<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/4115" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/4115.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/4115.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/4115.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/12/4115/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/4115.atom" />
<meta name="HW.pisa" content="ggg;9/12/4115" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.119.400451" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Liu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide association studies (GWAS) have gained central importance for the identification of candidate loci underlying complex traits. Single nucleotide polymorphism (SNP) markers are mostly used as genetic variants for the analysis of genotype-phenotype associations in populations, but closely linked SNPs that are grouped into haplotypes are also exploited. The benefit of haplotype-based GWAS approaches vs. SNP-based approaches is still under debate because SNPs in high linkage disequilibrium provide redundant information. To overcome some constraints of the commonly-used haplotype-based GWAS in which only consecutive SNPs are considered for haplotype construction, we propose a new method called functional haplotype-based GWAS (FH GWAS). FH GWAS is featured by combining SNPs into haplotypes based on the additive and epistatic effects among SNPs. Such haplotypes were termed functional haplotypes (FH). As shown by simulation studies, the FH GWAS approach clearly outperformed the SNP-based approach unless the minor allele frequency of the SNPs making up the haplotypes is low and the linkage disequilibrium between them is high. Applying FH GWAS for the trait flowering time in a large Arabidopsis thaliana population with whole-genome sequencing data revealed its potential empirically. FH GWAS identified all candidate regions which were detected in SNP-based and two other haplotype-based GWAS approaches. In addition, a novel region on chromosome 4 was solely detected by FH GWAS. Thus both the results of our simulation and empirical studies demonstrate that FH GWAS is a promising method and superior to the SNP-based approach even if almost complete genotype information is available." />
<meta name="DC.Contributor" content="Fang Liu" />
<meta name="DC.Contributor" content="Renate H. Schmidt" />
<meta name="DC.Contributor" content="Jochen C. Reif" />
<meta name="DC.Contributor" content="Yong Jiang" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association studies (GWAS) have gained central importance for the identification of candidate loci underlying complex traits. Single nucleotide polymorphism (SNP) markers are mostly used as genetic variants for the analysis of genotype-phenotype associations in populations, but closely linked SNPs that are grouped into haplotypes are also exploited. The benefit of haplotype-based GWAS approaches &lt;i&gt;vs.&lt;/i&gt; SNP-based approaches is still under debate because SNPs in high linkage disequilibrium provide redundant information. To overcome some constraints of the commonly-used haplotype-based GWAS in which only consecutive SNPs are considered for haplotype construction, we propose a new method called functional haplotype-based GWAS (FH GWAS). FH GWAS is featured by combining SNPs into haplotypes based on the additive and epistatic effects among SNPs. Such haplotypes were termed functional haplotypes (FH). As shown by simulation studies, the FH GWAS approach clearly outperformed the SNP-based approach unless the minor allele frequency of the SNPs making up the haplotypes is low and the linkage disequilibrium between them is high. Applying FH GWAS for the trait flowering time in a large &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; population with whole-genome sequencing data revealed its potential empirically. FH GWAS identified all candidate regions which were detected in SNP-based and two other haplotype-based GWAS approaches. In addition, a novel region on chromosome 4 was solely detected by FH GWAS. Thus both the results of our simulation and empirical studies demonstrate that FH GWAS is a promising method and superior to the SNP-based approach even if almost complete genotype information is available.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/4115" />
<meta name="citation_id" content="9/12/4115" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/4115" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/4115.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/4115.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/4115.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400451" />
<meta name="citation_pmid" content="31604824" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="4115" />
<meta name="citation_lastpage" content="4126" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Fang Liu" />
<meta name="citation_author_institution" content="Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3774-552X" />
<meta name="citation_author" content="Renate H. Schmidt" />
<meta name="citation_author_institution" content="Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8037-3581" />
<meta name="citation_author" content="Jochen C. Reif" />
<meta name="citation_author_institution" content="Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany" />
<meta name="citation_author_email" content="reif@ipk-gatersleben.de" />
<meta name="citation_author_email" content="jiang@ipk-gatersleben.de" />
<meta name="citation_author" content="Yong Jiang" />
<meta name="citation_author_institution" content="Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany" />
<meta name="citation_author_email" content="reif@ipk-gatersleben.de" />
<meta name="citation_author_email" content="jiang@ipk-gatersleben.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2824-677X" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Abo;citation_author=S. Knight;citation_author=J. Wong;citation_author=A. Cox;citation_author=N. J. Camp;citation_title=hapConstructor: automatic construction and testing of haplotypes in a Monte Carlo framework.;citation_volume=24;citation_year=2008;citation_pmid=18653522;citation_doi=10.1093/bioinformatics/btn359" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. C. Anderson;citation_author=J. Novembre;citation_title=Finding haplotype block boundaries by using the minimum-description-length principle.;citation_pages=336-354;citation_volume=73;citation_year=2003;citation_issue=2;citation_pmid=12858289;citation_doi=10.1086/377106" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.;citation_pages=796-815;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130711;citation_doi=10.1038/35048692" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. C. Barrett;citation_author=B. Fry;citation_author=J. Maller;citation_author=M. J. Daly;citation_title=Haploview: analysis and visualization of LD and haplotype maps.;citation_volume=21;citation_year=2005;citation_pmid=15297300;citation_doi=10.1093/bioinformatics/bth457" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate - a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Brachi;citation_author=G. P. Morris;citation_author=J. O. Borevitz;citation_title=Genome-wide association studies in plants: the missing heritability is in the field.;citation_pages=232-232;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=22035733;citation_doi=10.1186/gb-2011-12-10-232" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Haplotype phasing: existing methods and new developments.;citation_pages=703-714;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21921926;citation_doi=10.1038/nrg3054" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. R. Cardon;citation_author=G. R. Abecasis;citation_title=Using haplotype blocks to map human complex trait loci.;citation_pages=135-140;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12615007;citation_doi=10.1016/S0168-9525(03)00022-2" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=O. Carlborg;citation_author=C. S. Haley;citation_title=Epistasis: too often neglected in complex trait studies?;citation_pages=618-625;citation_volume=5;citation_year=2004;citation_issue=8;citation_pmid=15266344;citation_doi=10.1038/nrg1407" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=C. Y. Cheng;citation_author=V. Krishnakumar;citation_author=A. P. Chan;citation_author=F. Thibaud-Nissen;citation_author=S. Schobel;citation_title=Araport11: a complete reannotation of the Arabidopsis thaliana reference genome.;citation_pages=789-804;citation_volume=89;citation_year=2017;citation_pmid=27862469;citation_doi=10.1111/tpj.13415" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=A. G. Clark;citation_title=The role of haplotypes in candidate gene studies.;citation_pages=321-333;citation_volume=27;citation_year=2004;citation_issue=4;citation_pmid=15368617;citation_doi=10.1002/gepi.20025" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=D. V. Conti;citation_author=W. J. Gauderman;citation_title=SNPs, haplotypes, and model selection in a candidate gene region: the SIMPle analysis for multilocus data.;citation_pages=429-441;citation_volume=27;citation_year=2004;citation_issue=4;citation_pmid=15543635;citation_doi=10.1002/gepi.20039" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. Corbesier;citation_author=C. Vincent;citation_author=S. H. Jang;citation_author=F. Fornara;citation_author=Q. Z. Fan;citation_title=FT protein movement contributes to long-distance signaling in floral induction of Arabidopsis.;citation_volume=316;citation_year=2007;citation_pmid=17446353;citation_doi=10.1126/science.1141752" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=J. Y. Dai;citation_author=M. Leblanc;citation_author=N. L. Smith;citation_author=B. Psaty;citation_author=C. Kooperberg;citation_title=SHARE: an adaptive algorithm to select the most informative set of SNPs for candidate genetic association.;citation_volume=10;citation_year=2009;citation_pmid=19605740;citation_doi=10.1093/biostatistics/kxp023" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=O. Delaneau;citation_author=J. Marchini;citation_author=J. F. Zagury;citation_title=A linear complexity phasing method for thousands of genomes.;citation_pages=179-181;citation_volume=9;citation_year=2011;citation_pmid=22138821;citation_doi=10.1038/nmeth.1785" />
<meta name="citation_reference" content="citation_title=Applied regression analysis;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=O. J. Dunn;citation_title=Multiple Comparisons among Means.;citation_pages=52-64;citation_volume=56;citation_year=1961;citation_doi=10.1080/01621459.1961.10482090" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. Durrant;citation_author=K. T. Zondervan;citation_author=L. R. Cardon;citation_author=S. Hunt;citation_author=P. Deloukas;citation_title=Linkage disequilibrium mapping via cladistic analysis of single-nucleotide polymorphism haplotypes.;citation_pages=35-43;citation_volume=75;citation_year=2004;citation_issue=1;citation_pmid=15148658;citation_doi=10.1086/422174" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. B. Gabriel;citation_author=S. F. Schaffner;citation_author=H. Nguyen;citation_author=J. M. Moore;citation_author=J. Roy;citation_title=The structure of haplotype blocks in the human genome.;citation_volume=296;citation_year=2002;citation_pmid=12029063;citation_doi=10.1126/science.1069424" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. B. Gjuvsland;citation_author=B. J. Hayes;citation_author=S. W. Omholt;citation_author=O. Carlborg;citation_title=Statistical epistasis is a generic feature of gene regulatory networks.;citation_volume=175;citation_year=2007;citation_pmid=17028346;citation_doi=10.1534/genetics.106.058859" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations.;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variances and covariances of squared linkage disequilibria in finite populations.;citation_pages=54-78;citation_volume=33;citation_year=1988;citation_issue=1;citation_pmid=3376052;citation_doi=10.1016/0040-5809(88)90004-4" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=B. E. Huang;citation_author=C. I. Amos;citation_author=D. Y. Lin;citation_title=Detecting haplotype effects in genomewide association studies.;citation_pages=803-812;citation_volume=31;citation_year=2007;citation_issue=8;citation_pmid=17549762;citation_doi=10.1002/gepi.20242" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Q. Huo;citation_author=S. H. Wei;citation_author=K. J. Bradford;citation_title=DELAY OF GERMINATION1 (DOG1) regulates both seed dormancy and flowering time through microRNA pathways.;citation_volume=113;citation_year=2016;citation_pmid=27035986;citation_doi=10.1073/pnas.1600558113" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=Y. Imura;citation_author=Y. Kobayashi;citation_author=S. Yamamoto;citation_author=M. Furutani;citation_author=M. Tasaka;citation_title=CRYPTIC PRECOCIOUS/MED12 is a novel flowering regulator with multiple target steps in Arabidopsis.;citation_volume=53;citation_year=2012;citation_pmid=22247249;citation_doi=10.1093/pcp/pcs002" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. Jiang;citation_author=R. H. Schmidt;citation_author=J. C. Reif;citation_title=Haplotype-based genome-wide prediction models exploit local epistatic interactions among markers.;citation_volume=8;citation_year=2018;citation_pmid=29549092;citation_doi=10.1534/g3.117.300548" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. Kardailsky;citation_author=V. K. Shukla;citation_author=J. H. Ahn;citation_author=N. Dagenais;citation_author=S. K. Christensen;citation_title=Activation Tagging of the Floral Inducer FT;citation_pages=1962-1965;citation_volume=286;citation_year=1999;citation_issue=5446;citation_pmid=10583961;citation_doi=10.1126/science.286.5446.1962" />
<meta name="citation_reference" content="citation_journal_title=BMC medical genetics [electronic resource];citation_journal_abbrev=BMC Med Genet;citation_author=S. Knuppel;citation_author=J. Esparza-Gordillo;citation_author=I. Marenholz;citation_author=H. G. Holzhutter;citation_author=A. Bauerfeind;citation_title=Multi-locus stepwise regression: a haplotype-based algorithm for finding genetic associations applied to atopic dermatitis.;citation_pages=8-8;citation_volume=13;citation_year=2012;citation_pmid=22284537;citation_doi=10.1186/1471-2350-13-8" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. M. Laramie;citation_author=J. B. Wilk;citation_author=A. L. Destefano;citation_author=R. H. Myers;citation_title=HaploBuild: an algorithm to construct non-contiguous associated haplotypes in family based genetic studies.;citation_volume=23;citation_year=2007;citation_pmid=17586827;citation_doi=10.1093/bioinformatics/btm316" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=P. Li;citation_author=D. Filiault;citation_author=M. S. Box;citation_author=E. Kerdaffrec;citation_author=C. Van Oosterhout;citation_title=Multiple FLC haplotypes defined by independent cis-regulatory variation underpin life history diversity in Arabidopsis thaliana.;citation_volume=28;citation_year=2014;citation_pmid=25035417;citation_doi=10.1101/gad.245993.114" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Lin;citation_author=A. Chakravarti;citation_author=D. J. Cutler;citation_title=Exhaustive allelic transmission disequilibrium tests as a new approach to genome-wide association studies.;citation_pages=1181-1188;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15502828;citation_doi=10.1038/ng1457" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lippert;citation_author=J. Listgarten;citation_author=Y. Liu;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_title=FaST linear mixed models for genome-wide association studies.;citation_pages=833-835;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21892150;citation_doi=10.1038/nmeth.1681" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. D. Long;citation_author=C. H. Langley;citation_title=The Power of Association Studies to Detect the Contribution of Candidate Genetic Loci to Variation in Complex Traits;citation_pages=720-731;citation_volume=9;citation_year=1999;citation_issue=8;citation_pmid=10447507" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. J. Lorenz;citation_author=M. T. Hamblin;citation_author=J. L. Jannink;citation_title=Performance of single nucleotide polymorphisms versus haplotypes for genome-wide association analysis in barley.;citation_pages=e14079-e14079;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21124933;citation_doi=10.1371/journal.pone.0014079" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. C. Mackay;citation_title=Epistasis and quantitative traits: using model organisms to study gene-gene interactions.;citation_pages=22-33;citation_volume=15;citation_year=2014;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. I. McCarthy;citation_author=G. R. Abecasis;citation_author=L. R. Cardon;citation_author=D. B. Goldstein;citation_author=J. Little;citation_title=Genome-wide association studies for complex traits: consensus, uncertainty and challenges.;citation_pages=356-369;citation_volume=9;citation_year=2008;citation_issue=5;citation_pmid=18398418;citation_doi=10.1038/nrg2344" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. H. E. Meuwissen;citation_author=M. E. Goddard;citation_title=Fine mapping of quantitative trait loci using linkage disequilibria with closely linked marker loci.;citation_pages=421-430;citation_volume=155;citation_year=2000;citation_pmid=10790414" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=S. D. Michaels;citation_author=R. M. Amasino;citation_title=FLOWERING LOCUS C Encodes a Novel MADS Domain Protein That Acts as a Repressor of Flowering;citation_pages=949-956;citation_volume=11;citation_year=1999;citation_issue=5;citation_pmid=10330478;citation_doi=10.1105/tpc.11.5.949" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=J. H. Moore;citation_author=S. M. Williams;citation_title=Traversing the conceptual divide between biological and statistical epistasis: systems biology and a more modern synthesis.;citation_pages=637-646;citation_volume=27;citation_year=2005;citation_issue=6;citation_pmid=15892116;citation_doi=10.1002/bies.20236" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=R. W. Morris;citation_author=N. L. Kaplan;citation_title=On the advantage of haplotype analysis in the presence of multiple disease susceptibility alleles.;citation_pages=221-233;citation_volume=23;citation_year=2002;citation_issue=3;citation_pmid=12384975;citation_doi=10.1002/gepi.10200" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=J. E. Pryce;citation_author=S. Bolormaa;citation_author=A. J. Chamberlain;citation_author=P. J. Bowman;citation_author=K. Savin;citation_title=A validated genome-wide association study in 2 dairy cattle breeds for milk production and fertility traits using variable length haplotypes.;citation_pages=3331-3345;citation_volume=93;citation_year=2010;citation_issue=7;citation_pmid=20630249;citation_doi=10.3168/jds.2009-2893" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. C. Reif;citation_author=A. E. Melchinger;citation_author=M. Frisch;citation_title=Genetical and Mathematical Properties of Similarity and Dissimilarity Coefficients Applied in Plant Breeding and Seed Bank Management;citation_pages=1-7;citation_volume=45;citation_year=2005;citation_issue=1;citation_doi=10.2135/cropsci2004.0517" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=D. J. Schaid;citation_title=Evaluating associations of haplotypes with traits.;citation_pages=348-364;citation_volume=27;citation_year=2004;citation_issue=4;citation_pmid=15543638;citation_doi=10.1002/gepi.20037" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating the dimension of a model.;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. B. Sung;citation_author=R. M. Amasino;citation_title=Vernalization in Arabidopsis thaliana is mediated by the PHD finger protein VIN3.;citation_pages=159-164;citation_volume=427;citation_year=2004;citation_issue=6970;citation_pmid=14712276;citation_doi=10.1038/nature02195" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_title=1,135 Genomes reveal the global pattern of polymorphism in Arabidopsis thaliana.;citation_pages=481-491;citation_volume=166;citation_year=2016;citation_pmid=27293186;citation_doi=10.1016/j.cell.2016.05.063" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the Lasso.;citation_pages=267-288;citation_volume=58;citation_year=1996;citation_doi=10.18637/jss.v033.i01" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. A. Trégouët;citation_author=I. R. Konig;citation_author=J. Erdmann;citation_author=A. Munteanu;citation_author=P. S. Braund;citation_title=Genome-wide haplotype association study identifies the SLC22A3-LPAL2-LPA gene cluster as a risk locus for coronary artery disease.;citation_pages=283-285;citation_volume=41;citation_year=2009;citation_issue=3;citation_pmid=19198611;citation_doi=10.1038/ng.314" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=Y. Yang;citation_author=S. S. Li;citation_author=J. W. Chien;citation_author=J. Andriesen;citation_author=L. P. Zhao;citation_title=A systematic search for SNPs/haplotypes associated with disease phenotypes using a haplotype-based stepwise procedure.;citation_pages=90-90;citation_volume=9;citation_year=2008;citation_pmid=19102730;citation_doi=10.1186/1471-2156-9-90" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Z. X. Yu;citation_author=D. J. Schaid;citation_title=Sequential haplotype scan methods for association analysis.;citation_pages=553-564;citation_volume=31;citation_year=2007;citation_issue=6;citation_pmid=17487883;citation_doi=10.1002/gepi.20228" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Zhang;citation_author=M. H. Deng;citation_author=T. Chen;citation_author=M. S. Waterman;citation_author=F. Z. Sun;citation_title=A dynamic programming algorithm for haplotype block partitioning.;citation_volume=99;citation_year=2002;citation_pmid=12032283;citation_doi=10.1073/pnas.102186799" />
<meta name="citation_reference" content="citation_journal_title=Pharmacogenomics;citation_journal_abbrev=Pharmacogenomics;citation_author=H. Y. Zhao;citation_author=R. Pfeiffer;citation_author=M. H. Gail;citation_title=Haplotype analysis in population genetics and association studies.;citation_pages=171-178;citation_volume=4;citation_year=2003;citation_issue=2;citation_pmid=12605551;citation_doi=10.1517/phgs.4.2.171.22636" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association studies (GWAS) have gained central importance for the identification of candidate loci underlying complex traits. Single nucleotide polymorphism (SNP) markers are mostly used as genetic variants for the analysis of genotype-phenotype associations in populations, but closely linked SNPs that are grouped into haplotypes are also exploited. The benefit of haplotype-based GWAS approaches vs. SNP-based approaches is still under debate because SNPs in high linkage disequilibrium provide redundant information. To overcome some constraints of the commonly-used haplotype-based GWAS in which only consecutive SNPs are considered for haplotype construction, we propose a new method called functional haplotype-based GWAS (FH GWAS). FH GWAS is featured by combining SNPs into haplotypes based on the additive and epistatic effects among SNPs. Such haplotypes were termed functional haplotypes (FH). As shown by simulation studies, the FH GWAS approach clearly outperformed the SNP-based approach unless the minor allele frequency of the SNPs making up the haplotypes is low and the linkage disequilibrium between them is high. Applying FH GWAS for the trait flowering time in a large Arabidopsis thaliana population with whole-genome sequencing data revealed its potential empirically. FH GWAS identified all candidate regions which were detected in SNP-based and two other haplotype-based GWAS approaches. In addition, a novel region on chromosome 4 was solely detected by FH GWAS. Thus both the results of our simulation and empirical studies demonstrate that FH GWAS is a promising method and superior to the SNP-based approach even if almost complete genotype information is available." />
<meta name="og-title" property="og:title" content="Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/4115" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association studies (GWAS) have gained central importance for the identification of candidate loci underlying complex traits. Single nucleotide polymorphism (SNP) markers are mostly used as genetic variants for the analysis of genotype-phenotype associations in populations, but closely linked SNPs that are grouped into haplotypes are also exploited. The benefit of haplotype-based GWAS approaches vs. SNP-based approaches is still under debate because SNPs in high linkage disequilibrium provide redundant information. To overcome some constraints of the commonly-used haplotype-based GWAS in which only consecutive SNPs are considered for haplotype construction, we propose a new method called functional haplotype-based GWAS (FH GWAS). FH GWAS is featured by combining SNPs into haplotypes based on the additive and epistatic effects among SNPs. Such haplotypes were termed functional haplotypes (FH). As shown by simulation studies, the FH GWAS approach clearly outperformed the SNP-based approach unless the minor allele frequency of the SNPs making up the haplotypes is low and the linkage disequilibrium between them is high. Applying FH GWAS for the trait flowering time in a large Arabidopsis thaliana population with whole-genome sequencing data revealed its potential empirically. FH GWAS identified all candidate regions which were detected in SNP-based and two other haplotype-based GWAS approaches. In addition, a novel region on chromosome 4 was solely detected by FH GWAS. Thus both the results of our simulation and empirical studies demonstrate that FH GWAS is a promising method and superior to the SNP-based approach even if almost complete genotype information is available." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"iKls_DWC-_P021aT04AOJ_juRO25LmycXUz1yhGQr1g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494990","apath":"\/ggg\/9\/12\/4115.atom","pisa":"ggg;9\/12\/4115","ac":{"\/ggg\/9\/12\/4115.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/4115.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4115"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4115"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/4115":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Co6r0AFGdJgK528uoW6gS8KGXx5SjXn20WHEn0wCvXyZ72sGmsP6o53XjRuEXZXjuRTwEHzV6surmDVE+7PVkCBk9nSRsw14h2sYnpbFBln6KmH38Wi2TocHdzMKNoL2bCt7v7Mz1gVd3I695zhmQbBO9Y0n6o4Xq6IXGHcyFy6KIHEB8ANnsBwZ38dFGV38n0P8o8qtMZNinJK1WcgNaMmk177K658ldXaDxYDArmtnT4NP1hpUtrQ1mP74kDMBT5wtsfNYQsGZ+y4GbOGVDmv\\\\\\\/o0oSLsZGot8W5A18ecLSLlekCRoe6mCXUvRU\\\\\\\/6AKm45ME\\\\\\\/+XMZ9\\\\\\\/4V4qYvl4jwTmlC2Sb17TqG9d\\\\\\\/X\\\\\\\/72upZbcAaAupoTBg9tLMFLXOuym+1s0Icwxut33d4lWQCGRTWB7qgJ202uebcKfO\\\\\\\/7SE8FKdKzhhdcu5SlNEky0m41CjXVJ7thbaB47HZokdtENwL3NZANQMF8EHIcQ8mUgNkv1QIKSebhZiHjfg8P3ckjmnAPu68NCBmRmn4fe8Vwo4HAS8BkKtNbeEZ1lT\\\\\\\/jBaUKKgvI4uCrk+c3a5xI19i0ihTiIXJnSGeVUrDIOeBajHB\\\\\\\/N32F\\\\\\\/hCapuGQckZXbqL2Fh7wc7UucyuLPGC07tIzDpMNHnR1LQS5R6vUiyWy6qsPTP6XcbGeNjvXsPiX6Kns8tvPHEyd+vCSaMi8+rngFLfWFNiQFqHYwiyJCbXifNYvJtjSrkTGTtETHRqpoRJClXQrIyhV1L6FsH8fOd3ZB2t+cdX5AR9GHJ3QCFronHsYVZ3HskmngB1pXpLxKeasZrHyPrccoaY\\\\\\\/Unz1S6qYusFMn0Le8kaOiRe9Kploa4get3Sfg6hzTiuFQhdH6lJfcCK0\\\\\\\/jn2bWhRVeQEvxB9\\\\\\\/amDQKqqDmh3e7VOmE55wSEbIeJfrC4XZL5xQD21AZkoe74frPz4LPTu5RguDQq0yr2hkQeZvD15WkuYoiHeMW6FB286mjUuOgEMAqHHBu+Ib+vr9SJk2D3roys4kJC5JNtNL9SMcNhKgxkSYGZQvFyuoU7YDa9CNBv1OdAPEgwBGuBmEnDFZX38Y9HE2zfWM1M3ztAL8s34G2GLiS9ZOFdD1tINnXQRLHsRx\\\\\\\/oMZ4xeJ3PkGfsfivlXUSs48TCkNWUgSLdmUdnrK+OIX\\\\\\\/kWAuJvNVYlbp2UKPOAaHYC4A3\\\\\\\/ypXPP+4tqYxW3sRhd8+AeQnP5wcyw4zJ5wRBBD+Xl4+PYQcFJ3L28dFIDSJJpkp49oq5YjK6R5H\\\\\\\/AlPB1xX\\\u0022,\\\u0022iv\\\u0022:\\\u00220R72lsjCfX6A3Ek785jPPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c6aa96eef0dba1810c9ff46a3d32212f5a440cc33da1a9f1c4c07ea296b03ba0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KaCUD1wrIVeg4PnKqNGBdO3\\\\\\\/2uhTqLoIG3UlFHEcXJLZGSfVV85IoONpIe3glmCbt93c9sYcrj\\\\\\\/LhsIcAcuwYQuCPfu3Yy6qV3+s71cpko2+bhA+6JMZ0GPGXrG\\\\\\\/TOSte4xfc758uINDInZHr4WdLybtnkgcvY9DP2oE6yPDQPLsse+Kidb2G6XlI7iguWdr95W5J5q4HKLs42anU6rpe\\\\\\\/WAK8Y3xxskEZQXNXKq8a3KYjqApX40r1O1lnkBuVkJpN4xS+M6GslHP7zVm\\\\\\\/PuUb9JpZMFLnkPJIJbWLbWBN\\\\\\\/7AK36Kum+JD1KrTrI90g6YAFoXei94TugWM+mJQQq7KklX9S8L1JzgKjyVuVMZ0bg7x36v3RBNQ8ZjUriIldt53eMZEwi2gOCPOZaGv6ks3BzNG3Fjg+s8Uof9\\\\\\\/z7VoWKHLR\\\\\\\/AMGdGQMX5rn2hF\\\\\\\/qhsG5fWWyko5vdErJ3OgNQzY0shgEjZBpz45wD6hnejT2bP1L2yjnUDVpFsnmifsVBo\\\\\\\/OfluJvDhTe5mf+gjsYfSSEe6U+hfsL4cecjbWVUoFCfxKjTMHSKdRv1HnE+Bmb9bT\\\\\\\/lDL3oHgXDObFgFOtY31PSnt1c6KlRE13VOkMcgrxVBZ3CVMQOwAVuscYi6XCviaSWOOtKwSf86y0EQcRGSPbjQ6N3Ul14ObiihvcXAehMOZHcI1hbHs+qvpPa+EPYvXOxZcVBgf9vqZ+f95dq21WZYOjp3mLiKPq2\\\\\\\/My5zsyQVxpTutvaziMEPap+195GzhOJL5jPP9bO8pAZC3mFLYCxlwOupm4tao4vh6WwRy7WjYdnEcjNPJ1I1xhaMH9TMp1EaR5C\\\\\\\/+LFjLkqiE\\\\\\\/cLXGkM9lwppNkno3JDZR\\\\\\\/6rX0S9eXJahiWrrvXrqnY4KB5gD5wvx5ON81MDO1R5qP+zUjiaecFPvcxoaZ2oF5lkgYiWJOx3Gdh1uxKViHp98takTYKI9ecMjy3kmuo2J792iCO21ACpnjX+\\\\\\\/jsPKPBlsnSmHBH31HMn259R2P7C7CgEfGHN8joPaBPKAr\\\\\\\/\\\\\\\/Y4J1zfkNo7hLYKVjvcoOT63F55cPmal7Wr5JVC4YB\\\\\\\/C5bf7DnEAmyASbCgr2WV6ZKdkuFycikgUKwbmE\\\\\\\/Szf+n6Y7SWVJllDnGoIjTUbowWzAvLqmTSGb\\\\\\\/VXt6fLzsts5Uamkg0El\\\\\\\/R1TvS3dkOFfBWylQ3WlXTW0I22lPVP7lcU3s2MeoioWLorzDuGLnaF0stpkpTq4cPNNntOl\\\\\\\/fuP6IsfXZ0ena85QQZ2U\\\\\\\/zt5i0o9oqLT3HsNIJ9AtJeohscRtmcsQITRoIfGZ4Jgnl+uP2etFliFUUXLSOjbnL3Sr92WFV05ldubSMabzPIHSxNyMSZyV3qdCX24TPLIyCGXjof15GqGMlg+nvaphkE5ZmTkuiDw8yBK7H3h02w1VujfgqnHS+1hPg5IQX1s3K2IcuNPaHf\\\\\\\/zNU7KdBNGEVxyRLjSYpt9i+SgTc8Pxeb+n\\\\\\\/StUaoTMC7fUtF15291jx0Ej0vRdoxY+yF9zLwnzxQpqKL1M2DNOPFpWUmfUHUfO16W+5BTJjLFJ7X+PuRegmnyxKKvA5yiZdXK2RyCKlQnVbEuruCKUBda2rLMHnQb3GEKI+7Xy698SuH+xOdcbp0LatXC9zTElvg3PQZOIm0E0UnFJHDkjc1q341fMAtOAsEnqr6CRzYw5cnAvwxoM0UC86anJzQyFrxYwpPQwGDNVfjS50+1QhzB8sB9a\\\\\\\/ID\\\\\\\/dUmNyvsl59hqZx+SCAfPmOOo2MKlK6CLQizEI9LWTSvQX8C0lcJUX\\\\\\\/bezZZrCJXzAUbLh01rp36r8wKGXYNfrpUfTkTczpjtSdR9RDFILWvhXPJQozlDbdCAIfjn6IXFnFwGNZq2ZeR4ChQc7HxgLNLmDOuYPQmlfHYljYfLl9yEzuv14wP8vyM365asAqiSmYazVwYG158XqY5bIlHOmNZ3Y0Hh6q2EFn+4B+zgD\\\\\\\/nh+wFPThk8W0FEv53BmmhwXQcCatTh8z\\\\\\\/lO21HpWFcOHB2mzO+Qv1WBApyMibaZmBemtqMy\\\\\\\/2xeEJSyHSUh0lzgv00v99G3nsbznS338JbqSTUKZmz\\\\\\\/muPmDl0hUefbCvAGem\\\\\\\/3jpMrIS8ia1Wln1iXNq88BQez0z+oceny6KVm2g2rkvUwEmv15c7jci99OJ9OTQs36lN3cCOSgEjYVhun6GbNE80+nMB\\\\\\\/NB0BYXv+2dsAN4C2+9KLzPJxe9C2KyHz9xu1WrQoT0c4WoQex1r1kaZKH5VWArstkiG+vobXQqzmQTpc72bRq95uiyixfvUUosREV7zQT3nnabO9Rxr0SE\\\\\\\/i8UIA6hct2XUA3hDQc6alBtzzSv5tow\\\\\\\/6JMNfzr4HwKZ1LA\\\\\\\/NKUrFeM9jehcRGXVpoPjr6tTS2e5fzMbZ9mSF+RAeMNOqMrmV7sjAwnyL6YfDZx+V3a3MK+hVCLaZOslgatYSij\\\\\\\/hMjoLElg2IpqQqsC2OpDgTZ1jm8KrIH0iPkJ4CSdvdQCkLI3NpzLXDtij69F7gLjATbPQZSB1PXjVoliMTm61v66eftFaalOhOj3u3+\\\\\\\/YIq3bIChuzASw1\\\\\\\/1EO8QtWv34cfVakxn88N4rhGG46VMk23+f32YpC0aVZzTkbR5uw9piLhMHEAX\\\\\\\/YGp0AWYu3qyb0qqCIjn7uApFbFzN2jdmCQK+lvUUE8ZFwxT+dUDMytOuDvVJdgjpFsKR3bsLVZ1Cr6WhRN5SYLBsIYQiqn65icpXTy2Fj7ovKY0ucT6H3boo5nU2Qz5fD0vdFp7h0diR0g\\\\\\\/YcHTqMDD3TkXPUqnNOkoNXhikeBltsnwCUDdlf0GH4lTLxhebwZkRMwMLBMg3uMciNJ6dgDV5Gm1+GS874KNlMkR3phhhi3mIeGmooijucgi1aPSa\\\\\\\/hX4vgCjfszS977FM6hC8+UFgj+UNA\\\\\\\/BzlLthJvF54p2EaKI2FL9RO8Rg0DtLrwchlwHA0SdPGDzasq9Ahl9T8OQ\\\\\\\/lk9Qt1yZWq6iYYJOQh7aTfanE0j0vdtFEtu9tAlZPxQ5qwuuqIYkCDr5WwvXQu95l+hvsQzH1xiWys3lCy1IHkJJGrRINY+mSeiWmhMDyoZTmRyslRmBx1\\\\\\\/TtsTxAqJItJciUICQltUzGiEFuwiO\\\\\\\/\\\\\\\/kCl9KHAvpI62lcUWnj0paDcFifCGR2zuU1DsTi2Wjm1eM\\\\\\\/09oP1S8f1LVdC+HogkDSVVDOV66f6zY4V\\\\\\\/5iv0IvIBGHLlfMrT\\\\\\\/nqNcdJryccw5Hj0TiuaOa2oIoGccL2JX\\\\\\\/GSSZAHSC9DPZySiMnpI8Kten1y7I3IpncH3aj\\\\\\\/4X8eozxBTIIyHye8JfXD849zqqAufbOLxKCS70nJalLk22neiTOVgXThoi4l7dSOfuPcP7H\\\\\\\/XQ20gIom\\\\\\\/Masy+Z5849a\\\\\\\/hwhFOxR+hwQeBfRMj83JYoeRFqTBQY2gwUdAYBxcWRWf7lKbLHpHGiE\\\\\\\/AlPJaN\\\\\\\/PZvhp2Ds4V4GOER7aNmSo0ZpixQB6VjtC53KVs3FpT8zIUyItp6T55POosGkhCCta+IFFFQziQsuHdtPbKnfGo9KQL1i+lJnC\\\\\\\/F0D6suv8a7pv6C5U6BKGUKpu7GUiwdxWdYF16\\\\\\\/aTUhZOdkl2OnWHVUQb7j19Pu5o\\\\\\\/EYG0j9oqLepHPWSltVvGEOnU3sARaFzAyfb\\\\\\\/owwrW\\\\\\\/yFdYs1HMjU79XUZITCj+RWPf0FXPOEWX+zeNkztN7PtD1WfZwm+3zcQ21+mJ28jjSLV91hBjPG\\\\\\\/iTbpASsUoTz8+QujlJgmQnZMC+PdY2WslNW4L\\\\\\\/xUrjEVlzbQBS+763NMhz55IFgl9hpi+dRIgJBnNbKWpVFGxjLaQgaJspWsX1B6zszq83UWFp4gHD0iO7tBGr4pZ4ZiyJcieL6v30nsQATQN1bLWNhgBI+5xBnGnZHczjeNfpXs1agGV7pwZSo2JZwA17HZ9Z2x5MSMi5QWEZRiK16+0B\\\\\\\/k0xCO6CaWtoBmV3PfZRmzTgd1GIOkLkaiYPHsnPysjBQWctvRxI+dozXp2p8isTviD2Tiut4sVsdi89T8ucT7T4MWudrV5XnPI7Gp1dlXeVMPBF8B+yMcDwI2TwVQFnUiJpAo63i9zrlrL30zS0hAKS1xHAmBOKLXtccSjqEoPqPb6dxJsi\\\\\\\/V5AijQSir0nO2vvQUOVhzM3r54zlwclwv\\\\\\\/PeDRzIQPoY2q+EThobQXsuuDuYecofhqN55lgRLp57sVm0xvvH264ocr\\\\\\\/RBIbjobzu5fNqkqWhUvt1k1FiGyVBwrwMsSrnXbO3LRLFth\\\\\\\/ok5Gk8OGI8fem9SXhjwOUT9IrDToquHgPyDaFlDFk2FJwNxjZLO03EmPHMplimPd9+YfIhYR5aTcYQN6giylMugZ+91z+9\\\\\\\/ker1+rRBQ68DTQ77SNIhJCqeMXbiVMXGZo7B5f7qbHLBdFWGTnnsfY8Xyv5R\\\\\\\/gcAaDhFoBZIvMp6Yq4jmrHRPFmPqg4VOcctv8zQh3d22NlO\\\\\\\/O5BmN5MRCIDVAHP4UtSpWfv+cWB3ufCTF1GyxOgWuT4aGllnl85a\\\\\\\/58zV7w65nFNLAcZSeNBvDHMHXJA8GhZwiBqwl30oyiJ5rV7qeZKjFE4izYQ7St+tlflV6T4y8JngtSxjYshPTNMoaF7mKOnTP+fb87ptC\\\\\\\/fyMbA8W5ReYGiI2rusejn8CJpQOwtooO5\\\\\\\/EEzsY2eQ2virPBS+DNh0B0bXr350trhV4Wp8rb8r3pDPBeznL1tboRqliCsvjySPPlqrWoD2tfpOtLhXmEc7TnGoFn1md8wxyPFAMz2hOqdbmGhPC4OGOgDsX4wPIJ+0X2W+OzLApmRzoI0I6k7L2TVwqfa7tXJwgzzSM9bz\\\\\\\/EW8F3bOzR4D\\\\\\\/RNLG8Y1iyONWPwk+pl27Cosw7IwQqwvl+zlWmpvjlaA+L4ubl2oX8ul0HzllBh8BupOaSbmfZswZ2yW7jhIedjkUsJAxR8EH87ovndFHkn7H4YUaj+GAD6hnnxeQo6D0XRJeBRInWto4K9wccZWnZ2onfO\\\\\\\/ZEwyOpaukfl879IQUBt7usaiumwJ8zl\\\\\\\/tqTldim2pyvn+QUg35Gc1lkIcPhGUzd3S+es5JmRTors+jvTRUVqcqTNtT\\\\\\\/IgJ+40RGas9KdzY2W9Q7GU0sJhR51YgIektrZulW+dU2g3b+5\\\\\\\/F+VKu0+LQyDCVf0ELzViDhdafTP9mT3M3VcLMKFV\\\\\\\/oEuCSv0G9ngPkppbkYP8NYYS6T4AuxTZUOC3drJ4mh0jns\\\\\\\/BygqWGD0+JpC4UwMvhY80jeu4TWQwvpfMe73nwvoKhsTrX4YDtpfIMTV1MxE1RJ8Ycgo3+Gq\\\\\\\/\\\\\\\/RnvS5yc0AF9wztlSC0WEEzvN8YPcNrp7roobTzMx0pdbxOlohDNjMDsbbvGvsWP1S64GyoisaQ\\\\\\\/W9I98aUYXr6mApUpjiwo1t4Az+vqZK30A2dxpPQ8fiB+\\\\\\\/uq3mbCNKAhZ0PXg66U8lAqYf5+aLXWNThKw0OHEV9qRYnEyqjuWoO8mkgwxvUOjvvvqHO1vpgGXby7o5lHguBHyqP+nhxEuY\\\\\\\/vuapsQ0DuI4DudrdJc0mxNhJ6+A4GSHQotNb1athnRIBRuSQCcTLXXViSCC23qxyWpQRXG+tn7D4Ve\\\\\\\/EzwQHCnaN8aMuEsZqarRhcMyz7aqKazfz2rVohb6T7yYhO3H9sXHztrmr\\\\\\\/aq\\\\\\\/RK\\\\\\\/Z6xgCovbj+hHU\\\\\\\/4W+s9JLs86aMqRPZqVYsVtinbxS7C+14U9JhjdcbeJgGumhR9138nUSQc+0335eBF4+p1i5Jp+0C\\\\\\\/ylAnTmMRrhPQSycrBOSxGAta8uCzlynirACyN0GEjx3oZTFAP5VnjVjbzXHKUY3tQZxCMP+zEEvZsq3WzoFCxwM9UaveGPNe0JVep8rsIOwqv87PaCT17K9lDLPGLmp\\\\\\\/SBFDpjLMzlKxLQzs4VhyX4FFKJDivHNxMtEUea0koXlO7dLjFWZUV\\\\\\\/nUCVJWegz\\\\\\\/2NuN0wnYqcAdCwZcVdKfsywwxqsQLk0o0STAe0N50HzC1eniZRnS9YxDQLEV3bE4Y5n2nhK1tVB+FRJRIt6NOt6BO6HuY5faSmhZQrSAz3twBjv67BpqMnHJ9nd3b4ZcqP\\\\\\\/xgntdQURkiRlY8GYIVp+ixDaO89jGXId96nRkrMky1qRqn0adIV5FHqt9G89aL8jsSjZRFQA+wlpsmfTuy4Ff+NMAT12SFuWhxUayZREii+728vhrvkKnMLaVbaRUgkP6kyEhljabcxls6RjzWabj2idunMjzDWhZySsqz0RejN33izBdSZFM758r5rxY1tAHlwR8mP1C0aTGwJ8elhJ7lPujzpnL\\\\\\\/IahnEMwLUmTSUH9P73RtIqa4+2YlvM+AWanaqcqUd9tJqtgOepDP08GaDM18HyWwXZLkArpgDa0y8k1lDHP73b6p1kQfTvVOb3cEHifEoROwQoBXGduR+1TfQuRonDhLskNp0l2P8fUcN33YPKHNJy0hE0oawJkQt1hCSk0Kfb+MA9pyTOsURcyAfdpV3iS1dceOdAY9UVC+eS0T7fMluYpFfKOBLKES4Jk8fnItB25zA2lrlyABwjICnmU+lo5XRbzCnG0aH7\\\\\\\/bIydUAVq9D9leU28A99f2lVu0TzxwPW0NOyii\\\\\\\/iufu10clXvXHbvZ\\\\\\\/5bOswGOgaFMiJRcbDAVlFuiT5IdjuEc0e1eEtKn5pmqTffVoD8S93bS984wTLFhLfsm54ytXh17LZWcFqMq0OGR+0G6neZz4OnQZdVuciYB1A8lndCvQhwPxeCyIfz3ZDO3DdGzZqoj\\\\\\\/kslaqKlDqTq5qG5dBFk3NJEa0rd6gNpTjZEAbR1DP8+a8ePrvLfFxt+tU\\\\\\\/gFHuNQoAKD6SeW6hEXWcxrD2ITSJfl7lc9lzjTuViKwABZPm3BG4qTMLr7zqmJKQ9xIW2KnmYKsoyUrwq0nNCd9HFofl4tU67vudmOkMiw1c3Jd0UFF8ORD4lgATTmM95bdXbQCjGT9uS3aiV1cTcLW6TvMMPht24ZeoT1gHC3fDYEaObc2aps93Dhtqm9mdAn3IoYdVevAsNxPOLmUWSBcnNav\\\\\\\/OVV4jn9CC3wGjWp\\\\\\\/817691+nnI6fJ1kH3MWhIZpUI\\\\\\\/oZAq56MGUogS+M9TffON5ILuwmfWZt0rBkXjJJ4omLy\\\\\\\/\\\\\\\/X9Tn6EXi\\\\\\\/xiOTnuQ1npbwDBDrBbjpwkQCquWf87pH1CALNOD10kaivyZmUiX01SuPnURKVTu7lpgxTjRJsssIsOLV6rMA1UmgeSLFlgGlD7QaDQaOXFa6KC8dEN2Pw0veDPETyWSTN+iRKk1uoMXrYGK9ysDGw1brZA4WuTJGNrdonyuFywvNPRmmM4y9M\\\\\\\/7PBry2wYADuIiM5nO9+26BdpTVQoLoh+DHpuDgNIzSmz7vnEtH8Oi3VOrAk9+qFHivViQlW4UWcOQ1X52cpHz\\\\\\\/RrEIQm3qqpaxonUNGotWZrVuxtEvfONgR4AFb8aGNpidqoM3SddIHFxeKIdbBL3dzkG3DnaI7P7Dwanyjp5wWLaNEXewKbEoAD\\\\\\\/slazGW8\\\\\\\/oGTqYX6QM9Ztk7PNEVOSWWZScO1L70R+Yc7kXN\\\\\\\/YfR87twEGJ7hm1ulMq9fEl+I\\\\\\\/n\\\\\\\/CzSjda6X65cz8skZzKLoj6qYVdbG9V8c5MnbejsO7OeXjKY6+I0GIKxc0XNcjs+0+eMBxyMfixsa8mXyPHf6xlaYwiIyeIWPGrSvZ7gQ\\\\\\\/sYA33MN87xRg+V4+XwyfE+thAWUxjOatf+s5yzzXDokyBwKZJkX+eWWc0tWRF9Y6iKfhWj3WZ8apT2bsDZx27JGnA9mU9l9qWtk82Vt2BGZDN0gF3Cgg1Whb2S09eILqGywLznhIDeUOaAET2fMjcRy4+R4p4eHvXsz8T6QQTOP\\\\\\\/TOaKo4+y2YXp1ho\\\\\\\/vd9aBDoUsdVsB7VwFQKLj6kBjb8i2uC8Q4KBR2B6E6gIzwfcFGMYWWOgKGKrmBb0vwixQ\\\\\\\/0EnbglvDZBEP20SVVx6xoPUoxhf9V5zTqOHT+lBtx7B+NRX6QxLJoo+duIJYmd4xZ7SlM4sYfe5tnM2wt2PifC3x3TNyxYzq\\\\\\\/1+ZkQsftIKpK6B4b31Qh9OE0GHgsvQGui1396nAh+jBUU9uetg4tvb\\\\\\\/zwkLZRzF\\\\\\\/JYKYNlPBiD6r4fW5aBwPBSR4wW8NtvMERVB2vwpPQiSTVW9s0Mkyyo9jW6Do8ZII+IS1oMRIojGmHeRbGiK+wxwOMaSFApLmBGlft0Q9SnXdtoaFOsebov\\\\\\\/Mg6fsfWFrmavnKI23Z9JScMPMuaqJW4yoeYnbvzYUEOaB+lBLzQm+C5XBUWzoA\\\\\\\/dRZrsDZ8rCPKPPxY2rNhWbNsir0ZlCVVLW9w37NJwVlbrczXPZAxrimhkGluQAEA2UCVbvIXsnfzG9DZMyMhM5s6TfN1bl55gjCB3\\\\\\\/+kaq\\\\\\\/DsKexkcnTBHOTgnFDOfNSdcJRxadjJFQanZSQVKbIoycInhtCdcEOhRyhbLaX3T9lBggjsQCZonYCoD30qdb8mfXMnDApWz81w3mrdVtOuOjb6AANv0o3\\\\\\\/yqNxNfe5RcmaOTOuzbukot9WqZ8Fr69EsB6I\\\\\\\/n9c3lkfb5Cg1f7qJXz00iUP6F9W4XVn9QbkyZ4dpZDHIbSqLBzDsDJYEw3h5Myu84yY\\\\\\\/FEpSbyJy2eXdzq5H8BEGlxsDiDxYnH2M9Bppj3TFqDy\\\\\\\/DUVSrD0Ep4p8LbOS\\\\\\\/mBKZnglChW+KRFg3RPKoU9Dqrc3YiWr0wDF42NDS1FEjcpJv5nhYMwNck\\\\\\\/54y+H5sTzL132edPmHlXBgaYQzkiLUwS6KstQtsbk0+U9RBOJ5tqpZyIW7c8GrmhhWiNk\\\\\\\/mXduFlQ+\\\\\\\/0RcDICJEq5TZ40QG5hFS97PC3qcEuzf3p6Dg4kswRqf0V3MXqepksHtG0Tucm7VBY+m91VtOI4hR7cQabq11629MKXPYHfikOw4UGUWgHCutrM2B5o0lMr5GrrXEHWZ0rS6vFw8h8s+CmCT3fp3NhaxAN2FbZxVnZaW4Ch+zbejWRNBz0VNYpPig6oD03VD1HLEUSFmPIcGKkadtVYkZ71F1vn3n5LzuNdTcnmMXC5EiPbN5YJ522i+faJpyFkHCKJlAOpWdB7GTqwv\\\\\\\/nVggSEazbo0tWuwAPGo0W7dkG4KF93i85iblNVI7oapygIW+HkoIl0A\\\\\\\/AI3+HFLC53H016Cphu2joJR9sZldjsGU5fHgt6dJtFbF6RnJOVI\\\\\\\/1BTdnJmbWDIa\\\\\\\/E5dbdmdBRHd6NScaemaKixYy+pvix\\\\\\\/mVWORwvSHn0IO0wXXkVGccmxrDGf2qKxLnIkMxqBqZLj6mY8XeaTDZX2gPMU8qm1m+U4sVDCWA4YqxxP50xAKpF7g34hFQLR7lxpzub3nM0tfPVgcHZ48Saqq16flC5pgGsMmAkBjC3GIBOsTNPe8B7HReX57FnVUeH9TjpcL919vHSRcW\\\\\\\/QikUxUuCQcFGqBdPSx+O+IYD6jIryNMNpyRjrSlRc7zvCnTbn4aeqsM1bNAKouoceKgpraJbOSgIeKDpEAov2nlKxG9DQQjXROcZ2pNji9lG0F1HOb4IxiQ\\\\\\\/DI9fG5RwOPk\\\\\\\/iSLP7AUWu1dL4ntO7p4rrfNU6Bq8rv7\\\\\\\/aBWkp7R6Vz4YFZ+O3kwlCTFX0SSka2Z7+saQM26QUYhg2TSQOR3DLnHaEgB5\\\\\\\/bG130nu0zM8iwUaMWgoSMG9X++CkLml9LaKHmJdaRfjcF7zgdvkQVprXIt3RDq+fNqHgAmrZm+Ino8kPi9vyqnkD471FHZ01cu5D6LMAQ55WOm9Q2JqFTRrfDi9IBtzbiXPZHUm9sifMzaej7skbQgai1cj7ux4W\\\\\\\/cNp0\\\\\\\/FULA5Mx0glycsLlmK+xRJQIgvHRBu8LyEHxIone3BFLvbHTYh66lOMYy4rDWbJF9nK\\\\\\\/vGZoXbNFGihBNzNvB\\\\\\\/07od7kwiYgPtLgDyuoLy2cQRkDXi4cPQqtIt4wO3bLdIeUxfZqw9xpWYdsurxzd3AAcra3ryjrYppfs9CftpIdPSNEOGba1BiHQIOJXg09EET9rs9CEL1B0yQypqUbLaUW1Nr\\\\\\\/HL7t+gADKBSutwtAAsAHRgO\\\\\\\/tjj0pTIEdD60j8plzucy4LuPePvHFWruf7w7vd+BdLGbW\\\\\\\/+9QCHNlT\\\\\\\/CBfJld4Tr9uTAQvsqGFSrqvhv\\\\\\\/TDuZksZluwUjahdI5dX968s1gJIEHWCNcPyiTFhTk8BsvW3BoSIDCa3umMnf\\\\\\\/eziYwepd6+RvpoybFITXwt9653wTDynb5ZkmPUe2PJyykIA6HcdS9aV2DOuhZiixHwRuAzFYI3Ak+t5\\\\\\\/pXJsDL3uJjlXp+lXdfS2cQ93El9c\\\\\\\/eTrsXT\\\\\\\/3MuBYhFFCMoZZSt0N4q3zeT2cSLp07BaRKipwVK1v21QGPCoAFO2w1xuh2gFg4TSeTsI7FqNWkn0mOsFoI\\\\\\\/m4yET+gJj\\\\\\\/AXV4Fv6GNtavBZyikUoulHqbpAm2tj2Tl6TdJNnOR9b\\\\\\\/L+CfKDwLGNLLeQuLqwg3uNvk7XeM6vzq2OU9\\\\\\\/z\\\\\\\/0fHuFElh1iyc+ZQMXfmBhovRaJoUMnxp4qD\\\\\\\/piN6rn\\\\\\\/Etel+p5tmjmHbmba11fmhoTBBWXfcFoUFE8iFrG3CduoHShmDhWQePHwZSE6d27xGjOfhzQpY8nLJe0KROY86LjhFLvwP6d6Z9P45BUw\\\\\\\/gd5\\\\\\\/pvP17Buh+qhqr1fkWJrSPOHKM1NrZpJmjDkZNh279pZ1rS80+zMGzsE+LUM1LlhDb4INA7uxQiVLuuTBhV\\\\\\\/F7w8nnbnxcn1wanqbyV7KBsgNUJkTHW8YvSzGqiFg5x\\\\\\\/4KTsp9070J6pqfW\\\\\\\/Z9uSUoeQOPXoxjGd9kI04l9oTS59Vf2HKOn+2KBUPQ0a5y+F4mvov+n1GwsOrz9BbsZuuJuLkYSKZH3+C+Ez01jmw84XIRmuPwh1hThbcE98URFI6TYfzYipOYLWOXm6PIby5xYPfQftMlAiZrfSt9fBlxYF0HrFifSraOIio8BSCcI9fFDWt+a2jrlCJAAXEBwuA5eJJez+T+Ix4gpviVxy1Vj9j1kI\\\\\\\/f7Ky2pLp8H7jlbWj0OiEo34NvZgkTV5hGK3dYqhnF7ULUPNYRpNNJJ97YVdAIm1X6ssnSx3VxQ+7L\\\\\\\/CKcwZh5Yqm17QA8mn6WcdCzjuQFoSQYQVYYhnXWBApNE1zsKQUoy4qWYxAPuXpeCihIIRhhto5Y13hauNY\\\\\\\/0CCnJUdlArS5Ri7xcykPkBYUKsiq\\\\\\\/pgMT+1m13Z+YgnGohPkc0Fuy+5sF6n8IhaMp8iTHbYlCFBfXh3vmN\\\\\\\/WVw86w\\\\\\\/fm4BvK2WaiVsRt70MwwEP9Y9iEmAzIgPG9ZidjHyzUYC0sNuznZwn0I0GJa959F5ZeNioN1koMqrOvDXkV5dLsK\\\\\\\/7v9mvDmyYeCu81\\\\\\\/G7fLZkM7vzZMkSoVeBtPsQH1v0z\\\\\\\/Ki9OePXn0web6IjhCG5vKKpJB08v6ng5zzINIlmG48vX6bruswA3lt+A34nmwJqJJkuvkr7JnHAmDn7rKWePSE5YphCzzPuy0807k7NBICPIrPtV8LriJ+xnmCSv9yIhDa2mHVp3RcN53kCLJ1VgHk+4Gb32BGQqkxuiySY8FjzIsOAibGMuAHk+krbo3X8i1vQn1brXYZVDZvdA\\\\\\\/4KIA5F\\\\\\\/BVV4eW6bFO8jSs8qTLTHowNFTSEpEhKSTHj2IsxhbqPaL4QLUCs5zovqW2RlsDHjDT7xIM6hs8DPuDcxHksXOHbQGN6Z3RVlJZS0jN9ycjPabEU8DCuLTDzGPp8\\\\\\\/nl7YBQAvaB\\\\\\\/8nmqlXlrCA4k4BEIQ3s6Z9mVRO4WYp7umBR\\\\\\\/lAxD\\\\\\\/H\\\\\\\/Pxo+ShAN8VNapmz5jfHqjxF7oj9yssPOvvLDACt2fORMsblXPNst0dMMINGlRim8AWZYHxVQBA\\\\\\\/KC5xJqCC2jV0P33PL52qq2HOwbDZGsW1uCvAHERL7Ty56j698Srs+GW\\\\\\\/450GVfve5WUfu\\\\\\\/K8A5tKWKHnPI70n4bDHOAsW7xhpjSB0YhKIq8N6twR\\\\\\\/FUS1Y0RBzBN4Pu4iDwCXO80KtXwxbik8OUEXmSW5Ie8e2D8U+HI8UFXE8\\\\\\\/Mmn5M\\\\\\\/xOOocyKDxNOQNxhXKEERH9nFUpJIwFbyC7cbtphIwy9oBRDk1To+FjKjak+hT3eBPQp4kxfH0IPVScsMOF4E7AVaZ6YuTdSQlNmGNeeQL6w9hr\\\\\\\/x7DrEDKNznVP53exbUDGKDSjN34YHTSXx6zm9rx\\\\\\\/99cvEj81MI43SFC8vP+Z2ZvWUkdKmPq1t93NWC97JujW8OsYpgovn0vdAz89kownZSE2VNE\\\\\\\/hJnhJLaW1nPz2FLeV2\\\\\\\/hOa5LhoYgwdWJcQL6y5EACI8Y9A6sEB1S\\\\\\\/BYhMfPLt0IJOL+H+kVu1l7CUo\\\\\\\/xI9Z6\\\\\\\/E2C\\\\\\\/28RMXI1akIQJSPsDjrxTYVEc9sSAXvZOVrkH8wyPEOC3X3frC7ggFFYU6dfJj4ZgOcb2BC1mGauXEF09Pq2gWp14mWWoTAy9jORXXI+mTT8wKwI69oiCnlSA7ebr6dYn03MvhtuZkkYWO7+zoVJxn3Czq11NJNaNrgnFDSb4P+jV2mAu4V8o5s3xc23a8I7klRwF7QUa4sQrcJWbgG2Lzbz5qNuVWgglzPGIMS6iMVnhqPaq7ZfEr5eKw41EI3nDVFnRomsfcyLcNY3AVrmYJVE0jM67IxY6yeJP52zqTOQRf6LBvb\\\\\\\/XUcKgpO9MEZF9vBNjYh3GBkiDhfGyL9WqzBAumOqICVHr6rwtk4l4i9vYHzRaMOY\\\\\\\/+IoEkONelIZDsoS5t7wFU7YZHBizQvRRYQP7pnmYu2scVvkjkgLWRGfbOW+rw+YKCILfqv430qEWJ\\\\\\\/vAybNW1D1YGiWB7H8eKwPqlIuWbHhfsd6iMKHvWqVsBTvu1Xghp9OBaAwbCxA8eciUnCO2ZAwFyV4F7j+otw2JNAAhmzb2DPkKZryePXziQ\\\\\\\/QSQGyWlGPQI0S3BibYlr8WLDwnLr+LBNq6s9taUc3CeaK41X5Gy27FcK39NYSP+jVLkaYbplcAkj1gnGNwwar8AqURGv0gj+CSljSBWdZbI3g82KIwu8RHS8zDq471+MeSpcTi0h2YBC\\\\\\\/V4xOnWo0UWUeXpi9C6xHAsOkC72NE77B+B4rX9a0Hu1oyLWhafTzq3AurAZ27Hq8a2iEiJFp+jLUHxFycTe596TarYu02QLjMkF5IzQeUAXey64ICAlQZ\\\\\\\/SpbvbKViiKtQfTzvagjkWNGo4L2tvt0ELX6kcUBU\\\\\\\/j9h7OAQMUJ7APZar1zmKogpKw5LbY9U3c+ZJod1fbbiD2QAq71JCVtk\\\\\\\/dFYfQ4G9tiaxcYbdwN4tXHGge9cIuzTZLyLQAv1eF65Uj8yGWhQilI2pHDmU2BhXeCQtmRAG4kYUBBxQI\\\\\\\/pE6LUPYQdmp1CXiu7xb5bxIoMEpjjdI3dK8okIbpLuyOrG1xsYIrHujCWlcVAJF\\\\\\\/Xo5hLKvanKR63sIw83tzknL6T7GTM31KbNW1nlNiIgxu81\\\\\\\/dC+\\\\\\\/N+cuwgWQLWlpE4IltJaL0eWZf91LPrv0Ynia1Qo0vKWEojOKAEYbdKogC4huW+s1FVFWsbsIr2EXvNaHnI\\\\\\\/6s0s6\\\\\\\/WZvN5\\\\\\\/dmnJQAF4DHEZzNsA5w8EvcBGr1472OsoYuerTbMmyDAnst23Lo9vlTuxjpotsKzhGN0ALUVfy1vNEv6DVOJ87L3VPXuPR43mrAb\\\\\\\/Fozg2BcmcZ38QcE1+9Kb\\\\\\\/ZpUOs3r34l62HtAj4MVkmyh\\\\\\\/KgEGt8dEvgDl8HSWw6ko0xvHeOv78QOriIEunRxD2jmLT1JR0wlIrDN+SDpQgR3xb4xtPgSujdHhdYrKrZpLxN3F\\\\\\\/HrYk0kpRS5ILISXnwuVwjZqg8eNTPp0n2wH97t8x4ihxvKoRAVMngSjTvsJAghcwOpgJfVTTRQx+eGy46z67FoE9Ap72WOLc7nqNomL7CVCj+o7gsLooXSeSlNGNPkYjWbMrCrTneQ7B7Xv95E7kSLAP8RdHRbjLUXH3tcrNyvFHbdYninUh4H5HiDzN4RtGVNHBG9Nep4Ey3NOzSXNW84g5PRDqD7Tv\\\\\\\/MuZnt1cxmeea8gVJNl\\\\\\\/D4HltWfrO0T\\\\\\\/RvDAn35PniV5c3uSvn182oAlelajiRCZRjjFthN6YPh4WJwr1jK7w30BhtZCRieSe9FhSGuOaBbiaYtSjgFxxk930vkwsL4AzlLZ6BKvlNDPtuXJ1oVXfjHfvncUw0guqrnXY30JDQXX68VVJy8VuJI1\\\\\\\/wve8thxxA0LcDycrptkIl6yEegtFr18ZfvJWI\\\\\\\/2dRnU\\\\\\\/vBnzB4tUTWy2lgoEBj55+YRTrv6wfLOX92HiESuDjk+6Ah2kaqWsXTQC7Td3UjLbJawbubwVLdiRkIOzZHcQT9hTGVzYGTVQoBOeOBL5AU3wNtOTeSTjJg3fNTxtgF9mTQRp2Ps0EpqpulZTvr3fOc7b8unNDhp6a+VrrYH11R\\\\\\\/zYvdrrgYuOtHJzsFmx0Ryn98Uf3ln5QaqS2c\\\\\\\/fprutbgkwKaBvwMli49SYvfUBKZmL7rOxoU1ilFjgqeT66CEbciUt+TBVX8Bdy4Wawd6gor40x12Roj0oODJVFR3krcC4ZXECWAKbJ0h25YdICGLpihPOu9xeitcmZ7azc5LJOeQCLvxTtCehTwhmQ8Jr4mnUa3NFBMVJr8f0rKW1sigRiuiC0U+A5Ya67\\\\\\\/NEqZTKb4X2fmD8F7qAo1j\\\\\\\/HMiHCIleMiWOInUIRvrA6J56QcuUKMBN\\\\\\\/\\\\\\\/RiR2O3NWCGTwVVOsXukXirdwuZsg4ifMcQoCuhPl7EoYChDzyqiegRxueGS1iEBB\\\\\\\/9nuwLEEn76Obaex4RIIss4lPW779GwPt8XnaAk6t05XDw\\\\\\\/dZe1fnS4GQTaLFYBzC8FK876pHq5y8+n6Pq+PTmNuHF42+sf2eYlTteb97Sef3oib4um3Nr5PabwxjLdNVXaQyRzFmGHbXLJxt2PfdBEvcgKkPv+6CPH78qYdZWVJWoCSVxbJNTzDMEjpTO6M\\\\\\\/drl9V1HUMDYCh+G66DmVbt4FviczcQXsO00y7LBBkrOo5e6vJJt8rCBNVOVYMVShP\\\\\\\/jPPQelExPSJfOIghKYy57jJiC2Pnpy+Bh8c6tjJ6fNwA4a1RgLmWz+Md2a0lPQuaiMfeFlQAhzkGSnCBiEr06Yk5jhCYmbzLDWY8Zg7o4laSt2n+jKwrU2QO\\\\\\\/d2v\\\\\\\/Sifdmyr8E7CZjZC9ihsHRHbd\\\\\\\/+i3VozuQ196YFXi\\\\\\\/HQdlnn17Vj7yrk2L0nuACiF47GAOi08yZWlMGhQyK8IffKy5JCLNIEEytvQmRcz5cc\\\\\\\/DxrrFWUrlGwL8uWDY0f4V4xKrfI+ujwYMG0RNO9nCNN3JT01RnNJhJ1nOrEniLb+SMCjeJz1A4nxu+HWkse\\\\\\\/pYoVVEO5EKqUO2Al19XcSGQ0\\\\\\\/NMHkRMe5mJM2O5bZI9Gphj7Z2rAocF3VOH4otYXZL08Y+wKZG3neem+JLhEpq5d5u7Shw69wWIX+SQUNO4EiNydGfa6+GIL5pMUw+Ro3rgzjy+c8I+Cz0nW0Vw0XnUtYzDO5gPl26JQ7oLTXUEgjphKVehYHvFh5RI2Y3RXTghub6SMOTNbX5EEOQu4u3uNXayMUm9hat1dMpgSt7yK37ND9BJynGnb8zq07DPN52cminHYAHp7OoPvI2Tncidrn4fa2bbd8JM2jT8huxzzpK0wXAcg1YhaNf3h00eJyBu\\\\\\\/112Dhg\\\\\\\/WnbCEavkOH4Kqe9lSk+S9OCfjPlshRlIYeqgNse\\\\\\\/0YHa2QyWv0\\\\\\\/n6Su1dpaaWeLuXdjf12XFJ\\\\\\\/q8gd5qbIK1YbVgk5hFwwaMu2qb54Z68mcW1zjXmWyv0CsvdaxQ7b7CK7Rmz2Jrm0ViPCOrc7PhBQyKuYWS3CI9KSfWQugEKRx1bAx4hKE9QhjlCABENQI+wZPcwv9pNe5cjvLYQUOor3h4irBhSPKJCH+01CBfVl0XRkngnXPbA+fxMWTbJEF7HIhohCaVXxIGMr48ndes2Q7vBCgIx8ZgKOOjeAbNzZ2IKqsnPqaokylg2gpNUVYbZ++zHR4viStFAVGaCjKabXW\\\\\\\/ZHh+pN3G103AkfQYUhgvZ13SJxs1QJAn24qgLlEt7GdkRflpDLJDWFHF8k\\\\\\\/gsVFFQQ88OVU4yLkZ5VbxXUjcd9KRVvx7sPp8ps+w60LZIKdbuPuv84iX9XA3rNn4L22ojduTG14Kx3C\\\\\\\/HepiVnlDiXbgdpOwKhDiF\\\\\\\/ZLLBE\\\\\\\/rWxK7t9k5jNNmvj3N7mm3o11nytGhavK3PJCxmbTE1qAXXFtDuurGgOZJsNoefmfUu+HEeDEoKbmRCdmfc4BUzFIlJ\\\\\\\/enLg4+jc8XVOQbFWl\\\\\\\/5fpUs0PMoe9jTm1\\\\\\\/2XAXv9LuQKQnlXZeU39txe791UMM8pY28xh++8M8PiQMmUsXKLbW6zzPwsebZIolzfosF3GV7jyVGNHWOx3ynZwP3w+tXirja4soVlYZBldYkoOPxxycVzgc6smwT2NZwwyi8mso7AY8QIlYUUboYA\\\\\\\/nAyNW9R2qujqAqfRv\\\\\\\/J0yZpMHpvUALfZFASJi7tmcC6\\\\\\\/ow4i6WA+5CKbBw19O3leytbf2rM\\\\\\\/WguFwLaooCULYRfNJZDDuaxpuzxjMBJ5abY7HI45ZI5bzFAxYwbQF39ygTrTiT82HHNzh7cXWdkem7UBc5Ndx+Ao5RMDNXDZWYy43irw02sK4XglUP+bcesV2QtOnbYnvpOfBJeiP6A2w0ae33AcoD\\\\\\\/C1V5U86bM5L7fxa8IL8ote+tM5lX9pCROTP29K+bxpq8Lx9HmY3q4lYqKda7YBu8jUfj3A3\\\\\\\/DtWdin0H3U807dHirGZCNqKk801qpbFVNG3K6od7XUK0fDtPdp8eeVjDQY\\\\\\\/ofnNQ5ocGVXgiAFjtIyBqlQp0f2Ul8+a7BRjjZ1myJMfwGhUXwATuqPKD88y\\\\\\\/XluKPaXdJx6VnpwPZKJalaVSxuiUO7r5zgBpvA8nc9dMYpb4bAZYlEdngHDyunnhx92Byp3iDN+3JkuKOTplSVZbAjEEtE1UWg3Z8STTMtHKZbTqzI2mDJxqX+oWCSNnUePyl\\\\\\\/evBoCiaJOxjW2y377p+689YD8dVJwFHGlw7lI3EF1e6kC0bHIJ0CNp\\\\\\\/sxF4Sbt0Fara24Ukg33Tk+RVAhfLdmiWspHnATWFIuaUXhaBlbgpOcee+3dSb\\\\\\\/0329ReI0PdPiNyVpy39rMdTFa+PflQodKqypwH81aJe\\\\\\\/MALdZokecYlqbtjILoE0QabW4pdwWkGDxCXtssy+c9\\\\\\\/mD6tQwaQCz4SR+5qQNEpgygHYZpjalnX6axq6d2FIU99+C2dvPNHyishF9lW8ArKTi06AOs7nrMLFsVUBU5mePdIvDbLiX5OsX\\\\\\\/iwznolQwt8uJSxoiuB4B1tL+42AWzybhn+VXVUW6kISMxu\\\\\\\/ZI98MZGV3rj0Qq5PX1Kuv+eOemXhO4szzh9PL4Zl1FBYJdj9PNhIMtEmssca6jVyOEP7Kharu6kB3aRMbE60AO84DZauP6x7fuB872VYlMTB7Z89udj5ukir+GfI7\\\\\\\/OomE3zPH5UqjIokILsC6+qymp2RGYMhrbbNsNgA8NuyYNQzRlkFN0EvBczjzHGbvuDcloDxHk7\\\\\\\/HK9RJ1XIuMCfTYf8+6DZk24SG4NXieR2clhCvlLRJf3pB9HFhIFQD1y7+jYURPEAhCkFs+MZxH9rwgUjezYrDJ4HOGuINlMCfjhLcroK5LJanyolFmQBdmZvCe+4Z7skP64bVJ9CSKwm6YRa7OCBa92Kanl+J3WxjUzVQlCrqki\\\\\\\/ZFINOhOouJPzJeYOv\\\\\\\/QbqaH8qqV4+vlGd9m9ARYMdtOCWn1a75pdQI8yhvAkiedYJdK6jKcqrxut+\\\\\\\/cAVOZLHEK7wMPplk7XVnc69\\\\\\\/HBg6PrcLdmitiL8IhocMogOSKGhq8F1zLkVQew+y+URQWJqvwMr5xoh6nI9tnvv4cfS\\\\\\\/ty+zD8Cjc5yCvmcdVBik+OPPnZJuEflOS8j\\\\\\\/XMjIoQZ9WfGzSKuVlBl5FI2mgNRhIMfD\\\\\\\/b8FQJqEvdA9wEL1K\\\\\\\/\\\\\\\/0ZPmvJWDLls7T49LdasZBtGXQOhvfz9xzXi8GgeVLN5AptJQivWiDnaERxna7+dY2vuIZhusV+2uHeIGUe51wVbAytSwOKMKw+APKPAdhBmwGsyCiAFAZ8B4fMvMEp15whQdDn5LVaocnGiiSCA6K9Ve6aEMzycriGNPrj8PUAQ4fUoNWVCsSgqjrIow6FHF0nwrTL+5yqYuh\\\\\\\/HUtmw9xCcDAycNp9gHh\\\\\\\/D9jlztWKaYCtEs3MMyu5NF7mj0ecejLKRJ6xhW3uSQXNHrXRn1vp+rG5C2i62TU6BFeRtIcS4jAhyFA3jUAEQMWqYG6opL6RPMDBZAzSsiddOUg\\\\\\\/RfCjjyEeqRf1+5lFERQ\\\\\\\/PVcZ+jthkuF5DIJ1UZJ+tS84o0wBeoXLGnjpw0q1XzPVRP2hgEohd7lWebzvQdyNcKBIcqeO6spz535ZvJsx+rOWVUcTFaAOh2U+YDPr3o+e1qUL2Q9xj7Vn7JEumhO3UxAD71IESZRoYpf7f1Zn2y2Q10hE+\\\\\\\/hQf1Ic6BJIHRbJJxGu7+Vc0qG1c0RM40r3e2UPwROpK8kdTQbLKeUxtZIaL8gq\\\\\\\/6NvDAl+ZaUtsk\\\\\\\/MrsRNdIwRQ3bXNobO76SokU45VRur\\\\\\\/5jn3FTRe8fC8xC+qzEpOMqHF8csBl2VKtZtJdIjEsLk7vHMKxw5o++6qXvzDC1mplCeF4+M4CvokQnAKh5ARK5m7Idfv3m73KF7ov9FFXiqZ6d9dmlggrXF\\\\\\\/J5XFTbMi8FbsSLyBY+Ib86V7woI5AUa6oPXFonYKaUexEsRA+ktRy0XYEMqN1pzS6xKgHbKiBgIeDdho5v6S+Mi3QcXQOOxzxN5cYZ54DKTBcgPbsg+hPFMj2Oa2nOW6gk\\\\\\\/XMf8ST6eYuDp9r261P0VUpO74sHWpCfM+reN78U3TG\\\\\\\/G8K4XRKoNQexObF\\\\\\\/Cp8m8g\\\\\\\/P0mWkworGPGp5+rl6nYc08l\\\\\\\/hw2i4uDk\\\\\\\/fwTDyTXnUowYex\\\\\\\/3p7iNYPs8q5ECC9P97tWBEhR+4iDN9SlOyn\\\\\\\/82RKHIPDM1ti6oaX4p4AgTmc22kbcxrc3fewIh5CF72+PPVs4T+FxEc1XVo94G0fCh+bUdnBKY25KgHJqVrzoZritm78DqjmW9yNVOkoEi1s+2guGmQMcR3TRCXN\\\\\\\/8xOpAKfS8xupYlZNLRt6IjGO4lkbUOeohr7YGnlQTaj73CSM98q0YDnFbdf9zuiHK\\\\\\\/oQ+i0PP8slJAKCRrnz19GtFNYs6tzOUh8u7P7TamoG79TVtOnGEP+OxaKuL8TWRT1boFeALLG99G786MAdn2mYe+GrAJy+sLlZf6Zg0wdha0Y\\\\\\\/dz1XbwiQGhbq7+IN+Dg6qLMt\\\\\\\/KqYOT6nfjjnIL4ArB\\\\\\\/dpHS3hicSgyak4LQvxn07KI20xU4FPQXjNl5Yet7N97MR5QyPOH7ZuoytAr4ylI7h2gRJNCG9zptOewJjJvghNJI736IyMpbYbCnd75\\\\\\\/8o6UECbL2Aht\\\\\\\/IKPko+G5YJIr9txgbd1\\\\\\\/7zbtNDtKC7kM6xHH+avgakGrKc+DgjKG+i8vGqFlvLnDpmS6ZgesHGx20qcOhrVzOhFP3IdDjQSvOX6LiMWFYQKWte3M6lxDNfhrqwJd00cpCZrlxtKBJuo8+V4+A3+GXUy4NBJMnTLcYVwwB7\\\\\\\/0P3zdcOPWyoMO2Fa6bMO4flDNXzxbTrCHjm66kYuL+1Q4IMsC74nXv9tyFxOpBniGb8Fl4LA3kW8lvrwUN+PhTJ\\\\\\\/kCWdFLU1lmLdoxBl5OdJl8+TEpd229a2a2rgV\\\\\\\/Jw5SX0CTbfH+Iuw68SuiuhXMi9a\\\\\\\/lAWbHcGqQaucbNkhohI4DilqmNX5ZpH35aNAt5ldsn49hnF9iyktruvnqTb\\\\\\\/FPZ9GCgsHHMOUDwV6I\\\\\\\/WnmzB5nOwONO2+I291TCd8iaS3GKWrDGaHKFczaEtu3NZcROOn9be9M5y9lzOpruUV0XsaDMEJSes0nJfwjo3qxt+xjgeHilNUGlNj3Ph7lE1NykrR1jrECt8X7sxlXF8zjFyBH660CCNZtQ9fk+PBZfeEq5qtXO6TRZI+0JDCsnD7S1V3vD6MsnAd\\\\\\\/onoyF7XwxOBLgEUiDemaV3Lk2ycAtr98h8sOi9EFOFgtBC72HZZYpIWjaQpfTXl\\\\\\\/8ZCFrTjfdgQ3Ei+tlPpV2sb3eMPfef1k32rss8EeRvDiAM8\\\\\\\/esEfarT8IiHeyefuRN6VVu0VDbcwJjzjogRjAOYN2unVeLAK0xc0r6wT36Xr2T6JYTT6aXHheZ9KMXoq\\\\\\\/Y9AEB\\\\\\\/4M9HQIHVyGaFWs3kOgxe+rzZKmA4ea60lnvthUKK22jQlcOVvb8A1ewreFKkWcMzyW8kavmpjRLawZU1qUMQDq2DxcD1sYht2FkMyfXqlixa98cE\\\\\\\/pjjXSfOJ0dXCkSFhzc1nKGS0S5quElJFGc54O4h+i0kPLKmd938LbZ+cLxWMZzfbPoGbFxlGIOjnWJ0xTGWeIbSRjHm3vvHrsl6R7qk7QOJrW+fRS1Yt0f1njFavad5PB7jPgggYGKxnIuO8WEdJuG\\\\\\\/xpB85RL9Yu6hiY1oCt\\\\\\\/OfT7rmedaYhNu0RGVzBFii2kFUoOTS3YIN\\\\\\\/hBjeQc7druSiKtFTGDL2y6q9JbWR4P4WZIQhd7QeLMN2uBgZN2unH+rLHK1iEsnM0sQrI\\\\\\\/oTXqlQXjvQBiS+rpt6hvYJ1gapHiXtSM8VfhvFl9QvfHj4UF7TUQ0cFf1ba6yxkpAe0zU0kRFR0V\\\\\\\/mFrqEYPuZuMeFskpLfGGxpRMzVW4VMeOi4XI66Avy8MDyv9EsH4i6NYE4aPxdXqAo\\\\\\\/CXTwr+JRMV0bTq2FXExiamyajRKZFReiBj9aP1QFWZpghK7VK+xz\\\\\\\/KuyeXOzkc6UT7glMjazkB3o4XDOOzhIThk400un55kWjAnunvU11UgUyy+hpnGWZUKWMhIGB1V6qvvizq62Y6XMjCQnzzorq\\\\\\\/ehWy3XyAAIlLGM7kceuh+bY5RSI3gWTPNmJl6V4YUG9od8Z+CkM4DgSTVXBmZfEHqk7nOLYFma3mo5cBKfwiirwcWR5F3hI3KQCUBTmxC04OjURmMB9uheozLJrys5wrTBmeH7P9paxl7sxVPxUnNEIGIRvgY6gVCXETMXF2AsPvpWj78bkBftBhvVF0QQLktGrQlHTHczbUoeMuHT7Nci0ReP\\\\\\\/DwvxBR466E7mnd2RZC02ayBhrRgYQypohk9EcgSYpNbqCQbPGM+Zw2rusBb8dDVQed58HSIqqkPjmnhNNiNTXUb6P2AvyC0JyW7mDwARHLXCwBQyG+T\\\\\\\/2\\\\\\\/jpi\\\\\\\/eyQQKmSdwhMWvjl+pkVqT058dNMdQRqqu8vWPAOX39tolwv\\\\\\\/QC4xGVdLg9JSc8Vh0KvwI8QCRus20UC+j4ub\\\\\\\/S5m1\\\\\\\/k1wcSQg0gB07Hyecnue6LaMir\\\\\\\/RXr2I7BK1DAg84CdruN0YmKjDW9\\\\\\\/AoHcf4sMuycAlfTmfW6YfyyAvV3og7ttK0AAHp6ixsmCg8eInuzky6oZAnmTaCFWmBubGgZ57yGhEApgzqeTh1J6\\\\\\\/EwSmFRLYT5UbpS990Wj+SxtEFCsYVEqPjNQusm6c7ZeHNe3xyB6cAzLITr31hSh3bgzf+5YOmRKcG28WRcDdrGcrs3Onj20oAaT\\\\\\\/PN8wjrzcch1TQJ9VxljR15pJ2JU7pZl3g6bnXTNijr1AWalZNNPDncvvnKiyG6MbXDlNyre9M+Vp67DxANzAOSM5gD9ZNj4umFVIleSfTLYwl5mRx+d6sfiEjrV\\\\\\\/RdUW+1a11GD22g6n941Zpuz3t6DGcphZqcL2zbB6R97s1YZFrP7OjELEmeWRZv+8Xm4Pu1NfLf3pVPGvPIgk5tByZCyAnY9eGpdtw51Y8sqLuxFVVN27VWn5Q6Llxoo24Su15E7ktozedGF\\\\\\\/P4h3UgdF9oPoTAo+LnjWlowyarkk1vYqckTQ\\\\\\\/flntwjYkRSIalTxr0zeB+w3Ab9ZaWAwfYtWuDSGs1tX87bDvki+xvujwLElNqQy2D+5cqtox6mwqeNUEa\\\\\\\/IblVGY\\\\\\\/PnX\\\\\\\/U0Huue4DHJRdIlRL7xrOhYyOx43gYU0u2WUy3CLrqCjMZbWj1vsuJ\\\\\\\/wx3VwJ2XcWb7DuQIENWBMC+GQEbtY9AHcEU+\\\\\\\/jQAxB10pfykJoGk0nCAuN8MENdQKHtdPgvwrPUr3uE9rco58EA35QLAR6njjeVSMYW4YQxX4A9LbXyWwbJVZjkXJEfFtOjFXdDeIDD8ldO2Dx1AVTv9M5X6eAAAn7eSI8NzryPQs3ktCHwcNTe4RMrrk2l7cSEQVbU+z6Y2kQ2cqNQaD+j8yy8iwJTRcZyGBzsrewHniFPRQKd6h4bsF0XOUASgCPMB0XZB+0cfd9E3kOp78dEyonXsb8avVDyYwhr4gONZiRwwGzIWaXoRUOEqe+Adq6nUHq7kOtF2yBD+xY\\\\\\\/nQFEdorx2L1RljNcvFOTgRly4Ded1nfaObFO4Fbwc0MjRIw+dhuG6mtJQcPxTxJ9ho9nWZg0G6OXVwZLFAd2koYokzmGLD0BXLTMPZFcFi\\\\\\\/ZmjBoNO1MwHut\\\\\\\/CX2uVHh\\\\\\\/rMTfoFP09C3xVZ10Oj7B8U5gYBBqReC4tY9WpeHT5NAeSngUsNi0ZJqSvksFh3Q0ZqzPnlB229jMlFgdBmQGuGzXLGLsNHVjgip\\\\\\\/vK4E5ErfQAMdj2adANvl8yRNKT2QwGdk3RUZriTOzoA0qGjN7mfkGXYdW8dBqiYf8mwcG3RK3FwYnrm7ULJWEf3kHLAKYgC8nyoZmoiyt9n9MklsOAMlfHw9FoAcDKlo4xGU4JGy5\\\\\\\/ThooxVZ0I7jZR\\\\\\\/i5SBEVB89oF7IOelXuwKG18jdr2EDmPDFjrxqDR6x5g89+xcQg2e6qnb8mjUsSlNidCluFjTpOp8dRWqoBGhWGcA+DpM+m1hzCQdJUHhsxsVmSlYgD15GyTt1x610F1j44\\\\\\\/Ag\\\\\\\/i7i2+fFgLmlTbP6ZPPJTlTlU8B3+VHt2ZCtlWjh7gXhZqvn2PtrIbjnQhuQ6fbXeBzQJVke9nsUu1mwyCBn1iOTtlr\\\\\\\/R4+xdkulHW+uSund9YiXy09tVT72Ae9iAx3kde2hlb+zoeTppNTSCOCOLmHLqndjOx026jPpQ1ZQLj8yj6mdgq9ulr9G2cODC5TVDSEl7zn3RItFbB\\\\\\\/GAdqiSoWXV4xqzN+O4Stbc1iVxGqkyFmm5SwJUWg9U5k1LCzGsGf9xYHJGxu6A0DV0wd2v1V7APv2B1pc8weYn7OYkXOQANsJvcIFrFlDs6QuerN95Vg9ZyEcx9AWodI3HFtGYbPrBJGZvFrqTL6+5TkGFjQihU6DKCahZNaAyJ+8tVCJCGwIfbmVqpgtscwbbEm2bZKbOTRYQJMgv7loA897uVI4xQPDyOcDQfAnjHT6EiRrJ250O\\\\\\\/3O8Jydovlgkf27EJi3TWp3xOhwxOQUYjRKv9bw\\\\\\\/SiY55TvgcS3rZ+QDcCI3\\\\\\\/5QKVDqpVYfMYUL+1uLZZRNV2I6busaP4P9GOhotOIa7V8MAqVEUzNKwI\\\\\\\/SKu6EsFkKYrMcsbrphNYDRbA8lEsbY+Dc6omaYQJ93AxzBz9iMuQxQDh1xq9RztdVHUXfvieB9U5dZOQQ84zPMBaTkvQ\\\\\\\/drSyg+IWwBwKJqSjHyPy81utj80plqvHeeK8QV1TMRHDzdseYhQuVbiwUouwdCtdkpVBBiTKsxRmV8xocNgAU\\\\\\\/6RkaW+s+O6xkXMUYF0aoqgGnBtVwL3i4IxS7y3UNnwkmN7URmVF9sQ5UOgyJSU\\\\\\\/o9JO1lpDN9qF6F7kdA+pt9XDdC9uMxVm8+x6Fpf228HV\\\\\\\/pD0r+XUbfQiSXEhJYhN+pHWV18NXScbrOKyRdGl3tXx4nhrPBeX0VJGB+C1DJ9HbcuvmbvDt08fDQsCqAlujJG+3WWDGCYafCQJoumgAxyzqiqNMZnxqorykHjordcraMKYHvVVCessetCfD\\\\\\\/M4cSGobu0yVE1SuRW+Xu8APsSPkx1rgQVjuKEq1QPWZCTxQJqNin68pqqS0w5\\\\\\\/NqzB8VsoHcr3\\\\\\\/Vy6XUxlFe7qOaVcO8LGfDFNH+CRaF6W3ElbAmNHkI95EnJ4+BE7BPvv\\\\\\\/mKdabDRWTYtXU1MNTLK\\\\\\\/f\\\\\\\/a6tRbVPgh0kSCMjOxqgrwGUuwlCV5z8PVb8fs9wn84UI8VfzGiiFhlzDaYoqDJElL7t0av+YbseTPtqeXX4T5VWks3htaT0xY36JAxT3sG5iM7uW7TpFbSKYf08SzwTF9pWUtizJ1YLRNDQ9HvBKzqOGm51nK0oO8495Bb06D4SKwTqcAm6M6KWTdpIYKEljg+dvrIk0LHIKK5VLy4wZ7XcapPx0dNw1lydQ6AVN+UW7L2EdRnl2QVAVt3bY4rU9VXs41mmLiZ3+4R2rluHP4Q+4NxfONGtFjAYBY6zhIKOPBQKh4m0\\\\\\\/UV2t\\\\\\\/yDeM\\\\\\\/OigbxvrWpPoQ4\\\\\\\/6AHbCJYn4JklNLL8B4\\\\\\\/TWSasXb\\\\\\\/ANN2Eqv1mIrGg61R3gjAs8MqF8ZH2oBiB8ZA8baca14ed9xItHJp3AzMRu0E\\\\\\\/eRjiO9\\\\\\\/i\\\\\\\/BKgPDJuw7\\\\\\\/8sfIwYgjCwuV\\\\\\\/ciaIHv\\\\\\\/m0CS6wLHkHtpQY9Cl0mFYf7jyZ\\\\\\\/qqTkTFRTqd7hvRBZUrhzSnRxpnIeqLA4lqlc1jJ80SQbJtA7qxV8YdS8R1lMqaj3ror8VIBCMa8IGqKe7uu4J+AntQIVmOAwXiVckAbXXPn2Z1Ea09GwnGfdH8cjFYLG1v9a7nsMMF49eQhaE+k9Aft7SRXg8ySXT8pEAlMneR6ncvnVUsOSV+dZhHY1bvtvaDM7cy7JQPyoKjB4uw1MYDZYUzpYJavm3JlqNXxv\\\\\\\/bp6nPfDNuY04M4b8pLYglzy2C5h4I4QejOn+REcLeTg7kf4w5p9W3Mz+si7pdBc20tbja\\\\\\\/mc1GeibAWDzFK2XVoIq1t3CtG+LMClgsxbYfmXLRf6hBAjqvrmhnVzwnMHE\\\\\\\/aa0TsTrqjKdtKH357+9lU2y4SOpNq5zTTdI2t6jANOPhHQH37g60v+mO4WgA7+EPLQCkj8KahBvsuIseAFBwZ\\\\\\\/PnaTdphyoAHnyPJmJbP498zJkSeRtN1jRPhsPD3QRsEhloZ4VQH7c+UMVX+WHQtrahWBE8v\\\\\\\/hssDW\\\\\\\/50ESILXfS7jUZCG1AiInCVEvxmdv74P+Y8Z7qDilT00UiD7n9TBIuH9SUqcKVICxkRTUVZM2nwuzJ6N\\\\\\\/HRaAxf96zgtP7jyWexlEG+9dI5eEhWIghaWxiJXWxW743k7GIbhu4kP6KOSlTdCvgaCLHJZhaB17I5\\\\\\\/LJ++HXvJz2q6aD1HLVnWbeXFlxOiXSbc1897oimFimCTVPgVENlyEc\\\\\\\/FftEeQWQE5tw9Yuzcfx6jLBmfl85hsd5nvA10wrwn9y7OYBSpcye+BV2nXEh0kxugp4RIvZqfZZw3VUlNVlAUkzNcZw5+CKjt56LGQ\\\\\\\/3nS8K7FD7\\\\\\\/EaA7vxITSii76KEr9i9pS8fEjLmp6Yp72o4rO24EwvsD2O+xZlVRPQcvsc7ha\\\\\\\/pH5MXpe9eVHx3CAscSibj2OMB07PdZTbv2iIQaK0EqkEeGMdyQQSDmsMRFQI1\\\\\\\/w6nKDqDltHj1lIFUmkLprVbyhbOTG6+VmAG+zcLCaJWG+i7jniTw9zdzwTBwdJ2EmApiYms6eKDlLU4+SUH5oYh75CTdAdjKx4OrjXrRnxo7LVu9\\\\\\\/yCki1sbnV6v\\\\\\\/+\\\\\\\/+7JLPhBoFV6zMAR9\\\\\\\/2VjjX1GNiVXL\\\\\\\/zZdUWUGBSzuUTeimp3eFCSWJi+0ye\\\\\\\/c020NwZ6uEY4MBn72VDxBiGIoCC58Jlc3Zqc2rgNpZqpvcjkggFduPTwUBWNbqOlyBdsVk2HhzKdhbWUTHt7zk\\\\\\\/n3Ck5XSZoUiI6eEYXOBq3lIoR4JQQ6ne2VdUVj9swGOa4696OCboTOXc6H5ejRpkqOMZjQmSSliUBLVY3ELSdK1gjD2Mv2XT3TE8YqFiDhhDAKGpJCXnPIfAFum6qyjeUvmUgGCynPHE6UBRNB8kS+aVIfxGagataYXFk2OkAMNDZHlU3G6VZPGlUVX5vlDk75oeE0fMVlQekP8Lrnhl\\\\\\\/ZkVLPjq5tKtRpYRFNIxbQAshZRk2Ph2U11XMHgWN\\\\\\\/f+bBDzcA7VNhUMGPZMKj2dMEc6iNN0Ld93NT035sfs2qQ5CeM0hS2zVdYUNV4xRtDyNul1IXbJnlLDM1DqnyvseGjkCfHXSbig7jTKGpL3pEgNGL5amtTXBb2R\\\\\\\/SZHVyDrTTletadpmWFUVHTHgtinwbJpVuA+y9wtqdI86fJANVxEYF0EyGo5oDhOIrqT+SsyEAFPKeF7mmLg9qCnpd2BHI8FNIYjrRewMRZc+4O+kajQQblyGuh9BVK2IMGqCdKkpqDzzkJVjIUpQ4hEhpyr+Z5TyKSV3w93jeBVBKZfjGjW4g3o2elb+7ZC1kdcZzv9XeAT2Nnp3m2o4zmoPsDZtkf31fnVHh0RryJVPw0kWROeGWDAB6ge9l0muZIVFVMOvBcNSwfjbA9uKRUrOu0NwJFnVDPI3EU3QYgmTm50HtfZF6hhhh+wcSMF\\\\\\\/qSqEw7vj2f4LdHwx+vM2Symg4REysE5T1t9mYli1sTv4c5\\\\\\\/x8S90n1xcNz1w\\\\\\\/SLRqwIKIYQFtsVNPVZNAhdvJUTqgpLH2rhcgM\\\\\\\/e2DqVPjtbxU2OgqOwx2zZoNEATh\\\\\\\/UqAKCty9XvA3bFT8dlXmxrIYC6IrRTc2zqq\\\\\\\/Q0p7PfRx+rUv4SH9PUWtnEV3xpqvKalZ63+H3H5N3VOh9RnY5DseVggax859C3m1\\\\\\\/Go1ITcYWcO\\\\\\\/xxI1YHHr5JFs6QVoZP7IMGZrFUGHRH5Jh0ycprgpjmTiqjSkfYt52acc93HpF6LyA5NNJA0HvfIPLIPK+\\\\\\\/\\\\\\\/grjLfqRmsyhaLLvmUlmb1RSi15br9jhMBUFIq8ITJninEbfBsQK0fjcrjaklhkAds7K3\\\\\\\/BOzIasw3s+F7p\\\\\\\/P0h2xR9fyS0s9B4+KyJuzjV8sSiM+R3aCIXiUQxTMRn9RRC0Qd1XEbNH1W+eD5PYx3EJZTl1x73pAEqnPs5F0Ej0qVUudKiF9jOJSBgPbMSwDSyA1L+XjAyY0+rAGdXCzXMFWdikLB25qcwo5HTqyWSfU7QK0yjRpUmyJUufMONGjq\\\\\\\/eqE9sBkitb+FrbuVDN\\\\\\\/aFvOY7b+eoxoqL003FEmT3dgoOct2Ig+5ZgnPT0\\\\\\\/xn0lR2vYOBPy7nYpbJtTnSkrFtwU68vkDGOLncDErED\\\\\\\/l79L9xxof5DUUg9HXQPulvfqoQjS12rLAcz7nihf8d+WREw29aIqNB8cJZArTbt7V\\\\\\\/4OWzBXMoNcvGiPKRgfUUNHUXIzl1\\\\\\\/olTcrQhM8vxR\\\\\\\/pjogiHoVH1SXy++JvJtEQ3T4lCrESkhK4jaewjQ9b7MSFbIR7PaDuYMiLp+xy2AW35S3+Wa9oP2vNbeeatXndGiiL4kBo\\\\\\\/ugWdOJGR6lba+z8xAu8A\\\\\\\/1C0eUgHyeR1jc6ilC3ctm77IxCGhpnhcmvlVJJyx\\\\\\\/XdzBc89eLp+9lHkm2v5BAZikzeiSLjAM9iR+gITZ9P43euKIYD7KnoR1ddcdgNWJUsWd2bkGlccMlzBLRcXq5ifAqHGEiCcTdlmOLVeV4rtt\\\\\\\/zgBC\\\\\\\/Yq6dxP\\\\\\\/KGUTYwdFrzmH1LKfy\\\\\\\/0hvnOgufATaUV1c6PyNsNcPMiecoXE4Zl2zdt93jy9W08hboOHl0LEZwwi\\\\\\\/YLIA4VC3lnvKCp1D2Japp3PCL3ARj6ADrdyA3ppVIHuxLDHDUO9npVGlVfH5D1nccRRzXr5xSCQSFiu+ukcMLviaMO1KnAd64BMh\\\\\\\/\\\\\\\/1IUY0\\\\\\\/WyDu93Bya0UXZ6\\\\\\\/j6DcBgoNzxcMGxohkxfvpsds9\\\\\\\/gk8ECg0WxaTs8Ldi7m3TB7C3bvWNWNmz63FoMQ+vk\\\\\\\/ksIFYxxSccF0AL5KCg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022e30vb5143+HkLuNJ6oRpnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002269fd15475d180345ea36923c34445ca32ac15a6363a0d04fc51e30eed37e6c42\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kPiCoVJ7DUZ5kBKRy5QC0c+ttJR8uMUdC+B+MjaQ+zeiuKzg7G1UKs6rI3Rc1bZPuryFzW4LUuS6BjQHtqZH7bM2c5WWbuTd1xfy7r2hFk+jXxTTe7wG2wJePqNkjUPN\\\\\\\/GDr+t6NzuOj6fggGLzfQXplhuvE7vwFVFVNj\\\\\\\/LUv+WiTXwVO1q5cmBAcIml7B9pksVOiCBYdqzKH2SkJnrGfy0W1Xw5YywjUc+YFtlLXKaCx\\\\\\\/fL4DMwBXNnRmQbjJ6WVjlfV0WmarixzqGSuXXZ37GJFRn+t3X+jlo8iddOVdjhh\\\\\\\/ExKfdmfsDeC+nzLtl7HIB5tO\\\\\\\/tPdiNAwhDN2wtjUn6Afx62dEq5R4OxJgCAwuesTzw6HR2JsDfN+\\\\\\\/rAWbAH5gh2uNp2mHHcnAxSMAqK7kYUd+mZp7136hYUCZOxwZB\\\\\\\/bbzXX\\\\\\\/k0IdKkuW8cZWOCGYMlHf7ROGOBXuFDtqjR+7AI4nN4DIxGrryQL0qXCbVS2TIzVjjzyDkeXebOL\\\\\\\/MhkAcO2rKvL5g3Ma5wSop2qDvOW\\\\\\\/HxafeoJ4DV+74DYGHDhdk7cWzSLW93pvDVhFEMOkQfJtno4UhMJjcZr\\\\\\\/mNmNYyURtYaEntT+ezxtmLQqwzImKEaij\\\\\\\/CBuIneURphiKhDg+394dWKhoA9m2WwnvRdkWOV6o1y2JcwPt9sBL5rsZbDaz7kKrACwuThkbh3BwapHOam6qUVru\\\\\\\/Keb+WoVYKm3x48QOuI1CrfjsNNa0rZIqajbsAsMEmUb6y4sv9HYQtghQtWm8VcmWoUxgX0BNsvurMHszzjRLAMOVyrvX5UCOB6ACDYLa2gcrOtsNuYGZj6SrgVCk43NDilkUkzykEjRTFSSI0Am2OB\\\\\\\/R50ibnxmzmuQJmnTe37fXvO\\\\\\\/LaHRVFTMM+XWiBl2oOTSsNrdNNNNa0Qen\\\\\\\/Xweo0O3XUpFYaPr2WgvqiobLN87SVHB6FceN0W5owqPFuUDMyf+KbSn4VK3jpZpk9NUp84tcLpYMuQgqmmKM8JMCnR5fmLP5B\\\\\\\/y8R+j9C2tmj8PSOdmo3svi0JybjzZ4\\\\\\\/H3OUr7kYczfQ4ereBY5MlXqiNJ9BHZ9RdFx\\\\\\\/ClEDmJ7+E7E38Wq\\\\\\\/DMiY77Wuu3znyMr+skK4NlFgJE1cH+pYiPeuCLhiQnsUUcFhDgX+RtxUAICVwWUTvHHjXgMZsbXwb53sYU0aDnp\\\\\\\/ySZWOYcdtd7Lb+LxjVMd7ZXyhoLo2ICLlHizu9enHqULC4nr+LyMZbijLFIuw4GCPyVz1+ljIKj2fzCwaY0NGf8fgGuTMaWKnyVWPa09dMU\\\\\\\/Sfn508W6dhLiP71Ci7LQ9KyHfE\\\\\\\/hUak1x5IGeclxwsMJeMDFufyJ+RV4XNo0x69qshPAkjJNSc3l200AGjUTBtFiqvZOktMFWy5bzWiCgwfzQfavmYG1IPD78ZkeL8KMTvBR+h4OjLoo3pYzC4roMSzz847SVTI7Z4I1iDxd9CjLmp3VEfjl82UIRATOZhhFpp4hXe2TpRrkAv4uYocjS+9IhTQqBNgsQ0i5SYVne93FLQzsiImpQbwMspyhEB+5kIQWzFlXB6kifzUVEzr4DblEBsrSNKN0ECVCRXmt1dVT50APlV0e0odp22KCK8WxhDU4z9M3BSu4aWw0XdfS+3EHih6LWaY45ZWLQ2B0z0GMlbvWORkoYolKsDkKYcFZ2vOS8Q2T57IOesSA8eUydHp6xYEbI2+6PZ86yM8NM1pTPothGRdE8hi4f3OmgAvhxkPminEt9jbJeMDfHLfzs+hmPnNBVlEKgYMDzraNLz58rht45hlbazDprwyGE+B3uviJZJqFYRHo\\\\\\\/Mr9+YN1s8t9tbtnAeMM6Y17PT01hXaHiTdwray9hzJST5Ichct4r1bmhBCwyal0l2pbVLRXqZcW7as6poCYPL7j5Ur2AzgR\\\\\\\/4r9j4kmWXn3ferZT56UnTl6+YX+FJEpB8x9q+DPV\\\\\\\/Cw74Lc1N7RJOn4ypokT2Rdf0na9Rv7wP299YyZmg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OVKXNHGbY+qbjc+\\\\\\\/YUmCwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002291c03f61243400e33d39bbc543ce6674c15eaf7b28d750e15d2dbeaaf1c3d841\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494990 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4115" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zI_hJ9DHWHTVunGsvCMLfm_m2NPjZLFMAREnT4lloCE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4115" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SFkln3R0jZoS8lD7B4t8Jim8TN1kbIvrSyuh1TGnhbQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4107" title="OsSPL10, a SBP-Box Gene, Plays a Dual Role in Salt Tolerance and Trichome Formation in Rice (Oryza sativa L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4127" title="Transcriptome Characterization of Reverse Development in Turritopsis dohrnii (Hydrozoa, Cnidaria)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494990" id="node494990" data-pisa="ggg;9/12/4115" data-pisa-master="ggg;g3.119.400451" data-apath="/ggg/9/12/4115.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3774-552X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fang Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renate H. Schmidt</span>, <span class="highwire-citation-author" data-delta="2">Jochen C. Reif</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4115-4126;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400451 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494990" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3774-552X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fang Liu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Renate H. Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renate%2BH.%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenate%2BH.%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renate H. Schmidt</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a>
<a href="mailto:jiang@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">jiang@ipk-gatersleben.de</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yong Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong Jiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a>
<a href="mailto:jiang@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">jiang@ipk-gatersleben.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/4115" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494990" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4115.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494990" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4115.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494990" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/4115.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494990" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genome-wide association studies (GWAS) have gained central importance for the identification of candidate loci underlying complex traits. Single nucleotide polymorphism (SNP) markers are mostly used as genetic variants for the analysis of genotype-phenotype associations in populations, but closely linked SNPs that are grouped into haplotypes are also exploited. The benefit of haplotype-based GWAS approaches <em>vs.</em> SNP-based approaches is still under debate because SNPs in high linkage disequilibrium provide redundant information. To overcome some constraints of the commonly-used haplotype-based GWAS in which only consecutive SNPs are considered for haplotype construction, we propose a new method called functional haplotype-based GWAS (FH GWAS). FH GWAS is featured by combining SNPs into haplotypes based on the additive and epistatic effects among SNPs. Such haplotypes were termed functional haplotypes (FH). As shown by simulation studies, the FH GWAS approach clearly outperformed the SNP-based approach unless the minor allele frequency of the SNPs making up the haplotypes is low and the linkage disequilibrium between them is high. Applying FH GWAS for the trait flowering time in a large <em>Arabidopsis thaliana</em> population with whole-genome sequencing data revealed its potential empirically. FH GWAS identified all candidate regions which were detected in SNP-based and two other haplotype-based GWAS approaches. In addition, a novel region on chromosome 4 was solely detected by FH GWAS. Thus both the results of our simulation and empirical studies demonstrate that FH GWAS is a promising method and superior to the SNP-based approach even if almost complete genotype information is available.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/genome-wide-association-study" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study" rel="nofollow">genome-wide association study</a></li></ul><p id="p-3">Genome-wide association studies (GWAS) have been widely applied to identify candidate regions on chromosomes influencing complex traits in plant (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brachi <em>et al.</em> 2011</a>), animal (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goddard and Hayes 2009</a>) and human populations (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McCarthy <em>et al.</em> 2008</a>). The most commonly used genetic variants to test genotype-phenotype associations in GWAS are single nucleotide polymorphism (SNP) markers. Alternatively, SNPs can be combined into haplotypes which has been popular in association studies since the structure of human haplotype blocks was revealed (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gabriel <em>et al.</em> 2002</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cardon and Abecasis 2003</a>). Empirical studies showed that haplotype-based GWAS was able to detect loci which failed to be identified in single SNP-based GWAS (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Trégouët <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pryce <em>et al.</em> 2010</a>). Nonetheless, contrasting results comparing the power of haplotype- and SNP-based GWAS were reported in previous studies (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lorenz <em>et al.</em> 2010</a>) and whether it is beneficial to use haplotypes as variants in GWAS has to be evaluated on a case-by-case basis (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Long and Langley 1999</a>).</p><p id="p-4">Potential advantages for testing associations between phenotypes and haplotypes, instead of SNP markers include: haplotypes may exploit epistatic interactions among markers within the haplotype blocks (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Schaid 2004</a>); contain more information on whether two alleles are identical by decent (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Meuwissen and Goddard 2000</a>); utilize the information from evolutionary history (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Durrant <em>et al.</em> 2004</a>) and provide more power than single SNPs when multiple alleles contribute to the trait (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Morris and Kaplan 2002</a>). There are, however, also drawbacks when using haplotypes as variants in association tests. Adding irrelevant markers to a possible causal genetic variant will dilute the contrasts among haplotype allele classes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clark 2004</a>). A haplotype consisting of <em>k</em> SNPs may have up to <em>2<sup>k</sup></em> different haplotype alleles, which will increase the degree of freedom and hence reduce the power of test (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zhao <em>et al.</em> 2003</a>).</p><p id="p-5">Among factors affecting the power of haplotype-based GWAS approaches, a fundamental one is how the haplotypes are constructed. The widely used methods group SNPs by sliding-windows of fixed or variable length (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lin <em>et al.</em> 2004</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang <em>et al.</em> 2007</a>), by the linkage disequilibrium (LD) between adjacent SNPs (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrett <em>et al.</em> 2005</a>) or by the diversity of haplotypes across samples (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhang <em>et al.</em> 2002</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson and Novembre 2003</a>). Common to these methods is that only consecutive SNPs, which are often in high LD, are combined into haplotypes. Consequently, in many cases the haplotypes are not much more informative than a single SNP because the SNPs in high LD provide redundant information (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Laramie <em>et al.</em> 2007</a>). This may provide one explanation for the contradicting results reported in the literature comparing the power of haplotype- and SNP-based GWAS approaches. Other methods were developed to search for haplotypes consisting of most informative and possibly non-consecutive SNPs within a certain region (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Laramie <em>et al.</em> 2007</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yu and Schaid 2007</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abo <em>et al.</em> 2008</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dai <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Knuppel <em>et al.</em> 2012</a>). Despite their potential, the high computational burden associated with these methods restricted their use mainly to association studies for candidate gene regions.</p><p id="p-6">In this study we addressed these limitations by developing a new method of constructing haplotypes, taking epistatic effects among SNPs into account. Epistasis has been identified as an important contributor to the genetic variation of complex quantitative traits (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carlborg and Haley 2004</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mackay 2014</a>). It has been reported for two- or three-locus examples that a model involving haplotype effects can be reparametrized into one including the main and epistatic effects among markers constituting the haplotypes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Conti and Gauderman 2004</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Schaid 2004</a>). More recently this relationship between haplotype and marker effects was formally proved in the framework of genome-wide prediction for homozygous populations (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jiang <em>et al.</em> 2018</a>). Capitalizing on these theoretical findings, we exploited epistatic effects among markers for constructing haplotypes and implemented this novel strategy in haplotype-based GWAS for a large <em>Arabidopsis thaliana</em> population generated by the 1001 Genomes Consortium (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">1001 Genomes Consortium 2016</a>). The results were compared to those obtained with two commonly used haplotype-based GWAS methods as well as the single SNP-based approach, and underlined the ability to detect hidden marker-trait associations using the newly devised strategy. Moreover, simulation studies revealed factors which determine whether the developed GWAS approach outperforms the single SNP-based method.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">Throughout the manuscript, a combination of (possibly non-consecutive) SNPs was termed haplotype. By haplotype effect we meant to consider the effects of all possible alleles together. When referring to a specific allele, the term haplotype allele was used.</p><div id="sec-2" class="subsection"><h3>The baseline model for genome-wide association mapping</h3><p id="p-8">A standard linear mixed model controlling the structure of genetic relatedness or the polygenic background effects (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yu <em>et al.</em> 2006</a>) was used for genome-wide association mapping. In this study the model was used for testing single SNP effects, epistatic effects among several SNPs and haplotype effects. It can be uniformly described as following:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where y is a n-dimensional vector of observed phenotypic values (n is the number of genotypes), <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-2.gif"/></noscript></span></span> is a vector of one’s, μ is a common intercept term, β represents the effects of the variables (SNPs, interactions among SNPs or haplotype alleles) being tested, X stands for the corresponding design matrix, g is the n-dimensional vector of genotypic effects and e is the residual term. In the model we assume that μ and β are fixed effects, g and e are random effects and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-3.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-4.gif"/></noscript></span></span>, where K is a marker-derived kinship matrix, I is the identity matrix, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-6.gif"/></noscript></span></span> are the corresponding variance components. Distance matrix was calculated with Rogers’ distance (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Reif <em>et al.</em> 2005</a>) and K was equaling one minus distance matrix. To reduce the computational load, an acceleration algorithm was implemented in which the linear mixed model was transferred to a simple linear model by applying eigen-decomposition to the kinship matrix (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lippert <em>et al.</em> 2011</a>). The significance of β was assessed by <em>t</em>-test.</p></div><div id="sec-3" class="subsection"><h3>The general procedure of functional haplotype-based GWAS (FH GWAS)</h3><p id="p-9">In genomic prediction, it was demonstrated that modeling haplotype effects is equivalent to modeling main and epistatic effects among markers within the haplotype block, except that the two models assume different covariance structures for the unknown parameters (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jiang <em>et al.</em> 2018</a>). The theory also applies to GWAS and in this case the two models are strictly equivalent because the parameters to be tested are assumed to be fixed effects (Equation 1) and hence without any covariance structure. Based on this theory, we developed a new haplotype-based GWAS approach, FH GWAS, with haplotypes based on the main and epistatic effects among SNPs. FH GWAS consists of the following four steps summarized in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The workflow for FH GWAS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-19753087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The workflow for FH GWAS.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4115/F1.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/4115/F1.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494916" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">The workflow for FH GWAS.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-4" class="subsection"><h4>Step 1: Preselecting SNPs to be combined into haplotypes:</h4><p id="p-11">GWAS for single SNPs is performed using the linear mixed model (Equation 1) and a mild threshold without correction for multiple testing is applied to identify candidate SNPs (<em>e.g.</em>, <em>P</em> &lt; 0.01). SNPs whose P values do not pass the threshold are excluded in subsequent analyses.</p></div><div id="sec-5" class="subsection"><h4>Step 2: Constructing functional haplotypes:</h4><p id="p-12">In this step, candidate SNPs showing significant local epistatic effects are grouped into haplotypes. First we need to determine two parameters: the window size for searching haplotypes (denoted by w) and the number of SNPs in each haplotype (denoted by l). Theoretically the choice of these two parameters can be arbitrary. But in practice one needs to consider the linkage disequilibrium in the population, the computational load and the power of the association test (More details were discussed in the Discussion section). Once the parameters are chosen, GWAS model (Equation 1) is then performed for any l-tuple of SNPs within the window size w, including the additive effects of each SNP and the digenic epistatic effects for each pair of SNPs. That is, the entries in the vector β include <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-7.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-8.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-9.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-10.gif"/></noscript></span></span>), where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-11.gif"/></noscript></span></span> denotes the additive effect of the i-th SNP, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-12.gif"/></noscript></span></span> denotes the epistatic effects between the i-th and the j-th SNP. In the case that <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-13.gif"/></noscript></span></span> is small, higher-order epistatic effects can also be included in the model. Next we determine the number of significant additive and epistatic effects (again under a mild threshold) required for grouping the l-tuple of SNPs into a haplotype, <em>i.e.</em>, when there are at least s (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-14.gif"/></noscript></span></span>) significant additive effects and t (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-15.gif"/></noscript></span></span>) significant epistatic effects, the l-tuple of SNPs is combined into a haplotype. Since main as well as epistatic effects are taken into account for haplotype construction we coined the term functional haplotypes. Importantly, for each trait to be analyzed in a given population, a different set of candidate SNPs and functional haplotypes will be obtained.</p></div><div id="sec-6" class="subsection"><h4>Step 3: GWAS using functional haplotypes:</h4><p id="p-13">All resulting functional haplotypes are applied in conjunction with phenotypic data for GWAS using the linear mixed model (Equation 1). Significant functional haplotypes are then identified using a stringent genome-wide threshold corrected for multiple testing, <em>e.g.</em>, <em>P</em> &lt; 0.05 after Bonferroni (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dunn 1961</a>) or Benjamini-Hochberg correction (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benjamini and Hochberg 1995</a>).</p></div><div id="sec-7" class="subsection"><h4>Step 4: Narrowing Down candidate regions:</h4><p id="p-14">In each region in which significant functional haplotypes are detected in Step 3, we fitted all significant functional haplotypes in a variable-selection model (<em>e.g.</em>, the stepwise linear regression model (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Draper and Smith 2014</a>) or the least absolute shrinkage and selection operator (LASSO, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tibshirani 1996</a>) to select representative significant functional haplotypes. In any region where significant functional haplotypes are found, the span of all representative haplotypes is considered as a final candidate region.</p></div></div><div id="sec-8" class="subsection"><h3>Two other methods constructing haplotypes</h3><p id="p-15">To compare FH GWAS with existing haplotype-based GWAS approaches, we considered the following two commonly-used methods of constructing haplotypes (Figure S1).</p><div id="sec-9" class="subsection"><h4>The overlapping sliding-window approach:</h4><p id="p-16">The sliding window approach constructs haplotypes with a fixed window length, <em>i.e.</em>, the number of adjacent SNPs (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Huang <em>et al.</em> 2007</a>). If the window is moved with a certain step size which is smaller than the window length adjacent windows are overlapping. In this study we chose the window length to be three, which is consistent with the length of our functional haplotypes, and the step size to be one.</p></div><div id="sec-10" class="subsection"><h4>The linkage disequilibrium approach:</h4><p id="p-17">The linkage disequilibrium (LD) approach groups SNPs into a haplotype if the LD between every two adjacent SNPs is equal or greater than a certain threshold, which allows physically close and non-randomly associated SNPs to be grouped together in the same haplotype (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Barrett <em>et al.</em> 2005</a>). In our study, the r<sup>2</sup> statistic was used to measure LD (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hill and Robertson 1968</a>) and the threshold was set to 0.9. Note, that in this method the constructed haplotypes may have different lengths. In all cases in which SNPs were not grouped into any haplotype, the single SNPs were considered as haplotypes of length one.</p><p id="p-18">GWAS based on haplotypes constructed via the above two methods are referred as SWH GWAS and LDH GWAS respectively. The significance of haplotypes was also tested using the linear mixed model (Equation 1).</p></div></div><div id="sec-11" class="subsection"><h3>Data sets</h3><p id="p-19">The study was based on published data of <em>Arabidopsis thaliana</em> from the 1001 Genomes Consortium (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">1001 Genomes Consortium 2016</a>). The genotypic data contained 1,134 accessions with 11,458,975 single-nucleotide polymorphisms (SNPs). The phenotypic data that were considered was flowering time for plants grown at two different temperatures (10° and 16°), which included phenotypic values for 1,163 and 1,123 accessions respectively. Combining the genotypic and phenotypic data, 1,003 (970) accessions were used in the 10° (16°) data set. In the following the two data sets were referred as data set FT10 (10°) and FT16 (16°) respectively. Only bi-allelic SNPs were considered for the analyses. After removing the SNPs with missing rate above or equaling 0.1, the remaining missing values were imputed with IMPUTE2 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Howie <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Howie <em>et al.</em> 2012</a>). Linkage phases were determined by SHAPEIT (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Delaneau <em>et al.</em> 2011</a>). SNPs with minor allele frequency (MAF) below 0.05 were also removed. For subsequent analyses the resulting 756,005 and 754,656 SNPs were used for data set FT10 and FT16, respectively.</p></div><div id="sec-12" class="subsection"><h3>Comparing FH GWAS with other methods using empirical data</h3><p id="p-20">We compared the performance of FH GWAS with that of SWH GWAS, LDH GWAS and the single SNP-based approach with the <em>Arabidopsis thaliana</em> data sets described in the previous section. The genome-wide thresholds for the different approaches were generally determined as <em>P</em> &lt; 0.05 after Bonferroni correction for multiple testing (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dunn 1961</a>). Thus for the SNP, SWH and LDH GWAS approaches, the thresholds were <em>P</em> &lt; 0.05/m, where m is the number of SNPs or haplotypes constructed in total. The proportion of phenotypic variance explained by each of the significant SNPs or haplotypes was calculated as the adjusted <em>R<sup>2</sup></em> in a linear regression model with intercept and the testing variable. For regions in which significant associations were detected in GWAS, annotated genes were retrieved from Araport11 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cheng <em>et al.</em> 2017</a>).</p><div id="sec-13" class="subsection"><h4>Implementation of FH GWAS:</h4><p id="p-21">In the procedure of preselecting SNPs, we filtered the markers with the threshold <em>P</em> &lt; 0.01. Then we set the window size for searching haplotypes to be 50 kb and the number of SNPs in each haplotype to be three. Thus the linear mixed model (1) was performed for any triplet of candidate SNPs within 50 kb, testing the additive effects of each SNP, the epistatic effects for each pair of SNPs and the three-way epistatic effects. If at least two of the additive effects were significant with <em>P</em> &lt; 0.05 and at least two of the pairwise epistatic effects were significant with <em>P</em> &lt; 0.1, the triplet of SNPs was grouped into a functional haplotype. In the test of all functional haplotypes, we again applied the Bonferroni correction for multiple testing. But the threshold for FH GWAS needed further adjustment to account for the pre-testing procedure for single SNP effects and epistatic effects. So a more stringent threshold was determined as <em>P</em> &lt; 0.05/(m+c), where m is the number of functional haplotypes and c is the number of tests performed in the pre-testing procedure. To select representative significant functional haplotypes, we used the stepwise linear regression model (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Draper and Smith 2014</a>) and applied a bidirectional elimination procedure minimizing the Schwarz Bayesian Criterion (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schwarz 1978</a>).</p></div></div><div id="sec-14" class="subsection"><h3>GWAS considering markers in perfect LD</h3><p id="p-22">SNPs in perfect LD (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4115/embed/mml-math-16.gif"/></noscript></span></span>) are virtually identical in GWAS models in the sense that they have the same estimated effects and P values. Thus for each group of SNPs in perfect LD, we recorded their positions and performed only one test in GWAS. This approach was termed SNP<sub>LD</sub> GWAS. Let n<sub>LD</sub> be the number of SNPs adjusted for perfect LD, meaning that SNPs in perfect LD were counted only once. Then the threshold for SNP<sub>LD</sub> GWAS was <em>P</em> &lt; 0.05/n<sub>LD</sub>.</p><p id="p-23">For any two haplotypes consisting of three SNPs, they may share k SNPs (k = 0, 1, 2). If the remaining 3-k pairs of SNPs are in perfect LD respectively, the two haplotypes can be treated as identical in GWAS for the same reason as above. Thus our FH GWAS approach can also be adjusted by considering SNPs in perfect LD, which was termed FH<sub>LD</sub> GWAS. Let m<sub>LD</sub> be the adjusted number of functional haplotypes and c<sub>LD</sub> be the adjusted number of tests performed in the pre-testing procedure. Then the new threshold for FH<sub>LD</sub> GWAS was determined as <em>P</em> &lt; 0.05/(m<sub>LD</sub>+c<sub>LD</sub>).</p></div><div id="sec-15" class="subsection"><h3>Decay of linkage disequilibrium</h3><p id="p-24">The genome-wide decay of LD in the population of data set FT10 was estimated by a non-linear regression model using Hill and Weir’s function (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hill and Weir 1988</a>). The same method was used to estimate the decay of LD for the five candidate regions detected in GWAS.</p></div><div id="sec-16" class="subsection"><h3>Simulation study</h3><p id="p-25">Phenotypic data were simulated based on genotypic data of the 1,003 <em>Arabidopsis thaliana</em> accessions described previously (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">1001 Genomes Consortium 2016</a>). Considering computational load, the simulations were restricted to all bi-allelic SNPs mapping to chromosome 2 regardless of MAF, in total 279,038 SNPs. In the simulation procedure three SNPs were always selected within a 50 kb window and main and epistatic effects were assigned to them. To clarify the influence of LD and MAF on the performance of haplotype-based GWAS, three ranges of LD (0-0.2, 0.3-0.6 and 0.7-1) between each pair of selected SNPs, three ranges of MAF (0-0.1, 0.2-0.3 and 0.4-0.5) for the selected SNPs and combinations thereof were considered. For each of the resulting nine simulation scenarios, the main effect of each SNP, the epistatic effects between each pair of SNPs and the three-term interaction effect were set to account for 6%, 3% and 1% of the explained proportion of genetic variance, respectively and the heritability was set to be 0.8. All remaining SNPs were required to contribute equally to the remaining proportion of explained genetic variance to simulate genetic background. Based on the resulting simulated phenotypic data, association mapping was performed (model 1) using the three SNPs of a particular haplotype individually and the haplotype. For each scenario, simulations were repeated 1,000 times.</p></div><div id="sec-17" class="subsection"><h3>Data availability</h3><p id="p-26">All data analyzed in this study have been published previously (1001 Genome Consortium 2016). Phenotypic data were downloaded from AraPheno (<a href="https://arapheno.1001genomes.org/phenotypes/?sort=study&amp;#x0026;page=1">https://arapheno.1001genomes.org/phenotypes/?sort=study&amp;page=1</a>). Genomic data were downloaded from the 1001 Genomes data center (<a href="http://1001genomes.org/data/GMI-MPI/releases/v3.1/">http://1001genomes.org/data/GMI-MPI/releases/v3.1/</a>, the data ‘1001genomes_snp-short-indel_only_ACGTN.v<em>cf</em>.gz’ was used in this study). FH GWAS was implemented using R (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">R Core Team 2017</a>). The source code and sample data sets which are subsets of the original data set for running the code can be found at <a href="https://github.com/Fangv1/Functional_haplotype_GWAS/tree/master/">https://github.com/Fangv1/Functional_haplotype_GWAS/tree/master/</a>. Supplemental files are available at figshare. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.8967986">https://doi.org/10.25387/g3.8967986</a>.</p></div></div><div class="section results" id="sec-18"><h2 class="">Results</h2><div id="sec-19" class="subsection"><h3>FH GWAS outperformed SNP-based and two other haplotype-based GWAS approaches</h3><p id="p-27">In this study, data for flowering time in <em>Arabidopsis thaliana</em> accessions that had been cultivated at 10° and 16° were analyzed (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">1001 Genomes Consortium 2016</a>), these compilations are referred to as data sets FT10 and FT16, respectively. Data set FT10 encompassed 1,003 accessions and after quality control 756,005 biallelic SNPs remained for subsequent analyses. To assess the performance of the proposed FH GWAS, we compared its results to those of single SNP-based GWAS and two other haplotype-based approaches in which haplotypes were either constructed using sliding-windows (SWH GWAS) or by considering LD of consecutive SNPs (LDH GWAS). The number of SNPs grouped into haplotypes and number of haplotypes analyzed in GWAS varied between the three approaches (Table S1). Applying Bonferroni correction for multiple testing (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Dunn 1961</a>) (<em>P</em> &lt; 0.05) resulted in significance thresholds of -log<sub>10</sub><em>P</em> = 7.03, -log<sub>10</sub><em>P</em> = 7.18 and -log<sub>10</sub><em>P</em> = 6.50 for LDH, SWH and FH GWAS, respectively. But for FH GWAS it was necessary to apply a further correction to account for the pre-testing procedure for single SNP effects and epistatic effects which preceded the construction of functional haplotypes (see <strong>Materials and Methods</strong> for details). Implementing this correction resulted in a more stringent threshold of -log<sub>10</sub><em>P</em> = 8.29. Applying the different GWAS approaches, significant associations were found in five chromosome regions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>-<a id="xref-fig-2-2" class="xref-fig" href="#F2">2D</a>). Importantly, all regions that were identified by SNP-based, LDH and/or SWH GWAS were also found by FH GWAS. Four regions, I, III, IV and V, were identified by all methods, but region II on chromosome 4 solely showed significant association with flowering time using FH GWAS. For each of the significant functional haplotypes detected in region II, an additional association test was performed with only the main effects of the three SNPs in the haplotype. We found that in all cases the –log(P) values decreased by three to five orders of magnitude. This clearly showed the important contribution of epistatic effects to the overall effect of a functional haplotype.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Association mapping results using four different GWAS approaches. (A-D) Manhattan plots illustrate the results for a single SNP-based (A) and three haplotype-based GWAS approaches for data set FT10 (B-D). Positions of SNPs or haplotypes on the five chromosomes are shown on the x axis relative to their -log10(P) values on the y axis. Haplotypes were constructed based on overlapping sliding-windows (B), linkage disequilibrium (C) or by using the functional haplotype approach (D). Thresholds after Bonferroni correction for multiple testing (Dunn 1961) (P &lt; 0.05) are displayed as horizontal dotted gray lines. Taking into account the pre-testing procedure for single SNP main and epistatic effects implemented in the functional haplotype approach a more stringent threshold resulted that is indicated as a pale blue dotted line in panels (D) and (G). (E-G) Plots showing the -log10(P) values of haplotypes on the y axis relative to the -log10(P) values established by SNP-based GWAS for the most significant SNP of a corresponding haplotype on the x axis. The P value relationships for SWH, LDH and FH based GWAS are illustrated in panels (E), (F) and (G), respectively. Thresholds after Bonferroni correction for multiple testing (Dunn 1961) (P &lt; 0.05) are indicated as horizontal dotted dark blue lines for haplotypes and vertical dotted red lines for single SNPs. The five regions in which significant associations were found were denoted with I to V and are marked by stippled lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-19753087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Association mapping results using four different GWAS approaches. (A-D) Manhattan plots illustrate the results for a single SNP-based (A) and three haplotype-based GWAS approaches for data set FT10 (B-D). Positions of SNPs or haplotypes on the five chromosomes are shown on the x axis relative to their -log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt;) values on the y axis. Haplotypes were constructed based on overlapping sliding-windows (B), linkage disequilibrium (C) or by using the functional haplotype approach (D). Thresholds after Bonferroni correction for multiple testing (Dunn 1961) (&lt;em&gt;P&lt;/em&gt; &lt; 0.05) are displayed as horizontal dotted gray lines. Taking into account the pre-testing procedure for single SNP main and epistatic effects implemented in the functional haplotype approach a more stringent threshold resulted that is indicated as a pale blue dotted line in panels (D) and (G). (E-G) Plots showing the -log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt;) values of haplotypes on the y axis relative to the -log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt;) values established by SNP-based GWAS for the most significant SNP of a corresponding haplotype on the x axis. The P value relationships for SWH, LDH and FH based GWAS are illustrated in panels (E), (F) and (G), respectively. Thresholds after Bonferroni correction for multiple testing (Dunn 1961) (&lt;em&gt;P&lt;/em&gt; &lt; 0.05) are indicated as horizontal dotted dark blue lines for haplotypes and vertical dotted red lines for single SNPs. The five regions in which significant associations were found were denoted with I to V and are marked by stippled lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4115/F2.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/4115/F2.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495025" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Association mapping results using four different GWAS approaches. (A-D) Manhattan plots illustrate the results for a single SNP-based (A) and three haplotype-based GWAS approaches for data set FT10 (B-D). Positions of SNPs or haplotypes on the five chromosomes are shown on the x axis relative to their -log<sub>10</sub>(<em>P</em>) values on the y axis. Haplotypes were constructed based on overlapping sliding-windows (B), linkage disequilibrium (C) or by using the functional haplotype approach (D). Thresholds after Bonferroni correction for multiple testing (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Dunn 1961</a>) (<em>P</em> &lt; 0.05) are displayed as horizontal dotted gray lines. Taking into account the pre-testing procedure for single SNP main and epistatic effects implemented in the functional haplotype approach a more stringent threshold resulted that is indicated as a pale blue dotted line in panels (D) and (G). (E-G) Plots showing the -log<sub>10</sub>(<em>P</em>) values of haplotypes on the y axis relative to the -log<sub>10</sub>(<em>P</em>) values established by SNP-based GWAS for the most significant SNP of a corresponding haplotype on the x axis. The P value relationships for SWH, LDH and FH based GWAS are illustrated in panels (E), (F) and (G), respectively. Thresholds after Bonferroni correction for multiple testing (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Dunn 1961</a>) (<em>P</em> &lt; 0.05) are indicated as horizontal dotted dark blue lines for haplotypes and vertical dotted red lines for single SNPs. The five regions in which significant associations were found were denoted with I to V and are marked by stippled lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">For each haplotype the P value of the SNP for which the lowest P value had been observed in single SNP-based GWAS was compared to the one of the corresponding haplotype. The proportion of haplotypes showing significant associations that contained at least one SNP, which had passed the significance threshold in SNP-based GWAS, varied between the three different haplotype-based GWAS approaches (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2E</a>-<a id="xref-fig-2-4" class="xref-fig" href="#F2">2G</a>). The highest proportion was found with 90.91% for LDH GWAS and the lowest one with 29.33% for FH GWAS (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495051/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495051/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Summary of significant associations in different genome-wide association studies obtained for the trait flowering time for <em>Arabidopsis thaliana</em> accessions using data set FT10</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Accounting for linkage disequilibrium in functional haplotype-based GWAS</h3><p id="p-32">Strikingly, FH GWAS identified several thousand significant haplotypes whereas SNP-based GWAS and the other two haplotype-based GWAS approaches revealed few significant associations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). Inspection of the significant functional haplotypes in a given chromosome region revealed many subsets sharing one or two SNPs. For example, nine of the 15 significant haplotypes in region II had two SNPs in common. Moreover, the SNPs distinguishing these nine significant haplotypes were in high LD to each other (Figure S2). This exemplifies that many different significant functional haplotypes may result in cases in which significant haplotypes are made up of SNPs which are in high LD with other SNPs in the region. Taking into account which SNPs are in perfect LD to each other it is possible to restrict the FH GWAS analysis to those haplotypes which provide non-redundant information regarding additive and epistatic effects, called FH<sub>LD</sub> GWAS hereafter. In data set FT10, the number of SNP combinations to be tested could be reduced in this manner from 8,932,265 to 2,460,993. Instead of 157,526 functional haplotypes in FH GWAS only 44,759 resulted in FH<sub>LD</sub> GWAS. However, owing to a less stringent threshold of -log<sub>10</sub><em>P</em> = 7.79 for FH<sub>LD</sub> GWAS compared to -log<sub>10</sub><em>P</em> = 8.29 for FH GWAS the number of significant associations increased (Table S2). Regardless whether FH GWAS or FH<sub>LD</sub> GWAS were used multiple significant haplotypes were found in regions I to V. In addition, a single haplotype passed the significance threshold in FH<sub>LD</sub> GWAS on chromosome 3 (Table S2, <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, Figure S3).</p></div><div id="sec-21" class="subsection"><h3>Representative significant haplotypes narrowed down the candidate regions</h3><p id="p-33">Depending on the region, the mean size of the significant functional haplotypes, defined as the distance in base pairs between the outermost SNPs of a particular haplotype, varied from 10.6 to 41.3 kb in FH GWAS. Moreover, the size of chromosome segments in which overlapping significant functional haplotypes were found differed, ranging from 54.3 to 167.2 kb (Table S3). The sizes of the significant functional haplotypes in conjunction with their high number hampered the search for candidate genes. Variable selection methods were therefore used to reduce the number of significant functional haplotypes (see Materials and Methods for details). For data set FT10, two to six and two to eight representative haplotypes were selected per region in FH GWAS and FH<sub>LD</sub> GWAS, respectively (Table S2). Taking into account the overlaps between all representative significant functional haplotypes of a given region and/or the area between them, small regions with few genes were detected (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, Figure S4, Table S4). In all cases a candidate gene was identified among these genes for which a role in flowering time control had been documented previously. <em>FT</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kardailsky <em>et al.</em> 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Corbesier <em>et al.</em> 2007</a>) represents a candidate gene for region I on chromosome 1 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). <em>DOG1</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Huo <em>et al.</em> 2016</a>) and <em>FLC</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Michaels and Amasino 1999</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2014</a>) are part of regions III and IV on chromosome 5, respectively (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). In these three areas, the significant SNPs that were significantly associated with the trait flowering time were also located in the candidate gene itself or in its immediate vicinity. It is important to note that the proportions of phenotypic variance explained by the representative significant haplotypes were in four out of five analyzed regions higher than those determined for any of the SNPs in these chromosome segments (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, Table S5). In region V on chromosome 5, the only SNP significantly associated with the trait flowering time mapped approximately 60 kb apart from the region with the candidate gene <em>VIN3</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sung and Amasino 2004</a>) which was indicated by the representative significant functional haplotypes in FH GWAS (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). In region II that had only been detected using FH GWAS and FH<sub>LD</sub> GWAS, <em>CCT</em>/<em>CRP</em>/<em>MED12</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Imura <em>et al.</em> 2012</a>) was identified as candidate gene (Figure S2, Figure S4).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Details of significant associations for the trait flowering time revealed by SNP-based and functional haplotype-based GWAS in two chromosome regions. Panels (A) and (B) refer to the analysis of data set FT10 for regions I and V, respectively. SNP positions on the different chromosomes are shown on the x axis relative to the corresponding -log10(P) values on the y axis. The depicted regions reflect the chromosome segments for which overlapping functional haplotypes had been obtained, but only those functional haplotypes are shown which passed the stringent adjusted significance threshold of -log10(P) = 8.29 as gray or pink lines. Pink lines highlight representative significant functional haplotypes. The positions of the first and third SNP of a particular haplotype on the chromosome mark the beginning and end of the line, respectively. A colored triangle indicates the SNP of a haplotype for which the lowest P value was observed by SNP-based GWAS. P values ranging from 1 &#xD7; 10&#x2212;4 to 1 &#xD7; 10&#x2212;2, 1 &#xD7; 10&#x2212;6 to 1 &#xD7; 10&#x2212;4, 1 &#xD7; 10&#x2212;8 to 1 &#xD7; 10&#x2212;6 are represented as black, orange and green triangles, respectively. Blue triangles represent P values smaller than 1 &#xD7; 10&#x2212;8. The translucent pale blue and red dots correspond to the P values of SNPs obtained in single SNP-based GWAS, red dots represent those SNPs that were part of significant functional haplotypes. Below the x axis the coding regions of genes in the region are shown as gray boxes, 5&#x2032;-regions are indicated as red lines. Two vertical pink dashed lines are used to mark the position of the coding region of the candidate gene. The red and blue horizontal stippled lines correspond to the significance thresholds for single SNP-based and FH GWAS, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-19753087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Details of significant associations for the trait flowering time revealed by SNP-based and functional haplotype-based GWAS in two chromosome regions. Panels (A) and (B) refer to the analysis of data set FT10 for regions I and V, respectively. SNP positions on the different chromosomes are shown on the x axis relative to the corresponding -log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt;) values on the y axis. The depicted regions reflect the chromosome segments for which overlapping functional haplotypes had been obtained, but only those functional haplotypes are shown which passed the stringent adjusted significance threshold of -log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt;) = 8.29 as gray or pink lines. Pink lines highlight representative significant functional haplotypes. The positions of the first and third SNP of a particular haplotype on the chromosome mark the beginning and end of the line, respectively. A colored triangle indicates the SNP of a haplotype for which the lowest P value was observed by SNP-based GWAS. P values ranging from 1 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt; to 1 &#xD7; 10&lt;sup&gt;&#x2212;2&lt;/sup&gt;, 1 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt; to 1 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;, 1 &#xD7; 10&lt;sup&gt;&#x2212;8&lt;/sup&gt; to 1 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt; are represented as black, orange and green triangles, respectively. Blue triangles represent P values smaller than 1 &#xD7; 10&lt;sup&gt;&#x2212;8&lt;/sup&gt;. The translucent pale blue and red dots correspond to the P values of SNPs obtained in single SNP-based GWAS, red dots represent those SNPs that were part of significant functional haplotypes. Below the x axis the coding regions of genes in the region are shown as gray boxes, 5&#x2032;-regions are indicated as red lines. Two vertical pink dashed lines are used to mark the position of the coding region of the candidate gene. The red and blue horizontal stippled lines correspond to the significance thresholds for single SNP-based and FH GWAS, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4115/F3.medium.gif" width="301" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/4115/F3.medium.gif" width="301" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494923" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Details of significant associations for the trait flowering time revealed by SNP-based and functional haplotype-based GWAS in two chromosome regions. Panels (A) and (B) refer to the analysis of data set FT10 for regions I and V, respectively. SNP positions on the different chromosomes are shown on the x axis relative to the corresponding -log<sub>10</sub>(<em>P</em>) values on the y axis. The depicted regions reflect the chromosome segments for which overlapping functional haplotypes had been obtained, but only those functional haplotypes are shown which passed the stringent adjusted significance threshold of -log<sub>10</sub>(<em>P</em>) = 8.29 as gray or pink lines. Pink lines highlight representative significant functional haplotypes. The positions of the first and third SNP of a particular haplotype on the chromosome mark the beginning and end of the line, respectively. A colored triangle indicates the SNP of a haplotype for which the lowest P value was observed by SNP-based GWAS. P values ranging from 1 × 10<sup>−4</sup> to 1 × 10<sup>−2</sup>, 1 × 10<sup>−6</sup> to 1 × 10<sup>−4</sup>, 1 × 10<sup>−8</sup> to 1 × 10<sup>−6</sup> are represented as black, orange and green triangles, respectively. Blue triangles represent P values smaller than 1 × 10<sup>−8</sup>. The translucent pale blue and red dots correspond to the P values of SNPs obtained in single SNP-based GWAS, red dots represent those SNPs that were part of significant functional haplotypes. Below the x axis the coding regions of genes in the region are shown as gray boxes, 5′-regions are indicated as red lines. Two vertical pink dashed lines are used to mark the position of the coding region of the candidate gene. The red and blue horizontal stippled lines correspond to the significance thresholds for single SNP-based and FH GWAS, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportions of explained phenotypic variance for the trait flowering time obtained by SNP-based and functional haplotype based GWAS in two different chromosome regions. Details for regions III and IV are illustrated for data set FT10 in panels (A) and (B), respectively. SNP positions on chromosome 5 and percentages of adjusted R2 values are shown on the x and y axes, respectively. Chromosome segments are illustrated for which overlapping functional haplotypes had been obtained, but only significant functional haplotypes are displayed as gray or pink lines. Representative significant functional haplotypes are indicated by pink lines. The beginning and end of the individual lines represent the chromosome positions of the first and third SNP of a particular haplotype, respectively. The SNP for which the lowest P value of a given significant functional haplotype was obtained is indicated as a colored triangle. Black, orange and green triangles represent P values ranging from 1 &#xD7; 10&#x2212;4 to 1 &#xD7; 10&#x2212;2, 1 &#xD7; 10&#x2212;6 to 1 &#xD7; 10&#x2212;4, 1 &#xD7; 10&#x2212;8 to 1 &#xD7; 10&#x2212;6, respectively. Blue triangles mark P values smaller than 1 &#xD7; 10&#x2212;8. Percentages of R2 determined for SNPs are displayed as translucent pale blue or red dots, those SNPs that were part of significant functional haplotypes are depicted in red. The coding regions of genes are shown as gray boxes and red lines represent 5&#x2032;-regions. The position of the coding region of the candidate gene is marked by two vertical pink dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-19753087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportions of explained phenotypic variance for the trait flowering time obtained by SNP-based and functional haplotype based GWAS in two different chromosome regions. Details for regions III and IV are illustrated for data set FT10 in panels (A) and (B), respectively. SNP positions on chromosome 5 and percentages of adjusted R&lt;sup&gt;2&lt;/sup&gt; values are shown on the x and y axes, respectively. Chromosome segments are illustrated for which overlapping functional haplotypes had been obtained, but only significant functional haplotypes are displayed as gray or pink lines. Representative significant functional haplotypes are indicated by pink lines. The beginning and end of the individual lines represent the chromosome positions of the first and third SNP of a particular haplotype, respectively. The SNP for which the lowest P value of a given significant functional haplotype was obtained is indicated as a colored triangle. Black, orange and green triangles represent P values ranging from 1 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt; to 1 &#xD7; 10&lt;sup&gt;&#x2212;2&lt;/sup&gt;, 1 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt; to 1 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;, 1 &#xD7; 10&lt;sup&gt;&#x2212;8&lt;/sup&gt; to 1 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;, respectively. Blue triangles mark P values smaller than 1 &#xD7; 10&lt;sup&gt;&#x2212;8&lt;/sup&gt;. Percentages of R&lt;sup&gt;2&lt;/sup&gt; determined for SNPs are displayed as translucent pale blue or red dots, those SNPs that were part of significant functional haplotypes are depicted in red. The coding regions of genes are shown as gray boxes and red lines represent 5&#x2032;-regions. The position of the coding region of the candidate gene is marked by two vertical pink dashed lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4115/F4.medium.gif" width="288" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/12/4115/F4.medium.gif" width="288" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494927" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Proportions of explained phenotypic variance for the trait flowering time obtained by SNP-based and functional haplotype based GWAS in two different chromosome regions. Details for regions III and IV are illustrated for data set FT10 in panels (A) and (B), respectively. SNP positions on chromosome 5 and percentages of adjusted R<sup>2</sup> values are shown on the x and y axes, respectively. Chromosome segments are illustrated for which overlapping functional haplotypes had been obtained, but only significant functional haplotypes are displayed as gray or pink lines. Representative significant functional haplotypes are indicated by pink lines. The beginning and end of the individual lines represent the chromosome positions of the first and third SNP of a particular haplotype, respectively. The SNP for which the lowest P value of a given significant functional haplotype was obtained is indicated as a colored triangle. Black, orange and green triangles represent P values ranging from 1 × 10<sup>−4</sup> to 1 × 10<sup>−2</sup>, 1 × 10<sup>−6</sup> to 1 × 10<sup>−4</sup>, 1 × 10<sup>−8</sup> to 1 × 10<sup>−6</sup>, respectively. Blue triangles mark P values smaller than 1 × 10<sup>−8</sup>. Percentages of R<sup>2</sup> determined for SNPs are displayed as translucent pale blue or red dots, those SNPs that were part of significant functional haplotypes are depicted in red. The coding regions of genes are shown as gray boxes and red lines represent 5′-regions. The position of the coding region of the candidate gene is marked by two vertical pink dashed lines.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>FH GWAS for the trait flowering time at two different growth temperatures</h3><p id="p-36">Association studies in which the trait flowering time had been comparatively analyzed for accessions cultivated at 10° and 16° had revealed fewer significant SNP associations in the latter data set (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">1001 Genomes Consortium 2016</a>). It was therefore of interest to extend the performance comparisons of SNP-based and FH GWAS to data set FT16, in which phenotypic data for 970 accessions and 754,655 biallelic SNPs that had passed quality control had been compiled. Two regions showing significant associations were identified by SNP-based GWAS as well as FH GWAS using data set FT16 (Figure S5), these corresponded to regions III and IV that had also been found for plants cultivated at 10°. In contrast, region II on chromosome 4 was solely identified by FH GWAS, regardless which of the two data sets was analyzed (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, Figure S5). Significant associations in regions I and V were not found in data set FT16. However, for FT16, FH GWAS detected in three additional regions located on chromosomes 1, 2 and 3 between one and three significant haplotypes associated with the trait flowering time (Figure S5).</p><p id="p-37">The three regions, which were identified by FH GWAS in both data sets, were analyzed in more detail. A comparison of the results revealed that the chromosome segments in which the representative significant functional haplotypes were found showed large overlaps in both data sets (Table S4) implying that the same three candidate genes underlie the trait flowering time in these regions (Figure S6). Interestingly, such congruence was not observed if for example the candidate SNPs that were considered for the construction of functional haplotypes were compared in the two data sets. Although around 650 candidate SNPs were identified in each of the two data sets, only 390 were in common. Similar results were found by assessing the SNPs that were grouped into haplotypes (Table S6). Although 67,865 and 57,479 functional haplotypes had been considered in GWAS using the two data sets, only 3,606 were in common between both data sets. A similar trend was seen if only those haplotypes were considered that had passed the GWAS significance thresholds. None of the representative significant functional haplotypes were identical in the two data sets (Table S6).</p></div><div id="sec-23" class="subsection"><h3>The influence of linkage disequilibrium and minor allele frequencies on the power of functional haplotype-based GWAS</h3><p id="p-38">Simulation studies were performed to gain insight under which circumstances FH GWAS outperforms SNP-based GWAS. Specifically, it was analyzed how the minor allele frequency (MAF) of the SNPs making up a particular haplotype and the LD between them influenced the results of FH GWAS, therefore three MAF and LD ranges each as well as all of their combinations were considered (see <strong>Materials and Methods</strong> for details). The P value distributions obtained for the haplotypes using the nine different simulation scenarios are shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> side-by-side with the results for the most significant SNPs of the different haplotypes. Mean P values were inversely correlated with the MAF range in FH GWAS and SNP-based GWAS, regardless which LD range was analyzed. In scenarios in which the MAF range was kept constant, inverse correlations were seen between the mean P values and the LD range. Exceptionally, analysis of the highest MAF range revealed very similar mean P values in case of FH GWAS for the three different LD ranges. In four out of the nine scenarios tested, the mean P values obtained for FH GWAS clearly outperformed those of SNP-based GWAS, in each of these four scenarios more than 96% of the haplotypes revealed lower P values compared to the values that had been established by SNP-based GWAS for the most significant SNPs of these haplotypes (Table S7). This was not the case in the five scenarios in which the lowest MAF range and/or the highest LD range were analyzed. The same trends were observed regarding the proportion of phenotypic variance explained by the haplotypes and the most significant SNPs of the different haplotypes (Figure S7).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of P value distributions for FH GWAS and SNP-based GWAS obtained for nine different simulation scenarios. The violin plots show the distributions of P values after 1,000 simulation runs. Plots are arranged in order of decreasing MAF and increasing LD range. The y axis corresponds to the -log10(P values). &#x2018;A&#x2019; represents the P values of haplotypes and &#x2018;B&#x2019; the P values of the most significant SNPs of the particular haplotypes. The black vertical line corresponds to the 95% confidence interval and the black vertical box represents the interquartile range. The white and red dots mark the median and mean values, respectively. The latter values are indicated in the plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-19753087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of P value distributions for FH GWAS and SNP-based GWAS obtained for nine different simulation scenarios. The violin plots show the distributions of P values after 1,000 simulation runs. Plots are arranged in order of decreasing MAF and increasing LD range. The y axis corresponds to the -log&lt;sub&gt;10&lt;/sub&gt;(P values). &#x2018;A&#x2019; represents the P values of haplotypes and &#x2018;B&#x2019; the P values of the most significant SNPs of the particular haplotypes. The black vertical line corresponds to the 95% confidence interval and the black vertical box represents the interquartile range. The white and red dots mark the median and mean values, respectively. The latter values are indicated in the plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4115/F5.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/12/4115/F5.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4115/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495092" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Comparison of P value distributions for FH GWAS and SNP-based GWAS obtained for nine different simulation scenarios. The violin plots show the distributions of P values after 1,000 simulation runs. Plots are arranged in order of decreasing MAF and increasing LD range. The y axis corresponds to the -log<sub>10</sub>(P values). ‘A’ represents the P values of haplotypes and ‘B’ the P values of the most significant SNPs of the particular haplotypes. The black vertical line corresponds to the 95% confidence interval and the black vertical box represents the interquartile range. The white and red dots mark the median and mean values, respectively. The latter values are indicated in the plots.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-40">We devised a haplotype-based GWAS approach, FH GWAS, for studying complex quantitative traits which capitalizes on a novel way in which main and epistatic effects among markers are considered to group SNPs into haplotypes. In FH GWAS we first select SNPs with a mild threshold for main effects and then search for combinations of consecutive and/or non-consecutive SNPs in a genomic region of defined size requiring certain significance for epistatic effects. In this way, only those SNPs having true contribution to the haplotype effects via additive and/or epistatic effects are combined into functional haplotypes. Thus, FH GWAS is able to overcome the constraints of combining redundant SNPs in high LD into haplotypes and meanwhile it avoids exhausted search for optimal combinations of SNPs which is too time-consuming. It is therefore expected to be more powerful than SNP-based and other haplotype-based GWAS approaches, which was confirmed by the empirical analyses for the trait flowering time in <em>Arabidopsis thaliana</em> using the data from the 1001 Genomes Consortium (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">1001 Genomes Consortium 2016</a>). Our FH GWAS approach detected not only all regions, which were detected in the SNP-based and the other two haplotype-based approaches, but also a new candidate region on chromosome 4 for plants cultivated at 10° and 16° (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, Figure S5). The FH GWAS approach can be generally applied to any quantitative trait in any homozygous species for which populations with appropriate SNP coverage and of suitable size are available. If multiple traits are studied, the functional haplotypes have to be constructed for each trait separately as the tests of marker main and epistatic effects are trait-dependent. Thus, FH GWAS enhances the power of GWAS in a way that is tailored for each trait, however, it has a higher computational load than other haplotype-based GWAS approaches in which solely consecutive SNPs are considered for haplotype construction.</p><div id="sec-25" class="subsection"><h3>On the implementation of functional haplotype-based GWAS</h3><p id="p-41">The first step of FH GWAS is a mild preselection of SNPs according to their main effects in order to reduce the computational load for the remaining steps. Thus, it is necessary for high density SNP data sets generated for example by whole genome sequencing projects as used in this study (<a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">1001 Genomes Consortium 2016</a>). Theoretically, the significance of a haplotype effect can be solely a result of significant epistatic effects, or cumulative (non-)significant main and epistatic effects among the SNPs. The preselection of SNPs is therefore dispensable and can be omitted if the computational load is acceptable.</p><p id="p-42">In the second step of the procedure, the construction of functional haplotypes, there are two important parameters to be determined, namely the size of the window in which the functional haplotypes are constructed and the number of SNPs to be grouped into haplotypes. The window size is essentially determined by the extent of LD in the population, however, gene density should also be considered. A too small window size leads to high LD among markers within the window, reducing the advantage of haplotypes according to the results of the simulation study (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), whereas a too large window size may yield functional haplotypes that span large regions on the chromosome involving many candidate genes. For the <em>Arabidopsis thaliana</em> population considered in this study, the window size was set to be 50 kb, where the LD (measured as r<sup>2</sup>) decayed to 0.03 (Figure S8A). On average 22 genes mapped to intervals of this size in the <em>Arabidopsis thaliana</em> Col-0 genome (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arabidopsis Genome Initiative 2000</a>). Interestingly, in the region with the steepest LD decay (Figure S8B), region I, median and mean haplotype sizes were substantially smaller than in the other four regions (Table S3).</p><p id="p-43">The number of SNPs in each haplotype is directly relevant to the power of association test, which decreases as the number of haplotype alleles increases. Usually only a small number of SNPs can be afforded unless the population size is very large, because the number of haplotype alleles grows exponentially with an increasing number of SNPs constituting the haplotype. It is also limited by the computational load because allowing more SNPs in a haplotype results in more possible combinations of SNPs to be tested. Thus in this study the number of SNPs in each haplotype block was set to be three.</p></div><div id="sec-26" class="subsection"><h3>Functional haplotypes boosted power of GWAS by exploiting statistical epistasis</h3><p id="p-44">The construction of functional haplotypes rests upon interaction effects among markers, which was termed statistical epistasis in quantitative genetics (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Moore and Williams 2005</a>). In general, the estimation of statistical epistasis is not directly relevant to biological mechanisms of gene interactions (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Carlborg and Haley 2004</a>), although some simulation studies showed that various functional dependency patterns of genes could result in significant statistical epistasis (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gjuvsland <em>et al.</em> 2007</a>). As we observed many significant functional haplotypes consisting of SNPs with non-significant main effects even in the region where SNPs with strong main effect were detected (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>), a variable-selection algorithm was applied to select representative haplotypes. This step was of crucial importance to narrow down the regions, which needed to be inspected for the presence of candidate genes. In each candidate region, detailed analyses of the representative haplotypes revealed several distinct two- or three-locus genotype-phenotype patterns. Moreover, although three of the candidate genes were identified in two different data sets, none of the representative significant functional haplotypes were identical in these two data sets (Table S5). These findings made it unlikely that the statistical epistasis exploited by the significant haplotypes reflected a biological mechanism of gene interactions but also revealed that the cumulative statistical epistatic effects among SNPs in haplotypes indeed enhanced the power of FH-GWAS. Hence, the approach is useful for detecting new candidate regions, which cannot be detected using SNP-based or other haplotype-based GWAS approaches. Previously, haplotype-based methods were used to boost power in GWAS mainly for incomplete genotype data (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">McCarthy <em>et al.</em> 2008</a>), whereas our study showed that FH GWAS is a promising method even if almost complete genotype information is available such as whole-genome sequencing data.</p></div><div id="sec-27" class="subsection"><h3>Further development of functional haplotype-based GWAS</h3><p id="p-45">In this study, FH GWAS was applied to an <em>Arabidopsis thaliana</em> population consisting of pure homozygous lines. Hence, the haplotype phase was known and only the additive-by-additive epistasis was considered in the construction of functional haplotypes. A generalization of the FH GWAS method for heterozygous populations is possible as algorithms inferring haplotype phases (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Browning and Browning 2011</a>) can be applied if the haplotype phase is unknown. It may, however, be necessary to consider other types of epistasis, additive-by-dominance and dominance-by-dominance, when constructing functional haplotypes. Note, that in these cases the relationship between haplotype effects and marker epistatic effects was only illustrated in two- or three-locus examples but not formally proved in general case (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Conti and Gauderman 2004</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Schaid 2004</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Jiang <em>et al.</em> 2018</a>). Thus, further theoretical and empirical studies are needed to develop an optimal strategy of FH GWAS for heterozygous populations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">Fang Liu was supported with funds from China Scholarship Council (CSC). Yong Jiang was supported by the Federal Ministry of Education and Research of Germany (Grant FKZ031B0184A).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-47">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.8967986">https://doi.org/10.25387/g3.8967986</a>.</p></li><li class="fn" id="fn-4"><p id="p-48"><em>Communicating editor: P. Morrell</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-17"><span class="received-label">Received </span>June 17, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-08"><span class="accepted-label">Accepted </span>October 8, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Liu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.1" data-doi="10.1093/bioinformatics/btn359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abo</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Knight</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cox</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Camp</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">hapConstructor: automatic construction and testing of haplotypes in a Monte Carlo framework.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btn359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn359%26rft_id%253Dinfo%253Apmid%252F18653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18653522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.2" data-doi="10.1086/377106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">E. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Novembre</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Finding haplotype block boundaries by using the minimum-description-length principle.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/377106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAnderson%26rft.auinit1%253DE.%2BC.%26rft.volume%253D73%26rft.issue%253D2%26rft.spage%253D336%26rft.epage%253D354%26rft.atitle%253DFinding%2Bhaplotype%2Bblock%2Bboundaries%2Bby%2Busing%2Bthe%2Bminimum-description-length%2Bprinciple.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F377106%26rft_id%253Dinfo%253Apmid%252F12858289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/377106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12858289&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184223900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.3" data-doi="10.1038/35048692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Arabidopsis Genome Initiative</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">815</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35048692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D796%26rft.epage%253D815%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bflowering%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048692%26rft_id%253Dinfo%253Apmid%252F11130711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130711&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.4" data-doi="10.1093/bioinformatics/bth457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Fry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Maller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Daly</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Haploview: analysis and visualization of LD and haplotype maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">265</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bth457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth457%26rft_id%253Dinfo%253Apmid%252F15297300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15297300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226308500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hochberg</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate - a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.6" data-doi="10.1186/gb-2011-12-10-232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Morris</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Borevitz</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association studies in plants: the missing heritability is in the field.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011-12-10-232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D232%26rft.epage%253D232%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bplants%253A%2Bthe%2Bmissing%2Bheritability%2Bis%2Bin%2Bthe%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-232%26rft_id%253Dinfo%253Apmid%252F22035733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22035733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.7" data-doi="10.1038/nrg3054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">S. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype phasing: existing methods and new developments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">714</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D703%26rft.epage%253D714%26rft.atitle%253DHaplotype%2Bphasing%253A%2Bexisting%2Bmethods%2Band%2Bnew%2Bdevelopments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3054%26rft_id%253Dinfo%253Apmid%252F21921926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921926&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.8" data-doi="10.1016/S0168-9525(03)00022-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>,  <span class="cit-name-given-names">L. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Abecasis</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Using haplotype blocks to map human complex trait loci.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(03)00022-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DCardon%26rft.auinit1%253DL.%2BR.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D135%26rft.epage%253D140%26rft.atitle%253DUsing%2Bhaplotype%2Bblocks%2Bto%2Bmap%2Bhuman%2Bcomplex%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252803%252900022-2%26rft_id%253Dinfo%253Apmid%252F12615007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(03)00022-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12615007&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181584500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.9" data-doi="10.1038/nrg1407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>,  <span class="cit-name-given-names">O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Haley</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epistasis: too often neglected in complex trait studies?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">625</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D618%26rft.epage%253D625%26rft.atitle%253DEpistasis%253A%2Btoo%2Boften%2Bneglected%2Bin%2Bcomplex%2Btrait%2Bstudies%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1407%26rft_id%253Dinfo%253Apmid%252F15266344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223039200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.10" data-doi="10.1111/tpj.13415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Krishnakumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Thibaud-Nissen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schobel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Araport11: a complete reannotation of the <em>Arabidopsis thaliana</em> reference genome.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">804</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13415</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D789%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13415%26rft_id%253Dinfo%253Apmid%252F27862469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27862469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.11" data-doi="10.1002/gepi.20025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The role of haplotypes in candidate gene studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D321%26rft.epage%253D333%26rft.atitle%253DThe%2Brole%2Bof%2Bhaplotypes%2Bin%2Bcandidate%2Bgene%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20025%26rft_id%253Dinfo%253Apmid%252F15368617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15368617&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225524800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.12" data-doi="10.1002/gepi.20039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conti</span>,  <span class="cit-name-given-names">D. V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Gauderman</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">SNPs, haplotypes, and model selection in a candidate gene region: The SIMPle analysis for multilocus data.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DConti%26rft.auinit1%253DD.%2BV.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D429%26rft.epage%253D441%26rft.atitle%253DSNPs%252C%2Bhaplotypes%252C%2Band%2Bmodel%2Bselection%2Bin%2Ba%2Bcandidate%2Bgene%2Bregion%253A%2Bthe%2BSIMPle%2Banalysis%2Bfor%2Bmultilocus%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20039%26rft_id%253Dinfo%253Apmid%252F15543635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15543635&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.13" data-doi="10.1126/science.1141752"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbesier</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Vincent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Jang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Fornara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. Z.</span> <span class="cit-name-surname">Fan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>FT</em> protein movement contributes to long-distance signaling in floral induction of <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1030</span>–<span class="cit-lpage">1033</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1141752</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1141752%26rft_id%253Dinfo%253Apmid%252F17446353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgyNy8xMDMwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.14" data-doi="10.1093/biostatistics/kxp023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Leblanc</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Psaty</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kooperberg</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SHARE: an adaptive algorithm to select the most informative set of SNPs for candidate genetic association.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">693</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biostatistics/kxp023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxp023%26rft_id%253Dinfo%253Apmid%252F19605740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxp023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19605740&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.15" data-doi="10.1038/nmeth.1785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delaneau</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Marchini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Zagury</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A linear complexity phasing method for thousands of genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">181</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1785</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D9%26rft.spage%253D179%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1785%26rft_id%253Dinfo%253Apmid%252F22138821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22138821&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4115.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Draper</span>,  <span class="cit-name-given-names">N. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Applied regression analysis</span>, <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.17" data-doi="10.1080/01621459.1961.10482090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>,  <span class="cit-name-given-names">O. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Multiple Comparisons among Means.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1961.10482090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D56%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1961.10482090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1961.10482090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.18" data-doi="10.1086/422174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. T.</span> <span class="cit-name-surname">Zondervan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Cardon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hunt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Deloukas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage disequilibrium mapping via cladistic analysis of single-nucleotide polymorphism haplotypes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/422174</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DDurrant%26rft.auinit1%253DC.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D43%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bmapping%2Bvia%2Bcladistic%2Banalysis%2Bof%2Bsingle-nucleotide%2Bpolymorphism%2Bhaplotypes.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F422174%26rft_id%253Dinfo%253Apmid%252F15148658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/422174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15148658&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222031300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.19" data-doi="10.1126/science.1069424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>,  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Schaffner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nguyen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Moore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Roy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The structure of haplotype blocks in the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">2225</span>–<span class="cit-lpage">2229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1069424</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069424%26rft_id%253Dinfo%253Apmid%252F12029063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTYvNTU3Ni8yMjI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.20" data-doi="10.1534/genetics.106.058859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>,  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. W.</span> <span class="cit-name-surname">Omholt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Carlborg</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Statistical epistasis is a generic feature of gene regulatory networks.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.058859</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058859%26rft_id%253Dinfo%253Apmid%252F17028346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzEvNDExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.21" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>,  <span class="cit-name-given-names">M. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.22" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Robertson</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF01245622</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.23" data-doi="10.1016/0040-5809(88)90004-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Weir</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Variances and covariances of squared linkage disequilibria in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(88)90004-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D78%26rft.atitle%253DVariances%2Band%2Bcovariances%2Bof%2Bsquared%2Blinkage%2Bdisequilibria%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252888%252990004-4%26rft_id%253Dinfo%253Apmid%252F3376052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(88)90004-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3376052&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988M750300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.24" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fuchsberger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Marchini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Abecasis</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.25" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>,  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donnelly</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Marchini</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000529</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.26" data-doi="10.1002/gepi.20242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. I.</span> <span class="cit-name-surname">Amos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. Y.</span> <span class="cit-name-surname">Lin</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Detecting haplotype effects in genomewide association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">812</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D31%26rft.issue%253D8%26rft.spage%253D803%26rft.epage%253D812%26rft.atitle%253DDetecting%2Bhaplotype%2Beffects%2Bin%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20242%26rft_id%253Dinfo%253Apmid%252F17549762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17549762&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251574600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.27" data-doi="10.1073/pnas.1600558113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huo</span>,  <span class="cit-name-given-names">H. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Wei</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Bradford</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>DELAY OF GERMINATION1 (DOG1)</em> regulates both seed dormancy and flowering time through microRNA pathways.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E2199</span>–<span class="cit-lpage">E2206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1600558113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1600558113%26rft_id%253Dinfo%253Apmid%252F27035986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzE1L0UyMTk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.28" data-doi="10.1093/pcp/pcs002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imura</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kobayashi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yamamoto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Furutani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tasaka</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>CRYPTIC PRECOCIOUS/MED12</em> is a novel flowering regulator with multiple target steps in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/pcp/pcs002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcs002%26rft_id%253Dinfo%253Apmid%252F22247249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcs002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22247249&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300497500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.29" data-doi="10.1534/g3.117.300548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Schmidt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Reif</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Haplotype-based genome-wide prediction models exploit local epistatic interactions among markers.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1687</span>–<span class="cit-lpage">1699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300548%26rft_id%253Dinfo%253Apmid%252F29549092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNS8xNjg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.30" data-doi="10.1126/science.286.5446.1962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kardailsky</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. K.</span> <span class="cit-name-surname">Shukla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Ahn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Dagenais</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Christensen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Activation tagging of the floral inducer <em>FT</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">1962</span>–<span class="cit-lpage">1965</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.286.5446.1962</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKardailsky%26rft.auinit1%253DI.%26rft.volume%253D286%26rft.issue%253D5446%26rft.spage%253D1962%26rft.epage%253D1965%26rft.atitle%253DActivation%2BTagging%2Bof%2Bthe%2BFloral%2BInducer%2BFT%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.286.5446.1962%26rft_id%253Dinfo%253Apmid%252F10583961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODYvNTQ0Ni8xOTYyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.31" data-doi="10.1186/1471-2350-13-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knuppel</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Esparza-Gordillo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Marenholz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. G.</span> <span class="cit-name-surname">Holzhutter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bauerfeind</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multi-locus stepwise regression: a haplotype-based algorithm for finding genetic associations applied to atopic dermatitis.</span> <abbr class="cit-jnl-abbrev">BMC Med. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2350-13-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmedical%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BMed%2BGenet%26rft.aulast%253DKnuppel%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DMulti-locus%2Bstepwise%2Bregression%253A%2Ba%2Bhaplotype-based%2Balgorithm%2Bfor%2Bfinding%2Bgenetic%2Bassociations%2Bapplied%2Bto%2Batopic%2Bdermatitis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2350-13-8%26rft_id%253Dinfo%253Apmid%252F22284537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2350-13-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22284537&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.32" data-doi="10.1093/bioinformatics/btm316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laramie</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Wilk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Destefano</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Myers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">HaploBuild: an algorithm to construct non-contiguous associated haplotypes in family based genetic studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2190</span>–<span class="cit-lpage">2192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm316</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm316%26rft_id%253Dinfo%253Apmid%252F17586827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586827&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249818300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.33" data-doi="10.1101/gad.245993.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Filiault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Box</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Kerdaffrec</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Van Oosterhout</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiple <em>FLC</em> haplotypes defined by independent cis-regulatory variation underpin life history diversity in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1635</span>–<span class="cit-lpage">1640</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.245993.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.245993.114%26rft_id%253Dinfo%253Apmid%252F25035417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI4LzE1LzE2MzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.34" data-doi="10.1038/ng1457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chakravarti</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Cutler</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Exhaustive allelic transmission disequilibrium tests as a new approach to genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLin%26rft.auinit1%253DS.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1181%26rft.epage%253D1188%26rft.atitle%253DExhaustive%2Ballelic%2Btransmission%2Bdisequilibrium%2Btests%2Bas%2Ba%2Bnew%2Bapproach%2Bto%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1457%26rft_id%253Dinfo%253Apmid%252F15502828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15502828&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.35" data-doi="10.1038/nmeth.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Listgarten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Kadie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. I.</span> <span class="cit-name-surname">Davidson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FaST linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLippert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D835%26rft.atitle%253DFaST%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1681%26rft_id%253Dinfo%253Apmid%252F21892150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892150&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">A. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The power of association studies to detect the contribution of candidate genetic loci to variation in complex traits.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLong%26rft.auinit1%253DA.%2BD.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D720%26rft.epage%253D731%26rft.atitle%253DThe%2BPower%2Bof%2BAssociation%2BStudies%2Bto%2BDetect%2Bthe%2BContribution%2Bof%2BCandidate%2BGenetic%2BLoci%2Bto%2BVariation%2Bin%2BComplex%2BTraits%26rft_id%253Dinfo%253Apmid%252F10447507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOC83MjAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.37" data-doi="10.1371/journal.pone.0014079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Hamblin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Performance of single nucleotide polymorphisms versus haplotypes for genome-wide association analysis in barley.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e14079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.001407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLorenz%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De14079%26rft.epage%253De14079%26rft.atitle%253DPerformance%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bversus%2Bhaplotypes%2Bfor%2Bgenome-wide%2Bassociation%2Banalysis%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014079%26rft_id%253Dinfo%253Apmid%252F21124933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124933&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.38" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epistasis and quantitative traits: using model organisms to study gene-gene interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.39" data-doi="10.1038/nrg2344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>,  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Abecasis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Cardon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Goldstein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Little</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide association studies for complex traits: consensus, uncertainty and challenges.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMcCarthy%26rft.auinit1%253DM.%2BI.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D369%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bcomplex%2Btraits%253A%2Bconsensus%252C%2Buncertainty%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2344%26rft_id%253Dinfo%253Apmid%252F18398418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18398418&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255057300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Fine mapping of quantitative trait loci using linkage disequilibria with closely linked marker loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D421%26rft_id%253Dinfo%253Apmid%252F10790414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10790414&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.41" data-doi="10.1105/tpc.11.5.949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michaels</span>,  <span class="cit-name-given-names">S. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Amasino</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title"><em>FLOWERING LOCUS C</em> encodes a novel <em>MADS</em> domain protein that acts as a repressor of flowering.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">956</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.11.5.949</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DMichaels%26rft.auinit1%253DS.%2BD.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D949%26rft.epage%253D956%26rft.atitle%253DFLOWERING%2BLOCUS%2BC%2BEncodes%2Ba%2BNovel%2BMADS%2BDomain%2BProtein%2BThat%2BActs%2Bas%2Ba%2BRepressor%2Bof%2BFlowering%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.11.5.949%26rft_id%253Dinfo%253Apmid%252F10330478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjExLzUvOTQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.42" data-doi="10.1002/bies.20236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">J. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Williams</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Traversing the conceptual divide between biological and statistical epistasis: Systems biology and a more modern synthesis.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.20236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DMoore%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D27%26rft.issue%253D6%26rft.spage%253D637%26rft.epage%253D646%26rft.atitle%253DTraversing%2Bthe%2Bconceptual%2Bdivide%2Bbetween%2Bbiological%2Band%2Bstatistical%2Bepistasis%253A%2Bsystems%2Bbiology%2Band%2Ba%2Bmore%2Bmodern%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20236%26rft_id%253Dinfo%253Apmid%252F15892116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15892116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229468500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.43" data-doi="10.1002/gepi.10200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">R. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Kaplan</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">On the advantage of haplotype analysis in the presence of multiple disease susceptibility alleles.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.10200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DMorris%26rft.auinit1%253DR.%2BW.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D233%26rft.atitle%253DOn%2Bthe%2Badvantage%2Bof%2Bhaplotype%2Banalysis%2Bin%2Bthe%2Bpresence%2Bof%2Bmultiple%2Bdisease%2Bsusceptibility%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.10200%26rft_id%253Dinfo%253Apmid%252F12384975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.10200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12384975&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178517800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.44" data-doi="10.3168/jds.2009-2893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bolormaa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Chamberlain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Savin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A validated genome-wide association study in 2 dairy cattle breeds for milk production and fertility traits using variable length haplotypes.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">3331</span>–<span class="cit-lpage">3345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DPryce%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D93%26rft.issue%253D7%26rft.spage%253D3331%26rft.epage%253D3345%26rft.atitle%253DA%2Bvalidated%2Bgenome-wide%2Bassociation%2Bstudy%2Bin%2B2%2Bdairy%2Bcattle%2Bbreeds%2Bfor%2Bmilk%2Bproduction%2Band%2Bfertility%2Btraits%2Busing%2Bvariable%2Blength%2Bhaplotypes.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2893%26rft_id%253Dinfo%253Apmid%252F20630249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20630249&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4115.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.46" data-doi="10.2135/cropsci2004.0517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Frisch</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetical and mathematical properties of similarity and dissimilarity coefficients applied in plant breeding and seed bank management.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2005.0001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DReif%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D45%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DGenetical%2Band%2BMathematical%2BProperties%2Bof%2BSimilarity%2Band%2BDissimilarity%2BCoefficients%2BApplied%2Bin%2BPlant%2BBreeding%2Band%2BSeed%2BBank%2BManagement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000226435300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.47" data-doi="10.1002/gepi.20037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaid</span>,  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evaluating associations of haplotypes with traits.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">364</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DSchaid%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D364%26rft.atitle%253DEvaluating%2Bassociations%2Bof%2Bhaplotypes%2Bwith%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20037%26rft_id%253Dinfo%253Apmid%252F15543638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15543638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225524800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.48" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Estimating the dimension of a model.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.49" data-doi="10.1038/nature02195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>,  <span class="cit-name-given-names">S. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Amasino</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Vernalization in <em>Arabidopsis thaliana</em> is mediated by the <em>PHD</em> finger protein <em>VIN3</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">427</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSung%26rft.auinit1%253DS.%26rft.volume%253D427%26rft.issue%253D6970%26rft.spage%253D159%26rft.epage%253D164%26rft.atitle%253DVernalization%2Bin%2BArabidopsis%2Bthaliana%2Bis%2Bmediated%2Bby%2Bthe%2BPHD%2Bfinger%2Bprotein%2BVIN3.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02195%26rft_id%253Dinfo%253Apmid%252F14712276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14712276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187863900036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.50" data-doi="10.1016/j.cell.2016.05.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1001 Genomes Consortium</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">1,135 Genomes reveal the global pattern of polymorphism in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.05.063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.063%26rft_id%253Dinfo%253Apmid%252F27293186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27293186&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.51" data-doi="10.18637/jss.v033.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the Lasso.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D58%26rft.spage%253D267%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v033.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v033.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.52" data-doi="10.1038/ng.314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trégouët</span>,  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. R.</span> <span class="cit-name-surname">Konig</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Erdmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Munteanu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Braund</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide haplotype association study identifies the <em>SLC22A3-LPAL2-LPA</em> gene cluster as a risk locus for coronary artery disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.314</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTregouet%26rft.auinit1%253DD.%2BA.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D283%26rft.epage%253D285%26rft.atitle%253DGenome-wide%2Bhaplotype%2Bassociation%2Bstudy%2Bidentifies%2Bthe%2BSLC22A3-LPAL2-LPA%2Bgene%2Bcluster%2Bas%2Ba%2Brisk%2Blocus%2Bfor%2Bcoronary%2Bartery%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.314%26rft_id%253Dinfo%253Apmid%252F19198611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19198611&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263640200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.53" data-doi="10.1186/1471-2156-9-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Chien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Andriesen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. P.</span> <span class="cit-name-surname">Zhao</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A systematic search for SNPs/haplotypes associated with disease phenotypes using a haplotype-based stepwise procedure.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-9-90</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.spage%253D90%26rft.epage%253D90%26rft.atitle%253DA%2Bsystematic%2Bsearch%2Bfor%2BSNPs%252Fhaplotypes%2Bassociated%2Bwith%2Bdisease%2Bphenotypes%2Busing%2Ba%2Bhaplotype-based%2Bstepwise%2Bprocedure.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-9-90%26rft_id%253Dinfo%253Apmid%252F19102730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-9-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19102730&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.54" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pressoir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. H.</span> <span class="cit-name-surname">Briggs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. V.</span> <span class="cit-name-surname">Bi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yamasaki</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.55" data-doi="10.1002/gepi.20228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Z. X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Schaid</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sequential haplotype scan methods for association analysis.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">564</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20228</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DYu%26rft.auinit1%253DZ.%26rft.volume%253D31%26rft.issue%253D6%26rft.spage%253D553%26rft.epage%253D564%26rft.atitle%253DSequential%2Bhaplotype%2Bscan%2Bmethods%2Bfor%2Bassociation%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20228%26rft_id%253Dinfo%253Apmid%252F17487883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17487883&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249476200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.56" data-doi="10.1073/pnas.102186799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Deng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Waterman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. Z.</span> <span class="cit-name-surname">Sun</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A dynamic programming algorithm for haplotype block partitioning.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">7335</span>–<span class="cit-lpage">7339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.102186799</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.102186799%26rft_id%253Dinfo%253Apmid%252F12032283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTEvNzMzNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDExNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4115.57" data-doi="10.1517/phgs.4.2.171.22636"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pfeiffer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Gail</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Haplotype analysis in population genetics and association studies.</span> <abbr class="cit-jnl-abbrev">Pharmacogenomics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1517/phgs.4.2.171.22636</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacogenomics%26rft.stitle%253DPharmacogenomics%26rft.aulast%253DZhao%26rft.auinit1%253DH.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D171%26rft.epage%253D178%26rft.atitle%253DHaplotype%2Banalysis%2Bin%2Bpopulation%2Bgenetics%2Band%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1517%252Fphgs.4.2.171.22636%26rft_id%253Dinfo%253Apmid%252F12605551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1517/phgs.4.2.171.22636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12605551&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4115.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181475000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/4115.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4107" title="OsSPL10, a SBP-Box Gene, Plays a Dual Role in Salt Tolerance and Trichome Formation in Rice (Oryza sativa L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4127" title="Transcriptome Characterization of Reverse Development in Turritopsis dohrnii (Hydrozoa, Cnidaria)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4115" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MhxINjfS-dyde5LRglQBgf8jt471S3uIbPoIyJLqCSY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/4115" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/4115" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/4115" class="active" data-icon-position="" data-hide-link-title="0">Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494990" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-q9ERvY-5VeunNdVM0Xh4YHcuAJjRdYqv7A1QCmtLlSU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4115.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/4115" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-P0Zbt7P8wkmuBrN2cBwxTahZeHc68eieGlMLQnw0WrY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/4115" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4115.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494990" id="node494990--2" data-pisa="ggg;9/12/4115" data-pisa-master="ggg;g3.119.400451" data-apath="/ggg/9/12/4115.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3774-552X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fang Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renate H. Schmidt</span>, <span class="highwire-citation-author" data-delta="2">Jochen C. Reif</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4115-4126;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400451 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494990--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3774-552X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fang Liu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Renate H. Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renate%2BH.%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenate%2BH.%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renate H. Schmidt</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a>
<a href="mailto:jiang@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">jiang@ipk-gatersleben.de</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yong Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong Jiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a>
<a href="mailto:jiang@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">jiang@ipk-gatersleben.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/4115" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/4115&amp;title=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/4115&amp;title=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/4115&amp;title=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/4115&amp;text=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/4115&amp;title=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/4115&amp;t=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/4115&amp;title=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/4115&amp;title=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494990/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494990" id="node494990--3" data-pisa="ggg;9/12/4115" data-pisa-master="ggg;g3.119.400451" data-apath="/ggg/9/12/4115.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selecting Closely-Linked SNPs Based on Local Epistatic Effects for Haplotype Construction Improves Power of Association Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3774-552X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fang Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renate H. Schmidt</span>, <span class="highwire-citation-author" data-delta="2">Jochen C. Reif</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4115-4126;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400451 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494990--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3774-552X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fang Liu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Renate H. Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renate%2BH.%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenate%2BH.%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renate H. Schmidt</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a>
<a href="mailto:jiang@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">jiang@ipk-gatersleben.de</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yong Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong Jiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a>
<a href="mailto:jiang@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">jiang@ipk-gatersleben.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=4115&amp;rft.epage=4126&amp;rft.atitle=Selecting%20Closely-Linked%20SNPs%20Based%20on%20Local%20Epistatic%20Effects%20for%20Haplotype%20Construction%20Improves%20Power%20of%20Association%20Mapping&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Liu%2C+Fang&amp;rft.au=Schmidt%2C+Renate+H.&amp;rft.au=Reif%2C+Jochen+C.&amp;rft.au=Jiang%2C+Yong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494990/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494990/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494990/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494990/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494990/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494990/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494990/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494990/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494990/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494990/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494990/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494990/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/4115" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/4115.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/4115.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/4115.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
