<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/4073" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/4073.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/4073.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/4073.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/12/4073/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/4073.atom" />
<meta name="HW.pisa" content="ggg;9/12/4073" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model" />
<meta name="DC.Identifier" content="10.1534/g3.119.400778" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Paris et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Detecting genomic regions under selection is an important objective of population genetics. Typical analyses for this goal are based on exploiting genetic diversity patterns in present time data but rapid advances in DNA sequencing have increased the availability of time series genomic data. A common approach to analyze such data is to model the temporal evolution of an allele frequency as a Markov chain. Based on this principle, several methods have been proposed to infer selection intensity. One of their differences lies in how they model the transition probabilities of the Markov chain. Using the Wright-Fisher model is a natural choice but its computational cost is prohibitive for large population sizes so approximations to this model based on parametric distributions have been proposed. Here, we compared the performance of some of these approximations with respect to their power to detect selection and their estimation of the selection coefficient. We developped a new generic Hidden Markov Model likelihood calculator and applied it on genetic time series simulated under various evolutionary scenarios. The Beta with spikes approximation, which combines discrete fixation probabilities with a continuous Beta distribution, was found to perform consistently better than the others. This distribution provides an almost perfect fit to the Wright-Fisher model in terms of selection inference, for a computational cost that does not increase with population size. We further evaluated this model for population sizes not accessible to the Wright-Fisher model and illustrated its performance on a dataset of two divergently selected chicken populations." />
<meta name="DC.Contributor" content="Cyriel Paris" />
<meta name="DC.Contributor" content="Bertrand Servin" />
<meta name="DC.Contributor" content="Simon Boitard" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Detecting genomic regions under selection is an important objective of population genetics. Typical analyses for this goal are based on exploiting genetic diversity patterns in present time data but rapid advances in DNA sequencing have increased the availability of time series genomic data. A common approach to analyze such data is to model the temporal evolution of an allele frequency as a Markov chain. Based on this principle, several methods have been proposed to infer selection intensity. One of their differences lies in how they model the transition probabilities of the Markov chain. Using the Wright-Fisher model is a natural choice but its computational cost is prohibitive for large population sizes so approximations to this model based on parametric distributions have been proposed. Here, we compared the performance of some of these approximations with respect to their power to detect selection and their estimation of the selection coefficient. We developped a new generic Hidden Markov Model likelihood calculator and applied it on genetic time series simulated under various evolutionary scenarios. The Beta with spikes approximation, which combines discrete fixation probabilities with a continuous Beta distribution, was found to perform consistently better than the others. This distribution provides an almost perfect fit to the Wright-Fisher model in terms of selection inference, for a computational cost that does not increase with population size. We further evaluated this model for population sizes not accessible to the Wright-Fisher model and illustrated its performance on a dataset of two divergently selected chicken populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/4073" />
<meta name="citation_id" content="9/12/4073" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/4073" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/4073.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/4073.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/4073.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400778" />
<meta name="citation_pmid" content="31597676" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="4073" />
<meta name="citation_lastpage" content="4086" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Cyriel Paris" />
<meta name="citation_author_institution" content="Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France" />
<meta name="citation_author" content="Bertrand Servin" />
<meta name="citation_author_institution" content="Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5141-0913" />
<meta name="citation_author" content="Simon Boitard" />
<meta name="citation_author_institution" content="Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France" />
<meta name="citation_author_email" content="simon.boitard@inra.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2499-5052" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=N. Alnahhas;citation_author=C. Berri;citation_author=M. Chabault;citation_author=P. Chartrin;citation_author=M. Boulay;citation_title=Genetic parameters of white striping in relation to body weight, carcass composition, and meat quality traits in two broiler lines divergently selected for the ultimate pH of the pectoralis major muscle.;citation_pages=61;citation_volume=17;citation_year=2016;citation_pmid=27094623;citation_doi=10.1186/s12863-016-0369-2" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=D. J. Balding;citation_author=R. A. Nichols;citation_title=A method for quantifying differentiation between populations at multi-allelic loci and its implications for investigating identity and paternity.;citation_pages=3-12;citation_volume=96;citation_year=1995;citation_issue=1-2;citation_pmid=7607457;citation_doi=10.1007/BF01441146" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=E. L. Bihan-Duval;citation_author=C. Hennequet-Antier;citation_author=C. Berri;citation_author=S. A. Beauclercq;citation_author=M. C. Bourin;citation_title=Identification of genomic regions and candidate genes for chicken meat ultimate pH by combined detection of selection signatures and QTL.;citation_pages=294;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4690-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. P. Bollback;citation_author=T. L. York;citation_author=R. Nielsen;citation_title=Estimation of 2Nes From Temporal Allele Frequency Data;citation_pages=497-502;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493066;citation_doi=10.1534/genetics.107.085019" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Bonhomme;citation_author=C. Chevalet;citation_author=B. Servin;citation_author=S. Boitard;citation_author=J. Abdallah;citation_title=Detecting Selection in Population Trees: The Lewontin and Krakauer Test Extended.;citation_volume=186;citation_year=2010;citation_pmid=20855576;citation_doi=10.1534/genetics.110.117275" />
<meta name="citation_reference" content="citation_title=Inference in hidden Markov models. Springer series in statistics.;citation_author=O. Cappé;citation_author=E. Moulines;citation_author=T. Ryden;citation_year=2005;citation_doi=10.1007/0-387-28982-8" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=L. L. Cavalli-Sforza;citation_author=I. Barrai;citation_author=A. W. F. Edwards;citation_title=Analysis of Human Evolution Under Random Genetic Drift;citation_pages=9-20;citation_volume=29;citation_year=1964;citation_issue=0;citation_pmid=14278497;citation_doi=10.1101/SQB.1964.029.01.006" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=H. Doekes;citation_author=R. Veerkamp;citation_author=P. Bijma;citation_author=S. Hiemstra;citation_author=J. Windig;citation_title=Value of the Dutch Holstein Friesian germplasm collection to increase genetic variability and improve genetic merit.;citation_pages=10022-10033;citation_volume=101;citation_year=2018;citation_doi=10.3168/jds.2018-15217" />
<meta name="citation_reference" content="citation_title=Mathematical population genetics;citation_author=W. J. Ewens;citation_volume=Vol. 27;citation_year=2004;citation_doi=10.1007/978-0-387-21822-9" />
<meta name="citation_reference" content="Fages, A., K. Hanghøj, N. Khan, C. Gaunitz, A. Seguin-Orlando et al., 2019 Tracking Five Millennia of Horse Management with Extensive Ancient Genome Time Series. Cell 177:1419–1435: e31. https://doi:10.1016/j.cell.2019.03.049." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. I. Fariello;citation_author=S. Boitard;citation_author=H. Naya;citation_author=M. SanCristobal;citation_author=B. Servin;citation_title=Detecting signatures of selection through haplotype differentiation among hierarchically structured population.;citation_volume=193;citation_year=2013;citation_pmid=23307896;citation_doi=10.1534/genetics.112.147231" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Ferrer-Admetlla;citation_author=C. Leuenberger;citation_author=J. D. Jensen;citation_author=D. Wegmann;citation_title=An approximate markov model for the Wright–Fisher diffusion and its application to time series data.;citation_volume=203;citation_year=2016;citation_pmid=27038112;citation_doi=10.1534/genetics.115.184598" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. Foll;citation_author=H. Shim;citation_author=J. D. Jensen;citation_title=WFABC: a W right–F isher ABC-based approach for inferring effective population sizes and selection coefficients from time-sampled data.;citation_pages=87-98;citation_volume=15;citation_year=2015;citation_pmid=24834845;citation_doi=10.1111/1755-0998.12280" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=Z. Gompert;citation_title=Bayesian inference of selection in a heterogeneous environment from genetic time-series data.;citation_pages=121-134;citation_volume=25;citation_year=2016;citation_pmid=26184577;citation_doi=10.1111/mec.13323" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=An MCMC-based method for Bayesian inference of natural selection from time series DNA data across linked loci.;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J.-N. Hubert;citation_author=T. Zerjal;citation_author=F. Hospital;citation_title=Cancer-and behavior-related genes are targeted by selection in the Tasmanian devil (Sarcophilus harrisii).;citation_pages=e0201838;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0201838" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T.-Y. J. Hui;citation_author=A. Burt;citation_title=Estimating Effective Population Size from Temporally Spaced Samples with a Novel, Efficient Maximum-Likelihood Algorithm.;citation_volume=200;citation_year=2015;citation_pmid=25747459;citation_doi=10.1534/genetics.115.174904" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Iranmehr;citation_author=A. Akbari;citation_author=C. Schlötterer;citation_author=V. Bafna;citation_title=Clear: Composition of Likelihoods for Evolve and Resequence Experiments.;citation_volume=206;citation_year=2017;citation_pmid=28396506;citation_doi=10.1534/genetics.116.197566" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Probab.;citation_author=M. Kimura;citation_title=Diffusion Models in Population Genetics.;citation_pages=177-232;citation_volume=1;citation_year=1964;citation_doi=10.2307/3211856" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lacerda;citation_author=C. Seoighe;citation_title=Population genetics inference for longitudinally-sampled mutants under strong selection.;citation_volume=198;citation_year=2014;citation_pmid=25213172;citation_doi=10.1534/genetics.114.167957" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A.-S. Malaspinas;citation_title=Methods to characterize selective sweeps using time serial samples: an ancient DNA perspective.;citation_pages=24-41;citation_volume=25;citation_year=2015;citation_doi=10.1111/mec.13492" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A.-S. Malaspinas;citation_author=O. Malaspinas;citation_author=S. N. Evans;citation_author=M. Slatkin;citation_title=Estimating allele age and selection coefficient from time-serial data.;citation_volume=192;citation_year=2012;citation_pmid=22851647;citation_doi=10.1534/genetics.112.140939" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=I. Mathieson;citation_author=I. Lazaridis;citation_author=N. Rohland;citation_author=S. Mallick;citation_author=N. Patterson;citation_title=Genome-wide patterns of selection in 230 ancient Eurasians.;citation_pages=499-503;citation_volume=528;citation_year=2015;citation_pmid=26595274;citation_doi=10.1038/nature16152" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=G. Nicholson;citation_author=A. V. Smith;citation_author=F. Jónsson;citation_author=Ó. Gústafsson;citation_author=K. Stefánsson;citation_title=Assessing population differentiation and isolation from single-nucleotide polymorphism data.;citation_pages=695-715;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00357" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=L. R. Rabiner;citation_title=A tutorial on hidden Markov models and selected applications in speech recognition.;citation_pages=257-286;citation_volume=77;citation_year=1989;citation_doi=10.1109/5.18626" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Sackman;citation_author=R. B. Harris;citation_author=J. D. Jensen;citation_title=Inferring demography and selection in organisms characterized by skewed offspring distributions.;citation_volume=211;citation_year=2019;citation_pmid=30651284;citation_doi=10.1534/genetics.118.301684" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. G. Schraiber;citation_author=S. N. Evans;citation_author=M. Slatkin;citation_title=Bayesian inference of natural selection from allele frequency time series.;citation_volume=203;citation_year=2016;citation_pmid=27010022;citation_doi=10.1534/genetics.116.187278" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Siren;citation_author=P. Marttinen;citation_author=J. Corander;citation_title=Reconstructing Population Histories from Single Nucleotide Polymorphism Data.;citation_volume=28;citation_year=2011;citation_pmid=20819907;citation_doi=10.1093/molbev/msq236" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. S. Song;citation_author=M. Steinrücken;citation_title=A Simple Method for Finding Explicit Analytic Transition Densities of Diffusion Processes with General Diploid Selection.;citation_volume=190;citation_year=2012;citation_pmid=22209899;citation_doi=10.1534/genetics.111.136929" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Stat.;citation_author=M. Steinrücken;citation_author=A. Bhaskar;citation_author=Y. S. Song;citation_title=A novel spectral method for inferring general diploid selection from time series genetic data.;citation_pages=2203-2222;citation_volume=8;citation_year=2014;citation_pmid=25598858;citation_doi=10.1214/14-AOAS764" />
<meta name="citation_reference" content="Storey, J. D., A. J. Bass, A. Dabney, and D. Robinson, 2015. qvalue: Q-value estimation for false discovery rate control." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Tataru;citation_author=T. Bataillon;citation_author=A. Hobolth;citation_title=Inference under a wright-fisher model using an accurate beta approximation.;citation_volume=201;citation_year=2015;citation_pmid=26311474;citation_doi=10.1534/genetics.115.179606" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=P. Tataru;citation_author=M. Simonsen;citation_author=T. Bataillon;citation_author=A. Hobolth;citation_title=Statistical inference in the Wright-Fisher model using allele frequency data.;citation_volume=66;citation_year=2017;citation_pmid=27486182;citation_doi=10.1093/sysbio/syw056" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Terhorst;citation_author=C. Schlötterer;citation_author=Y. S. Song;citation_title=Multi-locus Analysis of Genomic Time Series Data from Experimental Evolution.;citation_pages=e1005069;citation_volume=11;citation_year=2015;citation_doi=10.1371/journal.pgen.1005069" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Tobler;citation_author=J. Hermisson;citation_author=C. Schlötterer;citation_title=Parallel trait adaptation across opposing thermal environments in experimental Drosophila melanogaster populations.;citation_pages=1745-1759;citation_volume=69;citation_year=2015;citation_doi=10.1111/evo.12705" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=J. J. Vitti;citation_author=S. R. Grossman;citation_author=P. C. Sabeti;citation_title=Detecting natural selection in genomic data.;citation_pages=97-120;citation_volume=47;citation_year=2013;citation_pmid=24274750;citation_doi=10.1146/annurev-genet-111212-133526" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Zhao;citation_author=X. Yue;citation_author=D. Waxman;citation_title=Complete Numerical Solution of the Diffusion Equation of Random Genetic Drift.;citation_volume=194;citation_year=2013;citation_pmid=23749318;citation_doi=10.1534/genetics.113.152017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="Detecting genomic regions under selection is an important objective of population genetics. Typical analyses for this goal are based on exploiting genetic diversity patterns in present time data but rapid advances in DNA sequencing have increased the availability of time series genomic data. A common approach to analyze such data is to model the temporal evolution of an allele frequency as a Markov chain. Based on this principle, several methods have been proposed to infer selection intensity. One of their differences lies in how they model the transition probabilities of the Markov chain. Using the Wright-Fisher model is a natural choice but its computational cost is prohibitive for large population sizes so approximations to this model based on parametric distributions have been proposed. Here, we compared the performance of some of these approximations with respect to their power to detect selection and their estimation of the selection coefficient. We developped a new generic Hidden Markov Model likelihood calculator and applied it on genetic time series simulated under various evolutionary scenarios. The Beta with spikes approximation, which combines discrete fixation probabilities with a continuous Beta distribution, was found to perform consistently better than the others. This distribution provides an almost perfect fit to the Wright-Fisher model in terms of selection inference, for a computational cost that does not increase with population size. We further evaluated this model for population sizes not accessible to the Wright-Fisher model and illustrated its performance on a dataset of two divergently selected chicken populations." />
<meta name="og-title" property="og:title" content="Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/4073" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Detecting genomic regions under selection is an important objective of population genetics. Typical analyses for this goal are based on exploiting genetic diversity patterns in present time data but rapid advances in DNA sequencing have increased the availability of time series genomic data. A common approach to analyze such data is to model the temporal evolution of an allele frequency as a Markov chain. Based on this principle, several methods have been proposed to infer selection intensity. One of their differences lies in how they model the transition probabilities of the Markov chain. Using the Wright-Fisher model is a natural choice but its computational cost is prohibitive for large population sizes so approximations to this model based on parametric distributions have been proposed. Here, we compared the performance of some of these approximations with respect to their power to detect selection and their estimation of the selection coefficient. We developped a new generic Hidden Markov Model likelihood calculator and applied it on genetic time series simulated under various evolutionary scenarios. The Beta with spikes approximation, which combines discrete fixation probabilities with a continuous Beta distribution, was found to perform consistently better than the others. This distribution provides an almost perfect fit to the Wright-Fisher model in terms of selection inference, for a computational cost that does not increase with population size. We further evaluated this model for population sizes not accessible to the Wright-Fisher model and illustrated its performance on a dataset of two divergently selected chicken populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"HMnXtsp4aE95ki79bYLwHWsqNdp41Bea6ARCQ2mDOTU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494998","apath":"\/ggg\/9\/12\/4073.atom","pisa":"ggg;9\/12\/4073","ac":{"\/ggg\/9\/12\/4073.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/4073.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4073"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4073"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/4073":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TTp6565EB0zuwVzB8AbFlDUhyOznWUV\\\\\\\/46kCp7vqk88pMjIK6Anz1zBmn\\\\\\\/lJfYqCzeOytdqN7zC1O6iYRJNaU0tzx7\\\\\\\/EoG0t6zZsPDO5wmtXmdyxFWcQ1CJec9MB8YY9bUVlM3E9Id9cH4Pf08yuGkkDK8gTlwznDp7s1z9TTzWiYUWExADqEkfmULVS3P0QEiVqf2QrTdyNCQzmnUDLu95WcLE4bwfbG9uKjBMGfgOdOQqb0jQ\\\\\\\/vJCBHqvllu8iIAqzrp8n+NFJvIZ8xssapqF5xJG+sMB04zN3pbHU2VnWp+LGcMxhYBlrGmnHsbOr2f77O2m24tFYY6Jl1Xr7PZ+fpXS+ACXQ91\\\\\\\/hA2+2Sol4deHoounGyCmUOorz7q31ZCaymeDjEuh6VXohqHFhe0HdmUW4W04hP8CXfVxW+3bLGy5bdY6\\\\\\\/z3YwY6KIFneQfe2UKs9uScVRmNPilmQJou\\\\\\\/ZqiTtgRkRFRJ7WFohKq5ReY\\\\\\\/5oNDT+pj7pdOHtP6kIQP313\\\\\\\/sFXW+s8WhO\\\\\\\/T8827JEOB\\\\\\\/ZSlq0phGjueX7jHx9Dg9ZJh4nBcybGz9npPIiepCngY5eIwx+\\\\\\\/\\\\\\\/+gUpJdMA57Zq1o78K47Mzjh2hMjnIMpfP9LwDn0aVNlY4E8E+LsKNiewDCvVGyQayjPoKYDCGaCEDNxnXd1M01FNYIcyT02uNuRgtgPk194C+MLoCq0lUbE+rJ4U\\\\\\\/Me74pwWTB16vZX365Dny7p2jNF1FLs4oly+CI0eAKMlunby1bWIdNsKYaNYddZThu88wwEP3ABbHP2zV371LGVh9JTh\\\\\\\/o3miITFsroCGF6GcUAvr1nwYZ9F4uRH\\\\\\\/nEV\\\\\\\/atJ4moUul0O1wou4U2Z7+S4UXDCsKmlc9m0w64H9mD8\\\\\\\/5s6AlNoXQaNwHtv8Glc8AD8XvcWYobpHKvZvkiA9ztZ6rjcHakRgj\\\\\\\/nZsSxpaUPbh6UaaEnt6a7h4EMCQkBJKA0GF2Zw7EOv3ZR6F\\\\\\\/zweKPISOX+o1wCSq2M7gDYDg0q8nUevHfSkxxB20ops1++bggb\\\\\\\/qnUzFHvNitv2KlPyWUjpDTmoHheZlhTWDC2rKoihPQs5IwPnPSLz9pNiD0KH9dnm70KBIcTDncwY2ily6oermtdtf+ww6Q4BNZvBiTcxjLos+VBK+CwHdG45sY9LwC6psaAD\\\\\\\/oUWAmEhmyHyXRQ5zcWi1OHPg83ssWVNkoFPO5e3vu8nG3V4UYQieuXyL3u9oFKoLr+jF9xctPjOGj+GCt4A4hCWgxwkzNbsHVfkGN8Nl9q\\\u0022,\\\u0022iv\\\u0022:\\\u00226eWZwZM2twr6PXq45SHFlg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224ad9cc6cb2d9b3cd8d4c9ad1b7c8a0e378f8b8ce7421b94c0fb3d91d5e5b6870\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n091NpMAwk6kpgnacBBeY8JjnyffcInMs0wAo0FOlz4OXgS8zZbirCo2EKyOO5T431vTzRfHdZC7SuvdMUgubehJG0ws\\\\\\\/cqcExRlETca1on76\\\\\\\/iY7fzESZ4fFd7Q063DEdJCPiO\\\\\\\/YY+\\\\\\\/9sze1IXhVdz\\\\\\\/Zajz6dUWELfpp+UJhL7tQGgDmsXe0N62VttfRKOq9\\\\\\\/HiIOxcdkrOP0e8ZtstjJMTdjjV3xvlqQKOPmqgfnjWBkuIqCTB19V47Y5qaHkuVFZkbA3NzrXRMZ34+UGMFpsM8Ww4N54VLElrYV7Y\\\\\\\/+deekPwIW\\\\\\\/JNBu0t+sWusZe1UtkcyegCOFf1AUUSGYys9t\\\\\\\/8nEaOXjCLt\\\\\\\/KrJxtUTuzWp5QUZXlOaW\\\\\\\/E\\\\\\\/tVOdYCkW0k\\\\\\\/MxT8fePm1QGuyMaLUSPU1LWHAqoXU\\\\\\\/8p3duofhlW6SFNUDN7SFfaX8Rbs4bo35EXg\\\\\\\/eD\\\\\\\/ljpM8PCF60Zl9Fkt5Fr2M6+cqV+IbqCa+cKQFgrGrx+Uoj+joRvCgrheb9eb3OQZVX\\\\\\\/NusfN\\\\\\\/Ut0EgSiUndAXSjw7W2g368OzzMnSSD+Pm4z8mTDNlCA\\\\\\\/tKBnVKUwIZ2LZu+LGk42iE5HxapkOhPqyhdyke3GLcx5ggKvL80yi1WfJGng7eFXHcpEQCVVSJsc4iNf3D68PJv+x7AR4kVsgbx+kSBuVpEFpm5P2zwNt300z8T19kGfN6qxS8TW+hxNb6QK85hA1kz6WSJGKDnzHCrVPgEz3ajhWpWeUHw\\\\\\\/b\\\\\\\/KGAnERFAhTitGSEpwNZEWUEtjKcnURHjM+aJT9O8DHumDdvBDQtEr6RkBpDMSUwUY8eQenf4J4bQs6ZxfB\\\\\\\/njD95oyXE9\\\\\\\/loIazZb0vTiHc0sKlwbr4LXmMdx\\\\\\\/iGq9najVoEOoxSw7UZvFqkaPaX+VN0XUZLZZ38qPOPPbs8nyZ\\\\\\\/DX38bF8sSbLFFXIlFjOlqovlj5GwzexVGNJZUTBMioiI\\\\\\\/Y\\\\\\\/I62iDruCHT+ENzKdvy41oN1XjCE\\\\\\\/tz4vwVg0F0Xz+1t9kqvYpV8zdX6evRHUJrsBWgEWmV++230rIKVpgtONFQCxjtusXD2KQD5AKyGJdJmGRbhRtXTJTiJ9H6H\\\\\\\/HliQ9InVoNzyGTJlI6efpB09\\\\\\\/S59Ra\\\\\\\/Al8Lf5EA4GYQFsct1Gh52JNi7Z2Rnkdu7UfSvo+fOPh80Cu5HPTV+hg2Zzw\\\\\\\/x\\\\\\\/w7KwWjyRUN0lttP+XiAcI1nTYFWKuTP\\\\\\\/s5I5NTj31yadmtculrUGYwxdcGWKJPLcBaIPYSiCwGIWKTWMCXrVD7OANan5GqsR39XFgI1vSzzhteFB5rgDAKWa1uADFPALRC7S4IO0uX\\\\\\\/4dx6alylNEAd3rZ588HM36yci\\\\\\\/syKoxMPIW\\\\\\\/J4nLHrEWdvdVERibbIhi2Q8ARR7RsIcYu8cW5M2xe83UVx4osTbNZO1CSIWaWM6CFpl3IRKV54uDKTPRQQso\\\\\\\/Z9vbzDef+sr8NnsGMxL9lmY3OGKc0DtuqWgGW2IjpCefsPM3uChPxWlaRcG6o1WYEPa89Z0umOIdf60RUWtMwK0SNp\\\\\\\/orFvXWZu23+qM8NNCCandJkonTGVhCkmV8g5zsky7\\\\\\\/W0jRp7LZ5yq2JXFtu2M\\\\\\\/pOHNkH1L\\\\\\\/ghbtmVb64Q3LFbAolgxfha0EjVm\\\\\\\/+d4upFGmM2hhUgu47Iy+xu0OPKOcPO2G8MRvCtcWoyUZDT6s3f3LLH7StBzVfjkNCYwc3LoKqiNBwM1RDMO8g3sZP5d2GryHVwWCQ6mZkG3fWDS0aB04UeGfXTRuoNlNA3ROk+Rxj8Ivo90fdwUD6ZXHWelyUuiCGxW+xPQrcbWfFbElfmnfvfVyWVP4fchvMqGvcXCoujs2hQI7otRVERojV6gFpTnv239GXCaHwfdENIU2HE75J9Bh\\\\\\\/XTNfLzh8yOhUDSeK746bBtN3zijo3yS\\\\\\\/nyqEg7WA485I7qTOZ7NnI6ULrIMUQ2oeA3h8WkGOGzyzjq1sK\\\\\\\/MJz4hCdSmSa6lABwmG30i0XBZ0tzobx2uU3jDg51dT40Excn5qF8hYYPlqdm40WIbTPF\\\\\\\/wBLldYRThPFrQ3V\\\\\\\/G0hlPOEBCFiG9QnLY6cnWNUVXvlm1lncg3t2CaYYRqjPRe0xvKihVX2lvEFGwkS3d3Y7oNmTIk6uoK13OcBGH7BtQsROwNgmrd1dfnQ95BQuc8HK6lfjRt2md4IzJJcZEx0JW6J462J6DiJL7VA+x5w9qsI9zEfsFhGPTgpNjCs7omTtlEqADzRzq615PTFkWs\\\\\\\/4LiiMKHoXmcGwscMVurZCHPkJt6OOsksBr3WnWfUE8tGALYUPSEwAd1\\\\\\\/8ez5VdIRhMyR04v1ZJKTtQ9kL6sLhSafDN1d4xitNqGhB1P0okAwktAf4QH\\\\\\\/21a6WOwtl2Usi5Iiy\\\\\\\/wyo4R9EE2wOkzIwfldn4OYod7euMcifTpSMItGEEtnr6n4UqDcDxnhW7sU0DrgVb1NV0AQgRMX3hNOQVuECc2LqiihIPIXsuXNsCLhObp0mLHHmdXrSDkkNlaiEQNfREIwUb3oSWCjkPpgLcKRE+iqxFCnm9ZkaFrG0HlDQ2wEC00fflMvq2sKBtypdpY1yd1w5YQy0GXkkVf7BG2p9HzMBnjYA20gIV+uFlyZLITTs1fjfL\\\\\\\/ngAN61lcS8F4rGsNFZr4phH6GSfzDQhi8re8YN7eP2WhoO1u06gVzTA7TdbXKjn9mpBVAQAOzisGgYtd7voKnyI\\\\\\\/Z2yRQdOq9cbDl1iveD45Y25nIxVXwxWg4o6ndoNw1E5RV8z8WQG9R4pykAV8cYndxiW8+2AWGqQ7FntvfGu6BandNOk8ctghZClPAvCtIHDj4z\\\\\\\/rZ9B4AjUnFklOY0DntJiaDPeY1Q5nuyDuy4FM5Wqmb+sHd6CliL62Pyjgqvvk6l8BpZUa4nkQV9ipExpv4OPcgS1hINfylJbACmJbO9vfkSBwWGsJXOyZwpnbL6xBqQ\\\\\\\/S8Pn+SZq\\\\\\\/aJCS10\\\\\\\/w\\\\\\\/cNBbinLNL5hXq0u9dYbNjFfcPIdma+TpswPJB9N28cBcOsftv1IvpfXbjHa6RKaeeDxWjj\\\\\\\/eVWMnaqiVt0DnKbzv0T4key96vKYf1HLpU1SVBi0L6G2DUhjPiN607MDisek107TAwno6xaXsvARfrgR5h\\\\\\\/U0CmpylORbXqZeJLO5rkzcljzy7AIU1YNBu9+v+6cB\\\\\\\/Z\\\\\\\/9eoOqqppQhCwWY+8kDsQ1vQCGii6W2WuNsBkax8uT76flw2bmZoSfWAU7stMQUqJteF0+IMe8LWUSapn\\\\\\\/ojYv5t9vmERMhJKfSmjBIwEZP5trDifyq4M\\\\\\\/GN0Mqf2lie12GMtbfK8W29qjHcKxEo+eKcqC4BTLRXDGsoYm+Cy375Hdm1YpB2Q92619ZBjWWr\\\\\\\/\\\\\\\/1rfzRW96h1rSXvih5cJ61rRPI1x6lSiQ98MlCZcIFZOLsZfVT4B+Z\\\\\\\/xrJD2VSIfJfnVkfAo7D486BlT30yBf7utHr8Mm0U\\\\\\\/ZIxae286jIEqORQWv8wtoZCbjs8lukP6v5SsZVJ+JKTmpWvXcDehHBorsGr0erNIo6NhLsJHPQYeC11si0X2qsIC8mEQWtmskcPvTE4Xpb4yZqHGILZK7ZLFfZVvlToFXTU4RUMBuIBtEQyXXDO4IgCqv8oqONLN4xAj6g8OjqXtFFLIlZfsRfpcO062Wc3q6Dz164tmi+3a+tlHvDAW58ANqbN5Z1cseTswuUdnBP+lLKx4oIDrBM6MPUGsBhYtkBJ3\\\\\\\/27la+uHEFFwAod2mRCxTjUwbA91b3q9AOuzvT8Of0WRfObM\\\\\\\/eOfGwP8bTkYQ05dcxB6g399xgHe3OrfpjgdSQebMKxpB8Pn4KzP7iLB6fRpekiTRbjk791UnLJ5+F09+wYsm1pOFva7gxtMjbuoGxBMMTnpKlYkhmfwXBC9WwGrIs0C+DT2uHVFGVVltfLJ3Q1bPoX+xG5WdCaQpr42ptOMZyULQBiMngwdvxf3A5XUk8LfvMmVaUz3AJejxVv7QYNeNa1EjnoqjiVorPBQFsxsA2gpWi0Yu4jflA3\\\\\\\/d79NdJzSKrsCdG2YVMz2kBx0PZzDzfnxC5v7KdnbiFsPf2SrZyWzg75i8dn8QmYQvVzTQn\\\\\\\/xjIhoLWElnYXg6lh+vQoGQovqFLE0458GOjLfT9mqyJiluppcrofxwGZjuO8xvZcD5sQPGkphBwTRI51meoEPZ9M6866\\\\\\\/prEkTavl1lnUYbUdj9IJXiSPax846iG76fMJM45w4hg4baAZAoH3oO3jCrH64V3IcjSph25TeYk3qMsoz3chmDG1fgSuXDgTH47YDt4fMN10LNZcjhT\\\\\\\/Ehl+L5V8FFqIr0bq0rNgZhYX4bshBzR+GzuNRxyFzaZ4Koeb4jlf0Y5Vb6cxjxF+K6LV8DEqr+BQsVnSzskTkMcNpdblLkys0+0OLGyKmi8BK8AKWcWGGrMxMfOLFi0i34Rz4ziZY5wuoswXw3yStQm3XgNymgwBB+VJYOxJag7vzQTYai5yPS0Rk6Xwvmgc2n7GT4MNQ+Rtk1DvPbgaS5fFX\\\\\\\/P2gVD7XQ5jc4c3X8Sm9SRDKXH\\\\\\\/\\\\\\\/Ln7LWzEWBx+B9ob3bkV3aGmYZMiIWC2Lv1qJqpPdZHtxBOe7sXglysZRyHxy2U1Ba6K5Nwj3HbzzurqVlJ4IEMJd3CvuUdokluxA6zTAox4T+N6QByXRJbkZvpB1xcT9GwYoGasz0ElOmBGsCJCfscnz4g0R7qCXaHHAKRzishOf0cd\\\\\\\/051EOqH\\\\\\\/+aIS00IZ0VIEUg78G15qwUSj3sARdK+tCAyO74fNrkOFhCSGgCziiKptC3Z8eUHpcE6HpKoYZTeS9jFfzMFkVKUsDkEoypR+16XCP102z1lXSZcwxsP6PIH+Gu0LcpyFwpqczJvYkMeGdf4Z6shVqDSvPrnv+KBFxjilCBhBBNVhTPz\\\\\\\/V134RFKyaMjfLOP\\\\\\\/cXtmEjS08HOeKzCNc4Z7+Ptj5eAKzgpwc7sjADCGXQ\\\\\\\/OQ2kNbmQeE9fmodtTQK5ZM6jxzDL3oIeDwR99m9JFofKDaKg7r8Ln7q8oD+B20iembG8t4NHN8qOxJGQielbjuK6BhxUnVUuQop9UtDBToYDsYJzaaHRo+JvcxtYw9p3RZUIUXIvK58XrxCiCPnSIHJCPxkXd9fm4BySAifXYvoGLB\\\\\\\/g9qJbSb7OkcJloPAsS9yUMuN\\\\\\\/zm661ugBkSoltE\\\\\\\/bSmYucDqOiN9Z+GE3zM3yTpKo6ncrrb9o47tOvavJZ2Lv7D7T7PVe1nN6kaUkOkZ8HxnVeQ6SfGTDFohd0qIkDGmN2fLoRhkKhDH4DBe+xWmuahz5fKuNb7MKa0GWPwY4X\\\\\\\/1b43CTo4pVM\\\\\\\/IPDFUaiwGnLaAyQbsTm6xANSL5PkWJK9laPradlmKEm6Ssi23K1TEAOhOD8M5VVn2Ujv0CJHb3zNyUZPgfArd+mDcST315BJj9k1W5kAvx7u9uup2Azc+Rnfx0BRFVnjKQWq0gjSMlBEOyD4\\\\\\\/ilsF5rRWYuSWcRkMG3N6pmD1YlqRf+Vyrb7lWhXjsvRGqgyY+uAq7W3zIbzh8mwX9D537dKjnaDhAmifmwqLeLg0bvtz6l3\\\\\\\/4dwM2\\\\\\\/EJNFsqbmn\\\\\\\/U5SYXIuiM2Bs0BzBWfcSVQN4ETr\\\\\\\/aNO46EOekwUNXnH6PzPJjvE+oAv9Dragl7Y3G1b5pBwCM8Sf3FP4HR3eWGjn58gMrnvHiYEmzveb4ZXbJZt7vjKL4zCGjq0JL6vRszGNmSa7u6o34UZI6CPGW1qsPiew+RmjXEZ4PLPT\\\\\\\/hdOa9tl+jthz3iuEHdedWDkwCFN\\\\\\\/zrS4uRCK6dw0z1VfNEUrCtdc8D\\\\\\\/RQim3YFzFNae19r1AwS4NHc60yaH+ZojLh+AtGZQB5d36VAvwd5\\\\\\\/HG1vhBQuJcYWkddDWczZYtBoliFrDvR5qtcuyv2O+3BBMvTa7Z2BFM0xb9yYrzi7HUdqGg5QlW5Hj5s682Pcr2QWRNfkTyTCl9wEP6SAuolaPtoGN2TYZ+Ox\\\\\\\/VB1me5KrMX8OS7bYswqczE5Lq1Rs+IRXbDtd7OCFFZDKRm\\\\\\\/o5z2LYkwmyzDuHUro0JrYxgKYMknoPXOmbifivjOsSZpxnY0FUBPawTa7obAd9ZRUrTHZ6PvbgiUXF+kKg8jBc2JSbkEsQagql6jhurEJ4c+LpCfVFgjSpOq8iN0WNIzc7ou28eEsPrkpI5gFQ\\\\\\\/naWUQNtutE0ZAh2Hu\\\\\\\/lkYMmaUQtECiOYiRAbwFZtFZUOUhjaEy+hjTXFBzzuQu91o4TLtSoUdViGNV5gre7HS12t5jUOwBs29SNfVZ9bPyAYKMM\\\\\\\/BALsWt4UDcvhxYFPdFmALAYbeEAf9AIdL1JfXLBxA6Fq5XlRIGW63ntwi859Zg15hX2UKZJGNwIg1LH96BucfKbJdpDtJ1HRU4HXCxmGZOac\\\\\\\/arsNhSOWWn\\\\\\\/kr4zEV8+wSUnERlPOxhfQguwb8Xl2sOZpeOaxUcF8xoTSRsu8h2AJW2HqTUiKOpBFrFOmS92Bevs23z7dNHw6jOOiZXUvzkL5jcSvWE3fveUwz0iwMud0mNM2vwwGmW91wswPe7t7vXVXfPkOtaDovGxXf3omgKOvbUP8IMOgnkows01sia6ZExg9ePSFVerekhCMU51aGcjVzW6TrGsRYhwzkIBMXdAuAzDgRCiz8dPvFcplEhuVONnRmM1yWbGrIroEaxJP0lPiyN2GxYzrbKGYjZr0Z0rKr8GKZ4nwQ2SH0XUekLfouqXMd+iSPdbzhSt\\\\\\\/LhteJtTIso23EFUHxRO3Fy8vErCROYSLGfN1nfKWyEAHKjx+Yx8r4OX8UsWtgrG0lyubn5WnEtyqstg3rqiN5wiy9b00seVXJDxsWDJvSxvD4kV3ayk9AC167slUJ1GByP5nPLd3lZStu0K3CmRegiNofbqQYxaV3Qnjj8t3A6z0\\\\\\\/f2GkZWvGi4xFhMEbf6GQSaGlpRUoavclGRwVHZpno4yFIj1nV7+U+vu3Dilh5UsSlof2Aa+CbgfJF3G1X5jb8lgQDSxdkIJiAhGe6QSCaAMmGgkGA+\\\\\\\/Wn+3yAwNPbm+yNvTg9fJVqkZcmnh+8jDVSxDYk26PCpiNcCLKi70Cc3pkE9JSHTn6EtgnUpFWCKW\\\\\\\/vwdbeVxmh7KOkX90u9viKiw11NDSvv5deSooiQJF9v+ESrTaZTf6ed72HCqZR2DHfzqKrgEFgC+57Rlp1ujBj71auacMkOsU+TeJ3SzI7QZNvNn3YGAtXKcZIOGYXIxD6dw+IXJ0UEOyc+byMZ7UaXVELhR+VBzGH5y+gQPEACVg1\\\\\\\/StbAotPXqcE\\\\\\\/du9xcOIps5PnSIHFi0E4dShT\\\\\\\/PJQwYEMp5CDQS33KIxrC2OxKA0nQ5SINHv0psyZ+UPRLJ0UbPGBCXI0Tm5iGLRqRXjnaBVWl1RMqWXoYQiyGlct3RY+zFe+xijzpMVFhYjLIRl4f7zWw5\\\\\\\/KEQhptd7v1UbdiVtyWbUMlUhfrl9mGiyqnOKB92cs2UwI8Yu93PjLBZB\\\\\\\/zll70s29kNWZUndiGv5IzX2P5fGM0ypn8STcnWn6+9VgMoNk5Q7iRyHqM28wDjuUdWNoZKODhVZewIk1CJyRUDLVrOKB\\\\\\\/tWUTc5oKUaci1+cVzj1Wl7O9Z\\\\\\\/EY\\\\\\\/nvel\\\\\\\/shZfmxFs53sZE0FqxiMugJOFSGiX+ZnnguwFDZzABOcLolyEKUzK5++XUhgNtrLY9hXkki9UMP8z+lwcC+nNo5Ra8+I\\\\\\\/BNVgBDvprIQcTfQOs3TWxb4u4cDncPmylU0XuGZzVYBSsNvw07aWiBulwrotnagLmIXpAI2RbIXtll7sJvvLYz1x2wf45O+Cg679Vg8TdUhbYYar1G9WpMjLq\\\\\\\/GwavOua+XASZcneL7SQysTwUGZEt5Mcmvk7xw6zkrNFpIHguNdOUlriXJSK2l6Oz26jIkhb2S4ptKDotwXHFYY1eupYBBndrS8LTQLElP4jTotv5y1pfKLOOfQLEhZLlY9gkGb085Nhzc62BWfYpfIUoPl5pEwW2whjbVSCuTGNCrMMubB4bGeY5\\\\\\\/\\\\\\\/xJ8zsjT7rLN8BgM7o5UqvFRH+CUhJFCPtLxgO1RhTK9jqdqVbsJsPGj4SgIZ1b3rgwGY\\\\\\\/Rve9V6MBdhWEGPHyV8xSCmVljx4wMLIYPMKy1qIe5DncmnBI2aUpLlMBpd4MbawI2d3bW6tXfaTd1dR5ZvkUXdsQbtrfSYUDe4HBDszoz1FTrEAG9z+PFW7oYWMxiFYMciAzrIccS1xYtUMgCp9drO07bfUsOHbleTT0DoS3FCD77m4U+uIII90ZVXb05d3fHBKcHn24gHHMhl48tsDU7gGcS6wffmfCGDlm\\\\\\\/G4ddPyPcpN4uW\\\\\\\/Seo5zV3Fe54hbyP0N6shobmLeZKFL7mPRiiXy6vYC5fM8bNHeoaVYvFQ+yxH\\\\\\\/dSHNVo\\\\\\\/qPmeQLcISTabX33McfvTcvBEAfag66jY9C7Uvl4LtVRMp48chsmG2UaeoTqTrxwryhHpmGrb+2QwF4EzEKPepuQSmrKFDh2TB57k55VkczWqYbOdDrINYUnh6jF0e4\\\\\\\/gWW1\\\\\\\/X+zepKIGdODNwBi1ZyAU8\\\\\\\/czmFNL5TflFinO1I+N\\\\\\\/wxNcjaHKgbbeWIGE5jS022escntpsyiDIKb1eX3VOS45B9FG1g5ifUnVgUzG9Ko6httVSZuRDNiUnUTdERXWZ4j+HKdY7\\\\\\\/IgmRuGQL6I0iVzyso2MKhuydMWEx2HNYdaCrUeVlS1f8B\\\\\\\/E\\\\\\\/u2dj3hSIuEzsHdhN\\\\\\\/i6YMzHtWaN8jVDyRgCxp7JsfBRAepHR0k2KcmwxAcPSn0TyIeA8anUu\\\\\\\/hDCDG0e73V4VzSaNSY\\\\\\\/NPqYceGTJFyehIx68rAwptVERNVt3Q+6\\\\\\\/gtYU3uMfQ8JcR\\\\\\\/wgPE26SBJUiqSm58o5OnFxPiEJFcAftibSL29E8qx+\\\\\\\/W2D\\\\\\\/HnYf7uiFwLg7bv8OPVTvFZvtS5fZBmHwGN08VCUOFiFxWxw16BX+YDnSD8WKQhRKkrKKEqjyQdNhzhJFeKjdAjiOZuVE2fPQLNtq\\\\\\\/az0+9B2blozeufpEvG3tUV6\\\\\\\/lhw6iD6V5Xoj3K6KPPvDiFrba6o0NmU977LvK4c4u8Hg8imlIgCfQqI5OfS9cC0MJAr+MvS3bC0U77hl\\\\\\\/QOpsuo87y3k5WXJpb6wd8dzOMNDW6zC2X55cP7aLfwjl6UoyX\\\\\\\/cDhkhHVF+IMS0p8gmaQ1U0QX5Q7BYCJ52zh8ha4L15Pw7pCz+zRpezfAz+nRZ9O2KPEoDkSoXKL+jJa4NnL3xm+MbO\\\\\\\/2XQ1oQjpjgNYe+VHIqskpcAVZFGbNk9iUZfI7gXezXSZaxD\\\\\\\/kBNbZt6KrNhRotAXzVL3Hr8o\\\\\\\/5NXwnpFKw\\\\\\\/Lf1\\\\\\\/x3DYx2yR8TLIxTNTr\\\\\\\/\\\\\\\/5QUNPM22DT7Md59mf+A70R+YFGR1uR0+8E5KJ8lNkhOZpYMp2cSIyWI2XbMfEeAVW+ctSuTeGPEgXN+u5UoJt3GTbTRXMmrdvpXS2foEKZDzc7yMJPiIe8rq11r20Ti2ksVSByLXnD0dKeqPXMCdtoAZAtnDbTed304fEmLYbLl3Sm2b2YodMqBJ4xCZFmOTfITC9sqZ0mqJ4j6K2TRCvBJwh80q8Fijx3SjmUWiymk4mfgUSX8nEMAySRsbBuWEV06vY8xUMUozEjAtAVwRxXjFIhL2d42VjlfhlclvIHCjyw35pvq752V8SbQRTweLiR4UvnmHeviPB66MhXgw6mLJpiVsbTuSEOvN9LApJnvm7+0S+6mUzrmlBtjOGQoO1RRWnasjh3to3kaDcfTmzwwiFpG5m6FQenc6E+D2CO9CCYjukyFGYc0lzKDztle1SlWayS6Ay2F3yoQmGjhBhvp\\\\\\\/D8mDiLdc9LmZ+r8aGc16xuNanuORVmS40V34JVRvlgRZciG1m\\\\\\\/NTr0pdogWX9VRc8CRJpWrwc\\\\\\\/M6k5XXcyWTmnOSsI3uyizSLOcRDUBCn6Joo+66KU7IjCKgV33Ts+F93FE+dc+uUwxfkTCGC0B+L2CwIau9vUlOIwG\\\\\\\/vm0W\\\\\\\/GqZoO1qDb0lNBZFJLhwGsfKdvZ8RwgBNUx26loxaO3iost\\\\\\\/QyoWV82NOC90c8XT2Cll0XYKbBz9XRNbnRgqVQisa2TAOU0KHSAbsD03iMUKswXQLglCeSiWzDCIBvzChygx6OsEMuJDikSzXSMJLzV+opOwWGHktoKmXCIAXPmlC0g10I7i43lr2zrzulksFYSwzCbWwn7IDbhiGc5Cnthu+kTE5nQX7eZMUZ0LJJG3p+5D1++opR3atosUgj17kUOTWdc2XzwlPjJ6fajojaYLl5V0I7d2R\\\\\\\/ylmzuwAxsc\\\\\\\/H2\\\\\\\/GUq0GrlBPcQLJFDuFJNOgUdP\\\\\\\/6uohPGWgCsahaYR0kYuzksxaBXMUSXpywnWchz3+GriilZXcRiP8UTKnY3D5KTsSJVvLvjezI9VHmRdH0ynd5txoamkFV932RRKov4+yLDbAlFG83HIfkdCZ5Y5Fob5V1xH777e+lxyxkjLf5a93tl57uzaNqHYtG0joLgORDl9s07bN9ipwzdubRNwl\\\\\\\/7bgjuyaL8KY\\\\\\\/9PqYZhyF85oylGBSoVO7ZVuJQFSd1PhILp27LevB+5AWJ3gdzOjDLM7ZJPJlDVpZJO4UeI6wgwz2SDSncZP5ZLZkfo2amjBxUIoGOW9QEkOWxvoUOjwhKGxlGtwWjqjPy+UVn2WBNNtpRQTzEcB\\\\\\\/DzQGn2\\\\\\\/ELEpRBlAl\\\\\\\/kMx6+nLqQy7bLwJicjdUJhf+kk6xTaueQiWx5z4O9YmeYwB2zPxD+Z59FLXKrGvuKlGAbxluQFVo7FhpANV0PCg43ceGYqldRwZu4wfMmxk3vLVl3Nfybkx2Srv2LhBtfdVDW8y+lcFRIZ4VmbWYS5VmH3h25J0yHUsfM4A4KdtjdfvSBfXdZ+K5nAX\\\\\\\/Wd+mWBhljsxFO29xzVJa9v45DJpzvdGMrfibAH+mhuIwdhvsfazo6qSqryt0IxoQkWLCUQpeIR5yXsur99R9Xdoye9NrNCJhCe7RESLmxYE6xWrblwAsjofaGiyXCap6QR5JZ8MzWXIefib1epvBe0r+hLiKcec3IwjshMkaNJiTlVXnv2oFt\\\\\\\/wo0z7Rv5cdFl7mog8z4sHBIByw2h5qjJZSgqRy35KRKDpVknENjT68Q8M+sKbr4evAaYrcYBOziDXV1AHcLNtCi+GzxUQv97WbS\\\\\\\/oPlFlZMaQ4v4Ia6whQ31RihxD9pBFVV3mzesV8FZVu6dnK4UxaCchwZa+Rh0UqL\\\\\\\/aKwhzco166f+wqlRfRLBIojjIOjOXjjUtQco475LW35L8KlLiPywTQLqLFA2WuM4A4LkjGGRDM6Z\\\\\\\/ulNIzvDfwMk374utaGP0cuUBQolUftlorXbzTKbVHmC6Y+DLUAymkAeiSzWf9XGFsdz57HKQ2aDpDMS0FVc3gOnGwpJr0uWH1Y0CwsiBk9H+plxBxF8x5ljKp4OgP3WHn\\\\\\\/mUvDUAosHb2\\\\\\\/8c6PoWEM+kMwf3PnE2sS0N+uofOR6rbnLhpuWXhGnAis3LWkaLpMVKQT6j+5h7VdHnCx4vGo9wAwN9blAmLQuNSVGP6bowO11b+cYi4kvepvSbF5p+Y4010dJ4lC8zuVDAA69yi2J0\\\\\\\/BMqOpgkSuia85p5EpFq0UunAfLJiV6r1bc2AKGHBXgrFfQKMvzK9MlJHXlv0Yi54oij3PItwlrCf\\\\\\\/7TT8lLu+mLGtLzEzYJCTWdi11piu+D9q0NYPn+gLbuM8yvTktmehAMDYXI4SIP7wQDNxU+LDlWkP8XD7Ec7nO\\\\\\\/6rAFjNVOYtK7jXOgj+gkqHBx3sMrGuUKLT9NO9jLpzBZlfjjdGqZ7rj6h0yO3pSpBDw8RuU9u1ArcrR5dQ4Jjlm49ERNPxlWy5CzDOkUcuy7IIbA1JvHf5FLg+Ea+Rul7Fsg3Z3ErbcmEGqgGyBYF5VgbxyOs6IT34TQKk1W8+5qxI8+ginA150vB3dr3uGESxcinc7rDsJvkpn05XAfnA3ZMNsGdtQ3EQ9PLMYbFNowIShwBVfSfCljiK1oCwZNjY1BY+6TkAIp2JIxuzsSz1xLO3KFHGFnnY\\\\\\\/yqw7kXxn2IhJFTX2jXmqf9nvlKivqqIPvcO55rFrDg0AAOpEFJ7laipB+hsI\\\\\\\/rSbmYf+m4WPLbD0jYRZMVl2+NxxUroiaNQGXKqTGXfPxt\\\\\\\/LkOvnXWkMFhzYslTfAiV+BSxi+JJTDpMBTkcRVHD4cq58vvZRjJsNlfSEPqvv+Ihzs8K6khZOS4BOxRUmvVb7nEGvN0N6wVq7OFcxIQ59RS1XspbkLNrgUvNVrPBIGD4vKCQgg44d+0PsnaTdUaFMU7IFag7Q2KQfeebyrtxPEofEqiQBbifvoEWz6hoKMvT8Md+iWbXY13Z89S6Ms2nStuvIJJ1i8EZw8q1YFq2k3TRD1EQlprtFrO77pNwe8GGtdUcboFIAXTugfZIF3rGSK0y+\\\\\\\/6jwRAqns00Y58XlW0vC7otAVEA9zK1MAu655omV2ui0udGOTIhdjKFp8tO7ymSPRuzUxp3KDwinTaFUt+c3MpKwWIRjMUuPxG3+oi1budsmKC5\\\\\\\/kU+9aGuIlNzwmOYyxlcXMsYZWuruBm5fuS\\\\\\\/macpadfCpwtAiS1s9fs1zyYYEwXvcQPzauhrjt1qKDAyEWu+aVOsTuzgRdNKJpFVyj7dhDdMOd1H2uD02A1hbcyQmvnIQk2Ri5qnt1LVT7oGJCTHW8hSiDQNlZhlKH7OQdZtcWI0VjqZuugoCNuTQP1MzuWv2fxlXGUXWigcHjqCNgQh8d6LPO1d\\\\\\\/a\\\\\\\/WtLCTkvTa+o7lrw3PzNHpyFi3tBvpEvRMZYeeiNC0EUpSiUS8QKe9I9uH6iHlouOmqdPLMdzHuC+sqdILwrywI7hPC5dCKaDG6knbldoKH5tyu6dOKiZwDiu986Re5O6A1aBDZVWSQ1fuSpG4x3JraymA+krwZD2uvG3lzv27wxLIiVz5mbs3L3chHtut1Ox3M8DZ4RbzFfvG24R3UlKsbbd+KmVYln5K8edaPlFCVg0M8o2Yy8CeM6vs278Fj22eIBkZYmwh4WuQ9kI3I\\\\\\\/fUbOgQ6WLCDm\\\\\\\/o7EXHPi3qJTbQKruFP8q3Nj35bCgQAIt+DOPWzgozUJsbWIW\\\\\\\/AtcsVlamLYbhH3ll9UR\\\\\\\/\\\\\\\/lawQV0X6S4MbNAxrMDWQaGb2+i2L+F0pqaWxtsdaRhsZyKQqCoUralw\\\\\\\/EWpnVDCkDhZVIrNw0oSWJMD2mssA+UKqMf40wOooqgRHqOrll3mLSWV7e47MoW19YlQx16WhPquwh4Ic86gaCeYB6o0p8YDQTw1GvwV0CPefPkF6BKtbwAdd4gIWwVZxxbzPynBs\\\\\\\/dHShRFovMF0h7JfQ9\\\\\\\/xzhqxO8oiqaKMPIq8jVbyv\\\\\\\/vb6gKvE+so\\\\\\\/6VhPHU91t4xjGc+l5hpS24T4rnTzyXQG8SpmDOu8Hc3lnxT\\\\\\\/nKluZiAKY3KGR4WxK75uEf62YTFKHq7QuzS48LUnc4xNE6IUjSDAwEy5iVJJ72opabNlAclwmuhpElrj2nesr5T3MppEQZfd7AbmVxNOATt6wuM2UUB0P4iXjerjhFJUPOxLyUsZHXQgOb4cbI2ABTQXBMEWxXTDFOpYUvnooQettKxEtqOvXAh1nMsr2lpQ3wvFF4TuWmrCSWgveVk6A\\\\\\\/MWo0PUanexz3Rn6G9pWASw2rlchV0zNi1D+\\\\\\\/Bl6AWUEdfeOt3Bbrc2TbbBI2hGUmFz323t1FQCnNpcF6SDloDoeZPDgkxPBJ4AIqUhZlqjxUv7kl5iC\\\\\\\/sE32MaC\\\\\\\/2Xp0Avh0pf36PsFqA+DOBFcoamYOvY0HD6BxqN+Ao6cbTNOjtcXgnfrkmr\\\\\\\/0ZxtXxnlEtcfbna2UPcT2r04KKAXyTAg707mnBnX04MOGfku632XfPneUjIhD7D\\\\\\\/m7JIrzT0BK\\\\\\\/wXNb\\\\\\\/xz2WYXYAd9pAg+OhXynpQPdRlSlpXPx17E8TAxgrN3yF9RBaceyw3jAWfAjBfzZT7GbN\\\\\\\/y3FNs5ngvLgH\\\\\\\/8lpswmSPxt3M2CB5MOBarnfTK5GbcngP2E3X1veCYgEEi9NSXH78WG8L2pUeaAWZmXrNPB3Ut3oQU1E04Hb70gnA1s2ker\\\\\\\/tpCop07c3Rly5G\\\\\\\/mraDwY8JdER3uhb+Xjx1pSy\\\\\\\/4xdM2KcIeVv1YdGFD757oLtXEK8Rz0zg\\\\\\\/JpI1qrOWaduVKApbNHHM9MflNwYWC\\\\\\\/dMThKrVrYGM47wUfLqrhvQhMrY3G0b4BKgA46fBLWdurRiq4bcZbTlJ3EEagb8\\\\\\\/ua5zp6TEUl8cbb86uwWztVraGxr8Mq22E2j8Yc1GPh2\\\\\\\/48Zg2qW6ncpO328vzNZbI+vt+z9Ca+m6pfCyDgsT6WW+DICBiOfykHpdmfafWV66j\\\\\\\/poR3agQ+FNqEPPSrpZbb7pidGUrH7BBRh2vW3R6\\\\\\\/DLRYKYI8ooLYyHTAaJrGqeGJ4ZYSG2HC4aE12Mzpu4YNjVIatZiS1yMkt8ZcZ9emos9guRVHV2SX4Zp\\\\\\\/q2uvDKfny0fcO1j7q4kMUmB1UO7HHZNRWwvleWd0ASRr+JUeIPF3wD8Wz\\\\\\\/Pa8afrbFctAe0KuHKuKW1b5Dh6ws3MLWN4AaL3eQCeyKjdXLM8YUdVFz31JlK9JuiEslHAazL9q1v61Kf7Giwoe2EGJmDpIzz7VDQjJEPg+wmP49WwvXatNtuiTd8jCTtQx14aTwd3sZS\\\\\\\/FTyZlzRWKMeEOHNXT3r8TzBkPVgG05378PjwgShERzxFr+f6O+rVqs7oSSSOovOBzA0REtxRZ2Og69w5UP+Kb8\\\\\\\/o4f1gfONd9MtYPJj1Vg8gE403WMmIB09spxIg+yfQa8c666LH8XtXlOWGbTdXbKBSnBj60+WrLPPQLJfhpuVgGEsVdvr3G0oAX6bTXO5kHu3i2CXirbvfU4UUZSHNH7XCicuQoRjjIk2pmzO6DvfCJvPn7e4LeDI6hXMC53+rvGbRs8ywX6BY3f2Pe433AqHzFq8y7PR5uSz\\\\\\\/belI2cEZIeAhKCUEwNWhw2UakO\\\\\\\/RfuqaNgjZ7v2QGNuW\\\\\\\/2A7yVWPpGbknonm6cytquLXXa4rPAQ5N7asNmIH1miB9M1ikj7ghoaGXesZq+7YBHuSmVrTDjutz9HKYFsjajFQWjTrMRjc9iLMkVLEUSF39OM51NULKWKVfHIcrbADRAAnGWNXnO5huR7gxIXRQSEn3FtFufS1WgGW\\\\\\\/xOXnhfY\\\\\\\/ep7GnErOaGeLhg10YIpZK03ImsHGr\\\\\\\/7DW8N+5oC+j6HNLgBASOagiW2ZGig93QfyjUuuOiVCeP03hRRkYUi8RUq6AYr1CAWgZxlGahY6GGCAdtvTvS4rmxP2o4dZ9P+\\\\\\\/3MUoq3C1ennjgy6mWiX3puMrUzEOantiBqclMX8uW0h+N5dLsm3uImnkR3vmoGIOJWI5PzSyoVUfMw6gOxX+z45iZXs1MLRq0oQxuuYpUSBfvhbbKvV1kZRHCwsA5iph7JxSDPZZxRvb01gj9lL8\\\\\\\/92W9mkvy56UFgxzfEcdS1\\\\\\\/\\\\\\\/UbUuK87IB4B0viNdsYqzxvRPPOklnBx8ZSyKUacbcIfCTCLVnGyUipDyXmLa4+BypXWmKbFM2nQarpGgnfqCgceJIY33sCWcREQCqqvkFpuf6KiMjIPTOtDJqx2EdkxqQFigNCLrVvZPvOyXDREwqOEYizwmcnWyvym\\\\\\\/QXfmZZ2X+HlEBzSfY6A3TQJo2MaVV\\\\\\\/HAAOtkywWK2JZI4blmzxydmZ4jM+7xwNSIjFTNzjszqYWVVp0twY0NehE8LMxy\\\\\\\/sZP9X9\\\\\\\/zsbvjJ3pPTwzqpVVv3+V3LygyOZ08GuQmbbc9ZZTcQir3ilvm5+YEiMUrNJ4ljf\\\\\\\/nmUYniy+dk63L9s8diL\\\\\\\/iFzGhz9GvBz2a8QanV6WDJ0dgpocDZBhGWQD6KE81UzwcBMMT8hhqAJEVOKDFHR072ntoty2khutmRBZw5yIIHmpuSuQTjNBZYHt\\\\\\\/Fq+EY8kKmcYkUTISxbkWy\\\\\\\/brbgv7wID9W8U7fotMeu\\\\\\\/9fjgBTBaOPqv1zxnwG0V91X2E9bQnSUxsDT8KPEF3dwnocWaLwyg\\\\\\\/OZVVODgU3PhzcFiJI2khGHOu5Pyqo6xIL3opYxlD\\\\\\\/oaRkGGZiXdMJ4A4EsVaWzhtZseDshH9F2PPMU4ieHQKDmxihYzT2z8te\\\\\\\/7oYns82exxwYXZ01b+Z\\\\\\\/HspjEBb5iDivOkrfupK63wWkBYde0lGEswkd5USKTOD0MWjif1oh+T97pHMTCYwrlTSdfD+UIDNZ\\\\\\\/VD66l6crqLLck4ePIlUv1NH053YQNOgmNdlnHNBIUTYo+hF7C6CGt5UP31Rx8W9p4W4PWW6vnIQI4K4sNpexRv\\\\\\\/MAQ\\\\\\\/HniijIUaFIF2sSbqg8XMyGD35wTMbVV+U6\\\\\\\/rtW8nqbwnxY4REM1bank+b7moghAtKTtdXakSUAWC51Mb+eKSG9xSyOMYjpSjhtTNlxkOKz03kdDMZmQarozvssZuyH4HZFwsxdYncPXwMw95MnZtxRxdaO005lMEIQUeN8Cijj6kYkYE2j46sg4FcKZWLitt3pH7CmmCkf1iMIlPBpL7f5o\\\\\\\/RcgVb6b8ItyL\\\\\\\/1APpag4VU1qC0IT9cUJoMM0jwQxJLgclpxrm0g7ki19sT57JJs5mlErWuOVCLQiU4BK0rr\\\\\\\/aC5+XQ+C6MMtXwGc98ACgMl8BssLTToSElQjPAoSIMZZS0ivHBeGejtWs6kZY9nMIo+mtVcTfYwno+5LcWMTcP2IKpw8ikFZ\\\\\\\/7zI53Jgkukx9DwqDbBSz+9GbdolCMy374xl4SsJd1jHRtaIIZoRw5gK5vXv0l2ZFu7ZbNa5AI4qfyzT5ZLLfqODGmZW6YAimq1XAFS2JgSkdnkW5yvz9aNlFPCRnhYdr4DHCRj4IwbxrX1Ji\\\\\\\/Hf7PrhjdW+h4\\\\\\\/GZxZ0GMYbX34xnO0E8oPdlkg\\\\\\\/2jLnKZgXDlWS9pSwGs59kQh0brNFNnRxmeHy6LTWu+EwtiK0pBLEKhBrnKjIbl1Q6KJ\\\\\\\/9e9p5oYceAuaVl2uR6n8Nzdzog70CUoLRdQPVW2UcOLvFnPQ3nPorFGEGS9cow9Vw0RfT0LJ\\\\\\\/dgAStIpXzhOkX+GuhP69wl9FVWoucCxFLFHpDPbUIBrBAMXZxJFklW2vyLChvfdffOQnx5XT8QYyWxdCdnCx86aLq5TnQCEThLGIRMPhx8X8E\\\\\\\/PiGeMORHkixkn4LG6aCkD4+PyXCDEX7fkGEcoU8mxQFhU5sPH0SX+lGVpBlfZxXoEK25TYLtt2rta8G5xenzm7lbdzs4VaYQyPVlL4v0lvwMVDLSxDmACo5oUC9Ke6iec\\\\\\\/aCDp2\\\\\\\/G9N0BL1EiT\\\\\\\/Qu+si2M0CjDTMY14lMsFC\\\\\\\/sc7n9ud1EVZ0z8l7IspAYmy+N8Rqxpdq\\\\\\\/X\\\\\\\/BUQqWxCTlA8dzW7SSeQUqt34QZzfu5wOp\\\\\\\/NurxHN+j7qnZOPiRWQGfYwxyyXYd\\\\\\\/2f1XmFmB\\\\\\\/cuOFXoSq5O9knUSfZ2I2Qha3x6ob\\\\\\\/\\\\\\\/uzxXU4yvYjL\\\\\\\/qo1L7YJ8TyY2E150LhSZKiFrRgfn4T0qgT9JMUpb40P1vMbQ61O9zXFO7PgOYVOYkQ2wQh6ysIDd2\\\\\\\/Gc5cKB7Ng22svhNkJC9NJ1nBPT8S6ghw3v7+BB81Mfrq8bqVXlvvqVj5j0x2d2LyZs87DKCpNX0cXbjHVeVLRHffwfc4KlDB7SLPSq34enRbHyEFV4HSeZEUtxRt3Idelam7ni4\\\\\\\/oiqz7SArgyHzMSGLBZZ9eW5\\\\\\\/Hfe7BNgm1xXgTGLshcvbpmdEM2NumulfS2zCcJy0mYAErwlK3iuiEkUBp5KlHpPsuN0D67XXJNHI+OEqh2bhAQEKAkg+O9TT\\\\\\\/VXs+nHhqXNazs7YjqbrSZe0ww+GipWp9V\\\\\\\/vVbsbvd8wsZ8mkmv40g8PKacvruXgBl08F89hkj1Q5hTeLNDOvNZidhdHf6ooOsDj5KONv4Z8id9g9IhmoHxFtxoEdd8pT9P\\\\\\\/UppMv2JJ6cBWcViN\\\\\\\/nX5WSYn9FGFCS3uchUtoAV88TjWH0b6PUyTMSGUe+CMFJY9avpUu46K\\\\\\\/C7Vzdra\\\\\\\/krPsDq5IMqV5EDz7+rLNcLfjfpPHTUbqMpkdQcnrMnEJPPrBV4qdWtMYnvfRNm3e+s+rzqy6Su0wJyxbBJQQU0Gt8mcCyB5IXSTYA+V\\\\\\\/bgZfarYbDqj5EDV1vTAE\\\\\\\/USR90E8lLfaYyfXPHRnQd56GV2pLI+g6nbTIhbO4JGazAoTQxLBIh3unJfIKSQdVL6Hb2lDhpVoTl3V1b8fLcp9bRlzVpoh5FQLOWB\\\\\\\/lbeH3\\\\\\\/TXvBQhFJxMt43MMtoKyAyM3wbbJjCuEcm7ENXgTGvQ4heCf8WT5A8fSgLrU+KE0rjn6pGHcndDrdryKtKwvZPNVydz5p+ZFgwF3R\\\\\\\/eKXvG3hkqs\\\\\\\/zr13PWe8p6HpYE5O1RxAAWugs\\\\\\\/rFVoeEHUOxbiBDmgfBEchkg4OCTJoVCq7gtiWgWQUOm+B5D40d8uoTLp4N\\\\\\\/D08yEahbPvxMu0NRUeoe13jWAAX23EJsbrcDUuVVSrrtRBU6p59iQLzGsLIQaxKvT2uw3Lv0WLJsbONNpbOrKQ2K6XJ1tbnRPtgPKu7CIR5kAL6SF+BwclfIt8u73iVqTA\\\\\\\/zM2ykM0ut000AuRMA7b9qu0UG4k+z\\\\\\\/mQ3ldO7Kd3rV2SIkQpQ6nAUS549UHNDhCRjgy7w7gUVpVZ3O49+l0jKKd91MrxIm0K829lm30WbVoBXEeuPB7PLtXnF+RuHssWAqZIFYHllYiqq4yzFp0mKa3F1yPjezCusxMqnjG681fSc81PYgf3km5js4OJVF2+zFk1sJPwzK95ivPqRL79p8asPk9o01PY2GxGDV6aneznQeMvJu3yHAwHcnB9peZnLIgSAvRTmHG0dIYOxdYsN0SJdoX0jF2Kkpub8h+39oHQVtFyKEHgSvMkYggb6Ya7le3c552ej0dBzuBM5LXN6LH+TxHC+9HeP8PaVOKhQzQra51JnKVVR+k+\\\\\\\/ZIywnsN6kglvi+8wVyeO2M5+aFjL0\\\\\\\/N7Oma5qbz9AL\\\\\\\/6Jk0XueBq3ctp7BYNoduJFthl2D2EAI+qV0w5oVxqdnh5obIWDtfoHZqT6\\\\\\\/8CTzIg3C0qJGHv3r\\\\\\\/42UxOdc3apHL9cqM86nJ0U+FuuTAAQpqkGOaE5pOXFKA7u6HxpQuWuccFP4mBM6m8n\\\\\\\/ZNM\\\\\\\/gid47LtQLttHS9laFZ5P+II\\\\\\\/bUqCgDyvZ8M0XaEU\\\\\\\/GI+lgAFPz9RLnyyWkAYAbcQ95X89Kh53b5SxT+C2DK1DaoE3\\\\\\\/Hv9d4sM\\\\\\\/yjb0o4LwcKeYMWfta84HJnkQ2IUCdXHIsnvjwcCfmtjrY3UG9cgBd8nHpSg7s53mbx9kdiR4FDUNOz1WSkT80872qGOklvxPmC5WNoDc9h2xYQOCJujBaRXdcnh4Xsbo3zbMffC7TimSRJD9z+B3MzP3a5FFD7kDweVEeqDvmP5g4Mhi5D2gm6DMmPrRUeobV9sbkOQojQVrVnsQB9WmrSgpk\\\\\\\/rI+zmp6\\\\\\\/Rpgdp1j18PO5Y\\\\\\\/8i5ONQfRSx40tQwpbEurs2l\\\\\\\/6WLvzSiV5C5lIQlBOPAsPbk7DLFrRsy1Ec8Bm1SFEOoKfwyauAMiBVe+9I5rtG3klhueXaFksE3JDlyyTPFNVrk9eYmYoxMVxJNpVjWzeT5pJmmm7rhlLtPCzBFX4+2VWm6wH50MCMuWl7UTb1aXgLQS4aOW2+kQqsBb9jwjF\\\\\\\/wesJkFLBMv+G01ehav2EMal0cYs1\\\\\\\/NA9GtYGmQQtxPMqDq69fHzj8l4SFkBVEZc3UJ5Hll1mkvr\\\\\\\/PWMkZgpdo5vigosuvRVnREoCjLRUgWA8L+6KcaHX+P6ZgMeyBUd6jZTFNTsXxcTB2dx0\\\\\\\/GtiRBi5WJIob6\\\\\\\/aO7EtKfffyOAA3AsnaIK\\\\\\\/OuHNWYi6CkQANECeo0D4WEn7yF5x3ooqRM3Lj17Hjh7RcvEOvq8SdFM2CWNEHfvVBE+J3odOqGFvJzOgKG4znu68gJoN9M7Tp\\\\\\\/SXe5a+nbwCfsugfSzut\\\\\\\/odaicLU29qH2iCbrv2qwheLP1HLACIc6Seujk40b4nJyRBsHbMfojwGEtf27ja8ZREWB8pyUX7pzDCf54OvbyEE4j9mzuf0jhu3jr9EAJbzhbT23SkchHoqJVeeCcCr\\\\\\\/N9M7kcCVLxd132vTcmW1chTObmZfC8AdZclNiM8HxsBoFzVK3py9fN90fN+lBGj1MoRoG+j2nHSzGD+N7T6U\\\\\\\/VSwwABX0eDlAh\\\\\\\/uCsARSPehVeeNr2\\\\\\\/IwVCICIGc80hhZmpBPU0hGbKVsDTTa7SKcwL29\\\\\\\/A8mXw16hcCKQKkcc2D5gl69pZRpHZN0SZRhJcVkT\\\\\\\/AQKfHfNhN95QfBhztYIUFaEHCkGCtA9KGUwR1ZXqqKmGiLlYM4ND++H5qiZjOBxzJ\\\\\\\/fzelvIQ8ewb2Kv9RaFCsX+6u0QY3q\\\\\\\/lH6tsuw9Vqgyt1el\\\\\\\/n8eRfgaMLY\\\\\\\/btUVKVlbCkKWOecr8lh5LzZLzjRXE\\\\\\\/usK\\\\\\\/bE6iUjZAkJxaBGLSa8XuEmldFlh4B1OGmm9oFn7d8l5gnbEwEfALDCydA34zuvPXXBD24wkSrzwrdLftzoeV1DxuEzNuM+ldf3iQJ9KVIISMeU\\\\\\\/rDqat4hg2GTIsYz66zdNSJwo1wMEqcOpxdH86zp8LkbP5lIt7wwt+Hr+vxP+meEMKEFWBlfBR0pQ0tNSoYqB\\\\\\\/5FmR\\\\\\\/vsZa2SLB7B9l6Xj5aSFUbr16VCygVZTGG6bVBmxQG6O4jveCZ\\\\\\\/c8CCGsjzKhwkWKhn7wDvTeA\\\\\\\/GGPfKWlkZuQRIQMCbWT2F9WhMLzp1vCCOBFSSMPRpK8ox4mziR7z4jtOyRadlNT4rxsDk\\\\\\\/\\\\\\\/ohtTrmSvRX+vHOQ2T3rPwYb+ceW8teASWJRrVKHSmpS+Mfa2+d93+tOGwgrdtn1qrrLFG4i0s54y1GWdadNHj8kTP14iOWyDIQvC9t7KzL4jv8CzWSEjjyeXeIdkInUBmh0yzF2HASnbY5et0JX9I1Ac83C5\\\\\\\/JZguF+OjM598IQxrXMNpiWfUuJH5GODB+6lJfCxHFFkfi2YXyRvndJaAmznrG+EnmM9bdp50aRaDzCHdIzgH0O9MCdqNDCjAQ45vhDUfOmKBj0Z\\\\\\\/NPETPb\\\\\\\/mBAAo69UKuKxmeo76YunNLeAdVLrI\\\\\\\/B0WC6bnNbAyY82YJrenQXkRe\\\\\\\/K0Vxcyu7TtqjDx+QEAsP03rhYL1+mwy6sDo78YxEpLKTYDYGEOZwn1d9f56eYmIG6GaG9JG5ldRmfkXwOzUATv6LDj+NeL7c9cTwFkZHpaqG+LYtjH1MXXgCw1YuEK0uTKCTExC7RqSrzfGRur7nTGCBWOfgnGyk1lGX3iJBXpi\\\\\\\/ddrjBZVzpo8mob\\\\\\\/ofyXLj0kAEr4wausi1dmjqoF4NC90TN\\\\\\\/KYbVShS1rTZaasRsv83nN6WsOQes6ddwPvKZKwM1y4Zlj7SLaB9oZ4MtKHHyJ+ogbwhPQXcX1IvqidPOClDAFtx1629JvO492mJjhZXNoE+TMsUE4xqQ9RJOHtW\\\\\\\/gzrywT22MbBZf5nPTI+wIu3O1dpTRjg0G\\\\\\\/Ba7DFi2BDY\\\\\\\/LqdVaAGMe7tQcyuks84SR\\\\\\\/\\\\\\\/OkfBn3h4yvXn1FqGNCXZvTOlp0rBb9MpuB3BCy3F5XgqpPQqGSo87xM9xOMrSOANCUYuJc+SehOoWdy7b0tNYGasp+NX6VYjZv+mIqgM8XiOHYl77wNLl3E6QhVkkXBtutN\\\\\\\/W3Sufy00sQKUQI3dFXGK7vojZLB5AbpP55Ey\\\\\\\/V\\\\\\\/hPlAr8\\\\\\\/a0S9YkAswv6sgU91UiBvujK1qrT9ZXPUkXzaSqIgxF2\\\\\\\/A2Gs31MM+hJHraPnO9rs8r3BzBlgghlbTyUu5MRn54fH4llH1Dxffxh5sufV7X\\\\\\\/8aTwMZfuS4Leq2iJ83RMqzGNKPIcRsKLroBnSuWUa0bPNiOLMYSn\\\\\\\/0Ijoz3TB+i3wHTHqRNUSiXRBcgYHHokSFab4FhK\\\\\\\/7+brzDxza0KAAiIjpsrKNkY5gHnZf93q55LtCFQAxCzup1mRdswkejCvpg7Ip+Tm69yzuj1KRZXg3K6LD92ZmRmocbb36tjvZz4pcgFuztNxpKZiQe8qb0KGOHTzrQ4n3bb0nKwvNzeabjGA4gUPtHw1fcpHrYHvL7SdNRBJK9VCCzL\\\\\\\/CRjctTewso5Kx3iEmyM+rHhZtE8gFYHR1TdFtqHKX72h0jCZhOkXkoClpw+dHR+HX\\\\\\\/aPpLCkbR6EMADVODzb7tKVP\\\\\\\/e7Vkev+bp4Zblf3zdi24Yi4j7SEqBNhyeArOXf3vuTp4rmI1eC\\\\\\\/thJ9eXuh2M2ROuoQT6iVb2DBhvvlkKzm+BYQqbPGXUj4Xot2GQGHenFf4ufdlmCBLjC9I5Z2JVVOTznXDu73jpOGZx7T46Oxu47g08j7PvxFIha9ANc5FjVYuACqZ6AToQ\\\\\\\/jGNmSQZkgfGiAAEzHreu\\\\\\\/tWbZFeTd9MTpxl9HMt33uR8wtsE3eXeXHbCIJ2daBRowbJO86hB5AUl9Hr5dUtx428r1z4pYr1uXQ0NGG7TgaPZOou39lGYpsFmeVI5T1aLeTOYL88qzllAg5oHdrZPFIbyeFtta8tEZb611U2VwZHNtWtWVL6ezRD+fbXeL9zHWHZizLPbniFOcNH9Fc\\\\\\\/X8T8SQ2tEzFOS\\\\\\\/U+SWwG9r4UUtHX0XrSfFJi5KfZmP5xbtCUOw\\\\\\\/Ae5NJ3n5+f92tUBN1EOkBl9EZRvjfrRjjP\\\\\\\/SQhsDWouiVmemKtN7iYKsPYxPFEcThHOxSTP9Zhpnj7sI09XhnJfn60OzYZ1Gx7gwExxa99MVnEOAdsXDMQFvC1BpmoOkqj3dOfMXg7j6DROmOeBE39RXKu4Vjm90S9sxARtaNf60b710byNWaQOr6ClvK6uFm4cxfl7IdqPQUtZsSaUH8eMm4t62jhB6ulPH9ly\\\\\\\/12rYVKWhGdgisx9UkPBKXX+vm2Dgnox3IzpWbIEEGVKkz34Plq4xVtX+8C1hHp3eVGG\\\\\\\/eU4oAGZj2cTsgdUJFAeJXTosIAkv6kvWw\\\\\\\/Nm3R6RnKaPv4pkhpfitqosSM4si3HyokbPJG3uLnHdKGEcal7rfSxyLITuOFMnJ4hVs6ITgD29t5KG1mbq1jrk9PRogQ1FdJDh2XuOQxQCORXR8MwO+p3sGj3XmXCDQgOn\\\\\\\/grNLwcJH6\\\\\\\/T4BQ3l8o\\\\\\\/wNS6e7LvjfxrSzU8JmasARlbttYzMjnvpJXc1mqmPoqkk8wTzfgcAICD7TOWHPImUMqEcttCfPM2dBhyJxbjLkXj4CxybI9W3U1wzbTtQgAGzuuZTYrtf1IbujTrIiSX39l0e\\\\\\\/YprP2UOXoJOGr+GmOZPxxE9yUoZBgUY0\\\\\\\/Pw9XzzwYonwVizku616uLIF0uRGituch7j4\\\\\\\/npZ16iZsMDxvHD2iLIEb4UZZPp5R7HzIj1KtP3Qr4eKX7hgZ4VBzF1azj488RpO7yUuwRKIKey0koTR8LEacmgGzWYPFqLP+bn3BkrrTjqaBErp+GAj1Ewraydwp8It9l\\\\\\\/kYGqf61WU9fbvSewsk+xWX+Yep+6KA9x8TLkTA\\\\\\\/gPKdsVQlyyxwU11go6GlTKOguwqRMDnEH1GYrfbZ0clO9anDVoTOFWzz63eqv1ZHgVVk8py3pXzQKk\\\\\\\/WPyAab3PQvIt9BaTI72eACFnWUr9lw1VUWY00s1W3yKtHNx9XM2rgnDeM4+P4yCREsDXqii3hKXQTOt4FM5tIX70p+j+2KAAxTPUKJzBjxpqcrdh3vUefqS+nqd2tsJeAtSoOMoc1nKKjc5+xwExALLByCYwAOW2Eu384pOyll+q7X7CXkd3mMV9QkeX3I3FepsKpThh9E57y8kbnlH6YHiYtyi7MCs6e5BDOw895qYk28z83DBN1Dd5kdelXAv\\\\\\\/Iv448zPRsuQKH0\\\\\\\/ygcIP7MTlhPnHM732mir8fI4GEcreNtbzangw1ZE8FaiVNXJRm\\\\\\\/A9VLII0aIcwJpuWNQOZBAqY2125nA+hEOBp3\\\\\\\/1AgzS9xui1Lnk75IrQeDosTyyZH4itfk1auHcuNLKsvu1Yo7\\\\\\\/s8iNHZFABTfMWT7XwZrM70JuteP0YXH92vQb\\\\\\\/S3V2t67GXsSlllatKNoFmvgOUtPHEsWZ8325DakF6Nm55p1EuaP5fRJcHace17\\\\\\\/4IZJGWSPXuBMIZbXLXIfuRMoVuM\\\\\\\/ZwlN5LCZSLqZRxnaJTb7TLcnwHWcpFpY+xNaxdPSvvPgiKUrPSel9RQ0YxDHTAhpLUW2tiTpHWT3wx7dkW77QllfWyfglC4kGxhSY\\\\\\\/+X29wUeyDwEQA1RuZM1X0Ol2updhnY617psZUru2X8zlUdete8PiP\\\\\\\/LZFX6OjGj1N7nJIkEofj2owT4eGVo1WpFPcAzQgvI+jFsmHdf5O3JCwur\\\\\\\/98y4PvwPv16vjBMr+N6d5SJ\\\\\\\/WQXkpA883NOIM1WQckP7NiEVvQlnKkxSqqVRA4d6n3YDT1UO6n27tHkG4\\\\\\\/OQOclj9fbN0qlH9r60CkvjYmTyWFbZwvbEcBa23akd+gE1RPzcD+QyQ9JNYdMRvyWaTvG+yo6ihqrm4nsWw8KrsImA5e9dZwgTAI0Jw0jlWf\\\\\\\/lR0j5L\\\\\\\/nh7WK3x3ldciQEMhBdKy\\\\\\\/2zPC\\\\\\\/EcnYyXXGU2OCgPtfJbpxRERN8Fr6Lq1N3thE5qJjgVuFvV1Sez+YzDgZlv3RiafOiLb8wdF\\\\\\\/epm+\\\\\\\/jOfA24B7B6iK72UrNHKQCT9Z4a7sfK3fWsIXGCZX6u4qNmbjBcxFHa83+RXD3PgFM7KPfOdFNt\\\\\\\/otTokt+uQflOMgPfYO5YptNZis5j2qLnO0NQnCs0atT7L6RyTQfQAu6jHa5khHc3KxIEdKCkodV34HYBda9ByWOD+EmkYipZbx9ACXMhf6ZDsZw8s0T860DShv41ObqdQnq7ktg+x73fHTtjnZ8aDCxTbwYg7+Uvub49bN4+MDn2cmkosDBICegBFD9xtR74qwCg2exLyPmsTWFGWKD2iHLNgYu3Ietq2CPWIbouV1OSyHvWyp09c5y4u6OJzuK6rgYMrHVmZy+BCsuMJB9L3pV8r6QUuMldIiaLXAcBg812cFbbyihKEuLVRMTJ0xanTou1dIZNLjOZLbK11xxf+eh1Px9nWCfpnLE3mpX93J9I7SAW7L\\\\\\\/orgf6nnJzfymf9B5h8RcPP6cmKU9bXDbEKGm9WAOZxZOG9ec3lGZsnwYFa\\\\\\\/ZID6R0ENCDsEPf9XRD5ieedD0Xe8oeqyk8xD6wQ17axo29QOvlLOwoxatkWez\\\\\\\/Xmta+GQBJXhrogZP4UpePe+cCLjfYk96\\\\\\\/5q87JCnpZUEsKv3bhZobrRgOyKxGev+Gkv0Y9BLCwy4pTrvzDG85d+lanqBSw8JuiZdBAbMAo8ySK1mjenJ44QIcEvJ1qe+82H985oQp1mr\\\\\\\/HOg8J8WCTQHFddAuYKqjec74SkP3h9QJFJx\\\\\\\/AK59V0ybvT389KL523sd1fxmcJsSSw5tqGwSAlsQhARyLTFqx23uTvbtfsY2CGHla5DqF8rDpOit7F44jdlIMhTdlXsBD0XpdCZmLvcCM7kr4PL0rmuLhorrUxC+Va0I5Xlfstp9qYScn4pxPxTGmwrAoE82GTEPNdZni7t5KOmNITmMpMSvCTzotD\\\\\\\/cZW0g9C8DjwYNKcUZh+7YY6l72s3fn1g8gvNTZhwalfutn2ehNIvZLad\\\\\\\/C+12Rakt0NYWEN3yKpjkq2u3Qfi8wf6dUEE2+OMfgOi360fyM88wF4gbkdL4xkrV1yEZpBWfgg6oOU9l90RpNmYZMfeiI24yZ40IrxmF8g+hBPNzl\\\\\\\/3LKYReNsfCgOa1849RpnTr6am1aRgQkvTygqy0cn6XbJwPAbFEjunRyc5Yr2Ko0s15QPNAF08GFuXiav62HdVIBWYvO3y18eBLQmrqWnf\\\\\\\/UY8LGhZ\\\\\\\/dPfHv\\\\\\\/IBt9QYoGOP1garKHJvOZbN92jHv7DtK1u0JTecA8f+U7QVdbRtvxavxLbsSIu\\\\\\\/EG0D8BWzo3EvZcxQC7LX7VmdYl0PEBn8ALX\\\\\\\/Bfq0+NZUGLpYCJro4V32QUgqQAe83x2Dx9CCacKIxhHbOFmVB5Twgp0V\\\\\\\/HmCTLxyFLlEstpmuXAczVD5R9SCP7jdGf75GjQDyXvij1cOi8xtLCYVdTz1uPvbkKBcXsznETniaf+Un3d7SyVADBCo+7xT9vXDHDMW1hHnOT+I27Ee40G1MLilv4xuBLhekjpIYzJ2C6W66zv8p8Q+sugI8h90VnRHtLCZ1TvCQuOyQWSG9E8Zq8bYLzh+kD+GaMeY\\\\\\\/MRYKubFy\\\\\\\/8RJRLo\\\\\\\/kBvu5qvNoTHXRM6q\\\\\\\/WYseNAXOK4FM2eLh3FutBH12527VtD77hF0hvU2IT6xVIGlQMUK6aF79rvHsvfCf6+TBoKMveJwCGrP1Uu4Vw4WxfXWW4d3wxCAHmZdJIa92JElUdPX\\\\\\\/nBb3zspiRxX9wOYMOIeJp0EntlwmIKEow1sy1uyowY8PVKI4YHQBREkHx6+66mW0NWCEHFmXw9dNAudH\\\\\\\/9SifLSc5LOulP7QZB4JTpsVGaTlyPuRk3OL65zuQKLN5kCEU5BtzTry81bTLYAwkmHDKq02lj\\\\\\\/lv6cIw72YIM5dBrfqZj1w\\\\\\\/kiLkMSWpgPNOsvYPLPH09mEXmHgrXMYZdJVJw0nulHzo7cWcY\\\\\\\/Gm3DXBeuNwCmBxCeRKDa0FmWeq26Nzwbn0xs+kaEikEqH\\\\\\\/iBPPFHhArnz8XvtEnDHF79Wr2Nk7\\\\\\\/mFcu\\\\\\\/i5R8lYD9f+y1FxCyYOnhZzD4FlKtEnpEIn7NfLsKFSGlm6jPAv4ogrtzBM7GgdTvfL7vLZCByPa3nZG5WVh6PPfzHJWqU7e7PDh03cwGqKqKgReWBXTov894R2M99aJaMxYC2foc3K+wO9nyl827BIclLb9RCYwzWsdWNkPvGGg\\\\\\\/Z11QmnMQoEMSKZQC0NvDPZO18Ct+Ajql7tSvSpDeb3Z17N6vf5FCH93cLiZxYdRzRD8XepxoFW3u2BFwPYwO5p7ksBH8w\\\\\\\/2SxceOpyCRG1EvFp7EiZw4JOKSVVlK5J78TcQ+a0Y391wUjZ5N5DfAj2OUHRhyXFirqhU6Q9HfayGp1qkT9a+briQ+JQB8i4M+Vc1SondLLJZGImegGGz6c5Hl1OheLknq3\\\\\\\/5sIdZ5iD39qEexvJuoI+GJB9\\\\\\\/lBLIXJ48LufncdHrxICcbSk9rg5RRUxiPqfkEmaYHcqB2I21E6UnQuEKuYydhD11O5PEmPiYGZQJnvv4\\\\\\\/l9LuT+bPLnsrxnL752vVkmlRS0WeMaVaVulQi6eJTnTwS3hdJzSlfW6fzkiMffuJBjJlyYBbg5qMSEldASoazfE4xWAoxL6THTRZ8KdI0ayJ70yS31qJI0DhcAI4XZbZ\\\\\\\/6QFLInDKNyMcJBwfNv9MTQrJqcpb1tlrItM9xSzpkXgwcTpa4sKVVseMGE6BAmhtZ4UnBm\\\\\\\/a4oVkzKRBBbHZAXHvexwjK6gQBy0yM2ZQYEADOHnbQLUXg8F7A3rAMPsbCJFQrvF9LovcT3AUXlLBu1q8rJQG7Tamu6\\\\\\\/+wthNiaNfNzdTP7kOm\\\\\\\/iPiHfAQjPKcSPaJlp3VveK67q3dU4WOhtKFaCS0jKwE6Oz5li0td6F7UVsJznbRZ2zJc3nUpbZ6gt3Fyf+Wu5uLO8NhYrRcwxm2q\\\\\\\/27+SdTMMgca7pDn23snxWY9k7pLyM22acWGVRystbG7FMnWo6SC4G5zhGl80SHT0AEBDHiiPHE+gRFGZ7pJlE8c90\\\\\\\/076ItrsWO6Lfwmhwo65tZ2e2qZU5Qli4Tp5X34RW+kgTcGksB3MNX854bA8g5nZPELJ5hnj8pMTrhXzY\\\\\\\/T3KfMDkqtmBEcPri5HLeu8Rk9br7IlLj2ycSFh2x6ensEjpioM4w2ma0s5BqRAIC0lsae38Nkzp5u1yX1o\\\\\\\/0tx1Y4cU3Lfb752qxTw6YZRHGSzqXNuNF50DeEEmXvL736EUIvBiyoTjAF1ttD8aoRSll8rp37JLCVxwRA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RX5eb3rDbMOxHSzD7mmZ+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a0abd7a3fc250ddfc63a1e4e2e4742b69e2dc16a8926caa738efe01f09090487\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J67x43tFwNFkQDpYK\\\\\\\/cSzV8UmlflxRg\\\\\\\/vAicKzFFFNnEzhcRwVWxyyyhue4S+OB\\\\\\\/Y6xZUAa\\\\\\\/6JxggaYNAye5Rsnv5OZwhXZI6nap1wFNqZ6tT3Sjxd5iVTplP3ZVRu9wqUqLyDkJDYz6sVWGosVrX0yW83L1oDcbepFlC29rFMXcyCoRAlo8iu\\\\\\\/X+NGvUETxPOrvUzkfGiTqM9LgiWfVbRdqWHF\\\\\\\/Y6mdtTMpl1IfN\\\\\\\/az8ZwSmZfiwbPu8XNZfN75sn8jHxmtuNLhL5b+KCEm4NdmY0XIIyS5YUfPJnpppxBKbw30+AO16wwZ0nsAbsHXKdfUCoTfJXfDfgMN28ARMe98F3nKd3f2PBhJu8zXim1kybD7LdMzbMNXxbMFEhYBlOrSjX4M+SFW8cNb0l4DI+DNiizboDs8Z9WEpRetIPMtRJot4sLrB9lvrQoC5YPev3KaHTmekAMw0PEbBvZKD2wxMrD4OIkxRJnYQi5yLRAPTpAhjSooVvE\\\\\\\/\\\\\\\/loCZn8drkOkh1m0y312SA1H8GzoVS7b4TYESDAbef5cUJ09V9\\\\\\\/EZxEaL6GTeV1YPrtWglYlUh7CRpniCzdavDwz48pa9ZfT9M\\\\\\\/FKgBL3XascetJok3tXmgQvlu9+yHeQMrw0KpZxYy3CH9J3K4J6qjWQ+4V9RCH5kEW\\\\\\\/kLkRhNbUhcuZ4FNa93Ki7EZaY8H8VfdPW2dDkDPQatScRT1r9QQiw5khEzHvQl\\\\\\\/fcY1v5U+r\\\\\\\/UQDQEE7\\\\\\\/VBEjdEZirPRal1k11RmlISYbO6+snij9jgv5p8lShQguyK4Gf9sEItkiNhcT5193AmmiwBl4LiZzJH3RSkq2dHLQDKYSKCCsRIqOx2Htiw59ZJfduBvtGYs+lwls41FiraV2E8YyUf6zqrOn+72zH4t49YvRB8SXTz0G7cL2phuK1\\\\\\\/IY2Zd7ADe1P5zCZ4fplhy+SoyzJXwvhQQlSznSO8Y4koBUqmq7CWFSj0Xi+\\\\\\\/2Zi0WHq81XajB7r9w1YdQkRtrFIbv22a8Ua7QUYzoYgcc\\\\\\\/i0\\\\\\\/zB5Ssc382xkzjXMrPq84Vv0YarMWYBpuME5Y6kBq8fBqT1Z0Vgxc70QP30pd1KaKcyhfKnp8shdyjKc6p3ZYF5OQylF3QzBJuTZRjwJBO\\\\\\\/I0BnPRUndB0wqmPFnu2JSZEULZ17p4OjlhjxwGnwVcsuetxdIYb4JHVdsqeWPlbZ74BvL+dEhAMP8nzE9feg97GjXOTFz5Q+Zfdz4JBh594F9o9QnhqoMCwAhVMAy5Hq\\\\\\\/JAFDtpLoTEkBQQcDA\\\\\\\/4+4MeVO5xbp0iwG1rKKEVAt1hrO9xo1J4YmfTUNysavE5DhtzsNN8w38SGihpbd7iYthMy9CW9+vByNgK+p7uH3eB37D\\\\\\\/7daEAv7CkHIFpj8UacAQim9BZO1kQzsLhMSKSy5vVCqZ9+BpJw43Tditn5Q92UxtYMCgEY6jT5o+lzs3Jp1O7W39bKVgYAxaD\\\\\\\/weCe\\\\\\\/TuruvIg6T13bb6VtfXwKdt1m1mU7RwWAyEQ+xgiMGKg+GApShP5eFm\\\\\\\/+xrdvD9K1xdfys9f2B97BaGf1sTTEtrUIPVg4tVI\\\\\\\/EpmenlZK15ecC7Z5YzsRhEmzpNkAHURN1j2JVA+J\\\\\\\/NXEIcwtyjPXkk1LBLCxivvJ\\\\\\\/q+GwJWFXorcE+9YwTOXIY2ZveavJ2n6tEYSc97fZmNgY+g40QiROOwp6DDXs\\\\\\\/VEmZGHbd29st9lKbak3LNn92hQ5\\\\\\\/L7u4siZl15NJ\\\\\\\/ltq6n4kxY7FPTV\\\\\\\/B4VkhkTt2aTgAnah4ydTV32R2CGD5FhtVBEfYk6RDnIbhr8+aLh4\\\\\\\/vhNjvWykuSQbHEM3rT8FCfBX5jV1+l3WFi9CcI51Abc8x\\\\\\\/gPRAFhSWNtpm3\\\\\\\/RQdmnYNvuVb8GH1SFiPwsm600prWp7zl5vBdUEbv5BTSBKdOuGbQfO23\\\\\\\/W\\\\\\\/BFLf6\\\\\\\/aa7OFfjVkrxVXFmJC0tEAhSg4eEwXWjYinZgf4M3+29wiXGFtO1Su6m1IRCGaLklqD0A==\\\u0022,\\\u0022iv\\\u0022:\\\u00221g198Tm0oQozyLWv7EGPIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f0ca5d4456982a8417aa3b907c236c7402f7ca402c6014f70412a338e3a33c99\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494998 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4073" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_0Da7YS4Zngz4BIgbPq7fZbUL-KrFEOCeMF91PJMtrA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4073" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-po4IlSHcUE5sq5uLK9PIdwyhyUXau0JJgjBNQ6xKS1U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4059" title="Investigating the Influence of Glycerol on the Utilization of Glucose in Yarrowia lipolytica Using RNA-Seq-Based Transcriptomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4087" title="rmtA-Dependent Transcriptome and Its Role in Secondary Metabolism, Environmental Stress, and Virulence in Aspergillus flavus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494998" id="node494998" data-pisa="ggg;9/12/4073" data-pisa-master="ggg;g3.119.400778" data-apath="/ggg/9/12/4073.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cyriel Paris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5141-0913" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bertrand Servin</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2499-5052" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simon Boitard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4073-4086;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494998" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cyriel Paris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyriel%2BParis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paris%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyriel%2BParis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bertrand Servin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bertrand%2BServin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Servin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABertrand%2BServin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5141-0913" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bertrand Servin</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Simon Boitard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BBoitard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boitard%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BBoitard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2499-5052" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simon Boitard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:simon.boitard@inra.fr" class="" data-icon-position="" data-hide-link-title="0">simon.boitard@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/4073" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494998" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494998/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4073.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494998" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494998/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4073.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494998" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494998/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/4073.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494998" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494998/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Detecting genomic regions under selection is an important objective of population genetics. Typical analyses for this goal are based on exploiting genetic diversity patterns in present time data but rapid advances in DNA sequencing have increased the availability of time series genomic data. A common approach to analyze such data is to model the temporal evolution of an allele frequency as a Markov chain. Based on this principle, several methods have been proposed to infer selection intensity. One of their differences lies in how they model the transition probabilities of the Markov chain. Using the Wright-Fisher model is a natural choice but its computational cost is prohibitive for large population sizes so approximations to this model based on parametric distributions have been proposed. Here, we compared the performance of some of these approximations with respect to their power to detect selection and their estimation of the selection coefficient. We developped a new generic Hidden Markov Model likelihood calculator and applied it on genetic time series simulated under various evolutionary scenarios. The Beta with spikes approximation, which combines discrete fixation probabilities with a continuous Beta distribution, was found to perform consistently better than the others. This distribution provides an almost perfect fit to the Wright-Fisher model in terms of selection inference, for a computational cost that does not increase with population size. We further evaluated this model for population sizes not accessible to the Wright-Fisher model and illustrated its performance on a dataset of two divergently selected chicken populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-time-series" class="hw-term hw-article-keyword hw-article-keyword-genetic-time-series" rel="nofollow">Genetic time series</a></li><li class="kwd"><a href="/keyword/hidden-markov-models" class="hw-term hw-article-keyword hw-article-keyword-hidden-markov-models" rel="nofollow">Hidden Markov Models</a></li><li class="kwd"><a href="/keyword/method-moments" class="hw-term hw-article-keyword hw-article-keyword-method-of-moments" rel="nofollow">method of moments</a></li><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li><li class="kwd"><a href="/keyword/wright-fisher" class="hw-term hw-article-keyword hw-article-keyword-wright-fisher" rel="nofollow">Wright-Fisher</a></li><li class="kwd"><a href="/keyword/beta-spikes" class="hw-term hw-article-keyword hw-article-keyword-beta-with-spikes" rel="nofollow">Beta with spikes</a></li></ul><p id="p-3">Detecting the molecular basis of adaptation in natural species is one of the major questions in population genetics. Most studies in this context rely on contemporary genomic data and exploit several typical signatures left by the spread of a benefical allele in a population: decreased heterozygosity, extended linkage disequilibrium (LD), increased differentiation with other populations, see <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Vitti <em>et al.</em> (2013)</a> for a review.</p><p id="p-4">Genomic time series provide an even more direct access to past allele frequency changes, allowing a more precise estimation on the onset or the intensity of selection. With the combined advances in sequencing technologies, ancient DNA processing and cryoconservation, this type of data are becoming accessible to many researchers and are found in various contexts. On a long time scale, time series of ancient DNA have been used for example to detect the genetic basis of adaptation in humans from Eurasia (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mathieson <em>et al.</em>, 2015</a>) or to infer allele frequency trajectories at several causal variants or quantitative trait loci in domestic horses (Fages Antoine <em>et al.</em>, 2019). On a shorter time scale, genetic time series are provided by the sequencing of samples conserved in biobanks during decades. This approach allowed for instance to assess the recent evolution of genetic diversity in Holstein cattle and to predict the selective value that could be obtained by using semen from ‘historical’ bulls in this breed (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Doekes <em>et al.</em>, 2018</a>). Controlled evolution experiments also provide a lot of interesting temporal genetic data and concern a large set of organisms. Such experiments generally imply intentional selection for a given trait of interest, <em>e.g.</em>, thermal tolereance in Drosophila (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tobler <em>et al.</em>, 2015</a>) or intra-muscular ultimate pH in chicken (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bihan-Duval <em>et al.</em>, 2018</a>).</p><p id="p-5">Following the increased production of genetic time series, several methods allowing researchers to exploit such data for selection inference have been developed in the two last decades (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Malaspinas 2015</a>). Most of them infer selection at a given variant based on temporal observations at this variant only, <em>i.e.</em>, without exploiting LD information (but see (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Terhorst <em>et al.</em>, 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">He <em>et al.</em>, 2019</a>)). In this context, several statistical approaches have been proposed to model the link between the full temporal trajectory of a population allele frequency, which is informative about selection intensity at the locus, and the (partial) genetic samples observed at some specific dates along this trajectory. These approaches include approximate Bayesian computation (ABC) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Foll <em>et al.</em>, 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Sackman <em>et al.</em>, 2019</a>), Bayesian path augmentation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Schraiber <em>et al.</em>, 2016</a>) or hidden Markov models (HMM) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bollback <em>et al.</em>, 2008</a>).</p><p id="p-6">The HMM approach arises quite naturally when analyzing genetic time series, as it accounts both for the Markovian property of allele frequency trajectories over time and for the fact that these trajectories are latent variables. Besides, efficient algorithms have been developed for likelihood evaluation and parameter estimation in such models (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rabiner 1989</a>). Following this approach, one important choice concerns the modeling of the underlying allele frequency trajectory at a locus. In the initial work of <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bollback <em>et al.</em> (2008)</a>, this process was modeled by the Wright-Fisher (WF) diffusion and the partial differential equation (PDE) resulting from this diffusion between each pair of consecutive sampling dates was solved using a numerical scheme. More recently, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Song and Steinrücken (2012)</a> derived an analytical solution of this PDE based on an infinite series expansion and integrated this solution into the HMM framework to detect selection on ancient DNA (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Steinrücken <em>et al.</em>, 2014</a>). The diffusion process can also be approximated using Markovian continuous time processes with a relatively low number of discrete spaces (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Malaspinas <em>et al.</em>, 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ferrer-Admetlla <em>et al.</em>, 2016</a>). All these approaches are quite computationally demanding, moreover the WF process converges to the diffusion only for weak selection (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ewens 2004</a>).</p><p id="p-7">Considering the WF model as a reference, an even more natural approach is to compute transition probabilities of the Markov chain using this model directly (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Iranmehr <em>et al.</em>, 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hubert <em>et al.</em>, 2018</a>). However, this implies that the dimension of the transition matrix is equal to the population size so this approach is in practice only possible for small populations. A more efficient strategy is to approximate the WF process by a parametric continuous distribution, whose first two moments fit those of the WF and whose transitions can be computed using a simple analytical formula. This moment-based approximation was implemented by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lacerda and Seoighe (2014)</a> and <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Terhorst <em>et al.</em> (2015)</a> using a Gaussian distribution and by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gompert (2016)</a> using a Beta distribution. However, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Tataru <em>et al.</em> (2017)</a> showed that these two distributions accurately approximate the WF only for very short time scale and weak selection and that a much closer fit can be obtained by incorporating fixation probabilities in the Beta distribtuion, leading to what they called the Beta with spikes distribution. They illustrated the performance of this distribution to infer branch lengths in a population tree, but did not consider the potential application for selection inference.</p><p id="p-8">Here we compared the performance of these different approximations for inferring a constant selection parameter in a population of constant size. This choice aims to establish a likelihood calculation framework that could be generalized to a more realistic scenario (<em>e.g.</em>, time varying selection and population size). To compare performances, we implemented a generic HMM that allows to estimate the selection intensity by maximum likelihood and to detect selected loci by a likelihood-ratio test. Transition probabilities in this model can be computed using either the WF model, the Gaussian approximation, the Nicholson Gaussian approximation, the Beta approximation or the Beta with spikes approximation. We evaluated the quality of inference provided by these different transitions in a large range of selection scenarios and showed an application of the HMM approach to a dataset concerning the experimental divergent selection of two chicken lines (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bihan-Duval <em>et al.</em>, 2018</a>).</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Statistical models for genetic time-series</h3><div id="sec-3" class="subsection"><h4>Hidden Markov Model framework:</h4><p id="p-9">Let <em>A</em> be a locus of interest accepting two alleles: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-1.gif"/></noscript></span></span> is called the (arbitrary) <em>reference allele</em> while <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-2.gif"/></noscript></span></span> is called the <em>alternative allele</em>. Considering a diploid organism, we assume each genotype is associated to a particular fitness: the homozygote reference allele fitness is 1, the heterozygote form fitness is <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-3.gif"/></noscript></span></span> and the homozygote alternative allele fitness is <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-4.gif"/></noscript></span></span>, where <em>s</em> is the selection parameter and <em>h</em> the dominance parameter. Assume a population has been sampled at <em>n</em> different dates <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-5.gif"/></noscript></span></span> (in generations after the first sampling date) in order to assess the evolution of the frequency of allele <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-6.gif"/></noscript></span></span>. We note <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-7.gif"/></noscript></span></span> the true frequency of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-8.gif"/></noscript></span></span> at date <em>t</em>. At date <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-9.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-10.gif"/></noscript></span></span>), we note <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-11.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-12.gif"/></noscript></span></span> the total number of alleles sampled and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-13.gif"/></noscript></span></span> the number of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-14.gif"/></noscript></span></span> alleles sampled. Finally, we note <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-15.gif"/></noscript></span></span> the haploid effective size of the population.</p><p id="p-10">In this study we will follow a common approach to model such data, first proposed by <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bollback <em>et al.</em> (2008)</a> and use a Hidden Markov Model (HMM, see <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cappé <em>et al.</em> (2005)</a> for a general description). This HMM (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) is a bivariate Markov process <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-16.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-18.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-19.gif"/></noscript></span></span>) is a Markov process with <em>transition kernel</em> <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-20.gif"/></noscript></span></span> between <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-22.gif"/></noscript></span></span>. Given <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-23.gif"/></noscript></span></span>, observations are independant and the <em>emission kernel</em> of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-24.gif"/></noscript></span></span> given <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-25.gif"/></noscript></span></span> is noted <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-26.gif"/></noscript></span></span>. The distribution of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-27.gif"/></noscript></span></span> is noted <em>ν</em>. This model is fully specified by setting <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-29.gif"/></noscript></span></span>, and each of them can be thought of as a probability:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-30.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-31.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-32.gif"/></noscript></span></span>In this study we assume that the distribution of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-33.gif"/></noscript></span></span> is uniform on <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-34.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-35.gif"/></noscript></span></span>) (but see Discussion). As the observations <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-36.gif"/></noscript></span></span> are sampled from a population with allele frequency <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-37.gif"/></noscript></span></span>, the distribution of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-38.gif"/></noscript></span></span> given <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-39.gif"/></noscript></span></span> is a binomial distribution with parameters <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-41.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-42.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-43.gif"/></noscript></span></span>For a given <em>x</em>, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-44.gif"/></noscript></span></span> is the distribution of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-45.gif"/></noscript></span></span> given that the previous frequency <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-46.gif"/></noscript></span></span> is <em>x</em>. Note that the Markov chain <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-47.gif"/></noscript></span></span> is typically not homogeneous as for example the duration between successive sampling dates may vary. Apart from the time between samples, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-48.gif"/></noscript></span></span> also depends on the transition model and on parameters <em>s</em>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-49.gif"/></noscript></span></span> and <em>h</em>. One of the main objective of this study is to compare different possible transition models which are described below.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of the Hidden Markov Model for genetic time series." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Graphical representation of the Hidden Markov Model for genetic time series.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F1.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/4073/F1.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495079" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Graphical representation of the Hidden Markov Model for genetic time series.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">In this HMM framework, the likelihood is:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-50.gif"/></noscript></span></span>A direct calculation of this likelihood is very costly because the integral is over <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-51.gif"/></noscript></span></span>. The usual way to tackle this problem is to introduce the forward measure <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-52.gif"/></noscript></span></span>, which is the joint probability of the <em>k</em> first observations and the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-53.gif"/></noscript></span></span> state:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-54.gif"/></noscript></span></span>This forward measure is computed using the following recursion (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cappé <em>et al.</em>, 2005</a>), generally referred as the forward algorithm:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-55.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>The likelihood is obtained by integrating the last forward measure <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-56.gif"/></noscript></span></span> over all possible states <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-57.gif"/></noscript></span></span>, using the following relation:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-58.gif"/></noscript></span></span>Using this approach, the likelihood calculation involves only <em>n</em> integrals over <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-59.gif"/></noscript></span></span> instead of one integral over <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-60.gif"/></noscript></span></span>. For a later use, let’s also define the log likelihood:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-61.gif"/></noscript></span></span>This likelihood forms the basis of statistical inference on the population parameters <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-62.gif"/></noscript></span></span>, <em>s</em> and <em>h</em>. In this study, we focus on the inference on the selection parameter <em>s</em> and will therefore assume <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-63.gif"/></noscript></span></span> and <em>h</em> to be known. We estimate the selection parameter <em>s</em> by maximum likelihood:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-64.gif"/></noscript></span></span>To build a test for the null hypothesis (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-65.gif"/></noscript></span></span>), we use the log likelihood ratio statistic:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-66.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Under the null, λ asymptotically follows a <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-67.gif"/></noscript></span></span> with one degree of freedom, a property that allows to derive p-values which are useful quantities in particular in a multiple testing context.</p><p id="p-13">Having described the general framework used in this study, we now turn to its implementation that requires specifying the transition kernel <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-68.gif"/></noscript></span></span>. One objective of this work was to investigate different models for this transition kernel. A reference model in this context is the one-locus Wright-Fisher (WF) model under diploid selection pressure (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Ewens 2004</a>). However, this model has computational limitations and, in the HMM famework, must generally be approximated. We first present the WF model and then several continuous approximations of this model based on a common approach: the method of moments.</p></div><div id="sec-4" class="subsection"><h4>Wright-Fisher transition model:</h4><p id="p-14">The WF model is a discrete time model that assumes random mating and non overlapping generations. Let <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-69.gif"/></noscript></span></span> be the haploid (constant) effective population size. Under this model, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-70.gif"/></noscript></span></span> can take a discrete number of values in <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-71.gif"/></noscript></span></span> and the transition kernel of this process is a transition matrix with size <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-72.gif"/></noscript></span></span>. Let P designate the transition matrix of this process in one generation. Then, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-73.gif"/></noscript></span></span> and<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-74.gif"/></noscript></span></span>where <em>f</em> is the diploid fitness function (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Ewens 2004</a>):<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Note that this function can easily incorporate the possibility of mutations (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Ewens 2004</a>) but it is not considered here for the sake of simplicity.</p><p id="p-15">Given this one step transition matrix, the transition matrix of the HMM is obtained by taking P to the required power (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-76.gif"/></noscript></span></span>):<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-77.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>In the WF model, the effective population size <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-78.gif"/></noscript></span></span> is a critical parameter affecting computational properties. More precisely, the transition matrix memory occupation is <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-79.gif"/></noscript></span></span> and the forward algorithm computation time is at least <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-80.gif"/></noscript></span></span>. Moreover, the transition matrix <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-81.gif"/></noscript></span></span> is obtained by taking a <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-82.gif"/></noscript></span></span> matrix to the power <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-83.gif"/></noscript></span></span>, with a complexity in <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-84.gif"/></noscript></span></span>. It can thus be very costly for large <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-85.gif"/></noscript></span></span> and large intersample time. To overcome this issue, one approach is to approximate the Wright-Fisher process with a continuous space process such that integral calculations of the forward algorithm are made over a discretization of <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-86.gif"/></noscript></span></span>, with a grid size (<em>i.e.</em>, a number of discretization points) much smaller than <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-87.gif"/></noscript></span></span>. We followed this approach here and approximated the WF process using the method of moments.</p></div><div id="sec-5" class="subsection"><h4>Parametric approximations based on the method of moments:</h4><p id="p-16">The principle of the method of moments consists in modeling the transition kernel <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-88.gif"/></noscript></span></span> with a parametric distribution, matching its mean and variance to those of the WF model. The advantages of using parameteric distributions are (i) to be able to compute transitions faster than raising the transition matrix to a power equal to the number of generations elapsed, (ii) to use smaller transition matrix in the forward algorithm. This approach was first proposed by <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lacerda and Seoighe (2014)</a> with a Gaussian distribution and was then used in several other studies (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Terhorst <em>et al.</em>, 2015</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Tataru <em>et al.</em> 2017</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gompert 2016</a>). One prerequisite of the method of moments is to know the moments of a WF model for any value of <em>s</em>, <em>h</em> and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-89.gif"/></noscript></span></span>. Under selection (<span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-90.gif"/></noscript></span></span>) no analytical formula of these moments exists but they can be approximated using a recursion inspired by (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Lacerda and Seoighe 2014</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Terhorst <em>et al.</em>, 2015</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Tataru <em>et al.</em> 2017</a>). In the WF model, moments at generation <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-91.gif"/></noscript></span></span> can be derived from those at generation <em>t</em>:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-92.gif"/></noscript></span></span>For models with selection, the fitness function <em>f</em> (<a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equation (3</a>) is non-affine so this recursion cannot be solved analytically. However, using a Taylor expansion of <em>f</em> around <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-93.gif"/></noscript></span></span>, it is possible to build a recursion allowing the approximation of <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-94.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-95.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-96.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-97.gif"/></noscript></span></span> be these approximations at generation <em>t</em>, they can be computed as follows (see Supplemental File S1 for details):<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-98.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Note that as f is a rational fraction, its derivatives can be calculated exactly. To simplify notations in the following, we further define <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-99.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-100.gif"/></noscript></span></span>.</p><p id="p-17">Having established approximations to the Wright-Fisher moments in the general case, different parametric distributions can be chosen to approximate the transition kernel. We show on Figure S1 examples of approximations using models presented below.</p><div id="sec-6" class="subsection"><h5>Gaussian Model (Ga)</h5><p id="p-18">The first possibility, considered in the original description of the method of moments approach (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Lacerda and Seoighe 2014</a>), is to use the Gaussian distribution (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cavalli-Sforza <em>et al.</em>, 1964</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nicholson <em>et al.</em>, 2002</a>). Indeed, when <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-102.gif"/></noscript></span></span> are small, the rescaled Wright-Fisher process converges to a Gaussian process and the HMM transition kernel is:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-103.gif"/></noscript></span></span>One of the main issues with the Gaussian model is that <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-104.gif"/></noscript></span></span> can take values outside <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-105.gif"/></noscript></span></span>. To tackle this issue we chose to reject values distributed outside <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-106.gif"/></noscript></span></span> and appropriately rescale the Gaussian density so that it integrates to one.</p></div><div id="sec-7" class="subsection"><h5>Nicholson Gaussian Model (NG)</h5><p id="p-19">Another option to tackle the Gaussian distribution support issue, is to allocate the mass of the distribution outside <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-107.gif"/></noscript></span></span> to discrete fixation probabilities at 0 (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-108.gif"/></noscript></span></span>) and 1 (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-109.gif"/></noscript></span></span>) (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Nicholson <em>et al.</em>, 2002</a>). Hence, the resulting distribution is a mixture of a Gaussian continuous density on <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-110.gif"/></noscript></span></span> and a discrete distribution at 0 and 1.</p></div><div id="sec-8" class="subsection"><h5>Beta Model (Be)</h5><p id="p-20">A second possible parametric distribution is the Beta, already used in several contexts (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Balding and Nichols 1995</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gompert 2016</a>) including the HMM based analysis of genomic time series (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hui and Burt 2015</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Tataru <em>et al.</em> 2017</a>). Use of this distribution can be motivated by the fact that it takes values on <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-111.gif"/></noscript></span></span> and because it is the stationary distribution of the diffusion approximation under neutrality (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kimura 1964</a>). In addition, the family of Beta distributions encompasses a lot of different shapes that can fit many behaviors for allele-frequency variation. In this model, the HMM transition kernel can be written as:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-112.gif"/></noscript></span></span>with:</p><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-113.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span></div><div id="sec-9" class="subsection"><h5>Beta with spikes Model (BwS)</h5><p id="p-21">We finally considered the Beta with spikes model described by <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Tataru <em>et al.</em> (2017)</a>. Similarly to the Nicholson Gaussian model described above, this model explicitly accounts for the possibility of allele fixation: it is a mixture of a Beta distribution and two discrete fixation probabilities at 0 and 1. According to <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Tataru <em>et al.</em> (2017)</a>, this model is a better approximation of the Wright-Fisher than the Ga and Be models.Let <span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-114.gif"/></noscript></span></span> and <span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-115.gif"/></noscript></span></span>The transition kernel is:<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-116.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-117.gif"/></noscript></span></span> is a Beta distribution describing the transition from <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-118.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-119.gif"/></noscript></span></span> conditional on the fact that <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-120.gif"/></noscript></span></span> did not fix in 0 or 1. Parameters <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-121.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-122.gif"/></noscript></span></span> are obtained with a formula similar to (6), replacing abolute moments <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-123.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-124.gif"/></noscript></span></span> by conditional mean and variance <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-125.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-126.gif"/></noscript></span></span> given that the WF process did not fixed at 0 or 1:<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-127.gif"/></noscript></span></span>In the Beta with spikes model, fixation probabilities must be computed in addition to WF moments. Let us define <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-128.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-129.gif"/></noscript></span></span>. We used the following recursion, proposed by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tataru <em>et al.</em> (2015)</a> assuming Beta with spikes transitions:<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-130.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-131.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-132.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-133.gif"/></noscript></span></span>where <em>B</em> denotes the Beta function.</p></div><div id="sec-10" class="subsection"><h5>Implementation</h5><p id="p-22">In this study, the likelihood was always evaluated on a grid of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-134.gif"/></noscript></span></span> composed of 600 values ranging from <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-135.gif"/></noscript></span></span> to 300. For WF transitions the integrals involved in the likelihood computation using the forward algorithm (1) are finite sums. However for the parametric approximations to the WF these integrals are continuous and must therefore be numerically approximated. Here, this was done using the trapezoidal rule on an integration grid composed of 129 points evenly spaced on <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-136.gif"/></noscript></span></span>. Note that the calculation of these transitions is only required once per dataset.</p></div></div></div><div id="sec-11" class="subsection"><h3>Comparison of HMM transition models</h3><p id="p-23">We studied the quality of the transition approximation to the WF process. For this objective, we set the effective population size to <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-137.gif"/></noscript></span></span> haploids to render the comparisons computationally tractable. We compared the Ga, NG, Be and BwS models as approximations to the Wright-Fisher process, first by evaluating their fit to the WF distribution and second by measuring their impact on selection inference.</p><div id="sec-12" class="subsection"><h4>Approximation of the WF transition:</h4><p id="p-24">We first studied the quality of approximations of the WF transition kernel. For all parametric distributions, at least part of the WF discrete transition is approximated by a continuous density. Thus, measuring the fit of these approximations requires comparing continuous densities to discrete probabilities. In such a case, classical distances aimed at comparing densities, such as the Kullback-Liebler divergence or the Hellinger distance are not valid. Therefore we used the <em>Wasserstein distance</em> defined as the <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-138.gif"/></noscript></span></span> distance between the cumulative distribution functions of two distributions and which is well defined in all cases considered here. The fit of the Ga, NG, Be and BwS models were evaluated over the parameter set <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-139.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-140.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-141.gif"/></noscript></span></span>, and starting allele frequency covering all <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-142.gif"/></noscript></span></span> values for <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-143.gif"/></noscript></span></span></p></div><div id="sec-13" class="subsection"><h4>Consequences for statistical inference:</h4><p id="p-25">We computed the likelihood profile (see implementation paragraph) for genomic time series randomly simulated with the Wright-Fisher process. We set the initial allele frequency to 0.1 or 0.5 and its selection parameter varying from 0 (neutral case) to 1 (strong selection). We simulated a random sampling of <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-144.gif"/></noscript></span></span> alleles at 10 evenly spaced dates over a total range of <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-145.gif"/></noscript></span></span> generations (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-146.gif"/></noscript></span></span>). We ran 100000 simulations for <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-147.gif"/></noscript></span></span> and 10000 for other <em>s</em> (see Figure S2). To compare the Ga, NG, Be and BwS models in terms of selection inference, we focused on four summarizing quantities computed from the likelihood: the log-likelihood value at <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-148.gif"/></noscript></span></span>, the maximum likelihood estimator <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-149.gif"/></noscript></span></span>, the log-likelihood value at <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-150.gif"/></noscript></span></span>, and the log-likelihood ratio statistic (2).</p></div><div id="sec-14" class="subsection"><h4>Inferring selection using the HMM:</h4><p id="p-26">As the BwS model was found to be the best approximation to the Wright-Fisher (see Results), we further studied its statistical properties on simulations for higher <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-151.gif"/></noscript></span></span> and applied it to detect selection on a real data set. The NG model, which provided the second best fit to the Wright-Fisher, was also considered for comparison when analyzing simulated data.</p></div><div id="sec-15" class="subsection"><h4>Simulation study:</h4><p id="p-27">To study the statistical properties of NG and BwS models, we created new datasets designed to represent consistent selection dynamics as for the <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-152.gif"/></noscript></span></span> dataset but with larger <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-153.gif"/></noscript></span></span>. To do so, we considered that the diffusion approximation to the WF suggests that selection dynamics are driven by the two compound parameters <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-154.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-155.gif"/></noscript></span></span>. Therefore the new datasets were created by increasing <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-156.gif"/></noscript></span></span> to 1,000 and 10,000 but in each case adjusted <em>T</em> and <em>s</em> so as to keep the same set of <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-157.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-158.gif"/></noscript></span></span> values as for the <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-159.gif"/></noscript></span></span> dataset. Specifically we considered <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-160.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-161.gif"/></noscript></span></span>. We also kept the same sampling pattern of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-162.gif"/></noscript></span></span> haploids sampled at 10 evenly spaced dates over the total range <em>T</em> and an initial allele frequency of 0.1 or 0.5.</p></div><div id="sec-16" class="subsection"><h4>Real data analysis:</h4><p id="p-28">We applied our method using the BwS model on a dataset of two experimental populations of chicken divergently selected for the intra muscular ultimate pH (pHu) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alnahhas <em>et al.</em>, 2016</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bihan-Duval <em>et al.</em>, 2018</a>). This selection experiment was conducted for 5 generations and at each generation (including the starting population) samples were collected and genotyped for 40,199 markers. Sample sizes in each line (pHu- and pHu+) are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494913/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494913/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Haploid sample sizes across 6 generations of chicken populations selected for intra-muscular ultimate pH</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">Our current implementation of the method does not estimate the <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-163.gif"/></noscript></span></span> parameter which is not known in this real dataset. To remedy this issue, we estimated <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-164.gif"/></noscript></span></span> in each line using the NB R package (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hui and Burt 2015</a>). Motivation for using this method to estimate <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-165.gif"/></noscript></span></span> comes from the fact that the underlying model is a HMM with Beta transitions, similar to our general framework. To estimate <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-166.gif"/></noscript></span></span>, we removed trajectories starting near edges (frequencies below <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-167.gif"/></noscript></span></span> or above <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-168.gif"/></noscript></span></span>), running the analysis over 27,669 loci. For our analysis we considered the two lines as two different populations.</p><p id="p-30">As a further validation of the Beta with spikes approximation, we applied the HHH both with Wright-Fisher and Beta with spikes transitions. We removed from the analysis all trajectories fixing in one step as we used the <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-169.gif"/></noscript></span></span> distribution to compute p-values. SNP were called significant with a FDR threshold of <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-170.gif"/></noscript></span></span> estimated using the q-value R package (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Storey <em>et al.</em>, 2015</a>).</p></div></div><div id="sec-17" class="subsection"><h3>Data availability</h3><p id="p-31">The implementation of the HMM is freely available at <a href="https://github.com/CyrielParis/compareHMM">https://github.com/CyrielParis/compareHMM</a>. Observed allele frequencies in the two chicken lines, for all markers and generations, are available at: <a href="http://doi.org/10.5281/zenodo.3461580">http://doi.org/10.5281/zenodo.3461580</a>. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9959492">https://doi.org/10.25387/g3.9959492</a>.</p></div></div><div class="section results" id="sec-18"><h2 class="">Results</h2><p id="p-32">We have established a general HMM framework that allows to perform statistical inference from genetic time-series data. Within this framework, different transition models can be used and easily compared. In this section we will first present results comparing the quality of approximations of the Gaussian (Ga), Nicholson Gaussian (NG), Beta (Be) and Beta with spikes (BwS) transition kernels on their ability to approximate the Wright-Fisher (WF) kernel. Next, we study in more details the statistical performances of the BwS for detecting selection and estimating its intensity. Finally, we illustrate how the BwS model can be used on real data by re-analyzing an experimental evolution dataset.</p><div id="sec-19" class="subsection"><h3>Approximation of the Wright-Fisher model</h3><div id="sec-20" class="subsection"><h4>Approximation of the WF transition:</h4><p id="p-33">We first compared each continuous approximation with the WF using the Wasserstein distance. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows for each model and for different selection intensities the heatmap of the Wasserstein distance between the continuous approximation and the true WF distribution. In principle this distance ranges from 0 which is a perfect match to 1 which is, for an allele frequency, the worst possible distance. For this set of parameters, the Wasserstein distance ranged from 0 to <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-171.gif"/></noscript></span></span>. According to this distance, all approximations are valid for intermediate starting allele frequencies and a small number of generations but the accuracy of all approximations decreases as the number of generations increases. The accuracy of the approximation also worsens when the selection parameter increases, especially for small starting allele frequencies. Indeed, the figure shows that for starting frequency <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-172.gif"/></noscript></span></span>, generations <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-173.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-174.gif"/></noscript></span></span> all models performed badly in approximating the Wright-Fisher model. As the forward algorithm involves an integration over all possible allele frequencies at each sampling date, poor approximations of transitions from small allele frequencies may in principle affect the performance of HMM inference for any observed trajectory (with or without small frequencies). We evaluate this in the next setion.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Wasserstein distances to Wright-Fisher transition for four continuous approximations (columns) and for varying starting allele frequencies (x axis of each panel) and number of generations (y axis of each panel). Top panels: Neutral evolution (). Middle panels: Mild selection (). Bottom panels: Strong selection ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Wasserstein distances to Wright-Fisher transition for four continuous approximations (columns) and for varying starting allele frequencies (x axis of each panel) and number of generations (y axis of each panel). Top panels: Neutral evolution (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-175&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Middle panels: Mild selection (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-176&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Bottom panels: Strong selection (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-177&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F2.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/4073/F2.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495118" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Wasserstein distances to Wright-Fisher transition for four continuous approximations (columns) and for varying starting allele frequencies (x axis of each panel) and number of generations (y axis of each panel). Top panels: Neutral evolution (<span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F2/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F2/embed/mml-math-175.gif"/></noscript></span></span>). Middle panels: Mild selection (<span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F2/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F2/embed/mml-math-176.gif"/></noscript></span></span>). Bottom panels: Strong selection (<span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F2/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F2/embed/mml-math-177.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Concerning the comparison between models, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> shows that the Be and BwS models are better approximations than the Ga and NG models, the BwS model being overall slightly better than others. These results are consistent with those obtained by <a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Tataru <em>et al.</em> (2017)</a>, although they used a different approximation for the moments and a different measure of the distance between distributions.</p><p id="p-36">The error in the approximation can come from the model itself but also from the use of approximated moments. To evaluate what proportion of the distance to the true distribution comes from the moment approximation, we computed the Wasserstein distance of each continuous distribution using the true WF moments (<a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">Equation 4</a>, Figure S3). <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> presents this proportion for the BwS model under neutrality (left <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-178.gif"/></noscript></span></span>) and under selection (middle <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-179.gif"/></noscript></span></span> and right, <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-180.gif"/></noscript></span></span>). Under neutrality the error comes mainly from the continuous approximation. This is also usually the case under selection although for small starting frequencies (<span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-181.gif"/></noscript></span></span>) and large duration between samples the error in the approximation of moments is the most influential. As getting the true moments of the Wright-Fisher process can be costly especially for large population size, all results presented further are based on approximated moments.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of the Wasserstein distance of the BwS due to the approximation of moments. Negative values correspond to very rare cases where the transition with approximated moments is better than with true moments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of the Wasserstein distance of the BwS due to the approximation of moments. Negative values correspond to very rare cases where the transition with approximated moments is better than with true moments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F3.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/4073/F3.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494974" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">Proportion of the Wasserstein distance of the BwS due to the approximation of moments. Negative values correspond to very rare cases where the transition with approximated moments is better than with true moments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h4>Approximation of the WF likelihood:</h4><p id="p-38">Next, we evaluated the ability of continuous WF approximations to recover the WF likelihood of genetic time-series data. To do so, we simulated trajectories under different evolutionary scenarios (see Methods) and compared the inference obtained with each continuous model to the one obtained using the Wright-Fisher model.</p><p id="p-39">Simulations of time-series genetic data to evaluate the quality of parametric approximations to the Wright-Fisher were performed under this reference model. Hence, using the HMM with WF transitions should provide the best statistical behavior and be considered a reference inference method. Before assessing the quality of approximations to the WF model with respect to their accuracy for computing the WF-based likelihood, it is necessary to ensure that this model itself provides sensible likelihood inference. Results on simulated data showed that this was not always the case: a family of trajectories were found to lead to unreliable results even when using the WF model for inference. These trajectories correspond to cases where the observed allele frequency reached fixation in one time step (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-182.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-183.gif"/></noscript></span></span>). In such cases the likelihood function is monotonic in <em>s</em>: the <em>s</em> value that maximizes the probability of fixation at 1 in one time step is <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-184.gif"/></noscript></span></span> and similarly the <em>s</em> value that maximizes the probability of fixation at <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-185.gif"/></noscript></span></span> in one time step is <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-186.gif"/></noscript></span></span>, essentially corresponding to an infinite <em>s</em> for the other allele. In these cases, likelihood inference is not valid. They could be deemed merely pathological but they occur with a non negligible probability that is increasing as the intersample duration or the selection intensity increases. In the following, we excluded such cases when evaluating the statistical properties of inference models. However we studied their occurrence frequency (see below).</p><p id="p-40"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> presents four different statistics based on the likelihood of simulated data under various scenarios (see Methods), comparing results obtained for each parametric approximation to those obtained using the WF: <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4a</a> presents results on the likelihood at <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-187.gif"/></noscript></span></span>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4b</a> the maximum likelihood estimates (MLE), <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4c</a> the likelihood at the MLE (which can be different from the WF one) and <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4d</a> the likelihood ratio statistic. For all these statistics and independently of the scenario simulated, the BwS and the WF provided essentially the same inference. Only for the MLE there are few cases where the two transition model disagreed (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4b</a>). However, this is never true for the LMLE (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4c</a>) so when the two models disagree for <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-188.gif"/></noscript></span></span> they still give similar likelihood values at the MLE. Overall this shows that the few large differences in MLE between BwS and WF most likely correspond to cases where the data are not very informative on the selection intensity. On the contrary, there are many scenarios where the Be and Ga models have very different values than the WF for all statistics considered. Finally, the NG model showed significantly better properties than the Be and Ga, but it provided a less accurate fit to the WF than the BwS model, for all parameters studied.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of four continuous approximations to the WF transition for likelihood-based statistics. In each subplot, Be, Ga, BwS, NG indicate the compared model among Beta model, Gaussian model, Beta with spikes model and Nicholson Gaussian model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of four continuous approximations to the WF transition for likelihood-based statistics. In each subplot, Be, Ga, BwS, NG indicate the compared model among Beta model, Gaussian model, Beta with spikes model and Nicholson Gaussian model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F4.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/12/4073/F4.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495094" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Comparison of four continuous approximations to the WF transition for likelihood-based statistics. In each subplot, Be, Ga, BwS, NG indicate the compared model among Beta model, Gaussian model, Beta with spikes model and Nicholson Gaussian model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">One reason why BwS and NG models are better approximations to the WF than the Beta and the Gaussian is that they include fixation probabilities. In the HMM framework, fixation events are considered by these models while others fail to correctly interprete vanishing observed allele frequencies. Indeed, when fixation events are unlikely, all models approximate efficiently the Wright-Fisher and the likelihood approximation is good. Fixation events depend mainly on the total duration of sampling <em>T</em> (see Figures S4-S7). For large values (<span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-189.gif"/></noscript></span></span>) early fixation events (apart from the pathological cases mentionned above) are not interpreted by the Beta and Gaussian models as fixations. These models will consider that the constant trajectory subsequent to the fixation event is consistent with a neutral process. The resulting maximum likelihood estimator therefore underestimates the true <em>s</em> (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4b</a>). As the maximum is not well found by Beta and Gaussian models, the likelihood-ratio statistic is inconsistent and is not calibrated as a <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-190.gif"/></noscript></span></span> distribution (Figure S8).</p></div></div><div id="sec-22" class="subsection"><h3>Inferring selection using the Beta with spikes model</h3><p id="p-43">Given that the Beta with spikes model closely mimics the Wright-Fisher model in the HMM framework, we studied the statistical properties of the HMM with BwS transitions for population sizes of 1,000 and 10,000 haploids, which are computationally prohibitive when using the WF transitions. In terms of statistical properties, we studied the ability to detect selective loci and the accuracy in estimating the intensity of selection. The same analysis was conducted using the NG model and comparable results are provided in Supplementary Figures S9-S11.</p><div id="sec-23" class="subsection"><h4>Detection of selection:</h4><p id="p-44">The HMM framework developed here allows to derive a likelihood-ratio statistic that can be used to test for the null hypothesis <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-191.gif"/></noscript></span></span>. Asymptotically, this statistic follows a <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-192.gif"/></noscript></span></span> distribution. However, the asymptotics in the HMM framework are measured in terms of the number of dates sampled. In our simulations, and most likely in typical real datasets, this number is not very high (10 here). Our first goal was thus to evaluate whether the <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-193.gif"/></noscript></span></span> was a good fit to statistics computed under the null hypothesis. Then we evaluated its power under different alternatives.</p></div><div id="sec-24" class="subsection"><h4>Calibration under the null:</h4><p id="p-45">We checked for the good calibration of the BwS under neutrality. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows on the y axis the quantiles of the empirical likelihood ratio distribution and on the x axis, the corresponding quantiles of a <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-194.gif"/></noscript></span></span> distribution. Generally the fit of the LR to the theoretical distribution was good although the test would be slightly conservative for large <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-195.gif"/></noscript></span></span>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Calibration of  under the null: comparison of empirical vs. theoretical  quantiles for different population sizes, initial allele frequencies and inter-sample times." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Calibration of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-196&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under the null: comparison of empirical &lt;em&gt;vs.&lt;/em&gt; theoretical &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-197&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; quantiles for different population sizes, initial allele frequencies and inter-sample times.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F5.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/12/4073/F5.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494956" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Calibration of <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F5/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F5/embed/mml-math-196.gif"/></noscript></span></span> under the null: comparison of empirical <em>vs.</em> theoretical <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F5/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F5/embed/mml-math-197.gif"/></noscript></span></span> quantiles for different population sizes, initial allele frequencies and inter-sample times.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h4>Power:</h4><p id="p-47">With the LR statistic and the <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-198.gif"/></noscript></span></span> quantiles, the empirical power of the test can be computed for different scenarios. As in this case we know that all true <em>s</em> parameters are non negative, we considered that all negative <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-199.gif"/></noscript></span></span> having a high LR should not be considered as a success in detecting selection. Indeed, even if the LR is high in this case, the final conclusion of such results would be that the wrong allele is under selection which is even worse than not detecting selection.</p><p id="p-48"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the power of the test as a function of <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-200.gif"/></noscript></span></span> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6a</a>) and as a function of <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-201.gif"/></noscript></span></span> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6b</a>), for two different initial allele frequencies (0.1 and 0.5). As expected, the power increased with <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-202.gif"/></noscript></span></span> (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6a</a>). We can also see that for larger <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-203.gif"/></noscript></span></span>, the power starts to increase for smaller <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-204.gif"/></noscript></span></span>. This can be explained by the fact that in such scenarios, selection has more time to affect the allele frequency and even small <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-205.gif"/></noscript></span></span> give rise to trajectories that are consistent with selection. However, when starting at an initial frequency <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-206.gif"/></noscript></span></span> the power decreases when <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-207.gif"/></noscript></span></span> is too high. In these cases, the probability to observe large variations of allele frequency under neutrality becomes larger and so the ability of the LRT to distinguish neutrality from selection decreases.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power of the likelihood ratio test under different simulated scenarios." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Power of the likelihood ratio test under different simulated scenarios.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F6.medium.gif" width="440" height="129"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/12/4073/F6.medium.gif" width="440" height="129"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494924" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-49" class="first-child">Power of the likelihood ratio test under different simulated scenarios.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h4>Parameter estimation:</h4><p id="p-50">Figure S12 shows that under the parameters considered, <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-208.gif"/></noscript></span></span> has no influence on estimation, except for large <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-209.gif"/></noscript></span></span>, for which the maximum likelihood estimator has a higher variance for small <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-210.gif"/></noscript></span></span>. In this case, the diffusion approximation is not valid so different <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-211.gif"/></noscript></span></span> are not equivalent for given <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-212.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-213.gif"/></noscript></span></span>.</p><p id="p-51"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows the MLE distribution for different <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-214.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-215.gif"/></noscript></span></span> pooling results from different <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-216.gif"/></noscript></span></span>. It shows that the estimation is more accurate (with lower variance) for large values of <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-217.gif"/></noscript></span></span>. Moreover, when fixation events (<span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-218.gif"/></noscript></span></span>) are unlikely, the estimation is unbiased. On the other hand, when fixation events are likely (<span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-219.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-220.gif"/></noscript></span></span>), the trajectories that are not filtered out are those that are consistent with lower values than the true <em>s</em> and therefore the MLE underestimates the true <em>s</em>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimation error distribution in different scenarios for . Each column stands for a scaled time parameter . The first line indicates the absolute error distribution in each scenario and the second line represents in each case, the proportion of rejected trajectories (red lines for fixations in 1 and blue lines for fixations in 0)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimation error distribution in different scenarios for &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-221&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Each column stands for a scaled time parameter &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-222&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The first line indicates the absolute error distribution in each scenario and the second line represents in each case, the proportion of rejected trajectories (red lines for fixations in 1 and blue lines for fixations in 0).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F7.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/12/4073/F7.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494971" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-52" class="first-child">Estimation error distribution in different scenarios for <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F7/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F7/embed/mml-math-221.gif"/></noscript></span></span>. Each column stands for a scaled time parameter <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F7/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F7/embed/mml-math-222.gif"/></noscript></span></span>. The first line indicates the absolute error distribution in each scenario and the second line represents in each case, the proportion of rejected trajectories (red lines for fixations in 1 and blue lines for fixations in 0).</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-27" class="subsection"><h3>Analysis of a selection experiment</h3><p id="p-53">To illustrate the statistical performance of the HMM approach on a real data set, we applied it to allele frequency trajectories observed at 40,199 SNPs in two lines of chicken divergently selected for the intra muscular ultimate pH (pHu). This dataset was previoulsy analyzed by <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Bihan-Duval <em>et al.</em> (2018)</a> using the FLK <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bonhomme <em>et al.</em> (2010)</a> and hapFLK <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fariello <em>et al.</em> (2013)</a> tests: considering each generation of the experiment independently of the others, these authors looked for regions showing a significant excess of allele (for FLK) or haplotype (for hapFLK) frequency differentiation between pHu+, pHu- and the founding population. Thus, this case study offers the opportunity to compare the regions detected under selection by two different approaches: genetic differentiation and time series.</p><p id="p-54">We first estimated population size in the two lines using the NB R package (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Hui and Burt 2015</a>) and found <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-223.gif"/></noscript></span></span> haploids in pHu+ and <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-224.gif"/></noscript></span></span> haploids in pHu-. Based on these values, we applied the HMM in each line and detected a total of 39 significant SNPs under selection at a FDR of 5% (Table S1). We also conducted the analysis using the Wright-Fisher model and obtained exactly the same results (Figure S13). These SNPs were clustered in 12 genomic regions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, Figure S14: eight of them were detected only in the phu+ line, one was detected only in the phu- line and three others were detected in both lines. Moreover, for almost all significant SNPs, the allele that was selected in one line (positive value of <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-225.gif"/></noscript></span></span>) was also counter selected in the other line (negative value of <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-226.gif"/></noscript></span></span>) (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). This is consistent with the experimental design of divergent selection between the two lines.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495047/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495047/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Significant regions (for HMM)</span><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of  in pHu- vs. pHu+. Levels of gray represent the density of , significant SNP are represented in colors, the size representing the  associated p-value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485654301" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Distribution of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-227&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in pHu- &lt;em&gt;vs.&lt;/em&gt; pHu+. Levels of gray represent the density of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-228&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, significant SNP are represented in colors, the size representing the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-229&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; associated p-value.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4073/F8.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/12/4073/F8.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4073/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495015" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-55" class="first-child">Distribution of <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F8/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F8/embed/mml-math-227.gif"/></noscript></span></span> in pHu- <em>vs.</em> pHu+. Levels of gray represent the density of <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F8/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F8/embed/mml-math-228.gif"/></noscript></span></span>, significant SNP are represented in colors, the size representing the <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F8/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/F8/embed/mml-math-229.gif"/></noscript></span></span> associated p-value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">Among the 12 regions detected by the time series approach, six were detected by FLK or (and) hapFLK and six were new. SNPs in these six new regions were associated to an allele frequency significantly increasing or decreasing in one line but rather constant in the other line (Figure S15): in such cases, genetic differentiation was elevated but did not result in a significant FLK or hapFLK p-value. In contrast, SNPs detected only with FLK were generally associated to a moderate allele frequency change in both lines, that was not found significant when analyzing each of them independently (Figure S16). Overall, the differentiation and time series approaches were thus found complementary. Note also that the number of regions detected (<em>i.e.</em>, the detection power) with the time series approach was very similar to that obtained by FLK. Detection power was significantly higher with hapFLK, which was related to the use of haplotype information.</p></div></div><div class="section discussion" id="sec-28"><h2 class="">Discussion</h2><p id="p-57">In this study we have described a general statistical framework for the analysis of genetic time series. It is built upon combining a Hidden Markov Modeling approach, first proposed by <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bollback <em>et al.</em> (2008)</a> in the context of genetic time series and parametric approximations to the Wright Fisher process. Within this framework we have shown that using the Beta with spikes distribution proposed by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Tataru <em>et al.</em> (2015)</a> to approximate the Wright Fisher transitions provided a statistical inference comparable to that of the Wright-Fisher model while having a computational cost that does not depend on population size. We have shown this framework allows to perform hypothesis testing by relying on the asymptotic distribution of the likelihood-ratio statistic and that the maximum likelihood estimate of the selection intensity parameter had generally good statistical properties.</p><div id="sec-29" class="subsection"><h3>Advantages and limits of parametric approximations to the Wright Fisher process</h3><p id="p-58">Using parametric distributions to approximate Wright-Fisher transitions requires the user to know or approximate the moments of the Wright-Fisher process, including fixation probabilities in the case of the BwS and NG models. We have shown, in the lines of previous work by <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Lacerda and Seoighe (2014)</a>; <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Terhorst <em>et al.</em> (2015)</a> that this could be done using a Taylor expansion of the fitness function (see Supplemental File S1). Despite generally good performance of the approximation of moments, there still exists situations were the approximation performed poorly. For the BwS model, this only happened for low starting allele frequencies (<span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-230.gif"/></noscript></span></span>), large number of generations (<span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-231.gif"/></noscript></span></span>) and strong selection. Note that the poor performance for low starting allele frequencies is relevant for all <em>underlying true</em> allele frequency as for computing the likelihood the transition is integrated over all possible starting frequencies in the forward algorithm. So this poor performance could have impacted our simulation results. However, we showed that the WF likelihood was still well approximated by the BwS model. In cases where better approximation of the WF moments might be required, different approaches could be followed. For small population sizes (<span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-232.gif"/></noscript></span></span>), the WF moments can actually be computed exactly using <a id="xref-disp-formula-15-2" class="xref-disp-formula" href="#disp-formula-15">equation (4)</a> but this computation quickly becomes computationally prohibitive (the cost is approximately <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-233.gif"/></noscript></span></span>). A solution would be to develop the Taylor expansion some degrees further and possibly include correction terms as was done by <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Terhorst <em>et al.</em> (2015)</a>. This could be important for datasets covering large timescales such as in ancient DNA studies where sampling times can be widely spread.</p><p id="p-59">Here, we compared four different parametric distributions for approximating the Wright-Fisher transitions. The Gaussian distribution is a widely used approximation to the evolution of allele frequencies, starting from (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cavalli-Sforza <em>et al.</em>, 1964</a>), including for the analysis of genetic time series (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Lacerda and Seoighe 2014</a>; <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Terhorst <em>et al.</em>, 2015</a>). Our results confirm what is widely known: this approximation is only valid for small time scales (in units of <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-234.gif"/></noscript></span></span>) and when the starting minor allele frequency is high. The Beta distribution has been less used in this context (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Siren <em>et al.</em>, 2011</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Gompert 2016</a>) although we have shown that it offers a better approximation to the transitions than the Gaussian model. However these two models present poor performance when it comes to approximating the likelihood of data simulated under the Wright-Fisher model. This it not true for the Beta with spikes model that provides good approximations both for the transition kernel and the likelihood. Moreover, despite poor approximation of WF transitions for some parameters, the Nicholson Gaussian model generally provided a good fit of the likelihood. The good fit of NG and BwS is most likely due to the fact that they incorporate fixation probabilities that become large when the other models fail (large selection intensity, large intersample time, low minor allele frequency).</p><p id="p-60">The BwS model should therefore be preferred to the other three at least in the context of analyzing genetic time series using Hidden Markov Models. Given the good performance of the BwS model in this framework, it could be an interesting model to use in more general contexts. One interesting extension would be to use the BwS in the context of time series data sampled in multiple populations. However, one important limitation of using the Beta distribution (with or without spikes) for such data are that the multivariate Beta distribution is essentially unusable in practice. Using this class of distribution in a multiple populations context therefore requires developing a specific factorization for calculating the multivariate likelihood. An example of such an extension for the problem of estimating branch length in population trees can be found in <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Tataru <em>et al.</em> (2015)</a>. For such data, the Gaussian model remains much more practical but still suffers from the limitations mentioned above.</p><p id="p-61">In this study, we only focused on comparing parametric distributions to approximate the WF transitions. Another standard approximation to the Wright-Fisher process under selection that was not considered here is the diffusion process. This process has been shown to provide a good approximation to the WF process. However this approximation is only valid for large <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-235.gif"/></noscript></span></span> and small <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-236.gif"/></noscript></span></span> values and has two additional important limitations. The main limitation is that using the diffusion model in practice implies solving the diffusion equation which is a partial differential equation. One approach to do so is to write the transition distribution as an infinite sum of elementary solutions of the diffusion equation (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Song and Steinrücken 2012</a>). This approach was considered in a HMM for genetic time series by <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Steinrücken <em>et al.</em> (2014)</a> but exhibited prohibitive computational cost. Another approach is to consider numerical schemes to approximate the transition density (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bollback <em>et al.</em>, 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zhao <em>et al.</em>, 2013</a>). These approaches involve numerical integrations and are also computationally demanding. The second limitation of the diffusion process is that it does not naturally incorporate the fixation probabilities at 0 or 1. However, <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Zhao <em>et al.</em> (2013)</a> showed that the discretized diffusion transition density had atoms on these edges that could be used to represent the fixation probabilities. This suggest that the diffusion approximation could be extended to include fixation probabilities. This would require specific developments and evaluation.</p><p id="p-62">Overall, the choice of a particular transition model for genetic time series should depend on the study to analyze. For small <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-237.gif"/></noscript></span></span>, WF calculations are tractable and should be preferred. For larger <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-238.gif"/></noscript></span></span>, the diffusion approximation could be improved but parametric approximations seem more adapted (<a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Lacerda and Seoighe 2014</a>). In the class of parametric distributions, our results show that the Beta with spikes model is probably more flexible as it is adapted to both large <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-239.gif"/></noscript></span></span> and a wider range of <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-240.gif"/></noscript></span></span>.</p></div><div id="sec-30" class="subsection"><h3>Inference of selection from genetic time series</h3><p id="p-63">In principle, genetic time series can provide a lot of information on selection at a locus. This comes from the fact that the allele frequency trajectory underlying the observed sample is greatly affected by the selection process. Indeed we found that in many scenarios that we simulated likelihood inference provided essentially unbiased estimates of the selection parameter. The power to detect selection was substantial for a wide range of situations when the selection intensity <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-241.gif"/></noscript></span></span> was greater than 10. For large populations this corresponds to a relatively modest fitness effect. For example for a population of size 10,000 this corresponds to <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-242.gif"/></noscript></span></span>, <em>i.e.</em>, a 1% increase in fitness for an individual homozygote for the beneficial allele compared to the other homozygote.</p><p id="p-64">Our approach builds upon likelihood inference and in particular maximum likelihood estimation. While this approach was found very efficient in most cases, the maximum likelihood theory could not be applied to allele frequency trajectories where the reference allele was fixed or lost between the first and second sampling dates. Indeed, the best explaination for such events is <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-243.gif"/></noscript></span></span> (for allele loss) or <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-244.gif"/></noscript></span></span> (for allele fixation), in other words the MLE is reached on an edge of the domain. The occurrence of such events implies that the LR distribution under neutrality is not <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-245.gif"/></noscript></span></span>. An approach to ensure properly distributed p-values in this situation would rely on performing neutral simulations to obtain empirically the distribution of the LR under the null. However, a more fundamental issue is that the ML estimator gives no accurate information about the true <em>s</em> here, because sampling dates are too distant to capture how fast the allele frequency increases (or decreases).</p><p id="p-65">We chose to treat these trajectories specifically and excluded them from further analysis. However, the exclusion of fixed trajectories leads to an underestimation of <em>s</em>. This happens because the trajectories that are not excluded correspond to a biased sample of trajectories (those that reached fixation more slowly) and the resulting <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-246.gif"/></noscript></span></span> is lower than the true <em>s</em>. A way to compensate this effect would be to consider the transition process conditionally on the allele not being fixed. This can be done in the method of moments framework by using the conditional moments, and with the diffusion model by modifying the diffusion equation (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Ewens 2004</a>). However, this methodology may exclude a potentially large number of loci that can be properly analyzed within our framework, <em>i.e.</em>, all trajectories including allele loss or fixation <em>later than</em> the second sampling date. In real data, the way to avoid such situations is through the experimental design by chosing an appropriate sampling strategy.</p><p id="p-66">In this study we only examined one sampling strategy: 10 sampled dates equally spread along the total duration with 30 alleles sampled each time. With this strategy, we showed that the statistical quality of the model is driven by the <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-247.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-248.gif"/></noscript></span></span> parameters: if the selection is too weak, increasing the duration considered is needed so as to capture its effect on the allele frequency trajectory. But if the total duration is too large, distinguishing the effects of genetic drift and selection is harder. The consequence of our results is that, for a given dataset, as the statistical accuracy of the estimation and the power of the test depend on the sampling dates distribution and the sample sizes, the interpretation of the results becomes more difficult. Some selected loci will escape detection although they can be potentially important for fitness. Hence, for a given dataset, we would suggest performing simulations (possibly using the scripts we used here) matched for sampling dates and sizes under different selection intensities to help interpreting selection scan results. Moreover, such simulations could actually be used to compare possible sampling strategies (<em>i.e.</em>, depending on resources such as cost and sample availability) to optimize the power of the eventual statistical analysis. Deriving general guidelines for the optimization of sampling strategies in the context of genetic time series would require a dedicated study.</p><p id="p-67">One more aspect requiring further research would be the impact of the initial allele frequency distribution used in the HMM. In our simulation study the initial allele frequency was fixed (to 0.1 or 0.5), but we preferred ignoring this information that is unknow in practice and used the uniform distribution. When analyzing real data the allele frequency distribution is not expected to be uniform. Using a more realistic distribution may help the algorithm and make it more confident about its inference. However, this confidence persists even if the given initial distribution is actually wrong, which may then bias the inference. The choice of an informative initial distribution should thus be done with caution. For instance, while the hyperbolic function (<span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-249.gif"/></noscript></span></span>) represents a natural choice for the site frequency spectrum, one should be aware that it is only expected for a neutral population with constant size. Another solution would be to estimate the ancestral allele frequency distribution from the data, which is possible if time series are available genome-wide. Finally, note that these considerations really matter only if sample sizes at the oldest sampling dates are small, otherwise the emission probabilities at these dates are informative enough to overcome any prior on the initial distribution.</p><p id="p-68">Our evaluation of the HMM framework focused on its performance to detect and estimate selection. To do so we have assumed that other parameters, in particular effective population size, were known. In addition, we assumed that each parameter was constant over the sampling duration. In practice most of these assumptions are deemed to be wrong. Regarding the estimation of <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-250.gif"/></noscript></span></span>, in our analysis of the real dataset, we used a two-step approach by first estimating <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-251.gif"/></noscript></span></span> using <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Hui and Burt (2015)</a> and then <em>s</em> at each locus. The issue with such an approach is that all loci are used to get the <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-252.gif"/></noscript></span></span> estimate. If some of them are under selection, they will exhibit larger allele frequency deviation. The consequence is that the <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-253.gif"/></noscript></span></span> estimate is going to be under estimated and it becomes harder to reject the null hypothesis (genetic drift). Hence, while this two-step approach is practical, it could be improved by jointly estimating <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-254.gif"/></noscript></span></span> and <em>s</em>. The framework described here can in principle be used to do so as the HMM likelihood incorporates both <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-255.gif"/></noscript></span></span> and <em>s</em>. This will require fitting the HMM model on multiple loci jointly as the information on <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-256.gif"/></noscript></span></span> lies in the variance of the allele frequency trajectories over the whole genome. Also, it is now well established that populations of many species have experienced large changes in effective population size over time. There is also many situations where the selection intensity of an allele (expressed by <em>s</em> in our model) will change over time (<em>e.g.</em>, time varying environmental conditions, selection for an optimum trait value …). In principle the joint model described above to jointly estimate <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4073/embed/mml-math-257.gif"/></noscript></span></span> and <em>s</em> could also be extended to allow for parameters to vary in time (<em>e.g.</em>, at each interval between sampling dates). The likelihood framework described here can form the basis of such models.</p></div><div id="sec-31" class="subsection"><h3>Conclusion</h3><p id="p-69">Genome-wide assessment of the genetic diversity of a species is becoming more and more accessible in particular thanks to the development of cost effective genotyping and sequencing techniques. With the increase in genomic data, the evolution of genetic diversity in time becomes accessible either as a by-product of data accumulation or through dedicated projects in artificial populations (<em>e.g.</em>, experimental evolution) or natural settings (<em>e.g.</em>, ancient DNA sequencing). As genomic time series data become more widespread, their analysis requires dedicated methods that have good statistical and computational properties. In this study we have established a general statistical framework to this aim that we believe can contribute to further developments for the analysis of genetic time series.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">The research leading to these results was conducted as part of the IMAGE project, which received funding from the European Union’s Horizon 2020 Research and Innovation Program under grant agreement no. 677353. It was also conducted as part of the Haptitude project, which was funded by INRA metaprogramme SelGen. Data analyzes were performed on the computer cluster of the Genotoul bioinformatics platform Toulouse Midi-Pyrénées (<a href="http://www.bioinfo.genotoul.fr">www.bioinfo.genotoul.fr</a>). We would like to thank Thomas Bataillon and Paula Tataru for useful discussions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-71">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9959492">https://doi.org/10.25387/g3.9959492</a>.</p></li><li class="fn" id="fn-2"><p id="p-72"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-05"><span class="received-label">Received </span>July 5, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-07"><span class="accepted-label">Accepted </span>October 7, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Paris <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.1" data-doi="10.1186/s12863-016-0369-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alnahhas</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Berri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chabault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Chartrin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Boulay</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic parameters of white striping in relation to body weight, carcass composition, and meat quality traits in two broiler lines divergently selected for the ultimate pH of the pectoralis major muscle.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-016-0369-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D17%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-016-0369-2%26rft_id%253Dinfo%253Apmid%252F27094623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-016-0369-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27094623&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.2" data-doi="10.1007/BF01441146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>,  <span class="cit-name-given-names">D. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Nichols</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A method for quantifying differentiation between populations at multi-allelic loci and its implications for investigating identity and paternity.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF01441146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DBalding%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D96%26rft.issue%253D1-2%26rft.spage%253D3%26rft.epage%253D12%26rft.atitle%253DA%2Bmethod%2Bfor%2Bquantifying%2Bdifferentiation%2Bbetween%2Bpopulations%2Bat%2Bmulti-allelic%2Bloci%2Band%2Bits%2Bimplications%2Bfor%2Binvestigating%2Bidentity%2Band%2Bpaternity.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01441146%26rft_id%253Dinfo%253Apmid%252F7607457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01441146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7607457&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RG20800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.3" data-doi="10.1186/s12864-018-4690-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bihan-Duval</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hennequet-Antier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Berri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Beauclercq</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Bourin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Identification of genomic regions and candidate genes for chicken meat ultimate pH by combined detection of selection signatures and QTL.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4690-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4690-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4690-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.4" data-doi="10.1534/genetics.107.085019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">York</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimation of 2Nes from temporal allele frequency data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.085019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBollback%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D497%26rft.epage%253D502%26rft.atitle%253DEstimation%2Bof%2B2Nes%2BFrom%2BTemporal%2BAllele%2BFrequency%2BData%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085019%26rft_id%253Dinfo%253Apmid%252F18493066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvNDk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.5" data-doi="10.1534/genetics.110.117275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chevalet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Servin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Boitard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Abdallah</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detecting Selection in Population Trees: The Lewontin and Krakauer Test Extended.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.117275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.117275%26rft_id%253Dinfo%253Apmid%252F20855576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvMjQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4073.6" data-doi="10.1007/0-387-28982-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cappé</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Moulines</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ryden</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>  <span class="cit-source"><em>Inference in hidden Markov models. Springer series in statistics</em>.</span> <span class="cit-publ-name">Springer science &amp; Business Media</span>, <span class="cit-publ-loc">Berlin</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/0-387-28982-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInference%2Bin%2Bhidden%2BMarkov%2Bmodels.%2BSpringer%2Bseries%2Bin%2Bstatistics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F0-387-28982-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/0-387-28982-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.7" data-doi="10.1101/SQB.1964.029.01.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>,  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Barrai</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W. F.</span> <span class="cit-name-surname">Edwards</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Analysis of Human Evolution Under Random Genetic Drift.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/SQB.1964.029.01.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DCavalli-Sforza%26rft.auinit1%253DL.%2BL.%26rft.volume%253D29%26rft.issue%253D0%26rft.spage%253D9%26rft.epage%253D20%26rft.atitle%253DAnalysis%2Bof%2BHuman%2BEvolution%2BUnder%2BRandom%2BGenetic%2BDrift%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1964.029.01.006%26rft_id%253Dinfo%253Apmid%252F14278497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjY6IjI5LzAvOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDA3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.8" data-doi="10.3168/jds.2018-15217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doekes</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Veerkamp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bijma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hiemstra</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Windig</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Value of the Dutch Holstein Friesian germplasm collection to increase genetic variability and improve genetic merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">10022</span>–<span class="cit-lpage">10033</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2018-15217</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D101%26rft.spage%253D10022%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2018-15217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2018-15217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4073.9" data-doi="10.1007/978-0-387-21822-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewens</span>,  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Mathematical population genetics</span>, <span class="cit-vol">Vol. 27</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-21822-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bpopulation%2Bgenetics%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-21822-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-21822-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.12.4073.10"><div class="cit-metadata unstructured">Fages, A., K. Hanghøj, N. Khan, C. Gaunitz, A. Seguin-Orlando <em>et al.</em>, 2019 Tracking Five Millennia of Horse Management with Extensive Ancient Genome Time Series. Cell 177:1419–1435: e31. <a href="https://doi:10.1016/j.cell.2019.03.049">https://doi:10.1016/j.cell.2019.03.049</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.11" data-doi="10.1534/genetics.112.147231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fariello</span>,  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Boitard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">SanCristobal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Servin</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detecting signatures of selection through haplotype differentiation among hierarchically structured population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147231</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147231%26rft_id%253Dinfo%253Apmid%252F23307896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvOTI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.12" data-doi="10.1534/genetics.115.184598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admetlla</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Leuenberger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wegmann</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An approximate markov model for the Wright–Fisher diffusion and its application to time series data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.184598</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.184598%26rft_id%253Dinfo%253Apmid%252F27038112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzIvODMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.13" data-doi="10.1111/1755-0998.12280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shim</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">WFABC: a W right–F isher ABC-based approach for inferring effective population sizes and selection coefficients from time-sampled data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12280</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12280%26rft_id%253Dinfo%253Apmid%252F24834845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24834845&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.14" data-doi="10.1111/mec.13323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gompert</span>,  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bayesian inference of selection in a heterogeneous environment from genetic time-series data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13323</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13323%26rft_id%253Dinfo%253Apmid%252F26184577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26184577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Dai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Beaumont</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Yu</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">An MCMC-based method for Bayesian inference of natural selection from time series DNA data across linked loci.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.16" data-doi="10.1371/journal.pone.0201838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>,  <span class="cit-name-given-names">J.-N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zerjal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Hospital</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Cancer-and behavior-related genes are targeted by selection in the Tasmanian devil (Sarcophilus harrisii).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e0201838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0201838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0201838%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0201838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0201838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.17" data-doi="10.1534/genetics.115.174904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hui</span>,  <span class="cit-name-given-names">T.-Y. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Burt</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Estimating Effective Population Size from Temporally Spaced Samples with a Novel, Efficient Maximum-Likelihood Algorithm.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.174904</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174904%26rft_id%253Dinfo%253Apmid%252F25747459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzEvMjg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.18" data-doi="10.1534/genetics.116.197566"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iranmehr</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Akbari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Bafna</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Clear: Composition of Likelihoods for Evolve and Resequence Experiments.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1011</span>–<span class="cit-lpage">1023</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197566</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197566%26rft_id%253Dinfo%253Apmid%252F28396506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8yLzEwMTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.19" data-doi="10.2307/3211856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Diffusion Models in Population Genetics.</span> <abbr class="cit-jnl-abbrev">J. Appl. Probab.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/3211856</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BProbab.%26rft.volume%253D1%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3211856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3211856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.20" data-doi="10.1534/genetics.114.167957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacerda</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Seoighe</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population genetics inference for longitudinally-sampled mutants under strong selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1237</span>–<span class="cit-lpage">1250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.167957</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167957%26rft_id%253Dinfo%253Apmid%252F25213172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC8zLzEyMzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.21" data-doi="10.1111/mec.13492"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>,  <span class="cit-name-given-names">A.-S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Methods to characterize selective sweeps using time serial samples: an ancient DNA perspective.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13492</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13492&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.22" data-doi="10.1534/genetics.112.140939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>,  <span class="cit-name-given-names">A.-S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Malaspinas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Evans</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Slatkin</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating allele age and selection coefficient from time-serial data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.140939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140939%26rft_id%253Dinfo%253Apmid%252F22851647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.23" data-doi="10.1038/nature16152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathieson</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Lazaridis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Rohland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mallick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Patterson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide patterns of selection in 230 ancient Eurasians.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">528</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature16152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D528%26rft.spage%253D499%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16152%26rft_id%253Dinfo%253Apmid%252F26595274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26595274&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.24" data-doi="10.1111/1467-9868.00357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jónsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Ó.</span> <span class="cit-name-surname">Gústafsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Stefánsson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Assessing population differentiation and isolation from single-nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">715</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1467-9868.00357</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D64%26rft.spage%253D695%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.25" data-doi="10.1109/5.18626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabiner</span>,  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A tutorial on hidden Markov models and selected applications in speech recognition.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/5.18626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D77%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.18626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/5.18626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.26" data-doi="10.1534/genetics.118.301684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sackman</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Harris</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Inferring demography and selection in organisms characterized by skewed offspring distributions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">1019</span>–<span class="cit-lpage">1028</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301684</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301684%26rft_id%253Dinfo%253Apmid%252F30651284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIxMS8zLzEwMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.27" data-doi="10.1534/genetics.116.187278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schraiber</span>,  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Evans</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Slatkin</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bayesian inference of natural selection from allele frequency time series.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.187278</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.187278%26rft_id%253Dinfo%253Apmid%252F27010022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvNDkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.28" data-doi="10.1093/molbev/msq236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siren</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marttinen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Corander</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reconstructing Population Histories from Single Nucleotide Polymorphism Data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">683</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msq236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq236%26rft_id%253Dinfo%253Apmid%252F20819907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msq236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20819907&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285418600064&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.29" data-doi="10.1534/genetics.111.136929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Steinrücken</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Simple Method for Finding Explicit Analytic Transition Densities of Diffusion Processes with General Diploid Selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.136929</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136929%26rft_id%253Dinfo%253Apmid%252F22209899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC8zLzExMTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.30" data-doi="10.1214/14-AOAS764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinrücken</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bhaskar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A novel spectral method for inferring general diploid selection from time series genetic data.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Stat.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2203</span>–<span class="cit-lpage">2222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/14-AOAS764</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat.%26rft.volume%253D8%26rft.spage%253D2203%26rft_id%253Dinfo%253Adoi%252F10.1214%252F14-AOAS764%26rft_id%253Dinfo%253Apmid%252F25598858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/14-AOAS764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25598858&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.31"><div class="cit-metadata unstructured">Storey, J. D., A. J. Bass, A. Dabney, and D. Robinson, 2015. <em>qvalue: Q-value estimation for false discovery rate control</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.32" data-doi="10.1534/genetics.115.179606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tataru</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bataillon</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hobolth</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inference under a wright-fisher model using an accurate beta approximation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.179606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179606%26rft_id%253Dinfo%253Apmid%252F26311474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS8zLzExMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.33" data-doi="10.1093/sysbio/syw056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tataru</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Simonsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bataillon</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hobolth</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Statistical inference in the Wright-Fisher model using allele frequency data.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">e30</span>–<span class="cit-lpage">e46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/syw056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyw056%26rft_id%253Dinfo%253Apmid%252F27486182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syw056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27486182&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.34" data-doi="10.1371/journal.pgen.1005069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terhorst</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multi-locus Analysis of Genomic Time Series Data from Experimental Evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005069</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005069%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.35" data-doi="10.1111/evo.12705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Parallel trait adaptation across opposing thermal environments in experimental Drosophila melanogaster populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1759</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D1745%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.36" data-doi="10.1146/annurev-genet-111212-133526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitti</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Grossman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Sabeti</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detecting natural selection in genomic data.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genet-111212-133526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-111212-133526%26rft_id%253Dinfo%253Apmid%252F24274750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-111212-133526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24274750&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4073.37" data-doi="10.1534/genetics.113.152017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yue</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Waxman</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Complete Numerical Solution of the Diffusion Equation of Random Genetic Drift.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">985</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152017%26rft_id%253Dinfo%253Apmid%252F23749318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/4073.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4059" title="Investigating the Influence of Glycerol on the Utilization of Glucose in Yarrowia lipolytica Using RNA-Seq-Based Transcriptomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4087" title="rmtA-Dependent Transcriptome and Its Role in Secondary Metabolism, Environmental Stress, and Virulence in Aspergillus flavus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4073" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e4q767R-IgVHXMWRXNmGISCVE1sZdrnGTYNkgAvIJSo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/4073" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/4073" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/4073" class="active" data-icon-position="" data-hide-link-title="0">Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494998" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ytVwmhN40GqlQ-75KYN8ycSCVD0MuIT4EEJPKdetBUw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4073.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/4073" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NjkCI0ihg6LDlUVmXNpJeMvTx6BU3-DZIC_--R5Upgw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/4073" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4073.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494998" id="node494998--2" data-pisa="ggg;9/12/4073" data-pisa-master="ggg;g3.119.400778" data-apath="/ggg/9/12/4073.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cyriel Paris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5141-0913" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bertrand Servin</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2499-5052" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simon Boitard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4073-4086;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494998--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cyriel Paris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyriel%2BParis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paris%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyriel%2BParis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bertrand Servin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bertrand%2BServin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Servin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABertrand%2BServin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5141-0913" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bertrand Servin</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Simon Boitard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BBoitard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boitard%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BBoitard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2499-5052" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simon Boitard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:simon.boitard@inra.fr" class="" data-icon-position="" data-hide-link-title="0">simon.boitard@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/4073" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/4073&amp;title=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/4073&amp;title=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/4073&amp;title=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/4073&amp;text=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/4073&amp;title=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/4073&amp;t=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/4073&amp;title=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/4073&amp;title=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494998/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494998" id="node494998--3" data-pisa="ggg;9/12/4073" data-pisa-master="ggg;g3.119.400778" data-apath="/ggg/9/12/4073.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inference of Selection from Genetic Time Series Using Various Parametric Approximations to the Wright-Fisher Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cyriel Paris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5141-0913" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bertrand Servin</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2499-5052" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simon Boitard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4073-4086;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494998--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cyriel Paris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyriel%2BParis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paris%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyriel%2BParis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bertrand Servin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bertrand%2BServin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Servin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABertrand%2BServin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5141-0913" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bertrand Servin</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Simon Boitard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Génétique, Physiologie Systèmes d’Elevage, INRA, Centre Occitanie-Toulouse, 31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BBoitard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boitard%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BBoitard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2499-5052" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simon Boitard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:simon.boitard@inra.fr" class="" data-icon-position="" data-hide-link-title="0">simon.boitard@inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=4073&amp;rft.epage=4086&amp;rft.atitle=Inference%20of%20Selection%20from%20Genetic%20Time%20Series%20Using%20Various%20Parametric%20Approximations%20to%20the%20Wright-Fisher%20Model&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Paris%2C+Cyriel&amp;rft.au=Servin%2C+Bertrand&amp;rft.au=Boitard%2C+Simon"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494998/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494998/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494998/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494998/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494998/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494998/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494998/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494998/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494998/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494998/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494998/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494998/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/4073" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/4073.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/4073.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/4073.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
