<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/3981" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/3981.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/3981.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/3981.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/12/3981/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/3981.atom" />
<meta name="HW.pisa" content="ggg;9/12/3981" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants" />
<meta name="DC.Identifier" content="10.1534/g3.119.400677" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Ceron, Crossa. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The constrained linear genomic selection index (CLGSI) is a linear combination of genomic estimated breeding values useful for predicting the net genetic merit, which in turn is a linear combination of true unobservable breeding values of the traits weighted by their respective economic values. The CLGSI is the most general genomic index and allows imposing constraints on the expected genetic gain per trait to make some traits change their mean values based on a predetermined level, while the rest of them remain without restrictions. In addition, it includes the unconstrained linear genomic index as a particular case. Using two real datasets and simulated data for seven selection cycles, we compared the theoretical results of the CLGSI with the theoretical results of the constrained linear phenotypic selection index (CLPSI). The criteria used to compare CLGSI vs. CLPSI efficiency were the estimated expected genetic gain per trait values, the selection response, and the interval between selection cycles. The results indicated that because the interval between selection cycles is shorter for the CLGSI than for the CLPSI, CLGSI is more efficient than CLPSI per unit of time, but its efficiency could be lower per selection cycle. Thus, CLGSI is a good option for performing genomic selection when there are genotyped candidates for selection." />
<meta name="DC.Contributor" content="J. Jesus Cerón-Rojas" />
<meta name="DC.Contributor" content="Jose Crossa" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The constrained linear genomic selection index (CLGSI) is a linear combination of genomic estimated breeding values useful for predicting the net genetic merit, which in turn is a linear combination of true unobservable breeding values of the traits weighted by their respective economic values. The CLGSI is the most general genomic index and allows imposing constraints on the expected genetic gain per trait to make some traits change their mean values based on a predetermined level, while the rest of them remain without restrictions. In addition, it includes the unconstrained linear genomic index as a particular case. Using two real datasets and simulated data for seven selection cycles, we compared the theoretical results of the CLGSI with the theoretical results of the constrained linear phenotypic selection index (CLPSI). The criteria used to compare CLGSI &lt;i&gt;vs.&lt;/i&gt; CLPSI efficiency were the estimated expected genetic gain per trait values, the selection response, and the interval between selection cycles. The results indicated that because the interval between selection cycles is shorter for the CLGSI than for the CLPSI, CLGSI is more efficient than CLPSI per unit of time, but its efficiency could be lower per selection cycle. Thus, CLGSI is a good option for performing genomic selection when there are genotyped candidates for selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/3981" />
<meta name="citation_id" content="9/12/3981" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/3981" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/3981.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/3981.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/3981.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400677" />
<meta name="citation_pmid" content="31570501" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3981" />
<meta name="citation_lastpage" content="3994" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="J. Jesus Cerón-Rojas" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2885-6831" />
<meta name="citation_author" content="Jose Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=V. Börner;citation_author=N. Reinsch;citation_title=Optimising multistage dairy cattle breeding schemes including genomic selection using decorrelated or optimum selection indices.;citation_pages=1-1;citation_volume=44;citation_year=2012;citation_pmid=22252172;citation_doi=10.1186/1297-9686-44-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=S. Mugo;citation_author=A. Tarekegne;citation_author=R. Babu;citation_title=Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.;citation_pages=154-163;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.07.0460" />
<meta name="citation_reference" content="citation_author=J. J. Ceron-Rojas;citation_author=J. Crossa;citation_author=V. N. Arief;citation_author=K. Basford;citation_author=J. Rutkoski;citation_author=D. Jarquín;citation_author=G. Alvarado;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=I. DeLacy;citation_year=2015;citation_pmid=26290571;citation_doi=10.1534/g3.115.019869" />
<meta name="citation_reference" content="citation_title=Linear selection indices in modern plant breeding;citation_author=J. J. Céron-Rojas;citation_author=J. Crossa;citation_year=2018;citation_doi=10.1007/978-3-319-91223-3" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. J. Cerón-Rojas;citation_author=J. Crossa;citation_author=F. H. Toledo;citation_author=J. Sahagún-Castellanos;citation_title=A predetermined proportional gains eigen selection index method.;citation_pages=2436-2447;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2015.11.0718" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=J. C. M. Dekkers;citation_title=Prediction of response to marker-assisted and genomic selection using selection index theory.;citation_pages=331-341;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076470;citation_doi=10.1111/j.1439-0388.2007.00701.x" />
<meta name="citation_reference" content="citation_title=Introduction to quantitative genetics;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=D. A. Harville;citation_title=Index selection with proportionality constraints.;citation_pages=223-225;citation_volume=31;citation_year=1975;citation_issue=1;citation_pmid=1164534;citation_doi=10.2307/2529722" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. N. Hazel;citation_title=The genetic basis for constructing selection indexes.;citation_pages=476-490;citation_volume=8;citation_year=1943" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Y. Itoh;citation_author=Y. Yamada;citation_title=Comparisons of selection indices achieving predetermined proportional gains.;citation_pages=69-82;citation_volume=19;citation_year=1987;citation_doi=10.1186/1297-9686-19-1-69" />
<meta name="citation_reference" content="citation_title=Genetic Data Analysis for Plant and Animal Breeding;citation_author=F. Isik;citation_author=J. Holland;citation_author=C. Maltecca;citation_year=2017;citation_doi=10.1007/978-3-319-55177-7" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=O. Kempthorne;citation_author=A. W. Nordskog;citation_title=Restricted selection indices.;citation_pages=10-19;citation_volume=15;citation_year=1959;citation_doi=10.2307/2527598" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic selection in plant breeding: knowledge and prospects.;citation_pages=77-123;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=J. Mallard;citation_title=The theory and computation of selection indices with constraints: A critical synthesis.;citation_pages=713-735;citation_volume=28;citation_year=1972;citation_doi=10.2307/2528758" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. W. Podlich;citation_author=M. Cooper;citation_title=QU-GENE: a simulation platform for quantitative analysis of genetic models;citation_pages=632-653;citation_volume=14;citation_year=1998;citation_issue=7;citation_pmid=9730929;citation_doi=10.1093/bioinformatics/14.7.632" />
<meta name="citation_reference" content="citation_title=Methods of multivariate analysis;citation_author=A. C. Rencher;citation_year=2002;citation_doi=10.1002/0471271357" />
<meta name="citation_reference" content="citation_title=Matrix analysis for statistics;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=L. R. Schaeffer;citation_title=Strategy for applying genome-wide selection in dairy cattle.;citation_pages=218-223;citation_volume=123;citation_year=2006;citation_issue=4;citation_pmid=16882088;citation_doi=10.1111/j.1439-0388.2006.00595.x" />
<meta name="citation_reference" content="citation_journal_title=Jpn. J. Genet.;citation_author=G. M. Tallis;citation_title=Constrained selection.;citation_pages=151-155;citation_volume=60;citation_year=1985;citation_doi=10.1266/jjg.60.151" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=K. Togashi;citation_author=C. Y. Lin;citation_author=T. Yamazaki;citation_title=The efficiency of genome-wide selection for genetic improvement of net merit.;citation_volume=89;citation_year=2011;citation_pmid=21512116;citation_doi=10.2527/jas.2009-2606" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="The constrained linear genomic selection index (CLGSI) is a linear combination of genomic estimated breeding values useful for predicting the net genetic merit, which in turn is a linear combination of true unobservable breeding values of the traits weighted by their respective economic values. The CLGSI is the most general genomic index and allows imposing constraints on the expected genetic gain per trait to make some traits change their mean values based on a predetermined level, while the rest of them remain without restrictions. In addition, it includes the unconstrained linear genomic index as a particular case. Using two real datasets and simulated data for seven selection cycles, we compared the theoretical results of the CLGSI with the theoretical results of the constrained linear phenotypic selection index (CLPSI). The criteria used to compare CLGSI vs. CLPSI efficiency were the estimated expected genetic gain per trait values, the selection response, and the interval between selection cycles. The results indicated that because the interval between selection cycles is shorter for the CLGSI than for the CLPSI, CLGSI is more efficient than CLPSI per unit of time, but its efficiency could be lower per selection cycle. Thus, CLGSI is a good option for performing genomic selection when there are genotyped candidates for selection." />
<meta name="og-title" property="og:title" content="Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/3981" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The constrained linear genomic selection index (CLGSI) is a linear combination of genomic estimated breeding values useful for predicting the net genetic merit, which in turn is a linear combination of true unobservable breeding values of the traits weighted by their respective economic values. The CLGSI is the most general genomic index and allows imposing constraints on the expected genetic gain per trait to make some traits change their mean values based on a predetermined level, while the rest of them remain without restrictions. In addition, it includes the unconstrained linear genomic index as a particular case. Using two real datasets and simulated data for seven selection cycles, we compared the theoretical results of the CLGSI with the theoretical results of the constrained linear phenotypic selection index (CLPSI). The criteria used to compare CLGSI vs. CLPSI efficiency were the estimated expected genetic gain per trait values, the selection response, and the interval between selection cycles. The results indicated that because the interval between selection cycles is shorter for the CLGSI than for the CLPSI, CLGSI is more efficient than CLPSI per unit of time, but its efficiency could be lower per selection cycle. Thus, CLGSI is a good option for performing genomic selection when there are genotyped candidates for selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"V9WhUIdkuyyTO-WP6RNLRxuPsM7BxRuOUGPzdzFK1UA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494999","apath":"\/ggg\/9\/12\/3981.atom","pisa":"ggg;9\/12\/3981","ac":{"\/ggg\/9\/12\/3981.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/3981.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/3981"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/3981"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/3981":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TywWjllE3KdTYjovUu\\\\\\\/KFxapj1T1VGNO8OPiFY\\\\\\\/nRTXnpyfps3tXbpNIyNWJpgI8tzvWQHR+MxAdwKGOiC6ll\\\\\\\/nDUtBYZmcQeKs+QiMsvUEfhznMm\\\\\\\/Xr8EIQi4vDIf1YyxWxrsdSMkystR2wsv+iGk3gcQEio36Jcqy1zBMr6bDVjWlDCOcRu2fwlcx9uQezw0uqiTXD1\\\\\\\/VrJa\\\\\\\/JGnn3d2SV5McOaLkHJM5Iv0CnUqkwOF\\\\\\\/CEz3p\\\\\\\/Rd1a4WqGU+iEt3rsQlx0R52F7cWSAg+iXUDfDUDyfcDmg7yXrn7z7Dk5oTXgZ29TVftBVEe3S6QrWTewdweeNS2DkyUYltP0sO\\\\\\\/p+3j6fc3ymS58z7NxnmEZ\\\\\\\/FnkaIA5R\\\\\\\/sT1ESdQxK8Iar41hBhR5iIqrTqK1g8eleJgp8smGpTyrV4wEyFPV4tJzoWE4LJco1lE6ppaWJE5VLTPpLAG07y3H7ugflPcNQ3DQMNizc+tDUp1cN+4k9HuC3coO7qxvUpo+cpgJiTVbdXkjCXLsCUUGQrje\\\\\\\/OgugvmAOqf+Jd+Bv6jDV7zq0IFdA7mlmi\\\\\\\/GyZn2EMghH54CLkzb\\\\\\\/SoddpDo+1wP3xNCRbZq+dlSqa82E6\\\\\\\/jbEQZRgJCobNDNara1GzLwOirFl7+UX\\\\\\\/OuQbIFpEMxQG44klnAJwnRon+V1Id56dMEDwBWQX4jOYV7b\\\\\\\/SqNeuIQ0tZdA4uKYd+zJQ6LfrrEA41aEiB8Dn9UXkyCOAwUezSxb1EURyIBZz71E6Z60XNwotZdhDgVJAa+Weg1vn3u667No3YQMCjCefu\\\\\\\/UObmfKvY5bLO4GCv49nkyYj5pslj1dSaGDcHUoJtzvDmZJH8ZSr3Fxwbu0zLigDAOA9B8J3CmFGPMV4urm9Ej5sHrHPzkqKSVZSamf70mwI4oKnZ0Ni3GIeBhPxKSAVsLl6Lc3CvtAIuOjMorDzKUJDNllc7BBekDd4HsCrOtCs6NZxsBoo4ydAJo+xHkJS4hcptbhY9S430QpK4D0KsiULjrK9jmZt8irAjTGx2COdarLSAmK0TWVpNr3GXfiqlNts7XWu\\\\\\\/ui49VPas7bzyaM9+iUTp3gV1faEs1XtoLUo0xrMrhXMoNQIxJ17PEh0\\\\\\\/EJKaZ0kI3ijjY7vJBhDyP8ZkPLINowsoeGCS\\\\\\\/6gd\\\\\\\/aTGKBuIfQ5sutnH+WCTmYrF9w64cn0WqTUKxgXyCAUSN64TMqiJP0Tmsqx\\\\\\\/Hs2\\\\\\\/5DCQPtHd4zYBv5sf8Z4FGRqYufCQ+x+Tmd3GtOeTw26NHwVxGKq7+im\\\u0022,\\\u0022iv\\\u0022:\\\u0022LgdQVnrs\\\\\\\/r9VJbbxQFIirQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a5e3fbb43d250e14f946f2eee368acd04999077cfc67d3c3b490150eded4e327\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jxPTwSjVmQnMN069\\\\\\\/sIRcmLiKSNe4gQAIpiNkDt8jJwsD7GQDNzyCdVoBQIGyLYcB2e35IilzcBOyck1zz95qLE5JzYf0d8A6OLpyyBvM5R8xBLRaPqYy151frb\\\\\\\/0hTZXfOFQXgBS92pLhOZCtCMdQF8xR6R1Kvef9Ctg40\\\\\\\/BcRc\\\\\\\/SfM5wKrwgJueSqwBWjibQV1bRQvk1WrfGzWJM0GOdtbcBk75R4qcccrLOx+ltMAgfZMsfSyyCvn1ooAxGHe7vm3LZYw7KhFVT0df9j9Bw3Lxnjmij3WX43e8bXs8DB0i21n69\\\\\\\/\\\\\\\/qRcsr\\\\\\\/xPWLlsnY3YozuiwlOvZiuQeGWG9WhzkPAnf0+i6Js9rGlEyFep7A82g4OYiQL5+RUd34dV65q3iayLNtG5+lALS8GYm54tjxpi01hHKb11COiTlMzhvtpQsnRFlgBDnjWvoIVCY2zo8MA3lt\\\\\\\/ijywPTdEyNGwAO0XLngLoakYp2f7g12+QM3eVSsMDAKvJtloiVbHmlG+acZb6gO3CBAkg\\\\\\\/kQCtLKvprQxDLEvnHRM6SgMvEY\\\\\\\/bASJTXZqUolshC1kIsGdEDlPoiQG6hNQBipX72rpHDlVTVIcLunEmUIHTI4z1X7SzBMgkwwf6jnuu2W3rbJSsn2OxhoB4V0Hk\\\\\\\/aN1V893fXRntY2qli8AwXFXr9SmO63qJ5LB1gCYZmIBbqqYU\\\\\\\/IlYURqsFVCc6SFoSKSfE7w6Cq0vHkiLdtQQyuwWtnFNTIP7PaKQNrsthddAZEcNfBu\\\\\\\/2uTP74\\\\\\\/IC4oNePIQj2dArujscllO0zNnNt5GVUC\\\\\\\/73t7+cogd8xZmURgIhJCIPumQkSknP22QUwjksSQWn0lamxpHAh5JTow4kmpk2AMKMhCEK5HHAQS+gF33YUjD\\\\\\\/TdMZCmKnrSbI\\\\\\\/vhHUA4a627XznCImuUcgahrzw\\\\\\\/EZXRaZqhuS17fUTlcSpgnDI5JoIkopwgvVjd4U5kRjQ8MEF3EEg0JUJnWPWP\\\\\\\/rI4LR\\\\\\\/cYB2lKq78GmtdOMtj3J5j4f2yjifc49fgT23vkzZf9aQQTEgnhnuGjVTbsNGLXV+Jrs9jOctoipzvmlKwHCyTkcs\\\\\\\/5E0TDFP+Ahc6nvpO6Hxg7f0VUUrsS4gXdqlnvWf1MA8Km3cTC9gbLqYdK9siEQE4\\\\\\\/M9FrBVFyOInG1XsYmhFXmUlB2wAdWcP14So\\\\\\\/tXgMWpJj7EAy6Gkvjjk9dIdRxV9vGkXuCosnKcaPAmx4RYKXvu8Qt4SyHByy2xVJWNx11AHOXKWsl5ShwjwQ2n1t23YO9GM0hN00n2P4m41zTO8rLJkUL2t2XD4u3xDjWzUEk\\\\\\\/haOleJ0eKVQHfn2tq0YbYTaqFQcyJT7BK7kLSk5mGeVX4wZVRazJ8jAhT6BQE263cNlxo6EIpBYMvAie\\\\\\\/CSRZjQWNB7G5LA8cgJgmqw0Oqb0NqBukU70SOReUu99uBqwcBNaBwrazAG0b1vHUGg3b1mLqkKS4dAhJMXnBhVpfzMzaISpYSrvlgwBg4JXDm+8kDuwY6+KB1WyxGBn4ILpwElupBaVJoCiWbCQ+Ux7szMCfA7L8cT5kXurmAryvCq4f15jw6dRVv03mvGHYv0lySW0awM+bXYG93HqC8RAxrXG0Z4gr7LZgm1Ofn4iHx\\\\\\\/U5kWrGyao4FlxrY6NO1VcHjClJlULKZJp0oGHoseO59ZN+rn0787YHsBkoCKSVP4foaABrxTJi7P+rfo7kXs9FLUomgzUYloq\\\\\\\/Hy+xnPBCdDljes4QfaKhRSv5Bv7l44JnVhuIYpOtxxB6XGoy8oInIDNyZi4N0+nSxefeYdLKYSj5Jn4K6uvSYUfZra3\\\\\\\/MRjCqde89KTUJ6uIwQYtFDmoo53ZiNpiQUM36HGxf+vRz5mJEemCGNYcNZsGb3d3oBxRvRfuk7nCwE+wo3u2bNyksD0WnxtByI4I0Iz1K0ICwSpupi9fdtc1K93NY\\\\\\\/BDJnfMVxoLgCzRX6KsydI8FKmGIwTGTxtGk1xT5RlsuuOFTRUulByiz\\\\\\\/IKNEtbmIJD1ZedCidEFgLyhDXKwKghCreSKWDJHRABYA1OAM3ifQXa2M9EuvlVpxxbiSEY885K3kQ+WfhkKa7HnX52Jb\\\\\\\/7SPO56Ztf9+KQJX9aCefuzOxNU5n2kij8zqUaVZo96nrSfcjdHQNCAr5\\\\\\\/dVncIFPgGLiBJS6BgOG7JdFiV0bypyTCrTk+8ys2XSvG41mlr3oEAFzm7LCdKUdaf9rjrmBOZ3wZJ+RVmrf9zATb4LwIpMFU9YKrQi3+0fU1Gy\\\\\\\/3W3gfjtx3gj53ay+MCrkPRPQHtsilqSiGdQ7wW6vqPQk1XfoMWvMr+j78hJl1F3qXl1j86ovv1KRlPEBXKhG6FOvK+3usxe6O+MWGkp8828UFSglwCftZN7tv7GstXrg9LfD9cV3jETY4A++NhG5pVcetKHoQLzH\\\\\\\/QRYWFYDsq1zZ0\\\\\\\/JPEQvUeXcIcClApTbvD9tXuzIOlm9dZ2DZUs4Kr3uUo8o3ocR9X22wSq9TMHPwvPHWXnY0c3UniUTvygHnl06fr6qYDs\\\\\\\/1Z89sOjBkLZaXHX4FA1xHwbm7Oa6AQjGMD8Xj3\\\\\\\/8wEGOJN9\\\\\\\/Z0IyP4UI4kWl2yjB34ZbvjyPRHkQZ\\\\\\\/X1RFApuzNuX0pxPont8wkZzoVM8nHB0s4Q9F624dq5gajKEfsIJUJwjiBvanuN6vttU7ghi2jzVqS5Qzw0pIIWNA3KhIypFha\\\\\\\/wu\\\\\\\/HuuN03LGx71nUxFM46B6uZSHlPNcBonq1yNg6vZfo7G8pDvkFvd4b9n3xt2sGzVQBLphy5SlO\\\\\\\/BFYM\\\\\\\/u88mc0T9oH62X6FNcWRnKnP6rzEpMSh91QG2mnc6xclhfkVSgYmF6FeAW0PvDaFElKWfHyq6B7wBCNSTV+1hA5VLab2duqmOeU2\\\\\\\/IH6J16xt8TvHK\\\\\\\/IdZhQ9cyvoZJ6nsJtRa4SOef1t+8gUWnWiFDDD7yrlb4AaqHZCzHU\\\\\\\/EkB4A0UZVOaOHxQ7G+JlaB14e4+xYnnMNskoyStw6xbkHmqyCkDR5VR4XPyPazHUyk\\\\\\\/5JzFie5gae+D5PAffO1WQrfeAMTTO\\\\\\\/1pGwBmAe4cYCVoYz3SVVcObjXG1ahsMONExqLTo1NDQ9SEGaVsNXU7MAfwlJvS6eTPL1AYCK2trWpPZ47tB91zVYEfg1O9uEHJb0O6V1a4FagtinS\\\\\\\/TTd++BjNRBirRAaFbIYy0dk5gij2FCtMj84knT2E1cwh0s+bkwhk4u2CTQi6yj8yW+\\\\\\\/laFq8Hjndm5tG6ax2ZeMiT6cYJ\\\\\\\/CxBfqzW8aX+bspBM2QFb3b9sI6Q4mE5m+DW0MXww91\\\\\\\/tHuJR4GK4PC0IQOqcykKzRTQdBLjTofLkJvi9EHEI2EzNbKrHELGKb37Ra8f9BfbqFkLz\\\\\\\/8YPQdVizRrK2ffRembBXsCfYC1RS20mWLO9LBREeREg2HMzrxKfG++nEY\\\\\\\/sYT0t7NKE+N1n8rIG17\\\\\\\/f8zcEzvJB81QRPhOEhAIMJFAt0fdWG9YbowE0mkFQemsQm+dqslyriuaLPuTNVtmlnrmHZ4pJ8mEEymxZGoqJEqpZHW66oZKCgXmUBC6xEjBVKuoRDxtQv8cqsKA\\\\\\\/E8b\\\\\\\/Yp7aLpGMpAYhsH0DHSdJcvIcADL1EqRem3LV6\\\\\\\/4VxD12KkhV9BrPp1b6aIYf6\\\\\\\/X\\\\\\\/ssF5N6rwjz\\\\\\\/pcapSrca9q\\\\\\\/XmSfbsLE\\\\\\\/VDPwDOD\\\\\\\/XMJAb988gxudnIvmnwYRDo2P5gOOF9LRvBg3scc7h\\\\\\\/nHkqFYokbXcaWVwtgpco2ggTg6MNUKp7xScNjNTTO8uSREmp3F2Cwqn5cekZaEfhrzajB++VAojq+CytVFC8B4RwuCBtDzTAptQYsFL8yKrP+H6tbXhFMiK4RZOs\\\\\\\/IH\\\\\\\/keI29Z6ZljRmTgFmPZFtcwssf+79XYNeriOf9tnrmSQEZQSO1IonE1qq2YO2gwQDJR5NfW18mAiMF3UyRiWum66Lhu1iaW+yi6LmOcocwPjeX2iU92ApRLtUF0FcQeGXrtOVdlKRQfVYo\\\\\\\/Az3yn8Gki7kxog3Axm9EjNYukwE6S2hRl6UCeqHcTHQ83xYnBBHIUhqsBgD1zdDd+eqvuEaBOvgd+DakhAxclihR\\\\\\\/aPdl06pmMMOmkO3K5KhSbb\\\\\\\/X+xwl9jridcVy\\\\\\\/H6pwEcTSef9LN5Ug4oi+3PRwNoIX3G8f3M7VXeWLKkJ2ckGiIu+nl4MF1NJyROeOGenVmxXxet7L82VmGFybV1DK8Y5GFjsznfK63Kl4yhdg4dOUxADjR87lhFpK+\\\\\\\/ejBQ6IpjS5M30ND5B5my+p2LgOP9Z5mAsqsTsF2qcQR1EvqL6wPMEVl6p5TCLZWz+quvIyD1cF1kblkngTNEYjYHfbRBPehvfj+PPyD1EPdk5kQ\\\\\\\/05LGmXkk619h7h4ML2x8e+LFQg5pjHrKH8JY2FPdeAXJY3q\\\\\\\/dN9h+Kbpxdg5rQlxxwmY8MUwpoX64b+Psph\\\\\\\/uNl7e1eTut\\\\\\\/2xuAYVe+6+rpnHfnYS6qtckYgcfSPeRN3O1IH8a35T+shCPp8aBt3TTQBQcnzgaw6a8h1qA8nkuE1+NhxwFTVp1ZNHTSIjy1pSsQE5C3A7vL9O1yMRM5RUZNlyBEoZ6R\\\\\\\/tkMC9Y\\\\\\\/SZg6xtJqSjoDTB2fUhNWVvS2GJIBtUmz+pbIGbsx3+hZ3oZD5Y9HEK9+Z3CPUe37mfYYn5joI\\\\\\\/8Wc9zLz\\\\\\\/x3PMlNV82mkKChIr5XC82qYxFgEHG6uRX2JiLFq25YtHyInYM8xy5VnOgS9MdVpZxjdt620Flw8BZ8\\\\\\\/fV\\\\\\\/w0Hw59YT6+AyT2lcDnINLGoc3HiuvGXzgDKq8I1XVqKRcVBe6hgwG\\\\\\\/CQLH37Qsskk4\\\\\\\/C+x7msEqfjxW2Djd4DdYx+qCYJGAhORH+VjnlzimZzy5F4ihiq06bP86q7H1\\\\\\\/8d6RyRoFirmg0PT+RzvH+v2PAtuBj6Mv91K8nLK8EGdCe5v9qGzC\\\\\\\/4bJD8GLNdmKweJ\\\\\\\/kKOsFvI7V9HEbvCmfQz+Z41rn53ryCb9UsdZ\\\\\\\/54W1XkKoB6cCAN76UD1sVu76nzHTDM0B7ASQ9SdrwATvvYXFzZVkXbhuXQVA60\\\\\\\/sJlivBB4Rb5s5V57kgktrLeUWsf85AXPEcLEbKpJ5wI6YQ34PaKNwIQrHi3CIycGSF5UjV2kQracl+guIrZT+tdLEoowFO94B2HY4eawd6iHLqHql+J\\\\\\\/9X+kZ\\\\\\\/aBC1RLYU32\\\\\\\/K1V7+4ScarBG9yJccn87dKy4RcOncwjP2\\\\\\\/TAhWJVDfTLyxWvV8b94Kzcc601c8PVP5d6SMttYo5HrmkavxjU+9vxfGBwdX7eVwsgMg+dM\\\\\\\/8RivIFfxEcxYXRkyO93p3js\\\\\\\/RnFKuCll+VFwExkXqOIcjnccGfbOp6JnRnyZ0+Lf5HS5vFMV1UdEAcmzSJz6fa5UrzPLGHnQt5kqxhB0XNR1A2Z3za2RlQmdBXSxBTTrrorEHyQ0P6J9pMPsEmwMBvvi1D3eyDPs8Tbq12AoZ08dem8ix0vsUMq1q2LJY9JFNGAt3ht3E\\\\\\\/D69r7qVZuspePhrUjmDYJTl\\\\\\\/z6y7nQ5lMRKdnZsJiRmFNxVo\\\\\\\/W31zHTs1VbYiI31t13XC3qm86FhVSS2HfYVe4cC1P0kyHzeijINH\\\\\\\/nUq8N76O12fnVBDELsITT4JSEjGeQmDiz533ZrYzqNgisJ75\\\\\\\/ZDpmc\\\\\\\/C7bwY+G3bOwsrL615OHqQwe4lgTCXdm3BfDxB9ZgzYk2tr8AS1P7wKEW4MOdsyWy0bsyWbBqR1KwBHuAQ+2cI78VfVdpELrn8C4+p0RRz3pwcV8xVEhWZ7Agrs\\\\\\\/KzHzLBf6RSNAhlaHmXCb5nHXYb3d\\\\\\\/HGJeym+K7UJGM5zpcn2Ab1O8jbtRfhlDv6p2YEp0TSpfeekQ0g4W2B+twgp8A95NSeIpsylr73+S19HofPXlINrRq4042rp+pLiiFevxApYSP7LHuMcgHe8hiXGxpmBmLrTEsIy6eWNh7br7+gLtf7Q6HwGNlBlNKZpsHuvkIYYSPa8Y0bF7fVjhYC6R2Vaig6Tc8x3FEkEQDee0tpfPQiF9L+y1Yu5pBO3fu0ySGjFugH22n1DOacn9gut8AGgbZtdxyBWuBoHxYdpXW5sV5XkQ3L+ueys0c+MAHb4\\\\\\\/cb2zXtB90XVndkwJdk4kINrcdhCshL\\\\\\\/a68b\\\\\\\/YhpA9AaHgrYxZwEOH+1mBjPL0qJ8QJj05Jb9OzsGX702T+pammSdzidX4aD0m02oVavevsd7zu8MYron6Gr5dBlJ6rdbCQi9n+qxmVnnWDv7j3h7CvPT\\\\\\\/uf73XzT6DfAIdNGTBzQ6DSyr9Ab92k3W4OMDKlg6KNdITRKbEQJ0qtFniW5r3ARzgGXqJ5m+5ONK3DIRyQ5n1heW4H2E4UkV+ZqOpsYO37jRhkCjiPRsKJvlGWewI4JDF2AKpUzrwnrwBgEem1V4Sc0DcOJ1ppQsGNIVXvKmDSzynx1K+iMj+thVyUqKYKk9cwv\\\\\\\/2BcVDaVF0CKQcULOyzNhhdfAYRhMOFyKjI\\\\\\\/nO31CA7cgLWlS5p5gMzlj9Uv+0TCaAzWL2hvitxuoPA9zDggPgDjGyPzFReJhD6G2Iy155pWIdmxDhBj0G02sEY3P8zMUN7ds1E9+4WWzzIjY\\\\\\\/pFiwjwqTSxdBqLppyFXz6Hr2TkJ\\\\\\\/rEijKgf+0TEqhFB6dXDSH9z4NDrTIZi4cOWFLFyHjEOg6\\\\\\\/Yi\\\\\\\/GXh0Lifdu6Sx4hIVvXbD3w7xVro2+64oMPtEHQ\\\\\\\/hGrSKRlEoiYV9SA7g1Cc5TWVFgHVOJDa0wsAQ1\\\\\\\/dN+mcyLRXsUpl14E5\\\\\\\/OGMohv1nTD7Tk\\\\\\\/ZdAunlBCNQW7u4eeNQwDrWIT8ghc5vyduWlot565AMPAXdDzpihiJBP\\\\\\\/BTiUKAwR0hDi117ZDQe9Ytzcz4L8ShajWj4SFyksj8C4flxyMPKUfShwRRoTPnji1gIpkQtndFKo2F0o3lc8\\\\\\\/i8tI0UUOAxVTEIZW+RjZvkKIStNGPq5TZkv5YVBBdYdHb\\\\\\\/WKh9SzVllr43BDFl612KCE8xyV5p5Ez\\\\\\\/UZ\\\\\\\/qqaDC8tRs24nWsGnso14OODN5XTEoRTiuAPvCzL0PiUHVnrgKTIE5otIaFpiaFCTqi1ik1zoa8zGM9Q2tud94kJ86GuM\\\\\\\/ZDW3eCNNBJtIUWHW1Riibwzv3pStycpGa99QWXfv4QjhX509WoRznq9X+Uu\\\\\\\/Ka3mPqlRZS5\\\\\\\/BAUI6h+6dLPBI59jLaIf\\\\\\\/tGwCKI7PTPJRUkxTBzpu9ez6LKCVxsqLgNtIAXpj8MqWSC1aItIM04gdW1R2bzQwhVWaxj1ww9i1bsmb0WoIwLhDP8s4t7fKjjPk7wvGAxBUT7pZiqBZrOpX2\\\\\\\/zZgmd\\\\\\\/oUc2gxHUov5d6i0ohSTb4m9IpXAzxDHQDLgyiHKRUioI5m5MAjDpENvNCDEaz7ySEjErYW1+toyQZfNGmy7QGkoa1yj9TjjBy9Se7P6UrUqTxyXTYcW\\\\\\\/FSvyUNUhs+7TLvia3210mVwikkCvJBVKyGaUjQcKT7MiNWy4xbSknRgw9eD\\\\\\\/l+kqIl3pkmZUqyCWgIs9gMXbhHrc8DovF2kf0fruz\\\\\\\/X10QFELqZ85pivu0RglDkNy7a1xAQXRidyExWzJJxdfaEGdOiIE4C7WxGg8wI2fNG7V+vv0y0CjKtZMflkLDb5w\\\\\\\/WTPrND1oTnGsl+zwCFVdqpJZG+LTnyMWYWcmSa4CD0XSJ6IFoSgaL0gP+19j9iuOc7yMPg8K8h8blgChqcQRA\\\\\\\/wZL1sE3AwYtuaRxPmDTfvR0\\\\\\\/OIM1JdNC1TQaD91i+cBs8cktrSJD2lzWHqX5HVmgrfLnMnyIQsfeK2MI7dUP3OXXLt2Le\\\\\\\/qYFDet3OlX+gYn0kE+rozSQLfF4JLckXTE7zP64KtD3CM9yNCT3LZUhUtBLGfy3vloVi\\\\\\\/8\\\\\\\/oFn7ilEsBuCAYUHbKuJfR7VG38L6MqWlpzqpELu8YBGEXafOkGR8Dzs4UOdMS1CvBbQQEphlREKv7g9ODsNJR+eP6QheYXSMOMaHvSslzIVV8Qv6UZsMZquB84eYHv\\\\\\\/xnbqVzm1cfdaEuNfOgHov0xkaqNWmRZEVZhj31iJT1S+6WoNh01Rw7FyztN+IZ8kPznm77inU2Zm6dxqI\\\\\\\/PDRq6F0k4JZAoF98Zt99YXzc6m7\\\\\\\/zOYDCjnOBGcgdM96ZBoncD5MVfkKwcuD7XvvYJZlk3zKfwEQwwoLli0xlsZJ5Q7dK3WyS2EA\\\\\\\/K6liIAZPe5HwBws0B7KdpPWjXrA10jc2WkYWHCZudyVSJQ3usR2JfTqNaxwy45RchqAk13\\\\\\\/cDnRCWLT1\\\\\\\/fWMX8GSUUIr5F+t8Bil1Y8EDLEGllPa8Fo59DN9xyvhqKET+pdTeRn5Ip4hEO4ticdL23xoUAwnP1Mcbi6imTwLJfYcucl0S+l\\\\\\\/1s7GxA671ofanWsbgBYev1HyCyM9yPppUcP1eCAw0AAjOtKgks8Kim4LTMdwtp\\\\\\\/\\\\\\\/tpnEeRun17En87c5OepESubhC99A2GonMvIu9QIBhQcV0NP9WN5SaJ8lQko5y4ZowE6LhyjNVY8QC6FCX2nCfMDz2j9R8Gcbhs5TgoqyDnDPhpJ2Nb6MhJvDVR1ZYE5Qb7KrHwRFCAt6teCl7iWAbI9l2qflLxXXi2jMEt9DPktemGR4CCtQTeldaIC3wmIEeWJj4VhCvvz24xZ\\\\\\\/UP9NKWkw1rQ1KnvTuhX2g8xyOzCq86ZE2gabtz4XWHstJr1ZouB0l0TO0adwOxKgEyUiEtL16Whih\\\\\\\/WypU3ute0P7J7gQvUjlrQ+6T2+OAMoF7TDjh3fmB2QKJMXLFR8MFmBI+eYinVR7cKVPWGznok95ugbc9NjZO014WpNlMmPidbCtE9ki8X8JD0bckYXuZhdHDWRUS7SgUPDlv4ccL3\\\\\\\/yNmWAJ7mYQbNoTjAPpl9YvMFXUhoTnP9YfAMIQq4\\\\\\\/mWnFXBZWIFRKhYPl6cvIYixhsVjOVnH2scQv6VsIo7GtFu6P8TEIoAd3gF2W1fJrNDsp1pu+tMVxuSvV2UJe8DHk9AnxYm5OelpOiRZHMUKCYaK8bUtrR\\\\\\\/akqElChxzO3l1KM92On8y5b\\\\\\\/EVxwIFTgVcfC0pjMqasTVjsFQgblS6cqpO9DFEKKYC9W6M7Eo9EPS9pNtJ04YX8TQ5ceLf5Tp0tohPLxnS0B+qkdu8kMc42VPxDwkPe2Se4YnEd1jp4WYh3sPMcoRbOLfrjw9SHIOokrYfwZoRFvOhKOoruKN9j5e7IfBQONTVCoTM6ko6GPgubjgwZi\\\\\\\/jsGX5j8f9tSoTpdGkfH0iLCp61qr5PMbS9eMSERykDOB4QkllobhKcUjhd6Sk6mg9Wp\\\\\\\/L8LH\\\\\\\/mNnNg9W4rCVAv1izuDUcPY3Y+10mM3sa+ATraa2gRvy2EzVjyGWdI\\\\\\\/uXr8NMlYT4+xzc0Z8ksE0ybWWLo0KmK0k0eI1bW20ZAmI1IMXe1QoF7hDoPu6i2QwKhPkH9OwlI4fh+vh2ca1EWkNl+F3vCfMJ4nT9DMV0iccDUygo5DNQOuGbakT7Ob60LTvS6seYs8oUPvSKPhTU3bGJjyfnfAiPL1nO6xLegu9jiH92LFy6Vi0SG1jjjU6gVbYjeK+igyeDy9WitatiE9JbKbsjClW579WP0Hk6KdbTF\\\\\\\/0zJB+tYpy5xkLXZfJF34B1xWhjYPcvcE5Q2chJ7qYvGOUr1v5\\\\\\\/LPtdUZMFdDrPoSXea2IEd07FLTdeMLExUe\\\\\\\/8VoQa+UmVAT5fmCqS\\\\\\\/0KpypZ6l1E1\\\\\\\/\\\\\\\/2HP6CLK9hcHkINqmO++MSheKjaEH2tGObJaAQhnbjxtwNwcXO0ATmzHpr5n9n19SApAOguQ6jVfcC4t4f1H7p2oBO2JFeez1AnvKXMXi+9TQUsnOhnBrHnFr9E7SERXOZnWaSm2R\\\\\\\/GdWC9iY4qptdOawavY4\\\\\\\/MUKmr1W\\\\\\\/0XsE7WZ55mi6xVXxWDRRQluWHXinbFp+s2MrzGsUlhDPaSdicjKgUMCuoA7xSLHs0fUvFlEX7brilS4AhUIoYRdI+xz0EcJwBDK6kStIf\\\\\\\/YRt4Op+8ZckcoSDFvAUR2hsj245S7QIKtp9NnQUKQb6Nj307jYCz2yMrjLpzUp841LZQFJ0\\\\\\\/F6uv2CFlaHKIlHu9zVIQvOGXb9KNiLSScrjd95SbtdfFBa3tplz\\\\\\\/Bo+knNBtwksXs0tU8kffIuHiryqjIf23SfIa4fGdqdl32nvnPAdgcY4DoUCnumAHFhMd8E7Q4iAV+WdPHeRdD9Xyt2UDIQHbpmnoIfUcQTBYoy\\\\\\\/cVGL3mV8PUgYzHLxPAVr50nBRqH7tLSZP2xdceLPxbG\\\\\\\/YMhjXcUbPhrLdxTrQIr3X9YHile46SBMv6aQzyaS99YJ+08w+gCNw4LLljy30mPXAIlx57zn2RvkUPqdvItJlYbjDqztXoKz2OuunQsaKSRZy2gnZr8Diyp93INKMjVtWgnNthXopAlv9GZmSogJk\\\\\\\/jgjz5LDGsMi1daCA0RFldEFVEaoQqb7TBoonsM4ZIaCQDvrosK8\\\\\\\/9U65DXr95\\\\\\\/TahmgoOZbHxkZPZ4ePuooM2RYpx8+YTCaDLhBRgq9Mc0DA2K0YqCR+wGvaf9IQVDBSrZ7MxeUNWlCN5S4xXtvetSbmbhmLvUJ\\\\\\\/77Rfpij+ze5SICS+4ixw52YQBpfleumdlTePKvPleFplM+vjbxHHRVx2TrJ9n74k0cf\\\\\\\/Q+qg21sHNeLQoi4aoOy+02xozILomC1Uey+Kwxgj\\\\\\\/O3Enyv4HcJAhu4L0OSx34IaVvDJr7Q+9ojXHDAcs\\\\\\\/uutJqNgIOk1MF+qLFlNSTmBLaQ8UABxyovnSKHAb7UuPvjW8BE4Vio9xf254I1BkFepV71vh1BAbALwWNXtR9J02wB80f\\\\\\\/kYLVYJWw5i9KypBrh7pMqY+3qKl4GNd6mrQMsX4GwQiN5+HGOR9vKMAYKZy7xhdhNH4REjXzHRgMooOqCNGbtC+cTUzOUH60gGYNyUz\\\\\\\/5kw6IvFbayetQGKn4Tuvy\\\\\\\/GaDGLEM36O1Z\\\\\\\/IR9Cha4jW4XOLt0gxurKzm7N3mS9GZbpT1vle3ed+u6qIPponJIzROMlIvoQ16UXfpUrKD+FIbJ\\\\\\\/xW3eCy1WBPLnoHpppi38ITUE\\\\\\\/99LXP\\\\\\\/PVwdW+b1iqWkFQePOVR5+46yXb8GZi530xYvMjpbtTUok5eG3YTNaRp2u\\\\\\\/0bvR1P9IoYZSY\\\\\\\/t+Z3L6PyyjKKokrdJEAWEmWhPu4pG64Xl6MFIHdyoAaJ8hQetPyLYZc+QQJd6j0irrGLGYMBTfniqOxcf5Onmgb+59GaByDjGeweKyGRiwTII7neR2FXRtFrc\\\\\\\/m546Vsd9KkFLlbMnWVZu6pEHPYk0sLnpmjo0uSd3lzUxE9EulP7ymgx\\\\\\\/f1iSCbZYCwWzuC6MCtSO\\\\\\\/delP5\\\\\\\/UwerbOA5++Wkn\\\\\\\/pyIm9tnJAl9ZADcaHcNZa1stX\\\\\\\/84kORK1bVvDVmqu\\\\\\\/VlQ+hJlUJxTEK0\\\\\\\/fmDtVzdWrWIBmWvJwyNCRN3p6S0Yr1DZrfCS+xPaACdqMWUj+yrxX30xERylhYDERlOZSpZSGtiKv6a0g5lVgkLFJjY8p4fYYVYImDfog7b0KCOx5oBkw96zq84c4td+Ss+0EUagw182ex2eYU4sTyeOMVSxPr76rLeD5vrAw4biGwqwcg6thLEi5Det1FJpqBfsxTTbib82rJMcmPfI2Q6akBRdzdSAZ6TSiWuebNsfm37q\\\\\\\/LGboxTH5GZ\\\\\\\/xfYIe\\\\\\\/u35tYdfG3tE7FSOtIT57fhS6iWeZyR7xe\\\\\\\/671P4KJcxoYJ\\\\\\\/5iW27M8KoRlB2Gifeu1R5deFV0Zhy9Kfps9Wp2wY+jaTI7PmDVRJ7PACrBcM9pQ0lr10kiBzxsakpHSo8hvsnK4Zd+EXFgdIWxx9ZfGX7JQgTxH5mMVn3kEc1rkkxJEES0K2HIp6C0tHHCXeCzB+bPOGnxfZbXK2JTCsUc46jaqW\\\\\\\/f4SlK3wrn3ei7V9TcdVAIxbv4+KPYRrNEZiaLeur8OCcnLZt+lEa9I1F4cIv1Sr2804SwovLkEHgRNFsmmA5ZA7vZqImjjqPQqF\\\\\\\/K+RY2CYJ+zx\\\\\\\/oxA4A3W9D\\\\\\\/gif8hu4E\\\\\\\/gVc8yXhrgmCUwBvgJubxnxySRGhtIUDUBHCbKTGPl9q7Zw6w088I7APM2MhxdJWMbc7cq\\\\\\\/q2cAUtjL+w1gCEUWH5GFkfMWqw3HgOrh1\\\\\\\/sHe1XJQVdxMeq9N1uRmRexVOs8EmFBeGM0cBU6NYBDxhed3mK68Uub7rYaEuQMBNALrtka5jhE4n1HvKLBC1YtDf2PqPVrQENfmazZme6IvoztGaxk2JuMy3qjJJVpW13GxZqAsuMIFBm+bfqKPoxM0ui9XvWlbceh6irOsUTfdrJc4pD+d6PG\\\\\\\/4emw7c6l9XNlIIRe1vqtWfAsCpRIgpnv99HzrkFlJLNdd9zO5e378k0SrzCztPLjdSEoL+\\\\\\\/7X+9XMc3pTKp9yDFA+HPmJQxs2dGn91KmOiw84KsNzdQ79krVBuDfWPyURSoO7cibCRvkR7+5jY+6Dl1ABa65IKBjldFYzxt9fZxi93EgeKd\\\\\\\/+PIHGLPK5rWC406xNbQd4cpAVbmRL8BgmIMxoi5q\\\\\\\/0FF6KZT2YvrTfmidcgU0oD+yOyJoAqEm5BwPLKwXR39prS79fg3zoQUFHsWCR1HMUzTCn4AwnSTY9dc\\\\\\\/WdlwijctVrwYNIUSnkEp1rCGMXs+5tTMKDTCVKVUmvRHv19TJB0+DYrkFJlwgZ0HzFZAXFccr\\\\\\\/lJYykn+h0Lrn74HZjMeyywZYwqK8EwjPmDtrxBCCfgmgXdW1QSOUQu8\\\\\\\/znVPg57uoveBX2oKIP\\\\\\\/up+LDJMEHf6JAFqQfSqwniIZsX5CxmqeVSYRb5VIotpgVHbTZDSCWGgRx9zx2+cjXRaNon2DGp9aORBTr\\\\\\\/tu2mOx68A2x9lyPXIr8AzschLZdpuVU6SoOfbUnRVm+FpCjP\\\\\\\/V35+mGRaKZ1V1qsfVE33TRzXzGxqfddMj60aSGyLpuLe13DwBkDg9ayL\\\\\\\/LoDPUJuQ5pbLkxYabwCWuD+K1tEnNm+yb0CdERz9jRRYxGrovVgiM9R7ter1dLUusk8P7Xq8H7Bmu7Z9SHxtJPcJNsQiOclEeMYE501ABVFb353qNXx8ckeZxamNekMRANB1AduUCZzGdG4zTpdawm4xkNM2ANLlVBumhMY\\\\\\\/flkiShqpHvHeBtEgfPXgOqUgvkWQalEhI3P9tZIBVEEKPnuQ+274VgKSJ42Dx\\\\\\\/NwSpkJSs3yfolma9V3paCYLomFOxWVclNxmS0wOC6S5V6BHuWSQp5sOlk1tWTHxJ9PDZubYsYfI0fZ+gpX9egenRbtoV8hsr1DXCONuG4Pnz8JpZV3PDqFqCvZwvoRI9Dysp0ZrI1txh\\\\\\\/IOl2R0NjIOMNhyeobvAQzfj1QI0\\\\\\\/\\\\\\\/e4lPiv86KwL67lndd+tHrnSPHjYRAiAImT8UdDrWTzoNEHXa42Ipvn6nogZPyBxYhVFXPUZzr0jc+KfOQZop04S+sTFuldMV2EJhYvICC8eey1aFyLP0xgSHYU1+qkyGNUvue6zdL0ZcjDYjFdARNpqGWOHwRrX+3wuAoCrWwv4Z0lsNaBol4Tda+JNAU4Qm16KXNPNySJd54a0NtcajON57i7pgAJYPxF6qzk3iV2mBCFT84z47no3ou3zABpCqjQi58DUrqzD4JC2sV6k34kVXEC17ei0FvknxHQxn6xS+htRP705pHX3ae2GVSKSvFlJY4KbDQ3Db9K6PgSIHLdI0O6HOh3oQVW+niWltN2B\\\\\\\/D3fJ19YGkI1GSDe7SOIwpOyuQi8uzaYvOsiCqXClmxCzoW39v\\\\\\\/MbCQi35EgypRpFjurvLCeL4DwE1L2sgVqfxL5zUUEHK+URd1I9dm+e30lHVDv9KRNZWJwOyFJFheXPFLHQo69L6KO3QD96jSa683hqbrS8Rh4uJ6QyOPQZQOWtBGH66ZszOisBsDxvCl0kKSV6fBehuwlh73kVjKWMcuz1NmfNJxvyA4EZOrZIu\\\\\\\/DHQU1bpV9jc+gIjTB9WC15rL4AfhyYBvP8U4A0CzOECc+c8mQLU5WHEfBscBfule5BoWVL67ngZpKL3cGDGKTKFtHRCUhgBiY\\\\\\\/9tzETzsWaQyn1WTb9fjlWkpwplz6mdTlyfaDyAKTaCF7XJma8uH\\\\\\\/V+FBa17xlvuXpEa43FVkN7FoXXWdtHm0d9R1hM9KO2cQBJk\\\\\\\/QzSm+Nfq37bQ2+uLX+enD1+IiWLwTlcX1upMYwLZV2FrCQZLrZhzzAHa0hT2Vw64gxda\\\\\\\/UDjQLwqcaHxd2uQuoNLLQaZMBotLgOblnEZn8CzBcKCaHDoYFaMaqa246LD4UsVZb8+0jlgVcpOOTofJ4AK5VXDA1dlV+i8POaSfmfzFRWl0uz9z6fy0nGSCT0kAaAfsU9GAFsnUOKcQQQoqcR1yiE6tP4StKPgGNtC1bHjMMCXdnI8+EWEHI9jPnXE+wdbddo6mEGcc\\\\\\\/pOSmGpIocDBUFOaYy7fFCqXiiWZNgNlBBzxyUr3zg8mdiXh0RJevhn1n4AO0\\\\\\\/79Vl0Rjtqp2AFlIQXw+xAw8RUTjpYOmG+2JLkgd5aMcl1cQCNuCTP3AphqDp+sxjP1F\\\\\\\/aJRc1UUDVkFPpjat6O1dTkqv1dkWYix8YJOw6o0g802clf4eNjoBfcBGLz4+EeItxROLNTdnvzR\\\\\\\/JXUR0DJ0eBrfPCRot3bjFAYrjvONzE7j5VWN3fSm0L6wMHvlsGQwqsWnhbPzqb\\\\\\\/H9GxXH82E+mAqv8aILXHTKusS2UcpJ4otjk6NaZvnagNMZ6VWzbqwGbJ4NHIpIOS2sZ3h43qYAC30lmkljptU4\\\\\\\/kbWBAUGpIYFzZRYAHoGEN7Y4YtkhSi90rBQPqfB4QzKmOJfLIXdpzMABBcova8sEpmlZkEeTBRD4bbxx3soh5uI9Rkkcb\\\\\\\/3IMiaVoDjmgfy9ed+XyCmaTHA4kDpbqLdrAYLUg676shdBPFqOoe1QkYusL\\\\\\\/nciBYyIROr1fPsCdxTG0Fy1L7E5CgMixKZBIe4yh\\\\\\\/5yiPm+\\\\\\\/chq4qc+Xe669ek2tYPSZrgXcfChxkobwlTdezGxAYHvrMku1YJkRFxFG\\\\\\\/iSULjo4ChuwXl51QHdarqlgnlIBZ9FO4mRpxqmBOiJ7T5FtnnE8E0SRv1sxsLy8tczICapVq8hmvQtFzZg5nMEw5VeJUvreyf30hKKBnLSiH2w6nI\\\\\\\/KBKk3KqUKTaiJj+TZyBnBzVC5aNFzmhM1pYcv54sZjYscNoJzFODcTeSPGcooiWc0TuElm3lQvAcklIK9PxugfDreqrxljXlBrK\\\\\\\/\\\\\\\/J4SO3uZEvLLm7e2g\\\\\\\/VcAn7Nn6MXzQzSBRmJD56gDIKaUpA9t4dmOw3lJO5fxNvW0HOklOZ9ZYhAX7tritGUfrwM5iunH5giswq8IpsyqNhgOgB\\\\\\\/ROYHs4VOTBroRecWAA+fuE2446Yav7UYPn2RkcZJkfZNW1ou\\\\\\\/m4sMe6D+byTr3zOXB\\\\\\\/1iMAS5Fkm3F6xfzzEJ4luc7s6lq4ZBeze4PnC+0OjkslYGgjgwmQeFQt1n3NxSxH+edM9cyqLXhmJp68Cgzf8vsM7SVVKB32XLzba2JCUCI7O0\\\\\\\/RZGwv3BWoblKucH\\\\\\\/W6uw+Dx2bopc7iiFjwD4xpcnnAjfPCczfDZXms4zQQ2VTNUQWcOzySTroBM9LZZicKVGQnWeHDBDkisewHpGfdWqiEzx\\\\\\\/B2O87gT7PEer\\\\\\\/cg5LxoUVNPN1gt1tOg4e2sC3jW4QzIOnTaxoQzARFj+j9SDRfGcCiciyCsaUk3Ne0ULi5P1C2YARIUrDdKvwlOxRXqPUwTg5D9oZpZe6kXoi9CBVKUnkaLTTWDeuNN4oQ3bgkiKtsHEl5dKcVqpECKLEA1\\\\\\\/HrLRIBu4khF\\\\\\\/\\\\\\\/TPzAunb1D7ZRSFYxUvGun3X1kH2chnDUch649PEonjVIcZCerFCyFEY7u6eShy86FUbHa3TU+rltZ5KT8rW7KIyIQruvIbOUIQObAWmY4wp0Ck0u3BOPUQ5H+JeTefh5dDxAE+tRNVa2yJB1hQC\\\\\\\/bRchWBAPkWCrJtMxoDrMB4f5+Yrgyxc2mctaNxb74Kr\\\\\\\/7R\\\\\\\/4VykfE8WQpb64x0f1Fk2Lh98NGNOYRxAJvPlvH0DEDMA1WcTppujKGai\\\\\\\/e2inaOcMQwTyIzNb6v17cVt\\\\\\\/qOZ2A1j9H+ZjbVLP14nhHBTuZr2XzUXExulXmkHeNdMIp1uZN7n3MIVu26Qq+BqKAhjhSgXppF\\\\\\\/GPflE0M7QapAjLMx\\\\\\\/9\\\\\\\/SxCRfFityuYc2qWQ\\\\\\\/\\\\\\\/7WD7jFG3bkL8jkFkBPL7Fep2etvYYuFIoZ64ukIH69JvwrOVtNiCPmae2ar+IeMepG03OgGTCxxokjtONt6i3n22Rx51wOlM0rskmcIhfKG6WvEtPCFyEDghkn3GQLKRAyENmQNVWBzmqM++JicjbfwvHexyfZVPdOkzgV2SSCnNHmB2tw6U+2c9lxotQyQ4nUtNaRuHdiwPWwZd7lZ3wWxEngHbX5Ye+faX5k\\\\\\\/U8nMN4MGwkZIrHTAiLQ1o4Olxu+6rrYpSf3aAeI9348a2gk5sGFaY1W33L5qWQtqV490qBV36JfMnT7jNTJjs90PF3brREh+EbTMrPceyAYt6cRr\\\\\\\/vfnqfabu5Box36Om89oapRxa5vgMPKbwAphfgTUsCF\\\\\\\/sew8eSEyHWsksqm8LNCjP7dkBOyISFmLG9FNmrLFygpSgmxSmyAmJpTzu8oTuYdFnRlYhfsmgYxSM9vW\\\\\\\/lkiflmHFEWb+g4crUg4JkuOKajaEt7MrCzWS5pDkMLokISY9lsQDoYnHE+bNpYL34xvivmehfovzLG01nbHDuUtZu3XTQpr4Sh0olxTNCD1McHSJFiBhTYeupdzsTVqlp1lhJ4+K0ad0xIkzQGSam96DTl1CjVD2No4T2mxn9FBrn0MlfK+EiUlRzr2NQcT5mF9GQsATHLYUYnKA5t6f8mGxROQj3vGwdiJyX+SL0AtJIJR3DuNmSANn5b3P83lfthaqClYaJiRJ95SsAdrFRZ57nYFthKqlmCtlkISnCSbCi7jW3cIZafWTmibyIOQ3MEvg4p8Kos9+fmO1ERDYSlYuG8FX3yVmAWiUuYyv6TQYJRX8UmR6FNCsZZT9uSuOn9agvKYQBeckW9j4ao8MoiBxw0hLWpOIrB6kTHGbWCE7U0Jn9NrD8J23LoKvTY1emQVjleTuPcAgoIwSgeejsZ4Q\\\\\\\/nNBiWIrD+khx\\\\\\\/bdBw2oqhLNQO+RvuMMc6SSDkgZ8NPxFSrvOZMMgEEzcZ0H4ItQ7tCJvKjypO2MIhtOdclgD1EFD9WckenOtc5WtT1MuJDtP0361buB7uS837FJztaIndIzOoWVWYyIaL3pRm78nz4Q2Tk9kLoYe+eifCGN+QVHfl6AScosxVW01CqS6CBExE0s86JSXr9kg0d40+J7mRhqumATB+mQ9wXuSKC7GhS1e77NFJFmkEB0l\\\\\\\/QPsDvH+V8ZsQMF\\\\\\\/VHnqD+gJeSXrQ4TAxpE\\\\\\\/1mKdRvqQYP6dg5jloUa2QJ5Lp1R1\\\\\\\/uuSdClQovOu2EbKKKfRjAahA0TbVtsiRcmMtXeeiJIXHYtCYa8Y+AOMrXXlKpJf\\\\\\\/o0cUGpyMgGtRWDj4o2NG0twxXhVWNcfoO0sWSvj3ZgfJbO66gbY7ZSlsa4JQJX7Qy7X6W1rVWKlBwOLGbDwvYZLQyebR9HL8FnzQnYS3j5EyXLlSd5ihnrkw6tMdA+DLKxpSHPzIrZcceTLNzvAthIkeVfrIwTMdHTc6Xo3RZoeloyB6O433qNBftpEqOUZ8ftCjtYJChcE38qOWbkD86CgAAuwiT4Ohj1XVwN42Apjfj9aZ1EWM6FXAleN2It\\\\\\\/w0CjU179xTEk\\\\\\\/CPNhK3RVfKCs\\\\\\\/by+jjWt3tZrTMF5uR9KYbu+9PGNTmPrKzxqTi2vsckJKHMOptRwksjMorqlOgxHRL2rl7Yl3ProCZ\\\\\\\/H\\\\\\\/Ju4oXn5gPsz\\\\\\\/lpwX+RalEihP2vgKoN8mRWqA4SAYfe8aavMh4SGl+mb9Qc9AQ7aGVADoXl0Gu5EmEOHqdb1kNx1glKjkySzlo2T1RuRW\\\\\\\/NlX6FutYx+Br+3hggcIGwfM7gO3gXu9hWs4t\\\\\\\/1TVY+ETcFppiz4gDWfxu2ALoJj7j9K++sZ5lT8rIQHzlmFcrWSVO9TLeRCZywOutVpZpQKbEtZ73DctK\\\\\\\/iuwI9HS0XpMRdAcERmSlWxE0CebC1K7Bj9pNKEP\\\\\\\/FsxkolGxYQuqhLHR07Q1NRn444NFgCH5v54Beu4BgBCP6bhBaaHYm730w3I3YNvJRWxLuw1Q7BdwL4JddMLsmpDEb35CGcDdvv0WpDJbo8At2YDRnGwdrb83YvjPr1gKcoCXDcKxpU0kEjSQcpteGp2d+EcdrIPyMrKy8Mz3d5LGPmHR0AxFqQENptuq9Ex8wPgKJbc\\\\\\\/9nYfAf6dKr+Ea\\\\\\\/TBdwBy5cFh8OAP5Uqq5aqMRLBBhbgXCk7Im\\\\\\\/SBNaIBKkMoYJq+RXloqUDQOL31TesFzXeXdbdYb78IEYRBwVWDJAc78vc6H9135FdF5u5Jd\\\\\\\/CZhYMZ6oTJzGRYN0bhNcUjx\\\\\\\/NeJEbxUO8fCIuJdklARLn9xUnm2zE9pOcXizLGLTmNvSntT\\\\\\\/zyjp+LOKMXq5ZPKPyrKZ5w5ESDqp0KBZj9DhF\\\\\\\/hr+1t9WiZb07ZI04OVl\\\\\\\/4gX4eHHHqnm2UJnRA\\\\\\\/f21O8A1mt3vH4B+pHNtIdFi\\\\\\\/p2lfWR\\\\\\\/MuQW+QOQBREtJbmqAgV+u0\\\\\\\/x6kVi+b8VuE1Is5lBD3dYgE3OEfpBcgSur3pa\\\\\\\/8SuNEmTe\\\\\\\/jLPqhsqcDg1mqXOzUBbtm6gm6oaCaPkMsKrXBipc0kwkJLHhHF2a51TiBbuz1ToyGxSlT6IRIn3Fh4ZVDNjEXFqOAdZnC68jMhW1EKxW\\\\\\\/GZA3bGhKHEu5b0SzoGQt1oWaSLZC6R5Fb7DsV\\\\\\\/RFTSbQeYgN5baEnka0sjBM4dIg0cu8inu6RnDUSA4vwr9lxYz1d7vYC\\\\\\\/6sQ7eX8cHWkfxZD+baDQc50XSJZP4Cb5nkxOfSRgQxiXuQQLJm\\\\\\\/0D6Ud6yAKNX68xeioTNAMRWKnofTOkzhgmEsN49FmVzulIBav4S\\\\\\\/XAT3bJ2BvwSfoT1NcPzjHZ0YhovfPtoxnf84HE0e0A4gzZOH+yn0lX\\\\\\\/7U2E2Wr4S1gzn9waM0EgdqqCQPiF3ALOD2KRCJyE71cVwHeicxmPhOrdiUvkp939iVKk6v4nU2aJWUugdHYvuGkJJGx56bJJgf3cua\\\\\\\/ptEVI5wZlBIIvIEVg4bEhpZjzTwsr+AKh3krlnIASsGSAwa2tMeX42i\\\\\\\/dOaQxi2AZbL\\\\\\\/W\\\\\\\/xXjXwqF2PW3O5PFV3a0TTkqnM9XJFsY7PZq6oXpAqnsbZAEo4zf8yu1h8g6yH9rpmlcbrFCKasfo+R3HXBvi1rmi2\\\\\\\/ATebTwuuGsDzSE41NJeLZ3TjHS+dQy6m6qGCVJFG0EZMuvgaeRvIUPbiiqFyl6ASp4+yaISRiBalLZJCQnvZZnrQlqLzA1kl1GE\\\\\\\/tS8RGj4G5Rggrak3TmFZvjKVd6q18hCigbaZ3Or9Ezage5DldLKHTRRArz3SwCmYqZWeYhKkMN9nTvGueJTNF6ac3l2puQuTBTyayx6epL4umTuSpfWroM1m+Hze4VBRyI0IpVl1EJTDAtoopaR3xaQAEqMuT86KnkOlaeNBeJkUrAR0tC7SHnm3xueVbhxGLVnvFkMTXsZrwmvb0zrxhb435drpWMNzcCNawrQf48P+ZfuzxpV6XpWE3LOWLQa4oLHnAo8nCPrGeNr01K++Zu4XS76S78ahN9EjU47ABfESthbcx83hDqfi\\\\\\\/XvUPFg5ZacM82DVDA\\\\\\\/+hgsfaUr+k8vnqlaYxp6VGJxGb\\\\\\\/ScJIriPsda1VDcg7KkPC\\\\\\\/89PBqMDEn+ow6cPBd01L\\\\\\\/2ZeCPXVJj7dqEM7pGSnXNAOopv+n7HV06baqHNWYC9qsdGKIxzjxoOcmodiGvbjCd+0Xy03\\\\\\\/7ys65VbhJObeFFwl1PbmvnEeEIUA5pEenygv0+iHTo8hlBVd5iqoajVd3lp+7\\\\\\\/1gkgI5rD9379+sO2g8UWT5ovoM7lcll9Av9FZyeYgACk7A+ID\\\\\\\/MGeYkJTuvItsFGoFuw0aAV7aIgvwqyWeiIy5Zek3kuwrhEID58s8R1th2qWfaokr6W1QPcLAgTVcvsBjjLYg0aceT26s0lKtMe5AzEucp8Oy\\\\\\\/e0rYoKhWOvR28PwmoghGTTeMFBf9MCC+SPoNCrfSweWQUXFxRNiJGnXZ+jAAT4paWYfCqChoBYfqMzmwI2yA\\\\\\\/EPGm52XfkX25Xo6k8B\\\\\\\/iD\\\\\\\/2yBsws1aEUIsmqa3hT646drtL80lyK6mLmZ3sKAxVePVkR+QuRPb30DHs2nosgNiOKx7r4DtEODvIoICxEw0WLgoBcGmZnfuVfP8iTwocXCCsASp6jZ5jO4e6xfyPTDL5OXWIDhHM5A3KUgcNwqbfv86fIsKEhRApvZlNiSfd+ZDNc8pbRCP\\\\\\\/MbuMCv3\\\\\\\/JiBLXGGPs3nXU6fSW4d0XOw4i33bfyh8hjGxDYrXnKHSw3fkkC5QypFMhNA4\\\\\\\/LMa7Va8AcV1KroVDfPqxVEjnuRoI6y8EXaTe1N8DRfi7\\\\\\\/WBLtbj+AWmjPNKoRuXZT7Em3kMzLao1ft4D0YwPa05728OT6G0vHlPztuHgcBjx7hrluLI+5Kyij9fKeY9vzVnKVN3Gh3Q6GMI31MQMSiAJKzY2ONNlNN24wnjqd2tbOQZJdpCQN6zHPOyJqOR4l1fEhkcOQvemhWM+bxG+SjV1\\\\\\\/+jBfBfFlTefUiugai5opREQQcFlJzcJmgv08RD3V7dJ52pdFZ0yTm\\\\\\\/obxQXW1nFCI6q2hgXT7wVYLl4NI3krAzFqiGgul5YhWFzEt209VyRpKv6LxLC4dE5x91ZxPcBu0k9ECcGg+juELB+quGzQGZbn8S8RndmK\\\\\\\/JZxFznk8cHskMxNrKh3qumha3w66NRHxNpYv4KiqH89pzlNzQXDV4Q7XjdVvcMw4VUHH5ynIpw+PJMdHf434IhFCQGwPAkbe333G++8Lg4xyjgQfx0R5edvD7zFNKzOnlRFaQkvVhnQ8XW8xlnzaVpLM7V\\\\\\\/XVFGZ9DWxVrewNdsNGXNQelD6UFW3yLADBYfaKb2Ww4JqKdFuGjzDHS3a9uM1KPEr1FsqJJe5D3q3pPu76\\\\\\\/9F\\\\\\\/IJjhiyBtqbx4YFeOMpRqFkCK8onqjfj+axYztWt8UIPnpMf7krhjlHwJmO\\\\\\\/vnd7h\\\\\\\/bSTsHFZkWgUpjBtij4IPeuV3Z5cMKJJ+TyImPR8jdObmWGVtKLRHrsPLrf+kn\\\\\\\/iYrsf8lMNWAx2jCFbzLW2Ef9WXzj5rz6cDKaCWeClqBDcCIhnDau1iIp0JtfGPhDvI\\\\\\\/dMItw+hAhZ4ypNLH459YvsGnDGEmX1PvBFnCdxxenwY8wLJUPD6ZF35zIkvLuVoHhBK0ceNL2R7VAUBIrYYpxBIXjJy3pO2t5136WqQjBr3T7lTmkN9MT\\\\\\\/jz3sdl8vUloFvukkH1IOHOBKyBTeXQWNWoklJZZZafg8mNO2KFwL\\\\\\\/OVXLZUu28yLD2zy7UpUp+wkEMzJsxKqs\\\\\\\/guw91U1Fetn2L3JoGc45CpaKytA+SlGp5YamtAxMNuYy4N6l4919sUDVhI5LklmGKI4cUyUj79J8zLBEq7AoqdzBWrktiuAT0hnFeDMG8tqL1KGL6Iqj5jKVP+nD3KploYpCCdQ9CRmvlkRx4GayygFb98zJlEpmk7Zlds8L4ZgWhmUjSD4zzFRuKWGlnlQt32T8fLkl8DgikQSezZ5cyDDPYtlNgFZAkts+zHeqyL08UcdzLaVmyKYQ2XDSt9LRSr7RybY06GfrcMOqN1YPugU\\\\\\\/9F+pcTk8IAwiV2RvBBkbRl\\\\\\\/5Fev6n3D4klxQK7jgjC+wUqM7oLhbjYGWvXI54XNLUPF\\\\\\\/kbyIldgiZUAMv\\\\\\\/aplJVBY4wYIpQJ3bOXw0D9BLMg35UHf18a81FNqQSKHnNT\\\\\\\/AB+a0MR3DEQDZeT+E8YfJLVTxdA9JaXvt2ZZJsdOZQ\\\\\\\/mOCXkCHrzad+U691pzdCEaUR4CWE9XsHIM1DSCIbo9+MAMBipGTe+1WsYsR+X2dDm5YNkBbo1LQug7Ord9l9ENCia8qCbdUjf+wJ+kDVq8rVR+Azrr99RlY4cuX8Ax\\\\\\\/PxKb89XgDaG5UxuRAZuUZK\\\\\\\/vnd7LBtWNX1kSkIkUggrJCKXMfMl3FtxXF6ZmW6Na7SFN2O+3RXHY17XIw4UOGLG1Ix31SzgG\\\\\\\/S3rJmyr4shW6gfOsCJJxS0Ad8izJfN21I\\\\\\\/9uMCBLvx\\\\\\\/ZyA\\\\\\\/nRq9RNmJiuzfSa12nAU581Jk1EyFBMTWIkxK2K4XrhCLO2IlsQPZEIiIKJp1niLkGiyH4TAZAHYLomRnEa5bMwwx\\\\\\\/8EOM0nj0juqOG\\\\\\\/quZ7ZjOZIgcR2XwMHp5tjCMsfInmE8UdpWQTOH21O+rkB5w9NFeA3osF+\\\\\\\/udU+9ZLDw3WFOHdGSPONufi4OxGEpXiw8s7cT\\\\\\\/BnuP+pWCbsF8CDlmx14XXuu5I\\\\\\\/82o\\\\\\\/VI+OO9Tq60enVU5fGtyWzTQ64aKbJYvtpxm3vXwoeL+IVkyoB11Moe0gWptPPX7PvK+Ip08pcKnsXWuO238V3otmyFbM+KLNrC8KvTwvdG3WSV5mDd\\\\\\\/f9vbtpUNj91kGUv6ZpbMHHsQdtOwYBbynbcF2Ev\\\\\\\/sNEsJYNv7Cp0rZU8ZMHg3bO0s6iwpLxkSS+Kbduu2JynricfqMt9P0xkmCD5jXM0hi5xYhypUWwG8XNJXG\\\\\\\/ta\\\\\\\/iivDLVKzyHqzM802z9zs05Dq4JNqc0aXPmAjlxK8paVxI\\\\\\\/A9DQvxBMukE4rJO\\\\\\\/eXPzIqFE8cJsj6fZtWokpGq\\\\\\\/9vvMTdTPSoJcg3ULJJ0YaDy0ijscmDcz4\\\\\\\/EeoJDh4bnIF6VD0zPs2OJJHCKWASqQ2tL25SUXcGCseq6RURKoNaxbLtg9BT4BTbvjdMi1wWhKD4v1iR0SqyKSdRv80wv8eBogOj6G42WjxdX+HVmI\\\\\\\/+0+sV16JSVT7+4JdZ32qgx9olsZdT82la9DKMnrS80xeSi+Ik6fNFM9VXS\\\\\\\/jzlZ1oEAYWBFbSW6fMHCAjOVWFKQzLatAcAUKX8d1+8WSKyNiKAurlOJWFS5uzYqdigSqSKIt\\\\\\\/MvNwbl1I48L7sMBQuc9rJh24T7Z9v4SLdnCNY+ohKsBRHP37i6quLCT0yopHXk5w0CaZqquH5Am6EncH9YW7NyWt03KCmx654xshshMaXDFvxdDR4zf38uKhFrsNWYlfX\\\\\\\/kzGs2UW3pRF\\\\\\\/n42pLJ1Ng13Rpmq86yg+Ie8\\\\\\\/vmbff7JFjI3qw1BHlXjRl3ZREBnl11rOQ+tVYNa8Qh9QGtTznL96H42OVNjumUVPubNH1vIe+zRWbEtyxpDj3Z599TPiOcCsp3Rpcf9eDf1vPVUo3ykPeWEkznKH6k3X3GYLYJ6oEdqFnIq0VkNy3vXh2QkI2xyxP8HWewShjDF5ST5Vt9azN7QH3pE9P5vIJGIlCSbjC7cMMMwj+Eyk0t9SmRAGSVAhE\\\\\\\/RttTT0teCe2xYRTuNO\\\\\\\/DkjiNlZR+zGLkhDqkUJ1HzOebDkmJRHwYWO2JuDDNctx7Wx7WPR\\\\\\\/iVWVYuX2tKOCX7G8yteKPsD3Eym22nc9fkyjko4GqUh4Vp4dR38wt\\\\\\\/U9bLBQlAaeNg8qS8SwTnInNFvW+YEueoVLEx8lSPmdiWzuKuKT7f7kadhv4oQldHRSOLJxSA7iuz4HU5oQgvCP\\\\\\\/wIUUbkExE7fzfyKkZKQLBst9WmRAftv50vPhquroujnVIPq+cxxnaWiDJLLbKxwegN5ef9zYq0c3Rzm4gZN6i6wPc49OtOMltXbgxDsBpgP1R0dcMwsZUJerErOM3uC3B7zNIDzxA1BuWo+ngopuQaDJlLskwi8coG367tMoLW3\\\\\\\/EF1OQkdakiBVQzywbPp+Y\\\\\\\/LqyAJBgG4ym5uDFcF9U98zsvDf9JAUIcOXS+JJczlTvKY+diLgiURqeLKda52htGl8Z+HGzKqoWvqb6D0ESYcM0KpSn0\\\\\\\/1+aicnheahk\\\\\\\/F2B88ospSsO2rt7K24wI6oZCgKGJj0ZLa6GzTSavMl4nXxHdPz5Xxp4aJjcb+CqRz41BpiNNtb8DOe8ZhDj+Eov7YLBAd9szp\\\\\\\/Pa69ETCr5Ce1eJX++uLZVtNvIDo5vmmuIJWNrIJ4O07maTd\\\\\\\/FACuZMj86Zs9lDHIfCCR1SbtxzVIS7ucRj5KXqVlKKQ5rKSuob\\\\\\\/GYlchp3gktN0xp421k2zAepiFzstJnT6DgSqy0furSFyKNnTVsfI+TfKWd0DMKQ0VBWnyEfGsgKUTprvwcCYmCQQT2f8g8eQnC\\\\\\\/atT2LwqCM+EreUzoFMicvTe5qK7NocubhcQVgRNYmTygaAjpBabzB\\\\\\\/wAFtuMD5NR\\\\\\\/\\\\\\\/VfFxqxDUHSDDIvL4100AeZDm5zkPW\\\\\\\/wSytUufsS7\\\\\\\/OKYiv7NYGkN5wvajeZr7TzVxmz1NAlr2LQi\\\\\\\/F66HSDBf9H8XI5R0lsgGb5TQDhO47Cv\\\\\\\/0izIbzlYqiQmUdpNLkmLofNQV1l\\\\\\\/LMkuA\\\\\\\/8FLeJuwCpqHkLnGuURl5XBAqlyTq5v2FwRX9gcgnKHK8uGnQQ+6n1pkON32OnUe+quP34UL18Q1mCYU\\\\\\\/Dj9a0kJYuUIfBRstVPwi2Vg3kjaftKqMXFKKusaooZkxZvt0Kus5+s+VMSp+8jJGQYBpumIwtwJAa\\\\\\\/JY7eL4TwAJzQGrbB2q+i1ktCX9RqzVejYO5yVfxXtYqHs81OLDyRkJLGLYGP8iMdka+Ur0edq2YuadD6kqdzD8iDPpDzKoxG6AF30G397XSB4IZiRbv\\\\\\\/2DI5oaQBQkr+H9yyjEelump4lsL0vLMl6rZ18PRYYTRM17Q8Oxmi4T0rPk1l044zghxaNQ8doztkzIHxfOvD4+HWo9IUO311r0DIOo8pLmHHwFMhwgJ0L42H4BfoZm\\\\\\\/BBu6oZx+ikrDF25g4N91d1J1Uhy3r5txuBljJdZs84NmOgV\\\\\\\/wuoCNt0mjmvP0kX\\\\\\\/5tJwhAvpbcsDvwtAH\\\\\\\/i0+GrkIdrIilJh0n9JmPRUlaP9j\\\\\\\/TaCkhDysrepOcJHjJHwhFd0a\\\\\\\/C2Vd6oMluOuq5+Kcaz31vnYtKzxdceCD8jNJH7M7bxlT7FZm4X2GL0G5XKDI28ZhkHzKGWYGXUyB3kimqNMzgvfZUZzGTfNwBMeueQ7k5EhI0u16yF2eD0RXvLcjECa6gK3gtI3ZM3oLpcxYJpvua2ZHnym8bUNkH8fevhorcPbtTB1xkknjp7gHjRg2JbUpkhoqf8HN5JhckwVWKuO\\\\\\\/zQfTPMGZsdeu+pCeZe2H+cel7tHc0PAhg8NW5XnPmTD2kPnfAtWMw8ahS4v+bNIjuHtVAAuD7GSunRA0deGg0FApGxXuPYOzOnoII62rNpgWD5zJZ0gOcQC6CzwtIyGtHro9nJpu\\\\\\\/7GFBOFes\\\\\\\/zwsmsuztaxA78uJN4BEhcA1xt96Sr4a13g9OwNzgot4pQnaKNmu\\\\\\\/AJz+C712JPnC68nAjiMIYOAqPAZ\\\\\\\/krIUHFUpJieHcyFoUIUhUZavW\\\\\\\/wIOpNDhARiikyH7AlLGn3y3ehBsi8N7OMZqjyRCji4HdflIpxs+a8Al2wRDGfMBgpV0tm4S1MPVoawPW5tLiMH5QXecYqmKaFn4DHYIulNT6gK8pY2jz0MN2DhA5vhOmaIehNdVEo\\\\\\\/u7poyggf3edVGDRQE9vkqlzr6qaxeFcnjizMzHem\\\\\\\/ejWgMLIRJH9ji7fDyBL\\\\\\\/J\\\\\\\/TdQpRC0bSTnGmoX1DDPLsdTz4zas73bu7ggiRQGI2xuvAX0sy81iuROEZfb92kod0L2W\\\\\\\/C7Dfvu\\\\\\\/jPyYO54MSMqVDe+dSrw77fjSSJrwMDeWj7gS3SKe8S+VK8GHW4dHVAFLfr4xtoyROqmE\\\\\\\/EPaR23LjDuUooeZoEdVUT7yWiiZqk4TpLP891dvlf1IpHcgrw3SrmnobygtbVEGjJj0kGsqbb88WKWqg6Yg4xzd6s00xXFYyp6z04tBuGAZP+C6D9ZAl\\\\\\\/x5ubDjZlQcKSupnV2Xz0K8k483NtEadyvD6kzezZchknDeh824+ESLigG95pqz043nDQKKUDC9C8xSPDNRgAS3zjY6Sz4jH2X0terHVkd+U49EY9QEftlXvifz3o+thVPOwJosCX4XhYlX+Rksi1ml9xkzZKvwFCKr0AQiNF7wshbVmCPdIE3p6cXWf0vYWhTxc4wyY\\\\\\\/4GO1Jz8WeWpPaVtAfdmcAwEHKSk2SSRGNBsqOegmkoXHahyWOnGLfZxeU5PrKi8qijFZvEEoqExsLPrrGEshXHrR0flzyhsVCcSGf3\\\\\\\/SvIYD0DX5kB68DQONfMMsZz4dKb+ThF8FF3ERFb6HKunpYnXp2lLNq4m+GQ8MLatiN1MRf8EqEWClgVKZ1\\\\\\\/Zi6GPdXyAIE0nGwGTFAQYXwt6VlviZ4QOVmoLf0svTu77j4FomD\\\\\\\/1VqyDXwZyKAGjgvhwi7djo+ZopKbStcj8\\\\\\\/gGrbFYE2022qSL9D77l+4lQzhmzB46WrlPOQA6JXJ9XD9OWPTp3i51nHYgGbDudEAvEa73nhYRbIMcxEdm3DwBtT34r9ZEn4WDPDn96eVFXg+pZpynxKT2j5MyHCRUII\\\\\\\/e5firJQRAvysaWto\\\\\\\/13Iyf3tlQotCOJSocf0wH8tsvHhurpbmrcnAkCXwZCKodz7Be9Pu3BpFGOGsohrjoGycb+knFNHp6OsvvXHFG4iF2WJ9Tga0eLBJeCn+liRP+UVFkHnQ7iT4gFQYompVM3006pwo7FEe4rqkVSnJ29ANZEgufeZ7jvuLqkB8hrebXd8QrCK42\\\\\\\/KenTFgXzBRahF3KlJWK1I85EMqXrP77lDmvkZq1wLjPo5X9bxV3E9hcKIfUb2LTlcLefe+uyLLKeuvbKET\\\\\\\/bOdfTWUM\\\\\\\/OAx8Dn5BkrgiEgZRHZn517T8glJHtnKgnen\\\\\\\/Ka5yl8bbiEsGFZk3dFBOBjN7AK7nyuS\\\\\\\/cDngflEgO19gVR2HMK8nnXTOZB767oPwHo+LPfOpfcMoQ\\\\\\\/MLEawVWLTnM1E6ZU\\\\\\\/TIX3oEx4IggiKZRa6e6UlbtOYnhdDwQl7D3QVJ7u4D23C4lIxAWbqW\\\\\\\/zoDUWR4X1LldxRbXZyp0X4Z7BOx1jgW55plyy2ueb7u4wJi4Kk6SM7y0WaFFhRPy50Wi7l59jbqVUgRN\\\\\\\/D1DpPuWBsOsRF2iGsyIZA60T6uz5xWkFppjYVCMcwgbVHhgpuM50JMyP6XCQ+oZaZMlaMH1FqI8w6M2K0W+9vGeiBmNO+WFJMqzs+KVV0Ra58OXh7yJx9Jqy40q1Gr79iB8LjwEX0a4SbM0SqVnUUGKfLCwrnog5ej2Hcgbd+ALiLAWgPySXw6O43Gc44Vawsvgi110bFqReZHDyJozKD8xMHSPA3atiJD37j+hqNYkZGFfWb7qdQtKgxkHD3idS\\\\\\\/t5u5NlSoNK2O3xw1eKo4PTbz6POP\\\\\\\/3N6v5L6qk8\\\\\\\/ylMTJuddUdNYSkdDRx2mg2YWMfAAMAbliibbLdGDqnmGqF1273CXfQkxNsINHzKJ30iYkcih8bdOim8zEY2QiB5kKEzFeA+nqkZVw\\\\\\\/VSNDDWUcUlexcreiajAvPBrbv+IrjGDOGUCIwM67GvAVDQi6+G+4F0\\\\\\\/zz11KUQ9Tabe4ojoR4xM7tTFtKQ0RZg3ctguuhbWwAC8wk7JO9ilmg7SU7Gqt+k5xxOIuK1+ws0HySLm\\\\\\\/aFGKJhfR7NC\\\\\\\/x6HQrpfGFqa+2a0qCMMLKFsD9ERxg\\\\\\\/e9eS6k\\\\\\\/Ty6OWoDf5lOi2gLwJBmzGStLz\\\\\\\/tPZsaC8blPG3eO7R1Ap4lDPmn55+a1pL4\\\\\\\/SXCKN\\\\\\\/w4Fq0F\\\\\\\/rdhoOOGPQOBNnxY+K0BndiVloot4f2mlZaO4lN4YEQXRK3CTMluQIRDHb72\\\\\\\/22ddr7rXM6y0poSlF2qp5lW3dHYgANrn6LSmz9Cw==\\\u0022,\\\u0022iv\\\u0022:\\\u00229zujeyUkh2k46ZOnRLchrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e461b6f40177f32b1fe3f28bb1e0e1864b817a203e25ca60c69dc15134560e48\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Okps47zn8Hx9brJ9sMtC2biu80ixjHKucTpnIKVsu0N+KKXid4yq49fM1MFO1kdRwsBFFzk97LqcxphzbgD1YoZZwvzKv\\\\\\\/nyM5q0ZuDFEI2YXGe4Q4d+F1\\\\\\\/0OsKbN3YwgwHRojMqhPktSgsxDV\\\\\\\/weXVQA8VmiB176OPUWfS7dw\\\\\\\/1hn+bM7+YyweMI3z3gsvNG+M5iGQVQ8X9ozvuZZQtPe2gZrgEQ5pGd1xcXzyasY+GfmqfxtXd3v1XsOOKHp3EHuscQo1zaSQnYJNIJNiOcPmt17+smv8zdxbenJdI0s7\\\\\\\/B4k0h8DYMknAV20nNnvu9OXt35lWNpk6sKpx73qO\\\\\\\/4SnC3T+SGj+qXrbeUn5rGElYmFPUirXitAL0EVAWd1SLrNBLWRzTdEZ+mJd5MZJBRSVha8kdPM0pyLwwMbtCY8UAr09Qu0jJipWKDkDelqLaEoLYMzlUlerhyeMIi\\\\\\\/41DVeWzsyus45yGwfbKTA0iKDzCBy0QEIT7LhEb2qLb23uZqDqcE3qpksHS9xoBSDhqU7hiiWJ6\\\\\\\/Yo1KUskmGgrjMzaathx6SCSPp1sq+eJlAdKLt2YV89CJmwDKQfLH1VLFiXvbvcjZUB7r+ree6s1Ui0n230Qy8XhDKNJR2ivugsmSlkyeOjMr\\\\\\\/L0tscrPiFTdFnKQlQmFDq\\\\\\\/+b68vqAxlAerdgLvNKWG8M9xKeww9LKKJlpUSIxCR0gOUaa3hDFGqj4zeq4rAtOgNmSE4tcdbo2TMsr56UJoZSaai1PcbAdwnXeM6BVYDR12IwtxIlSoRRvGmhvZQSk1KcsXKU1ahrXgNUJSJyk3o8hbNYiCQBslHwDXn0uvMBi7KK+UDMH9yMD8Ir7mlj2NgxUnXjiuc0Tk3D4V1VJ7\\\\\\\/jBiLrHwpPWlVy\\\\\\\/C0XMZFQE31Zs9R\\\\\\\/MrUJYHvRkq4nb9TL+1J4xQy6mtLCiQ1XAl8wTKhoMH00zlkoN1vvzYa4h6k9dhK7IRTW54mxBleeJmcDKjaCSJFCRISjd5vmGHfy4mJsj4wkFhF2AvWxU\\\\\\\/mja10Pdgys74mGNZ9gk1Ddjyf4yAwrf3uwC8b93q+GrFFT3pYPT5JGOzyItzDqhaZH8LhbwkiM7XuTQcBSarr4Krb2BbF4l2orP+pEhc+RPJK1ke47eLJdU2MNUmI1Ymqfu7rUqSpnDKIj0R8SvQiKxZn+UPwo+fIvDhJLmfLIlPsyEi6NvTtSjG2QBWsYZxZkFsUXap2oH1bkMBDqHu9R6WtLjSg6oBpIFGZyH0RxNCZeWhB3gFE03kLW6KTWnALWWLa8x9ejNYcsMbDHbkAVAYSEwm1I0CAg\\\\\\\/1P\\\\\\\/OjeDj14ZzxEgpxiDpofgFu53y9w8PzBFOsbiduub6BuZ9SdfDu\\\\\\\/WOaBMQ9Zdh5yDvYmKsrOrlvXQe7OFaBenmvNCdSbRsvJUlQbAFV+Ml\\\\\\\/2pXrlnFRVB35Urm2T1oT5v06tYuwW8qAjahnyWy1v0QrVajVlEh5W+TODEht4\\\\\\\/xC1AhOGlqoUAemaaBAuujWJ6ZODJ2IA35HXZcm1BJqBYgWTOTkuby4YeHdNClQRACK3obyQIOszxM85n2v+q1MNhl3GYRiaJHQhaCD7bFBpy2Cyoisyeq09dWCsQSxDHqgSJg7zs7qnfnnR2Cdr3hOaEHt0cb5Avhzarq6lir1MOdZVl68TylnkX5ssoyFTDmqcV9lvQAYyN0at0f9kNzq4srw3un8jqY\\\\\\\/ofxqlZ3iSMEUoe1zrZd150U1H4ysWNC2yZZv\\\\\\\/+WnVrn4Mboxr9xTROJHCjs7G8\\\\\\\/wD2y8L+MiiRdXJMf8kjG0d+EPv3OwnWDHBp\\\\\\\/bzYuaeaJa\\\\\\\/rZxtPU0Jd8D+GNBvEiTb\\\\\\\/Va8vI3duN7O008uYYEs9TYkBpnoPx+zRVlAFD58yl5s9pNFVSPfcG8vWbNxmEKrSw6wyaOVCPdcG\\\\\\\/zUjfgxtCJbocA\\\\\\\/P8QE+Cxn5eV8pg\\\\\\\/K\\\\\\\/4WHR1vHUrN4aJrAPsqSLxjtVmTSKRI5TwRqM1kKv8r8+Vg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OZQo0n7IH8t5IcEKK9P9hg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220839bafb6c3dca85b3bd19979d901eadc934e64b9cf2fcc4f0afec74d4fefb40\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494999 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/3981" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yd4gOCFNmydJ1xx5QcV-TxSKO8NGkPXxgsUemX1IDd0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/3981" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2cIgRIzguwqlTw87Z6wUxP0j3ExQziaz-qWTJ8FaSOs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/3973" title="Genomics Analysis of L-DOPA Exposure in Drosophila sechellia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/3995" title="Natural Genetic Variation Screen in Drosophila Identifies Wnt Signaling, Mitochondrial Metabolism, and Redox Homeostasis Genes as Modifiers of Apoptosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494999" id="node494999" data-pisa="ggg;9/12/3981" data-pisa-master="ggg;g3.119.400677" data-apath="/ggg/9/12/3981.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2885-6831" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. Jesus Cerón-Rojas</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3981-3994;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400677 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494999" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">J. Jesus Cerón-Rojas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJesus%2BCer%C3%B3n-Rojas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cer%C3%B3n-Rojas%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BJesus%2BCer%25C3%25B3n-Rojas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2885-6831" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. Jesus Cerón-Rojas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/3981" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494999" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494999/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/3981.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494999" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494999/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/3981.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494999" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494999/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/3981.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494999" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494999/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The constrained linear genomic selection index (CLGSI) is a linear combination of genomic estimated breeding values useful for predicting the net genetic merit, which in turn is a linear combination of true unobservable breeding values of the traits weighted by their respective economic values. The CLGSI is the most general genomic index and allows imposing constraints on the expected genetic gain per trait to make some traits change their mean values based on a predetermined level, while the rest of them remain without restrictions. In addition, it includes the unconstrained linear genomic index as a particular case. Using two real datasets and simulated data for seven selection cycles, we compared the theoretical results of the CLGSI with the theoretical results of the constrained linear phenotypic selection index (CLPSI). The criteria used to compare CLGSI <em>vs.</em> CLPSI efficiency were the estimated expected genetic gain per trait values, the selection response, and the interval between selection cycles. The results indicated that because the interval between selection cycles is shorter for the CLGSI than for the CLPSI, CLGSI is more efficient than CLPSI per unit of time, but its efficiency could be lower per selection cycle. Thus, CLGSI is a good option for performing genomic selection when there are genotyped candidates for selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-estimated-breeding-value" class="hw-term hw-article-keyword hw-article-keyword-genomic-estimated-breeding-value" rel="nofollow">Genomic estimated breeding value</a></li><li class="kwd"><a href="/keyword/selection-response" class="hw-term hw-article-keyword hw-article-keyword-selection-response" rel="nofollow">Selection response</a></li><li class="kwd"><a href="/keyword/expected-genetic-gain-trait" class="hw-term hw-article-keyword hw-article-keyword-expected-genetic-gain-per-trait" rel="nofollow">Expected genetic gain per trait</a></li><li class="kwd"><a href="/keyword/molecular-marker-effects" class="hw-term hw-article-keyword hw-article-keyword-molecular-marker-effects" rel="nofollow">Molecular marker effects</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li></ul><p id="p-3">The unconstrained linear genomic selection index (LGSI) is a linear combination of genomic estimated breeding values (GEBVs) and was originally proposed by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Togashi <em>et al.</em> (2011)</a>; however, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em> (2015)</a> developed the LGSI theory completely and applied the LGSI theoretical results to real and simulated data. The LGSI is useful for predicting the net genetic merit, which in turn is a linear combination of true unobservable breeding values of the traits weighted by their respective economic values (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hazel 1943</a>). In LGSI, all marker effects of the genotyped individuals in the training population are estimated using marker and phenotypic data; these estimated effects are then used in subsequent selection cycles to obtain GEBVs that are predictors of the individual breeding values in the testing population for which there is only marker information about the candidates for selection. In LGSI, the GEBVs can be obtained by multiplying the genomic best linear unbiased predictor (GBLUP) of the estimated marker effects in the training population (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">VanRaden 2008</a>) by the coded marker values obtained in the testing population in each selection cycle. Applying LGSI in plant or animal breeding requires genotyping the candidates for selection to obtain marker effects and GEBVs, and then predicting and ranking the net genetic merit of the candidates for selection.</p><p id="p-4">The LGSI was developed in the genomic selection (GS) context in which animals and plants are selected based on the GEBV of the candidates for selection. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Meuwissen <em>et al.</em> (2001)</a> developed the GS theory and showed that it increases the accuracy of predicting the breeding values of the candidates for selection, and reduces the intervals between selection cycles and the costs of the breeding programs. <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Meuwissen <em>et al.</em> (2001)</a> suggested estimating all marker effects jointly using linear mixed models and Bayesian methods to increase the accuracy of the predicted breeding values because many genes affect the quantitative traits, which are of most concern to plant and animal breeders. <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Isik <em>et al.</em> (2017)</a>, however, have indicated that GS has not changed the fundamental basis of breeding methods, since in GS the individuals are ranked based on their estimated breeding values, which has been the selection method used by animal and plant breeders for a long time. Nevertheless, because GS decreases the generation interval, it leads to a much higher genetic gain per year. For example, in the plant breeding context, a four-year breeding cycle, which includes three years of field testing, can be reduced to only four months, <em>i.e.</em>, the time required to grow and cross plants (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lorenz <em>et al.</em> 2011</a>). <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Börner and Reinsch (2012)</a> have indicated that GS could replace traditional progeny testing when maximizing the genetic gain per year, as long as the accuracy of GEBV is higher than or equal to 0.45.</p><p id="p-5">One of the main problems associated with the LGSI theory is that the values of its expected genetic gain per trait (or multi-trait selection response) can increase or decrease in a positive or negative direction without control. In the phenotypic selection context, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog (1959)</a> developed a restricted linear selection index that allows imposing restrictions equal to zero on the expected genetic gain of some traits. Other authors (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mallard 1972</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Harville 1975</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Tallis 1985</a>) extended the <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog (1959)</a> approach and developed a constrained linear phenotypic selection index (CLPSI) that attempts to make some traits change their expected genetic gain values based on a predetermined level, while the rest of the traits remain without restrictions. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Itoh and Yamada (1987)</a> showed that the <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Mallard (1972)</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Harville (1975)</a> and <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Tallis (1985)</a> indices give the same results. The CLPSI is the most general index and includes the unconstrained and restricted phenotypic indices as particular cases.</p><p id="p-6"><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa (2018</a>, Chapter 3) developed a constrained linear genomic selection index (CLGSI); however, these authors did not evaluate this index completely. For example, they did not give results associated with the CLGSI expected genetic gain per trait, which is the main parameter of this index because the breeder imposes constraints on it to make some traits change their expected genetic gain values based on a predetermined level, while the rest of the traits remain without restrictions.</p><p id="p-7">This study had two main objectives: first, to apply the CLGSI theoretical results to two real datasets and to seven simulated datasets using only GEBVs for selecting non-phenotyped candidates for selection, and second, to compare the relative efficiency of CLGSI and CLPSI using real and simulated datasets in a single-stage context. The criteria we used to compare CLGSI <em>vs.</em> CLPSI efficiency were the estimated expected genetic gain per trait values, the selection response, and the interval between selection cycles. One additional criterion was that the CLGSI and CLPSI proportional constant value should be positive (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa 2018</a>, Chapters 3 and 6 for details). The results indicated that because the interval between selection cycles was shorter for CLGSI than for CLPSI, CLGSI is more efficient than CLPSI per unit of time, but its efficiency could be lower per selection cycle.</p><p id="p-8">We applied the CLGSI theory assuming that the GEBV values have multivariate normal distribution and that the CLGSI (CLPSI) and the net genetic merit have joint bivariate normal distribution. Under this last assumption, the regression of the net genetic merit on any linear function of the phenotypic or GEBV values is linear (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog 1959</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Objectives of the constrained linear selection index</h3><p id="p-9"><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cerón-Rojas <em>et al.</em> (2016)</a> and <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa (2018</a>, Chapter 3) described the CLPSI theory; thus, in this work, we shall describe only the CLGSI theory. Let <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-1.gif"/></noscript></span></span> be the population mean of the <em>i</em><sup>th</sup> trait before selection. One of the main CLGSI objectives is to change <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-2.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-3.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-4.gif"/></noscript></span></span> is the <em>i</em><sup>th</sup> (<em>i</em> = 1, 2, …, <em>r</em>; <em>r</em> = number of constrained traits) trait constraint or predetermined proportional gain imposed by the breeder on the CLGSI expected genetic gain per trait. Additional CLGSI objectives are to maximize the selection response; to predict the net genetic merit (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-5.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-7.gif"/></noscript></span></span> are <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-8.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-9.gif"/></noscript></span></span>number of traits) vectors of economic weights and true unobservable breeding values, respectively); to select individuals with the highest H values as parents of the next generation; and to provide the breeder with an objective rule for evaluating and selecting several traits simultaneously.</p></div><div id="sec-3" class="subsection"><h3>The constrained linear genomic selection index</h3><p id="p-10">Let <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-10.gif"/></noscript></span></span> be a vector of genomic breeding values for t traits (<strong>Appendix 1</strong>, Equations A1 to A4 for details). The CLGSI used to predict the individual net genetic merit of a candidate for selection is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-12.gif"/></noscript></span></span> is the CLGSI vector of coefficients and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-13.gif"/></noscript></span></span> (<em>i</em>= 1, 2, …, t) is the genomic value associated with trait <em>i</em><sup>th</sup>. <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em> (2015)</a> described Equation (1) in the unconstrained LGSI.</p></div><div id="sec-4" class="subsection"><h3>The CLGSI selection response</h3><p id="p-11">The CLGSI selection response (R) can be written as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where k is the selection intensity, <em>L</em> denotes the interval between selection cycles, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-15.gif"/></noscript></span></span> is the standard deviation of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-16.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-17.gif"/></noscript></span></span> is the covariance matrix of <strong>g</strong>, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-18.gif"/></noscript></span></span> is the correlation between <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-19.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-20.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-21.gif"/></noscript></span></span> is the covariance between <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-22.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-23.gif"/></noscript></span></span>, <strong><span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-24.gif"/></noscript></span></span></strong> is the covariance matrix of genomic breeding values (<strong>Appendix 1</strong>, Equation A2 for details), and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-25.gif"/></noscript></span></span> is the standard deviation of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-26.gif"/></noscript></span></span>. Equation (1) indicates that the genetic gain that can be achieved in <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-27.gif"/></noscript></span></span>by selecting for several traits simultaneously within a population of plants, is the product of k, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-29.gif"/></noscript></span></span> (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog 1959</a>). Thus, to increase selection progress, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-30.gif"/></noscript></span></span> should be as large as possible. <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa (2018</a>, Chapter 3) described the selection response of the CLPSI, which is very similar to Equation (2).</p></div><div id="sec-5" class="subsection"><h3>The CLGSI expected genetic gain per trait</h3><p id="p-12">The CLGSI expected genetic gain per trait (E, or multi-trait selection response) is the covariance between the genomic breeding value vector <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-31.gif"/></noscript></span></span> and the index, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-32.gif"/></noscript></span></span> (Equation 1), weighted by the standard deviation of I, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-33.gif"/></noscript></span></span>, multiplied by the selection intensity (k) and divided by the interval between selection cycles (<em>L</em>), <em>i.e.</em>,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-34.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Equation (3) is a <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-35.gif"/></noscript></span></span> vector, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-36.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-37.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-38.gif"/></noscript></span></span> 1, 2, …,t) is the expected genetic gain of trait <em>i</em><sup>th</sup>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-39.gif"/></noscript></span></span> is the <em>i</em><sup>th</sup> genomic breeding value, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-40.gif"/></noscript></span></span> is the index value (Equation 1) for one individual, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-41.gif"/></noscript></span></span> is the covariance between I and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-42.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-43.gif"/></noscript></span></span> is the covariance between <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-44.gif"/></noscript></span></span> and the j<sup>th</sup> (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-45.gif"/></noscript></span></span> 1, 2, …,t) index genomic breeding value. We defined all the other terms of Equation (3) in Equation (2).</p></div><div id="sec-6" class="subsection"><h3>The CLGSI vector of coefficients</h3><p id="p-13">Maximizing <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-46.gif"/></noscript></span></span> (Equation 2) is equivalent to minimizing the mean squared difference between the net genetic merit <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-47.gif"/></noscript></span></span> and the index <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-48.gif"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-49.gif"/></noscript></span></span>, with respect to the vector of coefficients β under the restriction that Equation (3) values are equal to the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-50.gif"/></noscript></span></span> values (<em>i</em> = 1, 2, …, <em>r</em>) imposed by the breeder. Details of the process used to obtain the CLGSI vector of coefficients (β) are given in <strong>Appendix 2</strong> (Equations A6 and A7). Here we present only the main result. The CLGSI vector of coefficients is<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-51.gif"/></noscript></span><span class="disp-formula-label">(4a)</span></span>where <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-52.gif"/></noscript></span></span> is a vector of economic weights, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-53.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-54.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-55.gif"/></noscript></span></span> is an identity matrix of size <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-56.gif"/></noscript></span></span>, D is a <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Mallard (1972)</a> matrix described in <strong>Appendix 2</strong>, and U is the <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog (1959)</a> matrix described in <strong>Appendix 3</strong>. When <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-57.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-58.gif"/></noscript></span></span> for all traits), matrix K can be written as <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-59.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-60.gif"/></noscript></span></span>. In this last case, the CLGSI is a null restricted LGSI similar to the <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog (1959)</a> restricted index. When <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-62.gif"/></noscript></span></span> is a null matrix, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-63.gif"/></noscript></span></span>, the vector of economic weights or the unconstrained LGSI vector of coefficients (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em> 2015</a>). Thus, the CLGSI is more general than the LGSI and includes the null restricted and the unrestricted LGSI as particular cases. The vector of coefficients β should maximize the selection response (R, Equation 2) and make Equation (3) values similar to <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-64.gif"/></noscript></span></span> values.</p><p id="p-14">It is possible to show (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Itoh and Yamada 1987</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa 2018</a>, Chapter 3) that another way of writing Equation (4a) is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-65.gif"/></noscript></span><span class="disp-formula-label">(4b)</span></span>where <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-66.gif"/></noscript></span></span> is the proportionality constant and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-67.gif"/></noscript></span></span> is the vector of the constraints imposed by the breeder. According to <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Itoh and Yamada (1987)</a>, θ should be higher than zero (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-68.gif"/></noscript></span></span>) because when <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-69.gif"/></noscript></span></span>, the CLGSI moves the population means in the opposite direction to the predetermined desired direction. In addition, when <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-70.gif"/></noscript></span></span>, we would have the CLGSI with null constraints similar to the <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog (1959)</a> restricted index. Thus, θ is a good criterion for determining when the CLGSI moves the population means in the desired direction, which will occur when <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-71.gif"/></noscript></span></span>. While Equation (4a) is associated with the <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Mallard (1972)</a> constrained index, Equation (4b) is associated with the <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Tallis (1985)</a> constrained index; however, Equations (4a) and (4b) express the same result in a different mathematical way.</p><p id="p-15">In <strong>Appendix 1</strong> (Equation A5), we give a method for estimating matrix Γ, and in <strong>Appendix 2</strong> (Equation A8), we explain how to estimate β.</p></div><div id="sec-7" class="subsection"><h3>Maximized CLGSI selection response and optimized expected genetic gain per trait</h3><p id="p-16">The maximized CLGSI selection response is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-72.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>while the optimized CLGSI expected genetic gain per trait is the same as Equation (3). In Equation (5) we omitted <em>L</em> to simplify notation. Whereas in Equation (2) the selection response can take any value, in Equation (5), R gives the maximum value of Equation (2). In <strong>Appendix 2</strong> (Equations A9 and A10), we indicate how to estimate R and E.</p></div><div id="sec-8" class="subsection"><h3>Obtaining the genomic estimated breeding values (GEBV)</h3><p id="p-17">Several authors (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">VanRaden 2008</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em> 2015</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Isik <em>et al.</em> 2017</a>, Chapter 11; <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa 2018</a>, Chapter 5) have given detailed descriptions of how to obtain trait genomic breeding values (GEBV) which are predictors of trait unobservable breeding values. In CLGSI, we fitted phenotypic and marker data from the training population in a statistical model to estimate all available marker effects; these estimates were then used to obtain GEBV that are predictors of the individual traits’ true genomic breeding values in a testing population for which there is only marker information. We obtained the GEBV in the non-phenotyped testing population by multiplying the estimated marker effects obtained in the training population by the coded marker values obtained in the testing population at each selection cycle.</p></div><div id="sec-9" class="subsection"><h3>Criteria for comparing CLGSI efficiency <em>vs.</em> CLPSI efficiency</h3><p id="p-18">The criteria used to compare CLGSI <em>vs.</em> CLPSI efficiency when making genomic and phenotypic selection were as follows. First, the estimated theta value (θ) should be positive (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-73.gif"/></noscript></span></span>). Second, the estimated expected genetic gain values should be close to the constraints (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-74.gif"/></noscript></span></span>) imposed by the breeder. Third, the estimated selection response value should be equivalent to the true selection response value. Fourth, the size of the interval between selection cycles (<em>L</em>) should be short to increase the selection gain per year.</p><p id="p-19">In this work, we do not consider the problem associated with the cost of obtaining measures of each phenotypic trait and the process of genotyping individual candidates for selection. However, some authors (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Schaeffer 2006</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Börner and Reinsch 2012</a>) have considered this problem in the animal breeding context.</p></div></div><div class="section materials" id="sec-10"><h2 class="">Materials</h2><div id="sec-11" class="subsection"><h3>Real data</h3><p id="p-20">We used two real maize (<em>Zea mays</em> L.) F2 populations: “JMpop1 DTMA Mexico optimum environment” and “6x1020 WEMA Africa optimum environment” (hereafter we shall refer to the first and second maize F2 populations as dataset 1 and dataset 2, respectively). The training population (C0) of each dataset contains genotypic data and four phenotypic traits: grain yield (GY, t/ha), plant height (PHT, cm), ear height (EHT, cm), and anthesis days (AD, d). In addition, each dataset has three sets of individuals from the training population (C0) and two testing populations (C1 and C2). We present the number of individuals and molecular markers in each population in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Assuming that the breeding objective was to increase GY while decreasing PHT, EHT, and AD, the vector of economic weights in C0, C1, and C2 for GY, PHT, EHT, and AD was <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-75.gif"/></noscript></span></span> for both indices and the two datasets. For illustration purposes only, in this work we used three selection cycles (C0, C1 and C2) for both datasets to illustrate the theoretical results and the efficiency of CLGSI and CLPSI.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494921/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494921/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Two real maize (<em>Zea mays</em> L.) <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/T1/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/T1/embed/mml-math-76.gif"/></noscript></span></span> populations and the number of genotypes (<em>g</em>) and molecular markers (<em>m</em>) used in three selection cycles (cycles 0, 1 and 2)</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">For illustration purposes only, to select traits GY, PHT, EHT and AD, we imposed two sets of restrictions. First we restricted traits GY and PHT with vector <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-77.gif"/></noscript></span></span> and matrices <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-79.gif"/></noscript></span></span> (<strong>Appendices 2 and 3</strong> for details), and later, we restricted traits GY, PHT and EHT with vector <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-80.gif"/></noscript></span></span> and matrices <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-81.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-82.gif"/></noscript></span></span>for both datasets and for the two indices (CLGSI and CLPSI). The total proportion (p) of retained value for these datasets was <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-83.gif"/></noscript></span></span> 0.10 (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-84.gif"/></noscript></span></span>) for both indices and the two datasets with the two sets of constraints.</p></div><div id="sec-12" class="subsection"><h3>Simulated datasets</h3><p id="p-22">The datasets were simulated by <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em> (2015)</a> with QU-GENE software (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Podlich and Cooper 1998</a>) using 2500 molecular markers and 315 quantitative trait loci (QTL) for eight phenotypic selection cycles (C0 to C7), each with four traits (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-85.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-86.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-87.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-88.gif"/></noscript></span></span>), 500 genotypes and 4 replicates for each genotype. The authors distributed the markers uniformly across 10 chromosomes and the QTL randomly across the 10 chromosomes to simulate maize (<em>Zea mays</em> L.) populations. A different number of QTL affected each of the four traits: 300, 100, 60, and 40, respectively. The common QTL affecting the traits generated genotypic correlations of -0.5, 0.4, 0.3, -0.3, -0.2, and 0.1 between <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-89.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-90.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-91.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-92.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-93.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-94.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-95.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-96.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-97.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-98.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-99.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-100.gif"/></noscript></span></span>, respectively. The economic weights for <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-101.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-102.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-104.gif"/></noscript></span></span> were 1, -1, 1 and 1, respectively. Additional details of the simulated data can be seen in <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em> (2015)</a>.</p><p id="p-23">We used seven phenotypic and genomic selection cycles (C1 to C7) with <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-105.gif"/></noscript></span></span> 0.10 (<span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-106.gif"/></noscript></span></span>) in each cycle. We selected all four traits in each selection cycle. For illustration purposes only, to select traits <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-107.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-108.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-109.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-110.gif"/></noscript></span></span>, we imposed two sets of restrictions. First we restricted traits <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-112.gif"/></noscript></span></span> with vector <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-113.gif"/></noscript></span></span> and matrices <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-114.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-115.gif"/></noscript></span></span>, and later, we restricted traits <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-116.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-117.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-118.gif"/></noscript></span></span> with vector <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-119.gif"/></noscript></span></span> and matrices <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-120.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-121.gif"/></noscript></span></span>for both datasets and both indices.</p></div><div id="sec-13" class="subsection"><h3>Real and simulated data availability</h3><p id="p-24">The real and simulated datasets are available in the <em>Application of a Genomics Selection Index to Real and Simulated Data</em> repository, at <a href="http://hdl.handle.net/11529/10199">http://hdl.handle.net/11529/10199</a>. The two real datasets used in this work are the folder named “File Real_Data_Sets_GSI” that contains four folders called “DATA_SET-3, 4, 5 and 6”. Each of the four folders in turn contains four Excel data files. The four Excel data files within the folder DATA_SET-3 are as follows: DATA_SET-3_Markers_Cycle-0, 1, 2, and DATA_SET-3_Phenotypic_Cycle-0. The first three Excel files contain the marker coded values for cycles 0, 1 and 2, while the Excel file DATA_SET-3_Phenotypic_Cycle-0 contains the phenotypic information of cycle 0 (training population). The Excel data files of the other folders were described in a similar manner as for folder 3. In this work, we used datasets 3 and 6 to make selections and to estimate the theta values, the selection response and the genetic expected gains. The results are presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">3</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495042/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495042/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Two real datasets for estimated CLGSI<sup><em><sup>a</sup></em></sup> parameters constructed with four traits (GY, PHT, EHT and AD) for two and three constraints in three selection cycles</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495121/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495121/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Two real datasets for estimated LGSI<sup><em><sup>a</sup></em></sup> parameters constructed with four traits (GY, PHT, EHT and AD) without constraints for three selection cycles</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Folder Simulated_Data_GSI contains two folders: Data_Phenotypes_April-26-15 and Haplotypes_GSI_April-26-15. In turn, folder Data_Phenotypes_April-26-15 contains two folders: GSI_Phenotypes-05 and PSI_Phenotypes-05. Within folder GSI_Phenotypes-05, there are six Excel data files, each denoted as C2_GSI_05_Pheno, C3_GSI_05_Pheno, C4_GSI_05_Pheno, C5_GSI_05_Pheno and C6_GSI_05_Pheno, corresponding to the simulated phenotypic information for the genomic selection index for cycles 2-7. In addition, folder GSI_Phenotypes-05 contains eight Excel datasets denoted as C0_Pheno_05, C1_PSI_05_Pheno, C2_PSI_05_Pheno, C3_PSI_05_Pheno, C4_PSI_05_Pheno, C5_PSI_05_Pheno, C6_PSI_05_Pheno, and C7_PSI_05_Pheno corresponding to the phenotypic simulated information for the phenotypic selection index for cycles 0-7. File Haplotypes_GSI_April-26-15 contains the haplotypes of the markers for cycles 0-7 of GSI. We present the results of the simulated datasets in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Tables 4</a>, <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">5</a> and <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">6</a>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495072/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495072/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Simulated data for estimated CLPSI<sup><em><sup>a</sup></em></sup> and CLGSI<sup><em><sup>b</sup></em></sup> parameters constructed with four traits (T1, T2, T3 and T4), two constraints and true maximum responses in seven selection cycles</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494964/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494964/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Simulated data for estimated CLPSI<sup><em><sup>a</sup></em></sup> and CLGSI<sup><em><sup>b</sup></em></sup> parameters constructed with four traits (T1, T2, T3 and T4), three constraints and true maximum responses in seven selection cycles</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495012/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495012/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Simulated data for estimated LPSI<sup><em><sup>a</sup></em></sup> and LGSI<sup><em><sup>b</sup></em></sup> parameters constructed with four traits (T1, T2, T3 and T4) without constraints and true maximum responses for seven selection cycles</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Real data</h3><div id="sec-16" class="subsection"><h4>The normality assumption:</h4><p id="p-47"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> presents the frequency distribution of the GEBVs associated with traits GY (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1a</a>) and PHT (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1b</a>) for Dataset 1, in cycle 1. In addition, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> presents the frequency distribution of the CLGSI values for real Dataset 1 (in cycle 1) with two constraints (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a>, <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-122.gif"/></noscript></span></span>), whereas <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2b</a> presents the frequency distribution of the CLGSI values for real Dataset 2 (in cycle 2) with three constraints (<span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-123.gif"/></noscript></span></span>). Based on these results, we can assume that the GEBVs associated with traits GY and PHT, and the CLGSI values for the two set of restrictions, approach the normal distribution.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the GEBVs (genomic estimated breeding values) associated with traits GY (Grain Yield, Fig. 1a) and PHT (Plant Height, Fig. 1b) for real Dataset 1 in cycle 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979623712" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of the GEBVs (genomic estimated breeding values) associated with traits GY (Grain Yield, Fig. 1a) and PHT (Plant Height, Fig. 1b) for real Dataset 1 in cycle 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/3981/F1.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/3981/F1.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495098" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-48" class="first-child">Distribution of the GEBVs (genomic estimated breeding values) associated with traits GY (Grain Yield, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1a</a>) and PHT (Plant Height, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1b</a>) for real Dataset 1 in cycle 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the estimated CLGSI (constrained linear genomic selection index) values with two (Fig. 2a) and three (Fig. 2b) constraints for real Datasets 1 and 2, in cycles 1 and 2, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979623712" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of the estimated CLGSI (constrained linear genomic selection index) values with two (Fig. 2a) and three (Fig. 2b) constraints for real Datasets 1 and 2, in cycles 1 and 2, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/3981/F2.medium.gif" width="440" height="390"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/3981/F2.medium.gif" width="440" height="390"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494937" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-49" class="first-child">Distribution of the estimated CLGSI (constrained linear genomic selection index) values with two (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2a</a>) and three (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2b</a>) constraints for real Datasets 1 and 2, in cycles 1 and 2, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>Estimated maximized CLGSI parameters:</h4><p id="p-50"><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">3</a> present the estimated CLGSI and LGSI parameters, respectively. The estimated parameters are the theta (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-124.gif"/></noscript></span></span>) values (for CLGSI only), the expected genetic gains per trait <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-125.gif"/></noscript></span></span> (<strong>Appendix 2</strong>, Equation A10), and the selection responses <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-126.gif"/></noscript></span></span> (<strong>Appendix 2</strong>, Equation A9). In <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, all parameters were estimated for datasets 1 and 2 with two (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-127.gif"/></noscript></span></span>) and three (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-128.gif"/></noscript></span></span>) constraints, whereas in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, the parameters were estimated, without constraints, for three selection cycles (0, 1 and 2). The estimated theta values were all positive (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) for the two real datasets, which indicates that the CLGSI moves the population means in the desired direction, as we would expect.</p></div><div id="sec-18" class="subsection"><h4>Estimated CLGSI expected genetic gains per trait:</h4><p id="p-51">In <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">3</a>, there are two means: Mean1 and Mean2 = Mean1/<em>L</em> (<em>L=</em> interval between selection cycles) associated with the estimated CLGSI and LGSI expected genetic gain per trait and selection response for the two real maize (<em>Zea mays</em> L.) F2 populations, respectively. Mean1 is the average of the three selection cycles, whereas Mean2 (Mean1/1.5, where 1.5 is the interval between selection cycles or the time required to complete one selection cycle (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> 2015</a>)) is the average of the genetic gain per year.</p><p id="p-52">For dataset 1, <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-129.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-130.gif"/></noscript></span></span> and two constraints, the estimated CLGSI expected genetic gains per trait for cycles 0, 1 and 2 were <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-131.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-133.gif"/></noscript></span></span>, respectively. Each <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-134.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-135.gif"/></noscript></span></span> 0, 1, 2) value was associated with the mean values of traits GY, PHT, EHT and AD. Traits GY and PHT were constrained by <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-136.gif"/></noscript></span></span> values. The Mean1 of the <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-137.gif"/></noscript></span></span> value associated with traits GY and PHT (0.67 and -1.35, respectively) overestimated the <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-138.gif"/></noscript></span></span> values by 35% (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). However, note that the genetic gain per year, obtained as Mean2 = Mean 1/1.5, underestimated the <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-139.gif"/></noscript></span></span> values by 10% because the Mean2 of the <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-140.gif"/></noscript></span></span> values associated with traits GY and PHT (0.45 and -0.90, respectively) were lower than the <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-141.gif"/></noscript></span></span> values.</p><p id="p-53">When we constrained three traits (GY, PHT and EHT) by vector <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-142.gif"/></noscript></span></span>, we found results similar to those for two constraints. That is, the Mean1 of the <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-143.gif"/></noscript></span></span> values associated with traits GY, PHT and EHT (0.75, -1.50 and -0.75, respectively) overestimated the <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-144.gif"/></noscript></span></span> values by 50% (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>). However, for Mean2 = Mean1/1.5, we found that the <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-145.gif"/></noscript></span></span> values associated with traits GY, PHT and EHT (0.50, -1.0 and -0.50, respectively) were equal to the <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-146.gif"/></noscript></span></span> values. Thus, for dataset 1, the values of the vector of restrictions, <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-147.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-148.gif"/></noscript></span></span>, were closer to Mean2 than to Mean1 values. This means that the estimated maximized expected genetic gain per trait estimated the genetic gain per year better than the genetic gain per selection cycle for dataset 1.</p><p id="p-54">For dataset 2 (<span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-149.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-150.gif"/></noscript></span></span>), with two (<span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-151.gif"/></noscript></span></span>) and three (<span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-152.gif"/></noscript></span></span>) constraints, Mean1 for the <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-153.gif"/></noscript></span></span> values associated with traits GY and PHT and those associated with GY, PHT and EHT, were closer to the <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-154.gif"/></noscript></span></span>values than the Mean2 values (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). In this case, the estimated expected genetic gain per trait estimated the genetic gain per year better than the genetic gain per selection cycle.</p><p id="p-55">Dataset 1 gave higher estimated maximized CLGSI expected genetic gains per trait and genetic gains per year than dataset 2. This means that the number of genotypes affected the estimated values of the expected genetic gains (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The number of genotypes in real dataset 1 for cycles 0, 1 and 2 were 247, 320 and 303, respectively, whereas for dataset 2, the number of genotypes for those cycles were 181, 274 and 274 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), respectively.</p></div><div id="sec-19" class="subsection"><h4>Estimated maximized CLGSI selection response:</h4><p id="p-56">The results of the estimated maximized CLGSI selection responses for both datasets and constraints, were as follows. For dataset 1, the Mean1 of the estimated maximized CLGSI selection responses for two and three constraints were 4.24 and 4.29, respectively, while for dataset 2, those estimated values were 2.79 and 3.01. For dataset 1, the Mean2 of the estimated maximized CLGSI selection responses for two and three constraints were 2.82 and 2.99, respectively, while for dataset 2, those estimated values were 1.86 and 2.01. These results indicate that dataset 1 gave higher estimated CLGSI genetic gains per year. Again, we explain these results by noting that the number of genotypes in real dataset 1 for cycles 0, 1 and 2 were 247, 320 and 303, respectively, while for dataset 2, the number of genotypes for those cycles were 181, 274 and 274 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), respectively. This means that, in effect, the number of genotypes affected the estimated values of the selection response.</p><p id="p-57">When we compared the CLGSI results (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a>) with unconstrained LGSI results (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>), we found that the estimated CLGSI expected genetic gains per trait were different to the estimated unconstrained LGSI expected genetic gains per trait for both datasets. However, the estimated selection responses of both indices were very similar (<a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">3</a>); this means that the two sets of constraints imposed on the CLGSI when we obtained its vector of coefficients mainly affected the CLGSI expected genetic gain per trait, as we would expect.</p></div></div><div id="sec-20" class="subsection"><h3>Simulated data</h3><div id="sec-21" class="subsection"><h4>Correlation of the GEBV With the true breeding values of the traits:</h4><p id="p-58"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> presents the estimated correlations between the GEBVs and true breeding values for four traits in six (C2 to C7) selection cycles. Each selection cycle contains four columns: the first column (from left to right) corresponds to the correlation between the GEBV and the T1 true breeding values; the second column corresponds to the correlation between the GEBV and the T2 true breeding values, and so on. In this figure, all correlation values tend to decrease. In C7, the correlation values between the GEBVs and the traits’ true breeding values were 0.40, 0.55, 0.54, and 0.50 for each of the four traits, respectively, whereas in cycle two (C2), these correlations were 0.52, 0.74, 0.69, and 0.73 for each of the four traits, respectively. In percentage terms, the correlation values of C7 were only 76%, 74%, 78%, and 68% of the correlation values in C2. That is, the correlation between the GEBVs and the traits’ true breeding values decreased more for traits 2 and 4 than for traits 1 and 3. We can explain these results by the number of QTL that affected each trait and the size of the QTL effects on the traits in each selection cycle. In all selection cycles, the estimated correlations were higher than or equal to 0.45; thus, the GEBVs obtained with the simulated data were good predictors of the individual breeding values, and so the CLGSI was a good predictor of the net genetic merit because the CLGSI is a linear combination of GEBV.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between the genomic estimated breeding values (GEBVs) and the true breeding values for four traits (T1, T2, T3 and T4) in six (C2 to C7) selection cycles. For each selection cycle, each column correspond to the correlations between the GEBV and the true breeding values for traits T1, T2, T3, and T4, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1979623712" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlations between the genomic estimated breeding values (GEBVs) and the true breeding values for four traits (T1, T2, T3 and T4) in six (C2 to C7) selection cycles. For each selection cycle, each column correspond to the correlations between the GEBV and the true breeding values for traits T1, T2, T3, and T4, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/3981/F3.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/3981/F3.medium.gif" width="440" height="263"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/3981/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495024" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-59" class="first-child">Correlations between the genomic estimated breeding values (GEBVs) and the true breeding values for four traits (T1, T2, T3 and T4) in six (C2 to C7) selection cycles. For each selection cycle, each column correspond to the correlations between the GEBV and the true breeding values for traits T1, T2, T3, and T4, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>Estimated maximized CLPSI and CLGSI parameters:</h4><p id="p-60"><a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">5</a> present the estimated maximized CLPSI and CLGSI parameters, whereas <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a> presents the unconstrained estimated maximized linear phenotypic and genomic selection index (LPSI and LGSI, respectively) parameters for seven simulated selection cycles. <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa (2018</a>, Chapters 2 and 5) have given details of how to estimate the LPSI and LGSI parameters. We imposed two sets of constraints on the CLPSI and CLGSI expected genetic gains for seven selection cycles (C1 to C7). The vector for two constraints imposed on traits T1 and T2 was <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-155.gif"/></noscript></span></span>, whereas the vector for three constraints imposed on traits T1, T2 and T3 was <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-156.gif"/></noscript></span></span>. The estimated theta values were all positive for the simulated dataset in the seven selection cycles for both sets of constraints, which indicates that CLPSI and CLGSI move the population means in the desired direction.</p><p id="p-61">We shall compare the estimated maximized CLPSI parameters to the estimated maximized CLGSI parameters using the results in <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">5</a>. In <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">5</a>, there are three means: Mean1 and Mean2 (Mean3) = Mean1/<em>L</em> (<em>L=</em> interval between selection cycles) associated with the estimated maximized CLPSI and CLGCI expected genetic gain per trait and selection response, respectively. Mean1 is the average of the seven selection cycles, whereas, for the CLPSI, Mean2= Mean1/4 (4<em>=</em> interval between selection cycles, <em>L</em> (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> 2015</a>)) is the average genetic gain per year. For the CLGSI, <em>L</em> = 1.5 (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> 2015</a>), from where Mean3= Mean1/1.5. This means that the interval between selection cycles is a characteristic of each index and that the interval was shorter for the CLGSI than for the CLPSI.</p><p id="p-62">The total proportion retained for both indices was <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-157.gif"/></noscript></span></span> 0.10 (<span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-158.gif"/></noscript></span></span> 1.755) for all seven selection cycles. Each estimated CLPSI and CLGSI expected genetic gain per trait <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-159.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-160.gif"/></noscript></span></span>1, 2, …, 7) for two and three constraints is a vector of values associated with the expected genetic values of traits T1, T2, T3 and T4 (Equation 3). Some authors (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cerón-Rojas <em>et al.</em> 2016</a>; Céeron-Rojas and Crossa 2018, Chapter 3) have described how to obtain <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-161.gif"/></noscript></span></span> for the CLPSI. In <strong>Appendix 2</strong> (Equations A9 and A10), we describe how to estimate the maximized CLGSI selection response and expected genetic gain per trait.</p></div><div id="sec-23" class="subsection"><h4>Estimated and maximized CLPSI and CLGSI expected genetic gain per trait:</h4><p id="p-63">For two constraints (<span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-162.gif"/></noscript></span></span>), the Mean1 of the estimated CLPSI expected genetic gain values associated with traits T1 and T2 were 6.99 and -2.80, respectively, whereas the Mean1 of the estimated CLGSI expected genetic gain values associated with traits T1 and T2 were 5.91 and -2.36, respectively. This means that the CLPSI overestimated the <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-163.gif"/></noscript></span></span> values by 38.80%, whereas the CLGSI overestimated those values by 18.20%.</p><p id="p-64">For three constraints (<span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-164.gif"/></noscript></span></span>), the Mean1 of the estimated CLPSI expected genetic gain values associated with traits T1, T2 and T3 were 5.86, -2.34 and 3.52, respectively, whereas the Mean1 of the estimated CLGSI expected genetic gains associated with those traits were 4.69, -1.87 and 2.81, respectively. This means that while the CLPSI overestimated the <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-165.gif"/></noscript></span></span> values by 17.20%, the CLGSI underestimated those values by only 6.20%. Thus, the expected genetic gain per trait per selection cycle of the CLGSI was closer to the true gain than was the expected gain of the CLPSI. In addition, because the interval between selection cycles was higher for the CLPSI (<em>L</em> = 4) than for the CLGSI (<em>L</em> = 1.5), the genetic gain per year associated with the expected genetic gain per trait was also higher for the CLGSI than for the CLPSI (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">5</a>).</p><p id="p-65">Finally, note that the estimated LPSI and LGSI expected genetic gains per trait (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>) were more different from the vectors of constraints, than the estimated CLPSI and CLGSI expected genetic gains per trait. Thus, the two sets of constraints imposed on the CLGSI (CLPSI) when we obtained its vector of coefficients, affected the CLGSI (CLPSI) expected genetic gain per trait, as we would expect.</p></div><div id="sec-24" class="subsection"><h4>Efficiency of the CLPSI and CLGSI selection response:</h4><p id="p-66">In this subsection, we shall compare the average (Mean1) of the estimated CLPSI and CLGSI selection response to the true selection response obtained for both indices (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-6" class="xref-table" href="#T5">5</a>). In <strong>Appendix 2</strong> (Equations A11a and A11b), we indicate how we obtained the maximum true selection response for LPSI (LGSI) and for CLPSI (CLGSI). The maximum possible true selection responses for the economic index was the same for the genomic and phenotypic indices when we started the selection process, as this did not depend on the weights used in a particular selection index. However, from selection cycle two to selection cycle seven, the maximum possible true selection responses for both indices were different (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em> 2015</a>).</p><p id="p-67">Again, let <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-166.gif"/></noscript></span></span> 0.10 (<span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-167.gif"/></noscript></span></span>1.755) be the total proportion retained for both indices for all seven selection cycles with two set of constraints, as indicated in the last subsection. For two constraints, the average of the estimated CLPSI selection response (13.91) explained 84.66% of the true selection response (16.43), while the average of the estimated CLGSI selection response (11.94) explained only 80.0% of the true selection response (14.89) (<a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>). In a similar manner, for three constraints, the average of the estimated CLPSI selection response (13.14) explained 89.02% of the true selection response (14.76), while the average of the estimated CLGSI selection response (10.63) explained only 79.0% of the true selection response (13.41) (<a id="xref-table-wrap-5-7" class="xref-table" href="#T5">Table 5</a>). That is, in this case, the estimated CLPSI selection response was closer to its true response than the estimated CLGSI selection response was to its true response. Note, however, that because the interval between selection cycles was higher for the CLPSI (<em>L</em> = 4) than for the CLGSI (<em>L</em> = 1.5), the genetic gain per year associated with the selection response was higher for the CLGSI than for the CLPSI (<a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-8" class="xref-table" href="#T5">5</a>) for both constraints.</p><p id="p-68">Finally, when we compared the estimated CLPSI and LPSI selection responses, we observed that they were very similar. The same was true when we compared the estimated CLGSI and LGSI selection responses (<a id="xref-table-wrap-4-9" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-6-4" class="xref-table" href="#T6">6</a>). This means that the two sets of constraints imposed on the expected genetic gains of both indices did not affect the selection response of the indices, at least for this dataset.</p></div></div></div><div class="section discussion" id="sec-25"><h2 class="">Discussion</h2><div id="sec-26" class="subsection"><h3>CLGSI (CLPSI) expected genetic gain per trait and selection response</h3><p id="p-69">For the two real and simulated datasets, the CLGSI and CLPSI results indicated that CLGSI was more efficient than CLPSI per unit of time, but not per selection cycle, when the criterion for comparing the efficiency of both indices was the estimated selection response. However, when the criterion used to compare the efficiency of both indices was the estimated expected genetic gain per trait, the CLGSI was more efficient than the CLPSI per unit of time and per selection cycle. In addition, we found that when we compared the estimated CLPSI selection response with the LPSI selection response, they were very similar. The same was true for CLGSI and LGSI when we compared their selection responses. This means that the two sets of constraints imposed on the CLGSI (CLPSI) when we obtained its vector of coefficients, mainly affected the estimated and maximized CLGSI (CLPSI) expected genetic gain per trait, not its estimated selection response.</p></div><div id="sec-27" class="subsection"><h3>Interval between selection cycles</h3><p id="p-70">For the two real and simulated datasets, the time required to conduct a selection cycle is a function of the time required to collect the data needed to estimate the index parameters. For CLPSI, the interval between selection cycles was 4, while for the CLGSI it was 1.5 (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> 2015</a>). The interval between selection cycles was the main factor that made CLGSI a more efficient index than CLPSI when we measured the efficiency of both indices by the genetic gains per year. It could also be the main justification for implementing genomic selection programs instead of phenotypic selection programs.</p></div><div id="sec-28" class="subsection"><h3>Accuracy of the GEBV and the CLGSI values</h3><p id="p-71">For the simulated datasets, the average of the accuracy of the GEBV and the estimated correlation of the CLGSI with the net genetic merit were higher than or equal to 0.5. The accuracy of the GEBV in cycle 2 was 0.67, while in cycle 7 it was 0.5. The average of the estimated correlation of the CLGSI with the net genetic merit (data not presented) for the seven selection cycles was 0.8 for two constraints and 0.72 for three constraints. Thus, the correlation between the estimated CLGSI and the net genetic merit decreases when the number of constraints increased. <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Céron-Rojas and Crossa (2018</a>; Chapters 5 and 6) have given a method for estimating the correlation of the CLGSI with the net genetic merit. In addition, <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Cerón-Rojas <em>et al.</em> (2016)</a> found similar results in the CLPSI context. These results indicated the reliability of the GEBV for predicting the breeding value of the traits and of the CLGSI for predicting the net genetic merit; thus breeders can use the CLGSI as a good option for performing GS when there are genotyped individuals.</p></div><div id="sec-29" class="subsection"><h3>The multivariate normality assumption</h3><p id="p-72">Based on the normality assumption of the estimated CLGSI (CLPSI) and GEBV values, we developed and applied the CLGSI (CLPSI) to real and simulated data. The histograms of the GEBV and the CLGSI values indicated that these values approached the normal distribution. The multivariate normality distribution is very important for breeding plant and animal quantitative traits because these traits show continuous variability and are the result of many gene effects interacting among themselves and with the environment. That is, quantitative traits are the result of unobservable gene effects distributed across plant or animal genomes, which interact among themselves and with the environment to produce the observable characteristic plant and animal phenotypes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Falconer and Mackay 1996</a>). Under the multivariate normal distribution assumption, the traits under selection can be described using only means, variances, and covariances. In addition, if the traits are not correlated, they are independent; linear combinations of traits are also normal; and even when the trait phenotypic values do not have that distribution, the normal distribution serves as a useful approximation, especially in inferences involving sample mean vectors, which, by the central limit theorem, have multivariate normal distribution (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Rencher 2002</a>). By this reasoning, the fundamental assumptions in this work were that the GEBVs have multivariate normal distribution, while the net genetic merit and the index have bivariate normal distribution. Under the latter assumption, the regression of the net genetic merit on any linear function of the phenotypic values was linear.</p></div><div id="sec-30" class="subsection"><h3>Criteria for evaluating the relative efficiency of the indices</h3><p id="p-73">We used four main criteria to compare CLGSI <em>vs.</em> CLPSI efficiency when performing genomic and phenotypic selection. Those criteria were the estimates of the theta values, the expected genetic gain per trait values, the selection response, and the interval between selection cycles. For real and simulated data, the estimated theta values were always positive, as we would expect. The estimated expected genetic gain per trait indicated how close these estimates were to the constraints imposed by the breeder for each trait, whereas the estimated selection response predicts the mean value of the net genetic merit in the progeny population. The interval between selection cycles is the time required to collect information to evaluate the index and complete one selection cycle. The four criteria were useful for evaluating and comparing the efficiency of both indices.</p></div></div><div class="section conclusions" id="sec-31"><h2 class="">Conclusion</h2><p id="p-74">We compared the relative efficiency of a CLGSI <em>vs.</em> a CLPSI. We determined the efficiency of both indices based on four criteria using real and simulated datasets. In both type of datasets, we found that the CLGSI genetic gain per year was higher than the CLPSI genetic gain per year because the CLGSI interval between selection cycles was shorter than the CLPSI interval. Therefore, breeders should use the CLGSI when performing selection.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-86">We are thankful for the financial support provided by the Bill &amp; Melinda Gates Foundation (for maize and wheat breeding programs), the CIMMYT CGIAR CRP (maize and wheat), as well the USAID projects. We acknowledge the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through Research NFR Grant 267806.</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1</h2><div class="section" id="sec-32"><h3>Genomic breeding values</h3><p id="p-75">In the training population, the <em>i</em><sup>th</sup> phenotypic value (<span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-168.gif"/></noscript></span></span>) can be denoted as <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-169.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-170.gif"/></noscript></span></span> is the breeding value (<em>i.e.</em>, the average additive effects of the genes an individual receives from both parents) and <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-171.gif"/></noscript></span></span> is the residual. It is assumed that <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-172.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-173.gif"/></noscript></span></span> are independent and that both have normal distribution with an expectation equal to zero and variance <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-174.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-175.gif"/></noscript></span></span>, respectively. The vector of genomic breeding values <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-176.gif"/></noscript></span></span> associated with the vector of observations <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-177.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-178.gif"/></noscript></span></span> number of observations for trait i, <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-179.gif"/></noscript></span></span>1, 2,…,t; <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-180.gif"/></noscript></span></span> number of traits) of the candidates for selection for trait i can be written as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-181.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span>where X is an <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-182.gif"/></noscript></span></span> matrix (<span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-183.gif"/></noscript></span></span>number of observations and <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-184.gif"/></noscript></span></span>number of markers in the population) of coded marker values (<span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-185.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-186.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-187.gif"/></noscript></span></span> for genotypes <em>AA</em>, <em>Aa</em>, and <em>aa</em>, respectively) associated with the additive effects of the quantitative trait loci (QTL), and <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-188.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-189.gif"/></noscript></span></span> vector of the additive effects of the QTL associated with markers that affect the <em>i</em><sup>th</sup> trait. It is assumed that <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-190.gif"/></noscript></span></span> has multivariate normal distribution (MVN) with mean <strong>0</strong> and covariance matrix <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-191.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-192.gif"/></noscript></span></span> is the additive genomic variance of <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-193.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-194.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-195.gif"/></noscript></span></span> additive genomic relationship matrix; <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-196.gif"/></noscript></span></span> in an F<sub>2</sub> population; q is the frequency of allele <em>A</em> and <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-197.gif"/></noscript></span></span> is the frequency of allele <em>a</em> in the <em>j</em><sup>th</sup> marker (<span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-198.gif"/></noscript></span></span>).</p><p id="p-76">Let <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-199.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-200.gif"/></noscript></span></span>1, 2,…,t, <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-201.gif"/></noscript></span></span>number of traits; <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-202.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-203.gif"/></noscript></span></span>number of observations) be the <em>j</em><sup>th</sup> element of the vector of true genotypic breeding values <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-204.gif"/></noscript></span></span>; then the covariance between <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-205.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-206.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-207.gif"/></noscript></span></span>. That is, the covariance between <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-208.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-209.gif"/></noscript></span></span> is equal to the additive genomic variance of <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-210.gif"/></noscript></span></span> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dekkers 2007</a>). Let <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-211.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-212.gif"/></noscript></span></span> be <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-213.gif"/></noscript></span></span> vectors of genomic and true breeding values, respectively, for t traits; then<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-214.gif"/></noscript></span><span class="disp-formula-label">(A2)</span></span>is the covariance matrix of genomic breeding values.</p></div><div class="section" id="sec-33"><h3>Estimating marker effects</h3><p id="p-77">Let <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-215.gif"/></noscript></span></span> be a vector <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-216.gif"/></noscript></span></span> associated with t traits. The <em>i</em><sup>th</sup> vector <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-217.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-218.gif"/></noscript></span></span>1, 2,…,t) of marker effects in the training population can be estimated as <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-219.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-220.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-221.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-222.gif"/></noscript></span></span>, and the other parameters were defined earlier. In addition, we can estimate vector <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-223.gif"/></noscript></span></span> in the multi-trait context as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-224.gif"/></noscript></span><span class="disp-formula-label">(A3)</span></span>where <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-225.gif"/></noscript></span></span>, “⊗” denotes the Kronecker product (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Schott 2005</a>); c and X were defined in Equation (A1); <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-226.gif"/></noscript></span></span>, R and C are the residual and genotypic covariance matrices for t traits, respectively; <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-227.gif"/></noscript></span></span>∼ MVN(μ, V) is a vector of size <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-228.gif"/></noscript></span></span>, with covariance matrix <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-229.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-230.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-231.gif"/></noscript></span></span> are identity matrix of size <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-232.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-233.gif"/></noscript></span></span>, respectively; <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-234.gif"/></noscript></span></span> is a vector <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-235.gif"/></noscript></span></span> of means associated with vector y, and 1 is a vector <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-236.gif"/></noscript></span></span> of 1's.</p></div><div class="section" id="sec-34"><h3>Estimating genomic breeding values</h3><p id="p-78">We can estimate the vector <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-237.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-238.gif"/></noscript></span></span> of genomic breeding values for t traits in the testing population as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-239.gif"/></noscript></span><span class="disp-formula-label">(A4)</span></span>Equation (A4) is the vector of GEBV for the multi-trait case and <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-240.gif"/></noscript></span></span> (Equation A3). Thus, in the testing population, the only thing that will change in Equation (A4) will be the coded values in matrix X, while <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-241.gif"/></noscript></span></span> will be the same in each selection cycle. In Equations (A3) and (A4), we assumed that μ, C and R are known.</p></div><div class="section" id="sec-35"><h3>Estimating the genomic covariance matrix Γ</h3><p id="p-79">Let <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-242.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-243.gif"/></noscript></span></span> (Equation A4) be the genomic estimated breeding values (GEBVs) of <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-244.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-245.gif"/></noscript></span></span> (Equation A1), respectively, and denote by <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-246.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-247.gif"/></noscript></span></span> the arithmetic means of the values of <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-248.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-249.gif"/></noscript></span></span>. We can estimate matrix Γ (Equation A2) when there is no phenotypic information, as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-250.gif"/></noscript></span><span class="disp-formula-label">(A5)</span></span>where <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-251.gif"/></noscript></span></span> is the covariance between <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-252.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-253.gif"/></noscript></span></span> values (<span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-254.gif"/></noscript></span></span>1, 2, …, t); g is the number of genotypes; 1 is a <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-255.gif"/></noscript></span></span> vector of 1’s and <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-256.gif"/></noscript></span></span> is the additive genomic relationship matrix (<a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em>, 2015</a>, for details).</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix 2</h2><div class="section" id="sec-36"><h3>The CLGSI vector of coefficients</h3><p id="p-80">Let <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-257.gif"/></noscript></span></span> be a <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Mallard (1972)</a> matrix <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-258.gif"/></noscript></span></span> of constrained values, where <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-259.gif"/></noscript></span></span> is the <em>i</em><sup>th</sup> element of vector <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-260.gif"/></noscript></span></span> (r is the number of traits constrained) of constrained values imposed by the breeder on the expected genetic gain of the trait (Equation 3), and let <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-261.gif"/></noscript></span></span> be a <a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog (1959)</a> matrix <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-262.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-263.gif"/></noscript></span></span>number of traits) of 1’s and 0’s described in <strong>Appendix 3</strong>. To obtain the CLGSI vector of coefficients (β) that maximizes Equations (2) and (3), we will minimize the mean squared difference between <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-264.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-265.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-266.gif"/></noscript></span></span>) with respect to β under the restriction <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-267.gif"/></noscript></span></span>.</p><p id="p-81">Let <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-268.gif"/></noscript></span></span> and suppose that Γ, <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-269.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-270.gif"/></noscript></span></span> and w are known. To minimize <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-271.gif"/></noscript></span></span> under the restriction <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-272.gif"/></noscript></span></span>, we need to minimize the function<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-273.gif"/></noscript></span><span class="disp-formula-label">(A6)</span></span>with respect to vectors β and <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-274.gif"/></noscript></span></span>, where v is a vector of Lagrange multipliers. Equation (A6) derivative results are: <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-275.gif"/></noscript></span></span>, from where the vector that minimizes <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-276.gif"/></noscript></span></span> under the restriction <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-277.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-278.gif"/></noscript></span><span class="disp-formula-label">(A7)</span></span>where <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-279.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-280.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-281.gif"/></noscript></span></span> is an identity matrix of size <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-282.gif"/></noscript></span></span>. If in Equation (A7), <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-283.gif"/></noscript></span></span> for all traits, then <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-284.gif"/></noscript></span></span> and the CLGSI will be similar to the <a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Kempthorne and Nordskog (1959)</a> index, but in the genomic selection context. When <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-285.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-286.gif"/></noscript></span></span> is a null matrix, <span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-287.gif"/></noscript></span></span>, the vector of economic weights or the unconstrained LGSI vector of coefficients (<a id="xref-ref-3-9" class="xref-bibr" href="#ref-3">Ceron-Rojas <em>et al.</em>, 2015</a>). Thus, the CLGSI is more general than the LGSI and includes the LGSI as a particular case.</p></div><div class="section" id="sec-37"><h3>Estimating the CLGSI vector of coefficients</h3><p id="p-82">We can estimate <span class="inline-formula" id="inline-formula-275"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-288.gif"/></noscript></span></span>(Equation A7) as<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-289.gif"/></noscript></span><span class="disp-formula-label">(A8)</span></span>where <span class="inline-formula" id="inline-formula-276"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-290.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-277"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-291.gif"/></noscript></span></span>. We defined all the parameters of Equation (A8) earlier.</p></div><div class="section" id="sec-38"><h3>Estimating the maximized CLGSI selection response and expected genetic gain per trait</h3><p id="p-83">The estimated maximized CLGSI selection response (Equation 5) is<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-292.gif"/></noscript></span><span class="disp-formula-label">(A9)</span></span>while the estimated optimized CLGSI expected genetic gain per trait (Equation 3) is</p><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-293.gif"/></noscript></span><span class="disp-formula-label">(A10)</span></span></div><div class="section" id="sec-39"><h3>Maximized true selection responses</h3><p id="p-84">We obtained the maximized true selection response for the unconstrained linear phenotypic and genomic selection indices (LPSI and LGSI, respectively) as<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-294.gif"/></noscript></span><span class="disp-formula-label">(A11a)</span></span>respectively, where matrices C and Γ are the covariance matrix of true genotype values associated with the phenotypic (C) and the genomic values (Γ, Equation A2) in each selection cycle, k is the selection intensity and w is a vector of economic weights. In addition, the maximized true selection responses for CLPSI and CLGSI are<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-295.gif"/></noscript></span><span class="disp-formula-label">(A11b)</span></span>respectively, where <span class="inline-formula" id="inline-formula-278"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-296.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-296.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-279"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-297.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-297.gif"/></noscript></span></span> are the vector of coefficients; <span class="inline-formula" id="inline-formula-280"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-298.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-298.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-281"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-299.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-299.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-282"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-300.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-300.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-283"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-301.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-301.gif"/></noscript></span></span> (Equation A7). Matrices C and Γ were defined in Equation (A11a). We obtained these last parameters only for the simulated datasets.</p></div></div><div class="section app" id="app-3"><h2 class="">Appendix 3</h2><div class="section" id="sec-40"><h3>The <span class="inline-formula" id="inline-formula-284"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-302.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-302.gif"/></noscript></span></span> matrix of restrictions</h3><p id="p-85">In addition to matrix <strong>D</strong>, to obtain the CLGSI (CLPSI) vector of coefficients, we need the matrix <span class="inline-formula" id="inline-formula-285"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-303.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-303.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-286"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-304.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-304.gif"/></noscript></span></span>(<span class="inline-formula" id="inline-formula-287"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-305.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-305.gif"/></noscript></span></span> number of traits; <span class="inline-formula" id="inline-formula-288"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-306.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-306.gif"/></noscript></span></span>number of traits restricted), which is a matrix of 1’s and 0’s (1 indicates that the trait is restricted and 0 that the trait is not restricted), and depends on the number of restricted traits. Thus, suppose that we restrict only one of t traits; then, we can restrict the first of them as <span class="inline-formula" id="inline-formula-289"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-307.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-307.gif"/></noscript></span></span>, the second as <span class="inline-formula" id="inline-formula-290"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-308.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-308.gif"/></noscript></span></span>, the third as <span class="inline-formula" id="inline-formula-291"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-309.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-309.gif"/></noscript></span></span>, etc. When we restrict two of t traits, matrix <span class="inline-formula" id="inline-formula-292"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-310.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-310.gif"/></noscript></span></span> could be constructed as follows. We can restrict the first and second traits as <span class="inline-formula" id="inline-formula-293"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-311.gif"/></noscript></span></span>, the first and third traits as <span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-312.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-312.gif"/></noscript></span></span>, the second and third traits as <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-313.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-313.gif"/></noscript></span></span>, etc. If we restrict three of t traits, matrix <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-314.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-314.gif"/></noscript></span></span> will have the following form when the first, second and third traits are restricted: <span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-315.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-315.gif"/></noscript></span></span>; if the first, second and fourth traits are restricted, <span class="inline-formula" id="inline-formula-298"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-316.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-316.gif"/></noscript></span></span>, if the second, the third and the fourth traits are restricted, <span class="inline-formula" id="inline-formula-299"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-317.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-317.gif"/></noscript></span></span>, etc. The procedure used to construct matrix <span class="inline-formula" id="inline-formula-300"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-318.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/3981/embed/mml-math-318.gif"/></noscript></span></span> is valid for any number of restricted traits (Cerón-Rojas and Crossa 2018, Chapter 3, for details).</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-22"><p id="p-87"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-08-28"><span class="received-label">Received </span>August 28, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-26"><span class="accepted-label">Accepted </span>September 26, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Ceron, Crossa</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.1" data-doi="10.1186/1297-9686-44-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Börner</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Reinsch</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimising multistage dairy cattle breeding schemes including genomic selection using decorrelated or optimum selection indices.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DBorner%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DOptimising%2Bmultistage%2Bdairy%2Bcattle%2Bbreeding%2Bschemes%2Bincluding%2Bgenomic%2Bselection%2Busing%2Bdecorrelated%2Bor%2Boptimum%2Bselection%2Bindices.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-1%26rft_id%253Dinfo%253Apmid%252F22252172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252172&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.2" data-doi="10.2135/cropsci2014.07.0460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Semagn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mugo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tarekegne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Babu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.07.0460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.07.0460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.07.0460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.3" data-doi="10.1534/g3.115.019869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ceron-Rojas</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. N.</span> <span class="cit-name-surname">Arief</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Basford</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rutkoski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Alvarado</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Beyene</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Semagn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">DeLacy</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> A genomic selection index applied to simulated and real data. G3 (Bethesda) 5: 2155–2164.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.019869</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019869%26rft_id%253Dinfo%253Apmid%252F26290571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMjE1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvMzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.3981.4" data-doi="10.1007/978-3-319-91223-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Céron-Rojas</span>,  <span class="cit-name-given-names">J. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">Linear selection indices in modern plant breeding</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Cham, the Netherlands</span>., Available at <a href="https://link.springer.com/book/10.1007/978%E2%80%933-319%E2%80%9391223%E2%80%933">https://link.springer.com/book/10.1007/978–3-319–91223–3</a><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-91223-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLinear%2Bselection%2Bindices%2Bin%2Bmodern%2Bplant%2Bbreeding%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-91223-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-91223-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.5" data-doi="10.2135/cropsci2015.11.0718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cerón-Rojas</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. H.</span> <span class="cit-name-surname">Toledo</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sahagún-Castellanos</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A predetermined proportional gains eigen selection index method.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2436</span>–<span class="cit-lpage">2447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.11.0718</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2436%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.11.0718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.11.0718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.6" data-doi="10.1111/j.1439-0388.2007.00701.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>,  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prediction of response to marker-assisted and genomic selection using selection index theory.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00701.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DDekkers%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D331%26rft.epage%253D341%26rft.atitle%253DPrediction%2Bof%2Bresponse%2Bto%2Bmarker-assisted%2Band%2Bgenomic%2Bselection%2Busing%2Bselection%2Bindex%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00701.x%26rft_id%253Dinfo%253Apmid%252F18076470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00701.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.3981.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to quantitative genetics</span>, <span class="cit-publ-name">Longman</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.8" data-doi="10.2307/2529722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harville</span>,  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Index selection with proportionality constraints.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">225</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2529722</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DKemp%26rft.auinit1%253DC.%2BD.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D223%26rft.epage%253D225%26rft.atitle%253DIndex%2Bselection%2Bwith%2Bproportionality%2Bconstraints.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529722%26rft_id%253Dinfo%253Apmid%252F1164534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1164534&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hazel</span>,  <span class="cit-name-given-names">L. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">The genetic basis for constructing selection indexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.10" data-doi="10.1186/1297-9686-19-1-69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yamada</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Comparisons of selection indices achieving predetermined proportional gains.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-19-1-69</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D19%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-19-1-69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-19-1-69&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.3981.11" data-doi="10.1007/978-3-319-55177-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isik</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Holland</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maltecca</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Genetic Data Analysis for Plant and Animal Breeding</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Cham, the Netherlands</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-55177-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BData%2BAnalysis%2Bfor%2BPlant%2Band%2BAnimal%2BBreeding%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-55177-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-55177-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.12" data-doi="10.2307/2527598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>,  <span class="cit-name-given-names">O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Nordskog</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Restricted selection indices.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2527598</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D15%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2527598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2527598&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.13" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. G.</span> <span class="cit-name-surname">Asoro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Heffner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hayashi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-385531-2.00002-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.14" data-doi="10.2307/2528758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mallard</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">The theory and computation of selection indices with constraints: A critical synthesis.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">735</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2528758</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D28%26rft.spage%253D713%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528758&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.15" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzM5ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.16" data-doi="10.1093/bioinformatics/14.7.632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>,  <span class="cit-name-given-names">D. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cooper</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">QU-GENE: a simulation platform for quantitative analysis of genetic models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">653</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/14.7.632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DPodlich%26rft.auinit1%253DD.%2BW.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D632%26rft.epage%253D653%26rft.atitle%253DQU-GENE%253A%2Ba%2Bsimulation%2Bplatform%2Bfor%2Bquantitative%2Banalysis%2Bof%2Bgenetic%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.7.632%26rft_id%253Dinfo%253Apmid%252F9730929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/14.7.632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9730929&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076226300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.3981.17" data-doi="10.1002/0471271357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rencher</span>,  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Methods of multivariate analysis</span>, <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471271357</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bof%2Bmultivariate%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471271357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471271357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.3981.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schott</span>,  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Matrix analysis for statistics</span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, New Jersey</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.19" data-doi="10.1111/j.1439-0388.2006.00595.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>,  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Strategy for applying genome-wide selection in dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2006.00595.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D218%26rft.epage%253D223%26rft.atitle%253DStrategy%2Bfor%2Bapplying%2Bgenome-wide%2Bselection%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2006.00595.x%26rft_id%253Dinfo%253Apmid%252F16882088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2006.00595.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16882088&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239006300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.20" data-doi="10.1266/jjg.60.151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tallis</span>,  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Constrained selection.</span> <abbr class="cit-jnl-abbrev">Jpn. J. Genet.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1266/jjg.60.151</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJpn.%2BJ.%2BGenet.%26rft.volume%253D60%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fjjg.60.151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/jjg.60.151&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.21" data-doi="10.2527/jas.2009-2606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Togashi</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. Y.</span> <span class="cit-name-surname">Lin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yamazaki</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The efficiency of genome-wide selection for genetic improvement of net merit.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2972</span>–<span class="cit-lpage">2980</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2009-2606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-2606%26rft_id%253Dinfo%253Apmid%252F21512116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-2606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21512116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3981.22" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/3981.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/3973" title="Genomics Analysis of L-DOPA Exposure in Drosophila sechellia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/3995" title="Natural Genetic Variation Screen in Drosophila Identifies Wnt Signaling, Mitochondrial Metabolism, and Redox Homeostasis Genes as Modifiers of Apoptosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/3981" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-t83fuv-uBOwq3TMSmZ2hCyAFJogIKxbYcEaj8IAhR-o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/3981" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/3981" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/3981" class="active" data-icon-position="" data-hide-link-title="0">Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494999" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zkQRxyeOY7QJnhM2DliAi_bR_LoxMLK6Z1wPvZRzStg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/3981.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/3981" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-TszwLtk8ZxE_VIVB_V2rshmmkqiwR4mOFjZQo56s7z0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/3981" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/3981.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494999" id="node494999--2" data-pisa="ggg;9/12/3981" data-pisa-master="ggg;g3.119.400677" data-apath="/ggg/9/12/3981.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2885-6831" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. Jesus Cerón-Rojas</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3981-3994;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400677 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494999--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">J. Jesus Cerón-Rojas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJesus%2BCer%C3%B3n-Rojas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cer%C3%B3n-Rojas%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BJesus%2BCer%25C3%25B3n-Rojas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2885-6831" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. Jesus Cerón-Rojas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/3981" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/3981&amp;title=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/3981&amp;title=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/3981&amp;title=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/3981&amp;text=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/3981&amp;title=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/3981&amp;t=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/3981&amp;title=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/3981&amp;title=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494999/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494999" id="node494999--3" data-pisa="ggg;9/12/3981" data-pisa-master="ggg;g3.119.400677" data-apath="/ggg/9/12/3981.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Efficiency of a Constrained Linear Genomic Selection Index To Predict the Net Genetic Merit in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2885-6831" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. Jesus Cerón-Rojas</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3981-3994;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400677 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494999--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">J. Jesus Cerón-Rojas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJesus%2BCer%C3%B3n-Rojas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cer%C3%B3n-Rojas%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BJesus%2BCer%25C3%25B3n-Rojas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2885-6831" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. Jesus Cerón-Rojas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico City, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3981&amp;rft.epage=3994&amp;rft.atitle=Efficiency%20of%20a%20Constrained%20Linear%20Genomic%20Selection%20Index%20To%20Predict%20the%20Net%20Genetic%20Merit%20in%20Plants&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cer%C3%B3n-Rojas%2C+J.+Jesus&amp;rft.au=Crossa%2C+Jose"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494999/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494999/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494999/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494999/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494999/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494999/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494999/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494999/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494999/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494999/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494999/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494999/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/3981" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-25" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#sec-31" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 2</a></li><li class="even"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 3</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/3981.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/3981.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/3981.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
