<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/4183" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/4183.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/4183.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/4183.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/12/4183/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/4183.atom" />
<meta name="HW.pisa" content="ggg;9/12/4183" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Classification-Based Inference of Dynamical Models of Gene Regulatory Networks" />
<meta name="DC.Identifier" content="10.1534/g3.119.400603" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Fehr et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cell-fate decisions during development are controlled by densely interconnected gene regulatory networks (GRNs) consisting of many genes. Inferring and predictively modeling these GRNs is crucial for understanding development and other physiological processes. Gene circuits, coupled differential equations that represent gene product synthesis with a switch-like function, provide a biologically realistic framework for modeling the time evolution of gene expression. However, their use has been limited to smaller networks due to the computational expense of inferring model parameters from gene expression data using global non-linear optimization. Here we show that the switch-like nature of gene regulation can be exploited to break the gene circuit inference problem into two simpler optimization problems that are amenable to computationally efficient supervised learning techniques. We present FIGR (Fast Inference of Gene Regulation), a novel classification-based inference approach to determining gene circuit parameters. We demonstrate FIGR’s effectiveness on synthetic data generated from random gene circuits of up to 50 genes as well as experimental data from the gap gene system of Drosophila melanogaster , a benchmark for inferring dynamical GRN models. FIGR is faster than global non-linear optimization by a factor of 600 and its computational complexity scales much better with GRN size. On a practical level, FIGR can accurately infer the biologically realistic gap gene network in under a minute on desktop-class hardware instead of requiring hours of parallel computing. We anticipate that FIGR would enable the inference of much larger biologically realistic GRNs than was possible before." />
<meta name="DC.Contributor" content="David A. Fehr" />
<meta name="DC.Contributor" content="Joanna E. Handzlik" />
<meta name="DC.Contributor" content="Manu" />
<meta name="DC.Contributor" content="Yen Lee Loh" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Classification-Based Inference of Dynamical Models of Gene Regulatory Networks" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cell-fate decisions during development are controlled by densely interconnected gene regulatory networks (GRNs) consisting of many genes. Inferring and predictively modeling these GRNs is crucial for understanding development and other physiological processes. Gene circuits, coupled differential equations that represent gene product synthesis with a switch-like function, provide a biologically realistic framework for modeling the time evolution of gene expression. However, their use has been limited to smaller networks due to the computational expense of inferring model parameters from gene expression data using global non-linear optimization. Here we show that the switch-like nature of gene regulation can be exploited to break the gene circuit inference problem into two simpler optimization problems that are amenable to computationally efficient supervised learning techniques. We present FIGR (Fast Inference of Gene Regulation), a novel classification-based inference approach to determining gene circuit parameters. We demonstrate FIGR’s effectiveness on synthetic data generated from random gene circuits of up to 50 genes as well as experimental data from the gap gene system of &lt;i&gt;Drosophila melanogaster&lt;/i&gt;, a benchmark for inferring dynamical GRN models. FIGR is faster than global non-linear optimization by a factor of 600 and its computational complexity scales much better with GRN size. On a practical level, FIGR can accurately infer the biologically realistic gap gene network in under a minute on desktop-class hardware instead of requiring hours of parallel computing. We anticipate that FIGR would enable the inference of much larger biologically realistic GRNs than was possible before.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/4183" />
<meta name="citation_id" content="9/12/4183" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/4183" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/4183.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/4183.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/4183.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400603" />
<meta name="citation_pmid" content="31624138" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="4183" />
<meta name="citation_lastpage" content="4195" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="David A. Fehr" />
<meta name="citation_author_institution" content="Department of Physics and Astrophysics, and" />
<meta name="citation_author" content="Joanna E. Handzlik" />
<meta name="citation_author_institution" content="Department of Biology, University of North Dakota, Grand Forks, ND 58202" />
<meta name="citation_author" content="Manu" />
<meta name="citation_author_institution" content="Department of Biology, University of North Dakota, Grand Forks, ND 58202" />
<meta name="citation_author_email" content="manu.manu@und.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7677-7228" />
<meta name="citation_author" content="Yen Lee Loh" />
<meta name="citation_author_institution" content="Department of Physics and Astrophysics, and" />
<meta name="citation_author_email" content="yenlee.loh@und.edu" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=M. Akam;citation_title=The molecular basis for metameric pattern in the Drosophila embryo;citation_pages=1-22;citation_volume=101;citation_year=1987;citation_issue=1;citation_pmid=2896587;citation_doi=10.1038/msb.2012.72" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=M. Ashyraliyev;citation_author=J. Jaeger;citation_author=J. G. Blom;citation_title=Parameter estimation and determinability analysis applied to Drosophila gap gene circuits.;citation_pages=83-83;citation_volume=2;citation_year=2008;citation_pmid=18817540;citation_doi=10.1186/1752-0509-2-83" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=M. Ashyraliyev;citation_author=K. Siggens;citation_author=H. Janssens;citation_author=J. Blom;citation_author=M. Akam;citation_title=Gene circuit analysis of the terminal gap gene huckebein.;citation_pages=e1000548-e1000548;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19876378;citation_doi=10.1371/journal.pcbi.1000548" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Balaskas;citation_author=A. Ribeiro;citation_author=J. Panovska;citation_author=E. Dessaud;citation_author=N. Sasai;citation_title=Gene regulatory logic for reading the Sonic Hedgehog signaling gradient in the vertebrate neural tube.;citation_pages=273-284;citation_volume=148;citation_year=2012;citation_issue=1-2;citation_pmid=22265416;citation_doi=10.1016/j.cell.2011.10.047" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Bonzanni;citation_author=A. Garg;citation_author=K. A. Feenstra;citation_author=J. Schütte;citation_author=S. Kinston;citation_title=Hard-wired heterogeneity in blood stem cells revealed using a dynamic regulatory network model.;citation_volume=29;citation_year=2013;citation_pmid=23813012;citation_doi=10.1093/bioinformatics/btt243" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Phys.;citation_author=K. W. Chu;citation_author=Y. Deng;citation_author=J. Reinitz;citation_title=Parallel simulated annealing by mixing of states.;citation_pages=646-662;citation_volume=148;citation_year=1999;citation_doi=10.1006/jcph.1998.6134" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Collombet;citation_author=C. van Oevelen;citation_author=J. L. Sardina Ortega;citation_author=W. Abou-Jaoudé;citation_author=B. Di Stefano;citation_title=Logical modeling of lymphoid and myeloid cell specification and transdifferentiation.;citation_volume=114;citation_year=2017;citation_pmid=28584084;citation_doi=10.1073/pnas.1610622114" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=J. Cotterell;citation_author=J. Sharpe;citation_title=An atlas of gene regulatory networks reveals multiple three-gene mechanisms for interpreting morphogen gradients;citation_pages=425-425;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=21045819;citation_doi=10.1038/msb.2010.74" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=A. Crombach;citation_author=K. R. Wotton;citation_author=D. Cicin-Sain;citation_author=M. Ashyraliyev;citation_author=J. Jaeger;citation_title=Efficient reverse-engineering of a developmental gene regulatory network.;citation_pages=e1002589-e1002589;citation_volume=8;citation_year=2012;citation_issue=7;citation_pmid=22807664;citation_doi=10.1371/journal.pcbi.1002589" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. H. Davidson;citation_author=M. S. Levine;citation_title=Properties of developmental gene regulatory networks.;citation_volume=105;citation_year=2008;citation_pmid=19104053;citation_doi=10.1073/pnas.0806007105" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. H. Davidson;citation_author=J. P. Rast;citation_author=P. Oliveri;citation_author=A. Ransick;citation_author=C. Calestani;citation_title=A Genomic Regulatory Network for Development;citation_pages=1669-1678;citation_volume=295;citation_year=2002;citation_issue=5560;citation_pmid=11872831;citation_doi=10.1126/science.1069883" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=E. H. Davidson;citation_author=J. P. Rast;citation_author=P. Oliveri;citation_author=A. Ransick;citation_author=C. Calestani;citation_title=A provisional regulatory gene network for specification of endomesoderm in the sea urchin embryo.;citation_pages=162-190;citation_volume=246;citation_year=2002;citation_issue=1;citation_pmid=12027441;citation_doi=10.1006/dbio.2002.0635" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. Driever;citation_author=C. Nüsslein-Volhard;citation_title=The bicoid protein is a positive regulator of hunchback transcription in the early Drosophila embryo.;citation_pages=138-143;citation_volume=337;citation_year=1989;citation_issue=6203;citation_pmid=2911348;citation_doi=10.1038/337138a0" />
<meta name="citation_reference" content="citation_journal_title=Physica D;citation_author=R. Edwards;citation_title=Analysis of continuous-time switching networks.;citation_pages=165-199;citation_volume=146;citation_year=2000;citation_doi=10.1016/S0167-2789(00)00130-5" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=E. D. Eldon;citation_author=V. Pirrotta;citation_title=Interactions of the Drosophila gap gene giant with maternal and zygotic pattern-forming genes;citation_pages=367-378;citation_volume=111;citation_year=1991;citation_issue=2;citation_pmid=1716553" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=L. Glass;citation_author=S. A. Kauffman;citation_title=The logical analysis of continuous, non-linear biochemical control networks.;citation_pages=103-129;citation_volume=39;citation_year=1973;citation_issue=1;citation_pmid=4741704;citation_doi=10.1016/0022-5193(73)90208-7" />
<meta name="citation_reference" content="citation_journal_title=J. Math. Biol.;citation_author=L. Glass;citation_author=J. Pasternack;citation_title=Stable oscillations in mathematical models of biological control systems.;citation_pages=207-223;citation_volume=6;citation_year=1978;citation_doi=10.1007/BF02547797" />
<meta name="citation_reference" content="citation_journal_title=Physica D;citation_author=V. V. Gursky;citation_author=J. Jaeger;citation_author=K. N. Kozlov;citation_author=J. Reinitz;citation_author=A. M. Samsonova;citation_title=Pattern formation and nuclear divisions are uncoupled in Drosophila segmentation: comparison of spatially discrete and continuous models.;citation_pages=286-302;citation_volume=197;citation_year=2004;citation_doi=10.1016/j.physd.2004.07.003" />
<meta name="citation_reference" content="citation_journal_title=Biofizika (Moskva);citation_journal_abbrev=Biofizika (Moskva);citation_author=V. V. Gursky;citation_author=K. N. Kozlov;citation_author=A. M. Samsonov;citation_author=J. Reinitz;citation_title=[A model with asymptotically stable dynamics for the network of Drosophila gap genes];citation_pages=235-249;citation_volume=53;citation_year=2008;citation_issue=2;citation_pmid=18543766;citation_doi=10.1134/S0006350908020085" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=V. V. Gursky;citation_author=L. Panok;citation_author=E. M. Myasnikova;citation_author=M. Manu;citation_author=M. G. Samsonova;citation_title=Mechanisms of gap gene expression canalization in the Drosophila blastoderm.;citation_pages=118-118;citation_volume=5;citation_year=2011;citation_pmid=21794172;citation_doi=10.1186/1752-0509-5-118" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=R. N. Gutenkunst;citation_author=J. J. Waterfall;citation_author=F. P. Casey;citation_author=K. S. Brown;citation_author=C. R. Myers;citation_title=Universally sloppy parameter sensitivities in systems biology models.;citation_pages=1871-1878;citation_volume=3;citation_year=2007;citation_issue=10;citation_pmid=17922568;citation_doi=10.1371/journal.pcbi.0030189" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. K. Hamey;citation_author=S. Nestorowa;citation_author=S. J. Kinston;citation_author=D. G. Kent;citation_author=N. K. Wilson;citation_title=Reconstructing blood stem cell regulatory network models from single-cell molecular profiles.;citation_volume=114;citation_year=2017;citation_pmid=28584094;citation_doi=10.1073/pnas.1610609114" />
<meta name="citation_reference" content="citation_title=The elements of statistical learning: data mining, inference, and prediction;citation_author=T. J. Hastie;citation_author=R. J. Tibshirani;citation_author=J. H. Friedman;citation_year=2009;citation_doi=10.1007/978-0-387-84858-7" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. B. Hengenius;citation_author=M. Gribskov;citation_author=A. E. Rundell;citation_author=C. C. Fowlkes;citation_author=D. M. Umulis;citation_title=Analysis of gap gene regulation in a 3D organism-scale model of the Drosophila melanogaster embryo.;citation_pages=e26797-e26797;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22110594;citation_doi=10.1371/journal.pone.0026797" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=M. Hoch;citation_author=E. Seifert;citation_author=H. Jäckle;citation_title=Gene expression mediated by cis-acting sequences of the Kruppel gene in response to the Drosophila morphogens bicoid and hunchback;citation_pages=2267-2278;citation_volume=10;citation_year=1991;citation_issue=8;citation_pmid=2065664;citation_doi=10.1002/j.1460-2075.1991.tb07763.x" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=T. Hong;citation_author=J. Xing;citation_author=L. Li;citation_author=J. J. Tyson;citation_title=A simple theoretical framework for understanding heterogeneous differentiation of CD4+ T cells.;citation_pages=66-66;citation_volume=6;citation_year=2012;citation_pmid=22697466;citation_doi=10.1186/1752-0509-6-66" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=S. Huang;citation_author=Y. Guo;citation_author=G. May;citation_author=T. Enver;citation_title=Bifurcation dynamics in lineage-commitment in bipotent progenitor cells.;citation_pages=695-713;citation_volume=305;citation_year=2007;citation_issue=2;citation_pmid=17412320;citation_doi=10.1016/j.ydbio.2007.02.036" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Hülskamp;citation_author=W. Lukowitz;citation_author=A. Beermann;citation_author=G. Glaser;citation_author=D. Tautz;citation_title=Differential regulation of target genes by different alleles of the segmentation gene hunchback in Drosophila.;citation_pages=125-134;citation_volume=138;citation_year=1994;citation_pmid=8001780" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=J. Jaeger;citation_title=The gap gene network.;citation_pages=243-274;citation_volume=68;citation_year=2011;citation_issue=2;citation_pmid=20927566;citation_doi=10.1007/s00018-010-0536-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Jaeger;citation_author=M. Blagov;citation_author=D. Kosman;citation_author=K. N. Kozlov;citation_author=E. Manu;citation_title=Dynamical Analysis of Regulatory Interactions in the Gap Gene System of Drosophila melanogaster;citation_pages=1721-1737;citation_volume=167;citation_year=2004;citation_issue=4;citation_pmid=15342511;citation_doi=10.1534/genetics.104.027334" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of development;citation_journal_abbrev=Mech Dev;citation_author=J. Jaeger;citation_author=D. H. Sharp;citation_author=J. Reinitz;citation_title=Known maternal gradients are not sufficient for the establishment of gap domains in Drosophila melanogaster.;citation_pages=108-128;citation_volume=124;citation_year=2007;citation_issue=2;citation_pmid=17196796;citation_doi=10.1016/j.mod.2006.11.001" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Jaeger;citation_author=S. Surkova;citation_author=M. Blagov;citation_author=H. Janssens;citation_author=D. Kosman;citation_title=Dynamic control of positional information in the early Drosophila embryo.;citation_pages=368-371;citation_volume=430;citation_year=2004;citation_issue=6997;citation_pmid=15254541;citation_doi=10.1038/nature02678" />
<meta name="citation_reference" content="citation_journal_title=Development genes and evolution;citation_journal_abbrev=Dev Genes Evol;citation_author=H. Janssens;citation_author=D. Kosman;citation_author=C. E. Vanario-Alonso;citation_author=J. Jaeger;citation_author=M. Samsonova;citation_title=A high-throughput method for quantifying gene expression data from early Drosophila embryos.;citation_pages=374-381;citation_volume=215;citation_year=2005;citation_issue=7;citation_pmid=15834586;citation_doi=10.1007/s00427-005-0484-y" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=L. Jostins;citation_author=J. Jaeger;citation_title=Reverse engineering a gene network using an asynchronous parallel evolution strategy.;citation_pages=17-17;citation_volume=4;citation_year=2010;citation_pmid=20196855;citation_doi=10.1186/1752-0509-4-17" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kirkpatrick;citation_author=C. D. Gelatt;citation_author=M. P. Vecchi;citation_title=Optimization by Simulated Annealing;citation_pages=671-680;citation_volume=220;citation_year=1983;citation_issue=4598;citation_pmid=17813860;citation_doi=10.1126/science.220.4598.671" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kondo;citation_author=T. Miura;citation_title=Reaction-Diffusion Model as a Framework for Understanding Biological Pattern Formation;citation_pages=1616-1620;citation_volume=329;citation_year=2010;citation_issue=5999;citation_pmid=20929839;citation_doi=10.1126/science.1179047" />
<meta name="citation_reference" content="citation_title=Parallel Computing Technologies;citation_author=K. Kozlov;citation_author=A. Samsonov;citation_title=Deep - differential evolution entirely parallel method for gene regulatory networks;citation_year=2009;citation_doi=10.1007/978-3-642-03275-2_13" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=K. Kozlov;citation_author=S. Surkova;citation_author=E. Myasnikova;citation_author=J. Reinitz;citation_author=M. Samsonova;citation_title=Modeling of gap gene expression in Drosophila Kruppel mutants.;citation_pages=e1002635-e1002635;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22927803;citation_doi=10.1371/journal.pcbi.1002635" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=R. Kraut;citation_author=M. Levine;citation_title=Mutually repressive interactions between the gap genes giant and Kruppel define middle body regions of the Drosophila embryo;citation_pages=611-621;citation_volume=111;citation_year=1991;citation_issue=2;citation_pmid=1893878" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. Y. Kueh;citation_author=A. Champhekhar;citation_author=S. L. Nutt;citation_author=M. B. Elowitz;citation_author=E. V. Rothenberg;citation_title=Positive feedback between pu.1 and the cell cycle controls myeloid differentiation.;citation_volume=341;citation_year=2013;citation_pmid=23868921;citation_doi=10.1126/science.1240831" />
<meta name="citation_reference" content="Lam, J., and J.-M. Delosme, 1988a An efficient simulated annealing schedule: Derivation. Technical Report 8816, Yale Electrical Engineering Department, New Haven, CT." />
<meta name="citation_reference" content="Lam, J., and J.-M. Delosme, 1988b An efficient simulated annealing schedule: Implementation and evaluation. Technical Report 8817, Yale Electrical Engineering Department, New Haven, CT." />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. Laslo;citation_author=C. J. Spooner;citation_author=A. Warmflash;citation_author=D. W. Lancki;citation_author=H.-J. Lee;citation_title=Multilineage transcriptional priming and determination of alternate hematopoietic cell fates.;citation_pages=755-766;citation_volume=126;citation_year=2006;citation_issue=4;citation_pmid=16923394;citation_doi=10.1016/j.cell.2006.06.052" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Levine;citation_author=E. H. Davidson;citation_title=Gene regulatory networks for development.;citation_volume=102;citation_year=2005;citation_pmid=15788537;citation_doi=10.1073/pnas.0408031102" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=C. Li;citation_author=J. Wang;citation_title=Quantifying cell fate decisions for differentiation and reprogramming of a human stem cell network: Landscape and biological paths.;citation_pages=e1003165;citation_volume=9;citation_year=2013;citation_pmid=23935477;citation_doi=10.1371/journal.pcbi.1003165" />
<meta name="citation_reference" content="citation_journal_title=Parallel Comput.;citation_author=Z. Lou;citation_author=J. Reinitz;citation_title=Parallel simulated annealing using an adaptive resampling interval.;citation_pages=23-31;citation_volume=53;citation_year=2016;citation_doi=10.1016/j.parco.2016.02.001" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. Manu;citation_author=S. Surkova;citation_author=A. V. Spirov;citation_author=V. V. Gursky;citation_author=H. Janssens;citation_title=Canalization of gene expression and domain shifts in the Drosophila blastoderm by dynamical attractors.;citation_pages=e1000303-e1000303;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19282965;citation_doi=10.1371/journal.pcbi.1000303" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. Manu;citation_author=S. Surkova;citation_author=A. V. Spirov;citation_author=V. V. Gursky;citation_author=H. Janssens;citation_title=Canalization of gene expression in the Drosophila blastoderm by gap gene cross regulation.;citation_pages=e1000049-e1000049;citation_volume=7;citation_year=2009;citation_issue=3;citation_pmid=19750121;citation_doi=10.1371/journal.pbio.1000049" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=A. A. Margolin;citation_author=I. Nemenman;citation_author=K. Basso;citation_author=C. Wiggins;citation_author=G. Stolovitzky;citation_title=Aracne: an algorithm for the reconstruction of gene regulatory networks in a mammalian cellular context.;citation_pages=S7;citation_volume=7;citation_year=2006;citation_pmid=16723010;citation_doi=10.1186/1471-2105-7-S1-S7" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=J. S. Margolis;citation_author=M. L. Borowsky;citation_author=E. Steingrimsson;citation_author=C. W. Shim;citation_author=J. A. Lengyel;citation_title=Posterior stripe expression of hunchback is driven from two promoters by a common enhancer element;citation_pages=3067-3077;citation_volume=121;citation_year=1995;citation_issue=9;citation_pmid=7555732" />
<meta name="citation_reference" content="citation_journal_title=Cell Stem Cell;citation_author=G. May;citation_author=S. Soneji;citation_author=A. J. Tipping;citation_author=J. Teles;citation_author=S. J. McGowan;citation_title=Dynamic analysis of gene expression and genome-wide transcription factor binding during lineage specification of multipotent progenitors.;citation_pages=754-768;citation_volume=13;citation_year=2013;citation_pmid=24120743;citation_doi=10.1016/j.stem.2013.09.003" />
<meta name="citation_reference" content="citation_journal_title=Physica D;citation_author=T. Mestl;citation_author=C. Lemay;citation_author=L. Glass;citation_title=Chaos in high-dimensional neural and gene networks.;citation_pages=33-52;citation_volume=98;citation_year=1996;citation_doi=10.1016/0167-2789(96)00086-3" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Novershtern;citation_author=A. Subramanian;citation_author=L. N. Lawton;citation_author=R. H. Mak;citation_author=W. N. Haining;citation_title=Densely interconnected transcriptional circuits control cell states in human hematopoiesis.;citation_pages=296-309;citation_volume=144;citation_year=2011;citation_issue=2;citation_pmid=21241896;citation_doi=10.1016/j.cell.2011.01.004" />
<meta name="citation_reference" content="citation_journal_title=Biophysical Journal;citation_journal_abbrev=Biophys. J;citation_author=S. Palani;citation_author=C. A. Sarkar;citation_title=Positive receptor feedback during lineage commitment can generate ultrasensitivity to ligand and confer robustness to a bistable switch.;citation_pages=1575-1589;citation_volume=95;citation_year=2008;citation_issue=4;citation_pmid=18469073;citation_doi=10.1529/biophysj.107.120600" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=T. J. Perkins;citation_author=J. Jaeger;citation_author=J. Reinitz;citation_author=L. Glass;citation_title=Reverse engineering the gap gene network of Drosophila melanogaster.;citation_pages=e51-e51;citation_volume=2;citation_year=2006;citation_issue=5;citation_pmid=16710449;citation_doi=10.1371/journal.pcbi.0020051" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=A. Pietak;citation_author=J. Bischof;citation_author=J. LaPalme;citation_author=J. Morokuma;citation_author=M. Levin;citation_title=Neural control of body-plan axis in regenerating planaria.;citation_pages=e1006904;citation_volume=15;citation_year=2019;citation_doi=10.1371/journal.pcbi.1006904" />
<meta name="citation_reference" content="citation_journal_title=The Journal of experimental zoology;citation_journal_abbrev=J Exp Zool;citation_author=J. Reinitz;citation_author=E. Mjolsness;citation_author=D. H. Sharp;citation_title=Model for cooperative control of positional information in Drosophila by bicoid and maternal hunchback.;citation_pages=47-56;citation_volume=271;citation_year=1995;citation_issue=1;citation_pmid=7852948;citation_doi=10.1002/jez.1402710106" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of development;citation_journal_abbrev=Mech Dev;citation_author=J. Reinitz;citation_author=D. H. Sharp;citation_title=Mechanism of eve stripe formation.;citation_pages=133-158;citation_volume=49;citation_year=1995;citation_issue=1-2;citation_pmid=7748785;citation_doi=10.1016/0925-4773(94)00310-J" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Rivera-Pomar;citation_author=X. Lu;citation_author=N. Perrimon;citation_author=H. Taubert;citation_author=H. Jäckle;citation_title=Activation of posterior gap gene expression in the Drosophila blastoderm.;citation_pages=253-256;citation_volume=376;citation_year=1995;citation_issue=6537;citation_pmid=7617036;citation_doi=10.1038/376253a0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=L. Sánchez;citation_author=D. Thieffry;citation_title=A logical analysis of the Drosophila gap-gene system.;citation_pages=115-141;citation_volume=211;citation_year=2001;citation_issue=2;citation_pmid=11419955;citation_doi=10.1006/jtbi.2001.2335" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=C. Schulz;citation_author=D. Tautz;citation_title=Zygotic caudal regulation by hunchback and its role in abdominal segment formation of the Drosophila embryo;citation_pages=1023-1028;citation_volume=121;citation_year=1995;citation_issue=4;citation_pmid=7743918" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Segal;citation_author=M. Shapira;citation_author=A. Regev;citation_author=D. Pe’er;citation_author=D. Botstein;citation_title=Module networks: identifying regulatory modules and their condition-specific regulators from gene expression data.;citation_pages=166-176;citation_volume=34;citation_year=2003;citation_issue=2;citation_pmid=12740579;citation_doi=10.1038/nm0203-166" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=A. A. Simcox;citation_author=J. H. Sang;citation_title=When does determination occur in Drosophila embryos?;citation_pages=212-221;citation_volume=97;citation_year=1983;citation_issue=1;citation_pmid=6404675;citation_doi=10.1016/0012-1606(83)90078-7" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Simpson-Brose;citation_author=J. Treisman;citation_author=C. Desplan;citation_title=Synergy between the hunchback and bicoid morphogens is required for anterior patterning in Drosophila.;citation_pages=855-865;citation_volume=78;citation_year=1994;citation_issue=5;citation_pmid=8087852;citation_doi=10.1016/S0092-8674(94)90622-X" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=S. Surkova;citation_author=D. Kosman;citation_author=K. Kozlov;citation_author=E. Manu;citation_author=E. Myasnikova;citation_title=Characterization of the Drosophila segment determination morphome.;citation_pages=844-862;citation_volume=313;citation_year=2008;citation_issue=2;citation_pmid=18067886;citation_doi=10.1016/j.ydbio.2007.10.037" />
<meta name="citation_reference" content="citation_journal_title=Mathematical Modelling and Scientific Computing;citation_author=D. Theiffry;citation_author=M. Colet;citation_author=R. Thomas;citation_title=Formalization of regulatory networks: A logical method and its automatization.;citation_pages=144-151;citation_volume=2;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=D. Thieffry;citation_author=L. Sánchez;citation_title=Dynamical modelling of pattern formation during embryonic development.;citation_pages=326-330;citation_volume=13;citation_year=2003;citation_issue=4;citation_pmid=12888004;citation_doi=10.1016/S0959-437X(03)00079-0" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Cancer;citation_journal_abbrev=Nat Rev Cancer;citation_author=J. J. Tyson;citation_author=W. T. Baumann;citation_author=C. Chen;citation_author=A. Verdugo;citation_author=I. Tavassoly;citation_title=Dynamic modelling of oestrogen signalling and cell fate in breast cancer cells.;citation_pages=523-532;citation_volume=11;citation_year=2011;citation_issue=7;citation_pmid=21677677;citation_doi=10.1038/nrc3081" />
<meta name="citation_reference" content="citation_journal_title=Physical Review Letters;citation_journal_abbrev=Physical Review Letters;citation_author=S. Vakulenko;citation_author=J. Manu;citation_author=J. Reinitz;citation_author=O. Radulescu;citation_title=Size regulation in the segmentation of Drosophila: interacting interfaces between localized domains of gene expression ensure robust spatial patterning.;citation_pages=168102-168102;citation_volume=103;citation_year=2009;citation_issue=16;citation_pmid=19911861;citation_doi=10.1103/PhysRevLett.103.168102" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=B. Verd;citation_author=E. Clark;citation_author=K. R. Wotton;citation_author=H. Janssens;citation_author=E. Jiménez-Guri;citation_title=A damped oscillator imposes temporal order on posterior gap gene expression in drosophila.;citation_pages=e2003174;citation_volume=16;citation_year=2018;citation_doi=10.1371/journal.pbio.2003174" />
<meta name="citation_reference" content="citation_journal_title=Front. Immunol.;citation_author=B. R. Weston;citation_author=L. Li;citation_author=J. J. Tyson;citation_title=Mathematical analysis of cytokine-induced differentiation of granulocyte-monocyte progenitor cells.;citation_pages=2048;citation_volume=9;citation_year=2018;citation_doi=10.3389/fimmu.2018.02048" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=K. R. Wotton;citation_author=E. Jiménez-Guri;citation_author=A. Crombach;citation_author=H. Janssens;citation_author=A. Alcaine-Colet;citation_title=Quantitative system drift compensates for altered maternal inputs to the gap gene network of the scuttle fly megaselia abdita.;citation_volume=4;citation_year=2015;citation_doi=10.7554/eLife.04785" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=H. Wu;citation_author=R. Manu;citation_author=R. Jiao;citation_author=J. Ma;citation_title=Temporal and spatial dynamics of scaling-specific features of a gene regulatory network in drosophila.;citation_pages=10031;citation_volume=6;citation_year=2015;citation_doi=10.1038/ncomms10031" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Classification-Based Inference of Dynamical Models of Gene Regulatory Networks" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="Cell-fate decisions during development are controlled by densely interconnected gene regulatory networks (GRNs) consisting of many genes. Inferring and predictively modeling these GRNs is crucial for understanding development and other physiological processes. Gene circuits, coupled differential equations that represent gene product synthesis with a switch-like function, provide a biologically realistic framework for modeling the time evolution of gene expression. However, their use has been limited to smaller networks due to the computational expense of inferring model parameters from gene expression data using global non-linear optimization. Here we show that the switch-like nature of gene regulation can be exploited to break the gene circuit inference problem into two simpler optimization problems that are amenable to computationally efficient supervised learning techniques. We present FIGR (Fast Inference of Gene Regulation), a novel classification-based inference approach to determining gene circuit parameters. We demonstrate FIGR’s effectiveness on synthetic data generated from random gene circuits of up to 50 genes as well as experimental data from the gap gene system of Drosophila melanogaster , a benchmark for inferring dynamical GRN models. FIGR is faster than global non-linear optimization by a factor of 600 and its computational complexity scales much better with GRN size. On a practical level, FIGR can accurately infer the biologically realistic gap gene network in under a minute on desktop-class hardware instead of requiring hours of parallel computing. We anticipate that FIGR would enable the inference of much larger biologically realistic GRNs than was possible before." />
<meta name="og-title" property="og:title" content="Classification-Based Inference of Dynamical Models of Gene Regulatory Networks" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/4183" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cell-fate decisions during development are controlled by densely interconnected gene regulatory networks (GRNs) consisting of many genes. Inferring and predictively modeling these GRNs is crucial for understanding development and other physiological processes. Gene circuits, coupled differential equations that represent gene product synthesis with a switch-like function, provide a biologically realistic framework for modeling the time evolution of gene expression. However, their use has been limited to smaller networks due to the computational expense of inferring model parameters from gene expression data using global non-linear optimization. Here we show that the switch-like nature of gene regulation can be exploited to break the gene circuit inference problem into two simpler optimization problems that are amenable to computationally efficient supervised learning techniques. We present FIGR (Fast Inference of Gene Regulation), a novel classification-based inference approach to determining gene circuit parameters. We demonstrate FIGR’s effectiveness on synthetic data generated from random gene circuits of up to 50 genes as well as experimental data from the gap gene system of Drosophila melanogaster , a benchmark for inferring dynamical GRN models. FIGR is faster than global non-linear optimization by a factor of 600 and its computational complexity scales much better with GRN size. On a practical level, FIGR can accurately infer the biologically realistic gap gene network in under a minute on desktop-class hardware instead of requiring hours of parallel computing. We anticipate that FIGR would enable the inference of much larger biologically realistic GRNs than was possible before." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>Classification-Based Inference of Dynamical Models of Gene Regulatory Networks | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__-8wSh-loC8aZQjVpkiWXeUnH4axtmjVhZGuSdz8kVhA__4doWch-QYHBi-iQ7qgOafMkMn_zUNZJNYpe8oI0O6J8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fNIriR66yelh3T3Xn8FCYQqAHYCWPmaeWoekBRHOVfU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494995","apath":"\/ggg\/9\/12\/4183.atom","pisa":"ggg;9\/12\/4183","ac":{"\/ggg\/9\/12\/4183.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/4183.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4183"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4183"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/4183":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Lklrin1A4qYpuL70oEZBByanc1fgHt11mKahbnMz3himSCx0OqTOz\\\\\\\/Wakf9t0arCvBPmt1NyXjcQ61qT3sQAQflhlIFKpzIrNiu4Efh6u4R8S4g84jcv92wZikJ4sBMUkI5Zd6ODIUmk+p8t1fL8KXm5n0+ECt0YxhiYQjUzchvuW\\\\\\\/YPGzJkC02JMKEJmj9oh1uo0P78IK59zUpCTeAiPFnLonn1ofR+QANMnPKVrXqauEatU5R5OqBLs\\\\\\\/MQ6mMeIyai9WxTn9Vwp5GkYc40k9y3wnqzC++LBfrTolXrOjkisL6tFjX8aqpaTyUMdwVPE+hk8xz7DkGLvsHXV313LPyrPDlUb3ge6bsW+daczDnACVkafyj9Lw4ftLRwsSS1AotTnAiUaPZeYOjoobt1HXRXveNFBZ3UikGBZXH6chB0FusT4OgkQPBO7tH4P\\\\\\\/ZT6pESrZFT3uRyxonbX0Fp3hfX5fINJ9NK0D1oEbVfs7aAPE29NFPYm6M0\\\\\\\/ZMn3sbJ42\\\\\\\/+hlY9q3Jt9+piG0MuhhS\\\\\\\/RKoifOgsyZ7vXTwxEwwhB1oxvRDNgFDMZhRyOVl4yTJMaqhtPhPpJgS1gJtJ5bdVbZmFlZdpA1MvsdS1UE\\\\\\\/Xmi3I\\\\\\\/tXCYgFrc+ClMRcGGNA+C6HBFDezHA6ic4Q\\\\\\\/hl4sVU5cWOEVoQCT9xPKAcD1xYb4PwkDWpXcpAgIaePwW+gjXFEFL\\\\\\\/sl8MFDpEXX3n+nu6t8mqhyhR5+6TVDaB+CVuF+WF1g1vOjwiX42rnLJ1f\\\\\\\/yHhcxU5dAbuVxLkESOw7JsxFBlHuQq6rR5ElDHWJ2lYvA5r7B26fuMMQEmy+so0aKx29WsPi5m0I8DYi7cuhgPvJAVsrGOYBpwLVuqrcxueJ6XwdEDGYGbHneOU0wTtzomlmXZ6\\\\\\\/953rFRX0n3tiCiuMCF3V89qv0lmBJSxqrTtkODiA1sJc2gjUPOwTfeX6x1em3jqKwy\\\\\\\/UI+2ObQicc5WG04IX+AsrzaLDpThR2uqPk2FTdbBFfR1Mr2xRtk4HUtSkJtnz8eRQPWOz2Y+et0\\\\\\\/gtPxovz7dNcLAUrsqFx0egB62jcsurnGcUamVDo23fF6wfQLXI6XRoS3kvUKc1JFHGrPf5DQtiVmT+DDB5s2wxrek5wizkfPIuybb3qEAAZSI099lOTrHH\\\\\\\/JJDqqE4HyGR+cb3Aqi2Mm75kL6gEuUSvKulO8gvP7EJmVMOZzzLUQAeU\\\\\\\/hJ2DxbrHyIXgNJeV7yLFygccitdJCM2fGtdwzjgGgiRZ8ysv8Z0AY\\\u0022,\\\u0022iv\\\u0022:\\\u0022cgt9O4QhAHjdIIJoPkVt4w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f79e3ed710d0fb45a7f626dc3c13363205a389e85c5f845ff880a842e1587efb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uYQhYCkxIc2CAXRG9G9PCc9JPTiYZuPezNDl+khx6TnI94R04dypwloDkAXgPN6WMZqwyGo\\\\\\\/M9BLmp6Hhca8WF7f17EZHYQM9vjoWX3fiWIz8Q5rBqbpWdMTS\\\\\\\/hin87miRfm6YSEm+dcRF3alu\\\\\\\/\\\\\\\/uriDxRECx5Ixtb7tgX6\\\\\\\/vi2HMP0gH92\\\\\\\/DLNrTALhqsOa7igHUIHgmIFU0AmXnvzT8SYdLdpQa1XdxtKkYKO7Q+B59DFde7qkd3UX7T2e+U73Q\\\\\\\/a9xLlswmF8Rfl5t45GqFAubhzJ5VyqsesOZNcUYNuubGU1idkiOiqqHLL6M795Q+dGqdF0\\\\\\\/RbxcIknUv0hMpQhMEPh50pyG8QD1glmyKGnJlIgW9ZMOb+wxrH3ElBTIhvhU+m2cEPIGrvZvSXHQj9kH4d9oLtCTHWcxOVNqLIFIeDObvBSforyKKYwhxnAtbCIa3zoDnEAO5vdSpS6sVG6GTaV43G6pntTFsk8t9KgQzyT7VAhhqNN68ZjBehaQoE1n048X55livI9qfBEjboRPnaEk9pOspkyqwSwBLivEx51JRuRrzAdVJ9NTjTp7ku4DsSHXf1CiIcAUZn+OvrHzIy+Cxf2Y9xUx\\\\\\\/2BeLjm0G\\\\\\\/ooF0fbta0WZDNBLWanjANGD27m\\\\\\\/46bHunnU0UZUtwK8TJ4eyPQiAbKAJBHbDfUQvUxy+5s\\\\\\\/59\\\\\\\/a5oALL51EHvUesIpNeeQa20ksdTaRCiyayq6O3E9c2\\\\\\\/FYWQnmh\\\\\\\/T+tJIhzYwX501UWKg4wZ0T+KS2HJ2UU+xtUb9NJZDi+9r+PGtjS8BeDPft8pvLaneXLRq3ZVgnG12PbhoWr9+42Z1Az197FaGleWcDd+Pxx0qbNr0kJ4r1Xnp6l8btcnAgUYBxWftuV\\\\\\\/SKHN4agqSezo5MntH3fBvrLwhmswTDof0w2aSBWq6bwYe7Gln499jAjo6sxBO+b\\\\\\\/PtguMGG+NxpuD\\\\\\\/ot9P4NeIZeUTn2BxFXiaSN\\\\\\\/5zmCUgNj\\\\\\\/y540KWmAzWDXKygj\\\\\\\/Os9w5I6OaJuR6yyaVpNbRRJJWjXETGuiFJTbN76T\\\\\\\/1d3wev1f169PoZC1UPXroPA0c828JFdm1yVFsUBb77fOYuMfvpZ2rFkiuGP7e1tihaJGyMrWKFPPaAHGM6moSGmXzfAbQdfpDI1E5PWtQlne7dm8VBkH\\\\\\\/cESE4cGu1vfZtZ8kfzqk1Lc4fj8Ln44f+M5iYYvFPfakiEA4yCS+xo0WhTlW8bQYFy6k2z7xt8sumQOGzoh5AGMtGfyC9Bz+LVftR8ZX4FtIrT\\\\\\\/b3fAlYNdtsz6oyidRrui4Rhkg6s3vEE964jOGGssOpIaxusdTjmP4AA8CofUbcdpiFzeJpQt+y3LpoTycQ7t5s9wh91gTdpr78DEyW2LkugXazzgcHpFte72EBiWxkqciA0++nS7\\\\\\\/RSrhc5fVIIgxxHw7Tzayna5aKk596uFNF+6pY01r1+5f6gZWmFIc0TX2LuQa3njtZiMPW6wF73rRhYClPaK15eweY7s\\\\\\\/u\\\\\\\/\\\\\\\/F0yqC4WR\\\\\\\/7lpopWdpH++a6TZYPq1iwveMtwVqX9LQPsAwDn\\\\\\\/KuKOG7G8\\\\\\\/2heuDIzmilgNghdnx\\\\\\\/oQe38BapzNugTtZTjOE2rf25A5N5OlTK7bz8Rf9fN7edfzPGi7ATxIMlERt50nY0xp+wgJwIKFnohupPIZUs+7kZDJv\\\\\\\/FHzdkmOS4ZoLSBN2UhKtWwHacqLwQtc0PcnsX5zCFpAzGOeAl5jJJtbG\\\\\\\/91zGA+UGGCUw2XLPIVoehsJY8PGPI8ihbe0mhqhQ+OPeAUZgX1+As2QsmT6ZiAcpjba+xP4IeQfiToRN+BKH6+6ERiyKoDw+Lhx2+OehwwFJbmBafRqlHon8u0FHdrOPhtK4Z5vKWkHS0+f6a1RWZ8N7YmufrtqINfKLB7V3xNN0KprJsHg7ql3XK6Rbd7+9YFajq6Qu10NIjWdMVMMiot2CVj4jcCrMM3btu6bPTcLescRMFWEOIlDdOspeX6SJgeiZv8+L6ChOu687+9ATjpMSOEBW0TgUelqCSz5yKd9XyNxU6l3tqt35KlMqbfK+LCYIVhzO768S7Jrz131BTYAmdkr0kjUc3bgWSwhgJNhcrJ0G6pyGPdoPZdVHGrYqS\\\\\\\/S0MgXKMMCKCUCKp9T4MiR6niEPtiYmuxVPJ\\\\\\\/dYV9vmIgHClfmnycjjQb6Rxzd9GqSyAnTme\\\\\\\/kO\\\\\\\/W54DwyqVQ9\\\\\\\/LilbWtQxptpD5\\\\\\\/WZdysd\\\\\\\/HDbSBwa39g+oP77ZiVFB\\\\\\\/30cnfkxC1G8r6MGC\\\\\\\/RNYPwWKKKRZyGRdI0q8ugJa+16Mul1WuCQKw3n2GJqRCX\\\\\\\/t61r6IeqwcpSQpm4722KetpSXzT9cVbXT2O5fN6qagcqR5JTaleh1rJH68WBrDbm+GD2TwXsKgE7gobH8IONRgdyAxQGLpvz6lYe8R1Fb4xD5rhsj6fFcekwWEPcLASDHnh+fUTe2YFJrZNBkkry9sMX0\\\\\\\/sSiMFUtiUvUpC3\\\\\\\/h5079hmZ+6vn3Afx931Ka9BUV+blC\\\\\\\/Xy2xtM4ZFyKFnMlmScUkLjcQHwjaSHHtY40HxPj45WGxGvCxv3N7snkczGBDtv2H6N5NJ7BCsMhFWE4N8Vlm\\\\\\\/ZB\\\\\\\/cXB2gko6KIeDIp\\\\\\\/D2+XfVvo01xWlYQ4VoUgLxJIXltnDorWsAbD3MEQnAVcX4634QD6nN08h\\\\\\\/CN400+J3VBRRT4KB\\\\\\\/bQBKhdk\\\\\\\/K8DfXxkRKQV1Yioq5mWgPdeFD2qgzTRvOFsFL8H7BOr1LsSkGlQK3Rt97GlHNZzycNdf7g9FEG8THLhlj79QtXnww28WUMJ9uAnvMlY0UHFzvHJg72YUrw1N6a\\\\\\\/ixdo44vV5rOoB9tJlCxkNp4mjuDSm923nrkLrn2run2ArOP82dL3OcbAslCOOgClxjQScRoNPRQcP7Z6I19HB1QzCo0PgvVt6txgD\\\\\\\/AlKzzcMxB8ud7Hks5m6v93bUYxt4RWdGI30JpCJW6stM4\\\\\\\/KVAMk\\\\\\\/SDXsrx5Q2aeccK0UfVVJIbIFZADNcz+d9hSaA14GMx2vkC\\\\\\\/YkNzmGGDRAiWrYcmmV50vOOguSfMaCeTrPd87a5KO6ywJlelktoQy3e48qbUFMW28\\\\\\\/d+DteXwmLqskJolhV8f5kJrAIVx2rIh0VBgouDuJbGaTG\\\\\\\/uJCeQmeCfFADlzjVqGoi83xpQf\\\\\\\/o\\\\\\\/q9VhHqQv8yGCnQ3wJxfw1Yurwt6JbEESdnwYmIkyR58QDOeuHn3Cy59BDEY3GczJliNT7QErcuhFgpjO2nnYzqnHMMRNOuK2HWlStwe5cWTf2thVCjYS7JLDcs1+LCxpFSEqN3PLanlqsl5wcimgK1qar0Ep2YJqrwmS\\\\\\\/HW\\\\\\\/ppWJgbq1GSRztEMkFNyXKDvr\\\\\\\/+jdqstOzz7QwiH\\\\\\\/IyUeJrIl5Dr87qRUc0gM\\\\\\\/JX7QBqFKvXtGPFQaT59sqHE3Frg8H0vfQAD\\\\\\\/yBmIY2Bivq2U5Ykk4jCQPMuBZOW\\\\\\\/ogRORBtupRvbQR6T3FM1j+kDE461DJqbHj8fb3xFitLUb8J5P+bJ18RZkQsr02unLQ38tCej59y4+4MKAb+P43bq7c\\\\\\\/rKm4s4IJh\\\\\\\/MaoF+e0ZWMmyvwlgWjFBZ2DBQzCIDZnU0h\\\\\\\/Q4ok0sjTs6rGKcBmTOnDpmejhui8bzZQIMg1nme6wE4ofoMAvYEDCDelh6Q9NgsZVWCy2ChXsqtuclocKV8f5Ej9QShSPx3t29LqphZYtLvMkJmU4YDmigW7nOxt\\\\\\\/N8AN9\\\\\\\/zgEjVhJg5FKWXtR7fLF1oiCi65y4wchzz241FVGeJeuJ2h5auYIdcwJhg0xpABPk2SXNgo9TfYE+cvQubW2gfHWouz0rAXZwEzkzWIwArgR7xduC5BPSLBr+h1tXMPy7\\\\\\\/YB4vn7NdpyKGLV\\\\\\\/m8CUZxP7oGk6o35oe8gAdAcsiWl10OOuKC3AaAJOKYf3qxGnQDX0rPsecYoQoJ3HO7ZPJyHgYZ22zbtC626DfZORZlGRmvSlqXUh\\\\\\\/h9Fsu5DOruzOAZtNRmcsd9Mav5E\\\\\\\/phiHzZvGVMVKzJ5fqk3H4OIUjLT3w\\\\\\\/oNejyKsVNrjQzRdzsbecPx\\\\\\\/fRwrHD9v9qgta23K42EyR2Ir4G+p\\\\\\\/9sX5XnOToBha6tBsNbMRvLlmzZigkJbhjb7h9\\\\\\\/1vm5VbbbOQXLpXj7DnwcOemuQf89CtL3WzOfgQ+FJofVW4CkxWWzVRMaCN1M2IB0sSppr2JUH\\\\\\\/rFGz\\\\\\\/wPml0iYvBLZjP5I7iTnQmYDuNnx84G8qcifVMhyMfM\\\\\\\/XYtJwfT11tAyTjvkt4uwo+nyqfXF7gbRqHtS\\\\\\\/WNGT4T8RdLH6yv\\\\\\\/B6etvK\\\\\\\/bjRWM46eYo0NPNUXn8EV\\\\\\\/8zrbIHdiIhs70sr+Cmf4R+goy7YiWD\\\\\\\/10mw9qs6ztTETV36W1\\\\\\\/FPMMNLYLr0i44EeU8eYJIKkTAtDzoM5Q1\\\\\\\/3JOOkHEHKYbW3cIGCvVoLyI\\\\\\\/i1UE20tw3gS7MGphUogkv8FHMhxskCoxSkE4+jpq\\\\\\\/hUMY83oE6btyRhKT9cdLOy5CIQVvIN+hJeWnVbWEO4CNHoO3Vhy1idhkf8GCdmZv4+xxkWgYAaCjVswc2SS0XudiYWfhMbyAh1fGqymHK+4laI\\\\\\\/di7FKYHWo5HjKVMeH0q\\\\\\\/16iNQ3r4Rlg0CPP9cpujuC3W990o4SKnLW047lTIqxSn21N0er+Hu+eCtKjFIZtq4aTiz8mtMUzbkMrQrQnv\\\\\\\/Dcxk2SnoAXOXzbAbtoYCLNRD6Ez0B8dxGKXxmwGrHKlmTdEtTWkXfBHP4sDmEPwPvgVodrj4yPeH1S9zfGSQcMm1Y8wXXO45mODMwWE9WS6kcpYgK+biRFb2TWuCz6wFyZSAIAFc1IGlrMcRb5ZEMsCbCE4PUJKF5YhGXFx9SNjsdi7cRSdj3mAfYFybJMEZexG91\\\\\\\/cMUHdd85jz45qMWVd7O4XAy+ntQl15hky9638dnGyvrPrExMgCVSWoGqbKzHUi\\\\\\\/hr52mM28ICpRiFi8Cjojz9be\\\\\\\/M3LSIJhVIufsQfh8WTp+jnI1tCFhj2xsfBFw63DTSRMwbzTiFyGKStNuCbP84Y5ehdCJAuzyBKxUmotUpekS88\\\\\\\/nzJkZqAKdhC9nYtHre+We6CfXlgBgrEoE7UdlN1SN8vO2vfSVj5EAGbB1jLHNPlnk4MFSXZvOGngApt4+\\\\\\\/jgHIwWI09iWazn5kFSzXHuOCY1myK2jJ2c+EIdrxavUAUs9+KzMISDAfs6vswWd1jX6+0a8ll8hMa15MF9gf6urzxxo4KyyyHHcZCffhL284ErhHZIy2FMotStxidU6mmW+Dx2dkX8gLa4YL1oIhxat+EOVohWmZJRNzf4Akk2a5dO5unh\\\\\\\/3asD4Rg7ZolEGijoQ9nf7BuUTm1AsqCjW8o7IrxytDcc7MN9eFfutdGcJFbfkVYBl1I0HaJADaZ9g1ZY2Z9txOX2JLK5+EkMLvoelxURrMb5VKd+PlakxRN1db6HM4mNIZfPMBnydh0eI97Ke5DD909qTsY6hzrCI+NZBEAMgwmrXXWJ7M9O5VjD1KMVyBacuO5mb5r22JvJ9mx6xVeTXZKyK49QFAIidV0kMI9F+\\\\\\\/Plw9Qyg+lAqx7pr71z2xVF3oGTZwRQb8P4mG3EP3sEK7LGob3hHoQvfwekgFoLV0XesHZD2YkmBo468JXNUts2hifn9j7OwMD35YjwU74jOzvGiX7GaT5AkHccUb+uscnWFgZHoTx49RjZE0t7wbnBY1jRyHo4\\\\\\\/ZmyCWLFBODYE4DlBNOzzx34X22Afo\\\\\\\/YZb6PxWlK\\\\\\\/RO2OZSB9XYTEOM9iXtK6pDt3ZKYhILh70hi7\\\\\\\/ds4cn34hAEzGUEgFNMTwB\\\\\\\/ZlnSVxiXGMbakCOPBkR7tJxIWQrioGFR2OQWmiQ8H124HrTuKwZ7bICRIj4b9mBfg9Y+U6\\\\\\\/1wuvToQVIu3MLgWAWan4E\\\\\\\/GtK7ysUbpvQrphzOCMyzcw5agTirBCveMYRM+13dIABIpGD4DPNw3SP+QhhIgxRpg+uz0q+ne731xDnG1ppZmWrj+9FuNGQqn6qFrOjOTvNtw2LMLCfpPBESL1gfswuG0wBY9R6xBs7CVzKIBLNUuQQgZUq097TuL3HMPXArUbyFac+OoFXuc8ZPZuNDRykik0oYGLyEHSSxhqs340OII68T2UG31zh1H3ny8ggH\\\\\\\/SGlN7cR70\\\\\\\/n3E5GmUGsUu00y9h4zfVEev87UWwSZRBZ48BWHWljs6V70y05J3cKt\\\\\\\/Ilwa\\\\\\\/uO+bysIu3Z+lik4Y0pSRZpwe05UGkDKdMiRkVz\\\\\\\/o95bHMIwmuvoYbNQh4AnqjbR80L3SvPG3QSsn157sIlnsIwF5DkA8OAMgqWJoBr7uyhOVHBJo\\\\\\\/naaVM\\\\\\\/CugOr5GvaKeMVFu6swEViz3Ba9EDIZOadb6vMoWp1y3Rgx2Rlf2khfRLnMZYPtHQRXM\\\\\\\/DtZsuEpoSAXcEgulYyvbbjSbB4f6oOv89wOA5161\\\\\\\/rfDPwic36zww0z9b7BbJFkXeX7moB7Ak\\\\\\\/eoCjQAE8b4xQ6Paw+vN3z0oebHU3npaDPg2eoQvP11iCnHjIJfnGFR16jbMx8eXR4PNAV0QLkkgry\\\\\\\/3TtQ49mq5jHu6BSlUHASMOLSHfBulGxRelCWQzyZrg4ufyaHd0sRAreVziNdAlTgYWZ8mtnGt9p9reCIzU5bjA7weCJmWyrq3AEZ30yzlGtspil\\\\\\\/KC\\\\\\\/I8xf6tL9yTtEM9NuV77z+tzC8LDeQru5GMrJkAUNxGCtVZBy5kK5VgIyeYAh87XYW4GwEfg5Dw83C5hwmPX5Fg3gsq9pRPdaFz1\\\\\\\/5\\\\\\\/6yaCf1qmB2RRUxF2Dib7oYdKtUAkeRgFQOQCwM4UpF29teYANcKGjd3TH92d4tpQU9LgzSuya7rCs20DMhom7\\\\\\\/p1dE0JP3tgXBfCBfUee1N5gYdkLKjo86AHEOxRXAitV1LnRv1tUkZykPJRPcM\\\\\\\/NPed5knhInzlZeOBCv9bOkrJzUi+1EphmJPJDKoXCKGcwGLBGXaEWCDGZFWliK7vPT\\\\\\\/xTfFdwdcFDSHOj363EiOpmwUp+5A0EafjPaU8UE9TgRR3u1PNHykGbZQTe22r\\\\\\\/7P4zwTwo7vpe38mdP14Aa6FpYTLIJ2n+rFDjB23MBfUBhOvAQKl+tAK5nsxwqpnFitw2kWyL\\\\\\\/+NdS9XppWi6n3\\\\\\\/I8Lkec4GPAXg4HDxukBiruSFWyTwBy8y8zuYuc3f+yUH+PE6sVtwuJ7QQZnOwpvVU\\\\\\\/+WNYIpfvZhviODoFt\\\\\\\/nr+JvCCypb8hP4JmLU8hpj4G7F+mDI1EM72\\\\\\\/JoYABk9NR2tHC3otMTRRdVUxv9pfKvmevAFlljKEe8Ql4aH6APaQ7k42RiGX8slIqHDhO1Zo7bNHGTQr6kj9fweFELJd+JIIgn+HjpkTbXxvqMwL70OiOxlhHgCPn4rWxRh5KLvM4ccFxKQAnAKBt8YScr8KFxbD8aoCrwNQpm+zQNfgb+21G6gyYekZzFI78e7rUiX\\\\\\\/qKFL2Ue2wUMST020mZ6NLrhtT5YM35dmNAWtqqGUnxCX9r\\\\\\\/J3F\\\\\\\/0Pgs9Zi83R2KinVur4zKO9fIAO3zLydfTGrPUX4HR+iDgTT2D3R8oroyUVhX6nVMKCGg99f910tBfgvIk9yS6pCfU+fDc02Mjf5XZV51sKY2JCW+nlR724x4\\\\\\\/ejc5E0fZ6ilXGdvxU0CtsYHmGKXpM7YLGcmAjrg5GlOQWe3qEHUFEVa56QTp2X5VR6Za4wLg4wEvXH\\\\\\\/hyIuSYVEEdAJmkTG5KmJmEGHs6WLRZyYVNeSefNLY14NbYddQH8OEVwU8yhR4gLUPg2c+BF9ATJ6MwfvlGmJGMmV+5geFoKGzzL8ceJ+H8D94VGJYvpVWUOrSEiSMr9wHfCVPoAPFY+068igZlbRNipFJ4I\\\\\\\/VXi2nP1KxzH4anv3kYMNd9pIJpoT2beqOb1oEDaqffcQpxf9UsrDk9maIi1HVEIQNaczaDTM2z+3HBLIC6SETPeWlDdibQvPMtKxQ8bXm6nK7eCbjKOIma6laVr866Lc8w7tZBsUYXoSS3zpYUW7aGx0DAxxRT2rUvHH2GYutkkRNRdULfWxZtWfp7bGio7QD4U6Z9\\\\\\\/EPsnzsghsX2mU6ucTbMaWh2aoq3rmJN4uUyGkKP2Q5vj7Qqh4MspnD947GiNJhGfmLHSEFUbwCvJIeey6aoBFCgksRJV7ziCQeb+jtFg269l7sAYX\\\\\\\/KsAsBCf43flMMLWi7+O0Ydz5J69Kh9gX2xRJvu26OUaAjlQL69vHfuosVezBAY++U5VLby3N0Kw1Ejc\\\\\\\/sBW04j8OyDjqhtqIZA5P1dbBlKl4U+2j9PQ\\\\\\\/9CfK94GZmfB9QENotE3c72W1waMCC\\\\\\\/9\\\\\\\/t913rHdGSWICxDMn8ghI+fvTCrcFn1QpyNKqzrPAqa\\\\\\\/VtwvzWwTLWl0axwNml2S0iN9uIoADB0rL2GS+OpRh1y2s1qQLWSqndpIL5P4Fx0wWBq6yLSN3oTuTB5St6JjSc+E2SKZA3aXVkI6Al+DOSYJBOqBEUh8VGjErXBkMtLXrTL0jlK2x1vaoBJSt\\\\\\\/zXG7mJUfotuAe9wj5HMJGKq7XudHOIU0K7hTd0nc8rQvox0FzUjd5s7YnFKuvGbVCwJR\\\\\\\/5dldwlh3CMNYeP3iLzKwq9KiNDqX8Zw0PQCNxtloBBZ3hcOET6p1jJPNMvCu1xnTxBRvzx1WRRug8xVvQnUZo3t1rXF06Ghe+UJxakIEx0i54Li0Bwz+aKznuO9sWoDC+ArwUdT2P6ro0sOl+b1dxsCuVGYZvUjmxkptw5I464HF8QltusZTKpRPd4YCDPuezg6iE54jqvL2p7qEL5Pnjrd98ouolO1rDRJiFPlgofzzdrnVHhmeOEBOzbPBw2OakOU0F2U85qTAf7islmuCjZlRiqzTSfAJKdaOUV5y03zsR9+Sh+22LxNNC\\\\\\\/RKjXCY0Xr17GNEBPI9orwANsYuYf54MrEEY\\\\\\\/HPxKw9Sek4xvDFme97JClBEi5A5ijLVjF9qPuddUz2K5YrmfG5BVVIXVxMxWCQDJROmJjT4Ziu8kkje1t9txNBtPvXG2K\\\\\\\/PrAevlW7S+dIErTWvDJYFJtPOyk81IZohxbFT+5bjL4BI67nEvXkbq9knV0RZU8MbAkJS1Ah5Ru3QVPhTcmSiyPst0TujNwcWQyk+Hep85MKLrZyx0VJ2jI\\\\\\\/Lvx+m\\\\\\\/zZ6b+VM1HG4PjTzvRRHSZLodLxDppHtD4KRF6tfsR1Dz6opOo64urqBx1pHff1WVnw0PORO46kxOVAAxgNkiKWMckv0fafSpSz+GbGsNVknaYkbJvXHNoXdlaOVaeLOHrm0FrL4fikVUPZCoBjh+xJhLIZ0VTIkNrShO9xOpIkNoaTTMto0qb071TCxEGDrT5zyhaV\\\\\\\/etTIySb\\\\\\\/Spxaun8yBfUhUaGE9c67pRv2xJAanlYyO\\\\\\\/OHkqJntNmwhgdKU3poSHXO16oocGZAOLiFvnNn0xXfOYZSmzy8XXbHwv3ThMqD4uUZp5ika\\\\\\\/9fjMYI9PHNmCO+fctO6zZreKE5cnA1hT9+s8yXjrtYFODSRo5DmQ8qeUKXH6eDM8gFr6U7gr2baJqylXn3x9AK8Rhxgwg3SV2NKghnCcIigjs+Aqz9NPyut+6r4s99zH4\\\\\\\/aJRO7HQmk5KhuAzcRGtxo5TujKqGDWPF4eJ5Z8eOP++FKCJ8Siaao+ZbeaVoHxOJM2p\\\\\\\/PSjdh7Zf8ObF6O+Ws1Ycj55S+7HJ6gGvfG4dTz2S77kE7J0HZ4uWklOY9\\\\\\\/6YvRa0lbZjvg5CP0Fh5s\\\\\\\/wmias\\\\\\\/KZTjzFMeHDokpHE6\\\\\\\/1PdNS85cFXYCr3y5llawBjo52bFFd2xqXJq24vG6UpVoDDInTxuNgPHNeBS3aainvXVRET2a88J9q5S3y8hGvyyH+pu7nsbWoBX8\\\\\\\/c1h6EoD3HQEmKH6PLCgThDn3JPdcGMeDqhm2DCGpIHrHVODI17hADxRR7jJo47uxb8lKBNjsrXKXAQRQLjJAqDEZkfyHFQdWV3jPk9RbLbA9taUtuYtrPHmx6z1aRT3OsjfyqcBDKhyeKvz1nKXTcCztmg21yCh6h2ZCv6yLetQMtH002GnBzP1OtKlUGhPng9AKa3400BQmMXck7utbB+rKMLkVbh+\\\\\\\/+2R2mhTgX83K9vfHsR6CV2IKof4TsxdjeGTiTzCZ7VYJkFcnhfayVOch98Z8H+VecgpH\\\\\\\/k+6\\\\\\\/LIG\\\\\\\/2gj8i60VvNUnEzkIF1mTFa\\\\\\\/EdFnqLFpclkXNSAsRGXx\\\\\\\/oahMIJLJRn2OKijTh3oXRqoOs23D3+bIrl5XbhScSYb9KWqDVmQWcDwGVoxQyJyO0oSYkwNRmBG04lAERQnp0OWF\\\\\\\/GZMcKzq30TwsKhDcAzZ9hLnHHUyDh3wY\\\\\\\/wTG7mD\\\\\\\/jy+jieLEsKzh0RvMozZCqfhw3zOPiDA+4PbkKy\\\\\\\/\\\\\\\/uqk120eZHgb45dhpaxxy\\\\\\\/IdBCbL6OSDkD+BLmKzYWops1LaMJbqXbFwEUhWKacLXqRcBY0IkF1G4lDZJPBEj4Pvu49RJZW0VcCS0aUHswzmJKsD3YHgtMX6oAtcB9gmDruikBXjtDjpEBRjSdc88tQ045Y4v+RIW\\\\\\\/lJ2pY+bsci5zoTJLOU99j1j9OlaKWIbYSKYvskf9IBPvmw8rk+XRmDd\\\\\\\/jUlUMvmfTFE5IU\\\\\\\/8LFLGZcCiFZoMLU3O1P3BYtk6MO4fuA7bsKmpxOUngD4DAMdYblCR8xC3Dc5XOJwLc8h47RztJ5ltfRyz+uIsobmbFmgMgO3DBdHiYuOgVMY75wH35ylLN7C1PVKS1ueOcCr6zG+UcxCKCVxPqIv8yXRPhc+fSQIx1vBllUT0QirTkTdoDX1Ec8E0eGv3xtPjfKVOmd9\\\\\\\/zH5iEjHyXump7FaRIsCJ5nlm2Hmf44idK1OzLinORc2G1bISfmsejl+5uzA1QNLKCwsMQqJLNLI6xSJqAySBlXdq319eFoepWj2TSDcu7L7L29cYI8R66Dcuj4Q2Hlaht9ylscczAWhXoSS5RbD\\\\\\\/3eqgLz\\\\\\\/LLSKVuhO9Zuom0QU8Shwyq0Gb6NhHeW6r\\\\\\\/YiUtOmNUKq4NHm1gRLMRZ\\\\\\\/6lTtTfXn04cpVU0o72YZe2xd4lQm7LwX5zv7p6GwqlqmtEnk1vkucCPEvU798sakNptIweJ6OfZIEbhQVf2Ys0oJfsuiGumx7a7DEvGZQoOKz2yKnYfzGs0PV\\\\\\\/zOzRMwUVKLIH1lBVPmIbxODwHnKEj79DxvzrUw2dvkLawcuZ\\\\\\\/ilifW8Wj9P0w6CzcvfNKFld45Fq\\\\\\\/tT+gNSZyRQyI1PTobfLaUPuKb\\\\\\\/uInRnYJP3bYkl3I3ffNoV2LIuampgqzOH2agJm7ibixtIMUb4x0q+VjmFzHvfezV31Yy45K8SmL9hQHu3\\\\\\\/bnlR6Ujpkh9+yNsquZcertf7r\\\\\\\/NiZyPDjfGZkW32I62aRM83v8uCL3PnAMY8dx22tetl4Q\\\\\\\/hPHZycVOXWA3jHmYxx\\\\\\\/mk+w+xsbsVkEvc\\\\\\\/5FiAPZHcXVlXs9XWcY7TPCr+kxDt4Q3ATfxejhShxsL2QRAQXhMyVTVrVFO8u3Da9HBpF1l3EHiNGn9KJi8AZkVUis5oNcI5OCvz4ap9P717ZHKrwvv5E3GdcNjQnTmCLf1ukJSOTFEsYhRx5yIcqZoqVoekPlslCJdbyzVQszv9YreTBFZiHnuu5ZU4saBg\\\\\\\/LbX9mq+Wc\\\\\\\/HPf4SYCx0GEtk60Qu9B+xheabgozWmyZOpvb1CMNQgnumvRI\\\\\\\/ZdANV6um8BET0XMTV5HM2pUUSnmEoxEjmLgmgpPW\\\\\\\/Qa6nNQfu0UwQId2KyeFGF4bVizLszysc+OehisoC4igeXSUOYfYsABpgDwHCUQBMqa\\\\\\\/66uuaMi6SWy2JRh5oJ3dl1R2E9aJsbRdiVrH\\\\\\\/G70aydiTfj6a+S1gplBiZNL8OkCpnqowLetFmXPwVayZtsX8Ujuo4r5kZqLKWu1ZrZIiBwUhZqcK0UdNmYQy4aq7c7ttsUgi1gydX2FOdNyqcVPHhVcR4QbhigREMCNumKBfjERQpE+RKRDJMbvarVlJlZntBo8Pdc8KzWtMgSAoelwBBoDcC3N7wZ8mKQw1UOfAsXJ8g1+vBbktxBoLA4Zp65eQ2JJWocVrCzwwnxYWyVV65cxf5+DjYpaLVefmJ\\\\\\\/gLRY2sbysZDDNYxMc6MU\\\\\\\/5Ic8mBfBUJlHiBRMXvTN7KAqMDm\\\\\\\/RRzpSjImRNrNyF45jf+tDvI3uyCL\\\\\\\/KEziFT1M6kchpUKC+7aqDISP3s+c1rB11hDQcK5ktyr4gOAQ6R9GFL2WHXfQr+MD5Yw9jDo\\\\\\\/0tVqLK4nLAy02GUV3409xxOSYDuaTbGnSyFrPJevs6aUxRvjTZzIMsp6s0CdcCn1ae8zKrkq26bPjNhp2XnPPnBTf1+Ute0Y0ltW2qOqmQcHw5FuE7wuXU\\\\\\\/Vip3WUOAsDFWDA0Nd4xb3J1\\\\\\\/o5PZfMM0LbyFcfj9K2xC1BzIK9LvlIWTM7S5kf\\\\\\\/71WI\\\\\\\/CynoeZuLvaLpmznvgTjaZEPPQ1RVEefFFoIKRWMkutgA2VC\\\\\\\/7MhC7Iuz4YR+8tu154wn02JBmpgAEp8L+aAHRaaLldzZBIjGHufpkJwy1dWF9yaCc0Qrbvy8mWZgQyRmi3Fqdfhmb9mBvWrJIdwjh7K0563sRKp3srQvRo19GeuFB5d1jeGThymBkpTOgIfQR6lIZC7Mg95VGtSyHR1fabvzsXstWu1AXqQkv0S6MxgsiyuHmbl3fwuCC4j2Ofvb1lrIpFbxnSkC9UtB743l6dzb9oNBMOEp51Y3\\\\\\\/ZGzHS36vKxEZaNGR+xkUu8DmlrBTpQKfLR1\\\\\\\/O7iBZMSPlXOQIOAghB4VUhAQGOHEUH9abwzaxwv+FSh2WJJm4SgJf32UPJ6Cg8P+ZXAr7KByFZ+eWUuBJXICgJB5lfvHe7PkLA3kkYIUJTfuacXCnVYG\\\\\\\/uETe3bU24UwP7gGM7qHx5d0sdvL1xaV6Y6juC3bD+HTn46zVi5844n\\\\\\\/fZB3NiLfLJKQ4LT6U3+1cWYY+VpX82rP1XERJK\\\\\\\/ngV6p8RBklJf3pA8Q57aFNZZEyj+INnFnHBmwzgSSBZHVA9IRBb7FC2QcZN4KiRUy5gb1\\\\\\\/ylO\\\\\\\/3zAfjC20FdTC6He3H8AfRdYCYLA+5piZKCG4m3hq18uhUuu3cgpdqYWEDpMZCpeeLNYmA1dAsG8xEFgattplakSWH3DUPDJTwr1m5DEKrJiavS4hc14nSQ7\\\\\\\/kkFy6fggvKa+PZDMWmlmGfRO6BUClxlRQDF2l7MuUMlRm4lfqArkxwEfTwPa07cVwKGSA6lc5GG1omssjMA8tKKtQn5YbNATr\\\\\\\/TDLb355xG0tH8Ugyvtzf8RBYC8fDNVKsSzLwzNkG7gSI6AmWnfUEyX04ZOhIG+At2ums0AcFiL\\\\\\\/\\\\\\\/gI80RmA43yRrxNyvlNyjmdU8CuG8hGYlOmjCpOPpwoQ0Um\\\\\\\/EfkrhKTXqS83\\\\\\\/KMugrFMlWaByZJk8zT2Z+6Uaq6heWoDvKsmAjVN2mleZs5x4+BNKfBG4LWyni+q615NSU5g1vW+\\\\\\\/4FtHoQhoJ0iHNQv3WSeY9OgHgLyPD2Iw7wuv1EF6htr9B8DKrlbPym9vGHrUezJMsyCCyzEIVujoq9YlGGsJfu81Wet+V\\\\\\\/W9O8bbgQxcfZfRCmKhyb2ItI31cuS8gHoOu8gV9mwHnFZ7zTclk4H7rn\\\\\\\/rELJo5oRb8jRTSGjwL5rYQQ5FjslFbmrqrBFewUsayI42H2gE6wTcIyaQxMcUC8PPxxKW5LrZqj4mZjdJmLOtn9EUotFWuR7WYZupXX6R\\\\\\\/em1PNuCf8QsaJsYVz58AmPvN60TzSlov49sMbjgUCgP7HaYHHMvWue9eqQngcfDWizxqcv8foupH+RjIg8elWbFJHM9OEPgDyb0ae4NdNIyM77KpW85KAYuSx02VUqLgkg9UlwT8OOz8EEIvhHpTBXoKM2Yk3kEDd9GGK0I1vgvwNYnYhFnrwEiumjWdRk7MP6+ZJ\\\\\\\/D16WBzYXvFFRraPJBazun5yznyTMj1RDQjmb4YilI116p+H+\\\\\\\/hI04f2YcGPP7VwDILStR8rCr2U0tRm\\\\\\\/R4zGLIhZjihD8weo+rc1Cr83p08WWkDoDZM8PMB7XmxWZJUbEWbciuaoKLnzfoXFH+Ixm7uuTItums8cdULZ+NWWgaszlrpsqr76oGMXL9pqhqfx7dXcNDg4QrgJ0\\\\\\\/JMvEkwsgv6vGIvDw3Cr\\\\\\\/s+CXhCdDmQzf4EQxbFeG40yyM3hpgrKy6I5r8ORrH+QQy\\\\\\\/\\\\\\\/\\\\\\\/4k26NTMbTPt6cdmCFoXgmb23kNfCa7frBCaIvcECmBMwpkt+zLbZQ5H5C9DEETFvG1VHbgjwjitk7QVlMWXCgYF5mBc81LMHQ9b5D+yvlKCy4iCY+7Zw7Gd72WhgrQcpT6CNs0ImLm+xWo1bphOC43VrCKuW31evyQ9rhTUQCvBvsRc3BsZfjHUkGLm1Hy6UeCenDjjjCqcarzujqQZS99QPZXZa27ggHGrXYive8eE39r0h5Y8xP\\\\\\\/PAc6Ab9QkOrfWJ5XMUGInnNzJjT9Xi4IMVLu5eAXCtr3psK6gSFmUomJx2EgSwAQ4dIrKQ1m1z8SAaC70jj9yuKpZDNCC7Km4d4hOAJkQpP49DlrJa9ca1uIrnecniO0Eez1CvdsT932gf1yUR9ToveQU5l36epfaCN5XuH4DVyQRKrx9RtdhL\\\\\\\/yDF2RLB0q1eh0E5FbbQrUuClISg\\\\\\\/CEiDPv5YEJKj2QUWAC90oL0uEjjpMN3u5DpgamAURl8tRs\\\\\\\/rUyJ8PjtEeNkY2EgGuP7DGkpkk8QweoUym\\\\\\\/jp\\\\\\\/wD07ULOcqoWtaGJuQPGX8IT+mvoGEx+JAumJ9pWXlrZYJyDV4kxWzAVungsnEl1VUWwJAxoNep3VjkzGBh0d\\\\\\\/U1P3BdHMac+CtLOJ0g\\\\\\\/2lO34\\\\\\\/PmXXQYJQG+fSnmpE+r14GYE+r78rXXyZHGXN63LgzM41tSzJatZKtb9I6Z4I3DdKQ0aq02Amj2hRHkX2XGyoyqXOLSYFPb7vE6y8UKwBIonMLjT+84u6oF1L1jYlkjkAUFnYbKVHHh1IGyV0vKCK2AB7TvUD4XswkSm5dbJgJBKU60Dd565FoqudFltadjvzybdP2myFW2frJLAzqwPv5Q4CBRKLEEZJSaPiPsZyFILRImsLkioNQ9jRXdiRJKq02+ne2UtsQR9gdMqtLz7dOkeeeQSyZLKG0\\\\\\\/osv+x88WIASrg7I42McC7YD+FjN\\\\\\\/L9ng47ITnsV08twdPV7rDAHy5foTikAnA0ikQV5quDyxMmqM+sJyiBWsyKB1W3jbFvpGm3RSlQ7sadJ6Xn1VAL7UGS3PtJYmvZcfj6KKRAHwTlrcbuSQrAfBfEa2Lfev1BTow+j3egAM4gp3ekZgAFLgVTKvRKt4beQyaZtzgLprJCkGpdMDk\\\\\\\/Nx7FabL7dlJSxoGfINzzY11vcmMi2yIfo8le3S+74sk+sNG+czMnDYJKwtF9SzN6F+mfox\\\\\\\/UXwXqTeUIZ1aUBwmYOCxarw8cmA+31f5cdDdspJbtw\\\\\\\/zgzp7bjYrrvj6U0ZJv1bZdwsYR\\\\\\\/c029au\\\\\\\/wHwZdcum5yBEQhwbJxI1STlbRZditAmq4IcP4FhnbDuZKk38OG7xtmu+6zRPzo1foqi42HQfuSaCG55+7zN75xhkJlx1xuv1yonoVjbs+ydsJ7BO8FpECahxqZaFyzgcZ\\\\\\\/7r7IFjuaR0PYSiOqzuV1\\\\\\\/cnb4LsKDkWsNukF\\\\\\\/awaaEBRdzY2y4QsP6dzi7nx4O1McbcKW\\\\\\\/kCck5+CIuMWQboimbfP0KOWpfv7g2XKJrFBseUPm2jZXlhXO7OEcXZIvsk2RH6GN6nRf57RCOQb63X0e4Plc+UyWMbjNuRJ2ZJUm7501nCSyLNIgPMTtQNUOcLiFU19Xo9Leyt9MzJvmRKdM7BLIrvPr6upPyaGWT2WD7xwH+j2fAmvATQ2p77BKpoipwCBdWcQUvpmmkq8WfsrvTNpOjaUbI1AZ3P+uIHB\\\\\\\/CTLzweUAePJk6dW90VI4Ez\\\\\\\/+SzZvrnnpg6KjC0n8UKw93n5anYPEPk8CDdW\\\\\\\/mZ\\\\\\\/lj32lzesfjkqoXx23Z7wvIULM7A6zs\\\\\\\/224lzJqzyPMPR6h19SkeSr7BSLaiwUfqCMNdkwTyiMR9QGm3ZWX4bnQ4vDZ9jCaMMoa7BRYuYQNQy7z2nstcFXZSsxDzaYuYMfY8fNw288z8kQShT2mHmkPLwgzBMcwt8h\\\\\\\/qTuclzFuQ6F8F7UAz53wf7kp3KEVHDQ90V1DmiWoN20XlW41B477AL8\\\\\\\/Qhghqmw0EB2SFkJeOBrIoHRiLVPGiqAJ\\\\\\\/xJqOMs1xBwa2g2Y5L+\\\\\\\/fpRUoLOoBRZggL4MXV6kVNE803ume30vzSgyUKSvrG6hwcu9ByfwdXpG2AxRWcwy6kIvq0Rx+5IglEdq3O341qhDZOtxTtXH2r\\\\\\\/udaJQS80BIR3tcuis7srPMQPasv2DDmySmf6+hY232NfIkYMgQwNvojjqnjUN9TlBQE5u2RrzKwlQYMdgLpP+4DFT5r\\\\\\\/7mwiZOkuh7HknhDaFFB0ONtQL\\\\\\\/8bnpmptO2Ls867YE\\\\\\\/5UlE7wd\\\\\\\/AiZ\\\\\\\/FWFBA5\\\\\\\/TCvVttRujmPKERR6H7H6vUl04FoZzXy+GNuqrx5bYh4UISiYDgHqIjjA3fS6D+vCpfD74vrQ7Dl+7E0JBPkIlw2Sllq+Z\\\\\\\/V1ZlW5wUyTE\\\\\\\/vwtbbvYPs0tIIeaMcEqbX4jbJSLLSbLn41wwnjtnn0aGcKaKprG4F9q2E+vV+DQgSigJyQy6nN\\\\\\\/B1cazXlz4QJb5emtTl302O1TvABQEjcxGwzefEF6otBAoqIviGIZf8tGyCqwWxjtywY3taM2igS0uau0ethPU\\\\\\\/7E0T8F+9ZTGEXNiurqOj6+oT1jZuV9NyzfTch8TBm+dwuVrBxHx\\\\\\\/ll9ytCRvPJStpAgTuHU2jYjPwcLpOSZ3j+fhzoN6r3bVkj+8WPE\\\\\\\/oeMuArVI0vfcTMglYGNuj2zQ5O4wyciEqIbwXp4m89DtiWi9Ft6OxOiDT+opVlo9dmfGSQm7XPUQklp+cMzjcLJAVaM3Q0wGKyRF2Is3gd58ag1RpPSJW83IIu2rH6Zmn66UUA84LOi6AWr5HoGekwWj6VsdEgGB7d3+olb7v68XAPUYYcRuCF+9c3gHO5SiSQGBvNgfC930D2iMaCbcrWRLunbAIr+G2GajKl5HAjGnzOsPPrFTiayvyFVlFEmAsTR18Trml3vMCWj4s9pt2GzO+G\\\\\\\/19FgcKHIVsDpng0FFQZtzTAHHdTl4zCh9vAbtM17jV6IWwPC2uciqb8lZOA4MjT7S4KeHBtGrw0xwRmzcdARKgCR36EOPvmrX4q2qDMDnkkWhLXT6y\\\\\\\/bfdIUUEr9KbffFIsv7hMsix\\\\\\\/bB4FtM8xzJdwT9QUAjjVEw9poeLNsGwiI8RhyhrEYLKZh50Rvfoc20S9XJyfJUf0KRFIJKbQCzHz7BdaB+OY2KQC7wUtVd61T0FalK8XkFpCioSyaTsjSepoYUBxXNjFJkTzB0uHZoPZcF4ikxiHcgYX1KgchOwoZ\\\\\\\/5RODGH\\\\\\\/GnJLsE75JCSRbSVwyeG7z7A+0fM6a0ZYrzYTW+Y7xmBGcB\\\\\\\/ULF0n8tFQjN77\\\\\\\/hrKF2Z2u3\\\\\\\/2CtqpbBYaB5tv88zzuWPROrgod8KjVi9FkboHzwTJpYmLCsMoEUNC+XvanMbGJO5cE7KPND8CiszDj9XKyQeX4kuFWAtsdNzxDLzyFC4xPvZpkJ7GzByNci09GYjE9ATZ7g7B1P++9SeurCHnABWaJfXHxeyenUu6WCFLP4zTMfX0SWPZC86kbAp2nXBmcKX4TEIOQFjep0Zuda7XDVxD\\\\\\\/nzUbSLGM1+MzCo8+6Y+btB2FzRpbdQatu0mVtZ63N44RLWsfFOhCTTMvuK6cCfzeZ8MrS6k+oqLZCSmrO79ZoF7bwalCb5A+IIv9\\\\\\\/wgNUEDoOfcLcAP41146JgSCE5f0KuBagYPm8QHmNq2tUx7GrZGIH5JGnwjdethz0pDWnwC\\\\\\\/xTArBC\\\\\\\/1LT5EOswfxjfW73jxDHZlIPS2JFK7NniyDFMykM+NM0SFrzPAAmwFCBi\\\\\\\/\\\\\\\/o3cHOZCzotf\\\\\\\/U1DdYccOziWkkt4gwVLZ3G6Pfr7zduqFqgkFpXA5\\\\\\\/t589heIdRXamBqz47nbV9928Z\\\\\\\/CoEJsH83Iesw+mMQZH2JMlrEChurHqB616YWXcmk6YtfjmzSydePiDhIQnDPmUsHFNt0uA2e3uywZ1WIAseKRzXK681C6k2W63aeBiSIzJCtgtj428oucmsE6CTZfd9oVKgXxU5Kttc5hWaV1PWwF8cdgwCHbIYObzytWB\\\\\\\/Xip5Sftsno1JdkgO3\\\\\\\/QxjgobqT0L1\\\\\\\/70cmFykKJPIYJDjm+tLjntpD7ToWHTjj6z8X6q0VmQQHOLCsa2UecTqeEDYg2Nt\\\\\\\/NuQXzjizipwengNhl2EBJiVBHU+vGXGppjGLJORp5x5aUd8b3Rgj8NbCLGYWXPQ3LcebGpDgRqlIw4+3X\\\\\\\/Sr1QD83aehoeysOt7Ttra55KJ47Ld2sCR4fHa27p5wRie7tvUIJygyowo4ZhcaCWUPAdZJmS7s77pKRRRSdzQrLoWmKYFz4j0I0ce6bXs92ZS5otst6Tqk4w95juh3g590\\\\\\\/OrM1m0gHQ7PVMCGGhPi2hjaGnc2XX8jzBvQ47lEo15hGbqTZ1VXMr4c6ty9Kd1p8FEl2IssDVFUBXU\\\\\\\/bOyl81qAthDIaBSO8rdN\\\\\\\/wCS\\\\\\\/qDNkI8lb+y4lMJhVvqHszffUMKSCEaOH8ypLSfVs3N1RB03hWLv9trqV1hoGC1Ukd\\\\\\\/hPlQdhY458Mar2URBMWViCsP6ZwAJqgu0bj9CwfF6azqBRyRYgEEP\\\\\\\/nNK14pi9hoim3\\\\\\\/\\\\\\\/12sb80VnacfOcLmchCyt9wowb0F4buvon\\\\\\\/bAXDF5bhbxePLTQEiX\\\\\\\/tS+58qy0CSTpbZOPqGib30HLkZmmt163kIjdHPRy\\\\\\\/5gTRiIhkHrAUhF57l1QmWWe8yvH\\\\\\\/\\\\\\\/i81sa01RD2ljn6PcBfEPNYiHBnuPbjn0eepAtXOBcZ3SyGWk17F3kyTaqXK+YoLv8fx4j2lI33z9KnSGTIUt\\\\\\\/YJNX1koJgz8a+T836\\\\\\\/33ULtNtcBSloHGzyMq+k1RfNoYQ2jlCkdT3EwAKX2kjtXA0dkMWji75nlCHvPZpXb2KTQYS9Jdr2gInzpT2NT8zNsilB15JeSjXuUVdkWf0fK1uZxtuWvEBGrKme6Ivq8f91vLg1iernaRnbWBLeqPsXbB\\\\\\\/bKVAXF6HdR\\\\\\\/NOeySBUd+mVP+1KSM7CGSgLVtPU6Keb9HcVqDN0OFhi2UltiCoiZdDKo\\\\\\\/MScjp6OkuRQBu7\\\\\\\/9+bpKLcSyt\\\\\\\/lp\\\\\\\/FwbBISf4BFTf53FfxUAZj3s\\\\\\\/dT50iu4iS8ioeDjewB+YjpfLq8xaGiBCcKVYF+zC1QTdd6+\\\\\\\/LNTFAbsH9JTihhTT19qhgERmuA8kblUsmAN9xnRlEuayRPX6Jdtjw1nSqWvSG9OT7mJD01cjY4HWSkU3wSV67uC6ggbPNQ+Y50guGeSP45NMV1QY+Mm4ZtwIgavFW20BscG0V6yjaVdd1ZGyYV2saUXl2bA54OapEhhJli01iKCQUIY2PeXuKUcnFXl5IJUQNrjMuCrUoESFZW3MR9u6zOquKktAdqXLDovjMtfIOrLW6uaCPoBmHE9Ynpg6ewy6O9gMeKuqjku3NJqBTPqJSm6jEVfouKK1LSZ9ai8btxNSTT8ATBL+nIITolfq+5mIhYRK6bkWnRkZ6iGVaMzzMpJ3of1Zo6RglE7AvIXXAJv6AOVns2frR1dlsLXmSDcMXlQfyeMWO7Ve9amLpyxliKBuxLaMq2+qwXqOGDcwLKjrRm\\\\\\\/avOrp9HVGCvAh5UFZUepHVmlQN\\\\\\\/gwvf17KjWFUI1CerTbh6t3hQUP7HNnwRgk0ato53QTD2hO8OtszYzra\\\\\\\/ND6Vl+vUXR9FJCwgxBvdWYNruLi4WFMj6SEKsubhL8jkNkCP0SfQDlsHnm+p90IhM+3U3BgKTn+WDznIXCM75Use+ZPXKDsuSg2O4gOQHUhI\\\\\\\/PvNYRM6VGQH2YAybTKYBwSFeixA8lF2fkO0Ei8Ml\\\\\\\/ahetEKcChskVSATNI2mJHnFUm3cgaTj4U+kHAV0TXb2ZtWbHWtqCdo2wa89VR0GVLEJ\\\\\\\/EyaMxt2UYnU2SDB+o2GFAQWjxOdoAVOtlW5vXnx3tpQxPwOaiJ3ScpD2kIpEuyNyv1Spu5RY8Lfx7QTFqXMHOIqC1jLXEdEGBxShaX8QER78lYQL5vDww7Uu40N596iUSW9I9Qov+2rzKJ+LHGl3GAuNEgS1oZPPtj3kLTzRXA+EKk1vWCPlUht2ZCQnhNsU1Sy5Xk349Iygsed7tn9LnSpFoxmeyGyHb6TpsVFWDJ04X4SEVHCy9Mf+I+kJtGrPbrG978TsNW4TCOT34XgbH2lwuJkoyc8imteqA0noQlvXq80DXE\\\\\\\/mRVLzaricdm\\\\\\\/KWwP62S7059s8zSmEJhXUqUoJFwaNGDparH15pzcOYfch6liFNpIAEadCseP4bi7JuZGH6MAPWyIAfghUyT\\\\\\\/HxU8fLPKi82alfkvqT8nu\\\\\\\/qBUmveXXD4LH2G9WhwAAs9F8CX\\\\\\\/DZXb\\\\\\\/KhO6G3nEHGT5cRypvE\\\\\\\/iUZAMAxAB6z7xdABULMCogGfyEWvlduk82ep3yH2cNppRwFM00mGBZAiKRUAQrf14Ol0tpRkTe0yUs4scRK68+o9TX7sX59ifrZ9MqlM5DbvXIE9ivzDojwM8jXTA8wJmxrguEbYgOsT1myg0vnl3bLyiYKAGc3ZE5vjTU26bPEJZjLEWPa7DmyunYoWyC8SA4AHm4s248ThFeit\\\\\\\/L0cEO0ckeMDECfnPWjTVZzWbwpnGMCpGiP749tbc0wAhxanMpcfGwW+rgxT7dtI4H8sH8S6OFgw6qimDbbLrCpkaawlwk7a705ywVzr6qBRAEj+r0XH995L2ePfFRa\\\\\\\/gw+Lu8KUb1\\\\\\\/9yjcTJ7o\\\\\\\/W+cE1DHdjVT5Mdx1sJwj3nkstU6bkJ1RTPF3eDWPBIJaHF76i5k+cx+0Wa0BvsNkMJYeQpu1abJQZFhEvIVY5X2823XTJcbx1AZOQd40Lkg0\\\\\\\/64J9AqNhdN4OXQ39jhOhBUy+H0BZV0anV\\\\\\\/du5uah+qygOPbuuPT6l+eJzh\\\\\\\/SylrSzV8q8+Rsee0NO\\\\\\\/FUUJa8g9+5y+e5OphjL7ax4Pn9oH1q7RI8UXOBpxFyYIfPwnM8mrHAhm+7+ZfvNAjj8nyRVxobV+VgyCyHkvEGQYEVQP6NNivNUspDfYmDzKThGP0Mic6lWON4I6YkojA4LJLHjw1EuoxZ8jejSdC4eKSrUE41XfY4qPVwu0jFoX1tHESjkFYKaAjKV2MN8dgwaRwjwSqY3z3JWztJRuZVydhxykox7UGhO3gJs\\\\\\\/rpNbKnL7KrQRg0MzIlLrYr\\\\\\\/cDTdIA6utmHt+ekbOb9FxXdA2IWUy0jgCzcCg8RdyhxXAXid9Z+AgBJQ8Ne9S+81Aq+F2byReABwLaRd+plL\\\\\\\/rYsJymCSiPzChXIl5wnVBKBtgNc+QbwVVvRbj5H0N+lwBAGWpQv0nAUoT\\\\\\\/EMo+ZzJP2HtVgKwyvqCUzZxS11a5ICP\\\\\\\/MWl8mcv0tizF9YaomvBueAgH8yYN+fK5yXjz7Dnn4F0lD0\\\\\\\/j+sYUijdkDhzyCKDNsucL6hV+IpTL8LcxFQLuWMu8XdaxgnGNpHRIIF4q9ofDo8vNBVJ8Lg\\\\\\\/q3adkyBUQDnDPiwxeO1P7a3hArDsZFnOPuFdcjrHasWihbb2\\\\\\\/BolGp4rj4iOxkWC03gn1bMYc6zXFf1caTWjuQJ8N6i\\\\\\\/8YjiK5vupaaUzjQgeo64lRSYwtPkwlXrQBwsnUZqrsAx22QiO2X2dFmXIMbKrgN8ODrZ8PzsnuQGiafQq0AXDfyLycG0GWhgetnsYTuAk4QDi3pCN2SRwuBOukYhmqnfg7OtxvYFyaCv5gT2jYYv1Yib5TZM15XspkVBPwM4Ro8W7iM3+AgVCUVsjnZCM6ZjGW9WZnDe1xzNZYBg8AHRNHxsJyyMrjLsFGJ7r4lgSmeCpSI3p+pN+uImNB0upBAG4FJTcsBV6YINUds9wdRrkN6hntIH\\\\\\\/qYq5bpzR\\\\\\\/+jhtAxKP8lazUN5dUcmOHS9OP7Se9fzIEZZy2q8K0iRteZ3n1hJa2FS4+lvNQyZPlbjLy1iZCaNXvLgtvvBY761wMk4JR0R4Ol+4YPNWJE+5O6yJvcTUuLUU6QLQcghKg8j3qCEfvahW0QqeDGbryNNvY\\\\\\\/9JmZ\\\\\\\/MeyCeFkSrt7QTm4cUlYZnNww49SfEHfT7RcNYoFm8rrvKSuddwSRV+PxeHoc0yXlPjzpX4zN+OeLd73rkPQIe1QxzmULX64nEup\\\\\\\/4ChHaG7vU35E16Esdbtg2N6xtVewE\\\\\\\/yeGNUU+sYs1m9orT3qwQlD6TWU4rZHRldlzkIFlZacti9l5k2Qz8hAqpzuZ6ZXEqyf6+1GmdZAVddwq1rC\\\\\\\/bxkemTulOzGNN4HgqT9bYwgZAip1rx+ETbTSEe74KA3BSy\\\\\\\/SwBRaXI94dZhvy26nMtFkJXiBEpdJ\\\\\\\/mGGvV7WPi27HQe+wHx2T8djm5Y+x20qf6A2H\\\\\\\/fhP6CXKF8dn+tVKxbc8BxvZ+8hvg\\\\\\\/OLRl32w9kGEkZgbCpyJANwyZh19RMJmXS2+JCCkGENcNgH\\\\\\\/WAJHuCgS2SrQ55jbiTeNpzQjGO6\\\\\\\/XZ\\\\\\\/kq22eHdwgo3FAtFRYxAg51h72GxcHFORBzYXAqU962KthoIDYzZBqQOruAtKWDE36jC2SeejZXoeyqHfGTnbK+YJAGYtmAhBQ+voIDYD091U3+Ktw7v36HZKLmMJ+bJ8t4ITbQqfldRvoCvfAGTvIUwVwyNsu72rn7xNQkG2r2YBL4QavcbqHZ78T1Ht3yd\\\\\\\/uVYsi+F9YkLcMFJCK326JS2Vv1cVbb6+il72Z6CxwRnUuMQj9kC0zaWGSIvp3A2lsSwE6UKZsO3V8m2PjAHA15dK1aS6wyYHbDu7\\\\\\\/MPR\\\\\\\/aFWH0YMXm40ZVrOpgkJjUuVWErT8GifVyC5TYtsbgPNlcyZIyD9VpbMowfjWV+LGADWSJ4bmHlFjyA1HkwRYPNf2Wefj6uzGj+g1DDVQXTOaVRWEBPCute\\\\\\\/q+HkHzTQ3IE+x2KTmoqlXaQV7\\\\\\\/SnAaCc32AN9LyjnDv6NbvoL\\\\\\\/0cRKcXEStEXGVcsBpngO6NR44jROg7VWvuc8BC7+KvbEv5K1g3MFOmNhv3lvBDzl+Htge5hiCWWEGp1h4gPUI1RVaYQ6vCgvgCkQDu\\\\\\\/Oq988sJY9Kx1xgFA2C\\\\\\\/iniPPlGsZoAeiS+6g6DKZRrAVrHD+Vfbk\\\\\\\/uUM01CNGwlKD7EOzzmyPgkb5tW+q4AG97k8gx6Zgk48wqXGfOneC1ZUPJTF7IMRZ5FWM4gRcNUCSOaP78IxvYloQ0fwNZny3IwtKVI2NV8jFMLzx5Pd2oDX1p\\\\\\\/BIdn6zavn9wOFnjKti7s1jdtHfSDjDQ58uZLfMIIIMW6zA3wfikc3meKJrwS\\\\\\\/Y08wJmKRlus+2P6JfJ70UwB7Ecm6Ou4tjzxAASnOq3J0fIyR5WG4P8eSredRgqEKkBV0oXyCEZPS4wsI3uFC7folSsMXMhYtCpqlclWMPCbjnccZrW5kO+8DUAqkAOX9Rx8779luAYPhPFpWUzSjHEhf7y5MblAT56\\\\\\\/03d2gLduV\\\\\\\/Z+IjRG7mxSb8OtRA1xPJipH3LQcPFaOslItkV1SAF5WRNWiKF54YyMO+o\\\\\\\/5+RI5BASUzqlIHmFLXUdM0ZBWMpAbORAo5bMA5VZ8uva9SF4dtC9kHeBV0jClyHOpqI+xw4RNyrjKSMZ4SEXosBMlNlYEVEfEU3pzYy6KqH2wAF2aenmofrPdfu\\\\\\\/5P8ligE6GHQzdWuSWl2k9Jjsk9\\\\\\\/cuGznrgmvw8V6uUhGlo5iSXVPwJOiHxzvUBcOljBoli2rV1V6Cs0X+NnxkmW2F1rKu3oz5\\\\\\\/86gLrB7ErgRomHDwx5Afda2OBeeJADkdNJd4GdHut6jiAwhgqRBj37F58w4MRMR7XokRAaZGusjoxbRfNOKpcPpLieT5BHIh3j89twmo48gri\\\\\\\/K6TerkgQ6J1FWkkH7+uvdu76VGTdAti4GOONmhw+mq3ER64Kzlw7FCehn3QkAK2qX8ccL+dK35ib\\\\\\\/aHmRgneS9VG6Lx41BcrAWOkNQDLZJhlAX7Bpaa6vCqaXwWDp6bHd0N+8CeS4kZdbfv4ak5dd2ST0azFwzvRk\\\\\\\/iuUbL348N9sZZnK3eYrvtGj0IsCZiK8nPL+towAU85sLGNiCHucBD1881912zBFBjsk7ZlVAJiDGGhpx5GqUYR9GrMF+ndpkpj9y8H5zZOItgEGBiBUdsmvG1D9BHJOpihVRv30jTYdHzGgrhLQALFFZTmQQz4hAOKM+WYrG6qlw1yCYEZb7d0M5R1Je7XxhhpoZZYj76eB8uH9tB7T2SPuxMTPB4wWO\\\\\\\/5Mj6VLrpBJxeTfmNOvia+x9zT7o5RiLZZ3AgEPJUmUTJNJ+5virhbezkx8K272o5w6eXnKo+JPaaj7Cz6AAd8yl8nVF5hCYj9WL1AwqpB3462X7XdDGTJX+RCmuBG537C0AZTOqa73JSub0bRngIYIzGt4LDQFEdK+Lce1d\\\\\\\/5xuWKYupCDL8hCtaN1Y\\\\\\\/GSvBTJ\\\\\\\/zb8U8xa2hZsCvYNo0oEv6JLwCM2nPzeI5PQzrHtLfDhpA0hLuzBLLItOTELWGuSOVQGts4lrA8Hrv73pUPeH3IdvCvLNaWvf60n+aidek14rbomPXbciNjYKDqPl9W6SzsEDN6sOusv5LB66VUidw5JudWWjnCoCvkT4D1LNvG\\\\\\\/0LSWE6dU2cxzVjo6iH8QWWPCAHOPC+qGuLUMKCru4jsyE33RSfEb4j\\\\\\\/YEaBrJXbjygSP4UQiqA0t6cOKAXHzypsdMoKpzkr4zHg7\\\\\\\/S3K4kgzZfT\\\\\\\/7jGroCQAPVD2420hKzE6TQ9g6QpV22itw\\\\\\\/YTzsodERr+qzy+47JK2l3uSgjcSWwcRPzF+fq0NIgbTjAUL1uaYwKR1e30Mf9nochy2mdvN3j6yxWwiKcKSEMfCuVJzsdc+o1hxEOhU589mfk\\\\\\\/j1hFR4R2AC9joYVem0MqIZx5f7e8uBafsuPQIqCm0ztSnyX4cJXD1UHgEiP1kR0p02UxXvCDPCUGe4BQxZKebJZ8\\\\\\\/xbl4DaMS4r05t5ikVHGssuGDiQme4LwNP8DuJQpXJHTLE5CiMa\\\\\\\/kAObcj11oMSbxDx2DfdvK20r4TMfcQHOo1FmNGSQtyNdRiMujjBVSkaKB6\\\\\\\/otneke+RVFKnZbc91+exdYbxvPFKbOIjteGsJ8OP0Q2BaTD7hjE70Xgq0BdlMmP3G1ECvt5NZF1KU5xAHPiUQibyylrj5KW3LZ6tEbtrok4JMh+w2LWfAueqICrluuldcVmrHUoK8JrWmjdwFli2vr659abKFhBk1HTuiYUSWs1CJyL\\\\\\\/oHFqb9Plr+MDIY2IAl79qwt7wTfHUblwkXVaMNtrdy0qOoitHfcxFYbE6hypcA9Y21apcLOBTsPY1uNjMb4MJoSbSWg+l1Ue\\\\\\\/ZhtD+NmdyydDXY2oY\\\\\\\/eGEMiYarK3vru71TfER2KSxz\\\\\\\/Z\\\\\\\/anyRrS4lCgb7hTmIyuubBCs4KDBQ2eu0WLAuRR52uuLE5W8\\\\\\\/nLrcgXfhwTcrwA5ztr9jcR9Inmb8BhhlsUo0qzEgS\\\\\\\/V3zM9irrIDiu7G3S7W4Hl5jP3Ty1DMiSLnb5YMi1iLlFtpILg6ldmUKoBSzR9+XmRyUHFRYUYgn237iToV5lBCd+T7fZen\\\\\\\/4YRRVOq7CZ2HP6qkOsfDExINmw0jaDpR3NaR3VfInIJmH8+5565BJ\\\\\\\/k30k5XdP68UnpGQobAaZbso4szo1lEoCA+3bDQsjr\\\\\\\/njHClthKI6BzXn4iJxjL0YsY4lWSAuifpULevFvNcVJeYzgg9MPrP25g9w2zX+1u4\\\\\\\/nJ0xS6DXYtO6iinAglyadIAiNZqEKal5QRINac\\\\\\\/dcVudnaWsMloyTQS6MWybxVB2yfvEueSgitYqVXBU4gNzZfmVK+KssebjIqBT\\\\\\\/P2SYnhA+o9zkg8JpYnbcuQywcN6MQpcm7j+j7i8jMyHEgv+ug1ug5tmZmUXvrY3+KbmTdhrlKZ6aAmvOPZMVM2LJ0E+eXqjpYmG1enSr3JxT\\\\\\\/wwCILIEa6VoAaNsHe0qOGiv92Ee+azhquiHRBlAvDLxmkroz+AcVY8Gcd2tdxahUYlSAiE9134tRnkJRo4tIAzKJR5lVfiJ9Sz5lmHvEAKpnFQN77GifiTKounR\\\\\\\/L7u6tFRez\\\\\\\/a5yfUUaLCDrfW1sV3WesTCrEpt4bed6jZeXVa8iZFsORYusCWiHngyi3M5\\\\\\\/4IX0OoEF71mVC8y9pV7dsOOOcD+bHtEVM2PS8WkpzrqirEcadtzn+6s3ZBmX5mWZVC7qmwgWhcJybZasbIOr5z5sqKqs\\\\\\\/xXYvpl0pgDPYmW\\\\\\\/rh1s++GxtbDsgJ6v1T4kHlGSg5zcQzcOeq+4ejFZcIDvcOOeEz9UH091qMnv01Rz9QUc1n46vEawXHSRrDCGN9Ziq2Dx7avSprcCACNNPMa2qmdgJKd1OD8+E3hjfkBqiIHZvjOOzKGRVeNT1F5kWCPcD9LXfsihO7ZfOqVBcFTJfv8ALGa7rw4PQBHmIBkP61EKJFr7Sndql+Ervsu4M2Qs1XbL48kHlDDsdyFDTl3UpUCyxuVqxZT5qrB4NMNRAD24St3JMF8iZzPaBfx2jpwi\\\\\\\/wygsm0Gm5Dtp10JGgK\\\\\\\/kopYDuMI+uHIxSFCQ3+Ou3YHKjGYXeY0loU71oGlF39MKooZtSWggSHW393zTu2SunU\\\\\\\/rIWdtPfFPhVlcmLCk7kjGcN9LJjJzRE0xnQ9ehsEhB2SFXLsNbMTbTmPJkk7Y6C+\\\\\\\/uEWsQ+1qMci+7C83jcxqOpMrfV+RmvkPD8B8FqpC8KWwUH8oZO1DtO9qQKznVkZMguxbgXhdXBvrGQlYwcLdp5vWehJv1giq8llRz2u6IykgOGvfcxfu8c7d7tBxHyGldIODAnmDVhLFcVTC26t82BO8TEOerXffGOfzqTr7h4\\\\\\\/UL5jYZ\\\\\\\/hIejP2IAvvOdtngZaLbMwb8sb1xBTb\\\\\\\/0PbbUqaVNMGDfBnYm3q4aD+IdJVkUHKkEXX\\\\\\\/NdPNZU9rdXWvA2pvJZ0ayjXp92L8MqJyKli3fISiZlkWhKmuzlEXgbgDzIRkrZmTwG\\\\\\\/q1wMi8ANIbo9hLSXzh0gbK1LBZMWaEYAqwQuB\\\\\\\/c08n6AqNpcpcNDxQu2GuFbARiKn\\\\\\\/LJzgB975zmhfyGMIwKodHWubcoYfx\\\\\\\/5YqmgOfnen1c8okQmUpMTW+\\\\\\\/tdmJev8Hup9TaraFGkElgFu47k9T+4KOTvYnMCWYYQTS07GSEwiVwOuqKMGJiwKeM0eXGnZw\\\\\\\/h5sT6HCqdyjD2Ha4vgcRKaFLLsUkZWAxGrwUpo+e9Y9+p+6kcBN5Jzhzj0hHsgLLBBD265Ch2SzOhZRvSR9rXw4ZgqqjDMmF\\\\\\\/1q3yVSl16rQ2ljwaG0A4gyP5lJQ+bH9JkMvVIz\\\\\\\/qJ2lkX6FHKhgAKCXmQfwO4lxuWbNr\\\\\\\/DtvLuFOXNgSXKy6oXznAFNbIB32EBKDykSn9zFCVxp0tYoGi118rPynn6PH526rVwj7ghmStYohRmt4bkDoxeLO4cxp+abpbHWaRghAWZh4mvrtEzas\\\\\\\/II6F2arthvvWQ3Ln5Y2BlMKW9W5QEHhq+9yvgfOiLCBU1n91IBk0rPzWmQrOmZbMEYeoM3kt\\\\\\\/3T8QOi\\\\\\\/fh6Kz92dHAKS0jaJw2trl4ZvpdWV+TGu5kD9ZQqia3m9QIzZsxnooScuP8BcTocxy3G8QD21Vr0+\\\\\\\/5deH85BEF\\\\\\\/guoSlArXm\\\\\\\/JNkWWHYjf6d7zxqtQrd8+ErGsbO5iWYlHBHE9BmrNXx5bHb0gwpK7x5GO\\\\\\\/llNVjJJ03lAmHcK2zNvVkIMZLGe6YX9+ncGs3LvdcQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ivKdtb9U1FkEWaUN3TMFvA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222a94022dd52ba3348a2e7b4c64e50c086ca2192a377e56abb719ca8a0d560382\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022B39Fri0cpy+S9tbpvloEE4CnB4FrdPMJ8d11Lc8I\\\\\\\/MF0ox129762WSP41TUt\\\\\\\/dxYHYTiyLSRWKfc7L5OHE2veKsKFLOHrvPvB\\\\\\\/2gaH7ckCt4l7IxACa8MCeW4PC0NvLj4yUXkKEt+cpVVVGgx4CXfF4vZdRvS0TZ3KncPB\\\\\\\/F+3ul\\\\\\\/diqVyPblBEAIkA6EXXr9OWq2xvPQdpyvpNACPcrr1DzEgXIOb3u1FolGIFqK+FO5XHgKjukqkJxVJBETR\\\\\\\/1eE4SDHnXquEG9QZo\\\\\\\/WO5gsdigyOl7QHPThgRuE1anMhcKgzLzkE1Q9r80vLlhEs4PFy450KubkFhEZOdS9vS80TlFobEf7VlNnpSvLpSg8fm+NDGSdHwtAfNoEH4\\\\\\\/uhprS8AtnvPsax59VK6IPeEslW5T3Hin8LSGSVWdTuRdyvHXfIFBgq+KV2IhXo7Q26yHiLXbnrqwxKkXRpZ7c1\\\\\\\/zDMu0BvqSRoPNYcCvwGbU9UVMg+ALoEsI\\\\\\\/kWvD1r8+h9fM3QyFnos99M0iEyzifqU57rqqQzQg9ZDM3c9TgndSaw5D2ouTqYpTwl6BiXzRfhLS9tWWzSji3YbfORHw+QcE\\\\\\\/tybMKi6fXMP8O1wVxYkBMOagqmODtqNMDdibFarYeM55MoVN0RrIVCGtASaAVChowlJj7rsyQ4aB+TVn1OGbA032juFVdhiO\\\\\\\/IuqCS4DpdgF5eqG9Yd8F6elcgLfwQdCbRkW7EAKmuNwGHNeF\\\\\\\/a8g1D7PHxfP5ao7fYn40uh6GNvfyBv4cB\\\\\\\/Fgbofn7wqDJAQQQxxDfNRPZEwqjIrYiWqc\\\\\\\/qOoAHjPTjkexDFEjZmmzm3WzL\\\\\\\/rXAIMEPLR+Xr4UkQQoX1j3FARv1110T+wyHB\\\\\\\/f4bqi4mEJKRI7xWfHepcppwjuHuPqTIPgutP5ICRtKFlIVX7JnAepO4RR1jHuaBaP4s5jgS0jN57BXDRbkArfFAfWhv\\\\\\\/tdK5O5ejGpSCG6776j67RXY6AZq0Dc9O7AbW\\\\\\\/NVscNsNQZcjAED3T3zuoHl0i8AwXZcCzEiZpoDjS7AphRQdYcnQ3+zLbHgSmX0292y3MCTJfV2uR0tt4CXxE4hnaVrXzcWu7lcgfWk4vPuh7RVhhZDryfHfjjNOFCQ8p1VS\\\\\\\/UWhcw0uWXafEwIqm1szVGputU0gjWlLCyvSDW6+X592Yoy8vM1zWJqjboXHFSOIlLHGGobwUwboxwxXDMTVm10WVslf0DNb9nxvgNCE5UtDv8ETy2VjMQTfG31Q8PzQmI1DDF9IURs+9QaaJyjl030me81ZPTZ7n7qtTW4\\\\\\\/DraP+r0rz2wXcmqvhCrDXoN\\\\\\\/TaiBeuuKSUy9Boshn5HncHritaWZ+UZ5eXDmzXLG6WjDfKJeWlRg2647pq2OfCw\\\\\\\/IqzLmU18yYh92CEY089eURjsSKLb0gt1Vx\\\\\\\/dq4wQUbTNCMNVFu63avD\\\\\\\/Kk7f1QqQIr8xR9jFIM4YTwPMLGbHKSTUyRq6xfKBK8LCjqUItHPAY3zX40XCgjIE8LgMD8PBkACskme7u1vV2vdFfGTIzwLk5sEqwLWtN82OWX9W2RBBGPVsutO4mD1JSJ3j81daNlZIlRispZNKzfuRI4z52RLzgAbKCOeq8zJ8k3t\\\\\\\/W2nD5mql36OJexWl4Q7oiyfarWCm+mqSG0B8Q4UbUYUgxNFxbz21UDjAxVGOi2myF07Yf3uVXxQ5OxVy1Wm27cgAWaOrU9EwBFgWYOzG6z7x+yxZY3TvYTFViz8hahkTk36JHeFK2P5TAou4WQnE3WiZ7obSfDrCXWrLdokygz+3hU2gWfTLFmH7+Bllx31sOV6uQtSA70iXB81wjIwfcZoLBXpfpaQW5EPP9N7GIxRXWliYJ2q5tIVKUsVqS1j5u\\\\\\\/sNpKTrkZw5QjdAzDVkyjnVEc9wDFg2+i\\\\\\\/4fJJ1SW8VKL3bABY5NKeEjh8JZmzDuEuEIQ\\\\\\\/3WS0p8iW9pxKxaWPF4jgYrmMUREpZgaTEaPOsmsXuzHIAy9FtnFE1Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/ax41hpeLU\\\\\\\/l9XborNSZPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223c5c3aba7aef339b0fcb9d4ac51e14cab8c9ce28c6ab3cffb0cebf6697c8f632\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494995 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4183" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MeF89i8yTmQdEvmJPlb6lcEvab2WdS_jgOr-woFxMck" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4183" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-s8Ls1hd8DGiI5Mq3IhLQVgWUp9xqqQs98Yn0XmmWbEk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4169" title="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4197" title="Expression of Heat Shock Protein 70 Is Insufficient To Extend Drosophila melanogaster Longevity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494995" id="node494995" data-pisa="ggg;9/12/4183" data-pisa-master="ggg;g3.119.400603" data-apath="/ggg/9/12/4183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Classification-Based Inference of Dynamical Models of Gene Regulatory Networks</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David A. Fehr</span>, <span class="highwire-citation-author" data-delta="1">Joanna E. Handzlik</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7677-7228" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manu</span> and <span class="highwire-citation-author" data-delta="3">Yen Lee Loh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4183-4195;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400603 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494995" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David A. Fehr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Physics and Astrophysics, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BA.%2BFehr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fehr%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BA.%2BFehr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joanna E. Handzlik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Dakota, Grand Forks, ND 58202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BE.%2BHandzlik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Handzlik%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoanna%2BE.%2BHandzlik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name"> Manu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Dakota, Grand Forks, ND 58202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%2BManu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manu&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%2BManu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7677-7228" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for  Manu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:manu.manu@und.edu" class="" data-icon-position="" data-hide-link-title="0">manu.manu@und.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yen Lee Loh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Physics and Astrophysics, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yen%2BLee%2BLoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loh%20YL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYen%2BLee%2BLoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yenlee.loh@und.edu" class="" data-icon-position="" data-hide-link-title="0">yenlee.loh@und.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/4183" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494995" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494995/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4183.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494995" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494995/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4183.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494995" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494995/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/4183.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494995" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494995/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Cell-fate decisions during development are controlled by densely interconnected gene regulatory networks (GRNs) consisting of many genes. Inferring and predictively modeling these GRNs is crucial for understanding development and other physiological processes. Gene circuits, coupled differential equations that represent gene product synthesis with a switch-like function, provide a biologically realistic framework for modeling the time evolution of gene expression. However, their use has been limited to smaller networks due to the computational expense of inferring model parameters from gene expression data using global non-linear optimization. Here we show that the switch-like nature of gene regulation can be exploited to break the gene circuit inference problem into two simpler optimization problems that are amenable to computationally efficient supervised learning techniques. We present FIGR (Fast Inference of Gene Regulation), a novel classification-based inference approach to determining gene circuit parameters. We demonstrate FIGR’s effectiveness on synthetic data generated from random gene circuits of up to 50 genes as well as experimental data from the gap gene system of <em>Drosophila melanogaster</em>, a benchmark for inferring dynamical GRN models. FIGR is faster than global non-linear optimization by a factor of 600 and its computational complexity scales much better with GRN size. On a practical level, FIGR can accurately infer the biologically realistic gap gene network in under a minute on desktop-class hardware instead of requiring hours of parallel computing. We anticipate that FIGR would enable the inference of much larger biologically realistic GRNs than was possible before.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow">drosophila</a></li><li class="kwd"><a href="/keyword/gap-gene-regulation" class="hw-term hw-article-keyword hw-article-keyword-gap-gene-regulation" rel="nofollow">gap gene regulation</a></li><li class="kwd"><a href="/keyword/differential-equation-models" class="hw-term hw-article-keyword hw-article-keyword-differential-equation-models" rel="nofollow">differential equation models</a></li><li class="kwd"><a href="/keyword/pattern-formation" class="hw-term hw-article-keyword hw-article-keyword-pattern-formation" rel="nofollow">pattern formation</a></li><li class="kwd"><a href="/keyword/parameter-inference" class="hw-term hw-article-keyword hw-article-keyword-parameter-inference" rel="nofollow">parameter inference</a></li><li class="kwd"><a href="/keyword/binary-classification" class="hw-term hw-article-keyword hw-article-keyword-binary-classification" rel="nofollow">binary classification</a></li><li class="kwd"><a href="/keyword/development" class="hw-term hw-article-keyword hw-article-keyword-development" rel="nofollow">development</a></li><li class="kwd"><a href="/keyword/gene-regulatory-networks" class="hw-term hw-article-keyword hw-article-keyword-gene-regulatory-networks" rel="nofollow">gene regulatory networks</a></li></ul><p id="p-4">Development is controlled by gene regulatory networks (GRNs) that integrate extrinsic signals and intrinsic cell state to make decisions about cell fate (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Levine and Davidson 2005</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Davidson and Levine 2008</a>). Modeling of GRNs is an important approach to understanding a wide variety of developmental processes such as pattern formation (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">B</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Verd <em>et al.</em> 2018</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Balaskas <em>et al.</em> 2012</a>), cell-fate specification (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hamey <em>et al.</em> 2017</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">May <em>et al.</em> 2013</a>), pluripotency and cell-fate reprogramming (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Collombet <em>et al.</em> 2017</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Li and Wang 2013</a>), oncogenesis (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Tyson <em>et al.</em> 2011</a>), and regeneration (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pietak <em>et al.</em> 2019</a>). Over the past decade or so, it has become clear that developmental GRNs comprise tens to hundreds of densely interconnected genes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davidson <em>et al.</em> 2002a</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Novershtern <em>et al.</em> 2011</a>) rather than a few so-called master regulators. Moreover, developmental GRNs are wired recursively since the genes encoding transcription factors (TFs) are themselves regulated by other TFs or indirectly by non-TF gene products (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Palani and Sarkar 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kueh <em>et al.</em> 2013</a>). Their large size and high interconnectivity make the modeling of developmental GRNs a challenging problem.</p><p id="p-5">Coupled ordinary or partial differential equations (ODEs or PDEs) are a natural choice for modeling GRNs since GRNs are nonlinear dynamical systems (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Weston <em>et al.</em> 2018</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Li and Wang 2013</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Laslo <em>et al.</em> 2006</a>) whose time evolution depends on their state. The state is defined by the concentrations of gene products and the equations are parameterized by constants with a biochemical or biophysical underpinning, such as synthesis and degradation rates and binding constants. Estimating the values of these parameters is necessary for simulating the time evolution of GRN state but direct <em>in vivo</em> biochemical measurement of the large numbers of parameters involved is generally infeasible if not outright impossible. One approach to estimating parameter values is to search in parameter space for broad regions that reproduce the qualitative behavior of the system (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Huang <em>et al.</em> 2007</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Laslo <em>et al.</em> 2006</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Li and Wang 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hong <em>et al.</em> 2012</a>). The other approach (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Reinitz and Sharp 1995</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">May <em>et al.</em> 2013</a>) to parameter estimation is data-driven, that is, parameter values are inferred by fitting the ODEs or PDEs to quantitative observations of GRN state sampled in space and/or time. In inferring parameters from quantitative data, data-driven differential equation modeling of GRNs provides a framework for understanding developmental cellular decisions at a quantitative and predictive level.</p><p id="p-6">Here we focus on a specific data-driven and predictive ODE modeling framework, termed <em>gene circuits</em>, that has been particularly successful in inferring and modeling developmental GRNs from spatiotemporal protein (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>,<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">b</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hengenius <em>et al.</em> 2011</a>) or mRNA (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crombach <em>et al.</em> 2012</a>) data. Gene circuits determine the time evolution of protein or mRNA concentrations using coupled nonlinear ODEs in which synthesis is represented as a switch-like function of regulator concentrations. The values of the free parameters define the regulatory influences among the genes in the network. Gene circuits do not presuppose any particular scheme of regulatory interactions, but instead determine it by estimating the values of the parameters from quantitative data using optimization. Gene circuits infer not only the topology of the GRN but also the type, either activation or repression, and strength of interactions. Most importantly, the inference procedure yields ODE models that can be used to simulate and predict developmental perturbations (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jaeger <em>et al.</em> 2004b</a>; <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>,<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">b</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wu <em>et al.</em> 2015</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Verd <em>et al.</em> 2018</a>).</p><p id="p-7">Despite its successes, the gene circuit method suffers from the drawback that parameter inference is computationally expensive. Efficient optimization methods, such as steepest descent (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gursky <em>et al.</em> 2004</a>) are guaranteed to find the global minimum only if the cost function, usually the sum of squared differences between model output and data, is convex—has a unique minimum—which is not the case in such problems. This implies that the only practical approach currently available for fitting gene circuit models is global nonlinear optimization with techniques such as simulated annealing (SA; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kirkpatrick <em>et al.</em> 1983</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lam and Delosme 1988a</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">b</a>), that minimize the cost function by searching the high-dimensional parameter space stochastically. Not only do global nonlinear optimization methods need to make millions of cost function evaluations in order to find the minimum, but each evaluation is itself quite costly since it involves solving a set of coupled differential equations. Furthermore, the computational cost scales poorly, as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-1.gif"/></noscript></span></span>, with gene number <em>G</em>, since <em>G</em> ODEs are solved in each function evaluation and the number of cost function evaluations required is proportional to the number of parameters (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-2.gif"/></noscript></span></span>). High computational cost and poor scalability have hamstrung the application of the gene circuit method to larger networks or more broadly in development. Gene circuits have only been inferred for relatively small networks so far (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Reinitz and Sharp 1995</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cotterell and Sharpe 2010</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">May <em>et al.</em> 2013</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Verd <em>et al.</em> 2018</a>).</p><p id="p-8">One approach to speeding up the inference procedure has been to explore different global optimization methods such as evolutionary algorithms (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kozlov and Samsonov 2009</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>) and scatter search (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdol <em>et al.</em> 2017</a>). Alternative global optimization methods do not circumvent the problem of high computational cost since each cost function evaluation still involves the solution of coupled ODEs. Another important strategy for inferring gene circuits in a reasonable amount of time has been the development of parallel optimization algorithms such as parallel Lam simulated annealing (pLSA; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chu <em>et al.</em> 1999</a>) and Differential Evolution Entirely Parallel (DEEP; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kozlov and Samsonov 2009</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>), including attempts at reducing communication overhead (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jostins and Jaeger 2010</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lou and Reinitz 2016</a>). Although parallel methods reduce the absolute amount of time required to infer a gene circuit of a given size, they nevertheless suffer from the scaling problem.</p><p id="p-9">In this paper we present an alternative approach, FIGR (Fast Inference of Gene Regulation), for determining gene circuit parameters that is significantly more computationally efficient than global nonlinear optimization. Our algorithm exploits the observation that the inference of the connectivity of a given gene can be rephrased as a supervised learning problem: to find a hyperplane in state space that classifies observations into two groups, one where the gene is ON and the other where the gene is OFF. Our algorithm determines whether a gene is ON or OFF at a given observation point by computing the time derivative of concentrations in a numerically robust manner. It then performs classification using either logistic regression or support vector machines (SVM) to determine the equation of the switching hyperplane. The genetic interconnectivity can then be computed from the coefficients of the hyperplane equation in a straightforward manner. We have implemented the algorithm in MATLAB and tested its ability to recover the genetic interconnectivity of random GRNs of up to 50 genes from simulated data. The algorithm works as expected and recovers parameters accurately, provided that sufficient data are available. We also demonstrate the ability of our method to correctly infer the gap gene regulatory network of <em>Drosophila melanogaster</em> from empirical data. We observed a ∼ 600-fold speed up relative to simulated annealing on the gap gene problem.</p><p id="p-10">The results are presented in two parts. In the first half of the results, we develop the algorithm and validate it on synthetic data. In the second half, we describe the inference of the <em>Drosophila</em> gap gene network from empirical data. We begin the first half by introducing gene circuit equations for cell-autonomous GRNs (Section Gene circuit models of GRNs). Next, we develop the FIGR algorithm (Section FIGR: Classification-based inference) for inferring gene circuits for cell-autonomous GRNs. This is followed by tests assessing the accuracy of FIGR in recovering known parameter values from synthetic data (Section Validation of FIGR on synthetic data). In the second half (Section Inference of the gap gene network from empirical data), we first adapt FIGR to spatially extended systems such as the <em>Drosophila</em> blastoderm (Section FIGR for a spatially extended system) and then compare FIGR-derived gap gene circuits to data and SA-derived gene circuits (Section Gap gene circuit inference and comparison with data and SA).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Validation of FIGR With synthetic data</h3><div id="sec-3" class="subsection"><h4>Generation of synthetic data from random gene circuits:</h4><p id="p-11">Random gene circuits were generated and simulated to generate synthetic data as follows. The synthesis rates <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-3.gif"/></noscript></span></span> and degradation rates <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-4.gif"/></noscript></span></span> were drawn uniformly from the interval <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-5.gif"/></noscript></span></span> . We chose the genetic interconnectivity coefficients <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-6.gif"/></noscript></span></span> and threshold <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-7.gif"/></noscript></span></span> such that the switching hyperplane passed through a random point <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-8.gif"/></noscript></span></span> drawn uniformly from the bounding hypercube (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-9.gif"/></noscript></span></span>), and the normal to the switching hyperplane (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-10.gif"/></noscript></span></span>) was drawn uniformly from the unit <em>G</em>-sphere, where <em>G</em> is the number of genes in the GRN. We generated <em>N</em> trajectories starting at random initial position <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-11.gif"/></noscript></span></span> drawn uniformly from the bounding hypercube by integrating the Glass equations without diffusion (<a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation 3</a>) using MATLAB’s ode45 solver. We stored the values of these functions <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-12.gif"/></noscript></span></span> at <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-13.gif"/></noscript></span></span> timepoints equally subdividing the interval of the simulation (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-14.gif"/></noscript></span></span>) to serve as synthetic data for both FIGR and SA.</p></div><div id="sec-4" class="subsection"><h4>Inference with FIGR:</h4><p id="p-12">Gene circuits were inferred using FIGR as described in Section FIGR: Classification-based inference. The user-defined options and parameters utilized in this study are provided in Table S1.</p></div><div id="sec-5" class="subsection"><h4>Inference with SA:</h4><p id="p-13">SA was carried out with gene circuit C code in serial largely as described previously (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>) save for a few modifications. The quality factor <em>λ</em> was set to 0.001 and the averaging parameters <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-16.gif"/></noscript></span></span> were set to 200 and 1000 respectively. The stopping criterion was 0.001. The parameter controlling the search space of the regulatory parameters Λ was set to 0.1. The search space of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-18.gif"/></noscript></span></span> were set to <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-19.gif"/></noscript></span></span> . The Glass equations (<a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Equation 3</a>) were solved using a 4<sup>th</sup> order Runge-Kutta solver. Since SA is a stochastic method, different optimization runs yield slightly different gene circuits. The inferences were carried out in 5 replicates, each starting for a random set of initial parameter values. For each synthetic dataset, several replicates having low RMS could be identified. The circuit with the lowest RMS was chosen for further analysis.</p></div></div><div id="sec-6" class="subsection"><h3>Inference of gap gene circuits</h3><p id="p-14">Gap gene circuits were inferred from a publicly available whole-mount immunofluorescence dataset (Surkova <em>et al.</em> 2008) of the spatiotemporal expression of the segmentation genes. Data from the 50 min-long cleavage cycle 14 are staged into 8 time classes, T1–T8, spaced 6.25 min apart. We utilized integrated data (<a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>) obtained by subtracting background fluorescence from raw single-embryo data, aligning the spatial patterns of different embryos belonging to a time class, and averaging over several embryos in each time class. See Janssens <em>et al.</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Janssens <em>et al.</em> 2005</a>) for details of how the data were processed.</p><p id="p-15">The user-defined options and parameters utilized for fitting the gap gene circuit with FIGR are provided in Table S1. Wall clock time was measured with MATLAB’s tic/toc functions. SA was carried out with gene circuit C code as described previously (<a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>). Wall clock time was measured with C’s time function.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-16">Figure S1 shows the fraction of genetic interconnectivity signs inferred correctly from synthetic data. Figure S2 shows the training error of SA-inferred gene circuits. Figure S3 shows the inference of <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-20.gif"/></noscript></span></span> and kinetic parameters from synthetic data. Figure S4 compares the spatiotemporal pattern of gap gene expression with the output of gene circuits inferred with FIGR and SA. Table S1 lists user-defined options and parameters utilized in FIGR code. File S1 describes an alternative method for determining kinetic parameters in FIGR. FIGR Source code is freely available at <a href="http://github.com/mlekkha/FIGR">http://github.com/mlekkha/FIGR</a>. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9249245">https://doi.org/10.25387/g3.9249245</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Gene circuit models of GRNs</h3><p id="p-17">We consider a GRN of <em>G</em> genes whose state at time <em>t</em> is defined by the concentrations of the gene products <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-21.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-22.gif"/></noscript></span></span>. We assume that the GRN functions cell autonomously, that is, the expression of the genes is independent of the state of other cells. Gene circuits (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Reinitz and Sharp 1995</a>) describe the time evolution of <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-23.gif"/></noscript></span></span> according to <em>G</em> coupled ordinary differential equations,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-25.gif"/></noscript></span></span> is the maximum synthesis rate of product <em>g</em>. <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-26.gif"/></noscript></span></span> are genetic interconnectivity coefficients describing the regulation of gene <em>g</em> by the product of gene <em>f</em>. Positive and negative values of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-27.gif"/></noscript></span></span> signify activation and repression of gene <em>g</em> by gene <em>f</em> respectively. The threshold <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-28.gif"/></noscript></span></span> determines the basal synthesis rate, and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-29.gif"/></noscript></span></span> is the degradation rate of product <em>g</em>. Nominally, all genes in the model also function as regulators, so that both <em>g</em> and <em>f</em> run over the range <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-30.gif"/></noscript></span></span>. Sometimes such gene networks include upstream regulators that are not themselves influenced by other gene products represented in the model. For example, in the <em>Drosophila</em> segmentation gene network, maternal proteins such as <a href="https://identifiers.org/bioentitylink/FB:FBgn0000166?doi=10.1534/g3.119.400603"><em>bicoid</em></a> activate the zygotically expressed genes, but are not regulated by their targets (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akam 1987</a>). An upstream regulator <em>g</em> can be represented by setting <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-31.gif"/></noscript></span></span> for all <em>f</em>.</p><p id="p-18"><span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-32.gif"/></noscript></span></span> is the regulation-expression function, which determines the fraction of the maximum synthesis rate attained by the gene given the total regulatory input <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-33.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-34.gif"/></noscript></span></span> is required to have a switch-like dependence on <em>u</em> and to take values between 0 and 1. If the total regulatory input has large positive values, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-35.gif"/></noscript></span></span>, as a result of high activator concentrations, low repressor concentrations, or both, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-36.gif"/></noscript></span></span> and the gene product is synthesized at the maximum rate <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-37.gif"/></noscript></span></span>. If the total regulatory input has large negative values, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-38.gif"/></noscript></span></span>, so that <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-39.gif"/></noscript></span></span>, the gene product is not synthesized. One sigmoid function that satisfies these properties,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-40.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>has been utilized almost exclusively in previous studies (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Reinitz and Sharp 1995</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>; <a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>). However, any function that satisfies these rather general properties is a valid regulation-expression function.</p><div id="sec-10" class="subsection"><h4>Glass networks:</h4><p id="p-19">In what follows, we show that one choice of the regulation-expression function permits a radical simplification of the gene circuit inference problem. If the regulation-expression function is chosen to be the Heaviside function,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-41.gif"/></noscript></span></span>the resulting differential equations (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) are piece-wise linear and are referred to as Glass networks (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Glass and Kauffman 1973</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Edwards 2000</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Glass and Pasternack 1978</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mestl <em>et al.</em> 1996</a>).</p><p id="p-20">Using the state vector <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-42.gif"/></noscript></span></span> to represent a point in the <em>G</em>-dimensional state space of the model and the vector <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-43.gif"/></noscript></span></span> to represent the <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-44.gif"/></noscript></span></span> row of the genetic interconnectivity matrix, the Glass equations may be written as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>The gene may said to be “ON” or “OFF” depending on whether the gene product is being synthesized or not respectively. <a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">Equation 3</a> implies that<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-46.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Thus the “gene <em>g</em> ON” and “gene <em>g</em> OFF” configurations are separated in state space by the hyperplane defined by the equation <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-47.gif"/></noscript></span></span>. We call this the <em>switching hyperplane</em>. <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-48.gif"/></noscript></span></span> is the normal to the switching hyperplane and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-49.gif"/></noscript></span></span> is the perpendicular distance of any point x to the hyperplane. Furthermore,<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-50.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span><a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 4</a> and <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation 5</a> imply that for Glass networks, the <em>regulatory parameters</em>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-51.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-52.gif"/></noscript></span></span>, and the <em>kinetic parameters</em>, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-54.gif"/></noscript></span></span>, are separable. The former determine the switching hyperplane, while the latter determine the trajectories on either side of the hyperplane. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1D,G</a> shows examples of the switching hyperplanes and trajectory of a two-gene gene circuit (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A,B</a>) having a stable spiral equilibrium solution.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Classification-based inference of an example gene circuit. A. Theoretical parameter values are listed by row for each gene. The T matrix is shown in the first two columns, one column per regulator. Green (red) indicates activation (repression). B. Schematic of the theoretical gene circuit. C. Parameters inferred by FIGR. D,G. Trajectory in state space (purple) overlaid upon the Heaviside regulation-expression function. Green (red) is ON (OFF). The switching hyperplane is plotted as a blue line. Switching hyperplanes for genes A and B are showing in panels D and G respectively. E,H. Sampled gene expression trajectories and assignment of ON/OFF state for genes A (panel E) and B (panel H). Trajectories are numerical solutions of Equation 3. Detected ON or OFF state (Section Determining ON/OFF state) is indicated with green stars or red circles respectively. F,I. Switching hyperplane (dashed blue line) inferred using Logistic regression for genes A (panel F) and B (panel I). Sampled trajectories annotated with ON/OFF state are plotted in state space." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1090008564" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Classification-based inference of an example gene circuit. A. Theoretical parameter values are listed by row for each gene. The T matrix is shown in the first two columns, one column per regulator. Green (red) indicates activation (repression). B. Schematic of the theoretical gene circuit. C. Parameters inferred by FIGR. D,G. Trajectory in state space (purple) overlaid upon the Heaviside regulation-expression function. Green (red) is ON (OFF). The switching hyperplane is plotted as a blue line. Switching hyperplanes for genes A and B are showing in panels D and G respectively. E,H. Sampled gene expression trajectories and assignment of ON/OFF state for genes A (panel E) and B (panel H). Trajectories are numerical solutions of Equation 3. Detected ON or OFF state (Section Determining ON/OFF state) is indicated with green stars or red circles respectively. F,I. Switching hyperplane (dashed blue line) inferred using Logistic regression for genes A (panel F) and B (panel I). Sampled trajectories annotated with ON/OFF state are plotted in state space.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4183/F1.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/4183/F1.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495096" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Classification-based inference of an example gene circuit. A. Theoretical parameter values are listed by row for each gene. The T matrix is shown in the first two columns, one column per regulator. Green (red) indicates activation (repression). B. Schematic of the theoretical gene circuit. C. Parameters inferred by FIGR. D,G. Trajectory in state space (purple) overlaid upon the Heaviside regulation-expression function. Green (red) is ON (OFF). The switching hyperplane is plotted as a blue line. Switching hyperplanes for genes A and B are showing in panels D and G respectively. E,H. Sampled gene expression trajectories and assignment of ON/OFF state for genes A (panel E) and B (panel H). Trajectories are numerical solutions of <a id="xref-disp-formula-4-4" class="xref-disp-formula" href="#disp-formula-4">Equation 3</a>. Detected ON or OFF state (Section Determining ON/OFF state) is indicated with green stars or red circles respectively. F,I. Switching hyperplane (dashed blue line) inferred using Logistic regression for genes A (panel F) and B (panel I). Sampled trajectories annotated with ON/OFF state are plotted in state space.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-11" class="subsection"><h3>FIGR: Classification-based inference</h3><p id="p-22">Let the expression of each gene be measured at <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-55.gif"/></noscript></span></span> time points <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-56.gif"/></noscript></span></span>, along trajectories starting from <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-57.gif"/></noscript></span></span> initial conditions. The goal of GRN inference is to estimate the values of the gene circuit parameters <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-58.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-59.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-60.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-61.gif"/></noscript></span></span> given the measurements <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-62.gif"/></noscript></span></span>.</p><p id="p-23">In FIGR, we exploit the separability of the regulatory (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-63.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-64.gif"/></noscript></span></span> ) and kinetic (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-65.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-66.gif"/></noscript></span></span>) parameters to break up the inference problem into two distinct tractable subproblems. For inferring the parameters of any given gene, we classify the data points into two classes—one in which the gene’s product is being synthesized (ON class) and the other in which the product is not being synthesized (OFF class). The regulatory parameters are inferred by determining the optimal <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-67.gif"/></noscript></span></span> dimensional hyperplane separating the two classes. The kinetic parameters can be inferred either by fitting the piece-wise linear Glass equations to estimates of the rate of change of gene product concentrations or by fitting <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">Equation 5</a> to the gene product concentration time series.</p><div id="sec-12" class="subsection"><h4>Determining ON/OFF state:</h4><p id="p-24">We will assume that the gene product concentration, including initial concentration, is bounded by the maximum concentration determined by the synthesis and degradation rates, that is,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-68.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Let<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>represent the ON/OFF state of gene <em>g</em>. Then,<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-70.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>This implies that the ON/OFF state of a gene can be determined by ascertaining the sign of the <em>velocity</em>, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-71.gif"/></noscript></span></span> .</p><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-72.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span><p id="p-25">Gene expression data, such as those obtained from immunofluorescence or high-throughput sequencing, inevitably contain noise. If the gene expression level is close to its maximum (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-73.gif"/></noscript></span></span>) or minimum level (<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-74.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-75.gif"/></noscript></span></span> is theoretically close to zero, but noise causes <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-76.gif"/></noscript></span></span> to fluctuate, which might be interpreted as spurious switching events. To avoid this problem, we identify a gene’s ON/OFF state as follows. If the gene expression level <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-77.gif"/></noscript></span></span> is increasing (decreasing) at a rate greater than a user-supplied <em>velocity threshold</em> <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-78.gif"/></noscript></span></span>, then the gene is classified as ON (OFF). Otherwise, if the expression level is above (below) a user-supplied <em>expression threshold</em> <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-79.gif"/></noscript></span></span>, then the gene is classified as ON (OFF). This can be summarized as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-80.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>In our implementation of FIGR, cubic smoothing splines are fit to time series data and differentiated to estimate velocity. <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1E,H</a> illustrates the determination of <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-81.gif"/></noscript></span></span> for an example two-gene network.</p></div><div id="sec-13" class="subsection"><h4>Determining regulatory parameters:</h4><p id="p-26">Within the Glass model, the ON/OFF state of a particular target gene <em>g</em>, whose index we shall omit from now on, is given by <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-82.gif"/></noscript></span></span> Suppose that gene product concentrations have been sampled <em>P</em> times, in time and in one or more conditions or cell types. The gene ON/OFF state <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-83.gif"/></noscript></span></span> is determined for each experimentally measured state vector <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-84.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-85.gif"/></noscript></span></span>, according to the method described above (Section Determining ON/OFF state). Then, the regulatory parameters can be inferred by finding <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-86.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-87.gif"/></noscript></span></span> such that<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-88.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>is satisfied for as many <em>p</em> as possible. Inferring the regulatory parameters therefore reduces to the problem of linear binary classification (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hastie <em>et al.</em> 2009</a>).</p><p id="p-27">There are many well known supervised learning algorithms for linear binary classification. We have used both support vector machines (SVM) and logistic regression. An SVM finds a hyperplane buffered by the biggest possible margin such that the number of points <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-89.gif"/></noscript></span></span> belonging to each class, “gene ON” or “gene OFF”, is maximized on opposite sides of the margin zone. This can be accomplished by minimizing the cost function<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-90.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>where the first term is a regularization penalty that maximizes the margin. The second term is the hinge loss function, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-91.gif"/></noscript></span></span>, which is non-zero only for points that transgress their class boundary, each such point contributing an amount proportional to its distance from the margin. The parameter <em>λ</em> is used to choose the relative weight of the penalty and loss terms. Two-class logistic regression models the posterior probability of the ON/OFF state of a point as a logit transformation of its distance from the switching hyperplane. The optimal switching hyperplane can be found by minimizing <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">Equation 12</a> with a Binomial deviance loss function <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-92.gif"/></noscript></span></span>. <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C,F,I</a> illustrate binary classification for an example two-gene network.</p><p id="p-28">Minimization of <a id="xref-disp-formula-13-2" class="xref-disp-formula" href="#disp-formula-13">Equation 12</a> is a convex optimization problem, which can be solved by quadratic programming or the Newton-Raphson method quite efficiently, even for large <em>G</em>. This is the key benefit of the separation of regulatory and kinetic parameters enabled by the Glass equations.</p></div><div id="sec-14" class="subsection"><h4>Determining kinetic parameters:</h4><p id="p-29">Having identified the ON/OFF state of a gene, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-93.gif"/></noscript></span></span>, for <em>P</em> measurements of its concentration, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-94.gif"/></noscript></span></span>, the Glass equations (<a id="xref-disp-formula-4-5" class="xref-disp-formula" href="#disp-formula-4">Equation 3</a>) can be rewritten as<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-95.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>The velocities <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-96.gif"/></noscript></span></span> are estimated by differentiating cubic smoothing splines fit to the time series data (Section Determining ON/OFF state). Thus, for any particular gene, <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equation 13</a> takes the form of <em>P</em> equations that are linear in the two unknowns <em>R</em> and <em>λ</em>. This is an overdetermined linear system, so best estimates for <em>R</em> and <em>λ</em> can be extracted by least-squares linear regression. In practice, the error in the spline, and hence in <em>v</em>, is the largest when a gene is switching states. We therefore exclude a user-configurable number of time points nearest to switching events. This method is implemented as the <strong>“slope”</strong> method of FIGR. Alternatively, <em>R</em> and <em>λ</em> can also be determined by fitting <a id="xref-disp-formula-6-3" class="xref-disp-formula" href="#disp-formula-6">Equation 5</a> to the concentration data (see File S1).</p></div></div><div id="sec-15" class="subsection"><h3>Validation of FIGR on synthetic data</h3><p id="p-30">As a first test of FIGR, we tested its ability to recover known parameters from synthetic data. In each test, 100 randomly generated gene circuits (Section Validation of FIGR with synthetic data) were simulated using the Glass equations (<a id="xref-disp-formula-4-6" class="xref-disp-formula" href="#disp-formula-4">Equation 3</a>). For each gene circuit, <em>N</em> trajectories starting from random initial starting points were computed and sampled at <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-97.gif"/></noscript></span></span> time points to obtain synthetic time series data resulting in <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-98.gif"/></noscript></span></span> observations per simulation. The quality of the inference depends not only on the effectiveness of the method but also on how well determined the inference problem is. A gene circuit of <em>G</em> genes has <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-99.gif"/></noscript></span></span> parameters. If <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-100.gif"/></noscript></span></span>, then the problem is well determined and the accuracy of the inference depends primarily on the effectiveness of the algorithm. On the other hand if <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-101.gif"/></noscript></span></span> then there isn’t a sufficient amount of data to infer the parameters accurately irrespective of the effectiveness of the algorithm. We checked how effective FIGR is at inferring parameters of gene circuits of various sizes by exploring different combinations of the number of free parameters and the number of data points. With the exception of the 50-gene network, we also inferred the parameters with SA (Section Inference with SA) to serve as a point of reference. Inference of 100 random 20-gene networks took 5 days on 500 CPUs with SA, and hence it was impractical to infer 50-gene networks.</p><p id="p-31">The inferred parameter values were compared to the known values by computing the discrepancies between them. From the viewpoint of correctly predicting a gene’s ON/OFF state, the accuracy of individual genetic interconnectivity coefficients <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-102.gif"/></noscript></span></span> is less important than the accuracy with which the switching hyperplane has been inferred. Accordingly, we judged the accuracy of the genetic interconnectivity matrix by computing the magnitude of the vector difference between the unit normals of the theoretical (<span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-103.gif"/></noscript></span></span>) and inferred (<span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-104.gif"/></noscript></span></span>) hyperplanes, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-105.gif"/></noscript></span></span>. When the angle between the unit normals is small, <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-106.gif"/></noscript></span></span> gives the angle between them. <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-107.gif"/></noscript></span></span> implies that the inferred hyperplane is orthogonal to the theoretical one, and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-108.gif"/></noscript></span></span> is the maximum value possible, implying that the two normals are in exactly opposite directions and the assignment of ON/OFF state has been reversed. We also computed the null distribution of <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-109.gif"/></noscript></span></span> which results from choosing the inferred unit normal at random uniformly on the unit <em>G</em>-sphere. The discrepancies in the other parameters were computed as <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-110.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-111.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-112.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-113.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-114.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-115.gif"/></noscript></span></span> are inferred parameter values.</p><p id="p-32">In the first set of simulations, we simulated networks of size ranging from two to fifty genes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A–E</a>) with <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-116.gif"/></noscript></span></span> trajectories sampled at <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-117.gif"/></noscript></span></span> time points. The switching hyperplane is inferred with high accuracy (<span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-118.gif"/></noscript></span></span> or 6° angle to the theoretical normal) for the vast majority of two-gene random gene circuits, showing that FIGR is capable of recovering the true values of the parameters if a sufficient amount of data are available. As the size of the gene circuit, and consequently the number of free parameters, increases, the accuracy declines. However the inference is still fairly accurate for 20-gene networks since 75% of the inferred hyperplanes have <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-119.gif"/></noscript></span></span> or less than a 30° angle to the theoretical hyperplane. Inferring the signs of the genetic interconnectivity coefficients <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-120.gif"/></noscript></span></span>, that is, whether a regulator activates or represses a target, is an important goal in gene circuit analysis. FIGR achieves 90% accuracy in inferring the signs of <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-121.gif"/></noscript></span></span> for 20-gene networks (Fig. S2). For 50-gene networks, the accuracy of quantitative inference is quite low and most inferred hyperplanes have <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-122.gif"/></noscript></span></span> or larger than a 30° angle to the theoretical hyperplane. This is not entirely surprising since a 50-gene network has 2,650 free parameters, while the model is being inferred from only 2,100 observations. Although 50-gene networks are inferred with poor accuracy, the <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-123.gif"/></noscript></span></span> distribution is significantly better than the null distribution, suggesting that the inferred parameters still contain information about GRN structure. In fact, the signs of 82% interconnectivity coefficients are inferred correctly (Fig. S1) suggesting that FIGR still works reasonably well at a qualitative level in a highly underdetermined problem.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference of genetic interconnectivity coefficients from synthetic data. The distribution of the discrepancy between inferred and theoretical switching hyperplanes, , in inferring 100 random gene circuits with FIGR or SA is shown as boxplots.  and  are unit normals to the inferred and theoretical hyperplanes. Note that each boxplot is constructed from  discrepancies since each random parameter set contains G switching hyperplanes, where G is the number of genes. The box lines are the first quartile, median, and the third quartile. The whiskers extend to the most extreme values lying within 1.5 times the interquartile range, and any datapoints outside the whiskers are shown as crosses. The blue violin plot (&#x201C;RAND&#x201D;) shows the null discrepancy distribution that would be obtained, , if  were not inferred but instead picked randomly from a uniform distribution over the surface of the G-sphere. The width of the violin plot is proportional to , and blue and red vertical lines indicate quartiles and median respectively. A&#x2013;E. Number of trajectories  and number of timepoints . The number of genes G was varied between 2 and 50 for FIGR and 2 and 20 for SA, since SA was impractical for . F,G. Number of trajectories  and number of genes . The number of timepoints  was varied between 41 (panel F) and 11 (panel G). H&#x2013;J. Number of genes  and number of timepoints . The number of trajectories N was varied between 10 and 50." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1090008564" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Inference of genetic interconnectivity coefficients from synthetic data. The distribution of the discrepancy between inferred and theoretical switching hyperplanes, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, in inferring 100 random gene circuits with FIGR or SA is shown as boxplots. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-125&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-126&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are unit normals to the inferred and theoretical hyperplanes. Note that each boxplot is constructed from &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-127&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; discrepancies since each random parameter set contains &lt;em&gt;G&lt;/em&gt; switching hyperplanes, where &lt;em&gt;G&lt;/em&gt; is the number of genes. The box lines are the first quartile, median, and the third quartile. The whiskers extend to the most extreme values lying within 1.5 times the interquartile range, and any datapoints outside the whiskers are shown as crosses. The blue violin plot (&#x201C;RAND&#x201D;) shows the null discrepancy distribution that would be obtained, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-128&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, if &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-129&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; were not inferred but instead picked randomly from a uniform distribution over the surface of the &lt;em&gt;G&lt;/em&gt;-sphere. The width of the violin plot is proportional to &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-130&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and blue and red vertical lines indicate quartiles and median respectively. A&#x2013;E. Number of trajectories &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-131&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and number of timepoints &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-132&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The number of genes &lt;em&gt;G&lt;/em&gt; was varied between 2 and 50 for FIGR and 2 and 20 for SA, since SA was impractical for &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-133&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. F,G. Number of trajectories &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-134&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and number of genes &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-135&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The number of timepoints &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-136&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; was varied between 41 (panel F) and 11 (panel G). H&#x2013;J. Number of genes &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and number of timepoints &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The number of trajectories N was varied between 10 and 50.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4183/F2.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/4183/F2.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495062" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Inference of genetic interconnectivity coefficients from synthetic data. The distribution of the discrepancy between inferred and theoretical switching hyperplanes, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-124.gif"/></noscript></span></span>, in inferring 100 random gene circuits with FIGR or SA is shown as boxplots. <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-125.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-126.gif"/></noscript></span></span> are unit normals to the inferred and theoretical hyperplanes. Note that each boxplot is constructed from <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-127.gif"/></noscript></span></span> discrepancies since each random parameter set contains <em>G</em> switching hyperplanes, where <em>G</em> is the number of genes. The box lines are the first quartile, median, and the third quartile. The whiskers extend to the most extreme values lying within 1.5 times the interquartile range, and any datapoints outside the whiskers are shown as crosses. The blue violin plot (“RAND”) shows the null discrepancy distribution that would be obtained, <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-128.gif"/></noscript></span></span>, if <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-129.gif"/></noscript></span></span> were not inferred but instead picked randomly from a uniform distribution over the surface of the <em>G</em>-sphere. The width of the violin plot is proportional to <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-130.gif"/></noscript></span></span>, and blue and red vertical lines indicate quartiles and median respectively. A–E. Number of trajectories <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-131.gif"/></noscript></span></span> and number of timepoints <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-132.gif"/></noscript></span></span>. The number of genes <em>G</em> was varied between 2 and 50 for FIGR and 2 and 20 for SA, since SA was impractical for <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-133.gif"/></noscript></span></span>. F,G. Number of trajectories <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-134.gif"/></noscript></span></span> and number of genes <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-135.gif"/></noscript></span></span>. The number of timepoints <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-136.gif"/></noscript></span></span> was varied between 41 (panel F) and 11 (panel G). H–J. Number of genes <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-137.gif"/></noscript></span></span> and number of timepoints <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F2/embed/mml-math-138.gif"/></noscript></span></span>. The number of trajectories N was varied between 10 and 50.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Hyperplanes inferred with SA also show a trend of decreasing accuracy with increasing network size, suggesting that declining accuracy is a result of progressive reduction in the determinacy of the problem rather than an intrinsic inability of FIGR to infer larger networks. In fact, in nearly all cases, SA’s inferences were more variable than FIGR, and were slightly less accurate than FIGR on the 20-gene problem. The relatively lower accuracy of SA is not a result of poor fitting since the RMS of most of the random gene circuit fits is less than 0.04 (∼4% error; Fig. S2).</p><p id="p-35">In the second set of simulations, we simulated random 20-gene networks, but varied the number of sampled time points <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-139.gif"/></noscript></span></span> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2F,G</a>) or the number of trajectories <em>N</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2H–J</a>). Increasing or decreasing the number of time points to 41 or 11 respectively had a minimal effect on the quality of the inference of the switching hyperplanes by FIGR or SA (compare to <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). This suggested that 11 time points were sufficient to reliably estimate the genetic interconnection coefficients. In contrast, decreasing the number of trajectories progressively reduced accuracy and both FIGR and SA inferences were indistinguishable from the null distribution when only 10 trajectories were sampled. Once again, this is not surprising since the 460 free parameters of a 20-gene network are being inferred from only 210 observations. These results imply that while increasing temporal resolution beyond a certain point provides diminishing gains in accuracy, the number of trajectories or conditions the trajectories are sampled from is a crucial parameter influencing the quality of the inference.</p><p id="p-36">The inference of <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-140.gif"/></noscript></span></span> (Fig. S3) was quite accurate and behaved like the inference of the switching hyperplanes. Increasing gene network size or reducing the amount of data tended to reduce accuracy, although the effects were less pronounced than what was observed while inferring switching hyperplanes. The kinetic parameters were also inferred accurately by FIGR (Fig. S3). The accuracy of both <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-141.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-142.gif"/></noscript></span></span> increased with increasing number of time points but did not depend on the number of genes or trajectories. This can be understood as a consequence of the separation of regulatory and kinetic parameters in Glass gene circuits—the inference of the kinetic parameters occurs independently for each gene and depends only on the sampling frequency.</p><p id="p-37">Although the inference is quite accurate, the discrepancies are not zero for most gene circuits and can be fairly large for a small number of gene circuits, even in the 2-gene case. This results from constraints imposed by the intrinsic dynamics of gene circuits and finite sample size. For instance, trajectories move away from the switching hyperplane for autoactivating genes. In this case, the initial conditions act as support vectors for the inferred hyperplane, which then strongly depends upon the random sample of starting points. Another situation that arises is that of a hyperplane that divides the bounding hypercube into ON and OFF regions in a lopsided manner. Since initial points are sampled uniformly, this results in too few sampled points in the vicinity of the hyperplane and poor inference. Given that FIGR was at least as accurate as SA, these failure modes are not specific to the inference methodology but likely represent fundamental limitations of inferring differential equations models. These considerations are also valid when inferring GRNs from empirical data. Such insights and their implications for parameter identifiability will be described elsewhere. Notwithstanding these constraints, our analysis demonstrates that FIGR is capable of inferring parameters quantitatively when provided with a sufficient number of data points and qualitatively (signs of <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-143.gif"/></noscript></span></span>) even when the problem is underdetermined as in the 50-gene case.</p></div><div id="sec-16" class="subsection"><h3>Inference of the gap gene network from empirical data</h3><p id="p-38">We tested the efficacy of FIGR on empirical data by inferring a gene circuit for the gap gene network acting during <em>Drosophila</em> segmentation. The gap gene network is one of the best characterized developmental GRNs at both the experimental and theoretical levels, and therefore serves as a benchmark for gene circuit inference. We provide a very brief summary of the segmentation system here; a more complete description may be found in reviews by Akam (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akam 1987</a>) or Jaeger (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jaeger 2011</a>). The segmentation proteins pattern the anteroposterior axis during the first three hours of embryogenesis. During this period, the embryo is a syncitium, so that nuclei lack cell membranes and undergo 13 mitotic divisions, termed cleavages. After the tenth cleavage cycle, the majority of the nuclei migrate to the periphery of the embryo and are arranged in a monolayer, forming a syncitial blastoderm. Cellular membranes form by invagination of the plasma membrane in the latter half of cleavage cycle 14, at the end of which the embryo undergoes gastrulation.</p><p id="p-39">Near the end of cleavage cycle 14, the segmentation genes are expressed in spatially resolved patterns that specify the position of each cell to an accuracy of one cell diameter. Segmentation gene expression is initiated by shallow protein gradients formed by the translation of localized mRNAs, such as <a href="https://identifiers.org/bioentitylink/FB:FBgn0000166?doi=10.1534/g3.119.400603"><em>bicoid</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0000166?doi=10.1534/g3.119.400603"><em>bcd</em></a>) and <a href="https://identifiers.org/bioentitylink/FB:FBgn0000251?doi=10.1534/g3.119.400603"><em>caudal</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0000251?doi=10.1534/g3.119.400603"><em>cad</em></a>), deposited in the oocyte by the mother. These maternal protein gradients regulate the gap genes, which commence mRNA expression in cleavage cycle 10–12 and are expressed in broad domains ∼ 20 nuclei wide during cycle 14. The gap genes in turn regulate the pair-rule and segment-polarity genes that provide the molecular prepattern for the subsequent segmentation of the embryo. All the maternal and gap proteins are known to act as transcription factors that regulate each others’ expression in a complex GRN that has been modeled extensively (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Reinitz <em>et al.</em> 1995</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>,<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">b</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gursky <em>et al.</em> 2008</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Vakulenko <em>et al.</em> 2009</a>; <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>,<a id="xref-ref-49-9" class="xref-bibr" href="#ref-49">b</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gursky <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ashyraliyev <em>et al.</em> 2009</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wotton <em>et al.</em> 2015</a>; <a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Verd <em>et al.</em> 2018</a>).</p><div id="sec-17" class="subsection"><h4>Gene circuit equations for a spatially extended system:</h4><p id="p-40">The gap gene circuit describes the time evolution of the concentrations of the gap proteins in a one-dimensional row of <em>N</em> nuclei lying along the anteroposterior axis of the embryo (<a id="xref-ref-49-10" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>) during cleavage cycle 14. The lack of cell membranes in the syncitium implies that proteins can diffuse between nuclei (cells) and <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> is modified to incorporate discretized Fickian diffusion and anteroposterior position, so that<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-144.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>Here <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-145.gif"/></noscript></span></span> is the expression level of protein <em>g</em> in nucleus <em>n</em> at time <em>t</em>, <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-146.gif"/></noscript></span></span> is the diffusion constant for protein <em>g</em>, and <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-147.gif"/></noscript></span></span> is the sigmoid regulation-expression function (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>). Zero-flux boundary conditions are used at the ends of the modeled region, while initial conditions are given by the empirical data from cycle 13.</p></div><div id="sec-18" class="subsection"><h4>FIGR for a spatially extended system:</h4><p id="p-41">The determination of ON/OFF state and inference of regulatory parameters <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-148.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-149.gif"/></noscript></span></span> is carried out as described above in Sections and. However, inferring gene circuits in spatially extended systems requires two modifications to the algorithm. First, the diffusion constants <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-150.gif"/></noscript></span></span> must be inferred in addition to <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-151.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-152.gif"/></noscript></span></span>. Second, for gene circuits with diffusion, the correspondence between velocity <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-153.gif"/></noscript></span></span> and ON/OFF state <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-154.gif"/></noscript></span></span> (<a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Equation 9</a>) will be violated for a few nuclei lying adjacent to nuclei in which the gene is ON. Such nuclei would have positive velocity despite the gene being OFF because of diffusion of the protein from adjacent “ON nuclei”. Although this effect is limited to a few nuclei, it would nevertheless result in slightly inaccurate parameter estimates. For this reason, once the regulatory and kinetic parameters have been inferred, they are refined by local optimization.</p></div><div id="sec-19" class="subsection"><h4>Inference of kinetic parameters including <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-155.gif"/></noscript></span></span>:</h4><p id="p-42">We exploit so-called kink solutions to the gene circuit equations (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Vakulenko <em>et al.</em> 2009</a>) to estimate the kinetic parameters. Let gene <em>g</em> be ON in an anteroposterior domain <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-156.gif"/></noscript></span></span> so that there is net diffusion out of the domain into surrounding OFF nuclei. Then the balance of synthesis, degradation, and diffusion will establish a stable gradient,<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-157.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>outside the domain. Here, <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-158.gif"/></noscript></span></span> is the characteristic length scale of the gradient at steady state.</p><p id="p-43">In order to infer the kinetic parameters, we first determine the time class in which a gene is expressed at the highest level, since the spatial gradient would best approximate steady state at that time point. Next, we identify gene-expression domains as local maxima in the spatial pattern in that time class. For genes having multiple expression domains, such as <a href="https://identifiers.org/bioentitylink/FB:FBgn0001180?doi=10.1534/g3.119.400603"><em>hb</em></a> and <a href="https://identifiers.org/bioentitylink/FB:FBgn0001150?doi=10.1534/g3.119.400603"><em>gt</em></a>, the domain with the highest expression is chosen. The gene is expressed at half maximum at the domain borders (<a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">Equation 15</a>). Therefore, border positions (<em>l</em> or <em>r</em>) are determined as nuclei where the expression is half of the domain peak. <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-159.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-160.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-161.gif"/></noscript></span></span> are then determined by fitting <a id="xref-disp-formula-16-2" class="xref-disp-formula" href="#disp-formula-16">Equation 15</a> to the observed expression data for <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-162.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-163.gif"/></noscript></span></span> using MATLAB’s lsqnonlin function. This is implemented as the <strong>“kink”</strong> method of FIGR.</p></div></div><div id="sec-20" class="subsection"><h3>Refinement of parameters</h3><p id="p-44">The parameter values estimated using classification and fitting to kink solutions serve as a starting point for an unconstrained local search using the Nelder-Mead algorithm implemented in MATLAB’s fminsearch function. The cost function is <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-164.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-165.gif"/></noscript></span></span> are data and <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-166.gif"/></noscript></span></span> are solutions of <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">Equation 14</a>.</p><div id="sec-21" class="subsection"><h4>Gap gene circuit inference and comparison with data and SA:</h4><p id="p-45">We inferred a gene circuit for the gap genes <a href="https://identifiers.org/bioentitylink/FB:FBgn0001180?doi=10.1534/g3.119.400603"><em>hunchback</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0001180?doi=10.1534/g3.119.400603"><em>hb</em></a>), <a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Krüppel</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Kr</em></a>), <a href="https://identifiers.org/bioentitylink/FB:FBgn0001150?doi=10.1534/g3.119.400603"><em>giant</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0001150?doi=10.1534/g3.119.400603"><em>gt</em></a>), and <a href="https://identifiers.org/bioentitylink/FB:FBgn0001320?doi=10.1534/g3.119.400603"><em>knirps</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0001320?doi=10.1534/g3.119.400603"><em>kni</em></a>) using both FIGR (Section FIGR for a spatially extended system) and SA (Section Inference of gap gene circuits). The model includes the upstream regulators <a href="https://identifiers.org/bioentitylink/FB:FBgn0001180?doi=10.1534/g3.119.400603"><em>Bicoid</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Bcd</em></a>), <a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Caudal</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Cad</em></a>), and <a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Tailless</em></a> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Tll</em></a>), so that <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-167.gif"/></noscript></span></span>. Our implementation matched previous gap gene models (<a id="xref-ref-49-11" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>) so that the upstream proteins regulated the trunk gap genes but not vice versa. The time-dependent concentrations of Bcd, Cad, and Tll were determined by linear interpolation of the expression data. The gene circuit models the protein expression of these genes between 35–92% egg length during cleavage cycle 14 (Section Inference of gap gene circuits). The dataset is comprised of immunofluorescence protein concentration measurements at the resolution of individual nuclei at eight timepoints during cleavage cycle 14 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Classification-based inference of the Drosophila gap gene network. Horizontal axes indicate anteroposterior position. Vertical axes represent time elapsed since the thirteenth nuclear division in the downward direction. A. Integrated spatiotemporal gap gene expression data , where , used to infer the gene circuit. Data at the first time point are from cycle 13 and serve as initial conditions for the model. The rest of the time points correspond to eight time classes in cycle 14 separated by 6.25 min. Hot (cold) colors indicate high (low) expression levels. B. Classification of ON/OFF state. Green (red) colors indicate ON (OFF) states  determined from velocities . C. Prediction of ON/OFF state  using parameter values inferred with FIGR. D. Gene circuit output using parameters obtained by FIGR and then refined using local search (RMS = 13.29). E. Gene circuit output using parameters inferred by SA (RMS = 11.03). F. Comparison of the spatial patterns of gap gene expression (blue), output of gene circuit inferred by SA (red; SA), and output of gene circuit inferred by FIGR (green) at  min (time class T7). Slightly shallow posterior border of the anterior Hb domain in FIGR output is indicated with &#x25C1;. Slight ectopic bump in the anterior border of the posterior Hb domain in SA output is indicated with &#x25B7;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1090008564" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Classification-based inference of the &lt;em&gt;Drosophila&lt;/em&gt; gap gene network. Horizontal axes indicate anteroposterior position. Vertical axes represent time elapsed since the thirteenth nuclear division in the downward direction. A. Integrated spatiotemporal gap gene expression data &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-168&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-169&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, used to infer the gene circuit. Data at the first time point are from cycle 13 and serve as initial conditions for the model. The rest of the time points correspond to eight time classes in cycle 14 separated by 6.25 min. Hot (cold) colors indicate high (low) expression levels. B. Classification of ON/OFF state. Green (red) colors indicate ON (OFF) states &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-170&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; determined from velocities &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-171&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. C. Prediction of ON/OFF state &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-172&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; using parameter values inferred with FIGR. D. Gene circuit output using parameters obtained by FIGR and then refined using local search (RMS = 13.29). E. Gene circuit output using parameters inferred by SA (RMS = 11.03). F. Comparison of the spatial patterns of gap gene expression (blue), output of gene circuit inferred by SA (red; SA), and output of gene circuit inferred by FIGR (green) at &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-173&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; min (time class T7). Slightly shallow posterior border of the anterior Hb domain in FIGR output is indicated with &#x25C1;. Slight ectopic bump in the anterior border of the posterior Hb domain in SA output is indicated with &#x25B7;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4183/F3.medium.gif" width="389" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/4183/F3.medium.gif" width="389" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">Classification-based inference of the <em>Drosophila</em> gap gene network. Horizontal axes indicate anteroposterior position. Vertical axes represent time elapsed since the thirteenth nuclear division in the downward direction. A. Integrated spatiotemporal gap gene expression data <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-168.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-169.gif"/></noscript></span></span>, used to infer the gene circuit. Data at the first time point are from cycle 13 and serve as initial conditions for the model. The rest of the time points correspond to eight time classes in cycle 14 separated by 6.25 min. Hot (cold) colors indicate high (low) expression levels. B. Classification of ON/OFF state. Green (red) colors indicate ON (OFF) states <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-170.gif"/></noscript></span></span> determined from velocities <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-171.gif"/></noscript></span></span>. C. Prediction of ON/OFF state <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-172.gif"/></noscript></span></span> using parameter values inferred with FIGR. D. Gene circuit output using parameters obtained by FIGR and then refined using local search (RMS = 13.29). E. Gene circuit output using parameters inferred by SA (RMS = 11.03). F. Comparison of the spatial patterns of gap gene expression (blue), output of gene circuit inferred by SA (red; SA), and output of gene circuit inferred by FIGR (green) at <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F3/embed/mml-math-173.gif"/></noscript></span></span> min (time class T7). Slightly shallow posterior border of the anterior Hb domain in FIGR output is indicated with ◁. Slight ectopic bump in the anterior border of the posterior Hb domain in SA output is indicated with ▷.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">In the first step of FIGR, the ON/OFF state of each gene is determined by estimating the velocity <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-174.gif"/></noscript></span></span> (Section Determining ON/OFF state). <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> shows that the assignment of ON/OFF states by FIGR correctly recapitulates the extent and dynamics of the gene expression domains. In the next step, the regulatory parameters <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-175.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-176.gif"/></noscript></span></span> were inferred by supervised classification (Section Determining regulatory parameters). Predictions of the ON/OFF state from the inferred parameter values, <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-177.gif"/></noscript></span></span> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>), are in good agreement with the empirical ON/OFF classification and the gene expression domains, suggesting correct inference of the regulatory parameters. In the third step, the kinetic parameters were estimated by fitting the kink equations to gap gene domain borders (Section Inference of kinetic parameters including <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-178.gif"/></noscript></span></span>.). In the presence of diffusion, the correspondence between velocity <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-179.gif"/></noscript></span></span> and the ON/OFF state of the gene will be violated for nuclei abutting gene expression domains due to diffusion of protein from ON regions into OFF regions. The inferred parameters likely deviate slightly from optimal values and were therefore refined using local search (Section Refinement of parameters.) in the fourth step of the algorithm. During refinement, the solutions were computed using gene circuit equations that represent synthesis with the sigmoid regulation-expression function (<a id="xref-disp-formula-15-2" class="xref-disp-formula" href="#disp-formula-15">Equation 14</a>) instead of the Heaviside function. Therefore the final model obtained did not rely on the assumption of binary switch-like regulation, which was utilized only during binary classification while obtaining the initial guesses of parameter values.</p><p id="p-48">Gene circuit simulation with refined parameters produces output that agrees very well with the data and the output of gene circuits inferred using SA (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D,E</a>), and has an RMS (<span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-180.gif"/></noscript></span></span>) of 13.29. Careful comparison of the data and the outputs of the FIGR- and SA-derived gene circuits shows that there are only minor discrepancies between model output and data (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3F</a> and Fig. S4), some shared between FIGR and SA output and a few unique to each method. Both methods underpredict Hb, Kr, and Gt expression in time class T1 (<span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-181.gif"/></noscript></span></span> min), a discrepancy that is resolved at later time points (Fig. S4). In FIGR output the posterior border of the abdominal Gt domain retracts a bit earlier, in time class T2, than in data, another discrepancy that is resolved at later time points (Fig. S4). SA output has an ectopic bump in the anterior border of the posterior Hb domain while FIGR produces a slightly shallower posterior border of the anterior Hb domain (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3F</a>). Despite these minor discrepancies, FIGR output agrees well with data overall, having the same border and peak positions, and reproduces the dynamic anterior shifts in gap gene domains during cleavage cycle 14 (Fig. S4).</p><p id="p-49">The inferred gap gene network (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) is in good agreement with experimental evidence (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Jaeger 2011</a>), the network inferred by SA, and previous analyses (<a id="xref-ref-49-12" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>). One of the organizing principles underlying the resolution of gap gene domains, termed “alternating cushions”, is that genes with non-overlapping domains, Kr and Gt or Hb and Kni, strongly repress each other (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kraut and Levine 1991</a>). The genetic interconnection coefficients of Kr/Gt and Hb/Kni inferred by FIGR have negative values of high magnitude, which is consistent with the alternating cushions mechanism. Another key feature reproduced by FIGR output is the movement of posterior gap gene domains to the anterior during cycle 14 (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Jaeger <em>et al.</em> 2004b</a>). These shifts have been understood to occur in a cell-autonomous manner (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Jaeger <em>et al.</em> 2004b</a>) due to asymmetric repression between gap genes in reverse order of adjacent gap gene domains, Kr ⊢ Kni ⊢ Gt (<a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>). The FIGR gene circuit encapsulates this mechanism since 1) genes expressed in adjacent domains repress each other weakly or not at all and 2) Gt represses Kni but not vice versa and Kni represses Kr but Kr does not repress Kni.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of gap gene parameters inferred by FIGR and SA. The values of , , , , and  are listed one gene per row. The T matrix is shown in the first seven columns, one column per regulator. Green (red) indicates activation (repression). Since  and  can only be determined up to a constant factor, they have been normalized () to allow comparison between FIGR and SA. A. FIGR after refinement. B. SA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1090008564" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of gap gene parameters inferred by FIGR and SA. The values of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-182&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-183&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-184&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-185&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-186&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are listed one gene per row. The T matrix is shown in the first seven columns, one column per regulator. Green (red) indicates activation (repression). Since &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-187&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-188&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; can only be determined up to a constant factor, they have been normalized (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-189&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) to allow comparison between FIGR and SA. A. FIGR after refinement. B. SA.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4183/F4.medium.gif" width="440" height="384"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/12/4183/F4.medium.gif" width="440" height="384"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4183/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495089" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-50" class="first-child">Comparison of gap gene parameters inferred by FIGR and SA. The values of <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-182.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-183.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-184.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-185.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-186.gif"/></noscript></span></span> are listed one gene per row. The T matrix is shown in the first seven columns, one column per regulator. Green (red) indicates activation (repression). Since <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-187.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-188.gif"/></noscript></span></span> can only be determined up to a constant factor, they have been normalized (<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/F4/embed/mml-math-189.gif"/></noscript></span></span>) to allow comparison between FIGR and SA. A. FIGR after refinement. B. SA.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">The activation of the gap genes inferred by FIGR is also consistent with experimental evidence. The gap genes are activated by Bcd and Cad (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Driever and Nüsslein-Volhard 1989</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rivera-Pomar <em>et al.</em> 1995</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hoch <em>et al.</em> 1991</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eldon and Pirrotta 1991</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Schulz and Tautz 1995</a>). Hb autoactivates strongly, in agreement with experimental evidence (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Margolis <em>et al.</em> 1995</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Simpson-Brose <em>et al.</em> 1994</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hülskamp <em>et al.</em> 1994</a>). Although there isn’t clear evidence that the other gap genes autoactivate (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Jaeger 2011</a>), the FIGR GRN, like previously inferred ones (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>; <a id="xref-ref-49-13" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>; <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>), features weak autoactivation by Kr and Gt. Two main differences are observed between the FIGR- and SA-inferred gene circuits. First, whereas Hb is strongly activated by Bcd in the SA gene circuit, it is only weakly activated in the FIGR gene circuit. However in other studies using both wildtype (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>) and wildtype/<a href="https://identifiers.org/bioentitylink/FB:FBgn0001325?doi=10.1534/g3.119.400603"><em>Kr</em></a><sup>−</sup> (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>) data, SA infers Bcd → Hb interactions ranging from repression to strong activation. A second difference is that whereas Tll activates Hb in the SA circuit, it represses Hb in the FIGR gene circuit. Experimental evidence about whether Tll activates or represses Hb in the posterior is inconclusive (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Jaeger 2011</a>). In summary, the agreement between FIGR output and data are the result of genetic mechanisms that have been substantiated by previous experimental and modeling analyses.</p><p id="p-52">With compiled MATLAB code, FIGR with refinement takes 48 sec compared to 29,067 sec taken by SA (∼600-fold speed up) to achieve the same RMS. In summary, FIGR infers the gap gene network accurately but at a fraction of the computational expense.</p></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-53">Gene circuits (<a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Reinitz and Sharp 1995</a>; <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>; <a id="xref-ref-49-14" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>) provide many unique advantages for inferring and modeling developmental GRNs. The differential equations are biologically realistic in representing gene regulation as a nonlinear switch-like function of TF concentrations. Gene circuits not only infer the topology of the network but the directionality (causality), sign (activation/repression), and strength of regulatory interconnections. Most importantly, gene circuits are not limited to inference but are capable of accurately simulating and predicting gene expression patterns. Finally, the use of differential equations allows gene circuits to compute transient solutions, an important factor in simulating development since fate determination can occur before equilibrium is reached (<a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Simcox and Sang 1983</a>). Despite the promise held by gene circuits, their application, as of other data-driven differential equation models, has been limited to smaller networks so far. Analysis of larger networks is limited to correlative approaches (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Margolin <em>et al.</em> 2006</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Segal <em>et al.</em> 2003</a>) that neither infer causality nor simulate or predict the time evolution of GRN state.</p><p id="p-54">A major challenge in broader application of gene circuits is the high computational expense of inferring the free parameters from time series data. Currently, the approach for inferring parameter values (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chu <em>et al.</em> 1999</a>; <a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Reinitz and Sharp 1995</a>; <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abdol <em>et al.</em> 2017</a>) is to solve (“integrate”) the ODEs to obtain trajectories, compare with experimental trajectories, and refine parameters using global optimization techniques such as SA. This procedure is slow and expensive because it requires performing multidimensional optimization on a complicated cost function <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-190.gif"/></noscript></span></span> with many local minima and each function evaluation involves solving a system of ODEs. Moreover, the computational complexity grows rapidly (<span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-191.gif"/></noscript></span></span>) so that global optimization approaches for gene circuits scale poorly with G.</p><p id="p-55">In contrast, FIGR directly attempts to fit the differential equations, which describe how the velocities <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-192.gif"/></noscript></span></span> depend upon the concentrations <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-193.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-194.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-195.gif"/></noscript></span></span> are determined using binary classification (support vector machines or logistic regression). Both of these algorithms reduce to quadratic programming, and thence to convex optimization. Subsequently, <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-196.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-197.gif"/></noscript></span></span> can be determined by linear regression against velocities or non-linear regression against concentrations using the piece-wise analytical solutions of the ODEs, which are even simpler optimization problems. Each inference can be completed in a fraction of a second on a consumer-grade computer, even with interpreted MATLAB code. We found that in the gap gene circuit, the values of the regulatory parameters <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-198.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-199.gif"/></noscript></span></span> thus inferred were consistent with those obtained by SA, and ODEs solved with the inferred parameters produced gap gene domains correctly positioned in space and time. Furthermore, the parameter values, which were inferred under the assumption of a Heaviside regulation-expression function, served as a good starting point for equations with a sigmoid regulation-expression function. Gene circuits with a good RMS could be obtained with an “off-the-shelf” Nelder-Mead simplex method built into MATLAB. This refinement takes under a minute in serial compared to the eight hours taken by SA. Finally, logistic regression has a computational complexity of <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-200.gif"/></noscript></span></span>, where <em>G</em> is the number of features (genes), which implies that FIGR has a complexity of <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-201.gif"/></noscript></span></span> when inferring <em>G</em> genes. Consequently, FIGR scales with problem size much better than global nonlinear optimization techniques.</p><p id="p-56">The computational efficiency of FIGR does not come at the expense of accuracy. In testing the recovery of known parameters from synthetic data (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, S1, and S3), we found that FIGR and SA had comparable accuracy for smaller gene circuits, while FIGR had slightly higher accuracy than SA for 20-gene networks. We speculate that the lower accuracy of SA results from “sloppiness” (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gutenkunst <em>et al.</em> 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ashyraliyev <em>et al.</em> 2008</a>; <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>)—insensitivity of model output to certain genetic interconnectivity coefficients. If a certain parameter gives similar solutions over some interval, then SA can infer any value in the interval rather than the true value. This insensitivity can result from the compensatory and redundant roles many parameters play in the model (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ashyraliyev <em>et al.</em> 2008</a>; <a id="xref-ref-39-9" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>). For example, a high expression level can be achieved by having higher activating genetic interconnection coefficients, by having higher synthesis rates, or by having lower degradation rates. The higher accuracy of FIGR could perhaps be attributed to the separation of the regulatory and kinetic parameters, which limits the opportunities available for redundant parameters to produce similar solutions. It is not possible to decide which method was more accurate in inferring the gap gene circuit because the “true” values are not known. However, there are hints that FIGR suffers less from the compensation problem in the gap gene case as well. Using SA, it is common to get fits where a gap gene is either activated by Bcd and has low basal synthesis (<span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-202.gif"/></noscript></span></span>) or a gap gene is repressed by Bcd but has high basal synthesis (<span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-203.gif"/></noscript></span></span>). The former is biologically accurate, while the latter is not. For this reason, in the SA optimizations performed here and in previous work (<a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Jaeger <em>et al.</em> 2004a</a>; <a id="xref-ref-49-15" class="xref-bibr" href="#ref-49">Manu <em>et al.</em> 2009b</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gursky <em>et al.</em> 2011</a>; <a id="xref-ref-39-10" class="xref-bibr" href="#ref-39">Kozlov <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Perkins <em>et al.</em> 2006</a>), <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-204.gif"/></noscript></span></span> was constrained to be negative. No such constraint was used in FIGR, and it infers low basal synthesis for all the gap genes autonomously.</p><p id="p-57">In representing synthesis as a binary ON/OFF choice, Glass equations (<a id="xref-disp-formula-4-7" class="xref-disp-formula" href="#disp-formula-4">Equation 3</a>) are similar to Boolean or logical models, which have been applied to a broad range of developmental GRNs (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Theiffry <em>et al.</em> 1993</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sánchez and Thieffry 2001</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Davidson <em>et al.</em> 2002b</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Thieffry and Sánchez 2003</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Collombet <em>et al.</em> 2017</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bonzanni <em>et al.</em> 2013</a>). Given this similarity between Boolean models and Glass equations, FIGR should be readily applicable to a large class of GRN modeling problems. Moreover, Glass equations relax the assumption made in logical models—that genes are expressed at a small number of discrete levels—to allow expression at any arbitrary level. This makes Glass models more general than Boolean models and capable of simulating transient dynamics during development. The accurate inference of the gap gene network using FIGR (Section Gap gene circuit inference and comparison with data and SA) demonstrates that the assumption of discrete ON/OFF synthesis may also be relaxed—binary classification provided estimates of parameters which were close enough to the optimal values that further refinement by a local search resulted in good fits.</p><p id="p-58">The separation of the inference procedure into regulatory and kinetic components bears a superficial similarity to the multi-step approach of Perkins <em>et al.</em> (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Perkins <em>et al.</em> 2006</a>) for inferring gap gene circuits. In the first step of their approach, the diffusion and decay parameters are inferred by assuming that the domains where each gap gene is synthesized have a quadrilateral shape in space-time. In the second step, the regulatory parameters are inferred by fitting a sigmoid synthesis function to the quadrilateral production domains inferred in the first step, followed by a third step of refining all parameters by solving the coupled differential equations. In contrast to the method of Perkins <em>et al.</em> our method does not assume any particular shape for the production domains and does not require manual analysis to define the domains. The production domains result naturally and automatically from our classification procedure (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>). The lack of a strong assumption about the shape of expression domains means that FIGR should be applicable to genes having any production domain shape, and with 2D or 3D spatial data. The second point of divergence is that we infer the regulatory parameters using binary classification instead of fitting the sigmoid synthesis function to the production domains. As a consequence, FIGR is faster than the method of Perkins <em>et al.</em></p><p id="p-59">Although in our tests FIGR was shown to be at least as effective as, and much faster than, SA, it does have a few limitations. First, in order to estimate the velocity and determine ON/OFF state reliably, FIGR requires that the data be sampled sufficiently frequently in time. Roughly speaking, during each time period in which a gene is in a particular state (ON or OFF), its product concentration would have to be sampled at least three times in order to ascertain the velocity and state. FIGR therefore would not be suitable for datasets that have been sampled sparsely in time. Methods reliant on solving the ODEs will, in contrast, attempt to fit the trajectories to a sparsely sampled dataset, even if the actual inference achieved is poor (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2J</a>). FIGR was successful in inferring the gap gene circuit even though the gap gene dataset (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3A</a>) has only 8 time points because most nuclei don’t change state and a few nuclei at gene expression borders change state once, allowing sufficient sampling for a reliable detection of the state.</p><p id="p-60">In spatially extended systems, the sign of the velocity, which can be estimated empirically, and the unobserved gene ON/OFF state are not strictly equivalent due to diffusion of the gene product out of the region of synthesis. In the gap gene system, we found that the sign of the velocity nevertheless corresponded quite well to gene state (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a>); well enough, in fact, to provide a starting point for a local search that yielded good fits (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3D</a>). The success of this approximation is likely due to the fact that gap gene patterning is largely cell-autonomous and diffusion plays the limited role of averaging gene expression in space (<a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Manu <em>et al.</em> 2009a</a>; <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Jaeger <em>et al.</em> 2004b</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gursky <em>et al.</em> 2004</a>). In spatial systems where patterning is largely driven by diffusion (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kondo and Miura 2010</a>), FIGR’s decomposition of regulatory and kinetic parameters might be less successful, and require a greater reliance on the refinement of parameters by solving the differential equations. Successfully extending FIGR to spatially extended systems besides the gap gene network may require the recognition of situations where patterning is driven by diffusion. The simplest empirical means of assessing the importance of diffusion is to compare mRNA and protein expression patterns. A large overlap between the two would imply that diffusion is less important and that parameter estimates obtained through binary classification would be close to optimal. If, on the other hand, protein expression domains extend much farther than the mRNA expression domains, it would imply that the initial estimates are not close to optimal values and more aggressive refinement is likely needed. If mRNA expression data are not available or difficult to obtain, it might be possible to estimate parameters under slow or fast diffusion scenarios by constraining the ON domains to be larger or smaller respectively during binary classification. This could potentially be achieved by adjusting the velocity and expression thresholds <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-205.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4183/embed/mml-math-206.gif"/></noscript></span></span> prior to binary classification in FIGR.</p><p id="p-61">Besides the problem of computational efficiency, the broader application of gene circuits, and indeed all nonlinear differential equation models, is limited by a lack of understanding of parameter identifiability. Most commonly, <em>a posteriori</em> confidence intervals for the parameter estimates are computed (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Ashyraliyev <em>et al.</em> 2008</a>). Such calculations are based on the strong assumption that the solution is linear in the parameters and that the measurement errors are normally distributed. <em>a posteriori</em> parameter identifiability analysis also does not provide any hints to improve experimental design for achieving better identifiability in future studies.</p><p id="p-62">Although we have not directly addressed the problem here, we anticipate that conceiving of and visualizing gene circuit inference as a classification problem will lead to insights into parameter identifiability. For example, it is evident from state space plots (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1F,I</a>) that sampling gene expression trajectories closer to the true switching hyperplane of the gene will lead to less “wiggle room” for the inferred hyperplane and result in more accurate parameter inference. This implies that datasets that measure gene expression near steady state, for example in differentiated cell types, are unlikely to lead to accurate parameter inference irrespective of the number of data points sampled or the precision of the experiment. Sampling transient trajectories densely when genes are turning ON or OFF is the best strategy for accurate parameter inference. Another less obvious implication is that it is easier to infer the regulation of negatively autoregulated genes than positively autoregulated ones. Trajectories move toward or away from the switching hyperplane for negatively or positively autoregulated genes respectively, making it more likely that sampled data points will lie near the hyperplane in the former. This analysis will be reported elsewhere.</p><p id="p-63">In summary, we have exploited features of the mathematical structure of gene circuits to break a difficult optimization problem into a series of two, much simpler, optimization problems. We have demonstrated that FIGR is effective on synthetic as well as experimental data from a biologically realistic GRN. We have validated the inferred gap gene model by comparing its parameters against models inferred with SA as well as comparing its output against experimental data. The improvement in computational efficiency and scalability should allow the inference of much larger GRNs than was possible previously.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">This work was supported by the National Science Foundation [1615916 to M.]; the University of North Dakota Office of the Vice President for Research [to M. and Y.L.L.]; and North Dakota EPSCoR [to D.A.F.].</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-65">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9249245">https://doi.org/10.25387/g3.9249245</a>.</p></li><li class="fn" id="fn-3"><p id="p-66"><em>Communicating editor: J. Ma</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-08-05"><span class="received-label">Received </span>August 5, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-16"><span class="accepted-label">Accepted </span>October 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Fehr <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdol</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Cicin-Sain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Kaandorp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Crombach</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Scatter search applied to the inference of a development gene network. Computation <strong>5</strong>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/computation5020022</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.2" data-doi="10.1038/msb.2012.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akam</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The molecular basis for metameric pattern in the <em>Drosophila</em> embryo.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DAkam%26rft.auinit1%253DM.%26rft.volume%253D101%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DThe%2Bmolecular%2Bbasis%2Bfor%2Bmetameric%2Bpattern%2Bin%2Bthe%2BDrosophila%2Bembryo%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2012.72%26rft_id%253Dinfo%253Apmid%252F2896587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo3OiIxMDEvMS8xIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.3" data-doi="10.1186/1752-0509-2-83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashyraliyev</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jaeger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Blom</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Parameter estimation and determinability analysis applied to drosophila gap gene circuits.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1752-0509-2-83</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DAshyraliyev%26rft.auinit1%253DM.%26rft.volume%253D2%26rft.spage%253D83%26rft.epage%253D83%26rft.atitle%253DParameter%2Bestimation%2Band%2Bdeterminability%2Banalysis%2Bapplied%2Bto%2BDrosophila%2Bgap%2Bgene%2Bcircuits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-2-83%26rft_id%253Dinfo%253Apmid%252F18817540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-2-83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18817540&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.4" data-doi="10.1371/journal.pcbi.1000548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashyraliyev</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Siggens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Janssens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Blom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Akam</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene circuit analysis of the terminal gap gene huckebein.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000548</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1000548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DAshyraliyev%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000548%26rft.epage%253De1000548%26rft.atitle%253DGene%2Bcircuit%2Banalysis%2Bof%2Bthe%2Bterminal%2Bgap%2Bgene%2Bhuckebein.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000548%26rft_id%253Dinfo%253Apmid%252F19876378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19876378&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.5" data-doi="10.1016/j.cell.2011.10.047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balaskas</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ribeiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Panovska</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Dessaud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Sasai</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene regulatory logic for reading the sonic hedgehog signaling gradient in the vertebrate neural tube.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2011.10.047</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBalaskas%26rft.auinit1%253DN.%26rft.volume%253D148%26rft.issue%253D1-2%26rft.spage%253D273%26rft.epage%253D284%26rft.atitle%253DGene%2Bregulatory%2Blogic%2Bfor%2Breading%2Bthe%2BSonic%2BHedgehog%2Bsignaling%2Bgradient%2Bin%2Bthe%2Bvertebrate%2Bneural%2Btube.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.10.047%26rft_id%253Dinfo%253Apmid%252F22265416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.10.047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22265416&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299540700030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.6" data-doi="10.1093/bioinformatics/btt243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonzanni</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Garg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Feenstra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schütte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kinston</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hard-wired heterogeneity in blood stem cells revealed using a dynamic regulatory network model.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">i80</span>–<span class="cit-lpage">i88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt243</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt243%26rft_id%253Dinfo%253Apmid%252F23813012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23813012&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.7" data-doi="10.1006/jcph.1998.6134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Deng</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Parallel simulated annealing by mixing of states.</span> <abbr class="cit-jnl-abbrev">J. Comput. Phys.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">646</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jcph.1998.6134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BPhys.%26rft.volume%253D148%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjcph.1998.6134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jcph.1998.6134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.8" data-doi="10.1073/pnas.1610622114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collombet</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">van Oevelen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Sardina Ortega</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Abou-Jaoudé</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Di Stefano</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Logical modeling of lymphoid and myeloid cell specification and transdifferentiation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">5792</span>–<span class="cit-lpage">5799</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1610622114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1610622114%26rft_id%253Dinfo%253Apmid%252F28584084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzIzLzU3OTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.9" data-doi="10.1038/msb.2010.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotterell</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sharpe</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An atlas of gene regulatory networks reveals multiple three-gene mechanisms for interpreting morphogen gradients.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/msb.2010.74</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DCotterell%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D425%26rft.epage%253D425%26rft.atitle%253DAn%2Batlas%2Bof%2Bgene%2Bregulatory%2Bnetworks%2Breveals%2Bmultiple%2Bthree-gene%2Bmechanisms%2Bfor%2Binterpreting%2Bmorphogen%2Bgradients%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2010.74%26rft_id%253Dinfo%253Apmid%252F21045819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjYvMS80MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.10" data-doi="10.1371/journal.pcbi.1002589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crombach</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Wotton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Cicin-Sain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ashyraliyev</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jaeger</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient reverse-engineering of a developmental gene regulatory network.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002589</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DCrombach%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002589%26rft.epage%253De1002589%26rft.atitle%253DEfficient%2Breverse-engineering%2Bof%2Ba%2Bdevelopmental%2Bgene%2Bregulatory%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002589%26rft_id%253Dinfo%253Apmid%252F22807664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22807664&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.11" data-doi="10.1073/pnas.0806007105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">E. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Levine</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Properties of developmental gene regulatory networks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">20063</span>–<span class="cit-lpage">20066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0806007105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806007105%26rft_id%253Dinfo%253Apmid%252F19104053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUxLzIwMDYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.12" data-doi="10.1126/science.1069883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Rast</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Oliveri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ransick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Calestani</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">a</span> <span class="cit-article-title">A genomic regulatory network for development.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">295</span>: <span class="cit-fpage">1669</span>–<span class="cit-lpage">1678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1069883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDavidson%26rft.auinit1%253DE.%2BH.%26rft.volume%253D295%26rft.issue%253D5560%26rft.spage%253D1669%26rft.epage%253D1678%26rft.atitle%253DA%2BGenomic%2BRegulatory%2BNetwork%2Bfor%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069883%26rft_id%253Dinfo%253Apmid%252F11872831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTUvNTU2MC8xNjY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.13" data-doi="10.1006/dbio.2002.0635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Rast</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Oliveri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ransick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Calestani</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">b</span> <span class="cit-article-title">A provisional regulatory gene network for specification of endomesoderm in the sea urchin embryo.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">246</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.2002.0635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DDavidson%26rft.auinit1%253DE.%2BH.%26rft.volume%253D246%26rft.issue%253D1%26rft.spage%253D162%26rft.epage%253D190%26rft.atitle%253DA%2Bprovisional%2Bregulatory%2Bgene%2Bnetwork%2Bfor%2Bspecification%2Bof%2Bendomesoderm%2Bin%2Bthe%2Bsea%2Burchin%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.2002.0635%26rft_id%253Dinfo%253Apmid%252F12027441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.2002.0635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027441&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176065900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.14" data-doi="10.1038/337138a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Driever</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Nüsslein-Volhard</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The Bicoid protein is a positive regulator of <em>hunchback</em> transcription in the early <em>Drosophila</em> embryo.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/337138a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDriever%26rft.auinit1%253DW.%26rft.volume%253D337%26rft.issue%253D6203%26rft.spage%253D138%26rft.epage%253D143%26rft.atitle%253DThe%2Bbicoid%2Bprotein%2Bis%2Ba%2Bpositive%2Bregulator%2Bof%2Bhunchback%2Btranscription%2Bin%2Bthe%2Bearly%2BDrosophila%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F337138a0%26rft_id%253Dinfo%253Apmid%252F2911348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/337138a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2911348&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.15" data-doi="10.1016/S0167-2789(00)00130-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of continuous-time switching networks.</span> <abbr class="cit-jnl-abbrev">Physica D</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0167-2789(00)00130-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysica%2BD%26rft.volume%253D146%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0167-2789%252800%252900130-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0167-2789(00)00130-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eldon</span>,  <span class="cit-name-given-names">E. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Pirrotta</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Interactions of the <em>Drosophila</em> gap gene <em>giant</em> with maternal and zygotic pattern-forming genes.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DEldon%26rft.auinit1%253DE.%2BD.%26rft.volume%253D111%26rft.issue%253D2%26rft.spage%253D367%26rft.epage%253D378%26rft.atitle%253DInteractions%2Bof%2Bthe%2BDrosophila%2Bgap%2Bgene%2Bgiant%2Bwith%2Bmaternal%2Band%2Bzygotic%2Bpattern-forming%2Bgenes%26rft_id%253Dinfo%253Apmid%252F1716553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMTEvMi8zNjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.17" data-doi="10.1016/0022-5193(73)90208-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Kauffman</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">The logical analysis of continuous, non-linear biochemical control networks.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-5193(73)90208-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DGlass%26rft.auinit1%253DL.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D129%26rft.atitle%253DThe%2Blogical%2Banalysis%2Bof%2Bcontinuous%252C%2Bnon-linear%2Bbiochemical%2Bcontrol%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-5193%252873%252990208-7%26rft_id%253Dinfo%253Apmid%252F4741704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-5193(73)90208-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4741704&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973P354200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.18" data-doi="10.1007/BF02547797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pasternack</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Stable oscillations in mathematical models of biological control systems.</span> <abbr class="cit-jnl-abbrev">J. Math. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02547797</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMath.%2BBiol.%26rft.volume%253D6%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02547797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02547797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978FQ92900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.19" data-doi="10.1016/j.physd.2004.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gursky</span>,  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jaeger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. N.</span> <span class="cit-name-surname">Kozlov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Samsonova</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Pattern formation and nuclear divisions are uncoupled in <em>Drosophila</em> segmentation: comparison of spatially discrete and continuous models.</span> <abbr class="cit-jnl-abbrev">Physica D</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.physd.2004.07.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysica%2BD%26rft.volume%253D197%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.physd.2004.07.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.physd.2004.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.20" data-doi="10.1134/S0006350908020085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gursky</span>,  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. N.</span> <span class="cit-name-surname">Kozlov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Samsonov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A model with asymptotically stable dynamics for the network of <em>Drosophila</em> gap genes.</span> <abbr class="cit-jnl-abbrev">Biofizika</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1134/S0006350908020085</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiofizika%2B%2528Moskva%2529%26rft.stitle%253DBiofizika%2B%2528Moskva%2529%26rft.volume%253D53%26rft.issue%253D2%26rft.spage%253D235%26rft.epage%253D249%26rft.atitle%253D%255BA%2Bmodel%2Bwith%2Basymptotically%2Bstable%2Bdynamics%2Bfor%2Bthe%2Bnetwork%2Bof%2BDrosophila%2Bgap%2Bgenes%255D%26rft_id%253Dinfo%253Adoi%252F10.1134%252FS0006350908020085%26rft_id%253Dinfo%253Apmid%252F18543766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1134/S0006350908020085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18543766&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.21" data-doi="10.1186/1752-0509-5-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gursky</span>,  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Panok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Myasnikova</span></span>. </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Manu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Samsonova</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mechanisms of gap gene expression canalization in the drosophila blastoderm.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1752-0509-5-118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DGursky%26rft.auinit1%253DV.%2BV.%26rft.volume%253D5%26rft.spage%253D118%26rft.epage%253D118%26rft.atitle%253DMechanisms%2Bof%2Bgap%2Bgene%2Bexpression%2Bcanalization%2Bin%2Bthe%2BDrosophila%2Bblastoderm.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-5-118%26rft_id%253Dinfo%253Apmid%252F21794172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-5-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21794172&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.22" data-doi="10.1371/journal.pcbi.0030189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>,  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Waterfall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. P.</span> <span class="cit-name-surname">Casey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Myers</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Universally sloppy parameter sensitivities in systems biology models.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1871</span>–<span class="cit-lpage">1878</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.0030189</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253D1871%26rft.epage%253D1878%26rft.atitle%253DUniversally%2Bsloppy%2Bparameter%2Bsensitivities%2Bin%2Bsystems%2Bbiology%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0030189%26rft_id%253Dinfo%253Apmid%252F17922568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0030189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17922568&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251113500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.23" data-doi="10.1073/pnas.1610609114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamey</span>,  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nestorowa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Kinston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Kent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. K.</span> <span class="cit-name-surname">Wilson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Reconstructing blood stem cell regulatory network models from single-cell molecular profiles.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">5822</span>–<span class="cit-lpage">5829</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1610609114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1610609114%26rft_id%253Dinfo%253Apmid%252F28584094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzIzLzU4MjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4183.24" data-doi="10.1007/978-0-387-84858-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>,  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Tibshirani</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Friedman</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>The elements of statistical learning: data mining</em>, <em>inference</em>, <em>and prediction</em></span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-84858-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Belements%2Bof%2Bstatistical%2Blearning%253A%2Bdata%2Bmining%252C%2Binference%252C%2Band%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-84858-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-84858-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.25" data-doi="10.1371/journal.pone.0026797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hengenius</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gribskov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Rundell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Fowlkes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Umulis</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of gap gene regulation in a 3d organism-scale model of the drosophila melanogaster embryo.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e26797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0026797</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHengenius%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De26797%26rft.epage%253De26797%26rft.atitle%253DAnalysis%2Bof%2Bgap%2Bgene%2Bregulation%2Bin%2Ba%2B3D%2Borganism-scale%2Bmodel%2Bof%2Bthe%2BDrosophila%2Bmelanogaster%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0026797%26rft_id%253Dinfo%253Apmid%252F22110594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0026797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22110594&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.26" data-doi="10.1002/j.1460-2075.1991.tb07763.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoch</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Seifert</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jäckle</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Gene expression mediated by cis-acting sequences of the <em>Krüppel</em> gene in response to the <em>Drosophila</em> morphogens Bicoid and Hunchback.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2267</span>–<span class="cit-lpage">2278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/j.1460-2075.1991.tb07763.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DHoch%26rft.auinit1%253DM%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D2267%26rft.epage%253D2278%26rft.atitle%253DGene%2Bexpression%2Bmediated%2Bby%2Bcis-acting%2Bsequences%2Bof%2Bthe%2BKruppel%2Bgene%2Bin%2Bresponse%2Bto%2Bthe%2BDrosophila%2Bmorphogens%2Bbicoid%2Band%2Bhunchback%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fj.1460-2075.1991.tb07763.x%26rft_id%253Dinfo%253Apmid%252F2065664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/j.1460-2075.1991.tb07763.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2065664&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FW47300038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.27" data-doi="10.1186/1752-0509-6-66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Tyson</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A simple theoretical framework for understanding heterogeneous differentiation of cd4+ t cells.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1752-0509-6-66</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DHong%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DA%2Bsimple%2Btheoretical%2Bframework%2Bfor%2Bunderstanding%2Bheterogeneous%2Bdifferentiation%2Bof%2BCD4%252B%2BT%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-6-66%26rft_id%253Dinfo%253Apmid%252F22697466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-6-66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22697466&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.28" data-doi="10.1016/j.ydbio.2007.02.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">May</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Enver</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bifurcation dynamics in lineage-commitment in bipotent progenitor cells.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">713</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2007.02.036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DHuang%26rft.auinit1%253DS.%26rft.volume%253D305%26rft.issue%253D2%26rft.spage%253D695%26rft.epage%253D713%26rft.atitle%253DBifurcation%2Bdynamics%2Bin%2Blineage-commitment%2Bin%2Bbipotent%2Bprogenitor%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2007.02.036%26rft_id%253Dinfo%253Apmid%252F17412320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2007.02.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17412320&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246461000027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hülskamp</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Lukowitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Beermann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Glaser</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Tautz</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Differential regulation of target genes by different alleles of the segmentation gene <em>hunchback</em> in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D138%26rft.spage%253D125%26rft_id%253Dinfo%253Apmid%252F8001780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8001780&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PC81800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.30" data-doi="10.1007/s00018-010-0536-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaeger</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The gap gene network.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00018-010-0536-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DJaeger%26rft.auinit1%253DJ.%26rft.volume%253D68%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D274%26rft.atitle%253DThe%2Bgap%2Bgene%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-010-0536-y%26rft_id%253Dinfo%253Apmid%252F20927566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-010-0536-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927566&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.31" data-doi="10.1534/genetics.104.027334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaeger</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Blagov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kosman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. N.</span> <span class="cit-name-surname">Kozlov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span><span class="cit-comment">a</span> <span class="cit-article-title">Dynamical analysis of regulatory interactions in the gap gene system of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1721</span>–<span class="cit-lpage">1737</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.027334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJaeger%26rft.auinit1%253DJ.%26rft.volume%253D167%26rft.issue%253D4%26rft.spage%253D1721%26rft.epage%253D1737%26rft.atitle%253DDynamical%2BAnalysis%2Bof%2BRegulatory%2BInteractions%2Bin%2Bthe%2BGap%2BGene%2BSystem%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.027334%26rft_id%253Dinfo%253Apmid%252F15342511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny80LzE3MjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.32" data-doi="10.1016/j.mod.2006.11.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaeger</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Sharp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Known maternal gradients are not sufficient for the establishment of gap domains in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mech. Dev.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">108</span>–<span class="cit-lpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mod.2006.11.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bdevelopment%26rft.stitle%253DMech%2BDev%26rft.aulast%253DJaeger%26rft.auinit1%253DJ.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D108%26rft.epage%253D128%26rft.atitle%253DKnown%2Bmaternal%2Bgradients%2Bare%2Bnot%2Bsufficient%2Bfor%2Bthe%2Bestablishment%2Bof%2Bgap%2Bdomains%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mod.2006.11.001%26rft_id%253Dinfo%253Apmid%252F17196796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mod.2006.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17196796&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244539700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.33" data-doi="10.1038/nature02678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaeger</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Surkova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Blagov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Janssens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kosman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span><span class="cit-comment">b</span> <span class="cit-article-title">Dynamic control of positional information in the early <em>Drosophila</em> embryo.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">368</span>–<span class="cit-lpage">371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02678</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJaeger%26rft.auinit1%253DJ.%26rft.volume%253D430%26rft.issue%253D6997%26rft.spage%253D368%26rft.epage%253D371%26rft.atitle%253DDynamic%2Bcontrol%2Bof%2Bpositional%2Binformation%2Bin%2Bthe%2Bearly%2BDrosophila%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02678%26rft_id%253Dinfo%253Apmid%252F15254541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15254541&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222631200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.34" data-doi="10.1007/s00427-005-0484-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janssens</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kosman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Vanario-Alonso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jaeger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Samsonova</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A high-throughput method for quantifying gene expression data from early <em>Drosophila</em> embryos.</span> <abbr class="cit-jnl-abbrev">Dev. Genes Evol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00427-005-0484-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%2Bgenes%2Band%2Bevolution%26rft.stitle%253DDev%2BGenes%2BEvol%26rft.aulast%253DJanssens%26rft.auinit1%253DH.%26rft.volume%253D215%26rft.issue%253D7%26rft.spage%253D374%26rft.epage%253D381%26rft.atitle%253DA%2Bhigh-throughput%2Bmethod%2Bfor%2Bquantifying%2Bgene%2Bexpression%2Bdata%2Bfrom%2Bearly%2BDrosophila%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00427-005-0484-y%26rft_id%253Dinfo%253Apmid%252F15834586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00427-005-0484-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15834586&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230515900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.35" data-doi="10.1186/1752-0509-4-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jostins</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jaeger</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reverse engineering a gene network using an asynchronous parallel evolution strategy.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1752-0509-4-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DJostins%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.spage%253D17%26rft.epage%253D17%26rft.atitle%253DReverse%2Bengineering%2Ba%2Bgene%2Bnetwork%2Busing%2Ban%2Basynchronous%2Bparallel%2Bevolution%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-4-17%26rft_id%253Dinfo%253Apmid%252F20196855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-4-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20196855&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.36" data-doi="10.1126/science.220.4598.671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Gelatt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Vecchi</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Optimization by simulated annealing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">220</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">680</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.220.4598.671</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKirkpatrick%26rft.auinit1%253DS.%26rft.volume%253D220%26rft.issue%253D4598%26rft.spage%253D671%26rft.epage%253D680%26rft.atitle%253DOptimization%2Bby%2BSimulated%2BAnnealing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.220.4598.671%26rft_id%253Dinfo%253Apmid%252F17813860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyMjAvNDU5OC82NzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.37" data-doi="10.1126/science.1179047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Miura</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reaction-diffusion model as a framework for understanding biological pattern formation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">1616</span>–<span class="cit-lpage">1620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1179047</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKondo%26rft.auinit1%253DS.%26rft.volume%253D329%26rft.issue%253D5999%26rft.spage%253D1616%26rft.epage%253D1620%26rft.atitle%253DReaction-Diffusion%2BModel%2Bas%2Ba%2BFramework%2Bfor%2BUnderstanding%2BBiological%2BPattern%2BFormation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1179047%26rft_id%253Dinfo%253Apmid%252F20929839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjkvNTk5OS8xNjE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4183.38" data-doi="10.1007/978-3-642-03275-2_13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozlov</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Samsonov</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deep - differential evolution entirely parallel method for gene regulatory networks</span>, pp. <span class="cit-fpage">126</span>–<span class="cit-lpage">132</span> in <span class="cit-source">Parallel Computing Technologies</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Malyshkin</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-publ-name">Springer Berlin Heidelberg</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-03275-2_13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParallel%2BComputing%2BTechnologies%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-03275-2_13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-03275-2_13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.39" data-doi="10.1371/journal.pcbi.1002635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozlov</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Surkova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Myasnikova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Samsonova</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Modeling of gap gene expression in drosophila kruppel mutants.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DKozlov%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002635%26rft.epage%253De1002635%26rft.atitle%253DModeling%2Bof%2Bgap%2Bgene%2Bexpression%2Bin%2BDrosophila%2BKruppel%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002635%26rft_id%253Dinfo%253Apmid%252F22927803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22927803&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraut</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Levine</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Mutually repressive interactions between the gap genes <em>giant</em> and <em>Krüppel</em> define middle body regions of the <em>Drosophila</em> embryo.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DKraut%26rft.auinit1%253DR.%26rft.volume%253D111%26rft.issue%253D2%26rft.spage%253D611%26rft.epage%253D621%26rft.atitle%253DMutually%2Brepressive%2Binteractions%2Bbetween%2Bthe%2Bgap%2Bgenes%2Bgiant%2Band%2BKruppel%2Bdefine%2Bmiddle%2Bbody%2Bregions%2Bof%2Bthe%2BDrosophila%2Bembryo%26rft_id%253Dinfo%253Apmid%252F1893878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMTEvMi82MTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.41" data-doi="10.1126/science.1240831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kueh</span>,  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Champhekhar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Nutt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Elowitz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Rothenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Positive feedback between pu.1 and the cell cycle controls myeloid differentiation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">670</span>–<span class="cit-lpage">673</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1240831</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1240831%26rft_id%253Dinfo%253Apmid%252F23868921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0Ni82NzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.42"><div class="cit-metadata unstructured">Lam, J., and J.-M. Delosme, 1988a An efficient simulated annealing schedule: Derivation. Technical Report 8816, Yale Electrical Engineering Department, New Haven, CT.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.43"><div class="cit-metadata unstructured">Lam, J., and J.-M. Delosme, 1988b An efficient simulated annealing schedule: Implementation and evaluation. Technical Report 8817, Yale Electrical Engineering Department, New Haven, CT.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.44" data-doi="10.1016/j.cell.2006.06.052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laslo</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Spooner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Warmflash</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Lancki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-J.</span> <span class="cit-name-surname">Lee</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Multilineage transcriptional priming and determination of alternate hematopoietic cell fates.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2006.06.052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLaslo%26rft.auinit1%253DP.%26rft.volume%253D126%26rft.issue%253D4%26rft.spage%253D755%26rft.epage%253D766%26rft.atitle%253DMultilineage%2Btranscriptional%2Bpriming%2Band%2Bdetermination%2Bof%2Balternate%2Bhematopoietic%2Bcell%2Bfates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.06.052%26rft_id%253Dinfo%253Apmid%252F16923394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.06.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16923394&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240276700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.45" data-doi="10.1073/pnas.0408031102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Davidson</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene regulatory networks for development.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4936</span>–<span class="cit-lpage">4942</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0408031102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0408031102%26rft_id%253Dinfo%253Apmid%252F15788537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE0LzQ5MzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.46" data-doi="10.1371/journal.pcbi.1003165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quantifying cell fate decisions for differentiation and reprogramming of a human stem cell network: Landscape and biological paths.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003165</span> <span class="cit-comment">EP –.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1003165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1003165%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003165%26rft_id%253Dinfo%253Apmid%252F23935477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23935477&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.47" data-doi="10.1016/j.parco.2016.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>,  <span class="cit-name-given-names">Z.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Parallel simulated annealing using an adaptive resampling interval.</span> <abbr class="cit-jnl-abbrev">Parallel Comput.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.parco.2016.02.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParallel%2BComput.%26rft.volume%253D53%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.parco.2016.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.parco.2016.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.48" data-doi="10.1371/journal.pcbi.1000303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manu</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Surkova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Spirov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. V.</span> <span class="cit-name-surname">Gursky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Janssens</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Canalization of gene expression and domain shifts in the <em>Drosophila</em> blastoderm by dynamical attractors.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1000303</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DManu%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000303%26rft.epage%253De1000303%26rft.atitle%253DCanalization%2Bof%2Bgene%2Bexpression%2Band%2Bdomain%2Bshifts%2Bin%2Bthe%2BDrosophila%2Bblastoderm%2Bby%2Bdynamical%2Battractors.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000303%26rft_id%253Dinfo%253Apmid%252F19282965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19282965&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.49" data-doi="10.1371/journal.pbio.1000049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manu</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Surkova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Spirov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. V.</span> <span class="cit-name-surname">Gursky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Janssens</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Canalization of gene expression in the <em>Drosophila</em> blastoderm by gap gene cross regulation.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1000049</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000049</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DManu%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1000049%26rft.epage%253De1000049%26rft.atitle%253DCanalization%2Bof%2Bgene%2Bexpression%2Bin%2Bthe%2BDrosophila%2Bblastoderm%2Bby%2Bgap%2Bgene%2Bcross%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000049%26rft_id%253Dinfo%253Apmid%252F19750121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19750121&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.50" data-doi="10.1186/1471-2105-7-S1-S7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margolin</span>,  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Nemenman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Basso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wiggins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stolovitzky</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Aracne: an algorithm for the reconstruction of gene regulatory networks in a mammalian cellular context.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">S7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-7-S1-S7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D7%26rft.spage%253DS7%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-S1-S7%26rft_id%253Dinfo%253Apmid%252F16723010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-S1-S7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16723010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margolis</span>,  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Borowsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Steingrimsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Shim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Lengyel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Posterior stripe expression of <em>hunchback</em> is driven from two promoters by a common enhancer element.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">3067</span>–<span class="cit-lpage">3077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DMargolis%26rft.auinit1%253DJ.%26rft.volume%253D121%26rft.issue%253D9%26rft.spage%253D3067%26rft.epage%253D3077%26rft.atitle%253DPosterior%2Bstripe%2Bexpression%2Bof%2Bhunchback%2Bis%2Bdriven%2Bfrom%2Btwo%2Bpromoters%2Bby%2Ba%2Bcommon%2Benhancer%2Belement%26rft_id%253Dinfo%253Apmid%252F7555732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTIxLzkvMzA2NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.52" data-doi="10.1016/j.stem.2013.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Soneji</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Tipping</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Teles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">McGowan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dynamic analysis of gene expression and genome-wide transcription factor binding during lineage specification of multipotent progenitors.</span> <abbr class="cit-jnl-abbrev">Cell Stem Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.stem.2013.09.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BStem%2BCell%26rft.volume%253D13%26rft.spage%253D754%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2013.09.003%26rft_id%253Dinfo%253Apmid%252F24120743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stem.2013.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24120743&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.53" data-doi="10.1016/0167-2789(96)00086-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mestl</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lemay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Glass</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chaos in high-dimensional neural and gene networks.</span> <abbr class="cit-jnl-abbrev">Physica D</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0167-2789(96)00086-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysica%2BD%26rft.volume%253D98%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0167-2789%252896%252900086-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0167-2789(96)00086-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VQ29700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.54" data-doi="10.1016/j.cell.2011.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novershtern</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Subramanian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. N.</span> <span class="cit-name-surname">Lawton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Mak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. N.</span> <span class="cit-name-surname">Haining</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Densely interconnected transcriptional circuits control cell states in human hematopoiesis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">296</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2011.01.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNovershtern%26rft.auinit1%253DN.%26rft.volume%253D144%26rft.issue%253D2%26rft.spage%253D296%26rft.epage%253D309%26rft.atitle%253DDensely%2Binterconnected%2Btranscriptional%2Bcircuits%2Bcontrol%2Bcell%2Bstates%2Bin%2Bhuman%2Bhematopoiesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.01.004%26rft_id%253Dinfo%253Apmid%252F21241896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21241896&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286459900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.55" data-doi="10.1529/biophysj.107.120600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palani</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Sarkar</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Positive receptor feedback during lineage commitment can generate ultrasensitivity to ligand and confer robustness to a bistable switch.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1575</span>–<span class="cit-lpage">1589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1529/biophysj.107.120600</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D1575%26rft.epage%253D1589%26rft.atitle%253DPositive%2Breceptor%2Bfeedback%2Bduring%2Blineage%2Bcommitment%2Bcan%2Bgenerate%2Bultrasensitivity%2Bto%2Bligand%2Band%2Bconfer%2Brobustness%2Bto%2Ba%2Bbistable%2Bswitch.%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.107.120600%26rft_id%253Dinfo%253Apmid%252F18469073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1529/biophysj.107.120600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18469073&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257889400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.56" data-doi="10.1371/journal.pcbi.0020051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>,  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jaeger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Glass</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reverse engineering the gap gene network of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e51</span> <a href="http://compbiol.plosjournals.org/perlserv/?request=get-document&amp;#x0026;doi=10.1371/journal.pcbi.0020051">http://compbiol.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pcbi.0020051</a>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.0020051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DPerkins%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253De51%26rft.epage%253De51%26rft.atitle%253DReverse%2Bengineering%2Bthe%2Bgap%2Bgene%2Bnetwork%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020051%26rft_id%253Dinfo%253Apmid%252F16710449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16710449&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.57" data-doi="10.1371/journal.pcbi.1006904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pietak</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bischof</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">LaPalme</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Morokuma</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Levin</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Neural control of body-plan axis in regenerating planaria.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-elocation-id">e1006904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1006904</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D15%26rft.spage%253De1006904%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.58" data-doi="10.1002/jez.1402710106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinitz</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Mjolsness</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Sharp</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Model for Cooperative control of positional information in <em>Drosophila</em> by <em>bicoid</em> and maternal <em>hunchback</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.1402710106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bexperimental%2Bzoology%26rft.stitle%253DJ%2BExp%2BZool%26rft.aulast%253DReinitz%26rft.auinit1%253DJ.%26rft.volume%253D271%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D56%26rft.atitle%253DModel%2Bfor%2Bcooperative%2Bcontrol%2Bof%2Bpositional%2Binformation%2Bin%2BDrosophila%2Bby%2Bbicoid%2Band%2Bmaternal%2Bhunchback.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1402710106%26rft_id%253Dinfo%253Apmid%252F7852948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1402710106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7852948&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QH42800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.59" data-doi="10.1016/0925-4773(94)00310-J"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinitz</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Sharp</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Mechanism of <em>eve</em> stripe formation.</span> <abbr class="cit-jnl-abbrev">Mech. Dev.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0925-4773(94)00310-J</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bdevelopment%26rft.stitle%253DMech%2BDev%26rft.aulast%253DReinitz%26rft.auinit1%253DJ.%26rft.volume%253D49%26rft.issue%253D1-2%26rft.spage%253D133%26rft.epage%253D158%26rft.atitle%253DMechanism%2Bof%2Beve%2Bstripe%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0925-4773%252894%252900310-J%26rft_id%253Dinfo%253Apmid%252F7748785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0925-4773(94)00310-J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7748785&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF46700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.60" data-doi="10.1038/376253a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rivera-Pomar</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Perrimon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Taubert</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jäckle</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Activation of posterior gap gene expression in the <em>Drosophila</em> blastoderm.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">376</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/376253a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRivera-Pomar%26rft.auinit1%253DR.%26rft.volume%253D376%26rft.issue%253D6537%26rft.spage%253D253%26rft.epage%253D256%26rft.atitle%253DActivation%2Bof%2Bposterior%2Bgap%2Bgene%2Bexpression%2Bin%2Bthe%2BDrosophila%2Bblastoderm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F376253a0%26rft_id%253Dinfo%253Apmid%252F7617036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/376253a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7617036&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.61" data-doi="10.1006/jtbi.2001.2335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Thieffry</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A logical analysis of the <em>Drosophila</em> gap-gene system.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jtbi.2001.2335</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DS%25C3%25A1nchez%26rft.auinit1%253DL.%26rft.volume%253D211%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D141%26rft.atitle%253DA%2Blogical%2Banalysis%2Bof%2Bthe%2BDrosophila%2Bgap-gene%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.2001.2335%26rft_id%253Dinfo%253Apmid%252F11419955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.2001.2335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11419955&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169848200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Tautz</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Zygotic <em>caudal</em> regulation by <em>hunchback</em> and its role in abdominal segment formation of the <em>Drosophila</em> embryo.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1023</span>–<span class="cit-lpage">1028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DSchulz%26rft.auinit1%253DC%26rft.volume%253D121%26rft.issue%253D4%26rft.spage%253D1023%26rft.epage%253D1028%26rft.atitle%253DZygotic%2Bcaudal%2Bregulation%2Bby%2Bhunchback%2Band%2Bits%2Brole%2Bin%2Babdominal%2Bsegment%2Bformation%2Bof%2Bthe%2BDrosophila%2Bembryo%26rft_id%253Dinfo%253Apmid%252F7743918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTIxLzQvMTAyMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.63" data-doi="10.1038/nm0203-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segal</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shapira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Regev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Pe’er</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Botstein</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Module networks: identifying regulatory modules and their condition-specific regulators from gene expression data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegal%26rft.auinit1%253DE.%26rft.volume%253D34%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D176%26rft.atitle%253DModule%2Bnetworks%253A%2Bidentifying%2Bregulatory%2Bmodules%2Band%2Btheir%2Bcondition-specific%2Bregulators%2Bfrom%2Bgene%2Bexpression%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm0203-166%26rft_id%253Dinfo%253Apmid%252F12740579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm0203-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12740579&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183200500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.64" data-doi="10.1016/0012-1606(83)90078-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simcox</span>,  <span class="cit-name-given-names">A. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Sang</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">When does determination occur in <em>Drosophila</em> embryos?</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0012-1606(83)90078-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSimcox%26rft.auinit1%253DA.%2BA.%26rft.volume%253D97%26rft.issue%253D1%26rft.spage%253D212%26rft.epage%253D221%26rft.atitle%253DWhen%2Bdoes%2Bdetermination%2Boccur%2Bin%2BDrosophila%2Bembryos%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252883%252990078-7%26rft_id%253Dinfo%253Apmid%252F6404675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(83)90078-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6404675&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QS13100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.65" data-doi="10.1016/S0092-8674(94)90622-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson-Brose</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Treisman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Desplan</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Synergy between the Hunchback and Bicoid morphogens is required for anterior patterning in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">865</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(94)90622-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSimpson-Brose%26rft.auinit1%253DM.%26rft.volume%253D78%26rft.issue%253D5%26rft.spage%253D855%26rft.epage%253D865%26rft.atitle%253DSynergy%2Bbetween%2Bthe%2Bhunchback%2Band%2Bbicoid%2Bmorphogens%2Bis%2Brequired%2Bfor%2Banterior%2Bpatterning%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252894%252990622-X%26rft_id%253Dinfo%253Apmid%252F8087852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(94)90622-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8087852&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PG29600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.12.4183.66" data-doi="10.1016/j.ydbio.2007.10.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Surkova</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kosman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kozlov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Myasnikova</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Characterization of the <em>Drosophila</em> segment determination morphome.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">844</span>–<span class="cit-lpage">862</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2007.10.037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSurkova%26rft.auinit1%253DS.%26rft.volume%253D313%26rft.issue%253D2%26rft.spage%253D844%26rft.epage%253D862%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BDrosophila%2Bsegment%2Bdetermination%2Bmorphome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2007.10.037%26rft_id%253Dinfo%253Apmid%252F18067886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2007.10.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18067886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252541900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theiffry</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Colet</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thomas</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Formalization of regulatory networks: A logical method and its automatization.</span> <abbr class="cit-jnl-abbrev">Mathematical Modelling and Scientific Computing</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">144</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2BModelling%2Band%2BScientific%2BComputing%26rft.volume%253D2%26rft.spage%253D144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.68" data-doi="10.1016/S0959-437X(03)00079-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thieffry</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sánchez</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dynamical modelling of pattern formation during embryonic development.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0959-437X(03)00079-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DThieffry%26rft.auinit1%253DD.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D326%26rft.epage%253D330%26rft.atitle%253DDynamical%2Bmodelling%2Bof%2Bpattern%2Bformation%2Bduring%2Bembryonic%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252803%252900079-0%26rft_id%253Dinfo%253Apmid%252F12888004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(03)00079-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12888004&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.69" data-doi="10.1038/nrc3081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyson</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. T.</span> <span class="cit-name-surname">Baumann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Verdugo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Tavassoly</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dynamic modelling of oestrogen signalling and cell fate in breast cancer cells.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">532</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrc3081</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DTyson%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D523%26rft.epage%253D532%26rft.atitle%253DDynamic%2Bmodelling%2Bof%2Boestrogen%2Bsignalling%2Band%2Bcell%2Bfate%2Bin%2Bbreast%2Bcancer%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc3081%26rft_id%253Dinfo%253Apmid%252F21677677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc3081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21677677&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292028300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.70" data-doi="10.1103/PhysRevLett.103.168102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vakulenko</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Manu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Reinitz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Radulescu</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Size regulation in the segmentation of <em>Drosophila</em>: interacting interfaces between localized domains of gene expression ensure robust spatial patterning.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. Lett.</abbr> <span class="cit-vol">103</span>: <span class="cit-elocation-id">168102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1103/PhysRevLett.103.168102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BLetters%26rft.stitle%253DPhysical%2BReview%2BLetters%26rft.aulast%253DVakulenko%26rft.auinit1%253DS.%26rft.volume%253D103%26rft.issue%253D16%26rft.spage%253D168102%26rft.epage%253D168102%26rft.atitle%253DSize%2Bregulation%2Bin%2Bthe%2Bsegmentation%2Bof%2BDrosophila%253A%2Binteracting%2Binterfaces%2Bbetween%2Blocalized%2Bdomains%2Bof%2Bgene%2Bexpression%2Bensure%2Brobust%2Bspatial%2Bpatterning.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevLett.103.168102%26rft_id%253Dinfo%253Apmid%252F19911861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevLett.103.168102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19911861&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.71" data-doi="10.1371/journal.pbio.2003174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verd</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Clark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Wotton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Janssens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Jiménez-Guri</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A damped oscillator imposes temporal order on posterior gap gene expression in drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-elocation-id">e2003174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.2003174</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D16%26rft.spage%253De2003174%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.2003174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.2003174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.72" data-doi="10.3389/fimmu.2018.02048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weston</span>,  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Tyson</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mathematical analysis of cytokine-induced differentiation of granulocyte-monocyte progenitor cells.</span> <abbr class="cit-jnl-abbrev">Front. Immunol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fimmu.2018.02048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BImmunol.%26rft.volume%253D9%26rft.spage%253D2048%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffimmu.2018.02048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fimmu.2018.02048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.73" data-doi="10.7554/eLife.04785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wotton</span>,  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Jiménez-Guri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Crombach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Janssens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Alcaine-Colet</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative system drift compensates for altered maternal inputs to the gap gene network of the scuttle fly megaselia abdita.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e04785</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.04785</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.04785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.04785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4183.74" data-doi="10.1038/ncomms10031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Manu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Jiao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ma</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Temporal and spatial dynamics of scaling-specific features of a gene regulatory network in drosophila.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">10031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms10031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D10031%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms10031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms10031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/4183.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4169" title="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4197" title="Expression of Heat Shock Protein 70 Is Insufficient To Extend Drosophila melanogaster Longevity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4183" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W9e5BLJYaooAUDTeHR-kWxnuOopqBYJ2u7NzdKtOAhg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/4183" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/4183" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/4183" class="active" data-icon-position="" data-hide-link-title="0">Classification-Based Inference of Dynamical Models of Gene Regulatory Networks</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494995" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-YCUVrie-pqND5PEJ0svRw_90-8O0cx992uKWQ5jNXqo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4183.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/4183" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-gEI-LYTl_N2dLtvbVakJadxcCgh-5ko3IMyB9w-R3ek" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/4183" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4183.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494995" id="node494995--2" data-pisa="ggg;9/12/4183" data-pisa-master="ggg;g3.119.400603" data-apath="/ggg/9/12/4183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Classification-Based Inference of Dynamical Models of Gene Regulatory Networks</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David A. Fehr</span>, <span class="highwire-citation-author" data-delta="1">Joanna E. Handzlik</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7677-7228" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manu</span> and <span class="highwire-citation-author" data-delta="3">Yen Lee Loh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4183-4195;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400603 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494995--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David A. Fehr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Physics and Astrophysics, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BA.%2BFehr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fehr%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BA.%2BFehr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joanna E. Handzlik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Dakota, Grand Forks, ND 58202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BE.%2BHandzlik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Handzlik%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoanna%2BE.%2BHandzlik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name"> Manu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Dakota, Grand Forks, ND 58202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%2BManu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manu&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%2BManu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7677-7228" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for  Manu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:manu.manu@und.edu" class="" data-icon-position="" data-hide-link-title="0">manu.manu@und.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yen Lee Loh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Physics and Astrophysics, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yen%2BLee%2BLoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loh%20YL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYen%2BLee%2BLoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yenlee.loh@und.edu" class="" data-icon-position="" data-hide-link-title="0">yenlee.loh@und.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/4183" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/4183&amp;title=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/4183&amp;title=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/4183&amp;title=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/4183&amp;text=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/4183&amp;title=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/4183&amp;t=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/4183&amp;title=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/4183&amp;title=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494995/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494995" id="node494995--3" data-pisa="ggg;9/12/4183" data-pisa-master="ggg;g3.119.400603" data-apath="/ggg/9/12/4183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Classification-Based Inference of Dynamical Models of Gene Regulatory Networks</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">David A. Fehr</span>, <span class="highwire-citation-author" data-delta="1">Joanna E. Handzlik</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7677-7228" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manu</span> and <span class="highwire-citation-author" data-delta="3">Yen Lee Loh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4183-4195;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400603 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494995--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David A. Fehr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Physics and Astrophysics, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BA.%2BFehr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fehr%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BA.%2BFehr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joanna E. Handzlik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Dakota, Grand Forks, ND 58202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BE.%2BHandzlik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Handzlik%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoanna%2BE.%2BHandzlik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name"> Manu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Dakota, Grand Forks, ND 58202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%2BManu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manu&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%2BManu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7677-7228" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for  Manu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:manu.manu@und.edu" class="" data-icon-position="" data-hide-link-title="0">manu.manu@und.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yen Lee Loh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Physics and Astrophysics, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yen%2BLee%2BLoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loh%20YL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYen%2BLee%2BLoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yenlee.loh@und.edu" class="" data-icon-position="" data-hide-link-title="0">yenlee.loh@und.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=4183&amp;rft.epage=4195&amp;rft.atitle=Classification-Based%20Inference%20of%20Dynamical%20Models%20of%20Gene%20Regulatory%20Networks&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fehr%2C+David+A.&amp;rft.au=Handzlik%2C+Joanna+E.&amp;rft.au=Manu%2C+&amp;rft.au=Loh%2C+Yen+Lee"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494995/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494995/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494995/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494995/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494995/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494995/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494995/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494995/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494995/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494995/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494995/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494995/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/4183" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/4183.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/4183.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/4183.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
