<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/3/427" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/3/427.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/3/427.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/3/427.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/3/427/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/3/427.atom" />
<meta name="HW.pisa" content="ggg;3/3/427" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy" />
<meta name="DC.Identifier" content="10.1534/g3.112.005363" />
<meta name="DC.Date" content="2013-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Rutkoski et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection, a breeding method that promises to accelerate rates of genetic gain, requires dense, genome-wide marker data. Genotyping-by-sequencing can generate a large number of de novo markers. However, without a reference genome, these markers are unordered and typically have a large proportion of missing data. Because marker imputation algorithms were developed for species with a reference genome, algorithms suited for unordered markers have not been rigorously evaluated. Using four empirical datasets, we evaluate and characterize four such imputation methods, referred to as k-nearest neighbors, singular value decomposition, random forest regression, and expectation maximization imputation, in terms of their imputation accuracies and the factors affecting accuracy. The effect of imputation method on the genomic selection accuracy is assessed in comparison with mean imputation. The effect of excluding markers with a large proportion of missing data on the genomic selection accuracy is also examined. Our results show that imputation of unordered markers can be accurate, especially when linkage disequilibrium between markers is high and genotyped individuals are related. Of the methods evaluated, random forest regression imputation produced superior accuracy. In comparison with mean imputation, all four imputation methods we evaluated led to greater genomic selection accuracies when the level of missing data was high. Including rather than excluding markers with a large proportion of missing data nearly always led to greater GS accuracies. We conclude that high levels of missing data in dense marker sets is not a major obstacle for genomic selection, even when marker order is not known." />
<meta name="DC.Contributor" content="Jessica E. Rutkoski" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="DC.Contributor" content="Mark E. Sorrells" />
<meta name="article:published_time" content="2013-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection, a breeding method that promises to accelerate rates of genetic gain, requires dense, genome-wide marker data. Genotyping-by-sequencing can generate a large number of &lt;i&gt;de novo&lt;/i&gt; markers. However, without a reference genome, these markers are unordered and typically have a large proportion of missing data. Because marker imputation algorithms were developed for species with a reference genome, algorithms suited for unordered markers have not been rigorously evaluated. Using four empirical datasets, we evaluate and characterize four such imputation methods, referred to as k-nearest neighbors, singular value decomposition, random forest regression, and expectation maximization imputation, in terms of their imputation accuracies and the factors affecting accuracy. The effect of imputation method on the genomic selection accuracy is assessed in comparison with mean imputation. The effect of excluding markers with a large proportion of missing data on the genomic selection accuracy is also examined. Our results show that imputation of unordered markers can be accurate, especially when linkage disequilibrium between markers is high and genotyped individuals are related. Of the methods evaluated, random forest regression imputation produced superior accuracy. In comparison with mean imputation, all four imputation methods we evaluated led to greater genomic selection accuracies when the level of missing data was high. Including rather than excluding markers with a large proportion of missing data nearly always led to greater GS accuracies. We conclude that high levels of missing data in dense marker sets is not a major obstacle for genomic selection, even when marker order is not known.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/03/01" />
<meta name="citation_mjid" content="ggg;3/3/427" />
<meta name="citation_id" content="3/3/427" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/3/427" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/3/427.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/3/427.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/3/427.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005363" />
<meta name="citation_pmid" content="23449944" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="427" />
<meta name="citation_lastpage" content="439" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jessica E. Rutkoski" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Manhattan, Kansas 66502" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853" />
<meta name="citation_author_institution" content="USDA-ARS, Ithaca, New York 14853" />
<meta name="citation_author" content="Mark E. Sorrells" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853" />
<meta name="citation_author_email" content="mes12@cornell.edu" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Akbari;citation_author=P. Wenzl;citation_author=V. Caig;citation_author=J. Carling;citation_author=L. Xia;citation_title=Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome.;citation_pages=1409-1420;citation_volume=113;citation_year=2006;citation_issue=8;citation_pmid=17033786;citation_doi=10.1007/s00122-006-0365-4" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random forests;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=R. Dassonneville;citation_author=R. F. Brøndum;citation_author=T. Druet;citation_author=S. Fritz;citation_author=F. Guillaume;citation_title=Effect of imputing markers from a low-density chip on the reliability of genomic breeding values in Holstein populations.;citation_pages=3679-3686;citation_volume=94;citation_year=2011;citation_issue=7;citation_pmid=21700057;citation_doi=10.3168/jds.2011-4299" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B. Met.;citation_author=A. P. Dempster;citation_author=N. M. Laird;citation_author=D. B. Rubin;citation_title=Maximum likelihood from incomplete data via the EM algorithm;citation_pages=1-38;citation_volume=39;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=T. Druet;citation_author=C. Schrooten;citation_author=A. P. W. de Roos;citation_title=Imputation of genotypes from different single nucleotide polymorphism panels in dairy cattle.;citation_pages=5443-5454;citation_volume=93;citation_year=2010;citation_issue=11;citation_pmid=20965360;citation_doi=10.3168/jds.2010-3255" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_title=Applied Statistical Genetics With R: For Population-Based Association Studies;citation_author=A. S. Foulkes;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection using low-density marker panels;citation_volume=182;citation_year=2009;citation_pmid=19299339;citation_doi=10.1534/genetics.108.100289" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=R. Babu;citation_author=G. de los Campos;citation_title=Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs;citation_pages=654-663;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.07.0358" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. Iwata;citation_author=J.-L. Jannink;citation_title=Marker Genotype Imputation in a Low-Marker-Density Panel with a High-Marker-Density Reference Panel: Accuracy Evaluation in Barley Breeding Lines;citation_pages=1269-1278;citation_volume=50;citation_year=2010;citation_issue=4;citation_doi=10.2135/cropsci2009.08.0434" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=B. W. Kennedy;citation_author=D. Trus;citation_title=Considerations on genetic connectedness between management units under an animal model;citation_pages=2341-2352;citation_volume=71;citation_year=1993;citation_issue=9;citation_pmid=8407646" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=Y. Li;citation_author=C. J. Willer;citation_author=J. Ding;citation_author=P. Scheet;citation_author=G. R. Abecasis;citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes;citation_pages=816-834;citation_volume=834;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Li;citation_author=Y. Li;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_author=A. J. Slater;citation_title=Performance of genotype imputation for rare variants identified in exons and flanking regions of genes.;citation_pages=e24945-e24945;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21949800;citation_doi=10.1371/journal.pone.0024945" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=Y. Lin;citation_author=Y. Jeon;citation_title=Random forests and adaptive nearest neighbors;citation_pages=578-590;citation_volume=101;citation_year=2006;citation_doi=10.1198/016214505000001230" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic selection in plant breeding: knowledge and prospects;citation_pages=77-123;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_author=S. Myers;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A new multipoint method for genome-wide association studies by imputation of genotypes.;citation_pages=906-913;citation_volume=39;citation_year=2007;citation_issue=7;citation_pmid=17572673;citation_doi=10.1038/ng2088" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=H. A. Mulder;citation_author=M. P. L. Calus;citation_author=T. Druet;citation_author=C. Schrooten;citation_title=Imputation of genotypes with low-density chips and its effect on reliability of direct genomic values in Dutch Holstein cattle.;citation_pages=876-889;citation_volume=95;citation_year=2012;citation_issue=2;citation_pmid=22281352;citation_doi=10.3168/jds.2011-4490" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y.-F. Pei;citation_author=J. Li;citation_author=L. Zhang;citation_author=C. J. Papasian;citation_author=H.-W. Deng;citation_title=Analyses and comparison of accuracy of different genotype imputation methods.;citation_pages=e3551-e3551;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18958166;citation_doi=10.1371/journal.pone.0003551" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=J. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_title=Variance Components;citation_author=S. R. Searle;citation_author=G. Casella;citation_author=C. E. McCulloch;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. J. Stekhoven;citation_author=P. Bühlmann;citation_title=MissForest - nonparametric missing value imputation for mixed-type data;citation_pages=112-118;citation_volume=28;citation_year=2011;citation_pmid=22039212" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B. Met.;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the lasso;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=O. Troyanskaya;citation_author=M. Cantor;citation_author=G. Sherlock;citation_author=P. Brown;citation_author=T. Hastie;citation_title=Missing value estimation methods for DNA microarrays;citation_pages=520-525;citation_volume=17;citation_year=2001;citation_issue=6;citation_pmid=11395428;citation_doi=10.1093/bioinformatics/17.6.520" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_title=Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle.;citation_pages=5423-5435;citation_volume=93;citation_year=2010;citation_issue=11;citation_pmid=20965358;citation_doi=10.3168/jds.2010-3149" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. C. Whittaker;citation_author=R. Thompson;citation_author=M. C. Denham;citation_title=Marker-assisted selection using ridge regression.;citation_pages=249-252;citation_volume=75;citation_year=2000;citation_issue=2;citation_pmid=10816982;citation_doi=10.1017/S0016672399004462" />
<meta name="citation_reference" content="citation_journal_title=G3: Genes, Genomes, Genetics;citation_author=V. S. Windhausen;citation_author=A. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments;citation_pages=1427-1436;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.003699" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. K. Wong;citation_author=R. Bernardo;citation_title=Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.;citation_pages=815-824;citation_volume=116;citation_year=2008;citation_issue=6;citation_pmid=18219476;citation_doi=10.1007/s00122-008-0715-5" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=Z. Zhang;citation_author=T. Druet;citation_title=Marker imputation with low-density marker panels in Dutch Holstein cattle.;citation_pages=5487-5494;citation_volume=93;citation_year=2010;citation_issue=11;citation_pmid=20965364;citation_doi=10.3168/jds.2010-3501" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=W. Liu;citation_author=T. Würschum;citation_author=H. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations;citation_pages=769-776;citation_volume=124;citation_year=2011;citation_pmid=22075809" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection, a breeding method that promises to accelerate rates of genetic gain, requires dense, genome-wide marker data. Genotyping-by-sequencing can generate a large number of de novo markers. However, without a reference genome, these markers are unordered and typically have a large proportion of missing data. Because marker imputation algorithms were developed for species with a reference genome, algorithms suited for unordered markers have not been rigorously evaluated. Using four empirical datasets, we evaluate and characterize four such imputation methods, referred to as k-nearest neighbors, singular value decomposition, random forest regression, and expectation maximization imputation, in terms of their imputation accuracies and the factors affecting accuracy. The effect of imputation method on the genomic selection accuracy is assessed in comparison with mean imputation. The effect of excluding markers with a large proportion of missing data on the genomic selection accuracy is also examined. Our results show that imputation of unordered markers can be accurate, especially when linkage disequilibrium between markers is high and genotyped individuals are related. Of the methods evaluated, random forest regression imputation produced superior accuracy. In comparison with mean imputation, all four imputation methods we evaluated led to greater genomic selection accuracies when the level of missing data was high. Including rather than excluding markers with a large proportion of missing data nearly always led to greater GS accuracies. We conclude that high levels of missing data in dense marker sets is not a major obstacle for genomic selection, even when marker order is not known." />
<meta name="og-title" property="og:title" content="Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/3/427" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection, a breeding method that promises to accelerate rates of genetic gain, requires dense, genome-wide marker data. Genotyping-by-sequencing can generate a large number of de novo markers. However, without a reference genome, these markers are unordered and typically have a large proportion of missing data. Because marker imputation algorithms were developed for species with a reference genome, algorithms suited for unordered markers have not been rigorously evaluated. Using four empirical datasets, we evaluate and characterize four such imputation methods, referred to as k-nearest neighbors, singular value decomposition, random forest regression, and expectation maximization imputation, in terms of their imputation accuracies and the factors affecting accuracy. The effect of imputation method on the genomic selection accuracy is assessed in comparison with mean imputation. The effect of excluding markers with a large proportion of missing data on the genomic selection accuracy is also examined. Our results show that imputation of unordered markers can be accurate, especially when linkage disequilibrium between markers is high and genotyped individuals are related. Of the methods evaluated, random forest regression imputation produced superior accuracy. In comparison with mean imputation, all four imputation methods we evaluated led to greater genomic selection accuracies when the level of missing data was high. Including rather than excluding markers with a large proportion of missing data nearly always led to greater GS accuracies. We conclude that high levels of missing data in dense marker sets is not a major obstacle for genomic selection, even when marker order is not known." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
    <title>Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yf4YBcaZlZ19u2VUICoKB3x9LynfAQkpr0GwpSB_6l4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459422","apath":"\/ggg\/3\/3\/427.atom","pisa":"ggg;3\/3\/427","ac":{"\/ggg\/3\/3\/427.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/3\/427.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/3\/427"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/3\/427"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/3\/427":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221FzHJifdVFCMZtxE6JCApH5POtFax5YlY89fdRw0FMBBCKsnS64sYu6YfIlRWvncD8gpo7O04GFXkbJwaUCZaCXoG5Z9piQNyZRSK6n68RmCEUbG58E+2LMpILDkKMkHse3zh5pTB+vMnAYO5vJtOTu\\\\\\\/zIo5QZANYA7inMSGK9T7W\\\\\\\/avz5HyZA++uXwvJw4sH0WkS3r9Z3Uz16QJvvEC38jtGXySuuQU2JRZ3pL5vcuSn7ZvXcjjoQlMbZeJLLfDQtxQesL3SXhjeQxu9HiAhFfM+EKBkfU6EdZsGfSs5lUGR1+qqPEBix1iCRfH44+o4n1cqt1OhnBvYL4MFi3FbN8Ej+bior5Sm3WvitLVQuk3pZ5QszvBzQUO2mS0kgbIf9544uE2sjz7IZtSDTyergHIgaALe\\\\\\\/t\\\\\\\/lMA2lq0HMu4nJHoeC5kaCL+DS7Qr7wZb5s61mNydbjSqUBBZZHgysiICKziGevmQkI\\\\\\\/w5+wX3EAh6QwGm3PNNgxIF3vjttYASJyIfDWTFKCuFdS+z\\\\\\\/+C2Fcpa5tWBWUpqQaJeB\\\\\\\/w9Xpc97XqtfEu1t6SUVJX6XH4J3uJt\\\\\\\/3Ty9euCfnDX+eKn0v\\\\\\\/a20aKoArYnbINxn3v0VvOO8MUJEyGi5pXrQq2ubj9ZyA68BFsvzCvJsmuMDQvSB3+\\\\\\\/Xc+DZ5fUWzfJG1GZjmqKszbEanzSGbY7tFDOJyRYSvi\\\\\\\/eunNUXG4guZoFoVYvQaglJbQ6LBtV76Uo2a+JXXs8oWDYbgf1F2HJtR6fyHlS0kowmDAGhXiSzTJcEI8+f\\\\\\\/TXIz1YcUyxD0Brt6SjhL2Co6GgOqrXofkViASNnxDGyzE5+YaBZHahop2YZytguN+QEjr+rSCNs8DlAFBtXUFfBMT53PEyUjiOl8YeC9rqxtPRTzOMepnc1tz8th+\\\\\\\/SdBOo8nC8YizfSZRjPxxThv7EHlou8za\\\\\\\/Fr\\\\\\\/82FwrDtWW\\\\\\\/O6TGClC09TWjgIe\\\\\\\/qswv6+2hkh0QEMmKwHrOmJeOFtKa3ks3gEJfpX7xD8UydyOVz2avpxCThga+jrTuUawaekFha8vSuhH+q0EqnczG\\\\\\\/fzn0xQCveLk5M8Eg8J0EDewt+fmrvIH4MT06\\\\\\\/2UF1EP3uFeFBoYU8+WRhem7b\\\\\\\/l2eqgfKpr1AI34oOK10N5JH8XLxmC6gYt1og03Twam4JS8NtP8NNcqWJVVm77cqrQvxGDtc2z4\\\\\\\/t5kzxy\\\\\\\/YmCwpn7XDTT4ZxnhyRwg9YRw1SYHI2QBoU4GXPJ9Kgl6UIm64UgVq5W05q5IkX9bhr\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ms44BZ2hDXf1ic2g9BxjgA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022798edd3bc0ec170eb9c573720152f35e9a999128a9e9b37d7b252e4811272663\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mCGdKj9FE8iuwDlftAdCaedwQgdxlgDYwFL3p+LfKVdPxOpBrLQrTJH4n7LhgNhIWL4uuHL11afXEShdEEfOnJo4KYjYIpxNM8+zxzdMrKSHXOQkm4WXSKtit\\\\\\\/K7r5jCSWjD4DZZLwXY\\\\\\\/PfElxJx\\\\\\\/yyusRFCzLfObw4AM5fqBwjUy2nb\\\\\\\/jJHZEE1W8xvrEB7t8L9Ae0JrceAKPckxmq8mATpud1uohjS35LyOuCaWEZe3rJv5Swsr4FSGUN8qaABBHLOKCxEFXYesJf1L7CZfRb8Ql+VKOann9\\\\\\\/xj2jQfNuvDuZaHERPUuhjdW7XBBjt3ckpiNGeDfkBtYoNSFOOibplaUXBpwxgf1lgVv3fod2aODCqOW1cWXEHFyHEdUz0VMpgqckV7JiTO6kayQjD2mljyEj+O7snNc48qjs7WEBgkbq7zqWzUloJmrKJ4cgwXRBN3wPPYRsLrvMtnPgRTmGHGxiITssI6lTmcFRwTqAQBIHhNw2fWg9502iiM90KZWgFM7VlKIkS\\\\\\\/SUCrtqA0DejAt4LyjiWjGnItRJQQXWeiaiHJAtlVYcV2cXT2QezlELKX9rNtrDR5b5LCZqNR1B1kIoICZeEhdnwOR53Ne1YKOYLsOPlZIWQMIxjD8kLZlDOjeM2g3CwaOKQvdxN0R5pooD1LjCnS7DndYxg8x48ICtUbWifiV+iyo\\\\\\\/L9OZk5QA9WabZu9yrxccNzwcC1xWYbW0TUZaKn5YjRBHQNWnFm8mzDVN71CjfPlPUaXpRyy3v6kok7m928i310IPdGy+z3BCJHzT5DQgV0F3Ypbv2XeB3QQh7A7DEtT6qQRRyLidt4\\\\\\\/6iE0wAJqBCqRvDIukmwW\\\\\\\/YrZokHx+UbtjudFy8O0vQIxZOLzF6hB59NxqBcHfMsHuGZBUOviSWkpdKevKlyyQJMbcYssOVp9\\\\\\\/pl\\\\\\\/B6puC2rlzit7d1EgZo9x6wvKU\\\\\\\/4DGzF+T64OJfGQEah8cxZe\\\\\\\/sITJxOFH5Y7ymcH8lNJy2cTEyphQxQrmUwRTRwDz6BgAejeIFuiLHaapbAySJo8vosTle8k7oPjN9HHl4LivO2cyE9NaANeHeO58RmjdopXUX3NMMrKrNb9MX5VARVz+JUt7mIdLMbsRKdpNcJxe8bpm5\\\\\\\/P\\\\\\\/Kcd\\\\\\\/O1L0hwptLwDhNgsQ7y3+u1vbM2FF0O9WDJoHbegGNe2haiDbaDk2QFa+J5q6doiCv\\\\\\\/lvGTYcGE6uu9EWZPC96tv5COroWE19HuwlwjGKCKnsyumn47KHh2uAKWnwvqNPp3lKWFAPPaVT673QF6HD3k+oCVa3Ktmk6\\\\\\\/8skrcgeXg+Kx8czG6qjtx3uQB8pl6awdLedzGHjQ6h9+TqNdTajLwIUJ+7lIAGAhKGj5YmTzNkUEHKZBVP8P7VAmjUCwp5A7pCFvjtDBO6EvDQ5LdJXkbGo1qIA0PyPqY9xJ0j1c4NcnMzybRwZwV4TtwquwFj28tVgv0HvDCR2nwsnMrMkARx\\\\\\\/IE8vEUjaJBcPzzFGPHE4XFUgdFezu+c8szdtL9z3wk69i1Mdy6Oz4zaCoyy9psqwyft3vtYu8l5Q3JYiDSM8TXx0jgQXfOQstqj\\\\\\\/GYS4fv5sae4UMOKYxp2R6\\\\\\\/4cgjeURehkkg4YLWN6v97OB8j0xYQs5XuK57Ssl9eGqNycVcsxX+w9rE3lEt8oYK7g7rKPJq8oyktNffZ9mQGdNviQyMNCM2+AJvyZHrkm4cEdi3pJJmJcCC1OXC+80u3Ct8EGJ0ZfbDDEAa6AJ+\\\\\\\/nflBZdYJ+1AqcdCZ9YIX1LUpd4G+6DC38cVMTVswRC\\\\\\\/Whf803j1kGEAm4goop2L6JNdXUApCv6l0pqxT4WcBqMrZquwlpruCj0B4PI8WN9jaWX5WLoeZWUIXQm30XRkDKJJFC2+\\\\\\\/3lywzd8lAFPXrvI9b\\\\\\\/BhEhJ\\\\\\\/harBTvugOIjkmvhNafNXMm8ifuwEEY+Nhk+4Nl0hK1X\\\\\\\/CQBM1BcRg+utJKA7ALN6cxfzutLzu5YXjkuDM9zK1vIN02gBus3wLh+i\\\\\\\/yEmxwJPtST9HL3OJqsdg73kCF\\\\\\\/ImzuYxNTMZ1fDC\\\\\\\/McvmvSjLOVdJ2Whvj6Wo0PZHDBbZ5wRuVaKvR5DLuM+LOk3QdHfjRYcZGEBi6KcrHIqJDCkkhQ2C1YKwNMx\\\\\\\/AFvNQLnGjCzcs+lG6IgmwHONrYG2t16gii7yydRUv1ihKEK9E5hooOyMWQt3H9Tlm2M2nz4ne9nmdPa6QL\\\\\\\/Alu+561Hzx\\\\\\\/hNkCvcAUhh2Mt6wmNMnA5xc2A6vCBDQPdW7It4YXhwuHCuWErykMSyQ9VkrvLV+2W7Gb753WPQ8Pc9p3hwpT9nAhxmWtORTQGaikeJqY+Ets8jEBlxiF4U2WpEVyadOeldQ2rDcw\\\\\\\/VTeZinqEXM7Dzb0k6cIOkkEjvT8n0EzwyupH3902\\\\\\\/2VLiDA\\\\\\\/Eug7Ye7dNQx9TMeF8JJ3mhzc3Rtgd3ANZqylLFhsWhG\\\\\\\/glKC1XD1PAKn4qctdtJPvojdfkRPawYKNVaSfdejAfmd4H5Wqk4OK81AQfh0nOQ735HA\\\\\\\/aRFcbd5DlAYAwxu9kxfeIyBxz74Y1P8raLn90JZVk2PRmqmdA7hrprVfyb4KJpJfDEhBU5FCnAPi0BMvpt4ig6xJ\\\\\\\/FVM5trQ+t1mZiiAMRRI92fybEfPYLG0hFZt7NJ8Xtqek2f8oSww2IdvIp0kuNKstOxdLWjV5Dz79Er0E2+kxFrQZ2ADWQMfU4UlM2Uze1s6KGSQLmJ0R2S4ImfVQO3jCa9BZACfempRkMG3iqv2zg8+r5DHnl\\\\\\\/pKb\\\\\\\/9NsnseSL04Ks+9yCJSt7txM8yKYbmOyFJnzfVWlJKxNPAEUklyIjjtRJKyfADX1SXxvGLz2Pey4GkwgV+uyvIbmzsodGf3vmfhjykiBwI2cGe\\\\\\\/s6nEG4oZq9iiz0IPRsjKauWDLDMoV5zq5buRiJxfe0n0H1yLd4wF6c9I7FE\\\\\\\/t7XkjBlh8YJ8oAj6\\\\\\\/fnmdNKcP0N5f4HQMJOIcrEb56xTFCpgKaIObLc5O+cuzEbodKhcF0r8luGVccxgZCOJn7+ET0EUlAchus7diVW3Ff2pxdgGPKCo+x3wPezauLYkWjydS2k9P+QSjQ8EfHRcPnBh0nKF3e8Wc3cc0zm3gZe0ssDa2SfjrY9wIiWsZuFzpY9qbn100sDRU4zde3xmjhIHBk3WAVtdooNG9E+B55XTD0WlmNNsiQwjxKxYMk95Zr8jtsFuNUDrHXl07M6HR2Pus\\\\\\\/2DbNjaaJY5xlqOLyOn7nWrfww\\\\\\\/ZbdAD\\\\\\\/aD10MztkVA1wvCF6xGOYzDdZo0kmcSxJieXJToe4jof5XyIU9Rl0lOWDxxI3yvYrvXxp0mPUP8CRwoTQ9SgDjNjyCUUjEa0KbPWeq7LMMfjXXEz1qycr6iOKZXJc0XIccBKHRzDjS8bcwAMx89QifXDOxql9P3wKBnkWfTKiW7qJB3WS1l0ITYOKzXR7MLvuaCRWnwHXKmMhzycEe4c1ae7yoY4Rn7\\\\\\\/RN8jRqR5SgcSi6vEsn40ZqvuuiQMhen\\\\\\\/8l21SXdVHc5MzxjV0zoUa07gbgLagJp9WmECBzwiMZVuPOOgmt59Em1PWqVxtmmlBGGCgo9NX9PcWkItU3kHoEALOAdGWrxvR+hr2525HEMboGITiLfEVv6VJb02kRenOMNxBJHxYMynqioYBdd+eF88M5rpt9n1ppcROpmt47Jnkky8JCXy5YDBhdVYXetcsiqrK6DcxCSslzJNKa9YYSGwFt0l9hfizSZxRqU503oxkdRLFuF2mAE3azpijZ1HkdN7tQxuW\\\\\\\/4kk2P4PfVdtjR8j1YfgGTKuWT6ArJbRNPhg6L13IEHG60gqW9y09jKIqRcmdzgbsCdi\\\\\\\/0cOwk\\\\\\\/3H69M1YonXSX0ZRjCN7CJhCBN2HvT1yRLKrA1ufb14f2ct\\\\\\\/0SndIPeeiotf1+os3Wb9KksrzZs1tuXUdYlGdx\\\\\\\/JrRa53Fip5Kirns4aBIcKzj9OAyKZ4k6vxha6JM1byszwiy\\\\\\\/YghcSB2VATpGhtJMLeEaRfrP+oPhuP5vzUSjueDtJUNnC58vWLRPbgLM4t8nTnwT0+mASHE3f8UkdQrcoA5N\\\\\\\/XM\\\\\\\/1bbNs+lHdM8KrX\\\\\\\/xyped3bkn7Otxu5wa3AFXu7Rfq4hQOqI\\\\\\\/H\\\\\\\/1Fpji6cdQ3SUBpr9j6wLjvjy+2GBwLMEqLtm6PjOHQ1e1BLioe1m5POOGRxpMW22q3Jepv4blDK6mrIr++ypABkinIIWAhuifXAbqLlSGh1ADUICvVQAn42RyXXVADdnnDkt1EVI9CQZEvBQ6f2eMEPy4Z\\\\\\\/SzBeW5JYBrkuUK1t09hj2zGQ6ViarEM9vmptvShR53sDFi7GCjoqvGy0U5iEGBkJosrvtxf6HKM5ZqXOVYWBrhUFxSLUomls0liAn1V\\\\\\\/TkNOrjyaweoXPbIGtPDMe+eHUefeD\\\\\\\/MXIzIuB4KKk5NFuzy72L6CJu0xD3Bl2AZUcLDVOzJp9swZiP0Sdav32\\\\\\\/a81nwjQ88paNlrWK\\\\\\\/1rFF7q\\\\\\\/cb7Y9TvE\\\\\\\/MaEZbUFrHaiw43tLNMION+burvLzffnLIx73e8rcbEdPcOPFXPS66AtIYOOcS7OgFktGFBGY7MYgtD0Tp\\\\\\\/bKJREu8A3DW0hMdsnexyrV\\\\\\\/jzA+n3ZYTgmvU0nojoJVHjERGN2LiPvPtzlWIj7uV9u5Y8oEmrlEaAl0skaIS1cS48fFgYkHAVvNRL+fYePkZXLIKqTIJ2UQCPSDOipqPYy1Mj89KQ1ahsIvZ6x2KFOH+1IMoUckC5OCaCvK64JWqtrFbSGPSt2EQV9P1YnUGMC2X3C7FuyzqjHtfMZU0hQdibTlWnRSJWCmXqoJc+sie4F5CTIAAM4grTlvts1q\\\\\\\/Q32bG4nNU+k1UVd1sNa7+MzV4rQrLb9L4hp0q90\\\\\\\/FjoL1Isvp285C8ZR+BFKuOLSm7CMdLHWqSQpHsj9jDAnM+ZJ56\\\\\\\/ff5LYtS\\\\\\\/9NE5Xb5on4ocMBeV\\\\\\\/03IIteqGm+m+yUdoas3WsOAjUhxRYnJXZB55N0rraPPQPksSc3Zq5nBezpX5wB3v3HE+wqHNyTR7\\\\\\\/7Y525U6qw6nPz2hm6SoEUXl0lxGJJ6duOx90XoQteHyVZbWybvIXCCfW2hxBooGQJ8mwi3JgQq4IX16ZXx\\\\\\\/hVymy25c7I0w+5oNiOFOFHqU5whjNh21Enng9vmMfdwo5rZO\\\\\\\/vz\\\\\\\/3khtblbJ7pwUmwkKNQksfUGMy7hqgNE0AD3ntLzUZ5+lz47jLMZFk4QQNekIRmDsB53kzzWFjDKkRmpXlVWsoxCA7zO0q+6YUNmQW6Y5VPD\\\\\\\/DJQLJiI\\\\\\\/T81r\\\\\\\/JEL+8WYM1KrQQsnvM9ZeQRPelZMwlgX7edhJjULBTMFaBdvH\\\\\\\/QrnOJ\\\\\\\/y\\\\\\\/NF0kVYsC4zLFjA6h\\\\\\\/HyuVa\\\\\\\/hZ05G2+H9cEfBlU6tGdfLysii7toLALC1hqEHQ+eJBpd3tIysKjGGJFHRndVi092p8GBsKLjy1n4hzbFPEC3lZ2UobTt9pXnT\\\\\\\/CLWKcTEGJLpcyrJ9hMj1PFfeoFOLftznhiAVj5RFerciea1oy2CItvIlmh0A\\\\\\\/d61wXdDrzSbpC\\\\\\\/IVyUgJ5k02POQcje4mESsfwYAukDhaKiUU1GCYzlNi1JttdUm\\\\\\\/XixQzOfdki\\\\\\\/V+N42FIwvesmfRUJiUmVCQ7wC0Gk0dx3duT7Znu++GzxKD6Dnd9HDufRTpjvJCuym9ig9zpGBidfaE04w8zMwFC6uMUH\\\\\\\/w5b31aoHU3pzxg5cWKmSs0\\\\\\\/KxWIqm0FzOgPB2XGc9l\\\\\\\/vDiU+h3tIEyFnIO+q4jcRFU45B0Rx\\\\\\\/qnZO9iafjKtwieG4ZQYVQ55totBLaWqVFZ\\\\\\\/4o5\\\\\\\/d6h6uHl2o4yil9QtYfe\\\\\\\/ISXqqBcz4cRLpylWwqzrG5uHJ8pVkuXi77itwOxe4kZFP3yKO7x3c3ByEC1k+xMoOkLh+ibDtQh01Cle0aRtqIoL9\\\\\\\/x1Q5D4ADhWQ6Z1U2McVoYfg\\\\\\\/MRfts3lU39RL44QfvHqIbMxyTnSbjnzXtHZfWlMzLIOZ5OPbO1gHKhRP8ZSxs9jeF7nI+SgoXRyOWTqxMre1xNzvnaWDsaEhh0Be2WkfX5bifEeltUceQuOIQ\\\\\\\/iYQnXiNQ1GcWrohpfm\\\\\\\/\\\\\\\/vsRIpFy+BqsVs1WZ6hScx6vaXcgf+bLiC3sc5kgMAhzqIRNRGmiHL6a3502j8rUEBJuxyrd5aVNzqx7UJcED\\\\\\\/w11nvJiEaOVil9FV9s7ds7wFAjvwe5C\\\\\\\/PKafQX\\\\\\\/HYN4qhOo8sKbdMBFS1tJmMd1R5ONgj\\\\\\\/OFi6ctn\\\\\\\/LD74gWUhr2Ou5tBlRw7QlGR7dSosd\\\\\\\/tRibgmEJIJNaZu\\\\\\\/AM1kaJ+DuJnwWhoaOaf5WvaBDgnKDTDf40p1hMAF311nHtazcuqRlGovssL29iO5mUORw7X0ibWQAZgAt1UWnixAKeAHGqUOgiWJpKN1QhGGjPsL8HeymKfKdOlf\\\\\\\/szlbR5M6pHyatONVuxGGAEnAtGvBtfgoubARJZ1VjY8YBOIclkqzjZfmpZ\\\\\\\/iLCDhgWImTUojXtHk95RQvUWCybT3bPlCXWqO5gRQAKeQPIHqFJ0fNDiJc1iZY3pFbErUkfoLgN6nlDIvQsawGIJNJHJ2QVTl0cnGHcwtdNHBUIUMxHtQQQ6ypwkkx5B4dqafMtdyj+cQicZKzqkroOvQ\\\\\\\/E7Lt+t98gVLAjgSWZOZnSc7xua+i2g+7XV5Sih7YfossniPg1J6vi\\\\\\\/bTpbKQNhz4k3ywVJL69\\\\\\\/Va6KY6qdIe280fC54t1i12xCjtSzXGrXwfq7xaSyllWcRUagY3m6X8cv4VmmZYl7if2AqwWvaKOu9XRaGVjgD6CFsHQKyeV26moSRjNXwm9MOh+aA5V32sKShilrQgxjfyhGKlP2pMPvU4W\\\\\\\/UrUGbdhxc3TClMgNO8mzgb3MyeOx6sjNShyNxBvpBM0blOC6Dgnberm6NqwIkaguc3h9V\\\\\\\/64Bn18xrf5fW+MjvCGXRgqW+toA0+9etQkgWgKKTD9i5UUNQy6Ok\\\\\\\/T6O7cAwKhYqc5v7sol+2tWozhnYV6ogMBowlwWFYMkIt+FFOvwKQHDzI+VKliP6wp3DCexkt7z2msWtftvKwS8OVf86TRV6PL2jvfw00LYbPM5JSrlcKybOmtJH5MQvv+wsOg5iq8CJ6fY1MeKeZQMUSTXh3t2\\\\\\\/YAXFp4JCo6tI4EPSWDFHzTWxbGAY+ILnuuXvbe6mlOSl3ffwVK5kVWGY2perro64c3NOMb6Rs6Si9f3ldV2ent7eVDM9K2eIvoQGQvoejIdy\\\\\\\/0kee298wzR0xGoAwL5\\\\\\\/KAU1jKQ81y7HO5tb4rtY2OL+NbiZHtKmItHCZcYVS9VNdFPIqIatzbZt0sjPbPP2hj0mGJGf01IS6kZYOiWHB\\\\\\\/\\\\\\\/ZpUvS6paKqt4QLqZ7dKG7rw7XVeLrHEf6TedqI8nwMVGwa3fEKQvRgYCk+3Uy7PWdStkJY1SMaO\\\\\\\/bMFAc7Q0BwTGEW1zhNBG6wF3er4hn4cKCZcFDc1RGUM0YClyWd\\\\\\\/2mBpZS8daOmQ2vgFXK0owhWsmpKJXjZBkVJWRNPKvrXk3+0ltM2cNWxC9p\\\\\\\/onhaA92HJTUxW8djETUTKjkSPQbXSieKEsTEbENxIz7i3YDDGfqG42spBU1k+xv2ZY2qr6MWLzYvXInyLEic6ZelC1fC2XyMfKTCxVWgYkYR9627XHbmxKZqCyi1HnhhGMcmTea221s4QbwFXD5rI\\\\\\\/\\\\\\\/+mNOfkoK+oRmaOUxgHDYqwUtKSpm2Zcfn9v\\\\\\\/zBT3LlnVrOiHuESXe+l7C9RUwezr+bdNTE7rf9Mv5UYUtUY3ItFKlxfzcYRwr8+Sko9J0cwSGt6UpvFDV\\\\\\\/8fTav6LdMSIFUQ3MZ12EoTCaRkQdJrzZ3let8ajuYZbYEuf6Gp96yXshdjAQY8fsOuXbYbOScUPuEd+yS3+h\\\\\\\/dfB5AQ03H4YvFN9CiD0kxiynOXf8UMxpaml4arPitLD59prRBFG3fbtSFPGdyJlZEsekrq9l+ohu6c1UxYYhazfZod3RTN6kxtBkVramDdIg8Cuzwf8plJiSBSt+qNZgDgUjP1zt7Xh8AHIKAj5mMHu77BwQUmMl5nDO2WxvykgoJzgS5cqIhPEQCBjCfSH7SdHbjgRsTGX770Y\\\\\\\/ZHFIhh6RAVhkB+BdktJSIt7S5XYS4c15hJIycR9T2Jss7grAqfRpGWuSQKUjHoZWzNa3lCqq1leUJ0FSV78MWIamipzJ4XyXrX3Lrolgg0qCftrqDbsNGyRN6diJwxyUf24XS\\\\\\\/RGWaBrgyG7qw01K+NC0jy49hUU6BA4Zk2N0+GRYhMUi16DDAGVCwdWS7wgISyyw1mBgDostfA2ksbVv29PjtsVTG\\\\\\\/y\\\\\\\/ICG3hxAWTpeyrIxtzAimx07pZ0gXxfKqAKQN7nCOOtju2lty6uO\\\\\\\/fcvgNZoQRlDkP1+bUbmqJaisR2LN3otwzAC9ALRDR3KOKRWkpuKM6p\\\\\\\/KIu6LQgiyD6+9J+YPlQX+ucxrx76xWBszQbG47uczrn5NXbuSff+5R78lGM3WnVDfPGpdyaygyAbnYxX6u4gi6K\\\\\\\/H6\\\\\\\/W\\\\\\\/YWoX2mJWQKH8atfADnStG5UybdVRj3A2FmZeN2r1yJeGo4HRtUyySO1o2bIjdH8lf+xHTbysXq\\\\\\\/W6Szd1XkucPr+Dp3JpkpEHeWqRTsYXuAeIjh492oQ3MEoCjMTZ+qn1nOWfJfbNfTrchWeirLBXXHgx5kzc1qyzP84hZoUC0Se69C5Afsyvwy9+8XRu8xPjvPf9Mmqj1hfugo7rgq8RtcHZwyI3GV761HjCYBLrFf1RtX8gPCT08RCdAGtwdYLgiFftVyjfkPojP1LrrTEVRTpzGOUh4\\\\\\\/H6iiFyjzOxFiGV7Hdy6Yj8698Hb0ySCAIXlYYDL6qP\\\\\\\/DGlwbBAqb6ivAaNyXUzQdehuPZ0oHMV7Ubu8tyQttsHilv99xrh\\\\\\\/jjh2HH7vuywCViH4jSOOP\\\\\\\/9mZ6j8H\\\\\\\/v1PoBBnhDfE9JvnvjID6KDnkoU9xEQ08c7bEpxlCp1KXo8yaoSextoQ5lxNeKbnoTR2jIVOYQQWGwzmgy823oOPqLW6T0bp+bgBmUPCz3daIdfJdL+3tzVNvP0WnHgC3hOTXk\\\\\\\/KSLk3iaNImsTEokJTKzy\\\\\\\/6rwgNAcaVtkG\\\\\\\/9vviBgUwIKf2vcll1ZwVn4lD1gDvso2wtz5My5EOkGqNNsyMlRTIsuXw6KRd+ytn1eitq+S7kMT3sH9JauU2b9Hnt6aooD0tWj4baZoTAODy\\\\\\\/jf0yAvdoiNcY3WA\\\\\\\/kRIIhOmxtm65lMthEQdxL1Mtd+E3p1yL\\\\\\\/50HYnDUB+Ix3CIcyj8epp28TnwDG0Y6c7jT7GKsRthfwvCWhO8p01d4TdkB62kXAPiCYoSQPSNLbYrJ58MiKKX4pxlXS1tv+oCPb\\\\\\\/y4lCQXThkcItDGXRbIHS\\\\\\\/sodXKv7+OlJM+ojuH5s3da1g3WuxgU5y9LCiHBy3Uxo8fdWt7Uc5Aw8l0JArO3KKfAgUwKATj5AIKnmW8t1nC3qb+GSGAbefOH\\\\\\\/cSV34lOZjinEZhXzPsN39aBqCzimlFAdiPNk8\\\\\\\/LG2GoWYFWHOqZmHnC5sBl68MjPAM3r2pNRznVSLTeBtS3Xo4Yfpz0iDCZtOW42MLsQda9JFKcDITQ478vM\\\\\\\/Xo\\\\\\\/x\\\\\\\/owu255DhfVK9y9q8L8jgQYjPzvk+NXb0BEz89SznWQWa347kZGr1MCSmhYUmRBxVUnoMa8cYK1xV0koF+WcFGbqOzsI7\\\\\\\/co\\\\\\\/9MVhWKKdNe3bctptzLJ1IjQB4XjKCaX36ugBu9+H2XH1xqP687QVH\\\\\\\/nIAz1Fynhx5rGd0aH8rn\\\\\\\/jRFMtdkeAfwMhUIeJvx4Go7ms7\\\\\\\/1wzuI3cmLN17PMx+4dESLySUI+7xSTpRI0F7c\\\\\\\/c5C4XzNv17USEJpOCAYfNDYyNpG6dO+mUsfvIh3B3eojQpY8tOlXOnckA+Opz\\\\\\\/As9xCVwIJs9JYFf79XdGXQKwXQk5SygOaCwjhxNVOo6zlr77JfGKSBvs4F7PHaoqJC2dMM3EccCkeJeCZk3mS7xs8hEcLeBOuv\\\\\\\/XKJQxQL+HywvrNk9cTN8LnBpNl23XywPRhSgU0+7wQm6PyBqR6iJCczlyMKy4fN2k1WjIuut1GmTqq5NUk7XIYs5Bf3FdUqhz7l0kcMZN5bMijDSp9Fk5ftOh3BRqfINafXsW0LYShIcnB8z8s5RjWEeuO\\\\\\\/3\\\\\\\/VqKrC0nROspFauwDPjyjlVXjQ2hhqfglW7QyDQZlR0iLLfx09oXSe7GFXqD7ZKC02zLExhZs01gkOSFQWVa3JTWoHC60ugzKLudgXlM5eWHszAXUJWPDLFI5rpQEFVCgxkFnpd00t\\\\\\\/iMXH6ByLY4sha6XQ9vF1s1v9733gpooMqget23DF34GUuuuNJt0n33hxTIKaPQhhHKVan12OWkhOB7blCzt2fPvK6TnNRWMFj23HOOqNGw2\\\\\\\/0DHZbmc+OSttYJfW89LrqPtDO+voyMDjqjH7W0bJKSDJ3\\\\\\\/gcOJLb0Wi+QK3oimGMChKeEY\\\\\\\/mcWbdzyzCFAtYbwU5Bhr0Q4\\\\\\\/YDGWU3X8AX\\\\\\\/4L2VzGb4v3dxpnX0PoUgl0N1+GPHtIMJWWytGUaeoyupdZVpZmbo8qixNve846qZbXzM7KaPdmtES6nhMWuOYzuoNHcTghlXFOM2pVbkXkKnmgSLXpt3CDlfKzbuIkDUJK6Ly3w5Z4J2p3mLPcnrlaO+yipK1PlbKcRu3yefbj7o2IyUc\\\\\\\/yItrpo\\\\\\\/BYtdMSraOlMgbeov9flrJjS1Fr+qHkdRH5aATdIRPsGcsh8AIDHPj1NRESv4N\\\\\\\/QFrsz3xUfUCT\\\\\\\/cZpiStvX6iokntKsNQva38+TMFG07VuQFE\\\\\\\/9ZFsf1ng7Ge8epkA121ZC2+3kzNr39DH635nvR2cayUaiAUgdf03tv+eiFS53kep\\\\\\\/9ixv7cgv08zfNFIb0u+3frgoYn63jk2wFPF69y5x6XSlhZJIHH7om1B0rlYP0jv0xGAGT3trTbbAHh3\\\\\\\/177GnGjVznu\\\\\\\/YcEHFh7+mgwQViPw47v17cPR3aIqUWWIRnsY2JnWHC459rY3oVgiRfuN9+yRddiRFGVTJRxALZbC2l6J1oQHHdvKaJr5ps2huI7JzEflKvckx\\\\\\\/un\\\\\\\/E\\\\\\\/uE3Ny19Rbxf188ZUG6GaLm77u04jAMzod75\\\\\\\/daxOA+0T3Aaaze59cPdyCJFU3DcJ8TvTIxSBItAm7imdLIGZvBFRzXH+NUlQRLrZRolIYFC2xMj7OMknbnIGEiKXwA\\\\\\\/4NM4SRaGiDibyC\\\\\\\/qeLjHsrww43agyeeLINOJYQ9wrCY\\\\\\\/JSwIYfvDWPgV6SjLlriRgoO4UHaZDCNSdCj6TxOExqmGklqKaPkrPMXopJOWOuv3Ps5UQi7ZCMsQaf1CkbcjRW8yDqjmHIdzPjNR4psD9la8sRvy4yMo3s1XG7W4+u+RHrT1e3U\\\\\\\/fPuCAqBtSS5YbVrft+Y8DqpOJT+UB5Sb+VBFwM1CkaZQx65LNnVdrCuNj3wPlmjXmnPEtXOI+ZqhLC+8prKP6DDZqwMYfuQpehmd9+xk6eJDtwJfsgUdyueBStMBCDy72ajjp7IY5crMmx4cXKWyRDHwkvasFFiLmeXNAFPbtnFiC++jm1DdH\\\\\\\/ncvPWdyyKCHQjsqY4gEmiP9SAUQh7KeSvGTj7lhxq2NXsHBCUEMZWvtszoiEp0JADB7QLC+QlLBQ\\\\\\\/mYbk46c5VuEcr16Di8m99xLoTR4RtmFHAFytYd0RM7vJetp7H+Kbo5\\\\\\\/XUuuIY4P0mm5IwwoRJjVwvk1GjuJ3Nm3vznVN01kCQU+SICBYqF\\\\\\\/saLQrcbmhZXrtauKCWa8dGE3zSmqfD\\\\\\\/5v+SY5MdhtSddF1rljB+AonNJxLXzmQNfrhf9g4mwB3eVRx9oDxcRmYnylEbYGj+bqVqJMRhFJQhQDPRS4+ogvbBO66yQGOmXFhqQmjXhvLt3dAhLe1oK0GK31dR1jk5wT4OnAyECafUkzu2gR9Cb6A0kNSlfJ+Dbz6Xi3zOXNOtNLZ3GCT\\\\\\\/9bt8RjWWveay36+hxJnpRwRCz8nyo9PNlSgZgJKT3zWPdVBNmtLUtAAzn73qvihV77OL0NkO5y+K0Lq3js66w+4V1Z+HQ+J37nDijWV2v2l+BodYUqCKVVA85ZLm0UMYkMt577oPIL5XMfWYZvTYCFThKuwzMRMluBy7dNzsYsNfwsoQTSnigNkYgHRdue4xDlyfAIf3\\\\\\\/lZjUlSV6hZm\\\\\\\/luAQrRcTFQCc6MYBkqtZUlwXIPYU6exNhZETeGsIcipZ2a4wB8uJkRFDwzQ7Huy9BzmDTziU7dI4J5oHkdpMM4Y8Ehx7A0buxYg7Hzk9WPaQjTVPDrCACwosgFyyw1awdq9wZdKuB\\\\\\\/kgwl9xAZcysbt8b35sWxoJUHuBTDwhgz1x17Ob8zclnApCPtaQWuAO80qQd841DvJYsvOHXO3UiLzOdZ6JZftOv8sCeYsYt2IoKJr2f\\\\\\\/dncoD5DcvfhGditZdU9d+KVyHiHXvlMiYtYdw3eY1vRDE+1cep7w8MvBoBKDRAoqSDuRNYEXVcJfB\\\\\\\/r2lsZXUEzOhOyXqJfgnfOlMne+qO0+IzMlGKDdl2eBpH6Ukm616101wFdfgkmQd5swmDkFS1BVzxcGbSZHfBGDfoJW15ihS6ZVIPzwSgWhRf2Z3xHVqjZRUHukHAZSYx6ia036mQYeEuinWNMbC4AVM6QREVV32orN8mbEg5mnO7iMCzUQiBBk0H9yVFq87xaf81WuUGcJD7PxkbqUTbnCxzpD7ZkpYEglFCZof5EkuutyhOOPL48UMeKrNjWjwEzu4rept9aWKUZ\\\\\\\/wUT7bmrZ3jj6PJOvCWFebJKDYnPUvqgEhWnhjiZ\\\\\\\/p2QVHZ7XCkM38ynwWpHe6K5qpHcqMwj\\\\\\\/+T0UVASXVGTxF2giw3MnNlvBhXdGMoLRCovPbC5UkYfAqZc+aJb6ERSMJl4qt8Kb2L1WqILMoFS7LASlm\\\\\\\/RNm\\\\\\\/9xJRh1cCxkNbd7EykV5JJ249\\\\\\\/Ax3Ir5SWBKhrNms5Ka2q5vxGi\\\\\\\/qcB0L8xtfAsH\\\\\\\/8HEnWyasLofCYzaK+xwaImvgVPl+bs8fcAAyPTTeRXB+NGTbPEZm19z73dpri9\\\\\\\/w+ZEkikjaTPPvQFOFFauA+\\\\\\\/KBw4\\\\\\\/Q8DiwMiWOmPNjlZdFoxnqd63ztbcOQ9RMm7c9V9CJ40REbPHSYQBIFqFOl8glHS99ogCM\\\\\\\/JqA1+Az46PVdTbddv4G3v0xpUwq3kB5E4kA03\\\\\\\/UUCZbX7MpMJT3BtyX3mUpvn4YsgyWH6g6QKNs1yAGakvny\\\\\\\/FjBO0I8boGyon\\\\\\\/CPVEIYiqQ5BjSS+RKndEaN5+TY8wh79k4\\\\\\\/gIQ2zGEHEeFMGiOs+FJ\\\\\\\/a6b6XeuvTcbBtJwvKXU2c1V+wcurvh3o0cL7xr1e49V4Wnfzl1uW26IeFXjiCSmYPowRDxlHuoYf+Y8z+3A9ihF0GSDWBWVWL0a4a4ZJWOt\\\\\\\/3kMudUkF9n+es3yo7SCQY1DrDrNC\\\\\\\/aqAZkekaR\\\\\\\/BjO7utndFHVY2bGYuH6U9JkBdhaGVmzLHBuW\\\\\\\/op9Q0PO7eB8j9sggcnrlO1xrEK8xCVfB4BrUCDFGrAP8LJBTBn\\\\\\\/rXKggm7pCS0xM03V7+37IoT\\\\\\\/1rSXVC4z9hOKsVbW\\\\\\\/qt0btKJg0a1+\\\\\\\/sJf5MAdfuDhZnfZJUbROo4DB5Qmjm\\\\\\\/IfMJqUL+pKXYe9O9B7Q2uNthcto+ONCtx2+9H80uMdpwy71BihOAjw7aktG1ft5SU0ObDSE9LVBfWMWLNfX6lzIUnrdcHZr5Ur2Bl4uRbJsXKeqeqIZ3nEFDUcs+7riYVPgdH4eaHpEbyjuKCm1yhJPKd\\\\\\\/8n1W7rpZCvCmiOXy9vfXjCiq+IO0mjQmzbpLw\\\\\\\/B3z2FGsbpHg3r0m2dYi0v+65CBeHS456eekSTN2\\\\\\\/JjU0u8DMdfJRuV2bClqH2b1W4i0thlxwTpAy8v\\\\\\\/1ZLyUJkUjgKkKmZLXJsdIb7LAxgwRc770PeSQky0lpkkgakuSzNRfG9A4TVtzpOKogCI2GleAyux8hwM20z1JKfGMbg8DRoj0fDAPgXZk+Z\\\\\\\/V55UnV9ykfoXLBGbPSEjuOb9xbjonMKjEz3jSqxfKx0pFH6aft56lV7BM50gs94wH\\\\\\\/SIgFVBj+PEC\\\\\\\/sfRtu9nIYgzbHw9aQKzhZB93aXoAgkzFyF717iXwGqy3GmRSb1oOVcPEc7bWkoeCYopNtLjyqOQ\\\\\\\/gmYQC\\\\\\\/t6gQfxhtsju1MjdpHe13CbnMlGPYL+rhUMslLl4sUbOgzho+Ri82+CiBsMrONI2og+yu+HNTeanjMBZERLkyZQMnTvYtNwAVwPrdBFenkTisNrbTalGUJ+lwkPBT4ivWsLMhjUVH198XRmPQ+7L6fLU0K3hnQMYk+zbWtD8A1T3Ds2Tl3ChEcgZ2zFrY2KTEKisTmsDPH62PoYZdUiMKMy+pvGnCWzb\\\\\\\/+PaBdiVsYBUa1Gj\\\\\\\/VoDBl\\\\\\\/ikwweA48eY5DYzPVPo8VfLySUKidUVVBdEukWVr1Idq47R3hD6wiqMG6\\\\\\\/BsXzfTavqX3dVHLuQ+Rwn9gySm3GGzPP+wiy\\\\\\\/mffUpVEx4SCU9K2sBmilmbfJfIxYNpXD1fmz1NxEkZ6xgFQHFev9GHIJBSTjfFVAEBrl0BenDrW1y4TW0+LxYjluOcb6qZ\\\\\\\/ER6\\\\\\\/Ni7wZimGbVTG+hwHlWSZCt3TQwF5br0ICaURFvJ1nS6iO+G1UpNtwPtCMAUpd4r1DgFIAfH7UC0IQB3tr6EhTtkidEuxHd7rQbf\\\\\\\/\\\\\\\/N1f4pW62IU1dD+YWKmNwjMq\\\\\\\/lyOhvaxZu+sedUhX2fQRy0U0\\\\\\\/yWtx96NrvICsmMve7i8H+c\\\\\\\/WiMZnlFfj6AM8BC0D2KGOeXvofAJy9YPsneehQb6PmFYEY\\\\\\\/TqJDI05e8M0kiliXFg8d0HnDEMKZBDO9xVr7pEgpgba37n12IHupE27MLMm2O11EQzFdF9VG+NzhT3ePRfg9OuQQzB5qw24SGJiEiBbOb05u5w0GLCIW5tCOvJidZ5SPj4kC0MALTsqyJkqFzj0FL0ktplIExc0oDs2DBWI3rrW\\\\\\\/9qy2Gmk7AXLbiyipaQIA53A7XxdnXMoZzU7gPytTvSgwBcHT02Iwr7rjhV54f9ZRjY352Z2ig\\\\\\\/nSyQRk1e83IMdM2oeNq4fWouTbXHanwlkxHYyGIPOcQgnZtz0f7ZurXjkZd9nxbjDBnKH1MaLmVH80sqNDyB4wGEou2xnpvzoHwimCBZIs\\\\\\\/GrVSjCfJIaupuR3QhoRKPl6UNB8NkIX6GODmn2X1Qs+uQAmiNG2KuXFNcNqkk6RnLHnIxZHLj46dfXPbsDV7\\\\\\\/m8v8hBakRaZv85fvO+szM3G08zhMkH7DDudSa8PJd6g+oOMl804775ovE6LK+ebVdbe37JgEKll+AH07yrV2ObqS+DS3KX51JNljSVVC\\\\\\\/RaqwwnPklCyuvlrPnbUXDfInsnA9spgGq0+ccJI3rda51FrROBO8Xw3sk7t0EAW6h224ACcZF+90msPAKuXpgLBw7SVaihI1el+rCa8UyAx0UvP+bo0Wfu5w7PthJnZrI2knUR3V4SXqkxEvHMhpcyNsfAjUbPeEbwnraN0igwN7Bte0zhpysxNtpN+jTYoC1N5dqJYnbrcr6FIo5TUcSJ0ZVfdwx8DHD5LZGv6Y+5Tc3RIK+JhZyER6lhZO9qnusv4KMc8kw8JWg7EujMOjwGzpveI2lP1xpcUUv+llK0KOFbVEzmZlvZhlo6M\\\\\\\/7HF2jWjb9KzkyLaVVa\\\\\\\/K6LtdhbDNaXJw3\\\\\\\/\\\\\\\/DJ7Xh6+m3YyxXhxmaMEGZsJe2siJPUeB8TQOlWuWZiU0frZkma1Edi4HXe1SiEV17uEAni8rN3OKq9l9iROs3VszN7VctPmrKS3y9MtxND2P+G57iOhTMZpN0G7CeBmREAPc1re2EtYPZEAvDilLb2Y5ByY4qEUj6TKT6kJIo9\\\\\\\/Tmpz6eQBQ6SxTkF6WBYzfdWWrqZ1hxBTw24tz9SzPjXhFhIvfqWTzUD3jvQ5enGFocPpIikfhDIk75Lq4nhy25oWyrV\\\\\\\/IvDw9i4Oq6uHsY+TEFQq1Qa33Ou+IbxKN+I0gjARG2f6w7NF+I5hPXepDGgwEwaAAReEMW+6gK6RyFxaenIJA3hKAaE8NNkyuwmlJEyQ4x20sNmWZx4pibMP0+I1EP2hB7ogpZTxr8YkTsxiDw4k1zeLrQUAKXNteS4dBNKgBOo55+AMQAmEpIIgdS3ujFzoMeBW+Nb2d\\\\\\\/uB1LZH3r3xZqVCz2nCde9wyjdvr6aVb\\\\\\\/3ohSNHM\\\\\\\/fRPjl+9W1Xq03S3I+lzbKM\\\\\\\/xvW6NqmgI5kshk1VzkzqEw8937pj4Dn29HtPNEMD607HDZiJxaQlEIw8zkhaqMkt+7RITidDyYf3kKWuPFnoLFc82TEmaReIwi5\\\\\\\/VYH\\\\\\\/9MgG7Lo7sTpbvyDqo6yESkU6uoMcal1Yq\\\\\\\/lB5t3JZjqit8mzomHzi9YHwiyg+A7YAdQ7RDY221h7nB2wkmW9A\\\\\\\/9VZTgcNpGQkX0wmPefqzM8vkXG4I5AFWQ\\\\\\\/8wlOoK9UfPwuE9g63k62okiURUVUsSP3l9tljD5ZBbuK04OdN8YKMwZgq15KGJWHNUUIDwyIZWKme3t0W0DuD7rfXYh1+8ZokfoxyjPv\\\\\\\/XDh8kJYR4asHuk1gz+YAkBM6iGSYaKzx3BQybftNl9HtSDtJM07dCDeZoDFtc+JegMyB6r+SwtjvXUgWC2EU7MKkl3nWMv1NKr21a\\\\\\\/coCvEQuVr3uk8rBsnQ5MxM6i2BmEcidUXASVz8EClQEj9NOMpb9maKFVSZfBHEj3Qc3EW++l4z2kTsirT0T8MMZ4cE+nV7df73FOR34zUdN\\\\\\\/7At8e\\\\\\\/809IqkEfaEyZ5UTo7UzM\\\\\\\/7EBPgNvxx1CVkyVCubUwxGmK92krKKlSN49V7ASgNL9tu9yJTnh3R++xYyAm0ZeO9ep01l397SOcV7PyaMWdJciUaMqqzfIwS5SupYKeDDNsCt4gKFKQmm\\\\\\\/AD95tPg1+31rcgjaxxq600r1E3\\\\\\\/H\\\\\\\/iN9urXPK7s89yi4pBZpraOb1OumtmXmhTvy5l1KXJj512bjbTyAbtH7A4lWTZ4ZXjjSbxIJ3cscrAvVq62JybWuwhMAe1Ap9tN8nCl8ZyR+POlH3kpHdtXnIrrUwh6FVPEahR4qN39Hkag0tERaDfiABgdUCY3+qMpTjIRGS8Mlw12jbJ121rk0biw6jXynuDrfP8X61B85+NpC0LYnn1SMzti2Y9Ix\\\\\\\/wifbEcJS6iLGDdSeVXLbie\\\\\\\/rp5Cwjl6cHU18a+naq1FPOIAif5XlTgMotBU\\\\\\\/8PTnM6SIcZI1xRnsimsahKHzzlSUn+4sTXQYONtpqtiRc7tmuRkGuO6DPSUIXW65kXxqyFA4CpYE7OahKhVMMPFlAYwjQx4RSkp436sz2rbM\\\\\\\/WR2go3f8tLdZfmmzWwamuKnwpaA5zJ8CVMKQvoPLFDO4IvAcYs8d2PO9hdZvbta0j\\\\\\\/TalJldmjk6deRfneU8oVx7lPn8wExSBeUscGkCoxJ2ar65ryXmKzB9y6JlRFbOQzRZMqjc2vW5K6JiMN0sthxEBOd51YYv715lPuskx2r\\\\\\\/vj4Ub9JLY9SkB14lv4iisxY80HTUHlBQYB\\\\\\\/XhP4KLnoNYPCxi9FAczlXoBPlIdmSXh5tAQZpXw8y9mpdo1idh1AqZgI\\\\\\\/NjdHDV3g80iltqTFoPwag9Zmsyx4bU8BP18wBmwz7778mgS+AUqg+lpXF4+dluOe26aLReceKRN80kx3FIGKyHr5VN1NsFa3vT+IhV93k4mhqFNomNv41F0JzQCpidJKZHnzNs6pUw0oNQ+lTpS\\\\\\\/s48+Ax8CJrqXlyKIQJ9uyQO6JLaTiaJtDJJVUITMrpVBVSk+RHXmL\\\\\\\/u3+p8qLSFrMxbyQyMcW5+fwt2KQ2OxtmesKXKZi+F2kFgSOrFKGN70RdfxEPU89XS5FWKIJrSvfHjOrulUazo4vFf7IzRvr95UJduoNyZ9fC6aKvOBEDyRayTejzWitqhjGPfqensNzvRHZQWMl+fXQrlBT3sCPz7t1z3S5tzT48beDNEe20gFlkA5iaLDb58V5c6+HldilBi4VB\\\\\\\/DGDht8KGy4RNle\\\\\\\/1Dw0KnL1K1xf8pwVG2TqYrVBK4Lvmm+eGAmDau3Ds5X0\\\\\\\/jPecZw91Cc6+yxhpJ8cDsZHZbv02mFPOfOYg50z\\\\\\\/SnPE0FkZQPyVdlmLdJRT8U3Syilrlts9kFLH0szRhSnh3xRJMtsFa5UJev3qsBLiid8c35YKlf28Bf0ARw7Xl60\\\\\\\/qtfVKgcIabekxxt3V3ydXvAOy+7e5sdgaIpnKYNX377l0omlQ7aoE\\\\\\\/\\\\\\\/ngecEz9jdY\\\\\\\/w\\\\\\\/NdICVN0WBAPE+9m1PP3qqtap6Cz+z\\\\\\\/gNgDYHzUuQ1MXvRtTwUKaSHtZC1AF+8agRxieCMGKGFDGp\\\\\\\/UZSE+3r\\\\\\\/w7V2+PVtDRjIZRmfC1neYa5wZiLp3WMgQb09d3SM2pOsOMXNecXisaXO1oClmKCvn29Fl7qdwslLcjZVeSjnQTDRGmRh5e3lBysq+hRpBS1d7DpPbxmApqwe4n3UijqlRPGFyfszetBMnuQPAwzecpxl2LPSRXpVB4CHY37zn\\\\\\\/yiezKvzMZRRb6PRdZHQ8vbOYLpWf8W5jVJe95kd2u++5CwRbnOmXb9vPPPHRwD5TUB7oDmue5rLlJ2qPiNp5HvCjm6WwD5AXnNsokyAq7\\\\\\\/0cMamSTIqGuL59oRemKZi4N85uVxPBWn64aSTGpa8+pR9QLvlp7xjhbCnyRGG+ER3uKJStKuS7LGwl00VF1GtveuYoDHzfFXbZL0ZWHIzVvYId8rA5amUPAlvWJwnMcGIxsAgrlTrMD2\\\\\\\/GN2PKPWTf4EU8OWvzTC0G7FHwmdvGt7UC7Dt48PoYzhnjBdaMkK1LbAJvwm55j7uyo5vUsmLe3Fcgpey+qXSBPRHbel14dfQpZt775+ufOzMP\\\\\\\/mtQtrC6k+IVNWx9b3HSfRIrBBSovR96gs3dP0HDjHFZGaj5N8G+\\\\\\\/mKzLWhjdjXphfjMHRsVlZ2Zl55WL4tawRiWdgJznDS9pRQpG0KWzo1QUSzeATUJNx6fouMR2sKOELEIjJDjtUu8OUk4cIi7707k\\\\\\\/HbuX6hStwTArfS7wobTxoME\\\\\\\/YBegPXTjfRSaY\\\\\\\/UFB2bBV33hGGiACKxWI9\\\\\\\/pcW5JKYf0+twUJfsXYJ\\\\\\\/TBvA\\\\\\\/xy6F5PsYIcy+huA1UsWtlgDjHI+1xTJZGlMq4j6zT1agsMEpvgUR0QSaL+feNlPnpoqyLPOlePNis1BZ0rV5JOXlIFp03z\\\\\\\/cFgMZhQiwyOz4cea3IA0rXX2ERLgR3Jb0e2S2yAc2pDwajAyWLtkq69ON76hPSIje+x15Gow0bC3mJgc\\\\\\\/tV\\\\\\\/T1boJKEZZyXqnHSOhu+7M8wNTjvN8lKz4I2p\\\\\\\/QVxxDhk4HIdZZqDmgTKUT8TJ3GW4LDivjMUI1Af0s\\\\\\\/VzHvn73Tz\\\\\\\/D0xENuYC+NpJ4uxvHpd54I7E025Ba6raufSL9k8LeiZ7IEdp2iKSgGX16pS+lo0Wg7MnnIVm8JHG1uhhk\\\\\\\/7WOCduyQppNheNEu6zeUEkBWXPh7\\\\\\\/PAAgwzVK2W38ZRCrI3IS6K1oaR\\\\\\\/7SYMZCGKOZ6S9Z4QCdPZFJZF9jSnTLd83Akv5nLQ5CbW73ZOjXx64xXGzKNK41w\\\\\\\/R8EUYpMEroOj29NbKu3C8k5IZOTSCQhOL\\\\\\\/j2XsNi\\\\\\\/SLL3uOE7OE\\\\\\\/QhG5XB0O0ikHbHTg49Rw7s0dzkLI\\\\\\\/u0mJdDVIkj+jJ6HJHCPvlnqv8wtBloENnCotCCYwXI7JjCwO5ZTL0n5Pp1fM+SL9Bh6Jj9vWOo6ZAynMrSHKl1JdtpXeCR63ceYLwicIxSuu00yUHRcda5q94KReNF92XmFw7t0lR2OwTUkqmnzu+QsjML248zaTocXG+qzCVdTt45sxVmgvh8FMEsetNubMPYLL9R1GBKzaJgQQ1lwH99DO4Dc8SXYfTZw4W12ekAbJ0gplnnYutO2KGZ3Vo2C6IUlF2yAA0OXb5P5gXY5ldkFM4J0q5GwJGNCbj0ssxa7Ozv\\\\\\\/USrB8kuSY9jiqGvZe6tnOY23uGZJ7VJO6U6tKIszll10hT2n9p7Ynik6+ndb8j+xryPOPOhb54pO+EDYJ\\\\\\\/FFlUQZaYIbA+jOzLfrZSJr4LjL\\\\\\\/11DcVfghleZPoT7kk\\\\\\\/Gksygfqw8xr0h6B4klXjz1T6OS7fEDFLVujVH9fQPjUaCjEpc++XIm2I4lRO\\\\\\\/O4Ud+Y4VnyRREvQz1pi1E6sMNqrABifFNl9WWPVDYW+7A\\\\\\\/6xHGLusaKqGuXIs3ypvimm0OWZxoQ33g5JHsU2kQKtE3TshSK6gC9tE+HQPU1Jga+5Fo6OelcyNmlPbpWr1fUreTf+ISis4Ak2a\\\\\\\/ObBxTSg1v05W7zorQQzJj6DVix\\\\\\\/m9xxlACZ\\\\\\\/XlC+OdEKnp0pSP4X+HehiZlTLtzZFiWYWoWMpXw201GTeyH3e36HiZPwSOpG2OKOX4auHEatDNUPF11DpyQX6dM3OP\\\\\\\/LqaAYUSJeoMq821V6pbDeMbVXZlRzxumLygH7kSG6IwjtSN64Ol6T9Eit3ycAE0HYNfIU4qYm9uOaNiE3Tsk2oltL8WzuF2Fr\\\\\\\/Pbis7ItEZufTSd8Pqq3Nur+ceExSNQGU0fRNJUieqiTj6oT+teXRXrrvbGMcBUog\\\\\\\/+AKN7jMeKuRAVzVdmGLlwu\\\\\\\/6gOlC6O8E25NjGZjfqw5szlcjzmuyMrWqziGxTdZOUc8xPNve2OcAwlXF1CW2UwDhi3xugSKR12VieEAWRjuVsnWNPmMS+z\\\\\\\/tzyOfwNiXb9rapKYfjeyJturxF+Snh5pG0P+nNglgCFFoncua3vPNA9p7vKvfRrhGbSLwX6wCIfsr+WnMIDq6\\\\\\\/adqah3E38gzGmEZk8yiMXUu06Gw987khUkIhR36ZAFLsTZDC0N5F4jvzjtOTy01UjI0N2yeZW6dtwTCDLJS4JbR\\\\\\\/1T\\\\\\\/3FXjAS9Y0kGEbII\\\\\\\/25fH6W1mfLzWw5vqWP5Wgib6p+GDruKbVE2d2YtPBEDPs6Er4K2FIFGmWJYURalOD4PYjR68HHTWrZLewvw3bXBKdAcY2ubomaa9corBKrSl9JaX+cGjQ4kJT3\\\\\\\/E4aSer2lfPdX2pnUNJgJ0X3Cev4phcKKwuCbhv7\\\\\\\/iQSmTDtz1pU6n5h4LGGuOHIpBSu+hnPqstzU4T21TkslsjFoe4YJpN6axV6F\\\\\\\/WUEDcGGb6kaBEG\\\\\\\/\\\\\\\/fzB0JdOpqk8i6o4\\\\\\\/KYAiqq\\\\\\\/I7pNRviD9HuG1CjrARRUsD2fei+tIyA+vyipc8dWafBUhAYSjC+2PZBkvHPzEKsnWKTIA263\\\\\\\/vQen01hNqMEZm\\\\\\\/aMI5RFem18+DsWIM7ZBBUtlWsPOMm\\\\\\\/8YCwlmGQAOSn9USCMqUW1q3Blct5c28JvKIOoKn8zBeRu60nCziDa2xPy8QA5U2\\\\\\\/\\\\\\\/2yssxuPEftz2vR5gUgMUPt6HKdsbChMMsOi93k6oMZ+pI96HRw8OoZrU1sseCzhVKnFBSDgJHJtx7AOUTS\\\\\\\/AJGypyZjxROtvyBMNwwbhbpyZrdlyVF9RyFAXTLDL71NQ1keKPWHMm+Bt9q2+ZRGuJvm+4zef1vw\\\\\\\/wXkC1WdypBOfwHvsPinkbRG7MUFn7BEHrmchXWAq697osC6zvLEXcUZ9sFeCWNy9P\\\\\\\/dujVvzW3gjHsnUHjWdncmrpdb2v5ivXVQWVNF+CGVMqqmLClQmjdsonBcDZ1CsjU5VV8kMNKwFU48YxKHfTc3vKIcf8D5D4USoJBnNI++TJZNLZO41hkWJYJKasuFEUkRaWD52LRzH2jkcwoUaQRA7ZfiAcs4ndfjBArLkMetdjB\\\\\\\/vvCsQZvzvsfNQz4jmVkItO1ln8ZaCROwMmcQQMV5buEvZKKx63dgRNgS0LyeWhzlC1yPLWDn8MwXSu2ci\\\\\\\/oNNDoqHT8Kul43Z2ezdQ46o9N7Hks97FfC8W7x2Tw5kgCijGyeyEHW8OXhcCASjFkYSVFNhYiVx3y0etMNiXArXLw2Ua57iu7eX9emJAk+Sm1J41XwZ\\\\\\\/Vr4VV5W9moaCHlgfijhbIL4cOvrwNoqdsGHJVnIx+y0x7d0kTxbBDbRhZ7L4WhzCH7+fgXfTm0lurWSHHvL9t6Ut8wQ0pJH2+84pWd6ex593ee3hh1VR1ZLAE8BiZxi\\\\\\\/xhCokaBJqEAICLZdGkYi7FD\\\\\\\/ybyguFPF+48syCwwt8z7qXSWzEHzSLIHNHwDUObVBI\\\\\\\/o6EqUjJ7v\\\\\\\/VxKUrl6WVPEp+T4Gr4Z\\\\\\\/dXJec9kpSTS2YsfT0UCAlvWY3BRaAvMuKr2Uw9uxVx4\\\\\\\/rQ+une\\\\\\\/vNOntxdAC6I0T4O7xnTlcIv\\\\\\\/RfwKTN3wn80dxP7AjJk5qv8lNAF5u+XmmtA09nEhds8M768vk5OSiew\\\\\\\/AbNiol48JJYNfs6hYn4KaONVq2eVWbdOWncW6nnN8+QOzvuRl\\\\\\\/65e60PENRgPR8FKSZ7td+XGveNS0oaHo0kZSgmJhJZRCoS\\\\\\\/lgFTyU6Jzt14em+QH8UnXk69HiIbWj685doVD8jDWWRlVt45tH3sNSBX37giNKYXM10yeymV+DYZqc7qdDxBpwB+ruRlrsD7Yw2hmhfpIVWgoVHzk5Yb8PWFUJQIyM+TI+yX9JX6RShbMVgYh5Mbrk\\\\\\\/1OpvOjCUg\\\\\\\/spM6cgHxCg1YZpwJrW85i5+ZU3CcR6ujwcAkMEoZMIrmcu9Y0xuTvYF1pk2gFFOfuakrfz\\\\\\\/S7pyWWQq5Hnt3VLfnafnR8zMhnIjBnVxQRpmUGsccJN+t\\\\\\\/vB0MGHDd+7N0rWsiI6O78K37\\\\\\\/wuKfg6bruWKpxAiKusiy8WCV8RxfzuId+OOEok0G50ar17uU7u111ISWkWt8kMrsG3w+3yfX\\\\\\\/zIHxtW6LVK1YiQVoXMcBedN4Mk\\\\\\\/mOoQjwxiuzworqroTRQWVqjaio16cJd1Hdg\\\\\\\/oxUsfoQf94HBhXmlPT2XVLdCCKtNnWP54kA74ji6MxpHYe8NvmyaSGdup1JveTzLo5siyRtbgM5zEGcErK5MgcCLbwGvYEMTyXw96HcINGuyUKv+IQ2p4qirnaXB2dgv9dGd4oEU3HVGufKS8MhLiXDliS+v02V492llOikiD7WDNS4STbg5ish28GIx46eQWV6niCQQtZoGmyRw5zOKBhNm1ovZ\\\\\\\/fVnmgW1kJB6T4oXmKhzMtpHjexI0EW4KJ2HsrhCLcrqd6KhHOM6emlcXcxdwY6KpozrPshHVgbAN4nV4l6vxUa\\\\\\\/vlQv+ssLYbEcmyHl2AGRvygBRbZ83+hk82tpewSxD8GPutnToXNiw6Qq0\\\\\\\/tQ4gUYUoI1gEB+YKDqbOp9p9v6SrptHyYYj0dzybsXfGvSYDvQsIjoIR70r5oIcke4Z3tjJpvGOIU40tejj8u8Rka3jpdupEMINtkVvhxx6OOTRmXGOL27byVGaov6HjWA\\\\\\\/01mTcPJGWpMuaScR\\\\\\\/tDTvLxzol0\\\\\\\/EM2ouCFAPN6G8kFBJX0Cn0SgwKaUa4rUEnUPXXz8v7+Maok6bmMxU1cF3i5wn0ztUjWYqIE8kClFPm3BjJZ\\\\\\\/xkS1DuvQIFoh6t1ycqvYNMoUJm58ktX9m1O5IL3VoSA4Jrpu50gYLVqXgltFwkQSmK94R9WDeuOZiJT+M++l7Z7k\\\\\\\/UaswwfkQ2ZEcfX5ssDiC7wfhz1D1GvmG4eRtmIiePjmuHgxe6eGPoc12u95ZNL8s0E3qTu72H9xe36FDp65BKWMU\\\\\\\/8qO1UGBiiWiCnrU9xfkqKyNDA9yqa1iIkPycgShHDOC4SQGZ3r7KgjgGnwatINQcFY299YhbF+E6oSQ6BmSZtdu49s8SiNkZIbll9gyfms3yCVsWka43pdedRV6TZtK0ZmqseFJLCN\\\\\\\/B\\\\\\\/SQVpaweH9GEm0ryGdaRkcc5rABogwtAjLEjbqda74vtfDWSq3DnRyGpK21xHrg2ak+akVUrvBb+lcuEuphj\\\\\\\/vnGzGEW9\\\\\\\/A2spkRGRN9qASqBq35I+Gg\\\\\\\/0Ouq24IONttxSiRwlsBVQs+KfuPjxtqfVQBxCDXTpeeaOp9QM5WmsMy5TsdFTBkzvjfByRuIf6w8f2+gUV6Fj62\\\\\\\/IWZf2QKjELSM\\\\\\\/5jCKBiYNd+u9bvVUKxFmMWKA\\\\\\\/hvufr0lG4oGm\\\\\\\/mKKg+xw7zBezflhfx8ly36YMPwzWdsQghyHBZNxcklXjgdfcqaDPp1UMODPEc8SB7kYMyB56JGyIBIEvk\\\\\\\/x2HkWZYJLVny0fRKQ3bU3ZitGjvPLB7uidMhfhz7uPU1f4\\\\\\\/uZCu97Hmt0m8i7GsDNNmCq+WYfzhxGRfUNQq+fW0L1IqRM8RICvgwUXr05jOvLkEggcmCgkq0BnReh+96dn2Pvj1wY3Dq0niNlF\\\\\\\/0MRQUIldZiFAZoqAQ0DR+hCAQwLTnW59UiO2cUd8YrQOL\\\\\\\/7yuwxp9O5W2R685YyC7007bdfqKn9JjVrvMHteK1lKIGlQXZS6FI+PMMa6gfWXbE82aed0BYXYm7FHGzzpFxlAYg+DGYKeGbp\\\\\\\/joM+IqdPqS1qww3kA5ugaRlqejsZMzYmkW+\\\\\\\/iEgXk80x3TCTtiWJjmvtA1Mo6\\\\\\\/kt+kY9nh9CwweJrE8earjkFrtr6YvkABktlktvi\\\\\\\/iw3GI5EXFPqe3hCR4eafcRCOtOD+4tJIi5TbFy5WIev63MZjosAhtc42AIJZ2M87QUipsRz9yIccHleNoAk0JCJJqjdkPiStKcMsEFA8YOlEJFWYMgdEULCwrcrx4iqtPaL2v0ORG4bXRL8bvFFD6xVdGuOtHNiBjY8P+FruixGHHZ6Ts3e8yAI\\\\\\\/GrR1rBdpX7DQ5RW5TIRWSX9yKuPs30b7ksbTm98KRAmGk9svoGEI7sNCyWot8qSAsxB9a3SRgnQa7BjAm+t45U6emtzW8hvQvYXuurznWFU6D9MzVbo5wunkl5B1znbloCAtZArf9ij64sU8SdRC7ovT531DJl2SnNX0cPCh48CS94LeBmObl81s5+bzFm+ncc+3ZIVB8yVwM42pSSGYSAYKZnKZ49DRrSbSnCtqnfyjhTSIADsso\\\\\\\/jOi\\\\\\\/RLndugm8pYo1xBOlvx3u9qjknfET0wsrczpmEV0vcYiV0qv+AxeppMNoiavWTF\\\\\\\/FhNJosKT3LqLiTqIzgke+uN4YhNHJQ7Z6gFYEvD\\\\\\\/hytCQX2luRF8b1Rvmb+hIVbcb8v05PJqwAocflb+9KPB7jn77SwRdsUtVwpTPIGkoY+SjuJfjm\\\\\\\/N+XGWxpLelY2aB8epKtHkfgYQuMT3Urg5X7M9ovAmFzujeDsg7wl9wkeXPGdTxBgCvbWtUARXZg8D2U7cVDYwx5coJWI68aFPppNNf5LC9DezMoFEtfBazw5sZ0Jy4tgN\\\\\\\/OT3jrLaVvfvV7uoeo1rueW9S96uYosNMzacVGFKOwtUIw4HgvAIrmst4Po6iD34KVhqtl4BE7JTqSnu7ZcbSzSrVtQOGTTXQENIiVaDblKxxxzxupEIK43+D4+6P\\\\\\\/Kx0rp+3VTHO0jMGH5QQtSpmnD2+b7+82ok+jHYrvwoEB\\\\\\\/Z7zgfh7Y6qd2PeRhTg\\\\\\\/W0Q\\\\\\\/8efIhtt8dbstuItYcBxoCtog2ehACXw\\\\\\\/27x2swdQ7aj+bd55z17zZCqfqqdWiHhRzOi8ciMAR7Bx1yvilYigLG5O\\\\\\\/QpV7ygoeNuftgRl8NxVR2qaQajAs3\\\\\\\/XnnOslO1ATE7XVlLRwj+7q6OT\\\\\\\/dhZDKAV6zybZB3ki6dUikTUfynyOONYd9CmJ9VwSMeYrHYlNa7DzEkZ6TEuykVoKzYw0\\\\\\\/rEfYoqkWhfEzG2hfMtTG7oDObk0NdZsW4wnsvnSIsuqFxOD9qUoDSa32gk+kGOb0JzgI3Ri7JQCP2HGYPE06Y8biAphgEjuBfp9m8D3LPpfnz\\\\\\\/4n5t+CjHXxaJCLANsmnD46jHZoJoKPP9NF7fpa9onYP0+Huon7kX9nuAjLcFZiSB6egqV0xFXGXmIZk4uJh\\\\\\\/swmXxyzHS0pAIf6BzyGOb31HrHVsg38FTIYAbgZgXEOW0JJS1BD77KHV8IUqH0y4YWdy8br7XA+\\\\\\\/Oi3x7wZcpizJNSjkDimpnSWLb8UizeJ\\\\\\\/NPL8rMAzvrr5eRSMGGBKBP7tfRroB2PqRcJE4ele2AW25mk1MhUEDO0uRYauQmVOvKhgw2zRZyaTAe\\\\\\\/x\\\\\\\/YVjQpHPb7n1VpGCbFMsx70L9YgMzqPOAdqGhhaQNRfndlQAvrsgnxoIQUxiPz3+T3zptiB6HshmualmrU+0zbkT8IpYy7zqsXXWjYE+wjnLZGbdR6r7IAxQMDdB0gEyClZ7LzBHgqRyIUs+dZda3K1Z+e\\\\\\\/exMKuDk+yPD90DGeI8FlfCyLRCrz11IEcjTl1dfvWk\\\\\\\/Jjfl0n3LO1kAbbDEqR10dpCGb7aKMaV+8BSHeXvcpmTlJVEfYJlZh5d+Lq5DnG9+D1G0\\\\\\\/Ek8PhgTsCBn8xCTKNzkqTKad\\\\\\\/R\\\\\\\/8T00ljL\\\\\\\/LwSwgBAMw9aRqn1VurVvRAsvwnBElwHOy5zQD3uFHH+s5ucCn3lTxq903w8FajZuFXLA\\\\\\\/amzx6r4WFWsq8m+pjTguv8r9E5K69E+Ho1z0xbUkEDfnnlam88iBbSFB0K349JwPE6FQR23JS\\\\\\\/5r8ztwM+Pbz3+Kxi8FFtVHHYuYQaXQ2b+EIAXIvoCFGOneeags++E4dEAOfdnnvpnd46\\\\\\\/0Fjt4o8xyXY82BocXc6P0PZIhYd2oI7sl5eEmgIpjOa2hMFuft08NtGN\\\\\\\/n7rJefM4ETVA+v5cECxbUAIbgkBmLXXWuukW6GbdQ+fkqfRRbjTMuwM1vmKz0c3W8058UbusBrMuEPjYymyPtBF4cL30LEHFG2dUgX60U+u07Rl6hOcqyXmRWpkm38VzayOJWEcKJsmTH7jOKQonQgMtM5rg3b2gX9JNXJrAG069aMlEpKK2NqhnCKX8FH3N7WoEwOPfj04Le5BOy305ZJ89Ti9gcWATcfiEU4FeF0U+9AuKNZWKJnFEp5opfRJxxDJLp2nHUDiL66tUGcKVH0gXiLuzmutAKpFXOxyG27a9kdol3c6GWtvkzCWoGtu42g+Zy8XjdCS4X4dmOH8e36uRaSUgJjpCjh0coHq1kHjnrUyPLB4scATWwOz4Be1SotUrN9NLDH5dZG86t+iWsl34TPU6QRyqgJq2oqbYRNQG8SuzvaMcOWtZKKjYkRe6IuN7dvKbaMVn6eL3WsVNo\\\\\\\/WMsdX2sAR8IYo2puf4\\\\\\\/9sN0TuTjwbirsyl3LO+ecas2iFLTTVrjrYhfBmZWkPtjuniPZaWCxhEtm+leW3Se0n2zJW6Q1tGpeKGOvJzTiJVIG8KnT1diNgqI1sfsFUTpSE7Qc6UzUYzHZicKGE6SelB89G8rjImrxU0pZpnYFU4oTAnHb9RVG1rHAqRoSQ\\\\\\\/rop7\\\\\\\/K32n558JoqdBUItuRQtYQpcA8xMkpe6rTeohX4HA682BEIGNzfzEl+XPb15U21T1VLXAZZ0lSodo6VblPRvLHKteWpw6ZfYCYU2tvX6t2FiUsTFczPIrulJhGUZn+WSQQU4aryH8ndkcW9y6U4ONlbR1zsKYaJZ079sl9iDfKpIhvfTYYQ6yGAP90uaeWUYXzHF+82Y3zwZtXFqriTutgQZuO1UaeqlQVLeqPZToe94CzrAwQctXTWIXZl+OehTkAexFhRYbzZg5u5DQXdNg6LY9EPZqyaMZ1wOFYRbRK0KFPZEgBMrD4ik42L0pPhOuNeR6Gr2EMVqMIlKOhJrEOEpa6uHYprcxhrOF6osxAfrFC1fsFeRyB14IJntQ8CiugjDKVUPpSpB4yr6ffNYqVHjdFFuqduWGN\\\\\\\/HdH3yMlt7c8rpHmGKn8p+5To9DOEMcIuqzZJ78Ycf6FUh0cpUEa7E6GS16jhK3g9KkY6xNiqD+gZ1t1NAMKYVVPw9MsXd6Q3a9Yg1kRuegVcJxIPKXLNdAS1B91geIOUNV9YafZbmSpnr1zZyoR4VoVw==\\\u0022,\\\u0022iv\\\u0022:\\\u00228AcykZx0qDGMK8zAROOXkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222e53b176020063859b15516593236f028f048c6bdbb22be4c8c800e8dda51684\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vP+h45QGfl0j2D1Ta8y\\\\\\\/vSjizLVM4dtVfq07ONf3NKcX+C3noACNWSK92x4V1KgRvQuwfIL3UCYM9Rhkg52bLmG\\\\\\\/P9aZ6iKTmVQPy40nFL4h558s1mJfPzXhrcGmxrQHoBJyAfv5K6ccQMq7xUzeW\\\\\\\/Hnqj1\\\\\\\/1Ylp72UBSju96LX5Hp3GJBLFhvlqZODTfJiMacRsOfOZHvhhoPDunVJgrJ2yEoiz8FDL3pGmcsFxEM5n2OTPhvS5ieDi9m0rIq\\\\\\\/d9UKwLrJ9Zfv+Kz6W3Gr1YE\\\\\\\/QaK\\\\\\\/wqraui5AlaGO9VG4aUA\\\\\\\/J+hRpssbadXX\\\\\\\/VAIYBfve\\\\\\\/2+27HV3u+8vPlBlOKzzU4jJb9yt9L4oRAwst5coe1oNRJrKx9oI7zYoEhF83NC1NPZp6F2ciH5AKTRUPKgFwE\\\\\\\/p9CBv0VWqTVqvYtzEtI29G4CIUCoNedvZQ9JEVfqXy1Vp5IomQipFbuorrTlnsWE7ZrZ6qloOMI+MhLsDBMhytkQREVSw2SVyPNjvmSUK1oRMzOSVJ2Oc\\\\\\\/TPLNPXkCQaTOZ2PMmaHevRrHJaCimVp\\\\\\\/cmGat47r3t\\\\\\\/jmlJlZUCHK3qDD787mJ+4\\\\\\\/sCBE2ZYEweQ8qmM2WUeFBhimo2B9UeaAaZ7LxUJW+agPNIp9L2yEsLkEgWA91C7t6CHGFbJ5DVhpDTo9MmcCbWyL0kPCu9NfojAtHPVnkEgx8ao8MZdFmm5tsgz42Ivbk2zZpTVckcxGUy3ijdTa5qiiI2DL+DSaEenUG3cd2mSSNou5QnMdrJe6C254Dl3OEPYp+NmfMgI0IokhUBvOy9qT5hg\\\\\\\/JF2R+N0eklbwiAK3KElYi9x6Y1ip0BNAYQjToMK\\\\\\\/aZD7HHEL7XaZUSgaNhcPLevl\\\\\\\/QoSyBiZqqOJB6LsPxjqwGf9XMIBhZ7V7VtFTXUTrnngM0MTuvWM7LX+09+lWKOk+uKcc0QV7U1k\\\\\\\/3H5oHQ0QqvTOOfcwMzCo+YLpWmm0Ztll4mq4KA3AqNrNNt5RRSn6UN\\\\\\\/NyLM+N7twM4V8cIbJcs+N4hGYJnLf7TwfIWe8JLgBniN\\\\\\\/uPn1TjUdLUo3\\\\\\\/F3A3xlNTQPDq7mC8GbQsT3gG2edOSCALWnpOkH7LJri9v\\\\\\\/jRUmkz16ai1mS7OrjC5tfT1mbQm67aWPELl\\\\\\\/mEFpHduDgbUvkQy7zRjkxJ5RedektKuiJ0HqV5K3dncLlOUHwZDY00GXNNbpxqm3Ji1cMnS2lgBta3qPH1dT+VyJivvd57dRfZY0xnmZ5F\\\\\\\/lgJrKe7QKtpeMt+3wZKeXsOYggntrgX+vIk9J7Q\\\\\\\/C0arG3vVjqF8e5XZwa5iXav8\\\\\\\/YfQ+\\\\\\\/JRhBEksxKbe3zLNV6Uo6uHhuC\\\\\\\/UPG5rx4tlFVUtpiCKP77bpTVRnpaG4LqDqux4l+UzfKnd8lkIi7zGrEJDW33Q3tP9Q8R+j66P0KjTrNwKTHjaUTJoVTI1A5dt\\\\\\\/Dq\\\\\\\/Pksh4rjaPX44jJ8rGBwWe\\\\\\\/Ov+ICYdwUyIGuYbE+DQWmmjLQVxHHgU8whJx8oeRhTBlIEzePDVADM+H\\\\\\\/OBCUltMdAd2GTQLPdTeHqE5Zr1r\\\\\\\/uWctUE34s8tAnru00sbudeYQc2kxEuhnyHvsMBG7xIn+nLTW8owNjtVLfJMVdZx6XhH5SYGL80\\\\\\\/wpg32R8Pptcakd8F\\\\\\\/mBpQb0MXzSo0zmr31VBwhQbYO\\\\\\\/CowSY1iZNvHdZPr7nmaegG8isEsToIxwQLrs2dwQmvvbfpdHHAz3LTQ6EpuyBfuzBtGOja\\\\\\\/rrAg0Copnhc\\\\\\\/LerSY9CIEbOW4CyRY\\\\\\\/r59WEwzQS32gE\\\\\\\/6X7RmyESJVPA0ZYq4oZ3XBWtsYJMFQdMhkMCk9CqvGVqPtpa2SNzlgv1O2Q8a1cPMzVWgLCekQivq\\\\\\\/YNB3OXB9ZDYihsv28y50XMLN04ldkznXrvuwK1QIrH6MFo+7KMZHRHhOKoIeClDglgyNKz5RfQbYIHmIqmvJl8nvQ+BLKmChNQnrhUpqaAQbDd65TiDHOMjCoZ3zvOnxBnUwTw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CucKBc328ILUIKKFVh91LA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f78b7d3dc9f464d8f0b7cac78ac60a038a6f0d3beb18722b81aa6421fe55a40\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459422 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/427" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jd1n92oqZxDtypMGnN6LOeWwxU6FnCx6mLyQmrr5n-A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dRLHzMjRL4XIvLOyhyzKqtqhVBEBTO4eiQAs_cEx9xM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/409" title="A Mosaic Genetic Screen for Genes Involved in the Early Steps of Drosophila Oogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/441" title="The Genetic Architecture of Degenerin/Epithelial Sodium Channels in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459422" id="node459422" data-pisa="ggg;3/3/427" data-pisa-master="ggg;3/3/427" data-apath="/ggg/3/3/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jessica E. Rutkoski</span>, <span class="highwire-citation-author" data-delta="1">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="2">Jean-Luc Jannink</span> and <span class="highwire-citation-author" data-delta="3">Mark E. Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-439;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005363 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459422" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jessica E. Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BE.%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BE.%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Manhattan, Kansas 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mes12@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mes12@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/3/427" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459422" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/427.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459422" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/427.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459422" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/427.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459422" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/3/427.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459422" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genomic selection, a breeding method that promises to accelerate rates of genetic gain, requires dense, genome-wide marker data. Genotyping-by-sequencing can generate a large number of <em>de novo</em> markers. However, without a reference genome, these markers are unordered and typically have a large proportion of missing data. Because marker imputation algorithms were developed for species with a reference genome, algorithms suited for unordered markers have not been rigorously evaluated. Using four empirical datasets, we evaluate and characterize four such imputation methods, referred to as k-nearest neighbors, singular value decomposition, random forest regression, and expectation maximization imputation, in terms of their imputation accuracies and the factors affecting accuracy. The effect of imputation method on the genomic selection accuracy is assessed in comparison with mean imputation. The effect of excluding markers with a large proportion of missing data on the genomic selection accuracy is also examined. Our results show that imputation of unordered markers can be accurate, especially when linkage disequilibrium between markers is high and genotyped individuals are related. Of the methods evaluated, random forest regression imputation produced superior accuracy. In comparison with mean imputation, all four imputation methods we evaluated led to greater genomic selection accuracies when the level of missing data was high. Including rather than excluding markers with a large proportion of missing data nearly always led to greater GS accuracies. We conclude that high levels of missing data in dense marker sets is not a major obstacle for genomic selection, even when marker order is not known.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/imputation-algorithms" class="hw-term hw-article-keyword hw-article-keyword-imputation-algorithms" rel="nofollow">imputation algorithms</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li></ul><p id="p-4">Genomic selection (GS) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Meuwissen <em>et al.</em> 2001</a>) is a relatively new breeding methodology reviewed by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em> (2009)</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Heffner <em>et al.</em> (2009)</a>, and <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lorenz <em>et al.</em> (2011)</a> that is increasingly attractive for the genetic improvement of various species because of its potential to increase the rate of genetic gain (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wong and Bernardo 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Heffner <em>et al.</em> 2010</a>). With GS, a training population having both phenotypic data and genome-wide marker data is used to develop a prediction model for the trait of interest. Before phenotyping, this prediction model is then applied to selection candidates that have been genotyped. Genomic-estimated breeding values are calculated for the selection candidates and selections are made using these values. These breeding values are estimated using genotypes instead of phenotypes; therefore, selection can occur in early stages on a single plant basis or in situations in which phenotyping is either not possible, unreliable, or too expensive, thus leading to shorter selection cycles.</p><p id="p-5">One of the requirements for GS is genome-wide marker coverage. In general, one marker should be in linkage disequilibrium (LD) with each segregating segment of the genome. The choice of marker platform is driven by the available genotyping technology and the cost per data-point. Genotyping-by-sequencing (GBS) is gaining popularity because it can be less expensive than other platforms and can provide genome-wide marker coverage for species that lack genotyping resources such as pre-designed single-nucleotide polymorphism platforms (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Poland and Rife 2012</a>). Polymorphic loci scored by GBS can contain a large proportion of missing data across samples because random fragments of the genome are sequenced at low depth, leading some loci to have zero coverage in some individuals (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elshire <em>et al.</em> 2011</a>). The proportion of missing data depends on the sequencing depth and library complexity. Greater sequencing depth leads to a smaller proportion of missing data but increases genotyping cost. Less-complex libraries, on the other hand, will have less missing data but a fewer markers. To generate a large number of markers at low cost, low sequencing depth is commonly used, leading to a large proportion of missing data points. Most analyses require a complete dataset; therefore, marker imputation is a necessary step before GBS data can be used for most purposes.</p><p id="p-6">Imputation has been shown to increase power in association mapping studies (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Marchini <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Marchini and Howie 2010</a>) and, for GS, imputation can enable the use of low-density genotyping without a major loss in accuracy because a closely related reference panel genotyped at high density can be used to impute markers not present in the low-density marker panel. (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Habier <em>et al.</em> 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Weigel <em>et al.</em> 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dassonneville <em>et al.</em> 2011</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mulder <em>et al.</em> 2012</a>). Although several highly accurate and widely used imputation algorithms have been developed to assign allelic states of missing values in genotype data (reviewed by <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pei <em>et al.</em> 2008</a> and <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Marchini <em>et al.</em> 2010</a>), these algorithms were designed for human genetic data, and they require that the order of the markers be known because they are based on constructing haplotypes. For species lacking a reference genome and complete reference linkage map such as wheat, <em>Triticum aestivum</em> L., the majority of markers typed on a given population are unordered, and current genotype imputation methods cannot be used. Although for biparental populations linkage maps can be constructed, breeding populations for GS are derived from multiple parents and not well structured for developing genetic maps. Thus, alternative imputation strategies that are map-independent are necessary when GBS is used for species lacking a reference genome sequence and for populations unsuitable for linkage map construction. There are many general imputation methods that do not require any prior information about the variables to be imputed. Although these methods are used across many disciplines, they have not been tested for imputation accuracy of genome-wide marker data. It is also not known how imputation with a general and potentially less accurate method before GS model training will affect the GS model accuracy. However, we expect these imputation methods to improve the GS accuracy because during the imputation step, genotypic information from both the training and selection sets is used to estimate missing values. Thus, the validation set helps improve imputation of the training set and vice versa.</p><p id="p-7">The objective of this study was to evaluate imputation strategies that do not require previous information about the order of the markers. The imputation methods compared were: mean imputation (MNI), k-nearest neighbors imputation (kNNI) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Troyanskaya <em>et al.</em> 2001</a>), singular value decomposition imputation (SVDI) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Troyanskaya <em>et al.</em> 2001</a>), expectation maximization imputation (EMI) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dempster <em>et al.</em> 1977</a>), and random forest regression imputation (RFI) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stekhoven and Bühlmann 2011</a>). By using array-based genotypic datasets with varying levels of simulated missing data, we compared these methods in terms of their imputation accuracy, computation time, and impact on GS prediction accuracy. The factors affecting imputation accuracy for each method at the marker genotype and individual genotype level were also examined. Finally, we determined whether excluding rather than including markers with high levels of missing data could lead to higher accuracy.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Original datasets</h3><p id="p-8">We used five different datasets consisting of genome-wide markers and breeding value estimates. These datasets are referred to as winter wheat (WW), spring wheat (SW), drought tolerant maize (DTM), North American barley (NAB), and stem rust resistant wheat (SRRW). The WW data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/005363SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS2.zip">File S2</a>) consists of 374 elite inbred individuals originating from the Cornell winter wheat breeding program. The markers consisted of 1158 polymorphic diversity array technology (DArT) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akbari <em>et al.</em> 2006</a>) markers coded as “−1” and “1.” For a more detailed description of this dataset, refer to <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heffner <em>et al.</em> (2011)</a>. The traits used for the evaluation of cross-validated GS accuracies for WW were grain yield, height (HT), protein, and days to heading. The SW data are a historical dataset consisting of 599 elite inbred spring wheat lines originating from the International Maize and Wheat Improvement Center (CIMMYT) wheat breeding program. The markers consist of 1279 polymorphic DArT markers coded as “0” and “1” and the trait used for the evaluation of cross-validated GS accuracies was grain yield in CIMMYT mega-environment 1. The DTM data consists of 264 tropical CIMMYT maize lines. The trait used to calculate cross-validated GS model accuracies for DTM was grain yield. The marker data consists of 1135 single-nucleotide polymorphisms coded as “−1”, “0”, and “1.” For more details about the SW and DTM datasets, or to access these datasets, refer to <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> (2010)</a>. The NAB dataset consists of a North American spring barley association mapping panel evaluated from 2006 to 2008 as part of the <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barley Coordinated Agricultural Project (2011)</a>. The panel consists of 911 individuals with 2146 polymorphic single-nucleotide polymorphisms. The trait used to calculate GS model accuracies was beta-glucan content (B-glucan). The data can be accessed at <a href="http://triticeaetoolbox.org/barley">http://triticeaetoolbox.org/barley</a>.</p><p id="p-9">The SRRW dataset consists of 360 recent, elite CIMMYT spring wheat lines that have been selected for quantitative resistance to stem rust caused by <em>Puccinia graminis</em> f.sp. <em>tritici</em>. The markers consist of more than 130,000 GBS polymorphisms. Three different versions of the SRRW GBS data, described in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, were created based on different per-marker percent missing data thresholds. For the first version referred to as SRRW version NA20 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS3.zip">File S3</a>), markers were excluded if they had more than 20% missing values, which resulted in 2014 total markers. For the second set and third sets, referred to as SRRW versions NA50 and NA70 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS4.zip">File S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS5.zip">File S5</a>), markers were excluded if they had more than 50% and 70% missing data, respectively, and then 2014 markers were randomly selected. The percent of the data points that were missing in the original WW, SW, DTM, and NAB datasets was between 0.2 and 3%. This low level of pre-existing missing data was assumed to have a negligible effect on the imputation and GS accuracies and for these datasets the original marker data are referred to as version NA0.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462588/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462588/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of datasets used for imputation and genomic selection</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Calculation of LD between marker pairs</h3><p id="p-13">For the original WW, SW, DTM, and NAB datasets, LD between all marker pairs was measured using the r<sup>2</sup> statistic, where r<sup>2</sup> between two markers was calculated using the formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-1.gif"/></noscript></span></span>where <em>D</em> = <em>x</em><sub>11</sub> -<em>p</em><sub>1</sub><em>p</em><sub>2</sub>; <em>x</em><sub>11</sub>is the probability of observing the combination of allele 1 at marker <em>j</em> and allele 1 at marker <em>l</em>, <em>p</em><sub>1</sub> is the probability of allele 1 at marker <em>j</em>, <em>q</em><sub>1</sub> is the probability of allele 2 at marker <em>j</em>, <em>p</em><sub>2</sub> is the probability of allele 1 at marker <em>l</em>, and <em>q</em><sub>2</sub> is the probability of allele 2 at marker <em>l</em>. A maximum likelihood estimate of <em>x</em><sub>11</sub> was obtained using an expectation maximum approach reviewed by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Foulkes (2009)</a>. All calculations of the r<sup>2</sup> statistic were implemented in the R package “genetics” (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Warnes <em>et al.</em> 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Missing data simulation</h3><p id="p-14">For each of the WW, SW, DTM, and NAB datasets three versions of the genotypic data, summarized in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, were created with different levels of simulated missing data. In each of the versions: NA20, NA50, and NA70, missing values were introduced at random but the maximum percent missing data at a given marker was set to 20%, 50%, and 70% respectively. Examples of the simulated markers sets are illustrated in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FigureS1.pdf">Figure S1</a>. A total of 10 replicates of each simulated dataset were created, and the mean percent of total data points that are missing across the 10 replicates is shown in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. The distribution of per-marker percent missing values from the SRRW data versions NA20, NA50, and NA70 were used to assign the percent missing at each marker for each of the WW, SW, DTM, and NAB datasets to produce versions NA20, NA50, and NA70, respectively. Across all the missing data versions of all the datasets, the percent missing per marker distribution had a long left tail and a large concentration of values near the threshold level.</p></div><div id="sec-5" class="subsection"><h3>Imputation methods</h3><p id="p-15">In all cases, the genotypic data were considered continuous variables. The methods MNI, kNNI, SVDI, EMI, and RFI were used to impute the simulated missing values. For all methods the input was an <em>m</em> x <em>n</em> genotype matrix M with <em>m</em> individuals and <em>n</em> markers. For MNI, each missing data-point <em>x<sub>ij</sub></em> at a given marker <em>j</em> was replaced with the mean of the non-missing values at that marker.</p><p id="p-16">For kNNI (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Troyanskaya <em>et al.</em> 2001</a>), the data points were imputed by replacing them with the weighted average of the data points at the k closest markers. Euclidean distance was used as the measure of marker distance. Euclidean distance between marker genotype vectors <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-3.gif"/></noscript></span></span> of length <em>m</em> was defined as<em>:</em><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-4.gif"/></noscript></span></span>In detail, (1) missing values were first replaced using MNI and the Euclidean distance between all of possible pairs of marker vectors was computed. Each marker was included in the marker matrix twice, both in its original and flipped state, to ensure that markers in negative LD would not be considered distant to the marker of interest. (2) For each marker <em>j</em>, markers were sorted based on Euclidean distance to marker <em>j</em>. (3) For each row <em>i</em> of marker <em>j</em> the weighted average of the k closest markers with nonmissing values at row <em>i</em> were used as an estimate of marker data point <em>x<sub>ij</sub></em>. The weight of each marker was 1/<em>d<sup>2</sup></em> were <em>d</em> is the Euclidean distance between marker <em>j</em> and the marker to be weighted. kNNI makes no assumptions about the distribution of the data.</p><p id="p-17">For SVDI (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Troyanskaya <em>et al.</em> 2001</a>), a singular value decomposition of genotype matrix M was used to obtain a set of the k most significant Eigen-vectors of the markers. These k Eigen-vectors were then used as the predictors for linear regression estimation of the missing data points. SVDI was implemented in R (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Development Core Team 2011</a>) using the package “bcv” (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Perry 2009</a>). The genotype matrix M can be described as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-5.gif"/></noscript></span></span>Where U has dimensions <em>m</em> × k, V has dimensions <em>n</em> × k, and Σ is a k × k diagonal matrix. U contains the left singular vectors with are equivalent to the Eigen-vectors of the markers. The corresponding singular values are in the diagonal elements of Σ. The singular values are equivalent to the square root of the Eigen-values. The k most significant Eigen-vectors of the markers were those with the k largest Eigen-values. The imputation procedure is described as follows: (1) Missing values were originally imputed using MNI. (2) Singular value decomposition was used to estimate the k most significant Eigen-vectors of the markers: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-6.gif"/></noscript></span></span>. (3) For each marker <em>j</em>, linear regression coefficients of each column of <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-7.gif"/></noscript></span></span> were estimated by the multiple linear regression equation:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-8.gif"/></noscript></span></span>Where Y is a column vector for marker <em>j</em>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-9.gif"/></noscript></span></span> is an <em>m</em> × <em>k</em> matrix of k Eigen-vectors, β is a vector of regression coefficients and ε is a random error term. Only individuals with nonmissing values in Y were used to estimate β. (4) <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-10.gif"/></noscript></span></span> and the estimates of the regression coefficients, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-11.gif"/></noscript></span></span>, were used to estimate the missing values at marker <em>j</em>. (5) Using the current version of the genotype matrix, we repeated steps two through four for a total of 10 iterations, sufficient to meet the convergence criteria, which was:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-12.gif"/></noscript></span></span><em>RSS</em> is the residual sum of squares between the nonmissing values and their SVDI model approximation. <em>RSS</em><sub>0</sub> and <em>RSS</em><sub>1</sub> are the <em>RSS</em> values of successive iterations. SVDI assumes that the genotype matrix is multivariate normal distributed. For the optimal k value calculation methods and results for both kNNI and SVDI, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS1.pdf">File S1</a>. Optimal k values are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/TableS1.pdf">Table S1</a>.</p><p id="p-18">For EMI, the nonmissing marker data were used to obtain maximum likelihood estimates of the vector of means, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-13.gif"/></noscript></span></span>, and covariance matrix <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-14.gif"/></noscript></span></span> of the individuals based on the markers. These estimates were then used to obtain multiple linear regression estimates of the missing marker values. <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-15.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-16.gif"/></noscript></span></span> were then re-estimated and were used to re-estimate the missing marker values. This process was repeated until the difference between the new estimate and the previous estimate of <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-17.gif"/></noscript></span></span> was 0.02 or less. EMI was implemented using the R package rrBLUP (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Endelman 2011</a>). For a more detailed description of this EMI algorithm, refer to <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Poland <em>et al.</em> (2012)</a>. For a more through description of the EM imputation algorithm in general, refer to <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Dempster <em>et al.</em> (1977)</a>.</p><p id="p-19">For RFI, missing marker values were estimated using random forest regression (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Breiman 2001</a>) using all available data to predict the missing values for every marker. RFI was implemented in R using the package “MissForest” (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Stekhoven and Bühlmann 2011</a>). The imputation procedure was: (1) for marker matrix M, markers were sorted from lowest to highest percent missing and missing values were imputed using MNI. (2) At each marker <em>j</em> containing missing values, the nonmissing values, Y, were used to grow 100 random forest regression trees Θ1… Θ100. Each tree was grown using a bootstrapped sample of individuals Y and a random sample of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-18.gif"/></noscript></span></span>marker predictors were used where <em>n</em>-1 is the number of markers excluding marker <em>j</em>. Each tree Θ contains the terminal node values and a set of instructions for recursively partitioning the observations into the terminal nodes: these instructions include the split variables at each node, and the value of the split variable used for partitioning. (3) Missing values at marker <em>j</em> were imputed as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-19.gif"/></noscript></span></span>where <em>x</em> is an input vector.</p><p id="p-20">(4) Marker <em>j</em> was then updated in marker matrix M by using the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-20.gif"/></noscript></span></span> values as the estimate of the missing values. (5) Steps two through four were repeated for each subsequent marker until all markers were imputed. (6) Then, using this imputed matrix, we repeated steps two through five until convergence or for a maximum of 10 iterations. Convergence was declared as soon as the Δ<em>N</em> increased for the first time where:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-21.gif"/></noscript></span></span>M<sub>1</sub> and M<sub>0</sub> are the newly imputed and previously imputed marker matrices respectively. If the convergence criterion was met, M<sub>0</sub> was used as the final estimate of M. RFI makes no assumptions about the distribution of the data. The implementation of all imputation methods is demonstrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS6.zip">File S6</a>.</p></div><div id="sec-6" class="subsection"><h3>Imputation accuracy calculations</h3><p id="p-21">The per-marker imputation accuracy, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-22.gif"/></noscript></span></span>, was described using the <em>R</em><sup>2</sup> value between predicted data points and the original data points for a given marker vector or individual vector <em>x</em> of length <em>j</em>. The <em>R</em><sup>2</sup> was defined as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-23.gif"/></noscript></span></span>The <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-24.gif"/></noscript></span></span>, as well as the imputation <em>R</em><sup>2</sup> of the individual genotypes, referred to as <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-25.gif"/></noscript></span></span>, were calculated. For each dataset and missing data level, average <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-27.gif"/></noscript></span></span> across the 10 missing data simulations were also calculated and referred to as <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-29.gif"/></noscript></span></span>.</p><p id="p-22">To compare with imputation accuracies reported in other publications, for each <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-30.gif"/></noscript></span></span> value, the equivalent percent correct was also calculated. Because imputed values were continuous, the percent correct for each marker could not be directly calculated. Instead, for each marker, equivalent percent correct values were determined by simulation using each marker’s MAF and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-31.gif"/></noscript></span></span> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-7" class="subsection"><h3>Computational time</h3><p id="p-23">For the first replicate of simulated missing datasets, whenever a dataset was imputed, the number of seconds required for imputation to be completed using one central processing unit was recorded. All jobs were submitted to the Computational Biology Service Unit at Cornell University, which uses (1) a 240 core Windows cluster consisting of 60 Dell PowerEdge 1855 nodes with two x64 Pentium 4 Xeon 3.4 GHz, 4 GB RAM, and 144 GB HD each and (2) a 400 core Windows cluster consisting of 200 Sun V20Z nodes with two AMD Opteron 248 2.2GHz, 2 GB RAM, and 300 GB HD each.</p></div><div id="sec-8" class="subsection"><h3>Assessment of factors affecting imputation accuracy</h3><p id="p-24">For each imputation method factors affecting the imputation accuracy were assessed. A marker’s minor allele frequency (MAF), number of nonmissing data points, and level of LD with other markers were considered as factors that could impact its imputation accuracy. The distance between an individual and its closest relative and the expected prediction error variance (PEV) were considered as factors affecting the imputation accuracy on an individual genotype basis. The impact of each of these factors was assessed for each imputation method using the WW, SW, DTM, and NAB datasets post imputation.</p><p id="p-25">First, the impact of MAF on the imputation accuracy was assessed. For each dataset-imputation method combination, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-32.gif"/></noscript></span></span> was averaged across dataset versions NA20, NA50, and NA70 and this overall estimate of marker imputation accuracy is referred to as <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-33.gif"/></noscript></span></span>.The median <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-34.gif"/></noscript></span></span> for each value of MAF rounded to the nearest tenth was calculated. The relationship between the median <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-35.gif"/></noscript></span></span> and the MAF value was then plotted to characterize the relationship.</p><p id="p-26">The impact of the number of nonmissing data points at a marker on the marker’s imputation accuracy was assessed for each dataset-imputation method combination using data from all 10 replicates and versions NA20, NA50 and NA70 combined. For each marker, the number of nonmissing data points was rounded to the nearest factor of 5, and for each value the median <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-36.gif"/></noscript></span></span>was calculated.</p><p id="p-27">To determine the impact of the LD level with other markers on the imputation accuracy, markers were first classified as markers in low LD with all other markers or markers in at least moderate LD with at least one other marker. Markers whose highest r<sup>2</sup> statistic was less than 0.5 were considered to be in low LD with all other markers. A marker that had at least one r<sup>2</sup> statistic greater than or equal to 0.5 was considered to be in at least moderate LD with at least one other marker. The median <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-37.gif"/></noscript></span></span> of markers in low LD and of markers in at least moderate LD with at least one other marker was calculated. The ratio of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-38.gif"/></noscript></span></span> for markers in low LD to the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-39.gif"/></noscript></span></span> for markers in at least moderate LD was then examined.</p><p id="p-28">To assess the effect of the genetic distance between an individual and its closest relative on the individual genotype imputation accuracy, the Euclidian distance was calculated for each pair of individuals and the <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-40.gif"/></noscript></span></span> of each dataset was measured for each simulated dataset and imputation method combination. The mean <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-41.gif"/></noscript></span></span> values across all replicates, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-42.gif"/></noscript></span></span>, were averaged across versions NA20, NA50, and NA70 of a given dataset-imputation method combination to calculate an overall mean <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-43.gif"/></noscript></span></span> for each individual which is referred to as <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-44.gif"/></noscript></span></span>. The Euclidian distance between each individual and its closest relative, rounded to the nearest whole number was plotted against the median <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-45.gif"/></noscript></span></span> to examine the relationship.</p><p id="p-29">The relationship between PEV for the genetic values and the <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-46.gif"/></noscript></span></span> was also examined. An individual’s PEV is a measure of genetic connectedness to the other individuals (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kennedy and Trus 1993</a>) where an individual’s connectedness is determined by the number and strength of the genetic relationships between that individual and the other individuals in the dataset. For example, a low PEV indicates high connectedness and high degree of genetic relationship. To measure an overall PEV value for each individual, a vector of PEVs was calculated for each marker using the mixed model equations (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Searle <em>et al.</em> 1992</a>) implemented in the R package “rrBLUP” (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Endelman 2011</a>). The genetic and error variance components were estimated using restricted maximum likelihood, and the genomic relationship matrix, excluding the response variable marker, was used as the covariance matrix between genotypes. The sum of the PEV vectors across all markers was used as the overall PEV vector. Because PEV is a reflection of the number and strength of the genetic relationships between individuals, it is expected to be a useful indicator for how well and individual’s missing data can be imputed using all other individuals as a reference.</p></div><div id="sec-9" class="subsection"><h3>GS accuracy calculation</h3><p id="p-30">All 10 simulations of missing data versions NA20, NA50, and NA70 of the WW, SW, DTM, NAB, and SRRW marker sets were imputed with each of the imputation methods: MNI, kNNI, SVDI, EMI, and RFI. Then, each of the 10 replicates of the marker set-imputation method combinations was used to calculate the 10-fold cross validation GS accuracy for both Ridge-Regression (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Whittaker <em>et al.</em> 2000</a>) and Bayesian LASSO (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">de los Campos <em>et al.</em> 2009</a>), see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS1.pdf">File S1</a>. GS accuracies are computed as the Pearson’s correlation between the phenotype estimated breeding values and the genomic-estimated breeding values. The mean accuracy for each marker set-imputation method-prediction model combination was computed. GS accuracies were also computed using version NA0 of the WW, SW, DTM, and NAB genotypic data.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>LD between markers</h3><p id="p-31">For each dataset, the LD between marker pairs was quantified using the r<sup>2</sup> statistic. Markers that had at least one other marker associated with r<sup>2</sup>≥ 0.5 were considered to be in at least moderate LD with at least one other marker. In the WW, SW, DTM, and NAB datasets, 62%, 74%, 12%, and 69% of the markers had at least one other marker in at least moderate LD, respectively. Comparatively, LD between markers was high in the SW, NAB, and WW datasets and much lower in the DTM dataset.</p></div><div id="sec-12" class="subsection"><h3>Imputation accuracy</h3><p id="p-32">The imputation accuracy reported as the median <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-47.gif"/></noscript></span></span> is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for kNNI, SVDI, RFI, and EMI. For all dataset-imputation method combinations, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-48.gif"/></noscript></span></span> values were non-normal, and there were many extreme values. The median <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-49.gif"/></noscript></span></span> values and the equivalent percent correct values are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The population with the highest median <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-50.gif"/></noscript></span></span> for each of the levels of missing data were the NAB population, whereas the lowest imputation accuracies were observed with the DTM population. As expected, median <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-51.gif"/></noscript></span></span> values always decreased as the level of missing data increased. RFI always produced the highest accuracies; kNNI generally produced the second highest accuracies, followed by EMI and SVDI. The rankings were slightly different for the DTM dataset, where RFI was most accurate followed by EMI, SVDI, and kNNI. The rankings of the methods for each dataset according to the median percent correct are the same as those according to the median <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-52.gif"/></noscript></span></span>; however, the median percent correct values could not be compared across datasets because percent correct values are influenced by the MAF which differs among datasets.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Median of each imputation method across all datasets. (A) WW; (B) SW; (C) DTM; and (D) NAB. For each population median  obtained using kNNI, SVDI, RFI, and EMI are shown for the three dataset versions: NA20 (black), NA50 (gray), and NA70 (white), which contain up to 20%, 50%, and 70% missing values per marker, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1920275025" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Median &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-53&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;of each imputation method across all datasets. (A) WW; (B) SW; (C) DTM; and (D) NAB. For each population median &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; obtained using kNNI, SVDI, RFI, and EMI are shown for the three dataset versions: NA20 (black), NA50 (gray), and NA70 (white), which contain up to 20%, 50%, and 70% missing values per marker, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/427/F1.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/427/F1.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462554" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-33" class="first-child">Median <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F1/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F1/embed/mml-math-53.gif"/></noscript></span></span>of each imputation method across all datasets. (A) WW; (B) SW; (C) DTM; and (D) NAB. For each population median <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F1/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F1/embed/mml-math-54.gif"/></noscript></span></span> obtained using kNNI, SVDI, RFI, and EMI are shown for the three dataset versions: NA20 (black), NA50 (gray), and NA70 (white), which contain up to 20%, 50%, and 70% missing values per marker, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462478/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462478/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Median <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/T2/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/T2/embed/mml-math-55.gif"/></noscript></span></span> and median percent correct for each imputation method and across all datasets</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Computational time</h3><p id="p-36">Large differences in the computational requirements for the imputation methods were observed (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). kNNI, SVDI, and EMI required relatively little computation time on average, while RFI required at least 95x, 760x, and 65x more computation time than kNNI, SVDI, and EMI, respectively. For SVDI and kNNI, the computation time required for determining optimal k values was not included in the estimates of the average computational time because the computation time for optimal k estimation depends on the method used for estimation. The 10-fold cross validation approach that we used to estimate optimal k values for SVDI and kNNI requires approximately 50 runs of the SVDI and kNNI respectively. If the time required to estimate optimal k values for SVDI and kNNI was included in the total computational time, EMI would be the fastest of the four imputation methods.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462562/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462562/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">CPU minutes required to complete the imputation of one dataset</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Factors affecting imputation accuracy</h3><div id="sec-15" class="subsection"><h4>MAF:</h4><p id="p-39">For all datasets, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-57.gif"/></noscript></span></span> values for markers with MAF &lt; 0.1 were low compared with that of markers with MAF &gt; 0.1; however, the relationship between MAF and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-58.gif"/></noscript></span></span> for markers with MAF &gt; 0.1 was different for each dataset (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In general, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-59.gif"/></noscript></span></span> increased as MAF increased as long as MAF &lt; 0.4; however, with the NAB dataset (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>) there was no relationship between MAF and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-60.gif"/></noscript></span></span> for MAF &gt; 0.1. Accuracy in terms of percent correct had a strong negative linear relationship with the MAF across all imputation methods and datasets. Markers with lower MAF values tended to have higher percent correct values (data not shown).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between the MAF and . The median obtained for a given MAF rounded to the nearest tenth is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1920275025" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Relationship between the MAF and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-61&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;sub&gt;.&lt;/sub&gt; The median &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-62&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;obtained for a given MAF rounded to the nearest tenth is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/427/F2.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/427/F2.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462512" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-40" class="first-child">Relationship between the MAF and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F2/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F2/embed/mml-math-61.gif"/></noscript></span></span><sub>.</sub> The median <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F2/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F2/embed/mml-math-62.gif"/></noscript></span></span>obtained for a given MAF rounded to the nearest tenth is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Number of nonmissing data points:</h4><p id="p-41">With almost all dataset-imputation method combinations, as the number of non-missing data points increased, the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-63.gif"/></noscript></span></span> levels increased in a linear fashion (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The strength of this linear relationship was similar for all imputation methods; however, with the DTM dataset, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-64.gif"/></noscript></span></span> for kNNI and SVDI were close to zero regardless of the number of nonmissing data points.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/427/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between the number of nonmissing datapoints and . The median  obtained for a given number nonmissing data points rounded to the nearest factor of 5, is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1920275025" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between the number of nonmissing datapoints and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-65&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The median &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-66&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; obtained for a given number nonmissing data points rounded to the nearest factor of 5, is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/427/F3.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/427/F3.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/427/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/427/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462480" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-42" class="first-child">Relationship between the number of nonmissing datapoints and <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F3/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F3/embed/mml-math-65.gif"/></noscript></span></span>. The median <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F3/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F3/embed/mml-math-66.gif"/></noscript></span></span> obtained for a given number nonmissing data points rounded to the nearest factor of 5, is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>LD between markers:</h4><p id="p-43">The ratio of the median imputation <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-67.gif"/></noscript></span></span> for markers with no other markers in moderate LD to the median imputation <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-68.gif"/></noscript></span></span> for markers with at least one other marker in moderate LD was always less than one (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), indicating that the imputation accuracy for markers without markers in moderate LD was always lower than that for markers that had at least one other marker in moderate LD. Across all datasets, the <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-69.gif"/></noscript></span></span> ratios for the two classes of markers was much smaller for kNNI compared to the other imputation methods, indicating that the imputation accuracy of kNNI was more strongly influenced by the level of LD between markers compared to the other methods. With the WW, SW, and NAB datasets the <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-70.gif"/></noscript></span></span> ratios for the two classes of markers was similar for SVDI, RFI, and EMI indicating that the accuracy of these three methods is influenced by the level of LD between markers to a similar degree. However, with the DTM dataset, the <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-71.gif"/></noscript></span></span> ratio for the two classes of markers was closer to one for SVDI compared to the other methods, indicating that for this dataset, the accuracy with SVDI was less affected by the LD between markers, compared to the other methods.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462600/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462600/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Ratios of median <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/T4/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/T4/embed/mml-math-72.gif"/></noscript></span></span>of markers having no markers in moderate linkage disequilibrium (LD) to that of markers with at least one other marker in moderate LD</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Distance from the closest relative and PEV:</h4><p id="p-45">Regardless of the dataset or the imputation method, the smaller the distance between an individual and its closest relative, the higher the <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-73.gif"/></noscript></span></span> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). One exception was observed with the DTM dataset, where for kNNI there was no relationship between the distance between an individual and its closest relative and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-74.gif"/></noscript></span></span>. We observed very similar trends between <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-75.gif"/></noscript></span></span> and the overall PEV (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FigureS2.pdf">Figure S2</a>). As an individual’s PEV increased, indicating a decrease in the strength and number of genetic relationships between that individual and all other individuals, its <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-76.gif"/></noscript></span></span> decreased in all cases except when the DTM dataset was imputed with kNNI.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/427/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between the distance from the closest relative and . The median  obtained for a given Euclidean distance between an individual and its closest relative rounded to the nearest whole number is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1920275025" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Relationship between the distance from the closest relative and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-77&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;sub&gt;.&lt;/sub&gt; The median &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-78&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; obtained for a given Euclidean distance between an individual and its closest relative rounded to the nearest whole number is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/427/F4.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/427/F4.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/427/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/427/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462532" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-46" class="first-child">Relationship between the distance from the closest relative and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F4/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F4/embed/mml-math-77.gif"/></noscript></span></span><sub>.</sub> The median <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F4/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/F4/embed/mml-math-78.gif"/></noscript></span></span> obtained for a given Euclidean distance between an individual and its closest relative rounded to the nearest whole number is plotted for each dataset: (A) WW; (B) SW; (C) DTM; and (D) NAB. Each color and symbol represents a different imputation method: kNNI, orange triangles; SVDI, purple squares; RFI, red circles; and EMI, blue crosses.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-19" class="subsection"><h3>Effect of imputation method on GS accuracy</h3><p id="p-47">In nearly all cases, GS accuracies did not differ greatly from one imputation method to another, with the exception of MNI, which sometimes led to much lower accuracies compared to all other methods when the NA70 dataset version was used (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>). Overall, GS accuracies were least affected by the imputation method for dataset version NA20, and most affected by the imputation method for dataset version NA70. The relative performance of each method in terms of GS accuracy after imputation depended on the dataset, and dataset version; however, RFI consistently performed well across all datasets. For the WW datasets, the relative performance of the imputation methods in terms of GS accuracy was inconsistent across the four traits tested.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/427/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GS accuracy obtained using ridge regression after imputation. Mean GS accuracies obtained using the dataset versions NA0, NA20, NA50, having up to 0%, 20%, 50%, and 70% missing data per marker, respectively, imputed with either MNI (black stars), kNNI (orange triangles), SVDI (purple squares), EMI (blue crosses) and RFI (red circles) are shown for (A) WW-yield, (B) WW-height, (C) WW-protein, (D) WW-days to heading, (E) DTM, (F) SW, (G) NAB, and (H) SRRW datasets. Each plot has a different y-axis range. Error bars depict SE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1920275025" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GS accuracy obtained using ridge regression after imputation. Mean GS accuracies obtained using the dataset versions NA0, NA20, NA50, having up to 0%, 20%, 50%, and 70% missing data per marker, respectively, imputed with either MNI (black stars), kNNI (orange triangles), SVDI (purple squares), EMI (blue crosses) and RFI (red circles) are shown for (A) WW-yield, (B) WW-height, (C) WW-protein, (D) WW-days to heading, (E) DTM, (F) SW, (G) NAB, and (H) SRRW datasets. Each plot has a different y-axis range. Error bars depict SE.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/427/F5.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/427/F5.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/427/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/427/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-48" class="first-child">GS accuracy obtained using ridge regression after imputation. Mean GS accuracies obtained using the dataset versions NA0, NA20, NA50, having up to 0%, 20%, 50%, and 70% missing data per marker, respectively, imputed with either MNI (black stars), kNNI (orange triangles), SVDI (purple squares), EMI (blue crosses) and RFI (red circles) are shown for (A) WW-yield, (B) WW-height, (C) WW-protein, (D) WW-days to heading, (E) DTM, (F) SW, (G) NAB, and (H) SRRW datasets. Each plot has a different y-axis range. Error bars depict SE.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/427/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GS accuracy obtained using Bayesian least absolute shrinkage and selection operator after imputation. Mean GS accuracies obtained using the dataset versions NA0, NA20, and NA50, having up to 0%, 20%, 50%, and 70% missing data per marker respectively, imputed with either MNI (black stars), kNNI (orange triangles), SVDI (purple squares), EMI (blue crosses) and RFI (red circles) are shown for (A) WW-yield, (B) WW-height, (C) WW-protein, (D) WW-days to heading, (E) DTM, (F) SW, (G) NAB, and (H) SRRW datasets. Each plot has a different y-axis range. Error bars depict SE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1920275025" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GS accuracy obtained using Bayesian least absolute shrinkage and selection operator after imputation. Mean GS accuracies obtained using the dataset versions NA0, NA20, and NA50, having up to 0%, 20%, 50%, and 70% missing data per marker respectively, imputed with either MNI (black stars), kNNI (orange triangles), SVDI (purple squares), EMI (blue crosses) and RFI (red circles) are shown for (A) WW-yield, (B) WW-height, (C) WW-protein, (D) WW-days to heading, (E) DTM, (F) SW, (G) NAB, and (H) SRRW datasets. Each plot has a different y-axis range. Error bars depict SE.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/427/F6.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/427/F6.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/427/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/427/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462496" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-49" class="first-child">GS accuracy obtained using Bayesian least absolute shrinkage and selection operator after imputation. Mean GS accuracies obtained using the dataset versions NA0, NA20, and NA50, having up to 0%, 20%, 50%, and 70% missing data per marker respectively, imputed with either MNI (black stars), kNNI (orange triangles), SVDI (purple squares), EMI (blue crosses) and RFI (red circles) are shown for (A) WW-yield, (B) WW-height, (C) WW-protein, (D) WW-days to heading, (E) DTM, (F) SW, (G) NAB, and (H) SRRW datasets. Each plot has a different y-axis range. Error bars depict SE.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">For, a given dataset and dataset version, the rank of each method based on <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-79.gif"/></noscript></span></span>, was not consistent with the rank based on GS accuracy using RR or BL post-imputation. The rank of the imputation methods, however, was consistent between the two different GS models. We also found that including rather than removing ‘sparse’ markers, those with large amounts of missing data, nearly always led to higher GS accuracies (methods and results described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/TableS2.pdf">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FigureS3.pdf">Figure S3</a>), especially when RFI, kNNI, or EMI were the imputation methods used (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FigureS4.pdf">Figure S4</a>).</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><div id="sec-21" class="subsection"><h3>Imputation accuracy</h3><p id="p-51">This study found that map-independent imputation methods other than MNI can be surprisingly accurate, especially when LD between markers is high and the genotyped individuals are related. RFI was the most promising method overall because of its consistently high performance in terms of imputation accuracy and subsequent GS accuracy; however, it was the most computationally intensive method evaluated. kNNI, although less accurate than RFI, may be a good alternative to RFI if there are computational limitations to completing the imputation. It is likely that RFI and kNNI produced comparable levels of accuracy because both use a similar model free approach for imputation that involves weighting a selected set of k important variables according to a distance metric (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lin and Jeon 2006</a>). The weighted average of these variables is the predicted value of the variable of interest. For kNNI the distance metric was the Euclidean distance and k was a fixed number across all variables. For RFI, the k important variables and their weights are determined by the splitting scheme of the tree that is determined using the response variable. The increased accuracy but greater computational burden of the RFI method compared with kNNI is due to its adaptive weighting of variables that takes into account the response variable.</p><p id="p-52">A possible reason that EMI and SVDI were less accurate than RFI and kNNI is that the genotypic datasets that we used may have violated multivariate normality, an underlying assumption for EMI and SVDI. Alternatively, EMI and SVDI may not have been as effective at ignoring uninformative predictors. If true, linear regression based imputation methods involving variable selection could be as accurate as kNNI or RFI. However, due to multicollinearity, attempts to test imputation based on subset selection methods such as stepwise regression were not successful. Regression imputation using variable selection methods which can cope with multicollinearity, such as the least absolute shrinkage and selection operator (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tibshirani 1996</a>), would be interesting to test in future studies.</p><p id="p-53">EMI performed consistently better than SVDI which is likely because EMI incorporates all the marker data as predictors whereas SVDI first used a data reduction step, potentially eliminating useful information. SVDI may have outperformed EMI if the datasets had a greater rate of genotyping error because it is expected to better cope with noisy data (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Troyanskaya <em>et al.</em> 2001</a>).</p><p id="p-54">For all methods, average median imputation accuracies on an individual genotype basis <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-80.gif"/></noscript></span></span> were not always homogenous across population sub-groups as illustrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FigureS5.pdf">Figure S5</a>, which shows individuals plotted according to the first two principal components of their marker genotypes and color coded according to their imputation accuracy. With the DTM and WW datasets, small subgroups of individuals that clustered together according to the first two principal components of marker genotypes tended to have similar ranges of accuracy. However, with the SW and NAB datasets <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-81.gif"/></noscript></span></span> was relatively homogenous across population subgroups. An association between <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-82.gif"/></noscript></span></span> and population subgroup is undesirable because it may create or worsen an association between GS accuracy and population sub-group. Using large datasets with minimal population structure for imputation and GS is advocated to avoid heterogeneity of imputation and GS accuracies across subgroups of individuals.</p><p id="p-55">Population structure may also lead to increased imputation accuracy for markers with high levels of population subdivision (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Iwata and Jannink 2010</a>) because an individual’s allelic state can be predicted largely based population sub-group alone. Accuracy levels for datasets with many markers highly subdivided by population may be high largely because of structure; we therefore calculated <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-83.gif"/></noscript></span></span> excluding markers with high levels of population subdivision as indicated by their Fst values, where high Fst indicates high population subdivision (for methods, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FileS1.pdf">File S1</a>). For markers with MAF &gt; 0.1, on average, <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-84.gif"/></noscript></span></span> excluding markers with the 25% highest Fst values were 0.9, 1.17, 1.02, and 0.9 times those of overall <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-85.gif"/></noscript></span></span> for the WW, SW, DTM, and NAB datasets, respectively. Thus, for the WW and NAB datasets, the high imputation accuracies we observed may have been in small part due to population structure.</p><p id="p-56">Comparing our imputation accuracy results with those of other studies is difficult because each study uses different populations of different sizes, levels of missing data, MAF distributions, and levels of LD between markers. In addition, accuracy reported as percent correct cannot be compared across datasets with different MAF distributions. Nevertheless, we assume that map-dependent imputation methods would outperform the map-independent methods that we evaluated (given the availability of an accurate genetic or physical map) because physically linked markers are used to predict missing values. These physically linked markers should be more reliable predictors compared to markers that are in LD but may not be physically linked. As genetic and physical maps develop for wheat and barley the assumption that map-dependent methods would outperform the map-independent methods can be tested.</p></div><div id="sec-22" class="subsection"><h3>Factors affecting imputation accuracy</h3><p id="p-57">Markers with very low MAF had low <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-86.gif"/></noscript></span></span> values. There are two possible explanations for this observation. First, because of the way <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-87.gif"/></noscript></span></span> is calculated, a single imputation error has a much larger negative impact on the <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-88.gif"/></noscript></span></span> for markers with lower MAF values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1/FigureS6.pdf">Figure S6</a>). Thus, it is harder to achieve high <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-89.gif"/></noscript></span></span> for markers with a low MAF. Second, individuals with the minor allele at a given marker are not well represented, making their marker genotype more difficult to predict. A similar relationship between MAF and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-90.gif"/></noscript></span></span> was also found by studies by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Iwata and Jannink (2010)</a> and <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> (2011)</a>, which used map-dependent imputation methods. Unlike <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-91.gif"/></noscript></span></span>, imputation accuracy in terms of percent correct had a negative linear relationship with MAF (data not shown), this is because markers with lower MAF can always be imputed with a reasonably high percent correct based on the marker mean alone. Other studies of map-dependent imputation methods report a negative relationship between MAF and percent correct (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Pei <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hickey <em>et al.</em> 2012</a>).</p><p id="p-58">The number of nonmissing data points, analogous to reference panel size in other studies was found to positively impact the <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-92.gif"/></noscript></span></span>. This finding is consistent with other studies in which researchers tested the effect of reference panel size on the imputation accuracy using map-dependent methods (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Pei <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Druet <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2010</a>). For RFI, EMI, and SVDI, which involve a model training step, fewer missing data points means that more individuals are available for model training. With kNNI, a smaller number of nonmissing data points at a given marker leads to a more accurate estimate of its distance from all other markers. However, with the DTM set there was no trend between accuracy and the number of nonmissing data points with kNNI. This was because accuracy with kNNI for this dataset was very low overall.</p><p id="p-59">The presence of one or more markers in moderate LD (r<sup>2</sup> statistic ≥ 0.5) was a more important factor for kNNI compared with RFI, EMI, and SVDI because kNNI bases its predictions on a fixed number of close markers, whereas RFI, EMI, and SVDI use information from all markers in the dataset to generate predicted values for the missing data points. The LD between markers on a whole dataset basis also appeared to be an important factor affecting the <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-93.gif"/></noscript></span></span> of all methods because accuracies with the DTM dataset, which had low levels of LD between markers overall, were much lower than accuracies with the WW, SW, and NAB datasets. Other publications that have evaluated the effect of LD on imputation accuracy for map-dependent methods have found similar trends (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Pei <em>et al.</em> 2008</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hickey <em>et al.</em> 2012</a>).</p><p id="p-60">We found that imputation accuracy on an individual genotype level was negatively correlated with the distance from the closest relative in the dataset, and the PEV, which is an indication of the relationship between an individual and other genotypes. A similar relationship between imputation accuracy and relationship has been found by other studies of map-dependent imputation methods (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Druet <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang and Druet 2010</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Hickey <em>et al.</em> 2012</a>). It is clear that to ensure effective imputation, the dataset to be imputed should contain related individuals. If the dataset is suited for GS, it is likely that the individuals are already related. However, to increase the chances that an individual will have close relatives in the dataset, all available genotypic data for the germplasm pool of interest should be combined before imputation.</p></div><div id="sec-23" class="subsection"><h3>GS accuracy</h3><p id="p-61">The GS accuracies that we observed may be sufficiently high to lead to increased rates of genetic gain compared to phenotypic selection (PS), depending on the accuracy of PS and the selection cycle duration of both PS and GS. It is important to note that all GS accuracies reported for a given dataset are global estimates across all potential subpopulations. Based on other studies evaluating GS accuracies within and across subpopulations (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhao <em>et al.</em> 2011</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2012</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Windhausen <em>et al.</em> 2012</a>), this global accuracy estimate may be greater than the accuracy measured within individual subpopulations.</p></div><div id="sec-24" class="subsection"><h3>Effect of imputation method on the GS accuracy</h3><p id="p-62">Improved accuracy of GS after application of map independent imputation methods was another important finding of this study. Based on our results, unordered markers with missing data can be included in the dataset to improve accuracy through imputation with RFI, kNNI, EMI, or even SVDI rather than MNI. However, for datasets with low levels of missing data (up to 20% per marker), imputing with MNI is sufficient. Although our results do not support removing markers with high levels of missing data prior to GS, in many datasets markers with low levels of missing data may be sufficient to saturate the genome. With the datasets used in this study, the average number of markers with up to 20% and 50% missing data were 18 to 37 and 99 to 186 respectively, and these reduced marker sets were not sufficient to saturate the genome. Thus, including markers with larger amounts of missing data led to improved GS accuracies. Interestingly, a low median <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-94.gif"/></noscript></span></span> was not reflective of the merit of imputation prior to GS. The median <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-95.gif"/></noscript></span></span> for the datasets with up to 70% missing data per marker were the lowest of all the missing data levels; however we saw the greatest gain in GS accuracy from kNNI, SVDI, EMI, or RFI relative to MNI with this level of missing data. This was especially apparent for the DTM dataset, which had a median <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/427/embed/mml-math-96.gif"/></noscript></span></span> near zero for most methods when there was up to 70% missing data per marker. However, RFI on this dataset produced GS model accuracies 1.3 times greater than those achieved when MNI was used before GS. Surprisingly, the most accurate imputation method was not always the method that gave the greatest GS accuracy. This may be caused by nonrandom imputation errors. If some imputation errors are similar for related individuals, these nonrandom errors may able to capture some genetic relationships in the GS model. The idea that the imputation errors may capture some genetic relationships was suggested by a study by <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Weigel <em>et al.</em> (2010)</a>.</p><p id="p-63">This study has important implications for species that lack a reference genome, complete reference map, and predesigned high-throughput genotyping platforms. First, unordered markers can be imputed with high levels of accuracy, and even greater accuracies may result if additional reference genotypes can be added to the dataset prior to imputation. Based on the results of this study, if a large number of marker genotypes are produced (so that markers are in LD with each other), and the population contains individuals with some genetic relationship, missing data can be imputed with reasonable accuracy even if the level of missing data are high; up to 70%. Future work to improve upon and reduce the computational burden of the most promising methods in this study, RFI and kNNI, would be especially useful if these methods are to be used widely. The second implication of this study is that a large proportion of missing data in dense marker sets is not a major concern for GS. As long as the marker density is sufficiently high, the accuracy does not appear to be strongly negatively affected. In cases where missing data does negatively impact the GS accuracy imputation using a method other than MNI before GS model training and validation can help improve the accuracy. Overall, map-independent imputation shows promise for the feasibility of applying GS, enabled by emergent sequence-based genotyping technologies, to almost any species regardless of the availability of pre-existing genotyping resources.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">This research was funded by The Bill and Melinda Gates Foundation (Durable Rust Resistance in Wheat) and the United States Department of Agriculture-Agricultural Research Service (USDA-ARS) (Appropriation No. 5430-21000-006-00D). Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the USDA. USDA is an equal opportunity provider and employer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005363/-/DC1</a></p></li><li class="fn" id="fn-10"><p id="p-65"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-08-04"><span class="received-label">Received </span>August 4, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-12-28"><span class="accepted-label">Accepted </span>December 28, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Rutkoski <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.1" data-doi="10.1007/s00122-006-0365-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akbari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenzl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caig</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAkbari%26rft.auinit1%253DM.%26rft.volume%253D113%26rft.issue%253D8%26rft.spage%253D1409%26rft.epage%253D1420%26rft.atitle%253DDiversity%2Barrays%2Btechnology%2B%2528DArT%2529%2Bfor%2Bhigh-throughput%2Bprofiling%2Bof%2Bthe%2Bhexaploid%2Bwheat%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0365-4%26rft_id%253Dinfo%253Apmid%252F17033786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0365-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17033786&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241798000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-3.3.427.2"><div class="cit-metadata unstructured">Barley Coordinated Agricultural Project, 2011 Introduction to project. University of Minnesota, St. Paul, MN. Available at: <a href="http://www.barleycap.org/">http://www.barleycap.org/</a>. Accessed: July 30, 2012.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.3" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forests</span>. <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.4" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.5" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.6" data-doi="10.3168/jds.2011-4299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dassonneville</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effect of imputing markers from a low-density chip on the reliability of genomic breeding values in Holstein populations</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3679</span>–<span class="cit-lpage">3686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DDassonneville%26rft.auinit1%253DR.%26rft.volume%253D94%26rft.issue%253D7%26rft.spage%253D3679%26rft.epage%253D3686%26rft.atitle%253DEffect%2Bof%2Bimputing%2Bmarkers%2Bfrom%2Ba%2Blow-density%2Bchip%2Bon%2Bthe%2Breliability%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BHolstein%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4299%26rft_id%253Dinfo%253Apmid%252F21700057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21700057&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempster</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laird</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Maximum likelihood from incomplete data via the EM algorithm</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B. Met.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB.%2BMet.%26rft.volume%253D39%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.8" data-doi="10.3168/jds.2010-3255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrooten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Imputation of genotypes from different single nucleotide polymorphism panels in dairy cattle</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5443</span>–<span class="cit-lpage">5454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DDruet%26rft.auinit1%253DT.%26rft.volume%253D93%26rft.issue%253D11%26rft.spage%253D5443%26rft.epage%253D5454%26rft.atitle%253DImputation%2Bof%2Bgenotypes%2Bfrom%2Bdifferent%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bpanels%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3255%26rft_id%253Dinfo%253Apmid%252F20965360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20965360&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.9" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.10" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP</span>. <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.427.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foulkes</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Applied Statistical Genetics With R: For Population-Based Association Studies</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.12" data-doi="10.1534/genetics.108.100289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection using low-density marker panels</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100289%26rft_id%253Dinfo%253Apmid%252F19299339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.13" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection in dairy cattle: progress and challenges</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.14" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.15" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.16" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program</span>. <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.17" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.18" data-doi="10.2135/cropsci2011.07.0358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">654</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D654%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.07.0358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.07.0358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.19" data-doi="10.2135/cropsci2009.08.0434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Marker genotype imputation in a low-marker-density panel with a high-marker-density reference panel: accuracy evaluation in barley breeding lines</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DIwata%26rft.auinit1%253DH.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1269%26rft.epage%253D1278%26rft.atitle%253DMarker%2BGenotype%2BImputation%2Bin%2Ba%2BLow-Marker-Density%2BPanel%2Bwith%2Ba%2BHigh-Marker-Density%2BReference%2BPanel%253A%2BAccuracy%2BEvaluation%2Bin%2BBarley%2BBreeding%2BLines%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.08.0434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.08.0434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000279138200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trus</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Considerations on genetic connectedness between management units under an animal model</span>. <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">2341</span>–<span class="cit-lpage">2352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DKennedy%26rft.auinit1%253DB.%2BW.%26rft.volume%253D71%26rft.issue%253D9%26rft.spage%253D2341%26rft.epage%253D2352%26rft.atitle%253DConsiderations%2Bon%2Bgenetic%2Bconnectedness%2Bbetween%2Bmanagement%2Bunits%2Bunder%2Ban%2Banimal%2Bmodel%26rft_id%253Dinfo%253Apmid%252F8407646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8407646&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LV19300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">834</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D834%26rft.spage%253D816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.22" data-doi="10.1371/journal.pone.0024945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Performance of genotype imputation for rare variants identified in exons and flanking gegions of genes</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De24945%26rft.epage%253De24945%26rft.atitle%253DPerformance%2Bof%2Bgenotype%2Bimputation%2Bfor%2Brare%2Bvariants%2Bidentified%2Bin%2Bexons%2Band%2Bflanking%2Bregions%2Bof%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024945%26rft_id%253Dinfo%253Apmid%252F21949800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024945&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949800&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.23" data-doi="10.1198/016214505000001230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeon</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Random forests and adaptive nearest neighbors</span>. <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D101%26rft.spage%253D578%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214505000001230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214505000001230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000238033200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.24" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects</span>. <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.25" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.26" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.27" data-doi="10.1038/ng2088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new multipoint method for genome-wide association studies by imputation of genotypes</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">906</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D906%26rft.epage%253D913%26rft.atitle%253DA%2Bnew%2Bmultipoint%2Bmethod%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bby%2Bimputation%2Bof%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2088%26rft_id%253Dinfo%253Apmid%252F17572673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572673&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247619800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.29" data-doi="10.3168/jds.2011-4490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrooten</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Imputation of genotypes with low-density chips and its effect on reliability of direct genomic values in Dutch Holstein cattle</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">876</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMulder%26rft.auinit1%253DH.%2BA.%26rft.volume%253D95%26rft.issue%253D2%26rft.spage%253D876%26rft.epage%253D889%26rft.atitle%253DImputation%2Bof%2Bgenotypes%2Bwith%2Blow-density%2Bchips%2Band%2Bits%2Beffect%2Bon%2Breliability%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bin%2BDutch%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4490%26rft_id%253Dinfo%253Apmid%252F22281352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22281352&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.30" data-doi="10.1371/journal.pone.0003551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">Y.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papasian</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.-W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analyses and comparison of accuracy of different genotype imputation methods</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPei%26rft.auinit1%253DY.%2BF.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3551%26rft.epage%253De3551%26rft.atitle%253DAnalyses%2Band%2Bcomparison%2Bof%2Baccuracy%2Bof%2Bdifferent%2Bgenotype%2Bimputation%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003551%26rft_id%253Dinfo%253Apmid%252F18958166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18958166&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-web" id="cit-3.3.427.31"><div class="cit-metadata unstructured">Perry, P. O., 2009 bcv: Cross-Validation for the SVD. R package version 1.0. Available at: <a href="http://CRAN.R-project.org/package=bcv/">http://CRAN.R-project.org/package=bcv/</a>. Accessed: July 30, 2012.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.32" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics</span>. <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.33" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing</span>. <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-3.3.427.34"><div class="cit-metadata unstructured">R Development Core Team, 2011 R: A Language and Environment for Statistical Computing, Vienna. Available at: <a href="http://www.r-project.org/">http://www.r-project.org/</a>. Accessed: July 30, 2012.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.427.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Variance Components</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stekhoven</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MissForest - nonparametric missing value imputation for mixed-type data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D28%26rft.spage%253D112%26rft_id%253Dinfo%253Apmid%252F22039212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22039212&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the lasso</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B. Met.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB.%2BMet.%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.38" data-doi="10.1093/bioinformatics/17.6.520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Missing value estimation methods for DNA microarrays</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DTroyanskaya%26rft.auinit1%253DO.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D520%26rft.epage%253D525%26rft.atitle%253DMissing%2Bvalue%2Bestimation%2Bmethods%2Bfor%2BDNA%2Bmicroarrays%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.6.520%26rft_id%253Dinfo%253Apmid%252F11395428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNy82LzUyMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-3.3.427.39"><div class="cit-metadata unstructured">Warnes, G., G. Gorjanc, F. Leisch, and M. Man, 2011 Genetics: Population Genetics. R package version 1.3.6. Available at: <a href="http://CRAN.R-project.org/package=genetics/">http://CRAN.R-project.org/package=genetics/</a>. Accessed: July 30, 2012.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.40" data-doi="10.3168/jds.2010-3149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5423</span>–<span class="cit-lpage">5435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWeigel%26rft.auinit1%253DK.%2BA.%26rft.volume%253D93%26rft.issue%253D11%26rft.spage%253D5423%26rft.epage%253D5435%26rft.atitle%253DAccuracy%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bderived%2Bfrom%2Bimputed%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenotypes%2Bin%2BJersey%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3149%26rft_id%253Dinfo%253Apmid%252F20965358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20965358&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.41" data-doi="10.1017/S0016672399004462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denham</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection using ridge regression</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWhittaker%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D75%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D252%26rft.atitle%253DMarker-assisted%2Bselection%2Busing%2Bridge%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004462%26rft_id%253Dinfo%253Apmid%252F10816982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10816982&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086569100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.42" data-doi="10.1534/g3.112.003699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D2%26rft.spage%253D1427%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.003699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.43" data-doi="10.1007/s00122-008-0715-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWong%26rft.auinit1%253DC.%2BK.%26rft.volume%253D116%26rft.issue%253D6%26rft.spage%253D815%26rft.epage%253D824%26rft.atitle%253DGenomewide%2Bselection%2Bin%2Boil%2Bpalm%253A%2Bincreasing%2Bselection%2Bgain%2Bper%2Bunit%2Btime%2Band%2Bcost%2Bwith%2Bsmall%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0715-5%26rft_id%253Dinfo%253Apmid%252F18219476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0715-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18219476&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254262600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.44" data-doi="10.3168/jds.2010-3501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Marker imputation with low-density marker panels in Dutch Holstein cattle</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5487</span>–<span class="cit-lpage">5494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D93%26rft.issue%253D11%26rft.spage%253D5487%26rft.epage%253D5494%26rft.atitle%253DMarker%2Bimputation%2Bwith%2Blow-density%2Bmarker%2Bpanels%2Bin%2BDutch%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3501%26rft_id%253Dinfo%253Apmid%252F20965364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20965364&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283532700051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.427.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D124%26rft.spage%253D769%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/3/427.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/409" title="A Mosaic Genetic Screen for Genes Involved in the Early Steps of Drosophila Oogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/441" title="The Genetic Architecture of Degenerin/Epithelial Sodium Channels in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/3"><span>Volume 3 Issue 3, March 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/3.cover-source.jpg?itok=w5x3d6rp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o-GUq1pI_kWR5WTTNlIrLo7WFb9ykUBA3puUgzKE24A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/3/427" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/3/427" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/3/427" class="active" data-icon-position="" data-hide-link-title="0">Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459422" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9N0HzKX5EQKBwXTSSVlbzVnI4VkKofWWNHIi_VuUdQ4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/427.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/3/427" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-pLV7YR4XQ8zVmlMEAREBkc4fFwMp9XqY6cfzin2Fq4o" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/3/427" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/427.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459422" id="node459422--2" data-pisa="ggg;3/3/427" data-pisa-master="ggg;3/3/427" data-apath="/ggg/3/3/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jessica E. Rutkoski</span>, <span class="highwire-citation-author" data-delta="1">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="2">Jean-Luc Jannink</span> and <span class="highwire-citation-author" data-delta="3">Mark E. Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-439;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005363 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459422--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jessica E. Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BE.%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BE.%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Manhattan, Kansas 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mes12@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mes12@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/3/427" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/3/427&amp;title=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/3/427&amp;title=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/3/427&amp;title=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/3/427&amp;text=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/3/427&amp;title=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/3/427&amp;t=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/3/427&amp;title=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/3/427&amp;title=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459422/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459422" id="node459422--3" data-pisa="ggg;3/3/427" data-pisa-master="ggg;3/3/427" data-apath="/ggg/3/3/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jessica E. Rutkoski</span>, <span class="highwire-citation-author" data-delta="1">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="2">Jean-Luc Jannink</span> and <span class="highwire-citation-author" data-delta="3">Mark E. Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-439;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005363 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459422--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jessica E. Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BE.%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BE.%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Manhattan, Kansas 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mes12@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mes12@cornell.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=427&amp;rft.epage=439&amp;rft.atitle=Imputation%20of%20Unordered%20Markers%20and%20the%20Impact%20on%20Genomic%20Selection%20Accuracy&amp;rft.volume=3&amp;rft.issue=3&amp;rft.date=2013-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rutkoski%2C+Jessica+E.&amp;rft.au=Poland%2C+Jesse&amp;rft.au=Jannink%2C+Jean-Luc&amp;rft.au=Sorrells%2C+Mark+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459422/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459422/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459422/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459422/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459422/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459422/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459422/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459422/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459422/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459422/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459422/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459422/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/3/427" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/3/427.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/3/427.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/3/427.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/3/427.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
