<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/3/505" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/3/505.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/3/505.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/3/505.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/3/505.atom" />
<meta name="HW.pisa" content="ggg;3/3/505" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria" />
<meta name="DC.Identifier" content="10.1534/g3.112.005231" />
<meta name="DC.Date" content="2013-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Raymann et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Bacteria and eukaryotes are involved in many types of interaction in nature, with important ecological consequences. However, the diversity, occurrence, and mechanisms of these interactions often are not fully known. The obligate bacterial endosymbionts of Paramecium provide their hosts with the ability to kill sensitive Paramecium strains through the production of R-bodies, highly insoluble coiled protein ribbons. R-bodies have been observed in a number of free-living bacteria, where their function is unknown. We have performed an exhaustive survey of genes coding for homologs of Reb proteins (R-body components) in complete bacterial genomes. We found that reb genes are much more widespread than previously thought, being present in representatives of major Proteobacterial subdivisions, including many free-living taxa, as well as taxa known to be involved in various kinds of interactions with eukaryotes, from mutualistic associations to pathogenicity. Reb proteins display very good conservation at the sequence level, suggesting that they may produce functional R-bodies. Phylogenomic analysis indicates that reb genes underwent a complex evolutionary history and allowed the identification of candidates potentially involved in R-body assembly, functioning, regulation, or toxicity. Our results strongly suggest that the ability to produce R-bodies is likely widespread in Proteobacteria. The potential involvement of R-bodies in as yet unexplored interactions with eukaryotes and the consequent ecological implications are discussed." />
<meta name="DC.Contributor" content="Kasie Raymann" />
<meta name="DC.Contributor" content="Louis-Marie Bobay" />
<meta name="DC.Contributor" content="Thomas G. Doak" />
<meta name="DC.Contributor" content="Michael Lynch" />
<meta name="DC.Contributor" content="Simonetta Gribaldo" />
<meta name="article:published_time" content="2013-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Bacteria and eukaryotes are involved in many types of interaction in nature, with important ecological consequences. However, the diversity, occurrence, and mechanisms of these interactions often are not fully known. The obligate bacterial endosymbionts of Paramecium provide their hosts with the ability to kill sensitive Paramecium strains through the production of R-bodies, highly insoluble coiled protein ribbons. R-bodies have been observed in a number of free-living bacteria, where their function is unknown. We have performed an exhaustive survey of genes coding for homologs of Reb proteins (R-body components) in complete bacterial genomes. We found that &lt;i&gt;reb&lt;/i&gt; genes are much more widespread than previously thought, being present in representatives of major Proteobacterial subdivisions, including many free-living taxa, as well as taxa known to be involved in various kinds of interactions with eukaryotes, from mutualistic associations to pathogenicity. Reb proteins display very good conservation at the sequence level, suggesting that they may produce functional R-bodies. Phylogenomic analysis indicates that &lt;i&gt;reb&lt;/i&gt; genes underwent a complex evolutionary history and allowed the identification of candidates potentially involved in R-body assembly, functioning, regulation, or toxicity. Our results strongly suggest that the ability to produce R-bodies is likely widespread in Proteobacteria. The potential involvement of R-bodies in as yet unexplored interactions with eukaryotes and the consequent ecological implications are discussed.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/03/01" />
<meta name="citation_mjid" content="ggg;3/3/505" />
<meta name="citation_id" content="3/3/505" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/3/505" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/3/505.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/3/505.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/3/505.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005231" />
<meta name="citation_pmid" content="23450193" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="505" />
<meta name="citation_lastpage" content="516" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kasie Raymann" />
<meta name="citation_author_institution" content="Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France" />
<meta name="citation_author_institution" content="Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France" />
<meta name="citation_author" content="Louis-Marie Bobay" />
<meta name="citation_author_institution" content="Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France" />
<meta name="citation_author_institution" content="Institut Pasteur, Microbial Evolutionary Genomics, Departement Genomes et Genetique, Paris, 75724 Cedex 15, France" />
<meta name="citation_author_institution" content="Centre National de la Recherche Scientifique, Unité Mixte de Recherche 3525, Paris, F-75015 France" />
<meta name="citation_author" content="Thomas G. Doak" />
<meta name="citation_author_institution" content="Indiana University, Department of Biology, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Michael Lynch" />
<meta name="citation_author_institution" content="Indiana University, Department of Biology, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Simonetta Gribaldo" />
<meta name="citation_author_institution" content="Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France" />
<meta name="citation_author_email" content="simonetta.gribaldo@pasteur.fr" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=N. Akiba;citation_author=T. Aono;citation_author=H. Toyazaki;citation_author=S. Sato;citation_author=H. Oyaizu;citation_title=phrR-like gene praR of Azorhizobium caulinodans ORS571 is essential for symbiosis with Sesbania rostrata and is involved in expression of reb genes;citation_volume=76;citation_year=2010;citation_pmid=20382809;citation_doi=10.1128/AEM.00238-10" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_title=Paramecium: Genetics and Epigenetics;citation_author=G. Beale;citation_author=J. R. Preer;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=C. L. Beier;citation_author=M. Horn;citation_author=R. Michel;citation_author=M. Schweikert;citation_author=H. D. Gortz;citation_title=The Genus Caedibacter Comprises Endosymbionts of Paramecium spp. Related to the Rickettsiales (Alphaproteobacteria) and to Francisella tularensis (Gammaproteobacteria);citation_pages=6043-6050;citation_volume=68;citation_year=2002;citation_issue=12;citation_pmid=12450827;citation_doi=10.1128/AEM.68.12.6043-6050.2002" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=M. R. Brown;citation_author=J. Barker;citation_title=Unexplored reservoirs of pathogenic bacteria: protozoa and biofilms.;citation_pages=46-50;citation_volume=7;citation_year=1999;citation_issue=1;citation_pmid=10068997;citation_doi=10.1016/S0966-842X(98)01425-5" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. W. Buchan;citation_author=S. M. Ward;citation_author=A. E. Lobley;citation_author=T. C. Nugent;citation_author=K. Bryson;citation_title=PROTEIN ANNOTATION AND MODELLING SERVERS AT UNIVERSITY COLLEGE LONDON;citation_pages=W563 568;citation_volume=38;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=A. Criscuolo;citation_author=S. Gribaldo;citation_title=BMGE (Block Mapping and Gathering with Entropy): a new software for selection of phylogenetic informative regions from multiple sequence alignments.;citation_pages=210-210;citation_volume=10;citation_year=2010;citation_pmid=20626897;citation_doi=10.1186/1471-2148-10-210" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. A. Dilts;citation_author=R. L. Quackenbush;citation_title=A mutation in the R body-coding sequence destroys expression of the killer trait in P. tetraurelia;citation_pages=641-643;citation_volume=232;citation_year=1986;citation_issue=4750;citation_pmid=3008334;citation_doi=10.1126/science.3008334" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. C. Edgar;citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity.;citation_pages=113-113;citation_volume=5;citation_year=2004;citation_pmid=15318951;citation_doi=10.1186/1471-2105-5-113" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Ecol.;citation_author=M. Erken;citation_author=M. Weitere;citation_author=S. Kjelleberg;citation_author=D. McDougald;citation_title=In situ grazing resistance of Vibrio cholerae in the marine environment;citation_volume=76;citation_year=2011;citation_pmid=21314704;citation_doi=10.1111/j.1574-6941.2011.01067.x" />
<meta name="citation_reference" content="citation_journal_title=Antonie van Leeuwenhoek;citation_journal_abbrev=Antonie van Leeuwenhoek;citation_author=M. J. Espuny;citation_author=C. Andres;citation_author=M. E. Mercade;citation_author=M. Robert;citation_author=M. A. Manresa;citation_title=R-bodies in Pseudomonas aeruginosa strain 44T1.;citation_pages=83-86;citation_volume=60;citation_year=1991;citation_issue=2;citation_pmid=1804030;citation_doi=10.1007/BF00572696" />
<meta name="citation_reference" content="citation_journal_title=Antonie van Leeuwenhoek;citation_journal_abbrev=Antonie van Leeuwenhoek;citation_author=J. Favinger;citation_author=R. Stadtwald;citation_author=H. Gest;citation_title=Rhodospirillum centenum, sp. nov., a thermotolerant cyst-forming anoxygenic photosynthetic bacterium.;citation_pages=291-296;citation_volume=55;citation_year=1989;citation_issue=3;citation_pmid=2757370;citation_doi=10.1007/BF00393857" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=M. Fusté;citation_author=M. Simon-Pujol;citation_author=A. Marques;citation_author=J. Guinea;citation_author=F. Congregrado;citation_title=Isolation of a New Free-living Bacterium Containing R-bodies;citation_pages=2801-2805;citation_volume=132;citation_year=1986;citation_issue=10;citation_doi=10.1099/00221287-132-10-2801" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=R. J. Gast;citation_author=R. W. Sanders;citation_author=D. A. Caron;citation_title=Ecological strategies of protists and their symbiotic relationships with prokaryotic microbes.;citation_pages=563-569;citation_volume=17;citation_year=2009;citation_issue=12;citation_pmid=19828317;citation_doi=10.1016/j.tim.2009.09.001" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=I. Gibson;citation_title=Transplantation of killer endosymbionts in paramecium.;citation_pages=127-129;citation_volume=241;citation_year=1973;citation_issue=5385;citation_pmid=4633042;citation_doi=10.1038/241127a0" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=D. Hernandez-Romero;citation_author=P. Lucas-Elio;citation_author=D. Lopez-Serrano;citation_author=F. Solano;citation_author=A. Sanchez-Amat;citation_title=Marinomonas mediterranea is a lysogenic bacterium that synthesizes R-bodies;citation_pages=2679-2686;citation_volume=149;citation_year=2003;citation_issue=9;citation_pmid=12949192;citation_doi=10.1099/mic.0.26524-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=D. P. Heruth;citation_author=F. R. Pond;citation_author=J. A. Dilts;citation_author=R. L. Quackenbush;citation_title=Characterization of genetic determinants for R body synthesis and assembly in Caedibacter taeniospiralis 47 and 116.;citation_pages=3559-3567;citation_volume=176;citation_year=1994;citation_issue=12;citation_pmid=8206833;citation_doi=10.1128/jb.176.12.3559-3567.1994" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. Jeblick;citation_author=J. Kusch;citation_title=Sequence, transcription activity, and evolutionary origin of the R-body coding plasmid pKAP298 from the intracellular parasitic bacterium Caedibacter taeniospiralis.;citation_pages=164-173;citation_volume=60;citation_year=2005;citation_issue=2;citation_pmid=15785846;citation_doi=10.1007/s00239-004-0002-2" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=G. Jobb;citation_author=A. von Haeseler;citation_author=K. Strimmer;citation_title=TREEFINDER: a powerful graphical analysis environment for molecular phylogenetics.;citation_pages=18-18;citation_volume=4;citation_year=2004;citation_pmid=15222900;citation_doi=10.1186/1471-2148-4-18" />
<meta name="citation_reference" content="citation_journal_title=The Journal of experimental zoology;citation_journal_abbrev=J Exp Zool;citation_author=A. Jurand;citation_author=B. M. Rudman;citation_author=J. R. Preer;citation_title=Prelethal effects of killing action by stock 7 of Paramecium aurelia.;citation_pages=365-387;citation_volume=177;citation_year=1971;citation_issue=3;citation_pmid=4938048;citation_doi=10.1002/jez.1401770311" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. A. Kanabrocki;citation_author=R. L. Quackenbush;citation_author=F. R. Pond;citation_title=Organization and expression of genetic determinants for synthesis and assembly of type 51 R bodies.;citation_pages=40-48;citation_volume=168;citation_year=1986;citation_issue=1;citation_pmid=3759909;citation_doi=10.1128/jb.168.1.40-48.1986" />
<meta name="citation_reference" content="citation_journal_title=Protist;citation_author=J. Kusch;citation_author=L. Czubatinski;citation_author=S. Wegmann;citation_author=M. Hubner;citation_author=M. Alter;citation_title=Competitive advantages of Caedibacter-infected Paramecia;citation_pages=47-58;citation_volume=153;citation_year=2002;citation_pmid=12022275;citation_doi=10.1078/1434-4610-00082" />
<meta name="citation_reference" content="citation_journal_title=Zeitschrift fÃ¼r allgemeine Mikrobiologie;citation_journal_abbrev=Z Allg Mikrobiol;citation_author=J. Lalucat;citation_author=F. Mayer;citation_title=&amp;quot;Spiral bodies&amp;quot;--intracytoplasmic membraneous structures in a hydrogen oxidizing bacterium.;citation_pages=517-521;citation_volume=18;citation_year=1978;citation_issue=7;citation_pmid=735162;citation_doi=10.1002/jobm.3630180708" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. G. Langille;citation_author=F. S. Brinkman;citation_title=IslandViewer: an integrated interface for computational identification and visualization of genomic islands;citation_volume=25;citation_year=2009;citation_pmid=19151094;citation_doi=10.1093/bioinformatics/btp030" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Lartillot;citation_author=T. Lepage;citation_author=S. Blanquart;citation_title=PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating;citation_volume=25;citation_year=2009;citation_pmid=19535536;citation_doi=10.1093/bioinformatics/btp368" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=C. Matz;citation_author=S. Kjelleberg;citation_title=Off the hook--how bacteria survive protozoan grazing.;citation_pages=302-307;citation_volume=13;citation_year=2005;citation_issue=7;citation_pmid=15935676;citation_doi=10.1016/j.tim.2005.05.009" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=C. Matz;citation_author=T. Bergfeld;citation_author=S. A. Rice;citation_author=S. Kjelleberg;citation_title=Microcolonies, quorum sensing and cytotoxicity determine the survival of Pseudomonas aeruginosa biofilms exposed to protozoan grazing.;citation_pages=218-226;citation_volume=6;citation_year=2004;citation_issue=3;citation_pmid=14871206;citation_doi=10.1111/j.1462-2920.2004.00556.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. Paul;citation_author=G. Pohnert;citation_title=Interactions of the algicidal bacterium Kordia algicida with diatoms: regulated protease excretion for specific algal lysis.;citation_pages=e21032-e21032;citation_volume=6;citation_year=2011;citation_issue=6;citation_pmid=21695044;citation_doi=10.1371/journal.pone.0021032" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. Philippe;citation_title=MUST, a computer package of Management Utilities for Sequences and Trees;citation_pages=5264-5272;citation_volume=21;citation_year=1993;citation_issue=22;citation_pmid=8255784;citation_doi=10.1093/nar/21.22.5264" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=F. R. Pond;citation_author=I. Gibson;citation_author=J. Lalucat;citation_author=R. L. Quackenbush;citation_title=R-body-producing bacteria.;citation_pages=25-67;citation_volume=53;citation_year=1989;citation_issue=1;citation_pmid=2651865" />
<meta name="citation_reference" content="citation_title=Symp. Soc. Exp. Biol.;citation_author=J. R. Preer;citation_title=The hereditary symbionts of Paramecium aurelia;citation_pages=125-144;citation_volume=29;citation_year=1975;citation_pmid=785658" />
<meta name="citation_reference" content="citation_journal_title=Genet. Soc. Am.;citation_author=J. R. Preer;citation_title=Sonneborn and the Cytoplasm;citation_pages=1373-1377;citation_volume=172;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Experimental cell research;citation_journal_abbrev=Exp Cell Res;citation_author=J. R. Preer;citation_author=P. Stark;citation_title=Cytological observations on the cytoplasmic factor &amp;quot;kappa&amp;quot; in Paramecium aurelia.;citation_pages=478-491;citation_volume=5;citation_year=1953;citation_issue=2;citation_pmid=13117017;citation_doi=10.1016/0014-4827(53)90234-1" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=J. R. Preer;citation_author=L. B. Preer;citation_author=A. Jurand;citation_title=Kappa and other endosymbionts in Paramecium aurelia.;citation_pages=113-163;citation_volume=38;citation_year=1974;citation_issue=2;citation_pmid=4599970" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. L. Quackenbush;citation_author=J. A. Burbach;citation_title=Cloning and expression of DNA sequences associated with the killer trait of Paramecium tetraurelia stock 47.;citation_pages=250-254;citation_volume=80;citation_year=1983;citation_issue=1;citation_pmid=6571998;citation_doi=10.1073/pnas.80.1.250" />
<meta name="citation_reference" content="citation_author=R Development Core Team;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. L. Salzberg;citation_author=D. D. Sommer;citation_author=M. C. Schatz;citation_author=A. M. Phillippy;citation_author=P. D. Rabinowicz;citation_title=Genome sequence and rapid evolution of the rice pathogen Xanthomonas oryzae pv. oryzae PXO99A.;citation_pages=204-204;citation_volume=9;citation_year=2008;citation_pmid=18452608;citation_doi=10.1186/1471-2164-9-204" />
<meta name="citation_reference" content="citation_title=Microbiol. Monogr.;citation_author=A. Sanchez-Amat;citation_title=R-bodies;citation_pages=331-341;citation_volume=1;citation_year=2006;citation_doi=10.1007/3-540-33774-1_14" />
<meta name="citation_reference" content="citation_journal_abbrev=Eur J Protistol;citation_author=M. Schrallhammer;citation_author=S. Galati;citation_author=J. Altenbuchner;citation_author=M. Schweikert;citation_author=H. D. Gortz;citation_title=Tracing the role of R-bodies in the killer trait: absence of toxicity of R-body producing recombinant E. coli on paramecia.;citation_pages=290-296;citation_volume=48;citation_year=2012;citation_issue=4;citation_pmid=22356923;citation_doi=10.1016/j.ejop.2012.01.008" />
<meta name="citation_reference" content="citation_journal_title=Proc. Am. Philos. Soc.;citation_author=T. M. Sonneborn;citation_title=Mating types in P. aurelia: diverse conditions for mating in different stocks; occurrence, number and interrelations of the types;citation_pages=411-434;citation_volume=79;citation_year=1938" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. M. Sonneborn;citation_title=Gene and Cytoplasm: II. The Bearing of the Determination and Inheritance of Characters in Paramecium Aurelia on the Problems of Cytoplasmic Inheritance, Pneumococcus Transformations, Mutations and Development;citation_pages=338-343;citation_volume=29;citation_year=1943;citation_issue=11;citation_pmid=16588623;citation_doi=10.1073/pnas.29.11.338" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biotechnology;citation_journal_abbrev=Journal of Biotechnology;citation_author=F. J. Vorholter;citation_author=T. Thias;citation_author=F. Meyer;citation_author=T. Bekel;citation_author=O. Kaiser;citation_title=Comparison of two Xanthomonas campestris pathovar campestris genomes revealed differences in their gene composition.;citation_pages=193-202;citation_volume=106;citation_year=2003;citation_issue=2-3;citation_pmid=14651861;citation_doi=10.1016/j.jbiotec.2003.08.011" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=M. Weitere;citation_author=T. Bergfeld;citation_author=S. A. Rice;citation_author=C. Matz;citation_author=S. Kjelleberg;citation_title=Grazing resistance of Pseudomonas aeruginosa biofilms depends on type of protective mechanism, developmental stage and protozoan feeding mode.;citation_pages=1593-1601;citation_volume=7;citation_year=2005;citation_issue=10;citation_pmid=16156732;citation_doi=10.1111/j.1462-2920.2005.00851.x" />
<meta name="citation_reference" content="citation_title=Micron;citation_author=B. Wells;citation_author=R. W. Horne;citation_title=The ultrastructure of Pseudomonas avenae II. Intracellular refractile (R-body) structure;citation_pages=329-344;citation_volume=14;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Winstanley;citation_author=M. G. Langille;citation_author=J. L. Fothergill;citation_author=I. Kukavica-Ibrulj;citation_author=C. Paradis-Bleau;citation_title=Newly introduced genomic prophage islands are critical determinants of in vivo competitiveness in the Liverpool epidemic strain of Pseudomonas aeruginosa;citation_volume=19;citation_year=2009;citation_pmid=19047519;citation_doi=10.1101/gr.086082.108" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=T. Yamagishi;citation_author=A. Kai;citation_author=H. Kawai;citation_title=Trichocyst ribbons of a cryptomonads are constituted of homologs of R-body proteins produced by the intracellular parasitic bacterium of Paramecium.;citation_pages=147-157;citation_volume=74;citation_year=2012;citation_issue=3-4;citation_pmid=22447322;citation_doi=10.1007/s00239-012-9495-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Zhou;citation_author=Y. Liang;citation_author=K. H. Lynch;citation_author=J. J. Dennis;citation_author=D. S. Wishart;citation_title=PHAST: A FAST PHAGE SEARCH TOOL;citation_pages=W347 352;citation_volume=39;citation_year=2011" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="twitter:description" content="Bacteria and eukaryotes are involved in many types of interaction in nature, with important ecological consequences. However, the diversity, occurrence, and mechanisms of these interactions often are not fully known. The obligate bacterial endosymbionts of Paramecium provide their hosts with the ability to kill sensitive Paramecium strains through the production of R-bodies, highly insoluble coiled protein ribbons. R-bodies have been observed in a number of free-living bacteria, where their function is unknown. We have performed an exhaustive survey of genes coding for homologs of Reb proteins (R-body components) in complete bacterial genomes. We found that reb genes are much more widespread than previously thought, being present in representatives of major Proteobacterial subdivisions, including many free-living taxa, as well as taxa known to be involved in various kinds of interactions with eukaryotes, from mutualistic associations to pathogenicity. Reb proteins display very good conservation at the sequence level, suggesting that they may produce functional R-bodies. Phylogenomic analysis indicates that reb genes underwent a complex evolutionary history and allowed the identification of candidates potentially involved in R-body assembly, functioning, regulation, or toxicity. Our results strongly suggest that the ability to produce R-bodies is likely widespread in Proteobacteria. The potential involvement of R-bodies in as yet unexplored interactions with eukaryotes and the consequent ecological implications are discussed." />
<meta name="og-title" property="og:title" content="A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/3/505" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Bacteria and eukaryotes are involved in many types of interaction in nature, with important ecological consequences. However, the diversity, occurrence, and mechanisms of these interactions often are not fully known. The obligate bacterial endosymbionts of Paramecium provide their hosts with the ability to kill sensitive Paramecium strains through the production of R-bodies, highly insoluble coiled protein ribbons. R-bodies have been observed in a number of free-living bacteria, where their function is unknown. We have performed an exhaustive survey of genes coding for homologs of Reb proteins (R-body components) in complete bacterial genomes. We found that reb genes are much more widespread than previously thought, being present in representatives of major Proteobacterial subdivisions, including many free-living taxa, as well as taxa known to be involved in various kinds of interactions with eukaryotes, from mutualistic associations to pathogenicity. Reb proteins display very good conservation at the sequence level, suggesting that they may produce functional R-bodies. Phylogenomic analysis indicates that reb genes underwent a complex evolutionary history and allowed the identification of candidates potentially involved in R-body assembly, functioning, regulation, or toxicity. Our results strongly suggest that the ability to produce R-bodies is likely widespread in Proteobacteria. The potential involvement of R-bodies in as yet unexplored interactions with eukaryotes and the consequent ecological implications are discussed." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
    <title>A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wHfNaaynFVq9dYB6M8lu6-3TWxsCerzlqYQcXAI6EDA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459427","apath":"\/ggg\/3\/3\/505.atom","pisa":"ggg;3\/3\/505","ac":{"\/ggg\/3\/3\/505.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/3\/505.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/3\/505"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/3\/505"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/3\/505":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225yA7JW1RUI+N\\\\\\\/U9msko6M7G2+vBYN2sUhuUd8+tazAu1039RJ4MxUuJIzWCFUvmqS+At4ah5emXm3gOylKdKcXb0L3KkakHlrJ6fIfcA8OCJ04DDOuybJN0xS2rdqSzvSaY6ht1hq52poAQhB\\\\\\\/bKYvo0IgV1lAxOWkyC\\\\\\\/GuXvl9P41tPNbpuii8BxJ7vBzsMm2X+1iYN\\\\\\\/WWpewYtQxdq2mFC0i3PQAwkPEbu5SxXKMSbNsJrvPQu4srC6pNVac2CpQ8yJxy1nsf3ePs6ob5Z2YlhlcOM+WabRCH8mEEEqRvcLy87TWGD76vKu9AevhGvhByzE6yNfEWra\\\\\\\/hsAoLFkwBZtMh2MAiVs4bd57VgS4c26qWXy\\\\\\\/MKG1r3lNLtil8RhoEYnj4ABbEFWQwroQXS7TNRJ9R0Qv5a1o2Xy8eapeblAwpQJyB3Ve9LtPDbO7hjuKsk5ON0V2g2zOtLiLdVw4kxJvQ6riN\\\\\\\/sQANAz7UYpk9GrH4DsGHd2rSpFVE2NMt84szex1VtGZAna43BX3Oo3fmPGu49gczIubOtKl\\\\\\\/ghVcEfkqJADx5MSBVh8Qb5vWO78Adl9A3mznXz8E+raJVihZpwQgmC13nvcIgr\\\\\\\/x7SZk5nji+1EzIDmEj7tBXt7P1keavfliIhAjBljmwltR51yCyXLxRUO0FDADGIJZCZuGzsnBCNKRTQnbKcfgglRlGd05SkgoOiRj9V5nBEYyERimw\\\\\\\/GCberGam4xn67kACyRBbAgY2UTrwi8QPpPe7+v8qbX801152fDhAmxvfp\\\\\\\/Cz9EKyPcE9LdsQQCKfjxvO1BxSfwMXwwJ5VKXZTfpBa4aR\\\\\\\/fstGToyooUXFbvr4Mr445e0+SQ3kM4qR4f9TSG0hoiBPy0o6urX9XFwDshO+JXtym+k90L07JfUM57skTlJlxe2Ix+HiwYzUKBfXppyWaLPUvvXsG\\\\\\\/XgdD6ndlxQ+UoJ8l429q3J1\\\\\\\/Hs2l6KjZ4VJOaW\\\\\\\/WJZIDGWhLt6zRZlSGywjsbNUtmpLbwSCzOoZeMpk38tFFSigWWi8oJGR5ZXZmoXLVwO3qt46z6+wZ9QMs3RB1JzshVcyUcsRUbDO5hihLf6zyTgC9HY5+7t3a2p9JJeBN2tLd5IxVc6KN5y+xzaSzBuGrjhrbdVM23z0n3dNcMWdldY\\\\\\\/VL62ZoztO99buXUy+plkF5uTYA1\\\\\\\/\\\\\\\/sKw5ZdWmpuebDZz+cOOaFvsV1mJWQbswufo4iajmOTPbsiCPEwzhga9S4dfaTpJlqXgVPHLgkyV3Lm9Drk\\\\\\\/G3Vi\\\u0022,\\\u0022iv\\\u0022:\\\u00220L38L\\\\\\\/9SY4LC5Be3vLOwlA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c137593b8f31e562f4e3bf7ba26caf84fdbad7efed688ec984498bafa9518fb7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022k7YJVMYA3TGnBHmT3dMSHM8\\\\\\\/EfU773D8SDj+ue5+nLlqCL5mBp8jR+kUkGucBKAUMB9L0pOPu63wOvHDueDSpzL6KoPFPMXwbuG3K0JMFk5mZVOGAVFWBBoZdsZzCBC0Fnc7chuj5fOsqds6hMWY856x7SvUvtTItXJSl2bJCS\\\\\\\/On2e9BQBTPVD4vYK7IeMECmlO4urn2V13OGEliv8rNFodbS0TfKXaSe2k\\\\\\\/ywnA1xlcptNbfC79W59GcRjdppP\\\\\\\/ahZKT5IGdYa2OcFAx8VOGR+g6JPSMVOFFWo24s7\\\\\\\/oe8PbkZlSIL\\\\\\\/KTjhqlHoiqFFWrVM97gXUmZvUOq1lwMZcL698Ag0Vryop6Ivw02SkGxy7Fmky3pkhNXCesT529p6pjCpL7jaPxb7DHiX1R9UmUCQ3JjnqeTTIN2QEf4J8+5BzbnqzWWU6r9wBePSfLGkmpz1VVajMbiejfJQux7bVAsBx+1gGmSxw503b2niNSm5ZFV5YGLk8oEUf\\\\\\\/4bXkaEHAP2hy4tKFGHYcwfjA\\\\\\\/O4gve\\\\\\\/bdUAB+6mfH3PxoWmjPg8blX7JLY3y+0dYPr18GRyClryMiTKLsNgdCKkQHZDnmk4ujHyjUBzq0pdCzzo0WEMh5Cu7eCju3BAx7S2WymzvxO5OzaluFg6muJsNZ9nJlE3ec5d2+pyZZi7v7mRPqZc+D3PHbD827ASFqVPLUFryXvxD53nCbwC3zgMSeUFGiPVN56SPT+D1PTuv7519ZG3q0UYZP4Xan+uW4+vqn5vC6eA1DQAmlDc\\\\\\\/jt+IR6gp2Oq2m+WjkFj5XazX810XvPPT2tHujrj8\\\\\\\/d1dAJvSxtivWRzUvcn4zQft8emkwNIKzQxx3tb8b82NfEMOqFOlDqgo2IyU2tfTZAYPpubQvKeP2Td+JxyOSu21C0wzTVRMUSh7IwTxIuAqrCpURVmsaFG7SgFkZZgJapbwAXAgp3Fbw7bYEvU\\\\\\\/BPsORCnMzDb3epOtjHPyAZ1imesvCSIMKcAhFwotvovpw4UXLxMzl2qAA5DUX09IHznwE8oiaHZrXo6Vq7xVutwtIPTw\\\\\\\/wXxp00m9m0XiQa5h2cvIltR4QVLOiT\\\\\\\/A6+\\\\\\\/ZIwbiMHEvFyEAUiOlzOG+vBe8ttssyAKbqSFiz5vO9R5EVKopTxIEz4qD6qKDBF\\\\\\\/Lw6zcBoIJcvwMDliwPG3gQE6GYgElyREyBiD\\\\\\\/rV8k1EIKcPei1JqzcyUQf5fDMCxs1dT2pYmnRLRILsTvVRhai\\\\\\\/AOy3yLkbyVDpRVmXW1OsLFQTVSQWFOP9V\\\\\\\/K9aDYCd7VOKcBPXa1ky8Xa4LkHlx+W1IySUZR06DKwhljpK+q9JNJ1trV6Cn6Pe1I2kxkfR\\\\\\\/8FZPnki6XoK4XOJaLkUiIPTqaHwcEv2mMIwMaAxGT6FCxgnBZBf\\\\\\\/MN67QTP4zDchMwqKsGSK\\\\\\\/AdKCjR+CcgSLhYkT84tys\\\\\\\/WSJuRPv4PawnUDWnJWAM7uWc9z4+zA2sFeq9h2XSVTHX2oKagBjxaLU+4zjRGVLrx\\\\\\\/lYjcLhOOBuEWVLwHIEmE0NKE9eBhrwhYaTMklaBCuySfX3ysaJIy56puEO5\\\\\\\/evhcjr+4Vb+hQZH33ZcmjRZh77n+ZT6284f7vRkUhUc++4\\\\\\\/uwexFuUjpx81Cq3PDMh73EsILzalPMTSWz0HrV3ztETWLnN6QJ409npmLN6lS4N6B0jkHAxV0wjGtOdTMzsqGCXkd4Q64gwMTHFT35DsqsrUTgw+8v7uvYBHVZquktU\\\\\\\/Y8ulbquJSY8LROOJbjStAEmISIwy69rrbM7eaYUSctUPgrFQcQq\\\\\\\/n4P\\\\\\\/oDVwwyVy3g4ivCbEpm7j71LEiRl8DsNaHxs+E0dYRyE7FaISuojHh6uIrLIU\\\\\\\/wwNNKZqjOCKQqN\\\\\\\/B2OTOgkJSkE5jxQD87hWH0HQf455NU31Qa\\\\\\\/2AXZYkZnvWdmUxss1pUHo8vDERw94MuNA2Ngx218\\\\\\\/BX+naya+hz4mkwsWODeGGuPOipfBMOvp+ZdFwzLKaIdVol\\\\\\\/BXa9m0VbJRaML6G3vez1x+fOLkVS7O7YCHUvMpC+NGyoeXnIwmsA4sF2EIXMwYGfqRXEx2mbMYD3PT2jKT+boX4WKXpD1YP6Cm8+bsjGQORA53o1w9Lc2B9nJR+TmUPoY0n10upR+zOE\\\\\\\/fmUaMTCRuddPlMqzHYv4IETfIMmLEdgYuFFb\\\\\\\/BxXYMwvzGgQouxAii2SvYlMGMEeOy6Fy\\\\\\\/8VFvinIboV4WTxr4+d1EELSPWaEHXHwgFHgpiuu3pCqFrKj1TYEXz8s\\\\\\\/qMoiC1O41gXzaU2RDzb5By2\\\\\\\/tx3yunp\\\\\\\/9VvICr04RlXFhVdP9kOxK3fhA2WYwdDqUgq0cGEuoKjnAIDxvsbT4oh6UXoNpWFRXtBgs8+uinPx0UvzBzQfu6alVoN7sHCc2McwoHcjvEzY30h7JawSxBfUk2Z7L6dnfpOiNyZPRpR\\\\\\\/3Q1NNxfvOMnr5zcB4V5paOnXgq\\\\\\\/aKAiHp79iS4Cu3dVJkIgbjwMf9UTkjiuW5AawmIpcuXH8W4umk75tPH7RVdns7Nu6pQjKfWWwCL5MUA5wfhj6vQDAW\\\\\\\/Zgt4hzoIBMpBfn1Olc8YAe+Z56YSK9cwQPoix3pJFhAxkRkhhwZT\\\\\\\/iI57BatxZ20ER5wT\\\\\\\/64o7YSqRG6fxf1cTxsmOLOqLW6\\\\\\\/mFd7cgodFvaWsvIKf9QPhjzYskIOubl99uDAdg9AQQNfP62q7pw5cczd696Ly5ZODG4UV+HzU28xhTOXAtIAOSMeDnnsRpu7Nd0TbdvALpv7Mm19xaOq2eHz2rCiAbLho6gKuxgJaLUAA+ctPOGZcKzILGZcDzs5Rh6XY9bPn0+\\\\\\\/bUkU1pohnZ9EeVC7Cqaw8tMZ636I0sylWYCGeDBzF6Pfvr\\\\\\\/oELOu\\\\\\\/dIJnF9OmSy\\\\\\\/sCPIIbkkITlqIPGJQA+25eKABjhzQkosTLkW6vjfMGQs5xladS2T5mDCa1VBb2wJzvL7sYd90Wq\\\\\\\/QwczkwKaTcDgV+vuip+XxE9SrgWofLOe5PzlvW0txOsun\\\\\\\/i7f7JyI8Po\\\\\\\/ySPsP342zyI0IsawcaHsTV1VHDfn4Ylr1m8RRPDajKYRLg\\\\\\\/Mz2TrVRmWoytJxtdjyZ3m+7pxEqfWPUwj6lJF85kIiIuL1EZOy0UykGi7DM5F6aYXFChEiG5rM8+mxHbmxThgwbJY7AjeB56f12qYJ41rNivn9Nmu9MaPc7RNc8jP1tHYgSNO8ryLRSuMY0vzd40Q0oVdV5DRh4qOxgZwMMwoc4+cwi9M0oG+6p3r5A9ObrifVG9GOK2lsMPsOGA66S90P3HEOBduXAyepMvNgYJV3oli7n5vW7sHDraQJAhm6LBSJZqSU0VDf8KdBloZSZOrvYVm9tVh2TpjRjhTZEVRsvRA2b0RsXC1hmqxXR+ovoOjLeZaBMCzfDXtTwmw0o\\\\\\\/5M+dBpE2TAfmBzGuiKQ11bylFTIEDZQEx8eWheFgIuFZcKPb4GtHBlpRfPSr3EsMF9C4o\\\\\\\/sFIARhDr+wnrhx+0vHDvSy9wIiuMA8LjmPHXaS0ZutDOE2GHle8Y\\\\\\\/cm1uHHn7oNyNZvANTYxj3cdL038B+RirI2MxPZdHDAUMEEDn+LQC1POKJszjtikmT57j+XJignrLYm3ZwikP+7k\\\\\\\/C0WdO8Az4\\\\\\\/ZDqB6VVW1yuy\\\\\\\/0K1Wb4HX2pdHEVX1+BwH2\\\\\\\/SHdZp3jYUnPqWMEdBARNUAiNgn36B6NohfUpbTWXp5dPDwQuWao0kKaaYcoPBLQR+D8gsPKS3lgtobsbELg\\\\\\\/R0Y8bYoizhUjQ+xbOWjkmtjTWkjPNrbtEJ3AMbp8qcMLslTL1bxTE8WD0ukENjssaui1uhO\\\\\\\/5kwtOCrsnOrmy1N2yW8L6nqhRa99kC2D+AvfqYa5mBaZBelcjre68fZGUGrP2iMXg1Nkd6YO5vkizkni9lxhFmupbCC2CSRgX1q3XLQBvjeKECm+F6cXR5TQnSXwhr0KcVPIu+Yzi20IT3N2xLLEO+G\\\\\\\/D7Vdjv9kyXSBHHJ8734KliSPGEg32jumDB3X3wMVTy6YfST1T7WVA2d3sgu9NVMFSrQnd7rvbJlZ+31cUgk9ik9A+D3GcuC5rJuR24Jc2cpJNKtujiTQhr\\\\\\\/BbuBYGpra\\\\\\\/j\\\\\\\/X8zw2\\\\\\\/46TnH7h9MKLLW89xXxdH9e8km56NlHTUSd1P1k\\\\\\\/OUo8ph1QyjFCvFlRxrlpjTDYDg8t+o5uEqyrmJD56LkMvUg30IgNICPOSkklRtck5b6b9P0b\\\\\\\/VuEDQLYFqcC+jfrKc3EUptfQ4LlKj20bYPXdZfvGBg1M9CUipPDelkxadJm6+gTsBMjYddMfHQfCTI2ljGHk2iFwcoZN8c5L8bRSQO2SnfHuKVnbThmborjKtZ8Y4s0UQ3RPzq2WsOPh9D5XnzimbYCZay2pFArrZ74XkfdLMMdRr2oBy8oqdifxPJfaf3428q6UBZCiZNCe7bU682Dvdz7phqwsuBhixGCUUf4TVEO\\\\\\\/2\\\\\\\/22pNfhThcFtyN4bw6Y82NMh3nl8mawFDdKrXTA9MwuA9uk3scqVwAmP1PF5arBHgovY76m5wl++LovKw7tJS7E6Y4rTmgy93jVehuK8ZkulLu7OwaBDqVg4IsGaDCDOfpXAG9Cf\\\\\\\/fohi6TL\\\\\\\/5OwpuVzO8FX6UeVnslWmlgJksEWl5LEUv5RvqQby+XECOa0hu3KduPg7T0N9JzfTnFzQXWjaOxOJu1sAwdKHTi2zngItqYluDG5zo9DFUJlwKVqli\\\\\\\/1W4QYZMwsyayTLAngmWpe0X1zqmjvGG+Hmdzwkm6YtmtWjOubZ3D6khmzSODc+slSPbMvz9FQ83kGHWYu5G8o3Arpy5O4GfXv0z6FxRXtDQRs8wAQl4m8v0fw3PjfvfoXUQCYckM5dEqgD6P181d8GWudKOeD7w6i8hfbq6z5sZ8YJPQQW\\\\\\\/VWcEMs9yAh+VE7WXntc3aQYpxo2nWXkRsdkgbUguJWE5Mk1BmVXNuKycMZDPF6IVY4ur7wr11zxua\\\\\\\/nEqLyqGD513oF9GvnVxparMSeHDOUddyUp7uKMeV4kgqv58lxi+DmGrJ7ojrdAgrV+6FqRFaEAkXOI22ypWdY\\\\\\\/jjaPBfBOXxdq13TYx0fBZg2BgZUHPpHlemOtsB8pfIm4pGvBOGekRrqw827innY7ZjRef\\\\\\\/\\\\\\\/NGO0XW1GCAABCEtT6krgz00U0\\\\\\\/Bv6xpcZy6\\\\\\\/bMJNBoOgOj3uOWbCj6c+GGM97VMGtkfAVsxby4fV7laTqGq2TpXTBIutADrWaSEKfiiPvU\\\\\\\/AlDVnOCSODCYdhTCvQ3uQqE4RNbwnj1JZ0O7N7F6df4hcnnFTbXuihcl\\\\\\\/91BK\\\\\\\/i2rfTRy4baYDaOuBfIBaQfo+K2RhkLFGWIxqiCHGIqJ4oNJBGACQDqdJHZUwzfg5do\\\\\\\/o7owPN19idRleigKUegctZNXAupLtR8wwcefT5biGAoYnrOj5W49kTEzOdsBfxiKWezm7h9A9yzewYQuYbDF\\\\\\\/WeJ8BjG5TRcwxWF3EKDL9adrqiLiENqgfUYzepJG0LZhDvyd+ENn0eT3AI02mIKKCsUKYA7enTSAzdpjS95SS\\\\\\\/qXT9nVID5Y3qlGlYvl9LlDzru0xP+gDfC87BFzvaFxs2qwVRrkGDXcAWLWsuQUlYgVk3i6pBy45WbOyilL1l0ez8E44tu10yQjqyxkXzMb9PcXnyXY5g5DSA7acfxJkuU8s9jheClHtYlDQUyF3kARRt74duEq7FAh9afHf+aKfMBKSrbxBZQSvIKgZVFkGyHp60OH6rodaqaD2qa3PRTdrjg93Ed1pNnoZj8PumO6KVQvZmDwaiB67ZmAqTdKKXa3FTLrpbLgtMb3rO1c\\\\\\\/vqkJLTmgF2NXnso7J0lKIv1t\\\\\\\/kV\\\\\\\/QJkvYcX\\\\\\\/xoFX\\\\\\\/YAHnW7vKUOFJdgOjGlCWcmG74BPXz76lwqI1ymEHCtuaa3bFBaIkG04D8BdTWnR7a1\\\\\\\/TfSSnDrzK3o2CzkFlD+9fuw2yJpH+LKNrliYuCoSeE1vp6Uiv\\\\\\\/3ptIKP+RiI0prRIZQFGN\\\\\\\/T4afdzUIL4lzUqVsJeRfLJOpOB8Qz4K5wftPaTWkAczP5vKaBp9nmIAZKmVNZjjSGJJkCQszngZJBqkMiZ9X3aHVatrGDAgtmhVxpp6vD0b27pB6kdVgBUkbmRyu+\\\\\\\/KyO0PDRPOl0R0VDSQS5N1TbjRvvBUJ1lYOfLi37CPSePQwJW87ofUOf2cb0qK5Cu+kziQw22vpK1R5FrBjldHejC9xl2IuA4CgJvEfuDsO04JQVAdO8n3614BFyGWo9BOiFIfqxWhLu9gAipbVMRUMlY5HvyKBw5DN2qRIvaeSoj9VgUlcCsgIQsNP5XGMihnMSlbq492JpPnPkwGGyRZYagzy6SIr0xkzqRHEyRvABllYnbNDJz2lPbRIRuvTlv8Tmk1hs0FKJx9f9h9aIJudfcQ01BQZxNDnaKUzY8WT\\\\\\\/9yoHbWL\\\\\\\/esMTaQVUm0n+kVjb63hk1gka7biwvLLhIYTQDjqhJXWcwEEcOt7hS9YGn\\\\\\\/FxHK0EQJGGIkUYylHz3oVNIoffknZgRZI\\\\\\\/mMrgrZgI2lViRdOEFsaIj0vsu3mHkacFfr2dcfwBttx9jLn7fVSkcH6sfFFSRKytBaR8NezcyXCbMMAEAfIIrBYeQEF6xZpe1dBIaXCCCiNOlu4XhSp9hlXcPIu0DWCOB6jkAGcjg1D8cyybpKRcNjjQZM8UL+JwTjabgLcZkahnTmV5MvrooU53Q+W0+E5TQbL0m+c4izCoXtCzVPjckMUjqRHoWakCxAaEFvosTq9Vo2zYCpsfyWrcajZzK76dd2MpFapAA+e8a3z3JwLnr9jzheZypiQQku9FNB2GZgcLq5LQa7RlDGrhUsTm60pcjU1WTcf+qnIXwWXzsvMlV4tt3iAxrI2Tro4UaWV1jfwWWqqdRhX\\\\\\\/aLDM9+G00WnkhceRMRYO6DRJ5cK8IjxZpxWZ2H0O06y4st2vki6WqNVnX26JPzsKrMicTcZbFSAsvd1sGl5jG71yTfBOF071jvmj6+R31LaiTLqi5gDDlYQA4zSa3xSAN8KMeIe\\\\\\\/TrhSJ64JuJrudv7HWfqegygLxvdVbnBBPFOfWFQzhaNypVJbOZ+4HFCXVKjnl6bLyFLc4vLmDRx7ThZusFQbHMdlOJTG7sD3fp4DSJBcSVMvWgkJY9LIle3914qVQhrxnpoDA88nQayI4DSBiUKuTYSCKO2lfp7J6vumCsANXbwnuU9WrC6C4tFBWAbeNSjzup1d2RbJ\\\\\\\/rLSvPVlMFamVxjMRI9CwtEfefqxXMsYlUWpRwGt0TGN+5epDCdOl7FERuwGz8coEMFF9TGNStDKwOdQgJgq2QBYxk6tgWdwp6INV2wMwZ2f5askDRoRyu2cN0UuNi98KMG\\\\\\\/YLNMUH\\\\\\\/X7mqxIOvzElv1GQe916TYXa8GoX1HpVs0mh5K\\\\\\\/qcGFQPPk+My7Kf+w77ZWhSVyaUel3r07sSBZlp9BOBFzVmBW+vz6RIs+bdUqgtnU93hsT8MrYsJ4I2tepfVe68j9Mrk7mbJSwqxvWmGtYppyc+FFsfx7cQjtaeB6NYnA0J8B138ntzVzoDAqgYXSY7EbVGKcsVdJwRffwaQcH7zVxbEe977bIDX4c6+TLeCUFwt7Xx3rjGXySG6X7KrQ9j8vqFOTf07atxa1L1yK2G6DQ+3EMRvrtty5DLFboXL10KJRyZCNeePJAu1P2Yp3AG8nX8c\\\\\\\/LXJTMMxSTT3+8P2bVsOaW+dGduxrL4qPorJ+9PTYbiktybFJ6LMvtIi57V7RMBqb6mjGy0jKn0VOpNE5JXCoJfNGXounFDxEJ6FVTfej6\\\\\\\/0WBoPer3mste5\\\\\\\/+NA+EC\\\\\\\/CRlJOpY+MIWZBqhbcL\\\\\\\/LZ1rBad+N8uDVAeM3T34XofTxuDjR+xhwmq9mnOpmLls\\\\\\\/CUyV2JMBlhsabNFL7zSZiI2dTEYebe8F0\\\\\\\/qQtXXEObxbh7SIgRARKyFRyOetTOTbBDClH7p8P+8RZMG4zBY9iEFrBmij5QyZBIRAz2iPjITgaywCh1SGbFba4RJnXosQpxSfwWtYs1B0r4u4g\\\\\\\/4j+S8OUjAdIOXEqXsCbaHs9KbavfgXoeiPseQHHnpF57IUWxC1wsmM8rpJe1e1XCZqZf37GiPtyUt\\\\\\\/meRxHyH6MP1z\\\\\\\/p2E7TG0sz\\\\\\\/8G5ahilzsRk1c5cheF\\\\\\\/hwxIBN9RarGAdHLKQP4Cv3u5Q\\\\\\\/dydQR9IWIhhOV9YsCUW8YkA7ATEOdt4Xs75Q8cnc32\\\\\\\/adU\\\\\\\/YqyBknX94NulIpuwRN8SHUpDWfbKjXxBUTuzGX+3NR2EKPbtnRY7xXhMS7Ot203xdn5kKq0vhhTJWwCpfjoYvdZ7coOBNlDZ4NrU1X2izRVzHxLQ2ijegt+9gB3Bx1Aa9pUQx9R59PQd8jn8thT81CTfm13C5Kn+xsLr7ZHST1LRCt4M9tPGX3HgqYFIM9xPhj66YIKaxuTqBTx4DgpIPuXnGSYJnoSggR0vHJddGRYjnhKJaQTE36jOHyhdxL\\\\\\\/ocdFHJV7Hj58ayitlg+D5gTRuDeFt67WaqiO2TS50rlJIvgH0Zv8MrHChQWNRPzQ\\\\\\\/u6TaGUoQEjW3TEKGumGXJ60vFkp0hZHW\\\\\\\/dzF5fIlCugvp7+bJ4aOw94DDqCMjx7f43z4XixTj8\\\\\\\/hB4KNRIFUvmsg6RL\\\\\\\/Pb12GsNrL6hrwXjE5SsJuDWF886Az35yhhSqAHeQ+xyzlAcgUpnwVSF4Ly10cotOSSDV\\\\\\\/zxtMhOWfvWyQSL+txW6pRQS2B9ln+eOoKkkT+w+SyeS8y9zqc2rYklh3wcpasjcixln+TRYjGpTUmCnFn1yhHrY2poAVHrqAzFGMfw\\\\\\\/3qeuhUhPTJZ92ViMnfyBHAjcYdbJCyVxwXxEXUuqr0j3NDeCoFBGqWgAXugxNJf7MNH8PynT8fq3gbDNjjhA+27Rk29ytMWugfWxkNw3wEaq7\\\\\\\/0fX0GgdgH3H6ulm6iLTsiE+SbM48\\\\\\\/9H95nuojGNSUMJgmfDHUGE1QtZtdfp1d0XkZwcReuTW1OIpWMLqSv5cdGZl5Vo\\\\\\\/mxhdF5EXmq\\\\\\\/kKUIQcQ0LcBAE2sMKBQ1KUS1gKAkqnArknQjmtSSrrWZMKj193P0cpDuSxyyeY8lmnJepL8fPTmlGVRAM3E6hr88gwm7zJ36RmScsd4i76Xr2NLf0Sb07Ly4hs7gTrRrkZVHaix\\\\\\\/+xZew0rMsN5ZkmUNGlHPAKC3OjhKWFp7D4KO9jy++lvdo1EatDFpSRnfjGa46bTpqfaXebYSTSHmi55KVTQVV4dNQZqGUB9\\\\\\\/Yk5L4XRhtPL1wbao5h5dRKbe9MJa2LfF6fAXQPzsdiNJDuu5jF5efoi4JmvgYKyHsPMZLxi7k5OvfIKnmoEs5r3Tpmt7ouuzzoLBXjFhCVD4GMiX0nEw8EXqPKS3owdiz1RSdeJxMJM3wqg4qu9v3tFQdgv77nnEstwu9yhcKW2f3Lpoj87xBbjn2sacSt93ndlpbVNXsLOeZv7KR+l\\\\\\\/OLUhI3ZtyO70XamItPkxaqU\\\\\\\/8k52VsyCcPC2TH2g2f9dBnDjVCq+Ymf7\\\\\\\/0UL\\\\\\\/zn63ERQ1pwNF\\\\\\\/YFbrQ0BWGKMWeWtvl4gGZ79rEnsl+hVecV7XFCzab4ejj2L1E9y2fB0NVyykqOfR4JhVAc15QEv8R3lEO0Mq6xbEOGrGorKs0PEmBPClIUgIxUo6ys5T\\\\\\\/IS1Y+xFQ2U0cmY++HfXvtZA0OxgTZqFYBqXrH86WqI\\\\\\\/xdtdyv1N1PNmkxpaz1UfmXroLBc8hXnkDxD6rFrpMk6CY51dxDUmao1JT3\\\\\\\/+wclHd\\\\\\\/tMa0v\\\\\\\/LH+y7pbwOC76NV\\\\\\\/8F9jf8mCYrw\\\\\\\/vSc1+Zv5blK2GPRTrnqkJ0aqRibycaJRSbtW9b6z9hvsPi7WV5cirCO62EaqQrpy2WfeltsFWK4w16kdUZSLvlUM+Qg1hd8x87GLqggtzvFd0vo+wT\\\\\\\/5cd3AVGSIo8Hwzrm3+wI+BHtd4vwkklV\\\\\\\/o+SeQUY8+bKRPVkNE\\\\\\\/GiyAXPPIFj4DYOcXHUMwuMAJPYDiL67qu1ypAu2UC95Y5j83pqrgiDbyDkT\\\\\\\/kd8LHGyA9F1s1tXVQtmR6aT2juND6mLNJJHGVm7wKS+6i0M5JMOXWZTy7i4whtqEjeuBDAfNKwU\\\\\\\/cCdI6VX288nJOEQGcvfIfQQdKNCWUehVHmDwR5V87pKw6xZ4\\\\\\\/vnc+xBWMGb9AI52jV\\\\\\\/lVLus31n\\\\\\\/q5wDJ6FWJW32BCkiA6FJWVseyRKSp4m8jhHV+3ZAMYGnYso4yHHEm3Wy6ONCuywPg2d0fHz4y49E7S\\\\\\\/9\\\\\\\/WCa5OuCKCqohxWjJ0MBQNyGmzkLeTSaaDgsu8qzc+1V\\\\\\\/c3nH1Ffkds5+rJJsGFBE+LgI8IggNSyPJoeS9C9BFyR26kV5ZMgkrUaeRnPjkqfjE3maf1DADLri+d7nC8xyFoXBraIAoBe7xwJkuI1plMjfjflj+iOX+7B9QRmYP7DRAIwZ2gNGFe8UNI8N9auIT6I1suC5fR8jxW5yLpd0cj5Q5\\\\\\\/0hgUN3vprxIG6wHKV8jbbz40Yz9cOrQgG8WfeygUnQmnBLVx0N1MnlxlaBiUjw2sYlI9goBk+V4lkoHMpIdJXtGsVeFQynd8bhG0U7cdWedyARcBXTaPjyaV1bovZl+fd8KVlobLoo\\\\\\\/beOrfq+PZB3wWw4u5AqU0MQ+zkcqWuqU5Gp3HaJlQLugn2SKs3EbRne1F2MWUoNp4On2XNGhPIYXKsOBBy1Pg8XXzmTjAuFkMJ3LZCprdNqB+cguoBpMXdvkmNbPI944uAVrzQYfBB0hMMNx2MrLOQkwPPAwHfdm5YHqgG2x8hxtS+FRKWDBi0riosZVLuailh+yediz370O\\\\\\\/Hrodi79rHBE1hvSS4IpuksZcPBo6nphT+t3lD6WkYKUNOkDx7hmRNhRiKN3IFTj4ihSl4kBM8TZjnPy9zZ0VDxw426K0YsaOOZPIut19RZm8\\\\\\\/UUfdVuOa+9kwipAdqripdjG6q1\\\\\\\/FwyZT8TpFpWNtKSfSnM6Kpo9Zizv4ooWc9onz0ViSa4Qry3bqoRNVGB2NSZPZCl9drq9lTLuNl7FhIAz5moXDgfxV8Aayl3Tqf04TnPUw\\\\\\\/85eSwSvqxZPfYY+lplmMHrTBTqbDjXY5lDRW62qu6UqGGL2unDyJFNqBhpGAfMGOXwSXK+XXVqJKuN\\\\\\\/8bHSdWdEryd8369dWsQaKmZBjqWG\\\\\\\/xd\\\\\\\/3GHU7dId27Bi7BRLNIxlAjGTz\\\\\\\/FkWfEnpoR01lBHe3KGsiT4Id7fdAQ+nPF3+97BJHi5AeAQ1T7vgdxmSgOJQrv1Q2vYzstLfYRCfqu3FfXU7slf7klhT2VgtBLqw2Jo2hEJ1HWzf8306JshnPNEaF3QMjIKgUSf9YrbabG7PVNGWIDCylFVOkLtQNQ6xfThCnbbn5rnjWuFbUu8D31sppQC+Pdta4hPnqOE9fyJtp8wi\\\\\\\/VjKrRwJXmGDTsMP3Lrx67F9KjsYgsrLnZIZv8svnyof99YJa\\\\\\\/yiB8FirYJgR7tjxwUuagUxkK\\\\\\\/5Imuqu3qKz1ZVsb34+R+2Cepaz0P62Ywyw4ULb3Dc8J70DoI3tKyAoSwvixs4El9hnoI\\\\\\\/YKtnbCSwez7OA7SGS3aM\\\\\\\/F1iHC9R80b1ygts0lizfEBqBW0MB0Iqx5f2EkZqNZamDt1oVxc8MBsWubvqBGNd1wC2AjEdVOMfEaVwhyO+hG0fxr9RsnPhbrAz0Ue3bG4iidUpFkN1NwJJdbq0RNVHnZwhoubFyKjNAEE695dWPmFvNtvp6YtZtqsYCFKwrgLsUg\\\\\\\/bkFRUwXd2SK8Ge6HzwUdxwSMh6qkiKGYRJsHxRTu6UeBrtnH3aiVklzOpI1rMalK7YZwLOLlGz8E5i29XBqlujiULs+T0Hpx2NPzx+Q1h\\\\\\\/av3OW2GTEsNUWfs0v9k4e2jhDDBc1+k3krzlmOEMwhZW8DppptNYyZ+Qy3lHB80yG3lP8ZcGx0PdMZGVmpR\\\\\\\/E\\\\\\\/mLoacTRfagnIPHJCfQn6kACDm4gLrlVGns0y43C6nNIa\\\\\\\/YVHhzHgycfLBlS\\\\\\\/Hr5OL5x5eq4UMpLiINVoxBDi2huqDfRPoLn37IRN\\\\\\\/sFCnuvGLn6Mg7uwu4LyzZdH65od8Arvf2xhdEMrzjjEW4Ss8on6vpNQEyg5imj8mowVqeJ0VwDv038NYiRgoXebpfhGpO1tOrBd2gy9tpyg2nVad92hPQ2M5Ce8Zd0RGPdEwo5TiW+iMf392WX0dFtCINJBK1b6ydTe8zrh4\\\\\\\/elopkTQgzrRsuXMgbQqQbHwgiO\\\\\\\/Z4xs8txeNtuCi3cWzfKUufZNzFmIarTQ8gkshcIu1KPp3fxWg3auw4XUow9LpnWIT9IPM06oVMppYgbPeLx5oMyrGnCp\\\\\\\/rAD1m6iYfoR0TSvjXwCPT3hwYbUXrBKIscEfI11ugxuHjNuTNCsrvDTADMbASs7hP29kuiewO++VJ9YlXDpl7VsfZkI9RGq3qILMGuT03bcXdytbpuBi\\\\\\\/wncIN9wevpJdr74k0kEbi2BuMKbSVBAkDi0Nfi3BVF3Lv2jH5svdPMegnceTiZMIbNDyI79qoXh7ODU3N8hDXJZnhnPT0aoHX0cxMO5xwM1rfjwj2dAD\\\\\\\/AqcyFhd\\\\\\\/ieVw3BylLFdBzJ0NB1Qr18avDDusN418jAr2zMaR9HMI2N+B67sRdN+gQmJ\\\\\\\/4WktSz8lIYJOmGObX1LRtAhnIWNb+7H6C6bmuRrug4qisJ4Xn9v1ILpXU0LTbW0cfwB27h9R\\\\\\\/usiE5e9UtJOvBFrBTQx3+QxCqTmcClb78xdHWlaJGRAOmyx4fPk4zB2NnTZoDhecZU8OwGhrDThWUFP79KWYhlzHbYIeBOtZhVOWRaCydLmHesPufdB8IQqdcWLTsYwlNPssa58jcu5M9Jbeo535ZX+3KcoD5NJGa2QcT\\\\\\\/ZxYiuo3\\\\\\\/Z9FsWP+qbSIWbLdO7tu4uAEtuTBa09dkz1NTCDoEopdHpfYVblipUrOu0uU9ec4AYwWBF5ExDXAE2M+nJBS5YnwZkJCcU1pE4JbcJbXZLDcZeCOlsMZ7W4pBPO4SnT0ye4hF+XFLqZ7eUHNWZ8kvuLuFTKJyUaVJNnMj0p\\\\\\\/TFbKuhXTbZfBL4jheNwOiPA35SuAA\\\\\\\/NuzjrV+7acZYw2QG1Yo5zyTLme\\\\\\\/qRcWLBPCkIsAZ7lVujjShSdCL0KglRv1v6P71PdbYT1fGYqcur6gOnogxApqoRxqW1\\\\\\\/bw6V7ZXgIYFpO\\\\\\\/N7ecXj8EAiOUJ+8X4SON9aqI+hjjT7gApZ09at6WQpdjV7anT7yJkBf7MM4fUirrwIcB6N2zoE07tu+TDoATfqM1dultbHDglIsmpE4vaf\\\\\\\/9iC2weG\\\\\\\/tQY7Krg7EHA2lJNAbWnN8VO5yt+nl8\\\\\\\/NqGYRkp0Cyb80oh0k38nGQyLxMISWvyUGJ+nOLipWA7V5EiQwgFHsC5lm8jVxRrviXs0Q9H7YfHQDZtzeTH1Aw9Qfsis+5LoYW2tqFA8zA2w\\\\\\\/+Rw7rHpHl7r6A1OXQBpbSJ7TZFAvWNbSpLOI51R+8+sTztJRG9fxyf32ycNMkn0nTYz0f7dWszLxVa2D4Q65qSVUzQZyfBwtLQB0L\\\\\\\/keiVrlur29XkWfxDKl7DAh+cjZ50ScG88M4DfPJ8qNc23NJldwJuaF6nj1DF5OQnMKLwLNoowLGk2MRdvFkqe50Damz78dJRjUYzSjm+NY4hBXTDD15s4Ba\\\\\\\/nG0BPemthEz4Av1KlV824fUlPS3HB+7W3aPcA3g9X6MeTkUMR2OXvhag8zyrGVlKSZLQtBWvGwqJPxQwjAHui2yRpXcSe3rLZhBIWfE5IaCffsfR3YXyNHFWa1sOcC2aV5Ia5wul0aREOu8CD6TEAyFy5Kvr8fnM25QmDnXZCpLKQeNCI4lE+meDf13UcH8BOTKf\\\\\\\/IKguDlRK82ZKwIzTCywKVin\\\\\\\/gyXJnQMHuLmog2qT0Oc4o+6NpuAWJhmbviViDifKsMHPZJFnK8pfmqYMfatLZm2g0tXEF1HKNQPo0wkzKK8Aq+0GHvU5HZjkK7ZNhOSoHgkKU1XBJwDktUak28qcrjVwcvWeqStEEZHmI67T8OM70jI7SIVpJvT8zUNsYZz1OYK6Zpult691ZKeUbhr+T1fwxinje\\\\\\\/T2Qtidgtr5jBH6jtKjXo7OE3ISfqMSOYMOJx9lanQEQ8b021+bF32lsKvJZ+QZsWjWIRpEFmmMqsmRf9HQ4OxQGe8dVc4jPpa+VFx2W\\\\\\\/7DSEhiRsCGIS0RwYBPU9Ixe6WuHzi0vDV9vHSgjctsg8BwKLHCg8WgxtVjS8H8763uPoDXQvDFIG6ln8FaWV6Kq3Eq8Xfar8zNB1wDfuqytTxIGl8FzFh2BopJkD3K+xNLDGw+mBApZ2LQXmzbFN7G1WgHEg4WpSH3kKnW3yyl859mQwLTltMSCRD9r5ySSd8zWci+TdeQuDCJ0HdfEsctchZmT\\\\\\\/JJJ4kxFrWo8B\\\\\\\/FHJkUYWIlRu2A6M1jZQ+vxc+NT6JzHKHEqzmRPShIir+i9obB2EEWrmBmP9ibmQvqAhzSN1mF64L+eGsUNR6Q77HGWU+wlL75fEF7ciIPh6WX8jCjQLAWFtSb3kSsyVl\\\\\\\/JZxoxh5+rT\\\\\\\/KrhP20A6mbWWDpPL2PF4JWHti3xcm8tMZa98e4yHSAEehCUVHzBD0ufYyCnfbt7fVPeTMPix9N4K28Q4czGz3QsYNVCeZEZYLH+GNa9UHhvsTajznxyuJ7SQ3yKy4Rg8JdY9Q6S2de0u4pgEe\\\\\\\/I4WXlj1KWSHByylDfY08cQyWGqN3HLPArS4PXS6bPzBV3GknuJdOhJ3Zv6QPCk\\\\\\\/6wMjmuUoO9aVFaPY7V1abouolQNQENYd82o+KvuVL0yN1AYT6nbZCFzwTvwjFgFYp0aSU1Kqe\\\\\\\/ztyQY4ZS6GrAD5n\\\\\\\/0qwqh5lzZMLvcf1Q0jHMKonDRoJzrHW5glmqSVEeckHRjnNAW40YIfKi6cyQTDdV3CohqyaJKQlamZD11mxO9Q+LzFqxc60BQ\\\\\\\/IldO2y+3ZY\\\\\\\/LWjXZxfKMZ\\\\\\\/lbbSyP4pAoNXXvCmrbsWKKAqpyxGgbzVwgx5UKoH7Cjy9khUEclWD6X69puUitMHmdA7HgSYL8\\\\\\\/7z9b8FxIPqmCRZd4S9tlFWeiERuaIUDnju0RPC4Ocp4ZdtQzswQzZTnyistHtmhbQlMFE28ax8ziQsqJ1NKwBEReSZH9YnCj+CrEedMTp0J0umBbRPZ7joi9U82klLVVE78MVByGtglGd\\\\\\\/OsFuzPDz7WKImBODTVxDG0Z3xyS0siRYFCxw81rFX7XycnZ51MWvwUrDdeSfTYV309knbNKqVULAk2HtL2hoHzfBjJo2KMoYO31SPPDrx7PFCVXjNCVt3B0QeLkS5D3KlA6FARL3kAllZyKc\\\\\\\/yw5h2ORkvK33R4TRiRBqRY+oPHpVH3WS2MBLe4uy7oZ9ebybEOk1ZRvZCBO\\\\\\\/pFYorzY3TYBQ3Q0GnvF08AUYmWSGNdp72JcP78evAu9y94K+cGafVOjQ9lIIP3v5Czgx3Jz0YIH7EwAIwoKfMyDhK48e9AiOjjXWbiuMUgJUEKbAwJWH66plWlEEfAKOaq7hBlR4SZgIXsfP+ov65tpOhX5t04MSwUw4s9dLbK+VwpnTR4uL1wtLV8IrNQWOY+HrZdveopjk8JYCqAqmBYlQjJiPBBmma3ZrQhRv47sitx47dE848Wsgl\\\\\\\/KKi9PlZ9a0FtB97a\\\\\\\/8vlMBBdIbsQ27qDSEMjlO49HgWAqHJmx9ecSo9Qjt9SJlpfaicqB8LxtsQOcSGUpqH5fqLl2ecymrNjzsN7D2e2Fka\\\\\\\/PaaOgW5yK04rO9wvIQvOh5g90+w+niAgC\\\\\\\/7mj8lcTezo7Dqt2bKbKKXblFefVxVl2c2KkihbiDNjS4Wr5UwVmC5m4hmrtB13Se+24m0Z6tkxJeqemS9wwErys8FUaYChhXG69E9IoTbw\\\\\\\/6abbzh6V0\\\\\\\/3wxnBz4Zo8aoWdoRAvfWWQLJP+HPn+hyZHuCRHf210nbxwflC\\\\\\\/WaQt6pfBjsdFFoZ120fq39BHwq0wOdqPyv\\\\\\\/w7LUMYWxQiFzVKHaNF6cjVNg1V79HvgAQQmVHLjGjw1LhfF97nM\\\\\\\/el3J7FvcEX9uLXItj6KTo2+yBQoAczgwMsvS\\\\\\\/gHBLYuZyTFl2fvO27LZWBWYEQuqdyqURMYzw7f9if5YO8qquhJ2Jh67CkhdwrfzMaQ46DqJ\\\\\\\/os54IbyU\\\\\\\/CWrAMJHWEse31MA9AkXJEprJ23821fPFw2VdRFwWJO0iGkbf7XEtGOMAOLYjPy8GcNgFFBBcziX\\\\\\\/54dG2W0FlzKfwNbJz\\\\\\\/CiDnKKvAqUiFaQDIdEz6b7qXObJbY2xdJ7JjG5HOLAWcivLTP+w14EqOjwBflQ5DwOW9mcPKrFCbmaJDdRAnFaRUKDzmbrxtqi336S1b1V2pW1O3hxyITpMukKEnJY0+vTLI8CCC8iz50mKSylKz1gKjmgT0PtoVlnKJ7PIekv9+5iGRd+wR4ZtjSGBjSjFOpPV\\\\\\\/JdzJhbGXGrka1Xnd8m3EIv0F2tvSf5JIcM1iJ37hB+m+K0JGftw7esirCz8t+TM2nCAWU4lWOsJx0+EQNxqy+eIVOk7\\\\\\\/oRXdUiJYUDzRDWFbdM+\\\\\\\/a3HAaUZDqJSFqSKyJM65sMgXxoddHy+uLKrjQh4iGqWXBpraGJhZGn\\\\\\\/aT+11c7pSxcxz7VI3LS5mPeFaSOmq6hMxdX3wuqVS7QQgF9iIYKlP\\\\\\\/i1ySFAwknl1uMDUiIu5hy\\\\\\\/4kLeHq5szgj64u1A\\\\\\\/RsYeLNATzemAR24ziEQhNcU5FHMs5wdKXcCfwI7qKTU7ZpiQjZoGn1EdoD3RKXQu8Q\\\\\\\/adV+FvoCsI+Cse9wqmCgM3uHuqHJIhG7MdeirCBkx5W6vp\\\\\\\/mroF7JarI+yJgJIGj1qaots0X3FH9k6Gon8FEiGQH7yAP+\\\\\\\/44Ctxi85QpA6CSmNXX1sd\\\\\\\/XYC32p\\\\\\\/32VHtCJ5mILgS87VFuYezSeNmAFiR8\\\\\\\/mvgXv0iqxG1uT1QnL2PgyVCKWk91nydHG50lQbtdj+T+JrePZ+s8WuqNMbcTPyz6nBIlO\\\\\\\/CbiDzOxbPyxyhE3ay5Ttcri7wvf8Bw3kh8BSnsG\\\\\\\/0ZWRvaYcKnoplL8BtFIjtzU8OT1TBpyNJ1DsnzTbCwtHTZtkiZrSyTPyqyQtU\\\\\\\/N1x0ho9V8kRkI4MGpnS4D0WRjrAkNL0xVVpXIiHrl+cY3uQIhFddsVp2E0ukPoo9+0Lyv+SAWNdjw+D2yHOqohFwLYsJxJWzCn5Y98eycgWcGv7t3LXUvflvW+pN7qcvSm0AeH9OkodwAQDaewuBrMksMNhvdEgJOq9vPzIqv3BqCTPXJfnXSIn5wMPfocmMtvTA9O+jSzP3voE7evNFRSdTL2ACCAH05Z4UjjtB3PRchruWtrivuvvt0fmQWbw4YaOFH4PWNf9PekqCxDP66pltIT5u7e6eUBOdDrkACa65WhN936pnhQVmZLrB88+29yzP4svyF8f1YWgUdwDs6\\\\\\\/u9JoGrYxk9Lwe90hODPwYDUvhJa9tXVWPKeReIH0hAqQsK9MWZVoy5+6VFDn81MxeGT8Bpd+SdEEMkrbjGNI1w2cJG0ONuZQ5KUECURokBO9TWfbTN0i\\\\\\\/q9DKiUhTxbmNJY7eNeK3JdvDjzD\\\\\\\/18B80u5JpWv2FPHko2vKnsOPbBssQ3b2N2Tn1suEHFTHy5Nd7JZVxlTAmocgZWPoQO4Jl5rZWZ0Yno3AMgPAMfp1+GAZ3ydwGHURryJP6V4agZiltugOnLr2e074pmWWQMIRv9xJP6A6Atj\\\\\\\/KmF7SnNQ46ciF8GHcnAwIjNaS5nKqRxKuH8aO5YEt7E2DBFpQi4o5Zhe7io02U9LKO7y3GzKM\\\\\\\/a+kW527WvcCBPaKhhchEMOO2WxCLBwo5RGuhbkCMDkRxAzQ9kxr6W2Y6JjrWWSE1Ywkm56r5mI+RQlUBS3qGK5WGMjn1gPufRcvmPbt+ErHAULH42xSY1mQMzUSdaIHkLc+tuiZOAczoZPLLJbT6d38HVR06buL8783E\\\\\\\/4XK3Y0hoGlVJ+lau6d37Sw\\\\\\\/5hA7A\\\\\\\/YeZPbwgvzPS5BCKkZczvFhYu5s1N3UrVRkifiahZxcyYXtP+un7pK8vTMlWMrRnhNez+KC73MbKqTVBlt4BdfmmlYNU2Ji0JWTDpnl7KZ\\\\\\\/YGU4EMI9PXx5Bub1gX++Dur2VI\\\\\\\/FHqmQNfb9fANJy3uDUdhGDf+KjiBFJUMpFJRzhada0p2DjMT+MJ3VocLxNP11wCZjiz9UJ3+fQQdYyO8Q5PGpDxaFSDDpKVFeVzinhZQzhxJUPSzr9yQYAAXsZBTSXew0Cud2khw1BgkeYZ8C0p1rOrNEUqSNTs5Mrva+CDzAzYAhS7Ctx3af1qKjMRJerZwJyOh2+ERWSHzBXBjUWC0GNZepVMNj2iImYSaZ0v+1S2p9K5jupguCoDed0yro+TlF0k0E1ZVuSYEUSDkVBSqdGhVzvWggQ3GV8Tv9wY\\\\\\\/upZ+EYUs1efetrdx6CsmPs2pHYb17gwU0RiyPGkpJHIGy6J\\\\\\\/FauzKub6BzlBLkjniptKMf6e6wLThKun4A7c2m\\\\\\\/CD17LqyRTOy1\\\\\\\/zpk\\\\\\\/gHo4l7Lhns+lEUDId2Gua1Wct9ayj5McofeUvuUsPalYyCDcOosDk2tHaZLfr7qp3HuaY2Oniag8gcrZmeGu1nf8ip5KX+wQaBWnfwWr\\\\\\\/LTPD76hiNTYPbEIFS+5RI3gvrEA5OURA03wgc7V3fmEg6pRNDSpZwBDhzAKoHw0+g2HJRAmD5Rk9Vilh55OlYawUOsl\\\\\\\/O9m5\\\\\\\/Mb6OJ8zUyZtMHjwNml9HgNpHTlM1l92FJH3Dmk4PmPnKtjMo6+AS13VnhFMDHA9A98+UEiCjFqmAwfXTK3QkCpIJfdOaRRkthwk9\\\\\\\/owi9pFw0vodL1S5YAF3bL824C6cp3Glw7VHvk2lmgiNiKjEnCpClOG9oU9IuOOK1xg6sUvKOl69eZo3AP78wBVGRZha9NO+ZLW1B9nht3WgVhcgsG7yyHJ3EjzZidXGgM\\\\\\\/iOEhbBacEfGrowRAnxyKBobjoAYCef81f491mITzwucOedOixvpGEdlDV5ZyT\\\\\\\/P\\\\\\\/fTgCQrh\\\\\\\/TT6YWB2ErOedxSAODKzLoN84olUztUyozPXBLxq+Mk5WkEdxhR65n7BpI4FlD6eS7H2liEw9mDy7zVeeiotRdqFlixumTCZRst\\\\\\\/YY987imjQo+DNPEYbtcibli2M3SLFvRVXgc528zipo5Z18CcJFA+wS\\\\\\\/2obcBzPJpO5HFmeHerVO1d8BCBAvXmp+ss6aiRrgT+0RpZ37jxx9Ora57mDYueU8ft2LhfjaKgL4HEx\\\\\\\/bkkq8VHYDb\\\\\\\/PHVDbykF8VZ4kyljFPsXdehXnr6R8tZ0pGzD9XmYrd1IlwM++FTe09LIgC\\\\\\\/PCFlvf9yDXRyLDbfG5vrtbAYjOVr8xyVyN\\\\\\\/UeLQtMenbAs3IsWOaGcCXlDDYKq+DVZXgIBz8EdohiJsZ85tYHKlvoFdwUbGrwwocctCx8JgSjNMS8SD9hkDUWqJJhI100dqUXPn9CoI1aYz5BguugsNi8ID8z3hUt3vjDDBqMghkJcHwzGcwSazYWnEKerqZe7KCPUO2b0TmHoq1u4WdZDl7iuAh+Gtv9vYOLQ8xJwn3\\\\\\\/0ooeVkj81kObwIr7vrBWRb3+Z5cO7aUEfhmWaK6eDTo9ZctKngnLw8I2OfFdk+KRiANY+Cd0Ju5aZRsDay4EN5h\\\\\\\/i8g1AomylzB0g1wMaAswMju6npWh0CGPSPwV81fX30p71QbYST7AEky9C7Gzl7nXvV8nHgI5fFIb5M\\\\\\\/t8MaxdDwjHojuh+flBOi8sI\\\\\\\/SIVHIRH4S0aOo68wuixfiE5\\\\\\\/\\\\\\\/Gw5mo0WeVcjRJ3j1eYjEa59m4ZAk8JntC45kYsYdmLtvjQlSQapZkZvZT4NvgXPaVBQ5+b+RJNG49YvBNNnnKbyNXVD3\\\\\\\/39CpZOvRVAKVQKFc+jHj7VCjwQky\\\\\\\/6me7jH+BoqyENLnYZpstNayzs0Clm44rApt3RkMiH15u2WYEE8xZGFlldjlwOZqmywVUCSbxjxjtdR2sZXz8FQcEvTyoB\\\\\\\/OSIz2kqIfmL+GSg\\\\\\\/W09g4NFCosm5e0yWP\\\\\\\/P1AzjMpd04EsS2f3ZtZSNogdcHxg1a2MbKMbzi\\\\\\\/zZv8w\\\\\\\/EcuM1kWtqioHlU0+WS0pBTBnUMCnVPFw+fHqKC9\\\\\\\/bEafE7hbP2pP6o4SiwtwkEBb+MTaB3LLBTSkEt+91cyA50u+Tk2tb2S9S9BxbeazCJ7E+8ZRcTKDJ9kwEH3b8MuxJ5HcOeLKmwzhA88oQUoT5+UVzBSbT1FBTlhHzCwaw1HlbrBX2jxynfsFxKdP2KDgAnhQtropCxayVvQn0r4P9fD0ADM2efAfEn\\\\\\\/jyE6u2XK9wv\\\\\\\/+f9iHzwVNYU9NjxJx5mf2LiInOMp9\\\\\\\/jjBTok83S2Jco6pLhshsJ0fzNURpyqtt5iZo1Ge492Tdp7\\\\\\\/YZabSmniDpIiBQiIHOkUK2BXiLycli3kl1uE0\\\\\\\/BR3x7zzy7pfUKq1kuZummOVFvbeeCg3R7YUcsozihYIqTKuonvq7PFRAA\\\\\\\/7UD0Tb2qHIgWRpesrQ5rMQpfj1T2Ncr2WTzP8IYfoQtv2hTTajcRP6b09QIg8HohLoxdK9plLLb+5pOa7s2sPWaY\\\\\\\/v56dHAXC6T12nB4PiHE21F8gyR+TUHHBCP44Z4FBL6nII59FoFBG8OBoFRtaoE85u2UWqlu3jaWfgreGdUkH7qp0GGfTnRAmekkWPDLo\\\\\\\/7xZgOsiR89NqKlO8Md9j4IA73jGZ5ibs+U8xf6s4pbdtNELMZTu2Q2bUXFBafIQfyeEYJ+t52\\\\\\\/cHgj6heOJTURhxgeNzAU1ZEl4D2GbVg3bzj99WTIktG+RHRTX5YTkFS4BARyRNBFHXoIYLor3BzspK\\\\\\\/v5TzVQGpJ3JP6lWlFFajACPdyXcvANNPgY5tGMUTELwLSGpOXJUql5ZpByLZEnoKO5U+up0hmAnb3H7CDORhuUKmaiz1P3+fQT1HtoWyMn8oW2CS+i2Dl5sUoFXeXgJYEjrlVNIjbhbmF2OLrpXcjb3nJi4jSt9uvkUnQe5ngPOMddH2QfJSxuGaJ+r3+MEN+tIF2m4BC18kQjoVUQhqVPyxtqMQfXlhAgyRtDn4tVUfI4zWrdl1KsqcfEDxOPzSl0llnR8EEL4fFxpIZvM6HTrf\\\\\\\/9LpKjQ+Yui9ivgGjjr3QQb8EN7BWklcMB7ni3MqJyi7f2\\\\\\\/ZBBk8+J79F3CSodbdD8Trrp9byTmLujIassQswVBg0L4RFDtqjuKaZk5Q4rq3\\\\\\\/XFJ2ApmU52i+kUYqlb3Hw6Q1TYV15eOdzsB4zCcc4e3DsqUZ+GLdOUcV1E0zdJ5tHDiMjK\\\\\\\/1WwudoX5cnO9CqCbq2miE1lgMGeMiwiZogDzKFdt6uRswdqzBCDqRmrL6a4urnFA7Dqs2TmZkP8U8QdPn0lPNgfBTh7OwdaPJ1yVopLmb1JkIe03BmItw8eVVSURoegWkcdygD3kZi5a\\\\\\\/V1xIAcltYkU4ypn51DLcfIJGeLm4wMEB+Ft01FI18LtFM0p\\\\\\\/NjMhHBbX2ZvhrE82i4i6Cr\\\\\\\/6tR9hOcCgRdYSZRPXDlMTRsm48Ohga1F+PVD2wF7RvwV72aedE61paTCRNK03WKJR1Ox2sPIoYIggS\\\\\\\/ByHa4DF9hDpm\\\\\\\/ZiiFcfRAHIj1vTH9AHN5pko6Rnm2VUWjRZMDSGeO3SwlWtIu3\\\\\\\/pCr1Cqk1ELfxObEQ4YwrdIRuoIH8bWvT4PVgnQGvLzw6dmmBalbkHFw6D5kp3oiNJwy7B2nHpgwxOF7lJnZxF8VsWg2ZW7U\\\\\\\/yyaUzGSMAzg8ZrshFnrwSeT\\\\\\\/quF38XXnCaivV3D+ar+ZIio0RVfz7k2pJbQnOQy+PLBzcs3E9b2dkEan38+tuLQegD8ls0raDbct1ZkiDORITEPdU375GNHoWmc5x6\\\\\\\/zeiotAWsTInaT1Hi5Ym0UNDkwpJmxYsHyaFGrwVk8hzZUiEO2M3yjZXHhmrwv8oPqkkQ1o\\\\\\\/YgqsNMWdh00p6KVY+33Iejg59kc6zuu2ypdNbdAfYjEFatqjEw1EXjnklXI0YkdXG\\\\\\\/5RG1C1h+lVt7v2EKBYEhdSaOS7LLTF5wC4RjmzdPGHFqjuS7c1xUZiIAkcBjoqiNG7FGI2SkNpCwBwhGeOTwkvFAZHGTSsGsmzY247+gcWosdWxhyKukfIxvvVuRyG8dFCpt+AfGAgkDecZeSTL\\\\\\\/OyvxrgOX72A5k0IZIok0GKf9FT54pDLn5kDPdysx8IOtoNSjIfEH3oNnskjfLRs\\\\\\\/cPMji1F9uoaEbbiH1tin63HWuqTarVEi4AsCREB2jFKBxdfJmwA6BjM4GC50ZiUf3AYLWr6OYs\\\\\\\/1Q9D4ihCGFGR3RBSTTUCeXfX+F9oV4tXsgAKSZyNNghXhD0kZpLcftI4dFEPqTzfi9Be0lEbot3cDpFFPgKrXVRjuRShOZ5sNI5y8XsgJmQuebOVbnFjlYQLRyntjkaFdXqEF5ymXtIoH4pW58h0INNzpCrXQ4PCop8ibIBqahl28XHUC3fkMNXJZJow\\\\\\\/o7qnqjbHBu3YmvLfOK1vM7tQCTk2vYx0l7GTCQ4Bmb1u8\\\\\\\/lzAkXzaP6sfx5v+KXHXlomsj0dL5DtJH4iJlUu74wSfJzmgBjmloMaa7Vtkerq\\\\\\\/PRXK9F\\\\\\\/PK3L9qO58uW6DUPawVXQYgKS5ecaKE\\\\\\\/7GjNLFAgACdnQDEPIPf2exHi\\\\\\\/jEt6aM+y3konUSBlgi4xER5lmnibcGwm4RLR6xYPIkUYIgOa0gTtoXYIGJiMOqvIIyDcxRKfj+t00HjluafhG6Ss9OhSmbeLsH1BGnVhqzKRU07fV5kjfAk7kFr6UGutqMWVBZpqR2e5yEIZ+6Wa4jsOSBCC+k2T4ITA756B8NgJOWEwtRPqBf2auvMv9cHdCcw8LpeO1WoEjcKrXtWDtR\\\\\\\/ATZutoDcbjKZvJdNkS\\\\\\\/yks0grrAUfyiF24vQACLWKAePUKRnCW+CAgif747yu9eaolA1C8uiRRL3jRhSdK0pRvXtLoySDl15FeE6+6ulGC5a+dcp0u2JBQV\\\\\\\/Xxr7JkLLXNrm3ywiTd8mh\\\\\\\/8lB5bEY\\\\\\\/FY1ZB0OOn7HGbI3sY3XthBU3iVjVrIRiMJOId\\\\\\\/stsuFq+8jpHR\\\\\\\/AvpqDQ7WOpF9XOJiKdj2y7pIGqF\\\\\\\/0MXKNHWGUH+ku3rnFWBK7f21VbgULbkvxlIgRLlaTTB9tcWNyLqV4YF2qWHXKzSd\\\\\\\/9g7mDzvPOpgiXp+6DzOQvxZ9vbF5ljmHufNvZGdFxSOOX1i4aMOfXO9ExuxWVcPbAlzy7nvtvsDboFsO63K7dBvPWp5eoBd3IlrtL7u6WVlD8XnFCxn5eMZ2VHiDHjjmZBQu9zEjGIpjaMDUS4qZsmMMBpraS8VqJQkM\\\\\\\/EvilTxdqicfIRih1lgRG60uzmHpCMvfyTJbW\\\\\\\/whnmqvujx2WB7a5TKs+DSZcC10V1+osZzfBepn\\\\\\\/\\\\\\\/UUFCqOfsSODJlv99MNMax46tj9pulgQou\\\\\\\/rxMuTyEzCil3dksixS09INH0MbeLuMWvn3qxZuu73GD8buK4DVE14zq+zB5XSZmx+XjMTeR57Y4p\\\\\\\/fI6AR\\\\\\\/z+j9rhhKDtSN2KVk+TP6g+KcXihiiusgmI7ypqXO6EIghwjXCdYJafqMu7PbXCgbh\\\\\\\/mv3RoLYa9hfTnfmEjQd5S\\\\\\\/+09JDTbUQJMB5DjjjbIbunDTXk7+QQ45JL\\\\\\\/fL0kB6D3gDXjCFbJMmDBzsMwjq4TUZHXxHNbcKU8OPuuKIUEWGkJR635\\\\\\\/Qi3pQIcVXXVAxYHXaF1OyrrDu1N3lAVeuK\\\\\\\/\\\\\\\/z2ccA0lAdiaRRT0BeauY6l8gGqG+NDZjS9d9AvyF70oaapn8XpROwvMz47pOmAUMHLUxYiM1DHfVEXdwUgI+Q2fih0aSAZ0W2QKVBtQnarQa6tS\\\\\\\/NBEbYaD7kUbBnfBP9BdJbITfCuWg9Tn1nZvT1UJXdFCb1pzUcMECtP2dzkzGceXRcIs2+zdQ6flcqbICuaGxXXieH16mT5FCs+J65dJ\\\\\\\/qKY9AS3TxbsgAC9VedSxPNxOzXtET+HZF92Hq4hufXTLTKMrC4ioBDRRWoRVBvU4IOgVt61YGlkMwl5i2V8A7ZePROmjSicuHrHmeggdv9ljOAATdeZfOBWXW1kTssF9DM40Y71FcgFydP0lF\\\\\\\/otMnLyu1B2SJ3Vmdo89FSWsj3TwJa0PJuP+IrAc0qpmHMXuV\\\\\\\/ve7DNfzGChG5YjxbxqLQSCPv9JUSM4ivCXOL+mh07tgrIPl9VoAfLpmHP68HScJhGb1n8miL4K4s9VQ01bN69CGSOqatBIwbifGU4hKl3XgCkvrpt8d\\\\\\\/HlYnHb8ZfKTdH2HtbOL+cz6vb2pXZR4sQWNF7dxEsr9bYhF1jE51U6rtVEhXLMTzuGiPyB7IvVY76kN\\\\\\\/zpac\\\\\\\/PJDmp8MGGIkEiO8QMRF3p2S7UOC\\\\\\\/5lzuHI8Ia85jYgmI72V3A5QknN6THE0AAjIkYpCdvpNGgrljpxl8hpMCVQxWShgc1ZpECFfoxmohvMeOupgHh4YWxqlD6h7rGYoPZfs6par5EbIJqZw53Ay09bZH8n8xd+3FirSXv5rGfzLZzCCQVtad77ef7au6FPDjyHRDaR77138JXWcNjDH3BrcDTGayX219NHLaSzO5dBWaj34Xe24p4Ty5WSt\\\\\\\/e5A96Dyk4OvIHM07pwuDnA8JI9Re1b50Ef1q+Mz9NSF+cDUI8Mkl5kmXxT5bxxMG6zdOkdoSdGuYIYMS4Ef2Qi9M8LP0Ga0hlLbwkRw+1Br0M1ahspnZ39eIaRdahO4GO4sHJlkKtf4seU7r2f7AKnxhs3CKLVbOjpYCThlItHg20VfbACNDajrAlojLGq9zetzOmKickzKjCMp0i9sWkZRHg+fxTjRw88lm0yjzW821CnO5mMXli6Vj11bska+GhIThmmvwiijqAFM56H65yGNigb0o4kwJN7Hsp\\\\\\\/oit6wflhoAwdH4JnHhYmt6m6dEiOMkvWwyP6h17o9JxWqNMHaKigYVigaRtqV0avYbdLCL1Q7BgAzYGNTWOdlJHf9qenK7UOwYybkxYSsesbsCqeCTtnbwoKAqwqV1pavRi8f7uV0KuAgkCuoWzW8Jyc+rywVDgcgqalS5Zha\\\\\\\/BZH6jfDuy7vtlMdUwGKj4AVNrNpifZLLnBtp8Ify3\\\\\\\/IXb\\\\\\\/Ow1lDKo7QMrpjNUhY9Kq4G9yB30tiDEZvVnQJpbSXgFhyZem3i+KlojcJDx2QklZck3TdpMSaSrcT6D2yA6IYaANK1gPr+\\\\\\\/VkzLiyxORRArMTXoc5CrPt3qYED4pe1ftHgrGR0ZWot2j76vqduVi4G7HdrRQiwmkFj05NJ9jJuogHtc0yLG4saf\\\\\\\/r2z9nuuQDmPsy2Kcr\\\\\\\/TyZTLV8vLutWaM\\\\\\\/uISKVTic3yJHFOBClT8udzdMbADJ4vJeZH0hgL1io2bpPkJeydsJZA6QJss4Ez+WEoTEb0if1jmvBcogI\\\\\\\/z7brw\\\\\\\/8I68UMiOAYf\\\\\\\/byEBJYXiNMPHnnmkE3TD9oeLJRfhYLlXHL18h4JT3JQ2Qn58Bom7Pk7T2q+mPkdaqJGfBFsccbEq4jtEcdqNda3nXVFD4Ez4gmZK2kpfvSwaSlCGN\\\\\\\/4cUcdAjOfY39bVmMZFiCUXk\\\\\\\/hFNVc8v\\\\\\\/DjhF919WlYcaH6NDfN0iGbaO2bLdugjFSqoUuJxTJEW+lMPQQ09cdDD1RUnu9\\\\\\\/BNJwPxd7kTawmW5NK8byrHkpI5H750PQxPmpNRuNvtg9xoXRJiOmifx7VponoJSCKYKJYPZYMfDuVhm8OSimxn4gK4jn5j+hQX\\\\\\\/ow2jrnTmtBl1qtfvz4GSgiWho5BES\\\\\\\/uKwc4bix6j9ACq3EJW4OWVakp0GGPlOUClqx7pP9L6OcPnEPPVx++cFq59WiSg+wewttn4XyMXiYYsAPy\\\\\\\/ULC4HRDxVZpb3wpA3w61Oj5hjDatrkhpUgkUXJRumLQlTq7ipCyjUOxAJEjO0WVfyRJ9ORRAtC7Ctm05J8CsfFDrEEMN31ABMmnhnh\\\\\\\/6wVFRFJZOtkUWF2Yu3fVMb7US\\\\\\\/JKb60wadvUpdmdYKbbIhEp79dIb+SL\\\\\\\/h+KmpE37v6ugDHi+fH4Ja5rZJkCDH\\\\\\\/b4GVJLdK2gH56PCy+YYTXFLpFpWYl4\\\\\\\/P1y5+3VEcpa8Y15PRY2ADOqUGFfCN47BdJTPNx3+LEfwek7qqfRAVG2f39CdZtK+wMtwJm6IXxqES6C75BfVaXsCkGkwfAuCw3wSk+hI225hZWMGpQ09I8TXzHlSkNujsSjf3h9ZN\\\\\\\/PCIlxG7cvUD4m+wPhNkg2MDBQ+6AV\\\\\\\/oLOLqXW8BfMpIIoSQK2MdR0d+L9iqBU1irL5kYR7C\\\\\\\/gwr53y6gfab+DgC+JyiIOH96KntAjJnzS+Fv78lVWnoH\\\\\\\/ufUbnMPWDf9oId1rN4dxKyGG2ZeiHfexE89frT47trHogioE7TnIaorRKYij2+0+Z1vbCJ3LiNOAMUry9FUh3GXiYEy4\\\\\\\/Gt+Po2WcaRMd5eKDEhjMJOaoJbTQMxjzKGpFhzWLPfRMVNy89joaRZT\\\\\\\/zUtqw0aeHtINS3Akp\\\\\\\/x0tbc3OqT5qiGwCB1PhAjIK\\\\\\\/Vx+BiLnfcL9kSqbBlCBy1M\\\\\\\/wUFQLraFBiAyezkTLkLnP3Qk42FOl5vmXTgx98YYD0qqbKOA9w8H6UbCWdKFE\\\\\\\/7qJ84aLNVEX6zueE3SPqYt+MqDFGPrEJkia2NYphATYvIDyaF9hAQcJ+r4j6be3RTkjkT8VHzeg9Heq9WU1M40wy5F9PAPUeV5kUaN1L8UAOGhv++WLQqyiFtwRM5LFfBM3ULDWeAC1IH6QvtkYXEMRo1xaV9l6WcUhOQ2X6xy7fLxaXG9KbAqer+OHB3E4gbKd+EPMCseUncTykBGeZ3kxv\\\\\\\/MQaApAWkZ8hPx42FPKv926JZqUN9ZSajfnw28qAKvojRO6S80NxmiD+biOxPhL5Bp3fn5bT7kVm\\\\\\\/YNYUGMsSRVSDZMiAhE9eIwVQM8rrUJJxFapJa39nvns7IM0DevBg2+7RRQO3IHLDqJjSLX+s7KWMatvBYE5VgrnMTYKSDwTAS8vW8lqzEh3A\\\\\\\/f\\\\\\\/F2VswEXvaFD6KlDwtoIDnY1+Z7qUrOwSw8NSBp+A9zBXE+QjqaPfVq1AT60xDAcUGXhZpaO40z6ScIijfxQc3VCrSnKGiOtGTu7q0jwafkoA6aH2HVQTl+p\\\\\\\/RV4yahuNHvhCI2n4lNr4aXAoUxUNGWbidC71zi7HT0xapOIeHfRsiujXY7Eqetd4QYKsKCtjP+eHuhElAOB5+g7vWlIzzMbpCjjqyrXD\\\\\\\/g7Fth6P+zTEXGO6WvU7y5kiLmjNkN3a4LrvYQJ7\\\\\\\/d9FB\\\\\\\/GXS0ETLi1SNSYUezu2Yw+m2P6wl7lrwxDF4o9IG2sEX31WkaTUnOS04tdMds2KgeHBO+p4qjHbstOAXsGTPC2BYM+RFUeFN29130UNvNC+XJlFqM1s7q+7vF1Gp\\\\\\\/PDnEn2mlDAJH4eTe24qYGT2GqAlB1vz5CodsLg92\\\\\\\/ca5d\\\\\\\/ftF4TRYQ3A0u1HGGeTB9f5LqNr+R\\\\\\\/bVn67gqu+UGAWHsZEBqbjPTaaqiyHE2oE9qd8SXaZA0j\\\\\\\/2oPBP6PLWikL\\\\\\\/AqExOI9wTaUbgjmpAB3Io0MlzAbEr3qpgSLfDkpblISsE3pZKkt6WBFt9VVSreQsGvDwoDZ9Szc52JdZmlmTAsjgbVpNHd73ixwDIXQtAWg7uCkCtfdit9OUz\\\\\\\/DKz289h9tsPWBHO+hhgwwV65brqpDsohJttmmwQ7LkmblOFfkq2U0fbklWWL4k4vIT2eg\\\\\\\/C7OwB7nmwO4I3S5fxlrQm1nrCZh0TYEEl5Cum3nvOsJR9Y0f7QQuf0o7iYGYOSP8NqqTD1ydpWgnaKUvf\\\\\\\/4dc7+iOlCcABfqfACi0fTNbjSzrNIj0Akbg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022L0VvU2cAPlAD6C1JxwC5sw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224b99c06e11385da17e5ee2b04085426eb314b4916345619253649436bce64d09\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uLA2yoghRjxX5i9khKSL6bBCoIJPfxbxyuumiENKhwZsl\\\\\\\/29\\\\\\\/ScvRNvhKTQ3BDJ667ek6CGp1C2CDfFV1FK9QPyI2gJoeBl\\\\\\\/2sLiDr1oZBb1CHipaczQtc4Cb78ITIgi4XCo\\\\\\\/LiMNawCrEklVG4oy5XTZT59Rh\\\\\\\/vuDEm9+EXqG6njV1g0I\\\\\\\/STOtUucIGvpe8gaxG+qoIauqBJxF10\\\\\\\/3m7ROxGEt+fbHe4OEOqtWhy\\\\\\\/o474YmU9gGtMWTe+sFKTMm+aJbLcz91ADI\\\\\\\/qa+ngBCqQ2yAsJIZpn0Sq7rrCTbJMa3ldueFL3h5GBoFzGq5wJfJyqokbhhItkJ0LxqeGyJEprlgYKC4OomnplBCcRNE2FId5G4+blGMEFaBr9fxq\\\\\\\/JIgOOKYDs9dffF2DD4QVoS2lAD\\\\\\\/zpWv2eZnqQ20OuV5DHcHTaPqv5bfOz97rUMbm4iNl6XuxNbsZoYCkSYKeIVMtkjhVih7A1Ha43DCUqQnAMeIAxQ\\\\\\\/7+ElChZ9JpUgatQVRPCFid3E0ntiUxapz1z7Z8f0eBypJ\\\\\\\/S+TnvCtX9GlggVYjO1Y9Ula31B\\\\\\\/KnKWT3LBzJohk7u88uVEmlSBQ5Pq30vWSRfA4bkKpgzcNqgU3IStv++kOxG0KUWZpgEta8bfaS0MMG2I3sYcGOhUdwDY460yQ4lKZ2YbRvxnuT7Z4\\\\\\\/rLFIjAGcAEShRpE7lZ4JmYLapK9tEq1d1hT2zsbnbxDhiulc7ImAIeH0FChczrxdnTujFM\\\\\\\/eYgeV2QcWxHab0UX3I6b2XFphLulLWFe++L5rEXnqFq61Rk+21HGI0MKBmukllBDszir0KeCobtkvX7LVlzFa1kvOSdD1Y3p1GVycXCfbgxRGZltg92zQCiLQp+AHQYETflQaNk5VNy8UEXwi9G594jAtBKBPlhRGm8kcoxTTc+G30pnrXvz88VspxV4gPOPTZbKNUeLUmOpkZRd4j5H1Y7aW5Lud09K3S\\\\\\\/YtsPblscKspqzf8gzGKgtOaE3a0BDDyRD9PIeL2PbUMX5Ooze6k0ZXM3jyRAlstbLCD3udo\\\\\\\/ksUTsj9E8ZdsSOCsoSffpyL1BdMcH7VZMmUasqnY\\\\\\\/s6BRu6LmAh1SoSTMk3ifyNiB5Ud5qeuiBnf4NXalB9K1g4Slh3UbJK7nBr0nbV51VWAae4g1WhYkbanoORQ5ePkZuEB3CYUbudHuwPoOct7SBAoxmFvLHXp04eapNzn1YU6AyC8eZnj+osplWBW8Tp8ha3RsHWdwuv\\\\\\\/jrSg94moR5tLfIxS0wdkH+nZ+hL+vUR8UTFFOdXvHg9HZMBl+oNa6VMtv2EjkqHxa7ifwpUxb56Z4Wp3Fp9FsRVD7VbWdntrx9P77075g51QZyEnmxsbG0KrD6LUC9ZPuhVqvO0eh2GSGg2q\\\\\\\/6K+r4ClajiO4TKJK1RvBbL1EbtZSLMXQJtLZ8oCZFBQtdS56jJwpTS0iRvEVD0EaG3SbdAV2Gu\\\\\\\/glk2XMroWHiKUtTY3mE9rtVb5sb\\\\\\\/S+9PCbMjkIp0Mq15Bjvznhg8Ep7QeMYTlDg26\\\\\\\/TvToUoI0Pk1Pv0rFMZMxkwObgpdYwZ2QQu43iB1nPjtgeCcU4TAa2SWYqe1RB5j73+WVgbbv23DwwE5xqFVtbu0cbqzh6+m0epiLOcK5byckJmYjLkojtnsiGHW9jlpPSA07u6m4anLBxHtI8VlZXWqdaZpFHedSISt6biPAgxnIF8471UBrjWFGou7YThyeEZQDSphn3M3tl+T04fNAWSa0z8x6oltIgv4zDeUdFGompj2kdOGfqRXNwTUkXCwECbcNIt\\\\\\\/dHK8HRRUWa56+maaMVRfe4a5n7KCpLMkQ4HHCIKqQ\\\\\\\/QtTroEQAggHCh\\\\\\\/f71SEC31R6slQP3zTVvr9PfHhy7N\\\\\\\/rBYFXYhgH+RgCQZIK2EYFyit1tXYbyUpU+7SVM\\\\\\\/IdUkDlJq+DGhnFOUSlw5uUP3aKHqim5ymOmprLWqV4swTvXGAhW9XbeH4\\\\\\\/ahbMu0\\\\\\\/CZ27ir2Cg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022j5blFA6QmmaTWhnzGkeFFA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022be11aba6b7b86388b591ba0de73895667e6c68de415bb59b5e75b86bd1f97ffc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459427 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/505" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qY2rJHpMMdM5LqXXIufWx1Q566NH1GR0B0KIUgun1dQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/505" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JDSFQUAmeDWSmH5r4amZ9SVwD0mmDIFxN9aUJBNt4VQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/493" title="Genetic Interactions of Arabidopsis thaliana Damaged DNA Binding Protein 1B (DDB1B) With DDB1A, DET1, and COP1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/517" title="PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459427" id="node459427" data-pisa="ggg;3/3/505" data-pisa-master="ggg;3/3/505" data-apath="/ggg/3/3/505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kasie Raymann</span>, <span class="highwire-citation-author" data-delta="1">Louis-Marie Bobay</span>, <span class="highwire-citation-author" data-delta="2">Thomas G. Doak</span>, <span class="highwire-citation-author" data-delta="3">Michael Lynch</span> and <span class="highwire-citation-author" data-delta="4">Simonetta Gribaldo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">505-516;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005231 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459427" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kasie Raymann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kasie%2BRaymann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raymann%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKasie%2BRaymann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Louis-Marie Bobay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut Pasteur, Microbial Evolutionary Genomics, Departement Genomes et Genetique, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre National de la Recherche Scientifique, Unité Mixte de Recherche 3525, Paris, F-75015 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis-Marie%2BBobay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bobay%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis-Marie%2BBobay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas G. Doak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Indiana University, Department of Biology, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BG.%2BDoak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doak%20TG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BG.%2BDoak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Indiana University, Department of Biology, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Simonetta Gribaldo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simonetta%2BGribaldo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gribaldo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimonetta%2BGribaldo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:simonetta.gribaldo@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">simonetta.gribaldo@pasteur.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/3/505" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459427" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/505.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459427" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/505.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459427" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/505.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459427" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/3/505.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459427" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Bacteria and eukaryotes are involved in many types of interaction in nature, with important ecological consequences. However, the diversity, occurrence, and mechanisms of these interactions often are not fully known. The obligate bacterial endosymbionts of Paramecium provide their hosts with the ability to kill sensitive Paramecium strains through the production of R-bodies, highly insoluble coiled protein ribbons. R-bodies have been observed in a number of free-living bacteria, where their function is unknown. We have performed an exhaustive survey of genes coding for homologs of Reb proteins (R-body components) in complete bacterial genomes. We found that <em>reb</em> genes are much more widespread than previously thought, being present in representatives of major Proteobacterial subdivisions, including many free-living taxa, as well as taxa known to be involved in various kinds of interactions with eukaryotes, from mutualistic associations to pathogenicity. Reb proteins display very good conservation at the sequence level, suggesting that they may produce functional R-bodies. Phylogenomic analysis indicates that <em>reb</em> genes underwent a complex evolutionary history and allowed the identification of candidates potentially involved in R-body assembly, functioning, regulation, or toxicity. Our results strongly suggest that the ability to produce R-bodies is likely widespread in Proteobacteria. The potential involvement of R-bodies in as yet unexplored interactions with eukaryotes and the consequent ecological implications are discussed.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/kappa-particles" class="hw-term hw-article-keyword hw-article-keyword-kappa-particles" rel="nofollow">kappa particles</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/caedibacter" class="hw-term hw-article-keyword hw-article-keyword-caedibacter" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Caedibacter</em></a></li><li class="kwd"><a href="/keyword/phylogenomics" class="hw-term hw-article-keyword hw-article-keyword-phylogenomics" rel="nofollow">phylogenomics</a></li></ul><p id="p-4">During more than two billion years of coexistence, prokaryotes have established various forms of interaction with eukaryotes. Examples include the mutualistic symbioses that benefit eukaryotic host by providing nutrients, defense, competition, and adaptation to new environments (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gast <em>et al.</em> 2009</a>). At the same time, bacteria have developed various ways to defend themselves against grazing by eukaryotes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Matz and Kjelleberg 2005</a>), with potential implications for the emergence of pathogens (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown and Barker 1999</a>). However, the extent and diversity of bacterial/eukaryotic interactions in nature remains largely underexplored. As a growing amount of genomic data covering a large fraction of bacterial diversity becomes available, hints of such relationships may be gathered from <em>in silico</em> analyses. These can be linked to experimental observations, providing useful directions for further work.</p><p id="p-5">A fascinating example of a bacterial/eukaryote relationship is provided by the killer endosymbionts of the ciliate Paramecium. In the 1930s Tracey Sonneborn discovered that some strains of the <em>Paramecium aurelia</em> complex have a killer phenotype toward sensitive strains (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beale and Preer 2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Preer 2006</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sonneborn 1938</a>). Sonneborn could show that this phenomenon is not controlled by nuclear genes, providing one of the first examples of cytoplasmic inheritance (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sonneborn 1943</a>). It was later discovered that the killer phenotype is conveyed by an obligate endosymbiotic bacterium (also referred to as a kappa particle), and each killer paramecium strain harbors its own specific endosymbiont that usually resides in the cytoplasm but can also be in the nucleus (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Preer 1975</a>). Whereas all these obligate endosymbionts initially were placed in the genus Caedibacter, they were later shown to belong to different proteobacterial lineages (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beier <em>et al.</em> 2002</a>).</p><p id="p-6">In Caedibacter, the killer trait is directly linked to the production of R-bodies, unusual cytoplasmic refractile inclusion bodies (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dilts and Quackenbush 1986</a>) [for review, see (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pond <em>et al.</em> 1989</a>)]. What is known about R-bodies comes primarily from <em>Caedibacter taeniospiralis</em>, which belongs to the gammaproteobacteria family of Thiotrichales (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Beier <em>et al.</em> 2002</a>). R-bodies are highly insoluble protein ribbons that are typically coiled into cylindrical structures. They are produced by only a fraction of the endosymbiont population, which then stop dividing. When R-body−containing bacteria are released into the environment and captured by sensitive strains, killing occurs very rapidly. The exact mechanism for killing is not known, but it is thought that internalization of the R-body-containing bacteria into the food vacuole triggers unrolling of the R-body, which penetrates the phagosomal membrane and delivers a killer toxin to the cytoplasm (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jurand <em>et al.</em> 1971</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Preer <em>et al.</em> 1974</a>). Isolated R-body−containing bacteria are capable of killing sensitive Paramecium strains in various ways, with prelethal symptoms such as paralysis, vacuolization, and opposite swimming rotation. On the contrary, exposure to non-R-body−containing Caedibacter is not lethal [for review, see (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Pond <em>et al.</em> 1989</a>)], and a mutant Caedibacter strain unable to make R-bodies loses its killing ability (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dilts and Quackenbush 1986</a>). Different strains of Caedibacter produce different types of R-bodies, which vary in diameter (0.25−0.8 mm), length (&lt;10−30 mm), ribbon morphology (tapered or blunt termini), mode of unrolling (from the outside or from the inside in a telescopic fashion), and the nature of the stimulus for unrolling (changes in pH, temperature, ionic strength) [for review, see (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Pond <em>et al.</em> 1989</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sanchez-Amat 2006</a>)]. Interestingly, the unrolling of type 51 R-bodies has been shown to be reversible (they unroll when the pH is dropped &lt;6.5 and reroll when the pH is again raised &gt;7.0) (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Pond <em>et al.</em> 1989</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/505/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of the C. taeniospiralis R-body toxin delivery system (see main text for details and references)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-97130018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of the &lt;em&gt;C. taeniospiralis&lt;/em&gt; R-body toxin delivery system (see main text for details and references).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/505/F1.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/505/F1.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/505/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/505/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462510" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-7" class="first-child">Illustration of the <em>C. taeniospiralis</em> R-body toxin delivery system (see main text for details and references).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Studies of the genetic determinants of R-bodies began in the early 1980s, and in <em>C. taeniospiralis</em> the R body−coding region was found to lie on a plasmid (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Quackenbush and Burbach 1983</a>). When a region from the pKAP47 plasmid (from <em>Paramecium teraurelia</em> California strain 47) was cloned into <em>Escherichia coli</em>, R-bodies were produced, but the clones did not exhibit toxicity toward sensitive Paramecium strains (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Quackenbush and Burbach 1983</a>). Therefore, whereas production of R-bodies is necessary for killing by <em>C. taeniospiralis</em>, as described previously, it is not sufficient for killing by recombinant <em>Escherichia coli</em>. This excludes a direct cytotoxic effect of R-bodies and indicates a requirement for an essential unknown toxin encoded either by the plasmid or the <em>C. taeniospiralis</em> genome (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Preer and Stark 1953</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Quackenbush and Burbach 1983</a>). These data have been recently confirmed, and it has been shown that recombinant <em>E. coli</em> expressing the four <em>reb</em> genes of the <em>C. taeniospiralis</em> pKAP298 plasmid (from <em>Paramecium teraurelia</em> Panama strain 298) were capable of producing R-bodies but were not toxic toward sensitive Paramecium strains (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schrallhammer <em>et al.</em> 2012</a>).</p><p id="p-9">Very little information is available on the assembly process of R-bodies. At least three polypeptides of 10, 13, and 18 kDa were found to be involved in the structure and assembly of type 51 R-bodies (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kanabrocki <em>et al.</em> 1986</a>) and were later shown to be encoded by three genes: <em>reb</em>C, <em>reb</em>B, and <em>reb</em>A, respectively, the last two being homologous (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heruth <em>et al.</em> 1994</a>). These early data proposed that the major structural protein is RebB and that RebA may act as a scaffold to facilitate the polymerization process whereas RebC may act as a transcriptional regulator (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Heruth <em>et al.</em> 1994</a>). Finally, it was suggested that RebB might be modified posttranslationally, with the possible involvement of RebC (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Heruth <em>et al.</em> 1994</a>). The role of a fourth gene in the <em>reb</em> locus, <em>reb</em>D, coding for a homolog of RebA and RebB, is unclear but it was shown not to be necessary for R-body production in <em>E. coli</em> (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Heruth <em>et al.</em> 1994</a>).</p><p id="p-10">The complete sequence of the Reb-harboring pKAP298 plasmid of <em>C. taeniospiralis</em> strain 298 was obtained in 2005 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jeblick and Kusch 2005</a>). It was found that this plasmid contains 63 open reading frames, 23 only having similarity with proteins with known function, and a few being similar to proteins encoded by phages or prophages, which led to the suggestion that the plasmid originated from a bacteriophage (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jeblick and Kusch 2005</a>), which is consistent with early observations of the association of phage-like particles with R-bodies (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Preer <em>et al.</em> 1974</a>). A protein with homology to the Soj-ParA family of membrane-associated ATPases was suggested as a possible candidate for the toxin, which would kill the host by somehow affecting its membrane, although a precise mechanism was not proposed (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Jeblick and Kusch 2005</a>).</p><p id="p-11">The harboring of an endosymbiont that produces R-bodies gives a competitive advantage to its killer Paramecium host with respect to sensitive strains (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kusch <em>et al.</em> 2002</a>). In turn, R-body production seems to play a role in defense against predation and creates a benefit for the Caedibacter strains at the population level (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sanchez-Amat 2006</a>). However, many important questions remain to be clarified. For example, it is not known how obligate symbiosis is established in the first place or how sensitive strains can pick up Caedibacter and become killers, nor how killer Paramecium strains are protected from their own specific R-body producing endosymbionts (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gibson 1973</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Preer <em>et al.</em> 1974</a>).</p><p id="p-12">Interestingly, casual observations of coiled R-body structures of various types have been reported in several free-living bacteria: the hydrogen-oxidizing β-proteobacterium <em>Pseudomonas taeniospiralis</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lalucat and Mayer 1978</a>), now known as <em>Hydrogenophaga taeniospiralis</em>; the soil β-proteobacterium <em>Pseudomonas avenae</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wells and Horne 1983</a>), now known as <em>Acidovorax avenae</em> subsp. <em>avenae</em>; the soil strain <em>Pseudomonas</em> sp. EPS-5028 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fusté <em>et al.</em> 1986</a>); the anoxigenic photosynthetic N2-fixing α-proteobacterium <em>Rhodospirillum centenum</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Favinger <em>et al.</em> 1989</a>); the soil strain <em>Pseudomonas aeruginosa</em> 44T1 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Espuny <em>et al.</em> 1991</a>); and the melanin-producing marine γ-proteobacterium <em>Marinomonas mediterranea</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hernandez-Romero <em>et al.</em> 2003</a>). However, no further study on these R-body structures has been reported for any of these species [for review, see (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Sanchez-Amat 2006</a>)], nor have they been linked to the presence of Reb homologs in their genomes. Therefore, the role of these R-bodies in these diverse bacterial remains puzzling. A recent study has shown the presence of Reb homologs in the genome of the rhizobiale <em>Azorhizobium caulinodans</em>, a microsymbiont of the tropical legume <em>Sesbania rostrata</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akiba <em>et al.</em> 2010</a>). Interestingly, deletion of the putative transcription factor <em>praR</em> caused aberrant nodule formation and was linked to greater expression of the <em>reb</em> locus. On the contrary, a double <em>reb</em> and <em>praR</em> mutant had a restored wild-type nodule formation. The authors hypothesized that <em>praR</em> is essential to suppress the killer trait conferred by the <em>reb</em> locus and establish symbiosis between <em>A. caulinodans</em> and <em>S. rostrata</em> (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akiba <em>et al.</em> 2010</a>). However, it is not known whether <em>A. caulinodans</em> is able to make R-bodies. The authors also reported the presence of Reb homologs in a number of Proteobacteria and in the Bacteroidetes member <em>Kordia algicida</em> OT-1 (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Akiba <em>et al.</em> 2010</a>).</p><p id="p-13">Here, we have performed an exhaustive phylogenomic analysis of Reb homologs in currently available bacterial genomes. Reb homologs are widely distributed in members of <em>Proteobacteria</em>, comprising many free-living taxa as well as symbionts or pathogens of various eukaryotes, including humans. The evolutionary history of <em>reb</em> genes appears very dynamic, involving vertical inheritance, horizontal gene transfers, and gene duplications. By combining phylogenetic, genome synteny, and genomic content analyses, we highlight a few potential candidate partners of Reb proteins. Finally, we found no clear signs of <em>reb</em> loci originating from defective prophages, or from recent transfer via mobile elements. Ecological implications are discussed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Homology searches</h3><p id="p-14">Reb proteins (A-D) encoded in the plasmid pKAP298 from <em>C. taeniospiralis</em> (AAR87077.1, AAR87076.1, AAR87131.1, AAR87075.1) were used as seeds to search for Reb homologs in the nonredundant protein database at the National Center for Biotechnology Information (NCBI). Homology searches were performed by BlastP (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1997</a>) and all hits within an <em>e</em>-value cutoff of 1 × 10<sup>−3</sup> were retained. PSI-BLAST and tBLASTn programs (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1997</a>) also were used to search for highly divergent or misannotated Reb homologs. Searches were reiterated by using a number of seeds from various taxa. We also performed targeted searches against the metagenome and the viral sequence databases at the NCBI, and against all eukaryotic genome sequence available at the Joint Genome Institute (<a href="http://genome.jgi.gov">http://genome.jgi.gov</a>). Sequences were aligned using Muscle 3.8.31 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Edgar 2004</a>). Poorly aligned or divergent sequences were manually removed. Finally, HMM searches were performed with HMMER 3.0 (hmmer.org) against a local databank of 841 complete bacterial genomes (only one representative per species), including 435 from Proteobacteria downloaded from the NCBI ftp Genomes server using a model built on the multiple alignment of all previously recovered Reb proteins, but no additional homologs were found.</p></div><div id="sec-3" class="subsection"><h3>Sequence analysis</h3><p id="p-15">Sequence secondary structures were predicted using PSIPRED (<a href="http://bioinf.cs.ucl.ac.uk/psipred/">http://bioinf.cs.ucl.ac.uk/psipred/</a>) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buchan <em>et al.</em> 2010</a>). Alpha helical wheel diagrams were created using the tool created by Don Armstrong and Raphael Zidovetzki (<a href="http://rzlab.ucr.edu/scripts/wheel/wheel.cgi">http://rzlab.ucr.edu/scripts/wheel/wheel.cgi</a>). PredictProtein was used to search for additional structural features (<a href="http://www.predictprotein.org/">http://www.predictprotein.org/</a>). We searched for Reb homologs with available 3D structures using sequence based-PSI-BLAST (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1997</a>) searches at the PFAM and Uniprot databases by using HHpred (<a href="http://hhpred.tuebingen.mpg.de/hhpred">http://hhpred.tuebingen.mpg.de/hhpred</a>) and FFAS03 (<a href="http://ffas.ljcrf.edu/ffas-cgi/cgi/ffas.pl">http://ffas.ljcrf.edu/ffas-cgi/cgi/ffas.pl</a>). WebLogo analysis was performed at <a href="http://weblogo.berkeley.edu/">http://weblogo.berkeley.edu/</a>.</p></div><div id="sec-4" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-16">The final dataset of 203 Reb homologs was aligned using Muscle 3.8.31 (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Edgar 2004</a>) and trimmed using BMGE (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Criscuolo and Gribaldo 2010</a>) with the less-stringent parameters (Blosum30), giving a dataset of 73 unambiguously aligned amino acid positions for phylogenetic analysis. A Bayesian tree was obtained using Phylobayes 3.3 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lartillot <em>et al.</em> 2009</a>). (See figure legends for details on analyses.)</p><p id="p-17">16s rRNA sequences from Proteobacterial taxa representative of the diversity of this phylum were downloaded from the NCBI, as well as from the specialized Silva (<a href="http://www.arb-silva.de/">http://www.arb-silva.de/</a>) and the Ribosomal Database Project (<a href="http://rdp.cme.msu.edu/">http://rdp.cme.msu.edu/</a>) databases. Sequences were aligned using Muscle 3.8.31 (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Edgar 2004</a>) and manually trimmed using the ED program of the MUST suite (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Philippe 1993</a>). Maximum likelihood trees were obtained using Treefinder (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jobb <em>et al.</em> 2004</a>). (See figure legends for details on analyses.)</p></div><div id="sec-5" class="subsection"><h3>Genome synteny and genome content analysis</h3><p id="p-18">For genome synteny analysis, we retrieved the five open reading frames upstream and downstream of the <em>reb</em> loci (or extracted from contigs when the whole genome sequence was not available). Protein sequences were defined as homologous when sharing at least 40% similarity and less than 20% difference in length. Pairs of homologous proteins were then expanded to homologous protein families by including all proteins homologous to at least one member of the family.</p><p id="p-19">For genome content analysis, families of homologous proteins were built from 861 fully sequenced bacterial genomes downloaded from the NCBI ftp Genomes server. Protein sequences were defined as homologous if sharing at least 50% similarity and less than 20% difference in length. Pairs of homologous proteins were then expanded to homologous protein families by including all proteins homologous to at least one member of the family.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Taxonomic distribution</h3><p id="p-20">Although the production of R-bodies has been observed in a few free-living bacteria (see Introduction), the effective distribution of Reb homologs in prokaryotes has not been clear. We carried out an exhaustive search for Reb homologs in current sequence databases (see Materials and Methods) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We found no additional homologs of RebC other than the <em>C. taeniospiralis</em> pKAP298 plasmid, indicating that this protein is specific to the Caedibacter Reb system. On the contrary, RebB, RebA and RebD are homologous and widely distributed. We identified 203 Reb homologs from 64 taxa belonging exclusively to <em>Proteobacteria</em>, with the one exception of <em>Kordia algicida</em> OT-1, which belongs to the phylum <em>Bacteroidetes</em>, as recently noticed (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Akiba <em>et al.</em> 2010</a>). Reb homologs are widely distributed among representatives of four of the six subdivisions of <em>Proteobacteria</em>, Alpha, Beta, Gamma, and Delta (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). We found between one and nine Reb homologs in each genome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Although <em>reb</em> genes were first identified on the <em>C. taeniospiralis</em> pKAP298 plasmid, we found Reb homologs on only two additional plasmids: the megaplasmid from the α-proteobacterium <em>Ruegeria pomeroyi</em> DSS-3, and the AZOBR_p4 plasmid from the α-proteobacterium <em>Azospirillum brazilense</em> Sp245 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). The availability of a complete genome for these two taxa indicates that no additional homologs are present on the chromosome. We could not find any other Reb homologs in viruses, Archaea, or Eukarya, apart from four homologs from the Global Ocean Sampling marine metagenome sequence database that are closely related to Proteobacteria (not shown).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/505/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of Reb homologs. Presence/absence of Reb homologs in proteobacteria and Kordia algicida. Colors indicate the different proteobacterial subdivisions. For each genome that harbors Reb homologs, we included complete genomes of closely related taxa without any reb genes, when available. Taxa with no available complete genome sequence but harboring Reb homologs are highlighted in gray. For these taxa, the presence of extra Reb copies cannot be excluded. When present, Reb homologs are indicated by their corresponding accession number. Reb homologs located on plasmids are indicated by an asterisk. See main text for discussion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-97130018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of Reb homologs. Presence/absence of Reb homologs in proteobacteria and &lt;em&gt;Kordia algicida&lt;/em&gt;. Colors indicate the different proteobacterial subdivisions. For each genome that harbors Reb homologs, we included complete genomes of closely related taxa without any &lt;em&gt;reb&lt;/em&gt; genes, when available. Taxa with no available complete genome sequence but harboring Reb homologs are highlighted in gray. For these taxa, the presence of extra Reb copies cannot be excluded. When present, Reb homologs are indicated by their corresponding accession number. Reb homologs located on plasmids are indicated by an asterisk. See main text for discussion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/505/F2.medium.gif" width="334" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/505/F2.medium.gif" width="334" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/505/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/505/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462575" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-21" class="first-child">Distribution of Reb homologs. Presence/absence of Reb homologs in proteobacteria and <em>Kordia algicida</em>. Colors indicate the different proteobacterial subdivisions. For each genome that harbors Reb homologs, we included complete genomes of closely related taxa without any <em>reb</em> genes, when available. Taxa with no available complete genome sequence but harboring Reb homologs are highlighted in gray. For these taxa, the presence of extra Reb copies cannot be excluded. When present, Reb homologs are indicated by their corresponding accession number. Reb homologs located on plasmids are indicated by an asterisk. See main text for discussion.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/505/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of Reb-harboring taxa across Proteobacteria. Unrooted Maximum likelihood phylogenetic tree of 16s rRNA sequences from 60 taxa representative of proteobacterial diversity. Proteobacterial orders that include members containing Reb homologs are highlighted in red. The number of Reb-harboring taxa over the total number of available complete genomes is indicated in parenthesis. Caedibacter taeniospiralis belongs to the gammaproteobacterial family of Thiotrichales (indicated by a red arrow). The tree was obtained using Treefinder with the J1 model of nucleotide substitution and a discrete gamma distribution with four categories to take into account among-site rate variation. Numbers at nodes indicate bootstrap values (BV) for 100 replicates of the original dataset. For clarity, only BVs greater than 50% are shown. The scale bar represents the average number of substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-97130018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of Reb-harboring taxa across Proteobacteria. Unrooted Maximum likelihood phylogenetic tree of 16s rRNA sequences from 60 taxa representative of proteobacterial diversity. Proteobacterial orders that include members containing Reb homologs are highlighted in red. The number of Reb-harboring taxa over the total number of available complete genomes is indicated in parenthesis. &lt;em&gt;Caedibacter taeniospiralis&lt;/em&gt; belongs to the gammaproteobacterial family of Thiotrichales (indicated by a red arrow). The tree was obtained using Treefinder with the J1 model of nucleotide substitution and a discrete gamma distribution with four categories to take into account among-site rate variation. Numbers at nodes indicate bootstrap values (BV) for 100 replicates of the original dataset. For clarity, only BVs greater than 50% are shown. The scale bar represents the average number of substitutions per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/505/F3.medium.gif" width="271" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/505/F3.medium.gif" width="271" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/505/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/505/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462538" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-22" class="first-child">Distribution of Reb-harboring taxa across Proteobacteria. Unrooted Maximum likelihood phylogenetic tree of 16s rRNA sequences from 60 taxa representative of proteobacterial diversity. Proteobacterial orders that include members containing Reb homologs are highlighted in red. The number of Reb-harboring taxa over the total number of available complete genomes is indicated in parenthesis. <em>Caedibacter taeniospiralis</em> belongs to the gammaproteobacterial family of Thiotrichales (indicated by a red arrow). The tree was obtained using Treefinder with the J1 model of nucleotide substitution and a discrete gamma distribution with four categories to take into account among-site rate variation. Numbers at nodes indicate bootstrap values (BV) for 100 replicates of the original dataset. For clarity, only BVs greater than 50% are shown. The scale bar represents the average number of substitutions per site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Of importance, for three taxa with available sequence data, we could link for the first time the previously reported observation of R-bodies (see <em>Introduction</em> and references therein) with the presence of Reb homologs. In particular, <em>R. centenum</em> has six copies, <em>A. avenae</em> has four copies, and <em>M. mediterranea</em> has nine copies (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Reb-containing taxa display a wide variety of lifestyles. Albeit many taxa harboring Reb homologs harbor free-living lifestyles in a wide variety of environments, from marine to terrestrial, a few taxa other than <em>C. taeniospiralis</em> appear to have an interaction with eukaryotes. For example, <em>Pseudovibrio sp. JE062</em> is a symbiont of sponges; <em>Vibrio fischeri ES114</em> is the specific bioluminescent symbiont in the light-emitting organs of certain squids and fishes; <em>Labrenzia alexandrii</em> and <em>Oceanicaulis alexandrii</em> have been isolated from dinoflagellates; <em>Stenotrophomonas maltophilia</em> R551-3, <em>Azospirillum brasilense</em> Sp245, and <em>Pseudomonas fluorescens</em> Pf-5 are plant growth−promoting endophytes; various strains of <em>Burkholderia ambifaria</em> are important in the biocontrol of pea plant phytopathogens; and <em>Azorhizobium caulinodans</em> is a nitrogen-fixing proteobacterium involved in mutualistic rhizobiale symbioses with plant roots.</p><p id="p-24">Other than in the algae pathogen <em>Kordia algicida</em>, we also found Reb homologs in a number of important pathogens of plants such as <em>Acidovorax avenae subsp. avenae</em> ATCC 19860; various strains of <em>Xanthomonas</em>; <em>Burkholderia gladioli and B. glumae</em>; <em>Ralstonia syzygii</em> R24; but also in important pathogens of aquatic animals such as shrimp and corals (<em>Vibrio nigripulchritudo</em> ATCC27043; <em>Vibrio coralliilyticus</em> ATCC-BAA450). Reb homologs were also found in the genomes of opportunistic pathogens of humans, such as various strains of <em>Pseudomonas aeruginosa</em>, including the hypervirulent LESB58 strain; in various strains of <em>Burkholderia pseudomallei</em>, the causative agent of meilodiosis, in <em>B. mallei</em>, that causes glanders and pneumonia; and <em>Stenotrophomonas maltophila</em>, a rare but serious threat to patients who require catheterization. We also observed some interesting patterns by looking at the distribution of Reb homologs in closely related strains (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). For example, although the sponge symbiont <em>Pseudovibrio sp</em>. JE062 harbors <em>reb</em> genes, its closely related free-living relative <em>Pseudovibrio sp</em>. FO-BEG1 does not. Similarly, <em>Burkholderia thailandensis</em> MSMB43, an opportunistic pathogen that causes meilodiosis, has <em>reb</em> genes, whereas the closely related <em>B. thailandensis</em> E264, a common soil and avirulent strain, does not (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>).</p></div><div id="sec-8" class="subsection"><h3>Sequence analysis</h3><p id="p-25">Despite the widespread presence of Reb homologs in many bacterial taxa, it remains to be proven experimentally that these are responsible for producing R-bodies. However, some hints can already be gained from sequence analysis.</p><p id="p-26">Reb homologs are 95 amino acids long on average. They show good conservation at the sequence level and all display a basic alpha helical secondary structure with no significant structural difference among sequences (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Very little is known about the regulation and mechanism of R-body assembly [for review see, (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Sanchez-Amat 2006</a>)]. We could not observe any particular pattern in the sequences that allows distinguishing the equivalents of RebA, B, and D of <em>C. taeniospiralis</em> in other taxa, and this was also confirmed by phylogenetic analysis. Overall sequence conservation is high, suggesting that these Reb homologs are likely functional and conserved at the structural level. A WebLogo analysis highlighted highly conserved positions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>) that may be important for R-body assembly and/or unrolling-rolling, and should be the target of choice of future mutation studies. Unfortunately, we could not identify any homologous proteins with a solved crystal structure in extant databases (see <em>Materials and Methods</em>). Small proteins assembling into structures frequently display amphipathic helices, which consist of hydrophobic amino acids concentrated on one side and hydrophilic or polar amino acids on the other, and these can be highlighted using helical wheel diagrams (see <em>Materials and Methods</em>). However, we found no evidence that the helices of Reb homologs display amphipathic character (data not shown). Obtaining the 3-D structure of an R-body will therefore be essential to understand how Reb homologs assemble and function.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/505/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence analysis. (A) Secondary structure of the RebB of C. taeniospiralis predicted by PSIPRED [http://bioinf.ucl.ac.uk/psipred (Buchan et al. 2010)]. The same structure was substantially conserved in all other Reb homologs. (B) Conserved amino acid positions identified using Weblogo on an unambiguously aligned excerpt of the entire alignment of the 203 identified Reb homologs. For clarity, only 15 representative Reb sequences are shown. Position numbers refer to the RebB of C. taeniospiralis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-97130018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequence analysis. (A) Secondary structure of the RebB of &lt;em&gt;C. taeniospiralis&lt;/em&gt; predicted by PSIPRED [http://bioinf.ucl.ac.uk/psipred (Buchan &lt;em&gt;et al.&lt;/em&gt; 2010)]. The same structure was substantially conserved in all other Reb homologs. (B) Conserved amino acid positions identified using Weblogo on an unambiguously aligned excerpt of the entire alignment of the 203 identified Reb homologs. For clarity, only 15 representative Reb sequences are shown. Position numbers refer to the RebB of &lt;em&gt;C. taeniospiralis&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/505/F4.medium.gif" width="440" height="274"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/505/F4.medium.gif" width="440" height="274"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/505/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/505/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462526" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-27" class="first-child">Sequence analysis. (A) Secondary structure of the RebB of <em>C. taeniospiralis</em> predicted by PSIPRED [<a href="http://bioinf.ucl.ac.uk/psipred">http://bioinf.ucl.ac.uk/psipred</a> (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Buchan <em>et al.</em> 2010</a>)]. The same structure was substantially conserved in all other Reb homologs. (B) Conserved amino acid positions identified using Weblogo on an unambiguously aligned excerpt of the entire alignment of the 203 identified Reb homologs. For clarity, only 15 representative Reb sequences are shown. Position numbers refer to the RebB of <em>C. taeniospiralis</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">When present in multiple copies, Reb homologs are clustered on the genome, mostly lying side-by-side or separated by a few intervening genes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We generally found only one <em>reb</em> locus per genome, with the exception of <em>M. mediterranea</em> whose nine homologs are organized into two different genomic regions, and <em>Xanthomonas axonopodis</em> and <em>Chromobacterium violaceum</em>, which both have an extra Reb homolog located far from the main cluster (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Given their short length, the phylogeny of Reb homologs is globally poorly resolved, but a few monophyletic groups are apparent which are consistent with genomic synteny (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005231/-/DC1/005231SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005231/-/DC1/FigureS1.pdf">Figure S1</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). This allowed us to infer the evolutionary history of these Reb homologs (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). In some cases, Reb homologs from the same taxon are more closely related to each other than to Rebs of other taxa, suggesting that these have arisen from species-specific duplication events.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/505/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary inference of Reb homologs based on phylogenetic analysis of the 203 Reb homologs (Figure S1). Here we have highlighted a few of the monophyletic groups. For each taxon, the genome locations of the corresponding Reb proteins are shown. Reb homologs highlighted in green are orthologs that were inferred to have been inherited through speciation events; those highlighted in blue represent paralogs issued from species-specific gene duplications; and those in red are the Reb homologs that have likely originated via horizontal transfer. Adjacent reb genes are indicated in gray. Open reading frames between reb genes are shown in white, and black slash-like symbols represent large intervening regions between reb genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-97130018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evolutionary inference of Reb homologs based on phylogenetic analysis of the 203 Reb homologs (Figure S1). Here we have highlighted a few of the monophyletic groups. For each taxon, the genome locations of the corresponding Reb proteins are shown. Reb homologs highlighted in green are orthologs that were inferred to have been inherited through speciation events; those highlighted in blue represent paralogs issued from species-specific gene duplications; and those in red are the Reb homologs that have likely originated via horizontal transfer. Adjacent &lt;em&gt;reb&lt;/em&gt; genes are indicated in gray. Open reading frames between &lt;em&gt;reb&lt;/em&gt; genes are shown in white, and black slash-like symbols represent large intervening regions between &lt;em&gt;reb&lt;/em&gt; genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/505/F5.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/505/F5.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/505/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/505/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462542" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-29" class="first-child">Evolutionary inference of Reb homologs based on phylogenetic analysis of the 203 Reb homologs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005231/-/DC1/FigureS1.pdf">Figure S1</a>). Here we have highlighted a few of the monophyletic groups. For each taxon, the genome locations of the corresponding Reb proteins are shown. Reb homologs highlighted in green are orthologs that were inferred to have been inherited through speciation events; those highlighted in blue represent paralogs issued from species-specific gene duplications; and those in red are the Reb homologs that have likely originated via horizontal transfer. Adjacent <em>reb</em> genes are indicated in gray. Open reading frames between <em>reb</em> genes are shown in white, and black slash-like symbols represent large intervening regions between <em>reb</em> genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">This is for, instance, the case of five <em>reb</em> genes from <em>Marinomonas sp</em>. MED121, which are all more closely related to each other than to any other Reb (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). The same can be said for the six of the seven <em>reb</em> genes from <em>Vibrio fischeri</em> ES114 (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). In other cases, there is clear evidence for vertical inheritance of Reb proteins from the ancestor of a specific <em>Proteobacterial</em> family (<em>e.g.</em>, <em>Xanthomonas</em>) (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>). In yet other instances, Reb proteins are most closely related among distantly related lineages, suggesting horizontal transfer of the whole locus, for example in the case of Marinomonas mediterranea MMB-1 and Shewanella denitrificans OS217 (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). Horizontal gene transfer was also suggested for extra <em>reb</em> copies in a few taxa (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>). Finally, phylogeny could not help in assigning the equivalents in other taxa of the RebA-B-D of <em>C. taeniospiralis</em>, as these are more closely related among themselves (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005231/-/DC1/FigureS1.pdf">Figure S1</a>). It is therefore difficult to make analogies between the previously reported data on the role of the RebA-B-D of <em>C. taeniospiralis</em> in the assembly process of its R-bodies and what occurs in the other taxa. Moreover, because we found no homologs of RebC outside <em>C. taeniospiralis</em>, it is possible that other proteins have analogous function in Reb-harboring taxa. This finding would be consistent with RebC being a transcription regulator and therefore potentially species-specific.</p></div><div id="sec-9" class="subsection"><h3>In search for potential partners of Reb proteins</h3><p id="p-31">It has been shown that RebA, B and C from <em>C. taeniospiralis</em> are sufficient for production of type 51 R-body in <em>E. coli</em> but not for the killing phenotype (see <em>Introduction</em>), which indicates that yet-unidentified partners coded on either the plasmid or the chromosome of <em>C. taeniospiralis</em> are involved in the killing. Interestingly, we found no homologs of the 63 proteins encoded in the <em>C. taeniospiralis</em> pKAP298 plasmid in any of the Reb-harboring taxa. This may indicate that none of these proteins is a likely candidate for the killing toxin, which would be then encoded in the <em>C. taeniospiralis</em> genome (yet unavailable). Alternatively, the <em>C. taeniospiralis</em> toxin may well be on the plasmid but is not conserved in other bacteria, which may either display no killing activity or use nonhomologous toxins.</p><p id="p-32">To search for candidate partners of Reb proteins, we carried out a genome synteny analysis of the <em>reb</em> locus in 41 taxa for which a complete genome or sufficient genomic structure information is available (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, see <em>Materials and Methods</em>). Two mutually exclusive synteny patterns could be observed (hereafter referred to as Group 1 and 2, respectively; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Strikingly, the genes included in these conserved synteny patterns are exclusively present in Reb-harboring taxa, strongly suggesting a functional link with the Reb system. The Group 1 synteny pattern is defined by four proteins annotated as hypothetical: HP1.1 (red), HP1.2 (blue), HP1.3 (yellow), and HP1.4 (orange), which are only found in the surroundings of the <em>reb</em> locus and are only present in Reb-harboring taxa. The HP1.4 (~60−80 aa) and HP1.3 (!170 aa) proteins appear to be distant Reb homologs. However, they lack some of the conserved amino acid positions characteristic of other Reb proteins, and the HP1.3 protein is approximately twice as long as a typical Reb (data not shown). The HP1.1 (~360 aa) and the HP1.2 (~110−120 aa) proteins display no putative conserved domains. These four proteins often exhibit the same genomic organization (HP1.1,HP1.2,HP1.3,HP1.4; <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). In three cases, another hypothetical protein (HP1.5, purple) is associated with this context, and is distantly related to the HP1.3 protein (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/505/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome synteny analysis of the reb locus mapped onto an Unrooted Maximum likelihood phylogenetic tree of 16s rRNA sequences from the 64 reb-containing Proteobacteria. The tree was created using Treefinder with the GTR model of nucleotide substitution and a discrete gamma distribution with four categories to take into account among-site rate variation. Numbers at nodes indicate BVs for 100 replicates of the original dataset. For clarity, only BVs greater than 50% are shown. The scale bar represents the average number of substitutions per site. Species where Reb homologs are located on a plasmid are marked by a black circle. A white star in a red circle marks the fully sequenced genomes used in the analysis. Reb homologs are shown in green. Homologous genes are represented by the same color. For clarity, only genes discussed in the text are indicated. Black slash-like symbols represent large regions in between genes. The RebC of C.taeniospiralis (AAR87131) is shown in light green and outlined in black to indicate its lack of homology with the other rebs. The genome context for the Flavobacterium K. algicida is shown separately." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-97130018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome synteny analysis of the &lt;em&gt;reb&lt;/em&gt; locus mapped onto an Unrooted Maximum likelihood phylogenetic tree of 16s rRNA sequences from the 64 &lt;em&gt;reb&lt;/em&gt;-containing Proteobacteria. The tree was created using Treefinder with the GTR model of nucleotide substitution and a discrete gamma distribution with four categories to take into account among-site rate variation. Numbers at nodes indicate BVs for 100 replicates of the original dataset. For clarity, only BVs greater than 50% are shown. The scale bar represents the average number of substitutions per site. Species where Reb homologs are located on a plasmid are marked by a black circle. A white star in a red circle marks the fully sequenced genomes used in the analysis. Reb homologs are shown in green. Homologous genes are represented by the same color. For clarity, only genes discussed in the text are indicated. Black slash-like symbols represent large regions in between genes. The RebC of &lt;em&gt;C.taeniospiralis&lt;/em&gt; (AAR87131) is shown in light green and outlined in black to indicate its lack of homology with the other &lt;em&gt;rebs&lt;/em&gt;. The genome context for the &lt;em&gt;Flavobacterium K. algicida&lt;/em&gt; is shown separately.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/505/F6.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/505/F6.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/505/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/505/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462482" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-33" class="first-child">Genome synteny analysis of the <em>reb</em> locus mapped onto an Unrooted Maximum likelihood phylogenetic tree of 16s rRNA sequences from the 64 <em>reb</em>-containing Proteobacteria. The tree was created using Treefinder with the GTR model of nucleotide substitution and a discrete gamma distribution with four categories to take into account among-site rate variation. Numbers at nodes indicate BVs for 100 replicates of the original dataset. For clarity, only BVs greater than 50% are shown. The scale bar represents the average number of substitutions per site. Species where Reb homologs are located on a plasmid are marked by a black circle. A white star in a red circle marks the fully sequenced genomes used in the analysis. Reb homologs are shown in green. Homologous genes are represented by the same color. For clarity, only genes discussed in the text are indicated. Black slash-like symbols represent large regions in between genes. The RebC of <em>C.taeniospiralis</em> (AAR87131) is shown in light green and outlined in black to indicate its lack of homology with the other <em>rebs</em>. The genome context for the <em>Flavobacterium K. algicida</em> is shown separately.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The Group 2 synteny pattern is defined by the presence of two proteins annotated as hypothetical: HP2.1 (light blue) and HP2.2 (fuchsia) (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). These proteins are approximately the same size (~205−220 aa) and display no putative conserved domains. They frequently co-occur, but their genomic organization varies in different taxa. An interesting characteristic of Group 2 synteny pattern is the frequent association with a putative RNA polymerase sigma-factor protein (HP2.3, light pink) and a transcriptional regulator/cyclic nucleotide binding protein (HP2.4, dark pink), which might be involved in transcription regulation of <em>reb</em> genes in these taxa. It should be noted that the conserved Group 1 and Group 2 synteny patterns are generally consistent with the Reb clusters highlighted by phylogenetic analysis (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a>) and have phylogenies similar to the Reb one (not shown), indicating a common evolutionary history and providing further suggestion of a functional link between these proteins and Reb proteins. As an additional strong indication, the plasmid sequence of <em>A. brasilense</em> contains the four proteins characteristic of the Group 1 synteny pattern.</p><p id="p-35">A few taxa did not present any particularly conserved genomic context nor did they harbor the conserved genes found in Group 1 or Group2 synteny patterns. It is therefore possible that other genetic elements important for <em>reb</em> function are located in different positions of the genome in these taxa. To this end, we sought to identify additional proteins specific to Reb-harboring taxa by carrying out a whole-genome content analysis (see <em>Material and Methods</em>). Using the complete genomes of 841 bacterial taxa—including 25 Reb-harboring complete genomes—we constructed protein families having at least 50% identity and 80% size conservation (see <em>Material and Methods</em>). This analysis confirmed that the only protein family exclusively present in Reb-harboring taxa is the Reb family itself, along with the protein families specific to Group 1 and 2 synteny patterns (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Although genomes of taxa not containing Reb homologs harbored distant homologs of the HP2.3 and HP2.4 (light pink and dark pink) proteins of genome synteny patterns, these fell outside of subfamilies which are exclusively present in Reb-harboring taxa.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/505/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Whole genome content analysis. Graphical representation of protein families created using the R software (R Development Core Team 2011). The 25 fully sequenced Reb-harboring taxa are represented on the x-axis and the other 816 fully sequenced bacterial taxa analyzed are represented on the y-axis. Each point on the graph represents a protein family (see Materials and Methods for details on how protein families were defined). For example, the Reb family (indicated by a green box) is present in 24 fully sequenced Reb-harboring taxa but in none of the remaining genomes. The 4 Rebs of Acidovorax avenae subsp. Avenae ATCC 19860 did not fall into the Reb protein family because they are very divergent (see Figure 5 and Figure S1). The other five unique protein families specific to Reb-harboring taxa (see main text) are shown with boxes corresponding to colors as defined in the legend to Figure 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-97130018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Whole genome content analysis. Graphical representation of protein families created using the R software (R Development Core Team 2011). The 25 fully sequenced Reb-harboring taxa are represented on the x-axis and the other 816 fully sequenced bacterial taxa analyzed are represented on the y-axis. Each point on the graph represents a protein family (see &lt;em&gt;Materials and Methods&lt;/em&gt; for details on how protein families were defined). For example, the Reb family (indicated by a green box) is present in 24 fully sequenced Reb-harboring taxa but in none of the remaining genomes. The 4 Rebs of &lt;em&gt;Acidovorax avenae&lt;/em&gt; subsp. Avenae ATCC 19860 did not fall into the Reb protein family because they are very divergent (see Figure 5 and Figure S1). The other five unique protein families specific to Reb-harboring taxa (see main text) are shown with boxes corresponding to colors as defined in the legend to Figure 6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/505/F7.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/505/F7.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/505/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/505/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462613" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-36" class="first-child">Whole genome content analysis. Graphical representation of protein families created using the R software (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">R Development Core Team 2011</a>). The 25 fully sequenced Reb-harboring taxa are represented on the x-axis and the other 816 fully sequenced bacterial taxa analyzed are represented on the y-axis. Each point on the graph represents a protein family (see <em>Materials and Methods</em> for details on how protein families were defined). For example, the Reb family (indicated by a green box) is present in 24 fully sequenced Reb-harboring taxa but in none of the remaining genomes. The 4 Rebs of <em>Acidovorax avenae</em> subsp. Avenae ATCC 19860 did not fall into the Reb protein family because they are very divergent (see <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005231/-/DC1/FigureS1.pdf">Figure S1</a>). The other five unique protein families specific to Reb-harboring taxa (see main text) are shown with boxes corresponding to colors as defined in the legend to <a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">These few conserved proteins might be either involved in regulation of R-body assembly and function, or represent the toxin, and should be priority targets for future studies. It will also be interesting to test whether the Reb-harboring taxa that harbor none of these candidate partners are able to make R-bodies or display killing activity. Finally, it should be noted that none of these proteins belong to the Soj-ParA family or any annotated membrane-associated ATPase, weakening the previous hypothesis that these types of protein may represent the toxin responsible for killing (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Jeblick and Kusch 2005</a>).</p></div><div id="sec-10" class="subsection"><h3>A phage origin?</h3><p id="p-38">Jeblick and Kush emphasized the presence of phage-related genes on the <em>reb</em> carrying plasmid (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Jeblick and Kusch 2005</a>) and Preer (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Preer <em>et al.</em> 1974</a>) observed an association of phage-like particles with R-bodies, suggesting that R-bodies may be encoded by defective phage genes. Moreover, the evolutionary analysis of Reb families (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5</a>) and their genomic context (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6</a>) suggest horizontal gene transfer events, for which bacteriophages are known to be major contributors. However, we found no Reb homologs in genomic sequences from phages. We therefore sought to see if Reb homologs are part of integrated elements or prophages. We examined the 40 kbp on each side of the <em>reb</em> locus in the 25 Reb-harboring taxa for which complete genome sequences are available for the presence of prophages (integrated phages) or other phage-related elements. First, we searched using the PHAST database [<a href="http://phast.wishartlab.com">http://phast.wishartlab.com</a> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhou <em>et al.</em> 2011</a>)], which contains phage proteins that have been associated with a clear phage function. However, none of these regions were positive for prophage sequences. As a complementary analysis, we specifically searched a comprehensive local databank of 1130 bacteriophage sequences downloaded from GenBank (December 2011), which included 248 phages isolated from 33 proteobacterial genera. These regions did not display any specific similarity to phage elements. We also looked at whether <em>reb</em> loci are embedded in genomic islands by running searches on the IslandViewer server [<a href="http://www.pathogenomics.sfu.ca/islandviewer">http://www.pathogenomics.sfu.ca/islandviewer</a> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Langille and Brinkman 2009</a>)], which combines several prediction methods: (1) atypical dinucleotide content; or (2) codon usage; (3) identification of unique regions not present in closely related genomes; and (4) presence of genes that are functionally related to mobile elements. However, none of the analyzed genomes displayed identified potential genomic islands adjacent to or surrounding the <em>reb</em> locus. We also verified from the literature whether Reb homologs were present in any previously reported genomic region of potential exogenous origin. For example, <em>reb</em> genes did not fall into any of the two atypical regions identified in the genome of <em>Xanthomonas oryzae pv. oryzae</em> PXO99A (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Salzberg <em>et al.</em> 2008</a>), nor in the four atypical regions highlighted in the genome of <em>Xanthomonas campestris pv. campestris</em> str. ATCC 33913 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vorholter <em>et al.</em> 2003</a>), and were not included in any of the prophage islands identified in the genome of the <em>Pseudomonas aeruginosa</em> hypervirulent LESB58I strain (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Winstanley <em>et al.</em> 2009</a>). Finally, among 1062 plasmid sequences available from <em>Proteobacteria</em>, we detected <em>reb</em> genes only on two plasmids other than the <em>C. taeniospiralis</em> plasmid pKAP298: the <em>Azospirillum brasilense</em> Sp245 plasmid AZOBR_p4 and the <em>Ruegeria pomeroyi</em> DSS-3 megaplasmid.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-39">Despite having been continuously and intensively studied from the 1930s through the 1980s, recent data on R-bodies have been scarce. With a whole array of novel technology, studies on the diversity and role of these puzzling bacterial structures can now be fully tackled. Our exhaustive analysis shows that Reb homologs are widely present in Proteobacterial genomes spanning the diversity of this major bacterial phylum, indicating that they are much more widespread than previously known. In the perspective of obtaining experimental data, our analysis remains for the time being largely descriptive, but nevertheless provides a number of interesting hints for discussion and future work.</p><p id="p-40">Sequence analysis suggests structural and functional conservation, indicating that Reb homologs are likely responsible for the production of functional R-bodies in all the taxa where we found them, although this needs to be verified experimentally. Moreover, the presence of Reb homologs in bacteria where R-bodies have been previously observed is already a good hint. The occurrence of R-bodies in a wide range of bacteria harboring Reb homologs should be tested, with priority given to those that have medical, agricultural, and ecological implications. Our data will also help direct mutational studies to characterize the system further through structural and functional analysis of <em>reb</em> genes from Caedibacter but also other taxa, including those that harbor multiple <em>reb</em> genes and those that have only one copy. Also, it will be important to verify the involvement in R-body production, assembly, regulation, and killing of the likely partners that we have identified by genome context and whole genome content analysis. Because none of the Reb-harboring genomes identified in this study possess homologs of the genes carried by the <em>C. taeniospiralis</em> plasmid, it is possible that the killing toxin is encoded in the <em>C. taeniospiralis</em> genome and is perhaps one of the candidate genes that we have identified. The completion of this genome will therefore be very important. Another possibility is that the R-bodies are delivery systems for species-specific toxins.</p><p id="p-41">Our analysis of taxonomic distribution shows that Reb homologs are present in taxa displaying very different lifestyles, suggesting that the role of R-bodies in nature could be quite diverse. Moreover, we show that intact <em>reb</em> loci have been spread among Proteobacterial taxa via horizontal transfer, indicating that an advantage exists in acquiring and keeping R-bodies. However, we found no clear signs of a phage origin for the <em>reb</em> loci. It remains possible that this is due to an undersampling of phages from the Reb-harboring taxa or that these horizontal transfers correspond to events old enough to have allowed sequences to adapt to the new genome, or that all traces of the transfer vectors have been erased from the genome after transfer.</p><p id="p-42">Some of the Reb-harboring taxa have very important ecological, agricultural, and medical relevance. In addition, by observing the pattern of presence/absence of <em>reb</em> genes in closely related strains, we found intriguing links between virulence and presence of <em>reb</em> genes that will surely be worthy of further investigation. R-bodies may be involved in mediating interactions of these bacteria with eukaryotic cells, perhaps through the triggering of unrolling when ingested in the vacuole, similarly to what observed in the Caedibacter/Paramecium interaction.</p><p id="p-43">We found Reb homologs in many free-living bacteria. R-bodies in these bacteria may be used as a defense mechanism against grazing by eukaryotes. Bacteria have in fact developed various strategies against protozoan predation in nature (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Matz and Kjelleberg 2005</a>). Many examples have been reported of cytotoxicity responses against grazing by eukaryotes in different bacteria such as <em>Pseudomonas</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Matz <em>et al.</em> 2004</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Weitere <em>et al.</em> 2005</a>) and <em>Vibrio</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Erken <em>et al.</em> 2011</a>). Also, it is tempting to speculate that some of these free-living, Reb-harboring taxa can establish transient “killing” symbioses with ciliates or other protists, as seems to be indicated by the fact that some Reb-harboring taxa have indeed been isolated from aquatic microbial eukaryotes. It will be interesting to perform feeding experiments to test their killing potential in Paramecium and also other protistan taxa, such as algae. Indeed, we confirmed the presence of <em>reb</em> genes in <em>Kordia algicida</em>, a planktonic bacterium recently highlighted as a killer of diatoms by a yet unclear mechanism involving an unidentified protease (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Paul and Pohnert 2011</a>). Our study suggests that delivery of this protease could be performed via R-bodies, which would therefore be important players in the regulation of algal blooms. Similarly, it would be interesting to verify whether the presence of Reb homologs in the powerful coral pathogen <em>Vibrio coralliilyticus</em> is linked to R-body production and if these are somehow involved in delivery of the killing toxin. The killing factor produced by <em>K. algicida</em> is triggered independently of the presence of the diatom target but rather likely depends on a quorum sensing mechanism when the population size reaches a certain density (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Paul and Pohnert 2011</a>). Similarly, the triggering of R-body production in a fraction of the Caedibacter endosymbiont population in Paramecium, a phenomenon that is not yet understood, may be regulated by a quorum sensing mechanism.</p><p id="p-44">In addition, our study suggests that R-bodies may be involved in the interaction of Proteobacteria with several multicellular organisms, such as plants and animals. The recently reported involvement of <em>reb</em> genes in the regulation of a rhizobial symbiosis (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Akiba <em>et al.</em> 2010</a>) and the presence of Reb homologs in a number of Proteobacteria known to interact with plant roots is intriguing, and it is not excluded that R-bodies may help the bacterium to move through plant tissues, via delivery of specific lytic compounds. Indeed, we found Reb homologs in a number of bacterial strains known to be able to penetrate the xylem of plants. A similar mechanism may be used to move through tissues by some Proteobacteria that interact with animals, such as <em>Vibrio fischeri</em> with its squid host. Finally, the presence of <em>reb</em> genes in important pathogens of eukaryotes, including humans, some of which are responsible for emerging and poorly characterized infections, should prompt the study of their potential involvement in the infection process, perhaps by helping tissue invasion.</p><p id="p-45">If our predictions are verified, bacteria may represent a largely overlooked role in the regulation of microbial eukaryotic abundance and distribution, in addition to the much more studied impact of viruses. This regulation may be performed at different levels, by direct killing of eukaryotic grazers, but also by providing mechanisms used for defense among eukaryotes, as is the example of the Paramecium/Caedibacter symbiosis. Interestingly, it was recently reported that the thricocysts of eukaryotic algae belonging to the Cryptomonads, ejectile organelles that are probably used with a defensive role against predation, are composed of four proteins that share similarity with Reb proteins (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yamagishi <em>et al.</em> 2012</a>). The authors proposed that these proteins were acquired horizontally from Proteobacteria. R-body−harboring bacteria could therefore play a larger role in the origin and spread of defense mechanisms in eukaryotic microorganisms. Finally, elucidation of the mechanism of rolling/unrolling/toxin delivery of R-bodies will surely open the way to interesting biotechnological applications.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">We would like to thank Céline Brochier-Armanet for kindly sharing 16S rRNA trees of Proteobacteria and for useful comments on the manuscript, along with Patrick Forterre and Eduardo Rocha for interesting discussion. We also wish to thank Mart Krupovic for advice on secondary structure analysis. K.R. is the recipient of a Pasteur Ph.D. International grant and a Paul Zuccaire fellow and would like to thank John R. and Louise B. Preer for early training in Paramecium and killer endosymbiont biology. After completion of this work, we became aware that an analysis partially similar to ours was independently carried out and discussed in the Ph.D. thesis of Martina Schrallhammer (2010, unpublished) under the supervision of Giulio Petroni, whom we thank for bringing it to our attention. We also thank two anonymous referees for very useful comments that helped improve the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005231/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005231/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-47"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-12-03"><span class="received-label">Received </span>December 3, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-01-09"><span class="accepted-label">Accepted </span>January 9, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Raymann <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.1" data-doi="10.1128/AEM.00238-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akiba</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aono</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyazaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oyaizu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">phrR-like gene praR of <em>Azorhizobium caulinodans</em> ORS571 is essential for symbiosis with Sesbania rostrata and is involved in expression of reb genes</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">3475</span>–<span class="cit-lpage">3485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00238-10%26rft_id%253Dinfo%253Apmid%252F20382809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8xMS8zNDc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.2" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.505.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beale</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preer</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2004</span> <span class="cit-source">Paramecium: Genetics and Epigenetics</span>, <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.4" data-doi="10.1128/AEM.68.12.6043-6050.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beier</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweikert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gortz</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The genus Caedibacter comprises endosymbionts of Paramecium spp. related to the Rickettsiales (Alphaproteobacteria) and to <em>Francisella tularensis</em> (Gammaproteobacteria)</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">6043</span>–<span class="cit-lpage">6050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DBeier%26rft.auinit1%253DC.%2BL.%26rft.volume%253D68%26rft.issue%253D12%26rft.spage%253D6043%26rft.epage%253D6050%26rft.atitle%253DThe%2BGenus%2BCaedibacter%2BComprises%2BEndosymbionts%2Bof%2BParamecium%2Bspp.%2BRelated%2Bto%2Bthe%2BRickettsiales%2B%2528Alphaproteobacteria%2529%2Band%2Bto%2BFrancisella%2Btularensis%2B%2528Gammaproteobacteria%2529%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.68.12.6043-6050.2002%26rft_id%253Dinfo%253Apmid%252F12450827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI2OC8xMi82MDQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.5" data-doi="10.1016/S0966-842X(98)01425-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Unexplored reservoirs of pathogenic bacteria: protozoa and biofilms</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DBrown%26rft.auinit1%253DM.%2BR.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D46%26rft.epage%253D50%26rft.atitle%253DUnexplored%2Breservoirs%2Bof%2Bpathogenic%2Bbacteria%253A%2Bprotozoa%2Band%2Bbiofilms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0966-842X%252898%252901425-5%26rft_id%253Dinfo%253Apmid%252F10068997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0966-842X(98)01425-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10068997&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080898200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchan</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobley</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nugent</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Protein annotation and modelling servers at University College London</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W563–568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D38%26rft.spage%253DW563%2B568%26rft.atitle%253DPROTEIN%2BANNOTATION%2BAND%2BMODELLING%2BSERVERS%2BAT%2BUNIVERSITY%2BCOLLEGE%2BLONDON%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.7" data-doi="10.1186/1471-2148-10-210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Criscuolo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gribaldo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BMGE (Block Mapping and Gathering with Entropy): a new software for selection of phylogenetic informative regions from multiple sequence alignments</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DCriscuolo%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.spage%253D210%26rft.epage%253D210%26rft.atitle%253DBMGE%2B%2528Block%2BMapping%2Band%2BGathering%2Bwith%2BEntropy%2529%253A%2Ba%2Bnew%2Bsoftware%2Bfor%2Bselection%2Bof%2Bphylogenetic%2Binformative%2Bregions%2Bfrom%2Bmultiple%2Bsequence%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-10-210%26rft_id%253Dinfo%253Apmid%252F20626897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-10-210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20626897&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.8" data-doi="10.1126/science.3008334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dilts</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">A mutation in the R body-coding sequence destroys expression of the killer trait in P. tetraurelia</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">232</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDilts%26rft.auinit1%253DJ.%26rft.volume%253D232%26rft.issue%253D4750%26rft.spage%253D641%26rft.epage%253D643%26rft.atitle%253DA%2Bmutation%2Bin%2Bthe%2BR%2Bbody-coding%2Bsequence%2Bdestroys%2Bexpression%2Bof%2Bthe%2Bkiller%2Btrait%2Bin%2BP.%2Btetraurelia%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.3008334%26rft_id%253Dinfo%253Apmid%252F3008334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyMzIvNDc1MC82NDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.9" data-doi="10.1186/1471-2105-5-113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DMUSCLE%253A%2Ba%2Bmultiple%2Bsequence%2Balignment%2Bmethod%2Bwith%2Breduced%2Btime%2Band%2Bspace%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-113%26rft_id%253Dinfo%253Apmid%252F15318951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15318951&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.10" data-doi="10.1111/j.1574-6941.2011.01067.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erken</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weitere</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjelleberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDougald</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">In situ grazing resistance of <em>Vibrio cholerae</em> in the marine environment</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol. Ecol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">504</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6941.2011.01067.x%26rft_id%253Dinfo%253Apmid%252F21314704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2VjIjtzOjU6InJlc2lkIjtzOjg6Ijc2LzMvNTA0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.11" data-doi="10.1007/BF00572696"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Espuny</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercade</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manresa</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">R-bodies in <em>Pseudomonas aeruginosa</em> strain 44T1</span>. <abbr class="cit-jnl-abbrev">Antonie van Leeuwenhoek</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.stitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.aulast%253DEspuny%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D60%26rft.issue%253D2%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DR-bodies%2Bin%2BPseudomonas%2Baeruginosa%2Bstrain%2B44T1.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00572696%26rft_id%253Dinfo%253Apmid%252F1804030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00572696&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1804030&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.12" data-doi="10.1007/BF00393857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Favinger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stadtwald</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gest</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Rhodospirillum centenum, sp. nov., a thermotolerant cyst-forming anoxygenic photosynthetic bacterium</span>. <abbr class="cit-jnl-abbrev">Antonie van Leeuwenhoek</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.stitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.aulast%253DFavinger%26rft.auinit1%253DJ.%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D291%26rft.epage%253D296%26rft.atitle%253DRhodospirillum%2Bcentenum%252C%2Bsp.%2Bnov.%252C%2Ba%2Bthermotolerant%2Bcyst-forming%2Banoxygenic%2Bphotosynthetic%2Bbacterium.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00393857%26rft_id%253Dinfo%253Apmid%252F2757370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00393857&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2757370&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U396000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.13" data-doi="10.1099/00221287-132-10-2801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fusté</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon-Pujol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marques</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guinea</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Congregrado</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Isolation of a new free-living bacterium containing R-bodies</span>. <abbr class="cit-jnl-abbrev">J. Gen. Microbiol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">2801</span>–<span class="cit-lpage">2805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DFUSTE%26rft.auinit1%253DM.%2BC.%26rft.volume%253D132%26rft.issue%253D10%26rft.spage%253D2801%26rft.epage%253D2805%26rft.atitle%253DIsolation%2Bof%2Ba%2BNew%2BFree-living%2BBacterium%2BContaining%2BR-bodies%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-132-10-2801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-132-10-2801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.14" data-doi="10.1016/j.tim.2009.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gast</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caron</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ecological strategies of protists and their symbiotic relationships with prokaryotic microbes</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DGast%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D563%26rft.epage%253D569%26rft.atitle%253DEcological%2Bstrategies%2Bof%2Bprotists%2Band%2Btheir%2Bsymbiotic%2Brelationships%2Bwith%2Bprokaryotic%2Bmicrobes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2009.09.001%26rft_id%253Dinfo%253Apmid%252F19828317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2009.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19828317&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.15" data-doi="10.1038/241127a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Transplantation of killer endosymbionts in paramecium</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">241</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGibson%26rft.auinit1%253DI.%26rft.volume%253D241%26rft.issue%253D5385%26rft.spage%253D127%26rft.epage%253D129%26rft.atitle%253DTransplantation%2Bof%2Bkiller%2Bendosymbionts%2Bin%2Bparamecium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F241127a0%26rft_id%253Dinfo%253Apmid%252F4633042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/241127a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4633042&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.16" data-doi="10.1099/mic.0.26524-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez-Romero</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas-Elio</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Serrano</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Amat</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title"><em>Marinomonas mediterranea</em> is a lysogenic bacterium that synthesizes R-bodies</span>. <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">2679</span>–<span class="cit-lpage">2686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DHernandez-Romero%26rft.auinit1%253DD.%26rft.volume%253D149%26rft.issue%253D9%26rft.spage%253D2679%26rft.epage%253D2686%26rft.atitle%253DMarinomonas%2Bmediterranea%2Bis%2Ba%2Blysogenic%2Bbacterium%2Bthat%2Bsynthesizes%2BR-bodies%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.26524-0%26rft_id%253Dinfo%253Apmid%252F12949192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.26524-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12949192&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.17" data-doi="10.1128/jb.176.12.3559-3567.1994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heruth</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dilts</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Characterization of genetic determinants for R body synthesis and assembly in <em>Caedibacter taeniospiralis</em> 47 and 116</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">3559</span>–<span class="cit-lpage">3567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHeruth%26rft.auinit1%253DD.%2BP.%26rft.volume%253D176%26rft.issue%253D12%26rft.spage%253D3559%26rft.epage%253D3567%26rft.atitle%253DCharacterization%2Bof%2Bgenetic%2Bdeterminants%2Bfor%2BR%2Bbody%2Bsynthesis%2Band%2Bassembly%2Bin%2BCaedibacter%2Btaeniospiralis%2B47%2Band%2B116.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.12.3559-3567.1994%26rft_id%253Dinfo%253Apmid%252F8206833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3Ni8xMi8zNTU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.18" data-doi="10.1007/s00239-004-0002-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeblick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusch</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sequence, transcription activity, and evolutionary origin of the R-body coding plasmid pKAP298 from the intracellular parasitic bacterium <em>Caedibacter taeniospiralis</em></span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DJeblick%26rft.auinit1%253DJ.%26rft.volume%253D60%26rft.issue%253D2%26rft.spage%253D164%26rft.epage%253D173%26rft.atitle%253DSequence%252C%2Btranscription%2Bactivity%252C%2Band%2Bevolutionary%2Borigin%2Bof%2Bthe%2BR-body%2Bcoding%2Bplasmid%2BpKAP298%2Bfrom%2Bthe%2Bintracellular%2Bparasitic%2Bbacterium%2BCaedibacter%2Btaeniospiralis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-004-0002-2%26rft_id%253Dinfo%253Apmid%252F15785846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-004-0002-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15785846&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.19" data-doi="10.1186/1471-2148-4-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jobb</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">TREEFINDER: a powerful graphical analysis environment for molecular phylogenetics</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DJobb%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DTREEFINDER%253A%2Ba%2Bpowerful%2Bgraphical%2Banalysis%2Benvironment%2Bfor%2Bmolecular%2Bphylogenetics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-4-18%26rft_id%253Dinfo%253Apmid%252F15222900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-4-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15222900&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.20" data-doi="10.1002/jez.1401770311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurand</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudman</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preer</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1971</span> <span class="cit-article-title">Prelethal effects of killing action by stock 7 of <em>Paramecium aurelia</em></span>. <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bexperimental%2Bzoology%26rft.stitle%253DJ%2BExp%2BZool%26rft.aulast%253DJurand%26rft.auinit1%253DA.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D365%26rft.epage%253D387%26rft.atitle%253DPrelethal%2Beffects%2Bof%2Bkilling%2Baction%2Bby%2Bstock%2B7%2Bof%2BParamecium%2Baurelia.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1401770311%26rft_id%253Dinfo%253Apmid%252F4938048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1401770311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4938048&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.21" data-doi="10.1128/jb.168.1.40-48.1986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanabrocki</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Organization and expression of genetic determinants for synthesis and assembly of type 51 R bodies</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKanabrocki%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D40%26rft.epage%253D48%26rft.atitle%253DOrganization%2Band%2Bexpression%2Bof%2Bgenetic%2Bdeterminants%2Bfor%2Bsynthesis%2Band%2Bassembly%2Bof%2Btype%2B51%2BR%2Bbodies.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.168.1.40-48.1986%26rft_id%253Dinfo%253Apmid%252F3759909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiMTY4LzEvNDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.22" data-doi="10.1078/1434-4610-00082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kusch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czubatinski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Competitive advantages of Caedibacter-infected Paramecia</span>. <abbr class="cit-jnl-abbrev">Protist</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtist%26rft.volume%253D153%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1078%252F1434-4610-00082%26rft_id%253Dinfo%253Apmid%252F12022275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1078/1434-4610-00082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12022275&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.23" data-doi="10.1002/jobm.3630180708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lalucat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">“Spiral bodies”–intracytoplasmic membraneous structures in a hydrogen oxidizing bacterium</span>. <abbr class="cit-jnl-abbrev">Z. Allg. Mikrobiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZeitschrift%2Bf%25C3%25BCr%2Ballgemeine%2BMikrobiologie%26rft.stitle%253DZ%2BAllg%2BMikrobiol%26rft.aulast%253DLalucat%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D517%26rft.epage%253D521%26rft.atitle%253D%2526quot%253BSpiral%2Bbodies%2526quot%253B--intracytoplasmic%2Bmembraneous%2Bstructures%2Bin%2Ba%2Bhydrogen%2Boxidizing%2Bbacterium.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjobm.3630180708%26rft_id%253Dinfo%253Apmid%252F735162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jobm.3630180708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=735162&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.24" data-doi="10.1093/bioinformatics/btp030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langille</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkman</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">IslandViewer: an integrated interface for computational identification and visualization of genomic islands</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">664</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp030%26rft_id%253Dinfo%253Apmid%252F19151094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNS81LzY2NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy81MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.25" data-doi="10.1093/bioinformatics/btp368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lartillot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepage</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanquart</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2286</span>–<span class="cit-lpage">2288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp368%26rft_id%253Dinfo%253Apmid%252F19535536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTcvMjI4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy81MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.26" data-doi="10.1016/j.tim.2005.05.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjelleberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Off the hook–how bacteria survive protozoan grazing</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DMatz%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253D302%26rft.epage%253D307%26rft.atitle%253DOff%2Bthe%2Bhook--how%2Bbacteria%2Bsurvive%2Bprotozoan%2Bgrazing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2005.05.009%26rft_id%253Dinfo%253Apmid%252F15935676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2005.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15935676&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230778200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.27" data-doi="10.1111/j.1462-2920.2004.00556.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergfeld</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjelleberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microcolonies, quorum sensing and cytotoxicity determine the survival of Pseudomonas aeruginosa biofilms exposed to protozoan grazing</span>. <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DMatz%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D218%26rft.epage%253D226%26rft.atitle%253DMicrocolonies%252C%2Bquorum%2Bsensing%2Band%2Bcytotoxicity%2Bdetermine%2Bthe%2Bsurvival%2Bof%2BPseudomonas%2Baeruginosa%2Bbiofilms%2Bexposed%2Bto%2Bprotozoan%2Bgrazing.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2004.00556.x%26rft_id%253Dinfo%253Apmid%252F14871206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2004.00556.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14871206&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188820700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.28" data-doi="10.1371/journal.pone.0021032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pohnert</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interactions of the algicidal bacterium <em>Kordia algicida</em> with diatoms: regulated protease excretion for specific algal lysis</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e21032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPaul%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De21032%26rft.epage%253De21032%26rft.atitle%253DInteractions%2Bof%2Bthe%2Balgicidal%2Bbacterium%2BKordia%2Balgicida%2Bwith%2Bdiatoms%253A%2Bregulated%2Bprotease%2Bexcretion%2Bfor%2Bspecific%2Balgal%2Blysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0021032%26rft_id%253Dinfo%253Apmid%252F21695044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0021032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21695044&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.29" data-doi="10.1093/nar/21.22.5264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">MUST, a computer package of Management Utilities for Sequences and Trees</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5264</span>–<span class="cit-lpage">5272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPhilippe%26rft.auinit1%253DH.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5264%26rft.epage%253D5272%26rft.atitle%253DMUST%252C%2Ba%2Bcomputer%2Bpackage%2Bof%2BManagement%2BUtilities%2Bfor%2BSequences%2Band%2BTrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F21.22.5264%26rft_id%253Dinfo%253Apmid%252F8255784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8yMi81MjY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalucat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">R-body-producing bacteria</span>. <abbr class="cit-jnl-abbrev">Microbiol. Rev.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DPond%26rft.auinit1%253DF%2BR%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D67%26rft.atitle%253DR-body-producing%2Bbacteria.%26rft_id%253Dinfo%253Apmid%252F2651865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo3OiI1My8xLzI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-other" id="cit-3.3.505.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preer</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1975</span> <span class="cit-article-title">The hereditary symbionts of <em>Paramecium aurelia</em></span>. <span class="cit-source">Symp. Soc. Exp. Biol.</span> <span class="cit-vol">29</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSymp.%2BSoc.%2BExp.%2BBiol.%26rft.volume%253D29%26rft.spage%253D125%26rft_id%253Dinfo%253Apmid%252F785658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=785658&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preer</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2006</span> <span class="cit-article-title">Sonneborn and the Cytoplasm</span>. <abbr class="cit-jnl-abbrev">Genet. Soc. Am.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1373</span>–<span class="cit-lpage">1377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSoc.%2BAm.%26rft.volume%253D172%26rft.spage%253D1373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.33" data-doi="10.1016/0014-4827(53)90234-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preer</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Cytological observations on the cytoplasmic factor “kappa” in <em>Paramecium aurelia</em></span>. <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">478</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bcell%2Bresearch%26rft.stitle%253DExp%2BCell%2BRes%26rft.aulast%253DPreer%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D478%26rft.epage%253D491%26rft.atitle%253DCytological%2Bobservations%2Bon%2Bthe%2Bcytoplasmic%2Bfactor%2B%2526quot%253Bkappa%2526quot%253B%2Bin%2BParamecium%2Baurelia.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-4827%252853%252990234-1%26rft_id%253Dinfo%253Apmid%252F13117017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-4827(53)90234-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13117017&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preer</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preer</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurand</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Kappa and other endosymbionts in <em>Paramecium aurelia</em></span>. <abbr class="cit-jnl-abbrev">Bacteriol. Rev.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DPreer%26rft.auinit1%253DJ%2BR%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D113%26rft.epage%253D163%26rft.atitle%253DKappa%2Band%2Bother%2Bendosymbionts%2Bin%2BParamecium%2Baurelia.%26rft_id%253Dinfo%253Apmid%252F4599970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo0OiJtbWJyIjtzOjU6InJlc2lkIjtzOjg6IjM4LzIvMTEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.35" data-doi="10.1073/pnas.80.1.250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burbach</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Cloning and expression of DNA sequences associated with the killer trait of <em>Paramecium tetraurelia</em> stock 47</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DQuackenbush%26rft.auinit1%253DR.%2BL.%26rft.volume%253D80%26rft.issue%253D1%26rft.spage%253D250%26rft.epage%253D254%26rft.atitle%253DCloning%2Band%2Bexpression%2Bof%2BDNA%2Bsequences%2Bassociated%2Bwith%2Bthe%2Bkiller%2Btrait%2Bof%2BParamecium%2Btetraurelia%2Bstock%2B47.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.1.250%26rft_id%253Dinfo%253Apmid%252F6571998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4MC8xLzI1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy81MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-other" id="cit-3.3.505.36"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>, <span class="cit-pub-date">2011</span> R: <em>A language and environment for statistical computing</em>. R Foundation for Statistical Computing, Vienna. Available at: <a href="http://www.R-project.org">http://www.R-project.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.37" data-doi="10.1186/1471-2164-9-204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowicz</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome sequence and rapid evolution of the rice pathogen Xanthomonas oryzae pv. oryzae PXO99A</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSalzberg%26rft.auinit1%253DS.%2BL.%26rft.volume%253D9%26rft.spage%253D204%26rft.epage%253D204%26rft.atitle%253DGenome%2Bsequence%2Band%2Brapid%2Bevolution%2Bof%2Bthe%2Brice%2Bpathogen%2BXanthomonas%2Boryzae%2Bpv.%2Boryzae%2BPXO99A.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-204%26rft_id%253Dinfo%253Apmid%252F18452608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18452608&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-other" id="cit-3.3.505.38" data-doi="10.1007/3-540-33774-1_14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Amat</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">R-bodies</span>. <span class="cit-source">Microbiol. Monogr.</span> <span class="cit-vol">1</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BMonogr.%26rft.volume%253D1%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1007%252F3-540-33774-1_14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/3-540-33774-1_14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.39" data-doi="10.1016/j.ejop.2012.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrallhammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galati</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altenbuchner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweikert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gortz</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tracing the role of R-bodies in the killer trait: Absence of toxicity of R-body producing recombinant <em>E. coli</em> on paramecia</span>. <abbr class="cit-jnl-abbrev">Eur. J. Protistol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEur%2BJ%2BProtistol%26rft.aulast%253DSchrallhammer%26rft.auinit1%253DM.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D290%26rft.epage%253D296%26rft.atitle%253DTracing%2Bthe%2Brole%2Bof%2BR-bodies%2Bin%2Bthe%2Bkiller%2Btrait%253A%2Babsence%2Bof%2Btoxicity%2Bof%2BR-body%2Bproducing%2Brecombinant%2BE.%2Bcoli%2Bon%2Bparamecia.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ejop.2012.01.008%26rft_id%253Dinfo%253Apmid%252F22356923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ejop.2012.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22356923&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonneborn</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">Mating types in <em>P. aurelia</em>: diverse conditions for mating in different stocks; occurrence, number and interrelations of the types</span>. <abbr class="cit-jnl-abbrev">Proc. Am. Philos. Soc.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BAm.%2BPhilos.%2BSoc.%26rft.volume%253D79%26rft.spage%253D411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.41" data-doi="10.1073/pnas.29.11.338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonneborn</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">Gene and cytoplasm: II. The bearing of the determination and inheritance of characters in <em>Paramecium aurelia</em> on the problems of cytoplasmic inheritance, pneumococcus transformations, mutations and development</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSonneborn%26rft.auinit1%253DT.%2BM.%26rft.volume%253D29%26rft.issue%253D11%26rft.spage%253D338%26rft.epage%253D343%26rft.atitle%253DGene%2Band%2BCytoplasm%253A%2BII.%2BThe%2BBearing%2Bof%2Bthe%2BDetermination%2Band%2BInheritance%2Bof%2BCharacters%2Bin%2BParamecium%2BAurelia%2Bon%2Bthe%2BProblems%2Bof%2BCytoplasmic%2BInheritance%252C%2BPneumococcus%2BTransformations%252C%2BMutations%2Band%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.29.11.338%26rft_id%253Dinfo%253Apmid%252F16588623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIyOS8xMS8zMzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.42" data-doi="10.1016/j.jbiotec.2003.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vorholter</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thias</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparison of two Xanthomonas campestris pathovar campestris genomes revealed differences in their gene composition</span>. <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiotechnology%26rft.stitle%253DJournal%2Bof%2BBiotechnology%26rft.aulast%253DVorh%25C3%25B6lter%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D106%26rft.issue%253D2-3%26rft.spage%253D193%26rft.epage%253D202%26rft.atitle%253DComparison%2Bof%2Btwo%2BXanthomonas%2Bcampestris%2Bpathovar%2Bcampestris%2Bgenomes%2Brevealed%2Bdifferences%2Bin%2Btheir%2Bgene%2Bcomposition.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiotec.2003.08.011%26rft_id%253Dinfo%253Apmid%252F14651861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiotec.2003.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14651861&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187350200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.43" data-doi="10.1111/j.1462-2920.2005.00851.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weitere</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergfeld</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjelleberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Grazing resistance of <em>Pseudomonas aeruginosa</em> biofilms depends on type of protective mechanism, developmental stage and protozoan feeding mode</span>. <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1593</span>–<span class="cit-lpage">1601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DWeitere%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253D1593%26rft.epage%253D1601%26rft.atitle%253DGrazing%2Bresistance%2Bof%2BPseudomonas%2Baeruginosa%2Bbiofilms%2Bdepends%2Bon%2Btype%2Bof%2Bprotective%2Bmechanism%252C%2Bdevelopmental%2Bstage%2Band%2Bprotozoan%2Bfeeding%2Bmode.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2005.00851.x%26rft_id%253Dinfo%253Apmid%252F16156732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2005.00851.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16156732&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231737100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-other" id="cit-3.3.505.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horne</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span>  <span class="cit-article-title">The ultrastructure of Pseudomonas avenae II. Intracellular refractile (R-body) structure</span>. <span class="cit-source">Micron</span> <span class="cit-vol">14</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicron%26rft.volume%253D14%26rft.spage%253D329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.45" data-doi="10.1101/gr.086082.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winstanley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langille</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fothergill</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kukavica-Ibrulj</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paradis-Bleau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Newly introduced genomic prophage islands are critical determinants of in vivo competitiveness in the Liverpool epidemic strain of <em>Pseudomonas aeruginosa</em></span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086082.108%26rft_id%253Dinfo%253Apmid%252F19047519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE5LzEvMTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.46" data-doi="10.1007/s00239-012-9495-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamagishi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawai</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Trichocyst ribbons of a cryptomonads are constituted of homologs of R-body proteins produced by the intracellular parasitic bacterium of Paramecium</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">74</span>:<span class="cit-fpage">147</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYamagishi%26rft.auinit1%253DT.%26rft.volume%253D74%26rft.issue%253D3-4%26rft.spage%253D147%26rft.epage%253D157%26rft.atitle%253DTrichocyst%2Bribbons%2Bof%2Ba%2Bcryptomonads%2Bare%2Bconstituted%2Bof%2Bhomologs%2Bof%2BR-body%2Bproteins%2Bproduced%2Bby%2Bthe%2Bintracellular%2Bparasitic%2Bbacterium%2Bof%2BParamecium.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-012-9495-2%26rft_id%253Dinfo%253Apmid%252F22447322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-012-9495-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22447322&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.505.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wishart</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PHAST: a fast phage search tool</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">W347–352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D39%26rft.spage%253DW347%2B352%26rft.atitle%253DPHAST%253A%2BA%2BFAST%2BPHAGE%2BSEARCH%2BTOOL%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/3/505.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/493" title="Genetic Interactions of Arabidopsis thaliana Damaged DNA Binding Protein 1B (DDB1B) With DDB1A, DET1, and COP1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/517" title="PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/3"><span>Volume 3 Issue 3, March 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/3.cover-source.jpg?itok=w5x3d6rp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/505" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Afqw0rhizVUkXIKhhnAoaVvxsUbNp6yphFw9y5eBaNs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/3/505" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/3/505" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/3/505" class="active" data-icon-position="" data-hide-link-title="0">A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459427" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XZIbloJRqEU4raTKnXBu0lKDhqs8DN_5w9P6dYM0RxI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/505.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/3/505" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-CBGLz1HmUS51vpcNh74qNCcZ1EiID231gk4sd0SaFZY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/3/505" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/505.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459427" id="node459427--2" data-pisa="ggg;3/3/505" data-pisa-master="ggg;3/3/505" data-apath="/ggg/3/3/505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kasie Raymann</span>, <span class="highwire-citation-author" data-delta="1">Louis-Marie Bobay</span>, <span class="highwire-citation-author" data-delta="2">Thomas G. Doak</span>, <span class="highwire-citation-author" data-delta="3">Michael Lynch</span> and <span class="highwire-citation-author" data-delta="4">Simonetta Gribaldo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">505-516;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005231 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459427--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kasie Raymann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kasie%2BRaymann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raymann%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKasie%2BRaymann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Louis-Marie Bobay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut Pasteur, Microbial Evolutionary Genomics, Departement Genomes et Genetique, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre National de la Recherche Scientifique, Unité Mixte de Recherche 3525, Paris, F-75015 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis-Marie%2BBobay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bobay%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis-Marie%2BBobay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas G. Doak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Indiana University, Department of Biology, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BG.%2BDoak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doak%20TG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BG.%2BDoak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Indiana University, Department of Biology, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Simonetta Gribaldo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simonetta%2BGribaldo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gribaldo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimonetta%2BGribaldo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:simonetta.gribaldo@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">simonetta.gribaldo@pasteur.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/3/505" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/3/505&amp;title=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/3/505&amp;title=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/3/505&amp;title=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/3/505&amp;text=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/3/505&amp;title=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/3/505&amp;t=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/3/505&amp;title=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/3/505&amp;title=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459427/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459427" id="node459427--3" data-pisa="ggg;3/3/505" data-pisa-master="ggg;3/3/505" data-apath="/ggg/3/3/505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kasie Raymann</span>, <span class="highwire-citation-author" data-delta="1">Louis-Marie Bobay</span>, <span class="highwire-citation-author" data-delta="2">Thomas G. Doak</span>, <span class="highwire-citation-author" data-delta="3">Michael Lynch</span> and <span class="highwire-citation-author" data-delta="4">Simonetta Gribaldo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">505-516;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005231 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459427--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kasie Raymann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kasie%2BRaymann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raymann%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKasie%2BRaymann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Louis-Marie Bobay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Université Pierre et Marie Curie, Cellule Pasteur UPMC, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut Pasteur, Microbial Evolutionary Genomics, Departement Genomes et Genetique, Paris, 75724 Cedex 15, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre National de la Recherche Scientifique, Unité Mixte de Recherche 3525, Paris, F-75015 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis-Marie%2BBobay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bobay%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis-Marie%2BBobay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Thomas G. Doak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Indiana University, Department of Biology, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BG.%2BDoak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doak%20TG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BG.%2BDoak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Indiana University, Department of Biology, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Simonetta Gribaldo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Pasteur, Unité Biologie Moleculare du Gene chez les Extremophiles, Departement de Microbiologie, Paris, 75724 Cedex 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simonetta%2BGribaldo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gribaldo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimonetta%2BGribaldo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:simonetta.gribaldo@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">simonetta.gribaldo@pasteur.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=505&amp;rft.epage=516&amp;rft.atitle=A%20Genomic%20Survey%20of%20Reb%20Homologs%20Suggests%20Widespread%20Occurrence%20of%20R-Bodies%20in%20Proteobacteria&amp;rft.volume=3&amp;rft.issue=3&amp;rft.date=2013-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Raymann%2C+Kasie&amp;rft.au=Bobay%2C+Louis-Marie&amp;rft.au=Doak%2C+Thomas+G.&amp;rft.au=Lynch%2C+Michael&amp;rft.au=Gribaldo%2C+Simonetta"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459427/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459427/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459427/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459427/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459427/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459427/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459427/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459427/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459427/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459427/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459427/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459427/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/3/505" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/3/505.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/3/505.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/3/505.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/3/505.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
