<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/3/481" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/3/481.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/3/481.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/3/481.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/3/481/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/3/481.atom" />
<meta name="HW.pisa" content="ggg;3/3/481" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment" />
<meta name="DC.Identifier" content="10.1534/g3.112.004911" />
<meta name="DC.Date" content="2013-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Lorenz. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Allocating resources between population size and replication affects both genetic gain through phenotypic selection and quantitative trait loci detection power and effect estimation accuracy for marker-assisted selection (MAS). It is well known that because alleles are replicated across individuals in quantitative trait loci mapping and MAS, more resources should be allocated to increasing population size compared with phenotypic selection. Genomic selection is a form of MAS using all marker information simultaneously to predict individual genetic values for complex traits and has widely been found superior to MAS. No studies have explicitly investigated how resource allocation decisions affect success of genomic selection. My objective was to study the effect of resource allocation on response to MAS and genomic selection in a single biparental population of doubled haploid lines by using computer simulation. Simulation results were compared with previously derived formulas for the calculation of prediction accuracy under different levels of heritability and population size. Response of prediction accuracy to resource allocation strategies differed between genomic selection models (ridge regression best linear unbiased prediction [RR-BLUP], BayesC π ) and multiple linear regression using ordinary least-squares estimation (OLS), leading to different optimal resource allocation choices between OLS and RR-BLUP. For OLS, it was always advantageous to maximize population size at the expense of replication, but a high degree of flexibility was observed for RR-BLUP. Prediction accuracy of doubled haploid lines included in the training set was much greater than of those excluded from the training set, so there was little benefit to phenotyping only a subset of the lines genotyped. Finally, observed prediction accuracies in the simulation compared well to calculated prediction accuracies, indicating these theoretical formulas are useful for making resource allocation decisions." />
<meta name="DC.Contributor" content="Aaron J. Lorenz" />
<meta name="article:published_time" content="2013-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Allocating resources between population size and replication affects both genetic gain through phenotypic selection and quantitative trait loci detection power and effect estimation accuracy for marker-assisted selection (MAS). It is well known that because alleles are replicated across individuals in quantitative trait loci mapping and MAS, more resources should be allocated to increasing population size compared with phenotypic selection. Genomic selection is a form of MAS using all marker information simultaneously to predict individual genetic values for complex traits and has widely been found superior to MAS. No studies have explicitly investigated how resource allocation decisions affect success of genomic selection. My objective was to study the effect of resource allocation on response to MAS and genomic selection in a single biparental population of doubled haploid lines by using computer simulation. Simulation results were compared with previously derived formulas for the calculation of prediction accuracy under different levels of heritability and population size. Response of prediction accuracy to resource allocation strategies differed between genomic selection models (ridge regression best linear unbiased prediction [RR-BLUP], BayesC&lt;i&gt;π&lt;/i&gt;) and multiple linear regression using ordinary least-squares estimation (OLS), leading to different optimal resource allocation choices between OLS and RR-BLUP. For OLS, it was always advantageous to maximize population size at the expense of replication, but a high degree of flexibility was observed for RR-BLUP. Prediction accuracy of doubled haploid lines included in the training set was much greater than of those excluded from the training set, so there was little benefit to phenotyping only a subset of the lines genotyped. Finally, observed prediction accuracies in the simulation compared well to calculated prediction accuracies, indicating these theoretical formulas are useful for making resource allocation decisions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/03/01" />
<meta name="citation_mjid" content="ggg;3/3/481" />
<meta name="citation_id" content="3/3/481" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/3/481" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/3/481.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/3/481.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/3/481.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004911" />
<meta name="citation_pmid" content="23450123" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="481" />
<meta name="citation_lastpage" content="491" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aaron J. Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583" />
<meta name="citation_author_email" content="alorenz2@unl.edu" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_title=Proceedings of the 49th Annual Corn and Sorghum Research Conference;citation_title=The power and deceit of QTL experiments: lessons from comparative QTL studies;citation_author=W. D. Beavis;citation_pages=250-265;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Molecular Markers and Selection for Complex Traits in Plants: Learning from the Last 20 Years;citation_pages=1649-1664;citation_volume=48;citation_year=2008;citation_issue=5;citation_doi=10.2135/cropsci2008.03.0131" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_author=R. Bernardo;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for genome-wide selection for quantitative traits in maize;citation_pages=1090;citation_volume=47;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Accuracy of predicting the genetic risk of disease using a genome-wide approach;citation_pages=10;citation_volume=3;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=A. Gimelfarb;citation_author=R. Lande;citation_title=Simulation of marker assisted selection in hybrid populations.;citation_pages=39-47;citation_volume=63;citation_year=1994;citation_issue=1;citation_pmid=8206366" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_author=T. H. E. Meuwissen;citation_title=Using the genomic relationship matrix to predict the accuracy of genomic selection.;citation_pages=409-421;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059574;citation_doi=10.1111/j.1439-0388.2011.00964.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=J. Jannink;citation_author=H. Iwata;citation_author=E. Souza;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy for grain quality traits in biparental wheat populations;citation_pages=2597-2606;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.05.0253" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=K. Kizilkaya;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic prediction of simulated multibreed and purebred performance using observed fifty thousand single nucleotide polymorphism genotypes;citation_volume=88;citation_year=2010;citation_pmid=19820059;citation_doi=10.2527/jas.2009-2064" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. J. Knapp;citation_author=W. C. Bridges;citation_title=Using molecular markers to estimate quantitative trait locus parameters: power and genetic variances for unreplicated and replicated progeny.;citation_pages=769-777;citation_volume=126;citation_year=1990;citation_issue=3;citation_pmid=2249768" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for Fusarium head blight resistance in six-row barley;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. E. Melchinger;citation_author=H. F. Utz;citation_author=C. C. Schon;citation_title=Quantitative Trait Locus (QTL) Mapping Using Different Testers and Independent Population Samples in Maize Reveals Low Power of QTL Detection and Large Bias in Estimates of QTL Effects;citation_pages=383-403;citation_volume=149;citation_year=1998;citation_issue=1;citation_pmid=9584111" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=L. Moreau;citation_author=S. Lemarie;citation_author=A. Charcosset;citation_author=A. Gallais;citation_title=Economic Efficiency of One Cycle of Marker-Assisted Selection;citation_pages=329-337;citation_volume=40;citation_year=2000;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Schon;citation_author=H. F. Utz;citation_author=S. Groh;citation_author=B. Truberg;citation_author=S. Openshaw;citation_title=Quantitative Trait Locus Mapping Based on Resampling in a Vast Maize Testcross Experiment and Its Relevance to Quantitative Genetics for Complex Traits;citation_pages=485-498;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166171;citation_doi=10.1534/genetics.167.1.485" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Utz;citation_author=A. Melchinger;citation_author=C. Schon;citation_title=Bias and Sampling Error of the Estimated Proportion of Genotypic Variance Explained by Quantitative Trait Loci Determined From Experimental Data in Maize Using Cross Validation and Validation With Independent Samples;citation_pages=1839-1849;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10866652" />
<meta name="citation_reference" content="citation_title=Quantitative Genetics and Selection in Plant Breeding;citation_author=G. Wricke;citation_author=W. E. Weber;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic Design and Statistical Power of Nested Association Mapping in Maize;citation_pages=539-551;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. C. M. Dekkers;citation_author=R. L. Fernando;citation_author=J. L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="twitter:description" content="Allocating resources between population size and replication affects both genetic gain through phenotypic selection and quantitative trait loci detection power and effect estimation accuracy for marker-assisted selection (MAS). It is well known that because alleles are replicated across individuals in quantitative trait loci mapping and MAS, more resources should be allocated to increasing population size compared with phenotypic selection. Genomic selection is a form of MAS using all marker information simultaneously to predict individual genetic values for complex traits and has widely been found superior to MAS. No studies have explicitly investigated how resource allocation decisions affect success of genomic selection. My objective was to study the effect of resource allocation on response to MAS and genomic selection in a single biparental population of doubled haploid lines by using computer simulation. Simulation results were compared with previously derived formulas for the calculation of prediction accuracy under different levels of heritability and population size. Response of prediction accuracy to resource allocation strategies differed between genomic selection models (ridge regression best linear unbiased prediction [RR-BLUP], BayesC π ) and multiple linear regression using ordinary least-squares estimation (OLS), leading to different optimal resource allocation choices between OLS and RR-BLUP. For OLS, it was always advantageous to maximize population size at the expense of replication, but a high degree of flexibility was observed for RR-BLUP. Prediction accuracy of doubled haploid lines included in the training set was much greater than of those excluded from the training set, so there was little benefit to phenotyping only a subset of the lines genotyped. Finally, observed prediction accuracies in the simulation compared well to calculated prediction accuracies, indicating these theoretical formulas are useful for making resource allocation decisions." />
<meta name="og-title" property="og:title" content="Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/3/481" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Allocating resources between population size and replication affects both genetic gain through phenotypic selection and quantitative trait loci detection power and effect estimation accuracy for marker-assisted selection (MAS). It is well known that because alleles are replicated across individuals in quantitative trait loci mapping and MAS, more resources should be allocated to increasing population size compared with phenotypic selection. Genomic selection is a form of MAS using all marker information simultaneously to predict individual genetic values for complex traits and has widely been found superior to MAS. No studies have explicitly investigated how resource allocation decisions affect success of genomic selection. My objective was to study the effect of resource allocation on response to MAS and genomic selection in a single biparental population of doubled haploid lines by using computer simulation. Simulation results were compared with previously derived formulas for the calculation of prediction accuracy under different levels of heritability and population size. Response of prediction accuracy to resource allocation strategies differed between genomic selection models (ridge regression best linear unbiased prediction [RR-BLUP], BayesC π ) and multiple linear regression using ordinary least-squares estimation (OLS), leading to different optimal resource allocation choices between OLS and RR-BLUP. For OLS, it was always advantageous to maximize population size at the expense of replication, but a high degree of flexibility was observed for RR-BLUP. Prediction accuracy of doubled haploid lines included in the training set was much greater than of those excluded from the training set, so there was little benefit to phenotyping only a subset of the lines genotyped. Finally, observed prediction accuracies in the simulation compared well to calculated prediction accuracies, indicating these theoretical formulas are useful for making resource allocation decisions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
    <title>Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VcF6QRPhYWIi_H7CZ-OOJ2AB8U5RBGoGkakHKifnK-g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459429","apath":"\/ggg\/3\/3\/481.atom","pisa":"ggg;3\/3\/481","ac":{"\/ggg\/3\/3\/481.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/3\/481.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/3\/481"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/3\/481"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/3\/481.short":true,"\/content\/3\/3\/481":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/uaSjSO+Au+yTf6eh+ZidPg7SyePG7XmFh+8N99ui9bT3emho+Nb5dclUoMiuW5cGANa7lVYfP9MbNbqhhy0JPgJ\\\\\\\/KFA+DV9Lq5fnbksPcQ5TkgZXWyxSUX9RKJnfcCIGft9VEiM55ItytmEX8RPfEFyHejRKjreht6VvuKpe1abkVgBVX02bB0C\\\\\\\/UUTdBn36S7jltP603LPlxmI+Szw4PlZID85SLA+u5yopfFS+yYswrg5ktn+Ei0qj90rbqGlYF0rhui36MgHRxY4ajZN88uXptUBGzqJygTlRU2gBxnMBjD1HK+4lqLmE8HOvng3z+hsYTrVNFtjTNEzy1zSUDus2jvEk76yiS9GMfGKEnTiOO+i94PmVfi3yYH3AMrrVaomAEBjY0v8RU3bDsD5L28uAOppwYXvSH6qF1Fjv9xWqZ82VIS97kfvoqKDkx+X6cNj887Sh5xfjFzVXbBmSy02iDNg+qIxmOVYFJt94kwLBCOyibkGEKo9oqevS2D6powl\\\\\\\/O1R+iVo0P+G8EWYimY4wdioz55kLV2nGA+Ye6pMltHDLW+uZ60K101HewYT9HfKgNswoKu+wCT1PliQ4N6Z4FrPfQyZNo6KLiQq5aLJ5LUUDYB5FBplNeCzVVRfa5aJlc3KTLpH0\\\\\\\/AK0YRuaulV4Ghdu46aSQwZ8Z2FKBsEIzGK6QBOUxOuMYd9U4b46LJqEW7GX2\\\\\\\/conf3Uz\\\\\\\/Oi+GP\\\\\\\/mNFMPJOS7ZXXxKSGZZoJAQetOAW9VCyceNCrPfBtlSuQXaHDwb7Z0oeHsXtJW10zQYazroXmWQpXZlRYAS1h6qseof2MiECwvph1eVU4GSighQQOMOW64AcFWo7eo35LlKwc2gnSY21k6Tif9Bwl2h3FeOpeYh2f3OgxIPe5QrNugxe3KAYyB3ACWmQVmsFjbz2+dbvvItqaxRDhYR\\\\\\\/KXnn90HDjoqIK\\\\\\\/ZZfuvwdL5ZCiDXVIqa2TuSQLDxXW7s+RBD+Spdb8QFC26IO9IbebcO4Yb1jV3xUeZsZXRxskUR9Su\\\\\\\/AdEHIBOkp2zG7Rc6IM9itrTloC6O12ZwBKL7P5WI3Q8T7PFsYq\\\\\\\/lqgWgNvLyfKhYQeDXDEBHikqebqhRArFqWD7Gdbbkz5wNsWEymwfrAQoMxXRywDAB4R7ueoxiggr\\\\\\\/7p3LXfziselIY1QHkpqShJE8R\\\\\\\/ri1ogVt4E8VEFi\\\\\\\/OdKgAjdjvEuILo+f2uJ0Gz2gPAI20hJYqVgI6NxtJn6TJtCrBIOWjJ1l66SJArt0brEunQz6ydehC3U\\\u0022,\\\u0022iv\\\u0022:\\\u00227VgxG79e7biJZdD9UIPpkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224d6a370274fe6def03e9fec31c29f450129baaa31e016b4df8023060728f4801\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223ni2t0Cj2iLbEbOKm0ZspQrcbTuelnqIsCnCk1HDFZHBVLJNBX6iGw\\\\\\\/icow76jFbOwczxZyEgKbcskPHGhmp0YkiR5cwAUH9Ra1gQciqoI0u5X6oF8auDWV1xDHZMn9cYzS+FHlYZSO02Ct3J+J+nIJpBWXi8WCgcPEo2q1ryYI0LAdnF8JtQAlrUYLzowvujxgJlLIZcL\\\\\\\/x\\\\\\\/4ewBGV+TWYiTi3jUxBhj\\\\\\\/2EF+UDN174c3uoL\\\\\\\/znOX4brb9ZC93vDffAt6z22m+nHTXAKFc9g0UpuATfQ4GLrqGM3Z+S59Pznyq3MkM\\\\\\\/OebR6P3ySP9379IKeWmvk2vcufpbb0BSF5snx+Zg9XIqQp+FN4xjvCeud2uE21SQ4DTRpUmOlDZpCD66vLSBYNMA7llrmhh0RxEL0QMk0CZEdNq22kMroPvtq1\\\\\\\/5rpvAV5mUvGWp9usu5cPINcjz+dDg7CPalrRnX1SoqwW297F0j0EAcWXhtZ1bqSFiYAHOhVG5smFXQMdOpo\\\\\\\/jjJNVF3wnyE2EuyLSirSEIHNoncqVNsRdGTXRvd+U0dQ1Ytr7a2ECLAdUBIgl5AvJg7EM8Qk5hjFVuuvwcAPzWbKITteQbN5WiurI8Ox3ebNda3I50OVuIEFltnuBFuzijep3WwnsyYsuPQK+t4ehnklcNKSpgxaB6yHY3LoY+\\\\\\\/IEbvxMIKNvRKslSRB\\\\\\\/22dbMspw5VjKf2TK6YzIH50wsL7kKGukh9RFRYxx8rtvK\\\\\\\/wLi\\\\\\\/fD2AWnHJGClP3rncyQ+tzRtAFGBcrSVzryNVAxe+9lS1vcmcoWPxvvSRrNmZnDch6TnzFY92pPRnucW4cf7Bdm+vQ+DKVVmIzWHLZkm70rdJH6kN4m9kqyax8rKJWmwBkAV7tubfS9fSA4c4PeHtYn1XNAW1kLWRk+aKNKp529e2uQJ5R6mYA\\\\\\\/Bd19VfIAjzpJcY0M6RNuxdMa8\\\\\\\/G+rW8LncSjkwUY8E1yOvtASEJyKeGzXTnP5gIvQ3s3C5ZTd2Sus2IuMLIC4VnHAEF+IIH8prZJmrZrrl2kJSURHrJl0dLdbWTTgZiEKMKQ0X3mRlicpN1hXAV5tiQWq0EQGyE9KsJIfSDyHwrFufxNgJ\\\\\\\/XZjUc+NgNwb9\\\\\\\/gpD2C06AsL6y9bl6fg\\\\\\\/Ch4HVmUVXCKeu3BmRb9wyjsBfMv045f06\\\\\\\/o9sjtbKoTd8LHQ3unpofkSwHLqoBsMp23fAz7\\\\\\\/LcnvqIuk809BzoDqr5s9b4G2wwo1jR4mKTSZ1WnuTWK443idDcNr7+r7FQrTPZE\\\\\\\/v7\\\\\\\/cQ31kd0pWzNerCNg6ln9VUCxKblE7j3C5l3r1pZIcmsKJBGeet4K7EIv0B4zanNFQufk6RB2z0fKnJFalm\\\\\\\/gLwjnEyjK1ivYwkBa1eg\\\\\\\/qhIHUOV0yH7FmbUDvTmex0NY7RYKwRaGy2edZGIfH8h3lcyHaKUFOV9swQv5b98RuZyZkYsv3Y7iSK6GhT1LVacvpZHG25ghF+uwQQTICtBGYi\\\\\\\/+c66YA7ORJXZE29Os5sRpDnefvoCNyEPfmNQiJMCOVJgoMhOgH\\\\\\\/2t5TTdpjcPRgbjVtox5XMEDDt6xraecTau7n853a9fMlsrb4LLtZE7zFQPyog15FchF5rAkohP7pGRexF4bv0c7cxvxSu5qCghsDconVLj8IkvFrTkkn9kd0l41c2do3rLFRqytsOJbDfV+PQsJvO9R5q0XLbJ939f3EF\\\\\\\/2SzDPpTDlYXs0K9hzSFDVmhKPfWgWbyLw4YTCxfqG\\\\\\\/WVDIns70X8gHcyrhDwfR0nxA3WwgckW3EFMK553gr3W1aCCDZW56BhK5EcMOt2TSDhe36WZ072Z37Sfu\\\\\\\/d0VHnAifA\\\\\\\/YtgVfZNzJ\\\\\\\/FUVj8NQAOdyUs8UD8SvJonsC8qlAbUo5y80xXP67qzsG72flh8y0iR\\\\\\\/DSLgoxeO0rJ9W3bazvekZrKoMphTj5v1nqcgW5mu4F+aXXo8hWgrcaoWYWOxzqd7j9GzDMOgFjNY9wqZ0wwc9bd247nYJYlAYarNHo1NtrRiHs6WXQpMMDoA0DIMO+dLlq02SZhTnP985CSmSSEYzmRXSdW26GOG1yDH4RqoXbh\\\\\\\/X9JcMdMCacI2DTxOsErR7JcMx8BHwvY3xRsiVQ6FzmuPEq58KxB+4eDrtb7HEgGskE5n8sFkn9HG3Hp4+GakTu6QL7GXLHMmmY26ZabdrhZNd71mv1ITcZSq95TRlfV1vGfwP5VUaxgknf07lms+UyoN51vaPk8BfbIi+pBrEDoofNVVIXcgZQTcz2\\\\\\\/eDpaW2RUqktpFQz7b8ENBP+w8rodrgHp0G\\\\\\\/Dmo1C23mTJkWJf5RYvPtY2bGeieVSTN5uvPgfbpE00FHIqcrI3VIMsy6idl\\\\\\\/bZoRUUmS\\\\\\\/LMzj5w3sec5NKeHCJ2L0kb8SncdxLv\\\\\\\/yDVG7yVTOgnnEcFdAnjN\\\\\\\/lGcXELlBzcC1LFab5bKIox8dlMjlhqgF+NBPXjOTCbBaqSu5lWzgNtCuV89caMqdQ307lis1U\\\\\\\/fPukyIfE6kYwkYzkH1wqGs9zFxBmD0QXZ3ZUwY4pj4h8tK36QeMJVumRO1pEwvxMo0bpV6dik5ce7btL4wIKGpp8y7jHNE\\\\\\\/N+4uW0\\\\\\\/rcDqMTdI9FzlyKk\\\\\\\/6LFgBFoAMMlHX6mZJvUBuaDb4J3OLqOKtlpR+raGk+Yni8o68vDdY02xElTgmsXxcCEXq3113jx\\\\\\\/PMrf2QC7DraRN6FbHufhYGFBSQU19YXHqvlDKD\\\\\\\/6l29XefFLnQhsOFrIulSaXJpGu3m9QMZpMUZPQxqQtv6ulaetqOmYzNUuD\\\\\\\/VEw0dKky8J\\\\\\\/DJvTTiDipOEdspKxZsWw9ZWa6Y9LRQK22kaPwoFspRoIBQFvkpujfE3ScZkDpvb4uDAAkvvCTjNXyZZm18NnXG+D7Vj+K4LwMGWk+Y\\\\\\\/F4KECp6r\\\\\\\/Crj\\\\\\\/3M9EIuVwECeAD1rcoyvgJ+Cwbo\\\\\\\/wrG1p8bRVxFYZ1eqAvk\\\\\\\/gGVFoyoXGNafRK5sfvY9UQuOXUNnvHgeSPK947kD3RD\\\\\\\/eiRxCsjp1NiIx3FxqCTwHgUBZFJKRrijfPCg1dp5FcDQ\\\\\\\/fzHJ1t4cvKLp44KdxIWH4mVCwqijyCDiStcUEkYTAQZqsDaazga2SaXA1NzKdazpGBeOk2xdFCA5srwxfQ5MBzGw+SIZC7YKQeJ33iiWCxc7WhZijsEbh5nM8VwZX+VtuPyeggH7rOJKCU+CUnn55xsd5JeCj1SIcGX98k\\\\\\\/f+PbBA0lHpj1hvGfLMO8qaLQtdkZcEAoLNfJVHuXjuiJNWFaT1nsIQPoK1pF26L0udcXjU8wbQ5JFD0T6O7SlfZpvPYnTP265xGUtbSzGWVHMg7hcF7oGTR\\\\\\\/sRRMuF0Ng1imV6wSXTRq\\\\\\\/7Of\\\\\\\/E0sFGnWDgaNMMId+mSq3Vl77QreaqLx1LjgwKgUV2fBpIVMgu2qRGvedNnaOfe6fhoSZkqX9j7FXxS3oR4WFGEsZi+hNTx\\\\\\\/wCTwJKzVeaIaT8nPL8jVX7nK7QZUtMpyDu7GBWj6k0irMoCGxJ8g5HFRAS0mTwiPZx5OZQA3CIe3I9izW9C0cbWoawf5t1qxZPppL7QjlBWFViqGrxhhXtervFOAMESzqElrNlXGCwj\\\\\\\/HXB0qZnUcwCdZebeE9JL0jnru7lzbZJ\\\\\\\/J47NVsiWMHRtr5MbOlbQp9AF+ZWqM2RubaDSvb57euppGsQfCa183x\\\\\\\/kFPeKgqtTv21EOpaUEwxyX+iV+O8f0Wpo0d4kxXuR7Z1513VY0xDqw2g6S0+g07l3mQmrtDtZUgTQw4O1ZraPIc+KSU8dhzwOaQAv4mjRlhxUM3do8Kf41jj73L829LjFeqOlhD7MO1h1jgti3bbNIqxAzmFEzqll0o7TAq8hWNhqnSKDwC1WnmvN0xeTn\\\\\\\/z5wEFBVO8gleEqTtqb\\\\\\\/pjlE8eKsa5tprvrzReffNa\\\\\\\/raNiCGSCWTDmjTe2YJHHCd9yXiR\\\\\\\/B5dqEp5nfUOtG3uMK\\\\\\\/QxI7crLMOMiCYH7ss3Iif4n5OONDl1m\\\\\\\/om6QoLCXN1ObIIkka\\\\\\\/aBgRDW70r+dqBFNRl\\\\\\\/n+tKLObvAL8oT4CQohWzwaG7ITKQxzM5I6CJDFx\\\\\\\/z7p1SjHDE2NvQggy\\\\\\\/FLnb38gtVvj52DLvRr9IfFRui6W3LYgHG2BVZBg+s4ltvQpNpC3Cs6ejDtkDKPuDvb569QR\\\\\\\/JRAW6MadAeY7bbfqAMxKfBAiKH8SdlMRoizr+uKqYcsXz\\\\\\\/sRn2DPAU8Uy6HQNzZn5y4lvCtLcL5H1JiTJ9PaIZoKWJiKZvb5jj4f0ZZDcCBnjaMu6Cy+DvDuQh4eUGXYzcTa+56axsuHCpyZMad0mPgYBSc2TB5i+uptikEbmIAyk7u4ejmxBv7AgbqeEismviXvujaMM5nvfugwIy\\\\\\\/vOkPvfSLDl41HJA5lj+ak+2TDtxz\\\\\\\/cYKuj4cYolRXUga+YPFD1P4+VxrOJC3pPNMHtTGDDSybD1Xi8xNX0ZEgERyh2cWcAQYN9jzp0lTRynY5BZXkO8mZrUNG7t9dsbqGbqbnM1F43M4CNW\\\\\\\/KVUMb2nJwmXiAS+JKqHEmw5isdAp4MJqNf4wNybXJb201mBCSglR0YDVPUrSo6AwIJKWemLCrB+iVxCRz+4kFQl\\\\\\\/rnWz9QM8EYB8Vd+XcJrgcxUKAaEvn1xU\\\\\\\/VHLatlgA7rUHBjJfer9VKvkLnzEnLY9CENnBwnlWkREu4H8TPkkctblvNybemoj6H08BB7AJIeyOLEOcu5Qc9KxNQaSp1CzVF84GgVNflMDyWMFQi\\\\\\\/5dUPWfENeAphLSldZwhHYHb4RrSLIXy\\\\\\\/21QMKpL6V\\\\\\\/6w7iwbBOovNTbEHwoidyO4AmHIaplWmVu3plGALYYfozBB5YYm3m5PDGYYKEKEXdtklGZTEqbArqiWfvFfst63Z7m9ZGMkgiHvVW1TONr3z3VTaTYHv1\\\\\\\/0jetR7qw\\\\\\\/wMm3XFwajCgUd+jXhF8YenYY+yr5Z70YS3bR7HIcRKAJr4fxAIt8eNomxYoQEM7aX32mWo\\\\\\\/24C59BjDaDBz50MwA99QgW6WsrY0EKjqSgSZqmZXKN+T5pQF7AUmdJpmh\\\\\\\/FRtBDkK1efIIrjQ5QYmkgXNP1Is8u5I\\\\\\\/j1zYbUu9Pmo2+Lvl1ttnEykirjVc\\\\\\\/5MBTUKod18RYaiFJngsw00g9A3SXsLoLUuSeoHncY\\\\\\\/WyrFIHvHHqFexNpKO6nGXX35BufJCmTn9s5XkiaA9Jl5Nb3Esgh3CST1jc0YInnZBP4zY6rHKn3rTEIiTyat0s3mxQkww4oUf9nXmF6X7lC618dx\\\\\\\/5DwvGkqhcrbKxCAOw1BV7bHZXOM\\\\\\\/\\\\\\\/kaVb2fnYlvh0kyVPN2wQHUamXA9t\\\\\\\/1f1pvtUg0dpl\\\\\\\/EBHL3DYyES2Ic+r8cQflQsw73cCWVA6HcrBd9fn0KDnO\\\\\\\/OBLT+nmxO3DQ\\\\\\\/JaaedeNpGXpKkHKnZA9YC3X9+TEM6C6NH7k5dPzSkHj03Z+AOK0FXE0hLXBvxdZmvPcaVpbPsxYdXJbXUzOjfsV+a8qa307uR1QIdvVdnJYblFH8LrPLZb2TwhvvcMxjbpB3Xwdrfd7rqCAnzN3ucScNfONG2dB7PKFvfHf+tozFTxk8In2GnI5rqDtgwEOWAOBb6aqzMYToY8ySUxQWdh+S2WTgbe2rKs1\\\\\\\/OJ5aXSSXw9egmy1KDTPrlhTTbKE4mJhHeBxdTtIrh7G1g8I1nNo5cnRd5l02JmUgQCWO99GasKlXwPPDSyVWKttf8SnxV+ZvrJPLnKvxdtlUmr9\\\\\\\/pYDoqJohzu+1w0kW71Q4mNMyNDxM+qgFuvAAYaQeY6yi2w\\\\\\\/8MUe6OfYU+I4yivwv23vqGrehdbM8gMEvMLXyppFgfkINSMBFgoR2siphC1BBjw1ZdoIrabQaYpuKYPJZHUMyHYEGITxkSNv4xC44+mNwDWvp3RA\\\\\\\/5GYA5kCVLUsZrFdQbAS+TjtVm3IscaqrpNw803brMVGAU2zIqk1rwIu2SLtEztG0O8SjJigAXyZkDNkJR6QNe4xS6iXCCyRmMz\\\\\\\/L\\\\\\\/gLuU8Fr7lOyZv0UiJBnOs1smEUbbB5wFTMHaOM3\\\\\\\/sNOHrmg6KrPyHDWztXAWjrOsW7gAScNJvNhWigaTnW3c60iUoApBzsgzcp78qm5QX1DrWoVGFe81mQoRgNS1+Q7oQh7zC3XL2tZR6V3F7bTVMPKNlYZxEp21H1VkpH0ZTZZZ8sHt\\\\\\\/jlkDKac4zVWbE7\\\\\\\/otky4P4xdGD+cok8aS9qq8BPWZAe94jQsyW7HCXvtjUBoMah0UIVAIJdCjI5lpxhmEz7LJonJAynVxnKmTnxew8cCK0LG1eFd0aL8wC0QP9cRSYKMXnlQ6vuo8kFeLH4+qWXT56GqFD\\\\\\\/aBa3jxzn83wfWLNZxe8exAUyVAKObRp1PVGLblOwIYm66o93rZDWTVEXbeLfZHMpzsoQn1vUqYhW2aJySpc4u16cB3TcEj6cy5wAOeKh9SETmE34cz237q4dDdWQPsz0vav9gc23QQ6Dg5LVD4zBbkeqZOky0nSk+3ff6kMtWDI3z5Oec2XB6lr+ZVFu2Vs5z\\\\\\\/tYCMSKmLH\\\\\\\/glBGjI6nxfvkepndyDLAqsixGVnTZzyPEZzAno6dJYWm73WdIpR7IOB398PA\\\\\\\/ISryRUIVD\\\\\\\/YLUnq0Kt0S0Wk3DDwHcufvlX2E7c4jywBXUC1SVlMJvPYnSkAYQpyqevX4CI3ti0\\\\\\\/A4Ssakdiyvyz48U8\\\\\\\/wUOo0jNd9OU0t4+fVuZKtYgkzoZu201N4QKeLZyt6ShuY4JX5kSLZURpCgWNPZDlQaZfAUc1mmNIPJIT\\\\\\\/BHqM5mrV5bxlRCqzpaj+og3JOVXI9FnaCMD8gBLtOizOxg7STgbaBCi6Ms2FGbS3hSfPTKN7kzaV+lMzT8WLJKmqjNGGVLCDWEMTMyTjZecQaXy9c9nhU6d1HI2T6ISCxwjiN8d\\\\\\\/7KTVm4laGBm1jqrNv42gxtQY9yotrP9RkrLB5zrzxL\\\\\\\/15CKB\\\\\\\/+Xu0bDLVxXyxz\\\\\\\/+cvs0mf7cxdIUTLsPvFFXHMaR5tQ4irjeottvzEIURovQE0ne9JALRXhWGG6GkD40IgqLblsMgwkGW\\\\\\\/tAkUwt0dQvh9kLqNXzv78zsfeiRobZCoDi3genDtB9dGH9iYuUW1SOpA8bab0zi9B9oJzMrLIaOI1J7lyBHr4dwmqWSztQHfpK75qx+7XLWcqicLVnY5F1z4fFoneoxSm4bhdeZV7Lotpo\\\\\\\/IsM5zGyiOBp2KnUQQWZXsL2oUoI01UPNdE2D\\\\\\\/IRL6T2gRjVg\\\\\\\/5fsYeqfu4JdUDdOFYDbPmMGN++FpdbNyz+LuYDKgeMzq1\\\\\\\/fSUhEnXaJv87C3CeACKNWr6qAeISuGVU07m9TEGcYi1spCrc4wfj1ITGbY2OpmWztbeYrhUGrFDWJecT0lkm9XJMRAMgs+PZjZ7nBHvS2rLucz262\\\\\\\/0BlT0RobHr7UYFR6YzkOD8W20Y1n+R7mwD2lFQotvlJvH9gfU6fNp81+c7CVcCc+eIlca\\\\\\\/uPWG2Pqsf+rOAxWQi8zVUnmwNumiow0iZbDJ7E34zM2Ul9HGMB50Bbsdb8iNpnyE3EY7etOmamxdzH2AFmSJ2KRlL5E1rf\\\\\\\/S0FJtIKzrJYGzc9UE80L2FPdqxm0IkEmiIDqsvMk3lRKcfGUfVRx\\\\\\\/oZad0T5uT4+bk1NallwkeIVGPs9HcIB1z8rRGnLvj5YytzTu2DdvRdL3HybFese3SJ9ZcNw\\\\\\\/HQG1+ThowuYZ\\\\\\\/CvPfEBHWoUf4zPaMRuuSIBGXHpbMsQrVYX5zaTOcuoo2Vp3rC6Opr\\\\\\\/GVtYiXGvXenRZjRLN+kfw493sY+p3AnxHnEbut0O\\\\\\\/bvBOQ8FgiILca+r1TNg8Fam+5yJYB+ZCcM3xMH+AbmTYmCUpi08HtesYPVJkE23us0srRE1D5AmafBFDjoxCLElFtXleXR1Td9oawjrCSqLTOQiX37nGOgllVKrW39brRrEA4N3tualXRqT5HSZSv2KszOkNGz38n5TIuz24UVMjh8JCjtVb+5tx5A6ErgwVw\\\\\\\/5OG3wFPVLKtOdUmY\\\\\\\/4qTB36h2qtJBEdeSArHw9cfR3orTnfD+HmWPCcHaf3MFCoB2Q2ZcU2CRTALLYWbfWjHhllim46cX1dM+CITHX+UnYOl0Fge3YuoNnP8BqWIXkHftQ19PGQ0hxaNiBosJmLsMakqd6s3oZPjnfJekpU0TecguC69ux+ydIpYnoME1hN+aaYHgJoDjZU8IYD5R1kx1Fhx90LwVUkFnDMLRn27iM\\\\\\\/W37KBGk\\\\\\\/6ZDGxlZgh\\\\\\\/nTond0Fb5WoLLXFvvUJxaqfHvY5cn9NtTn74J8PEAfPz1sNAim+hD4xyMpKUXqwdS6eOiC710FwDFncbrWX\\\\\\\/ATeiGEIxgzoGXeBMjxbjH5RK8qYy1DpBppo6IX6K91tC9dBzuWCkjKARACqpjxeuZQ+O+utT8qggqKJMlvCH4qjTdwxJp7Plqf+QMPjW+2TkIY2Ycr5DqxyGnCfxgf+nQlBB5kV0EEqtD7E8LpRQZ6c4kxN3YjGuL2srr3RnKTRy4\\\\\\\/7kUzRJIx2zJqFZ6Pt3+Y5cBQXgLd\\\\\\\/gaIau0i7\\\\\\\/IQPmKOu3KEAX34ZByye1SYRWtRL\\\\\\\/f46OEKttNyfvOCYbp9P2DV0GNlGyNWKLQU1+H71Zws4YPzQZqcIhvCTcbiGglXEsgRJTVpd6MiaMOKZTh1UyFtB\\\\\\\/O8RE64HOglG7uq7ft90TOldsmD0I2lsLE\\\\\\\/I49IOJfMM+H3lo75RIbuZq0sI9qMTdfDWrlF0nMT\\\\\\\/dV9x7YPbsKJgIpZaTT9uuS8vI8uh1fM+4bXzPn3O5jvjG8eNaazR\\\\\\\/UWeUmsIzTXhnuD7qliEorYMLoC2YswtlrqULppvpy9e8UAiF0iyE3U+ypzt4bAcA81brHJisdiRDVSMugsZiT7VxvcKqxFZ3oAvTp+h6T1Re6SFXLDrrRBtT+tGEGVqxF9MZ8uZuFAnT8sWWYII9RyI7SZNIQBBlgS\\\\\\\/BbVSPabCOqwzunlTgnxcBahzjegC7eah2HTck3GPsGFxRGEZPyuU9fgQ4mjIWafSm4k\\\\\\\/7+Dlv1GOalMtmXbHQKVjdB5Mry3wNvWPcYEvLrzxZ12hpF9hyt1\\\\\\\/ts7syqUAWJpM9Nlj9drJ2YielHDZCxQ5fV90KHBy6orgFiMmxVOA3SX2YEmG04g\\\\\\\/tffURRMraWFdE0wyH\\\\\\\/iGDvQIx\\\\\\\/AF4xNXhlsOjFhJhTg8bm2M70CLSIRL73wG+tgQB7uBBcMLbH4MCQ09MyocjMFCmIIVN5dnl8BjUfB7AykfyZcjIU96ndkDLRQYRPqR4ny9vdhTbei3N4R53b2RJ7HyCLhmhcMLXqokUERaEGvyQzt6fzQuD4zbcD60ofzs3sQDasfDezSqXIXZAC5X7FwvOPHRt0\\\\\\\/DEVGG4xwXcUi1cswePYSsJI14um3VhutsxD87xwbRkih09fHMQmQt4m4YSaURXMAmUwVMTC1I843zYCwWrUQA68agNfOYvs6sjJotKPv1VrS6zoXZ6YqXPr8PWFea0bKBsnSEcX92\\\\\\\/7nFevHpcN1Cg7IFgxcPmWflMMoixJ8n7V6DGFYMN5fw5G2JUG0driyE7xX\\\\\\\/z7\\\\\\\/jp+cg48OTBvfUCHGqb3eYyjsaHsIPSee4lHvluQbh5+B3d71XfNRjqcbuz3fqXoXM3IfkeUYA+KGNJx4XGtVrkmlrg6zuFy7UKV21ca8tm6JIMDzNE0nPDcs16jemfv5J\\\\\\\/J4+fAYSfaaMzdKhX1vEtCy+3SqfUdYSIc3yN8vPJj2ZjVDoTJzftUEKrVqomO57p4RCaaO1F36\\\\\\\/05ER5q4wgIpy4hcSfscWqQT1v260KTxxi2SFkwhJ91NqFCtxKNIzb2dVQFMKhF9j0S6o+wgSPrMxOqEX1y3+iJUIvu8+9j+LYB7EkWtrh+2B9RXuDtelxfQ5WfjJqd79ZLwmZHaISo9RulFs1j5kauqZaUDO4AG0Slzg+qdANRL5D8yySw1pQ\\\\\\\/VQb\\\\\\\/pxjmM4f7z\\\\\\\/lxa1sdpkZ1\\\\\\\/cOYXxpMCi4QVg9O5IyXKY2Tw9ynLkzriBPNQESR7XNU4OwBVFCLPOggTl0NDIJLVjpTFX3nzqQ80EBfrGWM9Yk3XcZpBbG57a5zE9lZAhUTH3YU3MXUuYR\\\\\\\/0XQTrtUROSTxj+HmUCL3UJyLMR8lpALvjcR4c2pS2UA72zKido1dqfiBYTdwE8e1t7QuvhmBtZQZUZZMqkBQdOTcpIp4Jyp55aFykq0am2B5KeBDpm6CLhaU6FNjQDc0fFzEqdlQrfBIIW2qO3pebattdU+L7tgnBSStfObNbdSQKddI8GwQQunY5gECMvjl24qtDPmU7NCHY4MiBhRx1qBPcAc+a2P\\\\\\\/1q9+Y5H2Z0cNErWZLdtP27xcM4rNQtzmrVd\\\\\\\/ThA50QASlqfmq1\\\\\\\/KpEZ27k8V6Sla3eyKebKjH23rHyw9fuSz8ODlI2z9LiTluG\\\\\\\/uwZf80i\\\\\\\/sgk47fsKRnKh+nIgukuXh\\\\\\\/5stCccjoigL+Qz8xDLH4cEr1I46YPQxuBAT87yDDEFnuexIkd5EA1hDrp86zDeEGqUtHLJn8BHHuUCpKfI5yQUp1g0FVPI5VA\\\\\\\/eIu5WKenNIS1MXp7TLFsYZcqrcVb6hdICa1ouzZoLTbajMoMCxyewRUkKSZ8\\\\\\\/fwZsj0v79wrWaz057I8+WOzxXGUPS+xlyWA6GxiIMCcxu\\\\\\\/0t3N6ZYdBx59Ks8Fd6JtNXBs730aSNj2dKDYphxiYDaWyXUQivxDK2kX5v5sMZbzkK05F3dPF1twHovvkTdijPitqJPM\\\\\\\/ZSkIBpYLrlPO6b4hzmLhnegU4nXGdBzyi4t6aeXRx0Mn0e9UNMBM9l\\\\\\\/hviTY4YvkH1jSu98PDFqZcU0relGY5He9gl9twdzc7yKU5vVLST\\\\\\\/5SPlssM1ueqXdCbxKtINRJZZxLYixfbnDxAufejsijP16w8fd52j7WaAwBRQhE4qbx4qB\\\\\\\/9p7VGY1aL9+yfyru0J493Q1AnyVg368UFh\\\\\\\/CVd4qWF\\\\\\\/1woLEFQDp8IUOwkAhpMaIOXvOwPJvsj6TE3pr2tv\\\\\\\/+v9\\\\\\\/428ElGrLGmHvb+A93w832Ms9ZxhxBoamDFL4O3RBDCMZeGPpITgLWU6WBsZW2Z7zGzwPzgkFyVBSfWthAZPfb2gMBAcIwCX\\\\\\\/Xg5NTS03QXSVHy3fGljIXyY81kqVw4x+r6ePYXlOD21mpAyaidwiyIUDb8fwC8JjIW6CPr7swqC9fKu44+LYsOSF1coE9i8FnPHhWPXLP49+wiWm+THvkHDDU\\\\\\\/8W\\\\\\\/osSqmzVh2ttuHxnPkL68Nds7vrzHxVR8LOiYihdPxZKQxUGfLdbAi4XT1u6Qi2ibM2De05WdRRW3\\\\\\\/mI+RAdXDskkYGLqd6XBxnNQ\\\\\\\/Kh2UnqU2h+tC5hNgA5XdJx7MOKwZVKiMd7+3MFk\\\\\\\/4ukY2bTZeYUDcD8YeS898mMAR0pufIoVK1tur78zwR654PETo2akMuRPYJJqFrG76VSN2yaKPXtCt8OUGI9Neq7wWoHxEmAYxs+PB1wWW7\\\\\\\/AYr9V2wLjbsUMlnDddAN+QKEz0OWPN9iobgs+nxOHpdL62XQDJAvB3QiRp3DyDKM8T2jn4CpGR4Y3Sjpf2Mh0v5ZS3FQXE+K4PPzEOecThpYNPZA8+hZE1EjlApK8JhJq7VEetvHqQcH7vjgbCUo6O2yA21ZNI0pQhinPbz\\\\\\\/iEENwf58QhUlyVeBBi01vT17qOxbrC7kXyJGijUxPcKelcV40T6PJ0eM39OJGAotPpRC+NZD4fSroYndwzFhBXdX0dsBw3C0ZbYH0dW6HX2nqOC0RjdeKvUGu2XBo4\\\\\\\/kwgejWuHMv5659W03O84u+kPzzY\\\\\\\/garBGTEHtuu0E37K7iIiLu1ov6t7+M5vfSnD0gsL5ns3GXod0YdN22RNgAvEjK49S2V9WlEhZbRSRy2WZZcCai2a3OxeAjl91gk+dedioHYkHoRoX1V\\\\\\\/cc4h1zi7A8z3sffLBDXUxjDzCyXl+aqvTBPAio8zJm0r2oHDpM7O2wSgcVeF+nN0EQOhzJXuop1nA9O2qoA\\\\\\\/dXykDrTyluAi4TerC5uFXXd+oww9dkeOtft7XLF3Cg9bD\\\\\\\/O+aZKaRt1gs7d6t87dDJywuBKJSMrPQg2KYKoIas95n5AuGZkpcOd8ApJL72a25+hCMvta87ilJrRcDUltS\\\\\\\/ov+W8AzqHi39q2JJNJ6amL0LMWinff89d9j24NQENtyDwT96yewNAEeA88vNuyVxTncTcsnWcKkGO1Gkz2EhQe0H7WcKFhKbmaUAPy5VbmeiHu5SnVRoiEydViqXVgilGS35FY35ySQ\\\\\\\/L9uD2gWIyqJZiYY5f1Hz9LSjhr4Ayf2jqNG5jaHv2M1KXEol1EDcMw7qxa8iHQmjeyZe99t1BI0\\\\\\\/UnOn5\\\\\\\/JoJ275G2gEGgEtjhexSyntQ+e7Wn+k5VzunCwCQ\\\\\\\/EQRoOMw0CCZ5fT63jG1ysmIrLPGvT8RYyrbjh4WTiVpt0ltaUQLQWC9SUFJXiDL\\\\\\\/ALZJ8K5Zn\\\\\\\/YHdUSVx+sf6STviMW9D79AFi4bgaHZCmDvkbyg1ba00AwsWy\\\\\\\/5nfNdS37VdNK5LidYkqC51yCYTcXHvWs4846BX\\\\\\\/voXEH1nGoQOolCSmxeH6XlEHXlGuym1suhB+FgoZqBkc4XJ7FXZcht4Nf4vfmJxrVbc\\\\\\\/HBrQeTicuDuGCWIV01Qhk+ntjhKh4BAJTVCU6Mt8do3RDkBJm2DYDzZQ8tVzxapQjUWb+7ylwCDBPnwe68SJsvdT13XQ\\\\\\\/gJTcsWM1Stvr1hnzqM\\\\\\\/Vv8I4EjiZAGVhpMDRozeVyzjUYkUH9XZoeQcLycy5rj+keYmdjupOQxE3bWpsYd9CvJHwWUMqWYHlXNOqD1Ip48c9URBN8QJ7y9SHr7g+IbYeBs20ZnwcCwcMNi2Ch9pvj3kOEgcf9ERc7HjZh4TwkX1fVNO8P1DNrIubvR1uLq9z9IDtK4KDka3guQBokGb06U\\\\\\\/DF7\\\\\\\/GMS42ygUADQ\\\\\\\/pfcBExMqky5fwIINxVZZjGzkqTLmugHExGf8vuMI25iGMm9Mti2xPFwKbsrsC74CNuEAXR2b8Xan\\\\\\\/N3rAQFCrYgOhvLyqZV0EfVLfjgmmfVfctncLEbjmzMaAb3SFCCwC5hyRf8KXhtqHycirde6NMfYd31vmiCCQU8v2NxqX1XhRPEukZjtBMHMGKBHgTzz6IUGTePejSq8JQCectPmFFQTzcUuB3XJMlKpsUPZlXB3UgrJQpOWVHwh5e+TEk82TdvtiYUMqF29Hr0Xh32lpnW3f3LESPmTi0x\\\\\\\/RUh6ywqO0W6A+Yf2IKXY8Pu0tSf9XneJWEuA412uOPwBpgi9QAzDxIhPI\\\\\\\/EfdXp9CrqZ3s98kWk5OYqG4NQQsZWrfiaKZ0H2Ait+LHkrHdk+XY99Q49PeQySMh+LIESV0IdZTHIPECFLwklx0HKLz2kjW03KYWDIlCy7k61M4icNknVhclO5TzGIMJQ31YDX5F9p9eedRTS1h0Sw54pcKPHhMA08YIfWK4zLscu9GZaD3xVyhAsK7XdSp8CVkh5cFHMQucrEEMEzB+JULXAhWwZCXCubZWB7l24ii33i3SxAAE7liQPGBCsOGfiW2omCdlFB\\\\\\\/q3b2jodTF\\\\\\\/YfIuAiTMEVTZ48348nUiFWJzSbBRS8yYM98\\\\\\\/PXW60wYvfgqp9zL2C7p416dGujv7XWZyYMWlophjs7JOqxZuHOa0f0GxnkcwPIXcudzq0mZciN4E4MTMYpN9hjWx5T5cgtFW3OqpTKr8IFVWvWeIMbUQavXZMsEzwtGSsndfgMgj3AQk\\\\\\\/l95vLPIEW1xufJ\\\\\\\/Mv1BpljhGZcAExv9beHL+ga9RvcUUKggMwjSsdfEY4x06IsStN4IFXb0bUkxqgdMVbN6BabkHtXZ5JaWb1GUdA5T43JnHsdjA8Fjw627LglJN6ItjZdz9rjGBIBbQodHvbGeB\\\\\\\/7xmlqt9AxQLTmExcbfGX2+Ub3\\\\\\\/Bt7opND5goRLD8kI1E3hKpCR364LNwdQIuLmxnKQVDpref7VBSnWTNNNVi1nHxNz+pCqHi9h2HhB48U0M9yOG4+Yc01Wq3s0mU1GskLdTToCUP+JjWY0FuzH1lxqwr168dCU07x3\\\\\\\/fagGWhrNHjQUkq8egLLR5MamcUGXZP6TjfBE1v23AC0+6d\\\\\\\/U2AOY5gWuVQeX26\\\\\\\/AEyNmkfvMS8KOAUqTpYZKTWlc\\\\\\\/\\\\\\\/l6TjCL9sxqAWGhGqIv2lBYrkWhY9xiC\\\\\\\/MsQ94iNGsnvqV6HDMOgfeSpz06v+XPSCFqI0BMxGlh2shplw1NFa0inRyeOYAl8hurb1BPJFdmNd\\\\\\\/NAqbT5wkCk\\\\\\\/zwT\\\\\\\/zdkOmtU+zsn8jIjnB7IEO5aqeObD7u0vmebxuSt448cuUR7Kv8BN7ILa8+sFwSJAEiPa0wpFnQpKFhxu1KOSGADsXoKjOszj\\\\\\\/TEg4KDTCr3t\\\\\\\/1iiEkicjiOJMjRAaBHlbjwtzAvAH2G\\\\\\\/OBnCLcm95NtXR+opvFzpl\\\\\\\/dcGdfZbaSKgeM+tMxHTvYev7sIUZGDGhnssfPmWIQXYMqb6QugVf8jdbKKRm9dUWwhUeiBfrl8HOv8j\\\\\\\/q\\\\\\\/1jT9y5nj\\\\\\\/DB2Dkun\\\\\\\/OaZmtxl9fAbDAAQL3n1wMY91fZBvc97UpB3zKJmDToFerw3fBG79\\\\\\\/j2508DIksYrfk8hZSx3MM9N8c7YsCjtD08dL0YLM5C1kVOSGbGH53uUQoWRcWTfWPF+zCdF0Ba62M7PdRUUBG48mKugN+oPLD6TuCr4jObqUIqZZQkm877eeqhwm8xHNyTc+guh1iAVXa8Te5nAPulx1Vo+kLwNjS2D3FWiArqyzyEAcSmKp5kyVfcMAu\\\\\\\/Rml9PJNCHinSgukzFigB2Ui1l+\\\\\\\/JR2EGv4NggCcULIxtGK41ufnND0ZFamYEd39C2BmAs+UMEc3tvpAxKR96n4cV0u8WyP0\\\\\\\/0tHNu8WM7Q3+733xbulIwot+zJQPS+d5lYZ1Jola1mXrZoIcuZHSuql+mdjZ8JBTXm9AKapcEorbK4CPSW56+IxGZUkCcCZQoGxA\\\\\\\/8VQwGGtgPY+rHVga0SC4a0rL\\\\\\\/moxviuhoz62IzbYAHseCsz7bQBIS1YgiV8\\\\\\\/8evIzIPuHpd\\\\\\\/e0jfqj5Mf4GhZWkQfNWkhsUTPXG8nUm37OxdV4H0pkXF\\\\\\\/sO78v92wylv6ewFqaJUSw\\\\\\\/TBg+NBjIjAqGEsY3JPbvr36H8ELrxg9EcX7NMDQqO2gVTOma0lA8w6vFPKjMLJoJQCOBmPNZyfZFftyq6uSoo+jwYKA+R02k9xZvNaphiI5nd4i0GQCqu223Xz\\\\\\\/t9BD1XO0l0xwHcAZ43h\\\\\\\/MlgDxSqkP3ny8lg1Y+hcZlei4zLWOPVS70Dsm1rXke9d0C\\\\\\\/7ncce+zB75l8njTzQANtsQAw4D9Dj0Mhlupanba+nO4doC91JVjY+PSUSNR4oArogC1sXRMyvevYNgXXJL+3lXbz6JxB+EzriGNc3SzlqD\\\\\\\/iY7Hon8GtAYXlH3dSNcQ54wLFA5kdxNFQwGnNOEfqr5Ei8LAdpUL7jey2ZCHul\\\\\\\/NBL9tXsLhssD+d2aygDVOUxnSQY0RMKgdf3FFi78soYnx1f0ZzZBrFVfX\\\\\\\/jA6+Z\\\\\\\/z0hUFC6Rus3JuKgLnG44LvFj4J9B7yMDdY5hsxJaYl213qZrFpM1WtaYxDFFSdopsJB4bNiB8cxQ+MQM0jROh15fTt2KjIjsL66jAey1hbZR6R80C3h9FIvBwd94aH+3E\\\\\\\/XPczYPZurZUYKrLgflmAxzfpf5id6lMUroY84TJWDXpCNtUdmy363GW583uqsA6iJIMFjo\\\\\\\/h\\\\\\\/ZGi6zFr+pMSizKM0e3dRp5n6hzN\\\\\\\/B8Mz2ZSYHyBP27XvERtfBFND73H2A4f0bCi75gnsrCCMA3ElM1bbVjWDFJ7JGrR1qJI2Jc9yCORI\\\\\\\/OfntaGfa4Qan7JKWijP818X5yReIfHf3JRR3TVEl\\\\\\\/EE1u+QlpLjsML1dm8HdVVVapkCw9RzzSdCx5kxYZRP0J0ULs+y+\\\\\\\/AB3jrRevqm+CJ\\\\\\\/ViI2JIMmBQJBKBQrDPDUZmbQ2Hba9Z6WBQeTWBJC0S91yvFWePqKIofIMmphblm6vMLYXX9kouucZYuDVJKRBmhuaPj9mvlegUU4wFuYPphvz+28ze7kND1NuEWovGuyDQxR2jd9yL0uVLE9AWB\\\\\\\/+XH0H1KCzMz3ePvouRErvQUplMPTT0ZsHM5VZluhmg+uWSfeLuPmxyhVK9VBDPV6VQHlWePiwZ86pPAMrcuQJk9oGRd9pV2RczCUJf30RpaSRaRP05aGOlZIDR\\\\\\\/xs1cJUYtofOt0tT1D8Tt80GZnG0oT6xZ42QE\\\\\\\/wJO3AjPyNTFVeXxej\\\\\\\/5mkPAcKX0ps\\\\\\\/0ubh6Xp51JdxHR\\\\\\\/qfcLpDouu2mOwv0mFL880EC60UrxctJgFJWPQ9EH2QdjcZCED4OUBXpi8vIQBIVWQTDqwtAk4B84pUV\\\\\\\/vjtW9JQQhZe5yFDoGoRuv\\\\\\\/qvJ+nHvQfusBSos+uX4FsUsCHHL4OBSOpK+sSzSNSqQmH\\\\\\\/vg0mc6jifdbOFr\\\\\\\/1Ob7Z6ur8joq7zFwiQ\\\\\\\/feRfcdCbhj6FlkfbLckfX+Dri6FN+5kW9CUbKOy6d3Ife83R71JVU1RN84TPRkXbjVmxHSUmsMoLn1AZSAo+iyLE42lC5CyS5HKehKzy519yhHRnSRhjZQJmTay8SGC6NL1oSRS0y871slpp6oO8T5W4WRkleUM8LQA8T877hfzFy8RsZBGHbb3Lign3LUUayivT7OEHMq60Ao3+rmYczboztyJJTbUVFs\\\\\\\/e+BpWH7nWfiBgEUQTk\\\\\\\/D1Yf5ofUyRL5H5vT1mIufiYSK+b9AXwYhv311LSqVjdeR1HgvmkI5bfiEmYJGxzf5lrlXxTVE4GbN6lZsXlxlU2NpjJ9o8UKVcxbVPU5jKvI84dZQQnDTeJgNQX8UunmhylT1\\\\\\\/AnIfgdHEqe13TsQXsJ7OgfDbFfSyKVZdMXQDMnGRKvYCcJs12QhMrFkWWoC8DmhhUifViUvuB7gGPEyFyB2TdAJ1xC4Vjntbv1DjlWpMZPstDsvG4WidSo80OP2llk9utMdMgXea8H0U76FNBhPgFMTQy6GbGeiJwl0jWLQU0B8DXOLpR1uDHbRD\\\\\\\/HI5d+yDJZAtB5rvlOxc8c6J7zEWOEG3qlhX1ROJVNkmatOKD4ZMS24w0mcQ++8M8lxiq5KRPMas0+D0dMtCxT1dBVj90YLriq+X0b0EIahxVH0jCI\\\\\\\/xwg5EiaEZAuuPYfbOThymbkfTdEVTCWQx3DO2qU4HH9Mi17xYh8g041w5lzXxmncNL47\\\\\\\/fTba\\\\\\\/RTp7g\\\\\\\/DNUqH6tD1AG4epn2KYPmPQXbGjIIMnjIX+brejMFVz2MgtQ87JXOo6ldcruAOvTWbahLAd0BmQptdxzR3Odjhtzgw9SgHrWhmBwbowT6q7q9PBgGrx50WFjDTbQy01AnhaBOGKovC1ZvKEC2yGqcj9FFyoQg1dyMkzNVcNhXq6WD9XwuObYYmiviZftyLWtIPOblHZWtlNZsRupTwtAoN6ULEOgrPfX82ztMCI00ElBOXDkzjitHTrJvUyTsqtGM0ehkQ9akL+JryPN2RyuIQnBkRGmtqfvKNatC9ZSjFwxESCYd6mpgF4yzV\\\\\\\/Sz0qblyJawNuO2qEHBtS+\\\\\\\/Tqc3aaltt78J0SRO\\\\\\\/JTKSPjVh2CzqZSc1\\\\\\\/68OEG8GkwFa\\\\\\\/bpFXSARdDTBh8YmCJqIxyu5YUC8xs8AIcBMYbpMd6fI9+pdwYLYhHptaaBe+Id1gv9hy1UT+Df6J3Y9jsfhW3zZxvYatNkb0gyOtZ2Q83SCb3dBwmkiEfuCEvdVL1JGjPNTvwZRNpSsR0lDzvy3qAg5DD\\\\\\\/qEIfy6Sb2Tqfm0ni2Z6Kwc3K3zAj3hUklWRhJYc7+wvP3GKEYD1Dttsroo6NC+hohOMUiRceisHmJQwKheNacw8ItcrKksNpDwghZdmE5JH\\\\\\\/4ILCxBYfZWKe7s8no3iW4XvTUNcqnspFVbyPdNu61emuvhC13wIoB5ENm\\\\\\\/hd9nPVZ8Nrj3NdccdpLnZ7RAyB5JC5y63tUM5AUo4A3kTrPzXHKEVIJh0BqucaL9lUMVUJDREo0es9uFYZcJPijg8tri9vrGNfpJpd+YrR1qhbkXJl7SYE0FmT9spZOkSeaCPX6UEprtVmoep3nuDjclrMhxLpljLy3DyfMr+nL1XxVLePtFBKJqZcnGbfFc4E5\\\\\\\/cAAqn+yMbznTPSh6JJOA5iaKLgPB7wh\\\\\\\/iX8bLTYwHIA6kesM7nzc6U\\\\\\\/FhmkkWS8fUwekjx0rh31TZLk\\\\\\\/aOZc3LZVjsZIfbcLV1R4VYO9707TxeGOihJp+RTYxbzJKPXPFXQjATCcdDWmA71Z8b5cef\\\\\\\/zPTKHP+IIH+k0Ott+5oY2vtZEoh\\\\\\\/h7GwPnhwB9wiOT2WcOpBI0JhOI8a6hi5sqEoigO1kodvUwh5IH3oNA1tizjduRgF429Wuvy0Lu6XOjgGiIRlesOTMM5S2u2eLJOOvcDznsCFk0+OoRvSSNXRtzradxJUIh7F1kiHil2lrdYNEkDXDehAOeSrjWBLrvfMpCefihSdJA58b+fU1H0ixQW2dCSmfHzzRsgmnAA7iXU7jOEJTxUtuuhdFeSbp4bMgVG9ZKPAd9Htvcl0AF1NoBF4gPPRxt3V5VsHeAlACWxuSngKvNSFdPEmnBajWcAYVQgg0VFgpxD77eKb+acd3VUL0lwqQ+YaP6AR52FBcupneGQIAapCYr6nNLn2AJPOIEOUB2eKo+F1E591zFJWXs+060vYJI+FHFH1feBQMYzG+iX0RR4pRBhjEQFC0P5\\\\\\\/icdpdiaL5N6aRIqdR8Doldf\\\\\\\/ikRF4WqMzGbvI3iD5Q3LN6IU\\\\\\\/feQ0e1PDbA9zrGlS2Jbq\\\\\\\/ltvpzV\\\\\\\/MRfjctB8cHKa514xuoXOiWHCkTxf1lX+HxsAcTeR4RP1aeGSp2\\\\\\\/OFttWXqJui\\\\\\\/2\\\\\\\/A\\\\\\\/xbkU2KKfGGpeIsIacsAK8VOArU6oTU85SDdYO9MSpRlThK2mEEiaJ8xEOGK5lbqgrocSdjQB1hr1vhGDKxyk\\\\\\\/YWj4GNMYmOl\\\\\\\/fTRMJdxVR4lIVQf7vLeemB7j9yyIsfMz9xyPfGfIV8r16lbW+MfKFU3EiqGKo1j43dC4nfc6mYm4inT\\\\\\\/WuUh94AYz3RT9xiYp4je0Kuccdk8B6Nwqyv44gB9mcoTo\\\\\\\/UKNhPrj\\\\\\\/imJF6pHj0jgGDo0DXBydkkP276pNdxcYy9FGPCgnCXXQuyVDzIKAcZ6RSxPIGPHRIECsR1u2jlIG5Glem+vy5+qjgbbJnE+lpAKutZY8c41A0HaTWeHvIi9wnjyRhEF7fLV5lgjU7g4+cTQGuiFUeWFCf3f+YELk5V9NO+t29PQzM3PiUQDx5wv4\\\\\\\/5cLLYM0nyECW3th3eDfw103GDcONoNjXgop6M4hI+XPTlUVqnUtdSwKCVfEFbqHbWT0PaIuS+okTpLFifdeNPYZpmbs7T7Gnlzh6S0udQ3zlBdiNJbBv9cP93SozWtLHVSdAfDqDWGcWAWLNECCGjY7UK9r\\\\\\\/w1GKLw8Wi8tl4L+EXI1F9IOLI0Qh3g3dDBCL1+1Hm\\\\\\\/8BGLejWQVVpZszZ2ybuvHKTM9XNqNd89OQh9cdF1OF25GuvkMWUgOFjPt2ot9u1YftgTyXdingPXA5ARDFT8dprL+FvK06KmjAlFGA\\\\\\\/2N1fsD3og7lVBCLn9R4\\\\\\\/kobDCv2FEH+cbGT7lgoDm7stNylMrd7xAglYI2hfq4lyR+XOw5Jf9Qis\\\\\\\/tFD+CFZit4rnMQjPG6HB5Fn2Rf6Qtd7PDLKIxWnfUxjOHjkg3LJWfqSKIUHy5LBNQ49BILmVPu46\\\\\\\/UDWgC8lD1wv1XDH1ixc3HfOKBNT8ER0sWsdJMTSS29oXLTMuS9uiX\\\\\\\/OCjcC3oSmhZrk1x5FqIXIdoYvWcuxfudWhGDUmwOHgYqP92B40W2fD0RzQJ1\\\\\\\/EPtAMpscgv+l\\\\\\\/7hyUYOI4oPQ20mmRHY1Ybv5N\\\\\\\/hVrbnI3AMehIDbBJOhM25c4rSlp95CoBibnjm4gsSGfvvMquM0dwES\\\\\\\/7ZpgvaRy1zdRtKnqRcmlt\\\\\\\/mWDrn7e9fbuahTSbfSvS3bksooqWVwfIdPvET8cgr8go\\\\\\\/kzTsainlNCx2Al7ti4WlKHPnNqNNUCYs9k9AEQ+4Ogn7LWytmFWr7aPnJIYZ6X6cu4Lw0JBjhUxHQ50uUcs0bhZ1FCCnof8GUWmvu0HMQNym\\\\\\\/FXhK74d57ImtAggIPF9hhVl6jEEZbUcW5M20TAlBrPZkBE\\\\\\\/d1AxO8ajGLdci8mxWX+0ElvdMkU+8qLn7t17kBKnDXqx2s5CrSfEDzfYcs3JKPXcEzFA66wYArq84t\\\\\\\/LJO6EOjYf9aF9FTogDecvG8Jlt\\\\\\\/vqCJgOKzhiMXRUOLJ17C+gbOaubKihT6jNx6fPF\\\\\\\/WnvcuvZc4b8BnyrqYa774XgjnRGusHDgsEc2AaPcLrcJisWCyh70FNVdPz9kUaBNO8FFROrg5iEmxCGZk2jpMq4Gntzebq3mF4uNvaqd53iGrUIxphpAYa\\\\\\\/bDYsGLlExGWm\\\\\\\/ZhzMfwbl3BCLrpBw4oSOQgZlmK2ULfBnQDmQM9heg3t0wMuEP+a+lvq+7gcgAgTP4UjgOCQga7jlE6tS5RJZWWU8a45n2\\\\\\\/zwLHjkqCprVidIV5ZNnTvVPyABtl\\\\\\\/yJRqCEMqPEB7hAWDBeZn0cp99YjLxsd0pJ+gJDbXJscCHe+l9msAkLOCbZ17+jZdUlYcOk7FWscyE\\\\\\\/\\\\\\\/hBSPdwwxBUCOpl7mw8Diz36uL78APCiSYUchzRkq+9QHKqU4N7ci6nH4iTgoNZ996MmwjY3QxkCbGQlKDwEN5srjIvFd4IbVFhuZaSTiEld5qs5c2o+oy3yzvOP3vV1JR+pk1hgpLKE345ZTX\\\\\\\/vHo7\\\\\\\/2TkJ5uMsvT3VKcGodNJjHnCpsc+fMmO+9UlW4GO3tX8L0Wk785hEIg5900UrXVxm1+qUxCJyox5YUjATIQQ2avWL\\\\\\\/Fl72zhw97dhNvg+yuUXCKwuamJ8mNh4ylYMJgsKqQC\\\\\\\/JD2GsDFSSwaflZNtOxduBTW5bW\\\\\\\/ETRZbZTcx03Si57MkYibcfMPnX7wcnvbjQCI5AhWClN4J35YpaIrMZnumUAxO8PnDQImJG+VqSwxLpp0qvcX0A5SsesGZyygMOlDNXY5r7XvJs6bfKYBhbldq1e\\\\\\\/9r1vWrIU4eHThlLfuvGntYKjiILuLzYZ0\\\\\\\/GC7Mj3ALMzyQwJIC1UO4khHwawupBfdPWL5FgY7jd13FyqDBiZxG9sMbK2rK4HlWC0StfqCk72QheMd24hnD4sVpuQFJFraKQ9yRdsuSCeISOidFyfsa4GK1+AixfS7g3kSdr5F07rsrYb9oVrct8nqHW8xFepLtp+KOdF44J3MgIJUEebKcfbQn4wmlI\\\\\\\/oaB+WU+XGK0lMZQJ3y+DVZtIA2SDwTkxYPLCvwPsNHXhMUHjQkm2o8dVMRLyMuz0GjrFfFLXIo4kel0CHMEk3NVx7lRqnjKvP1azgHRX\\\\\\\/qP4jtU2gWu5YdPjv+gV5gNdjWs\\\\\\\/Il8zObwYtMt\\\\\\\/4wg7z5ludhCicJztIwOaRSIlp8dHwEa9ctwEjOdfe0JWdN1b9H\\\\\\\/24wnIrm7d+QRawCcZEWipT8+H7+Q8ZtfB5S8rJvHSTOH5Rg6819w11hnQsYzPqkpdLf\\\\\\\/FqzdUI+Xkl69s5bP4oh9BX9sBqj8syCtBm6rUFWBCh+1Px1LlSRTq+PbjbPqcWGJehvrPOYI1vzqaNJhrOAMngbAwzPPgMDIoH2\\\\\\\/nBPWbMmja\\\\\\\/eqv1MWekl2iIsQaOx8ADwrG8xWr5QDKoAcQ6zwYlLDg8GCdvsyj9tfV5LjG73dTFKFMfshpGBSLQ3uzbnhDCm1cw7N8n9hLTuZeSZVyXbmpScLFeorEq4EpcGS9gzkxhr6ta2aNPTErHrVEBRmdSaJOMapYKO7gQJxc0S0j2P5f\\\\\\\/0lhL+DYw5fviz3GvujiQILLMTeOKSAxXV7SsxtJdVEZUfV3vAhH5ZJhI3Nnxtoq7w96f9gPZQD5AbKWJmtrJROspcZ0jAsTy91+9WzSHENlGIv2+LPp\\\\\\\/RI+6WigWu\\\\\\\/lADNNE2dcnJIMXRYvyRlMNsiuYup9SXlofgZmq0drHoDLyTzBHwZRdPYQKlauIXoi+Nm2KGjcCRyLRFbNXrzi+y5w8yWcr\\\\\\\/KbVyy5qD0oxvRNNQTM4pL3eZlZOIZx0l\\\\\\\/Gfgl2pjh6HErscclL8A3md46Y6cjLUEKOazmTNyqZS6PS48eksWvhafa2cMLLi697OyU8bwv\\\\\\\/PXRHf40kJGY1FKRmZWb+M8WLKguRdjfJszrcdpyzfa0diMsrfisL2T4dQxifVqkxbU1GKQczRRt2PAz3I8Q9iqreI479x5mgSMjM8ROCn5H9QCtEywP3Fb+ThwnYoVcnAj2ZoesHNeeEhhuOwJUScj7b6wAbJczRD4YQQCFBFJkJwY6MPwxqncH+wMNThKu06EDa+XIIPD9mPUw9bDgawsOeSvZHU4PPs7Ysmg\\\\\\\/Ng1Cg0oLXHHI8pT+KzpDnns8FnsQmXoDws6PTvtT+OI37Yh+OmQt5KV+gWZ1Ap4FblQ6wyXPKrq1fO3jjUBuD0LZN3xiYay+0u+NziwmLad1kXqO0DxBCedWg0ZFWG7VHcNk4mk2spcavukmMXKxw+l53EyPmjwQWhZgUg6cnsqBk0ZbSkY\\\\\\\/iSKsk2qxb7RSsf2EL+c7O3l+LtCAfnts2RTOyg8V90u1puADVj\\\\\\\/d8\\\\\\\/\\\\\\\/66ZTokx4OJIn6p7e+GgPOPhwDjBoh3TKb7HbRgiU0ybfeQiaM8V\\\\\\\/lXBzMezfW+f\\\\\\\/rNKLWp6M36srhJyp\\\\\\\/fF+iepk7j91Nvmj0CxsqQ8jmeBp6Vc21W00332EuXEpDTSrVGt1qjWxWM4K6YXO1rtynpdcH3pgcDP7dSKdUb0rhYbNm43CSxq7zOaT0Q1KiILjKgpYWvvtIbMa9OLJ7sC3OMCwloWjz9ngPQfSAGq2cxoBa500GURLtIn3wxGWwTyHWkkGSfXL9xLAlsEsu+rkeyoCtyvOw4ZqeCiirUfIg9Ljwzqv2eKT9wiSualO77IFay3tL7571WSXXFrJ0LmArLbvLchyli0hPXHJTjmB6zhFXJ3zwFzuVGtpuW+x9Y4Kz5WGplK9G4\\\\\\\/dKWUDVxmrOjZoUnDsshN\\\\\\\/D8LsrfECwRe5+As+jZP6804nevqeM\\\\\\\/GAuxNPDqCFdOAfg+Sst29vOoAfk1kcKy7ZW43wcGlmo7d2XNMpQRaQBnwVN8kkWqi8yd8V40MCP0gS6PcFFdBhjQU83KNeZFN\\\\\\\/QBn8RChDD77Du07D5txLQgCO3tLGpFmADMlfD\\\\\\\/i5DkQJPnyOGks4gzgOoEri6RZBiTmW2l4S2EUMewdetcSrn+t+wgmCZ4cw8mJgKhRauvm47Us7Kk5SjvFcyhtl7s1DC8Kur4aRdGuR3eFXxQirFFuI0uSUFx0pywLDu2zdii2RhMPw6R+pN9laLBlhkVksis9SmnfZt2r7IxKfWqbDrX0TIh8IkgtZmzDoxy8EhotsdEFngxvd4GQef3gV2nD+1qE4yiBh3vv+NHHiCvTcIganaDjy3cUI4EO0v4dujCTPOs+i0ZIx8kWbw9n0cDogG3X5rTrwO62Dq\\\\\\\/T6jrxTiQXa2Wrd\\\\\\\/FwZhtbntbC7gCF+jIVAQrrsYol2+580J2P3\\\\\\\/PjmMGAyYghvVPAud4BmBIFaHuZua57cYOFeBVm5ETcRGjbi+03jwxDGTDAz\\\\\\\/dkyzG7G0BzYQFOnX4RYhk4mqfpeckY8WqR4pOrfXZymgYyNZzHZvdQav9Rf47hHocJjhpRad9hVqO+w7blbu5HZ+bTywSsvoMWi6w3IfUgrWCDUex+3z5\\\\\\\/gMAV\\\\\\\/XJgzlrwGOnnXWuU7qLrO08rnrYDl\\\\\\\/OXDdLQQ5hvwM3HEdtbVnc81AMMGvuGxO+gLrkFK6kfHayiEz5x6EArGpcyXA9O1xYA3sA0c6Xj+UoFiCM82eItwI2yWAc8biMqm99FiqbyLHMCoYomPux3I+Hr2MkYKGtmm4shlqqniWf+e6utIluCktQ1\\\\\\\/QagSoeV3NhhIPd1rGV7tTC9psIOSqUpeCoOeGJ3CmFnajTSg5d\\\\\\\/NpJ26AOXOZvJNS6pH3JztmxT\\\\\\\/u7m2pAeqJeRItUwmaQkzx+Bqxd4ltNshcB6cLM3gwm6caCCGXaFc6HX49ign64AbPKvSrDPDfFODf6n95mOdOLienFG7tX+ZamuPa19v4fGcV0tIhSN5U2nJx7cgsgBpUPTup3JuADTZFRpSbbFBsmeD8dwqi8z+pYYGqRQoAhYdAwwVOuCjPEHuoeb76XzcEyGJkLdGFroLFOwWtueV3H5d92yYEKD5La9Fp+sUNWk4g\\\\\\\/4pecIocinmqROQolC0EMXFvM2wffe0f+sHDlREDYTFmlu4iysL3Y3ZSYIhrBjvqG1WQEP8bajsty0OQWirjZi3z3mz7tck2p48MmrfSZ\\\\\\\/Eq0hE4TS4u5ECM+COIZwbujT++1ywGL2NlnJk2ceE5cnuj0DJDRmGgc7dNBXiWUPBmRGo09oIMX7\\\\\\\/PINmYAk1CDe2Km+xxAVUPvrTwpzUy\\\\\\\/N6tdYh2aUIMEZhHU1YUy1w+DHs48y2noD7ePxCN\\\\\\\/gjgB5lPYp7jrryJrA\\\\\\\/qYrIKwzgLh+txoKtdTCKJe4ec8JT\\\\\\\/UzXZeUXvubx4EPSt7inMAT6IIfqRY7ciIdOB0zLRwbxKOblnMS8H9ZIIWdptEFEWwOhmsq9r\\\\\\\/aIPGO5tTV4plNWay9xHal5WVcKCtRqmCumLo6FhcPVh3\\\\\\\/x4HqCCQZ90jQRbazhE0tXVNQOwX5OzemYM3HwKKSrBnsH2SuKginf4CIiphrFFPZaJ3JPjshM18whMx2fcze2WxHP2hY+w8yoaoQSqCmBr5\\\\\\\/DsdIGqMrEYW1QUq2omkF1YJK8bT1b2qsuFpIVxnDynh7EFGp5QG9HTznIWr3ofuF48r2WpjisK12tl7TwUMxlWzwIOBo+4euCm+4vcbQIB2pKstiz7YjuWqmuT8p1P\\\\\\\/Di1waVPLeyElxESIjkbEq0YamFXWYiiU\\\\\\\/WDXNkbHPi\\\\\\\/iP2UHdQ17UM5gg5ZrpHELa1pc1Y9NSFkxLMpIAD98qoPZywaFXgD0xdszag25Xx8n5P16UfNy3HWFY7AowuNGXaerTDDjfDXJscNUfq2e9cCxQDZAD8\\\\\\\/n08EKWTH39u6emAjbaVv\\\\\\\/4x9JDJlD6cWtqyWCCARzNKrBMitdgpvqrCMWXhpT8eYTp8\\\\\\\/KmvwKEjWVCUPs2ZfRs7Tt7YilpwCIw6q\\\\\\\/daQDDWp0dP8n7bB532j5GCV2KxBKstRcMgXzIz9OB2DDq9jO1pLGZw9xl+\\\\\\\/eV2aAJ6F1rE0olzyEhKYl4ic6o3APyYb8gJ+126RSpqYR9UYd1rUO6Yi2tHb6B4GIAWhgunC9VPF2kkdzkmJ+APh8HGuB2sBO5aWsNPl7exXkL4jnLGHH9ueXrtDPyP1Q3oKqp0nPglSREkRxHck5dAwEzuObPlWAXUCDBAyAKXF\\\\\\\/Z9nghrMC80WlQJSmHtj525UBSmJxTBanFIbG3sjK9\\\\\\\/15YWyJtSlaqhvBHr8VkNhKkYvAAct099wEU1v7smhvr0ZTvOT0USl8i0TlsIW72UW0pLJSzGAXz6p0LRBgXDqS9gr+ImLor\\\\\\\/KmLg1VAqrXNGJwFAl5Pcs+slWzD0QgxrsYPf8EbUTjC9d8Bi81VmONQtwTqdRhzZCLPSDD2gyjYTe0EaYaTtbUVqd1zzXIL8FBxjc9+XIM9EkZRn+PWJWMZZU7M8JFGLYf+yEQHeOYdIisO2cNvOZ9AMMimWDiTV1MD3M8W2C1c5LIPLofJr7yDrXcbcpiFe3RTMrjuCXcO6mYgixYGbDdMrMOQeDXtjpLqrV9TNR\\\\\\\/pvRuaNn244N9FeboPEjFEnAPcafwSOnBNP2W6irDozCe+wKkEFXkss4iePnRCo\\\\\\\/NJOfP4XsVKmndQHfB4ey6THELZCc4NCwVd+2ppqxddQKJHsw\\\\\\\/2tEHWQUgABKtDbevf+yEcUBsWmb+SBHcdd6WnudDIpO6zuRNd\\\\\\\/1EKAhxgZON75LGx9k602N2IEY+6lSy4W5\\\\\\\/4hiKnG3ia3ZuXk+bQkJPEbTJ3LVvo2kSsgWzPrX\\\\\\\/POSPCatzKQY\\\\\\\/7tE60SyBExhxRnNmvcavCK1Yh4uWUZhx5\\\\\\\/3bgDEhmrq\\\\\\\/AqbT78hP7DyKDjocbd4Qp1623Ztv5US5rm4\\\\\\\/NEKbtGD8zhPGVdHFbImpxYQXQKHTqVVhHjoE4XgbGyQSn4IHzO6IblcNNUDGNUlHu4ddqtluJiZz5SHsE5uVs4YVzeJxqUUAhkaKV7AyvimGB\\\\\\\/Nv\\\\\\\/QMS20ArzAHV1UbUtjukjG6wjNFOiJag8OpBnOVFVzrhb+8vkZrDsFxYTwm85eT0D9QDlnvDu\\\\\\\/w1mwvX05JXGN3rVyTA5CtTsdbc4ptoqWmnZ6vGlbllv+XZE76PdRqkWEsz696EX8tZKrBLPKBAL+LsY4P\\\\\\\/Ev9oqva0dahAtMg78rfKU39DeJrg9t3FvvHBMA7FlkbYfGJ3MvW53z3rPFAAG1JzMn3CeGTX5F10bme84mLdOabwl7Wfk4H2KF7Xr6m4dgg2vUz9zLupzslbs2Ts+jelY2bDlE5qYkOC9kk\\\\\\\/2rZN3wKAzuHx+Fh34pONOzKv2qlU7A\\\\\\\/ItkMZ6geAFtykK4cD\\\\\\\/fwXQM7SMrJsAJq9FUJ4ds+6TwU+YxZLgJ+PQo527mQoMN861cBXekcPa2VjpOXOX3kHvZwkQ7LKsiurWj7g46fM8IrptxdtPZMumhUhh6sgB1YBk54LIPirG4ABCrWTwwTGzS8Ntji4tJapFDSY7Z+jwSv0w0vAqNVb4HFLeE6zc51kFlIYRb\\\\\\\/Kgyh15axCy8koCNLGmDXsFZo2uaGAFGr3AXz\\\\\\\/6M4NmNGTIBvf63I9mFLAO4UTQO+4mcDeD9pm+\\\\\\\/X8sTHIdqtEL\\\\\\\/0WJDl8RXI9URl92cJhXD3j7BcrMXdioYtoxBM2ECC8k\\\\\\\/HgBAaXSkvQPrPb1w1l+lxscC15Opg0rAPait6cc3WN8ybJwouQJ5UmT84d9tjw657m9VQbeMQirbFhaomkH3yLW+wo7JXEbG8lIrIrDkEVO1RIGBoeUqat+J4DcxbN6Y+1YeNyplD1p+jIqXIyFBaa3hueCoUNr\\\\\\\/R8kQmd9vSUXsvuP1cjEKeHmzixUREB8XWyc961T7CZg9jt09+TBvc1BoWjz9PM\\\\\\\/Yxyipvz6kttkTUlrw8uTdk5HJq7Vp0Ys+eVewNoF9GzHEV3dEiqaEqHUY65twu2q6olfKrJejImlXNbpImOEYKkGoE+pGOZxhCRRi6qyg8n9ybQK+aF7tqStI5oMkF+qk99QqZOtdweD\\\\\\\/KvEIsRu7hDVerOFEp9uhWbC3QtzZH6fwcHACPwktMJtcw5FBPhaiUZymzG57rMlaZnsR6oacQUM\\\\\\\/hz7Fjk1DElHjoJmtr8I62r\\\\\\\/wj1KQ\\\\\\\/GBZwlCenKpjJydiT0VAJyGRxJX80tqOWZeurK42hk4mHzsyjxg7AL5XQuwNuIbEZeTbhYBPJLl3O4S+\\\\\\\/\\\\\\\/j60898zMDQS2uVY3PYYmOKp4XDsKUEnxzEDz\\\\\\\/AspG15QKnbYd9665gQ9DH6VirwBpjIfTsj4Zrp7S5giXgi8H9xtds5xM3ET0uMxTv7WixbMghYAMt5fUavusK45XS0CdlY6J8x7addyNBfm4DstAb9Xzrozg\\\\\\\/js0vNISyLWgrZOBBMNOV+SgB3+F69UQ1H\\\\\\\/D+28ahiryDD3bZZP2ucRECYaRK7HFRqXy7EbP8cTTTEvVg98j0+RQlfxUWra1Dn2fHRxxSqQC5CZUGnGgADF4gZLgmvmNpu20RMjdAshvlk\\\\\\\/0aKaJ1sadoCt4jHLJvCTpABn5weoXJMn0Ay2NNGAx99Kus\\\\\\\/7sEvKvlpEyCnX2NffpEMeHkaPb07+rAAIpE34\\\\\\\/TE6HdYq4\\\\\\\/WTXjr35QopIjp\\\\\\\/x0eCrsFTQDywi4wL9zlSvBXVEIks7PwY0N2ibgiO24ym54pvkjBXKl\\\u0022,\\\u0022iv\\\u0022:\\\u00229t9epPujsguUvU6cXXxvoQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022852d0ffc923f2fcda5cba035670a0679800b086ab6a46fa0b3f56acf8c8cd585\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Y9NGn6+dN81yazcX5eqS+vXMblVchCojDbKGmPvOi8085BvAEJ3NXvSQwY5+8Pq5OfkPbT0ISVNTxm4ajhmUF0TVgtLkRv\\\\\\\/68xdy9gYHO2+inqKtL4+l76BPG8f2yE19ZuhFDR\\\\\\\/nWMd3VbkV0Lli1s8rLUoBihrGoxSryC7t1hs94woPHTjteSN5BIMcCB3Mma8U\\\\\\\/oIpN86xNcyqFRRQw13sOHhLz7rjHp4rhVQreCm3f\\\\\\\/TYocX9F5UTuz9dEQxaTA12igFRKHzMK4aayRfys3aDoXMiujJo+6PJPSO4qJ7Et7BmWrFjVJfsA4R0T9\\\\\\\/fetnGia6yuurYRmrElfnAkk43V2zPLcfI1yraGDXA6DaPVZVscFdP\\\\\\\/3QyYnWpqCyEI+VoPmY5\\\\\\\/BPNOWxEH6+i0LPmSTpJkvxuIQRTNBJSi31P3O8S+lHeNO6REKO201PyfG7Y4y7I2F3UhKVaYi9MgEKrxni4fqnZYRAtCi2GehBUjXq\\\\\\\/zhHtfjzmyKUbgAqEgcO3ybxKacJwdReY6qEeEGCb98ejOJeQgF+uHinkNWdxTC6DxVHYoaD6vWX\\\\\\\/BhQd+akEYkwJ1yHYdOh5V4AQpXjorTJ+yRCxralPTqD8Azyr7BP16FeRenMoxzvvNPq2vN+X6u1YiTu8zLIUWmyPu5m6Rdp5qCIDi0WaEvyDwTM98kuu8bi6DPQEH3E5wK2zY4SJmDkOSmIV2IASO1sxwPAgrPaFVb5HC2gFus990z8zQwNL\\\\\\\/XpMEbajmmrBv\\\\\\\/C1xFU95dkdJL2mYNU\\\\\\\/IW4FTJ643PqDloixI+H0bRLZcYwk7QcRNtCmQwGkkKoUMugZJagkiiK6RD4yZ8zsAr0Al0XYzwzG\\\\\\\/uNG1Wl4\\\\\\\/sXuCs0vyFpR35hcIt5+n+tzJAJbL1WMTGVeS2HilY2JTAKOKdcLatN\\\\\\\/sMj8zdG9aIeUU5ZCq6Y5D1q9CSNZQkvcGpmpzWm+2hhQJ0tsXTe6eBrgmoY13vCIH3GA+jYuYA9x4Eh94OtaGR4IzsBcF+oggou7IYmuwpTsL7t1TS1OQL+qzuKSScC4NIneGULe09UvFfBdDFoqZoXlNa34bx5rrd7Fd+LTcwSes7j8qMBUTXXy71A\\\\\\\/yGGioZVBfN8c2S6Ug6h+MEEZ6Z4wVvqpUey1UZmrjwbc3U7oK\\\\\\\/OYFa1mxdGyDaC\\\\\\\/\\\\\\\/xshaUhyGqni4sf\\\\\\\/OLRUIISt14aYyw3IERwn29ZNJ\\\\\\\/nRl1bhGUVz64stEXBl3ePZiMgqCA7red\\\\\\\/ATWVojjjdzJ33\\\\\\\/ziAHPgTmsROtfuV\\\\\\\/ceGHd9rWKhSMEuIsTnskYFEf0RTIRoP1acepChOhcpAYGtNrCwdks0rNR0tJIkWU3B2ooapNBctUgMcHCEKgwxlIyohJS3XszWYQRIvQhW87v81CD3ocTGmPb4q+oPgRkQwKtZv1kQVqnmFI2QHTaf2Bawp2Ltbpt\\\\\\\/9bpbmzyOViZrC1FjHn0MCB\\\\\\\/nFbMPSPCD7J7lJbRnvgTiPLAt6W7xQMYKOoLo4MuL8YNEj+kla4fQZWPuPC7wFh9Pw9mo26rqMjZ029rgpTbiJMb2edjQ36mlg6Q3gkj7v0AO1Lu0YSTR1eEdtcswMngKUC6PZbEJt9IvfxiDZ3K0iz1Nq7UHa16VoPRqvpkSuNauwzERJ2KOXGAg37acSj1A9jcfrP6Dl\\\\\\\/jqxiMhFa07oeL2vLgkGNgXuxkNcOpudiYfbo9fIQJ\\\\\\\/a8qtqUEFQmy\\\\\\\/1DcFTL2zwykJ3Hb7lVY0yl3HvqwbEgmAhK0OK9N0C1YNsGGtFpb29m7vOpssBnOi\\\\\\\/P85r+m7HwdAE1QX1yMRpNpbomvcanCjRrpSoMtkder1RpYgDrmS785ydzDlJ+XyJVLs\\\\\\\/\\\\\\\/jRjqaAMdds4b6FAsPSYZHOdFC8rCqGJVDqDBxmO90Bh9+2p7P0nKfnmAtNQTJZlHAmeaZQACJmqbJsFwi9062ddwKdqpspwiu3ZajJ\\\\\\\/RiTA1fgaqrKhep4dLS0gXK2EKTFQPdOzq+x62+KoLFk4bbOssM7be9qgKfZCMrOH8me6RPc3yShpa3qdZvJRI+L8\\\u0022,\\\u0022iv\\\u0022:\\\u0022TIJvQVY27bt+gaGr9zojYg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223a9ca5ad82006cbaf9ff89726a5d5b0d807f75fb4f7c1cc3fe52b8cbc07754ea\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459429 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/481" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tzleUJPycvvlSQmX1ahog9MXxPE2ij-N-O_gb6m9-eg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/481" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BgfI5OMeC4A2lXBonPL1RUxYe1Z3eP-qXUzeHj_pvVc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/465" title="Unequal Recombination and Evolution of the Mating-Type (MAT) Loci in the Pathogenic Fungus Grosmannia clavigera and Relatives" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/493" title="Genetic Interactions of Arabidopsis thaliana Damaged DNA Binding Protein 1B (DDB1B) With DDB1A, DET1, and COP1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459429" id="node459429" data-pisa="ggg;3/3/481" data-pisa-master="ggg;3/3/481" data-apath="/ggg/3/3/481.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">481-491;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004911 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459429" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alorenz2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">alorenz2@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/3/481" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459429" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459429/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/481.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459429" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459429/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/481.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459429" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459429/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/481.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459429" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459429/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/3/481.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459429" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459429/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Allocating resources between population size and replication affects both genetic gain through phenotypic selection and quantitative trait loci detection power and effect estimation accuracy for marker-assisted selection (MAS). It is well known that because alleles are replicated across individuals in quantitative trait loci mapping and MAS, more resources should be allocated to increasing population size compared with phenotypic selection. Genomic selection is a form of MAS using all marker information simultaneously to predict individual genetic values for complex traits and has widely been found superior to MAS. No studies have explicitly investigated how resource allocation decisions affect success of genomic selection. My objective was to study the effect of resource allocation on response to MAS and genomic selection in a single biparental population of doubled haploid lines by using computer simulation. Simulation results were compared with previously derived formulas for the calculation of prediction accuracy under different levels of heritability and population size. Response of prediction accuracy to resource allocation strategies differed between genomic selection models (ridge regression best linear unbiased prediction [RR-BLUP], BayesC<em>π</em>) and multiple linear regression using ordinary least-squares estimation (OLS), leading to different optimal resource allocation choices between OLS and RR-BLUP. For OLS, it was always advantageous to maximize population size at the expense of replication, but a high degree of flexibility was observed for RR-BLUP. Prediction accuracy of doubled haploid lines included in the training set was much greater than of those excluded from the training set, so there was little benefit to phenotyping only a subset of the lines genotyped. Finally, observed prediction accuracies in the simulation compared well to calculated prediction accuracies, indicating these theoretical formulas are useful for making resource allocation decisions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">plant breeding</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li></ul><p id="p-4">A critical aspect to the design of plant breeding programs is the allocation of limited resources between population size and replication. Larger population sizes allow greater selection intensity and probability of identifying superior recombinants, whereas increased replication improves heritability, especially for highly complex traits prone to measurement error and random environmental deviations. For such traits, such as grain yield, theory suggests at least some replication across relevant environments should be conducted, the exact amount depending on the resource level and heritability of single plot measurements (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wricke and Weber 1986</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Moreau <em>et al.</em> 2000</a>). Level of replication also depends upon stage of the breeding pipeline. It is common practice to evaluate progenies using less replication during early testing and save more resources for greater replication of fewer lines during later testing when parent and cultivar release decisions are to be made (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo 2010</a>). The bottom line is that selection decisions made upon phenotypic information solely require extensive replication across the target population of environments to ensure the genotypic values of the selection candidates have been precisely estimated.</p><p id="p-5">When the purpose of phenotypic evaluations turn from informing phenotypic selection to calibrating statistical models for genomic selection, the unit of evaluation switches from the whole genotypic value of a progeny to the additive genetic value of a single allele. Hence, so-called hidden replication of alleles can be increased by increasing population size. It is well known that power and precision of quantitative trait loci (QTL) mapping and marker effect estimation is greatly influenced by population size and heritability (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beavis 1994</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gimelfarb and Lande 1994</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Schon <em>et al.</em> 2004</a>). Surprisingly few studies have thoroughly studied the tradeoff in detection power and effect estimation between increasing population size and increasing heritability through additional replication. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Knapp and Bridges (1990)</a> analyzed the expected mean squares of a model describing the sources of variation included in a QTL mapping experiment with replicated progenies and calculated power of QTL detection for different resource allocation strategies, not considering the cost of genotyping. They showed that when residual genetic variation exists within groups of progeny sharing QTL genotypes, power is most efficiently maximized by increasing population size at the expense of replication. Using real data from a very large QTL mapping experiment, <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Schon <em>et al.</em> (2004)</a> found that increasing population size generally increased QTL detection power more than increasing number of test environments. For more typical QTL mapping population sizes (<em>i.e.</em>, 150−300), some degree of replication is needed for some QTL detection power (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Schon <em>et al.</em> 2004</a>).</p><p id="p-6">Genomic selection is a marker-based selection method that uses all available marker information to predict genetic value of breeding progenies for selection. By using winter nursery and greenhouse facilities, it is possible to make at least two to three generations of selection and recombination per year in annual crop species. In temperate environments, only one relevant phenotypic measurement can be made per year, meaning that marker effect estimates applied to progeny selections need to be robust across generations of recombination to maximize genetic gain per unit time. Many studies on genomic selection in plants have been performed using deterministic calculations (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Heffner <em>et al.</em> 2010</a>), simulations (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo and Yu 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Zhong <em>et al.</em> 2009</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jannink 2010</a>), and empirical results (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Heffner <em>et al</em>, 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lorenz <em>et al.</em> 2012</a>), but none have explicitly investigated the effect of experimental resource allocation choices during model development on total genetic gain and accuracy of genomic selection models for predicting genetic value of progenies at least one generation of recombination removed from the training population. Some work on resource allocation has been performed on traditional marker-assisted selection (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Moreau <em>et al.</em> 2000</a>). In line with the aforementioned QTL detection power results, these authors showed that when genotyping and phenotyping costs are approximately equal, maximum gain from marker-assisted selection is achieved when more resources are allocated to larger population sizes rather than replication. However, some key differences in assumptions could preclude the findings of <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Moreau <em>et al.</em> (2000)</a> from being generalized to genomic selection. First, these authors consider a trait being controlled by only 5−10 QTL. Genomic selection is aimed at and works best for traits being controlled by many more QTL [<em>e.g.</em>, 100 (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bernardo and Yu 2007</a>)]. Second, <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Moreau <em>et al.</em> (2000)</a> consider marker effects as fixed effects and used a statistical threshold for deciding which markers to include in the marker score. Thus, proportion of genetic variance described by markers is closely related to power of QTL detection. Genomic selection, on the other hand, does not apply a statistical threshold for choosing markers, but rather estimates effects of all markers simultaneously and sums across all markers to obtain the marker score.</p><p id="p-7">This paper reports the results of simulations aimed at exploring the effect of resource allocation choices, in terms of population size and replication, on (1) prediction accuracy and (2) total genetic gain of genomic selection models applied to progenies in a genomic recurrent selection scheme. Results from simulations are also compared to predictions made using available theoretical formulas.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Theory</h3><p id="p-8">According to basic selection theory, response to selection is equal to the product of the selection intensity (<em>i</em>), selection accuracy (<em>r</em><sub>A</sub>), and SD of breeding values (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-1.gif"/></noscript></span></span>).<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-2.gif"/></noscript></span></span>The selection accuracy is defined as the correlation between true breeding value and estimated breeding value (EBV). When genome-wide marker information is used to construct a prediction, the EBV is referred to as the genomic EBV (GEBV). The accuracy of GEBVs is the most important criterion for evaluating the potential usefulness of genomic selection and for comparing different GS models and strategies. Equations for the calculation of GS accuracy before data collection have been derived by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Goddard (2009)</a> and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> (2008)</a>. Extensions and alternative formulations have been provided by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Goddard <em>et al.</em> (2011)</a> and <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em> (2009)</a>.</p><p id="p-9">Consider that the true breeding value of individual <em>i</em> is <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-3.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-4.gif"/></noscript></span></span> is the allelic state at locus j and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-5.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-6.gif"/></noscript></span></span> is the true allelic substitution effect at locus j. Similarly, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-7.gif"/></noscript></span></span> represents the estimated breeding value, where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-8.gif"/></noscript></span></span> is the estimated substitution effect at locus j obtained by regressing the phenotypes on the allelic state at locus j. The goal is to express<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>in terms of population size and trait heritability so that effects of resource allocation on genomic selection accuracy can be calculated before data collection.</p><p id="p-10">Using basic principles in quantitative genetics and simple linear regression and making a couple of conservative assumptions, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> (2008)</a> showed that<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>n</em> is the number of individuals with phenotypes in the training population, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-11.gif"/></noscript></span></span> is the effective number of factors (or loci) for which substitution effects need to be estimated, and <em>h</em><sup>2</sup> is the trait heritability. Some discussion on <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-12.gif"/></noscript></span></span> is provided herein. It is assumed the individuals used to estimate allele substitution effects and the individuals whose genetic value is being predicted are different sets of individuals but are from the same population. Also, this formula is derived by considering that the effect of each locus is estimated by regressing phenotypes on genotypes one locus at a time. The error variance is reduced by fitting all loci simultaneously. <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> (2008)</a> proposed an approximate adjustment of <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-13.gif"/></noscript></span></span> (<em>m<sub>e</sub>/n</em>), so that<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Goddard and co-workers (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Goddard 2009</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Goddard <em>et al.</em> 2011</a>) derived the same formula but took a slightly different approach based on the genomic relationship between individuals in the training population and selection candidates, which is used in place of a pedigree-based additive relationship matrix in the standard animal model.</p></div><div id="sec-3" class="subsection"><h3>Effective number of loci</h3><p id="p-11">Effective number of loci is defined and used to derive genomic prediction accuracy formulas. This quantity represents how many essentially independent effects need to be estimated. It is a function of the rate of linkage disequilibrium decay in a population, and is equivalent to the number of freely segregating loci of equal effect giving rise to the observed population range and genetic variance (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lynch and Walsh 1998</a>). The effective number of loci cannot be greater than the number of independently segregating chromosomal segments. In complex, randomly mating populations, the number of independently segregating chromosomal segments is a function of genome length and effective population size (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em> 2009</a>). Within families, this quantity is related to the degree of relationship between family members, genome size, and chromosome number. In double haploid populations derived from an F1, the number of independently segregating chromosomal segments can be calculated as the haploid number of chromosome plus the expected number of cross-over events (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lynch and Walsh 1998</a>). The expected number of cross-over events is equal to the genome length in Morgans.</p><p id="p-12">Equation 3 was used to calculate expected prediction accuracy under various combinations of <em>n</em> and r. The population modeled was a population of DH lines from a biparental cross of inbred parents. The genome of maize consists of 10 chromosomes with a total genetic distance of 1796 cMs (Genetic 2008 composite map; <a href="http://www.maizegdb.org">www.maizegdb.org</a>), resulting in 28 effective loci.</p></div><div id="sec-4" class="subsection"><h3>Simulations</h3><p id="p-13">A breeding program applying genomic predictions to recurrent selection within a maize biparental population was simulated. This is essentially a marker-assisted recurrent selection strategy (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo 2008</a>), but molecular marker scores comprise summation of all marker effects rather than just those determined to be significant in a QTL analysis. The genetic distance of each chromosome was set according to the Genetic 2008 composite map found on the Maize Genetics and Genomics Database (<a href="http://www.maizegdb.org">www.maizegdb.org</a>). The total genetic distance of the Genetic 2008 map is 1796 cM. Five-hundred single-nucleotide polymorphisms (SNPs) were simulated per chromosome. One-hundred random SNPs across the genome were designated as QTL. QTL additive effects were calculated according to a geometric series as in <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bernardo and Yu (2007)</a>. All genetic variance was simulated as additive genetic variance. Two hundred evenly distributed SNPs were designated as markers. A preliminary analysis found increases in <em>r</em><sub>A</sub> with additional markers plateaued around 200 markers (data not shown).</p><p id="p-14">A population of DH lines derived from the F1 between two parental lines polymorphic at all loci was simulated. Negative and positive values of QTL alleles were randomly assigned to parents. The genetic value of each DH line was calculated by summing effects of all QTL alleles carried by an individual. Measurements of single plots were simulated by adding a random environmental deviation to the genetic value. Environmental deviations were normally distributed with variance <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-15.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-16.gif"/></noscript></span></span> represents the additive genetic variance and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-17.gif"/></noscript></span></span> represents the heritability of single plot values. For marker effect estimation, the mean of <em>r</em> plot values per DH line was calculated.</p><p id="p-15">Where experimental design parameters were constrained by a budget, the tradeoff between replication number and population size was made according to a budget expressed in units of field plots (B), <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-18.gif"/></noscript></span></span>, where C is the cost (in plots) of genotyping a DH line, <em>n</em> is the number of DH lines, <em>r</em> is the number of replications, and <em>F</em> is the proportion of DH lines placed in field trials. Levels of <em>F</em> considered were 1, 0.75, and 0.50.</p><p id="p-16">Two uses of the DH population to investigate model accuracy were deployed: cross-validation and selection-recombination simulation. Cross-validation involved splitting the DH population into a model training set and validation set. The size of the model training set (<em>n</em>) was variable. Five-hundred DH lines comprised the validation set. Phenotypes in the training set were used for marker effect estimation. GEBVs of the validation set were calculated by summing estimated marker allele effects for each DH line. Prediction accuracy was calculated as the correlation between validation-set GEBVs and true genetic values.</p><p id="p-17">Simulation of selection and recombination involved phenotyping a DH population of size N, training a genomic selection model on that phenotypic data, and performing at least three generations of selection and recombination. Although phenotypic data are available on Cycle 0 (C0) DH lines, genomic predictions were used for selection because preliminary analyses showed selection on markers alone was more effective in C0 compared with either phenotypic selection or selection on an index combining phenotypes and genomic predictions (data not shown). In the first cycle, 10 DH lines were selected and randomly crossed to form 5 random F1s. The F1s were randomly mated to create 200 C1 S0 plants. The genomic selection model trained using C0 DHs was applied to the 200 C1 progenies to select 20 C1 S0 plants with the highest GEBVs. Selected plants were recombined to form C2. This procedure was repeated to create C3. Total genetic gain was calculated as the difference between C0 and C3 average genetic value.</p><p id="p-18">Because different levels of <em>n</em> inherent in this study cause differences in selection intensity and thus genetic variation in subsequent cycles, it is impossible to examine the influence of <em>n</em> and <em>r</em> on model accuracy in subsequent cycles. To circumvent this issue and isolate the effect of varying resource allocation on model prediction accuracy in subsequent cycles, another set of simulations was performed without the aspect of selection. The DH lines of C0 were randomly mated with replacement to create 1000 F1s, which were randomly mated to create 1000 C1 S0 plants. This procedure was continued to create C2 and C3 and ensured that allele frequencies were stable and that genetic variation present in the C0 population was held constant, allowing a comparison of different resource allocation options in producing a genomic selection model with predictive accuracy of populations more than one generation removed from the training population.</p><p id="p-19">All simulations were performed using the statistical software package R. Genetic recombination was performed using functions obtained from the R package <em>hypred</em>. All routines were repeated at least 200 times, starting with simulation of population and genetic architecture. Genetic gain evaluations were performed with RR-BLUP only, which is computationally less intensive, and therefore routines for this part of the study were run 500 times. Prediction accuracies and genetic gains were averaged across iterations and standard errors of the means were calculated. An R script containing functions used in the simulation and a user script executing the functions are provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1/004911SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1/FileS1.r">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1/FileS2.r">File S2</a>.</p></div><div id="sec-5" class="subsection"><h3>Models</h3><p id="p-20">Because all the data in the simulations are balanced, and all individuals belong to the same biparental family, the basic model relating marker scores to phenotypes can be represented as <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-19.gif"/></noscript></span></span>, where <em>y</em><sub>i</sub> represents the entry mean of the <em>i</em><sup>th</sup> DH line, <em>μ</em> is the population mean, <em>z</em><sub>ij</sub> is the allelic state of marker <em>j</em> for individual <em>i</em> coded as {A<sub>1</sub>A<sub>1</sub>, A<sub>2</sub>A<sub>2</sub>} = {−1, 1}, <em>u</em><sub>j</sub> is the estimated effect of marker <em>j</em>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-20.gif"/></noscript></span></span> is an indicator variable for the inclusion of marker <em>j</em>, and <em>e</em><sub>i</sub> is a residual.</p><p id="p-21">Three models were used: ridge regression best linear unbiased prediction (RR-BLUP), BayesC<em>π</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Habier <em>et al.</em> 2011</a>), and a multiple regression model in which marker effects were estimated with ordinary least squares (OLS). In RR-BLUP, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-21.gif"/></noscript></span></span> = 1 and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-22.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-23.gif"/></noscript></span></span> is estimated by maximum likelihood. This means that all marker effects are sampled from the same distribution and thus are shrunken toward 0 to the same degree. The R package <em>rrBLUP</em> developed by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Endelman (2011)</a> was used to implement the model. This implementation estimates <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-24.gif"/></noscript></span></span> through the spectral decomposition algorithm of <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kang <em>et al.</em> (2008)</a>. BayesC<em>π</em> is a modification of BayesB. Under BayesC<em>π</em>, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-25.gif"/></noscript></span></span> = 0 with probability <em>π</em>, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-26.gif"/></noscript></span></span> = 1 with probability 1 – <em>π</em> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-27.gif"/></noscript></span></span>. Parameter <em>π</em> itself is estimated from the data. The prior distribution of <em>π</em> is uniform between 0 and 1. The method estimates <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-28.gif"/></noscript></span></span> jointly over all nonzero markers (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kizilkaya <em>et al</em> 2010</a>). The previous distribution for <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-29.gif"/></noscript></span></span> follows a scaled inverse χ<sup>2</sup> distribution with 4 df (<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-30.gif"/></noscript></span></span>) and scale parameter <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-31.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-32.gif"/></noscript></span></span> represents the variance of additive effects of a random sampled locus.<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-33.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-34.gif"/></noscript></span></span> represents the variance of DH line means, 200 is equal to the number of markers, and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-35.gif"/></noscript></span></span> represents the marker score variance averaged across loci.</p><p id="p-22">Three thousand Markov chain iterations were run, with 1000 discarded as burn-in. This was judged to be sufficient because previous results showed stable estimates and because all parameters of interest in the simulation were estimated by running all simulations at least 200 times. The OLS model was selected using a forward-backward model selection algorithm. All markers were first fit individually. The marker with the lowest <em>p</em>-value was added to the model, and each remaining marker was individually fitted again. The next marker with the lowest <em>p</em>-value was then added to the model and this process was repeated. After each round of forward selection, backward elimination was performed by eliminating any markers upon model refit with significance levels that dropped below the statistical threshold. The <em>p</em>-value for both forward selection and backward elimination was set to 0.20. This algorithm was iterated until convergence.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Effect of replication and population size on prediction accuracy</h3><p id="p-23">Genomic selection accuracies were calculated for various levels of <em>n</em> and <em>r</em> for plot heritabilities of 0.20 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) and 0.60 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Accuracy increased with training population size and replication number as expected (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The rate of accuracy increase in response to greater <em>n</em> varied between the different models. RR-BLUP faired quite well even when <em>n</em> was less than 75. BayesC<em>π</em>, on the other hand, performed quite poorly until <em>n</em> was greater than 75, and accuracy slowly increased beyond <em>n</em> = 100 at about the same rate as it increased for RR-BLUP. As <em>n</em> increased greater than 125, accuracy slowly increased, and the curves presented in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> show diminishing returns between <em>n</em> = 125 and <em>n</em> = 300. For the OLS model, on the other hand, accuracies remain low until <em>n</em> = 125, then continued to rapidly increase through <em>n</em> = 300.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/481/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy (rA) as a function of replication number and population size for each of three statistical models. (A) Heritability of single plot measurements set to 0.20. (B) Heritability of single plot measurements set to 0.60." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1660122557" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy (&lt;em&gt;r&lt;/em&gt;&lt;sub&gt;A&lt;/sub&gt;) as a function of replication number and population size for each of three statistical models. (A) Heritability of single plot measurements set to 0.20. (B) Heritability of single plot measurements set to 0.60.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/481/F1.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/481/F1.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/481/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/481/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-24" class="first-child">Prediction accuracy (<em>r</em><sub>A</sub>) as a function of replication number and population size for each of three statistical models. (A) Heritability of single plot measurements set to 0.20. (B) Heritability of single plot measurements set to 0.60.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Interestingly, a substantial boost in accuracy from <em>r</em> = 1 to <em>r</em> = 2 was observed across levels of <em>n</em> for <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-36.gif"/></noscript></span></span>. It was expected that the advantage in model accuracy achieved by increasing <em>r</em> = 1 to <em>r</em> = 2 would diminish as <em>n</em> increased, but the boost in accuracy was remarkably consistent. For example, increasing <em>r</em> = 1 to <em>r</em> = 2 increased <em>r</em><sub>A</sub> by 0.09 for <em>n</em> = 100:model = RR-BLUP: <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-37.gif"/></noscript></span></span>, by 0.09 for <em>n</em> = 200, and by 0.08 for <em>n</em> = 300. To test the extent to which increasing <em>r</em> improved <em>r</em><sub>A</sub> despite large population sizes, the simulation was taken out to very large, impractical population sizes for model = RR-BLUP: <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-38.gif"/></noscript></span></span> and model = OLS: <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-39.gif"/></noscript></span></span> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1/FigureS1.pdf">Figure S1</a>). When RR-BLUP is used, there are diminishing returns in <em>r</em><sub>A</sub> by increasing <em>r</em> = 1 to <em>r</em> = 2 as <em>n</em> is increased, but only after <em>n</em> exceeds 300. Even then, <em>r</em><sub>A</sub> for <em>r</em> = 1 approaches that of <em>r</em> = 2 only very slowly, and a difference of 0.05 is still observed at <em>n</em> = 700.</p><p id="p-26">One thing that is apparent in these results is that RR-BLUP achieves much greater accuracies at all levels of <em>n</em>, but the difference is much greater at relatively small values of <em>n</em>. When <em>n</em> = 100, RR-BLUP is more than 34% more accurate than the OLS model, but when <em>n</em> &gt; 500, RR-BLUP is only 15% more accurate than the OLS model. This property of RR-BLUP indicates that resource allocation decisions will differ between traditional marker-assisted selection and genomic selection.</p></div><div id="sec-8" class="subsection"><h3>Effect of tradeoff between population size and replication</h3><p id="p-27">In reality, breeding programs have budgets to contend with. Budgets of 250 and 500 field plots per single DH population were considered. I assumed different genotyping costs (C), expressed in field plot equivalents of 0, 0.50, and 1. A situation in which genotyping is free (<em>i.e.</em>, C = 0) could be where a population was genotyped for another reason not related to the breeding program, or a futuristic scenario where genotyping technologies have advanced to the point of <em>in situ</em> genotyping assays, for example. BayesC<em>π</em> was no longer considered for subsequent analyses because it performed so similarly to RR-BLUP, and it was decided to focus the results on one model.</p><p id="p-28">When B = 500, prediction accuracy measured by cross validation was always maximized at <em>r</em> =1 and the greatest value of <em>n</em> for OLS (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). This was the case for both <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-41.gif"/></noscript></span></span> for all levels of C. Large differences in accuracy were observed between <em>r</em> =1 and <em>r</em> =4. Prediction accuracy of RR-BLUP, on the other hand, responded very little to allocating resources between <em>n</em> and <em>r</em>. For example, when C = 0, so that there is a direct tradeoff between level of replication and population size, <em>n</em> = 500:<em>r</em> = 1 resource allocation strategy produced a model only 4% more accurate than a <em>n</em> = 125:<em>r</em> = 4 resource allocation strategy, whereas the increase in accuracy was 67% for OLS. This essentially means that slightly greater accuracy with RR-BLUP can be achieved by replicating alleles across individuals, but the advantage isn’t nearly as great as it is for OLS. Following this, whenever C &gt; 0 so that genotyping costs reduce <em>n</em> disproportionately as <em>r</em> increases, the advantage of maximizing <em>n</em> quickly disappears. The RR-BLUP accuracies for all resource allocation strategies are remarkably similar, indicating greater flexibility in experimental design to maximize genomic selection accuracies. As can be seen in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>, rate of increase in genomic selection accuracy with increasing <em>n</em> tapers off above <em>n</em> =100, and increasing replication from <em>r</em> =1 to <em>r</em> =2 boosts accuracy by 0.09. The combination of these factors leads to almost an even tradeoff in accuracy between <em>n</em> and <em>r</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/481/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy (rA) for two statistical models as affected by tradeoffs between replication (r) and population size (n) for various levels of relative genotyping costs (C) expressed in field plot equivalents. Total budget is set to 500 field plot equivalents." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1660122557" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy (&lt;em&gt;r&lt;/em&gt;&lt;sub&gt;A&lt;/sub&gt;) for two statistical models as affected by tradeoffs between replication (&lt;em&gt;r&lt;/em&gt;) and population size (&lt;em&gt;n&lt;/em&gt;) for various levels of relative genotyping costs (C) expressed in field plot equivalents. Total budget is set to 500 field plot equivalents.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/481/F2.medium.gif" width="154" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/481/F2.medium.gif" width="154" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/481/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/481/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462534" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-29" class="first-child">Prediction accuracy (<em>r</em><sub>A</sub>) for two statistical models as affected by tradeoffs between replication (<em>r</em>) and population size (<em>n</em>) for various levels of relative genotyping costs (C) expressed in field plot equivalents. Total budget is set to 500 field plot equivalents.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The situation is only slightly different for B = 250 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1/FigureS2.pdf">Figure S2</a>). When C = 0 and <em>n</em> can be as large as 250 so that <em>r</em> =1, a big boost in accuracy is observed for OLS. Whenever <em>n</em> is less than this for other resource allocation strategies, OLS performs quite poorly. As when B = 500, RR-BLUP is much less sensitive to tradeoffs between <em>n</em> and <em>r</em> compared to OLS. Differences in accuracy are greater, however, for B = 250. This is likely due to the fact that values of <em>r</em> greater than two force <em>n</em> to be smaller than 100, which substantially reduces accuracy (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>).</p></div><div id="sec-9" class="subsection"><h3>Comparison between theoretical prediction accuracies and observed</h3><p id="p-31">Theoretical prediction accuracies follow a very similar trajectory as that observed in the simulation (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) and are only slightly higher than the accuracy observed in the simulation, being on average 0.026 higher for <em>r</em> = 1 and 0.013 greater for <em>r</em> = 2. A key use of the theoretical accuracy calculations would be for resource allocations decisions. The <em>n</em> and <em>r</em> choices according to values of C = {0, 0.5, 1} and B = {250, 500} discussed above were used in the formula derived by <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> (2008)</a>. The resource allocation strategy (<em>i.e.</em>, <em>n vs. r</em>) predicted to give the greatest accuracy was compared with that observed in the simulation. The predicted optimal resource allocation strategy largely agreed with those observed in the simulations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Only in three of the 12 instances was there a disagreement, and this often occurred when the difference between different levels of <em>n</em>:<em>r</em> was vanishingly small (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). These results suggest that the formula derived by <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> (2008)</a> can be used to guide genomic selection strategies in plant breeding, thereby avoiding time consuming efforts in data mining and computer simulations.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/481/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between calculated prediction accuracy (dashed lines) and observed prediation accuracy (solid lines) in simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1660122557" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison between calculated prediction accuracy (dashed lines) and observed prediation accuracy (solid lines) in simulations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/481/F3.medium.gif" width="440" height="382"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/481/F3.medium.gif" width="440" height="382"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/481/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/481/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462571" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-32" class="first-child">Comparison between calculated prediction accuracy (dashed lines) and observed prediation accuracy (solid lines) in simulations.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462564/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462564/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Population size and number of replication numbers achieving the greatest prediction accuracy according to theoretical calculations and simulations for different resource allocation situations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Effect of varying n and r on multiple cycles of selection</h3><p id="p-35">Resource allocation strategies were applied to generating phenotypic data for model training. One goal of this study was to determine how different strategies, constrained by budgets, affected prediction accuracy not only in Cycle 0 but also the prediction of progenies in cycles 1 to 3. To isolate prediction accuracy from the confounding effect of shifting genetic variances caused by different selection intensities and changes in allele frequency, a model was trained in Cycle 0 followed by randomly mating the population for three generations. Models trained in Cycle 0 were applied to progenies in Cycles 1−3. No change in rank across cycles of recombination was observed for the different resource allocation strategies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). For example, when C = 0, a <em>n</em>:<em>r</em> ratio of 500:1 was optimal across all cycles; when C = 1, a <em>n</em>:<em>r</em> ratio of 125:3 was optimal across all cycles. The exact same finding of no crossover in rank across cycles was observed for all other scenarios (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1/FigureS3.pdf">Figure S3</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/481/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy (rA) for each relative genotyping cost and resource allocation strategy across generations of random mating (Cycle). Population sizes corresponding to each level of r can be observed in Figure 2. Total budget was set to 500 field plot equivalents. (A) Heritability of single plot measurements set to 0.20. (B) Heritability of single plot measurements set to 0.60. Average standard error of prediction accuracies was 0.005 and ranged from 0.002 to 0.007." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1660122557" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy (&lt;em&gt;r&lt;/em&gt;&lt;sub&gt;A&lt;/sub&gt;) for each relative genotyping cost and resource allocation strategy across generations of random mating (Cycle). Population sizes corresponding to each level of &lt;em&gt;r&lt;/em&gt; can be observed in Figure 2. Total budget was set to 500 field plot equivalents. (A) Heritability of single plot measurements set to 0.20. (B) Heritability of single plot measurements set to 0.60. Average standard error of prediction accuracies was 0.005 and ranged from 0.002 to 0.007.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/481/F4.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/481/F4.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/481/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/481/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462484" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-36" class="first-child">Prediction accuracy (<em>r</em><sub>A</sub>) for each relative genotyping cost and resource allocation strategy across generations of random mating (Cycle). Population sizes corresponding to each level of <em>r</em> can be observed in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. Total budget was set to 500 field plot equivalents. (A) Heritability of single plot measurements set to 0.20. (B) Heritability of single plot measurements set to 0.60. Average standard error of prediction accuracies was 0.005 and ranged from 0.002 to 0.007.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Genetic gain is a function of selection intensity and genetic variance in addition to selection accuracy. The relative genetic gain of two methods for performing the first cycle of selection was evaluated: selecting C0 progenies on the basis of phenotypic value alone <em>vs.</em> selection based on marker scores. The latter holds potential to be advantageous because of extensive replication at the allelic level, and therefore prediction accuracy will benefit from larger populations, whereas phenotypic selection does not. <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Bernardo and Yu (2007)</a>, however, reported that simulated selection based on phenotypic value alone in the first cycle was at least as successful as an index value combining phenotypic values and marker scores. In the simulations performed for this study, selection on marker scores alone in cycle 0 led to greater genetic gain than selection on phenotypic values or GEBV-phenotype indices (results not shown), and therefore all selections made in cycle 0 were based on marker scores alone.</p><p id="p-38">Three options were considered for phenotyping: (1) phenotype 100% of DH lines, (2) phenotype only 75% of DH lines, (3) phenotype only 50% of DH lines. The latter two options were chosen to save resources for phenotyping, thereby allocating more resources to genotyping larger populations. Total population sizes and number of DH lines phenotyped at four levels of replication are displayed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. Surprisingly, phenotyping only a fraction of the DH population provided no benefit (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a>−<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">5</a>). This was largely because only a limited increase in selection intensity was achieved, as well as the fact that lines phenotyped and included in the training dataset were predicted with higher accuracy than lines not phenotyped.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462591/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462591/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Response and prediction accuracy of several resource allocation and phenotyping strategies for a budget of 250 total plot units, relative genotyping cost of 0.50, and plot heritability of 0.20 (<em>i.e.</em>, B = 250, C = 0.50, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T2/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T2/embed/mml-math-44.gif"/></noscript></span></span>=0.20)</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462518/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462518/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Response and prediction accuracy of several resource allocation and phenotyping strategies for a budget of 250 total plot units, relative genotyping cost of 1, and plot heritability of 0.20 (<em>i.e.</em>, B = 250, C = 1, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T3/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T3/embed/mml-math-45.gif"/></noscript></span></span>= 0.20)</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462604/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462604/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Response and prediction accuracy of several resource allocation and phenotyping strategies for a budget of 500 total plot units, relative genotyping cost of 0.50, and plot heritability of 0.20 (<em>i.e.</em>, B = 500, C = 0.50, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T4/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T4/embed/mml-math-46.gif"/></noscript></span></span>=0.20)</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462476/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462476/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Response and prediction accuracy of several resource allocation and phenotyping strategies for a budget of 500 total plot units, relative genotyping cost of 1, and plot heritability of 0.20 (<em>i.e.</em>, B = 500, C = 1, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T5/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/T5/embed/mml-math-47.gif"/></noscript></span></span>=0.20)</span><div class="sb-div caption-clear"></div></div></div><p id="p-59">With respect to comparing genetic gain across resource allocation strategies, results similar to those for prediction accuracy were found. That is, a degree of flexibility exists with equivalent genetic gains being observed over a number of different strategies. For example, when B = 250 and C = 0.50, no difference was observed between <em>n</em> = 167:<em>r</em> = 1 and <em>n</em> = 100:<em>r</em> = 2 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The same was observed for B = 500 and both levels of C. When B = 250 and C = 1, however, the greatest genetic gain was observed for <em>n</em> = 62 and <em>r</em> = 3. The small population size here was offset by the increase in accuracy through additional replication. In summary, if genotyping is relatively inexpensive (<em>i.e.</em>, half the cost of phenotyping a single experimental unit), one or two replications can be used without any difference in genetic gain. Allocating more resources to additional replication is not advisable. If genotyping costs at least as much as phenotyping one experimental unit, replication is beneficial, especially when budgets are small. If the budget is large, several different strategies can be used without sacrificing total genetic gain. It appears increases in selection intensity through larger population sizes are offset by increases in prediction accuracy through more precise phenotyping.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-60">Genomic selection can be incorporated into a breeding program in several different ways including, but not limited to, predicting and selecting within biparental populations (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Bernardo and Yu 2007</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Heffner <em>et al.</em> 2011</a>), across multiple related biparental populations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>) and among a panel of inbred lines or accessions (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Riedelsheimer <em>et al.</em> 2012</a>). When genomic selection is applied to any of these populations, GEBVs can be used to select progenies genotyped but not phenotyped, or GEBVs can augment or replace phenotypes if they are more predictive of true breeding values. The latter may be advantageous if large populations and a high degree of allelic replication provide highly accurate allelic effect estimates (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gimelfarb and Lande 1994</a>). In either case, a key question is whether resources should be used to maximize population size or if some resources should be allocated to replication in order to increase heritability.</p><p id="p-61">This simulation study on a single biparental population of DH lines showed that when the objective is to predict nonphenotyped progenies, plot-based heritability is low, and cost of genotyping is at least 50% that of phenotyping an experimental unit, RR-BLUP prediction accuracy slightly benefits from replication. When genotyping is free or <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-48.gif"/></noscript></span></span> is high, allocating resources to a larger population maximizes prediction accuracy. The differences between resource allocation strategies, however, were quite small across scenarios. This stands in contrast to what has been previously reported for QTL detection power (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Knapp and Bridges 1990</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Schon <em>et al.</em> 2004</a>), as well as what was found for a conventional marker-assisted selection approach tested in the present study. When marker-based selection was carried out by using OLS estimation and a model selection algorithm, prediction accuracy was clearly and consistently maximized when all resources were allocated to maximizing <em>n</em> and minimizing <em>r</em>. <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> shows that accuracy of OLS and RR-BLUP responded to replication approximately equally. A large difference between the two models, especially when <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3/481/embed/mml-math-49.gif"/></noscript></span></span>= 0.20, is that OLS accuracy didn’t respond until <em>n</em> exceeded 125, whereas RR-BLUP accuracy increased between <em>n</em> = 25 and <em>n</em> = 125, and actually began to level off just above <em>n</em> = 125. Accuracy of OLS continued to steadily increase until <em>n</em> was greater than 300. Because <em>n</em> was always less than or equal to 250 when <em>r</em> =2, boosting <em>n</em> by decreasing <em>r</em> provided a substantial benefit in accuracy to OLS, whereas RR-BLUP benefited less at these larger population sizes.</p><p id="p-62">It is well known that OLS estimates of marker effects and model selection algorithms suffer from inability to explain sufficient proportions of genetic variance for complex traits. This is primarily caused by lack of statistical power to detect small-effect QTL comprising genetic variation of complex, agronomically important traits. Power is especially low when population size is relatively small. In fact, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Utz <em>et al.</em> (2000)</a> found that power was insufficient for detecting grain yield QTL even at population sizes of 300. Compounding the problem of insufficient power is upward bias of QTL effect estimation, which is inversely related to power (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Beavis 1994</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Melchinger <em>et al.</em> 1998</a>). As population size decreases, both QTL detection power diminishes and estimation bias increases, resulting in marker-assisted selection model of dramatically reduced prediction accuracy. An RR-BLUP genomic selection model, on the other hand, does not suffer from lack of statistical power because all marker effects are estimated and used for prediction. Population size does affect marker effect estimation accuracy and thus genomic prediction accuracy (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lorenz <em>et al.</em> 2012</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Heffner <em>et al.</em> 2011</a>), but apparently not as much as power and estimation accuracy of OLS MAS models.</p><p id="p-63">A lack of rank changes among resource allocation strategies across generations of random mating was unexpected. It was expected that a resource allocation strategy better able to separate markers in terms of their effects and thus give greater weight to markers more closely linked to QTL would perform better on progenies several generations of random mating removed from the training population. It was hypothesized that maximizing population size, thus creating more recombination events for increased resolution, would always produce a model with superior prediction accuracies on progenies several generations removed from the training population. This was observed by <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Zhong <em>et al.</em> (2009)</a>, where more resources allocated to replication improved prediction when the training and validation populations were only one generation removed, but allocating more resources to larger population size resulted in greater prediction accuracy when training and validation populations were separated by four generations of random mating. These authors used a different population structure (several related biparental populations derived from actual barley genotypes) and did not account for genotyping cost. Another example of rank changes across generations of random mating is included in <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Habier <em>et al.</em> (2007)</a>, where different genomic selection models performed similarly when training and validation populations were removed by one or less generations, but BayesB performed markedly better as the number of generations between training and validation populations increased. This was because BayesB put more weight on markers more closely linked to QTL. Despite these theoretical expectations, I observed no change in rank across generations, indicating that a particular resource allocation strategy will be superior across at least three generations. One explanation is the high level of linkage disequilibrium (LD) between markers and QTL and an insufficient number of generations to break up linkages, although predictions accuracies generally declined as expected. The number of effective factors was 28, but the number of simulated QTL was 100, resulting in a situation where QTL are expected to be in high LD with several markers.</p><p id="p-64">With respect to total genetic gain, simulations under the budgets and genotyping:phenotyping cost ratios considered in this study again indicated a degree of flexibility in terms of resource allocation. In three of the four cases studied (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Tables 2</a>−<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">5</a>), multiple strategies were not statistically different from one another in terms of genetic gain. In each of these three cases, maximizing population size and phenotyping 100% of the DH lines resulted in at least as much genetic gain as any other strategy. When the budget was relatively small and genotyping cost was relatively high, it was beneficial to conduct three replications to increase heritability. Phenotyping less than 100% of the DH lines that were genotyped to save resources provided little benefit. It appears this is because prediction accuracy of lines included in the training dataset was always substantially higher than those not included. To my knowledge, in no other studies have authors specifically investigated this in the context of either genomic selection or marker-assisted selection, so an attempt at an intuitive explanation is warranted. Prediction accuracy is partly a function of LD between markers and QTL. Linkage disequilibrium between markers and QTL not physically linked can be generated by random genetic drift, or sampling. Because we are dealing with training populations of finite size, the sampling effect can generate slight LD between unlinked markers and QTL. This source of LD is useful for purposes of prediction within the same sample of individuals that generated that spurious LD. However, when marker effects are applied to individuals outside the original sample of individuals, the same exact spurious pattern of LD between unlinked markers and QTL no longer exists. Rather, this source of LD just generates noise, reducing marker effect estimation accuracy Increasing the heritability of measurements increases the accuracy derived from this source of LD when predictions are applied within the training population. However, when predictions are applied outside the original sample, increasing population size has a larger effect because it reduces noise caused by spurious LD (J-L. Jannink, personal communication).</p><p id="p-65">It is acknowledged that one expense not considered for allocating resources was cost of developing DH lines, which is certainly a considerable cost for breeding programs. One reason this expense was not considered is because of uncertainty in its relative cost and greatly varying costs between organizations. Nevertheless, the finding of little benefit to phenotyping only a subset of the genotyped population would not change, and actually factoring in cost of DH line development would only make phenotyping the entire population more beneficial.</p><p id="p-66">An important factor not considered in this study, and all studies on resource allocation for MAS, is genotype-by-environment interaction. The simulations carried out implicitly assume that the phenotypic data comprising the training dataset was collected in an environment perfectly representative of the target environment. In an actual breeding program, multiple locations and possibly multiple years are needed so allelic effects are estimated across a set of environments representative of those in which the cultivar is grown in subsequent seasons. Results reported herein pertain to the effect of replication within any given environment. The effect of genotype-by-environment interaction on the allocation of resources across environments needs further study using data from actual field trials. For example, is it more efficient to evaluate 100 different progenies in each of five environments for a total population size of 500, or evaluate 100 common progenies at five environments? The former strategy allows for much larger population sizes, but each progeny is only evaluated in one environment. Given that alleles are shared across progenies, the strategy of spreading populations across environments may be more effective.</p><p id="p-67">The main finding of this simulation study applicable to plant breeding programs is the flexibility found in resource allocation in genomic selection for highly polygenic traits. Unlike with traditional MAS approaches, where clearly population size should be maximized, genomic prediction accuracy and total genetic gain are less affected by allocating more resources to replication for low heritability traits like grain yield, and in some cases increased replication results in the maximum genetic gain. It was also shown that the deterministic calculations for prediction accuracy (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> 2008</a>) are useful for comparing resource allocation strategies in biparental populations.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">The motivation for this study originated in discussions with Jode Edwards. I am also indebted to Dr. Edwards for providing critical feedback as the study progressed. This manuscript was further improved by valuable input from Jean-Luc Jannink and Natalia de Leon. An R script for the BayesC<em>π</em> algorithm was obtained from the 2009 Iowa State Animal Breeding and Genetics short course titled “Use of High-Density SNP Genotyping for Genetic Improvement of Livestock.”</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004911/-/DC1</a></p></li><li class="fn" id="fn-24"><p id="p-69"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-02"><span class="received-label">Received </span>November 2, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-01-03"><span class="accepted-label">Accepted </span>January 3, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Lorenz</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.1" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-based prediction of testcross values in maize</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.3.481.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The power and deceit of QTL experiments: lessons from comparative QTL studies</span>, pp. <span class="cit-fpage">250</span>–<span class="cit-lpage">265</span> in <span class="cit-source">Proceedings of the 49th Annual Corn and Sorghum Research Conference</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. B.</span></span>. <span class="cit-publ-name">American Seed Trade Association</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.3" data-doi="10.2135/cropsci2008.03.0131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular markers and selection for complex traits in plants: learning from the last 20 years</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1649%26rft.epage%253D1664%26rft.atitle%253DMolecular%2BMarkers%2Band%2BSelection%2Bfor%2BComplex%2BTraits%2Bin%2BPlants%253A%2BLearning%2Bfrom%2Bthe%2BLast%2B20%2BYears%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.03.0131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.03.0131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000259792100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.481.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Breeding for Quantitative Traits in Plants</span>. <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D47%26rft.spage%253D1090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of predicting the genetic risk of disease using a genome-wide approach</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D3%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.7" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP</span>. <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gimelfarb</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Simulation of marker assisted selection in hybrid populations</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DGimelfarb%26rft.auinit1%253DA.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D47%26rft.atitle%253DSimulation%2Bof%2Bmarker%2Bassisted%2Bselection%2Bin%2Bhybrid%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F8206366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8206366&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NE88600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.9" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.10" data-doi="10.1111/j.1439-0388.2011.00964.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using the genomic relationship matrix to predict the accuracy of genomic selection</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D409%26rft.epage%253D421%26rft.atitle%253DUsing%2Bthe%2Bgenomic%2Brelationship%2Bmatrix%2Bto%2Bpredict%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00964.x%26rft_id%253Dinfo%253Apmid%252F22059574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00964.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059574&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.11" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.12" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.13" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.14" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.15" data-doi="10.2135/cropsci2011.05.0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy for grain quality traits in biparental wheat populations</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.05.0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.05.0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000295839200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.16" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.17" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.18" data-doi="10.2527/jas.2009-2064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic prediction of simulated multibreed and purebred performance using observed fifty thousand single nucleotide polymorphism genotypes</span>. <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">544</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-2064%26rft_id%253Dinfo%253Apmid%252F19820059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-2064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19820059&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274040700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bridges</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Using molecular markers to estimate quantitative trait locus parameters: Power and genetic variances for unreplicated and replicated progeny</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnapp%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D126%26rft.issue%253D3%26rft.spage%253D769%26rft.epage%253D777%26rft.atitle%253DUsing%2Bmolecular%2Bmarkers%2Bto%2Bestimate%2Bquantitative%2Btrait%2Blocus%2Bparameters%253A%2Bpower%2Band%2Bgenetic%2Bvariances%2Bfor%2Bunreplicated%2Band%2Breplicated%2Bprogeny.%26rft_id%253Dinfo%253Apmid%252F2249768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI2LzMvNzY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.20" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for Fusarium head blight resistance in six-row barley</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.21" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.481.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Quantitative trait locus (QTL) mapping using different testers and independent population samples in maize reveals low power of QTL detection and large bias in estimates of QTL effects</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMelchinger%26rft.auinit1%253DA.%2BE.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D383%26rft.epage%253D403%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2B%2528QTL%2529%2BMapping%2BUsing%2BDifferent%2BTesters%2Band%2BIndependent%2BPopulation%2BSamples%2Bin%2BMaize%2BReveals%2BLow%2BPower%2Bof%2BQTL%2BDetection%2Band%2BLarge%2BBias%2Bin%2BEstimates%2Bof%2BQTL%2BEffects%26rft_id%253Dinfo%253Apmid%252F9584111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ5LzEvMzgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemarie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Economic efficiency of one cycle of marker-assisted selection</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMoreau%26rft.auinit1%253DL.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D329%26rft.epage%253D337%26rft.atitle%253DEconomic%2BEfficiency%2Bof%2BOne%2BCycle%2Bof%2BMarker-Assisted%2BSelection%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000089410300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.25" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.26" data-doi="10.1534/genetics.167.1.485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Openshaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait locus mapping based on resampling in a vast maize testcross experiment and its relevance to quantitative genetics for complex traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchon%26rft.auinit1%253DC.%2BC.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D485%26rft.epage%253D498%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2BMapping%2BBased%2Bon%2BResampling%2Bin%2Ba%2BVast%2BMaize%2BTestcross%2BExperiment%2Band%2BIts%2BRelevance%2Bto%2BQuantitative%2BGenetics%2Bfor%2BComplex%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.485%26rft_id%253Dinfo%253Apmid%252F15166171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzEvNDg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Bias and sampling error of the estimated proportion of genotypic variance explained by quantitative trait loci determined from experimental data in maize using cross validation and validation with independent samples</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1839</span>–<span class="cit-lpage">1849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DUtz%26rft.auinit1%253DH.%2BF.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1839%26rft.epage%253D1849%26rft.atitle%253DBias%2Band%2BSampling%2BError%2Bof%2Bthe%2BEstimated%2BProportion%2Bof%2BGenotypic%2BVariance%2BExplained%2Bby%2BQuantitative%2BTrait%2BLoci%2BDetermined%2BFrom%2BExperimental%2BData%2Bin%2BMaize%2BUsing%2BCross%2BValidation%2Band%2BValidation%2BWith%2BIndependent%2BSamples%26rft_id%253Dinfo%253Apmid%252F10866652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE4MzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.481.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wricke</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">W. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-source">Quantitative Genetics and Selection in Plant Breeding</span>. <span class="cit-publ-name">Walter de Gruyter &amp; Co.</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.29" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D539%26rft.epage%253D551%26rft.atitle%253DGenetic%2BDesign%2Band%2BStatistical%2BPower%2Bof%2BNested%2BAssociation%2BMapping%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.481.30" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/3/481.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/465" title="Unequal Recombination and Evolution of the Mating-Type (MAT) Loci in the Pathogenic Fungus Grosmannia clavigera and Relatives" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/493" title="Genetic Interactions of Arabidopsis thaliana Damaged DNA Binding Protein 1B (DDB1B) With DDB1A, DET1, and COP1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/3"><span>Volume 3 Issue 3, March 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/3.cover-source.jpg?itok=w5x3d6rp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/481.short" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4k3IPguJza7NAI28BudK-x-QkGTygblKZGNjuccF97Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/3/481" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/3/481" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/3/481" class="active" data-icon-position="" data-hide-link-title="0">Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459429" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NWQAspSrB774twg9ETt_1yTI3-afHrUt5b5VpwNxg5Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/481.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/3/481" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-UdfIqawYn4JZUto2DBIevlLfFv1RB7hwwYAEu73Yy78" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/3/481" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/481.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459429" id="node459429--2" data-pisa="ggg;3/3/481" data-pisa-master="ggg;3/3/481" data-apath="/ggg/3/3/481.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">481-491;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004911 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459429--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alorenz2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">alorenz2@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/3/481" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/3/481&amp;title=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/3/481&amp;title=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/3/481&amp;title=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/3/481&amp;text=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/3/481&amp;title=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/3/481&amp;t=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/3/481&amp;title=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/3/481&amp;title=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459429/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459429" id="node459429--3" data-pisa="ggg;3/3/481" data-pisa-master="ggg;3/3/481" data-apath="/ggg/3/3/481.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">481-491;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004911 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459429--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alorenz2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">alorenz2@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=481&amp;rft.epage=491&amp;rft.atitle=Resource%20Allocation%20for%20Maximizing%20Prediction%20Accuracy%20and%20Genetic%20Gain%20of%20Genomic%20Selection%20in%20Plant%20Breeding%3A%20A%20Simulation%20Experiment&amp;rft.volume=3&amp;rft.issue=3&amp;rft.date=2013-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lorenz%2C+Aaron+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459429/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459429/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459429/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459429/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459429/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459429/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459429/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459429/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459429/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459429/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459429/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459429/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/3/481" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/3/481.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/3/481.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/3/481.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/3/481.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
