<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/3/399" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/3/399.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/3/399.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/3/399.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/3/399.atom" />
<meta name="HW.pisa" content="ggg;3/3/399" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="On the Mutational Topology of the Bacterial Genome" />
<meta name="DC.Identifier" content="10.1534/g3.112.005355" />
<meta name="DC.Date" content="2013-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 Foster et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="By sequencing the genomes of 34 mutation accumulation lines of a mismatch-repair defective strain of Escherichia coli that had undergone a total of 12,750 generations, we identified 1625 spontaneous base-pair substitutions spread across the E. coli genome. These mutations are not distributed at random but, instead, fall into a wave-like spatial pattern that is repeated almost exactly in mirror image in the two separately replicated halves of the bacterial chromosome. The pattern is correlated to genomic features, with mutation densities greatest in regions predicted to have high superhelicity. Superimposed upon this pattern are regional hotspots, some of which are located where replication forks may collide or be blocked. These results suggest that, as they traverse the chromosome, the two replication forks encounter parallel structural features that change the fidelity of DNA replication." />
<meta name="DC.Contributor" content="Patricia L. Foster" />
<meta name="DC.Contributor" content="Andrew J. Hanson" />
<meta name="DC.Contributor" content="Heewook Lee" />
<meta name="DC.Contributor" content="Ellen M. Popodi" />
<meta name="DC.Contributor" content="Haixu Tang" />
<meta name="article:published_time" content="2013-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="On the Mutational Topology of the Bacterial Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;By sequencing the genomes of 34 mutation accumulation lines of a mismatch-repair defective strain of &lt;i&gt;Escherichia coli&lt;/i&gt; that had undergone a total of 12,750 generations, we identified 1625 spontaneous base-pair substitutions spread across the &lt;i&gt;E. coli&lt;/i&gt; genome. These mutations are not distributed at random but, instead, fall into a wave-like spatial pattern that is repeated almost exactly in mirror image in the two separately replicated halves of the bacterial chromosome. The pattern is correlated to genomic features, with mutation densities greatest in regions predicted to have high superhelicity. Superimposed upon this pattern are regional hotspots, some of which are located where replication forks may collide or be blocked. These results suggest that, as they traverse the chromosome, the two replication forks encounter parallel structural features that change the fidelity of DNA replication.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/03/01" />
<meta name="citation_mjid" content="ggg;3/3/399" />
<meta name="citation_id" content="3/3/399" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/3/399" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/3/399.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/3/399.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/3/399.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005355" />
<meta name="citation_pmid" content="23450823" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="399" />
<meta name="citation_lastpage" content="407" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Patricia L. Foster" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_email" content="plfoster@indiana.edu" />
<meta name="citation_author" content="Andrew J. Hanson" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Heewook Lee" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Ellen M. Popodi" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Haixu Tang" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Agier;citation_author=G. Fischer;citation_title=The mutational profile of the yeast genome is shaped by replication;citation_volume=29;citation_year=2012;citation_pmid=22114361;citation_doi=10.1093/molbev/msr280" />
<meta name="citation_reference" content="citation_title=Second International Symposium on Information Theory;citation_title=Information theory and the extension of the maximum likelihood principle;citation_author=H. Akaike;citation_pages=267-281;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=T. E. Allen;citation_author=M. J. Herrgard;citation_author=M. Liu;citation_author=Y. Qiu;citation_author=J. D. Glasner;citation_title=Genome-Scale Analysis of the Uses of the Escherichia coli Genome: Model-Driven Analysis of Heterogeneous Data Sets;citation_pages=6392-6399;citation_volume=185;citation_year=2003;citation_issue=21;citation_pmid=14563874;citation_doi=10.1128/JB.185.21.6392-6399.2003" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=T. E. Allen;citation_author=N. D. Price;citation_author=A. R. Joyce;citation_author=B. O. Palsson;citation_title=Long-range periodic patterns in microbial genomes indicate significant multi-scale chromosomal organization.;citation_pages=e2-e2;citation_volume=2;citation_year=2006;citation_issue=1;citation_pmid=16410829;citation_doi=10.1371/journal.pcbi.0020002" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Benzer;citation_title=ON THE TOPOGRAPHY OF THE GENETIC FINE STRUCTURE;citation_pages=403-415;citation_volume=47;citation_year=1961;citation_issue=3;citation_pmid=16590840;citation_doi=10.1073/pnas.47.3.403" />
<meta name="citation_reference" content="citation_journal_title=EMBO Reports;citation_journal_abbrev=EMBO Reports;citation_author=M. Berger;citation_author=A. Farcas;citation_author=M. Geertz;citation_author=P. Zhelyazkova;citation_author=K. Brix;citation_title=Coordination of genomic structure and transcription by the main bacterial nucleoid-associated protein HU.;citation_pages=59-64;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=20010798;citation_doi=10.1038/embor.2009.232" />
<meta name="citation_reference" content="citation_journal_title=EMBO Reports;citation_journal_abbrev=EMBO Reports;citation_author=N. Blot;citation_author=R. Mavathur;citation_author=M. Geertz;citation_author=A. Travers;citation_author=G. Muskhelishvili;citation_title=Homeostatic regulation of supercoiling sensitivity coordinates transcription of the bacterial genome.;citation_pages=710-715;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16799466;citation_doi=10.1038/sj.embor.7400729" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=L. Boe;citation_title=Mechanism for induction of adaptive mutations in Escherichia coli.;citation_pages=597-601;citation_volume=4;citation_year=1990;citation_issue=4;citation_pmid=2191182;citation_doi=10.1111/j.1365-2958.1990.tb00628.x" />
<meta name="citation_reference" content="citation_journal_title=Structure (London);citation_journal_abbrev=Structure (London);citation_author=W. K. Cho;citation_author=C. Jeong;citation_author=D. Kim;citation_author=M. Chang;citation_author=K. M. Song;citation_title=ATP alters the diffusion mechanics of MutS on mismatched DNA.;citation_pages=1264-1274;citation_volume=20;citation_year=2012;citation_issue=7;citation_pmid=22682745;citation_doi=10.1016/j.str.2012.04.017" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=I. G. Duggin;citation_author=S. D. Bell;citation_title=Termination structures in the Escherichia coli chromosome replication fork trap.;citation_pages=532-539;citation_volume=387;citation_year=2009;citation_issue=3;citation_pmid=19233209;citation_doi=10.1016/j.jmb.2009.02.027" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. F. Elena;citation_author=R. E. Lenski;citation_title=Evolution experiments with microorganisms: the dynamics and genetic bases of adaptation.;citation_pages=457-469;citation_volume=4;citation_year=2003;citation_issue=6;citation_pmid=12776215;citation_doi=10.1038/nrg1088" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=I. J. Fijalkowska;citation_author=R. M. Schaaper;citation_author=P. Jonczyk;citation_title=DNA replication fidelity in Escherichia coli: a multi-DNA polymerase affair;citation_volume=36;citation_year=2012;citation_pmid=22404288;citation_doi=10.1111/j.1574-6976.2012.00338.x" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=D. L. Halligan;citation_author=P. D. Keightley;citation_title=Spontaneous mutation accumulation studies in evolutionary genetics;citation_pages=151-172;citation_volume=40;citation_year=2009;citation_doi=10.1146/annurev.ecolsys.39.110707.173437" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. E. Hudson;citation_author=U. Bergthorsson;citation_author=H. Ochman;citation_title=Transcription increases multiple spontaneous point mutations in Salmonella enterica;citation_pages=4517-4522;citation_volume=31;citation_year=2003;citation_issue=15;citation_pmid=12888512;citation_doi=10.1093/nar/gkg651" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. S. Jeong;citation_author=J. Ahn;citation_author=A. B. Khodursky;citation_title=Spatial patterns of transcriptional activity in the chromosome of Escherichia coli.;citation_pages=R86-R86;citation_volume=5;citation_year=2004;citation_issue=11;citation_pmid=15535862;citation_doi=10.1186/gb-2004-5-11-r86" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. T. Kibota;citation_author=M. Lynch;citation_title=Estimate of the genomic mutation rate deleterious to overall fitness in E. coli.;citation_pages=694-696;citation_volume=381;citation_year=1996;citation_issue=6584;citation_pmid=8649513;citation_doi=10.1038/381694a0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Lee;citation_author=E. Popodi;citation_author=H. Tang;citation_author=P. L. Foster;citation_title=Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing;citation_volume=109;citation_year=2012;citation_pmid=22991466;citation_doi=10.1073/pnas.1210309109" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=F. J. Lopez de Saro;citation_author=M. G. Marinus;citation_author=P. Modrich;citation_author=M. OâDonnell;citation_title=The beta sliding clamp binds to multiple sites within MutL and MutS;citation_volume=281;citation_year=2006;citation_pmid=16546997;citation_doi=10.1074/jbc.M601264200" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. Louarn;citation_author=F. Cornet;citation_author=V. Francois;citation_author=J. Patte;citation_author=J. M. Louarn;citation_title=Hyperrecombination in the terminus region of the Escherichia coli chromosome: possible relation to nucleoid organization.;citation_pages=7524-7531;citation_volume=176;citation_year=1994;citation_issue=24;citation_pmid=8002576;citation_doi=10.1128/jb.176.24.7524-7531.1994" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=J. B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Malik;citation_author=A. Bensaid;citation_author=J. Rouviere-Yaniv;citation_author=K. Drlica;citation_title=Histone-like protein HU and bacterial DNA topology: suppression of an HU deficiency by gyrase mutations.;citation_pages=66-76;citation_volume=256;citation_year=1996;citation_issue=1;citation_pmid=8609614;citation_doi=10.1006/jmbi.1996.0068" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=M. G. Marinus;citation_title=DNA methylation and mutator genes in Escherichia coli K-12.;citation_pages=71-76;citation_volume=705;citation_year=2010;citation_issue=2;citation_pmid=20471491;citation_doi=10.1016/j.mrrev.2010.05.001" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=I. Martincorena;citation_author=A. S. Seshasayee;citation_author=N. M. Luscombe;citation_title=Evidence of non-random mutation rates suggests an evolutionary risk management strategy.;citation_pages=95-98;citation_volume=485;citation_year=2012;citation_issue=7396;citation_pmid=22522932;citation_doi=10.1038/nature10995" />
<meta name="citation_reference" content="citation_title=A Short Course In Bacterial Genetics: A Laboratory Manual and Handbook for Escherichia coli and Related Bacteria;citation_author=J. H. Miller;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Mukai;citation_title=THE GENETIC STRUCTURE OF NATURAL POPULATIONS OF DROSOPHILA MELANOGASTER. I. SPONTANEOUS MUTATION RATE OF POLYGENES CONTROLLING VIABILITY;citation_pages=1-19;citation_volume=50;citation_year=1964;citation_issue=1;citation_pmid=14191352" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. Niki;citation_author=Y. Yamaichi;citation_author=S. Hiraga;citation_title=Dynamic organization of chromosomal DNA in Escherichia coli;citation_pages=212-223;citation_volume=14;citation_year=2000;citation_issue=2;citation_pmid=10652275;citation_doi=10.1101/gad.14.2.212" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. Ochman;citation_title=Neutral mutations and neutral substitutions in bacterial genomes;citation_volume=20;citation_year=2003;citation_pmid=12949125;citation_doi=10.1093/molbev/msg229" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. G. Pedersen;citation_author=L. J. Jensen;citation_author=S. Brunak;citation_author=H. H. Staerfeldt;citation_author=D. W. Ussery;citation_title=A DNA structural atlas for Escherichia coli.;citation_pages=907-930;citation_volume=299;citation_year=2000;citation_issue=4;citation_pmid=10843847;citation_doi=10.1006/jmbi.2000.3787" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. Pluciennik;citation_author=V. Burdett;citation_author=O. Lukianova;citation_author=M. OâDonnell;citation_author=P. Modrich;citation_title=Involvement of the beta clamp in methyl-directed mismatch repair in vitro;citation_volume=284;citation_year=2009;citation_pmid=19783657;citation_doi=10.1074/jbc.M109.054528" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Puigbo;citation_author=A. Romeu;citation_author=S. Garcia-Vallve;citation_title=HEG-DB: a database of predicted highly expressed genes in prokaryotic complete genomes under translational selection;citation_volume=36;citation_year=2008;citation_pmid=17933767;citation_doi=10.1093/nar/gkm831" />
<meta name="citation_reference" content="citation_title=Mathematical Statistics and Data Analysis;citation_author=J. A. Rice;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=MBio.;citation_author=M. A. Sanchez-Romero;citation_author=S. J. Busby;citation_author=N. P. Dyer;citation_author=S. Ott;citation_author=A. D. Millard;citation_title=Dynamic distribution of SeqA protein across the chromosome of Escherichia coli K-12;citation_pages=e00012-e10;citation_volume=1;citation_year=2010;citation_pmid=20689753" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. M. Schaaper;citation_author=R. L. Dunn;citation_title=Spectra of spontaneous mutations in Escherichia coli strains defective in mismatch correction: the nature of in vivo DNA replication errors.;citation_pages=6220-6224;citation_volume=84;citation_year=1987;citation_issue=17;citation_pmid=3306672;citation_doi=10.1073/pnas.84.17.6220" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Schneider;citation_author=R. Lurz;citation_author=G. Luder;citation_author=C. Tolksdorf;citation_author=A. Travers;citation_title=An architectural role of the Escherichia coli chromatin protein FIS in organising DNA;citation_pages=5107-5114;citation_volume=29;citation_year=2001;citation_issue=24;citation_pmid=11812843;citation_doi=10.1093/nar/29.24.5107" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Schuster-Bockler;citation_author=B. Lehner;citation_title=Chromatin organization is a major influence on regional mutation rates in human cancer cells.;citation_pages=504-507;citation_volume=488;citation_year=2012;citation_issue=7412;citation_pmid=22820252;citation_doi=10.1038/nature11273" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Biosyst;citation_author=V. F. Scolari;citation_author=B. Bassetti;citation_author=B. Sclavi;citation_author=M. C. Lagomarsino;citation_title=Gene clusters reflecting macrodomain structure respond to nucleoid perturbations.;citation_pages=878-888;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21165487;citation_doi=10.1039/c0mb00213e" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Sobetzko;citation_author=A. Travers;citation_author=G. Muskhelishvili;citation_title=Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle;citation_volume=109;citation_year=2012;citation_pmid=22184251;citation_doi=10.1073/pnas.1108229109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies;citation_volume=100;citation_year=2003;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Touchon;citation_author=C. Hoede;citation_author=O. Tenaillon;citation_author=V. Barbe;citation_author=S. Baeriswyl;citation_title=Organised genome dynamics in the Escherichia coli species results in highly diverse adaptive paths.;citation_pages=e1000344-e1000344;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19165319;citation_doi=10.1371/journal.pgen.1000344" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=A. Travers;citation_author=G. Muskhelishvili;citation_title=Bacterial chromatin.;citation_pages=507-514;citation_volume=15;citation_year=2005;citation_issue=5;citation_pmid=16099644;citation_doi=10.1016/j.gde.2005.08.006" />
<meta name="citation_reference" content="citation_journal_title=Biochimie;citation_author=D. Ussery;citation_author=T. S. Larsen;citation_author=K. T. Wilkes;citation_author=C. Friis;citation_author=P. Worning;citation_title=Genome organisation and chromatin structure in Escherichia coli;citation_pages=201-212;citation_volume=83;citation_year=2001;citation_pmid=11278070;citation_doi=10.1016/S0300-9084(00)01225-6" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=M. Valens;citation_author=S. Penaud;citation_author=M. Rossignol;citation_author=F. Cornet;citation_author=F. Boccard;citation_title=Macrodomain organization of the Escherichia coli chromosome;citation_volume=23;citation_year=2004;citation_pmid=15470498;citation_doi=10.1038/sj.emboj.7600434" />
<meta name="citation_reference" content="citation_author=P. Wessa" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Worth;citation_author=S. Clark;citation_author=M. Radman;citation_author=P. Modrich;citation_title=Mismatch repair proteins MutS and MutL inhibit RecA-catalyzed strand transfer between diverged DNAs.;citation_pages=3238-3241;citation_volume=91;citation_year=1994;citation_issue=8;citation_pmid=8159731;citation_doi=10.1073/pnas.91.8.3238" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="On the Mutational Topology of the Bacterial Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="twitter:description" content="By sequencing the genomes of 34 mutation accumulation lines of a mismatch-repair defective strain of Escherichia coli that had undergone a total of 12,750 generations, we identified 1625 spontaneous base-pair substitutions spread across the E. coli genome. These mutations are not distributed at random but, instead, fall into a wave-like spatial pattern that is repeated almost exactly in mirror image in the two separately replicated halves of the bacterial chromosome. The pattern is correlated to genomic features, with mutation densities greatest in regions predicted to have high superhelicity. Superimposed upon this pattern are regional hotspots, some of which are located where replication forks may collide or be blocked. These results suggest that, as they traverse the chromosome, the two replication forks encounter parallel structural features that change the fidelity of DNA replication." />
<meta name="og-title" property="og:title" content="On the Mutational Topology of the Bacterial Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/3/399" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="By sequencing the genomes of 34 mutation accumulation lines of a mismatch-repair defective strain of Escherichia coli that had undergone a total of 12,750 generations, we identified 1625 spontaneous base-pair substitutions spread across the E. coli genome. These mutations are not distributed at random but, instead, fall into a wave-like spatial pattern that is repeated almost exactly in mirror image in the two separately replicated halves of the bacterial chromosome. The pattern is correlated to genomic features, with mutation densities greatest in regions predicted to have high superhelicity. Superimposed upon this pattern are regional hotspots, some of which are located where replication forks may collide or be blocked. These results suggest that, as they traverse the chromosome, the two replication forks encounter parallel structural features that change the fidelity of DNA replication." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
    <title>On the Mutational Topology of the Bacterial Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"24USVRI1UZW8L1-kZd5aA2fKhfNSvP25fNEipY52jqw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459430","apath":"\/ggg\/3\/3\/399.atom","pisa":"ggg;3\/3\/399","ac":{"\/ggg\/3\/3\/399.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/3\/399.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/3\/399"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/3\/399"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/3\/399":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225N1FpzLB2Oz\\\\\\\/bHsDOvaXOb8+QVyXjqi2tci42JSznzTZKdrAKM\\\\\\\/0sYnfyIpB7dvpEDP\\\\\\\/7neVwU6wUn+Vp3rs17JIiyHXbc9ERPErvA+3z\\\\\\\/LecYd\\\\\\\/xUtE+niSxQwtVxDlsjeMSBZOL+m1DOacGJn\\\\\\\/ECuGjcmb+ZswQ7\\\\\\\/+zrX58WXCTyKFia0iHwMTGQPBJC23hAaY92YQdrUI2lD3FtIhl3uLCFv2BpxeCoYm+HcRxToJcZfntYzHISMKV0xIZSMe0I3d90ed3zlkTXrRqes2xOnDBQxrv1F4Cbcikj+2dycAzl0sj1m6Vw42widk\\\\\\\/WPv2FACvucT\\\\\\\/Tnm+2ZVUNzW\\\\\\\/G53bdcIgvRnMGM6KbNtRlZnfJStX8lzJbq3r41bg0w6nvgd21JqL3wpGcK8D6gAPJ7IgT17zsOlru4\\\\\\\/U709\\\\\\\/A1GN5+Q6o86NQvYLz4CSljDke9p592U3vNozWrDLDM+W4jf50U+YipDHkQCgGcukyZ4wql7DI7uQHDUVovPNOVHEXWamXuEqUDZeyDjHbnixzQvC3n\\\\\\\/ivTktk6ALgolN+DpfltPZj4dgMnvyp4iN98iO\\\\\\\/xHGtBwBJnNSZ8lk9\\\\\\\/po1zm8vpK2t2EAnafVtYyGkAYME5uNmwA4cCES6eDGbW97eWb0dfE5tPHDC5Mil9\\\\\\\/ZIWS9YaD0wQaAPA7VUT9\\\\\\\/QJ\\\\\\\/QH+4Sf82NQxyUoIzjHF6jP2DLNcw8kyLcnaz5XxL5gkUJymqvqsbgtjhsR88AJqu1gqp98CPmCeml7TilofnK5SzEN4S2WcEUtNlPswL6T4BqhDFTNgxn6ZpE8r8s7rVqQXvnJSrkgTwpttUz6QtoeStbZ87PN1+SgyVdhbMd96P7y4jgjjsYSXoxOoPvluYBmoj5wgrTS0Ey3dFbJmvYBcg3MuGHppwgqmhOUAbxW\\\\\\\/q8rkXSyvciAMUvrFVL8edcLgoaE11s3\\\\\\\/xNZ15BHYK\\\\\\\/L+QoHESUJeII1cacTZfrXnYYquC4SOScjIJjCuTqOtWuKrwYQR6aeD7CTd\\\\\\\/wXRbUeAdJHLFfIuZ6H6FRpdsdxHftVRGfhjfBIJJB81rcAzVOYZAFR3WKQKk\\\\\\\/9toSpIKyXMU7gYc5q1xCmG55dCvSR6X7QNH9rx3x+o4yPnpXk5JTz\\\\\\\/rwxhXSLfJ60dGSDG7Jwm8EiZM8locM8Q8mcVRY5jz8YZspiSmjJc4AdvRxjofz76Y48NiAv8ihlgvtxp5vybpdUC5G2juLwjoPr1td0oGJBZGzzzOzcokGs36cyu7TpKGWA91TTW8ZK6utwdK\\\u0022,\\\u0022iv\\\u0022:\\\u0022333HJSk\\\\\\\/gSltSZI97vFY9w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224648705dd33fa5fd8f4634437f62334301ac0886ef6a4c4720c2c3cf62fded10\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kaB8+BFO8TTNLyQU7au0CHSTX9VCa0M+asBKcY51zOrzU88P0CNEJOWZByxkNRXrjqlAAqx2+aGKihlkhg9PNRimlHMR4NSlQOb5PRzgK2cduOu06DejTSlCFiGOk6DhmHvncssv5Y6FozUsQRDlHo4BYBn9BvP+2Hm\\\\\\\/ZIITH2bGW45YpJCH2IawPDm\\\\\\\/ybVCDZfjFzXPxt+nToqcRReOEtNUG3VLsRTtyoWCvIMDIQx0szJAQaRuwuZTMbTZhUkuTcMIDz00t7evqfVhWk49LhSpyJ8vJF5sbbvi8VHYAFIvlclrHQAK0Ewef8vA5ldmWQ7mFrqVZ0E10kE+\\\\\\\/PNGwItOUBqC7\\\\\\\/kHlXJO5xX9oZBNbip51VVfe7uFqnCJkjvzfgL4ICQa6su4+2Vve7UIl0bzWZ0F5CUToWIE7Gx8YHSoVH6SM\\\\\\\/2cCKqJV4CKVGkSJc8FRIFbZ9uIt384q3Yt+fjC+CEIkXU4x3wbuSdaMUbXfzZtCPWOG6LKVk0mSEfCioB6TC52v0N7lG\\\\\\\/dUxT2jr9eL1a3bChXU7FrutebYqGChSxIarIRcXCdThRJYlrrhkgdBolzQS2Ytz+J9xf5vKtOzePY\\\\\\\/7AfwMdRs6pan\\\\\\\/hiGFHJTQUzGCmPTdtPr9iUtQ\\\\\\\/9GqyHZ49bQbdpTLkKWOuhSmj0ZqvzMbRXhy1qaRoAMAR0FoN0uzKsZqgNCVR9E5q1GM0ZCiBuTdKi5OU0XUzxdoZ9wdAGPRram66DnMPKsHmE4zkYju\\\\\\\/\\\\\\\/tNPusbKe9hfXUTxtbMXMOVCwGnUFfRt1IbTjgJDBlxVieN9fKt3oGFgxrNoaj6MXmnU13qm9aNJYGKY9V\\\\\\\/q88CbvKQv5a6XeSE1zl2CpQklrOD6HcLqJjM9Luf9TIudXs3Jx3V7hGYiP4A9zXQRiY+Z7l7a4sI6IOkGf\\\\\\\/\\\\\\\/dZ8aLl8Uv1Jgy6kA5hr1udC\\\\\\\/1RWQDzQV3qt1aW5voxH0uG3SyC8Y9yEdkXtiq7rPTjAKVJDK8kH9eXZlQBE9gh4bPw1bYkjKUjghqCYkXS3CtaAqsXofZVNyuUaDiIGb\\\\\\\/hvzB1HyLFdyL0n88MZuCtFwDh0K2REeAoSYuinG79VY\\\\\\\/y2hUOoRWC48omH91cOjktFH8YdYigFFvnecFOFY\\\\\\\/BPfKssfnhP\\\\\\\/j5PtjQZJwhx4ctixgrh4gWgUjTa4t6bHEtPKgUFwSnhhoIu85gzwcB9OW51MUwvc5lPA4tHOBc551utT+yBWAWwuTO2bIjlCX5FHyiujKdLNcWRU+EGe1ixmDatzuSORt1tC6RLVa82aTqG5ClFSeWLbiUAO9D6NnOWogFFte7sg0MA+veytwdsU4h32kSTp124FdVMSLPEt6UF\\\\\\\/jN8fXk1IqT6WS6NzLR5dw6omOYl9Ceh8u8Ufwr4ZlVP1xGovpRvuplgDdgs+POSTL4P52r514R8JqDyWqkjxFfKmSNq+OCQ7JjBY0zO\\\\\\\/1rald8pjgx6v8hDAist81TJxtD2EITar6ZOh9Ps7GwfO5U1nlFEv4YNuzJdG0zSZ8gO2L36p3sFmJb2U\\\\\\\/n4v2ODXEfHAXcGi8LJ7hM6PG86fzPJtn0FmEJ2s\\\\\\\/VnUwgRnvtO3HAs0ykf\\\\\\\/xtZ2Pd7kbTu64AGDdmx9jHrxsjQWePnStsfsNyz7C54QvsyhF0UZ40cOncoSvqKu37JK3r5sIgWlw9ScgPNDRE\\\\\\\/pi65uNzKRMZzw56chzfMzy73q1q2GHaz0FxYpYLf3pDB5j2kwlwbqC9s7Ujv4f4eKFW7hlrASvRiM04ZeUqsn48\\\\\\\/WlCU\\\\\\\/BUCw9+xBKWGlXBWWgZjwSmxgFkuuqNwecUD2U3slEBAeRVp4f5otz7qEKjxyAbdlaO2pM2OEaHKMtee79EKDz01dk8p1QhHF35qx3SnG3LBwtJ7cvMzkyDrSCJNcL5zGVaRddROjrHa1++cOPSd\\\\\\\/Jnpf7bnyinMacaOj7hQXoEAaS4gyWnitxX2CKhTy2uRAyYbbbKve2us5bp+FR0hEyF9Eya9WOI4lkEbkd9+e3hJMO9+vjUAnGZtFdFhWOO5QV2TRKS04XFboHrbO4R+evjsz+IdcamDKTvI\\\\\\\/Dg6BBsTHd+Vh+iRi+ssdF+f9BtLrwAE2LIgVkEwRFWWR5kEqFesjUZPGqpwOZqXsfhyI\\\\\\\/SK73UUYPa5rioDYPoFf8MTlG+j9O34zlJ18MjlBVLDwS373lDZnT\\\\\\\/GdUonPOJc1BcYNrzg1vCiNozQdcZ3nqpNhWaw+ntLoGuufaRqIk9+vXhXN+fjsj\\\\\\\/W8YntSn+BdY2XMtlfJGFkZnbrccRA8HoJ0GgSXcsyWH8iOTX4Zz0m2RodozUGpA5SU29U0tk6yEG8cmEPDE0wTacPY56CALS3\\\\\\\/mE2u+3eHoiWKMyPlerusYKfsxYlHsNjNqZBH8xfFhXpU1SGmxuLdbvjORvPzWaZJLkIG00Mcz4rbQz4Dy9hHTGtWDCXM9tqvCKFwV\\\\\\\/xbR2GgliQt1CUaHefdnGNjA5fiu65Zzj3o0cJchT3LAKfK1800zEvi0ACCG57Ut9OpP2UVhCXgT2rPfIBn+wOqdZKE1l78sq5Vebw71alGLEqKAfPZ4vQwGODhb93oRWpe55JNvGyR2HYiaQqUt+DXjlShMlPJbNfeRitY7Z2JbTB3yMDVwgHiCYAQfeYt2cYYkPlJh0YiL\\\\\\\/6gzx6YktYrnZvSO9CxivtT7QJqy1tYqswcNpc3qk6wQkA9HN+ZiXzUIFjBmqmBwlB8ml7E5R3ygLW4HjdkN1rLF532ICwMbWmI3URUSheCPlmo28WpGsdlDLEQA0Iew5UtkASxREWcuRy5kSHB9SPo9toH9xpiRedS7o2gPGRNlok2H6PCmZg0dQcgzSysd3vSllM2XFmW8Odot+XIhEp0JDuB5UGZtiGWj8dAMSCzxLYOeCrQB2LLwYBWDwaFmXLrVMjd5L2JjtjF2Xqk9QMe7WV8CbKh+mZZUx\\\\\\\/UGL6QbJFLmGg1UyhgAR6SJmga8amdXEi2j95Xgb7Ctb1zziu9Vaj72xWWCL3uoKl5Vn2dyVoJ\\\\\\\/bOhx+AW6G2TPdv6rDSo6xFWZaCHGrydExUnhZQs4m4zPmoGgcUL3SO1V7QU5wumbFeOeyHw18oBh+MIcya8oEVa4T42iZu\\\\\\\/tsNg6gcfYv95VWiRrbPxtLFF2vKyTe9sXqmWy1OrIWAKevoQCYJY32p0xbV+7ISuYBWcBv66+i2TD+xwgjz1NAtvPG0X4KfU35WqkgobHe2FlvaTWGJdMdVn36ycw6oeBe+wXsWJD9lepg1Eylhrm7IufRZdx+Rajb2JrUoF+EedAm2tjsIJfZE94SIgTlYVSKFZVxzO79xTgqWt6vQVUvATkwhjIfXj+YqWgvzLwSpK\\\\\\\/LqHDlF6sFGacPZ7ksN859y5VIXgJeXO0iW+xUn5K4oq4NhZ8rurE5PBX5OYhI44ZjXz3kaqAcuVDoWl48RZ2qWDg9kW7GZwwYz9ssR29ONRON6IWOq\\\\\\\/lLEklKnd1YYXF6kPSYE7TMHo1MqFAW1X1+Hg8D4OyP4BzKTK3po1jCLIbyvR7ZXF3p0a675lr0oEncuLLTpepP1jOC53eDcYA6cH3Ap0O+CGkvv2qPb3T0+4cubesowVdNW8XANivgzIdjHgjrLSQxdVrxjvERFFJ06PU+ZTZkdI6DIXEpcSlPKYo\\\\\\\/RjUp9f57hSYwxNaWZvnpWAnvjfX6OEVSCMRomWvJ\\\\\\\/XaTDwecWdKVDBuIslWXmAjsjFsXPEyNQMgViOWMrrdgX0ONvF8zzhSuJQXn5SRy6jIcO+gCLeu3yTlLwRrglhg8HXESimGmUc7z6PZ05xL7+hOIlPR+ryf2I18PSPPoRAE910klZuHfRcugvKgQF\\\\\\\/lxEP2maBX6wHkD7MSUZm5jVNLC6K02ANCMi2WW49YgmQMlgqphpNHhUofPJhfanIubvBI0NJaYoO8HRw4GjW9rh78XfaLP1kYFlbuSLjpjhu1+sadnSQVLoLP4oJanEf8d0UHHmIy6aQjboZ5b101a\\\\\\\/qNkZ2qanqg8AlYb6XObhhadIUCD6H1\\\\\\\/upWfBAxOktRyZyTU2NmRPnOTqyQwHamMlUskxrAjZa9ITwK5\\\\\\\/Jmi\\\\\\\/uNeo\\\\\\\/yau\\\\\\\/92HVwpuJ3luICPydqhEdm9K35zxha9qwNoFyt+Vv0MaemaWb60GY+Vjjpp8i\\\\\\\/AN57\\\\\\\/c6DahJ6o7+Y2reKUROcVLXSiwBs4KTI+DDEWhnFHoA5cYVpvvNXNWcbrHufvfWyyXGYcVjcTDlgfuvkMJJbsnAn0JVoOVHhwu7zSEQj5tKgecWoQdEFJ2V9O6mtdgO8Fhb0YkRf9mkJtDHXehAR6amxq\\\\\\\/jQI6qXWMJ+4e5lsZ0JXbWFYlrqOfwhROr5o16URsUKPKBvCL0eETIFfsxMvob8\\\\\\\/JSaeEyjvclc085GJLdTntEuFvOeMBXOGOoYBv6tZH6su5z\\\\\\\/MF3tH1Ok\\\\\\\/INk9dPlh9I8IbvJOIUdnZUhxsh7BZIarjw3QM69LN2Xv2d6fIQCDyQPkqFAY3qCCRCV+oZ5ng6D7y295PWvK5lzYbZDuEOSm13KUAdK9aXG1UNWJYAyjS\\\\\\\/RrzxrNpVa5912oQnJKrFuV8ki7TlCkqBx+JLpqQ\\\\\\\/u0EptckBJbfqVui96tWfoho57+Iu2qRLJTLijZB7EmsqH\\\\\\\/hB\\\\\\\/s+X3Z4VB3Sk\\\\\\\/o9d3utXawnUmQb3LRhzThI6lpmH8WrdcAydTwZ1kfVk2z4wAi3rBWz1oBLSy24kPZwMXB3Jd2M97IQSHte6uRpmRfr4C3l4VUocc8hrYHbgfetIqHOze0GrUPoUkIoLDqgJXG9pOgKEBBkblmsuA0jHMn7qw0DrMVgQpAbVypG85t6LK5bBnpeLkqgHn9zVlcRcvxCckn8D1WjI81RaJ6MDT54iNn6YazWyOLzTyjA1NqJvxwJ\\\\\\\/Jgkylb5iHS+VsvW9RoQz6\\\\\\\/HVGAf6kulaXWluzSbJx72byyDRIcRrRXseBl5Jo19ATGJ2vDvQy9yAqTt7AfZJIyZ+7qFmkSZszqw1IfhNpUJpxkftA2EL1FuBVz5dj6v6q3BGLRK5fNZkXNjzv7WEJkaVjRN3XTYxswU\\\\\\\/PxwtOhNmNwaTGcSuHxsY7SMNth9oV2\\\\\\\/QnPYWz\\\\\\\/gdVjcY\\\\\\\/b9DGTUSUICw9j3xngZAAq7+l5qCEm9VyBJ5pgcoPMH\\\\\\\/HEK3XwOVxuc6aRdZy9BvQAyBp+6ImbHME3h13LwW7a0CHn3Pr6rg5H5zHs6+HEkUHBR2hN+EFvZn1q5LD0IhrXbl6pkA1uVM4y8U+Uj1VfzzYVTkb0lTCZjO9W+cm0XzhdVJwDujqVIdkSEnS5Wca2S32Aa59HcG1x4CDLqRaw\\\\\\\/nt0WG8AztBz8lKeNudofyTJ5z1jCwK9TyPm9f44eBEDS+DJIRfBtywxnQcCnN\\\\\\\/ipjy30pum34X\\\\\\\/zSHQItvHjquQFZU50UEKNGkloPGvThjchLz9DLtpr6oYl2BYCPQk\\\\\\\/SB7dYUy+hFSkPQfsKKyDrnEUFjTSZX4u04fqRLWv2XcTJbigWliGiSWKqD0xeRTAo\\\\\\\/xhxp6Oiv2F3LmzX6DykehjGmwsaiLy1awedHjryXOB1Z4vBFjf8o7IRiUoeNMyCKtuw+hYzMKi44FtlBd0Gr+eg1GrZPTT3005GomJRkCwKp7aIKPUJGsF9ChzFPZ\\\\\\\/r9v9notEqz981\\\\\\\/ex5i4ERLENwvbQK+\\\\\\\/jFY9djXcUOtAiC1bHCyGl\\\\\\\/A7aMiDylpOpSx6xXCWYaWToGADbtjz8G0jKpJGyzJfPxE4VaO5t9Ex+RvqEkRp7SwAPcLXt0EN7A886vrE+n7y6Zai+AouhvE\\\\\\\/8BqTvOFZdLOxjYwizRORZv5vQRvR\\\\\\\/Vf5kmnExDFA2PVK7uAYVw12GYNxWp8mASJ4C+37Fikmtrr9sP008W+O8pFMFuQra7NXzuhmIKvGPBGD0VDOnRFgfAcxKDaCrbO6HnK2wYuZierNJ+06IVAvF\\\\\\\/HflAXqKVpms3QzDsEbOciQyOCc35k+mnn08vJffAJb2CY6alpqgugmB+GTVzKzjtaW+UuAO\\\\\\\/Mry2Dbs34554AadCtXXBPsg70MOAT1yvQI6WyoAkuYMRdqTSHNxf+JGI16rqXS0JTYXGDu7\\\\\\\/gzg73wLPKeZPDy\\\\\\\/MZzK5Id3mWQ69ZbtanAHruom+k19nNp7nWKrJAGf2ZvVtg5+UkkNM49D\\\\\\\/dWAZmQb+oXszFJmh6k7g3ydASShQiYqDAJUJFDrHwVl2EojAea6GpD8NRCPgieiz\\\\\\\/hd6hzgjGU4p8rBCWkRZKyRhL2Ts8u8uM+ZXUa5+H7c38dOwykNxpbahNugZ3x4V6Ny4WISPx4B+pjrCMnEKjIwIX3PVeRUty\\\\\\\/YvkYnSTBpgpF3y7uQpNY73pPmSxlmeEqOq8D5zISAdSHKBzgGmb3dOfKu3xpUbmQPr08u9Xz4AipyBpoLABKLKErjVuT9G6+ACOLGslg7Zxu9R47A8zS1sLAKiACOwbb9LtPKFszD6i7EHkkXMiPn8If0DibxlXcYtHe\\\\\\\/bV5tr8eVmrQH6t8A4G\\\\\\\/InXS8BtSFRb+AsdBf1KWfc9FmYdUo+dFV7eix1SmKn7mGRADjcadENR+nkLoP3h\\\\\\\/QllFAqUJGPYKwwIKGs+p1rAQyeDyNkFBBPSUWZUJ7WbG6v47\\\\\\\/xS0MoepVcTaYTs4UVVjad0hf4WD1TdO4K8efuPpAhyhGGoIzDSK0RY4KeS4RzkNHLT4IQ+VoiS+smqIxkHjRkrw0sGBhT\\\\\\\/hk2G2zg4\\\\\\\/qKedegZTlmtHZeE\\\\\\\/0JIXvua7z2B1U51GMapt1Bf2Hp+RmkYELnOqAMejbYSbQW5gziLWMNQmvFdmOTuEnJrV2ZSuWONOAqE2\\\\\\\/wPc3oulgeSMZNDap5OFM35V4PvknfCaBF102+ruGPbuRREpEm4wnKBl0Twh+20C59j8Pmg8Gj3dSWTQ\\\\\\\/JQk2n1dY3OKdc0U7WGu3PEAmUIJWAzLL703om95ZO5gA7ZzSNQS1KHAy8LCNjOhWL4yyY7NJReVHfAo00LwS1agBk5bM2zLET\\\\\\\/OQTzHCks4xAaDM884JWnVwGUxBecTSM\\\\\\\/WrdWt4XD191FU3ee7N7ucT0lkw1tG1xmV74Dm8\\\\\\\/MNBGMKOAFcjDit8cDmhibj6MQe1abYw6XBQ5I6OmjKe1y+1fYATNZhYokXoq+oL34zl3ea+CR4FnijD7tp7RXsGshkodjFSU1bZzT1joThjLgu26dmfRxRN9juumso27GZWLg6rwvpC1ML4LfHCIwNSi53Tq6Al5Y+eBbh6dRFWZ3pUNXtOtq8JdgzTrvT4iuwJBVEnuJ6ynV1rsSk06nNJqR\\\\\\\/vBC7C3SGpXAZhxC\\\\\\\/ei1dKyR\\\\\\\/xTdTbnzH\\\\\\\/CExF65bUbyMsAz61iJwU+kUkKIXk7T6QfLQ0eW\\\\\\\/kI4UzhXWvb9BIlJHWXTnkIRsAadPPhXXRwiPHZJZXOVSyTX4jbZv7Oxir0E7e2TsrYt52XkKZCLGaCvVpsf9MlX4tlRytTEyCDfoHiSU+NnbyGCndRKql872n\\\\\\\/cj2vmquOBVSD\\\\\\\/pCqvivS9bpiLtqSMgE1Kaeg3WMz\\\\\\\/W9\\\\\\\/DvAosmZ5YavKsyjUSzG7UAhVsx2ehlWzuh41EyXQyCitR99Wkw9DezIg\\\\\\\/PSO170\\\\\\\/I25\\\\\\\/RH11H9CoMnRLqXrxuu8lvCO7CawZq\\\\\\\/lSSH8DuwEp8tUh+ni8c0ZwhWepwnuDgHwpTi7vhhRMj6h46X3YczHLX5aurTOmDBGJac6Ih77I3aLEeKQd6FMjKoikLjWENXrZwllLeTrsg\\\\\\\/NKA0MikTUFsrEcVQ2tHz6Tsfb1G5evNZMtyCXwM+G9qLY1H49rQBumnX5JHLoCNT79sjTP25FyoU8UYOJ31MUICI\\\\\\\/dOWvjd\\\\\\\/o3GLhZ60PrQRrrL7FNDILxkixue4sfK+6Icl3iC0BP40DOFQdx9r+s4kyOk27QVOHMoGN3DX6wsrrTlNtsC4CyEbZeVKFgTfgjvGjZTwlgWoXQ1SE\\\\\\\/AAAgS3oVCYFMr1d0Z16BBEjnh0frKCCXF8TW12yqrfFnJUideM45gm9EIxKo\\\\\\\/mz4ePzR09dTkSaU8cjLHk6RXz87yjvB8hEPmxXBPqKuUvgGschLMEaEmSdvv0pHr\\\\\\\/SSemYq3bCclhpG4qCBa4qc06eO4x818hRuppVw7WIeH7hNuTzoT2jwhrPjHB53s3WvDGaeWcjzrMkZyZKIrMy\\\\\\\/cwusX+xWFuZaF13mbQRWpp+10fxsMocU7BKTvavKaM1X0PyqoJS86uS6huOBPCK0+8zzWsRHIbYEM75XjRScP7NU88RIcz0+WkNYAbjA9DWcRkdOgjsnbpD7Cam2xkvACcTLRyjKFSNm3cK+\\\\\\\/3PUBtlbyBMYVoViHfxIgN6lT0HlED3juZfZEIToe9zM2lHxQIhAxEGTl0X5yKuO5Y4DlT+LoChE4xCXuqEjyUWVvSK7mnNeHJRx+wGq996p81d6FP\\\\\\\/MPgaEDBxTWcDa9HHlQjl7UTlPH4KL4H0Vk+lzE0VjVg4P4WjungJPkuLXEemA1kXyyXZ\\\\\\\/pOSFQndkDqrWKEf5N9MTyFbFXaIMI+5fG\\\\\\\/VUNH0x5HCfzpOOyvJqCgCmAiQW1Pc3bdc9856R8G\\\\\\\/+gFQuyWFqFWhkv0X4M5GMMsTXbo6IUe5jKfPOyFeoG2gU5VzFP7I3XyDG9dKZDUTrEnHdhJqrgLkWni1sNIJHwmCk5\\\\\\\/Ox58V2nKEMxVFp6a2b6ZaT80e1HrSodxvITQxb9z9XZL63xTS0\\\\\\\/8LP684xucteHlEp4fHYLoRQlS5MyjpyMyV8\\\\\\\/2jx7C8iD5W5PGr7AU0hR110Y2KFcS4WCz397+f9fels2pjIHKsjWnFzMLbHg9zeOu8nYcSVtqSn+xAfgRbte+aMGdb7EZYOj0rNtgIBfMiohvBi2T6br+V0PvO2Pv+4MRswS0\\\\\\\/ZZxXxrNd3URaCggPnByF5x4deiZlI8V3CreGqaM8Npw2i7RhKLZ8g56urFDI85xY2CKGsY\\\\\\\/Rbrm6G3JP2JiHeT1zIAy9CLOH\\\\\\\/aU9jDqzDM5eJBYhRjY5\\\\\\\/IJh6A4QlbHHbL0QLKJrGsZU7yTOhwZ5d0hjg45QLTTADz3gvGof8KdT6r2NZ6QCcFxguIxJeIgkzCfb+2NbAIMSdCAXSxI4by84s\\\\\\\/CvF+MaJDEjzbRO\\\\\\\/y+5m2\\\\\\\/NOR8FcvuXxvtC99GF2545X4BztwEHgUJ6c7Xc8Go2NpiZIFBb3p2pvTpa9YiJGskqaCj5IXMMjMK7vLzLQMYQzVaYVtWfxwBhQvc0ZWFCdqcXlCXwEw2Yr3sZ4OPQfwQ3eP6LyjyEHFVP0XCqBlIP2ZQ+NkdEhUQRXn53k\\\\\\\/Y9H+YWzSQXvgS7wOPMV\\\\\\\/VOZ8fmWV4K094Kk8lNh2SsbKxffsWlBL4WYZY2Y2ZGMXmMqgk0Ls3wwSJET6X7CN6fkvHjb5YSkwwnpB0wn09UiWpCEBGPmYo7NHSdF5bXNW4GtngC7JTgNKYQVxbWvgC6qAjmnHPlFAp8UStMBZjFG0vl7Z2uSTfHyQIjOtUzmI6NhRVKadU2Lm5Rs+6VkbZDhQepFdzwSvxiUsr1WYjwzYHw9XmEtIben1bmTR5OaR8E7kfaerOixpBVMasRHLZsnufZxvihfs1W9MSclCV6l2RHNxHF2K6CyrsHJ1Mv49S5W2AQjtQkb+ZZ5tUBU6LegGHci5o05mKPz+qQm\\\\\\\/Xvnm5Z2k30eKxEtm6\\\\\\\/Hl3FoL2J4gyecTEKG0Jr5OVGfSHuULN3KF\\\\\\\/2k\\\\\\\/QvfrQe1SBXPFJtG2ADu\\\\\\\/mAWkMPsWB41PJtjsLjg5xvKWuxoKjFGn0KI\\\\\\\/eFaPFhkbc7aWhftzHNeCNSzNhUvlgenU2tvvXfSGqZ2aqyjy1kKyhIoxbuptZPdMHHjKnyaSQI\\\\\\\/LcmeY84Whl7JYVUsrJMzEpMfnVA1AKtDN0OXMv8+rHkNYobS45eSXwpt8vlMPN2TVvF\\\\\\\/pFZ+I+bgN26jI3oV1vrlukD0j6KdBfK7kzOXpFw\\\\\\\/62sCx2LqjOAkbYNlgSqmsZTYqfB1QdbR0SZZ\\\\\\\/toD47MsG6zs5GR9CqXCzgOX076SJgSv9unRIPxRI3L5Jr+YT40XMDpaC1JMcmxteazPq8ZK7h2uCFJjaJOasLxP9ayMxyzDmug\\\\\\\/9fuFsvPAH7bH4tbfkYwoRgFOD+Pq5SAhVRHvbrm5K1Ld4F3g5gZDCX2lAS8B2btZl\\\\\\\/VAp1I0um7u3R4bSC2WNKtD0YLSB6Fgec0bwH6nQKrZSctKr6csM+7eGNZP6940jqXL8nGmcWNEapzccFUW9p8kq1Rjx0n5Uuc8FMV33nlooXq3KJhanR9WZ8vhC5MOGzFhnvmSvv8DGiKp6yAOYwxuqmtXYv\\\\\\\/g+FAut5XEU2eByigliqAkU0lZRcQrzy3QSQAT0tRchqlh+bXCSbRUyfAios+fASE9kNxk1LZV6gVC3RMBqDeIXqlRNhqulCftrumt3CByTMuEtjyPef2AzsjORCNkM0uOpW0I6lg7D5M4UaHOL+DH1e4KjrutJl+GxNgvfxh4cI\\\\\\\/SmvtEYTIdHmZO7Q5MD0pxzHIQT4uQ\\\\\\\/M5CQYJ7uyBPekS1A1YzXsZfUkGShh4k8zVRSHt7g2Pijn5te+rL4AsXEyE8NQbHDVNJkqVPtL\\\\\\\/ugzc4XlG4eXZLo0Z4++\\\\\\\/5IzXTEfwrAYWxjFffyCQ9BQc92yIIdChQ19Tlw5zUHXKTlBCP69zTOIp4GbRfvcrPT4TsPq6LfkR5pRq8JHnd6WXjQ0vULmantBIkKzGaRRQXJU+rP9PTP5iv\\\\\\\/c5sivoTvwQycnxhHUDShzdAZIXkdKC+QdQC1V+FvB6p0o+tkXbUOX9yf9W1G+T4vDHND4A9f\\\\\\\/Uw278h04Ri5dpg8dMt+ZjD0LMCTLdNTeGDk+nL3v2JGAjYjfM07ao2wb\\\\\\\/jpp7wwxM4B4EV14eI2\\\\\\\/GXctB6WVxMT4vvTx\\\\\\\/4JPQRtKzCj8iX2VrUtx491p\\\\\\\/S6thLeSihRyZDGP7pvQmyM8\\\\\\\/huHbKwOxgJWV6Xl0+HrfWlb820FddN4kCs+U5aA66uLJOOyT2q0z7Bzcye8llF9JUBMa2E25O7mH3ZwKlADJeope9\\\\\\\/\\\\\\\/H3o6RcteFNKIEZo5WFrWpR6ry76qQXh9Ink7sntTR661aUY\\\\\\\/wMAqYuULpTJb2FIMvpDCa6828mLS+u7F0UonRTuNDZwJRbu\\\\\\\/+XDB+lVhWmRTqH6fbTv2tmPegL+BWylkE5iAh0G5mc\\\\\\\/8NGYg\\\\\\\/kcUaLHV89kNH0TF68D33zhnJOrz7wdMfLLreSR0xVNblnzYO08VdQXIQWrIsCpSL5YjqwnSF0fMs5UpFKK+f5NYq\\\\\\\/YMp0PeMxWWoarFRO6dJj1NYwKYNDcZUxG4a20riY+jIUVaRDU7tl2MzsiXve0GIpjZELDtLIA0yztQ5N0kHAbl+LBI9HJa\\\\\\\/3HvdWMLs4ox6v4NB9jGcx\\\\\\\/n8SPNx8DewWjKEi9y0Z1rnXUfBDJQhBIoNMovs20uROTMJzR2sCkAZihibUZmLPVugfobcLoHAi0Wv2SolCJOcCxs8KAOCtvmcESDNtREm+0ShwUr9BrPElCuvHbUC5ECponqGPJH7xBgMMBg9x9FbtVa2N+r4q65toQj8ri7p713DZuJYGAgNXgkQzXlferW\\\\\\\/lTXOHF46vhrm4IcDXuFo+H\\\\\\\/eiqk9syGRZFmRAQaABtYYbp2xVVvye1\\\\\\\/lo+R5mF2fd21xyexGvVAFlP71UfchlPgtqP538t7mZc2GCNTkU0RSVyVlmElIceYCH0zQkJewuL5XzKHp+dNpqMn+9R4x3MsWYKnaohUPK+XE8M6xrLUUaSqiWQCjy8WjPLBpkt7l1jJclD6c+vDJXd8MPy6fb9uj3u8i\\\\\\\/8lsQh+xIzIx2mR6wYu7pY6mXqux0H2Qtn0AfHc3jm4yp+uBweUls1fSHCpzJwJTLRfTUZ+fbB625Ce1hCvk1y\\\\\\\/u09WkIRi5Vs1ThYrG9\\\\\\\/ZX92ks+RrqEvVqmrQOUY7D1QF6aiD1+OTdI64O2LGZNVfxjAS3hI7RhqqJYDBKwoa\\\\\\\/1FeEGy1mkAwgKvuLpERQTfa\\\\\\\/z+\\\\\\\/8dkx+sj3AQsN63X7\\\\\\\/8XylgxLKTReDqfs\\\\\\\/hXFoiizLn6VYQKp3dhazzzwL\\\\\\\/u7LB8W7sw9cSGwV87Wxue5HQRMj9oRTYkEcMSvt1ETTZaIqzQcCT2eV3mmeoo7nNXEZg56UPjXRLqtRe+si82aGnV4\\\\\\\/\\\\\\\/hSnRmkbmpaIc7dlPWD++8OSdZH0ft5kFyJ\\\\\\\/RzxUl8RJkMly2eNuCDnnfWIF39Ku4SI3fem54wLElg6obPhCU+dnE0rmj8NHVhDwL0uCJdvRYYslvLZLhGPf6L0tGbuRQidYREgnD40RLpPn3w\\\\\\\/iaoW8czECk\\\\\\\/5P9Jq18uuWZebFzvebqhT+0XpstzDXWd5CsGz4MwTEaAKxqrUTdslBczxuA1N9AIpiHehUpgM4Vzg4cQmCX4ACQezwF7frgjwgIe0lpkonIJ+HClUBJAYB0zFCczOWTZzaqjRXtgqrEeWg3OQy1059fTswkTNmqq35yvmACaZ5MD+ESdzhsMNABBhjNfyjMCPYEdKR\\\\\\\/wz13u9HFfslvvKUpEmNWEZle+lFnlj5UDa6ThV3kxZdyky5QvMSGBli5VV2ixnUTOLQmFCxmgQF\\\\\\\/9\\\\\\\/V1HK0KR2J5GlP5cggk\\\\\\\/NSunywzmmRsVdI0yVyG5EniV4IW9s+YiegqnAihRyMOqgj0pFSyJqwVT92N3vikv3MUeDMrpoLmW2k99t1vVavUvhp4FtfYdOu6QdZ0s+QijRDE64VIsZ8UnjM50Vl\\\\\\\/vsQcziNgDdW46CzXcEKFUyXgkJbHbDsv7iVcQ6lMLiBsBI88gMvHcNckkWq94aU+C2Y6Y9LZK\\\\\\\/eoBmYPojT4F52k1PR06L1nI4t7Opp0o84ulYmUmXoLIPhfRKexacSf6lnE8Tnj3Ql84prP2WV1jyqw5iYxF+QE\\\\\\\/Dj7mTCCZykEpxGmyXcr0pHtX6olYmTtIu7piKXl2CEIk43WGLdZP4LncLjCPv5f9jbu9YQEojmxzhqZnPLJI\\\\\\\/tPBePsjv6k5Ek0pB9EPGRT0Dc1Maja9EaTo2fRBtAAIyUS4xGE4cZsJOnWymbtHyn\\\\\\\/frz\\\\\\\/OKm57\\\\\\\/fzta3DgwZKlsnCabPD7lpRJYOkw0ESG5p4SwFLU0B7wVuDM\\\\\\\/7HCs4j41EVahZNSof\\\\\\\/JUjfdgWfDNCyxrs9kP7\\\\\\\/zbsDzMFBFR5SQdZWVcK\\\\\\\/Ppo9s2YWoOEyKSElSgiDW7z1U7Nb7snnaUEW8Ca2\\\\\\\/UcDO3YJNa3FFpc4yZXQvg2Mu\\\\\\\/zJs7IZ11Kg+BoD6mrKSZ3nMVsfSxKNCXxW2OtBi6Wk9DpGkwJsyGBynucUPk9t42huYG5l\\\\\\\/j1pjDm3lP9T83dmvMrSEOMP8wM4CWVL6tZnM0r6T8KMYG7569\\\\\\\/16An2wCJN0BA8C1\\\\\\\/1VQbQbpRLV0JlDys1PyiVRggOVCHkpKdnzkh6zwtfcLBp0k3Mt8wwoPO\\\\\\\/HiQE8AKjsEv8Dn9OHauZdmR\\\\\\\/BpjTwtlezmZxnaSt+vn+GTnc\\\\\\\/Xi\\\\\\\/HrueGMVLt8bxo14LP+ZcRoPjMX4eazoi+zLzfGRw4\\\\\\\/lpSNPJQAEdqIrOya13MEiO0RF+XFa6sdO6+QkOgjQaEe8htg6ZV1+R+qCmPrmmrXflwr89PsmzplCxt0JXdsjxJkWJpokW1f36BLOj3dfEHrFKN3nNHG4h0ozC77CRPSxq7a1T68Jyh8\\\\\\\/\\\\\\\/ltyhctdCy8vaJFw18qV+ImSOQVMrY18xy31I0tuPMKtQuhfHl587RzDDBppMDlOViQWnfnufHrRPSX97n0\\\\\\\/1Q3aMzB4cMe9rPXVOZk168i4tRoyYRLQtnrRbSkYSs9lil5Wc4xl4Dxs1QfkZG3rchV7ufnbGOX8mRnlJdmYVIwK00al\\\\\\\/6HPetQlvoPU1QpFu2Fw4cgLH8obMxqE52mObUlFfFI8wrLSUvcvhhdDqKUGU4ypJ+9bGc3Cm0o+pl\\\\\\\/W46m3y8VTBAPXhfzgkq0LbFmpdJvKnaw16TnFNf5KmpW\\\\\\\/44tmru6oVXJNqvRIsVuPj2WVi1Tx1E7EgAzT3KhrjG2QakjF7+zKyBzRQwYPbJGqKm\\\\\\\/AcbyzXu5RBizby3B8XN2vJcUYPO9qlkYM4PagyXgqrb2cqa5XkwGYCE0XkNivKSyZ3e9BU3EEl8oXerDEfd6\\\\\\\/OI1vY5QtuLRDe3nhgsiGg3+7dNKXegGYbQ3hJXYuGUagIxjdfhuXMRM3nskE9rzSvYeltQejs26g\\\\\\\/yCWkQ2Domyu5YaMwcLc0L4wePyVBHMYAwuivYzyqdpDElgU+nKatOj1sZh+t4JDAPXDck7AGWQFb3C93VSe08q3aTTKuuPb49cB4rqMCQx2E3cidXoa1h7BJeMiW20ZBWwlKUstLz23Q1mG7JVrkQS0rVB4ITHSN4St24jIPRCamTtDFhhRnfDqWZBfgqUnEw7g7QLJ+\\\\\\\/DMI8PlBOW7z7EKct0yQz6WnuEE0PArsUq0RfpIA0U3h8YlAqZ5mn794NqRfJhQdWPzspP+kiZv7s22yA4pcrH2SzLH9Es1JKwAOk6JBf\\\\\\\/po+7emoE54NY5QJtVh3+MU9mN+iGX35GxCASfigPdFj7jQ2aZSBGGzVGLzqYp7EGhb9ESR+JldaumozSQ4\\\\\\\/5Ky1CMB7U86G6kYbggsoj7W0NbM4dNCrmOXXHJR1+goX+O2NYQfbcKSSBwhFk3d6lgpvENm3MFOmFWIhTRc7hrJb2fKTcreNF8v2bVlBZRvAPNmZehc9hzZFqLstUO89C3\\\\\\\/G2WumHqnJevtENkl3qFfPmcNl6wMGB8AjtzGNPP+cDsEAioic2JqiXJZb1KQn9Fod2OtxR1hv8kC5nSRWe2R3HNavie272lvzzEo4+TgOrm3h4Q7r0I6DSdQrMijZ9lP6L+WT5XUGWtjUnsw1UIxPZfvHJ\\\\\\\/z1hWnQHq5NY2LHpDE+Pdf4bmn2MyKpIpgV\\\\\\\/Citq2y8mn6hKSqBeSwzJsF9FKRRwk2REeN1Vsy\\\\\\\/CHIMhni7SERVMLvNVPXC6BnJ9ScM5SA3DsowpdqBeZprHR4Z\\\\\\\/fm5\\\\\\\/N0O8SMCFEPGvFniOKXykEOVRcCG2wj4O2x22zNA0aa2VB7KZlkmeeLC4paUc587dikyIXPmW+S41yQbX9wNsgr4X7dnQ5dEP9ysJqvLdMkoXi4xmPyFhdrH0r4\\\\\\\/9l\\\\\\\/IaN5mSabDKW8dqJIAucX85nHkq4JzAgBtQN4QMrmO05yeKBOe8RASpsmhuI34R6D0HgHe6Y4y+ywJ8+Nf+nCKVMIzHzH69pte1ssx7kJjNKHTeNOiJOsUH\\\\\\\/gr7oKcdmpJDUI17k+EFcPCM37Op4KZIKE5j+E8uwxKm4GqVCOQfvnzowLGhSiZPgL8KOwDtiyLKb9Qy+iVoKvc\\\\\\\/j3KtyQAZYAu0KHxQ75GDvgnJYca\\\\\\\/RKXdrpykTe+U6ObQyNuYnEbg1yQP2IPPd9Gzeo1kdWNo1S\\\\\\\/W4hY75OHtMsgKU2EPA7cNPqfnkVtmpoE2ea2pjI4PB0jE7ie1ombDxL8jRdeWd0E7is81UQMRDaIcoKeEn7HhLykwGfNDAgg9HNEiIC3nFTAG7CR8sFSKMT15XTsjF2OajpFRzfUsDj8Bo8V5dvnCu5iv5m7QSaQeoXVfmasM1wn\\\\\\\/JbcoXIeeeDbO5o7oWVwHf6J8QANW9xnoG01Tix\\\\\\\/EYc2CxC6BN3\\\\\\\/tWubFEvK28k1YpmlIZcd2oj8kvvzPK\\\\\\\/KIKkjcf7ZgC6gQzHFvNBQWwYOs9FKbGPTwCGR9MWApdiVnJcJpaywjldWvelQHLBdyILw6gRazEQkc5C5Y6KKP6YPe7SKvNDyvPPP5XJFAnhVqSnKuCvkeC\\\\\\\/pajpQqzinMwT6WZmu77bgBg\\\\\\\/ygoomBAuRPjdfROhda41P+YWRqsOFByOOtRE73muUdNyYwjgEofPoxdOtaxSKs\\\\\\\/VXoiFamK25TuyQ3yWVsDwnb4cgwuoKin7mTHbZTYyavr3+TDIlI6ggI01CFy0yoGP\\\\\\\/zhzwCzDZYfy+VEsjGHPYX2ioE\\\\\\\/HrHjAnkWejc0+0U4tHGqKM2UvJFNzVPTtPdVkKhM\\\\\\\/IU7OTxUasmJVu1zBkYrIndfwHHyD0uCQ0NgRvqoQ9m5xzapVAwm55b1rjf7Bn3ImPSlX7Z8TAM6V\\\\\\\/gKUXIZLMHX1nSGBDPm75ZcU4AAA02UPowQ5nyYqOggUoYw8JsIduDsTMqRsH8OGIv3oH5rSx0JbX5JuetemqcuUNkSZ+fezF9bNoYzUj73kToILQkpUnZjSRWOtt2AGWPiBfkJftxlCKA42vBNGXwNhZI++TemnAoOKHqkGMXnKPXRLVYXyKV30aBAMtwQQr5FS6hRQKVgo0aiVQVEKVWUIjHKCKB7PY\\\\\\\/9vSBcGCB+6oo3Myyte4UsbtqH9a710eurFDZ3uyclvhCqDgCqXS2rbEmzWTDMSnyLdRQ\\\\\\\/IO+XxulH4zXnDA1QgXLKv6J6ZzUtX4FUShAFKXgBokswMwVW9vkGBMua8ey7Y9BIEr22mzmansiNt2COa64UbRiBFEXbElEZsUDt+\\\\\\\/DmH4N+4Pm1n3OAMsQwWjwgCixJ07SWsYZiMycWa3Ufiti49XIb\\\\\\\/ZAy1HCngJlXmer7aIPpN1HxSsWI+tt0PLFrbiPxGP4q+ssWeRyIHgkMyha6j\\\\\\\/lyXCuKCeGh1rtAC4aW8qC7N+oQn7Ys+wwF+tlnpdwziqZVsgovastB21c2VO5jP4PDcSntU\\\\\\\/ZuiX9mVTKNe2VGLa5MH45+v6Ga7\\\\\\\/qzNTkwwRAZBTdVncuv0ZeRohGOkG\\\\\\\/Guk6Y+AQp5GjUpAclz9YSeUlUho6vEv3Dh2FmWyyLvNDodxSfyd6Y3ppe8JqdxHy9fsJDanogRnJUJUu+2Y26Gk8TlANIoWb4Fg5t+2ZHBUP1am+f11CsW1Bb40ZO3WqncDr0UrwYFooK7AJL85usIYItC3sN41OUkesaYwT09PsiaG6\\\\\\\/64gHe8PdsUMzl5rf23glE\\\\\\\/9I8CCQCM1CeltYxc\\\\\\\/MMejNh7RPD2+1yKWTb5D9Dg4z4rsgGZ8KmJXYAF1MZYn4FvT0wjtGFlb2\\\\\\\/kUK2uIckmYdGNn7fbYwT2HfsCQr5cYe0x\\\\\\\/1Ta+SJusjrSISnu0hL7w9jzPHUo3oKyTLvHpmkHSLvgoV+QI9P1APk8SMsrphggS1vG25wTc3GnPE3GOBoTdrZ+wU+G52MojaTquIQJcQxhYHGBoYo8qh+DpDjAvQOXPwThj1Vi9Ha2lG1Z6JU+b6EDiQrtDYvMICCgC4XL+1sK+mbqIBMoNilCrzjkt0F0bDpLlQkrfm+Ycbza8mrOx182m20bymjNHry+N6mz1PhLT1fvxpgK8qXSAT7fdz40pd6pQUFwblQvXftnVplE+vxEfCRA6au563eGGoHb61tpwZ89EBJRmegapYwTFfrQahHUs6reKdIbURl0+5yIOr99s8swNa2kqbOIgY7s5zMf8cGFt8BNmkb\\\\\\\/3A3nAbPTN8Byh1eqYUVzzdDj3+6uw+abD1D3HZihV8u7fWiA3Xz4CCsEngwlcgXdzSubP1\\\\\\\/L+9TJS4Kq7UOGwDEwiHCU+l9r+xlWawJoXb8yDR54S7HSfoDVDI1FzBCdgu6mi0GrEnqOmNJzECxRPelezyJiraaFN\\\\\\\/itIuXIYcgkxQijfYK9NZvKpmPj7HJoyOeyqt4fpjP1lHwMuIW9488s88jkCvnlJ9CP0RVwl7lXjKFRkuA0CmUFj\\\\\\\/mk05mdPxP6+ocHDi7ER3Neb44IeIXkKypE7NJFOzh1aEXxo5DNr+WeEqAf2ksufhe0Ru45FYXN0kdM8uq6YpY6ZUx08oXyKBRMkXcwdLgPchmXnAKpeieXuHdaEKegxKN17yUaeadzj0ozM0AYTLF98oCPuzQVbmiEz7JFq0u+S8bbg\\\\\\\/roDdEUvmdeIGNsZ8fTvClaLgKtQ0JlNA474bl67v48UUsikYPzm2aasoMaSFnl6BUq0wx6yQRXV+6uYLWqBzm7+RBY6gXmSjsvMXKeW8Qol1Koz6R1qdQvWa8GZmjHnCSRm3vkCnGHy1R9I+awDDOaYwm\\\\\\\/W2tPQ0m+Dh0Kl+njX6K9sp9ulbu+Q5774j47cHlSzxgs3aRrTQMInnSH3d8pWK69thEv87dtlPTSiMxj5GzZBtM9kX\\\\\\\/Oxvnzr\\\\\\\/uOGvUIW4cQXhbcDyUCa3u\\\\\\\/vwrwoQa2HL6GL3ZMfMeJUSJIYIKU13tNvhzBwzMTDZIbsuzJXUaeJNJGybCp0K97aCtZbyPxZF6z7SMOb9OH+yufSURgbWfdpeMMzJVQu\\\\\\\/2f24BH6hKj+zDpP4W0o5\\\\\\\/167XfAvS3Fdb06VVZyR+tyBcBo\\\\\\\/1BByjwb0N0bEjBkTP2vlkh1XQ6tu0yzHhrds7raFS8XIw5lurZvwcO0OWCeuA2GRr2Y8vYcdMBh5RfGOZ08UfAfHU+GQLn4a8NeAIhgEVSw5K+SFJ+OTYdvjSRGkepSqwyKBQlB9mJNLsPATfcF74krNSsCqsDH4DtAvmNyU4u\\\\\\\/dH+qrGrav8ICWqA2PXLQOZQeh7LsBod80WCtSZU\\\\\\\/7rq4MdX5HvFhdoiN3ahq+3VDhr0P+M2JEQyKufKC9ZPrO2FP35rzci5hZoTPShHpdLYlKScNTGvJdk2T0p1DoXWGxU8or4F5aKCatRQXN9X1llqCMIYfBeRTsMhxXFw4HSDcjxlCUmKg3BxsjqSIAp8JG\\\\\\\/nT5Bf6WYBbGzJtMGVLBp71d5qKQdqVH2nlQ071TkwXmylA00QE7b3FF0VhCRMEqJEkXTzfm09ZMk8w9TqSMZm2fRxknRbf54oKZO5SlIV9+kLTux8U4PzXvQinvCGp5yf0+6FXgBrDcSEVMKE0fb+Jpk2ZU9K\\\\\\\/Tf2hxG9nJ0TT5SP\\\\\\\/oJRhPwhIPJh6FUQtnG8JVURoV1pRA06SXwzLvTi0HAg5KAQSXVPmeSt1Tw4+xnbkG5ObKtgAStptDJlL3M+wFKeUxbzTezoC6KO+qgRI7CZzEZ3BxRyDANJD560ZFSPDl\\\\\\\/9a8oF9Eoo1jQ9AtvMvrbpF2wkABGjxvyiiKonzrZCTe12aoTm7vUC+kph4ISUphl5egCRO6Wd03ptQRThDLELVkpm35\\\\\\\/3Tm99HK0FZuYbP8OfOTQXLrfgonQRkqjc4qK1CU0\\\\\\\/q4WNw4zgo+FGjpJbYD2UZGj0o5Ilyznyk51GW5MDPlVnQuincZLgvaV1wxl7OCOncMyrDhigltORzZiT66kDdH0uIkU6vMp9pexEC2dNkYGXoJoynQg1c9gRZGfiMkOBQHY4VBkNBQHTCcxJHvuA2VjjQDEz8gKhp7ShaA17AjioK69JzIAg2dJ3cisFfxHTRFh3iTGLKmo2DJK7d0OngaEmkcvDLXI65FmsQYicJfCrT\\\\\\\/v5jGjaOpXEGDUuwSVEw8hcXKrkMrV58hItLv+4kOG8OFsNV5cwNqcro2PsBa74vDMJ7txulqTurpIZtHlYdQQkcxwQIgOFCVKMk7j6xRKWA61DiuaWWPiU9BMnF51RtY20Ycn7cMtIPHYjGDAZesOM6mHQyw1VwCGcF4\\\\\\\/M\\\\\\\/ldI3zihSIvWclDgrnCtmJXBN2zmwOlprWxvE\\\\\\\/7Soa\\\\\\\/tc\\\\\\\/5covZG1NrgmUP\\\\\\\/wKz\\\\\\\/mRNXkLr6WhBHnm1tzuWJQBDj7xVAVP\\\\\\\/JbbF+5mQwDdpOuQHOUfB02QY+8wIBZ2VoAfyyrEDyu3hpWU7dTV97Bkf13Nph21bNKZpJdMxeVNTKHI57EUmJWHa8922M09kA5+h1fJbCBSGx2po+lHEc\\\\\\\/jOTZ3yxv0UmZohV1Q7OTEfVxuZYUPa8iAFaBNn3QpIDTHN8TwaWguLdmkj5thot6q4nHgIn1lw\\\\\\\/JFqY5Dq2U3MKZOi20Pe5P20LnveIXjzDgOU6U7cC6dRk878o+Ta7o2DV+ziwI+T1hPaYAiOaLz0zJazuGWxmQI7tjeToyHkHoW12PgoOPZuJ3DrUcxL13IKvIgn4tzc9es4Ghs\\\\\\\/dd221EuJuAlGVguLiaUDVtekzjgbejEvPUMmV0ZgwJd\\\\\\\/1tz05vG92Y4yyAJn56Uu274ucSs0OhssOjjkiCbrwtSZCADSEs4VUwQiCNjKsIT+H4PoMaA86dq3ovoBGeJIAQtuYgjEOoNu9NhUaV4CvIRs37Hmyw5IB8Fn0EgaXLRdDcNEe1C6ljd\\\\\\\/tiLJJ\\\\\\\/TKKsiSWcDUzqsAXL\\\\\\\/F1LimMBjep1WKRAKnFpdUeA\\\\\\\/CeKvcNcHPnmS0MrToAPFPlxkeDqadwu0ofZqW0JVyW11QLrQdgfOsMqXh61A6NINEd+25Iv944Lz5+H4RtjcQE1fcTHPREO6JfqvCCy1PtsKNVptGlKGx0+9XokFEyvVbQTZ4RCXuVT4mRlW+4CsrK1squ6HTHZyB0zNzlReMrabuUdVmMZjP9PEihpcZxrRK3xXpEncLb2uuV+IBYxC2ANOfcwpwnIj2BDeme5A7UtmZ+88IDjvsA7ynKR3zyKWTqTtva4dIcT4+X6yV4XV+PGe1OHvHxxumEVtEbuZEt9hQfSvyyxpgRWMF0cZy6dOruxdYpMHc0h0UXMARLI393\\\\\\\/JNlBNMTOVVUxwxLAHMKjT7xTVR3mZc5YHXXbnzWfJcHdKem4g4uk5QJHwYr7BbXivNo7MZGsadvY11Bee0INO2tWtuDsOLNNfmYwc+O6Ujvj66xwVqq\\\\\\\/Kjuvjruwob0HR9PV6ARRBXz8ugOKssDLbogVQcNxunU6c3YeCFcBG9K8ZgHTqwq\\\\\\\/T3LO0oHvxfpN4l\\\\\\\/OKlTuzxPmyE340B+qZjeWRa3R8+EkiqiTWO7QxgvCc3qcjnTndV\\\\\\\/eIJi7oDNUztvPDnO8HFetz+c1nlBA3zwAs5WvcGhg9FgXoxi9kodpO0HGoP19PQZcWhpI\\\\\\\/QS+SKDD1abb4rz0nmdTlceRM87C3Z9n\\\\\\\/U1XdmJSIUp90gfRAiliRN8WlSMd4\\\\\\\/WOdDZ+F8MzImG9ZsLVeeVnOjmzNgQNBeYU4tHanaJReRTWwG\\\\\\\/Dl4wR49gnHx19slX91i\\\\\\\/0UX2kgcaFR7wkzjsd9LBXQSzsQaGiwbkeJ6kKdk7mwemVnQSlMSFF0HXDnuGKLHcRgy+A+eBy3oXxKGPIRmSXxgNokUrii3vrE8xoW158JfCZZkEK0wmv5elWlLkF2U\\\\\\\/cx6oAEaKz+0uIK2TMZ7k8MsMz7Fus1WujYDMWCQVTAmmVRmS+EOFhOkqiSA\\\\\\\/jq4sff3IGeuQqxPW3XUzfqsxtlQ69GeP+YZ8y5DUN8BoZxHdP1HOtYnfui1rtQTBqlV7A2+M7Kn8iJaUlQrcsnj3K3oAtpkZAN28JTRnvkanoNcRgo7xCj8xxKREk5ZHtYT3WnkNoWzAmfmnIbicyq1+kk2ZjcoCL4pnJG+oSmpmQSEO++zPLoluQ39GUbnTuzN77CvGmhVkFZzI1zBs7krv7SZpQ9croaZd6r1rHtcLTzSXbTswibq7kcyi5qmUkGSd4STyhuTaHKxPp8BAPbqvfyM+3gvqyDefYR6Ru8BIuNL3jrjRhtH5aDCq0nD0V6621\\\\\\\/5QdJXTHvn4XN5ijvW89cXIgKyKFd0detr0qhdbTb5e8z7D3jlJnBFFiZZh7u5vjWkKS1wi0oY8dcLyq+j96Ee2iux1zs3nFOWhdJ5NS3Dn\\\\\\\/6xUNDo+4iJ7zTk2H8IwVaz4DBk5c3MBQNDtLxK+Vqepdq+PlzC44IcMY7ZCn1amEsFJZBmx0sXHn2e37uEpuFkz0tS1Xnc0oaAjw+OdN9XcMpGuUb1PazSS1M7ERWIbArjHsepKChUzP0CopBYekowtW0fsbnx1m8WKBrNTyZERkqpEpuZh+xc7QslHcsHSVPBHTJ2HEQBItgqzO4TsRO8IThXa9+\\\\\\\/CJun4kaVgKyik+YhN0\\\\\\\/ECIMBLzHayoHKHEfAk4Cu3xUsvPnV+cDbgtfOzNSRvWdFzIlDAsJIGgY+xvRREBgW4ANXVgnDOREWmWfQG5NChyo7JVyG5xA9FpgxXe+mL4NaRb3RKyhuWkO2oX1TguiKXo8UFHUJalEmHlmW+S4NRr+yMtvmilhecKjX10LPzbT9vl1GJzUehmz3J8MCyGMj+xue45zXWGkK9oXqQUxwH8ME7zf6moQ0qqumEMsKnpQxOVuqD+g7YXm6eMt\\\\\\\/ivoCrwTcOMV6bHr+tbrnAtBAQHX88Z2yD6UaYgBW46TQO0Pvfw3Dxm5zyL6JhLulC7RXAO1ZFrVy1KaMVsa2NCw1409ixtYKjy\\\\\\\/fvwUglXZE5XtYtNrzFReAeuUzIPqgahJQAAOcNKordf8490ZhBiQFIDYT6nQy0bsMC\\\\\\\/fOD\\\\\\\/vtvS+LIdzRZd3C\\\\\\\/lJ3DXUiEQs2aQua6Ao8gEBFPrYuVjMJek\\\\\\\/pi13pVPuwTdn4x+MyBUKXqh6yySdosuP6CVCXKtMDoFvNUBvhKO\\\\\\\/5wbK4SKvB39j4o+rJsisTf\\\\\\\/iZFfTKjhM9k21ws7eg7JAzdu\\\\\\\/6F4ETDe3xMw6TahX9Djh0OWBcXUqRwTsvWQDyfem++WUnNI7oUcx++PNuJHsKTsNSznUhUyDexCj7jPHLg1qlF4s52elHUeoOxLPoA2vs0REGiqAijlygvfG2U04T59VP0ZgfD8ETqbxTGhD5ws6u7NEkcCVLtG4+uOMjuJB02bUJdvCBGIV36v9l+UGgvETp5m4w+jrd5ocxNGtFji86T4JLot\\\\\\\/jBwgtiCxxj3uTzqJrvYdEFPTGwD28PFTD+ddzKQuVg6R6Yy1XqbG0mGUh5yL8bQII+duqwMgJqA2PIsx8NZHhsrbbJkSD1Ij2J2Qvx07yG2U7tkVB47kl4vFCnTM3qOVpIHgTUQJG1wo2xvEoGeWetPRQHD5WYNdyH1+sx\\\\\\\/PZr3qbbNbRwTG6kpEf\\\\\\\/V9ZJIZZgHV7Iv04jYOS4DebOr3BJ6gdQxXKy7rXK7beumLLmBGdYu6YUscQZaRxM2\\\\\\\/\\\\\\\/IIWKwQsog\\\\\\\/l+g5\\\\\\\/MvCU1pcbT7Eq5NoWNnvYppIhLajay9SJ33pjKPiTQVBPBJN3rd3vvECCR4e4XmKrYrmCMM3yKRA8Y4+VP+q8p2tJxhwLflllTifBiQCaS8Bi8WMRTZCYpxicWOt5Di+axmGiz4LNVraZwHjaEQfsFsiml78tP0PJBfQlVZ5\\\\\\\/XEIezaNRzoTqvmDmOZ6eA85TNLQP\\\\\\\/YOiWwV+yKfykCM2XjDgFh2w8fwJmFzebHLtsR3q0xuidMxQfGuagB3GiQWbes8RENB5iUx9DJ0pK6yPxXnmHPE2z2Pz\\\\\\\/zUxS47qjmUs16yixRTB4dnUTFKjBNEcP+ts\\\\\\\/LmBJ4Fa21tpLLUzcK2xBtP9d9j\\\\\\\/TnRCAZ\\\\\\\/BbCKufOpJs47lP7OVMNfih0hjhW9mcPF0EwIbv2Rox07ShmUBeI3a5Vzhb\\\\\\\/ceEXA2VMz+1nF5VHY4bHTg3me7qdt2SmhqBIAmKk6eYZOF4dgbO70trlDqhKmbk7pPwT+kWkQid0e3He+xzOZ7DwcUFskkEhalATnQNhatcvyOLrWM3XO9XDEhbYDCerBuAssg9eLtMEeRXtdFGda5AnZCHg3sgM6zbMqcrDvuzgmeTkYVuWeufrWkIikQqK1E4i+RhszZHtOo\\\\\\\/Y16KG5oXnHbRZnKgjAtHEQe49LyS4qnvmMryZPBR8LheQtfF4ioP9U9g1kAJ6hVidbdUE6ceBGty07ipFwn6piA3hJ11IvM5+KBSSTJ+mbRK4BVahDovE0rvrnP7ygjasUrC9WiiVBKBH2tM\\\\\\\/YJheEwiHLuHI8YlQ99J0+hJWdzkpGcR5b+n80O\\\\\\\/DUB9ZbWja72blPESLGTnQIAjb8DlVobznhdhA9F0aCCBuRBWpCxqie9vGv318sbCcxUsac\\\\\\\/EREadeU+iXBmy8ZwbRlGK8OhbXBfbsQVA8F0jIwRamlWrn\\\\\\\/UQliSLIMr2Fd3JP5lP2XtyUS1CoKr2jq\\\\\\\/pa64myYsAhdMpo2os7THcklw9\\\\\\\/f5tWNtso2pH9hnaKCwCBTavYcy8vUcqXMAbi3fzTrVTbP4dcTUR7Ill7SipLNnNBYIXY7dJfYMu8ld0wCTQShMmEdI20Z+cJHcbmyhzcxZ1vyy0\\\\\\\/38JYDlBz1521yD\\\\\\\/MGF08toSBUprpe\\\\\\\/s74IaixhSU4pYzHP3cu3pJMjwBr27U9S8s6wgLIVIokLyREYyxbKKlgE8gh37mQGthd22cCQT3qiWgxMZirOOcHzIGQTJ53iZBmCscUpZ4s1YtEEkVUnFyDSiap7XNV13wPYGq1K\\\\\\\/LZs6ybJ9AGHcH3uSrzVZIZI67HeqZytlj9SbD+6\\\\\\\/fwemJVWwH1WR7OlLSpnctKiFiegHYiuE3kGMzegCvQXiun+RSAc+MkcKLyx0IagYUnm5KQmnlMw+asKdsnudXRuo99jJ7zanw7iAexfZgHVWNvcoxkH97mcNxuwHzjR+5ohL3UaD8uqmHEAYpR92zaW1GARVVuGUTkg1Xgf3hVXhqguZQXYpQAdlh41VmgG6TAH+wNqfMYbcX1EPRWKp9SVGJ3QKLCLhryQUt8aUra+LoG4LfFAzrz\\\\\\\/\\\\\\\/iJpNtiO6IM3BWvfuu+Tfi0Yzpm+lzC9bPCqj5ne2MPtfPV3uO7QNzJuNCzRSBxefdnZ4QE9\\\\\\\/90P6+8D9LNR4cGOdQ9y3O6yLvt\\\\\\\/ssdvQhT31RxuqxtoLUeTMD+mWfa4FrLOnYNjjUbK8HbqA+f4ASe+BASJYEyDVnAU8GjKtQYSzBeZrSFGAsMY9BIDW9B7LzCk4WSEqkWc2o3RyMsWgpfPEx9hZ8vgoUFTJQidgECgGCnsXKpZQysqoGweqZoDUpZI8QklYNBG\\\\\\\/Sutu3etL9QHeQGFVdXrPQXO9I017eewKXEJacy5BnllTzqxGmWVXYs1ud+mvLS2lqz77hsGxUu+Qo41u2XSls\\\\\\\/VtbbVUh70xNgK1AsOq+7LggA7vE2X1Arw9IJr9QEEdf\\\\\\\/KJ8gxr1j1a+F9OQhnB6J\\\\\\\/rG6Han32rg0Gz+KJM0sb4VDUsgdgZvIti0eyg6mxwizmYyqvUtS9izK1DRShGKrQ4g2BG+5XVQ7c4P6C\\\\\\\/uyJmVv\\\\\\\/G4ll0tzvGGdK3xwUkwDcnMYEA3CEye6hnm1eW1IyjNSX7bMIFYvoLaHYR9CzGXJ52tQrLjMZyhyNtlnTaWGX9mnT6pB5gRJ26z0XedQZYu07qO0+NYjXIbgSM6a9Pj5leaj5gmkmi+Ej4605NW9rLHBB4hwLf5FYdCOkiDQtTa2vfAU6SrIaTtnjN+T9doBskrcb+6R3wKa+axpzkjYOIIYaf881wG1aHB5tcQqaSSkpHR08QUO1OyLs4p9Fl4mTNAqGYhWgIkOV54NGmhuqwn6lQHUWiJ8jYBFm24n7Kp4SBbBUbcyTmTCCEDeTwgi0Mt5hO\\\\\\\/sOdB6D4LjPH82ZgwXmB+L549HKowMfEONFtXD33tkz4q0I6vlZLt1OfHXm2e3VVmxQv+Qxml\\\\\\\/MpiD0hDtftQl16t\\\\\\\/XPEcpfLmj\\\\\\\/oU+F7Dwv1SGtmkMNh3+Bbq07vw3YLaFxNmASbM6PWI8jMn55a9FSJ12J4uH5IT6XvqXMxMistYBq8LkhV\\\\\\\/YL2EZm8OHmTPbSGzAuE+N+0gFgZJ9ZubiufZY\\\\\\\/G6BGups+oxL4aYudMrlCzmBZlOcXMOnVefLjSM\\\\\\\/EkyQ6cNalOsBcZkbt58buT+Zn\\\\\\\/Nv\\\\\\\/2T06scaKCpHnGHSw0vEYazMEbiC+KkAEweXl0nLgo1hpE8WRup3TwrmN7YtuuqiJxV6PJRuAT5x0Y6cNljQk2dyxMAWwLwU3qYpfpft+e5ovR12plI8x0lwh1GRoqXIzusKN4XdLNQgra0eMqFEH7q6MsdtwtmgpryrP7V\\\\\\\/ccMjySv7c\\\\\\\/XIQuEIgQRNdRC+JJ71+CJ3fEprqQw+by5jWWL6nU23U6bwpTET74aMM9\\\\\\\/C7SBO90NwDut8NWnKqFlgYxZzLJn1Hsp3HLkfK9DamdGYTKNtO\\\\\\\/VHuJ6A+gPL0DmJg90BCRhIxJJO3V0HFHdf9xiyb7zOZQD6GFDOg+10+xKEJX8ZW7erVPXx3gT3aatcedKw5E4+mSIesq2CKxeq0BpwGhJCR1YRt39yxpItiJVf+zOQLbpXhA65IIRiVmruDoUiyzGMeapwfua2S0Tu4uM8mZat2wCy6h0nXIZtJG7ghhNesdiCelixNWaTPDw4lX25hk8FOdUZgE2huzIQxyTMBt794urMHQAnUigJgPm8aieTbUM\\\\\\\/XY855JzdHhft3cTsGyWhoNRT+BeqqrewWidIYOG\\\\\\\/cP6aWy9qr8eZiiNYF8GE7BHLJnbLaSDrr2oUvGwVmnuEAahAy\\\\\\\/ctFNGi1F7eeohR1kfZ7Fx3GxZ1NFyYu7LQKVF9iES\\\\\\\/aTwF\\\\\\\/mJegdp1vLPFdMMMGVwokKonbmm4iaz0kHd3SnGhSsmdCp3n3JRUm2ozBOl2XG1pa47LzwH8nXGL6CC9Y4DEXke\\\\\\\/n6uglgVe+6bJujm1EC4tN61V6OyarMADNA+6UZ0RYuoyDItnYSleDj1O0FY1lq540bBIJ+wnQ6BvRmqNqz1sTDu6URgfupAVQjqMTemq7JECglFGY6bbh0qS3t\\\\\\\/2\\\\\\\/DwbdRR0cGkzmk7WZt4iBPu\\\\\\\/CRu1\\\\\\\/f\\\\\\\/w69NFx\\\\\\\/Gqmlhc2Mviv7h+HjK78W7SFkGnt8Qgqf7gpl5oIMNgSBcrdLa++8w3dDLkF+4enK8smubGHcvCRCEBeyX495cYz8rnUaqiqDefGDgS+3nGLpQYpa4X8+0RAhKfSRhizMN1mTImhzU+l4DBl7fG0m3EAPip2JIzVdQS9gCysxuVJNGiD3RieuIGbvXEHTB8vBIETiO4IW81ID7hr1U\\\\\\\/crZ8VtCXg5aqEy\\\\\\\/irdiIz2bjgDkjMxrsptgtetokdu6iN\\\\\\\/oZz8oZ04rlTY0HcHjOhy9fWKwuDSFXfgGztpJ4V5b\\\\\\\/aMcIgLq\\\\\\\/N2fTEzs2lgjxu1Lt2rJuG9oIj6XfddzL3rPIyIevcy4E+IAFtnd6lmoUTaXwH3clnlxvDKWoz5BjsWlCaeBbHPxFzpZbXEfeH9SnTce0pOylHUregBgSCrx4Mqjs0f2WhNtKk9hbj2lsbs2zMckzWYQEXgDqO6XniDxtvvQwjFo7t8jrbsLmbAsb8kGd2JAn4E+tioXwKLLozQ1HioA3ocQ+VbKXI\\\\\\\/iTVD7PramFay1G6B0CAijtwGseJ1Tp3BNuaVnO9T5UhX375oB8UlH6C3mIpbtA4KYQC1lgv4o3wHyJlXjfkxDay52ROE9Hh2rZh6ZrJz4QN5C4IdvDSgVHr7U74TV7f8WF7ULG1R+cYh4cgCB295UqBnaA\\\\\\\/vmCvmQWxBUqJ2izjb4l9pqj9cxWIhbcUDnOHllsyPG5MnKOc+vwgykeefqP15LiHGHu8Z+GUOgxQEBKQJj9IWklI5au1wAHnFrjRPdRGTVw651WTJ2fmAWgZW9DAvqTDhUDqFwIuz3kiDwgI+WokntILSefoEnWZ66d6CWoLhnpYMhKsonOqOSwDreBj8cOPbttnxw397U4D67MFtTuG7U2ma9r0dSAEY\\\\\\\/spAyoj3WlAs8AGjgWwA\\\\\\\/LZkjSmEe9Bi58pRCs2TjldMY60nSuHNKyGmowwcMDE2jl+yG7ITGUR1gURMJp4YKYgVwHElwkQa3iaCvH\\\\\\\/Zgt2dD3D5Nh0VEhSBAfd4MbI5jZx0w9sQjZ5p2y7MHNwLICSV+h+OjKA2cuaXETlh7cud981IfIiQKx\\\\\\\/Wk7sMC0n0OfLMYplugLMxAI+GsjgmbEU92ayuzIA6nBKlEvo5QC80gbwjQGB79AkVyZXy8oiQZEPV0q9jvy7sYrXR3vdGygJz3FbLZGksA1DNxAIdSckGXXUWyi8yZNeWpS2sPqTE1MR0ORVDY1ChuSm1AmlGpNh58xrF7w7RVJCWwdDAgAJM94H0x2Vf4Iqb+zmC+BA\\\\\\\/HZV9JnRwpMmQ4qW\\\\\\\/PVH\\\\\\\/FjouxqbleCzqH2yDfBkZoYbCS18Jt5qMdWfRQXp+2HjwmLpvRXFGGqzi34fPeJlgClVwwLNhsaJW6ptrZW5zs6NksSywKti2zi6UPHqgyHI0c3+imfdSTrH0DS49ztMsqI6a\\\\\\\/gyfHGT96cwVL7wVsQ51aE2dl1+Uzloy1Tw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Npx+gIJU689dmqI559Cccw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229cc752d3792f747d6a2652b8e8496875d959f93e352c149d1e84665d3d4b3fe0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222\\\\\\\/QBcsY2Ly7QdoGLvgVB8d\\\\\\\/MsHC\\\\\\\/rigjD7os4wjsPYpKfpmEEGod4h9GFFfDTO8WoqDMIW\\\\\\\/3n855Ht3bXWexXJp3X8VgujmYS29RGc\\\\\\\/IXeSzWuA7qZkHDaMckbH3mSotbypgQtyd0GVShwxEEd0PCaSU9js1wpQoJI97WAVj7DMzlBFlc3\\\\\\\/pvPtOoY0tZbtrrxVjWzJ3iXMGc\\\\\\\/\\\\\\\/NWE2koX47XneHp8I\\\\\\\/L5c9r7ZSLRIN7jPDsnyREzO2VgKorlwOeglhiI0mWFuE3xm6dWy5PVB9NtTMF6X12Fy8IZefpCNJlVXBn+TKN2T1GvvAKUEAnUA9L0qt0GmTTFbQuK5adcm7kH2FG8FsxgZJjzcLjrFAF+EJI27UpiuP0lQdV6o40TpewaQdETPwQKYnnnJKgj8Ob8RDHy8mwDV0VouVv8OgO5lTZl7vPLg128w4RfIWfgCo\\\\\\\/DlD5atfQOhuhzYgJmwgXmm4O1xGzAPfIRsCOkGZg2JAGjXOcp7qOMI0RZaNdBQjIm8BgZeanjNcvnI+Tm1M3gvozND+mXsEE+N8LzFaYm+gwC8iclgUqnA5kQwB2FndjTwiZ6iw1pnNUZ4lnQQUTSOvTsEYhWwcejWyjPfPC1H+Kv9uHV4QVa\\\\\\\/6SRtXVbDWl7nbEs9kQDLCF04LAUhhanwc5jje7ZtBmteJR+1aG3aaEKLIwg\\\\\\\/CX2vJgsW2y3J4shrtEoRknJZplKSPylCEfRx6IVAhW1oXjkwQTSxlS2I1Nuuo7iYPQ\\\\\\\/P\\\\\\\/7sL3F\\\\\\\/949vWlw+wXQDVu+ZPEu7TmdQyd7ao6T25Vk8n6WeWN3unFQpL3nZzhVNQcXjuDGeLRZGzHsF50skFjQWR\\\\\\\/Oy\\\\\\\/\\\\\\\/bsN8KVm50luHxNVniFO14JO97y\\\\\\\/c\\\\\\\/uSlmzauWrRJ0ofWrgZygODtiqZ7guWIJD2OTj95Pm1xYRD7cxIkrJ5m9EHn0fRYLXp6TxIyH8ciTm+j5U+LBViehp+qGgVq+DUTo5lglyprXM1CxKa2OOniFokfSmNqCoU37Eizgq2bdIzAHtcGXCHQEaHr1wB1mo35GpBLXVswfOmA86LznKSRVKkznMpv+qDNUTDMOhHm7sSfOw+qOZIQPG+01ItrC4FfhFtuWzYy0Bofk2tbtZEyBDr7DK1OUjMm\\\\\\\/izP6ULdErdPIMLl3NcqewrvpWPBj6sd1MIhmXwMYQSIbcaWpyfCIaorvNvQEjvVf1ixIKb63QbPTKYjfJQT+STjU5YOMkQopJ9VFEE2\\\\\\\/6mtbNnuEsJe3\\\\\\\/oABySBgDfgYLuwSq7YmWtLRl4d+RA8IxgLUIvn\\\\\\\/ncEkR3w2WstxQx6itFlNVsAv0tTFYfLr5NJP6xczK0Y\\\\\\\/9bkJICyR4W7wws7cTnocEs4rl3Y+8y0eQb2p5xf+\\\\\\\/Fdc3VcsD6pqVYnPSvighM2UzuqPfaW2+JfXLj4cxAX7Yg4gZHxZHA+IxVD9FSBQGTMoKIWlLI8xo79j91pvBe6rBqfcSF7MgtGp+9rSIEvC0ZTT6WrSdQFIgew1bi\\\\\\\/HowrzHwEyPmq4scnEs9SYjx2cD6BvTR6sYEFC\\\\\\\/Scp0oIL1cF0Xlg0eGrpB6\\\\\\\/tz91CEIX2ET8QP9s5qdlelYgQk96Zk8mawss4RmvCu066RXeamvPxRnAWrGsbNUb+hHAyRtIbo24lNnUoInVRhc60YzUdjUD9vN6E2eXxmSGD1gLMgMsiAaZ9Xz97WAV99BTxIhji\\\\\\\/Gu3CIvJIcLxL0rNHS7JjBDNtZvsW6aJf2KhxwF78SGzr5gp0vE7+CFzhVrqwrhIag2ly\\\\\\\/45k8JKgfu1meaIhKXHTNE\\\\\\\/UzG+7W9RbuGWIHysS+6U8fOlIYWNw5\\\\\\\/A4iqKEdgH7tWbYDJrdxGkK4IFdPZukkSuOjT4Ng3cPXihLy1ZkjSISnmaxTa5zoNz\\\\\\\/AVQEStAgriIc5TuwnJoGBqqe0kacQzu49KHwkFcvsCqBwzVkH0KO7Mo7YXYim9yFN9p69KSOIU31UGDeNtm\\\\\\\/oXcxOOIy4G9zuYXR6IdoW9Sg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022r6cnt\\\\\\\/2aJM\\\\\\\/Ev4p9UR1HYg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229fe9d77e5e160d7529ec9a42500974bbb8b95ef31b0616fa153a07abe2bc3bfa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459430 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/399" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-N-XUQEp5Wi_W6SjwXs-ZZWqPQq0EAH8S_mL6BH_GMIQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/399" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2d9AKF1V1glJq9qtLBqsB6Klch7odiu2_3V3A_fA_Cg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/387" title="Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/409" title="A Mosaic Genetic Screen for Genes Involved in the Early Steps of Drosophila Oogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459430" id="node459430" data-pisa="ggg;3/3/399" data-pisa-master="ggg;3/3/399" data-apath="/ggg/3/3/399.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">On the Mutational Topology of the Bacterial Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Patricia L. Foster</span>, <span class="highwire-citation-author" data-delta="1">Andrew J. Hanson</span>, <span class="highwire-citation-author" data-delta="2">Heewook Lee</span>, <span class="highwire-citation-author" data-delta="3">Ellen M. Popodi</span> and <span class="highwire-citation-author" data-delta="4">Haixu Tang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">399-407;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005355 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459430" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patricia L. Foster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BL.%2BFoster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foster%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BL.%2BFoster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plfoster@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">plfoster@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew J. Hanson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BHanson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hanson%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BHanson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Heewook Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heewook%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeewook%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ellen M. Popodi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ellen%2BM.%2BPopodi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Popodi%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEllen%2BM.%2BPopodi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Haixu Tang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haixu%2BTang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaixu%2BTang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/3/399" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459430" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459430/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/399.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459430" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459430/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/399.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459430" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459430/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/399.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459430" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459430/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/3/399.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459430" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459430/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">By sequencing the genomes of 34 mutation accumulation lines of a mismatch-repair defective strain of <em>Escherichia coli</em> that had undergone a total of 12,750 generations, we identified 1625 spontaneous base-pair substitutions spread across the <em>E. coli</em> genome. These mutations are not distributed at random but, instead, fall into a wave-like spatial pattern that is repeated almost exactly in mirror image in the two separately replicated halves of the bacterial chromosome. The pattern is correlated to genomic features, with mutation densities greatest in regions predicted to have high superhelicity. Superimposed upon this pattern are regional hotspots, some of which are located where replication forks may collide or be blocked. These results suggest that, as they traverse the chromosome, the two replication forks encounter parallel structural features that change the fidelity of DNA replication.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate" rel="nofollow">mutation rate</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li><li class="kwd"><a href="/keyword/replication-fidelity" class="hw-term hw-article-keyword hw-article-keyword-replication-fidelity" rel="nofollow">replication fidelity</a></li><li class="kwd"><a href="/keyword/chromosome-structure" class="hw-term hw-article-keyword hw-article-keyword-chromosome-structure" rel="nofollow">chromosome structure</a></li><li class="kwd"><a href="/keyword/dna-polymerase-errors" class="hw-term hw-article-keyword hw-article-keyword-dna-polymerase-errors" rel="nofollow">DNA polymerase errors</a></li></ul><p id="p-5">A little more than 50 years ago, Seymour Benzer published a paper entitled âOn the Topology of the Genetic Fine Structureâ (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benzer 1961</a>). In that seminal report he asked âwhether mutable sites are distributed at random, or whether there exist portions of the map that are unusually crowded with or devoid of sites.â The advent of accessible whole-genome sequencing has now made it possible to begin to answer Benzerâs question for the entire genome. In the study reported here, we analyzed the distribution of spontaneous base-pair substitutions (BPSs) that accumulated in a mismatch-repair defective strain of <em>E. coli</em>, MutL<sup>â</sup>, whose mutation rate is approximately 150 times that of its wild-type parent (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lee <em>et al.</em> 2012</a>). To recover an unbiased spectrum of mutations we used the mutation accumulation (MA) strategy pioneered by Mukai (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mukai 1964</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Halligan and Keightley 2009</a>). The basic protocol starts with a founder individual from which parallel lines of clonal populations are established. Each line is then subjected to repeated single-individual bottlenecks. Because the effective population of each line is one, the ability of natural selection to fix or eliminate mutations is minimized. Applied to bacteria, the MA protocol consists of repeatedly streaking parallel lines for single colonies on agar plates; since colonies develop from a single cell, each passage is a one-cell bottleneck followed by approximately 28 generations of growth. Because most new mutations arise when the population is large, selection within the colony is minimal. In addition, most mutations have negligible fitness effects (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kibota and Lynch 1996</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lynch and Walsh 1998</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elena and Lenski 2003</a>). Application of whole-genome sequencing to parallel MA lines allows an unprecedented picture of nearly unbiased mutational profiles.</p><p id="p-6">Although replicative DNA polymerases make errors of the order of 10<sup>â4</sup>-10<sup>â5</sup> per nucleotide, DNA repair pathways reduce these to 10<sup>â9</sup>-10<sup>â11</sup> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fijalkowska <em>et al.</em> 2012</a>). Perhaps the most important of these pathways is the methyl-directed mismatch repair (MMR) system, which recognizes errors, destroys the newly synthesized DNA strand, and forces repolymerization templated by the old, presumably correct, DNA strand (reviewed in <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Marinus 2010</a>). Eliminating MMR not only increases the error rate of replication but can also reveal the nature of the mistakes that are made by the DNA polymerases.</p><p id="p-7">By sequencing the genomes of 34 parallel MA lines of the MutL<sup>â</sup> strain, each of which had gone through 19 single-colony bottlenecks for a total of 12,750 generations, we identified 1625 BPSs. The mutational spectrum of these mutations has recently been described (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Lee <em>et al.</em> 2012</a>). In this report we describe and analyze the spatial distribution of these BPSs across the <em>E. coli</em> genome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Bacteria strains and media</h3><p id="p-8">The bacterial strains and media used are described in <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Lee <em>et al.</em> (2012)</a>. The <em>E. coli</em> strain is PFM5, which is MG1655 <em>rph</em><sup>+</sup> Î(<em>mutL</em>).</p></div><div id="sec-3" class="subsection"><h3>MA protocol</h3><p id="p-9">MA lines of PFM5 were established from single colonies isolated on Miller Luria-Bertani (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Miller 1992</a>) agar plates. Subsequently, each day a well-isolated colony closest to a line drawn down the center of the plate was chosen for single-colony purification. After 19 such passages, resulting in 375 generations per line, genomic DNA purified from 34 lines was sequenced by the Beijing Genome Institute using the Illumina HiSeq2000 platform. Further details about the MA protocol, DNA extraction, sequencing, quality control, and single-nucleotide polymorphism (SNP) calling protocols are given in (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Lee <em>et al.</em> 2012</a>). Both the sequence reads and SNP calls have been deposited in the National Center for Biotechnology Information Sequence Read Archive, <a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>, accession no. SRA054031, and the SNP calls are available at the IU Scholar Works Repository, <a href="http://hdl.handle.net/2022/15192">http://hdl.handle.net/2022/15192</a>.</p></div><div id="sec-4" class="subsection"><h3>Analysis of the gap-size distribution</h3><p id="p-10">The distribution of the sizes of the intervals (gaps) between random events occurring either in time or in space is described by the exponential distribution; the mean of this distribution is the total length of time or distance divided by the number of events (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rice 1995</a>). Thus, the mean gap-size predicted for the BPS data is the length of the chromosome, 4640 kb, divided by the number of BPS, 1625, and equals 2.86 kb. However, analysis of the distribution of the gap-sizes was complicated by the fact that the location of BPSs in repeated sequences [insertion sequence (IS) elements, rRNA operons, and other smaller repeat sequences] could not be defined and thus were excluded from the data. To account for this complication, we eliminated all the gaps that included IS or rRNA operons (other repeated sequences are too small to bias the results). This adjustment, which left 1581 BPSs over 4284 kb for a mean gap-size of 2.71 kb, made little differenceâwith or without the repeat elements the distribution was significantly different from the expected distribution. Removal of these repeat elements did not change the pattern of mutational density across the chromosome (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/005355SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/FigureS1.pdf">Figure S1</a>A).</p></div><div id="sec-5" class="subsection"><h3>Bin analysis</h3><p id="p-11">The 1625 BPSs that accumulated in the MutL<sup>â</sup> strain were collected into 46 bins, each bin approximately 100 kb in size, starting at the origin of replication (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS1.pdf">Table S1</a>). The mean number of BPSs per bin was 35.3 with a variance of 95.2. The number of bins was chosen to be 46 because that number: (1) is close to the square-root of the sample size, a common ârule of thumbâ for choosing the number of intervals for a histogram; (2) describes the data clearly (see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>); and, (3) divides into the total number of nucleotides in the genome with an acceptably small remainder. However the mutation-density pattern was stable against changes in bin-size from about 50 kb (91 bins), which gave a mean number of BPSs per bin of 17.9 with a variance of 35.7, to about 220 kb (21 bins), which gave a mean number of BPS per bin of 77.4 with a variance of 307 (see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The pattern was also unaffected by a 50% displacement of the bin starting point and was preserved when the binning started at the traditional zero point of the <em>E. coli</em> chromosome instead of at the origin of replication (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/FigureS2.pdf">Figure S2</a>).</p></div><div id="sec-6" class="subsection"><h3>Wavelet transformation</h3><p id="p-12">Fourth-order Daubechies wavelet transforms were performed on the binned mutational data using the built-in program in Wolfram Mathematica 8.</p></div><div id="sec-7" class="subsection"><h3>Correlations and linear regressions with chromosomal features</h3><p id="p-13">Data describing various chromosomal features were obtained from the referenced sources (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a>) and collected into the same 46 bins used for the mutational data. Discrete data (such as number of transcription factor binding sites) were summed. Qualitative data (such as gene expression) were both summed and averaged and correlations with the mutational data calculated for each result; in every case the average value in each bin gave the best correlation. Pearsonâs product-moment correlation coefficient Ï<sub>p</sub>, was calculated for each of the 38 features listed in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a> compared with the binned mutation values. The product-moment correlation tests for a linear relationship; to ensure that no correlations were missed, we also calculated Spearmanâs rank correlation coefficient (corrected for ties), which is distribution free. With one exception, the rank correlation coefficients were similar to or less than the linear ones, indicating the relationships were better described by linear comparisons. The single exception was the relationship between the mutations and the number of relaxation-induced genes, which had a non-significant (<em>p</em> and <em>q</em> = 0.09) rank correlation coefficient of 0.235 but a linear coefficient of 0.077 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a>).</p><p id="p-14">The significance of the correlation coefficients was evaluated from the two-tailed t-distribution [using the web-based tools at <a href="http://www.wessa.net/">http://www.wessa.net/</a> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wessa 2012</a>)]. The resulting <em>p</em>-values give the probable false positive rate, <em>i.e.</em>, the probability that a correlation that is actually zero will have a coefficient equal to or greater than the given value; the traditional <em>p</em>-value cutoff for accepting the correlation is 0.05. We also computed <em>q</em>-values, which are a measure of the false discovery rate commonly used when multiple comparisons are being simultaneous considered (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Storey and Tibshirani 2003</a>). These were computed using the Qvalue program (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Storey and Tibshirani 2003</a>) with the false discovery rate set to 0.05. The <em>q</em>-values give the probability that a correlation considered to be significant is actually zero; applied to our results, among the eleven correlations with a <em>q</em>-value â¤ 0.05 (see <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) the probable number of false-positive results is less than one. The <em>q</em>-values so calculated are conservative because many of the comparisons were not independent (<em>e.g.</em>, up-regulated and down-regulated genes are subsets of the total number of affected genes) and so the number of comparisons is inflated.</p><p id="p-15">Linear regressions were performed using the built-in function of the XLSTAT package (Addinsoft SARL) for Microsoft Excel (Microsoft Corp). The best model was chosen by minimizing Akaikeâs Information Criterion (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akaike 1973</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-16">The <em>E. coli</em> strain used for this study has a nonpolar deletion of the <em>mutL</em> gene, which encodes MutL, a protein required for MMR (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Lee <em>et al.</em> 2012</a>). A total of 1625 BPSs were recovered from the MutL<sup>â</sup> strain and their spatial distribution across the 4640 Kb <em>E. coli</em> genome is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Although the distribution appears to be random by eye, it is not. If the BPSs were distributed at random, the size of the gaps between adjacent BPSs would have an exponential distribution. However, the actual distribution is significantly different from expected (Ï<sup>2</sup> â10<sup>4</sup>; <em>p</em> âª 0.0001). Below a gap size of about 0.2 kb the observed distribution fits the expected distribution fairly well, but gap-sizes began to deviate from expected at values greater than 0.2 kb (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Over most of the range the gap sizes are smaller than expected, meaning that the mutations are clustered.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/399/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of BPSs across the genome. The 4640-kb E. coli chromosome is shown with the traditional zero point at the top. Each blue line with a red cap indicates the position of a BPS that accumulated in the MutL&#x2212; strain; the thickness of each line is equivalent to approximately one kb, centered on the position of the BPS. OriC, the origin of replication at 3924 kb; TerD, TerA, TerC, and TerB, strong termination sites at 1279 kb, 1340 kb, 1607 kb, and 1682 kb, respectively (Duggin and Bell 2009). Clockwise (rightward) moving forks are halted at TerC or TerB and counterclockwise (leftward) moving forks are halted at TerA or TerD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1898225262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distribution of BPSs across the genome. The 4640-kb &lt;em&gt;E. coli&lt;/em&gt; chromosome is shown with the traditional zero point at the top. Each blue line with a red cap indicates the position of a BPS that accumulated in the MutL&lt;sup&gt;&#x2212;&lt;/sup&gt; strain; the thickness of each line is equivalent to approximately one kb, centered on the position of the BPS. OriC, the origin of replication at 3924 kb; TerD, TerA, TerC, and TerB, strong termination sites at 1279 kb, 1340 kb, 1607 kb, and 1682 kb, respectively (Duggin and Bell 2009). Clockwise (rightward) moving forks are halted at TerC or TerB and counterclockwise (leftward) moving forks are halted at TerA or TerD.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/399/F1.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/399/F1.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/399/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/399/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462509" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-17" class="first-child">The distribution of BPSs across the genome. The 4640-kb <em>E. coli</em> chromosome is shown with the traditional zero point at the top. Each blue line with a red cap indicates the position of a BPS that accumulated in the MutL<sup>â</sup> strain; the thickness of each line is equivalent to approximately one kb, centered on the position of the BPS. OriC, the origin of replication at 3924 kb; TerD, TerA, TerC, and TerB, strong termination sites at 1279 kb, 1340 kb, 1607 kb, and 1682 kb, respectively (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duggin and Bell 2009</a>). Clockwise (rightward) moving forks are halted at TerC or TerB and counterclockwise (leftward) moving forks are halted at TerA or TerD.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/399/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of the gaps between BPSs. Shown are the four quartiles of a quantile-quantile (Q-Q) plot (Rice 1995) of the observed sizes of the intervals (gaps) between BPSs vs. the sizes predicted by an exponential distribution. For this analysis, gaps that contained large repeat elements have been removed (see Materials and Methods); this procedure left 1581 BPSs distributed over 4284 kb of the chromosome, giving a mean gap size of 2.71 kb. The observed distribution is significantly different than expected (&#x3C7;2 &#x2248;104; p &#x226A; 0.0001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1898225262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distribution of the gaps between BPSs. Shown are the four quartiles of a quantile-quantile (Q-Q) plot (Rice 1995) of the observed sizes of the intervals (gaps) between BPSs &lt;em&gt;vs.&lt;/em&gt; the sizes predicted by an exponential distribution. For this analysis, gaps that contained large repeat elements have been removed (see &lt;em&gt;Materials and Methods&lt;/em&gt;); this procedure left 1581 BPSs distributed over 4284 kb of the chromosome, giving a mean gap size of 2.71 kb. The observed distribution is significantly different than expected (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; &#x2248;10&lt;sup&gt;4&lt;/sup&gt;; &lt;em&gt;p&lt;/em&gt; &#x226A; 0.0001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/399/F2.medium.gif" width="440" height="418"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/399/F2.medium.gif" width="440" height="418"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/399/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/399/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462629" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-18" class="first-child">The distribution of the gaps between BPSs. Shown are the four quartiles of a quantile-quantile (Q-Q) plot (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Rice 1995</a>) of the observed sizes of the intervals (gaps) between BPSs <em>vs.</em> the sizes predicted by an exponential distribution. For this analysis, gaps that contained large repeat elements have been removed (see <em>Materials and Methods</em>); this procedure left 1581 BPSs distributed over 4284 kb of the chromosome, giving a mean gap size of 2.71 kb. The observed distribution is significantly different than expected (Ï<sup>2</sup> â10<sup>4</sup>; <em>p</em> âª 0.0001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">A more interesting departure from random is revealed by the locations of the mutations. In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> the 1625 BPSs are collected in 46 bins, each bin approximately 100 kb wide, starting at the origin of replication. The left and right sides of <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> display the same data collected in opposite directions, reproducing the movement of the two replication forks as they traverse the chromosome, but arranged as if each fork continued around the chromosome and back to the origin (<em>i.e.</em>, the lower left quadrant is the inverted mirror image of the upper right quadrant, and <em>vice versa</em>); the color changes from blue to magenta at the midpoint of the chromosome. <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a> reveals that the mutations are distributed across the genome in a large-scale, periodic pattern that is repeated nearly in mirror-image in the two replichores. The distributions of the mutations per bin between the two replichores are highly correlated (Pearsonâs correlation coefficient, Ï<sub>p</sub>, = 0.701, <em>p</em> = 0.0002) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>); thus, the pattern appears to reflect genomic features that affect the fidelity of the two replisomes in parallel as they move from the origin to the terminus. Starting from OriC (the top of <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>) the density of mutations in each replichore drops for about 500 kb (bins 2â6, 45â42), then increases, reaching a peak at about 1000 kb from OriC (bins 10 and 37). The density then drops to another low at about 1400 kb from OriC (bins 13â15, 34â32) before climbing to a second peak at about 2100 kb from OriC (bins 20, 25), which is immediately before the terminus region. The density of mutations drops again at the point where each fork would pass the strong termination sites TerD and TerA (located in bins 20 and 21, respectively) for the clockwise-moving fork and TerB and TerC (both located in bin 24) for the counter-clockwise moving fork. Within the inner termination region (bins 21â24) the mutation density is relatively constant and close to the average across the whole genome.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/399/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of binned BPSs across the genome. (A) The 1625 BPSs that accumulated in the MutL&#x2212; strain collected into 46 bins, each bin approximately 100 kb in size, starting at the origin of replication. Each side of the histogram displays the binning in opposite directions, reproducing the movement of the two replication forks as if each continued across the whole chromosome (i.e., the lower left quadrant is the inverted mirror image of the upper right quadrant, and vice versa); the color changes from blue to magenta at the midpoint of the chromosome. The strong termination sites, TerA and TerC, in bins 21 and 24, respectively, are indicated; not indicated are the alternative strong termination sites, TerD in bin 20 and TerB in bin 24. The four MDs defined by the efficiency of recombinational exchange within each domain (Niki et al. 2000; Valens et al. 2004) are indicated: green, Ori MD; cherry, left MD and right MD; cyan, terminal MD. (B) The bins in (A) reoriented so to directly compare the mutational pattern of the two replichores. Note that the Ter sites are not symmetrically oriented with respect to the origin; the midpoint of the chromosome lies between bins 23 and 24, close to TerC. Thus, the peaks in mutational density in bins 20 and 25 surround the terminal region bounded by TerA and TerC, whereas the peak in bin 27 is well outside of this region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1898225262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distribution of binned BPSs across the genome. (A) The 1625 BPSs that accumulated in the MutL&lt;sup&gt;&#x2212;&lt;/sup&gt; strain collected into 46 bins, each bin approximately 100 kb in size, starting at the origin of replication. Each side of the histogram displays the binning in opposite directions, reproducing the movement of the two replication forks as if each continued across the whole chromosome (&lt;em&gt;i.e.&lt;/em&gt;, the lower left quadrant is the inverted mirror image of the upper right quadrant, and &lt;em&gt;vice versa&lt;/em&gt;); the color changes from blue to magenta at the midpoint of the chromosome. The strong termination sites, TerA and TerC, in bins 21 and 24, respectively, are indicated; not indicated are the alternative strong termination sites, TerD in bin 20 and TerB in bin 24. The four MDs defined by the efficiency of recombinational exchange within each domain (Niki &lt;em&gt;et al.&lt;/em&gt; 2000; Valens &lt;em&gt;et al.&lt;/em&gt; 2004) are indicated: green, Ori MD; cherry, left MD and right MD; cyan, terminal MD. (B) The bins in (A) reoriented so to directly compare the mutational pattern of the two replichores. Note that the Ter sites are not symmetrically oriented with respect to the origin; the midpoint of the chromosome lies between bins 23 and 24, close to TerC. Thus, the peaks in mutational density in bins 20 and 25 surround the terminal region bounded by TerA and TerC, whereas the peak in bin 27 is well outside of this region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/399/F3.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/399/F3.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/399/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/399/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462572" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-20" class="first-child">The distribution of binned BPSs across the genome. (A) The 1625 BPSs that accumulated in the MutL<sup>â</sup> strain collected into 46 bins, each bin approximately 100 kb in size, starting at the origin of replication. Each side of the histogram displays the binning in opposite directions, reproducing the movement of the two replication forks as if each continued across the whole chromosome (<em>i.e.</em>, the lower left quadrant is the inverted mirror image of the upper right quadrant, and <em>vice versa</em>); the color changes from blue to magenta at the midpoint of the chromosome. The strong termination sites, TerA and TerC, in bins 21 and 24, respectively, are indicated; not indicated are the alternative strong termination sites, TerD in bin 20 and TerB in bin 24. The four MDs defined by the efficiency of recombinational exchange within each domain (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Niki <em>et al.</em> 2000</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Valens <em>et al.</em> 2004</a>) are indicated: green, Ori MD; cherry, left MD and right MD; cyan, terminal MD. (B) The bins in (A) reoriented so to directly compare the mutational pattern of the two replichores. Note that the Ter sites are not symmetrically oriented with respect to the origin; the midpoint of the chromosome lies between bins 23 and 24, close to TerC. Thus, the peaks in mutational density in bins 20 and 25 surround the terminal region bounded by TerA and TerC, whereas the peak in bin 27 is well outside of this region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The pattern shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A</a> is unlikely to be obtained by chance. A random distribution of 1625 mutations in 46 bins is expected to be Poisson with a variance equal to the mean of 35.3. But the actual distribution has a variance of 95.2 and differs significantly from the Poisson (Ï<sup>2</sup> = 143, <em>p</em> â 10<sup>â12</sup>). We performed 1000 Monte Carlo simulations of 1625 mutations reproducing the MutL<sup>â</sup> mutational spectrum, distributed at random across the genome and then gathered into 46 bins. The variances of the 1000 distributions averaged 35.7 and ranged from 15.3 to 63.6. Thus, the maximum variance of 1000 random distributions was only two-thirds that of the actual data. Comparisons of a subset of 100 of the simulations to the actual data showed that all the distributions were significantly different from the symmetrical pattern shown in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3A</a> (the minimum Ï<sup>2</sup> = 103, <em>p</em> = 2 Ã 10<sup>â6</sup>).</p><p id="p-22">The underlying symmetrical pattern of the mutational density is revealed clearly by the Daubechies wavelet transform shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>. Applying this wavelet smoothing across continuously varying bin sizes preserved the underlying histogram structure in the range of bin sizes from roughly 50 kb (91 bins) to 220 kb (21 bins), with degradation outside that range (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). The Daubechies wavelet transform also reveals two regional hotspots, one 600â700 kb from the origin (bin 7) in the right replichore and one before the terminal region in the left replichore (bin 27), that do not fit the symmetrical pattern.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/399/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Wavelet transformations of the mutational distribution. (A) The numbers of BPSs in each of 46 bins are plotted in green with the bins arranged proceeding clockwise across the chromosome starting and ending at the origin of replication (OriC). The Daubechies wavelet transform is plotted in blue. (B) Two-factor model. A linear regression of the number of mutations against 10 chromosomal features significantly correlated with the mutational data (see Table S3) produced an optimal two-factor model. The model is: mutations = 16 + (7.8 &#xD7; HU) + (1.5 &#xD7; FIS), where HU indicates the HU response per gene minus hupAB and FIS indicates the number of genes up-regulated in a Fis&#x2212; mutant (see Table 1). For this model, r2 = 0.335, p = 0.0002. The numbers of mutations per bin predicted by this model (dashed, magenta) and the corresponding Daubechies wavelet transform curve (blue) are compared to the observed numbers of mutations per bin (green). (C) Five-factor model. To produce the five factor model, a linear regression of the number of mutations was performed against the ten features that were used to generate the two-factor model plus seven additional features that had positive or negative correlation coefficients with the mutational data of &#x2248; 0.2 (see Table S3). The model is: mutations = 68 &#x2013; (97 &#xD7; CAI) + (5.0 &#xD7; HU) + (1.0 &#xD7; FIS) + (0.9 &#xD7; RR) &#x2013; (0.8 &#xD7; H-NS), where HU and FIS are defined as previously; CAI indicates the average gene CAI; H-NS indicates the number of genes down-regulated in an H-NS mutant; and RR indicates the number of relaxation repressed genes. For this model, r2 = 0.435, p = 0.0003. The numbers of mutations per bin predicted by this model (dashed, magenta) and the corresponding Daubechies wavelet transform curve (blue) are compared to the observed numbers of mutations per bin (green)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1898225262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Wavelet transformations of the mutational distribution. (A) The numbers of BPSs in each of 46 bins are plotted in green with the bins arranged proceeding clockwise across the chromosome starting and ending at the origin of replication (OriC). The Daubechies wavelet transform is plotted in blue. (B) Two-factor model. A linear regression of the number of mutations against 10 chromosomal features significantly correlated with the mutational data (see Table S3) produced an optimal two-factor model. The model is: mutations = 16 + (7.8 &#xD7; HU) + (1.5 &#xD7; FIS), where HU indicates the HU response per gene minus &lt;em&gt;hupAB&lt;/em&gt; and FIS indicates the number of genes up-regulated in a Fis&lt;sup&gt;&#x2212;&lt;/sup&gt; mutant (see Table 1). For this model, r&lt;sup&gt;2&lt;/sup&gt; = 0.335, &lt;em&gt;p&lt;/em&gt; = 0.0002. The numbers of mutations per bin predicted by this model (dashed, magenta) and the corresponding Daubechies wavelet transform curve (blue) are compared to the observed numbers of mutations per bin (green). (C) Five-factor model. To produce the five factor model, a linear regression of the number of mutations was performed against the ten features that were used to generate the two-factor model plus seven additional features that had positive or negative correlation coefficients with the mutational data of &#x2248; 0.2 (see Table S3). The model is: mutations = 68 &#x2013; (97 &#xD7; CAI) + (5.0 &#xD7; HU) + (1.0 &#xD7; FIS) + (0.9 &#xD7; RR) &#x2013; (0.8 &#xD7; H-NS), where HU and FIS are defined as previously; CAI indicates the average gene CAI; H-NS indicates the number of genes down-regulated in an H-NS mutant; and RR indicates the number of relaxation repressed genes. For this model, r&lt;sup&gt;2&lt;/sup&gt; = 0.435, &lt;em&gt;p&lt;/em&gt; = 0.0003. The numbers of mutations per bin predicted by this model (dashed, magenta) and the corresponding Daubechies wavelet transform curve (blue) are compared to the observed numbers of mutations per bin (green).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/399/F4.medium.gif" width="209" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/399/F4.medium.gif" width="209" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/399/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/399/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462479" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-23" class="first-child">Wavelet transformations of the mutational distribution. (A) The numbers of BPSs in each of 46 bins are plotted in green with the bins arranged proceeding clockwise across the chromosome starting and ending at the origin of replication (OriC). The Daubechies wavelet transform is plotted in blue. (B) Two-factor model. A linear regression of the number of mutations against 10 chromosomal features significantly correlated with the mutational data (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS3.pdf">Table S3</a>) produced an optimal two-factor model. The model is: mutations = 16 + (7.8 Ã HU) + (1.5 Ã FIS), where HU indicates the HU response per gene minus <em>hupAB</em> and FIS indicates the number of genes up-regulated in a Fis<sup>â</sup> mutant (see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). For this model, r<sup>2</sup> = 0.335, <em>p</em> = 0.0002. The numbers of mutations per bin predicted by this model (dashed, magenta) and the corresponding Daubechies wavelet transform curve (blue) are compared to the observed numbers of mutations per bin (green). (C) Five-factor model. To produce the five factor model, a linear regression of the number of mutations was performed against the ten features that were used to generate the two-factor model plus seven additional features that had positive or negative correlation coefficients with the mutational data of â 0.2 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS3.pdf">Table S3</a>). The model is: mutations = 68 â (97 Ã CAI) + (5.0 Ã HU) + (1.0 Ã FIS) + (0.9 Ã RR) â (0.8 Ã H-NS), where HU and FIS are defined as previously; CAI indicates the average gene CAI; H-NS indicates the number of genes down-regulated in an H-NS mutant; and RR indicates the number of relaxation repressed genes. For this model, r<sup>2</sup> = 0.435, <em>p</em> = 0.0003. The numbers of mutations per bin predicted by this model (dashed, magenta) and the corresponding Daubechies wavelet transform curve (blue) are compared to the observed numbers of mutations per bin (green).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/399/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The stability of wavelet transform against changes in bin size. Daubechies wavelet transforms were applied to the binned data with the number of bins per chromosome varying from 11 to 141 (corresponding to bin sizes of 422 kb to 33 kb), resulting in the curves plotted here. Black lines indicate 21, 46, and 91 bins, with the terminal black line indicating the raw, untransformed 46-bin data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1898225262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The stability of wavelet transform against changes in bin size. Daubechies wavelet transforms were applied to the binned data with the number of bins per chromosome varying from 11 to 141 (corresponding to bin sizes of 422 kb to 33 kb), resulting in the curves plotted here. Black lines indicate 21, 46, and 91 bins, with the terminal black line indicating the raw, untransformed 46-bin data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/399/F5.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/399/F5.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/399/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/399/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462481" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5â</span> <p id="p-24" class="first-child">The stability of wavelet transform against changes in bin size. Daubechies wavelet transforms were applied to the binned data with the number of bins per chromosome varying from 11 to 141 (corresponding to bin sizes of 422 kb to 33 kb), resulting in the curves plotted here. Black lines indicate 21, 46, and 91 bins, with the terminal black line indicating the raw, untransformed 46-bin data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We have attempted to correlate the periodic pattern shown in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3A</a> with various features of the chromosome gathered into the same 46 bins. <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> gives the significant correlations; nonsignificant correlations are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a>. The MutL<sup>â</sup> mutational spectrum is dominated by A:T &gt; G:C transitions (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Lee <em>et al.</em> 2012</a>); thus, the number of mutations and the A:T content of each bin are correlated (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). However, the mutational pattern is not simply due to the distribution of A:T base pairs across the genome; if the numbers of mutations at A:T and G:C base pairs are normalized by the A:T and G:C content of each bin, the pattern does not differ significantly from the one shown in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3A</a> (Ï<sup>2</sup> = 0.3, <em>p</em> = 1.00; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/FigureS1.pdf">Figure S1</a>B).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462548/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462548/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Correlations of the numbers of mutations per bin with various genomic features</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">Gene expression, as measured by the number of genes that are transcriptionally active under a variety of conditions, has been found to oscillate across the genome with periodicities of 600â800 kb (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allen <em>et al.</em> 2003</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2006</a>). These oscillations, particularly in the terminus region, correspond roughly to some of the peaks and valleys in the mutational density pattern. However, although the distribution of mutations is symmetrical in the two replichores, the distribution of gene expression is not, and the overall correlation between the number of mutations and gene expression values used in <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Allen <em>et al.</em> (2006)</a> is low (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a>). Gene expression measured in a number of other studies was also poorly correlated with the pattern of mutational density reported here (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a>). There is a significant negative correlation between the number of mutations and the average gene Codon Adaptation Index (CAI) (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>); because CAI is indicative of gene expression, this correlation suggests that mutation rates are low in regions of the chromosome that contain a high density of highly expressed genes. However, the average CAI and the A:T content per bin are negatively correlated (Ï<sub>p</sub> = â0.744, <em>p</em> âª 0.0001), so it is not clear which of these two is the most important causal factor.</p><p id="p-34">In a recent study, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Martincorena <em>et al.</em> (2012</a>) analyzed the numbers of synonymous BPSs in 2659 genes in 34 diverged <em>E. coli</em> genomes. The distribution of synonymous diversity values (related to BPSs per gene) across the genome has some similarity to the mutational pattern we observe, again particularly in the terminus region (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a> in <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Martincorena <em>et al.</em> 2012</a>). The synonymous diversity values in the right replichore declined near the positions of TerD and TerA, the same point at which the mutational densities reported here also declined. In the left replichore, the synonymous diversity values declined at about 2.1 Mb, well before the replisome would reach TerB and TerC but still within the general terminus region. There may be other correlations between the distribution of synonymous BPSs and the mutational patterns we observed that are yet to be discovered by further analysis of the data. However, the examples given by <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Martincorena <em>et al.</em> (2012)</a> of a cold spot at 848 kb and a hot spot extending from 4314 kb to 4334 kb are not corroborated by our data; when the number of BPSs that we observed in each of these regions was normalized to size of the region in nucleotides, the density of BPSs was lower in the âhotâ spot than in the âcoldâ spot. In addition, the mutation-density distribution reported here is not correlated with the gene expression data reported in <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Martincorena <em>et al.</em> (2012)</a> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a>), whereas they reported a negative correlation of their synonymous diversity values with gene expression (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Martincorena <em>et al.</em> 2012</a>).</p><p id="p-35">The <em>E. coli</em> chromosome has been divided into four macrodomains (MDs) defined by the efficiency of recombinational exchange within each domain (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Niki <em>et al.</em> 2000</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Valens <em>et al.</em> 2004</a>); the approximate borders of the MDs are shown on the histograms in <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/FigureS2.pdf">Figure S2</a>. The mutational density pattern does not correspond well to the MDs because, unlike the mutation pattern, the MDs are not symmetrically located in the two replichores. In general, mutational densities reported here tended to be low in the Ori MD and in the right and left MDs, and greater in the terminal MD and in the nonstructured regions that flank the Ori MD (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/FigureS2.pdf">Figure S2</a>). Comparative studies of diverged strains have also indicated that mutation rates are greater at the terminus than at the origin (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Touchon <em>et al.</em> 2009</a>; <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Martincorena <em>et al.</em> 2012</a>); although this conclusion is not well supported by experimental data in bacteria (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ochman 2003</a>), it is supported by studies of <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agier and Fischer 2012</a>).</p><p id="p-36">The <em>E. coli</em> chromosome is organized into supercoiling domains that are constrained by nucleoid-associated proteins (NAPs) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Travers and Muskhelishvili 2005</a>). Long-range periodicities in supercoiling-sensitive gene expression have been found across the <em>E. coli</em> genome and correlated to gyrase binding sites (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Jeong <em>et al.</em> 2004</a>). Clusters of supercoiling-sensitive genes have also been found at the edges of the Ter domain, corresponding to binding sites for two NAPs, FIS and H-NS (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ussery <em>et al.</em> 2001</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Scolari <em>et al.</em> 2011</a>). The histone-like protein, HU, does not have well defined binding sites, but genes that respond positively to loss of HU are located toward OriC and genes that respond negatively to loss of HU are located toward the terminus (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sobetzko <em>et al.</em> 2012</a>). The mutational patterns reported here correspond in part to these patterns (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). The distribution of BPS-density is positively correlated to the distributions of genes that are up-regulated in the absence of FIS (presumably genes that are repressed by FIS binding), genes that are down-regulated in a gyrase mutant (presumably genes that are activated by negative supercoiling), and genes that are down-regulated in the absence of HU (presumably genes that are activated by HU binding). The last two correlations may reflect the fact that HU and gyrase act together to promote superhelicity (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Malik <em>et al.</em> 1996</a>) and suggest that the mutation rate is increased in regions of the genome with high superhelical density.</p><p id="p-37">To find a predictive model for the observed pattern of mutational densities, we performed linear regressions with 10 chromosomal features that were significantly correlated with the binned mutational data (positive or negative correlation coefficients &gt; 0.3; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS3.pdf">Table S3</a>). Just two features, the average HU response (HU-activated genes) and the number of FIS-repressed genes, account for 33.5% of the variation in the mutational data (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). The inclusion of seven additional features that are less well correlated with the mutational data (positive or negative correlation coefficients â 0.2; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS3.pdf">Table S3</a>), increased the predictive power of the model. The five-factor model (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>), which accounts for 43.5% of the variation in the mutational data, adds as a positive factor the number of relaxation repressed genes and as negative factors the average gene CAI and the number of genes down-regulated in a H-NS mutant. Adding more features contributed little to the predictive value of the resulting models.</p><p id="p-38">Although both the two- and five-factor models predict the general shape of the mutational pattern, both also fail to account for the high density of mutations in certain bins, particularly 7, 20, and 25. If the values in these bins are replaced by the average, linear regressions will yield models accounting for up to 50% of the variation in the mutational data. It is, of course, possible to fit the mutational pattern with nonlinear combinations of many parameters (<em>e.g.</em>, a 35-parameter nonlinear equation achieves an 80% predictive value), but the biological meaning of such complex combinations of parameters is not obvious.</p></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-39">The BPSs that accumulated in the MutL<sup>â</sup> strain fell into wave-like patterns of increasing and decreasing mutational densities that are symmetrically arranged in the two replichores (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">4</a>). This pattern suggests that as the chromosome is replicated, the DNA polymerases encounter parallel chromosomal features that change the fidelity of replication. In general the mutational density was high in regions of the chromosome where gene expression is responsive to NAPs. Specifically, the mutation density was positively correlated to the density of HU-activated genes and FIS-repressed genes (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). Both of these NAPs are strong determinants of chromosome architecture; HU contributes to negative superhelicity by constraining supercoils and compacting the DNA into nucleosome-like particles, whereas FIS also constrains supercoils but produces branches in the supercoiled DNA (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schneider <em>et al.</em> 2001</a>). We interpret the correlation of the mutational density to regions where genes respond to these NAPs to be due to the structure of the chromosome, not to the gene expression <em>per se</em>. That is, the gene response is an indicator of regions of high chromosomal structure, and these regions result in high mutation rates when encountered by the replicating polymerases.</p><p id="p-40">Additional evidence for this interpretation comes from the correlation of the mutational density with genes that are down-regulated in a <em>gyrA</em> mutant (presumably genes that are activated by negative supercoiling) (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>) and the inclusion of relaxation-repressed genes as a positive factor in the 5-factor model (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C</a>). This hypothesis was not supported by the inclusion of the number of genes down-regulated in an H-NS mutant (presumably H-NS-activated genes) as a negative factor in the 5-factor model. However, H-NS, which also compacts DNA, acts antagonistically to FIS in gene regulation (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Travers and Muskhelishvili 2005</a>), suggesting that FIS and H-NS may have different effects on chromosome structure. All of these NAPs have patterned distributions and contribute to the spatial and temporal regulation of the superhelicity of the chromosome (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Sobetzko <em>et al.</em> 2012</a>). Recently, the density of the somatic mutations in human cancer cells was found to be correlated with chromatin organization (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schuster-Bockler &amp; Lehner 2012</a>).</p><p id="p-41">On the basis of previous studies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hudson <em>et al.</em> 2003</a>; <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Martincorena <em>et al.</em> 2012</a>), we expected mutational density to be correlated either negatively or positively with gene expression. However, the results were conflicting and no convincing correlations with gene expression were found (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1/TableS2.pdf">Table S2</a>). Mutational density was significantly negatively correlated with the average gene CAI, and this feature appeared in the 5-factor model (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4C</a>). Because high CAI is indicative of high gene expression, this correlation suggests that mutation rates are low in regions of highly expressed genes. Our failure to confirm this with gene-expression data may mean that the data that we analyzed was inappropriate to the conditions of our experiments.</p><p id="p-42">To reveal errors made during replication, we used a strain defective in MMR. MMR is initiated when a homodimer of MutS recognizes and binds a mismatch. MutS then recruits MutL and together they activate the downstream events that repair the mismatch (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Marinus 2010</a>). For the experiments reported here we inactivated MMR with a nonpolar knockout mutation of MutL. We chose to eliminate MutL instead of MutS because we thought it possible that loss of MutS has additional unwanted phenotypes. Examples are: MutS may travel with the replication fork via its interaction with the beta-clamp (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lopez de Saro <em>et al.</em> 2006</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pluciennik <em>et al.</em> 2009</a>); MutS alone can block recombination between diverged DNA <em>in vitro</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Worth <em>et al.</em> 1994</a>); and certain <em>mutS</em> mutant strains have been reported to have an unexplained loss of viability (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boe 1990</a>). Previous studies have found little difference in the mutation rates or mutational spectra of <em>mutL</em> and <em>mutS</em> mutant strains (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schaaper and Dunn 1987</a>). However, loss of MutL may also have unwanted phenotypes. After binding the mismatch, MutS leaves the site and rapidly slides along the DNA; another MutS homodimer may then repeat the process (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cho <em>et al.</em> 2012</a>). In the absence of MutL and with the resulting persistence of the mismatch, this process might continue for some time and possibly affect the topology of the chromosome and/or interfere with replication fork progression. Although this process might cause mutations to be clustered, it is difficult to see how it could account for the symmetrical pattern of mutational density that we observed unless the initiating mismatches themselves had a symmetrical spatial pattern.</p><p id="p-43">The data reported here suggest that as they traverse the chromosome, the two replication forks encounter in parallel regions of high DNA structure that result in a decrease in the fidelity of replication. Interestingly, sharp peaks of mutation density are located nearly symmetrically around the terminus region, close to the strong termination sites TerD and TerA on the right replichore and TerB and C on the left replichore (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figures 3A</a> and <a id="xref-fig-4-8" class="xref-fig" href="#F4">4A</a>). Ter sites create a replication fork trap that prevents the two forks from leaving the terminus region; the clockwise moving fork is halted at TerC or TerB and the counterclockwise moving fork is halted at TerA or TerD [additional Ter sites are positioned as backups (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Duggin and Bell 2009</a>)]. The <em>E. coli</em> terminus region is unique both structurally [with high curvature (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pedersen <em>et al.</em> 2000</a>) and low superhelicity (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Sobetzko <em>et al.</em> 2012</a>)] and genetically [with few genes and a high recombination rate (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Louarn <em>et al.</em> 1994</a>)]. The arrangement of the peaks in mutational density suggests that a spike in the mutation rate may occur after a replisome traverses the terminus and is blocked at the relevant Ter site, or when the two replisomes collide at a Ter site.</p><p id="p-44">Blocking or impeding the replication fork by collisions or by DNA structural features could impact replication fidelity in several ways. For example, if the replicating polymerase stalls and disengages from the DNA, one of the cellâs error-prone DNA polymerases could gain access and synthesize a tract of error-containing DNA. Alternatively, DNA repair pathways other than MMR may be less active in regions of highly structured DNA than in less structured regions, or highly structured DNA may be more susceptible to spontaneous damage. Further research is needed to resolve these questions. The simple linear models predict that, in general, mutation rates increase when the replication fork encounters regions of high structure and, possibly, low gene expression. Superimposed upon this pattern are regional hotspots, some of which appear to be correlated with replication fork blocks or collisions, and some of which remain a mystery.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank D. Osiecki, D. Simon, K. Storvik, J. Townes, N. Yahaya, and A. Ying Yi Tan for technical assistance; E. Eisenstadt, S. Finkel, M. Lynch, and members of the Lynch laboratory for discussions; and, A. Khodursky, G. Muskhelishvili, B. O. Palsson, G. Plunkett III, and P. Sobetzko for help in obtaining and interpreting datasets. This research was supported by Multidisciplinary University Research Initiative Award W911NF-09-1-0444 from the U.S. Army Research Office (to P.L.F. and H.T.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005355/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Data are available at the NCBI Sequence Read Archive (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) under accession no. SRA054031 and at the IU Scholar Works Repository (<a href="http://hdl.handle.net/2022/15192">http://hdl.handle.net/2022/15192</a>).</p></li><li class="fn" id="fn-10"><p id="p-46"><em>Communicating editor: R. Sclafani</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-02"><span class="received-label">Received </span>November 2, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-12-27"><span class="accepted-label">Accepted </span>December 27, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 Foster <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.1" data-doi="10.1093/molbev/msr280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The mutational profile of the yeast genome is shaped by replication</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">905</span>â<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr280%26rft_id%253Dinfo%253Apmid%252F22114361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI5LzMvOTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-book" id="cit-3.3.399.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span>â<span class="cit-article-title">Information theory and the extension of the maximum likelihood principle</span>, pp. <span class="cit-fpage">267</span>â<span class="cit-lpage">281</span> in <span class="cit-source">Second International Symposium on Information Theory</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">V. N.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Csaki</span>  <span class="cit-name-given-names">F.</span></span>. <span class="cit-publ-name">Academiai KiadÃ³</span>, <span class="cit-publ-loc">Budapest</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.3" data-doi="10.1128/JB.185.21.6392-6399.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrgard</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glasner</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Genome-scale analysis of the uses of the <em>Escherichia coli</em> genome: model-driven analysis of heterogeneous data sets</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">6392</span>â<span class="cit-lpage">6399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DAllen%26rft.auinit1%253DT.%2BE.%26rft.volume%253D185%26rft.issue%253D21%26rft.spage%253D6392%26rft.epage%253D6399%26rft.atitle%253DGenome-Scale%2BAnalysis%2Bof%2Bthe%2BUses%2Bof%2Bthe%2BEscherichia%2Bcoli%2BGenome%253A%2BModel-Driven%2BAnalysis%2Bof%2BHeterogeneous%2BData%2BSets%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.185.21.6392-6399.2003%26rft_id%253Dinfo%253Apmid%252F14563874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4NS8yMS82MzkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.4" data-doi="10.1371/journal.pcbi.0020002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palsson</span>  <span class="cit-name-given-names">B. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Long-range periodic patterns in microbial genomes indicate significant multi-scale chromosomal organization</span>. <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DAllen%26rft.auinit1%253DT.%2BE.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253De2%26rft.epage%253De2%26rft.atitle%253DLong-range%2Bperiodic%2Bpatterns%2Bin%2Bmicrobial%2Bgenomes%2Bindicate%2Bsignificant%2Bmulti-scale%2Bchromosomal%2Borganization.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020002%26rft_id%253Dinfo%253Apmid%252F16410829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16410829&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.5" data-doi="10.1073/pnas.47.3.403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benzer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span>â<span class="cit-article-title">On the topography of the genetic fine structure</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBenzer%26rft.auinit1%253DS.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D415%26rft.atitle%253DON%2BTHE%2BTOPOGRAPHY%2BOF%2BTHE%2BGENETIC%2BFINE%2BSTRUCTURE%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.47.3.403%26rft_id%253Dinfo%253Apmid%252F16590840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI0Ny8zLzQwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.3.399.6" data-doi="10.1038/embor.2009.232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farcas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geertz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhelyazkova</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brix</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Coordination of genomic structure and transcription by the main bacterial nucleoid-associated protein HU</span>. <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">59</span>â<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BReports%26rft.stitle%253DEMBO%2BReports%26rft.aulast%253DBerger%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D64%26rft.atitle%253DCoordination%2Bof%2Bgenomic%2Bstructure%2Band%2Btranscription%2Bby%2Bthe%2Bmain%2Bbacterial%2Bnucleoid-associated%2Bprotein%2BHU.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2009.232%26rft_id%253Dinfo%253Apmid%252F20010798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/embor.2009.232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20010798&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273044300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.3.399.7" data-doi="10.1038/sj.embor.7400729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mavathur</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geertz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muskhelishvili</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Homeostatic regulation of supercoiling sensitivity coordinates transcription of the bacterial genome</span>. <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">710</span>â<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BReports%26rft.stitle%253DEMBO%2BReports%26rft.aulast%253DBlot%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D710%26rft.epage%253D715%26rft.atitle%253DHomeostatic%2Bregulation%2Bof%2Bsupercoiling%2Bsensitivity%2Bcoordinates%2Btranscription%2Bof%2Bthe%2Bbacterial%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.embor.7400729%26rft_id%253Dinfo%253Apmid%252F16799466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.embor.7400729&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16799466&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238687800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.8" data-doi="10.1111/j.1365-2958.1990.tb00628.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boe</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Mechanism for induction of adaptive mutations in <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">597</span>â<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DBoe%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D597%26rft.epage%253D601%26rft.atitle%253DMechanism%2Bfor%2Binduction%2Bof%2Badaptive%2Bmutations%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.1990.tb00628.x%26rft_id%253Dinfo%253Apmid%252F2191182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.1990.tb00628.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2191182&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA85300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.9" data-doi="10.1016/j.str.2012.04.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">ATP alters the diffusion mechanics of MutS on mismatched DNA</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1264</span>â<span class="cit-lpage">1274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DCho%26rft.auinit1%253DW.%2BK.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D1264%26rft.epage%253D1274%26rft.atitle%253DATP%2Balters%2Bthe%2Bdiffusion%2Bmechanics%2Bof%2BMutS%2Bon%2Bmismatched%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2012.04.017%26rft_id%253Dinfo%253Apmid%252F22682745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2012.04.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22682745&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.10" data-doi="10.1016/j.jmb.2009.02.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duggin</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Termination structures in the <em>Escherichia coli</em> chromosome replication fork trap</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">532</span>â<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDuggin%26rft.auinit1%253DI.%2BG.%26rft.volume%253D387%26rft.issue%253D3%26rft.spage%253D532%26rft.epage%253D539%26rft.atitle%253DTermination%2Bstructures%2Bin%2Bthe%2BEscherichia%2Bcoli%2Bchromosome%2Breplication%2Bfork%2Btrap.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2009.02.027%26rft_id%253Dinfo%253Apmid%252F19233209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2009.02.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19233209&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.11" data-doi="10.1038/nrg1088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Evolution experiments with microorganisms: the dynamics and genetic bases of adaptation</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">457</span>â<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DElena%26rft.auinit1%253DS.%2BF.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D457%26rft.epage%253D469%26rft.atitle%253DEvolution%2Bexperiments%2Bwith%2Bmicroorganisms%253A%2Bthe%2Bdynamics%2Band%2Bgenetic%2Bbases%2Bof%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1088%26rft_id%253Dinfo%253Apmid%252F12776215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12776215&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183202600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.12" data-doi="10.1111/j.1574-6976.2012.00338.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fijalkowska</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonczyk</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">DNA replication fidelity in <em>Escherichia coli</em>: a multi-DNA polymerase affair</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1105</span>â<span class="cit-lpage">1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2012.00338.x%26rft_id%253Dinfo%253Apmid%252F22404288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjk6IjM2LzYvMTEwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.13" data-doi="10.1146/annurev.ecolsys.39.110707.173437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Spontaneous mutation accumulation studies in evolutionary genetics</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">151</span>â<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D40%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.14" data-doi="10.1093/nar/gkg651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergthorsson</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Transcription increases multiple spontaneous point mutations in <em>Salmonella enterica</em></span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">4517</span>â<span class="cit-lpage">4522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BE.%26rft.volume%253D31%26rft.issue%253D15%26rft.spage%253D4517%26rft.epage%253D4522%26rft.atitle%253DTranscription%2Bincreases%2Bmultiple%2Bspontaneous%2Bpoint%2Bmutations%2Bin%2BSalmonella%2Benterica%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg651%26rft_id%253Dinfo%253Apmid%252F12888512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xNS80NTE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-6" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.15" data-doi="10.1186/gb-2004-5-11-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khodursky</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Spatial patterns of transcriptional activity in the chromosome of <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DJeong%26rft.auinit1%253DK.%2BS.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DSpatial%2Bpatterns%2Bof%2Btranscriptional%2Bactivity%2Bin%2Bthe%2Bchromosome%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-11-r86%26rft_id%253Dinfo%253Apmid%252F15535862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-11-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15535862&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.16" data-doi="10.1038/381694a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kibota</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Estimate of the genomic mutation rate deleterious to overall fitness in E. coli</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">381</span>: <span class="cit-fpage">694</span>â<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKibota%26rft.auinit1%253DT.%2BT.%26rft.volume%253D381%26rft.issue%253D6584%26rft.spage%253D694%26rft.epage%253D696%26rft.atitle%253DEstimate%2Bof%2Bthe%2Bgenomic%2Bmutation%2Brate%2Bdeleterious%2Bto%2Boverall%2Bfitness%2Bin%2BE.%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F381694a0%26rft_id%253Dinfo%253Apmid%252F8649513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/381694a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8649513&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UR97900051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.17" data-doi="10.1073/pnas.1210309109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popodi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Rate and molecular spectrum of spontaneous mutations in the bacterium <em>Escherichia coli</em> as determined by whole-genome sequencing</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E2774</span>â<span class="cit-lpage">E2783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1210309109%26rft_id%253Dinfo%253Apmid%252F22991466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQxL0UyNzc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.18" data-doi="10.1074/jbc.M601264200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez de Saro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marinus</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OâDonnell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">The beta sliding clamp binds to multiple sites within MutL and MutS</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">14340</span>â<span class="cit-lpage">14349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M601264200%26rft_id%253Dinfo%253Apmid%252F16546997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjAvMTQzNDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.19" data-doi="10.1128/jb.176.24.7524-7531.1994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Louarn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francois</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patte</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louarn</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Hyperrecombination in the terminus region of the <em>Escherichia coli</em> chromosome: possible relation to nucleoid organization</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">7524</span>â<span class="cit-lpage">7531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLouarn%26rft.auinit1%253DJ.%26rft.volume%253D176%26rft.issue%253D24%26rft.spage%253D7524%26rft.epage%253D7531%26rft.atitle%253DHyperrecombination%2Bin%2Bthe%2Bterminus%2Bregion%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bchromosome%253A%2Bpossible%2Brelation%2Bto%2Bnucleoid%2Borganization.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.24.7524-7531.1994%26rft_id%253Dinfo%253Apmid%252F8002576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3Ni8yNC83NTI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-book" id="cit-3.3.399.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-source">Genetics and Analysis of Quantitative Traits</span>. <span class="cit-publ-name">Sinauer Associates, Inc.</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.21" data-doi="10.1006/jmbi.1996.0068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malik</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensaid</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouviere-Yaniv</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drlica</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Histone-like protein HU and bacterial DNA topology: suppression of an HU deficiency by gyrase mutations</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">66</span>â<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMalik%26rft.auinit1%253DM.%26rft.volume%253D256%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D76%26rft.atitle%253DHistone-like%2Bprotein%2BHU%2Band%2Bbacterial%2BDNA%2Btopology%253A%2Bsuppression%2Bof%2Ban%2BHU%2Bdeficiency%2Bby%2Bgyrase%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1996.0068%26rft_id%253Dinfo%253Apmid%252F8609614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1996.0068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8609614&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TU63900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.22" data-doi="10.1016/j.mrrev.2010.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marinus</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">DNA methylation and mutator genes in <em>Escherichia coli</em> K-12</span>. <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">705</span>: <span class="cit-fpage">71</span>â<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DMarinus%26rft.auinit1%253DM.%2BG.%26rft.volume%253D705%26rft.issue%253D2%26rft.spage%253D71%26rft.epage%253D76%26rft.atitle%253DDNA%2Bmethylation%2Band%2Bmutator%2Bgenes%2Bin%2BEscherichia%2Bcoli%2BK-12.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mrrev.2010.05.001%26rft_id%253Dinfo%253Apmid%252F20471491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mrrev.2010.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20471491&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282467600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.23" data-doi="10.1038/nature10995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martincorena</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seshasayee</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luscombe</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Evidence of non-random mutation rates suggests an evolutionary risk management strategy</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">95</span>â<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMartincorena%26rft.auinit1%253DI.%26rft.volume%253D485%26rft.issue%253D7396%26rft.spage%253D95%26rft.epage%253D98%26rft.atitle%253DEvidence%2Bof%2Bnon-random%2Bmutation%2Brates%2Bsuggests%2Ban%2Bevolutionary%2Brisk%2Bmanagement%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10995%26rft_id%253Dinfo%253Apmid%252F22522932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22522932&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303451900041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-book" id="cit-3.3.399.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-source">A Short Course In Bacterial Genetics: A Laboratory Manual and Handbook for Escherichia coli and Related Bacteria</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukai</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span>â<span class="cit-article-title">The genetic structure of natural populations of <em>Drosophila melanogaster</em>. I. Spontaneous mutation rate of polygenes controlling viability</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMukai%26rft.auinit1%253DT.%26rft.volume%253D50%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D19%26rft.atitle%253DTHE%2BGENETIC%2BSTRUCTURE%2BOF%2BNATURAL%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BMELANOGASTER.%2BI.%2BSPONTANEOUS%2BMUTATION%2BRATE%2BOF%2BPOLYGENES%2BCONTROLLING%2BVIABILITY%26rft_id%253Dinfo%253Apmid%252F14191352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo2OiI1MC8xLzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.26" data-doi="10.1101/gad.14.2.212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaichi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiraga</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Dynamic organization of chromosomal DNA in <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">212</span>â<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DNiki%26rft.auinit1%253DH.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D212%26rft.epage%253D223%26rft.atitle%253DDynamic%2Borganization%2Bof%2Bchromosomal%2BDNA%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.14.2.212%26rft_id%253Dinfo%253Apmid%252F10652275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTQvMi8yMTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.27" data-doi="10.1093/molbev/msg229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Neutral mutations and neutral substitutions in bacterial genomes</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2091</span>â<span class="cit-lpage">2096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg229%26rft_id%253Dinfo%253Apmid%252F12949125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8yMDkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.28" data-doi="10.1006/jmbi.2000.3787"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staerfeldt</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ussery</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">A DNA structural atlas for <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">907</span>â<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPedersen%26rft.auinit1%253DA.%2BG.%26rft.volume%253D299%26rft.issue%253D4%26rft.spage%253D907%26rft.epage%253D930%26rft.atitle%253DA%2BDNA%2Bstructural%2Batlas%2Bfor%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.3787%26rft_id%253Dinfo%253Apmid%252F10843847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.3787&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10843847&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087680400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.29" data-doi="10.1074/jbc.M109.054528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pluciennik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burdett</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukianova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OâDonnell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Involvement of the beta clamp in methyl-directed mismatch repair <em>in vitro</em></span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">32782</span>â<span class="cit-lpage">32791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.054528%26rft_id%253Dinfo%253Apmid%252F19783657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvNDcvMzI3ODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.3.399.30" data-doi="10.1093/nar/gkm831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puigbo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romeu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Vallve</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">HEG-DB: a database of predicted highly expressed genes in prokaryotic complete genomes under translational selection</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D524</span>â<span class="cit-lpage">D527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm831%26rft_id%253Dinfo%253Apmid%252F17933767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNi9zdXBwbF8xL0Q1MjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-book" id="cit-3.3.399.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-source">Mathematical Statistics and Data Analysis</span>. <span class="cit-publ-name">Wadsworth Publishing Company</span>, <span class="cit-publ-loc">Belmont, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.3.399.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Romero</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busby</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ott</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Dynamic distribution of SeqA protein across the chromosome of <em>Escherichia coli</em> K-12</span>. <abbr class="cit-jnl-abbrev">MBio.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00012</span>â<span class="cit-lpage">e10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio.%26rft.volume%253D1%26rft.spage%253De00012%26rft_id%253Dinfo%253Apmid%252F20689753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20689753&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.33" data-doi="10.1073/pnas.84.17.6220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">Spectra of spontaneous mutations in <em>Escherichia coli</em> strains defective in mismatch correction: the nature of <em>in vivo</em> DNA replication errors</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">6220</span>â<span class="cit-lpage">6224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSchaaper%26rft.auinit1%253DR.%2BM.%26rft.volume%253D84%26rft.issue%253D17%26rft.spage%253D6220%26rft.epage%253D6224%26rft.atitle%253DSpectra%2Bof%2Bspontaneous%2Bmutations%2Bin%2BEscherichia%2Bcoli%2Bstrains%2Bdefective%2Bin%2Bmismatch%2Bcorrection%253A%2Bthe%2Bnature%2Bof%2Bin%2Bvivo%2BDNA%2Breplication%2Berrors.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.17.6220%26rft_id%253Dinfo%253Apmid%252F3306672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODQvMTcvNjIyMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.34" data-doi="10.1093/nar/29.24.5107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lurz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luder</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tolksdorf</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">An architectural role of the <em>Escherichia coli</em> chromatin protein FIS in organising DNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">5107</span>â<span class="cit-lpage">5114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSchneider%26rft.auinit1%253DR.%26rft.volume%253D29%26rft.issue%253D24%26rft.spage%253D5107%26rft.epage%253D5114%26rft.atitle%253DAn%2Barchitectural%2Brole%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bchromatin%2Bprotein%2BFIS%2Bin%2Borganising%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.24.5107%26rft_id%253Dinfo%253Apmid%252F11812843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8yNC81MTA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.35" data-doi="10.1038/nature11273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuster-Bockler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Chromatin organization is a major influence on regional mutation rates in human cancer cells</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">504</span>â<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchuster-Bockler%26rft.auinit1%253DB.%26rft.volume%253D488%26rft.issue%253D7412%26rft.spage%253D504%26rft.epage%253D507%26rft.atitle%253DChromatin%2Borganization%2Bis%2Ba%2Bmajor%2Binfluence%2Bon%2Bregional%2Bmutation%2Brates%2Bin%2Bhuman%2Bcancer%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11273%26rft_id%253Dinfo%253Apmid%252F22820252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11273&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820252&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307761600035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.36" data-doi="10.1039/c0mb00213e"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scolari</span>  <span class="cit-name-given-names">V. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassetti</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclavi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagomarsino</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Gene clusters reflecting macrodomain structure respond to nucleoid perturbations</span>. <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">878</span>â<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DScolari%26rft.auinit1%253DV.%2BF.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D878%26rft.epage%253D888%26rft.atitle%253DGene%2Bclusters%2Breflecting%2Bmacrodomain%2Bstructure%2Brespond%2Bto%2Bnucleoid%2Bperturbations.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc0mb00213e%26rft_id%253Dinfo%253Apmid%252F21165487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/c0mb00213e&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21165487&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.37" data-doi="10.1073/pnas.1108229109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sobetzko</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muskhelishvili</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E42</span>â<span class="cit-lpage">E50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1108229109%26rft_id%253Dinfo%253Apmid%252F22184251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDkvMi9FNDIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.38" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Statistical significance for genomewide studies</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9440</span>â<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.39" data-doi="10.1371/journal.pgen.1000344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Touchon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoede</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbe</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baeriswyl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Organised genome dynamics in the <em>Escherichia coli</em> species results in highly diverse adaptive paths</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTouchon%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000344%26rft.epage%253De1000344%26rft.atitle%253DOrganised%2Bgenome%2Bdynamics%2Bin%2Bthe%2BEscherichia%2Bcoli%2Bspecies%2Bresults%2Bin%2Bhighly%2Bdiverse%2Badaptive%2Bpaths.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000344%26rft_id%253Dinfo%253Apmid%252F19165319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165319&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.40" data-doi="10.1016/j.gde.2005.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muskhelishvili</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Bacterial chromatin</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">507</span>â<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DTravers%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D507%26rft.epage%253D514%26rft.atitle%253DBacterial%2Bchromatin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2005.08.006%26rft_id%253Dinfo%253Apmid%252F16099644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2005.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16099644&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232454200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.41" data-doi="10.1016/S0300-9084(00)01225-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ussery</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkes</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worning</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Genome organisation and chromatin structure in <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">201</span>â<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D83%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0300-9084%252800%252901225-6%26rft_id%253Dinfo%253Apmid%252F11278070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0300-9084(00)01225-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11278070&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.42" data-doi="10.1038/sj.emboj.7600434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boccard</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Macrodomain organization of the <em>Escherichia coli</em> chromosome</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4330</span>â<span class="cit-lpage">4341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600434%26rft_id%253Dinfo%253Apmid%252F15470498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7600434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15470498&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225301500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-web" id="cit-3.3.399.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wessa</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, 2012 Free statistics software, Office for research development and education, version 1.1.23-r7. Available at: <a href="http://www.wessa.net/">http://www.wessa.net/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.399.44" data-doi="10.1073/pnas.91.8.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Worth</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Mismatch repair proteins MutS and MutL inhibit RecA-catalyzed strand transfer between diverged DNAs</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3238</span>â<span class="cit-lpage">3241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWorth%26rft.auinit1%253DL.%26rft.volume%253D91%26rft.issue%253D8%26rft.spage%253D3238%26rft.epage%253D3241%26rft.atitle%253DMismatch%2Brepair%2Bproteins%2BMutS%2Band%2BMutL%2Binhibit%2BRecA-catalyzed%2Bstrand%2Btransfer%2Bbetween%2Bdiverged%2BDNAs.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.8.3238%26rft_id%253Dinfo%253Apmid%252F8159731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5MS84LzMyMzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/3/399.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/387" title="Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/409" title="A Mosaic Genetic Screen for Genes Involved in the Early Steps of Drosophila Oogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/3"><span>Volume 3 Issue 3, March 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/3.cover-source.jpg?itok=w5x3d6rp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/399" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UD2U-gXaWzfqpFP_i9pMX9qCCOdSqhqXMQ5y6ZTZXJ0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/3/399" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/3/399" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/3/399" class="active" data-icon-position="" data-hide-link-title="0">On the Mutational Topology of the Bacterial Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459430" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-j9PEk8kD5JASNsu3rE0ERpMers3ww2P68lyu0Z_ptV4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/399.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/3/399" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EcI8oHTq2xnGmDIZnC351XXwCYEiV0w-EkHPxa2jf2U" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/3/399" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/399.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459430" id="node459430--2" data-pisa="ggg;3/3/399" data-pisa-master="ggg;3/3/399" data-apath="/ggg/3/3/399.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">On the Mutational Topology of the Bacterial Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Patricia L. Foster</span>, <span class="highwire-citation-author" data-delta="1">Andrew J. Hanson</span>, <span class="highwire-citation-author" data-delta="2">Heewook Lee</span>, <span class="highwire-citation-author" data-delta="3">Ellen M. Popodi</span> and <span class="highwire-citation-author" data-delta="4">Haixu Tang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">399-407;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005355 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459430--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patricia L. Foster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BL.%2BFoster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foster%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BL.%2BFoster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plfoster@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">plfoster@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew J. Hanson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BHanson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hanson%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BHanson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Heewook Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heewook%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeewook%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ellen M. Popodi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ellen%2BM.%2BPopodi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Popodi%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEllen%2BM.%2BPopodi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Haixu Tang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haixu%2BTang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaixu%2BTang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/3/399" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/3/399&amp;title=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/3/399&amp;title=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/3/399&amp;title=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/3/399&amp;text=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/3/399&amp;title=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/3/399&amp;t=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/3/399&amp;title=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/3/399&amp;title=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459430/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459430" id="node459430--3" data-pisa="ggg;3/3/399" data-pisa-master="ggg;3/3/399" data-apath="/ggg/3/3/399.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">On the Mutational Topology of the Bacterial Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Patricia L. Foster</span>, <span class="highwire-citation-author" data-delta="1">Andrew J. Hanson</span>, <span class="highwire-citation-author" data-delta="2">Heewook Lee</span>, <span class="highwire-citation-author" data-delta="3">Ellen M. Popodi</span> and <span class="highwire-citation-author" data-delta="4">Haixu Tang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">399-407;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005355 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459430--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patricia L. Foster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BL.%2BFoster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foster%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BL.%2BFoster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plfoster@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">plfoster@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew J. Hanson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BHanson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hanson%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BHanson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Heewook Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heewook%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeewook%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ellen M. Popodi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ellen%2BM.%2BPopodi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Popodi%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEllen%2BM.%2BPopodi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Haixu Tang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haixu%2BTang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaixu%2BTang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=399&amp;rft.epage=407&amp;rft.atitle=On%20the%20Mutational%20Topology%20of%20the%20Bacterial%20Genome&amp;rft.volume=3&amp;rft.issue=3&amp;rft.date=2013-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Foster%2C+Patricia+L.&amp;rft.au=Hanson%2C+Andrew+J.&amp;rft.au=Lee%2C+Heewook&amp;rft.au=Popodi%2C+Ellen+M.&amp;rft.au=Tang%2C+Haixu"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459430/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459430/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459430/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459430/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459430/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459430/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459430/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459430/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459430/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459430/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459430/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459430/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/3/399" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/3/399.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/3/399.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/3/399.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/3/399.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
