<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/3/451" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/3/451.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/3/451.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/3/451.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/3/451.atom" />
<meta name="HW.pisa" content="ggg;3/3/451" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells" />
<meta name="DC.Identifier" content="10.1534/g3.112.005173" />
<meta name="DC.Date" content="2013-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Baptiste et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Dinucleotide microsatellites are dynamic DNA sequences that affect genome stability. Here, we focused on mature microsatellites, defined as pure repeats of lengths above the threshold and unlikely to mutate below it in a single mutational event. We investigated the prevalence and mutational behavior of these sequences by using human genome sequence data, human cells in culture, and purified DNA polymerases. Mature dinucleotides (≥10 units) are present within exonic sequences of &gt;350 genes, resulting in vulnerability to cellular genetic integrity. Mature dinucleotide mutagenesis was examined experimentally using ex vivo and in vitro approaches. We observe an expansion bias for dinucleotide microsatellites up to 20 units in length in somatic human cells, in agreement with previous computational analyses of germ-line biases. Using purified DNA polymerases and human cell lines deficient for mismatch repair (MMR), we show that the expansion bias is caused by functional MMR and is not due to DNA polymerase error biases. Specifically, we observe that the MutSα and MutLα complexes protect against expansion mutations. Our data support a model wherein different MMR complexes shift the balance of mutations toward deletion or expansion. Finally, we show that replication fork progression is stalled within long dinucleotides, suggesting that mutational mechanisms within long repeats may be distinct from shorter lengths, depending on the biochemistry of fork resolution. Our work combines computational and experimental approaches to explain the complex mutational behavior of dinucleotide microsatellites in humans." />
<meta name="DC.Contributor" content="Beverly A. Baptiste" />
<meta name="DC.Contributor" content="Guruprasad Ananda" />
<meta name="DC.Contributor" content="Noelle Strubczewski" />
<meta name="DC.Contributor" content="Andrew Lutzkanin" />
<meta name="DC.Contributor" content="Su Jen Khoo" />
<meta name="DC.Contributor" content="Abhinaya Srikanth" />
<meta name="DC.Contributor" content="Nari Kim" />
<meta name="DC.Contributor" content="Kateryna D. Makova" />
<meta name="DC.Contributor" content="Maria M. Krasilnikova" />
<meta name="DC.Contributor" content="Kristin A. Eckert" />
<meta name="article:published_time" content="2013-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Dinucleotide microsatellites are dynamic DNA sequences that affect genome stability. Here, we focused on mature microsatellites, defined as pure repeats of lengths above the threshold and unlikely to mutate below it in a single mutational event. We investigated the prevalence and mutational behavior of these sequences by using human genome sequence data, human cells in culture, and purified DNA polymerases. Mature dinucleotides (≥10 units) are present within exonic sequences of &amp;gt;350 genes, resulting in vulnerability to cellular genetic integrity. Mature dinucleotide mutagenesis was examined experimentally using &lt;i&gt;ex vivo&lt;/i&gt; and &lt;i&gt;in vitro&lt;/i&gt; approaches. We observe an expansion bias for dinucleotide microsatellites up to 20 units in length in somatic human cells, in agreement with previous computational analyses of germ-line biases. Using purified DNA polymerases and human cell lines deficient for mismatch repair (MMR), we show that the expansion bias is caused by functional MMR and is not due to DNA polymerase error biases. Specifically, we observe that the MutSα and MutLα complexes protect against expansion mutations. Our data support a model wherein different MMR complexes shift the balance of mutations toward deletion or expansion. Finally, we show that replication fork progression is stalled within long dinucleotides, suggesting that mutational mechanisms within long repeats may be distinct from shorter lengths, depending on the biochemistry of fork resolution. Our work combines computational and experimental approaches to explain the complex mutational behavior of dinucleotide microsatellites in humans.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/03/01" />
<meta name="citation_mjid" content="ggg;3/3/451" />
<meta name="citation_id" content="3/3/451" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/3/451" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/3/451.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/3/451.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/3/451.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005173" />
<meta name="citation_pmid" content="23450065" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="451" />
<meta name="citation_lastpage" content="463" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Beverly A. Baptiste" />
<meta name="citation_author_institution" content="Intercollege Graduate Program in Genetics, Huck Institutes of Life Sciences, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author" content="Guruprasad Ananda" />
<meta name="citation_author_institution" content="Graduate Program in Bioinformatics and Genomics, Huck Institutes of Life Sciences, Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Noelle Strubczewski" />
<meta name="citation_author_institution" content="Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author" content="Andrew Lutzkanin" />
<meta name="citation_author_institution" content="Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author" content="Su Jen Khoo" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Abhinaya Srikanth" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Nari Kim" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Kateryna D. Makova" />
<meta name="citation_author_institution" content="Department of Biology, Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Maria M. Krasilnikova" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_email" content="muk19@psu.edu" />
<meta name="citation_author_email" content="kae4@psu.edu" />
<meta name="citation_author" content="Kristin A. Eckert" />
<meta name="citation_author_institution" content="Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_institution" content="Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_email" content="muk19@psu.edu" />
<meta name="citation_author_email" content="kae4@psu.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=W. Amos;citation_title=Mutation biases and mutation rate variation around very short human microsatellites revealed by human-chimpanzee-orangutan genomic sequence alignments.;citation_pages=192-201;citation_volume=71;citation_year=2010;citation_issue=3;citation_pmid=20700734;citation_doi=10.1007/s00239-010-9377-4" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Amos;citation_author=D. C. Rubinstzein;citation_title=Microsatellites are subject to directional evolution.;citation_pages=13-14;citation_volume=12;citation_year=1996;citation_issue=1;citation_pmid=8528243;citation_doi=10.1038/ng0196-13" />
<meta name="citation_reference" content="citation_title=Gen. Biol. Evol.;citation_title=Distinct mutational behaviors distinguish simple tandem repeats from microsatellites in the human genome;citation_author=G. Ananda;citation_author=E. Walsh;citation_author=K. D. Jacob;citation_author=M. Krasilnikova;citation_author=K. A. Eckert;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Environmental and molecular mutagenesis;citation_journal_abbrev=Environ Mol Mutagen;citation_author=B. A. Baptiste;citation_author=K. A. Eckert;citation_title=DNA polymerase kappa microsatellite synthesis: Two distinct mechanisms of slippage-mediated errors.;citation_pages=787-796;citation_volume=53;citation_year=2012;citation_issue=9;citation_pmid=22965905;citation_doi=10.1002/em.21721" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. I. Bell;citation_author=J. Jurka;citation_title=The length distribution of perfect dimer repetitive DNA is consistent with its evolution by an unbiased single-step mutation process.;citation_pages=414-421;citation_volume=44;citation_year=1997;citation_issue=4;citation_pmid=9089081;citation_doi=10.1007/PL00006161" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=S. E. Bennett;citation_author=A. Umar;citation_author=J. Oshima;citation_author=R. J. Monnat;citation_author=T. A. Kunkel;citation_title=Mismatch Repair in Extracts of Werner Syndrome Cell Lines;citation_pages=2956-2960;citation_volume=57;citation_year=1997;citation_issue=14;citation_pmid=9230208" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Mol. Biol.;citation_title=Galaxy: a web-based genome analysis tool for experimentalists;citation_author=D. Blankenberg;citation_author=G. Von Kuster;citation_author=N. Coraor;citation_author=G. Ananda;citation_author=R. Lazarus;citation_pages=19.10.1-21;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. L. Bourn;citation_author=I. De Biase;citation_author=R. M. Pinto;citation_author=C. Sandi;citation_author=S. Al-Mahdawi;citation_title=Pms2 Suppresses Large Expansions of the (GAAÂ·TTC)(n) Sequence in Neuronal Tissues.;citation_pages=e47085-e47085;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23071719;citation_doi=10.1371/journal.pone.0047085" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=H. Buerger;citation_author=F. Gebhardt;citation_author=H. Schmidt;citation_author=A. Beckmann;citation_author=K. Hutmacher;citation_title=Length and Loss of Heterozygosity of an Intron 1 Polymorphic Sequence of egfr Is Related to Cytogenetic Alterations and Epithelial Growth Factor Receptor Expression;citation_pages=854-857;citation_volume=60;citation_year=2000;citation_issue=4;citation_pmid=10706093" />
<meta name="citation_reference" content="citation_journal_title=The Journal of pathology;citation_journal_abbrev=J Pathol;citation_author=H. Buerger;citation_author=J. Packeisen;citation_author=A. Boecker;citation_author=N. Tidow;citation_author=C. Kersting;citation_title=Allelic length of a CA dinucleotide repeat in the egfr gene correlates with the frequency of amplifications of this sequence--first results of an inter-ethnic breast cancer study.;citation_pages=545-550;citation_volume=203;citation_year=2004;citation_issue=1;citation_pmid=15095477;citation_doi=10.1002/path.1542" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=E. Buschiazzo;citation_author=N. J. Gemmell;citation_title=The rise, fall and renaissance of microsatellites in eukaryotic genomes.;citation_pages=1040-1050;citation_volume=28;citation_year=2006;citation_issue=10;citation_pmid=16998838;citation_doi=10.1002/bies.20470" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Calabrese;citation_author=R. Durrett;citation_title=Dinucleotide repeats in the Drosophila and human genomes have complex, length-dependent mutation processes;citation_volume=20;citation_year=2003;citation_pmid=12679531;citation_doi=10.1093/molbev/msg084" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. S. Chandok;citation_author=K. K. Kapoor;citation_author=R. M. Brick;citation_author=J. M. Sidorova;citation_author=M. M. Krasilnikova;citation_title=A distinct first replication cycle of DNA introduced in mammalian cells;citation_volume=39;citation_year=2011;citation_pmid=21062817;citation_doi=10.1093/nar/gkq903" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. S. Chandok;citation_author=M. P. Patel;citation_author=S. M. Mirkin;citation_author=M. M. Krasilnikova;citation_title=Effects of Friedreich’s ataxia GAA repeats on DNA replication in mammalian cells;citation_volume=40;citation_year=2012;citation_pmid=22262734;citation_doi=10.1093/nar/gks021" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=P. C. Chen;citation_author=S. Dudley;citation_author=W. Hagen;citation_author=D. Dizon;citation_author=L. Paxton;citation_title=Contributions by MutL Homologues Mlh3 and Pms2 to DNA Mismatch Repair and Tumor Suppression in the Mouse;citation_pages=8662-8670;citation_volume=65;citation_year=2005;citation_issue=19;citation_pmid=16204034;citation_doi=10.1158/0008-5472.CAN-05-0742" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Di Rienzo;citation_author=A. C. Peterson;citation_author=J. C. Garza;citation_author=A. M. Valdes;citation_author=M. Slatkin;citation_title=Mutational processes of simple-sequence repeat loci in human populations.;citation_pages=3166-3170;citation_volume=91;citation_year=1994;citation_issue=8;citation_pmid=8159720;citation_doi=10.1073/pnas.91.8.3166" />
<meta name="citation_reference" content="citation_journal_title=Neurobiology of disease;citation_journal_abbrev=Neurobiol Dis;citation_author=E. Dragileva;citation_author=A. Hendricks;citation_author=A. Teed;citation_author=T. Gillis;citation_author=E. T. Lopez;citation_title=Intergenerational and striatal CAG repeat instability in Huntington's disease knock-in mice involve different DNA repair genes.;citation_pages=37-47;citation_volume=33;citation_year=2009;citation_issue=1;citation_pmid=18930147;citation_doi=10.1016/j.nbd.2008.09.014" />
<meta name="citation_reference" content="citation_journal_title=Molecular carcinogenesis;citation_journal_abbrev=Mol Carcinog;citation_author=K. A. Eckert;citation_author=S. E. Hile;citation_title=Every microsatellite is different: Intrinsic DNA features dictate mutagenesis of common microsatellites present in the human genome.;citation_pages=379-388;citation_volume=48;citation_year=2009;citation_issue=4;citation_pmid=19306292;citation_doi=10.1002/mc.20499" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. A. Eckert;citation_author=S. E. Hile;citation_author=P. L. Vargo;citation_title=Development and use of an in vitro HSV-tk forward mutation assay to study eukaryotic DNA polymerase processing of DNA alkyl lesions;citation_volume=25;citation_year=1997;citation_pmid=9060443;citation_doi=10.1093/nar/25.7.1450" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=K. A. Eckert;citation_author=A. Mowery;citation_author=S. E. Hile;citation_title=Misalignment-mediated DNA polymerase beta mutations: comparison of microsatellite and frame-shift error rates using a forward mutation assay;citation_pages=10490-10498;citation_volume=41;citation_year=2002;citation_pmid=12173936;citation_doi=10.1021/bi025918c" />
<meta name="citation_reference" content="citation_journal_title=Molecular carcinogenesis;citation_journal_abbrev=Mol Carcinog;citation_author=K. A. Eckert;citation_author=G. Yan;citation_author=S. E. Hile;citation_title=Mutation rate and specificity analysis of tetranucleotide microsatellite DNA alleles in somatic human cells.;citation_pages=140-150;citation_volume=34;citation_year=2002;citation_issue=3;citation_pmid=12112308;citation_doi=10.1002/mc.10058" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Ellegren;citation_title=Heterogeneous mutation processes in human microsatellite DNA sequences.;citation_pages=400-402;citation_volume=24;citation_year=2000;citation_issue=4;citation_pmid=10742106;citation_doi=10.1038/74249" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Ellegren;citation_title=Microsatellites: simple sequences with complex evolution.;citation_pages=435-445;citation_volume=5;citation_year=2004;citation_issue=6;citation_pmid=15153996;citation_doi=10.1038/nrg1348" />
<meta name="citation_reference" content="citation_journal_title=Neurobiology of disease;citation_journal_abbrev=Neurobiol Dis;citation_author=V. Ezzatizadeh;citation_author=R. M. Pinto;citation_author=C. Sandi;citation_author=M. Sandi;citation_author=S. Al-Mahdawi;citation_title=The mismatch repair system protects against intergenerational GAA repeat instability in a Friedreich ataxia mouse model.;citation_pages=165-171;citation_volume=46;citation_year=2012;citation_issue=1;citation_pmid=22289650;citation_doi=10.1016/j.nbd.2012.01.002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Falcon;citation_author=R. Gentleman;citation_title=Using GOstats to test gene lists for GO term association;citation_volume=23;citation_year=2007;citation_pmid=17098774;citation_doi=10.1093/bioinformatics/btl567" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. W. Feldman;citation_author=A. Bergman;citation_author=D. D. Pollock;citation_author=D. B. Goldstein;citation_title=Microsatellite Genetic Distances With Range Constraints: Analytic Description and Problems of Estimation;citation_pages=207-216;citation_volume=145;citation_year=1997;citation_issue=1;citation_pmid=9017402" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=M. D. Frank-Kamenetskii;citation_author=S. M. Mirkin;citation_title=Triplex DNA Structures;citation_pages=65-95;citation_volume=64;citation_year=1995;citation_issue=1;citation_pmid=7574496;citation_doi=10.1146/annurev.bi.64.070195.000433" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=F. Gebhardt;citation_author=K. S. Zanker;citation_author=B. Brandt;citation_title=Modulation of Epidermal Growth Factor Receptor Gene Transcription by a Polymorphic Dinucleotide Repeat in Intron 1;citation_pages=13176-13180;citation_volume=274;citation_year=1999;citation_issue=19;citation_pmid=10224073;citation_doi=10.1074/jbc.274.19.13176" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Gemayel;citation_author=M. D. Vinces;citation_author=M. Legendre;citation_author=K. J. Verstrepen;citation_title=Variable tandem repeats accelerate evolution of coding and regulatory sequences.;citation_pages=445-477;citation_volume=44;citation_year=2010;citation_pmid=20809801;citation_doi=10.1146/annurev-genet-072610-155046" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Giardine;citation_author=C. Riemer;citation_author=R. C. Hardison;citation_author=R. Burhans;citation_author=L. Elnitski;citation_title=Galaxy: a platform for interactive large-scale genome analysis;citation_volume=15;citation_year=2005;citation_pmid=16169926;citation_doi=10.1101/gr.4086505" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Goecks;citation_author=A. Nekrutenko;citation_author=J. Taylor;citation_author=T. Galaxy;citation_title=Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.;citation_pages=R86-R86;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20738864;citation_doi=10.1186/gb-2010-11-8-r86" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. E. Hile;citation_author=K. A. Eckert;citation_title=Positive correlation between DNA polymerase alpha-primase pausing and mutagenesis within polypyrimidine/polypurine microsatellite sequences.;citation_pages=745-759;citation_volume=335;citation_year=2004;citation_issue=3;citation_pmid=14687571;citation_doi=10.1016/j.jmb.2003.10.075" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. E. Hile;citation_author=K. A. Eckert;citation_title=DNA polymerase kappa produces interrupted mutations and displays polar pausing within mononucleotide microsatellite sequences;citation_volume=36;citation_year=2008;citation_pmid=18079151;citation_doi=10.1093/nar/gkm1089" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=S. E. Hile;citation_author=G. Yan;citation_author=K. A. Eckert;citation_title=Somatic Mutation Rates and Specificities at TC/AG and GT/CA Microsatellite Sequences in Nontumorigenic Human Lymphoblastoid Cells;citation_pages=1698-1703;citation_volume=60;citation_year=2000;citation_issue=6;citation_pmid=10749142" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. E. Hile;citation_author=X. Wang;citation_author=M. Y. W. T. Lee;citation_author=K. A. Eckert;citation_title=Beyond translesion synthesis: polymerase Î° fidelity as a potential determinant of microsatellite stability;citation_volume=40;citation_year=2012;citation_pmid=22021378;citation_doi=10.1093/nar/gkr889" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=Q. Y. Huang;citation_author=F. H. Xu;citation_author=H. Shen;citation_author=H. Y. Deng;citation_author=Y. J. Liu;citation_title=Mutation patterns at dinucleotide microsatellite loci in humans.;citation_pages=625-634;citation_volume=70;citation_year=2002;citation_issue=3;citation_pmid=11793300;citation_doi=10.1086/338997" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=J. Hui;citation_author=L. H. Hung;citation_author=M. Heiner;citation_author=S. Schreiner;citation_author=N. Neumuller;citation_title=Intronic CA-repeat and CA-rich elements: a new class of regulators of mammalian alternative splicing;citation_volume=24;citation_year=2005;citation_pmid=15889141;citation_doi=10.1038/sj.emboj.7600677" />
<meta name="citation_reference" content="citation_journal_title=Science (New York, NY);citation_author=R. E. Johnson;citation_author=G. K. Kovvali;citation_author=L. Prakash;citation_author=S. Prakash;citation_title=Requirement of the yeast RTH1 5′to 3′exonuclease for the stability of simple repetitive DNA;citation_pages=238;citation_volume=269;citation_year=1995;citation_doi=10.1126/science.7618086" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=A. Kantartzis;citation_author=G. M. Williams;citation_author=L. Balakrishnan;citation_author=R. L. Roberts;citation_author=J. A. Surtees;citation_title=Msh2-Msh3 interferes with Okazaki fragment processing to promote trinucleotide repeat expansions;citation_pages=216-222;citation_volume=2;citation_year=2012;citation_pmid=22938864;citation_doi=10.1016/j.celrep.2012.06.020" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Karolchik;citation_author=R. M. Kuhn;citation_author=R. Baertsch;citation_author=G. P. Barber;citation_author=H. Clawson;citation_title=The UCSC Genome Browser Database: 2008 update;citation_volume=36;citation_year=2008;citation_pmid=18086701;citation_doi=10.1093/nar/gkm966" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. Kavathas;citation_author=F. H. Bach;citation_author=R. DeMars;citation_title=Gamma ray-induced loss of expression of HLA and glyoxalase I alleles in lymphoblastoid cells.;citation_pages=4251-4255;citation_volume=77;citation_year=1980;citation_issue=7;citation_pmid=6933474;citation_doi=10.1073/pnas.77.7.4251" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. D. Kelkar;citation_author=S. Tyekucheva;citation_author=F. Chiaromonte;citation_author=K. D. Makova;citation_title=The genome-wide determinants of human and chimpanzee microsatellite evolution;citation_volume=18;citation_year=2008;citation_pmid=18032720;citation_doi=10.1101/gr.7113408" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y. D. Kelkar;citation_author=N. Strubczewski;citation_author=S. E. Hile;citation_author=F. Chiaromonte;citation_author=K. A. Eckert;citation_title=What is a microsatellite: a computational and experimental definition based upon repeat mutational behavior at A/T and GT/AC repeats;citation_volume=2;citation_year=2010;citation_pmid=20668018;citation_doi=10.1093/gbe/evq046" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_author=C. W. Sugnet;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=T. H. Pringle;citation_title=The human genome browser at UCSC;citation_volume=12;citation_year=2002;citation_pmid=12045153;citation_doi=10.1101/gr.229102. Article published online before print in May 2002" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=M. Koi;citation_author=A. Umar;citation_author=D. P. Chauhan;citation_author=S. P. Cherian;citation_author=J. M. Carethers;citation_title=Human Chromosome 3 Corrects Mismatch Repair Deficiency and Microsatellite Instability and Reduces N-Methyl-N'-nitro-N-nitrosoguanidine Tolerance in Colon Tumor Cells with Homozygous hMLH1 Mutation;citation_pages=4308-4312;citation_volume=54;citation_year=1994;citation_issue=16;citation_pmid=8044777" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=M. M. Krasilnikova;citation_author=S. M. Mirkin;citation_title=Analysis of triplet repeat replication by two-dimensional gel electrophoresis.;citation_pages=19-28;citation_volume=277;citation_year=2004;citation_pmid=15201446" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. M. Krasilnikova;citation_author=S. M. Mirkin;citation_title=Replication Stalling at Friedreich's Ataxia (GAA)n Repeats In Vivo;citation_pages=2286-2295;citation_volume=24;citation_year=2004;citation_issue=6;citation_pmid=14993268;citation_doi=10.1128/MCB.24.6.2286-2295.2004" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Kruglyak;citation_author=R. T. Durrett;citation_author=M. D. Schug;citation_author=C. F. Aquadro;citation_title=Equilibrium distributions of microsatellite repeat length resulting from a balance between slippage events and point mutations;citation_pages=10774-10778;citation_volume=95;citation_year=1998;citation_issue=18;citation_pmid=9724780;citation_doi=10.1073/pnas.95.18.10774" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=F. Lazzaro;citation_author=M. Giannattasio;citation_author=F. Puddu;citation_author=M. Granata;citation_author=A. Pellicioli;citation_title=Checkpoint mechanisms at the intersection between DNA damage and repair;citation_pages=1055-1067;citation_volume=8;citation_year=2009;citation_pmid=19497792;citation_doi=10.1016/j.dnarep.2009.04.022" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=Y. C. Li;citation_author=A. B. Korol;citation_author=T. Fahima;citation_author=A. Beiles;citation_author=E. Nevo;citation_title=Microsatellites: genomic distribution, putative functions and mutational mechanisms: a review.;citation_pages=2453-2465;citation_volume=11;citation_year=2002;citation_issue=12;citation_pmid=12453231;citation_doi=10.1046/j.1365-294X.2002.01643.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. C. Li;citation_author=A. B. Korol;citation_author=T. Fahima;citation_author=E. Nevo;citation_title=Microsatellites within genes: structure, function, and evolution;citation_volume=21;citation_year=2004;citation_pmid=14963101;citation_doi=10.1093/molbev/msh073" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=H. T. Lynch;citation_author=A. de la Chapelle;citation_title=Hereditary Colorectal Cancer;citation_pages=919-932;citation_volume=348;citation_year=2003;citation_issue=10;citation_pmid=12621137;citation_doi=10.1056/NEJMra012242" />
<meta name="citation_reference" content="citation_journal_title=Clinical genetics;citation_journal_abbrev=Clin Genet;citation_author=H. T. Lynch;citation_author=P. M. Lynch;citation_author=S. J. Lanspa;citation_author=C. L. Snyder;citation_author=J. F. Lynch;citation_title=Review of the Lynch syndrome: history, molecular genetics, screening, differential diagnosis, and medicolegal ramifications.;citation_pages=1-18;citation_volume=76;citation_year=2009;citation_issue=1;citation_pmid=19659756" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. E. Madsen;citation_author=P. Villesen;citation_author=C. Wiuf;citation_title=Short tandem repeats in human exons: a target for disease mutations.;citation_pages=410-410;citation_volume=9;citation_year=2008;citation_pmid=18789129;citation_doi=10.1186/1471-2164-9-410" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. T. McMurray;citation_title=Mechanisms of trinucleotide repeat instability during human development.;citation_pages=786-799;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20953213;citation_doi=10.1038/nrg2828" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=E. V. Mirkin;citation_author=S. M. Mirkin;citation_title=Replication Fork Stalling at Natural Impediments;citation_pages=13-35;citation_volume=71;citation_year=2007;citation_issue=1;citation_pmid=17347517;citation_doi=10.1128/MMBR.00030-06" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=P. L. Opresko;citation_author=J. B. Sweasy;citation_author=K. A. Eckert;citation_title=The mutator form of polymerase beta with amino acid substitution at tyrosine 265 in the hinge region displays an increase in both base substitution and frame shift errors;citation_pages=2111-2119;citation_volume=37;citation_year=1998;citation_pmid=9485358;citation_doi=10.1021/bi9722711" />
<meta name="citation_reference" content="citation_journal_title=Genes, chromosomes &amp; cancer;citation_journal_abbrev=Genes Chromosomes Cancer;citation_author=J. Ou;citation_author=M. Rasmussen;citation_author=H. Westers;citation_author=S. D. Andersen;citation_author=P. O. Jager;citation_title=Biochemical characterization of MLH3 missense mutations does not reveal an apparent role of MLH3 in Lynch syndrome.;citation_pages=340-350;citation_volume=48;citation_year=2009;citation_issue=4;citation_pmid=19156873;citation_doi=10.1002/gcc.20644" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Parsons;citation_author=G. M. Li;citation_author=M. Longley;citation_author=P. Modrich;citation_author=B. Liu;citation_title=Mismatch repair deficiency in phenotypically normal human cells;citation_pages=738-740;citation_volume=268;citation_year=1995;citation_issue=5211;citation_pmid=7632227;citation_doi=10.1126/science.7632227" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. E. Pearson;citation_author=K. Nichol Edamura;citation_author=J. D. Cleary;citation_title=Repeat instability: mechanisms of dynamic mutations.;citation_pages=729-742;citation_volume=6;citation_year=2005;citation_issue=10;citation_pmid=16205713;citation_doi=10.1038/nrg1689" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. Pelletier;citation_author=M. M. Krasilnikova;citation_author=G. M. Samadashwily;citation_author=R. Lahue;citation_author=S. M. Mirkin;citation_title=Replication and Expansion of Trinucleotide Repeats in Yeast;citation_pages=1349-1357;citation_volume=23;citation_year=2003;citation_issue=4;citation_pmid=12556494;citation_doi=10.1128/MCB.23.4.1349-1357.2003" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=J. Pena-Diaz;citation_author=J. Jiricny;citation_title=Mammalian mismatch repair: error-free or error-prone?;citation_pages=206-214;citation_volume=37;citation_year=2012;citation_issue=5;citation_pmid=22475811;citation_doi=10.1016/j.tibs.2012.03.001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Sainudiin;citation_author=R. T. Durrett;citation_author=C. F. Aquadro;citation_author=R. Nielsen;citation_title=Microsatellite Mutation Models: Insights From a Comparison of Humans and Chimpanzees;citation_pages=383-395;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454551;citation_doi=10.1534/genetics.103.022665" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. M. Samadashwily;citation_author=G. Raca;citation_author=S. M. Mirkin;citation_title=Trinucleotide repeats affect DNA replication in vivo.;citation_pages=298-304;citation_volume=17;citation_year=1997;citation_issue=3;citation_pmid=9354793;citation_doi=10.1038/ng1197-298" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=S. N. Shah;citation_author=K. A. Eckert;citation_title=Human postmeiotic segregation 2 exhibits biased repair at tetranucleotide microsatellite sequences;citation_volume=69;citation_year=2009;citation_pmid=19155293;citation_doi=10.1158/0008-5472.CAN-08-3499" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. A. Shishkin;citation_author=I. Voineagu;citation_author=R. Matera;citation_author=N. Cherng;citation_author=B. T. Chernet;citation_title=Large-scale expansions of Friedreich's ataxia GAA repeats in yeast.;citation_pages=82-92;citation_volume=35;citation_year=2009;citation_issue=1;citation_pmid=19595718;citation_doi=10.1016/j.molcel.2009.06.017" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Sogo;citation_author=M. Lopes;citation_author=M. Foiani;citation_title=Fork Reversal and ssDNA Accumulation at Stalled Replication Forks Owing to Checkpoint Defects;citation_pages=599-602;citation_volume=297;citation_year=2002;citation_issue=5581;citation_pmid=12142537;citation_doi=10.1126/science.1074023" />
<meta name="citation_reference" content="citation_journal_title=Pharmacogenetics;citation_journal_abbrev=Pharmacogenetics;citation_author=K. Stangl;citation_author=I. Cascorbi;citation_author=M. Laule;citation_author=T. Klein;citation_author=V. Stangl;citation_title=High CA repeat numbers in intron 13 of the endothelial nitric oxide synthase gene and increased risk of coronary artery disease.;citation_pages=133-140;citation_volume=10;citation_year=2000;citation_issue=2;citation_pmid=10762001;citation_doi=10.1097/00008571-200003000-00005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Strand;citation_author=T. A. Prolla;citation_author=R. M. Liskay;citation_author=T. D. Petes;citation_title=Destabilization of tracts of simple repetitive DNA in yeast by mutations affecting DNA mismatch repair.;citation_pages=274-276;citation_volume=365;citation_year=1993;citation_issue=6443;citation_pmid=8371783;citation_doi=10.1038/365274a0" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. X. Sun;citation_author=A. Helgason;citation_author=G. Masson;citation_author=S. S. Ebenesersdottir;citation_author=H. Li;citation_title=A direct characterization of human mutation based on microsatellites.;citation_pages=1161-1165;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22922873;citation_doi=10.1038/ng.2398" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=A. Umar;citation_author=J. C. Boyer;citation_author=D. C. Thomas;citation_author=D. C. Nguyen;citation_author=J. I. Risinger;citation_title=Defective mismatch repair in extracts of colorectal and endometrial cancer cell lines exhibiting microsatellite instability;citation_pages=14367-14370;citation_volume=269;citation_year=1994;citation_issue=20;citation_pmid=8182040" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=W. J. A. A. van den Broek;citation_author=M. R. Nelen;citation_author=D. G. Wansink;citation_author=M. M. Coerwinkel;citation_author=H. te Riele;citation_title=Somatic expansion behaviour of the (CTG)n repeat in myotonic dystrophy knock-in mice is differentially affected by Msh3 and Msh6 mismatch-repair proteins;citation_pages=191-198;citation_volume=11;citation_year=2002;citation_issue=2;citation_pmid=11809728;citation_doi=10.1093/hmg/11.2.191" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. M. M. van Oers;citation_author=S. Roa;citation_author=U. Werling;citation_author=Y. Liu;citation_author=J. Genschel;citation_title=PMS2 endonuclease activity has distinct biological functions and is essential for genome maintenance;citation_volume=107;citation_year=2010;citation_pmid=20624957;citation_doi=10.1073/pnas.1008589107" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=I. Voineagu;citation_author=C. F. Surka;citation_author=A. A. Shishkin;citation_author=M. M. Krasilnikova;citation_author=S. M. Mirkin;citation_title=Replisome stalling and stabilization at CGG repeats, which are responsible for chromosomal fragility.;citation_pages=226-228;citation_volume=16;citation_year=2009;citation_issue=2;citation_pmid=19136957;citation_doi=10.1038/nsmb.1527" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=Y. Wang;citation_author=X. Liu;citation_author=Y. Li;citation_title=Target genes of microsatellite sequences in head and neck squamous cell carcinoma: Mononucleotide repeats are not detected;citation_pages=195-201;citation_volume=506;citation_year=2012;citation_pmid=22771919;citation_doi=10.1016/j.gene.2012.06.056" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=R. D. Wells;citation_title=Non-B DNA conformations, mutagenesis and disease.;citation_pages=271-278;citation_volume=32;citation_year=2007;citation_issue=6;citation_pmid=17493823;citation_doi=10.1016/j.tibs.2007.04.003" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. D. Wells;citation_author=R. Dere;citation_author=M. L. Hebert;citation_author=M. Napierala;citation_author=L. S. Son;citation_title=Advances in mechanisms of genetic instability related to hereditary neurological diseases;citation_pages=3785-3798;citation_volume=33;citation_year=2005;citation_issue=12;citation_pmid=16006624;citation_doi=10.1093/nar/gki697" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Whittaker;citation_author=R. M. Harbord;citation_author=N. Boxall;citation_author=I. Mackay;citation_author=G. Dawson;citation_title=Likelihood-Based Estimation of Microsatellite Mutation Rates;citation_pages=781-787;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807796" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=N. A. Yamada;citation_author=G. A. Smith;citation_author=A. Castro;citation_author=C. N. Roques;citation_author=J. C. Boyer;citation_title=Relative rates of insertion and deletion mutations in dinucleotide repeats of various lengths in mismatch repair proficient mouse and mismatch repair deficient human cells.;citation_pages=213-225;citation_volume=499;citation_year=2002;citation_issue=2;citation_pmid=11827714" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="twitter:description" content="Dinucleotide microsatellites are dynamic DNA sequences that affect genome stability. Here, we focused on mature microsatellites, defined as pure repeats of lengths above the threshold and unlikely to mutate below it in a single mutational event. We investigated the prevalence and mutational behavior of these sequences by using human genome sequence data, human cells in culture, and purified DNA polymerases. Mature dinucleotides (≥10 units) are present within exonic sequences of &gt;350 genes, resulting in vulnerability to cellular genetic integrity. Mature dinucleotide mutagenesis was examined experimentally using ex vivo and in vitro approaches. We observe an expansion bias for dinucleotide microsatellites up to 20 units in length in somatic human cells, in agreement with previous computational analyses of germ-line biases. Using purified DNA polymerases and human cell lines deficient for mismatch repair (MMR), we show that the expansion bias is caused by functional MMR and is not due to DNA polymerase error biases. Specifically, we observe that the MutSα and MutLα complexes protect against expansion mutations. Our data support a model wherein different MMR complexes shift the balance of mutations toward deletion or expansion. Finally, we show that replication fork progression is stalled within long dinucleotides, suggesting that mutational mechanisms within long repeats may be distinct from shorter lengths, depending on the biochemistry of fork resolution. Our work combines computational and experimental approaches to explain the complex mutational behavior of dinucleotide microsatellites in humans." />
<meta name="og-title" property="og:title" content="Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/3/451" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Dinucleotide microsatellites are dynamic DNA sequences that affect genome stability. Here, we focused on mature microsatellites, defined as pure repeats of lengths above the threshold and unlikely to mutate below it in a single mutational event. We investigated the prevalence and mutational behavior of these sequences by using human genome sequence data, human cells in culture, and purified DNA polymerases. Mature dinucleotides (≥10 units) are present within exonic sequences of &gt;350 genes, resulting in vulnerability to cellular genetic integrity. Mature dinucleotide mutagenesis was examined experimentally using ex vivo and in vitro approaches. We observe an expansion bias for dinucleotide microsatellites up to 20 units in length in somatic human cells, in agreement with previous computational analyses of germ-line biases. Using purified DNA polymerases and human cell lines deficient for mismatch repair (MMR), we show that the expansion bias is caused by functional MMR and is not due to DNA polymerase error biases. Specifically, we observe that the MutSα and MutLα complexes protect against expansion mutations. Our data support a model wherein different MMR complexes shift the balance of mutations toward deletion or expansion. Finally, we show that replication fork progression is stalled within long dinucleotides, suggesting that mutational mechanisms within long repeats may be distinct from shorter lengths, depending on the biochemistry of fork resolution. Our work combines computational and experimental approaches to explain the complex mutational behavior of dinucleotide microsatellites in humans." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
    <title>Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"iG4ZQIf6DnnqDojCwZqk8WKl3tSBtJ-MEVeYlA8pX5M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459428","apath":"\/ggg\/3\/3\/451.atom","pisa":"ggg;3\/3\/451","ac":{"\/ggg\/3\/3\/451.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/3\/451.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/3\/451"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/3\/451"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/3\/451":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lHFYmUqdn3D9Lz8NB\\\\\\\/XAMGLhcpsaNi+CTNxEy5fUwfaM1xZVvbC9smK30V3jLmWkqoLlDywSPeIX9zwq0lhUQ5VQ421xVEDWPi3JrvwyWXERfqa1WQcav9rNQvTOqRBD1YppcOn2VVZk9NP5p6ghVG6SxinmcO1N+dCBUXKeH\\\\\\\/zS88K59X69\\\\\\\/hnrvj8OiotveU4oYgA0vBrCDcwtTphsfqU6cPMzgCvsVgXAOgk3qDzUwp+k7DGKoiuXuefIBTl3tLsvOtzHIQVrfY7A5fu\\\\\\\/rK4vye11+GgDwsbgS9B05ydP7gM3Cli8IfHKnSNlLCDF5jXUgXzFp4zcDCtFuUixwKOC1pwIG4UCWQIYxeUfMra\\\\\\\/\\\\\\\/pdd80L488rBaOpz9YJkl3DO6d\\\\\\\/3Cez5gUpUHEwdVCMIbpq8pfLxYXOzkIwK4HpYdFAQhaQY1KXd9HFSd6hirUWmURcLU3gRNNRUo2QN5fFhCJN1x4fRfm4Nv\\\\\\\/5sBIY2IaljXHEZc7QYLIHPzduxTN4KAVJIjDB6C0sIBKJfdxG01QsGIALeGrlp0SlAKhyWV3pTpztgt0GpPkKvZk9caT3PwsGxXcnVJ+k63TnXUsLulQ6uklYQXCYNgmthGbYUBB7V11rKUgnpQzfOtMcqAHz1yC3oKwxgbnoU0Pm2cFR8IJ5spJ+cZC3MOusIFoXj6O+Xx98rUOgSeSqzXFLMvwHluc9\\\\\\\/oGnieW\\\\\\\/Gkh1C891GgQHk04dJRaDCL7cN3Vu3y6L2NIt1ZN2KYS3hiZnFbHEoOKAsuQ4dXqaqYggZqa8htGnC4gKegV4H2u\\\\\\\/D0f1Um4kybgP7jLkkbryEGAGTroUYCH4epY1UUDUZ2cLrytg3dLdK97kgP7yVOc\\\\\\\/EqeD8mSPNUjisyfzQksUFXtXRyI5PVWhO6GDHJyI+aHxqeLWBSd9LPbWQFXPSqkA\\\\\\\/\\\\\\\/VB2DrwOsPaaE4dgk3f9HthZXIQeK2PA2b4vELSCka+r7ag52uj+l+Uh5oyd6DxoN6KoGm6JI8UCdSmdY9EG1eRUlIgVkmzuLH+TRwkj7hp57MXAMwGADXNz7JKUWiGB\\\\\\\/ldaptJftcSC3mh3rJ1F8YAIjHvJbodFPIjy8Ujug8KpI3udropE3sE34ONsyX8YpBaOFsIOIMj\\\\\\\/sav1FvttvKZFPx1plTfBswSCIkZy9CGm1Slxrs5pp28\\\\\\\/0c\\\\\\\/1QTeIFi5gc3lU7cux524XhMEF3A717VKsAQ+HqcI1wt2oxggoBNi0sxJQUwk8b07F6sn\\\\\\\/kd+ANRjRWGdl3rGlqqHRJg4P\\\u0022,\\\u0022iv\\\u0022:\\\u0022sZN69yXotAgk\\\\\\\/6Q1OxR7uQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022af70873e7aafceaff44f337a3f17215bc29c9cc68d07d99fe514dd439093225b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DzYGpthABnODfOAGYtZ5rZybEwUsohzRvCg7+zIeQusR7yLuJne9suEVJP6wVhbJ2XNfEHiD6sqEbeLkse089BvBCjuDVqwPV8lhaQNXzWJ4DDOZBhHJ9drgRFZTgI2s67AlxIvlQrOoqkRhBFM86TTWizS2lKdimL1Z\\\\\\\/wqAeWJNPHCyK4NCWhdjeKQ9vXPl9t6ld4hNLERt0FAvST4jVVBjg0IW5+4ii9fw1vr\\\\\\\/6RART\\\\\\\/yf7nuwCZskvkjGFuCQV2KqxfPn2jCQpaEWBayMcZm+plqZQt3iC\\\\\\\/B1ABjYFNuZ3iQwI8gYtX6BwfaHfdLa9nGrD1OlCewBEmdMxufxvBKnGWr3YqEIftpDE4V3klN6DmRPIvpaULmr\\\\\\\/miZjd8z\\\\\\\/4hTS4Kjd0YlA1ZrXhtyHUGZmgupEmcAwty02FzRnwOxGpI8wASkEvasfDuea3iD2ZpjkkQvDVmy\\\\\\\/pPhA\\\\\\\/FAXhtRW9we7CTdhK+rDlACSMTrt8XdFqvPplDb9JS037ZJKXQigcUSG\\\\\\\/kriklTWu51VkugY65Su0eonVWJ02CVT3EqziSiVr\\\\\\\/ED3dSMA9yJRXE6CWc2uFFm1+seYYrk7w\\\\\\\/t6Fbrnz0NAWrWE1HegyZUvLnBW8vneV5TtZhgWBlMn4emN7DcX3\\\\\\\/vz5RFjNecJu38cZU+g233XSiejNTqHMYpza2uNwrD7NDaQCgAJO9L9TTDjMbLuK9L+sfVwgMHjNBevpeuYq4xGQrtvzEuDMIw4bobrE8CqnCGbHiWiwKp\\\\\\\/CW11LtSXt+oZHEWImHzXpqP1+iLI4vAD9DA0suFYiFjtdbuJ7Hau3UWAzd4ICklLFRgoMF5dGV6evUDbPeZ6G7uBtII7BwUzZoY07vco99\\\\\\\/LHn7VQpeSiM\\\\\\\/zeY7RA6JOjCSaN\\\\\\\/txJ2UARIiW5rXGQyRlQ24eUt0BTt6RF4D5AxZUc6ZbblgI\\\\\\\/3HVtR5fUe5Yb9wL45XGrMgz7RXHo9ZHXDlKFym7UCmlACbLnuc3weovAxQad9zsakP3u3IxfnqMGrLPLpPwxQY4GnpFCSXKLrR9yDU8VCGFRTZRBL4l+QkAmQxEG7a+iEwltfzwzKS7kCbuGg\\\\\\\/Bxvi55\\\\\\\/tveUBdCdqDprLMZ59ZDSqo5Gi8Lpb6M0i5Y1j+vWqy4evFR0OC4NCuspw4eDylqIDKvIeUSPXG5I7\\\\\\\/eQ+JS7yw4Jmyr3grBceQAhLTnFzfaKchn\\\\\\\/grcwHiEXs2eoMCsPBNXq34gg19Db1L5KnDRuRdqV\\\\\\\/++ZEenS4eAmlRWMuBnww8K8o+qSDjBD08dxduyfCPjdWXMGwlScMUxoGQQD6\\\\\\\/9B7CVSWQBOGuDkq3FElc2INBa+Xf6RXlVNZpdNiUnzZ\\\\\\\/9iJq1Ka9sfg7s6WT8tB80ZzgCH0dt9kcNiwGH6\\\\\\\/n47RMihReQFO2\\\\\\\/+gCCB25vsJnKcTbRYZOrcVbUvlkRKmbDVhpqXzY49uZckh76FC1lc17zOjvucvx9fEw4Cjmquk\\\\\\\/t+WKk6OSWyvvoFPADrUh7uRuahFvaJ6ifPNUZ1kPSgXXwrr0MUX+M4epqKDa2IPCwCykGEvpB6Oozvi\\\\\\\/fu6kpxDXUohnFnjye1coFlKXkkedbi9j9tvDpnC4eWMmj2Ohs7oJMrX1mnGItgIrW4yXChZkA02+4ZJN1nky0Tw\\\\\\\/MNpqiPMtEF4oau5F3gPCEX5ickLGaaSKh6qq2LYwONyhEMdYpsdwOTUrKD++BZ4Pvflil+IpGv9e93mLJWlnkqGLyte3C5EUVDdYXRC+mXgAi6DJd0E5+8X28OhrVfmoVk8BY1kENuEB7g9TWiTh3bz33iaWWWD4C7KryLIR\\\\\\\/jkXd\\\\\\\/vRF6OUl\\\\\\\/I8HMU+oNsGp3IiKRln9yy7saad5wMOleYXpsfz2rxZg67pUhfnwwXUD520KIP5IgP9+0CE3UnhJmlajZ+b1dbtzzjXytdSAfECoOuSCMIJgF\\\\\\\/EdxrmqvvPd3yOpsPHSXPVsERTjT8BN3SmPe\\\\\\\/5CY7EAV2fXSFCeUhUMlWTsrGVZRqY10SH37yNTQWxLZkTUP6bNwEeOwRmmC7qWNJYi92wa0iBcL99MornrWctS4G60Fsr5z5te1caY0Drihn2OarkV05f3kcpNQFcVlN9j+87IX8ClQ0oEouAussT7sPicV\\\\\\\/uR0TLkuhZb\\\\\\\/7x6FgKdYKE7Q09LR1RmkPzReQ7MSsCKycQaGR6wMaxMSxYqcwPkNYlUNiWbLcpWsF+iHBhihT2PZgGENoTQcGd1zJmK45gV4DzIkdCCxT1in+cbuRdRKlt8yweUcoe+udlx7S8aYJ6E50Z6lSwhl+KFpkn5MOvep5dyb9l7WPQYO5xUpoVZafD\\\\\\\/GRXKxXVPX2h98VNLR7ixi9oGM4rzE63TyGc5VxpuDopETPAM5+svzAVPbQoK8ReUkxfgXeKncACKwm+7eeTYNjY\\\\\\\/Aa53yRb7Va6bCWTWaeQWpU0iTgHrirfM7S2dlbTr\\\\\\\/Um6j8G2F93MTq3jfODnAZdpiO+o8U2qJMXNUdaF4QfbhpIZ3H5Ratfi\\\\\\\/4GXa+xbWs9IojMES2nM\\\\\\\/TquIPYwQ5U7UCPsEqMlOrqQ+A884s0NbGHf+A66gNVt7gHFZPSkowXUzWUAH4KxF0A+y2tHjiGP083yCCU+NyYfFKdW+X7RKjWTboU4\\\\\\\/M4lN37+svvmPCQJBwWrOJUtIWgd0qFyVzeOKGbQXBDO8J\\\\\\\/Mmep66LOcx9Efcif5X9vRlEXyFZdYpKUWxkeywEQBM9wKmGPR73PUCS+lyDz7trn5niOL4lJknFHBLHWxWstlR6kTtWm8uXlITewjBy34w24wmpUI57p1i0MLgku30ssbDV7YFM0h7qgyj84hgdMcQWq4WQk0qmYM9r38hS2DeCBePzTnWaTcP55HkqDj7qxMYhIrl7m5HjfCsfzuQf19JUiCs5VmLgcds4AvulmKMBeCSzBrxeIcn2Y2aQ0X6c\\\\\\\/uT0G1AFo69KhiRtFrhvQLA7E\\\\\\\/orY2USBT5L2aKgd5N75xf1Dg0Xd9r8nX0WNRcuYbexPCz6bCd2+3v+x\\\\\\\/rSTaCJd0fVuPn5CKEdJLKL2T0nQSpFlre+6RbKGKRQ09PrxMWAnO3O6W58cAoUXlIEnO8osEVFE+HQ45qtNY8lRk5XoOy8kVlao11qlf4TrjP+QwDlokfSNlOmQVUQEYuKlz8r6dun7zhBMnDDp0vTiyqPkqGjw9469ata3MPnPXxSSmSnBGn\\\\\\\/hiF9jYwAuxfoWMNyJRXzdkic\\\\\\\/QhccC33X4nD0u8506lXD5HY9z0EhOp0iLJvzaM8o7vp0gtMv9fCXZA4YR+yv72gu3zqJsbBXopkrll0TZgi952kUlrBdRg3mKa1yzGuzc3j0lK9z+ZqQbz4gbBcM6mx9mOZujDtApG4z6Vj02QPNmGtfahbWNwUqCB+Gmiq7GzILa+JzMABT6jw89E3LThrc789Jp2xj4AqHhCfMKNMBci8aRREcbHnD690\\\\\\\/FQX8sxZ3B2VCy2C9xH4qxrC\\\\\\\/Z0Mzjo5uvbmdA+OK2DsbbGWrtJKitFeGpQpxLuKv2F8yjB+4Z54t64EROeu7rRbRw2irD4Ldt4dUScdijf1k85qRvi9ScKlEV0cl+\\\\\\\/h+nyZ0WtgS6dZai+w37nySf9bpeg4heR8\\\\\\\/IXwTw0Rb0HFYme1hDYDceZygEOGHwgiJ62mVT523VQBhmcrpSg8oN0M3T2xCl+mWZ7GY57F2s1oO4dWZnaYkyoohiCxyZsbue0np4d8kihnpglhO4cia+6PLzNvkCk6O3kzk8MdGOuIznSZ5xep3xAoiMCV4PWMPRcmslZe2fwG5zi6M12bfwbYwdGAbjvZN0HNP5OsOtms0LBtYvrvZ0XhJ+qS+Iw9GaDsJn7MwUKLm3QdRdmLjKyxwoUBOnGbN8c2UxN+XpniAv71L0CC0UAw16y\\\\\\\/JzPO9ReZkphWbS6xT94z4b0EBuPZcZ+w6G5Yb4DMFKY9JBWN\\\\\\\/HF7EBjxbRYapwKtUek0UGzL\\\\\\\/CKVESRvBE4lgJ2tl2cMDTwz0gRduUftlC9pR\\\\\\\/14vGPhRZPpJpttQhCEwoCO6AQLwI3V0Z0XFdNe6bRMeKTgru3RgRtxNzo0f7rZy9SYNnkH155ILlI8x8LcOYpOrUzWdT6AbqOwcuBNm4TJdLOq\\\\\\\/ikqfSwP16fywrjMTX0crhTx2f5cl\\\\\\\/PM3NEVggTHT\\\\\\\/v3TChPwwvrmLLEH6EFAkOjF9vyk3Mwx6\\\\\\\/egoX+FzuItJ+h0AEcXeJMA7f55iRME+OplmM2MQO+3BhAnHuxbnml1Pp4fuaE\\\\\\\/6YeSBBg57+qYyLaXyfZ\\\\\\\/DCSwChEht\\\\\\\/hKw3YPMIkgfI6f3hTvrdX45sIUl\\\\\\\/epvY0AOQOHfhI+5UfBcKPsOMC9Cj5lZQRcG2BgAqwmIYrkXO+aBN\\\\\\\/6NDo9mLtIhFgE+BQqXHOEl7t7uWZ0q+zLCw1a2d4WOqgZ6KTJmFwOw3n7SB\\\\\\\/iJ0ZI0rQL32IGh1XdlGHontmbDjBBXHX1tzp1WYdrHlX\\\\\\\/\\\\\\\/4O7EkUJtDrBsZnoEIp1\\\\\\\/osmzGO1YffSnaFtsAMV6x6+1HOG9wnhMJcf1wxtBrVbM05T569KgSuLeNeQmXwV273oot8ma10lePvSybUyEvpBFwNLwUwLrJe5fgxlK77Hf0Ad8uwoyUCwv7h1lib9us4OmhW3XXtjEbuv6O80AF3Gun5Ue2PPyqpxa6Qt+CUniHFZaSvKjzB3oWf3yUG\\\\\\\/EbgJXNepTTkd5ZTXLHVjLNBlMtfeU71NZ4MV56FmL5vbmpKWzKOUVhn63Xxn+oLIgOyrhrPLOoj+oST5DLdRHez9xvMSNlF94C7KXH+v9cgIQcujYL+aiqtr1m4WNbeKpfbtkztxc2WAuFSUamqkt6A0QvG9vQ7oHUCwgfSt6rJ0+QRIWefu5qc592rJDPVegajZRlk7lu2z70W6XtNChYF6d9RVfNN5oTv6Wkn9oFLkoqMRTvjkBou\\\\\\\/9IASuYgFlvbKbo0vMRF84JZHbholXcMtyuQM\\\\\\\/t3Gwp2FB8PiHR6mfnBaGnsOV29Beyx2gZMx7nO0lrTaxuFlF4YkOFeYpZgmy24cHvDsoKjZRa6e+NrwvCtbdEyZPffHGHOYbJXSbqlLEi30qxQiXAtyMZM6LYMglR898P7AKS2kLJsyH1n6sidT2+zoyNCAclMqabQ6RhMPvftcnNr0MQgdrgzio62yQvVyoQZ9W5OoJl9t+oVAGhrQRoKf5AHYErJ0ZnHcu+U2Z2VBXfjVEXE4lbw4jAUdUoCcOSn1ITsXgjbC0oYQNRw9svjxAeEioIgqwhhmWyOR\\\\\\\/6ixrSt5pW\\\\\\\/myBynofbpaMAd4ORcfl+x8ihSPcgc0bYyBVr0W85JpK4eahkljZtudAGqL9Xj\\\\\\\/ZZMkpwksh04oSOrkiMMXhFu0zcHjfDmhYLz0IV5W69GYShOfJ5n\\\\\\\/wfMvI80My6tCDYSkmVz8KujYNnRsu8ikMsir4a0JjwlY+RYoGgX+TlQWUX2m1fdftwYC8vLjuvRE8cZMEqbYWT2pzOy5oZWy+fYGiCj53dvCOaPXqrsj+HWl7TnfdjcAhmiOdrbgTyR5C0jE8GfL3onWmJ+vhFuDVJDkF9aEo8VCaAeguMkEOuoc88cPYsAOnfoWWBvsnN9lfAs3sFCNyo1ZVLExVamKYQpQK5H52xCBVdyJomOaoZmTQuVL2CSTSJhl00UJS\\\\\\\/\\\\\\\/SqubTxK4qf\\\\\\\/gsXNLDrKAcqesh3848wc1UmbhzOc3yQY7sUMmLnYdhr45cYqRucWi8J16SBP5Al8wdUgmuv88wyctN0Ht8io335dov3FJp+1qXtBcRZ2dqvQIG5ZTDMWPOg9aAF41yc29u29QHMJckHwYyURv1Lrvlq2IqSS4vHmJJC55PCtKSKZ+7rZo5fxzWootiod1kWGfotOqRh6RoQuxlSXQnqNVpPi\\\\\\\/tiyXUejts9X\\\\\\\/vNs5fG6BOt\\\\\\\/gIIYkjDkdEqjxOQgLUJSjY5ABG5THNhCairndCYXNd1XdmTyQ3vGxPfKNtOgtWL4\\\\\\\/EPjFNUm89cGbNE\\\\\\\/w\\\\\\\/nN23BnBmhWGmb1oB9AJxTPPQNdxvS3JVzqQObrhXig8b16EsdF5fFq5DLv\\\\\\\/74bE8KxfhisayKUfcgZhqeGCUC2YMBWiVbo\\\\\\\/46Id0iolIer99DJDXqVKZHv56LXBOdsyXgT\\\\\\\/C\\\\\\\/kqxtzQCPUyrGopnLdEGtRLO1xFZ39Rbtm4YI\\\\\\\/9uwv6Whe3s0S+8cylkuZQmJyGzyEb9sJuk9sOL09HzsACQ1gmYxmtUfuoR\\\\\\\/soG2EPlU9OWXXBIXtE0TnUkDTvTQOP8Fyb8H83atbtFVBrhHPFTXXUnGetTU7FGWHvUQBCKttFN\\\\\\\/FCsBpJiPhVi9nfqWkbZyEBIEpDJK9GkV3QK39eEXli\\\\\\\/CcUpWctgFmBj+qUJpdeNohckNTPW1d83vSG4bxl\\\\\\\/kAs1NUED3rVcxB\\\\\\\/HX1Ens0+SkLXqBOosO4d3z21T2bUPLb8WAfSK9Kk4ihSFr+EjVX6ctLs2XxSU\\\\\\\/hQHiGnP6V8K1i8+BZei0AkDcYePhVoo1PmtW1pLXVjzZoVbhbPSeyHV9XN3gh8w+jRLRUROTAyH+fBW0aLG55NYB49wlybLwcOvgMHRqXZrSJ5hzUs2yuDs\\\\\\\/XM5F94rnchcgWYTvxiV4ilhPmyWw58hLvYjocELFz0\\\\\\\/ZBMMmtPoWzuQWuig8tz7+kMQo2DMAis8KQvBhZjMGHLCaksWghJehu5gU4+IlGp\\\\\\\/ZvlYlwu1hd9Zi5QgwKJNCvLArqE+uzHvWjY1DGFEhmhJS9Z7i\\\\\\\/YrAOeiz0GVqYPvopzgfZ2C7TwbN7hHCyUzHamfIRM62WVq0MqaJc5mLTbngr4Nec7uLyOPl19YTWaUcCtCI8TC3FUe6V6SwoN0OXw4RfuOpeS9rQGX+hdQU48K7fY8NyYLbRFZmebYJ0qGTgqe6S6Ro\\\\\\\/uSnVi7N5DRUhbD2jex9E2X25QPEQeIJpBsY4lKM0kv925TTCHk0t6usE395JyN+DqmCXmmW5i9YH729Cn1Y5fi5hYI5fNN4pDFe2dk3UPx+HkeagNT0IyNZ\\\\\\\/IlxI2tMdIwRauRwJBaLERKlCoiGwxe3ef8VP778cCPKNUQS4OwkPs\\\\\\\/MP\\\\\\\/wGivuIFp3nPjDewk93l41mXnvhM6znGjHYIvEXK8C8Y3nCyZwhd\\\\\\\/86Mi\\\\\\\/J3eNZ41HjBlOHNrj0A0273zz9IwIuicXSvpHulVIMyw97gx\\\\\\\/NI\\\\\\\/ZFTWtYeUl0NAWZM6bCOwvVy9Kqgm5\\\\\\\/d2rLs4Ur91T049E+VMd7L\\\\\\\/mMi1U5bpDGvN+RuBIG7+Yya1gXOUOpTBSasjugOF1Af7VdeoxiW9urjkV\\\\\\\/3vKIZUb4GAzrm0PcGtY85UBKJrHYOb2vCVHkACGhlOrT+FXdS8JgZBMK\\\\\\\/5jaVjacDPLU03pxLNWne1+RKhJCqDiy1Wt9TSWLhQyZUd4TyL1GF1+DDqLFr924rU0G0GvAPppSig8esC3D2f3uEdj92ePxgByuyOjG4ftrcKZ5GS9rUgZPEAApT5SVcL1XfWwSiNCTlmltroTR48+Wg2CwAxisgloLnyRIuMBxTxuG79Ig6yxJ1K+txX\\\\\\\/dhJNtWAPlfTWZXtCvJD3oFpC93CO\\\\\\\/Fkr31t7edRdYjlIkc88BZTaKqkQk\\\\\\\/Aeo7eZCDb6TgWRFa2EKpa+Ta1zMTuFsWNdYz6EUusXFaBG2sJjp9GKGC0UdMGRX9a1azDiq\\\\\\\/v6vecWfeuINlc8Ew5dibyl8dYAdsvIelK8t4OSo7I0+n9sHqLyBTOpnFf1WmPsxYKK345AI4PcGNlEPEs7DlT13WypD2f5O4zEM82a3fu3\\\\\\\/m6DaD4gjGyLkty3i3zLSJfnDKbSeCNIEtkRW+SQftm+5dFS4Sa3YWDHkxR3MW7mdIKt6B2DLPynEokTMSQlQJexGNE2dpC9Y44Tlp5ZuuLg1woUlOfTieSRFY69koaHtmVJPPl9SeYKPeaiLyNYG568Fr\\\\\\\/P\\\\\\\/qwlnwrihz43DK8M0xiLandFclvHQEnaD3NUPjJ7UyyU14dSgpcGgVSOVP3MV2ScK2jHNWN9vVjWbQi80YQy7azGnjTzaWYI351fiikGzFQJ+H33b1XEvffB8NjFboxLuxkw0CL4ptB6\\\\\\\/RyAAuvZrnydaxoZszq9P6atO1Yiv3Et64oD5DIlXplDOEKvOLdO0dbbgqDcCTiFE+x7lmu58UZsfQYvvi43aK2f3T8SUrn8alBCS6\\\\\\\/pQ+odjlhjaL2FPEXIuep5\\\\\\\/Nhao97wn39sZsQNLxZn3ISmxzNqtQ6N2EYeuAmptJKV6Fbk+asjIA1PDS171a\\\\\\\/U8J5Iro6Hjb1CHuPii\\\\\\\/F+kWkJNV1zAhxx0OkL7b3fnRamveyzYFDQJvmMFLNQOhjowagsSPPbZGIHbxYE1ZX4a1gu3WQVWqOpKW6s8IRlI71PejX+C6VsXlRf7cM7ba1G79gNISZBHPGp3dIR\\\\\\\/IwSvMs9C9pvLomtmE5vp68PmBi8IWVkUazilQsRsBTm3mppA9SW551ZKFwECrHJFYr6CvwnHgxX\\\\\\\/kptGcn7kkSpftDHIk9MDHiEBGEDfC8Vz3sGGWfd2J0gY1nMtuHrbP379u1FTTED9XAi78HzQe2Muh3JP3dlcP4ikFjk2sDjpAO48DZTXtQxftLI19PBnWnCVDY7Y21lWoy\\\\\\\/F1GzgfsIoVvv2+gCYA5BIT0pK\\\\\\\/Exq1m3GGME0f+Ve6QY0XurBmq6qj5Jblf35exjhqeCBJ94sRZX80Y3cWIiWXaHyLETwtWNfuAZVGi3Hmo\\\\\\\/SQSTrOcwqgY6Vg3Oan6kdMxYU+oLdBG89OzwvDu7HaNKmlawxihVpumR9icmic0n+TVSwb9OF1rqFqgp65pEjaGT1sd28IEcW2kuLUIeyFnJMKspRUPoG\\\\\\\/e0TlISLsmAFjR8TylqsugNwOCJz5akssuYR0+AV8tnyTngXkotA74eDGWFIXG5J62ZaKH4M0DYdDEt\\\\\\\/tux8TZCoAwbd0Ql0k5PZH0vGGfqJdbGN3FsM3ErrLZbnHWwB6xJ\\\\\\\/VA5XujS8qpf+tOqfF+s+w0R1uh+JK05PMrzhD+V6+dGhGHBpd8X\\\\\\\/ujFHmUE7\\\\\\\/fDhd4sLUJZT4Zg09vQvnv4f94zIT8gpzUCYyiyyrXRl+LLqrHYZgE524b3vQzQwG+VS3yZfS+YUYRym6BuGelXgpm\\\\\\\/5magQiK3sm\\\\\\\/vBTzQMeCHR3XtsjDeJeWv1MAqK8Ta8sR+DVugSGEnfW9xTGPpomfdJ9SWzfnWN43aVNpW8M5cy9vkr7186ZN+2rICzVZb94Yr17cgGSHHTMc9xAGvDjNSwbDCqvDyoQHKPQXnuIActh0LReJh\\\\\\\/Gx+uDoA3tlIbVrAuO2TjkJRRrmrzZrq7VPxTWAQpQbiUPbW0sdoLEJ7CG7Omn1wzhjR3IfYbWYqBzOZ6YdvzK9T0aRXMxXadpKsTddx3dbdLcIqjnxJQmOqTRUPksFpmfHvfX9aQHcgwqgeAb9MAaAi1C648EfNZaiKB4mhZneMbP+NqObK9TzmB5Pz9H7pmBK391w1ERb\\\\\\\/6zPt1bYtgdiEKdAyNgYQi26R6qoWH3g4NMrBNhRQ9iDDHzteUx4r2WCrpVXoPa+8QsIoRWZC7DLlMHnUuWzyqVNvmL4tkxiFClt1nN8vPrACpTV6ueQOoQmkzOljB8b0CG5O9zoOagmi4RNqGfDM5mQ+iEbMAIWw36y95\\\\\\\/VsB4Jn78jf6ZsFOuXhFRbgNaJA\\\\\\\/ds+Ed9AxbH9PwG0xp3rhMrDUC\\\\\\\/hWUccRXX70twof83CNIl1DghG7m63j0JITKBHuJ1\\\\\\\/MdwSGYrmBJ8j\\\\\\\/qXd1+3WESYjRMICw3O3\\\\\\\/3rqNn3OdVYn0fjR51ZG+HG6jJogNf4Goy0d\\\\\\\/rUlB+XCJ3Pfj05IW6+QpCwbNuX6tZj7+aZmJw1\\\\\\\/JCzKxyjVjfGd5XVzX3gyJ1oMu8bF59MgPU+3TnCLd57ZZ0lYJ0Z1Uw84mWzXAs\\\\\\\/cCd1M87HI+7FGerD74hsIKxiLzvDlQyIFFTy9P73x7dryx+5Cd0be5ZXeeB6h472CJmoSi63cdNJbJWtaIbniFpqKTdPw6e2klKi9nHeytjg7yHPAuDCrQe7NBeM3JXGcZPwN2tB6EznqAq9QO2IFLCUVlMYOZwHySVxYDQ9djlEoGTLaknFM1EgioJ8BUgMzU\\\\\\\/f5Xq6Kf52cglnNI7pwkaBihL2dpAY56ipijOZMtfWnVTVIdoJ0MbzUWmridACqA9+JgRTGrqifXhtkjE7NJW15MCY5QlimKJ6izk5TSzqIYfo\\\\\\\/wTqkFuHNW3ri+ad\\\\\\\/KKVLcv5JwQ9t9\\\\\\\/glulqdTmDCUDEYexwRt\\\\\\\/Wk5XbL+fZuYlrSAdch5TdSl6WrgOK3pPGbSfQ1hAMosins4u\\\\\\\/cnXIoRcEvAY727FT3eo2\\\\\\\/7GpBI+kztGf1CutOEdqizES\\\\\\\/N9u\\\\\\\/LwVBJ07rsXUdKVfwAqvUJjKLWhLdqwagNmq+sTr27gcw9taI8ysD43m0NvL+tYOhmJ1h4AJn2d2zJZrI55Lru+GvUQji2jgSFftXjakyLGhzeJJJyHYI0YN8v6kNsJpLdbIPN0WygpN9ATReiWehQf7LBx0jQsRA+pF3y4+1YDWQlybl4hdh0i22YDuaHd7znfk4vD9Lvpn905Td0Ut9EoSpmjL\\\\\\\/EaFGLsRe0RDDYcC6fX\\\\\\\/IHJ86kJSPz+2GJW34Ao3t+khGLc3mFxe8hbUR68xEexofwOURighqtvf4PvDiHMFlDdFKlmP6TcOfDATAbdJ8UUOLRt+p+x2IoxCY2bSYs+TYbMG1K1I4+fL7lOaUwOnEGhM2r3L6+yPv4gQb7mw78v6wdQfIyk6OITsUJSJmutnjFk0hdPaSAA3ZwGmcI\\\\\\\/cNsOIUkY5wwg3Wq8JZD6150mHmT\\\\\\\/uoIkcLTFmbDsy5QOj0kDmkHvsTi6IgE6jAhWGZI3aMBdoaMJ0PucWSQ+MQgoNc1KBEMYVNqPq8BAykzgGqw5nKejcHY4wcIBM+NDPSbE4WGNUNx2RYXmC9BqR5sXArptoWYIwJKATBuoLdLGbGErMYsgcjs+9b8EdjA6\\\\\\\/yoBXXmjwcNrChoiYWm1esrez1nZK9Oyy2grUDZeRWrLk9+b2jIHMDvxLOgAUT+TtNOtVTfYEO2U4a45r0SZx6xJgK1t8\\\\\\\/FnVXsKVYoDWCgOYeUcgvuM7J\\\\\\\/3eR6P6UZzvMzn8PyhDMljcR4b46soGdesQ7X+T0DA5Cu3i3Yi6v1HGak\\\\\\\/13F6lRDuvptHtTZZSYb5GMHSYdWtmke+3cZxS1uA832lwtTHEESNZ\\\\\\\/P17krlX1z9jXaO4s0HEeYcRp9PrWRtuvuCOjrO05HFzpgF3wF9CHUfcGyD5pdYh\\\\\\\/T8RS9gN9Nib\\\\\\\/SbySMJ5tDx9BAA5SBpMoI6dU3wSG2oqSQaSsTr75CeusdJ99c0zS7KD6W4W1pQZwBa9a36zzVUwoI2ze13ZzgbhcHyyiMXvUeB9WUkN+oaoDMrO3ZupEQoBkGBQ1Gs46M0IujEisZuELY9kRHcIYtnXIRHBQalOymXWUQN5YTmTdv\\\\\\\/bocUWUagK8kGeRXDTd+aTXOkiUy5KjT2uRGdItDf3CWHxyrxijvwXmUsmQyYhm9pxF91qDusYbW1EZ9+h+zlT1GgH4IBjijJBChsniXQiknmFJ7wxDi4SOwwaAXVXhsN53aqA0GVOokenfKzVsZXTXuQeihucwITbqSuR57b32gl9h15btjq11X9WpbEgwRiOgk1FtkGdTx5BJrzE2Yg+QnUY4sXkL+2ILLQRJJFkyf\\\\\\\/lKB46OmR\\\\\\\/T5Ym9tTRstMrIcWL6Vy6gu4ie7qKtU23\\\\\\\/02TGa\\\\\\\/EAxTiilnSNlZ0QQ\\\\\\\/OZBVSIbqYWwykBns9\\\\\\\/kbRYyZ5uSBj5PwczQ6Fd5zFOfVeso7y0k21Sht6wdckOS8cjBIbR5NylAAqz0QaiEvA5aHaqTL+6G3V9TwX6\\\\\\\/aS3f6F8KeuMWNKTSm6q\\\\\\\/ycGAl\\\\\\\/t0twQGcEXYCibEvHyACQKu4Wu\\\\\\\/HLp9MEJ7lswEOozpmkzHC+sP5xUYU6xGC9zJoLuClg8114eUQ+jpEvLJ0i5iV8hsCu7Mx8UVplSHyS1gXacMcJs1bU9v+4oUzqyapFfyAkkJnbl6zysuADsT9dfg\\\\\\\/kn0uBsWLJqNqMmPqyTAxJSYv7sO61KJFHJuuD0iFXydnpsgDFxwalkF6k5G82T+BbRgi9UmoIzNqdxi\\\\\\\/4L5fhJ4ehuNITahGkddzdONQuA1CRsTtM2AWTKD+uNDCvisB3Olu4xpBO3ErAttK4kMTw3ZMfCKXoErSmAze1trbq5rYJHj2kngXFJs5FgS9oVIeYWTHogSnHVcdcC7Km18Elkw3BQda3R6anxhDh810Wku+6EF6v2m5VzjCJHapVMGv8MMP49Vp7U2YAMNb8s6wr2CsQeQ\\\\\\\/aT\\\\\\\/61HFc35FlmVuMAiZkqqB81AHoJS3vwIhw8b4rwUfSopjRfaRC\\\\\\\/U6YqHSZVjkpYDOMGvaP\\\\\\\/\\\\\\\/MncktZwoGLRKFqQ97LMpibEvKtLeE\\\\\\\/w6k6DSmmgdHv3YNuR7vx3a12uFxd+3k7O3SjWtUUKapF++jr\\\\\\\/sphcyxbt90Vq9bSSgKaBT\\\\\\\/Uec6g31L5cy7HpxFCLJpcZJWXSu+fYqRJNzLg8fDoB3P5UT1ri1zhuCfHcZgKHrYWPpWXHlt0NhlzqdOsVz4jq14T7MYI7WdlJZqSS625GDUKDKKuFdXtS4bWfTwrHJLdOtTRdPRFveDPWU5ZH7G5bKtArF3WW8XEAFZLo4CEHX7+\\\\\\\/mtkPYNGn8W9uctyzagliAyl\\\\\\\/cdnizWat3OMhQvuIAtyTJmHmftUanXQQ5cIiy8H70MRnKqVtJ\\\\\\\/bN6wnFmEtigvtedpXl1pwgm4lGPiI\\\\\\\/ibAyhD0QV2\\\\\\\/sUuiFpjX\\\\\\\/+CoAv9MQAt35\\\\\\\/LHMLzNVGUtSqLc3+T21\\\\\\\/hcPLeyz6b2bKm9n+Qk0kTaOQKbjTcQHidvsjikOdmyLGz7bFbx70jQ76qYoQh3jYvy\\\\\\\/BUN0dO7jRNVpLK7+Am3wvY7sxXcn4\\\\\\\/73jhabSgu3V9A+BMAQZGOBTpaGk0SuMrjPTUHW\\\\\\\/igVbKfRe9lJuGbqjLCXq9HYaMggdJ0MFkqWJJ3BZ+LUI7ue3h8t3FaUTDmiSMtPaat+cGLUBA7xuUZitAflUbxiXzRDdtQs476VMQKTxMLbQ3eXn7SmbLkTIJDi9vje\\\\\\\/kwI5HfPAewldwjCL8zHcPdMEgViIz370mAvCM6zNIbhLD6CXG2D75K\\\\\\\/RqN3hCbEdfBSGsj9GpgcInbDLmlhzxEYEoQCtPmf+TITp3G4L19pmE08Tsz0hX5\\\\\\\/XCNcSswD5cj2KqjtxK+Kh9lHBTOxhhjOvRynppQ6RAW0C+UOu6j\\\\\\\/6pOfdbze3pqGnRAvAM8VjsnhtFEvP5aLfPlQQTzapWo2QK9fp5dtIw5uviEdjKL5oH\\\\\\\/0HolkH92ZFr6FjPLZ2sFmOXwsolGTWbaMLekqK6Ovb6W0DvOcPnBNo6sjHQEQx0A52XGS2ftD0lHwWdPZdcX3\\\\\\\/P4rwR39kZai8xmScBOQmMBf3i\\\\\\\/+iqXcAAD8OkPyo70YnQOpOm6f+pu\\\\\\\/evrb+R4887uGVOmXe\\\\\\\/UuwsAD2RYCr2RRtW+zPIO4jS+ydvu0accMD5t4vE0t9AIWAq68FAITBJzOoDrUGE94zCWs6021sELuWpamMDdFfW9hHest1ZSZ5IG\\\\\\\/mUhCa6R4Ak9yE4oj8LV9sOBPvug0IeDmwvqKvoG9YNzg5G1qfxZc750Z9sXI\\\\\\\/Lcb2T1aOJkacTKFNmbLzTl+iq8qlSyPYHNhg8wOnk0qzvqnKlmMbnf\\\\\\\/5cHGcJuLVpj8fS\\\\\\\/T3Z5Xk0RlCRAXCexCYtyPmhn3zt\\\\\\\/9oHSf2cPXCLGnCsCEgN8la8HsObQ+om+lRk+lmKkZ\\\\\\\/9JtzPzM26+GkUobF3OPdLX5QMWumfquXH6b4bBh\\\\\\\/kTDJ58AxoIUPpKBsTHZdjYigExSF96DD\\\\\\\/9LYb0KBBY1a5kFzFQjoJHBN3f8cUHWyBCH8wAm6iaFRD4sODf56qMyqqn3tr9GEb5B8UJFURZmBPKdpOcWYqHU6uOOio\\\\\\\/D19QdBvJYuoAkgei1UIsFSSIwXXm8w9mMYWiQ7s\\\\\\\/QMqFD2XhiSTPFv\\\\\\\/aHEwL+ZY5IIoTPtKgoFneRJeCCai\\\\\\\/IFnAYqC985YZBu4ueAXhERq3YaPE96S2oIE1uc82agwOiFZmq0VjCgZh5hd15bI5SiOvYCCve8GNGboTW9+O\\\\\\\/knrp937o3F8GvtXvOmQkdky61BAVMIAqqX9T27SYB3LfODzRf0dvgoYZXGXU3PzhH1YmbsJ4jj2opYfwj5g6FCUWQ8ZUix751hPjLnQhkUE3+U0vtuRPu0EsZbJ75c2ZAIwldk1JFWpCsytCv5yw3gRIS8eJ7teBVPLiNHNDdebxAV1E3dXCKLH5RCMUBM\\\\\\\/RWUbjdn1fit8JhRVxYRmCeKkxbrsIMrZY1b7Z\\\\\\\/xNmRdb4lt6xgkgzWnfAySlTWojeIeJey1shE3pB8u5Bmma8bi6\\\\\\\/w1H6TXiGnu0fpQabudAIOKA7V8sdgruZbeX+J6sHhdEXgysGpYLY+rp2JQvBjW5ZL4SiJMzEUqrJkgvNzHigeZx0EgBuWCWpEEAw5pIBWKEOkR13YuwfElT\\\\\\\/gbkZ1E+0ppsTh9oq6jb7qh9gYEAxBSZmhh0nxXx83VpDQ+EZs8XKiSn0KbL+Gw0VQWM4WXUdJbUVmOAeB8z47Ncva7LHkoovUnaiQKt3aBhhN9tfo19083Kf\\\\\\\/P9yaPhPHtlt94Rofpu4KA0\\\\\\\/XJfKc2eTXpZO4lUqcQK6WB71PFPtuxhtPzJj5CYVbjt\\\\\\\/Q\\\\\\\/\\\\\\\/GRwMtlu8sRHJ7P+q5C4\\\\\\\/ptsuL6MB79S9sMRxvoFQZrTCbdUT6CRL3IwwA2UJv9tKTBy8hCwbItiglpjm1NKfdbq\\\\\\\/kVnmfnOa8o6p7\\\\\\\/L4CtUXp1Pb4lOsT8JxrIjgGcCQiUYrFDCixjBqZxxaeW12heJgKXKgp\\\\\\\/VE7xPj7ZU3MafxvSPY9ed4S+kY7VGTJkIGmOz4edcSl83VOccy5xo\\\\\\\/qi8lyUidw2\\\\\\\/YIBjIxwZBmosWk2KKnKZGaD03yEBShwJhv5sPRfYh5VqFoK93VXqUFzRi9S3iHjhzqr\\\\\\\/RIveyyu5AG3reersaaufX8BG2NDJ2ED3rJAJ5zRLBQsXsyxUb46Rw8Z7KaNQzXhc27w\\\\\\\/4fdEfstFBAezbygK1NhfolF2DzSjRkzyfcOh81SvC\\\\\\\/mPZ86jWqnv4fBg4PMq8qHgz0SqyL\\\\\\\/JcpkatOIFZEHJ26cVEp4Kvqb6ylEUojvCldfZV86InpWkBFcNLfYy1jFQbu73Zo\\\\\\\/Q6\\\\\\\/E+3wqAxBRmgfH0NqdUou\\\\\\\/kuIxfsH+FLGmqmfk+ED\\\\\\\/5Yx77T8qgwF1tjHYamIeWNAwal8P9i0N4j0\\\\\\\/ub3j5hSeGI2567XY8ZBL6mfn8aA8KhRvLMP10fMoFFxl9q0FhxcMvq1LEML9USAzfbMEZOS\\\\\\\/QaMB0MgebDEaXRJ9dDtBeNbTDnjhq+7LNfYxOdmfeiJdZgMqR9LtmwmbfQuQS313Rbapzqd9hSDtLxzIc1oGzzVCsbZUZUhKBG5Is5ySk4TbnOGpfq1XtxhFu7YSnVcqakKNHRMxcear4ZlfZWvw85g74SqpGiiWRT3R7uRqhsrSwSitkzBtY+8LvIggLNSVZQdQLhuqtsgQrMyqYfTs6uvUwUP2RM4ip5zdz4Rg\\\\\\\/Pun+2cQ6kqFaqcQHlJyqV2DByU1vpTkIrsqKxYrNn5G1B9JUCYwKO1utOZb+hbL4v8t5HJ9rH3poyu\\\\\\\/2v+lPEImx29ZSzjspPaJTMwhNKKa2I98MWc3xXReWI8qGu8w4CatNwNf33p5t01yClK+qtJ+c2lTRuMjRt7Ze0pg6BLbyyz+JVZzFk\\\\\\\/o2T+BlonvXdMCj0ejQXkZA1U73gekvfQtX847HFE3JQnv\\\\\\\/wMbyCtFot4dKse8gh1KYw35gaPmVz8hGFfl8QUag5bmdlRQeZLoUqqg\\\\\\\/RWljfuHcBdYWpHjp\\\\\\\/XM8cWT\\\\\\\/Exxh5SIir5xqS8kCWluURXRyJWI4r\\\\\\\/QIT1NxKcrBwaq\\\\\\\/YuYQ9GEhQPQJQPLQl+qG6m4sDUtZpz9q\\\\\\\/qYw9E4DJ2QcsSbwJ5B4N+b2QAtX+KWN7zXnNMyGSS2dZKl34Y8frCCBvp0N5cfIYHlH74myOAENS\\\\\\\/ZwG\\\\\\\/4fkVHPbF\\\\\\\/fPUh4gHcGleA1cXYD7tnJ8TEWlgFGBeC4sjRS0+UrnRQ8Z6Vp7ZPVCJY\\\\\\\/LnYE6UsCCzE44qbh1z5ueD4B1qpO9Ow612tXc5nazNvs2dCyP+nJ4buWa20X8KSjwysP8KwXpAGGlflrhj+p00lLfc6XmnFhIz1hkihlOLDKJaCrfxPYhxWCsCgz+iO2jU+YiDwSD9PMiz9R0asTedecdOWokkqmV3c+tkk0eNRn+Szkxo\\\\\\\/heGyJc\\\\\\\/tm0avqvs0j9z7+VVOKdGpihAKGj1rE19\\\\\\\/rJpzsAGf9Nr\\\\\\\/RTXm3OeXpP9YnFvHvPdvWMyfL\\\\\\\/AokNhPKqyD0zEgF\\\\\\\/Ejnpso18Cji6sHYUce5IQRvwUS79xWES16AIb2TgzXngJ6ZtPUnnO\\\\\\\/GBvq57RUo5cUBdOahbSiOJkapY1d5KA6cZDVhh0QgJMUn5NWWW\\\\\\\/Kr8Re7G+h7nz9GmOfz1QhGbghktV2LHm2CPoVIWw5ir1RbIBr9AlMTwhb4hsZOgeFhQr\\\\\\\/7Vq83oEFl1OqBumbsngRPvAOtwtr5u8qKwqeKq2ohsW++X340SaGRr3Zm7YLva6HQPFVGMYbkdhoxZIo2q2v3BtR+ms4\\\\\\\/UPpFgHKcdJCzDy5KwOmCvHV3TuinJPo4u9ReTgN0T8vNlmUao2rVF7yVNNBpIhBHSkD3slNQvQqzU4t\\\\\\\/CWfibhF25nSEd+lESE3rK7Ni6I1Ujm+k8gKRNNjAFSYp6prNMRx6EUUGE4fQA0HWPZkqWLKAah4FvTi75hnV+BdFAWTC9J\\\\\\\/rpqTZBUcAUblJiIqsAkoLKm0nWjUw4AB7Jb\\\\\\\/yzG7Z07+2riKTlpTXNfPZFQ86plTo7KkhSCzkyRmszD8yKvbfNKMlZopNf9O\\\\\\\/E\\\\\\\/Gy2apxQFJtfKikihS3ax1qd8wVvqrnD1jU+0BWaJb0KvwYIMDLhjWLti\\\\\\\/YcMCWZowtrECYja5sF8D74KTDH9CSvsDR7zwon+MsaQg4a5xVPo\\\\\\\/F\\\\\\\/1NvEaMSDt3KBtrT9zdZznRRqqFedSGIKgnQSaJ2Ef8aCBHmWcTsRGcueGZeKReetbw41Dc5RTEQec8CjnNJ7mJ8\\\\\\\/H3DrsKbSugT2fuY\\\\\\\/c1epWlddGL0nqZ3rZLDMzINYy8RT12LRZU7G1tJG0bvmmmxL4EqWsPCxORm+V\\\\\\\/xgG\\\\\\\/5KbMU2pt\\\\\\\/4V4uCGzOBHs5qUsoaA\\\\\\\/dcIrLX+ysbFROR2RkH5CMn47dDC\\\\\\\/wTWWGmwodn88gtkcpKRyI2h12I\\\\\\\/iOQIfEBzd6fJ\\\\\\\/oTz0J2gMV9kzhb2zvqxFPRzd+mdWSyuAX2vUeW1PqoKKjkvt01rHF9GMU9un6RosltyRbJ+zJKPL5m5OwVq3oLq2M98zFOUiBBNs90xuBPz1Rf3Ad8raxjzg+cKOMR2HYl\\\\\\\/dbqZq+5TbAj+4YQcr9mBAOZxJGao9x05qBHYTpAnTg6nEwAMbXELLW9kHVbdHVKgXMcWpAq4drinEhymAc+tHauBKpP3kFDW\\\\\\\/jGIXW95CnXs+JUaYRUPOxNmbEpBdTQu0eAxBU3WG1DFeeb+ADoDdArkRrmLBWny7gGdlF8kBbSeG0GoXbHIQ61zJ4ZKEbOZDXvQh1742QnGRESgDr2bWs\\\\\\\/PH2q41ljv8HQLCxH77PF77SaTMgyP7qH3eErw3EJNUhx7AapAh01gIAlSEIfFGgM5drUoHhKamfrqCKykma4YwJgpEWnlba4KfgY4ZPMh1dB0Am9nJoPd33Ie9AGMrgvkNF\\\\\\\/oC4thdEPrbn2mhrTeeViTdxbMI94pFkjweSy15FGLkKqnnYPAM6kWt6znVpMzdJ+i6r80rkQkrEH84XtOkHBQ1u37lvSZHcWyhNH+61nNhLSPCKYK3vA0PvviJU\\\\\\\/pOE3zomUQWDMT3TqIhhtCubF7qCZyRHTvj\\\\\\\/2QOfA7q5o68KnSSbtcl1Wotgtlt2p7Y8Yc\\\\\\\/SSsnIOLE4swKiPKVMlZv1PEQtwPEhSgwmoGRuX2iGHnKwvJUzRFT\\\\\\\/sebXiq1waQnoMxEYxv73QBPEjV+5PwwTUQueIrAevOqXnxSqdD4D5LcxCwzHbX0eYdnDdULqAeEjMBnGbx91wACOSdNzhLyv3g+oN7fSES3mB4zBx3XAywg\\\\\\\/Cjza6nH0m4X0xECI9o71hRPX3Hcv88cGqav3x2xCCHLvLUhps5eFI4q4ZNtYUWGdV7XPsaBGKWv9CYve9LjoLaL1S0P8gnbvJHPu3xaM4wg9hrl3ZhHA1c9zcts5uzHpmye7CEIDj1+aHrbKFyX3hUu4Ejgiw6WvD\\\\\\\/sqR85Oci5+arICkCMO96VouowaEpJEHjstEyCC3S66jOCgJ1KPl+6h74nx61d4GM9+FJsoAYXn40Lx2O73PZ+9r3nVjdBzlcLRPx+Hko7ceTWhXKtz+FdbSIbmfPgPPfP8YCa4A7KHwkBhiXmWnDR4XPyUCJ+tDUc58GbYwUsz7HjbQltExumqvp06N7Ap6FXe7fHBoq0x7GMyLtaqsbD2z7cOYRXZMn4HX3NxiMLgqxxEqgn1bPqURfZs9bUXFb4Ty1F\\\\\\\/i3R9KipvOQV\\\\\\\/KaeZhfssa3f8N0b\\\\\\\/WrK8elIFQSUh7abgeyswQEN+1haPZkIp2d2AMS2NoV0VjDFSTmTmOQKlkHqRJRIgxjGAJXSKAqqLq2DRrYS+x5k7AL0gGOyo\\\\\\\/EilfGCb8bntKLsusiqMm0p\\\\\\\/j16gBaIT09hNtD23UNxgzL9jcy1ls2G03zwCVqsZ+vLkI14usztNJsVLdf8zkYMhsW5+H3AS7XFAfeZmSgi0V9DXjIhNhW4Q8345WJS+4ig5ksg1Yy0wy3z550zGBAdC3afCRAr+E2Zz6uPotDWX6zWnobMDQMj6kA7L+++RWDCdxDyDNEuLq8bluCU0\\\\\\\/M8euRF8+v64T6wnQqHiVF0aCCaIwtW7ATkVxuIqS\\\\\\\/JQg43OmKvqYTXn60h5P3Nc+UjxfCIiLhKpDRPNWeKHc4Cm5opMpzrAKZgA\\\\\\\/++JL8zB4ArFEcEyCN2u+kDX+YrF2dG3pGgKJxKrQqTzis+XX8B5rxd3VsKwJ7JZX8paq61hFC8sz7DePk7mmfcnWdx4yoeZvulSvrqFP6mfmrzOrarWIIOkdk671CFhhXOanrupP7bFy0wVtz1VwbJd2g5duevTQ9QPczzZCIka\\\\\\\/PN4i6Vc5\\\\\\\/NmV8W9NqFabGiFKqNhXZl51ETRGPXKqq+Z53wjzWreO\\\\\\\/w8A7P8\\\\\\\/XtV99gubgvPnPB86sSkFdksK3oNG1GQ\\\\\\\/tMo53Dn+d6i\\\\\\\/\\\\\\\/O7d\\\\\\\/3JkxvPCJ4xzsKPbd3FNH+QFlP0NYAKEkGJniv7g4QoKG7OzPsRWnu9hficFY2CGVfoQRfN330XItvfXzyNahD2yGR8nxBdczalEyIfUfSv\\\\\\\/Mo8Y1env27blhLOSZBRsJdE4+7tl1rV8QKEUGc5qlGXAa3lHWvW+5tTSDHUA\\\\\\\/CeiCsHjmpY9G9E3lvTRgyMp0gmw6rRqhtuWqMr7Y3MaCCsKeUiXvzpw5VKtC6gxe4K+upVNrUkUHnqKabmGkHBTUveDLljjX4RyN0Kh3HReHyr9e1d5WmLAVD8DKr598X2+CB3ztbJL3dD\\\\\\\/t5x\\\\\\\/a5xvIT9RG67gn0mafuUfeMFdbzOaaNNv4cFDBdi2rJQIZ6YIUBX09eOdPgnPrwlFOySj\\\\\\\/qYsNU+\\\\\\\/cWw1mxrpF\\\\\\\/LrPM1vXNYX3Aqaxg5wCbxvb73FxzwqbSaHuqU\\\\\\\/9g69ZSchJswRBfQtw6DAR2SJ4prprI7gj7hTgIVbI119SGcavRAM3P\\\\\\\/MHtI3ME2m1pHmp3HnLrMLOOtwkWUoW5GpQsqrytDnLXBufO8KjP9COfS5L8xO5Ad7cevPy8UdXlxvSkjHN7orovWRL6ZLsXsVLTTR638TiFJIeiaxoBAFQHqGvlEVPC2TKJwI8nvRl1PalIs9a6SH8NiKDdoK6krjgrKzrBp\\\\\\\/YOOPdCl+jUv\\\\\\\/+c88zdjhJ8t0j1eUJsUP8HFVk4ZLB3MXKmAUKljK+z3+4HIdIQ\\\\\\\/kruwYlHbYDMJfpr6wnUJjMbf+rtpxVhFHH\\\\\\\/7tHCQ4eWSQwjxrEAj1qCRug0JR5x0qnrn1jMkIRM5Z9HYcpmfhSzVu9NJ830EiWMAM26kALG8i6nAdkg3+R4HdVdpn\\\\\\\/soJnPW2P+QMQL2CD+s8CeNYKBhZCDqkpBTZF6GTjU278J1FYztHEYbvgmNWPRVpkHAVdDlDU67lPDlZ6g2QBgJmlJR4SRmcGb2mlnRB6O3D+tLaOdHpXQNhS\\\\\\\/1r8KA+elKae67dgxtYMbeAC0IFIcfn0kLe08UypcNBeZ\\\\\\\/IKvFV3P0iXTAGuhR9X30nXdr6qjmg4rmjCf6euc5pfz5\\\\\\\/C69lFPqBbWY\\\\\\\/lJ\\\\\\\/oN1cbMTWrwvEAsh5Rz\\\\\\\/gwzsn5IBnKrGeyw3wnwXBNDRTdBUfWuicxnrphM4tYXyBK3DNGhlDvMC4P5NU2rvw34+uyWA6uzka2YOAXvq37wWSDx4vR5uLPyhyAK+z6Gy5D75rklOaKlDUoLTmJz4gehP+89PBOj+pgucav7oNko0fVIsLbPQVeDnxpjyUEPrhGRvpr228fwftkYhsxAcaf3JbCFQgSl0ypgcp3t61LlwxGYvo1dSgHWJM\\\\\\\/yqx7yk7\\\\\\\/uIQr6IWVy379bDtUSFt0sYsa\\\\\\\/Np+N9+iqDhLOb7gK+8t6VFDiF8hSN1g+Lv5p1hrt5B7cKDxNGaBsLSNYEeVYQ31PIDWSDswI8HX3f9X6l7DUvwiJWzJtkzmYRYF2U+ID58WHXCsMkCqhk7aZnAT6bH7pUTV9rR6lYd8ifM4OY6\\\\\\\/NgiF2SIoDHJWxu4TD3l+\\\\\\\/jWGtP6H3fqv2G9xuYmzzBzdWgIYEytDsAqe5ithfdZttSxMZmnOVXWC3WjJShhuFWoQkQkM1801LSnGXArWrNZBdhSgL0KoSQ\\\\\\\/qLVvEUPFfzmk7Bh+yfd70HA+oe4+\\\\\\\/VPdI0LgvPMG77at29z5zF7GnqXaEtdOoQeAWjZJ7kM3obwGZunuu7m9KYhQwBoTCZM5Cy\\\\\\\/90XXkT+OjlgqH07wU7HsBdJlFmnaYCXS2QkkxiIKlo2WiKApMLROFLz3ozDWOxfTOAgx+e880OVar\\\\\\\/WsiJZU0rhh\\\\\\\/XsIIFmYqI9+KsKzAx7cwcghxildMwGKGyHzukCC3DutLanZE2qiMXSZLFRy6bY\\\\\\\/BbHgUPenlcmJaudpgg3OAsclSDAumzLko2Kpwxo6i3SqFLfRcaRbdZq3hT+d\\\\\\\/TDTe4BcmvdoTMu5F\\\\\\\/T4RVtba80nJTaPhzhmaU\\\\\\\/mltXz26upF2GoYQgwo7Hf4qXmmGNm8ojehonSufQCHnNLlI5hxg+xKLQ979GjcixlDdletSqG1spjyK4cbmA+Rcn35MLvuemfaZUOa7mXBxudciE0WqvLm0KHRgLK5YRRUgUYd6hOx33Bac6I38iDBG6sLB3SEQziUXysxOLXA2W\\\\\\\/Cb1dP6fv\\\\\\\/7jJsdLWu4yZk7SGR62ijfYK+\\\\\\\/cZmB5hntIjlvz25oZaGcZDaIjN3jaLwg\\\\\\\/aOb1vyRhjOnZ2B0Kxx07LpKlTqCb4PD5tMnbJlpsmQ\\\\\\\/Bj+\\\\\\\/CHspQz8kcNp+hC41h4eh+yOcOFY88IcN8+en2elfNcDWwFrqQGyL\\\\\\\/cAm7+GGXzbWbz7IuK5J8iS3ajrG6nz+lPxRBc2Y2mTOmmH1A2NJotmQgx8HK4xFDzIbVlXfxF5IWbgKdhCv7HQuye8Y8ArPTWYudgNr\\\\\\\/yQYMjqDUAk7Lm3EL+Lfnhjpa9z21ipFkQf20Dk0aYrwIQcU8E66dwl669ZjlGoelLNjrp788gBYnFG\\\\\\\/HsBMKci+t8\\\\\\\/tTZopPQP+fy7JHezkgccFjMDVzWg+28F65diUq2nVXAe3Cxm2C84cSxc0c+o2BbxcUkmfes8a1Yw4oR3feGEOEe1mP8NrFXeW8S\\\\\\\/+OpG0Px38mqzgyLOWSl72gC3ZTT5KOnwT6aqg8+bx1BTQoZBtS7PweQKHjKhqdVrhTHxyzhDt5TrSWx\\\\\\\/FqNCDXc5JCdDXayY\\\\\\\/guzHPoM1LYaXzEvSxTXflY6S+TIhw\\\\\\\/U30Y6lGD\\\\\\\/8SfxXddpSQ40zPXcuNSsbMKlmeduSu8BJ3pJnG1tvIxZNm+cusaFUj5G1EGDvyzf8QRcFvyG9p2Xpinh6emM+l6VJuLvV0fT27yVQz2IU4ciNN02lnuwpX+IjtXL54IXTxhV63rX7yf0Hm+66Tmsd3cNBRD+6saCFgixUKVZMxF9HpStubp39bejOWYw6owueUR0ABCaJzWXYo0JZI8tmzFOskD+\\\\\\\/X22iKOPgTrh2Tb5obMDHJbC1w9nrlPuE7ibe07xgBFZKzohX7yC75ZevEp1RXy9xiuCGaEbANTTQuJ8I4cm+88sMDIL1SH9EY\\\\\\\/6YyCoE0oYidB4keM\\\\\\\/v11n+OLpy0f1IW3vW1rC6GuXglJgOdGZ6P9HZnIF9EOcW6BLqQdGODWWXM9sUcr05L1AoXdjw18vePT\\\\\\\/PZgYFTXdNcZhwwv0\\\\\\\/c0gegHCNzdQFT4Q2yCAiRJTrGSLmV8KbJ2xzR6Agk80S2mZoxqtiG2VILFfKnFFeLCy3if12KpQ2rRVHukTcbus68CrTUaLXnrTS0fG2bTTntYAn8MOTgpAiJrZ53kNBHtvehDHM4TJ\\\\\\\/FoGAhXDqODon3E8W9aayDOo8L9fCeltPZ0z8V48oJ9tmkGleGrHRpW\\\\\\\/pDrhuM4zlqIhtGVVYq+pnjBz+18rv2vl4SN9XThflj\\\\\\\/2slIPQdpowst1oqwMTUDDDoNk6TbZWFc400yF6fy7c++8ZEMMePyziGVMk3YenKSlVBj5s6ZwhNhXuNrIXCBDSe\\\\\\\/6H9ejJ7Qs4T29A6Zc43UK\\\\\\\/Z30Ki5QrCCmI0MblwAEtCuXU3QA4CiQQq5evT7EE\\\\\\\/voyvPnCXf\\\\\\\/9Zz23JU\\\\\\\/V3CS5udpitNWWFXworuxuUUFIqNMZNZc2gN4k3F18JLK6JzQLSbT4qFroyiNuP9DAsmIgyEJrKgckrlYWkNpIywu73g3LSyD5OBJNv24NNgb1NsdVdthpWSFpjtvA1U612UlAF9ZF9Ewr\\\\\\\/iQGvY1nyuWP4O8hMqdpL+95OHLqwdoyu2rdolFlG0NAffoX1t87nDGSvTWQZU9j6sGF142EMYE7FIV\\\\\\\/NvOJGxrEgxNf9r3e8R1F75VWRT3vnxMyRr7sYfl+cuVcSXjI+sFizyAnurHqcwDwUuCBHNhM2o1Qw21FWBUbNrHkNS9eUlaxFLOoezXXf8M+MTc+Mrj15VOjWShQDQRrsR7VXNk8PIrI04YzxPb2aHTIeNMVD6J1MefVebrOnO1r9RXJC\\\\\\\/NP4SCT6TW4kGPdan50wXOAbjKcny7CZIUhp8\\\\\\\/TI4uM8iG411aT3nFpYQmRvGMdysMJ6fuc9CbognGgRd+cf8MSlM2bD+46Udkzituw3qjDQPs8WAnm8KNll3gMc4ndRMy+1NXilA2DRSgmEFQeR9Pmo+UInnmoKcRKahO5Fx8U9avf1XDcoETF95aCSRmkGHn3yT3wzRQdmce5VvYVPQ2if7ulE6XGGeRV8SCPay7rqS7OKovnbamez9OJ\\\\\\\/aQBietJjw+HFuKYRyBwVBxXKOKcQuqz5LLFFgVSJNvK7HbT\\\\\\\/PZ\\\\\\\/5f3LpRMmDzc5LYmKTBPAdkEsnFreeQ3P06BuCrancUz9OQLj9JieA2BnJMoKcDlDf5BxYj18OxUWtmrpFRjEaD6ckbqQBCO0XRLKxAe9N1VsHie46R8c2NVqeFQR\\\\\\\/sBWlH9k\\\\\\\/xtnhO4s1ZpqiUcUMF3qV2g6Yh9FNwvY3xkzcK4VadoYL3abfD7pfhEhSj6dWqys2WlWDzV7d5EM8s8xUpZlcVU4gVbhhDJhxX5PMX33ymAeGz2GAsRpjEnj0Yy6XYZuKNAYap7iSnkgz+\\\\\\\/FtUQ+0Xb8iLFlQBFCZJzYb5\\\\\\\/HjB\\\\\\\/Dp16XuOjPg6fZOa\\\\\\\/s6TB6s\\\\\\\/Qp2R2JDV4xrlPZGSgZTKMZyfGtuZEqWtWbTgIBN78eu19Er66QVdJgbk20hep6tMxjIkKIQQ+zZZsey5bfVhBKS36vJHxuJvoGG5m8r1jAw977NuDOaU2rhv3xcpyj2b0eQUccut8MtZkyUQvoueW3bfw1HhT0LcPMa4oIs3AsqSgCXgsPnQVFM8H9edfWx196ObaMWIXnOHRPsqXgd1Ubtg1cHcRBz1N14uWDZUq6+5xDM0T9kVsfze22dndMyS7oeO2F1uyPbzs60iYQNuBrIAoHeo3jjbpsZobObAtZn8vUtKg1xtORVD7hgElG0Wt468IIXiX9PddSJNRhq29IoxUFBXBfLyCf9+FDeAQM5UOgyuLUXCXl8TQYXRyQPDQiwu0yueWCA7IaNEWTDtpHKJ7YaNGVj1QtjPzr51zvlkQ224Z9hI24qEyMbiDmeXgJR2TqWTbbHDWk\\\\\\\/XAyT2folBLJUNSZtgF6wse7gNwTqTXyYn0e5WaDiJaxkWbCR3A8Sl595TOS75GUNJcjt+Nfjw6+LaAzaqI8CqMtOWkY3c2yUWXkpzksyUOJw9ZZVFLsnECXgbSWfqTXHKanxn58QR0z6rqmgwfVO3XfQT3WWzRtsKnpxyQx\\\\\\\/fERwIciZ\\\\\\\/xrzeWvK3FxWpO3FXLXhX35qZL3PjRYyDxn38AoH48\\\\\\\/iomu8NDs+oGwrqfv4vwWQa\\\\\\\/JMk3lEPNcrW2pK0gzQ6SSSVFW+\\\\\\\/v7FvKuWy21lt8r7XjDnjKa2xqtiCHWSuETxMQTw1E9xQ\\\\\\\/cWXSr+Vbtt2ONy\\\\\\\/4wASIY9hQFWb9xBohpbWqyoF6NhmkI+CIKYHBKi51VEsuJDl6mS68+AWnEGDffhndIt8vHcEN5HGeSvcwVSzVZO71C5yuk+mMjhYLdKzbIjIB\\\\\\\/a4aAVK0yCXdldr+zguc7vtpR6zWY3mfZqKFURG\\\\\\\/DGQTbRa998fbOgUk9l0z\\\\\\\/iAzyVwkZaGhaluNI9uX3pgb9ruUEPhdNmxAPvEaGxIQwGXDzDqXNRAG1VdG+d2mjf2GrxJLqfdAYYc6uEpNUuD31jPgGvsOnkj8e7dloyxSKxhnxjaYuYtaFqGex8NqDHCh8a6b0FXlKg9TSylELcoOCkI1\\\\\\\/9BFS17KJhUm3ejmyPUJlz602x6sLPu3SDsFmCGgmcp+GK2XE0h9RAzHsnb0X\\\\\\\/VarYTAWkcxqE7OUp3STwl+mdiBRDHRa7aaMI+z+4hhn4GBQJiKyqfhDj5lM5rI16HVlAwhaBoNO9e\\\\\\\/7tj1uUiS7yk3TFKMQpbo0IwDeIKPCg5asETZEUX9i4vp17hxAlz+tavk+Qo7D4JORrS682+sXrai8iR4fL61u4dfGNAp8yJtHgXyPSkVukcltuOumj3pfPjJeMZex8aBbVytY9RqxYLzKDFESvA3g5KRTNLrs8qbPiX31lfG4X\\\\\\\/XRueFx\\\\\\\/AgcqnxkKr9ToXR6B0bcv8AqEzUYmcrf16mJJWyFdP+8OHbMDWIn8\\\\\\\/gnbjw\\\\\\\/hy7Kj+e8Z5ioIygzPv10uDv\\\\\\\/2cTwvyAjqCUDi4WDIDD6ROSmlAbv+s5AVqIeAQaCpgD3xP1lHUjaUdqFRBxu9nFNLTz9N2jWiMqQDPh\\\\\\\/BV3EGTCL9++mmQ9+JQaf4XRHaT7mY6P3LXO+O54iwynNybdmNI+3xP2wM7LREcZdFKkjociBMVQASkjwqZ97VRWKZUIi+dPVogJcbXl4arWN54BaCRqxbxx2SNx1eoPjG46AXuh2FOddQixQd43R\\\\\\\/VDeCHCfKlJRHB7jSlYvD7Q919hJmW0\\\\\\\/wtEnJmugABh6rhbOywwx5pbqZlGGkUdk\\\\\\\/HNZYhKa82il9FVN9UWRV9zL2H5kNLLRkit1RrBHb3OEv2YjC\\\\\\\/4cD0QUtJlXEC0yr1Xuvxs4cFGZc13Q6auYgXQMfQSvn3JMCF\\\\\\\/EPW+WKUUrAeZKWK4l6k2STr9\\\\\\\/oWWydzmEY6dEL9ytBC+2S92PXxVqfPBRTyGzG1EvyfRLB6wXetsYg6ShVkeOcrYty3JKFh+iR+Uf1LvaWU3UxRepGqtVsirI\\\\\\\/59DH459mT2ctA5aJRNIWJNlhJdOHUjWgZyZ\\\\\\\/GiLfBw0qP0bL9YHRx0XwFqEwK74ZRt3HST4uJltpY46hnLgikUApOowLdeS4RT\\\\\\\/9FFdJCZMGBYdecuERyD7LWyLPsv6QnC+uuwUb+6M6yXRbzwmol5jM79m3qGskbSdx5dNokAMU9ihTiOSSCSNst4atyQlS1a0t+pMAkXgl9I72laAp6toGG9ozpw0yrkdSC1bwb\\\\\\\/FDkHov\\\\\\\/nXUS8KELPq4YC9M5uO13ysfj2AX6oHZ\\\\\\\/SZyeRyMbgcsUpFo8CaqSH9TwewxLMCwe4R8iDIZd8m4mD\\\\\\\/Msv9UE+FJWVd2\\\\\\\/M\\\\\\\/WusX7tTzqfar5EwFL6Bvn8RmC896AtvsabXX0718TpJmi\\\\\\\/q9JV9KQSa+KPgkZ1pxk8Ysjhtoy7f3wcSxNmnmq6atPDwqB3m5vuxK5FL48WBFvhRXxlRbonxI8X9YGSkMEwW5Ie37KCJlDFMgfKxOz5fhPY+m0ZhYWo2ZuPG\\\\\\\/lk9KgksABOTG4cKRa1qNLPvaktEmMGlZcYSGcK8WSPf2ruKn7zc1Fycd8WaHTI6rn6vHrE55DPSDLStLuOgzt\\\\\\\/qaU0WIpV+5fymQrN5U\\\\\\\/uoSW4MsaB0PJ5BxGbIUqDYMX5+e9a64AvLTH2KZj0U89PGxJZdazlQSgY5o8Z07Yg5KOYoPd5jemXt9GWdKjMuuHMCs9leZ2vBdGt2acwIDTz7JdSh9na\\\\\\\/34j7Gg6LzH5kSqfP297ubEa5cecsBJ9cfPUbXIazJHc3UBm3qssgGhpO8U7tKwBE4VVXdjtndxvv8KCVi0mnGRWfjtxO9lJIJT80q8\\\\\\\/cH8ukOSGbmuDDP6mrOnTq0h\\\\\\\/3u8KhN6jYOZLw8pWF4TG5YVL3MTnSLnbTV54WgdZgU86wseoSEAyw9TGE+kMPJaen5HQB2lI5kOiWzt4dLj1FDZrg20AkrxUH3ktpPpntzXux9UrUqWi\\\\\\\/x3mKQ607O\\\\\\\/FDW1IvegerILjJNNSyS\\\\\\\/oqxgHMDYZHMW3jjmTRzA+cWV0wfvlhZfTE4PS9PE9zcTzo\\\\\\\/ss2X3kp6iEP7Ea5\\\\\\\/1pf0lLlbsq1gIePeYQPY3JQwgyHTiNFk6leAJA\\\\\\\/k55TIHfzFG6QPPMLK1O74UHCJCYRqlel\\\\\\\/ZF5WZJofhr+lKGkTov8qjgmOm\\\\\\\/503U+H\\\\\\\/kGwWZRUw3niWW\\\\\\\/luw4byAyywpG7PB4UkSRPHNJSUxmnY2uEot1vUhiiZKfE4+ZWSxg0n8ZxvsPeHJh\\\\\\\/hkNrVwU3KFfQTo2DQtV2p8rMwcVihDXgdOW3LAISDzyd+8FWOq8XKZZxpT\\\\\\\/1bibOKvZo4De9iTPzNzZ91JeNYrzC+FDplrImUhgDhvxuk7ITx53wn4d\\\\\\\/1znbAo3Do0KvQUa38K7WeRMWPEDGaxtfesRo86cxMCe8AHw7tIouATrLGrzSDNn4Q7\\\\\\\/FhNXD\\\\\\\/s6R\\\\\\\/Z9n7pWdaQuwwbjKQ6umMZWlVZsZsdr5y6ccFyUU+5GvHthtrntc\\\\\\\/+c5h9kD22UthOc9yYIeiz2ddsW08j0UXG\\\\\\\/wU8wwKr+o\\\\\\\/o4X0s2ky1MCAv2up5JRrXKziNIOgaNIqikzNW1j1CUfMp6jfutHMERgZZlGVgMK76WokA+pZWJKExwAi8f98ni\\\\\\\/hKU3WdP3qVdHh\\\\\\\/ZtxNyMGVCijOVcfFOT\\\\\\\/pWuAfi\\\\\\\/fyx9lbc4uuMkbYrVKMCyNqvYAgG1PK1fal\\\\\\\/m0Z9EEDHvQfOQ+VV7\\\\\\\/MKHGANY\\\\\\\/FfeZHs0FvJdspUdmESH32x7UcWIBGX4ZNfBxr3yYwLNpPVPOHRWZT9lOgTOFiY3ZyOoD1RL4qSrfawaW2ODvXe7e07qjS0bScF2fmeDKqhkb6PeLXXmEJqd9zj3uN1YOcvxANxUfC3YT+mwoJ3k7lx6zbEYSGvYOQ6z7e7romo\\\\\\\/mEpF46DRqSQtMBKiSK2cQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lYmBOrwLfXMbax\\\\\\\/ln0umXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022daf85f357c3f3e9549fe09d327ca7b2999601f9d860abe81c52e605bbe9abe3b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XR\\\\\\\/cl5vAMRn\\\\\\\/LbYHF4j+7dPub3Ti4B5ORtFBMKNSjTBfIdsE4PV+wvjybZM\\\\\\\/mjnbNcu1vcuoUEWp4cZ+bPeWsbdsPITvvgxLL2CYEu7M1khwoXbNRor4HgAdLR1IZTONsseJ6PP3fVxzmIS6L3i0aL+VlknQhfCVU6iUYFfqbVUbuBajZTcfCQA4wnluuEeVtSqt6ZM+aYv4GQAWOMMdbQ5Dbd1XcxcfBQX92gdwvmS0kr1F9iA7XCd1ENJQfUUC5zSuyroIBudTBJ1iOw5KiUQpY0bVEWaYIpNkwaBX6UzQlASQIdpVt29RO3eZj4q\\\\\\\/CCSDe9yjbMnm8p3TpSRhE9jnpLQUtpiqxPH1ytT8pm0Dn2rlI4uhXM4Oyu7tmptqCqdAvTnirzT4WDPNHXUnD35UWgPtlC1vpzdpk+48JsN4izB+6lDpE+h6FNww7zODxXzmYocSk5lobvcZprjYEO\\\\\\\/9KnLmFWs7p9K68Q5XUc5H141Zti1IA2HOESfwyVeNnhvLG+a373Kf\\\\\\\/uuo7QHg8NZ6T4QAhc6SFKhHz2mGfuqZ2gqqgYzgp+XbE5dAbvOmrPLNLS5lh5Kpgkh9Vw8eT1lFtCUrlzODb9wQ\\\\\\\/b\\\\\\\/fY57eoyjAsKfWHwZvkVYvzqOoPh+J8EJkNRCPBYUAuY0kqdoQsGpW4WUDGEwJW7U17rUnfZ+2C59AKnnbQodBs9LM6jZezBCiTCjiCWPfClhrf1rF42Dktu4sXq1Sm4YSCGiln3UNtPLZDL98DM4xj7gRJXXGfpszAC3r3MnM8RvfctmKuXATktdH61wFiCEN+uLi3rQYNBHq+myGi2+y51Sfxnx2LLbZvqNyk9lGOMYOTsl3H89UE8fzV0DarJ5NpvyoMVKF60GaZzWIB2H\\\\\\\/tYV67kPr4Bo2AebNRwOW7PDQBYV4tLUODxwXNyx3EYbDKLUx7k+WL4+9yTBBctplJ4MQhINQ00YnDHoxWMgnL9XvCaOjxrfzpxp9MgMJQ6jpJegMBU+vF0KLi1YubdO0Ahzusw\\\\\\\/uj2WgZtCKLnne9njn14Qph+TMSS03AtBYeejGLSQzOdeikc3t5KbpQVzERU2PUA3eKPjelkV\\\\\\\/pP06s75wYBvymCrvy7EgJxsyY4uuB\\\\\\\/8Ipg6F+nSY7+U0e8dW5BYrzpCjPXct9+5Utk+ekxf6b00Fo2KzDlYstJS\\\\\\\/OlRISEo5kcmdAYkRWDaW6ZUqC\\\\\\\/WdMkGmbP6LyfKDjLOG1OkAqFcSC4OpJZjPfxFhCjZ\\\\\\\/unpRwH2SMlLo3GKDifOZvsRmaRgmdB4LTwFMWOIRsu\\\\\\\/+uX3IfHoFpBclTnhrFKsAGA1xu4kl\\\\\\\/NnEa\\\\\\\/gnkL1ACQnlH\\\\\\\/0wHpgkjMgEXEbK3rWNIUNzMgs8K3Kuz3yIfYcpq34Bwls+1BuP1nFQdvrETE3Ou60b0PjordaDTgP5hRn9XUICT1wUMhJ9VUqG7I9Qi3m7ubxS+xZROru6EGmHotJFuRrhx9nER+tVlk3u1gYsCKM\\\\\\\/ZpEHNlvZhsw79RmU5moapVF+m\\\\\\\/mqepoYfR6CznC73PDiPNQOeuLxU6lKIKB30GFGl1iDGij6m9DMME9OY6QTjwnfAE58AAMWPsW\\\\\\\/3hokcgYZCbxh3zseCY\\\\\\\/+rzCi2sGJTFWAFjWVJBHToe2YkP8mFd2NYQm9bcnGsvfUMhyKZe8WxONKz1CNGbAS3X6MVdN17yjO8x81jCFuqeAZhM8mQ4wz\\\\\\\/rUAiR1HCgf7MFLa5Xgk+x6SGjapknSypcnGvyIOrdljz2PzFwmmHkGCWDErjTp7RkFNZ066X2NzGLJQrxL8R0F6B3UnbSDbVBc+RYPxEFolXq9OSWzvgi7orjBJmduHhiChQ\\\\\\\/oWu\\\\\\\/AFQGa0u6Nftiy7RxVSyrIZ5QDTNcIepJLQPW3AxnJAq5IIYmybGpziIHHHrAS+G\\\\\\\/+6jS9MOUmjuKgUenLehmtca0sngBBZm7\\\\\\\/hTiTGGhf9Bk3L96+szlE4qjNvQIxgHJQtv9vLyTNqo+O0nU2TiUhV9GGklg==\\\u0022,\\\u0022iv\\\u0022:\\\u00224I4VBC2fZzOvacCZ\\\\\\\/15ToA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022673d9222f8412c986a4747894aec9893a29a00802ed608165a13eff1fbc6e730\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459428 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/451" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FPyLw24v8NxRtLzZq_dS2DSOW8qt7nw5eSPO9OvlS1Y" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/451" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qVZ9CaR6hZPNf0F1fEHP68olho3s9cfRUP2NuxJ3HDg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/441" title="The Genetic Architecture of Degenerin/Epithelial Sodium Channels in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/465" title="Unequal Recombination and Evolution of the Mating-Type (MAT) Loci in the Pathogenic Fungus Grosmannia clavigera and Relatives" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459428" id="node459428" data-pisa="ggg;3/3/451" data-pisa-master="ggg;3/3/451" data-apath="/ggg/3/3/451.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Beverly A. Baptiste</span>, <span class="highwire-citation-author" data-delta="1">Guruprasad Ananda</span>, <span class="highwire-citation-author" data-delta="2">Noelle Strubczewski</span>, <span class="highwire-citation-author" data-delta="3">Andrew Lutzkanin</span>, <span class="highwire-citation-author" data-delta="4">Su Jen Khoo</span>, <span class="highwire-citation-author" data-delta="5">Abhinaya Srikanth</span>, <span class="highwire-citation-author" data-delta="6">Nari Kim</span>, <span class="highwire-citation-author" data-delta="7">Kateryna D. Makova</span>, <span class="highwire-citation-author" data-delta="8">Maria M. Krasilnikova</span> and <span class="highwire-citation-author" data-delta="9">Kristin A. Eckert</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">451-463;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005173 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459428" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beverly A. Baptiste </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Intercollege Graduate Program in Genetics, Huck Institutes of Life Sciences, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beverly%2BA.%2BBaptiste%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baptiste%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeverly%2BA.%2BBaptiste%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guruprasad Ananda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Program in Bioinformatics and Genomics, Huck Institutes of Life Sciences, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guruprasad%2BAnanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ananda%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuruprasad%2BAnanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Noelle Strubczewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noelle%2BStrubczewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strubczewski%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoelle%2BStrubczewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrew Lutzkanin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BLutzkanin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lutzkanin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BLutzkanin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Su Jen Khoo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Su%2BJen%2BKhoo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khoo%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASu%2BJen%2BKhoo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abhinaya Srikanth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhinaya%2BSrikanth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srikanth%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhinaya%2BSrikanth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nari Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nari%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANari%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kateryna D. Makova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology, Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kateryna%2BD.%2BMakova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Makova%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKateryna%2BD.%2BMakova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Maria M. Krasilnikova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BM.%2BKrasilnikova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krasilnikova%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BM.%2BKrasilnikova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:muk19@psu.edu" class="" data-icon-position="" data-hide-link-title="0">muk19@psu.edu</a>
<a href="mailto:kae4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">kae4@psu.edu</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kristin A. Eckert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristin%2BA.%2BEckert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eckert%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristin%2BA.%2BEckert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:muk19@psu.edu" class="" data-icon-position="" data-hide-link-title="0">muk19@psu.edu</a>
<a href="mailto:kae4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">kae4@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/3/451" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459428" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459428/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/451.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459428" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459428/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/451.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459428" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459428/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/451.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459428" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459428/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/3/451.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459428" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459428/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Dinucleotide microsatellites are dynamic DNA sequences that affect genome stability. Here, we focused on mature microsatellites, defined as pure repeats of lengths above the threshold and unlikely to mutate below it in a single mutational event. We investigated the prevalence and mutational behavior of these sequences by using human genome sequence data, human cells in culture, and purified DNA polymerases. Mature dinucleotides (≥10 units) are present within exonic sequences of &gt;350 genes, resulting in vulnerability to cellular genetic integrity. Mature dinucleotide mutagenesis was examined experimentally using <em>ex vivo</em> and <em>in vitro</em> approaches. We observe an expansion bias for dinucleotide microsatellites up to 20 units in length in somatic human cells, in agreement with previous computational analyses of germ-line biases. Using purified DNA polymerases and human cell lines deficient for mismatch repair (MMR), we show that the expansion bias is caused by functional MMR and is not due to DNA polymerase error biases. Specifically, we observe that the MutSα and MutLα complexes protect against expansion mutations. Our data support a model wherein different MMR complexes shift the balance of mutations toward deletion or expansion. Finally, we show that replication fork progression is stalled within long dinucleotides, suggesting that mutational mechanisms within long repeats may be distinct from shorter lengths, depending on the biochemistry of fork resolution. Our work combines computational and experimental approaches to explain the complex mutational behavior of dinucleotide microsatellites in humans.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/short-tandem-repeats" class="hw-term hw-article-keyword hw-article-keyword-short-tandem-repeats" rel="nofollow">short tandem repeats</a></li><li class="kwd"><a href="/keyword/mismatch-repair" class="hw-term hw-article-keyword hw-article-keyword-mismatch-repair" rel="nofollow">mismatch repair</a></li><li class="kwd"><a href="/keyword/microsatellite-instability" class="hw-term hw-article-keyword hw-article-keyword-microsatellite-instability" rel="nofollow">microsatellite instability</a></li><li class="kwd"><a href="/keyword/strand-slippage" class="hw-term hw-article-keyword hw-article-keyword-strand-slippage" rel="nofollow">strand slippage</a></li></ul><p id="p-4">Microsatellites, short tandem repeat DNA sequences with base unit sizes ranging from 1 to 6 base pairs, are abundant in both intragenic (introns, exons, untranslated regions) and intergenic regions of the human genome (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Li <em>et al.</em> 2002</a>). Approximately 17% of human genes contain microsatellite repeats within open reading frames (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gemayel <em>et al.</em> 2010</a>), and intragenic microsatellites can play a prominent role in regulating gene expression and protein function (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Li <em>et al.</em> 2004</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gemayel <em>et al.</em> 2010</a>). In this study, we focus on the mutational behavior of dinucleotide microsatellites. Allele-length polymorphisms at specific dinucleotide microsatellite loci are implicated as genetic risk factors in a number of diseases. For example, the length of a polymorphic [GT/CA] allele within intron one of the EGFR gene is inversely correlated with transcription (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gebhardt <em>et al.</em> 1999</a>), and EGFR expression is increased in breast tumors with [GT/CA]<sub>15</sub> alleles, relative to tumors with [GT/CA]<sub>18</sub> alleles (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Buerger <em>et al.</em> 2000</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2004</a>). Length changes of a [GT/CA] allele in the eNOS gene affect splicing regulation and as a result are associated with the risk of coronary artery disease (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stangl <em>et al.</em> 2000</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hui <em>et al.</em> 2005</a>). Mutation of dinucleotides within exons is expected to directly affect protein sequence and potentially also function; of importance, instability of exonic dinucleotides within 14 cancer-associated genes was detected in tumors of head and neck squamous cell carcinoma patients (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2012</a>).</p><p id="p-5">A defining characteristic of microsatellites is their dynamic mutational behavior and high level of germline polymorphism among individuals (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ellegren 2004</a>). Based on changes in mutational behavior, we have defined the threshold length at which a short tandem repeat becomes a microsatellite (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kelkar <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ananda <em>et al.</em> 2013</a>). The threshold length for dinucleotides defined in this manner is five units (10 bp). The major factors influencing microsatellite mutability are specific to the microsatellite itself; these intrinsic factors include motif size, motif composition, and the overall number of units in the microsatellite (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kelkar <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Eckert and Hile 2009</a>). Recently, we demonstrated that distinct cellular mechanisms might contribute to dinucleotide microsatellite mutability at different repeat length ranges before and after the threshold length (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ananda <em>et al.</em> 2013</a>). In a previous comparative genomics study of microsatellite mutability, we observed distinct phases of mutability as a function of allele length (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kelkar <em>et al.</em> 2008</a>). Together, these observations argue for unique mutation mechanisms within microsatellites of lengths above the threshold.</p><p id="p-6">Genome-wide studies have identified directional biases in the mutational behavior of long microsatellites. Early studies of human germline mutations at dinucleotide microsatellites indicated that expansions outnumber contractions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ellegren 2000</a>). Subsequent studies, based on larger data sets, also demonstrated an expansion bias, with a contraction bias seen only for very long alleles (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Huang <em>et al.</em> 2002</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sun <em>et al.</em> 2012</a>). These latter studies are also consistent with computational modeling interrogating human dinucleotide microsatellites using their genomic distributions (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Calabrese and Durrett 2003</a>) or human-chimpanzee interspecific comparisons (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sainudiin <em>et al.</em> 2004</a>). A recent study found a pattern of an expansion and contraction biases for tetranucleotide alleles (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Sun <em>et al.</em> 2012</a>) that is similar to the one observed for dinucleotide microsatellites. Thus, the directional biases that exist for in microsatellites in the human genome seem to depend on their repeat number (length). Computational models have been derived that extend the stepwise mutation model to assume higher mutation rates at long microsatellites (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bell and Jurka 1997</a>), to allow different rates of expansions and deletions depending on length (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Whittaker <em>et al.</em> 2003</a>), to impose an upper limit on allele sizes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Feldman <em>et al.</em> 1997</a>), to incorporate occasional mutations involving a large number of repeated units (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Di Rienzo <em>et al.</em> 1994</a>), or to integrate slippage and point mutations depending on microsatellite allele length (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kruglyak <em>et al.</em> 1998</a>). Clearly, understanding the mutational mechanisms operating within long microsatellite alleles is necessary to correctly model the evolution of these sequences.</p><p id="p-7">Several potential mechanisms may underlie mutational biases within microsatellites, including DNA polymerase errors during synthesis and postreplication mismatch repair (MMR) [reviewed in (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Eckert and Hile 2009</a>)]. MMR proteins act in multiple DNA metabolic pathways to modulate mutagenesis: the canonical pathway, which removes base-base mispairs and insertion/deletion loops (IDLs) generated during DNA synthesis; homologous recombination and double-strand break repair pathways; and DNA damage signaling pathways (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lazzaro <em>et al.</em> 2009</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Pena-Diaz and Jiricny 2012</a>). With the use of mouse models, a complex effect of MMR on both the germline and somatic mutability of very long, disease-associated trinucleotide microsatellites has been demonstrated (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">McMurray 2010</a>). In these studies, MMR proteins differentially affect expansion and contraction (deletion) mutations in a manner that is dependent on both the identity of the specific MMR protein and the sequence of the microsatellite (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">van den Broek <em>et al.</em> 2002</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dragileva <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bourn <em>et al.</em> 2012</a>).</p><p id="p-8">Replication fork stalling and/or the perturbation of DNA repair synthesis caused by non-B DNA structures formed within microsatellites also is correlated with expansion of disease-related microsatellites in several model systems [reviewed in (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pearson <em>et al.</em> 2005</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Mirkin and Mirkin 2007</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wells 2007</a>)]. Replication fork stalling has been demonstrated primarily at expanded trinucleotide microsatellites (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Samadashwily <em>et al.</em> 1997</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Pelletier <em>et al.</em> 2003</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Krasilnikova and Mirkin 2004b</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Voineagu <em>et al.</em> 2009</a>). Replication perturbations have been proposed to generate repeat expansions in several models, which include replication fork stalling followed by fork reversal (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sogo <em>et al.</em> 2002</a>) and fork stalling followed by template strand switching events (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shishkin <em>et al.</em> 2009</a>). The extent to which dinucleotide microsatellite repeats lead to replication fork stalling has not been previously investigated.</p><p id="p-9">The goals of this study were several fold. First, we sought to understand the biological significance of long dinucleotide microsatellites by analyzing their locations within the human genome and identifying the genes harboring such microsatellites within exonic sequences. Second, we quantified the types of mutational biases present in human cells using direct experimental approaches. Third, we tested possible mechanisms influencing the mutational behavior of mature dinucleotide microsatellites by using both <em>ex vivo</em> and <em>in vitro</em> assays. Fourth, we examined whether replication fork progression is affected by the presence of long dinucleotide microsatellite alleles. We find that mature microsatellites up to 20 units, present in numerous protein-coding regions, display a mutational bias toward expansion that is lost in the absence of mismatch repair. Additionally, long dinucleotide repeats do stall replication fork progression, which may affect mutational mechanisms. Our results have uncovered unexpected mechanistic parallels between dinucleotide microsatellites commonly found within the human genome and rare, expanded trinucleotide microsatellite alleles.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Gene Ontology (GO) enrichment analysis</h3><p id="p-10">We obtained the set of dinucleotide microsatellites from the reference human genome (hg19) by using custom scripts [see (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Ananda <em>et al.</em> 2013</a>) for details]. Using Galaxy (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Giardine <em>et al.</em> 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blankenberg <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Goecks <em>et al.</em> 2010</a>), we intersected these microsatellites with a list of exons obtained from the UCSC Genome Browser (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kent <em>et al.</em> 2002</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Karolchik <em>et al.</em> 2008</a>). The list of exonic microsatellites was then combined with HUGO gene annotations (HUGO Gene Nomenclature Committee at the European Bioinformatics Institute) to map each exonic microsatellite to the associated gene. Next, using functions in the R package ‘GOstats’ (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Falcon and Gentleman 2007</a>), we investigated whether the mature microsatellite-containing genes were enriched for specific GO functional annotations (in comparison to the genes in the remainder of the genome). Specifically, we used hyperGTest function (with a <em>P</em>-value cut-off of 0.01, and ontology specified as “molecular function”), which uses a hypergeometric test to compare the two gene sets (mature microsatellite-containing genes <em>vs.</em> all other genes in the genome) and determines an over/underrepresentation of GO “molecular function” annotations in a selected gene set (mature microsatellite-containing genes in our case).</p></div><div id="sec-3" class="subsection"><h3>Reagents</h3><p id="p-11">Antibiotics and 5-fluoro-2’-deoxyuridine (FUdR) were purchased from Sigma-Aldrich Co. (St. Louis, MO). Fetal bovine serum was purchased from Hyclone Laboratories, Inc. (Logan, UT) and gentamycin was purchased from Mediatech, Inc. (Manassas, VA). Recombinant DNA polymerase β (pol β) was purified as described (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Opresko <em>et al.</em> 1998</a>).</p></div><div id="sec-4" class="subsection"><h3>Cell lines</h3><p id="p-12">LCL721 cells are an Epstein-Barr virus (EBV)-transformed cell line derived from the B lymphocytes of a clinically normal female donor (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kavathas <em>et al.</em> 1980</a>). LCL1261 cells are an EBV-transformed cell line derived from the B lymphocytes of a patient with Turcot syndrome and are PMS2 deficient (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Parsons <em>et al.</em> 1995</a>). Cells were cultured in buffered RPMI 1640 supplemented with 10% (LCL721) or 15% (LCL1261) fetal bovine serum and 50 mg/mL gentamycin. The MMR protein expression profile of LCL721 and LCL1261 cell lines has been previously reported (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Shah and Eckert 2009</a>). HCT116 cells were derived from a human colorectal carcinoma and are deficient in MLH1 and MSH3 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bennett <em>et al.</em> 1997</a>). The HCT116+chr3 cell line has been complemented with an additional chromosome 3 to restore MLH1 gene function (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Koi <em>et al.</em> 1994</a>). Both HCT116 cell lines were cultured in buffered Dulbecco’s modified Eagle medium + F12 supplemented with 10% fetal bovine serum and 50 mg/mL gentamicin. Culture medium for HCT116+chr3 cells also contained 400 µg/mL Geneticin. The presence or absence of MMR proteins in the HCT116/HCT116+chr3 cell lines was confirmed by immunoblot analyses (data not shown). Replication analysis of plasmids was performed in two mammalian cell lines: 293A (purchased from Invitrogen/Life Technologies, Grand Island, NY) and COS-1 cells (purchased from Sigma-Aldrich, St. Louis, MO). COS-1 cells and 293A cells were grown in Dulbecco’s modified Eagle medium supplemented with 10% newborn calf serum (COS-1) or fetal bovine serum (293A).</p></div><div id="sec-5" class="subsection"><h3>Vector construction</h3><p id="p-13">The herpes simplex virus thymidine kinase type 1 (HSV-<em>tk</em>) gene-containing vector, pSStu1, is a derivative of the pGem3Zf (-) phagemid and has been previously described (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2002a</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hile and Eckert 2008</a>). The oriP-tk shuttle vector (pJY1) contains the HSV-<em>tk</em> gene and the oriP replication origin sequence from EBV (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hile <em>et al.</em> 2000</a>). The psGSV-<em>tk</em> shuttle vector contains the HSV-<em>tk</em> gene and the replication origin sequence from Simian Virus 40 (SV40 ori). psGSV-<em>tk</em> was constructed by amplifying the SV40 ori from pEGFP-N1 and cloning the amplified product into the <em>Xba</em>I site of pGTK4, followed by <em>Bam</em>HI digestion and religation to remove the chloramphenicol resistance marker. HSV-<em>tk</em> gene cassettes containing microsatellite alleles were constructed by inserting tandem repeats in-frame between bases 111 and 112 of the target HSV-<em>tk</em> gene, in the sequence context GT^TCTC, as described previously (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2002a</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">b</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Kelkar <em>et al.</em> 2010</a>) HSV-<em>tk</em> gene cassettes containing various microsatellites were subcloned from the pSStu-based vector into pJY1 and psGSV-tk shuttle vectors. Several subclones were isolated and re-analyzed to confirm wild-type HSV-<em>tk</em> function and DNA sequence. For microsatellite replication stalling assays, oligonucleotides containing [AT/TA]<sub>n</sub>, [TC/AG]<sub>n</sub>, [GT/CA]<sub>n</sub>, and [GC/CG]<sub>n</sub> repeats were first cloned and elongated in pYES plasmid, as described (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Krasilnikova and Mirkin 2004b</a>). Plasmid pUCneoH was obtained by inactivating the existing <em>Hin</em>dIII site of pUCneo, and insertion of oligonucleotides containing <em>Hin</em>dIII at the blunt-ended AatII site. Microsatellite-containing pUCneoH plasmids were obtained by inserting the corresponding repeat-containing <em>Hin</em>dIII fragments of pYES into the <em>Hin</em>dIII site of pUCneoH.</p></div><div id="sec-6" class="subsection"><h3>OriP-<em>tk</em> shuttle vector assay</h3><p id="p-14">Shuttle vector constructs containing the various microsatellite alleles were introduced into LCL populations by electroporation, and cells containing shuttle vector DNA were selected by the presence of 300 μg/mL (LCL721) or 100 μg/mL hygromycin (LCL1261) for 5−7 d, and maintained with 150 μg/mL and 50 μg/mL, respectively, as described (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Eckert <em>et al.</em> 2002b</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Shah and Eckert 2009</a>). Briefly, selected cell populations were cloned by serial dilution, and individual clones were expanded to a population size of ~2−3 × 10<sup>8</sup> cells. An alkaline extraction method was used to purify shuttle vector DNA, which was used to electroporate <em>Escherichia coli</em> strain FT334. To select for HSV-<em>tk</em> mutant plasmids, bacteria were plated in the presence of 50 μg/mL chloramphenicol (Cm) and the absence or presence of 40 μM FUdR. FUdR selects for bacteria harboring a plasmid with any mutation that inactivates the HSV-<em>tk</em> gene. The HSV-<em>tk</em> mutant frequency is defined as the number of FUdR<sup>R</sup> + Cm<sup>R</sup> colonies divided by the total number Cm<sup>R</sup> colonies. The mutation rate was estimated by dividing the observed HSV-<em>tk</em> mutant frequency by number of cell population doublings between the time of cloning and shuttle vector DNA extraction, as described (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Eckert <em>et al.</em> 2002b</a>), for each human cell clone. To derive specific microsatellite mutation rates, the DNA sequence changes of 20−30 independent FUdR<sup>R</sup> Cm<sup>R</sup> mutants were determined from several human clones. The proportion of mutants arising within the microsatellite region (not within the HSV-<em>tk</em> gene coding region) was multiplied by the estimated mutation rate for each clone to calculate the specific mutation rate of the microsatellite. Statistical differences in the mutation rates observed among various shuttle vector sequences were analyzed using nonparametric tests and mutation rates derived for at least three human cell clones per vector. Statistical analyses of mutational biases were performed by pooling microsatellite mutations within a given vector observed among all cell clones, followed by Fisher exact test (two-sided).</p></div><div id="sec-7" class="subsection"><h3>SV40-tk shuttle vector assay</h3><p id="p-15">For each template, ten 75-cm<sup>2</sup> plates were transfected with 4 µg each of psGSV-tk and pPVU-o, a plasmid containing SV40 T Antigen. Cells were grown for 3 d before harvest. Cell pellets were washed with phosphate-buffered saline and then frozen at −80°. Plasmids were extracted from the cells using QIAGEN Plasmid Minikit (Valencia, CA). Dpn1 digestion was performed to isolate plasmids that have undergone complete replicated in human cells. For mutational analyses, <em>Eco</em>RV and <em>Mlu</em>I digestion was performed to isolate the HSV-<em>tk</em> mutational target sequence and the resulting fragment was hybridized to gapped DNA heteroduplex molecules, as described (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eckert <em>et al.</em> 1997</a>). Successful gap hybridization was verified by gel electrophoresis, and the resulting hybrid molecules were electroporated into <em>E. coli</em> FT334 followed by selective plating for mutational analyses, as described previously. The HSV-<em>tk</em> mutant frequency was determined after two independent transfections for each cell line. Independent mutants were isolated and sequenced to derive the mutational specificity.</p></div><div id="sec-8" class="subsection"><h3><em>In vitro</em> DNA polymerase assay</h3><p id="p-16">Linear DNA fragments and single-stranded DNA (ssDNA) were prepared from pSStu vectors and used to construct gapped duplex molecules for each construct, as described (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2002a</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Hile and Eckert 2008</a>). The <em>in vitro</em> reactions contained 1 pmol of primed ssDNA template at 40 nM concentration and 10 pmol of pol β. Two independent polymerase reactions were performed for each tandem repeat-containing template, as described (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2002a</a>). The polymerase mutant frequency for each strand of the microsatellite (<em>e.g.</em>, GT <em>vs.</em> CA) was determined separately from two independent polymerase reactions per strand. To control for pre-existing mutations present within the DNA synthesis template, we determined the HSV-<em>tk</em> mutation frequency for each ssDNA by electroporation of FT334, followed by selective plating on media containing 250 µg/mL carbenicillin, with or without FUdR (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2002a</a>). For each template strand, the polymerase error frequency was calculated by subtracting the ssDNA background mutation frequencies from the observed pol β HSV-<em>tk</em> mutation frequencies. To determine the polymerase error frequency (Pol EF) within each microsatellite target region, a mutational spectrum of 20−30 mutants was generated for each template, using mutants isolated from two independent polymerase reactions per template. The Pol EF for a specific microsatellite allele was calculated from the proportion of the microsatellite insertion/deletion (indel) mutants (among the total sequenced), multiplied by Pol EF for each template. The Pol EF for each microsatellite allele (<em>e.g.</em>, GT/CA<sub>10</sub>) was estimated by adding the Pol EF for each strand (<em>e.g.</em>, GT<sub>10</sub> + CA<sub>10</sub>). Statistical differences in the observed mutational specificities among the microsatellite alleles were analyzed using the χ<sup>2</sup> or Fisher exact test and the numbers of mutants in each class (summed for both strands).</p></div><div id="sec-9" class="subsection"><h3>Replication pausing assays</h3><p id="p-17">Two different assays were performed to analyze the first and subsequent replication cycles of plasmids. For the analysis of the first replication cycle, 293A cells were transfected with plasmids using Lipofectin (Invitrogen) according to the manufacturer’s instructions and lysed after 6 hr. To analyze the replication of plasmids in the subsequent replication rounds, COS-1 cells were transfected using Turbofect (Fermentas, Glen Burnie, MD) according to manufacturer’s protocol. The cells were grown for 30 hr before isolation of replication intermediates. Isolation of replication intermediates from mammalian cells and their analysis via two-dimensional neutral/neutral gel electrophoresis were performed as described (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Krasilnikova and Mirkin 2004a</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Voineagu <em>et al.</em> 2009</a>).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-18">The focus of this study is mature dinucleotide microsatellites in the human genome, which are expected to be at increased risk for mutation. Here, we define mature microsatellites as pure tandem repeats, longer than the microsatellite threshold, that are not expected to contract to lengths at or below the threshold in a single mutational event. We and others have described the microsatellite lifecycle in three phases: birth, when a locus acquires the necessary numbers of repeats to attain the threshold length; adulthood, a dynamic phase when a locus is above the threshold for mutagenesis; and death, when the locus repeat length dips below the threshold (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amos and Rubinstzein 1996</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Buschiazzo and Gemmell 2006</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Kelkar <em>et al.</em> 2010</a>). Adult microsatellites can be further divided into “young” and “mature” based on the ability of the microsatellite to degrade below the threshold in one replication cycle (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). For dinucleotides, the threshold was determined to be five units (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Kelkar <em>et al.</em> 2010</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Ananda <em>et al.</em> 2013</a>). We empirically defined mature dinucleotides as repeats 10 units or greater in length, as we and others have not observed mutational events greater than four unit deletions in previous studies of dinucleotide microsatellites (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Kelkar <em>et al.</em> 2010</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Ananda <em>et al.</em> 2013</a>). Therefore, dinucleotides of 10 units are not expected to contract to lengths below the threshold.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/451/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dinucleotide microsatellites in the human genome. (A) Definition of mature microsatellites. (B) Distribution of genic mature dinucleotide microsatellites in the human genome (hg19) as a function of motif sequence. Solid black line, GT/CA motifs; gray line, TC/AG motifs; dashed line, AT/TA motifs. Only one mature length exonic GC/CG motif (11 units) was identified (not shown)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1019279126" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dinucleotide microsatellites in the human genome. (A) Definition of mature microsatellites. (B) Distribution of genic mature dinucleotide microsatellites in the human genome (hg19) as a function of motif sequence. Solid black line, GT/CA motifs; gray line, TC/AG motifs; dashed line, AT/TA motifs. Only one mature length exonic GC/CG motif (11 units) was identified (not shown).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/451/F1.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/451/F1.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/451/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/451/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462545" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-19" class="first-child">Dinucleotide microsatellites in the human genome. (A) Definition of mature microsatellites. (B) Distribution of genic mature dinucleotide microsatellites in the human genome (hg19) as a function of motif sequence. Solid black line, GT/CA motifs; gray line, TC/AG motifs; dashed line, AT/TA motifs. Only one mature length exonic GC/CG motif (11 units) was identified (not shown).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h3>Mature dinucleotide microsatellites in the human genome</h3><p id="p-20">We computed the number of mature length dinucleotide alleles in the reference human genome (hg19) as a function of a region’s gene annotations. There are 83,840 dinucleotide loci with ≥10 repeats (mature microsatellites), of which 35,654 are present within genes. Of these genic dinucleotide loci, 34,831 (97.7%) are intronic and 823 (2.4%) are exonic. Among mature dinucleotide microsatellites located in exons, the predominant motif is [GT/CA]<sub>n</sub> (636 alleles), followed by [TC/AG]<sub>n</sub> and [AT/TA]<sub>n</sub> (with 95 and 91 loci, respectively). Only one mature-length exonic [GC/CG] locus is present in the human genome (of 11 units). Given that the human polymorphism incidence measured for dinucleotides of 10 units in length was ~40% (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Ananda <em>et al.</em> 2013</a>), we expect that mature microsatellites will be highly polymorphic in human populations, when not under selective pressure. Unfortunately, as shown in our recent study (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Ananda <em>et al.</em> 2013</a>), we cannot reliably measure human polymorphism rates at dinucleotides above 10 repeat units (or 20 bp) from resequencing projects, due to limitations imposed by the short read lengths of current datasets.</p><p id="p-21">Dinucleotide loci as long as 37 units (74 bp) in length are present within exons (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Mature dinucleotides are present in 450 exons within the human genome and correspond to 385 genes (as annotated by HUGO; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1/005173SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1/TableS1.xls">Table S1</a>). We examined whether these 385 genes are significantly enriched for specific functions categories by comparing GO terms for this set <em>vs.</em> these for genes in the remainder of the genome (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We found that genes with mature length, exonic dinucleotide microsatellites are significantly enriched (<em>P</em> ≤ 0.01) for several GO functional terms, including transcription factor activity and sequence-specific DNA binding; ion binding; various channel activities; protein domain specific binding; and GTPase regulator activity (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462566/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462566/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">GO molecular functions significantly overrepresented in dinucleotide microsatellite containing genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Our computational analyses have defined a set of genes that are potentially at high risk for mutational inactivation due to microsatellite length variation. In the following sections, we analyze mature dinucleotide microsatellite mutagenesis in human cells to elucidate the mechanisms underlying their mutability.</p></div><div id="sec-12" class="subsection"><h3>Mutation rate and directional biases in immortalized human cells</h3><p id="p-27">We previously reported the use of HSV-<em>tk</em> gene cassettes containing in-frame insertions of defined microsatellite sequences to study the relationships between DNA sequence and microsatellite mutagenesis in mitotic human cells (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hile <em>et al.</em> 2000</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Eckert <em>et al.</em> 2002b</a>). Forward mutational analyses allow the quantitation of mutation rates in the artificial microsatellites after vector isolation and genetic selection in <em>E. coli</em>. Our established <em>ex vivo</em> assay (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) uses an <em>ori</em>P-based episomal shuttle vector system to study microsatellite mutagenesis during stable DNA replication in EBV-transformed cell lines. Using the <em>ori</em>P system, we directly measured mutation rates of mature microsatellites, and tested whether mutational biases exist among dinucleotide microsatellites of varying length and sequence composition. The mutation rate of a series of [GT/CA]<sub>n</sub> or [TC/AG]<sub>n</sub> microsatellite alleles ranging in length from 10 to 20 units was measured after stable replication in the non-tumorigenic LCL-721 cell line. (We are unable to analyze mutagenesis at [AT/TA]<sub>n</sub> motifs of mature length using our experimental system due to high background mutation frequencies, so this motif was not analyzed in this assay.) The individual microsatellite (MS) mutation rates for each clonal population, determined after DNA sequence analyses of independent mutants are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1/TableS2.pdf">Table S2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/451/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of the HSV-tk experimental system. (A) HSV-tk mutational target. Short tandem repeat (STR) sequences were inserted in-frame between bases 111 and 112 of the HSV-tk mutational target to create artificial microsatellites. Inactivating mutations can arise within the MS sequence, as well as within either the entire HSV-tk promoter and gene sequence (ex vivo assay) or an ~100 base-pair region (MluI&#x2212;StuI) of the HSV-tk gene (in vitro assay). (B) Illustrations of the mutagenesis approach. Top line: Ex vivo oriP-tk shuttle vector assay. MS-containing HSV-tk gene cassettes were cloned into the EBV-oriP derived pJY1 shuttle vector (Hile et al. 2000). The location of the MS sequences is indicated by an inverted triangle. Human lymphoblastoid cell lines (LCL) were electroporated with shuttle vector, and vector-bearing populations were selected using hygromycin. Clonal subpopulations were isolated by limiting dilution, and expanded ~20&#x2212;40 cell generations. Episomal DNA was extracted and shuttle vector purified. Middle line: SV40-tk shuttle vector assay. MS-containing HSV-tk gene cassettes were cloned into the SV40 ori, psGSV shuttle vector. HCT116 and HCT116+chr3 cells were transfected with shuttle vector and pPVU-o (contains SV40 T-antigen). After 3 d, plasmids were harvested from cells. Fragments containing replicated mutational targets were digested with restriction endonucleases and hybridized to gapped heteroduplex molecules. Bottom line: In vitro DNA polymerase assay. Gapped heteroduplex molecules were created by hybridizing the MluI-StuI large fragment from the pRStu vector to ssDNA derived from pSStu vectors. Gel-purified gapped substrates were used as templates for DNA synthesis reactions containing purified human DNA polymerases. In all three assays, product DNAs (purified oriP-tk shuttle vectors or gap-filled heteroduplexes) were introduced into E. coli (upp, tdk) for mutational analyses. Cm selects for bacteria bearing the shuttle vector, or bacteria derived from the heteroduplex CmR strand; FUdR selects for HSV-tk-deficient bacteria. DNA sequence changes of independent FUdRR mutants are determined to derive a mutational spectrum for each MS vector." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1019279126" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic of the HSV-&lt;em&gt;tk&lt;/em&gt; experimental system. (A) HSV-&lt;em&gt;tk&lt;/em&gt; mutational target. Short tandem repeat (STR) sequences were inserted in-frame between bases 111 and 112 of the HSV-&lt;em&gt;tk&lt;/em&gt; mutational target to create artificial microsatellites. Inactivating mutations can arise within the MS sequence, as well as within either the entire HSV-&lt;em&gt;tk&lt;/em&gt; promoter and gene sequence (&lt;em&gt;ex vivo&lt;/em&gt; assay) or an ~100 base-pair region (&lt;em&gt;Mlu&lt;/em&gt;I&#x2212;&lt;em&gt;Stu&lt;/em&gt;I) of the HSV-&lt;em&gt;tk&lt;/em&gt; gene (&lt;em&gt;in vitro&lt;/em&gt; assay). (B) Illustrations of the mutagenesis approach. Top line: &lt;em&gt;Ex vivo&lt;/em&gt; oriP-&lt;em&gt;tk&lt;/em&gt; shuttle vector assay. MS-containing HSV-&lt;em&gt;tk&lt;/em&gt; gene cassettes were cloned into the EBV-oriP derived pJY1 shuttle vector (Hile &lt;em&gt;et al.&lt;/em&gt; 2000). The location of the MS sequences is indicated by an inverted triangle. Human lymphoblastoid cell lines (LCL) were electroporated with shuttle vector, and vector-bearing populations were selected using hygromycin. Clonal subpopulations were isolated by limiting dilution, and expanded ~20&#x2212;40 cell generations. Episomal DNA was extracted and shuttle vector purified. Middle line: SV40-tk shuttle vector assay. MS-containing HSV-&lt;em&gt;tk&lt;/em&gt; gene cassettes were cloned into the SV40 ori, psGSV shuttle vector. HCT116 and HCT116+chr3 cells were transfected with shuttle vector and pPVU-o (contains SV40 T-antigen). After 3 d, plasmids were harvested from cells. Fragments containing replicated mutational targets were digested with restriction endonucleases and hybridized to gapped heteroduplex molecules. Bottom line: &lt;em&gt;In vitro&lt;/em&gt; DNA polymerase assay. Gapped heteroduplex molecules were created by hybridizing the &lt;em&gt;MluI&lt;/em&gt;-&lt;em&gt;Stu&lt;/em&gt;I large fragment from the pRStu vector to ssDNA derived from pSStu vectors. Gel-purified gapped substrates were used as templates for DNA synthesis reactions containing purified human DNA polymerases. In all three assays, product DNAs (purified oriP-tk shuttle vectors or gap-filled heteroduplexes) were introduced into &lt;em&gt;E. coli&lt;/em&gt; (upp, tdk) for mutational analyses. Cm selects for bacteria bearing the shuttle vector, or bacteria derived from the heteroduplex Cm&lt;sup&gt;R&lt;/sup&gt; strand; FUdR selects for HSV-&lt;em&gt;tk&lt;/em&gt;-deficient bacteria. DNA sequence changes of independent FUdR&lt;sup&gt;R&lt;/sup&gt; mutants are determined to derive a mutational spectrum for each MS vector.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/451/F2.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/451/F2.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/451/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/451/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462608" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-28" class="first-child">Schematic of the HSV-<em>tk</em> experimental system. (A) HSV-<em>tk</em> mutational target. Short tandem repeat (STR) sequences were inserted in-frame between bases 111 and 112 of the HSV-<em>tk</em> mutational target to create artificial microsatellites. Inactivating mutations can arise within the MS sequence, as well as within either the entire HSV-<em>tk</em> promoter and gene sequence (<em>ex vivo</em> assay) or an ~100 base-pair region (<em>Mlu</em>I−<em>Stu</em>I) of the HSV-<em>tk</em> gene (<em>in vitro</em> assay). (B) Illustrations of the mutagenesis approach. Top line: <em>Ex vivo</em> oriP-<em>tk</em> shuttle vector assay. MS-containing HSV-<em>tk</em> gene cassettes were cloned into the EBV-oriP derived pJY1 shuttle vector (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Hile <em>et al.</em> 2000</a>). The location of the MS sequences is indicated by an inverted triangle. Human lymphoblastoid cell lines (LCL) were electroporated with shuttle vector, and vector-bearing populations were selected using hygromycin. Clonal subpopulations were isolated by limiting dilution, and expanded ~20−40 cell generations. Episomal DNA was extracted and shuttle vector purified. Middle line: SV40-tk shuttle vector assay. MS-containing HSV-<em>tk</em> gene cassettes were cloned into the SV40 ori, psGSV shuttle vector. HCT116 and HCT116+chr3 cells were transfected with shuttle vector and pPVU-o (contains SV40 T-antigen). After 3 d, plasmids were harvested from cells. Fragments containing replicated mutational targets were digested with restriction endonucleases and hybridized to gapped heteroduplex molecules. Bottom line: <em>In vitro</em> DNA polymerase assay. Gapped heteroduplex molecules were created by hybridizing the <em>MluI</em>-<em>Stu</em>I large fragment from the pRStu vector to ssDNA derived from pSStu vectors. Gel-purified gapped substrates were used as templates for DNA synthesis reactions containing purified human DNA polymerases. In all three assays, product DNAs (purified oriP-tk shuttle vectors or gap-filled heteroduplexes) were introduced into <em>E. coli</em> (upp, tdk) for mutational analyses. Cm selects for bacteria bearing the shuttle vector, or bacteria derived from the heteroduplex Cm<sup>R</sup> strand; FUdR selects for HSV-<em>tk</em>-deficient bacteria. DNA sequence changes of independent FUdR<sup>R</sup> mutants are determined to derive a mutational spectrum for each MS vector.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">As expected from previous studies (<a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Kelkar <em>et al.</em> 2010</a>), we observed an increase in MS mutation rate with increasing allele length (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For the [GT/CA]<sub>n</sub> series, as the length of the allele increased ~2-fold, from 10 to 19 units, the median MS mutation rate increased ~30-fold (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), a difference that is statistically significant (<em>P</em> = 0.0002, Kruskal-Wallis test). In comparison, the MS mutation rate of the [TC/AG]<sub>n</sub> series increased sevenfold between 11 and 20 units (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), a difference that also is statistically significant (<em>P</em> = 0.0052, Kruskal-Wallis test). Interestingly, the [TC/AG]<sub>n</sub> MS mutation rates did not change substantially over a span of five [TC/AG] units: 3.3 × 10<sup>−6</sup>, 2.3 × 10<sup>−6</sup>, and 4.1 × 10<sup>−6</sup> for lengths of 11, 14, and 17 units respectively. In contrast, the median MS mutation rates of the [GT/CA]<sub>n</sub> alleles increased progressively, with a 3-fold change from 10 to 13 units (2.1 × 10<sup>−7</sup> and 6.9 × 10<sup>−7</sup>, respectively) and an additional 5-fold change to 26 × 10<sup>−7</sup> at 16 units. However, when similar repeat numbers are compared, the [TC/AG]<sub>n</sub> alleles are usually more mutable than the [GT/CA]<sub>n</sub> alleles. For example, the rate for a [TC/AG]<sub>11</sub> allele is 16-fold greater than that for the [GT/CA]<sub>10</sub> allele (<em>P</em> = 0.038, Mann-Whitney test), whereas the rate for a [TC/AG]<sub>20</sub> allele is ~4-fold greater than that of a [GT/CA]<sub>19</sub> allele (<em>P</em> = 0.016, Mann-Whitney test). Given the slope of the [GT/CA]<sub>n</sub> <em>vs.</em> unit number median mutation rate curve (not shown, derived from data in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>), the lower mutability of the [GT/CA] alleles than the [TC/AG] alleles cannot be accounted for by the one unit differences in total allele lengths.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/451/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Microsatellite mutation rates and directional biases observed in nontumorigenic, mismatch repair-proficient human lymphoblastoid cells. The oriP-tk shuttle vector assay was performed using LCL-721 cells. (A) MS mutation rate (mutation frequency per cell generation) as a function of GT/CA units. Data are box plots for the number of clones indicated. (B) MS mutation rate as a function of TC/AG units. Data are box plots for the number of clones indicated. (C) Mutational biases within GT/CA alleles as a function of length. Total number of MS mutants observed among all clones is shown in parentheses for each allele. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units (D) Mutational biases within TC/AG alleles as a function of length. Data for the [TC/AG]17 allele are taken from reference (Eckert et al. 2002b). Black bars, expansions of 1 or more units; gray bars, deletion of one or more units." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1019279126" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Microsatellite mutation rates and directional biases observed in nontumorigenic, mismatch repair-proficient human lymphoblastoid cells. The oriP-tk shuttle vector assay was performed using LCL-721 cells. (A) MS mutation rate (mutation frequency per cell generation) as a function of GT/CA units. Data are box plots for the number of clones indicated. (B) MS mutation rate as a function of TC/AG units. Data are box plots for the number of clones indicated. (C) Mutational biases within GT/CA alleles as a function of length. Total number of MS mutants observed among all clones is shown in parentheses for each allele. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units (D) Mutational biases within TC/AG alleles as a function of length. Data for the [TC/AG]&lt;sub&gt;17&lt;/sub&gt; allele are taken from reference (Eckert &lt;em&gt;et al.&lt;/em&gt; 2002b). Black bars, expansions of 1 or more units; gray bars, deletion of one or more units.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/451/F3.medium.gif" width="440" height="361"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/451/F3.medium.gif" width="440" height="361"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/451/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/451/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462593" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-30" class="first-child">Microsatellite mutation rates and directional biases observed in nontumorigenic, mismatch repair-proficient human lymphoblastoid cells. The oriP-tk shuttle vector assay was performed using LCL-721 cells. (A) MS mutation rate (mutation frequency per cell generation) as a function of GT/CA units. Data are box plots for the number of clones indicated. (B) MS mutation rate as a function of TC/AG units. Data are box plots for the number of clones indicated. (C) Mutational biases within GT/CA alleles as a function of length. Total number of MS mutants observed among all clones is shown in parentheses for each allele. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units (D) Mutational biases within TC/AG alleles as a function of length. Data for the [TC/AG]<sub>17</sub> allele are taken from reference (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Eckert <em>et al.</em> 2002b</a>). Black bars, expansions of 1 or more units; gray bars, deletion of one or more units.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Computational studies have suggested that there is a directionality bias in germline dinucleotide microsatellite mutations, such that expansions are more likely than deletions (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Amos and Rubinstzein 1996</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Ellegren 2000</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amos 2010</a>). We analyzed our data set for the directionality of mutations occurring at a microsatellite allele, namely unit-based expansions <em>vs.</em> deletions. In somatic human cells, we observed that the proportion of expansion mutations is greater than deletion mutations for both dinucleotide motifs at all except the shortest alleles tested. For the [GT/CA]<sub>n</sub> series, no expansion mutations were observed at the shortest length examined (10 units), whereas greater than 50% of the MS mutations were expansions within the longer alleles (13−19 units; <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). For the [TC/AG]<sub>n</sub> series, ~40% of the microsatellite mutations were expansions within the shortest allele (11 units; <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>). Moreover, a statistically significant increase in the proportion of expansion mutations was observed as the allele length increased (14−20 units), relative to the [TC/AG]<sub>11</sub> allele (<em>P</em> &lt; 0.0001 to 0.006, Fisher’s exact test). A full 80% of the microsatellite mutations arising within the [TC/AG]<sub>20</sub> allele were expansions (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>).</p><p id="p-32">To summarize, we observed two types of mutation biases in mature microsatellite alleles using our somatic cell mutation assay: greater mutation rates for [TC/AG] than [GT/CA] alleles and a bias toward expansion <em>vs.</em> deletion errors in alleles 13 to 20 units in length. To examine the mechanisms underlying these biases, we investigated the roles of DNA polymerase slippage errors and MMR.</p></div><div id="sec-13" class="subsection"><h3>DNA polymerase error specificity within dinucleotide microsatellites</h3><p id="p-33">We used our published <em>in vitro</em> DNA polymerase mutagenesis assay (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2002a</a>) to investigate the sources of the sequence composition and directionality biases observed above. Previously, we reported that DNA pol β error frequency for insertion/deletion (Indel) errors within complementary [GT] and [CA] repeats increased ~30-fold from 4 units to 13 units (<a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Kelkar <em>et al.</em> 2010</a>). Here, we extended our studies to include [GT/CA]<sub>16</sub> and [GT/CA]<sub>19</sub> templates, as well as [TC/AG] templates of 8 to 14 units in length. (Longer [TC/AG] alleles could not be analyzed using the <em>in vitro</em> assay due to the high background mutation frequency of the single-stranded DNA preparations.) The relationship between allele length and pol β error rate for unit-based (two nucleotide) Indels is exponential between 10 units and 19 units in length for the [GT/CA] allele (R<sup>2</sup> = 0.97) and between 8 units and 14 units in length for the [TC/AG] allele (R<sup>2</sup> = 0.99; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). For the alleles tested, the pol β error rates within the [TC/AG] alleles are higher than those within the [GT/CA] alleles of similar length (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>), similar to the <em>ex vivo</em> assay results.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/451/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pol &#x3B2; error rates and specificity within dinucleotide alleles. (A) MS indel error rates as a function of allele length. Error rates were estimated from two independent reactions per template strand. Error rates for each complementary strand were summed to derive the error rate for each allele length. Solid line, GT/CA alleles; dashed line, TC/AG alleles. Lines are an exponential fit of the data. (B) Mutational biases observed for GT/CA and TC/AG alleles, as a function of length. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units. (C) Size distributions of MS indel deletion errors, as a function of allele length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1019279126" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pol &#x3B2; error rates and specificity within dinucleotide alleles. (A) MS indel error rates as a function of allele length. Error rates were estimated from two independent reactions per template strand. Error rates for each complementary strand were summed to derive the error rate for each allele length. Solid line, GT/CA alleles; dashed line, TC/AG alleles. Lines are an exponential fit of the data. (B) Mutational biases observed for GT/CA and TC/AG alleles, as a function of length. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units. (C) Size distributions of MS indel deletion errors, as a function of allele length.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/451/F4.medium.gif" width="440" height="233"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/451/F4.medium.gif" width="440" height="233"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/451/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/451/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462568" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-34" class="first-child">Pol β error rates and specificity within dinucleotide alleles. (A) MS indel error rates as a function of allele length. Error rates were estimated from two independent reactions per template strand. Error rates for each complementary strand were summed to derive the error rate for each allele length. Solid line, GT/CA alleles; dashed line, TC/AG alleles. Lines are an exponential fit of the data. (B) Mutational biases observed for GT/CA and TC/AG alleles, as a function of length. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units. (C) Size distributions of MS indel deletion errors, as a function of allele length.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">The majority of pol β errors created in both the [GT/CA] and [TC/AG] microsatellite alleles are deletions. Thus, we observed no bias <em>in vitro</em> toward the production of expansion mutations in longer [GT/CA] or [TC/AG] alleles (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>), in contrast to the <em>ex vivo</em> assay. Interestingly, we observe a statistically significant increase in the size of pol β deletion errors as the [GT/CA] alleles increase in length, such that 22–25% of deletions were 5 units or more for the [GT/CA]<sub>16</sub> and [GT/CA]<sub>19</sub> alleles (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>; <em>P</em> = 0.027, χ<sup>2</sup> test, 3 d.o.f.). This type of mutation was not observed or rarely observed among microsatellite errors within the [GT/CA]<sub>10</sub> and [GT/CA]<sub>13</sub> mutational spectra.</p><p id="p-36">We previously reported that the human DNA polymerase δ also creates a high frequency of Indel errors within the [GT]<sub>19</sub> microsatellite (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baptiste and Eckert 2012</a>). The mutational specificity bias toward multi-unit [GT] deletions that we report here for pol β is similar to our previous measurements for polymerases δ and κ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1/FigureS1.pdf">Figure S1</a>). Moreover, the specificity of pol β errors within the [TC]<sub>11</sub> allele (~85% deletions, 15% expansions) is nearly identical to that previously reported for the replicative polymerases, pol α-primase (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hile and Eckert 2004</a>) and pol δ (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hile <em>et al.</em> 2012</a>).</p><p id="p-37">In summary, our <em>in vitro</em> results show that differential DNA polymerase error rates can contribute to the observed bias regarding motif sequence. However, the specificity of errors created by replicative (pol α, pol δ), repair (pol β), or specialized (pol κ) DNA polymerases cannot readily explain the directionality bias toward expansion mutations with increasing allele length that was observed in human cells.</p></div><div id="sec-14" class="subsection"><h3>Role of MMR in generating a mutational bias toward expansion of [GT/CA]<sub>n</sub> alleles</h3><p id="p-38">In the canonical pathway associated with correction of DNA synthesis errors, MMR proteins recognize premutational intermediates in which the newly synthesized DNA strand (nascent DNA) differs in sequence from the parental DNA strand (template DNA). Failure of MMR to repair the intermediates results in mutations after the next round of DNA synthesis. Microsatellite expansion mutations result from premutational intermediates containing IDLs in the nascent DNA strand, whereas microsatellite deletions result from premutational intermediates containing IDLs in the template DNA strand. Human cells have specification in MMR, which is achieved through the combination of different heterodimers of the MutS and MutL components of the MMR machinery (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Pena-Diaz and Jiricny 2012</a>). Human cell lines deficient in one or more MMR proteins have been identified, and we used three such cell lines to examine the role of MMR in generating mature dinucleotide microsatellite mutational bias (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462528/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462528/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of [GT/CA]<sub>19</sub> mutagenesis results from <em>ex vivo</em> and <em>in vitro</em> assays</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">First, we examined mutational specificity in the HCT116 human colon cancer cell line, which carries loss-of-function mutations in both the <em>MLH1</em> and <em>MSH3</em> genes and has been previously shown to be deficient in repair of two-nucleotide loops (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Umar <em>et al.</em> 1994</a>). Because these cells do not express the MLH1 protein, they are deficient in both MutLα and MutLγ repair complexes. Therefore, although HCT116 cells express MSH2 and MSH6 proteins (MutSα), they are functionally MMR-deficient because they lack a functional MutL heterodimeric complex. We expected that the mutational events observed in HCT116 cells would reflect errors generated by DNA polymerases, which we have shown here and previously to be biased toward deletions. Second, we analyzed mutational biases in HCT116 cells complemented with chromosome 3 (HCT116+chr3). This complementation restores MLH1 protein expression, but does not affect expression of the MSH3 protein. HCT116+chr3 cells are MutSα proficient, but MutSβ deficient. This gain-of-function comparison of HCT116+chr3 cells to the HCT116 parental cell line allows us to determine the role that MSH2/MutSα plays in generating mutational bias. Third, we assessed the role of MutLα in mutational bias using the human lymphoblastoid cell line (LCL1261), which does not express the PMS2 protein, a key component of the MutLα heterodimer. This cell line does express proteins of both the MutSα and MutSβ heterodimers, as well as MLH1 (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Shah and Eckert 2009</a>) and MLH3 (data not shown), proteins of the MutLγ complex. A loss-of-function comparison with LCL721 cells allows us to determine the role that PMS2/MutLα plays in generating mutational bias.</p><p id="p-42">To measure microsatellite mutagenesis experimentally in epithelial cells such as HCT116, we modified the <em>ex vivo</em> mutational assay by incorporating the SV40 origin of replication (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Three days after transient transfection of the [GT/CA]<sub>19</sub> construct, plasmids were recovered from HCT116 cells and analyzed for mutation frequency and mutational specificity. As predicted for cells containing no functional MMR complexes, we measured mutant frequencies that are of the same magnitude as those measured in the <em>in vitro</em> DNA polymerase assay (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Of importance, the frequency of expansion mutations generated after replication in MSH3, MLH1-deficient HCT116 cells (4.8 × 10<sup>−3</sup>) is the same as the <em>in vitro</em> frequency of pol β expansion errors in the same allele (5.6 × 10<sup>−3</sup>). In addition, no expansion bias is observed after [GT/CA]<sub>19</sub> replication in HCT116 cells, and the ratio of expansion: deletion mutations is similar to that observed for purified polymerase <em>in vitro</em> (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-43">The mutation frequency of plasmids after replication in HCT116+chr3 cells (5.7 × 10<sup>−3</sup>) was ~2-fold lower than the frequency after replication in HCT116 cells (1.2 × 10<sup>−2</sup>), suggesting that MSH2/MutSα contributes to MMR of replication errors of mature [GT/CA]<sub>n</sub> dinucleotide repeats. The proportion of microsatellite expansions also was lower for HCT116+chr3 cells (15%) than for HCT116 (40%) cells (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Specifically, we observed that the frequency of expansion mutations was 6-fold lower in HCT116+chr3 cells (8.0 × 10<sup>−4</sup>) than in HCT116 cells (4.8 × 10<sup>−3</sup>) (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Thus, the presence of MSH2/MutSα suppresses expansion mutations.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/451/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GT/CA microsatellite mutation rates and directional biases observed in mismatch repair-deficient human cells. (A) Mutational biases of GT/CA19 microsatellite replication in HCT116+chr3 and HCT116 cells. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units. (B) Mutation rate as a function of allele length determined after oriP-tk shuttle vector replication in PMS2-deficient human lymphoblastoid cells. Data are box plots for the number of clones shown in parentheses. (C) Mutational biases within GT/CA alleles as a function of length. Total number of MS mutants observed among all clones is shown in parentheses for each allele. Solid bars, expansions of 1 or more units; stippled bars, deletion of one or more units." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1019279126" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;GT/CA microsatellite mutation rates and directional biases observed in mismatch repair-deficient human cells. (A) Mutational biases of GT/CA&lt;sub&gt;19&lt;/sub&gt; microsatellite replication in HCT116+chr3 and HCT116 cells. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units. (B) Mutation rate as a function of allele length determined after oriP-tk shuttle vector replication in PMS2-deficient human lymphoblastoid cells. Data are box plots for the number of clones shown in parentheses. (C) Mutational biases within GT/CA alleles as a function of length. Total number of MS mutants observed among all clones is shown in parentheses for each allele. Solid bars, expansions of 1 or more units; stippled bars, deletion of one or more units.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/451/F5.medium.gif" width="440" height="362"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/451/F5.medium.gif" width="440" height="362"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/451/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/451/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462524" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-44" class="first-child">GT/CA microsatellite mutation rates and directional biases observed in mismatch repair-deficient human cells. (A) Mutational biases of GT/CA<sub>19</sub> microsatellite replication in HCT116+chr3 and HCT116 cells. Black bars, expansions of 1 or more units; gray bars, deletion of one or more units. (B) Mutation rate as a function of allele length determined after oriP-tk shuttle vector replication in PMS2-deficient human lymphoblastoid cells. Data are box plots for the number of clones shown in parentheses. (C) Mutational biases within GT/CA alleles as a function of length. Total number of MS mutants observed among all clones is shown in parentheses for each allele. Solid bars, expansions of 1 or more units; stippled bars, deletion of one or more units.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">We previously demonstrated that MMR mediated by PMS2 (MutLα) also is biased toward repair of expansion mutations within tetranucleotide microsatellites (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Shah and Eckert 2009</a>). Here, we tested whether a similar result would be observed within dinucleotide microsatellites. Using the assay <em>ori</em>P<em>-tk</em> shuttle vector assay, [GT/CA]<sub>n</sub>–containing shuttle vectors were stably replicated in LCL1261 cells. As expected, the observed MS mutation rates were 200- to 1000-fold greater than those measured for MMR-proficient cells (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1/TableS3.pdf">Table S3</a>). Across all allele lengths examined, we measured a statistically significant 180-fold increase in the median MS mutation rates for the [GT/CA] motif (<em>P</em> &lt; 0.0001, Kruskal-Wallis test). Analysis of mutational spectra revealed a striking expansion bias in LCL1261 cells, wherein 91–100% of the microsatellite mutations observed within each [GT/CA] length were expansions (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). These results demonstrate that the presence of PMS2 suppresses expansion mutations within dinucleotide alleles. The absolute frequency of expansion mutations in PMS2-deficient LCL1261 cells is similar to that measured in MSH3,MLH1-deficient HCT116 cells (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Similar to loss of MutSα (comparison of HCT116 + chr3 with HCT116), loss of MutLα (comparison of LCL721 to LCL1261) resulted in an increased frequency and proportion of expansions. Taken together, our data suggest that functional MMR generates directional biases within dinucleotide microsatellites and identifies the MutSα and MutLα complexes as protecting against expansion mutations.</p></div><div id="sec-15" class="subsection"><h3>DNA replication pausing within dinucleotide microsatellites</h3><p id="p-46">In our next analysis, we considered whether mutational processes in addition to polymerase strand slippage errors and MMR potentially act during DNA replication to produce expansion mutations. Replication fork stalling and template switching mechanisms are well known to correlate with the production of very large trinucleotide repeat expansions (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wells <em>et al.</em> 2005</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Mirkin and Mirkin 2007</a>). We used an established experimental system (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chandok <em>et al.</em> 2012</a>) to analyze whether DNA sequence composition and length affects replication fork progression through mature dinucleotide alleles. To quantitate DNA replication inhibition, [GC/CG]<sub>n</sub>, [AT/TA]<sub>n</sub>, [GT/CA]<sub>n</sub>, and [TC/AG]<sub>n</sub> alleles of varying lengths were cloned into vectors containing a defined SV40 origin of replication, and DNA replication intermediates isolated from primate Cos-1 cells were analyzed by 2D gel electrophoresis. We observed replication fork stalling at all types of dinucleotide microsatellites, detected as bulges on replication arcs that were not present for the control plasmid (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). The position of stalling corresponded to the position of the microsatellite within the plasmid, based on the distance traveled in the first direction of electrophoresis. The strength of the stalling was quantitated by measuring the amount of radioactivity in the bulge, relative to the intensity of the arc (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Pelletier <em>et al.</em> 2003</a>). Stalling intensity was clearly motif sequence and length dependent (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>), and a dependence of the stall on motif orientation relative to the replication origin was not observed for any of the repeats (data not shown). At an equivalent number of units, the rank-order for pausing was: [GC/CG] &gt; [AT/TA] &gt; [GT/CA] = [TC/AG] (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). We also analyzed dinucleotide microsatellite replication in 293A cells, prior to assembly of a regular chromatin structure. Because 293A cells lack expression of SV40 large T antigen, the plasmid is replicated transiently using an alternative replication mode that initiates randomly throughout the plasmid (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chandok <em>et al.</em> 2011</a>). In 293A cells, the only dinucleotide repeat that caused a significant stalling was [TC/AG]<sub>n</sub> (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>), where stalling is detectable at a length of about 20 repeated units, and increased with the subsequent increase in the repeat length. Surprisingly, the repeats that have high tendency to form hairpin structures such as [AT/TA]<sub>20</sub> and [GC/CG]<sub>9</sub> did not cause stalling in 293 cells (data not shown), although these sequences have a profound effect on SV40-origin dependent replication (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/451/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Replication fork stalling within dinucleotide alleles as a function of allele length. (A) Representative two-dimensional gels after replication in Cos-1 (top panels) or HEK293A (bottom panels) cells. (B) Quantitation of pausing in Cos-1 cells. (C) Quantitation of pausing in 293A cells. (D) Schematic of SV40-origin vector indicating position of the microsatellite sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1019279126" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Replication fork stalling within dinucleotide alleles as a function of allele length. (A) Representative two-dimensional gels after replication in Cos-1 (top panels) or HEK293A (bottom panels) cells. (B) Quantitation of pausing in Cos-1 cells. (C) Quantitation of pausing in 293A cells. (D) Schematic of SV40-origin vector indicating position of the microsatellite sequences.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/451/F6.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/451/F6.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/451/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/451/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462582" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-47" class="first-child">Replication fork stalling within dinucleotide alleles as a function of allele length. (A) Representative two-dimensional gels after replication in Cos-1 (top panels) or HEK293A (bottom panels) cells. (B) Quantitation of pausing in Cos-1 cells. (C) Quantitation of pausing in 293A cells. (D) Schematic of SV40-origin vector indicating position of the microsatellite sequences.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-48">Microsatellites of 1−6 basepairs per unit are abundant in the human genome, and ~90% of known human genes have microsatellites within exons (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Madsen <em>et al.</em> 2008</a>). Exonic microsatellites shorter than 33 bp in length were shown to be overrepresented in disease-related genes, particularly cancer and immune system disorders (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Madsen <em>et al.</em> 2008</a>). Here, we demonstrate that mature dinucleotide microsatellites (20−74 basepairs in length) are present within exons of 385 genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1/TableS1.xls">Table S1</a>) and are significantly enriched in specific classes of genes encoding important biologic activities, such as transcription factor and membrane channel activities (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Furthermore, we measured a significantly elevated rate of mutation within mature dinucleotide alleles in normal human cells (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>). Therefore, our analysis not only identifies genes potentially at high risk for mutational inactivation due to microsatellite length variation in somatic cells, but also demonstrates the potential functional significance of mature microsatellites and hence the need for a thorough investigation of their mutational behavior.</p><p id="p-49">Our study used various experimental approaches to examine the mutational mechanisms operating within mature microsatellites, which we define as alleles that are not expected to mutate to lengths below or at the threshold for microsatellite mutational behavior in one round of replication. We explored the sources of sequence and directional biases in microsatellite mutability, and our results uncovered several novel facets of dinucleotide mutagenesis. First, our data from nontumorigenic human cells indicate that the [GT/CA]<sub>n</sub> alleles are usually less mutable than the [TC/AG]<sub>n</sub> alleles of comparable length (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>), and identify DNA polymerase error rates as the underlying basis of this sequence bias (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). Second, we demonstrate a directionality bias in somatic human cells that favors expansion mutations for mature dinucleotide microsatellites of 13−20 units (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>). This bias cannot readily be explained by DNA polymerase error biases (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). Genome-wide studies have identified germline expansion biases in the mutational behavior of dinucleotide microsatellites (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Ellegren 2000</a>), with a contraction bias seen only for very long alleles (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Huang <em>et al.</em> 2002</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Sun <em>et al.</em> 2012</a>). Possibly, our <em>ex vivo</em> data capture the portion of the dinucleotide mutation curve right before an expansion bias switches to a contraction bias. Comparisons of the exact nucleotide range of the bias switch observed in our study in somatic cell mutations and computational studies of human germline mutations (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Huang <em>et al.</em> 2002</a>; <a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Sun <em>et al.</em> 2012</a>) are challenging, because the latter studies report standardized and not actual allele lengths. However, some computational studies indicate this switch occurs at ~20 dinucleotide repeats, although the length at which the switch occurs may be sequence dependent (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Calabrese and Durrett 2003</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Sainudiin <em>et al.</em> 2004</a>).</p><p id="p-50">Third, we found that MMR is a likely source of directional bias within common microsatellites in the human genome. Strikingly, total absence of cellular MMR (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>) mimics the expansion <em>vs.</em> deletion balance seen in our <em>in vitro</em> results (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). Comparisons of MutSα-deficient/proficient and MutLα-deficient/proficient cell line pairs identified these two repair complexes as protective against expansion mutations (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>). Fourth, we demonstrate that DNA replication fork stalling occurs within mature dinucleotides and is motif and length dependent (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>). Thus, mutational mechanisms within very long dinucleotides may be different than those within shorter alleles (closer to the threshold), depending on the biochemistry of fork resolution/restart.</p><p id="p-51">A constant balance of expansions and contractions of microsatellites may play a role in maintaining their genetic stability over time. Our results show that for dinucleotide alleles in the 13 to 20 repeat number range, expansions occurred more frequently than deletions (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>). The only exceptions seen were the two shortest alleles, [GT/CA]<sub>10</sub> and [TC/AG]<sub>11</sub>, in the MMR-proficient cells. A bias favoring expansions over deletions within [GT/CA] alleles was previously demonstrated using a yeast model system (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Strand <em>et al.</em> 1993</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Johnson <em>et al.</em> 1995</a>). In contrast, we find no directional bias for dinucleotide microsatellite mutations in functionally MMR-deficient HCT116 cells, using a forward assay (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). In fact, the mutational specificity that we observed after replication of the [GT/CA]<sub>19</sub> vector in HCT116 cells is highly similar to our <em>in vitro</em> observations of errors produced by pol β and pol δ at the same microsatellite allele. Our HCT116 cell results using a forward mutation assay differ from the previous conclusions of <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Yamada <em>et al.</em> (2002)</a>. One explanation for the different conclusions of the two studies may be the fact that we compared our HCT116 results with chromosome 3-complemented HCT116 cells as a control for MMR, whereas <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Yamada <em>et al.</em> (2002)</a> used MMR-proficient mouse cells as the control comparison. Our complemented cells still lack MutSβ, whereas the mouse cells are presumably proficient for both MutS complexes. Also, mouse cells may replicate and repair looped intermediates differently than human cells.</p><p id="p-52">Our PMS2-deficient cells lack MutLα (MLH1•PMS2) but retain MutLγ (MLH1•MLH3) expression (<a id="xref-ref-66-5" class="xref-bibr" href="#ref-66">Shah and Eckert 2009</a>), and likely, activity. Strikingly, of the ~200 independent mutants analyzed from LCL1261 cells deficient in MutLα, only 2% displayed deletion events within the [GT/CA] microsatellite. Our estimated deletion frequency for dinucleotides in LCL1261 cells is ≤ 10<sup>−4</sup>, which is lower than the deletion frequency observed in the fully MMR-deficient HCT116 cell line (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a>). Therefore, our data indirectly support a role for the MutLγ MMR complex in suppressing deletion mutations within dinucleotide microsatellites. Mouse studies have shown that although <em>Pms2</em><sup>−/−</sup> (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">van Oers <em>et al.</em> 2010</a>) and <em>Mlh3</em><sup>−/−</sup> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chen <em>et al.</em> 2005</a>) single knockout mutants both display microsatellite instability and tumor susceptibility, only the double knockout (<em>Pms2</em><sup>−/−</sup> <em>Mlh3</em><sup>−/−</sup>) mice are indistinguishable from <em>Mlh1</em><sup>−/−</sup> mice for tumor susceptibility, reduced life span, microsatellite instability, and DNA-damage response (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Chen <em>et al.</em> 2005</a>). Thus, the two MutL homologs are partially redundant, possibly explaining the 40:1 ratio of <em>MLH1</em> to <em>PMS2</em> gene mutations seen in Lynch syndrome tumors (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lynch and de la Chapelle 2003</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Lynch <em>et al.</em> 2009</a>). This redundancy also may help to explain the low prevalence and penetrance of <em>PMS2</em> mutations in MMR-deficient colon cancers (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Lynch and de la Chapelle 2003</a>) and the reported lack of biochemical data to support MLH3 in Lynch syndrome (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ou <em>et al.</em> 2009</a>).</p><p id="p-53">Large expansions of trinucleotide microsatellites are associated with numerous neuromuscular and neurodegenerative disorders, and models involving both DNA repair and replication have been proposed to explain these expansions (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">McMurray 2010</a>). Although our expansion bias within dinucleotide microsatellites of common lengths in somatic human cells differs qualitatively from massive trinucleotide repeat expansions (we observe only a small number of units added per mutational event), unexpectedly our results uncovered mechanistic parallels to disease-associated trinucleotide expansions. Specifically, we demonstrate that (1) the dinucleotide expansion bias is likely caused by MMR functions, and (2) long dinucleotides cause replication fork stalling in a length- and sequence-dependent manner. We have shown here and previously (<a id="xref-ref-66-6" class="xref-bibr" href="#ref-66">Shah and Eckert 2009</a>) that loss of PMS2 in human cells results in a significant bias toward di- and tetranucleotide microsatellite expansion mutations. Recent studies using Friedreich’s ataxia mouse models have demonstrated that PMS2-deficient mice display an increase in expansion mutations within very long [GAA/TCC] microsatellites (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bourn <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ezzatizadeh <em>et al.</em> 2012</a>). Therefore, PMS2 generally limits expansion mutations within di-, tri-, and tetranucleotide microsatellite alleles. Because pre-mutational IDLs for expansions occur on the nascent strand during DNA synthesis, we propose that PMS2/MutLα-mediated MMR plays a vital role in repairing polymerase errors formed on the nascent strand (<a id="xref-ref-66-7" class="xref-bibr" href="#ref-66">Shah and Eckert 2009</a>). This model is not restricted to DNA replication because excision repair, strand break repair, and recombination pathways all require DNA re-synthesis and involve MMR proteins. Indeed, the protective effect of PMS2 toward [GAA/CTT] expansions was suggested recently to occur during the homologous recombination/strand synthesis steps of double strand break repair (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Bourn <em>et al.</em> 2012</a>). Yeast-null mutants of <em>RAD27</em> (<em>RTH1</em>) Flap endonuclease, an enzyme required for lagging strand DNA synthesis and DNA repair, display an increased microsatellite mutation frequency and a mutational bias favoring expansion mutations within [GT/CA] microsatellites. These effects are epistatic with MMR gene defects (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Johnson <em>et al.</em> 1995</a>). MutSβ (Msh2p and Msh3p) has been implicated in promoting trinucleotide repeat expansions in yeast (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kantartzis <em>et al.</em> 2012</a>). MutSβ binds to nicks in the lagging strand before Okazaki fragment processing is complete, allowing the repeated sequence to loop out and bind to downstream homologous sequence. When the flap is processed, the loop remains, resulting in an expansion of one to two repeat units. The mutability of very long [GAA/CTT] alleles also is increased in MSH3-deficient mice, with an increase specifically in deletion (contraction) mutations (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Ezzatizadeh <em>et al.</em> 2012</a>). However, in Huntington’s disease and myotonic dystrophy mouse models, loss of MSH3 decreases somatic mutability of very long [CAG/GTC] and [CTG/GAC] alleles, but has no significant effect on germline mutability or directionality biases (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">van den Broek <em>et al.</em> 2002</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dragileva <em>et al.</em> 2009</a>). Future studies are required to examine the specific role of MSH3 in mutational biases of dinucleotide repeats of varying sequence in human cells.</p><p id="p-54">Replication fork stalling and template switching mechanisms correlate with the production of trinucleotide repeat expansions, although a direct cause and effect relationship has yet to be established (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Wells <em>et al.</em> 2005</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Mirkin and Mirkin 2007</a>). We observed replication fork stalling at all dinucleotide microsatellite sequences in a length-dependent manner, suggesting that replication fork restart/template switching mechanisms may also be operative within these regions of the human genome. Our previously developed system (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Chandok <em>et al.</em> 2012</a>) allowed us to analyze DNA replication fork progression through dinucleotide repeats in two situations: the first replication cycle when the DNA is not completely covered by nucleosomes, and the subsequent replication cycles that occur after a regular chromatin has been established. The various dinucleotide repeat sequences had very different effects on pausing in the first and subsequent replication cycles. Replication fork stalling in the subsequent cycles directly correlated with the hairpin-forming potential of the repeat sequences. In contrast, replication fork stalling in the first cycle was significant only for the [TC/AG] repeat, and at lengths greater than 20 units. Interestingly, we observed a similar replication stalling in the first replication cycle of DNA containing [GAA/CTT] repeats (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Chandok <em>et al.</em> 2012</a>). Both the [TC/AG] and [GAA/CTT] repeats have the propensity to form H-DNA (triplex) structures. The dependence of fork stalling on repeat length may be explained by the instability of H-DNA structures formed at or within shorter repeats, or the unfavorable energetics of H-DNA formation at shorter repeat lengths, due to a higher proportion of unpaired nucleotides (at triplex borders and in the free strand)(<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Frank-Kamenetskii and Mirkin 1995</a>).</p><p id="p-55">In summary, our current study has uncovered several new facets regarding the mechanisms underlying dinucleotide microsatellite stability, sequences that are highly abundant in the human genome. First, we demonstrate that the motif sequence-dependent differences in dinucleotide microsatellite mutation rates can be explained by differences in DNA polymerase error rates. Second, a directionality bias favoring expansion of human genome dinucleotide microsatellites has been inferred in computational studies, and our study demonstrates this bias experimentally in human cells, while providing a mechanism to explain the bias. Based on our studies of replicative, repair and specialized polymerases, it is unlikely that the observed directionality bias favoring microsatellite expansion in cells reflects the inherent error specificity of DNA polymerases. We observed that loss of MutSα or MutLα functions promotes directionality biases toward expansion mutations, while total loss of MMR eliminates the expansion bias entirely. Our data raise the possibility that in MMR-proficient cells, IDLs on the template strand are very efficiently repaired by either the MutSβ or MutLγ complex, resulting in a net bias toward expansion mutations, a scenario that could be tested in future studies. Finally, we demonstrate that replication forks are stalled within dinucleotide microsatellites, implicating fork restart and recombination pathways in the stability of long dinucleotide microsatellites. Interestingly, our mechanistic studies of common dinucleotides uncovered mechanistic parallels with the rare trinucleotides, suggesting a complex involvement of MMR in microsatellite genome stability.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank Amanda Breski for her contributions to mutation rate analyses in human cells. We thank Todd Schell (Penn State Hershey) for his gift of plasmid pPVU-o. The HCT116 and HCT116+chr3 complemented cell lines were a generous gift from Thomas Kunkel, National Institute of Environmental Health Sciences. This work was supported by the National Institutes of Health grant R01GM087472 to K.D.M and K.A.E. (Multiple PI). This project is funded, in part, under two grants (SAP 4100050904 to B.A.B. and K.A.E. and SAP 4100047645 to K.A.E) from the Pennsylvania Department of Health using Tobacco CURE Funds. The Department specifically disclaims responsibility for any analyses, interpretations, or conclusions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005173/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-57"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-10-20"><span class="received-label">Received </span>October 20, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-12-30"><span class="accepted-label">Accepted </span>December 30, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Baptiste <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.1" data-doi="10.1007/s00239-010-9377-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutation biases and mutation rate variation around very short human microsatellites revealed by human-chimpanzee-orangutan genomic sequence alignments</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DAmos%26rft.auinit1%253DW.%26rft.volume%253D71%26rft.issue%253D3%26rft.spage%253D192%26rft.epage%253D201%26rft.atitle%253DMutation%2Bbiases%2Band%2Bmutation%2Brate%2Bvariation%2Baround%2Bvery%2Bshort%2Bhuman%2Bmicrosatellites%2Brevealed%2Bby%2Bhuman-chimpanzee-orangutan%2Bgenomic%2Bsequence%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9377-4%26rft_id%253Dinfo%253Apmid%252F20700734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-010-9377-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20700734&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281397000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.2" data-doi="10.1038/ng0196-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubinstzein</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Microsatellites are subject to directional evolution</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAmos%26rft.auinit1%253DW.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D14%26rft.atitle%253DMicrosatellites%2Bare%2Bsubject%2Bto%2Bdirectional%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0196-13%26rft_id%253Dinfo%253Apmid%252F8528243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0196-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8528243&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TM55500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-other" id="cit-3.3.451.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ananda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Distinct mutational behaviors distinguish simple tandem repeats from microsatellites in the human genome</span>. <span class="cit-source">Gen. Biol. Evol.</span> doi: 10.1093/gbe/evs116.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.4" data-doi="10.1002/em.21721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baptiste</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DNA polymerase kappa microsatellite synthesis: two distinct mechanisms of slippage-mediated errors</span>. <abbr class="cit-jnl-abbrev">Environ. Mol. Mutagen.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Band%2Bmolecular%2Bmutagenesis%26rft.stitle%253DEnviron%2BMol%2BMutagen%26rft.aulast%253DBaptiste%26rft.auinit1%253DB.%2BA.%26rft.volume%253D53%26rft.issue%253D9%26rft.spage%253D787%26rft.epage%253D796%26rft.atitle%253DDNA%2Bpolymerase%2Bkappa%2Bmicrosatellite%2Bsynthesis%253A%2BTwo%2Bdistinct%2Bmechanisms%2Bof%2Bslippage-mediated%2Berrors.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fem.21721%26rft_id%253Dinfo%253Apmid%252F22965905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/em.21721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22965905&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.5" data-doi="10.1007/PL00006161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The length distribution of perfect dimer repetitive DNA is consistent with its evolution by an unbiased single-step mutation process</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">414</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBell%26rft.auinit1%253DG.%2BI.%26rft.volume%253D44%26rft.issue%253D4%26rft.spage%253D414%26rft.epage%253D421%26rft.atitle%253DThe%2Blength%2Bdistribution%2Bof%2Bperfect%2Bdimer%2Brepetitive%2BDNA%2Bis%2Bconsistent%2Bwith%2Bits%2Bevolution%2Bby%2Ban%2Bunbiased%2Bsingle-step%2Bmutation%2Bprocess.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006161%26rft_id%253Dinfo%253Apmid%252F9089081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9089081&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WR61800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshima</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monnat</span>  <span class="cit-name-given-names">R. J.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mismatch repair in extracts of Werner syndrome cell lines</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">2956</span>–<span class="cit-lpage">2960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DBennett%26rft.auinit1%253DS.%2BE.%26rft.volume%253D57%26rft.issue%253D14%26rft.spage%253D2956%26rft.epage%253D2960%26rft.atitle%253DMismatch%2BRepair%2Bin%2BExtracts%2Bof%2BWerner%2BSyndrome%2BCell%2BLines%26rft_id%253Dinfo%253Apmid%252F9230208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI1Ny8xNC8yOTU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Von Kuster</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coraor</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ananda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazarus</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Galaxy: a web-based genome analysis tool for experimentalists</span>. <abbr class="cit-jnl-abbrev">Curr. Protoc. Mol. Biol.</abbr> Chapter 19: Unit <span class="cit-fpage">19.10.1</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.8" data-doi="10.1371/journal.pone.0047085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourn</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Biase</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al-Mahdawi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Pms2 suppresses large expansions of the (GAA· TTC) n sequence in neuronal tissues</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47085</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBourn%26rft.auinit1%253DR.%2BL.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De47085%26rft.epage%253De47085%26rft.atitle%253DPms2%2BSuppresses%2BLarge%2BExpansions%2Bof%2Bthe%2B%2528GAA%25C2%25B7TTC%2529%2528n%2529%2BSequence%2Bin%2BNeuronal%2BTissues.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047085%26rft_id%253Dinfo%253Apmid%252F23071719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071719&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buerger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebhardt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beckmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutmacher</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Length and loss of heterozygosity of an intron 1 polymorphic sequence of egfr is related to cytogenetic alterations and epithelial growth factor receptor expression</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">854</span>–<span class="cit-lpage">857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DBuerger%26rft.auinit1%253DH.%26rft.volume%253D60%26rft.issue%253D4%26rft.spage%253D854%26rft.epage%253D857%26rft.atitle%253DLength%2Band%2BLoss%2Bof%2BHeterozygosity%2Bof%2Ban%2BIntron%2B1%2BPolymorphic%2BSequence%2Bof%2Begfr%2BIs%2BRelated%2Bto%2BCytogenetic%2BAlterations%2Band%2BEpithelial%2BGrowth%2BFactor%2BReceptor%2BExpression%26rft_id%253Dinfo%253Apmid%252F10706093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjg6IjYwLzQvODU0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.10" data-doi="10.1002/path.1542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buerger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Packeisen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boecker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tidow</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kersting</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Allelic length of a CA dinucleotide repeat in the egfr gene correlates with the frequency of amplifications of this sequence–first results of an inter-ethnic breast cancer study</span>. <abbr class="cit-jnl-abbrev">J. Pathol.</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bpathology%26rft.stitle%253DJ%2BPathol%26rft.aulast%253DBuerger%26rft.auinit1%253DH.%26rft.volume%253D203%26rft.issue%253D1%26rft.spage%253D545%26rft.epage%253D550%26rft.atitle%253DAllelic%2Blength%2Bof%2Ba%2BCA%2Bdinucleotide%2Brepeat%2Bin%2Bthe%2Begfr%2Bgene%2Bcorrelates%2Bwith%2Bthe%2Bfrequency%2Bof%2Bamplifications%2Bof%2Bthis%2Bsequence--first%2Bresults%2Bof%2Ban%2Binter-ethnic%2Bbreast%2Bcancer%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpath.1542%26rft_id%253Dinfo%253Apmid%252F15095477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/path.1542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15095477&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220938400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.11" data-doi="10.1002/bies.20470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buschiazzo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemmell</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The rise, fall and renaissance of microsatellites in eukaryotic genomes</span>. <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1040</span>–<span class="cit-lpage">1050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DBuschiazzo%26rft.auinit1%253DE.%26rft.volume%253D28%26rft.issue%253D10%26rft.spage%253D1040%26rft.epage%253D1050%26rft.atitle%253DThe%2Brise%252C%2Bfall%2Band%2Brenaissance%2Bof%2Bmicrosatellites%2Bin%2Beukaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20470%26rft_id%253Dinfo%253Apmid%252F16998838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16998838&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241023500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.12" data-doi="10.1093/molbev/msg084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calabrese</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrett</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dinucleotide repeats in the Drosophila and human genomes have complex, length-dependent mutation processes</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg084%26rft_id%253Dinfo%253Apmid%252F12679531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzUvNzE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.13" data-doi="10.1093/nar/gkq903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandok</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapoor</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brick</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidorova</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A distinct first replication cycle of DNA introduced in mammalian cells</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">2103</span>–<span class="cit-lpage">2115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq903%26rft_id%253Dinfo%253Apmid%252F21062817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM5LzYvMjEwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.14" data-doi="10.1093/nar/gks021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandok</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effects of Friedreich’s ataxia GAA repeats on DNA replication in mammalian cells</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">3964</span>–<span class="cit-lpage">3974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks021%26rft_id%253Dinfo%253Apmid%252F22262734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQwLzkvMzk2NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.15" data-doi="10.1158/0008-5472.CAN-05-0742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dizon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paxton</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Contributions by MutL homologues Mlh3 and Pms2 to DNA mismatch repair and tumor suppression in the mouse</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">8662</span>–<span class="cit-lpage">8670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DChen%26rft.auinit1%253DP.-C.%26rft.volume%253D65%26rft.issue%253D19%26rft.spage%253D8662%26rft.epage%253D8670%26rft.atitle%253DContributions%2Bby%2BMutL%2BHomologues%2BMlh3%2Band%2BPms2%2Bto%2BDNA%2BMismatch%2BRepair%2Band%2BTumor%2BSuppression%2Bin%2Bthe%2BMouse%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-05-0742%26rft_id%253Dinfo%253Apmid%252F16204034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2NS8xOS84NjYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.16" data-doi="10.1073/pnas.91.8.3166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garza</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdes</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Mutational processes of simple-sequence repeat loci in human populations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3166</span>–<span class="cit-lpage">3170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDi%2BRienzo%26rft.auinit1%253DA.%26rft.volume%253D91%26rft.issue%253D8%26rft.spage%253D3166%26rft.epage%253D3170%26rft.atitle%253DMutational%2Bprocesses%2Bof%2Bsimple-sequence%2Brepeat%2Bloci%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.8.3166%26rft_id%253Dinfo%253Apmid%252F8159720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5MS84LzMxNjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.17" data-doi="10.1016/j.nbd.2008.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dragileva</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendricks</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Intergenerational and striatal CAG repeat instability in Huntington’s disease knock-in mice involve different DNA repair genes</span>. <abbr class="cit-jnl-abbrev">Neurobiol. Dis.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurobiology%2Bof%2Bdisease%26rft.stitle%253DNeurobiol%2BDis%26rft.aulast%253DDragileva%26rft.auinit1%253DE.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D47%26rft.atitle%253DIntergenerational%2Band%2Bstriatal%2BCAG%2Brepeat%2Binstability%2Bin%2BHuntington%2527s%2Bdisease%2Bknock-in%2Bmice%2Binvolve%2Bdifferent%2BDNA%2Brepair%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.nbd.2008.09.014%26rft_id%253Dinfo%253Apmid%252F18930147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.nbd.2008.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18930147&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262457900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.18" data-doi="10.1002/mc.20499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Every microsatellite is different: Intrinsic DNA features dictate mutagenesis of common microsatellites present in the human genome</span>. <abbr class="cit-jnl-abbrev">Mol. Carcinog.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcarcinogenesis%26rft.stitle%253DMol%2BCarcinog%26rft.aulast%253DEckert%26rft.auinit1%253DK.%2BA.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D379%26rft.epage%253D388%26rft.atitle%253DEvery%2Bmicrosatellite%2Bis%2Bdifferent%253A%2BIntrinsic%2BDNA%2Bfeatures%2Bdictate%2Bmutagenesis%2Bof%2Bcommon%2Bmicrosatellites%2Bpresent%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fmc.20499%26rft_id%253Dinfo%253Apmid%252F19306292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/mc.20499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19306292&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264918500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.19" data-doi="10.1093/nar/25.7.1450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargo</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Development and use of an <em>in vitro</em> HSV-tk forward mutation assay to study eukaryotic DNA polymerase processing of DNA alkyl lesions</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1450</span>–<span class="cit-lpage">1457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.7.1450%26rft_id%253Dinfo%253Apmid%252F9060443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMTQ1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.20" data-doi="10.1021/bi025918c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mowery</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">a</span> <span class="cit-article-title">Misalignment-mediated DNA polymerase beta mutations: comparison of microsatellite and frame-shift error rates using a forward mutation assay</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">10490</span>–<span class="cit-lpage">10498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D41%26rft.spage%253D10490%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi025918c%26rft_id%253Dinfo%253Apmid%252F12173936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi025918c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12173936&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177435100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.21" data-doi="10.1002/mc.10058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">b</span> <span class="cit-article-title">Mutation Rate and Specificity Analysis of Tetranucleotide Microsatellite DNA Alleles in Somatic Human Cells</span>. <abbr class="cit-jnl-abbrev">Mol. Carcinog.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">140</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcarcinogenesis%26rft.stitle%253DMol%2BCarcinog%26rft.aulast%253DEckert%26rft.auinit1%253DK.%2BA.%26rft.volume%253D34%26rft.issue%253D3%26rft.spage%253D140%26rft.epage%253D150%26rft.atitle%253DMutation%2Brate%2Band%2Bspecificity%2Banalysis%2Bof%2Btetranucleotide%2Bmicrosatellite%2BDNA%2Balleles%2Bin%2Bsomatic%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fmc.10058%26rft_id%253Dinfo%253Apmid%252F12112308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/mc.10058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12112308&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176601400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.22" data-doi="10.1038/74249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Heterogeneous mutation processes in human microsatellite DNA sequences</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">400</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D400%26rft.epage%253D402%26rft.atitle%253DHeterogeneous%2Bmutation%2Bprocesses%2Bin%2Bhuman%2Bmicrosatellite%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F74249%26rft_id%253Dinfo%253Apmid%252F10742106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/74249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10742106&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086192800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.23" data-doi="10.1038/nrg1348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microsatellites: simple sequences with complex evolution</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D435%26rft.epage%253D445%26rft.atitle%253DMicrosatellites%253A%2Bsimple%2Bsequences%2Bwith%2Bcomplex%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1348%26rft_id%253Dinfo%253Apmid%252F15153996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15153996&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221759700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.24" data-doi="10.1016/j.nbd.2012.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ezzatizadeh</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al-Mahdawi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mismatch repair system protects against intergenerational GAA repeat instability in a Friedreich ataxia mouse model</span>. <abbr class="cit-jnl-abbrev">Neurobiol. Dis.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurobiology%2Bof%2Bdisease%26rft.stitle%253DNeurobiol%2BDis%26rft.aulast%253DEzzatizadeh%26rft.auinit1%253DV.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D165%26rft.epage%253D171%26rft.atitle%253DThe%2Bmismatch%2Brepair%2Bsystem%2Bprotects%2Bagainst%2Bintergenerational%2BGAA%2Brepeat%2Binstability%2Bin%2Ba%2BFriedreich%2Bataxia%2Bmouse%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.nbd.2012.01.002%26rft_id%253Dinfo%253Apmid%252F22289650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.nbd.2012.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22289650&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.25" data-doi="10.1093/bioinformatics/btl567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falcon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using GOstats to test gene lists for GO term association</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl567%26rft_id%253Dinfo%253Apmid%252F17098774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMy8yLzI1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollock</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Microsatellite genetic distances with range constraints: analytic description and problems of estimation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFeldman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D145%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D216%26rft.atitle%253DMicrosatellite%2BGenetic%2BDistances%2BWith%2BRange%2BConstraints%253A%2BAnalytic%2BDescription%2Band%2BProblems%2Bof%2BEstimation%26rft_id%253Dinfo%253Apmid%252F9017402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ1LzEvMjA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.27" data-doi="10.1146/annurev.bi.64.070195.000433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frank-Kamenetskii</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Triplex DNA structures</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DKamenetskii%26rft.auinit1%253DM.%2BD.%2BF.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D95%26rft.atitle%253DTriplex%2BDNA%2BStructures%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.bi.64.070195.000433%26rft_id%253Dinfo%253Apmid%252F7574496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.bi.64.070195.000433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7574496&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RH35300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.28" data-doi="10.1074/jbc.274.19.13176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gebhardt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanker</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Modulation of epidermal growth factor receptor gene transcription by a polymorphic dinucleotide repeat in intron 1</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">13176</span>–<span class="cit-lpage">13180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DGebhardt%26rft.auinit1%253DF.%26rft.volume%253D274%26rft.issue%253D19%26rft.spage%253D13176%26rft.epage%253D13180%26rft.atitle%253DModulation%2Bof%2BEpidermal%2BGrowth%2BFactor%2BReceptor%2BGene%2BTranscription%2Bby%2Ba%2BPolymorphic%2BDinucleotide%2BRepeat%2Bin%2BIntron%2B1%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.19.13176%26rft_id%253Dinfo%253Apmid%252F10224073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMTkvMTMxNzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.29" data-doi="10.1146/annurev-genet-072610-155046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gemayel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinces</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variable tandem repeats accelerate evolution of coding and regulatory sequences</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DGemayel%26rft.auinit1%253DR.%26rft.volume%253D44%26rft.spage%253D445%26rft.epage%253D477%26rft.atitle%253DVariable%2Btandem%2Brepeats%2Baccelerate%2Bevolution%2Bof%2Bcoding%2Band%2Bregulatory%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-072610-155046%26rft_id%253Dinfo%253Apmid%252F20809801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-072610-155046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20809801&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286042600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.30" data-doi="10.1101/gr.4086505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giardine</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riemer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardison</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burhans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elnitski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Galaxy: a platform for interactive large-scale genome analysis</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4086505%26rft_id%253Dinfo%253Apmid%252F16169926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.31" data-doi="10.1186/gb-2010-11-8-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goecks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galaxy</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGoecks%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DGalaxy%253A%2Ba%2Bcomprehensive%2Bapproach%2Bfor%2Bsupporting%2Baccessible%252C%2Breproducible%252C%2Band%2Btransparent%2Bcomputational%2Bresearch%2Bin%2Bthe%2Blife%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r86%26rft_id%253Dinfo%253Apmid%252F20738864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20738864&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.32" data-doi="10.1016/j.jmb.2003.10.075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Positive correlation between DNA polymerase Î±-primase pausing and mutagenesis within polypyrimidine/polypurine microsatellite sequences</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHile%26rft.auinit1%253DS.%2BE.%26rft.volume%253D335%26rft.issue%253D3%26rft.spage%253D745%26rft.epage%253D759%26rft.atitle%253DPositive%2Bcorrelation%2Bbetween%2BDNA%2Bpolymerase%2Balpha-primase%2Bpausing%2Band%2Bmutagenesis%2Bwithin%2Bpolypyrimidine%252Fpolypurine%2Bmicrosatellite%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2003.10.075%26rft_id%253Dinfo%253Apmid%252F14687571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2003.10.075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14687571&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188066900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.33" data-doi="10.1093/nar/gkm1089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">DNA polymerase kappa produces interrupted mutations and displays polar pausing within mononucleotide microsatellite sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">688</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm1089%26rft_id%253Dinfo%253Apmid%252F18079151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjM2LzIvNjg4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Somatic mutation rates and specificities at TC/AG and GT/CA microsatellite sequences in nontumorigenic human lymphoblastoid cells</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1698</span>–<span class="cit-lpage">1703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DHile%26rft.auinit1%253DS.%2BE.%26rft.volume%253D60%26rft.issue%253D6%26rft.spage%253D1698%26rft.epage%253D1703%26rft.atitle%253DSomatic%2BMutation%2BRates%2Band%2BSpecificities%2Bat%2BTC%252FAG%2Band%2BGT%252FCA%2BMicrosatellite%2BSequences%2Bin%2BNontumorigenic%2BHuman%2BLymphoblastoid%2BCells%26rft_id%253Dinfo%253Apmid%252F10749142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjk6IjYwLzYvMTY5OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.35" data-doi="10.1093/nar/gkr889"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. Y. W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Beyond translesion synthesis: polymerase Î° fidelity as a potential determinant of microsatellite stability</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1636</span>–<span class="cit-lpage">1647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr889%26rft_id%253Dinfo%253Apmid%252F22021378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQwLzQvMTYzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.36" data-doi="10.1086/338997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Q. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mutation patterns at dinucleotide microsatellite loci in humans</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DQ.%2BY.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D625%26rft.epage%253D634%26rft.atitle%253DMutation%2Bpatterns%2Bat%2Bdinucleotide%2Bmicrosatellite%2Bloci%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F338997%26rft_id%253Dinfo%253Apmid%252F11793300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/338997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11793300&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173880000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-3.3.451.37"><div class="cit-metadata unstructured">HUGO Gene Nomenclature Committee at the European Bioinformatics Institute. Available from: <a href="http://www.genenames.org/">http://www.genenames.org/</a>. Accessed: January 18, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.38" data-doi="10.1038/sj.emboj.7600677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hui</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schreiner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumuller</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Intronic CA-repeat and CA-rich elements: a new class of regulators of mammalian alternative splicing</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1988</span>–<span class="cit-lpage">1998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600677%26rft_id%253Dinfo%253Apmid%252F15889141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTEvMTk4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.39" data-doi="10.1126/science.7618086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovvali</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Requirement of the yeast RTH1 5′to 3′exonuclease for the stability of simple repetitive DNA</span>. <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BNY%2529%26rft.volume%253D269%26rft.spage%253D238%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7618086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.7618086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.40" data-doi="10.1016/j.celrep.2012.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kantartzis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surtees</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Msh2-Msh3 interferes with Okazaki fragment processing to promote trinucleotide repeat expansions</span>. <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D2%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.06.020%26rft_id%253Dinfo%253Apmid%252F22938864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22938864&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.41" data-doi="10.1093/nar/gkm966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karolchik</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baertsch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clawson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The UCSC Genome Browser Database: 2008 update</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D773</span>–<span class="cit-lpage">D779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm966%26rft_id%253Dinfo%253Apmid%252F18086701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNi9zdXBwbF8xL0Q3NzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.42" data-doi="10.1073/pnas.77.7.4251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kavathas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bach</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeMars</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Gamma ray-induced loss of expression of HLA and glyoxalase I alleles in lymphoblastoid cells</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">4251</span>–<span class="cit-lpage">4255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKavathas%26rft.auinit1%253DP.%26rft.volume%253D77%26rft.issue%253D7%26rft.spage%253D4251%26rft.epage%253D4255%26rft.atitle%253DGamma%2Bray-induced%2Bloss%2Bof%2Bexpression%2Bof%2BHLA%2Band%2Bglyoxalase%2BI%2Balleles%2Bin%2Blymphoblastoid%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.7.4251%26rft_id%253Dinfo%253Apmid%252F6933474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ny83LzQyNTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.43" data-doi="10.1101/gr.7113408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelkar</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyekucheva</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiaromonte</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makova</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genome-wide determinants of human and chimpanzee microsatellite evolution</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7113408%26rft_id%253Dinfo%253Apmid%252F18032720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE4LzEvMzAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.44" data-doi="10.1093/gbe/evq046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelkar</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strubczewski</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiaromonte</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">What is a microsatellite: a computational and experimental definition based upon repeat mutational behavior at A/T and GT/AC repeats</span>. <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq046%26rft_id%253Dinfo%253Apmid%252F20668018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjIvMC82MjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.45" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugnet</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furey</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roskin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The human genome browser at UCSC</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauhan</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherian</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carethers</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Human chromosome 3 corrects mismatch repair deficiency and microsatellite instability and reduces N-methyl-N’-nitro-N-nitrosoguanidine tolerance in colon tumor cells with homozygous hMLH1 mutation</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">4308</span>–<span class="cit-lpage">4312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DKoi%26rft.auinit1%253DM.%26rft.volume%253D54%26rft.issue%253D16%26rft.spage%253D4308%26rft.epage%253D4312%26rft.atitle%253DHuman%2BChromosome%2B3%2BCorrects%2BMismatch%2BRepair%2BDeficiency%2Band%2BMicrosatellite%2BInstability%2Band%2BReduces%2BN-Methyl-N%2527-nitro-N-nitrosoguanidine%2BTolerance%2Bin%2BColon%2BTumor%2BCells%2Bwith%2BHomozygous%2BhMLH1%2BMutation%26rft_id%253Dinfo%253Apmid%252F8044777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI1NC8xNi80MzA4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span><span class="cit-comment">a</span> <span class="cit-article-title">Analysis of triplet repeat replication by two-dimensional gel electrophoresis</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DKrasilnikova%26rft.auinit1%253DM.%2BM.%26rft.volume%253D277%26rft.spage%253D19%26rft.epage%253D28%26rft.atitle%253DAnalysis%2Bof%2Btriplet%2Brepeat%2Breplication%2Bby%2Btwo-dimensional%2Bgel%2Belectrophoresis.%26rft_id%253Dinfo%253Apmid%252F15201446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15201446&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.48" data-doi="10.1128/MCB.24.6.2286-2295.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span><span class="cit-comment">b</span> <span class="cit-article-title">Replication stalling at Friedreich’s ataxia (GAA)n repeats in vivo</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2286</span>–<span class="cit-lpage">2295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKrasilnikova%26rft.auinit1%253DM.%2BM.%26rft.volume%253D24%26rft.issue%253D6%26rft.spage%253D2286%26rft.epage%253D2295%26rft.atitle%253DReplication%2BStalling%2Bat%2BFriedreich%2527s%2BAtaxia%2B%2528GAA%2529n%2BRepeats%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.6.2286-2295.2004%26rft_id%253Dinfo%253Apmid%252F14993268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI0LzYvMjI4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.49" data-doi="10.1073/pnas.95.18.10774"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrett</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schug</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Equilibrium distributions of microsatellite repeat length resulting from a balance between slippage events and point mutations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">10774</span>–<span class="cit-lpage">10778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKruglyak%26rft.auinit1%253DS.%26rft.volume%253D95%26rft.issue%253D18%26rft.spage%253D10774%26rft.epage%253D10778%26rft.atitle%253DEquilibrium%2Bdistributions%2Bof%2Bmicrosatellite%2Brepeat%2Blength%2Bresulting%2Bfrom%2Ba%2Bbalance%2Bbetween%2Bslippage%2Bevents%2Band%2Bpoint%2Bmutations%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.18.10774%26rft_id%253Dinfo%253Apmid%252F9724780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMTgvMTA3NzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.50" data-doi="10.1016/j.dnarep.2009.04.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lazzaro</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giannattasio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puddu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellicioli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Checkpoint mechanisms at the intersection between DNA damage and repair</span>. <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1055</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D8%26rft.spage%253D1055%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2009.04.022%26rft_id%253Dinfo%253Apmid%252F19497792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2009.04.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19497792&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.51" data-doi="10.1046/j.1365-294X.2002.01643.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beiles</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Microsatellites: genomic distribution, putative functions and mutational mechanisms: a review</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2453</span>–<span class="cit-lpage">2465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLi%26rft.auinit1%253DY.%2BC.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D2453%26rft.epage%253D2465%26rft.atitle%253DMicrosatellites%253A%2Bgenomic%2Bdistribution%252C%2Bputative%2Bfunctions%2Band%2Bmutational%2Bmechanisms%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2002.01643.x%26rft_id%253Dinfo%253Apmid%252F12453231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2002.01643.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12453231&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179492800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.52" data-doi="10.1093/molbev/msh073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microsatellites within genes: structure, function, and evolution</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">1007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh073%26rft_id%253Dinfo%253Apmid%252F14963101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.53" data-doi="10.1056/NEJMra012242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Chapelle</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Hereditary colorectal cancer</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">348</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DLynch%26rft.auinit1%253DH.%2BT.%26rft.volume%253D348%26rft.issue%253D10%26rft.spage%253D919%26rft.epage%253D932%26rft.atitle%253DHereditary%2BColorectal%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra012242%26rft_id%253Dinfo%253Apmid%252F12621137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMra012242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12621137&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181341100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanspa</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Review of the Lynch syndrome: history, molecular genetics, screening, differential diagnosis, and medicolegal ramifications</span>. <abbr class="cit-jnl-abbrev">Clin. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bgenetics%26rft.stitle%253DClin%2BGenet%26rft.aulast%253DLynch%26rft.auinit1%253DH.%2BT.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DReview%2Bof%2Bthe%2BLynch%2Bsyndrome%253A%2Bhistory%252C%2Bmolecular%2Bgenetics%252C%2Bscreening%252C%2Bdifferential%2Bdiagnosis%252C%2Band%2Bmedicolegal%2Bramifications.%26rft_id%253Dinfo%253Apmid%252F19659756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19659756&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268051600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.55" data-doi="10.1186/1471-2164-9-410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villesen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiuf</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Short tandem repeats in human exons: a target for disease mutations</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMadsen%26rft.auinit1%253DB.%2BE.%26rft.volume%253D9%26rft.spage%253D410%26rft.epage%253D410%26rft.atitle%253DShort%2Btandem%2Brepeats%2Bin%2Bhuman%2Bexons%253A%2Ba%2Btarget%2Bfor%2Bdisease%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-410%26rft_id%253Dinfo%253Apmid%252F18789129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18789129&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.56" data-doi="10.1038/nrg2828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMurray</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mechanisms of trinucleotide repeat instability during human development</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">786</span>–<span class="cit-lpage">799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMcMurray%26rft.auinit1%253DC.%2BT.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D786%26rft.epage%253D799%26rft.atitle%253DMechanisms%2Bof%2Btrinucleotide%2Brepeat%2Binstability%2Bduring%2Bhuman%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2828%26rft_id%253Dinfo%253Apmid%252F20953213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20953213&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.57" data-doi="10.1128/MMBR.00030-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Replication fork stalling at natural impediments</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DMirkin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D71%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D35%26rft.atitle%253DReplication%2BFork%2BStalling%2Bat%2BNatural%2BImpediments%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00030-06%26rft_id%253Dinfo%253Apmid%252F17347517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo3OiI3MS8xLzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.58" data-doi="10.1021/bi9722711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Opresko</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweasy</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The mutator form of polymerase beta with amino acid substitution at tyrosine 265 in the hinge region displays an increase in both base substitution and frame shift errors</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">2111</span>–<span class="cit-lpage">2119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D37%26rft.spage%253D2111%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi9722711%26rft_id%253Dinfo%253Apmid%252F9485358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi9722711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9485358&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072299900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.59" data-doi="10.1002/gcc.20644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westers</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jager</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biochemical characterization of MLH3 missense mutations does not reveal an apparent role of MLH3 in Lynch syndrome</span>. <abbr class="cit-jnl-abbrev">Genes Chromosomes Cancer</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bchromosomes%2B%2526%2Bcancer%26rft.stitle%253DGenes%2BChromosomes%2BCancer%26rft.aulast%253DOu%26rft.auinit1%253DJ.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D340%26rft.epage%253D350%26rft.atitle%253DBiochemical%2Bcharacterization%2Bof%2BMLH3%2Bmissense%2Bmutations%2Bdoes%2Bnot%2Breveal%2Ban%2Bapparent%2Brole%2Bof%2BMLH3%2Bin%2BLynch%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgcc.20644%26rft_id%253Dinfo%253Apmid%252F19156873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gcc.20644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19156873&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.60" data-doi="10.1126/science.7632227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Mismatch repair deficiency in phenotypically normal human cells</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">738</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DParsons%26rft.auinit1%253DR.%26rft.volume%253D268%26rft.issue%253D5211%26rft.spage%253D738%26rft.epage%253D740%26rft.atitle%253DMismatch%2Brepair%2Bdeficiency%2Bin%2Bphenotypically%2Bnormal%2Bhuman%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7632227%26rft_id%253Dinfo%253Apmid%252F7632227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjgvNTIxMS83MzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.61" data-doi="10.1038/nrg1689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichol Edamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleary</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Repeat instability: mechanisms of dynamic mutations</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPearson%26rft.auinit1%253DC.%2BE.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253D729%26rft.epage%253D742%26rft.atitle%253DRepeat%2Binstability%253A%2Bmechanisms%2Bof%2Bdynamic%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1689%26rft_id%253Dinfo%253Apmid%252F16205713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16205713&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232316800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.62" data-doi="10.1128/MCB.23.4.1349-1357.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelletier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samadashwily</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahue</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Replication and expansion of trinucleotide repeats in yeast</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1349</span>–<span class="cit-lpage">1357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DPelletier%26rft.auinit1%253DR.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D1349%26rft.epage%253D1357%26rft.atitle%253DReplication%2Band%2BExpansion%2Bof%2BTrinucleotide%2BRepeats%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.4.1349-1357.2003%26rft_id%253Dinfo%253Apmid%252F12556494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjIzLzQvMTM0OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.63" data-doi="10.1016/j.tibs.2012.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pena-Diaz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiricny</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mammalian mismatch repair: error-free or error-prone?</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DPena-Diaz%26rft.auinit1%253DJ.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D206%26rft.epage%253D214%26rft.atitle%253DMammalian%2Bmismatch%2Brepair%253A%2Berror-free%2Bor%2Berror-prone%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2012.03.001%26rft_id%253Dinfo%253Apmid%252F22475811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2012.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22475811&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304232400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.64" data-doi="10.1534/genetics.103.022665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sainudiin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrett</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microsatellite mutation models: insights from a comparison of humans and chimpanzees</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSainudiin%26rft.auinit1%253DR.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D383%26rft.epage%253D395%26rft.atitle%253DMicrosatellite%2BMutation%2BModels%253A%2BInsights%2BFrom%2Ba%2BComparison%2Bof%2BHumans%2Band%2BChimpanzees%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.022665%26rft_id%253Dinfo%253Apmid%252F15454551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvMzgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.65" data-doi="10.1038/ng1197-298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samadashwily</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raca</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Trinucleotide repeats affect DNA replication in vivo</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">298</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSamadashwily%26rft.auinit1%253DG.%2BM.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D298%26rft.epage%253D304%26rft.atitle%253DTrinucleotide%2Brepeats%2Baffect%2BDNA%2Breplication%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1197-298%26rft_id%253Dinfo%253Apmid%252F9354793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1197-298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9354793&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YD66700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.66" data-doi="10.1158/0008-5472.CAN-08-3499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Human postmeiotic segregation 2 exhibits biased repair at tetranucleotide microsatellite sequences</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-08-3499%26rft_id%253Dinfo%253Apmid%252F19155293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjk6IjY5LzMvMTE0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy80NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.67" data-doi="10.1016/j.molcel.2009.06.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shishkin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voineagu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherng</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernet</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Large-scale expansions of Friedreich’s ataxia GAA repeats in yeast</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DShishkin%26rft.auinit1%253DA.%2BA.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D82%26rft.epage%253D92%26rft.atitle%253DLarge-scale%2Bexpansions%2Bof%2BFriedreich%2527s%2Bataxia%2BGAA%2Brepeats%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.06.017%26rft_id%253Dinfo%253Apmid%252F19595718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19595718&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268003300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.68" data-doi="10.1126/science.1074023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sogo</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foiani</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Fork reversal and ssDNA accumulation at stalled replication forks owing to checkpoint defects</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSogo%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D297%26rft.issue%253D5581%26rft.spage%253D599%26rft.epage%253D602%26rft.atitle%253DFork%2BReversal%2Band%2BssDNA%2BAccumulation%2Bat%2BStalled%2BReplication%2BForks%2BOwing%2Bto%2BCheckpoint%2BDefects%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1074023%26rft_id%253Dinfo%253Apmid%252F12142537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTcvNTU4MS81OTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.69" data-doi="10.1097/00008571-200003000-00005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stangl</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cascorbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laule</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stangl</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">High CA repeat numbers in intron 13 of the endothelial nitric oxide synthase gene and increased risk of coronary artery disease</span>. <abbr class="cit-jnl-abbrev">Pharmacogenetics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacogenetics%26rft.stitle%253DPharmacogenetics%26rft.aulast%253DStangl%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D140%26rft.atitle%253DHigh%2BCA%2Brepeat%2Bnumbers%2Bin%2Bintron%2B13%2Bof%2Bthe%2Bendothelial%2Bnitric%2Boxide%2Bsynthase%2Bgene%2Band%2Bincreased%2Brisk%2Bof%2Bcoronary%2Bartery%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00008571-200003000-00005%26rft_id%253Dinfo%253Apmid%252F10762001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00008571-200003000-00005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10762001&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086148500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.70" data-doi="10.1038/365274a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prolla</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liskay</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Destabilization of tracts of simple repetitive DNA in yeast by mutations affecting DNA mismatch repair</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">274</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStrand%26rft.auinit1%253DM.%26rft.volume%253D365%26rft.issue%253D6443%26rft.spage%253D274%26rft.epage%253D276%26rft.atitle%253DDestabilization%2Bof%2Btracts%2Bof%2Bsimple%2Brepetitive%2BDNA%2Bin%2Byeast%2Bby%2Bmutations%2Baffecting%2BDNA%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F365274a0%26rft_id%253Dinfo%253Apmid%252F8371783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/365274a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8371783&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LX47100061&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.71" data-doi="10.1038/ng.2398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helgason</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebenesersdottir</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A direct characterization of human mutation based on microsatellites</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1161</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSun%26rft.auinit1%253DJ.%2BX.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1161%26rft.epage%253D1165%26rft.atitle%253DA%2Bdirect%2Bcharacterization%2Bof%2Bhuman%2Bmutation%2Bbased%2Bon%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2398%26rft_id%253Dinfo%253Apmid%252F22922873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22922873&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Umar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risinger</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Defective mismatch repair in extracts of colorectal and endometrial cancer cell lines exhibiting microsatellite instability</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">14367</span>–<span class="cit-lpage">14370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DUmar%26rft.auinit1%253DA.%26rft.volume%253D269%26rft.issue%253D20%26rft.spage%253D14367%26rft.epage%253D14370%26rft.atitle%253DDefective%2Bmismatch%2Brepair%2Bin%2Bextracts%2Bof%2Bcolorectal%2Band%2Bendometrial%2Bcancer%2Bcell%2Blines%2Bexhibiting%2Bmicrosatellite%2Binstability%26rft_id%253Dinfo%253Apmid%252F8182040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvMjAvMTQzNjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.73" data-doi="10.1093/hmg/11.2.191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van den Broek</span>  <span class="cit-name-given-names">W. J. A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelen</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wansink</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coerwinkel</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">te Riele</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Somatic expansion behaviour of the (CTG) n repeat in myotonic dystrophy knock-in mice is differentially affected by Msh3 and Msh6 mismatchâ€“repair proteins</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253Dvan%2Bden%2BBroek%26rft.auinit1%253DW.%2BJ.%2BA.%2BA.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D191%26rft.epage%253D198%26rft.atitle%253DSomatic%2Bexpansion%2Bbehaviour%2Bof%2Bthe%2B%2528CTG%2529n%2Brepeat%2Bin%2Bmyotonic%2Bdystrophy%2Bknock-in%2Bmice%2Bis%2Bdifferentially%2Baffected%2Bby%2BMsh3%2Band%2BMsh6%2Bmismatch-repair%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F11.2.191%26rft_id%253Dinfo%253Apmid%252F11809728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjg6IjExLzIvMTkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.74" data-doi="10.1073/pnas.1008589107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Oers</span>  <span class="cit-name-given-names">J. M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werling</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genschel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">PMS2 endonuclease activity has distinct biological functions and is essential for genome maintenance</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">13384</span>–<span class="cit-lpage">13389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1008589107%26rft_id%253Dinfo%253Apmid%252F20624957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzMwLzEzMzg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.75" data-doi="10.1038/nsmb.1527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voineagu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surka</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shishkin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Replisome stalling and stabilization at CGG repeats, which are responsible for chromosomal fragility</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DVoineagu%26rft.auinit1%253DI.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D226%26rft.epage%253D228%26rft.atitle%253DReplisome%2Bstalling%2Band%2Bstabilization%2Bat%2BCGG%2Brepeats%252C%2Bwhich%2Bare%2Bresponsible%2Bfor%2Bchromosomal%2Bfragility.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1527%26rft_id%253Dinfo%253Apmid%252F19136957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19136957&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.76" data-doi="10.1016/j.gene.2012.06.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Target genes of microsatellite sequences in head and neck squamous cell carcinoma: Mononucleotide repeats are not detected</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">506</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D506%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2012.06.056%26rft_id%253Dinfo%253Apmid%252F22771919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2012.06.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22771919&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.77" data-doi="10.1016/j.tibs.2007.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Non-B DNA conformations, mutagenesis and disease</span>. <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DWells%26rft.auinit1%253DR.%2BD.%26rft.volume%253D32%26rft.issue%253D6%26rft.spage%253D271%26rft.epage%253D278%26rft.atitle%253DNon-B%2BDNA%2Bconformations%252C%2Bmutagenesis%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2007.04.003%26rft_id%253Dinfo%253Apmid%252F17493823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2007.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17493823&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247654100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.78" data-doi="10.1093/nar/gki697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dere</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebert</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Napierala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Son</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Advances in mechanisms of genetic instability related to hereditary neurological diseases</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">3785</span>–<span class="cit-lpage">3798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWells%26rft.auinit1%253DR.%2BD.%26rft.volume%253D33%26rft.issue%253D12%26rft.spage%253D3785%26rft.epage%253D3798%26rft.atitle%253DAdvances%2Bin%2Bmechanisms%2Bof%2Bgenetic%2Binstability%2Brelated%2Bto%2Bhereditary%2Bneurological%2Bdiseases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki697%26rft_id%253Dinfo%253Apmid%252F16006624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMy8xMi8zNzg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbord</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boxall</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Likelihood-based estimation of microsatellite mutation rates</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWhittaker%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D781%26rft.epage%253D787%26rft.atitle%253DLikelihood-Based%2BEstimation%2Bof%2BMicrosatellite%2BMutation%2BRates%26rft_id%253Dinfo%253Apmid%252F12807796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNzgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzQ1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.451.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roques</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Relative rates of insertion and deletion mutations in dinucleotide repeats of various lengths in mismatch repair proficient mouse and mismatch repair deficient human cells</span>. <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">499</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DYamada%26rft.auinit1%253DN.%2BA.%26rft.volume%253D499%26rft.issue%253D2%26rft.spage%253D213%26rft.epage%253D225%26rft.atitle%253DRelative%2Brates%2Bof%2Binsertion%2Band%2Bdeletion%2Bmutations%2Bin%2Bdinucleotide%2Brepeats%2Bof%2Bvarious%2Blengths%2Bin%2Bmismatch%2Brepair%2Bproficient%2Bmouse%2Band%2Bmismatch%2Brepair%2Bdeficient%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Apmid%252F11827714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11827714&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F451.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174045500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/3/451.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/441" title="The Genetic Architecture of Degenerin/Epithelial Sodium Channels in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/465" title="Unequal Recombination and Evolution of the Mating-Type (MAT) Loci in the Pathogenic Fungus Grosmannia clavigera and Relatives" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/3"><span>Volume 3 Issue 3, March 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/3.cover-source.jpg?itok=w5x3d6rp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/451" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZCaqfXrjPoqikflwrjdaFXki_F2nc94_W3RGKG3Ku4Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/3/451" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/3/451" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/3/451" class="active" data-icon-position="" data-hide-link-title="0">Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459428" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JD1P0zAs86W7eY3CUL9IF-MWFe81FhGlQe92LeI3l7c" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/451.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/3/451" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-QhymKejA0XaCL6uefXFaeUT1hs6iPVTq3laYMZ0Izrk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/3/451" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/451.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459428" id="node459428--2" data-pisa="ggg;3/3/451" data-pisa-master="ggg;3/3/451" data-apath="/ggg/3/3/451.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Beverly A. Baptiste</span>, <span class="highwire-citation-author" data-delta="1">Guruprasad Ananda</span>, <span class="highwire-citation-author" data-delta="2">Noelle Strubczewski</span>, <span class="highwire-citation-author" data-delta="3">Andrew Lutzkanin</span>, <span class="highwire-citation-author" data-delta="4">Su Jen Khoo</span>, <span class="highwire-citation-author" data-delta="5">Abhinaya Srikanth</span>, <span class="highwire-citation-author" data-delta="6">Nari Kim</span>, <span class="highwire-citation-author" data-delta="7">Kateryna D. Makova</span>, <span class="highwire-citation-author" data-delta="8">Maria M. Krasilnikova</span> and <span class="highwire-citation-author" data-delta="9">Kristin A. Eckert</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">451-463;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005173 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459428--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beverly A. Baptiste </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Intercollege Graduate Program in Genetics, Huck Institutes of Life Sciences, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beverly%2BA.%2BBaptiste%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baptiste%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeverly%2BA.%2BBaptiste%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guruprasad Ananda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Program in Bioinformatics and Genomics, Huck Institutes of Life Sciences, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guruprasad%2BAnanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ananda%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuruprasad%2BAnanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Noelle Strubczewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noelle%2BStrubczewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strubczewski%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoelle%2BStrubczewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrew Lutzkanin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BLutzkanin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lutzkanin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BLutzkanin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Su Jen Khoo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Su%2BJen%2BKhoo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khoo%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASu%2BJen%2BKhoo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abhinaya Srikanth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhinaya%2BSrikanth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srikanth%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhinaya%2BSrikanth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nari Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nari%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANari%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kateryna D. Makova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology, Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kateryna%2BD.%2BMakova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Makova%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKateryna%2BD.%2BMakova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Maria M. Krasilnikova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BM.%2BKrasilnikova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krasilnikova%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BM.%2BKrasilnikova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:muk19@psu.edu" class="" data-icon-position="" data-hide-link-title="0">muk19@psu.edu</a>
<a href="mailto:kae4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">kae4@psu.edu</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kristin A. Eckert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristin%2BA.%2BEckert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eckert%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristin%2BA.%2BEckert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:muk19@psu.edu" class="" data-icon-position="" data-hide-link-title="0">muk19@psu.edu</a>
<a href="mailto:kae4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">kae4@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/3/451" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/3/451&amp;title=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/3/451&amp;title=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/3/451&amp;title=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/3/451&amp;text=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/3/451&amp;title=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/3/451&amp;t=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/3/451&amp;title=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/3/451&amp;title=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459428/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459428" id="node459428--3" data-pisa="ggg;3/3/451" data-pisa-master="ggg;3/3/451" data-apath="/ggg/3/3/451.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mature Microsatellites: Mechanisms Underlying Dinucleotide Microsatellite Mutational Biases in Human Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Beverly A. Baptiste</span>, <span class="highwire-citation-author" data-delta="1">Guruprasad Ananda</span>, <span class="highwire-citation-author" data-delta="2">Noelle Strubczewski</span>, <span class="highwire-citation-author" data-delta="3">Andrew Lutzkanin</span>, <span class="highwire-citation-author" data-delta="4">Su Jen Khoo</span>, <span class="highwire-citation-author" data-delta="5">Abhinaya Srikanth</span>, <span class="highwire-citation-author" data-delta="6">Nari Kim</span>, <span class="highwire-citation-author" data-delta="7">Kateryna D. Makova</span>, <span class="highwire-citation-author" data-delta="8">Maria M. Krasilnikova</span> and <span class="highwire-citation-author" data-delta="9">Kristin A. Eckert</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">451-463;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005173 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459428--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beverly A. Baptiste </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Intercollege Graduate Program in Genetics, Huck Institutes of Life Sciences, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beverly%2BA.%2BBaptiste%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baptiste%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeverly%2BA.%2BBaptiste%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guruprasad Ananda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Program in Bioinformatics and Genomics, Huck Institutes of Life Sciences, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guruprasad%2BAnanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ananda%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuruprasad%2BAnanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Noelle Strubczewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noelle%2BStrubczewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strubczewski%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoelle%2BStrubczewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrew Lutzkanin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BLutzkanin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lutzkanin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BLutzkanin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Su Jen Khoo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Su%2BJen%2BKhoo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khoo%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASu%2BJen%2BKhoo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abhinaya Srikanth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhinaya%2BSrikanth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srikanth%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhinaya%2BSrikanth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nari Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nari%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANari%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kateryna D. Makova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology, Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kateryna%2BD.%2BMakova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Makova%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKateryna%2BD.%2BMakova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Maria M. Krasilnikova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Pennsylvania State University College of Science, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BM.%2BKrasilnikova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krasilnikova%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BM.%2BKrasilnikova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:muk19@psu.edu" class="" data-icon-position="" data-hide-link-title="0">muk19@psu.edu</a>
<a href="mailto:kae4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">kae4@psu.edu</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kristin A. Eckert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pathology, Gittlen Cancer Research Foundation, Pennsylvania State University College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Medical Genomics, Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristin%2BA.%2BEckert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eckert%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristin%2BA.%2BEckert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:muk19@psu.edu" class="" data-icon-position="" data-hide-link-title="0">muk19@psu.edu</a>
<a href="mailto:kae4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">kae4@psu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=451&amp;rft.epage=463&amp;rft.atitle=Mature%20Microsatellites%3A%20Mechanisms%20Underlying%20Dinucleotide%20Microsatellite%20Mutational%20Biases%20in%20Human%20Cells&amp;rft.volume=3&amp;rft.issue=3&amp;rft.date=2013-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Baptiste%2C+Beverly+A.&amp;rft.au=Ananda%2C+Guruprasad&amp;rft.au=Strubczewski%2C+Noelle&amp;rft.au=Lutzkanin%2C+Andrew&amp;rft.au=Khoo%2C+Su+Jen&amp;rft.au=Srikanth%2C+Abhinaya&amp;rft.au=Kim%2C+Nari&amp;rft.au=Makova%2C+Kateryna+D.&amp;rft.au=Krasilnikova%2C+Maria+M.&amp;rft.au=Eckert%2C+Kristin+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459428/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459428/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459428/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459428/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459428/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459428/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459428/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459428/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459428/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459428/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459428/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459428/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/3/451" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/3/451.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/3/451.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/3/451.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/3/451.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
