<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/3/387" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/3/387.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/3/387.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/3/387.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/3/387.atom" />
<meta name="HW.pisa" content="ggg;3/3/387" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.112.004812" />
<meta name="DC.Date" content="2013-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 Tilgner et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Precise identification of RNA-coding regions and transcriptomes of eukaryotes is a significant problem in biology. Currently, eukaryote transcriptomes are analyzed using deep short-read sequencing experiments of complementary DNAs. The resulting short-reads are then aligned against a genome and annotated junctions to infer biological meaning. Here we use long-read complementary DNA datasets for the analysis of a eukaryotic transcriptome and generate two large datasets in the human K562 and HeLa S3 cell lines. Both data sets comprised at least 4 million reads and had median read lengths greater than 500 bp. We show that annotation-independent alignments of these reads provide partial gene structures that are very much in-line with annotated gene structures, 15% of which have not been obtained in a previous de novo analysis of short reads. For long-noncoding RNAs ( i.e. , lncRNA) genes, however, we find an increased fraction of novel gene structures among our alignments. Other important aspects of transcriptome analysis, such as the description of cell type-specific splicing, can be performed in an accurate, reliable and completely annotation-free manner, making it ideal for the analysis of transcriptomes of newly sequenced genomes. Furthermore, we demonstrate that long read sequence can be assembled into full-length transcripts with considerable success. Our method is applicable to all long read sequencing technologies." />
<meta name="DC.Contributor" content="Hagen Tilgner" />
<meta name="DC.Contributor" content="Debasish Raha" />
<meta name="DC.Contributor" content="Lukas Habegger" />
<meta name="DC.Contributor" content="Mohammed Mohiuddin" />
<meta name="DC.Contributor" content="Mark Gerstein" />
<meta name="DC.Contributor" content="Michael Snyder" />
<meta name="article:published_time" content="2013-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Precise identification of RNA-coding regions and transcriptomes of eukaryotes is a significant problem in biology. Currently, eukaryote transcriptomes are analyzed using deep short-read sequencing experiments of complementary DNAs. The resulting short-reads are then aligned against a genome and annotated junctions to infer biological meaning. Here we use long-read complementary DNA datasets for the analysis of a eukaryotic transcriptome and generate two large datasets in the human K562 and HeLa S3 cell lines. Both data sets comprised at least 4 million reads and had median read lengths greater than 500 bp. We show that annotation-independent alignments of these reads provide partial gene structures that are very much in-line with annotated gene structures, 15% of which have not been obtained in a previous &lt;i&gt;de novo&lt;/i&gt; analysis of short reads. For long-noncoding RNAs (&lt;i&gt;i.e.&lt;/i&gt;, lncRNA) genes, however, we find an increased fraction of novel gene structures among our alignments. Other important aspects of transcriptome analysis, such as the description of cell type-specific splicing, can be performed in an accurate, reliable and completely annotation-free manner, making it ideal for the analysis of transcriptomes of newly sequenced genomes. Furthermore, we demonstrate that long read sequence can be assembled into full-length transcripts with considerable success. Our method is applicable to all long read sequencing technologies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/03/01" />
<meta name="citation_mjid" content="ggg;3/3/387" />
<meta name="citation_id" content="3/3/387" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/3/387" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/3/387.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/3/387.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/3/387.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004812" />
<meta name="citation_pmid" content="23450794" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="387" />
<meta name="citation_lastpage" content="397" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hagen Tilgner" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University, Stanford, California 94305" />
<meta name="citation_author" content="Debasish Raha" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, Yale University, New Haven, Connecticut 06120" />
<meta name="citation_author" content="Lukas Habegger" />
<meta name="citation_author_institution" content="Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120" />
<meta name="citation_author" content="Mohammed Mohiuddin" />
<meta name="citation_author_institution" content="Roche, Branford, Connecticut 06405" />
<meta name="citation_author" content="Mark Gerstein" />
<meta name="citation_author_institution" content="Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120" />
<meta name="citation_author" content="Michael Snyder" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University, Stanford, California 94305" />
<meta name="citation_author_email" content="mpsnyder@stanford.edu" />
<meta name="citation_reference" content="citation_journal_title=J. Royal Stat. Soc. B.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. J. David;citation_author=J. L. Manley;citation_title=Alternative pre-mRNA splicing regulation in cancer: pathways and programs unhinged;citation_pages=2343-2364;citation_volume=24;citation_year=2010;citation_issue=21;citation_pmid=21041405;citation_doi=10.1101/gad.1973010" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=T. Derrien;citation_author=R. Johnson;citation_author=G. Bussotti;citation_author=A. Tanzer;citation_author=S. Djebali;citation_title=The GENCODE v7 catalog of human long noncoding RNAs: Analysis of their gene structure, evolution, and expression;citation_pages=1775-1789;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955988;citation_doi=10.1101/gr.132159.111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Djebali;citation_author=C. A. Davis;citation_author=A. Merkel;citation_author=A. Dobin;citation_author=T. Lassmann;citation_title=Landscape of transcription in human cells.;citation_pages=101-108;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955620;citation_doi=10.1038/nature11233" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Eid;citation_author=A. Fehr;citation_author=J. Gray;citation_author=K. Luong;citation_author=J. Lyle;citation_title=Real-time DNA sequencing from single polymerase molecules;citation_volume=323;citation_year=2009;citation_pmid=19023044;citation_doi=10.1126/science.1162986" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. R. Graveley;citation_author=A. N. Brooks;citation_author=J. W. Carlson;citation_author=M. O. Duff;citation_author=J. M. Landolin;citation_title=The developmental transcriptome of Drosophila melanogaster.;citation_pages=473-479;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21179090;citation_doi=10.1038/nature09715" />
<meta name="citation_reference" content="citation_title=Genome Biol.;citation_author=J. Harrow;citation_author=F. Denoeud;citation_author=A. Frankish;citation_author=A. Reymond;citation_author=C. K. Chen;citation_title=GENCODE: producing a reference annotation for ENCODE;citation_pages=S41-S49;citation_volume=7;citation_year=2006;citation_issue=Suppl. 1;citation_doi=10.1186/gb-2006-7-5-r41" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Harrow;citation_author=A. Frankish;citation_author=J. M. Gonzalez;citation_author=E. Tapanari;citation_author=M. Diekhans;citation_title=GENCODE: The reference human genome annotation for The ENCODE Project;citation_pages=1760-1774;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955987;citation_doi=10.1101/gr.135350.111" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Johnson;citation_author=J. Castle;citation_author=P. Garrett-Engele;citation_author=Z. Kan;citation_author=P. M. Loerch;citation_title=Genome-Wide Survey of Human Alternative Pre-mRNA Splicing with Exon Junction Microarrays;citation_pages=2141-2144;citation_volume=302;citation_year=2003;citation_issue=5653;citation_pmid=14684825;citation_doi=10.1126/science.1090100" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_author=C. W. Sugnet;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=T. H. Pringle;citation_title=The human genome browser at UCSC;citation_volume=12;citation_year=2002;citation_pmid=12045153;citation_doi=10.1101/gr.229102. Article published online before print in May 2002" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lander;citation_author=L. M. Linton;citation_author=B. Birren;citation_author=C. Nusbaum;citation_author=M. C. Zody;citation_title=Initial sequencing and analysis of the human genome.;citation_pages=860-921;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237011;citation_doi=10.1038/35057062" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Magen;citation_author=G. Ast;citation_title=The importance of being divisible by three in alternative splicing;citation_pages=5574-5582;citation_volume=33;citation_year=2005;citation_issue=17;citation_pmid=16192573;citation_doi=10.1093/nar/gki858" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. B. Montgomery;citation_author=M. Sammeth;citation_author=M. Gutierrez-Arcelus;citation_author=R. P. Lach;citation_author=C. Ingle;citation_title=Transcriptome genetics using second generation sequencing in a Caucasian population.;citation_pages=773-777;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=20220756;citation_doi=10.1038/nature08903" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=U. Nagalakshmi;citation_author=Z. Wang;citation_author=K. Waern;citation_author=C. Shou;citation_author=D. Raha;citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing;citation_volume=320;citation_year=2008;citation_pmid=18451266;citation_doi=10.1126/science.1158441" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Q. Pan;citation_author=O. Shai;citation_author=L. J. Lee;citation_author=B. J. Frey;citation_author=B. J. Blencowe;citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing.;citation_pages=1413-1415;citation_volume=40;citation_year=2008;citation_issue=12;citation_pmid=18978789;citation_doi=10.1038/ng.259" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. Parra;citation_author=E. Blanco;citation_author=R. GuigÃ³;citation_title=GeneID in Drosophila;citation_pages=511-515;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779490;citation_doi=10.1101/gr.10.4.511" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. D. Pruitt;citation_author=T. Tatusova;citation_author=G. R. Brown;citation_author=D. R. Maglott;citation_title=NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy;citation_volume=40;citation_year=2012;citation_pmid=22121212;citation_doi=10.1093/nar/gkr1079" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Roberts;citation_author=C. Trapnell;citation_author=J. Donaghey;citation_author=J. L. Rinn;citation_author=L. Pachter;citation_title=Improving RNA-Seq expression estimates by correcting for fragment bias.;citation_pages=R22-R22;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21410973;citation_doi=10.1186/gb-2011-12-3-r22" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. C. Venter;citation_author=M. D. Adams;citation_author=E. W. Myers;citation_author=P. W. Li;citation_author=R. J. Mural;citation_title=The Sequence of the Human Genome;citation_pages=1304-1351;citation_volume=291;citation_year=2001;citation_issue=5507;citation_pmid=11181995;citation_doi=10.1126/science.1058040" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. T. Wang;citation_author=R. Sandberg;citation_author=S. Luo;citation_author=I. Khrebtukova;citation_author=L. Zhang;citation_title=Alternative isoform regulation in human tissue transcriptomes.;citation_pages=470-476;citation_volume=456;citation_year=2008;citation_issue=7221;citation_pmid=18978772;citation_doi=10.1038/nature07509" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=57-63;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19015660;citation_doi=10.1038/nrg2484" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Q. Wu;citation_author=L. Habegger;citation_author=P. Noisa;citation_author=A. Szekely;citation_author=C. Qiu;citation_title=Dynamic transcriptomes during neural differentiation of human embryonic stem cells revealed by short, long, and paired-end sequencing;citation_volume=107;citation_year=2010;citation_pmid=20194744;citation_doi=10.1073/pnas.0914114107" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=C. L. Zheng;citation_author=X. D. Fu;citation_author=M. Gribskov;citation_title=Characteristics and regulatory elements defining constitutive splicing and different modes of alternative splicing in human and mouse;citation_volume=11;citation_year=2005;citation_pmid=16251388;citation_doi=10.1261/rna.2660805" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="twitter:description" content="Precise identification of RNA-coding regions and transcriptomes of eukaryotes is a significant problem in biology. Currently, eukaryote transcriptomes are analyzed using deep short-read sequencing experiments of complementary DNAs. The resulting short-reads are then aligned against a genome and annotated junctions to infer biological meaning. Here we use long-read complementary DNA datasets for the analysis of a eukaryotic transcriptome and generate two large datasets in the human K562 and HeLa S3 cell lines. Both data sets comprised at least 4 million reads and had median read lengths greater than 500 bp. We show that annotation-independent alignments of these reads provide partial gene structures that are very much in-line with annotated gene structures, 15% of which have not been obtained in a previous de novo analysis of short reads. For long-noncoding RNAs ( i.e. , lncRNA) genes, however, we find an increased fraction of novel gene structures among our alignments. Other important aspects of transcriptome analysis, such as the description of cell type-specific splicing, can be performed in an accurate, reliable and completely annotation-free manner, making it ideal for the analysis of transcriptomes of newly sequenced genomes. Furthermore, we demonstrate that long read sequence can be assembled into full-length transcripts with considerable success. Our method is applicable to all long read sequencing technologies." />
<meta name="og-title" property="og:title" content="Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/3/387" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Precise identification of RNA-coding regions and transcriptomes of eukaryotes is a significant problem in biology. Currently, eukaryote transcriptomes are analyzed using deep short-read sequencing experiments of complementary DNAs. The resulting short-reads are then aligned against a genome and annotated junctions to infer biological meaning. Here we use long-read complementary DNA datasets for the analysis of a eukaryotic transcriptome and generate two large datasets in the human K562 and HeLa S3 cell lines. Both data sets comprised at least 4 million reads and had median read lengths greater than 500 bp. We show that annotation-independent alignments of these reads provide partial gene structures that are very much in-line with annotated gene structures, 15% of which have not been obtained in a previous de novo analysis of short reads. For long-noncoding RNAs ( i.e. , lncRNA) genes, however, we find an increased fraction of novel gene structures among our alignments. Other important aspects of transcriptome analysis, such as the description of cell type-specific splicing, can be performed in an accurate, reliable and completely annotation-free manner, making it ideal for the analysis of transcriptomes of newly sequenced genomes. Furthermore, we demonstrate that long read sequence can be assembled into full-length transcripts with considerable success. Our method is applicable to all long read sequencing technologies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
    <title>Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"90yka9pgj6qOmlOb-kgihdfxKSSTySw9yx34EWrfU6Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459437","apath":"\/ggg\/3\/3\/387.atom","pisa":"ggg;3\/3\/387","ac":{"\/ggg\/3\/3\/387.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/3\/387.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/3\/387"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/3\/387"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/3\/387.full":true,"\/content\/3\/3\/387":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qLlF\\\\\\\/eLWNqnCAOiKNFjOc0R\\\\\\\/J++2PM57HtJ+uWpIj1HGRyCEGqPeKjvLp0LyxeROGTkDS8SSasm82PLjLcg0fEBt\\\\\\\/TOa1OX6onSaspBzWi8nar7wfs9Og2UeW3YoV9Q9owLuDyAS2lDVerGltgVaWwqTxgwtePjE9ZkVW8OnS8gdpl\\\\\\\/qhwIkB\\\\\\\/rhM\\\\\\\/ebRPpzzQz5dnbHAiFONUeyNJdjVo4yLAc7I8ioRjGa\\\\\\\/TtXwjBjyf3+sKKYMj645LJgV1SMYJH\\\\\\\/lvha6KXC+7m5R+kBsK3nieSjZaSpNvEr0KUSVoJSUqgO2yKYQW1Qs5APcaTomC0a6pnNY3Lq4qZfHAlRik9w6Lu\\\\\\\/\\\\\\\/lmWRhOxC2h36yYm5HFsOVGAP35+rfxjiCOEDk07OzCmNG2MPJJFlX2exmCETfU0FqLr817Adl1jAsEdjccNDRklogd+WWqOSGTXOPTqOqR7xYuYqMRTZ12BxRIlHj41k272VYz5v7ygCjTx82eDwbp+\\\\\\\/sxsGINi0G\\\\\\\/hTY9ZmGpOWj2f0TZrLxmCfq8Ih0j6x22mSIOiKkqw2Xh9vLy1k45B0b7H+VwEiVePnj1oE3yachlNzGos+iiwFmis3Npwo3dFz49A0zhnHVBk9Rq6oLBT9uL1vwPSL5soE2qqLF73j5\\\\\\\/5jEzDn\\\\\\\/L1Jf9VZuyl2YqJkwvkh2XfS6ftY\\\\\\\/heLVJdTqFsbj23HHiNvb8ZCSthPbhryFQreaZFUeB6vMNvT70QodAP7OmFNdCTOqOce8jj72JdlqwgNjPlUFT3HYTqAh2m5mpV4GayYAzW3fINRu7bZGNAnD0YP0psTRub8hDfmEKGK3vwRHwIuhL0u5ahmI0DZikbLWC+\\\\\\\/H3Q+qSIGKT8zecviGSnBshEbX1kWfKLFaUhadfyj0neL5dyxy++YG0bzsEGDJs\\\\\\\/LTKssb7W1zRJQ0wmRgBfBWdi+uQwY2hXITYLnubDoMwpWumvwpw14PXcd+nynhI+JqGJVCL27EzgjNNZzp7dU377LKMJRx7rMVjMAH4T73hDfVLErc4YyvYKuuxGS1HziYVxGqIkvcbZuAQkHrbsosNle9TOwcG0lOOUDXJW1aMDIaKqGA8yNpKZoT+k1khd2D+slR2B+g32rzPWz2LiqJ5eF9RHZC9jwStGa7xWX9dNL9qRZMCNLzko2J\\\\\\\/Q3xEnRDVsb5GtQM7m\\\\\\\/gRk+DCDZE9wKCrKadjnpmbaRzSitmIBWSFjK\\\\\\\/xL+fyuxNx69Uf0TMzZODZbJ60b\\\\\\\/sVCiSDzIDK173W0oSeqm7Pt6AaLrIRf\\\u0022,\\\u0022iv\\\u0022:\\\u00220w2p5adtNWOq6M4Kp65Jlw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262866444dd8e2831b0ed7e99feef90df461a7e3f1a07b93621db1927c4f4809f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227WMo9oBuywoc759HheqUSoDSaT\\\\\\\/ftAp2roaYi7vLMMPdrDhp0JgdtlX+1HHAJUWpgU16ZvmXsPgk0joGM7Uz6zhibUqwjUJDk1wveoUHTCWoOl2AZKuXzKX07KtA6LtRbsn02BR8enN3zJWYmKV7re7MrKj1ue99ji4DfYgIPD+uROaKGPWrWjmRehNWi7diMT+b+3J0nLMWa1c1gxR5JCbRMxaxDhH6+Urcp35dDGqRrI0UFoatrD7yW\\\\\\\/tnZ1PIL6WCHAfIGOkVHL2TPr+T5riflGBvjP09EhGTRR1yGQQFhUpKlPPMGI6o05pPkn\\\\\\\/GAn4RB4lAumB\\\\\\\/vxBqLp12QufvCzr0Ywl9REqMyg\\\\\\\/CtupjHg4naazmgmsI2aEPAcO7nELm+F33QAr5A3OtCDDyWSaQhfh1o+y+iOQ+XYKIqUEEcKg0iqE\\\\\\\/If62l6vMG5NitefMFDnXPP0IJX0asGM6Zy92G72g3f19JP7krzlpnhtuFbzutNZM5osvXL6vmorYEI6Gk73g8di0X8DlJDs\\\\\\\/Mtc9NcxgmeZ4cx6rXaJ3+rldE3KwofkBnScCMWiBl0pvNo2gS8nAB51AvLC8o5rgrvHzzbYwHUaQrsSIGyuGOxrir4yBcYR2KfXyrI0Rfa1c+vuojJLMcTQK9ofwpmc0oPTfEQMMBzuIO8R+GGIsiY6+FlEAs6sYV2BIiGegKnoyvUaqIh+JRpmEG73Ym6QuZhs\\\\\\\/olhDiTnKu0B\\\\\\\/\\\\\\\/100A\\\\\\\/22K0SncWBVD7J7rQCwFjl87owFos1fWbEiUxHGrOvVQhT1b7UsSB9NyxrKnxwfwgu5X2snIeuZl+4+6nVGe\\\\\\\/skkfrhk6laobm459I\\\\\\\/Fte+JU2+T6eDurf7gR4cFfmkGkCifYyBdF642rk7Et8neXypL0tMTo52S4XGGBfnhMxwRjQC7WD6qM3b0ez\\\\\\\/TfMX90syjxL6I0eUsl0mDGybv5oOZgCG5hSWXdYQoSnD1DpL8pdMvcd0Pt+69svo5uVWRXSCqfJ6HnVN2yyjlr9yTV4b9rLBAo3xEDe7LqZZGa+v8SoxHtbGQBrZlyT0fptkl3qDfFpgGsaDUj9BeE+y8J0nA8dopKuUrr95Q1psJAJUqoB8fmbQ60KU8xrsOmBLH+P7sU44IwowdNsEBdeuS7TZlQSzphd0ERkmxuW4dU9ebDtIV9aKL1hjd+S0mZ9I0yPV1I3Lw1o8sprZM4+1YbrrtEHUIv+4vA+2xOpPCCh0\\\\\\\/uzkdSMuX0XSFDM6TEG8G\\\\\\\/5SdCrHQapt+Ur2AV+QCZRzH55MpSPKCC62+8qsocump4WkDymMtE27ynbGkUJTZKszfNSZvnxU4u+cI2orPoSVMv62ATey5o1aG1JyrXt0YO3VU0pocp1RSR6Zm78EQnZsKISHVfvzRwIf+U2DOR1wdIcJoFG+k4JD9i1\\\\\\\/EC7vrXF8UlOTAgWYFAi4fWw1ms1ZTQjdgUjFbtART6R9C2MfcCAxEt69EOjoTFZyVXuaPT8nQ\\\\\\\/jwb6S5jhdJLO1fPLpVQ2m1OxlGN++6XRRQY6dHHxfTMbd\\\\\\\/291cKYIA6VAVLQevviq+QdWXXz5gFwype5Dp9ul1XdPyCAsYbr2+BmTXBptauQnCjHhH2nrRhFl4EqNVYMVQhFnxvSwU8wMX3vB4EugoIUvja2S61GUQ5hyRmQRiQUdrk2I709pMaDeEBKdg6wkjU8H7wsso94VHmAyTifRW\\\\\\\/EHXSQt24GMk9A9R82p3kMbne5evbFsgotg1jvfcjR8wI\\\\\\\/vIp1hXXs+GJSWYOLRI9mlHROUcTRzEU+vkPn5KXTva0YdgHVembT3LCisyzN7BLId9IUo+T+n45PS6QL510NjS2vAm7YUMlbr0Z\\\\\\\/\\\\\\\/hd44Vs4hU136M1IYDSSuFjQQVKfuQk\\\\\\\/Rjwfbk\\\\\\\/pUbyGJlu38KzOjE5vSv00TOo5Nhq+75LpHnwXSpxGIfXvYUptYSPyYiRApO2wHChP6JPx\\\\\\\/LhvA98nVSK4AQwQxa9Dq7MZMxa0C6slqVfPlHY5nys3CjLb806m8K5lW0TvSWNDWQouSIuL1E1m9EpJjN2VHJ4YWDRAx1N\\\\\\\/2hZkXc0k+bT6yvSrX8TpQT\\\\\\\/Bu\\\\\\\/axF8RaZTcmZq9ZuZRjZRxkuPjGfQHEjdOO8aipshBL6zZYjtSxQ\\\\\\\/R7+RNAMJFBvcV\\\\\\\/fy+4T91JbpLjIQLIj\\\\\\\/CzovE3dNf\\\\\\\/6dQFA1GAR68F1fR5rpJj4e8fnkmKQWagKiUU0J5JgQRNKSj2d6KSlStJHKDV7Hd4sntjvE+BmQIh00\\\\\\\/BsWykrUWUjkGgN1CwCJ8G05ShHNe8ybzyXFC26AYsu4Tb6i0Kf9pZlXRJGnnLA2WDqqHPdI\\\\\\\/yT97LrpUkblR1zEqlTy8t+I9cSEDROX6+ekgkQiv84\\\\\\\/\\\\\\\/gqMMSQx6rLwUfwmTgVRM83NyYX1diI9pbMJ9nw5fFKrgA\\\\\\\/\\\\\\\/4gABi2Im0cruJwIswFBM+7qCXnyTT08P2SRsuvgBpMvmr7rUYaVsy80Zd+1n8eeRASxfrD3rvy41mYI2aoo0q4k6aGcni+AelhRf08Vm7VKLkmnar\\\\\\\/pR\\\\\\\/opirstnuKe6LE\\\\\\\/rxrMR4+2z1\\\\\\\/Da+hS5ft0dUhP5Y2brEPSvUNM6+HrRivEI5qQbF4OnsTtT5IR4tmVrtBO8KRUG2+iLjDEpUsywsV1JpUdB13Xn4MWDNSRiH9XFdeyBs1gRiFc+oiZWZa68UeMGv3yeI9VrWpl2ZdLjt4RSd\\\\\\\/S6icO0Dr3M4twad\\\\\\\/rk2ejM+l2F5l2DuxYnwWxrpk3m5LBUUlS5ha\\\\\\\/xBIrUpTHVE3wRh4SjWQjelx1N1vOBbEK3JN7Lk7+6auuYZHb7XDUFjZ3RCMjMUl8Z2OTRIzZTDre1n5zKRbIrHDOWE264h4j4EEYQneqB+UX8Z\\\\\\\/H8ydNDB\\\\\\\/2rrGSi+tAgeAxaAg+JRpXgcgMtNovVPTYjghML8VmP1W0f33fDCn\\\\\\\/fGcMaQpuh8BFM4Mm7BUPlAfvjW4d20IkzUZbFj7eBd0vm+OiW3bGnwkGmPVNbC1vbMkpy+hujk0J9JVygE058Qd1wVORlJlEwHSGIxly9VsBR\\\\\\\/ZfbFQCJoYLCRCoYvhU+CPtBKtc7qMWfYGVA1F+wJk68T4E9O8zeV+hi4DHpdYwOa19kd4I79lI3yM+BkCet31R6\\\\\\\/UELtVi0G\\\\\\\/EqdETy51bf4ueXgiHg+8lf7VJ6\\\\\\\/JVYX5ofzpCve+hqrB\\\\\\\/ojNwfLGyGOMhaoGwlBkljZENmyM8aOusuIT1NWhvY8vvRM1n8NYPgxl2EbHYADLVzBV+gf1iqr0NWxeAYxhbtcHye0NOBz1hQtw4zi2FOogor4iyDigQISYs9ADglhmmMO+aM57fIiFkvJ7ks1KJ4wVyaiyhzhM5Pc8ZthNI3MBHN2KY\\\\\\\/eAo1uzqzPOfTKb756nzDqYkosRktKRBF9sK0cCrcVcYmkijBddjWbWJPiO7zYZBjnV4EC4WZjH1YG0LT65gIfAteZ9A7+jXZ7t3lS+KVzhOkmHsDuJMvLaBEeQ3gGQCFD8aLynBojyyWXOVvWnxkxtaC7BrIejMYFNhY\\\\\\\/LkyQ1033xRkvQmHXQrEXV2MchZcjhLAquhQA4sRoPh6D9ImI7OMvbsto+LVqSrc8EbM\\\\\\\/3PSJG0LKdBLTlU4VUu+4X77pzptI0O6nk7ES0\\\\\\\/4l0BbdAONcMmqABWfhV6wVQi0HzuVI5w+UeX5GzOy6LdoAUIWoToZW4jn9UEUBTGqZ22urVMe41k3NtI7jUZO2VbFG1YkTEvAP4dDjB1rUfN5y8qp1bIIR0vg+l2DgcJl+\\\\\\\/vI50xkcV8M2eDQCAS0Srzim5zznh0JzEnd7zPFcKQk2SsnYZv+3A9MnMI738awrmDuXPhLV6NbIw2XKYOfKpaGgz28pGxbIJbAKDzvk1ZsGYzw1Egq0Oy70Eu4Fg0roH6RNxly18gAnNJurGrOMa9jdfV0vcxqwRK+ExcDKhMpoXSbZiZ8zWL2HEr+KJHUaehssmdypS0Q4wn3DL2wqjzdGFvL2OcyTQVAXwSWARmmyeN2W8NkaeZ5RFpGxQOC8sMH\\\\\\\/vdFoVO+ZmquprHjtp3zDeZjP2Maevenz7\\\\\\\/+kNsk0h8mLX6GxESuLfRDvhCZpl1ORN7MKS5SRoACL8\\\\\\\/nlOjTN8Ymu6+GzcXpw5nNWb+KlD7UGqqI8qzjc80EkLU7j6lwUddPTLdgxsD47oSvzNRe3a\\\\\\\/qzW5fD3loxIt9s2mjzsROfLkogAjCIXZjvlVU4ln+JyeQpaQ3pJbW8Znpsfl4SM+tHYFXgcuGRFElUxbVyetsswPDL8qWqBpV18\\\\\\\/maXrcJWSJ1IfCOwk9uQPgQHxiBPMC7yfPXuXMKjzUbwaK+lC+qSAf7Lk9vqtTyCSpopyb+mbRmev5xVQBhyxDzdTG7yJhpAOmc5SMGHUEv4ESyCJvzXlS4aXJVvZawEJLL4jj2GfFDxB4WPObnFmpQ4DlSic6W6XHGhIE0C0tRjrg0AoJOE8KqmoUOQQcNYXqubQL7187eImlVSNDJi7dL49qwjnR3nIsLs+MmiRDavnCfQnTrjfLinsg00PW\\\\\\\/sT4xAQW0b0eSUgNMRRj0fSyV1Zd\\\\\\\/RwlFNNPZ0M6inZ3xWVemCCMe7+GDL5WPTziXZQZ3ZSWrNgRKU0DMx6cNjWUsF3dRzwTVw+MfRGzk81gVymvWlKZXautzFfYlVNJdtiR8Ho7\\\\\\\/OQrJmJO3nA4wa1G9rQGqdG4A39vdsnelvI69WOJBCq83+22IefG+ywxKVKJMtMXNSpcWV2P20OQbljpg7u+T1vkCucDr7\\\\\\\/qHhjJTRZA7Chm4\\\\\\\/I15465q02aB7k\\\\\\\/fDOLR+djSrWtqMUtBsdxHwRV6m6Gk+\\\\\\\/OVZy4\\\\\\\/U7ILj23tz47GJDsixpJv8L9\\\\\\\/y45vNQJH8HkRPtoHGEMVYwk5kRH7JITtClDdIb5W9z78FiRUH0wrghOgO0nQPDJe+QbCrFnfOPItwnZaZqv6zlKKDwT8rB6UJzoMEqCG0s3P9K1Z\\\\\\\/vV5nTRjENzuSXVCYpyAwcq5ovGMztlekf2TBsbIbtWl3DzSyEANUpC2cQn7SYeltDkMNzUUr3g3HagR3PU9s0BDEe6aLjjPRJ14EpsnVhMpFSnFHrZP6qG7111+kmuGgzW7HsOViwJS0IlFaWTKbOashYgePVjt6wv04yJE6euozfhZs8lC8+fs49VNHYILMpfyR2Ubq+k\\\\\\\/SDjw5AHnB7b9isT7hVMjvxrd+B0GX7qgjePyxU6e+rhRn1Mx3F0RxJodBOH444ep272Kii2flGIvBn8ZKOHxnrKTEjqbviP6\\\\\\\/JL3w08GM2jMb5gIXVhmjZI+VJdxmx3kO6NFSVc\\\\\\\/yzlUiHC\\\\\\\/avcmpBAMIVutcJJJlsI3NPqFQ+8r5a1fizLpjHYSrzV\\\\\\\/nT3iJrqy9Bochzysw5vGjwZC7Zg9KoBhR+UhK08+iyEgKPbOf5kq\\\\\\\/vGP6J4hH\\\\\\\/JLAe7PcFcQGggpfXguE6PWLv7T4Df26q9\\\\\\\/P2I7980BtwxU6C6AcJ\\\\\\\/qVkJ18UKAXbAubr4yAmgaOxcEB5TsCU+H8O2mJi4cVcwjDsUq87AuyjemAyAV7sjQ2\\\\\\\/6JYpw8EjbzcWoSFwv0+RkMFodzpb+Gbp1kiZlO+eH7+LaaSslEkFrhOmoyPuYRosbxayL5CFBAAK52Y8nkXG1IlIMZIiyno8s9uMepYFyQ3TqU6Ky4pugMIy2s72diNmfmXn0X55+e9AsSPOJFCgA3FDXEvUpkgSHlekEOw4pP\\\\\\\/x6NohhofmxSUR\\\\\\\/fSLc3EqrRVlKE9DvHjqx0OK6Y00zUZUtlHaKcfgISZJt937369XSsv79YT1DbxS87RgYUgdmE0eu70XGehLO+BGXMeFDmsOPynd5fSe8aQpxtQ461a\\\\\\\/WE5OfzYhS0CFPWi5PCS+quIwifWHh8y4WxHw5hepgYWVDN4ZRv4edawKijXUj86N6798oovBQOyNAjMW+PlPBpsAlwRTPXVESiupOwBYjBGEPyCWa150\\\\\\\/52OB2AUIcfQbbM7jS3mmdJX4C690m3ooSVWmcSGemFeOf0+JZMB1TEX27xmSpz9QPLYzG9j438TmoY8uZWz1GGXRZNIrUaPA6V5tkOcFQYdjByT+LF77ksS92TUdUeE\\\\\\\/YHzA4l2lnUYmfCqoUVL9pp\\\\\\\/hjMTS0w05e+EpvY3+nydbj3lS23begjfL2yaTIZkaTTMGmZJCxPuU0jG\\\\\\\/uQPmnVPx67AI06niu6CiNyaZLo60i5jxlvnxyG+xiLVKh4QzXjmJW2d6pMW2mtqAEOX5Nj4pgS58tU0680ztw64A9or+4MnX7ZD\\\\\\\/cnWeEySf4h8MmBR4ibB2cbQjZ3JufpgGP3Nz4BlN4Q0Jk6AloLcRBvBlQtVVThBOxwQkWbeHdnhavUMnBvLreg5OfH\\\\\\\/z57X7e5VFXRL1nMZ6naYUWr\\\\\\\/vMZbNGOyLqvhRWPjuvI7lV4eInIHQ2A7kiQXodeMd582+yuOVPMFIr27zn+5JSxfqCaB4N1VerkGjf3W9IW0bQlmmjSVdecPoyEi6w1+sHsyk3nLse\\\\\\\/zwX6iUNo4jzhChRCOzuKH2ViS+7uHTpNYH4pPmrOKOj+FR1iI\\\\\\\/\\\\\\\/H6YuIMVGhYB6qnq1\\\\\\\/ZBr15d\\\\\\\/YvvkCYX9EysRWE1PfYOLsTRgYdHk9e+vfhCpqmGuo9J4j4p76+pl9zcni\\\\\\\/AvAqN\\\\\\\/OMznikrJB3WQBvUPzzqFzxMslpBQ16mPhJvN\\\\\\\/o0jilPFd\\\\\\\/zQlSgGs3WRfaMFH+ewAHABZ+33AVYZf1k08pVaJJ7PiDfJaku+6hAultw9AAr3C67Cecj2FqU\\\\\\\/JqITxwiir0MJrKWsGnJaJXf79fn7ZHb9XP5BHJ\\\\\\\/QMjqKF3CW5y8RCoWA2c2zl6XMZivIakHhHP21hu6PgRvBC6TvOzQi4ys8aW7M6LQ+oz64c6F0jYiuOxtwUKjHmf78l7Ncxd8O19LVBRg1jgYDYdxamKsMo+9OsFMpzEzdNMpA4fJOxTyOLZfQ7a7C+BYfKmyrdOU\\\\\\\/0G3eglFidKSwtCVfH\\\\\\\/LOCnJFgV9YZiQ8BwmkuiQ1fhHYULuSrkNP58EinHF+QFUOqTSX\\\\\\\/pBTrgbQhyyZocXkxnjBAs+Z6fmrP7gnkA5kDhkaBjI4JzDeBhYhqKhhGONQnOPPhMPPSunXUedSwXMtmK5WFVUc5jt0iHTvX0E+OPrOXYJZXUdMW87ZqN+6Uwqy+fRd+3JzdWQTUg8M6qkwqE\\\\\\\/hiiV4GyZIaCKtzu1LyHv1HBa0WDocGg56OkX5XwuXeA8nMOB5qU33SNDDbBj+kzOWv6lxtcs8HFgm5cPPEUTDreTfS94UfSlGz4FdgSCx8\\\\\\\/9HZ00W8SpzdPB7Ng1YCeGx5g5GI12wIYRSB+rGPTjnMWuyeRc0CQrm6MiQTWJfrEjZ0NG8iw\\\\\\\/qBt0ZbkNQGauVthwnbhSaiRFRevhxHqXzmTRw1DYnDNrWfldrwWy0hqLT5Mtm7Bk+XjkvOvP1ZyJ6m03mCAFgcZYveFwIUKqyQTyhlnNtr+OMMAYgzl0TRcwfJcO9BRXUS138556VNr\\\\\\\/T8iuuBKO4\\\\\\\/GXptpalHxcBt8vI73ib119rzIaVp\\\\\\\/B7AhOWy7A4crKBpSbDXfNm6dT3b6NzLGwfshOP0oXd0qz8dc1KuA9\\\\\\\/+EKt1gbjtGXwQnk2rXbxXUGqnyWC74h0NVVoEqBX74mXwCbv1vL971mzG9HKU7OCkFab87KMUGRltovC8+SwvzJOwcP30FkK72P1fPubhesc6JuRlfynqn0s+2BZV+7SE5GA3oOMelrJCh8WnzEA\\\\\\\/NTZRYKZ5gUIFaXDhjcbJt3TXiJk8bY\\\\\\\/cQoET1eG0fc8ya25eoROt4EowrpHuU6FQaeAqF6\\\\\\\/gSaO09lGdmvH26eXIz1\\\\\\\/NqqfA4IdzPum4eP73eQYeu7n9ZtBJQ312XkZWvSgo4l5ZZQNm70obG43MfrpI3IJ6pr7Md32HO62ZCxjlmtT9x7VKcQjsDBkY6juAcnBVertFQ2HUucEud4SaAnL85jUlnd8yu3RqSiDH0nZMvUEsWspn6mHnMB7g16ptTvGQCOZGs+A5SzC8khnBmxtXnrt7PalWxau1v\\\\\\\/gCjw4KkalfHcbDA6NNN3aXe8fvusJwoQPdDUMi2f6\\\\\\\/Y2hfNqAIFI+9x0YkHtz3s8G2HyW5RCNXQvd6XbJ\\\\\\\/Qdp4cgyfMwPkSE7e6UXOVDWAvqeffYQ8VS+\\\\\\\/YOw\\\\\\\/uvvAJcQcrBQ+s1KrzBaTBWUrF\\\\\\\/QuUNI8uQ2rESZtEDdI\\\\\\\/i\\\\\\\/9gv1sbeWUHQWWdfyp\\\\\\\/0vU5OwQfUeMO8YS3LoNRZzuoDpN273xXOlUMMTKHPZjAiX6YWylfgYd0VMkhznh+jXvF7RpO4kl1SppM8HCBKYqQUdup7Ng6uw8QSW\\\\\\\/3XCMGiKgWWGwn7YzMb9j4ftIyBc8jaWcovFA9KMEUsJGHGJ0mbOdEAC6iVulV15oTpL+\\\\\\\/emc3e6qJR8wSCyRF3w14h6Qh0mnMRHDs8gUO7htDoWtbP1HRIWt3S02qIQPtw1aFZF\\\\\\\/UxGR\\\\\\\/PAhpbx++XowRODQkBzef1h5m+O9he6byEEzBcMiRLuhQY5Rnw+xts7JqIFVGlSXBGobvw58LTwgH+iG8qh5H2EE79oqR9YEvqe1HHt6KVOVXegAHDexL+rDs0TRAuzPFBTIryyNIRFfLikpwD4I73hMsohYvx84rQqsk7Fq27FdDDl0tZI8d\\\\\\\/y8Iji8TUoFHrOhOZ7DLJj0cGFXWOS6TH\\\\\\\/Gs6P70jBz2y7F207xLdFIVPMM\\\\\\\/9ysAC7aWy5B3wboiHv3P\\\\\\\/NYyhYCerZ732q1JIKkHiQXZRiEGdwh6MsVu4LbHkK7gYdrua2avWiq6nN45RCN9KrTgK1QJMjieaW+Y+ks8czdB\\\\\\\/rvlMmvU6KF\\\\\\\/BH6V3ue5ZFoTPrPzH+0IvvhEvLX9T0quRdd0gLqmiRcbduNvaJRoJowSolQEuvO7mdtr4mbNhKUXMR5ZgJhmqIS687aZpqEtQsjTnBLQpIwCglUGKTX17RZAIeywhh4n2nv2NcUZPZvHNt3GExLcs\\\\\\\/AlUtY3wddfPYCTK8h5P1nyTH+cN8L40JFf2T1lX90UMPMGm57c21Hpom+2GVSw4TucYcP+Qy1UwjZ0W90sC3k3X05goKEuwLQIjSfEjQf8bTxrublg5HnUCWSgDfb0yOmVZzq\\\\\\\/feO8VvIJLKw5zDvoW8hX\\\\\\\/06EKw021\\\\\\\/S7TxaJVngOE5aEBpgfKz87el\\\\\\\/BRry2snBZRHZ\\\\\\\/VDapIgGwZqer6wv1a5euJQF222qp\\\\\\\/5xkt9AXOvPUQXboRAJy6PUE3pW8aWZHvNWhXGlK\\\\\\\/TKUcP8duPmXtZocrtmDas7CIxJZ1CugNAWI9G8BZevbcBeHDaV\\\\\\\/ebTBEBawipGvhqwHNA5JS74ADCKhM9rry6XlXYphWxuVMKfcLlefGNbYj5dtA8bXgvg4o5sfa8c6SayLF49f0YiZdsYtYxF1Ug2o\\\\\\\/OH3BqlNqWtoDGR23WiN1neCkDpP4\\\\\\\/8fXLrEJ8mx3Y8q+Ddcf\\\\\\\/Q3QhlHGjclUNcjqE\\\\\\\/DJV+6H\\\\\\\/h18btmu62ZhFpvuIpSESv1oK77uOKflNBRMQGkERQxFMJiV5yr8RsmXgsBXoWvgUHiASVNmzTP\\\\\\\/EiMcWjghclqrl22LoeLxmF66oPF966lBXZBJuz\\\\\\\/nvTiWV3vUXoT6W9Rbe2foR3VBfrrkTv+63JxVMOfceJRxGkRYP\\\\\\\/nBTzYi1+F2J\\\\\\\/M9pdO14ai9hDm1wAKcPvcazFpAjSETDPIpe8ycTvkQiDhcq7Sd1uthNhJ0STl\\\\\\\/l8lGgT8YQyGP6eSwupqTjON7+2kC90RMjx\\\\\\\/tMVANpEm8PP2uO\\\\\\\/ragtWca2lR189CoDLCXBgFA0Y+\\\\\\\/CxCtvzDCixFkIcVekBHiWuRj3e6covOk6cvPyHdszkaJnoSFrzoQuecFR6LkAFhb\\\\\\\/7jAyxpCPlgeLuOL+VtUw0EEMKORO9nyLE5ClcP1DAnK47YZ0t2re5xirlnhCG5FLFXrtu30nNo5DdOocGlvEYYUtDummTX6guClQ4Uz3jtbwQV\\\\\\\/uTaas1b08a1L6et0ExtWKhOhKTp5mpRHLplI\\\\\\\/ugmlVQuWL+JXWHrPzb+4IOoH62DKxxT5FyaVUpXGk1Y+g35MTDK+ENrhDi\\\\\\\/HjTxm7ZuEdUpVq8nExZJSsYk30UZL47WlDMbq4gXNkHIHDSOn3FT2jfKfaUzi1TICbNzgNGyFu9MT9bFVTcNO4aG1bDquFVa1zDvTydyrJk91LELU1SFerevdJ16DtBlN9JIyc2Qms5VTDI23fkiYz5fxDR9dDJx6E66xFTaIEK\\\\\\\/JAeAFGjP5uKb7d9OHayST0PudSwaiu2fzJoCp0NRbbnzVN+StmDKNEnTgTGDWfEvhNsQ1NzKMdn0H7Po4XutYe4DinHKE5smB92RR6vk68D2noYEaCXmXdEDg\\\\\\\/89PGzxRz\\\\\\\/+I+LzulDLJBmUfDZu\\\\\\\/zdbYwrmYyN7rbWvYQt1fnvYA3cUaVwYNeYuMraQ6jJfEi90379rWAwibb6qKdh8qDlphenNi5Gooqm\\\\\\\/JVY9Dd7cNzUkzHzpiTOffbBqerlkpbgmYITZJT\\\\\\\/Qx898M7Wd\\\\\\\/+IqZO2OBIxKUsoTT7dlrMIW7m5G0nzmgXoKgjpe1WfyBWSapeyABTUQtelx\\\\\\\/ZeKb5F\\\\\\\/WC9pQmKUQk0BNiwQ7RW12uJ4vjBm4j6SZJqeI5HSX1j21hsk35moaMM9+58PgrKHdtiC+m4oVvEZwAWB\\\\\\\/1k39Nzcgwe8swrnJmdIJU6Q7QUYHRWLn5Ni74i1G+xRZqj1EWiEAQPw6LT3sarr\\\\\\\/f3f1Yb0bfyE\\\\\\\/h8JtE+uc6\\\\\\\/FC7CINhTdMkZM9Exk7M5UKp9\\\\\\\/Tz1ZEQk8lPqnx9ShE7y8nZ5y5Dv24\\\\\\\/DqUc+7S5p9sBxoCBSemZ\\\\\\\/Dz1\\\\\\\/z3Ue2SDV7dr0YkLFKTmyGt8c9YLNw55CNnAM5Dt17ieunFIwPjdIXTFolJnn9wKG8RY2yPj83xos2o2L45mYdhMyTD4o9+FyTigs7OamEtiNUNiq3aAF6EwwlRWlW\\\\\\\/moeVeHoCsaOZtVkrXvrRcbf3T8Fhv3od09mcgcrKLJCtv3r1UIpiHUZudeJr3aO85RPOgO99I+aRJKk3PTYsdGdzRpc11YHaQ9vfpWy2GLtJkmeFtP+MApN2DY4tBn2y0MyTieKIKs2kWxI3HyIHrRNXAZo+Deg46HcPfTqYVOFEh2c\\\\\\\/vAqsGPFKqS3LA9FmMhHSmnXTZwt3gkM7\\\\\\\/zTPt\\\\\\\/VgYLRBg3p+KhJrrhlZUY\\\\\\\/YhRKI9HTXRAnrNvcIcqYoSjgX2HOL8oZWHi1H5VES6\\\\\\\/v31L6vs+rpK1s3RSTdswhvXxVHVsjshd3NhgNx\\\\\\\/nHYx05EyMESpj+jMipgcAwyNp7OfYO5Fi8p5JJGQmh5Q4oitxPwJp3KsG3U2TE2vitO963eNiL1Rkydq12hVbn5NbBJYpfHuNCNa\\\\\\\/XukYRnEcb0fV9AEX\\\\\\\/76yP4F40UrHDQpwCfYnn\\\\\\\/MoU8fNpff0RtIWZJsBNA77LUR0nfN7w0RW\\\\\\\/Ql21wlizbpDohJXUk\\\\\\\/2AZAdPFE6rUcAGkfGr5ZHfFW7OMycsqTOuJVZXqBDPLqafyLxhaU4o+IGJGgXVK+oGTZz+c7qo7ExVHD2P7Q+2B8Q9MIgKjdW2PMWCTBCovqDNjsvTGLzzYoKwrZ5pBExtipP04NBZ0nbfw+yrclSW\\\\\\\/XMi0KYk4zW3WZGvnPKz4wE2uKvHg9b+ClGp1c4d86LOzrsE74AIX7V2U5PmQtcrKUjE1LZXrkQgr3HSTJRMPaN2z16ui1O+v6jqJ3Vy8+cwm10zHwoAtWLfr\\\\\\\/pW1o1iVrEaUMlyWVWXoV\\\\\\\/ViZu2eowZMVsu3BSeqPRipAa6WzsI7YB8K9uiFRvDsK\\\\\\\/G8RROGNtYNqtjhZX9ExPXLIY7gkeSivD9Hyxj7z7iAQXMvRwTcyBQYxqa72oCL2YuuERr1\\\\\\\/2tWt7\\\\\\\/D4Q2M8m0XhoKVzsuREL5WGHJzyiItZ56RW39d7kSPve4FCcH65N\\\\\\\/NSv3t\\\\\\\/UHOxf3o0dFN05Dy2EP9UwtFT2Ljuyp3qSg3fPnabaRHOc+j\\\\\\\/jsiSxhg1UVIKoBYyqG9\\\\\\\/6aFbsd1g4uYUX2HBZ3WWWRDVEY2LobwNRc\\\\\\\/RPkrHdTcnlIMN7wEjiFM0rRUnfwJR6dC0ExfGLWflMzBTEddA\\\\\\\/N+27VA\\\\\\\/8Lsct1BGlRL3GAS0iPvHfXPIMDZ\\\\\\\/J86E55dEo1iqBIL\\\\\\\/qLmEm05trYwsvO0D8+91u3Do9cPP+JF9l6yGbZwJS23UJQRUeg1V0sDNClwLz+yHq2K9+SMCsY53+kuslY1sPORNFe3msvYunsRDNZiQmuEKXn+DwmLA0NZZSnBzjArOZX7BKpcLQBnn18hr5d9l\\\\\\\/RWedfU+zFLfFOkjzsqOHfy21XZVbNL\\\\\\\/KOg\\\\\\\/Op+LFvWQryMdjLeP4G1Js3J9ss1B5Pxim5gXHBMCXvB8WgNLst6cpZuQyZiJ70f4tM+w9pzE2c8YdlTS5xpohQFp0YpAA+QoUFDnWPNYlMEIrvmIs4SwbCKtQHGttTwMit89Wgg6\\\\\\\/QLBSdZDMJBw5d7BJBNOGTVl1ljSXx0Z+B22rdGyDmYJONP1dn1CY4ZpgT64QJ8R0PziJZr42xvULHdJJx2yu1zlUgCjRwQ3sRajBUDO4jnj79Gxhg+FDfPsUsHYMsnmDUe4y5hc3EtwJi\\\\\\\/gS6\\\\\\\/ssCZswxVn3+9V6em8Q6rtRhmowE6OsB+cjyXTzzNZl8Ph8ujKZyKNGzJDVxxwj9fGmqMSLuvOoMhUxtu8H+cRJOs0Jj41cru+l8twJT0dNwSW5IaoXs9WDym5z69ndGW9Lcj0aooDKMpgW21zB1kdpQC\\\\\\\/egBufOsAnhwP8uSYicMBXwuA\\\\\\\/Eo8tYn88XeBB6+9Ig+uZ8l7n\\\\\\\/TeJELcsgoMc5iFBwjYuxXQgvVdVaoIWUKToYD+RpEolliR\\\\\\\/8XfjsT6N17iqxBg2\\\\\\\/uMWLBLao7qn3d7wzy+31saIKBo59YyywB8g9OdZaHPYxYFoHC0o2s+DKoz8EzdeBPqoKDNbiBHiDLd9VjNcNRujV7hFrhwj+72vz3Q8yQDNySDHWWizoaCZpytSbpVPNQUv\\\\\\\/qW+JEauCgGwfVIAeeQ44jeDPlzkzO8TQuhQ5bjC9se8QcubKPPorq+0NGMS7X4ySYxM9texS7rXcTEfOgwYxYs21gmEJiiMIUmwAJJhW9uHW4jC6wuAbgX2FpTlQbLo6ZIaefi4PtYX7+ZWuy\\\\\\\/NsYfWs8HW\\\\\\\/VpxGLSlBY83BOY\\\\\\\/CehpHg8kek25dRk6y9DVBk3aOYJINc6ikAKUlhcPvPDTxVVyh2vGrc4VhRDdm8Cp4o0I2ODv1yYRpBPMfQuemLoxXwK\\\\\\\/nkE6wCZc3ERvAxM49O0o8Q8cNcZ82uGdaIj0ULV67YXjFO\\\\\\\/gjd800n1skD4FbVIQJeUNMrvuIWHYqCcuMFOK2zCxk8Uh+9p+DH9+HU0cRJKOKQsZiv0YXJ60hwndESo2eAoBGMX2mRS3ptGjMfQyAWu7qnzgCMyOy+r0TtA+B0RyNbX+9Rl77hUho53E2ViEB2wwgUtgCOYwSpTy7zvFiGVSzsCXGlItQO1ArSoJOZ9dr5blabcb5lmUdMSfJNn+hjb5tWfdQUgfrpAqH7uKaN1D8JZ9qr5zoIKZUd\\\\\\\/tTtE8Sg60OmPvtj\\\\\\\/4p4WQsb0hWc2lQ3jBeu2GE9bhaSYyVCjzgKWWl77svAtBZYG9JaxhID2Iwpeg1YjMvbjUR1L7ZCH26TQzFAWjtcK3VHfbVZZlyriTEUc\\\\\\\/auyCLrASuceJQdxXAmaYg3\\\\\\\/RuzmHNl19rC65E8e8IVnm+g7y938yaDhWRPxNHxWJL2ZC5jcfaGjYWM5WQDyQmzZBHU4D4s3WequtFm7OR7oCKruWj\\\\\\\/uxwll+g+9YeXIp67tyAY9edNZs5zPZxJqQeeTKSwqQuN508aHNooV3ZSUnGODyt7K9\\\\\\\/qjKhE+OfAcTn2hQ2nlKHg86D\\\\\\\/OYeUnkBiqvn0+dHNXSPBUIk+DJX4kEm5ow0HrRXnCWbrMqlfN16QOoQ\\\\\\\/xclw5AUmPmpoOUfhVnX+vuElo29Azdh+RVtgfQVmtnaQ86XyJXYv\\\\\\\/rWsmH1iquQj3nLNBy5UAen0zouVD4gMZgA57XxLZjit7\\\\\\\/qHCt\\\\\\\/\\\\\\\/ZZgpaOEXm1MnFFiHZTGSKk9S1tRLzxrTQgtUbZahW2bY1FGdgELbiGAwMXkOwYQwucSvidpITstc1ksCVXmL05YG1CWb6uMCsBiq5MY5v30KHbeYRV0tbpLOC0HeaSi8uI9rGgEnsE6IOPCUB318p5PY90uXKz2fS6zICKF8YUk+L1UTN96ipLg4HyEKoLyHBHjERXxq6FC+wbJ7hHq1EtHL0fHPIsDEokwQ40USiO6iU2MFzKueaHCPDofTspzD5bOEQSw9hlsJdGLZjtDhcz3cL4jFaXm2a9t6eYmr7nbeID9wdbi4UeZOo5wcpq+EjyzkQ3JfFFSxa6fKyD5EJ4ytfvjyKaBqRCq0oXbqVv2AuZ0+bSCC99BiBNcGqZ9uumhDMxfEXTAuddDYdE9hKBo8AYrZqxJqlDe2pK59fOVfQ6m8F5oS+ITfr60l2f6WFmjk9emktEH2HMGxzpcQUaCR9vDNkkBpyqDR8dhDapjfUlimWikGFoxZiBWs8f\\\\\\\/gO2IZTTMkCUWHEbFAFfCJQexEdh96PTrYYF3Ftds5JDrs7if7H7uu9ubWo8kWFNWjmaHud29Po83uu29JLb1EKf+tCSryffpz\\\\\\\/a2+tkAQMGhq3FQJ64HsRDcuUM9zD1u2xLRP8XQJ\\\\\\\/Y2L77IO94txZ8tkixqUgpc4a2IHLNv+hnuZZrPXjVOI82Nbq\\\\\\\/0zSV6iLH4yUz1ILO8GgG\\\\\\\/wGqpFqes4i6ftCAGYpneMTSq6Nk922CVsRmTLd2yOAmR3rkojwBRAImx44EIjv7M1ma7AKTSqiZwznhhsE6n2x7Z4hJQqzr3Rfok2z23o4weeoMYmvQ3yfAKjvSbX65hAcV0ThIvhjXDLUVGREPF5LVUDzC5Ht55xewSp53L+ZQQlCy7d7WohNWxp5UEKL6paMHc8NiJq4I3tVE7s3R0xeYSh8zqsWc0Pbn7gs7D3pZccOxfZi+nPONmrFbozr+YLnuE38\\\\\\\/ERBBlgMP+FzT5d8gCg8eOK0Mv0AB4UtyydQaXcZRjPiRtoi\\\\\\\/L7LiT5h4pX\\\\\\\/Yd50MEHcVuFKSi1QpIiPqdpKSDlLy484mMMZM46DagQoIVRCnz9QBrXxbwuqs88nXi3B\\\\\\\/fnJIqJ\\\\\\\/kif\\\\\\\/TylnrJ9bu3RpvEGv106aUCB95ICBPDUBv5sfigIsmFAvsO1EurQX3ZULVBSogRkFScKX9drHsaYwl7S2\\\\\\\/OGdyEYmyROY\\\\\\\/NlottpmMXOiUX9kJRDHDZCA2mDVfEBGv0VsifF9pLuNhebskSdKulIhnZzYLmFQPRSz81Bx7pI8jKEuVxMzJEIgag594AaB06ZHmOHHU90fb\\\\\\\/YjJQR2oHUx06nAFFHhSq0K+++wD9CqeKZCS0JmJsLxUvfmdLvwVuDGO2SWi8NLA6w2nrNF+Qcwb97LwZIboLj8KRcve8oP\\\\\\\/lvmFJO\\\\\\\/7oawovPaEily2omBpeq7mxGczXFkpwBQT33cbbAWgDo927PGxOwy5qwqxXjEyTY6jUARYig77dRboHkgrtOAD9wNSxJhXvMmeAzsPb7bXYyrzU5zHQODYueT\\\\\\\/3j\\\\\\\/xz8IpLWqL0sHKBVx\\\\\\\/0yigI8BKjrvio4IkXF9U3pTCRBpS9e\\\\\\\/QsLuDOKG1MQcOGYUB71T63T4G9uzxDsl8tfJQ4k\\\\\\\/z6Fu7+w9GcnJ4gE7i4MNe+uYoNP9hj97hbNetLnFX3hiGkVjobRd9n\\\\\\\/70xCI5cpqLx8LlQLFuVo17KeF+oAkoj6w17kDimC3ebyQSFmd7UGvEQIvUwj0fOWgxE9GcJ1PN7ro1C7N7MhiDq\\\\\\\/ZuOGcQ07UkIjrJ2F\\\\\\\/OVIzYwD\\\\\\\/M1iD665jrtmLocuo0QseVoeNA2r9cRHv04uXCouEOIKKnjw0WzAhLBFntZv+hn0NjSNkcMpDWEC2G7eRb5fvuBngXYTO6MW1K2AAnFt9xrllCqPU0gjj4uM8t685i\\\\\\\/Kij8DsMsHeRVxuXBfdUsImwzCpASNnon3oM4OsItxTOk99+31GodtOVrX1CI9a0hKfDQkTmcDZGXApEjSg2Y6VNVWlphZlMT9NY1SIc\\\\\\\/zX\\\\\\\/\\\\\\\/omg3qAG8bRJp4K2RBYTuskhHGsbKS+XkqG0JExhH9Zx\\\\\\\/AL427xjMJ26sOc74iJLD2ia3\\\\\\\/pXZmlXpLaNN2p81PzrloQKMsIjomYh32eRksEjCrrmCF7mVZMY1A61eBgxwgLfhdM+i5ia14Y9zGRmXm5TatMeyZFG2xqAJwe5\\\\\\\/IfiGH4ZK+47zxIpkNggjUVsHGKNy8HlvvO9bgEA+DnbLcPUusHMk7CEASkjfLPm4DfigMztO0OFikdeDaAHyxVoU5uI\\\\\\\/DkHCn7LTmld0Bnliq2C5RJHogmCj1l6Okh7xwO3ew\\\\\\\/YHtbH6rcwy4IQ1x57btT6oDtMs1YsAyCA3svIbT8JbG6n3T9fZjM3AFDS72\\\\\\\/JnAcWr6z4D4PU6xxK1yATB7nsTVyWmH5zV3K9gvTat8+7ZI+MYzAaIHAZG4ATWv5n8CZgU5rwYLNOF6klraYDnmmFQ1ZxdisAIx0qwsSJs05YeOXR6sY5m5RejxWY7fcgjXt33gfXPMZuMUN0QG0LG1Z5uk2pI8Jb1A6S1wTJClJcwZVzvCe4tSjVGycuIT5mRfYGmxWKKtYUdeuTnOxj3oXtFxV66K4Jf4PjHc\\\\\\\/+GlSAybEbTNJWPZvrISgHMZb6kHtIwmQjAAw0dmcgZDzuYp2KuTWlMR2I7jEHscPWxUetiMsmqM4je3f\\\\\\\/ekjpWQ8EsggjwpA79NA9Z6y58GBFcXIK7jUPh9H0b5H9umek4kjalmGzwoYVrMBBbauupUTcHM\\\\\\\/joREHHkX8gT2sJAM7T+jgggyp0v7VC8DQFbKDuLWZ0P0e1G21GHz+dvfNJR7vn\\\\\\\/Jq6Kk17h8cL9Qw5BthZvYvQKFZZavNjcN2V4zuweUuL5b8zJwGUE8p\\\\\\\/2gZEkCXw3Mw4GzBvLjdzIfbYYi9YF+QA0zkLqlrLfKkXJiCCDYj6USJEZKjeXWyE195NdRHeraMqbatE\\\\\\\/Q6Y+aRItT+h53nE0eGjSsj01iuzgxU9ZIEwIVA6gd0yPJWppxC5\\\\\\\/Q0zTB0QFhJJEq9c1IEfEwNFyg35Qu3hc+NvPmCkoyXuVT8J6TcB2gOQfu8aiknpi9OCEg7ubFvRGpwgtvyEuxvzEzZ7TJXKqd\\\\\\\/Ia+B\\\\\\\/2wkb3J3HIJnRhAmR8YeNTW7tRybTKtxdLiR8bkpK9cqaZUsu1P0hrStUWIcIxMuZKjHyPRcyYOxj6wBw5d1VP9u6IfZLM0zN0cPHQ\\\\\\\/XVdmZr4LEkBwlyVZAAWaw1QQK+k6MOtaiLLrtOjiHTj+Pus5VFXKUgQiEjsvTiuwtwUifm0zh1KTaqhNd6vYIJOoX49e487aMoaHQi6ef3T+Ky3hCvEgJI4K\\\\\\\/MXpxWo187KGU9enz8x0C882RyB+8pwqdIRkxaj4562qWG6iOMQOpX\\\\\\\/g4oWFcKOau1PHr1t5ZIiS5CmUzg8lOgtqGuHzneiEBVg5jRS+haSSymvmLHXjwgG6NN1J2ceAwaA4mbkGA8zI7CzXhBN\\\\\\\/\\\\\\\/6+rj\\\\\\\/9d3j76LIakpRDLDXXl2dCnBT+hdAw7L6g0TnKA+NdA2cfJka6DunzsFeX7IMosD083bSdNn218+pnHirxTpe30eEhcJiJ7ovGexB5qEa4AdXcvAp4w9UfQYlmR4V7VMb2xYgqZvXVpRq6o1G7bAJyb\\\\\\\/MRxOtrP5pS1cjAfW7qKWnlTTxJs7y7tEDuYMFIngzhCEVHK7sc98PFuoEPqXtWy5Gy4Q\\\\\\\/cUmEGX41S7PAlMIZZ0omPNFxNrmALnJPpJjm0AIwwrQyjjumPk1EgoosuJ4yU898JCrD4EYkgXhkyu6m9JMYG7KKxM7A+vac80QxADEC\\\\\\\/k+LmvDYG6BkC2s2LQo1OLh+ZZcF\\\\\\\/gCQrVSq0vKv\\\\\\\/Hu0BUreZTevJlHrtbxqJclOnslx1nGhFnmRcmiOgXk+aeQ\\\\\\\/vc2HOzUohn1Xm2a79hebIYJqLr1tfV+FVvsXkgCkHy\\\\\\\/Ssyc\\\\\\\/0+KfI0aBDfhJyGU36Vycr2hTTb0Ba47MSoXHX1FF10CUmb4a8\\\\\\\/Ka0XZ02VZEpZuwqQqVDX5eHcjTtzAobihDO0S9WON6YHSSG8Vy5T69g2\\\\\\\/gTa29xBvLbhLgz1Jr1Ks+aFl5buxnLKMtG06iEtSY\\\\\\\/dbasvO1E1talB5b38cAXkWXvaOKIhAL6fvLcUmVBeEfDye2AY4vaqf\\\\\\\/OUVbDxeKH3d188nystM1eOzH8ENdzHZSK6AKLkqlQtacsq4DgALBXZeNb3TmLF26WMs7lPpC6\\\\\\\/PJk3uvQih4BPlmt7Jlc3Ycg4sdaux1rOkW+aXkTUkvXG5dqXSspjV1fKBgQjmb6kTicVTDuL4aBkHUNBosM8KhtAsC0y9id4DIdgp4pr1RTfjhYELRZ2b9K+sWTak\\\\\\\/a2I8jmIply+fScNBnBLaG+\\\\\\\/\\\\\\\/UB9ypmuiqZXcHfUG3LnX3A3uggwfkBGpQn4eh+6tkS8ktaN1pnE1bkV0tLIkZD8Uc\\\\\\\/JmhG23fRPimv+ry4U+zKIwcNU0j0n44TqE5NmsjXN2XXx2xTUubrqM41y\\\\\\\/eYphdTQvhTkVVUVarocdeE874kh+aZdJZB2N8bpVWZUszegHiyT\\\\\\\/JnigqASYWqrBnGr2X0pPR9xQR5M59csU+TC64eMhM+JAvsVhPlsbUZilkc\\\\\\\/13rQ4pD34ZDWvPcOWxA0E2seSWB8DchU54BReIX5+u9CNnXPKrklIcAgbN7+4pRxO\\\\\\\/qLVagmnodpTtUJxCuDUynDtRHSP7imRtunKKhUjHK+Qz2i5lLmBDr0zmjcwmGO77T\\\\\\\/E+wQKhp9AezKO9FJMS4k4nf2bQg2U+7M5w34\\\\\\\/\\\\\\\/Dmdxe4Gl\\\\\\\/MP7abvou30AXsY4lu3l+fXYXuFwKhDc4lkdUJ4F7Bbv22BFKtLAzfgXz3dO8vaduoclocKRFCZXQaBs6yMVrzP574zUbtYiiuyouPHBk9lry4wQcS1fmRe1r3NMWMFxih4toQOX\\\\\\\/j+HUi9bWnyJfLzz8H5i5CY5SUQ8FIyaUj27AgJp8dQmVQmmjEfiJGsSBVXZSeahn2xScB47KYwrsvU+MZh+DwKLxSvuphmU6V0zZG\\\\\\\/nyr9SMDqW9IiCz7fyJdDTRTmKIuDRwl7uVWinpvIofPZbJDAmS1nWMt7Qaizq0iIpvUZA215QQgspfQqDdhoxKuc8V5Sam1T4TmTv1cXuEBLhg2jGVMiEazAv75sxJDp8fbZln5uqJMJj44b3yOpifotg1k+Cxx3wMCUdX4Q7tKzRfDe0nDWAqP3N3qndap\\\\\\\/bW36LNL4Qvm1JtfxLO7fGdQyZ+98gWAVWSJoJk62E95P32ZXGDyLDeal9RmNdle3qSCAQaq2PGscuLLDE7jVoRE\\\\\\\/2nA7KKCjNoaER563nVdn5R1mVhOWQsYOfkltucRrlNXkTwNl42ArISWgD05R0nLuol7\\\\\\\/thHqaFOhs4BXqmrVjWKvGUEpa7T\\\\\\\/dECJ2isQ3ld6iZhy10QzbCfawwRpjwODw3ZoTzulcFTulfJAxfrmCfD2o2Fu8ttX7iPKioCh84xDbHB9eFa+unpQ5g8ggRhRzKWgAyErDqivW+3p0hBLJa9V7RwXDQFTlAMKwEhxfShPA0ccZeVSKC0eMxPdN+khD6QzNGwbLW2bv3OJj0xlutHv8Xca\\\\\\\/pSIMNfKBjffIkrPANX3EwmPnpX+j2BMdpZs9DOM6YDjAarAoJCeTZJvu603obLfgTb3iXpnfFw0gaW8quEDy523Rk9u5SjoxOdDZz0DLPhwWaszC9IXLbLJy1w+OAazf3ww\\\\\\\/SCEk4HKxr6gyCe4MXY98puBflmtgDiJakVMdOEAc+4TlZVs6NhBe3N3Ax89L5pom1cmpc0HWvarIY9D91bKdMMkmULhlmp+gmUxi9OX9D7NFr2CI2jcL0m9TrKukAYqOKsliliwzguvGQEV8zjUWJJLlIPuV8gv5G0xMYQC6NTwk7Lxi153uDnDiuIklLd0eoufd07ldQRmDrHAuZ3f5KxXzidqW2IkmMzHogfjlyqMJolvdLdV5I0+UFrSO03p69zjKoLHuDWwkpIPDygMHgN1mjjNTPQinvFdhJlAnID6Mx3rluPTypu2RlbfuZkf7frS2tN4c3VUZ9mz7lDoJoED3ATlImS7k4Bzo84JIcjpEiSDclsiNsXE7wKH0Whi8PmDGw8OuLO2l1thbWEC7AfRAMP3CIJHclS769CuDK3ZcjWbwa74IYU+nBslrG+iMu\\\\\\\/V\\\\\\\/0tdJ3fJuUMWeOZ6FKot6PyLtuY1\\\\\\\/MWqyJOYPKd08Q9ejaYTRgfaXX3rW5Km1grT7x38tOTEPeGSMs1g1elp6Abd3BsEhnqgrc+lfK34HqpSgfAAeTamKRdy2dvxBzNYtTFb10xoB5Y+UI8C5N+H6Q+M5V+UZfMMAcaHIdeThyKirdpROvDQS33aTwpf+umuTSs\\\\\\\/fvSYPkbRFktM6\\\\\\\/iGf6cwq\\\\\\\/AcdYSkrbF4NZ3ig01aVk9xPlbPOxFQUvYCu0nnHdkInQYqEXcCku3Uxy+n5jQd2AUGQEcXMZQCVD0NsnpxEbXaGThv6bPu0JqQmPgB7kHICsZPQYp3r6vPe2QybkaOsclg2\\\\\\\/nRG9rM9c5cm8LZE4\\\\\\\/lcZTUoQeVLZM1fkStZzsz7ghwPCtJSWCvI+kL2PQ508mtMTTgMO9kbHxPARL83jcT23sTzfpXUC9dFXgCw3WZjqiFJTa2c0qm1sNlb20T\\\\\\\/3k\\\\\\\/8OwAOECOHKl5KojH5q32bbu9Yb0DddkjOkc22UdkSjTzK1\\\\\\\/k1HOrvlN3PklQlQmsa2gKye+0bFHtnixvpf6Q+fBTdHgnrGCWNKIcHveyH4uStYvF\\\\\\\/lapnz+IDu5Ao0jxcKp6dZQMuFwSPL4rexOpoLlaWAOMWh5Sep3q9kJULnK5C+9Ie2dakzymgJn1Rb5CWa6EaMLRoEHMwcUHnGg\\\\\\\/UNCIm5j1kNY1v7iv\\\\\\\/XfgD\\\\\\\/WdF\\\\\\\/Xqg3aN6F0fYH7HVr4Itymii95j3FYTCDzdtMH9aKeqZ838U\\\\\\\/R5e53nzsZ5oJEOTv5mhXzIB0kxHB1zOafvbZPoWUMU+gwusOBMD8rai1gnlD7wgu03LaSc7503XG8P5rgHQKrXgF3nYNEzowerm535M\\\\\\\/RYk0Wjum1pMns7KIPU\\\\\\\/ObLHKpOcQykoaSusRrXaZJfrgo0WCyhXRYmpXtFYqB3DFtuHnQ0PC7ptwbC4cjYxziGeOjUkUjtT38ImhTwguIUX7DPwf0EIscdIT3KL\\\\\\\/0sNMVS6SVDEj+JW1xMy34fysRJM\\\\\\\/WxjAkYDdAEjdd5z+xIpz0O9qG5WMCA9TJ6Vsjm4RAjKVRFZ\\\\\\\/8IMcLPfLu9q9gL3B+N6X+31H5G5jHiOlcWKl2S1kwY5OT0ObydxBV9RCkWRO+afp9O9sRJ0XeeYslW9U9sS+aVYc0szf1NGD2ASbjHIJrH1MNbfgLxegKrFlLEccg\\\\\\\/UbKrNJ4WsyACIM7AANJ4DsRD9OTKtFsxNZ1W5RKiQS+Wb6jNTz5V0DA3q2mKf+0OlYlxHEnvV5PDd6DuNNsHejBhjgFBfHdC01OETszBUkLLFBPIZXdy68OpTi1XephIT9VnoOn20SVRURnCG83OnUWqwDKVdqlnk6VzvnlJsfGkzh7H8xCNj61w\\\\\\\/PU7u1gn8Ml6dZjRHxBkBg9f8QepN5MhZUXg2EHp\\\\\\\/lPILmvIj242B5YFhp5b669bPhyxoo3ytFpzLYleddFZdf1NuUioYg5yWuAhtxhE1Csi92gKvTvHRWofF04d3UXkRsRT+Ci5KfO8GnwJPKlPZdBk0Tw\\\\\\\/+RksHsi7R+ZkwsXHmCbmBONYpA9hCNoWOMDMBRx\\\\\\\/ORDwhOINxKvoqthR0ZMvOCF99ghv\\\\\\\/ghiBi2K4hUc6gtXC6Ck16rKA0LY0hZehk+3k6b167M31CjGWpymHsn0xsBVk8gbvWQGzL0xwR2T9eRExshQ+3NZbjDab+NJ0GkKisCWHp0fD+oEJGO5jLqg1XVVMi3iCDDAAl1ngmNUVVTFGgq\\\\\\\/PL3kMuaKtO9vSRfySiBRC4FuOP7D3JENOXevzSV+vyDP\\\\\\\/re+9MslrDOsOKEeaYvcroZX\\\\\\\/1xQd9122YeHtMszpQKiMLd\\\\\\\/ksQ\\\\\\\/kKbfY6UtIKbKlhHQP7lp9zeTJlehz2HLicBHh5\\\\\\\/mqqbM+JDJGOifelPHmF56Qt9FaY5H26XczuwjsIL1dQUZw8FNT+Bogy1fDx9UvGrti8epvLcMWGIKgd20Owbiig7Ca0Ats62jLA4+HNYHxc4J7tFfN9JF9qeFKVZJ\\\\\\\/ozW2cL1s+f6jYaVCLrmtjSw1RZPjdwuhTGeC1XkU3WaaqpWtcGIz528tNGYvymFn1bLv8derMeGisIR+6AP3UwbB92+nT0ichstb3SOwKD+sC3Ai2qbDAHTHNHC\\\\\\\/sig2f4za67ClHCcvab3k4H2HpZSJsBHR1yRjrio2TNWfsmK6xzed2Dl9YlhihAmm4n8AUSszuQ1O93J+goqzmkZlj4pzbPMxTQ7aQSJbOWnksqjs9ZCOO6JQAKCgY\\\\\\\/h79djH9JFAaH+kqLGUVZ3CMGGu8rj24\\\\\\\/viG9v6ON\\\\\\\/m2XFXVStjnRE00P1RCVQWEMGRz656kALsvvlA48lse3iAwUTl5CfqhdjC+p5YeusfZejaZh\\\\\\\/r0vjo8x8RH+le+xB+PzsQ8ui1KWc9eo+vug01\\\\\\\/WpMKba1GYo+k4\\\\\\\/VUJYwYm6R4v9rV3aLrfYUx2UNg\\\\\\\/uxhX+aYQaXTE\\\\\\\/wZcEVjtcYkbJVCEv0CfL\\\\\\\/6qGijj72SgKUfGe0mt48JpgWfUnp60PNxOJZPK1I5ihAkCFOKCMnVXv+8B7UTcWqnWMUYHF28O8ZaQcDWJCX3cBhxIULlG5owoyl95pTAOVDQmwNggrSpbS4V4vdb+8kIMGMUzROsm0BCKY1NjjzndkPF+FvqENFQmsKQRPLVozMVGHsbZZ\\\\\\\/BwNXX0Zn3Fspk1BfDwfqY0HGULe18uRxXhQ0hLssv87vC\\\\\\\/bYd\\\\\\\/HzlOKRfTMsbDD0nHHFwXOaFnJSaUc6OAIIlQe\\\\\\\/oBaBrtsVANysDKTok1wDV2atANtKKVhJBD\\\\\\\/6kcrBJdV7oDMOtm2ES6\\\\\\\/ccjhl5s2DZUDhA5fQwJ9KpwSRQs3vbDYyK9LGQD+vKBaSCvOn+RzxqRSHyEm6BlngIDJP0rVdeciI\\\\\\\/JT31SPKAlZMwfV92IwxdAnFPtYxR5T2qIvSs0Ps+S4awhCsLmA5sLfTVPZuXlvGWu0yWV+tAbwOk2KBU\\\\\\\/Yp1zsr1txcAbcxl\\\\\\\/J5sstSMzik7d1qtVWiDEqbQolqE3HLA2CQ0NndG3XHPOchBnEVt5nqj2x3BESloRTQP5xhQJUZHCoiGJwQYsfhi6Y58nQrVmjZxhDfk6gVusoo2iUwtowojSoH2ZSJrjzIjUhgwC5nRfWcScEDabevM4qdUjfpuZ1Xk78YyTM6SChjFrgSlLpiI\\\\\\\/GlEuqCubB0X8Mfk4ZF4AHa2+x846jRIFwXaF51NFKylq47KT8D24AnG4\\\\\\\/3TUDC84bjp3GIIXnma8Fm3gS+JjbvBvfWS9OBh4rRYvNSp0R4LqDmM4NQB9CCy6+n9Nib1OkWNXNhTuKXRMQ5XU0nGG8s4yi\\\\\\\/C3T0ruqRamswS0\\\\\\\/OOUMcAfqITnYiWW9jEZzQdtpTViBNXxo4Up7yBNepvcSnRBM0Ju1XTSvNUUYbHeo4+gQgv9PD66Q7NmQpzWKFmtDNyQlTXSPmXdxhTxBe5wQOO6sI6h43uHpAYVKYjlKNiLRfjgEXTYDK\\\\\\\/Pul1MDag5wIUJh3AJLgnN9OqNg\\\\\\\/a0CDfu7dMXvhgNsBZI4C9N2vFICnxRWuvhkpvXfTbbp6vNLo5V16p942eNAI+N7o5oWSAAwBYU\\\\\\\/MdyzE5wU\\\\\\\/Wi2DfK\\\\\\\/T0dcizCxqKsBlP4v198mdwd0YCAsMwY\\\\\\\/qNslmLwOefpAM18wdaMGOqnHUIvRvKdzwLtFXkzEx6Zq3GhzeseY3bp\\\\\\\/kYjGOOEuf14BOW5iv4XZGgG87u40t67IPe8XfdBlseslKB8Kyi1nbwMmeoduKNPwpB+7qnGqAAgJLvQLK4y7tLz\\\\\\\/L3q9VpoGmTq0oc3Fl0A3\\\\\\\/ZAYjb1lGb1+4hfP+aoghud4DkpcWgKxFScPeCtiHOfCiwUWnH7LHpEv2OrdOb36691rrM5I9nimtGPezE0RKiGz1kIGvRO24S1bfeqoXRMDlzl2M3BkXassOnYvssHOh6a4cX\\\\\\\/mZTAU3X3kXdfKdQN0p8fOpWxpy+HDO2h+3LXyAYPuefZ8\\\\\\\/wvbvsy3sz6Kiy7zreVCZF4RXfOHZxN6+5rRAn5gjfh4dGrhkb+cib0GDh4zCU4t+LD1OER\\\\\\\/RzZfdxjHHlrEHxi\\\\\\\/UhEZneyB\\\\\\\/WLwFxIhK0\\\\\\\/\\\\\\\/MMKZkq5gvfgweOkiQElXn3Jh5nls0i7\\\\\\\/P2j0iXVYYm37Pujt0Qt\\\\\\\/nwISP30UIbt8Z9Dla\\\\\\\/m07gnT2Xdozd\\\\\\\/bOOBlXpf65hZiKYUYp0br1LA35vhSPUWrC1bceuKHl6BCnbGDR1u4KC98u3LhUuejG0i\\\\\\\/HezPYOr1QF5Y4vRwSnPVyewvbNXEBoktssT55w6OuXX88d2o2zOvFsgj\\\\\\\/07o9Qc0JG+SskEpFIX3Z5kWbf\\\\\\\/2Y8KHwkA7kSGhEIFdQqm7IN\\\\\\\/RHyXOIJ0JoywMgI8gR3Jh4dKeP8c4NM7Q5YdJ8VDke+DUNc5m8bpnKMWwJx++bQXGhFlQJ7cB4t96z\\\\\\\/nfNBEbB5phE13uYncVTe+oFYDJ26T4U5beekQ4Iwnd9yH\\\\\\\/uBfLIVnxBTrry+B1qj5Zw3lEcj8hxOkCU3eProboBZPtzK\\\\\\\/60t1+j+6DZxn4\\\\\\\/6UP6GRKYb34I3h3mbu0iZbyyDibnW0QzZBNs+KuWowV53ySiU4LMlqM+jzUYnSHUQTxtwh2y+TlYJRQgK8gV9oj2XKZgnTEjLDi3pKP9AwI\\\\\\\/uLoTSj+wstVt9C5BTfSI56g5HRroHsQcNv89JarYKr6Xcd\\\\\\\/NS+je6RNn0VC0EiNTgD0Wb5x+\\\\\\\/BLehXzkrYW7e9u3eEQpiMqV4eShDLMlKztlm9u6h7pWgW69\\\\\\\/4b9XDwoajtfgxQdl0UbNVfzizEoXoQ\\\\\\\/ceMhpQv03s38zjX4CtN3q67u6WOMuIaO5o7aowvOOxphX+uVwSYbSqI59dHxRzMPpXIiCYAqEDNEeHgL71+Kxrrz\\\\\\\/tTNgcPYzquNXcaEf+Dma\\\\\\\/NGkwhIoK+Nj+rgBf2s7OFG\\\\\\\/c1XpzMemHhX15SXJ4wbQ6saDTfqvc943xVd2qj9iVUBUDIm5MKTTQNzLWg9ngTqD58KGKFmkT4VS6PYmMtGjPQgZtOF3pRWkXro1AhWKQPZ33mCqFLxOIT14TD6iDYT5g\\\\\\\/E5kcNlFgmC3+oQT8aWvwx3X1oxmuaQruQvhdlxDdaMPk\\\\\\\/lazeVKMfnCjSHHSk8Fz5P\\\\\\\/hO2TRCAasKD0R64\\\\\\\/Ro9UxYLyQ+8dGbmUMlJbMdwmuLlmTUmiZaE5WUfiMeqjV05p2flV1a+JZIdd2KrLIKVA6t2smW6D0CKzsr6jn+\\\\\\\/ms\\\\\\\/oumEllYBHnI7VefwON5MKXhIfTY+aLWzzHW8bz9Xq8oF\\\\\\\/oEB\\\\\\\/spFnEEImORWiQa5M3e\\\\\\\/Lv\\\\\\\/9ue3VzmKtIDVabBQhO6P2AKeEFtcSwSNzm+iWd7B77rSNnzmBE\\\\\\\/Uidi5XFj6yy33ck9dYOb4XW8hwEvm03hfBbwZLr3IrqPKk+kIKoSNt8\\\\\\\/+UwWnNMgVZS3ColHlzYdaRi2ykNbg+FCu43oAL5fSDDJU1L2MwmmZakdoFpQKevbdrYIp6Rmv\\\\\\\/NV6qXEMgyzx32oQ7nkk+ixNDSMWsuxNc8S2wyfXZGrueOy\\\\\\\/PutPR0PKTgp81ndYe1q8qwi30eLKaozuFWKNxEkY2gd3ACALl04RsFdPz3aNT4A620CgygmPuJl2DIMABbipQzLGQItn9Lzu\\\\\\\/W\\\\\\\/TOsEJuClVYbhB8r4RmfuiGuFS7EeSubSVYWM5bCqoPx3mDxyEmVALoOao6SOPvO0WV1qLSFYq7c1aAqh60LTBsp81PFPm1Lj\\\\\\\/zL2OrACmaeCyNoqffblGuJS6rIztXuUMTvpevpgxKoX3AeC5sLdm9D2Dd109bj6mHOXjvBCJ01huRc6jEnBI4Mf3p\\\\\\\/blAYqhRq3Y439Ixn+Ct6xOZ0VlikoUxcuo1J+WtCmN8fm3CH2+6mRrnF6kuNFgskU1bTCa\\\\\\\/kyveHoRgt8z+vlDALrRf1wP9ktmcp\\\\\\\/3N9xM+04p36U+zqM+YNObY+MvMbDmp9rJQ3P\\\\\\\/bTq2QCE5Hpgi\\\\\\\/nP8rsaWgPcpGmrSSbniSt1E6iUQ2VPo5pDn9cNduuqbmoIo3J3lcADJI1HvmPI\\\\\\\/vnHgwACPQn7ogC1JnSz6s6O6PFjXW\\\\\\\/7IHUYqV3FzBM6f17gzUWonuaAPPP72N8q+qCixjJre11\\\\\\\/F91ek\\\\\\\/DLZvbCKtTlrnhfbB\\\\\\\/Rk4J1V0neJ9ySP6NxGgFLPo+r+coWKD0b0SPCsw\\\\\\\/d8xwymk4WwOI+jWP3bSL\\\\\\\/Y2S5Acdq5csDC+3PIak+Fy7AprZrNiYcZ0rTrxmeHZZwef\\\\\\\/MghXuEnAz\\\\\\\/ro3HVcRQkKwrZXlU5qsRab3TT9MvjgokYQaVxIdqgDuOX0e2KuXSUmrWZUirhFCQHBAL6ek1OOieE9otrO7YkRM2KVES9WCbJU8Gt60fmilAjpnKrw3redCjT23gUR4jynGDpcMvJ6qHscOGSa7p96caBjanvPZHFpvOI0d29ZIJjy0NGQWpPfqowHkiXwJuFHJm4AxGxAE7wvMU08QP6knAnFI827vsXZrVKpXpoL5GVmqXljikL8BWfBuTEJig2PpVfADIjObufjIENAQq5+qBOlptLsj9APLThW914mNpw3VbtWt49hklRXrEfa+VhKVmCQgtOf1pi4VouUE+ZM7aBT5l8kNrUJyqPInm3YK6aeNwg+oFRuXXLv8IyzGor6HY2cjIENuEm4DHJgt08YkLfYV\\\\\\\/j1ryd\\\\\\\/tORpgZqzxBbHz4uEWXHhxssWVWS\\\\\\\/WZ2d5Xq8VraoWdTJoCLWohF8Y3W3SaFgI0dnjujNBEPaQbLHM01XaR4Ibz6JT2++NtUmzsa+0Eu0xhotfoPXAT0f8YPzSBcU+O6w+uKizIc0vPCP6Cm2E7tmQ6xbIdvlaZdFWY2CHX8sl\\\\\\\/5GEn5Z1vlykauMEVCoHz+5FQ9vYvT6LhyoRG+mguLI23XB\\\\\\\/DKFPEF1861VBNefRKRwaSpWIFWK6asrPgKa055eGmo2VraYFeaP8hpKspeZB\\\\\\\/0BkACsPvwHpnujUfY7DSmXAgtsbpKbWTQz+FJmayug0zjo8F63j4ccP4NI1tvRgZAP62uqlllWDYCRgZqJTuRe6GQBlVWP8Ogb2Sk7DCxVCIcD4hMa78ZhZxwI4h\\\\\\\/Y7dfdSgkKQb3ylxCl9HseRaQn3R9G3S+Su5gtHajDX+UoHGus7Ke\\\\\\\/TDcaU+yr5ebEX4hdY2O+EaYufJzHYpkNv4mUQ5ccZblAl+e1M2oQqAO796f+wVRhdjxHlMTMTETKpLsVvsgPshA7S7D+Vs6yacmmKzOWh+qWxAG\\\\\\\/xVzkwcNZ09fpYtHK3NLD0+0yDDKMompkwjkLPtLUlZxJR2a2XF42Bic686mQX81x11cIvjYCBTCQVW6xtZwrytoEYEc64yrnPmfnLEw5OQLhpnJZ4+OkTMhCwKZSeEgLeIOtSIWepWKyhUJp8zhDtSwG3tQiPBHE2BTA+0x1vTCuvWQpav9FZLGBuKFPCblxdhAZw9Q+6UlgYRBMu7kHW84vY57lbJDzKKWL9NYLRjbM+rTsktJfYLPBZs9iHRbXSGO2AUEF8OGy4XFwMMA0FGASnmJ08JicKEJun6YJUevs52ACaSlf3a+rChJ2wWrD9gWro9xcpGXMccwDMvZU25PNZUPs2BI02hMopn+aF+riIEYClCRe3Gea\\\\\\\/3Mbil+1D3I9l5ztTznfveCx49wRO\\\\\\\/xxBWwJxDYHzTL0X7p1nHx\\\\\\\/v\\\\\\\/LNH18paFU4pNy61\\\u0022,\\\u0022iv\\\u0022:\\\u0022QPhnUQQgjQEWv\\\\\\\/AMt\\\\\\\/sIqQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e8a20f979357dcdfedb3c5dc908797f0f7b5fa2224215efeb4e6aff22181ee8c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227U+JupDLUkCVwPWtWOTvuJct9+5dP+mOIFw0ErXkNYDKt4852l+id+QkCOZArFBLe2AQetXw3sUEgNQzpIWYo+FTD7oP+HXo5VFzhEE+wDlE+CI3JRULxC9GMZU+y6zIVVF7szC0IIFa\\\\\\\/q2HirrIYEgpTRz3ipQhXL++gc5nPeRfco2SftB5l5wkXx9j2F2SJi6RhpX20RYyqfaRNc5n9usb2cY9Nq2YIB+bGCTDWoG5KipNrMkbJYzx39zDKk9N7e8iDTn6EmdRWaBWY2MmdLHtk9GWiI\\\\\\\/O3nVcRrChdZy831+uMrh2F6pBAldhhXUF+rPU2\\\\\\\/FUaVa3TPPncgPctI77aTAUcsjS8J4vrtrQ68o+k2qdQ1rYTEIJZ0ns9OpDb9bPHrsxc4obxK53jmTYLRO8v6OO8IIoXQhkJLJJGGbBitsigOm\\\\\\\/PvGc+DfmHVaulBCKaCkieoHyVJSMgT4XVisufGxGYFtMS\\\\\\\/+VodfDSGDQefkAndVq2EjdvWEXSMAg5TEKr9PPfTG+D03DWgr\\\\\\\/obD2L\\\\\\\/YP0PofMhZM\\\\\\\/FNTJ\\\\\\\/afOKOphf1qsVhI6fXpmgcDghyagqz9znIygPE6rdq2s6WhFlasaaP+VO2fOhnRMSp2HmnPTgozm1Z5ugrBd1rpFOdhI4JwT2jIo4RDjCs0ehKNfWMwhM\\\\\\\/KZRRVvR7KA1WqmpySKTbOUx6UokhVktuVTK35BWFmA7DCbGY+p9k8bvlaPaGc8xcsTl5nckhUqkhGOF6pSRZgwli\\\\\\\/ZI0XLW6uY0xi2v1Ku2mj9nPeTgZrB53VscPTOQUb9Nt70mFl+Ucmiz4MSzbedBMT5fUaUwRn2mR+Fw3CbyK38v3tzsCJN5PJjC+5nRpSvJ3AnVJiTIUoziPbrq2El9agOPb1NfnbSvPBDqACF+7z74NIRQgHP5OwXsUGmh9PQ5HoeFPUBGa7FrCRMsjpfNRsK4TONqzx8D6PmAuisc37ytFjD6JPPBGKNa31yF5oH4vf9aDrI\\\\\\\/yHoqAVmCbQBc8PLjR0Qck+W8HEkUGEBWpuo0c3wI\\\\\\\/zACVsUUpGLdJlafj6qKxpcv1FZAJI+EZ9XZCjtHPZDDnEdnZXCc2cRpMEj5goBtNAvaCbOah0ltYEfNJ6WoR1qoHLC752\\\\\\\/T5LyMvMeq+edWsbepufQsX7nrRc6xPd1lm0na5aXFmfmryDWCb9x\\\\\\\/iFgt6FaHdBKx05Xrjm\\\\\\\/cYvxGdsZkIDdR62K7LpvCaPORZzuft+nsIj0CDjk7l90wpX2cRhrdhX1HBq0\\\\\\\/bjvfvCOL1BOZeqMxM7b70tJFNUX4hIW3KBkIxJ7IKcbE\\\\\\\/mNd3KVs+CHuWZ+T5goMGDcLx9p5ngPpzLRG6DZ9ctl2CZ5hgxcY1T6shdCJki3Llf8veG7ZlXF9PBmKhAHu0feSytpI72frhNK8qo0b\\\\\\\/2AAHcDdj7NCF5fnLeo\\\\\\\/RTlhU3BMxwWO03JldC8Ej7eqYpv75OQcnOgcWxsmzAifGxvrKn5dTxfyfb9lm7WcyDMiqXbIe\\\\\\\/49nZVVHu3qk9inm+w7VNt6ozrWtssy0P+i\\\\\\\/z4KjOf0+QOJq69byIwOK7MODIkRHghVy2tz3YkU5oJ8DhdC6T\\\\\\\/1bAVRSPOgDJzw5IOCAgzsq\\\\\\\/iDIiel9tQTZjSsWz9WL0y9OSlFGJ+83hDIZnbkm+OgR75s06tUYP2QVA9uja7\\\\\\\/O2KcE2XTG\\\\\\\/z6GkxQc6WtrXlkoqYmH5FCTfECsXqKjlviiux5WjwDEpPSPHcXc4hu21xswRhY0CetyTSvNSP9dnXnL94mt\\\\\\\/RVZcvHXkpEyoXp9psDXwziWffd1Hn3Y7mmXxPdqx8FX9qDDqZNShO0vXH82GdjaNB5yr+7ODUcAFm54tj5\\\\\\\/QgQy3PJlpogFBfKYu0oori5Zr4LG8xlf09qwpBn1SR4OIUsEgNWKpkUU67\\\\\\\/m6+8YPWpbmilMu3Dt1O5MMFwKifXWAj1khRQVBTv7qlqsytfLdUMeW4aoZxjZ+zdYJd6FMwUH8OWuv70PEdBRZPzGTEcBXiy\\\\\\\/W9nSg8qwEJFc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022nqeHJAv9bvlx7DrnuMIyyQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222ecc861e37236977e6af500ec13f14ca4657c085a71129038344fbcf8618f14f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459437 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/387" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XCuCNtj2CaHOb2i_m-T5EVvU1RNB1gmgGR7LXR9lXws" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/387" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ID0FPwMUpQIbszY-0YvuVGNFuI59Pb196O2AdXjjkrI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/379" title="Strain Specific GenotypeâEnvironment Interactions and Evolutionary Potential for Body Mass in Brook Charr (Salvelinus fontinalis)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/399" title="On the Mutational Topology of the Bacterial Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459437" id="node459437" data-pisa="ggg;3/3/387" data-pisa-master="ggg;3/3/387" data-apath="/ggg/3/3/387.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hagen Tilgner</span>, <span class="highwire-citation-author" data-delta="1">Debasish Raha</span>, <span class="highwire-citation-author" data-delta="2">Lukas Habegger</span>, <span class="highwire-citation-author" data-delta="3">Mohammed Mohiuddin</span>, <span class="highwire-citation-author" data-delta="4">Mark Gerstein</span> and <span class="highwire-citation-author" data-delta="5">Michael Snyder</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">387-397;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004812 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459437" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hagen Tilgner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hagen%2BTilgner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilgner%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHagen%2BTilgner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Debasish Raha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular, Cellular and Developmental Biology, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Debasish%2BRaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raha%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADebasish%2BRaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lukas Habegger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lukas%2BHabegger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Habegger%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALukas%2BHabegger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mohammed Mohiuddin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Roche, Branford, Connecticut 06405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohammed%2BMohiuddin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohiuddin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohammed%2BMohiuddin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mark Gerstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BGerstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerstein%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BGerstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael Snyder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSnyder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snyder%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BSnyder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mpsnyder@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">mpsnyder@stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/3/387" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459437" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/387.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459437" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/387.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459437" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/387.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459437" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/3/387.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459437" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Precise identification of RNA-coding regions and transcriptomes of eukaryotes is a significant problem in biology. Currently, eukaryote transcriptomes are analyzed using deep short-read sequencing experiments of complementary DNAs. The resulting short-reads are then aligned against a genome and annotated junctions to infer biological meaning. Here we use long-read complementary DNA datasets for the analysis of a eukaryotic transcriptome and generate two large datasets in the human K562 and HeLa S3 cell lines. Both data sets comprised at least 4 million reads and had median read lengths greater than 500 bp. We show that annotation-independent alignments of these reads provide partial gene structures that are very much in-line with annotated gene structures, 15% of which have not been obtained in a previous <em>de novo</em> analysis of short reads. For long-noncoding RNAs (<em>i.e.</em>, lncRNA) genes, however, we find an increased fraction of novel gene structures among our alignments. Other important aspects of transcriptome analysis, such as the description of cell type-specific splicing, can be performed in an accurate, reliable and completely annotation-free manner, making it ideal for the analysis of transcriptomes of newly sequenced genomes. Furthermore, we demonstrate that long read sequence can be assembled into full-length transcripts with considerable success. Our method is applicable to all long read sequencing technologies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rna" class="hw-term hw-article-keyword hw-article-keyword-rna" rel="nofollow">RNA</a></li><li class="kwd"><a href="/keyword/roche-sequencing" class="hw-term hw-article-keyword hw-article-keyword-roche-sequencing" rel="nofollow">Roche sequencing</a></li><li class="kwd"><a href="/keyword/human" class="hw-term hw-article-keyword hw-article-keyword-human" rel="nofollow">human</a></li><li class="kwd"><a href="/keyword/splicing" class="hw-term hw-article-keyword hw-article-keyword-splicing" rel="nofollow">splicing</a></li><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li></ul><p id="p-5">Although genome sequencing has become commonplace, a significant challenge is to obtain an accurate annotation of each gene locus as well as a quantitative description of the transcript isoforms produced from it. This information is crucial for inferring biological meaning from each gene locus and, given the role of alternative splicing in diseases such as cancer (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">David and Manley 2010</a>), will likely play a role in analysis of human disease. Currently, gene annotation is usually addressed using reverse-transcription polymerase chain reaction experiments (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Harrow <em>et al.</em> 2006</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Pruitt <em>et al.</em> 2012</a>) and a quantitative description is carried out using short read sequencing projects (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mortazavi <em>et al.</em> 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Nagalakshmi <em>et al.</em> 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Wu <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Djebali <em>et al.</em> 2012</a>). Thus, quantitative description often relies on the presence and quality of existing annotations.</p><p id="p-6">In contrast to lower eukaryotes such as yeast, human transcripts contain more introns on average. It has been estimated that between 74% and 100% of human multiexon genes are alternatively spliced (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Johnson <em>et al.</em> 2003</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Harrow <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Pan <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Wang <em>et al.</em> 2008</a>). These studies demonstrate that alternative splicing can be highly regulated. Therefore, proper interpretation of eukaryotic transcriptomes requires accurate detection and quantification of full transcripts with accurate assignment of transcription start site, poly-adenylation site, and splice junctions. Mapping short reads against a genome annotation (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Wang <em>et al.</em> 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Graveley <em>et al.</em> 2011</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Djebali <em>et al.</em> 2012</a>) and its junctions provides quantifications of single junctions and exons. Yet prediction of entire transcript structures requires advanced analysis tools (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Montgomery <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Roberts <em>et al.</em> 2011</a>). Despite recent advances in the field, it is not always clear how well they reconstruct full-length transcripts when used <em>de novo</em>.</p><p id="p-7">Here, we sequence millions of complementary DNAs (cDNAs) in the human K562 and HeLa cell lines using the 454/Roche long-read sequencing technology. Although the reads are shorter than what can be achieved on the Pacific Biosciences platform (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Eid <em>et al.</em> 2009</a>), the presented data are considerably deeper than what can be achieved at this point on that platform at a reasonable cost. These 454 reads usually span multiple introns and therefore show a level of information that is not present in short read sequencing projects. We show that the partial, but long, gene structures provided by alignments of 454 reads correspond well with annotated transcript structures. In addition, we detected novel, unannotated spliceforms. These results show both the high quality of the annotation and of our sequenced cDNAs, and that the process of genome annotation and transcriptome quantification can be performed without <em>a priori</em> knowledge of transcript structures. Similarly, we show that high-quality cell type/condition/individual-specific splicing analysis can be achieved using this approach. The demonstrated approach is in principle applicable to all long-read sequencing approaches, including Pacific Biosciences (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Eid <em>et al.</em> 2009</a>), and its success will increase with sequencing depth and read length. Overall, this approach is ideal for transcriptome analysis in recently sequenced genomes that lack a detailed annotation or in cases where using an annotation could introduce a bias into the results.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data access</h3><p id="p-8">The 454 reads (sff-files) are available at the Sequence Read Archive (SRA) under accession no. SRA063146. These data also are available currently at <a href="http://homes.gersteinlab.org/people/lh372/ENCODE_454/K562/">http://homes.gersteinlab.org/people/lh372/ENCODE_454/K562/</a> and <a href="http://homes.gersteinlab.org/people/lh372/ENCODE_454/HeLaS3/">http://homes.gersteinlab.org/people/lh372/ENCODE_454/HeLaS3/</a>. Well-aligned alignments (used in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/004812SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS2.pdf">Figure S2</a>E) for K562 and HeLa S3 are available as supplementalFileS1.gff.gz and supplementalFileS2.gff.gz at <a href="http://stanford.edu/~htilgner/2012_454paper/454.index.html">http://stanford.edu/~htilgner/2012_454paper/454.index.html</a>. Note that for spliced alignments the alignment strand has been changed to RNA-direction (as indicated by the dinucleotide-consensus) and alignments overlapping ribosomal RNA genes have been removed.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/387/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Read length histogram for the K562 cell-line. (B) Total number of reads in the K562 cell-line and number (and percentage) of reads that could be mapped using GMAP. Percentages in light blue bars are given with respect to the previous light blue bar. (C) Chromosome distribution of read-mappings. (D) Number of reads (and percentage) that were considered mapped with high confidence (&#x201C;well-mapped&#x201D;) and number of reads (and percentage) of reads that did not overlap ribosomal RNA genes. (E) Chromosome distribution of high confidence read mappings that did not overlap ribosomal RNA genes. (F) Number of reads falling entirely into regions without annotated transcription (WAT), intronic, and exonic regions. (G) Number and percentage (with respect to the previous light blue bar) of reads containing a split (first bar); number and percentage of reads containing at least one split and having intron-consensus di-nucleotides at the ends of all splits (second bar); number and percentage of reads containing at least one split and having intron-consensus di-nucleotides at the ends of all splits and having at least one split-end as an annotated splice site for all splits (third bar). (H) Number of introns in these reads (with respect to last blue bar in G). (I) Intron length distribution for the previous introns, showing only introns of up to 500 bps. (J) Percentage of annotated genes identified when using increasing number of reads. (K) Percentage of annotated exons identified when using increasing number of reads." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560449566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Read length histogram for the K562 cell-line. (B) Total number of reads in the K562 cell-line and number (and percentage) of reads that could be mapped using GMAP. Percentages in light blue bars are given with respect to the previous light blue bar. (C) Chromosome distribution of read-mappings. (D) Number of reads (and percentage) that were considered mapped with high confidence (&#x201C;well-mapped&#x201D;) and number of reads (and percentage) of reads that did not overlap ribosomal RNA genes. (E) Chromosome distribution of high confidence read mappings that did not overlap ribosomal RNA genes. (F) Number of reads falling entirely into regions without annotated transcription (WAT), intronic, and exonic regions. (G) Number and percentage (with respect to the previous light blue bar) of reads containing a split (first bar); number and percentage of reads containing at least one split and having intron-consensus di-nucleotides at the ends of all splits (second bar); number and percentage of reads containing at least one split and having intron-consensus di-nucleotides at the ends of all splits and having at least one split-end as an annotated splice site for all splits (third bar). (H) Number of introns in these reads (with respect to last blue bar in G). (I) Intron length distribution for the previous introns, showing only introns of up to 500 bps. (J) Percentage of annotated genes identified when using increasing number of reads. (K) Percentage of annotated exons identified when using increasing number of reads.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/387/F1.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/387/F1.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/387/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/387/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-9" class="first-child">(A) Read length histogram for the K562 cell-line. (B) Total number of reads in the K562 cell-line and number (and percentage) of reads that could be mapped using GMAP. Percentages in light blue bars are given with respect to the previous light blue bar. (C) Chromosome distribution of read-mappings. (D) Number of reads (and percentage) that were considered mapped with high confidence (âwell-mappedâ) and number of reads (and percentage) of reads that did not overlap ribosomal RNA genes. (E) Chromosome distribution of high confidence read mappings that did not overlap ribosomal RNA genes. (F) Number of reads falling entirely into regions without annotated transcription (WAT), intronic, and exonic regions. (G) Number and percentage (with respect to the previous light blue bar) of reads containing a split (first bar); number and percentage of reads containing at least one split and having intron-consensus di-nucleotides at the ends of all splits (second bar); number and percentage of reads containing at least one split and having intron-consensus di-nucleotides at the ends of all splits and having at least one split-end as an annotated splice site for all splits (third bar). (H) Number of introns in these reads (with respect to last blue bar in G). (I) Intron length distribution for the previous introns, showing only introns of up to 500 bps. (J) Percentage of annotated genes identified when using increasing number of reads. (K) Percentage of annotated exons identified when using increasing number of reads.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>RNA extraction and sequencing</h3><p id="p-10">For RNA isolation the cells were grown to 60â70% confluency and lysed using Trizol (Life Technologies). Total RNA was extracted from the lysate following the protocol provided by the vendor (Life Technologies). RNA was digested with DNase, extracted with phenol:chloroform (1:1), and precipitated using ethanol. Ribosomal RNA was removed using RiboMinus technology (Life Technologies), and mRNA was purified using oligo-dT columns (Life Technologies). Purified mRNA was provided to 454 for subsequent processing. Roughly speaking, from 200 ng of messenger RNA as starting material, a cDNA library was prepared using Roche kits (Primer Random, cDNA Synthesis System, and the GS Rapid Library Prep Kit), and 32 cells were sequenced for each cell-line.</p></div><div id="sec-4" class="subsection"><h3>RNAseq mapping of long RNAseq reads</h3><p id="p-11">Fasta sequence was mapped to the human hg19-genome sequence using the Genomic Mapping and Alignment Program (GMAP) and a minimal intron length of 25 (âminIntronlength = 25).</p></div><div id="sec-5" class="subsection"><h3>Criteria for high-confidence mappings (âwell-mapped readsâ)</h3><p>For each read-alignment, we calculated two numbers:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-13">mapLength: the ratio of the number of aligned bases and the read length; and</p></li><li id="list-item-2"><p id="p-14">mapQuality: the fraction of identically aligned bases, averaged over all exonic blocks of the read-alignment.</p></li></ul><p>Read-alignments for which mapLength &lt;0.67 or mapQuality &lt;0.75 were discarded. For reads with multiple alignments, the read-alignment A that maximized mapLength* mapQuality was retained if and only if 0.98*mapLength<sub>A</sub>* mapQuality<sub>A</sub> â¥ mapLength<sub>B</sub>* mapQuality<sub>B</sub> for all other alignments B of the read.</p></div><div id="sec-6" class="subsection"><h3>Identification of genes for a mapped and spliced read</h3><p id="p-15">To each spliced read, we assigned the spliced gene with which it shared most splice sites. If two genes fulfilled this criterion, both were retained.</p></div><div id="sec-7" class="subsection"><h3>Inclusion and exclusion reads for mapped internal exons</h3><p id="p-16">For each cell type <em>t</em> and all exons <em>ex</em> that appeared as internal in at least one read, we determined two numbers:</p><ol class="list-ord " id="list-2"><li id="list-item-3"><p id="p-17">IR<sub>t</sub>(ex): the number of reads linking a splice site of this exon to an upstream exon and/or a downstream exon.</p></li><li id="list-item-4"><p id="p-18">ER<sub>t</sub>(ex): the number of reads linking an upstream exon of <em>ex</em> to a downstream exon of <em>ex</em>, with at least 50nts separating ex and both neighboring exons</p></li></ol></div><div id="sec-8" class="subsection"><h3>Minimal evidence for alternative splicing</h3><p id="p-19">Each internal exon ex of a read mapping was considered for testing of AS if and only if the following conditions were met:</p><ul class="list-unord " id="list-3"><li id="list-item-5"><p id="p-20">the number of reads (in both cell types combined), in which ex appeared as an internal exon was at least 2;</p></li><li id="list-item-6"><p id="p-21">we found at least one read, indicating the usage of at least one of the splice sites in one cell type, and at least one read skipping this exon in the other cell type (or vice-versa); and</p></li><li id="list-item-7"><p id="p-22">the sum of IR<sub>K562</sub>(ex), ER<sub>K562</sub>(ex), IR<sub>HelaS3</sub>(ex), ER<sub>HelaS3</sub>(ex) was greater than (or equal to) 0.75 times the number of reads overlapping the exon ex.</p></li></ul></div><div id="sec-9" class="subsection"><h3>Inclusion level (or Î¨-value) definition</h3><p id="p-23">For each identified AS-exon we defined the Î¨-value in cell type t as the ratio of IR<sub>t</sub>(ex) and (IR<sub>t</sub>(ex) + ER<sub>t</sub>(ex)).</p></div><div id="sec-10" class="subsection"><h3>Splice-site strength measure</h3><p id="p-24">For each exon we used geneid (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Parra <em>et al.</em> 2000</a>) to calculate an acceptor score and a donor score. Non-AG acceptors were assigned a score of â15 and non-GT donors a score of â7.</p></div><div id="sec-11" class="subsection"><h3>CDS coding exons</h3><p id="p-25">Exons, which were entirely coding in at least one annotated transcript, were labeled âCDS.â Importantly, this does not exclude the possibility that they might be partially or entirely noncoding /UTR in other transcripts.</p></div><div id="sec-12" class="subsection"><h3>Alternatively skipped exon calling</h3><p id="p-26">For exons with âminimal evidence for ASâ (see the section <em>Minimal evidence for alternative splicing</em>), we constructed a 2 Ã 2 table, containing the numbers IR<sub>K562</sub>(ex), ER<sub>K562</sub>(ex), IR<sub>HelaS3</sub>(ex), ER<sub>HelaS3</sub>(ex). All resulting tables were subjected to two one-sided Fisher tests, and a Benjamini-Hochberg correction was applied to both directions.</p></div><div id="sec-13" class="subsection"><h3>Using cufflinks to predict full-length transcript structures based on 454 reads</h3><p id="p-27">We discarded 454 reads for which any aligned block (exon) had a GMAP score of less than 98 (<em>i.e.</em>, &lt;98% of the aligned base-pairs are matches). The remaining reads were submitted to cufflinks using a minimal intron length of 25bps (-min-intron-length 25).</p></div><div id="sec-14" class="subsection"><h3>ENCODE predicted full-length transcripts (based on Cufflinks and ENCODE short read RNAseq)</h3><p id="p-28">As part of the ENCODE project, the previously cited Djebali and coworkers have produced large amounts of paired-end 76-bp RNAseq data. Among other things, these authors have predicted transcript models in different subcellular compartments and cell types using Cufflinks. We downloaded the full-length predictions corresponding to K562, whole-cell, long, and polyadenylated RNA-seq from <a href="http://hgdownload-test.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeCshlLongRnaSeq/releaseLatest/wgEncodeCshlLongRnaSeqK562CellPapTranscriptDeNovo.gtf.gz">http://hgdownload-test.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeCshlLongRnaSeq/releaseLatest/wgEncodeCshlLongRnaSeqK562CellPapTranscriptDeNovo.gtf.gz</a>. These transcript models sometimes contained very short introns (<em>e.g.</em>, five base-pairs and less). For equivalence with the GMAP alignments (and not to overstate the performance of our 454 approach), very close exon-pairs (separated by less than 25 bp) were joined into one exon.</p><p id="p-29">More information about the original dataset we downloaded can be found at <a href="http://genome.crg.es/~jlagarde/encode_RNA_dashboard/hg19_RNA_dashboard_files.txt">http://genome.crg.es/~jlagarde/encode_RNA_dashboard/hg19_RNA_dashboard_files.txt</a>; the geoSampleAccession is given as GSM765405.</p></div><div id="sec-15" class="subsection"><h3>Human genome sequence</h3><p id="p-30">The human genome sequence(<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Lander <em>et al.</em> 2001</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Venter <em>et al.</em> 2001</a>) version hg19 was downloaded from the UCSC browser (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kent <em>et al.</em> 2002</a>) on August 21, 2011.</p></div><div id="sec-16" class="subsection"><h3>Comparison of intron structure</h3><p id="p-31">We first denoted each intron by its genomic position (chr_start_end_strand). Each 454 read was then represented by a tuple X = (x<sub>1</sub>,x<sub>2</sub>,â¦,x<sub>n</sub>) where each x<sub>i</sub> is an intron. Note that this representation of a read represents perfectly the introns (splits) of a read but loses the information of the start and end of the read. In the same way, we represented each annotated transcript as a tuple Y = (y<sub>1</sub>,y<sub>2</sub>,â¦,y<sub>m</sub>). We say that tuple X is contained in Y, if and only if there is a k â¥ 1, so that x<sub>1</sub> = y<sub>k</sub>, x<sub>2</sub> = y<sub>k+1</sub>,â¦, x<sub>n</sub> = y<sub>k+n-1</sub>. In other words the intron structure X (given by a 454 read) is a substructure of the intron structure Y (given by an annotated transcript).</p></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><div id="sec-18" class="subsection"><h3>Transcriptome sequencing and annotation-free mapping in two human cell lines</h3><p id="p-32">Whole-cell cDNA libraries from messenger RNA molecules (<em>Materials and Methods</em>) were prepared in two human cell lines: the myelogenous leukemia cell line K562 and cervical cancer HeLa cells. These cDNA libraries were sequenced on the 454-platform using multiple runs, yielding a total number of 4.1 million reads with a median length of 521 bp for K562 cells and 4.6 million reads with a median length of 538 bp for HeLa cells. Read length rarely ever fell below 450 bp, whereas reads with lengths of longer than 800 bp could readily be observed (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/004812SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS1.pdf">Figure S1</a>, A and B, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS2.pdf">Figure S2</a>A).</p><p id="p-33">All reads were mapped to the hg19 version of the human genome sequence (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Lander <em>et al.</em> 2001</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Venter <em>et al.</em> 2001</a>) using the spliced aligner GMAP (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Wu and Watanabe 2005</a>). For K562 cells, 94.3% of all reads could be mapped (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The remaining 5.7% unmapped reads were on average of much lower quality and also longer than the mapped reads. A more detailed analysis of unmapped reads can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS3.pdf">Figure S3</a> and its legend. Mapping numbers per chromosome broadly correlated with chromosome length (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Chromosome Y received a very low number of read mappings (~19,000). These mappings are unlikely to reflect the true biological origin of the corresponding RNAs but rather homology between DNA sequences on chrY and other chromosomes, because K562 cells were created from a female donor. In contrast to short-read technologies such as Illumina, long-read technologies such as 454 usually obtain reads of varying length. The typical approach of fixing a number of mismatches, above which a mapping for a read is not considered, is therefore not suitable for long-read technologies.</p><p id="p-34">Thus, we devised more flexible criteria (<em>Materials and Methods</em>) to define high-confidence mappings (or âwell-mapped readsâ), aimed at ensuring that the read mapping (1) consisted of a large number of identically matched nucleotides, (2) represented a large fraction of the read and (3) that any secondary mapping for the same read was clearly worse than the read mapping in question. More precisely, we only considered mappings for which at least three quarters of the aligned nucleotide-pairs were identical and the mapping length was at least two thirds of the read length. These criteria were very stringent to guarantee correctness, rather than completeness. A total of 83% (3.25 million) of the mapped reads were considered high confidence (see <em>Materials and Methods</em>) and did not overlap ribosomal RNA genes. Thus, 17% the readsâ mappings were removed by our filter (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). On chromosome Y, however, 33 of 34 reads were removed by our filter (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1E</a>). This shows that our filter achieves what it intended to achieve, that is, to remove almost all mappings that we know to be false (<em>e.g.</em>, mappings to chromosome Y in a female cell-line) while removing few other mappings.</p><p id="p-35">Using annotation projections (A. Tanzer and R. GuigÃ³, unpublished data) defining regions without annotated transcription (WAT) [according to the gencode v7 annotation; these regions contain no protein coding genes, no long-noncoding RNAs (lncRNAs), no small RNA genes, <em>etc</em>.], exonic, and intronic regions, we assessed how many mappings fell entirely within one of these nonoverlapping categories. Very few fell entirely within regions without annotated transcription (3.3%) or single intronic regions (6.3%). Hence, at least 9.6% (~310k) of the reads originate from transcription units that are not yet appreciated. A considerable portion (25.6%) fell entirely within single exonic regions and presumably most often represent single exon genes (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1F</a>). The remaining reads overlap regions of at least two different categories. Slightly more than two million reads were split-mappings and for almost all of these we found all split points to respect the intron consensus (GT-AG, GC-AG, or AT-AC) and to use at least one annotated splice site (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1G</a>). We conclude that in nearly all cases, it is possible to determine the location from which an RNA was transcribed as well as the precise location of all its introns, using nothing but the program GMAP and the genome. This entire collection of alignments contained a total of 5.9 million introns (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1H</a>). Annotated human introns are very infrequently shorter than 70 bps, but 70â80 bps introns are frequently observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS4.pdf">Figure S4</a>), suggesting that this is a minimum length for introns to be appropriately processed. The distribution of intron lengths in our 454-mappings is highly consistent with the length-distribution of annotated introns(<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1I</a>).</p><p id="p-36">We next monitored how many annotated spliced genes were covered by at least one read, limiting our analysis to genes and reads that contain at least one splicing event. Using only one lane (~90,000 reads), we found that approximately 20% of such genes were covered (defined by splice site identity; see <em>Materials and Methods</em>) by at least one read, and this percentage increased gradually to 38% when we used more and finally all 32 lanes (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1J</a>). The coverage percentage began to approach a plateau after about 25 lanes, although saturation was not attained, even after 32 lanes. A similar analysis on exon level showed that about 23% of annotated exons could be found in at least one 454 read (requiring both splice sites to be precisely identified, <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1K</a>).</p><p id="p-37">For mappings in HeLa (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS2.pdf">Figure S2</a>), we generally observed similar trends, although both the numbers of mapped and highly confidently mapped reads were considerably lower. As in K562, unmapped reads tended to be longer and of lower quality (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS5.pdf">Figure S5</a>, A and B; see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS3.pdf">Figure S3</a> for an explanation in the K562 case), suggesting again that some longer reads contain very low-quality nucleotides. Consistent with this interpretation, we found a longer read-length median and a lower read quality in HeLa (than in K562, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS5.pdf">Figure S5</a>, C and D), coinciding with a lower number of mapped reads in this cell type.</p></div><div id="sec-19" class="subsection"><h3>Comparison of multi-intron arrangements against annotated transcripts</h3><p id="p-38">We next assessed whether our read-mappings gave evidence for transcript structures that were not included in the annotation. To this end we transformed each transcript of the annotation into an ordered tuple of introns and then proceeded similarly for our read-mappings. We then asked how often the entire intron-structure (tuple) of one of our reads was contained in the intron-structure (tuple) of an annotated transcript, starting with the K562 cell-line (<em>Materials and Methods</em>). In other words, we asked whether an annotated transcript contained all the splice sites indicated by a 454-alignment, and no other splice site in between these splice sites. Note that for this analysis we used only the reads for which each intron respected the splice-site consensus without considering the annotation during the mapping in any way.</p><p id="p-39">An example of such a novel mapping, in which three exons are simultaneously skipped, is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>. The ~2 million read-mappings we used in this analysis had at least one intron and more than half a million reads had four introns or more (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Very few (&lt;13,000, 0.6%) had an aligned block with an alignment quality of less than 75%, which might indicate that the K562 genome might not be well represented in hg19 in these regions or that some details of these alignments might be questioned. The vast majority (~93%) of these 454-intron-structures corresponded to partial intron-structures of the annotation (that is, the annotation contains a transcript, that contains all the introns that the read contained and no other introns in between, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). This result highlights three facts: First, the high quality of the intronâexon structure in the annotation; second, the high quality of the intronâexon structure in our 454-alignments; and third, although we have used the annotation here to check our results, we would not have needed to do so because our annotation-independent mapping approach finds, in large parts, the same gene structures. The fraction of read-mappings that contained novel elements with respect to the annotation increased quite strongly with the number of introns in the mapped reads and with length of the read from 400 bp on (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, D and E</a>). Our results also indicate that there appear to be a class of longer gene structures, for which not all transcript structures are described in the annotation. 7% of reads (~133,000 reads) had intron structures that were not fully consistent with annotated intron structures. Only very few fell entirely into intronic or intergenic space (&lt;3000 reads for each of these two categories) and supposedly represent novel genes. Of the aforementioned 133,000 aligned reads, 85% (~113,000) however, had at least one splice site in common with a spliced gene and supposedly represented novel isoforms (or in some cases partially processed transcripts) of these genes. These novel-isoform reads affected 9902 spliced genes and for 4961 spliced reads we could find at least six such reads (data not shown). This finding shows that the vast majority of our reads represent RNA products of known genes, although sometimes with unknown intron structure. In summary, these results show that sequencing long reads is of great use in the annotation of genomes. The simplicity of the RNAseq process in comparison to cloning techniques is a further advantage in this respect. Largely similar results (using HeLa cells; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS6.pdf">Figure S6</a>) show that transcript reconstruction can be achieved for multiple cells types using this approach.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/387/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="454-read mappings in the K562 cell-line. (A) Example of a 454 read showing a partial gene structure that was not annotated. (B) Distribution of intron number in aligned reads (with consensus splits). (C) Pie chart of partial gene structures (given by alignments of 454 reads) that (1) correspond to parts of annotated gene structures and (2) those that do not correspond to parts of annotated gene structures. (D) Fraction of reads whose intron structures are not included in annotated gene structures as a function of intron number in the read-alignments. (E) Fraction of reads whose intron structures are not included in annotated gene structures as a function of read-length. Note that there are very few reads that have between 0 and 400 bp." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560449566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;454-read mappings in the K562 cell-line. (A) Example of a 454 read showing a partial gene structure that was not annotated. (B) Distribution of intron number in aligned reads (with consensus splits). (C) Pie chart of partial gene structures (given by alignments of 454 reads) that (1) correspond to parts of annotated gene structures and (2) those that do not correspond to parts of annotated gene structures. (D) Fraction of reads whose intron structures are not included in annotated gene structures as a function of intron number in the read-alignments. (E) Fraction of reads whose intron structures are not included in annotated gene structures as a function of read-length. Note that there are very few reads that have between 0 and 400 bp.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/387/F2.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/387/F2.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/387/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/387/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462537" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-40" class="first-child">454-read mappings in the K562 cell-line. (A) Example of a 454 read showing a partial gene structure that was not annotated. (B) Distribution of intron number in aligned reads (with consensus splits). (C) Pie chart of partial gene structures (given by alignments of 454 reads) that (1) correspond to parts of annotated gene structures and (2) those that do not correspond to parts of annotated gene structures. (D) Fraction of reads whose intron structures are not included in annotated gene structures as a function of intron number in the read-alignments. (E) Fraction of reads whose intron structures are not included in annotated gene structures as a function of read-length. Note that there are very few reads that have between 0 and 400 bp.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Implications for long-noncoding-RNA gene structure</h3><p id="p-41">A class of genes that has received considerable attention in recent years are those that produce fairly long mature RNA products yet do not encode proteins: lncRNAs. The gencode v7 annotation already contains many thousands of such lncRNA genes, so that along with protein coding genes and small noncoding-RNA genes, the total number of genes in this annotation is approximately 50,000. The properties of long-noncoding RNAs in this annotation have recently been described in much detail (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Derrien <em>et al.</em> 2012</a>). These authors provided a classification of âintergenicâ and âintronicâ (both with respect to protein coding genes) and also showed that lncRNA genes are on average shorter than protein coding genes and often consist of few exons only. It follows that long-read technologies, such as the 454 reads we employ here, are often able to capture the entire exonâintron structure of such lncRNA genes. We therefore investigated how many of our reads described lncRNA genes. Despite the low and often nuclear expression of lncRNAs (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Derrien <em>et al.</em> 2012</a>), we could find spliced reads that mapped to 842 spliced intergenic lncRNAs (of 5774 defined by Derrien and coworkers; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) and for many of these we could find multiple reads. Some of these were exclusively expressed in either in K562 or in HeLa S3 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Similarly we could find reads for 193 intronic lncRNA genes and again some were expressed cell type specifically (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and D</a>). Although protein-coding genes have been a topic of research for decades, lncRNA genes have captured attention only much more recently. We therefore investigated how many unknown isoforms our 454-reads could uncover for genes that encode lncRNAs (lnc) and for those that donât (non-lnc), the latter group being mainly protein coding genes. Although for non-lnc genes, only 6% of all reads supported novel isoforms (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a>, left bar), we found 37% of all reads mapping to lncRNAs to support isoforms that are not known in the gencode v7 annotation. These novel isoforms were given by 4486 spliced reads and affected 599 lncRNA genes. An example of a lncRNA for which novel structures can be found using 454-reads is shown in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3F</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/387/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Histogram of long-read numbers found for intergenic, defined as &#x201C;between protein coding genes&#x201D; (see Derrien et al. 2012), lncRNAs combining the reads found in K562 and in HeLa S3. (B) Scatterplot for log10-transformed read numbers in HeLa S3 (x-axis) and K562 (y-axis) of these lncRNAs. Colorscale from white (no lncRNAs) to red (large number of lncRNAs) (C) Same as A but for intronic lncRNAs. (D) Same as for B but for intronic lncRNAs. (E) Fraction of reads mapping to non-lnc genes that represent novel isoforms (left) and fraction of reads mapping to lncRNA genes that represent novel isoforms (right). (F) First (and not cherry-picked) example of 454 reads (top, red) showing novel isoforms for a lncRNA gene (annotated transcripts in green, bottom). The read that led to the choice of this example is the second to the last." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560449566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Histogram of long-read numbers found for intergenic, defined as &#x201C;between protein coding genes&#x201D; (see Derrien &lt;em&gt;et al&lt;/em&gt;. 2012), lncRNAs combining the reads found in K562 and in HeLa S3. (B) Scatterplot for log10-transformed read numbers in HeLa S3 (x-axis) and K562 (y-axis) of these lncRNAs. Colorscale from white (no lncRNAs) to red (large number of lncRNAs) (C) Same as A but for intronic lncRNAs. (D) Same as for B but for intronic lncRNAs. (E) Fraction of reads mapping to non-lnc genes that represent novel isoforms (left) and fraction of reads mapping to lncRNA genes that represent novel isoforms (right). (F) First (and not cherry-picked) example of 454 reads (top, red) showing novel isoforms for a lncRNA gene (annotated transcripts in green, bottom). The read that led to the choice of this example is the second to the last.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/387/F3.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/387/F3.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/387/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/387/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462560" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-42" class="first-child">(A) Histogram of long-read numbers found for intergenic, defined as âbetween protein coding genesâ (see <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Derrien <em>et al</em>. 2012</a>), lncRNAs combining the reads found in K562 and in HeLa S3. (B) Scatterplot for log10-transformed read numbers in HeLa S3 (x-axis) and K562 (y-axis) of these lncRNAs. Colorscale from white (no lncRNAs) to red (large number of lncRNAs) (C) Same as A but for intronic lncRNAs. (D) Same as for B but for intronic lncRNAs. (E) Fraction of reads mapping to non-lnc genes that represent novel isoforms (left) and fraction of reads mapping to lncRNA genes that represent novel isoforms (right). (F) First (and not cherry-picked) example of 454 reads (top, red) showing novel isoforms for a lncRNA gene (annotated transcripts in green, bottom). The read that led to the choice of this example is the second to the last.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Comparison of multi-intron arrangements against transcript structures predicted <em>de novo</em></h3><p id="p-43">We next assessed whether the partial transcript structures found by mapping long RNAseq reads have been previously obtained through short-read sequencing followed by bioinformatic transcriptome-assembly techniques. To this end we downloaded <em>de novo</em> cufflinks-predictions from the ENCODE project (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Djebali <em>et al.</em> 2012</a>) that are based on short reads (<em>Materials and Methods</em>). These predictions were derived from approximately 470 million short reads and often contained exon-pairs separated by few base-pairs. In order to guarantee similar treatment (in comparison with the 454 reads), we joined exon-pairs separated by less than 25 bp within the same transcript. We then compared our 454 alignments against the short-read-cufflinks predictions. More precisely we performed a similar analysis as in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> using all aligned 454-reads showing at least one intron and for which all introns respected the splice site consensus. The annotation was not considered in this process. Roughly 15% (compared with 7% in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>) of the intron-structures generated by our alignments of 454 reads were not part of transcript structures predicted within the ENCODE project (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). For aligned 454 reads with one, two, or three introns, 90% were found by the ENCODE short-read <em>de novo</em> approach; however, this number shifted dramatically when 454-alignments with 7, 8, or 9 introns were considered. For these read-alignments, up to 60% could not be found by the short read <em>de novo</em> approach (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). This finding highlights that sequencing longer reads will improve our understanding of transcript structures, particularly for genes with larger number of introns. When only considering reads longer than 400 bp (the majority of all 454 reads, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS1.pdf">Figure S1</a>A), a greater fraction of novel transcripts was observed for reads between 600 and 700 bps (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). Similar results were obtained for HeLa cells (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS7.pdf">Figure S7</a>). An example of a multi-intron structure that was not correctly identified by the short-read <em>de novo</em> approach, but for which 454-alignment and annotation concurred is shown in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS8.pdf">Figure S8</a> shows the next four examples. Two of these involve intron arrangements, for which the short-read cufflinks approach found all parts, but not all existing combinations. Two others are simple alternative splice sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS8.pdf">Figure S8</a>). These results clearly show the need to complement short-read-sequencing with sequencing technologies that can span many introns for accurate transcriptome analysis.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/387/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Pie chart (for the K562 cell line) of partial 454 gene structures that (1) correspond to parts of full-length transcript structures predicted using ENCODE short reads and (2) those that do not correspond to parts of these predicted transcript structures. (B) Fraction of reads whose intron-structures are not included in predicted transcript structures (based on ENCODE short reads) as a function of intron number in the read-alignments. (C) Fraction of reads whose intron-structures are not included in predicted transcript structures as a function of read-length. Note that there are very few reads that have between 0 and 400 bp. (D) Example of partial transcript structures given by 454 reads (red) that are not included in predicted cufflinks structures (based on ENCODE short reads from the same cell line, top, blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560449566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Pie chart (for the K562 cell line) of partial 454 gene structures that (1) correspond to parts of full-length transcript structures predicted using ENCODE short reads and (2) those that do not correspond to parts of these predicted transcript structures. (B) Fraction of reads whose intron-structures are not included in predicted transcript structures (based on ENCODE short reads) as a function of intron number in the read-alignments. (C) Fraction of reads whose intron-structures are not included in predicted transcript structures as a function of read-length. Note that there are very few reads that have between 0 and 400 bp. (D) Example of partial transcript structures given by 454 reads (red) that are not included in predicted cufflinks structures (based on ENCODE short reads from the same cell line, top, blue).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/387/F4.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/387/F4.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/387/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/387/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462489" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-44" class="first-child">(A) Pie chart (for the K562 cell line) of partial 454 gene structures that (1) correspond to parts of full-length transcript structures predicted using ENCODE short reads and (2) those that do not correspond to parts of these predicted transcript structures. (B) Fraction of reads whose intron-structures are not included in predicted transcript structures (based on ENCODE short reads) as a function of intron number in the read-alignments. (C) Fraction of reads whose intron-structures are not included in predicted transcript structures as a function of read-length. Note that there are very few reads that have between 0 and 400 bp. (D) Example of partial transcript structures given by 454 reads (red) that are not included in predicted cufflinks structures (based on ENCODE short reads from the same cell line, top, blue).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Analysis of cell typeâspecific alternative splicing</h3><p id="p-45">Since gene expression can vary considerably between cell types, conditions, and individuals, it is crucial to obtain an accurate transcriptome analysis in multiple cell types. Although assessing differences in gene expression is relatively straight forward, analysis of cell type specific alternative splicing often is more complicated. Cell typeâspecific alternative exons have been determined previously by mapping short reads to exonâexon junctions and collecting for each exon the reads indicating inclusion and exclusion. One can then assess statistically whether the ratio of inclusion reads to exclusion reads is significantly different in two conditions(<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Wang <em>et al.</em> 2008</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Graveley <em>et al.</em> 2011</a>).</p><p id="p-46">To test whether this approach can be performed in an annotation-independent fashion, we considered all split reads for which the split-point respected the splice site consensusâchecks that can be performed in a newly sequenced genome. For all genomic stretches that were indicated to be an internal exon using at least two reads, we collected numbers of three types of reads: (a) reads linking this read to an upstream or downstream exon (or both: âinclusion readsâ), (b) reads skipping this exon and 50 adjacent nucleotides on both sides (âexclusion readsâ), and (c) the reads overlapping the exon but not falling into category (a) or (b)âsee <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1/FigureS9.pdf">Figure S9</a> for an example. Whenever category (c) is relatively small in comparison to category (a) and (b), the exon in question has few interfering factors such as alternative acceptors, donors, overlapping transcription start site or poly-adenylation sites that can confuse the analysis. Of these exons, ~47.000 showed minimal evidence of cell type specific splicing (<em>Materials and Methods</em>) and were subjected to two one-sided Fisher tests, in which we controlled for multiple testing using the Benjamini-Hochberg method (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Benjamini and Hochberg 1995</a>) (false discovery rate â¤ 0.05).</p><p id="p-47">A total of 146 exons were found to exhibit cell-type specifically increased inclusion levels. Note that sometimes two or more exons are part of one splicing event, such as double exon-skipping or mutually exclusive exons. The inclusion level or Î¨-value (âpercent-spliced-inâ) under these circumstances can be defined as the ratio of (a) and (a) + (b) and the definition of the ÎÎ¨ between the two cell types follows directly (<em>Materials and Methods</em>). <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> shows the ÎÎ¨-values for all alternative exons more highly included in HeLa cells and those more highly included in K562 cells. Of these exons, 86% showed a ÎÎ¨-value equal or higher than a threshold of 10% (corresponding for example to 40% inclusion in one cell type and 50% inclusion in the other, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), which has been suggested to guarantee increased biological significance(<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Wang <em>et al.</em> 2008</a>). A total of 40% of the aforementioned alternatively skipped exons showed corrected <em>p</em>-values that ranged from 0.05 to 0.005, but the majority had far lower <em>p</em>-values (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). Because most sequencing technologies have intrinsic biases, it is of utmost importance to independently assess the results and rule out technical artifacts. In the case of alternative splicing, a large body of research has established that alternatively spliced exons exhibit properties that can be readily determined from the genome. Alternatively spliced exons have been shown to exhibit lower splice-site strength and shorter exon length relative to constitutive exons (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Zheng <em>et al.</em> 2005</a>). In addition, when coding, alternative exons tend to show a length divisible by three (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Magen and Ast 2005</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Zheng <em>et al.</em> 2005</a>), thus keeping the reading frame unchanged when in-/excluded. Consistently, the exons our approach found as alternative showed shorter exon length and increased divisibility of their length by 3 than tested exons that did not pass the significance threshold (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5, C and D</a>). We then calculated acceptor and donor scores for all exons using geneid (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Parra <em>et al.</em> 2000</a>). As expected, the exons our approach found to be alternatively included between the two cell types showed lower acceptor and donor strength than exons whose inclusion did not differ significantly between the two cell types (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5, E and F</a>). Hence it is possible to search for high quality alternative exons in a completely annotation-free manner using long-read technologies.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/387/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Testing of exons for cell type specific exon inclusion. (A) &#x394;&#x3A8; (i.e., &#x3A8;HelaS3 &#x2212; &#x3A8;K562) distribution for exons showing significantly different inclusion levels between the two cell-types. Exons more highly included in the HeLa S3 cell line (blue) and exons more highly included in the K562 cell line (dark red). (B) P-value distribution for exons passing the significance threshold of 0.05. (C) Boxplots of exon length for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (D) Bar plot indicating the fraction of exons whose length is a multiple of three for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (E) Boxplots of acceptor scores for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (F) Boxplots of donor scores for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (G) Fraction of cell-type specific alternative exons that are annotated as exons (&#x201C;known&#x201D;), and those that are not (&#x201C;novel&#x201D;). (H) Fraction of cell-type specific alternative exons that are annotated as alternative exons (&#x201C;known&#x201D;), and those that are not (&#x201C;novel&#x201D;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560449566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Testing of exons for cell type specific exon inclusion. (A) &#x394;&#x3A8; (&lt;em&gt;i.e.&lt;/em&gt;, &#x3A8;&lt;sub&gt;HelaS3&lt;/sub&gt; &#x2212; &#x3A8;&lt;sub&gt;K562&lt;/sub&gt;) distribution for exons showing significantly different inclusion levels between the two cell-types. Exons more highly included in the HeLa S3 cell line (blue) and exons more highly included in the K562 cell line (dark red). (B) &lt;em&gt;P&lt;/em&gt;-value distribution for exons passing the significance threshold of 0.05. (C) Boxplots of exon length for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (D) Bar plot indicating the fraction of exons whose length is a multiple of three for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (E) Boxplots of acceptor scores for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (F) Boxplots of donor scores for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (G) Fraction of cell-type specific alternative exons that are annotated as exons (&#x201C;known&#x201D;), and those that are not (&#x201C;novel&#x201D;). (H) Fraction of cell-type specific alternative exons that are annotated as alternative exons (&#x201C;known&#x201D;), and those that are not (&#x201C;novel&#x201D;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/387/F5.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/387/F5.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/387/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/387/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462541" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5â</span> <p id="p-48" class="first-child">Testing of exons for cell type specific exon inclusion. (A) ÎÎ¨ (<em>i.e.</em>, Î¨<sub>HelaS3</sub> â Î¨<sub>K562</sub>) distribution for exons showing significantly different inclusion levels between the two cell-types. Exons more highly included in the HeLa S3 cell line (blue) and exons more highly included in the K562 cell line (dark red). (B) <em>P</em>-value distribution for exons passing the significance threshold of 0.05. (C) Boxplots of exon length for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (D) Bar plot indicating the fraction of exons whose length is a multiple of three for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (E) Boxplots of acceptor scores for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (F) Boxplots of donor scores for tested exons not passing the significance threshold (light blue) and for those passing the significance threshold (dark blue/dark red). (G) Fraction of cell-type specific alternative exons that are annotated as exons (âknownâ), and those that are not (ânovelâ). (H) Fraction of cell-type specific alternative exons that are annotated as alternative exons (âknownâ), and those that are not (ânovelâ).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">Although the annotation was not used to define these alternative exons, practically all of the AS-exons found (97%, 142 of 146) corresponded exactly to known exons (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5G</a>), showing again the high fidelity of this approach. However, approximately 30% of these alternatively included exons were not annotated as alternatively spliced, showing that long read sequencing can be used to improve annotations (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5H</a>).</p></div><div id="sec-23" class="subsection"><h3>Construction of full-length transcripts</h3><p id="p-50">The aligned 454 reads represent partial transcript structures sometimes containing double-digit numbers of introns. In the absence of a method to sequence full-length transcripts, it is crucial to combine these partial gene structures into full-length gene structures. We therefore used cufflinks to predict full-length transcripts from our aligned 454 reads (<em>Materials and Methods</em>), a task much easier due to the multiple introns that most 454 read-alignments contain. This resulted in a total of 9488 predicted full-length structures. We also downloaded all cufflinks predicted gene structures that were obtained from short read sequencing within the ENCODE project (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Djebali <em>et al.</em> 2012</a>) (<em>Materials and Methods</em>), focusing only on those that overlapped our long-read-cufflinks gene structures. The former set of predicted transcripts (âlong-read-cufflinksâ) showed significantly more introns than the short-read-cufflinks transcripts, but this difference was not very pronounced (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). The absolute number of predicted transcripts was twofold greater in the case of the short-read-cufflinks approach (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). For more than 50% of long-read-transcripts, we could find an annotated transcript with identical introns, showing the extraordinary quality of the predicted long-read-cufflinks transcripts. For short-read-cufflinks transcripts, this was only true for 22%, showing that long reads provide a considerable advantage in terms of constructing full-length transcripts (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). Note, that in both cases many more transcripts may still have considerably similar intron structures in comparison to annotated transcripts, and the difference may be limited to one or two splice sites. Because of the greater number of predicted transcripts when using short-reads, this however resulted in almost as many transcripts for which an intron-identical annotated transcript existed for the short-read-cufflinks approach as for the long-read approach (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6d</a>). Almost 2500 annotated transcripts were identified by both approaches. However, approximately 2400 annotated transcripts were found only by the long-read approach and almost 1900 only by the short- read approach (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6E</a>). The former showed significantly more introns than the latter, showing that when it comes to assembling transcript structures with high intron numbers, the long-read approach provides a significant advantage over short read sequencing (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6F</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/387/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Intron number distribution for short-read-cufflinks transcripts and long-read-cufflinks transcripts. (B) Total number of predicted transcripts for short-read-cufflinks and for long-read-cufflinks. (C) Fraction of predicted transcripts (for short-read-cufflinks and for long-read-cufflinks), for which an annotated transcript with identical introns could be found. (D) Total number of predicted transcripts (for short-read-cufflinks and for long-read-cufflinks), for which an annotated transcript with identical introns could be found. (E) Total number of predicted transcripts with intron-identical annotated transcripts that could be found only by &#x201C;short-read-cufflinks,&#x201D; only by &#x201C;long-read-cufflinks,&#x201D; or by both. (F) Intron number distribution for intron-identical annotated transcripts that could be found only by &#x201C;short-read-cufflinks&#x201D; or only by &#x201C;long-read-cufflinks.&#x201D;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560449566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Intron number distribution for short-read-cufflinks transcripts and long-read-cufflinks transcripts. (B) Total number of predicted transcripts for short-read-cufflinks and for long-read-cufflinks. (C) Fraction of predicted transcripts (for short-read-cufflinks and for long-read-cufflinks), for which an annotated transcript with identical introns could be found. (D) Total number of predicted transcripts (for short-read-cufflinks and for long-read-cufflinks), for which an annotated transcript with identical introns could be found. (E) Total number of predicted transcripts with intron-identical annotated transcripts that could be found only by &#x201C;short-read-cufflinks,&#x201D; only by &#x201C;long-read-cufflinks,&#x201D; or by both. (F) Intron number distribution for intron-identical annotated transcripts that could be found only by &#x201C;short-read-cufflinks&#x201D; or only by &#x201C;long-read-cufflinks.&#x201D;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/387/F6.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/387/F6.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/387/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/387/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462565" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6â</span> <p id="p-51" class="first-child">(A) Intron number distribution for short-read-cufflinks transcripts and long-read-cufflinks transcripts. (B) Total number of predicted transcripts for short-read-cufflinks and for long-read-cufflinks. (C) Fraction of predicted transcripts (for short-read-cufflinks and for long-read-cufflinks), for which an annotated transcript with identical introns could be found. (D) Total number of predicted transcripts (for short-read-cufflinks and for long-read-cufflinks), for which an annotated transcript with identical introns could be found. (E) Total number of predicted transcripts with intron-identical annotated transcripts that could be found only by âshort-read-cufflinks,â only by âlong-read-cufflinks,â or by both. (F) Intron number distribution for intron-identical annotated transcripts that could be found only by âshort-read-cufflinksâ or only by âlong-read-cufflinks.â</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-52">An accurate description of the ensemble of RNA molecules that is encoded by a genome is an integral step toward understanding the biology of the corresponding genome. Usually it takes years until a relatively complete and cell typeâspecific transcriptome description is made available. Here, we show that using long read sequencing technologies we can provide a very large part of such a description almost instantaneously without further input information.</p><p id="p-53">The accuracy of the provided information is evident by the fact that the vast majority of partial gene structures we find are already part of the gencode v7 annotation, although a large number of novel structures can be found. For lncRNA genes we find a particularly high fraction of novel isoforms, which demonstrates that long-read sequencing can further the understanding of lncRNAs. When comparing our partial gene structures to predicted gene structures based on short read sequencing technologies, however, we find that many of our mappings cannot be reconstructed from short read cDNA sequencing. Hence, sequencing longer reads adds information to short-read analysis of transcriptomes. Nevertheless, one should note that there are many more short-read-predicted transcripts than long-read-predicted transcripts, thus enabling statements about genes, which the long-read approach did not cover.</p><p id="p-54">A clear advantage of short read sequencing is that its low cost allows very deep sequencing, thus enabling quantitative statements about gene or isoform expression and exon inclusion. We therefore explored whether an example of such an analysis, cell type specific exon inclusion, can be performed using long reads. We show that despite the limited sequencing depth and other biases affecting 454 sequencing, such an analysis can be performed in an annotation-independent manner. The thus-defined cell typeâspecific alternative exons are highly reliable, as 97% of them coincide with already known exons. A total of 30% of these alternative exons were however not alternative according to the annotation; hence, our approach can complete the annotation in a cell type specific manner. The defined alternative exons show characteristics that correspond perfectly to what is known about alternative exons: They show weaker splice sites and are shorter in terms of exon lengthâand above all, in roughly two-thirds of the cases, they keep the reading frame of the encoded protein.</p><p id="p-55">The dataset we present here comprises millions of reads that span multiple exon-exon junctions. However, it falls short of sequencing entire transcripts. Therefore, the question of whether one can accurately predict full-length transcripts from these reads becomes important. We show that based on our 454 reads we can predict ~9500 full-length transcripts and that for more than half of these an annotated transcript with identical introns exists. This is contrasted by the &lt;25% of all transcripts predicted from short reads that fulfill this criterion. Furthermore, despite the extreme sequencing depth of short reads, more than 2300 annotated full-length transcripts can only be predicted correctly (that is correctness of each single splice site) using 454 reads. Conversely, there are also almost 1900 annotated full-length structures in the regions covered by 454 reads whose exact exon-intron-structures are only found based on short-read-sequencing. This shows, that the length of long-read sequencing and the depth of short-read sequencing complement each other and that their combination can be used to obtain an accurate description of transcriptomes. Importantly, new advances in sequencing technology could help reduce the cost of long read sequencing in comparison to the approach employed here. Coupling isolation of ~450-bp cDNA fragments with 250-bp paired-end MiSeq sequencing could lead to ~450 bp reads and to results that come close to ours. Assuming that Illumina continues to increase read length (from 25 or 36 initially to 150 bp paired-end at the time of writing) this could also be true for that platform as well. Finally, using the Pacific Biosciences platform, we could obtain much longer reads. Due to the lower throughput of this platform, it might be worth to couple this platform with capturing techniques in order to represent all genes independently of their expression level.</p><p id="p-56">In summary, these results show that long-read cDNA sequencing is ideal for transcriptome analysis in species for which an annotation is not available or in cases in which relying on an annotation can introduce biases. In species in which an annotation is available, long-read cDNA analysis can successfully complete the annotation and complement short read sequencing analysis.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank Andrea Tanzer and Roderic Guigo from the CRG in Barcelona for providing annotation projections. We thank the ENCODE Project (National Institutes of Health/National Human Genome Research Institute [NIH/NHGRI]) for funding (Application IDs 3U54HG004558 and 1U54HG006996). M.S. proposed the project. D.R., L.H., and M.M. designed and performed the experiments. H.T. designed and performed the analysis. M.G. and M.S. supervised the project. H.T. and M.S. wrote the paper. All authors discussed the results and commented on the manuscript. M.S. is a consultant for Illumina and on the scientific advisory board of Personalis and GenapSys. All other authors declare no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004812/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Reads from this article have been deposited at the Sequence Read Archive (SRA) under accession no. SRA063146.</p></li><li class="fn" id="fn-3"><p id="p-58"><em>Communicating editor: C. Nislow</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-10-26"><span class="received-label">Received </span>October 26, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-12-26"><span class="accepted-label">Accepted </span>December 26, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 Tilgner <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</span>. <abbr class="cit-jnl-abbrev">J. Royal Stat. Soc. B.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>â<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRoyal%2BStat.%2BSoc.%2BB.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.2" data-doi="10.1101/gad.1973010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manley</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Alternative pre-mRNA splicing regulation in cancer: pathways and programs unhinged</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2343</span>â<span class="cit-lpage">2364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DDavid%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D24%26rft.issue%253D21%26rft.spage%253D2343%26rft.epage%253D2364%26rft.atitle%253DAlternative%2Bpre-mRNA%2Bsplicing%2Bregulation%2Bin%2Bcancer%253A%2Bpathways%2Band%2Bprograms%2Bunhinged%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1973010%26rft_id%253Dinfo%253Apmid%252F21041405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI0LzIxLzIzNDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.3" data-doi="10.1101/gr.132159.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derrien</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussotti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1775</span>â<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDerrien%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1775%26rft.epage%253D1789%26rft.atitle%253DThe%2BGENCODE%2Bv7%2Bcatalog%2Bof%2Bhuman%2Blong%2Bnoncoding%2BRNAs%253A%2BAnalysis%2Bof%2Btheir%2Bgene%2Bstructure%252C%2Bevolution%252C%2Band%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132159.111%26rft_id%253Dinfo%253Apmid%252F22955988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy8zODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.4" data-doi="10.1038/nature11233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lassmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Landscape of transcription in human cells</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">101</span>â<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDjebali%26rft.auinit1%253DS.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D101%26rft.epage%253D108%26rft.atitle%253DLandscape%2Bof%2Btranscription%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11233%26rft_id%253Dinfo%253Apmid%252F22955620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955620&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.5" data-doi="10.1126/science.1162986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eid</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fehr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Real-time DNA sequencing from single polymerase molecules</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">133</span>â<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162986%26rft_id%253Dinfo%253Apmid%252F19023044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMC8xMzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.6" data-doi="10.1038/nature09715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">The developmental transcriptome of <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">473</span>â<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGraveley%26rft.auinit1%253DB.%2BR.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D473%26rft.epage%253D479%26rft.atitle%253DThe%2Bdevelopmental%2Btranscriptome%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09715%26rft_id%253Dinfo%253Apmid%252F21179090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179090&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-other" id="cit-3.3.387.7" data-doi="10.1186/gb-2006-7-5-r41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denoeud</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reymond</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">GENCODE: producing a reference annotation for ENCODE</span>. <span class="cit-source">Genome Biol.</span> <span class="cit-vol">7</span>(<span class="cit-issue">Suppl. 1</span>): <span class="cit-fpage">S41</span>â<span class="cit-lpage">S49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D7%26rft.spage%253DS41%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-5-r41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-5-r41&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.8" data-doi="10.1101/gr.135350.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tapanari</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diekhans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">GENCODE: the reference human genome annotation for The ENCODE Project</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1760</span>â<span class="cit-lpage">1774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHarrow%26rft.auinit1%253DJ.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1760%26rft.epage%253D1774%26rft.atitle%253DGENCODE%253A%2BThe%2Breference%2Bhuman%2Bgenome%2Bannotation%2Bfor%2BThe%2BENCODE%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.135350.111%26rft_id%253Dinfo%253Apmid%252F22955987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc2MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy8zODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.9" data-doi="10.1126/science.1090100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrett-Engele</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loerch</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">2141</span>â<span class="cit-lpage">2144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJohnson%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D302%26rft.issue%253D5653%26rft.spage%253D2141%26rft.epage%253D2144%26rft.atitle%253DGenome-Wide%2BSurvey%2Bof%2BHuman%2BAlternative%2BPre-mRNA%2BSplicing%2Bwith%2BExon%2BJunction%2BMicroarrays%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090100%26rft_id%253Dinfo%253Apmid%252F14684825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDIvNTY1My8yMTQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.10" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugnet</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furey</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roskin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">The human genome browser at UCSC</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>â<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.11" data-doi="10.1038/35057062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Initial sequencing and analysis of the human genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">860</span>â<span class="cit-lpage">921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D860%26rft.epage%253D921%26rft.atitle%253DInitial%2Bsequencing%2Band%2Banalysis%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057062%26rft_id%253Dinfo%253Apmid%252F11237011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237011&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.12" data-doi="10.1093/nar/gki858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ast</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The importance of being divisible by three in alternative splicing</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">5574</span>â<span class="cit-lpage">5582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMagen%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D17%26rft.spage%253D5574%26rft.epage%253D5582%26rft.atitle%253DThe%2Bimportance%2Bof%2Bbeing%2Bdivisible%2Bby%2Bthree%2Bin%2Balternative%2Bsplicing%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki858%26rft_id%253Dinfo%253Apmid%252F16192573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMy8xNy81NTc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.13" data-doi="10.1038/nature08903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez-Arcelus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lach</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Transcriptome genetics using second generation sequencing in a Caucasian population</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">773</span>â<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMontgomery%26rft.auinit1%253DS.%2BB.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D773%26rft.epage%253D777%26rft.atitle%253DTranscriptome%2Bgenetics%2Busing%2Bsecond%2Bgeneration%2Bsequencing%2Bin%2Ba%2BCaucasian%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08903%26rft_id%253Dinfo%253Apmid%252F20220756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220756&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.14" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">621</span>â<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.15" data-doi="10.1126/science.1158441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raha</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1344</span>â<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158441%26rft_id%253Dinfo%253Apmid%252F18451266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4MS8xMzQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.16" data-doi="10.1038/ng.259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shai</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blencowe</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1413</span>â<span class="cit-lpage">1415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPan%26rft.auinit1%253DQ.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1413%26rft.epage%253D1415%26rft.atitle%253DDeep%2Bsurveying%2Bof%2Balternative%2Bsplicing%2Bcomplexity%2Bin%2Bthe%2Bhuman%2Btranscriptome%2Bby%2Bhigh-throughput%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.259%26rft_id%253Dinfo%253Apmid%252F18978789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18978789&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261215900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.17" data-doi="10.1101/gr.10.4.511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">GuigÃ³</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">GeneID in Drosophila</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">511</span>â<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DParra%26rft.auinit1%253DG.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DGeneID%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.511%26rft_id%253Dinfo%253Apmid%252F10779490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.18" data-doi="10.1093/nar/gkr1079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maglott</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D130</span>â<span class="cit-lpage">D135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1079%26rft_id%253Dinfo%253Apmid%252F22121212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9EMTMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.19" data-doi="10.1186/gb-2011-12-3-r22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaghey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Improving RNA-Seq expression estimates by correcting for fragment bias</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRoberts%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253DR22%26rft.epage%253DR22%26rft.atitle%253DImproving%2BRNA-Seq%2Bexpression%2Bestimates%2Bby%2Bcorrecting%2Bfor%2Bfragment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-3-r22%26rft_id%253Dinfo%253Apmid%252F21410973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-3-r22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21410973&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.20" data-doi="10.1126/science.1058040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venter</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mural</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">The sequence of the human genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">1304</span>â<span class="cit-lpage">1351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DVenter%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D291%26rft.issue%253D5507%26rft.spage%253D1304%26rft.epage%253D1351%26rft.atitle%253DThe%2BSequence%2Bof%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1058040%26rft_id%253Dinfo%253Apmid%252F11181995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTEvNTUwNy8xMzA0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.21" data-doi="10.1038/nature07509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khrebtukova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Alternative isoform regulation in human tissue transcriptomes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">470</span>â<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWang%26rft.auinit1%253DE.%2BT.%26rft.volume%253D456%26rft.issue%253D7221%26rft.spage%253D470%26rft.epage%253D476%26rft.atitle%253DAlternative%2Bisoform%2Bregulation%2Bin%2Bhuman%2Btissue%2Btranscriptomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07509%26rft_id%253Dinfo%253Apmid%252F18978772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18978772&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261170500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.22" data-doi="10.1038/nrg2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">57</span>â<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.23" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>â<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.24" data-doi="10.1073/pnas.0914114107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Habegger</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noisa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szekely</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Dynamic transcriptomes during neural differentiation of human embryonic stem cells revealed by short, long, and paired-end sequencing</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">5254</span>â<span class="cit-lpage">5259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0914114107%26rft_id%253Dinfo%253Apmid%252F20194744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzExLzUyNTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvMzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.3.387.25" data-doi="10.1261/rna.2660805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gribskov</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Characteristics and regulatory elements defining constitutive splicing and different modes of alternative splicing in human and mouse</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1777</span>â<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2660805%26rft_id%253Dinfo%253Apmid%252F16251388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMi8xNzc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/3/387.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/379" title="Strain Specific GenotypeâEnvironment Interactions and Evolutionary Potential for Body Mass in Brook Charr (Salvelinus fontinalis)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/399" title="On the Mutational Topology of the Bacterial Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/3"><span>Volume 3 Issue 3, March 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/3.cover-source.jpg?itok=w5x3d6rp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/387.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GYLSK3hLaid42XOCYz2lIpLdbMPXJS1GYziL_cX9gig" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/3/387" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/3/387" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/3/387" class="active" data-icon-position="" data-hide-link-title="0">Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459437" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-A6QO38Z_zAXvRTrbWatS61Z5DVrgfbbXJ7I6XtiQV1g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/387.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/3/387" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3VQ6gdpqD3RqLzX2S4e8qYuocDJGRwodVpD-slH2bko" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/3/387" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/387.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459437" id="node459437--2" data-pisa="ggg;3/3/387" data-pisa-master="ggg;3/3/387" data-apath="/ggg/3/3/387.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hagen Tilgner</span>, <span class="highwire-citation-author" data-delta="1">Debasish Raha</span>, <span class="highwire-citation-author" data-delta="2">Lukas Habegger</span>, <span class="highwire-citation-author" data-delta="3">Mohammed Mohiuddin</span>, <span class="highwire-citation-author" data-delta="4">Mark Gerstein</span> and <span class="highwire-citation-author" data-delta="5">Michael Snyder</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">387-397;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004812 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459437--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hagen Tilgner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hagen%2BTilgner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilgner%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHagen%2BTilgner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Debasish Raha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular, Cellular and Developmental Biology, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Debasish%2BRaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raha%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADebasish%2BRaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lukas Habegger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lukas%2BHabegger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Habegger%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALukas%2BHabegger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mohammed Mohiuddin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Roche, Branford, Connecticut 06405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohammed%2BMohiuddin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohiuddin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohammed%2BMohiuddin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mark Gerstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BGerstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerstein%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BGerstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael Snyder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSnyder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snyder%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BSnyder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mpsnyder@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">mpsnyder@stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/3/387" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/3/387&amp;title=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/3/387&amp;title=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/3/387&amp;title=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/3/387&amp;text=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/3/387&amp;title=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/3/387&amp;t=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/3/387&amp;title=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/3/387&amp;title=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459437/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459437" id="node459437--3" data-pisa="ggg;3/3/387" data-pisa-master="ggg;3/3/387" data-apath="/ggg/3/3/387.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Identification and Analysis of Human mRNA Isoforms Using Deep Long Read Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hagen Tilgner</span>, <span class="highwire-citation-author" data-delta="1">Debasish Raha</span>, <span class="highwire-citation-author" data-delta="2">Lukas Habegger</span>, <span class="highwire-citation-author" data-delta="3">Mohammed Mohiuddin</span>, <span class="highwire-citation-author" data-delta="4">Mark Gerstein</span> and <span class="highwire-citation-author" data-delta="5">Michael Snyder</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">387-397;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004812 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459437--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hagen Tilgner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hagen%2BTilgner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilgner%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHagen%2BTilgner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Debasish Raha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Molecular, Cellular and Developmental Biology, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Debasish%2BRaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raha%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADebasish%2BRaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lukas Habegger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lukas%2BHabegger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Habegger%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALukas%2BHabegger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mohammed Mohiuddin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Roche, Branford, Connecticut 06405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohammed%2BMohiuddin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohiuddin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohammed%2BMohiuddin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mark Gerstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Program in Computational Biology and Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut 06120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BGerstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerstein%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BGerstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael Snyder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSnyder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snyder%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BSnyder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mpsnyder@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">mpsnyder@stanford.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=387&amp;rft.epage=397&amp;rft.atitle=Accurate%20Identification%20and%20Analysis%20of%20Human%20mRNA%20Isoforms%20Using%20Deep%20Long%20Read%20Sequencing&amp;rft.volume=3&amp;rft.issue=3&amp;rft.date=2013-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tilgner%2C+Hagen&amp;rft.au=Raha%2C+Debasish&amp;rft.au=Habegger%2C+Lukas&amp;rft.au=Mohiuddin%2C+Mohammed&amp;rft.au=Gerstein%2C+Mark&amp;rft.au=Snyder%2C+Michael"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459437/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459437/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459437/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459437/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459437/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459437/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459437/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459437/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459437/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459437/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459437/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459437/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/3/387" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/3/387.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/3/387.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/3/387.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/3/387.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
