<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/3/517" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/3/517.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/3/517.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/3/517.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/3/517.atom" />
<meta name="HW.pisa" content="ggg;3/3/517" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms" />
<meta name="DC.Identifier" content="10.1534/g3.112.005298" />
<meta name="DC.Date" content="2013-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Page et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Read mapping is a fundamental part of next-generation genomic research but is complicated by genome duplication in many plants. Categorizing DNA sequence reads into their respective genomes enables current methods to analyze polyploid genomes as if they were diploid. We present PolyCat—a pipeline for mapping and categorizing all types of next-generation sequence data produced from allopolyploid organisms. PolyCat uses GSNAP’s single-nucleotide polymorphism (SNP)-tolerant mapping to minimize the mapping efficiency bias caused by SNPs between genomes. PolyCat then uses SNPs between genomes to categorize reads according to their respective genomes. Bisulfite-treated reads have a significant reduction in nucleotide complexity because nucleotide conversion events are confounded with transition substitutions. PolyCat includes special provisions to properly handle bisulfite-treated data. We demonstrate the functionality of PolyCat on allotetraploid cotton, Gossypium hirsutum , and create a functional SNP index for efficiently mapping sequence reads to the D-genome sequence of G. raimondii . PolyCat is appropriate for all allopolyploids and all types of next-generation genome analysis, including differential expression (RNA sequencing), differential methylation (bisulfite sequencing), differential DNA-protein binding (chromatin immunoprecipitation sequencing), and population diversity." />
<meta name="DC.Contributor" content="Justin T. Page" />
<meta name="DC.Contributor" content="Alan R. Gingle" />
<meta name="DC.Contributor" content="Joshua A. Udall" />
<meta name="article:published_time" content="2013-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Read mapping is a fundamental part of next-generation genomic research but is complicated by genome duplication in many plants. Categorizing DNA sequence reads into their respective genomes enables current methods to analyze polyploid genomes as if they were diploid. We present PolyCat—a pipeline for mapping and categorizing all types of next-generation sequence data produced from allopolyploid organisms. PolyCat uses GSNAP’s single-nucleotide polymorphism (SNP)-tolerant mapping to minimize the mapping efficiency bias caused by SNPs between genomes. PolyCat then uses SNPs between genomes to categorize reads according to their respective genomes. Bisulfite-treated reads have a significant reduction in nucleotide complexity because nucleotide conversion events are confounded with transition substitutions. PolyCat includes special provisions to properly handle bisulfite-treated data. We demonstrate the functionality of PolyCat on allotetraploid cotton, &lt;i&gt;Gossypium hirsutum&lt;/i&gt;, and create a functional SNP index for efficiently mapping sequence reads to the D-genome sequence of &lt;i&gt;G. raimondii&lt;/i&gt;. PolyCat is appropriate for all allopolyploids and all types of next-generation genome analysis, including differential expression (RNA sequencing), differential methylation (bisulfite sequencing), differential DNA-protein binding (chromatin immunoprecipitation sequencing), and population diversity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/03/01" />
<meta name="citation_mjid" content="ggg;3/3/517" />
<meta name="citation_id" content="3/3/517" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/3/517" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/3/517.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/3/517.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/3/517.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005298" />
<meta name="citation_pmid" content="23450226" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="3" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="517" />
<meta name="citation_lastpage" content="525" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Justin T. Page" />
<meta name="citation_author_institution" content="Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062" />
<meta name="citation_author" content="Alan R. Gingle" />
<meta name="citation_author_institution" content="Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia, 30602" />
<meta name="citation_author" content="Joshua A. Udall" />
<meta name="citation_author_institution" content="Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062" />
<meta name="citation_author_email" content="jaudall@byu.edu" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=K. L. Adams;citation_author=J. F. Wendel;citation_title=Polyploidy and genome evolution in plants.;citation_pages=135-141;citation_volume=8;citation_year=2005;citation_issue=2;citation_pmid=15752992;citation_doi=10.1016/j.pbi.2005.01.001" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=I. Bancroft;citation_author=C. Morgan;citation_author=F. Fraser;citation_author=J. Higgins;citation_author=R. Wells;citation_title=Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing.;citation_pages=762-766;citation_volume=29;citation_year=2011;citation_issue=8;citation_pmid=21804563;citation_doi=10.1038/nbt.1926" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. A. Birchler;citation_author=N. C. Riddle;citation_author=D. L. Auger;citation_author=R. A. Veitia;citation_title=Dosage balance in gene regulation: biological implications.;citation_pages=219-226;citation_volume=21;citation_year=2005;citation_issue=4;citation_pmid=15797617;citation_doi=10.1016/j.tig.2005.02.010" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. L. Byers;citation_author=D. B. Harker;citation_author=S. M. Yourstone;citation_author=P. J. Maughan;citation_author=J. A. Udall;citation_title=Development and mapping of SNP assays in allotetraploid cotton.;citation_pages=1201-1214;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22252442;citation_doi=10.1007/s00122-011-1780-8" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J.-M. Chia;citation_author=C. Song;citation_author=P. J. Bradbury;citation_author=D. Costich;citation_author=N. de Leon;citation_title=Maize HapMap2 identifies extant variation from a genome in flux.;citation_pages=803-807;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660545;citation_doi=10.1038/ng.2313" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Cokus;citation_author=S. Feng;citation_author=X. Zhang;citation_author=Z. Chen;citation_author=B. Merriman;citation_title=Shotgun bisulphite sequencing of the Arabidopsis genome reveals DNA methylation patterning.;citation_pages=215-219;citation_volume=452;citation_year=2008;citation_issue=7184;citation_pmid=18278030;citation_doi=10.1038/nature06745" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Coulondre;citation_author=J. H. Miller;citation_author=P. J. Farabaugh;citation_author=W. Gilbert;citation_title=Molecular basis of base substitution hotspots in Escherichia coli.;citation_pages=775-780;citation_volume=274;citation_year=1978;citation_issue=5673;citation_pmid=355893;citation_doi=10.1038/274775a0" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=R. C. Cronn;citation_author=R. L. Small;citation_author=T. Haselkorn;citation_author=J. F. Wendel;citation_title=Rapid diversification of the cotton genus (Gossypium: Malvaceae) revealed by analysis of sixteen nuclear and chloroplast genes;citation_pages=707-725;citation_volume=89;citation_year=2002;citation_issue=4;citation_pmid=21665671;citation_doi=10.3732/ajb.89.4.707" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Cui;citation_title=Widespread genome duplications throughout the history of flowering plants;citation_volume=16;citation_year=2006;citation_pmid=16702410;citation_doi=10.1101/gr.4825606" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Y. V. de Peer;citation_author=S. Maere;citation_author=A. Meyer;citation_title=The evolutionary significance of ancient genome duplications.;citation_pages=725-732;citation_volume=10;citation_year=2009;citation_issue=10;citation_pmid=19652647;citation_doi=10.1038/nrg2600" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Dubcovsky;citation_author=J. Dvorak;citation_title=Genome Plasticity a Key Factor in the Success of Polyploid Wheat Under Domestication;citation_pages=1862-1866;citation_volume=316;citation_year=2007;citation_issue=5833;citation_pmid=17600208;citation_doi=10.1126/science.1143986" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. M. Durbin;citation_author=D. L. Altshuler;citation_author=R. M. Durbin;citation_author=G. R. Abecasis;citation_author=D. R. Bentley;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=L. E. Flagel;citation_author=J. F. Wendel;citation_title=Gene duplication and evolutionary novelty in plants.;citation_pages=557-564;citation_volume=183;citation_year=2009;citation_issue=3;citation_pmid=19555435;citation_doi=10.1111/j.1469-8137.2009.02923.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. E. Flagel;citation_author=J. F. Wendel;citation_author=J. A. Udall;citation_title=Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton.;citation_pages=302-302;citation_volume=13;citation_year=2012;citation_pmid=22768919;citation_doi=10.1186/1471-2164-13-302" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. Freeling;citation_author=B. C. Thomas;citation_title=Gene-balanced duplications, like tetraploidy, provide predictable drive to increase morphological complexity;citation_pages=805-814;citation_volume=16;citation_year=2006;citation_issue=7;citation_pmid=16818725;citation_doi=10.1101/gr.3681406" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=R. T. Gaeta;citation_author=J. C. Pires;citation_title=Homoeologous recombination in allopolyploids: the polyploid ratchet.;citation_pages=18-28;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20002315;citation_doi=10.1111/j.1469-8137.2009.03089.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. Garber;citation_author=M. G. Grabherr;citation_author=M. Guttman;citation_author=C. Trapnell;citation_title=Computational methods for transcriptome annotation and quantification using RNA-seq.;citation_pages=469-477;citation_volume=8;citation_year=2011;citation_issue=6;citation_pmid=21623353;citation_doi=10.1038/nmeth.1613" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Gore;citation_author=J. Chia;citation_author=R. Elshire;citation_author=Q. Sun;citation_author=E. Ersoz;citation_title=A First-Generation Haplotype Map of Maize;citation_pages=1115-1117;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965431;citation_doi=10.1126/science.1177837" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. R. Graveley;citation_author=A. N. Brooks;citation_author=J. W. Carlson;citation_author=M. O. Duff;citation_author=J. M. Landolin;citation_title=The developmental transcriptome of Drosophila melanogaster;citation_pages=473-479;citation_volume=471;citation_year=2010;citation_pmid=21179090" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. Griffith;citation_author=O. L. Griffith;citation_author=J. Mwenifumbo;citation_author=R. Goya;citation_author=A. S. Morrissy;citation_title=Alternative expression analysis by RNA sequencing.;citation_pages=843-847;citation_volume=7;citation_year=2010;citation_issue=10;citation_pmid=20835245;citation_doi=10.1038/nmeth.1503" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. L. Harper;citation_author=M. Trick;citation_author=J. Higgins;citation_author=F. Fraser;citation_author=L. Clissold;citation_title=Associative transcriptomics of traits in the polyploid crop species Brassica napus.;citation_pages=798-802;citation_volume=30;citation_year=2012;citation_issue=8;citation_pmid=22820317;citation_doi=10.1038/nbt.2302" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Jiao;citation_author=N. J. Wickett;citation_author=S. Ayyampalayam;citation_author=A. S. Chanderbali;citation_author=L. Landherr;citation_title=Ancestral polyploidy in seed plants and angiosperms.;citation_pages=97-100;citation_volume=473;citation_year=2011;citation_issue=7345;citation_pmid=21478875;citation_doi=10.1038/nature09916" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=K. Kaufmann;citation_author=J. M. Muiño;citation_author=M. Østerås;citation_author=L. Farinelli;citation_author=P. Krajewski;citation_title=Chromatin immunoprecipitation (ChIP) of plant transcription factors followed by sequencing (ChIP-SEQ) or hybridization to whole genome arrays (ChIP-CHIP).;citation_pages=457-472;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20203663;citation_doi=10.1038/nprot.2009.244" />
<meta name="citation_reference" content="citation_title=Science Translational Medicine;citation_journal_abbrev=Sci Transl Med;citation_author=J. O. Kitzman;citation_author=M. W. Snyder;citation_author=M. Ventura;citation_author=A. P. Lewis;citation_author=R. Qiu;citation_title=Noninvasive Whole-Genome Sequencing of a Human Fetus;citation_pages=137ra76-137ra76;citation_volume=4;citation_year=2012;citation_issue=137;citation_pmid=22674554;citation_doi=10.1126/scitranslmed.3004323" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=F. Krueger;citation_author=B. Kreck;citation_author=A. Franke;citation_author=S. R. Andrews;citation_title=DNA methylome analysis using short bisulfite sequencing data.;citation_pages=145-151;citation_volume=9;citation_year=2012;citation_issue=2;citation_pmid=22290186;citation_doi=10.1038/nmeth.1828" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=K. Lai;citation_author=C. Duran;citation_author=P. J. Berkman;citation_author=M. T. Lorenc;citation_author=J. Stiller;citation_title=Single nucleotide polymorphism discovery from wheat next-generation sequence data.;citation_pages=743-749;citation_volume=10;citation_year=2012;citation_issue=6;citation_pmid=22748104;citation_doi=10.1111/j.1467-7652.2012.00718.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. W. Laird;citation_title=Principles and challenges of genomewide DNA methylation analysis.;citation_pages=191-203;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20125086;citation_doi=10.1038/nrg2732" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=1000 Genome Project Data Processing Subgroup;citation_title=The Sequence Alignment/Map format and SAMtools;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Lister;citation_author=J. R. Ecker;citation_title=Finding the fifth base: Genome-wide sequencing of cytosine methylation;citation_volume=19;citation_year=2009;citation_pmid=19273618;citation_doi=10.1101/gr.083451.108" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Lister;citation_author=R. C. O’Malley;citation_author=J. Tonti-Filippini;citation_author=B. D. Gregory;citation_author=C. C. Berry;citation_title=Highly integrated single-base resolution maps of the epigenome in Arabidopsis.;citation_pages=523-536;citation_volume=133;citation_year=2008;citation_issue=3;citation_pmid=18423832;citation_doi=10.1016/j.cell.2008.03.029" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Lister;citation_author=M. Pelizzola;citation_author=R. H. Dowen;citation_author=R. D. Hawkins;citation_author=G. Hon;citation_title=Human DNA methylomes at base resolution show widespread epigenomic differences.;citation_pages=315-322;citation_volume=462;citation_year=2009;citation_issue=7271;citation_pmid=19829295;citation_doi=10.1038/nature08514" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. J. McManus;citation_author=J. D. Coolon;citation_author=M. O. Duff;citation_author=J. Eipper-Mains;citation_author=B. R. Graveley;citation_title=Regulatory divergence in Drosophila revealed by mRNA-seq;citation_volume=20;citation_year=2010;citation_pmid=20354124;citation_doi=10.1101/gr.102491.109" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Nègre;citation_author=C. D. Brown;citation_author=L. Ma;citation_author=C. A. Bristow;citation_author=S. W. Miller;citation_title=A cis-regulatory map of the Drosophila genome.;citation_pages=527-531;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21430782;citation_doi=10.1038/nature09990" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. C. Osborn;citation_author=J. Chris Pires;citation_author=J. A. Birchler;citation_author=D. L. Auger;citation_author=Z. Jeffery Chen;citation_title=Understanding mechanisms of novel gene expression in polyploids.;citation_pages=141-147;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12615008;citation_doi=10.1016/S0168-9525(03)00015-5" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. J. Park;citation_title=ChIP-seq: advantages and challenges of a maturing technology.;citation_pages=669-680;citation_volume=10;citation_year=2009;citation_issue=10;citation_pmid=19736561;citation_doi=10.1038/nrg2641" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=Y. Van de Peer;citation_author=K. Vandepoele;citation_title=Ancient duplication of cereal genomes.;citation_pages=658-661;citation_volume=165;citation_year=2005;citation_issue=3;citation_pmid=15720677;citation_doi=10.1111/j.1469-8137.2005.01347.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibers;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=P. Varilly;citation_author=B. Fry;citation_author=J. Lohmueller;citation_author=E. Hostetter;citation_title=Genome-wide detection and characterization of positive selection in human populations.;citation_pages=913-918;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943131;citation_doi=10.1038/nature06250" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=A. Salmon;citation_author=L. Flagel;citation_author=B. Ying;citation_author=J. A. Udall;citation_author=J. F. Wendel;citation_title=Homoeologous nonreciprocal recombination in polyploid cotton;citation_pages=123-134;citation_volume=186;citation_year=2009;citation_pmid=19925554" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. E. Schranz;citation_title=Novel flowering time variation in the resynthesized polyploid Brassica napus;citation_pages=242-246;citation_volume=91;citation_year=2000;citation_issue=3;citation_pmid=10833052;citation_doi=10.1093/jhered/91.3.242" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. S. Senchina;citation_title=Rate Variation Among Nuclear Genes and the Age of Polyploidy in Gossypium;citation_volume=20;citation_year=2003;citation_pmid=12679546;citation_doi=10.1093/molbev/msg065" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_author=J. C. Pires;citation_author=A. Kovarik;citation_author=J. A. Tate;citation_title=Recent and recurrent polyploidy in Tragopogon (Asteraceae): cytogenetic, genomic and genetic comparisons;citation_pages=485-501;citation_volume=82;citation_year=2004;citation_doi=10.1111/j.1095-8312.2004.00335.x" />
<meta name="citation_reference" content="citation_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. E. Stajich;citation_author=D. Block;citation_author=K. Boulez;citation_author=S. E. Brenner;citation_author=S. A. Chervitz;citation_title=The Bioperl Toolkit: Perl Modules for the Life Sciences;citation_pages=1611-1618;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368254;citation_doi=10.1101/gr.361602" />
<meta name="citation_reference" content="citation_title=Variation and Evolution in Plants;citation_author=G. L. Stebbins;citation_year=1950" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. A. Udall;citation_title=A global assembly of cotton ESTs;citation_volume=16;citation_year=2006;citation_pmid=16478941;citation_doi=10.1101/gr.4602906" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Udall;citation_title=A novel approach for characterizing expression levels of genes duplicated by polyploidy;citation_volume=173;citation_year=2006;citation_pmid=16702424;citation_doi=10.1534/genetics.106.058271" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Valouev;citation_author=D. S. Johnson;citation_author=A. Sundquist;citation_author=C. Medina;citation_author=E. Anton;citation_title=Genome-wide analysis of transcription factor binding sites based on ChIP-Seq data.;citation_pages=829-834;citation_volume=5;citation_year=2008;citation_issue=9;citation_pmid=19160518;citation_doi=10.1038/nmeth.1246" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=M. Lee;citation_author=T. A. Wilkins;citation_author=A. Kozik;citation_title=Sampling nucleotide diversity in cotton.;citation_pages=125-125;citation_volume=9;citation_year=2009;citation_pmid=19840401;citation_doi=10.1186/1471-2229-9-125" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=M. W. Vaughn;citation_author=M. Tanurdžić;citation_author=Z. Lippman;citation_author=H. Jiang;citation_author=R. Carrasquillo;citation_title=Epigenetic natural variation in Arabidopsis thaliana.;citation_pages=e174-e174;citation_volume=5;citation_year=2007;citation_issue=7;citation_pmid=17579518;citation_doi=10.1371/journal.pbio.0050174" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=57-63;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19015660;citation_doi=10.1038/nrg2484" />
<meta name="citation_reference" content="citation_title=Adv. Agronomy;citation_author=J. F. Wendel;citation_author=R. C. Cronn;citation_title=POLYPLOIDY AND THE EVOLUTIONARY HISTORY OF COTTON;citation_pages=139-186;citation_volume=78;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. Wilbanks;citation_author=M. Facciotti;citation_title=Evaluation of algorithm performance in ChIP-seq peak detection.;citation_pages=e11471-e11471;citation_volume=5;citation_year=2010;citation_issue=7;citation_pmid=20628599;citation_doi=10.1371/journal.pone.0011471" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. E. Wood;citation_author=N. Takebayashi;citation_author=M. S. Barker;citation_author=I. Mayrose;citation_author=P. B. Greenspoon;citation_title=The frequency of polyploid speciation in vascular plants;citation_volume=106;citation_year=2009;citation_pmid=19667210;citation_doi=10.1073/pnas.0811575106" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads;citation_volume=26;citation_year=2010;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. S. Yang;citation_author=F. Cheung;citation_author=J. J. Lee;citation_author=M. Ha;citation_author=N. E. Wei;citation_title=Accumulation of genome-specific transcripts, transcription factors and phytohormonal regulators during early stages of fiber cell development in allotetraploid cotton.;citation_pages=761-775;citation_volume=47;citation_year=2006;citation_issue=5;citation_pmid=16889650;citation_doi=10.1111/j.1365-313X.2006.02829.x" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=J. Z. Yu;citation_author=R. J. Kohel;citation_author=D. D. Fang;citation_author=J. Cho;citation_author=A. Van Deynze;citation_title=A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome;citation_pages=43-58;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Zhang;citation_author=J. Yazaki;citation_author=A. Sundaresan;citation_author=S. Cokus;citation_author=S. Chan;citation_title=Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis.;citation_pages=1189-1201;citation_volume=126;citation_year=2006;citation_issue=6;citation_pmid=16949657;citation_doi=10.1016/j.cell.2006.08.003" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
<meta name="twitter:description" content="Read mapping is a fundamental part of next-generation genomic research but is complicated by genome duplication in many plants. Categorizing DNA sequence reads into their respective genomes enables current methods to analyze polyploid genomes as if they were diploid. We present PolyCat—a pipeline for mapping and categorizing all types of next-generation sequence data produced from allopolyploid organisms. PolyCat uses GSNAP’s single-nucleotide polymorphism (SNP)-tolerant mapping to minimize the mapping efficiency bias caused by SNPs between genomes. PolyCat then uses SNPs between genomes to categorize reads according to their respective genomes. Bisulfite-treated reads have a significant reduction in nucleotide complexity because nucleotide conversion events are confounded with transition substitutions. PolyCat includes special provisions to properly handle bisulfite-treated data. We demonstrate the functionality of PolyCat on allotetraploid cotton, Gossypium hirsutum , and create a functional SNP index for efficiently mapping sequence reads to the D-genome sequence of G. raimondii . PolyCat is appropriate for all allopolyploids and all types of next-generation genome analysis, including differential expression (RNA sequencing), differential methylation (bisulfite sequencing), differential DNA-protein binding (chromatin immunoprecipitation sequencing), and population diversity." />
<meta name="og-title" property="og:title" content="PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/3/517" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Read mapping is a fundamental part of next-generation genomic research but is complicated by genome duplication in many plants. Categorizing DNA sequence reads into their respective genomes enables current methods to analyze polyploid genomes as if they were diploid. We present PolyCat—a pipeline for mapping and categorizing all types of next-generation sequence data produced from allopolyploid organisms. PolyCat uses GSNAP’s single-nucleotide polymorphism (SNP)-tolerant mapping to minimize the mapping efficiency bias caused by SNPs between genomes. PolyCat then uses SNPs between genomes to categorize reads according to their respective genomes. Bisulfite-treated reads have a significant reduction in nucleotide complexity because nucleotide conversion events are confounded with transition substitutions. PolyCat includes special provisions to properly handle bisulfite-treated data. We demonstrate the functionality of PolyCat on allotetraploid cotton, Gossypium hirsutum , and create a functional SNP index for efficiently mapping sequence reads to the D-genome sequence of G. raimondii . PolyCat is appropriate for all allopolyploids and all types of next-generation genome analysis, including differential expression (RNA sequencing), differential methylation (bisulfite sequencing), differential DNA-protein binding (chromatin immunoprecipitation sequencing), and population diversity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/3.cover-source.jpg" />
    <title>PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mqwESJkZ_mKnQtWY8sfFdIZXRET3HIpkY05oIIwfKZg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459424","apath":"\/ggg\/3\/3\/517.atom","pisa":"ggg;3\/3\/517","ac":{"\/ggg\/3\/3\/517.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/3\/517.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/3\/517"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/3\/517"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/3\/517":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AYxvfUqTqACNfkdYAXhgWxKje75TPEBiWROz4D8BNrJUgL0pEFZ5eB1fo6SMJF+p\\\\\\\/i626JRX1Uhqg4YXjp2OZnc60Fu7ahgN79y2wvG9Fwc7u4bmeL5ur94Jc1CW7rNRjhAaTXNL4jZB4zMuoEJCvbV063smu7noNY5HYpsxcPGIbbZaGseiBkKJUuLLma5Sbhr+5a5P1KTYMWo0SXlITcNhkzINPDfRY43xUDxAurypNU7RhvQRkg9NqkPqTiKBav\\\\\\\/Miry2eNybYK8LqxkXqxr1dc0tEC\\\\\\\/pmEkwfUYgI+PmBH5zUXTYCxf\\\\\\\/8\\\\\\\/YEbt\\\\\\\/othGqbtRO+fS23JJSViTcuTuuUlr+3yG7Qtqjg+cH\\\\\\\/wnhgypaJpRLKVgqDB9dac5J0bjkgsjHj4gr8dqr60IWAPIJhdbE2N\\\\\\\/O2OesMREh1L3DVRoXgIaFbJ2k+qHku3VtUWVCMym4JcKlytnFiJinzWmT2e+\\\\\\\/443Tz9Z4AygRO1vkvZZRdjpREvtV58KGgSHPAmeZ5t8u9Ja3Qqn68pe5APH5POlMlSm4xwVgijewhNtqmGQGn4f5B8xLWmR3mJGa9jBvEXrY1VdvIZKOurrudUjzCFcuPjF0S+zszzZ879PtBxo561VvsqNfxQhvb+54gC5qm+dKr7Eh\\\\\\\/BiBomgj9wtAOjv1zDCbfGYsy0x8CNb2ngBXDuOqW61ZkLFG651S9xQEZMv6sewwUZY9uw95bfvUNGhyPYvuRP6hONaFDpYzkdYxjgzTwUkzhX9IsIgFIPoHd0IHVc9u2+Vm1GbMgZlnQMHFAG0KofT036YcGA7JiDny\\\\\\\/5DoM4ddUCNlfAvJaNRb0zf44BQW\\\\\\\/aJ7+68hp+EjMqZQgepOFKopjnLTyHDCBOH7Mm0uFgttvDZ+wGFYgywRBmQL\\\\\\\/++RFH6DDHYkyz+OmLTaChnhlXsKC9MIx9GmRjScxQ6e4qBBIS3O6mPTC1V03bttqDd0QD4xpFk9Oxad+5Jbyp82XZaSBhB4EiV\\\\\\\/bDXk9iR9IgKdRpswMa7sI2OknCIoAIXxEDwhR4xpHOUmYyuCjXQdKTbn5hMXfQ2zpgkQ6rAQvlv9I48AmIPr4s31naaAXq7bW8hfNg3tkeKJxZ53wRz0f69Zz5v7au1CuDw4lobccQ9FZ8cxGnqq6eOndYJtp7DZZHujLCpEXsOqDf6r+iThiWJln1a3nrS88woBNP0Bvg6qATlcKc2eu06XoTK1CjgwO1H0\\\\\\\/Swprolj\\\\\\\/GVglpvcpHqapgfsuR\\\\\\\/kj6YesgzM0OktkdjJvlOP\\\u0022,\\\u0022iv\\\u0022:\\\u0022QnyF8ivMauBigSDywWLmrg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002241b34fd2ea2a95bba099bf3b281449c2d5bca9928d180197d9e36bb89dafaac8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n2MPNYyN\\\\\\\/t\\\\\\\/VYlcoWqQc7Y8O03QtiUhlyp3ILsTU8Jr5uucnWVwY+rwCv4Gp2ixwxlWQsg0JVGdvf+zjA\\\\\\\/Vw8EzSZ2wPX77kyaVu2z0AH38d0qxHZFcQ3d3UaZKSzR3CrQjCY91x45UqcgznNlxEH1Prf6+RMgPYF1DbEW9PM8n9Bw2shCA5sPZOnZfz8UqQoz5ZJ\\\\\\\/v7D7pV1HGozGeM+2t977pEB8GcC7WoV1UN1\\\\\\\/JyfpJjsuqHZ+82KRT2xuHHUEIJYbvOGethUOmDKuC+gqUtmxBvByubLJwBSEp9R7R3KjGGzii7LjtXVLStqAjGCHf4rbIDzCCo8tCGwgB56f0aO2jtDVSNfGcvxfdXAQefyf\\\\\\\/HuiV0+79FwVZwWBo7Y4yH6e5UNaeHOC5aALz9YaUunFxOl8GXmM3XPjdJzBZv4g6gtIYnwPkTuIgaHu+OXQPmyuu\\\\\\\/R2CziGzY0+tW07525gwj4zW4LAhhOn\\\\\\\/1nNVYtbHeLnVCzp9jah\\\\\\\/de5xY00rHU8mv2SZBo03gK+ln\\\\\\\/LYrthMbB2SfnAYza3g21hVeeEQZl1+AeJicv+YAQn8tvsbIKqOExhSYtY40Mv1kJSMlXjhWsjI6xgbUPAcqKZWt1x3HmDImQUZiTIhPfrSMjH16ipSPfa3oZhflho\\\\\\\/45COmzS8RZWfgLrTN9uRg0GkLyUxcMX4enrBwx2s0DZi3NSJihZe5vgGVjLO36JwFsXbbnsztibYCiqKDiqxyR1QRdyARL53B3jSUAoNqEruyRjbaBKbySypAXdFQ0x5Vge0KH1A9SQwihyB\\\\\\\/uTiR2TWQPg4QXuArjk6H8N8Hope+J5CjSnsCChZzME5Ijr59n7YpcYgTQLm\\\\\\\/8dwuJJvlHd+rcusu4ftm2jOTC\\\\\\\/uhwS8B+5jTSU24f0tzPYyrXDs3bWyDiXXD6XbQR3VJFzozxeSPj4z5kPDmptmPeGoLr64oS7tQ4pHfwGIkpg5sTvYSBtv5d7dupbglcrePryLtz+0sIhetlQ7\\\\\\\/YBnmHWFoDYB3SNezJZa5ygYboj0jrRH5b2ofZXry7XTQJS1ums3iycpG9ou4LTw8PVtYosh7xF6C496M7hyRkqnBpPsF\\\\\\\/4Jn\\\\\\\/sxFGUkquVflRKIHqRFa5B0lmFl1um0287ykhDrQB0cjIChucmGzfMCSFrIYdQecg+Az4egWhm2IGZvqxJoc+VtV0KrIBhNCTUmmcSkdw2867KtV7owiYEKjxawwECzWCM0BhkegXx0asv0O1OiaDYV7CAETZvf29GoXQYZkHweMQoZGfJ\\\\\\\/2sf5HZ+F7V3G12U+GQLR1f1eqMl89eLKBv60PjZvT0zujWyhioVyn10wV\\\\\\\/UUfETIS+EoLCHWLL9HrEU\\\\\\\/cDvuR9JhkH\\\\\\\/uAXb\\\\\\\/mUlu8jNnrqSDSFVQa40EqL7QpFeOgCD38LSghd5SerjzaGKngtkjAKVaOnc0PUZb2kef5VQ4GMSh7ou6MMoGfMryVEgE+taIDbKZy0NMmUZWAbQ+EsWvibLcgujxcyooiLAxH33yzcFH1\\\\\\\/1rXnWwLNflqDIiPh\\\\\\\/MsXcsKuAMmszgcYpkJLrBaOmNgay+Po+5WhglWGwbPNidPgkurYVIzAtRkzHQuDKnaTwI39yKaIFaLRSjairLm1Y\\\\\\\/AhsXMS6ZlGI3Vmx2Qwjq2dal\\\\\\\/QVdw8wikG6h7fK3NuHIHzDGCAa72onx4jIucSHw\\\\\\\/f+I8+wnoXDhNd5ma4dOTs3fOdu5Kp9wvKHzOe0J7\\\\\\\/TivYLxa4\\\\\\\/gi3DPexuOMcT7t\\\\\\\/iMn9O7Y8SpULQ1+XZQ6DVEmsuLu5p3EcAuF8zJVUzCA5u4VS6Wf3ASniJCt3pDIG8r5dHNIsiylmKfjqfyXPFQBWn0BHWXR2iUPTTwURYIIos1PgQo8hEeQuwj3ptX7fSl+6W6eQyRfjaG5CS6PjXXiRDmwBHYopD0vgCrk+V+XxQpT1a5hapcHf4ydJVfT96LDF9H5bE2aTWgtRjfPPiywq8Xvp\\\\\\\/fPKcnarE\\\\\\\/jvBGCNFDj9NeldTMMbX8gSS1AJ3cMQeZjujlVhxcpXVh4z7idcpQGHklGl\\\\\\\/7fx26oVB7vQvP1oXRFD1ac4rwKWmQj66uAnB2urfe1xSRUrfhhD\\\\\\\/ULX1tOMe1ZVyGO0KJNd4JB9o9bTp5f7NfufcLVySBUhaKz+oWRSIfl7VkUGRbu8C3cpEHgj1gX5iSiuyBgaj76Ef0RakEIfg7ot3g3S0TGVLMaosQwiWswBdG2dfHp7AM9QIw8duDL5b3pp+kKo7uqgxNFGEOFpuZZ6PbM6mGwzOtvsYj68jf6qfY\\\\\\\/70yTisKZ22h4acwratKFLCL4WglrijacFK\\\\\\\/0zwUc2pYJMlEJiw90u8opqDn5kVJAdqv+Iud34Zk7gf8un7V9+I1pnIuIbgz9MjCvqqFoetse8cdpuCgl0zaKIM6lVil9pAwQacl0+2rVSQSLsXCEp337AyjiMLbJPnXKo8Avcjmiazwpsi1HWA9ke7rdb8sWTBlf1E2G8LyZznNGYAFSaY7FrIH2Ag0JASGzYeFI5PrRK\\\\\\\/a1OXC3kMKiN3w9518P3+IBXDw2X+UBKrAmj407YUYj\\\\\\\/EZmmhTuc9cHXwFfVm0SYdmiYBWTbl4EqkC2JZUd90V30wshdZ0pF5Vk18+Uugnsae+ypqrf+g+lgB\\\\\\\/ZQyacfS6WfwM+3+cq5dPVYD3CMxUnp40TtpXLh4J6GvBstunHn1IqU7ZrJ\\\\\\\/RDgQGO8tb8AUxFGyxeWkiRkO7ASwkW5Rpi5YYFvYwDFAEi+ZK256jjWSjt+4Q8OoDzLQcOUDLaCOOmChCTzXLIODv098+W2Dgiz6UjL5fQBNL2GrSO9DERcQtsEG0j66EDejSrMC53V6wpAynDMnbubEKosq5+e9kWEfpiPIDJhrZdN1WWluOGuxsX8Gd5r9jVi1XE5YPEK\\\\\\\/cmIRMl9S5D+j257mrE34bcaSKJ8LrY4m5tx82JjxCf0p6sNM\\\\\\\/RAHhHBca40T2krY6d4LRnYmZ+QPCq0p3syaX1npQk1hTOWKdo\\\\\\\/721sfwAVSjW6xsBJczO8XVwZTH4AgZ\\\\\\\/rBEwtSMi+7SPuLWXTL0\\\\\\\/d\\\\\\\/MH4B7Z5R8Fr0dzt6rdM096kV7oI3sBhbyrA90jJeJiu5IcuSau+UHxWC6aGkfvKrkssyKnI2qhitcbqD2fUpfh9nGWoOnN4tRzL7rgNvgIj+WkCG\\\\\\\/6S1d8CgVS59NP0a0Y7JetDiie+\\\\\\\/8zvSLMTRdsH4AurWu0IwNdY\\\\\\\/pPFVx4DqTnqac3RVFHet\\\\\\\/moJ1nfy\\\\\\\/esaMUmfgGerZ6+q7zTVGnmZKrPqGmq6yAtp+Vt36XbzvDjr6hiBK4cBAmUA9tihSksIlpX\\\\\\\/WNXuk13tNCBe9WHHMVubgJbCXYPAeEn5jOxQHr6kkauVkhsQ\\\\\\\/P93r24xpF5QZ7lYTU\\\\\\\/Pt2zXEAhhol\\\\\\\/TSU3ohC\\\\\\\/R\\\\\\\/5N+goTOLLKtU9WrYkyo5GqdqoQS8OoEcew1Jks9TsOiOcXoyrFFaUZfy4U0FlQBP9AAP4jNQzxVTs76VH+PPcttjmezGz9KZIWPTfaaVFAmT2GLcs2PTzW3B8op250Kxr5hRpsvOLkI8QcIOvaJbfULQEUJ3yCFB9v9ddVeiPvUAkD9N+OxmemSyGNMYVsiTbVN49oeKTT7\\\\\\\/Fih0Rk+iQiB8\\\\\\\/L3W2J2x574UO1kzDKlTukfxTLfApip1qdP5WlNJSo\\\\\\\/Jyi0w4dILnR8qQGBxbSnVBTHZkezbEMcTPGx1RYb3xceK9FbY\\\\\\\/8YOsohcB+IJ3fbpwPjfPRaA4N0oOf7NXyMnm\\\\\\\/eyfbQIVIfqB1V8RaqTf5XeWe+FzoRIZi3WVEsoQE1oyEcNLJ03gKpnjpvAg88YvJoH5xsPhO8SUDfkmyGuSzSkGB+Q6DkuFN8JMSlHMdmCgW2PHZ99UqH3Ara8kybDtPY5yvVv1u1fOmRKJdZGf9hXvjLDcFWcS+M\\\\\\\/1s4mRGJZy+IVb4jxa3kq9qCJc2HQ86vNlUAlnhf9J2fw1i6HdWEdqR2vHQzQgvwSPMPbriF2o\\\\\\\/Ng+Q6MH9UvFVQlX78j8Vdx2wRbWItM8I7DEteaGjKBDHV3Op8C1m+glNGOoFWYW\\\\\\\/YB\\\\\\\/qP5JSlgkYOAxjeGB1dVbbe\\\\\\\/hArQSt+JAa5G89ggOcGeiNcjw6bzH7BmgOvVV+GIl+Ql86oWzcW3VzZ6xKSRAPuOjL5pqCuUK7pBINht5\\\\\\\/DO\\\\\\\/Bxh7woH8Eu1KFfDL0Gq0rJ8buBJYvDm2a7IWgbh1YGEf4KdnuR\\\\\\\/TyFCaCExma0YZPXcf7LOoob9ZO97V4OCp3RIjxCqkI4c5FKDvrd8nFB3sePurT5aAkgkqzdjYIrKim7IrEOPzpPEgg5XSCGypGpiqOiHHKUMTCE3LFlAt4SVSPNhHniKuhYyAXsy9xA+CfoGPwLb0UFTzgVOMaXMgKFypgHDSkI1aPqepDvYN3EFfDcokfcZHUE3PcuV5bkmoHrgN3XLNCB\\\\\\\/uldst6iGrMC4hlrWbH93NDU7p37UxYrAhXanzJtVFR6MV3Z8WomNDLtRz2rBLuQeZq34ubpPlgo5TUhqccH3ngXd6OP7VJ1\\\\\\\/4iExNO\\\\\\\/L1dN18ddnjCq\\\\\\\/y60BNOvsI1ti\\\\\\\/LwXzTDcuvs7eYveOxBOffiwvAvHxIgl50Z8NmfTibBLzh6hq04TgjtYemVcTB64XB\\\\\\\/XVzrbtOJoiLzzEMnjHA0uruT+h\\\\\\\/rbMj5Ly5B66WI2aPmePopcyVAPEhwcm+WuT6xWkbd9Ey2ovPnEcL8Dgu\\\\\\\/a9n21GgoD9EV\\\\\\\/vQqkuHWtJ\\\\\\\/NYbSTibpgl0f6VUDoNKFsqULGuJ\\\\\\\/aZVmPWVH46IMYJd6CgYNeOrk5h\\\\\\\/N1PIZjc9NSTvHhuWsIu3EXJQn1GCotNOOWUpeRNPKYjsDfhFqXQbrkeKew1QEi5s6Qqm120FhsSyIO8u8TYNGcef\\\\\\\/o+IzIzjKlRHXSWy4VBuLq9ul\\\\\\\/xP9hHS\\\\\\\/hvSRXIT0\\\\\\\/H4ApUlEpfB2k8mEtoKt\\\\\\\/r78LzOqhExYiDQnFbTN6Pmzf0Xm0KfKDwsGJ0l994LYmAYTrrQpCRsnE9rfO83x\\\\\\\/TBK7wce4Fee9yU\\\\\\\/9wz5NrCS6w8jJHmsys+V0Ei3GKtblBSRuo5OE8zC74LZk7hssR1832KQ85uvPREgs0XDgkXYWkWuH80ACLyG4RgNiRppMRZoJpRx\\\\\\\/xhSozc6ZJwgGaNRVNlclCCughLnK2wnHgF9GSjAch3FwIKhv7erlPgi3xpjgsYIs+rnpOlWDPac+D2Mg4lnbGt76cdo2PmSbsygvyr6sy5E0PAtF3P01id5b0XY7l8WXME9EPLJz8Vix5uMUtO3KoE4j5iTGm5jZswVKzZKdCb0rPAcd+fML5gS7GDWGj60Hv0VbLtv49fe5W0Jq4ah0PVedrtf75dxwTf59RYWFM40pK6XyM6W2gjDuhuz56A7zUi1jbNiWNMFyXrucAkF2OCMaHZx8kT0xmrpLxB5OctjZHcdebLCm11w6eo\\\\\\\/oFTmHTwiYqoe8bIpsRDi9TGlEZJTAF\\\\\\\/GkeCjEu8wLZKF\\\\\\\/Mq5IuCC2cbpZtrOHkMz4xxL7OUEPSQX99dZhJpnyEKiTj8y2UWumUNvjl4trOW8r2eUfEhvp7U+0fDZUY6PyDOTY8dNzstWrA8e\\\\\\\/WeP3IkP7XOoR2g\\\\\\\/aTlZXHR6w1KZfjG3yUlv\\\\\\\/yOUqdOHA64biH7SNsdIUf7JoAHVidOhCHdZuuwjr1ycQOmtFQbuDFP6BUOeLuovRw7A6ESEL9GMPpCJ9zOPc1N9RXstBBZzJBaHtT1wtU2Fx2HV0PH1sWxx78LqlfSEu+knhJCcV\\\\\\\/rXU\\\\\\\/NGzAt4i6Zjqw50RnLTvjp+mJy\\\\\\\/UB3mF2sX9z0vOM1EEoRpX8YyzVwzwkxXytdG\\\\\\\/LSiALGRUwhMY+1hh3SXqs27zgzwQe6\\\\\\\/CryWK\\\\\\\/8IIPIVyLcH9Ov+ZekL8z8V6zqTCAPc3jIB72LoFLudfRNrF7ejdRhaeN9+e9dV4E4ygW\\\\\\\/UMWgcT\\\\\\\/ETnjHdDwnHNVPecbHfzYtPIcStL4uSrotpf+7ECFovZWYxRI0YQ3orVWedRlhDXljzNDLb0OSqQeCPOFkM1+ju1OYycNVNrPLDvsJJW3N53Qw3snXZDXm1LZIsZ5L1QnZZZPnEivr89h6tU0z8GgyV7haWyv4\\\\\\\/etSncGIqma2tJ8v0Wx+Znp6rQO\\\\\\\/mT+biSc3WpuYOkB3wZG9lvyiZ9pPHebYVAalXlcWd59fJVMurHxTmdfjWXGnSfxSWAvdzWkQxbOXDPX+HAnCfkzw4ZCm7wHIspIivfIK9gsOS9ulPQYB4llgXnbXQ3+evh1s9\\\\\\\/fLBq\\\\\\\/0X649uC7+9KSj9AYsdE237UaFVmEt+2Tjfj5Gg2NWTr0QjDl8hMpkFn\\\\\\\/ASM7VAzZ\\\\\\\/uLihoEWI9GX6y9arJTRG05RG2nxDCq1i0AANjFqc3QvGuNulawNQsHkXQx8tAO7uabSX2oU0xQ9yombW1\\\\\\\/DaMTR9\\\\\\\/ZQIVJYua9X79azxRNhmiJAa+TA4CFxAQoUEWM\\\\\\\/eklCTGpvXf6RNAPbcujelt68MoH\\\\\\\/HkCZSeJz\\\\\\\/pkLYx3MxC+iKkaSSy3EaQeFUAQRVacZeVEIdVSjL8e0V4jTHZXZ+6to0O6I6ltnIwl5aRybpJYMwDWOU5cImES\\\\\\\/9wtC6\\\\\\\/tUGdJig8OfbKZEKz3aDKZB\\\\\\\/tDJ349qP6wY2cfa6uCU4sKueslhxXsZjm46zv3jwWoVW+glDRnfqRk6zmRpXdRanm4t5UPm9NneWPRKgd4hPsioisd2LRyuy+vXnesyZmLGEOzG03TpXU1J\\\\\\\/lYjPhYEQmxIYR+LDDIyfKD5IZc4kp7amT7x5Kr3xhw+Nn9oBUbsz09vTLsUtoT3SZ0pf+rjkXgU1SIVg2lxiKX8Vl4xrOClDPLH+5HrT2NYnuktTs2kONp\\\\\\\/FXlGnpsK4Mez2hPeVV7f\\\\\\\/8k7lI+l9z0ervUNChm84yNORp83wzA9pmUKuJOfn2CguIZNxoroQxAADof9Zv\\\\\\\/BnxvTdHS4LYkbjK5n9IbXGX+hg4AnS4bd58ODPTI0h\\\\\\\/feQM\\\\\\\/A8hAa07V9u+UtTNW8IjDyy7Vit93yxPtNQnu8AoMAe8Adh7nF00VFiUjttU2aFKsrvX3tKxpkijdD3cLV4FiYAyb\\\\\\\/ZG1lSce64W1x1aYRfulpRRSKZDiwY3zECBUDNcazbXEOs04bneiYrygP2vxwbIW2sRIWjIfqahQC0v5ILSqernovGin1YyBkRi4cdoazf+kW2PqFxgDCAfNj98hLa6fXzs0iVaqa3D0AemVEbPi7E3wWBuq0\\\\\\\/PB8sCZ03j\\\\\\\/ldBUutT\\\\\\\/ujywxP5TD41A5JqXF9bucJsk3WzvAMgUuJTkIynXXCQ\\\\\\\/4Nu9174Kf5jW5DJGoRmTOELp4yI9O310NBMOfYTLln4Xh2WtreRdhvzk2kEjOOp1sTs4RvrnTKQOxYrFMyFS6EnOfbXpDYWnx2oEIj+QwhxS3gAoF6Bc+QwBSIpxQh17g6Fk6F5mUY6Sr191YTb2k8VCMrDTIOCkyPSiz0SBF8k\\\\\\\/f\\\\\\\/42TejqR3wOY7KWOmoc+K\\\\\\\/aXrvEg8TUcxSx1wft\\\\\\\/MZS42EgSkG0PPbrFF4xsfMqa3W4jirh\\\\\\\/BAp+TkPdtzKCYZ3oi1cnVy7tB8D1sAe7\\\\\\\/OUiH2BR\\\\\\\/vbHd+oh0go86HasffNKtMwUiHCPQoCm5X3LI+S3Mh1MmUxC1qDLydw7owZCm4fiEnXxBbDg\\\\\\\/aL0wUAjUHAb16DzsyOOft7Q\\\\\\\/eUuWmseQh3YldQKuKSponlgI6Xs7ZXe6+nMKdTiUd0QUABOUPJzhSTMexila4ugEP6dfEJGUycB0t8VSZw7UdtdYuNpMXjIxRLALtwvqFGqp5RqTZLLkwrNbrkAQKBLhp0bGPDNRcEsD0EOl23CzOohMUQTzMeYLf5DXeeqkh2DvM2KUZQTZ+u4pq9080DqzzKdhc2cRJjkBSSZGl2bzxOlw8yVoqy2\\\\\\\/x72+lGtGD3CNVrTwASaC5A7cEGSpO5oJqwXdJFQddjCBfQOjhEHZc3pdbnVtjgg8JcQEENxJU7GwcNhGiHcmUIkGNZFwWbbrZoJD5\\\\\\\/Tp8dYCJFBG7NyqRT0ZEUrvHD88vKjrNNqb\\\\\\\/YS+2roG2VQCa1RCdXR74aI+zmqdqT1mCsDV5HQp643JWgkD0d7I01owXR3adRuhCpwquLH0\\\\\\\/\\\\\\\/XjWfU0cPKtapuUEwElLPMC3oUcT5PA8GIy50E1WzLlPSkMY\\\\\\\/SxV6UVr6tNnqvb6n6Jyr9m6Qm5uuT+K7wkHtvKP0IXmAdWPrIV9T1xiHt8b8jdVvE9LxsEiF2\\\\\\\/x4ewWKw2Ri\\\\\\\/9mw4UBjuU8SjLxYcLg1v\\\\\\\/ozpHuARnwUxN2Xy\\\\\\\/D0eVI06ltpzRMcdlKgZVMIFCMVG6aybQcnJKdxZP\\\\\\\/o9DAQS7KO\\\\\\\/3HUY\\\\\\\/zmwGxfCJw7NoLiASbqzx9N4FHTrCo6GtSVyZocjBFV1O4Exc8DXwYeaY2rBNuVGh9qshdcbsGUeLdSUUhQ3hIFLQtKDVZjSUpIuCbgLt+nH8h8YyNgQSy3\\\\\\\/tbVnm2YQbD7q0LbbQ20I+wkGzFfnr4OV55Ji4IcK5DFiOLgDMVKLzTYNnEwvN0UdFDjs7JjESmHErtKeZmmXBm31i3cQhY1iJ\\\\\\\/IKMWcaBISTNzdQKeLiQ+8Pat\\\\\\\/nXTZIc7RgnWj8lDag5CZ7ugJ7WFX\\\\\\\/+OUNhW8kCBiwug8xzKMKHa1zAWfPkG1lfvd5wxvo4axhnFLdfEXLGDfhdkdtVV3tLBlv+779L4pczoyFLwVwEsdBW12brJcngxTul97sm8vA7Dli6V99zrcTwZ+Oo1wW+8l10bM4KV+QE5ZCcSopZ7gqaWD4+TQnj259Kvb+M+Lsl9LAvxRmdMzjQim8bwfc6BFwLlcAsg3RRMCF6NHWJDq8f4MB6N6lwTi\\\\\\\/MgeaX2y8AEHej3KtvLtE7e0CQcHFRIFv8U7SVsxbvzwuHLKtfDObXXGNYaNibDodV\\\\\\\/Jze4V5PP+\\\\\\\/Rg0JinGfHnveaGXgamS0EJjwFUqv6K5xHATZKFTkG6XC3EK\\\\\\\/9\\\\\\\/o45YX0Zh7Ksfkq0lLX2KKJqV5L+9woBmqN89hd2lNI2GElPX3Nk6\\\\\\\/CrfbV3kaVUXJUgSQCb2yvO9ebn+SmNoFspjiOvmobzxBkgfUP6wRMjSnCoDdfOSJQe3v1wEDGxP2VqqUsyoT9GUnA\\\\\\\/LFnqWV8qWxaHVl3+YmgFp9GNVK73jgl7qDhRjk1zQ6ds060GL9JpD+WoENFoHg9Ruv\\\\\\\/nzIW+F3a6CKI8AM7ftAcxeaLvwr1v9zEocMcHpcUkXvNDmu3dM\\\\\\\/1nG1QnbVKB83ypU0Mjb6zc11MTKGiGRLt5ODgSCa21Z+Xx7HBbQkmPF+ihBLATWPpwtuIhat9dTTsnqnrAKxd8l22DuO4ml67\\\\\\\/gncqo6DumNEdzqRAW\\\\\\\/CPEo7FDfm2rXZBbNWiZZaCkNt8i3\\\\\\\/pDATF1tBBxywu8f09fXzoo5S\\\\\\\/o+ezgobPoit0V\\\\\\\/jmvPfn83g360bQcBwZ3H+w\\\\\\\/Ul++ip4H+QGkke15EhIDQw0KYa\\\\\\\/zmlDgeXUjt7vzJ7xpBY31Ehu9XiMViIKsGByrviIpT8lQUNBmwbD36rtMWYNOm9xRqVIpeX57O53mzHCZki2Q3VDa6FXcDJRyET1AnLZpDLE2F1\\\\\\\/4RQN9oSarI2sjjGDjmrZAi7wq6lPo1FXP8eTxb24Ru7LwNpv8jNpv\\\\\\\/xLz4BdM03ZkxOCfEjgw9kOVsDwmyWKHkhi\\\\\\\/R7rcPTJ\\\\\\\/zOgvQQxm3nDknhOk+m9GlZStYQARoCA5+S2EJMmTJ4M91anXPlkX07LS7I1GTPF1GdriJdqTYt1zfmtbBzt7xXR4\\\\\\\/GOzsDgH16v2DKnUNFKRFS65RZSNgql4f\\\\\\\/EK32mc1vVOqX0qs1bgQfuA1A+3BFNqZjTFrgXHOefmbRZ66upKhWc\\\\\\\/AqIKBlQI51QTQZ5xAonLXc+chH+varJ5Fp\\\\\\\/JOKKdYFhcuRwUKPE\\\\\\\/N6gWc6lOCPMwKWzzq\\\\\\\/CTiB9yqIFuSETvUyrwwwbxu3q\\\\\\\/uHkenSig2QixI3ds9mGJej2zQrLXl7TPdNy94+AP8QK5Cy\\\\\\\/+vjOOlMQVrL58I5AieASSA49TSS+bO1fb\\\\\\\/qeT581ZfQiqIKCeANHIaqN1aFUEWV+\\\\\\\/qopwBa93GVv7RqcVutvQAG1762RGyXaY+BEmlG67z+A+RBS1EU3bFY3hXBHuCuZJ58nc1cRIjF+Ci9kN5lnJ+DFp3mqGwQfVcOM3kIl+onDI3UAo5bV7WpbkfeeOwjhe2xDQxRrcg5Pu\\\\\\\/81QmVmniWwKjqNewDuWb401HidKKdpHXSGBIYq9+LOoO7k1TRUlvI+yEMYvfcUaAe9wTWecX1v0hXX5oOiuZzS2HlLsYpfvfCYVoV0nUxMXkJxDWb8he1LWjJckfM1ENzJhvbGXnTgHuYLEN8rTgE8iKzciGvHZwXGWidOReXqqAQyK+B88\\\\\\\/13bH7g69F6GA4lPw0sc7y67b6nonDUNsw2U2PwhsAOjvjHTtpch7+tYAUm7DseVAdz3enveuEZK3MpY5RDxeH6cj1qAZKyyUEOQywtA7E6QJgLTWlxJNYVh1jvOjlXt\\\\\\\/qIB1PXORTP6wJJDWk2BeGOUFPtTQfqgdyvCohiG1tbRfUvMEFE2G14EWfnoliy\\\\\\\/gAmHCwWSLm\\\\\\\/gGWwSP50gmHZNMFM1MpDXYskn28WTzFjyUJkXGXvLiuYa32IUbe+\\\\\\\/UcezmTIqgAN1VeuLgJh3kOebugI+mEjKxXZ79Mpq5wIuDIctc6tYo5qCzLfBMq0GOwJodzIB0M+RfilBeNlqucZHatQK1I+yajPKaVYkAy1J47\\\\\\\/aL8V\\\\\\\/Yx6KTkb\\\\\\\/zFbuKfazowGpL2H9t6uUUTbPaYhP3rXEJLt8xb3HCO4BlT8epGlJfedChAMakku8jomUPjRZOcJLweOrbT7s\\\\\\\/EGyo\\\\\\\/tCNEJWqo8lMVeC5OGycHfUsR9d9+sFRy9SPbaaeNms0sryqJTAmdpCa+bLj4ovptBpGjdOwi2P8oiQfw64RDPrV\\\\\\\/2p1wIvY0e73M0rS84QiTGisppNOK6XlqgTCH0a52X8KO9WB\\\\\\\/VTFYA3S4j008O42Hx3SO8nnpPUDdnF4zIRrCpsCNf+cLvLuT6cfDrTIgeY5rOtGGC6VQvqIY3bPZoqMuqjZ2kuTQ9vxQqzlAMS2xe6IZSdO9oGX+5e\\\\\\\/\\\\\\\/kTrRoMvH3Tx4NEyH\\\\\\\/xIeoDRY3rKYyl+Pxy2TC8IR4UrQOPA6y1P8HfOD+nJvwJiKrs5I2PHuHOsIkPyYHGgo\\\\\\\/kETODeN0ymCtJcPqp\\\\\\\/XnLBwfw0b9Dj4YxAX1S\\\\\\\/3NkBC\\\\\\\/ICC6FeQnVBghxDSDBb0d9Y2wYW6qe1vh+IlrJHTha5iZe9LbKPM6fvP7QnVKgWJvBu1H1RuvNrcnMjZ3E9kCgSC+aKHo18R8aBImznQa1\\\\\\\/xVpskLQNTn3kMY0YvMFPYvi5vYNvZmBZWKS1JLYHvuUTdyxOXhY56wjotqXe43VbbGmb8J4Jz6ZdxKWWorNT1AEH39UJvtZqViwusFBAnxpV58se6PXgmgH9RErpvpQ3MT2tK6OpynUM31HG32Z4ExxHSpCGPq6Xs\\\\\\\/NedaKWyJjRU+tOYU3lkqVUVYqaBDbWJyvRLQS+NZlmtPHBlHQt2LqQ6aTgZEiIAynemrBf9YfrcLZyE45Jh5nocn9U\\\\\\\/7O5EMeIwaQt1U7fFvGE1CcaqD\\\\\\\/LBzVlHgYesf5y329IqSq\\\\\\\/u4X5XpNt0H2hd3h4sIDG7ZJ56YisZyCx+Xv3CZxUuyUyqpwGxJbrFtqh8fVL2cNF0sxBIjsrvAvEr0cwFu54mUo\\\\\\\/VKDVFeDJu18y6UihHskPQ6ohe9qnJpoerykF2QogOjZtyDfafGSUJ8a7UfQsY9+VXjrxVurp0RsVhs\\\\\\\/XJiUrbcaXx597AjZ483ts2\\\\\\\/srBtWFoABNCdlXL8FA8BkqDzeviN5ZBfHmciWhTIaTEyU95rmir+QqhfIYN\\\\\\\/7fkw5kOYzVSQr1byZSbvQKw3SNaLH+FmYlrlAt4kfZ3Nty2V9EMtIwXlribENOQb9wBrzGpnL0zVZbDcQRSpp4cmlG7A+LHUMQXBf1zumStZBxANR1RoBqLfuAui9TZLH3ROelEvhRU2PidN267PQKuwkGTj1cmb21rAoERAIKPB\\\\\\\/Rk2N60c9VCZBRqZIbyUimhBzSRZSd4grX93GZw5mVwWUxmBojc3m70NH43jr60VBYsDlgfZcOO4sgtZPktmICOoeHH\\\\\\\/JKZBfCcb7sppOc3Wim+GVDfVpA8fT+FAQDsdCfoeh1N8f8f2x7GnFSzJlUQCqDpjHslWnqHhDK8btDNacxFOqLsTxddBbOwDXlbcdQwiq33deB7Hnwm3A5aQWkpziVi27XZplHS\\\\\\\/YL0ptWj6JDZ1qvCjXe+8Y7fip2g\\\\\\\/ApOo8Mmi2+tQy4DcERgP8ao0BtY2BqADT6hyRJp\\\\\\\/YXZtKjL8OGK5pxP7ivLIGW9ymwWp\\\\\\\/avl\\\\\\\/eq2sXKK6y5VYh0TqkwHQcpkW3anUkwaGStvHe0C7cuhCzKwx53t4qN7RfUUSzw0w0xrNDr9F7ku+nDe7lHskWJ3HbuC\\\\\\\/Bf4Pv3YrQ5zPXC2eHfLvDgx3cGpofBUqBsIEkABPkdNKPy+TwMbfkllpWPBWAcZ2CGf7klkNKS7KvNCt1VHBluFGhTKidY8CDRrulMVnOwjY4geO6BdPU9uXGdBHOg7spjDLn6PotpqfRTwXgujgaRIfCcH8VLKJYgKj2ItLPRzA+XvcQ1x2nsixi+5HqDv01+Af2+PwmYx2+2dAPBKEb2kfPEk1Y32uQB\\\\\\\/5W54i9ApTMgnUe9GmBoGvcso92WgJaaZOawRMy1eukIbMP2XvwX2Y7xYAEpDmX5pZ7CS0\\\\\\\/K0dYQ6h9UQijI3x8gGoYzJPGt+mX2OTHB0wPKLivbCo47wWHWxTXz45QjtGysmBMlE0Ir+t230W5LzPq\\\\\\\/+5JLB4UNFEezSfw\\\\\\\/4i0m\\\\\\\/K1Z1OAIC8ntuKuhk+A0TxkaAQNK93yYHQTSQx\\\\\\\/mQeRLajv9P9IhMhUiqVU+ftq\\\\\\\/lP0d7CxqhO7Oo8MxQhF5En2NAfVTH\\\\\\\/HmNYVl2mOxgPemMv73wdgEJFEzzfnBXTXBulyRMRfOIDPHULVlGhfzvODDX\\\\\\\/Z6BXC9kLTAcxvG6gYBt51PYOCEgva5\\\\\\\/dE8TPV+JpieNOGCtAHnHHH5zhACy4CuMMAXaCm+\\\\\\\/QV\\\\\\\/\\\\\\\/qkWV+RZMk5Xtk5Mps4MRZrQakuAH3Aw+Yw2l0K\\\\\\\/NB\\\\\\\/GVxOgW49iSUEnYH8vbaY0BnW2E46FbnY4061Jv9M5vHCygWVqp8lq+E\\\\\\\/7Ippl7feS3G0\\\\\\\/R6KhWyG2kaQP8P75Po\\\\\\\/tUriR2pFWZ6NXxukVzXdo9KBxqYdSUVwwgtn3T\\\\\\\/jwOpJD79JG3fRR0RoMBCOlrEznB3jZ+eqtDHzsV5bdvddXPGelm8K9UjQSUEchf5wni4bvCqWnhw4k6BsvDHy3o4cWkGKn2kvoyrgHFG+hQYI45Lw3cZShIXRQuipVhO8DZ+T488b7OKHF3lJm\\\\\\\/skUAkpq9TH2UzR4tXQ7uKmbls8fFxVptOvxHqWtWYX9qUmbNj9Z5yIUPDCxkr8MC\\\\\\\/fwEzfeQujRfLo6k9LSJnzBsg3ybRciMzGjR\\\\\\\/EA10Bo1T+a\\\\\\\/\\\\\\\/SXtMyiKBLdFCyFsUhY5iA0lkIy0SNnvjeyq7mOMSjjcgSH33nmHXqbU3HsPTpBL5609Av6L40WO3qk71jzEEi32fxaCKnCDxNROmv+EHNRMsneY0uJrPqFh06Dz\\\\\\\/pQ2BKT1FsqZM1DjEKAzVwRdPm7M\\\\\\\/6OJqcGSeg0Nm8P9hhQRC3rKWS6kRJTUzApELnWS1GYYEPySKvHOF3f7yxgfStYQP5c2mssW0TqTkGKhuANY6njwJqFstFe6TaI20cPAt8jUJ+eisgegReAF0MmE1GjIjJuCoSd1GL36TY7hSAiF6bmZ+FMxOFIDyO+8157gBNn51wtvIF8X4pQXfD5bpUBUq5ulPXxzcivmPUln\\\\\\\/0FHwSkVPCZp\\\\\\\/OLjDX9XrznnRvt\\\\\\\/5S2lD1bLhQSybfq2qbwvMVSAmuNV+4Zgwf6c7Y8YKrzUBCr\\\\\\\/eBJnHpbuNRsrdP+\\\\\\\/cNIh8aN5DQgC6g+rFG3lZECDX6bNQXgQbmZab\\\\\\\/bImug6SxaxdsxBp089B4qQ7xgkCZZIowfNeG1iae2rBAFbkAfJEMzcRauR\\\\\\\/NX+cZ11Nz7\\\\\\\/T4KirY4rzUyJ2M\\\\\\\/NpbOL5B+zHF1tQWLqRPo6B2U4Vzv+ta0Ok7Giyivj0S\\\\\\\/msTHe0lhsXbd7tvJRE8VZfbaBqCA\\\\\\\/wwM3CUl6Ssl4hGh8z77rPp\\\\\\\/Kb7VAwo4QkNWISuJmrg4yei2jU9U4R03bu4pm94tgLFI35N9ZKa\\\\\\\/gvcuDXEfmPMMdpznaiQJcU1ALYx1oDMoe15R\\\\\\\/lWVKEA+R9I49FfsQcWxdozsV1g4nltK0x6CG8ykTOr7NR0z4HUjc0HURWUhAHA+BoDj0P9Xg4S5t9IFBZpyKhTXCTJw7PmBoNhkRHnN\\\\\\\/9FZY20W2ZAQfsg9hukCCIaWX8T282zMv8Gir0p3I7fbzrzTPfcSxcG4k8+xSFJAmBvFSZvhbRuKgqg8xhTKrCueQMNUwjCxQFxiIjcJ5ZI4hpCNAA1by+\\\\\\\/kSvFevOyEgy8q1G2bGzbczYJpFmBcgBl0yyXhSlRsaJpxrneh8BrfLe4ZpDEbpnB4j77xGohWpXS4HqA4EWcptXZwh\\\\\\\/tYc11ShV2mHrDZlt6oHGtnS2snl9POvk+qIWFfQu\\\\\\\/C+B5WWakzcA5ptI33xZRQZDlzCaUSQgMVm+gvvM9SuN7voK0IGht2h75hE6JOvNPQgk24E81EVcbyDYhaKf0coQXto+64s0lOGU1ob9ciwqQciHpnikSF2tv8XFO5gcKosnnhPTu12J45UPOGxvHDxrlEDttJk5OPGTtj5iUKuIDhhgu7AXJr5R6kxVSr8p9977\\\\\\\/putG3TZqMN6pw6btKV\\\\\\\/82LirzFZLKMj0Pt3ja7koUqgsXako4PnMrLv56l+ZHOm\\\\\\\/2v2KF6vV7nYZfSCT3UffYlFwQUraYEzgSCl07DBQbfUsm\\\\\\\/4fd+Ejw+ghVdA+E9WlJtBY9PkMEZmZfqkQZOhfpLBnptgms\\\\\\\/WA4ROBdX\\\\\\\/wmgyC+ZnXkdOStd0qLEhlgp6aygGDIA+NyWe+cGANsUyDRsp115bz3UqONguSn9GIbkWz6naqW7cDSfANwwIzyTbDeacrmXPwIz01U1hyiPjQDdMCl6VzLrzWAm2g1X26+Sf4ZwOK9M7mGN54DtFtXVJmGxHnV2Qdib5bvLsCot7\\\\\\\/ainxdnHWnSaG0RyKYoURWC\\\\\\\/762SZ9Y+Cqx5JEtSgoPY6W7ub0feOrZC4PJCHn4QHgr9lxyQB+sDs7gqF0jgh9TMtlmr3c4ZuVNaDKgaIpfl\\\\\\\/GyopMIxFEyL+5F7Glyib9pfwByMBGsKGoEZK\\\\\\\/ZZwWoLfe\\\\\\\/qn2AXonY7ZLVxC1ZvrCYV8JztTA6xYfO\\\\\\\/1AxHFj7EB8O8RB9Mks\\\\\\\/Jg2er0DRNCWXVIvQ428bpioJWwnmCjz4BoBM9K08FJaBNcIPXKYrVgkiLNBXYNqwbvblcaVo4rdkGN2QWVMD\\\\\\\/ANgwI8VVOhD+aOIlodcIqgIZrVmU15wAPpebNU3p4LGTDWrNf2Bjq7h7okNuUx+ZcU0Kped1DkB51o0BP3PRP9Dd2WAdlXXDy+lwe1\\\\\\\/eJnvsYHhVaLukdkMUqTxP2xbZWwlQvs3cDvDTfNWbiyF7C3mbPQknY+EawVhZPul1qFzpKKHx+GVVjYIK5lwv7oqfUJJSJzmi4dAiY1hLrNGy+TZbQ3zwJ6tedVvPP42mpkKaa0JVQJZF6Hjkethijhx6iJ3fv9fYS\\\\\\\/fPHnxzsQQdQPJ4FOSijs9pp\\\\\\\/JnJiAfOuVsde5Ufe5b2DwnEwnTioQfAX407AKnHh+THvMx\\\\\\\/eip4YsX00yCk6CnHEVeUBry17m+yK\\\\\\\/OltQWseQJJhjfPu9uPbmYlycnH42imJS6wURCRr4V7LyWaKNayGsa7QXmrXAMBx3NhzP\\\\\\\/Q28vQbIe7bh8tql3OvLGy+d37xovRg9k6BZC3GGKHjQWIaiBXQ4Ji1ZQEQEgO8MZJNN7bWgVE+1F2FhQjhlxEXCcYuAyYBw1FoUFdUM0KqSZMQy\\\\\\\/Ylsa6g+rfMX\\\\\\\/IHXEjwj3qd5AfbrHMnQL5A5I67h8zfU0wW0AbcPzGsSWIYBGYYFayI3lAZP5gkxD1Hm6UyqxoeoHfXK3lD4YkEm3T50QV8selwnT2McONuzoNk5RC4dW4AMhobr\\\\\\\/XWactLD5PXovBoAPamd3F\\\\\\\/0aR7BHMjR76ce8e\\\\\\\/yP6R9xhOdFoHAgVWNmY3xS3TMCc2m+ER5a+SN2ZJqdHQObkUdzfVrK8Tuyoim7on3lyDZsf7UfC0QzuxiaBnPC9XPCPwKdTY7eV7MLBq0PdajTbF1O36HOGg4jVPEAM4k73ETDGDK9gK4Uco6ui\\\\\\\/IsNHo21JssSHCA3TDWW3jSnna\\\\\\\/FVM2vwucNI58x5cN9OabRQn\\\\\\\/lo0tyReCMLJ3jUm2i6RCCDzihyGpPYj0n\\\\\\\/qULMEfXYCN6JOpgsndk50ZT7AD\\\\\\\/zjOUtFJt\\\\\\\/pCxceUfOgTTAVtbITgEu6gdWhpVvLbNTKtGWYUxrTnH4bGYzuuz3WChEPiDn+w+1IOYWykQ5Mzt5\\\\\\\/NUKeEkCOItFXAg\\\\\\\/O0VZX33bipMVCJ7+5LLHNtjMEEDi2oQSCOm4\\\\\\\/W8cF7NHDOJoBygqOSd7go4IaS4efLQStk2x\\\\\\\/tX9fqjPakw5QPUk1YtbCjKcGyMekSQ1MTvQDOU74RuWwfY6TTNksWQ9TrNG3QTfWHKbwkNcGbHwtirkaUCcce5LC2qvfQNAwGmtNHsPVGZUfgZXNZVvk3YAjt6GBnb4nSm4jVA0zCWvjE9\\\\\\\/tnGtXo2Blvh+GnO2vMuF\\\\\\\/mBSFXwSgOYWwMDRk3ui\\\\\\\/6ykvf6hcDFlnbNSa9fzApAUDt\\\\\\\/rxiw5jyLmOKtig6tk0MmsoRhVY0HX1t\\\\\\\/Bc2s1hvir0mHjw0Fbh3S\\\\\\\/Wyxg4ARBRZzehT6Y19O\\\\\\\/M7z7FIvM7w\\\\\\\/VxHqPBeuDEGHtXM+SGPEjYcjtgiEhRysbwb7PY8Cqh9ZnTzcy3qTeBe2JttGZJgtD1OjDjSlmDNTOZ1C5gyNwjMvESqAy+4YqI5oE7MbTuAXLebEH71GsOkrxwFmIujdywLEKBjAdqGqwQqYPLqcCvndDecTLTSZbHi7N\\\\\\\/PPFPayaShG\\\\\\\/hj9ISwHB6XP8GViLeNm61jPmDVUPjiyFTvifg69V6tq\\\\\\\/+FnkXbb1JbMdhy7nOlOVN\\\\\\\/AWg9wdhorp4hGVDEJQice5H7L\\\\\\\/qmQ\\\\\\\/okX7G+q4GI7Szt6dOE297ldoHH0FZqTSBEXuIbw4ngvJLXki8tZshbC9\\\\\\\/0VylUUktF7Jg6dByyesGeErBJJgTlsTzhrOh\\\\\\\/flQbfo97RGlc1vqGCfFfaje3qw7VRsG3Qrl7PD2cLlg\\\\\\\/smcYZ\\\\\\\/4XSoHPG6fWYcNfaW62+Ece+vSiU5fEWN2TlJvogCwEPdiRy+Z6o9mkEkWtKFqtCjTVB+Ef7Oj51pewVvcZBu4SbJv9RWt4HlAKSfZvnwbhMZsijmfM06v2bdr\\\\\\\/JCrWNA5jTt+d+aTrdJWsQhQQE8M1x0X2A8d2ASDf6nn0QViZdCDhvil9Dycabtpi1+6mZuzsLUBMFPOS7OG9wQSuREAnldEuPc5afsKMMYcx+M+iVuJ48o6quc6WjThBz66XnKFwWbcEcxWXqkdJ5aQg3fRylUW6Q58N4yqFKlEVIR+fTGuUFF2mSTu5JLUjzLWo8\\\\\\\/C\\\\\\\/G0j8RB7i3z7vIh2+5yPWDWT3dNuB+pHRJMdEJy+WNGtiU58JBlrVsY1YqaTTao56JC9OHfr+LUiEwFilxlfKcblGJOztc0OKMIodqfjKtuXcTS3Ri1t7DUTYcm7Z7JHxpBvcrgUgpTDX6mblom8p03EOKgp+OqgtupcKF0T7USuVoEGr6zh03qiRNCV2aAPVw8gqTh9VRZ1ypBsOHH2t\\\\\\\/4DFlI63S0ow+bn6ZdweMkGjaGdUnbK3ACKORKc5X4tcFzzyO8ZA+oCfk8p+CDAcHaWR2W+2tzmoL72sh\\\\\\\/txm8qfROAEjiUckoj9O5aTlsOwweXMK7\\\\\\\/wM7AvDJEC8mcE6eauBwZK6ox1qY9C9YFgTlCCs2a52RQq8GViSliijFL6LwuQoWh5ZRdtNuqpE1oSpqB\\\\\\\/qt4FUB3r+YWJFcrxsxQS29afgoKa15FbwBC04ZHgBb04VQ5vkT1eY3l5GGwsRKYFTdk6X2CvM1P4poWLBMjGGp7C0vnYp8wsv7v46YR3KJcBf7mgg\\\\\\\/PsnB+FZU4ACzNOWYbu3fKKOWgigC21tk0KNiZPg2a0e3pw9BSqek8AjBE2Xb\\\\\\\/dBGGM99GS+JZN7tB+tBwRW7vDSDE73uGWZRRnJvxSC7CIVZBdTYX\\\\\\\/GkgjyZSVKBOUImt\\\\\\\/hct0qKq2UoJapsq\\\\\\\/TQur9WF2TBVVdUjz4AhT5jhjadwNgTMs6Ol0JuV5OzpyAPaorn5n00JKBrd0wlKCI7FvBhb2n9riBl1XkZRZhbfbytO8mF7Z6oTm2MnzU7l0oMlLRSsnqnmCllYpSgPY2lkpo\\\\\\\/w02xuk9Ch4+ZF6jDMsOsY3LaR5p2Mfu4W6mOB\\\\\\\/nwXxdZzDYKeW43Q3Z4i8BlIg51W2aozJ\\\\\\\/0k7wTR+KpxUkdl19seTiB576n9QezlHZZL41c2GmwG7S7Imzd+Szo3B2ej784lU1OK80Qpe4GzyXVQE\\\\\\\/BvT0s7J9HXNBXtdMsxg+zGh4qtUWeyzbAbx26px7qGNsCpzHsBSMFaOW8yeiNxVPDjK0VD3xR1rQa+PbPik5x4yMBVDDIr7\\\\\\\/0DaykCtgRByawG5NfPuQ3zbeWqDdWV6sUPKxlHv05jX21QBVsGph8Jpm+Ge1zxe3FNSmZx9qorbKr6RcyUnnaBJJAWi+iVizQw0YyGm37T6HjNpWvdZPr\\\\\\\/mjhI\\\\\\\/iu6b+M4SzyNhZ3nsRvREngk3pn4kRAJGQ+P5q3N91QE8X4BErq+CGAWkGnpurHJWnFpjT595W45cS2FPOLdWEMJSN5x19P2Xx98kCpdoShZtuCyOyBsEEMTFVB7pMQenKjoF4RoOxM5Fo1+TDHtKQZcWLEGXSSdr\\\\\\\/zl5sK\\\\\\\/g4oVGdgwyTM+dmpFFb0vieJA2s5h14eLTJ6rbXpZ57lMcxZOCeQVY66Ehkd41JoxeEttidirL9Dub+q4DeeW+Tzsc0qpnHivKkbLuz8bu3t8sTQ9AfXqzxPgaq4MJmCiU+vWOg8uldH9LRqh\\\\\\\/L\\\\\\\/mmNP\\\\\\\/UfEgWL0l9q0Sp3RP6Fkpi54u6hb+N8KjaqtU3Xvf4lE0qR0MKUPqYoToS4htJjSfU76JadlcxqZpFrA9QORSUpI9s232AoOo5Nyr3beMFD6rwRXFZqnfxZPsKXN3oPzvuxXxVGe5tQPmnKCuobFmS1JQH92M7CWxp5wl6rW841lpdwhAUYhMu45sD29VM7JKjGZplCJi7hT5YLlF2CbpgTVVTb483WZbTDTaIF0lqgXDGi1ZELliYICjzRdMCH6UffKuYWnXs+EJAF9rBeK2BoAs4BYeCDEtLcPq5nuZz2QAzjSGl36ZzxTOCZX1Sgn5q\\\\\\\/o4agYS5CiZcu5XKS4IMsGUbq0dcOF+7ATd9\\\\\\\/qSD3QxOi0y5cgvXki38UiwlP2JbkWRNbYXn9jTbFatd99Yl65bDh93y04CwiSWnTsyY1W5tka0h4ss2jttleTt6ZW+oWGVRmzM8LautexJa0xk7aYUSQIfbqLqG7DpUtq4DtN0JargF4\\\\\\\/9JmJnhyyFy\\\\\\\/xWcZfq02xcP6kLK+PDPutOZ60VwCCi\\\\\\\/sHgBgo5itee9SZxopL3Kn5QiaP5Ujt6V6xH7+YzmQqgd3CglwX6FeikcL+PWAZBx7W4QD0vxiL0ssf58AqspCqG7K7HmhUnMT2rNT\\\\\\\/a3cdoc3Ab8pj6GMSAmpO\\\\\\\/4EXnXSFiG4m2iyoV8C7gWqMW7KRDYTT1iNMC3dG6Gl9e2xL8YuY3ahMaUtR3zSRinx08TLdefYRQn6l7VCnxOCJXlvKZnLz\\\\\\\/M2q6CyGyclK56ioG9R7DwphuNVwMIAwME1oGaOgz87hNXkV+A3dArTPWGbrXZkUhyOjrSVZcL2Gmyh8BA5uMEfE7pAEgw3wWE6aYs0huM6N255ae0R6L+eCNYDvUHO8F8GUOgCkGSCherxfK\\\\\\\/W8M07rmgIc1wLXNxqOEhJZ8mwOux9BHoq\\\\\\\/EKwgi1bR2x4HAyI4cOlkLLMH7URCymurhO20i1lXj3mOM+TPTphiEP0SbPPR035aRkU0CiMsEIORRjasH5bdMYkCG+vQ0huF7wQdSmWUI3o+kZjnP1iZidn3Pu1SnAAgSANzS188gtJXYitoabJnPdfVW1KZiYQA2dxkwF5rUraV8AyAZBP54QIx2cy2Y9fdR0GsNFHtFud0RJ+XpdGSJj\\\\\\\/vrgcE7SR4WiYVDyTyIDsoT+GG4nJtaZxARAgCvOaQavBsAChaQeQDVIzudKlctIDogXYPw+qLY2qFNnk2nAiEpxq9XHY2FX418x3V4OBdoGczgBKD2eWVVXpuX+v31hC8zxSYaq8sImjsUti\\\\\\\/JqwBhGVrHUycxgCK1Y6Z4HFKukyS1SJjRPe6F5iOm5i9BttA8AWaNlm3xU+ZpvR8u2NFocwWBby2\\\\\\\/2e6rdRAK3SC6NyqMK6dpVIDK3wc3EpqeqiLmLB5GHBDJrx+0F6cSigg\\\\\\\/utYIP9E2+WC17EeXxrEiGkTStjjyHc5SjJEFyb5+XylyvwswFjU5RfE5r8YkpCU1xx094fimT7edLGmAZ4mf9FE+9WVAsnMaRPkqn7SFk7uCny\\\\\\\/W8cUefLOyNDqPbnKFO+TqB4p81RSEiP4J0kldG\\\\\\\/6HjOaFJInS8Igyv1wh8eFPtUzX5QU5gnf9epPXZGTJmeaFo7FA5Xbklf8eopSGymGIdHs7t3do0Al8kkWw9xB3xLsDhjRpVwVgMXowad4kwrVyJthMZplfmw9uFIp0qjmR5GqtdQ+qHzdKV07xS+ZNHfO0DjOVlkm6V+zc6janKB4YSzXc4hF58z1U8BXOqqDr1bJhu0+ejNdpZvqD5t0WoTt10b7iJlNi\\\\\\\/AqqRmJaCiUjkW\\\\\\\/OhAGafGMLi1\\\\\\\/oDsLaL28s6pl0jkLuVo9dUpTQCDx9vw6Pu+b3VVPR\\\\\\\/1By+STSG0mvRh50U678bbJHLD9gMG6nOUCpHfkBEJQ2KGoNCGIs+FZ9U4NGkBDlgDpDLDRGGLXwIXnGA5ExG3zWFZi3M2J8ovp7i39XN1n5TAEjAXTIDiwvvim\\\\\\\/Wn7p4ndRXsN53Nw7TYXEGU5h2bgmQ\\\\\\\/xYlCWzG8uMtVWL85ZwI7WfBVetent\\\\\\\/2T8S2uuFBzpzcIyUc4wV5tVCmejxn\\\\\\\/6f17JsCwFXCOLuojEK3zHF347MgFskInUrFf\\\\\\\/s6TFWlKztBAsPhMBOMm1zyFltLy\\\\\\\/0CQgWkHJz4QkAx\\\\\\\/tIz2SSFqUcd+YdYNcTOPY7EF9ouvZBtfZMi1RJHCDovyOmFZCP+jHg4mttvtk8htHzgGPnkzr5x8n9obvKL66\\\\\\\/NihqjSMBamFoEyoZxocTMhlp9UUiPN5FwbU4HzHSX1TcfAy6cesSylRlsj22a4mMJTYP7hkY60nigH8fEAqL2q1nB\\\\\\\/u0ilAHmuniVOfC5zPYp7aIlq9wl0jGRf8OddnzuHJrkVsDvNjRFmPYr1GL8LKidTxefoGKMqDBEjtlsSA28eoEtI03o8pT504YUfoQiUVlQa0w3DDwnkBLian1a1pJlQEsQLaJUb8pdRPrABLiB4WoiS\\\\\\\/LtiiNF5RtbDvjXs1kSo0B25fCMpRiZ4JNZOgV5TQYuGT19U6MzbwwATEJPW\\\\\\\/s+pfDrAhu5qSKEZ+mgnPyp5Pp9udHmuzPpUQdCGlms17HwI7XCV5xZxrMvLzlC7PsFvY+ROPJXJPcKJAlffDKaUAlac+LiR1rJc7gH2FNzSvSyyXqcwDrlQUMpuwUE5d3\\\\\\\/S8TYnd1OV0gv50jqZ8xVu38H4kiWXy25tuJlUE+Vb7Po3GgECNckpY8B5EWF9jjrhP9gN2TBFfFEqXPXkdNsMFVIh7inIxEEckHhP8RAjHA6nzWuhlehQ7BdVJce07uALWP+DMa+AFTIZs+GRReqGcaizjc7TM+rODaMHB9jZepkwY3\\\\\\\/icdePHxFJ2ieNHvXZc2FEgu1DgZpksJQPkx94vX9PgRRQF22hiy7O0WlWc+u+ZzmWN\\\\\\\/n1iH1rvC3uMCRf53a+Y97xj5ntpyaci\\\\\\\/dYNIoms8j9gtsqA8uCLwltqhIdyl\\\\\\\/aoBxj65NR+V1Y6kkSuzz+A9QWon6X+6xBidSvwL6GSPOcF9Habo9W0fSIOhsWICsnxa0EG2UDJoRqxlhyKdwB0DZHW+eiH7Y+Ph0ImOyQx34tyiTvnr+KTxUecYf20jdtRFA65Hq2KOcJMN5qWGXbJYtxQzEO7c3EWZkFnpwfKX0VuKCIo7YzSulj6Rg+HgMAZAvupi2sr4jY90eSchp\\\\\\\/97HIUCzpNv5+K+3uNuzpiJMkzcZlR+djoghbBoApNsnJZgl+0RhJqA2SyVVppRBIMDuS7x9P83itmiN2xwpy13hNFVuHQ5CyxKHliQYGB41Tf8457EiKFdkrUU\\\\\\\/zcTBmJYxqIht0f2HzoslYoQiRV8k6F3GxG\\\\\\\/J\\\\\\\/IqQU+SLA7h4QhDhojQHTufJNVfMTymHzL2s1dtlyt6btkZ80\\\\\\\/1GpRoRxhFNuOa4d7qBRmDnJPBaBO7viDmgb8+nPbAGK5r1XfRHplG+7InvZ\\\\\\\/NICioI79ooWFU+qUJSqAmuC9s+C9pp3AakWLi3WZVceSlErytgCNjnePgwl3JVA\\\\\\\/VaSEmExBtNL4OslWx7QXxT58KdoBmYGxBFNUuf1RAQ0xIGMCnZbT+l\\\\\\\/rJShP1fUpyLQvl\\\\\\\/DnwSMa3dZ7vY8N6SWrSnDfiwiDgcZft9Tyyy5OC85gPRPYqKZjdtlnhEcosyTAKJBOZAzTg64UICWTVTvkTb8OHb62BSAMYa5fPrQekXuBNT2goPNklHfHRRHACaOyeEEYIkis6i7D8WOG7FJx77IbAwZVsBmWtk7plWNcb3jKv8xUusJuq44DEEdxOb1l7igHO9RI789pNtdD5UTuXytKUz1HfnuXatTHtGBbl3pBhlHW6utU\\\\\\\/lDb8ypK\\\\\\\/Xr7gFPPnVupig2GvpjaHcMhmg97oVh9\\\\\\\/Oj6i3vBTOw3jnqNp8TaysH7cI2gp83ZEq6SpwM+MGrHlkE3SpITx5o0fagIYy17BFuMtYG94WfR\\\\\\\/sZQFk+KXewBTUT4B\\\\\\\/1Gj6V4je\\\\\\\/rLfRdfecCj7WnlXoRP3OotJXnKNSHpbxdPc8vqyg67kCzSbuPk5hUDh6jm7Vx5IBzG6daCJJWjDglJsBozreZTFKeLVmu6oNQwQzYjI48EsmtKWZ2huKO3Vqlr9gfp+DO0BtzZomPFTHgC06A3XSqn1wtPKzKxX\\\\\\\/I2APEnwVgHKGEA9tekS2XxxQaI7op9NlDSbQ93bc3C27wNeEI09Ld1Zv08oAiPjHszTD0nrHH3AERQDMZay4ZSvgd9ECd9QMGzG1YPV05XC2MhJf89TaZwnW0ekuWCD9vO8onsF6WCL\\\\\\\/\\\\\\\/Acu6njONpFBt+6OWFxlZ+Tntg7At8GA159J+ox4Ba7CdvBZRHAJMC1gykcvTs36KefxEhzO1UM9oq5Fk+fDVtsxfInRKYq8JPK6BmO+TgochxddnWVDDBcSqfe6olMcR2tNtmdxjVJiQsJlx3qhfw6VMZaMHrTynjqLUt0f0xfreyGCyz4otCIZplTNuotQme2kiQFT720h3eLeRValGjBKSUQWvhZzdQDkr4qOldrf90hTQh0qBSewlEZCYeJQ14hdOi5IBxc3iJaP2CISBGvmftBu3h7lADJ9H7\\\\\\\/9lKiT7aZDlbmS4fGDlUVWIVZmKIMPD0smFqnIAGXYlhglXFwJpOeHXT4r+rE9EJZFqbYOtXjyDcVANUFbOfiOIfHW\\\\\\\/hytfDCMSWxlP5jKEBLnKaq4OW6JgPZVWHjgFrhw4CYlQcz+8levDRJHqXG\\\\\\\/MYhil0fFTPnbHWsAiJoIpm1LW51ExnA\\\\\\\/3ccn38drmfDalKuFln0pa0WkAIAbHcfYkiPb7nIrB82fBueu+WpXLKuG01rWYR2GHHqC6MiYm7XRLXkQyilbKHjbmCtVqlQzGKq1DbZIv5ITG75qvg5l6ls7bC6uW2FOSCFWBWb6708E1C6qGhxte5Xo2FrNyA37mIHMF3XAyoYZDB7LU5wv4GFFfK1VSn4NbsojporjblWm1CRkZ4xZeK8dyV23TZdaKtifBZFXcj8oJzzcrZIp6x6DoflD3UWClT0QprcRXxbHushZoxyZ8yEL6QozoNL80AwCXGcN\\\\\\\/l6bzdvFsqiQDRwEIcjvZEk\\\\\\\/0qFoAHUoZKLwG4TbKLyxoQck6OI2hQAnpMw1ajuIyy56gmLD5LyavPb8CakBpJtPPCOuDbPwKZEYXu9OdIitzkUeYUWc3C5sZGvlpsXgMgiBpb4AsCjDK+pA\\\\\\\/TveIfJTukrUT61YdFjwV1yQsEjBe93qGAAgDZhfGg6fJUGCRVpW\\\\\\\/JMFvhLNk8xDMhQsm3IWeiX6StxifQOKEo1pMPd4fk7bIgiiabCW2VN6+Xdh\\\\\\\/AWgb04KY2jdtA5xt\\\\\\\/E\\\\\\\/BYihhCmSfHcoDE7GupZixFxHwilxvI4bTh2uFiXucJ3b4twTSemUx0gp3ePD5Gwf8TTGQK3S38y7Czbn\\\\\\\/jooSvDuCsu2sLLoLjM0d\\\\\\\/pzZiPyfHm+l7WhbNSTflOD\\\\\\\/HBTe4CsOSiiZkB0MUYv5bq5S3yLTixmOMFNfVWpuPvX2q4vewEr8N5HCAFee99z4k7NsGkF7NY9X2xGfAIKEt0iizkNvh\\\\\\\/KGA\\\\\\\/\\\\\\\/xrUfQY7lpIQawERflJLWv\\\\\\\/lVRnhuee+WaUTDuY\\\\\\\/gTuKY9FjqnT1iUj\\\\\\\/cryzHZE44rdOLCthjJYeAy3s9foc3pRx6InTte7HbGd4kdrrpnZtS19ho261cr89df3k8TRi0mggIo5UCRH+CpBVZg+\\\\\\\/3uUcNevTum\\\\\\\/1Vv+pZOX4mhnBYsNKjupyL902H9RI\\\\\\\/E5GFNnVzLkCs9uVUp2y2ACofd35j\\\\\\\/tEtA\\\\\\\/fRux194Mv4\\\\\\\/aDYuJDLXFleuBPZZvS9\\\\\\\/nfk1smvFnfKjSMUTOqS9yTZFivbFX+Wki8+hUG6+9cuPPMvWw1EP7qxzAdyvsORVEZx+9tpNlz7pGMgEA2mHcQM2UErVhSJFiK6ZeQ59Dg+bbBxs7Uileb3Ne2vXGxuL3RtHuONM99ymmBAKfmPeclzfKnn3xWPqR999s37zgWOfALkSOXIPdHmYwaoTWLfBn0OYGF\\\\\\\/vSrho9y6Vi09GyoNe\\\\\\\/4CPhIxrGCfk6utbmR+G2MxqyF72gfPULLkDy+GI5po2c\\\\\\\/r7Ov2SVT7QjwNxt5pIRWYwTAbDCshe4M3BHFlGeIJMX369Xrit5g3XGV0G3xI1U\\\\\\\/NIwz+lcepecWqVMNiAUL4gYWEgVl2EXX\\\\\\\/KYEDxx32vM7ZpYA+SmjXGyQDcn0TOqupzu46Vd3AokZVzzhtDb+0NECqfNJZFAZAjp3IdweVc046Z\\\\\\\/ya6Yr8rKqa5st1z\\\\\\\/OfE\\\\\\\/O4i+9BmqrRinyaGgDyB9k7yXn9XiIW9Ohv4F55cG25+5U9E33yZOLNZ8odmT41\\\\\\\/NRm8TM8Kdy9dZO1g2SeAGxPkuKpk\\\\\\\/jdm+lbCvA8yF1KI9K0gL\\\\\\\/qdmJLdRBMJBfPJMXIbj74cprNTdKzIoSMRwfYxMIh35EIRenEPumaeX9FBLxKeSeoAESri+tuWsGUayEuOr4vAW\\\\\\\/ktPJcnUypapOKFfwtQWar1q+4Boe+U6wVKEFg+FcWtuIaDz87KW5YEUHvNsPRNBtgGYLm7ZysoHrxnlW\\\\\\\/N+9ByHcZbl1VrQFzyL7p+BOAgE6sQrKQ72WZcUewMHo7k+iDP\\\\\\\/e45YyAW+4lkKj23beUscBItzQoS8y8oWHKVW04haqSVl++X2iQ42yAPD+tKAMZOgnTIMDlCBGTzAvO4BXqKvEfvEZh0EDJXpypyNS39VQMKjVqrFOv4egtgi1vrrNfCowszsYDSt5OLmNu2h5GnCP1Q\\\\\\\/i1Fpq3EAiz\\\\\\\/xuMCplBSAkmMVDvnf6EJRoCTayalfAWxvSv2SBhJLatfqFiXSbfH7\\\\\\\/MCX2aYO5O+aD4p1FAGz73UlHsGEr0VvFc0E1WVlRQsf9Ht8fFJV+kBS5ICMk1mSg68ROKd0f2Id5BCk8IrYv\\\\\\\/gPEqxhVLhjSyBtPgUZEOtIADE6njsP1jx83W3Z2zTqUFfZXd5BEF0Hvx4\\\\\\\/GWOW\\\\\\\/G8gYF2CgBIsVCHVWp\\\\\\\/N4Yn0RpynPBCZDm6anIissQA0DEY8lwwT0GTBq8hmN4Bu7xEhCzWpruXxOqhdHfD1e4zTm\\\\\\\/f4Y+4gUxISw3XflmoMYzhaOCIx+pDboEk1ZJRTH5UaWS5lI72m5uSn01ZpGWGeR6yYbC7s8xNgQ84zFHyNdob+ECYO40PrpgV63BOm9F1LHZK5nNirN4ObJLpcGXSp6uVQHBoP88s6wAW77VtpH8T4zmUhSaxBVRO3vFqgfze10qwNlSrVdkLUGTQ1tcDovNWTXyduwGyid8sPUCvji1PZ7+m10OYuii0nYcq2K03Bo\\\\\\\/2LAE8jPatlsCJwVPk7y3rKeR2xezyY3J6dMSjZXbg4T1zomVJeBFVCm6Mt2RuX585weDPzUBneCBqnrD9CQtCUsYOqKCAKxKWaMuFS9yl1j9huOGJvJTeKxXspsbJhaYd6nKFFyi\\\\\\\/rXIuUz+fkic3HgDLqc2j8RB9C0I2J7qV\\\\\\\/5sxEYsT83xFouW6W6zU\\\\\\\/+64q+oB+5+Ja9PNJdbSUXSBQaH12uIAyAn5yxlZtFuvlrsLiXImUEkngCgN2XBBg+292xrfpK3T2ZBe\\\\\\\/naENkyOPv4ga8eYE4pdQMs7\\\\\\\/j4CJFiyPExZ9n2Si6sRvo4tGRvLddBEG9tbvpG4wEfCQdeX1a+omkJd4qxXUGwiVsNYKqTpvEIqu\\\\\\\/Fhcn75LgMeyB17ftbIwI8d0ooSB6lqRuVbNIhR4YjM8FZ7SDV0\\\\\\\/Pt\\\\\\\/wPZi\\\\\\\/7FOu9oV4PyoDp3Pkw3fSqByoNKhihtfS3oZQlV4Li4eGQiX\\\\\\\/xRQqoUklOAX9n7mCtI0xW9T900tb3g41fM1rhEggkvqQNP4rb6bhB0jTswdm6JVohH3G34k8X\\\\\\\/0deGRam0Dfzi0IEG7ttKwB117YEMyIwiIf3E3EmzOrxc7IgS9Arg7t+0pLo1yzItIsK1lRC4+7qHo4rr8Qo32qn1fQp19PykbEazXub51ejuljdYFMMbAOy5603Iu\\\\\\\/PciN4OL\\\\\\\/R0i0sl0oqlL\\\\\\\/qGVatG33zYdY3j40zg6uQvnGUMNPq3rHUsZJc+Xk5gZ\\\\\\\/1qNbAQBE+kgthb4oTdSIjB1JQXDWzmznAhuHsLzUhQ8BMygXSr2q\\\\\\\/toK\\\\\\\/vBa7K\\\\\\\/aYt5c06GZola9uGyleS2+VUrJhiJ3qwOlzZkbXTI1I7GT7pDCt5tPjJ3NPit8OzT0rKfqSHkBAg4FPIFH7tQw9FJChVNd1he1TIEvBQ1JRlsagpS7h0H0Ot\\\\\\\/tMl4QsmgZK70ORFDpsxULkzQzO0nz8X2wXsECsXDxIofcOZP6mAW5Xve0UCR\\\\\\\/hXGp19a1tOlAH1ZZKyNkfjv0e3rrhjgpoHewuMVEbsK77tgkearkq4E8JUdgbRFa0hpbWcLA3V9Fuh8pHF6s7RQtzPK0X7Y9ckzZRkXVUhVdSTLLM8X+B\\\\\\\/9P0Ji36w3CL56cdCNRMxivCjEQBCkuXMpf1lRKrFHdEdQhgpUSLwzQio2AAN7BMDC0iDk777YCxcEbRQob3AOL3HzgsGVLLkLFK0ZOVrxLu4TrT+NSX5hnS8RGe2JivH7JT28C8qOsvnW0MphUT6V+do86c5j79svfQ+wIJXrqW6PyNOXS8TVJWwWzgIduGVp6vtUE5N8rKDepFy3hAa0CL7XwnHbxJ6u5+SQmhp\\\\\\\/VyC0XgifHatMETPpPTNeerJX4yw+bD72Ia9Dr6BI2b4H0g+BBmEgDG7YjL1JbR48p+2AcjPcwzrLoET6QKlgufO3Mjw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022X9zLqaitRW5PBkafVYFYFQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f814b3babc64ef06fa506204781698dc3fcce7e93da73382899dac7610d5d245\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F0Ziy0EHhvfKiznUVVR5qx9bH\\\\\\\/b2DtTjJYGOMG7MkUxhCM8J6MYp4HUTmHRhc6iOcp9o07cUpe4ZChBz8G+UuDcK+EHgEdXgGd\\\\\\\/eX6adP2XKyXaIIGuXSo8GzKjKxeJS0FOlxK8MwGf\\\\\\\/EqDGSbn0rCAcnSJKVlmDJQorhuSJCrJnCt9RMJnzBDpkK8jc6Q0HGyX3d1pckjvbiteWpjHmbvZg5sR8wJkhpzFLnNfm51Xkhxs8jrQsO7XC7IuV3VigyAGKp8FlSqqMxRAtweVmv3AmH1TqgOihEfmFolEMuHTR7yho6azUqZ+JdEo+curQDR+sGGCLlU7R8uwTDOOm\\\\\\\/vBIMSI1a1H9cbREgrjBSHLl41KQxGg1UqkRNN0Y7CiARm\\\\\\\/BeVdGLKOLNybdrX50BDpACt5MQczNHWRg5zCnm0DVTBMv5g+\\\\\\\/xHgAJ9pKGzcXngAQ9Hp\\\\\\\/Xn59OkgqFNtwCR3pgW8IxyA30GwR45pV8hXm2jY\\\\\\\/5eVX4Lye0UEKWR04euHauXirASto8ZIEIMlmKXTJBfpFzlGtyZfkN4uKY0l06y3kXNmC7JsT7r+KH3krvgax0\\\\\\\/iiTRKCeXoShDE0oYIOqS\\\\\\\/VtCjlV7CZ0kpoFYY1BsZFtzWUJ2Z9KIEaWBl+JWBMRt1dBVMH7Vsv5833O15A4plM7Q8Zn7Moz92MnDpALKFZPNyvIwJmFnsWp5VedBN0kjMvDQyILdXcdL9stOq12plhFidsAKwwp6++91ZaoHZ6IWHEdUfNrF7ToJPNwgHaux8dSgnzdFk2j7QLi25cazQt2Q4DqlRXmDw78OChb4Ek885oj6OY6vziURWcpCPHUheA+5LVMkiZNnthso5iwPv0Qjddn8LZea41HGbK3I5wKaOnzSgXRXgsiql9dQwGXjMs8OUODP2bMXHiXUILOWeh9Wzl9Lt+d612XkcRU\\\\\\\/4tGQUFQvxLMpy7VRy0oW+5exxQ\\\\\\\/FXtrOR3TAmy\\\\\\\/QFdbqaoZebF8WGFBmIH5rlbNZ9M8CDV8+8NNT5C4GsKAuSHtJb2gk\\\\\\\/w095zAbg9lBYH7TJVV0rf2DEbociDE7+eBTx\\\\\\\/jEFhtVsjCUgyTAAnDdjOTkgYm0p8enOMDmw85gSrcGPW4rakfk7A2vVzTf7E+3G6ix5Px2bhQmAG6Bn5pf1cnh99Ln7zqgFdFBnRzBLExhpc2NM45lxt4RsRt4FUDq+zuZ\\\\\\\/bI8Ed9wh5M7+zWZllbCtfXaG9u+31+IEDAgHUxkrAb+KDOqe9cHtW+r5VPpTbrbAa9TUxsrwPwApFZMB1o2wZ6XimHQ4UDCEdNYPskQuFxKkUYYWH4DRv0LqPzY4qLZ48OYF9qvR13MdxMVeZqb+2TInWaGk4jdM1OU\\\\\\\/C1XKsqLKIBNVvCpcKvwgMJzQE+19wdIrG36PRbetDJN3vATluOG6smZRbFb51k6RJoPBvHJ0H5eCqtKRIitWFmBNYEzb2f2O3JXj0loKyRKfQjESpkFWzCiDiHDyoUTsFdO1SclAUI74Iz6KEOmbWFPF3BC6JwyKRHAAmf24Tj1ID98EX+Zul9hWzVE83HFk8nsVTiJrXogz3wF8ukRzzf8xqstkVtjK0FQ1UQJsQ1x0+e0JT4f4O09y0M6+s2jyHtfmTIaHBTtRg1OE2wUkSzg1Un6Yj927FXjOjIJbi5cuzfcfsNwGS3MpV05WsUJfbSyVZ3Q+cTtja5FCBfXWOA6iMbe2Ufl8csv8Kx3lWVH8quv67dIz4V+60aZAAF2GIeyV7DYIQEG1D1szOFJMPtgXuowIGAvSJZ8qcAHhEM4Vm8TPX3+W93UpzEBV3sMXRcyvh+yXqzy32MrccWNd55gMI022Km02x4JqNRnuB2f\\\\\\\/egOp3R9PENx3MhnQHHDv+Swr+wejLjXCaj0IbD4ymqYh+u30nf0NmplKHLRXlhYdJIcqJBCCcwD9wzJArePc+kjyHxC2nWiZ7ffyAzltJbR0gDuXaa0jSSVnRayqIkzWjs\\\\\\\/4bMBhzvokxBK7mNQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/NKonEL3+Y291jzXh8XRlg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228c354fac44e48ddc6914b94a57b096df9963d8cdc6f1bdf5aa2cdb410ad03219\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459424 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/517" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pRcJO0yyFvdFy0kABn-MkwooVhqtFfIW-G5iInVeRgA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/517" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r3witKkejOXmlVI7l41Mzvi_66TnQSXF5m3f4b-mfGg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/505" title="A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/527" title="Calcineurin Governs Thermotolerance and Virulence of Cryptococcus gattii" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459424" id="node459424" data-pisa="ggg;3/3/517" data-pisa-master="ggg;3/3/517" data-apath="/ggg/3/3/517.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin T. Page</span>, <span class="highwire-citation-author" data-delta="1">Alan R. Gingle</span> and <span class="highwire-citation-author" data-delta="2">Joshua A. Udall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">517-525;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005298 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459424" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin T. Page </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BT.%2BPage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Page%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BT.%2BPage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alan R. Gingle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia, 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BR.%2BGingle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gingle%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BR.%2BGingle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaudall@byu.edu" class="" data-icon-position="" data-hide-link-title="0">jaudall@byu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/3/517" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459424" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/517.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459424" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/517.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459424" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/3/517.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459424" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/3/517.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459424" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Read mapping is a fundamental part of next-generation genomic research but is complicated by genome duplication in many plants. Categorizing DNA sequence reads into their respective genomes enables current methods to analyze polyploid genomes as if they were diploid. We present PolyCat—a pipeline for mapping and categorizing all types of next-generation sequence data produced from allopolyploid organisms. PolyCat uses GSNAP’s single-nucleotide polymorphism (SNP)-tolerant mapping to minimize the mapping efficiency bias caused by SNPs between genomes. PolyCat then uses SNPs between genomes to categorize reads according to their respective genomes. Bisulfite-treated reads have a significant reduction in nucleotide complexity because nucleotide conversion events are confounded with transition substitutions. PolyCat includes special provisions to properly handle bisulfite-treated data. We demonstrate the functionality of PolyCat on allotetraploid cotton, <em>Gossypium hirsutum</em>, and create a functional SNP index for efficiently mapping sequence reads to the D-genome sequence of <em>G. raimondii</em>. PolyCat is appropriate for all allopolyploids and all types of next-generation genome analysis, including differential expression (RNA sequencing), differential methylation (bisulfite sequencing), differential DNA-protein binding (chromatin immunoprecipitation sequencing), and population diversity.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/read-mapping" class="hw-term hw-article-keyword hw-article-keyword-read-mapping" rel="nofollow">read mapping</a></li><li class="kwd"><a href="/keyword/polyploid" class="hw-term hw-article-keyword hw-article-keyword-polyploid" rel="nofollow">polyploid</a></li><li class="kwd"><a href="/keyword/cotton" class="hw-term hw-article-keyword hw-article-keyword-cotton" rel="nofollow">cotton</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/homoeo-snp" class="hw-term hw-article-keyword hw-article-keyword-homoeo-snp" rel="nofollow">homoeo-SNP</a></li></ul><p id="p-4">Read-mapping is a fundamental part of next-generation genomic research. Read-mapping was the essential first-step in pioneering studies of gene expression (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mortazavi <em>et al.</em> 2008</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> 2009</a>), quantification of genome methylation (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lister <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2009</a>), estimation of DNA−protein interactions (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Park 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wilbanks and Facciotti 2010</a>), and assessment of population diversity (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sabeti <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Durbin <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chia <em>et al.</em> 2012</a>). Researchers have largely applied these methodologies to diploid genomes of model organisms, including <em>Arabidopsis thaliana</em> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zhang <em>et al.</em> 2006</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Vaughn <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cokus <em>et al.</em> 2008</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lister <em>et al.</em> 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kaufmann <em>et al.</em> 2010</a>), <em>Drosophila melanogaster</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Graveley <em>et al.</em> 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McManus <em>et al.</em> 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nègre <em>et al.</em> 2011</a>), and <em>Homo sapiens</em> (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Mortazavi <em>et al.</em> 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Valouev <em>et al.</em> 2008</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lister <em>et al.</em> 2009</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Trapnell <em>et al.</em> 2010</a>).</p><p id="p-5">Read-mapping will also be used to analyze the polyploid genomes of many important plants. It has been recently established that all seed plants are paleopolyploids, with all angiosperms sharing an additional event (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jiao <em>et al.</em> 2011</a>). Thus, all flowering plants have undergone at least two paleopolyploid events in its history. Although all flowering plants have a history of whole-genome duplication (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stebbins 1950</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams and Wendel 2005</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Paterson <em>et al.</em> 2005</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cui 2006</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wood <em>et al.</em> 2009</a>; 2011), ancient duplications do not significantly complicate read-mapping because duplicated loci diverge over time, permitting confident placement of a large majority of sequencing reads. On the other hand, more recent whole-genome duplications challenge read mapping by causing a twofold increase in chromosome number and DNA sequence while preserving gene order, coding and noncoding sequence, and chromosomal elements such as centromeres and telomeres. The increasing capacity of DNA sequencing will allow future studies to address the evolutionary and molecular hypothesis of recent polyploidization events (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Osborn <em>et al.</em> 2003</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adams and Wendel 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de Peer <em>et al.</em> 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Flagel and Wendel 2009</a>) and the effects of polyploidization on plant phenotypes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gaeta and Pires 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Soltis <em>et al.</em> 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schranz 2000</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dubcovsky and Dvorak 2007</a>). Accurate assignment of sequencing reads to their genomes-of-origin will be essential to elucidate the underlying principles and consequences of polyploid evolution.</p><p id="p-6">Because most read-mapping software has been written for the analysis of diploid genomes (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Griffith <em>et al.</em> 2010</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wu and Nacu 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Garber <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Langmead and Salzberg 2012</a>), they are unsuited for mapping sequencing reads from polyploid samples for two reasons. First, mapping reads from a polyploid to a related diploid genome results in differential mapping efficiencies because one coresident genome matches the reference better than the other. Differential mapping efficiency biases subsequent comparisons of the two genomes and skews quantitative analyses. Second, existing tools cannot distinguish between the two genomes to assign quantitative results to one or the other. Other phenomena, such as copy number variation, cause different problems for interpreting read mapping results and are not the focus of this effort (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kitzman <em>et al.</em> 2012</a>).</p><p id="p-7">The problems related to analysis of polyploid data can be mitigated by <em>a priori</em> single-nucleotide polymorphism (SNP) identification within and between extant diploid relatives. Most of these SNPs are vertically inherited from diploid ancestors to allopolyploid derivatives, so they are present both between diploid relatives and between coresident homeologous genomes of the allopolyploid. These “homoeo-SNPs” can be used to reduce mapping efficiency bias through the use of SNP-tolerant mapping, as with heterozygous genes in humans (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Wu and Nacu 2010</a>). After mapping, the genome of origin for individual reads can be identified based on a comparison between the bases at the homoeo-SNP locus and the respective bases of related diploid species—a process we call read categorization.</p><p id="p-8">Bisulfite-treated data present additional challenges to read mapping and read categorization because transition SNPs cannot be distinguished from bisulfite (BS) conversion events. Because transition SNPs comprise a majority of all SNPs, including homoeo-SNPs, treatment with BS causes a majority of homoeo-SNPs to be potentially uninformative for categorizing BS sequencing (BS-seq) reads.</p><p id="p-9">Here we present PolyCat: a pipeline for mapping and categorizing sequencing reads from allopolyploid genomes. PolyCat was developed and tested on data derived from various species of cotton (genus <em>Gossypium</em>). The most common form of domesticated cotton (<em>Gossypium hirsutum</em>) is an allopolyploid composed of homeologous A<sub>T</sub>- and D<sub>T</sub>-genomes, where the ‘T’ subscript indicates genomes within the tetraploid nucleus (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wendel and Cronn 2003</a>). Two extant diploid cotton species have genomes closely related to those contained in the polyploid nucleus, namely the A<sub>2</sub>-genome of <em>G. aboreum</em> and the D<sub>5</sub>-genome of <em>Gossypium raimondii</em>. The A<sub>2</sub>-genome is more closely related to the A<sub>T</sub>-genome than the D<sub>5</sub> genome to the D<sub>T</sub>-genome (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Senchina 2003</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Flagel <em>et al.</em> 2012</a>); however, the diploid D<sub>5</sub>-genome recently was sequenced because of its smaller size (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Paterson <em>et al.</em> 2012</a>). This characterized trio of genomes was used to develop and evaluate the read mapping and read categorization of PolyCat.</p><p id="p-10">The PolyCat source code and the current cotton SNP-index is publically available for other studies (<a href="http://cottonevolution.info">http://cottonevolution.info</a>), along with a web portal in which evaluation sequence data sets may be submitted for mapping and categorizing. PolyCat produces genome-specific BAM files as output, which may be immediately used by most current bioinformatics tools for downstream analyses, such as differential expression (RNA-sequencing [RNA-seq]), differential methylation (BS-seq), differential DNA-protein binding (chromatin immunoprecipitation sequencing), and population diversity.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sequence preprocessing and SNP index generation from diploid-derived data</h3><p id="p-11">Sickle (<a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a>) was used to trim all sequence reads with a quality cutoff of 20. We used the Genomic Short-read Nucleotide Alignment Program (GSNAP) (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Wu and Nacu 2010</a>) to map 1,140,550,335 reads from <em>G. raimondii</em> (D<sub>5</sub>), and 4,070,680,434 reads from <em>G. arboreum</em> (A<sub>2</sub>) to the <em>G. raimondii</em> reference genome (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Paterson <em>et al.</em> 2012</a>), accepting only unique best hits and allowing for novel splice sites (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). SAMtools (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2009</a>) was used to generate two pileups, one for A<sub>2</sub> and one for D<sub>5</sub>. We compared the resulting pileups with each other and with the D<sub>5</sub> reference at each nucleotide position to identify homoeo-SNPs between the genomes, as well as allelic SNPs within the A<sub>2</sub> and D<sub>5</sub> genomes with at least 4× coverage and a minor allele frequency of 40%. Sequences used in this effort are available through the National Center for Biotechnology Information Sequence Read Archive (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462570/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462570/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Contribution of different DNA and RNA sources to construction of a SNP index</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>RNA-seq read categorization</h3><p id="p-13">We illustrate read categorization with RNA-seq reads from cotton petals in two allopolyploid cotton species: <em>G. hirsutum</em> (cv. Maxxa Acala and referred to as Maxxa) and <em>Gossypium tomentosum</em>, an endemic polyploid cotton species of Hawaii. Because the cotton A<sub>T</sub> and D<sub>T</sub> genomes are more similar to their extant diploid relatives than they are to each other (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Flagel <em>et al.</em> 2012</a>), SNPs between diploids approximated SNPs between their respective allopolyploid genomes and were considered putative homoeo-SNPs. These SNPs were used to categorize reads from <em>G. hirsutum</em> and <em>G. tomentosum</em> as originating from either the A<sub>T</sub> or D<sub>T</sub> genomes (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Udall 2006a</a>,<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">b</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Yang <em>et al.</em> 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Flagel <em>et al.</em> 2012</a>). After mapping to the D<sub>5</sub>-genome reference as described previously, PolyCat was used to compare the nucleotide at each SNP position to the SNP index and categorized it as A<sub>T</sub>-genome or D<sub>T</sub>-genome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), depending on its unique match in the SNP index. PolyCat then assigned each read to a category based on the number of A<sub>T</sub>-genome and D<sub>T</sub>-genome matches. Reads with at least 75% (a user-specified parameter) of matches for one genome were categorized as A<sub>T</sub> or D<sub>T</sub>, accordingly. Reads with matches to both were categorized as chimeric (X). Reads without SNP positions or matches were categorized as unknown (N).</p></div><div id="sec-4" class="subsection"><h3>Bisulfite sequencing</h3><p id="p-14">Bisulfite treatment deaminates unmethylated cytosines to uracils. During subsequent polymerase chain reaction, the uracil is interpreted as a thymine for complementary strand synthesis. After sequencing, cytosine-to-thymine mismatches (C→T) between the read and the reference sequence indicate unmethylated cytosines on the sequenced ‘+’ strand. Guanine to arginine mismatches (G→A) indicate unmethylated cytosines on the sequenced ‘−’ strand. This conversion looks like a transition SNP and requires tracking by PolyCat to avoid data loss.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/517/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A diagram of the PolyCat read categorization process. (A) Reads from diploids are used to generate an index of homoeo-SNPs. (B) Reads from tetraploids are assigned to a genome based on the sequenced base at each overlapped SNP position. (C) Categorized reads from tetraploids can then be realigned into genome-specific assemblies and used to improve the SNP-index." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1604448285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A diagram of the PolyCat read categorization process. (A) Reads from diploids are used to generate an index of homoeo-SNPs. (B) Reads from tetraploids are assigned to a genome based on the sequenced base at each overlapped SNP position. (C) Categorized reads from tetraploids can then be realigned into genome-specific assemblies and used to improve the SNP-index.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/517/F1.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/517/F1.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/517/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/517/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462619" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-15" class="first-child">A diagram of the PolyCat read categorization process. (A) Reads from diploids are used to generate an index of homoeo-SNPs. (B) Reads from tetraploids are assigned to a genome based on the sequenced base at each overlapped SNP position. (C) Categorized reads from tetraploids can then be realigned into genome-specific assemblies and used to improve the SNP-index.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">For BS-treated data, PolyCat first inferred the origin strand of each read by counting C→T and G→A conversions. More C→T conversions indicated ‘+’ strand, whereas more G→A conversions indicated ‘−’ strand. Ambiguous strands were counted as half reads for both strands. For ‘+’ strand reads, PolyCat accepted a thymine as a match for a cytosine allele; for ‘−’ strand reads, PolyCat accepted an adenosine as a match for a guanine. Knowing the strand origin allowed PolyCat to maximize information from each SNP. Because transition SNPs comprised the majority of the SNP index (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), most reads would be uncategorizable if transition SNPs were made uninformative. However, C-T SNPs were uninformative only on the ‘+’ strand, and G-A SNPs only on the ‘−’ strand (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). So PolyCat could use C-T SNPs to categorize ‘−’ strand reads and G-A SNPs to categorize ‘+’ strand reads to minimize data loss.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462486/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462486/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Composition of SNP index by SNP type</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/517/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homoeo-SNPs in BS treatment. (A) Suppose there is a C-T SNP on the &#x2018;+&#x2019; strand between the A and D genome (green characters). After BS treatment, reads &#x2018;descending&#x2019; from the &#x2018;+&#x2019; strand may have a C or a T, depending on the methylation state. All reads from the &#x2018;&#x2212;&#x2019; strand will have a &#x2018;C&#x2019; at that SNP position, regardless of methylation state. And in this case, all reads from the D genome will have a T, regardless of the strand. Thus, a T base at the SNP is uninformative because it could be from the D genome or an unmethylated A genome. However, if it were known that the T nucleotide was descended from the &#x2018;&#x2212;&#x2019; strand, then the T would be fully informative (i.e., it would indicate the read was unambiguously from the D-genome in this example). As mentioned in Materials and Methods, we impute the original read strandedness based of the frequency C&#x2192;T and G&#x2192;A transitions. (B) Suppose there is a G&#x2192;T SNP; there is no ambiguity, then, about the genome origin of the original strand because A-genome reads will have a G or an A, whereas D-genome reads will have a T." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1604448285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Homoeo-SNPs in BS treatment. (A) Suppose there is a C-T SNP on the &#x2018;+&#x2019; strand between the A and D genome (green characters). After BS treatment, reads &#x2018;descending&#x2019; from the &#x2018;+&#x2019; strand may have a C or a T, depending on the methylation state. All reads from the &#x2018;&#x2212;&#x2019; strand will have a &#x2018;C&#x2019; at that SNP position, regardless of methylation state. And in this case, all reads from the D genome will have a T, regardless of the strand. Thus, a T base at the SNP is uninformative because it could be from the D genome or an unmethylated A genome. However, if it were known that the T nucleotide was descended from the &#x2018;&#x2212;&#x2019; strand, then the T would be fully informative (&lt;em&gt;i.e.&lt;/em&gt;, it would indicate the read was unambiguously from the D-genome in this example). As mentioned in &lt;em&gt;Materials and Methods&lt;/em&gt;, we impute the original read strandedness based of the frequency C&#x2192;T and G&#x2192;A transitions. (B) Suppose there is a G&#x2192;T SNP; there is no ambiguity, then, about the genome origin of the original strand because A-genome reads will have a G or an A, whereas D-genome reads will have a T.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/517/F2.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/517/F2.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/517/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/517/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462487" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-17" class="first-child">Homoeo-SNPs in BS treatment. (A) Suppose there is a C-T SNP on the ‘+’ strand between the A and D genome (green characters). After BS treatment, reads ‘descending’ from the ‘+’ strand may have a C or a T, depending on the methylation state. All reads from the ‘−’ strand will have a ‘C’ at that SNP position, regardless of methylation state. And in this case, all reads from the D genome will have a T, regardless of the strand. Thus, a T base at the SNP is uninformative because it could be from the D genome or an unmethylated A genome. However, if it were known that the T nucleotide was descended from the ‘−’ strand, then the T would be fully informative (<em>i.e.</em>, it would indicate the read was unambiguously from the D-genome in this example). As mentioned in <em>Materials and Methods</em>, we impute the original read strandedness based of the frequency C→T and G→A transitions. (B) Suppose there is a G→T SNP; there is no ambiguity, then, about the genome origin of the original strand because A-genome reads will have a G or an A, whereas D-genome reads will have a T.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">After categorizing each read, PolyCat reported the number of cytosines and thymines, or guanines and adenosines, at each cytosine or guanine reference position, along with the methylation context—CG, CHG, or CHH—according to the D<sub>5</sub>-genome reference (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lister and Ecker 2009</a>). Separate columns reported the total number of cytosines and thymines, as well as the counts for each genome (A<sub>T</sub>, D<sub>T</sub>, X, or N).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Homeologous SNP index</h3><p id="p-19">A SNP index largely composed of differences between homeologous loci was created by comparing the alignments of reads from A- and D-genome diploid species (A<sub>2</sub> and D<sub>5</sub>, respectively) to the D<sub>5</sub>-genome reference. We refer to these single-nucleotide differences between homeologous loci as homoeo-SNPs. Our SNP index consisted of 2,633,689 SNPs (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Of these, 1,543,513 (~58.6%) were transitions (tr) and 1,055,479 were transversions (tv), a ratio of approximately 1.5 (34,697 SNPs had multiple allele possibilities in one of the two genomes and could not be classified). The gene-dense Maize HapMap 1 had a tr/tv ratio of approximately 1.0 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gore <em>et al.</em> 2009</a>), and the more uniform Maize HapMap 2 has a tr/tv ratio of approximately 2.0 (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chia <em>et al.</em> 2012</a>), demonstrating a greater abundance of transition SNPs in intergenic regions in which natural selection does not prevent spontaneous cytosine to thymine mutations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Coulondre <em>et al.</em> 1978</a>). These values, together with the cotton SNP-index tr/tv ratio of 1.5, suggest a correlation between the genic skew of a SNP collection and the tr/tv ratio (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005298/-/DC1/005298SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005298/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-20">SNPs were distributed evenly across the genome, reflecting the gene density of the <em>G. raimondii</em> genome. The average SNP density across all chromosomes was approximately 3.51 SNPs/kbp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005298/-/DC1/TableS2.pdf">Table S2</a>). Chromosomes 6, 7, and 9 had slightly more than 4 SNPs/kbp, whereas Chromosomes 5, 10, and 13 had slightly less than 3 SNPs/kbp. The remaining chromosomes had between 3 and 4 SNPs/kbp.</p><p id="p-21">A total of 1,123,129 SNPs were in annotated genes, including 579,259 in exonic sequence (9.4 SNPs/kbp). This increased SNP density in genes was likely due to increased sequence conservation between the A- and D-genomes. (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cronn <em>et al.</em> 2002</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Senchina 2003</a>). The number of SNPs varied greatly between genes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). A binomial distribution of genes with 9.4 SNPs/kbp and 1.6 kbp of average length predicted 0 genes with no coding homoeo-SNPs, but 4161 genes actually had no coding homoeo-SNPs. These data suggest strong purifying selection on these genes, possibly due to their connectedness (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Birchler <em>et al.</em> 2005</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Freeling and Thomas 2006</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/517/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histogram of SNP frequencies by gene as annotated in the initial draft of the D-genome reference sequence. Most genes (mode) had between 20 and 30 SNPs. A total of 7235 genes with low coverage (RNA-seq or WGS) from the diploid datasets were removed from the distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1604448285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Histogram of SNP frequencies by gene as annotated in the initial draft of the D-genome reference sequence. Most genes (mode) had between 20 and 30 SNPs. A total of 7235 genes with low coverage (RNA-seq or WGS) from the diploid datasets were removed from the distribution.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/517/F3.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/517/F3.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/517/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/517/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462520" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-22" class="first-child">Histogram of SNP frequencies by gene as annotated in the initial draft of the D-genome reference sequence. Most genes (mode) had between 20 and 30 SNPs. A total of 7235 genes with low coverage (RNA-seq or WGS) from the diploid datasets were removed from the distribution.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>SNP-tolerant mapping efficiency</h3><p id="p-23">SNPs between diploid relatives can approximate homoeo-SNPs between coresident genomes of an allopolyploid (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bancroft <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Harper <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lai <em>et al.</em> 2012</a>). SNP-tolerant mapping uses these SNPs to improve mapping efficiencies of sequence reads from allopolyploid genomes, but previous efforts (<em>e.g.</em>, <em>Brassica napus</em> and <em>Tuber aestivum</em>) have not used SNP-tolerant mapping. To demonstrate the effectiveness of SNP-tolerant mapping, GSNAP (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Wu and Nacu 2010</a>) was used to map sequence reads from A<sub>2</sub>, D<sub>5</sub>, Maxxa, and a synthetic F<sub>1</sub> hybrid to the D<sub>5</sub> reference. The mappings were performed with and without SNP-tolerant mapping. For comparison, Bowtie also was used to map the WGS reads from A<sub>2</sub> and D<sub>5</sub> to the D<sub>5</sub> reference (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Langmead <em>et al.</em> 2009</a>).</p><p id="p-24">The SNP-tolerant mapping substantially improved the mapping efficiency of reads from A<sub>2</sub> or allopolyploid cotton to the D<sub>5</sub>-genome reference (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The mapping efficiency of D<sub>5</sub> reads to the D-genome reference was unchanged. GSNAP mapped more A<sub>2</sub> reads than Bowtie, and a substantial increase of mapping efficiency was observed with SNP-tolerant mapping enabled. Of that increase, approximately 50% was observed whereas mapping A<sub>2</sub> BS-treated reads because of the reduced sequence complexity typical of BS treatment (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lister and Ecker 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Laird 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Krueger <em>et al.</em> 2012</a>). The overall mapping efficiency also improved for the allopolyploid reads ince allopolyploid reads included both A<sub>T</sub>-genome and D<sub>T</sub>-genome reads. The improved efficiency of allopolyploid cotton reads was a result of accurate mapping of A-genome reads to the diploid D-genome reference.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/517/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping efficiency with and without SNP-tolerant mapping. Reads were mapped by Bowtie (WGS only), GSNAP, and GSNAP with SNP-tolerant mapping (GSNAP-snp). WGS reads from G. arboreum (A2), G. raimondii (D5), were mapped to the reference genome of G. raimondii. Subsequently, RNA-seq and BS-seq reads from A2, D5, G. hirsutum (Mx) and the F1 diploid hybrid (F1) also mapped using SNP-tolerant mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1604448285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping efficiency with and without SNP-tolerant mapping. Reads were mapped by Bowtie (WGS only), GSNAP, and GSNAP with SNP-tolerant mapping (GSNAP-snp). WGS reads from &lt;em&gt;G. arboreum&lt;/em&gt; (A2), &lt;em&gt;G. raimondii&lt;/em&gt; (D5), were mapped to the reference genome of &lt;em&gt;G. raimondii&lt;/em&gt;. Subsequently, RNA-seq and BS-seq reads from A2, D5, &lt;em&gt;G. hirsutum&lt;/em&gt; (Mx) and the F1 diploid hybrid (F1) also mapped using SNP-tolerant mapping.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/517/F4.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/517/F4.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/517/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/517/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462617" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-25" class="first-child">Mapping efficiency with and without SNP-tolerant mapping. Reads were mapped by Bowtie (WGS only), GSNAP, and GSNAP with SNP-tolerant mapping (GSNAP-snp). WGS reads from <em>G. arboreum</em> (A2), <em>G. raimondii</em> (D5), were mapped to the reference genome of <em>G. raimondii</em>. Subsequently, RNA-seq and BS-seq reads from A2, D5, <em>G. hirsutum</em> (Mx) and the F1 diploid hybrid (F1) also mapped using SNP-tolerant mapping.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Read categorization of sequence reads</h3><p id="p-26">After mapping, PolyCat categorized each read based on matches to the SNP index (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). To test accuracy, reads from diploids were also categorized. Most reads were assigned to their correct genome (0.3% of D<sub>5</sub> reads categorized as A<sub>T</sub> and 0.8% of A2 reads categorized as D<sub>T</sub>). Erroneous categorization occurred most frequently in BS-treated reads (2.1%). A small number of reads from diploids (&lt;1%) were categorized as chimeric, indicating nucleotide matches at separate loci (within a read) to both the A- and D-genomes. Chimeric reads were slightly more common in A<sub>2</sub> than D<sub>5</sub>. The low level of erroneous or chimeric categorization shows that PolyCat successfully categorized the vast majority of sequence reads.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/517/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentages of read categorization. Reads were mapped to the G. raimondii reference with GSNAP and SNP-tolerant mapping, then categorized as A-genome (A), D-genome (D), chimeric (X), or unknown (N)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1604448285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentages of read categorization. Reads were mapped to the &lt;em&gt;G. raimondii&lt;/em&gt; reference with GSNAP and SNP-tolerant mapping, then categorized as A-genome (A), D-genome (D), chimeric (X), or unknown (N).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/517/F5.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/517/F5.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/517/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/517/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462536" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-27" class="first-child">Percentages of read categorization. Reads were mapped to the <em>G. raimondii</em> reference with GSNAP and SNP-tolerant mapping, then categorized as A-genome (A), D-genome (D), chimeric (X), or unknown (N).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">For allopolyploid reads, erroneous categorization was impossible to definitively identify, but the rate of chimeric categorization was low, albeit greater than in reads from diploids (4.4% in RNA-seq and 3.8% in BS-treated reads). Two factors may explain the increase in chimeric categorization in reads from allopolyploids: (1) The SNP index was based on A<sub>2</sub> and D<sub>5</sub>, so it includes false homoeo-SNPs that are really allelic SNPs specific to A<sub>2</sub> or D<sub>5</sub>. (2) After polyploidization, gene (or intergene) conversion events between the allopolyploid genomes could have replaced the nucleotides of one genome with the nucleotides of the other. At homoeo-SNP positions, conversion events can be detected in reads from an allopolyploid (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Salmon <em>et al.</em> 2009</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Flagel <em>et al.</em> 2012</a>), and the rate of nonreciprocal homeologous exchange had been extrapolated to be approximately 2% between the two genomes (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Salmon <em>et al.</em> 2009</a>). A greater rate of nonreciprocal homeologous exchange (6.8%) was recently detected in a global assembly of expressed sequence tags from <em>G. hirsutum</em> and <em>G. barbadense</em> (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Flagel <em>et al.</em> 2012</a>). If homeologous exchanges did not overlap a homoeo-SNP position or if they were larger than individual read (or expressed sequence tags), then they were not detected. Thus, these numbers likely underestimate the true number of historical exchanges between the two genomes.</p><p id="p-29">Approximately one-half of the polyploid reads could not be categorized because they did not overlap a homoeo-SNP. The uncategorized fraction of reads varied by length and by quality of reads. In the reference genome, only 163 Mb of 749 Mbp were within 100 bp (the length of Illumina HiSeq reads in our dataset) of a homoeo-SNP, resulting in a 21.78% theoretical probability of any whole genome shotgun read being categorized. Genic regions (120 Mbp) had a greater density of putative homoeo-SNPs than intergenic regions because of our large collection of diploid RNA-seq data. In these regions, the theoretical probability of categorization was higher (60.7%) than the remainder of the genome (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). These data illustrate the dependency of polyploid reads categorization on SNP density.</p><p id="p-30">The BS-treated reads had a decreased level of uncategorized reads because of the information loss caused by BS conversion. Each transition homoeo-SNP was only informative for half of the reads (C-T SNPs for ‘+’ strand reads and A-G SNPs for ‘−’ strand reads). Although the same portion of the genome (120 Mbp) could have been theoretically be categorized after BS treatment, the combination of transitions confounded with BS treatment and of uneven distribution of homoeo-SNP density (<em>e.g.</em>, &lt; single homoeo-SNP/read) caused fewer reads to be categorized in some regions than would have been otherwise categorized had only one of the individual causes been a factor.</p></div><div id="sec-9" class="subsection"><h3>Allele-SNPs within individual allopolyploid genomes</h3><p id="p-31">After read categorization, SAMtools (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2009</a>) was used to call allele-SNPs within each genome-specific assembly (A<sub>T</sub> and D<sub>T</sub>). These allele-SNPs represented loci that were heterozygous within the subgenomes of <em>G. hirsutum</em> and <em>G. tomentosum</em> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). <em>G. tomentosum</em> had slightly more allele-SNPs, representing slightly more genes, than <em>G. hirsutum</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005298/-/DC1/TableS3.pdf">Table S3</a>). The D<sub>T</sub>-genomes had more allele-SNPs, representing more genes, than their coresident A<sub>T</sub>-genomes. Approximately 75% of allele-SNPs were novel (<em>i.e.</em>, not indexed). A small number of indexed homoeo-SNPs also appeared as allele-SNPs within the genome-specific assemblies. These SNPs may reflect homeologous gene conversion events, or they may be false homoeo-SNPs.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/517/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNPs in G. hirsutum and G. tomentosum compared with the SNP index. Numbers inside blue circles represent the total number of SNPs for that genome, whereas underlined numbers between blue circles represent SNPs that are shared between two different allopolyploid genomes. This is not a formal Venn diagram because the numbers between blue circles are represented twice&#x2014;once inside the circle(s) and once between the circles. They simply indicate the number of shared SNPs between the blue circles. Numbers inside the large red circle are indexed, while those outside were not contained within the SNP index." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1604448285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNPs in &lt;em&gt;G. hirsutum&lt;/em&gt; and &lt;em&gt;G. tomentosum&lt;/em&gt; compared with the SNP index. Numbers inside blue circles represent the total number of SNPs for that genome, whereas underlined numbers between blue circles represent SNPs that are shared between two different allopolyploid genomes. This is not a formal Venn diagram because the numbers between blue circles are represented twice&#x2014;once inside the circle(s) and once between the circles. They simply indicate the number of &lt;em&gt;shared&lt;/em&gt; SNPs between the blue circles. Numbers inside the large red circle are indexed, while those outside were not contained within the SNP index.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/517/F6.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/517/F6.medium.gif" width="345" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/517/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/517/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462580" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-32" class="first-child">SNPs in <em>G. hirsutum</em> and <em>G. tomentosum</em> compared with the SNP index. Numbers inside blue circles represent the total number of SNPs for that genome, whereas underlined numbers between blue circles represent SNPs that are shared between two different allopolyploid genomes. This is not a formal Venn diagram because the numbers between blue circles are represented twice—once inside the circle(s) and once between the circles. They simply indicate the number of <em>shared</em> SNPs between the blue circles. Numbers inside the large red circle are indexed, while those outside were not contained within the SNP index.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">By comparing the A<sub>T</sub> and D<sub>T</sub> alignments, we found that only a small number of novel homoeo-SNPs were identified in genic regions (77 in <em>G. hirsutum</em> and 59 in <em>G. tomentosum</em>), which suggests that most existing homoeo-SNPs between the A<sub>T</sub>- and D<sub>T</sub>- genomes were identified using the diploid genomes as surrogates. Therefore, increased sequencing of tetraploid transcriptomes will only minimally augment the number of “new” homoeo-SNPs; however, it would likely decrease the number of false-positive homoeo-SNPs resulting from diploid specific nucleotides.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><div id="sec-11" class="subsection"><h3>The phylogenetic context of SNPs</h3><p id="p-34">Read mapping in polyploid genomes is a natural application of DNA sequencing, although the practical challenges of mapping to the duplicated loci of polyploid genomes have not received much attention. These challenges include (1) mapping duplicated reads to a single reference genome, (2) the difference in similarity between the subgenomes of an allopolyploid and the diploid reference sequence, (3) gene conversion, (4) allopolyploid autapomorphies, and (5) diploid autapomorphies. Carefully classified SNPs can be used to address some of these challenges, despite the lack of a read-mapping program capable of mapping to a duplicated reference genome. For evolutionary and plant improvement studies, reads are best classified within a phylogenetic context using SNP positions and their corresponding nucleotides.</p><p id="p-35">In the simplest case involving allopolyploid formation, the genomes of Parent 1 (P<sub>1</sub>) and Parent 2 (P<sub>2</sub>) are combined into a common nucleus and form an F<sub>1</sub> (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Assuming that such a sexually reproducing hybrid could be created, little nucleotide substitution will have occurred between the parental genomes and their counterparts within the polyploid F<sub>1</sub> hybrid. Thus, SNPs between the diploid parents accurately predict homoeo-SNPs between the subgenomes of the F<sub>1</sub>, allowing for improvements in polyploid F<sub>1</sub> read mapping efficiency and read categorization. For example, a sterile cotton diploid F<sub>1</sub> hybrid (a nascent allopolyploid) was created by a recent hybridization. Categorization of reads from F<sub>1</sub> had fewer chimeric (X) reads than reads from the natural allopolyploids (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/3/517/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phylogenetic contexts of SNPs within a polyploid genome. (A) Immediate formation of an F1 is largely additive in terms of DNA content. SNPs between the contributing diploid genomes can be readily detected in the newly formed hybrid (red SNP marks) and F1 reads can be readily categorized as originating from the P1 or P2 genome. (B) For most allopolyploids, a significant amount of time has passed since the initial genome duplication (represented by dashed arrows where time is on the y-axis). Nucleotide substitutions since polyploid formation (autapomorphies) resulted in allele-SNPs (green SNP marks). (C) SNPs can be placed within a classical phylogenetic context. Red and green SNP marks represent their respective SNP-types. Additional homoeo-SNPs (red, dashed SNP marks) were identified by comparing alignments of categorized reads (e.g., AT-genome reference alignment to DT-genome reference alignment). (D) The blue circle represents identified SNPs (allele- and homoeo-SNPs) that are useful for improving mapping efficiencies of allopolyploid samples. Potential false-positive homoeo-SNPs (i.e., diploid allele-SNPs) that are autapomorphic for each diploid do not negatively impact read mapping if one of the diploid alleles is common to one of the allopolyploid genomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1604448285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The phylogenetic contexts of SNPs within a polyploid genome. (A) Immediate formation of an F&lt;sub&gt;1&lt;/sub&gt; is largely additive in terms of DNA content. SNPs between the contributing diploid genomes can be readily detected in the newly formed hybrid (red SNP marks) and F&lt;sub&gt;1&lt;/sub&gt; reads can be readily categorized as originating from the P&lt;sub&gt;1&lt;/sub&gt; or P&lt;sub&gt;2&lt;/sub&gt; genome. (B) For most allopolyploids, a significant amount of time has passed since the initial genome duplication (represented by dashed arrows where time is on the y-axis). Nucleotide substitutions since polyploid formation (autapomorphies) resulted in allele-SNPs (green SNP marks). (C) SNPs can be placed within a classical phylogenetic context. Red and green SNP marks represent their respective SNP-types. Additional homoeo-SNPs (red, dashed SNP marks) were identified by comparing alignments of categorized reads (&lt;em&gt;e.g.&lt;/em&gt;, A&lt;sub&gt;T&lt;/sub&gt;-genome reference alignment to D&lt;sub&gt;T&lt;/sub&gt;-genome reference alignment). (D) The blue circle represents identified SNPs (allele- and homoeo-SNPs) that are useful for improving mapping efficiencies of allopolyploid samples. Potential false-positive homoeo-SNPs (&lt;em&gt;i.e.&lt;/em&gt;, diploid allele-SNPs) that are autapomorphic for each diploid do not negatively impact read mapping if one of the diploid alleles is common to one of the allopolyploid genomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/3/517/F7.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/3/3/517/F7.medium.gif" width="328" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/3/517/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/3/517/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462584" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-36" class="first-child">The phylogenetic contexts of SNPs within a polyploid genome. (A) Immediate formation of an F<sub>1</sub> is largely additive in terms of DNA content. SNPs between the contributing diploid genomes can be readily detected in the newly formed hybrid (red SNP marks) and F<sub>1</sub> reads can be readily categorized as originating from the P<sub>1</sub> or P<sub>2</sub> genome. (B) For most allopolyploids, a significant amount of time has passed since the initial genome duplication (represented by dashed arrows where time is on the y-axis). Nucleotide substitutions since polyploid formation (autapomorphies) resulted in allele-SNPs (green SNP marks). (C) SNPs can be placed within a classical phylogenetic context. Red and green SNP marks represent their respective SNP-types. Additional homoeo-SNPs (red, dashed SNP marks) were identified by comparing alignments of categorized reads (<em>e.g.</em>, A<sub>T</sub>-genome reference alignment to D<sub>T</sub>-genome reference alignment). (D) The blue circle represents identified SNPs (allele- and homoeo-SNPs) that are useful for improving mapping efficiencies of allopolyploid samples. Potential false-positive homoeo-SNPs (<em>i.e.</em>, diploid allele-SNPs) that are autapomorphic for each diploid do not negatively impact read mapping if one of the diploid alleles is common to one of the allopolyploid genomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">This simple model of polyploidization lacks the passage of time since polyploid formation, during which additional nucleotide substitutions will have accumulated (autapomorphies in the diploid and polyploid genomes; <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). The nucleotide substitutions within each genome after polyploid formation are called allele-SNPs because (1) they occurred independently in various allopolyploid individuals (<em>e.g.</em>, accessions) and (2) they originated in only one genome and in only one of two germline chromosomes. After a single base substitution, drift, selection, or both will move the allele frequency of the derived base toward fixation or elimination. Thus, allele-SNPs can be found within individual genomes where a particular accession is heterozygous or by the comparison of two different homozygous accessions. These allele-SNPs would independently assort during meiosis after nucleotide substitution, regardless if they were identified in homozygous or heterozygous individuals. SNP identification efforts in other species have used confusing, alternative notation (<em>e.g.</em>, hemi-SNP, <em>etc</em>.) if the allele-SNPs were initially identified in a heterozygote as opposed to a homozygote (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bancroft <em>et al.</em> 2011</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Harper <em>et al.</em> 2012</a>). We do not use that context-dependent terminology in cotton.</p><p id="p-38">Allele-SNPs can be identified by remapping categorized reads to the reference sequence and searching the alignments using common SNP-finding tools developed for diploid genomes (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">McKenna <em>et al.</em> 2010</a>). As an example, by using SAMtools we identified more than 1000 new allele-SNPs within both allopolyploid genomes of <em>G. hirsutum</em> and <em>G. tomentosum</em> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). These allele-SNPs would be the most useful type of SNPs for cotton improvement because they have been bioinformatically discriminated from homoeo-SNPs and because they could be expected to segregate in Mendelian fashion (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Van Deynze <em>et al.</em> 2009</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yu <em>et al.</em> 2012</a>).</p><p id="p-39">Comparison of independent alignments of categorized reads identified a limited number of new homoeo-SNPs because the extant diploid relatives used for initial homoeo-SNP identification were not perfect surrogates for the actual ancestral genomes that formed the ancestral allopolyploid, AND because of autapomorphic substitutions since polyploid formation (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Resequencing multiple diploid accessions from each genome could identify the true diploid autopomorphies and reduce the number of SNPs erroneously classified as homoeo-SNPs. With our current dataset, these two SNP types were indistinguishable in our SNP index. Fortunately, the rate of false-positive homoeo-SNP (or false-positive allele-SNPs) had a negligible impact on read mapping because neither allele was penalized as a mismatch during SNP-tolerant read mapping. Thus, PolyCat used a conservative approach where if any SNP were included in the index (regardless of its source) its respective bases would be essentially masked during mapping.</p><p id="p-40">Finally, SNPs can be placed on a traditional phylogenetic tree, but only a portion of those SNPs (homoeo-SNPs and allele-SNPs within the allopolyploid) impact mapping of sequence reads from allopolyploids (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>). Allele-SNPs identified in subsequent re-sequencing of additional allopolyploid accessions can be easily added to the SNP index. Thus, improvement and extension of the PolyCat’s SNP index will be an iterative process (although SNP discovery will likely reach a saturation point and plateau). The combination of both types of SNPs (homoeo- and allelic) was included in the cotton SNP index for read mapping, and a similar collection of SNPs could be compiled for other allopolyploid genomes such as <em>Brassica napus</em> (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bancroft <em>et al.</em> 2011</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Harper <em>et al.</em> 2012</a>) and <em>Triticum aestivum</em> (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lai <em>et al.</em> 2012</a>).</p></div><div id="sec-12" class="subsection"><h3>Effectiveness of the PolyCat pipeline</h3><p id="p-41">The SNP index and read categorization process facilitated the analysis of allopolyploid cotton by reducing the bias in mapping efficiency between the two genomes and by providing a means to separate data generated for each allopolyploid genome (A<sub>T</sub>- and D<sub>T</sub>-genomes in cotton). Mapping all sequence reads to a single genome reference allowed for an aligned, comparative analysis between the two genomes within a given accession, as well as for more accurate analyses between accessions. Although these tools have been developed for cotton, they can be readily applied to any allopolyploid by providing an appropriate genome reference FASTA file, SNP index, and sequencing reads.</p><p id="p-42">PolyCat is ultimately limited by the density of homoeo-SNPs across the genome. Reads belonging to a particular region of the genome can only be categorized if it has one or more homoeo-SNPs because every categorized read must overlap at least one SNP. The use of longer reads could improve the rate of categorization.</p><p id="p-43">PolyCat is written in C++ and Perl, using BamTools (<a href="https://github.com/pezmaster31/bamtools">https://github.com/pezmaster31/bamtools</a>) and Bioperl (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stajich <em>et al.</em> 2002</a>). The custom scripts, the cotton SNP index, and a demo web application for demonstration of allopolyploid cotton read categorization are available online (<a href="http://bioinfo3.pgml.uga.edu/polyCat/upload.html">http://bioinfo3.pgml.uga.edu/polyCat/upload.html</a>). In the online version, 1 GB of sequence reads (non BS-seq) in FASTQ format can be categorized by PolyCat in approximately 15 min. Additional sequencing and development of software algorithms and tools will provide continued insights into polyploid genomes, their interactions, and their resultant phenotypes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Michael Gore and Jonathan F. Wendel for valuable comments and discussion. We also thank all primary contributors of public cotton sequence data, particularly Jeremy Schmutz, Andrew H. Paterson (D-genome WGS and reference sequences), and Allen Van Deynze (A-genome WGS sequence). We thank the Brigham Young University Mentored Environment Grants, the National Science Foundation Plant Genome Program, and Cotton Inc. for generous support.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005298/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005298/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-12-07"><span class="received-label">Received </span>December 7, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-01-10"><span class="accepted-label">Accepted </span>January 10, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Page <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.1" data-doi="10.1016/j.pbi.2005.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Polyploidy and genome evolution in plants</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DAdams%26rft.auinit1%253DK.%2BL.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D135%26rft.epage%253D141%26rft.atitle%253DPolyploidy%2Band%2Bgenome%2Bevolution%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2005.01.001%26rft_id%253Dinfo%253Apmid%252F15752992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2005.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15752992&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227932600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.2" data-doi="10.1038/nbt.1926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bancroft</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBancroft%26rft.auinit1%253DI.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D762%26rft.epage%253D766%26rft.atitle%253DDissecting%2Bthe%2Bgenome%2Bof%2Bthe%2Bpolyploid%2Bcrop%2Boilseed%2Brape%2Bby%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1926%26rft_id%253Dinfo%253Apmid%252F21804563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21804563&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.3" data-doi="10.1016/j.tig.2005.02.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riddle</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auger</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Dosage balance in gene regulation: biological implications</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DBirchler%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D219%26rft.epage%253D226%26rft.atitle%253DDosage%2Bbalance%2Bin%2Bgene%2Bregulation%253A%2Bbiological%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2005.02.010%26rft_id%253Dinfo%253Apmid%252F15797617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2005.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15797617&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228286300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.4" data-doi="10.1007/s00122-011-1780-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harker</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and mapping of SNP assays in allotetraploid cotton</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DByers%26rft.auinit1%253DR.%2BL.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1214%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSNP%2Bassays%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1780-8%26rft_id%253Dinfo%253Apmid%252F22252442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1780-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252442&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.5" data-doi="10.1038/ng.2313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maize HapMap2 identifies extant variation from a genome in flux</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChia%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D803%26rft.epage%253D807%26rft.atitle%253DMaize%2BHapMap2%2Bidentifies%2Bextant%2Bvariation%2Bfrom%2Ba%2Bgenome%2Bin%2Bflux.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2313%26rft_id%253Dinfo%253Apmid%252F22660545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660545&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.6" data-doi="10.1038/nature06745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cokus</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merriman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Shotgun bisulphite sequencing of the Arabidopsis genome reveals DNA methylation patterning</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCokus%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D452%26rft.issue%253D7184%26rft.spage%253D215%26rft.epage%253D219%26rft.atitle%253DShotgun%2Bbisulphite%2Bsequencing%2Bof%2Bthe%2BArabidopsis%2Bgenome%2Breveals%2BDNA%2Bmethylation%2Bpatterning.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06745%26rft_id%253Dinfo%253Apmid%252F18278030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18278030&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253925600040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.7" data-doi="10.1038/274775a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coulondre</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farabaugh</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Molecular basis of base substitution hotspots in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">775</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCoulondre%26rft.auinit1%253DC.%26rft.volume%253D274%26rft.issue%253D5673%26rft.spage%253D775%26rft.epage%253D780%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bbase%2Bsubstitution%2Bhotspots%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F274775a0%26rft_id%253Dinfo%253Apmid%252F355893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/274775a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=355893&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978FL81400026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.8" data-doi="10.3732/ajb.89.4.707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Small</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haselkorn</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Rapid diversification of the cotton genus (Gossypium: Malvaceae) revealed by analysis of sixteen nuclear and chloroplast genes</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DCronn%26rft.auinit1%253DR.%2BC.%26rft.volume%253D89%26rft.issue%253D4%26rft.spage%253D707%26rft.epage%253D725%26rft.atitle%253DRapid%2Bdiversification%2Bof%2Bthe%2Bcotton%2Bgenus%2B%2528Gossypium%253A%2BMalvaceae%2529%2Brevealed%2Bby%2Banalysis%2Bof%2Bsixteen%2Bnuclear%2Band%2Bchloroplast%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.89.4.707%26rft_id%253Dinfo%253Apmid%252F21665671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijg5LzQvNzA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.9" data-doi="10.1101/gr.4825606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Widespread genome duplications throughout the history of flowering plants</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">738</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4825606%26rft_id%253Dinfo%253Apmid%252F16702410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzM4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.10" data-doi="10.1038/nrg2600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Peer</span>  <span class="cit-name-given-names">Y. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The evolutionary significance of ancient genome duplications</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DVan%2Bde%2BPeer%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D725%26rft.epage%253D732%26rft.atitle%253DThe%2Bevolutionary%2Bsignificance%2Bof%2Bancient%2Bgenome%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2600%26rft_id%253Dinfo%253Apmid%252F19652647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19652647&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269965100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.11" data-doi="10.1126/science.1143986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome plasticity a key factor in the success of polyploid wheat under domestication</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1862</span>–<span class="cit-lpage">1866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDubcovsky%26rft.auinit1%253DJ.%26rft.volume%253D316%26rft.issue%253D5833%26rft.spage%253D1862%26rft.epage%253D1866%26rft.atitle%253DGenome%2BPlasticity%2Ba%2BKey%2BFactor%2Bin%2Bthe%2BSuccess%2Bof%2BPolyploid%2BWheat%2BUnder%2BDomestication%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143986%26rft_id%253Dinfo%253Apmid%252F17600208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgzMy8xODYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.12" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.13" data-doi="10.1111/j.1469-8137.2009.02923.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene duplication and evolutionary novelty in plants</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DFlagel%26rft.auinit1%253DL.%2BE.%26rft.volume%253D183%26rft.issue%253D3%26rft.spage%253D557%26rft.epage%253D564%26rft.atitle%253DGene%2Bduplication%2Band%2Bevolutionary%2Bnovelty%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.02923.x%26rft_id%253Dinfo%253Apmid%252F19555435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.02923.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19555435&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268100800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.14" data-doi="10.1186/1471-2164-13-302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DFlagel%26rft.auinit1%253DL.%2BE.%26rft.volume%253D13%26rft.spage%253D302%26rft.epage%253D302%26rft.atitle%253DDuplicate%2Bgene%2Bevolution%252C%2Bhomoeologous%2Brecombination%252C%2Band%2Btranscriptome%2Bcharacterization%2Bin%2Ballopolyploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-302%26rft_id%253Dinfo%253Apmid%252F22768919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22768919&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.15" data-doi="10.1101/gr.3681406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Gene-balanced duplications, like tetraploidy, provide predictable drive to increase morphological complexity</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">805</span>–<span class="cit-lpage">814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DFreeling%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D805%26rft.epage%253D814%26rft.atitle%253DGene-balanced%2Bduplications%252C%2Blike%2Btetraploidy%252C%2Bprovide%2Bpredictable%2Bdrive%2Bto%2Bincrease%2Bmorphological%2Bcomplexity%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3681406%26rft_id%253Dinfo%253Apmid%252F16818725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzcvODA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.16" data-doi="10.1111/j.1469-8137.2009.03089.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaeta</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Homoeologous recombination in allopolyploids: the polyploid ratchet</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DGaeta%26rft.auinit1%253DR.%2BT.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D18%26rft.epage%253D28%26rft.atitle%253DHomoeologous%2Brecombination%2Bin%2Ballopolyploids%253A%2Bthe%2Bpolyploid%2Bratchet.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03089.x%26rft_id%253Dinfo%253Apmid%252F20002315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03089.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20002315&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.17" data-doi="10.1038/nmeth.1613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Computational methods for transcriptome annotation and quantification using RNA-seq</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGarber%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D469%26rft.epage%253D477%26rft.atitle%253DComputational%2Bmethods%2Bfor%2Btranscriptome%2Bannotation%2Band%2Bquantification%2Busing%2BRNA-seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1613%26rft_id%253Dinfo%253Apmid%252F21623353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623353&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291031800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.18" data-doi="10.1126/science.1177837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A first-generation haplotype map of maize</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGore%26rft.auinit1%253DM.%2BA.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1115%26rft.epage%253D1117%26rft.atitle%253DA%2BFirst-Generation%2BHaplotype%2BMap%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1177837%26rft_id%253Dinfo%253Apmid%252F19965431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The developmental transcriptome of <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D471%26rft.spage%253D473%26rft_id%253Dinfo%253Apmid%252F21179090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21179090&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.20" data-doi="10.1038/nmeth.1503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">O. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mwenifumbo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrissy</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Alternative expression analysis by RNA sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">843</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGriffith%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253D843%26rft.epage%253D847%26rft.atitle%253DAlternative%2Bexpression%2Banalysis%2Bby%2BRNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1503%26rft_id%253Dinfo%253Apmid%252F20835245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20835245&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282274700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.21" data-doi="10.1038/nbt.2302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clissold</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Associative transcriptomics of traits in the polyploid crop species <em>Brassica napus</em></span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">798</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHarper%26rft.auinit1%253DA.%2BL.%26rft.volume%253D30%26rft.issue%253D8%26rft.spage%253D798%26rft.epage%253D802%26rft.atitle%253DAssociative%2Btranscriptomics%2Bof%2Btraits%2Bin%2Bthe%2Bpolyploid%2Bcrop%2Bspecies%2BBrassica%2Bnapus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2302%26rft_id%253Dinfo%253Apmid%252F22820317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820317&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.22" data-doi="10.1038/nature09916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickett</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayyampalayam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanderbali</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landherr</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ancestral polyploidy in seed plants and angiosperms</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">473</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJiao%26rft.auinit1%253DY.%26rft.volume%253D473%26rft.issue%253D7345%26rft.spage%253D97%26rft.epage%253D100%26rft.atitle%253DAncestral%2Bpolyploidy%2Bin%2Bseed%2Bplants%2Band%2Bangiosperms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09916%26rft_id%253Dinfo%253Apmid%252F21478875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478875&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290218300038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.23" data-doi="10.1038/nprot.2009.244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaufmann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muiño</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Østerås</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farinelli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krajewski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Chromatin immunoprecipitation (ChIP) of plant transcription factors followed by sequencing (ChIP-SEQ) or hybridization to whole genome arrays (ChIP-CHIP)</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKaufmann%26rft.auinit1%253DK.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D457%26rft.epage%253D472%26rft.atitle%253DChromatin%2Bimmunoprecipitation%2B%2528ChIP%2529%2Bof%2Bplant%2Btranscription%2Bfactors%2Bfollowed%2Bby%2Bsequencing%2B%2528ChIP-SEQ%2529%2Bor%2Bhybridization%2Bto%2Bwhole%2Bgenome%2Barrays%2B%2528ChIP-CHIP%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.244%26rft_id%253Dinfo%253Apmid%252F20203663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20203663&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275234900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-3.3.517.24" data-doi="10.1126/scitranslmed.3004323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-pub-date">2012</span> <span class="cit-article-title">Noninvasive whole-genome sequencing of a human fetus</span>. <span class="cit-source">Sci. Transl. Med.</span> <span class="cit-vol">4</span>: <span class="cit-fpage">137ra76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DKitzman%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D4%26rft.issue%253D137%26rft.spage%253D137ra76%26rft.epage%253D137ra76%26rft.atitle%253DNoninvasive%2BWhole-Genome%2BSequencing%2Bof%2Ba%2BHuman%2BFetus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.3004323%26rft_id%253Dinfo%253Apmid%252F22674554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjEzOiI0LzEzNy8xMzdyYTc2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.25" data-doi="10.1038/nmeth.1828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krueger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreck</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DNA methylome analysis using short bisulfite sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DKrueger%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D145%26rft.epage%253D151%26rft.atitle%253DDNA%2Bmethylome%2Banalysis%2Busing%2Bshort%2Bbisulfite%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1828%26rft_id%253Dinfo%253Apmid%252F22290186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22290186&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300029600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.26" data-doi="10.1111/j.1467-7652.2012.00718.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berkman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenc</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Single nucleotide polymorphism discovery from wheat next-generation sequence data</span>. <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DLai%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D743%26rft.epage%253D749%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bfrom%2Bwheat%2Bnext-generation%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00718.x%26rft_id%253Dinfo%253Apmid%252F22748104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00718.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22748104&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306131400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.27" data-doi="10.1038/nrg2732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laird</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Principles and challenges of genome-wide DNA methylation analysis</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLaird%26rft.auinit1%253DP.%2BW.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D191%26rft.epage%253D203%26rft.atitle%253DPrinciples%2Band%2Bchallenges%2Bof%2Bgenomewide%2BDNA%2Bmethylation%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2732%26rft_id%253Dinfo%253Apmid%252F20125086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20125086&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274649900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.28" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.29" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.30" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>.<span class="cit-auth cit-collab">1000 Genome Project Data Processing Subgroup</span>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy81MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.31" data-doi="10.1101/gr.083451.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the fifth base: Genome-wide sequencing of cytosine methylation</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083451.108%26rft_id%253Dinfo%253Apmid%252F19273618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzYvOTU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.32" data-doi="10.1016/j.cell.2008.03.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonti-Filippini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Highly integrated single-base resolution maps of the epigenome in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLister%26rft.auinit1%253DR.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D523%26rft.epage%253D536%26rft.atitle%253DHighly%2Bintegrated%2Bsingle-base%2Bresolution%2Bmaps%2Bof%2Bthe%2Bepigenome%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.03.029%26rft_id%253Dinfo%253Apmid%252F18423832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.03.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18423832&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255503900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.33" data-doi="10.1038/nature08514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelizzola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowen</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Human DNA methylomes at base resolution show widespread epigenomic differences</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">462</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLister%26rft.auinit1%253DR.%26rft.volume%253D462%26rft.issue%253D7271%26rft.spage%253D315%26rft.epage%253D322%26rft.atitle%253DHuman%2BDNA%2Bmethylomes%2Bat%2Bbase%2Bresolution%2Bshow%2Bwidespread%2Bepigenomic%2Bdifferences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08514%26rft_id%253Dinfo%253Apmid%252F19829295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19829295&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271899300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.34" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy81MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.35" data-doi="10.1101/gr.102491.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McManus</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coolon</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eipper-Mains</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regulatory divergence in Drosophila revealed by mRNA-seq</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102491.109%26rft_id%253Dinfo%253Apmid%252F20354124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvODE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.36" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.37" data-doi="10.1038/nature09990"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nègre</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bristow</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A cis-regulatory map of the Drosophila genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNegre%26rft.auinit1%253DN.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D527%26rft.epage%253D531%26rft.atitle%253DA%2Bcis-regulatory%2Bmap%2Bof%2Bthe%2BDrosophila%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09990%26rft_id%253Dinfo%253Apmid%252F21430782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09990&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21430782&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.38" data-doi="10.1016/S0168-9525(03)00015-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chris Pires</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auger</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Understanding mechanisms of novel gene expression in polyploids</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DOsborn%26rft.auinit1%253DT.%2BC.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D141%26rft.epage%253D147%26rft.atitle%253DUnderstanding%2Bmechanisms%2Bof%2Bnovel%2Bgene%2Bexpression%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252803%252900015-5%26rft_id%253Dinfo%253Apmid%252F12615008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(03)00015-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12615008&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181584500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.39" data-doi="10.1038/nrg2641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">ChIP-seq: advantages and challenges of a maturing technology</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPark%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D669%26rft.epage%253D680%26rft.atitle%253DChIP-seq%253A%2Badvantages%2Band%2Bchallenges%2Bof%2Ba%2Bmaturing%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2641%26rft_id%253Dinfo%253Apmid%252F19736561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19736561&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269965100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.40" data-doi="10.1111/j.1469-8137.2005.01347.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandepoele</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ancient duplication of cereal genomes</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">658</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D658%26rft.epage%253D661%26rft.atitle%253DAncient%2Bduplication%2Bof%2Bcereal%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2005.01347.x%26rft_id%253Dinfo%253Apmid%252F15720677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2005.01347.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15720677&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227229800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.41" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibers</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.42" data-doi="10.1038/nature06250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varilly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hostetter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide detection and characterization of positive selection in human populations</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D913%26rft.epage%253D918%26rft.atitle%253DGenome-wide%2Bdetection%2Band%2Bcharacterization%2Bof%2Bpositive%2Bselection%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06250%26rft_id%253Dinfo%253Apmid%252F17943131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943131&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Homoeologous nonreciprocal recombination in polyploid cotton</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D186%26rft.spage%253D123%26rft_id%253Dinfo%253Apmid%252F19925554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19925554&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.44" data-doi="10.1093/jhered/91.3.242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schranz</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Novel flowering time variation in the resynthesized polyploid Brassica napus</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DSchranz%26rft.auinit1%253DM.%2BE.%26rft.volume%253D91%26rft.issue%253D3%26rft.spage%253D242%26rft.epage%253D246%26rft.atitle%253DNovel%2Bflowering%2Btime%2Bvariation%2Bin%2Bthe%2Bresynthesized%2Bpolyploid%2BBrassica%2Bnapus%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F91.3.242%26rft_id%253Dinfo%253Apmid%252F10833052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6IjkxLzMvMjQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.45" data-doi="10.1093/molbev/msg065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senchina</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Rate Variation Among Nuclear Genes and the Age of Polyploidy in Gossypium</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg065%26rft_id%253Dinfo%253Apmid%252F12679546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzQvNjMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.46" data-doi="10.1111/j.1095-8312.2004.00335.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovarik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Recent and recurrent polyploidy in Tragopogon (Asteraceae): cytogenetic, genomic and genetic comparisons</span>. <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D82%26rft.spage%253D485%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2004.00335.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2004.00335.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223808600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-web" id="cit-3.3.517.47" data-doi="10.1101/gr.361602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Block</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulez</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chervitz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Bioperl toolkit: Perl modules for the life sciences</span>. <span class="cit-source">Genome Res.</span> <span class="cit-vol">12</span>: <span class="cit-fpage">1611</span>–<span class="cit-lpage">1618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DStajich%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1611%26rft.epage%253D1618%26rft.atitle%253DThe%2BBioperl%2BToolkit%253A%2BPerl%2BModules%2Bfor%2Bthe%2BLife%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.361602%26rft_id%253Dinfo%253Apmid%252F12368254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNjExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.517.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stebbins</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-source">Variation and Evolution in Plants</span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.49" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.50" data-doi="10.1101/gr.4602906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">A global assembly of cotton ESTs</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4602906%26rft_id%253Dinfo%253Apmid%252F16478941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNDQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.51" data-doi="10.1534/genetics.106.058271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">A novel approach for characterizing expression levels of genes duplicated by polyploidy</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1823</span>–<span class="cit-lpage">1827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058271%26rft_id%253Dinfo%253Apmid%252F16702424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE4MjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzMvNTE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.52" data-doi="10.1038/nmeth.1246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valouev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundquist</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anton</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide analysis of transcription factor binding sites based on ChIP-Seq data</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DValouev%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253D829%26rft.epage%253D834%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Btranscription%2Bfactor%2Bbinding%2Bsites%2Bbased%2Bon%2BChIP-Seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1246%26rft_id%253Dinfo%253Apmid%252F19160518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19160518&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258912700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.53" data-doi="10.1186/1471-2229-9-125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sampling nucleotide diversity in cotton</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BDeynze%26rft.auinit1%253DA.%2BE.%26rft.volume%253D9%26rft.spage%253D125%26rft.epage%253D125%26rft.atitle%253DSampling%2Bnucleotide%2Bdiversity%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-125%26rft_id%253Dinfo%253Apmid%252F19840401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840401&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.54" data-doi="10.1371/journal.pbio.0050174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanurdžić</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippman</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrasquillo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Epigenetic natural variation in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DVaughn%26rft.auinit1%253DM.%2BW.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De174%26rft.epage%253De174%26rft.atitle%253DEpigenetic%2Bnatural%2Bvariation%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050174%26rft_id%253Dinfo%253Apmid%252F17579518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17579518&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.55" data-doi="10.1038/nrg2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-3.3.517.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polyploidy and the evolutionary history of cotton</span>. <span class="cit-source">Adv. Agronomy</span> <span class="cit-vol">78</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgronomy%26rft.volume%253D78%26rft.spage%253D139%26rft.atitle%253DPOLYPLOIDY%2BAND%2BTHE%2BEVOLUTIONARY%2BHISTORY%2BOF%2BCOTTON%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.57" data-doi="10.1371/journal.pone.0011471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilbanks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Facciotti</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evaluation of algorithm performance in ChIP-seq peak detection</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWilbanks%26rft.auinit1%253DE.%2BG.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De11471%26rft.epage%253De11471%26rft.atitle%253DEvaluation%2Bof%2Balgorithm%2Bperformance%2Bin%2BChIP-seq%2Bpeak%2Bdetection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011471%26rft_id%253Dinfo%253Apmid%252F20628599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20628599&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.58" data-doi="10.1073/pnas.0811575106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takebayashi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayrose</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenspoon</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The frequency of polyploid speciation in vascular plants</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">13875</span>–<span class="cit-lpage">13879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0811575106%26rft_id%253Dinfo%253Apmid%252F19667210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzMzLzEzODc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8zLzUxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.59" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzg3MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMy81MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.60" data-doi="10.1111/j.1365-313X.2006.02829.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Accumulation of genome-specific transcripts, transcription factors and phytohormonal regulators during early stages of fiber cell development in allotetraploid cotton</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSamuel%2BYang%26rft.auinit1%253DS.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D761%26rft.epage%253D775%26rft.atitle%253DAccumulation%2Bof%2Bgenome-specific%2Btranscripts%252C%2Btranscription%2Bfactors%2Band%2Bphytohormonal%2Bregulators%2Bduring%2Bearly%2Bstages%2Bof%2Bfiber%2Bcell%2Bdevelopment%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.02829.x%26rft_id%253Dinfo%253Apmid%252F16889650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.02829.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16889650&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239700000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-other" id="cit-3.3.517.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome</span>. <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.3.517.62" data-doi="10.1016/j.cell.2006.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yazaki</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundaresan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cokus</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZhang%26rft.auinit1%253DX.%26rft.volume%253D126%26rft.issue%253D6%26rft.spage%253D1189%26rft.epage%253D1201%26rft.atitle%253DGenome-wide%2Bhigh-resolution%2Bmapping%2Band%2Bfunctional%2Banalysis%2Bof%2BDNA%2Bmethylation%2Bin%2Barabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.08.003%26rft_id%253Dinfo%253Apmid%252F16949657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16949657&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F3%2F517.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240897600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/3/517.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/3/505" title="A Genomic Survey of Reb Homologs Suggests Widespread Occurrence of R-Bodies in Proteobacteria" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/3/527" title="Calcineurin Governs Thermotolerance and Virulence of Cryptococcus gattii" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/3"><span>Volume 3 Issue 3, March 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/3.cover-source.jpg?itok=w5x3d6rp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/3/517" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ES01Jct7O26etv1gEPpDNKiojeiqLX8HPf5iI_lnJ60" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/3/517" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/3/517" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/3/517" class="active" data-icon-position="" data-hide-link-title="0">PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459424" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hopWxHDYOCog_1-Zmo5q2TbYQRWz8DDxIwKSlcNSOK8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/517.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/3/517" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KJLXedr1fTGjvqsRtArBo9vW80xDPboYDbMIpsWXMMI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/3/517" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/3/517.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459424" id="node459424--2" data-pisa="ggg;3/3/517" data-pisa-master="ggg;3/3/517" data-apath="/ggg/3/3/517.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin T. Page</span>, <span class="highwire-citation-author" data-delta="1">Alan R. Gingle</span> and <span class="highwire-citation-author" data-delta="2">Joshua A. Udall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">517-525;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005298 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459424--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin T. Page </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BT.%2BPage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Page%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BT.%2BPage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alan R. Gingle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia, 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BR.%2BGingle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gingle%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BR.%2BGingle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaudall@byu.edu" class="" data-icon-position="" data-hide-link-title="0">jaudall@byu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/3/517" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/3/517&amp;title=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/3/517&amp;title=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/3/517&amp;title=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/3/517&amp;text=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/3/517&amp;title=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/3/517&amp;t=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/3/517&amp;title=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/3/517&amp;title=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459424/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459424" id="node459424--3" data-pisa="ggg;3/3/517" data-pisa-master="ggg;3/3/517" data-apath="/ggg/3/3/517.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Justin T. Page</span>, <span class="highwire-citation-author" data-delta="1">Alan R. Gingle</span> and <span class="highwire-citation-author" data-delta="2">Joshua A. Udall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">517-525;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005298 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459424--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justin T. Page </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BT.%2BPage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Page%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BT.%2BPage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alan R. Gingle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genome Mapping Laboratory, University of Georgia, Athens, Georgia, 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BR.%2BGingle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gingle%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BR.%2BGingle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant and Wildlife Science Department, Brigham Young University, Provo, Utah 84062</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaudall@byu.edu" class="" data-icon-position="" data-hide-link-title="0">jaudall@byu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=517&amp;rft.epage=525&amp;rft.atitle=PolyCat%3A%20A%20Resource%20for%20Genome%20Categorization%20of%20Sequencing%20Reads%20From%20Allopolyploid%20Organisms&amp;rft.volume=3&amp;rft.issue=3&amp;rft.date=2013-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Page%2C+Justin+T.&amp;rft.au=Gingle%2C+Alan+R.&amp;rft.au=Udall%2C+Joshua+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459424/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459424/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459424/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459424/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459424/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459424/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459424/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459424/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459424/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459424/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459424/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459424/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/3/517" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/3/517.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/3/517.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/3/517.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/3/517.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
