<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2277" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2277.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2277.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2277.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2277.atom" />
<meta name="HW.pisa" content="ggg;9/7/2277" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="De Novo, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in Pristionchus Nematodes" />
<meta name="DC.Identifier" content="10.1534/g3.119.400326" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Prabh, Rodelsperger. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Homology is a fundamental concept in comparative biology. It is extensively used at the sequence level to make phylogenetic hypotheses and functional inferences. Nonetheless, the majority of eukaryotic genomes contain large numbers of orphan genes lacking homologs in other taxa. Generally, the fraction of orphan genes is higher in genomically undersampled clades, and in the absence of closely related genomes any hypothesis about their origin and evolution remains untestable. Previously, we sequenced ten genomes with an underlying ladder-like phylogeny to establish a phylogenomic framework for studying genome evolution in diplogastrid nematodes. Here, we use this deeply sampled data set to understand the processes that generate orphan genes in our focal species Pristionchus pacificus . Based on phylostratigraphic analysis and additional bioinformatic filters, we obtained 29 high-confidence candidate genes for which mechanisms of orphan origin were proposed based on manual inspection. This revealed diverse mechanisms including annotation artifacts, chimeric origin, alternative reading frame usage, and gene splitting with subsequent gain of de novo exons. In addition, we present two cases of complete de novo origination from non-coding regions, which represents one of the first reports of de novo genes in nematodes. Thus, we conclude that de novo emergence, divergence, and mixed mechanisms contribute to novel gene formation in Pristionchus nematodes." />
<meta name="DC.Contributor" content="Neel Prabh" />
<meta name="DC.Contributor" content="Christian Rödelsperger" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="De Novo, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in Pristionchus Nematodes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Homology is a fundamental concept in comparative biology. It is extensively used at the sequence level to make phylogenetic hypotheses and functional inferences. Nonetheless, the majority of eukaryotic genomes contain large numbers of orphan genes lacking homologs in other taxa. Generally, the fraction of orphan genes is higher in genomically undersampled clades, and in the absence of closely related genomes any hypothesis about their origin and evolution remains untestable. Previously, we sequenced ten genomes with an underlying ladder-like phylogeny to establish a phylogenomic framework for studying genome evolution in diplogastrid nematodes. Here, we use this deeply sampled data set to understand the processes that generate orphan genes in our focal species &lt;i&gt;Pristionchus pacificus&lt;/i&gt;. Based on phylostratigraphic analysis and additional bioinformatic filters, we obtained 29 high-confidence candidate genes for which mechanisms of orphan origin were proposed based on manual inspection. This revealed diverse mechanisms including annotation artifacts, chimeric origin, alternative reading frame usage, and gene splitting with subsequent gain of &lt;i&gt;de novo&lt;/i&gt; exons. In addition, we present two cases of complete &lt;i&gt;de novo&lt;/i&gt; origination from non-coding regions, which represents one of the first reports of &lt;i&gt;de novo&lt;/i&gt; genes in nematodes. Thus, we conclude that &lt;i&gt;de novo&lt;/i&gt; emergence, divergence, and mixed mechanisms contribute to novel gene formation in &lt;i&gt;Pristionchus&lt;/i&gt; nematodes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2277" />
<meta name="citation_id" content="9/7/2277" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2277" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2277.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2277.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2277.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400326" />
<meta name="citation_pmid" content="31088903" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2277" />
<meta name="citation_lastpage" content="2286" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Neel Prabh" />
<meta name="citation_author_institution" content="Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany" />
<meta name="citation_author_institution" content="Department of Evolutionary Genetics, Max-Planck-Institute for Evolutionary Biology, August Thienemann Str. 2, 24306 Plön, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3099-1007" />
<meta name="citation_author" content="Christian Rödelsperger" />
<meta name="citation_author_institution" content="Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany" />
<meta name="citation_author_email" content="christian.roedelsperger@tuebingen.mpg.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7905-9675" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. Aguilera;citation_author=C. McDougall;citation_author=B. M. Degnan;citation_title=Co-option and de novo gene evolution underlie molluscan shell diversity.;citation_pages=779-792;citation_volume=34;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=P. Baskaran;citation_author=C. Rödelsperger;citation_author=N. Prabh;citation_author=V. Serobyan;citation_author=G. V. Markov;citation_title=Ancient gene duplications have shaped developmental stage-specific expression in Pristionchus pacificus.;citation_pages=185;citation_volume=15;citation_year=2015;citation_pmid=26370559;citation_doi=10.1186/s12862-015-0466-2" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. Borchert;citation_author=C. Dieterich;citation_author=K. Krug;citation_author=W. Schutz;citation_author=S. Jung;citation_title=Proteogenomics of Pristionchus pacificus reveals distinct proteome structure of nematode models.;citation_volume=20;citation_year=2010;citation_pmid=20237107;citation_doi=10.1101/gr.103119.109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Cai;citation_author=R. Zhao;citation_author=H. Jiang;citation_author=W. Wang;citation_title=De Novo Origination of a New Protein-Coding Gene in Saccharomyces cerevisiae;citation_pages=487-496;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493065;citation_doi=10.1534/genetics.107.084491" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A.-R. Carvunis;citation_author=T. Rolland;citation_author=I. Wapinski;citation_author=M. A. Calderwood;citation_author=M. A. Yildirim;citation_title=Proto-genes and de novo gene birth.;citation_pages=370-374;citation_volume=487;citation_year=2012;citation_issue=7407;citation_pmid=22722833;citation_doi=10.1038/nature11184" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Chen;citation_author=A. L. DeVries;citation_author=C. H. Cheng;citation_title=Evolution of antifreeze glycoprotein gene from a trypsinogen gene in Antarctic notothenioid fish;citation_pages=3811-3816;citation_volume=94;citation_year=1997;citation_issue=8;citation_pmid=9108060;citation_doi=10.1073/pnas.94.8.3811" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=S. Chen;citation_author=B. H. Krinsky;citation_author=M. Long;citation_title=New genes as drivers of phenotypic evolution.;citation_pages=645-660. Erratum: 744;citation_volume=14;citation_year=2013;citation_pmid=23949544;citation_doi=10.1038/nrg3521" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J.-Y. Chen;citation_author=Q. S. Shen;citation_author=W.-Z. Zhou;citation_author=J. Peng;citation_author=B. Z. He;citation_title=Emergence, retention and selection: a trilogy of origination for functional de novo proteins from ancestral LncRNAs in Primates.;citation_pages=e1005391;citation_volume=11;citation_year=2015;citation_pmid=26177073;citation_doi=10.1371/journal.pgen.1005391" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=W.-Y. Chung;citation_author=S. Wadhawan;citation_author=R. Szklarczyk;citation_author=S. K. Pond;citation_author=A. Nekrutenko;citation_title=A first look at ARFome: dual-coding genes in mammalian genomes.;citation_pages=e91-e91;citation_volume=3;citation_year=2007;citation_issue=5;citation_pmid=17511511;citation_doi=10.1371/journal.pcbi.0030091" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=L. Delaye;citation_author=A. Deluna;citation_author=A. Lazcano;citation_author=A. Becerra;citation_title=The origin of a novel gene through overprinting in Escherichia coli.;citation_pages=31-31;citation_volume=8;citation_year=2008;citation_pmid=18226237;citation_doi=10.1186/1471-2148-8-31" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. Domazet-Loso;citation_author=J. Brajković;citation_author=D. Tautz;citation_title=A phylostratigraphy approach to uncover the genomic history of major adaptations in metazoan lineages.;citation_pages=533-539;citation_volume=23;citation_year=2007;citation_issue=11;citation_pmid=18029048;citation_doi=10.1016/j.tig.2007.08.014" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=B. Dujon;citation_title=The yeast genome project: what did we learn?;citation_pages=263-270;citation_volume=12;citation_year=1996;citation_issue=7;citation_pmid=8763498;citation_doi=10.1016/0168-9525(96)10027-5" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=A. M. Gontijo;citation_author=V. Miguela;citation_author=M. F. Whiting;citation_author=R. C. Woodruff;citation_author=M. Dominguez;citation_title=Intron retention in the Drosophila melanogaster Rieske Iron Sulphur Protein gene generated a new protein.;citation_pages=323-323;citation_volume=2;citation_year=2011;citation_pmid=21610726;citation_doi=10.1038/ncomms1328" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Gouy;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=SeaView Version 4: A multiplatform graphical user interface for sequence alignment and phylogenetic tree building.;citation_volume=27;citation_year=2009;citation_pmid=19854763;citation_doi=10.1093/molbev/msp259" />
<meta name="citation_reference" content="citation_title=Evolution of Living Organisms: Evidence for a New Theory of Transformation;citation_author=P.-P. Grassé;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=Y. Guan;citation_author=L. Liu;citation_author=Q. Wang;citation_author=J. Zhao;citation_author=P. Li;citation_title=Gene refashioning through innovative shifting of reading frames in mosses.;citation_pages=1555;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-04025-x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. J. Hangauer;citation_author=I. W. Vaughn;citation_author=M. T. McManus;citation_title=Pervasive transcription of the human genome produces thousands of previously unidentified long intergenic noncoding RNAs.;citation_pages=e1003569;citation_volume=9;citation_year=2013;citation_pmid=23818866;citation_doi=10.1371/journal.pgen.1003569" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=T. J. A. J. Heinen;citation_author=J. A. Tobias;citation_author=F. Staubach;citation_author=D. Häming;citation_author=D. Tautz;citation_title=Emergence of a new gene from an intergenic region.;citation_pages=1527-1531;citation_volume=19;citation_year=2009;citation_issue=18;citation_pmid=19733073;citation_doi=10.1016/j.cub.2009.07.049" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=F. Jacob;citation_title=Evolution and tinkering;citation_pages=1161-1166;citation_volume=196;citation_year=1977;citation_issue=4295;citation_pmid=860134;citation_doi=10.1126/science.860134" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. H. Jan;citation_author=R. C. Friedman;citation_author=J. Graham Ruby;citation_author=D. P. Bartel;citation_title=Formation, regulation and evolution of Caenorhabditis elegans 3′UTRs.;citation_pages=97-101;citation_volume=469;citation_year=2010;citation_pmid=21085120;citation_doi=10.1038/nature09616" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=N. Kanzaki;citation_author=E. J. Ragsdale;citation_author=M. Herrmann;citation_author=W. E. Mayer;citation_author=R. J. Sommer;citation_title=Description of three Pristionchus species (Nematoda: Diplogastridae) from Japan that form a cryptic species complex with the model organism P. pacificus.;citation_pages=403-417;citation_volume=29;citation_year=2012;citation_issue=6;citation_pmid=22639812;citation_doi=10.2108/zsj.29.403" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Kawasaki;citation_author=A.-G. Lafont;citation_author=J.-Y. Sire;citation_title=The evolution of milk casein genes from tooth genes before the origin of mammals.;citation_volume=28;citation_year=2011;citation_pmid=21245413;citation_doi=10.1093/molbev/msr020" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. K. Keese;citation_author=A. Gibbs;citation_title=Origins of genes: &quot;big bang&quot; or continuous creation?;citation_pages=9489-9493;citation_volume=89;citation_year=1992;citation_issue=20;citation_pmid=1329098;citation_doi=10.1073/pnas.89.20.9489" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=K. Khalturin;citation_author=G. Hemmrich;citation_author=S. Fraune;citation_author=R. Augustin;citation_author=T. C. G. Bosch;citation_title=More than just orphans: are taxonomically-restricted genes important in evolution?;citation_pages=404-413;citation_volume=25;citation_year=2009;citation_issue=9;citation_pmid=19716618;citation_doi=10.1016/j.tig.2009.07.006" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_title=FEBS J.;citation_author=S. Klasberg;citation_author=T. Bitard-Feildel;citation_author=I. Callebaut;citation_author=E. Bornberg-Bauer;citation_title=Origins and structural properties of novel and de novo protein domains during insect evolution.;citation_pages=2605-2625;citation_volume=285;citation_year=2018;citation_doi=10.1111/febs.14504" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. G. Knowles;citation_author=A. McLysaght;citation_title=Recent de novo origin of human protein-coding genes.;citation_volume=19;citation_year=2009;citation_pmid=19726446;citation_doi=10.1101/gr.095026.109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. T. Levine;citation_author=C. D. Jones;citation_author=A. D. Kern;citation_author=H. A. Lindfors;citation_author=D. J. Begun;citation_title=Novel genes derived from noncoding DNA in Drosophila melanogaster are frequently X-linked and exhibit testis-biased expression.;citation_volume=103;citation_year=2006;citation_pmid=16777968;citation_doi=10.1073/pnas.0509809103" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=C.-Y. Li;citation_author=Y. Zhang;citation_author=Z. Wang;citation_author=Y. Zhang;citation_author=C. Cao;citation_title=A human-specific de novo protein-coding gene associated with human brain functions.;citation_pages=e1000734-e1000734;citation_volume=6;citation_year=2010;citation_issue=3;citation_pmid=20376170;citation_doi=10.1371/journal.pcbi.1000734" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Long;citation_author=E. Betrán;citation_author=K. Thornton;citation_author=W. Wang;citation_title=The origin of new genes: glimpses from the young and old.;citation_pages=865-875;citation_volume=4;citation_year=2003;citation_issue=11;citation_pmid=14634634;citation_doi=10.1038/nrg1204" />
<meta name="citation_reference" content="citation_journal_title=Computational biology and chemistry;citation_journal_abbrev=Comput Biol Chem;citation_author=I. Makalowska;citation_author=C.-F. Lin;citation_author=W. Makalowski;citation_title=Overlapping genes in vertebrate genomes.;citation_pages=1-12;citation_volume=29;citation_year=2005;citation_issue=1;citation_pmid=15680581;citation_doi=10.1016/j.compbiolchem.2004.12.006" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. G. Mayer;citation_author=C. Rödelsperger;citation_author=H. Witte;citation_author=M. Riebesell;citation_author=R. J. Sommer;citation_title=The orphan gene dauerless regulates dauer development and intraspecific competition in nematodes by copy Number Variation.;citation_pages=e1005146;citation_volume=11;citation_year=2015;citation_pmid=26087034;citation_doi=10.1371/journal.pgen.1005146" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=A. McLysaght;citation_author=L. D. Hurst;citation_title=Open questions in the study of de novo genes: what, how and why.;citation_pages=567-578;citation_volume=17;citation_year=2016;citation_pmid=WOS:00038151;citation_doi=10.1038/nrg.2016.78" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Milde;citation_author=G. Hemmrich;citation_author=F. Anton-Erxleben;citation_author=K. Khalturin;citation_author=J. Wittlieb;citation_title=Characterization of taxonomically restricted genes in a phylum-restricted cell type.;citation_pages=R8-R8;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19161630;citation_doi=10.1186/gb-2009-10-1-r8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. N. Murphy;citation_author=A. McLysaght;citation_title=De novo origin of protein-coding genes in murine rodents.;citation_pages=e48650-e48650;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185269;citation_doi=10.1371/journal.pone.0048650" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Nekrutenko;citation_author=S. Wadhawan;citation_author=P. Goetting-Minesky;citation_author=K. D. Makova;citation_title=Oscillating evolution of a mammalian locus with overlapping reading frames: an XLalphas/ALEX relay.;citation_pages=e18-e18;citation_volume=1;citation_year=2005;citation_issue=2;citation_pmid=16110341;citation_doi=10.1371/journal.pgen.0010018" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Ohno;citation_title=Birth of a unique enzyme from an alternative reading frame of the preexisted, internally repetitious coding sequence.;citation_pages=2421-2425;citation_volume=81;citation_year=1984;citation_issue=8;citation_pmid=6585807;citation_doi=10.1073/pnas.81.8.2421" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=N. Palmieri;citation_author=C. Kosiol;citation_author=C. Schlötterer;citation_title=The life cycle of Drosophila orphan genes.;citation_volume=3;citation_year=2014;citation_pmid=24554240;citation_doi=10.7554/eLife.01311" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=N. Prabh;citation_author=C. Rödelsperger;citation_title=Are orphan genes protein-coding, prediction artifacts, or non-coding RNAs?;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12859-016-1102-x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. Prabh;citation_author=W. Roeseler;citation_author=H. Witte;citation_author=G. Eberhardt;citation_author=R. J. Sommer;citation_title=Deep taxon sampling reveals the evolutionary dynamics of novel gene families in Pristionchus nematodes.;citation_volume=28;citation_year=2018;citation_pmid=30232197;citation_doi=10.1101/gr.234971.118" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. Rödelsperger;citation_author=C. Dieterich;citation_title=CYNTENATOR: progressive gene order alignment of 17 vertebrate genomes.;citation_pages=e8861-e8861;citation_volume=5;citation_year=2010;citation_issue=1;citation_pmid=20126624;citation_doi=10.1371/journal.pone.0008861" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=C. Rödelsperger;citation_author=K. Menden;citation_author=V. Serobyan;citation_author=H. Witte;citation_author=P. Baskaran;citation_title=First insights into the nature and evolution of antisense transcription in nematodes.;citation_pages=165;citation_volume=16;citation_year=2016;citation_doi=10.1186/s12862-016-0740-y" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=C. Rödelsperger;citation_author=J. M. Meyer;citation_author=N. Prabh;citation_author=C. Lanz;citation_author=F. Bemm;citation_title=Single-molecule sequencing reveals the chromosome-scale genomic architecture of the nematode model organism Pristionchus pacificus.;citation_pages=834-844;citation_volume=21;citation_year=2017;citation_doi=10.1016/j.celrep.2017.09.077" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Rödelsperger;citation_author=R. A. Neher;citation_author=A. M. Weller;citation_author=G. Eberhardt;citation_author=H. Witte;citation_title=Characterization of genetic diversity in the nematode Pristionchus pacificus from population-scale resequencing data.;citation_volume=196;citation_year=2014;citation_pmid=24443445;citation_doi=10.1534/genetics.113.159855" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=C. Rödelsperger;citation_author=W. Röseler;citation_author=N. Prabh;citation_author=K. Yoshida;citation_author=C. Weiler;citation_title=Phylotranscriptomics of Pristionchus nematodes reveals parallel gene loss in six hermaphroditic lineages.;citation_pages=3123-3127.e5;citation_volume=28;citation_year=2018;citation_pmid=30245109;citation_doi=10.1016/j.cub.2018.07.041" />
<meta name="citation_reference" content="citation_title=eLS;citation_author=C. Rödelsperger;citation_author=A. Streit;citation_author=R. J. Sommer;citation_title=Structure, function and evolution of the nematode genome;citation_year=2013;citation_doi=10.1002/9780470015902.a0024603" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. Rogers;citation_title=Adding resolution and dimensionality to comparative genomics: moving from reference genomes to clade genomics.;citation_pages=115;citation_volume=19;citation_year=2018;citation_doi=10.1186/s13059-018-1500-7" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Rošić;citation_author=R. Amouroux;citation_author=C. E. Requena;citation_author=A. Gomes;citation_author=M. Emperle;citation_title=Evolutionary analysis indicates that DNA alkylation damage is a byproduct of cytosine DNA methyltransferase activity.;citation_pages=452-459;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0061-8" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=J. Ruiz-Orera;citation_author=M. Mar Albà;citation_title=Translation of small open reading frames: roles in regulation and evolutionary innovation.;citation_pages=186-198;citation_volume=35;citation_year=2019;citation_pmid=30606460;citation_doi=10.1016/j.tig.2018.12.003" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=J. Ruiz-Orera;citation_author=P. Verdaguer-Grau;citation_author=J. L. Villanueva-Cañas;citation_author=X. Messeguer;citation_author=M. Mar Albà;citation_title=Translation of neutrally evolving peptides provides a basis for de novo gene evolution.;citation_pages=890-896;citation_volume=2;citation_year=2018;citation_pmid=29556078;citation_doi=10.1038/s41559-018-0506-6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Sabath;citation_author=A. Wagner;citation_author=D. Karlin;citation_title=Evolution of viral proteins originated de novo by overprinting.;citation_volume=29;citation_year=2012;citation_pmid=22821011;citation_doi=10.1093/molbev/mss179" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=P. Sarkies;citation_author=M. E. Selkirk;citation_author=J. T. Jones;citation_author=V. Blok;citation_author=T. Boothby;citation_title=Ancient and novel small RNA pathways compensate for the loss of piRNAs in multiple independent nematode lineages.;citation_pages=e1002061;citation_volume=13;citation_year=2015;citation_pmid=25668728;citation_doi=10.1371/journal.pbio.1002061" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. J. Schmid;citation_author=C. F. Aquadro;citation_title=The Evolutionary Analysis of &quot;Orphans&quot; From the Drosophila Genome Identifies Rapidly Diverging and Incorrectly Annotated Genes;citation_pages=589-598;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606536" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=K. J. Schmid;citation_author=D. Tautz;citation_title=A screen for fast evolving genes from Drosophila;citation_pages=9746-9750;citation_volume=94;citation_year=1997;citation_issue=18;citation_pmid=9275195;citation_doi=10.1073/pnas.94.18.9746" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=J. F. Schmitz;citation_author=K. K. Ullrich;citation_author=E. Bornberg-Bauer;citation_title=Incipient de novo genes can evolve from frozen accidents that escaped rapid transcript turnover.;citation_pages=1626-1632;citation_volume=2;citation_year=2018;citation_pmid=30201962;citation_doi=10.1038/s41559-018-0639-7" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=A. Sinha;citation_author=C. Langnick;citation_author=R. J. Sommer;citation_author=C. Dieterich;citation_title=Genome-wide analysis of trans-splicing in the nematode Pristionchus pacificus unravels conserved gene functions for germline and dauer development in divergent operons.;citation_volume=20;citation_year=2014;citation_pmid=25015138;citation_doi=10.1261/rna.041954.113" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. S. C. Slater;citation_author=E. Birney;citation_title=Automated generation of heuristics for biological sequence comparison.;citation_pages=31-31;citation_volume=6;citation_year=2005;citation_pmid=15713233;citation_doi=10.1186/1471-2105-6-31" />
<meta name="citation_reference" content="citation_title=Pristionchus pacificus: A Nematode Model for Comparative and Evolutionary Biology;citation_author=R. J. Sommer;citation_year=2015;citation_doi=10.1163/9789004260306" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=R. J. Sommer;citation_author=P. W. Sternberg;citation_title=Evolution of nematode vulval fate patterning.;citation_pages=396-407;citation_volume=173;citation_year=1996;citation_issue=2;citation_pmid=8606000;citation_doi=10.1006/dbio.1996.0035" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Srinivasan;citation_author=W. Sinz;citation_author=C. Lanz;citation_author=A. Brand;citation_author=R. Nandakumar;citation_title=A Bacterial Artificial Chromosome-Based Genetic Linkage Map of the Nematode Pristionchus pacificus;citation_pages=129-134;citation_volume=162;citation_year=2002;citation_issue=1;citation_pmid=12242228" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. C. Stein;citation_author=Y. Yu;citation_author=D. Copetti;citation_author=D. J. Zwickl;citation_author=L. Zhang;citation_title=Genomes of 13 domesticated and wild rice relatives highlight genetic conservation, turnover and innovation across the genus Oryza.;citation_pages=285-296;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0040-0" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Suyama;citation_author=D. Torrents;citation_author=P. Bork;citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments.;citation_pages=W609-W612;citation_volume=34;citation_year=2006;citation_issue=Web Server issue;citation_pmid=16845082;citation_doi=10.1093/nar/gkl315" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Tautz;citation_author=T. Domazet-Lošo;citation_title=The evolutionary origin of orphan genes.;citation_pages=692-702;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21878963;citation_doi=10.1038/nrg3053" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdóttir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Vakirlis;citation_author=A. S. Hebert;citation_author=D. A. Opulente;citation_author=G. Achaz;citation_author=C. T. Hittinger;citation_title=A Molecular Portrait of De Novo Genes in Yeasts.;citation_pages=631-645;citation_volume=35;citation_year=2018;citation_doi=10.1093/molbev/msx315" />
<meta name="citation_reference" content="citation_author=J. L. Villanueva-Canas;citation_author=J. Ruiz-Orera;citation_author=M. Isabel Agea;citation_author=M. Gallo;citation_author=D. Andreu;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. S. Werner;citation_author=B. Sieriebriennikov;citation_author=N. Prabh;citation_author=T. Loschko;citation_author=C. Lanz;citation_title=Young genes have distinct gene structure, epigenetic profiles, and transcriptional regulation.;citation_volume=28;citation_year=2018;citation_pmid=30232198;citation_doi=10.1101/gr.234872.118" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=L. Wissler;citation_author=J. Gadau;citation_author=D. F. Simola;citation_author=M. Helmkampf;citation_author=E. Bornberg-Bauer;citation_title=Mechanisms and dynamics of orphan gene emergence in insect genomes.;citation_volume=5;citation_year=2013;citation_pmid=23348040;citation_doi=10.1093/gbe/evt009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D.-D. Wu;citation_author=D. M. Irwin;citation_author=Y.-P. Zhang;citation_title=De novo origin of human protein-coding genes.;citation_pages=e1002379-e1002379;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22102831;citation_doi=10.1371/journal.pgen.1002379" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Xie;citation_author=Y. E. Zhang;citation_author=J.-Y. Chen;citation_author=C.-J. Liu;citation_author=W.-Z. Zhou;citation_title=Hominoid-specific de novo protein-coding genes originating from long non-coding RNAs.;citation_pages=e1002942-e1002942;citation_volume=8;citation_year=2012;citation_issue=9;citation_pmid=23028352;citation_doi=10.1371/journal.pgen.1002942" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Sci. China Life Sci.;citation_author=W. Zhang;citation_author=Y. Gao;citation_author=M. Long;citation_author=B. Shen;citation_title=Origination and evolution of orphan genes and de novo genes in the genome of Caenorhabditis elegans.;citation_pages=579-593;citation_volume=62;citation_year=2019;citation_doi=10.1007/s11427-019-9482-0" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="De Novo, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in Pristionchus Nematodes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="Homology is a fundamental concept in comparative biology. It is extensively used at the sequence level to make phylogenetic hypotheses and functional inferences. Nonetheless, the majority of eukaryotic genomes contain large numbers of orphan genes lacking homologs in other taxa. Generally, the fraction of orphan genes is higher in genomically undersampled clades, and in the absence of closely related genomes any hypothesis about their origin and evolution remains untestable. Previously, we sequenced ten genomes with an underlying ladder-like phylogeny to establish a phylogenomic framework for studying genome evolution in diplogastrid nematodes. Here, we use this deeply sampled data set to understand the processes that generate orphan genes in our focal species Pristionchus pacificus . Based on phylostratigraphic analysis and additional bioinformatic filters, we obtained 29 high-confidence candidate genes for which mechanisms of orphan origin were proposed based on manual inspection. This revealed diverse mechanisms including annotation artifacts, chimeric origin, alternative reading frame usage, and gene splitting with subsequent gain of de novo exons. In addition, we present two cases of complete de novo origination from non-coding regions, which represents one of the first reports of de novo genes in nematodes. Thus, we conclude that de novo emergence, divergence, and mixed mechanisms contribute to novel gene formation in Pristionchus nematodes." />
<meta name="og-title" property="og:title" content="De Novo, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in Pristionchus Nematodes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2277" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Homology is a fundamental concept in comparative biology. It is extensively used at the sequence level to make phylogenetic hypotheses and functional inferences. Nonetheless, the majority of eukaryotic genomes contain large numbers of orphan genes lacking homologs in other taxa. Generally, the fraction of orphan genes is higher in genomically undersampled clades, and in the absence of closely related genomes any hypothesis about their origin and evolution remains untestable. Previously, we sequenced ten genomes with an underlying ladder-like phylogeny to establish a phylogenomic framework for studying genome evolution in diplogastrid nematodes. Here, we use this deeply sampled data set to understand the processes that generate orphan genes in our focal species Pristionchus pacificus . Based on phylostratigraphic analysis and additional bioinformatic filters, we obtained 29 high-confidence candidate genes for which mechanisms of orphan origin were proposed based on manual inspection. This revealed diverse mechanisms including annotation artifacts, chimeric origin, alternative reading frame usage, and gene splitting with subsequent gain of de novo exons. In addition, we present two cases of complete de novo origination from non-coding regions, which represents one of the first reports of de novo genes in nematodes. Thus, we conclude that de novo emergence, divergence, and mixed mechanisms contribute to novel gene formation in Pristionchus nematodes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>De Novo, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in Pristionchus Nematodes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NuFUSaE47La4W6cP4SPusFFb5EKjg6-cu_vpCXAAAdI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493459","apath":"\/ggg\/9\/7\/2277.atom","pisa":"ggg;9\/7\/2277","ac":{"\/ggg\/9\/7\/2277.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2277.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2277"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2277"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2277":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ObpJi09fcIPooBYxIuPiuw0WnR4HYYlas3DW1EyMQCRL\\\\\\\/UYws2QANH3Or6nwdn2cZZEgfVzMawgTsKTvRgTSb93az5GOeGy4hl3GlkpeYspBD169gsTIh+30WiPyVatyN7g4g7mSkXjEdQd8XhWz0+hFvx+vG2FKDfwYSUsk4smygbPBtGWemaT+wuEok2EFXWUUNqumNRyaaEiasFL0U4wFNm5oAvWBfLcWR7YKG7avSjQSbHrIIvi6PqN9NHM6xuj7XyhsJX7Wvw8mwniy8stISNsJyLBJD1sCRiVcME97DwAT2a2lmuc7XWWJmBLyI\\\\\\\/T2Tq3HLLn7jV51QIZpqNyPQ7LQRIZhczkzWpfq9g3GKOQfEjOAfsPCuVjpsGyRfnNOykaw5nR9tkKmNCzxyAZsHYWCN9\\\\\\\/t5eOeebx+0xIjv5aqe3XAmqL2ca0ck1\\\\\\\/Edh1ZBvPhXXRZU8qaffsDtHvwbN8MoNBEhdU2K150XzfXInMesejOoRp45rM3LA8wx9gVUK\\\\\\\/\\\\\\\/kYQima\\\\\\\/YKvjfdJy7EZw8u5hJnpnavzXGADNQ+xuFzKv0MXiRBNeNkXDeHIxJ7jD\\\\\\\/15OoAPhtf9W7vMiz+cASHqqCG4mKxHgw2dEojYPUNunUgup2xe2QpTPCjs\\\\\\\/RJ2xjtwFW+GUQ+VvrnT1nggEYWNvcg+0n7KycYUjChveqyyhwSFBNTEamQTqoJvXHhhwYQiEpf\\\\\\\/Z\\\\\\\/SSwNZcZN5ZsO7KylgUA4XfWGT9lQM87Y4mH\\\\\\\/hpH4brd3F4O+bkKODLCdPzfI+xbF4zYWvH5NiJXGSsBBg67zKbTSpe0eKvBDtlChO7O0xNHcavEef2PJJwCPYnZY9\\\\\\\/FNe5ESz6+1E3AYIaDcZT92DjXNGzNN4qIA34ByLw06x\\\\\\\/cBLFxuxy1UZWg3jJvYUVJaO3fzGxoT0JjY0uxKMxaU6kUxkoooyb12TlyJoT723+rii+GLflf9IhxAaMdH+3Qim5GXNsDF9rtEjDCK6RtmAvOSQG4rsmGFR614zeVaQGj0etj1RNxSqLs9+gTGHOwIsQgIIkDpkw22omrf3Hxyw8hrGOlf6WhcAh6vAoGlccaxn5Aow32rjGw0wHCkOLjaNgVXRev38NoJtwcg0fSEFUtNQxpsfCnAcssnHavHy5LwYtADScGlVQXdEOEqoY8RLvamKfVCRl1w+C22lKnjOsZ2cRTE6hUsEc9XppP0OdwLOH00RtrkooM1\\\\\\\/ya10xbsZ7fq3Qtzav3T9E\\\\\\\/8DBK3jRpQh\\\\\\\/BVwDLIWWrpIxOv4WgjY2dI\\\u0022,\\\u0022iv\\\u0022:\\\u0022DoqQ+CuzrfSpOeqHK96+lw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225a77be96a7f99c856f0e1669e39f2cd04128dcb5578b1ac81adb572df7f5704c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+EA4deM6WMyMQXmtwDki808CCBUdlBCNJnoZURSt+G8i9+nKXB7QM6ZqRup5ZGwIq6ij\\\\\\\/CSyENcn0f2q0VGXMPBo8YsN6lvGE99Ghs3+2560qs3xogwJ2\\\\\\\/xqzzpMTVntX9Q8IwrrV81cxi3gUEduPdpOHUekKpPeo+AtaVYS+Ey0UedpI+tQs7Gd9R0pqir3DQwOL4mSSRs1hGQ6zTNxqIhs\\\\\\\/b8JFsJPRVHB7D\\\\\\\/+EA8Xfb\\\\\\\/Gjdeo9i9vI3w56gwa450yqQNCOPIZN5itBNER1qRCC\\\\\\\/xS1laqfmhtlRTH4UPu7OBPKHD4bHchEBRPjKy+XkiWyl\\\\\\\/8AxLKgMiF2jUblbAyPUxJeKtfDxj34gxEkDCjaue\\\\\\\/DVKzxobVEXmmnpNSWfFvfbwEIaA\\\\\\\/AjLB3H+xChu6WCtCGkHlOg3ru88R1uKYwAnP3kspP1GmntIyhjvSndADHq+niGzMyVcKkou8XFboy1\\\\\\\/GMGxwUClpZW\\\\\\\/iADtA5Px0e6tWGnUD3sNkuAscJ6+cQf3I7EwgQJ\\\\\\\/PJO840xZY8wLmrFUyV5DiHVQxdJ18FpkXn3JN834j7xZ5V8pEFBOA0FfDYlchwQSPA0GZ1uDjrm4W0Hq2dr+H\\\\\\\/kPTrSV65C7n9BT7Yx1LQHPD6llcRMY9SxVPEsqD1EajrXgIR46AFrpABwQVUyd1yAEFjSk6EWbXwdHLurFB2jwZE3YsTERWJuf7yv8es6o5q6qQEq+SGvfIdnDurishxsy2n68IgzjQUTO6CKQzsgvlDvwT3kQPqJtirn0kX6NwTCIyO3Zdr41RudtBcbXlBE0fkGPMmmRCnApKkmw8o4llC+H7ldpD698zXQ\\\\\\\/ksyCn4k7L\\\\\\\/fS3m10fo7Dczu2X1A7Eo4Y1owIQUFzIlyc5QiGiNhjEwpv+rmoNjpn4JfsGlMHEDedWt\\\\\\\/SqxusfCpDjXHKBFdByMc2Tdf2SR8YZ\\\\\\\/GrhLZlNc\\\\\\\/1YfdoodLesM9Y\\\\\\\/K3q1roy91h1vf9aYEOcKWncZ+Q7FKElahlflKk\\\\\\\/sYkMrKhexwUL3T1VV32W1lmdDdgSPQAuSn6IhxTMxOnePxbh7Qs\\\\\\\/ys\\\\\\\/S9V4uKMV1V1ek10gh4MMk1kEk6sYZ3d9TXxCp8dp0VoWpX2iY0GSOm+cXZKAzS7lbav5OYyYCEAt1PI+sDI1RlXjjvl6y3wQETi8U+nNEgtMuWhPHF7sQwJGm7b6NUyOtOxY+RmcfRrrALuZT48sZ7Rqe7Wp+px1HyiqSsz\\\\\\\/SHkUkcbFBH5wekQnFUOIMHBrjqgEhsREOVw1CjsMQT68xd\\\\\\\/eSyZmhsMUOUltlB79FNurbIAmMxQE0FzQRGM6tsvWv57ZrmX6pv44GEhkAi1FO\\\\\\\/0Xnzmz4g4s3fE7YFfoSlEh+LD8fJW+vhn5EfMpcsCO5jk5I5wm2YRK5\\\\\\\/wEFGym\\\\\\\/A+gV\\\\\\\/+ANVM3ZLwEwNQztGpXxYKP3YA6gc8AW8iGDSxaIutW44\\\\\\\/wiRtWtxt3pfMSo2HjqaOe5kdJi8rqYIXUdafJb5hmpEQIeQMT+fI4dVu\\\\\\\/8M0yvY6XPpTa28zLM55B5D+\\\\\\\/Ku28f2fCwjTjX3QHTXM5JHFDJhpDhXEPAGyLsMKXcCjxQGAQJpKipmMShQ2f4QRnexE2TVatGxMt15r4s1KuiYO+zADa98U4nKJM+33SQaf0o2Wth7ENGJiNpNqab4TyuAxVrmvIzqnST1+vjTUwYlvC1n7xOfCouBUH4xrKgcI2FcgcZEo3+d\\\\\\\/RP0iwGDyhnptZy\\\\\\\/TsGQmmaUT124RIy9UO0ZmvCm8ZX6QiHL8Xj4Y3DsoZe1Hrib\\\\\\\/Lzjw7cTFz0Tz1YzDHOaq+UOCw\\\\\\\/76SFMQQDQKGjXNBVV8mBjcPuWYFcK862Ex+DbZe2KHfZZ8eD+6gLmhZspM7yy49QCpnnPYIRribCfYpRloI6ZwOOYlPHc35Vh+JbHiup2Km2Jv1JZ8w\\\\\\\/l3VqHNAvQsjSI5BiifNn+YR5gdZy2V3TWJbXwYXbVYnnrjWV6nEZDObdGTfes7Zcnb0kKGEtqnk3cjU\\\\\\\/xfTUBC0AEFz0jbW24qhhVZ5p9rCWnxkoZf1bcb64A9XVog\\\\\\\/8pRKv2xmoMAtzz3A5ciZkosjW8IBZllwY37OIKbBxh1iMz8vWVFqet8bZOlf\\\\\\\/QvaRrrvwre+nf5eocQo6YBWxR33a9VFTqr2KcjHxi4BLiOFgg5QBrhi8LbiJqSOdSmeURb0XdLCyLml\\\\\\\/5YP27Cs3Sn2e4fo\\\\\\\/oznXLXgPmE7YFA9I\\\\\\\/UZZAhRrO2HO63zbGU33HSAfMjhEW36i0awdGX4rHtvEbM0o1Z3mpd7Z0rK1QOjM6L24sea033PJVYaxkHqkG2N1nrh8RQS80ZEhVWD0BAxU2iFNCWcOzfDq+dhCvvMmKs8YCoevTjfO07sljCzqa+ive5ttaEDlYPid3E\\\\\\\/oKbIhwRwYKKogq+M4eQNSrXvysufXI8DFf9+1a+3WySio7Y8HS1oBZGe2NPD\\\\\\\/ClxZ+IlkvpTK06sLm2eQ4kyOOX9s13i+60MKfIlXr\\\\\\\/uARCbZGNiGK6eUmNpLniVxuxdTGMJNkUvdzOm8VzRkxH4SuPHxaxH+FGDYYDKotdk2LOo3O5AMBqgjJ+VLnGLX+YN4VO\\\\\\\/Jc3oFackYOp5X9mnupYWyh4TBQPrEESgv+H0vH4WdI3alp8\\\\\\\/hVAlAjDzMUWZUPTDcc2EhGYO20wJpMfHZ\\\\\\\/7hlTbklCz7AM\\\\\\\/smaEfIZe+Xn1o9T5UfqphrhzqRRugYYpOatw\\\\\\\/RHl7rTO2xBtkTKfxlyCEDPsD7vILgmemNJ1ZkG1tKVLr04Cx9XSUoSTxWYOPNreBwMaV7WPH9e6b8+BPOUpAlIosVkIx2jN5oo9aigKmFW2QKPHqnw20WJ7m6HFevlFYBVZv6oFR3RXgR1buaUVJ0ClYfsS2ewfd5grMcTSDQXtq+JH6KQw9zTCyTLWANaL+YXT3wB+WZpGRKHPPdq8jUcUXKLZzH2gffQfCoLpaSqec73QZrje88n75Vq8t4v1NVqEVqLg8DLJWRIGIYzLlQomqjvmOSjTRlZwcG2zCZtffC8iBhNphICX2MKGAqAr3GFBDWn6mKitEc5DeFcj\\\\\\\/mIyI+0+DboLzZ3kKAb+lpU+HuPlMyvhWxVnQ8rrHovIDk6qTzwAZI\\\\\\\/uKwr9Ho4wNpnuIKl0k37isDKiKs1YwctY4MpCoBULAUMyK1JA3HQ0Dl0arapv2RyMVfCMayDTkL\\\\\\\/fXOPHP7eCNVNzNI7AbMwuLAbqeO++NbB9ElEA5QyCKneT8p6c9Dwzscgms4ufb6o+FTXZfCpOx4heRc5qDSC3tsBIrctLoSwwXYeqbKfu9tHOvcm4IMahIEQNAUkF5W++mLyVOTLryHNHs4W0IB7rdYFe7W4i9wFikWTEP98Jz\\\\\\\/qjaJbs8Nz+BICBJr1lv5eMyeRfUcTh32yAaJwz5Lhj7nxvthDvaGSmBdQaCaYCPk6i5Lv\\\\\\\/k16u7YVxOugYXp\\\\\\\/SaXFXQpQx0q1NpDPKaCkilQ7g2Q5O9YPoTZHTKzmaN74PDTWWK8kqmtPZnAc2I0HgimfFE1df2oy6c\\\\\\\/\\\\\\\/TvahszGpebMnGs5oGe87nw9rCJheZR4keo2f8YjeDBvFENh30s2usBFLB5EcdHLv69NO2Kl5E8pvgRF18WihkkHTzFSiVXT+16d6WHvJNo2CWoilmQq0FAl9PEpzrwRjIYzVnUdRwNx2Bx7oWi+k45+NxoNz1RzZYf92Q94arSeQyh\\\\\\\/nV+CwCOO4w\\\\\\\/LpXnZ3MUmdVhdB3J9fYK+AnizCYb8aJC0jotbHjqDUxzd55Lz7cGsVwlnHeyrKgGGLjVatr82+GvWIygVYx76BMjS17hHoy\\\\\\\/FzoFE+Awax25xBNBJmF7k9nPMTACINueGDABf6agbfr9HYEQK7kXWyB8MI\\\\\\\/zyUSz24Crcv3C6sp05Mk3xwy1q9IxW\\\\\\\/PIzdIGKjpbzUGh8kEW3wkRz3iiosDIC2qKeyQRRkOiFX5U8lHUs3d7Dqxm6Q7AlITQ\\\\\\\/2aH2n84EFcBBgH3b40A85Q4w9WzsK7iQDBQgFGFQoprJO8WA6z+CdUVfutLC8Gc9XSZ+cpfmGYkJRzBK32Y2fj5rBP4P3wys4LJCCDdea1aXyKqniYq8Rm2YghFG76iGIThRRCn6ToQSPKHvMiYG95GX838hUxVmwgY31bd38xGgzVyBRlrvhxwMAe6xo9TR9NfqLJMmeOj\\\\\\\/PGGiHxe4DXwoOWowCxl+k\\\\\\\/pJXiSE+TEXC17xVQQTE9Sbz7DMUf3tYsQUJKzOWxxRCWVAPm+4jbWZFyJoyRY88d1pgnooG37JlAKIkRkuQ0HKjoka\\\\\\\/psKH7+MBtHoU63Jompx+hqZV11wR8Ma\\\\\\\/RZTsRTsvOdZwckkkxUjBPD84l20jt+B38hqiOK75T5+RTrw2+zJMbpAnf\\\\\\\/38ffUiPFqID2P\\\\\\\/SpA41cvCBzws4epgU1pNkPb43NMqMWR5g\\\\\\\/xpdm9IWicdh60IE9euQbufJBHYm2YSKNEBL6Ut\\\\\\\/d3EK75nFEhxn7sEkks8LryzsBBIV05AjFUsPqSDBQCEWa\\\\\\\/Zv1JT\\\\\\\/3QZjWtl6dOoLBUL7oXWuGIqTKHdA3mJ1+6BtZ0+Df1YQImHRmUyprYKrBfwUy2VAkxe+x4Fx4hyzv\\\\\\\/fAsuDHeuJMjO0rm\\\\\\\/DXjmD4Kd+QR5JBmsSSM6q+jW3vcPs5gxy+Jvk0I49v3vEkPBredJhjxVKkGYVzPxwcw1BXGPlFyZNoDM+WtiIzAmZogCJjrhyfyHnhkb9AJGkbJBtZu2f3GL\\\\\\\/X3zMbf9gWz9QkohaVui2JMpKpCSPy37ZYOnZ4K9se+H6vLx9l5SvPr3899To3XriBRnMQ\\\\\\\/SKcwL\\\\\\\/WPyAoEehu0TZDmDBeUWLqkN3lnUuiDd5MZIxGBWDMwstZ016++2jlyvESvpS4oczLwfOishH6v7W5b83aQLJORcUEJFkVtJkXh4j84qNdSrXcXVnPPH0t9xowYWO\\\\\\\/RmyT5ZOrfh7miIDZ5MRuEZ6kMZ8uZcvwGtMXQeg9ASgIaIKUyqDbvssatZfT4b9SmdBj3TUPLQanu+j7mcsvEEFWUcJWKhOzS+Ub\\\\\\\/NLAtUuEu7CQwm4gaTIW8LFgdV7rPzYBDltejm0pUBBFm3B+GcDaiyiBpyb\\\\\\\/ynGopE7E3mOdgcTxotxOgma\\\\\\\/DOrN0i3PIaqppzjW+RFKbBwwHy1ZXWDy7ZGkvBTcrUEce+xLwe6vxxP8BsH9VFnk0uj7HOjc\\\\\\\/xlATk\\\\\\\/iSqbJbdNJcDQTZ8T5teW48DcFrxwefQvqQeXLeJclp2UtsuLnMUmWSBSL4fa4L8oKaH63dwprjZDMpU+peRRE66T2YZRUK4\\\\\\\/CdjdXNOCeVpx5hrULI7nhfmvCdY2oobEU2yHdyFGG\\\\\\\/u5uwwX6BkrLx8TQ5ZKgnQRaFx4KvXnf4BbC+BuzSLFgNpS6rbg0vh+kRaJtUjXkNoUx3qX7ykCe+tSSkmsi9W5P+sRH6O+syp6fCPdPJRdSb4FZg2Xo2GSu5NZ\\\\\\\/c1zs8wBLKIqBJnj3h3HrfGYgqWx4qWKa73cvIDSa\\\\\\\/RbQGce35FhADVnqaY4gF6jd8FjFXnOV8sGVxm\\\\\\\/YJ+fNaB2ZRQBXfLmxybdTCL9Nbp\\\\\\\/oEherPtbczqzHFD3ORBWY8wd0OQP433kpjYec0QgrZPy5Uh22LqF5P3L6o\\\\\\\/coNX\\\\\\\/G2mjSZRK0I6Oed+osYOHWzSm5L7zuAiP0g4ZAyOyqeQgzcxapBoN9RwfgwrWOufPHKs3b6WR648yFkSF7weabHi2Scy1Xy890FbsK+L\\\\\\\/YTJpMW7YTD5m+WK2lCH8tosJDlR\\\\\\\/jT1KC2\\\\\\\/4XiB1pOxyuYG6IUH0iI2+rgnndRQQmvLZhOh5dKyDyRH76TWthYEt1\\\\\\\/R8C3m5LsIkl3DgaUs+LsTIHGnc1oFISZ0KLU9GfyMBSBo3iZ1xe7YivWGhAzz4otxUbL4CbQuynZaUVLg1U7jAl7AmMWH4lE4bNUguISrjC2XotnzahTIpgZWRiZp6sqPXcQxJGN8woBgjWLGtXeVHg8K\\\\\\\/v6fMW6T23BbTzmkajNXq5JC6YLyksm253AEFjvPT+HV6s82KXirGkLmTVV3c48ZfrJtuoxaBdKqL9RBdLQparkoz0dEJSv2u0Om5By2XrxMI\\\\\\\/TQvynkqVbjt\\\\\\\/PDWkdC6jT\\\\\\\/EEpIirn1Hr4ydAu3edUAJeaArSb\\\\\\\/znSugEr6TykDM+76kG7P+G8kYXEc6HCot+l0T3HFachjXg1fTDb1u3S\\\\\\\/KM5Au7bPsCWd46ecHx2LA1C6YCwgafPg89dPYVHyrdQFZ+QFwvUCs60OYWZ\\\\\\\/q64Xrs9SfpSbeuSOLzS2XaFT90dxeo8nbdU8XjUzHp+UBjaXe6zet7aXoGbRSAE9LIf9Yd4VTf0X7hqwE6VBWKlAociq8hafHyVoApF7MHNpNWLg7aPtbfDCKkmRRemWt9WZNWHE4bgj\\\\\\\/TlnkbEBFrrne98GABYFfzEKAd7kHxTyWvDCReii7CXvMXxGUGJ+ZXT25ctout\\\\\\\/+5Ip3\\\\\\\/oh\\\\\\\/aU95xuxU8bGxVmo45yV0hpr7H64L\\\\\\\/rxayHfMkTJtZQZx0tzdR5rs6jGGMYVMGUQmpHyIWzwIjxqvG\\\\\\\/iGM2iq2LqoFOryMnuS2FDatLwXlMzfapUFDb0\\\\\\\/fLeuFomWO3tcKhAFA\\\\\\\/W2nLcrlul7BEcIQjlw3d2T+xXtTeZvjnKk5kALuvzkq30N16Shf91fLFiHi9l\\\\\\\/bE0V+8x8de\\\\\\\/ByU4gDsGiE5pGQPqwZwvwZOp+yqMe5ZK0\\\\\\\/OZ7y91iiawmrHNgCEoV8gy\\\\\\\/WJE1riI1OoT1hXdV5Gl5DNd6i04Z9R0BlqpaFJrcBTDmO6sfdn1wlJ\\\\\\\/dB7nVG7MaxoNbw0tJbgZDKpYLvrxQj446miUZ6A2tClOMtyVCqKBj2ktSAwhtAbpfb+0k7d0JM9WEXMbtG0H1EZuAh10OwbjFVsMcF6nBDIHzAdLdUj9N+cCYc46\\\\\\\/h8YtlbwpRo0+ggdelOQvstTqU5kLJJsdf5B3\\\\\\\/lHE6FKIAjj\\\\\\\/MqckWKBsN1U1\\\\\\\/TMhf8QOcqymRQvjN39y9OUPAIicaUGyoJVMIWgCacB3x9XWgMuihMwwUS6vPCaT2zNZAZCgxKalkLogawO\\\\\\\/J8kECXuRUcGrIJP0bSxBbPK0oAn1\\\\\\\/8\\\\\\\/YTwiqHatTPxCcaIBhNqvfW3+XJL5klZPgBg0oKarDebANyqVRX+IxYiUiZ3Lk87RfArQLWhVz+nKpMBT8ZwS77y53xt2yZmaEavMcRGvjWDvr51jgSGT+8OftBfaYBxix34vdvtHkXH0Ludb2v3fmNHHQPoZMLJvsthvsEyvHJKqHDM0WJs1KbWKNB7jZO\\\\\\\/Xc0d6HsBit7j4KZkrj5e1StGU429kC8\\\\\\\/JF5M8sKmuM1K7+ibCbfJcMrEUEw8kI1lxSWYR6+zs5j32KqQ3dqa4YXViI74hGSC5GRmo8B8fI2BMC3EAQ6EFOcb\\\\\\\/J0uRtsCPcvwTpY3NPXsHGY9tt0bz8A8bkjBe6x1afFrtBPd1LpiDwcf5e778A7E0hEGwllWp+vGz1U5HdQYQs2PtGagEBSi7qNPPnYbQ7+ZGEntUd50vqw4XNBMf8BF\\\\\\\/xq8GGavWoJIrijbKD65bL2Rm+kvH5BlQkw3oz9Pe0WNiVadEovWO4PhHA1jx6TliJ1I8mp9T3AFIDx35sxfKj+CTFN0fxUS9ps6QYgWgg+S+ZaDxmdpnJp3EAGSTa+uEZJhFfI62Ut6d0tOgl7hfwfGPTfwnseVJE5ZTOSZrOGPGTBct\\\\\\\/af5teHuwZvyzBhyd4\\\\\\\/9Tnwt2PVSCKgd2mLN3\\\\\\\/33hVEhtKuiXsK8cgKLgEjLKJWax47FhtP9Vx6kY2LT6Oohsb9E7P2ETWM3+xKolOgbzZTpwjawniYJ2k1VxVGWpntZJmU06vECZIuX3W2FP8BCjqV6+yoL\\\\\\\/04PgQTYI2Wb4dR5MXQ+\\\\\\\/dx6KrAQpg9NTduQi3XOiKVmsnkcIG60pqDfr6+E3x6YNA0z1lz5Nsu9xnNtZYh736BzbOUHHWeiIKMzocvXc5rsb5pj7z0+DbwNmNlnu+nA5GE+RA0gGreElWySJweIDBOFrdOhhtZnZnNKCfxv63btp9k9aypUHZaSGhLb3SxUGKVALlBTuvQk7V\\\\\\\/StSi4tE0LTHdzwXiT3Kevc2nt54lDoZxWgoRlXKKFKJ+ddMkuDWydJWb22XPxURULxevTIbV\\\\\\\/bUewsFzEbzKQa7OkySVdcuKiplnOqYCaO5bG6vKjoSCkI3aH13BGckO0rxFHkMznEI8jXw1AxRN+QQ6lc8kS7DfUwZiZ3ynnDzluyjwRdZ5y8GW\\\\\\\/ezCzC2IteTwdxpUTSCZNLc35hzbzaEaNRIxBDLPktU8tigu7mWhVU1T3pXlFZPX3YaNxdIkVLZZvRn+SiZQC\\\\\\\/84eAorqNj43HPFQO92hZyRkKB2Y+atu4s4EbQlkDRwOxTNO7jtaX1BP0XyTKkCB5wicUbqXdLKKCxIOUvPUx4Axme9EgpFOvOuWB5pVhoPLSiDBWr6BahVP0nBD6SmcNY9J8bmIzmIK4LS5VgJckK0uZZv6lIZQcEKTMrIC5vwr8PjVpTz8W0zuYCgc\\\\\\\/HJkiOwXcWK2sPApy1vNbLTxR4P5ez2PYo5hjwrTOq10\\\\\\\/qUwevk53Mf4WyK0IuaKdMSgL27ojTXmNmRvKwbBXJZ6IokYBNi2pOV4XvIhwNXSn8wSueqQj7OvOeTCmSe4NIYACVzFRJCuRMsxmH77Fbxgg7d2pHhPy3B9xZWNY8LTlnAy4seZVUPkUgJVZ09m2MKRYni\\\\\\\/kf43eI3TdCaCkmL3P9Pdnox2EFwA74CLtYUj7hwApi2TxyY\\\\\\\/F5SkhVFpRz0EbElE0mcp\\\\\\\/IN77mAG\\\\\\\/inZy1JZgNAaqyQe2nPRv1ZbySUd5OT1AuqcHkjG5g5RFtkijumMBERURJACBvg3lPXfuHK7DB9WLXZB933PiETf1jmDUlnXklgK04y2+8I5cvOQlrqy62PlDQMk5SWlUYu335QH1bgbkZBk2XbeVOwQ32xjGp2g5\\\\\\\/P7cFuxvNRGj5QWcHavJcH795vhDA48nsBVJ7FvOqKJDUxECA80sCiNsGg1035OXufCGAVyKCXU25ayMVtt1bSKVuv9wI4uC+7D8g+WXWs1lSwTqviJmk\\\\\\\/24CwuFeJernTUYnxyg5kT7uXXYQguxi0vKSDMK9zSn4\\\\\\\/rqS0smp0m1h9KuiP7a\\\\\\\/6ahI8cFJLtdYp2LzEFj0gmAon+9bwzdM5WzcRNsKOpFXYrqb8kUEW75WAW2TdN6wL9RR6P5bcasYaQG+KjlJig43x5eF3kxaJBfYI97qTSVRilWqNOb4XHsWgrobV4pfPAejE2BVWGJ9IoLZD0uBo7VxIvnBKe4KPKj8StELpEaDWWfJzgIOQG6opfNycOLmdKBWroZYs9hmPuNw3USEchdlkdExl7jRN4IOways4hJ99TxYtsgbGAS3z4ZrlgSHWj6isDJiWEIkkGMYsegGYPqZRyG7LPKes5qKx1ZgbpBmv0Mujk68Ju75MCxr5fOiEvVnFwhRcwJR6C6NwjvC3BS5mEPfmkwW4qPtFr6ZU+NDBRPHilbI2cdoX9IVVQltcaJOi+FotICzaGXqpq2oIE2UJBiuNC32dlbFtaPY7VyQ6aJYLXAPB61gegtg0VHh\\\\\\\/CH8wY66hXkdIS115PxTCgf4wq7MxDek3XNYjfCxYdiOKC2f+x3jn4xoD4Mjs0rqsUS6RTeMUqXJR9GPW1tXDLrTfVpSjvtm\\\\\\\/122iFQUaWmrlaADoQFNI3If7\\\\\\\/xd5WTTNm7gSg1iIh+3L9mCY\\\\\\\/C87YPjVtNq5pglQzHxOittA5sti8Kmgn+HgxL6HLoYkVgk1JSw864wsA7Wq+HlLpgu7gXP4BMr4vKFNqCq0RIakfDXWUJDwjMGz6k5CxK38XLixiaK0+YSL5LwJ6UoAIzPpUQ1zkYQnQDKPiDkwClGMvow9xGEotvPnDI29ErzRBIkBiEYL+AwT1sZ7xxqvYJy2I\\\\\\\/ORGTE3KUt5BrD0tT0h5PTp6u66kI9b1u81Wm7\\\\\\\/ayF6k3lGark1sb2OWdLMG9SIcQ02sG5hB+Ez0q51Qbx0o7aJJCQJUOe2zN1L\\\\\\\/iiaROLFekT1\\\\\\\/6QMJP40olYjhIwAz5a+WKvxeivZQNPQ\\\\\\\/Fccpj2tmLLrQQTQf033VGaDANsh3xlJ4nMmzWAOljQnwpng9mWP6cCiqEpZUyt0WEaOtXhw719\\\\\\\/2A6kSpKD4bEUew2Y4Tt5T9rp\\\\\\\/+ZmgV9ymh9NcEp9drWD3zjp8xLfR9MalZi9b\\\\\\\/ohfzLnqKPhB1HaHEMlqZyrAtXVKZcukC3RxD1t6ZrIRJiZrQ0uBOTEwfw5Uqz2p8pCWRK1PsFRhnmYg8n0xfSkAzH48TmbjIukCDjaBqKrk\\\\\\\/9nWYE3OQzTC0F\\\\\\\/5VnSmSeIeMdsKHEKoLsxldqd+iuKm3xpsHnWoPu94IVm1BATu8fwklJgQYNvQQKU8PoV0Ttq+Q46aqtwz3ssJqrLsOO3tdXTzVIE1rj\\\\\\\/BC2Z7CPT+KjIh5mT2A9\\\\\\\/+Af25ColLotJOn1sPkY+Jm7BQxy3QwZep7BSl4UhJb1Sdbgj1hnyrgNcdajgq1AsqfiSOC5X\\\\\\\/gTyQHELpsk874rYZrT6lems4PONEH9BhfJ\\\\\\\/tz\\\\\\\/T+gFo68KZhsn+LXVPyeoMHbXg4Zx\\\\\\\/fnZSPrGsbe89ntAp11wVYYdKOuaiNfj6Cz+gttjAuyPq3v3sS6ayurIVE3A\\\\\\\/TqZUjas5JEAhFiOqjD5qGsl992JLp+7UDPGIEP1nVQuD+uNlkE\\\\\\\/zRn+O3ic88MW4kTnDX3zbUQ6MH\\\\\\\/+7J8zlt9hhi1zB8QwR74XeCihHMGL8aEwv3SUd0fFndB1lOloYehvflzba9+9Ec2Xu4ZrNHlV2aUpV3RkRPs8YwpeMhy9oArvEZFx8b1\\\\\\\/hY4ZTn+fBLbcedVR7SSH78xSGJe\\\\\\\/oEc6mkQqF5M3VHXZ4slk1UtUlZNLVXDWEFQz9M\\\\\\\/iCJwGOkdFAdUOJLdsTdth15U9rMmgYU9R8Da0ZsRbXXMqQgS0JyApobpSKbptS0CaHyxtvqS7n5gJNw8xXlfr2uzrvidtSoSIk4QbrGQkuLcewDywM+XTdK+knf94gmzclF32UE7PVc5hx1Z5A36abjWvL3IOdQ76FbGun0xDXUZ+bUZ6rpFz+7dqsBz02SX4SnjThvYJY9FPnMQE5fCTqR+haUN6fKipjFvQDOi135eJaFYKwa0896e6pE6cW2I\\\\\\\/pZsF54oFPeqCDnW9+xObmfNiK7m\\\\\\\/Fdrv5+fjIc430WH\\\\\\\/3pZy9S+vZBarkyjPkASElcZB5XaJpBsNRRXAjnWTQoGORPn6W1oXbGotKeZtvuFEU5RoPuwWH8ux0VfhWZMPu82uUuv8OWVm3g+ZoOUlSL3d9l42bchYCQgqrq\\\\\\\/DCtOVxKtcjFV8m\\\\\\\/LOM26kqNgSZ47LA8Hhqbv\\\\\\\/bSUrBlLaZDIwlKs1O7cyscfMAIa+J5N5ct5H5H7ts7USs21O0RSz6rcW\\\\\\\/wvA5YPSn\\\\\\\/yMOOOkgm3tFZN8t4CLY5GTiF78g+4eYzoUlFo\\\\\\\/EDgzm3K9tXdPCONQeicXYxSjWWQipkkdECrUu4wc12bdN5+U5iHBdLo5ho4gOa22YzOvoRusqhMZiz5mJSKX\\\\\\\/Qfbz0TqTS2O5ZDqpFDEDLQ0DXlTDObXwAirkHyj7Pu1KN2\\\\\\\/O0ajRj\\\\\\\/BEs4VE+UMu9UgELmoJ5pelKniSNfzK+hakdBjYuh7e6R\\\\\\\/nKAUjIxUbH+x4J575zWbArO\\\\\\\/rtthxi7blg3UAjiDgvqU72Pe1HVXor5STomFNLy62g2cHscY6YsDUOLbnap8OCnHGH6zwGs9U\\\\\\\/7hm4K2X6kYp9U6BRKbqrwTwUQXGYBv9V7E\\\\\\\/7s7gt+7B59xmVnh7xqVcvZ5ILh+eglzKJ7W6W\\\\\\\/ZC6\\\\\\\/iwWyNabDhofe7idxLfZJjiWiHtR6+NuKOghxWXHFZWtwDkSeeyJgOYnapMbTpbAtlDRBF6NC\\\\\\\/YZ8d5oX4Yy3zv8n8zuOXK\\\\\\\/q8HvMZA\\\\\\\/z\\\\\\\/mitvhY+j7o1jwUQ9n13fxNxC3oAou\\\\\\\/7VzOowp3J++tFFRvC+WxQme9sUdwOnnFjoJdwhpCKazVxtwLUH1jSUz5JGlyOynWXKG14fNex6oRDSQheYyDxs2PDt76Eze3rm7FxDDt5EpOwFesmGnNwmZr\\\\\\\/E9ycrob9HSWxA1dZG\\\\\\\/MNxgn0eYJeFYeukq9GLeGp0fNBM6Rd5sGk7gmcwW1A1QNd7ZtNSQKQZuzdrZLGNrqWJo5FTmkS2NhKcBTiy6rGrIzzmBBTpC9T6X1dMv807BVTFGYFNa2p6AK2m6P9N1SwY+kJD4lPwG4a+GgN7TO557Y1lfTaE5harUowbcixguqpj30S2pmHq9TTP5G8mryrCFtpPnRnOgpKQqTy6orFL5mnqOXd1Br+T2lz21DBB\\\\\\\/D2EFuBfvoLi+QRAkoagjN3jVTkZKEUluY619BnYo3f8LGA4mPkJUwxp7\\\\\\\/321IUvrMwHcp+NI5kru05BvX2RhmlXrkDr9B\\\\\\\/WvNLBj25bEeJPrZSFIFL8Zi9aAAbmVU6rfuUWuxIC27b7fJlriZYwLNL4wx\\\\\\\/2mDwgZWLvgd5tLqyUo7LR73uFXzBV99XMAqTzR6kGM5VFbQas8CZTWkMvYZ7PCCRDOscGZkZc1CtN8x5T5R5S1z0zUI4FANwhpvSsYMUXY+prttYc4SpH6XX+eXOzuIIO0o5gspkovNxiSPOLaN5BAv0VXzN5lSsqCez9g8nlbKEpyhMmk92h1aJ9nqa38sMZAPF0JjUDjm\\\\\\\/5URw3hMWRzpP87exXuNNa+\\\\\\\/L5XYw2u3Xdla+6nMFjeecgRC6I\\\\\\\/tCdmRUktbT97CDXFffcMDCfnT4vWUIYzbDQNtUcJR25NN5\\\\\\\/fSZ4xX4I4G6yNRrXz5\\\\\\\/sxMQfM4HCk2zhHgcW2nrFtppmIWxpXq0E\\\\\\\/jv32DjbWfTQ5l1YY8k3Vjg\\\\\\\/nUWFBpIjPWMfakq+jBpKnoAMT0pzPseh+mNGtP1HM+Jkl9lNZVgs4g2pM5IWqij3qYeAabORPv6w+3S\\\\\\\/hF3nFTC+vIV8vLqmGTTL5PJ8gYF0Md3yV\\\\\\\/4qim6cisdxQaffIbrQ67of7Ekf5zeCMYXW+xrwWwiJzcz+mpACYLLFnuazKHZHixdZJbRALWZhG8XIRLQ+fKhkX88nCgjgpNzgUQvz\\\\\\\/LG8VxPLX3rzj+7abTp+VwC9wI\\\\\\\/dpWEOpwCfGgDgpTw9+5WSYj1yCXJ2\\\\\\\/QALii9M48UPk4rce0gtwRLhz8WFgp6Lo4NjScEjOzHsr4+Q41APWTbmxTv1iw64EpW3YyaObwRenoSzBZvj9+B9yQGhlxVgdRSv\\\\\\\/cRmN7nsVHdD+bW7NM6FPv+eqCK9o2fDHejVwyTWOjcgumoP+0ueHIp\\\\\\\/oARHgnkW2S4Sso\\\\\\\/TmKEy0vAhFj3gFgCzalKdx3hLf2YhJvQNxV1XBQe1vci0\\\\\\\/aZTMUzFHx1CeapQtYgIx8Gz\\\\\\\/+ht3Fu7PjGX7fHsTL18Rkl4Ky\\\\\\\/rP1o2Xtc1H7Gi2QTyJP5Bet0\\\\\\\/l5x6cpvWi7uOPme\\\\\\\/w43xWyYobrBOkujvgQ8yReu3sORQUb4pnXQFDQBjjXfmCk+WKTix6ll+d+lLFFOTQ9e2KQ7xJ6pL1juua2bcBVrauekRUhU4+966iQMidKHX2LqVvI3P2AZ6OhjLhsz3g1Wt2qevwUtfQF+cv7URH4W2RCg+fQigL4pEBjEtYfHMoNzG\\\\\\\/xyU06Lxy9yjj8i7GrwR1gDTPWms7EqRniW91on\\\\\\\/rYptdrW9rMJmfEkSPzTchnqbxfBomaGZK37IA8CmaZSMwNwzSCWRxD2lM\\\\\\\/jqyPPgwE6XTi5jleqWFy0II7m9\\\\\\\/w+gWO3r5U449SCHIsmHfo2Liwjpq9mQrEWoF3Oll18rePZTMcvltEnOQ7vPRFbox86rja3CFGsK5xJyDxtrSx23LQscbCtRmwfXz8sQxzZUGLAhq3ndPmehUv34PPoVWcWQAdCnpD3h9VgV3b9\\\\\\\/nB+o1KkTwZuU\\\\\\\/ZkQ14ek9WLzaqB8LxsT\\\\\\\/a1FXXNMXnvb2slKoFyxOIUjYUSkb9kAHS5ZWq9cMW7nqdOBESpE0wDcOXEfYkTzJPZf1QJzYBKTFTEvQLpISDVDz4zb6XVnAzBbv0NUckrEUMcP7jqEfnAEnfpwRdf\\\\\\\/nn0mumXMyrD\\\\\\\/3X2UekWWsz4nbxFmO+maY93X3z10UnaxuwYI7WjVY9HY1YQZIlR9VuJMgt0GJV7Nw+R4Q+fRAhREV9CXfaVs4tyW01BSm4Aw245rD11QD7ItmG7ggHdZjnNykmdZulwwZIg4gwB5ayR5\\\\\\\/XopAiipwpZPfNpo6fmm6mRbCERw9s5X6vd\\\\\\\/M2oqI33Mlu9jDSvwW3baVK8jrK\\\\\\\/ife\\\\\\\/a5fjDdUw+Ase65w7I6TebsMTpw8lSlYgPo00MFwEwdHt20NurcHvvT4ZImBUVWt2RSRoB5zQZbQGC32KuDPnkJfvMcn8Wh44jh+CsL4IJLJ3Wmf53laqdwn+R9OMuTxT+bxw3nz5P+ls+kPo\\\\\\\/SsHOlDamMQdIg6vfYi9L36QjerOklO8W+zL7FJAWAdeukOvOF7G5EruR8C4rw6goh9GiLZ\\\\\\\/aXsqlQCDFN7uwteqgTmyLsFdgCNEhZHM4XjLsSeHYIPR2vDO1SsfNRfPGFH9+CiHmafkxI1wI2vfyBxDKb6OFA+yOYUNLvnSsiSd6OwJdG+rxrJ0WVrY1xjPKfWVJeWNNCwsyB5kQLx\\\\\\\/ZUaLlQ82K4mOdz58ipXD2Oy9ku\\\\\\\/bhWQMgeLYoqC075KP6VQWcdVb\\\\\\\/4ld149wPgV9IWssrRUYPlbSbsbwToOF9NyUP5MMyMF5xEnTNOzFn3\\\\\\\/jCEYV1l6Dx+70CGdp5vSnN+8ldNkeGqzCBbnnMqGQm9e9tr4C4M7YNIEelsA1T9BIY066GLx3KTaxpcyjSq1jHQY+AIBkTFLGBlDLetubVR17+aJXUsE4XFxAWUtMPyQlW9LHaukmHaHl75j5ya1r+cEDH7Jur\\\\\\\/p3lhPwQMdYQka2lLElgBXilnansLg2rv39twTcqc5dxlrOEtF57nYkhUfHSCLgvujLgkpz2jUzhSUnjWA\\\\\\\/XvFw\\\\\\\/5wyFDxckQKK3G\\\\\\\/RDNG2Aos5SUwTdzdLTidvRMYklV3cW2wIqNXMWinE+kU1rL2Fh6+q9xC4hJ+taXnsWafEfdU6cn1jjM8kKe3\\\\\\\/jnvtcvDMVoweuT\\\\\\\/KvC2bznhfhh0yqyIwNfrYGxUXo+wIxzbkeO88CdHoAtOptAey2tzKarDbJQYGWge7FUBxL2PVpbaRUGbKmg97MUbFAJ4uJ+2Jowww3JXHFJFicSDEeMVALKi6JfyqwGrQbrWEeTDNx7teVPbl3n2PpNVVe60t2moKVsaBQ5J+4Tp3lkBvv8bDs7vc52OXio0+JCLO2\\\\\\\/aWGf1wrDDCrKc6Ry+\\\\\\\/WuV7y0Z47Wfk7Zr9syhs+MszGPsNHre37JwZ5Xut2kRuqcWe2keZScCt8XqYSkXPVk5DqQMwZt6SVaJ8NsOZQstrd5PdjHIrhLItsrmDh6+SWkbCvazOzQ4hopgBNxWe07Ks7YawW6TGQfl5RBsC5bA22uhyXbGShJ1BBKZKMxOftNRmjPHDD4v15kJZnP0R\\\\\\\/91uVriQuYWAXrQR6\\\\\\\/h8ufDsUdzUkbJmPbm9g2h656sEaaX631H8ApY0W7EyKcBKy3r2qycmw0euTpNlZZ4cMLOxe8j90lBqqneM1a5aQ2swaMPEQKouLFB6CHXNhFZCPaZeU1ALWW1oEEp4kvLCOiIt4C0zw85Fj4JREKM1nvrdWX3LosK142i0klAAdLcSnuefnHFLcmIqnJmwCQZuV3b1QWv5BA2gGWOunqAU0IHTM8nCxS0PC1+3vD2mlOwkCFF2\\\\\\\/jvF8JDzbr501NR6V52Rl5a8eFGIcMp3KP5RKuMEdsPOd0kl7RgHZMtK1XFt+oH31vZxw5gICrsHCSObsGLz\\\\\\\/X\\\\\\\/k2EzP0Gpe9A3sULYD1RN+FpNmfRpR9IRahFArk+wvjip22DKenVqmtex9E4eDgb3tKQ2Tj02669pSBx94IOFALHgR9oUTWmFv1eii26Y4twU3PNCWYcEKdJQvcGj01F75ROnYl97Ptj4wCWOxnHU26AeOQHwWvSORF7VCTO9h1Qr\\\\\\\/jeuoH8y3WgtHz8OhKQHWHp6JJZ\\\\\\\/AUfMQR14ydwxqLF7jy42gRmBX93seqeMqTD3d1Li+RuaqRBJVx4tYsiuc6AqwgljhzV+gDvRzxCLlnKY14VHoFn+rrVoKapTwqBHHxPZRvT2Jn6qzycoRodBhkhApPP3nloAJR+6c07kdYP6nBCTRjan1kQ0+2uceZcnH0c5IU8yjVTU1KPGmL0Mpby4UXMeTE2X5Qkjeq\\\\\\\/xtHWbyXOqSC4Yrt+r26tjrgMhxeGc2Qm8RBZQz4E4MrjG5CqTVwTdyaJomz9yvHeInBuRSiZAq9vFHPSMr\\\\\\\/CuvLJ0sdiLQ5k9w3hv5vhM3eQNRc4A23Vo7yrOQ1YlocyVBbuFhJ3+aPpZmkN9tJNF3mRec\\\\\\\/H\\\\\\\/SviWisaQxXBEzrk\\\\\\\/\\\\\\\/2I0QmvTYsWGqNMCVPvnxYAi4U50TwvxJrPnc8ivcl\\\\\\\/ng2HGeVYraBa518nO1UtTphORVX7dmm\\\\\\\/46s5ZQDJQ3Z8V4Y2utRF0M4Jehu3eTjTK9rYG7PjYftUTyPliziEuwodjDGhxwmJ7dH9N9ZanVGWyK5b8XrdMI51CiqQ9ubT9jsHmHdctkDriwpu9n8sfZyaw1uLCaoQM+K\\\\\\\/Q+b0XamM+sOsCzMy2avrhKZV3CNMyfjBX24f+JLjnjLRbgdXQWamBN+GItWxnPEufFuXXAJexcWcrdHcYWck37SLjjJN9uP1g29GW1YZyYGYGNcHY7x7pp40XYNLi7mc\\\\\\\/T6uSi7V2hHFwHVocub9g1yq5y3Jnn2frzQEe2hS4mFvteQ9er4Yy5gnB13Ec1Igy9IEakjW0yqq+LOrfpCLahsUIqaTqjDa7rxeLiBUr5lnbFCaJyE7B4xSz9Z8Hb33w+ya5dyyrlpCVZDBRQbZqUABGoL\\\\\\\/7KwDWmZEkeFdCj1evQtmPupDkOE4oTaymRWv\\\\\\\/k6DWu77t29cqgj5RQrI\\\\\\\/wiPTm9btWs0ew4VcPPoFIuHtn6p2uiQ0nLqdQppjYJqFRyocAgxR7wBwoE9oqgFS8zUOMUvVkThFsCPHj85IP66\\\\\\\/yp\\\\\\\/y5akvgWhPqTA82p0L+\\\\\\\/xfFKaswZMMSC+toR5PQaBS6HfKYrs+4n\\\\\\\/wFZcHfZxHT5LCydRAkot5+vSzwbvlthrhWyRp3nD2WjhB1JEAB7fFxM8TsKrEI7M2gNc3M5tyaI5iJAOqP0\\\\\\\/3E84hQL6fZs9p04RrkbCgsaaZHE3bkYp575TLjpXOJw1sSJwZT9WQ9XeoWLKhx3ZRursrmVBli8f9no4LxC6qOusPPU7D43ypmLgoT+98zJ\\\\\\\/jfbt7CbHLdRb7bmSwXIM+2PutYQKTqVR17W37CGUDNMG3XcHuW93BtoK8465ojOXVNhWOB51on1B7sJ\\\\\\\/jdpdz\\\\\\\/z5c5jy6oxli+4Pmhog1ptxzgwiyDrBx9xMCO3PlSGn00mGMSvLA3IouOrrmLg65ahBpDJ9Tfsb2oRcz7cCXHmem8NyFqulf1PAiK7hb18e1U4fS9JQVz9e5TpsoBF82MK7amI\\\\\\\/jYG1+gcYsW+lPzBw1Pvm2lJ19DlvlhBd0JGRbPcsyJEAv6CwEwONbsPSBLFjO8KtQBdwNg\\\\\\\/9F05FtmNQXQQhpbUNHxMl723wH1mjHfsbOKkSydK1i74Hvk6\\\\\\\/UoaYMuiG48tiIryhT0oLyspnEZtF2nntTWX6\\\\\\\/d8ENQsry9mpig2MeVrW\\\\\\\/IXvipVgbLHba1tiepK0jkm6F0b6V+hB6qIzZBvgzIR0h8wE+lgVo\\\\\\\/XXKwB3OR\\\\\\\/kRmcyx+FSPGB23BbfaK0ZB58MJn4oXS4MUKxzqyIT8QNUtePePKtdMnHfBVM1NkGDktkV8tyx1NX72AG7phgEe2w+R03yeuI0olRZxequH1KZP1RZ51AEHc8MevnucadCEdcjWxyPYzI\\\\\\\/EsM4+S+3XfbgP4Efk6Y4MDrSU3G+eF7p9dyFhXyL5gqqrzfRjU0UrShbmg6AeaEhJ8XD4IIHwQuZTORm5bA1gDt5yCu5TbaMtVfqxM5zGvzO2YuyyGifa1yKsDfBSPD2EFI\\\\\\\/HZOcb4QipIbMBcLW7bByYFoa9zjtDirlV2ECZ\\\\\\\/GQ2kwz8XvN+aJcWti2JNDFLzFbf0TrWQaKrqFOH5+jVtV\\\\\\\/lIvqTRAoQxNyGyt7HmV\\\\\\\/KRb5YmFtfEkhmY719GXQT5yQREakwZGV77OSBYavlqZ8\\\\\\\/JTBYmHFX+C+iYw6Z0EAPowIbcJu9RDZtF91oaMerRTwyXK2jiY3gxbCy4xQyuzwoxsJOodm0s+1XhTsddRt8hgRnYrGfte7VULVio+nre+tCSAKYh6N58gPMTA3GH1TvT1a8YBkO0Tny06cZoYAXRtSpbz2m0POsNl8oMEZ9b0qP0PNnGg+sU7VeJLt504XYly21IaWc6kDUzu805PmO4+1IHK8Cu7Nwc2FdOnJSyX4IOOpGJCGinRz81OouRH2awFd8MyiqAdQWVzJLyO8188iLI2HdujC2\\\\\\\/SxIDHqHXZN3Y6vcoXRhSMwG\\\\\\\/dWqFA6sSPsx2bMoFEX4YS\\\\\\\/DmN9uEgP\\\\\\\/hCArXmVsM\\\\\\\/heGphHPq6+zpxuzDFk4PUxXL93UIuO+qEC6Ch9WDj+QVnQUwRoFVE+F37G\\\\\\\/o9kjjXydT+fFSghKcQRxuvzqNcMUL788gi6D0RYVvPHQLJemOGrsV0ryBxG0QY3+ugwQNlbg4Pmp2CNcDNXEH7fwO7q3LVlTEj2NymYN5Qj2s4h+LeZK8dImVmygHZSgu\\\\\\\/e6\\\\\\\/rRcWUVSv3YA\\\\\\\/zuxr1r\\\\\\\/gJYph1emiBz0HEw1s8oe6Clulo0Bx09BbyG7W2kXZJMMOZsrMOIqGZpHqu89WhgjAt\\\\\\\/GIR63X0E0z5rxBeQFiiPKHXr9AuvL4TnEWefkilSKY27Q0kW2d\\\\\\\/6qt+oGuvy15eZ8Mimash7RhzRl9uqTJnbFXoZuO+ZQDhITDSq+gjj8CcmNbKaNNGYrfHfdyNbq+AwnX8rEF1VeQrUKujivHPw1nsJHFan2mLf8ejdzQWqoLWEXilhfOwWWk0qEihi3PDWOChKL+\\\\\\\/x2m0PofD5cH8+KfdFxEiy0IIeOiBl1PjPSvsztaryyMO0YAZcGvtOx8jUy9JW0ka4W4XLld7+JpAlsjg6g0usXt+tMcepf5RF396YlAettFg2JzRyuGeRnb\\\\\\\/kMms2JLKwOCyLegHHI68XR9VsMmuHbZmNiQpxGOym4ouh9KTgfMTRkHt+5Ev6TMKRfLEz+WLmsQE2TOPqUpQ3sVWFtAkV+DOjARAW14PxSz8UEXxAPUcQpZ44PxzzTBzu0rb0fcmXEb6oRWvGnlZgIgL9dB2vFAgZ7G\\\\\\\/VInJiBomZiYp72awAT\\\\\\\/OD+QF3gn2VzzUdmt1m3llIe6wB2iE12bAehK7bVpGUX3pv13j6m4aN43SElGmmkVkVsG8\\\\\\\/dx2AviWkGK\\\\\\\/8RBMluyfEPVZ+8Uy6vNxg5KtRUyX49vmQMAEjhvahaU3xsTEfyGQTKB2lCOk\\\\\\\/AejRHwQf2vGGeSOVkT0aZtBqrT4hL0uS2k2tc+ZDa51Phhhu1fXXPX1Xd8AQi08\\\\\\\/qHSEo6d0gyNze5WhrOeFcr3ck1+8hq6U0cGPquUIE1WaRX6JvEO1QfCP8Mpv2FFeDtpeM5MmjRIeVaFTddBV8brR0jWwDHlzHAUeiM27QieDSIrvay+S2ZhdsIdYq5J8SRXRYV85cj8SJuh6A90FvKIbJ3C2enYXl9\\\\\\\/NkPE1K0+LTP2vJzvOIGTbJqWCQTdvAEnco\\\\\\\/qaEqIURU3xGe1TbLF0cOsGDnPSpTDHiqu7Z255lYHQOhqBxmySeDdQqzZ\\\\\\\/pdAEKhnBabI8\\\\\\\/CFtiNzwNTUoPkOeFea70HHbWA8+5zeq7ZhdW512y1EZcWIMLXrCqhDLuKrhtiHSk2QokxrbH9ZkYfmiwVXJRz5oVoNzFaSJB9YOLBBLiGIBZ60bUCWaI5HQ6bRxJn+D3uVlUU+SEm3GE\\\\\\\/TWkSO91rAZnoSWVsVw0HB55DSGZCeMLpx7Lq1rB\\\\\\\/U\\\\\\\/\\\\\\\/5w6War2COnBJ5zZAzKjWHXUzNVfcrSbg94rN5vxKU5R2UWdDWMXYwdqQUFOLL7tJ8AltbmpFaFPM1rMwXBue2hrOC1r\\\\\\\/Bt6PguDi7eQMZcdGFQq2UUUH5YKOI6L+tKsEGjBeeqsxXXv9SKSvIRvB16FP8JdrYZqaUaQER3O7pbKnwo6q8rEml+Eb9VZ8eaLrU+vp2DrS2OHGMNN9da6U1v7v5G7AVVma\\\\\\\/T6Xkw1i9OkBwSSNnsyi7Rk\\\\\\\/1FDTRXzhNlLYKwR9hT1mA9gafj0X\\\\\\\/usMZQfAsBUDo0LRYkk7CppHW+fQNdhEqWusH4eAi2UpEtuDUnfDaUmTJVoQALhebGGzOxczef92ws9DPWQOKJcHoia3c+HWN7IpILVFEIgm3EXelakB17ngKQ73NCZ84PXu9yxzcPelpSRBmILkz3KGglWISOqojZ5w6Q8ogDvP1vG3ajFdm1CjvXiH66+CkEDdh+EzvZgdco+1ZLTrg6rkVJcVSeNqkjJeUkT\\\\\\\/jDdpuTVtjETzG2bxsLMYCudHzbotKW1jNNXTeN7WfX1Usdha0duuHubaUaIDRZ8vTJ8mF5UdiylTRgYeOOnLXx+QQhuZGs\\\\\\\/GSaGhcXljxkcE2BwjGcg+XfPlzTNuKRMUygKHFB\\\\\\\/YUWcPQtXfiAK\\\\\\\/3Ijh3c0kLeVHSTsc\\\\\\\/W+VveZYTXG8FHSrJj7wesk0anmtgOIq3WezhTbMKYS5tu7TtIO3qgs7T1TFMqk4xfF8FWRQiBR49A\\\\\\\/ksrrJi7e\\\\\\\/N5NZ9LyMeMEixy794HneEYaN696XkjuSPUhslP6\\\\\\\/3SDI3iAjKKC+qZISOMYqyNtS2qeQJM6L+2vptMXJe1kDApSE\\\\\\\/Le7kUSxhYa8Zmk+7nfvwifNeuNlZ8oYCQeECnazkMzSxFMlOIYGRZdZCddexUtoSdn+rmLiAUvKqjYYRbCrFQm8MDUz5Atn6ZJsCr1JcsE8oLBFnQssGPQ6eAb6Gb5LeLpTKzz21xEkEv8iTor3gh1ESYlRLn\\\\\\\/flJgaGvaUbqpuJwtmwL\\\\\\\/Nrrw\\\\\\\/h83TjXnn2FZfg8ZPPvZ8fQsuOTgnqnCSw8n5KBvr65lVxawzb5hUlMddTUFtxMLi5Tl+KM9kQxSmA0JCsR2KRzrAaMDu1BF6y2aUS3QRQRE5ZgUS7ZRnuv+0IWkh9qHdUgbp8nyJWDJ2RTDsKnjpsM42K9LExuay5vu\\\\\\\/3nzezUw714StdQVxN+jl21ahpinwagg8KkPDypSVFWIH2W0YgdczMDDl9k1iDQLNkpBsLn2gN4p0hxxdvqcqwNTmqlkhPx1Z2GwJ7t5xuSDZBgNIp2A7VBpAGrg3x0GLFTLvZ7hyFYCt+IjdGD\\\\\\\/diQu7JT69OjgHBTA+tYztIa1xn4p0HQhk0+5pwpEdIgoj\\\\\\\/+FXTvzbAYtCIynaHc\\\\\\\/+QjYBj\\\\\\\/mhq1dKASSifOTfgj4AZXh9QM2U6b6QANpg8tYkZwvTZpKoSMq4bu3dCa4XpL96iR+7UVdyvO\\\\\\\/qZZtu2GC+DAF1KFcoOJqxHilvIBJrcNr6ONd1lb6GMil\\\\\\\/HWC6IqreenEqwCUEtSv2m47qpOLUUSbbUaINSjbifB8sYb88WA9g3BUVS9oeUQquGMtpBTLW0E74GBjvRVJqwTDB26mO5w4lBINW3nvo2IH9xKPXfRuuRfTY3ky55Mv\\\\\\\/uM8q43gGjydDZ\\\\\\\/HizK6QOBCmzpZA4UaDZZhH9WvIhxhwyT417szJ5QZhAC7h6HnuNUS\\\\\\\/tKe3f8v42kch9+GjqasVgs42K0IrsqvSZah5JligQRLeWrjUhaC9UMbyC\\\\\\\/8Z6XpZp53vJpHTuyic5njzCBhL6057aKuNZh8cZWZAziIgBlnTWmyyqYRnh7b9QuaZMoL6CUx4AS0\\\\\\\/Oh\\\\\\\/YNJwKIBeekJZgrd3NwpbnuxomymBzfgrvIkia4avVoAOCivZEmr97EBeZOGE0yoZlKCxoHRodwzk7z7NFt3pj2y3+DovXZxRl8tlMyi5WrqQv7hF0xwPqZRCKj9a6SrobTRhftiTXoUj35bRq2LYn4R1n88HnuYmpTYKxjkDXUi1CS+ZWpscGvq7MDN3ii6o24k7e0r91H\\\\\\\/pikcgppQ0hbTDKluSUJdlPwT\\\\\\\/Odnz8RcN23NeFzSLYjoNyxdKPGhVimtO7yYUuCsKYC6K5S66JRbFHSWoPNZ7XR9yc4yxJeakFRTVgZ1Yl2BrwwwI+ZpIewW6oe2gqfajC\\\\\\\/iddYUVXA8XEww85cZX+kNAtcUrhSdY2ZW7kI1WdnNHzsNrN\\\\\\\/8TxuMBejwdRGj3rhukeZr0ZQzIlp8yAH4QyrpBWzf\\\\\\\/lOF4BxtqSGP2Kgg54de6iJ4WsYjuU49GC4rO2fiZ9J+dve81YEWWK8P7\\\\\\\/WVmn9436k9zauPyEQhVOX8q\\\\\\\/BHWengd0fUCK9w0BPF9kX\\\\\\\/yQs9XtlvOnJd5b8rwZ7H7tiUoWyHopOrkKJl5A8LhvYB9PbTP3AfQo8zvIJnE6tL+qJah8nFWdFaPSbEgoPmFAkIGMaR+NSyUwkHWgSeqUDmtviyFJ98SgsXEVyBPJpxGBK5AnaMhKclHtYHpcecncon942KAdLj3DgehpDaR2lxbsWrApGbbggkeG5OnrCIVjOp+p\\\\\\\/3RqOD0EX6SYkupGni5AoLd7i+ndDtPMOSTJSkV0QegCfejg84JNWKSaKxE9CoVzEuRXTST+OAXOMG5UZDJpUA91l8qj83A0Fx1hF8uvNGukSywsrUl+Di38fUr5UFGI0ULixVhl\\\\\\\/kCuPUENvIwjciGI\\\\\\\/vuYjKgtdEfd977JohhI4Qy1AGrmg4SyWABjU4D25qdI1l6FUn8dCa+PH22tdtIF2fLuVWNytNXKZpbvfT817rfoMSRgaMh\\\\\\\/3v0WRdOqdWjKpWVQ7GxqCiHgyNap8tosOHnKIdllz4836ilZKAUHyRACzTB3XuGqrE7r7mqkVng9wSh5PAGJa\\\\\\\/2+5yTkmWyR2DRzZZUQ+aIKUHhr0SzxdRTouy2lhCmhz2KIEwIM4zF9CK8+SIgN9EjQnkgy6wbxjBiUWcqCaj7Ig\\\\\\\/\\\\\\\/EEoSH64TEfDlvZ06Nh9XrG2Cx3nGtMWLoM3L3Nj8blu0pgvPBgoRobbVFUNWSQ4sdorYZggv1HeZFAKoWzevWYfzrPQp1Kf1dqHtCyPRgov1MrsNPPbwgNeDa+jrQvne9sMlmWUJQPXwjmTGWCTa0QdUH7Dd+erLUn+WHeMmop4A2E9flqjErWrNWfwNybYRw5CLcysoTiq+YgfSAJhvbxRD8vXpSBzYsdRIQ9q4yLYljs\\\\\\\/dv4+pChcOLcuVDrtmpnSZFkiy4FzV01GLgduoiMovFlxxtI90bT2uKi7oELkNtw9W9mnIGRkdmPyLHcr\\\\\\\/xMNVL1zw1OKFRYDAj9OLQQy\\\\\\\/5iO6L5utOx5KNIBtkVlO6D\\\\\\\/alIxSD0Vl1uRXLwBKxvY\\\\\\\/UUQT1hljI6KOpkkUM1EMZGTz9Fonf8Ev5S81xMxNA8EBWh9YUWyj4JaoCNUrEpLGUz2wK+pp0DEnjjRoaLZaz2vTr\\\\\\\/grwOf8m0Uf69sKWZcnN86fu+orElAiZhfvkkvJSwGHO4xln4cQmDHjg+0vuUEOe4vOEHxzKSVTmZXx56P68maM8biTHC6jKqYoMc2ii3A\\\\\\\/Q\\\\\\\/89b0soyEn8IlByPF8Ste2tfBwAONP6OBZ\\\\\\\/yEWxVrEtKSa5W+VffwqFf62czUxYvXYSJA6O58cZL0z3cLXb+E85SQJbqYTatBoD7orRdFBan5OAh0686MmZ1JcXrIZ+B1hbrrwqPslvgmJyzcQuNcJ1\\\\\\\/X1JdVkJ2voZ51TjQN+3a5Z\\\\\\\/2q\\\\\\\/tfTpDbMztDQ4AG3HxEwnpCbZlKWwVRA\\\\\\\/o0Bf1YJs0adiNA5BoRbKF2YT9Dpo7tcqu6s8CUNDXTdH0bJzQZYR1SuWPusFROHMenIGfywhLrcRXveAjomeNptK3ZkW9S0tC6p2tk3TtCifivXlKAS7u8uwabHgBayq\\\\\\\/6QEt+veIBYiUFy6TEAX3jj6idBrHwJSFy6\\\\\\\/3B7UtiF5mG1PL\\\\\\\/UW5MmimFf+IyppqhfW\\\\\\\/SOh8787gkjUU51FFud1XImnbduH+KgKLI9uTe66XumKsTC8XKzzKGOuB\\\\\\\/bTartSYeaGK5UQOWP80zgv5ZeIR3UuYZsOYsuwpnvvn8LW+gZQ7SKFGWcKPToUwxMbWFq5F0bV2tojTkT75xgD+tP+Df1HSN+DmUKNgKsfAwK7CaqmWBmoArA0dDKd9+Go3nz5TqNmsAuabrPBaoSuHuRlWp3Q84Ivv9+o5u5hlJIi5dTcjgUFsldYfbu7xv5NZBu85dfpnUb2YT6EGG3xXn+onGzKiSVVh4saW\\\\\\\/Ki\\\\\\\/qHgsSlZs\\\\\\\/iy4sbNnKS3\\\\\\\/kMSVz6CbRScDciwk1+q9euXVblWlh9QOsQWGzfktap8VKsYlUMWAjzjvfP\\\\\\\/rtRLIUscxsLo14MLO6YQzxmBkdArbRXuwkLBPYaRP96CXXj0xhbyzdLhhv0FmiYj3bqtdFGiOE1FSf2lUlWai\\\\\\\/M+gEmcLCX+7SNF5MYRhMvhQmOPhM9EvqI+E8Qvgm\\\\\\\/g9d\\\\\\\/jnAAe\\\\\\\/6MUMru+Qfzb5dJZmNpD7CKEoIF4NtXdVrwLDYl5c8z08vV\\\\\\\/n87ji0miph1F1tWGQihbV5wRINNY+N8PJZqKOGgVse\\\\\\\/SR+1PmTOtmDts3NZtqF8S+Fi7pLqLfNCPHZGX92ivVHtOrtii7yzrsErA8rR6iL1qkqU1nFVyzyMVhn6uS99hDYKg91cbUiGyG2bzBu8bAqI\\\\\\\/CAhbQKiupCihf3qYUtY6cpO+pN69wwGJroETRauWUKt3Ud6BCcwqmTnf4PYFO49hMk19v2jf5Nospl952e7SgAY0luEoJC8at0CT3twMI7WWY4O2vfTNzBRvDHrKTzAyefhoTgZrS79kMgPyedQiY5gw8NzsNc011pk1GW+UAs+YpXNLdofti2qULrpsPMvap1oermB4eGmQInYvKtuhi1QUDAvnPXaF4rZzwfAenJpJCPMUYVZnGo2WhhYjIrg5jg1RQlFIqjO6y2FRuMASDybS22iFvg\\\\\\\/UlJkgnB8QYCL\\\\\\\/VXK856RUD1zjDJrXxLdMBfB91+X7y04fRwqZZAWi3qjSfm6OT8oGamHn+MBIVCQPLPcb9vTAZE2InbaSotPhnGl8IPHwLpIVrRR\\\\\\\/uCissut04TDRBcWAckB9I0auMujvI0HM\\\\\\\/nzYnFfeQDWDe9rU4FEKkr7aI4M6qe7Q8iZpiWLC6u6khtE381RAMXoDzZ5reGyHsXUO6p5lh2a9KBe63rL4PTnFVksxqVskNXsg7Zi6RGtYdKLjVW8N4FnF6oxsTeTXT3I5ustdnolVoeXRymxIuXiYJYPVGINZkCoIUEyg0+XWZZAbU4fyJGrKxtviW3msooe4gWai19HEnqQtFPvD4mf5rUvBdQzYTwEacVL0HRKJgMA4XcPve1VnmfnhGKdmMHwyzEbR2S7iUKRC\\\\\\\/X6AEnHKUbkiJeYnAuq\\\\\\\/mZt84lY1NwIhnQO67w8OcSyRyqnUXogu4bo4oJQuj6pkMmjKEJnUmVRV3HgUE\\\\\\\/Lcb1RyXIJGDtnVQsoFyjVoOUnf4jh4SbJ+12foeVhbUgb487UViHPHrF2ZgIuo6pB9rEl4cYkyjCCD+3c+oyXoPHo\\\\\\\/P+qhu35uWVKZMCl8NHFuuoipK6jOZshLEtmKYhXo7XUmD4ZXMfETOH4niMbkfVlv5rJzjCBoqWeO4JzfdXDcyZC7jdxeqly1s6kFj4Yy3wRRJcP5fh\\\\\\\/qnhnnpxDGYyYHYGvv9jwc+xwPMp+krLv14R\\\\\\\/cudXgCkRu05IIoTc+ZfwR53EGUiwU1OqgmEWQWo82Uh69zigsRFPtwzoMlz8S0\\\\\\\/mXXQQSlivCbmyN7GWhN6xMuF2NUzo1le2stO606q+pW\\\\\\\/Avt2ieMkDIUwjIpUugRuI\\\\\\\/LWreVKliLCpG9BCZ4Fn95rJJyeKs4VRM+zDULPIZDDdLJv+\\\\\\\/stbr25793bWy0zNs+jiVR4gTA3JdikfA\\\\\\\/j5I8Pz6EzDD\\\\\\\/bGXgwA1OFor9TlQBk61Whg5ylstWpJeyTLlLcA\\\\\\\/vt8cxwDGQjGnfQxs5SpR63XKMijDMYKQMXu8QDM6mBDg+qxk6df7w5kXsrRuwm4K7Or8lcZJWwqVktmG5iUgu3IjzR7EVsxINtbu5wbDIWWiDHmzaVzxstFWRL\\\\\\\/YmsSGUHYm78aJtCzxISbgTbS04ZXwDD8L4IaQnBoWYlkpW2LrLOTpSa6GWmXwrfhXyZAYJXdm3gQZ64AskIa\\\\\\\/aGuLYC\\\\\\\/BrEb+ZKAxmUpP97GQpprflciX++jaCePanCvYWqtaWzG7LMh\\\\\\\/Ad4NWT1Ba3+SfZ1IXOeX3dlqp1BRkc6I25skdIt+fsJg8pXctSNvnUINeZco2SMOjtx4Fmwe389wF+8dVOKvHkcCbeY\\\\\\\/V4qXT86ewGfvKqMVNt0Gh+TFy\\\\\\\/UdlWXbKTYx58bfBCKxIUqDIju1FP5DoBsSxJSJ1ArvUfATPiSM5VUXTMxky1XWRAx4CTqPaMO+\\\\\\\/a142t0zX6KEgQNpos0GomoqKttNCVwlciHmbzugiL3WSrO3DhsmOxKI+EHgZjYDT6mVdj\\\\\\\/hLRjBaLWsnf7iV5nz9Pascykb+erwM\\\\\\\/14CBpu4UP4ekRZjBKx01GehpfS6FHngFpVAPb2z2pPQxO\\\\\\\/by07MnLD6VnMln+ogJab5up2RowJonf+RrHdttgU9h2TVwcGLXMjOGxEs\\\\\\\/emkKSZneHySiXtqui5Qsmbd0J1pw+OGagiJrLgPiImzuueh2C8Uc7MLnHjXkDupSJadIvRqwQZwTxOZ24ErnXAywj2W9qwOgRX8FAVWnQBkOrYXubLpuCNjlLdvQGtmeNPcfyMMbwjr2zJxAPiOFqqp2U\\\\\\\/ZVEyAfIPgnCCakQpkI2CqfegltHtoYDGQIJBXKJNaCKVkvFA4NrY15FIpJ9DRzVr1kEDjL\\\\\\\/itgy9WpRxpLR2E960ubkix4cn\\\\\\\/xHI0wTlUBB4hYeDc9u80dwrDjBUcoYcrtbS\\\\\\\/\\\\\\\/APrFO2rjbnapww\\\\\\\/H\\\\\\\/mUKgfGALzJrRNyX2ztf4N+JoW9HLsgJDZyz4fIQfO3zVcIk3iDxcMzvQzGhnWk2LVEpm6lgoZUAyhSHozwL3LOyPkZ+6L7CaLPJRxNN6oWAieIwufoEFDdKpzLgBZjH+QPR3A35KBZBvbvI1Q+Uwzgp1FkhoT05MdjOENhFOqgNkpJ\\\\\\\/1Dldpk2ilDmCWr3BkRZKbu\\\\\\\/0qrQ\\\\\\\/Y8qaNy\\\\\\\/\\\\\\\/52j5QKVW2mVczS6xdok95phEz4getm6jyw38E4I5jVnf3vU14WcTxjSKbe8YKu2xKIbjgk\\\\\\\/aditq78OyMbhuQt6b+p97NqsDQwTfm713u\\\\\\\/kQovIJI0R1iHW5lrKZO9SwZshb3X+w9k4N7OUWoZvOLO8YKbYEblYiF+YYtcQNrBB69+qn7w7OBDly4dNpfWoX9NtX5fiVBnzwx6gkdy0MjYHtxzDq+SmKMuq4bfRpwO8doOYxOVnuZOJk0hqGz5CmqjEWM9sZH9n1pVmYU5JKZ31xLwgAmr4MCTHHad3NrM3ACGg9Rr+EaeRE\\\\\\\/Kr8STxwZUqdnVvbw8+IVLWki56EKX87C3iDSemxEndBCi+ID7yiAbB1qVTBmqIadgGeY8p8PcNmNQrY6OC12Wld3q9uauxYco0OWJUaRYzJTAr+RkfOYODgEgAfnUnIvB8\\\\\\\/PUFeB1q9+o1ExVr3i62rG3ryEg\\\\\\\/kf1Ge9n9e8h8znlRI5uNeunTX3XA0XCj7t8dREFRA8G\\\\\\\/4iJreZlGYfbIEZrQ8PzhNKFuihSEx2P1Xyx6VzJ4vq87RnU4bH6YSomp7pQYPSv+kWGuGAlWTsoHEcy8GTxxMQOoKKBaYGin9zvxNGnHb5Fqnqey6jWl6BZrWxpb2Hu8iRWeY0g6X36fHL0JaK\\\\\\\/aSSuD9gjtzU8SbYzR7bukbzqmeOlfRSCQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022d43VfPZrRn7XcwnrsapMjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022692a5ba40432368b52b1123ce07c36a987a632938e46cb0f2b0bbb8fc61fb310\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229gBVZrji\\\\\\\/VPsBan87ek5l2a8nYW5hsRcu+4rsgRm9nfn\\\\\\\/8uZLchdc5oj8H+IEIlTvA6B7NdPmL6KQYkPD8zowKvjjTbc6a147mp1rEUnP6YqfOXK22HCTV4IYchX\\\\\\\/Gf4ipKJOeIY09CBLsZhmL1whIRmiwog9qXSD8hS0gKZ6eSwoIfoEnPTn37+gg\\\\\\\/JcGoDkcFxRR4G09rdErSw3c+ozJhUqbp+WJCM2J+RSr7HUpJomMCFdbDKuYdtHVzqIQhyDJIQYM6POmoc+IRo8GRXHGXnL4yHAayF830\\\\\\\/Vs+DuLxtDOlU76rDo20FV3\\\\\\\/Ldk4iyQ3sXHCIWZb8D0iu4zVrqnKtnanY12Erq8pcM4uctVom8HgBpD7jKxYmen66Ie7g4S0xmOMye+gtWgBcgjfb+wYa4zNhNbGKpePOnh2f7d0OziWaneI3Z\\\\\\\/6lZ7doC\\\\\\\/6sVpePz5iLmSbQYaRRgeSuB0SCImQKFM6aQpJxawQ6XsVMIsbIrv0ecJ7ixUGmN6IgQLLC71VpjU5ZkqNHKNgPBbGrH8ao3ktZxMlaZsIBke8VFON+2umlX4\\\\\\\/TWnoAv90H38MjksB+9Lh5a3EAFlXfxZy5KLUOOiOa00n9iH3tR3eSeTaplpgRyUBNmZH9vYPUqIJZIMj1m2LB7\\\\\\\/Wc8gmolg4dCxn8H1OonKVDEdXc0iS7Ms4TowGTLWV9sDzX+lJYBwjPu+ZYKOpRVxtIvVvVoK72KVNaeqsj8M\\\\\\\/zNhY4HSB2F957hgtoirxzFLRz+XWwe6sq9nC1dy53k4+\\\\\\\/FmAJR+Xw4Bm7vyB9c+za3ro34AX7jWpmVsurPKUDh1DvMpuu2FvhiRL675Ud9edhyPR9z4GvycRa\\\\\\\/yRxRN09X3+I6vyMV7i36lSvtBFDUG\\\\\\\/N+\\\\\\\/nWL2QGRN3+KcyZNeIwBRtKeaHcR9sn7Q1iZ9tlTy0BBxwvPNyvGhEBY9AFiC57XaqKF5cxg6wRXGie7pDe3ehM7ofWVe8M5Mjj4qN1LBosU1ZzNWd5AvTM7D+yjbcaYQEvZ5AY5ZF59WNgPoIKvmz\\\\\\\/Sk576kOHaVg9u0moSXZAeZbBKjzI6pTtMKMf\\\\\\\/NZzrpatEa6KxpNpSgVXB38kO74QUvxVl7nhOfUIsVaDd3dvLyJK+Wob\\\\\\\/fAiNwQmRxVzbSqG4EJhvLXADcEkqta8jVAfhr71aVh75hul7TfI7QVId2NsNVrTdEFKAq7KFfkxa\\\\\\\/X9icYZ0m4fdQ\\\\\\\/7B5H+zN+PaXapDG8jcYCRsHobs450I+tPzLv6s8+1ve5E4cmnsvz871MNCWTnYJ4IMISYyGDnrxD8MbyKnplYjjCdPh+7VXzIOME4Z3h5ZXvWAFzJSt5PxOyNLnuB\\\\\\\/ZhU2F8pSgJczbn0DjB2Ub4hBAA2WRAYGIbUcW2RytSKUrqBlPj+SHd7mrnP9D3BjE3cpzee+rbH5laaeig3qLEHhNdQO7HNdRG61K56p7Pm6+p+jKXIjorqduKPpT2qzRl2g6Klf\\\\\\\/8rnv4QQBThI9rNW5ByEpxWHG0KDe9TB4Lc9ZDPQX3f\\\\\\\/Bx2lMFDOiItpxvJug8tHCPFII6iv8a1CJUndjtGdsS7bZT9LG75KTBu\\\\\\\/8vswZVneS3Kj\\\\\\\/vGsweAZONq3gLI8N+1X5AgFnt1KwmxsIlI+trRlrw+bVrVzKiCVoKvJLz12vnkNpzI5K3LFW3ccAiHDB79eRCQkYI3jGHT+4EedAYOALEgAow5pTBz04wKzrihElhZA8nkVvLrrC1gwrki34xbBIZgwQRuCbPWMdaOLpsxxoicEZNitOJfzRNKepAC5KDlBRADrLrIfO3pOfFN2NB41rubO5l8L82D\\\\\\\/4cmFTFvyrqATCbAD2NRcHKwbMCoCw52WMvbLk\\\\\\\/2OY2MCzMdudty+GPo6pvfdg4I7m2CWM51Fa7WmUuWd\\\\\\\/PexlPC8LY4psDoiXJzM\\\\\\\/52yhdVm57TvvUA8u05LsKU2HlTym674gpdlKujD8y\\\\\\\/DSFc5htYCUIOMX98NqyFLX8B1Pn8bLOGAo+BSA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wP3wZ7Yr4FdVpNKYzHzAPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a0252eb5987d3e989579c62b6f5cbb6bcef0f00890db131e49d3a4cb36b042c4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493459 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2277" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aQ3aIbr3ydoE2D-D-7SrGv4Ftwdzd4Ogidbc0hLnul0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2277" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PL_s6md90MtX89iF9WaO_GIUz-bu6SxPBCbq-V-kMhw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2267" title="Expression Signatures of Cisplatin- and Trametinib-Treated Early-Stage Medaka Melanomas" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2287" title="Restriction of Cellular Plasticity of Differentiated Cells Mediated by Chromatin Modifiers, Transcription Factors and Protein Kinases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493459" id="node493459" data-pisa="ggg;9/7/2277" data-pisa-master="ggg;g3.119.400326" data-apath="/ggg/9/7/2277.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em>, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in <em>Pristionchus</em> Nematodes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3099-1007" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Neel Prabh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7905-9675" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian Rödelsperger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2277-2286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400326 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493459" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Neel Prabh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Genetics, Max-Planck-Institute for Evolutionary Biology, August Thienemann Str. 2, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neel%2BPrabh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prabh%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANeel%2BPrabh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3099-1007" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Neel Prabh</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian Rödelsperger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BR%C3%B6delsperger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6delsperger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BR%25C3%25B6delsperger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7905-9675" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian Rödelsperger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.roedelsperger@tuebingen.mpg.de" class="" data-icon-position="" data-hide-link-title="0">christian.roedelsperger@tuebingen.mpg.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2277" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493459" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493459/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2277.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493459" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493459/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2277.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493459" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493459/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2277.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493459" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493459/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Homology is a fundamental concept in comparative biology. It is extensively used at the sequence level to make phylogenetic hypotheses and functional inferences. Nonetheless, the majority of eukaryotic genomes contain large numbers of orphan genes lacking homologs in other taxa. Generally, the fraction of orphan genes is higher in genomically undersampled clades, and in the absence of closely related genomes any hypothesis about their origin and evolution remains untestable. Previously, we sequenced ten genomes with an underlying ladder-like phylogeny to establish a phylogenomic framework for studying genome evolution in diplogastrid nematodes. Here, we use this deeply sampled data set to understand the processes that generate orphan genes in our focal species <em>Pristionchus pacificus</em>. Based on phylostratigraphic analysis and additional bioinformatic filters, we obtained 29 high-confidence candidate genes for which mechanisms of orphan origin were proposed based on manual inspection. This revealed diverse mechanisms including annotation artifacts, chimeric origin, alternative reading frame usage, and gene splitting with subsequent gain of <em>de novo</em> exons. In addition, we present two cases of complete <em>de novo</em> origination from non-coding regions, which represents one of the first reports of <em>de novo</em> genes in nematodes. Thus, we conclude that <em>de novo</em> emergence, divergence, and mixed mechanisms contribute to novel gene formation in <em>Pristionchus</em> nematodes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/orphan-genes" class="hw-term hw-article-keyword hw-article-keyword-orphan-genes" rel="nofollow">Orphan genes</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/de-novo-genes" class="hw-term hw-article-keyword hw-article-keyword-de-novo-genes" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">de novo</em> genes</a></li><li class="kwd"><a href="/keyword/taxonomically-restricted" class="hw-term hw-article-keyword hw-article-keyword-taxonomically-restricted" rel="nofollow">taxonomically-restricted</a></li><li class="kwd"><a href="/keyword/frame-shift" class="hw-term hw-article-keyword hw-article-keyword-frame-shift" rel="nofollow">frame-shift</a></li><li class="kwd"><a href="/keyword/annotation-artifacts" class="hw-term hw-article-keyword hw-article-keyword-annotation-artifacts" rel="nofollow">annotation artifacts</a></li></ul><p id="p-3">The sequencing of hundreds of genomes lead to the discovery of new genes that do not share protein sequence homology with previously known genes. Over the years, these genes have been referred as young, pioneer, or orphan genes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dujon 1996</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Khalturin <em>et al.</em> 2009</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tautz and Domazet-Lošo 2011</a>). Orphan genes makeup a considerable fraction of every sequenced metazoan genome and as a result, the total number of orphan genes has far surpassed that of the known gene families (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Khalturin <em>et al.</em> 2009</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tautz and Domazet-Lošo 2011</a>). Recent studies have shown that the proportion of orphan genes tends to be higher in secluded species that are devoid of genome data from closely related lineages (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Khalturin <em>et al.</em> 2009</a>; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Tautz and Domazet-Lošo 2011</a>). Thus, deep taxon sampling of closely related species is needed to study their age, origin, and mode of evolution (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Palmieri <em>et al.</em> 2014</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Stein <em>et al.</em> 2018</a>).</p><p id="p-4">The nematode <em>Pristionchus pacificus</em> is an established model organism, which has been used for comparative studies with <em>Caenorhabditis elegans</em> (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sommer and Sternberg 1996</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sommer 2015</a>). Orphan genes constitute roughly one third of all <em>P. pacificus</em> genes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Borchert <em>et al.</em> 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baskaran <em>et al.</em> 2015</a>). Given that the estimated divergence time between <em>P. pacificus</em> and <em>C. elegans</em> is 75 (±15) mya (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Werner <em>et al.</em> 2018</a>), hence the high fraction of orphan genes in <em>P. pacificus</em> can be attributed to the depleted taxon sampling around it (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Prabh and Rödelsperger 2016</a>). Thus, to overcome the limitation of this scarce taxonomic representation, we recently sequenced ten closely related nematode genomes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rödelsperger <em>et al.</em> 2014</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">2017</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>), which created a ladder-like phylogeny around <em>P. pacificus</em>. All ten genomes were generated within a single laboratory and were annotated using the same pipeline to minimize technical variations and maximize comparability (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>). The ensuing analysis was based on orthologous clustering and assignment of the resulting gene families into age classes. This revealed that younger age classes tend to be located on chromosome arms, show less evidence of expression, evolve more rapidly, and have a higher propensity of being lost (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>). However, in our previous work, we neither segregated gene families into orphan and conserved classes nor did we infer their mechanism of origin. Although several mechanisms for the emergence of orphan genes have been suggested, rapid divergence and <em>de novo</em> emergence remain the most widely accepted (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Khalturin <em>et al.</em> 2009</a>; <a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Tautz and Domazet-Lošo 2011</a>). Previously, the <em>de novo</em> emergence of an open reading frame from an ancestrally non-coding region had been considered highly unlikely (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jacob 1977</a>). However, the initial finding of the first instances of <em>de novo</em> genes in <em>Drosophila</em>, yeast, <em>E. coli</em>, humans, and plants (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Levine <em>et al.</em> 2006</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Delaye <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cai <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Heinen <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Knowles and McLysaght 2009</a>), inspired several subsequent efforts to identify and characterize <em>de novo</em> genes in many other organisms including mammals, insects, and viruses (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2010</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carvunis <em>et al.</em> 2012</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Xie <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Murphy and McLysaght 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sabath <em>et al.</em> 2012</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wissler <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2015</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ruiz-Orera <em>et al.</em> 2018</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Vakirlis <em>et al.</em> 2018</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Klasberg <em>et al.</em> 2018</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Schmitz <em>et al.</em> 2018</a>). While <em>de novo</em> gene origin is generally inferred through identification of an ancestrally homologous non-coding sequence in a closely related genome (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McLysaght and Hurst 2016</a>), recognition of gene birth through divergence is complicated by the heterogeneous mechanisms with various degrees of sequence change that make automated homology detection untenable (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Schmid and Tautz 1997</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Schmid and Aquadro 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Long <em>et al.</em> 2003</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2013</a>). Thus, investigating whether an orphan gene fits one of these models is a difficult proposition and requires both exhaustive computational and manual analysis of individual cases. Accordingly, this study is divided in two parts. In the first part, we employ an automated pipeline that establishes distinct classes of orphan genes and takes stock of how these genes are distributed along the <em>Pristionchus</em> phylogeny. In the second part, we manually investigate a limited number of candidate genes to illustrate several mechanisms of orphan gene origin.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fraction of SSOGs is consistent within the Pristionchus genus irrespective of divergence time (A) The maximum-likelihood phylogenetic tree of the species analyzed in this study, adapted from R&#xF6;delsperger et al. (2018). Branch lengths denote the number of amino acid substitutions per site. The numbers correspond to the phylostrata from panel C. (B) The horizontal stacked bars show the fractions of Conserved genes, TROGs, and SSOGs. (C) The ten phylostrata depict the origin of P. pacificus orphan genes along the diplogasrid lineage. Blue boxes indicate presence of P. pacificus orphan genes and the most distant diplogastrid species that has homologs of these gene, red bars indicate absence of homologs, and gray bars indicated homologs may or may not be present. The number of P. pacificus orphan genes in each phylostratum are at the bottom. (D) The heatmap shows traces of homology for P. pacificus in genomic and transcriptomic data of other species. The rectangles indicate whether traces of homology were found (blue) or not (red). Manual inspection of P. pacificus RNA-seq data resulted in a high-confidence data set of 29 P. pacificus SSOGs which were taken as the starting point for origin analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2134227042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fraction of SSOGs is consistent within the &lt;em&gt;Pristionchus&lt;/em&gt; genus irrespective of divergence time (A) The maximum-likelihood phylogenetic tree of the species analyzed in this study, adapted from R&#xF6;delsperger &lt;em&gt;et al.&lt;/em&gt; (2018). Branch lengths denote the number of amino acid substitutions per site. The numbers correspond to the phylostrata from panel C. (B) The horizontal stacked bars show the fractions of Conserved genes, TROGs, and SSOGs. (C) The ten phylostrata depict the origin of &lt;em&gt;P. pacificus&lt;/em&gt; orphan genes along the diplogasrid lineage. Blue boxes indicate presence of &lt;em&gt;P. pacificus&lt;/em&gt; orphan genes and the most distant diplogastrid species that has homologs of these gene, red bars indicate absence of homologs, and gray bars indicated homologs may or may not be present. The number of &lt;em&gt;P. pacificus&lt;/em&gt; orphan genes in each phylostratum are at the bottom. (D) The heatmap shows traces of homology for &lt;em&gt;P. pacificus&lt;/em&gt; in genomic and transcriptomic data of other species. The rectangles indicate whether traces of homology were found (blue) or not (red). Manual inspection of &lt;em&gt;P. pacificus&lt;/em&gt; RNA-seq data resulted in a high-confidence data set of 29 &lt;em&gt;P. pacificus&lt;/em&gt; SSOGs which were taken as the starting point for origin analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2277/F1.medium.gif" width="440" height="392"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2277/F1.medium.gif" width="440" height="392"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493554" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Fraction of SSOGs is consistent within the <em>Pristionchus</em> genus irrespective of divergence time (A) The maximum-likelihood phylogenetic tree of the species analyzed in this study, adapted from <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rödelsperger <em>et al.</em> (2018)</a>. Branch lengths denote the number of amino acid substitutions per site. The numbers correspond to the phylostrata from panel C. (B) The horizontal stacked bars show the fractions of Conserved genes, TROGs, and SSOGs. (C) The ten phylostrata depict the origin of <em>P. pacificus</em> orphan genes along the diplogasrid lineage. Blue boxes indicate presence of <em>P. pacificus</em> orphan genes and the most distant diplogastrid species that has homologs of these gene, red bars indicate absence of homologs, and gray bars indicated homologs may or may not be present. The number of <em>P. pacificus</em> orphan genes in each phylostratum are at the bottom. (D) The heatmap shows traces of homology for <em>P. pacificus</em> in genomic and transcriptomic data of other species. The rectangles indicate whether traces of homology were found (blue) or not (red). Manual inspection of <em>P. pacificus</em> RNA-seq data resulted in a high-confidence data set of 29 <em>P. pacificus</em> SSOGs which were taken as the starting point for origin analysis.</p><div class="sb-div caption-clear"></div></div></div><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Identification of orphan genes</h3><p id="p-6">The genome, protein and transcript data of 24 non-diplogastrid nematodes were obtained from Wormbase (WormBase web site, <a href="http://www.wormbase.org">http://www.wormbase.org</a>, release WS254, date 7/18/16). The phylogenomic data set for the ten diplogastrid nematodes was gathered from our previous publication (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>) and is available at <a href="http://www.pristionchus.org/download">http://www.pristionchus.org/download</a>. To improve readability, we have abbreviated the original gene identifiers throughout the manuscript and a table with full identifiers and corresponding gene models on WormBase (WS269) and WormBase ParaSite (WBPS13) is provided in Table S2. All the Uniprot knowledgebase taxonomic divisions SwissProt data were downloaded from <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/">ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/</a>. The invertebrate taxon contained a single <em>Pristionchus</em> species gene, Q9NHZ4, which was removed from further analysis.</p><p id="p-7">We first identified all conserved genes for the ten diplogastrid nematodes using the following approach:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-8">Classify all genes that have blastp match (E-value ≤ 10<sup>−3</sup>) with any non-diplogastrid nematode protein as ‘Conserved genes’. For the remaining genes go to step 2.</p></li><li id="list-item-2"><p id="p-9">Classify all genes that have a tblastn match (E-value ≤ 10<sup>−5</sup>) with any non-diplogastrid nematode genome as ‘Conserved genes’. For the remaining genes go to step 3.</p></li><li id="list-item-3"><p id="p-10">Classify all genes that have blastp match (E ≤ 10<sup>−3</sup>) with any protein from any Uniprot knowledgebase taxonomic divisions as ‘Conserved genes’. The proteins classified as conserved genes at this step are candidates for horizontal gene transfer.</p></li></ol><p id="p-11">The remaining genes were classified as ‘Orphan genes’. All blast runs were conducted, with version 2.6.0+, under default parameters (including no filtering of low complexity regions by SEG) unless mentioned otherwise.</p></div><div id="sec-3" class="subsection"><h3>Classification of orphan genes</h3><p id="p-12">The availability of ten diplogastrid genomes provided us with the opportunity to further investigate <em>Pristionchus</em> orphan genes. Our first aim was to identify the orphan genes that have a homolog in at least one other diplogastrid species. Thus, for each species we selected the subset of orphan genes that have blastp match (E-value ≤ 10<sup>−3</sup>) with at least one other diplogastrid species. This subset of orphan genes was classified as ‘Taxon-restricted orphan genes’ (TROGs). The remaining orphan genes were classified as ‘Species-specific orphan gene’ (SSOGs), as they did not show blastp match with any other species. It is important to note here that for the identification of TROGs we have only used protein homology. We did not employ tblastn against genomes to avoid detection of pseudogenes or non-coding genomic regions as protein homologs. Further, since a ladder-like species phylogeny exists around our focal species <em>P. pacificus</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Rödelsperger <em>et al.</em> 2018</a>), we decided to trace the origin of <em>P. pacificus</em> TROGs and SSOGs on this phylogeny. For this, we employed the phylostratigraphy approach (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Domazet-Loso <em>et al.</em> 2007</a>). This approach is based on finding the oldest ancestral node of a given phylogenetic tree where the founding member of a gene family can be traced back to. Thus, we divided the diplogastrid family tree into nine phylostrata. ‘Phylostratum 1’ corresponds to the most recent common ancestor of <em>P. pacificus</em> and <em>P. exspectatus</em>. Additionally, we created ‘Phylostratum 0’ that includes <em>P. pacificus</em> SSOGs and hence is the youngest phylostratum.</p></div><div id="sec-4" class="subsection"><h3>Mapping of gene models from on the genome of other species</h3><p id="p-13">The synteny relation between genes from <em>P. pacificus</em> and the other species was derived using CYNENATOR (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rödelsperger and Dieterich 2010</a>). Pairwise blastp results for each species pair and two files containing genomic location of genes in both the species, were provided as input to the software. The output file contained a list of genes from both species within the syntenic blocks. Spliced alignment of gene models from one species to the genome of another species was done by employing the protein2genome model of the Exonerate tool (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Slater and Birney 2005</a>).</p></div><div id="sec-5" class="subsection"><h3>Gene structure validation</h3><p id="p-14">One of the main aims of this study was to elucidate the mechanistic details underpinning the birth of new genes. However, even with our structured approach of dividing the orphan genes into several categories and subcategories, we were unable to put forward a clear hypothesis on this matter. Thus, we decided to create a set of most reliable candidate genes to better understand the processes that foster new <em>P. pacificus</em> genes. For this, we limited ourselves to the <em>P. pacificus</em> SSOGs with confirmed gene structure. The validation of predicted gene structure was done by visual inspection, in IGV (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Thorvaldsdóttir <em>et al.</em> 2013</a>), of raw RNA-seq data aligned with the <em>P. pacificus</em> genome (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sinha <em>et al.</em> 2014</a>; <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>). We used TopHat v2.1.1 and STAR version 020201 for aligning the raw reads to genome (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dobin <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kim <em>et al.</em> 2013</a>). Single exon genes were filtered out. Only multi-exon genes with minimum two spliced RNA-seq reads aligning all coding exons and minimum two spliced reads straddling such exons, were assigned ‘fully confirmed gene structure’ status. If, only few, but not all exons of a gene qualified this criteria, then it was assigned ‘partially confirmed gene structure’ status. For overlapping genes from opposite strands, strandedness of strand-specific RNA-seq data were used as an additional confirmation step.</p></div><div id="sec-6" class="subsection"><h3>Selection analysis</h3><p id="p-15">For selection analysis of the SSOG candidates, their orthologous reading frames (including in-frame stop codons) from sister species were extracted and manually adjusted. Protein alignment of the candidate and its corresponding reading frames from one or more sister species was done using MUSCLE and visualization was done with SeaView (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Edgar 2004</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gouy <em>et al.</em> 2009</a>). The protein alignment was converted to codon with PAL2NAL (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Suyama <em>et al.</em> 2006</a>). Selection analysis was done with codeml suite of PAML (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yang 2007</a>). Species tree was passed as gene tree to PAML. If the corresponding homologous region from only one sister species was included in the analysis we generated a single ω value for the entire tree, else we generated independent ω values for each branch of the tree (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5C</a>). The statistical significance of the resulting ω values was calculated using the likelihood ratio test at the P-value threshold of 0.05. Only statistically significant results were reported.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-16">Sequences are available at <a href="http://www.wormbase.org">http://www.wormbase.org</a> and <a href="http://www.pristionchus.org">http://www.pristionchus.org</a>. Full accession numbers of all abbreviated gene identifiers are listed in Table S2. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8123768">https://doi.org/10.25387/g3.8123768</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Roughly 10% of all genes are species-specific irrespective of sampling depth</h3><p id="p-17">To quantify the amount of orphan genes among the ten nematode genomes, we applied a three-step filtering procedure (see <em>Methods</em>) that classified around one third of genes in each genome as orphan (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). We next explored the conservation of orphan genes within the diplogastrida family. Roughly 70% of all orphan genes have a homolog in at least one other diplogastrid species (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a> and Figure S1) and were therefore labeled as ‘Taxonomically-restricted orphan gene’ or ‘TROG’. Thus, approximately 10% of all genes in different <em>Pristionchus</em> species lack any homology at the protein level with any other species and were classified as ‘Species-specific orphan gene’ (SSOG). This lack of phylogenetic signal is unexpected, since the taxonomic sampling is much deeper around our focal species <em>P. pacificus</em> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>) and encompasses the two sister species, <em>P. exspectatus</em> and <em>P. arcanus</em>, that can still form viable but sterile hybrids with <em>P. pacificus</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kanzaki <em>et al.</em> 2012</a>). Hence we naively anticipated that this should result in a much lower fraction of SSOGs in our focal species and its close neighbors. While we cannot rule out that a constant fraction of erroneous gene annotations partially contributes to this pattern, these results are consistent with the idea that novel genes are frequently generated as a result of pervasive transcription but rarely reach fixation and are rapidly lost (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Schmitz <em>et al.</em> 2018</a>).</p></div><div id="sec-10" class="subsection"><h3>SSOGs make the most gene rich phylostratum</h3><p id="p-18">To gain more detailed insights into the age distribution of <em>P. pacificus</em> orphan genes, we separated them into different phylostrata that can be mapped to the most recent common ancestors of <em>P. pacificus</em> and the other diplogastrid species (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). Based on the parsimonious assumption that the breadth of a gene’s phylogenetic distribution is an indicator of its age, a gene that is shared by several species is expected to be older than a gene that is present in only one or two species. Thus, each orphan gene was placed into the phylostratum that points to the most recent common ancestor of <em>P. pacificus</em> and its most distantly related species that has a homolog of this gene (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Domazet-Loso <em>et al.</em> 2007</a>). <em>P. pacificus</em> SSOGs were placed in the ‘Phylostratum 0’ which is the most gene rich among all phylostrata (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). This gene set is likely a mixture of annotation artifacts and novel gene-like sequences that result from pervasive transcription and translation but do not live long enough to survive a speciation event (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hangauer <em>et al.</em> 2013</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ruiz-Orera <em>et al.</em> 2018</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Schmitz <em>et al.</em> 2018</a>). Due to the high abundance of <em>P. pacificus</em> SSOGs and the possibility to study their origin in multiple closely related genomes, we decided to investigate in further detail the processes that generate such SSOGs.</p></div><div id="sec-11" class="subsection"><h3>Most P. pacificus SSOGs have traces of homology in closely related genomes</h3><p id="p-19">The taxon sampling around our focal species <em>P. pacificus</em> allowed exhaustive homology search of <em>P. pacificus</em> SSOGs in the genomes of sister species, which could be indicative of their mechanism of origin. To this end, we performed various blast searches against the annotated transcripts, genome assembly, and transcriptome assembly (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a>). While tblastn searches against the genome assembly of other species may identify homologous non-coding regions of <em>de novo</em> candidates, we additionally performed a blastn search against the annotated transcripts to screen for potential cases of ORF switching, and a blastn search against the transcriptome assembly to assess the degree of missing homology due to assembly gaps. As a result, 504 (32%) of <em>P. pacificus</em> SSOGs show blast hits in all three target database types, which after closer investigation was seen to be largely due to overlapping gene structures. It is important to note here that nematodes possess high fraction of overlapping genes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jan <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rödelsperger <em>et al.</em> 2016</a>). Another 479 (31%) of <em>P. pacificus</em> SSOGs did not show hits in any of the databases and were labeled ‘Untraceable’. Among the remaining SSOGs, we find only 29 (2%) with a hit in the transcriptome assembly but not in the genome or the annotated transcripts. This fraction of putative assembly gap genes is constantly low for all our genomes supporting their comparably high quality (<a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Prabh <em>et al.</em> 2018</a>). In total, 1082 (61%) of <em>P. pacificus</em> SSOGs exhibit detectable traces of homology in the genomes of other closely related species, demonstrating that the taxon sampling of our phylogenomic data set is sufficient to study the mechanisms of origin for the most <em>P. pacificus</em> SSOGs in greater detail.</p></div><div id="sec-12" class="subsection"><h3>Identification of a high-confidence candidate set for origin analysis</h3><p id="p-20">Given that more than a thousand <em>P. pacificus</em> SSOGs show traces of homology in closely related sister species and that the gene structures of orphan genes in general are poorly supported by expression evidence (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Prabh and Rödelsperger 2016</a>), we first needed to define a high-confidence candidate set of SSOGs that could be used for detailed gene origin analysis (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1D</a>). We only considered SSOGs with more than one annotated exon, because we hypothesized that this additional layer of regulated expression involving the proper splicing of the transcripts would yield a more likely protein-coding gene candidate with confirmed regulated expression as opposed to pervasive transcription and translation (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hangauer <em>et al.</em> 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ruiz-Orera and Mar Albà 2019</a>). Additionally, the splice sites can be informative to better predict the correct orientation of the gene, which is essential to elucidate their origin and the reason why homology was not detected in the first place. We manually inspected RNA-seq alignments of all <em>P. pacificus</em> SSOGs except the untraceable genes, in total 1082 candidate loci, to find gene structures that are fully confirmed by raw RNA-seq reads and we insisted on finding a minimum of two raw RNA-seq reads aligned with each coding exon and two spliced reads that span such exons. Eventually, we established 29 SSOGs with fully confirmed gene structures (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1D</a>) that formed our high-confidence candidate set. Based on our investigation, we provide examples for six plausible mechanisms that explain the origin of SSOGs including two examples of <em>de novo</em> genes. Among the high-confidence candidates 21 can be explained by the proposed mechanisms, the origin of six candidates cannot be unambiguously concluded and the two remaining candidates were annotation artifacts (Table S1).</p></div><div id="sec-13" class="subsection"><h3>Divergence by recycling of ancestrally protein-coding fragments</h3><p id="p-21">The first mechanism alludes to chimeric gene formation resulting in an SSOG with two exons. Both exons are derived by partial duplication, but of separate source genes. The paralogous exons from both the ancestral source genes are duplicated and then get inserted in close proximity to facilitate the formation of a novel ORF (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Considering that such genes can be created by minimal contribution from existing genes, local alignment based tools may fail to detect the homology of these short sequence stretches with their paralogous exons. For example, PP378-0.29 is a <em>P. pacificus</em> SSOG with two exons. Its first exon has 100% protein identity with an exon from a <em>P. exspectatus</em> Gluthatione peroxidase gene (92% identity with the corresponding exon of the orthologous <em>P. arcanus</em> gene PA7-2.29), while its second exon shows partial identity with an exon of another conserved <em>P. exspectatus</em> gene (PE440-0.48, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Orthologs of both <em>P. exspectatus</em> genes are maintained in <em>P. pacificus</em> and given that the first exon shows high sequence identity with same exon of the Gluthatione peroxidase gene in both <em>P. exspectatus</em> and <em>P. arcanus</em>, we can establish that the the first exon of our candidate has been derived through partial duplication of an existing gene. However, blastp failed to detect homology with the paralogous exons from the two <em>P. exspectatus</em> genes. This demonstrates that even if a high percentage of identity is retained between paralogous exons, small chimeric genes can be classified as SSOGs.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence divergence and ORF shift erode evidence of homology. (A) The schematic overview shows an example of an SSOG with chimeric origin. Two exons gained from partial duplication of two distinct genes are joined together and with time sequence divergence occurs. Thus, traces of sequence homology with the original exons become hard to detect and such genes get classified as SSOGs. (B) This example shows a P. pacificus SSOG (PP378-0.29) of chimeric origin and its alignments with parts of two conserved P. exspectatus genes. Identical amino acid residues are labeled in black between the P. pacificus and P. exspectatus exons. Even though the first exon is 100% identical with its homolog, the stretch of alignment is not long enough to be detected by blastp at the stipulated E-value cutoff. (C) Schematic overview of a gene split with subsequent exon gain which results in an SSOG (D) The P. pacificus SSOG PP390-0.42 is homologous to the first exon of a conserved P. exspectatus gene. The neighboring gene shows homology with the remaining exons, indicating that the SSOG is derived from a gene split event. (E) The alignment of the P. pacificus SSOG with P. exspectatus is spread over multiple reading frames. Amino acid identity between the predicted reading frame of both the proteins are marked in black and those from the other reading frame of the exspectatus gene are marked in saffron. The residues corresponding to the P. pacificus SSOG in different reading frames of the P. exspectatus sequence are also labeled in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2134227042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequence divergence and ORF shift erode evidence of homology. (A) The schematic overview shows an example of an SSOG with chimeric origin. Two exons gained from partial duplication of two distinct genes are joined together and with time sequence divergence occurs. Thus, traces of sequence homology with the original exons become hard to detect and such genes get classified as SSOGs. (B) This example shows a &lt;em&gt;P. pacificus&lt;/em&gt; SSOG (PP378-0.29) of chimeric origin and its alignments with parts of two conserved &lt;em&gt;P. exspectatus&lt;/em&gt; genes. Identical amino acid residues are labeled in black between the &lt;em&gt;P. pacificus&lt;/em&gt; and &lt;em&gt;P. exspectatus&lt;/em&gt; exons. Even though the first exon is 100% identical with its homolog, the stretch of alignment is not long enough to be detected by blastp at the stipulated E-value cutoff. (C) Schematic overview of a gene split with subsequent exon gain which results in an SSOG (D) The &lt;em&gt;P. pacificus&lt;/em&gt; SSOG PP390-0.42 is homologous to the first exon of a conserved &lt;em&gt;P. exspectatus&lt;/em&gt; gene. The neighboring gene shows homology with the remaining exons, indicating that the SSOG is derived from a gene split event. (E) The alignment of the &lt;em&gt;P. pacificus&lt;/em&gt; SSOG with &lt;em&gt;P. exspectatus&lt;/em&gt; is spread over multiple reading frames. Amino acid identity between the predicted reading frame of both the proteins are marked in black and those from the other reading frame of the exspectatus gene are marked in saffron. The residues corresponding to the &lt;em&gt;P. pacificus&lt;/em&gt; SSOG in different reading frames of the &lt;em&gt;P. exspectatus&lt;/em&gt; sequence are also labeled in black.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2277/F2.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2277/F2.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493601" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Sequence divergence and ORF shift erode evidence of homology. (A) The schematic overview shows an example of an SSOG with chimeric origin. Two exons gained from partial duplication of two distinct genes are joined together and with time sequence divergence occurs. Thus, traces of sequence homology with the original exons become hard to detect and such genes get classified as SSOGs. (B) This example shows a <em>P. pacificus</em> SSOG (PP378-0.29) of chimeric origin and its alignments with parts of two conserved <em>P. exspectatus</em> genes. Identical amino acid residues are labeled in black between the <em>P. pacificus</em> and <em>P. exspectatus</em> exons. Even though the first exon is 100% identical with its homolog, the stretch of alignment is not long enough to be detected by blastp at the stipulated E-value cutoff. (C) Schematic overview of a gene split with subsequent exon gain which results in an SSOG (D) The <em>P. pacificus</em> SSOG PP390-0.42 is homologous to the first exon of a conserved <em>P. exspectatus</em> gene. The neighboring gene shows homology with the remaining exons, indicating that the SSOG is derived from a gene split event. (E) The alignment of the <em>P. pacificus</em> SSOG with <em>P. exspectatus</em> is spread over multiple reading frames. Amino acid identity between the predicted reading frame of both the proteins are marked in black and those from the other reading frame of the exspectatus gene are marked in saffron. The residues corresponding to the <em>P. pacificus</em> SSOG in different reading frames of the <em>P. exspectatus</em> sequence are also labeled in black.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The second mechanism of SSOG creation is based on splitting of an ancestral gene (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). After the split, either both or one of the fragments can diverge from the ancestral sequence and can also acquire new exons. If the fragments resulting from a gene split event are small, a moderate level of divergence can result in a failure to detect homologous sequences. The <em>P. pacificus</em> gene PP390-0.42, is an example of an SSOG created by gene split (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). Based on synteny information and spliced alignment, we mapped the first exon of this gene to the first exon of a conserved gene (PE158-0.48) in <em>P. exspectatus</em> and another <em>P. pacificus</em> gene is homologous to the remaining exons of the <em>P. exspectatus</em> gene. The <em>P. exspectatus</em> gene PE158-0.48 is the ortholog of <em>P. arcanus</em> gene PA73.-2.42 and both genes share the same first exon, which confirms that the first exon of our candidate gene is the result of a gene split event. Upon manual inspection, we found that the first exon of the <em>P. pacificus</em> SSOG has acquired insertions that shifted its reading frame and renders protein homology undetectable. Although some of the N-terminal residues are identical to the <em>P. exspectatus</em> protein (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2e</a>), the remaining residues from the first exon of our candidate gene were found to be derived from other reading frames of the orthologous <em>P. exspectatus</em> exon. Hence, it is clear that the predicted ORF from the first exon of our candidate gene is mainly derived from the non-ancestral reading frame. Moreover, the initial segment, which partially retains the ancestral ORF, is not large enough to facilitate homology detection. Ancestry of the second exon of the <em>P. pacificus</em> SSOG could not be established even after manual inspection. This suggests that the second exon has been acquired <em>de novo</em>. Thus, origin of the candidate gene can be attributed to gene split, partial ORF shift, and <em>de novo</em> acquisition of a new exon.</p></div><div id="sec-14" class="subsection"><h3>New gene creation through alternative reading frame usage</h3><p id="p-24">So far, we have discussed two mechanisms of new gene creation that require deviation from an existing gene structure but maintain the ancestral reading frame either fully or partially. Here we discuss a third mechanism that involves strand switching, which results in a completely new ORF (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The <em>P</em>. <em>pacificus</em> SSOG PP198-1.6 has two coding exons and is an example of such a mechanism. In <em>P. pacificus</em>, this gene is placed within an intron of a conserved <em>P. pacificus</em> gene (Figure S2). This intron is 2.1 kb long in <em>P. pacificus</em>. The corresponding intron of the <em>P. exspectatus</em> ortholog is 1.4 kb long and shows no homology to our candidate SSOG at the nucleotide or the protein level (Figure S2). Spliced alignment of the candidate SSOG on to the <em>P. exspectatus</em> genome did not generate any match. Thus, we performed a tblastn match against both the <em>P. exspectatus</em> and <em>P. arcanus</em> genomes at a relaxed threshold of E-value &lt; 10 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The resulting aligned genomic section was traced to a single exon of PE1052-0.1 gene whereby our candidate has some sequence identity with a reading frame from the reverse strand of the <em>P. exspectatus</em> gene (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, B and C</a>). The ortholog of PE1052-0.1 gene in <em>P. arcanus</em> (PA61-4.37) also maintains this exon and the neighboring exon-intron boundaries. Although the sequence identity between PP198-1.6 and PE1052-0.1 is not substantial (tblastn E-value = 2.37) and could be indicative of an ancient duplication event with subsequent losses, we propose that our candidate SSOG gene shares a common ancestry with sequences in <em>P. exspectatus</em> (PE1052-0.1) and <em>P. arcanus</em> (PA61-4.37) and originated from a combination of a possibly ancient duplication event, intron gain, strand-switching (Figure S2), and insertion at its current position.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Switching to an alternate reading frame gives rise to SSOGs. (A) Partial duplication in combination with an intron gain can allow opening of a new reading frame from the opposite strand. (B) The P. pacificus SSOG, PP198-1.6, is an example of both strand switching and exon splitting. Here we show amino acid identity and similarity between our candidate SSOG with the translation from P. exspectatus and P. arcanus genomes. (C) This is a two exon gene, and both the exons share a remote homology with the opposite strand of one single exon of a P. exspectatus gene at the aligned locus. The identical amino acid residues between the P. pacificus SSOG and its corresponding P. exspectatus ORF are marked in saffron. (D) The schematic overview illustrates a case of actualisation of an alternative reading frame by duplication. Overprinting describes a gene with two alternate ORFs. Gene prediction tools generally do not annotate alternate overlapping ORFs from the same strand. However, duplication might generate gene copies where the alternative ORF will be annotated. Nevertheless, in species with a single copy of this gene only one ORF gets predicted and due to lack of protein homolog in other species the alternate ORF will be categorized as SSOG. (E) PP49-3.6 is a four exon SSOG. Its P. exspectatus homolog is predicted from the same strand but in a different reading frame. Both genes maintain both ORFs. We found a P. pacificus gene, PP49-3.55, which is predicted in the P. exspectatus ORF and their identical amino acid residues are marked in turquoise between their exons and also in corresponding reading frame of our candidate SSOG. Comparison of this reading frame between the two P. pacificus genes shows two residues, in saffron, that are uniquely found in these genes. This indicates that SSOGs can be generated by prediction of an alternate ORF." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2134227042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Switching to an alternate reading frame gives rise to SSOGs. (A) Partial duplication in combination with an intron gain can allow opening of a new reading frame from the opposite strand. (B) &lt;em&gt;The P. pacificus&lt;/em&gt; SSOG, PP198-1.6, is an example of both strand switching and exon splitting. Here we show amino acid identity and similarity between our candidate SSOG with the translation from &lt;em&gt;P. exspectatus&lt;/em&gt; and &lt;em&gt;P. arcanus&lt;/em&gt; genomes. (C) This is a two exon gene, and both the exons share a remote homology with the opposite strand of one single exon of a &lt;em&gt;P. exspectatus&lt;/em&gt; gene at the aligned locus. The identical amino acid residues between the &lt;em&gt;P. pacificus&lt;/em&gt; SSOG and its corresponding &lt;em&gt;P. exspectatus&lt;/em&gt; ORF are marked in saffron. (D) The schematic overview illustrates a case of actualisation of an alternative reading frame by duplication. Overprinting describes a gene with two alternate ORFs. Gene prediction tools generally do not annotate alternate overlapping ORFs from the same strand. However, duplication might generate gene copies where the alternative ORF will be annotated. Nevertheless, in species with a single copy of this gene only one ORF gets predicted and due to lack of protein homolog in other species the alternate ORF will be categorized as SSOG. (E) PP49-3.6 is a four exon SSOG. Its &lt;em&gt;P. exspectatus&lt;/em&gt; homolog is predicted from the same strand but in a different reading frame. Both genes maintain both ORFs. We found a &lt;em&gt;P. pacificus&lt;/em&gt; gene, PP49-3.55, which is predicted in the &lt;em&gt;P. exspectatus&lt;/em&gt; ORF and their identical amino acid residues are marked in turquoise between their exons and also in corresponding reading frame of our candidate SSOG. Comparison of this reading frame between the two &lt;em&gt;P. pacificus&lt;/em&gt; genes shows two residues, in saffron, that are uniquely found in these genes. This indicates that SSOGs can be generated by prediction of an alternate ORF.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2277/F3.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2277/F3.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493512" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Switching to an alternate reading frame gives rise to SSOGs. (A) Partial duplication in combination with an intron gain can allow opening of a new reading frame from the opposite strand. (B) <em>The P. pacificus</em> SSOG, PP198-1.6, is an example of both strand switching and exon splitting. Here we show amino acid identity and similarity between our candidate SSOG with the translation from <em>P. exspectatus</em> and <em>P. arcanus</em> genomes. (C) This is a two exon gene, and both the exons share a remote homology with the opposite strand of one single exon of a <em>P. exspectatus</em> gene at the aligned locus. The identical amino acid residues between the <em>P. pacificus</em> SSOG and its corresponding <em>P. exspectatus</em> ORF are marked in saffron. (D) The schematic overview illustrates a case of actualisation of an alternative reading frame by duplication. Overprinting describes a gene with two alternate ORFs. Gene prediction tools generally do not annotate alternate overlapping ORFs from the same strand. However, duplication might generate gene copies where the alternative ORF will be annotated. Nevertheless, in species with a single copy of this gene only one ORF gets predicted and due to lack of protein homolog in other species the alternate ORF will be categorized as SSOG. (E) PP49-3.6 is a four exon SSOG. Its <em>P. exspectatus</em> homolog is predicted from the same strand but in a different reading frame. Both genes maintain both ORFs. We found a <em>P. pacificus</em> gene, PP49-3.55, which is predicted in the <em>P. exspectatus</em> ORF and their identical amino acid residues are marked in turquoise between their exons and also in corresponding reading frame of our candidate SSOG. Comparison of this reading frame between the two <em>P. pacificus</em> genes shows two residues, in saffron, that are uniquely found in these genes. This indicates that SSOGs can be generated by prediction of an alternate ORF.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The fourth mechanism deals with genes that can have more than one overlapping ORFs. This phenomenon is known as overprinting and has been reported in several studies (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grassé 1977</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ohno 1984</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Keese and Gibbs 1992</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 1997</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Makalowska <em>et al.</em> 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nekrutenko <em>et al.</em> 2005</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chung <em>et al.</em> 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gontijo <em>et al.</em> 2011</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Sabath <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guan <em>et al.</em> 2018</a>). Generally, gene prediction tools only annotate single ORFs. However, if an ancestral gene with two ORFs gets duplicated in a lineage, one of the duplicates can switch to the less common ORF (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>). This will lead to classification of the duplicated gene as an SSOG, as the corresponding ORF has not been annotated in any other species. We found that the <em>P. pacificus</em> SSOG PP49-3.6 is one candidate for such a scenario. Although it lacks protein homologs with any other species, this gene has a paralog, PP49-3.55, at the predicted transcript level (blastn E-value = 0.00, identity = 92%). The protein predicted from the candidate SSOG is in a reading frame that differs from that of its paralogous transcript. We found that both ORFs are available to both paralogs. The predicted ORF of the paralog is conserved within the genus and has its orthologous ORF in <em>P. exspectatus</em> (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3e</a>). Selection analysis indicates that the predicted <em>P. pacificus</em> ORF shows an ⍵ value of 1.6 whereas the ancestral ORF shows evidence of negative selection (⍵ = 0.38). This demonstrates how annotation artifacts such as inconsistent ORF calling can give rise to classification of genes into SSOGs. However, in the absence of conclusive evidence such as ribosome profiling data, we cannot completely reject the predicted reading frame and would point out the possibility that gene duplication in principle allows actualisation of such alternative ORFs.</p></div><div id="sec-15" class="subsection"><h3>Heuristic failures in homology detection contribute to classification as SSOGs</h3><p id="p-27">The fifth mechanism of SSOG formation specifically deals with the fact that blast programs implement a heuristic approach to find sequence matches and typically these programs are run with default settings. It is obvious that lowering thresholds (<em>e.g.</em>, E-value) or switching to a more sensitive alignment approach (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Slater and Birney 2005</a>) facilitates the identification of homologous sequences for a number of <em>P. pacificus</em> SSOGs that were missed by blast programs. This has been illustrated by the identification of homologous regions for the previously described divergence cases (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). During our investigation of high-confidence candidates, we encountered two repeat rich SSOGs, PP142-0.63 and PP81-0.14, where more detailed investigation of the syntenic region facilitated the identification of a homologous segment in the <em>P. exspectatus</em> genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Even when blast’s repeat filtering is switched off, it fails to detect homology due to the combination of a small non-repetitive match and indels as well as substitutions in the repeat-rich region (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Even though we cannot be sure, how specific this behavior is to repeat-rich genes, these two examples together with the previous examples illustrate that the failure of any heuristic approach to detect homology, will inevitably lead to the classification of certain genes with homologs as SSOGs.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Failures in homology detection lead to classification as SSOGs. (A) Conserved synteny may reveal loci where genes are incorrectly classified as SSOGs due to a failure of homology detection. (B) The P. pacificus SSOG PP142-0.63 is found in a conserved syntenic region with the P. exspectatus TROG PE68-1.70. Both proteins are &#x2018;GGX&#x2019; repeat rich proteins and share a small non-repetitive part, but blastp failed to identify both proteins as homologous." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2134227042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Failures in homology detection lead to classification as SSOGs. (A) Conserved synteny may reveal loci where genes are incorrectly classified as SSOGs due to a failure of homology detection. (B) The &lt;em&gt;P. pacificus&lt;/em&gt; SSOG PP142-0.63 is found in a conserved syntenic region with the &lt;em&gt;P. exspectatus&lt;/em&gt; TROG PE68-1.70. Both proteins are &#x2018;GGX&#x2019; repeat rich proteins and share a small non-repetitive part, but blastp failed to identify both proteins as homologous.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2277/F4.medium.gif" width="440" height="97"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2277/F4.medium.gif" width="440" height="97"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493490" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Failures in homology detection lead to classification as SSOGs. (A) Conserved synteny may reveal loci where genes are incorrectly classified as SSOGs due to a failure of homology detection. (B) The <em>P. pacificus</em> SSOG PP142-0.63 is found in a conserved syntenic region with the <em>P. exspectatus</em> TROG PE68-1.70. Both proteins are ‘GGX’ repeat rich proteins and share a small non-repetitive part, but blastp failed to identify both proteins as homologous.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Evidence for de novo genes in P. pacificus</h3><p id="p-29">All the five mechanisms described in the previous sections portray how new genes can be created from old genes. However, the <em>P. pacificus</em> SSOG PP23-6.60 is an example of <em>de novo</em> formation from an ancestrally non-coding region. It has two coding exons, placed within a single intron of the <em>P. pacificus</em> homolog of <em>C. elegans</em> <a href="http://www.wormbase.org/db/get?name=WBGene00016169;class=Gene">C27F2.7</a> (Figure S3). The intronic location of our candidate SSOG within a conserved gene helped us to identify the orthologous genomic locations in other <em>Pristionchus</em> species. Based on the spliced alignment of our candidate against the genomes of other species we were able to extract the orthologous sequences from <em>P. exspectatus</em>, <em>P. arcanus</em> and <em>P. maxplancki</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). No transcriptional evidence for the genomic regions corresponding to their extracted ORFs was found in <em>P. exspectatus</em>, <em>P. arcanus</em> and <em>P. maxplancki</em> (Figure S3). Nevertheless, the length of the <em>P. exspectatus</em> ORF matches that of the <em>P. pacificus</em> prediction. Additionally, the <em>P. arcanus</em> ORF aligns well with the <em>P. pacificus</em> ORF but contains two stop codons in the middle of the second exon. Furthermore, the sequence extracted from <em>P. maxplancki</em> has stop codons at the 11<sup>th</sup> and 14<sup>th</sup> position and no Methionine thereafter to make an abridged ORF. This suggests that the ORF at this locus was engendered in the common ancestor of <em>P. pacificus</em>, <em>P. exspectatus</em>, and <em>P. arcanus</em>. Moreover, the lack of ORF in <em>P. maxplancki</em> and alignable region in other species confirms the <em>de novo</em> origin of this gene. The protein coding nature of our <em>de novo</em> candidate was further supported by selection analysis of the <em>P. pacificus</em> ORF and the protein translation from the other species. In this analysis, we allowed each branch of the tree to have an independent ω value. Here, the branches leading from the common ancestor of <em>P. pacificus</em>, <em>P. exspectatus</em> and <em>P. arcanus</em>, toward the <em>P. pacificus</em> lineage are under extremely strong negative selection (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). This indicates that since its emergence, the <em>de novo</em> gene has been maintained as a protein coding gene in the lineage leading to <em>P. pacificus</em>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="De novo gene birth. (A) A de novo gene can originate as an antisense transcript in the intron of another gene. De novo creation of such an ORF can be verified by finding the corresponding intron in a related species that lacks this ORF. (B) PP23-6.60 is two exon P. pacificus gene that is located in an intron of another P. pacificus host gene. Based on the identification of the orthologous intron of the host gene in other species, we have created an alignment of our candidate and translation of its corresponding reading frame from other species. It is clear that the same ORF also exists in P. exspectatus. However, P. arcanus has two stop codons (*) in the middle of the 2nd exon and P. maxplancki has two stop codons in the 1st exon itself. (C) Selection analysis done on the alignment from panel B, indicates that the predicted ORF has been under strong selection toward the P. pacificus lineage. This trend may have started from the common ancestor of P. pacificus, P. exspectatus and P. arcanus. (D) A de novo gene can originate from ancestrally intergenic region. (E) The P. pacificus gene PP356-0.37 contains a single coding exon and its homologous reading frame in P. exspectatus is found at a non-transcribed intergenic location and has an early stop codon (*). This gene does not show sequence homology with any other species but P. exspectatus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2134227042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;De novo&lt;/em&gt; gene birth. (A) A &lt;em&gt;de novo&lt;/em&gt; gene can originate as an antisense transcript in the intron of another gene. &lt;em&gt;De novo&lt;/em&gt; creation of such an ORF can be verified by finding the corresponding intron in a related species that lacks this ORF. (B) PP23-6.60 is two exon &lt;em&gt;P. pacificus&lt;/em&gt; gene that is located in an intron of another &lt;em&gt;P. pacificus&lt;/em&gt; host gene. Based on the identification of the orthologous intron of the host gene in other species, we have created an alignment of our candidate and translation of its corresponding reading frame from other species. It is clear that the same ORF also exists in &lt;em&gt;P. exspectatus&lt;/em&gt;. However, &lt;em&gt;P. arcanus&lt;/em&gt; has two stop codons (*) in the middle of the 2&lt;sup&gt;nd&lt;/sup&gt; exon and &lt;em&gt;P. maxplancki&lt;/em&gt; has two stop codons in the 1&lt;sup&gt;st&lt;/sup&gt; exon itself. (C) Selection analysis done on the alignment from panel B, indicates that the predicted ORF has been under strong selection toward the &lt;em&gt;P. pacificus&lt;/em&gt; lineage. This trend may have started from the common ancestor of &lt;em&gt;P. pacificus&lt;/em&gt;, &lt;em&gt;P. exspectatus&lt;/em&gt; and &lt;em&gt;P. arcanus&lt;/em&gt;. (D) A &lt;em&gt;de novo&lt;/em&gt; gene can originate from ancestrally intergenic region. (E) The &lt;em&gt;P. pacificus&lt;/em&gt; gene PP356-0.37 contains a single coding exon and its homologous reading frame in &lt;em&gt;P. exspectatus&lt;/em&gt; is found at a non-transcribed intergenic location and has an early stop codon (*). This gene does not show sequence homology with any other species but &lt;em&gt;P. exspectatus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2277/F5.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2277/F5.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2277/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493661" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child"><em>De novo</em> gene birth. (A) A <em>de novo</em> gene can originate as an antisense transcript in the intron of another gene. <em>De novo</em> creation of such an ORF can be verified by finding the corresponding intron in a related species that lacks this ORF. (B) PP23-6.60 is two exon <em>P. pacificus</em> gene that is located in an intron of another <em>P. pacificus</em> host gene. Based on the identification of the orthologous intron of the host gene in other species, we have created an alignment of our candidate and translation of its corresponding reading frame from other species. It is clear that the same ORF also exists in <em>P. exspectatus</em>. However, <em>P. arcanus</em> has two stop codons (*) in the middle of the 2<sup>nd</sup> exon and <em>P. maxplancki</em> has two stop codons in the 1<sup>st</sup> exon itself. (C) Selection analysis done on the alignment from panel B, indicates that the predicted ORF has been under strong selection toward the <em>P. pacificus</em> lineage. This trend may have started from the common ancestor of <em>P. pacificus</em>, <em>P. exspectatus</em> and <em>P. arcanus</em>. (D) A <em>de novo</em> gene can originate from ancestrally intergenic region. (E) The <em>P. pacificus</em> gene PP356-0.37 contains a single coding exon and its homologous reading frame in <em>P. exspectatus</em> is found at a non-transcribed intergenic location and has an early stop codon (*). This gene does not show sequence homology with any other species but <em>P. exspectatus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Our second <em>de novo</em> candidate PP356-0.37 is a two exon gene with its entire coding sequence in the 2<sup>nd</sup> exon. Since the candidate could be mapped on to the genomes of none of the other species but <em>P. exspectatus</em>, we were only able to extract the orthologous <em>P. exspectatus</em> sequence from a conserved syntenic region (Figure S4). Nevertheless, the absence of transcription in <em>P. exspectatus</em> and the presence of a stop codon at the 4<sup>th</sup> position of the extracted <em>P. exspectatus</em> sequence confirms the non-genic and non-transcribed status of the <em>P. exspectatus</em> sequence. Even though the absence of homologous traces outside the two sister species did not allow us to conclusively infer the state of this gene in the ancestor of <em>P. pacificus</em> and <em>P. exspectatus</em>, we propose that the <em>P. pacificus</em> SSOG PP356-0.37 arose very recently and is a putative <em>de novo</em> gene that emerged from a previously non-coding intergenic region in the <em>P. pacificus</em> lineage. Together with a recent study of the <em>Caenorhabditis</em> genus (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zhang <em>et al.</em> 2019</a>), these genes are the first examples of <em>de novo</em> genes in nematodes.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-32">Genome sequencing projects identify novel genes in all domains of life. Many of these genes have been shown to be involved in lineage specific adaptations (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Milde <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kawasaki <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mayer <em>et al.</em> 2015</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Villanueva-Canas <em>et al.</em> 2016</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilera <em>et al.</em> 2017</a>). However, even with deep taxonomic sampling of genomic data sets, it remains unclear, what are the most common mechanisms to form novel genes. Given, that yeasts, mammals, insects, and nematodes have highly variable genomic architectures (<em>e.g.</em>, genome size and fraction of coding sequences (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rödelsperger <em>et al.</em> 2013</a>), presence of operons (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Sinha <em>et al.</em> 2014</a>), recombination (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Srinivasan <em>et al.</em> 2002</a>), transposon control (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sarkies <em>et al.</em> 2015</a>), and DNA methylation (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rošić <em>et al.</em> 2018</a>)), multiple studies in different clades are needed to characterize and compare processes that lead to emergence of novel genes. In this study, we bring the power of clade genomics to enumerate various mechanisms of gene birth in <em>Pristionchus</em> nematodes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rogers 2018</a>), this makes our study the first of its kind in nematodes.</p><p id="p-33">The exceptionally high number of SSOGs (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1, B and C</a>) may be due to a combination of erroneous gene models and short-lived gene-like sequences that result from pervasive transcription and translation (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Hangauer <em>et al.</em> 2013</a>; <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Schmitz <em>et al.</em> 2018</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Ruiz-Orera and Mar Albà 2019</a>). We discussed two cases, where either wrong ORF annotation or heuristic failure in homology detection resulted in an incorrect classification as SSOGs. However, as most SSOGs are only poorly supported by expression data, it is challenging to conclusively distinguish annotation artifacts from lowly expressed genes. Thus, it remains unclear, to what extent annotation errors and pervasive transcription and translation contribute to the abundance of SSOGs. The deep taxon sampling of our phylogenomic data allowed us to detect traces of homology for 1082 (61%) of <em>P. pacificus</em> SSOGs. This demonstrates the potential of the <em>Pristionchus</em> system to study the mechanisms of gene birth. However, in contrast to many other recent studies in mammals (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Ruiz-Orera <em>et al.</em> 2018</a>; <a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Schmitz <em>et al.</em> 2018</a>), insects (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Klasberg <em>et al.</em> 2018</a>) and yeasts (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Carvunis <em>et al.</em> 2012</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Vakirlis <em>et al.</em> 2018</a>), we did not aim only for the identification and characterization of <em>de novo</em> genes, but undertook an unbiased exploration of high-confidence candidates through manual investigation of gene structures and various sequence search methods (<em>i.e.</em>, blastn, tblastn, and spliced-mapping with exonerate) in our phylogenomic data set. In some cases, this lead to a reclassification of an SSOG as TROG, but more importantly this demonstrated that both divergence of existing genic segments and <em>de novo</em> creation of new genic elements contribute to orphan gene emergence. While <em>de novo</em> origin only requires the identification of an ancestrally non-coding sequence in a closely related species, the case of the <em>P. pacificus</em> SSOG PP390-0.42 (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, C, D and E</a>) shows that the distinction between <em>de novo</em> and divergence can sometimes be difficult to discern. The formation of this gene results from several steps, which include splitting of the ancestral gene, sequence divergence, reading frame shift and <em>de novo</em> acquisition of a new exon. Thus, we argue that this gene should be considered a product of ‘mixed origin mechanism’, as both divergence and <em>de novo</em> origin mechanisms have contributed to its birth. Moreover, in order to quantify the contribution of different origin mechanisms, we first have to establish a comprehensive catalog with detailed descriptions of all possible mechanisms and then develop computational tools to reliably detect them. This knowledge can be used in future to perform more systematic screens, possibly also on older phylostrata, in order to get better estimates of the relative contribution of various mechanisms to novel gene emergence.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">The authors would like to thank Bogdan Sieriebriennikov and Prof. Ralf Sommer for discussions. The authors would like to thank Dr. James Lightfoot for proofreading the manuscript. The work was funded by the Max Planck Society.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-35">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8123768">https://doi.org/10.25387/g3.8123768</a>.</p></li><li class="fn" id="fn-2"><p id="p-36"><em>Communicating editor: M. Rockman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-04"><span class="received-label">Received </span>February 4, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-11"><span class="accepted-label">Accepted </span>May 11, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Prabh, Rodelsperger</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilera</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDougall</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degnan</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Co-option and <em>de novo</em> gene evolution underlie molluscan shell diversity.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.2" data-doi="10.1186/s12862-015-0466-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baskaran</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serobyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markov</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ancient gene duplications have shaped developmental stage-specific expression in <em>Pristionchus pacificus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12862-015-0466-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D15%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-015-0466-2%26rft_id%253Dinfo%253Apmid%252F26370559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-015-0466-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26370559&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.3" data-doi="10.1101/gr.103119.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borchert</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dieterich</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krug</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schutz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Proteogenomics of <em>Pristionchus pacificus</em> reveals distinct proteome structure of nematode models.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.103119.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.103119.109%26rft_id%253Dinfo%253Apmid%252F20237107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvODM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.4" data-doi="10.1534/genetics.107.084491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>De novo</em> origination of a new protein-coding gene in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCai%26rft.auinit1%253DJ.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D487%26rft.epage%253D496%26rft.atitle%253DDe%2BNovo%2BOrigination%2Bof%2Ba%2BNew%2BProtein-Coding%2BGene%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084491%26rft_id%253Dinfo%253Apmid%252F18493065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvNDg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.5" data-doi="10.1038/nature11184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvunis</span>  <span class="cit-name-given-names">A.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolland</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wapinski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calderwood</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yildirim</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Proto-genes and <em>de novo</em> gene birth.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">487</span>: <span class="cit-fpage">370</span>–<span class="cit-lpage">374</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11184</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCarvunis%26rft.auinit1%253DA.%2BR.%26rft.volume%253D487%26rft.issue%253D7407%26rft.spage%253D370%26rft.epage%253D374%26rft.atitle%253DProto-genes%2Band%2Bde%2Bnovo%2Bgene%2Bbirth.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11184%26rft_id%253Dinfo%253Apmid%252F22722833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22722833&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306506500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.6" data-doi="10.1073/pnas.94.8.3811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeVries</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Evolution of antifreeze glycoprotein gene from a trypsinogen gene in Antarctic notothenioid fish.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3811</span>–<span class="cit-lpage">3816</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.94.8.3811</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DChen%26rft.auinit1%253DL.%26rft.volume%253D94%26rft.issue%253D8%26rft.spage%253D3811%26rft.epage%253D3816%26rft.atitle%253DEvolution%2Bof%2Bantifreeze%2Bglycoprotein%2Bgene%2Bfrom%2Ba%2Btrypsinogen%2Bgene%2Bin%2BAntarctic%2Bnotothenioid%2Bfish%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.8.3811%26rft_id%253Dinfo%253Apmid%252F9108060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NC84LzM4MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjI3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.7" data-doi="10.1038/nrg3521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krinsky</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">New genes as drivers of phenotypic evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">660. Erratum: 744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D645%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3521%26rft_id%253Dinfo%253Apmid%252F23949544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23949544&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.8" data-doi="10.1371/journal.pgen.1005391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Q. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">W.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Emergence, retention and selection: a trilogy of origination for functional <em>de novo</em> proteins from ancestral LncRNAs in Primates.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005391</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005391%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005391%26rft_id%253Dinfo%253Apmid%252F26177073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26177073&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.9" data-doi="10.1371/journal.pcbi.0030091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">W.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wadhawan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A first look at ARFome: dual-coding genes in mammalian genomes.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.0030091</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DChung%26rft.auinit1%253DW.%2BY.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De91%26rft.epage%253De91%26rft.atitle%253DA%2Bfirst%2Blook%2Bat%2BARFome%253A%2Bdual-coding%2Bgenes%2Bin%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0030091%26rft_id%253Dinfo%253Apmid%252F17511511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0030091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17511511&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.10" data-doi="10.1186/1471-2148-8-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delaye</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deluna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazcano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becerra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The origin of a novel gene through overprinting in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-8-31</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDelaye%26rft.auinit1%253DL.%26rft.volume%253D8%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DThe%2Borigin%2Bof%2Ba%2Bnovel%2Bgene%2Bthrough%2Boverprinting%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-31%26rft_id%253Dinfo%253Apmid%252F18226237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18226237&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.11" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.12" data-doi="10.1016/j.tig.2007.08.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Domazet-Loso</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brajković</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A phylostratigraphy approach to uncover the genomic history of major adaptations in metazoan lineages.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">539</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2007.08.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDomazet-Loso%26rft.auinit1%253DT.%26rft.volume%253D23%26rft.issue%253D11%26rft.spage%253D533%26rft.epage%253D539%26rft.atitle%253DA%2Bphylostratigraphy%2Bapproach%2Bto%2Buncover%2Bthe%2Bgenomic%2Bhistory%2Bof%2Bmajor%2Badaptations%2Bin%2Bmetazoan%2Blineages.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.08.014%26rft_id%253Dinfo%253Apmid%252F18029048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.08.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18029048&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251533600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.13" data-doi="10.1016/0168-9525(96)10027-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The yeast genome project: what did we learn?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0168-9525(96)10027-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D263%26rft.epage%253D270%26rft.atitle%253DThe%2Byeast%2Bgenome%2Bproject%253A%2Bwhat%2Bdid%2Bwe%2Blearn%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9525%252896%252910027-5%26rft_id%253Dinfo%253Apmid%252F8763498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9525(96)10027-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8763498&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UW03100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.14" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.15" data-doi="10.1038/ncomms1328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gontijo</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miguela</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whiting</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominguez</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Intron retention in the <em>Drosophila melanogaster</em> Rieske Iron Sulphur Protein gene generated a new protein.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1328</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DGontijo%26rft.auinit1%253DA.%2BM.%26rft.volume%253D2%26rft.spage%253D323%26rft.epage%253D323%26rft.atitle%253DIntron%2Bretention%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2BRieske%2BIron%2BSulphur%2BProtein%2Bgene%2Bgenerated%2Ba%2Bnew%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1328%26rft_id%253Dinfo%253Apmid%252F21610726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21610726&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.16" data-doi="10.1093/molbev/msp259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SeaView Version 4: A multiplatform graphical user interface for sequence alignment and phylogenetic tree building.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msp259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp259%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854763&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273704400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2277.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grassé</span>  <span class="cit-name-given-names">P.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-source">Evolution of Living Organisms: Evidence for a New Theory of Transformation</span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.18" data-doi="10.1038/s41467-018-04025-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Gene refashioning through innovative shifting of reading frames in mosses.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-04025-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D1555%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-04025-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-04025-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.19" data-doi="10.1371/journal.pgen.1003569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hangauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McManus</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pervasive transcription of the human genome produces thousands of previously unidentified long intergenic noncoding RNAs.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003569</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003569%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003569%26rft_id%253Dinfo%253Apmid%252F23818866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003569&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23818866&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.20" data-doi="10.1016/j.cub.2009.07.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heinen</span>  <span class="cit-name-given-names">T. J. A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staubach</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Häming</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Emergence of a new gene from an intergenic region.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1527</span>–<span class="cit-lpage">1531</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2009.07.049</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHeinen%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D1527%26rft.epage%253D1531%26rft.atitle%253DEmergence%2Bof%2Ba%2Bnew%2Bgene%2Bfrom%2Ban%2Bintergenic%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.07.049%26rft_id%253Dinfo%253Apmid%252F19733073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.07.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19733073&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270400400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.21" data-doi="10.1126/science.860134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Evolution and tinkering.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1161</span>–<span class="cit-lpage">1166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.860134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJacob%26rft.auinit1%253DF%26rft.volume%253D196%26rft.issue%253D4295%26rft.spage%253D1161%26rft.epage%253D1166%26rft.atitle%253DEvolution%2Band%2Btinkering%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.860134%26rft_id%253Dinfo%253Apmid%252F860134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTM6IjE5Ni80Mjk1LzExNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjI3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.22" data-doi="10.1038/nature09616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jan</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham Ruby</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Formation, regulation and evolution of <em>Caenorhabditis elegans</em> 3′UTRs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">469</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D469%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09616%26rft_id%253Dinfo%253Apmid%252F21085120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085120&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285921600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.23" data-doi="10.2108/zsj.29.403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanzaki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragsdale</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Description of three <em>Pristionchus</em> species (Nematoda: Diplogastridae) from Japan that form a cryptic species complex with the model organism <em>P. pacificus</em>.</span> <abbr class="cit-jnl-abbrev">Zool. Sci.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2108/zsj.29.403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DKanzaki%26rft.auinit1%253DN.%26rft.volume%253D29%26rft.issue%253D6%26rft.spage%253D403%26rft.epage%253D417%26rft.atitle%253DDescription%2Bof%2Bthree%2BPristionchus%2Bspecies%2B%2528Nematoda%253A%2BDiplogastridae%2529%2Bfrom%2BJapan%2Bthat%2Bform%2Ba%2Bcryptic%2Bspecies%2Bcomplex%2Bwith%2Bthe%2Bmodel%2Borganism%2BP.%2Bpacificus.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.29.403%26rft_id%253Dinfo%253Apmid%252F22639812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.29.403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22639812&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.24" data-doi="10.1093/molbev/msr020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawasaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafont</span>  <span class="cit-name-given-names">A.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sire</span>  <span class="cit-name-given-names">J.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The evolution of milk casein genes from tooth genes before the origin of mammals.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2053</span>–<span class="cit-lpage">2061</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr020%26rft_id%253Dinfo%253Apmid%252F21245413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21245413&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291756400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.25" data-doi="10.1073/pnas.89.20.9489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keese</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Origins of genes: “Big Bang” or continuous creation?</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">9489</span>–<span class="cit-lpage">9493</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.89.20.9489</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKeese%26rft.auinit1%253DP.%2BK.%26rft.volume%253D89%26rft.issue%253D20%26rft.spage%253D9489%26rft.epage%253D9493%26rft.atitle%253DOrigins%2Bof%2Bgenes%253A%2B%2522big%2Bbang%2522%2Bor%2Bcontinuous%2Bcreation%253F%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.20.9489%26rft_id%253Dinfo%253Apmid%252F1329098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODkvMjAvOTQ4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMjc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.26" data-doi="10.1016/j.tig.2009.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khalturin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemmrich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraune</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Augustin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosch</span>  <span class="cit-name-given-names">T. C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">More than just orphans: are taxonomically-restricted genes important in evolution?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2009.07.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DKhalturin%26rft.auinit1%253DK.%26rft.volume%253D25%26rft.issue%253D9%26rft.spage%253D404%26rft.epage%253D413%26rft.atitle%253DMore%2Bthan%2Bjust%2Borphans%253A%2Bare%2Btaxonomically-restricted%2Bgenes%2Bimportant%2Bin%2Bevolution%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.07.006%26rft_id%253Dinfo%253Apmid%252F19716618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19716618&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270185900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.27" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-4-r36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.28" data-doi="10.1111/febs.14504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klasberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitard-Feildel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bornberg-Bauer</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Origins and structural properties of novel and <em>de novo</em> protein domains during insect evolution.</span> <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">2605</span>–<span class="cit-lpage">2625</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/febs.14504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJ.%26rft.volume%253D285%26rft.spage%253D2605%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ffebs.14504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/febs.14504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.29" data-doi="10.1101/gr.095026.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLysaght</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recent de novo origin of human protein-coding genes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1752</span>–<span class="cit-lpage">1759</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.095026.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.095026.109%26rft_id%253Dinfo%253Apmid%252F19726446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xNzUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.30" data-doi="10.1073/pnas.0509809103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindfors</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Novel genes derived from noncoding DNA in Drosophila melanogaster are frequently X-linked and exhibit testis-biased expression.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">9935</span>–<span class="cit-lpage">9939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0509809103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0509809103%26rft_id%253Dinfo%253Apmid%252F16777968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzI2Lzk5MzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjI3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.31" data-doi="10.1371/journal.pcbi.1000734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A human-specific <em>de novo</em> protein-coding gene associated with human brain functions.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1000734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1000734</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DLi%26rft.auinit1%253DC.%2BY.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De1000734%26rft.epage%253De1000734%26rft.atitle%253DA%2Bhuman-specific%2Bde%2Bnovo%2Bprotein-coding%2Bgene%2Bassociated%2Bwith%2Bhuman%2Bbrain%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000734%26rft_id%253Dinfo%253Apmid%252F20376170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000734&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20376170&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.32" data-doi="10.1038/nrg1204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betrán</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The origin of new genes: glimpses from the young and old.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">875</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLong%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D865%26rft.epage%253D875%26rft.atitle%253DThe%2Borigin%2Bof%2Bnew%2Bgenes%253A%2Bglimpses%2Bfrom%2Bthe%2Byoung%2Band%2Bold.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1204%26rft_id%253Dinfo%253Apmid%252F14634634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14634634&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186325400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.33" data-doi="10.1016/j.compbiolchem.2004.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makalowska</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makalowski</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Overlapping genes in vertebrate genomes.</span> <abbr class="cit-jnl-abbrev">Comput. Biol. Chem.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.compbiolchem.2004.12.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2Bbiology%2Band%2Bchemistry%26rft.stitle%253DComput%2BBiol%2BChem%26rft.aulast%253DMakalowska%26rft.auinit1%253DI.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DOverlapping%2Bgenes%2Bin%2Bvertebrate%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compbiolchem.2004.12.006%26rft_id%253Dinfo%253Apmid%252F15680581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compbiolchem.2004.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15680581&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227611200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.34" data-doi="10.1371/journal.pgen.1005146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riebesell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The orphan gene dauerless regulates dauer development and intraspecific competition in nematodes by copy Number Variation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005146%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005146%26rft_id%253Dinfo%253Apmid%252F26087034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26087034&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.35" data-doi="10.1038/nrg.2016.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLysaght</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Open questions in the study of <em>de novo</em> genes: what, how and why.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2016.78</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.78%26rft_id%253Dinfo%253Apmid%252FWOS%253A00038151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=WOS:00038151&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.36" data-doi="10.1186/gb-2009-10-1-r8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemmrich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anton-Erxleben</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khalturin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittlieb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterization of taxonomically restricted genes in a phylum-restricted cell type.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2009-10-1-r8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMilde%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253DR8%26rft.epage%253DR8%26rft.atitle%253DCharacterization%2Bof%2Btaxonomically%2Brestricted%2Bgenes%2Bin%2Ba%2Bphylum-restricted%2Bcell%2Btype.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-1-r8%26rft_id%253Dinfo%253Apmid%252F19161630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-1-r8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19161630&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.37" data-doi="10.1371/journal.pone.0048650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLysaght</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>De novo</em> origin of protein-coding genes in murine rodents.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e48650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0048650</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMurphy%26rft.auinit1%253DD.%2BN.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48650%26rft.epage%253De48650%26rft.atitle%253DDe%2Bnovo%2Borigin%2Bof%2Bprotein-coding%2Bgenes%2Bin%2Bmurine%2Brodents.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048650%26rft_id%253Dinfo%253Apmid%252F23185269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185269&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.38" data-doi="10.1371/journal.pgen.0010018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wadhawan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goetting-Minesky</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makova</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Oscillating evolution of a mammalian locus with overlapping reading frames: an XLalphas/ALEX relay.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">e18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0010018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNekrutenko%26rft.auinit1%253DA.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253De18%26rft.epage%253De18%26rft.atitle%253DOscillating%2Bevolution%2Bof%2Ba%2Bmammalian%2Blocus%2Bwith%2Boverlapping%2Breading%2Bframes%253A%2Ban%2BXLalphas%252FALEX%2Brelay.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010018%26rft_id%253Dinfo%253Apmid%252F16110341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16110341&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.39" data-doi="10.1073/pnas.81.8.2421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Birth of a unique enzyme from an alternative reading frame of the preexisted, internally repetitious coding sequence.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">2421</span>–<span class="cit-lpage">2425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.81.8.2421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOhno%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D8%26rft.spage%253D2421%26rft.epage%253D2425%26rft.atitle%253DBirth%2Bof%2Ba%2Bunique%2Benzyme%2Bfrom%2Ban%2Balternative%2Breading%2Bframe%2Bof%2Bthe%2Bpreexisted%252C%2Binternally%2Brepetitious%2Bcoding%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.8.2421%26rft_id%253Dinfo%253Apmid%252F6585807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4MS84LzI0MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjI3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.40" data-doi="10.7554/eLife.01311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmieri</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosiol</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The life cycle of <em>Drosophila</em> orphan genes.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e01311</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.01311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01311%26rft_id%253Dinfo%253Apmid%252F24554240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.01311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24554240&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.41" data-doi="10.1186/s12859-016-1102-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prabh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Are orphan genes protein-coding, prediction artifacts, or non-coding RNAs?</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1102-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1102-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1102-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.42" data-doi="10.1101/gr.234971.118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prabh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeseler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Deep taxon sampling reveals the evolutionary dynamics of novel gene families in <em>Pristionchus</em> nematodes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1664</span>–<span class="cit-lpage">1674</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.234971.118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.234971.118%26rft_id%253Dinfo%253Apmid%252F30232197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMS8xNjY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.43" data-doi="10.1371/journal.pone.0008861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dieterich</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">CYNTENATOR: progressive gene order alignment of 17 vertebrate genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e8861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0008861</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRodelsperger%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De8861%26rft.epage%253De8861%26rft.atitle%253DCYNTENATOR%253A%2Bprogressive%2Bgene%2Border%2Balignment%2Bof%2B17%2Bvertebrate%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008861%26rft_id%253Dinfo%253Apmid%252F20126624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20126624&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.44" data-doi="10.1186/s12862-016-0740-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menden</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serobyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baskaran</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">First insights into the nature and evolution of antisense transcription in nematodes.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12862-016-0740-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D16%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-016-0740-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-016-0740-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.45" data-doi="10.1016/j.celrep.2017.09.077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bemm</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Single-molecule sequencing reveals the chromosome-scale genomic architecture of the nematode model organism <em>Pristionchus pacificus</em>.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">834</span>–<span class="cit-lpage">844</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2017.09.077</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D21%26rft.spage%253D834%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2017.09.077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2017.09.077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.46" data-doi="10.1534/genetics.113.159855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weller</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of genetic diversity in the nematode <em>Pristionchus pacificus</em> from population-scale resequencing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.159855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159855%26rft_id%253Dinfo%253Apmid%252F24443445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzExNTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjI3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.47" data-doi="10.1016/j.cub.2018.07.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röseler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Phylotranscriptomics of <em>Pristionchus</em> nematodes reveals parallel gene loss in six hermaphroditic lineages.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3123</span>–<span class="cit-lpage">3127.e5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2018.07.041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D28%26rft.spage%253D3123%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2018.07.041%26rft_id%253Dinfo%253Apmid%252F30245109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2018.07.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30245109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2277.48" data-doi="10.1002/9780470015902.a0024603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streit</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Structure, function and evolution of the nematode genome</span> in <span class="cit-source">eLS</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Ltd</span>, <span class="cit-publ-loc">Chichester</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9780470015902.a0024603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLS%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470015902.a0024603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470015902.a0024603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.49" data-doi="10.1186/s13059-018-1500-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Adding resolution and dimensionality to comparative genomics: moving from reference genomes to clade genomics.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-018-1500-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D19%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-018-1500-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-018-1500-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.50" data-doi="10.1038/s41588-018-0061-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rošić</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amouroux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Requena</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emperle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Evolutionary analysis indicates that DNA alkylation damage is a byproduct of cytosine DNA methyltransferase activity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">459</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0061-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D452%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0061-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0061-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.51" data-doi="10.1016/j.tig.2018.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Orera</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mar Albà</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Translation of small open reading frames: roles in regulation and evolutionary innovation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2018.12.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D35%26rft.spage%253D186%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2018.12.003%26rft_id%253Dinfo%253Apmid%252F30606460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2018.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30606460&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.52" data-doi="10.1038/s41559-018-0506-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Orera</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdaguer-Grau</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva-Cañas</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messeguer</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mar Albà</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Translation of neutrally evolving peptides provides a basis for de novo gene evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">890</span>–<span class="cit-lpage">896</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-018-0506-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft.volume%253D2%26rft.spage%253D890%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-018-0506-6%26rft_id%253Dinfo%253Apmid%252F29556078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-018-0506-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29556078&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.53" data-doi="10.1093/molbev/mss179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabath</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of viral proteins originated de novo by overprinting.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3767</span>–<span class="cit-lpage">3780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mss179</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss179%26rft_id%253Dinfo%253Apmid%252F22821011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22821011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310970000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.54" data-doi="10.1371/journal.pbio.1002061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarkies</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selkirk</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blok</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boothby</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ancient and novel small RNA pathways compensate for the loss of piRNAs in multiple independent nematode lineages.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e1002061</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1002061</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002061%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002061%26rft_id%253Dinfo%253Apmid%252F25668728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25668728&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The evolutionary analysis of “orphans” from the Drosophila genome identifies rapidly diverging and incorrectly annotated genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchmid%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D589%26rft.epage%253D598%26rft.atitle%253DThe%2BEvolutionary%2BAnalysis%2Bof%2B%2522Orphans%2522%2BFrom%2Bthe%2BDrosophila%2BGenome%2BIdentifies%2BRapidly%2BDiverging%2Band%2BIncorrectly%2BAnnotated%2BGenes%26rft_id%253Dinfo%253Apmid%252F11606536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNTg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.56" data-doi="10.1073/pnas.94.18.9746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A screen for fast evolving genes from <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">9746</span>–<span class="cit-lpage">9750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.94.18.9746</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSchmid%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D94%26rft.issue%253D18%26rft.spage%253D9746%26rft.epage%253D9750%26rft.atitle%253DA%2Bscreen%2Bfor%2Bfast%2Bevolving%2Bgenes%2Bfrom%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.18.9746%26rft_id%253Dinfo%253Apmid%252F9275195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTgvOTc0NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMjc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.57" data-doi="10.1038/s41559-018-0639-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ullrich</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bornberg-Bauer</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Incipient <em>de novo</em> genes can evolve from frozen accidents that escaped rapid transcript turnover.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-018-0639-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft.volume%253D2%26rft.spage%253D1626%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-018-0639-7%26rft_id%253Dinfo%253Apmid%252F30201962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-018-0639-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30201962&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.58" data-doi="10.1261/rna.041954.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langnick</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dieterich</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide analysis of trans-splicing in the nematode <em>Pristionchus pacificus</em> unravels conserved gene functions for germline and dauer development in divergent operons.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1386</span>–<span class="cit-lpage">1397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.041954.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.041954.113%26rft_id%253Dinfo%253Apmid%252F25015138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTM4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMjc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.59" data-doi="10.1186/1471-2105-6-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">G. S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Automated generation of heuristics for biological sequence comparison.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-6-31</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSlater%26rft.auinit1%253DG.%2BS.%26rft.volume%253D6%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DAutomated%2Bgeneration%2Bof%2Bheuristics%2Bfor%2Bbiological%2Bsequence%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-31%26rft_id%253Dinfo%253Apmid%252F15713233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713233&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2277.60" data-doi="10.1163/9789004260306"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite> (Editor), <span class="cit-pub-date">2015</span> <span class="cit-source">Pristionchus pacificus: A Nematode Model for Comparative and Evolutionary Biology</span>, <span class="cit-publ-name">BRILL</span>, <span class="cit-publ-loc">Leiden</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1163/9789004260306</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPristionchus%2Bpacificus%253A%2BA%2BNematode%2BModel%2Bfor%2BComparative%2Band%2BEvolutionary%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1163%252F9789004260306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1163/9789004260306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.61" data-doi="10.1006/dbio.1996.0035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Evolution of nematode vulval fate patterning.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">407</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.1996.0035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSommer%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D173%26rft.issue%253D2%26rft.spage%253D396%26rft.epage%253D407%26rft.atitle%253DEvolution%2Bof%2Bnematode%2Bvulval%2Bfate%2Bpatterning.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.1996.0035%26rft_id%253Dinfo%253Apmid%252F8606000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.1996.0035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8606000&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TU64600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srinivasan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brand</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nandakumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A bacterial artificial chromosome-based genetic linkage map of the nematode <em>Pristionchus pacificus</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSrinivasan%26rft.auinit1%253DJ.%26rft.volume%253D162%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D134%26rft.atitle%253DA%2BBacterial%2BArtificial%2BChromosome-Based%2BGenetic%2BLinkage%2BMap%2Bof%2Bthe%2BNematode%2BPristionchus%2Bpacificus%26rft_id%253Dinfo%253Apmid%252F12242228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYyLzEvMTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.63" data-doi="10.1038/s41588-018-0040-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copetti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zwickl</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomes of 13 domesticated and wild rice relatives highlight genetic conservation, turnover and innovation across the genus Oryza.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">296</span>. Erratum: 1608.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0040-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0040-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0040-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.64" data-doi="10.1093/nar/gkl315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torrents</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W609</span>–<span class="cit-lpage">W612</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSuyama%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW609%26rft.epage%253DW612%26rft.atitle%253DPAL2NAL%253A%2Brobust%2Bconversion%2Bof%2Bprotein%2Bsequence%2Balignments%2Binto%2Bthe%2Bcorresponding%2Bcodon%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl315%26rft_id%253Dinfo%253Apmid%252F16845082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845082&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200122&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.65" data-doi="10.1038/nrg3053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domazet-Lošo</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The evolutionary origin of orphan genes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">702</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DTautz%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D692%26rft.epage%253D702%26rft.atitle%253DThe%2Bevolutionary%2Borigin%2Bof%2Borphan%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3053%26rft_id%253Dinfo%253Apmid%252F21878963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878963&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.66" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbs017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.67" data-doi="10.1093/molbev/msx315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vakirlis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebert</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opulente</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achaz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A Molecular Portrait of De Novo Genes in Yeasts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D631%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2277.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villanueva-Canas</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Orera</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel Agea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> New genes and functional innovation in mammals.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.69" data-doi="10.1101/gr.234872.118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sieriebriennikov</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loschko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Young genes have distinct gene structure, epigenetic profiles, and transcriptional regulation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1675</span>–<span class="cit-lpage">1687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.234872.118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.234872.118%26rft_id%253Dinfo%253Apmid%252F30232198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMS8xNjc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.70" data-doi="10.1093/gbe/evt009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wissler</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simola</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helmkampf</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bornberg-Bauer</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mechanisms and dynamics of orphan gene emergence in insect genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evt009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt009%26rft_id%253Dinfo%253Apmid%252F23348040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23348040&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.71" data-doi="10.1371/journal.pgen.1002379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">D.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>De novo</em> origin of human protein-coding genes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DD.%2BD.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002379%26rft.epage%253De1002379%26rft.atitle%253DDe%2Bnovo%2Borigin%2Bof%2Bhuman%2Bprotein-coding%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002379%26rft_id%253Dinfo%253Apmid%252F22102831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22102831&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.72" data-doi="10.1371/journal.pgen.1002942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">W.-Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hominoid-specific <em>de novo</em> protein-coding genes originating from long non-coding RNAs.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002942</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002942</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DXie%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253De1002942%26rft.epage%253De1002942%26rft.atitle%253DHominoid-specific%2Bde%2Bnovo%2Bprotein-coding%2Bgenes%2Boriginating%2Bfrom%2Blong%2Bnon-coding%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002942%26rft_id%253Dinfo%253Apmid%252F23028352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028352&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.73" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2277.74" data-doi="10.1007/s11427-019-9482-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Origination and evolution of orphan genes and <em>de novo</em> genes in the genome of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Sci. China Life Sci.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">593</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11427-019-9482-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BChina%2BLife%2BSci.%26rft.volume%253D62%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11427-019-9482-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11427-019-9482-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2277.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2267" title="Expression Signatures of Cisplatin- and Trametinib-Treated Early-Stage Medaka Melanomas" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2287" title="Restriction of Cellular Plasticity of Differentiated Cells Mediated by Chromatin Modifiers, Transcription Factors and Protein Kinases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2277" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Agrcakjcr7aKNpu9seXK7iVaVUFP3gIxA4NIm_OOd5g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2277" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2277" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2277" class="active" data-icon-position="" data-hide-link-title="0">De Novo, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in Pristionchus Nematodes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493459" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LEROVPMXIZybfeQwy-NTJw-An7yOr-Ec572kBATojNg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2277.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2277" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JqQ1EuJNPxiWUTofNVkD5wryAnllrVVrhvqeuZ5le2k" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2277" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2277.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493459" id="node493459--2" data-pisa="ggg;9/7/2277" data-pisa-master="ggg;g3.119.400326" data-apath="/ggg/9/7/2277.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em>, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in <em>Pristionchus</em> Nematodes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3099-1007" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Neel Prabh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7905-9675" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian Rödelsperger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2277-2286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400326 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493459--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Neel Prabh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Genetics, Max-Planck-Institute for Evolutionary Biology, August Thienemann Str. 2, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neel%2BPrabh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prabh%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANeel%2BPrabh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3099-1007" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Neel Prabh</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian Rödelsperger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BR%C3%B6delsperger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6delsperger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BR%25C3%25B6delsperger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7905-9675" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian Rödelsperger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.roedelsperger@tuebingen.mpg.de" class="" data-icon-position="" data-hide-link-title="0">christian.roedelsperger@tuebingen.mpg.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2277" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2277&amp;title=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2277&amp;title=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2277&amp;title=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2277&amp;text=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2277&amp;title=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2277&amp;t=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2277&amp;title=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2277&amp;title=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493459/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493459" id="node493459--3" data-pisa="ggg;9/7/2277" data-pisa-master="ggg;g3.119.400326" data-apath="/ggg/9/7/2277.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em>, Divergence, and Mixed Origin Contribute to the Emergence of Orphan Genes in <em>Pristionchus</em> Nematodes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3099-1007" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Neel Prabh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7905-9675" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian Rödelsperger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2277-2286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400326 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493459--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Neel Prabh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Genetics, Max-Planck-Institute for Evolutionary Biology, August Thienemann Str. 2, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neel%2BPrabh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prabh%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANeel%2BPrabh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3099-1007" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Neel Prabh</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian Rödelsperger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Evolutionary Biology, Max-Planck-Institute for Developmental Biology, Max-Planck-Ring 9, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BR%C3%B6delsperger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6delsperger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BR%25C3%25B6delsperger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7905-9675" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian Rödelsperger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.roedelsperger@tuebingen.mpg.de" class="" data-icon-position="" data-hide-link-title="0">christian.roedelsperger@tuebingen.mpg.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2277&amp;rft.epage=2286&amp;rft.atitle=De%20Novo%2C%20Divergence%2C%20and%20Mixed%20Origin%20Contribute%20to%20the%20Emergence%20of%20Orphan%20Genes%20in%20Pristionchus%20Nematodes&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Prabh%2C+Neel&amp;rft.au=R%C3%B6delsperger%2C+Christian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493459/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493459/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493459/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493459/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493459/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493459/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493459/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493459/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493459/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493459/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493459/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493459/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2277" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2277.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2277.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2277.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
