<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2135" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2135.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2135.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2135.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/7/2135/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2135.atom" />
<meta name="HW.pisa" content="ggg;9/7/2135" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves" />
<meta name="DC.Identifier" content="10.1534/g3.119.400043" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Stegeman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Thermal reaction norms pervade organismal traits as stereotyped responses to temperature, a fundamental environmental input into sensory and physiological systems. Locomotory behavior represents an especially plastic read-out of animal response, with its dynamic dependence on environmental stimuli presenting a challenge for analysis and for understanding the genomic architecture of heritable variation. Here we characterize behavioral reaction norms as thermal performance curves for the nematode Caenorhabditis briggsae , using a collection of 23 wild isolate genotypes and 153 recombinant inbred lines to quantify the extent of genetic and plastic variation in locomotory behavior to temperature changes. By reducing the dimensionality of the multivariate phenotypic response with a function-valued trait framework, we identified genetically distinct behavioral modules that contribute to the heritable variation in the emergent overall behavioral thermal performance curve. Quantitative trait locus mapping isolated regions on Chromosome II associated with locomotory activity at benign temperatures and Chromosome V loci related to distinct aspects of sensitivity to high temperatures, with each quantitative trait locus explaining up to 28% of trait variation. These findings highlight how behavioral responses to environmental inputs as thermal reaction norms can evolve through independent changes to genetically distinct modular components of such complex phenotypes." />
<meta name="DC.Contributor" content="Gregory W. Stegeman" />
<meta name="DC.Contributor" content="Scott E. Baird" />
<meta name="DC.Contributor" content="William S. Ryu" />
<meta name="DC.Contributor" content="Asher D. Cutter" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Thermal reaction norms pervade organismal traits as stereotyped responses to temperature, a fundamental environmental input into sensory and physiological systems. Locomotory behavior represents an especially plastic read-out of animal response, with its dynamic dependence on environmental stimuli presenting a challenge for analysis and for understanding the genomic architecture of heritable variation. Here we characterize behavioral reaction norms as thermal performance curves for the nematode &lt;i&gt;Caenorhabditis briggsae&lt;/i&gt;, using a collection of 23 wild isolate genotypes and 153 recombinant inbred lines to quantify the extent of genetic and plastic variation in locomotory behavior to temperature changes. By reducing the dimensionality of the multivariate phenotypic response with a function-valued trait framework, we identified genetically distinct behavioral modules that contribute to the heritable variation in the emergent overall behavioral thermal performance curve. Quantitative trait locus mapping isolated regions on Chromosome II associated with locomotory activity at benign temperatures and Chromosome V loci related to distinct aspects of sensitivity to high temperatures, with each quantitative trait locus explaining up to 28% of trait variation. These findings highlight how behavioral responses to environmental inputs as thermal reaction norms can evolve through independent changes to genetically distinct modular components of such complex phenotypes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2135" />
<meta name="citation_id" content="9/7/2135" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2135" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2135.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2135.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2135.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400043" />
<meta name="citation_pmid" content="31048400" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2135" />
<meta name="citation_lastpage" content="2151" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gregory W. Stegeman" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Toronto" />
<meta name="citation_author" content="Scott E. Baird" />
<meta name="citation_author_institution" content="Department of Biology, Wright State University, Dayton, Ohio, 45435" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8084-9383" />
<meta name="citation_author" content="William S. Ryu" />
<meta name="citation_author_institution" content="Department of Physics, University of Toronto" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Toronto, Ontario M5S3B2, Canada" />
<meta name="citation_author_email" content="asher.cutter@utoronto.ca" />
<meta name="citation_author_email" content="wryu@physics.utoronto.ca" />
<meta name="citation_author" content="Asher D. Cutter" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Toronto" />
<meta name="citation_author_email" content="asher.cutter@utoronto.ca" />
<meta name="citation_author_email" content="wryu@physics.utoronto.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7141-0013" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. C. Andersen;citation_author=J. S. Bloom;citation_author=J. P. Gerke;citation_author=L. Kruglyak;citation_title=A variant in the neuropeptide receptor npr-1 is a major determinant of Caenorhabditis elegans growth and physiology.;citation_pages=e1004156;citation_volume=10;citation_year=2014;citation_pmid=24586193;citation_doi=10.1371/journal.pgen.1004156" />
<meta name="citation_reference" content="citation_journal_title=Physiological and Biochemical Zoology;citation_journal_abbrev=Physiological and Biochemical Zoology;citation_author=M. J. Angilletta;citation_author=R. B. Huey;citation_author=M. R. Frazier;citation_title=Thermodynamic effects on organismal performance: is hotter better?;citation_pages=197-206;citation_volume=83;citation_year=2010;citation_issue=2;citation_pmid=20001251;citation_doi=10.1086/648567" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. R. Anholt;citation_author=T. F. Mackay;citation_title=Quantitative genetic analyses of complex behaviours in Drosophila.;citation_pages=838-849;citation_volume=5;citation_year=2004;citation_issue=11;citation_pmid=15520793;citation_doi=10.1038/nrg1472" />
<meta name="citation_reference" content="citation_journal_title=Funct. Ecol.;citation_author=A. Ayrinhac;citation_author=V. Debat;citation_author=P. Gibert;citation_author=A. G. Kister;citation_author=H. Legout;citation_title=Cold adaptation in geographical populations of Drosophila melanogaster: phenotypic plasticity is more important than genetic variability.;citation_pages=700-706;citation_volume=18;citation_year=2004;citation_doi=10.1111/j.0269-8463.2004.00904.x" />
<meta name="citation_reference" content="citation_author=A. K. Bahrami;citation_author=Y. Zhang;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=R. L. Baker;citation_author=W. F. Leong;citation_author=M. T. Brock;citation_author=R. J. C. Markelz;citation_author=M. F. Covington;citation_title=Modeling development and quantitative trait mapping reveal independent genetic modules for leaf size and shape.;citation_pages=257-268;citation_volume=208;citation_year=2015;citation_doi=10.1111/nph.13509" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. W. O. Ballard;citation_author=R. G. Melvin;citation_author=S. D. Katewa;citation_author=K. Maas;citation_title=Mitochondrial DNA variation is associated with measurable differences in life-history traits and mitochondrial metabolism in Drosophila simulans.;citation_pages=1735-1747;citation_volume=61;citation_year=2007;citation_issue=7;citation_pmid=17598752;citation_doi=10.1111/j.1558-5646.2007.00133.x" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=M. L. Begasse;citation_author=M. Leaver;citation_author=F. Vazquez;citation_author=S. W. Grill;citation_author=A. A. Hyman;citation_title=Temperature dependence of cell division timing accounts for a shift in the thermal limits of C. elegans and C. briggsae.;citation_pages=647-653;citation_volume=10;citation_year=2015;citation_doi=10.1016/j.celrep.2015.01.006" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=M. R. Bernstein;citation_author=S. Zdraljevic;citation_author=E. C. Andersen;citation_author=M. V. Rockman;citation_title=Tightly-linked antagonistic-effect loci underlie polygenic demographic variation in C. elegans.;citation_year=2018;citation_doi=10.1101/428466" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009;citation_doi=10.1007/978-0-387-92125-9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ã. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_author=C.-C. Chang;citation_author=J. Rodriguez;citation_author=J. Ross;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Advances in Insect Physiology;citation_title=Physiological diversity in insects: ecological and evolutionary contexts;citation_author=S. L. Chown;citation_author=J. S. Terblanche;citation_pages=50-152;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Funct. Ecol.;citation_author=H. Colinet;citation_author=A. A. Hoffmann;citation_title=Comparing phenotypic effects and molecular correlates of developmental, gradual and rapid cold acclimation responses in Drosophila melanogaster.;citation_pages=84-93;citation_volume=26;citation_year=2012;citation_doi=10.1111/j.1365-2435.2011.01898.x" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=A. D. Cutter;citation_title=Caenorhabditis evolution in the wild.;citation_pages=983-995;citation_volume=37;citation_year=2015;citation_pmid=26126900;citation_doi=10.1002/bies.201500053" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. D. Cutter;citation_author=M.-A. FÃ©lix;citation_author=A. Barriere;citation_author=D. Charlesworth;citation_title=Patterns of nucleotide polymorphism distinguish temperate and tropical wild isolates of Caenorhabditis briggsae.;citation_volume=173;citation_year=2006;citation_pmid=16783011;citation_doi=10.1534/genetics.106.058651" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=N. J. Dingemanse;citation_author=A. J. N. Kazem;citation_author=D. RÃ©ale;citation_author=J. Wright;citation_title=Behavioural reaction norms: animal personality meets individual plasticity.;citation_pages=81-89;citation_volume=25;citation_year=2010;citation_issue=2;citation_pmid=19748700;citation_doi=10.1016/j.tree.2009.07.013" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=E. S. Dolgin;citation_author=B. Charlesworth;citation_author=S. E. Baird;citation_author=A. D. Cutter;citation_title=Inbreeding and outbreeding depression in Caenorhabditis nematodes.;citation_pages=1339-1352;citation_volume=61;citation_year=2007;citation_issue=6;citation_pmid=17542844;citation_doi=10.1111/j.1558-5646.2007.00118.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=F. Duveau;citation_author=M.-A. Felix;citation_title=Role of pleiotropy in the evolution of a cryptic developmental variation in Caenorhabditis elegans.;citation_pages=e1001230-e1001230;citation_volume=10;citation_year=2012;citation_issue=1;citation_pmid=22235190;citation_doi=10.1371/journal.pbio.1001230" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=M.-A. FÃ©lix;citation_author=R. Jovelin;citation_author=C. Ferrari;citation_author=S. Han;citation_author=Y. R. Cho;citation_title=Species richness, distribution and genetic diversity of Caenorhabditis nematodes in a remote tropical rainforest.;citation_pages=10;citation_volume=13;citation_year=2013;citation_pmid=23311925;citation_doi=10.1186/1471-2148-13-10" />
<meta name="citation_reference" content="citation_author=P. A. Fields;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Clim. Res.;citation_author=K. Fischer;citation_author=I. Karl;citation_title=Exploring plastic and genetic responses to temperature variation using copper butterflies.;citation_pages=17-30;citation_volume=43;citation_year=2010;citation_doi=10.3354/cr00892" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Fontanillas;citation_author=A. Depraz;citation_author=M. S. Giorgi;citation_author=N. Perrin;citation_title=Nonshivering thermogenesis capacity associated to mitochondrial DNA haplotypes and gender in the greater white-toothed shrew, Crocidura russula.;citation_pages=661-670;citation_volume=14;citation_year=2005;citation_issue=2;citation_pmid=15660955;citation_doi=10.1111/j.1365-294X.2004.02414.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Gaertner;citation_author=M. D. Parmenter;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_author=P. C. Phillips;citation_title=More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in Caenorhabditis elegans.;citation_volume=192;citation_year=2012;citation_pmid=23086219;citation_doi=10.1534/genetics.112.142877" />
<meta name="citation_reference" content="citation_journal_title=J. Neurophysiol.;citation_author=P. A. Garrity;citation_title=Role of adaptation in C. elegans thermotaxis. Focus on âShort-term adaptation and temporal processing in the cryophilic response of Caenorabditis elegansâ.;citation_volume=97;citation_year=2007;citation_pmid=17202237;citation_doi=10.1152/jn.01335.2006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Ghosh;citation_author=E. C. Andersen;citation_author=J. A. Shapiro;citation_author=J. P. Gerke;citation_author=L. Kruglyak;citation_title=Natural Variation in a Chloride Channel Subunit Confers Avermectin Resistance in C. elegans;citation_pages=574-578;citation_volume=335;citation_year=2012;citation_issue=6068;citation_pmid=22301316;citation_doi=10.1126/science.1214318" />
<meta name="citation_reference" content="citation_author=E. E. Glater;citation_author=M. V. Rockman;citation_author=C. I. Bargmann;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. S. Greene;citation_author=M. Brown;citation_author=M. Dobosiewicz;citation_author=I. G. Ishida;citation_author=E. Z. Macosko;citation_title=Balancing selection shapes density-dependent foraging behavior.;citation_pages=254-258;citation_volume=539;citation_year=2016;citation_pmid=PMID: 277996;citation_doi=10.1038/nature19848" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=E. W. Gutteling;citation_author=J. A. Riksen;citation_author=J. Bakker;citation_author=J. E. Kammenga;citation_title=Mapping phenotypic plasticity and genotype-environment interactions affecting life-history traits in Caenorhabditis elegans.;citation_pages=28-37;citation_volume=98;citation_year=2007;citation_issue=1;citation_pmid=16955112;citation_doi=10.1038/sj.hdy.6800894" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=N. J. Hausmann;citation_author=T. E. Juenger;citation_author=S. Sen;citation_author=K. A. Stowe;citation_author=T. E. Dawson;citation_title=Quantitative trait loci affecting delta13C and response to differential water availibility in Arabidopsis thaliana.;citation_pages=81-96;citation_volume=59;citation_year=2005;citation_issue=1;citation_pmid=15792229" />
<meta name="citation_reference" content="citation_journal_title=Funct. Ecol.;citation_author=A. A. Hoffmann;citation_author=J. Shirriffs;citation_author=M. Scott;citation_title=Relative importance of plastic vs. genetic factors in adaptive differentiation: geographical variation for stress resistance in Drosophila melanogaster from eastern Australia.;citation_pages=222-227;citation_volume=19;citation_year=2005;citation_doi=10.1111/j.1365-2435.2005.00959.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=D. K. Howe;citation_author=C. F. Baer;citation_author=D. R. Denver;citation_title=High rate of large deletions in Caenorhabditis briggsae mitochondrial genome mutation processes.;citation_volume=2;citation_year=2010;citation_pmid=20333220;citation_doi=10.1093/gbe/evp055" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=D. K. Howe;citation_author=D. R. Denver;citation_title=Muller's Ratchet and compensatory mutation in Caenorhabditis briggsae mitochondrial genome evolution.;citation_pages=62-62;citation_volume=8;citation_year=2008;citation_pmid=18302772;citation_doi=10.1186/1471-2148-8-62" />
<meta name="citation_reference" content="citation_title=Is a jack-of-all-temperatures a master of none? Evolution;citation_author=R. B. Huey;citation_author=P. E. Hertz;citation_pages=441-444;citation_volume=38;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=R. B. Huey;citation_author=J. G. Kingsolver;citation_title=Evolution of thermal sensitivity of ectotherm performance.;citation_pages=131-135;citation_volume=4;citation_year=1989;citation_issue=5;citation_pmid=21227334;citation_doi=10.1016/0169-5347(89)90211-5" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=R. B. Huey;citation_author=J. G. Kingsolver;citation_title=Evolution of resistance to high-temperature in ectotherms.;citation_pages=S21-S46;citation_volume=142;citation_year=1993;citation_doi=10.1086/285521" />
<meta name="citation_reference" content="citation_journal_title=Am. Zool.;citation_author=R. B. Huey;citation_author=R. D. Stevenson;citation_title=Integrating thermal physiology and ecology of ectotherms: a discussion of approaches.;citation_volume=19;citation_year=1979;citation_doi=10.1093/icb/19.1.357" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. C. Koboldt;citation_author=J. Staisch;citation_author=B. Thillainathan;citation_author=K. Haines;citation_author=S. E. Baird;citation_title=A toolkit for rapid gene mapping in the nematode Caenorhabditis briggsae.;citation_pages=236-236;citation_volume=11;citation_year=2010;citation_pmid=20385026;citation_doi=10.1186/1471-2164-11-236" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=C. A. L. Latimer;citation_author=B. R. Foley;citation_author=S. F. Chenoweth;citation_title=Connecting thermal performance curve variation to the genotype: a multivariate QTL approach.;citation_pages=155-168;citation_volume=28;citation_year=2015;citation_doi=10.1111/jeb.12552" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=C. A. L. Latimer;citation_author=K. McGuigan;citation_author=R. S. Wilson;citation_author=M. W. Blows;citation_author=S. F. Chenoweth;citation_title=The contribution of spontaneous mutations to thermal sensitivity curve variation in Drosophila serrata.;citation_pages=1824-1837;citation_volume=68;citation_year=2014;citation_doi=10.1111/evo.12392" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=C. A. L. Latimer;citation_author=R. S. Wilson;citation_author=S. F. Chenoweth;citation_title=Quantitative genetic variation for thermal performance curves within and among natural populations of Drosophila serrata.;citation_pages=965-975;citation_volume=24;citation_year=2011;citation_issue=5;citation_pmid=21306462;citation_doi=10.1111/j.1420-9101.2011.02227.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Li;citation_author=O. A. Alvarez;citation_author=E. W. Gutteling;citation_author=M. Tijsterman;citation_author=J. Fu;citation_title=Mapping determinants of gene expression plasticity by genetical genomics in C. elegans.;citation_pages=e222-e222;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17196041;citation_doi=10.1371/journal.pgen.0020222" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. X. Ma;citation_author=G. Casella;citation_author=R. L. Wu;citation_title=Functional Mapping of Quantitative Trait Loci Underlying the Character Process: A Theoretical Framework;citation_pages=1751-1762;citation_volume=161;citation_year=2002;citation_issue=4;citation_pmid=12196415" />
<meta name="citation_reference" content="Mark, S., J. Weiss, E. Sharma, T. Liu, W. Wang et al., 2019âGenome structure predicts modular transcriptome responses to genetic and environmental conditions. BioRxiv. 10.1101/517235" />
<meta name="citation_reference" content="citation_journal_abbrev=Neuron;citation_author=P. T. McGrath;citation_author=M. V. Rockman;citation_author=M. Zimmer;citation_author=H. Jang;citation_author=E. Z. Macosko;citation_title=Quantitative mapping of a digenic behavioral trait implicates globin variation in C. elegans sensory behaviors.;citation_pages=692-699;citation_volume=61;citation_year=2009;citation_issue=5;citation_pmid=19285466;citation_doi=10.1016/j.neuron.2009.02.012" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. J. Murren;citation_author=J. R. Auld;citation_author=H. Callahan;citation_author=C. K. Ghalambor;citation_author=C. A. Handelsman;citation_title=Constraints on the evolution of phenotypic plasticity: limits and costs of phenotype and plasticity.;citation_pages=293-301;citation_volume=115;citation_year=2015;citation_pmid=25690179;citation_doi=10.1038/hdy.2015.8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=D. Nettle;citation_author=M. Bateson;citation_title=Adaptive developmental plasticity: what is it, how can we recognize it and when can it evolve?;citation_volume=282;citation_year=2015;citation_pmid=26203000;citation_doi=10.1098/rspb.2015.1005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. F. Palopoli;citation_author=M. V. Rockman;citation_author=A. Tinmaung;citation_author=C. Ramsay;citation_author=S. Curwen;citation_title=Molecular basis of the copulatory plug polymorphism in Caenorhabditis elegans.;citation_pages=1019-1022;citation_volume=454;citation_year=2008;citation_issue=7207;citation_pmid=18633349;citation_doi=10.1038/nature07171" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. Prasad;citation_author=M. Croydon-Sugarman;citation_author=R. L. Murray;citation_author=A. D. Cutter;citation_title=Temperature-dependent fecundity associates with latitude in Caenorhabditis briggsae.;citation_pages=52-63;citation_volume=65;citation_year=2011;citation_issue=1;citation_pmid=20731713;citation_doi=10.1111/j.1558-5646.2010.01110.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. A. Ross;citation_author=D. K. Howe;citation_author=A. Coleman-Hulbert;citation_author=D. R. Denver;citation_author=S. Estes;citation_title=Paternal mitochondrial transmission in intra-species Caenorhabditis briggsae hybrids.;citation_volume=33;citation_year=2016;citation_pmid=27613821;citation_doi=10.1093/molbev/msw192" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. A. Ross;citation_author=D. C. Koboldt;citation_author=J. E. Staisch;citation_author=H. M. Chamberlin;citation_author=B. P. Gupta;citation_title=Caenorhabditis briggsae recombinant inbred line genotypes reveal inter-strain incompatibility and the evolution of recombination.;citation_pages=e1002174-e1002174;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21779179;citation_doi=10.1371/journal.pgen.1002174" />
<meta name="citation_reference" content="citation_title=Phenotypic evolution: a reaction norm perspective;citation_author=C. D. Schlichting;citation_author=M. Pigliucci;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Shao;citation_author=L. C. Burrage;citation_author=D. S. Sinasac;citation_author=A. E. Hill;citation_author=S. R. Ernest;citation_title=Genetic architecture of complex traits: Large phenotypic effects and pervasive epistasis.;citation_volume=105;citation_year=2008;citation_pmid=19066216;citation_doi=10.1073/pnas.0810388105" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=K. L. Sikkink;citation_author=R. M. Reynolds;citation_author=C. M. Ituarte;citation_author=W. A. Cresko;citation_author=P. C. Phillips;citation_title=Environmental and evolutionary drivers of the modular gene regulatory network underlying phenotypic plasticity for stress resistance in the nematode Caenorhabditis remanei.;citation_volume=9;citation_year=2019;citation_pmid=30679247;citation_doi=10.1534/g3.118.200017" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Biol.;citation_author=G. W. Stegeman;citation_author=M. Bueno de Mesquita;citation_author=W. S. Ryu;citation_author=A. D. Cutter;citation_title=Temperature-dependent behaviors are genetically variable in the nematode Caenorhabditis briggsae.;citation_volume=216;citation_year=2013;citation_pmid=23155083;citation_doi=10.1242/jeb.075408" />
<meta name="citation_reference" content="Stegeman, G. W., D. Medina, A. D. Cutter and W. S. Ryu, 2019âNeuro-genetic control over plasticity of C. elegans behavioral thermal performance. BioRxiv. 10.1101/583120" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=L. D. Stein;citation_author=Z. Bao;citation_author=D. Blasiar;citation_author=T. Blumenthal;citation_author=M. R. Brent;citation_title=The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics.;citation_pages=E45-E45;citation_volume=1;citation_year=2003;citation_issue=2;citation_pmid=14624247;citation_doi=10.1371/journal.pbio.0000045" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=J. R. Stinchcombe;citation_author=M. Kirkpatrick;citation_title=Genetics and evolution of function-valued traits: understanding environmentally responsive phenotypes.;citation_pages=637-647;citation_volume=27;citation_year=2012;citation_issue=11;citation_pmid=22898151;citation_doi=10.1016/j.tree.2012.07.002" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=F. Supek;citation_author=M. BoÅ¡njak;citation_author=N. Å kunca;citation_author=T. Å muc;citation_title=REVIGO summarizes and visualizes long lists of gene ontology terms.;citation_pages=e21800-e21800;citation_volume=6;citation_year=2011;citation_issue=7;citation_pmid=21789182;citation_doi=10.1371/journal.pone.0021800" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. G. Thomas;citation_author=W. Wang;citation_author=R. Jovelin;citation_author=R. Ghosh;citation_author=T. Lomasko;citation_title=Full-genome evolutionary histories of selfing, splitting and selection in Caenorhabditis.;citation_volume=25;citation_year=2015;citation_pmid=25783854;citation_doi=10.1101/gr.187237.114" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. D. Thomas;citation_author=M. J. Campbell;citation_author=A. Kejariwal;citation_author=H. Y. Mi;citation_author=B. Karlak;citation_title=PANTHER: A Library of Protein Families and Subfamilies Indexed by Function;citation_pages=2129-2141;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952881;citation_doi=10.1101/gr.772403" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. C. Ungerer;citation_author=S. S. Halldorsdottir;citation_author=M. D. Purugganan;citation_author=T. F. C. Mackay;citation_title=Genotype-Environment Interactions at Quantitative Trait Loci Affecting Inflorescence Development in Arabidopsis thaliana;citation_pages=353-365;citation_volume=165;citation_year=2003;citation_issue=1;citation_pmid=14504242" />
<meta name="citation_reference" content="citation_journal_title=J. Insect Physiol.;citation_author=M. Urquhart-Cronish;citation_author=M. B. Sokolowski;citation_title=Gene-environment interplay in Drosophila melanogaster: Chronic nutritional deprivation in larval life affects adult fecal output.;citation_pages=95-100;citation_volume=69;citation_year=2014;citation_pmid=24929224;citation_doi=10.1016/j.jinsphys.2014.06.001" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=S. Via;citation_author=R. Gomulkiewicz;citation_author=G. De Jong;citation_author=S. M. Scheiner;citation_author=C. D. Schlichting;citation_title=Adaptive phenotypic plasticity: consensus and controversy.;citation_pages=212-217;citation_volume=10;citation_year=1995;citation_issue=5;citation_pmid=21237012;citation_doi=10.1016/S0169-5347(00)89061-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Vieira;citation_author=E. G. Pasyukova;citation_author=Z.-B. Zeng;citation_author=J. B. Hackett;citation_author=R. F. Lyman;citation_title=Genotype-Environment Interaction for Quantitative Trait Loci Affecting Life Span in Drosophila melanogaster;citation_pages=213-227;citation_volume=154;citation_year=2000;citation_issue=1;citation_pmid=10628982" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. N. Weber;citation_author=B. K. Peterson;citation_author=H. E. Hoekstra;citation_title=Discrete genetic modules are responsible for complex burrow evolution in Peromyscus mice.;citation_pages=402-405;citation_volume=493;citation_year=2013;citation_pmid=23325221;citation_doi=10.1038/nature11816" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Wu;citation_author=M. Lin;citation_title=Functional mapping - how to map and study the genetic architecture of dynamic complex traits.;citation_pages=229-237;citation_volume=7;citation_year=2006;citation_issue=3;citation_pmid=16485021;citation_doi=10.1038/nrg1804" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_title=A computational approach for functional mapping of quantitative trait loci that regulate thermal performance curves.;citation_author=J. S. Yap;citation_author=C. Wang;citation_author=R. Wu;citation_volume=2;citation_year=2007" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="Thermal reaction norms pervade organismal traits as stereotyped responses to temperature, a fundamental environmental input into sensory and physiological systems. Locomotory behavior represents an especially plastic read-out of animal response, with its dynamic dependence on environmental stimuli presenting a challenge for analysis and for understanding the genomic architecture of heritable variation. Here we characterize behavioral reaction norms as thermal performance curves for the nematode Caenorhabditis briggsae , using a collection of 23 wild isolate genotypes and 153 recombinant inbred lines to quantify the extent of genetic and plastic variation in locomotory behavior to temperature changes. By reducing the dimensionality of the multivariate phenotypic response with a function-valued trait framework, we identified genetically distinct behavioral modules that contribute to the heritable variation in the emergent overall behavioral thermal performance curve. Quantitative trait locus mapping isolated regions on Chromosome II associated with locomotory activity at benign temperatures and Chromosome V loci related to distinct aspects of sensitivity to high temperatures, with each quantitative trait locus explaining up to 28% of trait variation. These findings highlight how behavioral responses to environmental inputs as thermal reaction norms can evolve through independent changes to genetically distinct modular components of such complex phenotypes." />
<meta name="og-title" property="og:title" content="Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2135" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Thermal reaction norms pervade organismal traits as stereotyped responses to temperature, a fundamental environmental input into sensory and physiological systems. Locomotory behavior represents an especially plastic read-out of animal response, with its dynamic dependence on environmental stimuli presenting a challenge for analysis and for understanding the genomic architecture of heritable variation. Here we characterize behavioral reaction norms as thermal performance curves for the nematode Caenorhabditis briggsae , using a collection of 23 wild isolate genotypes and 153 recombinant inbred lines to quantify the extent of genetic and plastic variation in locomotory behavior to temperature changes. By reducing the dimensionality of the multivariate phenotypic response with a function-valued trait framework, we identified genetically distinct behavioral modules that contribute to the heritable variation in the emergent overall behavioral thermal performance curve. Quantitative trait locus mapping isolated regions on Chromosome II associated with locomotory activity at benign temperatures and Chromosome V loci related to distinct aspects of sensitivity to high temperatures, with each quantitative trait locus explaining up to 28% of trait variation. These findings highlight how behavioral responses to environmental inputs as thermal reaction norms can evolve through independent changes to genetically distinct modular components of such complex phenotypes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__Ta-UXINC0YONAj3hv7M-Srfc-_B4Wn0KATTOQGieTps__agTSMicuoC50kRvu-VcTNoPubLJgflHIUuLq_Jt5dMY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"YjngOy3Sy49bxPU5Lv6Jmd_vba85FpoiPU3jJR4yX8o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493672","apath":"\/ggg\/9\/7\/2135.atom","pisa":"ggg;9\/7\/2135","ac":{"\/ggg\/9\/7\/2135.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2135.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2135"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2135"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2135":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MZOUFOkGZWSTodWr38SnSh3XvkxlpeMi5NYVmrIdDbDIsun6FGXjaL+yfLBCVCBSJz6VlQM7VTpYi1TE\\\\\\\/vCCchqvLjXzOo6Q5as6Rm5mi7QS5fpOnkzt5XNaDXKGgm6\\\\\\\/4lWWfExGYK4fpTL23o9WX+LMhsIcxt6XMcYbTDx4seBfB7ZrR7jddrvmr465zZFkZIaIKYqIlpuMavjzhpoTQVWAYv2o2d+jhGpJOvUdCoD\\\\\\\/vB3jcj8ySMwYME1ymByOnOvco28GshD+Z2pBYULq4UOqDq2kt5mS6nK\\\\\\\/IDOJAEkHwbz1Zg61JGfNosgfaJH9wmdAyUcefh0DUR19tVt7+TsWvjqkYazMBOKcyc8BkvN46pvVbKPVHG546MIwvb9Bzb4ZtrRUC2oIZ2sgkamIjwlg0KgatZx1chmoBvECfQ2o+A7xY5\\\\\\\/IAWC79x9uxQJcpp3LXl9o5kpmjvZm50QSOIvpEhMMedv2me9PtpxQKsuJ4apTtJCYVkNbaWJTGTIFfIV\\\\\\\/3\\\\\\\/5qAmCY8IO089iFnxiJcqUoHNXWeSktO8vgGOP5Z3cLu+ZtIKC20tkLrBtA5uHSW+z7kh5kgl1ENrFJHLw1y7fHTqcyEjKhF0g+N\\\\\\\/HxjVgSnoq1eJxMT3+19C9\\\\\\\/jO7qUOM3QnY3er3nFUNMwXI+6eVFNKSJtsk9HOwtlPBxaXkMz+16Zh\\\\\\\/4gqQvu3l5LjJMmAGR+MR6tTCb70y4iXrtxTXz5QFRfABhDNOPhYfON+F1ZBF5OD\\\\\\\/BW1eFOEJT4koT49zvU0UupNmoLKaxePfqDpKWv3ChPdrcBoYsBiLxCwSiNswfecrlLAGeTUPumX8\\\\\\\/1zdiD0TBRldlS1rfSctsPQsMpavX9kGsTOoiQeffoFybnWbJC8n1TjP8FeR60sKcMlS8kmNWC3k0Dr\\\\\\\/clvFzyvbjNT1HUltv3sH4cEPnInToShFnlwf7anZwGIm2oJfYlzBfLouFPlN\\\\\\\/BUKWYug7vcQjKvHfXGQ6AK5AR8MzeEfIxjiieBbbU03+S3aQY3RveljnrZjjqklpVy5U4sUDC\\\\\\\/lW64iziw\\\\\\\/KfzWjncz7wSe2eJfAFKC3w4T0HnIyk7NS2QcnOSH8H8oejO60kHCfO4ZGePIE\\\\\\\/xZFRSSM3VlFgPVyhLB4chDs89lONv2MJLcTSMQxVevX4ZPMAfnka0jXf4PkQTC+KM+uVISBFEWZUqiof2C7BtS08WzuDuxxZRLSdxze2wdwffSnpcjG\\\\\\\/\\\\\\\/c+2X0fGJbK8fsrbB3Ool6ptGLNoA90\\\\\\\/EIWVMmSZh0t\\\u0022,\\\u0022iv\\\u0022:\\\u0022VJTLvsoNABdW1qqTMVaBBQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225a033350bdd2372a74d02a6ff5d9180bf9e3b5ff550385bdd748443e717bb45d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HzZWsl4IlxQbUT1ngtkh77Ko\\\\\\\/uOHJ5braDdUQ4cLgvfwyyoWpiALsdx23ZRwVlIXh8NSHMG7CPN3iCZusPWEN9YYgZmcesfHcM4zvdPJYlBl2VuI07R\\\\\\\/ylxE2w\\\\\\\/V0dLkdVV\\\\\\\/qIyABVJciI+PzPei0kNxtZiwXOUntMuTvPIqJ\\\\\\\/bY0ZzG0r0gHJ3EP8HYGxW4EiRVYuYW7oJdHasK07Cb+68\\\\\\\/uJP0qyVHRle2vhp6E21djxiuH2+7KAFqGM\\\\\\\/UQGc1osroHWgCIK8PFjyfTgJ+5Rxs3DF4u6a5Tc9D4FtDikwFYdVPt8VvnHGmbMVysAWpvC1wVckyk8T9RI6c+kZCh+XC3boehNswK9s0gC9U7NNClZoviriEmuIGqyjuY+8Zt4V0hBKsH08R3tXCX2HJ6iVGEVJkslHRnBzFIVLCdtTClKC8QV09opiFdXZU4AVjnAPydb2JADsbWmLDGSG\\\\\\\/Jh55JdrjXpU70RdY90GCqKMfs26aqUD\\\\\\\/WzJM1ytNu2rWC0vi\\\\\\\/s81AHtKCLt8VEH7daJPPFQ1F0SL9BgiWoP3iZKQxvLjdOYAlCeTaoNVpo3h2vSE7lwbH0fPFkZ0mxiHDwsgIQ4pLwY48CKrYuv0AApajmoxyjgogq5gD1IsqUvbWKnz929C75E05QJR5DJWmJI5AW1xh8srcx7mzb\\\\\\\/qsxEltvZPFX9MXNtLh3WrgRW8IyVGvS6\\\\\\\/RI+b4L+DrRNZnRsx2yAaSt+Ag9KbSl6PFW91F4za1Pv+yQmA333qyseZ9w8wbedZ5lULuV9V\\\\\\\/PRgkVBNpc5QNqus3t86iECTsLdWxi2+KXg29kdojuydJbvHDKUaKBszTNTEh7rGNE0N1B12aaRn+JFcZpgXENMHkwZ8GUDP9PdYIRAmy0ZtYnB3nRbbBHgcTYsmVDMDa4vGE+haP7YYbhI26FUmCasrPnxhPb7zVpHcytcSmQACIx8zWMEbunAsgBmWTtfq5qZKn\\\\\\\/pppw5YiH0kRqsivVACXQEnkDT98G1dfYSbbEAsrXA6X6yLbo0kO8Wc6Y0GkviHN8hfnQ5WXRV7O9xtW+MWaYkxxhQ8dvu4Kyqxd6LXxobWQAEz5Cr5ax8UCRDomRixqvKObA1GxLCYPTYm\\\\\\\/r2bYK3xFqwurInrCM9SN8RTOE1p8JXMwwgeUQpwQWdgl+Yym\\\\\\\/mnk7q1MGkcW8gvjEct3lzPjNSfeIA9wqf6Te6ET19R\\\\\\\/co2KZPXS0ybrK14IflZMJtnG\\\\\\\/e8IK23pKN7Jf5+wYLkD8oHpqJYE0WYChAGyBPtSK9TuIRe3kosok1KNWD2XjXLimbVBrCoJWcSoZt5hryksusXVRX+H4w3tSn4Y9N12\\\\\\\/pB2xJM4hTuA\\\\\\\/I9O9DjpRRhgV2EYVHXw0kqQhC1jiOJzwWDGuy8xI9QyROOi9V3ASNVUrryFVe\\\\\\\/pvLgJXKVPFnA+tDkZDv9VwRS+g1zBK8cqdZqSyPWI4sBlIIbtNDQVmklMnBLnVwCyBT3qwu7RYk8S+xm50EQkwRDvBoWZ359PAXP3BO2rXbFAqwbWxfVMp0yrwSQ4n5wARiP7O5vOrtp1OeX5ghqdtnFq\\\\\\\/u+kd5te5\\\\\\\/tZ\\\\\\\/+l6GK\\\\\\\/g3j5fMzQ\\\\\\\/DTEBldx2\\\\\\\/UJsvU5QXTYCN8ATYNBaVihllnKfmAtQsPY2yKbSPzYpX6Tq4bX2\\\\\\\/8bjDcMrFRgrOJ5HjmN3NyiElQQtgAQXU+1TB31AWUU1Vd0J03TO6aK+vNvA8N+JvfDv78tYp9G8sf5fqSnhRvKQ5pbt41AvXgoHW27rmH0+rgyznCNSuTp5ZHpe4yRQiO0xfGb9vjvHGNqLzh+8gDv68YixAgJGm\\\\\\\/7GT\\\\\\\/v+b2\\\\\\\/s1K4Gu3g6xrFLWGCKJ45WTXQ1ysvRlpNjAwYclZP0bWBoGtesGGU7RL\\\\\\\/io1kbugZJBnLHpHYvL8c+fI0JT8WFCd5spCJC3PA8KSvzCKY3FO\\\\\\\/E5CUE0zZjh5WLBGyrlN8DKxfFNZPp6L2qTFuqmtGQQ+i7tm8Az+prniQhq4Wmvi5MtNVvWLGYsG1PF\\\\\\\/Z2+tNHeSAzLVfOZkHqpEmQcNNJAAsroKSKUydPKXuTa0M3rmBv8nQSnHZuW\\\\\\\/jOBSFJVhA0wwW3W4uCGkEw1JofegPon58T\\\\\\\/UG4eHBL9v2N8R+pINYs2BCu9M9dWRz1EUpzb3VM7e53MFSAcan8AIQLtKskvfjRvtKo11nUzGbhxDl6yE0EDOoaJ\\\\\\\/SWwegpKZ5L74bG9\\\\\\\/j6qAknQsWIiw9qFGRGhM7R6++eiTnf\\\\\\\/fOYQH3IF4Ch9DRYmFLLn13B0fA0II3Tf5dlseRqOz0aeyoAL9PsGVagSpiLBty0zmkjmD+g7oDj+hQwSFvsv\\\\\\\/VUYOk8AJWFBcPmMQjHzKztqODDpatd9gXxPytWgvWvKsqaFqllv2tf0NqbSiVx2e4PrHaZqlBCaC0l+eb5vRnEnvNH4hOGzE18jvpaapHeNue6ecrx5VAzWu6be0s\\\\\\\/ClRaRupyU2N94X4drfM5nU7KF7NlTGYJ2X1HENkcK4sIKHn0orDStIOPGBZO1KfMhGYrf3XJuRA9zNSowPe8Y3UEpJ+YaAPeFRvoYmyO1WKZ72Dq1y6WheY7Th1BB53HCSPDjPyJTTPx33yiXJGPACxAMks2Jx1jx4o05bRPlYBmUkGGrl3C3nFmBUWAC4YgNCzVaNQkevzPgVMIFdbJzZniULOD8rIvhwFee1uHDWzwfRFc81u3xbg0uZUgmy69JwZ71EQLPM43foE74EEipic\\\\\\\/Cb7\\\\\\\/QRcbUQLIGF\\\\\\\/x+MQHOv5BZPBBfmT9ks4JVpDcQDAUjwAwK3XiaSeLjp0tf0Q2HXWX5eW5XtD0zMI2XpGeJvZhMp3zsTbvG5\\\\\\\/YD9i1OQEKPM2gIKxGGzSJm58LYsDDv+nZr0RZ6W2GSxJXcJznuNBp25llCA1pO9r9QiEtTYj+idB1hZyXNjJXWzwIicAOUQ+CRloTOxFT\\\\\\\/jikPxuonaE9UXmJ13ZVHCXFGK8iUJxq3U7Olm\\\\\\\/9SXrwyY1is0RVWqZiLDI8vMv5CA9Z+wdxAYD8\\\\\\\/JGOBCb8jY7NnDGTmX9VLzyTZrI+oIUlDyKZuXkyyJK8cJ3TxUfBNAO5stpWlyQcTUyDcLHjgxisdW8PtQgP+Q9trnQlNej6ypAo3CENtXGwZq+xiGr1kifbp3UxkV83uZEyH\\\\\\\/yRQ+XoY\\\\\\\/SfBCjM1J6xvFVr2uenU8DA318Kgfp2YJ3fwE\\\\\\\/ytL0JdqC0qB223dAhHCdxmad+\\\\\\\/NUD8SK2JWPZ2lkRxE2d3KkFHppOiczbaK0CB5wbz3uEwyRdcVw+Ie6gWj8YRWpL\\\\\\\/\\\\\\\/MuFuoEAO\\\\\\\/iCw2a9ExNc1v8trK8KjaCocsu6Y7oEeJPfK4ssWIrkAGkb9p362Kk9IGyKnjhTDL+m6qzj1KWudnz7jwmztPhAWAreciSzmmpk8AU+dPMQOl+Ktjye15SPEjf3TLi8KGQmnICcDgdO+stf8JzZDqb5o5E1GGfIpKaagGijUnps8TYsQRdeiZtkqdyDYM+ei6vLx3xUNzvBcBeoVR0ECwoWc4jCIf\\\\\\\/vl\\\\\\\/NJYgIZ2gw7xUV1Q\\\\\\\/J2S++MAZLcUuQ4KAWoMNaYYOqoGALunbDk\\\\\\\/U9aI9cuUgjKa+aOZgulgGV02mUT8+bq2hA3r7A6q\\\\\\\/Xhtd+tJLcDxmRnqbzVZbrtOUxI9TntCKSEgmG1IEcTGn7gdhIuYYEFggq2lYW9u0drbpkali\\\\\\\/Vvjt6EgvqEQGtdLqTWU5d90Em6OW8hidwE4Nq5\\\\\\\/AAD6xqcXDM+gppPR9xyJuY1Yv7AYuhrW7ytDJu5+eh8llLz\\\\\\\/Pbv7zf2DT7WNQK\\\\\\\/zXZ6QyoFUWkXyzcOHRvwUsi6uhjiPou9T9cfub1Ir9QLF5F7vhq1jyFmnKaR7S8trOd7AjeDSlBt2oWVhRoJjw67I25y8VLKJ8eiJcbvx8lK1+p4Lcky9RONf9OkC9YzLPFnayvSPFIpSkCQTWnZv1\\\\\\\/51S\\\\\\\/LnkDNx6cuiJcUmZHSNhL6VulL4wG8i75RWrpIAvl1ioOMOOOdKLIkPsgC7uo4XMFu4XA1\\\\\\\/ORTWmKW5Ad3xC0rMVoLwJ51PCm9dgH6ILyv4QGzYWvOsHFwVGEAIo+RFlhiteIr0BaStGCWW8E9lc8ehtzPanYRI1HNSQEoDFXSwJV18MNyNYk0FDrMjU9uwEGWZmW6G7GNyWpz0O1FEnAxhwL6CoETIUAZ9PW6pR4FTeTnqxvsD4ZmA8k2+cf6rNJ3KIKyWL0fOiFe1GYVVVlyNHzwaFcb\\\\\\\/PZoERDLx\\\\\\\/liQg\\\\\\\/H8TVUJ6J7PhoyLKAbANfrsrI07+Oi+SmaoV4Qor33TwewtPuw20kuFS9lkmEIiLlBnPC\\\\\\\/ZS18IPoLdgOqc4iEbgoxZkvSIBZ+QQP1GiMUG\\\\\\\/g0Xnn7ymMNtqDHWXPhiWVKOsqLvG0mZ2Vab40PrmRsEjUuZooOuYii+nMiUX\\\\\\\/K1hiqNHzzmcRi87Ig\\\\\\\/1ogSTs9i\\\\\\\/LviG4qdfSFyUqJQiE3p9Vkx+fXz8hQ5ls66daHso7Zn3kbdWiU++B+iK4rQkdvJZG\\\\\\\/Hs9KBRu6PP\\\\\\\/ptR8rjvwsVD2sRentOI11m6H6\\\\\\\/4674ufLmImUQM22V\\\\\\\/cXwgFRiD95th2O3EixlZcqY9X0mfW5Oxe7ZqBthbbHeLiKWvYSA58epU33gmBq1c47X7W364SF61AB9pRxoqxlDY4fn9LykO8mSbLaqFnmisZZH4uoIPmQ4oAKiT0FBvYqe2x4MDmmP6V+XjynGr0y9\\\\\\\/Nxwl\\\\\\\/6LKWiP2tDAyfuS5DnHnyl6vjB0zcU259k0I2+Gi+ZW8e3oX8gx7zkdrlE9q6mvdh1u0i8WLF3TpmKnK+hPbFBFdFwMYmjJy+UQgjCA7A8ozjUDD0JhPWbHoJgsX6dYv062LhCMZ5+aSs8\\\\\\\/RzhM2X6rFciKgnwuLFiSkSsMCC5wNS255QQN7jGvHGZCk07BQdLu14275aCl24JFm1XVSx0ll5fkmNeRVja\\\\\\\/brCDG08XMZQXOp8KhbrYzxJbfkeSmV+fxATnO8aaWbKqKKS\\\\\\\/bsgvsjhI8z4IvJLeKvPnz2k1drzzvP587WmKnSwtZTZ1vzeRwvw6OtPo+pNIgHL23C7QQLn9zcTv22yekRy4VBOMJfBxX6BKzDzkp5rzesmCrPrlrnWJd6TVaIiqXsYXV6VzQytDjhnc878SExaaOtEyHoqfLoR9kpEjyUyStQAxaP1O6EWrmghb3ZxtK+XHZukNVEc4bXAWSFnHn5goJ6p27dEDN2Q+pdNxWK1suBet4ToospPK4KkRyeR\\\\\\\/ZNBmIGisWbVTJug+2viTJJDOp9iEUFtqYmUm47JhPTfsYCdussVbXrf5yDSW1vbAXGKvpRHqHtlEN4cTjDz3AGEtd2Ezdc6w+oijm8QYX0VM+9FvZ3TWUJCqzzPqQ\\\\\\\/5LV2aMIwN+in9c7rr5oWxhtMT9jgB87Y3X5fCcd7DcEGrQZoDilNEwENpjmgmv7IMIHF3278wuBTFzirDPXeMc4VdZPrCSTP1GHztvR9LEsOoBCFMolI\\\\\\\/gRAerb4FJVjM8HLe9\\\\\\\/TnYMqovIZ6SEfIKq0C+VoOHYNjWR9DeqpPs+xeuDae1rtei1pm3oa96mZzNxreZkeQw2gUQdgkg5m3XeoMS6wBNeImF+DQ3qH\\\\\\\/3USNYOymNeVRiyuTAF1hAlCFcCkqGNb0kL6Lx1RqwmA0dppvKepKAyH\\\\\\\/JJGLU9mnPXm0GphQ6nNYXh9XvCz4iVsE2L3gT1EgCi0MXn7vE7TMmw5i3yJEK0XRdgvwQDqWJx7PWO9YXmqx52NsO7yJpiAUp0T7Y05rVgijuGCuvFoa+LtlGcuDAx+OI\\\\\\\/8dSkJ8fcUQZfVubgzhCVgbFfsToeQH3g5CxjwavAPxzYa79AjGhRG3180ju42MMsrUpdeZT9YDZDBewPwHCutj5USbaHy1v27ccvBu5eBBbSc9Y3XnpIDaC0a2zUn3wGEmCcL7TSn0qPReEeNAgSg85QkKXt08kS95EGCkOdTqwxg\\\\\\\/98XvuxIXIww8QjsGcyBZTe0zpcl+L49ElmdzE\\\\\\\/xotm9WRwjf9gZnHagWnOpSlDjn1UHvK6x4TzjWNNICe3ziZ748pKA5bS9f6ilK\\\\\\\/fJ651XzQG0tscIyN7piu0pxP49iI2JhLbe3P+BtdNxzoTA05E6ENB5Jg40crSr2wLLp989uUKNSgKZI+q6Kieqeeh80ur2IpxI85okLT96cWfEYLGknM8XDIUWSJfuEfXRtmGM\\\\\\\/PeHZvizQ0ZSyHZ+t5Ci0ZaiTRB\\\\\\\/JXYV97tr4lQ5wpMBCLMQC3nZElNTNaIG4iq5dsqqAyzbAO9WfHXvWTtaLyerAj3ZPP4+uroUerNomxkpGk59csOFSsi\\\\\\\/yIn8HIgpp8Ggi6As+s8W8ahzV010IUSXeOu\\\\\\\/6u6Cp2uJ04qbNM9t7MwRiwJeK2n2DrlTSMbX\\\\\\\/eVLKN1pGPYY9O9JKPzY1mHbLSFloiluci8QL5MDoPLKjmChC2qt0QC0TGJkES5gfuL9mr1faUUgRAOXscrGAaQ4nd7Bq7PNGK8pArI\\\\\\\/45gGKKWk+uSMUJ\\\\\\\/ddwr5t2oP+KkjbXN6zdx\\\\\\\/vkM\\\\\\\/fac6pilsVgWdBynATuEtaSVGpc+ZXn4JJu2qsVC2egnwahjXvr+eE12\\\\\\\/KbdRWB319KV4nlRuLTdwaDApOQcrFT1fpNTT8dVkEFUfAdsBbFBnxMgX5Pw8lBaZyCMhLbZKbvdreiVvr4\\\\\\\/Iy8kfuEsZv2DNMAb7x7QxO8iMMnpSVwq37LnTbKKrFwZBYGnlmnGukngHXP0k++uHcLk+JMP2c2ZhzD0V7nDwpvyueiXYRRB55ZICldfNQ\\\\\\\/nclpU82iGz3DS4scCB9KWJVHYXKNNmeLtKq0YdiCCe0Zn28C9mCqxiku+A2xGRJ2Y17nY5rFtoXlM1IIBcLT6tsVYTkH+dFrvbijH1\\\\\\\/G4jc1b9pnCIcNV4UaoU+LI2NMOyKOkvmPOTr899nt4tAPTEfozYQ25wv9vdpsYH163kSbnfL9MjkQKXJUEDhp5+2ELXLTv4pCNeXk48SweMGELeFL9lzUf50qp052Vgz+f1vDsdyem5p0n48xJoW58CXzgHY1roi7TyJ2InWIT5DB+TE0xOj2ZjX4jPxqjJEHilzLirEv3yGJhvBnzgsThA2Zc5WPNwOjUxlvIKpkoKtnRxZxK5a4i3qbNnrTilZpjk2mMLfjgFws\\\\\\\/26dnhvpQpMELG75RZjMGuLv8IyWn9rvTB550mA\\\\\\\/ZrZ9HgikU2eRxSm3w4Dp+XlTJnCBC3geUVXze5OrRuBiWHEwBgXwA5AqJfXzvfMzkLW\\\\\\\/zTfUCmj7B0zd2ry0bE\\\\\\\/OfL\\\\\\\/hUkim+KY\\\\\\\/sSx\\\\\\\/bphZD+B2hPKGjGEM+eiQ5aZhKUY4bgGapPcDrmj7VHiGc6\\\\\\\/Kb6NF++cmY2CbAE1Mua8fQlDO0sZOmL1pdsMrNHrPvQtAp+c\\\\\\\/\\\\\\\/0q3\\\\\\\/k4ZguvFwnZAPnmX0P00W1LvAfuLoGAPZwAFOc2CBvRv6Twix\\\\\\\/ExvWmweP50iITHY+zptg0\\\\\\\/nMP3nTkVEuKTCoi3ENLB5j36UWWv1xwY3Ko8FONjxb8ObubGcuBWL39iIcJPKHJ7D\\\\\\\/d5HOz19MJyWlcTWHNK8t1vp2ra0tgQjNDTmIoQ+fO+qu6J1GAgIgCIuocxqE4kzaeHI18B022MzOfjHXCwRpd93cTObcHNa7JNrGs95SaL3elxTbwAXUABy3a7\\\\\\\/U9Enjegrp4l9IFEqkuYdspVheaHaK6SyjGz6bXC2F+kIoI7JAu6j+oXMNHq03fSmyIJf+RfmQss5NdApQSACnpn7pqzq1EINz1bCV4BUGPTmo0uTB1yU5IFpWloZ\\\\\\\/GgHXd+TbrjFQF9dqYFI8APk740hdKCBtIcx2RJmmrKMd\\\\\\\/ibAl1Wsv0AsG7dyVzun+qc0clhGuFMC74UDKY50IDXtyhZahCZ67okyR5gy43WNmjj7mrq6W8ik13dmNhq5KjkcR\\\\\\\/ygun5nAaqNd4D+RCQvileOnXd7vOP1\\\\\\\/UC7NuOEtxajQfwDvf4ZqfV9WwQcKVFPOwT25Ql2kmAn\\\\\\\/6GufTaizIREyKnF26TSMonFmmDfZeqlv5OYJdLi+w2VjnfaScGVvRnmtCKG\\\\\\\/SGOxall8yk1x4yHaclK2PTLTfsx7b10L5lv92n2lM9Svi6PmBjTIpxxQlOUtCIcxRouTGqwRr0X7MMJ+zir9vbCIkQCoUw\\\\\\\/Yh361\\\\\\\/Di7yAT5FpL8A\\\\\\\/JEWCxkUVKw6uIDSfSze5EMu2qRRaO0YYDJ29vqcSdRMfuhzI2Qsh3SS9pPmmbAJSoKg2YOemFW3ajEKiFYA8nmLC1dcMYTM11Z5E8meli1dk5nu8c71M\\\\\\\/YYP6vl9ALi9CVX26EsR66LapZ1b9llHs48y4E10TPvx+XjiipcKS\\\\\\\/eUECWa5zlQ6ZYx8PaR+6RNGp+oUvIVkYMz3s64jNC03mDZfimJO0SFMakLkE4cIdkJLVMpngh93FL7sduZ2i2C1vB2AKHDpB5AcSyfCY5bnKwEiow8AzhlIIhpIkoSlpj3y20gOnpfc3DfxNJXImsO3fi0\\\\\\\/mvLqJ579GRQY1R8li6hcDzZF6uza5lF28l2HBTm08Uu70fDSBK20qKsOBY8xTnytiy+3TVkNagsn7DvyRyb5IRAdzQmfPok\\\\\\\/iAK5xMea7m8lhve7hDuhftNg1QPxmNUC817CHdF2PPw4XEGIq1IDAa2SZ7qZFq4SzPPSWLYsi75hHvusB71ogXj3JtnuzFqA4a58vz8u6rrJfSA1f9ASthAh1fEO28Yyn\\\\\\\/dEVlHYFb\\\\\\\/0ITgFz49BBZNsDI9F5S5JEAwfc9pbaOYNyi3uPJuRugs0wd1KCxQPeiQ7HQzyuwli6nvAy6wqQGbwYs+f7xNJVwsmcBFTX1mZyWRh+QhX5rSCc9OoOY0CXUzz0EunC\\\\\\\/ZjtcMCitq8i0GnoFnEOQvIzxy75MNYlbLbXiUFJ3O2KcMLaeUlP9fxwQHoxUjhBT2AgsVFZuYk79wUcR2kQvX5QEsnjRaMaY8MYKoVFLSFr07+minpSLqJPWMITz1D+\\\\\\\/L5GztnpESOGnV3BVWvv3ViQ9AQ+p+OdiMR6U5HhgnNKmOt1a2gbInaQiD6Qojlffw+QzP73u\\\\\\\/HqjTnVhKwc4Nm5c4zrQxpZU6GJtHi1fPBoM9X1y0d321rxhxwu0h06MTQAzRmBnbzrBiZMbvr6Gn3BZ7YrOFgFhJH4h3Jw2rnUS8YpPK6oCkuhr\\\\\\\/3m0fhMhG8tPELJvISyoASR\\\\\\\/RfRL2UxRn3z7VoicJOgte2u4rfUWq8xYtdlXYzR43p3Tz8kf8oOLSdeoUkYJIvhImaAScvBEcJWl8GmQMl08w+BSXl4UViLQfWeRD+CGtO2aUK3Rn0M8ZVJ1FtGD93\\\\\\\/eR68V3LjpLnKjJz+IQSE840jdiU8SF95S\\\\\\\/XcUxMCBIBrwBQS5lrvwtmtEm5y8fQ3gvqFnVRqza0CvLtYDUSPrvoDykpWy8JClhtxtgBVoulvsvHvdH1zGpDAIScayIesnYxBTiMZV\\\\\\\/kHe9XZSUYUC7iWHAyQsUsWErNslM58nZbXIBGcVyqqtaG6FNhW4B4Mtn7YOomaSjLWzvR\\\\\\\/M3vkd\\\\\\\/s2EpGeVMYX9+V27uus8jySKcVSZFOrVBimcWgAkuW3oUdLAOuZWTUY0wQXRtcKeREtla27qntqA8WV3Iva00JwqizLcbRjNphQZbcuUZnxWQFgj90ixC8q6YMUMRu4R\\\\\\\/RrmGhZKjNDPx3DCtUGxwbVj2r1hDBOw+924cn6fSDErYIdRHrwuqEYiHj7gUOfO7gaFIUVWBGDe2lymmCR+51xOCPfjK1q6foxJ6vuUw\\\\\\\/OWWqLppufkUaXSSxPvB8fc++3Kv7lgUm6UXZ3rTYA9xwCUj1b3EVgGixsBZ1zRHEEyIUQnZx6dIonzfq0\\\\\\\/Stlty\\\\\\\/D6Z6pqgsFKk+JlTvpZGNj+5hicfiaDtyzQMGKlBR40BbfQY0mlvw2gQ8N+c5PPRhcR7YycEj8LnfEfVQES2NpaaQV5qIDuiVbf\\\\\\\/kDEs1DDUo8W+foLx0Yi5ppLeX\\\\\\\/AMdhDQhs0Zt5EzEtgO1JKmrhEhh4ya4KkAbkGC1CoM++H8vLq1kcvXzn4iNby0BBJkQZD1FwZxrl01quYTpn9JANQt34BaOwovF9UoC4oYCIjUqWFCe6ORWobWZ6ooDbGaZwR3cBvaKnAyqMaQDRk33\\\\\\\/alB3nIMxVtaTWMhXBsXWP99G7hF3tnrBGzLiq5ft9LKD1ZonEp1oRH8YQ74mbqAwe4UBK8B9uhOVIXeC1Ois\\\\\\\/AuEFCoRDvift2KqC29NNSMS2geRx6hc9yCHHzPyB4mvsUghuGKXGkWd8UQTIrmZ0QJLiorHXi0whjWXFBBa9tUbqfKHDCI2S2IhrjGodlLX9s5cmLH8JHdwM8Pp4kPdN6GS3ePa5aRoU559zvFOv39M6bYpLaoe2d4FFFGVM1ldRVY87it5P6qoAbvwRm85DSmM7DdNA+nYynw+GJBBUCz2FA4tBp+JN9c2OfCKE5RwWDHx+R1sZ+1s5\\\\\\\/ehHOIpv8IrU8ElvEkRH2kum8iT4\\\\\\\/VyFk\\\\\\\/tyOkZfbF7sGQcELropQe3xeBbLBGJ5lrne1oVtUYzq8wbSDEKqsxH3A9Vl3KOH7QnJ9\\\\\\\/lIqzVikicGjpOC+yurgKWdGeHAGTw3m14ZW8VnvTYfDWCrYbRHBQJtmt+gLkVdO8D4yLz3\\\\\\\/wN\\\\\\\/\\\\\\\/YQcI\\\\\\\/dDiTCI9vGUC227SMTN9tK7yB3sM93JhG610TXfuA5BCIUEFLebo6K65dJcwLaJwnCgXgX4ESa1nEdeyhCRcNrR+tEZWIQlj\\\\\\\/2BfsY2pd1R9XKP0Qvc5vBNjSfRrIej46nFCIpYixhN5mRZVD3g1A71Alg\\\\\\\/3U7F+CBefWYXmrtvtDg+NrR2OqlxBkK6f2zRCfnMSBBJGxrkFoHhCdtDcuP1USXdBvrfYbUdpjucDJaoHCJCuIiPv8fw5G9qTMsNfbbfLxSAaGcMJZgxhLkzMjujf649549LSW3ghmecJdjaAQI2nMrWMl9bOK3YRuXYYG3XG75R7G1b+taS3qF1944MKH1T3AP4KjD0bFNo05s28n3LsAm0kppcnGQPu\\\\\\\/8ON43qWyIiq5igz5VVhvSloWcg9wApfHjSduysj2\\\\\\\/MChfue967JkYsT3Gzke\\\\\\\/f26fOCmxMCQoFchAe11ZCNDG9EmTfF3fWw3O36fvKLE3ZcPQmpQfxPixBMF8jEw\\\\\\\/X7ehnEUssbzX1WV\\\\\\\/G4gtLbT5hjdZ4xVHdXfKb\\\\\\\/rOpBuhzPCpjEt7v6mlT5l44j0oDPTI1b5FoiwrOlMyAwi13vZKEum4O7BWs1+w9g9gCfHj+nWbHYf3KI1i4rSQopdLtqpmS6wMmecSAthwDGjIT\\\\\\\/G6L802QxYOlmHwSz3F3TlXhCc6r9PMcrTkJrAsvt74SWXic5xfiv5Y5mo5ct9C0g\\\\\\\/4N54zcOyIMQlJnWC5\\\\\\\/NegqI7Pzh5I06lqkyf8uLyCsBRVzYUO1f2DvJazuj1NGF5to3OAVNbuJjeWMpwvTIYMOsU2k\\\\\\\/IYr3XqaUhodoDA0ABfNTdVtIHI2xvYCdUGofyxJkP5EPhZe+a5mYbp0K3kEljmIdVlUPP8Z7J3eSht8YqzmKFsm6WMwBsZ4sl\\\\\\\/gREnwbh6T3k8eJsYcUweFVaNDNp6YrgXmWD\\\\\\\/q7cquQH84MI1IYeX+xolhQIDLPqPrQzaedrvDm8jgFO96i0aqJ0jSJ9oKO00G9kW4hF98\\\\\\\/xVNMFZnL51PWQSgh1A5DHu6rxaqj4lgECy\\\\\\\/rbNkS\\\\\\\/k7wRuo\\\\\\\/wnmg4rkjDXR4nWNvRYGgOK8ZaFD2aWNPrKOq1aeMBKJBjrZDGruwA7E8AsFjPgQGN8ak9MlndmS3AzKWZ75INwjdyllutyfuL8TvVusEdhRW5RR2Or9FKU5GzTQd71wLE1nLHUzgdxMWrtXdYa3Q7i5A05aG+xKcpbwUKc2sZiFtFvWy+6EK2iGEe6tZQ7OEw\\\\\\\/0R406ng2Ftghe6zVvTAae2o0wv0ve3WBXZvUrOKjjcfF3aFH10dh40FO\\\\\\\/G3fwdD3ch+BkOgs56I68qBLsZPFqzZ\\\\\\\/guHbgtFTjne1GPez3Drkgvv1YQexrbu+BGdku1ZwBCbhkoQvsgYfY7mN6AXDtHub5Tn7UemJrz3O8ptea4y7h72tf+acy7XK4hsBLT6RDRfERy3uH+joDwFfZ8FKVb4pzqgoWhrwliHlNw73Z29f4kKqYMKBfklbPx24b7P2CobXfJj2XYR9c3HHIX2xuk9tDNFdy8Tnlx4AeixZEXLZP5FQ6yHit2eNIw4TO50L9sxz4FQILUOm9k1QYq7JbRld9uxiXAvjs2JJZfIjYoybJ9OCRWLzdJJQ6u5kYAmBZEUiwEkzvUpQ\\\\\\\/xvjCBwWK5RzuzeZjyIDQ8WFgMRFAnON\\\\\\\/Q4uDyKheXWooQevIBGV72oT27Gps3Pqa4x+o+2pPh6ivOWObE9nipMjWizhC0VdO2LY0V2LLQtUK6vOSvnvwqs3jr8+cITpm8zXmfcxhTSVpEktQ8a+kk+AcPdoPYhYvpxHv+hZwnXivP4ESNs7Aeoqpg9mI0Gyjwczk6vDOguG0jzAuilR44yJwpCIbF9vzcTdkwj7xto1T\\\\\\\/\\\\\\\/IaH9oQn\\\\\\\/J3OwapNT1SaQY10UbeVW+hUqujiJ0drAXTOQTfp+mZWx874vwqWMxLJbm\\\\\\\/rZ4LDK60\\\\\\\/JcDsW3NAXVKGLTHahoQRe8d+SsAsKK2wysaIfsbU1YNc7TApISOst7ZczJQAgrXtEWRZUOK0nIcW1Ebafkkp4ni+gsuxmBEvxxcoALm96VGRjsdwjV26MkbhjqeNKZNFhR50zp98ZWlwgF+yArfwxZb2SkOBgV9MClIqgVBCXBTglmPOHRyGeBwZszH6np2SNqR+RHTrPy+nawEKmQ1pqglTAQT7aL+vwUCV8HIAX0e4EM78eEEDnqZRL0mj5aEnfkV\\\\\\\/9qkgdBQEG7dn3Jsoj5uvUEwtlkQOn3ekYSQIRlRs+mMqdQpD+WOxNLs1zJxlaCW6stA7lQHXTaPgqpYk0a9VN4vMFGWhmSlul7JofOMjPindvJCU6wXbRAYk31Xt8jMvnPulfw0BLacHqLxJ+3spurjvCxGNSIxLigfWlQSKh3ybxFOnAYvyJ+1XqCM9KYkJVkvVCceqQ0OrnkrFzCal39dGKpB5AJ7n31TiRPbyx9nnPFqLGDXfEQ\\\\\\\/pkia6CeGnGfajeFs7Ue30XqV7kFXxvsemY1AUfa7XNL+Jle2u6Y9TVCh3U2efra8AFvezZdtpsDBAzI5yJ6LbzViKoyjuOhJCqmy6gEvSSEL1JDKmILJ8zoFnv17MSty5huXbxtkK3C9xxQDl5f4VSuzWS\\\\\\\/lnmHI36QgMdWtkUnJ9+oP7IohbIWkwA0E6+Rs\\\\\\\/vbA3jqtZejBWKUDvM1TRDpcRyQgnRWzAkwRd9ZTeITHfO8eR\\\\\\\/11MPrM+CClwP1uDXAF2dWW4nCjrynuhiEJDNNRiIt3I2yMnKdqAmrdwQpO\\\\\\\/HyFwR35x8BkzrD1oU\\\\\\\/XeCakbuTLUOsJ5tEHPzblYGfo+KvpGCEhyz86dnnO0vX5gRXa6DlgG\\\\\\\/Bw3FQY6OEhWNRoAKqzNzke7i6nAyugX\\\\\\\/Qkx\\\\\\\/J6YAdOB2Khvo8N\\\\\\\/p3EAXhlWUe8YlvV7Rl4q8xD9dxzGYLhLjgEkgRV9G1lVUBzAZRe9EnCQHk+d48xaF2NLvtUkpovhRODCbVvYM2ayRvUp2YCfHK4x1fGUDxwiW+DyyxLOin6WmSt6Zumoo52kw9koHkiWdHEFaqdbPi83AJtC90Aojg0uiS2asEBinHnfDJ43fvD5Da3uh\\\\\\\/O4uUkAZlTmWIEwhevGwUSvhwcAwR2NghChqAJl2jp28DNyvMKu+sA1PjLtoHjcc7bF5rDbbXm4v6pZrHFnE165hyNKnFSkumFrjkh7Wm13oGvH12IdBVvbd6Ds3A8Mhbn2nhT2H\\\\\\\/bGFeqLCB+6XZV1N+tbT3m8bgUFuc0y\\\\\\\/gkJKtLwq74NluDUbJAB4U\\\\\\\/o3R0m8IvDfCSoWteNOuQGh5kAhnfvQWyxmCw3d71JmMLLEIN+rw0Jlt8xYN2DrXABCknEzAr\\\\\\\/WxAX2fy6RdfxZ18Gd3tsdL9FranMhBipt6qUpzOFE+MMidRBjSZUmghOuHBLRBbouugOrwgxWhLwHstX5HfKMosB0bTBZnak1KYDZnGYMyP2Ay9D\\\\\\\/xEuB+xv5zxXTD7uN24I3jidZLwXiTnN5YkpuACfX\\\\\\\/yauc65qBOBPylibjfEJl1su9A6VyFvHZEgOawyJ9OMzSG+cxThfVHSXPH3WIPBIlAPyoQVr781EalFcyIWIyvNjmyUj04qIJwVy9PUJEY0DqEbEsrIwqsULTbCN4WGVyEN8nh\\\\\\\/tMimIF0tiy3Pn82C7ec2lPOUIfd3qgBdm+x1Ut2hvb5av3w6N535dcCof\\\\\\\/SElYRzVzkAgDItMqOJjZoJIsWoqSXehgrB5XZmUTT+iwSYtIEFjlejoyZFpu3Y2MQIuaiGVUCl\\\\\\\/ibK3h2fpD+pGL2M242gmZzgiBXCMn8VNvAlDw5scDIvMNAsSkaWOU6Mo+PgRI8ZzL+SfheU1THK2qWQe6CBRAvIEeN8kir+ye03sBQdH55VkNyuSTR6SirnK8VkaKZhRAPC2SkTibVBUKjvmHY+ngKbirX1YQMprICuOajgMZcBxWhmSqmg\\\\\\\/m+uijGckgXeSC0LkjgoqpqzMLgP\\\\\\\/NFw2m+D7USDbYp4dcfEnp1JFzH0rwcHbEClOFAHuJvt\\\\\\\/H0OpkD89nn4Gn3aq6W7aazHJuUnyWN1WrOVjQk7mOiq2R7Id\\\\\\\/Bx58a0CqwRJIDuM3Cp1zQ\\\\\\\/KMcXmPSVRSRCJeer9IxVsZDged0\\\\\\\/l16gOXeaCa+tXosafWqq8voFP0uRqtVdqqXBurcHYYIIyrdgt5O5g8RL9UteSOv+aTd0aY1cjmDXyZUBdw7++8lMjPXN46Gwvbphd2Gc+GVWS+71bcwEKspPycoG8vR4ODxX9Zz15FQWM+lzfxPaG332hD1fuu7W2dGsMWoChGaeKMZFQ3mmjzjlxXCYGuR+KNfIGJsntHUzzzqBf\\\\\\\/k8WM2V52aLmqg7ACqP2RMzIuPyEqEMYGz3vxUjCY4yDeMTvn87KROx7pRVDozcL+MOXoyK9u7O21oCmyF\\\\\\\/2N1NKudlW1A8t0MMc7dMSBlMHs\\\\\\\/zFA2e+X9v4K2DaUNvRTupbSe2eBlRibbKIksUCaPLhVM0CzsGoLJqQw4UTYIE0hPHQe+QMOkrhG+fPkq4kR6VSC0RaxumpjtauLwOR6IQpt2mUpLbUhmio0l2jGYOTBUI+EbgxXfa+wfwCzGHlIbzYqLGkS\\\\\\\/brXL4+qLKKmgJXGzIYwSIc+O7TPmMjJ0ZZheYfPYFjfSIGRf8uNnI0tMJvwUvMod4KEx6K0+8vlolSM94wduZkk3tCHTB4\\\\\\\/kLRInX2TvTJk03qCqYgbszljeokGIu5\\\\\\\/4535ULCQFX11hzPNCCVgyxeiep\\\\\\\/+KAyrOyrc8FTb9UvGesa1PG3nbanb\\\\\\\/vjWdNPFkSLbazRirSTroikaZcOyCPXbj\\\\\\\/LlpU9SvxKNdnofz4ZOW0b\\\\\\\/+Pzni9wHECR6sNpkgbAEP5aVRd8uCzqpgdWy34JEgYsQgWp7orfx2YpNw+cHoLzVMKbEdQoGoIm4doxEVtvMYPm5wGhUa3Uj3\\\\\\\/Uxm4sO2GejRpMR+Pto3F7iB49re9b6DL+0zpFplIqfgnkgFmR1UaRqwxRyFCBlL1eifxXphjHbcntwviYPkYdcirDOImWWKnIsSdz6KQekVxI+44SxNgWbPO6csYPsSq6tbIqeryEFv1\\\\\\\/JrDpHm5+0CO\\\\\\\/u0CpuyWXPgigu1ew+h+fhx13VhR7fW97AD2q3mYRGVPkV6zPqe\\\\\\\/pf5ArEI2xETIHo+fPX7OvYo36Qmfi7mWCsRXatNSyWROQ3C9jJ1Oq4StYBDQc8lv\\\\\\\/\\\\\\\/hIl3iXsjw4xbof1ay3+E30U5hfhHaiTQZGgXPJs+CR+MCQBvIwnJOW5C64WBy7q7Wft\\\\\\\/aOqQXO1bP+F1Pimfng5deKHL+1T2a2FZ43+eLUZAanDYpUxraKyW0p56B\\\\\\\/44bmo8vXc\\\\\\\/Y4CF2\\\\\\\/hcPp9GBk9AvayX+0Ojs16HYXvedZjUdUHXhyB5O7yzerfHJsf5SOJKXwNtf34+EBym5H8L30Ac9LeCsy64eRvH9RNetMYVQlJp9QOfw\\\\\\\/jBCYIV22KA1Xx1sn6yhMQ90Q4sNstFcmiGzvGhTCkOdk9+mq4oSjbnCWvj80WYOkD2So8gN4gUmmCOcwhbpt5NOijt+bNZ7SylNegCkSnHqtqNTIyZY88sQfaBxKJqJe\\\\\\\/ww\\\\\\\/pRr2HM8P9NayJl2hN4nwtkfTgqOWg+5wHLWbD4EvbzTZeMai7vwvoHKPtyylwaT2i1t82rnKOzwEQDQL9uX85p4SwY55DJu+rifL\\\\\\\/ffcuPob5\\\\\\\/Fdv2fDurXrkZmLEKY5NEHmWdfmzWKXN+jQlpEcgkCZUbi6PZXx1WxS3nANlOcxRsEsOy940b0K8N+gNQcJfy2LKiZfozrjTJTsG++eRiPnUbrQWn7tqw7cXgkzwOCj96\\\\\\\/Hp7QonMDc1aoRuM3kw1BkRrka1ZeKwcKXj8Cbw7mizKzJCxJPAMG5gv2u+naYG0053r4yKYzjhzg14R00zGzLNZblnTAW+TiOnP7d7L0vzqjpWS9h679k9FV6kY8b4sYohaeO272yIeEzVp3vqmjFeFJHlIc\\\\\\\/fQ5bykvVIh9JnjxBwkttLoeM1PLgiet\\\\\\\/NW\\\\\\\/bHC5Ds4ghTx4xQ6x2H1AVJC6twPj+SmDKWu5Vh2T5PcIuT+DEoBz\\\\\\\/siYyN7Qfyv3RwsfDqdmdA7kcHjIMku1X5E7PiuJQU5\\\\\\\/7vUM7iMV5qW+qAaatAJVhkHGeO0V+o03y3+OWU53VNbBASO4\\\\\\\/5JznfxtIn9O2M8pJfcBBtVUyeRcQBwQI48zMujTRJHTw1l6\\\\\\\/MmIbFK4bAjCZ35a0j\\\\\\\/OMSUD2fZyQIQ86sd7B54REKROVgWc0sKNBnrMMhBg3+ioBZ9U42PMWRaV3gX2s\\\\\\\/O93mjJErkVkBJ+GmE8bGIqDHUTEplpFQvtdF9RFEq30R1DFN2DY7zJAG3gQHO2ziQm8I8iz\\\\\\\/ctpk4\\\\\\\/mHELYwNTntpRBp3S1zWf\\\\\\\/\\\\\\\/zubeJ1t9ADVZgJUaaeZ5je9ONTISMGApCU1n4UyZ9gksuLtYJr7\\\\\\\/c+SA2vDCSWQgVwdyV+qa0m5GTwqxpC8VdxR5q2UX\\\\\\\/cbHgIU0LU\\\\\\\/0iDaOccUcOK8Jp74O4KDl8dqBV7Ucb4teRjI\\\\\\\/MQgyz9N1lPWDZH6Hlz2+qx6eEsB54YAJJhdiX+cUJKzCh+ZWD5mpKtTcM7k\\\\\\\/XjXn6S1js2RCg1tZUdT3CSHyJcoRl\\\\\\\/Kml7yuncYWxNXlOuQx3Sy4ZoQubppBM5GX71Wz3As1zWRfHXF9HnA2AB1ZNHcStFsqU07vd5o1lS1EXrPYGLMZYw00FaT1YZpThykezp+RXH07ZdQ\\\\\\\/Tr\\\\\\\/8BudbcBDYmmtm+fEcDLjajteWVH7rRaW5Q02OkaP2NKmJ2pvrYU\\\\\\\/l+r8wWfAQtg4dJEQjCwdPmUCKv5svxHIM0+rOepQvsowhmkowwlNp3IXmFXFwCji1qqGvc9yhqIMkhdG8FTMlC6rHRQ9HoYWg7VY62KX2gJ+XeiGDbZpItJP6u7VSpBrZSSY2lYdIt4CBYM71vCVZmVVwpNFMguspDYxEwZy2uVIXfNtvA4HA5TtBK6wbQfqeUeMLUHKgsdrnDQx+6LOUOWh3VhtK+bKKhGLk1Ure\\\\\\\/SejNtr4j6U5a9rMDP6jPHLo8bQhpkrjKkYfGu5SYIZV2EgXmt2ZAVMY1q\\\\\\\/nqUE17N51+J0yQ1v6yo+6oiIhvii9nb3G5LQMbkYjA2M2hQwX1IwkhQ+KQp7zD8Nr+0ZZ+ijkuWnhmBsHfKggZnsKtVdxwERmQXCf+S86e49NJbGVWkhRSQF8Cy447pJh5VBczldRHVWf58RnqqZsI0fkrTWl+0HEIycVyI6OjGyvMoSHvxB5omlxeUMRbIeOwT3DFwQmuF6DCTcJjptZo\\\\\\\/h8viLgueiRlrsTpsdGt6XdMbTs3cmRhoscSwsnjjXFj0+d6sz7p7jX38tfEh8kZSWddxBNdVUwlPAUmWv\\\\\\\/PRv1CpFHFLclJa+bt7DLCJ0ktohmwaif9HN1EjTZAG\\\\\\\/CNV5o5+uh0M8fS60STCGgA2I65XtqPQj9jYQ2lGcUTKZnSPa81dc\\\\\\\/b2O7r7Z8CLO2LRwjjmLhodYNBYGrjb1NhNpt\\\\\\\/jjTrggSTd1xaIhdFTkjpS909+uvofHQAIEF7C7IxebF2WYhrlL2PbMPG88eLon6hhb2R87ICEMxEVbiZORfO+pW+fQKFUzCVH6SfostKdYSfVN\\\\\\\/rp1\\\\\\\/swIV9P9v+VEdf0+plomw+XXWmVAci1TBQLccHQ9lK9eEnKVcLWB16Z9nZ1o\\\\\\\/xVSfmkpC5Tb\\\\\\\/pQ0YExHXDnIm2TsaZRLel\\\\\\\/qqGZvaCvvfoXRUaYLrrAOEkirIVAV9VBKytfbzTQ0lJcxdFGqFQLbFvmePIYwBo6WmWIwuq8sijowH8tqslXdQtopBKF6+2QAgqnsD3IZMViFsEW94v2UNJEFyOI+m5qDZe4pMDLuIIj+MDHkRnS+gLnqY2EbGSKTy+hPXRiRRGxRGv3DqDiLWQKs51u0RpC4YYazyN176Fe\\\\\\\/djQGwJWSmO2ccS5aPQXdLIi4QkFxSKAIMLXNqHAM3jQWiMOhkwbCtbKp3SMehDjQRwc0hHUJAk9BgUuTI2skjRG\\\\\\\/RaUUfsd7kTeaxsbvmLZEms\\\\\\\/86lYusf2+qIDmt7sOyjwetGsF4Z0duvX1QknzGZXrPPl9EzNovnzK+rCAA++iwiPqkF5TPF7GJ9I\\\\\\\/BC+3MhzFIFwh47h0d3P3j62I36E2oVnyrzUumpdJlqddebbmp6futZbmmKGva62yTUwar\\\\\\\/ZVkIo95FTXd+RjnkMlD81kRIYDqq0Hw3CkGrDxVB4Z5Vl6\\\\\\\/cuUfpuIyRXSQ9BZrP3IM5qnfM78w3lwYIQ6e7pYPU9YlcF8dlr0JZRd0uFzo0MNZKm9aL2BJgPopEw5TY9p2pd6ngLtqUp7GqG5ZYZoxR1gPvQe8V54o5Y5K79yV1uiUOYJVeqGVUBQHkurKB5DuB8VJd129bhqji8MthoQulML9zAqoFn5UqgJWXGA9CQAa1Omm1kd12Ljigx9ctSMiNyuKeutG0ZWmCBKjUnOY0kB1McN3U+AzdiV6k04MacyyMxUXkpZ915QEEGpB5nMsJztvtWqvy674z0qldkad0pTxIzH4uKCbE8IpQOk6kigQCR\\\\\\\/ox+t6I8yuNoJM+E\\\\\\\/hBC1cLBaL4DgBrjJwILj8BLQG2kFN7O5dspegUI9Ibu5av87M1bIKAmurTmDDZF5v\\\\\\\/02zuoyJRuv4C\\\\\\\/8LV5YF6BmYV\\\\\\\/\\\\\\\/Kr+NcDh\\\\\\\/hoUoVB0XzZMh3K3Jgaj9UMc+DyYer8T8gR1CGYxLbh6Ag64jj9Vyd\\\\\\\/xSDuvA2dXUc6eu8J7fcNS4OPxl02mqIKwhbSIrbdx3BxrqHbMxN6x0kWbEv4FEnBtVMq2NDaTYLq8paS2sbRbpUGQlbvnT4OO2SupopeF8Hr77130IufWTN9n2pU1012T742I6B6sxaG\\\\\\\/Alf8MXsgP0VDwB9cRQiXSUe7fcNnKf2CwbkJOjwTMf+OZu9m1loZbrC7c3D6KiMAA8U5s9eGu3lf9xY371v88bIzZkjXp53xzgECZ0sezIo2mc5EZFW7waiVGKkIouInXgbn1qkx1tvNBzbPdsVa9l0P06vQzI+BhJKlNVknaG7Gg+NSy8BstneY3x3nAAJQaev2xdfYZpTUqPzDfL19ralY6mkpb4Bef7GPHNQeESALc5BcS6T+XL7qGIUua7gebukunZT26aTeSo4qyC6tbdUT3qN3M7JCoz4S\\\\\\\/ZMrQ39tCLHinql9amAObSPQ4eUHo3myrdBbEOLod8sASQuVusWWlpuyOdEdE781hjAC923yI1r+JxqpksYpdn18dnJYCG2+zOL7Avgk\\\\\\\/7G2Le6aDjiHx69Ta9YZqd0BlhtcIpVMXjjLjevt6Wr7UnklYtL0sKMb\\\\\\\/tN9jzA4Sok0sycS2WSz7VRORYAmL3Izl9GszcICd\\\\\\\/hjwdCzTR9KQvaA\\\\\\\/URtITrOdkjdFG0CzRDxUji+zwZG\\\\\\\/I9EyJ39JV2n919K2lOEry3w3EsN1THPSMNEpuaXk03+Ofj3yJ9On2iwQpeA\\\\\\\/HOuPzdAtSMDWjmneL1auXyXaJm6DsjOYsqVbnPOXRFbnnwB7j6L\\\\\\\/PD2eJxfL3L0oRBAOHckjP5YV8qajYtMN6eZHw9JaMvd2TstHrDzeCfSfmwSqJ94P2n6JYxfdYvbn7LlXqoii1BSFYPzol3qrv64++mc99xnuCJRbLldQR4X0lj2vDePQY0vYL1A0fuldV2DcrzfyLFBXptlKJePa3Pv9xmyU69EQh+\\\\\\\/X0Y0QWR92AjX4JipHkOOjKIntM3jNgPbOq4e1p5iXKTGTGMYTSltbgHr3go5jbPb6b29hV2Rd195Vvbg1yBdWy1p5UYTWU\\\\\\\/zwZfo\\\\\\\/BDTBMA4lWELZS2YIMJVnK56X42VY1y0O84SdMEu1Ajjgqb0aqsbjrgO7FgP+1qYOHau2\\\\\\\/YSvsGNbUZtV6TxouT1YucGvf1Mn8dswu\\\\\\\/x4\\\\\\\/Q3XNZ5fcaYEGAZeloz8tGMjEyyg1ppfO1Am3mtoMTAKILN9ooJjDHxzJ8fVAVN2+AXS0HgyVenfjF8F0QVn55KBuN\\\\\\\/yILwqfKN7JVFwfYiavWqeAZOt8SlJJgsgONbzZ\\\\\\\/47\\\\\\\/0KMoIxN2WUYZVPF2Pm8s6PibMcjMS3KaeR+2fuYyw+Bsxps8TfecOQSVXu+5GajurSFjIpk8Wew2g4GkrfArRbyxu8xf4ZwIY9tSzPw8aaNOWz4iB+UnrY6wZfSTD\\\\\\\/BV8gSF5sQU9Kqy1O0gwrf6p4IEPasJOPWP7j7ooy8D9ipDTNmdmDMWaogU8BBCrmvbM7kaQmwUIHXbg7\\\\\\\/sodLp6KdSHsTHJjuaqqmJxOf7d26ACRCo0LNzxyqqIykGaVrIzEyeT1+XkCsHFvInIKfyIgtl\\\\\\\/VAIm6Y32HLi9Bw3rQXBx4CNzhrgrmhbGEKVhuvW5XjYG+Md3LWSLUDIn0NU2MjP2vrqtsDZRURSXlmE8GlqErksdkJmQXLqCjA4iwCrKRw4P3E3kLXVTa4\\\\\\\/imAEBm56D4r1TbiHSy37j42hWG4MuT3jUSEjMePer76dfGTcIhBFIzqNP3secMFdomDH5GCUv3MkGsfyNur\\\\\\\/mK\\\\\\\/lzGQdzUnj52EliRjgk+3+845mFThebOFzxH0VXJ+glrEXf2j5\\\\\\\/mhRVX7Jgr2tlITp\\\\\\\/wT5PREsPtuV8jH+syZzfThGVPxoFiA9xuYjv1JoPuCOmngfnsJQtcfU9LyxdpaKqyCDkNfWBty6PmHDA6AmbaIX533\\\\\\\/HlaaK4m3A3gU9+Vc\\\\\\\/dJcZ51GhpH9ZFU4tt\\\\\\\/DaPKxmBd6iXAekkMwskpGkNJt9kzUss5SvmjBVB\\\\\\\/p8Z1j+i+RZGqi3uQ9FP8+dnecV4tMmJ+LJsAnte3u2FEsSXYDciNKNdDDi0d8Ma3rj9A7XVGIhG6VZdLl0zWWcNPLuEswfMFjt66p5z47k09881SxNjA1BuvXb57r5Hx7gz0uJzioi0av94dpzI08IWXf3FfjbUylRwof49pRm1KXx\\\\\\\/0Z74o6WPffGqraF2FhvVqgqatjhBLKUJtm+ZnmhIxTG+6P1aFVPQNanwWMvvMl+dXwvi+t+rJuvFlHjtu1CbcR6\\\\\\\/z3rWlGNBwFRkMutrrgssssoZOiy\\\\\\\/UsVH2+LJOaDMIPLolr3yy2rSZurbL2U8reFzN\\\\\\\/CKHCX5w\\\\\\\/a9n1WD8GgcsgqRDHkj2cZNH0swKVd1ZfIkMMl1c17KpvhEPrTWIxLp+hyb+kkcm0ploTsYqQYPCQ2o+zMwUZeHKiKWItml7zjQ\\\\\\\/SCmx86pz4NR94jQMK\\\\\\\/Us76Hh\\\\\\\/mVWTwL0zGrCl3qp\\\\\\\/MFE7hOoa\\\\\\\/Qtuh08skQk28laM0EyNR0p9sTR\\\\\\\/7\\\\\\\/hIpxyJEhsxaTIEgExT7WOw0oEUdPkGA9wyQqS+fadCxhBMlF+pczwN4UVMvY\\\\\\\/7NGi9cUfSZjJw2K92Tz9\\\\\\\/rbAvEaFtEuGTOIJmHee6z8lHBviMAO6Afr4WL0f116NjVAkFZ7EkmA+Gy6ZzhA2FgcNN0\\\\\\\/wjFV2y0hPgvRA2Q+xlVZ5c9XUuVnk\\\\\\\/iYLMTdXNIyYc6qvJiwbWO\\\\\\\/VuGsp0sIQktkQB1Of9q3c20vKt4\\\\\\\/T+tEGxoftSxs\\\\\\\/l\\\\\\\/1pIFKo2a08puOUNjcvCS8FzgA++a25DN1nALyw7RGHLp8Ylig2mAgNeB5A+XI8Dq8Lh88NbmYfKxNoU3Tv9pfGL7ybk+07CgxhTp0\\\\\\\/RnII+chu3zfxLNiJSBPh0+nyG5xcPVRipPtRTPlxvidSOfugdlzTv7MO8XMszgCFhKREUybHKLdbKXmR5HsfFhT+TzpglkdxQyvxyQoF4IH8urWEegC+Qtx9rEXhNVqjpx2+ZG\\\\\\\/iFkjkEAIGyyrVOHA\\\\\\\/OtW2ZDrFI5t0637DIuaUi5dQ32A5c+EHuieuAwjaTv61FASSo\\\\\\\/yGhdwTaftI3cO5N1a+iQIIDWNdB3jHm58x78R0RfNz5HDcz\\\\\\\/Qo3tkEoI9yAww6mIRouIxSAX8lufbIVUs0b0adx4wCgTTaeLbdxP9h5T4qREdc8SkOb3wmquS+VM3lgT5mblYdfuer9gSQqQ3VPbMc++8J08gpNA9MP6QxCH5\\\\\\\/y+vfqUMWW8coR3ERnuKY\\\\\\\/QpplbhRRxcVKRqyq7TSrZbP4C0H9y4U9sTfejZAgkHEz66iWtrOxhwogLiEpe7ojQGaSVEcCV37amQCZgMj6AqcR069OusZ9vNYMh8X9PPExJGW9QoKYRkg0WBd1MIEGCbFn9ZjsCwvds7nhAh+kb3LqsrVCAGAuB5iQnUCo5ty8UzuQHk5\\\\\\\/afhJiUwD3nq5B1YOsYclZdwlXaZfgvhbRSnhuLMvoVa1L2fKhBIrJ72+M96cTFk60bpk0rzyp3DVDRjJyrErhNnBO\\\\\\\/I\\\\\\\/nuvEBCb6uJ97kOtBPkTf0m\\\\\\\/Ng9mI8bB\\\\\\\/asx3tGT\\\\\\\/NIv3w\\\\\\\/8HOc3T8IMrWtadxRAgjN\\\\\\\/wDwT8pQuQNDMlV4J0giphjLZQjAqch0K44MSQShSp2Uw37OYmtfzByCvLB8xYIytKu6Inkcww6HY5nO\\\\\\\/XTcDExrYSBYm0L+EUJR2D4svh\\\\\\\/BcKWEo\\\\\\\/OZPzgv+EBDpaebykUE7ZsvAN6+aVo6WR6WGZoHVqkXhM66\\\\\\\/140lS1LzHdJPcmtMoH9TV8f6gcoXMIvsBtZKdCEmi6qORCNhFsmftfft3hwnnJUOT8WOVB6Dojx9SuV5wwp5r3Y0\\\\\\\/TgymghAwNFpxAdysN4\\\\\\\/iOic5fYgpVJMKbHOUPLIjYVdBAFltTN8GfLkAT3xzRdM29U72QiSMz5agPG1YHE2rpu6d\\\\\\\/wt4pLAS2NtZL+74GaaOcfOCD+jj35Baf254IXRA6SMOqAxGAr1\\\\\\\/z2pb2XN9mdqihAnLJY\\\\\\\/0UJEGvhwLnrlLQ4rHlKEv9K0uKmsJwiq89\\\\\\\/iQ9I1Loh6\\\\\\\/OzO\\\\\\\/n3gq3yo2f\\\\\\\/\\\\\\\/ESoUZvqThHfdIBX\\\\\\\/USVVFvEDxvNBdHCoLmwNtoMxjOQZgNmsz3FWIJL\\\\\\\/2QtHgIrbSEec07teP3s6mvon0Moc42OXurHoNmt6PeYSZiGB4P\\\\\\\/oADLqJa4m1QFQi97DIDf05LKQELutgJVX9wd7EW3iiH1PuHTfHEszKvIgPyTwceAKfAXcDWqP\\\\\\\/RQke23eiahP6ypUu+AUmc2Z2Bkp3IXUw2GnaI9KmaIHEugAVsNYPnxEOQbVVA3cajbX2Nl0GXHgrAOv3si4uPuG7B55leJZbFFG8f+XocF+LiwIYzqMEnLUWFgwpxdIJ6W3khqiO3ys6gPMDW6OT25pBVn2ORzITHWAKmuE5G6SxePVVfv7+gZIY1zUpprq0VuNdiIYfBFABBh2zr9fpl2Q26LfPxUYmubp6RhQCpFwh6ymgRnjvwf55GiV0QpZTdHCt86\\\\\\\/A7YGcuclep6fWAC+NUvdg8CnpNm5GB6WihAHDF23XLt8aA8TnuivUQXaDnGwaXI0KXaUCz\\\\\\\/RJxeLWkls6ZIHiFmhBrx4f4Ae2xMr1AN40NP\\\\\\\/JpwU6unHusKxC9vvZW\\\\\\\/OTqcpgNiAyHHaBPpyVsqDPqPaWxOAhmyr3VNDNoG1unWib6WYq5a8+s2T5AvoVmChxE+1jlaXU4Vh6EzQBaenqnkl55nofBtAtNf6Jvud0b6QctrEuSpa4Ddp31ro6c3LJ9Asv4d5VvDXtKkFDwtiAyIOKiwh8EYQQeCvImpByfaku5IyYFvyN2aaOiezShTU6hc5dXxv6qLFu1D9vXNuOctFuOPj8kHJE8+t4uVbHFDort3NQu9aZ4elySA0SHTf0inM\\\\\\\/nsf+emCqAQcc3hrab4u3FCQT4GLKPlsWRwIH+5d9XTrerpITRUeH+rTwKwZvzzljIFRRTzYu6PCtK4EfIePq4RMyYclHJOujyC+VfK4Nx\\\\\\\/ui5ycaDFf+WePuQV1mY1Lf8k0hiYcQj9VzWrVQIRgMRgAA1Ay2q5apX8H7e3BBIDuiJQgjHh96DGwJPyeunIZ21jPFEEQlr1eDFJlSF+GenV\\\\\\\/OYEYEEBdNvgxCwPt\\\\\\\/HVQQfa5qrXis4adHCCBWNl+sucehBLEMYqOVInXgt3eDDP1+j5UDyo2HPQfYmg89P4iUk2zfWZGjfWe0jQqviAo0\\\\\\\/V1Bo7rvsOCdJ33i5qYhbK9d\\\\\\\/ihbdds+4fg15JZSYoLdVScCWXc4gHUl0ogFZPx6AgLUHwAc2SDwo41DR4tghbBy9m8jzbRUqHoFozR2m7A98iDpTSm3J4vTI8YFq1sKRU8GQngqFOh6rXUwYOTEHNtSYnLPPdfpu3HidMnhW09BLzNmt\\\\\\\/lkXFdGnB9ocFaU3A6nHcqWaBq0jNLtitg6PSMAjZ42RESXNZcPZZi1qZQv\\\\\\\/u\\\\\\\/YrWIjqIQGuhCGcKxqUoDgpN31Nbein8nGyGeRF9J73UyMDkLQqUtY6bq+zNHwiQ8KhyZA60yRjBPeLaf+dxCqjj8eq2JNztLq670fE49LMla7FiUgPcgMhsLmbjKNzD39Q8GNa01Awwa2nDIHGoYLV14SPv1w6jK4rwhkqhAFQTkau2gohNTVeVkclSQzZTMoWXFuXzmxC2GtrATNjbXai1gA7ZCDw+S5VymAd1M1rfJzvHOmeVosDdgU0yJTmkPIofum1NyE0MwHelao+NtjKnN52anxsLR+LgwswOxXnqHmhGUZ9F77uFwgQRSAAYcH8O8JZ3T6vKPrBTvlr6qpoMWsy\\\\\\\/gye2CA9pd1KY\\\\\\\/fPbkG0rGloc9Dx8V5A7zgh76e6HqHjBxA06wqRAMP7EDPUy3tCrXUHSY89++Hvdtj5OS5URzGvOPg8WFOtMJK6ZmiYkMZZVRWk0OzY\\\\\\\/WEIWcqGwsvaKOTILeVnAMX8592ZCZtnvp67AtTNb+GZUY++F7gtj1waY6SKFaQddGHUUdZrzcS6dGzbis3JpvYvWHoLJ9qXdEM2r12dNsGuvfItCRCyTNmNu+NOeVtDKue5v05yxtYrNJUQke46cZkiLvuTovgRbN3h0MJ4xXY0mehAv2\\\\\\\/KKt7aD40DXmFAhoCRMWquBbCzkPvnp1\\\\\\\/wWleBPQkalP28snCnUlGmxtUCw3SqzS6++LQESv054r2DeXfHmalpNLs+QxXHCqOGT2eIImqOt7WZp0UP7Fh82BkJFUM0sgeScGRQMdOHl6hH+GY+870uoFIxFG\\\\\\\/ClCC8KzdWM5WqOCoSssFkKbszztLOXAjvS\\\\\\\/dyBu2Or3kIzfRTV4anLkw3\\\\\\\/hqdVIiWeWiEkyUoyiKQiT0dALm+UzVXjTo\\\\\\\/J31\\\\\\\/aySdpuZotorYJd8gCX2UQ2SrGt27knsfw62hmbqyc55tD7kUuAy9o8Y0rtEzhQyR2S4nWbJ9a2WEThOxz0EK6lqSnvGp2ajl1\\\\\\\/KFM3+73P03HCD4e17Eti9QjDDH46qeqnNhUDHbEKbOB+8OSu2sszLKKnZp9AZmtlFF528T2CxKcO+SmqlqnWuOx2kImrr4Jg77VstimRpYbF0sTgG5RIlyJYJOEL+ZWGUaKaCy88OUXTBMSxrQYY41zBAsb8pBjY8Q\\\\\\\/A7OlXe99eM0IQ0RzTaHoR0\\\\\\\/f+8SglvDvimBQODL+71Cx3ZPdeSE7GNI+sUPL+OTviKLLz0B+gKz2Sq5lzEMCx79VeZlc4RCjKM1dxV7f\\\\\\\/uSY12gWR0eoFRe8NOkQQI83alfl0DThGpCDtXf8tFJV\\\\\\\/vXPrzPNJIOl96RjyJIz\\\\\\\/cDnFmNS839UB2T4vH5St3yXfzCIH\\\\\\\/rmxSM\\\\\\\/2LluJKIUvHlOmKmXHlDGSOUpDnS07ZndcXWSN6HhtfUWzZrhckxubx+eoUnPK8eS\\\\\\\/FL2kK9kD8+TWLB9Fc+8VTdtyooz7aXex5tPEJQJBR9zS3vL+opj1F9gE7ptZqEnuX+hM0v9NSY1Hllv1WQdB\\\\\\\/DEKAF7uJ7n0WOfg\\\\\\\/mjQW2eHxepUlp5Jni8okRYC46ebqfkI6+11K9bsrCtdzQSgoKJGShNLzCpmWGuBepSoLgUElALyWlkNydWcjqlq1My8hIo2BoXwuWrNSyz7FZjxepr8vSW2Lu0BxjG1GWxIQ3Ynb1KNpjGYJRzDOhAK1DTqSbzgjOOMN9acD82heEz7pOu8k4ca263yspUOLV5nsnIGJqkW+ycr2NzVboKXC1gxe0fEWB6DYMEPCP9diT97ONlZD9ngVNN0HhFlBO7oPMZdmrS\\\\\\\/an3ohFy7Pwa8P3tU\\\\\\\/H7Vh2ys9hVcmNpW0WpSGqiXYm8EzJaZka8t7VLIrV+93UdLavL41a+rWrX6PaK7cMCQeRtaURcMbk73tPpYRHGe3Q5XUnBbS25JKMO0liR\\\\\\\/g\\\\\\\/C+BD8gMWm3kNwTX8SBHQrZgzu51TyWeZmS+RSPWYsnX60gvA86tuZyESk5Af7d+nbjd\\\\\\\/J+vWqQH\\\\\\\/nTsD3AAPrxgONmopCtmS5BVD5UlWrDigEcMzDv877TCOpShxXNy2GUruIXDn\\\\\\\/jpOiSEgo5e4+dGoaREv8aE\\\\\\\/qUGd3NHo8GobwitmTlPs4I63+zNMlZ24C98InhZE9LeMaCvKSmw0pBPT9fNPGPTs5w7jXCHCiRi8FGwP9GARwMSBBn54BJKrNsNyvkHzkk+ZTP4gUoogGEFsDn4WMpyoWSEcw0a5E5QeE1W+jJ4pDPEJaVf3PSqy7NQIQeEWJVBrhicwUivEx5r9yRPQHw\\\\\\\/HCtZ+CQUBL+PdHDWdso8U3ayITosJikzgHxkh7JUEx025Y\\\\\\\/NgJlJiVEfUgM\\\\\\\/RKJBXObpXFyyetNCrp4tD64EBBKXBtUD2\\\\\\\/beOXqrQ58OzdfImT2I04AjS5yZAb+j6\\\\\\\/G2So4QYMlLuPm6BNfVL5zsPHMhUdDsSp1OB\\\\\\\/uZuKOYu2rag0JgG5NbQQWVSFVUH5MeTzJia5NqOZyTQ97asmYp2rOq73S+aRlH2SYwm15apOCoOnofSGlH\\\\\\\/Tkz+1rWUnFUJAX+KkhKkdRae5VFKmc0OiqBLTJdpsnBF7ins4fJjmPUKQYKUgxZc6y1d7HV2\\\\\\\/gFQs3uI0YJ8Q0GBcriXEFsB++X7U49yd+UirtM+\\\\\\\/nJp3nQ++pGfg0VsdqJ2Vtpm2dFnAIk6sfLv0c41jhEEnrU9SIh8TKFVp+kYuKnr\\\\\\\/\\\\\\\/5LDJ0j6iGeiN5uww30cKHoIEyOOf1OZsdSW1NEBQqkkY2Tk+IqNPTZGKvsaoAmmDC5prk5A4sYeyP6ZdT0c3d+h69eitbMCaqPCDV1V8saEdQ8OxQLYMdW9W0itrT0eKYBAI\\\\\\\/Twz1vCljrIFqbWgmaHry2PR1eyN6tHcyumAr7Nsvl1WzYdR+rHFgMs0HixAUJL0xjDKopCR8x4YYqK\\\\\\\/JT7bMAt4B4wV2vOz34XGpnGf5jpKW3NGpcovDJZlBvf+YV+VQWnDDKtkLpe3lgXCk1heP1L\\\\\\\/F4xXPetyJXNbPzPHmKwLfC2GQanvtRlqQQ1lTNyC\\\\\\\/4xFg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022IiFzQIcesoWLkxZytpTRCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aa9f50564925a3013fa1f77da935156c89c682bad8cd42344b4e7df0deb58cb8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022abUhB9251BigPG5iDaBROTVEt\\\\\\\/Bryj3v9q25aXhDHD66fusR35hBvIoHmfrK6B3kqYgF1K\\\\\\\/7YE0QxA+v1dIQxUX9HfH8YNOtMlYxONRjXkHk+53W3Li\\\\\\\/apGOKvdIBM2LepTByg9UgDNc6G76qSOojmuaQi7tOFXmgY\\\\\\\/9r0pPwDl1Jmji\\\\\\\/XmiAnz3Szc4kkX3fmpwudD048ZLQ1LXNAFf9UaIqbUHjgfk9mHaoElw9aq9NPxpdmRXGYpP7nA11QIMLwPNW9WT0IAG2ml+u3\\\\\\\/LRmaIp1kBODHiE5lmRsNa8IvPUAHtSlZ9IciSTtxLmG2SFpQ8bCBtL5HSg5gHq54E9HUy3KETw0jMrFoGn4DP0FWiyYQXH\\\\\\\/0ok+VNM7huy1Xrz2ssEYMlPNAmjtXDZiQ0q45D3Q+C6bS11B031NAT\\\\\\\/DMDcyp59dsXIWtZc36vMTQYhJKMpN6arYTtAfYvhqvv2+Sh0P5q73A916A6iT1Y1TE6RbbenJDmdSLcv7RfxV\\\\\\\/oaaTEpjxWtvDg+FVL0ZhPdLIxPTjklacUbYIpcB6YBOSQ+yQUsag7Zn4\\\\\\\/qv9rbH\\\\\\\/BzlAWQnLXmyj6cxfuTS0zj6i9Cqdmp7s0owSxCsh3ITTPf2APK43+34rbUPyJqtN\\\\\\\/tYtQrBpOQFZ0IoLM68Yc8XUBG9pmHJ7hZyN6RuaMyC0Hffg8sDOCd4knSj7f5zbnIrlJKzQYUuAo4LqdrVyC4h4HiRjGG7BrFWosHcYvF1bhWqLl+TWCl\\\\\\\/xyi0wSDWRThTYcNOXv9ZkhafqZbCptcBlHlgnjswl0u4p8kGf2Ffo3qOrgPnsL1a2Re129HQ6xWbyNffRSsWb9xhWDOAV2tcU31xICs1XRGx2o\\\\\\\/EzG0eKblinoWxy+JpdNfUfxgZvOLYbgax9cngywP2BEvaP4gcYVb8IxHoObDEF+I5YgiJlZm1xRPXyP24k3BGQbdkvogQe\\\\\\\/UbMoAEVpcxCjcnFgT\\\\\\\/NgujdRuZ9Ys9J5cfL00WdgUi2A9VZpTy+fgxWYJiJwbmX8Qfs9WBFBaNi5fQ0a9q0dAW6s36PBahudc7rJUBUvvZC1tBvfMmZtQ3RevfcwkL1ST9L41anx+bot6ne7ItLZQyt3pwT9G33X+8aayphnhwvROJupafsGLfwdldRmVHt\\\\\\\/K2HEXhHKtOasRKCYvcsqsfnRaeAlVFJzhJiF0u\\\\\\\/LxsPLldebv0CGSDttnJ0kDcM06DRkdwZUbZYOuww0MvRAbquqf5j5lLK\\\\\\\/ZR5u\\\\\\\/+pQNluaiA540v77\\\\\\\/sGxXGCeLoK8lJX7FsYj7os8UT3ocF3ln1CnT+wqrgQE82Vf4oyalTDJWr4q9IanyON1kL6UjOTa6scRJ+oH5ghCHTo7bjrE0EK5eAp0eTLEEgRm+YTnL1JE5XbCKvLTsBhUvWWAs9bpy24GGexCLiJIvnnL+asd\\\\\\\/iMoUC4Cn7moE5irMf+TIDKRDeQin61UbQy45P0AxUsmiIOBvZUEznmUe\\\\\\\/Ql697Kmx55OzBzY1pjmwm1pUGPgV3Gvb+JbGEiwVnMI1V1Y9IhTANpCOSlbkuNcIWnEEb0Zi6KCGeI9b\\\\\\\/uBbL43x0BFLWQecPzAR\\\\\\\/YzTRxcm0tRCyQwcaDHM\\\\\\\/iUqbCHu7MKTb9S6XvgasctetVUCJjsqGAZg55Ux0mpeZF7B5db38xXEYBwAsAZzhPlv3ZFn0rsj\\\\\\\/H3XjmpqnMJqURBGFRZbMYrR1IAjKKJSUeIF0cyRJh8rw6IOWi0kySmVXekkYBUnIYRZE3nVrVn+E3JLcutgxf2WoFrmIbFHhjlFuSSpWJHEft\\\\\\\/eNXnnaYBojVtYXXi\\\\\\\/oOt0mLn413QO4iP4JK+JSwUG9ZmLsudvDm8UTEgmlYTcnt\\\\\\\/IrSGVhEdjkGqcGyyTJ621QP7qWvlTFg1EG4qtq7t6wdqrpn0UkTRbztQsneRr6eI0PkBtuvMTUeK0JhR7aQtSsUTGowE+IcA+9zQk4hhbrlB3Y0mmnKH+9+JBGI32wDipBbyw7IXkqqo+zn1g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Pw9ApbDm0vGmgHC++RfeNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a1439e6bbd12d58440ace5394c0ae8800c4c0603e5bdeddf5c7619874404f339\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493672 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2135" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G6L_M8xltKgHdJSLe5FfNOAjsgD4dN8PBe24TMkGpbg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2135" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rSfdMLgchVAS1xCj6L-gF7WO-6ABC_fh7xKxHzxWDAw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2123" title="Optimizing Selection and Mating in Genomic Selection with a Look-Ahead Approach: An Operations Research Framework" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2153" title="Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.]" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493672" id="node493672" data-pisa="ggg;9/7/2135" data-pisa-master="ggg;g3.119.400043" data-apath="/ggg/9/7/2135.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory W. Stegeman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8084-9383" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott E. Baird</span>, <span class="highwire-citation-author" data-delta="2">William S. Ryu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7141-0013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Asher D. Cutter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2135-2151;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400043 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493672" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory W. Stegeman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BStegeman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stegeman%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BW.%2BStegeman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Scott E. Baird </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Biology, Wright State University, Dayton, Ohio, 45435</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BE.%2BBaird%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baird%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BE.%2BBaird%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8084-9383" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott E. Baird</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William S. Ryu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Physics, University of Toronto</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Donnelly Centre, University of Toronto, Toronto, Ontario M5S3B2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BS.%2BRyu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ryu%20WS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BS.%2BRyu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asher.cutter@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">asher.cutter@utoronto.ca</a>
<a href="mailto:wryu@physics.utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">wryu@physics.utoronto.ca</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Asher D. Cutter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asher%2BD.%2BCutter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cutter%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAsher%2BD.%2BCutter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7141-0013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Asher D. Cutter</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asher.cutter@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">asher.cutter@utoronto.ca</a>
<a href="mailto:wryu@physics.utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">wryu@physics.utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2135" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493672" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493672/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2135.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493672" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493672/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2135.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493672" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493672/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2135.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493672" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493672/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Thermal reaction norms pervade organismal traits as stereotyped responses to temperature, a fundamental environmental input into sensory and physiological systems. Locomotory behavior represents an especially plastic read-out of animal response, with its dynamic dependence on environmental stimuli presenting a challenge for analysis and for understanding the genomic architecture of heritable variation. Here we characterize behavioral reaction norms as thermal performance curves for the nematode <em>Caenorhabditis briggsae</em>, using a collection of 23 wild isolate genotypes and 153 recombinant inbred lines to quantify the extent of genetic and plastic variation in locomotory behavior to temperature changes. By reducing the dimensionality of the multivariate phenotypic response with a function-valued trait framework, we identified genetically distinct behavioral modules that contribute to the heritable variation in the emergent overall behavioral thermal performance curve. Quantitative trait locus mapping isolated regions on Chromosome II associated with locomotory activity at benign temperatures and Chromosome V loci related to distinct aspects of sensitivity to high temperatures, with each quantitative trait locus explaining up to 28% of trait variation. These findings highlight how behavioral responses to environmental inputs as thermal reaction norms can evolve through independent changes to genetically distinct modular components of such complex phenotypes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/behavior-genetics" class="hw-term hw-article-keyword hw-article-keyword-behavior-genetics" rel="nofollow">behavior genetics</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/caenorhabditis" class="hw-term hw-article-keyword hw-article-keyword-caenorhabditis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Caenorhabditis</em></a></li><li class="kwd"><a href="/keyword/function-valued-traits" class="hw-term hw-article-keyword hw-article-keyword-function-valued-traits" rel="nofollow">function-valued traits</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/thermal-performance" class="hw-term hw-article-keyword hw-article-keyword-thermal-performance" rel="nofollow">thermal performance</a></li></ul><p id="p-3">Plastic responses to environmental inputs, reaction norm phenotypes, are pervasive across diverse organismal traits and crucial to organismal fitness (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schlichting and Pigliucci 1998</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>). Such phenotypic plasticity can manifest as irreversible switches during development, physiological or metabolic adjustments, or rapid behavioral reactions (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dingemanse <em>et al.</em> 2010</a>). This ability to vary phenotype in response to environmental conditions can be adaptive, though maintaining the biological architecture required for adaptive plasticity may be costly or evolutionarily constrained (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Murren <em>et al.</em> 2015</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nettle and Bateson 2015</a>). Like any trait, reaction norm phenotypes can vary genetically in the wild so that selection may act on the genetic variation to shape the magnitude and form of phenotypic plasticity (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>). Two outstanding key questions in the evolution of plastic traits include how genetically complex is the control of natural variation in such seemingly complex phenotypes and how genetically modular are they.</p><p id="p-4">Thermal performance curves (TPCs) represent a common type of reaction norm in which a trait depends on temperature conditions in a stereotypical way (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Huey and Stevenson 1979</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Huey and Kingsolver 1989</a>). Temperature is an ever-present environmental factor that strongly modulates diverse aspects of biology, especially traits of ectothermic organisms (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fischer and Karl 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Colinet and Hoffmann 2012</a>). Ectotherms have evolved diverse physiological strategies and molecular mechanisms to adjust to changes in temperature (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fields 2001</a>). Behavior, however, remains the principal means by which ectothermic animals regulate body temperature by sensing, orienting and navigating their thermal landscape in ways crucial for fitness (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Huey and Kingsolver 1993</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Garrity 2007</a>). Behaviors are notoriously plastic and dynamic traits controlled by networks of many genes that thus lend themselves to quantitative genetics approaches to relate genotype and phenotype (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anholt and Mackay 2004</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dingemanse <em>et al.</em> 2010</a>).</p><p id="p-5">Summarizing the complexity of such dynamic traits presents a challenge in the analysis and understanding of plastic reaction norm phenotypes (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>). One way to simplify them is to describe the functional form of the trait response to environmental changes with fits of appropriate mathematical equations to experimental data. The corresponding parameter estimates of the function can then be treated as phenotypes themselves, termed âfunction-valued traitsâ (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ma <em>et al.</em> 2002</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu and Lin 2006</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yap <em>et al.</em> 2007</a>; <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>). Functional forms with biologically intuitive interpretations of the parameters are especially appealing for distilling multidimensional phenotypes into a few key, distinct components. Partitioning complex phenotypes as function-valued traits in this way also is powerful for mapping trait variation to genomic variation with approaches like quantitative trait loci (QTL) mapping (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baker <em>et al.</em> 2015</a>). Using function-valued traits instead of a single trait or a long list of univariate traits as phenotypes for QTL mapping may provide extra biological insight and statistical power to detect the most relevant QTL affecting a reaction norm phenotype (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wu and Lin 2006</a>; <a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>).</p><p id="p-6">Analysis of behavioral reaction norms in <em>Caenorhabditis elegans</em> demonstrated a neural âdecision-makingâ contribution to temperature-dependent locomotory responses using the same experimental paradigm as we here investigate in its congener <em>C. briggsae</em> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>). <em>C. briggsae</em> may provide an even more powerful animal model system to investigate natural genetic variation in temperature-dependent behavioral reaction norms. In contrast to <em>C. elegans</em>, <em>C. briggsae</em> displays latitudinal genetic differentiation and adaptive differentiation to distinct thermal regimes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cutter 2015</a>), with genotypes from the âTropicalâ phylogeographic group having higher fitness at extreme warm temperatures and the âTemperateâ group genotypes more fecund at extreme low temperatures (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Prasad <em>et al.</em> 2011</a>). While the vast majority of isolates of <em>C. briggsae</em> found around the world fall into these two phylogeographic groups, five other genetically distinctive groups of restricted geographic origin also are known to contribute genetic variability to the species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">FÃ©lix <em>et al.</em> 2013</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Thomas <em>et al.</em> 2015</a>). These distinctive genetic backgrounds also show temperature dependent behavior differences (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stegeman <em>et al.</em> 2013</a>). Moreover, the high-quality genome assembly and annotation for <em>C. briggsae</em> and a library of advanced-intercross recombinant inbred lines (RILs) with dense genotype information provides a powerful experimental stage for connecting phenotypic differences to natural genetic differences (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>). This context prompted us to investigate temperature-dependent behavioral reaction norms among natural isolate strains and in genetic mapping populations of <em>C. briggsae</em> with the aim of providing insight into the genetics and evolution of plastic behavioral traits.</p><p id="p-7">Here we quantify behavioral reaction norms as thermal performance curves for locomotion in response to increasing or decreasing temperature. Quantitatively, one may distill the complex shape of TPCs into characteristic parameters that describe the functional form of the response, treating the emergent multidimensional phenotype as a set of function-valued traits (<a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>). We found that TPC variation induced by different environments can be dissociated into distinct genetic components. We demonstrate substantial natural variation in TPCs among wild genetic backgrounds in addition to âclassicâ developmental plasticity in TPC shape in response to distinct rearing conditions. By quantifying variation in behavioral reaction norms for F1 heterozygous lines, recombinant inbred lines, and introgression lines, we map distinct QTL for components of the TPCs as function-valued traits to show that genetically separable phenotypic modules underlie the genetic architecture of the overall emergent trait captured with behavioral thermal reaction norms.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Experimental quantification of natural genetic variation in locomotory behavior</h3><p id="p-8">We quantified nematode locomotory behavior as a function of changes in temperature using the liquid micro-droplet experimental paradigm that we developed previously (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>). In brief, we recorded video and quantified the swimming behavior of individual worms placed into 1.8ÂµL droplets arranged in an array of 18 droplets on a glass slide as we manipulated temperature with custom hardware (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>). Image capture and processing of swimming worms was performed with custom programs written in LabVIEW (National Instruments, Austin, TX) with further data analysis in R to compute metrics of locomotory activity (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>). Our standard assay involved an increasing temperature regime from 25Â° to 41Â° with 1Â° steps occurring every 80 s, recording video of the behavior for the last 20 s of each temperature step at 15fps. We also performed experiments on wild isolates with a decreasing temperature regime (from 25Â° to 8Â° with 1Â° steps).</p><p id="p-9">To investigate variation in TPCs, we reared 23 <em>C. briggsae</em> natural isolates at 16Â°, 23Â° and 28Â° to quantify genetic and plastic contributions to locomotory behavior profiles in the liquid micro-droplet assay when we exposed animals to increasing or decreasing ambient temperature changes from an initial 25Â° (Supplementary Table S1). Animals were reared at a given treatment temperature starting as zygotes from parents raised at the same temperature. The fitness optimum for the Tropical <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> strain is close to 21.5Â°, with 16Â° and 28Â° both yielding strongly reduced fitness (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Prasad <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Begasse <em>et al.</em> 2015</a>). These strains derive from genetically distinct phylogeographic origins known to exhibit phenotypic variation for temperature-related traits (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Prasad <em>et al.</em> 2011</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Stegeman <em>et al.</em> 2013</a>), with 16 of them having published genome sequence data (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Thomas <em>et al.</em> 2015</a>). We also generated and tested reciprocal F1 heterozygotes from crosses of two strains (Tropical <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> and Temperate <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>), using parental hermaphrodites that were depleted of self-sperm to ensure harvesting of non-selfed F1 offspring. Experimental assays included only well-fed adult worms with <em>E. coli</em> <a href="http://www.wormbase.org/db/get?name=OP50;class=Strain">OP50</a> from non-starved NGM agar plates kept at 23Â° unless indicated otherwise.</p><p id="p-10">For analysis of wild isolates, we tested 36 worms from each strain from each rearing temperature, with each block of 18 droplets containing 9 replicate worms of two strains. We randomized testing order combinations of the 23 wild isolate strains over a period of a week separately for the increasing and decreasing temperature assays.</p><p id="p-11">For QTL mapping, we phenotyped a panel of 153 recombinant inbred lines (RIL) that are genomic mosaics derived from <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> and <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> parental lines (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>). These 153 lines were genotyped at 1031 single nucleotide polymorphism (SNP) markers across the six chromosomes, which define 430 distinct genetic blocks for this subset of the full RIL collection created by (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>) (Supplementary Table S2). We performed our standard liquid micro-droplet assay on the RIL and parental strains with â¼1Â° temperature increments from 25Â° to 41Â° for animals reared at 23Â°. We tested 45 worms from each homozygous RIL strain in a randomized order with 9 replicate animals for each of two strains on each assay slide with additional replicates for the parental lines <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> and <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>. After quality filtering, we included data from 6907 individual animals across the 155 strains, with data from â¥40 animals for each strain (except PB1209 with <em>n</em> = 36). The number of animals that passed quality control at a given temperature point can be lower than 40 to still be included in analyses, provided â¥25 replicate animals were of acceptable quality for analysis (Supplementary Figure S1).</p><p id="p-12">We calculated several distinct but inter-related metrics of worm movement behavior: locomotion index with 1/15<sup>th</sup> second frame resolution (LI1), locomotion index with 7/15<sup>th</sup> second frame resolution (LI7), pixel center-of-mass displacement (at 1/15<sup>th</sup> and 7/15<sup>th</sup> second frame resolution), and the proportion of worms moving at each temperature step (calculated from a threshold of activity for LI7). We calculated the LI index of locomotion for each individual animal at each temperature step as the fraction of overlapping pixels of the worm between different frames of video: adjacent video frames 1/15<sup>th</sup> second apart (LI1) or 7/15<sup>th</sup> second apart (LI7). The LI values range from 0 (no movement detected) to a maximum of 1. We standardized the raw LI1 and LI7 values for each individualâs locomotion by subtracting its lowest value across temperature steps to account for the slight pixel noise across video frames even for immobile animals. The proportion of animals moving provides an overall indicator of activity across replicate individuals, calculated after we assigned values of 1 (movement detected with LI7) or 0 (no movement detected with LI7) for each individual worm from which we computed the mean and standard deviation at each temperature for each strain. These locomotion metrics are not fully independent and their individual values will correlate between adjacent temperature steps. Consequently, we sought to reduce the phenotypic dimensionality into independent components using two techniques: function fitting and principal component analysis.</p></div><div id="sec-3" class="subsection"><h3>Behavioral thermal reaction norms as function-valued traits</h3><p id="p-13">In order to compare thermal performance curves as a whole, rather than considering separately the metrics at each temperature, we fit a three-parameter logistic function to the behavioral thermal response curve from benign to extreme temperatures for each strain as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2135/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2135/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>T</em> is droplet temperature, <em>Î±</em> is the asymptote that describes the baseline locomotion at the beginning of the assay (performance at temperatures near 25Â°), <em>Î²</em> is the slope parameter that corresponds to how rapidly locomotion declines as temperature rises or falls, and <em>Ï</em> is the inflection point parameter, which represents the temperature at which the thermal performance curve switches from concave to convex (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Prior to function fitting, we normalized each individualâs locomotion by subtracting its lowest LI1 (or LI7) value (usually â¼0.06-0.12) from the scores at all other temperature steps to account for the slight pixel noise across video frames when worms are immobile that keep LI scores from reaching 0. We performed functional fits to mean LI1 or LI7 values from each temperature step from a given treatment by using the three parameter logistic equation in the non-linear model fitting procedure in JMP v10.0.0 (SAS Institute). This procedure could capture most of the change in strain mean locomotion scores at each temperature (<em>R<sup>2</sup></em> of 0.98-1.00). We compared pairs of strains by treating TPCs as function-valued traits in this way and used the 95% confidence intervals for the parameter estimates to determine whether pairs of strains differed significantly from one another.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Movement behavior thermal reaction norms as a function-valued trait. Three parameter logistic function fit for four example strains, including the RIL parents that correspond to Tropical and Temperate phylogeographic groups (AF16, HK104) and two RILs with extreme TPCs (PB1137, PB1219). The locomotion index is calculated as the fraction of overlapping pixels of the worm between different frames of video: adjacent video frames 1/15th second apart (LI1) or 7/15th second apart (LI7). For function-valued trait parameter estimation, the locomotion index values were normalized by subtracting each worm&#x2019;s minimum score from the value at each temperature step prior to fitting strain mean values to Equation 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Movement behavior thermal reaction norms as a function-valued trait. Three parameter logistic function fit for four example strains, including the RIL parents that correspond to Tropical and Temperate phylogeographic groups (AF16, HK104) and two RILs with extreme TPCs (PB1137, PB1219). The locomotion index is calculated as the fraction of overlapping pixels of the worm between different frames of video: adjacent video frames 1/15&lt;sup&gt;th&lt;/sup&gt; second apart (LI1) or 7/15&lt;sup&gt;th&lt;/sup&gt; second apart (LI7). For function-valued trait parameter estimation, the locomotion index values were normalized by subtracting each worm&#x2019;s minimum score from the value at each temperature step prior to fitting strain mean values to Equation 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F1.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2135/F1.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493644" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Movement behavior thermal reaction norms as a function-valued trait. Three parameter logistic function fit for four example strains, including the RIL parents that correspond to Tropical and Temperate phylogeographic groups (AF16, HK104) and two RILs with extreme TPCs (PB1137, PB1219). The locomotion index is calculated as the fraction of overlapping pixels of the worm between different frames of video: adjacent video frames 1/15<sup>th</sup> second apart (LI1) or 7/15<sup>th</sup> second apart (LI7). For function-valued trait parameter estimation, the locomotion index values were normalized by subtracting each wormâs minimum score from the value at each temperature step prior to fitting strain mean values to <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">Many analyses of thermal performance focus on thermal optima, or peak performance, which is not captured in the logistic function fits described above. In order to estimate TPC peak performance for natural isolates strains, we fit a Gaussian peak function (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>) to a subset of locomotion data. The TPC at relatively benign temperatures between 18Â° to 32Â° exhibits minimal skew, so we fit mean values combined for individuals from increasing and decreasing temperature assays with the non-linear model fitting procedure in JMP v10.0.0 (SAS Institute):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2135/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2135/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>T</em> is droplet temperature, <em>P</em><sub>max</sub> is the peak value for the locomotion index, <em>T</em><sub>opt</sub> is the âcritical pointâ temperature where the peak occurs, and <em>c</em> is the curvature.</p></div><div id="sec-4" class="subsection"><h3>Behavioral dimensionality reduction With Principal Component Analysis</h3><p id="p-16">As an alternate means of reducing the dimensionality of the data by integrating correlated phenotype metrics, we used principal components analysis (PCA) on the phenotypic measurements of the 153 RIL strains. We performed PCA using the rda (redundancy analysis) function with scaling enabled, from the vegan package (v2.3-3) in R (with strain as âsitesâ and phenotypes as âspeciesâ). We included univariate phenotypes for the strain mean and standard deviation at each temperature step for five metrics of locomotory activity (LI1, LI7, 1 and 7 frame displacements, and proportion moving). We also included the mean pixel area for each strain as well as the three parameter logarithm fit parameter estimates and standard errors for a total of 167 phenotypic metrics for each strain.</p></div><div id="sec-5" class="subsection"><h3>Genetic mapping of natural variation in locomotory behavior</h3><p id="p-17">We used standard interval mapping to associate genotype and phenotype of RIL strains, as implemented in the r/qtl package version 1.39-5 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Broman <em>et al.</em> 2003</a>) in R v. 3.2.3. To fill in potential break points between genotyped markers, we applied the multiple imputation method for standard interval mapping in this package. Specifically, we simulated 128 possible combinations of break points at 0.5cM spacing between genotyped markers. Mapping also included two âpseudo-markersâ to represent the mitochondrial genome, inferred based on the reciprocal cross origin of a given RIL strain. For the two-dimensional QTL analyses, we used Haley-Knott regression instead of multiple imputations in order to reduce the computational load.</p><p id="p-18">We first QTL mapped 8 âsyntheticâ principal component locomotion behavior traits arising from the PCA analysis, using 1031 markers across the six chromosomes plus two pseudo-markers representing the mitochondrial genome. We determined a 5% genome wide significance threshold for each trait from 2500 permutations and Bonferroni correction (<em>P</em> = 0.05/12; 8 PC traits, 3 function-valued traits, 1 univariate trait). We followed a similar procedure in QTL mapping and significance testing for the three function fit parameter phenotypes for locomotion index (normalized LI7). We then computed 95% Bayes credible intervals for log-odds (LOD) peaks on chromosomes with significant QTL. We estimated the broad-sense heritability of QTL at the peak marker location as 1 â 10<sup>-2âLODâ/â</sup><em><sup>n</sup></em> (<em>n</em> = number of RILs). We also performed two-dimensional QTL scans in r/qtl with a minimum distance of at least 5cM between markers and 2500 permutations of the data to estimate significance for a subset of phenotypes. Model type was determined by comparing the full model allowing for interactions (lod.fv1) to the additive model (lod.av1), with epistasis inferred by a significant effect of interactions (lod.int) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Broman and Sen 2009</a>).</p><p id="p-19">The collection of <em>C. briggae</em> RIL strains exhibited marker transmission ratio distortion (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>). These included significant bias toward <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> markers on Chromosome V near the QTL peak that we identified for high temperature locomotion. There were also allelic biases on Chr III and Chr IV (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>). This likely contributes to fewer novel breakpoints in the NILs for these regions than expected from cross-over rate estimates. While some inadvertent selection due to slower development during the intercrossing of the RILs, it is primarily associated with distortion on chromosome III (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>). The distortions on Chromosomes V and IV are only significant for one of the initial F1 cross directions, and show no LD with the X chromosome, which suggests a cytonuclear epistatic interaction (<a id="xref-ref-51-7" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chang <em>et al.</em> 2015</a>).</p><p id="p-20">To investigate the QTL region on Chromosome V, we generated 13 near isogenic lines (NILs) by backcrossing RIL strains PB1124 and PB1147 to <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a>. We used restriction-digest fragment length polymorphism (RFLP) marker-assisted selection of backcross individuals for genotyping near recombination breakpoints in the QTL region of interest through at least 7 generations of backcross. After selfing the resulting backcross lines, we then affirmed homozygosity and genotype along Chromosome V, as well as for three loci for each of the other chromosomes, using markers in (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Koboldt <em>et al.</em> 2010</a>) (loci with âcb#â SNP names in Supplementary Table S3). The three markers on other chromosomes were consistent with introgression only of DNA from Chromosome V, though we cannot exclude the possibility that small regions between markers on other chromosomes might have co-introgressed. We also designed and used four additional RFLP markers to increase marker density along Chromosome V (Supplementary Table S3). We tested these lines using the same droplet assay with increasing temperatures as we used for the RIL QTL analysis. We also tested the NILs and parental lines in modified droplet assays to explore alternative related phenotypes.</p><p id="p-21">We tested for gene ontology (GO) term enrichment within the QTL region on Chromosome V using PANTHER (<a href="http://www.pantherdb.org/">http://www.pantherdb.org/</a>), which we visualized using Treemaps created with REVIGO (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Supek <em>et al.</em> 2011</a>); 33 of the 115 genes lacked GO terms.</p></div><div id="sec-6" class="subsection"><h3>Data and reagent availability</h3><p id="p-22">All <em>C. briggsae</em> strains are available from the Caenorhabditis Genetics Collection (<a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a>, <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>, wild isolates) or from the authors (recombinant inbred lines, near isogenic lines). File S1 contains Supplementary Tables S1 and S3-S6 as well as Supplementary Figures S1-S12. File S2 contains Supplementary Table S2 (genotypes for each RIL used in QTL mapping) and File S3 contains Supplementary Table S3 (genes with coding changes and GO terms). Per-individual phenotype data are given in Files S4, S5, S6, and S7 for F1s, wild isolates, recombinant inbred lines, and near isogenic lines, respectively. Per-strain phenotypes of RILs used in QTL mapping for all univariate traits are provided in File S8. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7590779">https://doi.org/10.25387/g3.7590779</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-23">We hypothesized that rearing temperature would differentially affect thermal performance of swimming behavior of distinct genotypes, motivated by prior studies that demonstrated temperature-dependent trait differences among phylogeographic groups of <em>C. briggsae</em> (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Prasad <em>et al.</em> 2011</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Stegeman <em>et al.</em> 2013</a>). We therefore quantified natural genetic variation and environmental plasticity in locomotion behavior in <em>C. briggsae</em> by assaying thermal performance with a liquid micro-droplet experimental paradigm described previously (<a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>). In these experiments, we isolated individuals in â¼2Î¼l liquid droplets arranged in an array on a microscope slide and incremented their temperature from a benign starting point (25Â°) in â¼1Â° steps to hot or to cold extremes (41Â° or 8Â°) while taking video recordings for automated image analysis and behavioral quantification. Like many other organisms (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Huey and Stevenson 1979</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Huey and Kingsolver 1989</a>), including <em>C. elegans</em> (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Begasse <em>et al.</em> 2015</a>; <a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>), <em>C. briggsae</em> generally reacts to changing temperature with a functional response like a classic thermal performance curve (TPC): peak activity at intermediate temperatures, with a shallow rate of declining locomotion toward cold temperatures, and a rapid reduction in movement toward high temperatures (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We then quantified the behavioral TPCs as function-valued traits to reduce the dimensionality of the phenotype and extract the dynamic behavioral information into biologically-meaningful parameters of the function fits (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Thermal performance curves for 23 natural isolate strains from seven genetically distinct phylogeographic groups of <em>C. briggsae</em> (Table S1; (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">FÃ©lix <em>et al.</em> 2013</a>)) revealed substantial heritable and environmental contributions to variation in behavioral TPC shape (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>; Supplementary Figure S2). By then focusing on 153 recombinant inbred lines (RIL) derived from the Tropical and Temperate phylogeographic groups to investigate the genetic differences underlying variation in thermal performance, our quantitative trait loci (QTL) mapping identified genetically distinct behavioral modules that contribute to TPC shape.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Behavioral reaction norms differ due to rearing conditions and genotype. (A) Thermal performance of C. briggsae reared at three growth temperatures (purple = 16&#xB0;, gray = 23&#xB0;C, orange = 28&#xB0;C), with data pooled across experiments and genotypes for the raw LI1 locomotion index. Assays either incremented or decremented temperature from a 25&#xB0;C initial assay temperature (black vertical line). (B) TPCs for each of the 23 wild isolate strains reared at 23&#xB0;C (raw LI1; &#x223C;36 worms per strain), colored by phylogeographic group (8 Tropical red; 5 Temperate blue; 3 Nairobi yellow; 3 Kerala green; 2 Montreal purple; 1 Hubei orange; 1 Taiwan black). (C) Standard deviation of locomotion index values (raw LI1) for each wild isolate strain shown in (B). Rearing temperatures are denoted with vertical dashed lines; gray region in (A) indicates fertile temperature range. The spikes in SD and LI at temperatures below 12&#xB0;C are due to condensation artifacts in the image analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Behavioral reaction norms differ due to rearing conditions and genotype. (A) Thermal performance of &lt;em&gt;C. briggsae&lt;/em&gt; reared at three growth temperatures (purple = 16&#xB0;, gray = 23&#xB0;C, orange = 28&#xB0;C), with data pooled across experiments and genotypes for the raw LI1 locomotion index. Assays either incremented or decremented temperature from a 25&#xB0;C initial assay temperature (black vertical line). (B) TPCs for each of the 23 wild isolate strains reared at 23&#xB0;C (raw LI1; &#x223C;36 worms per strain), colored by phylogeographic group (8 Tropical red; 5 Temperate blue; 3 Nairobi yellow; 3 Kerala green; 2 Montreal purple; 1 Hubei orange; 1 Taiwan black). (C) Standard deviation of locomotion index values (raw LI1) for each wild isolate strain shown in (B). Rearing temperatures are denoted with vertical dashed lines; gray region in (A) indicates fertile temperature range. The spikes in SD and LI at temperatures below 12&#xB0;C are due to condensation artifacts in the image analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F2.medium.gif" width="440" height="396"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2135/F2.medium.gif" width="440" height="396"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493682" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Behavioral reaction norms differ due to rearing conditions and genotype. (A) Thermal performance of <em>C. briggsae</em> reared at three growth temperatures (purple = 16Â°, gray = 23Â°C, orange = 28Â°C), with data pooled across experiments and genotypes for the raw LI1 locomotion index. Assays either incremented or decremented temperature from a 25Â°C initial assay temperature (black vertical line). (B) TPCs for each of the 23 wild isolate strains reared at 23Â°C (raw LI1; â¼36 worms per strain), colored by phylogeographic group (8 Tropical red; 5 Temperate blue; 3 Nairobi yellow; 3 Kerala green; 2 Montreal purple; 1 Hubei orange; 1 Taiwan black). (C) Standard deviation of locomotion index values (raw LI1) for each wild isolate strain shown in (B). Rearing temperatures are denoted with vertical dashed lines; gray region in (A) indicates fertile temperature range. The spikes in SD and LI at temperatures below 12Â°C are due to condensation artifacts in the image analysis.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Function-valued traits differ among genotypes and rearing conditions for both the upper and lower portions of the thermal reaction norm. Parameter estimates from logistic function fits to locomotion index (LI1) thermal performance curves for each wild isolate strain (see Equation 1 and Figure 1). Parameter values for decreasing temperature TPCs (A, B, C) and increasing temperature TPCs (D, E, F) estimated separately for animals reared at each of three temperatures (purple = 16&#xB0;C, gray = 23&#xB0;C, orange = 28&#xB0;C). Dashed lines indicate the mean across strains within a phylogeographic group (colored by rearing temperature); error bars indicate &#xB1;95% confidence interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Function-valued traits differ among genotypes and rearing conditions for both the upper and lower portions of the thermal reaction norm. Parameter estimates from logistic function fits to locomotion index (LI1) thermal performance curves for each wild isolate strain (see Equation 1 and Figure 1). Parameter values for decreasing temperature TPCs (A, B, C) and increasing temperature TPCs (D, E, F) estimated separately for animals reared at each of three temperatures (purple = 16&#xB0;C, gray = 23&#xB0;C, orange = 28&#xB0;C). Dashed lines indicate the mean across strains within a phylogeographic group (colored by rearing temperature); error bars indicate &#xB1;95% confidence interval.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F3.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2135/F3.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493536" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Function-valued traits differ among genotypes and rearing conditions for both the upper and lower portions of the thermal reaction norm. Parameter estimates from logistic function fits to locomotion index (LI1) thermal performance curves for each wild isolate strain (see <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Parameter values for decreasing temperature TPCs (A, B, C) and increasing temperature TPCs (D, E, F) estimated separately for animals reared at each of three temperatures (purple = 16Â°C, gray = 23Â°C, orange = 28Â°C). Dashed lines indicate the mean across strains within a phylogeographic group (colored by rearing temperature); error bars indicate Â±95% confidence interval.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-8" class="subsection"><h3>Thermal performance curves are sensitive to genotype and environment</h3><p id="p-26">Under all three rearing temperature conditions (16Â°, 23Â° and 28Â°), the TPCs for locomotion are unimodal with declines in performance toward low and high temperatures. For animals reared at cool to moderate temperatures, the TPC shows the classic shape of a steeper drop in performance toward high than low temperatures, but those reared at a high 28Â° actually tend to show sharper performance declines toward low temperatures (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A,B</a>; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B,E</a>; Supplementary Figure S2). Thermal performance curves also shift position and width depending on rearing temperature, with cooler rearing temperatures yielding wider TPCs that show more locomotory activity at lower temperatures (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C,F</a>; Supplementary Figure S2).</p><p id="p-27">Despite rearing temperature yielding large and consistent differences in locomotion that give all genotypes a similar characteristic TPC shape (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ayrinhac <em>et al.</em> 2004</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hoffmann <em>et al.</em> 2005</a>), the genetic identity of strains also contributes substantial variation in temperature-dependent locomotion profiles (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). While strain-specific genotypic differences appear more pronounced than general differences among phylogeographic groups, visual inspection of TPCs for the genotypes from the Kerala âcladeâ strains exhibit cold-shifted TPCs compared to most other strains (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3F</a>). By contrast, the two Montreal âcladeâ genotypes tend to perform poorly at colder temperatures and the Hubei genotype has the highest and one of the broadest TPCs (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-28">To quantify these trends, we applied a 3-parameter logistic function (<a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>) to data from each half of the TPC (the âcoolâ half with subscript âCâ and the âhotâ half with subscript âHâ). From these function fits to data, we extracted the fit parameter values as simple and biologically interpretable summaries of locomotion, treating the reaction norm phenotype as a function-valued trait to compare TPCs across strain and rearing condition treatments.</p><p id="p-29">First, we characterized baseline swimming behavior at benign temperatures with the asymptote parameter (<em>Î±</em>), which is related to maximum locomotion. We observed <em>Î±</em> to be highest for most strains when animals were raised at a benign 23Â° and <em>Î±</em> lowest when raised at 16Â° (ANOVA <em>Î±</em><sub>C</sub> F<sub>2,66</sub> = 18.48, <em>P</em> &lt; 0.0001; <em>Î±</em><sub>H</sub> F<sub>2,66</sub> = 15.57, <em>P</em> &lt; 0.0001; Tukeyâs posthoc tests show differences among all rearing temperatures for both <em>Î±</em><sub>C</sub> and <em>Î±</em><sub>H</sub>) (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A,D</a>; Supplementary Table S4). Animals reared at 23Â° appear visibly healthier than when reared at the extreme temperatures, and this may translate into them being capable of faster locomotion at initial assay temperatures. Alternatively, the thermal jump from rearing temperature to the initial assay temperature (25Â°) is largest for animals reared at 16Â°, which might shock them into slow initial locomotion as captured by <em>Î±</em>. We also observed significant genetic variation in both <em>Î±</em><sub>H</sub> and <em>Î±</em><sub>C</sub> (non-overlapping 95% CIs among genotypes; <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3A,D</a>). The Hubei strain VX0034 shows consistently fast <em>Î±</em> across rearing temperatures, whereas the three strains from Nairobi consistently exhibit especially slow baseline locomotion when reared at 28Â° (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3A,D</a>). Strain genotypes also differ considerably in their degree of plasticity to rearing conditions, with some strains having nearly identical <em>Î±</em><sub>C</sub> values regardless of rearing temperature (overlapping 95% CI across rearing temperatures for <em>e.g.</em>, <a href="http://www.wormbase.org/db/get?name=VT847;class=Strain">VT847</a>, NIC65, GXW23) and others being very sensitive (non-overlapping 95% CI across rearing temperatures for <em>e.g.</em>, NIC21, <a href="http://www.wormbase.org/db/get?name=JU1348;class=Strain">JU1348</a>, VX0034; <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3A</a>). This variation in trait canalization also extends to the hot end of the TPC (<em>e.g.</em>, compare low <em>vs.</em> high plasticity in <em>Î±</em><sub>H</sub> for <a href="http://www.wormbase.org/db/get?name=VT847;class=Strain">VT847</a>, <a href="http://www.wormbase.org/db/get?name=QR24;class=Strain">QR24</a>, NIC20 <em>vs.</em> <a href="http://www.wormbase.org/db/get?name=ED3083;class=Strain">ED3083</a>, NIC21, VX0034; <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3D</a>). These strain differences in plasticity point to genetic variation for the phenotypic robustness of baseline locomotion behavior in response to rearing conditions, in addition to heritable variation for the raw locomotory response.</p><p id="p-30">Our function fit captures the sensitivity to changes in ambient temperature as the sharpness of the decline in locomotory activity, with this slope parameter (<em>Î²</em>) also influenced by rearing temperature and genotype (ANOVA <em>Î²</em><sub>C</sub> F<sub>2,66</sub> = 55.94, <em>P</em> &lt; 0.0001; <em>Î²</em><sub>H</sub> F<sub>2,66</sub> = 3.75, <em>P</em> = 0.029; non-overlapping 95% CIs among genotypes) (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3B,E</a>). The magnitude of the slope (|<em>Î²</em>|) is steeper for most strains when the assay increments toward hot temperatures than when it approaches cool temperatures, at least when they were reared under cool or benign conditions (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3B,E</a>; Supplementary Table S4). For animals derived from these same cool or benign rearing conditions, we also observe more genetic variation in <em>Î²</em> when they experience the warming portion of the TPC compared to cooling half of the TPC (F-test for variances: 16Â° rearing <em>F</em><sub>22,22</sub> = 0.137, <em>P</em> &lt; 0.0001; 23Â° rearing <em>F</em><sub>22,22</sub> = 0.097, <em>P</em> &lt; 0.0001; 28Â° rearing <em>F</em><sub>22,22</sub> = 0.828, <em>P</em> = 0.66). Moreover, the cooler the rearing temperature, the shallower the slope for the decline in locomotion toward low temperatures (Tukey posthoc tests indicates <em>Î²</em><sub>C 16Â°</sub> &lt; <em>Î²</em><sub>C 23Â°</sub> &lt; <em>Î²</em><sub>C 28Â°</sub>; <a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>); but the cooler the rearing temperature, the steeper the decline in locomotion toward high temperatures (Tukey posthoc tests indicates <em>Î²</em><sub>H 16Â°</sub> â¤ <em>Î²</em><sub>H 23Â°</sub> â¤ <em>Î²</em><sub>H 28Â°</sub> with <em>Î²</em><sub>H 16Â°</sub> &lt; <em>Î²</em><sub>H 28Â°</sub>; <a id="xref-fig-3-14" class="xref-fig" href="#F3">Figure 3B</a>; Supplementary Table S4). Despite considerable overlap in slope estimates among strains for the three rearing temperatures, especially at the hotter portion of the TPC, we also observe heritable strain differences in <em>Î²</em> in response to rearing conditions (<a id="xref-fig-3-15" class="xref-fig" href="#F3">Figure 3B,E</a>). For example, strain <a href="http://www.wormbase.org/db/get?name=JU1338;class=Strain">JU1338</a> exhibits virtually identical values of <em>Î²</em><sub>C</sub> regardless of the rearing temperature (similar for <a href="http://www.wormbase.org/db/get?name=QX1547;class=Strain">QX1547</a> and <a href="http://www.wormbase.org/db/get?name=JU1348;class=Strain">JU1348</a>), whereas <em>Î²</em><sub>C</sub> values range threefold depending on rearing temperature for GXW23 (similar for <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> and <a href="http://www.wormbase.org/db/get?name=QR25;class=Strain">QR25</a>). As we also described for <em>Î±</em>, these findings implicate heritable variation for phenotypic robustness of the dynamic behavioral response encapsulated by <em>Î²</em> as a function of rearing conditions. Even though both <em>Î±</em> and <em>Î²</em> derive from the same locomotory TPC, distinct genotypes can show robustness or plasticity separately for each trait, suggesting that these behavioral responses involve integration of the same environmental inputs with distinct genetic pathways.</p><p id="p-31">The inflection point parameter (<em>Ï</em>) identifies the temperature at which the TPC flips from concave to convex, heuristically indicating the center of the transition from normal to stopped locomotion. For all strains, low rearing temperatures led to colder values for the lower inflection point (<em>Ï</em><sub>C</sub>; ANOVA F<sub>2,66</sub> = 215.75, <em>P</em> &lt; 0.0001) (<a id="xref-fig-3-16" class="xref-fig" href="#F3">Figure 3C</a>). However, the upper <em>Ï</em> values (<em>Ï</em><sub>H</sub>) are similar regardless of whether worms were reared at 23Â° or 28Â° (<em>Ï</em><sub>H</sub>; <a id="xref-fig-3-17" class="xref-fig" href="#F3">Figure 3F</a>; ANOVA F<sub>2,66</sub> = 39.29, <em>P</em> &lt; 0.0001, Tukeyâs posthoc test indicates no significant difference between 23Â° or 28Â°), and 5 strains actually have a colder value of <em>Ï</em><sub>H</sub> when reared at 28Â° than 23Â° (non-overlapping 95% CI for <a href="http://www.wormbase.org/db/get?name=JU1338;class=Strain">JU1338</a>, <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>, <a href="http://www.wormbase.org/db/get?name=EG4181;class=Strain">EG4181</a>, <a href="http://www.wormbase.org/db/get?name=JU1348;class=Strain">JU1348</a>, NIC20). This suggests that limits to tolerance for high ambient temperature might not permit further upward shifts of the TPC with higher rearing temperature, with the notable exception of the Nairobi genotypes due to their unusually low <em>Ï</em><sub>H</sub> with 23Â° rearing (<a id="xref-fig-3-18" class="xref-fig" href="#F3">Figure 3F</a>). Plasticity in <em>Ï</em> as a function of rearing conditions is pervasive, with very few strains showing ârobustâ insensitivity to rearing temperatures (<em>e.g.</em>, <a href="http://www.wormbase.org/db/get?name=VT847;class=Strain">VT847</a> for <em>Ï</em><sub>C</sub>, <a href="http://www.wormbase.org/db/get?name=JU1908;class=Strain">JU1908</a> and VX0034 for <em>Ï</em><sub>H</sub>; <a id="xref-fig-3-19" class="xref-fig" href="#F3">Figure 3C,F</a>). Temperate clade strains consistently show the greatest plasticity for <em>Ï</em><sub>C</sub> in response to different rearing temperatures, with nearly a 4Â° shift among rearing treatments, whereas the Nairobi strains exhibit consistently high plasticity for <em>Ï</em><sub>H</sub> (<a id="xref-fig-3-20" class="xref-fig" href="#F3">Figure 3C,F</a>).</p><p id="p-32">The TPCs from animals reared at cooler temperatures exhibit a â¼1Â° wider span between the two inflection points (Î<em>Ï</em> = <em>Ï</em><sub>H</sub> - <em>Ï</em><sub>C</sub>), shifting position more on the cool end of the TPC and thus reflecting greater lability in response to cold ambient temperatures (ANOVA F<sub>2,66</sub> = 6.39, <em>P</em> = 0.0029, Tukeyâs posthoc tests indicate Î<em>Ï</em><sub>16Â°</sub> = Î<em>Ï</em><sub>23Â°</sub> &gt; Î<em>Ï</em><sub>28Â°</sub>; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A-C</a>; Supplementary Table S4). Likewise, most strains have a narrower TPC (smaller Î<em>Ï</em>) when reared at 28Â° than when reared under cooler conditions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>). The widest TPCs across the wild isolate strains are found consistently in those strains from the Temperate clade when reared at 16Â° or 23Â° (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A-C</a>), which could be interpreted as a feature of a âgeneralist genotype.â Rearing temperature alters TPC midpoint even more dramatically than does genotype (ANOVA F<sub>2,66</sub> = 157.95, <em>P</em> &lt; 0.0001, Tukeyâs posthoc tests indicate significant differences among all rearing temperatures; <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations among behavioral reaction norm features as a function of rearing conditions and genotype. (A) C. briggsae thermal performance curve midpoint shifts warmer with higher rearing conditions (locomotion index LI1) (ANOVA F2,66 = 157.95, P &lt; 0.0001, all conditions differ by Tukey&#x2019;s post-hoc test). The midpoint of the TPC also correlates positively with its width across wild isolate genotypes, for benign and hot rearing temperatures (Spearman&#x2019;s &#x3C1; = 0.64, P = 0.001 for 28&#xB0;C; &#x3C1; = 0.47, P = 0.024 for 23&#xB0;C; &#x3C1; = 0.22, P = 0.3 for 16&#xB0;C). (B) The temperature of peak performance (Topt) correlates positively with TPC width across genotypes reared under benign or high temperature conditions (Spearman&#x2019;s &#x3C1;=0.75, P &lt; 0.0001 for 23&#xB0;C; &#x3C1;=0.41, P = 0.051 for 28&#xB0;C; &#x3C1;=-0.046, P = 0.84 for 16&#xB0;C). (C) TPC widths (&#x394;&#x3C4;) are narrower on average for wild isolate strains reared at 28&#xB0;C than at lower temperatures (ANOVA F2,66 = 6.39, P = 0.0029, Tukey&#x2019;s post-hoc test). (D) The temperature of peak Topt is greatest for wild isolate strains reared at high temperature (ANOVA F2,66 = 55.33, P &lt; 0.0001, Tukey&#x2019;s posthoc tests: Topt 28&#xB0; &gt; Topt 23&#xB0; &gt; Topt 16&#xB0;). (E) The peak performance for locomotory activity (LI1) Pmax, however, occurs for wild isolate strains reared under benign intermediate rearing conditions (ANOVA F2,66 = 20.25, P &lt; 0.0001, Tukey&#x2019;s posthoc tests: Pmax 23&#xB0; &gt; Pmax 28&#xB0; &gt; Pmax 16&#xB0;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlations among behavioral reaction norm features as a function of rearing conditions and genotype. (A) &lt;em&gt;C. briggsae&lt;/em&gt; thermal performance curve midpoint shifts warmer with higher rearing conditions (locomotion index LI1) (ANOVA F&lt;sub&gt;2,66&lt;/sub&gt; = 157.95, &lt;em&gt;P&lt;/em&gt; &lt; 0.0001, all conditions differ by Tukey&#x2019;s post-hoc test). The midpoint of the TPC also correlates positively with its width across wild isolate genotypes, for benign and hot rearing temperatures (Spearman&#x2019;s &#x3C1; = 0.64, &lt;em&gt;P&lt;/em&gt; = 0.001 for 28&#xB0;C; &#x3C1; = 0.47, &lt;em&gt;P&lt;/em&gt; = 0.024 for 23&#xB0;C; &#x3C1; = 0.22, &lt;em&gt;P&lt;/em&gt; = 0.3 for 16&#xB0;C). (B) The temperature of peak performance (&lt;em&gt;T&lt;/em&gt;&lt;sub&gt;opt&lt;/sub&gt;) correlates positively with TPC width across genotypes reared under benign or high temperature conditions (Spearman&#x2019;s &#x3C1;=0.75, &lt;em&gt;P&lt;/em&gt; &lt; 0.0001 for 23&#xB0;C; &#x3C1;=0.41, &lt;em&gt;P&lt;/em&gt; = 0.051 for 28&#xB0;C; &#x3C1;=-0.046, &lt;em&gt;P&lt;/em&gt; = 0.84 for 16&#xB0;C). (C) TPC widths (&#x394;&#x3C4;) are narrower on average for wild isolate strains reared at 28&#xB0;C than at lower temperatures (ANOVA F&lt;sub&gt;2,66&lt;/sub&gt; = 6.39, &lt;em&gt;P&lt;/em&gt; = 0.0029, Tukey&#x2019;s post-hoc test). (D) The temperature of peak &lt;em&gt;T&lt;/em&gt;&lt;sub&gt;opt&lt;/sub&gt; is greatest for wild isolate strains reared at high temperature (ANOVA F&lt;sub&gt;2,66&lt;/sub&gt; = 55.33, &lt;em&gt;P&lt;/em&gt; &lt; 0.0001, Tukey&#x2019;s posthoc tests: &lt;em&gt;T&lt;/em&gt;&lt;sub&gt;opt 28&#xB0;&lt;/sub&gt; &gt; &lt;em&gt;T&lt;/em&gt;&lt;sub&gt;opt 23&#xB0;&lt;/sub&gt; &gt; &lt;em&gt;T&lt;/em&gt;&lt;sub&gt;opt 16&#xB0;&lt;/sub&gt;). (E) The peak performance for locomotory activity (LI1) &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;max&lt;/sub&gt;, however, occurs for wild isolate strains reared under benign intermediate rearing conditions (ANOVA F&lt;sub&gt;2,66&lt;/sub&gt; = 20.25, &lt;em&gt;P&lt;/em&gt; &lt; 0.0001, Tukey&#x2019;s posthoc tests: &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;max 23&#xB0;&lt;/sub&gt; &gt; &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;max 28&#xB0;&lt;/sub&gt; &gt; &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;max 16&#xB0;&lt;/sub&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F4.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2135/F4.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493520" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Correlations among behavioral reaction norm features as a function of rearing conditions and genotype. (A) <em>C. briggsae</em> thermal performance curve midpoint shifts warmer with higher rearing conditions (locomotion index LI1) (ANOVA F<sub>2,66</sub> = 157.95, <em>P</em> &lt; 0.0001, all conditions differ by Tukeyâs post-hoc test). The midpoint of the TPC also correlates positively with its width across wild isolate genotypes, for benign and hot rearing temperatures (Spearmanâs Ï = 0.64, <em>P</em> = 0.001 for 28Â°C; Ï = 0.47, <em>P</em> = 0.024 for 23Â°C; Ï = 0.22, <em>P</em> = 0.3 for 16Â°C). (B) The temperature of peak performance (<em>T</em><sub>opt</sub>) correlates positively with TPC width across genotypes reared under benign or high temperature conditions (Spearmanâs Ï=0.75, <em>P</em> &lt; 0.0001 for 23Â°C; Ï=0.41, <em>P</em> = 0.051 for 28Â°C; Ï=-0.046, <em>P</em> = 0.84 for 16Â°C). (C) TPC widths (ÎÏ) are narrower on average for wild isolate strains reared at 28Â°C than at lower temperatures (ANOVA F<sub>2,66</sub> = 6.39, <em>P</em> = 0.0029, Tukeyâs post-hoc test). (D) The temperature of peak <em>T</em><sub>opt</sub> is greatest for wild isolate strains reared at high temperature (ANOVA F<sub>2,66</sub> = 55.33, <em>P</em> &lt; 0.0001, Tukeyâs posthoc tests: <em>T</em><sub>opt 28Â°</sub> &gt; <em>T</em><sub>opt 23Â°</sub> &gt; <em>T</em><sub>opt 16Â°</sub>). (E) The peak performance for locomotory activity (LI1) <em>P</em><sub>max</sub>, however, occurs for wild isolate strains reared under benign intermediate rearing conditions (ANOVA F<sub>2,66</sub> = 20.25, <em>P</em> &lt; 0.0001, Tukeyâs posthoc tests: <em>P</em><sub>max 23Â°</sub> &gt; <em>P</em><sub>max 28Â°</sub> &gt; <em>P</em><sub>max 16Â°</sub>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Much of the thermal performance curve literature emphasizes peak performance. Most strains reach their peak performance (<em>P</em><sub>max</sub>) when reared at 23Â° rather than the other two temperatures (ANOVA F<sub>2,66</sub> = 20.25, <em>P</em> &lt; 0.0001, Tukeyâs posthoc tests indicate <em>P</em><sub>max 23Â°</sub> &gt; <em>P</em><sub>max 28Â°</sub> &gt; <em>P</em><sub>max 16Â°</sub>) (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4E</a>); this is similar to the pattern for the asymptote parameter (<em>Î±</em>) with respect to rearing temperature. Just three strains have faster peak locomotion at 28Â° instead of 23Â° (<a href="http://www.wormbase.org/db/get?name=ED3083;class=Strain">ED3083</a>, <a href="http://www.wormbase.org/db/get?name=VT847;class=Strain">VT847</a>, <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>; Supplementary Figure S3). To estimate the temperature for peak locomotion (<em>T</em><sub>opt</sub>), which the logistic function fit does not capture, we fit a Gaussian curve (<a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>) to locomotion data between 18Â° and 32.5Â° where the asymmetry in TPC shape is least pronounced (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). The temperature of peak performance (<em>T</em><sub>opt</sub>) is warmest when we reared animals at 28Â° compared to 16Â° (ANOVA F<sub>2,66</sub> = 55.33, <em>P</em> &lt; 0.0001, Tukeyâs posthoc tests indicate <em>T</em><sub>opt 28Â°</sub> &gt; <em>T</em><sub>opt 23Â°</sub> &gt; <em>T</em><sub>opt 16Â°</sub>), although for many strains there is little difference between rearing at 23Â° <em>vs.</em> 28Â° (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>). This is similar to the pattern we see with inflection point values (<em>Ï</em>) and lends support to the idea that rearing temperature acts to shift the entire curve. We observed no significant correlation between peak performance <em>P</em><sub>max</sub> and the temperature at which peak performance occurred (<em>T</em><sub>opt</sub>) (all <em>P</em> &gt; 0.53). We also found no significant correlation between peak behavioral activity (<em>P</em><sub>max</sub>) and TPC breadth (Î<em>Ï</em>) across the 23 wild isolate genotypes for any rearing temperature, despite a positive trend for animals reared under benign 23Â° conditions (Spearmanâs Ï=0.38, <em>P</em> = 0.073). By contrast, we observed significantly higher temperatures of peak performance (<em>T</em><sub>opt</sub>) for genotypes that have broader temperature breadths (Î<em>Ï</em>), provided the animals were not reared under cold conditions (Spearmanâs Ï=0.75, <em>P</em> &lt; 0.0001 for 23Â° rearing; Ï=0.41, <em>P</em> = 0.051 for 28Â° rearing; Ï=-0.046, <em>P</em> = 0.84 for 16Â° rearing; <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>).</p></div><div id="sec-9" class="subsection"><h3>Robustness of thermal performance among individuals within a genotype</h3><p id="p-35">Our plots of TPCs exclude error bars to reduce visual clutter, but we also explicitly considered variation within strains and how this variation responds to assay temperature and to rearing temperature to learn more about individual thermal responses. Of course, the lowest standard deviation (SD) in locomotory activity among individuals occurs when all worms are paralyzed at the temperature extremes (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>; Supplementary Figure S2). The SD is highest near temperatures that are slightly more benign than the inflection points in the TPC, where locomotion just begins to decline (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>). However, high temperatures induce greater inter-individual variability for a given genotype as some worms stop moving entirely while others continue relatively normal locomotion, being most pronounced when we reared the animals at 28Â° (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2C</a>; Supplementary Figure S2), which may associate with a wider range of individual body condition. The lower overall SD toward the cold end of the TPC indicates that the decline in locomotion tends to be more consistent across individuals, with this more stereotyped reaction indicative of a more canalized behavioral response to cooling than to warming ambient temperatures.</p></div><div id="sec-10" class="subsection"><h3>Genetically distinct TPCs of Tropical and Temperate strains</h3><p id="p-36">We next sought to characterize differences between the âTropicalâ <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> and âTemperateâ <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> strains in detail, as the parental genotypes of a library of recombinant inbred lines with which we could explore the genetic architecture of trait variation. When reared at 23Â°, the locomotory behavior of these two strains differed most at high temperature near 36Â°, also reflected by the change in proportion of animals moving, despite similar locomotory activity of the two strains in benign and cool portions of the TPC (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A-C</a>; Supplementary Figure S4). The LI7 locomotion index metric has greater resolution than LI1 when animals are moving slowly (<a id="xref-ref-57-7" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>), and LI7 reveals that near 36Â° many individuals are moving slowly in <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> whereas <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> ceases locomotion altogether. This distinction in performance near 36Â° is consistent across four large repeated experiments (Supplementary Figure S4). An instantaneous jump in temperature from a benign 23Â° to 35.6Â° also showed <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> and <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> to respond distinctively: the movement for <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> worms declined more quickly over time, but retained slow activity for the duration of the â¼21 min assay (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5D</a>). By contrast, locomotion of <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> declined more slowly toward complete immobility after â¼13 min of exposure to 35.6Â° (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5D</a>), suggesting that <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> can tolerate 35.6Â° longer than <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> can. Because of these consistent and conspicuous differences in high temperature responses for which the Tropical <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> strain outperforms the Temperate <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> strain, we devoted further effort to this aspect of high temperature locomotory performance.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Thermal performance for Tropical strain AF16 and Temperate strain HK104. (A) Decrementing and incrementing temperature TPCs show decline of locomotory performance with more extreme temperatures with three distinct metrics: (A) locomotion index with 1/15th second resolution (LI1), (B) locomotion index with 7/15th second resolution (LI7), and (C) the proportion of moving individuals. Upper and lower halves of the TPC start at an initial 25&#xB0;C assay temperature (vertical black line). (D) Thermal responses for AF16 and HK104 with increasing temperature steps (dashed lines, same data as in A) or with an instantaneous increase from 23&#xB0;C to 35.6&#xB0;C followed by a constant hold temperature (solid lines); top panel shows TPCs and bottom panel shows experimental thermal profiles). Error bars indicate &#xB1; 1 SEM (E) Comparison of locomotion index (LI7) TPC and (F) values at 35.6&#xB0;C (&#xB1; 1 SEM) for F1 heterozygotes to their homozygous parental genotypes (Tropical AF16 and Temperate HK104; purple = F1HA with HK104 as maternal parent; pink = F1AH with AF16 as maternal parent). (G) Comparison of proportion moving TPC for F1 and parental genotypes, with (H) point estimates at 35.6&#xB0;C (&#xB1; 1 SEM). (I-K) Parameter estimates from logistic function fits to LI7 TPCs for F1 heterozygotes and parental genotypes show more extreme values for baseline locomotion (&#x3B1;) in heterozygotes and intermediate F1 values for thermal sensitivity (&#x3B2;). All animals reared at 23&#xB0;C; AF16 shown in red, HK104 show in blue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Thermal performance for Tropical strain AF16 and Temperate strain HK104. (A) Decrementing and incrementing temperature TPCs show decline of locomotory performance with more extreme temperatures with three distinct metrics: (A) locomotion index with 1/15&lt;sup&gt;th&lt;/sup&gt; second resolution (LI1), (B) locomotion index with 7/15&lt;sup&gt;th&lt;/sup&gt; second resolution (LI7), and (C) the proportion of moving individuals. Upper and lower halves of the TPC start at an initial 25&#xB0;C assay temperature (vertical black line). (D) Thermal responses for AF16 and HK104 with increasing temperature steps (dashed lines, same data as in A) or with an instantaneous increase from 23&#xB0;C to 35.6&#xB0;C followed by a constant hold temperature (solid lines); top panel shows TPCs and bottom panel shows experimental thermal profiles). Error bars indicate &#xB1; 1 SEM (E) Comparison of locomotion index (LI7) TPC and (F) values at 35.6&#xB0;C (&#xB1; 1 SEM) for F1 heterozygotes to their homozygous parental genotypes (Tropical AF16 and Temperate HK104; purple = F1&lt;sub&gt;HA&lt;/sub&gt; with HK104 as maternal parent; pink = F1&lt;sub&gt;AH&lt;/sub&gt; with AF16 as maternal parent). (G) Comparison of proportion moving TPC for F1 and parental genotypes, with (H) point estimates at 35.6&#xB0;C (&#xB1; 1 SEM). (I-K) Parameter estimates from logistic function fits to LI7 TPCs for F1 heterozygotes and parental genotypes show more extreme values for baseline locomotion (&#x3B1;) in heterozygotes and intermediate F1 values for thermal sensitivity (&#x3B2;). All animals reared at 23&#xB0;C; AF16 shown in red, HK104 show in blue.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F5.medium.gif" width="388" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2135/F5.medium.gif" width="388" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493539" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Thermal performance for Tropical strain AF16 and Temperate strain HK104. (A) Decrementing and incrementing temperature TPCs show decline of locomotory performance with more extreme temperatures with three distinct metrics: (A) locomotion index with 1/15<sup>th</sup> second resolution (LI1), (B) locomotion index with 7/15<sup>th</sup> second resolution (LI7), and (C) the proportion of moving individuals. Upper and lower halves of the TPC start at an initial 25Â°C assay temperature (vertical black line). (D) Thermal responses for AF16 and HK104 with increasing temperature steps (dashed lines, same data as in A) or with an instantaneous increase from 23Â°C to 35.6Â°C followed by a constant hold temperature (solid lines); top panel shows TPCs and bottom panel shows experimental thermal profiles). Error bars indicate Â± 1 SEM (E) Comparison of locomotion index (LI7) TPC and (F) values at 35.6Â°C (Â± 1 SEM) for F1 heterozygotes to their homozygous parental genotypes (Tropical AF16 and Temperate HK104; purple = F1<sub>HA</sub> with HK104 as maternal parent; pink = F1<sub>AH</sub> with AF16 as maternal parent). (G) Comparison of proportion moving TPC for F1 and parental genotypes, with (H) point estimates at 35.6Â°C (Â± 1 SEM). (I-K) Parameter estimates from logistic function fits to LI7 TPCs for F1 heterozygotes and parental genotypes show more extreme values for baseline locomotion (Î±) in heterozygotes and intermediate F1 values for thermal sensitivity (Î²). All animals reared at 23Â°C; AF16 shown in red, HK104 show in blue.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The reciprocal F1 progeny of <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> and <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> exhibited largely intermediate TPC phenotypes at those high temperatures that most distinguish the parental strains (35.6-36.7Â°), with the F1s appearing slightly more similar to their maternal parent to indicate some parent-of-origin asymmetry (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5E,F</a>). These F1 data cannot distinguish mitochondrial <em>vs.</em> epigenetic parental effects, although we can exclude sex-linkage as a factor because all individuals were hermaphrodites (XX karyotype); QTL mapping with RILs found no evidence for a mitochondrial contribution (see below). The largely intermediate F1 heterozygote phenotypes imply that trait variation cannot be explained by a single dominant gene of large effect, although the inflection point parameter values are more ambiguous on this point (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5F-K</a>). These results also tell us that heterozygous genotypes do not exhibit outbreeding depression for locomotion in response to increasing temperature, in contrast to some other phenotypes in <em>C. briggsae</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dolgin <em>et al.</em> 2007</a>), with the asymptote parameter actually showing overdominant trait values that exceed both parents (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5I</a>). We next applied quantitative trait locus mapping using recombinant inbred lines to further dissect the genetic architecture of these TPC differences.</p></div><div id="sec-11" class="subsection"><h3>Locomotory thermal performance variation among recombinant genotypes for QTL mapping</h3><p id="p-39">We quantified locomotory thermal performance for 153 RILs, plus their two parental lines, with the liquid microdroplet assay for the âhotâ portion of the thermal performance curve. We then tested for associations between genotype and phenotype, yielding multiple QTL across the genome that each explain up to <em>H</em><sup>2</sup> = 27.6% of the variation in behavioral thermal performance traits (Supplementary Table S5). We summarized trait variation in temperature-dependent locomotory responses using a diversity of phenotypic metrics and reduced their dimensionality through function-valued trait analysis of the reaction norms. Most RILs produced phenotypes that spanned the trait space between parental values, but those that fell beyond parental performance indicate transgressive segregation and imply multiple loci controlling trait variation (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A,B</a>). For example, 52 RIL strains have steeper point estimates for slope (|<em>Î²</em><sub>H</sub>|) than the <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> parent whereas only 7 RILs had a shallower estimated slope than <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6D</a>). Similarly, only 6 RILs showed higher locomotory activity than <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> at 35.6Â° (LI7) and only 9 RILs have inflection point values (<em>Ï</em><sub>H</sub>) higher than <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6E</a>); <em>Î²</em><sub>H</sub> and <em>Ï</em><sub>H</sub> correlated only weakly across RIL strains (<em>r=</em>0.127 <em>P =</em> 0.0132), suggesting that the genetic basis to variation in these traits may be distinct. Many strains (83) have asymptote estimates lower than either parent, indicating that they have slower baseline locomotion (<em>Î±</em><sub>H</sub>; <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>). Overall, the range of variation among the RILs is similar to (or greater than) that seen among wild isolate genotypes of <em>C. briggsae</em>: mixing up the genomes of just two genotypes to create the RILs produced a range of phenotypic variation similar to the natural range of variation seen in the entire species (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6D</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation among recombinant inbred lines for function-valued traits and synthetic &#x201C;PC traits&#x201D; of temperature-dependent locomotory behavior. (A) Thermal performance curves for 153 recombinant inbred lines (RILs) (gray) derived from C. briggsae parental genotypes (Tropical AF16, red; Temperate HK104, blue). Locomotion indices (LI7) for each strain at 35.6&#xB0;C (dashed vertical line) is shown as a histogram of phenotypic values in (B). (C-E) Distribution of parameter estimates from function fits of locomotion index scores to each strain (LI7; see Equation 1 and Figure 1). Distribution of slope estimates (&#x3B2;H) for 23 wild isolate strains (orange) in (D) is superimposed on the distribution for the 153 RILs (gray) for function fits to the LI7 metric from TPCs of worms reared at 23&#xB0;C. Red and blue lines show values for strains AF16 and HK104. (F) Distributions of &#x2018;synthetic&#x2019; locomotion behavior trait values for 153 RIL strains for the first eight principal components (PCs). Red lines indicate AF16, blue lines indicate HK104." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation among recombinant inbred lines for function-valued traits and synthetic &#x201C;PC traits&#x201D; of temperature-dependent locomotory behavior. (A) Thermal performance curves for 153 recombinant inbred lines (RILs) (gray) derived from &lt;em&gt;C. briggsae&lt;/em&gt; parental genotypes (Tropical AF16, red; Temperate HK104, blue). Locomotion indices (LI7) for each strain at 35.6&#xB0;C (dashed vertical line) is shown as a histogram of phenotypic values in (B). (C-E) Distribution of parameter estimates from function fits of locomotion index scores to each strain (LI7; see Equation 1 and Figure 1). Distribution of slope estimates (&#x3B2;&lt;sub&gt;H&lt;/sub&gt;) for 23 wild isolate strains (orange) in (D) is superimposed on the distribution for the 153 RILs (gray) for function fits to the LI7 metric from TPCs of worms reared at 23&#xB0;C. Red and blue lines show values for strains AF16 and HK104. (F) Distributions of &#x2018;synthetic&#x2019; locomotion behavior trait values for 153 RIL strains for the first eight principal components (PCs). Red lines indicate AF16, blue lines indicate HK104.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F6.medium.gif" width="384" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/7/2135/F6.medium.gif" width="384" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493551" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Variation among recombinant inbred lines for function-valued traits and synthetic âPC traitsâ of temperature-dependent locomotory behavior. (A) Thermal performance curves for 153 recombinant inbred lines (RILs) (gray) derived from <em>C. briggsae</em> parental genotypes (Tropical AF16, red; Temperate HK104, blue). Locomotion indices (LI7) for each strain at 35.6Â°C (dashed vertical line) is shown as a histogram of phenotypic values in (B). (C-E) Distribution of parameter estimates from function fits of locomotion index scores to each strain (LI7; see <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> and <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Distribution of slope estimates (Î²<sub>H</sub>) for 23 wild isolate strains (orange) in (D) is superimposed on the distribution for the 153 RILs (gray) for function fits to the LI7 metric from TPCs of worms reared at 23Â°C. Red and blue lines show values for strains AF16 and HK104. (F) Distributions of âsyntheticâ locomotion behavior trait values for 153 RIL strains for the first eight principal components (PCs). Red lines indicate AF16, blue lines indicate HK104.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">A function-valued trait approach provides one powerful way to distill a dynamic multivariate phenotype down to a simpler representation (<a id="xref-ref-59-7" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>). Principal Component Analysis (PCA) represents another common approach in QTL analysis to reduce dimensionality, providing an interesting comparison to function valued traits alone. Therefore, we applied PCA to reduce the dimensionality of the 167 metrics for the 155 strains to yield 8 principal components that are statistically independent and together explain two-thirds of the total variation (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6F</a>; Supplementary Figure S5). Correlated phenotypes get weighted strongly together in a given principal component (PC) axis to represent an integrated âsyntheticâ phenotype, and we analyzed this subset of 8 PCs as composite traits for subsequent analysis. The first principal component (PC1) explains â¼20% of the phenotypic variability and relates primarily to locomotion at low to moderate temperatures up to 32.5Â°, where locomotion is normal or just starts to decline for many strains (Supplementary Figure S6). Subsequent orthogonal components explain smaller portions of phenotypic variation and become more dominated by a few phenotypes with high weights (Supplementary Figure S6). The variation in PC2 captures â¼16% of the variation among the RIL phenotypes, generally being associated with responses to high temperatures above 35.6Â° (Supplementary Figure S6). We also included measures of within-strain variation itself as âpenetranceâ phenotypes: the standard deviations for locomotion metrics and standard error for parameter estimates. These penetrance metrics tend to cluster together in PCs 3, 4, 6 and 7 that are distinct from the PCs loaded primarily with simple locomotion metrics (PCs 1, 2, 5; Supplementary Figure S6).</p><p id="p-42">To perform QTL mapping, we integrated the RIL phenotype data with the RIL genotypes for 1031 SNP markers (<a id="xref-ref-51-8" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>), which comprised 430 distinct genetic blocks in the genomes of the 153 RILs used here. We primarily focus our QTL mapping analysis and interpretation on the three TPC function fit parameters for the âhotâ portion of the thermal performance curve rather than PCs (slope <em>Î²</em><sub>H</sub>, asymptote <em>Î±</em><sub>H</sub>, inflection point <em>Ï</em><sub>H</sub>). These analyses uncovered at least one QTL on chromosome V that explains <em>H</em><sup>2</sup> =12â28% of the variation in locomotion at high temperature (depending on the specific phenotype) and another major QTL on chromosome II associated with locomotion at lower temperature (<em>H</em><sup>2</sup> = 12â19%; Supplementary Table S5). The function fit parameters as phenotypes are especially interesting because they give a summary of characteristics of the entire thermal performance curve and correspond strongly to some of the less intuitive summaries of phenotype captured with PCs. Nevertheless, we also mapped the eight principal component axes as âsynthetic traits,â using 2500 permutations to determine genome-wide significance thresholds based on a Bonferroni correction for multiple phenotypes. This reduction in data dimensionality allowed the maintenance of power to detect QTL while avoiding excessive multiple hypothesis testing and overly redundant phenotypes among the 167 metrics of temperature-dependent locomotory behavior. We then validated the biological interpretations of the âPC-QTLâ with genomic associations involving a key univariate trait (LI7 at 35.6Â°).</p></div><div id="sec-12" class="subsection"><h3>Locomotion at benign temperature maps to QTL on Chromosome II</h3><p id="p-43">The asymptote parameter (<em>Î±</em><sub>H</sub>), which summarizes baseline movement behavior at benign temperatures, maps to the center of Chromosome II (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Both of the synthetic traits corresponding to locomotion at benign initial temperatures in the liquid micro-droplet assay, PC1 and PC6, similarly map to this region of the genome (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A,B</a>). Other univariate phenotypes for locomotion at benign temperatures also had LOD peaks on Chromosome II in exploratory univariate analyses (Supplementary Figure S7; Supplementary Figure S8), many of which weigh heavily in PC1 and/or PC6 (Supplementary Figure S6). These QTL on Chromosome II may have a common genetic basis that affects all of these phenotypes. The <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> allele at the marker nearest to the QTL peak is correlated with higher values of locomotion metrics (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7C</a>; <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8C</a>), consistent with the parental strain differences (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5I</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping for function-valued traits describing behavioral reaction norms. (A) QTL mapping for three function fit parameter phenotypes for locomotion index (normalized LI7). Dashed lines indicate threshold for significance from 2500 permutations of the data (Bonferroni corrected P = 0.05/12). (B) LOD peaks (solid lines) and 95% Bayes credible intervals (vertical dashed lines) for chromosomes with significant QTL for each function parameter phenotype; significance thresholds as in (A). (C) Phenotype values of RIL strains segregated by parental genotype at the marker closest to the QTL peak in (B) indicated above each panel. (D) Multiple imputation QTL mapping analysis of the Locomotion index (LI7) phenotype at 35.6&#xB0;C, for all chromosomes and an expanded plot for the QTL on Chromosome V. LOD peak on Chromosome V at 37.35cM indicated by solid vertical line, with dashed vertical lines indicating 95% Bayes credible interval. Horizontal lines indicates the 5% genome-wide significance LOD threshold after 2500 permutations and Bonferroni correction as in Figure 8. Panel on right shows phenotype values of RIL strains segregated by parental genotype at the markers closest to the QTL peaks on Chromosome V (mean &#xB1; 1 SE for each marker allele shown adjacent to the distribution of points); values for parental strains shown as dashed horizontal lines (red AF16, blue HK104). Mean &#xB1; 1 SE across RILs for each marker allele shown adjacent to the distribution of points; dashed horizontal lines indicate parental strain phenotypes (red AF16, blue HK104). See Supplementary Table S5 for physical map positions of QTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL mapping for function-valued traits describing behavioral reaction norms. (A) QTL mapping for three function fit parameter phenotypes for locomotion index (normalized LI7). Dashed lines indicate threshold for significance from 2500 permutations of the data (Bonferroni corrected &lt;em&gt;P&lt;/em&gt; = 0.05/12). (B) LOD peaks (solid lines) and 95% Bayes credible intervals (vertical dashed lines) for chromosomes with significant QTL for each function parameter phenotype; significance thresholds as in (A). (C) Phenotype values of RIL strains segregated by parental genotype at the marker closest to the QTL peak in (B) indicated above each panel. (D) Multiple imputation QTL mapping analysis of the Locomotion index (LI7) phenotype at 35.6&#xB0;C, for all chromosomes and an expanded plot for the QTL on Chromosome V. LOD peak on Chromosome V at 37.35cM indicated by solid vertical line, with dashed vertical lines indicating 95% Bayes credible interval. Horizontal lines indicates the 5% genome-wide significance LOD threshold after 2500 permutations and Bonferroni correction as in Figure 8. Panel on right shows phenotype values of RIL strains segregated by parental genotype at the markers closest to the QTL peaks on Chromosome V (mean &#xB1; 1 SE for each marker allele shown adjacent to the distribution of points); values for parental strains shown as dashed horizontal lines (red AF16, blue HK104). Mean &#xB1; 1 SE across RILs for each marker allele shown adjacent to the distribution of points; dashed horizontal lines indicate parental strain phenotypes (red AF16, blue HK104). See Supplementary Table S5 for physical map positions of QTL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F7.medium.gif" width="324" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/7/2135/F7.medium.gif" width="324" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493509" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-44" class="first-child">QTL mapping for function-valued traits describing behavioral reaction norms. (A) QTL mapping for three function fit parameter phenotypes for locomotion index (normalized LI7). Dashed lines indicate threshold for significance from 2500 permutations of the data (Bonferroni corrected <em>P</em> = 0.05/12). (B) LOD peaks (solid lines) and 95% Bayes credible intervals (vertical dashed lines) for chromosomes with significant QTL for each function parameter phenotype; significance thresholds as in (A). (C) Phenotype values of RIL strains segregated by parental genotype at the marker closest to the QTL peak in (B) indicated above each panel. (D) Multiple imputation QTL mapping analysis of the Locomotion index (LI7) phenotype at 35.6Â°C, for all chromosomes and an expanded plot for the QTL on Chromosome V. LOD peak on Chromosome V at 37.35cM indicated by solid vertical line, with dashed vertical lines indicating 95% Bayes credible interval. Horizontal lines indicates the 5% genome-wide significance LOD threshold after 2500 permutations and Bonferroni correction as in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>. Panel on right shows phenotype values of RIL strains segregated by parental genotype at the markers closest to the QTL peaks on Chromosome V (mean Â± 1 SE for each marker allele shown adjacent to the distribution of points); values for parental strains shown as dashed horizontal lines (red AF16, blue HK104). Mean Â± 1 SE across RILs for each marker allele shown adjacent to the distribution of points; dashed horizontal lines indicate parental strain phenotypes (red AF16, blue HK104). See Supplementary Table S5 for physical map positions of QTL.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping for synthetic &#x201C;PC traits.&#x201D; (A) QTL maps for the 8 &#x2018;synthetic&#x2019; principal component locomotion behavior traits show logarithm of the odds (LOD) scores plotted against marker position by chromosome (&#x201C;mt&#x201D; = mitochondrial genome). Dashed lines indicate the 5% genome wide significance threshold from 2500 permutations and Bonferroni correction (P = 0.05/12). PCs with non-significant QTL indicated with gray colors in (A) and Figure 6F. (B) Significant QTL peaks shown for individual chromosomes (colored as for PC traits shown in A; significance thresholds as in A). Solid vertical lines indicates LOD peak for the marker loci shown in (C); dashed vertical lines indicate the 95% Bayes credible interval. (C) Phenotype values of RIL strains segregated by parental genotype at the marker closest to the QTL peak in (B) indicated above each panel. Parental strain values indicated by horizontal dashed lines (red AF16, blue HK104; PC colors as in Figure 6F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL mapping for synthetic &#x201C;PC traits.&#x201D; (A) QTL maps for the 8 &#x2018;synthetic&#x2019; principal component locomotion behavior traits show logarithm of the odds (LOD) scores plotted against marker position by chromosome (&#x201C;mt&#x201D; = mitochondrial genome). Dashed lines indicate the 5% genome wide significance threshold from 2500 permutations and Bonferroni correction (&lt;em&gt;P&lt;/em&gt; = 0.05/12). PCs with non-significant QTL indicated with gray colors in (A) and Figure 6F. (B) Significant QTL peaks shown for individual chromosomes (colored as for PC traits shown in A; significance thresholds as in A). Solid vertical lines indicates LOD peak for the marker loci shown in (C); dashed vertical lines indicate the 95% Bayes credible interval. (C) Phenotype values of RIL strains segregated by parental genotype at the marker closest to the QTL peak in (B) indicated above each panel. Parental strain values indicated by horizontal dashed lines (red AF16, blue HK104; PC colors as in Figure 6F).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F8.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/7/2135/F8.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493518" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-45" class="first-child">QTL mapping for synthetic âPC traits.â (A) QTL maps for the 8 âsyntheticâ principal component locomotion behavior traits show logarithm of the odds (LOD) scores plotted against marker position by chromosome (âmtâ = mitochondrial genome). Dashed lines indicate the 5% genome wide significance threshold from 2500 permutations and Bonferroni correction (<em>P</em> = 0.05/12). PCs with non-significant QTL indicated with gray colors in (A) and <a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6F</a>. (B) Significant QTL peaks shown for individual chromosomes (colored as for PC traits shown in A; significance thresholds as in A). Solid vertical lines indicates LOD peak for the marker loci shown in (C); dashed vertical lines indicate the 95% Bayes credible interval. (C) Phenotype values of RIL strains segregated by parental genotype at the marker closest to the QTL peak in (B) indicated above each panel. Parental strain values indicated by horizontal dashed lines (red AF16, blue HK104; PC colors as in <a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6F</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>High temperature locomotory activity maps to QTL on Chromosome V</h3><p id="p-46">The function-valued trait parameters for slope (<em>Î²</em><sub>H</sub>) and inflection point (<em>Ï</em><sub>H</sub>) both mapped to QTL in the center of Chromosome V, indicating that this genomic region contains important genetic variation explaining locomotory differences at high temperatures (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A,B</a>). Supporting this result, two synthetic PC traits mapped as QTL to the center of Chromosome V (PC2 and PC3), and PC3 also mapped to a QTL on the right end of the X Chromosome (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8A,B</a>). The QTL peaks for PC2 and PC3 on Chromosome V are 3.9Mb apart, with partially-overlapping 95% Bayes credible intervals, reminiscent of the âtwin peaksâ pattern of LOD scores for <em>Î²</em><sub>H</sub> (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a> <em>cf</em>. <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8A</a>). Univariate phenotypes related to locomotion at high temperatures (&gt;35Â°) load most heavily on PC2, whereas PC3 is most closely tied to âpenetranceâ phenotypes related to variability among individuals at moderately high temperatures (&lt;33Â°; <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>, Supplementary Figure S6). Consequently, we mapped the univariate 35.6Â° locomotion index (LI7), also yielding a QTL peak on Chromosome V (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7D</a>), in fact the highest LOD value for any phenotype (Supplementary Table S5), and consistent with the biological interpretation of at least one QTL on Chromosome V associated with high temperature locomotory activity. Additional exploratory univariate QTL analyses reinforce this interpretation, given that 32 univariate traits related to high temperature locomotion associate with this region of Chromosome V even after Bonferroni multiple-test correction (Supplementary Figure S6; Supplementary Figure S9). Strains with Tropical <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> genotypes at the peak locations of these QTL had values related to greater performance at high temperature, consistent with the difference between parental strains (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Figure 8C</a>; <a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7C</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Univariate trait loadings for PC2 and PC3 that map to Chromosome V QTL and two-dimensional QTL scans. (A) Biplot of 167 individual phenotype weights measured for the 153 RILs, for the orthogonal PC2 and PC3 &#x2018;synthetic&#x2019; trait axes that map to Chromosome V. Length and direction of vectors indicate weight of the phenotype in each PC with the text indicating individual phenotype metrics (Supplementary Figure S6). For example, the Locomotion index (LI7) at 35.6&#xB0;C is weighted strongly in both these PC&#x2019;s and can be found near the top-right corner (&#x201C;LI7_35&#x201D;). Circles indicate each RIL strain, colored according to their genotype for markers at QTL peaks for PC2 and PC3 on Chromosome V (marker cbv12146 for PC2 and marker cb49450 for PC3; red = both AF16 alleles, blue = both HK104 alleles, purple indicates different alleles between the peak of PC2 and PC3; peaks were 6cM and &#x223C;3.9Mb apart, Supplementary Table S5). (B-F) Two-dimensional QTL scans show pairwise LOD scores for a given phenotype, shown only for chromosomes with significant QTL. The upper-left triangle above the 1:1 line in each plot contains the additive-effect LOD values after subtracting the LOD values from the single-QTL model. The lower-right triangle below the 1:1 line in each plot shows the LOD values for interactions. Note different color scales in each plot and each off-diagonal triangle zone. Magenta circles highlight the significant QTL locations (Supplementary Table S5)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-360737558" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Univariate trait loadings for PC2 and PC3 that map to Chromosome V QTL and two-dimensional QTL scans. (A) Biplot of 167 individual phenotype weights measured for the 153 RILs, for the orthogonal PC2 and PC3 &#x2018;synthetic&#x2019; trait axes that map to Chromosome V. Length and direction of vectors indicate weight of the phenotype in each PC with the text indicating individual phenotype metrics (Supplementary Figure S6). For example, the Locomotion index (LI7) at 35.6&#xB0;C is weighted strongly in both these PC&#x2019;s and can be found near the top-right corner (&#x201C;LI7_35&#x201D;). Circles indicate each RIL strain, colored according to their genotype for markers at QTL peaks for PC2 and PC3 on Chromosome V (marker cbv12146 for PC2 and marker cb49450 for PC3; red = both AF16 alleles, blue = both HK104 alleles, purple indicates different alleles between the peak of PC2 and PC3; peaks were 6cM and &#x223C;3.9Mb apart, Supplementary Table S5). (B-F) Two-dimensional QTL scans show pairwise LOD scores for a given phenotype, shown only for chromosomes with significant QTL. The upper-left triangle above the 1:1 line in each plot contains the additive-effect LOD values after subtracting the LOD values from the single-QTL model. The lower-right triangle below the 1:1 line in each plot shows the LOD values for interactions. Note different color scales in each plot and each off-diagonal triangle zone. Magenta circles highlight the significant QTL locations (Supplementary Table S5).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2135/F9.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/9/7/2135/F9.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2135/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493546" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-47" class="first-child">Univariate trait loadings for PC2 and PC3 that map to Chromosome V QTL and two-dimensional QTL scans. (A) Biplot of 167 individual phenotype weights measured for the 153 RILs, for the orthogonal PC2 and PC3 âsyntheticâ trait axes that map to Chromosome V. Length and direction of vectors indicate weight of the phenotype in each PC with the text indicating individual phenotype metrics (Supplementary Figure S6). For example, the Locomotion index (LI7) at 35.6Â°C is weighted strongly in both these PCâs and can be found near the top-right corner (âLI7_35â). Circles indicate each RIL strain, colored according to their genotype for markers at QTL peaks for PC2 and PC3 on Chromosome V (marker cbv12146 for PC2 and marker cb49450 for PC3; red = both AF16 alleles, blue = both HK104 alleles, purple indicates different alleles between the peak of PC2 and PC3; peaks were 6cM and â¼3.9Mb apart, Supplementary Table S5). (B-F) Two-dimensional QTL scans show pairwise LOD scores for a given phenotype, shown only for chromosomes with significant QTL. The upper-left triangle above the 1:1 line in each plot contains the additive-effect LOD values after subtracting the LOD values from the single-QTL model. The lower-right triangle below the 1:1 line in each plot shows the LOD values for interactions. Note different color scales in each plot and each off-diagonal triangle zone. Magenta circles highlight the significant QTL locations (Supplementary Table S5).</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">We also found a QTL on the X chromosome for PC3 (<a id="xref-fig-8-7" class="xref-fig" href="#F8">Figure 8A,B</a>). Exploratory analysis with univariate metrics suggest that this QTL mostly corresponds to phenotypes related to within-strain standard deviation for locomotion at relatively benign temperatures (Supplementary Figure S6; Supplementary Figure S7). One possibility is that genetic variation on the X chromosome contributes to the robustness of locomotory responses among individuals following exposure to moderately increasing temperatures.</p><p id="p-49">Given the important role implicated by the Chromosome V QTL for behavioral performance at high temperatures, we sought to affirm a causative role for genetic differences between Tropical <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> and Temperate <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> in shaping locomotory thermal performance. Therefore, we created 13 NILs that had regions of <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>âs Chromosome V introgressed into the genomic background of <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> by backcrossing RILs to <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a> for 7-12 generations using marker-assisted selection. We observed significant differences in locomotion among the NILs at 35.6Â°, most of which exhibited phenotypes more similar to the <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> parent (Supplementary Figure S10). Consistent with QTL on Chromosome V, therefore, it can be sufficient for <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> genotypes in the center of Chromosome V of the NILs to shift the phenotype toward <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>-like trait values (<em>e.g.</em>, A1124-gs123 and A1124-gs124). Trait values for some NILs, however, suggest that the genetic architecture on Chromosome V might be more complex than the QTL analysis implies or that our NIL markers miss some introgressed segments of DNA (<em>e.g.</em>, A1124-gs124 <em>vs.</em> A1124-gs66). Despite failing to narrow the QTL region, these introgression lines are consistent with the QTL region on Chromosome V containing genetic differences that cause phenotypic differences in behavioral thermal performance at high temperatures.</p><p id="p-50">The QTL region for Î² and Ï spans nearly 11Mb of Chromosome V and contains 2837 putative genes; the Bayes interval for locomotion (LI7) at 35.6Â° spans almost 7Mb and contains 1784 putative genes. Approximately 70% of these genes have associated gene ontology (GO) terms, implicating potential structural or molecular functions that we cross-referenced with genes for which we detected coding changes (missense, nonsense, or indel alleles). We identified 115 of the 2837 genes (63 of the 1784 genes) to have such coding changes as fixed differences between 20 Tropical and 4 Temperate group strain genomes (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Thomas <em>et al.</em> 2015</a>) (Supplementary Figure S11, Supplementary Table S7). Fixed differences between these phylogeographic strain groups should represent changes that are most ecologically relevant to temperature-dependent phenotypic divergence. Sensory and neurological system genes, including membrane proteins, receptors, and signaling pathway genes, were significantly overrepresented relative to genome-wide expectations (PANTHER overrepresentation test; Supplementary Figure S11) (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Thomas <em>et al.</em> 2003</a>). Despite the enrichment for these genes with potential influence on behavior, many other kinds of genes could conceivably contribute, in addition to the potential influence of non-coding regulatory or RNA genes on thermal performance.</p></div><div id="sec-14" class="subsection"><h3>Testing for multiple QTL on Chromosome V</h3><p id="p-51">The LOD peaks on Chromosome V visually appear bimodal for <em>Î²</em><sub>H</sub> and LI7 at 35.6Â°, and the broad Bayes credible interval suggests that multiple causal loci might occur under the QTL region (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7A-C</a>). The primary peak is located between 37.4 and 38.0 cM (near the QTL peak for PC3) and the secondary peak across the âvalleyâ is between 25.0 and 31.5cM (near the QTL peak for PC2) (Supplementary Figure S9). Presuming that the two PCs capture distinct orthogonal aspects of the phenotypes, then the âtwin peakâ pattern on Chromosome V might hint at distinct genetic contributions to temperature-sensitive traits. Can we find statistical evidence for twin peaks? It was not possible to map them separately with the multiple imputation method used for our primary analysis, which assumes only one peak per chromosome, prompting us to consider alternate approaches. Only 15 of the 153 RILs have genotypes that differ between the 37.4cM and 31.5cM locations, however, suggesting limited power to disentangle separate loci. Nevertheless, we explored the possibility of twin peaks using conditional QTL analysis, which did not reveal evidence of a significant pair of QTL peaks on Chromosome V (Supplementary Figure S12).</p><p id="p-52">As an alternate approach, we performed two-dimensional QTL scans, which found evidence for possible new additive and interacting QTL that the single-QTL model could not detect (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B-D</a>; Supplementary Table S6). These two-dimensional scans test all pairwise combinations of intervals at least 5cM apart for interactions and epistatic effects for 10 key phenotypes, including those with significant QTL from our initial analysis (those in Supplementary Table S5). However, this procedure did not reveal any new QTL for the phenotypes slope (Î²), inflection point (Ï), PC6 or locomotion (LI7) at 35.6Â°. Instead, the two-dimensional QTL analysis revealed only one pair of loci with an epistatic interaction and eight pairs of loci with additive effects across different chromosomes for the remaining phenotypes (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7</a>; Supplementary Table S6). For asymptote (Î±) and PC1, this analysis recapitulates QTL at the center of Chromosome II. At least three QTL were also found on Chromosome V associated with the âsyntheticâ phenotypes PC1, PC2 and PC3. Interestingly, PC2 shows evidence for an additive interaction between QTL near the centers of Chromosomes II and V, which correspond to the two distinct QTL from the single-QTL analysis for the two separate phenotypes.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><div id="sec-16" class="subsection"><h3>Behavioral phenotypic plasticity in response to temperature</h3><p id="p-53">Phenotypic reaction norms inherently illustrate the plasticity of traits in response to extrinsic conditions (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Via <em>et al.</em> 1995</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Dingemanse <em>et al.</em> 2010</a>). Our analysis of locomotory behavior in response to thermal inputs show this phenotypic plasticity explicitly, reflecting both dynamic changes in temperature over time and the plastic responses of those dynamic reaction norms to distinct rearing conditions. We summarized this complex kind of phenotype with a function-valued trait approach (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Wu and Lin 2006</a>; <a id="xref-ref-59-8" class="xref-bibr" href="#ref-59">Stinchcombe and Kirkpatrick 2012</a>), using biologically interpretable parameter values from fits of a logistic function to the norms of reaction. This strategy distilled a highly multidimensional phenotype into three key components captured by: the baseline locomotory activity of animals at benign temperatures (asymptote parameter Î±), the rate of change in motility with changing temperature (slope parameter Î²), and the temperature mid-way through the transition from normal movement to immobility (inflection point parameter Ï).</p><p id="p-54">Our characterization of behavioral dynamics with changes in temperature revealed <em>C. briggsae</em> locomotory behavior to produce a response similar to a classic thermal performance curve (TPC) (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Huey and Stevenson 1979</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Huey and Kingsolver 1989</a>). The classic TPC shape of a steeper decline in trait performance toward hot temperatures from a peak at benign temperatures, relative to the shallower trait change toward cool temperatures, was most pronounced when animals were reared under cool conditions. Such cool rearing conditions also produced TPCs with a broader range of animal activity (larger ÎÏ), indicative of a more âgeneralistâ trait response to changes in the ambient thermal regime. Thus, âdevelopmental acclimationâ (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Colinet and Hoffmann 2012</a>) from early life experience cascades into producing adult animals with more generalist or specialist locomotory strategies. The greater breadth of the behavioral response profile, however, primarily reflected expansion of locomotory activity when experiencing cool ambient temperatures, both in terms of a cold-shifted inflection point (Ï) and a shallower slope (|Î²|). By comparison, the upper âhotâ portion of the TPC was much less sensitive to the rearing conditions of animals, indicating that the temperature regime that worms experience early in life is disproportionately important in shaping their response to cooling ambient temperatures. This finding of greater TPC plasticity in a cool regime is generally consistent with studies of thermal biology in other systems (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chown and Terblanche 2006</a>). Previous work in <em>C. elegans</em> demonstrated that nematode TPC responses to high temperatures are influenced, in part, by neural âdecision makingâ and that the sensitivity of TPCs to rearing conditions also has a genetic component (<a id="xref-ref-57-8" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>). This sensitivity of TPC shape to rearing conditions in both <em>C. elegans</em> and <em>C. briggsae</em> illustrates the profound influence that early life experience through developmental acclimation (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Colinet and Hoffmann 2012</a>) can exert on subsequent traits later in life, as is well-known in more complex animals like insects and humans (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Urquhart-Cronish and Sokolowski 2014</a>).</p><p id="p-55">Despite the variation in TPC breadth, both due to rearing conditions and to natural differences in genotype, we found no evidence supporting the âjack of all temperatures and master of noneâ hypothesis for the existence of specialist-generalist trade-offs due to functional constraints in thermal adaptation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Huey and Hertz 1984</a>). Specifically, we observed that TPC breadth (ÎÏ) and peak behavioral activity (<em>P</em><sub>max</sub>) did not correlate negatively. Curiously, we also failed to find support for the âhotter is betterâ hypothesis, an idea motivated by the logic that adapting to hotter environments would lead to a higher peak performance (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Huey and Kingsolver 1989</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Angilletta <em>et al.</em> 2010</a>): <em>C. briggsae</em> genotypes with hotter temperatures of peak performance (<em>T</em><sub>opt</sub>) did not actually show higher peak performance (<em>P</em><sub>max</sub>). Instead, we find higher temperatures of peak performance (<em>T</em><sub>opt</sub>) for genotypes that have broader temperature breadths (ÎÏ), which suggests that âgeneralist genotypesâ are more likely to have peak performance at high temperatures. The behavioral nature of our phenotypic analysis provides one interesting possible explanation for the absence of evidence to support these classic hypotheses from thermal biology. Specifically, because locomotory activity is controlled in part by neural decision-making (<a id="xref-ref-57-9" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>), the optimal (adaptive) performance at a given temperature might not correspond to peak activity. The behavioral decision to slow movement as the animal senses more extreme temperatures might be the feature itself that is subject to selection, potentially in an adaptive manner.</p><p id="p-56">Genetic mapping studies of plasticity have revealed gene-by-environment interactions for life history traits in diverse organisms, including <em>C. briggsae</em>âs nematode cousin <em>C. elegans</em> (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Vieira <em>et al.</em> 2000</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Ungerer <em>et al.</em> 2003</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hausmann <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gutteling <em>et al.</em> 2007</a>). However, analysis of <em>Drosophila serrata</em> locomotory activity in thermal performance curves from natural variation (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Latimer <em>et al.</em> 2011</a>), QTL mapping with RILs (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Latimer <em>et al.</em> 2015</a>), and via mutation accumulation (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Latimer <em>et al.</em> 2014</a>) provides an especially useful comparison to our work on <em>C. briggsae</em>. Like our analysis for <em>C. briggsae</em>, Latimer and colleaguesâ data did not support the âhotter is betterâ hypothesis, instead suggesting that tropical genotypes of <em>D. serrata</em> were more specialized, with a higher peak performance and narrower TPC than other natural populations. Most variation in TPCs among <em>D. serrata</em> RILs reflected overall locomotion, rather than changes in TPC shape or temperature-specific responses. By contrast, we observed for <em>C. briggsae</em> that all key characteristics of TPC shape that we analyzed differed significantly among genotypes.</p><p id="p-57">In addition to the influence of temperature changes on the average response of animal movement, we also observed consistent patterns in worm-to-worm variability as a dynamic response to temperature for a given genotype. Cooler ambient temperatures generally led to more stereotyped behavior, consistent with the locomotory response being more canalized or robust to perturbations in temperatures below benign conditions. By contrast, worm-to-worm variation was greatest at hot temperatures just preceding the rapid decline in swimming behavior in the TPC, indicating greater individual stochasticity in behavioral responses as ambient heating conditions become more extreme. This pattern suggests that selection might have played a greater role in shaping plastic responses to cooling temperature in combination with the greater sensitivity of cool-regime TPCs to developmental rearing conditions, analogous to insect systems (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chown and Terblanche 2006</a>).</p></div><div id="sec-17" class="subsection"><h3>Heritable contributions to behavioral thermal reaction norms</h3><p id="p-58">By quantifying the thermal performance curves for 23 wild isolate genetic backgrounds of <em>C. briggsae</em>, we demonstrated substantial heritable variation in behavioral reaction norms. Essentially all of the features of locomotory responses to changing temperatures exhibited heritable differences among distinct genotypes from across the range of this speciesâ genetic diversity, from baseline or maximal activity (Î±; <em>P</em><sub>max</sub>) to the sensitivity to temperature changes toward hot or cold (Î²<sub>H</sub>, Î²<sub>C</sub>), the temperature of maximum thermal sensitivity or performance (Ï<sub>H</sub>, Ï<sub>C</sub>; <em>T</em><sub>opt</sub>), and thermal breadth of activity (ÎÏ). We then interrogated the âhotâ portion of <em>C. briggsae</em> behavioral thermal responses in more detail for representatives of the âTropicalâ and âTemperateâ phylogeographic groups within this species (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">FÃ©lix <em>et al.</em> 2013</a>; <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Thomas <em>et al.</em> 2015</a>), followed by a library of 153 recombinant inbred lines (RILs) derived from these Tropical and Temperate parental strains (<a id="xref-ref-51-9" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>). These latitudinally-separated genetic groups within <em>C. briggsae</em> show evidence of adaptation to temperature differences in terms of fecundity (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Prasad <em>et al.</em> 2011</a>), and also exhibit distinctive behavioral responses in terms of thermal preference and isothermal tracking (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Stegeman <em>et al.</em> 2013</a>).</p><p id="p-59">The Tropical (<a href="http://www.wormbase.org/db/get?name=AF16;class=Strain">AF16</a>) and Temperate (<a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a>) genotypes showed distinctive responses to high temperature, in particular, with animals with the Tropical genotype continuing to move slowly above 35Â° whereas Temperate strain worms were quiescent. Consistent with multiple loci contributing to this difference, we found that F1 individuals exhibited intermediate behavioral responses and that the recombinant genotypes of the RIL strains produced a spectrum of responses that exceeded the parental trait values, in fact yielding a similar range of phenotypic variation as we observed across the genetically diverse set of wild isolates. Our quantitative trait locus (QTL) mapping analysis corroborated this idea of multiple contributing loci, with major QTL mapped to three chromosomes.</p><p id="p-60">Distinct components of the behavioral response to temperature correlate with genetic differences in unlinked portions of the genome. The baseline behavioral activity associates with the center of chromosome II (LOD peak at position 9.91 Mb for Î±<sub>H</sub>), whereas the behavioral sensitivities to high temperature map to chromosome V (11.62 Mb for Î²<sub>H</sub>, 8.08 Mb for Ï<sub>H</sub>). We observed similar genetic architecture for alternate ways of summarizing phenotypes for the behavioral reaction norm, whether using univariate metrics or Principle Component axes as synthetic phenotypes, which corroborates the biological importance of loci on chromosomes II and V. Moreover, univariate and synthetic âPC-QTLâ indicate that genetic variation on the X chromosome contributes to inter-individual variation in behavioral responses as temperature increases, affecting the robustness or canalization of the trait when perturbed by environmental changes to thermal regime. These genetically-separable components of the behavioral thermal performance indicate that it is encoded by distinct modules that together contribute to the overall complex behavioral response of the reaction norm. Independent genetic loci contributing to complex phenotypes in a modular way might be a general feature of behavioral phenotypes, as it also has been observed in tunnel burrowing behavior in oldfield mice (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Weber <em>et al.</em> 2013</a>).</p><p id="p-61">Genetic perturbations are more likely to affect locomotion detrimentally, especially at high temperature, than they are to enhance locomotion or tolerance to extreme higher temperatures (<a id="xref-ref-57-10" class="xref-bibr" href="#ref-57">Stegeman <em>et al.</em> 2019</a>). Many genes must work in concert for successful locomotion, so even slight genetic changes to the system could decrease tolerance to extreme temperature stress and consequently result in slowed locomotion. With phylogeographic differentiation, co-adaptation may have tuned gene variants to work best in combination within their âhomeâ genome, especially in a self-fertile species like <em>C. briggsae</em> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Dolgin <em>et al.</em> 2007</a>). By recombining genomes from genetically distinct phylogeographic groups, these co-adapted genetic networks would get disrupted. We see evidence suggestive of this idea here in that a preponderance of RILs slowed their locomotion in response to temperature shifts relative to both parental strains (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6</a>).</p><p id="p-62">Despite the complexity of a dynamic trait like a behavioral response to external inputs, we uncovered little evidence of non-additive interactions and just a handful of QTL across the genome. This finding in <em>C. briggsae</em> contrasts with an epistatic genetic architecture to thermal preference behavior in <em>C. elegans</em> QTL analyses (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gaertner <em>et al.</em> 2012</a>). We also found no contribution of mitochondrial differences, despite previous work in <em>C. briggsae</em> showing an influence of mitochondrial genotype on other features of their biology (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Howe and Denver 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Howe <em>et al.</em> 2010</a>; <a id="xref-ref-51-10" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chang <em>et al.</em> 2015</a>). This lack of a role for mitochondria in our analysis also contrasts with mitochondria contributing to variation in thermotolerance in insects and rodents, deriving from the possible contribution of mitochondrial membrane potentials releasing heat (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fontanillas <em>et al.</em> 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ballard <em>et al.</em> 2007</a>). Evidence of paternal inheritance of mitochondria in <em>C. briggsae</em>, especially those involving <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain">HK104</a> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ross <em>et al.</em> 2016</a>), might also mask our ability to infer mitochondrial QTL. Our QTL mapping analysis also suggests several additional potential QTL at the cusp of statistical significance that might contribute minor phenotypic effects to explain residual heritable variation that is not captured in the major QTL we identified on Chromosomes II, V and X. While our experiments with introgression line genotypes affirmed a causative role of the Chromosome V QTL in affecting behavioral thermal responses, fine-mapping the causative underlying genetic differences in this gene dense region with low recombination will require further inquiry (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stein <em>et al.</em> 2003</a>; <a id="xref-ref-51-11" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2011</a>; <a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Thomas <em>et al.</em> 2015</a>).</p><p id="p-63">Given the distinct phenotypes that map to overlapping locations on Chromosome V and the hundreds of genes with potentially function-affecting differences between the Temperate and Tropical strains, it remains plausible that multiple linked loci within the QTL region contribute to the locomotory reaction norm. Variation in both Î²<sub>H</sub> and Ï<sub>H</sub> mapped to QTL on Chromosome V, but these function parameter phenotypes are mostly independent of one another (<em>R<sup>2</sup></em> = 0.04, <em>P =</em> 0.0132,), and the orthogonal PC2 and PC3 synthetic phenotypes also both map to Chromosome V. Moreover, dissociating multiple factors from one another would weaken the individual signal of each factor so as to make it challenging to decipher the role of the separated genetic loci. For example, the characterization of natural variation in <em>C. elegans</em>â <em><a href="http://www.wormbase.org/db/get?name=WBGene00005934;class=Gene">srx-43</a></em> following QTL mapping of pheromone-dependent foraging follows this pattern of a more modest phenotypic influence of an isolated causal genetic factor (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Greene <em>et al.</em> 2016</a>). QTL mapping in <em>C. elegans</em> has identified multiple causal variants of large effect (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Seidel <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ghosh <em>et al.</em> 2012</a>), often associated with domestication to lab conditions (<em>e.g.</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00015964;class=Gene">glb-5</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00018866;class=Gene">nath-10</a></em>) or with a major functional disruption (<em>e.g.</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00004041;class=Gene">plg-1</a></em>) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Palopoli <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mcgrath <em>et al.</em> 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Duveau and Felix 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2014</a>). While other QTL mapping studies of <em>C. elegans</em> behavior variation did not reach the QTN level, they sometimes revealed epistatic and interacting QTL in behavioral phenotypes (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gaertner <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bahrami and Zhang 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Glater <em>et al.</em> 2014</a>) and, more generally, complex traits may often be influenced by multiple antagonistic-effect loci in tight linkage (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shao <em>et al.</em> 2008</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Gaertner <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bernstein <em>et al.</em> 2018</a>). Plasticity QTL for gene expression in <em>C. elegans</em> animals that had been reared at cold and hot temperatures reported a prominent role for <em>trans</em>-acting regulators, including a âmaster regulatorâ with widespread effects (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Li <em>et al.</em> 2006</a>). Chronic exposure to distinct thermal regimes during development of distinct genotypes of <em>C. briggsae</em> reveals extensive modularity of plastic and genotype-dependent transcriptome responses (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mark <em>et al.</em> 2019</a>), as do evolutionary responses to acute temperature stress in experimental populations of <em>C. remanei</em> (Sikkink <em>et al.</em> 2018). Should the temperature dependent behavioral QTL we uncovered provide a general source of temperature dependence of phenotypes and gene expression differences among genotypes, then it would point to genetically simple but highly pleiotropic underpinnings to adaptive differentiation between Temperate and Tropical populations of <em>C. briggsae</em>. The challenge remains to functionally characterize specific genetic variants controlling <em>C. briggsae</em>âs plastic thermal responses in behavior to test these additional aspects of its genetic architecture.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">We thank Joseph Ross for sharing genotype information for the RIL strains, Karl Broman for R/qtl troubleshooting and suggestions, Lanna Jin and Adrian Verster for help visualizing data in R. ADC was supported by funds from the Natural Sciences and Engineering Council of Canada and a Canada Research Chair. WSR was supported by funds from the Natural Sciences and Engineering Council of Canada.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-65">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7590779">https://doi.org/10.25387/g3.7590779</a>.</p></li><li class="fn" id="fn-2"><p id="p-66"><em>Communicating editor: M. Rockman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-14"><span class="received-label">Received </span>February 14, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-04-30"><span class="accepted-label">Accepted </span>April 30, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Stegeman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.1" data-doi="10.1371/journal.pgen.1004156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">A variant in the neuropeptide receptor <em>npr-1</em> is a major determinant of <em>Caenorhabditis elegans</em> growth and physiology.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004156</span> (erratum: PLoS Genet. 10: e1004316).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004156%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004156%26rft_id%253Dinfo%253Apmid%252F24586193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586193&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.2" data-doi="10.1086/648567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angilletta</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frazier</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Thermodynamic effects on organismal performance: is hotter better?</span> <abbr class="cit-jnl-abbrev">Physiol. Biochem. Zool.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">197</span>â<span class="cit-lpage">206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/648567</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiological%2Band%2BBiochemical%2BZoology%26rft.stitle%253DPhysiological%2Band%2BBiochemical%2BZoology%26rft.aulast%253DAngilletta%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D83%26rft.issue%253D2%26rft.spage%253D197%26rft.epage%253D206%26rft.atitle%253DThermodynamic%2Beffects%2Bon%2Borganismal%2Bperformance%253A%2Bis%2Bhotter%2Bbetter%253F%26rft_id%253Dinfo%253Adoi%252F10.1086%252F648567%26rft_id%253Dinfo%253Apmid%252F20001251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/648567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20001251&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274443400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.3" data-doi="10.1038/nrg1472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Quantitative genetic analyses of complex behaviors in Drosophila.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">838</span>â<span class="cit-lpage">849</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1472</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAnholt%26rft.auinit1%253DR.%2BR.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253D838%26rft.epage%253D849%26rft.atitle%253DQuantitative%2Bgenetic%2Banalyses%2Bof%2Bcomplex%2Bbehaviours%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1472%26rft_id%253Dinfo%253Apmid%252F15520793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15520793&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.4" data-doi="10.1111/j.0269-8463.2004.00904.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayrinhac</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debat</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kister</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legout</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Cold adaptation in geographical populations of <em>Drosophila melanogaster</em>: phenotypic plasticity is more important than genetic variability.</span> <abbr class="cit-jnl-abbrev">Funct. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">700</span>â<span class="cit-lpage">706</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0269-8463.2004.00904.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol.%26rft.volume%253D18%26rft.spage%253D700%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0269-8463.2004.00904.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0269-8463.2004.00904.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bahrami</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>âWhen females produce sperm: genetics of <em>C. elegans</em> hermaphrodite reproductive choice. G3: Genes|Genomes|Genetics 3: 1851â1859.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.6" data-doi="10.1111/nph.13509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markelz</span>  <span class="cit-name-given-names">R. J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covington</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Modeling development and quantitative trait mapping reveal independent genetic modules for leaf size and shape.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">257</span>â<span class="cit-lpage">268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.7" data-doi="10.1111/j.1558-5646.2007.00133.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J. W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melvin</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katewa</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maas</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Mitochondrial DNA variation is associated with measurable differences in life-history traits and mitochondrial metabolism in <em>Drosophila simulans</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">1735</span>â<span class="cit-lpage">1747</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2007.00133.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBallard%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D61%26rft.issue%253D7%26rft.spage%253D1735%26rft.epage%253D1747%26rft.atitle%253DMitochondrial%2BDNA%2Bvariation%2Bis%2Bassociated%2Bwith%2Bmeasurable%2Bdifferences%2Bin%2Blife-history%2Btraits%2Band%2Bmitochondrial%2Bmetabolism%2Bin%2BDrosophila%2Bsimulans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00133.x%26rft_id%253Dinfo%253Apmid%252F17598752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00133.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17598752&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247576400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.8" data-doi="10.1016/j.celrep.2015.01.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begasse</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leaver</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grill</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Temperature dependence of cell division timing accounts for a shift in the thermal limits of <em>C. elegans</em> and <em>C. briggsae</em>.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">647</span>â<span class="cit-lpage">653</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2015.01.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D10%26rft.spage%253D647%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.01.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.9" data-doi="10.1101/428466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdraljevic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Tightly-linked antagonistic-effect loci underlie polygenic demographic variation in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/428466</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F428466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0Mjg0NjZ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-book" id="cit-9.7.2135.10" data-doi="10.1007/978-0-387-92125-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-source">A Guide to QTL Mapping with R/qtl</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-92125-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2BGuide%2Bto%2BQTL%2BMapping%2Bwith%2BR%252Fqtl%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-92125-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-92125-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.11" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ã.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>â<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>âMitochondrial-nuclear epistasis impacts fitness and mitochondrial physiology of inter-population <em>Caenorhabditis briggsae</em> hybrids. G3: Genes|Genomes|Genetics 6: 209â219.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-book" id="cit-9.7.2135.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chown</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terblanche</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Physiological diversity in insects: ecological and evolutionary contexts</span>, pp. <span class="cit-fpage">50</span>â<span class="cit-lpage">152</span> in <span class="cit-source">Advances in Insect Physiology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.14" data-doi="10.1111/j.1365-2435.2011.01898.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colinet</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Comparing phenotypic effects and molecular correlates of developmental, gradual and rapid cold acclimation responses in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Funct. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">84</span>â<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2435.2011.01898.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol.%26rft.volume%253D26%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2435.2011.01898.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2435.2011.01898.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.15" data-doi="10.1002/bies.201500053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title"><em>Caenorhabditis</em> evolution in the wild.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">983</span>â<span class="cit-lpage">995</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.201500053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D37%26rft.spage%253D983%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201500053%26rft_id%253Dinfo%253Apmid%252F26126900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201500053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26126900&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.16" data-doi="10.1534/genetics.106.058651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">FÃ©lix</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barriere</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Patterns of nucleotide polymorphism distinguish temperate and tropical wild isolates of <em>Caenorhabditis briggsae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2021</span>â<span class="cit-lpage">2031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.058651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058651%26rft_id%253Dinfo%253Apmid%252F16783011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIwMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.17" data-doi="10.1016/j.tree.2009.07.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dingemanse</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazem</span>  <span class="cit-name-given-names">A. J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">RÃ©ale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Behavioral reaction norms: animal personality meets individual plasticity.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">81</span>â<span class="cit-lpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2009.07.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DDingemanse%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D25%26rft.issue%253D2%26rft.spage%253D81%26rft.epage%253D89%26rft.atitle%253DBehavioural%2Breaction%2Bnorms%253A%2Banimal%2Bpersonality%2Bmeets%2Bindividual%2Bplasticity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2009.07.013%26rft_id%253Dinfo%253Apmid%252F19748700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2009.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19748700&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274767100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.18" data-doi="10.1111/j.1558-5646.2007.00118.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolgin</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Inbreeding and outbreeding depression in <em>Caenorhabditis</em> nematodes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">1339</span>â<span class="cit-lpage">1352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2007.00118.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DDolgin%26rft.auinit1%253DE.%2BS.%26rft.volume%253D61%26rft.issue%253D6%26rft.spage%253D1339%26rft.epage%253D1352%26rft.atitle%253DInbreeding%2Band%2Boutbreeding%2Bdepression%2Bin%2BCaenorhabditis%2Bnematodes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00118.x%26rft_id%253Dinfo%253Apmid%252F17542844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00118.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17542844&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247125100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.19" data-doi="10.1371/journal.pbio.1001230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duveau</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felix</span>  <span class="cit-name-given-names">M.-A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Role of pleiotropy in the evolution of a cryptic developmental variation in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1001230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001230</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDuveau%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253De1001230%26rft.epage%253De1001230%26rft.atitle%253DRole%2Bof%2Bpleiotropy%2Bin%2Bthe%2Bevolution%2Bof%2Ba%2Bcryptic%2Bdevelopmental%2Bvariation%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001230%26rft_id%253Dinfo%253Apmid%252F22235190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22235190&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.20" data-doi="10.1186/1471-2148-13-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">FÃ©lix</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jovelin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrari</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">Y. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Species richness, distribution and genetic diversity of <em>Caenorhabditis</em> nematodes in a remote tropical rainforest.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-13-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-10%26rft_id%253Dinfo%253Apmid%252F23311925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23311925&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>âReview: Protein function at thermal extremes: balancing stability and flexibility. Comparative Biochemistry and Physiology a-Molecular and Integrative Physiology 129: 417â431.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.22" data-doi="10.3354/cr00892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karl</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Exploring plastic and genetic responses to temperature variation using copper butterflies.</span> <abbr class="cit-jnl-abbrev">Clim. Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">17</span>â<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3354/cr00892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClim.%2BRes.%26rft.volume%253D43%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.3354%252Fcr00892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3354/cr00892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.23" data-doi="10.1111/j.1365-294X.2004.02414.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fontanillas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Depraz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giorgi</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrin</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Nonshivering thermogenesis capacity associated to mitochondrial DNA haplotypes and gender in the greater white-toothed shrew, <em>Crocidura russula</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">661</span>â<span class="cit-lpage">670</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2004.02414.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DFontanillas%26rft.auinit1%253DP.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D661%26rft.epage%253D670%26rft.atitle%253DNonshivering%2Bthermogenesis%2Bcapacity%2Bassociated%2Bto%2Bmitochondrial%2BDNA%2Bhaplotypes%2Band%2Bgender%2Bin%2Bthe%2Bgreater%2Bwhite-toothed%2Bshrew%252C%2BCrocidura%2Brussula.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02414.x%26rft_id%253Dinfo%253Apmid%252F15660955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02414.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15660955&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226928500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.24" data-doi="10.1534/genetics.112.142877"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaertner</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmenter</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1533</span>â<span class="cit-lpage">1542</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.142877</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142877%26rft_id%253Dinfo%253Apmid%252F23086219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.25" data-doi="10.1152/jn.01335.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrity</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Role of adaptation in <em>C. elegans</em> thermotaxis. Focus on âShort-term adaptation and temporal processing in the cryophilic response of <em>Caenorabditis elegans</em>â.</span> <abbr class="cit-jnl-abbrev">J. Neurophysiol.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">1874</span>â<span class="cit-lpage">1876</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/jn.01335.2006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNeurophysiol.%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.01335.2006%26rft_id%253Dinfo%253Apmid%252F17202237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/jn.01335.2006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17202237&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.26" data-doi="10.1126/science.1214318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Natural variation in a chloride channel subunit confers avermectin resistance in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">574</span>â<span class="cit-lpage">578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1214318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGhosh%26rft.auinit1%253DR.%26rft.volume%253D335%26rft.issue%253D6068%26rft.spage%253D574%26rft.epage%253D578%26rft.atitle%253DNatural%2BVariation%2Bin%2Ba%2BChloride%2BChannel%2BSubunit%2BConfers%2BAvermectin%2BResistance%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1214318%26rft_id%253Dinfo%253Apmid%252F22301316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2OC81NzQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glater</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>âMultigenic natural variation underlies <em>Caenorhabditis elegans</em> olfactory preference for the bacterial pathogen <em>Serratia marcescens</em>. G3-Genes Genomes Genetics 4: 265â276.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.28" data-doi="10.1038/nature19848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobosiewicz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishida</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macosko</span>  <span class="cit-name-given-names">E. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Balancing selection shapes density-dependent foraging behavior.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">539</span>: <span class="cit-fpage">254</span>â<span class="cit-lpage">258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature19848</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D539%26rft.spage%253D254%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19848%26rft_id%253Dinfo%253Apmid%252FPMID%253A%2B277996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19848&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=PMID: 277996&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.29" data-doi="10.1038/sj.hdy.6800894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutteling</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riksen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Mapping phenotypic plasticity and genotype-environment interactions affecting life-history traits in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">28</span>â<span class="cit-lpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800894</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DGutteling%26rft.auinit1%253DE.%2BW.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D37%26rft.atitle%253DMapping%2Bphenotypic%2Bplasticity%2Band%2Bgenotype-environment%2Binteractions%2Baffecting%2Blife-history%2Btraits%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800894%26rft_id%253Dinfo%253Apmid%252F16955112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800894&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16955112&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242851000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hausmann</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juenger</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stowe</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Quantitative trait loci affecting Î´<sup>13</sup>C and response to differential water availability in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">81</span>â<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHausmann%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D96%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Baffecting%2Bdelta13C%2Band%2Bresponse%2Bto%2Bdifferential%2Bwater%2Bavailibility%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Apmid%252F15792229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15792229&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226737900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.31" data-doi="10.1111/j.1365-2435.2005.00959.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirriffs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Relative importance of plastic <em>vs.</em> genetic factors in adaptive differentiation: geographical variation for stress resistance in <em>Drosophila melanogaster</em> from eastern Australia.</span> <abbr class="cit-jnl-abbrev">Funct. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">222</span>â<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2435.2005.00959.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol.%26rft.volume%253D19%26rft.spage%253D222%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2435.2005.00959.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2435.2005.00959.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.32" data-doi="10.1093/gbe/evp055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baer</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">High rate of large deletions in <em>Caenorhabditis briggsae</em> mitochondrial genome mutation processes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">29</span>â<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evp055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevp055%26rft_id%253Dinfo%253Apmid%252F20333220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evp055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20333220&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.33" data-doi="10.1186/1471-2148-8-62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Mullerâs ratchet and compensatory mutation in <em>Caenorhabditis briggsae</em> mitochondrial genome evolution.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-8-62</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DHowe%26rft.auinit1%253DD.%2BK.%26rft.volume%253D8%26rft.spage%253D62%26rft.epage%253D62%26rft.atitle%253DMuller%2527s%2BRatchet%2Band%2Bcompensatory%2Bmutation%2Bin%2BCaenorhabditis%2Bbriggsae%2Bmitochondrial%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-62%26rft_id%253Dinfo%253Apmid%252F18302772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18302772&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hertz</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Is a jack-of-all-temperatures a master of none? Evolution</span> <span class="cit-vol">38</span>: <span class="cit-fpage">441</span>â<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.35" data-doi="10.1016/0169-5347(89)90211-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsolver</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Evolution of thermal sensitivity of ectotherm performance.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">131</span>â<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0169-5347(89)90211-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DHuey%26rft.auinit1%253DR.%2BB.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D131%26rft.epage%253D135%26rft.atitle%253DEvolution%2Bof%2Bthermal%2Bsensitivity%2Bof%2Bectotherm%2Bperformance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252889%252990211-5%26rft_id%253Dinfo%253Apmid%252F21227334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-5347(89)90211-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21227334&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U410400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.36" data-doi="10.1086/285521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsolver</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Evolution of resistance to high-temperature in ectotherms.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">S21</span>â<span class="cit-lpage">S46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/285521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D142%26rft.spage%253DS21%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LX94500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.37" data-doi="10.1093/icb/19.1.357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevenson</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>â<span class="cit-article-title">Integrating thermal physiology and ecology of ectotherms: a discussion of approaches.</span> <abbr class="cit-jnl-abbrev">Am. Zool.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">357</span>â<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/icb/19.1.357</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BZool.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252F19.1.357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/icb/19.1.357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.38" data-doi="10.1186/1471-2164-11-236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staisch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thillainathan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haines</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A toolkit for rapid gene mapping in the nematode <em>Caenorhabditis briggsae</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKoboldt%26rft.auinit1%253DD.%2BC.%26rft.volume%253D11%26rft.spage%253D236%26rft.epage%253D236%26rft.atitle%253DA%2Btoolkit%2Bfor%2Brapid%2Bgene%2Bmapping%2Bin%2Bthe%2Bnematode%2BCaenorhabditis%2Bbriggsae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-236%26rft_id%253Dinfo%253Apmid%252F20385026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20385026&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.39" data-doi="10.1111/jeb.12552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Latimer</span>  <span class="cit-name-given-names">C. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenoweth</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Connecting thermal performance curve variation to the genotype: a multivariate QTL approach.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">155</span>â<span class="cit-lpage">168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12552</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D28%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.40" data-doi="10.1111/evo.12392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Latimer</span>  <span class="cit-name-given-names">C. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGuigan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blows</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenoweth</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">The contribution of spontaneous mutations to thermal sensitivity curve variation in <em>Drosophila serrata</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1824</span>â<span class="cit-lpage">1837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D1824%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.41" data-doi="10.1111/j.1420-9101.2011.02227.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Latimer</span>  <span class="cit-name-given-names">C. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenoweth</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Quantitative genetic variation for thermal performance curves within and among natural populations of <em>Drosophila serrata</em>.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">965</span>â<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1420-9101.2011.02227.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DLatimer%26rft.auinit1%253DC.%2BA.%26rft.volume%253D24%26rft.issue%253D5%26rft.spage%253D965%26rft.epage%253D975%26rft.atitle%253DQuantitative%2Bgenetic%2Bvariation%2Bfor%2Bthermal%2Bperformance%2Bcurves%2Bwithin%2Band%2Bamong%2Bnatural%2Bpopulations%2Bof%2BDrosophila%2Bserrata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2011.02227.x%26rft_id%253Dinfo%253Apmid%252F21306462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2011.02227.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21306462&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289692700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.42" data-doi="10.1371/journal.pgen.0020222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutteling</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tijsterman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Mapping determinants of gene expression plasticity by genetical genomics in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020222</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De222%26rft.epage%253De222%26rft.atitle%253DMapping%2Bdeterminants%2Bof%2Bgene%2Bexpression%2Bplasticity%2Bby%2Bgenetical%2Bgenomics%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020222%26rft_id%253Dinfo%253Apmid%252F17196041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17196041&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Functional mapping of quantitative trait loci underlying the character process: A theoretical framework.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1751</span>â<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMa%26rft.auinit1%253DC.-X.%26rft.volume%253D161%26rft.issue%253D4%26rft.spage%253D1751%26rft.epage%253D1762%26rft.atitle%253DFunctional%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BUnderlying%2Bthe%2BCharacter%2BProcess%253A%2BA%2BTheoretical%2BFramework%26rft_id%253Dinfo%253Apmid%252F12196415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MS80LzE3NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-other" id="cit-9.7.2135.44" data-doi="10.1101/517235"><div class="cit-metadata"><cite>Mark, S., J. Weiss, E. Sharma, T. Liu, W. Wang <em>et al.</em>, 2019âGenome structure predicts modular transcriptome responses to genetic and environmental conditions. BioRxiv.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/517235</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1101%252F517235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1MTcyMzV2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.45" data-doi="10.1016/j.neuron.2009.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macosko</span>  <span class="cit-name-given-names">E. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Quantitative mapping of a digenic behavioral trait implicates globin variation in <em>C. elegans</em> sensory behaviors.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">692</span>â<span class="cit-lpage">699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2009.02.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNeuron%26rft.aulast%253DMcGrath%26rft.auinit1%253DP.%2BT.%26rft.volume%253D61%26rft.issue%253D5%26rft.spage%253D692%26rft.epage%253D699%26rft.atitle%253DQuantitative%2Bmapping%2Bof%2Ba%2Bdigenic%2Bbehavioral%2Btrait%2Bimplicates%2Bglobin%2Bvariation%2Bin%2BC.%2Belegans%2Bsensory%2Bbehaviors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.02.012%26rft_id%253Dinfo%253Apmid%252F19285466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285466&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264366900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.46" data-doi="10.1038/hdy.2015.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murren</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auld</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callahan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghalambor</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handelsman</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Constraints on the evolution of phenotypic plasticity: limits and costs of phenotype and plasticity.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">293</span>â<span class="cit-lpage">301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.8%26rft_id%253Dinfo%253Apmid%252F25690179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690179&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.47" data-doi="10.1098/rspb.2015.1005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nettle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Adaptive developmental plasticity: what is it, how can we recognize it and when can it evolve?</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">282</span>: <span class="cit-elocation-id">20151005</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2015.1005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2015.1005%26rft_id%253Dinfo%253Apmid%252F26203000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2015.1005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26203000&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.48" data-doi="10.1038/nature07171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palopoli</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinmaung</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curwen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Molecular basis of the copulatory plug polymorphism in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">1019</span>â<span class="cit-lpage">1022</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPalopoli%26rft.auinit1%253DM.%2BF.%26rft.volume%253D454%26rft.issue%253D7207%26rft.spage%253D1019%26rft.epage%253D1022%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bthe%2Bcopulatory%2Bplug%2Bpolymorphism%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07171%26rft_id%253Dinfo%253Apmid%252F18633349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18633349&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258591000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.49" data-doi="10.1111/j.1558-5646.2010.01110.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croydon-Sugarman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Temperature-dependent fecundity associates with latitude in <em>Caenorhabditis briggsae</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">52</span>â<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2010.01110.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DPrasad%26rft.auinit1%253DA.%26rft.volume%253D65%26rft.issue%253D1%26rft.spage%253D52%26rft.epage%253D63%26rft.atitle%253DTemperature-dependent%2Bfecundity%2Bassociates%2Bwith%2Blatitude%2Bin%2BCaenorhabditis%2Bbriggsae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.01110.x%26rft_id%253Dinfo%253Apmid%252F20731713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.01110.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20731713&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.50" data-doi="10.1093/molbev/msw192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman-Hulbert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estes</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Paternal mitochondrial transmission in intra-species <em>Caenorhabditis briggsae</em> hybrids.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">3158</span>â<span class="cit-lpage">3160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw192%26rft_id%253Dinfo%253Apmid%252F27613821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27613821&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.51" data-doi="10.1371/journal.pgen.1002174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staisch</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlin</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title"><em>Caenorhabditis briggsae</em> recombinant inbred line genotypes reveal inter-strain incompatibility and the evolution of recombination.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002174</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRoss%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002174%26rft.epage%253De1002174%26rft.atitle%253DCaenorhabditis%2Bbriggsae%2Brecombinant%2Binbred%2Bline%2Bgenotypes%2Breveal%2Binter-strain%2Bincompatibility%2Band%2Bthe%2Bevolution%2Bof%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002174%26rft_id%253Dinfo%253Apmid%252F21779179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21779179&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-book" id="cit-9.7.2135.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlichting</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pigliucci</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-source">Phenotypic evolution: a reaction norm perspective</span>, <span class="cit-publ-name">Sinauer Press</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.53" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>â<span class="cit-lpage">594</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1151107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.54" data-doi="10.1073/pnas.0810388105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burrage</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinasac</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernest</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genetic architecture of complex traits: Large phenotypic effects and pervasive epistasis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">19910</span>â<span class="cit-lpage">19914</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0810388105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0810388105%26rft_id%253Dinfo%253Apmid%252F19066216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUwLzE5OTEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.7.2135.55" data-doi="10.1534/g3.118.200017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikkink</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ituarte</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span>â<span class="cit-article-title">Environmental and evolutionary drivers of the modular gene regulatory network underlying phenotypic plasticity for stress resistance in the nematode Caenorhabditis remanei.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">969</span>â<span class="cit-lpage">982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200017%26rft_id%253Dinfo%253Apmid%252F30679247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjkvMy85NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.56" data-doi="10.1242/jeb.075408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stegeman</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bueno de Mesquita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryu</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Temperature-dependent behaviors are genetically variable in the nematode <em>Caenorhabditis briggsae</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">850</span>â<span class="cit-lpage">858</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.075408</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.075408%26rft_id%253Dinfo%253Apmid%252F23155083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjk6IjIxNi81Lzg1MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">âµ</a><div class="cit ref-cit ref-other" id="cit-9.7.2135.57" data-doi="10.1101/583120"><div class="cit-metadata"><cite>Stegeman, G. W., D. Medina, A. D. Cutter and W. S. Ryu, 2019âNeuro-genetic control over plasticity of <em>C. elegans</em> behavioral thermal performance. BioRxiv.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/583120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1101%252F583120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1ODMxMjB2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.58" data-doi="10.1371/journal.pbio.0000045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasiar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blumenthal</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">The genome sequence of <em>Caenorhabditis briggsae</em>: A platform for comparative genomics.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">E45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0000045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DStein%26rft.auinit1%253DL.%2BD.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253DE45%26rft.epage%253DE45%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2BCaenorhabditis%2Bbriggsae%253A%2Ba%2Bplatform%2Bfor%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0000045%26rft_id%253Dinfo%253Apmid%252F14624247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0000045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14624247&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.59" data-doi="10.1016/j.tree.2012.07.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stinchcombe</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genetics and evolution of function-valued traits: understanding environmentally responsive phenotypes.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">637</span>â<span class="cit-lpage">647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2012.07.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DStinchcombe%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D27%26rft.issue%253D11%26rft.spage%253D637%26rft.epage%253D647%26rft.atitle%253DGenetics%2Band%2Bevolution%2Bof%2Bfunction-valued%2Btraits%253A%2Bunderstanding%2Benvironmentally%2Bresponsive%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2012.07.002%26rft_id%253Dinfo%253Apmid%252F22898151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2012.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22898151&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313607200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.60" data-doi="10.1371/journal.pone.0021800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Supek</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">BoÅ¡njak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Å kunca</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Å muc</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">REVIGO summarizes and visualizes long lists of gene ontology terms.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e21800</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0021800</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSupek%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De21800%26rft.epage%253De21800%26rft.atitle%253DREVIGO%2Bsummarizes%2Band%2Bvisualizes%2Blong%2Blists%2Bof%2Bgene%2Bontology%2Bterms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0021800%26rft_id%253Dinfo%253Apmid%252F21789182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0021800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21789182&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.61" data-doi="10.1101/gr.187237.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jovelin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomasko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Full-genome evolutionary histories of selfing, splitting and selection in <em>Caenorhabditis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">667</span>â<span class="cit-lpage">678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.187237.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.187237.114%26rft_id%253Dinfo%253Apmid%252F25783854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvNjY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.62" data-doi="10.1101/gr.772403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kejariwal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlak</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">PANTHER: A library of protein families and subfamilies indexed by function.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2129</span>â<span class="cit-lpage">2141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.772403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DThomas%26rft.auinit1%253DP.%2BD.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2129%26rft.epage%253D2141%26rft.atitle%253DPANTHER%253A%2BA%2BLibrary%2Bof%2BProtein%2BFamilies%2Band%2BSubfamilies%2BIndexed%2Bby%2BFunction%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.772403%26rft_id%253Dinfo%253Apmid%252F12952881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjEyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halldorsdottir</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purugganan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Genotype-environment interactions at quantitative trait loci affecting inflorescence development in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DUngerer%26rft.auinit1%253DM.%2BC.%26rft.volume%253D165%26rft.issue%253D1%26rft.spage%253D353%26rft.epage%253D365%26rft.atitle%253DGenotype-Environment%2BInteractions%2Bat%2BQuantitative%2BTrait%2BLoci%2BAffecting%2BInflorescence%2BDevelopment%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F14504242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzEvMzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.64" data-doi="10.1016/j.jinsphys.2014.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urquhart-Cronish</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolowski</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Gene-environment interplay in <em>Drosophila melanogaster</em>: Chronic nutritional deprivation in larval life affects adult fecal output.</span> <abbr class="cit-jnl-abbrev">J. Insect Physiol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">95</span>â<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jinsphys.2014.06.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInsect%2BPhysiol.%26rft.volume%253D69%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jinsphys.2014.06.001%26rft_id%253Dinfo%253Apmid%252F24929224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jinsphys.2014.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24929224&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.65" data-doi="10.1016/S0169-5347(00)89061-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomulkiewicz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheiner</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlichting</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Adaptive phenotypic plasticity: consensus and controversy.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">212</span>â<span class="cit-lpage">217</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0169-5347(00)89061-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DVia%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D212%26rft.epage%253D217%26rft.atitle%253DAdaptive%2Bphenotypic%2Bplasticity%253A%2Bconsensus%2Band%2Bcontroversy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252800%252989061-8%26rft_id%253Dinfo%253Apmid%252F21237012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(00)89061-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21237012&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QU19400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasyukova</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genotype-environment interaction for quantitative trait loci affecting life span in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">213</span>â<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVieira%26rft.auinit1%253DC.%26rft.volume%253D154%26rft.issue%253D1%26rft.spage%253D213%26rft.epage%253D227%26rft.atitle%253DGenotype-Environment%2BInteraction%2Bfor%2BQuantitative%2BTrait%2BLoci%2BAffecting%2BLife%2BSpan%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F10628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzEvMjEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.67" data-doi="10.1038/nature11816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Discrete genetic modules are responsible for complex burrow evolution in Peromyscus mice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">493</span>: <span class="cit-fpage">402</span>â<span class="cit-lpage">405</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11816</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D493%26rft.spage%253D402%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11816%26rft_id%253Dinfo%253Apmid%252F23325221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23325221&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313615900051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.68" data-doi="10.1038/nrg1804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Functional mapping: how to map and study the genetic architecture of dynamic complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">229</span>â<span class="cit-lpage">237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1804</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D229%26rft.epage%253D237%26rft.atitle%253DFunctional%2Bmapping%2B-%2Bhow%2Bto%2Bmap%2Band%2Bstudy%2Bthe%2Bgenetic%2Barchitecture%2Bof%2Bdynamic%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1804%26rft_id%253Dinfo%253Apmid%252F16485021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1804&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16485021&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2135.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235384300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.7.2135.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A computational approach for functional mapping of quantitative trait loci that regulate thermal performance curves.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e554</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2135.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2123" title="Optimizing Selection and Mating in Genomic Selection with a Look-Ahead Approach: An Operations Research Framework" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2153" title="Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.]" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2135" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1_OfDh9-OHPbqPWhlAWRS0-__qbzurOp4o9WoTespOQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2135" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2135" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2135" class="active" data-icon-position="" data-hide-link-title="0">Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493672" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-GhuUWmMm4tpDQE2B2a3aUJD7zAwJq1V1KNptYS0qEDY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2135.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2135" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OXA56ZszTb6gvzSvS8AamYPQ6fWceN9KsKPZCSPLZVc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2135" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2135.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493672" id="node493672--2" data-pisa="ggg;9/7/2135" data-pisa-master="ggg;g3.119.400043" data-apath="/ggg/9/7/2135.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory W. Stegeman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8084-9383" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott E. Baird</span>, <span class="highwire-citation-author" data-delta="2">William S. Ryu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7141-0013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Asher D. Cutter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2135-2151;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400043 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493672--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory W. Stegeman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BStegeman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stegeman%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BW.%2BStegeman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Scott E. Baird </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Biology, Wright State University, Dayton, Ohio, 45435</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BE.%2BBaird%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baird%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BE.%2BBaird%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8084-9383" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott E. Baird</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William S. Ryu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Physics, University of Toronto</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Donnelly Centre, University of Toronto, Toronto, Ontario M5S3B2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BS.%2BRyu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ryu%20WS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BS.%2BRyu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asher.cutter@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">asher.cutter@utoronto.ca</a>
<a href="mailto:wryu@physics.utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">wryu@physics.utoronto.ca</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Asher D. Cutter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asher%2BD.%2BCutter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cutter%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAsher%2BD.%2BCutter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7141-0013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Asher D. Cutter</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asher.cutter@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">asher.cutter@utoronto.ca</a>
<a href="mailto:wryu@physics.utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">wryu@physics.utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2135" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2135&amp;title=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2135&amp;title=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2135&amp;title=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2135&amp;text=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2135&amp;title=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2135&amp;t=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2135&amp;title=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2135&amp;title=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493672/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493672" id="node493672--3" data-pisa="ggg;9/7/2135" data-pisa-master="ggg;g3.119.400043" data-apath="/ggg/9/7/2135.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory W. Stegeman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8084-9383" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott E. Baird</span>, <span class="highwire-citation-author" data-delta="2">William S. Ryu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7141-0013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Asher D. Cutter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2135-2151;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400043 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493672--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory W. Stegeman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BStegeman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stegeman%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BW.%2BStegeman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Scott E. Baird </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Biology, Wright State University, Dayton, Ohio, 45435</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BE.%2BBaird%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baird%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BE.%2BBaird%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8084-9383" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott E. Baird</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William S. Ryu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Physics, University of Toronto</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Donnelly Centre, University of Toronto, Toronto, Ontario M5S3B2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BS.%2BRyu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ryu%20WS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BS.%2BRyu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asher.cutter@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">asher.cutter@utoronto.ca</a>
<a href="mailto:wryu@physics.utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">wryu@physics.utoronto.ca</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Asher D. Cutter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asher%2BD.%2BCutter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cutter%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAsher%2BD.%2BCutter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7141-0013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Asher D. Cutter</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asher.cutter@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">asher.cutter@utoronto.ca</a>
<a href="mailto:wryu@physics.utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">wryu@physics.utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2135&amp;rft.epage=2151&amp;rft.atitle=Genetically%20Distinct%20Behavioral%20Modules%20Underlie%20Natural%20Variation%20in%20Thermal%20Performance%20Curves&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Stegeman%2C+Gregory+W.&amp;rft.au=Baird%2C+Scott+E.&amp;rft.au=Ryu%2C+William+S.&amp;rft.au=Cutter%2C+Asher+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493672/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493672/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493672/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493672/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493672/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493672/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493672/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493672/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493672/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493672/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493672/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493672/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2135" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2135.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2135.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2135.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
