<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2107" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2107.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2107.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2107.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/7/2107/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2107.atom" />
<meta name="HW.pisa" content="ggg;9/7/2107" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models" />
<meta name="DC.Identifier" content="10.1534/g3.119.400269" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Bourke et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="New genotyping technologies, offering the possibility of high genetic resolution at low cost, have helped fuel a surge in interest in the genetic analysis of polyploid species. Nevertheless, autopolyploid species present extra challenges not encountered in diploids and allopolyploids, such as polysomic inheritance or double reduction. Here we investigate the power and precision of quantitative trait locus (QTL) analysis in outcrossing autopolyploids, comparing the results of a model that assumes random bivalent chromosomal pairing during meiosis to one that also allows for multivalents and double reduction. Through a series of simulation studies we found that marginal gains in QTL detection power are achieved using the double reduction model when multivalent pairing occurs. However, when exploring the effect of variable genotypic information across parental homologs, we found that both QTL detection power and precision require high and uniform genotypic information contents. This effect far outweighed considerations regarding bivalent or multivalent pairing (and double reduction) during meiosis. We propose that autopolyploid QTL studies be accompanied by both marker coverage information and per-homolog genotypic information coefficients (GIC). Application of these methods to an autotetraploid potato ( Solanum tuberosum L.) mapping population confirmed our ability to locate and dissect QTL in highly heterozygous outcrossing autotetraploid populations." />
<meta name="DC.Contributor" content="Peter M. Bourke" />
<meta name="DC.Contributor" content="Christine A. Hackett" />
<meta name="DC.Contributor" content="Roeland E. Voorrips" />
<meta name="DC.Contributor" content="Richard G. F. Visser" />
<meta name="DC.Contributor" content="Chris Maliepaard" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;New genotyping technologies, offering the possibility of high genetic resolution at low cost, have helped fuel a surge in interest in the genetic analysis of polyploid species. Nevertheless, autopolyploid species present extra challenges not encountered in diploids and allopolyploids, such as polysomic inheritance or double reduction. Here we investigate the power and precision of quantitative trait locus (QTL) analysis in outcrossing autopolyploids, comparing the results of a model that assumes random bivalent chromosomal pairing during meiosis to one that also allows for multivalents and double reduction. Through a series of simulation studies we found that marginal gains in QTL detection power are achieved using the double reduction model when multivalent pairing occurs. However, when exploring the effect of variable genotypic information across parental homologs, we found that both QTL detection power and precision require high and uniform genotypic information contents. This effect far outweighed considerations regarding bivalent or multivalent pairing (and double reduction) during meiosis. We propose that autopolyploid QTL studies be accompanied by both marker coverage information and per-homolog genotypic information coefficients (GIC). Application of these methods to an autotetraploid potato (&lt;i&gt;Solanum tuberosum&lt;/i&gt; L.) mapping population confirmed our ability to locate and dissect QTL in highly heterozygous outcrossing autotetraploid populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2107" />
<meta name="citation_id" content="9/7/2107" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2107" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2107.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2107.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2107.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400269" />
<meta name="citation_pmid" content="31036677" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2107" />
<meta name="citation_lastpage" content="2122" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Peter M. Bourke" />
<meta name="citation_author_institution" content="Plant Breeding, Wageningen University &amp; Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands" />
<meta name="citation_author_email" content="peter.bourke@wur.nl" />
<meta name="citation_author_email" content="chris.maliepaard@wur.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0665-6508" />
<meta name="citation_author" content="Christine A. Hackett" />
<meta name="citation_author_institution" content="Biomathematics and Statistics Scotland, Invergowrie, Dundee DD2 5DA, UK" />
<meta name="citation_author" content="Roeland E. Voorrips" />
<meta name="citation_author_institution" content="Plant Breeding, Wageningen University &amp; Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4388-9244" />
<meta name="citation_author" content="Richard G. F. Visser" />
<meta name="citation_author_institution" content="Plant Breeding, Wageningen University &amp; Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands" />
<meta name="citation_author" content="Chris Maliepaard" />
<meta name="citation_author_institution" content="Plant Breeding, Wageningen University &amp; Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands" />
<meta name="citation_author_email" content="peter.bourke@wur.nl" />
<meta name="citation_author_email" content="chris.maliepaard@wur.nl" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. C. A. M. Bink;citation_author=L. R. Totir;citation_author=C. J. F. ter Braak;citation_author=C. R. Winkler;citation_author=M. P. Boer;citation_title=QTL linkage analysis of connected populations using ancestral marker and pedigree information.;citation_pages=1097-1113;citation_volume=124;citation_year=2012;citation_issue=6;citation_pmid=22228242;citation_doi=10.1007/s00122-011-1772-8" />
<meta name="citation_reference" content="Bourke, P. M., 2014 QTL analysis in polyploids. MSc thesis, Wageningen University, Wageningen" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. M. Bourke;citation_author=G. van Geest;citation_author=R. E. Voorrips;citation_author=J. Jansen;citation_author=T. Kranenburg;citation_title=polymapR - linkage analysis and genetic map construction from F1 populations of outcrossing polyploids.;citation_pages=3496-3502;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty371" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=T. Kranenburg;citation_author=J. Jansen;citation_author=R. G. Visser;citation_title=Integrating haplotype-specific linkage maps in tetraploid species using SNP markers.;citation_pages=2211-2226;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2768-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=R. G. F. Visser;citation_author=C. Maliepaard;citation_title=The double reduction landscape in tetraploid potato as revealed by a high-density linkage map.;citation_volume=201;citation_year=2015;citation_pmid=26377683;citation_doi=10.1534/genetics.115.181008" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Cavanagh;citation_author=M. Morell;citation_author=I. Mackay;citation_author=W. Powell;citation_title=From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.;citation_pages=215-221;citation_volume=11;citation_year=2008;citation_issue=2;citation_pmid=18295532;citation_doi=10.1016/j.pbi.2008.01.002" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=C. Cellon;citation_author=R. R. Amadeu;citation_author=J. W. Olmstead;citation_author=M. R. Mattia;citation_author=L. F. V. Ferrao;citation_title=Estimation of genetic parameters and prediction of breeding values in an autotetraploid blueberry breeding population with extensive pedigree data;citation_pages=87;citation_volume=214;citation_year=2018;citation_doi=10.1007/s10681-018-2165-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. B. Endelman;citation_title=Genetic variance partitioning and genome-wide prediction with allele dosage information in autotetraploid potato.;citation_volume=209;citation_year=2018;citation_pmid=29514860;citation_doi=10.1534/genetics.118.300685" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=L. M. Evans;citation_title=Narrow-sense heritability estimation of complex traits using identity-by-descent information;citation_pages=616-630;citation_volume=121;citation_year=2018;citation_doi=10.1038/s41437-018-0067-0" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. S. Gaut;citation_author=S. I. Wright;citation_author=C. Rizzon;citation_author=J. Dvorak;citation_author=L. K. Anderson;citation_title=Recombination: an underappreciated factor in the evolution of plant genomes.;citation_pages=77-84;citation_volume=8;citation_year=2007;citation_issue=1;citation_pmid=17173059;citation_doi=10.1038/nrg1970" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=C. Hackett;citation_author=Z. Luo;citation_title=TetraploidMap: Construction of a Linkage Map in Autotetraploid Species;citation_pages=358-359;citation_volume=94;citation_year=2003;citation_issue=4;citation_pmid=12920109;citation_doi=10.1093/jhered/esg066" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=C. A. Hackett;citation_author=B. Boskamp;citation_author=A. Vogogias;citation_author=K. F. Preedy;citation_author=I. Milne;citation_title=TetraploidSNPMap: software for linkage analysis and QTL mapping in autotetraploid populations using SNP dosage data.;citation_pages=438-442;citation_volume=108;citation_year=2017;citation_doi=10.1093/jhered/esx022" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. A. Hackett;citation_author=J. E. Bradshaw;citation_author=G. J. Bryan;citation_title=QTL mapping in autotetraploids using SNP dosage information.;citation_pages=1885-1904;citation_volume=127;citation_year=2014;citation_pmid=24981609;citation_doi=10.1007/s00122-014-2347-2" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. A. Hackett;citation_author=K. McLean;citation_author=G. J. Bryan;citation_title=Linkage analysis and QTL mapping using SNP dosage data in a tetraploid potato mapping population.;citation_pages=e63939;citation_volume=8;citation_year=2013;citation_pmid=23704960;citation_doi=10.1371/journal.pone.0063939" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=C. A. Hackett;citation_author=I. Milne;citation_author=J. E. Bradshaw;citation_author=Z. Luo;citation_title=TetraploidMap for Windows: linkage map construction and QTL mapping in autotetraploid species.;citation_volume=98;citation_year=2007;citation_pmid=17965198;citation_doi=10.1093/jhered/esm086" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. B. Haldane;citation_title=Theoretical genetics of autopolyploids.;citation_pages=359-372;citation_volume=22;citation_year=1930;citation_doi=10.1007/BF02984197" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. G. Hamilton;citation_author=R. J. Kerr;citation_title=Computation of the inverse additive relationship matrix for autopolyploid and multiple-ploidy populations;citation_pages=851-860;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-017-3041-y" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=C. D. Hirsch;citation_author=J. P. Hamilton;citation_title=Spud DB: A resource for mining sequences, genotypes, and phenotypes to accelerate potato breeding.;citation_pages=1-12;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_title=An introduction to genetic statistics;citation_author=O. Kempthorne;citation_year=1957" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Kloosterman;citation_title=Naturally occurring allele diversity allows potato cultivation in northern latitudes.;citation_pages=246-250;citation_volume=495;citation_year=2013;citation_pmid=23467094;citation_doi=10.1038/nature11912" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Knott;citation_author=D. Neale;citation_author=M. Sewell;citation_author=C. Haley;citation_title=Multiple marker mapping of quantitative trait loci in an outbred pedigree of loblolly pine.;citation_pages=810-820;citation_volume=94;citation_year=1997;citation_doi=10.1007/s001220050482" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. A. Knott;citation_author=C. S. Haley;citation_title=Maximum likelihood mapping of quantitative trait loci using full-sib families.;citation_pages=1211-1222;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1459438" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=A. Korte;citation_author=A. Farlow;citation_title=The advantages and limitations of trait analysis with GWAS: a review;citation_pages=29;citation_volume=9;citation_year=2013;citation_pmid=23876160;citation_doi=10.1186/1746-4811-9-29" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant. Genom.;citation_author=J. Li;citation_author=K. Das;citation_author=G. Fu;citation_author=C. Tong;citation_author=Y. Li;citation_title=EM algorithm for mapping quantitative trait loci in multivalent tetraploids.;citation_pages=216547;citation_volume=2010;citation_year=2010;citation_doi=10.1155/2010/216547" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=X. Li;citation_author=L. Li;citation_author=J. B. Yan;citation_title=Dissecting meiotic recombination based on tetrad analysis by single-microspore sequencing in maize;citation_pages=6648;citation_volume=6;citation_year=2015;citation_pmid=25800954;citation_doi=10.1038/ncomms7648" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Luo;citation_author=R. Zhang;citation_author=M. Kearsey;citation_title=Theoretical basis for genetic linkage analysis in autotetraploid species.;citation_volume=101;citation_year=2004;citation_pmid=15100415;citation_doi=10.1073/pnas.0304482101" />
<meta name="citation_reference" content="citation_journal_title=Constructing Genetic Linkage Maps Under a Tetrasomic Model Genetics;citation_author=Z. W. Luo;citation_author=Z. Zhang;citation_author=L. Leach;citation_author=R. M. Zhang;citation_author=J. E. Bradshaw;citation_volume=172;citation_year=2006;citation_pmid=16415363;citation_doi=10.1534/genetics.105.052449" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=Y. S. Ma;citation_title=Potential of marker selection to increase prediction accuracy of genomic selection in soybean (Glycine max L.);citation_pages=113;citation_volume=36;citation_year=2016;citation_pmid=WOS:00038214;citation_doi=10.1007/s11032-016-0504-9" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=K. Mather;citation_title=Reductional and equational separation of the chromosomes in bivalents and multivalents.;citation_pages=53-78;citation_volume=30;citation_year=1935;citation_doi=10.1007/BF02982205" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="Pinheiro, J., and D. Bates, DebRoy S, Sarkar D, R Core Team 2017 nlme: linear and nonlinear mixed effects models. R package version 3.1–131 https://CRANR-projectorg/package=nlme" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Genome sequence and analysis of the tuber crop potato.;citation_pages=189-195;citation_volume=475;citation_year=2011;citation_issue=7355;citation_pmid=21743474;citation_doi=10.1038/nature10158" />
<meta name="citation_reference" content="R Core Team 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria R version 3.3.2" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Syst.;citation_author=J. Ramsey;citation_author=D. W. Schemske;citation_title=Neopolyploidy in flowering plants.;citation_volume=33;citation_year=2002;citation_doi=10.1146/annurev.ecolsys.33.010802.150437" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. H. Reyes-Valdés;citation_author=C. G. Williams;citation_title=An entropy-based measure of founder informativeness.;citation_pages=81-88;citation_volume=85;citation_year=2005;citation_issue=1;citation_pmid=16089038;citation_doi=10.1017/S0016672305007354" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=U. R. Rosyara;citation_author=W. S. De Jong;citation_author=D. S. Douches;citation_author=J. B. Endelman;citation_title=Software for genome-wide association studies in autopolyploids and its application to potato.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2015.08.0073" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating the dimension of a model;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. T. Slater;citation_author=N. O. Cogan;citation_author=J. W. Forster;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Improving genetic gain with genomic selection in autotetraploid potato.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.02.0021" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. B. E. Sousa;citation_author=G. Galli;citation_author=D. H. Lyra;citation_author=I. S. C. Granato;citation_author=F. I. Matias;citation_title=Increasing accuracy and reducing costs of genomic prediction by marker selection;citation_volume=215;citation_year=2019;citation_doi=10.1007/s10681-019-2339-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. G. Uitdewilligen;citation_author=A.-M. A. Wolters;citation_author=B. B. D’hoop;citation_author=T. J. Borm;citation_author=R. G. Visser;citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.;citation_pages=e62355;citation_volume=8;citation_year=2013;citation_pmid=23667470;citation_doi=10.1371/journal.pone.0062355" />
<meta name="citation_reference" content="citation_title=MapQTL 6, Software for the mapping of quantitative trait loci in experimental populations of diploid species;citation_author=J. Van Ooijen;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. W. van Ooijen;citation_title=Accuracy of mapping quantitative trait loci in autogamous species.;citation_pages=803-811;citation_volume=84;citation_year=1992;citation_pmid=24201478;citation_doi=10.1007/BF00227388" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=C. A. Maliepaard;citation_title=The simulation of meiosis in diploid and tetraploid organisms using various genetic models.;citation_pages=248-248;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23013469;citation_doi=10.1186/1471-2105-13-248" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. G. Vos;citation_author=J. G. A. M. L. Uitdewilligen;citation_author=R. E. Voorrips;citation_author=R. G. F. Visser;citation_author=H. J. van Eck;citation_title=Development and analysis of a 20K SNP array for potato (Solanum tuberosum): an insight into the breeding history.;citation_pages=2387-2401;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2593-y" />
<meta name="citation_reference" content="citation_title=Mathematica Version 10.0;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=A.-M. A. Wolters;citation_author=J. G. Uitdewilligen;citation_author=B. A. Kloosterman;citation_author=R. C. Hutten;citation_author=R. G. Visser;citation_title=Identification of alleles of carotenoid pathway genes important for zeaxanthin accumulation in potato tubers.;citation_pages=659-671;citation_volume=73;citation_year=2010;citation_issue=6;citation_pmid=20490894;citation_doi=10.1007/s11103-010-9647-y" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=C. Xie;citation_author=S. Xu;citation_title=Mapping quantitative trait loci in tetraploid populations.;citation_pages=105-115;citation_volume=76;citation_year=2000;citation_issue=1;citation_pmid=11006638;citation_doi=10.1017/S0016672399004395" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=F. Xu;citation_title=A statistical model for QTL mapping in polysomic autotetraploids underlying double reduction.;citation_volume=15;citation_year=2013;citation_pmid=24177379;citation_doi=10.1093/bib/bbt073" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zheng;citation_author=R. E. Voorrips;citation_author=J. Jansen;citation_author=C. A. Hackett;citation_author=J. Ho;citation_title=Probabilistic multilocus haplotype reconstruction in outcrossing tetraploids.;citation_volume=203;citation_year=2016;citation_pmid=26920758;citation_doi=10.1534/genetics.115.185579" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="New genotyping technologies, offering the possibility of high genetic resolution at low cost, have helped fuel a surge in interest in the genetic analysis of polyploid species. Nevertheless, autopolyploid species present extra challenges not encountered in diploids and allopolyploids, such as polysomic inheritance or double reduction. Here we investigate the power and precision of quantitative trait locus (QTL) analysis in outcrossing autopolyploids, comparing the results of a model that assumes random bivalent chromosomal pairing during meiosis to one that also allows for multivalents and double reduction. Through a series of simulation studies we found that marginal gains in QTL detection power are achieved using the double reduction model when multivalent pairing occurs. However, when exploring the effect of variable genotypic information across parental homologs, we found that both QTL detection power and precision require high and uniform genotypic information contents. This effect far outweighed considerations regarding bivalent or multivalent pairing (and double reduction) during meiosis. We propose that autopolyploid QTL studies be accompanied by both marker coverage information and per-homolog genotypic information coefficients (GIC). Application of these methods to an autotetraploid potato ( Solanum tuberosum L.) mapping population confirmed our ability to locate and dissect QTL in highly heterozygous outcrossing autotetraploid populations." />
<meta name="og-title" property="og:title" content="Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2107" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="New genotyping technologies, offering the possibility of high genetic resolution at low cost, have helped fuel a surge in interest in the genetic analysis of polyploid species. Nevertheless, autopolyploid species present extra challenges not encountered in diploids and allopolyploids, such as polysomic inheritance or double reduction. Here we investigate the power and precision of quantitative trait locus (QTL) analysis in outcrossing autopolyploids, comparing the results of a model that assumes random bivalent chromosomal pairing during meiosis to one that also allows for multivalents and double reduction. Through a series of simulation studies we found that marginal gains in QTL detection power are achieved using the double reduction model when multivalent pairing occurs. However, when exploring the effect of variable genotypic information across parental homologs, we found that both QTL detection power and precision require high and uniform genotypic information contents. This effect far outweighed considerations regarding bivalent or multivalent pairing (and double reduction) during meiosis. We propose that autopolyploid QTL studies be accompanied by both marker coverage information and per-homolog genotypic information coefficients (GIC). Application of these methods to an autotetraploid potato ( Solanum tuberosum L.) mapping population confirmed our ability to locate and dissect QTL in highly heterozygous outcrossing autotetraploid populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Jrr53fzHjqOHFWB1ocmWSmV40mgCuDwPawftQ13_OWY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493468","apath":"\/ggg\/9\/7\/2107.atom","pisa":"ggg;9\/7\/2107","ac":{"\/ggg\/9\/7\/2107.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2107.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2107"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2107"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2107":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZVsUFbvfik5i\\\\\\\/FN9LsDX+OTCvt3istwCC86rmSsCtxDXLwKQy1HRssdsm\\\\\\\/6GNU\\\\\\\/Z2KFehZuc+GxtYvcpxJZFmho5+\\\\\\\/+MpViGStbBYQD6v4m+lY5ycxbWCNBR1lkt3gfGbnVCGQ7oDPb5LFWwZnwoa30TFZT\\\\\\\/gmCP6CPRbGNb8UBYh9puguojY6gbj\\\\\\\/0QBqIRIHYw+mxghHOkjEZV5E7ZZBTbt48T7Z1tmOB6C6wuIyQS7rWcCJSbTdNQQn1I+\\\\\\\/WEvG2iAqdkSPIW5Ep\\\\\\\/f7Gg+PeBnyrXhhrm6tLXPHySDEqC7lqv8nyq5BFrgiRt9Gypblw8IY1aH89SrswXzntn8+rE1trbByzjjiGUUPNLDf+ZVmkiOl21ZatR\\\\\\\/Ca2qOnJwaF9eyqgA02ctTiSwTniDc9KfbU+3aCSMroVrxuoywdCZSPXxakzMzB4t+WYI9MS7zdbX9GCvCx26B+PFY6QVDZfgorbDywy\\\\\\\/HoKOAsfU9x8U74eHiMbmgm3aIHTJbtAiTuPJjwVYFi54Vw5wNC7+baHcNvZLPIhgt7d5Uo87r09tjFpyk\\\\\\\/eJI0IOOBoi3eJZeag3iDeKrNkSl6QXf\\\\\\\/r3FqCmgnRQxKwDR68Tm540m3VvytXaszS6RGJuoemLjQ\\\\\\\/rw\\\\\\\/8AR5zfOmikFrzNvbWWV+8IbZ\\\\\\\/H5xeXfXaJMUbSfYhsXDnhh8l3LPwLA0HKZekhyZld9JfmTbZATLhsMvI1mFAsgwpMQO604CNHxFEXTK2zWChqC2MhdyqZ\\\\\\\/55QVjMF7bfkeEULfCKAc1++Af4FNKxLo9+ncRomb69m9Ynz3GWLjlALiwN8nR3TxeZUy4G\\\\\\\/zpQL6hA8YCjeglfjDSpJbTOpefiPDOO38qubKGLfYfpxqlVp1cMu+Pe5WN9tWGLHO5f0\\\\\\\/tPnNZhQKMkKQ2kHU9F\\\\\\\/67ssyZMqSXpB5jC0qUnkTX6rm5gNEMayKCcm1yZmXSlUkfD0WHBN0btttLrG0TXG36K9\\\\\\\/E0lUeo0yYeBXuD5v1PIQ\\\\\\\/HGMD2TXp4snZFF8Yh6oXTvJlDxN4TrVT1LA2R9f2FZJnOs6zyTpr2P8S7eZGItf0i1iH6qh+kqcH1As5Lp106ZeFg3oFuRHW9WbpDViKxwRFatrqlIVHtaepVmARME7Vp1hXRJDfUiiYGD9lcwgFR6PMVOR5lFNKgr5vP90D38\\\\\\\/FfZxAIUPpE6QOcQBwhtX5d0buHoGVvsPM0d64QQeMwe+Ls7piVbbA0KvS8QqVSprVq4F85caCAcxVnQUnkQ6aaCglZKLi2JPg9\\\u0022,\\\u0022iv\\\u0022:\\\u0022neL225OM4n5VidEZthaWNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cdc23177a4487d154e03c283a339404efddba5ea4f1bda372bde330d1afb0ade\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YqvuvE4mWT1A4bNssUkjqvsH73eWunk2BGDpcYaVU8NBp1FRXrPqVnqfqRxklwLQ8MTR4kEUJiB5g23y3puipGxtLUKR79sMVWEm1BS0cNJ3lBWXu8eVb25NcDWQJS3OfjconJpoUksv3bBQOIu\\\\\\\/6aZ9wAOxrYP72HfLIE33Qf+hucleZIaYRV3gPlBE2FOvr1M5LJLXgxGQY8jMLb4Dr1GubXw3UOT+q9mZALcbuB9MYjRISAi1DxVA9upkvZedBmJ1bSpBSOaG1hroKxplw7ASBOrsN9eETz\\\\\\\/eyxm3O1H6rByPeTtOrLLF3BuejFYhDkWIHEvVChOeqWy4zcrIPIzb3gUoGaLG3glLSB7Yy9RzJhbsxGO2NIbW2QUdHQcFlFef4K35RAx17IEFP26jgYFMgxLwIrPkVvKohj828QOdSq4MBMMS3DmFkfiAwsvA+2HRMu9xhJvsrsJj5lEnqjVSYjx4calf35dU733vJ1sTuBBN1ZBxBflEby67K4BK7xSHEu4Q+jInzOt\\\\\\\/S\\\\\\\/Xapht6AYGE1EGj9wZWyfQ3QfN9kRq28Cmb2GbiKe8KZP26\\\\\\\/RPwgfpsf62r4okh07kCfuQ\\\\\\\/D\\\\\\\/Z\\\\\\\/vM97KMJZy2cQF1vT+9T+kiEFuSlRbNatv+YR3SvI4eat4nao++oEZhQYDesNl\\\\\\\/SlwR7gPb4M5\\\\\\\/ShqbcwGJMwTVAuApyy6I0OJdysknD++6KBvNORXIlmqaoNpVQkPT0aGyk7tp18I4HkKAvwCXYjiZyT6JgayLstIlIi39Ko4gjZY67zeGG59V1C+cbvJ2P2KXh7q1b3nDvRz6bzNHuUrHwX79iTJXMg+J1zbPOcUNwHsg1NcHo9gblRw7FE8vJUo5p++ez2kiNdzyLqPVTfpoaX25f3YF6iDBCF8pdFIEoX6RGr29woh8aKkJou9urSOy354dSCSYMiov4e5+mox6iLTZcWVPNuSTSyjTLvbtZ5xtkMQG5nlxPhFK5XEdfL4biOjHl3YnqTp5GBUvsH4lDH7dbk8Dpe+4lsEslStoZkOqPLTA9EHqi616viHYnxShIHiTeUPvuOC2i1c+m5dOnyfsJqTJnWcmkoLqdw87mCNG8lXaPZ6KykJtvWr6eGdVNog+RhBZpY5HRc3lILeKa4zlG\\\\\\\/Aqf5YvjPPGHIvB8IKgKyxUTffOA\\\\\\\/u7hMwhut42Y65vXtWziiXF0rcCZCVKgEh2Sz64v3L6b6YlAojz6dmeWRYFjlLBeLwAHIb2K6zvhOwJd3EE3nhFflteKQyPMgOaUGSVzGyTflcjEvB9lzSieJlOLW31wJJ7U+1a1GPt9TzlYUETqnxAs5dZXBDOrvusAiG9TpBy7rLRare+aNLZk8idEA3+x+PgLq+mqcuxK7fQtwWtyqqqFr3DeO1iSZKqd2aUjBEIuHw5mjq2cHnH3u72GugalDGS+\\\\\\\/5hXzfKolou68q00NtoFaOXYpfQR7ldF54ZcrpfEKQjcGw8Jb0riZ2zq3z9FBXa9XU365CMAYI8v3vW3zKn5chBC3UacF9Q\\\\\\\/IFQYN5V\\\\\\\/qjYJ9gxlAsJo9v0ASHOHExfdapech4KT35sBYk1LKk5qKZScWCi2qK+eJy6xbrxRP8tc05yRVg6ZBLRVLgutv9DTNZhPmuaPuzCVBYamvflg48dpl3IJhr6y5AGA26GoMXzpJn2GaFR+vARA2bU8fK2T\\\\\\\/kGBgiUHIlhR3PnxN4xDX7ARcp4AiBkmjMP\\\\\\\/W+rE2+WAqz14egwQRokoIbm41PHyZ94lsQoTojodo2Ip++70OIAwC8GhD8BcFMQ1ZmHPNWo0DMtjnan7GUAAxCCyTE8qGcxuC5bKs1hnp7uL\\\\\\\/1xsbHOjhq7t4FHPrOIHEnW936+DnRAJrw5oFjxnSus5z3AZtzU4nHjHMJJfYX5FOvaGX\\\\\\\/ZDUoxqNW5KM5GnSFXcsy1wRZKmnDDND9HObB\\\\\\\/0YcD4YnFzqn6EyRM64ZR\\\\\\\/Dw6AdpxECvxdOri8U52XbT8WDlUTE6J8g9XLc1CBcdTORRu5ZGNogpKSqIP2sIQZsAOXrezP7898l+0mHdWM1J6+sYQ2yapE8Ru3g4fxP99usDTtxW8+mQf1UoRUtsv9\\\\\\\/WDPSn5HBCwcnMBuVd7Au0qeLJOIplW2uRflRVB8QZT4KW5GFwwfd+x8+Vc13E48ovUnzobZpobAPadpIvMmBq\\\\\\\/g672grsFVvf3FKW2REtL64Y8ckYagqj5aejaGyJ6m64Tn59NUDRMK\\\\\\\/Cj1QfiAzufJdGg2plROGXECF7aPj2IQq93Tnkg8tlKOdhGkLYCyt9vazN+RSujvhT+Cc4BYyRSXB9gHsX+xuci8HBblbpWnPLw8U\\\\\\\/ukN8lEEMB4jc0Ns\\\\\\\/\\\\\\\/LAhoU7YA4ZSwa6MAwl5pqZj0cw3YAIVXN0uQH5bEOML\\\\\\\/kPJLHzKA6O2ZF+G3+\\\\\\\/OwWoNgMU21Egg6EjWepEr0ji59jT6M8d\\\\\\\/IZgU7rt0IZWy0ab2Pgj2O+cCvUdPWR2GcbmeXFHrnu9nXmCfa2QFlCuuJoScP5f8p85PeKV33VdJYuk817hFEUNvuOkccv90f7ev+Yfoul7wStYdOAi8ZnKG1m\\\\\\\/Ps0LRG\\\\\\\/C4npRrc2l6M40M7CqPM\\\\\\\/JXjcQ5whK1fSx5NI0WvweRVrHNwSISm5BqnHsoVPw\\\\\\\/GslXfSh1\\\\\\\/qYOW9gTgUnXheGDngM41fdY9o01uSFZyq1bw\\\\\\\/Hz5FYCZpzhCE1ifHZhYd2Wryh7J72a4MWq97mr8Ycq9PNdu9ov35PncnlItT0Q7UCfj6kD\\\\\\\/EXDH\\\\\\\/LWs14Mp5YqAUPOcvHKnHXfYAOVw9xiS\\\\\\\/YlV097UUfxOONpzpqKMXQysM7tYnSsV0j5GjMbfzE\\\\\\\/2CGlbu\\\\\\\/2mVMZIvozF2F1+wqNhEESe4comX\\\\\\\/WNXolf6CcNB1fD1aFTyIcH8gdpwCr+YH08kd+gYTUnhIGzLUoH5BxLOAsDvnRLYaCGzKKNb5tySXZ+F025zLS0cYTbBf6GfU0Z0QdoU780iXqAMMgvwERDJzHZ1vJGF222cNlvmjlsNms7JI9w0G3IkSIVqfTY9aM0fXNY2y\\\\\\\/9mPjfpQmexn2zPvqDJ2268skDAms7TzFWXV+1t6XB\\\\\\\/5UXdSjvcZW\\\\\\\/r+58B9jacYMAn+m\\\\\\\/LQjth2VNNULl6CQzeJVComXrrEzEFRQiOWrgiP4e3unLW3WlydWccunYNUJJ0O0LkCHGXvnVph9DxAxdi26fiHuGG8k3\\\\\\\/ZUl1VcZlOiLgu0\\\\\\\/8mOZDL1mP5JaZDJ994zsxPGQ6bx5oZAMASE37K+qie\\\\\\\/7c0Lmb2nn\\\\\\\/0uSDz2lJ9oeT39obyPCaX+oUXcSI1wOOBHxOSqeDF\\\\\\\/URbGKFDn2YQZcRLUCU7k4st9HnnGPDTKs3NqPQ47Uf02IwpTozzGaF7QPSqhbPCkUnbgUxvRxipRpdD3iXmS57OlE7pIxwXVaIWIvb0Ekay9MoEIQE0TeCPZYRQs0ltU7tBm91LwU5lMYbpnwp8XnzGWSBudqeYiEz73HF+jTpprrapb1x1FVRMO0caZZXlGkPUk40ziMU0CGaC5QBHxlqxKfE7VN+Uylynosau8b8AKOS2iPWwTaDl95xVa5I8yBwonOG\\\\\\\/v5+RhfubZmzJtIIEBazn4oFTEmxZdAHDUpKZTbF4kNaHKLwRue5L0S\\\\\\\/\\\\\\\/0MVp4Y83AqfqaZNpst4TD9GImM5KyuBpgm2T1BFoPybQEsmNjyhwKTJrUjUY9xlkedtWy9JE0qJcW7wQgotd2duODderWivOLJwmsgeLdmjUgpXWqnQCY02lJ2Tym\\\\\\\/4Lkj\\\\\\\/6EIYZ6cwoEdNeNZau1qPjeuoEILdBvKl4OW9mDZUt4nuaBqlpak+mCrFAHyLHlFxWIAWjw5lsySvP++yj1kovM3PprZb4vUIqDF2t+n42mnaqFhkwkr5g+r2gAGxd4yH7QsAWurbt7YBforsshVKUlq0fSQ6xR+Zn3TYnPQGJCsIDTMNbeo2JsKST0travqPmQi0W\\\\\\\/6bckPBjXrCkViJBdC2oZtMU4JIken6y7muzlV3fuQYcMAN3JxPRrI8q8pFk37NzoUpKxI6PSBIAwL4n8qwVPazFtCDkNx5W96KmVBFNdWcI7tO0zCt0KD+H9UGqLE3b5CCrSPiAvhoi4i55SWRprQWObEq61Fcu9aGJm57tctk1vInIEwgoILdviP0UA802Om6vdJrNRd4ujXkYDPKfMcIEjLIB7ocYfQzdE0rHeeH3qQWrMStEcrgCUjFbU9NWHeO\\\\\\\/24ZHXomy0W1jPUxC6CZ\\\\\\\/jw5oyB+U0VJz\\\\\\\/CzYya4lIdEq8uG+dJSaD9MUqI0rEgdHiuKr0ZDIihHvAfMFZsTBSKggjnPITnRqGbzgaN3Vv9n56z0yPv78wFo+Ppxj1Q7dX+6G02eRkg54PoFwwuivuKRr+67zvPre+DOhkK3DrdebNib8N4X2MVsu9iiJO+95ScqEVSIPC8rIufivA+3ONk+pWo7r+4TLK2\\\\\\\/RVCv4lbMbqmAoaei48JfgoGlCKgT\\\\\\\/lBVUYO2VNg8I2WdfPpzJPRaC8zzvnKjNeJITERHpVLQsGzFsRvTphQo+gQnCp0L5IsvGM6m3oOLZLILkuUi4yKzOjVQcbhQL+rMdeoGvUr3n7RfiBEYZnHQFGsixAife3yEJidVtH5Zp4zF2n\\\\\\\/UZ88n15l0yJunMHLNDlj\\\\\\\/\\\\\\\/iXcMviUdwp+VeDadr3RJyTycpeHGiae38qx6vp9aGj\\\\\\\/D4AWPnrVmEImwAnBxbGcWy+j6Kq4e6iyjgmsIjdGZWAXxbFnBVrBut2s8QxFvyUX77QsSZDjZ3ZJgg1x0z6b\\\\\\\/dvhuSpMd3Mri\\\\\\\/owEWvaOQeN4A4pt2c7jdqU2DAsUrJk6j9xGQlmd1ZLEwGwiFB9j0nAI8MhUiKfgBBFBwKyRS7r7u86F\\\\\\\/Ar2v8\\\\\\\/32LJ6LskVzI\\\\\\\/22B80t+aILNtZ7DaQk7F+rWy5dtcklxDdC6H+4H3vnxcajnKa0pukP+fFirU6UokQaG9mbPDN9hsu+kGglyJiHEswLMXdHpnMt\\\\\\\/TYaYYZeqUV+ppAIkEa27tB+WOwJiVnbL0asfNedbDMrk8FCYwGy9pQEIYwg4YBrOiLAEjk9B5lsP4qpZJqrO9TNSl6WxvXwvw5mUZwYZ84D4yTIX6a3l\\\\\\\/ZSmM1LRJr1gx5z0S89J7\\\\\\\/OWHaXIHBC56ZJOVFPIBo0zwoSh+L1ViqvMX96kn3e9R\\\\\\\/tC45t21AxaQb+c\\\\\\\/kQLQfoHdSu3CdWc4yPhHyhOYroaV92\\\\\\\/GP5vRAF4BT1BIjDTqnhJU2+nRQKVGAncnaTUSZ\\\\\\\/WY\\\\\\\/jbv+e+XOIjcltqAYQhmyLL1NFKezz86dDDSh+RbAjd4uiOVSeiPevem7yD7t3jFqVrUY1jwyC\\\\\\\/8HAsAx6rNUbdjhmV0IAcJRXsgHb1s5vuoI9Tsk36ilIOaCLQjyHUXIZD1xvgzQJe7ksRwLMQtBA+KIo4OfpdKeszj7KHnLPKf2smwG1OBH1LrlzJDWQtY6208Be56mCP2eskqBlGZD0tKbw9SD3p8dxqDBEc2EOyY11Ho4TTgEkVOKpbHJ+lB1Jb876unYPDsn7hCcjTmhLDqe88tASjPZMdieSbdUkVAU+E9AeX1wA5c2wE5O0PqTDrpdQ58ENe5wHfR8DZDwm9dGa9tksn6t4Nkym\\\\\\\/GM56poT9\\\\\\\/IvNyjbkIjpTB1+CwcxHy8AyYqgfzlpEOUXxOEmmYwPyxpIm9ErfQVbRQqbdVEmVO\\\\\\\/uuo3pXgshRaskVNYc12gd\\\\\\\/FCazGFdnWz6IOU66g5SxQmGedSV5BSmXoDmcNmfO+8ER6CyigCem5lMuUt+QD4J7n5l5hFTataq99fmfVKkeGh5HRwrhbRgiALTwh\\\\\\\/lXanNoMOXrRynqfRT3PEwbO6wftDGHe6VoHOczzyCVwYpjRVsjgL2JOH\\\\\\\/vLlmFSRTqtQsiWR2pEmYB03s0ozI7KKw7wehrvxcsdqvAEPhaRMaenO2ycn+KKYop6wNbaFEL98IfZqc12HZ2\\\\\\\/Sg3UIuSgQ2UAqGvjLEL98YC7tq9TTnZb1PcclbVpn0XET4ltxNysVvgc7VCnsq\\\\\\\/LtmXrQQmU0CMz8ynge+eAW41tMnNWQ5xlLcepWoGterxLbE55C+yOjH\\\\\\\/jeUQyyH9av8+j6pYP8HqrW9IAPkSs4tHOq50lEy+MS\\\\\\\/EriGIxQ\\\\\\\/n9ENoGP+lWOwqlWjGXUluS+kNkT\\\\\\\/hjgFr7MZzf+Q9x88IyppAn9864km9DlBxZ0wQ5XvV9A4MQrPSRRzea4CB1ZW5CTyi5KFxgmkv0dLkBtZTxnKOqKPVwXqsImXwnTXTksJtMr0s2ImEcNTQDlQhWyNJITzjkeAS7By5KbGKjG2MYxaHmSBTGTuFW\\\\\\\/g9Q1NZorL0W0OCKE\\\\\\\/2TGlx6TIOYEa61oTeCUjoBOMk9uBz17gxs6EdusKY1bK4QcdvIaBr+\\\\\\\/K+iYsRgt9TXrcvtCPC0BGAZRmug9tpWGa4EsO\\\\\\\/bYu7jkkqlfzWI1gBnTCrgkuoT12tcgJGBfIOWnFAdd+P4ZV52dLpxorQ5V201eF7ChDRmPTDiJujVYr6gvTYM2HOVFeqbDmx0JyafmcC+Y+xVdj4c7B++QhHBIiPZnPOUL5WyJunMXprFUL0Xjq3KTq1Uz1GKpsPYfud\\\\\\\/h4MdRFX\\\\\\\/B0maXKiSOsoV7FazXBBHW+eK29VraUUhxrOGKvHV5gw2hdcJJB6FYxiOpITa7eXfJocgiTOfIoMZfaLEU9b1U2PTKRg8IKLBe7baUp4rGKBX0+RBdPKEB2OOPqv+ReSHK1fnIN0S\\\\\\\/ZJoKHxoyla3gDQHGCGec13XYi5vHKrTeRf+85\\\\\\\/L1Djome3E9Qum6SgCR27K6IA\\\\\\\/95EqkQJ23pQi8o+I9xdHpIwmlKLoFmjcXCWJ5pvSvFs5bC+2Ta6nJfsW+v4v8BtBWwWll\\\\\\\/+3FBNYcN5OFFRCsRJ36UEk6CeU+yNsULMekGL7HOMYCuxkQg+Hr8fTcrFREZnH839\\\\\\\/uT2gkjnW67ydkcYeON\\\\\\\/jeEn1SzjOQaRiKfctQ4\\\\\\\/q052tl71lSFhBobSrNPzyc\\\\\\\/nI1Du1ATbUshoL1HotnPB6qy9TilQP\\\\\\\/U92OvLagRJudvRbu29HyJz6uWhoZ1TS3F5Ydh4K45P6W00zmwO2iYX2JqQJ+8\\\\\\\/EPk8DelZZPd007b8wz8QycNBXw9UGVsQl5TsQPx6BxXF5N4+wyS3C3kYDZGP0Z5BMxhgBzB5rI1EAlvkeFcouLSVHfGUFqx0Ty40SNhXcBjnJbq6HQ57YIqpoXURiRJXUC7RHHg0RM+5hbfmZ1oB8VnIjDT8FPsaKtHIbigjgwCaPbymaaxtQINXs841bur6TF8y2jtRXhfc5JtT+B4g+ihJL1rlT8SyCmCb4nW9Ikp0+3JHvhzOOjJmLzOpdc5mAAa69+nSUuvzKlQA426DPW6kveB4kjplbGAFHYH87v\\\\\\\/00XrvT79SBdeh5lycvuvHKu7w8B31jh9yHM1uncLEBTwCe4Ug08hGq83xZK6xnjyG5m47D+kjh6jggKBxISlW4Ll2BgTsVLLKyz0yyaTAzt7FR6V8JQRWvIDnq473Js26h9ZO9gbqudHOqHfRueF\\\\\\\/bfoiIkmailksWdE1cnAHl3\\\\\\\/m1kQ3\\\\\\\/5ceoZGnmCu+Di7Vf7F\\\\\\\/bT6SUi7qm3\\\\\\\/dVrkqvFYEzYQpdVJq5yDthGdmcGjTSLhVaFfAJ1hpsCcO4hNqP7xIRWzK\\\\\\\/pDYBGzFAjMPTK4NZJtnzSScTXu4Le1txI7E0EH3Na1T9hwT7BhlEVPEhRmN+j\\\\\\\/ZIMB6nGVW8jCMbhsLi+UOQlOVKz5cZum+zl0PSVLZTdlSF7kGO\\\\\\\/p\\\\\\\/brZEQd+4UWF19dgsQVcalRJ0CSN+ooeYOlXurhOkfLGCzPMyDq28UM3FZVbN\\\\\\\/LVYcqZTfNnKMihmEh2cVk6IM04UjlJ6J5whLv8m2b2rGLBnE\\\\\\\/8zSWcQ1I\\\\\\\/\\\\\\\/gUIx5nUS\\\\\\\/b0QeVY+EJakE4dDdCA6ilLRcMnqTQHRaTrHeSVm8WpFRjs04yv\\\\\\\/AEUsQB62bxHQ15YM2UQtpl5wk+LN3K5MeCsZwqcLju40LSMJHRFqsh6GGHizwVbyLOHaPYm3Ek9F+4cHxwcKmNpEh0MNq7PUuc40SoRfvwTLZRjRoQISSFpDduIgXQAtLznk35aSJG3SbB6ptezYp45mzT9m\\\\\\\/aQHzRDY\\\\\\\/ziWsgu3kzT1nttWG44qJi7N7muqLee6YdZwoMZNVDWbmQO7O7VWx+EF\\\\\\\/m3b8NjCZJeCMVVtxgJVWBi1fBNuTXSLzPDUJWClgb4L2oun1Bvka+Hn1jfBntatrPJVxtviNqqqdJlmkvyFwfp8jNht4yfG8C7R9V2dAi5R8I9toT9QWW6t9uOb4aGPOWdzYcO1VPQjW\\\\\\\/eJe\\\\\\\/c4bGRY75e1SOAliFx49v4eBarRaC7zTU1O93Aehr97IsIECXqflNCDaasKSQCK1UspNw\\\\\\\/gFuDVTHCxVg7ILoK57Pr+lfgt6pKtvSPwenrvJXbxLqpu\\\\\\\/iWdujVQgsecFPhLkMm6tcJ2pMB329KAYwdvnjlvoVB9l\\\\\\\/XRVpdnCjYfg8\\\\\\\/OIX+UyTsWo\\\\\\\/HsmVKyH0en5eEphCBiZTYEqAn+2byZcGskAPapijTKk1evPY765VMx2GN9Ffz1GtN1BO0fKiEl5hxnjC1Dvv127H2BhblOPyXlSZq\\\\\\\/8XC66onHtzAAIKMzqg566Po6i1xGpq+CdZ1KCSX9enk6ZhLZMIbeHybrLuRwX9F\\\\\\\/Oq4fdFd6qXsdCV5SaNS7l2x6IGN2opt6Ikt+bF79h\\\\\\\/J5RSDgb7iT8fL07lKuPG0R3SzQdiDYluyFhFCOjR0VnggKtzzNmvjT+OSwGCRidPHTJLkPEN7SYz9wICMNw3WNQbffX2Be7uSxnMdrkcyJ0WmKxGD+HJP5B5iJtVAaE4WbMzDtCbJRMW28JM9vvwFFUCtxFtXNIyJEDm\\\\\\\/p1qpQwJ\\\\\\\/Pa3SovmoXr545PoPXUcs+hZ+zeH9gO3yu\\\\\\\/O24AKSMuemGKeE1tLpEAIM6q\\\\\\\/77ti+9qtDD4LSqjDwPfdQnGIfitMO9b6fPgQPJ3o8Mo2qIGRkYQ5MVWPLVN3Dn24DFDMDCadilCpVhGAJsCMg\\\\\\\/JqaC7EtYfGAqNVzigvJkFJSj3CRlL1vob39dXeWAtaXSCidGImhsQjJMNcaurQ1\\\\\\\/IYb\\\\\\\/meCgQFKZF1rM6Zij2kJl\\\\\\\/w+dUCiKNGHCAP7q4mh5CQwEy2ZPsOWn+pp9dt765M9Xm47yylNdXRM8eo9mJurl4PS5zS3kUXrq57mmFQpPXvcMnAXdcEwr6snBblH+v\\\\\\\/5vdHOHprsk++RG8r6Cpj5Ez0M8DPcAUWr97pzjiT\\\\\\\/hFm7oeENAW\\\\\\\/wfcJrS\\\\\\\/cQPUSwAX+LX2+MlOXwFDLIzapQelEbEzXCN\\\\\\\/7T2RaRpPexZrTlau+yZ5J54ruWYZnVFf6gqyLa8FXgNhzB9ERW5VkWmgl3\\\\\\\/fwqt2Q8nykOlBTJZJ+REpHOvfKEeKAL3qCRrT7PeHJO+zqnsbdKXDvsdlv7Zussz\\\\\\\/lI\\\\\\\/J3dR7hklZjjv1LP\\\\\\\/MxK1YWEoyBlasWsTf84gQTXDqONvxKdkOhIcUthhA9LFVyEcfuz8hjDt\\\\\\\/XU5lSYttu3vOWMUqM9RL8bZMnSLHw7jkm+Yd+K\\\\\\\/GnRMuCGxxb0ygypulMn6aKyB0vc3tXOxTFPLtcMSUbWW6CHNLCdrAXRf7mPI9vHpWvbdwfk9TEKEy68T5xRISm8LZFETJZR\\\\\\\/phjhkkd5MjP\\\\\\\/bPxnICh3MQQVgrySP19+qrjIN0fV8Z9dLMYDZo2AnGibXUV3qREoK24hcASQMMIQOYeK6zQtYL1DZfLTREHjHwsps+9NhmnnoGpHosTyd4HsyRo3bnQ3bK3EOfcmErLVhFq4avqN162vutw0hkQM00b3g\\\\\\\/\\\\\\\/a9x969pFIZWr0fchlGhMFZuOBcz6710nVBrhvJkmV4c5Ix+GO5Xi7NpCZfGKusJi8NLHWcyJhFV9yQqh6muMhpuFiCBF+8zpn4McEYMxsfYRbIZQ1yP1Jtt6x9OLCc84fYY9853pOYLmBjzsmOFcUGPvikq3W2sEfFDNvG0fIB1ThyKCLaCRZx6yG1XnIfxwVyyivnbBmO7aHgRx1Fb1y1OuJMucgSX8SAm3xqZg5b+poKYLZHscn5o26L3Jdqoznsv+75p02i+tnZsrYUf\\\\\\\/jGZ6bPhTRLgdd+UohoHnCpzZJ6hhVy1GJQGGtNE2PtgkUVhy5vDSRxUn\\\\\\\/A4FHjy3xNuucA1u48uqZRh1t3XSEYUwcLXDEYhL6Kgh2umSLLdRRE0H6tjKQPvx6MMFIRzrV0nXZ+eYGA4IHypSr7qHc1dxfC8kCScSr25wlSY3l5Cd9AeG1DXjhZwgDzrtoQ4RwmlQors+YKventsDkKuy2eseZD\\\\\\\/Cn2ssm1CoDpLBdvboRN+flDJZ55G3JxSZcWkyM0Mq5Xb2VqBPwKPs+ewJSeOCL87hCoLg3+PRJ5vWWrTBogsAsoUE\\\\\\\/zCGxS8xgv3P7C4Z3j6RJB3z9kyiVKQi9JdI05N2VAZbXaCj8H\\\\\\\/JONcXGReQM4\\\\\\\/6uuD\\\\\\\/T4MWrkcKQToQujlF2F692FyXlLegbex+ISthtkwqV7dXiemIs0+Wxj+gp6tIrDzki\\\\\\\/7coc9qp+Abgy64ZuSlx567cV0MFcqYPGVtGX0MlsXLmpPcjtNy\\\\\\\/tyR8IthJfXeV5cASAjAws5oLIH0G6EbptKiWEScF3Khqc3WDTmWZ6D9hjN+X4qbNx9vOi8qlOZh5phVg5xvZY2j\\\\\\\/Ra465cpj2AWHy1D3aqBWiwl7JaAUqi96Y8zIZ3xmyNsxAS8Ab+g0suu9ruUa0xTrEBl37ae1L\\\\\\\/moPEwfYtjvdc\\\\\\\/zSIeTmXt9kdJMKPm1OPhrhLxp4nJRrOZDCCPOlZDuf\\\\\\\/a9NLNeeONSZxZaPyQpOq272amUGQkwszf442S2tw+RVk2tp6whswZJdmMq3n4JLlz6wIkT\\\\\\\/vctmpRBXCBkUlP8eG6bFO1YrnNTr029eizmBk7niaohxyf931vq7sKu4vt64LBRWa9AAbBvfewrksLBLHmjxKwd+JrH00uBXia5lWzzP2r7BI7LQnFBbgody4hsE3uP2LbQaGTn2HFxANnt4TRsUYXkcr5aeDNKkfVya7sf4wZYSJTm0ZJbyqvczrNB+AMLH3n\\\\\\\/+CMbvNjd1HLJa0RqPbwRBraunoEmvKvwoLlDv4UF0kHsRzZy3cs7+3YeWSsVikIVUNtfyYHYATsReKrf3Hi3SnYfnErbcAr3W35hCG\\\\\\\/MaKWP9tzpVCnsajWh2pkufL1UgCie82Z8o1hwLa3RZF6G1gjVpqlrC2sSXSaK0ydyK\\\\\\\/w+POXHN0Fh7t7DG5UuRjnIc8P9YPuK33AWloAiNgj+XMyRckkp4Z2WtQyDz3JoWVta9TQyqfvgW9SM4Itg3STKQIynX+MwNP7sTVKpGqECeIF+67NCadsZgEf4pPlwoCYI0RD36dasseVKVF4hxisQD5PceN\\\\\\\/NYJV3uKksQmMg3nORSnVbrKw8V6rn\\\\\\\/95VbE7HBbv31kQriCs6I8d7RMKOWm4NDqFI+JcoiyeZVvbdSIzqao2cgdS0GjN7RuhTZIZW70AL\\\\\\\/FRkDdhK\\\\\\\/hehOObY8\\\\\\\/KQFxgk7ng+aoDXNyPLNHeq7Cdkdkrf7VJr\\\\\\\/hRmAa0bFm6xzWfdrkPxISWKNqsXA7nnIAbPlL9Ds7iEC7dyFmmaDshGQCnyaEjZAbxkI7QOym5+ZuCzrYVz5602B5jfYaufbaGO1DuwCpuZcZ1RHgmYAPVQuaS0v8ij3Tg6t6I6WM1DXLS2du8X8fMrhawAMW0xGVgqPFWhFK1PR4aFbcpbHmcaYThniJwEs6R\\\\\\\/WQ8vmMj1KljnH2xjVLM2yi5jHoeo1Y3am0S2io3M4EqA\\\\\\\/DiaYEqH0vuiO7RFnTUuK4WT38OBZT8LoIXn5CoV+lEC1Cv2IN18slx22nicIIj5L5wWChoA1E0MEGSoMzL6MCRBfHAkrFWCmJMzcjQmeZMAqk+F3siTqoLhcIsphBLIIGg+tS5lg9O4o0un4H9jhY\\\\\\\/7e1XJBsFVuufmGzGSPBkM+g5HuCTrKhR1V\\\\\\\/Y\\\\\\\/EDq+T0OZ2kYGiGTZihTlV84QqXbVt6DNUjRNDv1myOG0P1vrT42TueiXaf+MJaWrTgtWDtyf7nsDCV+mftLRzf9L03Y7x3fChz5QfZglD8BQM2UyLA3n0MuHMZeAVhm++GKqMySamM\\\\\\\/qnetVLegqDDm7KKKHbiCgjCdnHtEjzMakWxKx76V04UN32zVb9aFuJ3Sx6uDRyu\\\\\\\/KJvuQIWr+aGBwPvpqE9fywOxfeT+i8RDHe1p+WxevuuWF712enZje8mGKDUatjWUyONCEwxecpQDO9StMMudflld3gq8oP3p+jqmLidux9vp0W65iWII8n2yx2HCHhH4t2ldyWvmzTwlR1MtX097Gr8S3a3Xe0Kjy+fauhF66oFIZeFx\\\\\\\/wt1\\\\\\\/GqmWcik1L6aeaBRXYAStbC8KEzAceSK5lJE\\\\\\\/mhoG2MbHFcfcXj6xq9fzO15oOP1VmWMPXaHSaMsKxUS5RSGyk7dayItuvH++PDPpTEzcSfffIXGJ0cDJb92v9eBymdSL9q\\\\\\\/K1jleYjctVA8Cs0uY0i9j51Ih+N5pNZWGPvBA0dxbAgzjwcnrNQ+I+X2\\\\\\\/T54NMifc6OChXurWqteY+9wUB16RsIYFveZrEBSrKSDmciRr0HDjHYjPC9C4jfyQSa0LeUgmZUW5Bfw6lzVUh4xqx6WGkQ9N5T5Opf9ixN4z6bGtCbdIXKUgvYMGo\\\\\\\/JtnH8zXa5SKpSLL+bbAbeLGfWscYIx\\\\\\\/74ICZn5RKCSC8NIb2NZsDJGzPuJhUOiNYBHGIe+pbjIThFwOLYqK8ugsd80rrm5RU6G8i88p9s5lBzhtUpSshSPmVa1EnvKy9TJl6Zkhdfo7UWUhewWm7TbNNmBNPW+IN2kPQdEZj1q\\\\\\\/UZEkAowZ+EguC8PC56kB\\\\\\\/k61OSEdV1H7W4MY+xPTQmsDKYjvjInChkS29hbPtMRVzNn1X98+Ez\\\\\\\/ICZmbx65DSAWibAENOdwi+rEwldbFSYTiGFsXtYX3BFWVToFx8jZSKlQhQnYDwJgLSTjoAEfAhCkzNB41ZAwD5h+R6Lly7JFbgX9UdA5yg+S0yOXjcgZyytW1G2\\\\\\\/pVpRPhvXJmC2GYsGoG0Vwv6tYA7Lpfut1WJm6rfNhLPWYsxNOLnH1V9aZBzuBBbDS7TR1UsUiUEjVxRAPH5MKBEfyWpM0ipedUjfzG8QY5tzO1PL16TdGIRE3QMT0s1ToJw4+hNTQUzNeXR22Wkyj4p16TAf3QcTMKgWvwTIglon3Uht5n0Y++6bsTEmEj+2E3CYYDSpU3BgBpFX7iaS+HsAVQ9hIMpEAos5+qScd\\\\\\\/ulf5f\\\\\\\/BFK0jP8XysVPtE6g299lMDVHbtZG7KypCcmOY1PLsDZonKurxvZ1pDCZu4JFd5rvMtFDZ9tiISxLL3rU\\\\\\\/+XcFVAqTMZllecZ78LyNVqRC7QwvCJJdnWgEp7YoTedsAVnYK1RdHRoGge58R8f8dH\\\\\\\/VdA5O0Pu3vUisnExIeN9otrccYpDw7epFoqFr9oi82s7yKkxXbI5jON+mOkf9olXuPJJwa5bgf\\\\\\\/OLJn9woIRFVPwMvPA\\\\\\\/\\\\\\\/UQ4hJxyiPxqYWg6uAeA1DGt8W92pFv5n+i7nkODTXPWFKU+EW+hI4Xb7Nnz5LVIcgpu5HEXTnGg1ELskH4MJSeP3NpnyTfjwUG0rTNyrTqoq0CanoIWaBHtX0a445rCHEMulEIX44nZsmClHDW2DwxNM81il2vsgupIpChmvDSawUV4YF1euKt0BrFax6qLhI8uyOrZg6kpITU2S7cbgjAxNq9Ibg7IetONgNfm6jdtS4dcFS2WD0CkXmmPLMsH3vI1TvMBtDnTiMUZeyuEId2FxjUmITjP2AeobB6iJjdOHSyDyfiV\\\\\\\/wPzx6dtDwpDxissJg78VH6rV8iMCaC\\\\\\\/A+YqPYRUVaklu6omhA4WabV34bt91OBW1jrDzKpWwHqMEgk387n0jQunth4MDdHeFkSdkYSuXizbGCOr5EFWcqqVXWpVwg+BR0Nyf+xn01QPAHRd1GyC1fF4xsPFca4tB+SdM32uJjJv+HYTlBjKa3DXZQKui22N9E2xiIvBlKQvzzpbhnb2R5rRMvwnTFjK76hR9HT08GSSZszsdPOujQrc11hlAlFCgCzshq5yls4Q3VKk\\\\\\\/iJEpEgom3Pvxq6vcw7cnrf78jvRz51\\\\\\\/KNvwNIoCYofX\\\\\\\/yAIen0j217ezThZ\\\\\\\/0OHv4YRi1iY+1AdkIZSB76wsyVRT\\\\\\\/6M5OWWS7rkAo2TNm+p101AmYqSTKTkccDWMNY\\\\\\\/6RNDyAsXf26bpF2K3Pp2xFC1kSIHp8TOcbXgNOMaObcBqwUjabzd2w8LF4D\\\\\\\/s10V4oSdn4j8heCzPkr1Og8dCdIlzudKYYjRDYAdlihrkBfRO\\\\\\\/Opj8666iw4upzY7AoCyzZ6RyQzxQGqbE9z5pRIrMr8FlYNrsDwoceSy1CiBXxNO+PMN431X4E5mVy7ij9IJlXHCi3VkHZYAtv8gaJxV4npIpaks+UWIQ2rhqBJ\\\\\\\/Yvis4xWaoLCSJryQy7QasEKUVPUezV170EAKQT4+cIMAV7VBn6WhGTyjovFI8Kz2GvbFHP1FvByv49yb\\\\\\\/goWwQe+fFEhetW1SU0hy2qEwvi3KSxGE3ZlESSY4ndPqGUP9gjWhdRFHlFyig4viLU7mZpI25v1cP7pjisEQrTnmiYXfbq5wLgM0nZx2Y82vFg4UhazJF4fYFAcX0Hl2554UVhZwHh31mru\\\\\\\/s8QYHH5IjNwAMpfUjeI1rR3UuEoYdwMLKVQqgLUIKO7msqBS0U8eSfdYGQh7w1Yb6tXCwZkZ6L9NwH7dUsLVv0WGOPTSuUZbSQAM37YID+AFsbP+V6\\\\\\\/PbewDVwiAd+AMfSsshvTDO+M8Z2p6mDHOPvvBtpLhs+7kVjEttxgWYSpt6v0+HOZI7haHqOGm8QOKLAenxrgYf8ggp7BvH8hDmMajEcS4c1wKQsqZCVqEPhDTC1ENbQO4SQ0vz8HAgamtwlG8kOUWLhHyIWEXTtAunQk1L\\\\\\\/3AicDFPp0+YPfeW\\\\\\\/jDSlHYe9TLgcAPv7Fsf+5pLIAW7zUeTzzz2zVqB1UmqDtxinnBU4p7WhKhajzvHDCO8BrCjiFGaCLVlTudfUoICFVeVrgkNIkDkD8kZ8jNZoF1pA16TimZWCJuY+uW8ZYa\\\\\\\/gZGW2v3ba5cL+AfsUA61QXnTYJkYgWBULF61T6Rdm1C+Qf\\\\\\\/6rIR9n244zb1CVr1F\\\\\\\/XyxO4S6atqMTuGtscU1I2HPgjQB3kbh48AHm6PhMAOBTKkxAU\\\\\\\/zr2pmshOjhZPBS2NfWEloYcDQjRIsXHsYqiF7j4VR5UsJf5SMQcnCY0vC314Z\\\\\\\/DphpWCqB3KEJ9anQ2Ta9l8oqpWmQP2rQi7nKp6Y9q6MYfIEAhMxLMFhKmUX9W6sUZk002YAwqzBnrp3hSddiJ6Xvt3AKwR2+0cuLTDxQuQHBRQMVUi69VMPSaiLM+XflOPh2AFJLaKCIUXWEWsgxRzQ+tSlk0GN7x9hKXPObdmNRdxgUw377+Xv+yQolus2f67oQqfmixCSIUxuy+LBajUg07K3X5EM8Gdo9MvdaSwwShkYiI8Dz7fQJiVHAcKpUP4HCFKYcng21SVnEa+OOK1nRfoSqmJ2ZxMnQpaRHMsYXMyqPA79kZdSXCm0\\\\\\\/UXRMqX\\\\\\\/qEp7wyqjWLSyk4CgaWYDjtrVtmFPCUYwmVmjpWC\\\\\\\/ahLzvyWQMOXUvE3f2kd+xMie5m\\\\\\\/9DHkPgOeTgk\\\\\\\/AMwbhBXzpUufIa5KL0FGk31u4FU8bBIIc4wYSL1fDhG2hgqna3erabcUVe6slvAH5AvKs+o1vOMNrTmPl5OVleO2vAYXZyPF+pT3x4sbvxImIOovIgKeAYfdqttxxkvlyWEqAHz\\\\\\\/uBho7tidomUBE5qiWocptKk5Djt\\\\\\\/cB5miNRVoD6FAgE9OYz8z5MOe2LyavbjsMt4tbpttCTxQCfxlDw4oCLu5rNhAX2d\\\\\\\/4UESYyOD+EhWmO1ltIvGVGZE+2m6iQijv4Mf0+wYGddxJQan0GO9Qsue18hg6d3O6CzTZVdphAJhbGQQvJnqGTjDS+YF\\\\\\\/HKOaGC6U15uD4zfCB4B0ZeZ6E17c\\\\\\\/vZruAf25gqH+XHfaTduuVhKta9AvlIAudMWAdqHml5zUojWKmHm7jwnKiq49+7maxoBuJkkU4EBxxlrXIWMaWHRR5D8CfXszXqYKFqYrpLNl2jCGRl7AOzTSQDNlcd4nDYgpJZvZU579X75Ap6vCmE\\\\\\\/xoXQz9cFeKDMllJSRtkz7oXw6kUcy7d9\\\\\\\/nL3xTlKGGELcR6mNQSQ58WotYkTgxZJC5Vmy6NzOiGHtJ\\\\\\\/8TxNBxfS1PWjPjKQn8yUp26hMhDQyzPnmM9+\\\\\\\/OzjlN5lTMlYhLtQQUdznXL4z5Mpg\\\\\\\/sqBZW8kkmAEoaA5DHQInTNSARldxaShjGxLiQYvLItOMWn4i0AFyURrMBhYdYek8BWEqDw1CvW22y2N45sZuMVbHryDZxkOe3dIhI+E3yb5Qb6TXkzkWMqCTjSOASscOzUMtx4XZoTlJ1AWGdVRT6LTc4+GdlwQ4CfU1McgVmotAsNHlxOnsXcPeS+aZDgwrNTmY4DV4\\\\\\\/5mWPqqJcc9tmGSZCJh5rP2BuU5oI7g7EwbWoYbKAXodmxo5PPCSdj\\\\\\\/TtQN2KxlkJkzSo\\\\\\\/EYf6aB2NXpm6ot\\\\\\\/RMVtQGdn2D7p2WGPQRjdapifHGjBYuMebW1QmQk2+s82q3UADkSqLLJ1mMjL\\\\\\\/U+E+uj7xoDsGzkc\\\\\\\/T1OpweZ7mwAM8p\\\\\\\/Rk3WPZy4rGNxSgBFdk4QsL+R97MLcRPEljlm5fQvVSVNspaqSZ4DYE299M7BVlGijC7N+I6l\\\\\\\/opEnmuBTsvP0SuXHGwP4kMw7puHWCNg9Lrq5plZjbFGstEiIAqX7BkKTPhD+6zkvroTNQpKRQfvrYYRUOKzRGwRpa3NPYSMEUAarZaAEl\\\\\\\/OYLzJj5\\\\\\\/PgsIffpEB6ReMo3EH3bnMuiTeWcxRTI0UrMaTcFPEBvWMA5My1vrPBB6Y3U6f0jqNfnLyby59y0Uc9w6uA0fUv2xjCYKnxkSQi4NbxfpAjqIfa5JSOlZuWVnl1vvFxgB9WrurLNSIhO8o0x6gil5XWqJDXTXWByoktBK0hlJXPvpJEtikPqynwjR\\\\\\\/Yro7XVf86Nw4F9bIfHvmAFnitxVpGPPB3o06T7rw\\\\\\\/DkCF0Hpk2bxhqVLIpioZ9Dx3CipWMuohe1MAhCTeUcNSruaO5ndBsylYvsy+SPtiWR4WZVsFwuHcGFYXeDyVYTZ7Bxd2ic103ISCnFOEktCJ1Fx+N+Ykv2H+Qj3eWflgo50RJrA7uZw06J\\\\\\\/j8EYOsMSN1zp8SZ3WPZ40clGYKrZXzSp2NfdzarHptQtcFqbFioyuMssUiF5mLTCPCDeDdkhGyxxNadOxotehvbUYfXtL04BgwzLgSd4QR1tu3X3Hk9qU10xBOFMvbqd8LwYbDVQTarFT1hatVVrXOZ6tpne6GDJSpU8pIk9knC0kiuAMLtLXACvT86tK5cpxW5Y3fWXvkUV7EN0UwSJSWkn6l1Fats3\\\\\\\/gnA1Fv0ibMb6JzjKi8rRlBIVuPlYoONDX\\\\\\\/+tJRvMP\\\\\\\/wCwp4VS2dF1BekjlrGdAoUEOnRBOfZJ\\\\\\\/x+CzOdEDfL0tpRPsXKDITLFUCeRp84XAlhoFYYtsh2\\\\\\\/j1LJSY4QElDcsm7B1OWSTIahI9JEQgprjRz61dHz4820V6r2RUpc1FCIoDGett4P5NHaneK\\\\\\\/lMS82kLYadJGGxUd6JyIpB2\\\\\\\/CkzeQn1JpPZz8Go7G8utT1wIqrHvyvTHnx0x1HyR83V6hONJGEZIVOJskQ\\\\\\\/CT9RZucY+m3WPK3HHhA6ORRvnnlRu6S4lw+HuHnDK\\\\\\\/sQOhkVVCbA2\\\\\\\/To1McDgTvd5yL8mJoCZuo\\\\\\\/CDG\\\\\\\/GDjJy6UeUNu8Go14PRIi7mbS898C6P8bhNin\\\\\\\/tXLNEDIJUOsyg5XeSrofNJDWoUO8SLLPqmzkxWViMOHluq0NocKZBuOSeSc+jWA1TqEwZlxd3viCyQwQk+9RAlFHRHhz2eRePokmS\\\\\\\/+xPsWtEVrziAEYAYP1ku5S0aZ4hC\\\\\\\/odIhDApQaCrNBl6hvwYnB8lDEX9LFhx27IFwqeGDWiG8Q3BhXQblHXS1ZFlthH+CuIJavD+xvY9\\\\\\\/o0QBMX5c6zkL0qO6ymUqQjLXHsFOc9hraNs13x\\\\\\\/ewvA8fOJkaKmQiD4y1E+MnHzVzKvWRNZa+Lvd+nQNsEC0mlpcNBD+8fLoS1J1NW+rDilVKpCQDSDppCOXqFx5n5RDd6iev+ZpiqrNXlL4dyxYSA5v4ElKKycD\\\\\\\/mQOLjpsajLfD1bsuD\\\\\\\/jVlY4cxEBUgimOen8mHUhzGXWf4\\\\\\\/noicKPv6855VP2HR2t77jzn4Gl6JQkOJTS9ERljjF\\\\\\\/A8kEo0yx8c+tRimlpSM1GzQUxTQVNbY09mehHbMTwHXvYzZ6GnacDrlZnrowXRLlAtgnwuWv05LbgeJDqhsYQ6pWaiKiroQmPfCjslOwAiZm5EicvNUsh0SJwXyJGBt1XTqcL9eI8Yoo5AUmnoQLesv28auoHdQ5fu5NDax\\\\\\\/l3xLwXTuCMcBGXLEGf6GMr\\\\\\\/CKrXBjtQ\\\\\\\/C1VZqMo7lDxzE90QSQHfL9gakZU1nMQQxb15IRGd54iNrtdRgkIKa1gaqhH5Bob6DRLYZrPdqDzU7TzGZHeVzJ68wM+BKrxQgrGIcModeqyjJqk9ra1Jq57fNHTAP2l6mMInBK25EXnj17t68G6HvdgI9ILMGc\\\\\\\/uAgNaFKM7r8d6FDfYKWvVrHJ7\\\\\\\/WTWUHOU5cblGXRYB2IXBUwkAG33QffJWnhulR0Mg3F72uB1086ee\\\\\\\/ln9M5XcVhm1zNiXY5zg9+mRt3h3n4+kdIDLhFNHw76Gsp90\\\\\\\/qbvBp8Q+IjOSY+rncxMA48a7Tphwop+L0AEQTMSCpiEtaUXgjRZ2ejcBfUAUIiHucv60HWsbDTmXU3vT2CkEroYc8PpEaOm5RFYyFxFGT1hA86\\\\\\\/jubuL65SviH\\\\\\\/Kc7jFyFUw\\\\\\\/Xsaznyyx1jfhM5zvm5OeqX\\\\\\\/0EtRYl0Aew6WRqSUEcORCk7l15pcSpnABiPwe1Qb90PwYi0MH6qRVmtyR84IuTt4054X2K5pDqNbltKhr96hwBrXFHT2ngixxi7U8+i75B0hLpONmP55u1dHEKBtSahdlVBG\\\\\\\/stJiFj9RgW3uHHRYVncl19d3v+6rGHSwKewitK64AXChWry0RsTznmtEygdoWo1x4farZejt7JZz7R9E6fr24lkP6p0zKT2mqL+STsVWSlkAsOU\\\\\\\/5uHQHs15p3BQ5g\\\\\\\/slbSVKJR8W4pamDsVShljYqo0KmfEqSgfPasYbh8+GwFW7J6Gkkt9gW6nOvIR5AAMJC\\\\\\\/zUA9RQS4ceWzPqEJmNt7EU4gjzpHXduKh2AWoByeOu\\\\\\\/amo0QKSyF6Eez\\\\\\\/PfoBfdUSqvTkPrJdd1qYSJuAYJzPZ7WuiBZejOmk+dRvEsSLr91hyR7nuB3ujrxjLINDMgu0\\\\\\\/l4o89OEtijBdbqd1yZpNEG9Jb6JTVW1R7651mBn841d+eV80FLUtgrEkFQh4WK4VoiSKZmSg3mG+lMnI5dYSJY1yU\\\\\\\/9hWibBeFiwnY76BTBIwWnV2iDwfxJlCJqMQoGcXVQ0ih422QInplTuEZWYALgvZZnILwsbFIjwAz0YRxZXAxya6uVPt+mtIfG3T1biAkKY2Pps4uE3K9hJQDHERNQykx+\\\\\\\/k6pWWufaeobPZE3rO\\\\\\\/UjoI7JqHcIcm8mTmjTcZBZLXsJAfNou1vJM1uZ6ZXuc3fhvHWQLXwI2r9OXkWLtk13R69TlSKw92usWwN2pJ2vWrLSRX8wabK327iWS2wekt9WrrDLfHKVnduVXsF5UmzchtPXkQBVeYmcAbfoaxwQnoR12bDm8fa65tzUj7OoQmQLFz1Fc3CcB+9xyisrQt4lTMPQfkpyQJ2ul4yE\\\\\\\/EZ1yHf+DIrS5zACxo2aUlqbEKm9YCWLbbbR3Vwm5QEsSslIYfi65C3GRCM8\\\\\\\/Lalf8hM1v0rbAq13atRr8l9UvStegwHaehCsFrVRVJ1qHKUgGnguwCrlgAMOCImJjr7Ah53vFQuU2y9ZJj87ZtIV5swx0IDVistYtCYq+Mc1zAGbUGkUlKHIrWl4pWLfYm\\\\\\\/oGJ3JFNCpquwVwwSZuyL2Ql4KVlHCfYDoXvLyRHdBKkiJ8sixxv9VFAzSxI6GfdZB06sjS5XkWgTWqSSXMAGKnfdLySwubJYCMHwJQCqPwpMKiqew5Sy5jIKfbXL7WN\\\\\\\/ee2OHRzYo16mFnmx3Ap\\\\\\\/3E2kOV16Is5t7QCF5tS03vr9x1Sr7+Q1a1VbWjSZwt6FKyoeUC799MI4gAqP9DQQ+NMhamBRXkL1LFxqWLli17LQbuehSO1e4+6TvXVQmcOmf\\\\\\\/WzowZ7N8ITFEmkJfRd\\\\\\\/UUn5xH0+4Zd4UZMHfsag\\\\\\\/349yDjFsuIF\\\\\\\/aEdUyOKs6FKUmL8767ztCS9onf+TL4H9P4L9yUT7\\\\\\\/+ImWaNnIFXNExlqWG8I\\\\\\\/Je0afoInRBoQN6NLUKIh0W\\\\\\\/YGasaZLUA+NB0dPzg8+eM8peZxzc1RS9hpxs5Ov10n8lb+ggs\\\\\\\/CUSTVmqUhkpcVRtk+0V2502B8vPLrSeWJtctDGgRqQ0OcW0ll2GM\\\\\\\/DOTSxcT+VwhCfq6MYqSYxHJlQRH+BeAxcFhBurD6uv9GSLWcn7GDX1xLGCJ2J8uPDCr2bfnZOaFQt+cGhlCP0e8NgGC9FSdvFxbVjclxK9zgcDtTc8Ilptn829zy6kDGee7LcsY3tRXddg6zREXV\\\\\\\/8zqjltMbsoDs6jGCOXB6Ws+N55wbOBWeplE2rWQGOaNMjAFGCLOtCnOvT4dxm2Js\\\\\\\/oDNJmwXwrxs69cQMJJU4b4MJw1QLj0NrwjlHPy02Gol+jiordW3yeT31CB\\\\\\\/GeCdGs6CiUl\\\\\\\/bdGZ9bPinenYgkRnzKlh82HbV48YbEvZdHsrrxiNGxaNueyG0VPJ7g5QKyZNGVOYdrbWdlGTUpniZh5sQiVh2znNKQBwtcSsHS\\\\\\\/s2rg9X+cjtOqGpk81uxmlYHP5zFfuDe\\\\\\\/FNosTPwNiNaQ5HO\\\\\\\/dFUSES8uNGSTp7276mUGBS0o3zCzxW00w\\\\\\\/hGzR6HHsoYx5VYKQsmvuGxh6cBmg3NzbNIPH4FP2YEHCQ82nN1eytBsKcddXyeGogA0tw800kRPQJdUhWnD7OIBz7iYkSNeds9Q75zrcJFsvFGuTBSf68Svs0hVMkbn4wHgJExB8JLgi5rTle6cuZo2IWuh9chN3vY3If2iZE9jvWUh33Wrduxrd0Yx2LbFNBM4dqnWQDzTKQ7oxwMzTIaVnHj8cs5arxD\\\\\\\/x6DAEi\\\\\\\/7esaKCpy6nYdXG\\\\\\\/You85\\\\\\\/kqpjK1phlhj6GSBqkVBbIFn9ECTSc124Axy\\\\\\\/1SVtvI5qY9iqADP8H9olf18Zm8kq1VzCTiOj8TtJZhgSRZaR4F\\\\\\\/Yl8W8eVreYjGLsXf4ns1gxLocU5W3OtJdIcEpEACcWnXJZFdBVhPx1aH2IvPIWvhARJoze77\\\\\\\/pDkdalijARIPs0v4TvgIfJVjSyo0EFxVisih+g\\\\\\\/3ff0\\\\\\\/JEPtznS\\\\\\\/qzJXpcz\\\\\\\/oky+B8UffTJ\\\\\\\/KdCbO7cZdKpjDGspYR2bg0zZOhNqFttvq2sYXBi2HYeB301fAK1mCRs9S9NOs4+c1hf7JnWmQhwnFlBZPdkl2YrLq4wE5UD0iBZpAET9CXwEmrspF3GOPKPdpG2ttsrZs1\\\\\\\/uQ0GVkam+INxcdsIUMJGYVKS5mkFuVqve7Ob4dknMtg6MkEvyjs0Dw\\\\\\\/\\\\\\\/ubWvWHel0UVFBfHIixaaOA4TTTtdEBKv+1XQ12gOKQaDHa1ICp72Q34ZE\\\\\\\/XQVX5wrjaoiCZK7NHe+hSdgj1EgaOeMQmHi8+a6NXsr7GCTyPgCScNd8lTggPseU3xIPqhJcQZaylAxYrSuW1swRamDPvsBSEQw8WwWlOCsF2nGwxI+g5XHrikGvHBv411Ao2u3sU3YWvvfQq9i5BOjITQBVIaKyNZ0MBZZPH5s\\\\\\\/xF5FPTyhW+TMfupn5vS3FKf0IFQFn5ySEzR9KwfPcFxJVVcU0syrBLHqwfRNWaUcfTjVzNlaH6mfNBSgyf47Avklj5kzU6KGZcoFtK0DDa4nubWnKwkhkEzHNMjVCw9swPWqjmy5Kd0aue0xr9ns6GZH4iI8yzDeS9DA\\\\\\\/dmymFI1PjYzZp5UrZIvhXzWILaB2qeIWTUajNZAm4bgGezRJ36Hc9U5gkxMKB1iAq2dbmVGCN9tCZL5mnry1fYxKkq2WWxKGTmtE8mveBaY2Kz1M84bZYlILKdj5dhEOAo8ISbmUlIl0JAAt1g12Jpqlrl544K38JRdlwNGJzDuEgwZFpNQ9oktvLHxp7PQ\\\\\\\/WIbelvcZ\\\\\\\/JGfCoFZD4F8+wVh4hU0djLc\\\\\\\/D\\\\\\\/7DSE3Q5eCEXyvkeuKcfCVIl7N1Y\\\\\\\/hhfV0PZcVFJF1KpF\\\\\\\/ZycSeiYybyyvDiyrTJ6Tu2cWzkU0TuXFsd1fVBoH6oGn\\\\\\\/5gDNbN\\\\\\\/9zCpJJJrmnzOW2qxaOK4vA\\\\\\\/Lw6qPhc+48howwedPwCUsJOsG0KW\\\\\\\/ApyH1yAKaovq5qEWTYdYYZRDIoXjOMCSDedu953kVMbgZSvtGp4qDOtjEnURDTJwEQeT1mqNoShO1RhY1EcAGZvZrM0p\\\\\\\/oGha55uprkAmQgUAM9TXYRkWWq\\\\\\\/0r4SHh4hwywrFDGYmLlb2ouA379AT85coTZS9SHoGWynUztyAYQq4ls7yP2pcLW7EN3SIFpJ5ma56jCsN\\\\\\\/ImWnjYjPGB9JP7kVh2IC8j4yk6v0MxZWhcnzk21I6SzjepD2EW5fBFcGLUaTCikjmYmdgl6TZ\\\\\\\/3woDqocIkgIX88BEej9sbqnS8i2yV9q\\\\\\\/dxNVjllnA3CVgCE8Vu+8qfdoF9pbXOqswsMAe4F2A\\\\\\\/dfIxad5JBHfzcnCAtgHL7ithDFNIRQEmjQfsWpIOofmyIgIBw3EHGz+ZKwf+x6rH6XSSdNg9gg+y\\\\\\\/If6kwIk8mhQeDRf7pYC0gaJqLRsCqwpQ+c3kKTbIMk8qARbwp0hgb\\\\\\\/FSovGeCzfMtQkTbIENxvCYWLQFzA1qV+4OvAKJzpYQ\\\\\\\/xjKKnKxSWvKk7QrvatsRfkGbdrfRp9CIeNuOCNMOgj1G2lfoeLo\\\\\\\/Htf5ALRwl\\\\\\\/L+nQd8Bwd9oDEeAFWfcmw5W3eY0cLBagBhpVDwiexuy2OActH75s+Y4VlJIAJyEKNNI4YRRTDHG2BIuxEHayjMOBVF9y1UlqZ\\\\\\\/dPIac3xzMxHXUSYpDJvL1FmlSubanriKNw4rFWx1hzRM2EtCR0oQqIIfsu2O0heAyBMnNvvbluXODuWYo0QMyNWxMs0Fh\\\\\\\/1F61jFMy+5PwyY7R5gvWWQIGEz8YisB9JIaRpLf6VIgL0TEOLr5coG8H2zf8K6x6ENxKlhQyRlWT3o64FQe1zeutaDdLROy1YCxbPiudGpqjtegD0AAqVo1ZVWSeqOW7DTOKzMjS3e1vFt3mL9xHICsLakne7SZNiUYjboHHK2aeVBvHhSMXRBEn7U22LpHrFuepBeILPGb7xYgL1rQXkHUeW+ltbXoACNf5ycxU5QwpcVObpXUqYn6nUo3cfZCrVzRk4pK6BNTZbNneoa5wF\\\\\\\/IWYL3hzlNe\\\\\\\/M5PLwF9pHr+SCcQMQE8oCD2ul0KA0BQoVSH1m\\\\\\\/PMn8CmnO86lDw8W3GRnG5wGEVB+W03Ic4EB4J3IWKdy9TU\\\\\\\/9fQZ8WAKqGFaNg7iwXXp+WS0F+4zhjG\\\\\\\/LP4VoHKitTc2MYJ5XP\\\\\\\/0xmnmnDGwSvak2plO\\\\\\\/ybvvcKOcNFRGTgbRHVK2vk7NieT5rDutdwKkxcnUDMrq5HLN915PD5aXi9HwR\\\\\\\/2P1rAHwO0UVliyXhZK5UxWl4getYkrtsZHXJSBbd9cMFvkgDwzCKFAk7V\\\\\\\/QVALSMqil1JgwlebtgySvSuEEHHC7kjrESrR3SdP3hxKzXpYZFVZHyT\\\\\\\/5X0NOtBAon9py8AplGSvcaBIXibuFxQ+1NiZens9LWDYCXcr9zLcRffP+UjE8eYJpgtEzF60kInKQUvHUpCiLqxx\\\\\\\/kem5+ochfGHUVNAVNjyQXFy5hPWKqQwSGbYe3HnXPQyCuXu4Hf9OE+oAj4p+osGFoTBUsCAKPL7PB7oeknNsEuxPNwJeqJR1mmUt1G8BwXc8Ece4pJR6JypCD5ihU1qpyCNfn\\\\\\\/TZG8p0Svaqb0e0t4oCs6VfaLhuk+SrxrTlwiVZfv9oK78cNDvO0kvQ9QyQHNSJFkrRSudaQx80Hu5medk0E9bNgyQafX3drLF8kQtqaWjAAeNUF0+2DRLUpOmS\\\\\\\/2iqTG9B1GUzWz8Z92VQw1TqIKVBtZL3SmWA0tHZLM2Dpb8l8aLiqR5fCatpgwbRnkquazyd76wPgrtLTHRuZ8LkRn7619TtoGdIo0xcLFUJgDIEblvw8qtyD7ncylL3Fjp3YYplnCcs6QB++YQmnGyoSyFKfwQmXW2V7rfClz4OCpOFYr45fHHGKDhUeymJ6ypqopKdF+3ctVVmAPI9AVE8YPfMCCphtaKUKRlVEkiMhviv3Vn3+D0lTgeEA0OTVLeIypdiyWpwWnJ3C+R+qtFnM40yyVFfqAmDBwAikhFamcNVrXL1NCH7NLEVWy2XFXROkM48\\\\\\\/i9UvH72aYxTarJdOntwGQ4ilKwwHF4IllSbHNc8hlTjQiwEFp3dQLRncmCkKr8AviAfAidcGlkNEfJHBRsdtvDjHEAQEd3K0iKMxyZzwdjesCTRXS5f1r6J77WkShI5YdjEgdiLzH7Y7jxlusc+rfO4Qj++MydLZBxjzlF9XlNFqPKK9H152vF10zf7LXI4\\\\\\\/m9miedUrXK9TCzccHPrissE6yVrZscr35TLMLLjN5DZ1R\\\\\\\/jJ6cy9Stf+tjfBydeI8RBib4CiQxcQTYUQQFz4XkIJVdlSUTRKrL61Fb\\\\\\\/VCF\\\\\\\/TcPxwusY3zPA3dsbJxL2\\\\\\\/2C6mtWlUnmBdU8eb+7wHlAIyNg6HW9JESv8tO7Q\\\\\\\/h8ClWhFbb6BsAGocXsDbrYkM0qy5wuAFBDfBdv+NPMDPE9DzNmTfU7am8\\\\\\\/DBudCyLVo+j6jF95UlaIpx8IFT9Rs+nCRgYYicWSZBTGWUZn+7eIu35zF3MRNfvKHyx12m7jteFAiuKOwAyXWRz\\\\\\\/dLP3Kt8mWl79R\\\\\\\/knO1AfF7vBTTA6Erz3QlMf5MdXkNKM2Id6N+XE028dYLagAIeE794oJPSoaJXlhLpYsm4NUiVNSrG+UlOHLzbZyXDjVQQhk61Ds2e7CUOLogm+rrY3bwr5lhmKAYwhgCAvRA6OjxrgLNd6ehtqspnpMOreJL8U9mrvVgIxeINZ1yMlr22ZWtubOklSH6rO3Ix56ISuHyWA9cchEf4sGhHEguv8YuXuhHLTUT5zSa9xuHxmUn61s7GlG0jaG0aZpQi0lN82PVZ0IgDSdljk7UOj+fEcFdwxMvvYHh90mQl483HQ2PSGwDeL\\\\\\\/Xoh4p7V5YCY444npLRlGhDcLKjGWVHOEKJqLu0etSL2cKepLwR9F+c6L7Ed9imqqUcF8jDqz6gSbG4P24l8evsYhd38WHacMS4796VzN4wOc2JdT1Rw28NR4JrSIDCUGXqsuUjXyM2BFLs88dB+VeQhgHdNzyse6wwvXTSuweMgsRlbRpbp3s6+rLiy5Bag5EOljjDTJc\\\\\\\/ogWiqCJSUgUaZoJ1QHIVITFfw3qbxAdEbZ7lSqoKuQDFwA3vuSxIAOsP7NpHVqxPGvPrIvUTWyMn13BOQqNYfoAl8GakNUBXHKOpxynCAhVjSW3UJDbXPDSd3xYC2AebkvuCyI7nJPbqvVY0lTjzOfTGujQXgcvBbms9YG3Y2JLSFMXGGaqKAH4UXICyiHZHJ53493IG75S8E6K3iBAe6RULWc3VAl2Y4sos0+IsxKa6kL8EVJhzc6boEp1REg\\\\\\\/WH6k7t0rXTxRtxKjuwfRf0s9fPNzzhSVrG4K0JaAqxjoZwFDS6lQDQktVuN8dTTfBXbLAA7i3DaU9srHcV4KgE+iqemcVDV8wEJsCgmznKgm8n5pHFHh2yqx+oOfXt77RPHxmHe792BoDvAEQfTbN4e8h5iROpGuGrCo8HDRPWluxF\\\\\\\/gbnQ4SBOOP3V72BK2pIffrd5k5LTxljVfmYEqnNNH+HJuwLNQFAJ09D74kDTqreGV0b6cxn3Vjeo7TLaY7UeijqJdb9E995IRg\\\\\\\/iw7+htIY71a51Gc5BCrGL45KOKGSa\\\\\\\/\\\\\\\/nQwxqbz4Mj1\\\\\\\/oUuTdcpy9CCwsvWPAP\\\\\\\/UQKy9ZdgrkNmk+GbDog1P2AEVUudInGg9AUBDeqSDia\\\\\\\/F7PQMOvk0eNPrDpEQUnHHF6ovmDnf28ziUMmECt936EFSEJJblzn3\\\\\\\/eCSnWvEJEdwoMomnEOpQHjOCah25OdUqkm6t7qUyysoE9YHnNwX\\\\\\\/kngkZiXiRa4qgIFiOHo\\\\\\\/TWwRd\\\\\\\/Du8PJqPB9+9hBCsfejVnkcxR5UAK2Aengxc+FuwClvpLa5B6rvKI8cT7AUsWG6jhxjKxjusTGaUh7huhCpi+BweYrpP+wHDd+egP9\\\\\\\/dYl4lFJKvE6SaQ0j1capK4NtzEk5A2CTiwqIIZTVviVJH4LOmTMHU67zCp1N6KqbqBTH9IJwHRwMaTeMQrczJXSDyN3mS+u20jAC5ppPXO9ddxgxgrgU60NkX+Cj4deqqtGwGIAk63eAUD8Ku6OtditT1c84Mly0wf3Mk\\\\\\\/JRJNRfr8wMeryMakX\\\\\\\/ZlcrbElb9Htk0cXGPUdia6CLKbypxa5jNGyqLTvqBxplweNQt36gjMZdV74OIFZT8t6+iOE4u1sqqV3Vcs8JNKTbaDX\\\\\\\/ImeQMyOksQzHlnk5Jv93Gha0lfsm4a8gKQgr78YwWhypyjdM5kfLN8eCBApIbY6FKqQyYhlCKbUGMOHgQsdNy9HrIEa9sksGNf+e4\\\\\\\/0dOaEwJIPGOuBana6KZyUcFV2OiId0svmDdXUZHncoadAXAfCxgGqb1O44BUI8a+Wqfn5RDaWoy02LFMnsSqAH4E99U1PRj1tHbjBhp6kMYKxgzBIehV9\\\\\\\/AqIR0\\\\\\\/dqLzYoWyLGszD8umcthjHnF2sRw1zvf4kjgPorLtL\\\\\\\/Y+XrobQEOkR0mA00n3k+6KbZuZks6F8vBjmich3qrWv+N2dkH2hR\\\\\\\/HqIj1Nin+ItLYemHSyJsHwOMVtE3efdZkzA54+evjQLtwTDDEerp9tqHccYduLFstSCa\\\\\\\/lw1f8WQbhSPhieLb652KEyIj41QpAP8dwwBd\\\\\\\/GhgWGOV3mPm6OftOygaaV9cgbIqzcnlbDi9B0Qb9mSGkayJompOijQvcK\\\\\\\/1UiekiVffDujZaHmiKGClfrNbqmEAZmraV0o4XqfFfF2X4YAb8ZQ7AZfqVoi62n7bmTUuwVoxQNx1ux3iVOeQqG9OkXFFpuH73A1hu52GhqZMh0d\\\\\\\/OnijcD1lR9xtBfH67XCS0hb1ry\\\\\\\/Bg3QGTmUdeD1ZJvtd9l6inpeD\\\\\\\/a36Iuwf5FCaonqzJe\\\\\\\/KcxpAzy0aamvrrNlnb4vn7La6VvAL4BHywL1viSuS3U7kInjH+pjcGTf306LOGjvCYMDg8j8eBrJzERNqImhsFBy0kv53ncZLTkAgnTq6E5Iy6gqf\\\\\\\/\\\\\\\/Qvf5FytAooffbU8g3UoCDoFADtFOrmx7RHyMbJAC9SXxGCXvPfJLcCm1v5m7Tl3g\\\\\\\/h4aU61HryIu0IDxXzcInuIr7o2Fee\\\\\\\/P5tXB89Qnnaj49IMQzAQTvmhaecNibe1V9BsaztuOGEaiGEdQUENkgBEzTP\\\\\\\/\\\\\\\/yeoTOEm2iVa8mG4qQg4X0RIB0llWWwd\\\\\\\/1XcV6WgRlimHXx++JO6+U9q4P0P4mP4CwenFi9MhTKNckHNI+thRcaKYU1cuJkaz6WIcZa6ZwxM2rDIWDpQHb0qXiuIZk2B9SfLKJG70mGwaTE8F5DtAS950XEz90SOP+s7RJDdfHLR5deD8trti3Pgg9g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MtHqmk7YRkB2kVg22XoKQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002272efbb0a9bd558c9beda42f4fe14d05bc72e1552bb22e3394a75d114d3d5bcbb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pN4W8lB1zJY90TNbCIdqmcDDjLalN5fqKj+OCZvRU85hLI871wnguEaMGYYioFNMnNgypqAlymbauS26trOKtZAdFWIcvq4KU5xHjMW0i91wYKDbSdUYpS0LGQyDP\\\\\\\/dGLsQNn2CXnbSZ9sHX7msP39kk+9EjTKZ7kLz6V\\\\\\\/urUeY70rGnwwzd7PPP40hGadH++T\\\\\\\/IrC+YZLtRpI15J6dAuSZUF8b0mvv3hDbivltq7MPv8lSJ4tUDL5yioetqPS+b11PYYL8rOQL+jHGCEVLswt6dG8fmAKTpw9Pms5\\\\\\\/ByQAON3CLc9XWw+XU1GLXfU\\\\\\\/8v8zEIWaYOorIDTNFdKGSJKWef+E1Ke+ZptxrK8BjL2RTv146UJmKk81mBSJK2TmAFFUOyZ\\\\\\\/FvXS0HufVgBEiCQM6GFJSYwsncH1YfegCpNK3aRRL\\\\\\\/AzHwBdiIEgoc+LT4OSD1mg6kbd4CGAs9hjunnoPPqTTTY10Mm6NMr10NVaNz1L78sZhDVa81+pOOgp0MZsbJvjppRKVxUcdJepImOX\\\\\\\/ywJ8A4iZdtGi6giD\\\\\\\/6OSObu9PahygiMYrpISPYgNcDRd6AOhfbR5GEYSCox\\\\\\\/hZ29quAG+4oqy\\\\\\\/gLUTN6uNMPQ7Df5oqVL52kXiObDUHxDy4So5WENy8RrZwquGSeszEvMVsLQpJNM6mHgh\\\\\\\/vB3Kzb\\\\\\\/\\\\\\\/9RetWTJu4X6dvGYiDSVm9cVulMUQQCOby3cJNBX18rRws0s4y4owh37FIjXJEdLc2cV8U1rmIyw4ZFxYM7mQqXMm2hOpPI2H89AAYDiKUYptCFMktxsRFUHYnN66jX80W+2TK6mN1rIe3CZwJM8NjoKH82UUVpms+dXxKbTcaHync+wDGQN0Azedi2zbdwf4qfE5yuL6KP69PwDnvYNhoQvF3u8I4ede7XklORcLeO5SxtK95gxzH0eoqEy1Jcn+TX0lo2zxvzSvRXhNFC4B4rCLMjkBaVG1WQi0CTwHfpqZ9H8PoHhP04fJAZ3tcibXKC\\\\\\\/Koxx+vnPrCAoXHwHQw+HayZZmBcEzgGpwiokGG6N4ApTtLNb1xRaK+u6tC7cXXq8IPediG\\\\\\\/YUCyoAMTN2nlSEsw9lFjqrcvXKxJJhwC1+IjGxufIjmVYKPm8bA\\\\\\\/l0eNo3B80G2E5izhExadh3qPc7PAJc2j4S9DvdcaYNR1GP53457QWs\\\\\\\/\\\\\\\/k7aZe4BCw4bMNxBLL0fH2rJ5\\\\\\\/SDiejGO8Mwko5fQMRJW45YTnaxUDlldyHNoViu0w6fm3nK8byQDVANLWeIZPBquYQ9QQIMAIgtCm\\\\\\\/sZIyz2MDCv2BQTLejTCMzQ2nM5HpDJcwSHdHHdRQIZay1cxBPq2LnLITdCdbwqVeFrweLIBlEzlydFOgeE7YJ2xzUNKKL4FlegbdSz0UXafiuiOchRN1QT0QvxxzxBSfytr9CWfdNfrPI8B+h09jByqVu3LdYi9p5\\\\\\\/Ej7e2L8MUkPL5feHNI+3P0SClU5avias\\\\\\\/VwCH09wCooxvgPyuRwyzYWzwNQzEph8Ly5ilw67himBz8RZa\\\\\\\/zTK9BvNtRSZYM9tMRTzUoVLFGB+3WpnW00dn+tkKubu+CO+im7kZxmixOU7jHHWKjzQPzzrV0Ldlv\\\\\\\/OOn+6ckHbjait2L05ctaz9nGJNpx42g0D5GCa2d31fSQ5KQfNTGL7w5AA0CWG42L1rm6ZyACr4d8O\\\\\\\/MW6l0kBC5QI0jIuY\\\\\\\/wgib4ypRyZeZ+OFliDTVGTbCwAZdk9XisYrv5Lgxt7Sc6OROAd2ofpYoHmdixwJWmVjPYq8WIUmek7XUdcoEi90axcMnkXpXrJYq46\\\\\\\/MVzHpZ8ijBj\\\\\\\/Sgqy19n5OFuj6A4\\\\\\\/QtMArrVciAC8LwepiHqxoH9TEA0Jm6+fOBy3R8HpcfE8KI0zd6iClbHsn4dApwZBj3NfAUkj2tX6GC4JlwXVxuMn3WGAafiF0kajjdzTwm+K4oehYYosRyEce1piScMn06D0SqKZsXrSNIuevHVYSl32j9dDCC2tOUQIlIg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XFFnhlqZp4D1jmRpy01DcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223025da1356cd7aef0d6bad45c4b0b70bb22d6f61b9c1d05553812fc7079ac14b\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493468 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2107" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1U6PDPZO4Vtbfz7AMUtCboWdg0nL3Pb_DU1GTfeWh4M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2107" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MkzBaetPmpEv9AYfk5tZZCSE3xneMYmZnEJDYzJuC-Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2097" title="An Interscholastic Network To Generate LexA Enhancer Trap Lines in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2123" title="Optimizing Selection and Mating in Genomic Selection with a Look-Ahead Approach: An Operations Research Framework" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493468" id="node493468" data-pisa="ggg;9/7/2107" data-pisa-master="ggg;g3.119.400269" data-apath="/ggg/9/7/2107.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0665-6508" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter M. Bourke</span>, <span class="highwire-citation-author" data-delta="1">Christine A. Hackett</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4388-9244" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roeland E. Voorrips</span>, <span class="highwire-citation-author" data-delta="3">Richard G. F. Visser</span> and <span class="highwire-citation-author" data-delta="4">Chris Maliepaard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2107-2122;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400269 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493468" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Peter M. Bourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BBourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourke%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BM.%2BBourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0665-6508" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter M. Bourke</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.bourke@wur.nl" class="" data-icon-position="" data-hide-link-title="0">peter.bourke@wur.nl</a>
<a href="mailto:chris.maliepaard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">chris.maliepaard@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christine A. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biomathematics and Statistics Scotland, Invergowrie, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BA.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BA.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Roeland E. Voorrips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roeland%2BE.%2BVoorrips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voorrips%20RE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoeland%2BE.%2BVoorrips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4388-9244" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roeland E. Voorrips</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard G. F. Visser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BF.%2BVisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Visser%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BF.%2BVisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Maliepaard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BMaliepaard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maliepaard%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BMaliepaard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.bourke@wur.nl" class="" data-icon-position="" data-hide-link-title="0">peter.bourke@wur.nl</a>
<a href="mailto:chris.maliepaard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">chris.maliepaard@wur.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2107" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493468" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2107.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493468" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2107.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493468" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2107.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493468" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">New genotyping technologies, offering the possibility of high genetic resolution at low cost, have helped fuel a surge in interest in the genetic analysis of polyploid species. Nevertheless, autopolyploid species present extra challenges not encountered in diploids and allopolyploids, such as polysomic inheritance or double reduction. Here we investigate the power and precision of quantitative trait locus (QTL) analysis in outcrossing autopolyploids, comparing the results of a model that assumes random bivalent chromosomal pairing during meiosis to one that also allows for multivalents and double reduction. Through a series of simulation studies we found that marginal gains in QTL detection power are achieved using the double reduction model when multivalent pairing occurs. However, when exploring the effect of variable genotypic information across parental homologs, we found that both QTL detection power and precision require high and uniform genotypic information contents. This effect far outweighed considerations regarding bivalent or multivalent pairing (and double reduction) during meiosis. We propose that autopolyploid QTL studies be accompanied by both marker coverage information and per-homolog genotypic information coefficients (GIC). Application of these methods to an autotetraploid potato (<em>Solanum tuberosum</em> L.) mapping population confirmed our ability to locate and dissect QTL in highly heterozygous outcrossing autotetraploid populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quantitative-trait-locus-qtl-analysis" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus-qtl-analysis" rel="nofollow">Quantitative Trait Locus (QTL) analysis</a></li><li class="kwd"><a href="/keyword/autopolyploid" class="hw-term hw-article-keyword hw-article-keyword-autopolyploid" rel="nofollow">autopolyploid</a></li><li class="kwd"><a href="/keyword/double-reduction-0" class="hw-term hw-article-keyword hw-article-keyword-double-reduction" rel="nofollow">double reduction</a></li><li class="kwd"><a href="/keyword/bayesian-information-criterion-bic" class="hw-term hw-article-keyword hw-article-keyword-bayesian-information-criterion-bic" rel="nofollow">Bayesian Information Criterion (BIC)</a></li><li class="kwd"><a href="/keyword/genotypic-information-coefficient-gic" class="hw-term hw-article-keyword hw-article-keyword-genotypic-information-coefficient-gic" rel="nofollow">genotypic information coefficient (GIC)</a></li></ul><p id="p-3">Autopolyploid species, characterized by having more than two homologous copies of each chromosome, present a number of challenges to genetic research not present in diploids or allopolyploids (which are essentially already diploidized, genetically-speaking). They have therefore been somewhat left behind when it comes to tools and methods for their genetic analysis. Among these challenges are the complexity of modeling polysomic inheritance and the occurrence of double reduction (the phenomenon whereby a particular segment of a parental chromatid and its recombinant “sister” copy migrate to the same gamete, which can only occur if multivalent pairing structures are formed and maintained through meiosis 1 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haldane 1930</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mather 1935</a>)). However, we are now at a stage where the availability and low cost of genotyping tools (based on single nucleotide polymorphisms, or SNP markers) are making the analysis of autopolyploids not just feasible, but of practical importance to crop breeders, increasing the need for both the methods and the tools to conduct these analyses, as well as knowledge on how best to apply these methods.</p><p id="p-4">Breeders and researchers are often interested in knowing the genetic architecture of important traits, for example: 1. whether they are mono-, oligo- or polygenic; 2. where the quantitative trait loci (QTL) influencing the trait lie on the genome; 3. from which specific parental homologous chromosome (which we term “homolog”) the favorable alleles originate; 4. whether these alleles exhibit additive or dominant gene action; 5. whether they interact with alleles at other loci, or with the environment. Up to now, approaches such as QTL mapping in bi-parental populations or genome-wide association studies have been proposed, although not always addressing all of the above points. More recently, genomic selection has been advocated as a powerful method to increase genetic gain, without necessarily needing an understanding of the genetic architecture (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Slater <em>et al.</em> 2016</a>). For quantitative traits with hundreds or thousands of causative genes, this is likely to be more appropriate in breeding programs. However, for traits with a few major causative loci, QTL mapping remains a viable option that additionally offers the promise of both understanding the genetics underlying the trait (including the possibility of finding the underlying genes) while also facilitating selection for it through marker-assisted selection.</p><p id="p-5">QTL mapping in autopolyploids (particularly autotetraploids) has evolved in the last 20 years to keep pace with changes in genotyping technologies. Approaches have been developed for both co-dominant and dominant marker systems, and range from simplified models that only consider bivalent pairing to more complex models that also include double reduction. However, there has been almost no investigation into the applicability or advantages of different models. Despite this, it is often asserted that models that ignore double reduction are <em>a priori</em> inferior to those that include it (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Luo <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Luo <em>et al.</em> 2006</a>). More complete models of polysomic inheritance that include double reduction are often developed under the assumption of completely-informative marker systems (<em>e.g.</em>, (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Xie and Xu 2000</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Xu <em>et al.</em> 2013</a>)), thereby avoiding the statistical complexities imposed by partially-informative markers (such as dosage-scored SNP markers). In fact, it is quite telling that the only publically-available tools for QTL analysis in autopolyploids have adopted the simplifying assumption of random bivalent pairing (<em>e.g.</em>, TetraploidMap (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hackett and Luo 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hackett <em>et al.</em> 2007</a>) and TetraploidSNPMap (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hackett <em>et al.</em> 2017</a>)), whereas more complete QTL model descriptions remain unimplemented or unavailable.</p><p id="p-6">Recently, a method to reconstruct inheritance probabilities (or identity-by-descent (IBD) probabilities) under both bivalent and multivalent pairing models in autotetraploid F<sub>1</sub> populations was implemented in the software package TetraOrigin (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zheng <em>et al.</em> 2016</a>). IBD probabilities describe the relative likelihood of inheritance of all possible combinations of parental alleles at a locus, for all offspring in a mapping population (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). They replace marker genotypes by pooling information from neighboring markers together (following a Markov chain procedure), essentially transforming bi-allelic markers into probabilistic multi-allelic markers. They are therefore very powerful as a replacement for single marker genotypes in QTL analyses. Using polyploid mapping populations simulated by PedigreeSim (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Voorrips and Maliepaard 2012</a>) and the IBD probabilities generated using the TetraOrigin algorithm, we investigated QTL mapping in autotetraploids and autohexaploids, estimating QTL detection power and precision, the effect of double reduction and multivalent pairing (while comparing models that both ignore and include double reduction), the impact of population size, trait heritability and marker distribution, and the differences in QTL detection and diagnostic power (<em>i.e.</em>, correctly predicting the QTL position as well as the composition and phase of the QTL alleles) between simple or more complex QTL segregation types and different modes of action (additive or dominant). We also examined the well-studied traits of plant maturity (earliness) and flesh color in a bi-parental tetraploid potato population to further illustrate our findings, comparing the QTL locations to the physical positions of candidate genes underlying these loci. Although we restricted our simulations to autotetraploids and autohexaploids (the most commonly-encountered autopolyploid levels), double reduction is a phenomenon that can occur at all higher ploidies, and therefore we expect our results to be broadly applicable.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tracking the inheritance of parental alleles in an autotetraploid. A. There are 36 possible combinations of parental alleles at a locus in each tetraploid offspring from a bivalent-only model (&#x201C;noDR&#x201D;). When double reduction (&#x201C;DR&#x201D;) is also considered, 64 extra genotype classes arise (shown in green), leading to a total of 100 possible allelic configurations at each locus. B. Visualization of &#x201C;haplotypic&#x201D; IBD probabilities in an individual offspring for one chromosome, showing the inheritance of homologs 2 and 4 (maternal), and segments of homologs 5 &#x2013; 8 (paternal), with recombination break-points identified. Darker blue colors indicate a higher probability of inheritance. X-axis scale is in centiMorgans." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Tracking the inheritance of parental alleles in an autotetraploid. &lt;strong&gt;A.&lt;/strong&gt; There are 36 possible combinations of parental alleles at a locus in each tetraploid offspring from a bivalent-only model (&#x201C;noDR&#x201D;). When double reduction (&#x201C;DR&#x201D;) is also considered, 64 extra genotype classes arise (shown in green), leading to a total of 100 possible allelic configurations at each locus. &lt;strong&gt;B.&lt;/strong&gt; Visualization of &#x201C;haplotypic&#x201D; IBD probabilities in an individual offspring for one chromosome, showing the inheritance of homologs 2 and 4 (maternal), and segments of homologs 5 &#x2013; 8 (paternal), with recombination break-points identified. Darker blue colors indicate a higher probability of inheritance. X-axis scale is in centiMorgans.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F1.medium.gif" width="222" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2107/F1.medium.gif" width="222" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493604" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Tracking the inheritance of parental alleles in an autotetraploid. <strong>A.</strong> There are 36 possible combinations of parental alleles at a locus in each tetraploid offspring from a bivalent-only model (“noDR”). When double reduction (“DR”) is also considered, 64 extra genotype classes arise (shown in green), leading to a total of 100 possible allelic configurations at each locus. <strong>B.</strong> Visualization of “haplotypic” IBD probabilities in an individual offspring for one chromosome, showing the inheritance of homologs 2 and 4 (maternal), and segments of homologs 5 – 8 (paternal), with recombination break-points identified. Darker blue colors indicate a higher probability of inheritance. X-axis scale is in centiMorgans.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">One important aspect of QTL mapping that remains conspicuously absent from most published QTL studies in both diploid and polyploid species is the topic of information content. Originally it was noted that “marker information content” could adversely affect the estimated position of a QTL if markers of variable informativeness were located near a QTL (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Knott and Haley 1992</a>). Increasing information content was found to lead to an increased test statistic, which could bias the location of a QTL peak (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Knott <em>et al.</em> 1997</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Knott and Haley 1992</a>). Other authors have proposed alternative measures of information content than that of Knott and Haley, such as one based on Shannon’s information content (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Reyes-Valdés and Williams 2005</a>). In autopolyploid species the issue of information content is arguably even more important than in diploids, as information content generally varies between homologs. We prefer to use the term “genotypic information coefficient” (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Van Ooijen 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">van Ooijen 1992</a>) as it avoids confusion when dealing with IBD probabilities, which are multi-point estimates of homolog transmission probabilities and not the marker genotypes themselves. In this study we also extend the definition of the genotypic information coefficient (GIC) to autopolyploids and explore its usefulness in predicting QTL detection power and precision.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">Bi-parental tetraploid F<sub>1</sub> mapping populations were simulated based on previously-published genetic maps of tetraploid potato developed from dosage-scored SNP markers (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hackett <em>et al.</em> 2013</a>), while the hexaploid F<sub>1</sub> populations were based on a randomly-generated set of marker positions. In polyploids, marker dosages correspond to the allele counts of the “alternative” allele (as opposed to the “reference” allele) of a bi-allelic SNP marker. In heterozygous autotetraploids the possible dosages are 0, 1, 2, 3 and 4 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2.a</a>), with a marker being defined by its maternal and paternal dosages (<em>e.g.</em>, 1x0 means a dosage of 1 in parent 1 (the mother) and a dosage of 0 in parent 2 (the father)). There are nine “fundamental” marker segregation types to consider in an autotetraploid cross, namely 1x0, 0x1, 2x0, 0x2, 1x1, 1x3, 1x2, 2x1 and 2x2, rising to 19 in an autohexaploid, to which all other marker types can be converted without loss or distortion of linkage information (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2.b</a>) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bourke <em>et al.</em> 2016</a>). For convenience, we often refer to simplex x nulliplex or “SxN” markers to indicate both 1x0 or 0x1 markers (or indeed QTL). Similarly, duplex x nulliplex (DxN) implies either 2x0 or 0x2 (and SxS = 1x1).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker dosage considerations in an autotetraploid. A. In an autopolyploid there are ploidy + 1 distinct marker dosage classes at a bi-allelic marker, ranging from 0 copies to ploidy copies of the alternative allele. Here, dosage is assigned based on counts of the alternative (red) allele. B. Re-coding (&#x201C;converting&#x201D;) markers reduces the number of marker types to consider as well as helping visualizations of marker alleles by only showing segregating alleles. Here, four different possible simplex x nulliplex markers (1x0, 1x4, 3x0, 3x4) are converted to their most fundamental form (1x0)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker dosage considerations in an autotetraploid. &lt;strong&gt;A.&lt;/strong&gt; In an autopolyploid there are &lt;em&gt;ploidy&lt;/em&gt; + 1 distinct marker dosage classes at a bi-allelic marker, ranging from 0 copies to &lt;em&gt;ploidy&lt;/em&gt; copies of the alternative allele. Here, dosage is assigned based on counts of the alternative (red) allele. &lt;strong&gt;B.&lt;/strong&gt; Re-coding (&#x201C;converting&#x201D;) markers reduces the number of marker types to consider as well as helping visualizations of marker alleles by only showing segregating alleles. Here, four different possible simplex x nulliplex markers (1x0, 1x4, 3x0, 3x4) are converted to their most fundamental form (1x0).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F2.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2107/F2.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493662" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-10" class="first-child">Marker dosage considerations in an autotetraploid. <strong>A.</strong> In an autopolyploid there are <em>ploidy</em> + 1 distinct marker dosage classes at a bi-allelic marker, ranging from 0 copies to <em>ploidy</em> copies of the alternative allele. Here, dosage is assigned based on counts of the alternative (red) allele. <strong>B.</strong> Re-coding (“converting”) markers reduces the number of marker types to consider as well as helping visualizations of marker alleles by only showing segregating alleles. Here, four different possible simplex x nulliplex markers (1x0, 1x4, 3x0, 3x4) are converted to their most fundamental form (1x0).</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Potato chromosomes 1 and 12 of <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hackett <em>et al.</em> (2013)</a> displayed contrasting levels of marker coverage (Supplementary Figure S1) and were therefore used to study different aspects. We used potato chromosome 1, with its uneven marker distribution, for the investigations into the genotypic information coefficient (GIC), while chromosome 12 was used for the power analysis (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Unless otherwise stated, all statistical analyses and visualizations were performed using the R statistical computing environment version 3.3.2 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">R Core Team 2016</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493598/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493598/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Details of parameter levels used in the simulation study</span><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>QTL detection model</h3><p id="p-14">QTL analysis was performed using a weighted regression of the homolog main effects, weighted by the IBD genotype probabilities (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> 2014</a>). The QTL model for autotetraploids described by <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Hackett <em>et al.</em> (2013</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">2014)</a>, derived from the earlier work of Kempthorne (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kempthorne 1957</a>), can be written as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-1.gif"/></noscript></span></span>having taken the constraints <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-3.gif"/></noscript></span></span>into account (<em>i.e.</em>, we drop the terms <em>X<sub>1</sub></em> and <em>X<sub>5</sub></em> to remove collinearity between model terms). Here, <em>Y</em> corresponds to the trait values, <em>X<sub>i</sub></em> the indicator variables for the presence / absence of a particular parental homolog (1-4 for parent 1, 5-8 for parent 2), <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-4.gif"/></noscript></span></span> the intercept and ε the residual term. <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> (2014)</a> describe this as the “additive” model, and used the probabilities of the possible genotypes as weights in a regression using the above model form (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> 2014</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kempthorne 1957</a>). The corresponding model for an autohexaploid is:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-5.gif"/></noscript></span></span>again, having taken the constraints <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-7.gif"/></noscript></span></span> into account.</p><p id="p-15">The software package TetraOrigin (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zheng <em>et al.</em> 2016</a>) can determine IBD probabilities in autotetraploid populations under both bivalent and multivalent pairing models. We applied TetraOrigin (run on Mathematica version 10 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wolfram Research Inc. 2014</a>)) with input files derived from the integrated linkage maps and dosage output of PedigreeSim. Both bivalent_decoding options (False / True) were run to generate IBD probabilities under both a model that allowed for double reduction (DR) and one that did not (noDR), visualized in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1.a</a>. The other parameter settings used were parental dosage error probability (epsF) = 0, offspring dosage error probability (eps) = 0.001, and parental bivalentPhasing = True (<em>i.e.</em>, assuming purely bivalent pairing predominates to determine parental marker phase, for computational efficiency). The TetraOrigin algorithm is generalisable to all even ploidy levels (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Zheng <em>et al.</em> 2016</a>), although for simplicity we only ran the latter step of offspring IBD estimation for hexaploids using the parental marker phasing from the simulated linkage maps as input. The IBD probabilities at the marker positions were used to fit splines (using the smooth.spline function in R (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">R Core Team 2016</a>)) from which re-normalized probabilities were interpolated at a 1 cM grid of positions (using the predict function in R) for subsequent QTL analysis.</p><p id="p-16">We first applied this approach using the noDR IBD probabilities as weights (which we term the “no double reduction” (noDR) model, where all <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-8.gif"/></noscript></span></span> = 0 or 1). This is identical to the approach used by <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> (2014)</a> in their work on QTL analysis in tetraploid potato.</p><p id="p-17">However, the IBD probabilities generated by TetraOrigin <em>without</em> the constraint of bivalent pairing can also be used as weights in a similar fashion, although the indicator variable matrix <em>X</em> must be modified accordingly (Supplementary File S1). We termed this the “DR” model, which allows for the possibility of genotypes resulting from double reduction, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-9.gif"/></noscript></span></span> are no longer constrained to equal 0 and 1, but rather <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-10.gif"/></noscript></span></span> = 0, 1 or 2.</p><p id="p-18">The “logarithm of odds ratio” (LOD) score for the regression was calculated using the formula<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-11.gif"/></noscript></span></span>where <em>N</em> is the population size, <em>RSS<sub>0</sub></em> is the residual sum of squares under the null hypothesis of no QTL (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-12.gif"/></noscript></span></span> for trait values <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-13.gif"/></noscript></span></span> and overall trait mean <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-14.gif"/></noscript></span></span>), and <em>RSS<sub>1</sub></em> is the residual sum of squares from the regression model (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broman <em>et al.</em> 2003</a>). A chromosome-wide QTL scan was performed at 1 cM intervals and the LOD score recorded at each position.</p><p id="p-19">Significance thresholds were determined through permutation tests (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill and Doerge 1994</a>), with each of the 1000 simulated phenotype sets per parameter set (10 populations × 100 phenotypes) permuted once before recording the maximum LOD score from the chromosome-wide scan (<em>i.e.</em>, recording 1000 maxima). This generated approximate experiment-wise LOD thresholds by taking the 0.95 quantile of the sorted LOD values. A QTL was declared detected if the significance at the QTL position exceeded the significance threshold. Because the true positions of most QTL were not positioned exactly at the grid of 1 cM positions tested, approximate LOD scores were interpolated at the QTL positions (and used to derive QTL detection rates).</p></div><div id="sec-3" class="subsection"><h3>GIC study – potato chromosome 1</h3><p id="p-20">For each set of population parameters (all possible combinations of population size (Pop.) and rate of quadrivalent formation (<em>q</em>)) we simulated 10 separate populations using PedigreeSim and the phased linkage map of chromosome 1 from <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Hackett <em>et al.</em> (2013)</a> for the phased parental marker positions and dosages (visualized in Supplementary Figure S1, left-hand side). Each simulated individual carried a single chromosome. For each population, we generated 100 phenotype sets for all possible combinations of the factors heritability (<em>h<sup>2</sup></em>), QTL segregation type (QTL seg.) and QTL action (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The phenotype of the <em>i</em><sup>th</sup> individual (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-15.gif"/></noscript></span></span>) with QTL dosage <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-16.gif"/></noscript></span></span> was randomly sampled from a Normal distribution according to:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-17.gif"/></noscript></span></span>where μ = 10, Q = 1. The environmental variance <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-18.gif"/></noscript></span></span> was determined by first calculating the genotypic variance <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-19.gif"/></noscript></span></span> across the whole population given the individual QTL dosages (in the case of a dominant QTL these were taken as a dosage of 0 and 1 only). Offspring QTL genotypes were derived from the .hsa and .hsb output files of PedigreeSim (which provide the exact location and origin of recombination points along offspring homologs). Both the position (to 0.01 cM accuracy) and the configuration of the QTL (<em>i.e.</em>, from which parental homologs the various QTL alleles originate) were randomized for each phenotype set.</p><p id="p-21">The GIC values for homolog <em>j</em> at a particular locus were determined as follows:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-20.gif"/></noscript></span></span>using the noDR IBD probabilities, where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-21.gif"/></noscript></span></span> is the probability of inheriting homolog <em>j</em> in individual <em>n</em> at this locus (derived in Appendix 1). GIC values were calculated at all 1 cM splined positions used in the QTL scan. We considered the extension of the GIC to include the case of double reduction, but found a homolog-specific GIC was no longer easily defined when an offspring can inherit more than one copy of part of a particular homolog. Note that this definition of GIC is independent of ploidy level.</p><p id="p-22">To better understand the relative importance of GIC on the power of QTL detection, we applied a generalized linear model (GLM) (using a Binomial model with logit link) using the following model:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-22.gif"/></noscript></span></span>where <em>p</em> is the probability of detecting the QTL, and the explanatory variables are <em>q</em> (rate of multivalent formation), <em>Pop</em> (Population size), <em>h<sup>2</sup></em> (heritability), <em>QTLseg</em> (QTL segregation type), <em>QTLact</em> (mode of QTL action, either additive or dominant) and <em>GIC</em> (the product of per-homolog GIC values underlying the QTL alleles). We confirmed these results by also using the LOD score at the QTL position as the response variable (corrected for variable significance thresholds by subtracting the thresholds from the LOD scores first).</p><p id="p-23">To understand the influence of GIC on the detection of more complex QTL segregation types, we categorized the per-homolog GIC as either high (H) or low (L) using a threshold of both 0.9 and 0.95 for high GIC (so for example in the former, Low GIC &lt; 0.9 and High GIC ≥ 0.9). These cut-offs were empirically-chosen to divide the data in a relatively balanced manner between high and low GIC. A DxN or SxS QTL could then be categorized as either LL, LH, HL or HH, depending on the underlying GIC at each of the alleles with positive effect. For each parameter set we compared the power of detection of LL, LH / HL (since both have one low and one high-GIC allele they were grouped together) and HH QTL.</p><p id="p-24">Finally, we performed a small investigation of the effect of founder haplotype on the GIC measure using populations simulated from a founder haplotype pool of 5, 10 or 20 haplotypes, described in Supplementary File S2.</p></div><div id="sec-4" class="subsection"><h3>QTL power analysis – potato chromosome 12</h3><p id="p-25">We were primarily interested in understanding the factors that influence the power and precision of a QTL analysis in autopolyploid populations (with the influence of double reduction of particular interest). Here we define detection power as the rate at which a simulated QTL position has an associated LOD score that exceeds the LOD significance threshold. This is a more conservative definition of power than the alternative definition (which considers the rate of significant peaks detected). We also looked at whether simulated QTL fell within the LOD-1 and LOD-2 support intervals (the full range of positions with a LOD score whose difference to the maximum LOD is less than one, or two, respectively), which is yet another indication of QTL detection power. The precision of a QTL analysis was gauged in a number of ways – as the distance (in centiMorgans) between the QTL position and the peak LOD score, but also in how precisely the QTL configuration and mode of action was predicted.</p><p id="p-26">The simulations using chromosome 12 were similar to those of chromosome 1 with some differences (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Six different rates of quadrivalent formation were tested (<em>q</em> = 0, 0.1, ..., 0.5) and for each set of population-wise parameters, 50 separate populations were simulated. For each simulated population, 50 sets of phenotypes were generated as described above, except that the position of the QTL was confined to two positions, namely 14 cM (telomeric) and 49 cM (centromeric). The choice of these positions was not arbitrary: they were chosen to minimize the effect that differences in GIC might have on the results, while noting that QTL positioned at the telomere itself (0 cM) would be likely to suffer from lower detection rates due to lower information contents typically observed at the telomeric extremes. The centromeric position (49 cM) was selected for study as it is known that the rate of double reduction typically falls to zero at the centromeres (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bourke <em>et al.</em> 2015</a>).</p><p id="p-27">The QTL analysis and setting of significance thresholds was performed as described above (although permutation tests were now based on 2500 permutations, one of each phenotype set, with α = 0.05 as before). We also wished to investigate the rate at which the QTL segregation type and mode of action was correctly reconstructed. <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> (2014)</a> describe a QTL model-selection method by fitting the 36 phenotype means at the QTL peak and comparing the Bayesian Information Criterion (BIC) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schwarz 1978</a>) for SxN, DxN and SxS models.</p><p id="p-28">If the QTL model is fit using ordinary linear regression, the BIC is given by the formula:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-23.gif"/></noscript></span></span>where <em>p</em> is the number of parameters in the QTL model, <em>n</em> is the number of data points (either 36 or 100 for the noDR and DR models, respectively) and <em>RSS</em> is the sum of squared residuals from the linear model. In our approach, <em>p</em> = 2 in the case of additive or dominant QTL (Supplementary Table 1), while <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-24.gif"/></noscript></span></span> in the study of multi-allelic QTL, depending on the number of QTL alleles (Supplementary File S3).</p><p id="p-29">Whereas <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> (2014)</a> restricted their model search to three QTL segregation types, we expanded the search to include all possible bi-allelic QTL models (either additive or so-called “simplex” dominant, <em>i.e.</em>, a single dominant allele causes full trait expression (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rosyara <em>et al.</em> 2016</a>)), comprising in total 224 different models (listed in Supplementary Table S1).</p></div><div id="sec-5" class="subsection"><h3>Hexaploid study</h3><p id="p-30">Given the higher complexity of autohexaploid inheritance we restricted our study to a smaller parameter set (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), with 10 populations of population size 200 for each level of multivalent pairing (0, 0.5 and 1) simulated in PedigreeSim, and 10 traits simulated per population as described above. QTL were simulated to be bi-allelic, positioned randomly across the chromosome and with random phase configuration (in the case of dominant QTL, only segregating QTL types were considered). We tested both a low-heritability (h<sup>2</sup> = 0.1) and high-heritability (h<sup>2</sup> = 0.7) scenario. QTL detection power and precision were investigated using the methods already described.</p></div><div id="sec-6" class="subsection"><h3>Single-marker analysis</h3><p id="p-31">To provide a comparison between IBD-based and single-marker based approaches, we also ran a single-marker ANOVA using the raw marker dosage scores as the explanatory variable, <em>i.e.</em><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-25.gif"/></noscript></span></span>where <em>Y</em> is the vector of phenotypes, <em>D</em> the vector of marker dosages (missing values removed), μ the intercept and ε the residuals.</p></div><div id="sec-7" class="subsection"><h3>Multi-allelic QTL</h3><p id="p-32">A common simplification is to assume QTL are bi-allelic. However, it may be possible that multiple functional alleles exist at a QTL locus. We therefore also performed a separate simulation study to investigate the performance of our QTL mapping approach with multi-allelic QTL. For conciseness, this was performed in tetraploids only (Supplementary File S3).</p></div><div id="sec-8" class="subsection"><h3>Application to real data</h3><p id="p-33">The <em><span class="underline">A</span>ltus</em> x <em><span class="underline">C</span>olomba</em> (AxC) tetraploid potato mapping population (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bourke <em>et al.</em> 2016</a>) was used to explore both QTL models and test the methods described earlier for simulated data. The genetic positions of 6912 SNP markers from the SolSTW 20K SNP array (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Vos <em>et al.</em> 2015</a>) were taken from a previously-published high-density linkage map developed using this population (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bourke <em>et al.</em> 2016</a>). A subset of these markers was selected as input data for TetraOrigin (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Zheng <em>et al.</em> 2016</a>). Markers were selected so that each consecutive 0.5 cM window had (where possible) one marker of every segregation type (in a tetraploid there are nine (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bourke <em>et al.</em> 2016</a>)), selecting markers randomly among those with fewest missing values. TetraOrigin IBD probabilities computed under the assumption of no double reduction (noDR) or allowing for the possibility of double reduction (DR) were saved for later QTL analysis (after confirming that homolog numbering between the noDR and DR datasets was consistent).</p><p id="p-34">The two traits investigated were plant maturity and tuber flesh color. Both were scored on an ordinal scale, with maturity scored from 1 (very late) to 9 (very early) in increments of 1, and flesh color scored from 4 to 8 through varying shades (4 = white, 5 = cream, 6 = light yellow, 7 = yellow, 8 = dark yellow). Maturity was scored visually in the field during the growing seasons 2012, 2013 and 2014, with flesh color scored post-harvest for each of these years (three replicates). QTL analysis was performed as described in the previous sections, using splined IBD probabilities as weights in both a noDR and DR model for comparison purposes. Individual analyses per year were performed, as well as a general analysis using best linear unbiased estimates (BLUEs) generated using the lme function from the nlme R package (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pinheiro <em>et al.</em> 2017</a>) with Year as random effect and genotype as fixed effect. QTL were re-mapped by saturating the LOD-5 support intervals around the QTL peaks (no marker binning performed) and re-estimating IBD probabilities in TetraOrigin. QTL analysis was subsequently performed at the marker positions themselves (rather than at splined positions) to better estimate the peak positions. The location of the <em>CYCLING DOF FACTOR 1</em> (<em>StCDF1</em>) locus on chromosome 5 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kloosterman <em>et al.</em> 2013</a>) with gene annotation PGSC0003DMG400018408, and the <em>BETA-CAROTENE</em> <em>HYDROXYLASE 2</em> (<em>StChy2</em>) locus on chromosome 3 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wolters <em>et al.</em> 2010</a>) with gene annotation PGSC0003DMT400026363 from the potato genome sequence version 4.03 (SpudDB Genome Browser <em>S. tuberosum</em> group Phureja DM1-3 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hirsch <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Potato Genome Sequencing Consortium 2011</a>)) were used to compare the physical and genetic positions of the QTL peaks. The most likely QTL model at the peak positions was explored by selecting the QTL model that minimized the BIC among the bi-allelic and (simplex) dominant models listed in Supplementary Table S1.</p><p id="p-35">We also determined the average allele effect around the QTL positions as <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-26.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-27.gif"/></noscript></span></span> is the weighted average phenotypic score, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-28.gif"/></noscript></span></span> for IBD probabilities <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-29.gif"/></noscript></span></span> (using the noDR model) and phenotypic BLUE scores <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-30.gif"/></noscript></span></span> for individual <em>i</em>, and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-31.gif"/></noscript></span></span> is the overall population mean phenotypic scores (<em>i.e.</em>, overall mean of the BLUEs).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-36">Supplemental material available at Figshare: <a href="https://figshare.com/s/a97475f5802e3354f44e">https://figshare.com/s/a97475f5802e3354f44e</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Effect of GIC on QTL analyses</h3><p id="p-37">As expected, there was a clear relationship between the GIC per homolog and the marker coverage of that particular homolog (an example is given in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3.a</a>). Differences between coupling and repulsion marker information can be discerned, for example where a single 1x0 (SxN) marker tagging homolog 4 in parent 1 at 28.5 cM gave a large boost to the otherwise low GIC values in that region on homolog 4, but also slightly increased the GIC values on homologs 1, 2 and 3 (there is no information about the meiosis of parent 2 from such a marker). The GIC accounted for much of the variability in QTL detection power, although not as much as the population size or trait heritability (Supplementary Files S5 &amp; S6).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effect of variable Genotypic Information Coefficient (GIC) explored. A. Influence of marker distribution on GIC values (calculated using noDR IBD probabilities) on potato chromosome 1 (Hackett et al. 2013), with parent 1 values in the lower panel and parent 2 values in the upper panel. Average GIC values over the 10 simulated populations are shown as lines above the marker distribution (and carry the same color). On the y-axis, h1 &#x2013; h4 = parent 1 homologs 1 to 4, and h5 &#x2013; h8 = parent 2 homologs 5 to 8. Occasionally GIC lines overlap, obscuring each other, although their profile logically follows the marker allele distribution shown. B. Effect of variable GIC on the precision of QTL detection. True QTL positions per homolog are represented by gray dots (each from a separate analysis &#x2013; we did not simulate scenarios of more than one QTL per chromosome), with arrows indicating the position of the discovered QTL peak. Arrows are colored by the GIC content at the QTL position itself, with average GIC lines from Fig. 3.a shown above the arrows. The example shown corresponds to SxN additive QTL with a population size of 400, heritability of 0.2 and multivalent rate q = 0 analyzed using the noDR model. In this figure, all simulated QTL were detected (full power). C. Effect of variable GIC on the power of QTL detection, visualized on a per-homolog basis. Here, the power in a 10 cM sliding window is shown by a heat-map track below each homolog. QTL positions are shown as gray dots, with arrows indicating the position of the discovered QTL peaks. In contrast to (b), there was not full detection power (population size 200 and heritability = 0.1) &#x2013; hence variation in QTL detection power along each homolog is apparent, and corresponds quite well with variations in the estimated GIC per homolog, shown above the arrows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The effect of variable Genotypic Information Coefficient (GIC) explored. &lt;strong&gt;A.&lt;/strong&gt; Influence of marker distribution on GIC values (calculated using noDR IBD probabilities) on potato chromosome 1 (Hackett &lt;em&gt;et al.&lt;/em&gt; 2013), with parent 1 values in the lower panel and parent 2 values in the upper panel. Average GIC values over the 10 simulated populations are shown as lines above the marker distribution (and carry the same color). On the y-axis, h1 &#x2013; h4 = parent 1 homologs 1 to 4, and h5 &#x2013; h8 = parent 2 homologs 5 to 8. Occasionally GIC lines overlap, obscuring each other, although their profile logically follows the marker allele distribution shown. &lt;strong&gt;B.&lt;/strong&gt; Effect of variable GIC on the precision of QTL detection. True QTL positions per homolog are represented by gray dots (each from a separate analysis &#x2013; we did not simulate scenarios of more than one QTL per chromosome), with arrows indicating the position of the discovered QTL peak. Arrows are colored by the GIC content at the QTL position itself, with average GIC lines from Fig. 3.a shown above the arrows. The example shown corresponds to SxN additive QTL with a population size of 400, heritability of 0.2 and multivalent rate &lt;em&gt;q&lt;/em&gt; = 0 analyzed using the noDR model. In this figure, all simulated QTL were detected (full power). &lt;strong&gt;C.&lt;/strong&gt; Effect of variable GIC on the power of QTL detection, visualized on a per-homolog basis. Here, the power in a 10 cM sliding window is shown by a heat-map track below each homolog. QTL positions are shown as gray dots, with arrows indicating the position of the discovered QTL peaks. In contrast to (b), there was not full detection power (population size 200 and heritability = 0.1) &#x2013; hence variation in QTL detection power along each homolog is apparent, and corresponds quite well with variations in the estimated GIC per homolog, shown above the arrows.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F3.medium.gif" width="263" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2107/F3.medium.gif" width="263" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493492" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">The effect of variable Genotypic Information Coefficient (GIC) explored. <strong>A.</strong> Influence of marker distribution on GIC values (calculated using noDR IBD probabilities) on potato chromosome 1 (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Hackett <em>et al.</em> 2013</a>), with parent 1 values in the lower panel and parent 2 values in the upper panel. Average GIC values over the 10 simulated populations are shown as lines above the marker distribution (and carry the same color). On the y-axis, h1 – h4 = parent 1 homologs 1 to 4, and h5 – h8 = parent 2 homologs 5 to 8. Occasionally GIC lines overlap, obscuring each other, although their profile logically follows the marker allele distribution shown. <strong>B.</strong> Effect of variable GIC on the precision of QTL detection. True QTL positions per homolog are represented by gray dots (each from a separate analysis – we did not simulate scenarios of more than one QTL per chromosome), with arrows indicating the position of the discovered QTL peak. Arrows are colored by the GIC content at the QTL position itself, with average GIC lines from <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3.a</a> shown above the arrows. The example shown corresponds to SxN additive QTL with a population size of 400, heritability of 0.2 and multivalent rate <em>q</em> = 0 analyzed using the noDR model. In this figure, all simulated QTL were detected (full power). <strong>C.</strong> Effect of variable GIC on the power of QTL detection, visualized on a per-homolog basis. Here, the power in a 10 cM sliding window is shown by a heat-map track below each homolog. QTL positions are shown as gray dots, with arrows indicating the position of the discovered QTL peaks. In contrast to (b), there was not full detection power (population size 200 and heritability = 0.1) – hence variation in QTL detection power along each homolog is apparent, and corresponds quite well with variations in the estimated GIC per homolog, shown above the arrows.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Apart from the influence of GIC on detection power, we were also interested in understanding the influence of GIC on the accuracy of QTL analysis. For this we examined more closely the position of QTL peaks in relation to their true position, for SxN QTL only (since these originate from a single homolog and are simpler to track). We noted a dramatic influence of GIC on the QTL peak position in regions of variable GIC, even in situations with 100% detection power (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3.b</a>). Local maxima in GIC such as that observed in parent 1 homolog 4 at 28.5 cM serve as local “attractors” for QTL peaks, an effect seen across all homologs. Generally-speaking, there is a tapering of GIC profiles at the telomeres, a consequence of poorer marker information (coming from one side only). Where GIC is high, the true position and detected QTL peak closely corresponded (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3.b</a>, 40 – 100 cM region). A visualization of the homolog-specific variation in QTL detection power is given in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3.c</a>.</p><p id="p-40">For more complex QTL types such as DxN or SxS QTL, we were curious to know whether the presence of a single QTL allele on a homolog with high GIC would be enough to detect that QTL, or whether high GIC was needed on both homologs. As described in the Methods section, we categorized QTL as either LL, LH, HL or HH depending on the per-homolog GIC underlying the QTL alleles with positive effect. As can be seen in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, the detection power of LL-type QTL tended to be lower than that of LH- or HL-type QTL, which themselves tended to be detected less often than HH-type QTL. In fact, the intermediate class (having only one positive QTL allele residing on a high-GIC homolog) was detected at approximately the midpoint of the LL-type and HH-type detection rates (Supplementary Figure S2).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493602/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493602/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Power of detection of complex DxN or SxS QTL, categorized by the GIC on the homologs carrying the positive QTL alleles</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">Finally, we found that the GIC was influenced by the haplotypic diversity within the mapping population, with more diverse populations leading to on-average higher GIC values than in populations derived from more genetically-homogeneous material (Supplementary File S2). However, lower haplotype diversity also contributed to greater levels of homozygosity and fewer segregating markers, meaning fewer markers were available for IBD calculation in these populations.</p></div><div id="sec-12" class="subsection"><h3>Power to detect QTL</h3><p id="p-44">As noted in the previous section, population size and trait heritability were found to have the most impact on QTL detection power, followed by GIC. On potato chromosome 12 we deliberately chose two QTL locations to minimize the impact of GIC and allow a comparison of centromeric <em>vs.</em> telomeric effects (with average cross-homolog GICs of 0.95 and 0.98 for 14 and 49 cM respectively).</p><p id="p-45">The four most important factors in determining QTL detection power (excluding GIC) were population size, trait heritability, QTL segregation type and QTL mode of action (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). When we ran a GLM using all the available explanatory variables we found that neither the rate of multivalent formation (<em>q</em>) nor the form of the model used (DR or noDR) had any real impact on the QTL detection power overall (Supplementary File S6). However, there were some instances where the DR model could improve detection power. For example, when the population size or trait heritability is low and rate of multivalent pairing is high, the DR model does offer an advantage (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5.a</a>), helping to maintain the same level of power as that achieved when there is strictly bivalent pairing (<em>q</em> = 0). Regarding the precision of the QTL analysis, the average distance from the QTL peak to the true QTL position is also adversely affected by double reduction (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5.b</a>). However in this instance, no matter what model is used, the QTL analysis will become slightly less precise at higher values of <em>q</em> (although there is some mitigation of the loss of accuracy when the DR model is used). Here we used distance as an absolute measure – the direction of this distance appeared to be biased toward the side of the QTL with greater genetic length (Supplementary Figure S3).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of QTL detection power, grouped by main explanatory effects. Here we show the distribution of QTL detection powers among the 576 estimated QTL detection powers we determined (c.f. Table 1, chm. 12 power study for the set of experimental parameters). Most scenarios had a very high detection power (&gt; 0.95), although powers as low as 0.35 were also observed. Additive SxN QTL with a mapping population of 400 and heritability of 0.2 are likely to always be detected, whereas a dominant DxN QTL with a mapping population of 200 and heritability of 0.1 is unlikely to be detected more than 50% of the time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of QTL detection power, grouped by main explanatory effects. Here we show the distribution of QTL detection powers among the 576 estimated QTL detection powers we determined (&lt;em&gt;c.f&lt;/em&gt;. Table 1, chm. 12 power study for the set of experimental parameters). Most scenarios had a very high detection power (&gt; 0.95), although powers as low as 0.35 were also observed. Additive SxN QTL with a mapping population of 400 and heritability of 0.2 are likely to always be detected, whereas a dominant DxN QTL with a mapping population of 200 and heritability of 0.1 is unlikely to be detected more than 50% of the time.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F4.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2107/F4.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493561" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Distribution of QTL detection power, grouped by main explanatory effects. Here we show the distribution of QTL detection powers among the 576 estimated QTL detection powers we determined (<em>c.f</em>. <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, chm. 12 power study for the set of experimental parameters). Most scenarios had a very high detection power (&gt; 0.95), although powers as low as 0.35 were also observed. Additive SxN QTL with a mapping population of 400 and heritability of 0.2 are likely to always be detected, whereas a dominant DxN QTL with a mapping population of 200 and heritability of 0.1 is unlikely to be detected more than 50% of the time.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of various experimental parameters on QTL power and precision. A. QTL detection power for population size 200 and 400 compared over a range of different rates of multivalent formation (q). Solid lines indicate the results using the random bivalent noDR model, with dashed lines indicating the results using the DR model (including double reduction). Results over both trait heritabilities are combined here - the comparison between lower and higher trait heritabilities showed the same trend. B. Effect of double reduction on QTL precision, depicted by the average distance between QTL peak and true QTL position across all experimental scenarios. C. Comparison between detection power for a centromeric QTL (left) and a telomeric QTL (right). D. Rate at which QTL fall within LOD-2 and LOD-1 intervals. E. Specific example of the difference in power between the DR and noDR models for the case of an additive SxN QTL with population size 200 and trait heritability of 0.1, up to 5% higher power on average when the rate of quadrivalent pairing was high. Standard errors of the means () are depicted by error bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Effect of various experimental parameters on QTL power and precision. &lt;strong&gt;A.&lt;/strong&gt; QTL detection power for population size 200 and 400 compared over a range of different rates of multivalent formation (&lt;em&gt;q&lt;/em&gt;). Solid lines indicate the results using the random bivalent noDR model, with dashed lines indicating the results using the DR model (including double reduction). Results over both trait heritabilities are combined here - the comparison between lower and higher trait heritabilities showed the same trend. &lt;strong&gt;B.&lt;/strong&gt; Effect of double reduction on QTL precision, depicted by the average distance between QTL peak and true QTL position across all experimental scenarios. &lt;strong&gt;C.&lt;/strong&gt; Comparison between detection power for a centromeric QTL (left) and a telomeric QTL (right). &lt;strong&gt;D.&lt;/strong&gt; Rate at which QTL fall within LOD-2 and LOD-1 intervals. &lt;strong&gt;E.&lt;/strong&gt; Specific example of the difference in power between the DR and noDR models for the case of an additive SxN QTL with population size 200 and trait heritability of 0.1, up to 5% higher power on average when the rate of quadrivalent pairing was high. Standard errors of the means (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-32&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) are depicted by error bars.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F5.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2107/F5.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-47" class="first-child">Effect of various experimental parameters on QTL power and precision. <strong>A.</strong> QTL detection power for population size 200 and 400 compared over a range of different rates of multivalent formation (<em>q</em>). Solid lines indicate the results using the random bivalent noDR model, with dashed lines indicating the results using the DR model (including double reduction). Results over both trait heritabilities are combined here - the comparison between lower and higher trait heritabilities showed the same trend. <strong>B.</strong> Effect of double reduction on QTL precision, depicted by the average distance between QTL peak and true QTL position across all experimental scenarios. <strong>C.</strong> Comparison between detection power for a centromeric QTL (left) and a telomeric QTL (right). <strong>D.</strong> Rate at which QTL fall within LOD-2 and LOD-1 intervals. <strong>E.</strong> Specific example of the difference in power between the DR and noDR models for the case of an additive SxN QTL with population size 200 and trait heritability of 0.1, up to 5% higher power on average when the rate of quadrivalent pairing was high. Standard errors of the means (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F5/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F5/embed/mml-math-32.gif"/></noscript></span></span>) are depicted by error bars.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">As expected, there is essentially no difference between the two models for centromeric QTL, but differences do appear for telomeric QTL (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5.c</a>). If we consider the rate at which QTL were present in the LOD-1 and LOD-2 intervals instead, we see a very sharp decline in the performance of the LOD-1 interval at higher levels of multivalent formation if the noDR model is used (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5.d</a>). However, it is questionable whether the LOD-1 interval should be used at all – even in the case of purely bivalent pairing, on average 16% of these support intervals contain no QTL – a value which increases to almost 32% at the lower rates of population size and heritability. The widths of the support intervals around QTL peaks were also found to increase as the levels of multivalent formation increased, an undesirable effect that cannot be mitigated by using the DR model (Supplementary Figure S4).</p><p id="p-49">For dominant QTL, we found the detection power of simplex dominant QTL was somewhat affected by the presence of quadrivalents, but the detection of duplex dominant QTL (where two copies of the allele are required for complete trait expression) was severely compromized when quadrivalents were present (Supplementary File S3). If we remove the major sources of variation from the data by considering only an additive SxN QTL with population size 200 and heritability of 0.1, we find that the DR model has the potential to increase detection power by up to 5% when the rate of multivalent formation is high (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5.e</a>).</p></div><div id="sec-13" class="subsection"><h3>Accuracy in predicting the QTL configuration and mode of action</h3><p id="p-50">Apart from the ability to detect QTL, we were also interested in investigating methods to correctly predict the QTL configuration (<em>i.e.</em>, predicting from which parental homologs the QTL alleles originate (also called the QTL segregation type or QTL phase)), and what the most likely mode of action is (additive or dominant). We followed the procedure described in <a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> (2014)</a> for this, using the Bayesian information criterion (BIC) to compare different bi-allelic QTL models. All possible bi-allelic QTL models were tested, as listed in Supplementary Table S1. In most cases, the minimum BIC corresponded to the correct QTL configuration and mode of action, with near-perfect accuracy for all QTL types in higher-power experimental designs (N = 400 and h<sup>2</sup> = 0.2).</p></div><div id="sec-14" class="subsection"><h3>Hexaploid analysis</h3><p id="p-51">The TetraOrigin algorithm (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Zheng <em>et al.</em> 2016</a>) was also implemented for autohexaploids, but due to excessive memory demands we were unable to run a fully general analysis. We therefore only considered multivalent pairing in one or other parent, but not both. In hexaploids, multivalent formation is a poorly-understood phenomenon. We took the most general approach of considering hexavalent formation, which also accounts for genotypes generated by a quadrivalent + bivalent pairing. We found that the presence of multivalents caused a slight reduction in detection power and precision (here, looking at the width of the LOD-2 support interval) which could be mitigated to some extent by using the DR model (Supplementary Figure S6.a). At the higher trait heritability (h<sup>2</sup> = 0.7) we found that QTL presence in LOD-2 intervals actually dropped, although this was a result of much sharper QTL peaks and hence much narrower intervals (∼2 cM <em>vs.</em> ∼45 cM). Prediction accuracies of QTL phase and mode of action barely exceeded 10% at low trait heritability (h<sup>2</sup> = 0.1). while at higher heritability, QTL phase prediction was near-perfect.</p></div><div id="sec-15" class="subsection"><h3>Single-marker analyses</h3><p id="p-52">We also analyzed the hexaploid and (a subset of) the tetraploid data using a single-marker QTL model. The rates of QTL detection within LOD-2 intervals (defined as the maximum interval for which the LOD score differed by at most 2 from the peak LOD) ranged between 70–85% for both tetraploids and hexaploids, although these intervals were extremely wide (60 – 80 cM), covering most of the chromosome (Supplementary Figures S7 and S8 for tetraploid and hexaploid, respectively). In contrast, using the IBD model resulted in much narrower LOD-2 support intervals (Supplementary Figures S4 and S6).</p></div><div id="sec-16" class="subsection"><h3>Multi-allelic QTL</h3><p id="p-53">We performed a smaller complementary study using simulated multi-allelic rather than bi-allelic QTL (described in Supplementary File S3). On the whole, the results are consistent with what we discovered using simple bi-allelic QTL only. For example, the presence of quadrivalents (and hence double reduction) lead to slightly lower detection powers, although if the DR model is used this effect can be offset somewhat. The number of QTL alleles present did not appear to have much impact on QTL detection power. However, we did observe an increase in prediction accuracies if the DR model was used, but only in the extreme case of complete quadrivalent pairing (Supplementary File S3).</p></div><div id="sec-17" class="subsection"><h3>Application to real data</h3><p id="p-54">To help illustrate our findings we looked at two well-studied traits for which phenotypic data were available from the AxC tetraploid F<sub>1</sub> potato population. AxC is the result of a wide cross between the late, white/cream fleshed starch cultivar <em>Altus</em> and the early, yellow fleshed ware cultivar <em>Colomba</em>. On average the rate of quadrivalent pairing was 0.24 (and was similar between parents: parent 1 = 0.23 ± 0.07 and parent 2 = 0.25 ± 0.05) (Supplementary Figure S9), consistent with a previous estimate of 0.2 – 0.3 from this population using only SxN marker information (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bourke <em>et al.</em> 2015</a>). The phenotypic traits themselves (plant maturity and flesh color) are already genetically well-characterized, offering the opportunity to compare QTL peak positions with the physical location of the underlying candidate genes as well as an exploration of the most likely QTL models. For both traits a single major QTL was found with both the noDR and DR models (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). As can be seen from the bottom panel of <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, the GIC for some homologs was quite variable (<em>e.g.</em>, chromosomes 3, 4, 11 or 12) but was overall relatively high across both parental maps.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the QTL scans for traits plant maturity and tuber flesh color in the tetraploid AxC potato population (N = 222 for most analyses, c.f. Table 3). Results using the noDR model (random bivalent) are shown above those of the DR model (allowing double reduction) for both traits. LOD significance thresholds are shown as dashed red lines. The lowest panel shows the GIC per homolog for the eight parental homologs (using the noDR IBD probabilities), using the same color scheme as Figure 3.A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of the QTL scans for traits plant maturity and tuber flesh color in the tetraploid AxC potato population (N = 222 for most analyses, &lt;em&gt;c.f&lt;/em&gt;. Table 3). Results using the noDR model (random bivalent) are shown above those of the DR model (allowing double reduction) for both traits. LOD significance thresholds are shown as dashed red lines. The lowest panel shows the GIC per homolog for the eight parental homologs (using the noDR IBD probabilities), using the same color scheme as Figure 3.A.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F6.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/7/2107/F6.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493543" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-55" class="first-child">Results of the QTL scans for traits plant maturity and tuber flesh color in the tetraploid AxC potato population (N = 222 for most analyses, <em>c.f</em>. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Results using the noDR model (random bivalent) are shown above those of the DR model (allowing double reduction) for both traits. LOD significance thresholds are shown as dashed red lines. The lowest panel shows the GIC per homolog for the eight parental homologs (using the noDR IBD probabilities), using the same color scheme as <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3.A</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">There was a single QTL peak for plant maturity on chromosome 5 around 18 – 20 cM (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Using the DR model, slightly more of the variance was explained than with the noDR model (44% <em>vs.</em> 41%), and the width of the LOD-2 intervals was narrower (1 cM <em>vs.</em> 3 cM). When we saturated the QTL region with marker information and re-ran the IBD calculations and QTL analysis, we were able to increase the proportion of explained variance at the peak QTL position to 47%, with the peak occurring at approximately 20 cM using both models (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The most likely QTL model in a search among bi-allelic additive and simplex dominant QTL models (<em>c.f</em>. Supplementary Table S1) was an additive oooo x oooQ model, (where ‘o’ = no effect and ‘Q’ = having an effect, in this case positive <em>i.e.</em>, increasing earliness) (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). When visualizing the allele effects (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7.b</a>) the “early” allele from <em>Colomba</em> appears to have been balanced by a “late” allele (darker purple), somewhat unexpectedly for an early variety. The mean maturity of offspring that inherited the early allele from <em>Colomba</em> was 6.9 ± 0.7, and without was 5.7 ± 0.9.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493663/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493663/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Major QTL peaks for potato maturity (chromosome 5) and flesh color (chromosome 3) detected in AxC, phenotyped over 3 seasons (2012 – 2014)</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493506/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493506/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Exploration of major QTL peaks</span><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Re-mapping of the plant maturity QTL on potato chromosome 5 and the flesh color QTL on chromosome 3. A. Genetic vs. physical position of a subset of markers within the re-mapped QTL region of chromosome five (Ch. V). LOD-2 support intervals for the noDR and DR models overlapped, shown here as overlapping horizontal red / blue lines. The StCDF1 locus is highlighted by a vertical arrow at &#x223C;4.54 Mbp. Inset: Genetic v&#x2019;s physical position of markers used in initial TetraOrigin analysis on chromosome 5, with re-mapped region highlighted in red. B. LOD profiles of the re-mapping of the chromosome 5 QTL for plant maturity. noDR model results are shown in blue, with DR model results in red. Underneath the LOD profile, the average allele effects per homolog are shown (, where  is the weighted mean of the allele effect originating from parental homolog h (weighted by IBD haplotype probabilities) and  is the overall phenotypic mean). y-axis labels &#x201C;h1&#x201D; &#x2013; &#x201C;h8&#x201D; represent parental homolog numbering. The range of allele effects are shown by the bar on the right. C. Similar to (A), showing instead the re-mapped region of the chromosome 3 (Ch. III) peak for flesh color. The StChy2 locus is highlighted by a vertical arrow at &#x223C;43.94 Mbp. Inset: Genetic v&#x2019;s physical position of markers used in initial TetraOrigin analysis of chromosome 3. D. LOD profiles of the re-mapping of the chromosome 3 QTL for flesh color. noDR model results are shown in blue, with DR model results in red, as before." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Re-mapping of the plant maturity QTL on potato chromosome 5 and the flesh color QTL on chromosome 3. &lt;strong&gt;A.&lt;/strong&gt; Genetic &lt;em&gt;vs.&lt;/em&gt; physical position of a subset of markers within the re-mapped QTL region of chromosome five (Ch. V). LOD-2 support intervals for the noDR and DR models overlapped, shown here as overlapping horizontal red / blue lines. The &lt;em&gt;StCDF1&lt;/em&gt; locus is highlighted by a vertical arrow at &#x223C;4.54 Mbp. &lt;em&gt;Inset&lt;/em&gt;: Genetic v&#x2019;s physical position of markers used in initial TetraOrigin analysis on chromosome 5, with re-mapped region highlighted in red. &lt;strong&gt;B.&lt;/strong&gt; LOD profiles of the re-mapping of the chromosome 5 QTL for plant maturity. noDR model results are shown in blue, with DR model results in red. Underneath the LOD profile, the average allele effects per homolog are shown (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the weighted mean of the allele effect originating from parental homolog &lt;em&gt;h&lt;/em&gt; (weighted by IBD haplotype probabilities) and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-35&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the overall phenotypic mean). y-axis labels &#x201C;h1&#x201D; &#x2013; &#x201C;h8&#x201D; represent parental homolog numbering. The range of allele effects are shown by the bar on the right. &lt;strong&gt;C.&lt;/strong&gt; Similar to (A), showing instead the re-mapped region of the chromosome 3 (Ch. III) peak for flesh color. The &lt;em&gt;StChy2&lt;/em&gt; locus is highlighted by a vertical arrow at &#x223C;43.94 Mbp. &lt;em&gt;Inset&lt;/em&gt;: Genetic v&#x2019;s physical position of markers used in initial TetraOrigin analysis of chromosome 3. &lt;strong&gt;D.&lt;/strong&gt; LOD profiles of the re-mapping of the chromosome 3 QTL for flesh color. noDR model results are shown in blue, with DR model results in red, as before.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F7.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/7/2107/F7.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493579" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-59" class="first-child">Re-mapping of the plant maturity QTL on potato chromosome 5 and the flesh color QTL on chromosome 3. <strong>A.</strong> Genetic <em>vs.</em> physical position of a subset of markers within the re-mapped QTL region of chromosome five (Ch. V). LOD-2 support intervals for the noDR and DR models overlapped, shown here as overlapping horizontal red / blue lines. The <em>StCDF1</em> locus is highlighted by a vertical arrow at ∼4.54 Mbp. <em>Inset</em>: Genetic v’s physical position of markers used in initial TetraOrigin analysis on chromosome 5, with re-mapped region highlighted in red. <strong>B.</strong> LOD profiles of the re-mapping of the chromosome 5 QTL for plant maturity. noDR model results are shown in blue, with DR model results in red. Underneath the LOD profile, the average allele effects per homolog are shown (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F7/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F7/embed/mml-math-33.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F7/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F7/embed/mml-math-34.gif"/></noscript></span></span> is the weighted mean of the allele effect originating from parental homolog <em>h</em> (weighted by IBD haplotype probabilities) and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F7/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/F7/embed/mml-math-35.gif"/></noscript></span></span> is the overall phenotypic mean). y-axis labels “h1” – “h8” represent parental homolog numbering. The range of allele effects are shown by the bar on the right. <strong>C.</strong> Similar to (A), showing instead the re-mapped region of the chromosome 3 (Ch. III) peak for flesh color. The <em>StChy2</em> locus is highlighted by a vertical arrow at ∼43.94 Mbp. <em>Inset</em>: Genetic v’s physical position of markers used in initial TetraOrigin analysis of chromosome 3. <strong>D.</strong> LOD profiles of the re-mapping of the chromosome 3 QTL for flesh color. noDR model results are shown in blue, with DR model results in red, as before.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between genotypic information coefficient (GIC) and the inheritance probability of a particular parental homolog (i.e., the IBD probability of inheritance of a single haplotype) in any individual n (&#x3C0;n)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1270628746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between genotypic information coefficient (GIC) and the inheritance probability of a particular parental homolog (&lt;em&gt;i.e.&lt;/em&gt;, the IBD probability of inheritance of a single haplotype) in any individual &lt;em&gt;n&lt;/em&gt; (&#x3C0;&lt;sub&gt;n&lt;/sub&gt;)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2107/F8.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/7/2107/F8.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2107/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493613" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-60" class="first-child">Relationship between genotypic information coefficient (GIC) and the inheritance probability of a particular parental homolog (<em>i.e.</em>, the IBD probability of inheritance of a single haplotype) in any individual <em>n</em> (π<sub>n</sub>)</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">For flesh color, the noDR and DR models resulted in the same rate of explained variance at the chromosome 3 peak (53%), with the LOD-2 interval for the noDR narrower than that for the DR model (3 cM <em>vs.</em> 6 cM), in contrast to the results for plant maturity. When we searched for the most likely QTL model, we found in both cases (noDR and DR) that a dominant model with segregation type oooo x QQoo fit the data best (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Visualizing the allele effects at the QTL confirmed that <em>Colomba</em> (the yellow-fleshed parent) most likely contributed two copies of the beta-carotene hydroxylase gene to the population (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7.d</a>).</p><p id="p-62">We were interested in comparing the position of the QTL peaks with the physical location of the candidate genes <em>StCDF1</em> (for maturity) and <em>StChy2</em> (for tuber flesh color). As described in the Methods section, we saturated the LOD-5 support interval around the QTL peaks with markers and re-ran both the IBD calculations and QTL analysis. In both cases, the gene position fell within the LOD-2 intervals of the QTL peaks, although the LOD-2 interval was narrower using the noDR model, and gave a better indication of the QTL position than the DR model for flesh color; for plant maturity the support intervals from both models overlapped completely (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7.c</a>). For both traits, there appeared to be sufficient marker coverage on the important homologs within the QTL support intervals, reflected by relatively high GIC values. For plant maturity, the <em>StCDF1</em> region had far more mapped markers than elsewhere, suggesting that this locus was specifically targeted in the development of the SolSTW SNP array (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Uitdewilligen <em>et al.</em> 2013</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Vos <em>et al.</em> 2015</a>). As can be seen from <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7.a</a>, we were unable to separate these markers genetically due to the limited population size used for linkage map construction (N = 235), highlighting the inadequacy of this population size for fine-mapping work.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>The effect of variable Genotypic Information Coefficient (GIC)</h3><p id="p-63">Although reported as early as 1992 (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Knott and Haley 1992</a>), the influence of a variable GIC in the vicinity of QTL has essentially been ignored in many subsequent QTL studies both at the diploid and polyploid level. In this study we hope to re-emphasize its importance by demonstrating the effect of low GIC on QTL detection power, as well as the effect of variable GIC on QTL precision. According to our analysis, the GIC is one of the most important considerations in a QTL study (as well as population size and trait heritability), suggesting that dense marker coverage across all homologous chromosomes is important for successful QTL mapping.</p><p id="p-64">GIC values were found to drop at the telomeres, a consequence of the one-sided information available at these regions in the multi-point IBD estimation. This has the unwanted effect of biasing the QTL detection positions inwards, making it unlikely for a telomeric QTL to be found at the correct position. This could be cause for some concern, given that telomeric regions tend to be more gene-rich than more centromeric positions. However, as telomeric regions are also known to undergo more recombination (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gaut <em>et al.</em> 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2015</a>), the extent of this effect is likely to be diminished by the genetic extension of telomeric regions.</p><p id="p-65">Particularly in the case of autopolyploids, knowledge of homolog-specific GIC values is helpful in predicting whether a QTL is likely to lie beneath a QTL peak, since variable GIC profiles can lead to a significant bias in the estimated QTL position. We were unable to model homolog-specific GIC in the context of double reduction, as it was not obvious how GIC should behave when an offspring can inherit more than one copy of part of a particular homolog. This could also be seen as an advantage of using the noDR model, where a homolog-specific GIC is a clearly-defined concept (Appendix 1).</p><p id="p-66">The GIC cannot be further increased by increasing mapping population sizes (which is often thought to be the only way to increase power), above the limitations imposed by marker density, distribution and informativeness. If GIC values are found to be low on certain homologs, it could be worthwhile to develop more markers within that region on the affected homologs. In scenarios where this is impossible (<em>e.g.</em>, due to long stretches of homozygosity across homologs), the investigator remains blind to any potential QTL within that region, although it could be argued that such regions are unlikely to harbor segregating QTL either. For complex QTL types with more than one positive allele contributing to the trait, our results show that it is preferable to tag all QTL alleles through nearby informative markers rather than just one, or none. We also found evidence that haplotype diversity contributes to higher GIC values, although this was likely to be due to the higher number of segregating markers available in an F<sub>1</sub> cross when parents are on-average less related (containing greater haplotype diversity). This implies that parents of a mapping study should be carefully chosen to maximize haplotype diversity, particularly considering that haplotype-poor population types such as F<sub>1</sub> populations still predominate in polyploid mapping studies.</p></div><div id="sec-20" class="subsection"><h3>The merits of a complete polysomic model</h3><p id="p-67">We initiated this study to determine whether it is worthwhile to include double reduction in a QTL model. Through a large simulation study we have demonstrated that there are some improvements to QTL analysis, but they are relatively minor. We saw at most a 5% increase in detection power for “low-power” situations (population size 200 and trait heritability of 0.1) when a significant proportion of multivalents are formed (<em>q</em> = 0.5). In practice however, one is unlikely to encounter rates of multivalent formation this high (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bourke <em>et al.</em> 2015</a>). Double reduction introduces both incompatible genotypes (<em>e.g.</em>, a dosage of “2” from a SxN marker) as well as altered but otherwise credible genotypes (<em>e.g.</em>, a dosage of “2” from a SxS marker, which appears as if the marker allele was inherited from both parents, whereas in fact it came from only one). In an analysis which ignores double reduction (noDR), some noise is inevitably introduced in the IBD probabilities of certain individuals. However, this noise is essentially random across the population and the genome. On a per-individual basis the haplotypic reconstruction in the affected region can be dramatically altered. With a very limited population size (&lt; 50 individuals) this might be problematic, but in a population of 200 or 400 individuals there is generally ample QTL signal to drown out this noise. There is no huge computational burden to running both analyses and comparing results, and this would seem to be the preferred strategy. The time-limiting and computationally-intensive step is the calculation of parental marker phase in TetraOrigin, which is arguably a redundant step given current linkage mapping methodologies which also determine parental marker phase (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bourke <em>et al.</em> 2018</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hackett <em>et al.</em> 2017</a>).</p></div><div id="sec-21" class="subsection"><h3>Investigating dominance effects in a polyploid</h3><p id="p-68">One of the aspects that we tried to include in our study was dominance effects, by simulating complete dominance of QTL and checking whether such QTL could be accurately detected and predicted / diagnosed. However, the topic of dominance effects, more generally termed inter-allelic interactions, becomes rather complicated in a polyploid species with multiple functional QTL alleles acting at a locus. Kempthorne described the partitioning of QTL effects into a population mean, the additive main effects of each allele and the non-additive effects composed of first-order (di-allelic), second-order (tri-allelic) and third-order (tetra-allelic) interactions (<a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> 2014</a>; <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kempthorne 1957</a>). We tested whether a first-order interaction model increased our detection power for dominant QTL, but found the power was reduced slightly in comparison to the main effects model (data not shown). We therefore only included main effects in the initial QTL detection scan, which was previously recommended as a robust strategy for the initial detection of QTL effects (<a id="xref-ref-15-10" class="xref-bibr" href="#ref-15">Hackett <em>et al.</em> 2014</a>). Non-additive effects have also been shown to influence traits in autopolyploid populations, and such variation can be captured in a breeding program if treated appropriately (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Endelman <em>et al.</em> 2018</a>). However, we have confirmed in this study that dominant QTL can be detected at a relatively high rate using Kempthorne’s main-effects model (in fact, this appears to be the optimal detection strategy). Complete dominance (either simplex-dominant or duplex-dominant (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Rosyara <em>et al.</em> 2016</a>)) is distinguishable from additivity, but may not be the most realistic model for quantitative traits. Therefore, although we have attempted to include dominance effects in our study, we realize that this topic is probably more complex than presented here and merits further investigation.</p></div><div id="sec-22" class="subsection"><h3>Diagnosing QTL segregation type and mode of action</h3><p id="p-69">One of the main advantages of IBD-based QTL analyses over current single-marker approaches is the ability to determine the QTL configuration and mode of action. There was little difference in the performance of the BIC model selection procedure if double reduction was included or not, but we did find the DR model performed better in the case of complete multivalent pairing (Supplementary File S3). However, this is hardly a realistic scenario biologically-speaking (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ramsey and Schemske 2002</a>). We also found the accuracy of the procedure decreased greatly if we compared more potential QTL models (<em>e.g.</em>, testing only bi-allelic models (Figure S5) outperformed testing for multi-allelic models (Supplementary File S3), while tetraploid testing outperformed that of hexaploids (Figure S6)). This is to be expected – the more models that are compared, the greater the chance of finding an incorrect model as the most likely. However, this leads to a conundrum. There is quite a high likelihood (particularly at higher ploidy levels) of having more than two distinct functional alleles at a single locus. Restricting the model search procedure to a subset of models (as is done for example in the TetraploidSNPMap software (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Hackett <em>et al.</em> 2017</a>)) might lead to the incorrect model being chosen (simply because the correct model was not tested). On the other hand, if too many models are simultaneously compared, the overall accuracy drops.</p><p id="p-70">When we tested the BIC approach using a real potato dataset we found that a search among a restricted set of bi-allelic models gave a clearer result (whether a single model or multiple models emerged as plausible) than a wider search encompassing vast numbers of multi-allelic QTL models (data not shown). However, the visualization of allelic effects (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7 b&amp;d</a>) can at times be misleading. With simulated data we found that “positive” alleles usually appear to be balanced by one or more “negative” alleles (and vice versa), even though the remaining alleles were simulated to have <em>no</em> effect on the phenotype. QTL detection methods confined to a single genetic background such as an F<sub>1</sub> population have no reference against which to objectively compare allele effects and therefore both the presence and absence of a “strong-effect” QTL allele will have an apparent contribution. With a single functional allele one would expect this balancing effect to be spread evenly over the remaining alleles of that parent (three in a tetraploid). In practice, due to random deviations from the expected segregation of allelic combinations, a different picture can emerge.</p></div><div id="sec-23" class="subsection"><h3>Future outlook</h3><p id="p-71">In this study we have investigated QTL detection models within the context of bi-parental F<sub>1</sub> mapping populations only. The relative merits of QTL mapping <em>vs.</em> genome-wide association studies are already well-documented, <em>e.g.</em>, (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Korte and Farlow 2013</a>), with bi-parental populations capturing limited allelic diversity and having low resolution due to a lack of recombination events. Multi-parental populations such as MAGIC populations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cavanagh <em>et al.</em> 2008</a>), or mapping using pedigree information in connected populations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bink <em>et al.</em> 2012</a>) offer some of the advantages of both methods, sampling a greater allelic diversity while benefitting from the extra power that comes from a more balanced population structure. Extending the methodology for IBD probability estimation to these population designs would enable more advanced QTL detection approaches to be implemented in autopolyploids. They may also provide a clearer contrast for the estimation of genetic effects, or enable the investigation of QTL effects in different genetic backgrounds. Furthermore, the impact of double reduction in wider germplasm could be investigated along similar lines to those employed here. IBD information could also be used for other applications, <em>e.g.</em>, improved estimation of complex trait heritabilities (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Evans <em>et al.</em> 2018</a>).</p><p id="p-72">Genomic prediction has also been advocated as a strategy to accelerate genetic gain in autopolyploid breeding (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cellon <em>et al.</em> 2018</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Endelman <em>et al.</em> 2018</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hamilton and Kerr 2018</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Slater <em>et al.</em> 2016</a>). One of the topics currently under investigation is how to optimize marker set selection for use in genomic prediction (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ma <em>et al.</em> 2016</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sousa <em>et al.</em> 2019</a>). In autopolyploids (and to a lesser extent, in outcrossing diploids), marker coverage exists in two dimensions – along a chromosome, and across homologs of that chromosome. Here we have shown how the GIC quantifies how much information is captured by marker data across both these dimensions. It could therefore also represent a useful measure to optimize in the selection of marker sets for genomic prediction, assuming that maximizing GIC might also maximize prediction accuracy where marker number and distribution is concerned. We also did not attempt to define what constitutes a sufficiently-high GIC for QTL detection. These topics fell outside the scope of the current study, but would be interesting avenues for future research.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-76">Funding for this research was provided through the TKI polyploids projects “A genetic analysis pipeline for polyploid crops” (project number BO-26.03-002-001) and “Novel genetic and genomic tools for polyploid crops” (project numbers BO-26.03-009-004 and BO-50-002-022). PMB received an EMBO short term fellowship to work in the group of CAH at BioSS, Dundee, Scotland (ASTF number 228 – 2016), during which this research was initiated. The work of CAH and the TetraploidSNPMap software development were funded by the Rural &amp; Environment Science &amp; Analytical Services Division of the Scottish Government. The authors wish to acknowledge the potato breeding companies Averis Seeds B.V. and HZPC B.V. for providing the phenotypic data on the AxC population, Michiel Klaassen of WUR Plant Breeding for helpful comments and Professor Glenn Marion of BioSS for suggestions on the final manuscript.</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1</h2><div class="section" id="sec-24"><h3>Derivation of expression for Genotypic Information Coefficient, GIC</h3><p id="p-73"><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Van Ooijen (2009)</a> described a procedure to decompose the variance associated with a QTL (<em>V<sub>Q</sub></em>) into that which is explained by the markers (<em>V<sub>M</sub></em>) and a residual variance for which uncertainty remains (<em>V<sub>R</sub></em>). The GIC is then defined as (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Van Ooijen 2009</a>):<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-36.gif"/></noscript></span></span>We consider the case of the GIC for one homolog of an autopolyploid, and limit our attention to the assumption of random bivalent pairing (noDR model). The derivation of the GIC per homolog when double reduction is admissible is less intuitive and has been omitted here.</p><p id="p-74">At a given locus, we assume we have already calculated the IBD probability of inheritance of homolog <em>j</em>. This can be performed using TetraOrigin (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Zheng <em>et al.</em> 2016</a>) or some alternative method (<em>e.g.</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bourke 2014</a>; <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Hackett <em>et al.</em> 2013</a>)). For tetraploids, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-37.gif"/></noscript></span></span> while in hexaploids, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-38.gif"/></noscript></span></span> <em>etc</em>. We can consider the IBD probability of inheritance of a particular homolog to be the inheritance probability of the alternative allele in individual <em>n</em> (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-39.gif"/></noscript></span></span>), with the corresponding probability of inheritance of the reference allele <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-40.gif"/></noscript></span></span>.</p><p id="p-75">Using the definition of variance as Var(X) = E[X<sup>2</sup>] – (E[X])<sup>2</sup> <sub>,</sub> and using a mean of +1 for the reference allele and -1 for the alternative allele, the residual variance (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-41.gif"/></noscript></span></span> is given by:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-42.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-43.gif"/></noscript></span></span>and since <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-44.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-45.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-46.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-47.gif"/></noscript></span></span>where the sum is over <em>N</em> individuals in the F<sub>1</sub> population. If we assume that the QTL is not under selection so that the expected proportions of the reference and alternative allele are both 0.5, then the total variance is simply 0.5(1)<sup>2</sup> <sub>+</sub> 0.5(-1)<sup>2</sup> = 1, and therefore the genotypic information content for homolog <em>j</em> is given by<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-48.gif"/></noscript></span></span>However, it should be noted that substituting for <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-49.gif"/></noscript></span></span> in this equation using <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-50.gif"/></noscript></span></span> yields an identical formula (as a function of <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-51.gif"/></noscript></span></span>), and therefore without loss of generality we see that<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2107/embed/mml-math-52.gif"/></noscript></span></span>This is a quadratic function of the IBD probabilities for the inheritance of homolog <em>j</em> (averaged over a population of size <em>N</em>), taking a minimum (per individual) when the IBD probability equals 0.5 (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>).</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-77">Supplemental material available at Figshare: <a href="https://figshare.com/s/a97475f5802e3354f44e">https://figshare.com/s/a97475f5802e3354f44e</a>.</p></li><li class="fn" id="fn-8"><p id="p-78"><em>Communicating editor: A. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-28"><span class="received-label">Received </span>January 28, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-04-26"><span class="accepted-label">Accepted </span>April 26, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Bourke <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.1" data-doi="10.1007/s00122-011-1772-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totir</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">ter Braak</span>  <span class="cit-name-given-names">C. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">QTL linkage analysis of connected populations using ancestral marker and pedigree information</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1097</span>–<span class="cit-lpage">1113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1772-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBink%26rft.auinit1%253DM.%2BC.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D1097%26rft.epage%253D1113%26rft.atitle%253DQTL%2Blinkage%2Banalysis%2Bof%2Bconnected%2Bpopulations%2Busing%2Bancestral%2Bmarker%2Band%2Bpedigree%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1772-8%26rft_id%253Dinfo%253Apmid%252F22228242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1772-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22228242&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.7.2107.2"><div class="cit-metadata unstructured">Bourke, P. M., 2014 QTL analysis in polyploids. MSc thesis, Wageningen University, Wageningen</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.3" data-doi="10.1093/bioinformatics/bty371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Geest</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kranenburg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">polymapR - linkage analysis and genetic map construction from F1 populations of outcrossing polyploids.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">3496</span>–<span class="cit-lpage">3502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3496%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.4" data-doi="10.1007/s00122-016-2768-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kranenburg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Integrating haplotype-specific linkage maps in tetraploid species using SNP markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2211</span>–<span class="cit-lpage">2226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2768-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2211%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2768-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2768-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.5" data-doi="10.1534/genetics.115.181008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The double reduction landscape in tetraploid potato as revealed by a high-density linkage map.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.181008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.181008%26rft_id%253Dinfo%253Apmid%252F26377683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzMvODUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.6" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.7" data-doi="10.1016/j.pbi.2008.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2008.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBeyer%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D221%26rft.atitle%253DFrom%2Bmutations%2Bto%2BMAGIC%253A%2Bresources%2Bfor%2Bgene%2Bdiscovery%252C%2Bvalidation%2Band%2Bdelivery%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.01.002%26rft_id%253Dinfo%253Apmid%252F18295532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18295532&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255623300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.8" data-doi="10.1007/s10681-018-2165-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cellon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amadeu</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmstead</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattia</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrao</span>  <span class="cit-name-given-names">L. F. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Estimation of genetic parameters and prediction of breeding values in an autotetraploid blueberry breeding population with extensive pedigree data</span>. <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D214%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-018-2165-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-018-2165-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.9" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.10" data-doi="10.1534/genetics.118.300685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic variance partitioning and genome-wide prediction with allele dosage information in autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300685</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300685%26rft_id%253Dinfo%253Apmid%252F29514860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA5LzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.11" data-doi="10.1038/s41437-018-0067-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Narrow-sense heritability estimation of complex traits using identity-by-descent information</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-018-0067-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D121%26rft.spage%253D616%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-018-0067-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-018-0067-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.12" data-doi="10.1038/nrg1970"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination: an underappreciated factor in the evolution of plant genomes</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1970</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGaut%26rft.auinit1%253DB.%2BS.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D84%26rft.atitle%253DRecombination%253A%2Ban%2Bunderappreciated%2Bfactor%2Bin%2Bthe%2Bevolution%2Bof%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1970%26rft_id%253Dinfo%253Apmid%252F17173059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1970&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17173059&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242934300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.13" data-doi="10.1093/jhered/esg066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">TetraploidMap: construction of a linkage map in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">358</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esg066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D94%26rft.issue%253D4%26rft.spage%253D358%26rft.epage%253D359%26rft.atitle%253DTetraploidMap%253A%2BConstruction%2Bof%2Ba%2BLinkage%2BMap%2Bin%2BAutotetraploid%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesg066%26rft_id%253Dinfo%253Apmid%252F12920109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esg066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12920109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184922700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.14" data-doi="10.1093/jhered/esx022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boskamp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogogias</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preedy</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">TetraploidSNPMap: software for linkage analysis and QTL mapping in autotetraploid populations using SNP dosage data.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esx022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D108%26rft.spage%253D438%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesx022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esx022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.15" data-doi="10.1007/s00122-014-2347-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL mapping in autotetraploids using SNP dosage information.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1885</span>–<span class="cit-lpage">1904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2347-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1885%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2347-2%26rft_id%253Dinfo%253Apmid%252F24981609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2347-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24981609&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.16" data-doi="10.1371/journal.pone.0063939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Linkage analysis and QTL mapping using SNP dosage data in a tetraploid potato mapping population.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e63939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0063939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De63939%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0063939%26rft_id%253Dinfo%253Apmid%252F23704960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0063939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23704960&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.17" data-doi="10.1093/jhered/esm086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TetraploidMap for Windows: linkage map construction and QTL mapping in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">729</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esm086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesm086%26rft_id%253Dinfo%253Apmid%252F17965198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esm086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17965198&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251867400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.18" data-doi="10.1007/BF02984197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">Theoretical genetics of autopolyploids.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02984197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D22%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02984197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02984197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200610100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.19" data-doi="10.1007/s00122-017-3041-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Computation of the inverse additive relationship matrix for autopolyploid and multiple-ploidy populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">860</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-3041-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D851%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-3041-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-3041-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, K. L. Childs, J. Cepela, E. Crisovan<span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Spud DB: A resource for mining sequences, genotypes, and phenotypes to accelerate potato breeding.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2107.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-source">An introduction to genetic statistics</span>, <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.22" data-doi="10.1038/nature11912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Naturally occurring allele diversity allows potato cultivation in northern latitudes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11912%26rft_id%253Dinfo%253Apmid%252F23467094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23467094&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316652300053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.23" data-doi="10.1007/s001220050482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Multiple marker mapping of quantitative trait loci in an outbred pedigree of loblolly pine.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">810</span>–<span class="cit-lpage">820</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050482</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D94%26rft.spage%253D810%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XE63000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Maximum likelihood mapping of quantitative trait loci using full-sib families.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1211</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BA.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D1211%26rft.epage%253D1222%26rft.atitle%253DMaximum%2Blikelihood%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Busing%2Bfull-sib%2Bfamilies.%26rft_id%253Dinfo%253Apmid%252F1459438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzMi80LzEyMTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.25" data-doi="10.1186/1746-4811-9-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The advantages and limitations of trait analysis with GWAS: a review</span>. <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1746-4811-9-29</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D9%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-9-29%26rft_id%253Dinfo%253Apmid%252F23876160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-9-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23876160&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.26" data-doi="10.1155/2010/216547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">EM algorithm for mapping quantitative trait loci in multivalent tetraploids.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant. Genom.</abbr> <span class="cit-vol">2010</span>: <span class="cit-elocation-id">216547</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2010/216547</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant.%2BGenom.%26rft.volume%253D2010%26rft.spage%253D216547%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2010%252F216547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2010/216547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.27" data-doi="10.1038/ncomms7648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dissecting meiotic recombination based on tetrad analysis by single-microspore sequencing in maize</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms7648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6648%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7648%26rft_id%253Dinfo%253Apmid%252F25800954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25800954&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.28" data-doi="10.1073/pnas.0304482101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearsey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Theoretical basis for genetic linkage analysis in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">7040</span>–<span class="cit-lpage">7045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0304482101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0304482101%26rft_id%253Dinfo%253Apmid%252F15100415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE4LzcwNDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.29" data-doi="10.1534/genetics.105.052449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leach</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <abbr class="cit-jnl-abbrev">Constructing Genetic Linkage Maps Under a Tetrasomic Model Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2635</span>–<span class="cit-lpage">2645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.052449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConstructing%2BGenetic%2BLinkage%2BMaps%2BUnder%2Ba%2BTetrasomic%2BModel%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.052449%26rft_id%253Dinfo%253Apmid%252F16415363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI2MzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.30" data-doi="10.1007/s11032-016-0504-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Potential of marker selection to increase prediction accuracy of genomic selection in soybean (<em>Glycine max</em> L.)</span>. <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-016-0504-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-016-0504-9%26rft_id%253Dinfo%253Apmid%252FWOS%253A00038214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-016-0504-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=WOS:00038214&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.31" data-doi="10.1007/BF02982205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1935</span> <span class="cit-article-title">Reductional and equational separation of the chromosomes in bivalents and multivalents.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02982205</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D30%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02982205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02982205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.32" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2107.33"><div class="cit-metadata unstructured">Pinheiro, J., and D. Bates, DebRoy S, Sarkar D, R Core Team 2017 nlme: linear and nonlinear mixed effects models. R package version 3.1–131 https://CRANR-projectorg/package=nlme</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.34" data-doi="10.1038/nature10158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Potato Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome sequence and analysis of the tuber crop potato.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXu%26rft.auinit1%253DX.%26rft.volume%253D475%26rft.issue%253D7355%26rft.spage%253D189%26rft.epage%253D195%26rft.atitle%253DGenome%2Bsequence%2Band%2Banalysis%2Bof%2Bthe%2Btuber%2Bcrop%2Bpotato.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10158%26rft_id%253Dinfo%253Apmid%252F21743474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21743474&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292690500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2107.35"><div class="cit-metadata unstructured">R Core Team 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria R version 3.3.2</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.36" data-doi="10.1146/annurev.ecolsys.33.010802.150437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Neopolyploidy in flowering plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.33.010802.150437</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.33.010802.150437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.33.010802.150437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000180007000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.37" data-doi="10.1017/S0016672305007354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Valdés</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An entropy-based measure of founder informativeness.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672305007354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DReyes-Vald%25C3%25A9s%26rft.auinit1%253DM.%2BH.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D88%26rft.atitle%253DAn%2Bentropy-based%2Bmeasure%2Bof%2Bfounder%2Binformativeness.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672305007354%26rft_id%253Dinfo%253Apmid%252F16089038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672305007354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16089038&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228943000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.38" data-doi="10.3835/plantgenome2015.08.0073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosyara</span>  <span class="cit-name-given-names">U. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Software for genome-wide association studies in autopolyploids and its application to potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.08.0073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.08.0073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.08.0073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.39" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Estimating the dimension of a model</span>. <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.40" data-doi="10.3835/plantgenome2016.02.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improving genetic gain with genomic selection in autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.02.0021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.02.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.41" data-doi="10.1007/s10681-019-2339-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">M. B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyra</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granato</span>  <span class="cit-name-given-names">I. S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matias</span>  <span class="cit-name-given-names">F. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Increasing accuracy and reducing costs of genomic prediction by marker selection</span>. <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">215</span>: 18.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-019-2339-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-019-2339-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-019-2339-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.42" data-doi="10.1371/journal.pone.0062355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A.-M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borm</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e62355</span> (erratum: PLoS One 10: e0141940).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0062355</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062355%26rft_id%253Dinfo%253Apmid%252F23667470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2107.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>MapQTL 6</em>, <em>Software for the mapping of quantitative trait loci in experimental populations of diploid species</em></span>, <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.44" data-doi="10.1007/BF00227388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Accuracy of mapping quantitative trait loci in autogamous species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00227388</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D84%26rft.spage%253D803%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00227388%26rft_id%253Dinfo%253Apmid%252F24201478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00227388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24201478&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JT30000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.45" data-doi="10.1186/1471-2105-13-248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The simulation of meiosis in diploid and tetraploid organisms using various genetic models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-248</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D248%26rft.epage%253D248%26rft.atitle%253DThe%2Bsimulation%2Bof%2Bmeiosis%2Bin%2Bdiploid%2Band%2Btetraploid%2Borganisms%2Busing%2Bvarious%2Bgenetic%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-248%26rft_id%253Dinfo%253Apmid%252F23013469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23013469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.46" data-doi="10.1007/s00122-015-2593-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G. A. M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eck</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development and analysis of a 20K SNP array for potato (<em>Solanum tuberosum</em>): an insight into the breeding history.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">2387</span>–<span class="cit-lpage">2401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2593-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D2387%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2593-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2593-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2107.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Wolfram Research Inc</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Mathematica Version 10.0</span>. <span class="cit-publ-name">Wolfram Research Inc.</span>, <span class="cit-publ-loc">Champaign, Illinois</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.48" data-doi="10.1007/s11103-010-9647-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A.-M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutten</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of alleles of carotenoid pathway genes important for zeaxanthin accumulation in potato tubers.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">671</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11103-010-9647-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DWolters%26rft.auinit1%253DA.%2BM.%26rft.volume%253D73%26rft.issue%253D6%26rft.spage%253D659%26rft.epage%253D671%26rft.atitle%253DIdentification%2Bof%2Balleles%2Bof%2Bcarotenoid%2Bpathway%2Bgenes%2Bimportant%2Bfor%2Bzeaxanthin%2Baccumulation%2Bin%2Bpotato%2Btubers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-010-9647-y%26rft_id%253Dinfo%253Apmid%252F20490894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-010-9647-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20490894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279607200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.49" data-doi="10.1017/S0016672399004395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mapping quantitative trait loci in tetraploid populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672399004395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DXie%26rft.auinit1%253DC.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D115%26rft.atitle%253DMapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Btetraploid%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004395%26rft_id%253Dinfo%253Apmid%252F11006638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11006638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089181400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.50" data-doi="10.1093/bib/bbt073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A statistical model for QTL mapping in polysomic autotetraploids underlying double reduction.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1044</span>–<span class="cit-lpage">1056</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbt073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbt073%26rft_id%253Dinfo%253Apmid%252F24177379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbt073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24177379&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2107.51" data-doi="10.1534/genetics.115.185579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Probabilistic multilocus haplotype reconstruction in outcrossing tetraploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.185579</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185579%26rft_id%253Dinfo%253Apmid%252F26920758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvMTE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2107.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2097" title="An Interscholastic Network To Generate LexA Enhancer Trap Lines in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2123" title="Optimizing Selection and Mating in Genomic Selection with a Look-Ahead Approach: An Operations Research Framework" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2107" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-X_GhpQBAdBLxNG5bm4SnbkoypoB8_VYXheuqAVHlKq8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2107" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2107" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2107" class="active" data-icon-position="" data-hide-link-title="0">Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493468" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WURvQSt8itfU67bf-yAQe4GMGkIGeO_y0irbm8HwRkE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2107.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2107" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3XvNssRrOnjZ7nu6vhSRU7L4lpnxlBO_Lswt8XRiUik" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2107" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2107.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493468" id="node493468--2" data-pisa="ggg;9/7/2107" data-pisa-master="ggg;g3.119.400269" data-apath="/ggg/9/7/2107.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0665-6508" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter M. Bourke</span>, <span class="highwire-citation-author" data-delta="1">Christine A. Hackett</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4388-9244" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roeland E. Voorrips</span>, <span class="highwire-citation-author" data-delta="3">Richard G. F. Visser</span> and <span class="highwire-citation-author" data-delta="4">Chris Maliepaard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2107-2122;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400269 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493468--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Peter M. Bourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BBourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourke%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BM.%2BBourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0665-6508" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter M. Bourke</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.bourke@wur.nl" class="" data-icon-position="" data-hide-link-title="0">peter.bourke@wur.nl</a>
<a href="mailto:chris.maliepaard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">chris.maliepaard@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christine A. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biomathematics and Statistics Scotland, Invergowrie, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BA.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BA.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Roeland E. Voorrips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roeland%2BE.%2BVoorrips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voorrips%20RE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoeland%2BE.%2BVoorrips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4388-9244" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roeland E. Voorrips</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard G. F. Visser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BF.%2BVisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Visser%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BF.%2BVisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Maliepaard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BMaliepaard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maliepaard%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BMaliepaard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.bourke@wur.nl" class="" data-icon-position="" data-hide-link-title="0">peter.bourke@wur.nl</a>
<a href="mailto:chris.maliepaard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">chris.maliepaard@wur.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2107" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2107&amp;title=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2107&amp;title=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2107&amp;title=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2107&amp;text=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2107&amp;title=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2107&amp;t=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2107&amp;title=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2107&amp;title=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493468/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493468" id="node493468--3" data-pisa="ggg;9/7/2107" data-pisa-master="ggg;g3.119.400269" data-apath="/ggg/9/7/2107.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantifying the Power and Precision of QTL Analysis in Autopolyploids Under Bivalent and Multivalent Genetic Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0665-6508" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter M. Bourke</span>, <span class="highwire-citation-author" data-delta="1">Christine A. Hackett</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4388-9244" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roeland E. Voorrips</span>, <span class="highwire-citation-author" data-delta="3">Richard G. F. Visser</span> and <span class="highwire-citation-author" data-delta="4">Chris Maliepaard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2107-2122;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400269 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493468--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Peter M. Bourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BBourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourke%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BM.%2BBourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0665-6508" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter M. Bourke</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.bourke@wur.nl" class="" data-icon-position="" data-hide-link-title="0">peter.bourke@wur.nl</a>
<a href="mailto:chris.maliepaard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">chris.maliepaard@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christine A. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biomathematics and Statistics Scotland, Invergowrie, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BA.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BA.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Roeland E. Voorrips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roeland%2BE.%2BVoorrips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voorrips%20RE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoeland%2BE.%2BVoorrips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4388-9244" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roeland E. Voorrips</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard G. F. Visser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BF.%2BVisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Visser%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BF.%2BVisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Maliepaard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Wageningen University & Research, Droevendaalsesteeg 1, P.O. Box 386, 6700 AJ Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BMaliepaard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maliepaard%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BMaliepaard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.bourke@wur.nl" class="" data-icon-position="" data-hide-link-title="0">peter.bourke@wur.nl</a>
<a href="mailto:chris.maliepaard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">chris.maliepaard@wur.nl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2107&amp;rft.epage=2122&amp;rft.atitle=Quantifying%20the%20Power%20and%20Precision%20of%20QTL%20Analysis%20in%20Autopolyploids%20Under%20Bivalent%20and%20Multivalent%20Genetic%20Models&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bourke%2C+Peter+M.&amp;rft.au=Hackett%2C+Christine+A.&amp;rft.au=Voorrips%2C+Roeland+E.&amp;rft.au=Visser%2C+Richard+G.+F.&amp;rft.au=Maliepaard%2C+Chris"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493468/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493468/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493468/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493468/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493468/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493468/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493468/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493468/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493468/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493468/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493468/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493468/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2107" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2107.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2107.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2107.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
