<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2089" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2089.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2089.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2089.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2089.atom" />
<meta name="HW.pisa" content="ggg;9/7/2089" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data" />
<meta name="DC.Identifier" content="10.1534/g3.119.400185" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Orjuela et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The extensive generation of RNA sequencing (RNA-seq) data in the last decade has resulted in a myriad of specialized software for its analysis. Each software module typically targets a specific step within the analysis pipeline, making it necessary to join several of them to get a single cohesive workflow. Multiple software programs automating this procedure have been proposed, but often lack modularity, transparency or flexibility. We present ARMOR, which performs an end-to-end RNA-seq data analysis, from raw read files, via quality checks, alignment and quantification, to differential expression testing, geneset analysis and browser-based exploration of the data. ARMOR is implemented using the Snakemake workflow management system and leverages conda environments; Bioconductor objects are generated to facilitate downstream analysis, ensuring seamless integration with many R packages. The workflow is easily implemented by cloning the GitHub repository, replacing the supplied input and reference files and editing a configuration file. Although we have selected the tools currently included in ARMOR, the setup is modular and alternative tools can be easily integrated." />
<meta name="DC.Contributor" content="Stephany Orjuela" />
<meta name="DC.Contributor" content="Ruizhu Huang" />
<meta name="DC.Contributor" content="Katharina M. Hembach" />
<meta name="DC.Contributor" content="Mark D. Robinson" />
<meta name="DC.Contributor" content="Charlotte Soneson" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The extensive generation of RNA sequencing (RNA-seq) data in the last decade has resulted in a myriad of specialized software for its analysis. Each software module typically targets a specific step within the analysis pipeline, making it necessary to join several of them to get a single cohesive workflow. Multiple software programs automating this procedure have been proposed, but often lack modularity, transparency or flexibility. We present ARMOR, which performs an end-to-end RNA-seq data analysis, from raw read files, via quality checks, alignment and quantification, to differential expression testing, geneset analysis and browser-based exploration of the data. ARMOR is implemented using the Snakemake workflow management system and leverages conda environments; Bioconductor objects are generated to facilitate downstream analysis, ensuring seamless integration with many R packages. The workflow is easily implemented by cloning the GitHub repository, replacing the supplied input and reference files and editing a configuration file. Although we have selected the tools currently included in ARMOR, the setup is modular and alternative tools can be easily integrated.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2089" />
<meta name="citation_id" content="9/7/2089" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2089" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2089.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2089.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2089.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400185" />
<meta name="citation_pmid" content="31088905" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="2089" />
<meta name="citation_lastpage" content="2096" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Stephany Orjuela" />
<meta name="citation_author_institution" content="SIB Swiss Institute of Bioinformatics, Zurich, Switzerland" />
<meta name="citation_author_institution" content="Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland" />
<meta name="citation_author_institution" content="Institute of Molecular Cancer Research, University of Zurich, Zurich, Switzerland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1508-461X" />
<meta name="citation_author" content="Ruizhu Huang" />
<meta name="citation_author_institution" content="SIB Swiss Institute of Bioinformatics, Zurich, Switzerland" />
<meta name="citation_author_institution" content="Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3285-1945" />
<meta name="citation_author" content="Katharina M. Hembach" />
<meta name="citation_author_institution" content="SIB Swiss Institute of Bioinformatics, Zurich, Switzerland" />
<meta name="citation_author_institution" content="Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland" />
<meta name="citation_author_institution" content="Department of Quantitative Biomedicine, University of Zurich, Zurich, Switzerland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5041-6205" />
<meta name="citation_author" content="Mark D. Robinson" />
<meta name="citation_author_institution" content="SIB Swiss Institute of Bioinformatics, Zurich, Switzerland" />
<meta name="citation_author_institution" content="Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland" />
<meta name="citation_author_email" content="mark.robinson@imls.uzh.ch" />
<meta name="citation_author_email" content="charlotte.soneson@fmi.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3048-5518" />
<meta name="citation_author" content="Charlotte Soneson" />
<meta name="citation_author_institution" content="SIB Swiss Institute of Bioinformatics, Zurich, Switzerland" />
<meta name="citation_author_institution" content="Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland" />
<meta name="citation_author_email" content="mark.robinson@imls.uzh.ch" />
<meta name="citation_author_email" content="charlotte.soneson@fmi.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3833-2169" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Afgan;citation_author=D. Baker;citation_author=B. Batut;citation_author=M. van den Beek;citation_author=D. Bouvier;citation_title=The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2018 update.;citation_pages=W537-W544;citation_volume=46;citation_year=2018;citation_pmid=29790989;citation_doi=10.1093/nar/gky379" />
<meta name="citation_reference" content="Allaire, J., Y. Xie, J. McPherson, J. Luraschi, K. Ushey, et al., 2018 rmarkdown: Dynamic Documents for R. R package version 1.11." />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=R. A. Amezquita;citation_author=V. J. Carey;citation_author=L. N. Carpp;citation_author=L. Geistlinger;citation_author=A. T. L. Lun;citation_title=Orchestrating Single-Cell analysis with Bioconductor.;citation_year=2019;citation_doi=10.1101/590562" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Conesa;citation_author=P. Madrigal;citation_author=S. Tarazona;citation_author=D. Gomez-Cabrero;citation_author=A. Cervera;citation_title=A survey of best practices for RNA-seq data analysis.;citation_pages=13;citation_volume=17;citation_year=2016;citation_pmid=26813401;citation_doi=10.1186/s13059-016-0881-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Cornwell;citation_author=M. Vangala;citation_author=L. Taing;citation_author=Z. Herbert;citation_author=J. Köster;citation_title=VIPER: Visualization pipeline for RNA-seq, a snakemake workflow for efficient and complete RNA-seq analysis.;citation_pages=135;citation_volume=19;citation_year=2018;citation_pmid=29649993;citation_doi=10.1186/s12859-018-2139-9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=N. Doumpas;citation_author=F. Lampart;citation_author=M. D. Robinson;citation_author=A. Lentini;citation_author=C. E. Nestor;citation_title=TCF/LEF dependent and independent transcriptional regulation of Wnt/β-catenin target genes.;citation_volume=38;citation_year=2019;citation_pmid=30425074;citation_doi=10.15252/embj.201798873" />
<meta name="citation_reference" content="Ewels, P., R. Hammarén, A. Peltzer, D. Moreno, rfenouil, et al., 2018 nf-core/rnaseq: nf-core/rnaseq version 1.2." />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Ewels;citation_author=M. Magnusson;citation_author=S. Lundin;citation_author=M. Käller;citation_title=MultiQC: Summarize analysis results for multiple tools and samples in a single report.;citation_volume=32;citation_year=2016;citation_pmid=27312411;citation_doi=10.1093/bioinformatics/btw354" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Frankish;citation_author=M. Diekhans;citation_author=A.-M. Ferreira;citation_author=R. Johnson;citation_author=I. Jungreis;citation_title=GENCODE reference annotation for the human and mouse genomes.;citation_pages=D766-D773;citation_volume=47;citation_year=2019;citation_doi=10.1093/nar/gky955" />
<meta name="citation_reference" content="Girke, T., 2018 systemPipeRdata: NGS workflow templates and sample data. R package version 1.10.0." />
<meta name="citation_reference" content="citation_title=Transcriptome Data Analysis: Methods and Protocols;citation_title=QuickRNASeq: guide for pipeline implementation and for interactive results visualization;citation_author=W. He;citation_author=S. Zhao;citation_author=C. Zhang;citation_author=M. S. Vincent;citation_author=B. Zhang;citation_pages=57-70;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=W. Huber;citation_author=V. J. Carey;citation_author=R. Gentleman;citation_author=S. Anders;citation_author=M. Carlson;citation_title=Orchestrating high-throughput genomic analysis with Bioconductor.;citation_pages=115-121;citation_volume=12;citation_year=2015;citation_pmid=25633503;citation_doi=10.1038/nmeth.3252" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=L.-H. Hung;citation_author=J. Hu;citation_author=T. Meiss;citation_author=A. Ingersoll;citation_author=W. Lloyd;citation_title=Building containerized workflows using the BioDepot-workflow-builder (bwb).;citation_year=2018;citation_doi=10.1101/099010" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Köster;citation_author=S. Rahmann;citation_title=Snakemake–a scalable bioinformatics workflow engine.;citation_volume=28;citation_year=2012;citation_pmid=22908215;citation_doi=10.1093/bioinformatics/bts480" />
<meta name="citation_reference" content="Love, M., R. Patro, P. Hickey, and C. Soneson, 2019 tximeta: Transcript Quantification Import with Automatic Metadata. R package version 1.1.16." />
<meta name="citation_reference" content="Lun, A. and D. Risso, 2019 SingleCellExperiment: S4 Classes for Single Cell Data. R package version 1.4.1." />
<meta name="citation_reference" content="Lun, A. T. L., Y. Chen, and G. K. Smyth, 2016 It’s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using Quasi-Likelihood methods in edgeR, pp. 391–416, Statistical Genomics, edited by E. Mathé and S. Davis, Methods in Molecular Biology, Springer New York, New York. 10.1007/978-1-4939-3578-9_19" />
<meta name="citation_reference" content="Marini, F., 2018 ideal: Interactive Differential Expression AnaLysis. R package version 1.8.0." />
<meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.;citation_author=B. Monier;citation_author=A. McDermaid;citation_author=C. Wang;citation_author=J. Zhao;citation_author=A. Miller;citation_title=IRIS-EDA: An integrated RNA-seq interpretation system for gene expression data analysis.;citation_pages=e1006792;citation_volume=15;citation_year=2019;citation_doi=10.1371/journal.pcbi.1006792" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=M. Nowicka;citation_author=M. D. Robinson;citation_title=DRIMSeq: a Dirichlet-multinomial framework for multivariate count outcomes in genomics.;citation_pages=1356;citation_volume=5;citation_year=2016;citation_doi=10.12688/f1000research.8900.2" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=R. Patro;citation_author=G. Duggal;citation_author=M. I. Love;citation_author=R. A. Irizarry;citation_author=C. Kingsford;citation_title=Salmon provides fast and bias-aware quantification of transcript expression.;citation_pages=417-419;citation_volume=14;citation_year=2017;citation_pmid=28263959;citation_doi=10.1038/nmeth.4197" />
<meta name="citation_reference" content="citation_author=D. R. Powell;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=K. Rue-Albrecht;citation_author=F. Marini;citation_author=C. Soneson;citation_author=A. T. L. Lun;citation_title=iSEE: Interactive SummarizedExperiment explorer;citation_pages=741;citation_volume=7;citation_year=2018;citation_doi=10.12688/f1000research.14966.1" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=C. Soneson;citation_author=M. I. Love;citation_author=M. D. Robinson;citation_title=Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.;citation_pages=1521;citation_volume=4;citation_year=2015;citation_pmid=26925227;citation_doi=10.12688/f1000research.7563.1" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=M. J. Steinbaugh;citation_author=L. Pantano;citation_author=R. D. Kirchner;citation_author=V. Barrera;citation_author=B. A. Chapman;citation_title=bcbioRNASeq: R package for bcbio RNA-seq analysis.;citation_pages=1976;citation_volume=6;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Cell Syst.;citation_author=D. Torre;citation_author=A. Lachmann;citation_author=A. Ma’ayan;citation_title=BioJupies: Automated generation of interactive notebooks for RNA-Seq data analysis in the cloud.;citation_pages=556-561.e3;citation_volume=7;citation_year=2018;citation_doi=10.1016/j.cels.2018.10.007" />
<meta name="citation_reference" content="citation_journal_title=JOSS;citation_author=K. Tsyganov;citation_author=A. James Perry;citation_author=S. Kenneth Archer;citation_author=D. Powell;citation_title=RNAsik: A pipeline for complete and reproducible RNA-seq analysis that runs anywhere with speed and ease.;citation_pages=583;citation_volume=3;citation_year=2018;citation_doi=10.21105/joss.00583" />
<meta name="citation_reference" content="citation_journal_title=RNA sequencing data: hitchhiker’s guide to expression analysis;citation_author=K. Van Den Berge;citation_author=K. Hembach;citation_author=C. Soneson;citation_author=S. Tiberi;citation_author=L. Clement;citation_pages=e27283v2;citation_volume=6;citation_year=2018;citation_doi=10.7287/peerj.preprints.27283v2" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=D. Wang;citation_title=hppRNA-a snakemake-based handy parameter-free pipeline for RNA-Seq analysis of numerous samples.;citation_pages=622-626;citation_volume=19;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Wu;citation_author=G. K. Smyth;citation_title=Camera: a competitive gene set test accounting for inter-gene correlation.;citation_volume=40;citation_year=2012;citation_pmid=22638577;citation_doi=10.1093/nar/gks461" />
<meta name="citation_reference" content="citation_title=R Markdown: The Definitive Guide;citation_author=Y. Xie;citation_author=J. Allaire;citation_author=G. Grolemund;citation_year=2018;citation_doi=10.1201/9781138359444" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. R. Zerbino;citation_author=P. Achuthan;citation_author=W. Akanni;citation_author=M. R. Amode;citation_author=D. Barrell;citation_title=Ensembl 2018.;citation_pages=D754-D761;citation_volume=46;citation_year=2018;citation_pmid=29155950;citation_doi=10.1093/nar/gkx1098" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="The extensive generation of RNA sequencing (RNA-seq) data in the last decade has resulted in a myriad of specialized software for its analysis. Each software module typically targets a specific step within the analysis pipeline, making it necessary to join several of them to get a single cohesive workflow. Multiple software programs automating this procedure have been proposed, but often lack modularity, transparency or flexibility. We present ARMOR, which performs an end-to-end RNA-seq data analysis, from raw read files, via quality checks, alignment and quantification, to differential expression testing, geneset analysis and browser-based exploration of the data. ARMOR is implemented using the Snakemake workflow management system and leverages conda environments; Bioconductor objects are generated to facilitate downstream analysis, ensuring seamless integration with many R packages. The workflow is easily implemented by cloning the GitHub repository, replacing the supplied input and reference files and editing a configuration file. Although we have selected the tools currently included in ARMOR, the setup is modular and alternative tools can be easily integrated." />
<meta name="og-title" property="og:title" content="ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2089" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The extensive generation of RNA sequencing (RNA-seq) data in the last decade has resulted in a myriad of specialized software for its analysis. Each software module typically targets a specific step within the analysis pipeline, making it necessary to join several of them to get a single cohesive workflow. Multiple software programs automating this procedure have been proposed, but often lack modularity, transparency or flexibility. We present ARMOR, which performs an end-to-end RNA-seq data analysis, from raw read files, via quality checks, alignment and quantification, to differential expression testing, geneset analysis and browser-based exploration of the data. ARMOR is implemented using the Snakemake workflow management system and leverages conda environments; Bioconductor objects are generated to facilitate downstream analysis, ensuring seamless integration with many R packages. The workflow is easily implemented by cloning the GitHub repository, replacing the supplied input and reference files and editing a configuration file. Although we have selected the tools currently included in ARMOR, the setup is modular and alternative tools can be easily integrated." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wMYNSMcoif4Q6UIK85zSn-Ui63l3gyJFvVkiDAKklro","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493465","apath":"\/ggg\/9\/7\/2089.atom","pisa":"ggg;9\/7\/2089","ac":{"\/ggg\/9\/7\/2089.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2089.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2089"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2089"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2089":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OKGbXM8+X823m\\\\\\\/Yy\\\\\\\/Ghd0lGkaZ59LeGrjSfqsMdqjk7uCTtnh0+PRLfpXZtYm1ByytBHRGnvapctX\\\\\\\/Eni2q\\\\\\\/6\\\\\\\/78g3vkQn8FvL0Z115eIBJH5yNwvfTWLgJs1Yp+9I4RQZTNJghPNfOa\\\\\\\/xtWkS8ehSgc7ht+7Ro0a2t4K00uU4Fr+QtE9k8cQ9r8mHC5dcPhzAYgNCOP9sieGoDCFOMqPOTjWn0z1ngaA4jTuFefcCPwiay22u1n1m+HEGtNPdI\\\\\\\/KsIdERRrvtskdkSAq4WEBumhNQMt2QQCDD9f9ra3mSuDWbZe4ay4moZYGhQCIGEZH+iEpzSyGJ14RviY9+jbB0ccttANmZ8lNXaeWKMFR0Y+Av2mnxvXlGmogB+mKphgpwYL6BPzQaii5Mj+173zI3gLxOI6\\\\\\\/0bEnlHQ\\\\\\\/XhtJfj2LWZbGxsRinJEu6Kgjyth1ym+TrSlFqolDJAJNuRmIIef5YTnKlnjg0XdNkdUZgPwHjOp9WhuKIql7z60eDjGTN69wqRBMSD7TMvEgGNy+griN9\\\\\\\/YzcwIfPNbj22Da\\\\\\\/uWoXcW0563x9nERV97maAxzO0nKKJ5fTEvab1QvAtOMg7HHC5gF+wMwzzkusDaiKAzxDIiy6EQiURJd7dwGcydXx8mvnWNxokyI8bDjrqW5FKxm3uT7ypHnk4hm1evc6vRt+jgTL21ddzA+vym4UL73rFLFcT+8VVMCGkD9QDPgJfDLHI7H5AfodUAjlhDW40ZR01NYrnaMDkthOXA1zT7GAnYg2J8xT8UhrTxmVqOvo\\\\\\\/R+wZkFtGlgnGQMRignu3NIeOvO7m3amLeAZmmmVgyDM9MQZ4r4T0ZkE\\\\\\\/BoIGKFM1WGfhjMIt1CRn3EHGalU1C6mxfgRchj3T3S77jLc3jx8g8x8+U7IsS3R204PMc0cV5jpj841gE0AeVtMXTyYmlrzg2CMh1AJY\\\\\\\/O0+KGVrDr1u\\\\\\\/\\\\\\\/oL\\\\\\\/ceLb9\\\\\\\/mXpX15bQQMCvczfeN59E7dMxA2fglUI1nFMZsdq4q0PRXPjlo2MjH62Walty4qIEqbLwcGX9HKObqWhpoeG+JeauJcj9F8H7ElI0ar0Pt6WkuL1mIldftB42Q6J\\\\\\\/YSiJCVu6FQrKL5mZ00BpBVasdVnpAGJ5rny9OG9AWQ0QRlkLu\\\\\\\/INojru56wPrEwlitaXcOY01DnGbcFC8s6Kfd1w6N8ULYcTTUcLd360PgMpClh3wtOfRwhxzG5sIlzENcOCAUUpgCSwos4FbB8TWmMwXwrIIsiBK5Nhau+ZINmUiqvsvRb0q4\\\u0022,\\\u0022iv\\\u0022:\\\u0022Osrlk773C1w2lGxyFMgfCg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225320fcff92adcc428aa91201fffb51d0246741247512f0bd0f6b257b94648794\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MbXXOAhUA0Wn5+e+NnbSGbEjBX7W56Q2O4es0\\\\\\\/BkCq81bZnXjlxzLC794YHn\\\\\\\/WbAOZTegYp4tpMCIHV4\\\\\\\/pAnoHmirfG5SwzS7goyRu8LtSrjvl1Vb3M4o08ST5wqILQgVv5ozGMUoSoKnUaM1jYmnkQYBaek1mP00YA+90JelO2fbcEj41UAz\\\\\\\/Il4HGuL1hopGkm\\\\\\\/y\\\\\\\/0QgrQInqkoT85S\\\\\\\/Lb6SCsHJuzezEQe6VCCuSr0twC4q6GUTtK\\\\\\\/bIdg1ZRmRzuB4S9zcnB10pbkfwx6JlKL8cS7SGQWWOv8vIeKMbeFx2Oc6\\\\\\\/Cj5PQ5unGN2i\\\\\\\/tU12cm3I2NBiSecl+Yis7zlWaUiiOsuMJyHZagU7nM2qeJPFjuj28o6iwVZmNpin4twf+Vcz9Z1s6tD\\\\\\\/VlAOKxhPWbrOvSQPgbsB3IJQgeBYOgzGpbHf7gw\\\\\\\/MhYV6VYCDL7cc4zJSbpliW9TC9AHRvCcbhyATaNPaK+reOg91usMJxDadiAAW8YkjPqQ1QGv6TGxAaZO2zJloInaBI8jkmL2\\\\\\\/97yzQrQqMDgc1kE9KMISbAEQbgi7isQ1qg55+aRyACsQB1+Nyk\\\\\\\/VZcGicFQAmaxxRWrFo9UIFSdcIKznLX2NOBT8nzZArsB+ypI39SiB6aOpsJc1qOyknFCaWgO\\\\\\\/\\\\\\\/wChBdWjDpUOsRZ3Ur+9A\\\\\\\/JEIDDZsr2suOw\\\\\\\/dofxlKRWUiqYSnfEnIw5KaYez\\\\\\\/sigf0f8FZLGJGTwNOfR+bcfI82x1srAk+Vqrr37bNv9Zut\\\\\\\/0qqI5qF00NxcfM\\\\\\\/IHIK1gIpMN3pWG7lxOymzMnOdPUAt0b+hWJec8kX8t0vhkJvaxmic\\\\\\\/Uwcaf\\\\\\\/nG+T7HoY+zXqVlEJvPqwgqmmcwoDLyfGNObC9mdek1xzBxFv6FZXMVARAhaUB01Of3wOpZcBjQqzToonck+sf6tHUwCgfR8iWwdRNq6VbPXK8z8Dx7RylGxQBhpBeruMptysWaMMUKSb5xAlWUor62fp3uQ5hNW+YpnRZ3gahJpuKlJNtiAiepzlDoN6gMoq3PNuEo0RfG9w5RGAl4glylRcZo4KVDWa4MNP4224n3OzOkFKsGRFFe542qK0kPEYfmyEhpWi414dxfAm2eR1dX7QMqn6Qu+fwc86Pt5J3UMYvn06rQaA6Bg7DoWi9v3xaX\\\\\\\/BC0gGZwTts6nw3wAe5hX20a5w+twVi2AhEJnIjnhaGUb68UzNv5L58RhVl\\\\\\\/EMTCNSKYzOndwa1tF5TqRgz55wwAb7wj2aLDMmbUtfzj+mD9Y28eCMfD9nFuScuD7VCz34nhi6ZiCqcOSRsOPe\\\\\\\/yEkL\\\\\\\/CmsMJPs5gC1MrhuH\\\\\\\/Mvg5c+N0nABD+7nVHr+mpFYEynCkWBicix\\\\\\\/Orxx1QoOqq1xa8oLMWTUsRTRvEroPq6Z2HtW9ymGzYXx\\\\\\\/lN6Wt1R+04GtfZjQH8wk+U\\\\\\\/pdGxnW8FjldgcrdqKxYHm5XnxBTbmFd48b4FgN0atFlPLjgec\\\\\\\/+8Q4fdZA4Bi\\\\\\\/MuD8uI4MNeAmNwkza\\\\\\\/5jR+jEhFWRWFk+l4pqXxIuR2+5ENFKhx\\\\\\\/8FeJfsli9bszH6OVn45k7SC6gbIVRb3xWrwnoryIr4TVJGAB46Of4vIw\\\\\\\/WZFgszSBnSSBwfi2HY\\\\\\\/YrmY6HRPZyp9vQ3TUZHr7OSTA7XrPlZ4M8ZjZmJjGw0ulqgb7TEdMWZo7MSjPVI\\\\\\\/mo5e9QpDgNHVh0BW5rdxe52hz6g5ecHcERSRd6n7uDwXM8GgSX2I4dIi0krTBIvy+TQso\\\\\\\/G8VRxUUt6KwrNMd\\\\\\\/bJa4TF5pnNB2hCjUz\\\\\\\/l9qTyZWVmq3tUgqQ9WoUeFXShyqKuNNl2A+F9IeCAjhwIuNDqKzkDb15LJ8OUcFkXJPldM+tYfH+JDqQXKM\\\\\\\/vHJJeKGhX2NacionXVkazSi4TgTIb1nLtzc8YG80byDcoEfqFXjyvp3f8XVD3uJGoMMw0XJ89bitppfOWDy6Bx03DkYLtzxmcEHFfHOZBDZgKZdmYw+DzfMX2+VrE8TuleR7W8UrXJG9osENnsAxsfXc3EMUGTwN1FlaCcVNLqg8cdFwK6jsT1G5zBn+sYNdAZ4m0TJaGLbAjXuFkGC5y7fawbHXLP7Xifo+uxhDPPRLQ3L9dZRbqh7rYQU2o5GGHRsTODrw+PT43TmyqEdQ+vA2MQeNo2iDBNZeLqQFT4HxhWmxVhX3TnhI2OBi9TNIPq1bbKEC91qa1F5DSwGWhn+7WF1WyoDpDUW2\\\\\\\/eBExO5uZKjpuzY+GVIDA\\\\\\\/xcOFhcYiBjp0PShBuTbYAujkdSVFbZV3YLY42CVL63BolTVK+9NBMWDQ1+I5mWPywrAKasskZRh6b+Mphp5oR++ckEdOBEc2lDjUcByaXE6WlQ29nmbZsrzmJevLxfUU7xDnnFzIg\\\\\\\/5lY\\\\\\\/rB2ROao4ZOVPezF9jeFvBxu0ar2fgSJU+vGP7t\\\\\\\/nLqzRy9sG7vQinj1kuyJKzs7IPcYPOjQ8eF0ZBzE523N2vdkYw4cmBJaj7HcCjtLAMyaeWzDyBIOHmlqCruPJZMFiGZRHVm2BPqBW2qmFEdgAt43qbZ5UFI0z\\\\\\\/YKFB\\\\\\\/3GWFm8H3VEAGiR8hAx8RNPpouc4VWYAn576oS+NTLlkPoEkTi+liPwKX6xDrgKcztE\\\\\\\/00PyqnRbQBLNI5fYEThf7eC6zgTn8btcyHNtYnEG358YEIIluDrMGAf03J3r\\\\\\\/HRYAC2dC47FHVRMoiCfCL5tMMWaYYw0YRvDWbAOCXuEEG8aIo9vKRgbVzP\\\\\\\/gVRUxs+u4SJgHM4iV5eHdjthnZgp32tSto9NjJka\\\\\\\/zlozSrIRaWqwO8WjBWobmSDo\\\\\\\/U\\\\\\\/GlOixjR+lXj1olWs1O4VX+m3ALN+wXAG7CJbzOcJ8Y\\\\\\\/BDG8WK8g7yfgKRCIayK81Yy3nO7tPkYBrCxqup1OxhbLrhB6CX5zZyal9EGw3kbRL8hZZQUuwwB11bJ+Jd+s8OqXhR1lJl90OcbI6QXnpdvc30K8SMBRv8yg5XUsEnjrBGqTRfh0QAFOIKZpV3aPtT76atVJbYK0tTW3h3koEZVK5MTGiObz79C+\\\\\\\/cvAAhoO3P1Og1LW2xZLuQfmCBvU2T4ZqzloNwigB6p4BwBuaX3IkrGSr2iORiSFnP7hns9YN9FlWhKO7ttyArJZiW3d6PGelMMIdOKy541gEU3gBvrvkiuBlWUj2rc+5kZaRpmSf\\\\\\\/fc\\\\\\\/4ZsdOKfl\\\\\\\/SuX3kWDt1DCYU2oOYN4PB\\\\\\\/Opx48jtw9ZbGuOJc80y6\\\\\\\/9P+baO8vxpuSv3Y2hy\\\\\\\/xWewVv1AmHrtQfqckvIlUM6ZrpeJTldk7CR98iVpuZ35IikhPop2eev8r+0nkI\\\\\\\/mdsiZCln1Gzo7C8XtA9tsyWrulbF2aAByPdgfKcDagp4sRAQqdPr6r+7DPJYVpwxG9S\\\\\\\/48cYd2Ri1roTQo1akUS+6uRDcXTXUGgFIyjl7lAR9dnnMNSF1eJtjTru6DbI4WUqvTWsSZJoAs7+Iubt2JhRx\\\\\\\/2hj69YqDikH5EfQQPKPPlg5uwjXzY6LOW0SrbGEctHMpzosdvDwTwOQMYG1ZF07zXj+BFLGGPzbefi6QGgBhAbmpooUCrRwcXoDC2LVy73UDJ80ZDeFAArNfsC5gQwWtgi5rzxldIATrkOdUGiFdMzhoHlX59I3rLGJXLis9+mNILPsDStOpDlaucYHNHKJC88wpUIuK2rKpJOjWGx9Li5XydkEfEYLXTaFWW4v1gkOZV2+VQQFMXSzVj5lObdZwVKJL9ACeFH2hRc8no4EOo\\\\\\\/W7I5sAJ4qgprfItJecKJTFakvUUuJUa1Z5eKKQpjbPjb9dEsPKWHvC+rOd1D9f83gI+gQ41H0jY\\\\\\\/IrPxav8IOZYUvnilCjptAZBgcVnNhpVNKg+XTo6WEdHH\\\\\\\/X6U93hwXK8llylcyfkYQS1dpkhhorX9nHXo9HWJHnIJvXaloMx+gIQWRR+N0oBKLvrb2D0QyFyTLbu1m6yg9PuT2DFBfVAp5x6mu9pYiXXh+PoqnYVdhYdsZfK4i9GUyymqNUB+tVhdZHLzURdDCnASZRmfQ2r0FygoyEwGKAaaOuYrWtmND+ZXYPgd20TOMF3e3+caUI1GrJQ9EVpklLR6JJxyjPHQgRXcFcWVoyMa3Uwp0lh35Ru+K6O0wi44D1Fzy+DCer2+xG4hGeh0qO3e0r9NO5YFK4tpvWfkzOT2T\\\\\\\/\\\\\\\/stcUWHndM106W8x55j7EkCh0CiP7YXZ413My0YamzPjaBv30QtEB+XdTWLBu2yneysqd7pYftn0uuVmg75U1iLVuAbESc0jcImVuyWJUJXTkbnCdgF9W6q8lF4zEk4mZ0av6r60DqnwGHDIJytUpVtMHbeWVeR07Nekda0BGfFFplL\\\\\\\/St5PEeUC6LAwx+Nc\\\\\\\/ZAfAYNLvXNiyU+qQBzOVlF7kEbAme6gAgHzHBilZcPn++PHUmxNixPOIDEheUfznBn+bUB3nhnfByKOiGQi1NbqLeHT8MNBBWXcyJg7bTmXu4OIKgKNkIwFdLw+YrJyh7gJu+SJ4sFUig0pYeVk7XubwQcQ0w6STHowquszGqQa5Cw3uMytQblDvSu9oUriIdx9Pt0BLAj7TUIW+nb6+QhOOp3YVGstfj81A3JtVKJcW\\\\\\\/d\\\\\\\/Ek8fmirGDtKcvP2mJBIkq5KTsTZDMn9ktq1x2trKbLwQHFa9baqubJ6YQLTFRUKp8uxAi4yxAWW1RCjj0j6AchBBF4cKorbhB\\\\\\\/xqiRFmcf1ds8m7A0vdiCAbAp3b3RW4QWv4xPRKzO\\\\\\\/tmE6\\\\\\\/GYXzeviePqiNWKP2CgWk4n+wzID+qkWHS2LAmppTGW8p\\\\\\\/7Zi\\\\\\\/zI95y6Nlm4JtID0FNT8E8PpsnSiR5V2n0oA4s1UQuKuWCM3I1ZKmFKmMYkBdz+L9QlK4yJ4f2XTvtoK+c4QQb7ngB2dP\\\\\\\/0Evo+Q2GKrKwQkvDvOkX3qG9ljWzF4SWdkDnIuRdMJ6ItQg\\\\\\\/98kWcAtCCyRHl0PVWFAu2TNFF+44pE1GZ8FoynftFZcecU1gNTxZ3\\\\\\\/iozrl6MSZwlLCzY4MQJAxUE9IZylKrhtnbbQN66ZvzOoXAzc+6XdC5LdgL0dx7jFgrjEpLZxePP3AdfsXv5s2SpOsSX3JgG2\\\\\\\/Gl+y4AQkmpd1DSLejnEDQ4ontwHyXboNzJjpeB6EFD3huEH5Fm8nnGtvKlF39sR51ZdyQJRluAxBCZpxVur5Ioflxsp2iBNcUzW95Jf\\\\\\\/dgr6DCXVismdQWRuKEmkcXSCK2ek4WpJW\\\\\\\/2q2gyVnJW2EjsvqNW1pfRQJIOCPPjHRgVjPtgQtrYxdbVHvp97voDRfiXKStttQ+EHDJWkM3IyqcssuB7KlT15KR43PW8m8WzdQNaJF\\\\\\\/twN4yfQ3iXJznourADLF8Dz2tn1Y7oqrcnyeCG6MvD0VICIwtr7huSxlebnWlU4TFY9at+5rPQWvHiatMutOGzZqEq380jMekcUyUDlojM4l35\\\\\\\/GZk0HqjlItRlt17xK4ndUcT+POv5yrcGN42keODguP0jE++W\\\\\\\/UTg8C5TI8pkC3FRDlMK+8MFCOL4yv6DF7aNcE2wPXLFOE9xPMwAqpPGwnO2mJz2Wqp8to69M\\\\\\\/h5YL6OenWF1991M+yfc01J+hyCc5J3weHCqXTJZk\\\\\\\/5R2Q97ywvuSdEcGKiV\\\\\\\/efpnNIgzr\\\\\\\/qD+UJbGN+hej7YVXdrzM3TOJaTdmpkrR4NimiDXQv+oJvPAo3HnYoC\\\\\\\/\\\\\\\/lUl84Z0K\\\\\\\/LqPyUMsYPQ2R\\\\\\\/aTqsYFLQ6QKNRikq87pTB63Byh2tshZhU03GvtEaYBYihlA6hEk6J9fQXQDMHT7VqslPA\\\\\\\/78yRJPVUXs7N\\\\\\\/bOdvp1FnNqCEVjCdApgPdAof9IKWFh8hAWYPmfmX6btm+WhjbU9gJtrV9qv84v8tlQtM6Oxz\\\\\\\/5JoUSJLmH1XHPJ1qjOp3JeLj8IPsTxtzZ5Pex4yfnUaqWPC3r\\\\\\\/gp46FgVyHXnP0VDt4sCFUQE26eMQXKkaLR8iKCjIvXIrbXLMyVkTNHUc4PYUl0Cq\\\\\\\/\\\\\\\/3XEhiMmrPvelxg77FnuEl6cIGoUBoABkhYQm+iPjkSQDa0ph\\\\\\\/5ynawEj8fzwLLr4moSnnOs0P\\\\\\\/5h8nwRVppr0rIAGSUnu05Ui0Y3Wjyr2wtxMXDHQk4GJWKUy5ix8ucKfrkd31nHPw3Fnwjr8rD52umxhCgwgQhExjNUhPl3I3OJO\\\\\\\/lp67UAKvsgW60vrhUcoJV8rD3IaIi+NMNtLtRCpLc7pem1iKtzXMB0MC3x0kvDovBtnCR8qdjVvsrvW+nl\\\\\\\/v+Gs2PxhPzkuWm51SiBB9lvVx5zYx2oyvij+UBsIaEPEXnjMp+i5ViKbILhh0uUzHCMtQaN3M9lp+bKwAP6OyrQ5Hj5RB0CtCOPf1HXSRBIipUYVKK98zL0qsJCXGWfX3scnxTBCdPm6n4R80YorGDIokPSjrih7R3XYIvSLH6qI5frk8aASuHEiBae5v+vWgRXAP8eGm3wwZy0RglRs9ZtrA4j6drS\\\\\\\/L5elX+omqrK85Pxm5jksgPe4ZohCW8ueLDw3IjiBk7t4UaKCKvtkrMSg096OHlYAcvIYxe04DQIV4AnCJYeWRRkfuT8Ia9s5Ex0XR1+ujxjM\\\\\\\/x1r4tAbPLUPgOuSqBgLvDNMvv8UVXnp0D5ippgfVbQLMBhy4j7VcJjN0FHuRBc8cd9LimaEOaYvaroEOaUZPB8dADIJS8dy5V696qW1+nNFHnuREPDVJ8NG18jjQpN8EY4+6j4sleK0KaY\\\\\\\/Eenw1qhF00dFDrSscWIEzo7VmPSFkFlIPFzQCCEbUnEjzqTRDGHYYrOLJJ4naPI1uMomk0LH30PUwUS5dRDU6N4runi25dLe6v2s8ceigCmYulZgi7PlihDghfGZXLfpZIXecQL\\\\\\\/cbPxyWDDFlmszfEdciiPR7GI3lxqvW40S6sXZz2LZ9xJzD2u+GxaU3i0ISZevpjl47x9AgCel+Ndq4TeWb4To10qo+7XRoCG71Q2Vjli2Si9wPD178bDhilbeD0mQhGNjGKqaeWv8V7J01bHzqgx8zB6JIAkxzMyo1TibJ5PD\\\\\\\/5rLRFtukyYpFE8s9zqUQYBS56LEnS7qBTK8cB1kUr9Mcng7z7n8LbfTF1m3xdJuAFgoVRAUfmAkRY8r+Hbq+Jo3Rq16YjVTvnaxU2jf4iEsQeUyTBBcpRG2IOMj99CAiZr6JEHd8fhzm9ATUJNjISLpYmHcdl19etn+FWUldxbGAozyTmVxGUgKLfetE2O\\\\\\\/weQBXaGBYn06u1yNiYrwytwz\\\\\\\/QapHZ6i6O6QR5Dgy8uS7ZNz5ammZSSOIvjC\\\\\\\/s8ABpuFgqlikPTNfm6Jn8ho0tgd4yWNMBAiYy3Ka2Tgq9dNhR2SfnZB6Dh4Tg7EOs4+q5btQ1rrt4THseYnBLecV2oeInxTLVCEKB49CtXDKPzQ\\\\\\\/fYBM1lfo0l\\\\\\\/eMErTu0TVc0xCpKerTQFod7O5wiorRpmCMscte096U8D7WGalHgAiHxVx7sb2witS0WpYPg63FMeXcbuv4rO5EHixrr7Xk6xDx25MncBvxKLYeE0huqT6xjcAX+IN4y7lriY2\\\\\\\/TY6BToWsnaiVzW0O6FUF4wWyT5aUZNsyQqTvVz49zzNoL1wDj45Ojp45eopeuL\\\\\\\/Ph8fTC+yvAff9oN02iFtAksrxj5p6\\\\\\\/LnceniasdkgTqwV3DoQTVStjh+\\\\\\\/0SBxpAyc\\\\\\\/SHoQl0bJwrmgX1T\\\\\\\/EjTkquKDkj+7OkwJPvtvn6LfvxC3tmflwcA\\\\\\\/QmoSMLdxv8rn+eAgPT7m68CHolqcS2bwZ+d7koA1WnPPK8HVKcyUzHqQsFaPt6rz74LwO9jAcCQqEESmsbOxpknBldVciH2z4IiQUA90nKrCdKlrfleyh0FRBVGdDcLppmbnRWLtG6jaMXHpC+n1wmK5fKPiOhp0a3NHtRCMFE+6dKJhYppEWDnx4ozFMN3LG35W\\\\\\\/f75Kr1BKf\\\\\\\/VuxsqgNesgHGwHdAJP3MgQZrQg41Sk51EPi4yDdrB5y6xvF+DcJdp+70m+Oh9ibjT\\\\\\\/rJ0O8lLOEtsNvmPNh1BGCJ\\\\\\\/1XNTpl3rXai9FRSefUVPXfD+4xyj4RnZjwNfxamgZuyg79KN4\\\\\\\/UrYAUgiPTMkmN8eBGdt5BTn+h4hIcnkatQKGdr9lGzlhYyiF618wqnngh\\\\\\\/+AZApW99lWTYZfe8snAzJu3Ys8jiWydOn1xWQjCaaYkBHDd\\\\\\\/qIjb7K5GgSo6nAakpBFVto0bxAp0zEC31d8\\\\\\\/kFZ\\\\\\\/DWnwxD3z4khN4or5BmCXbSiAVNIuneZfkJhjgw6x8HYIi0eHbsjQO2PaemX4kxuv0v5S5Qcijp6YVJ8iHx+c6daGazHH2YQp36xs2n6ti0CQ0FJZ\\\\\\\/uYQoEe9bq9t0cucGuS3YNJ91m1\\\\\\\/TJZLkltarw2f3OyHw+XJ45452c0CEdJk4HTgPEjq1Gu\\\\\\\/+gRm+YrqG7fVlvYOK00VkyriaUDOMwbsklO5mnLhESnXfuUxXWHoB+lD7Wl6eGHVdT+6N+AkoIUF5w6rWKbh1aY7GZnG+eCa0Yd+F+yk5P+xm5VZl3+SVZzK0dMDE0P66CgXZU5s90J7KnDNjWdaOCPvuyaLmVAvwZy2ML3bDOqMilIX4ep1nX8pdWKtnWrRlVSDAMscJXrltmgXRN4IuzgxHezMNIJT4W1gTDc8Ogybhu+HwMrq2KI+i5gsSjc1\\\\\\\/Jz2srqtpqgZZ0HjgRNXn5qGfQ5Z0Buewedf\\\\\\\/mIIWTMP4xzu2voY+WisEUqUUduTRxm6SO8GLBMdTtgW2bqZuWs1fnR0qG\\\\\\\/0T7Jbp7Q0\\\\\\\/wU8F7moeNLIJ4jM7b3yWNS7iM4Az0461kTSl0s7aiY3ZXrm4iLq4UfIU6kDm+hv1SO1Sc0seRaf4sVHQM7xzlfdRYWaqubMdPUsviZT6XJxL7WUx8FakjETI0X2xGUjCsThk8S8nAS32CKXnrqmST9q2d5aMPdRaxEekEeQi4NCCHDUTd7XWEG9SSUs1v7MeIHbnIJ4KBsMLf6Q2eeuzsM+idKZNUHcZJ2kXznDloE1bxX9YNlP+yasfmqTDoYQs5OMjMNa\\\\\\\/po0GIZSW82Lb82BcTN+iVgARNd+SQHGtwqywfHpn0YYXkOiXDONrI\\\\\\\/d50FGZQqJD2afcdjDJBVoHIoUq9spexhisd3C21EPFY9mtQe0R1l7qKgE8YWwkNwDpfWMhJ1gC1NIF4E8V4ch2kKvRdwo5wW6fdqazxfcN4ar+iKqmY5+ISsxr67+IJpdDxgaRn\\\\\\\/uYeqakBySWNQmPgHYpnzNfYy2CzWOv1CySYgdwnUlTa2TyST7z55st+ZCU+DK+nuhckrUMxUsIFWAc3VBGL4XIU4xE0XnyvpHM3jqdGGDKvlFwd8m3J96RfGAwQBILfrf41ynR\\\\\\\/EdJYc\\\\\\\/FHqzH8zJJZG9fueK381IuC1bwsR62MhxYQ1IBNNbtz7SRYcz5PIvFkaly3QLGvfMgT28YLN4zcR9ukhCm+FRtMTW7QmwsyO9ifvLuaX8pf2TNa4m0uwIma6KgiOsrpA\\\\\\\/uktR8F4K9q0M+r5CCEnaT8ci8agX8NaLCL0rAFE3mCFVc0monvRiPfiMxfyN8QgF36YIDA5ZuYPomzSdLcal\\\\\\\/9Ep3r3jMcGzUxhP84v+z+P80iF568tZvcXeTH0J9k7Y5IPsoOrocA\\\\\\\/W0SyBgB692O5pJSeGf\\\\\\\/\\\\\\\/BwM6UWYho1mV6jmSNs4IxxNjzXdBLwZCRlrOpGS2oujCFJG8av6B3adB0zXcHCn4DddpJgs5Un3ZGE1VFwVYu\\\\\\\/V0ZyLgSOzV51EQM\\\\\\\/mZvgf8GHiXpa8zb9IOZz7Y5nNA9nipP5Ag2oDQCQjOWoy\\\\\\\/mpIWaSuQezlcezrm4Qfv9aZ9MlmJ4o7h0nVr+NSzQptxSawbm2OhBDNkjxEQf2n4DPVEBb9M2mMS1kRMI+C+qS1008USv33BKmLiTobQU0COlf8S\\\\\\\/5Ce2QU3swAgDMfCweScaCNBf7NdhidzKuugOzslQCkNJRVdzGYzp5felJChNak9ehHtQJCu7K\\\\\\\/uK7VQB1nkMpyc99iBD+hr2L78bAraJCNat1lo7gpsMSsSxZfrWVQ9wdYMgYuB1PKFS3YVyMpJvK9em9LPvWoCG\\\\\\\/did9FaefGZRyTYfbtxJeR8N40AjL0canDfBXpJZyuRPJoNtrKSlCw2BkmPgDKVeLOWS5ZbkIiFJwK8Thu8PxRryEu959Wr24L463L2qGkxmCDeSKaX4HgphC3Pft+MVUzoapGSKOuUbUp22OVfY2VN7wj6W0YIuriluLTV\\\\\\\/m1W4QZPgIff2d2a5wK1FEiqoqgpyUyezxjAlLXa4i8oDHhwI1\\\\\\\/X\\\\\\\/6yuqUp73XNoQs44W9AZTtsZB6Kc5hnJTxS9lgppvvAq37E7PAkCPzjQ2h7s\\\\\\\/HXYyyHqcJSHZKlYg6QxH47c5H46pZA2m5bEdCchEEB4Iap5TnHG0HEQL1jp0ly\\\\\\\/AcLCMHGBYaoA7F7Fl4R48y6x+H\\\\\\\/ac2jwM+q8FtPAwZY5OwSmVZjz7b5NrimgISmPQKE12RqV4bAKlvfZCoCzj6Pf9PW9ZmfO3HGCuWCzZXxnpPQsoVflk\\\\\\\/DCvImkD9HHUQWfUJXPB3cpLziPhnmoQACQVjaLThWZfpH6mTksx8iPNOizNNt05\\\\\\\/PdnyF8voZChbNfLsD15QNHn03VPe0SVtHNdSzz8fn\\\\\\\/Rdbd6C5JGheg4KC3g5n9rw8DBDMvm\\\\\\\/Yu3VY7N3tGaWnasliIXiPRUrItHClsMWMe0Xb9XR8j4pdBZnadLsNYEClwjCE8SAYo90vNwdCRwIx1QkVOpy3ycyd5jYSAgC0OMa0sxq6N6OGJCVsYxP0cOeUtgzD0BvcIpj1kP8RIK2ERb4ATINNWT5Q3ovZhQwl9qaCCrCfUeDBJqaaHq1vY+qsz0xECy7dgRNw9edW407hH58qH12p6M8u5R3tdToIhvYozTiJXcGIZcIKD1Rr4IrhWFp2EdpKijA3u0zo9RqaazYpwRKSv0WzuyPk9HWwmCRdE2+N6jE0DAkWYzLBo5\\\\\\\/g\\\\\\\/7Yk2u7xlI9Hbt5bkRHXMigwWECGF4PMl86iQxVKcK8UBhkGp\\\\\\\/yvKFDyCy5eYcD+S5ZLB5GD6fD9P9PCHfERXB\\\\\\\/F4G5EBpKxrICoMABauNHy0xRv\\\\\\\/MOL50UOPxdB8AjF0ydT982AOvWj2xfUYUOKC01W9kTp5CG8MtygrS3Eo2csGyfjDQKit8UMYF3aL4UALcO4GHQ\\\\\\\/Dm\\\\\\\/4QECVVvp5eQKh+659mz0gVdy0nIHsRs+1bLK6NvTb9+RtCJaZISzoGMoAhuRmiY6e2VXWKs9trEhPsgpI9Iam2ZHEBo5TQGmUI\\\\\\\/7tbUq3aLOUNz\\\\\\\/cbGTHVPKIfepmz7HCsHX2gLCv8lJxmqWc0wRojBB9n2b1qMieWhayk7Qttz4eLZP7943EahLysVGc5a2dJTV7OSKPH\\\\\\\/bvDniyKotV9I15D4s5Bsh\\\\\\\/tWI9nNslCwCpY2sD160knvoEAU0kysIxJDPLvCYIo\\\\\\\/8zowf7soExHUo78fWFKacJMRavbUv9\\\\\\\/msypAjG4MW9R\\\\\\\/cxftE\\\\\\\/1XNiqnOVTiejaPdDsVD3hJ20hFGZMr1OmsJE1gQ+QPwkpzrUydLxhNQ\\\\\\\/axZtFucCfGqRjAtGEeb5oScr2jtNmKvYlSU43U4k6PbZa4w+8DPuJT\\\\\\\/fPDCdUVG1sfR9iyjZpwNmQ4P0Xm94Y5prksq2Vf+kmKIvtzA1IDZOAeIJu9fGRHtYg1UwqHJoJqSnGCg8N2sRc4TRnH9bD8QJdx6wQb42eS2SM5vH3luE3tqOgyYhTJzMXz4LaJVOid1iCKfmKKQaoJkyNTuFuTOUfCE4h+7GDuXXRZqrMgSLk\\\\\\\/7PyYJxaWr1JZE7Ak0I+yf3krZ\\\\\\\/VokUP24bAyODWa80ddyb7TaNGNHXY8YgB2Vcf1cwQc3PZgy+jLO\\\\\\\/v\\\\\\\/HDB2ZV1RlZB7IOnoIo9dpjncZvzY+iZ4NXfmGJSmvniopZn8Aap8QEuhpo\\\\\\\/aR3hkoLkdq+qIRPBROp7\\\\\\\/+uH8+dweFwdF7Acqb9ubt5JWeH5JeFnqWPd8muisshCtqdrgMDZ04g669OkOKVOipCKsIAxXVTLy+s+7OUw6EukYZGiKXOZBjxeByvBe\\\\\\\/laVaeM6XB5eUHeMZKinlgZxK1kyv6NJ2sjMjhdAG2\\\\\\\/CbNP4auSqXKMRgMAjDjdc1DdPo4LCQ2\\\\\\\/gqKdG\\\\\\\/2j2uA5jH70o2Uey32eui0XopLAgSh3HUZKNj7nAyJNaU1j\\\\\\\/2Yy1jfoQA7cl\\\\\\\/KWs1HutngXgpOgDXFWgSMi+pmxBpPwCltzowSwdRT4F0YRECOJGoLdFeN\\\\\\\/sJR5VudW3CGhuJ8z\\\\\\\/j+1JC8caKAtZGBnrwcwRsXg65QtZYzYEt2MVZTB9NfgZBI8aX8Dy6iudOzoDeCI18oyjX6QISdrfIIkByqSbe1sAt3NScXVZvN7gO+E4yQU9dCsP5uuSZHK\\\\\\\/\\\\\\\/4UMXSgIcYR2dM6jhdAiZEkuu9AUgXyN\\\\\\\/Ks4fQmcToaTiGydXYZJ6gDA3oaMoot+m4bG0KvgoAMBK3U1ecW25cCYUpNb\\\\\\\/vUk\\\\\\\/nDU4k62gUuXwVpWM5XFqjFLvqDn6owerrMj4TQ+q2hCPgAhDsfxqckyIsIV1NQ36EbCHByJ3KKUDookYrtzd\\\\\\\/784bB6igopta7gXqCaWk\\\\\\\/4hgVmY4kzgDM\\\\\\\/Byzp7NM17OQ1mvXw2D0msV5hTsDk7OwTCP38PhiVdBf9LHtmz\\\\\\\/V+sPh45uwcCJIpjBaZPziTVbvjAAxXj+HsyLQ2LojVI4AwF7\\\\\\\/eLH3yjOHDW2wqLOk4S5xVPOTlzqsGSHBzzx7PjkVzrQVw8\\\\\\\/1peFkKi5b0XzDx5pNC0dXdUtxjOasDeWiginlQ\\\\\\\/pbe1IlKtOJ0M6s0ROJHSUvpYBrHH9c6KIO5Q1ARlOn4uVenWNYO+DjgucsfVqdM6hUSFlQlrY+2JO4QLYp0jMAkm4CRkBa6iIsi6lVrOfGefDlfAJigqQgJlen6YhFkAQOiWk1sXiMrku8smEQ2KyvJPviG2fHORAhSf+iBPyzRuabQANIJfL98YLVt1LIFjt+CIr1yzw+ca9hYNluopnpkEIqLbU8MBD2QW8pItCbsHRGloIwoCbspWlZFlnJW8aI3U57bDCKdmcCPeGioEivheKEvBpNyaxeOJXJU2o\\\\\\\/jO1duF+w9IAOv\\\\\\\/KqsVDObRZotYW55oeBpzbRPZGCgvgEcBCf8utPovH0HeKVJ6aut+0a\\\\\\\/wQZxi\\\\\\\/XQbQaTiTb29Vh7BvfalZ4k5ERUhyCrqABXJc\\\\\\\/+FT3uKakYXuhZhp7s4VGUo6SeEoKR4GtgPqpZ8d18ka7AWPvE0NKP6kGp9KtEZyXdFlLLmKO2PLPl8MHsdFE+dqP3KCgLzqvYzONNhY3HbkGSZNoSCq24CFbuUhjtSE6Zzozm9M6MDN+umo+bmtORwMt6I45LgFBiNgaJefhk9UbNVWuouSn9tfG3Z0KQlLk7Kz9FWK2+HGYG43pUECSZAq\\\\\\\/O4b2wLk\\\\\\\/eHH+SBGiW\\\\\\\/bPzGmOx4xLBURpLx9VNtqo3MgLP8WTIDGVENRgzipvq0fHEcCy9RCDKgPHeIgcU+XXmbz5IGS4xk6SZxVjAen2vD1SSsdIfeUEusPBrWymnWVYsefeE9tTfUrKk0iydGY2Az6ltOMZ9e0xKsXIpwty1HeqHJViE2i3swycd1hPSd\\\\\\\/tuwGq8at+Be3BDTAh9nYouqyZSCJnQTKwMv73Jvtb5yelmSVXqOKJhBOrsMB7hyHW48KCz5i6cMTbODYiVRp6vVei5YJiQfmRb7GM3qM667aAlTKLMSlcGeeuyDjI4H\\\\\\\/OSdM97GN080617UOeDerNxlXH2BUhNQHDh+ImrFWhr0bTE9O9h6Iq\\\\\\\/8JPnI9g\\\\\\\/k4wxP5s7xsfRuKkJSymalu\\\\\\\/UHOVeAa7lR+tc0zKrM\\\\\\\/ePvosoUbwyqgkp5QGODPLMnM0T8IY8zXkAzb46ttouQVn6f2Xw0PisprTv2oege1FOh35Ecv2Rdp0igLdHP7hEu+5xcMtu7psEpv0vhhp2WQHJ4jzitVqD+bttE8haaFAbZSAaB4+DvVNn4ikp79h9dQMYzyD1n1gIBTnY6pljIGnsFLe1ULtxZImDtLMgrj1Gv2gTS54eza9jwS06xon3+J7PT+UdlaWIRcFQ9\\\\\\\/OV3K7fVu5yVHI78mAptcch2xKaPAiVjclOqeWnkr8w5KN33lHke6ct6iAibmK7Gp1mWjjgVaxWstuQBJ7oe9RKJbTxCV2NzXH\\\\\\\/qN03EHyGhvIy6sUO5qbzGRpc\\\\\\\/rUkCsGK5sgO3FVgeYXq0c6AKJhrTktJrkYQloV01+N3w4c6rHKaiYen7NmtcHJv\\\\\\\/3euUyRsIk1IAcXO5xOWMrxuo\\\\\\\/3aIbO0b\\\\\\\/oBgnn8b8m65SUqi+Guhl2Y0khwqI+1\\\\\\\/U683HX\\\\\\\/\\\\\\\/MxZQibTnmhQhtlYPw3A9\\\\\\\/+BqIMq+h0uueZQoivTgUYQ0vDVvVCu\\\\\\\/oBgd+Z28HjB\\\\\\\/GOcBP7zOE0FgnGGd4sh57NGr1YJ5OaS84d\\\\\\\/SMymrhXKMswOxrNOiLOtlLFD7DKCh1np7IqDNiA2nNEgc\\\\\\\/DYZTH3rn6+s2IfyarJfQgSRjqGpq5kVhyxsWWYGOvbAUelV6b9LcxhIuaCjIeVYbgz\\\\\\\/ERNq1X+MeO7miCHySRHOf8\\\\\\\/qtakNLtUQ08SYbowYJPWZMvWTiPl2iqYcIcK9mdAWH29oNfASb1zz3tMXMxkUJOjRdXxNbh7xTnj1333Y3vnohxXeFMyO7fMuSDCywk48DOl1rUW4qnCpekr4FfNXsOHHdFWmM\\\\\\\/DZAg+5iT7J5WSCiOhkdMkqbbqy+7d+JcZk8LycDu3fPCoz5FYLFBtLfzIXtGxWLf84j6buj\\\\\\\/tkzPfvHS74daeN1Vm0plGiqWf4RxFno\\\\\\\/q3E\\\\\\\/b9o8miO\\\\\\\/ClcplqxKaS\\\\\\\/liMlIZzLpYJPvzvpGUsZakRQrRsL6ACvfrzeFwcnX3mfEpPDP+xOYEvve0uv\\\\\\\/OQkQp48SSsNa8jVeX0vWU2mO3RhStWibnZx2mSTtwaeY7GNbzM6hiLi3+um\\\\\\\/ErzIy8jijQpLCW0JE\\\\\\\/kZFAdZY+gyNRF0drUVlTXztdqnyGgOjiw+LcMthzUNdnkMiBRrSkpBAzqm\\\\\\\/e3db51uYGlDJyDVinY3LoV7L8wEzVaWcdYUNTpaxAmBP0X3bqR2uoHY7Qopcr5IobggZEBkT2EFzez8Lp4sPJBQodFzp7gwIKHWRvz98mQ+ohb29\\\\\\\/PSI0AK1PBfmoymXKNV3h7A1mfNREf\\\\\\\/izpMw28Nos6C8GsGIwbagM+98uJwznnVtTG+JBEmlFLCLgGlaFWlegk1kXsEQkdCNnaC9ZM19\\\\\\\/hfm3VV8DmaLgm1O4ve5dx+ZHUwhxSLqhzVjnP8qtjUxaB54SpfNKBWOtKKeXdkSmK3YaWBUxkfhXK7t5rdmG43e1ibbblwjw7N9BZjd\\\\\\\/nq4H2tIRbWW8IPqFWLThd2fJsBsygtEunGiF\\\\\\\/BMfkaqJsPH6jwX81o6OgbLu703NTZ0X1kO4cjVLBvQi21JB8XjQvhtVtBKQApfDQp9sWj3jSDTUuBRjPVwOdcEGLy6hu9jzb8jbr\\\\\\\/irqs1G5AGlN2mLud0LmZ5apbU5+\\\\\\\/hwONyl5lRVTcKOiXuEe+8MMaZc\\\\\\\/\\\\\\\/4xh3\\\\\\\/tiQw8FUBfdEQ8ype\\\\\\\/CHK\\\\\\\/757HY8dahMR1pHwUEkWees9x+454Pm6bYqCbNVyZQ7rDV3vcIA7oBpBVRAXHRC0+W26+PZWAznIzt8aFBjQ0JLYESt\\\\\\\/GSzlJeSguQdp2nY0ed0Gi9vpZjKX3W6ZFJG9sGhugE7fnwmyiiVfhQhSfc05EyvetZxWCvNVjx0N01pCfj8xhlDGncfPrhItPiJCKj980ByDVRznni5fytu+4\\\\\\\/FymCRhZjNcMX2EFQD7d4ve7E+CIAicdFxDjKkqwseH1VxMXi4gqSDTbZQE3NSB5XjU35amZgCKGI78ShlXLLnkOjFPGF9CqJFvR\\\\\\\/\\\\\\\/Y0dvswkSJOMhtz1lxZyX3S\\\\\\\/3xxD8VdQqW+vrdB\\\\\\\/Qiixa1kYc5BKyj0tGbPKqn0ONSkOzC6GnvPOsv9ZmfXxyKWW2autUOD5U4Povekf7rNTwuXz+q4knGqbW9DXUhYVCuu2HZPrC2J3msyE5QHH9mCn4sx6uh0TLv1nor7Hq0X8cllvBzdMPAdwdBwa0J8VQ0oT0glDluOUr2WHsEP5d1C1OSPYHr8hEnjR55wjGvW8T1XPUi4xmij3SydEIa+91qb+UEs7nGAkzi+j40ReTiQr5nv\\\\\\\/uv6jxeYdsMET1gDuggwgxGK8saQ93UOzh4yeta5eKUAkNIT1BxOev0Wyk\\\\\\\/3Rb+Pmt1c09IV6ao4KkbrZvt5KArjY65oafwbCxLIdKNeFTRpGndA60sRuFbb31+H1RuRUTRnuutEse8rIjCrgKko4O1g9P\\\\\\\/opWbBfgCTrh+7TS7A9SNMImud6cYtDkpYSFAL2EiLAL7btYVvCm+oIMw88FlyYXk5oa2UIM36p7eTU42YaujDmgn7uFTQPxhmYdWhE1CacePRbmTxedgeW6o6qDKAlLz8J9SgHXRUgXMrV8qqV1qva5rxFJ7qKS3QRSorRKzC8K+Pf1uBTdZQyD7Ai9yB8nhRLXjqmAcAALSVeC5h\\\\\\\/yoyIFxZBoKGD9pdka0m5nUnItcIP\\\\\\\/tr4pk7mh1Kdxn0Ldf0vpOqRl8ZkuSs5GVvBDPlDZFQ4GTqszUc9gVQ7WerkeH9\\\\\\\/Nho5717qN48S4nXJUcH4\\\\\\\/YDFgmU5QRvpUb2S1xmCrOt4X4qyZ7xD+IJTRY+9scSfe+IBL8gsi0x4h0bUbdRYawjXGpbvKvGMiT\\\\\\\/xiWhmTqVml41dnBzA1ihtEudREe7MVBtEsDCrWTiFQ7SYicMSWN3LHI3y+ldCse10bwtWvNL\\\\\\\/KgcnU+hfXUgW5di60Lz6Dig2xr\\\\\\\/Mak7+UN7jcopKpCycnwSchMo1rrx7Bws38n31245Qn4hLBV3QBCJtXmb1h\\\\\\\/HjBlfP96\\\\\\\/FuTwSoXOxsVo8TFO0vmA97YuFe2sAZvQ8k1OhYtzUqUetjh945dxBc81QIdvjrGyeLWgd0q9hUNBk5TW\\\\\\\/3owKpeuZhRuGWXq2csVGq\\\\\\\/0ZyatHebA\\\\\\\/OWqosWuFW1tKo7lVqL50SbBILAVDCjw3C+UqJ\\\\\\\/7sgcE8\\\\\\\/nf5gJZdhRqXUNkAjwjeL6fjvtoaUv9ZTv5g2DS0CI7UT9PAifAGHzcYYYxQ0SH\\\\\\\/YJjMU5z\\\\\\\/EXUapUe7HRAwCsL22qNnALxNLhRQKJDK21NYyflyyGwZpcqSGgGNlu5fAqHpNhDLMA7DWuwnduHQdQrHaYdbO8fF98MpcY727xZgrVT9NbCM39X+S1yo924Q3dFmJmFWD89D\\\\\\\/R\\\\\\\/ZejLscij6LGmXlxG2hHMBRN+dlplXqGXyoliky0RB8c9GzwzidB+XYykTO7lpAZKvK\\\\\\\/ISws04VA8uOsHPygAFoPmZDJF5wa3sNExZOMTf3FNqTBbn+2bCvja52vGfio3i8PitIX8mGKHTP+jD1uLBWc6Uau2vJq1l9SKeHhkdW7pO2tIsFmMjc9tFo+tTRLemq\\\\\\\/f30dssY8q3sv3ZtvvsC4rF74ooQOOQS47yJFIltZ1xy7IDK9VVlRqdoeOpqBLMaoTk4MtIWq5d3RvIADQxvzE4ybWo1VdmoQZFQA6eJZpqO1LNMz7ehPDOSXXZCe9MT2DQsjcqYCYRMn+bz\\\\\\\/VTqsgJQwFHnYfICUArrs1f838MAynik7F+4KeJDdMq6R8gqlyyUz294E0g+EHmmZdCLlkU6+BBn0RqMG7EqVlOolXX0uMqS5RO+gXrYYq+c7VDlf8kTHk6acwizwd8D3Mui\\\\\\\/3PtlogesXhOFrc+18GYH5ytnSEf0l1KtNCrAnfVShxI7fW0mPjXLSy3vtsP6pm+LpfdC4DxKahkmPsbeJ2dDf+rfahI6CmEQ\\\\\\\/oVOrR0YldnCw1m\\\\\\\/\\\\\\\/Xy5qrcz6ruFcCsghTOmyFYyS9CrYQf2C6ysIZ\\\\\\\/KGxndaJoL9fMso1e+t8HmVmxAfFU4ag5vhsuY1cXcubiDY3ARnhPZw++w5h5CbTMwbX+3OXagQQwBJVYrnrPwxTZmWBkGzaTQuYgdSoMQ00ZCxH99Vqnkx9BXcU8457PBpZ4INSWHMRB2uGsKk1\\\\\\\/5olic1fVc7YU6XhPHAeDtoYc6apNfcCythCV5XEkwF6\\\\\\\/lcu2EEe37ueVMBtrgnf9a+g8l6I82HT9zDysBf9XkJbvUo8L9flC0ku558y1Jv83JpBIpmGyxr8vK2el62sh9Ii+MTvlpZYMo1TaFt9sYSdfh\\\\\\\/dJIZbhfpEDxJvTf84jsRclyeljea1BYbysXDbp5na8cqToRnLvOJjtd5e+khj5QlMPvRXl3mMoiqsz6xJqbt+IqmKfPdXWlG+mhEgO2dCZKE6wQWz\\\\\\\/v27LPnR0\\\\\\\/QhpEyGMT22AAoGNsWJ4ty+Y3VG2dTXlBxKMZESs+JMAVe\\\\\\\/YNbSTbaeX7Ro94mNnChSTbJ1abXBZdv1qQ16HWRaOjzwOTxUnXviuLgny5WypOX2OiOSGv+B3v8WjU0wJ97w5MgfGUyi0k4a144fviXWIJsjZYJNAmYJft1MrmKvxUaXDuQ5dAy6+BNzzGAE9RstpGT3Qm5fmNTwJJmrDSYxFL0OCoyMsCbCRo+RG19ZVeUOuloidCki8VH4QhSSafXTyMYY+sJRMMa7b\\\\\\\/qTHNEtE9XlLjjAICI7zs\\\\\\\/mv9bDIZ3Bb1JaERqKCqHPE5s7KqrJnAL1Ty8HZ6rXjTUSeO5WlUbMuAcfhNF+Cfj4cWdaCU5Uwu7GHjTudMU2J4Gk7u4lXKIIKuxG8XO6uPnKQteA3Z8bgs0k\\\\\\\/xgBEkW1QUc+uOCxbZ8j3e6RVYltACBQXF\\\\\\\/s34bLFWIvw8vAU16iRX0hvSMLH+Lhns6fhzbZfe8iY4po3FmZmrw1seifAcoMKabS\\\\\\\/jq9qAm9nIjET7CAMNl5izW2WLzFA04mJXgkUerafsjXfVVuz\\\\\\\/mmTm4oQM5BoGyoS6UrenJrWs8cN2WMB9VqywjDzjEeqFiQcXZQd4zoCSpPYOfnOFDpTIbwS6HgPjwextKCoS6Aw4UgD0LpJwdg2MlqbZwBkpSJhYEwo7tG0ujDuB8NwjrWdBYv\\\\\\\/TXqJAp1\\\\\\\/bNI1UEZrT0zCjnluP5\\\\\\\/vRYMtUpVuY6TFgZGjYb8w7HsBHVFSQpmLDHZQlICgdRVjfjNt4+QBs9BXJs7uWrW0ZFY2RH5cH2n24YpPRoZBpXlEDmyP\\\\\\\/WXETUYCjQqPcWr0Km8rKN13KESewvsLeoPhlmRaYLl3NC14wHLVQl4AZYbEzxDxAGlScq7dmqWg8p5Q0LN1O1wmA1hFxtyoAz1u8FKvnXqffa0gUDyYpY9mlOS+3QaWVC7xT56sx+7HyMJhhuHpuAeMeVyjdUfBMGHTBlcaGMOH3FF\\\\\\\/lCrn1+T9LrOHI2SaC\\\\\\\/fUtuROJXRxpx5YUszGXF73i3w2zc7JVoODVZ1l3hl3LJfri2o4Bxc8Mf+vdiUesvcxkrDNn+It0i0QUXgKw+yJsZMuGBWn37UXgR4Bgsz8Du1gv\\\\\\\/yvoE5DDcMQuyEINR9NeLXmkcbS+rb2QKQwkqbDIQWF7o50X\\\\\\\/F617ALRcKtLKF8wGULYAsbC50jBFCHL7RmaVFCNWvXvWiao\\\\\\\/DJ\\\\\\\/xuYoh3Cp24+s\\\\\\\/v+YvHlHdj3aA6cE4yH8pOeTWQg8fvuqadW8hhzXYYOCvOcUkt2xJwBaBLIKU9vQQ6hHj0qiCAJ50QgocHAAC2Q2DYjK1dsZUVKdbViTOpakCSo\\\\\\\/zh45nflAai2ErhjK3L9znJm4DZ9GkTgocMY9E63+z6kE0wiPCdtqrC8dWzazF5lke6nutRG3JPbTXaa5uHqcGSsc87jQhOTPmAMejS1wZIDPGS6dtzNXIC5VLTDH+lQ1qpb+2PNAdTIlGBefoCVOn9XSNOwYYOWSrXRBI382dFEnDbEgYsoCddgH7tKuvfRrAcsjflr2fhMqGqfBpSZLJYHcOv30s1MjJJ5s+Nd7vcZTQc8smZ3hWy99vOJOc3rP2Ip3feiT3cbV7WQg8BTDo6yBrNJNkH5q9XysLsvM\\\\\\\/1JCgOC0pTITwe+fOhsy3AtNDCYuB6OJJC1evaXQAT8YrVsCzBHI89zG7F2o0Xy4qGSwBBf0Kjr9yLX1LwFFCjSdYBFsrRJm\\\\\\\/LgaLGXfizIIII4fLLhxxX7DCoBZFU2WZ7mCgQIFs1Ro1Ds7nsFBtSL39EDZzivUn3vW0Ghvn9oIznHXjgA4xw\\\\\\\/zZETEF7bCfzITKNgi80W1QepU+YGYdJJUZvT3J8LrRlkzVtJuJmxU8igAata47eTf1L\\\\\\\/pVEUNiN22M6jk+39MYCI6RTRpi\\\\\\\/3A\\\\\\\/UbSssJv8iEaG+hPHvOvIZcYnurDz2q0918obGFutuQOV+aQCGMv+ICJVgC1CKnkzSQXDF9zG26chM8UpNDgzg7eYcL6Odh0boo3SE+8fBgJWssAaDzZZhhOtxTxltMnlanBzBHR6Og\\\\\\\/FOJq27LMGkyDA3rtPgM9VrKKX3YEigctygU7lMuPMKGWe+WoTbIca0FLdI9aF\\\\\\\/r1DLiL4U5UrPNUo1kHUAH2Sr8oUUDTIyY\\\\\\\/tiDN9NCwCT5gsWo3iVErJSclqLPhSMnNe+kZdT57JDLnniwU3udxpNspqAUx2C\\\\\\\/7PkK2NlEHfVdAJMhU+mwA5tw0lQliid\\\\\\\/ghH11\\\\\\\/8RCH65YWXvU0oPI3XAPTqoQw8mvs8jE0GWpqCHkNQ3nKHlyiN06dhoixfpAVRId+6VKSx8zB4zOO+bSfKNn9chBda9zAlmRouXleyIFAH7bJ+XWaY8cTGBdpyf7IcuDxyBRqoes6JzybXwYNrdUxqZB7sBVudr19KUuarOqcVrvB64oRG+KQkFrMIrFZSYV5JJzWJLUAyj8NHFMnpTCq1zOa7RfTMxymggwE+7R9guW7HKNA1xmAdr4iK8YZX8eSM2KzVCd1DEMcXt9+iG7bbrUKrTa9FWWKMeea3rgeWcOGWn6dIUBQMDTDrppfj2XoruQv4eoThyMai0E1GAigKj1YFHxwaN3ADrWeTK+TA7lb3XrF02s\\\\\\\/+SnJ8YVTB3+lzXolmxL1i4WWYbTZIjxASGS2iVOsyPLx4jhBA06\\\\\\\/8A\\\\\\\/DOHZZfAW1hwoEO7u6gsQ6EFTUYK\\\\\\\/+EdqKTVb+1fFUli3VtVvSmP5vp59J9PiyXzXEZf160WjarWbsWB8gIRw1R0k1hg7HeISZCq3RtM1wpnysUEp5CpCRNGEgst3Mer99bByGlfYtvQ2V\\\\\\\/jlKRYzWoSimdSMHcg2bgkWWW9cCMv1sZDQ8asEZU\\\\\\\/7UFlpeRGTVu9wjcq6djA\\\\\\\/8MUkiFoPVJHTQBc3HkZACV6DGttn9dqHy5bUrddEHPm7lYi+8Qn\\\\\\\/XPpL5AggPig5AnyBucL6rpcJ3iENxpxLqmUsVYFGQn9S3p8nOO6yys2XZXzzFpmZx+iQwQR+V9YwDyWYbnXhjvFv3LDjxdYxLwlVQ8\\\\\\\/CtxkxGc2Rz6AGDvxqDBq3enxIXpp3vy\\\\\\\/NNPPIhjJ41gGFaQAZEWc7EViQYSMlpqTLpR0huxmbGb9cmOny4bL0e8+Ls3ObjSj3r+wKn5a2fjcOEj2vh040ZMQ2mo7qimpQ+CrNfNAtQ5aHaTrMrpFhgfBwEEyw88045FHAEC0ERAHV2+FHxiYXlNom5QASiYt7KAPJ0tY7BJTrGNcVj94INuQxv+L6qzNblIVnKQp5E+Ye+44PgDGd0zQmDQQQ6wrmp\\\\\\\/OWy\\\\\\\/DZZzBaNMjQmv91dZeco+WMo0rwm1PDrhQdsPJB8YGjyInDZ6xJy9lNLi9ZKh5YQooqp\\\\\\\/lG2LBch0VRkalo8OXXqFqy5RCZT37GbWeeMhrjjkMPtisM+Bz1a27AVs2Jexl3q0L4fvJDy\\\\\\\/41XmcFhMA2CebJThDFRCDlh7LU4Un80tEncAR2RtsdIuocpl6H1ErvFHianwCt0nwy547k7C4YytElytiV\\\\\\\/G78L6MFFDIVDUDYRlIyAid6+PatrxVcKtS+1cGXc\\\\\\\/O1HsYFLEGjPGABD6hOmmr5GQAb8KXzdzRXLMnsXNN7MI4xyrlXbtXexTYkCJLkRPHF65pZlhUr3PhBUsIwffe+xsIvcHAspISn4KC3rFOfwbKeDuqRHroFp7DimaIJSQZM\\\\\\\/iuTxWp9EDf9pcsZbq9s8Sh4jHz4ZzoSOifFVIXqLKjPXJNWV0hd2e+88su8KBBz7InMYQoPkpuG45LqezaNId35MPIMO21WwLUmM5s0bUm+nFTGhzwdMK8zNgcZb7T7Twnnw1jMu5FxuEPUB7tY2rEfo6PwDj02y9MhXjZsJrsIh3dTlNeWZlg2JU1uij7aE1NVluJK1K4XXJkqH3Q3nuo+ArfhYgudLYE0cd+KPmkmLoBF2Zonr7f8r2zqiu9w9jofXvY4bHGqOPymHxCKtBOxLUOVQ3Uvw3TV4giNZqWMz7kAcnlQR6N03OQk3Zb7C1XFI3NmxgUgpLTwVO7qcKO4uFl33PVW6WyhROB+uMo0pUaBNaM2z0kETwSJZX\\\\\\\/9FqgZwRcZJFgZQueDemA2HhqwRNDp0PN00Vuh2tFi4t7QjUC43xCrBZven6OErRontvJ9AE3LSrj1IgcOF0csyzUE3hC3JwKFyCbLpKJs\\\\\\\/FiNPP4lv2rkTLW7V2BzUdrUU5fhM3XLlEv8LlAqEYRLOP898T+Ss1IteTev1DbtEGfLmIuxx+9HuTvAbSJsmN5QCVfCsCPnhMgmxLooIoGT1ExzRlfz0oR\\\\\\\/mLNGtMcDCmLoJTeWf2szAbOCpFVxvvxnhyKrJxzygzjbmWqE+5ylKXN51CMqtbEOZD1HstDWPQOILg0W9Jt39\\\\\\\/UywPUmWScZExry87v2jw6+nfNR9rCLwJpeADyUKkCNUjNynNGyw4KS1FFuxnfFF89T1N76fj78SP8hHC9UcgLTUzKD\\\\\\\/I5EVH57YLROe+gbYOwkVWHGga2YzGScIP3HIouFGPK58yIJvw2eylsUeH\\\\\\\/x+7FA9EhQbTiGEZ0LdsnJp+fA4gRhPJYvyEKiwxQXQ0RwO3rUd4vBwjQu\\\\\\\/p\\\\\\\/Vp6kKCt3sG+mk9kGRoF18m\\\\\\\/xNvuXw7ZbssKvfGDmse3vW78pk3LNK6k0A5pfeObfB7katy0ul44ES8Ab83t0LVvCIXiB9g1OBsbqD8uLRCADpybC72fcYwyT17Qs4OnrzJepm7iI5U\\\\\\\/Vf3CC1InPiAS9HytNrj+2vjOsaSGMM+73L6uNpcl9X5WXimPbEEbpwGxPpl4\\\\\\\/XCNc5xGqtHMZEYkrA0fQsAYdrkXf4tgcraqk6xomlhjAsXzO3hd5kvnZcjrn+L5YEoHUEQlpx9xngf9yCDsKZ7ccuTpU1F00uk9an9bk0aUWdBFt0\\\\\\\/IgMmHnBy7CwY0uIdaFrEX+xNfOrIUWW7+cMsq7zzI6BrwrRHxAf1NHAq\\\\\\\/xyoebkKVvDBcOQgFAm6EX29o9NIGe0U8l0fCgyZmaxr6sO8Zr4b13NtR1eZmQxDnrTSNLCaqZExyETYmugd9vNFPOEtyVWRj3G0F59yG5aiptjnpn244NPvJuK9mmt5Om3zD\\\\\\\/NTsOHw5hFlA8g+M7E04x8MZNHJOFSWHpRwau+hwfxYZvwuQVo+\\\\\\\/TkCogmaOgHchsJcteHX6y4yC0nnIdWFyefayUyUCkJZNCMEZnmYKGwAQYvidsPv5Zn9U0wtIVXQwG2Mwr7Jo6CpxJ1eOux23dIGj+B2VYe\\\\\\\/uALHFLg5OrHmFCBsVxfOYJiicD7d55Vtk2bhdvkUQbL706fF0\\\\\\\/0uXR4zs9tVJ7W4dhqQJ0O8pkZog2Bmjy1mFab9liZ0hg\\\\\\\/jDmolVw8HkMQsmre+AFokHxyMA3V+pEeLYg5lQUF99CgF9V2sf16zK1wYK2pQSMijH15EHC+LXgE3FVkT0aNNvdmWhPPkVrc3wA4O8hxSJ\\\\\\\/u3uZiR2n6aZMWLK5KS+dswR3K4dRlYL7kWfWtjCNhnqlOiqp2yXiVVi1oOLCJuwE6Z7v74Zmx6V\\\\\\\/Sfds7mOKvhmvT2iIDNILAIxgeNHrGPdOa3q6AzbLVWleufWYuyT7Rv9lcB3YqbGzkEG+H8czV0PXf+ZtUJ2E8Lbknb328N7sEGjrgQx43h2vsNuj+Ngh7Hx6G1yVqWFup4nqIY19gYnwfPRHEuqR7WXGGRsKA6EJjIyMRm3xPpLa0pNY\\\\\\\/vtFtr7h5eNCi8H3P3bDK0RJKnXaIMJBGmSTTPsLdyJTEYUVlV\\\\\\\/LBDaZDf8x3SwixxcV2zv70AlJvRYVZCf9zYF0\\\\\\\/APEwmJ5UwS7dv0z8AiZYKNE+z+k6nltN5mQ9lcWQ0AF4K7kRHrf5sM\\\\\\\/tJ+soMxABIv4CeMjvVOpCJIeZMa8OC+q8nd80z+Qoy2UYKE\\\\\\\/E7ISudB7VrbdUmCbtTSWaitzmWiC10DgIW0fnzQbcvscOmWw6ROhNh7\\\\\\\/3UC\\\\\\\/jcvreAghAdmH9OfczZ7Clu6NIxVifug0YbTT35k766hu8DVEuD\\\\\\\/R31h8\\\\\\\/YDAhMUB+\\\\\\\/0wjUJWHXmaq5gdeVyTnG6Eia7FKMGneJLp123iEiQB2zOYDMgF772CgsCaDoEHJdl4gi0q\\\\\\\/e++9NaJkaKAZ8AQPkL6nR2hoBNf7t++Ychk\\\\\\\/vRTAk4CWcVj1YWE+VQEbpQQCuKOy6TySMQzD70Ky1wuVZ8IskCFdtEKDSQPHbp4yoMY2BDzaecXYG5xrN3P3MHIAEGKlPk4k\\\\\\\/XMf2vVJU6fpa6kCl19vJvJ3yqby3dlxs72rM7HslKZZTMGVV7Ly9H89inWnZDCwKJTHDbXAkPV+7LimXBVI7iErlCedkcm4xaXXr+cnPA+aoyI4iubq3NPAYqwo4p0\\\\\\\/diWHOyooUuPXfb6wq8Qa72Q5qUJV3s2H\\\\\\\/HJHJCY+rWSDlUqRU7CkfpVjqYCHoaRxJhdY2lM8\\\\\\\/1eY3uwGr4Xp3\\\\\\\/anBGpaO5tOXXLdL4M9l7SQyAxlnc9aTNLltE4EQTon01IenX5JVJKSzYd05FkC2UHPWl7gnpy9MFj7fe8QY3vc2V5bJhww7CMLnrk4aAFu4X1N7C3wrZcDsBXPR2cepLENt\\\\\\\/+zxAT7oBacp9kUJ91TV\\\\\\\/TASVqCemE7FqEecR0Ug97paz89QUizGrSNsqXScdxPsYglkLXhz7Kxqkp9xU03fEzOs920D9uw0WtVUPNu+e0V4SojgKvRQMAlh0TRa5OcbADkIZMhjxPgZG0NLkpJrrxb5X9qi9mUJN+yH2taZ6irOL4GdIJwhzawPEmsHBz8UMGH8ZUrSSFkS1anJIHKY\\\\\\\/+Nr4l5aoLD\\\\\\\/fyyhp4AX5XlVuzjuT91D2ZJiIFcpAsfTP\\\\\\\/VVerwDGQVU7tt0EXyRWNWhWsuafpwyJbCbHv9PFZHLFtI\\\\\\\/YziuS\\\\\\\/BwP5vn8Uqn\\\\\\\/TqOUfHB92O2JZWUZ\\\\\\\/u40WLDYlJWISGC\\\\\\\/wLxqLZdhrIrJHMXUcGzRMkpZeZWPk++EJJBVReNBjYZS5butJUv3j68a4Z4Xdv0gORXLWuCrWzheD5\\\\\\\/jJw4CSyQ\\\\\\\/Wv1PR54\\\\\\\/AfZ7kfli32tItLcjKVAtSSV85wXbHwnpEFNnBMuP3m30QeIpMTimrmyDFo7S5TBvW78BMhKrkQ1\\\\\\\/VmTsi\\\\\\\/\\\\\\\/0L7QkM2FbfOZf0E\\\\\\\/SQYhrhOzPMxB4wUo9tFYn5WxYbfgu5bWSjjvYcNcZyOG2M2U2mkfa7FWyff1EiaO08ivxltzyhs9yf68iPeTJZKozNOe6AmgUpLUOldccRKCCc3de9E4tASTrzpbVar0RQMOZzC+87qP9YQRkubj3tC5H6+dlGoCiA0myV9AjZ\\\\\\\/7GDB+6Zp\\\\\\\/\\\\\\\/2bf6SBog4gsto9e48gejO44jxMAQoBiA54Vf6piP22RLXzc62g1eZ9J3Q180wgmjAJRFQ8T59eY\\\\\\\/UkzbUhTeIqwMaPhSBbBTvTIvKkZuCNSSOPhNsxWzC\\\\\\\/VBlPbnkYEVXCQClhLPwNFZCGYTNe+MiphzI9xzscdFjofQp5JNASo7kgwUa9uM73FzrOeC2Y\\\\\\\/UACTMSFEkiVqbYIDxqLcGBeYXQLT87EmSXho1o\\\\\\\/yF3uN4MwOY8iXgb4vhOVyTWx5ChB\\\\\\\/foQNdsf6YoeQU+H\\\\\\\/dY6U4rUeYJWCDyNq0YdW\\\\\\\/tCS7wWh3sMbGEVvYvOE5g\\\\\\\/ZEzf5guP8GnE27sRFbzJXgJBSfTL7BJrM1ioyPstq99P\\\\\\\/7\\\\\\\/lLlKtKfQmapZZQIqfRAyoDMSttb4mSrW29\\\\\\\/VHbF\\\\\\\/NblWoL4iQROZrStDjQuuUtZ5V2KVJO7IRUpP7EC4WscQhpUWzmEj+4iW2be0zItXx6Ri9s7ws4ucR4HbRvS6mI958IdHPNoGCw+SYw31V\\\\\\\/U0yr26pF1sGzzeab3EETA1FTHHnsK1QveWnJTTVOBkJ0X+up\\\\\\\/wVmqWV5DsMsncqnRT5yshTQAJHr\\\\\\\/BuGpdAXel3pUln7d823o0zoHGyPCj1wFvSvEqV24G9p7sCSEHO7ZMC0Zj5m32CzEFjpG9nlWRIIKgT4iVg73XtjY1DCN4JoQ9aflFegxPGgKpx4sIXyPwfmYCPG4eEfRk5cJFkxs2qS\\\\\\\/W6MP8rpHBC9OF9wGonzG1LJY90yYy3dG1aSgWWY5R5GJ7QhWSljEqhZvur6hduvjzwh9rsFdcfdg6xMNxDECd1worbuDkw9g3yuhAeRQxgR9rY7Z19ubXIc9QMIZjZJSAT7zg\\\\\\\/8HgJV6Vcif\\\\\\\/3LoYo2NqetwXIrzllCeVEPVt7DGnZEX76q7kaLhpHMmepsxgNp13iCaHwDbQmgmMipH1BmoBxiTcf9GmdTZ7NXvzO+IdqVZRjg5xYdkSjZ5lE\\\\\\\/Yvw32pev9ZcwEKg9scD3a55Aqg2WmViN6hlCM8oVzV53olWTVIA3xw8QOKHrfDer3fl89RYNWc8MB0WKWle\\\\\\\/xu+i0e9tIv75aWxYQ\\\\\\\/fhNhD6wSG3kejqy4ubt55gxVdR4irfiQcFm14Kg7cXfe9vlYQOYdlJ30LTqCGd73baGVBatFLu2\\\\\\\/\\\\\\\/OxVDeRVeOmhlWx64mrt5xZKDlnND7bCSjemLHqiWHwBnMyGivDvtmKrx4a7G67CuTxK4kLpNY8cOfl82bT8Wucdl8nbLyYpVwcFTSI4VvP0PvevoFsGvhe2V3uLUn31PlVZJMxw46EiKrpRmTjlK6kDa3VHDWjXP2vKbh5PaLEcmvhZ97+tmCByra7diboI2yMJ76U9NZ8f18i35LNClZtQM3JxPseTcIexwyd\\\\\\\/rEENpigsDnzyrjcFCW8\\\\\\\/pjfGWOV+IbwoQrB4mIUKjGgUPtaosQqKcogqdphVOnKDyrZ9w8OXjof6gXsLt+H43A4nlGjJm7J7rSSoIbp32l3I9w6vz704PHxVbIykLX2M+j3fDS8+M5W4xDz\\\\\\\/u8oiqwcOE0paCPAs4JGNTqARCj4WgBh1Li8Hn1fTqWVF+EjmbI0iU6fBVpaCWL8IcN299LOaPs\\\\\\\/cyD6Kn96MwWjlCnMKo4+62OIp2e3HM9YE3YHi2xg\\\\\\\/+WRuFoP\\\\\\\/IkZkPypnQ\\\\\\\/+GwSSMEj4x3orbkciJwl3RaXPfpRPMIL+nG9r\\\\\\\/JJP2xwMavNKjwD6xW\\\\\\\/QAJF8ALIzQ33B58OyzMVXrqI3jjyv0QOyV8n+s0awLyOB51CQvx+PAsR7+qRM7LC2g5VqYUcx4nM38f\\\\\\\/j6do\\\\\\\/97iZVeNxrZKNZXFZvYZeZDMjWTazflyQNGgXRgJWyYaIumw3FMRWci6l51sh7aTPNpJImIsp33wd6EP9yTo7dRgDtcGufe\\\\\\\/5Mewye67t\\\\\\\/s+AKGJ4nZSk2HrioRxnhqiIS7skmZ63kwS2oEHHeqHxgj6o8lnCg2BkTKUfrsKdmhV47uD3VVpOGjhixmyg7foERkpQz0jRY\\\\\\\/7la5zWpUZf0RoAs9tgM5+5p6wMzM863Dn3Wv8jSsisoP0g1QFxZnu6iaFnqVVlj4nKlw9xOGRsmlv9KEBU9JBaeLiArISCqAuSViss\\\\\\\/nzionhIyrN62+hv1AdeJvQpBq16pSLfmYqkPpRbaEbecYGFUiTrfc51lyJoyYBrDD\\\\\\\/QVOtwSWyrNK08wovpbnX7p6GsNyYlTDPvCcDZqaJ5wU6xC+zqxcPOlBlw\\\\\\\/s\\\\\\\/srQAWNTYsOQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UUb6x1ujSGPWwaP0DTjJjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002254392c695c47484b3eb78dea28f9c7296689ea458bfe394ac96353b4bdcb9689\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jttintIivHevc4ei2jIq7slGkA1IsWjwkcV9Eh8vSr7rpYCcpHhCTW18DqqMjXSCPVBS7YUeVp7ylUmuimg7ztc8F6YscESZ26uG34tMZeFHp9qiYY+scnWPngAZ3PfnherZYMgQWYG+0EfRE76lMxrZKUo+9VwKcSZawzNCxhwNbS3yEQB5W\\\\\\\/lw1HQLCg3EJxfQvmgRpg8CCYvCxNIC+ge7uf\\\\\\\/gJlYmZwfF4NzQO9aWJ7A3bzQXraCAxDwms2pG67EEveJ5NCBPMs+dKz5Bdo\\\\\\\/Zh7EQInHYLWI4glR3n+tuOO\\\\\\\/ZpYQBfXQWXZvMUmPh4C7\\\\\\\/cAZEu0PCupxhUjXvUFkmccupf4JktErkG0gL0xfVPajr+mCLLY+tmt2EHZR0O0y5JlGSOGpqsaK\\\\\\\/ozUI0U5aqstHPYyUPhpDYAQ9PZmA3ZEKPCrwvj9\\\\\\\/9pAGqbSGN6nlmgAzLz1je2Hgkv2bKGpvj6IcDnhOpQR5M5U9XyRNcIkYD0vgkFLxTxCuHSPm9cTGSNYwTQbdFFxwC2cjGTUPsp5HoFeZGDW4xTixNaNWkcnDtDL729GUdtwg9MpfN1Xtq7tid39Nxf9CBKJBN8uYcrkhZJLviS8Dr+H6HwdrYuiZOWRgTZLWkrBIaMe1nYE0dOcb+YssW0T6Wwfdtrth1luieTYD0JBGbY+ieWh0qr7ZCEk5mZtyNmQJMHHUtNyNS1ic2Za8SMljnnZMz\\\\\\\/vx\\\\\\\/eUZJK709ilzTB0RSYwdMth5TXiJ\\\\\\\/K0P9kBX7\\\\\\\/QqlB0+P5t\\\\\\\/MFhvpTj5Eeu0aJXfhgomTh9N4rwP0kUdSRtBSgysy04ksfTBWFKFRGbEIHhGqKEpmBO3P\\\\\\\/TAJ5AV34NnjCryzIABxsUf72yz2+sr4+sMTtSeLurtjKoDqMLKFYbCtOPy5OMiEadYLE+BiPlIUgIENvlZPkgujghmg6JDux2bjoqNUkk+QfmRhwUnK9Y3SSWROfpgwl4dbvUQmG3VXY66Ghal1IJ\\\\\\\/kcubULgJkm+DB+c2R3B4vz3wiyyEVpNgVU+k36E4M\\\\\\\/LYChm47\\\\\\\/GhLhDMGyr0uzwwE2O8UrkuCf3Z6NVYedh2bcc\\\\\\\/FgTYO73gpdfUOB0x2lt+zf9rbX3Z7Axli7sejPP5k4jNNrs8bVvFIXWLjfdI7bnILfZH6QL+5h5lDKRvhT3McMQ87WyZBdjrIgqHqCYu0FJ1OH6qnUBZExnHnDWX8oHTdMU6Zh7AKAKpIzT4mCtQVRK6QfEJHEgSaprCMmGU9Njg3C0tY4MbSEcCmpIaihDnLnFj1Fm4GlS7R57wJXU64a6aJJDOh04zv6ICbNY2ry0xOy0eVzhNGFl6gIHGArKpniZPymKzcQe5pHIjtuPKIA2Wmdg7GbkEYj2EhGmQdsSrPbFkT1oP2xTUwRyWQGqe2HqNFY2JRHiL9D+rAZN\\\\\\\/iz+\\\\\\\/xxa5s4mUuKGQVFoJgBRdq3AFLBNGTpnhyGidg5jGJZ4IpC4s+zUmmMcSd3K4zWuNclNSDsoJAH5W10azsMZ8kcrydNrZPB6cCDkmVamm\\\\\\\/UvKHg1mNW0VbNbuRO5MD8N0\\\\\\\/ope3VpaK3LbsbNh0wCUsi2bbNe1v1\\\\\\\/CHwbBh4LwjOgf9PBTTSRtzB+h2Tc2925ope+HLDVkVgiTVAO1F57mDziWO++ov2Xvf2sKlr4+7TLLn7Cbyrplis9H1twXnh+fmUJ48Jy+YiLmz8x3GQiAlGHwssZ2ekmioCjsUhoGqDG76tOAT9+gIWsYweCYEW7j8T9I3bAC65MXcR8VZqn5lF043EeGQ5Ij\\\\\\\/k+Y44ru+XFet6PUJ3Nk\\\\\\\/fogmcN33YleeXqg7467qpuO7PlMq6XXYiqGdcHT+uVG9cI7S+HzqxSksS\\\\\\\/f95iQ5avmsjstlrXE8gHjQB5GGgiVAD9fnW0FPIpmdtl5kd\\\\\\\/JDF+ZtsPcwMyZW0bVK2Ib+V8gYtR1vQhaHf5r2\\\\\\\/PXXTY6JVaTcqEhcGA+kM0e5sws708YG8NpL\\\\\\\/IkFqGfTHleR6EBLZ6dUpJwLg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xTwflTaX1+wpYD+QJM0LXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef9d12af55d915a1612e01ee4ceec1a86f8f72b82bdef34d935c27492433a5a1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493465 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2089" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3LLmXPEV4XaftHYZEF_ZVWdwTVqgvDT-Jgp_gXUtjkE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2089" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-S1FBaSmIw4gTthGv4qTtZkPx6Pz2zpNNWB9CV9wACdg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2071" title="Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2097" title="An Interscholastic Network To Generate LexA Enhancer Trap Lines in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493465" id="node493465" data-pisa="ggg;9/7/2089" data-pisa-master="ggg;g3.119.400185" data-apath="/ggg/9/7/2089.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">ARMOR: An <span class="underline">A</span>utomated <span class="underline">R</span>eproducible <span class="underline">MO</span>dular Workflow for Preprocessing and Differential Analysis of <span class="underline">R</span>NA-seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1508-461X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stephany Orjuela</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3285-1945" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ruizhu Huang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5041-6205" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katharina M. Hembach</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3048-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark D. Robinson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3833-2169" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charlotte Soneson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2089-2096;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400185 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493465" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stephany Orjuela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Molecular Cancer Research, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephany%2BOrjuela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orjuela%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephany%2BOrjuela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1508-461X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stephany Orjuela</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ruizhu Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ruizhu%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARuizhu%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3285-1945" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ruizhu Huang</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Katharina M. Hembach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Quantitative Biomedicine, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BM.%2BHembach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hembach%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatharina%2BM.%2BHembach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5041-6205" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katharina M. Hembach</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark D. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BD.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3048-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark D. Robinson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.robinson@imls.uzh.ch" class="" data-icon-position="" data-hide-link-title="0">mark.robinson@imls.uzh.ch</a>
<a href="mailto:charlotte.soneson@fmi.ch" class="" data-icon-position="" data-hide-link-title="0">charlotte.soneson@fmi.ch</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charlotte Soneson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BSoneson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soneson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharlotte%2BSoneson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3833-2169" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charlotte Soneson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.robinson@imls.uzh.ch" class="" data-icon-position="" data-hide-link-title="0">mark.robinson@imls.uzh.ch</a>
<a href="mailto:charlotte.soneson@fmi.ch" class="" data-icon-position="" data-hide-link-title="0">charlotte.soneson@fmi.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2089" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493465" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2089.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493465" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2089.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493465" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2089.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493465" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">The extensive generation of RNA sequencing (RNA-seq) data in the last decade has resulted in a myriad of specialized software for its analysis. Each software module typically targets a specific step within the analysis pipeline, making it necessary to join several of them to get a single cohesive workflow. Multiple software programs automating this procedure have been proposed, but often lack modularity, transparency or flexibility. We present ARMOR, which performs an end-to-end RNA-seq data analysis, from raw read files, via quality checks, alignment and quantification, to differential expression testing, geneset analysis and browser-based exploration of the data. ARMOR is implemented using the Snakemake workflow management system and leverages conda environments; Bioconductor objects are generated to facilitate downstream analysis, ensuring seamless integration with many R packages. The workflow is easily implemented by cloning the GitHub repository, replacing the supplied input and reference files and editing a configuration file. Although we have selected the tools currently included in ARMOR, the setup is modular and alternative tools can be easily integrated.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rna-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-rna-sequencing" rel="nofollow">RNA sequencing</a></li><li class="kwd"><a href="/keyword/differential-expression" class="hw-term hw-article-keyword hw-article-keyword-differential-expression" rel="nofollow">Differential expression</a></li><li class="kwd"><a href="/keyword/exploratory-data-analysis" class="hw-term hw-article-keyword hw-article-keyword-exploratory-data-analysis" rel="nofollow">Exploratory data analysis</a></li><li class="kwd"><a href="/keyword/quality-control" class="hw-term hw-article-keyword hw-article-keyword-quality-control" rel="nofollow">Quality control</a></li></ul><p id="p-6">Since the first high-throughput RNA-seq experiments about a decade ago, there has been a tremendous development in the understanding of the characteristic features of the collected data, as well as the methods used for the analysis. Today there are vetted, well-established algorithms and tools available for many aspects of RNA-seq data analysis (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Conesa <em>et al.</em> 2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Van Den Berge <em>et al.</em> 2018</a>). In this study, we capitalize on this knowledge and present a modular, light-weight RNA-seq workflow covering the most common parts of a typical end-to-end RNA-seq data analysis with focus on differential expression. The application is implemented using the Snakemake workflow management system (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Köster and Rahmann 2012</a>), and allows the user to easily perform quality assessment, adapter trimming, genome alignment, transcript and gene abundance quantification, differential expression analysis and geneset analyses with a simple command, after specifying the required reference files and information about the experimental design in a configuration file. Reproducibility is ensured via the use of conda environments, and all relevant log files are retained for transparency. The output is provided in state-of-the-art R/Bioconductor objects, ensuring interoperability with a broad range of Bioconductor packages. In particular, we provide a template to facilitate browser-based interactive visualization of the quantified abundances and the results of the statistical analyses with iSEE (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rue-Albrecht <em>et al.</em> 2018</a>).</p><p id="p-7">Among already existing pipelines for automated reference-based RNA-seq analysis, several focus either on the preprocessing and quality control steps (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">He <em>et al.</em> 2018</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ewels <em>et al.</em> 2018</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tsyganov <em>et al.</em> 2018</a>), or on the downstream analysis and visualization of differentially expressed genes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Marini 2018</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Monier <em>et al.</em> 2019</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Powell 2018</a>), or do not provide a single framework for the preprocessing and downstream analysis (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Steinbaugh <em>et al.</em> 2018</a>). Some workflows are based on predefined reference files and can only quantify abundances for human or mouse (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Torre <em>et al.</em> 2018</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cornwell <em>et al.</em> 2018</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang 2018</a>). Additionally, workflows that conduct differential gene expression analysis often do not allow comparisons between more than two groups, or more complex experimental designs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Girke 2018</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cornwell <em>et al.</em> 2018</a>). Some existing pipelines only provide a graphical user interface to design and execute fully automated analyses (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hung <em>et al.</em> 2018</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Afgan <em>et al.</em> 2018</a>). In addition to reference-based tools, there are also pipelines that perform <em>de novo</em> transcriptome assembly before downstream analysis (<em>e.g.</em>, <a href="https://github.com/dib-lab/elvers">https://github.com/dib-lab/elvers</a>).</p><p id="p-8">ARMOR performs both preprocessing and downstream statistical analysis of the RNA-seq data, building on standard statistical analysis methods and commonly used data containers. It distinguishes itself from existing workflows in several ways: (i) Its modularity, reflected in its fully and easily customizable framework. (ii) The transparency of the output and analysis, meaning that all code is accessible and can be modified by the user. (iii) The seamless integration with downstream analysis and visualization packages, especially those within Bioconductor (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Huber <em>et al.</em> 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amezquita <em>et al.</em> 2019</a>). (iv) The ability to specify any fixed-effect experimental design and any number of contrasts, in a standardized format. (v) The inclusion of a test for differential transcript usage in addition to differential gene expression analysis. While high-performance computing environments and cloud computing are not specifically targeted, Snakemake enables the usage of a cluster without the need to modify the workflow itself.</p><p id="p-9">In general, we do not advocate fully automated analysis. All rigorous data analyses need exploratory steps and spot checks at various steps throughout the process, to ensure that data are of sufficient quality and to spot potential errors (<em>e.g.</em>, sample mislabelings). ARMOR handles the automation of ”bookkeeping” tasks, such as running the correct sequence of software for all samples, and compiling the data and reports in standardized formats. If errors are identified, the workflow can re-run only the parts that need to be updated.</p><p id="p-10">ARMOR is available from <a href="https://github.com/csoneson/ARMOR">https://github.com/csoneson/ARMOR</a>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Overview</h3><p id="p-11">The ARMOR workflow is designed to perform an end-to-end analysis of bulk RNA-seq data, starting from FASTQ files with raw sequencing reads (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Reads first undergo quality control with FastQC (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) and (optionally) adapter trimming using TrimGalore! (<a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>), before being mapped to a transcriptome index using Salmon (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Patro <em>et al.</em> 2017</a>) and (optionally) aligned to the genome using STAR (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dobin <em>et al.</em> 2013</a>). Estimated transcript abundances from Salmon are imported into R using the tximeta package (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Soneson <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Love <em>et al.</em> 2019</a>) and analyzed for differential gene expression and (optionally) differential transcript usage with edgeR (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Robinson <em>et al.</em> 2010</a>) and DRIMSeq (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Nowicka and Robinson 2016</a>). The quantifications, provided metadata, and results from the statistical analyses are exported as SingleCellExperiment objects (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lun and Risso 2019</a>) ensuring interoperability with a large part of the Bioconductor ecosystem (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Huber <em>et al.</em> 2015</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Amezquita <em>et al.</em> 2019</a>). Quantification and quality control results are summarized in a MultiQC report (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ewels <em>et al.</em> 2016</a>). Other tools can be easily exchanged for those listed above by modifying the Snakefile and/or the template analysis code.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simplified directed acyclic graph (DAG) of the ARMOR workflow. Blue ellipses are rules run in R, orange ellipses from software called as shell commands. Dashed lines and light-colored ellipses are optional rules, controlled in the configuration file. By default all rules are executed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281360334" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simplified directed acyclic graph (DAG) of the ARMOR workflow. Blue ellipses are rules run in R, orange ellipses from software called as shell commands. Dashed lines and light-colored ellipses are optional rules, controlled in the configuration file. By default all rules are executed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2089/F1.medium.gif" width="334" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2089/F1.medium.gif" width="334" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493684" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Simplified directed acyclic graph (DAG) of the ARMOR workflow. Blue ellipses are rules run in R, orange ellipses from software called as shell commands. Dashed lines and light-colored ellipses are optional rules, controlled in the configuration file. By default all rules are executed.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Input file specification</h3><p id="p-13">ARMOR can be used to analyze RNA-seq data from any organism for which a reference transcriptome and (optionally) an annotated reference genome are available from either Ensembl (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zerbino <em>et al.</em> 2018</a>) or GENCODE (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Frankish <em>et al.</em> 2019</a>). Paths to the reference files, as well as the FASTQ files with the sequencing reads, are specified by the user in a configuration file. In addition, the user prepares a metadata file – a tab-delimited text file listing the name of the samples, the library type (single- or paired-end) and any other covariates that will be used for the statistical analysis. The checkinputs rule in the Snakefile can be executed to make sure all the input files and the parameters in the configuration file have been correctly specified.</p></div><div id="sec-4" class="subsection"><h3>Workflow execution</h3><p id="p-14">ARMOR is implemented as a modular Snakemake (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Köster and Rahmann 2012</a>) workflow, and the execution of the individual steps is controlled by the provided Snakefile. Snakemake will automatically keep track of the dependencies between the different parts of the workflow; rerunning the workflow will thus only regenerate results that are out of date or missing given these dependencies. Via a set of variables specified in the configuration file, the user can easily decide to include or exclude the optional parts of the workflow (shaded ellipses in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). By adding or modifying targets in the Snakefile, users can include any additional or specialized types of analyses that are not covered by the original workflow.</p><p id="p-15">By default, all software packages that are needed for the analysis will be installed in an auto-generated conda environment, which will be automatically activated by Snakemake before the execution of each rule. The desired software versions can be specified in the provided environment file. If the user prefers, local installations of (all or a subset of) the required software can also be used (as described in <strong>Software management</strong>).</p></div><div id="sec-5" class="subsection"><h3>Software management</h3><p id="p-16">First, the user needs to have a recent version of Snakemake and conda installed. There is a range of possibilities to manage the software for the ARMOR workflow. The recommended option is to allow conda and the workflow itself to manage everything, including the installation of the needed R packages. The workflow is executed this way with the command</p><p id="p-17"><kbd>snakemake‐‐use-conda</kbd></p><p id="p-18">The first time the workflow is run, the conda environments will be generated and all necessary software will be installed. Any subsequent invocations of the workflow from this directory will use these generated environments. An alternative option is to use ARMOR’s envs/environment.yaml file to create a conda environment that can be manually activated, by running the command</p><p id="p-19"><kbd>conda env create‐‐name ARMOR \</kbd></p><p id="p-20"><kbd>‐‐file envs/environment.yaml</kbd></p><p id="p-21"><kbd>conda activate ARMOR</kbd></p><p id="p-22">The second command activates the environment. Once the environment is activated, ARMOR can be run by simply typing</p><p id="p-23"><kbd>snakemake</kbd></p><p id="p-24">Additionally, the user can circumvent the use of conda, and make sure that all software is already available and in the user’s PATH. For this, Snakemake and the tools listed in envs/environment.yaml need to be manually installed, in addition to a recent version of R and the R packages listed in scripts/install_pkgs.R.</p><p id="p-25">For either of the options mentioned above, the <kbd>useCondaR</kbd> flag in the configuration file controls whether a local R installation, or a conda-installed R, will be used. If <kbd>useCondaR</kbd> is set to <kbd>False</kbd>, the path to a local R installation (<em>e.g.</em>, <kbd>Rbin:&lt;path&gt;</kbd>) must be specified in the configuration file, along with the path to the R package library (<em>e.g.</em>, <kbd>R_LIBS_USER=“&lt;path&gt;”</kbd>) in the .Renviron file. If the specified R library does not contain the required packages, Snakemake will try to install them (<em>i.e.</em>, write permissions would be needed). ARMOR has been tested on macOS and Linux systems.</p></div><div id="sec-6" class="subsection"><h3>Statistical analysis</h3><p id="p-26">ARMOR uses the quasi-likelihood framework of edgeR (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Robinson <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lun <em>et al.</em> 2016</a>) to perform tests for differential gene expression, camera (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wu and Smyth 2012</a>) to perform associated geneset analysis, and DRIMSeq (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Nowicka and Robinson 2016</a>) to test for differential transcript usage between conditions. All code to perform the statistical analyses is provided in Rmarkdown templates (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allaire <em>et al.</em> 2018</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Xie <em>et al.</em> 2018</a>), which are executed at runtime. This setup gives the user flexibility to use any experimental design supported by these tools, and to test any contrast(s) of interest, by specifying these in the configuration file using standard R syntax, <em>e.g.</em>,</p><p id="p-27"><kbd>design:“<sup>∼</sup> 0 + group”</kbd></p><p id="p-28"><kbd>contrast:groupA-groupB</kbd></p><p id="p-29">Arbitrarily complex designs and multiple contrasts are supported. In addition, by editing the template code, users can easily configure the analysis, add additional plots, or even replace the statistical test if desired. After compilation, all code used for the statistical analysis, together with the results and version information for all packages used, is retained in a standalone html report, ensuring transparency and reproducibility and facilitating communication of the results.</p></div><div id="sec-7" class="subsection"><h3>Output files</h3><p id="p-30">The output files from all steps in the ARMOR workflow are stored in a user-specified output directory, together with log files for each step, including relevant software version information. A detailed summary of the output files generated by the workflow, including the shell command that was used to generate each of them, the time of creation, and information about whether the associated inputs, code or parameters have since been updated, can be obtained at any time by invoking Snakemake with the flag <kbd>-D</kbd> (or<kbd>‐‐detailed-summary</kbd>). Using the benchmark directive of Snakemake, ARMOR also automatically generates additional text files summarizing the run time and peak memory usage of each executed rule.</p><p id="p-31">The results from the statistical analyses are combined with the transcript- and gene-level quantifications and saved as SingleCellExperiment objects (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Lun and Risso 2019</a>), ensuring easy integration with a large number of Bioconductor packages for downstream analysis and visualization. For example, the results can be interactively explored using the iSEE package (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Rue-Albrecht <em>et al.</em> 2018</a>) and a template is provided for this.</p></div><div id="sec-8" class="subsection"><h3>Multiple project management</h3><p id="p-32">When managing multiple projects, the user might run ARMOR in multiple physical locations (<em>i.e.</em>, clone the repository in separate places). <kbd>snakemake‐‐use-conda</kbd> will create a separate conda environment in each subdirectory, which means that the installed software may be duplicated. If disk space is a concern, building and activating a single conda environment (using the <kbd>conda env create</kbd> command as shown in the <strong>Software management</strong> section), and activating this before invoking each workflow may be beneficial. It is also possible to explicitly specify the path to the desired config.yaml configuration file when snakemake is called:</p><p id="p-33"><kbd>snakemake‐‐configfile config.yaml</kbd></p><p id="p-34">Thus, the same ARMOR installation can be used for multiple projects, by invoking it with a separate config.yaml file for each project.</p><p id="p-35">By taking advantage of the Snakemake framework, ARMOR makes file and software organization relatively autonomous. Although we recommend using a file structure similar to the one used for the example data provided in the repository (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), and managing all the software for a project in a conda environment, the user is free to use the same environment for different datasets, even if the files are located in several folders. This alternative is more of a “software-based” structure than the “project-based” structure we present with the pipeline. Either structure has its advantages and depending on the use case and level of expertise, both can be easily implemented using ARMOR.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The files and directory structure that make up the ARMOR workflow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281360334" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The files and directory structure that make up the ARMOR workflow.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2089/F2.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2089/F2.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493564" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">The files and directory structure that make up the ARMOR workflow.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Code availability</h3><p id="p-37">ARMOR is available (under MIT license) from <a href="https://github.com/csoneson/ARMOR">https://github.com/csoneson/ARMOR</a>, together with a detailed walk-through of an example analysis. The repository also contains a wiki (<a href="https://github.com/csoneson/ARMOR/wiki">https://github.com/csoneson/ARMOR/wiki</a>), which is the main source of documentation for ARMOR and contains extensive information about the usage of the workflow.</p></div><div id="sec-10" class="subsection"><h3>Data Availability</h3><p id="p-38">Supplemental file DataS1.html contains the MultiQC report for the data used in the <strong>Real data walk-through</strong> section (ArrayExpress accession number E-MTAB-7029). Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8053280">https://doi.org/10.25387/g3.8053280</a>.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>The ARMOR skeleton</h3><p id="p-39"><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> shows the set of files contained within the ARMOR workflow, and what is downloaded to the user’s computer when the repository is cloned.</p><p id="p-40">The example_data directory represents a (runnable) template of a very small dataset, which is useful for testing the software setup and the system as well as for having a structure to copy for a real project. The provided config.yaml file is pre-configured for this example dataset. We recommend that users prepare their own config.yaml and a similar directory structure to example_data, with the raw FASTQ files and reference sequence and annotation information in subfolders, perhaps using symbolic links if such files are already available in another location. We present an independent example below in the <strong>Real data walk-through</strong> section.</p><p id="p-41">Once everything is set up, running <kbd>snakemake</kbd>, which operates on the rules in the Snakefile, will construct the hierarchy of instructions to execute, given the specifications in the config.yaml file. Snakemake automatically determines the dependencies between the rules and will invoke the instructions in a logical order. The scripts and envs directories, and the Snakefile itself, should not need to be modified, unless the user wants to customize certain aspects of the pipeline.</p></div><div id="sec-13" class="subsection"><h3>Real data walk-Through</h3><p id="p-42">Here, we illustrate the practical usage of ARMOR on a bulk RNA-seq dataset from a study on Wnt signaling (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Doumpas <em>et al.</em> 2019</a>). For each of three genetic backgrounds (HEK 293T, dBcat and d4TCF) and two experimental conditions (untreated and stimulated using the GSK3 inhibitor CHIRON99021), three biological replicates were measured (18 samples in total). The number of sequenced reads for each individual sample ranges from 12.5 to 41 million. A more detailed overview of the dataset is provided in the MultiQC report generated by the ARMOR run (Supplemental File DataS1.html). An R script (download_files.R, which can be found at <a href="https://github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/E-MTAB-7029/download_files.R">https://github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/E-MTAB-7029/download_files.R</a>) was written to download the FASTQ files with raw reads from ArrayExpress (<a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-7029/">https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-7029/</a>), and create a metadata table detailing the type of library and experimental condition for each sample (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This table was saved as a tab-delimited text file named metadata.txt.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493489/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493489/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Metadata table for the Wnt signaling data</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">The raw data and reference files were organized into a directory, E-MTAB-7029, with the structure according to <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. The default config.yaml downloaded with the workflow was copied into a new file called config_E-MTAB-7029.yaml and edited to reflect the location of these files. In addition, the read length was set and the experimental design was specified as <kbd>“∼ 0 + condition”</kbd>, where the condition information will be taken from metadata.txt. Then, a set of contrasts of interest (<em>e.g.</em>, <kbd>conditiond4Tcf__chir-conditiond4Tcf__unstim</kbd>) were specified, as well as the set of genesets to use. The final configuration file can be viewed at <a href="https://github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/config_E-MTAB-7029.yaml">https://github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/config_E-MTAB-7029.yaml</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The suggested structure for the set of files that need to be organized to run ARMOR on a new dataset. The structure can deviate from this somewhat, since the location of the files can be specified in the corresponding config.yaml file." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281360334" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The suggested structure for the set of files that need to be organized to run ARMOR on a new dataset. The structure can deviate from this somewhat, since the location of the files can be specified in the corresponding config.yaml file.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2089/F3.medium.gif" width="408" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2089/F3.medium.gif" width="408" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493549" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-44" class="first-child">The suggested structure for the set of files that need to be organized to run ARMOR on a new dataset. The structure can deviate from this somewhat, since the location of the files can be specified in the corresponding config.yaml file.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">The set of files (not including the large data and reference files, which would be downloaded using the download_files.R) used in this setup can be found on the chiron_realdataworkflow branch of the ARMOR repository: <a href="https://github.com/csoneson/ARMOR/tree/chiron_realdataworkflow">https://github.com/csoneson/ARMOR/tree/chiron_realdataworkflow</a>.</p><p id="p-46">After downloading the data, generating the metadata.txt file and editing the config.yaml file, the full workflow was run with the command:</p><p id="p-47"><kbd>snakemake‐‐use-conda‐‐cores 20 \</kbd></p><p id="p-48"><kbd>‐‐configfile config_E-MTAB-7029.yaml</kbd></p><p id="p-49">and upon completion of the workflow run, the specified output directory was populated as shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The MultiQC directory contains a summary report of the quality assessment and alignment steps. In the outputR directory, reports of the statistical analyses (DRIMSeq_dtu.html and edgeR_dge.html), as well as a list of SingleCellExperiment objects (in shiny_sce.rds) are saved. The latter can be imported into R and used for further downstream analysis. Using the template run_iSEE.R (available from <a href="https://github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/E-MTAB-7029/run_iSEE.R">https://github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/E-MTAB-7029/run_iSEE.R</a>) and shiny_sce.rds (available from <a href="https://doi.org/10.6084/m9.figshare.8040239.v1">https://doi.org/10.6084/m9.figshare.8040239.v1</a>), an R/shiny web application can be initiated, with various panels to allow the user to interactively explore the data and results (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The set of output files from the workflow. This includes log files for every step and all the standard outputs of all the tools, such as R objects and scripts, BAM files, bigWig files and quantification tables. Note that the outputs for only one RNA-seq sample are shown; ... represents the set of output files for the remaining samples or contrasts. Directories ending in / contain extraneous files and are collapsed here." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281360334" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The set of output files from the workflow. This includes log files for every step and all the standard outputs of all the tools, such as R objects and scripts, BAM files, bigWig files and quantification tables. Note that the outputs for only one RNA-seq sample are shown; ... represents the set of output files for the remaining samples or contrasts. Directories ending in / contain extraneous files and are collapsed here.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2089/F4.medium.gif" width="219" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2089/F4.medium.gif" width="219" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493562" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-50" class="first-child">The set of output files from the workflow. This includes log files for every step and all the standard outputs of all the tools, such as R objects and scripts, BAM files, bigWig files and quantification tables. Note that the outputs for only one RNA-seq sample are shown; ... represents the set of output files for the remaining samples or contrasts. Directories ending in / contain extraneous files and are collapsed here.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screenshot of visualization of data and results from the real data walk-through using the iSEE R/Bioconductor package. The interactive application was configured to display an MDS plot colored by the sample condition (top left), a custom panel showing the observed read coverage of a selected gene (top right), a volcano plot for a specified contrast (bottom left, the selected genes are shown in the adjacent table) and an overview of the log-CPM expression values for each sample, for a gene selected in a second table (bottom right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281360334" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Screenshot of visualization of data and results from the real data walk-through using the iSEE R/Bioconductor package. The interactive application was configured to display an MDS plot colored by the sample condition (top left), a custom panel showing the observed read coverage of a selected gene (top right), a volcano plot for a specified contrast (bottom left, the selected genes are shown in the adjacent table) and an overview of the log-CPM expression values for each sample, for a gene selected in a second table (bottom right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2089/F5.medium.gif" width="440" height="351"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2089/F5.medium.gif" width="440" height="351"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493605" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-51" class="first-child">Screenshot of visualization of data and results from the real data walk-through using the iSEE R/Bioconductor package. The interactive application was configured to display an MDS plot colored by the sample condition (top left), a custom panel showing the observed read coverage of a selected gene (top right), a volcano plot for a specified contrast (bottom left, the selected genes are shown in the adjacent table) and an overview of the log-CPM expression values for each sample, for a gene selected in a second table (bottom right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-52"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the run time and maximal memory usage for generating each output file. Note that the <kbd>ncores</kbd> parameter in the configuration file was kept at 1, and thus each rule was run using a single thread. The most memory-intensive parts of the workflow, due to the large size of the reference genome, were the generation of the STAR index and the alignment of reads to the genome. The most time consuming parts were the generation of the STAR index and the DTU analysis with DRIMSeq. However, both of these can be executed using multiple cores, by increasing the value of the <kbd>ncores</kbd> parameter.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The required resources for the generation of each output file (grouped by the Snakemake rule) in the real data walk-through, as reported by the benchmarking directive of Snakemake. The four panels show the read and written bytes (in MB), the memory usage (in MB) and the run time (in seconds) of each rule. RSS = Resident Set Size. IO = Input/Output." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281360334" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The required resources for the generation of each output file (grouped by the Snakemake rule) in the real data walk-through, as reported by the benchmarking directive of Snakemake. The four panels show the read and written bytes (in MB), the memory usage (in MB) and the run time (in seconds) of each rule. RSS = Resident Set Size. IO = Input/Output.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2089/F6.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/7/2089/F6.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2089/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493513" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-53" class="first-child">The required resources for the generation of each output file (grouped by the Snakemake rule) in the real data walk-through, as reported by the benchmarking directive of Snakemake. The four panels show the read and written bytes (in MB), the memory usage (in MB) and the run time (in seconds) of each rule. RSS = Resident Set Size. IO = Input/Output.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-54">M.D.R. acknowledges support from the University Research Priority Program Evolution in Action at the University of Zurich, the Swiss National Science Foundation (grants 310030_175841, CRSII5_177208) and the Chan Zuckerberg Initiative (grant 2018-182828).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">2</span> The order of the shared first authors was determined randomly, using the sample() function in R v3.5.2, with the random seed 1552397284.</p></li><li class="fn" id="fn-4"><p id="p-55">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8053280">https://doi.org/10.25387/g3.8053280</a>.</p></li><li class="fn" id="fn-5"><p id="p-56"><em>Communicating editor: A. Whitehead</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-12"><span class="received-label">Received </span>March 12, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-13"><span class="accepted-label">Accepted </span>May 13, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Orjuela <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.1" data-doi="10.1093/nar/gky379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Afgan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batut</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Beek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouvier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2018 update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">W537</span>–<span class="cit-lpage">W544</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DW537%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky379%26rft_id%253Dinfo%253Apmid%252F29790989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29790989&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2089.2"><div class="cit-metadata unstructured">Allaire, J., Y. Xie, J. McPherson, J. Luraschi, K. Ushey, <em>et al.</em>, 2018 rmarkdown: Dynamic Documents for <em>R.</em> R package version 1.11.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.3" data-doi="10.1101/590562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amezquita</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpp</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geistlinger</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lun</span>  <span class="cit-name-given-names">A. T. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Orchestrating Single-Cell analysis with Bioconductor.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/590562</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F590562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1OTA1NjJ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMDg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.4" data-doi="10.1186/s13059-016-0881-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madrigal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarazona</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Cabrero</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cervera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A survey of best practices for RNA-seq data analysis.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">13</span>. Erratum: 181.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-0881-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0881-8%26rft_id%253Dinfo%253Apmid%252F26813401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-0881-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26813401&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.5" data-doi="10.1186/s12859-018-2139-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cornwell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vangala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taing</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbert</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Köster</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">VIPER: Visualization pipeline for RNA-seq, a snakemake workflow for efficient and complete RNA-seq analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-018-2139-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D19%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2139-9%26rft_id%253Dinfo%253Apmid%252F29649993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2139-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29649993&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.6" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.7" data-doi="10.15252/embj.201798873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doumpas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lampart</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lentini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nestor</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">TCF/LEF dependent and independent transcriptional regulation of Wnt/<em>β</em>-catenin target genes.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">38</span>. Erratum: e98873.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15252/embj.201798873</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembj.201798873%26rft_id%253Dinfo%253Apmid%252F30425074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMToiMzgvMi9lOTg4NzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjA4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2089.8"><div class="cit-metadata unstructured">Ewels, P., R. Hammarén, A. Peltzer, D. Moreno, rfenouil, <em>et al</em>., 2018 nf-core/rnaseq: nf-core/rnaseq version 1.2.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.9" data-doi="10.1093/bioinformatics/btw354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewels</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnusson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Käller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MultiQC: Summarize analysis results for multiple tools and samples in a single report.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">3047</span>–<span class="cit-lpage">3048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw354%26rft_id%253Dinfo%253Apmid%252F27312411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27312411&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.10" data-doi="10.1093/nar/gky955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diekhans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jungreis</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">GENCODE reference annotation for the human and mouse genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">D766</span>–<span class="cit-lpage">D773</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD766%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2089.11"><div class="cit-metadata unstructured">Girke, T., 2018 systemPipeRdata: NGS workflow templates and sample data. R package version 1.10.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2089.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">QuickRNASeq: guide for pipeline implementation and for interactive results visualization</span>, pp. <span class="cit-fpage">57</span>–<span class="cit-lpage">70</span> in <span class="cit-source">Transcriptome Data Analysis: Methods and Protocols</span>, <span class="cit-publ-name">Springer New York</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.13" data-doi="10.1038/nmeth.3252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Orchestrating high-throughput genomic analysis with Bioconductor.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3252</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3252%26rft_id%253Dinfo%253Apmid%252F25633503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25633503&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.14" data-doi="10.1101/099010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">L.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meiss</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingersoll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Building containerized workflows using the BioDepot-workflow-builder (bwb).</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/099010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F099010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/099010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.15" data-doi="10.1093/bioinformatics/bts480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Köster</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahmann</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Snakemake–a scalable bioinformatics workflow engine.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2520</span>–<span class="cit-lpage">2522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts480</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts480%26rft_id%253Dinfo%253Apmid%252F22908215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22908215&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309687500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2089.16"><div class="cit-metadata unstructured">Love, M., R. Patro, P. Hickey, and C. Soneson, 2019 tximeta: Transcript Quantification Import with Automatic Metadata. R package version 1.1.16.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2089.17"><div class="cit-metadata unstructured">Lun, A. and D. Risso, 2019 SingleCellExperiment: S4 Classes for Single Cell Data. R package version 1.4.1.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2089.18" data-doi="10.1007/978-1-4939-3578-9_19"><div class="cit-metadata"><cite>Lun, A. T. L., Y. Chen, and G. K. Smyth, 2016 It’s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using Quasi-Likelihood methods in edgeR, pp. 391–416, <em>Statistical Genomics</em>, edited by E. Mathé and S. Davis, Methods in Molecular Biology, Springer New York, New York.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-3578-9_19</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMETHODS%2BIN%2BMOLECULAR%2BBIOLOGY%26rft.volume%253D1418%26rft.spage%253D391%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-3578-9_19%26rft_id%253Dinfo%253Apmid%252F27008025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-3578-9_19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27008025&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2089.19"><div class="cit-metadata unstructured">Marini, F., 2018 ideal: Interactive Differential Expression AnaLysis. R package version 1.8.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.20" data-doi="10.1371/journal.pcbi.1006792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDermaid</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">IRIS-EDA: An integrated RNA-seq interpretation system for gene expression data analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-elocation-id">e1006792</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1006792</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol.%26rft.volume%253D15%26rft.spage%253De1006792%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.21" data-doi="10.12688/f1000research.8900.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowicka</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">DRIMSeq: a Dirichlet-multinomial framework for multivariate count outcomes in genomics.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.8900.2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D5%26rft.spage%253D1356%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.8900.2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.8900.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.22" data-doi="10.1038/nmeth.4197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duggal</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Salmon provides fast and bias-aware quantification of transcript expression.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">419</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4197%26rft_id%253Dinfo%253Apmid%252F28263959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28263959&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2089.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <a href="https://drpowell.github.io/degust/">https://drpowell.github.io/degust/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.24" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19910308&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273116100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.25" data-doi="10.12688/f1000research.14966.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rue-Albrecht</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soneson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lun</span>  <span class="cit-name-given-names">A. T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">iSEE: Interactive SummarizedExperiment explorer</span>. <span class="cit-comment">[version 1; referees: 3 approved]</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">741</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.14966.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D7%26rft.spage%253D741%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.14966.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.14966.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.26" data-doi="10.12688/f1000research.7563.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soneson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.7563.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D4%26rft.spage%253D1521%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.7563.1%26rft_id%253Dinfo%253Apmid%252F26925227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.7563.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26925227&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinbaugh</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pantano</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirchner</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrera</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">bcbioRNASeq: R package for bcbio RNA-seq analysis.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D6%26rft.spage%253D1976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.28" data-doi="10.1016/j.cels.2018.10.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lachmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma’ayan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">BioJupies: Automated generation of interactive notebooks for RNA-Seq data analysis in the cloud.</span> <abbr class="cit-jnl-abbrev">Cell Syst.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">561.e3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cels.2018.10.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSyst.%26rft.volume%253D7%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cels.2018.10.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cels.2018.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.29" data-doi="10.21105/joss.00583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsyganov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James Perry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenneth Archer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">RNAsik: A pipeline for complete and reproducible RNA-seq analysis that runs anywhere with speed and ease.</span> <abbr class="cit-jnl-abbrev">JOSS</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">583</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.21105/joss.00583</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJOSS%26rft.volume%253D3%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.21105%252Fjoss.00583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21105/joss.00583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.30" data-doi="10.7287/peerj.preprints.27283v2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Den Berge</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hembach</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soneson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiberi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <abbr class="cit-jnl-abbrev">RNA sequencing data: hitchhiker’s guide to expression analysis</abbr>, <span class="cit-publ-name">PeerJ Preprints</span> <span class="cit-vol">6</span>: <span class="cit-fpage">e27283v2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7287/peerj.preprints.27283v2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2Bsequencing%2Bdata%253A%2Bhitchhiker%253Fs%2Bguide%2Bto%2Bexpression%2Banalysis%26rft.volume%253D6%26rft.spage%253De27283v2%26rft_id%253Dinfo%253Adoi%252F10.7287%252Fpeerj.preprints.27283v2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7287/peerj.preprints.27283v2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">hppRNA-a snakemake-based handy parameter-free pipeline for RNA-Seq analysis of numerous samples.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft.volume%253D19%26rft.spage%253D622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.32" data-doi="10.1093/nar/gks461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Camera: a competitive gene set test accounting for inter-gene correlation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-elocation-id">e133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks461</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks461%26rft_id%253Dinfo%253Apmid%252F22638577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22638577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2089.33" data-doi="10.1201/9781138359444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allaire</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grolemund</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R Markdown: The Definitive Guide</span>, <span class="cit-publ-name">Chapman and Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, Florida</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1201/9781138359444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BMarkdown%253A%2BThe%2BDefinitive%2BGuide%26rft_id%253Dinfo%253Adoi%252F10.1201%252F9781138359444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1201/9781138359444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2089.34" data-doi="10.1093/nar/gkx1098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achuthan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akanni</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amode</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Ensembl 2018.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D754</span>–<span class="cit-lpage">D761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD754%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1098%26rft_id%253Dinfo%253Apmid%252F29155950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29155950&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2089.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2089.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2071" title="Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2097" title="An Interscholastic Network To Generate LexA Enhancer Trap Lines in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2089" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fU3ghORZ9rd9n25hL76Z7rohahvB1w4pnxyKSoY5uBk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2089" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2089" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2089" class="active" data-icon-position="" data-hide-link-title="0">ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493465" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-uUBjQWpEBMS80D8yw2F3hHh4_06ZcbZrZENGNeP9zxo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2089.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2089" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-XEcQ8Zu2cUUocBMZapK2eIleEkBe35VmAPcmHoZyUqY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2089" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2089.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493465" id="node493465--2" data-pisa="ggg;9/7/2089" data-pisa-master="ggg;g3.119.400185" data-apath="/ggg/9/7/2089.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">ARMOR: An <span class="underline">A</span>utomated <span class="underline">R</span>eproducible <span class="underline">MO</span>dular Workflow for Preprocessing and Differential Analysis of <span class="underline">R</span>NA-seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1508-461X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stephany Orjuela</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3285-1945" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ruizhu Huang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5041-6205" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katharina M. Hembach</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3048-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark D. Robinson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3833-2169" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charlotte Soneson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2089-2096;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400185 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493465--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stephany Orjuela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Molecular Cancer Research, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephany%2BOrjuela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orjuela%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephany%2BOrjuela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1508-461X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stephany Orjuela</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ruizhu Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ruizhu%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARuizhu%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3285-1945" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ruizhu Huang</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Katharina M. Hembach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Quantitative Biomedicine, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BM.%2BHembach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hembach%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatharina%2BM.%2BHembach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5041-6205" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katharina M. Hembach</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark D. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BD.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3048-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark D. Robinson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.robinson@imls.uzh.ch" class="" data-icon-position="" data-hide-link-title="0">mark.robinson@imls.uzh.ch</a>
<a href="mailto:charlotte.soneson@fmi.ch" class="" data-icon-position="" data-hide-link-title="0">charlotte.soneson@fmi.ch</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charlotte Soneson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BSoneson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soneson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharlotte%2BSoneson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3833-2169" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charlotte Soneson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.robinson@imls.uzh.ch" class="" data-icon-position="" data-hide-link-title="0">mark.robinson@imls.uzh.ch</a>
<a href="mailto:charlotte.soneson@fmi.ch" class="" data-icon-position="" data-hide-link-title="0">charlotte.soneson@fmi.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2089" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2089&amp;title=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2089&amp;title=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2089&amp;title=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2089&amp;text=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2089&amp;title=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2089&amp;t=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2089&amp;title=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2089&amp;title=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493465/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493465" id="node493465--3" data-pisa="ggg;9/7/2089" data-pisa-master="ggg;g3.119.400185" data-apath="/ggg/9/7/2089.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">ARMOR: An <span class="underline">A</span>utomated <span class="underline">R</span>eproducible <span class="underline">MO</span>dular Workflow for Preprocessing and Differential Analysis of <span class="underline">R</span>NA-seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1508-461X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stephany Orjuela</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3285-1945" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ruizhu Huang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5041-6205" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katharina M. Hembach</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3048-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark D. Robinson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3833-2169" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charlotte Soneson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2089-2096;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400185 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493465--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stephany Orjuela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Molecular Cancer Research, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephany%2BOrjuela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orjuela%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephany%2BOrjuela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1508-461X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stephany Orjuela</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ruizhu Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ruizhu%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARuizhu%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3285-1945" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ruizhu Huang</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Katharina M. Hembach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Quantitative Biomedicine, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BM.%2BHembach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hembach%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatharina%2BM.%2BHembach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5041-6205" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katharina M. Hembach</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark D. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BD.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3048-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark D. Robinson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.robinson@imls.uzh.ch" class="" data-icon-position="" data-hide-link-title="0">mark.robinson@imls.uzh.ch</a>
<a href="mailto:charlotte.soneson@fmi.ch" class="" data-icon-position="" data-hide-link-title="0">charlotte.soneson@fmi.ch</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charlotte Soneson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>SIB Swiss Institute of Bioinformatics, Zurich, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BSoneson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soneson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharlotte%2BSoneson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3833-2169" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charlotte Soneson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.robinson@imls.uzh.ch" class="" data-icon-position="" data-hide-link-title="0">mark.robinson@imls.uzh.ch</a>
<a href="mailto:charlotte.soneson@fmi.ch" class="" data-icon-position="" data-hide-link-title="0">charlotte.soneson@fmi.ch</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2089&amp;rft.epage=2096&amp;rft.atitle=ARMOR%3A%20An%20Automated%20Reproducible%20MOdular%20Workflow%20for%20Preprocessing%20and%20Differential%20Analysis%20of%20RNA-seq%20Data&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Orjuela%2C+Stephany&amp;rft.au=Huang%2C+Ruizhu&amp;rft.au=Hembach%2C+Katharina+M.&amp;rft.au=Robinson%2C+Mark+D.&amp;rft.au=Soneson%2C+Charlotte"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493465/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493465/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493465/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493465/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493465/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493465/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493465/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493465/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493465/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493465/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493465/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493465/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/933" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2089" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2089.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2089.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2089.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
