<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2317" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2317.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2317.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2317.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/7/2317/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2317.atom" />
<meta name="HW.pisa" content="ggg;9/7/2317" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Testing Pleiotropy vs. Separate QTL in Multiparental Populations" />
<meta name="DC.Identifier" content="10.1534/g3.119.400098" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Boehm et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The high mapping resolution of multiparental populations, combined with technology to measure tens of thousands of phenotypes, presents a need for quantitative methods to enhance understanding of the genetic architecture of complex traits. When multiple traits map to a common genomic region, knowledge of the number of distinct loci provides important insight into the underlying mechanism and can assist planning for subsequent experiments. We extend the method of [Jiang and Zeng (1995)][1], for testing pleiotropy with a pair of traits, to the case of more than two alleles. We also incorporate polygenic random effects to account for population structure. We use a parametric bootstrap to determine statistical significance. We apply our methods to a behavioral genetics data set from Diversity Outbred mice. Our methods have been incorporated into the R package qtl2pleio.

 [1]: #ref-14" />
<meta name="DC.Contributor" content="Frederick J. Boehm" />
<meta name="DC.Contributor" content="Elissa J. Chesler" />
<meta name="DC.Contributor" content="Brian S. Yandell" />
<meta name="DC.Contributor" content="Karl W. Broman" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Testing Pleiotropy vs. Separate QTL in Multiparental Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The high mapping resolution of multiparental populations, combined with technology to measure tens of thousands of phenotypes, presents a need for quantitative methods to enhance understanding of the genetic architecture of complex traits. When multiple traits map to a common genomic region, knowledge of the number of distinct loci provides important insight into the underlying mechanism and can assist planning for subsequent experiments. We extend the method of Jiang and Zeng (1995), for testing pleiotropy with a pair of traits, to the case of more than two alleles. We also incorporate polygenic random effects to account for population structure. We use a parametric bootstrap to determine statistical significance. We apply our methods to a behavioral genetics data set from Diversity Outbred mice. Our methods have been incorporated into the R package qtl2pleio.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2317" />
<meta name="citation_id" content="9/7/2317" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2317" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2317.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2317.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2317.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400098" />
<meta name="citation_pmid" content="31092608" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="2317" />
<meta name="citation_lastpage" content="2324" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Frederick J. Boehm" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1644-5931" />
<meta name="citation_author" content="Elissa J. Chesler" />
<meta name="citation_author_institution" content="Department of Statistics, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5642-5062" />
<meta name="citation_author" content="Brian S. Yandell" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8774-9377" />
<meta name="citation_author" content="Karl W. Broman" />
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="broman@wisc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4914-6671" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Automat. Contr.;citation_author=H. Akaike;citation_title=A new look at the statistical model identification.;citation_pages=716-723;citation_volume=19;citation_year=1974;citation_doi=10.1109/TAC.1974.1100705" />
<meta name="citation_reference" content="citation_journal_title=Journal of personality and social psychology;citation_journal_abbrev=J Pers Soc Psychol;citation_author=R. M. Baron;citation_author=D. A. Kenny;citation_title=The moderator-mediator variable distinction in social psychological research: conceptual, strategic, and statistical considerations.;citation_pages=1173-1182;citation_volume=51;citation_year=1986;citation_issue=6;citation_pmid=3806354;citation_doi=10.1037/0022-3514.51.6.1173" />
<meta name="citation_reference" content="Boehm, F., 2018 gemma2: GEMMA multivariate linear mixed model. R package version 0.0.1." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_author=D. M. Gatti;citation_author=P. Simecek;citation_author=N. A. Furlotte;citation_author=P. Prins;citation_title=R/qtl2: Software for mapping quantitative trait loci with high-dimensional data and multi-parent populations.;citation_volume=211;citation_year=2019;citation_pmid=30591514;citation_doi=10.1534/genetics.118.301595" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=E. J. Chesler;citation_author=D. M. Gatti;citation_author=A. P. Morgan;citation_author=M. Strobel;citation_author=L. Trepanier;citation_title=Diversity outbred mice at 21: Maintaining allelic variation in the face of selection.;citation_volume=6;citation_year=2016;citation_pmid=27694113;citation_doi=10.1534/g3.116.035527" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. M. Chick;citation_author=S. C. Munger;citation_author=P. Simecek;citation_author=E. L. Huttlin;citation_author=K. Choi;citation_title=Defining the consequences of genetic variation on a proteome-wide scale.;citation_pages=500-505;citation_volume=534;citation_year=2016;citation_pmid=27309819;citation_doi=10.1038/nature18270" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. A. Churchill;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=K. L. Svenson;citation_title=The diversity outbred mouse population.;citation_pages=713-718;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892839;citation_doi=10.1007/s00335-012-9414-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. A. Cubillos;citation_author=L. Parts;citation_author=F. Salinas;citation_author=A. Bergström;citation_author=E. Scovacricchi;citation_title=High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.;citation_volume=195;citation_year=2013;citation_pmid=24037264;citation_doi=10.1534/genetics.113.155515" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D.-J. de Koning;citation_author=L. M. McIntyre;citation_title=GENETICS and G3: Community-Driven Science, Community-Driven Journals;citation_pages=1-2;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236444;citation_doi=10.1534/genetics.114.169151" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=D. Eddelbuettel;citation_author=R. François;citation_author=J. Allaire;citation_author=J. Chambers;citation_author=D. Bates;citation_title=Using the R Package crlmm for Genotyping and Copy Number Estimation.;citation_pages=1-32;citation_volume=40;citation_year=2011;citation_issue=12;citation_pmid=22523482;citation_doi=10.18637/jss.v040.i01" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=B. Efron;citation_title=Bootstrap methods: another look at the jackknife.;citation_pages=1-26;citation_volume=7;citation_year=1979;citation_doi=10.1214/aos/1176344552" />
<meta name="citation_reference" content="citation_journal_title=Mass Spectrometry Reviews;citation_journal_abbrev=Mass Spectrometry Reviews;citation_author=X. Han;citation_author=K. Yang;citation_author=R. W. Gross;citation_title=Multi-dimensional mass spectrometry-based shotgun lipidomics and novel strategies for lipidomic analyses.;citation_pages=134-178;citation_volume=31;citation_year=2012;citation_issue=1;citation_pmid=21755525;citation_doi=10.1002/mas.20342" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Jiang;citation_author=Z.-B. Zeng;citation_title=Multiple trait analysis of genetic mapping for quantitative trait loci.;citation_pages=1111-1127;citation_volume=140;citation_year=1995;citation_issue=3;citation_pmid=7672582" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Keller;citation_author=D. M. Gatti;citation_author=K. L. Schueler;citation_author=M. E. Rabaglia;citation_author=D. S. Stapleton;citation_title=Genetic drivers of pancreatic islet function.;citation_volume=209;citation_year=2018;citation_pmid=29567659;citation_doi=10.1534/genetics.118.300864" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the Drosophila Synthetic Population Resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. A. Knott;citation_author=C. S. Haley;citation_title=Multitrait Least Squares for Quantitative Trait Loci Detection;citation_pages=899-911;citation_volume=156;citation_year=2000;citation_issue=2;citation_pmid=11014835" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Genes Brain Behav.;citation_author=R. W. Logan;citation_author=R. F. Robledo;citation_author=J. M. Recla;citation_author=V. M. Philip;citation_author=J. A. Bubier;citation_title=High-precision genetic mapping of behavioral traits in the diversity outbred mouse population.;citation_pages=424-437;citation_volume=12;citation_year=2013;citation_pmid=23433259;citation_doi=10.1111/gbb.12029" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=Joint estimates of quantitative trait locus effect and frequency using synthetic recombinant populations of Drosophila melanogaster.;citation_volume=176;citation_year=2007;citation_pmid=17435224;citation_doi=10.1534/genetics.106.069641" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=H. V. Meyer;citation_author=F. P. Casale;citation_author=O. Stegle;citation_author=E. Birney;citation_title=LiMMBo: a simple, scalable approach for linear mixed models in high-dimensional genetic association studies.;citation_year=2018;citation_doi=10.1101/255497" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. Meyer;citation_title=Restricted maximum likelihood to estimate variance components for animal models with several random effects using a derivative-free algorithm.;citation_pages=317-340;citation_volume=21;citation_year=1989;citation_doi=10.1051/gse:19890308" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. Meyer;citation_title=Estimating variances and covariances for multivariate animal models by restricted maximum likelihood.;citation_pages=67-83;citation_volume=23;citation_year=1991;citation_doi=10.1186/1297-9686-23-1-67" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=F. Ozsolak;citation_author=P. M. Milos;citation_title=RNA sequencing: advances, challenges and opportunities.;citation_pages=87-98;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21191423;citation_doi=10.1038/nrg2934" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Raghavan;citation_author=R. Mauleon;citation_author=V. Lacorte;citation_author=M. Jubay;citation_author=H. Zaw;citation_title=Approaches in characterizing genetic structure and mapping in a rice multiparental population.;citation_volume=7;citation_year=2017;citation_pmid=28592653;citation_doi=10.1534/g3.117.042101" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=J. M. Recla;citation_author=R. F. Robledo;citation_author=D. M. Gatti;citation_author=C. J. Bult;citation_author=G. A. Churchill;citation_title=Precise genetic mapping and integrative bioinformatics in Diversity Outbred mice reveals Hydin as a novel pain gene.;citation_pages=211-222;citation_volume=25;citation_year=2014;citation_pmid=24700285;citation_doi=10.1007/s00335-014-9508-0" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. E. Schadt;citation_author=J. Lamb;citation_author=X. Yang;citation_author=J. Zhu;citation_author=S. Edwards;citation_title=An integrative genomics approach to infer causal associations between gene expression and disease.;citation_pages=710-717;citation_volume=37;citation_year=2005;citation_issue=7;citation_pmid=15965475;citation_doi=10.1038/ng1589" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. J. Schaid;citation_author=X. Tong;citation_author=B. Larrabee;citation_author=R. B. Kennedy;citation_author=G. A. Poland;citation_title=Statistical methods for testing genetic pleiotropy.;citation_volume=204;citation_year=2016;citation_pmid=27527515;citation_doi=10.1534/genetics.116.189308" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. D. Stanley;citation_author=E. Ngoma;citation_author=S. O. Day;citation_author=E. G. King;citation_title=Genetic dissection of nutrition-induced plasticity in insulin/insulin-like growth factor signaling and median life span in a Drosophila multiparent population.;citation_volume=206;citation_year=2017;citation_pmid=28592498;citation_doi=10.1534/genetics.116.197780" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Tian;citation_author=M. P. Keller;citation_author=A. T. Broman;citation_author=C. Kendziorski;citation_author=B. S. Yandell;citation_title=The dissection of expression quantitative trait locus hotspots.;citation_volume=202;citation_year=2016;citation_pmid=26837753;citation_doi=10.1534/genetics.115.183624" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. Tisné;citation_author=V. Pomiès;citation_author=V. Riou;citation_author=I. Syahputra;citation_author=B. Cochard;citation_title=Identification of ganoderma disease resistance loci using natural field infection of an oil palm multiparental population.;citation_volume=7;citation_year=2017;citation_pmid=28592650;citation_doi=10.1534/g3.117.041764" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. L. Tyler;citation_author=L. R. Donahue;citation_author=G. A. Churchill;citation_author=G. W. Carter;citation_title=Weak epistasis generally stabilizes phenotypes in a mouse intercross.;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1005805" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. L. Tyler;citation_author=B. Ji;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=G. A. Churchill;citation_title=Epistatic networks jointly influence phenotypes related to metabolic disease and gene expression in diversity outbred mice.;citation_volume=206;citation_year=2017;citation_pmid=28592500;citation_doi=10.1534/genetics.116.198051" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=A. L. Tyler;citation_author=W. Lu;citation_author=J. J. Hendrick;citation_author=V. M. Philip;citation_author=G. W. Carter;citation_title=CAPE: an R package for combined analysis of pleiotropy and epistasis.;citation_pages=e1003270;citation_volume=9;citation_year=2013;citation_pmid=24204223;citation_doi=10.1371/journal.pcbi.1003270" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=N. A. Zaitlen;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=A. L. Price;citation_title=Advantages and pitfalls in the application of mixed-model association methods.;citation_pages=100-106;citation_volume=46;citation_year=2014;citation_pmid=24473328;citation_doi=10.1038/ng.2876" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic design and statistical power of nested association mapping in maize.;citation_volume=178;citation_year=2008;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z.-B. Zeng;citation_author=J. Liu;citation_author=L. F. Stam;citation_author=C.-H. Kao;citation_author=J. M. Mercer;citation_title=Genetic Architecture of a Morphological Shape Difference Between Two Drosophila Species;citation_pages=299-310;citation_volume=154;citation_year=2000;citation_issue=1;citation_pmid=10628989" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Efficient multivariate linear mixed model algorithms for genome-wide association studies.;citation_pages=407-409;citation_volume=11;citation_year=2014;citation_pmid=24531419;citation_doi=10.1038/nmeth.2848" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Testing Pleiotropy vs. Separate QTL in Multiparental Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="The high mapping resolution of multiparental populations, combined with technology to measure tens of thousands of phenotypes, presents a need for quantitative methods to enhance understanding of the genetic architecture of complex traits. When multiple traits map to a common genomic region, knowledge of the number of distinct loci provides important insight into the underlying mechanism and can assist planning for subsequent experiments. We extend the method of [Jiang and Zeng (1995)][1], for testing pleiotropy with a pair of traits, to the case of more than two alleles. We also incorporate polygenic random effects to account for population structure. We use a parametric bootstrap to determine statistical significance. We apply our methods to a behavioral genetics data set from Diversity Outbred mice. Our methods have been incorporated into the R package qtl2pleio.

 [1]: #ref-14" />
<meta name="og-title" property="og:title" content="Testing Pleiotropy vs. Separate QTL in Multiparental Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2317" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The high mapping resolution of multiparental populations, combined with technology to measure tens of thousands of phenotypes, presents a need for quantitative methods to enhance understanding of the genetic architecture of complex traits. When multiple traits map to a common genomic region, knowledge of the number of distinct loci provides important insight into the underlying mechanism and can assist planning for subsequent experiments. We extend the method of [Jiang and Zeng (1995)][1], for testing pleiotropy with a pair of traits, to the case of more than two alleles. We also incorporate polygenic random effects to account for population structure. We use a parametric bootstrap to determine statistical significance. We apply our methods to a behavioral genetics data set from Diversity Outbred mice. Our methods have been incorporated into the R package qtl2pleio.

 [1]: #ref-14" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Testing Pleiotropy vs. Separate QTL in Multiparental Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rSlaK7T-T368D1eOsUj6sjhFnK1UGTJB83Egrkf8jq8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493460","apath":"\/ggg\/9\/7\/2317.atom","pisa":"ggg;9\/7\/2317","ac":{"\/ggg\/9\/7\/2317.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2317.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2317"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2317"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2317":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GpEGjxskq65vEZz+5wheEjEappaKJ9XtaB+9tzeJmy57V4M+MB9LmO1u+qcWCr6rg0DDx1FGQHkyvO9Ii+q6ovLTpO6K0PVYg7S49XfqowbCZS5l\\\\\\\/+Fhxe2EG5aQd1Vk2kRTTxfnMDl4Pz8VcYYmvzmaUoTuUKFshsCZrJrwpD6Ulnj4d0u0b2MhWvr3nJhyUV1BGbCBZTKhX1DgLiH34jQPgUOP9oAyrQtnFQgbIUNHcegrQH+GGF5OXCM5cfwML8BTJ\\\\\\\/Y7hoDJGxfkhpRTM\\\\\\\/ENKi5yDLOnRXuqwEUZGuf1o3Jgg9m9pDe5qgUsRVQAWpMLBP8J4KVbvIsWXQpxpAq566St2FU9xZO8Poz6os6vsP0URS\\\\\\\/6zplGS0FldWBH1OodI7sW1sZG6aTyf8SfO8wL2RoHQkre6KtlwUH1qLMAL1WFHT0o+G1qMDQWIWn6jaIkq0u8Mt7u6YasU9WjQIzvn+FRnaMfPoVrf5VxK8T3ypWQLMCEaLIfH\\\\\\\/ohXUdkaSiioYwmlmeePQViHvcEIM36sVMBWfksfTxWaCuJipeIRWC5ykVrUGPgSDUSKzwxVLn6Mp6a8IsfBC5zM\\\\\\\/zXaRpn0Pd9oXPhliTqOy4zVlXMDiuHrwypy+ijELq4unBUkRDWWBUnnukJ7M2XDydGWHz8L8n5KJHvet+eqxE1DW+Vm9Ndos\\\\\\\/y3k6PGQgwtRpAVRC6rVot31INnuiPZBmSGHPP6zV3DW8C+ESuVHrLidBhausJUYyWgr3TDhaACSHrddg22Uwg0ZAxW1yMWMwrNMmTmwFhuVr+mOxF9AW4D6TsPwny\\\\\\\/CWvCgmMQGbIo\\\\\\\/qgq26L0OxsW5VzIDyqsp2wUhmnNah\\\\\\\/ump3TA1L8ZWgtRUnDdYIeSEDWL6yhGyxJwoK\\\\\\\/XSQvlZ5gOQL6ehnDAtBbdJIOQIojNeM48ZIR8tSoSAR1vO8NY1BCeHpg0pIPl2Fqi45qB4WZWcZtcLzZfbs99gneENE\\\\\\\/ni4HCBeUn6sqH1y5tvVkuq\\\\\\\/qzq9ZTgk0PsRZ7PeK3jjpwc+5zkhHwsRjRLW2MCrQTDLuAtW4nEdla0XLgpV+TwJg+0t6v0zFBuoqSE\\\\\\\/L0P8FZWvrkuMUFMTIC5zA6aFp6fnVdfL408cL+GD+GA6q0CSLZ3GOsMoJx3oqGEGnzL3zeAP6PEVB\\\\\\\/owcJAs+rypwgCuGO0ENpsdEqcYlzj+Xb4A6XWBocHDH8KSuHDDsH\\\\\\\/QRR2m4Zi4igAb7MhDHSgwFB6\\\\\\\/0S8\\\\\\\/sCq5e9oEPN0XayBenMRs0wpTKRVR\\\u0022,\\\u0022iv\\\u0022:\\\u0022btyPKSOPrLY2xw7Dm1x6SA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002235c646738dfbc9f3c4b317cc5249f3d4c2a9cfbc4838d24fe573cbde4ab33fd4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SnJ+jvTG4bpWR\\\\\\\/Ha8cSO82c+qAa+92U\\\\\\\/GUhAgbgqp\\\\\\\/VHjkyMj548y6fa4fim+1mY3+NOVO0v\\\\\\\/HjC4L1JjdpiACJwtzojgjHAxMQVwoJsszEmUacmQhouTaYbBlE0IG92gNGmPTxd1RTQpxHvrgrpdd7QIN\\\\\\\/KxWf1LBpjm8MOCmJys\\\\\\\/ijORIsOfZvFYzMvdTwLBrmX0Cpc5nCsDj47OfWM7JK7hr9964VDJrH7BAABOa1FoLYByAHPcoK67sdWHtMLJmw5fUBhrUWMJzsT3dcVUFZkpVoueg21kzMD8jfDX6scQzeYL8DgDYlUlx4tXeHHMY+fVaiEKgr8RSx5dwAfQdUqz3D8M5jtn7zs9xNKalcMn\\\\\\\/EW2xyrQkgtCGkJvbwlPInyvJbsmiiAOqSrFFd++KyuxV\\\\\\\/+He4K1R1H7hxW17ub4N7ly3YAtzYUE5X39MUISDDjQmcjed8xtqXpUnhJmZIx2eCWJvYvCNW8mrgTRRsynNC0wIZQOL+EBZdwXosLkTXB23svoA86omZdnb1YQHwrcRo0VxDj\\\\\\\/3SvMRmnYi\\\\\\\/Fdovtbc+rbZM9eHIwpi87FZgXqK4Dx5bSPaxWaD3SG1C0A1Wqn91WddSHdFTsHxrFC+4uVXkHa1U1W1TdqA7PeXXu\\\\\\\/Ba0pMqejHjsgj\\\\\\\/Z14zuK\\\\\\\/h2bjhyAXyf7wifUyInsgyEXZqGMQLkx1SGwBl+7fKZ5w1csBBAlkxopfz+czTt0HDLvPq\\\\\\\/rCodSZ1qeq9nSFFevnBRiyrsZSGW0g\\\\\\\/NV8OwqBvQUhtdXqPQmEbYXYQ0UIszyoZhXlmph4XhmSWZavjyA4Kg+7L2kPWdSr4EK7RF4Y6oQH4Fx2mmojX+rfF6WNQ8NSyN\\\\\\\/vcWpmAPETUVPuparxOtfSmDFcijTfbGnccOIXUojnKeEgoZGr2qF0gby1tKmgUIhwN4q8Yl7urf8gLE6Sp88vKGBkcDYVM7xPHks2exsvPDQeloS3TRRUSjjnuH5cskwSYdATA4uv\\\\\\\/igxEgtXqVt3kV1SspTXzrOB+MXM\\\\\\\/fo7YoGeNzKNfoy76PWg2WtITXyE8hT1p\\\\\\\/DMjfIWEli90z1WusYJP4c+EPAb4p\\\\\\\/ESmgv6JAIR\\\\\\\/cHLDWad5eUsBjx8P9o8O4RoIQi3WGQBv2IKTe6zKeRaR42cZgo6vc+hRp2OK3akf3swn9BzS9yz\\\\\\\/8HHouFrN7luaVo5\\\\\\\/226zQELOMEDqKyds3jNSo0d90uHweo0jHIwyVJM2aAo7SGLLENvDQQSl+va3dSxs9uc4qQP4Fe0MIjDvNqE94ZMctsre+6WlY1LxU\\\\\\\/4Tpx4a5Kwzn7EwJ9W1D9+7umk7m+fCcwG0jLkFkRkWeZmHbCyoUgTpBdnbcMtwVBjVafJmA67GOz9z9o1msYF6xfykPKLUnwwj0n29xY7zBW70l9ghYbyDIq9Uu1SZe8JzodYdk8V6fHc19qZaejW+hyKl5u1RxyltfnmWewiCb5HQu\\\\\\\/VuLunLce9mLeyHsj3ZsZH99+9X3Ly7hae15oIDSuPelWyCiaom1sreDkHCn77fjBBRp4VsfLxKaTlQ9iCB4iEsL5qUlqPE\\\\\\\/apcbMXO2JAIo6UMyk+a6BZGjoL75\\\\\\\/KJPV5ijWxyLqwuHqJvQeSw43Sr9zN6lslMzdN5gu\\\\\\\/fIokhZ9X7AHMEYrC35K6HU3l8qemKVByoYZkS\\\\\\\/qwHA0iQHWQ8AY7\\\\\\\/1grKcqx2cz9AmB1d+Jcz2bUaB\\\\\\\/FeZ4GsEcE4NKFXSfYGt3kZGAwdPsIfTmhgEBIuhLR24A29Y5Sebrv4a\\\\\\\/iySXGX6O7U1ZdEWDt6orYy4bLoVwahy\\\\\\\/3dU8SACsVwWEgfdBJj7FlyYLM96CtGCiWrgfg1UogyQzh2LUPzrNKTy38WNg5Uo3Z3qQSPj6jpixdufr6Xq3yxOODrA8nwnZXdLySfeagYh73hT1F6G\\\\\\\/SjnbpxfnoF2p\\\\\\\/uEYujrRk+Hxb1+ZEQw6vrbKE5b6ARAe11O1e7My68LMgyzQiZgUBmWCODvHBAoz0y5zM3i4q7MZ7dVHdzQpyejvotFr0CTiCr7eohiaWfb1QX8knN0drR6FFpHRcgAZXv14Z5o3tiFJZ2SjUqPJM3trCXBdnUoomGQ9uBj05vJkKZhvRjuLHcHenDaxgJtRLrkQwVmkXpkBcRc37OuzYbYThG9q1X84bMh4qIbwGYNLUJajQhIIMSmTAnMIPSsx6Z8xwaiMexoDB4veWPeBP0QUnTkPWdLRrGmOBYJRVjLlKVvylmM2ylObfM9x+kWTtftJaOVI4Du3s260h+ubHLDgK8dxaUpSi\\\\\\\/+WtzZhQS\\\\\\\/imaM2zeYBWnyJEzMejKc5fMr7nE47FoXvF+5SQmudKxXl9vBIFmf1nQe0L5p6L8Krnf41Kj\\\\\\\/9Totpa6sjcNN05Xe\\\\\\\/EYVDZilzBYGHai7g5Ics1s1Sx4\\\\\\\/9P249GXvXfA88NSijgYnmLf4GCfblEyI0IuwqZl6fZq5vnd1NTDQtoAO1mGhs9OBYzDq95AZpOtjUKIlDIkq56vw\\\\\\\/9MNWZN\\\\\\\/+EtnOmQLHvFtSqo8xFBvViwu7L+TJQP6itQ9329gWYCik+aDApycpC4vLLL6mfxpyKH6ApkARQM3G+iqxCJB5t73OpAsayC9sKSN4DEJi13IycJTCfSwYbicRzuTOUjPdGC3pU9d0s1BTYApcru1jZG9\\\\\\\/hLEVPgXlXwRwcRJ8myUH26TIRfSj3esdoyDccPn9hIXcGq8y9Bfamq+Nng422F4DCG1gzWGmuIGw+U5vaHVaHt3g0eikrvaKYDgvwHCn\\\\\\\/bcyiAjlUKHzCby05PcDrmVsAFMpLJACFO9gPk2UEekdLsRQRKw92GiKqwKxdE8QYfQD5KSoX5z\\\\\\\/Xvs5EgO6mCxJmy5SqfGWfHrxUdh\\\\\\\/gm\\\\\\\/bJQ2Gj8wNdjo8diftZsuDh5a\\\\\\\/dxAocN3H0AHgSebcIMAGQFzY\\\\\\\/J0fKLD8wvkeGIDmCqW36UvhmCQjFBhmacKQUxNzIfmihGVTjhyUqk5vANaNA9bKhWVcQxQQWzfcD7MNjmD72rE8tK7agsOX1y1AVg2+Zsm2UbJRCnuNkrfBw0vFN9Zlw5Ihq3whYP3wRDg6Sqmn65B1VJm8EO3VkOtIRszp97BitH5ym6DEDagjVGGfRi1NXlup40Yl8X+E3Lcp33sDgAFuQdbLWwJIvcv\\\\\\\/+eiNWnvF46zfJpsW4ZOJzfwL1VuIun7N3UdV5LEPIAm\\\\\\\/+kFkqkgPqyuJhrQAPtByxUTh5IY1CsfIQ1rOSzyHzFkC84TXaQ9V9QXH8pgDHvtvnMJombsauAiYH5y6G77lW2uRFeF1+mTLFZmWAYarAAvWyCYBJ8kgPRIJYGlRkc8jaeLo11NLrhpjxM7TCn\\\\\\\/vJ3fTSaRRkVC6s9qx5J2GW2MAGk80LRoa8owd6O3KGivQDVbplzXxMUOEv6l0O+7EkLdwZBhUxR6Pp1rQGk9aOhB5EEDYVc5I0XR0\\\\\\\/DKcSuwSoawe4vonb+aO+U5rr5jl+avCFMkaSbvRj9SpcYeHiwAdpe3fD3YaEnTd4yu5td1cnHFDaOwwGfxumW6khrzZZiOZJrq96W7K0J4b1X3s4XRgBhNLoe+Bhh1ZoPQptinu\\\\\\\/aXJCxyMRSMQv4+8P2s1ZjqGFTj4mRMEJ1sR4tsPmtHXthQZ2ojpxv45SFTQkpYc8oqyYcKAJVwQkO\\\\\\\/epSENOKFW01ImeGcetmGS9sFc0YPGV4LBLXe6kJN\\\\\\\/Ug4wr1xA67Up33QyjjG4dgLLLTgTENyzWmTsSVeFGGqpJp\\\\\\\/U6zduKz361BAbs4SSihC9HaBXsrLt4XV8XhYIEfHpwOoASzq0nFoZNAIbxcpEZ8y+GB4yEOBFY26REzYgEAou5jPcPU5WMfZi4hUwWmH6c0Ven+UcWAU3\\\\\\\/WLgAyI5MzXqR9Tn4ZPeWE0EJCgflC6bl9KS1BzljHPHFXoq1KxyJr\\\\\\\/WiLN95HIOWLHAFNX\\\\\\\/dqjbkOxZYWh7TrojNdX43n+QbayNnVK22xb8vA8EuaXpFbOb1cxobvy6sv9VOIfJTAZ9H1bPFwfjAEr2F\\\\\\\/gYSbcBdW0tGaNoZnr3I1AMrnSZHoO3O58EkCkbqnRaL+orvLn2amg0gpTLspwGZGEHEOP6iLr7mvWUaLboX374bDFcNxJ+ZQacS7BdkfZ2ILI1nQn69E30VBLiZ+UX5vDmHY585aerUGWkIxx2lRoRr4K2JglafgnDMUJ6gCWxbDSAtvrgrK5s\\\\\\\/ZHJqRtNiTKJeAyc1UMydm3mvYUwXbol7Xf1lXB86mCWAJbqBSxWhqV1Ml5FT\\\\\\\/55ucF6Ct52A22RlrAl3b7H0ekh4rVQcM+NlFi7ntEvQUkjPQ+OG27xvZ1omoEsuwARHQ5LW1Jvifwi4SjhyTR7MW9+PGE7qUtLvSybmY09tLawM+ilMIs6JPnr0HU5Eq3WBGH267ZPbRZbg\\\\\\\/M8vva7PAGXIIjWPG24\\\\\\\/5jaNfnsytSPCkHmzwhp0jm6\\\\\\\/OnPZxn1diHhdiWogHIZtDg3R+QsppAX2pmlUWb+7NviL6WRxKDX0OG991QwHC8ZT3iBNHLrGfKEGbim8lDzz834UU\\\\\\\/6WFvEwO6KIVSqWT8ApCaPi7sx+g0SObaexstcJdNK7bSdNMqtb2\\\\\\\/wu+YgWItjv\\\\\\\/88V\\\\\\\/ZPd606pesEtyrcvc+8LRNKa8sMp9DlYkJm3nRRA\\\\\\\/F8Os7l7zEK+5HdFVrveWqH6Y98In37GW8Grl8V4b8lUuHu5RgV1ipJ4ta8yF+orUMH4\\\\\\\/jZGI9or6UEkt1YfBmXCXNrMpMBmJt\\\\\\\/WGCPzbIQWwmFO5YQ\\\\\\\/LrINU56KRyw0Gdtj9glRE1mTC8OKUQ2GQfbfFN3jY+jzl0z\\\\\\\/H3JNyca9H3+nx876iW66sr9mJZrbr425InrpJXCX7HM\\\\\\\/1kz6iRkKBLTUbDMPQsZ8PXC3ylr\\\\\\\/02HKf6F+tDsTAeguxxTVSEuEFTt109vU+Fs5bpLyCA\\\\\\\/cjP8bMEdyT23BrXeJFyh7XqIG5IWGDxOlV2kahqmLCL533lc1xsOtdjQNvbVRWEIBOUU2GdDcQTYQdnHjJ11EVMIH0RhEmGbdngpPuLRdTSalsMnJWCrOoVzyxs+em\\\\\\\/+v0HOJvAiSCBskqU8E6FXWH0\\\\\\\/XNbU1y1WCy19kvhT7N7R2IgnzO5H8EctuwB0VzsH4AHdZF46zEfmrvwISgUY+mhKFzQOX4FPLRz\\\\\\\/ziGJTePS4m7V0FfCJVc\\\\\\\/OKtwg84f1E1xH3bF9LiKvcSqswBbiCuDA5RPd86+lm2Hbsumnws1XA0KotDMwAt8+3zzC9pUpjS3pvpIQ55Nmr8opdvRZDJLuZL+oHyRIrKnaJMvRTXISiQPr1fZvnfwzEAuO7G6E5FXa9M65\\\\\\\/2a1BvEBIilkN0XQxFyhOlJVkmyc1UH4N3bJ4FYasvayjy+Ca1A75bEmZGX2pfRXbdn9NGzFdAndr9XUiH1oDRnxoIdESNuK4HwZHBCc14mldvLUyRi8tAkPCJdvs6JlKvOrf0Q4RC7FHdYRqCDZ1rQssETw5lRANX8z3BT7rIqeB4esPptWv4Zx1U4B94TPkYx6c2UbqAQh4HW1ww57uBq\\\\\\\/jhJoNijkoh2VyV1fGu8KTOxP7Jpp2blTj7\\\\\\\/mZw\\\\\\\/A29DoH88EVCrvSYhKyLZyKmCAegWSE985YzMudOGVgtpefPrtnMbmEvDAduZg63tH9zsgdqvHKqP8YO+YFRs9Au\\\\\\\/4bOhtblcQinLCDzAgVbQ+7Ph2lop5iLJIlmXwJmP5FIUccKmZ99hldV6Wa7DGgGy02YCqiqwef47o0kxQ3qzweeYl\\\\\\\/lD4ppUhW+R7\\\\\\\/lD\\\\\\\/KDLbYwYjhLpFqz1QIq\\\\\\\/gScOt4kO\\\\\\\/iX3zJ+DsF3iZv4by\\\\\\\/uGn32ej9hJY2kG4zC5BTR9mTjW7U6LkVcqblEdvT9EtbTuQPxcqhhVOEc6D3zmIs0ThCKkxbdrs6FFJiM9fQ2PQh2+4i\\\\\\\/wvyuiMTPMrPpWpvH4y4WbjY80eV6llJdbzGcb+vhyM4jHbNiEb0hKtllQBwnzVsc4RVv+tfvR2V1FD8EGLa4L0K3NJVyeBpUWa8imv9Ur6JWqct20EU5XGq0duNdUNGT9EtcJ9MUW8i0dX9k22T2lJHfI+D86HXlU8v+YguSucM+hGpM4yOclvG7wM8RAQW47yAp5R10nUVUK7SKX37y6CNMDvCYCKwMaD++FzkY2VD9eGz7NQ+0iiV475AFVN22U7L8CYk\\\\\\\/4CUdu9DTBZo7zluxtuR9Rt9mquKbhq0TYGkVPNNc1MsoeEJvDEQfb0YExxswk3z9qwrrcv9\\\\\\\/NDjth0M0408y\\\\\\\/fcq3ZmTlZQecMOHzarWih2Yw81t5R5WVqdzeMFysJSdXZtGmo7wTtspYyuXzRFHIPdp5FYQXsIRhckbgpBIAV7ehlSZWaKxHB\\\\\\\/4V04eQaBVmrYKY6G6Zm9mPYiydGCBGTcZ4jb9t3mwGogNv2F+0rQPCB+k1+eHknLS+pgkCIijrcgqMySmQmSLaPCOr3aauO5rMYKRshvyFDPM5WNy57JgSeopnfjU3b9u25bcaB8sOtK5PeX4ymm4soKZrPI6ncF8wwcdIssAx6Ug+9HmlGQ5s2VsCKKsWPcejBGIUaor5WNAC5oheIkWgzrjo8mlgkDYEGfAWLbyEkthuqwO3HPwzhXy6IXeCOiHUykEAT\\\\\\\/lcGfB4I8hvRvoUwIadCdlkzeZRN67oM96RCP4RYHDwKzWqndv9Ibzjdqphf4YX+jPslwMZzXo7fayQi+PgWd9Nq5IEUN3B2eugLzEP4N9YTRAE\\\\\\\/j7FnsDodCD2mGp\\\\\\\/GQ+FUmvGFI0yZ6WjSUndC1UR68kGCJ1PbECAZ1XXMXEI0jFN5SSsdHYkf0zxuxPoVJwFGYPDon7IIZIVHDTxhPViabR4Myxoszwdq6fbQ55QP9pEma\\\\\\\/LClkbVKtcfzNDGxmCyu1m50b7P7+LyBrpLhnUgnwn2DbBHmGIGhAJ3U0NZmBt9mIh3hyaSfol8wL7exDkvu1dcEzsE7PjZnblLHNwNALdV4DVeUAL8yqCjhGelDwQF\\\\\\\/qfTbvu8GuG5TlwraHw\\\\\\\/IH2BfnVhbXh59xjMv8eszCvtvMd2IFvLW3teCaYjCzsv2Zb7nkXe2VTeKwQkVktLnnQi4xVwEWUj0N1YGvk7ez10tCsUpAT51PSIBTJuX+GMhYRtRUP5DOpRPKjluWQwKPUQ0rx1OnzP4MNIpR+zfNTIJ9JtBJJN1PaOch4AnLEhgRWqgQQqYWB\\\\\\\/QB4jKXga2OQfgExvzR6J8xqvpYKOcrsOyc\\\\\\\/b0bIuK+XEMeqtPhCbO2gabyce\\\\\\\/GuCCjBZxfnot3DQWEij92h\\\\\\\/7Fj9X3c\\\\\\\/SMR9bMufT3WL2Vf\\\\\\\/1jd\\\\\\\/7ZGpaffhK169u5QGtEH4qcm9DJ\\\\\\\/bN2BHCHdsTxYQNxpmUtn\\\\\\\/ItxmhKoow+pSHJYYQM2UNmuJyF3Dmt+4vqRCFdYnp9nM+MFH8pTYzpWBJ4y+k2NrNBZ8z+fpffU8lTSq6xSwhf6ozJSgJoXrHAIydN1gVK1tHnbmC2bbsVrh6DjWz\\\\\\\/RnO9Nm3DwiFlqzHEgrbsEImyMx9i\\\\\\\/LF+i7CWcAXm1kHQa5SutF\\\\\\\/Npr\\\\\\\/oG3B2D1xElKrOuf0HTE4mQQn7cZ9sqM\\\\\\\/bVpBx0+I9+T2XwX51O2qBW2BPEgah8qszFKeQrWnw1I24b+GjVgep\\\\\\\/Km\\\\\\\/8Zd3M2rc4M\\\\\\\/M4N0bXmB7b6lAi0ME4PTj5iTYmEc6G7ITyVCuCjBwOBWjhFJcjXyciC0CzCi0sW\\\\\\\/Lb07+yYcU35vEfwg48eF10amK\\\\\\\/T7JzxuvJDU\\\\\\\/i\\\\\\\/DPqMOZ6X9Hyhs+LIVgEFuAueMyupmrZpnwbTPlS+tfx4\\\\\\\/MNh6znDnSzrsM3NNwLBrOraqIGUQIvjyA4KdnltziNfYQZI+x7erV+j82G7tBnwWN3Qe1+XIDJzJGdjvt5jR0SuQAkULWu6na7+N8TXwqQbFKxceGWqIim63Y6W0BC02hBctrRh1VLnvgTob0NiOWkpbnGZj7RXRu2hHdDknLKTNQvvXRMxjTucbSEgI43n2e6kGFnKlIfL5PW5cJzcJy3MnvfwU15Vw+JXxLesyxKxqW1IDZTWdwH7ahvOF5PZ4DM6Ct6763a0FwGSjKmRB3wSRoLHop4wsulJCcq+l+AidGONHNcuId\\\\\\\/BpvOEbJB2D0hk6t6AEgiK66+Mc\\\\\\\/nKxFI\\\\\\\/tmdvcLzP0tVptsxpiLuofS\\\\\\\/4o78Zgi1EsBj6vsmDcxqN\\\\\\\/ClIod6sulSLW2EGasRIm0j9IDHjMet0lDXxri7OFtZsPvvAWXVTp6zqcbkBthkIz+ZUhBhRw56Gr8vxJloorKIU5wLer42\\\\\\\/QNlS6B1FSZ71pUTEP6Gy4D4hA\\\\\\\/Np6N+FUIQa5G\\\\\\\/OaMkRvkguwlkJsXBRUvuUjKpYYwjXhs058yzxx5JZGGvszhZ+6jumF2um0TQAWeOzP6qJcng+Op6HXa7wWO43qaff5u8hrIidZzeM9MR95J\\\\\\\/TEq0FfjW\\\\\\\/f6oxbAAYUu8OoAdWoR1k3yuJn7zuxylReNnpf6FQpamUSKeV6iVoeIb0JT\\\\\\\/2kqna7pRX64N9ykAj9Munc7A3ha1NWPIcKwZOAPjo5DKEV5+FrVDY\\\\\\\/Q43lmtlNGKdhi9oQip6eqd3zpB7G1n6XoGanxkV9oYtBR4XYrwFBhy0zwL2Z4FQ8hrYFwQEufWEfA4Z6waB7LmPxhU5db3BZM4qTdghEePNNJM+MxdYiyl0OIlSruv5d2hpi1PUGfslcZIzwbBlB80GODc\\\\\\\/eqPCE8DpUW1+TqrH3OqLmzMyCtdjbwjMi+OxSnOqBwaUWeJnV0pIv2uQLCBR\\\\\\\/8dQcnuaWUK2IDYYyUPLO\\\\\\\/4jfLAEGVAz2KyOS\\\\\\\/Q0UuOWmhoh7bi1J7PRwqgrMIwOpNSJU1RcoVLyOh6uYl0IYLopoSUtIYBnHRf5u6OjGSucwMqGTQhNZbPgxhmQEWsbSyMlR+jhvGsNI17VIamLighXXXmLG\\\\\\\/hIL9+LpvvZH+LDLnT5MmBk6TRCsN9x9tlHEMLpfzac5dgPnVo49vvTUs6bu4TJ5D9n3pTFWqjDG4XtH+jmoslRNg4gTKwTqR4kdpiXGO3zpGz6jTsFlKDYwR+sNofd1knZahfvTaksCZq21lYXOj0atdb86Btstxp4ioOSBypuntJWt\\\\\\\/0XXtuwWbbm0lSnNo4XaYXdAZOC6DWHxe9Kg\\\\\\\/V5w0HBdRru7Dex70qnD5OZd8ACmzCaL3wILH+\\\\\\\/SDevVSECtLlpBCd2pHxK57As8BYh\\\\\\\/0EEWhYTGzuhC2bHHERBweJs4ayp58Uc0JmbFXGJl9\\\\\\\/nHpXEbz0Q1NSxvhFyJStKNSZwtJuITB7S1DNdsC7Fcw38gX3Q6TcmEaUgP7ho6j9o\\\\\\\/tmGxfaeO5hhE2LQzCfnl+wBG88eFUmJH5QY9Hxl+fyJG6CZ4ItE1nmBB4tmONvkF9c30QTiFFzwuWQjUDbhT\\\\\\\/qKwySHpv1MaGf78triDQ3PQZIP2P9G7H5ualKWnnGkpyikoWk\\\\\\\/cq4X3rHDlTkDZpbDt1bNWK8C\\\\\\\/tqwWD6couYLNNC5MZ2qALYLhVnKWR1P7UoF9E3MNhsDAWhSSs1YPOMdD8wokSUNWOWW3TtP69++b3hcXneyRnuOd5t2bWjyNQvBpqdcqUHagDZQ5OwT7jPWUU+5e6MAKn9Ys5DRAjON6VPsH50eCRhfo2ORxVadZwMDvtzaDbOBrQUyeIG2PH49UH9hznrCn8Ml\\\\\\\/G8AtQ29+vuff5ISRGDOwYK1CTkTMKub6t0wV04d1QrXF6oOIPQFm7NxL\\\\\\\/zWOsuY\\\\\\\/yGWYxz\\\\\\\/YeT9O7C3mv3mwx5Pf8gpFdwaLbyQyST6jdJPZ+nBtkcapmygDqmNEbGVRIftaWF3\\\\\\\/c23m5A2sEEfLRpccebJSdTgv6cacV06OVpephEuVq9V0Wc6YkGzgWxsFkfUt20oroZ9jh32uc4EjVI26zCnVsckfUlSJVFdfjAGmioK9nkjr0kzS\\\\\\\/6XN9BRuRFjSzC41p9BxiDxW3A7PIzBF3sxz5F+KDg7yDn41lc7JFG0KBhlG16U2l2cDtgdSENJY2EXnUfXQqGA2A5c6PIFEWtq1chDStZWv+7lH6jRmmKIMMgdLaJlfuLjXLQPrd\\\\\\\/Mv0UdY8Q2UqsapyxK2szk72qkWtQhFtqXIi45OgZzNLnnBtbIot0DQgTpn3Jg\\\\\\\/v3kfp+UaLcDM5RHiyHgjrwCHeMY\\\\\\\/h182dBahjsq9d1hctYAZKD1CPrpraQsc1+91WjxOvDIrgSbX47ttH2JEIUrOIDZE1eH4Ut\\\\\\\/A3vhVfgFTKHfpnvTs2e2qaVPApQRUo98qPTXEaBOtRB8EoQA0RH69CTyOHfphcpwaeccWKYTOYC5SazqLQ+3pBQLrcr1fjSMnmLL4f7\\\\\\\/zhe1PFFjIuZCc7ht+gtqrGNBke4R2oMdQhlBCeZq1W+xn5VnU4KvPEa3AR+9dEt4Gy6S0LlAKfaW\\\\\\\/ba+Lxw\\\\\\\/n7mjYRlo\\\\\\\/royDvGn9xOCkXiiRlScLU30MtCFmVsEdqTCQln+SMvH8YiU47UJdX7+8pHlYjuU3iIt2lxM9ItWNoO1wiOMIR9i8mUM07vpUBnMUC1ECnVy9ly33SVFIs8YGl3JbqX0f1V0YUZt5KZEPEGTo3TeY3mxVEsYXGiW\\\\\\\/TOZtemDNjPuKdjNUFi\\\\\\\/GKmQbuDYIwczHJq1qEvQ7jM\\\\\\\/ovHUoo6gH\\\\\\\/m9pOJMtrhAxsPAxlNrjHoiMfkIPPoTFKfnjbRTRHuxAaeJpzw+hvJmfpKiQ5dGdtdsKocsllvsYKAKRGQMmzDgBBxrHF810a4wv4yP+SKLI34aJ28N7R4HbYDZ9I+J+5vE9oe5cWzUhj2f2Jn7b417A\\\\\\\/SThSbzlkXrEEVUefNvgCgiDBoIbE7LQSnNlOq3D7\\\\\\\/2LPOcjfE8ek3Nui1mm6VI50f\\\\\\\/pZ+xFkO27YiFyalT3VnisGsr7pU4no+S+\\\\\\\/lUQD9seWQySDKxlj4Z5WhoY1jb0504USJE5uTUT+Ub9Zj9yWFVr653k0MFtx0S+amx7Zvmmc\\\\\\\/\\\\\\\/Hq\\\\\\\/2k0lMQ\\\\\\\/MsZa9Vev\\\\\\\/7ybAwlef1qhl7zV2T0YhU6b9nMac9HReW8MLvTdpe4fntiflOILpxemkpJNH78yF5kUkNwqzKSsKvm2yZJirxnpCOSTttrqPsuA0wI8N9kMI4sg4PpAcFEHYs25mtH5v6ZCvzQjAYVivFz0gNoQ\\\\\\\/5g236OjLKnUo7lszgJ8KhxHAyRwKxcd7eo5HfA8BjzZHAd9XTenkbo4YyQWzj09CPWU8lw8bXg3jKf0lOyT6xao9Bk7CZtZqNaSkvlaG+WVLQb5lsF4I+uW+9\\\\\\\/IavDiRtwSzXjLkJxzrSedOJ7stK0i7mlWAxVjK+zuRjavuFfDupTAu0gBqGYLtQJ3QYpLZ\\\\\\\/nI1+u27uvSe2Tb27Qqst5gIcehuszmrJBBdy\\\\\\\/GotQi3y+vr3KABudCR2fx4lY4b205sn3u6t+uz6w90AQR8JclrLdV\\\\\\\/GBGNtzEqU7ihWoaEWKAk0LrTtAlXDu3\\\\\\\/eL9jEWiFVEMLHluQN8+amaJFYfe1nYyCDwo4ue+oy\\\\\\\/QOsSWjc6Cjq9pAfZlZgVEWdEel2MfQR183mWNbYG+Jnlbu\\\\\\\/sruKnCPEIfYMmfe2Jg3U5gUu47MVk7ukEwp1OVcn5yQpzfTUx6zO6JeR1Ey07LcH23plhnEjNcE0jzbBjCIIHssN4KcTYfLaYNUDYDszdv\\\\\\\/JjAit\\\\\\\/sH1reJczUhc\\\\\\\/z\\\\\\\/vfp+6D9EAxtJYcUFluDDbHpcncZn4teNUIrTlyEfyBDZOJyIUmI3euao\\\\\\\/BiohRrrLlo60\\\\\\\/ivqz3xooesmQ\\\\\\\/lloQ0SmUoyKMuG7IZWLJwXN1WjHkTTEwKw8JYq+Da+KHk1LvJZfvRbMgOlrNsjLfx7GyNeuTaFPr7Gys1\\\\\\\/+Fnd6Fga2Hl9ut2cRO1BllGeFOZO3ItEdi2zb6jfu+oe5ZG5ubcYi6kCPwdfcAlntT3n4CwdaugHwPZOGSG6dUaycwfGYpP8m1GIUZ8Y4bkC0tP1BvmjoNRF\\\\\\\/WbD08WpB09T5vk4cCkhaTO5+6XyEq8vtRWb8sgtMRlF+mFa85YjXscXkk+O+6S432PuG6sorReTaS8jXkAdTX+ogrRfeQ6vVtazX7E34OdpY++KDhOL7rcYu2GN1xxICqEQfpeqj+ve6WHd6f7XggNnkX\\\\\\\/KoclinqMfiB4\\\\\\\/lIhDc4sn71o6Wnc6PrOfsI9cvs2es1u8UqlL738N7YFm+m\\\\\\\/Cjy1OkpEHNZi+Vljjt62vP1BF13Clq+LIQH4DlhvucwaxdeIDit4OMvRecLI4Y96dsgGPHjN9L+JMbaD5iL8++DOHD636EQYLRwI9gdr+DUnjI+b6++xni1vMSEe0vISQowQqm5imZQ88mT6MmkJWbyczFlYLJ1uIhmluzQRpVZr0CB9kipr+f+wQ+AJHB+1iglNJlo0YyzAT+p383CQFxQaIUo5HD0+1uY2vxY7ci+gs9caiwmD0jSuFDr6shapht8EkUh\\\\\\\/Q3\\\\\\\/6Hy6aD\\\\\\\/H4Qkc4lGnz0jS8Dp9rLF3Uacty605jJ3xx2GtcW9QJS55Ypj917pe1JLl3jzw0gdanSWoZVadApEdd0OQbLm\\\\\\\/DsU165v1IH\\\\\\\/jxbGgIzjjdvNeglXScxEx7hMa3orfAWNPEobO4wc7\\\\\\\/AA2BowLWOfLUp69wiikLY+TPbYuuTMpipvW8046ao9OxUHouSuKp7tzbzlU38oGuBfFvO+\\\\\\\/IFYy1nWcSYL2CRnu8Bw+a4pB4yShBujE9sYLTz6OTu5h+3nFSacAtLMtIlMiVLi2MKvcjhnhMtULluvoTX7+N8lcJM9MMvuNcp8Mei+mQ7uL1szfmAX8iNybmcyfxTr32JjzgutGNiuuneD62PTCwos9ACIo+hdXloYYoQX1Vol+ybxQZZHTgQDLjRZFwCx7OBbv6srFsKCwadGczQI9my68B6iASyYNc5W\\\\\\\/L8oXH0KP3PDxnUgFpVdkP\\\\\\\/O7uMS3p89cKsOE0lgOuoA15Z2ejDtVO9JaudZA2GeMMwi62dphP8xfYXdWqyK4\\\\\\\/3q6eXgbw6LgCcsX2R3yZ8YFqyldeQ8FrDT8u5i2uXyXyiaxiruGluyBKN7AcQLQH51bc5HHHEZ3LS1cQY0WJZbsKbtTqA9G3IRQ6nZ1ltKsApu04XJlYpzVbOmZ45Z00QOJWdXfSNQ3CfGsmnKJAzLLWJLoGmuvdcpQhvi3WmdGwmmBcWHqXREepE0Yt5a2w4AenXaCbKpQVFkowNbcp6pPnikO2HyHv\\\\\\\/r4gpkHfgdNmAXpxuhnVRO9qGXyA7brPqWD194IwdD\\\\\\\/NBjT9ACNI+sDsTJX6JEh2TYqbrOE5lYmRy1y691JGvthurSu6mlY00XhgwfXKqu5NfCN9UE3hLTiJTYN420ijjfmEGN6miE1coZc8A+G3iao+\\\\\\\/K1kXoPKl6Gmap8WB9PqtiIiM6d4\\\\\\\/6l+XkPN5fv\\\\\\\/REO\\\\\\\/vwD+hR+Oynse4DhXFzgBwWtVRRnYyVp1h+MINT16Z3ZlZJP8NMxgenEMv7bRrnrWKGtpohsIi6MVj2uPf2BChC\\\\\\\/Ytke+CX03L2e9gATXcfgMZlhr306n6Op4Q8ydV5Hv4+WBdpCgsp1l3SG27wPS6KvuDwM7g4mq0V4U+6yqgC4dFFCaaekbXRjpCp7umpCm2285bU\\\\\\\/YEGFtnqeRms3lfhOBwzH2xnfGnvIvJug\\\\\\\/ULweMYWITWlSt1U5Sk7bKzaeY4BYjWFGaZ9nYaRwv0tAVfZXTtbyTBQxgPwc4ECb6GGWnBMRo3AgkjoP20KKs+OXb8XqOTcEM5agr0HYkYDjSmeVdnHMQkn6PNEHdpoF9Nhr0tdw9Pa8OoctNU74R7lzBIFu1i1+3sXnyeU\\\\\\\/OtewUwoWTnO2LqJw234A2YTlYf05NbuS\\\\\\\/hP0CS0bWdXP7H07lDEKBEoqBR0+MzdV5Dx9cjqahhGruqhXG9N5Scc7\\\\\\\/jNXkPL9f1nV8hPoWr9ds86kXrs51+L6dVublXL55FVEJ9QWAnqTAXBG\\\\\\\/+mztPsxTr\\\\\\\/wUkTLKpkTNpRqlp2Ktej2GCZa2jdJR1jZmVV14apfebRJ8i2tnQTImHILZX5ft9kxOp8nXWSToVhqdc7RORKz73md8Zto3Lm8HsoA\\\\\\\/0dyC0GQbgDpksvwl5KlUnJAnLhXuVmMCaIEDuDH40asCLAEGsSYgEk6YZDgzcf5HEvzZeOSjKh1xua7ojemHf9yYVMOJS9B3iC6vRSIMDem3JtNycVoV+Rlx2\\\\\\\/ah+WWyt86C4958BYkFI1u7cEjThQu6tlgcSe50sQEp4LtrhTt+0aj98oJML18iJldQj8VYCR4nGPpR3D9s4OWEUpI9oMbARh3Wc84igxI\\\\\\\/Y2Fgu6+MNl5A\\\\\\\/kUnDnFEu+AEh4GKPCp8lQG8pJRlNHxTTrFCcQfQZuw5S9p6MvIutWP2ufKLCWZHiJ8u+6\\\\\\\/9x6K5XUeAzag9TuO3RvcGRmDz01ZEhXbQIOM6\\\\\\\/QrE9bxX8dIvmw63YSFaeZEjxoENBOqjH8Agk+joZiZV8AmoV7xUEe4Vf2Ok60QoB7BZJbgeMdzFg8paZD0Dh++G6hLxSFnn+XF3t0Pj1d7ZJV5zY4t4ZetBhK40w6Y7gS4PBpzS8TanX8bDbADZ7Sq35HRL489tlxPnVgFQeJoTg+7PHpClLSRTdIdhpOxUEv3wW8bAo4eU2aVAXOw1utuLobCLYglKSZSSvnSvAOgpWcxSegsrOk9QQ1FtJxg7I5FTpzjh2UTWoe\\\\\\\/QmnqwzwVkHk96njP18XHKHxV4Im7Rq8l7qMSos8\\\\\\\/AUqkTG1kxNZotR8nIwgOFqKEV8ROks9Eos197ZlSzNxtCh3ohiDh25Uz8ddwBWdymcxnVg\\\\\\\/zcirGcRZTRl3dn9OyAhoOYW2vkzUTBZLgcTKTHJY1h5BnoMO7CmvIv9nNEkLZ1WPDOsCr8LLQhPFchAEfKsfmIEIApDnuFfK8R6JFhB7db\\\\\\\/+9uwCssCKd49OpZ7me7bHSvQNhhC5ovsYzvQWwWNBZ90XCOFX4aJKR99XzJ9lrc6HE5CdPu6lxP3jOx80+\\\\\\\/Vug70v908eZYCehc\\\\\\\/BPv7WDpyBSCKgFU7h5Tf7LQ40tuULnSI7Sqp1fBJtQ7Knq10ZnITyBDIHUVJgSe6Kxzc1xVTIhhbcBrRcHTKEAbJA4oISa1v4Fw8FD513YCdnb4E95JrHFG0w8ppzFJPW\\\\\\\/7HoZHJoYwhd1fqPQJJPGdxXOwMKSbE+Fpt+XqzQjdw3TS7v0F2+g5BrchosY1IBc\\\\\\\/\\\\\\\/sLg90L4cluQ+VgRzT\\\\\\\/uEfilXZAO\\\\\\\/J4HzBOwz\\\\\\\/zMVRVDH9zxYmWkQTw19A9Q7YenBv8Nyujks7WZm7ylj9OyZNVW\\\\\\\/u7LGmnYwzQRNjCwVmyRsGOEozjrCu\\\\\\\/zOUcizNO59v2yWReU9ol3WMoMoHp3Y+BI27hwEce3ga5bYVGrswNN3Zs+iJLns6KPOYbXhI6+fj8t\\\\\\\/8ygfJpB1hA7oTBYIL5Z9+hVcR3VPdv0SjZVUKiYH9B3Rqq4JI8KgcXikq0a69lTpf2hOgxXYKjsa3FZbm4wv\\\\\\\/+b9taENBRXsLiuqzNTFT2n5n3qmlSm7+J341vRwILTXRJK3gH48G\\\\\\\/K4btn+gHXlTs0BlW\\\\\\\/tmJgN0Tf5GIG0H91qjK1Qs7ZVTDdIPkHIrQO6XtiB18azNWcEulU2bxQxPjEdHZZQFNAb4o07bdWCGvlFHnF\\\\\\\/aXPUnp2URXElcqJ7otqaAixqHeoXxi\\\\\\\/1+3feNQD7Y6H9fESQ+R+EcNjtf5aFQOGcAoAlF5AcKKRVu\\\\\\\/u324dSfY4CTUToDhJoKhDaTOXU6mqptG0g3laIpT45U24NClgSpOKJIQKtqDc0gVNCRRnSLmoVtn+ZmOmu\\\\\\\/1N0AQDZUF6lNufCmiUPzBTyh7NV81Z5BHIiugB2oqWLVGB2RsCGiNJyAv0Ldk2HIUH8wsbS0zXHb3rnxlcjtGuz+FOQouOsm+wBkiBz7rv4L0uuFOvs5Mu\\\\\\\/1QZPsaRoXgnPkFEvQpCrczpM93nvjeKfxgD8XqUhYSb+U00DByFgm3ltmUuqkinbeUsA3vQ\\\\\\\/6V1owhcQmEadP3Kfi8tiUnzLq1EV8s7u1y3sEELbZewNgHirkDO5VIEuE2aaVnCk1X+g8CKloOljysg4GfBY8wqSc99+wXwxR+D+LQdbp6rIVPGl3plzkLeRqn4toGmCuaUuUHpTnl9fJnn\\\\\\\/aDAVyaHPEhfv1clXedU2Xh7Iz0WXJkHAdGHHo+CaarpJXVcXtcIF9Y+AtFAm6GmgmtrF7RDKU+aoCuHZbKFPoJSYMD0Oe0BEEYCWegA4tqCu58z\\\\\\\/B6k7V4DCTDgSiiuEz4Wrct3ggwlqMF5TKVZsGFXfYvoeZG1djBIk3wpaV8KG4qL+8vo4fTewRhbNDdcM3OrPp0+Ucw3ucTyqsP9NkXaj2fLKiA4fNYXcHJRkcgXJrivlFiC7MhnzCQcBgyYHiqfwlSbFskjwQrHWB\\\\\\\/l1x0zpwkpy6z1tlC2NEi+ZyoTp3XHXVNeuBy6GBJz0TvOCYZ6CxCwfIP4jEYBEsMBku55YIG7N+kwVbWoi4EejelLwETSsOsnqJqRbaS1oc0wJpnnagOtANTDeJmq+IT5tD2BfPJ2TT49RkQRiQnohebsGgEuGyZ3GyMGK72AiDBEtLaIpthLeNs2Sv3BGzlRYPl41QM9339k9mTc5udlq1l8rBzh6Ai6tdDIqPIundpZGlVm\\\\\\\/7zUOEumQKj+D2sRwIeTbIpJVfxN4nlmD33TVcsIrafczImyBOsgxSIE3wd9hRRgfRnZdSKJG6+cxxZUIufNrEbfDm1lRy+sLyvPtfcala\\\\\\\/ganWLQ2HVCwgn6PVdc2iyQ8EakP79sGwL6rZlY2YiBmqg\\\\\\\/utpZ1\\\\\\\/w7gD8tJbmQDUCgSXXua4NdcVt2BjL\\\\\\\/mzPSbb2C9OpUYugWyM23o0UABio1+Cz2GkBTYlge12gMT3QLG\\\\\\\/KOLQedWWUaD5Xcd\\\\\\\/p4oCRgiFDdjExGVojck2+TxKYLD+ikgNax1Q6iAN\\\\\\\/ALBg75mPtFhPVuO6T7ky86fhK9PnH6r7BaCyQUVTxByon3axjc8lZuUyFHDobBdxbXDcz3HN1IQfCjopTvIOVv72ka7EeNrQJ9BEC23S0klFbB3XuJDj2gAcKY\\\\\\\/GGs8BE+CZm1EaJfNDTB517ojlJhKoem6Q4mkkQz5j8SGkIjXhGh3XDmiaOYPO9zCfvRhGRJTMb3Uj+X7o8mGAzQ1QJp6FAqXkXwUxjwAzjqDCWREpS+xNfO1vIqr0qwODEWU+OigX5DVJbcJnADtDg1SuyWzEM0yI15b2xe4Yv2Uj7tdLE2cBQcgoF8ctFz9Z4I\\\\\\\/fAWFvDss5ksc0h6wO4vvEJb5dJAforMKw3kj5x4rJZf2QtGX+fOex8DeOqyorTKOY1OwjHJkBwq67rLrPX8WDX6cnGPbZ46hPyYV5gZkVQDb7l0bzA60W4guuZmBMLWYNHeQLMB2nXOmjWe6ZoHXROGd7T7k5tdvXk6DGD64X1u9nxdQhd6JTtgP5QVmhu2MiSYIDklshlKvvLpOaUzoPK38iJwEeeJRTbRcQJ\\\\\\\/upGTEg\\\\\\\/UkM+NVBmaxcmUvnAXS3Ox3MEUaa0h7rDJ0f0M7RQwg9KTO1Kah5Bk79jdUFznUnrwUAR4ysvGu7f4GRQEY5VGMoKwyh\\\\\\\/v+PeQfeveMQs9gjEzLfNnqgIJwigT3xQKPqRp1BL5OfnFfxq\\\\\\\/eD3IztdKPUn+Jcw\\\\\\\/fZhwRvPRNX7CClg\\\\\\\/CzL0ulYDACslcv1vbHJx7mZpP7GTWFCeZldIWZ7iE\\\\\\\/CTDVKIKd+J\\\\\\\/0uegTvPDjvPqP\\\\\\\/ZKPGQ6qaek9vF9\\\\\\\/zOYPJ1WiUQI8PrQ+pqKmR0UIP5qYdkoLArg0bbQktlvtS3s4JSULXPFsKQJeUgOKSOrW+PPobXltc2OKrLVTCsa\\\\\\\/ahf+W7Olf9TxA6KGHqhOHPoH9K\\\\\\\/7plPP6llaYjojPH4iH+VjUKNSRHnv+ImuVlPV49hy+mhrhkvoYTRIiwHXhF9plASle9y8spg50AN+BxtvC\\\\\\\/iyrBrRUNX3L6MlPwP5zILygp\\\\\\\/OdqvwYC5RfZGKdqprlN2u0Jy8m9CmCmYg63V3eyBbnz6DP4uVtde+ESv0eMua5VEUKMnR58N7OAqNdhvuwnALprRGpdbL6delDaGfKPc\\\\\\\/kzIs\\\\\\\/LpYxHzBHScN9Qyt5it8+hcjDHyYo8tUNBd7S6jgFRT1p1tf0nD8GdJ+msyVtmeLug6A6jSHz\\\\\\\/2XHpLHGIJl\\\\\\\/QmmrZkj\\\\\\\/x\\\\\\\/t6P20LXmvy8jk1oJYxg0WJaeyRkywGgz8UzCN\\\\\\\/jXmhS7o6wqGRbhZGfmqcppzXR3ElbQjbcsi9zQngi1CNZQZ6g5Bpy67rUzWjIYAd1xaRhtxqaH9jkj8OAoDnM9UzyGqf0dHrwkDv5pPE9Hp1dqDxuOiiCCVY165\\\\\\\/2GN\\\\\\\/IePghME3fQ22eByY6Qk6p+NpMU74vNi8LsZXuBnxkrU1Cq4YOi3k8q3q6YoC0OFiXrpOKIvz62Pn0hlCG2J6gtZ6GrjeSbm8Xery3VllmVmFuZWSn1Na7ikGKAKTqL9FaBGbgB5HuwvnKLHLX+c5pc3LsXn8le52x66vdbE8qEptlLXHcGIpQ1\\\\\\\/iNHRRayHag8UC+csLsSgxVFS2oXB5j\\\\\\\/nuMgcRdDiKz3g4fkCmi2knpsPzarVSxxGSICpsXOgqs2QIPPKKphm\\\\\\\/N6wCcYOgGzu0lZ4XgqahRRghdRdEU50QEfk5LZCGPraHGUZ2onNHNQNLPtN5BBx6jThl81vB4T0rteYnccOwhLw7iW\\\\\\\/ui8j\\\\\\\/sta+8iItnjLOs75mZYfhAidk90wau4dvnC3l8fj1nRoJUXtv5vEskZcoo8FMF3GNvLwd9HHCjmR1A4gdPPnmD8FcN99ysZwKoPtb0JvZ75SYEI85mlMcXErUiF99mxJjHxRG3a6K3o1E8sNwUji8sPoJjZQ40e3Q5sT3XNb+5Oaw0RkvQXYGIsqwOGb3EJhE0onY2SZpjMFl4z1YKxzAH3fRxviQ8WK3QD+Kit1gsAS2dtACgLL77ObPpdCMfi0BFPV1pc9QmrElTlQuDO9qvHM\\\\\\\/bH1AxbnoBB\\\\\\\/13gk16KRhaq70dVVA6NxDXUNi0J5UAnvyCJhB2TKmocGQgpK+Zr0tGnMuyBLQCfmyZLX4ssFYC922SiajMkjBhh24jlq9Uf2Oh+FjIe9UDKIjPZz7IVxII2YMXRhURuIxCrb58HCpiS6sSqe+OBZ3zIA0JGt\\\\\\\/sAaCazZ75yDINH+qDGfQE+HuUdto2S6EIgHhne2LkjUdEMN0MD8kvz77RD27OXTiYOYvXevotuEbBlDNR++hcdj+tz2tdb97vNic0aS7dUvgPiCyZu0755LMk1tBMsWzW49Kw7wM8M8FjMjpdkDzJU3rcH1sZ7xMtFOXfULE1wWbq4\\\\\\\/ZCgKTene63TRmDet+GwViuFbQNX98f8+pNI0WOGfA+Dis9Y8ke\\\\\\\/ycO3CRALiBHqhSwijF\\\\\\\/UeFQBapgDRNI5ZPKrb4X\\\\\\\/+SNAjQzdnErw9mn8eD9EyfpgEbchKRyscsui5Qq3VXkJ6x0+lMh8+0EPUB3TuAu8kqtyEABTMp9Kyx2BgE3JPMfvQa2C5bz2BYmKfZ0TwbHe8AnuiD52U3JkP5rkoNn4wTIHLzPcM1XUCm8jGhnBhtdhNG37YxI\\\\\\\/CTfWYXrrM\\\\\\\/C7CT6VvrczMk5SEZ4Phi2kk\\\\\\\/GO30uvPKaIxOtd3MSR21HGcL5AKsk27FfreV+rfGi2zm\\\\\\\/v8kSYICQO0oy2hN81+8uYUSju7zJy7BTYCMYgE8Kx8i5xs9BugBBM0qbZqzfizhR4VREpdVL3iSGwJaPigFOV4eIijLp7DEdFH6FrAD29V5I3\\\\\\\/0OVmnJm6jdNHDLVK9mGui93GYwzuBCBnnQ\\\\\\\/S6FJ3fzGmujLgzflG7190eIpIbUSoMI1rZwOPsk9+o42Vd9pSdGWXOU3SLKehUWeuVntYc8NQofLWYRFtdLhBOIkLRMzyOjJPzGpDf95ItyT+HTd6GvnVvx5N3jmc0G1vVQ4FAq2KBmsifcEOt2dQZFXkA4HFnXAccQJUWkTKo2Xgp3EPzJE3hAO558zTSQua0MSPrNYcVkqYScUhQR1F14n46laC90ALsdRTBn1oW77UVToJdIl2I2580bTX755qzsbzwcMsvBWpvfkseKWhDBe\\\\\\\/Eh7pt8z5w4smF+Fo2wTy42F\\\\\\\/Oqt9Oa5PFJaVnijAL0BE4\\\\\\\/HR0E7VdFv3vZ6P3Hiw0\\\\\\\/GKjDREnSgWmRPKcGcxJ235y6FjNH1E5nx\\\\\\\/lJRjrV3kLz5eh4S3\\\\\\\/qYLGPUWTS4nw7Mj03KyFbgAoubu01QNNe48jxuPMb2ROwRVdcXTjAwXCKKsT62NYepKD+1JboEAuN4WBs1OfktJmG+ur9fVMVx3ehBZ0pv2v5\\\\\\\/NN4c2nU4nOL0FIin\\\\\\\/\\\\\\\/sGim1AzsFkTPK03vcYMs4Tga39Ikb9cAYvhwiaR8pl7tBHJHfEQZ8758zEOIFCVVoK\\\\\\\/nLdYDoebpjZEYssj9QyEutMiYOHYfz\\\\\\\/eE\\\\\\\/Mp7Ghi8QITjFDj8d79QiiMorutDvixWBjaVmXio2Ycy+kblQGJ2r8xaqcO7cEaeYF39x+024XrNRXv1gDE2c6qBfOZxqa12nGH6rcP5QwhglAQBoSt+69CdMaOHKc9jcNKRiSwi+tZ8mfwiXm2szGG7IQH4vS9xnPQ0ng1KgsHMqh1sseK1oaiEO9ZKUhFOJqQp1S\\\\\\\/nTMk8+2GA4ZyYmJuexQemA3XLRIh91igrYWYuV3jzWk1zEqG25UdqOoydbZuf9Ws84kGHu02hVk7UPIJ3dNMfdxYg9NLOMFg7Y147PyBFxdZsl2RaJGzxkCPv\\\\\\\/k06ZFrpbyjbaBWN6xPJr7yvQ6Pj4gOlCtl5VCucA9tzaT5av53JkUs5F88CEITA\\\\\\\/3A8b6tuR2eXPoxsVLz6lyM6N9GAlDhMvdVS3EGFE4E\\\\\\\/kHOANl1mY5bp5\\\\\\\/MOEVaG12EMTYQEV7jlUvmeeOnDeokE88+rVeX4tMSbO2XVNqHy7yN2WXqYUa3CflFxYoLFADE9SgSv9LRlKS+8bZqY71+0mwpJuOvoguH8ljSFYWWV8aacGGQTYZWy4rAnABKAdMReoDC3uNa8ba2NYU4o7ZotVjWdevJaW8uKQ107RiisetsU781bB7c6LtZ82mQlXlffPT6kcbhZlQAbaxcSqsvGoYGDKty3UsTen\\\\\\\/qLDIT7\\\\\\\/Tvi83GpBIFEQEYMDmR04F6YzHlsD0M4hT2XZw+K0XoSoZlwyox9e8YW+2t\\\\\\\/KEdjQN8dS1V8ZEZ+vvRuwNH5VesRG92VW9aHBni+EIiTQbBqqGxJi3PRU4+u9fe+QQfAWmX1ey9ERl6Tu80U1HVIg8nWheuEF79SixFaA+0DksRSAGzccgd6PYO7wXSZ7HQpcvdXS+TJx80TO4Zo0uekE\\\\\\\/V1McvV8pYdcq8+ytKpGqcdtvdnfVlweMxSoCxNLGx7gm9Oxn7AJLitKQr2Y\\\\\\\/CIg62Yw8jxWtC3vcaapr7ZXbZsPotmRanF4\\\\\\\/GRrZ8VpICM0uUicbO2wPL2LdrFF3bokqvkwm1QdPYnImYdWikAWc0S8metbZSWPCV4+D41PnJbSt\\\\\\\/k5Q6t5dbAyQt7ZlOqpL9bCBmiRQ7+ZjDXnR0r9uLGjBn8mnWk7QRP5HL9Ws4+wENlt2qaglx7wN2lbbgnX9zySlRJQTffItgwE962fWGuSSY94MeRqku3nGQuc8U3uhTbR776Kudexw\\\\\\\/keGZHcl+clvXXdtpaXZGeLyqVlsXOktB0\\\\\\\/4NZVswJqY8xnTUntjBBe3mWj8MC17ZlqRoUeCm7vGzELujc+nIgtg5ug545YeziDfCDj6TJt2bQ5XqI0LsCSrFf\\\\\\\/GsYt8KCOxWFmUFFeLJ6HOSt0wxQgypFFoIlnz0GReB01X2Envc40Elc3z18kJXRP7\\\\\\\/5aPZw\\\\\\\/TPf96pvTCGU9R\\\\\\\/p0w8arY+eVVUe6ctykP4NRPHbpxvsqpN4m77lbrJIuNHnBuVnsbnhYlkbz1eAY0EsZtHQ4AW814TxngTbXeAK\\\\\\\/m1S4l3QUjYTtG7mfKwO1J0A7qmHdhIpkgVJLn5TnnSD+l47lMODldfR8Vgx9PKRngldYsU51DhewBANGSYgiqjCe0Ca7kz5lUkP1e2\\\\\\\/YJ\\\\\\\/zwBJtC4I+hBRfvA1TJpImCrrRiNd1FcFUZnavUtmxmfjKKCewWSLbSRihou3zQ3tXKXp5Ry2CJm+1Es4wojX1h+uQWaYoCxjr+iCBPbkxB5HJqDaQkDgA965EaDo2VLH1Lvhzkhl8+s4\\\\\\\/c1enoSYOxB4iW9syHAnmFfknj8AthWfB9JZw4OOOG8JktBdJ+y98mLWK4wdPBV86FF+ommEmKahBhj0EmYzBXWO8SZ32H6aIWoq6AV87oy0Cvb4bOJaGojmD0GnZGh+waik3szfTFtHiJxdmqVoaDsvYqNbgpoLWTU\\\\\\\/EDoR9pdtuRqsp156y+N+VZWYkw5UUQruWev4\\\\\\\/A2jeJ+3PGAenw94+r1E0d3izxgJKTJGlAhJqV+UtHuDtRQVgPHMacJhVtYQJhAAOO8ZzMt4SdpIbU0okVfYb1TAGX\\\\\\\/lRSAYpV+33evd8D55zTMwO9YV41zFy3WGfhDJpxjsg7UWfi25vL\\\\\\\/R5mVUpNyAHhdSjstnei9rKPm+aZpGuSnE+PEm\\\\\\\/V78hNyFEkzQwYlP4d5B18ejg5KeROApKmLzrbtTcer4CdElqXPDD2XAPeiqZg8eg02NlqXpbj5gKFY1JRWmyf\\\\\\\/py2hajeFLHCCHeNceIsZ37tFs8eT+6vixyYyiOGwFRSZuhycslI6EKkId34iThyhVT\\\\\\\/IeLxOUzrWsTh1epsw3g8oKwq31Rqu5p328onJBgpvuD7YCC\\\\\\\/8IkKPvHzadFyjwufPPa92piV3MzFJgae8ijqDROIVq29h+2+lj5NDYJ8ZKdIzBB4YsVS3m0zXABmSWrLCf169iftLvVZKikc19\\\\\\\/2q8SphOnfCYp+cYkNjzYglWanDRyDHXXpd1BGBCeC2SiqQYaEghvGJlp5pOjVL378fZ+YFr2ztUXYSflpU37Dv6kWmhiTyB7eSJFWQwKtN+Mdm1mwyUu0CQpCuk0aGXISjUA\\\\\\\/ELfA2U1qCiMRmEaJ8snXBpO8Ve9AX4EnpB6kz3w6m8VC\\\\\\\/YaKPBkVxetO87g\\\\\\\/gYQpePzAzmsd79jMWyEM6GlWEX4+94bT3NYDU0yW28HFgmxuC9itcA0y+jlAoBJ9tI7D0lpP6sWT+RMxO7Y5fTZO+XX8FhFccCzkCyyn7vtXanMdr0u0O+izR3JPGZjYn9lg2IzqVhaYA2vC7xJjVGroMLRQ8LEJ0t27R7d4bFiYw56qAOWuyvaGHvuXm3SbxF72rqadbsRmOaFDxHBYn3Rx4leeFvLRzOgpuUVxLG5y8sZDXeR2QcYpqkuoqCgPXZaO5XWmOxPRWtorpupwZtsoCbT4VVldXEhUPQXCFxF6T8Wzt3WYv6VPuJxNj93\\\\\\\/aRbo+Ml9s1Ne3YhDzv2RhS\\\\\\\/WiOi\\\\\\\/pmxHraeAA6Eilj7k9MzhVpuJN8NuHTHUDmaS55M1cYgOJP4sPGbkWqYduv5v9NkKZVUPrj9lAZGtxucYiszXDKgzOAJMBWQEJ7h2CyDoiWVrv1a7p2JRYGsaf1NfB7ndejrEKdOmi5A7QZPcOZw1B6ffGCqBE7LRnRGDPR\\\\\\\/cmN5PbaMnqiZ1J3CqtnypL+N71D\\\\\\\/Z+ZKu7lQPxcRCdacgswxTb7K1Ht5oMbVLmpFZC8pOtDbQoSSHgr5AWBnrpxXtrbo0or62QxPjvm88eaplt7+q030Ow4l1\\\\\\\/+yN7J4N0AOYB8rRCLzkcdEHMnN6t\\\\\\\/7sDl1PO9xbZj8il1CcAFa0rogWMY7Cz2CezWYjW105mUen5zBPboHIBuSB9Co8laiQaxQik48f1+sVUpFCqnGdgUT+szWCBRVY89RbpZO\\\\\\\/6QPHXowSyCCz6oG7EONedBr980zupSMRbO+0IPF6vi8vkduiNDeVsxtdrZaeZb57SJAfqtQoXBUfAIFYBlS8y7egmq6jbAkd+oM580zONuy6fHyV4PzBbtNU7ofpRzMqNL7fQRoSH8p94sjn8SmRxTRLMJhdt8Vz7ps72k\\\\\\\/l+uzRETfWec\\\\\\\/xM7wNtiD9oEDLAkXs+9feFlJVSDCvpXUxVmOy511hEiBl\\\\\\\/qGleMZ\\\\\\\/fLno5HF4ihCzuqmc70Ita4UWtxKaDYclM1MKlMUCAbkZv+H06wb\\\\\\\/uNHCZFEf0idpsI2VL\\\\\\\/ZU1ohzMWBo2IJBFXa1njul0EPfIKM2ikkL2q689iqxLU8V\\\\\\\/\\\\\\\/UvhVnjTDZLkfoqLePDTgdKP5r10x9PCdyNUqUDXOqd4pmLLkpU+VfThcHhmmmHNYkJYOC3Ng9HNTw9BNCtrD8mqVYTuuC8WyTt6wWPSSRutPwSoKgm0Thl3iahkApRpLdMaQYoB0qi9GcJki5WqKmx+C774t7wm5vm3hfF\\\\\\\/tDdLHfzI56Lnev3nl+Xwc8p\\\\\\\/i03nYeL3T1CWS3UBfSY2vjNg5kyheuy3Sc1t1EZ2JIWRF2xLkxbYwXvSHdpE3D0TbLnj5zK4A5KFlMfKPgzL3d\\\\\\\/neonfR3YyfbymGGCKtuAP\\\\\\\/txdhh7wki0CEnz5yYvbYD1rPrnk3g8kfEA9gy0ICB2mJqjydPD0SRgQJ6b+jzgxjNI5dBZWlwd3\\\\\\\/N20UberYiYA2HNgrgdI\\\\\\\/+N7T7B6DfWirEq0qwbM6WzpbRfptvePQRzJl2a7gyXKbHaw6xSAeIyNAwICbTctNCfK+nGo7eRS+Dfw4\\\\\\\/KR50rIL61gpkOAQYEFrg9bhcolMt0VDjEZQIQN6mB2sNHsHPPl3oj1PL+qeu6QwzyWdWXFXNcfYNXIG8rGgPTjjsYVdeeDeJMvouJZor639Y15zJ3K8Q0Y1fGjnVpYpIcmChM21DkCp9DEHzd\\\\\\\/182JQhZd4yTwkIiBBa3EXWEZRYwEQjXXbqdtlg3E2gwZtegdr0cF4T1MmSW+BeeOkk8pVun3LvcZ5\\\\\\\/Zl9fURHlXm2nFk8Huts0fOnjyn1FmqgUVO2cUNx5RKRm6BcE1iNtCm60l7FP4oICvXnO3OfiWDh+oSjxORVjz5zRnU5ET8yJVYWxKUGVd7LQ7nZ4GWcdeU\\\\\\\/Svu3hEPblaIL7UPYI7pGEQDYWz6j0GdVbUmgLWCMnq3BowPJvZzDN1dU+7QsWCPH6hKXGYh+j7VY7+Ui7hePXNvtdsHS4szdkxTfal8\\\\\\\/HIednC5VU9qM+XBUH57+PIuKXz2jfpY133p68NimgTAmTUkL0WENUIFkcffJnNuZjbP1vY\\\\\\\/blgA94zqEJhZhucg8EhTYoLZAhVoUZBm6U3UWfn8+i3tOChkzknoExOp3KlpMH\\\\\\\/kSd1JKvn7GLstyWvnhvsZYvF3ShkRNG\\\\\\\/TYRlPVeqGp3e8YiRqyLuytyGAgxv4ExAI1v4tkilNEY75qGDfCl1\\\\\\\/JsjVaiJ91J2xXqlMJA4WAZBhZXf5lE+Ha2nHezyFcGVPbnfwvgRPJZWXhSxHGSYiPO8xAf9gA1czIysu1fijPBLMEOEY48vyAOnOkxSuiWhLzRHAZWFeoltku5l8P40qSAb1gVtoTFs5x3u\\\\\\\/uoyJFuHNAx0ZAU0RM5P9jiT9jbfVZld5QUR6t95wjYGtBeVhUAy7PsgBhYpMxY2jvoGIMqHNU2sT0J4xZxpVbxPPUXBIiQ1dHcnAPcr+W5sD78G+I5ouCZAQjRAO92+KeJGS208g6\\\\\\\/vHD3zfJjq+3Xi3EiEQmxfCdRZTXSDOg\\\\\\\/l2K8lNYYDICBUxSGkHwJcJ79vJxfmKyCguYIsyV28pjBtdAT770BoRYVhYB\\\\\\\/YOfCWLjGl9vXKIE\\\\\\\/B72B0bxOTx4DYEcOEhYjFJItnzSLqAYoRwdmBuxzGiZ5aJtMo6SaTX+x0pILj078qNDcyXBR2NI1x6+JgmV4uZMxv56QW6NZmrvPmpTAjVTzb+uKXvQOpYTUCnshPgZoG+ulBDN0K\\\\\\\/fFx72Go9mfNCQeSMAREsrAG8bDIec0gXYoqzSUJaUYy8r3Nimc2Bj1SB562JhC7pSS12923Wtl7Mgyoga\\\\\\\/APt\\\\\\\/TqEIDf36JZNje0flzhqyVsN04YvLWaSb\\\\\\\/7YF8L73fK+1UKyciUFrmn362Mg12ubu1jfWGITfdUYLUvQZ0qCXbwuBcwsW6ZSGe6J367ss69B1W6Ca4w4VuBGmDmRzTmiDcl8EO0ESF3ocCiVrCg4SWpCIQBtfEgLMAw23L3VyUK1Y1+zO+RpEDmXLPAkJbb+z5tKh3KoC6QwcPlvNzy91Ewd8LSLF77JTq3U\\\\\\\/0VejL8JO\\\\\\\/k6fB7biTHpgN98Yu6vZMSttClvcaiSQ7lzIl2lWUV51t3e4BQN+SFX0vUOKYqdix4zJAEpmJipnQus15EB5Fmsn3riwBl4d4+K++gO3W6h0hlbghxFnLJp3MS5UfHCu734aL3wg05eYbALzrHO2F7T5xwa3cAMYB2+29muC05sWH4eVlC00M3K27yr4wo+qTMF5qxUYJ6c\\\\\\\/v15wNiQwKq88pYi9O6h+4nKPBLFmQdPoF04X8GmUzrA+Hz5xUH4iYpzv4uKIZjZqujYGbGN8pIigthi3SxZHeNwIxM8943ooAveNuyBpWbX9+LY5Abaq3xgfjf4U\\\\\\\/4wIwLl\\\\\\\/TFjn6AoH4GIKNx8PmS0VCyFrpnn\\\\\\\/oW3Qsu89EDOSO+sFdIzXHYEEJmXmU8vqy\\\\\\\/mkWYse9VUtfS1NPQ0zwnDITi6ISeBDKWxkK67OBk2eL\\\\\\\/Ayw4ts+ybbz2XQEduu+VhsXxSt04U8rlEeRgkF4sjPjHZ3tMc6AJBK0bzZEHitA0KA4amvSqaW6AcOuoxH+t0+0BYqUUW6ceXDFXEpVV7xM6h45Hf402XG58NG93AQaxIkTB2qFGXc5C5FbzDNwbfW\\\\\\\/HwMMCoJosP7y6RoYnKO2B0HCvL6oZ2ewy2sGDcfCeQELgPFdUVjm9KO45lX5sBepaG4Gip71foyETDI7K1tKNSyj+4sVVDfR2qRycfL906uUDoWUhaFLNPThsmzbhLITbcnCGYXN6yR9ADIJU+UzTaxvgV12zqy2tyhQPXz7obHI2qowXSLSUqWW9rWzqgijNINm5MSWRdkAQoEMb6tbssLCyoWvwxcvhrSadegAwOwLn4Ney3IH6alL6QWTscmMA5e47\\\\\\\/NnQx4CArwS5p7+Lm9X\\\\\\\/qpXW9kVdyI+uGMkMjCDG0YuxfGDuWph714XBFPL9H0fYligiPW4wNmaZRk3cuH4tJZN8lb\\\\\\\/DHBF9hT4\\\\\\\/Aho3fxs\\\\\\\/sWsKwtmIUuJ0PYRvddIDqk6ycgJlu0useMqhRyx1JxjKZj9f3Ol8hH5NItVmk35nA4+FMZbqw1v3cDxlZGtnge+qlsHGHcHq1umXKno\\\\\\\/jIyyVQmDdm+Pw3OJ0PRbb9vtF3Xbb80QC+8jQJzWi65wTRyDybwjvgygHAuHXVa3a8j2fWeHQdyfxca79yelZ096Emqw\\\\\\\/\\\\\\\/HMA\\\\\\\/nH+TuiRlRj12MAZmN2e7WegHoIq7yqJvn+nqEbYSIigwAsQfUzXxjZGKSUbVc+6gwbvLUaOCb4iIlZZmUspyaG+rqU46ZLKFkj+3kk1yWSmqVNHG+qxBICiK75oejZzzLXMQf628z+8c2T+UkTfs5YoHjIHyY0zlNl3GToO+PUgU54\\\\\\\/5tzQhhCU2tiO\\\\\\\/b7X\\\\\\\/oihdT\\\\\\\/tUcps4Xh0oI+kjfTYMDNMqRCyzJoV5dBqVdjM\\\\\\\/9ZJCM+1pkgUH1nEP5QFAgtwWa0Po6l8xKnfwZUSBauoUMIv1Vr\\\\\\\/bw5RL3yvZlCHUNq5OQjfbYw4b7DhoOarAGx+mlW8q1pWi0KWHZEL60WPTIT2aGtiY1jQJtZAg1wyehue+gHVTbP531544NJ6HlT\\\\\\\/tcqXp5IyfxVkvV\\\\\\\/4zweQLJe2VI6RA73sAX9\\\\\\\/3Ac\\\\\\\/9yDAoYSPoQT\\\\\\\/XKpoVIHERtOvDyVy08odLpczR+zoJIDGuEOS6jfmGMXSLHzGHuk1GXHR5xC9kZ6qwb93q7hPFwGuJS7wkeah3kiGCjgMXbBz\\\\\\\/lh2LcZDkbRhqbR6A+\\\\\\\/cf34c8VppumE63OaifAV3GdvDTe6gH9dXVz95MQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Bbohxkeqo0JX\\\\\\\/LiJONYJsA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228f78e6f71f20f737faf0cc14dc59d6df36115a261f257448c90cfad3c91a026a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LZeKtTV\\\\\\\/4og\\\\\\\/xA+crU9AtlNjkaktG2owatroZ7fFrlc\\\\\\\/zuPgda2xcPHMpyaCTQ07pi\\\\\\\/y552Tv0aVY43kloekW4oas4dHZkgt6ZkRWlgWjkVpGvh\\\\\\\/0B\\\\\\\/e9IJJVXD\\\\\\\/VbpHoy0gDA0MUGr\\\\\\\/fuHK9YrqE\\\\\\\/MF1ZU7TAc4J+MpShYaZl1ndh9RdXqtIzQhYBLWFLIl9hkNF5diFQHbKQHvgqZP0FtcYvJEKqPws1tA6TodYwz4BKBkTx6mq7bQfPGSLiTyqhS8TnKqGTrhu23fFGUYQoclcmFVkB6hTI9wj3cDwqQcsgr\\\\\\\/ZkZDArYxmrgyDneyrY8W66y0MLfkT47MwMgp93KaXE+cenjku+sy+8uEiKM86P\\\\\\\/Ibz8cRTxggxTXscjHCzJG00gNhXSGfwT5YgIg9u9KtKnYzA8dtjCv0okglGM\\\\\\\/WNfvW2W2roSDBM1A5GkzaZdX+8IDU0ceBKj0GgaTrNAVqKKWIHm21b3W7yBpXqla10VuT6+gC+nySKO8bhLNERnrC3rVnmWZfH0gJHKjIbn910lKezJjCRTKiHoeiCGKgmYBTfOTydTC6mU3Gw99eLvMSAWhxfkzyptpSt7SeEUHMlSiiIpD+sbobDdgr9E8lxVPXkLAnPtBmTUDHlxGZUEMJp94k\\\\\\\/+61Dm0G1nCO14rpX4G\\\\\\\/GOwQXxMGnH18sUJa7xSIvqvdr9WpRkMpfzemOeTYwgB5R\\\\\\\/dNGN6zIdSDZOUi7CmEtclW4UrrIuaN8sydw4gJCVALRXhhYxk5gZ0FXkCvcjhCUEId1iB6lgH5y8oLWFFxzb04+AB8K\\\\\\\/UrJUGV2pB0YsET2eG7uw6tjFebl170vclJfp3+OoWsghi4YP68tYsIeLfRari9TMvNb90Lm9KKCKI\\\\\\\/q3Kv2WHoWrp5ROvySL46ax35jKRz6oeJKWNZaRtqaAw3YgRfuEFrMaOT7pAqxa7OjuXPA3z\\\\\\\/V6XgASoqy6Qyx2h0Vq2acrf0df5l4W9SAjAMdnzjFPXKCG7Kq0hFy\\\\\\\/EJe5BDt25CfXl2kTH5sK0MM\\\\\\\/z0eBU35M81+YrbHdMDEWR5dKJ1MXTGcuBBFCsCLSJK9\\\\\\\/AF8VpRRuhbWgYNZBtxCAn6mvq77BaxtP9OckysVnmztCHkGDOjZcUM06pGyDestBdiI8bjAgmidXa6MZbIQljqt4L3gVN\\\\\\\/PjwMXIjSU+IR9KOrCaxMN42EbIjk4gghcdc7FNcxv3IdxlXdVWnQvcltqMxLhQRITBW3wz0fBIbLrwZuUHhkXClDtBTAy1p0+\\\\\\\/Lqo0wwRafI125Gmo\\\\\\\/8PHkMUotYLGu5btOl6HtrlVNIuoHR5GX4jlLpUm8FYfBNcztpdYGy4uu8XxaQlhdVW2Lpq\\\\\\\/N3gRJCFyHhNlNghemgytLUMV2CRlvA2ahiMhnFnEbbGrjdzz2GwDjcd9Q34xT5T9YE9stFahy+uLKWcQXnRc7WP4baDD0keQIKyQysmYTWu8c1onILzdCYgTzjgnopmVR2O+EPjUD+4wPe6fD8TVNLbd1W9CrZewhhTLtYAZVo\\\\\\\/cxlp65yt2TC\\\\\\\/FZr5rOccMXHNMA45BOtxIWJmCL2EPK5Q5Qxa2iWnjcho8a4oUtspSAQHfR4VBehKPANp6jMidWpJ99h\\\\\\\/E8VO\\\\\\\/YwV\\\\\\\/C+m2yT8cXnCqvyWfhsxGHuizFaa8WpMES+AQFKwVaZoyHXM9mYZmz6utC22p0dp5Khl2438Z2tbLb4aYRH6WDP3fStQT\\\\\\\/870Lv+l90Nc+hDF+xJ5fS8ht1+VQbY\\\\\\\/fdR8xPLFvw7Ny\\\\\\\/DOlVqv5xER87+BlL+B2jslkncpYFrLFT8Lm8s5YjXzEeyhDzybVH\\\\\\\/tkiiwVrdfnPPsQJVZfJVa2ADdV5N9qSZHcdt10QbEqZMdGhavYafIvh7GCuimG2qAMZ\\\\\\\/w2rd1VwAnLAnXyT76cFoaXCbacmAfhW7FJAx4mIdvOmzvs71XBggKfa80Qka0a7Vpq0qkx\\\\\\\/wDJcH4epgfN7h0kG0dvbge1ycxFPdUWbvcb1KrfY0GiwQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00223\\\\\\\/vQhGls7V\\\\\\\/Sevgk8R0vrA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022feb032eeb9959f6010e779f45fc13d8ce967ecf5f165c4de33945f1772ed056e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493460 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2317" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-N-dWmUmg6ZWXZGv9i2BHg5suWfRF28XFX187HwWghCA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2317" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Saase0Yo-MHkXc-0OrFz8524k7IZkkAdsOh2c38LHPM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2303" title="Contraction of the ROS Scavenging Enzyme Glutathione S-Transferase Gene Family in Cetaceans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2325" title="Genetic Diversity and Phylogenetic Relationships of Annual and Perennial Glycine Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1644-5931" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frederick J. Boehm</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5642-5062" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elissa J. Chesler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian S. Yandell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2317-2324;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400098 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493460" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick J. Boehm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BJ.%2BBoehm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boehm%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BJ.%2BBoehm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1644-5931" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frederick J. Boehm</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5642-5062" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elissa J. Chesler</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brian S. Yandell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Horticulture, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BS.%2BYandell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yandell%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BS.%2BYandell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian S. Yandell</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:broman@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">broman@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2317" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493460" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2317.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493460" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2317.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493460" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2317.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493460" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The high mapping resolution of multiparental populations, combined with technology to measure tens of thousands of phenotypes, presents a need for quantitative methods to enhance understanding of the genetic architecture of complex traits. When multiple traits map to a common genomic region, knowledge of the number of distinct loci provides important insight into the underlying mechanism and can assist planning for subsequent experiments. We extend the method of <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a>, for testing pleiotropy with a pair of traits, to the case of more than two alleles. We also incorporate polygenic random effects to account for population structure. We use a parametric bootstrap to determine statistical significance. We apply our methods to a behavioral genetics data set from Diversity Outbred mice. Our methods have been incorporated into the R package qtl2pleio.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quantitative-trait-locus" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus" rel="nofollow">Quantitative trait locus</a></li><li class="kwd"><a href="/keyword/pleiotropy" class="hw-term hw-article-keyword hw-article-keyword-pleiotropy" rel="nofollow">pleiotropy</a></li><li class="kwd"><a href="/keyword/multivariate-analysis" class="hw-term hw-article-keyword hw-article-keyword-multivariate-analysis" rel="nofollow">multivariate analysis</a></li><li class="kwd"><a href="/keyword/linear-mixed-effects-models" class="hw-term hw-article-keyword hw-article-keyword-linear-mixed-effects-models" rel="nofollow">linear mixed effects models</a></li><li class="kwd"><a href="/keyword/systems-genetics" class="hw-term hw-article-keyword hw-article-keyword-systems-genetics" rel="nofollow">systems genetics</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross-magic" rel="nofollow">Multiparent Advanced Generation Inter-Cross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Complex trait studies in multiparental populations present new challenges in statistical methods and data analysis. Among these is the development of strategies for multivariate trait analysis. The joint analysis of two or more traits allows one to address additional questions, such as whether two traits share a single pleiotropic locus.</p><p id="p-4">Previous research addressed the question of pleiotropy <em>vs.</em> separate QTL in two-parent crosses. <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a> developed a likelihood ratio test for pleiotropy <em>vs.</em> separate QTL for a pair of traits. Their approach assumed that each trait was affected by a single QTL. Under the null hypothesis, the two traits were affected by a common QTL, and under the alternative hypothesis the two traits were affected by distinct QTL. <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Knott and Haley (2000)</a> used linear regression to develop a fast approximation to the test of <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a>, while <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tian <em>et al.</em> (2016)</a> used the methods from <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Knott and Haley (2000)</a> to dissect QTL hotspots in a F<sub>2</sub> population.</p><p id="p-5">Multiparental populations, such as the Diversity Outbred (DO) mouse population (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Churchill <em>et al.</em> 2012</a>), enable high-precision mapping of complex traits (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de Koning and McIntyre 2014</a>). The DO mouse population began with progenitors of the Collaborative Cross (CC) mice (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Churchill <em>et al.</em> 2004</a>). Each DO mouse is a highly heterozygous genetic mosaic of alleles from the eight CC founder lines. Random matings among non-siblings have maintained the DO population for more than 23 generations (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chesler <em>et al.</em> 2016</a>).</p><p id="p-6">Several limitations of previous pleiotropy <em>vs.</em> separate QTL tests prevent their direct application in multiparental populations. First, multiparental populations can have complex patterns of relatedness among subjects, and failure to account for these patterns of relatedness may lead to spurious results (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2014</a>). Second, previous tests allowed for only two founder lines (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Jiang and Zeng 1995</a>). Finally, <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a> assumed that the null distribution of the test statistic follows a chi-square distribution.</p><p id="p-7">We developed a pleiotropy <em>vs.</em> separate QTL test for two traits in multiparental populations. Our test builds on research that <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a>, <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Knott and Haley (2000)</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Tian <em>et al.</em> (2016)</a>, and <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhou and Stephens (2014)</a> initiated. Our innovations include the accommodation of <em>k</em> founder alleles per locus (compared to the traditional two founder alleles per locus) and the incorporation of multivariate polygenic random effects to account for relatedness. Furthermore, we implemented a parametric bootstrap test to assess statistical significance (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Efron 1979</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Tian <em>et al.</em> 2016</a>). We focus on the case that two traits are measured in the same set of subjects (Design I in the notation of <a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a>).</p><p id="p-8">Below, we describe our likelihood ratio test for pleiotropy <em>vs.</em> separate QTL. In simulation studies, we find that it is slightly conservative, and that it has power to detect two separate loci when the univariate LOD peaks are strong. We further illustrate our approach with an application to data on a pair of behavior traits in a population of 261 DO mice (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Logan <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Recla <em>et al.</em> 2014</a>).</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-9">Our strategy involves first identifying two traits that map to a common genomic region. We then perform a two-dimensional, two-QTL scan over the genomic region, with each trait affected by one QTL of varying position. We identify the QTL position that maximizes the likelihood under pleiotropy (that is, along the diagonal where the two QTL are at a common location), and the ordered pair of positions that maximizes the likelihood under the model where the two QTL are allowed to be distinct. The logarithm of the ratio of the two likelihoods is our test statistic. We determine statistical significance with a parametric bootstrap.</p><div id="sec-2" class="subsection"><h3>Data structures</h3><p id="p-10">The data consist of three objects. The first is an <em>n</em> by <em>k</em> by <em>m</em> array of allele probabilities for <em>n</em> subjects with <em>k</em> alleles and <em>m</em> marker positions on a single chromosome [derived from the observed SNP genotype data by a hidden Markov model; see <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> (2019)</a>]. The second object is an <em>n</em> by 2 matrix of phenotype values. Each column is a phenotype and each row is a subject. The third object is an <em>n</em> by <em>c</em> matrix of covariates, where each row is a subject and each column is a covariate.</p><p id="p-11">One additional object is the genotype-derived kinship matrix, which is used in the linear mixed model to account for population structure. We are focusing on a defined genomic interval, and we prefer to use a kinship matrix derived by the “leave one chromosome out” (LOCO) method (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2014</a>), in which the kinship matrix is derived from the genotypes for all chromosomes except the chromosome under test.</p></div><div id="sec-3" class="subsection"><h3>Statistical Models</h3><p id="p-12">Focusing on a pair of traits and a particular genomic region of interest, the next step is a two-dimensional, two-QTL scan (<a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Jiang and Zeng 1995</a>). We consider two QTL with each affecting a different trait, and consider all possible pairs of locations for the two QTL. For each pair of positions, we fit the multivariate linear mixed effects model defined in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>. Note that we have assumed an additive genetic model throughout our analyses, but extensions to design matrices that include dominance are straightforward.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>Y</em> is the <em>n</em> by 2 matrix of phenotypes values; <em>X</em> is a <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-2.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-3.gif"/></noscript></span></span> matrix that contains the <em>k</em> allele probabilities for the two QTL positions and the <em>c</em> covariates in diagonal blocks; <em>B</em> is a <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-4.gif"/></noscript></span></span> by 2 matrix of allele effects and covariate effects; <em>G</em> is a <em>n</em> by 2 matrix of random effects; and <em>E</em> is a <em>n</em> by 2 matrix of random errors. <em>n</em> is the number of mice. The ‘vec’ operator stacks columns from a matrix into a single vector. For example, a 2 by 2 matrix inputted to ‘vec’ results in a vector with length 4. Its first two entries are the matrix’s first column, while the third and fourth entries are the matrix’s second column.</p><p id="p-13">We also impose distributional assumptions on <em>G</em> and <em>E</em>:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>and<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-7.gif"/></noscript></span></span> denotes the matrix-variate (<em>n</em> by 2) normal distribution with mean being the <em>n</em> by 2 matrix with all zero entries and row covariance <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-8.gif"/></noscript></span></span> and column covariance <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-9.gif"/></noscript></span></span>. We assume that <em>G</em> and <em>E</em> are independent.</p></div><div id="sec-4" class="subsection"><h3>Parameter inference and log likelihood calculation</h3><p id="p-14">Inference for parameters in multivariate linear mixed effects models is notoriously difficult and can be computationally intense (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meyer 1989</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">1991</a>). Thus, we estimate <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-11.gif"/></noscript></span></span> under the null hypothesis of no QTL, and then take them as fixed and known in our two-dimensional, two-QTL genome scan. We use restricted maximum likelihood methods to fit the model:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-13.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-14.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-15.gif"/></noscript></span></span> matrix whose first column of each diagonal block in <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-16.gif"/></noscript></span></span> has all entries equal to one (for an intercept); the remaining columns are the covariates.</p><p id="p-15">We draw on our R implementation (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boehm 2018</a>) of the GEMMA algorithm for fitting a multivariate linear mixed effects model with expectation-maximization (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Zhou and Stephens 2014</a>). We use restricted maximum likelihood fits for the variance components <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-18.gif"/></noscript></span></span> in subsequent calculations of the generalized least squares solution <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-19.gif"/></noscript></span></span>.<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where ⊗ denotes the Kronecker product, <em>K</em> is the kinship matrix, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-22.gif"/></noscript></span></span> is a n by n identity matrix. We then calculate the log likelihood for a normal distribution with mean <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-23.gif"/></noscript></span></span> and covariance <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-24.gif"/></noscript></span></span> that depends on our estimates of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-26.gif"/></noscript></span></span> (<a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation 6</a>).</p></div><div id="sec-5" class="subsection"><h3>Pleiotropy <em>vs.</em> separate QTL hypothesis testing framework</h3><p id="p-16">Our test applies to two traits considered simultaneously. Below, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-28.gif"/></noscript></span></span> denote putative locus positions for traits one and two. We quantitatively state the competing hypotheses for our test as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-29.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-30.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Our likelihood ratio test statistic is:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <em>L</em> is the likelihood for fixed QTL positions, maximized over all other parameters. The denominator concerns the likelihood for the null hypothesis of pleiotropy, where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-32.gif"/></noscript></span></span>.</p></div><div id="sec-6" class="subsection"><h3>Visualizing profile LOD traces</h3><p id="p-17">The output of the above analysis is a two-dimensional log10 likelihood surface. To visualize these results, we followed an innovation of <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zeng <em>et al.</em> (2000)</a> and <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Tian <em>et al.</em> (2016)</a>, and plot three traces: the results along the diagonal (corresponding to the null hypothesis of pleiotropy), and then the profiles derived by fixing one QTL’s position and maximizing over the other QTL’s position.</p><p id="p-18">We define the LOD score for our test:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-33.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-34.gif"/></noscript></span></span> denotes log10 likelihood.</p><p id="p-19">We follow <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Zeng <em>et al.</em> (2000)</a> and <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Tian <em>et al.</em> (2016)</a> in defining profile LOD by the equation<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>We define profile LOD<sub>2</sub><span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-36.gif"/></noscript></span></span> analogously. The profile LOD<sub>1</sub> and profile LOD<sub>2</sub> traces have the same maximum value, which is non-negative and gives the overall LOD test statistic.</p><p id="p-20">We construct the pleiotropy trace by calculating the log-likelihoods for the pleiotropic models at every position.<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-37.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>By definition, the maximum value for this pleiotropy trace is zero.</p></div><div id="sec-7" class="subsection"><h3>Bootstrap for test statistic calibration</h3><p id="p-21">We use a parametric bootstrap to determine statistical significance (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Efron 1979</a>). While <a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a> used quantiles of a chi-squared distribution to determine p-values, this does not account for the two-dimensional search over QTL positions. We follow the approach of <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Tian <em>et al.</em> (2016)</a>, and identify the maximum likelihood estimate of the QTL position under the null hypothesis of pleiotropy. We then use the inferred model parameters under that model and with the QTL at that position to simulate bootstrap data sets according to the model in <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">equations 1</a>–<a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">3</a>. For each of <em>b</em> bootstrap data sets, we perform a two-dimensional QTL scan (over the genomic region of interest) and derive the test statistic value. We treat these <em>b</em> test statistics as the empirical null distribution, and calculate a p-value as the proportion of the <em>b</em> bootstrap test statistics that equal or exceed the observed one, with the original data, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-38.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-39.gif"/></noscript></span></span> denotes the LOD score for the <em>i</em>th bootstrap replicate and LOD is the observed test statistic.</p></div><div id="sec-8" class="subsection"><h3>Data &amp; Software Availability</h3><p id="p-22">Our methods have been implemented in an R package, qtl2pleio, available at GitHub: <a href="https://github.com/fboehm/qtl2pleio">https://github.com/fboehm/qtl2pleio</a></p><p id="p-23">Custom R code for our analyses and simulations are at GitHub: <a href="https://github.com/fboehm/qtl2pleio-manuscript-clean">https://github.com/fboehm/qtl2pleio-manuscript-clean</a></p><p id="p-24">The data from <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Recla <em>et al.</em> (2014)</a> and <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Logan <em>et al.</em> (2013)</a> are available at the Mouse Phenome Database: <a href="https://phenome.jax.org/projects/Chesler4">https://phenome.jax.org/projects/Chesler4</a> and <a href="https://phenome.jax.org/projects/Recla1">https://phenome.jax.org/projects/Recla1</a>. They are also available in R/qtl2 format at <a href="https://github.com/rqtl/qtl2data">https://github.com/rqtl/qtl2data</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8126930">https://doi.org/10.25387/g3.8126930</a>.</p></div></div><div class="section other1" id="sec-9"><h2 class="">Simulation studies</h2><p id="p-25">We performed two types of simulation studies, one for type I error rate assessment and one to characterize the power to detect separate QTL. To simulate traits, we specified <em>X</em>, <em>B</em>, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-40.gif"/></noscript></span></span>, <em>K</em>, and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-41.gif"/></noscript></span></span> matrices (<a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equations 1</a>–<a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">3</a>). For both we used the allele probabilities from a single genomic region derived empirically from data for a set of 479 Diversity Outbred mice from <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Keller <em>et al.</em> (2018)</a>.</p><div id="sec-10" class="subsection"><h3>Type I error rate analysis</h3><p id="p-26">To quantify type I error rate (<em>i.e.</em>, false positive rate), we simulated 400 pairs of traits for each of eight sets of parameter inputs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We used a <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-42.gif"/></noscript></span></span> factorial experimental design with three factors: allele effects difference, allele effects partitioning, and genetic correlation, <em>i.e.</em>, the off-diagonal entry in the 2 by 2 matrix <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-43.gif"/></noscript></span></span>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493632/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493632/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Type I error rates for all runs in our 2<sup>3</sup> experimental design. We set (marginal) genetic variances (<em>i.e.</em>, diagonal elements of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/T1/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/T1/embed/mml-math-44.gif"/></noscript></span></span>) to 1 in all runs. <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/T1/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/T1/embed/mml-math-45.gif"/></noscript></span></span> was set to the 2 by 2 identity matrix in all runs. We used allele probabilities at a single genetic marker to simulate traits for all eight sets of parameter inputs. In the column “Allele effects partitioning”, “ABCD:EFGH” means that lines A–D carry one QTL allele while lines E–H carry the other allele. “F:ABCDEGH” means the QTL has a private allele in strain F</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">We chose two strong allele effects difference values, 6 and 12. These ensured that the univariate phenotypes mapped with high LOD scores to the region of interest. For the allele partitioning factor, we used either equally frequent QTL alleles, or a private allele in the CAST strain (F). For the residual genetic correlation (the off-diagonal entry in <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-46.gif"/></noscript></span></span>), we considered the values 0 and 0.6. The marginal genetic variances (<em>i.e.</em>, the diagonal entries in <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-47.gif"/></noscript></span></span>) for each trait were always set to one.</p><p id="p-28">We performed 400 simulation replicates per set of parameter inputs, and each used <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-48.gif"/></noscript></span></span> bootstrap samples. For each bootstrap sample, we calculated the test statistic (<a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation 8</a>). We then compared the test statistic from the simulated trait against the empirical distribution of its 400 bootstrap test statistics. When the simulated trait’s test statistic exceeded the 0.95 quantile of the empirical distribution of bootstrap test statistics, we rejected the null hypothesis. We observed that the test is slightly conservative over our range of parameter selections (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), with estimated type I error rates &lt; 0.05.</p></div><div id="sec-11" class="subsection"><h3>Power analysis</h3><p id="p-29">We also investigated the power to detect the presence of two distinct QTL. We used a 2 × 2 × 5 experimental design, where our three factors were allele effects difference, allele effects partitioning, and inter-locus distance. The two levels of allele effects difference were 1 and 2. The two levels of allele effects partitioning were as in the type I error rate studies, ABCD:EFGH and F:ABCDEGH (Table S1). The five levels of interlocus distance were 0, 0.5, 1, 2, and 3 cM. <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-49.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-50.gif"/></noscript></span></span> were both set to the 2 by 2 identity matrix in all power study simulations.</p><p id="p-30">We simulated 400 pairs of traits per set of parameter inputs. For each simulation replicate, we calculated the likelihood ratio test statistic. We then applied our parametric bootstrap to determine the statistical significance of the results. For each simulation replicate, we used <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-51.gif"/></noscript></span></span> bootstrap samples. Because the bootstrap test statistics within a single set of parameter inputs followed approximately the same distribution, we pooled the <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-52.gif"/></noscript></span></span> bootstrap samples per set of parameter inputs and compared each test statistic to the empirical distribution derived from the 160,000 bootstrap samples. However, for parameter inputs with interlocus distance equal to zero, we did not pool the 160,000 bootstrap samples; instead, we proceeded by calculating power (<em>i.e.</em>, type I error rate, in this case), as we did in the type I error rate study above.</p><p id="p-31">We present our power study results in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Power increases as interlocus distance increases. The top two curves correspond to the case where the QTL effects are largest. For each value for the QTL effect, power is greater when the QTL alleles are equally frequent, and smaller when a QTL allele is private to one strain. One can have high power to detect that the two traits have distinct QTL when they are separated by &gt; 1 cM and when the QTL have large effect. We provide example profile LOD plots from the power analysis in Figure S3.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pleiotropy vs. separate QTL power curves for each of four sets of parameter settings. Factors that differ among the four curves are allele effects difference and allele partitioning. Red denotes high allele effects difference, while black is the low allele effects difference. Solid line denotes the even allele partitioning (ABCD:EFGH), while dashed line denotes the uneven allele partitioning (F:ABCDEGH)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1704599465" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pleiotropy &lt;em&gt;vs.&lt;/em&gt; separate QTL power curves for each of four sets of parameter settings. Factors that differ among the four curves are allele effects difference and allele partitioning. Red denotes high allele effects difference, while black is the low allele effects difference. Solid line denotes the even allele partitioning (ABCD:EFGH), while dashed line denotes the uneven allele partitioning (F:ABCDEGH).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2317/F1.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2317/F1.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493515" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">Pleiotropy <em>vs.</em> separate QTL power curves for each of four sets of parameter settings. Factors that differ among the four curves are allele effects difference and allele partitioning. Red denotes high allele effects difference, while black is the low allele effects difference. Solid line denotes the even allele partitioning (ABCD:EFGH), while dashed line denotes the uneven allele partitioning (F:ABCDEGH).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section other2" id="sec-12"><h2 class="">Application</h2><p id="p-33">To illustrate our methods, we applied our test to data from <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Logan <em>et al.</em> (2013)</a> and <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Recla <em>et al.</em> (2014)</a>, on 261 DO mice measured for a set of behavioral phenotypes. <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Recla <em>et al.</em> (2014)</a> identified <em>Hydin</em> as the gene that underlies a QTL on Chromosome 8 at 57 cM for the “hot plate latency” phenotype (a measure of pain tolerance). The phenotype “percent time in light” in a light-dark box (a measure of anxiety) was measured on the same set of mice (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Logan <em>et al.</em> 2013</a>) and also shows a QTL near this location, which led us to ask whether the same locus affects both traits. The two traits show a correlation of <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-53.gif"/></noscript></span></span> (Figure S1).</p><p id="p-34">QTL analysis with the LOCO method, and using sex as an additive covariate, showed multiple suggestive QTL for each phenotype (Figure S2; Table S2). For our investigation of pleiotropy, we focused on the interval 53–64 cM on Chromosome 8. Univariate analyses showed a QTL in this region for both traits (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome 8 univariate LOD scores for percent time in light and hot plate latency reveal broad, overlapping peaks between 53 cM and 64 cM. The peak for percent time in light spans the region from approximately 53 cM to 60 cM, with a maximum near 55 cM. The peak for hot plate latency begins near 56 cM and ends about 64 cM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1704599465" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromosome 8 univariate LOD scores for percent time in light and hot plate latency reveal broad, overlapping peaks between 53 cM and 64 cM. The peak for percent time in light spans the region from approximately 53 cM to 60 cM, with a maximum near 55 cM. The peak for hot plate latency begins near 56 cM and ends about 64 cM.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2317/F2.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2317/F2.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493586" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Chromosome 8 univariate LOD scores for percent time in light and hot plate latency reveal broad, overlapping peaks between 53 cM and 64 cM. The peak for percent time in light spans the region from approximately 53 cM to 60 cM, with a maximum near 55 cM. The peak for hot plate latency begins near 56 cM and ends about 64 cM.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The estimated QTL allele effects for the two traits are quite different (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). With the QTL placed at 55 cM, for “percent time in light”, the WSB and PWK alleles are associated with large phenotypes and NOD with low phenotypes. For “hot plate latency”, on the other hand, CAST and NZO show low phenotypes and NOD and PWK are near the center.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome 8 founder allele effects for percent time in light and hot plate latency demonstrate distinct allele patterns between 53cM and 64 cM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1704599465" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromosome 8 founder allele effects for percent time in light and hot plate latency demonstrate distinct allele patterns between 53cM and 64 cM.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2317/F3.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2317/F3.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493583" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">Chromosome 8 founder allele effects for percent time in light and hot plate latency demonstrate distinct allele patterns between 53cM and 64 cM.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">In applying our test for pleiotropy, we performed a two-dimensional, two-QTL scan for the pair of phenotypes. With these results, we created a profile LOD plot (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The profile LOD for “percent time in light” (in brown) peaks near 55 cM, as was seen in the univariate analysis. The profile LOD for “hot plate latency” (in blue) peaks near 58 cM, also similar to the univariate analysis. The pleiotropy trace (in gray) peaks near 58 cM.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Profile LOD curves for the pleiotropy vs. separate QTL hypothesis test for &#x201C;percent time in light&#x201D; and &#x201C;hot plate latency&#x201D;. Gray trace denotes pleiotropy LOD values. Likelihood ratio test statistic value corresponds to the height of the blue and gold traces at their maxima." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1704599465" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Profile LOD curves for the pleiotropy &lt;em&gt;vs.&lt;/em&gt; separate QTL hypothesis test for &#x201C;percent time in light&#x201D; and &#x201C;hot plate latency&#x201D;. Gray trace denotes pleiotropy LOD values. Likelihood ratio test statistic value corresponds to the height of the blue and gold traces at their maxima.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2317/F4.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2317/F4.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2317/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493529" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Profile LOD curves for the pleiotropy <em>vs.</em> separate QTL hypothesis test for “percent time in light” and “hot plate latency”. Gray trace denotes pleiotropy LOD values. Likelihood ratio test statistic value corresponds to the height of the blue and gold traces at their maxima.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The likelihood ratio test statistic for the test of pleiotropy was 1.2. Based on a parametric bootstrap with 1,000 bootstrap replicates, the estimated p-value was 0.11. Thus, by our approach, the evidence for the two traits having distinct QTL is weak.</p></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-41">We developed a test of pleiotropy <em>vs.</em> separate QTL for multiparental populations, extending the work of <a id="xref-ref-14-10" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a> for multiple alleles and with a linear mixed model to account for population structure (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kang <em>et al.</em> 2010</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2014</a>). Our simulation studies indicate that our test is slightly conservative, with type I error rates below their nominal values (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Power is affected by many factors (including sample size, effect size, and allele frequencies). We studied the effects of interlocus distance and QTL effect on power, and we showed that our test has power to detect presence of separate loci, especially when univariate trait associations are strong (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-42">In the application of our method to two behavioral phenotypes in a study of 261 Diversity Outbred mice (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Recla <em>et al.</em> 2014</a>; <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Logan <em>et al.</em> 2013</a>), the evidence for the presence of two distinct QTL, with one QTL (which contains the <em>Hydin</em> gene) affecting only “hot plate latency” and a second QTL affecting “percent time in light” was weak (<em>P</em> = 0.11, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-43">Founder allele effects plots provide further evidence for the presence of two distinct loci. As <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Macdonald and Long (2007)</a> and <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">King <em>et al.</em> (2012)</a> have demonstrated in their analyses of multiparental <em>Drosophila</em> populations, a biallelic pleiotropic QTL would result in allele effects plots that have similar patterns. While we do not know that “percent time in light” and “hot plate latency” arise from biallelic QTL, the dramatic differences that we observe in allele effects patterns further support the argument for two distinct loci.</p><p id="p-44">We have implemented our methods in an R package qtl2pleio, but analyses can be computationally intensive and time consuming. qtl2pleio is written mostly in R, and so we could likely obtain improved computational speed by porting parts of the calculations to a compiled language such as C or C++. To accelerate our multi-dimensional QTL scans, we have integrated C++ code into qtl2pleio, using the Rcpp package (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eddelbuettel <em>et al.</em> 2011</a>).</p><p id="p-45">Another computational bottleneck is the estimation of the variance components <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-54.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2317/embed/mml-math-55.gif"/></noscript></span></span>. To accelerate this procedure, especially for the joint analysis of more than two traits, we will consider other strategies for variance component estimation, including that described by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meyer <em>et al.</em> (2018)</a>. <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Meyer <em>et al.</em> (2018)</a>, in joint analysis of dozens of traits, implement a bootstrap strategy to estimate variance components for lower-dimensional phenotypes before combining bootstrap estimates into valid covariance matrices for the full multivariate phenotype. Such an approach may ease some of the computational burdens that we encountered.</p><p id="p-46">We view tests of pleiotropy as complementary to mediation tests and related methods that have become popular for inferring biomolecular causal relationships (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chick <em>et al.</em> 2016</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schadt <em>et al.</em> 2005</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baron and Kenny 1986</a>). A mediation test proceeds by including a putative mediator as a covariate in the regression analysis of phenotype and QTL genotype; a substantial reduction in the association between genotype and phenotype corresponds to evidence of mediation.</p><p id="p-47">Mediation analyses and our pleiotropy test ask distinct, but related, questions. Mediation analysis seeks to establish causal relationships among traits, including molecular traits, or dependent biological and behavioral processes. Pleiotropy tests examine whether two traits share a single source of genetic variation, which may act in parallel or in a causal network. Pleiotropy is required for causal relations among traits. In many cases, the pleiotropy hypothesis is the only reasonable one.</p><p id="p-48"><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Schadt <em>et al.</em> (2005)</a> argued that both pleiotropy tests and causal inference methods may contribute to gene network reconstruction. They developed a model selection strategy, based on the Akaike Information Criterion (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akaike 1974</a>), to determine which causal model is most compatible with the observed data. <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Schadt <em>et al.</em> (2005)</a> extended the methods of <a id="xref-ref-14-11" class="xref-bibr" href="#ref-14">Jiang and Zeng (1995)</a> to consider more complicated alternative hypotheses, such as the possibility of two QTL, one of which associates with both traits, and one of which associates with only one trait. As envisioned by <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Schadt <em>et al.</em> (2005)</a>, we foresee complementary roles emerging for our pleiotropy test and mediation tests in the dissection of complex trait genetic architecture.</p><p id="p-49">Two related approaches for identifying and exploiting pleiotropy deserve mention. First, CAPE (Combined Analysis of Pleiotropy and Epistasis) is a strategy for identifying higher-order relationships among traits and marker genotypes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tyler <em>et al.</em> 2013</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2016</a>) and has recently been extended for use with multiparental populations, including DO mice (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tyler <em>et al.</em> 2017</a>). CAPE exploits the pleiotropic relationship among traits in order to characterize the underlying network of QTL, and it can suggest possible pleiotropic effects, but it does not provide an explicit test of pleiotropy. Second, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Schaid <em>et al.</em> (2016)</a> described a test for pleiotropy in the context of human genome-wide association studies (GWAS). Their approach is fundamentally different from ours, in that rather than ask whether traits are affected by a common locus or distinct loci, they ask whether the traits are all affected by a particular SNP or only some are. The difference in these approaches may be attributed to the difference in mapping resolution between human GWAS and experimental populations.</p><p id="p-50">Technological advances in mass spectrometry and RNA sequencing have enabled the acquisition of high-dimensional biomolecular phenotypes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ozsolak and Milos 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Han <em>et al.</em> 2012</a>). Multiparental populations in <em>Arabidopsis</em>, maize, wheat, oil palm, rice, <em>Drosophila</em>, yeast, and other organisms enable high-precision QTL mapping (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Tisné <em>et al.</em> 2017</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Stanley <em>et al.</em> 2017</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Raghavan <em>et al.</em> 2017</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cubillos <em>et al.</em> 2013</a>). The need to analyze high-dimensional phenotypes in multiparental populations compels the scientific community to develop tools to study genotype-phenotype relationships and complex trait architecture. Our test, and its future extensions, will contribute to these ongoing efforts.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">The authors thank Lindsay Traeger, Julia Kemis, Qiongshi Lu, Rene Welch, and two anonymous referees for valuable suggestions to improve the manuscript. This work was supported in part by National Institutes of Health grants R01GM070683 (to K.W.B.) and P50DA039841 (to E.J.C.). The research made use of compute resources and assistance of the UW-Madison Center For High Throughput Computing (CHTC) in the Department of Computer Sciences at UW-Madison, which is supported by the Advanced Computing Initiative, the Wisconsin Alumni Research Foundation, the Wisconsin Institutes for Discovery, and the National Science Foundation, and is an active member of the Open Science Grid, which is supported by the National Science Foundation and the U.S. Department of Energy’s Office of Science.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8126930">https://doi.org/10.25387/g3.8126930</a>.</p></li><li class="fn" id="fn-2"><p id="p-53"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-14"><span class="received-label">Received </span>February 14, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-14"><span class="accepted-label">Accepted </span>May 14, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Boehm <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.1" data-doi="10.1109/TAC.1974.1100705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">A new look at the statistical model identification.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Automat. Contr.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/TAC.1974.1100705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BAutomat.%2BContr.%26rft.volume%253D19%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTAC.1974.1100705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TAC.1974.1100705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.2" data-doi="10.1037/0022-3514.51.6.1173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baron</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">The moderator–mediator variable distinction in social psychological research: Conceptual, strategic, and statistical considerations.</span> <abbr class="cit-jnl-abbrev">J. Pers. Soc. Psychol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1182</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1037/0022-3514.51.6.1173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bpersonality%2Band%2Bsocial%2Bpsychology%26rft.stitle%253DJ%2BPers%2BSoc%2BPsychol%26rft.aulast%253DBaron%26rft.auinit1%253DR.%2BM.%26rft.volume%253D51%26rft.issue%253D6%26rft.spage%253D1173%26rft.epage%253D1182%26rft.atitle%253DThe%2Bmoderator-mediator%2Bvariable%2Bdistinction%2Bin%2Bsocial%2Bpsychological%2Bresearch%253A%2Bconceptual%252C%2Bstrategic%252C%2Band%2Bstatistical%2Bconsiderations.%26rft_id%253Dinfo%253Adoi%252F10.1037%252F0022-3514.51.6.1173%26rft_id%253Dinfo%253Apmid%252F3806354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1037/0022-3514.51.6.1173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3806354&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F285400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-other" id="cit-9.7.2317.3"><div class="cit-metadata unstructured">Boehm, F., 2018 gemma2: <em>GEMMA multivariate linear mixed model</em>. R package version 0.0.1.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.4" data-doi="10.1534/genetics.118.301595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furlotte</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">R/qtl2: Software for mapping quantitative trait loci with high-dimensional data and multi-parent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301595%26rft_id%253Dinfo%253Apmid%252F30591514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjExLzIvNDk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.5" data-doi="10.1534/g3.116.035527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strobel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trepanier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Diversity outbred mice at 21: Maintaining allelic variation in the face of selection.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3893</span>–<span class="cit-lpage">3902</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.035527</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.035527%26rft_id%253Dinfo%253Apmid%252F27694113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjYvMTIvMzg5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.6" data-doi="10.1038/nature18270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chick</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huttlin</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Defining the consequences of genetic variation on a proteome-wide scale.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">534</span>: <span class="cit-fpage">500</span>–<span class="cit-lpage">505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature18270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D500%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18270%26rft_id%253Dinfo%253Apmid%252F27309819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27309819&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.7" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Collaborative Cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1104-1133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.8" data-doi="10.1007/s00335-012-9414-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The diversity outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9414-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D713%26rft.epage%253D718%26rft.atitle%253DThe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9414-2%26rft_id%253Dinfo%253Apmid%252F22892839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9414-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892839&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.9" data-doi="10.1534/genetics.113.155515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scovacricchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155515</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155515%26rft_id%253Dinfo%253Apmid%252F24037264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjMxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.10" data-doi="10.1534/genetics.114.169151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics and G3: Community-driven science, community-driven journals.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.169151</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D2%26rft.atitle%253DGENETICS%2Band%2BG3%253A%2BCommunity-Driven%2BScience%252C%2BCommunity-Driven%2BJournals%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169151%26rft_id%253Dinfo%253Apmid%252F25236444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.11" data-doi="10.18637/jss.v040.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allaire</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rcpp: Seamless R and C++ integration.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v040.i08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DJara%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1%26rft.epage%253D32%26rft.atitle%253DUsing%2Bthe%2BR%2BPackage%2Bcrlmm%2Bfor%2BGenotyping%2Band%2BCopy%2BNumber%2BEstimation.%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v040.i01%26rft_id%253Dinfo%253Apmid%252F22523482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v040.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523482&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.12" data-doi="10.1214/aos/1176344552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Bootstrap methods: another look at the jackknife.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344552</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GL00300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.13" data-doi="10.1002/mas.20342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multi-dimensional mass spectrometry-based shotgun lipidomics and novel strategies for lipidomic analyses.</span> <abbr class="cit-jnl-abbrev">Mass Spectrom. Rev.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/mas.20342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMass%2BSpectrometry%2BReviews%26rft.stitle%253DMass%2BSpectrometry%2BReviews%26rft.aulast%253DHan%26rft.auinit1%253DX.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D134%26rft.epage%253D178%26rft.atitle%253DMulti-dimensional%2Bmass%2Bspectrometry-based%2Bshotgun%2Blipidomics%2Band%2Bnovel%2Bstrategies%2Bfor%2Blipidomic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fmas.20342%26rft_id%253Dinfo%253Apmid%252F21755525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/mas.20342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21755525&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298578100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Multiple trait analysis of genetic mapping for quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJiang%26rft.auinit1%253DC.%26rft.volume%253D140%26rft.issue%253D3%26rft.spage%253D1111%26rft.epage%253D1127%26rft.atitle%253DMultiple%2Btrait%2Banalysis%2Bof%2Bgenetic%2Bmapping%2Bfor%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F7672582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MC8zLzExMTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjMxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.15" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.16" data-doi="10.1534/genetics.118.300864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schueler</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabaglia</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapleton</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic drivers of pancreatic islet function.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300864</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300864%26rft_id%253Dinfo%253Apmid%252F29567659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA5LzEvMzM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.17" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic dissection of a model complex trait using the Drosophila Synthetic Population Resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1566</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.134031.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Multitrait least squares for quantitative trait loci detection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">899</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BA.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D899%26rft.epage%253D911%26rft.atitle%253DMultitrait%2BLeast%2BSquares%2Bfor%2BQuantitative%2BTrait%2BLoci%2BDetection%26rft_id%253Dinfo%253Apmid%252F11014835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzIvODk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.19" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000551</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.20" data-doi="10.1111/gbb.12029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Recla</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philip</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bubier</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-precision genetic mapping of behavioral traits in the diversity outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">437</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/gbb.12029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BBrain%2BBehav.%26rft.volume%253D12%26rft.spage%253D424%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgbb.12029%26rft_id%253Dinfo%253Apmid%252F23433259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gbb.12029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23433259&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319834800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.21" data-doi="10.1534/genetics.106.069641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Joint estimates of quantitative trait locus effect and frequency using synthetic recombinant populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.069641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.069641%26rft_id%253Dinfo%253Apmid%252F17435224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8yLzEyNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjMxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.22" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10811</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.23" data-doi="10.1101/255497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casale</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">LiMMBo: a simple, scalable approach for linear mixed models in high-dimensional genetic association studies.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/255497</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F255497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyNTU0OTd2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.24" data-doi="10.1051/gse:19890308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Restricted maximum likelihood to estimate variance components for animal models with several random effects using a derivative-free algorithm.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">340</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-21-3-317</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D21%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19890308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19890308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989CF64700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.25" data-doi="10.1186/1297-9686-23-1-67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Estimating variances and covariances for multivariate animal models by restricted maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-23-1-67</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D23%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-23-1-67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-23-1-67&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FE39200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.26" data-doi="10.1038/nrg2934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozsolak</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milos</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA sequencing: advances, challenges and opportunities.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2934</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOzsolak%26rft.auinit1%253DF.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D87%26rft.epage%253D98%26rft.atitle%253DRNA%2Bsequencing%253A%2Badvances%252C%2Bchallenges%2Band%2Bopportunities.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2934%26rft_id%253Dinfo%253Apmid%252F21191423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21191423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286319600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.27" data-doi="10.1534/g3.117.042101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauleon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacorte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jubay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaw</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Approaches in characterizing genetic structure and mapping in a rice multiparental population.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1721</span>–<span class="cit-lpage">1730</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.042101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.042101%26rft_id%253Dinfo%253Apmid%252F28592653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xNzIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.28" data-doi="10.1007/s00335-014-9508-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Recla</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bult</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Precise genetic mapping and integrative bioinformatics in Diversity Outbred mice reveals <em>Hydin</em> as a novel pain gene.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-014-9508-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-014-9508-0%26rft_id%253Dinfo%253Apmid%252F24700285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-014-9508-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24700285&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.29" data-doi="10.1038/ng1589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An integrative genomics approach to infer causal associations between gene expression and disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1589</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSchadt%26rft.auinit1%253DE.%2BE.%26rft.volume%253D37%26rft.issue%253D7%26rft.spage%253D710%26rft.epage%253D717%26rft.atitle%253DAn%2Bintegrative%2Bgenomics%2Bapproach%2Bto%2Binfer%2Bcausal%2Bassociations%2Bbetween%2Bgene%2Bexpression%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1589%26rft_id%253Dinfo%253Apmid%252F15965475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15965475&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230196400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.30" data-doi="10.1534/genetics.116.189308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaid</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larrabee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Statistical methods for testing genetic pleiotropy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.189308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.189308%26rft_id%253Dinfo%253Apmid%252F27527515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.31" data-doi="10.1534/genetics.116.197780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngoma</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic dissection of nutrition-induced plasticity in insulin/insulin-like growth factor signaling and median life span in a Drosophila multiparent population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">602</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197780</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197780%26rft_id%253Dinfo%253Apmid%252F28592498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.32" data-doi="10.1534/genetics.115.183624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kendziorski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The dissection of expression quantitative trait locus hotspots.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1563</span>–<span class="cit-lpage">1574</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.183624</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183624%26rft_id%253Dinfo%253Apmid%252F26837753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi80LzE1NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjMxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.33" data-doi="10.1534/g3.117.041764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tisné</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomiès</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riou</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syahputra</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Identification of ganoderma disease resistance loci using natural field infection of an oil palm multiparental population.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1683</span>–<span class="cit-lpage">1692</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.041764</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041764%26rft_id%253Dinfo%253Apmid%252F28592650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xNjgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.34" data-doi="10.1371/journal.pgen.1005805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donahue</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Weak epistasis generally stabilizes phenotypes in a mouse intercross.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1005805</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005805</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005805&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.35" data-doi="10.1534/genetics.116.198051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Epistatic networks jointly influence phenotypes related to metabolic disease and gene expression in diversity outbred mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198051%26rft_id%253Dinfo%253Apmid%252F28592500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNjIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.36" data-doi="10.1371/journal.pcbi.1003270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrick</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philip</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CAPE: an R package for combined analysis of pleiotropy and epistasis.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1003270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1003270%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003270%26rft_id%253Dinfo%253Apmid%252F24204223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204223&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.37" data-doi="10.1038/ng.2876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Advantages and pitfalls in the application of mixed-model association methods.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2876</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2876%26rft_id%253Dinfo%253Apmid%252F24473328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24473328&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.38" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic architecture of a morphological shape difference between two Drosophila species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.-B.%26rft.volume%253D154%26rft.issue%253D1%26rft.spage%253D299%26rft.epage%253D310%26rft.atitle%253DGenetic%2BArchitecture%2Bof%2Ba%2BMorphological%2BShape%2BDifference%2BBetween%2BTwo%2BDrosophila%2BSpecies%26rft_id%253Dinfo%253Apmid%252F10628989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzEvMjk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2317.40" data-doi="10.1038/nmeth.2848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient multivariate linear mixed model algorithms for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2848</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2848%26rft_id%253Dinfo%253Apmid%252F24531419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2848&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24531419&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2317.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2303" title="Contraction of the ROS Scavenging Enzyme Glutathione S-Transferase Gene Family in Cetaceans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2325" title="Genetic Diversity and Phylogenetic Relationships of Annual and Perennial Glycine Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2317" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oCD6ALHpEFOsuhIPyK2qruJF6hfROh5Rpd3-GXl1LrU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2317" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2317" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2317" class="active" data-icon-position="" data-hide-link-title="0">Testing Pleiotropy vs. Separate QTL in Multiparental Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493460" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UFA-0H6grACoCyRrOEjV3unGSgeYEFDlnT5v5QWCJW8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2317.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2317" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MlusCQ-fpg-ViP-sKlaFUCuQUTn-B1fFtvGtrCoontY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2317" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2317.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460--2" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1644-5931" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frederick J. Boehm</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5642-5062" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elissa J. Chesler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian S. Yandell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2317-2324;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400098 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493460--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick J. Boehm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BJ.%2BBoehm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boehm%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BJ.%2BBoehm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1644-5931" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frederick J. Boehm</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5642-5062" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elissa J. Chesler</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brian S. Yandell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Horticulture, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BS.%2BYandell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yandell%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BS.%2BYandell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian S. Yandell</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:broman@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">broman@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2317" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2317&amp;title=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2317&amp;title=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2317&amp;title=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2317&amp;text=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2317&amp;title=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2317&amp;t=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2317&amp;title=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2317&amp;title=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493460/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460--3" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1644-5931" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frederick J. Boehm</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5642-5062" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elissa J. Chesler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian S. Yandell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2317-2324;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400098 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493460--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick J. Boehm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BJ.%2BBoehm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boehm%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BJ.%2BBoehm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1644-5931" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frederick J. Boehm</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5642-5062" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elissa J. Chesler</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brian S. Yandell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Horticulture, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BS.%2BYandell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yandell%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BS.%2BYandell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian S. Yandell</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:broman@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">broman@wisc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2317&amp;rft.epage=2324&amp;rft.atitle=Testing%20Pleiotropy%20vs.%20Separate%20QTL%20in%20Multiparental%20Populations&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Boehm%2C+Frederick+J.&amp;rft.au=Chesler%2C+Elissa+J.&amp;rft.au=Yandell%2C+Brian+S.&amp;rft.au=Broman%2C+Karl+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493460/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493460/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493460/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493460/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493460/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493460/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493460/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493460/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493460/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493460/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493460/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493460/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2317" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Simulation studies</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Application</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2317.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2317.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2317.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
