<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2183" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2183.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2183.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2183.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/7/2183/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2183.atom" />
<meta name="HW.pisa" content="ggg;9/7/2183" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Synthetic Physical Interactions with the Yeast Centrosome" />
<meta name="DC.Identifier" content="10.1534/g3.119.400117" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Howell et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The yeast centrosome or Spindle Pole Body (SPB) is an organelle situated in the nuclear membrane, where it nucleates spindle microtubules and acts as a signaling hub. Various studies have explored the effects of forcing individual proteins to interact with the yeast SPB, however no systematic study has been performed. We used synthetic physical interactions to detect proteins that inhibit growth when forced to associate with the SPB. We found the SPB to be especially sensitive to relocalization, necessitating a novel data analysis approach. This novel analysis of SPI screening data shows that regions of the cell are locally more sensitive to forced relocalization than previously thought. Furthermore, we found a set of associations that result in elevated SPB number and, in some cases, multi-polar spindles. Since hyper-proliferation of centrosomes is a hallmark of cancer cells, these associations point the way for the use of yeast models in the study of spindle formation and chromosome segregation in cancer." />
<meta name="DC.Contributor" content="Rowan S. M. Howell" />
<meta name="DC.Contributor" content="Attila Csikász-Nagy" />
<meta name="DC.Contributor" content="Peter H. Thorpe" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Synthetic Physical Interactions with the Yeast Centrosome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The yeast centrosome or Spindle Pole Body (SPB) is an organelle situated in the nuclear membrane, where it nucleates spindle microtubules and acts as a signaling hub. Various studies have explored the effects of forcing individual proteins to interact with the yeast SPB, however no systematic study has been performed. We used synthetic physical interactions to detect proteins that inhibit growth when forced to associate with the SPB. We found the SPB to be especially sensitive to relocalization, necessitating a novel data analysis approach. This novel analysis of SPI screening data shows that regions of the cell are locally more sensitive to forced relocalization than previously thought. Furthermore, we found a set of associations that result in elevated SPB number and, in some cases, multi-polar spindles. Since hyper-proliferation of centrosomes is a hallmark of cancer cells, these associations point the way for the use of yeast models in the study of spindle formation and chromosome segregation in cancer.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2183" />
<meta name="citation_id" content="9/7/2183" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2183" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2183.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2183.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2183.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400117" />
<meta name="citation_pmid" content="31076383" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2183" />
<meta name="citation_lastpage" content="2194" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Rowan S. M. Howell" />
<meta name="citation_author_institution" content="The Francis Crick Institute, London, NW1 1AT UK" />
<meta name="citation_author_institution" content="Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2479-180X" />
<meta name="citation_author" content="Attila Csikász-Nagy" />
<meta name="citation_author_institution" content="Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK" />
<meta name="citation_author_institution" content="Faculty of Information Technology and Bionics, Pázmány Péter Catholic University, Budapest, 1083 Hungary" />
<meta name="citation_author_email" content="p.thorpe@qmul.ac.uk" />
<meta name="citation_author_email" content="attila.csikasz-nagy@kcl.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2919-5601" />
<meta name="citation_author" content="Peter H. Thorpe" />
<meta name="citation_author_institution" content="School of Biological and Chemical Sciences, Queen Mary University, London, E1 4NS UK" />
<meta name="citation_author_email" content="p.thorpe@qmul.ac.uk" />
<meta name="citation_author_email" content="attila.csikasz-nagy@kcl.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1649-6816" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=C. Arquint;citation_author=A.-M. Gabryjonczyk;citation_author=E. A. Nigg;citation_title=Centrosomes as signalling centres.;citation_volume=369;citation_year=2014;citation_pmid=25047618;citation_doi=10.1098/rstb.2013.0464" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. S. Avena;citation_author=S. Burns;citation_author=Z. Yu;citation_author=C. C. Ebmeier;citation_author=W. M. Old;citation_title=Licensing of Yeast Centrosome Duplication Requires Phosphoregulation of Sfi1.;citation_pages=e1004666;citation_volume=10;citation_year=2014;citation_pmid=25340401;citation_doi=10.1371/journal.pgen.1004666" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=E. L. Barnhart;citation_author=R. K. Dorer;citation_author=A. W. Murray;citation_author=S. C. Schuyler;citation_title=Reduced Mad2 expression keeps relaxed kinetochores from arresting budding yeast in mitosis.;citation_volume=22;citation_year=2011;citation_pmid=21593209;citation_doi=10.1091/mbc.E09-01-0029" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=Y. Kim;citation_author=H. Ding;citation_author=J. Koh;citation_title=Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.;citation_pages=1017-1024;citation_volume=7;citation_year=2010;citation_issue=12;citation_pmid=21076421;citation_doi=10.1038/nmeth.1534" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_pmid=16632515;citation_doi=10.1093/biostatistics/kxj037" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=L. K. Berry;citation_author=G. Ólafsson;citation_author=E. Ledesma-Fernandez;citation_author=P. H. Thorpe;citation_title=Synthetic protein interactions reveal a functional map of the cell.;citation_pages=e13053;citation_volume=5;citation_year=2016;citation_doi=10.7554/eLife.13053" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Brachat;citation_author=F. S. Dietrich;citation_author=S. Voegeli;citation_author=Z. Zhang;citation_author=L. Stuart;citation_title=Reinvestigation of the Saccharomyces cerevisiae genome annotation by comparison to the genome of a related fungus: Ashbya gossypii.;citation_pages=R45-R45;citation_volume=4;citation_year=2003;citation_issue=7;citation_pmid=12844361;citation_doi=10.1186/gb-2003-4-7-r45" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. K. Caydasi;citation_author=A. Khmelinskii;citation_author=R. Duenas-Sanchez;citation_author=B. Kurtulmus;citation_author=M. Knop;citation_title=Temporal and compartment-specific signals coordinate mitotic exit with spindle position.;citation_pages=14129;citation_volume=8;citation_year=2017;citation_doi=10.1038/ncomms14129" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. C. Dittmar;citation_author=R. J. Reid;citation_author=R. Rothstein;citation_title=ScreenMill: a freely available software suite for growth measurement, analysis and visualization of high-throughput screen data.;citation_pages=353-353;citation_volume=11;citation_year=2010;citation_pmid=20584323;citation_doi=10.1186/1471-2105-11-353" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=E. Eden;citation_author=R. Navon;citation_author=I. Steinfeld;citation_author=D. Lipson;citation_author=Z. Yakhini;citation_title=GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.;citation_pages=48-48;citation_volume=10;citation_year=2009;citation_pmid=19192299;citation_doi=10.1186/1471-2105-10-48" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=B. Efron;citation_title=Large-Scale Simultaneous Hypothesis Testing.;citation_pages=96-104;citation_volume=99;citation_year=2004;citation_doi=10.1198/016214504000000089" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. B. Eisen;citation_author=P. T. Spellman;citation_author=P. O. Brown;citation_author=D. Botstein;citation_title=Cluster analysis and display of genome-wide expression patterns;citation_pages=14863-14868;citation_volume=95;citation_year=1998;citation_issue=25;citation_pmid=9843981;citation_doi=10.1073/pnas.95.25.14863" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=C. Fraley;citation_author=A. E. Raftery;citation_title=Model-Based Clustering, Discriminant Analysis, and Density Estimation.;citation_pages=611-631;citation_volume=97;citation_year=2002;citation_doi=10.1198/016214502760047131" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=J. Fu;citation_author=I. M. Hagan;citation_author=D. M. Glover;citation_title=The Centrosome and Its Duplication Cycle;citation_pages=a015800-a015800;citation_volume=7;citation_year=2015;citation_issue=2;citation_pmid=25646378;citation_doi=10.1101/cshperspect.a015800" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=Y. Gryaznova;citation_author=A. K. Caydasi;citation_author=G. Malengo;citation_author=V. Sourjik;citation_title=A FRET-based study reveals site-specific regulation of spindle position checkpoint proteins at yeast centrosomes.;citation_pages=e14029;citation_volume=5;citation_year=2016;citation_doi=10.7554/eLife.14029" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=S. B. Haase;citation_author=M. Winey;citation_author=S. I. Reed;citation_title=Multi-step control of spindle pole body duplication by cyclin-dependent kinase.;citation_pages=38-42;citation_volume=3;citation_year=2001;citation_issue=1;citation_pmid=11146624;citation_doi=10.1038/35050543" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W.-K. Huh;citation_author=J. V. Falvo;citation_author=L. C. Gerke;citation_author=A. S. Carroll;citation_author=R. W. Howson;citation_title=Global analysis of protein localization in budding yeast.;citation_pages=686-691;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562095;citation_doi=10.1038/nature02026" />
<meta name="citation_reference" content="citation_journal_abbrev=Nucleus;citation_author=S. L. Jaspersen;citation_author=S. Ghosh;citation_title=Nuclear envelope insertion of spindle pole bodies and nuclear pore complexes.;citation_pages=226-236;citation_volume=3;citation_year=2012;citation_issue=3;citation_pmid=22572959;citation_doi=10.4161/nucl.20148" />
<meta name="citation_reference" content="citation_journal_title=Annual review of cell and developmental biology;citation_journal_abbrev=Annu Rev Cell Dev Biol;citation_author=S. L. Jaspersen;citation_author=M. Winey;citation_title=The budding yeast spindle pole body: structure, duplication, and function.;citation_pages=1-28;citation_volume=20;citation_year=2004;citation_pmid=15473833;citation_doi=10.1146/annurev.cellbio.20.022003.114106" />
<meta name="citation_reference" content="citation_journal_title=Traffic;citation_author=S. M. Markus;citation_author=S. Omer;citation_author=K. Baranowski;citation_author=W.-L. Lee;citation_title=Improved Plasmids for Fluorescent Protein Tagging of Microtubules in Saccharomyces cerevisiae.;citation_pages=773-786;citation_volume=16;citation_year=2015;citation_pmid=25711127;citation_doi=10.1111/tra.12276" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. A. Mena;citation_author=T. P. Treynor;citation_author=S. L. Mayo;citation_author=P. S. Daugherty;citation_title=Blue fluorescent proteins with enhanced brightness and photostability from a structurally targeted library.;citation_pages=1569-1571;citation_volume=24;citation_year=2006;citation_issue=12;citation_pmid=17115054;citation_doi=10.1038/nbt1264" />
<meta name="citation_reference" content="citation_journal_title=International journal of cancer.  Journal international du cancer;citation_journal_abbrev=Int J Cancer;citation_author=E. A. Nigg;citation_title=Origins and consequences of centrosome aberrations in human cancers.;citation_pages=2717-2723;citation_volume=119;citation_year=2006;citation_issue=12;citation_pmid=17016823;citation_doi=10.1002/ijc.22245" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Ólafsson;citation_author=P. H. Thorpe;citation_title=Synthetic physical interactions map kinetochore regulators and regions sensitive to constitutive Cdc14 localization.;citation_volume=112;citation_year=2015;citation_pmid=26240346;citation_doi=10.1073/pnas.1506101112" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. Ólafsson;citation_author=P. H. Thorpe;citation_title=Synthetic Physical Interactions Map Kinetochore-Checkpoint Activation Regions.;citation_volume=6;citation_year=2016;citation_pmid=27280788;citation_doi=10.1534/g3.116.031930" />
<meta name="citation_reference" content="citation_title=Genome Instability;citation_author=G. Ólafsson;citation_author=P. H. Thorpe;citation_title=Rewiring the Budding Yeast Proteome using Synthetic Physical Interactions;citation_year=2018;citation_doi=10.1007/978-1-4939-7306-4_39" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. J. D. Reid;citation_author=S. Gonzalez-Barrera;citation_author=I. Sunjevaric;citation_author=D. Alvaro;citation_author=S. Ciccone;citation_title=Selective ploidy ablation, a high-throughput plasmid transfer protocol, identifies new genes affecting topoisomerase I-induced DNA damage.;citation_volume=21;citation_year=2011;citation_pmid=21173034;citation_doi=10.1101/gr.109033.110" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=U. Rothbauer;citation_author=K. Zolghadr;citation_author=S. Tillib;citation_author=D. Nowak;citation_author=L. Schermelleh;citation_title=Targeting and tracing antigens in live cells with fluorescent nanobodies.;citation_pages=887-889;citation_volume=3;citation_year=2006;citation_issue=11;citation_pmid=17060912;citation_doi=10.1038/nmeth953" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=D. Rüthnick;citation_author=E. Schiebel;citation_title=Duplication of the Yeast Spindle Pole Body Once per Cell Cycle.;citation_volume=36;citation_year=2016;citation_pmid=26951196;citation_doi=10.1128/MCB.00048-16" />
<meta name="citation_reference" content="citation_journal_title=Cells;citation_author=D. Rüthnick;citation_author=E. Schiebel;citation_title=Duplication and Nuclear Envelope Insertion of the Yeast Microtubule Organizing Centre, the Spindle Pole Body.;citation_pages=E42;citation_volume=7;citation_year=2018;citation_doi=10.3390/cells7050042" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. J. Saldanha;citation_title=Java Treeview–extensible visualization of microarray data.;citation_volume=20;citation_year=2004;citation_pmid=15180930;citation_doi=10.1093/bioinformatics/bth349" />
<meta name="citation_reference" content="citation_journal_title=Small GTPases;citation_author=I. Scarfone;citation_author=S. Piatti;citation_title=Coupling spindle position with mitotic exit in budding yeast: The multifaceted role of the small GTPase Tem1.;citation_pages=196-201;citation_volume=6;citation_year=2015;citation_doi=10.1080/21541248.2015.1109023" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Schafer;citation_author=K. Strimmer;citation_title=An empirical Bayes approach to inferring large-scale gene association networks.;citation_volume=21;citation_year=2005;citation_pmid=15479708;citation_doi=10.1093/bioinformatics/bti062" />
<meta name="citation_reference" content="citation_journal_title=R J.;citation_author=L. Scrucca;citation_author=M. Fop;citation_author=T. B. Murphy;citation_author=A. E. Raftery;citation_title=mclust 5: Clustering, Classification and Density Estimation Using Gaussian Finite Mixture Models.;citation_pages=289-317;citation_volume=8;citation_year=2016;citation_pmid=27818791;citation_doi=10.32614/RJ-2016-021" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=K. Shirk;citation_author=H. Jin;citation_author=T. H. Giddings;citation_author=M. Winey;citation_author=H.-G. Yu;citation_title=The Aurora kinase Ipl1 is necessary for spindle pole body cohesion during budding yeast meiosis;citation_pages=2891-2896;citation_volume=124;citation_year=2011;citation_issue=17;citation_pmid=21878496;citation_doi=10.1242/jcs.086652" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Song;citation_author=T. Z. Grenfell;citation_author=S. Garfield;citation_author=R. L. Erikson;citation_author=K. S. Lee;citation_title=Essential Function of the Polo Box of Cdc5 in Subcellular Localization and Induction of Cytokinetic Structures;citation_pages=286-298;citation_volume=20;citation_year=2000;citation_issue=1;citation_pmid=10594031;citation_doi=10.1128/MCB.20.1.286-298.2000" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=J. M. Tkach;citation_author=A. Yimit;citation_author=A. Y. Lee;citation_author=M. Riffle;citation_author=M. Costanzo;citation_title=Dissecting DNA damage response pathways by analysing protein localization and abundance changes during DNA replication stress.;citation_pages=966-976;citation_volume=14;citation_year=2012;citation_issue=9;citation_pmid=22842922;citation_doi=10.1038/ncb2549" />
<meta name="citation_reference" content="citation_author=M. Usaj;citation_author=Y. Tan;citation_author=W. Wang;citation_author=B. VanderSluis;citation_author=A. Zou;citation_year=2017;citation_pmid=28325812;citation_doi=10.1534/g3.117.040220" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=M. Winey;citation_author=L. Goetsch;citation_author=P. Baum;citation_author=B. Byers;citation_title=MPS1 and MPS2: novel yeast genes defining distinct steps of spindle pole body duplication;citation_pages=745-754;citation_volume=114;citation_year=1991;citation_issue=4;citation_pmid=1869587;citation_doi=10.1083/jcb.114.4.745" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Witkin;citation_author=J. M. Friederichs;citation_author=O. Cohen-Fix;citation_author=S. L. Jaspersen;citation_title=Changes in the nuclear envelope environment affect spindle pole body duplication in Saccharomyces cerevisiae.;citation_volume=186;citation_year=2010;citation_pmid=20713690;citation_doi=10.1534/genetics.110.119149" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=J. Wong;citation_author=Y. Nakajima;citation_author=S. Westermann;citation_author=C. Shang;citation_author=J.-S. Kang;citation_title=A protein interaction map of the mitotic spindle.;citation_volume=18;citation_year=2007;citation_pmid=17634282;citation_doi=10.1091/mbc.E07-06-0536" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Zackrisson;citation_author=J. Hallin;citation_author=L.-G. Ottosson;citation_author=P. Dahl;citation_author=E. Fernandez-Parada;citation_title=Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale.;citation_volume=6;citation_year=2016;citation_pmid=27371952;citation_doi=10.1534/g3.116.032342" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Synthetic Physical Interactions with the Yeast Centrosome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="The yeast centrosome or Spindle Pole Body (SPB) is an organelle situated in the nuclear membrane, where it nucleates spindle microtubules and acts as a signaling hub. Various studies have explored the effects of forcing individual proteins to interact with the yeast SPB, however no systematic study has been performed. We used synthetic physical interactions to detect proteins that inhibit growth when forced to associate with the SPB. We found the SPB to be especially sensitive to relocalization, necessitating a novel data analysis approach. This novel analysis of SPI screening data shows that regions of the cell are locally more sensitive to forced relocalization than previously thought. Furthermore, we found a set of associations that result in elevated SPB number and, in some cases, multi-polar spindles. Since hyper-proliferation of centrosomes is a hallmark of cancer cells, these associations point the way for the use of yeast models in the study of spindle formation and chromosome segregation in cancer." />
<meta name="og-title" property="og:title" content="Synthetic Physical Interactions with the Yeast Centrosome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2183" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The yeast centrosome or Spindle Pole Body (SPB) is an organelle situated in the nuclear membrane, where it nucleates spindle microtubules and acts as a signaling hub. Various studies have explored the effects of forcing individual proteins to interact with the yeast SPB, however no systematic study has been performed. We used synthetic physical interactions to detect proteins that inhibit growth when forced to associate with the SPB. We found the SPB to be especially sensitive to relocalization, necessitating a novel data analysis approach. This novel analysis of SPI screening data shows that regions of the cell are locally more sensitive to forced relocalization than previously thought. Furthermore, we found a set of associations that result in elevated SPB number and, in some cases, multi-polar spindles. Since hyper-proliferation of centrosomes is a hallmark of cancer cells, these associations point the way for the use of yeast models in the study of spindle formation and chromosome segregation in cancer." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Synthetic Physical Interactions with the Yeast Centrosome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"va3LtuDlYhOISF0-7KCp6S0hGVoo3RWtvdH6jnXu7m4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493671","apath":"\/ggg\/9\/7\/2183.atom","pisa":"ggg;9\/7\/2183","ac":{"\/ggg\/9\/7\/2183.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2183.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2183"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2183"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2183":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zs0qU8l7DCJ8w6ox4h4SqkNoP4b1cIEIn5WLmJqmVr0R43aJGzrahizoR9k+x7VcZ+06ucj7nWaOCzmGFaV8+DFjoRn4TNcI8nRsM2cjjb6WRgE5SYinC1ZZ70jEkR5Jif3tp21WyAkJ2oi6ojn2SK3gftSM+wZemE50llsj5m5INSvy62bjPhrRyID+WesRMDCDFA8qStv\\\\\\\/o+5EAUv0aR2foJwMg8gaX1W1H3sQQZ4Byf3+yoNR+QHXt\\\\\\\/JkHbSe4YwOmYha+X3bGtMqF4rDrxezGgcn1QCyTpLfYovD8sxF2Lp8NiEK4vQ\\\\\\\/a2vvXT6+CvkvJy7JgnQajREB2PxclTEZ\\\\\\\/rcclgYd2srDUh4+u2SnfDd1Xi7\\\\\\\/5q83gHpquFzzCdRLLTMQk5Mvrrf7GIfmV4SMKLTrRrM854aadq2FDlCW8SDYXBhuqHOj53C1ik89wxbqIBeCPj3N+zDEsR6V5Kn881ytP31kGQs0TPYZxDdZvYbSSHw5YPsYAtYc1k7CF2SjRj2cZVnxenaY7W03N+g6kGOUQEZZwwbi9pjVNvZhA3bSNINY+aXfz8zKCPqKm1T7M0JF1O\\\\\\\/iw6MLhm9mO7DQhtpcJjQxHj8aTeSVqqt\\\\\\\/k5t0\\\\\\\/KwSQhOg728Vyid31r5+YqArEdyRpuV7swPZxNZmuDBErfeGHwqVhelZ+IepLpaXwXw0Ssil3cUfskpkvWJjQ6V7tVgdKibL7zhljx3bz233iKadYgnI8ExE+s50IPSIscMV9TD3WBVhjSqGGWFfZKQdxfSX1DI\\\\\\\/rpo5fZ8AW7NVdz3i9\\\\\\\/VkVFh3MySPT099FkeLPSkw6a92QrD3aTJ6Y650kgrEV6T6SvgZHGeGEqkdXGv\\\\\\\/8m\\\\\\\/qARg892Ef2KTecNZsBHZTPYfNKbMyWZRWcNNwDZOllEw4ef1RT\\\\\\\/zIQqmD\\\\\\\/U6r0CQShhwtU7NCPxCQeNIDD1UjUhx6OMEbVnl9X8AZZnLEWTA6E8MIyj1rgz5eTDNJzSzlM84hPI6dglmN+fYpM1q6Jm4qFNlonY17U\\\\\\\/F49N0+dGbUlQa5hYMudhUtxzM3Og5i6p3UPULLYBnZAL0qU0Q42b+MM9TSOE8h9HYl7EnZ2OSbUal8utFiktEbYyNkSz8mOK2iK\\\\\\\/pdfVYIaiA04xADT0A\\\\\\\/0pamadb2e7V8foGw+tDT73WcylK2GCRAqGHxBOxigXJxdGqAdcj2aQxeXpen8KvNfyToRbJRSUeoEyqMewavOdAd7Dn4kyV0OSLIIkW4D4k8yBJ2xzAEmE6zT232KtjGSOtt\\\u0022,\\\u0022iv\\\u0022:\\\u0022humBYq+Xih6rVZ+Z\\\\\\\/S80mA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223ee1dbb51966728f5d84b231e8fb8440c1453411bc8513486da0d421f4cb5cc7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qyq1Z+gkM7rpm8dWips2Lk2GAioHXro9V5p1pDO8QUd9sbqbM7EBeM1UO4zOd1aIrmaOQn3wZpe0pE\\\\\\\/Bd460lY7SVRXP1wG\\\\\\\/4O6l0HKZXpO5O9A4j2ESjxG9GSD196AthmgH7DCZMmt\\\\\\\/VenVBtG6nAib+QHxoEQkcpTCcpJ3w2uzB2PE6QDvOxwmzgFDaVrtftOjyI\\\\\\\/eoHBoCau2E+qtbMljTdEbehjtg5uFuUyNFCXHgOY1D075I9dK1TOMJ0BI0iqzEjDSDRQV6qiKo2NYE5ZU6Dlk8gG6uCqEMxtpP5N7uGB56J\\\\\\\/NY18aL9pU0ARMEqo78btM+KMxIg8NfJdHim2UVeb3d12awW6MIiQ5hhyakrScJ+HK9kDd1D1oGKt8Pkf1g7BqpuD9EB\\\\\\\/lnzvDdlMkG3GuskPotsydOJBXqMG45I5f46y0Vy\\\\\\\/xnjBbBmHf4lXMoK3cCcuAg8XV9nM+uT3cQ2OhGVFdAvJDtX2yGxckRrih3isQRH3PYSNnHX+mWrDiZPixeTGGL+FjK91Grqb8SRPY4KvRHJgB9a24dGH2lE8GI0hKp3yCeYfaZJnGfwR5Z5JupYP7OlAIKMmIi6vyoaqZrXY1oAhNPa1ZXRrtgTUiL+RK7WygUhRcANOURgMcLvSQJGSZfopz9pykUAsFWNnThIJV827TbKaoaWQOSLVzvXrC4z5vsDVcvScEZ4sQSYWAQKMCR2+1qN6\\\\\\\/W9P8Jf\\\\\\\/XBy08kWoCbbFqz84PLfVEjZbmHX7zm9UbLfSNxkyvqk\\\\\\\/QHSBa64THQ3sEKOzOfpfoLV\\\\\\\/X0kUN7l4EeLDOFwapy80tq4LpHAZl5Jb8s90NHXr1zJY6PuouSztsH0Ba4fLJsA+hi\\\\\\\/7GnS+LJTnTOmZlrixUh06duQxpdznEMwhMoXaplaxzg3f3fIeL423DcVl0Ju+TVqwZmWH6qEcSBfPzfVpsS63iO7pIamV6nhHbKQSGJVuTnGBQ4OKHv20OysdUHqgS3X+h02c3QZz3sKFYu9zpVQikQ2sueT0juk050OJ0seZMVpUT7S+Egnka7XbC0nBpLb2O7b9cQc4DmomM4BBcTePULG\\\\\\\/bd9FKNHEqrHQBH0YnTDNEmtGM82GJl0kN18mtBs1Mfipo5w+ZfxKkfsW\\\\\\\/eqwWJ86NrzlXjlQlrRnruG\\\\\\\/OgZVk3\\\\\\\/LYPO6WxH\\\\\\\/YQTAckJav3DUqb\\\\\\\/vnHWeeBvxiPyg6nEpg\\\\\\\/DXrCyir\\\\\\\/cw\\\\\\\/IYtksl6AMnGs6iceuBhpfmwnioRX9bIsnTdjkFmfeQ3aQncQWoXMyoa8WLa2gSlhx6nyoFyH+1naTvNjANdkld1QsIIy26A7lyS\\\\\\\/eqyhtYBRI7HONdMA5dPnyDxj9mVEbntAzI96E4aGG0RNqMano\\\\\\\/Ayw4w2p9EVCutZeRZh8a2jVr5Y0wd2qlyAxhbwv9Z\\\\\\\/RKpvjUeQX3FyeKhIQZNIL8CEsjGtjdGoGgoJ8PRMx5x8MnW73rFe+2Y3eXRhbmbWm8\\\\\\\/9YvqlQDpI7zAquXLt0CmlvdrWh++P98vsV3\\\\\\\/t4I49wKpvio19lGXUV+R006N6bPbIsvSB\\\\\\\/Q2rcK\\\\\\\/GTh8QPSNYhElbVveWuhswGUz4tR9uSlMfUXOJuNNJbRXgis\\\\\\\/LZLzG+bQkh6BN\\\\\\\/Yz1P\\\\\\\/EwxLswV9S+mIzcFYqCr\\\\\\\/l8l4jptthWqN6Cnq9N3f4SYtsT7Nbq1wI5KaDNm5sv5N1N1MvLvx\\\\\\\/sGSVQq+JyecUqN1QHQRawCjMJgzjONZoh1ZlJ2rJ63c88vxVoGdc\\\\\\\/s510ymcj0SQpvO4D2eHAAaWapP8sd7d+u\\\\\\\/VOTy+vQYq8mBDNLCsiX\\\\\\\/wb\\\\\\\/ehihxDMzKM4X8WcD6TMFsXYoyA7y4iHFWyONYoqPk3HsU9Bk\\\\\\\/KSxIjfR1rvyMgH+YDlO6Wl63ypnUUM4crwxezfrNFkCJH2Vi30Z+8al7Px9BkEb4avabAhdRvhdiHXN1fpCl3JjURuLfRJPqulZDtE1OILpotfi+dAgZXXu2p48FY6\\\\\\\/Y0SX3IhtZXarU2k0A8HDs3cW646xwI1H+t4YtIC7JRZ2EKHgSDwqwlZ\\\\\\\/RaVB5ALi5R3oOzIYS4JnBui4z1ITBGhnj3WJXb5yc4xOXgWtje1D9RJJ1qPSymWRrADhPH4fD3uH1j8+wzJ+sCv0eYmWt2PRr\\\\\\\/FE8hzN6wYyRMwHuXAvtTEuvS+e4fjJ34pI9cvsf+myPBzPWaW1QiF9ZzL6z\\\\\\\/1SoypuKhZf\\\\\\\/pDNoGC8cp\\\\\\\/\\\\\\\/gr2UMvPgBvvkMxbif3pndCCNPr5l+tr0iG08LVGvoWytjHXGhE1KXdrdR542CGcNfPj1BAGsWAx4sxbYOjISCk6vcFfKuYn4toykGsrPMFIz1JwukL2AFsSD84xnw\\\\\\\/D4aB3FrRPjlC4X79fgFbeJ5nzrGJltwU7rPKJMW20me+vs6czkOwIbA\\\\\\\/EmjafQGIER0tXo2A\\\\\\\/Jb\\\\\\\/ZEc9H4Ao\\\\\\\/XjyWE6jaT+SpkF87nTwKKcrbohnEXXmoZ5OZ8nxJjaxKYTVO5h1KV34ukRl5gGk8L1VCm\\\\\\\/qOgG\\\\\\\/wRD44YJTC5\\\\\\\/4j4lx0yrIagfxHB3WeEaKbuAJedRvzuYjJET1X\\\\\\\/j9rcBbJ65OSQKpp4j2R+JYyQG8RTWtru1LAN1JCPlgRhC4VsJidX7uX2UKDudKSPUU3FcPJsDMoBVAt2wSQWa0uM63XYC7j8uy42TKaRU79NLuj7MPppSMIX6pdNwbK9rmGcXsGcwmgod8GRhUhXt+01DFxlIE2FuF+d\\\\\\\/YU4HxndWALL4DR8S8075s1QK02RM3H0EY9HSj9\\\\\\\/tI\\\\\\\/6xchhKt5P4RrB12BqR\\\\\\\/5byC+XZB9uqjRLEFaZ\\\\\\\/UHTHSHO4rjI9PPJeIZs\\\\\\\/aLqsl94f\\\\\\\/oTxIcM6nXmaW1uyELoi\\\\\\\/QvKe1ptpX0L\\\\\\\/6Od3cktk84AjuBC8u4RkdkH0WwbHq7SZa5lRy+MrsQPsozgrSJfmjW9RUzOYMjVvO4vkUBVAvkalyfusxKWtZAThvxfu+4hO3UXLsdqA4SzbNvHSMrH8jmRKhEa7GE0bZEn\\\\\\\/QrN0FZowY3W2J2OnusuHrAEOLhARlTwIxQp4bFPWjsXSV3EHXM3Kau5Nlh4Lr3H\\\\\\\/jCb8bNj0Gom6127LInSB2uLXZ\\\\\\\/pSWYuw3VyQPqu3EPdcFz5XbXzv2rKKIeW9UU4F9wgucnQ18DDg+tqrgeYZyXyfiiQtXhMc6d6iIiipjVWKD9WAFFJFD754MMQ\\\\\\\/5yS4zejpp1MdF13mfSEZ32UgRT0CmBE4zG5vse3829kYxdkDqAVwiFqZY9qgYl0MY5CzQ85Sh9T8pPe4dht8aain8gO5pbt5aBXjouIAv2Jfj6vZSKNCtkOMXsHG2cANdfUc2EOzy2PmsR72qV82sFUH50FIoRv6cv5PZoutcLSbbMiuDTCMiRxE+mga4WREggzapA1E1asI3cAnKD2QAp180OtTghHn0PLRfdkTt2uyVGeCjMB9gEmoZwrq91hhRyJLg8iH6oW9AQfl9zRyqWNFJX8ebR961POy6ZV7hVMIQ70nyf+xOceyIoBvxMWsjM6MAC3kQzKsbrp9U5Uzai4eKMuz0I1ku0BNFD4XMHm+9wUO4Gpn15+iE\\\\\\\/6C0xhBJC\\\\\\\/LTXwVcNXBzHYNmKjnr+HoVDR1GPaHeKYjCZwicnG4c14JL9a6ymxi6vTTK1mZROOlQdzpgOGYga0CZfhhwskcphRzkEGwsh+nlZCEgAKrGlnmqTxaHU9ne2ENIAAQEJPiiB5FkIhXP3sohJCYbyG7edYarpI2CQiWwFplEQobHghBupRMOcAGDZ7KCITJ9AGGfuC6WUjLUL76c2sFFkzkN4HR7gYvtCT\\\\\\\/fL5wxYXVwaA\\\\\\\/rPgew6qnZM6TYb4q6UpQEmtvXy6PqHk+HQaof34DJ9phLZ\\\\\\\/WJSqnclFGiZ4qacf6WI2UAa6d9kuwmj+J2gA5ct8bPK7JR+4ldfufXAeSOW\\\\\\\/yx8gJQEgrwaNC2rswVHAX8IluIiPxfDQHnS9JEuQTHGXxYeT6LJHSRbOuC5mIUbnuJLKbGNXSZchWoFasnxXN9FyIVCPx6YyBKSIcoFPJFsP+EoKspuIN3ETjtFgrPN3CuWnEekFb\\\\\\\/UTSA+LoRyQFwBbSmCWolE6qsUywW2WRgu4yTdW\\\\\\\/k9Rz9cjuvK6JC9xnRNBjNtKu6PfqAPfy0dJhqaRhxvAYIFCnpb\\\\\\\/cGGG\\\\\\\/qRXsQCdUfOKPauOkdUehA7PJ6H0jXUG4LJFzSQeFZZmYae3wB4yzNHwXG4dF0wWybXdRKMsN8rKkBlGU+z6Vf7vzr\\\\\\\/8qOROLRDVKGdm8HxHY+upbqxP9DsaWtw6OGbhzCW3Rk3+2jvvCQ0LYmEfNu\\\\\\\/q1ersHsKB4HxpaPPwW+w+e9FZc+2szTusSNVIMILlwcYj2qLTKsg8K03NFDogwt0xN7aTz5HVo3YNHn0qW\\\\\\\/TgBv4mRGYQIljbUeWSJhkmIrwxDDXP9BxQEd182y1qhG8ntLKmTl1HJvECNNVrpQif7rSimR2nQ3vHwsqvMT4fVhPW8hYNcVQiucL+Zm5xaFBnqkkENL\\\\\\\/pc6ZFXlk1sYCBeLTtNaoHc0+R7CaVKJQkEkxNnwV4vgdNtJkKWBxBLFidOlVBNz0+daxOWahdq6O6ZdTnaAnUCS5ZE9rTdtGJKhNxuTp8M75D\\\\\\\/Qo14nhTGtIrhTY7Nd46oAckAW7zNkxaBBMfx59aF+IkzvyxJedGeXurTm2db6jqLXXl8RSUqfe3U7PqmATQD7dcEPyKL9+9ewnvTtMW9fCPJA4UE5OQFYCmESCBOiiuQjw+45MtUYVqk0Obr71hDP24R+s4gzrx2T2VfvRn1dCxQ945uD\\\\\\\/\\\\\\\/vNlsDYhvZOyFy21XUBV1FMzPGKhO99Yw0OuTwzUf3wNTvD0dgNt\\\\\\\/ospCdVNsPwnWbbM\\\\\\\/fat252H8h6eYL4Djhkj4945P6M\\\\\\\/QM6iQUO2CY\\\\\\\/seJo2rgqKUJpReEgp5x0LjPVoULgkAi8YrqgCzYzscfKOXzY3VZeevfup4xd0NtJz6uQzTHIRql5eEwSPfjlv15\\\\\\\/3+keAaNt4MA1ueb8ZMGQP0ElaUmzDz8mQKZeGKZiKe1YKDz1kQAWDubnfX5PwPN\\\\\\\/PkWsh4QPUKjgXrMQGJaTl168YTBfd\\\\\\\/R9BGbP+gR3\\\\\\\/E6X29UzmZkTShOzsVkk04qsjUswLykMfU7QOBxBOxfdiadErjsMML9iXEX38hublcKSg4fH1hRQuY1xvSLxMJjtz+I0NwvjwAb+S2HmWBL2Q0AuBz+t89FgCQo+7R9Pz\\\\\\\/Ak9KLdHg9Y3wNJLRzemhOCsitmWlD7gZqKjQgduSwz0FrL\\\\\\\/v6y1lgH8xcLyLgXlg55Ff71qwZNK1LhNbZstDv1oW4ig7j15Y5htRZdFt9wzwl42do0ePLyEGADmintp6uqY4GSav2x8VYzY2ywCALB1Vt1FouLtX\\\\\\\/vmUSV4rJZDKdc9sB++HlGRpwRdkfMD178z0OqZD2qXKkiM+1ubodJqC+koirwKcNMPtxUf\\\\\\\/Kydnipu+GkvpgNw6MkQjPijcLAjWGmBU4nBvndnqHZwhaS0a3WjWpshhEmL3nGwUdGtJtzTMODHRLkPw6PQqV9PVWV9NyQLlal6JzXLwpnUtARVkWhwn2\\\\\\\/N9en6ASQCWXr2QPffl0O3AKfgg65X0Uywqa52FmpgzZ3ec6tyG2\\\\\\\/6hQvYvOdjnsQ3QGKeUjPQDMW8Gc2OqvpD0ApbV\\\\\\\/OD4NyOTMD50gEVYGRLkE0uVMW9M74VxZLwBkQubVqBMc143i7ZiDmwJ6bwDQ+4i7MJWj+SIuxiwlkCqq2JbkwQwm6cv5QVX62hBgsZFoAHErklQS4qy9e5KgkIRmyH1dgFZ7ZUtieUPQu2RitkRa7WNj7wKW\\\\\\\/pH7kY0rtl62r1Mm91FmMkW\\\\\\\/3\\\\\\\/XDmTligk9rYvOGcpSdgly6kvPJJ7gr5Lrf7eNo8exkf7tfBmnJoKdENVRaqPRPAxMpiwhwj7DhPhCI96hZBQ04NkCs0VIzdneXk\\\\\\\/QbYyxqtJXMWZfU+KG\\\\\\\/P8mpfRgPotSyw8xD8kx3rIwOsAhv+0Qk1vnkXafo+bIEJTMAaVrzx2k3JMKUGH\\\\\\\/ZHOiVSdag2FYvHHpSclIfsADmsF8KmwjGje2Xy8IS6vnyGDD\\\\\\\/1T+MrQP\\\\\\\/GDZJmoD+E5R6jEn+Y3gP\\\\\\\/5jjS8mkPquSJx7j+T1PaE22LqZ3RE+8FkjzvwzomUZVTToesRfjNXUBEgCpR+AVqK7yeIPvwBumMmPNgox6xpTKJxrZDmcps8XmVOGPGcPC\\\\\\\/B9Bf9MNnkw16Y4TFVtx0ijDGXGEfto817wP3k1\\\\\\\/d70gYQC6btGNKzUBB5l2dn+rR93FtpP+vVyNAtQdfJXehGeVXc0rZYwhEVBlK06qgMc0Hdn4wpQEFB0Q9lR6qChnA9VJvxFEMyunUQaWHMyVnCIl1Irmf38Tadtzb6HQiPIYH08avWpcPmDWydrfQvRxkqPXXDhm7NbToAGOUbhj+207ilOnb8haJtnIwzgRt6W75WDAUQeUNlzIYP6b5c\\\\\\\/cqYFpzmIm3+ZSkUnj5VY2+ViuqLGe0ZbtEmkeY28N\\\\\\\/mV3ewk7FE1KQkw\\\\\\\/DpYUdn5xppjmAQZ5lbxDdr7438qlRYJACoFYV2xw+FqrYpADrYOuNDbbqncq05VdwE3LScPO+23cYJo783YPIVIDyMxRJiYBIQLpKYPTAL9eX\\\\\\\/pX2xgUfzaqJ0dbri0L8gw6Oqt4DhSDbs2r9mWccOzgmfk6+2ezwZF6HmNMQUEaNX1lrBL5frjd1GpKn\\\\\\\/6cVUv99oF99rXALc4zesBmurTwIjt9pkWzkq2MYS3rrY0LhPQ4vOtKTSiU8HkvMZjUrXWwd6kNH8vvEkHqM8MjLlSMxhL5uPlcgRU\\\\\\\/ZbX7uREUxTEWIXe3O021sCtrEQlbLcd+hLcXEYU+paz8y3ZhxVCWqIJr5Jqbcdw4QDMn4IiuRI2DR+Fx2BlI8BodlWPvNFbLrsVkuc7yMm5xnVbMUV30rK9ffmNwxIr6TuLiRHeDynQmhhEzQAC6KW5RGZ7m3sNbWsb4pI0UCdsz7\\\\\\\/wM25WLbsF1oO2k5X6Ii9L885N8dfKftO46abXO0yU+hJa7\\\\\\\/ANMfnyKRZBPQ3Vt4ZixjjYlQF85EegfEh2qgiUENjsAi4db7dWhISv1+VDyNXrvNd2nd0bZl3h1AQ96t3kEP4t9mw8M1m0CkH7NaSwldZEJ5tc7Dr4M8dRXH4FF0LR45\\\\\\\/cXpWr4qeI1\\\\\\\/yeXcMuRRXQyCckuE51o8M9UyP7a\\\\\\\/j6KFputFqczIAIW03EA3sagzZkduLxYPMEGaKb93tr+YmRi+6f0Dlj9qedBiUo9296f8g23JyRAwK+9uIWm4bKCgDE1OyJPFxo6xZS4JHqw0yQmBwowFv1avR20A0r4+h+mEY+AbehsW+78L2rXBoeEQirXnzkBmwFhUyS5W78tIoXQ6IsONCVxegwjmR0FykRTcnJ6Kt20jRZyVKr8mC02PUij2XAML\\\\\\\/MEpTYzd1HqrSWaORJxqKRVXYScH9v6HGSPqCzOwlFMX9o1r+hmhToquPdsMqdyMijNS0o5Y73r8IR5baTQbVkZZch0fcxOqF6xWBweyocLDdgmAClNqd6O\\\\\\\/Q0HdbTUdNmV5lwHr7TORcLHTCOv\\\\\\\/6F\\\\\\\/5kYq4OykkZuLu4L5jybs5DOIvqoHKjTzTHWyQ4AlBG6PuoMwZ6j9VNgOVHG+OhRbdYi26ctpDWZerR9WwUeHW0m0KCwUiY8Rl80nPXo4jKE26CLffcq5EiaQ948Vh0PsAUvURsDQWy+v346kuVpJJ\\\\\\\/8Lv3ckee6mJM6472p8hwYm3\\\\\\\/AjUTeuocF8K1TTlnwaOZUrrU4qa\\\\\\\/BOxSC84u4+tK\\\\\\\/BEM65WUe1ITLQPK99M85J9EdoQkqt6ZQEqzMaFvE1XzMhRxxYZY3AB0IXSVPeHbjGKf\\\\\\\/ylnrdjfjdXIEBpCyJmKqrwh852y5HX5tP0rBgZw7hD9TlEUs0qHdUMPtZ7G3p2dj2+wp5uQFvgBmkOfR\\\\\\\/834IEuzIXlHlELLDVipJyTNbNoG3a+pt\\\\\\\/EHO74oO9yTh5\\\\\\\/Mttu7HUipmXEeKX3qb8+PIQSbid79slMuFlEKQSZxXA7gh7L2doBlBS2TzVkdPbn7ahH0XIr+oP6V0mk76OWO7yoTd\\\\\\\/lqw2d4Q0UgOoHt6lM9zOK8Z5lKNatY\\\\\\\/mRFpE1NhW+huaJvqyavBzt4JNOedOnnTiCkbNE98aY2epV4u97pHmGdfAWVM0PSexrhUtP4Bho0rR3yhT8hTtYRukS65Y2qhpYD7saSh3fz31LlL0RBLkF+5OCRgPUTHugt1bIqWf1hlRyPf+WMbTmiXeeKhY+KRZADE1ZGytWjasELlRFVLJn7UMRYkfuGokYPXlwzMzVWgJvp+tJ19OO3pF5BL9PznGoFUKJw2S0PKDu6eggpN4psj2kcvT7VXOxTPWx+2gO\\\\\\\/Yew7ue8Vta+HGzkbxG4QR4c6kPNxjvWFewj26oYFvQBe1n8x2Hlh73zQ3pGgL43KtbV\\\\\\\/dHp74wCy9WVH51GC9tI\\\\\\\/USVIo0XSAd7CHzSD\\\\\\\/DSzuboy4D98aaLk6lxeW5u7Y8ja8th7w8a7272iN+xZBh6cwHtNxKPCT8POM025WKLcZcDjTD0nYrgU2kptX9rouFEbZor3n3GMmxGOYuWZWqC2pn84lG\\\\\\\/Ed5yms2SZ7QD4bBwtGMWYQsETqdfC8pwXrSujOJTuQijZ21Xv+po2LLfVVRSzYMLl4eULevQg9jzu2UBF8jPsHJNZYvmmY697kENBXHDnzU4GRYyLBEuLezh9bW2VGMIDpardfGoCX8L59Kgjfh6LO3c3eAiqoZXHf8ZBpuDFBtQm+YLqdBOOZIKK0XPX3BRaJPPVarC2D3jt5YEnPX1dVMASXFGNl6qEFUIR8B7MpJKMG9JkvD+8aLghgFJ4AUtXAL\\\\\\\/72V2T82rexHECEyLf7CsTgF3PomoRnHDcSNya3rWvAnzQDfnAUiIPXO5sm6L+Kg3qww610dYyp0xwsD8ALlxqPvqJaFsemaYEEBP2OrLZfLtG6FwM\\\\\\\/R9srR0QDJ3e4KxCEjX9E9SGMIT46\\\\\\\/vqfu0w8ds0Iiv1OlDKO5i1do3AQTK\\\\\\\/BaZsXw9zYDEDAnW0nVZCcB\\\\\\\/fYaSZnfGth+oMrResdzOwFqk0CQg0Ss\\\\\\\/dDqRjtO+uVCDwNoD8IkB\\\\\\\/YEVv\\\\\\\/h10N+oZNKIb\\\\\\\/5V6ZLEKonIDIcZtYu8Q50WipGqzu8gSz3ljk6EZqLzYuLtlezvNWSQnaMIYdSk6\\\\\\\/hlt6oanW17QjnLQUigXsuIUfvT2sA6bTS9zVTTexD\\\\\\\/6+Ddt6J5tvmANF82ljO5nVLX6B4asTdJy2EhhPwzQdKEprrgz3MJpzpJSe8R5kElZynUGSVOxHRFptC8W+SfK91nBcA\\\\\\\/055RbojTnJneF\\\\\\\/uEBm9RCOgrk77cz0H\\\\\\\/\\\\\\\/eMDq8chefKwkAHClnD9sL\\\\\\\/TD6qJdHvgG67crSLTachHVKwcKZ7E9z3Kl\\\\\\\/ToE9ixFTM3RvJ7YSgsqZDbEXSfZhJ1THBro5kOREA+G1ZXta5Jr4z9aU+QH5FeiWOg9ELxoSMaIkNtCV18qkJUm5t0DVfbWe+VjCsn+8\\\\\\\/0DP5C3F\\\\\\\/uqKCbDL\\\\\\\/oxmozy3PKa4mifkw7fdYf1kKNRDOzyWc7H68QeOyhqJqESh6chomT0\\\\\\\/eTnZhJXdQkPkH7IqUKLURT1gJ+Mk2IE6JSDokFEg9rVWO45V1m5XgTAL+Og\\\\\\\/TPOpuuWUayNNPeFZRjDH\\\\\\\/3bZ9uf0kL4UpKNTmwL3qswuxvnvtDaABk8FpbRj915M+6GgmNk4oNKRPWhR4w58leMUEdGSf0HrkDBOV4nkzCrocqjEJ5gRlZ3NzpTRN8Lpn+o0fq3G3\\\\\\\/EbioZwJ6e6IyOXPJutwohRQ7RLU5ha45bA0cjyi6bV12uiP1nvsHkI++YyOWSz484PwjVYA8rrUDmnT\\\\\\\/cQzuSUWKLe2yJ5dAFvhDuLRNoDnf78ADYQcpr+5nHrO3gq3LFt1KO\\\\\\\/bJK2Ag8FhfWPrEZxdQVrKCgNujYg96CmxIoo4cl+ZRYJGpImLYUwhZ2BCevxsjst+Rx9n\\\\\\\/Bmf3MbCn5b8gANI\\\\\\\/7pyakQFCuCXTeeRHXIb+9hR2W2GrNS7csaLMDDvDU1o8R+xpzZq5uk3mnSJoS9H52pH8KtStAaxnoLVv\\\\\\\/xZWHrkneL7qdlQ3dNS29PTG3RNff8T1ETFy1Tb6eUHoWG1i2J2N+zxjPEXgbWUI8tJ7tmRS73NrXRKyh2qpogCDwwGuDzHlNBD0n5uTT5L\\\\\\\/CgoLwPHupkCmKWPdQYP1VjUP4GeFqyB8W9E2lmtfQPzPVjOcw2rNUfyreFsUrt\\\\\\\/q3dWFEifPX5EBfZ8owEyug0NJbikdrriqbdsJV16asJUJ64ZyvMcizJQqLVeoBq0g2yLt\\\\\\\/R30jBZRryOtLaCcyMQWWbHSTiWKzbJ711bMpS5ytMF1h\\\\\\\/OmDq9AVVBMHU4ZIQfcCKK+Z2usmOQ69lT2CHs2VvyDg0wprpzc67R1fW+vDy6Z6aOvUMpcJbPHYl9KcYX9o58Z0rj02XwPbCf5YBH1\\\\\\\/gtosUy9qWriMYexPvia+tfYqbUm00jVpzjqDGU0NGnz5HeWViPBjTt+\\\\\\\/Wfl1sNJJMTtC2k25Lh0DRabJN\\\\\\\/c4nuMVxR8ELm3IbyRtNJAxXTZ9EWru1BkAdRtugSnnudpnoScFwf3U1jdMLl8\\\\\\\/8wYMDFsSRC0wB2bIKXPWElNEzTGAadWIYvnXfdDrpeHD00kzw3munZJVor37RGA+DjGUOyFIM4GMZHlM9VlBzZfUC6ciMlO6hoEBBnmf4+Imp0D5iwvTnB0WPXN2TvPNBNSFJT+bAzzbPLrKWr9YbOye1Dz\\\\\\\/\\\\\\\/Ief2S48ALX0KHyIxj7tlaYmMj1eI3oFj++8U35gOD4DQLkR80czLiS4egvHCZ9otB\\\\\\\/iWQlPSJG3AZHrloc8OdeRpHuiuOtyd6eKfwp+GkzyJgOjUtJMm3K1h9t3jWoJMx4T6ibw4n0kall7qHxNIrCRmNFGgYf55r\\\\\\\/8gtvT4Kf+b9Y1zn1g8dCdXV94t5jh5WvT+Ei0G\\\\\\\/Zz2aE6JxM7G+aOSrJTDn3pwLc6AB1R7RhzG9lxY+mDRLfixr0AJGQ9mFHpiLi9AlL2izRWEyIopRpzH7jfeNxjfYcAwFmyh4qqstR5krSyZXNvyztickckDSs921IXOLXsGzSyZQQqdxA3COKTDABhooXWR5SXBPFEMaYCOlRe6iB1zGFFRguxo4iev3CcRUwfSZgRravz\\\\\\\/tAZN5FjuPXerpZEVcBhjZ9bMpkyIdx1m3aviFecDYk03ouTNVItzS\\\\\\\/9q8Xjes0xtvd82C5h7NdpQsCskyKkYT7sXSxCuS8sHV+7zbK\\\\\\\/2K31lYpplkmfMDvn9BEzS5MZcBEuQINdYdZ1NPfvUwLxNXUmWXYJO85C\\\\\\\/Si1GX6BMlVjMEOutmnhNaoJwaMHjCtq3gCmmdR\\\\\\\/zzBv0sLNtM9U9pI4odJbWqSzApdSjm2HCLXH4NY29y1B\\\\\\\/RWGa0C7w49iK8Z0kfNuWeMmn8zZfiVzaXOZnww7\\\\\\\/SJjwkuS4EaJDVIYGZ6KYCI2J77hqsrogIOm4ildiW6J7zJwLFssRx2Kxe927XIfgsGtHOYWm0x8K9hdHac8G0ChkdhVPFJhB6f9zz4V8eqxK\\\\\\\/R8teYR0\\\\\\\/RijhiFV0sO6iwMsprMahqPKYWFeparD42MpK0n3zMp5G9AR7YfP8HFHsTs+jdguzpriEdnpYBFBHXqGQ1PUNWsYFgP22hhB1HqvGm5xTcpcxgu9eFqtJqmDeS7vBaw4TGmp+wEK2qQSkqN0fOgB8hYVeKY\\\\\\\/bPNH59cudL77ko6+WmTRxbpHMp6cpOBfraPYfwODUZ2hdELDFGQ6UVVH4wSAl9J37XPiSG4lh0QpM1eI4\\\\\\\/8JVMnpEfXeYKgUUwqIyWP4d\\\\\\\/im2xVyYxhhKkVpJJT1A6hOFTQ8AezbI+Q33IJIZCa6zPrfr9ORFOYubwfdWG\\\\\\\/1XaLy2w6QPIWx65BT3J5NLgL6KniZzEX3uB0Qb+RYgZTDnxxh8zg3MoMTGzIJ8joxKfGX1C1e+u0A2m+W0H4JmiFhqMhxT6ili+Z4bMqA68\\\\\\\/jEXV71QyexQgfEMGSI1HBalmp0crDJZ3+l\\\\\\\/SbkH1sjTVFQiHibNjyXu5bRpQteuO7ZgAcsiqff9+UR7ZxuPanfaEbr+KSciacx2Kx3dUFIdheCY4OQRztUxF1A82oPHqHYiePLJcZz26RnUre6Y2N9ujAjnbsBuWcoXP9opzzSi2kEz2suoxMPkT4H0PPZk1x7kg4rP5plOqihpJIXTZ6m+pJiCpTQlqmJ6NFvjEUedY2ga9JnYRc8soNM2iz1\\\\\\\/MQIbj1iDUKqlb51Db0\\\\\\\/TUUDfE0UKcoVsixXgq8YEfaVasfH9AhEnASoFvKpeFVPhPR6MgbDCUeP+HFN6SkSHBYlVIGtWFFvPAgCo7YsQP82xSP+GEWDgp\\\\\\\/aa+1m3HjzD6w8n5XREeCVrRVSNp7JHZ97DLxePz6DefrXj9Lr1Kt6R+OIGRnjTMBpXzTxzwixQ\\\\\\\/Mu9hs7wQcEJkvZDCJQm6vKHCPrlZTt8NC1t+pwBxaOJ5TLjK4lpY\\\\\\\/kTn5YQPqploVH4mDezpcWPI4+0OX5LHjMrqErA5mGs538ABKV1\\\\\\\/wjdL6VKJ0SgvDfbiXw5iiSmA0Sauay0c2I0MrSGYIxvNkV\\\\\\\/ddb55NTpIOFa1by8g36q4L\\\\\\\/avOaXuuSnd6DAVK\\\\\\\/YaVo4HJsE8FXtopBOJhxaaPFuGKGa\\\\\\\/sDT\\\\\\\/nY+jfLSM\\\\\\\/j3I2ytUFGH7I6zlV8N8PSuo63DT1HCjs01LJpnrdaUZNE1f3RfOrXNz9rzFzrY4Vkk0BtEv9+od6J3VadKAJqXqGBOaTt+Sa0Ko3D9xC3r898JM+TFklx+SeIEH+cBx89pKChg3VO8eM++Sam7X1d4dcJOzkMVtm5UEigJmmz94cAK\\\\\\\/0IBwwmH3lUnqB6PfIr7z00QZYfAft3gBsCvW0RGUd0I0aPDGEwybcQJrMyS2I+XFHPznmwtPlpUfaog1SvbUTJf+PZvzqZ8rq1PBViJMsQrXxyiMa4wXO9qccam7\\\\\\\/E6K329+7HELgLOg5yRz1xGAolSv51BbBAYLMfPXP4TdaQbE+cq82wY08SbWdAgG0lLaKx7XtLE3QoNQW1tnsMiJp\\\\\\\/IEXkrgHVAOLCQt\\\\\\\/bmh4tOonUIGYlWm4fcumuSDrPXClK\\\\\\\/+I6YPY24pvY0lTSHKpTO\\\\\\\/L7+4\\\\\\\/tNEm+TaDd4JlXu17n3dtOFdVafZJm4H1+vdURGeTIZGCA9V5PixlTILAH8FvJkGOBWtknPGOgytJ9PrmEcuaJTYyQ5qYnmjS7ayHfH2vk5kX+8Csga8Yoa7KmV5MV0ImZTMkcmxPc\\\\\\\/fSQzYIf4Sg\\\\\\\/tkG5dheFPFkKSef\\\\\\\/gb7ofCP6ICNQZEqKuD3WfbSVLpTJ3iuSHuEuwzOB9V7F48jajtyUMyiehheik\\\\\\\/RhP8jmBj648Y98poAezfNDFev\\\\\\\/\\\\\\\/SaE9x2NEHlhes74FTMOIAiINdiPPWLLKPknLUQHxf+FoRufbernaxko4sVVkCDowd\\\\\\\/cwCFdJiEzMpIB9F6YL0G2XuwzMyxHT3MR1mUTktnJwUZPiMgB62v2LRn1GKuJchmQwwd0JJ3kiHy0UQfxvcMUllQGhMwF6hYM5WYkTOGKLqMdXlQCxzC+y4aITFb3mn+SUYetV8na4Xjk829aMtkVOnQQuQYfwTphCdxkUUtL45JeDocRyqIEC6hzHVXBZj6Gl4xYSvt8Lhjm\\\\\\\/\\\\\\\/5\\\\\\\/symL4BOVCAnr3xXA0Pco+OIvArvhDyxMTozaubHQAmaKJTNHQuU5l96nUBGGLWnIVKE4AWsfQPhcNTChd3R+I0IRYYzYVoFJa92HZjvgONDKxOlmgJ9FMcFHunUlvvuC1uci7DUrDMc2icN\\\\\\\/Cnr54AErECAo08lMIxIx4ZCXyzeiitWtBKFK32p2i2oQzyGHxLD\\\\\\\/6yRbMEMFVVTYvnZx8dMtnbmYIhumx0yBp4hy5xX+kFCVw766XsS3BywENGE9RKehz6GYziPa2l\\\\\\\/8Jy4dRSHRRGNtcJ5D4yOQL0RkA39Cr2mdyC9y5kJ\\\\\\\/0KIRtYVXA\\\\\\\/xlj6sO4WJYgEBRvsExGzUO9ROnOEDM0etZGmjlIpObukl1oO3ZtF7+vbuC89fFry+ylYbdSFXehUTkSrGfN+RrCmTEw\\\\\\\/h6IUGpAX1za8\\\\\\\/In5\\\\\\\/Kw5\\\\\\\/wa8jQOXglMgg6cgK0n1puHJVe\\\\\\\/h6QKo\\\\\\\/6QzEYT6pl\\\\\\\/VmTt+\\\\\\\/UkUmPp\\\\\\\/y+8gZoYVjjn\\\\\\\/2ZWWZe7NIrMrno2Fr3jVvIyv190\\\\\\\/74MR+A3V+32YDev9nV6TNS\\\\\\\/f4Hz3\\\\\\\/ooKsHhcYsKeC4wf6WOM4Eb0geCBi7dY5p7TWQ1\\\\\\\/PeYsbW\\\\\\\/3GUhB7cqWabl+3b3DMAfMPbyGb9Mk7swGgzxA1KvrvQ3Qds+wh3O\\\\\\\/bRvyIiQQmG8302A7lhMM44NNB56oB+nx0coJPE9+Eu1fjGf7ERHlt5sQ\\\\\\\/DVWmi1FslhLy2PGGudpw+ESZx\\\\\\\/cXeiO6xVxeToENMPBPhOAJMKPWn6raUJrxyKQeQuqRcKhJOwG0zwYAiiTSlE7Fa8OVY9e3zF6E7cZmT6RMtv7o8mv23IhnkkJjdzXriI39uRSqBn0cpxQ+I37QSHBBiL0ToQ9aDQerwMCbXgfRSGQGT1wtj7VZx0yl4uCnmdh5ASAbDU7BUPILoFtQDM6M9lYGUzSXUHYQwKKxejDTOqp1MVrvjRIVrKJl9j+TG2TLDZcRqfFdrfTIvEqlo\\\\\\\/KSC99BYuO84eS481xY26HPMfycmw+zKliU\\\\\\\/d6IGk5fDms+RQy+hY5QFbsBOPqgnDEEGDIenphkrQ5BQxs1oRA+K0zKRFt\\\\\\\/J2Z75ARtisbS6OkWF+DyQNQUbRyw4p8k34rzha7eH\\\\\\\/OA9WB8nxNuwH1\\\\\\\/pZoWSWM0tCFQ9cCcoGzPs5IuwwuT4TntZMLVU67lb6j8\\\\\\\/GCB9ySXG1ngOd2595uoKajjpN9BfP3M7tYAVDMFtt08EGur5+\\\\\\\/EmUu4JzPG7nfaaecNRPXuwjT7tkRXBb9dok0ClywUyWMyOempJxzzb6gsP0VJP8ozqTF1cJjXtnE\\\\\\\/oB1LVN1EOIz1MEC69RMNUCLgzSXt9S9nvCYFMSf4f1Y6BSF8IF7DQVl5CNfb5+yI3i8heuoiTPMenF8oGJ4CfyWOmpe1Q3K229NmzlScfCTyuLvqtyCkFH644qzLedt6qJWVuVx6xOwRCoFglIzjxZKeLO7dxm8GyZizXeWvubZaMOzqyeQvs+18Hm6DdQiprZ6NaNvR2jBTGvATndTpVIXdCT2Wfw3jSyEHsd0hYJAzdSO96nX8m4jmtjtTJtEV2lGCZotumfUlGMxkFkBoaP\\\\\\\/Zzuz9NYOJtVfol550ALGQZwOahRccvSWm1+lhvvz8pMpQ0c70HAqqgVIQEQHEEqyBweC1bVk1Ti3jQEruglL0Pa11yQgiZiJgP6gdBb1IprmjnvnvETvjDXcID6iOyD0gXhkmCkrTTA+9Iofk8rBppbtomexuitw9uByRW+u7WVouJWK0jHKcYCmuz2T70U9hdyQs6JRcsgXjc1qXXdJPsIkDVgoqbUTjLKe0Hod+1RTF1nfogJ3Ks8pqwQyS+3aX00vWZt3agy0ChHas6D7kDxdQxua2tk\\\\\\\/xfWFAuW6TNXiYDH4Kz1ns5hgVv0g1hqUvy7ALnYk7AAyF8EosPweJ+utVYz5RwlMBUns+iboBrLbKoTy7OG+W529i96FN0yRUnnjJCe3J2+vY4\\\\\\\/e\\\\\\\/7V150JiGiZL4WPWX5EdKX6xYPFofIdWnEm1xlwiTayXag2ZQ95O1y3Bh7s1XcnXFtJE+ThDa9KE+R3JuRRRzV5Ftz7ZmUJkvStVB8eFm9HvlmI7B513YVYAdrFc8O1QFCQ0c1v38juiiyPYk7kTA8Nl5KI3JJj8y9PEc22z44m5iWiqnr\\\\\\\/TEVyk+U5wu9u2SzSrIvY+BSy7vzhRfffHyN8qVL7m3RJ\\\\\\\/18duAg0jFN1vl1gYFMCkvPEUdxJp4KdINYEEHJB2GDPiI0yY\\\\\\\/RryUGCjP3a8HHPWzgZq4HW0w0lPk5CgSzjmsiZVSjVD4wdc4mRvEzD7zsOx8kFOSfbx\\\\\\\/mMf7+TkkPJeRfpXpNzfG8Eo7pnDoqbUOzWFdUP9ex9KuQs3hfNyjZApA0oPgKgGUirztIelP\\\\\\\/dIaAwVxfp\\\\\\\/\\\\\\\/0CTaEyXYxq1TyBoWDxWZpnKNAwjHEnva9ZwRcHIssee\\\\\\\/kUEHPRWjqvJIVynow82iRpY4aG57Mkhepp5GLrY2PcQjeszmf5IiDA2927MSJrq1jlW58QyTY8C93zsliOTUcEBv6t\\\\\\\/Nd+nWE5vVXK+XCnMdgKm\\\\\\\/u0TX5qy5feqy2WvX7\\\\\\\/siAkY\\\\\\\/t5wU944rYApoOrLzceAWfApridJE2LvQS\\\\\\\/YtRTb6\\\\\\\/KBPVrT6X\\\\\\\/Ok0GAYdjizdtT0\\\\\\\/wE2HrbigYjAh7r8KmmLV8QqhF9lhRY+oTMLRxxXqjSeNWWCwl3KWT1ITFb0G6f\\\\\\\/dokM\\\\\\\/OmoU0dhqoD\\\\\\\/FGy997rrTQqiTi5CFlvMJTNhXRuvxax2\\\\\\\/9+nZbqs93jbX0MauLeuBKpYyXAUpY+Z4STzQtGGeGgcHzghgeKAwNQPWYieSO9T3MA33UzY4gnXz9rNAF3zKqUlptgHPjFnPmpWXcScqyyKvv31iNelkENTJ5\\\\\\\/M7SSLC8wb8jIBODZJTb8FFLv7uA7ZYeH14cHWYQ7slNoYBzTwxICPjKJq+3J5pTfMywufl3ntZMVxphcTzRaBRS\\\\\\\/WSc1soxVdIuwW3ZHywVC012uq431tLWS\\\\\\\/TKXb9Tfj6uvJmlmCVsNwuIzqu9KAOVMx5+zp+ep6ydDgZIwOxuDPGpxRNUnIMj0Nf+kdbF7u7on1xuPV5K\\\\\\\/75MESltb0HP1reSi3NPYtOvMAfxKXX88uWd0ZWwQ0HfIW0pdHkfS1LM9vFDKHjQ3CAR8BqbXx3ylwdmQwAkR0oKkj7cysSTUGnm\\\\\\\/FJJ\\\\\\\/N1F9JZTYbH1NA2uBbir4dJp6i7Q9cWBennAc97Xsgpu6LyAoIjcWPkJVzYr8+p54SJULE7Y70J60DKzd8DbmhOCXq++0StLr6alcBJbpRNdnNWDG87mnYriQ8tEwLZYZwlxBrfQIKa\\\\\\\/QaAWHgaJjaY6zGSEZKFdbyQL10dYWbihueakTh\\\\\\\/ZbeVtyL6ZJJaD4U91jKcxCzbXOZXrnakb\\\\\\\/EU\\\\\\\/yKwInRXRShQnaSMXiq9E7bwEjb43CVlXpss0cUyzwdhqjSp1Kt2mzcSalLD1JWv4PZmxLdSFR6E+7XRdCOZls8Yx7fw5bdVuud67L7vog1Fi6fZY7rOLAIJ6uXOBCi1L0Q1PUl5VO+QMw6ZKOK6VqcmIcoZc6LJZpsvo94QGCWgxMuSZ6FthZuc7eUOffJX02\\\\\\\/ymEf9Kbb1O1e8tk+VLLpidilMe7Q8BmQyw+\\\\\\\/IBx2kh\\\\\\\/Y1rBOsBN\\\\\\\/mp4w8STBYSzc\\\\\\\/+bMXAVa+rQ\\\\\\\/k4aVbu6jb5yMBKKX4M2R6U\\\\\\\/x4ns97OU65EUM9i6jueLLbS+ZlwQRv5wPCYvf4sMNOKc4IfSYQWDCFZgSxyS0NG8pECJvovbPIXwTU98PXKnP\\\\\\\/wOocu6adrzc2vThn1yBa0Ghpcxu+uG8OmM4OIyZiBhletx2TblpUPilkMNsW\\\\\\\/qr4Fo28\\\\\\\/7\\\\\\\/jSHNy8tjObayO74R6KkFesXhNdDG8wK9\\\\\\\/Wq4+hv3ay2ddoNWbML0\\\\\\\/ZIpm9\\\\\\\/oG9SBPoqx2YbubISHQyBWSEi3PGCS4HJwm+sfuVAMZJNtZ9BE2zurQecdaBGuaBWLnPfYn5g6EyaiSNhIvxjvteB6hWDToHI4VdLisdEAIrvhQzjlj91ewfNNyDVx\\\\\\\/\\\\\\\/2c6bzIclyavxm15QoOal3Q1AveNgUOgULQGyy6N974svVmVLtyFRjlndS6hViNdNWTb2lTgnm96vV1n1PTI42kF94niMv3JsKB\\\\\\\/QE5PgDpD5\\\\\\\/P28\\\\\\\/H2HJy4m4lMgfFB4t4GEsSU\\\\\\\/wWc\\\\\\\/h0QxZDZCljey0SPYHnVXcE0RJSVlnkThg\\\\\\\/8hIzVp9UY+iqo5tinYtkPPRZlndpo4sMJ81wZ7aLwdLzdgjTpLIlutEb6z4KihfbGfurGupG\\\\\\\/iQcP9NDHYGaZxtfiVchpnIi917iar3AthfVJHMKGNJUR\\\\\\\/MUg73xctmB0X+AZJo\\\\\\\/HOXYhDbdy0hJtF7jDS4YmH\\\\\\\/tJfipXj4oMJzqwTpr8EYn1jbEKE+psKjmu2wt+TtDcswkU8gXD90fDKp6hLb72JF+rhooqBdh0SM1HuGHr4kVOcv2QMTWD0AEVUChcR1JSx+vBKSEyXRhOr1pac3ICQoFLB5EdNVWNJ2PWeybCGy1XGbdUWqxRB9w0yDLjGBpXnjzSTzMXki+8gO0qqQC59eXFBJromDoVv45GaVM5gg4fC753H83pUapA9yVsZKfq9cOheV2UYVzcvoMQPDAFtmGT2DQTnThEgatHxpUchftqk8lNb\\\\\\\/GAhcDwmepYAUYtUJ\\\\\\\/75wwN0gLI9AOymDkZGKEZOfvOsORcQW11k1NhyD5Gyqt41P4cGtNsqwfx4Hsl5zs1xr2XG36bCpUR1YcLEp6rxcATTPzxTJvllzL6SpGzUu79vXw4msdM8XVB9LPBA\\\\\\\/XW4O+TGYD7zf62PhOf3UIqyo0bVORciSJMUph02GfTOgv3cZKFhYPzMXP+vBEmUhuy8dqEqoArjmn5qRwh2elfgDJMiA9lLnUj+iRdbzTaSl7ntzsAsru1w\\\\\\\/IVykZBuJw16mJDmAfagLln66Qwym3pYkBiiVCslXIPrr8+Rwm9Xii9PEh\\\\\\\/L68oTolBpgIDirEd4+LgDwWOxAZiCdfKwFIp9sBdM5JGfAN7bWm9kQZztO4Dfcueg3RPqP4xWguvoPv7ohE0yQuTAY\\\\\\\/qnzeVrKYkikzjv8sGieDmyLsQI7SYO0\\\\\\\/+IKx0YneZ1THklnlWy\\\\\\\/QlMNM3cRyyEELk2\\\\\\\/uvI2mG8YD30G8lDuV+JleuL+\\\\\\\/sxCaJ1frpAA4EZM9fixA7ybPiacOtN3yIHrKUgA9eGBhDHtFzZ8YsCHoy\\\\\\\/RPnc\\\\\\\/CkIqersjbpdjN1Z7OVbRc7I5cbfk5yCphPHVQCbsm9YZR+oce5g8jqsUzZ\\\\\\\/B6woq3aSSywk8KBzgvkhFqQhJLQh09Sai5d9GFzB0ec+KYJx22fYmrgvbySOzAYpwB\\\\\\\/v\\\\\\\/db8jdSYdo4bfbk\\\\\\\/\\\\\\\/dYuDdLvTGdIUtjacrfQJvh3q\\\\\\\/Dp1HqLPYix3sz3rtpa7md+G9SqQQuq6Gu5BkygH8V2ycIt92GMmQY\\\\\\\/JQ0vvue+US7BHwlSvLyjcX2TxLMXNK6DPRQzfkZxIf\\\\\\\/H65KLINqJ7BqXrONzd1xOBBOWT01wKF0r\\\\\\\/Y6Qz4DVt0sGJltsDRX2+bfZ\\\\\\\/AFRwWZNsFuBaJfrhhZyADTeaMtEgPeV7+iNkkjLFJAdCH7gqon\\\\\\\/\\\\\\\/c5tdR39e\\\\\\\/L2f+\\\\\\\/5kD+UdEA7iGtupfaXDZAqPFSQSZHo5\\\\\\\/LUQgnf+jkWP9XSjR2o3CEisg0ePy8wSVUz6Kk96BorWfOsiNrpIZnfA9Ss2VWWseocgH0\\\\\\\/8dS2QWgQHnrd1io7KPdfHkWquF4J3yWuDDYcn0Ke8vaId+6o++pU\\\\\\\/cWuPyi8y7oOTxU4n+UwE\\\\\\\/JOhEbBMTrsTg6z9irw+YJnkvDBS8O\\\\\\\/ThNi0c2XfXOABzg7Fy9WIlK8wxGywCnAuYxSGahPUwJy0zXmEqQAMV4uOJUxnGlHafbM57QoWmuE6rmSMKWGmKHEM\\\\\\\/urnMsmWbVzvaEHJ+HBMoamoYrU3cnga9GRFt16h9p8OBvblOSbbtuPZYJWurlVMsVYOe1lC0SDqdgkQ0k3HmjcDIf31OwcuQ6J7suLqI4SKk+OpObrt3yXZmHn3+wSkQegABJNPrlXEcfxNTOTxG1UhZjGWPBiJSh283fcFRXZINUNkXTlA819EVAiCZk3l5AeFzqo2A+z\\\\\\\/jXWjKTG1Nk2ky4bfMNnQS3Hk+nk7B1DLCfTYKVAJeh7Bv3qmteK3Y2Volvfk2iPC56k08DVRU2vpf5NPpeQNVp5HH07ar7JVkBpb0KraWqurHKZ0D4xzeHH6zMUr9\\\\\\\/zZ8japQbnqLBzywLoToZbXsQUNWywudEUODvAoNVjX71hlthrdYl++XGswy5WkWP08wfJrX+JShf7u6BuwlesZUqW+KZGLkhIiraP01wpAyBkXS+KkXWzE8YxCfBNJN4vVamIg2BLK+3yld3zvLM0ylsw8IGrwuJd+aL594ULf0OfEGA9iPbjKDjo\\\\\\\/FBME4pko8VU1SNqM0jw\\\\\\\/AloQjDKzxqqHI45tThOLu+0MJpHYcchrKV9tVd9j4qJ1NHfPDcQ9b9tsOhZHoTEdGe4gH2Gflw+lRaYTmH2CZImwAQmtPm1sAeLLSVHfB8Xs2zUzSa0nv0oM59jvt+Ik7tKW7XjXj0d1bMP\\\\\\\/lJAsIYGCfSXs88GPvRUKbBRbNo1gb6+hqwFvolKH3xnZUpdEyB0\\\\\\\/sSoq5S28fkIKcD5atf\\\\\\\/wAQmNiZ55+RpuCVSs1GQAuW5Yc3\\\\\\\/lcDhteVm2ytVr21Drcp7PH2kYHKKeFg6LUtW0tDctPxNDLYreYaC6aCQBf8R06s\\\\\\\/0lUPPxWfRcCrSXzG3Lb\\\\\\\/1Y0PzShn+wudthoXV6027sEkPWE7q1U+XwJ089Xggv7KiMXvJK26cEP6lpdRLqiYgmYY\\\\\\\/IBk3iewogm89wMdaKgbhyeA83auem2iX45RtXEfLyQxrHhNLQJwnVFjESGdQoXRtZ6Dw1C8Xo0ZISrLcIEHR\\\\\\\/JBcFk7W1EvO6p3wDcw01IZa7i+ZLwCNJCkX1rvILUeSMtyiNnc+iYdrih97UWqz8P4meQKndI4aNhi3IfRV0uvYzAbX9PSzX1iwbIyelzk1KhacfK9ySKbqpW4XZjmASGQY5e9laRuspbXgsBphyAwgRmln5XaRFaxTv4Ki3sYDuxo6imJMXWYbqp4GhpjTl3kZ1cPXKGLHmhtn6R42GQb+lxK8yg74YsfpCt54Htio\\\\\\\/AD48x12EpgZ2Nvcs5Ba55luyAQFz5dPzHALtADXvkJ0GrjXJteQDLZ\\\\\\\/weYErEVna59rULaOcyJtPlNX70XvGmsc3yAW4rCCDf75cx9ieVkpYa8uDOJ9qVnNMN43dkjz0H2v0I3s+8PRT3vitwHt5MPL2f9j4PJZzAq6Dg7CkDGGW0Nmjy1AHLTvD7Mz3RDSv6cQCohe1Vd4HkGBsOmqhCoxxjX87vElpWQxClEfj3bEKRPPWE0cc5SZenrz4P1ythO4C7niuyaXm1XXnjv02\\\\\\\/NChL4L87\\\\\\\/9RbDCWiSA4sMHeKCgl3irJTQBZEFha2v+1quD+h0SThwkyOEeZgmghTEHsTMIl8upG3cdeHFNHDKX3PduB01W1\\\\\\\/23f0bNRBzw1Z3rCSu0mV9yO8QBX8bNh5Uc6JAW3se68sfeS6lc1fa3MdTKvG47ZNGDrMF4y5fCpS4dLL3Obg1XiBBZIIsmIa+E6zgT6C1FrQ\\\\\\\/d73UCDTd7gpeu+F8o19HrrWUbg7niEeDRwnApFt8OY\\\\\\\/kfDhEhdjLtmCc46sQ5yg2VtHGsT7qXOG9gJX57VRdvya4VDjg8h8lTeaSiU+8ztPfbC8xxr1j5NHSeCzlrDBD4gW2kIC3148stIwXqmQWJ68SserLcnR5CvatWR0liJigFWXsrOstqLzE2x15UUfqxOXVFNErBT1aBeSOSr70xFFZLc8FfMAq7Z5hCw4MwguQTsG0ZAGN0UJWouH5bAFSUtotmqNRZiSoPekqDuMLnmmkuPO4qGQSFUWOmqZx\\\\\\\/gpSUXn4SYSSCF4\\\\\\\/ugT+XyY2u5rWN\\\\\\\/dhBhnkH45LEsbo3EXiPFxnJfppqs3ihx3UDBJ68Ys3KqCpzKWiydHJc1fWCC2qyvz9oVmDoiNc4dShzPuIFqDhb3bmbuQOE2YdaIoKz+7wlx1EnI\\\\\\\/8JYuukYKEONwinso03hNNANAJ43vpYW8FKBQZjnGDhiAAvRnQ9ljODGvftcG4KZY1Ig\\\\\\\/vP8XiU9PPJcUz8++iUar9GY0CIyILFY0zQNjx6mhsuC8kqAzUatV6at9HMmWHcwHs7fQX+nP8Cfmv9mMe842tmFkrjk+UINOOYImXk+ftoRdY2Mc02iCtnNtCw5aHYU3t1UqtsGWGJqNAvB3D6DcZRDenwXn4Xj74Tr9qgaY2wqhhm0qRe54njA+\\\\\\\/zfovEIq87LqdNALnlArYw9SeV9jcnDKGyr2MA5voGHJ+KcYiCv1Rs6EHqDu+HTsOoPmo0h4AaPaEEQ+4Akgp7HZobSX4j9kcUfg7beOBirPE5\\\\\\\/hnGkGf5fU2c51aCYxxQ6tVO1ioI+qGZ+5W3wxY+AiZROmpEJao9KJA\\\\\\\/NlrwB92FXg6gzGVo7XXbxHN0gL+DuLHr\\\\\\\/CmaIcIRq7E9AesgPI3gwcM6Oa6sTPtkRWb7Hwh7JRJh2BPnlCQ38DWmiKn\\\\\\\/CZMW+JFfjfjmiOlPbVQrk+f5BU5b9YQf2BA+8S6FMhPaXrB3\\\\\\\/xwAtTYnQh3GgPEG2PT8Tdfgz0jsZcrMxYdcGa1P6dYZkSI9zbac8eJtx\\\\\\\/\\\\\\\/v6uCkF+giGOwVGE8XfeKqosH1T04Ps\\\\\\\/fda3L9qFtVWmVs55DQoL1TXtUp6v+d\\\\\\\/vTY00f4rxdoSwlv922jK9xsIllsYlTaclsqxLjhU6CSm7AlOhrfqp9zZREWOm1Z4dnN0Naw0d9YdJwL+KoUiW2gN3mK7Y7QD1olwjfgtXW1nG42SpPQO\\\\\\\/TBhScby0nV9ne\\\\\\\/JFoAFAXb0Ak7iqyIGMFp2y8jE2995bY4iH3G30KjcjAc+Pm13YnkLOC2Ip4RgJlr3OG2oDp0UVsvC1MgkYQaBr6+zlbqnbx51ey+2ACPkUYV5617M5o4NF4\\\\\\\/n4CYNhmITuK+Kj4cMCdndKfXot+Dt89ZopwlxSGxu25EXyAIJGz6xUw9u0a7NBC4T7t8I7lefx4WIYXIdh2PSSEMxP8C1K+P7Z5LGsHdNv4+YFe1s1Bk5Y5yudY+rXuFnDnzO+8aZx2ufBzJKixRH7IzEluTOursITs+eMgVDw1R844EEGJUDmn5q9MKSE0qyXllBwf8Pu2C28WkfCqf9YtHNflvVeRrLvy\\\\\\\/l8RLe42AbmHhw23gNgo8iU8ZiEMZicWeaosJ0ZGPWtl2Dn66IMetISKBXHusk33WyACLC4dbT7Ba83P0iG6n2jVSHC8pa9VHyxJl1zW6U\\\\\\\/N2g9MPpun6zBsP4djF42AEWAwK41839Mg0vWUUFPNPP1\\\\\\\/FBWk24cyZS1OlrzvwVVDODsi1\\\\\\\/1mDGHOh4Eaf\\\\\\\/E74hzfdSniuosV+4DTwuuvDj1NCyXPReo5lbikeOFrRzMLmUn\\\\\\\/jA0rsL7YEjfIHApgMFRSgvdHVEbkcteKKo0hAsntJX+1jkEaT9PHpEFHMCnATwChXmOBqKQBJCU+yxpJQzr3lHiVEn0Ycu9\\\\\\\/SZpjsS4O23TbsHCWkka0HCgChFigk31v8mNQPIhFeZb8NTd+FTpjcsu4r52HxTHBS2PaXcfEuh2CtYgM56RR55ZD+S0OF9Uj9yfdCoRCPVnDodTKjBS1nC1ptQgm4FsY+nLsq\\\\\\\/baRx5MQ9Q0xzlfDBmoNKrpNZOHAtMIh+5IAcJTseoSD1VrVzEtnPYg3Ftk0Y\\\\\\\/nMr4iJPWEZwttbtYkshzsB8LrsBaNw2EvOx3g1cLcR48NQt5N2TBjiZzyewcTFCVZlKOHnct\\\\\\\/N5YeRHodUJ+QpTq8XuNliqIwCCCz6r9+0kb9SGb4QQECtQnsZeIGeXuRIjH8iDnlU9WVlEuJ+T1Ge2JKCZekAz+TcMe47xIBJaIzt3rBTyw33i30D4pNJJodRUIZ031J1VRjBy2spHDkye4W8caadk36\\\\\\\/2OSqFFVdIq9Kqfw7be\\\\\\\/A15nPZlQMuoZlMUx26eBmTpDxXwz3b2QxAnsMtX2gO\\\\\\\/8wEdAcoYFzZXRR8IMvs7hdnJPEKq5BDBrVJTZs5+ghfxdoWJcZzg0TcaN4amYpj68M06Y8My\\\\\\\/2\\\\\\\/GAV5fHq\\\\\\\/TLarGEru7W1\\\\\\\/BiyaER5tDXFxETQfiHmOlvZa4eIltPj7gGvoSFZuDVUFPdH1cAv+C8yEd3AGiuu\\\\\\\/AvnrrDI9v11Zm0fVhNzOjDlI42o6+Ucogwh2w20RcWpgceeu3Gzb+Dim9f99uYdiwBCAo48vU40KyyvqAKJl9fiIhbrrQRRlT22MDQYx0j\\\\\\\/z8lWoKdL0o8lX4\\\\\\\/ChapTVRBTWpukcmVAalKZvgTxTKEndS5cIVnzlj+H8rgu655qVB9DpqKZJx5KKbn0uA6s5WXBRmwNUgw4ttI3Q1MJKJ8D0\\\\\\\/cineogiXbnI8Iq9lfSgJvWNaYnR1b4WhE+qNT9DTaUnoUUtsQ+7YSp21ZscedOh2fq7gDyFJ0ab8I5xdcBlMP0MbYbqLLIqKFQbsdyitB6+jPxdcYQnWGsGteNZ2GxYVYkiKM7JyJGrTaqwetW3dkH+F17JTjc1fQwe0CW8jPGtcVwECPYQ6XquKwbAoyF7IlQEh8EsgZe+gOxZoMLRLyAiAFgP7Yx2K0gQFlYlVKThpfOI8cXTbNWvN8\\\\\\\/sB7atxHuI7lsdrcl6NfkeYAoyrqbqF6Hnm4gWgRsGG5\\\\\\\/OPmc8dSM7a5I89lbwO2QCrYKSrwY6OB\\\\\\\/LAFV4YuUiCMeUxQdgdPdNZOExPYD1guuh3cc7h\\\\\\\/EXZTVy62DPlBJ1qMA+3EvEdze5NCHTihePiZXO\\\\\\\/m9dxF5A75i9+So8C+ASNJTQuuSWQw\\\\\\\/YOrTL3JIFizvegsvwGy7xY46di8BR+I6t5AQhnazrWZErtKqhanhb6V4UZM8DQYCktWrMgFQkcqAAjkeB0e8pVFyhXkyvhhf8XN+fzMCs0RXxANmtMSCfnJ8QCqK4zfrhCHiqteOCdboveAQjBVrl7kH0KoPOlKuEHJKXwQwrgu3gS4+juwbemrLlimCV5jMNNOfQ5gx1O0ngDeJ+Yjm1\\\\\\\/RbbiYkC2z9GkTH2A3wdDfj3JWXuvmuIRqUJRnOhjtixKIbwIn5XBhVUdTA4wH9g1Tr8Zm4Cteyv\\\\\\\/kE773Jh1D3xRPvMrRBdTZa0JSCBs0SRTK44vGIKPvsQzTBe2upR+utNTdCkPDIEOco+4z0yvmc0ezZht9eb7pi+EPTLhD5YC4cvN+FbytQNaqMpCfCUQHFjhDphKBYTAmdXm8fDydPgZ+X+41XbJmPCR+3yr2\\\\\\\/Y5GOF2MTq4fBhVTQKM\\\\\\\/5lsRV4Z7qynn2ccR87kceHtppuubO1Kf8olscWDvhU6ghFR\\\\\\\/4Re\\\\\\\/xVFZaRk5sB19gZQ1w183f81Vp0ULWEBWpionmPLsm+6w0+5SoUIXy5uGxJewX0GUpuxw7cJP8BeaC4lqVAdCbJ5pvCyy3TjocbzH1SOylcShAEb6LPNfKievQ5bGEY5qimmUw5HW9AkQ1vVMjI36FBb5g5ZXXSfZjAcpUdVb183nsCYDvetziH9H+F7IbBblSYQe7vEiyqN8l+73\\\\\\\/iApBZtXBFsd6ubD+2nTd+2DJd2fkgFyXHb4Nxff6GBqNawO8xe\\\\\\\/cMo+0l1CxYbcxtETvYDPuQfb\\\\\\\/SUZ1DN5KcUu9zcj6nrHnv6Y6FIS8Yb7Lqcnu\\\\\\\/aRxVcpjOkAd+grUMt2NpAMoDbV5yJPL8Z+PU3psp6ZbLs\\\\\\\/aFyOEBrxuX\\\\\\\/8+72AkaeBGw+jrLEo4UshexkVphIkSqwe6RNQgx4VKElGUTTFC5WFCT+0ZWcSmPk07tU432N1x3XxHnmFfvSrCOEVhkloEsJQf9ok20CjGsv0\\\\\\\/Wsx5FL3QDJTARj3vWxjOSU8xyahQgdas9kR4PUI34s+Lu54FWB9z88NQ0NzoNf4Rj9\\\\\\\/WHDFp4CMNQJzBi9a2zcC9InPmSZLT6bJ7eWYh0i3RGurZ1kP5TKWe9AO0ns7OuwtePJ0CvdxuWTdipXi4kVHnpJGt+zYwytx7hkiaF30hY\\\\\\\/E\\\\\\\/yy7oBxUh37QC7xWgWgcJXw\\\\\\\/U8dp8ZoMzJBXofR0hQGm1wcDdO6jzCsbVIsKugtO+w2Lk+WKJmfRSuQZHT3E5HnHifrlJdQSJ21\\\\\\\/uUy83Cd279keN2bHYJslEmasAFLH0B2wWnxEwT+WWWcy1SR6tNp+Qh5VErB4qbh1VJSmIffFlZW0koMFRPWyyU6aLlz3kXxD843vwuNPuCPhwpf3u0Z0wLNuC8qVwXm0Qhbe31Dq2lbjFTieIVUlY58g478uWhTweNvD1w5UaCqJWHolwCVLdh3BfbMm4mEKwP7Xi0u\\\\\\\/V6W\\\\\\\/6F7eG\\\\\\\/q4zHPJvwBdYU5VpHN+VOoyamatHGuJ3lu+Be1e03Koq0sO4F\\\\\\\/WScLpBIF725tMwhMRTn4otvGChkkc3XH7GZx0SSC00MrCgVS8gv1AeJZaPAHv2wJIi6H8q8YN5ChJErITv1epoAFJYVL8dYoasdTIUAmfpOxzgj6khcCeHiI2f13JMGaS4I8Rs9SPcRYVOEEvAewBMJFdDKUqrKsPQxGFZ6pwmRyCqyF44J6fvYijs6l4j\\\\\\\/SI1PoSVie6a09CM8dYsOf8Hutq+LCUQyUQG2ux7wgsQDJsqqR\\\\\\\/bB9uqYm090OQFpTsVWGtBDcb1\\\\\\\/3h40OP20gZOZnsRlhKAkQUwOS8xrb2+0mxXsN1bbl8G8EygsvBOvf4TserlDeAf0uIIMhp7O172fgAaXc66V2+RyZBgmQIpjkuAnsc2x+Ef+5e0uCeSUZnAF+N8L+NkESJbvVXE\\\\\\\/GQG8js3dENa1iSyPSh+nKnQdKA52oky0S2mgtZWqiDja203n9kUz0L4eRRk+ygRiIuvOZSIkahdtv7T+cFVrJhV8Q\\\\\\\/LuMssswUI7ikVyBBT5yk2KcSsGo+bUZcwezhWwx9zv4xZMqEnj6DdqNgrFvI5vJj6VXY5aNs5F85VKglg2eGokJNWHY93w\\\\\\\/C68uciUUWDVTI1IFnl\\\\\\\/pZTk5OLDyrstOxqK1ZDvuEniPUhk+U80NZaZHNEovV3\\\\\\\/jm1tRMCN8loZiIbPXi02D8Fq71gjO0wq2roFGPtakLeY6+vL9xUcerc+egjo8VGHqywQad6rpIJRbDtL4QDPi8ni2Mxn4\\\\\\\/ydhrxeCJa6MxqOcfe2iJOMgNx9OfGPz+WyOUr8+KmlSBx\\\\\\\/JCP2qTpelPl\\\\\\\/McT8qdK1FVXQ4HYgAEvq5hae3jlbH0UeEKCDQCScdU3ZvP7pRY8wKabiXZfY5XBWxeZlck5rwdWldfs\\\\\\\/ocnNbYQW7oA6fbKsuQVhE6lkH+1fGW11xkgQ749LNwDIexqpRKco7\\\\\\\/g+2XABEkyhmLtkEbG9HQEcxUypXI+xumr\\\\\\\/ohy5VRNc0TyRHtgguvucOQBPJqkTwhtb0VAVAhDIt1fgN9XzqR8ow8apyfybqVWLJVL\\\\\\\/Zb91O1cZlfJ5X\\\\\\\/ORTVv3xQpSdjKxFUihFwD0W8+b19YIoVqpxlAf1KQWy9gJIB5rAIPd\\\\\\\/gALdVPTcLofxL23Ac4QJa4I7bjQQO1yYHeA2EP6sKSORrywb1Uk\\\\\\\/R254Z2CpqZIh2BQGaE9cfFnB7ISwQLhzjomyQ1xmHLklIZF8DUbjDJRQGqAxo42X2MPoS7VhJpgP1Acprl2ED6IaQrCwZOg+Wsy\\\\\\\/gj8k169IFGdKoj9AseJJp+e13NZwJx+IRShXbmMzxMAhKmsF7p798l3e7OLfiSj\\\\\\\/m8M9hjKRkN5D\\\\\\\/gXFLPsFiqcIZvPm68zEGkNT0QUtCIdFQ0jCbGpHeDKmorxjoNFIjCqAXn86SeVsBTZ6OVyN35SOz0cji\\\\\\\/84GHbgcyzyursnpuChGdGdtKb0KpuK+MHEKIiOoZKG4r1aTwdkUrZa1yvHKdbAiMlJlveZxL6z875XJnsqgf33bG6K++\\\\\\\/XSEsymUXLnKe2iCScDze7GVNnplA9+nL1Z21DarqRO+cviEuIyliIm6J09RQlsudDOD0sb2pBBlDiDMeZG8iJ6GsUxBNetVtaJTkt9BNanJqRs2mPAecAZwS4QrxZ\\\\\\\/vO8V3Lx9FwQgNIyi8FrrtFDq2rh6exJDh3m71f1uV+cmJi8HsDQIocrG7jHASWZeRGoVmXAnSqIPlZ4w+J3RAKwgBTtO0lwG+FJLxuAtUDygYSRdh46zJZsvA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022m9n2BICxdY2+jamSEIjJpw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a329e1c594032c42d2b8bb3453f8019c5c86a128d69d60338fba28c4fdf834c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+hSA4P5tRsldeLo0j97bjTNuimz4oDxeq9nDDv3+7O6QMjCqwDvrVT4pUV88gQbTaY7zGknSDbCe6AR2iNxsBR4XhNybW2yWibsndtRb8u5KL9wxtFjLcdk62uQ3SdFFYndZ9UnLg6\\\\\\\/\\\\\\\/A9En6sn6c4BaqEwBm4rW2lWM2PIJcJL0rD1lwI6bSO9J\\\\\\\/YuLGX8Jj\\\\\\\/+h3HmQLl59Bcy8TqLNPO59gwd+BvCq4s2JcjSIobUwH4j00RqVDjWC6CvySHRuC1CdqlDNJu2e1carSLHYkL2JJJVGGWJv2GpjXyTVhYOYVIH98bVQ6hsfurHDWsokfcYcYWNouqU3NuDnZUnuzJfZsGJGT1gyy9\\\\\\\/mNLFOjMB82k+VaVs2IrE9SLSdJ1JwqeFJMwNjiDZjSIovPmvZTXoMRErPb7R9zmbf5rl\\\\\\\/FnJL79+ZOgJvU8Ig4kFm5hQ\\\\\\\/g4oP1\\\\\\\/iynsftPwg44xJvpx5qvk+YivVXyGZH4KFpTariD+5Xr+L+0cYlw3by5K8+L+aG6EAey1qRR8glrAbtBRPiKgXM4hq8rNf7HfKh6bAy5\\\\\\\/XidyTkEwvts3dyLRnmu\\\\\\\/C2G8dfVoPK7GEcFi1gR7kK745XdJIcor7FYYFCQbea1lhbgakDhFPWVwzoSq0nfi1CC\\\\\\\/4oxHJXJa2fBcNtVav0WXjXzFgdzwiq+RVGEqD527dAHQTHuBKhzZVWbaGrTvGn6M2qBKjYlIHCOHSsBlV9fjl0DYDVkrB30U9IFnWh6bWZrHXf5+kD1Y+CU6XieA5etlh0Lj4d\\\\\\\/ZzR7kiFSyhL25NUI92cMmRqMNlicNxzTiXrhL0BBAWtNQ\\\\\\\/zqJFVsGAXgEtQdcLLosH1gIHgbjnwgW25wPmufsFMD17Dek1bTkoFYmovLvJRgzuHx2iIkzeg8FbNCt0RDjCt4xFGGmrBva3Tz8C2Ep5DvA4rN4V\\\\\\\/gnkAg1cwdJ3Wg68vVi2GudTtMQGR6wFsmJJmrrmZ8qmyNS3OPq+7u3H\\\\\\\/KK51CcnPwddT8LeJjLki82knxHWzL0+S8j0ZhZCabq8Ah7OBOq1N3W5qHr487qYWiKM63wMTbAPMmoFNWHrMbL4CWMmL\\\\\\\/HtSLggUUSz75pF1QGSSRQH2KdK7K7CpRrNq8GwSuH2hbpTIPaBd6bNW6LsX1+5cJsz5Hf1EasqvnG8eM8op8KzXV9AtFLKwjCGuAnbTP3uS662GyXpXyGigwGnMEFHq+fleWSppPSsV2oi\\\\\\\/WsuzxS73rzfcamSFoOUal7A7WHSTP4jh+53UMlYSOUhsd1iuDc+SMXWBeINF+g60Hkp19m1J\\\\\\\/WEFLf0kO2EK0ox+4DuXohG1s6yZ919x3UkEyhAgvQI5tdeqsgRk2hCCa3nPm+Idfmab5BseLpuPB\\\\\\\/bFoKNdNFBzLMK9kPorW6r7SBH\\\\\\\/dgFIaqjIU7tpAdM7Zs+FWMlVyYXW8JlnlpicZQgAyNih6bQb51EEBpzTfnYuB96XkFqGgI4pPk5+wsptlfNebA5YBnuyksCtk\\\\\\\/h85+7uEUcr0G6hbgyN9zyLh6v4POGw0\\\\\\\/z6GxaropcVRKb2YZgHCtQevu5tKPFTBmt7IPnR7ALO0L444RFkGUe3eOuS0K8TXDblObCW3Ei4JhMqSMwGo9B4021dw0VqByxI91ug5x\\\\\\\/m1DvGJlTdVvKXUVoNJ83XGJJ6257m1vJPEou0uRvpWhw5VEtjfeQPr0uatDledFJgXkMWiT\\\\\\\/FbKeP9t20CGZ0BhjremmabeXHd+7gw9dFZsKjlJjduySmE8TaEsUY3iwO9WrbXUAZP9\\\\\\\/\\\\\\\/ZNilPEzQzNdXQRCfwT81E+rDePtAfsmv\\\\\\\/KERjPQs3sZpz6cyQuJSRp0IS2ZlaxQtvcB0wXobMhibWJlhhGtM72ggg4o+1xCIZy6GnyRd9hgQALDPcd2pfqPI9rtJg0rw+THwMc\\\\\\\/uIqHywYxmnLvZlMJpUKTktV9dFXNOtnsrBldU+09f2NKD5LF\\\\\\\/AKzzjkRuHB\\\\\\\/UK3UD8kWPEnIp3Ap6QAEOqA75Jg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022fwPHJmdER0xUmMbrE46TTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b850bd70fe89f1e3c354bf5da8c7c2c72fc7fd0258822f815afffb1e2ecdc800\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493671 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2183" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LS3bADX1bGzF9dkRad422HpLxha5MXLxgNChsoepJyE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2183" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cyT-Wq_L0Nwjr5VmqtVrC789hYI_ZJY7KkaCdhETvoE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2171" title="Quantitative Comparison of the Anterior-Posterior Patterning System in the Embryos of Five Drosophila Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2195" title="Single-Copy Knock-In Loci for Defined Gene Expression in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493671" id="node493671" data-pisa="ggg;9/7/2183" data-pisa-master="ggg;g3.119.400117" data-apath="/ggg/9/7/2183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Synthetic Physical Interactions with the Yeast Centrosome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2479-180X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rowan S. M. Howell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2919-5601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Attila Csikász-Nagy</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1649-6816" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter H. Thorpe</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2183-2194;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400117 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493671" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rowan S. M. Howell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Francis Crick Institute, London, NW1 1AT UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rowan%2BS.%2BM.%2BHowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howell%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARowan%2BS.%2BM.%2BHowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2479-180X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rowan S. M. Howell</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Attila Csikász-Nagy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Information Technology and Bionics, Pázmány Péter Catholic University, Budapest, 1083 Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Attila%2BCsik%C3%A1sz-Nagy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Csik%C3%A1sz-Nagy%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAttila%2BCsik%25C3%25A1sz-Nagy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2919-5601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Attila Csikász-Nagy</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.thorpe@qmul.ac.uk" class="" data-icon-position="" data-hide-link-title="0">p.thorpe@qmul.ac.uk</a>
<a href="mailto:attila.csikasz-nagy@kcl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">attila.csikasz-nagy@kcl.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter H. Thorpe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Biological and Chemical Sciences, Queen Mary University, London, E1 4NS UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BH.%2BThorpe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thorpe%20PH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BH.%2BThorpe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1649-6816" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter H. Thorpe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.thorpe@qmul.ac.uk" class="" data-icon-position="" data-hide-link-title="0">p.thorpe@qmul.ac.uk</a>
<a href="mailto:attila.csikasz-nagy@kcl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">attila.csikasz-nagy@kcl.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2183" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493671" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493671/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2183.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493671" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493671/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2183.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493671" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493671/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2183.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493671" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493671/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The yeast centrosome or Spindle Pole Body (SPB) is an organelle situated in the nuclear membrane, where it nucleates spindle microtubules and acts as a signaling hub. Various studies have explored the effects of forcing individual proteins to interact with the yeast SPB, however no systematic study has been performed. We used synthetic physical interactions to detect proteins that inhibit growth when forced to associate with the SPB. We found the SPB to be especially sensitive to relocalization, necessitating a novel data analysis approach. This novel analysis of SPI screening data shows that regions of the cell are locally more sensitive to forced relocalization than previously thought. Furthermore, we found a set of associations that result in elevated SPB number and, in some cases, multi-polar spindles. Since hyper-proliferation of centrosomes is a hallmark of cancer cells, these associations point the way for the use of yeast models in the study of spindle formation and chromosome segregation in cancer.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/centrosome" class="hw-term hw-article-keyword hw-article-keyword-centrosome" rel="nofollow">centrosome</a></li><li class="kwd"><a href="/keyword/localization" class="hw-term hw-article-keyword hw-article-keyword-localization" rel="nofollow">localization</a></li><li class="kwd"><a href="/keyword/spindle-pole-body" class="hw-term hw-article-keyword hw-article-keyword-spindle-pole-body" rel="nofollow">Spindle Pole Body</a></li><li class="kwd"><a href="/keyword/empirical-bayes" class="hw-term hw-article-keyword hw-article-keyword-empirical-bayes" rel="nofollow">empirical Bayes</a></li></ul><p id="p-3">Microtubule Organizing Centres (MTOCs) are critical to the process of chromosome segregation in eukaryotes. Abnormalities in the structure or number of centrosomes, the metazoan MTOC, are strongly associated with human cancer (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nigg 2006</a>). In <em>S. cerevisiae</em>, the MTOC is the Spindle Pole Body (SPB). The SPB differs from metazoan centrosomes in its structure and in that it remains embedded in the nuclear membrane throughout the closed mitosis of yeast (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fu <em>et al.</em> 2015</a>). However, despite these differences, there is significant conservation between yeast SPB proteins and human centrosomal proteins (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jaspersen and Winey 2004</a>), making the yeast SPB a relevant model of MTOCs.</p><p id="p-4">Beyond their roles in microtubule nucleation, SPBs are thought to act as signaling hubs, with recruitment to the SPB a key step in regulation of certain signaling pathways (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Fu <em>et al.</em> 2015</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arquint <em>et al.</em> 2014</a>). Various studies have used the strong interaction between GFP and GFP-Binding Protein (GBP) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rothbauer <em>et al.</em> 2006</a>), to test the effect of forced localization to the SPB, for example (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gryaznova <em>et al.</em> 2016</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Caydasi <em>et al.</em> 2017</a>). However, no systematic study of forced relocalization to the SPB has been performed. We used the Synthetic Physical Interaction (SPI) methodology (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ólafsson and Thorpe 2015</a>) to test recruitment of more than <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-1.gif"/></noscript></span></span> proteins to five locations around the SPB.</p><p id="p-5">Proteome-wide SPI screens have been used in the past to probe the regulation of the kinetochore (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Ólafsson and Thorpe 2015</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">2016</a>) and a set of 23 SPI screens was used to generate a cell-wide map of proteins sensitive to relocalization (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> 2016</a>). Relative to the screens of Berry <em>et al.</em> our analysis shows that the SPB is particularly sensitive to forcible relocalization. As a result, we found that standard methods for analysis of genome-wide screens based on Z-transformations were unsuitable to analyze these screens. <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Efron (2004)</a> suggested an approach to multiple hypothesis testing, such as genome-wide screens, based around an empirically derived null distribution which he treated as a component of a finite mixture model to calculate significance of measured results. This empirical Bayes approach is widely used to analyze gene expression data, where it is used to classify the significance of correlations between genes, see for example (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schafer and Strimmer 2005</a>). We adapted this approach to the analysis of the 23 SPI screens conducted by <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a> as well as the five SPB SPI screens. We fit bimodal normal mixture models to our data, according to an approach outlined in (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fraley and Raftery 2002</a>). This approach overcomes the limitations of Z-transformations as well as providing a parameterisation to compare screens and tools to predict the rate of validation. <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a> concluded that only <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-2.gif"/></noscript></span></span> of proteins were sensitive to forcible localization, our analysis suggests that locally this may vary, with some regions, such as the SPB, far more sensitive than other parts.</p><p id="p-6">Global analysis of the SPI data shows that the SPB is sensitive to forced interactions with a variety of proteins, including proteins involved in microtubule nucleation, protein transport, lipid biosynthesis and the cell cycle. Proteins that caused growth defects when recruited to the SPB originated from the nucleus and chromosomes as well as membranes, especially the endoplasmic reticulum. Although we found significant variation in individual results between regions of the SPB, the data from the SPB SPI screens was found to be more similar to each other than to the screens with other parts of the cell. A particularly interesting finding is that tethering nuclear pore proteins to the SPB causes growth defects. A growing body of work (reviewed in <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jaspersen and Ghosh (2012)</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rüthnick and Schiebel (2018)</a>), argues that the process of SPB duplication and insertion into the nuclear membrane relies on machinery usually associated with the nuclear pore. We investigated whether these forced interactions between Spc42 and nuclear pore proteins resulted in abnormal SPB number. We found that forced recruitment of several nuclear pore proteins, as well as the SPIN (SPB Insertion Network) and some currently-unclassified membrane proteins showed evidence of SPB overduplication. The current model for SPB duplication is that it is tied to the cell division cycle through sequential activation by Cdc14 and CDK (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Rüthnick and Schiebel 2018</a>). Our work suggests that forced localization of proteins to the SPB can decouple the process of SPB duplication from the cell cycle, a finding that may suggest refinement of the current model or that SPB duplication can occur via alternative pathways. The development of yeast strains that reliably produce multi-polar spindles may facilitate research into these structures which are known to occur in cancer cells which exhibit high variability in centrosome number. (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Nigg 2006</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and methods</h3><p id="p-7">Yeast was cultured in standard growth media with <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-3.gif"/></noscript></span></span> (w/v) glucose unless otherwise stated. GFP strains in this study are from a library derived from BY4741 (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-4.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-5.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-6.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-7.gif"/></noscript></span></span>) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huh <em>et al.</em> 2003</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tkach <em>et al.</em> 2012</a>). For each screen we constructed plasmids expressing an SPB-GBP-RFP construct and the SPB protein alone from the <em>CUP1</em> promoter; all plasmids are derived from pWJ1512 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Reid <em>et al.</em> 2011</a>) and are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Plasmids were constructed by gap repair either through <em>in vivo</em> recombination or the NEBuilder plasmid assembly tool (New England Biolabs, USA). Linear products were created by PCR with primers from Sigma Life Science and Q5 Polymerase (New England Biolabs, USA). The sequence of azurite fluorescent protein (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mena <em>et al.</em> 2006</a>) was synthesized by GeneArt (ThermoFisher Scientific, UK). To visualize spindle morphology, Bbp1-, Nup170-, Nup133- and YJL021C-YFP mTurq-TUB1 strain were constructed using a linear construct to convert GFP to YFP and a linearized plasmid, “pHIS3p:mTurquoise2-Tub1+3’UTR::URA3”, which was a gift from Wei-Lih Lee (Addgene plasmid # 50635; <a href="http://n2t.net/addgene:50635">http://n2t.net/addgene:50635</a>; RRID:Addgene_50635) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Markus <em>et al.</em> 2015</a>). The sequence of all plasmids was verified by Sanger sequencing (Genomics Equipment Park STP, Francis Crick Institute and Genewiz, UK).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493580/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493580/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Table of plasmids</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>SPI screening</h3><p id="p-8">The SPI screening process is described in detail in <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a> and in <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ólafsson and Thorpe (2018)</a>. A library of GFP strains is transformed with a plasmid expressing either a fusion of a protein of interest with GBP or a control, through a mating-based method known as Selective Ploidy Ablation (SPA) (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Reid <em>et al.</em> 2011</a>). The plates are repeatedly copied and grown on successive rounds of selection media until a library of haploid GFP strains with the plasmid is produced. This library is assayed for colony size, giving a readout for the fitness of a given binary fusion between the GFP strain and protein of interest. Plates were scanned on a desktop flatbed scanner (Epson V750 Pro, Seiko Epson Corporation, Japan) at a resolution of 300 dpi. All plates were grown at <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-8.gif"/></noscript></span></span>C. All copying of yeast colonies was performed on a Rotor robot (Singer Instruments, UK).</p></div><div id="sec-4" class="subsection"><h3>Quantitative analysis of high-throughput yeast growth</h3><p id="p-9">Scanned images were analyzed computationally to extract measurements of the colony sizes. The online tool ScreenMill (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dittmar <em>et al.</em> 2010</a>) was used to perform normalization and calculate Log Growth Ratios (LGRs) and Z-scores by comparison of experimental and control colony sizes. Two controls were used (plasmids expressing GBP or the SPB protein alone) but, as in previous studies, we found strong agreement between the two and we used an average of the two values. In some cases, the library contained multiple copies of the same GFP strain, in these cases data were aggregated by averaging. In the proteome-wide screens plates were normalized to the plate median while in the validation screens GFP-free controls were used for normalization. LGRs were further normalized using a spatial smoothing algorithm as described in <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a>. Bimodal normal mixture models were fitted to the smoothed LGR data using the “Mclust” package (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Scrucca <em>et al.</em> 2016</a>). Further details can be found in the supplementary materials. R scripts for data formatting and analysis are freely available at <a href="https://github.com/RowanHowell/data-analysis">https://github.com/RowanHowell/data-analysis</a>.</p></div><div id="sec-5" class="subsection"><h3>Bioinformatics</h3><p id="p-10">The GOrilla website (cbl-gorilla.cs.technion.ac.il (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eden <em>et al.</em> 2009</a>)) was used to perform all gene ontology enrichment analysis. The “cluster” program (version 3.0) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eisen <em>et al.</em> 1998</a>) was used to perform hierarchical clustering of the SPI data; Java Treeview (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Saldanha 2004</a>) was used to visualize the results. Clustering was performed using the correlation of the LGRs, minimizing the average linkage of the clusters. Spatial Analysis of Functional Enrichment (SAFE) analysis of SPB SPI hits was performed using the Cell Map website (<a href="http://thecellmap.org">thecellmap.org</a> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Usaj <em>et al.</em> 2017</a>)). The Venn diagram in Supplementary Figure S1 was created at bioinformatics.psb.ugent.be/webtools/Venn/.</p></div><div id="sec-6" class="subsection"><h3>Fluorescence microscopy</h3><p id="p-11">To examine localization of the SPB-GBP-RFP construct, and GFP-tagged proteins, cells were grown shaking overnight at <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-9.gif"/></noscript></span></span>C in -leucine media, supplemented with additional adenine. They were then imaged with a Zeiss Axioimager Z2 microscope (Carl Zeiss AG, Germany), with a 63x 1.4NA oil immersion lens and using a Zeiss Colibri LED illumination system (RFP = 590 nm, YFP = 505 nm, GFP = 470 nm, mTurq = 445 nm, azurite = 385nm). Bright field images were obtained and visualized using differential interference contrast (DIC) prisms. Images were captured using a Hamamatsu Flash 4 Lte. CMOS camera containing a FL-400 sensor with 6.5 mm pixels, binned 2x2. Images were prepared with Volocity software (Perkin Elmer Inc., USA). To screen for abnormal numbers of foci in strains containing Spc42-GBP-RFP and GFP-tagged proteins, a plate of strains was prepared using the SPA methodology described above. In this assay, the Spc42-GBP-RFP plasmid (pHT11) was accompanied by a plasmid expressing Htb2-Azurite (pHT 706) with nourseothricin (<em>NAT</em>) selection. The Htb2-Azurite construct allowed for identification of the nucleus. On the same day, cells were picked from the plate and suspended in water and them imaged as described above. Dead cells were identified by a high level of dispersed fluorescence, and were excluded, as were cells with no visible fluorescence in the RFP channel.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-12">Plasmid details are shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, all strains are available on request. File S1 contains data from the colocalization assay. File S2 contains all LGRs from SPI screens used in this study. File S3 contains all LGRs from validation of SPB SPI screens. File S4 contains a list of hits from all screens used in this study, based on either the empirical Bayes method or Z-score where appropriate. File S5 contains full results of GO enrichment analysis. Table S1 contains a list of all empirical Bayes parameters and cutoffs used for screens in this study. Table S2 contains a comparison of empirical Bayes parameters fitted over entire GFP library and restricted to GB library for relevant screens used in this study. Table S3 contains the data from the supernumerary RFP foci screen. Table S4 contains the data for quantification of the supernumerary RFP foci phenotype. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8100749">https://doi.org/10.25387/g3.8100749</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Synthetic Physical Interaction screens with the SPB</h3><p id="p-13">The budding yeast SPB is embedded in the nuclear membrane with one face, known as the inner plaque, directed into the nucleus and the other, known as the outer plaque, facing into the cytoplasm (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jaspersen and Winey 2004</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). A central plaque links the inner and outer faces of the SPB and connects to a structure known as the half-bridge, which is involved in SPB duplication. In order to understand the effect of localizing proteins to different parts of the SPB, we performed genome-wide Synthetic Physical Interaction screens with multiple target proteins: Nud1, Spc42, Spc72 and Spc110 N-termini and Spc110 C-terminus GBP fusions. Nud1 and Spc72 are situated in the outer plaque of the SPB; the N-terminus of Spc110 lies on the inner plaque while its C-terminus is located, with Spc42, in the central plaque (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Jaspersen and Winey 2004</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Structure of the SPB showing location of GBP tags used. B: Colocalization of query and target proteins in the Nud1, Spc42, Spc110C and Spc110N screens. A selection of 48 GFP strains was chosen to represent different regions of the cell and a mixture of strong and weak growth phenotypes. Each strain was judged to have either colocalization of GFP and RFP at SPB foci or not. In some cases no live cells were imaged due to slow growth, these strains were removed from analysis. The  colocalization observed in each screen is consistent with previous studies (Berry et al. 2016). C: Validation of SPB SPI screens. For each GBP construct, 240 GFP strains were chosen and rescreened at higher density. These strains were considered to be validated hits if the growth defect measured was greater than a cutoff determined by GFP-free controls. In each screen, we found that strains with Z-scores less than 2 met the criteria for validation, suggesting the cutoff at a Z-score of 2 was overly restrictive. D: Ordered LGRs for each of the 5 SPB screens and 23 screens from Berry et al. (2016), this graph shows only strains present in the subset of the GFP library used in the SPB screens. The left hand side of the graph has left-justified values while the right-hand side shows the right-justified values, this is because the region closest to the edges is the most informative. The SPB screens, shown in color, are considerably separated from the screens performed with other regions of the cell." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1175178226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;A: Structure of the SPB showing location of GBP tags used. B: Colocalization of query and target proteins in the Nud1, Spc42, Spc110C and Spc110N screens. A selection of 48 GFP strains was chosen to represent different regions of the cell and a mixture of strong and weak growth phenotypes. Each strain was judged to have either colocalization of GFP and RFP at SPB foci or not. In some cases no live cells were imaged due to slow growth, these strains were removed from analysis. The &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; colocalization observed in each screen is consistent with previous studies (Berry &lt;em&gt;et al.&lt;/em&gt; 2016). C: Validation of SPB SPI screens. For each GBP construct, 240 GFP strains were chosen and rescreened at higher density. These strains were considered to be validated hits if the growth defect measured was greater than a cutoff determined by GFP-free controls. In each screen, we found that strains with Z-scores less than 2 met the criteria for validation, suggesting the cutoff at a Z-score of 2 was overly restrictive. D: Ordered LGRs for each of the 5 SPB screens and 23 screens from Berry &lt;em&gt;et al.&lt;/em&gt; (2016), this graph shows only strains present in the subset of the GFP library used in the SPB screens. The left hand side of the graph has left-justified values while the right-hand side shows the right-justified values, this is because the region closest to the edges is the most informative. The SPB screens, shown in color, are considerably separated from the screens performed with other regions of the cell.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2183/F1.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2183/F1.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493589" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">A: Structure of the SPB showing location of GBP tags used. B: Colocalization of query and target proteins in the Nud1, Spc42, Spc110C and Spc110N screens. A selection of 48 GFP strains was chosen to represent different regions of the cell and a mixture of strong and weak growth phenotypes. Each strain was judged to have either colocalization of GFP and RFP at SPB foci or not. In some cases no live cells were imaged due to slow growth, these strains were removed from analysis. The <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F1/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F1/embed/mml-math-10.gif"/></noscript></span></span> colocalization observed in each screen is consistent with previous studies (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> 2016</a>). C: Validation of SPB SPI screens. For each GBP construct, 240 GFP strains were chosen and rescreened at higher density. These strains were considered to be validated hits if the growth defect measured was greater than a cutoff determined by GFP-free controls. In each screen, we found that strains with Z-scores less than 2 met the criteria for validation, suggesting the cutoff at a Z-score of 2 was overly restrictive. D: Ordered LGRs for each of the 5 SPB screens and 23 screens from <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a>, this graph shows only strains present in the subset of the GFP library used in the SPB screens. The left hand side of the graph has left-justified values while the right-hand side shows the right-justified values, this is because the region closest to the edges is the most informative. The SPB screens, shown in color, are considerably separated from the screens performed with other regions of the cell.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">SPI screens aim to test the effects of forced relocalization of gene products across the genome (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Ólafsson and Thorpe 2015</a>). In each screen, a target gene tagged with GBP (GFP-Binding Protein) is introduced into a library of GFP strains (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Tkach <em>et al.</em> 2012</a>) to induce binary fusions between the target protein and the GFP-tagged query protein. Growth of colonies under these conditions is measured and an average LGR (Log Growth Ratio) between the experimental strain and two control strains is calculated, providing a measure of any growth defect caused by the artificial protein-protein interaction. Additionally, a Z-transformation is applied to assess the significance of the results. A Z-transformation assumes the data are normally distributed and uses the mean and variance of the data to transform each data point to a Z-score, which are distributed according to a normal distribution with a mean of 0 and a standard deviation of 1. This simplifies analysis, in Z-space the region <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-11.gif"/></noscript></span></span> represents the <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-12.gif"/></noscript></span></span> confidence interval, so a Z-score greater than 2 is indicates a significant deviation from the mean. Similar to genetic interactions, we say a forced association between proteins is a SPI only if this combination causes a significant growth defect.</p><p id="p-16">We predicted that, due to the structural integrity of the SPB, the GFP-tagged query proteins were more likely to be recruited to the GBP-tagged target protein than vice versa. We used fluorescence microscopy to analyze colocalization of GFP- and GBP-tagged proteins in 48 GFP strains chosen to represent a mixture of hits and non-hits as well as clearly-defined and distinct regions of the cell as well as regulators of mitosis. We found <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-13.gif"/></noscript></span></span> of strains viewed showed localization patterns consistent with recruitment of the query protein to the SPB (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, Supplementary File S1) in the Nud1, Spc42, Spc110C and Spc110N screens; a finding in keeping with the results of <a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a>. Genome-wide screens often have high rates of type I errors (false positives) so we validated a selection of strains with high or, in some cases, low, negative LGRs. Validation screens were performed with 16, rather than 4, replicates of each strain and “validation” of a result was defined by a LGR exceeding a threshold set by GFP-free controls. Each of the screens identified <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-14.gif"/></noscript></span></span> strains with Z-scores greater than 2 and we validated 240 strains for each screen. The remaining strains were chosen as those just below the Z-score of 2 cutoff and “growth enhancers” - strains with Z-score less than <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-15.gif"/></noscript></span></span>, these were found mainly in the Spc110C and Spc110N screens. The growth enhancers were found not to validate frequently in either screen, these strains are likely slow growing generally, which can lead to inaccurate LGRs (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). We were surprised to discover that almost all of the strains with Z-scores above 2 validated and many that lay below this cutoff validated as well (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>, Supplementary File S3). Furthermore, when we plotted the distribution of LGRs against LGRs from a dataset of 23 SPI screens (<a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> 2016</a>), we noticed that the SPB screens generally had more high-LGR strains than other screens (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>). We hypothesized that the SPB was particularly sensitive to forced localization and that these screens identified many true hits. However this was not reflected in the number of hits according to the Z-score. As the Z-transformation is based on the assumption that data are normally distributed, it will become inappropriate when the data deviates significantly from this distribution, as we would expect in the case of a screen with many hits. Therefore, we developed a novel statistical methodology to analyze significance in SPI screens.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Schematic of the mixture model analysis of SPI screen data for the Spc42 screen. The top panel shows scans of a single library plate with the plasmid expressing either Spc42-GBP-RFP (denoted Spc42-GBP) or Spc42 (the plate with the plasmid expressing GBP alone is not shown). The lower panel shows a histogram of LGRs in the screen, with two normal components of the mixture model shown in color. Five strains are highlighted to show the difference in colony size associated with different LGRs. Note that strains with low negative LGRs, such as that shown in orange are often the results of slow-growing GFP strains, which can register as having enhanced growth due to plate normalization and proportionally high levels of measurement error. B: Comparison of the bimodal normal mixture model and normal model of the Spc42 screen data, with the histogram of measured LGRs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1175178226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A: Schematic of the mixture model analysis of SPI screen data for the Spc42 screen. The top panel shows scans of a single library plate with the plasmid expressing either Spc42-GBP-RFP (denoted Spc42-GBP) or Spc42 (the plate with the plasmid expressing GBP alone is not shown). The lower panel shows a histogram of LGRs in the screen, with two normal components of the mixture model shown in color. Five strains are highlighted to show the difference in colony size associated with different LGRs. Note that strains with low negative LGRs, such as that shown in orange are often the results of slow-growing GFP strains, which can register as having enhanced growth due to plate normalization and proportionally high levels of measurement error. B: Comparison of the bimodal normal mixture model and normal model of the Spc42 screen data, with the histogram of measured LGRs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2183/F2.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2183/F2.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493577" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">A: Schematic of the mixture model analysis of SPI screen data for the Spc42 screen. The top panel shows scans of a single library plate with the plasmid expressing either Spc42-GBP-RFP (denoted Spc42-GBP) or Spc42 (the plate with the plasmid expressing GBP alone is not shown). The lower panel shows a histogram of LGRs in the screen, with two normal components of the mixture model shown in color. Five strains are highlighted to show the difference in colony size associated with different LGRs. Note that strains with low negative LGRs, such as that shown in orange are often the results of slow-growing GFP strains, which can register as having enhanced growth due to plate normalization and proportionally high levels of measurement error. B: Comparison of the bimodal normal mixture model and normal model of the Spc42 screen data, with the histogram of measured LGRs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Mixture models are an effective model for SPI screen data</h3><p id="p-18">Genome-wide screens, such as SPI screens, typically apply an experimental procedure to assign every gene in the genome a value. In yeast screens, such as SPI or yeast-two-hybrid screens, this measure often characterizes the growth of a colony. Analysis of these screens generally assumes that the distribution of colony sizes under equal conditions will follow a lognormal distribution, so that the logarithm of colony sizes is normally distributed. However, when performing a genome-wide screen, we expect some small but non-zero proportion of strains to have reduced fitness and grow more slowly. We hypothesized that in certain cases, where a significant number of genes are affected, screening data will not fit a normal distribution. In a previous study, <a id="xref-ref-6-10" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a> performed 23 SPI screens using GBP fusions in different compartments of the cell, in order to build up a map of protein localization sensitivity. We combined this dataset of screens with the SPB SPI data to assess the performance of Z-transformations in different proteome-wide screens.</p><p id="p-19">We found that the LGRs are not distributed according to a normal distribution (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We reasoned that, similarly to <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Efron (2004)</a>, we could take advantage of the assumption that the data contained two distinct categories, unaffected and affected by forced localization, to develop an improved statistical model of the data (Supplementary Methods). We used the “Mclust” package (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Scrucca <em>et al.</em> 2016</a>) to fit bimodal normal mixture models (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fraley and Raftery 2002</a>) to the SPI data (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). These mixture models matched the distribution of SPI data more successfully than unimodal normal distributions (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). We found that for 20 of the 28 screens the fitted mixture model matched our intuition of a “central” peak representing unaffected genes and a “hit” peak, shifted to the right representing genes affected by the forced interaction (Supplementary Table S1). The data for the remaining eight screens did not show well-defined hit peaks. An underlying assumption of our analysis is that the non-hits will be distributed according to a normal distribution, so in screens with few hits, we would expect a normal model to fit the data effectively. We interpret the failure of the mixture model to identify a well-defined hit peak in these eight screens as indicating that the screens have few hits and that therefore, in these cases, a Z-transformation would be appropriate. When present, the two overlapping, peaks in the data allows for the identification of two defined categories in the data. Component 1, or the central peak, contains genes unaffected by the interaction and is distributed normally due to noise in measurement. Component 2, or the hit peak, contains genes affected by the interaction, the shape of this distribution represents both effects of noise and the distribution of strength of real growth defects. We do not know <em>a priori</em> the shape of the distribution of interactions effects, but here we make the assumption it is Gaussian.</p></div><div id="sec-11" class="subsection"><h3>Tools based on mixture models</h3><p id="p-20">Having determined that mixture models are a more appropriate statistical model than a normal distribution, we developed metrics to determine the significance of individual results and cutoffs to distinguish hits from non-hits. A typical approach in genome-wide screens is to calculate p-values based on a null model of the data. In the case of mixture models, identifying Component 1 as an empirical null model for the data allows for calculation of p-values, which may be adjusted for multiple hypothesis testing, for example by calculating FDR q-values (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benjamini and Hochberg 1995</a>). However, in this context a more natural approach is to calculate the conditional probability of inclusion in Component 2. We define <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-16.gif"/></noscript></span></span> to be the probability of inclusion in Component 2 given a measured LGR of <em>x</em>. The point where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-17.gif"/></noscript></span></span> is the point where a strain with measured LGR <em>x</em> is equally likely to be in Component 1 or 2, and is therefore a logical point to place a cutoff. We define this point as <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-18.gif"/></noscript></span></span>, while the point where a Z-transformation of the data has value 2 is <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-19.gif"/></noscript></span></span>. We found that <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-20.gif"/></noscript></span></span> always sat below the <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-21.gif"/></noscript></span></span> but this effect was more pronounced in screens with more hits. Notably using Z-score as a cutoff limited the range of numbers of hits to 100-250. In contrast, using <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-22.gif"/></noscript></span></span> as a cutoff has a dynamic range of 100-700 hits (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, Supplementary Table S1). This makes the mixture model approach a more effective tool than Z-score to distinguish between screens with many or few hits. We compiled a list of hits for each screen based either on <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-23.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-24.gif"/></noscript></span></span>, depending on the success of the empirical Bayes approach (Supplementary File S4).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: The number of hits by both Z-score () and q(x) () cutoff for each of the screens where the mixture model was applicable. The q(x) cutoff has a higher dynamic range than the Z-score and is better able to distinguish screens with many hits. B: FPR prediction for the Spc72 screen. The FPR for the screen was predicted from the mixture model and this prediction is overlaid with estimates of the FPR using binned data from the validation screen. In this case, the predicted FPR was reasonably accurate, although the data are quite noisy. The points where the mixture model predicts  and  FPR are indicated with a dashed line. C: Box-and-whisker plot showing the difference between measured and predicted FPR at the point where the FPR is predicted to be  across the screens where the mixture model was applicable. This shows some bias, with the predicted FPR generally higher than the true FPR but generally achieving an accuracy around . D: Classification of mixture model fit for each of the 28 screens analyzed. The mean  and variance  of component 2 are good indicators of the success of the model with very low means or high variances indicative of the lack of a hit peak or poor validation prediction respectively. E: Classification of screen based on fitted parameters calculated using the subset of GFP strains used in the SPB screen. Each of the screens for which the mixture model fit was appropriate are plotted according to the proportion of strains affected () and the average strength of these effects (). The SPB screens Spc42 and Nud1 are positioned in the upper right portion of the graph, showing that a large proportion of proteins were sensitive to forced interaction with the SPB and these sensitivities caused significant growth defects." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1175178226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A: The number of hits by both Z-score (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-25&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and q(x) (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-26&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) cutoff for each of the screens where the mixture model was applicable. The q(x) cutoff has a higher dynamic range than the Z-score and is better able to distinguish screens with many hits. B: FPR prediction for the Spc72 screen. The FPR for the screen was predicted from the mixture model and this prediction is overlaid with estimates of the FPR using binned data from the validation screen. In this case, the predicted FPR was reasonably accurate, although the data are quite noisy. The points where the mixture model predicts &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; FPR are indicated with a dashed line. C: Box-and-whisker plot showing the difference between measured and predicted FPR at the point where the FPR is predicted to be &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-29&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; across the screens where the mixture model was applicable. This shows some bias, with the predicted FPR generally higher than the true FPR but generally achieving an accuracy around &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-30&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. D: Classification of mixture model fit for each of the 28 screens analyzed. The mean &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and variance &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-32&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; of component 2 are good indicators of the success of the model with very low means or high variances indicative of the lack of a hit peak or poor validation prediction respectively. E: Classification of screen based on fitted parameters calculated using the subset of GFP strains used in the SPB screen. Each of the screens for which the mixture model fit was appropriate are plotted according to the proportion of strains affected (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and the average strength of these effects (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The SPB screens Spc42 and Nud1 are positioned in the upper right portion of the graph, showing that a large proportion of proteins were sensitive to forced interaction with the SPB and these sensitivities caused significant growth defects.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2183/F3.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2183/F3.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493572" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">A: The number of hits by both Z-score (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-25.gif"/></noscript></span></span>) and q(x) (<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-26.gif"/></noscript></span></span>) cutoff for each of the screens where the mixture model was applicable. The q(x) cutoff has a higher dynamic range than the Z-score and is better able to distinguish screens with many hits. B: FPR prediction for the Spc72 screen. The FPR for the screen was predicted from the mixture model and this prediction is overlaid with estimates of the FPR using binned data from the validation screen. In this case, the predicted FPR was reasonably accurate, although the data are quite noisy. The points where the mixture model predicts <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-28.gif"/></noscript></span></span> FPR are indicated with a dashed line. C: Box-and-whisker plot showing the difference between measured and predicted FPR at the point where the FPR is predicted to be <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-29.gif"/></noscript></span></span> across the screens where the mixture model was applicable. This shows some bias, with the predicted FPR generally higher than the true FPR but generally achieving an accuracy around <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-30.gif"/></noscript></span></span>. D: Classification of mixture model fit for each of the 28 screens analyzed. The mean <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-31.gif"/></noscript></span></span> and variance <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-32.gif"/></noscript></span></span> of component 2 are good indicators of the success of the model with very low means or high variances indicative of the lack of a hit peak or poor validation prediction respectively. E: Classification of screen based on fitted parameters calculated using the subset of GFP strains used in the SPB screen. Each of the screens for which the mixture model fit was appropriate are plotted according to the proportion of strains affected (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-33.gif"/></noscript></span></span>) and the average strength of these effects (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F3/embed/mml-math-34.gif"/></noscript></span></span>). The SPB screens Spc42 and Nud1 are positioned in the upper right portion of the graph, showing that a large proportion of proteins were sensitive to forced interaction with the SPB and these sensitivities caused significant growth defects.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The top hits from genome-wide screens are commonly validated by repeating the screen either to verify key results or to establish metrics such as the False Positive Rate (FPR). Validation is undesirable as it requires further resources and, in some cases, may not be practical, so we developed a statistical method to predict the FPR. In the validation screens, 16 replicates were used as opposed to 4 in the original genome-wide screens. Furthermore, a hit was considered to be validated if the measured LGR was considered to be significant relative to GFP-free controls. Validation is considered to be a “gold-standard” for hit verification as it corresponds closely to other assays for growth defects such as spot tests. As the <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-35.gif"/></noscript></span></span> cutoffs were lower than the Z-score cutoffs we were concerned that they may not be reliable indicators of validation. Indeed, most of the <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-36.gif"/></noscript></span></span> cutoffs lay below the <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-37.gif"/></noscript></span></span> FPR point at which <a id="xref-ref-6-11" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> (2016)</a> stopped validating. It is worth noting that results that do not validate may still be reproducible and biologically interesting despite having relatively subtle effects on growth that are difficult to distinguish from the variability in wild type growth. Therefore, we developed a method to predict the likelihood of validation. Using the fitted mixture models, we developed a metric <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-38.gif"/></noscript></span></span>, representing the probability of validation for a strain with measured LGR <em>x</em>. <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-39.gif"/></noscript></span></span> is generally successful at predicting the rate of validation in a screen (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). For the 20 SPI screens which were fit well by the mixture models, the validation rate of 18 of these screens was predicted well by <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-40.gif"/></noscript></span></span>. The other two generally had very poor validation rates in general, making any kind of validation prediction unlikely to succeed. Plotting the variance and means of Component 2 for each of the SPI screens (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a>) shows that both of these screens are outliers with very high variances. Therefore, we recommend that when using this approach, great care is taken when the variance of Component 2 is high. Comparison of specific points, for example <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-41.gif"/></noscript></span></span> FPR, shows good predictive power (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>).</p></div><div id="sec-12" class="subsection"><h3>The SPB is especially sensitive to forced relocalization</h3><p id="p-23">When we compared the SPI screens using SPB components with the previous screens using other structures throughout the cell, we noticed some key differences. <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a> shows that the SPB SPI screens are among the screens with the greatest number of hits, both by Z-score and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-42.gif"/></noscript></span></span> cutoff. The fitted mixture models offer an additional way to understand this difference. Within a SPI screen, we may wish to distinguish between the case of a large proportion of strains being affected in a minor way and a smaller proportion of strains being very strongly affected. The fitted parameters <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-43.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-44.gif"/></noscript></span></span> reflect the proportion of strains affected and the severity of these effects respectively. Plotting these two parameters together therefore provides a graphical way to compare screens. This is shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3E</a> (Supplementary Table S1), where we see that the SPI screens sit in the top-right region of the graph as they have high values of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-45.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-46.gif"/></noscript></span></span>. In particular, Spc42 and Nud1 produce especially strong SPIs (high <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-47.gif"/></noscript></span></span>), while the Spc110 screens produce weaker SPIs but with many different strains (high <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-48.gif"/></noscript></span></span>). Spc72 is more midrange, possible reflecting the fact that in the S288C background, <em>SPC72</em> is a non-essential gene (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Giaever <em>et al.</em> 2002</a>). Notably, Loa1 has a high value of <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-49.gif"/></noscript></span></span>, Heh2 has a high value of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/embed/mml-math-50.gif"/></noscript></span></span> and Sec7 and Sec63 sit near to Spc42 and Nud1. All four of these proteins localize to the ER, Golgi or nuclear membrane, suggesting that these regions specifically may be the most sensitive to forcible relocalization. It should be noted that the screens of Berry <em>et al.</em> used the full GFP library <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Huh <em>et al.</em> (2003)</a>, while the SPB SPI screens were performed with a smaller subset of 4,500 strains with confirmed GFP fluorescence <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Tkach <em>et al.</em> (2012)</a>. The parameters fitted to the screens of Berry <em>et al.</em> by the empirical Bayes method are not significantly affected by restricting the analysis to this subset of strains (Supplementary Table S2).</p><p id="p-24">We used hierarchical clustering to compare the SPB screens to the other SPI screens in the dataset (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The data were clustered both vertically (by GFP strain) and horizontally (by screen). Clustering by screen shows the five SPB screens are more similar to each other than to other screens in the dataset, suggesting there is a characteristic set of proteins that are sensitive to forced localization to the SPB. A Venn diagram showing overlap between hits from the SPB SPI screens is shown in Supplementary Figure S1. Clustering the data by GFP strain identifies clusters of biologically related proteins with similar profiles of localization sensitivity, as previously reported (<a id="xref-ref-6-12" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> 2016</a>). Clusters of proteins with SPIs with the SPB group together, for example, the fatty acid elongases Elo1, Elo2 and Elo3; and the two paralogs of HMG-CoA reductase Hmg1 and Hmg2. These clusters also link together members of protein complexes such as the ER membrane protein complex (EMC) and oligosaccharyltransferase complex (OST). The clustering identifies a group of proteins that appear to enhance growth when forced to interact with both termini of Spc110. This group is not significantly enriched for any GO terms, however, it does include Mad2, consistent with the idea that partial Mad2 perturbation may accelerate cell cycle progression (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barnhart <em>et al.</em> 2011</a>). However, we found that growth enhancers were unlikely to reproduce their behavior in validation screens. The vertical clustering also identifies a collection of proteins that are sensitive to forcible relocalization to all or most parts of the cell. This group of proteins, known as ”frequent flyers”, are enriched for transcription factors and nuclear proteins.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cluster analysis of all 28 SPI screens used in this study. The data are clustered both vertically (by GFP strain) and horizontally (by screen, tree shown). The horizontal clustering tree shows that the SPB screen results are more similar to each other than the other screens. The vertical clustering identifies clusters of biologically related proteins with similar profiles of sensitivity to forcible relocalization." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1175178226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cluster analysis of all 28 SPI screens used in this study. The data are clustered both vertically (by GFP strain) and horizontally (by screen, tree shown). The horizontal clustering tree shows that the SPB screen results are more similar to each other than the other screens. The vertical clustering identifies clusters of biologically related proteins with similar profiles of sensitivity to forcible relocalization.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2183/F4.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2183/F4.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493558" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Cluster analysis of all 28 SPI screens used in this study. The data are clustered both vertically (by GFP strain) and horizontally (by screen, tree shown). The horizontal clustering tree shows that the SPB screen results are more similar to each other than the other screens. The vertical clustering identifies clusters of biologically related proteins with similar profiles of sensitivity to forcible relocalization.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">In order to understand which kinds of proteins are sensitive to forced relocalization to each part of the SPB, we used the online tool thecellmap.org (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Usaj <em>et al.</em> 2017</a>) to perform spatial analysis of functional enrichment and visualize the results (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). SPB SPIs cluster together in the region of the network containing mitotic regulators and transcription factors. Additionally, there is a cluster of hits from the Spc110 and Spc42 screens in the region containing proteins involved in cell wall, glycosylation and protein folding. There are further clusters for individual screens including Spc110 N-terminus clusters in ribosome biogenesis and vesicle traffic regions and a cluster of Nud1 hits in the rRNA/ncRNA processing region.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SAFE enrichment of hits from SPB SPI screen, visualized using TheCellMap.org. The S. cerevisiae genetic interaction network was clustered by density, identifying highly dense regions of space corresponding to shared function. Regions of space containing high densities of hits from each screen are highlighted accordingly showing visually which processes the hits from the screens are related to." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1175178226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SAFE enrichment of hits from SPB SPI screen, visualized using TheCellMap.org. The &lt;em&gt;S. cerevisiae&lt;/em&gt; genetic interaction network was clustered by density, identifying highly dense regions of space corresponding to shared function. Regions of space containing high densities of hits from each screen are highlighted accordingly showing visually which processes the hits from the screens are related to.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2183/F5.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2183/F5.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493485" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">SAFE enrichment of hits from SPB SPI screen, visualized using TheCellMap.org. The <em>S. cerevisiae</em> genetic interaction network was clustered by density, identifying highly dense regions of space corresponding to shared function. Regions of space containing high densities of hits from each screen are highlighted accordingly showing visually which processes the hits from the screens are related to.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We performed Gene Ontology (GO) analysis of the ranked LGRs for each of the screens using the GOrilla tool (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Eden <em>et al.</em> 2009</a>). Rather than using a list of hits from the SPI screen, we used the entire ranked list for each dataset, negating the need for a cutoff LGR. Heatmaps of significant enrichments are shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> (Supplementary File S5). The screens with Spc42, Spc110C and Spc110N fusions were all significantly enriched for proteins involved in lipid metabolic process and proteins from the ER. In particular, there was significant enrichment for proteins involved in biosynthesis of sterols, sphingolipids and those involved in fatty acid elongation (Supplementary Figure S2). The position of the SPB, embedded within the nuclear membrane (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Jaspersen and Winey 2004</a>), suggests that these growth defects may result from disregulation of nuclear membrane composition. Furthermore <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Witkin <em>et al.</em> (2010)</a> found that deletion of <em>SPO7</em>, a regulator of phospholipid biosynthesis, could partially suppress the monopolar phenotype of mutations in <em>MPS3</em>, suggesting that the membrane environment can impact on SPB duplication. We also found that the screens with Nud1, Spc72 and Spc110N, the proteins located closest to the sites of microtubule nucleation, were enriched for proteins involved in the process of microtubule nucleation. These findings suggest that targeting these proteins artificially to the SPB can induce growth defects, possibly due to problems with spindle formation or nuclear positioning. An intriguing result is the finding that all screens, except Spc42, were enriched for proteins involved in chromosome segregation and components of the chromosome and kinetochore; there is evidence for example from yeast-two-hybrid screens that kinetochore proteins physically interact with SPB components (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wong <em>et al.</em> 2007</a>). It is worth noting that these phenotypes may simply represent disruption of these structures by removal of the protein, although these proteins were not frequent flyers. Nud1 and Spc72 are thought to act as a signaling scaffold for proteins in the Mitotic Exit Network pathway (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Scarfone and Piatti 2015</a>) and screens with these proteins were enriched for mitotic cell cycle proteins. Finally, we found that the Spc42 screen was enriched for proteins involved in nuclear pore organization as well as subunits of the nuclear pore. Intriguingly, some of these findings overlap with known genetic interactions, for example deletion of <em>NUP157</em> suppresses the <em>spc42-11</em> mutation (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Witkin <em>et al.</em> 2010</a>), while we found that tethering Nup157 to Spc42 lead to a growth defect. Due to the proposed link between SPB duplication and insertion and the nuclear pore (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Rüthnick and Schiebel 2018</a>) we investigated these results further.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GO analysis of SPB SPI screens, performed using the entire, ranked dataset. A: Heatmap of process GO analysis, dark blue tiles represent no significant enrichment while the lighter colors represent significant enrichment, with warmer tones representing higher p-values. B: Heatmap of component GO analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1175178226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GO analysis of SPB SPI screens, performed using the entire, ranked dataset. A: Heatmap of process GO analysis, dark blue tiles represent no significant enrichment while the lighter colors represent significant enrichment, with warmer tones representing higher p-values. B: Heatmap of component GO analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2183/F6.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/7/2183/F6.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493596" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-29" class="first-child">GO analysis of SPB SPI screens, performed using the entire, ranked dataset. A: Heatmap of process GO analysis, dark blue tiles represent no significant enrichment while the lighter colors represent significant enrichment, with warmer tones representing higher p-values. B: Heatmap of component GO analysis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>SPIs with the SPB lead to SPB overduplication</h3><p id="p-30">We investigated whether we could detect any SPB duplication phenotype caused by forcible localization of proteins to the SPB. We screened 80 query proteins that we suspected would cause defects in SPB duplication against the Spc42-GBP-RFP fusion. These proteins included hits from our screen and other proteins known to function in the same pathways or complexes. For example, we identified Nsp1, a nucleoporin, in our screen and included other nuclear pore components such as Nup170, which was not a SPI in our original screen. Cells were imaged using fluorescence microscopy and the number of Spc42-GBP-RFP foci were counted. In strains expressing membrane or pore proteins tagged with GFP we observed recruitment of Spc42-GBP-RFP to these regions, however small regions with relatively high RFP signal were observed and these were counted as foci. We observed cells with 3 or more RFP foci in eleven different strains (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, Supplementary Table S3). In some cases, a single red focus was observed in large budded cells, however the slow-folding nature of RFP prevented us from ruling out the possibility of further SPBs that are unmarked by mature RFP.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493584/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493584/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Proteins identified in the microscopy screen for proteins that induce extra SPBs when forcibly relocalized to the SPB. * YJL021C overlaps the originally identified YJL020C ORF and so has been merged into YJL020C (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brachat <em>et al.</em> 2003</a>), however the GFP strain shows a punctate fluorescent signal. Database locations were accessed from <a href="http://yeastgenome.org/">yeastgenome.org/</a></span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Screening cells directly from the SPI screen meant that limited number of cells were available to image in slow-growing SPI strains. Therefore, we directly transformed these strains, alongside the four members of the SPIN network, with the Spc42-GBP-RFP plasmid as well as a control plasmid expressing Spc42-RFP. We were able to establish colonies of all strains except Crm1-GFP. Using these strains, we imaged larger quantities of these cells (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>, Supplementary Figure S3). We detected extra red foci in each of the strains expressing Spc42-GBP-RFP and quantified the proportion of cells expressing this phenotype (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>, Supplementary Table S4). In most of the GFP strains transformed with the control plasmid we did not detect additional RFP foci, however to our surprise we did observe additional foci in the Mps2-GFP strain and, to a much lesser extent, the Nbp1-GFP and Pom34-GFP strains. Forced recruitment to the SPB of all proteins gave rise to statistically significant proportions of cells containing additional RFP foci, with the exception of Mps2, Ndc1, YDL121C and Nbp1. Notably, we found that the strength of growth defect as measured by the LGR was not a strong indicator of the frequency of extra red foci, suggesting the growth defect does not arise entirely from this phenotype. Note that the protein denoted by its ORF, YJL021C, is included in these results however this ORF was determined to overlap <em>YJL020C</em> <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brachat <em>et al.</em> (2003)</a> meaning the GFP product in this strain is likely not a simple N-terminal fusion. Furthermore, the GFP strain shows a punctate fluorescent signal, meaning the extra red foci in these cells may represent relocalization of Spc42-GBP-RFP to YJL021C-GFP foci. These results suggest that the forced interaction of these proteins with the SPB results in aggregates of the Spc42 protein that may indicate extra SPBs. In order to gain a further insight into this phenotype, we constructed strains expressing mTurq-Tub1 and Bbp1-, Nup133-, Nup170- and YJL021C-YFP respectively, and transformed them with the Spc42-GBP-RFP plasmid. YFP has enough homology to GFP that it interacts strongly with GBP but the fluorescent signal is distinguishable from mTurq, allowing for identification of protein localization in three colors. Analysis of spindle morphology in the Bbp1-YFP strain revealed mTurq-Tub1 signal between multiple red foci, and in some cases showed a multi-polar spindle phenotype (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a> and Supplementary Figure S4), suggesting the additional red foci observed in this strain represent SPBs and not simply aggregates of Spc42-GBP-RFP. Data from the Nup133- and YJL021C-YFP strains also indicate that the red foci represent SPB-like structures that are capable of nucleating microtubules (Figure S5). In Nup170-YFP cells with multiple red foci, we observed only two foci associated with mTurq-Tub1 signal, suggesting the red foci in this strain may not represent functional SPBs.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Extra RFP foci phenotype. Plasmids encoding either Spc42-GBP-RFP or Spc42-RFP were directly transformed into the GFP strains identified in the screen for SPB number aberrations and imaged. A: Representative images of Bbp1, Nsp1, Pom34 and YPR071W -GFP strains expressing Spc42-GBP-RFP from a plasmid, each showing more than two RFP foci, interpreted as indicative of overduplication of SPBs. All scale bars are m. B: Quantification of the microscopy analysis, with key proteins highlighted. Three images were captured for each strain and the percentage of living cells showing more than two RFP foci was calculated. Error bars show  confidence intervals calculated with the Clopper-Pearson exact method; p-values were calculated using Fisher&#x2019;s exact test. C: Multipolar spindle morphology in the Spc42-GBP-RFP Bbp1-YFP strain, the spindle is labeled by mTurq-Tub1. Green lines indicate the observed spindle morphology, from the mTurq-Tub1 signal, relative to the cell morphology observed in DIC. Scale bars are m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1175178226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Extra RFP foci phenotype. Plasmids encoding either Spc42-GBP-RFP or Spc42-RFP were directly transformed into the GFP strains identified in the screen for SPB number aberrations and imaged. A: Representative images of Bbp1, Nsp1, Pom34 and YPR071W -GFP strains expressing Spc42-GBP-RFP from a plasmid, each showing more than two RFP foci, interpreted as indicative of overduplication of SPBs. All scale bars are &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-51&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;m. B: Quantification of the microscopy analysis, with key proteins highlighted. Three images were captured for each strain and the percentage of living cells showing more than two RFP foci was calculated. Error bars show &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-52&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; confidence intervals calculated with the Clopper-Pearson exact method; p-values were calculated using Fisher&#x2019;s exact test. C: Multipolar spindle morphology in the Spc42-GBP-RFP Bbp1-YFP strain, the spindle is labeled by mTurq-Tub1. Green lines indicate the observed spindle morphology, from the mTurq-Tub1 signal, relative to the cell morphology observed in DIC. Scale bars are &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-53&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;m.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2183/F7.medium.gif" width="269" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/7/2183/F7.medium.gif" width="269" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2183/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493496" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-32" class="first-child">Extra RFP foci phenotype. Plasmids encoding either Spc42-GBP-RFP or Spc42-RFP were directly transformed into the GFP strains identified in the screen for SPB number aberrations and imaged. A: Representative images of Bbp1, Nsp1, Pom34 and YPR071W -GFP strains expressing Spc42-GBP-RFP from a plasmid, each showing more than two RFP foci, interpreted as indicative of overduplication of SPBs. All scale bars are <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F7/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F7/embed/mml-math-51.gif"/></noscript></span></span>m. B: Quantification of the microscopy analysis, with key proteins highlighted. Three images were captured for each strain and the percentage of living cells showing more than two RFP foci was calculated. Error bars show <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F7/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F7/embed/mml-math-52.gif"/></noscript></span></span> confidence intervals calculated with the Clopper-Pearson exact method; p-values were calculated using Fisher’s exact test. C: Multipolar spindle morphology in the Spc42-GBP-RFP Bbp1-YFP strain, the spindle is labeled by mTurq-Tub1. Green lines indicate the observed spindle morphology, from the mTurq-Tub1 signal, relative to the cell morphology observed in DIC. Scale bars are <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F7/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2183/F7/embed/mml-math-53.gif"/></noscript></span></span>m.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>Analysis of SPI screens</h3><p id="p-33">Z-transformations are a common tool to analyze genome-wide screens, however their underlying assumption of normally distributed data means they can produce unreliable results, especially in screens identifying many hits. We developed an empirical Bayes approach to address the shortcomings of Z-transformations. Rather than use the Z-score to test significance we introduced a cutoff based on the probability of inclusion, which is more effective in discriminating between screens with many hits. Notably, some hits in the Spc42 screen, such as Nup133, had LGRs which would have been considered insignificant according to the Z-score cutoff, but showed a distinct phenotype in further assays. This shows that the lower cutoffs we propose can still be biologically relevant. Additionally, we developed a method to predict the validation rate of the screen with reasonable accuracy. These two metrics provide alternative viewpoints on the significance and strength of a given result within a screen. Furthermore, bimodal normal mixture models have 5 independent parameters, allowing for more effective parameterisation of the distribution than the 2 parameters of a single normal distribution. These parameters can be used to compare the distribution of results from different screens, providing a way to understand the differences between SPI screens. Previous analysis of cell-wide SPI screens concluded that only a small proportion of proteins are sensitive to forced relocalization however our re-analysis of this data and the inclusion of the SPB in this dataset suggests that some regions of the cell are far more sensitive to forced relocalization of proteins across the proteome. The empirical Bayes approach offers the most significant benefits in the cases where Z-transformations are least appropriate: when analyzing screens with large numbers of hits. However, we did find that when the fitted standard deviation of Component 2 was too large, the screens validated poorly and the mixture model was inaccurate.</p><p id="p-34">Mixture models allow for a quick and easy way to effectively parameterise screening distribution data, however they cannot provide perfect prediction with imperfect data. If greater levels of precision or reproducibility are necessary, further modifications to the experimental procedures would be required. Zackrisson and colleagues found significant variation in growth rates of colonies across a single plate and recommend local normalization of colony size to account for these effects to improve reproducibility (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zackrisson <em>et al.</em> 2016</a>). Baryshnikova and colleagues found that “batch” effects, caused by subtle differences in, for example, media composition or incubator temperature, between plates grown at different times, caused significant variation in colony sizes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baryshnikova <em>et al.</em> 2010</a>). While all plates in a SPI screen are generally grown concurrently, the validation screens were performed afterward, once analysis of the screen has been performed. This may explain the high FPRs in validation of some screens. Baryshnikova and colleagues propose using linear discriminant analysis to compute “batch signatures” which could be used to limit batch effects. Finally, the precision of measurements could be improved by a methodology that directly correlates growth measurement to rate, for example by calculating a growth curve using automated scanning of plates at regular intervals (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Zackrisson <em>et al.</em> 2016</a>).</p></div><div id="sec-16" class="subsection"><h3>SPB overduplication</h3><p id="p-35">Our current understanding of the SPB duplication cycle of <em>S. cerevisiae</em> is that alternating activities of the CDK-cyclin complex and Cdc14 phosphatase are responsible for once-per-cycle duplication of the SPB (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Rüthnick and Schiebel 2018</a>). This model suggests that SPB duplication is initiated while Cdc14 activity is at its peak but may not be completed until CDK activity increases and Cdc14 activity decreases later in the cell cycle. A reasonable prediction would be that forced recruitment of Cdc14 to the SPB would induce overduplication of SPBs, however we found that forced recruitment of Cdc14 only produced a growth defect with the outer plaque component Nud1 and we did not observe evidence of SPB overduplication when Cdc14 was recruited to any part of the SPB (unpublished observations). It is worth noting that a previous SPI screen found that the combination of Cdc14-GBP and Spc42-GFP induced a growth defect that depended on the phosphatase activity of Cdc14 <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Ólafsson and Thorpe (2015)</a>, however when the tags were switched in the screen reported here, we found no growth defect.</p><p id="p-36">It has been proposed that the SPB satellite is inserted into the nuclear membrane using molecular machinery that is responsible for nuclear pore complex (NPC) insertion (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Rüthnick and Schiebel 2018</a>). We screened NPC and SPIN proteins for SPIs with Spc42 and used fluorescence microscopy to count Spc42 foci in these strains. We found evidence that forcible recruitment of the SPIN component Bbp1 to the SPB induced formation of additional Spc42-GBP-RFP foci, which were capable of nucleating microtubules. We note that the ability of the extra SPB foci to nucleate microtubules does not necessarily mean that these foci represent canonical SPBs, a possibility which could be explored with further fluorescent SPB markers. We also found that the NPC components Nsp1, Nic96, Nup133 and Pom34 produced extra foci and that in the case of Nup133, these foci colocalized with tubulin. It is interesting that the deletion of either of two of the genes coding for these proteins, <em>NIC96</em> and <em>POM34</em>, were identified as suppressors of SPB duplication defects caused by <em>mps3-1 spo7</em>Δ mutation (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Witkin <em>et al.</em> 2010</a>). This finding suggests that in their wild type localization, these proteins inhibited SPB duplication, possibly by competing for binding partners, whereas our data suggests that when forced to the SPB, these proteins can induce the overduplication of SPBs. Additionally, we found evidence that the, as yet, unclassified proteins encoded by YJL021C, YPR071W, YPR114W and YDL121C as well as Apq12 similarly induce extra Spc42 foci indicative of SPB overduplication. There are several interpretations of these findings, which may apply to some or all of the phenotypes observed. First, it is possible that the RFP foci observed may represent aggregates of Spc42-GBP-RFP that do not contain other SPB proteins or function as MTOCs. It is worth remarking that in a systematic study of localization of target and query proteins, a small proportion were found to localize to a region of the cell where neither would localize in wild type cells (<a id="xref-ref-6-13" class="xref-bibr" href="#ref-6">Berry <em>et al.</em> 2016</a>). Second, it may be that forced recruitment of these proteins induce SPB overduplication through the documented SPB duplication pathway. This would require detachment of this process from the once-per-cycle regulation via CDK-cyclin and Cdc14. This could be explained if some aspects of this process were initiated by the presence of these proteins at the SPB, which in wild-type cells was induced by CDK or Cdc14 activity. Finally, it may be the case that targeting Spc42 to other structures in the cell, especially the NPC, can lead to the creation of <em>de novo</em> SPBs. The current model of SPB duplication suggests that SPBs assemble from a satellite formed of Spc42, Nud1, Cnm67 and Spc29 (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Fu <em>et al.</em> 2015</a>). It may be that a small amount of Spc42 is recruited to the NPC in these strains, seeding new SPBs in a manner completely distinct from regular SPB duplication. <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Witkin <em>et al.</em> (2010)</a> proposed an <em>MPS3</em> independent SPB duplication pathway and it may be this or some other pathway that is responsible for this phenotype.</p><p id="p-37">Further work is required to distinguish these models, in particular, assessment of the foci for presence of other SPB proteins and functionality of the foci as MTOCs is required to confirm them as real SPBs. If SPBs are created <em>de novo</em> we would expect that these strains would lose the requirement for proteins with an essential role in SPB duplication, such as Cdc31 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rüthnick and Schiebel 2016</a>). Many mutants have been identified that fail to duplicate their SPBs, for example the original MPS (Mono-Polar Spindle) genes (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Winey <em>et al.</em> 1991</a>) however there are fewer cases of genetic perturbations that lead to SPB overduplication. Overexpression of Cdc5 or mutations to Aurora kinase, Ipl1, have been shown to cause overduplication of SPBs in meiosis (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shirk <em>et al.</em> 2011</a>). In mitosis, deletion of all <em>CLB</em> genes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Haase <em>et al.</em> 2001</a>) and overexpression of Cdc5 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Song <em>et al.</em> 2000</a>) have been demonstrated to induce SPB overduplication however the pleiotropic effects of these mutations may limit their use as model systems to study multi-polar spindles. The <em>sfi1-C4A</em> mutation (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Avena <em>et al.</em> 2014</a>), causes SPB overduplication without perturbing key cell cycle control mechanisms however this strain also exhibits an SPB seperation defect meaning additional SPBs are kept tethered to the original SPB, preventing formation of multi-polar spindles. Therefore, the multi-polar spindle phenotype caused by forced association of Bbp1 and Spc42 may offer a useful system to investigate physiological responses to multi-polar spindles.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">The authors acknowledge funding from the Francis Crick Institute, which receives its core funding from Cancer Research UK (FC001003), the UK Medical Research Council (FC001003), the Wellcome Trust (FC001003). We thank L Berry, G Ólafsson, S Godinho, P Bates, F Caudron, S Santos, W Taylor, U Eggert, J Diffley and the Genomics Equipment Park STP (The Francis Crick Institute). The authors declare no competing financial interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-39">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8100749">https://doi.org/10.25387/g3.8100749</a>.</p></li><li class="fn" id="fn-2"><p id="p-40"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-21"><span class="received-label">Received </span>February 21, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-02"><span class="accepted-label">Accepted </span>May 2, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Howell <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.1" data-doi="10.1098/rstb.2013.0464"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arquint</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabryjonczyk</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigg</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Centrosomes as signalling centres.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">369</span>: <span class="cit-elocation-id">20130464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2013.0464</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0464%26rft_id%253Dinfo%253Apmid%252F25047618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2013.0464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25047618&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.2" data-doi="10.1371/journal.pgen.1004666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avena</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeier</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Old</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Licensing of Yeast Centrosome Duplication Requires Phosphoregulation of Sfi1.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004666</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004666%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004666%26rft_id%253Dinfo%253Apmid%252F25340401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25340401&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.3" data-doi="10.1091/mbc.E09-01-0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnhart</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorer</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuyler</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reduced Mad2 expression keeps relaxed kinetochores from arresting budding yeast in mitosis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2448</span>–<span class="cit-lpage">2457</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.e09-01-0029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E09-01-0029%26rft_id%253Dinfo%253Apmid%252F21593209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xNC8yNDQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.4" data-doi="10.1038/nmeth.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1024</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBaryshnikova%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1017%26rft.epage%253D1024%26rft.atitle%253DQuantitative%2Banalysis%2Bof%2Bfitness%2Band%2Bgenetic%2Binteractions%2Bin%2Byeast%2Bon%2Ba%2Bgenome%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1534%26rft_id%253Dinfo%253Apmid%252F21076421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076421&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284686300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.5" data-doi="10.1093/biostatistics/kxj037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxj037%26rft_id%253Dinfo%253Apmid%252F16632515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxj037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16632515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.6" data-doi="10.7554/eLife.13053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ólafsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ledesma-Fernandez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorpe</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Synthetic protein interactions reveal a functional map of the cell.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e13053</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.13053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De13053%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.13053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.13053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.7" data-doi="10.1186/gb-2003-4-7-r45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voegeli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Reinvestigation of the Saccharomyces cerevisiae genome annotation by comparison to the genome of a related fungus: Ashbya gossypii.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">R45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2003-4-7-r45</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBrachat%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253DR45%26rft.epage%253DR45%26rft.atitle%253DReinvestigation%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome%2Bannotation%2Bby%2Bcomparison%2Bto%2Bthe%2Bgenome%2Bof%2Ba%2Brelated%2Bfungus%253A%2BAshbya%2Bgossypii.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2003-4-7-r45%26rft_id%253Dinfo%253Apmid%252F12844361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2003-4-7-r45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12844361&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.8" data-doi="10.1038/ncomms14129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caydasi</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khmelinskii</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duenas-Sanchez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtulmus</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Temporal and compartment-specific signals coordinate mitotic exit with spindle position.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">14129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms14129</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D14129%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms14129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.9" data-doi="10.1186/1471-2105-11-353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dittmar</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ScreenMill: a freely available software suite for growth measurement, analysis and visualization of high-throughput screen data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-11-353</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDittmar%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D11%26rft.spage%253D353%26rft.epage%253D353%26rft.atitle%253DScreenMill%253A%2Ba%2Bfreely%2Bavailable%2Bsoftware%2Bsuite%2Bfor%2Bgrowth%2Bmeasurement%252C%2Banalysis%2Band%2Bvisualization%2Bof%2Bhigh-throughput%2Bscreen%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-353%26rft_id%253Dinfo%253Apmid%252F20584323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20584323&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.10" data-doi="10.1186/1471-2105-10-48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinfeld</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakhini</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-48</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEden%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DGOrilla%253A%2Ba%2Btool%2Bfor%2Bdiscovery%2Band%2Bvisualization%2Bof%2Benriched%2BGO%2Bterms%2Bin%2Branked%2Bgene%2Blists.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-48%26rft_id%253Dinfo%253Apmid%252F19192299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19192299&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.11" data-doi="10.1198/016214504000000089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Large-Scale Simultaneous Hypothesis Testing.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214504000000089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D99%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214504000000089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214504000000089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220638200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.12" data-doi="10.1073/pnas.95.25.14863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Cluster analysis and display of genome-wide expression patterns.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">14863</span>–<span class="cit-lpage">14868</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.95.25.14863</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEisen%26rft.auinit1%253DM.%2BB.%26rft.volume%253D95%26rft.issue%253D25%26rft.spage%253D14863%26rft.epage%253D14868%26rft.atitle%253DCluster%2Banalysis%2Band%2Bdisplay%2Bof%2Bgenome-wide%2Bexpression%2Bpatterns%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.25.14863%26rft_id%253Dinfo%253Apmid%252F9843981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMjUvMTQ4NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.13" data-doi="10.1198/016214502760047131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Model-Based Clustering, Discriminant Analysis, and Density Estimation.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214502760047131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D97%26rft.spage%253D611%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214502760047131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214502760047131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000176078300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.14" data-doi="10.1101/cshperspect.a015800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagan</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Centrosome and Its Duplication Cycle.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">a015800</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a015800</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DFu%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253Da015800%26rft.epage%253Da015800%26rft.atitle%253DThe%2BCentrosome%2Band%2BIts%2BDuplication%2BCycle%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a015800%26rft_id%253Dinfo%253Apmid%252F25646378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI3LzIvYTAxNTgwMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.15" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature00935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.16" data-doi="10.7554/eLife.14029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gryaznova</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caydasi</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malengo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sourjik</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A FRET-based study reveals site-specific regulation of spindle position checkpoint proteins at yeast centrosomes.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e14029</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.14029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De14029%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.14029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.14029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.17" data-doi="10.1038/35050543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multi-step control of spindle pole body duplication by cyclin-dependent kinase.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35050543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DHaase%26rft.auinit1%253DS.%2BB.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D42%26rft.atitle%253DMulti-step%2Bcontrol%2Bof%2Bspindle%2Bpole%2Bbody%2Bduplication%2Bby%2Bcyclin-dependent%2Bkinase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35050543%26rft_id%253Dinfo%253Apmid%252F11146624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35050543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11146624&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.18" data-doi="10.1038/nature02026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falvo</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein localization in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">691</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHuh%26rft.auinit1%253DW.%2BK.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D686%26rft.epage%253D691%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02026%26rft_id%253Dinfo%253Apmid%252F14562095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.19" data-doi="10.4161/nucl.20148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaspersen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Nuclear envelope insertion of spindle pole bodies and nuclear pore complexes.</span> <abbr class="cit-jnl-abbrev">Nucleus</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/nucl.20148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNucleus%26rft.aulast%253DJaspersen%26rft.auinit1%253DS.%2BL.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D226%26rft.epage%253D236%26rft.atitle%253DNuclear%2Benvelope%2Binsertion%2Bof%2Bspindle%2Bpole%2Bbodies%2Band%2Bnuclear%2Bpore%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fnucl.20148%26rft_id%253Dinfo%253Apmid%252F22572959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/nucl.20148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22572959&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.20" data-doi="10.1146/annurev.cellbio.20.022003.114106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaspersen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The budding yeast spindle pole body: structure, duplication, and function.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.cellbio.20.022003.114106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BCell%2BDev%2BBiol%26rft.volume%253D20%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.cellbio.20.022003.114106%26rft_id%253Dinfo%253Apmid%252F15473833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.cellbio.20.022003.114106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15473833&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225318200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.21" data-doi="10.1111/tra.12276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Markus</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranowski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improved Plasmids for Fluorescent Protein Tagging of Microtubules in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Traffic</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tra.12276</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTraffic%26rft.volume%253D16%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftra.12276%26rft_id%253Dinfo%253Apmid%252F25711127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tra.12276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25711127&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.22" data-doi="10.1038/nbt1264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mena</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treynor</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayo</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Blue fluorescent proteins with enhanced brightness and photostability from a structurally targeted library.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1569</span>–<span class="cit-lpage">1571</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt1264</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMena%26rft.auinit1%253DM.%2BA.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D1569%26rft.epage%253D1571%26rft.atitle%253DBlue%2Bfluorescent%2Bproteins%2Bwith%2Benhanced%2Bbrightness%2Band%2Bphotostability%2Bfrom%2Ba%2Bstructurally%2Btargeted%2Blibrary.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1264%26rft_id%253Dinfo%253Apmid%252F17115054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17115054&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242795800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.23" data-doi="10.1002/ijc.22245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nigg</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Origins and consequences of centrosome aberrations in human cancers.</span> <abbr class="cit-jnl-abbrev">Int. J. Cancer</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">2717</span>–<span class="cit-lpage">2723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ijc.22245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bcancer.%2B%2BJournal%2Binternational%2Bdu%2Bcancer%26rft.stitle%253DInt%2BJ%2BCancer%26rft.aulast%253DNigg%26rft.auinit1%253DE.%2BA.%26rft.volume%253D119%26rft.issue%253D12%26rft.spage%253D2717%26rft.epage%253D2723%26rft.atitle%253DOrigins%2Band%2Bconsequences%2Bof%2Bcentrosome%2Baberrations%2Bin%2Bhuman%2Bcancers.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fijc.22245%26rft_id%253Dinfo%253Apmid%252F17016823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ijc.22245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17016823&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242307800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.24" data-doi="10.1073/pnas.1506101112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ólafsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorpe</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Synthetic physical interactions map kinetochore regulators and regions sensitive to constitutive Cdc14 localization.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">10413</span>–<span class="cit-lpage">10418</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1506101112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1506101112%26rft_id%253Dinfo%253Apmid%252F26240346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzMzLzEwNDEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.25" data-doi="10.1534/g3.116.031930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ólafsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorpe</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Synthetic Physical Interactions Map Kinetochore-Checkpoint Activation Regions.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2531</span>–<span class="cit-lpage">2542</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.031930</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.031930%26rft_id%253Dinfo%253Apmid%252F27280788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOC8yNTMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2183.26" data-doi="10.1007/978-1-4939-7306-4_39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ólafsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorpe</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Rewiring the Budding Yeast Proteome using Synthetic Physical Interactions</span>, pp. <span class="cit-fpage">599</span>–<span class="cit-lpage">612</span> in <span class="cit-source">Genome Instability</span>, <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">New York, NY</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-7306-4_39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BInstability%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-7306-4_39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-7306-4_39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.27" data-doi="10.1101/gr.109033.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R. J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Barrera</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunjevaric</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvaro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciccone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Selective ploidy ablation, a high-throughput plasmid transfer protocol, identifies new genes affecting topoisomerase I-induced DNA damage.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.109033.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.109033.110%26rft_id%253Dinfo%253Apmid%252F21173034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzMvNDc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.28" data-doi="10.1038/nmeth953"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothbauer</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zolghadr</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tillib</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schermelleh</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Targeting and tracing antigens in live cells with fluorescent nanobodies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth953</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DRothbauer%26rft.auinit1%253DU.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253D887%26rft.epage%253D889%26rft.atitle%253DTargeting%2Band%2Btracing%2Bantigens%2Bin%2Blive%2Bcells%2Bwith%2Bfluorescent%2Bnanobodies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth953%26rft_id%253Dinfo%253Apmid%252F17060912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth953&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17060912&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241788500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.29" data-doi="10.1128/MCB.00048-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rüthnick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiebel</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Duplication of the Yeast Spindle Pole Body Once per Cell Cycle.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1324</span>–<span class="cit-lpage">1331</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.00048-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00048-16%26rft_id%253Dinfo%253Apmid%252F26951196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjM2LzkvMTMyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.30" data-doi="10.3390/cells7050042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rüthnick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiebel</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Duplication and Nuclear Envelope Insertion of the Yeast Microtubule Organizing Centre, the Spindle Pole Body.</span> <abbr class="cit-jnl-abbrev">Cells</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">E42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/cells7050042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCells%26rft.volume%253D7%26rft.spage%253DE42%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fcells7050042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/cells7050042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.31" data-doi="10.1093/bioinformatics/bth349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saldanha</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Java Treeview–extensible visualization of microarray data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3246</span>–<span class="cit-lpage">3248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bth349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth349%26rft_id%253Dinfo%253Apmid%252F15180930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15180930&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225361400039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.32" data-doi="10.1080/21541248.2015.1109023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scarfone</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piatti</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Coupling spindle position with mitotic exit in budding yeast: The multifaceted role of the small GTPase Tem1.</span> <abbr class="cit-jnl-abbrev">Small GTPases</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">196</span>–<span class="cit-lpage">201</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/21541248.2015.1109023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSmall%2BGTPases%26rft.volume%253D6%26rft.spage%253D196%26rft_id%253Dinfo%253Adoi%252F10.1080%252F21541248.2015.1109023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/21541248.2015.1109023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.33" data-doi="10.1093/bioinformatics/bti062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An empirical Bayes approach to inferring large-scale gene association networks.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti062%26rft_id%253Dinfo%253Apmid%252F15479708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15479708&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227562200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.34" data-doi="10.32614/RJ-2016-021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scrucca</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">mclust 5: Clustering, Classification and Density Estimation Using Gaussian Finite Mixture Models.</span> <abbr class="cit-jnl-abbrev">R J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">317</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.32614/RJ-2016-021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BJ.%26rft.volume%253D8%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.32614%252FRJ-2016-021%26rft_id%253Dinfo%253Apmid%252F27818791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.32614/RJ-2016-021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27818791&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.35" data-doi="10.1242/jcs.086652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirk</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giddings</span>  <span class="cit-name-given-names">T. H.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.-G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Aurora kinase Ipl1 is necessary for spindle pole body cohesion during budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">2891</span>–<span class="cit-lpage">2896</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jcs.086652</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DShirk%26rft.auinit1%253DK.%26rft.volume%253D124%26rft.issue%253D17%26rft.spage%253D2891%26rft.epage%253D2896%26rft.atitle%253DThe%2BAurora%2Bkinase%2BIpl1%2Bis%2Bnecessary%2Bfor%2Bspindle%2Bpole%2Bbody%2Bcohesion%2Bduring%2Bbudding%2Byeast%2Bmeiosis%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.086652%26rft_id%253Dinfo%253Apmid%252F21878496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjEyNC8xNy8yODkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.36" data-doi="10.1128/MCB.20.1.286-298.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenfell</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erikson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Essential function of the polo box of Cdc5 in subcellular localization and induction of cytokinetic structures.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">298</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.20.1.286-298.2000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSong%26rft.auinit1%253DS.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D286%26rft.epage%253D298%26rft.atitle%253DEssential%2BFunction%2Bof%2Bthe%2BPolo%2BBox%2Bof%2BCdc5%2Bin%2BSubcellular%2BLocalization%2Band%2BInduction%2Bof%2BCytokinetic%2BStructures%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.1.286-298.2000%26rft_id%253Dinfo%253Apmid%252F10594031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjIwLzEvMjg2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.37" data-doi="10.1038/ncb2549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tkach</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yimit</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dissecting DNA damage response pathways by analysing protein localization and abundance changes during DNA replication stress.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncb2549</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DTkach%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D966%26rft.epage%253D976%26rft.atitle%253DDissecting%2BDNA%2Bdamage%2Bresponse%2Bpathways%2Bby%2Banalysing%2Bprotein%2Blocalization%2Band%2Babundance%2Bchanges%2Bduring%2BDNA%2Breplication%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2549%26rft_id%253Dinfo%253Apmid%252F22842922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22842922&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308668600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.38" data-doi="10.1534/g3.117.040220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usaj</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanderSluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> The CellMap.org: A Web-Accessible Database for Visualizing and Mining the Global Yeast Genetic Interaction Network. G3 (Bethesda) 7: 1539–1549 p.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.040220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.040220%26rft_id%253Dinfo%253Apmid%252F28325812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNS8xNTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.39" data-doi="10.1083/jcb.114.4.745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goetsch</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baum</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">MPS1 and MPS2: novel yeast genes defining distinct steps of spindle pole body duplication.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">754</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.114.4.745</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DWiney%26rft.auinit1%253DM.%26rft.volume%253D114%26rft.issue%253D4%26rft.spage%253D745%26rft.epage%253D754%26rft.atitle%253DMPS1%2Band%2BMPS2%253A%2Bnovel%2Byeast%2Bgenes%2Bdefining%2Bdistinct%2Bsteps%2Bof%2Bspindle%2Bpole%2Bbody%2Bduplication%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.114.4.745%26rft_id%253Dinfo%253Apmid%252F1869587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjExNC80Lzc0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.40" data-doi="10.1534/genetics.110.119149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Witkin</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friederichs</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen-Fix</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaspersen</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Changes in the nuclear envelope environment affect spindle pole body duplication in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.119149</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119149%26rft_id%253Dinfo%253Apmid%252F20713690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzMvODY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.41" data-doi="10.1091/mbc.E07-06-0536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakajima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westermann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.-S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A protein interaction map of the mitotic spindle.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">3800</span>–<span class="cit-lpage">3809</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.e07-06-0536</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-06-0536%26rft_id%253Dinfo%253Apmid%252F17634282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMC8zODAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2183.42" data-doi="10.1534/g3.116.032342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zackrisson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ottosson</span>  <span class="cit-name-given-names">L.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Parada</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Scan-o-matic: High-Resolution Microbial Phenomics at a Massive Scale.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3003</span>–<span class="cit-lpage">3014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032342%26rft_id%253Dinfo%253Apmid%252F27371952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8zMDAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2183.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2171" title="Quantitative Comparison of the Anterior-Posterior Patterning System in the Embryos of Five Drosophila Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2195" title="Single-Copy Knock-In Loci for Defined Gene Expression in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2183" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-y8vcgj85tJoQn_6lJGR_ouP6uMepWcem4GJ59WesJuw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2183" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2183" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2183" class="active" data-icon-position="" data-hide-link-title="0">Synthetic Physical Interactions with the Yeast Centrosome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493671" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9_OB0CpRTc1u5i2C_XSbRy6SODIzjVP0GwdKXLsomP0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2183.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2183" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-PmI6BCzVDE_qV-YteRabjEx4tr5bKdQ-9rpl11s0EZ0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2183" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2183.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493671" id="node493671--2" data-pisa="ggg;9/7/2183" data-pisa-master="ggg;g3.119.400117" data-apath="/ggg/9/7/2183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Synthetic Physical Interactions with the Yeast Centrosome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2479-180X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rowan S. M. Howell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2919-5601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Attila Csikász-Nagy</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1649-6816" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter H. Thorpe</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2183-2194;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400117 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493671--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rowan S. M. Howell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Francis Crick Institute, London, NW1 1AT UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rowan%2BS.%2BM.%2BHowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howell%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARowan%2BS.%2BM.%2BHowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2479-180X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rowan S. M. Howell</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Attila Csikász-Nagy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Information Technology and Bionics, Pázmány Péter Catholic University, Budapest, 1083 Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Attila%2BCsik%C3%A1sz-Nagy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Csik%C3%A1sz-Nagy%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAttila%2BCsik%25C3%25A1sz-Nagy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2919-5601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Attila Csikász-Nagy</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.thorpe@qmul.ac.uk" class="" data-icon-position="" data-hide-link-title="0">p.thorpe@qmul.ac.uk</a>
<a href="mailto:attila.csikasz-nagy@kcl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">attila.csikasz-nagy@kcl.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter H. Thorpe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Biological and Chemical Sciences, Queen Mary University, London, E1 4NS UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BH.%2BThorpe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thorpe%20PH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BH.%2BThorpe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1649-6816" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter H. Thorpe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.thorpe@qmul.ac.uk" class="" data-icon-position="" data-hide-link-title="0">p.thorpe@qmul.ac.uk</a>
<a href="mailto:attila.csikasz-nagy@kcl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">attila.csikasz-nagy@kcl.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2183" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2183&amp;title=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2183&amp;title=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2183&amp;title=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2183&amp;text=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2183&amp;title=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2183&amp;t=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2183&amp;title=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2183&amp;title=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493671/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493671" id="node493671--3" data-pisa="ggg;9/7/2183" data-pisa-master="ggg;g3.119.400117" data-apath="/ggg/9/7/2183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Synthetic Physical Interactions with the Yeast Centrosome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2479-180X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rowan S. M. Howell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2919-5601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Attila Csikász-Nagy</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1649-6816" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter H. Thorpe</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2183-2194;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400117 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493671--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rowan S. M. Howell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Francis Crick Institute, London, NW1 1AT UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rowan%2BS.%2BM.%2BHowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howell%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARowan%2BS.%2BM.%2BHowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2479-180X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rowan S. M. Howell</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Attila Csikász-Nagy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Randall Division of Cell and Molecular Biophysics, King’s College, London, SE1 1UL UK</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Information Technology and Bionics, Pázmány Péter Catholic University, Budapest, 1083 Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Attila%2BCsik%C3%A1sz-Nagy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Csik%C3%A1sz-Nagy%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAttila%2BCsik%25C3%25A1sz-Nagy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2919-5601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Attila Csikász-Nagy</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.thorpe@qmul.ac.uk" class="" data-icon-position="" data-hide-link-title="0">p.thorpe@qmul.ac.uk</a>
<a href="mailto:attila.csikasz-nagy@kcl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">attila.csikasz-nagy@kcl.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter H. Thorpe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Biological and Chemical Sciences, Queen Mary University, London, E1 4NS UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BH.%2BThorpe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thorpe%20PH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BH.%2BThorpe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1649-6816" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter H. Thorpe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.thorpe@qmul.ac.uk" class="" data-icon-position="" data-hide-link-title="0">p.thorpe@qmul.ac.uk</a>
<a href="mailto:attila.csikasz-nagy@kcl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">attila.csikasz-nagy@kcl.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2183&amp;rft.epage=2194&amp;rft.atitle=Synthetic%20Physical%20Interactions%20with%20the%20Yeast%20Centrosome&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Howell%2C+Rowan+S.+M.&amp;rft.au=Csik%C3%A1sz-Nagy%2C+Attila&amp;rft.au=Thorpe%2C+Peter+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493671/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493671/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493671/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493671/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493671/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493671/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493671/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493671/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493671/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493671/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493671/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493671/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2183" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2183.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2183.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2183.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
