<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2153" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2153.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2153.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2153.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2153.atom" />
<meta name="HW.pisa" content="ggg;9/7/2153" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.]" />
<meta name="DC.Identifier" content="10.1534/g3.119.400093" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Happ et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Obtaining genome-wide genotype information for millions of SNPs in soybean [ Glycine max (L.) Merr.] often involves completely resequencing a line at 5X or greater coverage. Currently, hundreds of soybean lines have been resequenced at high depth levels with their data deposited in the NCBI Short Read Archive. This publicly available dataset may be leveraged as an imputation reference panel in combination with skim (low coverage) sequencing of new soybean genotypes to economically obtain high-density SNP information. Ninety-nine soybean lines resequenced at an average of 17.1X were used to generate a reference panel, with over 10 million SNPs called using GATK’s Haplotype Caller tool. Whole genome resequencing at approximately 1X depth was performed on 114 previously ungenotyped experimental soybean lines. Coverages down to 0.1X were analyzed by randomly subsetting raw reads from the original 1X sequence data. SNPs discovered in the reference panel were genotyped in the experimental lines after aligning to the soybean reference genome, and missing markers imputed using Beagle 4.1. Sequencing depth of the experimental lines could be reduced to 0.3X while still retaining an accuracy of 97.8%. Accuracy was inversely related to minor allele frequency, and highly correlated with marker linkage disequilibrium. The high accuracy of skim sequencing combined with imputation provides a low cost method for obtaining dense genotypic information that can be used for various genomics applications in soybean." />
<meta name="DC.Contributor" content="Mary M. Happ" />
<meta name="DC.Contributor" content="Haichuan Wang" />
<meta name="DC.Contributor" content="George L. Graef" />
<meta name="DC.Contributor" content="David L. Hyten" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.]" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Obtaining genome-wide genotype information for millions of SNPs in soybean [&lt;i&gt;Glycine max&lt;/i&gt; (L.) Merr.] often involves completely resequencing a line at 5X or greater coverage. Currently, hundreds of soybean lines have been resequenced at high depth levels with their data deposited in the NCBI Short Read Archive. This publicly available dataset may be leveraged as an imputation reference panel in combination with skim (low coverage) sequencing of new soybean genotypes to economically obtain high-density SNP information. Ninety-nine soybean lines resequenced at an average of 17.1X were used to generate a reference panel, with over 10 million SNPs called using GATK’s Haplotype Caller tool. Whole genome resequencing at approximately 1X depth was performed on 114 previously ungenotyped experimental soybean lines. Coverages down to 0.1X were analyzed by randomly subsetting raw reads from the original 1X sequence data. SNPs discovered in the reference panel were genotyped in the experimental lines after aligning to the soybean reference genome, and missing markers imputed using Beagle 4.1. Sequencing depth of the experimental lines could be reduced to 0.3X while still retaining an accuracy of 97.8%. Accuracy was inversely related to minor allele frequency, and highly correlated with marker linkage disequilibrium. The high accuracy of skim sequencing combined with imputation provides a low cost method for obtaining dense genotypic information that can be used for various genomics applications in soybean.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2153" />
<meta name="citation_id" content="9/7/2153" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2153" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2153.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2153.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2153.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400093" />
<meta name="citation_pmid" content="31072870" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2153" />
<meta name="citation_lastpage" content="2160" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mary M. Happ" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE 68503" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5897-2617" />
<meta name="citation_author" content="Haichuan Wang" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE 68503" />
<meta name="citation_author" content="George L. Graef" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE 68503" />
<meta name="citation_author" content="David L. Hyten" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE 68503" />
<meta name="citation_author_email" content="david.hyten@unl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6324-9389" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. G. Ardlie;citation_author=K. L. Lunetta;citation_author=M. Seielstad;citation_title=Testing for population subdivision and association in four case-control studies.;citation_pages=304-311;citation_volume=71;citation_year=2002;citation_issue=2;citation_pmid=12096349;citation_doi=10.1086/341719" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=W. Astle;citation_author=D. J. Balding;citation_title=Population Structure and Cryptic Relatedness in Genetic Association Studies.;citation_pages=451-471;citation_volume=24;citation_year=2009;citation_doi=10.1214/09-STS307" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. E. Bayer;citation_author=P. Ruperao;citation_author=A. S. Mason;citation_author=J. Stiller;citation_author=C.-K. K. Chan;citation_title=High-Resolution Skim Genotyping by Sequencing Reveals the Distribution of Crossovers and Gene Conversions in Cicer Arietinum and Brassica Napus.;citation_pages=1039-1047;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2488-y" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=S. A. Boison;citation_author=D. J. A. Santos;citation_author=A. H. T. Utsunomiya;citation_author=R. Carvalheiro;citation_author=H. H. R. Neves;citation_title=Strategies for Single Nucleotide Polymorphism (SNP) Genotyping to Enhance Genotype Imputation in Gyr (Bos Indicus) Dairy Cattle: Comparison of Commercially Available SNP Chips.;citation_pages=4969-4989;citation_volume=98;citation_year=2015;citation_doi=10.3168/jds.2014-9213" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A Flexible Trimmer for Illumina Sequence Data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype Imputation with Millions of Reference Samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=F. Cericola;citation_author=I. Lenk;citation_author=D. Fè;citation_author=S. Byrne;citation_author=C. S. Jensen;citation_title=Optimized Use of Low-Depth Genotyping-by-Sequencing for Genomic Prediction Among Multi-Parental Family Pools and Single Plants in Perennial Ryegrass (Lolium Perenne L.).;citation_pages=369;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00369" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. W. Chan;citation_author=M. T. Hamblin;citation_author=J. L. Jannink;citation_title=Evaluating Imputation Algorithms for Low-Depth Genotyping-By-Sequencing (GBS) Data.;citation_pages=e0160733;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0160733" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Environ. Biotechnol.;citation_author=Y. S. Chung;citation_author=S. C. Choi;citation_author=T.-H. Jun;citation_author=C. Kim;citation_title=Genotyping-by-Sequencing: A Promising Tool for Plant Genetics Research and Breeding.;citation_pages=425-431;citation_volume=58;citation_year=2017;citation_doi=10.1007/s13580-017-0297-8" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The Variant Call Format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Huang;citation_author=C. Raghavan;citation_author=R. Mauleon;citation_author=K. W. Broman;citation_author=H. Leung;citation_title=Efficient Imputation of Missing Markers in Low-Coverage Genotyping-by-Sequencing Data from Multi-Parental Crosses.;citation_volume=197;citation_year=2014;citation_pmid=24583583;citation_doi=10.1534/genetics.113.158014" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=T. R. Gaunt;citation_author=S. Rodríguez;citation_author=I. N. Day;citation_title=Cubic exact solutions for the estimation of pairwise haplotype frequencies: implications for linkage disequilibrium analyses and a web tool 'CubeX'.;citation_pages=428-428;citation_volume=8;citation_year=2007;citation_pmid=17980034;citation_doi=10.1186/1471-2105-8-428" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. T. Hamblin;citation_author=E. S. Buckler;citation_author=J.-L. Jannink;citation_title=Population genetics of genomics-based crop improvement methods.;citation_pages=98-106;citation_volume=27;citation_year=2011;citation_issue=3;citation_pmid=21227531;citation_doi=10.1016/j.tig.2010.12.003" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. He;citation_author=Y. Zhao;citation_author=M. F. Mette;citation_author=R. Bothe;citation_author=E. Ebmeyer;citation_title=Prospects and Limits of Marker Imputation in Quantitative Genetic Studies in European Elite Wheat (Triticum Aestivum L.).;citation_pages=168;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1366-y" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=R. Babu;citation_author=G. de los Campos;citation_title=Factors Affecting the Accuracy of Genotype Imputation in Populations from Several Maize Breeding Programs.;citation_pages=654;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.07.0358" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. N. Hirschhorn;citation_author=M. J. Daly;citation_title=Genome-wide association studies for common diseases and complex traits.;citation_pages=95-108;citation_volume=6;citation_year=2005;citation_issue=2;citation_pmid=15716906;citation_doi=10.1038/nrm1583" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=B. Howie;citation_author=J. Marchini;citation_author=M. Stephens;citation_title=Genotype Imputation with Thousands of Genomes.;citation_pages=457-470;citation_volume=1;citation_year=2011;citation_pmid=22384356;citation_doi=10.1534/g3.111.001198" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Huang;citation_author=C. Raghavan;citation_author=R. Mauleon;citation_author=K. W. Broman;citation_author=H. Leung;citation_title=Efficient Imputation of Missing Markers in Low-Coverage Genotyping-by-Sequencing Data from Multi-Parental Crosses.;citation_volume=197;citation_year=2014;citation_pmid=24583583;citation_doi=10.1534/genetics.113.158014" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Huang;citation_author=C. Wang;citation_author=N. A. Rosenberg;citation_title=The relationship between imputation error and statistical power in genetic association studies in diverse populations.;citation_pages=692-698;citation_volume=85;citation_year=2009;citation_issue=5;citation_pmid=19853241;citation_doi=10.1016/j.ajhg.2009.09.017" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=X. Wei;citation_author=T. Sang;citation_author=Q. Zhao;citation_author=Q. Feng;citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces.;citation_pages=961-967;citation_volume=42;citation_year=2010;citation_issue=11;citation_pmid=20972439;citation_doi=10.1038/ng.695" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. K. Jain;citation_author=P. L. Workman;citation_title=Generalized F-statistics and the theory of inbreeding and selection.;citation_pages=674-678;citation_volume=214;citation_year=1967;citation_issue=5089;citation_pmid=6049073;citation_doi=10.1038/214674a0" />
<meta name="citation_reference" content="citation_journal_title=Soybean Genet. Newsl.;citation_author=P. Keim;citation_title=A Rapid Protocol for Isolating Soybean DNA;citation_pages=150-152;citation_volume=15;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=M. Knapp;citation_author=M. Stiller;citation_author=M. Meyer;citation_title=Generating barcoded libraries for multiplex high-throughput sequencing.;citation_pages=155-170;citation_volume=840;citation_year=2012;citation_pmid=22237533;citation_doi=10.1007/978-1-61779-516-9_19" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=I. Kozarewa;citation_author=D. J. Turner;citation_title=Amplification-free library preparation for paired-end Illumina sequencing.;citation_pages=257-266;citation_volume=733;citation_year=2011;citation_pmid=21431776;citation_doi=10.1007/978-1-61779-089-8_18" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=F. Li;citation_author=J. Xie;citation_author=X. Zhu;citation_author=X. Wang;citation_author=Y. Zhao;citation_title=Genetic Basis Underlying Correlations Among Growth Duration and Yield Traits Revealed by GWAS in Rice (Oryza Sativa L.).;citation_pages=650;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00650" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic Selection in Plant Breeding: Knowledge and Prospects.;citation_pages=77-123;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=P. Ma;citation_author=R. F. Brøndum;citation_author=Q. Zhang;citation_author=M. S. Lund;citation_author=G. Su;citation_title=Comparison of Different Methods for Imputing Genome-Wide Marker Genotypes in Swedish and Finnish Red Cattle.;citation_pages=4666-4677;citation_volume=96;citation_year=2013;citation_doi=10.3168/jds.2012-6316" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: A MapReduce Framework for Analyzing next-Generation DNA Sequencing Data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Pasaniuc;citation_author=N. Rohland;citation_author=P. J. McLaren;citation_author=K. Garimella;citation_author=N. Zaitlen;citation_title=Extremely low-coverage sequencing and imputation increases power for genome-wide association studies.;citation_pages=631-635;citation_volume=44;citation_year=2012;citation_issue=6;citation_pmid=22610117;citation_doi=10.1038/ng.2283" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=G. Patil;citation_author=T. Do;citation_author=T. D. Vuong;citation_author=B. Valliyodan;citation_author=J.-D. Lee;citation_title=Genomic-Assisted Haplotype Analysis and the Development of High-Throughput SNP Markers for Salinity Tolerance in Soybean.;citation_pages=19199;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep19199" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y.-F. Pei;citation_author=J. Li;citation_author=L. Zhang;citation_author=C. J. Papasian;citation_author=H.-W. Deng;citation_title=Analyses and comparison of accuracy of different genotype imputation methods.;citation_pages=e3551-e3551;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18958166;citation_doi=10.1371/journal.pone.0003551" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=R. Poplin;citation_author=V. Ruano-Rubio;citation_author=M. A. DePristo;citation_author=T. J. Fennell;citation_author=M. O. Carneiro;citation_title=Scaling Accurate Genetic Variant Discovery to Tens of Thousands of Samples.;citation_year=2017;citation_doi=10.1101/201178" />
<meta name="citation_reference" content="Purcell, S., S. S. Cherny, and P. C. Sham, 2003 “Genetic Power Calculator: Design of Linkage and Association Genetic Mapping Studies of Complex Traits.” BIOINFORMATICS APPLICATIONS NOTE. Vol. 19. http://svn.donarmstrong.com/don/trunk/projects/research/linkage/papers/genetic_power_calculator_purcell_sham_bioinfor_19_1_149_2003_pmid_12499305.pdf. 10.1093/bioinformatics/19.1.149" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=A. Rasheed;citation_author=Y. Hao;citation_author=X. Xia;citation_author=A. Khan;citation_author=Y. Xu;citation_title=Crop Breeding Chips and Genotyping Platforms: Progress, Challenges, and Perspectives.;citation_pages=1047-1064;citation_volume=10;citation_year=2017;citation_doi=10.1016/j.molp.2017.06.008" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Q. Song;citation_title=Fingerprinting Soybean Germplasm and Its Utility in Genomic Research.;citation_volume=5;citation_year=2015;citation_pmid=26224783;citation_doi=10.1534/g3.115.019000" />
<meta name="citation_reference" content="citation_journal_title=PLANT GENOME;citation_author=Kelly. Swarts;citation_author=Huihui. Li;citation_author=J. Alberto. Romero Navarro;citation_author=Dong. An;citation_author=Maria. Cinta Romay;citation_author=Sarah. Hearne;citation_author=Charlotte. Acharya;citation_pages=1;citation_volume=7;citation_year=2014;citation_doi=10.3835/plantgenome2014.05.0023" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_author=R. C. Team;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_author=P. J. Brown;citation_author=H. Hung;citation_author=Q. Sun;citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population.;citation_pages=159-162;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217756;citation_doi=10.1038/ng.746" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=B. Valliyodan;citation_author=D. Qiu;citation_author=G. Patil;citation_author=P. Zeng;citation_author=J. Huang;citation_title=Landscape of Genomic Diversity and Trait Discovery in Soybean.;citation_pages=23598;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep23598" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Wimmer;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=C.-C. Schön;citation_title=Synbreed: A Framework for the Analysis of Genomic Prediction Data Using R.;citation_volume=28;citation_year=2012;citation_pmid=22689388;citation_doi=10.1093/bioinformatics/bts335" />
<meta name="citation_reference" content="citation_journal_title=Annual review of public health;citation_journal_abbrev=Annu Rev Public Health;citation_author=John S. Witte;citation_title=Genome-wide association studies and beyond.;citation_pages=9-20 4 p following 20;citation_volume=31;citation_year=2010;citation_pmid=20235850;citation_doi=10.1146/annurev.publhealth.012809.103723" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Z. Zhou;citation_author=Y. Jiang;citation_author=Z. Wang;citation_author=Z. Gou;citation_author=J. Lyu;citation_title=Resequencing 302 Wild and Cultivated Accessions Identifies Genes Related to Domestication and Improvement in Soybean.;citation_pages=408-414;citation_volume=33;citation_year=2015;citation_pmid=25643055;citation_doi=10.1038/nbt.3096" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.]" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="Obtaining genome-wide genotype information for millions of SNPs in soybean [ Glycine max (L.) Merr.] often involves completely resequencing a line at 5X or greater coverage. Currently, hundreds of soybean lines have been resequenced at high depth levels with their data deposited in the NCBI Short Read Archive. This publicly available dataset may be leveraged as an imputation reference panel in combination with skim (low coverage) sequencing of new soybean genotypes to economically obtain high-density SNP information. Ninety-nine soybean lines resequenced at an average of 17.1X were used to generate a reference panel, with over 10 million SNPs called using GATK’s Haplotype Caller tool. Whole genome resequencing at approximately 1X depth was performed on 114 previously ungenotyped experimental soybean lines. Coverages down to 0.1X were analyzed by randomly subsetting raw reads from the original 1X sequence data. SNPs discovered in the reference panel were genotyped in the experimental lines after aligning to the soybean reference genome, and missing markers imputed using Beagle 4.1. Sequencing depth of the experimental lines could be reduced to 0.3X while still retaining an accuracy of 97.8%. Accuracy was inversely related to minor allele frequency, and highly correlated with marker linkage disequilibrium. The high accuracy of skim sequencing combined with imputation provides a low cost method for obtaining dense genotypic information that can be used for various genomics applications in soybean." />
<meta name="og-title" property="og:title" content="Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.]" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2153" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Obtaining genome-wide genotype information for millions of SNPs in soybean [ Glycine max (L.) Merr.] often involves completely resequencing a line at 5X or greater coverage. Currently, hundreds of soybean lines have been resequenced at high depth levels with their data deposited in the NCBI Short Read Archive. This publicly available dataset may be leveraged as an imputation reference panel in combination with skim (low coverage) sequencing of new soybean genotypes to economically obtain high-density SNP information. Ninety-nine soybean lines resequenced at an average of 17.1X were used to generate a reference panel, with over 10 million SNPs called using GATK’s Haplotype Caller tool. Whole genome resequencing at approximately 1X depth was performed on 114 previously ungenotyped experimental soybean lines. Coverages down to 0.1X were analyzed by randomly subsetting raw reads from the original 1X sequence data. SNPs discovered in the reference panel were genotyped in the experimental lines after aligning to the soybean reference genome, and missing markers imputed using Beagle 4.1. Sequencing depth of the experimental lines could be reduced to 0.3X while still retaining an accuracy of 97.8%. Accuracy was inversely related to minor allele frequency, and highly correlated with marker linkage disequilibrium. The high accuracy of skim sequencing combined with imputation provides a low cost method for obtaining dense genotypic information that can be used for various genomics applications in soybean." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.] | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DXSzbdBSYm4su3B9gLTL7X_48Fkvz-cwFZ9-v1BCQtw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493461","apath":"\/ggg\/9\/7\/2153.atom","pisa":"ggg;9\/7\/2153","ac":{"\/ggg\/9\/7\/2153.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2153.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2153"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2153"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2153":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LB4Qbyi5ckxofdPTuraug7TX5pOvGTYtC25SRdMrLCYGtFenyHk5\\\\\\\/vSzTu10kJrSpYwJw\\\\\\\/qDdJFbQWm6IqQWmni4wjAaC3TCmZH3XdXideGKOJyAY\\\\\\\/JDwp5SFSboQZflYeNgC+hQlzvP44srQHsoIy4lwZhupZD8emcD80YdNFd2wVIj4MChQJ7FXiQk7574RVUO9VOrm+1SMonD7ZupYs8ku1v8ykOm4SVMUdcakbSDbJf6Ce1C6ZD5hR7KcbegLR5yYXsDUHWR+Wf5y31o+s0roCml8piJm8ev7LLKSD0BTk22dfoL1kL4ioiIjH7CpBhJY11bTaUzR\\\\\\\/MfWx\\\\\\\/fFnTQ+oNCtgzQqfCa6RP+wxHdUd6Rl2MZWr5HrRIscl8caUY9rxXLsB9EOFD+t5VIM\\\\\\\/Pxg+N7ywCydtodbzQ5XrV4uEnlzNRmEOhwAmB64\\\\\\\/LzauFR\\\\\\\/fyGTKs6ZLI\\\\\\\/NMtdG+b2JoWcKaTvwd3FPJ95o9RwY7JRPRAFxFFmVinft3quQJsYcYeAmXAq3pb7A1x+2u\\\\\\\/g6ZrqfXPZsStrmHHYd1Wuit0B51hz+or2HaPBygMfXmzOaR+ftARg+yGm4FirC1SbH9cYTpVlWSkhjlZgI0Y6yD0LPOjwC0UrNUrTOMTvRcB\\\\\\\/hP86GQA13c6mdzHBDjke\\\\\\\/0NLr\\\\\\\/WForVGZXfBg8JyZTikYAzVGMgcugjTMQYekpfgtxu90Wedn\\\\\\\/F2WI8pO3F4QA9CMdw0kTXMtP\\\\\\\/jwBGBeiAiqHszgn8D6eNpYTjd\\\\\\\/YV3r9tDh+OPvH3W6i5iEfODytOogB0MeqOzo4PShizZfPH4hWGFNRcnCqna0LJSGTjB6rHt7eedtqUwKoTARjJHmP9+dt03+pXxDJqCur+gqzETaAzNmzbUEzPPhgyRCEszPQ0sEhi4MU2+o\\\\\\\/y46CCF6ck5JW\\\\\\\/dKjcXruOighO+E2ep8kFobohMCPuPiK32i0ITV\\\\\\\/wWFm7IqmMUX5VAL9ReL9fVaREksVuIz1dxXa5Xfs29mVuDAfdxI5MJm\\\\\\\/7I20X6Onm5Vw3GtTZt1wBt9UCgRvc6WnbftREHnCzUh0R\\\\\\\/yPCCu8JaEL1A9PJUablhzazDr1KOthINpeoI\\\\\\\/kI9ZyAAHt+DIN58vyBP+KwRgzuh+WUyc8NZibLctuNO\\\\\\\/QT+oDjT40cTkTCgAui4EZOzn59KOFoctQU6ga4REFRAXa4uTjoeGyXG+wld7k+B3nu2PE8ZkKNuR+Q8hlE0R3uYxArd2VDJrSUFqw3Bp0gnGiBoxYQXRUzMF8j7olD61IGO\\\u0022,\\\u0022iv\\\u0022:\\\u0022xc5vmEi+YnYAGW4kmFGv3Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225512dfb079fb344582bd422630b974e8a3798edff8310ec65d17167961f2c6a9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022feYLcioRp2g9UEtK3VTUn4h5dFYJNQkvJ\\\\\\\/ER+5eE2VejBC1YwRpeMp3SXwj5qTjCw3Al\\\\\\\/FLk0bmbBtn5T4HHmgyid4UswvBu+HphSobqBoS0HEladMxO6Wo7pXxBTNNbjOBbEY8WcqAxAIRE+yLJyCCAbY7SKxAWAtGruyPzi2BPQgqjn1LrsvqR1dkxcdQjv\\\\\\\/0jPHZmvLk8g8JGHdSlHCx1+9FO+VzlFPFtZCB0dzX7tDxU+wI7ICMNlLN69Jlno19nchxR8lT\\\\\\\/kTwTZslmXdkrxhLztr4HIrjwhUt1kOesuoXYjkEJ7vvkAdSkVuWUxcBayES54Y0srlKRiYwiT56\\\\\\\/DI\\\\\\\/jIEflBMNiSoAk78LmT5AnRrxe1WvP1hqSW0lr8MFlSc7R2vltBlW3fmo7SMZVu8d\\\\\\\/YkxL7rf2vRLB2Np0E8oVwAy41KxB1jakUEqR0gS+531wwNRqxBjsizuIinnyhncP4DELFqtHZWlaPt14P+U9NPP6hePVQ7IioJ9tg4WHQ6ysTHqp1YUQYG4zpIrjXGD5lTXFDX1K2pbL6YeGtL4l1hmTHPMPjAxYugLQcPeiAXwqtUcFp7T8lw83z72oAP8kPxMKcbxagP9WUWlmZjEVN5gLnZ6zNAv4uKrBeyUjWPXYCzE6zI6xxIcdog0ygwwKeJ2+tVeWG0FhW0ED4oVH94yDS8Ld3iLMF+iib38S7tf5GC5ulczTSdkhHVE2kJ7muCsoqL72WysIKn0ylxeTAKSpfslG5e7yko4US22MQqkrajBQP5v55ViwqInFgYSWIRNwxGIWvc0kw4bCy0Ax9qrVtvhb8bIQdNbmC5W6Ww36Fe0diZMM94VDmjqPbxo+lK6lVRwj67S0++Dx8l6pTbn0GI9zCjKpGh4Xl2XOIFWBoJ8FwaksqoDAE2FSiQBFbIE3ScWYpxQp15jxU2r3YpAJGCvShskfyz0fy7weVSSUaRxsTWdwWlFoqr0LWYodVIqZiY7hqKBRkU4mEnqXrloo1P8y9uozAgmBBk10uR4f9sFVT8Fl0444+yO6y1zMt8fZCi0I2w5stMrHbXmobt\\\\\\\/Mb\\\\\\\/KXPpOIjSTUXS01mSnPUdUSeiLt\\\\\\\/eSIzYVWBB5tI8WQdfAC1v4D4NPf+cMpTtC0hlHof0FozaIUS9w+qGGrTJlJO7VlgKd660cY+M9O1eBf1P48kQ2704vl8\\\\\\\/MSz988iHJ3WOTHBV9Z4HvJTMsnaM4kG9z+J+STFGmXonY2YIm7oKe5IEaTmW\\\\\\\/EsZqS76pdEPTjM8rqg8nZAcfpk9jJsU0lL\\\\\\\/j49jVvNulb2wuBype1qlwc64UOc4kjkrGTSCjGJaR\\\\\\\/cRkq3G\\\\\\\/4M3ny5l906fJW5gRdvjTwiPPj\\\\\\\/VT951jEBQKWqwcEePqkcPhyHacxuxvuAtSAY3lkmpgr18EhO6WLAE2lv6TIpSl5v3VDh1YZVjf82j\\\\\\\/uL2Hu9xU4\\\\\\\/0YwrwtDplD2Ouh0RXjW9fHIsK3X1+3ZCLTxTlLzxptv+OgJygYD0jHVRKKrZT\\\\\\\/qHAvvxKJ+SZZLtOIfLSLuEZ08EakHWLkzDioZikY2axqFjtJD2MHgaXq0zoo4S7uUrL6sMPZ50sMfKZpXIlC82VM1\\\\\\\/Fw4Ma2p662mtbx0fQRuH2e8gtYr7iIDq5UgAocgrzix+Wvb2LI8AVLxkEHMgh4OMhSAvxTBLw+w2SyvZvA87jIRxIwc1cj2CE20Of\\\\\\\/FvtQ4yf1Nq1kzJr\\\\\\\/KLi6QhVeXnz\\\\\\\/2uK2\\\\\\\/7ie0geO\\\\\\\/yQ\\\\\\\/2bbZuoiIMdkudtxTUx435ub4W5mRcPdSELR5RwGe075RG5JiQktKKOnu494apJ8XRFq9sF0wx1nh\\\\\\\/2Lmb7ge4z1fLdbykgoh541\\\\\\\/pYOmzMDWJxkk5xgKZdEypZtV3LS+UE+HzW8FoAda4omX37CyFZeeW7Tb1olJTyUbd5ZKRfZqw7hcv1MzcZY0NAkDkg\\\\\\\/mDOjnYT1a0zaoMpzVPf\\\\\\\/8FfczfYauSRsFPo05TctFMeHn8qDYMyKRbiqAZIcNKC878k82tgl7JGn4M\\\\\\\/rSvjjHS9S2lmdKTASZzEf+r6SaH6s0zzi74AMXHW\\\\\\\/TxRYcWHhORRoZaFAsdUwVOilGwkFjw9ayGYTJcFRUeTNfxTH0Ymhfpl3T5Sy5Qpt5cSBezkDNoayusyy\\\\\\\/V37kFHQxFTMTqV50bw+oAH2LifJUo5sWDvbRhFrkZc66429xAB8l4N6kQzw4HFHEWbf1VVpNUw03W6C1sUMelw351OvsG2TaZ9K4cKTMnNY+5nI2Z2BzRSWSQ8V+qWMMlcXoWsDq6+6pfBG6691eBAXEHrRBmCFyjJW86Au3mvNNv3Rb69NIcwy\\\\\\\/tlU7Gs3JlNueFybRVftQXDKatIH1IpCR6M+suhCgFn\\\\\\\/1eANUvF69Dsc0tdK\\\\\\\/wrejqdvQy17upge7k72sSx+gU++zzm1Wfr5tTxjrZ0rxkbh9rnsqiQoqkYys6wHutICJl\\\\\\\/JlFwxLJS1xgYdHPKKWQfMnwKu\\\\\\\/B5c3ydFnvIDh8eaovbns9Optr4MH37fTKrBHxbm3jnzJAxrnJC6MP0rCw0Avp5iB9R9FLHyDu2uyr5Y+z4niTTOHPOIaAX\\\\\\\/\\\\\\\/ca8QvhZs37\\\\\\\/HrgG+RaSU3nFdLkQs4UHiP7MZAVIXUwHlTAWIVB2TPgRte\\\\\\\/VH07N\\\\\\\/Hgu0tVmPkJ4Saph1Tczu8VuJE0RNj14LbwE7OYqKOm1SKRZIEUu5OvDx5lQu68SuL18lxOUooSWHNGdWeGCh6CT62jeTN4WxvwOSflinTBFYDouQQGGLHEDWuCvEgD2st7gCCigVwCqTf1FXbH2ecPKnsWq\\\\\\\/+soaKzz6rIjJ0A41STwznMUzdz6Ipsk9nEX4fu9WNLDHUdwUgc1M7QuP+bQK8HnwcDc5aAu\\\\\\\/Pxo1xP3MMrl4fZ\\\\\\\/8BPa2ZmEHu13oCfRnfCIW0xA3h455UvWGSi9BHJ7u0PUhSPT8thlSgc\\\\\\\/oh0tsgh9uvl+Dxqp3iWbarRdaxjioithcwXgY5VExSqIX3U4tC5TumSXHbzzqBFIJ9RSSOiB9h0KrNc1CIFbX3jhLWacQ+XiAAJt3reL3Km1IBACVcLOzD5\\\\\\\/90JnB3IaG2nY8v12LhqGa1dw01z8gvHfgH\\\\\\\/IWu6SjKGq\\\\\\\/ngzw2qsdLIQufaVqAVgbelJtmEc82ponE4wmHQZfVizG7y5GpYJ1YIspSW4Jk2+Xut+nCCDTKnEI5JqaVtmgnB3DrVmg\\\\\\\/rtowPiNhHsGRmc038c6Hl2PtCusCs7hSc6PlToGzXzsCOx1LjOUpPItbCVd14sHtNWwf85\\\\\\\/ik0rb9yyx0gdqm+JPIlvaYTaUIQgZT0UgrO3lLyl2ZGb6GKalDgbDoU84KXEYGr1M1hBpKNJYhKTIh7Lt6xYHOgroHIwItBGOjWkurmSvULssdYqFmesf5sb5praLXMqItT4+wetgngaoenVrAtw4Srx7MnXM+iMfCkf9KNqUXC47PDPiP5YWId79Mp\\\\\\\/DLBtNW4WKdZ05OfPueGNc62+bCPR56fwPweEe8ukJ5LPF9dtdidVzDYwy8jKvuH4mzeOWzyLQOCqyoLOtBOgcfjAOKHEfMwUUwk37KgW1C1dxD4hDs2SeC41l68HMecD6FDHo+8qwJvFbDo6ndxF63PeK9UbRyC23io5jabBsVjMXUmSpSDN8dYk2ZdYOOjCgVloxhWQ9DnrrOe5ZtNVFmuB5WcUDOZwDA3N\\\\\\\/D3Fb8sJitQcSb7dp0cNZSDql2CirgOVI++BrTGuyYtOt0tNofHYrwiYa4F\\\\\\\/R25HpqnneGsC7LUv+FXutgtGh5NgK2nkiSyEXWBcLiqDDTg\\\\\\\/Av5oVWOcTg4bt3Y0ut2PDB6B8obO\\\\\\\/VaRM5IEyYBn0FneYa5LIW2ShJf\\\\\\\/8Zxh43WHwZcER77dGkj8yzBofmS+mlJrEVl7plVdPtrK+5NBlad610zq817tKdvNe9jvsyxSdWcFcbqe5zUQf8A8sMCEyumhqKNMa\\\\\\\/Molm\\\\\\\/z6nVNauFknaxNqJxfOc+T\\\\\\\/HUWwrLZxYxMimL1jgclmYxxY2Exo2WyaH3ZjGnDpelSGlDMBjfLb7eWOZvvaqBUzAIknYWK3cqs0B1T\\\\\\\/HLaACyWzzBaYTW65pwYcIFsVGrF\\\\\\\/ugSYRrnnaTvvFbE2LPhARc9FW\\\\\\\/UQPqLMgo310QwpcmP\\\\\\\/fwkOfRVN4aTmdwD8Avg+0ZrncMJR2HBc5srfBVOadkf2kEeXbviNxErcGmmqt3OGO0sPUKW6xvj+wSkAjY6khWpgt9FrD3TUWJ3\\\\\\\/roWimpG5Ry1TyFWCBJchCRNemPhda3Fzmp2Tgic94w0c67qgb+oY7k8ttmXHXU\\\\\\\/Gs6ur\\\\\\\/Grmr1B2meo0977ConojKB1SJVfvwvMSHCAb+bPNaPWg\\\\\\\/c+roCCD8kJ4JXBWLzp03hP2RmUTipA3X9C6sl0TkMjbsxd7wXqYidOd57QRXY6wblbMsFh48N9PmXYfYeCA7KL5x4+3wbWl3pv\\\\\\\/8c5\\\\\\\/\\\\\\\/bKtPoYK0RaUpRbF2s0BYsxi83xhJ0\\\\\\\/ajCaKDVSgWobAPBlMQcerOdvEhpW8MpXmT1MW++akWZSjqd+eb+OsmI5oUNlydpf2TV5MKQrK\\\\\\\/rAum7jZXvBESRb\\\\\\\/zHXW5\\\\\\\/7xxDSyXoCHIElHZEGlKVvJYe+YSSiQjuo76bnFFRYZ45kkqkPhGzCRFifQIXBYM9GuN4yfALFh+l\\\\\\\/AmHf9kTvAhomjpGsoxOc36uhYGtMnEGMPYPzj2lPgt9kN3MJ2nSWmHvR9hwFDvI7aUhQQjH4w5poHXtEX4QnC9YxxKyx+BLt5w\\\\\\\/bzKbv+G3tfx8CysVm5O+nLbtzw+Za8c0RN0i8IJwN+YkNZ9KM69by+i68znwkzcXCtt7NA5yqL+QEaOuCtx2yVDM46RR00G3weuvh+oWskv9J+z\\\\\\\/9qH6tMSVU2MiBw1AvDbhfv7HeS2PwwUqx8Tm+EHXAyUan5vp1BkJvPOlqqsnj7w83UQHYuNztYDYfblQIglveeQSqOMhlSAkOoXgu0\\\\\\\/1L2QLnJr\\\\\\\/DudaRgE5uurwDgubS42ahhwXnznTwERXLQ63+wVgjqhDPhjTc3qtqXp+R6A3\\\\\\\/2oEaiuE8mmOYdl+RF08Z97ZkkoVxuofTaLzv9Gay9F1ZGnES3BxcdGNJRSMrac5YmIt1aPzreqg9x9p7TZCTX4k64p0vEAf1tFFdrOpyo7zfjfjDl\\\\\\\/3bzZ\\\\\\\/e4nWas0PB\\\\\\\/Nw5kAdXZjNmfwVbihuBlhiO7d4hJTwgC4cpoYgZPx8oV184T\\\\\\\/U6Og95AD4CEH7B94ZF2XF+1HCoFeYC7tnAvXv2iIhRty8idc3dgKyKCJX7qoMTyg+b1jMjlJYvGw9oVI0wc0phP78DKoTgPk3oDgq8SSzZkyzbsc7tREkoAaxH6aC++1sXDcP4tzu8u3wjaXjyroo8ZhgXft7CQkE3ep8A75XBb2PRXFW2qgRLTMZ4Dk\\\\\\\/3Boy1yrun7pxfZy8RXoewsAjngxpVV8N0H5rKeMcuxO854xyPffifY0c5r+mOaHFGZiZ+n2CDr2UVfNqDwMbU56ksG6MRufyJiQZ8da1qcOzHRSDa8NIiQQPbAxmemKE2l+nhlmwXUnVxnN7wqWP2eOQeha8+asK7p\\\\\\\/bH5YGCw84vKDWAfkYUG9I\\\\\\\/XoESUcSbe04McmQ0O3g4RLBBkZ4XozLa8Dic7uZ5TztqN5OMsvyAVKnk3yaXeYMVBLfPljRkkYhOkVHztKoZBd2sd0NH6i5hHYrmfxSXwCiD\\\\\\\/iJUXYCHzmxM5sgayg3kiXiSbKM86yv3gD3o76\\\\\\\/GqAjjkU8AiJvAYGKBAsv9tY7g0tcfpDvf\\\\\\\/fn44TDS1nYF+eZ5jr1C5MODHD\\\\\\\/a\\\\\\\/1a9IJ2rFWMIWbcOHYAJYQxTVljoMBQySN+wioz8bvWN+uH07ME96EDZSwoY8\\\\\\\/+tmvACznMYNnoZXTTo84rBK+d2RB9KvZnNvcgEYvuGaNMOXt3m1rlZ85hsoEFI9m8dBzdk3m\\\\\\\/Uqwug0Nfo1tFz5s9nL0SAT7wtEn9ypxL1QG+O9Khx2vvpMb2MhP0ySpL8PDVZmlCZ1JDDHP+4DeLjh5b8LrPjP6zzaTB0g7t58Wt\\\\\\\/6prkxdp5D\\\\\\\/2uOsvEgOf8wIFkrhFHv0hoK4ejmT2Gk1+lXrQHCjKF3ukfHr4OUspqbJVCDxKH6I1UaPacey2YdAwKShd8g2iYMAmltB+A+wUAIbzKnyYYT8ddaG4kDCfb\\\\\\\/y3xZVDZ9LK+I+N4Ywcs7YVXlJGsONQYKU1qzzeM3WgMMaB3S0pska0HbvQOlJR+I5FLZhbY6E4pmBsCobGWzL5bg+g33qxkXYeTYcsJbjirk1UhjQHvL5rclaslWLADqG5dsqY8s81XmNXB4hMEsIisHhf\\\\\\\/+0VBQolZuj8ok264co1NKiJ+6u\\\\\\\/h2GybcdqdTX3z7MJ3AtiW3ew8fr6x1K+5vlBLNZtvfTX5i4RPcnpi+yT+qzxZmOYQYcQIwttQeeoFmRi1To1FQ85LCv8GSmxA4Ipmb4zNPegqF93djwzJ97bjv39M9MIU9MgHdf2AcyrymFHMtmTx\\\\\\\/d3KDhfB+bGu9i9rc7S1V6zfEQ\\\\\\\/c5Mki1vZy75+RS25rHFW3jpXJ0oanfpokk0p3jo9MAfvWHW27S1KpEOgr6jJzbp+3ehdw926zr7Vcoo2MBdU8I5DkwymrIVUXiM5QotVPye6qBmphubmd7R5ds2aeBSd0tMfpE3a4P0xN0NEGf+AGAHoIy5p\\\\\\\/vYcNxYr6OVqKwRgAmBs3FX3AYdnTt1mhM6m0iXQHeBF8EKKQ\\\\\\\/MeASH7Tj6JVZCIT8m+MqLcPFP5SxpgT6m1PMKhSzF7QBTd\\\\\\\/4MchWPwFmotTsS83gBcn9Vb1lNhoKQusir2S5DuRN4lrXhnHD\\\\\\\/cQpA5G6I2AVQ4sQxNpWlgcWilLLMHF48dqdpYcAoLqldSPJyq\\\\\\\/DfFyiLkioxQSAd62L3isysyRDfcwsN7PJKhSU8ep3zco\\\\\\\/z5BVxY7sVf6ipIB1M7qUjGH8\\\\\\\/4Y\\\\\\\/LSD+mOTvK19SFVzrHb85pGs7FCuyETIJ70iRs9NsP4XFDt0UeVEKHCj4+DYKFpGeXINsDkfjimfBTBhdN8\\\\\\\/nUOYQC4ndxvfU+U+hUfypZmo1I1S3PDs0GZ4EUVM8JxX9MJ2cg\\\\\\\/dnJDAk8MujxOp4Hsa5CmNG\\\\\\\/pquDp9S5yE4fq9pVH0PTe\\\\\\\/R8cTyOKA2+2p2vxRFUZmLX+K9GyxDWs\\\\\\\/Y7v\\\\\\\/XRbll4JQ3pq+7UW1vIjYIOYKj1rE7kR6YqvKVrqr4LwQd0qtsOJdZXx3PVkroCLnP\\\\\\\/OZtcbz4dRLod9Rc+62Z7BZcCqS6QcmT52yEDevdO6AQXBwGXMTlO+\\\\\\\/xFbL85UHsHLeZAZTJRFMkI+xaCWdZZOfvXcMLyr+cJhRq4i4ftzeXrvJHZDDetER\\\\\\\/5VkZkD3tevM9ZhrKIx0Mb7AsH8gW1a2UqnjYF1z1W3QPUzQWeKNP2bGZ6AbGpgo7WVl\\\\\\\/IMNKMZEetAoM3UxwYw8lLVKi9v7p6oMltPyPqV4XQTBZoSGa90iHyr54JKtZnPCUCoMT4SJLmrpNlKmdaQbM6Rnu0EWmdLJ+3nlABeDYLKeOcU5m\\\\\\\/DEhw0+n5kNdBDs7tpb3jfNF6D5MmfSuwK075USWQOkyRPJ5\\\\\\\/EYhyEaT\\\\\\\/CByVMPGY1huc0ryfcQGPTkwiA5WlbkS2DFVVvnFTxE\\\\\\\/XHbPHMLHqzP5lvE5NZ3JzEsAnYNFepxlEHYsiYfXl0AOrqaVxyiMOrzYkmbKtvn6inUJhfgBGoKHC4ldTIkpqTtnD6rHmS2hUitFK8H+ThBX9NLOy6ucS8yi27eA\\\\\\\/p\\\\\\\/P6qgP\\\\\\\/GpLiOd1mBpQelh94pAawrJK2\\\\\\\/ZSohW8QXXPNGzHNIlhWpNxBdxvQXriJjpFmT8PyVq+XLDSvwz8RVIXfnrQCPJpOpECyu3Z0Y4zmlyUnZkZTPrfXEr5v48RbeVksvuCHT6Wj\\\\\\\/EGmURgjSiRosPPqTlN3pZ8d68dDBRr\\\\\\\/fMxPJh7mCQ8qO1IiC2uijivD5QcNKZUUlybRonOfgRVt9VALm2bvnO0OGP0lbJL+\\\\\\\/JKU+NwV58BgS2ox7mebNcaBcdAHBm0VWq+LhnqUUFVh8aqnGbDdqQAS1mCvR7bv6HkS4KFbJOv0yAVZLNCsRikbAlChLOPXpqWgI8Nbt+xydo1dD2klxZEVevtpHett9vY0VvRCtmMPfxT1tnZgBIt81tb3DfDcMlQ98UbiuDBjA3bi3oG3N6lWBF16Q0DRxapWuJzfSZ+iibyC1xaaeEHORxCUtnqp4aJ9miGwhppANiUeelZP7si9o856EVpSpaymv09D2igVn6UDb7VHExcZndEE\\\\\\\/FOpuI4Ivcr2aSEVY5\\\\\\\/tWy08dtBKfbOOH546nITxlrh2VtYucLzquS54B55KXSB0ABP2bBoKL\\\\\\\/atsqXAKZmJHDmJVfK9giT9N9P2EnAgLupPbVj+5im6mWFTZdLyuahyVaxZasPIHkV84MgXILcyNsv7Y1A8ZBfnpnvpBgdolJ7jVs6sFCNdldttfIgMmNMaj6NYSzWFX2AcHOVA0BrrkcxyF8wXI1bAHMYiMXY6dlpwuX\\\\\\\/fi0oXZsG64bS+6d7fPAc1rd\\\\\\\/nJvC3nItuF5WkAez7ejQkc7uMM7O6FQBT8dVQpoByNH02CkUjb42v8GSNYYCpgOSZfOulBdK+7K3w3jUvBEGi9AZRqX2Z6c7eEaXdpeuxmJ9n+ecpWtfuJZkpdHL8DzK+Tf0f0kVItHS6flRp+nzuz679DbSDvLPhBzckYzriI5rZn8LtD8tQoaxazgcxMUvr5ggUtR061uEqqUkJ0d5749kqd4UdruWhEnexYbTyA0ynvgF3OQZ6BUO3eZi21VEQgLt5bempIApbRWY39ZnWiqYF9umiuBkCVvOomxaP44wHjjFHSZ2qTuFQfhwW\\\\\\\/gKL8uOxeByzp4wIAdTnh1YHkOo2vfVcwmu8gpedDB38co9AQBTfAGw3XHTXVGP5uIoEIcy6F7BC+fGBg1rKsxfT0AgX6sd3v0I7ll5UWGJb7nkq43e6P8btU\\\\\\\/ZkMC+WjCYuCF+ZG+oeYxFu7i8kxEbYPLU\\\\\\\/kp2v+ED2+kNN0t1syNigmubpwfIAYyjIQHquxL6iNY16seGuiyfUQLOp92EuLDpXGzKSUW6A9\\\\\\\/Ha9wio9B0YPaXzzvn2PnTIOg8FkaaSR0XpBMvW9lW4I8PYkjtqd\\\\\\\/E2BjfNv9aqrlLY1YNpDrCpFMJeiG8\\\\\\\/St+LBrAYO\\\\\\\/lA3szrhvX\\\\\\\/eGsvPoAYz1F+SS2M7MVcYF+uf8FH18qXOvyejCRnCEspL70ouuhj2uo8eKnpQOc1QASG1tvDWjtmjDR+66vtqmZQHzSYVxXXZZvnLxsIcjU\\\\\\\/zxcBXf2Ob5\\\\\\\/rhK9NigJjcNi\\\\\\\/tCIoSOMVwTDmJKu9uFRz+Twn1GyaQqXFOLztVZuKHvuYyoofCCy5WkNRQRwHYOnBWRT0GOhIHrearbatqFUyOTpw+3dNfNQ9g\\\\\\\/jPZ9zNcZYODf8QCDdBgpQswfcsDJZr\\\\\\\/BKjhcjI\\\\\\\/ORNC6lX9y9IFgGJlFOyHIzQNux\\\\\\\/04CazNR1YLlhXlvAQIN+au7b\\\\\\\/jAYwCKZq0GWsMAR5e2PFoBrlsVIa1K4aFTAlIZ0+hitRF1mImuygc\\\\\\\/UKiX2LzB7leCcA0aXGjyQKlIyh66iR3VcuWElbBriYcrHcpuLypdJ1GL+K5ZrGspnsLHg\\\\\\\/MJSUquGyFgzHPyIVuw6NYsJeLIr1J4uaQLcf2JUg6pLxDQJMsCQPLGIGOwtLacAXayVgVPflJ7lizWtKnPKzVRR5K18WPU3UYM8W7a4CeZwjI54mC\\\\\\\/ZB\\\\\\\/BeC5jY9jJnOsgKcYqkAYVrBdS9crwEx3QDoebdKR+3nWMBCoRJY6R\\\\\\\/xOQU8SX\\\\\\\/W6RZURv1ZdwzotwKvW3LC+qOjC63RH9Wg0BqTEsg5D0F4Kx4X0tn8p\\\\\\\/QbQuOJQwKEVkOSiH\\\\\\\/gkg7Jsu+U+EdT7Mle9siKH6pwDrq5QV+DGd7bvR2HvczeDHtq+JFbFL9MOM\\\\\\\/vgHvQW0gocvxALMQppTSr5ld8\\\\\\\/+QRBLAXy0CBmbXzM9BCqfJvNGXeac4KxdlTHNl13Nm7la7xNo42j2pz\\\\\\\/Cycp7unKSEJEEBOohvFz+\\\\\\\/1Xskf72Dxoo4ZJI2YCZmtlREF+iGottTqwQPqykn7v1AbtWYmO7OTy9GesC80xvssyClk2j9vtg1OtCyAPOiPus\\\\\\\/UCWx2I9XrP3+1jkzqctoDK616M+QbQa221zrF4YMw+KtyniQUcHg7QnF6nGe3EwF9IykrbFz1x0i4W3lcjIinvKAnKqT8XHCeC5YGESWmtFI1+d0WokAk51+vHTgbu\\\\\\\/9aydCsZColqlMTWFHi4z5GwHcmPjT1MldekOIIcu9\\\\\\\/xhZaAhAzzt913l14YdJ25p5GiIa7LI6y559VTIovvEfDDsWaql+TZo6fRM\\\\\\\/qJAPzRBNzYcp+GTe8UnWis\\\\\\\/lCuSjewzV953o7PYQKsN+ITWfrTc52SooypKzbPDfXnRCPIQLKoTmg\\\\\\\/yDaSWhmv1C6xzig35zYvN0fC5qEZRBOw9L0rJmPXP+3CWunWbpyxkn+r2PxvkbsPu0mXAlQzzMtNxXAMcNlmBMlxDnkIjuFxpVMr+hQh77GL1olWRL41\\\\\\\/7XM8ciomqZfJqr+Ze61eRUogbUG9OnyhpG9jEz\\\\\\\/eFvpchV72ADQSKAa6qcdx7dh6OC6u3W0nTdSVCw1niPkJYhrd\\\\\\\/3LR7J56dsFnviflVXyHbX+h5sKOt7Aa20hwfoBvxGLjHhS9pGzNTEAQkKauytxIW3QVZTmvnjG7035QYBtRxM0xey3GJBYv\\\\\\\/HWBVnFTGomubkrxY4oXXgsSlTkBlkLJw59+z3hNWYtetYd1oJNHPd5unjdi0Tr6Djm6me6P4NMPDFjdBOo\\\\\\\/SXWVYeroMdlbIwdQjjJ4jgZq74v0o5uh52BDLftNR866M+i70AzKKxgHyeyV2wDzIeMcDSqgNlCVY\\\\\\\/p1H314gGtxkd0CiaZvpbhBiGtV3JLoiM4N2LJc6lvYnj3XAZ0dQmI\\\\\\\/gE9h82Su4oCRhTzOBWjtMpne9U2+UzQp7JNsrKU7gdMsqH+CG3LTT7bi8wbzo0oANnghNxysF\\\\\\\/lcy0K6ascHFLN85Nt9veRV8v2IOJKbItssOxXyaT\\\\\\\/\\\\\\\/sATRT\\\\\\\/AKM66B1pBt\\\\\\\/F+BV2gmat4yOtROvDQNay99rKXJeQBvMOzKfopR5RiTJ9OBiPvJ8\\\\\\\/ObOmk3e1M0WA2RdgXS0CcUys4j0sTL8nHzQsdm+fXPa1qY8JNICOPV9oEvAT9jCPaEWpZC\\\\\\\/DfRfcAs0FmOeH5J91cGmpR2Z1JDBC39AvGWceIEKIvENe2Rk2GEnBpjaaWcIprbmUm5XmFDXILmYUSsAvkGuc6uFhpnmenqP2MFasQCLHUfEJSHxDqx0jRUpfuHWsnU3l3\\\\\\\/FSLjPV09JMhxT2jO6XzgG\\\\\\\/JTBogd46ZmK0W1lErNPkr91mLKIMV+yf5VcLk4IemYlRa0+J4y82HvYZeKeG1GnEBbOgsIaZq44M96BFsal349cmIpOzTjn2AR70MYqAW5NRVNBxZQ3dkOXV8YseKXHM83w5SKQiEtXYcF6CjAdkMs509L843e8D\\\\\\\/O6\\\\\\\/9L\\\\\\\/rqxb5E5fa8HW2rsS11Ixi64IZbjbSk\\\\\\\/nKWaE3gTx1jCzuzV80mq8kNQjgDZrYdUtHz0284IRnsrx0pimt+dCPBqJq9Mb0LAnGkpOtOE1GuR7ZlXIQ+KfH4O\\\\\\\/\\\\\\\/65F+maDdoAmPJAlH78\\\\\\\/ZNLJJXLJqTV0CBIy1hThO8aQQ2RJduotzn43B8SbWmx2rxc80db8n2YNg49776jmmkrcWjd1m746XjLIkZzdpZ+5++3HFt+VEvIBLoqklc6g+SjT44Hnf4uGGcVN3LppxNxTkxvtQYt+5OTbqRnUImG+4nzMI+2AaYhSDIWwE2UeT9ZCTIwxee9hnF8\\\\\\\/YhfR\\\\\\\/GoEFpDqtTVHzIMORLgjc\\\\\\\/rn8lRWQYzXpjnLIc5s1hJYPDX3kukp6vCy6toM3ZacWKOIj9RvBWPXMZT6iFKzkH7oS\\\\\\\/ATDntqCWxT1TEJT3\\\\\\\/8pXKYSMT6aVxhBGSDP215IHbOC90gl8Gu9bHVWecj8vaU2zmLQlz5hV71h9mhOOUAmjVaoNlNr+5Y50wxGw9reLJdwnd5kJjIwlaQt+gMWHORrSxmj\\\\\\\/emGkCBGS1fb9Zbffow7z+bKirQL9uQuuCzn3A1taEic6UOcjnPfscDeH4fU7MLEdV+boPSntDNdF1gR9wjvsXUu95D+lJ9uDuBj8P7UbWf9vltCMEXZukF8kMpFYEbq5FlZR7eSmxOiP6K5ulrFo\\\\\\\/bZOhVpp067TTNFcez316FwGNx22CZ0f+DaNxfToe5Kgz3qq\\\\\\\/3azKKFjqXlaRpvcfQrpTe38J7TI8QQoir+JhnxmoBgKIcbsf9tLxzpJ3\\\\\\\/iGqmDpa15+p4VXPzIzfd4hvWzFAPPkEuRrdwr06VW6wKd\\\\\\\/Q7+LberLf4jEjytYxcfjCvhLzOIRo2EljM2aR4cacO1cZyzTwjJY4i4OvysREEDv9Be8r3kBkI\\\\\\\/fegcVoYna8\\\\\\\/XtPi5q\\\\\\\/IGBWA7qnlLrmWwVxEq0s4eJzzfDGUTfjTznFZvPWxo3NOYgzR1hWqCghPI0EkdBZRgwyGOJRc3COXXSMNg13bZl1dQztxixYx1FdTqT0E2mxToJYRSU9mn5QdvphSoUAHx6odMxr7vYArykxTwVaX0brO92rq7471op855Pk0DEsz4d\\\\\\\/\\\\\\\/S4fBgAuOVb\\\\\\\/O7YogvwxD1qRlmxnkZf4QN6+\\\\\\\/CfiL6WVK5iCWf5HdPUT\\\\\\\/o7H+hOw3bSq8Z\\\\\\\/fkScangVjbELeGIv76cRBZ2vKLfj97E4itlcv+I4oQnKCl2PeQX1EkYl0XLKU\\\\\\\/PvXzq2sKCwXBhF2bR5UUmVxjcrXc1Iw+Q\\\\\\\/LTnMT6zGEmeRXgnxRCaUXi1uAakal1evMkgZItq6Lsf+AQBnD9BtSF+2LUmv2XbT0NAvUJlpuNGpAe6eFJfabU4eAGma9i+ZfeU9F9r\\\\\\\/D\\\\\\\/NdenZzX2iVhOPHOnogfPWD3P1khWZ0JwjSReaD1rX+a1gEpxbRZcA7UyzxRPTot3+uEAOO+0Gvyl0tBb4Qid+gWQ1ukzX\\\\\\\/wuV2LxSOO11nn6nnCocuAxWPZMsu4yt5uYLLnWyXcJ15qUdnTnAogTyxHPaW\\\\\\\/K3zFoJ4eKgL6EbHIDM4ivLdCC8z1xHHN0PzwP4mB6RIbEaW\\\\\\\/POFWhivjZIkCcUMLGwgmB0PheDooQvIWIrl9WT4S1HIrCu6FOtNrX1NDlxtnJFH9kr7FbD7Us7n9jm2bsG+RGSOOSEc8c9Qhk0FhyXPvLHYbv3xnC\\\\\\\/SpDi9eoM1dLM4VIYye281KgAGcDjUhVjurP2qoAtxDfMMTdlq6wgOZcU7TPJGOiL7M6RaXwwuxTmH2Sm+h725OLqcmBih0qhcK4NXqXyNxqHAtqMaHziT2FPVO7869cHMPBlUeDZpRvqQr7mD896yoPK7Q824PQVs+xkMeV672R0ijLRtmCJev4cEmG22uUtHqLjc7wx3243bCzw7KUOIsQ1ZqXtWslAYnBtkxmhZHbuAxvqpHBf5KDB5yNGufNPkdmvifRlGVdWR359kGhLlAe02p8Gc62\\\\\\\/1RI3dr1nm3kvR6fYHi7qh6hAUD6Ed6RLNjAT9pKkJadhNvac8bo0y3GzL0tS7ToWiuEqKtsYYb9TW3j9qFJwILWO1yLEABOrB+JG9P3+BtFMbHIOULMLZ8NkNrYmCZMK7hYb6XSQabFXqaqDRdYQ3KRW+Yvmqpsk02\\\\\\\/NiBeR8S4qhITO8IlSdkPBVD+aC7Sc0kNODBNACwGx2gN8+Eo9NOqdmdi6VlxLMhRRjlcbb2GoKMbIl2zAW0MHQGl1++8fP9VsMomxVst78hklKMlLVdM1Gts4\\\\\\\/C6ql5RY70wTwwP4LqcumqukhrVMigIz+0EGTLzcWXGCAifGP9zo4mcnrmBklGkYXr25vuW5pAlFWZWq0TKNwHSmwF4MjrgXyYS0i\\\\\\\/8rLRfIFpSJ\\\\\\\/6am2xPz1++F9ET2dvOm9ubL+Bg5FwlnchRgI\\\\\\\/wjPv6nvGs77yIEpgjk6lPnhCwtOz7AreWqh6pcG0dXKwGL0lAkMzovL7BZkl4TERjBlYw10JRmoF\\\\\\\/p1Gb\\\\\\\/9khML7QWmPvPM8GAT8F9VdpTchtRXlDWLzw7cWJQqfMXNr55KuvOliypriRL6tg0hsxfKYMSldXcUhHZlZs\\\\\\\/mjMtRwtST5lbgdY5SKKeBvZCOCAgj5VBAoCXHMW3+SGGqNYuKYZcW+H17jPH2k\\\\\\\/3\\\\\\\/9AC9qhwZuaNBLlR4N\\\\\\\/hCNl6rzn5AOnAgIWEgNKkjXLf5Xt+c3zr+yW8bdkXzBsc04SCU5maYBxmU42UkYmyr2DlE+HX8lKmUdoQTZ1LM+urb7gQR5K1rpdo6GahTfE35KAIoUn4HbbIiTJ8TzcHg70C4sjqqTIhigXKxDxqs8gGdYcOAW7NvRPYRbSZT7IkrfdkMcVQUIVkDXZcbA1nZA+sSFkiLdHz3PgAPE2TgE+vA5oWkoqx4VmaowGMJlG7V42Rw0OWJhoTK1ggJ2ywMtw0CtkTEKnK2XClosbn5B2HD+cQCawjvgVT0C\\\\\\\/rzGL4veG5GCb3bnV5v2vZo8mRuirid+NOY0gP4dy8LInCnRPPHa00VdfETwFe49rFe+reIJMPhvE7EqZioDdBuxn+SaXfrhqxHW9zEx+icvE4yH5\\\\\\\/6U1KREIzr7iqIhkYcv7yRPdyD1EeUJqmjW7ZELYuLvlwAB85W9CIVdwhCJMH5LRLQCSU7rWI\\\\\\\/UCbINeskdyf1yatCURR8j5tvGREv8vbCgZiJU00vUYlLaw8Oh7HJkaSuEaNlFrgQWKimu+2HiYu9KMsJ5klJYGC+irlGI97i58\\\\\\\/UHO9ogf9QL\\\\\\\/699Ok+0DoSqiONlmCyh98ICYhwAufmWSPj1ZTAu+kJ756Kr8RF4KtoEspYdYYI\\\\\\\/4NBr1TifFAebYWQomjiHLBb51su0edH8fSTuPoHxctl0HykFzuA2gDAa5vRrLnbPJdoRg\\\\\\\/MGLpnwXKX3uP39vOs7Mx74Z5XtgUDvn\\\\\\\/OU0lPyYZjJl2aTh\\\\\\\/vDmaI6n17YT\\\\\\\/w6HoRn+Dq3pam00f5NT4Cy\\\\\\\/zzz5NjiTwywLFGZBScay3bwbnEbZtkmKjjV+fafiPM9oekq2M6QlHO4tJ5SlOJ3FhLghEFTXIKM6qoIMlAWMXUcxPIPaMsRO2A2ksYkm+gSjL9pSANn2+BLCNab2zWvQa9KmM6jZfjYvMXIJR3u9bqok\\\\\\\/ZbleQMydzycKodX2PF0ch\\\\\\\/6P2KgBhz9ysa2x+eJFC7phI0SSvoJ4O\\\\\\\/xK+hswyJJO7czhIh+T3XDdQ81wzkyNqHxJH850VaNaLaHuzhHAA8I8QUjAE0KIuvkCMN\\\\\\\/0c3+rdfaYmFSfPr1w5fwBl25i\\\\\\\/AC9Jix4GjE944FmsIICH79fADvGVdNKGEYl7My4lZKKJuCteQuHOnni5tsXddNEo5Ryc8\\\\\\\/cRHNd6mTC67pbke4IJYVD5PyivV4qk\\\\\\\/dKiBg80Hr3PVw0hvqpKLDJO\\\\\\\/Yt\\\\\\\/160tfj2uNQNuErs8\\\\\\\/AmWLSztwQMvFgpCv7a7Z4GSxql+is488OxQciMj3XlEGXqnxFWYw7SISmgoSTEaAykH1FEwo\\\\\\\/2R8\\\\\\\/myEsPB9jB8vV0V++5ZU2wui9uUh\\\\\\\/gey9hiemWEg1sWmi2rhb6lqxeWud97p\\\\\\\/2E68iZk8HkXBxWwpUDb49iCY3hA\\\\\\\/Sq75eZrg1xvgPCrieGh+Ts7Z8vMJ9OZkH0oy08jshcL\\\\\\\/P9k5\\\\\\\/G0H6u7627IUnab3l4S1359accztCPnKFjAvPWW3su12u6Wd\\\\\\\/rOp6FmDYPeJZOUKTDNhwOvfXJVr5p6SfTyUSFuBAANNSBNIZGwXIUU5AKV9wmrHJeqIC\\\\\\\/fxwvvKAoSs1\\\\\\\/7e+gkGlggBNS5Dr7ySiRAJxJVp4xpeAr0qUIc1Q+rQgT4MWFUwNxNL6yAe1Kt\\\\\\\/lw7e9g\\\\\\\/2FXGta83iVxpL9a5YamHCCC+RbAwvVguXGjr5AAXY9WWgoDeVTJ2AxCwAZSrdzb64gWsuBdjwId0A3zKK9tAazSvyrAmw05n6JR6mdxohPvLIlFG7h05EDVhdhYhPTNgtQzTsSGFId7HYVB2rEFQJYhFVq0wYAvdmSzWCfJGI7kcGgtdS6AmpQyGdTZ2wjYlZW381Q\\\\\\\/Pu18VLcm\\\\\\\/2iZDWNT5RaTFaSh0sYC9yKrOKRplibAmvj\\\\\\\/h26qWSKzBmmtHfOe1BEYumUfIbwS8YTDySQru18fkFAhkQ7Q853zuOLQVUzxk2Mo3FUEv26r7fmz3QDdKSM\\\\\\\/YHSJGwo+goEdhZgTYsH7yBrnb6dMf+N1qUaTYJO0W2aPga15F5mxvZA0K35o0FtUYP3IzWTmyEKxLGWmg45n23y6uIo0Mh59i69\\\\\\\/ZwuRmJs5uDF4MB6V7kbsA1PxKiqMOPaVVCDRuScsEIcvw7BELhMruH75QHr8RveUYn+dCtutyBId5AZIdwqwqWwr1x11QJBvUKd1u0PE6J1fTz7dibYE85iMh9S\\\\\\\/rXr5KYQ1VtCMSwM9pbMTM9y0IYYdp1jpcehE6QH39fVlg6agdb2WBoa0vhA+DuknQVL4h3zWKh4MVQpayrLZRdeG6wyKO\\\\\\\/5QCFcoahs1tFj+6cqFva\\\\\\\/b1DJHfGxrsKv67NFyp5CBFQVmrRLUamHFurrmc\\\\\\\/VlnmrNzWUa0WDB33TLNfTv9P9Bdpto0mxyJIQwkLIV+\\\\\\\/mvXufP\\\\\\\/5bUPfE8qMvpgepLtTAnRO988+9hzVfZrdr6xlNoCncxtuwHShhCAOQuryVhs6GloouhwSkRXitkw11uY8b5+cS4I2sSD6OiL2mJsHz9k0o4ihbucTldRfnbrswcK03qsblC9mHLLf+Yr+OGkkQC85YU2\\\\\\\/6mSfDcqXSAd6\\\\\\\/yYyZJiV9+CtSvkvGFOWqHFA50CVcp0T5kXiPlMC4PUuQeCglPZW7RtiU2JBAGn0Rhr+qX6FU9OOSipgogpgEoRj5WSWeB1GMOjp7djYwxGkssD0zhkUQWjzChqg9j0HjY43cBoxqH\\\\\\\/a1mEgPhfFHVwdqMNEM4kLInVR71p\\\\\\\/CzOcBdgCWB7EGWevJNqfxe4a8VMh6kuoappGO\\\\\\\/peQocjiulSyFit8PO1UZM9ZbsO4mM9BPRTtQF6I2x8w9b9kMUm+miOZw5HbeY+VL4Sdmxb74hjHws+tXOU1n5EzCiiK6S8wdgHVYbrOaJJbw+RBx0XRRF\\\\\\\/xGo3nm\\\\\\\/0dkmTWB81tB7z2SkNe5PjI\\\\\\\/Wu9Sp3e4kxEli8\\\\\\\/HHpYRwKIv\\\\\\\/RSNbqux+ZAiDIOsvfSu8AO8zJHnurucPjRdfglBBIiJvLhv\\\\\\\/20Jlymlez2AhfiNijHFvPQUGj\\\\\\\/s9qU9Mq9jNplGfBcexDarQPZPqfFMuF6CEQnMt8FUr1DLrj4IAz45n+qvCeX0dG2AgMVa6WIQU46aYgw9qHuWLpew2S1Thvb2Vr1mT\\\\\\\/udbEFkaG\\\\\\\/lud5ASO35kUKXIajBjArUdbFazjL6xK1LdKo8Wgkh8FyipIxwCxrdApV+KoPx760l8hKjuX8+JlQgNdsIUr64khkDULL\\\\\\\/Z+WLn\\\\\\\/HqJxk56iN7xjQvf7QuMiKZ5n93FKRT0J934\\\\\\\/nF9j+WfPRzoWd4NCvJKbdssZJwUyhLRdiWK7F9sEGBA8HyK8n2SNOcBEamh\\\\\\\/B3qYb4srvGweay\\\\\\\/wUV3Xs368MxbwFcVaxNhS\\\\\\\/AIfHGVi9T28xte2HJLoMmLiLjjxZI8VizisU0TmQ1TwvIUsH+t+93852\\\\\\\/o0+LRYUMzPYpHweLqVwIOZKVtYrwuRfMpC3JhA4Hqgy6pqUW+AxcDPW+PkZckf6Ol7b8zZ0ecTn8KAdzpgSn6PE52FH7KQ1K4Mq7LQ6AxUU9ymse2XEzyySYp4nFZBaz88FfMrg7g\\\\\\\/dxrfXUxdc\\\\\\\/lSbZK+uMSRylBvJg\\\\\\\/0YTsdDK9qKC72DhVTtyG7IxUM7Z9T8BMfkU9D7dEEd9pNTJV8pM+\\\\\\\/CrN1mC9Dvyzk7Jt9wxqzBUVfXYOAqYTWtGrwyTrYbpu41RY0htv4WDQyepYjdHdIKTkf27p27sY21sCJsczFfosn+YxuXXSUCLJIrf3nEFUVByWSTzRE1+VZsoK+KlTVCt0qjNLElzrj0hdy3J7UJ68m01XFZwrZrLenlaKSvxPKKQwfCDJIzwJbn6gW\\\\\\\/8HU2rCfaDVz5fOL81D6pfAJb\\\\\\\/itx9ejcqznvfSLFhIT8dJTiDVL5JBtIcq8DT1fnXCEowMSHmgleRA9Dja\\\\\\\/jg3sxddLzUSEL5ZYWGlPmzx\\\\\\\/36mCSrbz466Oc8mOj2\\\\\\\/iz74+dA8wtzgEeN6LaYsraYF3yyLy2\\\\\\\/83O08ohmqQqbZQJhqYMeLpvPeZki8VLB\\\\\\\/Z++gtfr37rpzC2CwZA8tYaHP7NVIno5gXYGDwD+U+czyLT14oSPGgoB63mxESFjLE3vnLUDWDbHxrJAQYubv6cCQNk1ts28NWsgeruar2kJKZEME3pEYdcjuX4\\\\\\\/RX49hXS2eiCLNaKJxnpRxne9m17muNtc1QOdSSIEps\\\\\\\/4j390YK7Hyu9M7IOnPU5eiPhKr\\\\\\\/P+KcVe3jWLJUeQuHWMReoXRstl5\\\\\\\/KdPwft6gOyLWpAAmOjycQ2e4VZJ5rg1nhk9x7W+jkj5NxiGRsPqoQc1Z0gVtOqmv6nUeZCY11ywZ1qRCTCoVpb\\\\\\\/mzE2TT5sCwP3RmpTCq7wogdqVw5dQ4uUNLzEliSYfFdMl5q2H4+E\\\\\\\/xdBAwhXT0F5Zgd3fohdakCsE6aIuRpigYDnx\\\\\\\/+MUdgruVWO6Ac22pcVBKI1aFCds10hVQX5alHeeiVJDXR7uQ0cmIot0n8DpV00hvgnGR2noeg9Jj8zAbS4fQ5X7mwjnSYBCRsnhGZM+WDxrclQZx5ozV5hXmvkjDy00FOsWb1GWDCYXvAaIFi6N0QCFvGUrrwlb+e6mRM58doOHigG4pRiSbycjYCRHZNMld4wjmwyq5+JXvzZVPPKYsrbdWn5RBaIi9rqEDflv1XKin79jHRSwkLNo1GNNJJAhaKX0ZRCrgfQJUzkBJpGOU0ZmajEXnoe1pUWrOnysFFHFxxPgUcucpGQiPIDL5sqzoNxcyvb42Itz0JeGtKnL7j7JBzdxH9iogFmq2lJJLg2NOsyPckEmzpcS+KhCFMqC2qRVmzQTPlYAa6WhwyiWU8VDPlRwc5QXR9MpnYUNxhmtCxVNqojkvh4sBUzHxWvXQo4FxzKn43GvZZlJnI1tBqF9JXWKyYXbyGR3PUfDytEvI1OJxPuRjJczGmoLLcumUxRdELYQo5vj7ZyiD2o6Xv6uh\\\\\\\/H3ub3Zr3SxwVSPW2YApFPY0uFfwD8ELSjqFwM8ppwbpKQJUhdqslbkIXUU4Pvzj9rY2BZKxcGgOKB0PKBUaZEmwTuCJ1ZLqdHIbHomLwwyTvW2tmxqNoopDrrDGwjz5n07PeOOIGIFPZUQr8rFnd8bKdN1K9QYLPZrbEMZ+4g0RORGIGx55pDiTPspEczCTdhcs2DUGFHtgnJE1SHNRHeVLcrcMkhgbc1hPgyclotXdN0T3NqfzsAMk0XqEWPqeYmtB\\\\\\\/uK2HT+Yfs9b\\\\\\\/poGmJNlQC0X\\\\\\\/x5eRCJNk1242l1cvS4\\\\\\\/1TzHzAQ1Yx2GKVAsOLd7ZPKT65JCZxgJZE63N2S4vnSP7jHDL9JH8k0QDYj68fOrW8RGYoUSKnkfstvWPvhNGKvCek7pI\\\\\\\/EtJWq0c0uzeaCKpsXI\\\\\\\/kS2om86pdCTHfQtL5oncrRVK5NKl1zaX+\\\\\\\/u0vjewTk\\\\\\\/w6qe\\\\\\\/FxZMvZ0qQAtEtGR20deq8YbnWTZa+etAXaNgMic\\\\\\\/lZIyRqDi2Ga1YxO+5zH4z8emsubEudEIFSNOzQe57MLjgS3t9TJz3ThdPoxIKNzqHGbytPj42BErPOE6xlFzXkaJtZ7F+01nBoxzzi6I7UnGgqHYqW06VYIWTQSPHvA140Z31QO1nElIV8qpUVsGfiPok8VckirD0EIWHnA\\\\\\\/AlnUc5xtv7wW5TArOMfqCH57wu06LwcSPeO6G7MKQljygpOux7Uw7GDeDBWkfVgBuMhgktsppYe5oX\\\\\\\/jB805xLYkoArEPCkf\\\\\\\/MdTXdGAuJzZPmr6tgeppJoAfINL1etmh7A0iRLYhZPyqz3FFJLXxIzEwOXksiUckkqSLWE6a5nOwO4CJroCg+yisj6syHnYb2jbU7hIPM0o3SSK\\\\\\\/J\\\\\\\/dZDuTVzOcdrJ0qX9D56wYj7xaigR3YYUE3ZgHQrT4nARNsH5JVlziPQup5g4vHGrAb875ia\\\\\\\/7yuF2ReocHetdNbu8ghiUfO4vG\\\\\\\/O623fp9Mwkz\\\\\\\/3vaj246iDxA4H0xVLAHPo+NQxbS1FbtQPdvrra6ZYh1tRDfLO322Oa07DcxLQw6u6Lq42ktC\\\\\\\/NFeVGZaLcKyq96jILTZVrkvAdoOm\\\\\\\/rgRY6yEDviCUpK2cNzgduTYMeBWVisQuDnvvvJfh\\\\\\\/e3q9Kp4d9FeVSKrlvVnsApqkQMdehNzh8hyEpb3sVzWqGGmLBufsFIwoOraToPXz5+DX6kAZdFIFEuaBh5HFZ7blqwZBjXCvq5DhgNUr\\\\\\\/VzB7rjROumjPzWIinUS8DoI8ObYjb9vaF2k1PURehH3IkrsXBoi28PcrN4sf1pjnSPZNkDE\\\\\\\/MI2+ijPpw3frl7IDmWU1chB1ir+NVj6D\\\\\\\/Yli1wEs86bYLpVdv4ZJTSV0m2m\\\\\\\/AMbAM5feKyqp72A4geYCMiXLCSYsxMnhiypM9+bKQp0ZqQ54PCuIm2mIaOCak7SI5mNUD9IZH5c2skPdVNwu4HJ6j5dr9g20HeGD3zcQeTrDVESp9m28GQuwEE6xfHsoZTXQ9ucppN+mAbTYWcnYLHwv7mXIzW3\\\\\\\/9NSpodxLFL15nKbFPa4r6FpiNqJUvYlgVYAprZGlmslDygtsRDGNLGDPtS\\\\\\\/Fl6jQ+SjMTdjCiym79j40nvkK1mh\\\\\\\/X9mN5CSRrPe2j9LjXDsNqOIgEdem4\\\\\\\/TnHxtqVyMgOl+MVAQxHgSNZ67pIiU9bklkF+SOmGNjER7v20N4RGuycWYbwNZ6BxA2DDfMD4PEydJVfD1W4giKSfd65NOzPuPkVcFDUEkfK+cZA+pmoJXj2HN0hDxUkw+fU6KmBvOazUkC1UNzsKugev6E1CJZah9Y26j0axXkfZ6Mw+sTEhAjmFLBYa0K3UauCFUeVrWKsSpINAdJMgsohEt0E85VfsrW+ap3tIJeWsixf1sy6EYx61iqGADwKT5Zf+LdW2xiSIRaghF\\\\\\\/SaDATRSEdJq\\\\\\\/vlx1k2t5JmNpwGh4tYzSCc+oNj98olBmUvzZRBzHuIHijv7RWXSsJrAdlJ9Fhnm1HzxnUx7TWW\\\\\\\/2ENd5OaTaatqu0Ag52ENsfhdGE8clgaUH7TjE1BEpQpELpuVxwmVTpsOm6FJjSKMeyKEYtNQM7sRYigU4WBsbDhjsRh38Sol0GQzuxv6uhEkdj0uCHo9WsFIhj9mA29MecoSgnxYcLEDrVDXg22\\\\\\\/vEKxFC+ipa5viezcL6zDC6XLtEbRtFY9axyDN96ijT4avY2ZIpCSiJTn\\\\\\\/MVtSqA8HHVXUbllKM2HKM6HdKyPkUP812Skmcj7euV\\\\\\\/cCPZlq0Je8WLav3ixVmRlnf39epYIZdQJkMscTZeJm\\\\\\\/yP395\\\\\\\/\\\\\\\/ZgzNS0VmHNg7jajTHzievTang6rpF3pHpJ0G37zY8vvBWIR8jQoJFSjSTSl5NqApulTY244k993ANL14KVMn0VVg5eLr2T68+a25hMdB755RM3YbfV1EMdtNXoZhr5J6NXprincIbRMd\\\\\\\/tbo68gHsz9Sj\\\\\\\/HzcG4S9I\\\\\\\/5cFOjRsuGJhj94P8JJXqrzxLzAGeJiIrYQyMH7YuSHvMJC+pJBsNa2rNudY5ebpnBpQSSx\\\\\\\/PdDVFc2HGrDFhA+7n6FW7Pw\\\\\\\/Oy1RVuN5k2xApfZrc0OK9WcniOnkOAcNgOlIWT06qGOzUSbdaRcF+CYiFT0YSMUqhkxZUBVtplPu1fbUVGpE917VewIU6d2C7ldI\\\\\\\/NtMkvOeXUAN4dAabJh8\\\\\\\/ihbc3fU\\\\\\\/Y66E6jo1FeQxIoROU3EyWBQrbuPKEklu3LvGmuXZn1HgkDLlACsomWk94j3fnm9bC9uCOLDx8zhZZkpFgr0cedjfwb8GZDG7DoCChEpAlKwtdiZm8rLc\\\\\\\/01YfkbeRCoGKTptF66akK\\\\\\\/40FZ1TBg4evrkN1743Yss+jQQ9F2S+pbfNWPziwq5uIb+fxWjwAGJo5G+HQnaU8xB6sYFf80l6FRHTfQQIv6bRBjqeEu8oqXyadreB8zJwxTs5h4f9LlfEdfgEeEgSrvcKah9rYUAmB01PZv+pknZbhvKCgiNTNNU2JUBAA2tcX6\\\\\\\/XK3r92TlsFUplbp07Niw78kTclqRP5riMJ4YMq7L\\\\\\\/n4U4K8OnqyElKrlpIqfTNYjaaD3xFsIBBe6EdVtXfnJLQXASzqSv03zccEBVmzE+\\\\\\\/KOphiKm\\\\\\\/V2p6krDuAVrvWYwiSx1McMTSwRIavfgKLsiAuGKAFTxj2g9\\\\\\\/7\\\\\\\/xPj95Etz2DwpfQvAmBFDC+i3a8U2DbP8aByBEzfhhHJHN8OXFDrsNiSBThyz1JAJr6JWTqkpk2KO\\\\\\\/17XdwMJkkGmZDW2FyIa\\\\\\\/utR6ezReNkH0C1zqQkJUtcxn6Ivxw76qdBW1ylU9ErO4hp43pTTBoUK3+Ghf3a0dekft35ekvxaJLiHD6Jla2OhxmsTirLcwFZFkwFpadBvuhj2oJAjlTikajYqWLFXmO9\\\\\\\/DhM2ftyuJ7qGCyYuwSPYTW4C2RNuQaGKaWeuseRTOFjHn4to\\\\\\\/YGH8ikEc3hD86OsGo7\\\\\\\/HnuIQ7tDw8QSpGeb1zbEK7eGMm2I91H5own2s8wdvK6iEoPGTYIYFrJVnYV5Nk\\\\\\\/dUn0jDvNVTefPV8g0+7z0VPZgek1N8Zbu7H+XIQRQt3oWqVxTZyh4nKsJuAYP1Elwg5C8YN3fFMant0jsXvRYwX5U\\\\\\\/4ng15hjfBx0p1eIR4iueFFEiauG+Ywl66uUko4GQHlxDTqSyAXMrSonRyiJqyhx4+Pe56WoyWauT33vr5BHLCo9vfoj9md9NdNovkmYC1TUKD80SGSJmhJE+WZabv\\\\\\\/QYhggiKwq384UX8mkQ8Mr3NynPNbaoIQDuMNt6rXamPaYnwASzEGju9kccWW\\\\\\\/bznGjjkIvA9+YhbQgj3kO8sLHFECASdtLxnvSSEWIwXN1pgrYF2AtPlXGQPwnHYTVX\\\\\\\/azZNumf9Vnu7yzOhDZ4FSTnfpeH+vnZJhNYqjazNSePJ4XD21ha6x++dWufZAVeftHfUbtmxip\\\\\\\/P1kiHqm6wCCceaL00CFWOcpulMdXFTobZDaMNTh0ZvD1bs3+KRSEfB51ZTcNhqAuJyzFCa1HEgLODpifJ3IbQDKKc11V82cI44dmuL4VVEHWBU6QZTtirMsuI3EQq6dq6F5Ex1srob4JiNsmdq8AQ6e6aRlQ2WkdgNWRLtUHdX6aefWMZH655GDGolVJYQ3V1tgXhMS1PAkoUAdr\\\\\\\/OM0uEPSBDclmtFTsb2bJ9KtlYYcTZzSCF4Q7NLZp75COh2O0d7EK\\\\\\\/33qgwXuikGN\\\\\\\/AFwjAy3+0HH6ilTV0AGvAbXzVVFZ0qTT8rP\\\\\\\/IS0fTpipZaRoO+r4akfQHOStFtWeBYILUoCEmuPdKO7MnxEusfVz\\\\\\\/Y5qoE\\\\\\\/YkH9YEON0jEfyEc8C86TU9\\\\\\\/ynzJtJHeE5a9l\\\\\\\/ufgwpa2Rh2qpn2m33K6P\\\\\\\/oAEqRLTTA6m9de29EhxTAPm7grXjj3A4ULAVYWDZy0mGf\\\\\\\/k8VD1SXzivJHQOn7\\\\\\\/IP9PWX5pW2uwanJasO925Ce4kHdWLioavunWqahiNwW+ejgnHEBr9UwWB6txRbeOuF6QTyBRK1pKNCtXIBysEwwaQ3qStqmLWsWhmx57RcPyWa3T5RwkauRi4WbhOlsXSvGigiNn5O+aR3cExiJAiwtj4QPOKLuPB2mmFC2C4JDDtUveuqEUNGOqugWCsFf1H\\\\\\\/yVNqcjb9qTc0vs6R8yKIUwgl5xd6q37Xbn6XTExpkSoMyYDB8Uvyn5YRKC1\\\\\\\/D4BdyUp13XbhA1S5A0HTkO3cDsWIfkv2wEt6aXU+MWpqbSNwXoZE3evcRIieVMvIrwzoGzL8RGnCop\\\\\\\/tGSSO0TQ8q6CPHnhlma8nfhesexuj8ji3sYevmoQU4NUu9T85FDb8LkkresGMO5joGNHh9Xoj1WZYygD0H+\\\\\\\/c5XvL\\\\\\\/sG983bCdWklbrCHTdHVaNhrv4UQ2AwxUS0aNJgzRWtlp8MvBK8Ri+wkwEPATNlUumQGX6z4guz\\\\\\\/j4+0ZNaDJPPVt4jZYMViFtbtM8naAH+ZcWZXo\\\\\\\/hBTVBNlSObJ2YV9VinfK6Xnh\\\\\\\/lKs8JqaCqQCRLKF+btpLI3FLmGkxSKq3mvAQRl4V90P1lsTXKLb\\\\\\\/joidv5f1n1MdcU6wwzYNASZ9VODeOLr4uWJQioogaOImR7lQLAnnV9\\\\\\\/8mlBee0J4MK+CJT21NUaM56FMIyfeGUPVbn\\\\\\\/Sf7PLp+\\\\\\\/yYcvke8Ot8U9hRx68M9MaFBK25DA2GnNgl7CfKG2rGUAWa2zH8cFykJQhIVrglmJK\\\\\\\/7WG9B5vigFsMal8Wamr8YIStUAKdow6zKrOiVjeQzqRKbezd4V7Bx4CLGAtvEESgWZFil\\\\\\\/3863nM6A6I0s0CeCRDnwwATCcyyvM0Fc14dTEeQt7TYTpEK7DNK8LwTn2KI5ZeNVjWgNolWjz9Lln9w84GkK29SGVwjWNQuUaR+LnpijsXR7GCRQorEsCXO4i5vnu4PccS3qIcodMEACC4MQCGAWvh3bL3X+So93rZbJ3um0j0uBaqJO9JBY\\\\\\\/Bqym2CEC5Z\\\\\\\/e7b+f3v8GAWVD\\\\\\\/fX\\\\\\\/Gk01EvB8Hb+EQtmiohxJ9EEjuxkoOo2eI06p+Zl+pUW6r7romZkTdMjRWpeiKpVItvDIP5os0Aq0+3pyT55Of+ewI5EolzvZiAwxuRoQM5q0XAdzJFx6u9zk5o8R17i\\\\\\\/WWa\\\\\\\/oFshidB+jriZTotcYKge7tZs\\\\\\\/lXXKgKj9c6sib8etKEGdnw5zkS2wi5jhi2JOpAZTMuc4BmxIbgd39fnUza0gZ3BWGk2iEtReoCCFfPJ2WAzNXCngJZqV3DG3qqHwa9\\\\\\\/\\\\\\\/++ZbawZED6wu\\\\\\\/VUgwHuf+6MpqcLEcuSaJKQOb0bfl69lWJixm7L7XiW1upY8UzV7Tf0asiv+yY3V45r8uQeP3Fb7Gy481cPsHwFQ9RtygDxvFsmbmSEShWlnSYcpL3wnTpd9b+uC8hW9GN3+X+Rv2dmUe2FdFqEwGihwds2swcTmFtpIE7I+o1XAavuKAgck8psLVoDoW\\\\\\\/8Cw\\\\\\\/HVw2oGUyOyPOuWFAZmZEecAGic\\\\\\\/3IN\\\\\\\/NVPjxPeHhz5ZUYkJkO0Dr38p6NOT8zNO+6s+joCE6lqpWcuXL3sZQ6FZcYPPfTUNlYgBu\\\\\\\/XqZrDT3ST++SkUSYnzX2t3Cjb8GAMUIh+ttWrGVOMBNfjf7Xh4OKBbS9ShV8S0WGWf0yKHo8WfYEk0WFUJnMgu6qBt8N5kuTlGSr1UiKImGAXaoiw9xulL8FQ0WYbbZvGU7O69D1gI0bQm+WjDMtr65dldvLb88rg9+5f4jyCg8H6CFVK3iXvM8+Z4upFrLssIoJkB1Rl1ZCnpNLrN1Wklfs3VdKWrUr6B6KZA+SkvpiCAtx0YAiHxZQA6Lt5dzvxHK6jz3ujBkPXS45kozanlRcjyC0bf7h9tihGWyUVxO5NXfhnJQOzC32JRBWukri9Gx1oY4xMYjeioKiciPKh0mgwqivSgNcmthRPrMziKOolE258DI7jC1e8uM1yVHYIrbIQW37FKtwt6xRUDBNNz3oB\\\\\\\/SYYuaJMBV26FCh6L7dNmizYb7AAw967X+Dj8foV4oYbuOxeV2\\\\\\\/EWAKK85HF5UqL2e+egL9w7u0tpoUV\\\\\\\/ofQO0Y2SZMTCK7QXVEZnbcmvvMwo12+Gt+ai759qsUWpzVkLKYoP8KbOlFFk4buVk6bfg824RIwKeAIjcLH4AhB932+Vo2LunNXa9G3mnkGTeQUPaTgPrEh7qZVpmF+crYLl8tbqru8PRF+AHiheRXOTRFItF1sC3vEu9kekNfUjU0NLLFmEfAP7TJ0jnRkHoT9nnYVj64Tt4qiRQ1XdfhFsN0h0yhsnJq4xxZGE30ESlod1Sh3PhrczPljFgv1uzzFZvG6RDGMKzoqN5BtQn8qGOYW+o3pPXCrvISMgKcSkMVTInxPS8U7Sv\\\\\\\/\\\\\\\/9pYshX4ztGxPFrsGqhRHcbBtTJiA41ER4f\\\\\\\/HwUXoonw30l6mm7epntEB8\\\\\\\/QvKF+i4rglsBDT7b+rJq+IF8eNGSquXeziVsVNbiYb0iU14vbEZ88NW1884dhyQB66udQ2EPKPmuZ7zt89vuY+hq704ZW8FXL0a1abGQvqSzUw3z4anoDDsZC7VuJqOPXLqGK7B\\\\\\\/\\\\\\\/HvqV6D+mW+0gOhsdwnaBDGVZq\\\\\\\/DFnr0kzKaGPMJULmf0xg3FAzbftjLE8h1kr974zEWF2PfYoRyd039amKGtAA3\\\\\\\/pDeun1HT6d2Hazi+GvIiYUFumF5YLzuAnfsIZlIlHVR7g6SsGjO6ctiBYbTCPl5A1Dif0AJf9i465OKLcAyu9tEMb2TtXpqGNmJdXXlNISXgzUqNkng1ajw4grX3QiddYnPCbBHf5QCou1yY39cRiylJY+T2zXuODfBPgGkVvZdbOwGe3n8qxThYAXzYYiyirNRbOpp5a62JwNS1NK\\\\\\\/DJzEFS+QvNCu3Mz1Y+9F8PHcDovW1TCtN3ERBF5A9r4MHPLNtpek\\\\\\\/TIt6+DtQt97XafchGvI61jiwB9V4asSNm5aKfCSWGSoK6FT0f2YD2oIFhWta9VzoeDNewWljsUnbyTvLG\\\\\\\/\\\\\\\/uj1eilbl85TEIbbg4uXRcpyBnWVC64vtiflV9hzMzT94LIRNY+imi1nZkaVN31xojTDFEOwRC1vKdyu3mD5JK4QHcLLmPhEQ3hajqwl6ddnj9Fe7ogCJ8nMyZCsMLo\\\\\\\/CCUHZiIiwMP9pD5wEUoR8LSxwE7Mo+KbwY5J8AriY3rngcSgUKaqrbf9Tc9IR4swD0qYgkCdCi0BoBU5BKxQNi\\\\\\\/TKoRd7ZdCpqU3uIvZxV5MvXJGpQHZkfWeELEQB1HNlD4w4C1Mv4ZMKu5pPQ+2kAYUacEEMqhFFVd8BtIvMjArX0KiXWNdg47QVCSIRkfSzUgmUP72MXj51vwD8Qg\\\\\\\/53PVJzgxdOkb2wuDL02+WliAqGwnLUwzEeavo+FOhR5nqjB51IM5krf6bcCFZSUGa2O0nahpC9SYgoyP+f\\\\\\\/YMTMynH9lKUwoMawn2QsOb67QfA6o+RKrma8\\\\\\\/q8TEY2wVU7RijmHzSwZK0+R4XafMQG41k0QOJ2zHcFNiwRrsklNLu9LUS2E+B5nx67ppeYpMsP\\\\\\\/htTvu8OJiWgA1qCMECFIKP3snwNW96729Mp2L1HIB419O4YzpfvB+kRChGuzWlkVrl5AbfQpNQCuVtR6d3RH3kg4zQvsb9RN\\\\\\\/SQSplPXrWKPkxGQpk9tgDRw\\\\\\\/NsbEHdR1u+zgs+SgNY+y7Fxyxv1aCRRkOXiOntgGpDWitTnsZHflsc95oJ+0iBwWeeBwMI+wLmtvBPzA5HRA022xBpOWxwdlK0GhNpVMTNVgzKk2LtYMa2F+rmxr164wEHQrLXnCNXX8atSPZPsajkOAvHPDi5dwSwmYAwCfybaogNLv\\\\\\\/JHphB9vb4osel\\\\\\\/+l\\\\\\\/1A2cm5eqz8MJ5a1kLbhT4PCzAMiJwwV9cFXaP5O3EgkXHS3cwU5RNLTUMHvgVLAeKD9Tow6I5B+87oZJkTNCiEG12bxellOtG9FOJ9btbTlBxVNkXRCkvAQ34PNF3D9Ultk9I86hWXBA50YIDxVdfhmnFmwDsa2e+FtpoX8N9hIZGTW\\\\\\\/bzvqkjHm6qf3iQvbU2Ihy7d\\\\\\\/AwCBiqvKARUkiftmlxZNS6gSno6LIxORpu7vi54K\\\\\\\/cJE6xgUZbStgwHedOqLkj5zYaf6nnrde2lopqbCOwpKK+4k30\\\\\\\/+8nuU73e8yYAeldNIaKRLNVYA3Kvan4KIGhtbHW9CSM1ExJrte0HN+NHec\\\\\\\/MD9uVadSKpS1\\\u0022,\\\u0022iv\\\u0022:\\\u0022DvV3dXBpwoTvKbcBc9LBsQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f8361bdc4c3d90d35c1fd7008b04c441a5a66c587648c725ffb37d3133f613fc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pMPvyatjM\\\\\\\/61rG5SgXp9utF4SeTBdVN1EWXMMjln1Aen0g1PnUUvKPtDktCLUik\\\\\\\/FBPzjvYpmCHi+Mm84xleAUvO\\\\\\\/xqXTmkQdSQxAggBh82OCX3ZV2sKl1uCS2Wbd\\\\\\\/\\\\\\\/7GB3Mb170elYMUN3UYHfb423YbNpaastK0mUm4orkY3qWCJjSQrrEaK3bzf+gmcnFxK4Fo0HlWfyTMd5jLo5R3\\\\\\\/d2b4QtoeSboXN9aeMjFgpLK8lGYh1RyoK9OHavBfI\\\\\\\/k+0UgovnkjjL1GHBH4lZcdvcF8VwhM5D8XO4s2VVafquiBPZYDBAghWyrj1IPqaryL\\\\\\\/UWbdWXatIr6xw5jZCcCInpxCLjv0nZnWeW36qlXyyNaj0PPCwsapsY4uarDwC1yLMQfIDkDgoaOkFjb+kO3LwmVipbAvWcnHVTVe8P8t\\\\\\\/b\\\\\\\/gN\\\\\\\/rwSxw+CIAdLFDijii+ZUUruTLQhsA9VRtHY4nqPD0AEzJG\\\\\\\/S8YrBtOS21az+E10Mvj3pMxzsxJFzoPfNXbe\\\\\\\/nN2a8SbDoi+4nxOWVNln+DHP4m8+Qdw1SarYvGri4ZXMNkoigPq1ALXDeFYarvA0Ns2elS5m3KVycadHHvbGrV7Fh2olVxbxhNrETVbhz6eKG9glJ7V2lPuv6URvWZBZY9W7m60UikdqhI44G4psEMBQRSTCdEIbbID0zMwz6k9JEQmOrc3bNAAOudbpMjwsyoMe43Ni2EvNWAYFvwVELW6TC8RNZVwhOPBKRHs3ZK1q4dqIDISejso45Dy+L7MWZdtIlioZws+biLBrisrC\\\\\\\/vWhI\\\\\\\/FFgPaWytPf2R11IGBda23s7halv18cBFgd5sdF2FBX5R2iDI2+w1bg4C37qkzQLbKvbC989APEryb+rrkNCXFJndqjgD36rkssYJfWYmaeFuYzm2Y0rYMeF\\\\\\\/zUDcbkVwKNhTLyaP6Xxlpz3obE7\\\\\\\/OSVOBHczALNIejzSEOm2CvDqWawiA6zoy\\\\\\\/jH+2Y5OCmKB6ImhTp7BKT23GjAQBmZtTrfylzDDpYwFKKXkbVB1F4pG0voJTSYMvaY4BXFKHZvuEPYycFNKCOqiyHt9zYn270oiJUCpARGGAcY1g5dGFR7nVItdXrhgghAdM9A0267Zz0z89DK\\\\\\\/gfoMuFDLA7d1fmpDaR2zluPv1BtBsQz7Zj7RiBoWuBB+DgERm38oDuCfbOyVlV\\\\\\\/i6hTOvC0yyU5lqQpf32\\\\\\\/2K+O1EjNuv2v6s8oSgWcUiPUt45es24fVh+AWFvcA84OFxVca3thOF\\\\\\\/TDEBrB84RG2461pOGnHns6\\\\\\\/2BoDYjby8luNBdYqDQULmSqtw9ayKmnb1aPURXhW8LsgpN+TgfogK124\\\\\\\/Vfidk2qktOccJDjdPMJwhyRKBgG9ynyu\\\\\\\/b35IFBf4JtSIayJLKX6knkp8oPUCTGQcczr0ZbJxuAhHcwbJchfBF06Sx3BCjZiiiXPYcJdQpQAgzompF7Pr3IHXAAUIpyHrOUVLEGBlbaQCm9iCaqNFonfsT7MDBpceEJIUi\\\\\\\/QVeSxgAl1eHx9Uli0rqlMmc\\\\\\\/Dm5ifdCesQi58kuUKpnxQh41dzlVJ08BtoGEZD9nVz9tU\\\\\\\/L2f1PHpR5GXiB+jEP8AoAWXGnj8GQrVPMPtwru+InOLfij0NGUKiec6o8nBkyWakaJ5YRxP8c4F\\\\\\\/6E7tOuxYldhkwMEv1YYWshZEfa5mOq6PLl0JMTVi8rUFfC2SNN1O5lfqVb1wU2+vemQyMwORBhnCJDHQrC8QKIVJsVewoz2DVyoHaikWb75KXAhAVoD1pX+i7yEISgzJmpP4a4Yc0G6Z98HPwiShW5TAkctCOoW6Uahqd1UpIvuI\\\\\\\/f5H0fXRDtn0vJ9DD1nKN1D0IHTnEYNZ\\\\\\\/8RJagltsLy9q7u7ZsnPxTvXqogC7W1SiyERjsuNLL7ghXSYdL2vA3F2eTQ2lDfWA+LtIdRsVf0ezb+CRdcx78Vbx2lj7hix1XUjHq++NkyPWtmGaLHZqdAUf7xqS2EbmJO4rVjE6Pl9gc2YowZ+gQQr5XEOsHFmnflBSohNVkZOroYO3gUerlImn\\\u0022,\\\u0022iv\\\u0022:\\\u00225wglz3Y5Jv+VNCbysbcJ2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a3c6c5bbc0518b2b60f60bbc9d3e7fe8165059d117f117a3dda94b89f29a16bb\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493461 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2153" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-njCzSyDyLqkkwn9axVZaNi_pq0mdGYOOU1lxt0P4dwg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LUi8IyzMCGqyPPK73_5TDMPsvDfMdGeszMvaN5TfciA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2135" title="Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2161" title="RNA-Sequencing Analysis of Paternal Low-Protein Diet-Induced Gene Expression Change in Mouse Offspring Adipocytes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493461" id="node493461" data-pisa="ggg;9/7/2153" data-pisa-master="ggg;g3.119.400093" data-apath="/ggg/9/7/2153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Generating High Density, Low Cost Genotype Data in Soybean [<em>Glycine max</em> (L.) Merr.]</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5897-2617" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mary M. Happ</span>, <span class="highwire-citation-author" data-delta="1">Haichuan Wang</span>, <span class="highwire-citation-author" data-delta="2">George L. Graef</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6324-9389" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David L. Hyten</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2153-2160;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400093 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493461" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mary M. Happ </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BM.%2BHapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Happ%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMary%2BM.%2BHapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5897-2617" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mary M. Happ</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Haichuan Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haichuan%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaichuan%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">George L. Graef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BL.%2BGraef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graef%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BL.%2BGraef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David L. Hyten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BHyten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyten%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BL.%2BHyten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6324-9389" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David L. Hyten</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.hyten@unl.edu" class="" data-icon-position="" data-hide-link-title="0">david.hyten@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2153" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493461" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2153.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493461" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2153.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493461" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2153.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493461" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Obtaining genome-wide genotype information for millions of SNPs in soybean [<em>Glycine max</em> (L.) Merr.] often involves completely resequencing a line at 5X or greater coverage. Currently, hundreds of soybean lines have been resequenced at high depth levels with their data deposited in the NCBI Short Read Archive. This publicly available dataset may be leveraged as an imputation reference panel in combination with skim (low coverage) sequencing of new soybean genotypes to economically obtain high-density SNP information. Ninety-nine soybean lines resequenced at an average of 17.1X were used to generate a reference panel, with over 10 million SNPs called using GATK’s Haplotype Caller tool. Whole genome resequencing at approximately 1X depth was performed on 114 previously ungenotyped experimental soybean lines. Coverages down to 0.1X were analyzed by randomly subsetting raw reads from the original 1X sequence data. SNPs discovered in the reference panel were genotyped in the experimental lines after aligning to the soybean reference genome, and missing markers imputed using Beagle 4.1. Sequencing depth of the experimental lines could be reduced to 0.3X while still retaining an accuracy of 97.8%. Accuracy was inversely related to minor allele frequency, and highly correlated with marker linkage disequilibrium. The high accuracy of skim sequencing combined with imputation provides a low cost method for obtaining dense genotypic information that can be used for various genomics applications in soybean.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/high-density-snp-data" class="hw-term hw-article-keyword hw-article-keyword-high-density-snp-data" rel="nofollow">high density SNP data</a></li><li class="kwd"><a href="/keyword/skim-sequencing" class="hw-term hw-article-keyword hw-article-keyword-skim-sequencing" rel="nofollow">skim sequencing</a></li><li class="kwd"><a href="/keyword/low-cost-genotyping" class="hw-term hw-article-keyword hw-article-keyword-low-cost-genotyping" rel="nofollow">low cost genotyping</a></li><li class="kwd"><a href="/keyword/soybean" class="hw-term hw-article-keyword hw-article-keyword-soybean" rel="nofollow">soybean</a></li></ul><p id="p-3">Genomics research has yielded a variety of tools which allow for more efficient and precise translation of genetic variation into crop improvements. Panels of single nucleotide polymorphisms (SNPs) obtained through SNP arrays or genotyping-by-sequencing (GBS) are the most common tool used to explore and make associations between genetic and phenotypic variation. Genomics-assisted crop breeding continues to demand increasing densities of genotype information to successfully dissect and predict genetically complex traits (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hamblin <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lorenz <em>et al.</em> 2011</a>). Current approaches of directly ascertaining a high density of SNP genotype data on large populations are cost prohibitive or fall short of being able capture the maximum amount of genetic space.</p><p id="p-4">Fixed SNP arrays and GBS are popular options for SNP genotyping in crops. Panels ranging in densities of up to ∼600,000 variants are now common in several crop species (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rasheed <em>et al.</em> 2017</a>). However, recent genomics studies are utilizing datasets consisting of one million or more markers to answer complex, quantitative genetic questions. The need for this high density of markers is rendering current arrays and GBS approaches inadequate to generate the magnitude of data modern genomic studies require (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tian <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Patil <em>et al.</em> 2016</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2018</a>). High-depth whole genome sequencing can achieve these marker densities. One study utilizing high-depth whole genome sequencing in soybean found 9,107,000 high quality SNPs (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Valliyodan <em>et al.</em> 2016</a>). Despite advances and the plummeting cost of next generation sequencing (NGS) data, this approach still presents a heavy financial burden, as several reads are required at each variant site to ensure data quality and completeness.</p><p id="p-5">Decreasing genome coverage in the interest of cost savings introduces missing data, which decreases power and can produce biased results. Imputation of missing data has the potential to allow the researcher to recover nearly all of the missing data points resulting from skim sequencing, drastically reducing genotyping expenses associated generating complete, high quality, high resolution SNP datasets. By predicting the unobserved genotypes based on the surrounding variants and their correlation to a complete reference panel, missing data can be amended to the correct allele genotype. This technique has been developed and extensively used in human genomic research, and is now commonly extended to other organisms (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pei <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Howie <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Howie <em>et al.</em> 2011</a>). Seen frequently in plants is the use of imputation to fill missing data points in GBS data (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chan <em>et al.</em> 2016</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chung <em>et al.</em> 2017</a>). Specially designed populations such as bi-parental, nested, and multi-parent where the founders are genotyped to a high depth and used for the reference haplotypes has been shown to boost accuracy (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tian <em>et al.</em> 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Swarts <em>et al.</em> 2014</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bayer <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cericola <em>et al.</em> 2018</a>).</p><p id="p-6">Crop breeding programs working with inbred species and/or inbred lines are uniquely positioned to leverage imputation algorithms in an extremely accurate manner. Near complete homozygosity through inbreeding or double haploids allows calling of genotypes despite having sampled one allele at the site. Large haplotype blocks in historically inbred crops theoretically permit imputation accuracy to extend across large physical regions, where genotyped markers are sparse but in high correlation with each other. Success with such a combinatorial approach has been reported in rice, using ∼1X coverage sequence data of 517 individuals. Imputation of the missing genotypes in these individuals without a reference panel to produce a SNP panel of ∼3.6 million markers with &gt;98% accuracy (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huang <em>et al.</em> 2010</a>). This was confirmed in a later study that also included simulations performed down to 0.1X depth. Falling below a depth of 0.5X resulted in steep accuracy consequences, with concordance falling to 76% at the 0.1X level. (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2016</a>).</p><p id="p-7">Incorporation of a reference panel has been shown to result in large accuracy improvements at sequencing coverage less than &lt;1X in humans, where imputation at the 0.1X level was improved from less than 5% accuracy to ∼70% (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pasaniuc <em>et al.</em> 2012</a>). With the growing amount of sequence data present in public databases for many common crops, it is possible to generate an extensive reference panel that might improve accuracy at ultra-low sequence coverage and further cut per sample genotyping cost. In this study, we report on a low coverage whole genome sequencing with imputation approach in a naturally inbred crop, soybean, for producing a low cost, high quality, high density SNP dataset. A reference panel was generated using publicly available high-depth sequencing data for 106 lines, and employed for imputing the missing genotypes of 114 lines sequenced at ultra-low depth. Coverages from 0.1X – 1X depth at intervals of 0.1X were evaluated. The factors influencing error rates and extensibility within/outside soybean were investigated, and the consequences of error rates and types of error on a typical genome-wide association study (GWAS) were explored.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials &amp; Methods</h2><div id="sec-2" class="subsection"><h3>Reference Panel</h3><p id="p-8">The reference panel for genotype imputation was generated using publicly available sequence data deposited in the NCBI Short Read Archive from study number SRP062245 (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Valliyodan <em>et al.</em> 2016</a>). This unfiltered, raw dataset consisted of 106 <em>Glycine max</em> lines sequenced at an average of 17.1X coverage (Supplementary Table 1). The raw reads were filtered for adapter sequence contamination, base quality, and truncated reads using Trimmomatic (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>). Bowtie2 was used to map reads to the <em>Glycine max</em> Wm82.a2.v1 reference genome with the “very sensitive” option (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Langmead and Salzberg 2012</a>). Reads with a mapping quality score of less than 20 were discarded. SNPs were called using the GATK3.7 HaplotypeCaller tool for an initial panel of 13,052,759 SNPs across all lines (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Poplin <em>et al.</em> 2017</a>). SNP calls with five or less reads supporting the call were filtered out, as well as calls with a confidence score of less than 20. To control for potential sample contamination/mixing, the inbreeding coefficient, also called the F statistic (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jain and Workman 1967</a>), was calculated using the software Plink1.9 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Purcell <em>et al.</em> 2007</a>). As soybean is historically an inbred crop, one can expect F statistics close to one in <em>Glycine max</em>. Seven samples fell below a cutoff of 0.9 and were discarded from the final reference panel. All heterozygous calls in the remaining 99 lines were filtered, leaving only biallelic SNPs for consideration. The final reference panel spanned 10,803,148 biallelic homozygous SNPs in 99 lines compared to 10,417,285 SNPs found by Valliyodan <em>et al.</em> using the same data set.</p></div><div id="sec-3" class="subsection"><h3>Imputation Panel</h3><p id="p-9">To generate a low sequence coverage panel for imputation, 114 experimental lines selected from the University of Nebraska soybean breeding program (Supplementary Table 1) were sequenced to a depth of 1X or greater on an Illumina NextSeq 500 (Illumina Hayward, Hayward, CA) using the manufacturer’s protocol and 150 base pair paired end reads. DNA was isolated from lyophilized leaf tissue collected from twenty plants per genotype using a CTAB based extraction method (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Keim 1988</a>) scaled down for a 96 well plate by dividing all reagent volumes by 40. Extracted genomic DNA was fragmented using a Covaris S220 with the manufacturer’s recommended settings for generating ∼350 base pair length fragments (Covaris, Inc., Woburn, MA 01801). Double sided size selection was performed using KAPA Pure Beads to retain only fragments within the 250-450 base pair range using the manufacturer’s protocol and eluted in 40 μl of TE buffer (Roche Sequencing Solutions, Santa Clara, CA 95050). After testing DNA concentration, samples were standardized to 62.5 ng /μl. Libraries were prepared using a custom protocol adapted from literature to perform A-tailing and end-repair in one reaction, and avoid PCR after adapter ligation by extending the incubation time (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kozarewa and Turner 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Knapp <em>et al.</em> 2012</a>) . To perform end repair and A-tailing, 16 μl of fragmented genomic DNA for each sample was combined with 1 μl of T4 polynucleotide kinase (PNK) (10U/μl), 1 μl of T4 DNA polymerase (5U/μl), 1 μl of DreamTaq DNA Polymerase (5U/ μl), 2.7 μl of Cut Smart Buffer (10x), 2.2 μl of dATP (10mM), 0.8 μl of dNTP (10 mM), and 0.3 μl of ATP (10mM). Samples were incubated in a thermocycler for 30 min at 20°, and then immediately ramped to 65° and held at this temperature for 30 min. Samples then proceeded immediately to adapter ligation. To the 25 μl of end repaired and A-tailed product the following was added: 10 μl of T4 DNA Ligase Buffer, 3 μl of T4 DNA Ligase (2000U/μl), 3 μl of PEG 6000, 2 μl of PCR grade water, and 2 μl of uniquely barcoded adapters (30mM). Samples were incubated on a thermocycler for 45 min at 20°. After this time, samples were immediately cleaned using KAPA Pure Beads to retain fragments within the 350-550 base pair range and eluted in 20 μl of TE buffer. Multiplexing was performed by combining 5 μl of each individual library. Libraries were quantified using the KAPA Library Quantification Kit for Illumina platforms.</p><p id="p-10">To create subsets simulating depths from 0.1X to 1X at intervals of 0.1X, reads were randomly selected from the raw datasets based upon the total number of reads obtained for each genotype. Each dataset was trimmed for adapter contamination, base quality and truncated reads using Trimmomatic, and then mapped to the <em>Glycine max</em> Wm82.a2.v1 reference genome with Bowtie2 using the “very sensitive” option. Mapped reads below a quality score of 20 were filtered. The genotypes at all 10,803,148 SNP positions in the reference panel were called in the low coverage imputation panel using GATK3.7 Haplotype Caller. Genotyping SNPs from a single read has been found accurate in rice whole genome sequencing and maize GBS applications (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Swarts <em>et al.</em> 2014</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2016</a>) . Any heterozygous calls were discarded, as well as calls not matching the two allele options at that position. For each subset, a random 5% of calls were masked and considered “true” genotypes for evaluating imputation accuracy.</p><p id="p-11">To characterize how genetically distinct the experimental lines were from one another, a genomic relatedness matrix was constructed according to the van Raden metric using the R package “synbreed”. Prior to calculation, the imputed dataset was filtered to retain variants with a Beagle posterior genotype probability (GP) score above 0.9, pairwise r<sup>2</sup> LD metric below 0.4, and variant site missing rate below 5% using Plink1.9 (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Purcell <em>et al.</em> 2007</a>).</p></div><div id="sec-4" class="subsection"><h3>Imputation Concordance Evaluation</h3><p id="p-12">For the sake of computational efficiency, imputation was performed on a per chromosome basis using Beagle 4.1 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2016</a>) with the low memory option. To assess accuracy, the imputed genotype calls were compared to the masked calls, and the percent of those in agreement constituted overall concordance using GATK 3.7’s Genotype Concordance tool (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McKenna <em>et al.</em> 2010</a>). This accuracy assessment was performed across sequencing depths and minor allele frequencies. Three post imputation datasets were considered to quantify any accuracy improvement obtained by filtering poorly imputed sites. This included the raw imputed dataset, and two datasets filtered on GP. Values with GP scores under 0.45 and 0.9 were filtered for the latter two evaluation panels, respectively. VCFtools0.01.12a was used to bin by minor allele frequency, and Plink1.9 (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Purcell <em>et al.</em> 2007</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Danecek <em>et al.</em> 2011</a>) was used to filter on GP score. GP score filtering thresholds were determined after examining their relationship to error rate (Supplementary Table 2)</p></div><div id="sec-5" class="subsection"><h3>Error and Linkage Disequilibrium</h3><p id="p-13">Error in relationship to linkage disequilibrium (LD) was examined as a potential metric of extensibility to other soybean population and crop species. D' and r<sup>2</sup> statistics were calculated for all pairwise reference panel SNPs using Plink1.9 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gaunt <em>et al.</em> 2007</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Purcell <em>et al.</em> 2007</a>). Proportion of errors made at each SNP site across was calculated by comparing the imputed values to the masked values across all subsets of depths. To reduce noise, data were smoothed through the application of a rolling average window with a width of 1500 SNPs after ordering by the respective pairwise LD metric. A second order polynomial was fit to describe the D' and error relationship, and a simple linear regression was fit to describe the relationship between r<sup>2</sup> and error.</p></div><div id="sec-6" class="subsection"><h3>Relationship Between Samples &amp; Reference Panel</h3><p id="p-14">Close relatedness between the sample and reference genotypes has been previously reported to increase imputation precision. Relatedness matrices were generated based on five different coefficients and averaged the top five scores from each sample genotype as a metric for gauging degree of relatedness to the reference panel. These measures were plotted against concordance scores from the imputed data filtered for GP scores above 0.9 and averaged across all depth levels. A simple linear regression model was fit to assess potential correlation. Relatedness matrices were calculated using the R package “synbreed”, using options corresponding to measures described by vanRaden, Astle and Balding, Reif, Hayes and Goddard, and Euclidean distances (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wimmer <em>et al.</em> 2012</a>).</p></div><div id="sec-7" class="subsection"><h3>Genome Representation</h3><p id="p-15">Genomic studies improve as the linkage between the genotyped polymorphism and underlying causative gene increases. The extent of LD between two markers therefore constitutes proxy for the correlation of the marker and underlying gene(s) of interest. To assess how well the panel represented variation across the genome, the distribution of LD in the imputed experimental dataset was compared to the SoySNP50k Array positions extracted from the imputed experimental dataset (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Song <em>et al.</em> 2015</a>). SNPs with MAF below 0.05 were filtered out, a quality control step implemented in most genomic studies. Both D' and r<sup>2</sup> were calculated using Plink1.9, and distributions plotted in R3.4 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Team 2017</a>).</p></div><div id="sec-8" class="subsection"><h3>Error and Beagle Posterior Genotype Probability</h3><p id="p-16">To explore the possibility of using GP values as a post imputation filtering metric, proportion of error across depth subsets was plotted against GP. A rolling average window with a width of 500 SNPs was applied to the proportion error after ordering by GP, and a second degree polynomial was fit to describe the relationship in R3.4.</p><div id="sec-9" class="subsection"><h4>Error Type:</h4><p id="p-17">Allele frequencies exhibit some degree of influence on the results of many genomics studies. Therefore, how imputation error skews this metric is of significant interest. Masked and imputed datasets were coded according to the major allele in the reference dataset. Errors were binned into four categories, homozygous major to minor, homozygous minor to homozygous major, homozygous major to heterozygous, and homozygous minor to heterozygous, based on which allele was incorrectly imputed and which allele was true. Because all heterozygous calls were filtered in the initial data generation, no heterozygous to major, or heterozygous to minor category exists.</p></div></div><div id="sec-10" class="subsection"><h3>Power Analysis</h3><p id="p-18">In the interest of determining the potential cost of imputation error, a basic power calculation for minor to major and major to minor errors in a GWAS was performed. Using an R implementation of Purcell’s “Genetic Power Calculator” (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Purcell <em>et al.</em> 2003</a>), power was calculated to detect a moderate effect QTL across minor allele frequency bins from &lt;0.025 to 0.5. Simulations assumed an additive genetic model, 300 genotypes, LD between the QTL and marker of 0.8 D', a significance threshold that mirrored the Bonferroni correction for 1,716,234 SNPs (the final size of the SNP dataset after quality control filtering), and a QTL effect size of 1 standard deviation. Error rates from 1–10% were tested at intervals of 1%, with 100 iterations of the simulation performed at each error level. To investigate the possibility of including more genotypes to overcome power losses associated with imputation error, simulations were also performed for 150, 500, and 1000 genotypes for a 5% error rate at the same conditions as specified above.</p></div><div id="sec-11" class="subsection"><h3>Cost Analysis</h3><p id="p-19">Decreasing cost per sample allows a researcher to expand a study to overcome power loss introduced through the imputation error. To illustrate the impact of this, per sample sequencing costs were calculated using current Illumina NextSeq500 high throughput 300 cycle sequencing kit prices, cost analysis of a custom library prep protocol, and CTAB DNA extraction method (Supplementary Table 3). The retained cost per sample and average raw concordance were plotted as depth decreased.</p></div><div id="sec-12" class="subsection"><h3>Data Availability</h3><p id="p-20">Raw sequencing data directly generated by this project for use in creating the study panel has been deposited in the NCBI Short Read Archive under accession number PRJNA512147. The reference panel used for genotype imputation was generated using previously publicly available sequence data deposited in the NCBI Short Read Archive from study number SRP062245 (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Valliyodan <em>et al.</em> 2016</a>). Supplementary figures and tables can be found in “Supplementary Figures and Tables.” The 114 × 114 relatedness matrix is available in Supplementary File 1. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7975751">https://doi.org/10.25387/g3.7975751</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>SNP Genotyping &amp; Imputation</h3><p id="p-21">The reference panel for imputation was constructed using 106 <em>Glycine max</em> lines sequenced at an average of 17.1X coverage using publicly available sequencing data deposited in the NCBI Short Read Archive (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Valliyodan <em>et al.</em> 2016</a>) (Supplementary Table 1). After quality control measures were applied to the raw and mapped sequence data (see Materials &amp; Methods), a final reference panel of 10,803,148 biallelic homozygous SNPs across 99 lines was generated. SNPs discovered in the reference panel were used to genotype experimental lines in the study panel. This consisted of 114 lines that were sequenced to a depth of at least 1X. Coverages from 0.1X to 1X were analyzed by randomly subsetting reads from the raw sequence data. Of the 10,803,148 million markers discovered in the reference panel, the number of SNPs genotyped by this low coverage study panel subsets ranged from 133,747 to 1,288,463 markers. These subsets also ranged in missing data rates from 95.26 to 67.56% for those markers (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Using the reference panel, genotype values for all missing positions were imputed.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493476/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493476/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>The number of markers and genotyping rate in each low coverage subset from 0.1X to 1X sequencing depth. As coverage decreases, the total number of markers captured and completeness of the SNP panel decreases.</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Close relatedness between the lines in the experimental panel may bias the overall accuracy of the imputation results. To evaluate this, vanRaden relatedness scores were calculated using real and imputed genotypes. The resulting values ranged from -0.44858 to 0.91112, with a median value of -0.02851, mean of -0.00286, and standard deviation of 0.17650. Strong relationships are generally indicated by values over 0.4. Our experimental panel exhibits few strongly related lines, with only 2.7% of all possible pairwise combinations showing a relationship above this threshold. Therefore, we would conclude the majority of our experimental genotypes to be distally/non-related (Supplementary File 1).</p><p id="p-23">An alternative to this whole genome sequencing approach are fixed SNP arrays. However, this method provides less total SNPs for genomic studies and may not capture as much of the genome. High LD between SNPs can be extended to assume a strong correlation to other genomic variation between them. Plotting the density distributions of r<sup>2</sup> and D' LD measures for the Soy50KSNP Array and imputed dataset demonstrated that whole genome sequencing with imputation had a greater concentration of values toward higher linkage values. Generally, a D' or r<sup>2</sup> of over 0.8 between is considered “strong linkage”. The imputed dataset provided 1,716,234 SNPs after common quality control filters, with 36.00% and 85.66% of r<sup>2</sup> and D' values above 0.8, respectively. This is in comparison to the 42,133 SNPs in the fixed array, where 24.20% and 80.00% of r<sup>2</sup> and D' values are above 0.8 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). If high LD indicates a better tagging of underlying variation, the imputed dataset captures the genome’s SNP variation better than the Soy50KSNP Array.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing density plots for LD measures D' (A) and r2 (B) demonstrates that using whole genome sequencing with imputation results in a dataset that has a higher proportion of SNPs is strong pairwise linkage with each other, represented in the heavier tails in red near D' and r2 values of 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508692238" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparing density plots for LD measures D' (A) and r&lt;sup&gt;2&lt;/sup&gt; (B) demonstrates that using whole genome sequencing with imputation results in a dataset that has a higher proportion of SNPs is strong pairwise linkage with each other, represented in the heavier tails in red near D' and r&lt;sup&gt;2&lt;/sup&gt; values of 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2153/F1.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2153/F1.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493517" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Comparing density plots for LD measures D' (A) and r<sup>2</sup> (B) demonstrates that using whole genome sequencing with imputation results in a dataset that has a higher proportion of SNPs is strong pairwise linkage with each other, represented in the heavier tails in red near D' and r<sup>2</sup> values of 1.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Imputation Accuracy</h3><p id="p-25">Prior to imputation, 5% of genotype calls from the skim sequencing data were withheld to assess accuracy. Overall imputation accuracy was consistent for raw and filtered datasets as sequencing depth decreased from 1X until 0.3X, where accuracy drops off by an average of 3.5% from 0.3X to 0.1X (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, Supplementary Table 2). Assessing the error type of this study showed that 53.13% of the errors made were incorrect imputation of the minor allele when the major allele was true. Of the remaining errors, 35.10% were incorrect imputation of the major allele when the minor allele was true, and 11.77% were incorrect imputation of heterozygous calls. No heterozygous to major/minor errors exist as at heterozygous calls were filtered in the initial panels (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Overall accuracy of filtered and raw imputed datasets were plotted across the evaluated depths. For all study panels, concordance rapidly erodes below a sequencing depth of &#x223C;0.3X. B) Examining accuracy in the context of minor allele frequency (MAF) reveals that error occurs at higher rates as MAF approaches a maximum of 0.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508692238" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A) Overall accuracy of filtered and raw imputed datasets were plotted across the evaluated depths. For all study panels, concordance rapidly erodes below a sequencing depth of &#x223C;0.3X. B) Examining accuracy in the context of minor allele frequency (MAF) reveals that error occurs at higher rates as MAF approaches a maximum of 0.5.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2153/F2.medium.gif" width="315" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2153/F2.medium.gif" width="315" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493603" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">A) Overall accuracy of filtered and raw imputed datasets were plotted across the evaluated depths. For all study panels, concordance rapidly erodes below a sequencing depth of ∼0.3X. B) Examining accuracy in the context of minor allele frequency (MAF) reveals that error occurs at higher rates as MAF approaches a maximum of 0.5.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of errors made as categorized by whether the minor/major/heterozygous alleles was misimputed. In over half of all the errors made, Beagle overimputes the minor allele when the major allele is the true genotype. Incorrect heterozygous imputations make up a minor proportion of the total error and would likely be filtered out in inbred panels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508692238" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of errors made as categorized by whether the minor/major/heterozygous alleles was misimputed. In over half of all the errors made, Beagle overimputes the minor allele when the major allele is the true genotype. Incorrect heterozygous imputations make up a minor proportion of the total error and would likely be filtered out in inbred panels.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2153/F3.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2153/F3.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493585" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Proportion of errors made as categorized by whether the minor/major/heterozygous alleles was misimputed. In over half of all the errors made, Beagle overimputes the minor allele when the major allele is the true genotype. Incorrect heterozygous imputations make up a minor proportion of the total error and would likely be filtered out in inbred panels.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Filtering on Beagle’s posterior genotype probability (GP) to improve dataset quality was successful. When imputed positions with a GP score of less than 0.45 were discarded, accuracy improved by an average of 2.50% across sequencing depths. A more stringent filter that only kept positions with a GP score over 0.9 resulted in a 4.26% increase in accuracy (Supplementary Table 2). This practice did reintroduce some missing data, which varied across depth and filtering level. Data loss as a result of post imputation filtering was below 5% for all depths at a filtering level of GP &gt; 0.45, but quickly inflates when filtering for imputation quality of GP &gt; 0.9 to a missing data rate of 20.82% at 0.1X (Supplementary Figure 1). While filtering on Beagle’s posterior genotype probability may reduce falsely imputed genotypes, it must be balance with the reintroduction of missing data it causes.</p><p id="p-29">The error rate at individual marker loci may not be well captured by the overall concordance across all SNPs. Examining concordance in the context of minor allele frequency (MAF) reveals as MAF values approach a maximum of 0.5, concordance decreases. Application of post imputation filters of GP values increases overall accuracy through improved concordance at these increased MAFs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). This trend is uniform across all sequencing depths (Supplementary Figure 2). Through examining imputation accuracy in this manner, it is apparent that higher error rates are occurring at SNP positions at MAFs nearest 0.5 than is described by the average concordance measure.</p><p id="p-30">Error rates in imputation may be influenced by characteristics specific to the population and crop species to which it is applied. The correlation between variants is a cornerstone to the success of imputation. If the correlation between alleles is high then imputation accuracy should also be high and as the correlation between alleles decrease then the accuracy of imputation should also decrease. This correlation between alleles can be measured with LD. Soybean is a historically inbred crop with long ranging LD (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhou <em>et al.</em> 2015</a>). As D' and r<sup>2</sup> approach 1, where neighboring SNPs are in perfect linkage with each other, error rates are at their lowest. Both relationships demonstrate a very strong correlation with R<sup>2</sup> values of 0.98 and 0.89 for r<sup>2</sup> and D' respectively (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), indicating LD is an important factor to consider when applying this technique to other soybean populations or other crop species.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing the smoothed frequency of errors made at individual SNP sites with LD measures D' (A) and r2 (B) demonstrates the strong influence of linkage disequilibrium on imputation accuracy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508692238" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparing the smoothed frequency of errors made at individual SNP sites with LD measures D' (A) and r&lt;sup&gt;2&lt;/sup&gt; (B) demonstrates the strong influence of linkage disequilibrium on imputation accuracy.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2153/F4.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2153/F4.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493642" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Comparing the smoothed frequency of errors made at individual SNP sites with LD measures D' (A) and r<sup>2</sup> (B) demonstrates the strong influence of linkage disequilibrium on imputation accuracy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Relationship of the study genotypes to the reference panel genotypes has been suggested as a strong influencer of imputation accuracy. Plotting calculated values for five unique kinship metrics against concordance for each genotype did not demonstrate any strong linear relationships. The maximum correlation for any of the measures was for Reif’s method, at an R<sup>2</sup> of 0.26. Examination of the standard error, shows that the study population varies narrowly in terms of relatedness to the reference panel. Additionally, assessing the raw values suggests that the study population is weakly related to reference genotypes. This is best illustrated with the vanRaden and Astle &amp; Balding measurements, where a “strong” relationship is usually indicated by values approximately &gt;= 0.4. In both these cases, the largest measure does not exceed 0.18 and 0.16 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">VanRaden 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Astle and Balding 2009</a>). The combination of diminished values and narrow standard error indicates a weak relationship of the study panel to the reference panel (Supplementary Figure 3). The evidence of a weak relationship suggests that relationship was not a strong influencer of the high imputation accuracies obtained.</p></div><div id="sec-16" class="subsection"><h3>GWAS Power</h3><p id="p-33">Understanding the effect error rate has on genomic studies is important when selecting an appropriate genotyping technique. To determine the effect of the error rate of skim sequencing and imputation has on GWAS we performed power simulations of detecting a moderate effect QTL in a panel of 300 individuals. This power study showed significantly decreased power to detect QTL with increasing errors at MAF from 0.1-0.3. This was most pronounced when the minor allele was incorrectly imputed as the major allele. Above 0.3 MAF, power for QTL detection was minimally affected by error (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Studying the effect of three additional sample sizes, while assuming a 5% error rate demonstrates the potential for experimenters to recover power losses through inclusion of more genotypes. Including 500 individuals at this fixed error rate recovers and even slightly improves power at mid-range MAFs over studying 300 genotypes with no genotyping error (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) The power to detect a moderate effect QTL becomes increasingly sensitive to error for both major to minor and vice versa errors at intermediary MAFs. B) Comparing the power to detect the same QTL with 300 samples at a 0% genotyping error vs. 500 samples with a 5% error rate demonstrates that cost savings can be used to increase study sizes in order to recover power losses introduced by the imputation error of both major and minor alleles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508692238" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A) The power to detect a moderate effect QTL becomes increasingly sensitive to error for both major to minor and &lt;em&gt;vice versa&lt;/em&gt; errors at intermediary MAFs. B) Comparing the power to detect the same QTL with 300 samples at a 0% genotyping error &lt;em&gt;vs.&lt;/em&gt; 500 samples with a 5% error rate demonstrates that cost savings can be used to increase study sizes in order to recover power losses introduced by the imputation error of both major and minor alleles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2153/F5.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2153/F5.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2153/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493519" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">A) The power to detect a moderate effect QTL becomes increasingly sensitive to error for both major to minor and <em>vice versa</em> errors at intermediary MAFs. B) Comparing the power to detect the same QTL with 300 samples at a 0% genotyping error <em>vs.</em> 500 samples with a 5% error rate demonstrates that cost savings can be used to increase study sizes in order to recover power losses introduced by the imputation error of both major and minor alleles.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-35">This study illustrates the potential of low coverage sequencing with imputation as an economical approach to obtaining high density SNP genotype information in soybean. Accelerating improvement of complex phenotypes through genomics necessitates high quality, high resolution marker data. However, studies are often limited by the cost required to obtain this information through high coverage sequencing. The combination of low coverage sequencing with imputation presents an option that drastically cuts costs while retaining a high level of accuracy. Implementing a similar method in rice allowed researchers to generate a high quality, dense SNP dataset using 1X depth whole genome sequence (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Huang <em>et al.</em> 2010</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2016</a>). This analysis in soybean, which differs in the inclusion of a reference panel for imputation, determined sequencing depth could be reduced to 0.3X with no significant accuracy losses. Analogous results have been demonstrated in humans, where it was concluded that a reasonably accurate and dense dataset could be obtained from 0.2X coverage supplemented with imputation using a reference panel (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pasaniuc <em>et al.</em> 2012</a>). To our knowledge, this is the first work to examine using imputation with real sequence data at less than 1X coverage in the construction of a high quality, highly affordable SNP dataset in plants. The effect of imputation method and structure of the reference panel have not been specifically examined in the context of application to skim sequencing, providing future avenues for research and improvement.</p><p id="p-36">While SNP arrays and GBS are popular options for obtaining genotype information, high precision genomics demands markers to be in close linkage to the contributing genes. Regions of the genome with sparsely correlated markers may therefore contain overlooked causal variation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hirschhorn and Daly 2005</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Witte 2010</a>). Skim sequencing with imputation, as investigated here, tags a significantly larger portion of the genome in tighter LD than the current soybean 50k array. This effect may be presumed to extend to GBS datasets of a similar size. Such a boost in resolution may therefore reveal QTL in regions of the genome that would not have been captured through smaller datasets.</p><p id="p-37">The accuracy and extensibility of this approach in other soybean populations, as well as other crops is based on several factors. To explore potential limitations in this method, population LD and the relatedness of reference panel to study lines were examined. Both of these factors have been implicated as strong influencers of imputation accuracy due to the innate reliance of the technique on the presence of sample haplotypes within the reference panel, as well as the extent of correlation between observed markers (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">He <em>et al.</em> 2015</a>). The strong inverse relationship observed between the proportion of SNPs incorrectly imputed at a given position and LD measurements suggests that for soybean populations and other crops with shorter range LD, imputation accuracy will likely decrease. There was no significant relationship detected between kinship measures and accuracy. However, the study genotypes exhibited little variation for any of the calculated metrics, which can be seen in the low standard deviations. Without a wide range of values to examine, identifying a clear trend is unlikely. The positive effect of relatedness on imputation accuracy is documented in other literature (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ma <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boison <em>et al.</em> 2015</a>), and should therefore be a consideration in expanding this method to other soybean populations and crop species. The overall weak kinship between study and reference panels in this data may also be viewed as a positive, since high levels of imputation accuracy were achieved despite this populations being interpreted as distally related.</p><p id="p-38">The power to detect a QTL is partially dependent on the allele frequency at that loci (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ardlie <em>et al.</em> 2002</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tabangin <em>et al.</em> 2009</a>). Therefore, the relationship between imputation accuracy, minor allele frequency (MAF), and statistical power may be considered particularly important. In agreement with an analysis performed with maize, the data showed steadily decreasing imputation accuracy as MAF increased with the exception of very rare alleles (MAF &lt; 0.05) (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2012</a>). An opposite tendency was observed with respect to statistical power losses across MAF, so it can be interpreted that at the loci a SNP dataset would display the highest imputation error rates, the GWAS is least affected by them. This trend has also been supported in human imputation analyses looking at sample size inflation factors under different imputation error types (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 2009</a>). In both cases, power consequences were greater for incorrect imputation of the minor allele. It is unclear how a combination of error types at a SNP locus would influence genomic studies. Decisions on the level of decreased coverage that can be tolerated should consequently be made not on the overall average concordance, but by examining the concordance across minor allele frequencies in relation to the maximum allowable error to retain power.</p><p id="p-39">The cost savings associated with this method can be used to include more sample genotypes, not only recovering power losses at low minor allele frequencies, but potentially increasing total power. Similar results in humans have indicated sampling more genotypes with small error is more beneficial over fewer genotypes with perfect accuracy (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Pasaniuc <em>et al.</em> 2012</a>). Comparing the raw accuracy along sequencing depths along with per sample costs, shows that at the previously identified critical threshold of 0.3X coverage, there is only a 0.85% loss of accuracy relative to using a 1X sequence, while costs decreased 57% (Supplementary Figure 4). Moreover, the use of public sequence data to construct a broad reference panel eliminates the cost and limitations of assembling special populations and sequencing the founders to a high coverage to serve as the reference haplotypes.</p></div><div class="section conclusions" id="sec-18"><h2 class="">Conclusion</h2><p id="p-40">Here it is demonstrated that low coverage sequencing accompanied with imputation from a reference panel can be extended below 1X depth in soybean to capture high density, reasonably accurate SNP genotype information economically. The tremendous drop in per sample sequencing cost over high depth methods may allow researchers to expand the number of study genotypes in their investigations, while representing a larger portion of the genome than fixed SNP arrays and GBS data. The potential for success of this genotyping method within and outside of soybean is highly reliant on population LD. Furthermore, researchers should examine accuracy and power within the context of minor allele frequency to make informed decisions about sequencing depth tolerances. As genomics demands increasing SNP panel densities across a wide range of genotypes, skim sequencing with imputation constitutes a financially feasible and highly accurate way to meet these requirements.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">Research reported in this publication was supported by the Nebraska Soybean Board project #1726. The authors also acknowledge Dr. Reka Howard and Dr. Keenan Amundsen for providing their technical perspective during the compilation of project results and manuscript drafting. This work was completed utilizing the Holland Computing Center of the University of Nebraska, which receives support from the Nebraska Research Initiative.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7975751">https://doi.org/10.25387/g3.7975751</a>.</p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-13"><span class="received-label">Received </span>February 13, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-01"><span class="accepted-label">Accepted </span>May 1, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Happ <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.1" data-doi="10.1086/341719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ardlie</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunetta</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seielstad</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Testing for Population Subdivision and Association in Four Case-Control Studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">304</span>–<span class="cit-lpage">311</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/341719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DArdlie%26rft.auinit1%253DK.%2BG.%26rft.volume%253D71%26rft.issue%253D2%26rft.spage%253D304%26rft.epage%253D311%26rft.atitle%253DTesting%2Bfor%2Bpopulation%2Bsubdivision%2Band%2Bassociation%2Bin%2Bfour%2Bcase-control%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F341719%26rft_id%253Dinfo%253Apmid%252F12096349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/341719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12096349&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176977700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.2" data-doi="10.1214/09-STS307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Astle</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population Structure and Cryptic Relatedness in Genetic Association Studies.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">471</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/09-STS307</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D24%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-STS307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-STS307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.3" data-doi="10.1007/s00122-015-2488-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruperao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.-K. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-Resolution Skim Genotyping by Sequencing Reveals the Distribution of Crossovers and Gene Conversions in Cicer Arietinum and Brassica Napus.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1039</span>–<span class="cit-lpage">1047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2488-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D1039%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2488-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2488-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.4" data-doi="10.3168/jds.2014-9213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boison</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">D. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utsunomiya</span>  <span class="cit-name-given-names">A. H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalheiro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>  <span class="cit-name-given-names">H. H. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Strategies for Single Nucleotide Polymorphism (SNP) Genotyping to Enhance Genotype Imputation in Gyr (Bos Indicus) Dairy Cattle: Comparison of Commercially Available SNP Chips.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">4969</span>–<span class="cit-lpage">4989</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2014-9213</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D98%26rft.spage%253D4969%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-9213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2014-9213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: A Flexible Trimmer for Illumina Sequence Data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.6" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype Imputation with Millions of Reference Samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.7" data-doi="10.3389/fpls.2018.00369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cericola</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenk</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fè</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Optimized Use of Low-Depth Genotyping-by-Sequencing for Genomic Prediction Among Multi-Parental Family Pools and Single Plants in Perennial Ryegrass (Lolium Perenne L.).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00369</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.8" data-doi="10.1371/journal.pone.0160733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluating Imputation Algorithms for Low-Depth Genotyping-By-Sequencing (GBS) Data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0160733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0160733</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0160733%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0160733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0160733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.9" data-doi="10.1007/s13580-017-0297-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">T.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genotyping-by-Sequencing: A Promising Tool for Plant Genetics Research and Breeding.</span> <abbr class="cit-jnl-abbrev">Hortic. Environ. Biotechnol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13580-017-0297-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BEnviron.%2BBiotechnol.%26rft.volume%253D58%26rft.spage%253D425%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13580-017-0297-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13580-017-0297-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.10" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Variant Call Format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.11" data-doi="10.1534/genetics.113.158014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauleon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient Imputation of Missing Markers in Low-Coverage Genotyping-by-Sequencing Data from Multi-Parental Crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.158014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158014%26rft_id%253Dinfo%253Apmid%252F24583583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvNDAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.12" data-doi="10.1186/1471-2105-8-428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaunt</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">I. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cubic Exact Solutions for the Estimation of Pairwise Haplotype Frequencies: Implications for Linkage Disequilibrium Analyses and a Web Tool ‘CubeX.’</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-8-428</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DGaunt%26rft.auinit1%253DT.%2BR.%26rft.volume%253D8%26rft.spage%253D428%26rft.epage%253D428%26rft.atitle%253DCubic%2Bexact%2Bsolutions%2Bfor%2Bthe%2Bestimation%2Bof%2Bpairwise%2Bhaplotype%2Bfrequencies%253A%2Bimplications%2Bfor%2Blinkage%2Bdisequilibrium%2Banalyses%2Band%2Ba%2Bweb%2Btool%2B%2527CubeX%2527.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-428%26rft_id%253Dinfo%253Apmid%252F17980034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17980034&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.13" data-doi="10.1016/j.tig.2010.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Population Genetics of Genomics-Based Crop Improvement Methods.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2010.12.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D98%26rft.epage%253D106%26rft.atitle%253DPopulation%2Bgenetics%2Bof%2Bgenomics-based%2Bcrop%2Bimprovement%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.12.003%26rft_id%253Dinfo%253Apmid%252F21227531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21227531&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288685900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.14" data-doi="10.1186/s12864-015-1366-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mette</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bothe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prospects and Limits of Marker Imputation in Quantitative Genetic Studies in European Elite Wheat (Triticum Aestivum L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1366-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1366-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1366-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.15" data-doi="10.2135/cropsci2011.07.0358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Factors Affecting the Accuracy of Genotype Imputation in Populations from Several Maize Breeding Programs.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.07.0358</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D654%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.07.0358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.07.0358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.16" data-doi="10.1038/nrm1583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirschhorn</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome-Wide Association Studies for Common Diseases and Complex Traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHirschhorn%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D95%26rft.epage%253D108%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bcommon%2Bdiseases%2Band%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1583%26rft_id%253Dinfo%253Apmid%252F15716906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716906&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.17" data-doi="10.1534/g3.111.001198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype Imputation with Thousands of Genomes.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.001198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D1%26rft.spage%253D457%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001198%26rft_id%253Dinfo%253Apmid%252F22384356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.001198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22384356&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.18" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">“A Flexible and Accurate Genotype Imputation Method for the Next Generation of Genome-Wide Association Studies.”</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000529</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.7.2153.19" data-doi="10.1534/genetics.113.158014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauleon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient Imputation of Missing Markers in Low-Coverage Genotyping-by-Sequencing Data from Multi-Parental Crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">404</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.158014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158014%26rft_id%253Dinfo%253Apmid%252F24583583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvNDAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.20" data-doi="10.1016/j.ajhg.2009.09.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Relationship between Imputation Error and Statistical Power in Genetic Association Studies in Diverse Populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2009.09.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DL.%26rft.volume%253D85%26rft.issue%253D5%26rft.spage%253D692%26rft.epage%253D698%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bimputation%2Berror%2Band%2Bstatistical%2Bpower%2Bin%2Bgenetic%2Bassociation%2Bstudies%2Bin%2Bdiverse%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.09.017%26rft_id%253Dinfo%253Apmid%252F19853241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.09.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19853241&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.21" data-doi="10.1038/ng.695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-Wide Association Studies of 14 Agronomic Traits in Rice Landraces.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">967</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2B14%2Bagronomic%2Btraits%2Bin%2Brice%2Blandraces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.695%26rft_id%253Dinfo%253Apmid%252F20972439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972439&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.22" data-doi="10.1038/214674a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Generalized F-Statistics and the Theory of Inbreeding and Selection.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">674</span>–<span class="cit-lpage">678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/214674a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJain%26rft.auinit1%253DS.%2BK.%26rft.volume%253D214%26rft.issue%253D5089%26rft.spage%253D674%26rft.epage%253D678%26rft.atitle%253DGeneralized%2BF-statistics%2Band%2Bthe%2Btheory%2Bof%2Binbreeding%2Band%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F214674a0%26rft_id%253Dinfo%253Apmid%252F6049073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/214674a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6049073&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keim</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> “<span class="cit-article-title">A Rapid Protocol for Isolating Soybean DNA</span>.” <abbr class="cit-jnl-abbrev">Soybean Genet. Newsl.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">152</span>. <a href="https://ci.nii.ac.jp/naid/10015372412/">https://ci.nii.ac.jp/naid/10015372412/</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoybean%2BGenet.%2BNewsl.%26rft.volume%253D15%26rft.spage%253D150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.24" data-doi="10.1007/978-1-61779-516-9_19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Generating Barcoded Libraries for Multiplex High-Throughput Sequencing.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">840</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-61779-516-9_19</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DKnapp%26rft.auinit1%253DM.%26rft.volume%253D840%26rft.spage%253D155%26rft.epage%253D170%26rft.atitle%253DGenerating%2Bbarcoded%2Blibraries%2Bfor%2Bmultiplex%2Bhigh-throughput%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-516-9_19%26rft_id%253Dinfo%253Apmid%252F22237533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-516-9_19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22237533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321877600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.25" data-doi="10.1007/978-1-61779-089-8_18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozarewa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Amplification-Free Library Preparation for Paired-End Illumina Sequencing.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">733</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-61779-089-8_18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DKozarewa%26rft.auinit1%253DI.%26rft.volume%253D733%26rft.spage%253D257%26rft.epage%253D266%26rft.atitle%253DAmplification-free%2Blibrary%2Bpreparation%2Bfor%2Bpaired-end%2BIllumina%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-089-8_18%26rft_id%253Dinfo%253Apmid%252F21431776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-089-8_18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21431776&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289444000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.26" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> “<span class="cit-article-title">Fast Gapped-Read Alignment with Bowtie 2</span>.” <abbr class="cit-jnl-abbrev">Nature Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.27" data-doi="10.3389/fpls.2018.00650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic Basis Underlying Correlations Among Growth Duration and Yield Traits Revealed by GWAS in Rice (Oryza Sativa L.).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00650</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D650%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.28" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: Knowledge and Prospects.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-385531-2.00002-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.29" data-doi="10.3168/jds.2012-6316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of Different Methods for Imputing Genome-Wide Marker Genotypes in Swedish and Finnish Red Cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4666</span>–<span class="cit-lpage">4677</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2012-6316</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D96%26rft.spage%253D4666%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2012-6316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2012-6316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.30" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce Framework for Analyzing next-Generation DNA Sequencing Data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.31" data-doi="10.1038/ng.2283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extremely Low-Coverage Sequencing and Imputation Increases Power for Genome-Wide Association Studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2283</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPasaniuc%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D631%26rft.epage%253D635%26rft.atitle%253DExtremely%2Blow-coverage%2Bsequencing%2Band%2Bimputation%2Bincreases%2Bpower%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2283%26rft_id%253Dinfo%253Apmid%252F22610117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22610117&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.32" data-doi="10.1038/srep19199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patil</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuong</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valliyodan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.-D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic-Assisted Haplotype Analysis and the Development of High-Throughput SNP Markers for Salinity Tolerance in Soybean.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">19199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep19199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D19199%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep19199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep19199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.33" data-doi="10.1371/journal.pone.0003551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">Y.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papasian</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.-W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>. <span class="cit-article-title">“Analyses and Comparison of Accuracy of Different Genotype Imputation Methods.”</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003551</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPei%26rft.auinit1%253DY.%2BF.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3551%26rft.epage%253De3551%26rft.atitle%253DAnalyses%2Band%2Bcomparison%2Bof%2Baccuracy%2Bof%2Bdifferent%2Bgenotype%2Bimputation%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003551%26rft_id%253Dinfo%253Apmid%252F18958166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18958166&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.34" data-doi="10.1101/201178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruano-Rubio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Scaling Accurate Genetic Variant Discovery to Tens of Thousands of Samples.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/201178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F201178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyMDExNzh2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2153.35" data-doi="10.1093/bioinformatics/19.1.149"><div class="cit-metadata"><cite>Purcell, S., S. S. Cherny, and P. C. Sham, 2003 “Genetic Power Calculator: Design of Linkage and Association Genetic Mapping Studies of Complex Traits.” <em>BIOINFORMATICS APPLICATIONS NOTE</em>. Vol. 19. <a href="http://svn.donarmstrong.com/don/trunk/projects/research/linkage/papers/genetic_power_calculator_purcell_sham_bioinfor_19_1_149_2003_pmid_12499305.pdf">http://svn.donarmstrong.com/don/trunk/projects/research/linkage/papers/genetic_power_calculator_purcell_sham_bioinfor_19_1_149_2003_pmid_12499305.pdf</a>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/19.1.149</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D150%26rft.atitle%253DGenetic%2BPower%2BCalculator%253A%2Bdesign%2Bof%2Blinkage%2Band%2Bassociation%2Bgenetic%2Bmapping%2Bstudies%2Bof%2Bcomplex%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F19.1.149%26rft_id%253Dinfo%253Apmid%252F12499305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/19.1.149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12499305&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180463900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.36" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A Tool Set for Whole-Genome Association and Population-Based Linkage Analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.37" data-doi="10.1016/j.molp.2017.06.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasheed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Crop Breeding Chips and Genotyping Platforms: Progress, Challenges, and Perspectives.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1047</span>–<span class="cit-lpage">1064</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2017.06.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D10%26rft.spage%253D1047%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2017.06.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2017.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.38" data-doi="10.1534/g3.115.019000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, D. L. Hyten, G. Jia, C. V. Quigley, E. W. Fickus <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fingerprinting Soybean Germplasm and Its Utility in Genomic Research.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1999</span>–<span class="cit-lpage">2006</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.019000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019000%26rft_id%253Dinfo%253Apmid%252F26224783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMTk5OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.39" data-doi="10.3835/plantgenome2014.05.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">Kelly</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Huihui</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero Navarro</span>  <span class="cit-name-given-names">J. Alberto</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">Dong</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cinta Romay</span>  <span class="cit-name-given-names">Maria</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">Sarah</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">Charlotte</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2014</span> “Novel Methods to Optimize Genotypic Imputation for Low-Coverage, Next-Generation Sequence Data in Crop Plants.” <em>The Plant Genome</em> 7 (3).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLANT%2BGENOME%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabangin</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite> <span class="cit-pub-date">2009</span> “The Effect of Minor Allele Frequency on the Likelihood of Obtaining False Positives.” <em>BMC Proceedings</em> 3 Suppl 7 (Suppl 7): S41.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1753–6561–3-S7–S41</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2153.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Team</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>., <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0,28&amp;cluster=8918609904990403039">https://scholar.google.com/scholar?hl=en&amp;as_sdt=0,28&amp;cluster=8918609904990403039</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.42" data-doi="10.1038/ng.746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-Wide Association Study of Leaf Architecture in the Maize Nested Association Mapping Population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.746</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTian%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D162%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bleaf%2Barchitecture%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.746%26rft_id%253Dinfo%253Apmid%252F21217756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217756&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.43" data-doi="10.1038/srep23598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valliyodan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patil</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Landscape of Genomic Diversity and Trait Discovery in Soybean.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">23598</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep23598</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D23598%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep23598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep23598&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.44" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient Methods to Compute Genomic Predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> The Power of Inbreeding: NGS-Based GWAS of Rice Reveals Convergent Evolution during Rice Domestication. Mol. Plant. 9: 975–985.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2016.04.018</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.46" data-doi="10.1093/bioinformatics/bts335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Synbreed: A Framework for the Analysis of Genomic Prediction Data Using R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2087</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts335</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts335%26rft_id%253Dinfo%253Apmid%252F22689388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22689388&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.47" data-doi="10.1146/annurev.publhealth.012809.103723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">John S.</span></span></li></ol><cite> <span class="cit-pub-date">2010</span> “Genome-Wide Association Studies and Beyond.” <em>Annual Review of Public Health</em> 31: 9–20 4 p following 20.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.publhealth.012809.103723</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bpublic%2Bhealth%26rft.stitle%253DAnnu%2BRev%2BPublic%2BHealth%26rft.aulast%253DWitte%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D31%26rft.spage%253D9%26rft.epage%253D20%2B4%2Bp%2Bfollowing%2B20%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Band%2Bbeyond.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.publhealth.012809.103723%26rft_id%253Dinfo%253Apmid%252F20235850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.publhealth.012809.103723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20235850&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277908800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2153.48" data-doi="10.1038/nbt.3096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Resequencing 302 Wild and Cultivated Accessions Identifies Genes Related to Domestication and Improvement in Soybean.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">408</span>–<span class="cit-lpage">414</span> (erratum: Nat. Biotechnol. 34: 441).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D408%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3096%26rft_id%253Dinfo%253Apmid%252F25643055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25643055&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2153.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2135" title="Genetically Distinct Behavioral Modules Underlie Natural Variation in Thermal Performance Curves" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2161" title="RNA-Sequencing Analysis of Paternal Low-Protein Diet-Induced Gene Expression Change in Mouse Offspring Adipocytes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ej-XNH97BJyIqSR8LaRdrFLT2MNqu4jTiJCD0r1fgDI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2153" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2153" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2153" class="active" data-icon-position="" data-hide-link-title="0">Generating High Density, Low Cost Genotype Data in Soybean [Glycine max (L.) Merr.]</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493461" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6Ur2xAcPBgsrRUP79yqypIJ7EtMFNzrn3dfzsEEUJ1I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2153.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2153" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-i7uSMhtNVdn1RXQPaZTzgogasIDZXC_6bo7dRadD3ss" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2153" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2153.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493461" id="node493461--2" data-pisa="ggg;9/7/2153" data-pisa-master="ggg;g3.119.400093" data-apath="/ggg/9/7/2153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Generating High Density, Low Cost Genotype Data in Soybean [<em>Glycine max</em> (L.) Merr.]</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5897-2617" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mary M. Happ</span>, <span class="highwire-citation-author" data-delta="1">Haichuan Wang</span>, <span class="highwire-citation-author" data-delta="2">George L. Graef</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6324-9389" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David L. Hyten</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2153-2160;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400093 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493461--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mary M. Happ </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BM.%2BHapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Happ%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMary%2BM.%2BHapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5897-2617" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mary M. Happ</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Haichuan Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haichuan%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaichuan%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">George L. Graef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BL.%2BGraef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graef%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BL.%2BGraef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David L. Hyten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BHyten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyten%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BL.%2BHyten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6324-9389" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David L. Hyten</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.hyten@unl.edu" class="" data-icon-position="" data-hide-link-title="0">david.hyten@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2153" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2153&amp;title=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2153&amp;title=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2153&amp;title=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2153&amp;text=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2153&amp;title=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2153&amp;t=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2153&amp;title=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2153&amp;title=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493461/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493461" id="node493461--3" data-pisa="ggg;9/7/2153" data-pisa-master="ggg;g3.119.400093" data-apath="/ggg/9/7/2153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Generating High Density, Low Cost Genotype Data in Soybean [<em>Glycine max</em> (L.) Merr.]</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5897-2617" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mary M. Happ</span>, <span class="highwire-citation-author" data-delta="1">Haichuan Wang</span>, <span class="highwire-citation-author" data-delta="2">George L. Graef</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6324-9389" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David L. Hyten</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2153-2160;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400093 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493461--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mary M. Happ </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BM.%2BHapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Happ%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMary%2BM.%2BHapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5897-2617" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mary M. Happ</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Haichuan Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haichuan%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaichuan%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">George L. Graef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BL.%2BGraef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graef%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BL.%2BGraef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David L. Hyten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>University of Nebraska-Lincoln, Lincoln, NE 68503</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BHyten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyten%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BL.%2BHyten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6324-9389" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David L. Hyten</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.hyten@unl.edu" class="" data-icon-position="" data-hide-link-title="0">david.hyten@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2153&amp;rft.epage=2160&amp;rft.atitle=Generating%20High%20Density%2C%20Low%20Cost%20Genotype%20Data%20in%20Soybean%20%5BGlycine%20max%20%28L.%29%20Merr.%5D&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Happ%2C+Mary+M.&amp;rft.au=Wang%2C+Haichuan&amp;rft.au=Graef%2C+George+L.&amp;rft.au=Hyten%2C+David+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493461/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493461/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493461/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493461/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493461/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493461/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493461/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493461/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493461/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493461/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493461/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493461/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2153" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials &amp; Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2153.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2153.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2153.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
