<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2071" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2071.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2071.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2071.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/7/2071/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2071.atom" />
<meta name="HW.pisa" content="ggg;9/7/2071" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies" />
<meta name="DC.Identifier" content="10.1534/g3.119.400166" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Sakhanenko et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We describe an information-theory-based method and associated software for computationally identifying sister spores derived from the same meiotic tetrad. The method exploits specific DNA sequence features of tetrads that result from meiotic centromere and allele segregation patterns. Because the method uses only the genomic sequence, it alleviates the need for tetrad-specific barcodes or other genetic modifications to the strains. Using this method, strains derived from randomly arrayed spores can be efficiently grouped back into tetrads." />
<meta name="DC.Contributor" content="Nikita A. Sakhanenko" />
<meta name="DC.Contributor" content="Gareth A. Cromie" />
<meta name="DC.Contributor" content="Aimée M. Dudley" />
<meta name="DC.Contributor" content="David J. Galas" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We describe an information-theory-based method and associated software for computationally identifying sister spores derived from the same meiotic tetrad. The method exploits specific DNA sequence features of tetrads that result from meiotic centromere and allele segregation patterns. Because the method uses only the genomic sequence, it alleviates the need for tetrad-specific barcodes or other genetic modifications to the strains. Using this method, strains derived from randomly arrayed spores can be efficiently grouped back into tetrads.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2071" />
<meta name="citation_id" content="9/7/2071" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2071" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2071.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2071.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2071.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400166" />
<meta name="citation_pmid" content="31109921" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="2071" />
<meta name="citation_lastpage" content="2088" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nikita A. Sakhanenko" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, WA 98122" />
<meta name="citation_author_email" content="aimee.dudley@gmail.com" />
<meta name="citation_author_email" content="nsakhanenko@pnri.org" />
<meta name="citation_author" content="Gareth A. Cromie" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, WA 98122" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5265-1970" />
<meta name="citation_author" content="Aimée M. Dudley" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, WA 98122" />
<meta name="citation_author_email" content="aimee.dudley@gmail.com" />
<meta name="citation_author_email" content="nsakhanenko@pnri.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3644-0625" />
<meta name="citation_author" content="David J. Galas" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, WA 98122" />
<meta name="citation_reference" content="citation_author=A. J. Bell;citation_pages=921-926;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Physical review. E, Statistical, nonlinear, and soft matter physics;citation_journal_abbrev=Phys Rev E Stat Nonlin Soft Matter Phys;citation_author=A. Clauset;citation_author=M. E. J. Newman;citation_author=C. Moore;citation_title=Finding community structure in very large networks.;citation_pages=066111-066111;citation_volume=70;citation_year=2004;citation_issue=6 Pt 2;citation_pmid=15697438;citation_doi=10.1103/PhysRevE.70.066111" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=M. Clément-Ziza;citation_author=F. X. Marsellach;citation_author=S. Codlin;citation_author=M. A. Papadakis;citation_author=S. Reinhardt;citation_title=Natural genetic variation impacts expression levels of coding, non-coding, and antisense transcripts in fission yeast;citation_pages=764-764;citation_volume=10;citation_year=2014;citation_issue=11;citation_pmid=25432776;citation_doi=10.15252/msb.20145123" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. R. Fowler;citation_author=M. Sasaki;citation_author=N. Milman;citation_author=S. Keeney;citation_author=G. R. Smith;citation_title=Evolutionary diverse determinants of meiotic DNA break and recombination landscapes across the genome.;citation_volume=24;citation_year=2014;citation_pmid=25024163;citation_doi=10.1101/gr.172122.114" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Biol.;citation_author=D. J. Galas;citation_author=N. A. Sakhanenko;citation_author=A. Skupin;citation_author=T. Ignac;citation_title=Describing the Complexity of Systems: Multivariable “Set Complexity” and the Information Basis of Systems Biology.;citation_pages=118-140;citation_volume=21;citation_year=2014;citation_doi=10.1089/cmb.2013.0039" />
<meta name="citation_reference" content="citation_journal_title=Entropy (Basel);citation_author=D. J. Galas;citation_author=N. A. Sakhanenko;citation_title=Symmetries among Multivariate Information Measures Explored Using Möbius Operators.;citation_pages=88;citation_volume=21;citation_year=2019;citation_doi=10.3390/e21010088" />
<meta name="citation_reference" content="Jakulin, A., and I. Bratko, 2004 Quantifying and visualizing attribute interactions: an approach based on entropy. Computing Research Repository cs.AI/0308002 v3. http://arxiv.org/abs/cs.AI/0308002" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. L. Ludlow;citation_author=A. C. Scott;citation_author=G. A. Cromie;citation_author=E. W. Jeffery;citation_author=A. Sirr;citation_title=High-throughput tetrad analysis.;citation_pages=671-675;citation_volume=10;citation_year=2013;citation_pmid=23666411;citation_doi=10.1038/nmeth.2479" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Mancera;citation_author=R. Bourgon;citation_author=A. Brozzi;citation_author=W. Huber;citation_author=L. M. Steinmetz;citation_title=High-resolution mapping of meiotic crossovers and non-crossovers in yeast.;citation_pages=479-485;citation_volume=454;citation_year=2008;citation_issue=7203;citation_pmid=18615017;citation_doi=10.1038/nature07135" />
<meta name="citation_reference" content="citation_journal_title=Psychometrika;citation_author=W. J. McGill;citation_title=Multivariate information transmission.;citation_pages=97-116;citation_volume=19;citation_year=1954;citation_doi=10.1007/BF02289159" />
<meta name="citation_reference" content="citation_author=N. A. Sakhanenko;citation_author=D. J. Galas;citation_pages=161-164;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Biol.;citation_author=N. A. Sakhanenko;citation_author=D. J. Galas;citation_title=Biological data analysis as an information theory problem: Multivariable dependence measures and the Shadows algorithm.;citation_pages=1005-1024;citation_volume=22;citation_year=2015;citation_doi=10.1089/cmb.2015.0051" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. Sirr;citation_author=A. C. Scott;citation_author=G. A. Cromie;citation_author=C. L. Ludlow;citation_author=V. Ahyong;citation_title=Natural variation in SER1 and ENA6 underlie condition-specific growth defects in Saccharomyces cerevisiae.;citation_volume=8;citation_year=2017;citation_pmid=29138237;citation_doi=10.1534/g3.117.300392" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=A. C. Scott;citation_author=C. L. Ludlow;citation_author=G. A. Cromie;citation_author=A. M. Dudley;citation_title=BEST: Barcode Enabled Sequencing of Tetrads.;citation_pages=51401;citation_volume=87;citation_year=2014;citation_doi=10.3791/51401" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="We describe an information-theory-based method and associated software for computationally identifying sister spores derived from the same meiotic tetrad. The method exploits specific DNA sequence features of tetrads that result from meiotic centromere and allele segregation patterns. Because the method uses only the genomic sequence, it alleviates the need for tetrad-specific barcodes or other genetic modifications to the strains. Using this method, strains derived from randomly arrayed spores can be efficiently grouped back into tetrads." />
<meta name="og-title" property="og:title" content="Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2071" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We describe an information-theory-based method and associated software for computationally identifying sister spores derived from the same meiotic tetrad. The method exploits specific DNA sequence features of tetrads that result from meiotic centromere and allele segregation patterns. Because the method uses only the genomic sequence, it alleviates the need for tetrad-specific barcodes or other genetic modifications to the strains. Using this method, strains derived from randomly arrayed spores can be efficiently grouped back into tetrads." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mLupRoAGVro44mHk_C2TIAW4QQ8GRHYHSTsOaqCgslg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493673","apath":"\/ggg\/9\/7\/2071.atom","pisa":"ggg;9\/7\/2071","ac":{"\/ggg\/9\/7\/2071.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2071.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2071"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2071"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2071":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GI+mu1B0jwe2YwyJRo7JfMZOsjq2i54qF6VjwH7JocVmrhj11D8npJWF8H2+WQvTrN1tdrtdRld8s2bo7IhQy\\\\\\\/VoVzZn7xtmu+MFHBTZe8cbhnuryDoz8ZBC9FUTPI4Gec1YO6QFqNpS4OXj8avEasDq4X5xXtF+IwXWtMdp\\\\\\\/AgPOBsXyGBFfB7cEe2Bgq\\\\\\\/Wm61Ys0Iyb5ZnYiblZSIxQWl7HXZC9rVZUnh\\\\\\\/xsbHQFHjZurQIdAjv4N7T8wTXFVUG+gxSTkwTx6uZLZ8MF8\\\\\\\/BOgOHX\\\\\\\/iC93YcUhDQaUq8zsOZnryvVcIop5AFM+cnqe1F9522r9XD9SqzVDImG0F3jdWB8HOEyKEUzUii0aXe\\\\\\\/fCNCdz2j5WHtjmhvzUBluEL6unPE3P3o2o+5ojj1T\\\\\\\/8+bZ6597AnEfl3lNJKHnKivCw47e4JE6fOXpDwL5VcVJ1iAUaEmJyvaHLFyz5HUTsi\\\\\\\/59HUNSAGCnqVXWNminUzNaqp\\\\\\\/8ajpj1X9zBBCBsN17JcnSYdjMZ6Bjmp7MuC\\\\\\\/XQ2eAfRaIO2KaWiNUEv+mZgV6mjiwmarVrVFLdlna0oHeqDfRBxbbkVpai0DZH1VdT3ZFIOeIYWVuJnKJf6kuAO0FAASU3l86FCMse4Gpt9IGXpiN5teGRSP58ll2LlmW62yzl8lExoEL4ti2NmDmbaxpN+uuKP9rVWxJouLTVAPDFYx644LDZrcx+fY+UAIJQvBTNGR8bwa2QM+WMn183il4jl0iHVHuvEoO3tcecFVStJTGW61NTVGQnylkVTrvRFJ\\\\\\\/YBRzPBldn7flLCcXJSgEgmCbTX+PYGfCXXbLK9nXdkhF\\\\\\\/32lefN5O93f9n+Tkmsq50vp6g1q84mll3+z\\\\\\\/I6L96mrLoAFIZ3qCcrgqdDMIUTzdafspI5x7jxdT5G2MHrxowEJbma2RMnnKBO7ms61nDYSOuAV1JLJ7pNDCZU8zH3JMXv1GnJCcXCU7AvzsEcv2lub90G3FSV+q1pn6YXXRsgsZgEHnRWg\\\\\\\/u409EKvm6RaRjafTkIB6d9Wx8SPvm3jeBPfKrkA\\\\\\\/PnY9nazHXAHlnGJbMYrmLlTsTtj2srEWbqBnrsBY7BWsf4DVqHlrm9Vh\\\\\\\/9mzNTrZiVrOqfM3rxUR\\\\\\\/+LyhuEb6OdGgwS5UccOo5RLENFiCjGh7JaQLH9TwiW+dq0WO47M8cNLOknz5L0MQA3fO1K9LrGoMkq2uFq1VUMSnNoHWI4fltxM71mwLWm3D3Y\\\\\\\/iEFofpZ77BIgRDmwNO8nljW\\\\\\\/nu9Q6XuI96\\\u0022,\\\u0022iv\\\u0022:\\\u0022BLg8jIOSsnGaVICLyNz++w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002274c791f299e1e84542d4326e615e5e7bafc1a09cf755db540578b013edf40944\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IA5mgHEjqZ4htC0j7hTl9o9cc0kwpcAQ1V72E\\\\\\\/ZwHDtWE31+MYiH6PJqA7P9Q3467fvqRDj7\\\\\\\/EmoGfvhV4q+i75xXdVqE0tBHQhIvbs7jc2ZYwMRheNV+GRRjSKJfpPdnzuxrokzFiSjBXOFg0ZYc8dPQpGNlq1jtagqrob\\\\\\\/d7w5syzWLE5tlGRRYDqvrwNAA9s3Td+SUA4aZqDQqq4jnJeeFC5xBV\\\\\\\/pcnTJRfWNbrVQbQqdIFMQoIm+wTo2uobUqSLCA4144frEJxBXdb48wTXqnq1c9+ZEdu0\\\\\\\/p3OmaMbNYn1BBbr8nBMJF67qU8z5c3v7AFyet6AwlvQUeb4AFJ3G2BPytLlM2KtcZdLyK\\\\\\\/DOiKK\\\\\\\/HyEqAh2rxqHbWWVITbC0Wyt1QOISNVGEmUuOGC668su2Ws5Sjrywkw1qqAgivS088WB9RWidzxEtHn291eh3Iwy09CDwayqYKJ5kU5HXNx7HH7UBSYm4HMu2T0wsqGAu5UiILooTSKZ0qr+NfoUPXFYB5pglf1WJxyW+zv3uXallK3wC6DJglI7UKEAGCG7l2HrzG2sqpCyrLfR7qL\\\\\\\/fmkuoDzRJY+X8V7NSt0x8v0on6pobh6ICKWQWHJ73PtkR9bGgiOAj+HojVnRuZ7AlD5Pkzpsatm6K8\\\\\\\/BILPtDRTbrZZSwFjazererctyqmrkN800g+HAXkxcHljatn62\\\\\\\/tZ+3T6MZwkiRJy67daEIqy60Cx9ipIEfbbUb4l3sqaGtu6lUQkpNvlKKEpJspZhJwZvx6MHJo4iMhvFcIJoMhIRlgzzeTzm0wyduVpzb3R4Z94MM2GzYcH9sOnTGhsSM5xRjVan+YgIQNB8EZEtmCqK\\\\\\\/4WVRKZ58sS+s\\\\\\\/RxNMI++J00XbWePVu6MeWVoWqw81gNWdNDtD0S5Fte5e3NCYxoR2QVWUzsmakyAJgWiXdIcEL7wjaxYtN41sDK3HoWQMdyc1AOh5xJYN5o6c7A6OnlZWA8NopZCwdASzClpdq4OQrQ0Mk+wyiAylFPgW6UWK6C\\\\\\\/v\\\\\\\/a0qP7ZUpJ3RnZlGCg3D0ye+i\\\\\\\/UxOr9NuCBCypaxwSspJeqQYeHclgdfT4dTotsoMsP8sqFCqYxqyaiumjm9EN4HrTFaPAWM9UvUomqSG6lL3da3hYHgxd7pxB\\\\\\\/++UWWNoVXcDx0\\\\\\\/srZ\\\\\\\/9J\\\\\\\/NXLfgUiY4bIk6l3kMrEBhx5MmMSntbS32HldPZ+ETWJIWb6F+qatLLJayR6spJNz\\\\\\\/clWkWHrrpNU4\\\\\\\/4qrEPm1jGGDYAX0muJE32r4ecgzhinBWs5gbN0wBpI3K3392b0v5qEqiGdtSL2B4VBxNT8qTeoq2epZfw+2IRIKF+rgNvM+YTC7oZ\\\\\\\/5FjoQug4iuqWV9LNXQuOKeqef6arhppb8hO2DB+iUrq80hwLLr3GHD6wXRD5sEHlxq5SqD+UirrqKoiEymPzKedJdq4fiueYDgwilA9E8uaMMcDwclFBcrZi6nOhB4oSE2eC+NxiRUSCx4SY5Pbi8udkDOxgkWC8\\\\\\\/EfmkdugNk9WLHE+Tshf3a8YLul7Uz1hmQ63OwVv0Ds8vJDxA6m4ccMViTmYpFiBM1H5n1yhDLxlWMuAIbvPJvBbZwrVcQQni2PONJaXjD7E8SeZNOX2GTof35tKQM0dSG4JYLXeZh8tUQKS4iQqirDfbWoE26QFvGqDnqBdxYZl+H+UCLh\\\\\\\/388vdJfM\\\\\\\/irwsaEH7GPRqR15DMUb9XZS\\\\\\\/t5qtSeFi+DZ756nu\\\\\\\/bZETcz3In9tbc9Id7SaKvyqsgDWOMWxFInx1NlsBaD9Z3SZxByLeC5XdqG43Uy7TVZU6PkaCZCJLyXnueuYkK0ZvE1+LVnwRj9vuAAD+j2hDYl5wvIoG6fUtKJL2cCVdRxmDTKr8SJT\\\\\\\/0h9l0tAKZ9517IyTx9RrEJE3pBYgk69561QaEhj4TLH\\\\\\\/fjmEw1x9m5YBXKdk8Rp5gFfFYaUdhYbcNt3Zc2gkeXCZc3u0w0Yc7oYMLmPTB4crSZuLQjSTGzFPUg32ixsQY7QVKRdDwGE2qzRLiCMiYwCB\\\\\\\/Pr9MvfnvwuKSjRsaJnjBKsAI\\\\\\\/r\\\\\\\/sM1w65ge7qsz3gx5lZjHJ4duoRnxDhLLE455NuA6hs1AjNwvcuq\\\\\\\/ApGsWNAWMMmHg9IsfIaBjROcwScLQSuQsa6QwrgHme6uumKnmeMUmsIfp\\\\\\\/TfwQ5dDr9TJf7YMB\\\\\\\/5ffK7SZEyqQ4UMgF\\\\\\\/1U8UVnyzIopvKkOXeUgouxr0lmgEZmr796sfNMZJRIx0\\\\\\\/mPeliWcSPR3pVk\\\\\\\/w+FFwnol152nRskEbhlRbwBBlMug7po8APQTILj+a9RislSS852gnFM+iyYkH8YpM3pwE5WsbGT+xc5teKmi5WYu8dOhJUMUwG\\\\\\\/NQVZ9PzoumOZbsneouA+IkQCh6fGkIxOqVdvKDjrkr30qckbHoLqcQWDf4oEa5z2a02nDENWHmarBHEQyrOHDFGqXQAwh0tdnncsrQdUMXfgadw8F++wefnQsWpsRk610Z7diBNS5KuEDKZoFaOWBLiG5dOhzL8WKkzZPXzOik3jBclcEAP0c\\\\\\\/GssU2in+sFf91d3VikAIDPSP0GO2SdM4QHMF5UCAPxFKny5aAlAh3CWzQvJOvPs0RlMNfTukokzAhPNDCPyplqECQXbY4H1RDgeBgAZe21W3Uis\\\\\\\/W7rLSFeY1PDCilMMPQQF9S9f7uJAdBJZXTZHponifl+h4Q8dCxXnEnbiQUu3ShvmdLQD8siTk3a2xB3YdgzgqoIui3Nln91YsBm65c16+sA+50gYbHadc4MY4yznnnqz\\\\\\\/t5Lll+TTcXpv8Cp+9bdhQKqDFaY9FBQQMRwYsPuKNAv\\\\\\\/JrXupG3sEECJSXsLsfoHM7rDdg9NVH3FmE+2hkf8S3SS7CrIAj04HY0y9kLAIK1FgJqP+DPrJ8jPbPWUMR+sbBINmsixCS5m1L\\\\\\\/tFb1laltplOEzI6czoVbVLTiw7NP8p2PLXQwC0rYqDpXjT\\\\\\\/oINndtbbgyW8r7pgxD3LmGVvMZjN5CfGU8nNEhboNo3rhGKidwMyBWbN0VQLxgIDg3F2EXYr3fHxBfiX6\\\\\\\/8zAy2PNve3wUegPW3XejR5Z0fNTO8Tk\\\\\\\/x2teikAnhmEyzae9o5DGji6OElWcVkXaW2n\\\\\\\/r8jt+sbWYttHAiviH2S3bzHR9sUJjyQmDBwWnfTjriLmXt0LWYIB1Z2Lf2g+yf4J3JHAb+1JHO8Fc0R2Vo0Q2HxvpZjToLep4HuY3Hn13JWzheiv6YMezLBqJ0fZ+NFOAIFZTIt1lyhLvbHT3rue8y1JJ1DvKhyRNzzv1mMTYBADdWChambEc1cIxi3rqLfx+uuzbr61M8kiKYaHgY3hL0ARgojWwX7qbKNCeaUsRJr4o4H3Qe2ReTh0p0TEK0yh3wAQpv7BMk8VI9j2+MBuJhlRdQoBZh02pDW5N9TufrF6RAuRRsdJtZarvuPVDkt8+8Cu7kF2\\\\\\\/2xpbsLy3dlCzl8ewrR8qMz+4UWCZ63Q4RzCo88SKvznPt\\\\\\\/y3k9xOOjLzWw0HydWFN6RleBi69gMwtuRA3iQKFEKadmTutWyp28ggNvpgS2LPgv1jIF+5KhGaKsaHrY0yyiiZdwNgwh++75133vtrGU3uLDQNjgfCTi53JIjAqWPIwRgspNLFm7iIWEQoyYXR0accRf1IPDTxj8++ShQMDIh2UMYwAmJEvQkS9R5T6IxQOxUFqSdKtZq1bWYiCvzj67VLffDigZXsxVx8IHIn9PUXObXxh8XJgvkSzMUQBSWjEwwQgZm4OLQ0GXNB+hwpc4wo+8kO+oX3lS7N6LBBBbsnlyszGGUM03Jgq+8M7FiZyV3dyZ+LKCfdlaItzqW0HiWLbWPI4c\\\\\\\/JvRGoH3GG6CtoiRecq6sj5YVXZOVwujmwScSS8xFjIS6ca2W1SQGfC5IeNkeIB6Di17pi1PwU5Pdspmnqfm0Sg3ysa8Y2V\\\\\\\/3FeGt6\\\\\\\/xr32j4t2UKWSTM7pNeRbMmdM1JIOPesy4Z9wfUvAq8szmZ93YIrZcb\\\\\\\/1g5jsY7o5WucYLIwLb4QeNtIQAJfDFA61HBxWl1St3SnB95JFojaprME7KkRVnyc+Fl+KZwDyiLCOwvhhSO9RPCjdwoN6tVUbRH33GB78Ah8ckwftEYUI2IvhfmUOXmZYHmsEdr1uSNq4BNT\\\\\\\/SP\\\\\\\/9X\\\\\\\/ove\\\\\\\/rdkqhKIz\\\\\\\/f9Sz\\\\\\\/EipyflVrwcvKKUvpCemtvdKmQJq\\\\\\\/yU6y8rvAEhrDD9ze2AKSdxqLnu1wBCGL0whKL+j\\\\\\\/WfKbqN+Qy+miYtML7wyo4dBi60b0g0uIqmVDtibJPmRq4fGJPufuMnfX8\\\\\\\/hTKclGAR3lqT7edX1su2INlR4XoTmvHK+GU4izYsDTMl9cawy\\\\\\\/CNW82dQmtaj1+2XQYW2ctFEulU+Rbp5HAskjIi94L57SNh\\\\\\\/7OJePsEHbz8w+ThayRRsNftVQOxKTFWIhOvrc7wltudOKdLKn0cyFxfNVLVfEPpykTvqh2Y0ifBNXUW0Dw32H0d2xOj8WyKYFt9AvI4KVfCDjfNoG68OtvalFsqyACLGBjDOYBPrreOI2NWqMIA\\\\\\\/8DRQr8AqlnApdZSlVSTq0FrEIfyZtBEo2amMc3IxYdLRy90R4bwqWlqJr61pYkd9XKYgCoH7VcBtODTNChme\\\\\\\/9SSV\\\\\\\/aOy43w70j255aN9\\\\\\\/LxydlPZJ7KthHL234lSFT39xg8qvh\\\\\\\/uaFM7SQ8fG2Y4hRWsyI1iGu+rCnYySyIm+jKkq8zjrfvZRNu\\\\\\\/1P\\\\\\\/VQGGo\\\\\\\/V1\\\\\\\/LptMDBoD9z4+pBHu5IgQqv2U5Y2ZLKLYqWocZLqT6Wn5eca\\\\\\\/E0YxaByN5mYz\\\\\\\/bSfnPVu2soSjfxTVMTtK1G2PlIzjkjgq2jTXQL5hotn7IEZTvrvCBxbrJkorEPqn22dkEvnNU01Tzxg3TcaUnjkPuOsHwedXckmkKx1n+N5rWL+G4IImjikXrSdQnjWjVpZy4mnRMXQN2\\\\\\\/p9Bz8xYViaAnvsFu9onsHONt4SFulie9lmX2ADd\\\\\\\/wF\\\\\\\/AG6whqKeBlUYJTC+H5+Cn6actZOe2\\\\\\\/v+xEiTblVoHRUcPukJp1TiYWHxbut1YLkQn43hcvTMs7jzK\\\\\\\/WBrkCxB0KQwcjiSvUp1\\\\\\\/EHsBp+cjiFI9rpTZbaPJtPMqfuMGdxtrQOUx29vur7LVbHnqUeKHF0iV4ShOGY\\\\\\\/9wWBnGHKIFe02fuBzutkohwbJgRAPLSJQbl6eScdQAibSkq5OoJNRFbaltOUMXkGt9OkmB6gC0cuDVEmvne\\\\\\\/+sRAsiSRKitDz5Ud\\\\\\\/XbxlmZXKOqU4Y37HgxOSCbr+chzKKS1fIIpQT8gbqkDc92fe8IuV1W6VxKprJo7FSCUT9WP8Mw5+qL3cJR\\\\\\\/kZldnmVmZIz83xWcYZGt+\\\\\\\/OvwDO4i1msgqztx+DS93aOl5PK\\\\\\\/dQFy7B8ySpgNnoplPWctZQ6CS9XoK4bvkxe1CnxrPTQzOBFF1r4ULCffRZSTl+A7dnlmzgtBV\\\\\\\/i62yaz6OHnRIpGo7Mtv8bRkofy+mHzta9NuSohww7RjncT\\\\\\\/7SACl0kglya8AchYrx\\\\\\\/\\\\\\\/c3XT9Nf3AoCkbpEkqOGvLmc4R5FNquv8x05l3btsHi5lr4axrwahkknih9itIDtep51EQg2uM+Jx6VeLGsMpSRPCAatJJ9b9\\\\\\\/3PNdwZb2l7ScwNcHl8RbNPaVNt7I55fBlo8ktd72kcXO+y6rzKFL8p0atfukI1tUpKwHFdwYHmH3xqI7ZgNvync76s2COuyhTqwlosaorGbUvhz2Zu1kdSyNVTvRDFq6RFgIfJdiIE5O6lYayD6GO0e4jEi03j717A+wM08GqTd8T2h4G2bZt3l1myfluW\\\\\\\/g4PmSbX+YyQqZTq\\\\\\\/t+TGm\\\\\\\/8Yn8\\\\\\\/8Bxfy9+qZoqKuVHL+rsU48RtxVEOG9yKVAoG2BCugyCSnBVJU0kheEisYV4\\\\\\\/6CcUWT7rSffC0soEmXGfqKKq1vt4heScHTWIDPGO4LvvtPkBRN6hy0KHaWKp7qysGJKkMSDNpl3dqdAHmFinQyQBTI63aT+Bq7T\\\\\\\/923hgtxPUaUPV2HvUFov\\\\\\\/MzCasraDb9q0VzJrtLwSlmF\\\\\\\/ADqYQUOVapmWMzTh7IEtkpEtw5f+5K35T92kQHNPFFxS2ZdlhQ8BzTQKTZRvl\\\\\\\/yWdNEIDBPJ5wffZJq7S0XtC\\\\\\\/UdPyUvLffNTXUXDmdqSEzDkGwWbcPEt6P0qxOmQ4SvY4JGs1NmBEmQ4NGqMU3KDCfzWk+4mmBc2TNdGoXaeiVclCde9LXGnRovilxl\\\\\\\/MQK55sSwzes6V6a3gRPllKLnk20CjCLcE6mEzR8uMEqsyp7qzJ4PU8GMRWs0\\\\\\\/jXBESG1+eGFiAKmRWqIgPzeAMaND0gnO3YffQX3E90xlut2ZMiDg+PZ0viBhOpKnGZrBe2hGSt5RRtB7bSFVfEXMVmjDK4gf+Dk10Mpr8zPckdD+vqIX1ki4jZxSpD5F3RYonRb1dmtlRJBsSYHqKhlcQKDhKjlC4dnj3ycSwro8A7ffq18UdSdk7tU0rtoDT1OmF8VUGnw+zpQ50po\\\\\\\/6US3UUqpj9IW6e2noVN3sWXF2abWLTG8CggIP\\\\\\\/SWwaM6YQsEvO\\\\\\\/oN1goZTr77z0zKwSuW561a2YO9TOKPCUPYpSQUDr10d+wnKRhFdme1tlzvc0eKcPbCbR\\\\\\\/EKTauYAWdKT06w6NRkcsGEyOoA\\\\\\\/eVBYzrSa9dK5qdL9wfnWEMgkr+JrehJFEMj3bL34gBLFoOcP+jryXuqs20arf1p5D3jdvro8NHse7qx+eOcIVExAEvNI7pfMfPmfJpALDfOh5CmKIigkRqwcjjOCoGOBu04SshVGthcretpT6mEE6W6BiuKrQ3X+T5JM3+aYMX59KjGh87K3jbQQCG0FMMxBE7U\\\\\\\/vJJ0RtlX4PLoxuBIyG3Hsfl+OT1I0uZzsyRIVhRdRhyzBxYaXe7I7\\\\\\\/sj3Ng2Siszelwgrj1wh4wmXbUS3yN2SJhFCM9yGf3lLwZ1sufIuzZfG1TPR1W4owZOszbN5bKwL9R3DRbE6brCD3f1OCI\\\\\\\/Yn9beJRuYJzIXjQZrG1uYLTVGwvCZzO0Fu9gYtgkNgDIrpq1M52yF9JDuSGhkkXPkcETyIpmCEsiyDyYcN9GrSxoSscuUQV8WAgvjSuLlEqhYgrDwv0uxABOci6Tco0rZuDLFUFda9MyQgOBUrJ\\\\\\\/a51\\\\\\\/DTzoCez5jBUU8roYTfNbBIS+cqW9FmwPsMkEvAXcoADd7RZ5luraKlxc0rOAjyepn3WATLWWgUk7xs9KuxWAOJkgVP9RO6HmKfF1OfP926fjvJtH3YQzLSo2Bu1xFVzLvqBIyHEgcwt6pNwtZPW73FAup39U5CvogA+LBP3QQsmZhu+wWCI5IRbjTMg9DV5aWKJFGoq1vq2oXRlCIufLw4vvmkd1Cf9bC8hgIDgRbk6V3KLadqxuhYEf7\\\\\\\/IY89pXhq4UZrEdggP+Vbj1rKE0qSvvpFOK8LN9d3RR1UuznPynqfGUKGOdzzy6FlQOWAljsZzsCUGyrYzfquwD9mwHh4XleL75FuaHZEvPcYMSO89scZCVbXAsAahJNKCXKoQmcVDMCiNF9EZITBoe+J3AzyNq3dLYDCrXtpogVPM5e0u0Lm+nNP5saG4EWuun7KrsNw7MbmDZHnOLJPi1AIu+9uH4LHWtW99fBHahmN8\\\\\\\/1A5lMLEJXU2qcTTeni6T+ED0PmsnibPwZPudFDMp2va8EglCLQOu7cnqHgMQSD6X3I3s8+r6a7I7ruF1QbCMLNGwKknigFtzNh\\\\\\\/ztX0Z694Q4LEL8Z3ssotcLFJrhxjGUqXrrxv9G20sRmD5U89NFKgo4tG2JSFOdZaXUJar8m5zoWrFxBZrvFcu5gltsfxhpviI1Oycu5enZJAnRVeb0aF4aEVXMIQDrW6IGjnG7WNZsTinrGOyz03YZD3TOaOtY94ziflbhNFlxTGVnFCXPrncns0zPMJzD6BcxucTGdF86nxcuv6Oo6R1UbhMRDzd7tTWR62863ZQrun+mFtDHHnep4wPHF7dDcLkDEddBeMbBfUl1zJl7MAHFEFzTIne9OXsifZJAv0mPmhgTXPdu7m7hJz\\\\\\\/uIgJULbqKvGW6a01SP2MrzH6hArS4r8kkE4iC6JsGCepUErrin4yZbZYbcSnwDUCJ+gQVA5Y78rqHybG+sVJg8b6fc2Z7yc5C87LYb\\\\\\\/nueCrKmXhOybXERzsE3rWzmjP+I+5fch8RJfdNwl6Sn0FdTv9dpKVgPLvNULNDi1ehnXU9GWSQcggrqvxHbnFiy+OhNTP\\\\\\\/8IWTpAtcKRRHurQgrOSc+ueiXs6CdZFHxEUkbm\\\\\\\/qoBzsEThFb7H0kh4JZ5c4yldGrwl4C9rHE1qwyhS8nkrmkS\\\\\\\/eEnicDS5ws2ArcmLF8GIqcj26BNcCpDSh1qDKNFWwd1IVKgtRNaURBNVNJDKfv8FoXHBBmeDGJSw70WfvNeawzN1eJe2NMpYCqmwyL006QkoGWelfpZ003Zmf7ZL+ZCxO6eZDcYlEMqsDXZ2dfjB4s7vFQyvrrKSK4Te7zj4hjzH2TBba5n\\\\\\\/xMocQSRLy8b2ZfhJ4bB4O0Db+1icETpyi2DqMmClodvzdXjLp3VGGJom4CGlmKSzwYNfGUxfSDw8UwpxdoNME3kCmzKnkwvH+siHWJ3soLpRigL3t8eStH\\\\\\\/3Jj9G5r1WKjFx2eS3yyalkwDm5OtFGkLk\\\\\\\/nf1xhMmq9QZW88Xay7SCrMK1xULinv4HfvWaAjGeMmiDBpNxtn5GowmYojdTo6CP9QERCUM9gJGPfu4rVgEiuRpOTXjoHZDu515Dns\\\\\\\/LSBDY9IFX82RuwcTUgN+INrAU\\\\\\\/xGUONkWE6lV8W0jd3eu+UD8JRkLydCjFNQl+nvtzajsOAyNh34MsfwlIplrnfw0ZxStS+fK9RX0ylHN2qascb2FJD2nu5peGKhVFeQ\\\\\\\/XyPDAwz+Rlwq8RJDXdBZgzIVcz6he2QCA1zkRLkA6JzrUQJVdTprJDIrHTQYiu7EL\\\\\\\/eL\\\\\\\/NYczAV3gQwUXIa4awmvnLjowVQMf5BWoimvH1BKn8eXwpQGrwqp5NkGJldm8EnEkpzbwO9s0MH9O4GmhL7PDO0y79WeJMQN+SlKix6v+LSByQucIFvUykakkUCux21FXf2xGW7XSkzP7f5y7is6qNsx2VHnfJjUpZmsT\\\\\\\/I2Kkh4p\\\\\\\/f8symXaT6ao+JIAckKpOmts+SizpGdqfYLTs7CzIs4nUMd2rxHw71E70NbOQiSmLR3hHiBRfntYWUBjy81Erh+Pf8VLJoEm8Itiv2Lr8MP+6y4J7wZoAgNysTPNoYi7uFmEykbz97VmloGPKm3K9OWQnLEWey6GEwtCaqsScNvVOmDvg8pxF33IdquIKxLygdFygi8U85xMVxiVodW6m8jkl8mV4TlyyA2bIbRkuZoF\\\\\\\/zIaGkjBbs09g+EMZKOrkpwiQXeYF0xBQTl7wm\\\\\\\/5XGB6xzYouQEzh6hCdU4wLpGSwHlIue\\\\\\\/iXmi4ZGhj7er6GAWdPzkyOUVAp0jqOIy8lgyMdnaM77H+7LypENEyzaUSdHs2H\\\\\\\/MK+VTNY9WUz+lUEG6zfOLA+7l3BH5S5XzZn\\\\\\\/XXYFBazdD7lKOXNnEtUKmSKAQ8JcjEBD73Rhz6\\\\\\\/gVE6RW5AMF8dMqLbC\\\\\\\/cHsT0YO19j4ls1ehNt+aZXcfdcnLdtBtHy8JNInEVscpe57TOywP0XwNa22\\\\\\\/gG\\\\\\\/XCDl+7GrkjkBqsRJyy3NgBK92TNAwinclnRfoV962jaSUKdyP0VM43bpig\\\\\\\/pkzBjh29AI6QJ7s7a2+KssraznQvcm9HUS2gTwlUixvA\\\\\\\/QOpkKCvMPhr0nh7mZKb3epmuzVsv6N2GYWJ8ITuKGakctkIInHAYd9dFjammqKqP+VP5mf0I5WrGxR8bjrBktT6omtNekh+VZc42UZNEopYvKVjxPtkouT2Mo2umMahKRDfoSGPfIHikjXszk+sh+KXXcxRPnURbDVGo8FWeP2ohmJFoi1RhvOlvn2RWwSjrdzac8S4DN2Ro3jppcXscJFATvMIjIto7MIp7YRfSzbJDUPbwgnJtZBasz8jUDBWCwhb5aDfcsW5mJtN3OaRUMGeG0eA99Cbup3Rm2XEygmI2GVumVIDJ7KKWcmQSyJD8G4tuHbOj7mooSLFLVfMQMc1aqgoHiBysHpFkgfJOBuxsfmLS7gQhT727WeWMM5hbUbIyzbFn8dQwj+I1JcXt3eYNZq6Rv6Uf7WA3axG2ZvWqCiOZbVD87gLcnQiyVdRYvKoTmmKCGQcMs9nujQMStBm2uhaVxtPte5+o0dP0AN1hCyuYCXJ7L\\\\\\\/uYuHxzLpDUgmi6hD2OZerPuD9kZcp6Zmf3SEzLKz7\\\\\\\/k7dqQsS3zaA3F0+JL2NjjEnHSFeGtgbHuyHbYL0nyCOiHY9Ve8ldtWeuTKERykjqYtxbjASXZRcnejT7M7DdM4PbimeLJGapvaTSAaMRmgyCyRM6LfkonoFQKfwpqiIdPTgVZ2S3r0nLBOjB2FdUN\\\\\\\/Iil9BzJPNgmfiKUyKaPxyRiYkkZnjfVVvfJRlWLoiWUnpZ+BDxQiPkcKnK8YEIXDHmWbmSdGJLGWuSfNHokSQjdnIvzqMySjTGpyUz7Snc7jmvn4sgRgs1IfPuhGpdbwstM8bBT3Ty314+gR\\\\\\\/+E5kmdFKxxaQ5MAScubjsqff3z9oVu7DMuucStb0\\\\\\\/+ucpPc\\\\\\\/D7k088hk5+rNesGpO9eZJUkc3cMSZLyFkFqY+w+Kd7F9KSWVnmscG1aVYO+oxdAwZt5V79h50MkdgDD7vvvchqF+oFnXdoT\\\\\\\/8pRzy8YilMeBLsjGIOR7E4YjdqrMkwVCU+lwXSgGNEofwbufme17kQMqHs7XyDZjzh8fYTX94cN4qAM6keRSy4Y8u74bdtUd8FNj30i+X1JZ3JJe66H6ard16zA3jd0jU+PrTRlpqDkJSo67eIsMIpeCB4bOhif2GeVi6NbRRyLNRuKAfMOG3qhBeTLajIVefiXB1qcE8TLqSwmbaCTuErAKueAJL3XFcyFJgxT5+wDL24RezVE8NmkWNKxXrtJwoxOYymUJBEQAVNPTNjmwxL9YEguPPaiGKhcH6vuQ0jOWV2AegiA0pV32533xG+xX0jARctz8g6ywaew\\\\\\\/O4GIMhn+KuINscKkubKqelhOTGVB5cHYYnQf3pAHxab2ecn9SS4DjW\\\\\\\/8EHjmCFHGTvUfw9TKp+SolOzENedq8k3UJ15tRM+CIAPi+ettqk5s3A9oBLrNsYOxfkRSvVwolpypkZLOiEBkRg5EIXpmaNvcDlvyU4tB3A3BunJWuRCpACmX1SYYm5db3DqfGRU4jMRoFzNU3o+9hdN23k9LkUzWQYO\\\\\\\/MS\\\\\\\/m+s8GDqjofpTjCOTMVaj9GWOrm+ZCqcVRUWc0QF59cBeHA0tdYow8aT6O5MuY6gLb5xAkWw704VREWl18WnjCcx6YZELWj98dZ23M1MU0Zh31HtBZ25wCzAysOU4aL66uGD5IJjrnO4qhsPSbCLmZ3HSSQ3unper5aM8iIAx2\\\\\\\/LZrk\\\\\\\/LbPdfP5xusT5GJxiAz32PMCU\\\\\\\/RxhNrPIu34XMRFchNBNlT+uokRRRgDhlE8S+a9Hd\\\\\\\/8YbTJnPb\\\\\\\/0KjV+3Pf709ecgwKqt\\\\\\\/AXNUbP1jxoYYJFuma5THtOcv+c\\\\\\\/Q1EhckztLLmMGG\\\\\\\/J2n8zhovhqDyzxojWuj8lbLWgWfKxAc0lsRRLVtA9\\\\\\\/rSLaDWE+9xCyIOaTzgGyDE1l0RnthQ1e2Rl6bdcF3i46UjraXm7ezMSVa4Sfpml2E+tz8nBopy0u5fspKc3Tx6ev2ZzT9sKKoI6R9\\\\\\\/5SkqrXeAttPz+xadGQJp6nmo1z+NZJiYP9LQDcVSNOqpBLSXyUhylbTJN3DAjC1h1C6wzGs89lNHQKGPv638mi4jl1YZjQfNcitN41SkQXohRMikGwr+BiI6yE6+oZt4U8VzmjEFjzhOGYhz+eND7wqvmJY+6MVj3ay5kIvQmAG02dJ1ySvgLkJHKDRZJ9sm6eYIJ2L+Z3YlEjXHaW8hcig1U+CrUUXssB\\\\\\\/Ygn2jU9BGIOnOXshUYNQ13muHK3q16r0Z9zaBrvO6JsI+0oPFkmZPMPrg9+ZNLPLi9T+d1dVqBu8hJyBmeZ4bJqWJoNTHXSRdRsvJ1XZpokQr36bUmgUcGY759lTSTfFNUqTo\\\\\\\/Zts3sf1tzYQaXAsu4ZSOUQ1IAt67DlRzyYq6RqYDnXV4sGEmo5fLCuLOvHUKmVjufHTEIC2XqvA3ul40l6Htr43H6mC2fjPHzGUG+ap\\\\\\\/P6aOrFTX2eNnpmA0LC9PdjqRi3b67EGshQgs4QSeeO9sOmyiEs7qO5rDiaKiXk1ZTKkBEdNxPxxnx5Ae\\\\\\\/Dmfw1nbkE3v59qXqxDxPzc1Ui792BriQVVUz8D7Di48SUBwOX7lkyAfum1s9QZ59vVEelsqO09AgLfX8dkx3vqfj1IfDWPZG9jP58JqRRzO75cocPIUBQCKQqt+WIkIvCeqjULC0irEvIrAXUDNEbyIFW6AenEGO\\\\\\\/zpP67m1\\\\\\\/ovmfCGv1LgozLpTfuBeuWIb7+fwRVsuM59m8tAd9BeL1yF7QmYsq\\\\\\\/Chcq3LXn+FNBZWv9VYdT39hECnWHVz5xAVaPIZvBeUGnFFuTJk7TA82\\\\\\\/jPr3X465ZWigig2mpDCLBDRmKkTkKSJdhMqWdjBk+7Wg5\\\\\\\/DfekjeREt4j17uwmxcivWEFWiUcJFzelUf5hzx+hUx7ci0RxhZyh6NQcJq0u885Nu9ff33HKun+0VhZ0tSYMNQP7A43ydbRYwug06SzN7tFxE1T49GqEW0KmWPWzcOGOh44RnbWN52dA3f2iyiLEL1\\\\\\\/DM+1Syb0H6c5b0nWSC6qyTsagfABkNnfGGkqOEKvRrISfvpTIxeI7wYPcP+3WMQEuVeYqcUnaJiTntW6prABZOOjOba8Vw+vJDDcf5D\\\\\\\/bxs093EUCITpgZLBq7VvR8CIxxO\\\\\\\/cynH03ZBPyU\\\\\\\/i7KHBYbP2mHX2uXbu84KXSCMl8DXucuuxE731z6x9kbi02WhsTsFuzLx9gI8K2bqWyvBx1R7aR45KBPelL8Wm7WFQ3LAXJ0e6cM7Ej8WvlgCP\\\\\\\/Iik2FMIy50Y1xTYorf6zAJ2DuKTLf1KJf4iqiGuwJjjmIDAmfdYb9+ncM+A2C3M6JSvhCDPqx8lTzDZUNvmazMFt+ItPIW8oP+aVVmYb\\\\\\\/BzJnhepQBMmWn8Rfe6uN99+t2NZMMbFNbPOXzjPYnkzFHV8HPv24CARj+SACSYZMgf9pFcjruUtzKpU1Blm\\\\\\\/foZQK2FjNFXhmFdjwAklZ0qsNHhUJRqhywl\\\\\\\/MFMkng6vdNC2eOFGipBADkLbXS5cwrjfHP1K1Gwm7+xAnijnrJGuqIPA84JtMU24jLyZrQW1lnSuSk6ePd8ap+gfsOT0VD+p+FlDJNW3lFiFJCzobrZYpS63aiXKb3Dk68EluvZJN4Ow7zjamQXQ6pIh6+x5eZzLGwJ9WhIq9ZOaGaV18+hqiuzMZlLx7jPZrE4p8YI9R6VQmHuasZrg1f\\\\\\\/x8UQEykgEkZl3s87cl8qqWtIBcGJ5RmUX+HBqq8oCTulAHsAxQFpvaOBk6lCNkwFtrsxspLP0Jf0p8cvoJugpFdIJdUTIWNrcgF2Z2YfIKg\\\\\\\/rgSWknMWZfY4zzLg+7QWFTcHbEML1Zok4vV7CT3MYPWbQXUO7kPcGhoT84FSpjMWxcEPExaGgUtIrvIaGEf215vzEteSW\\\\\\\/oCbFXJUoQetJ5QPSBMIVIq1c5LSuJBALWR+F85NnxSs\\\\\\\/tIArFQKaVw1S2ahvRS\\\\\\\/z6XfzPMesYafN5kZcc1mA+lhPhiFI7t\\\\\\\/4cbvgKBl2thLS\\\\\\\/C6SqBmloPN1eSWQ29FxnaDnevlg99Jv7wx+eijnP\\\\\\\/3s50QC4Yq8oGHzHS0VVHZPcLXRABsY6IKKqtWx9theLDugMH5HCRFyQpC6xY5e61htsUhLaXeQcn7aKN4BDkt2btWaOhvvgkcWLxd1x1vSaRQM6y6IqmUNYa70\\\\\\\/E98c1qCUbpu0xA\\\\\\\/lRix+SiEbFkTNIRkPaXcvf2Q1aCD5uCK5zhgSMY\\\\\\\/J0mSG09Nj6BciTpaWO\\\\\\\/WTWuFeJciLaj1IaKF7oBiAtZEZah\\\\\\\/nirYUI6BTxwVU5j+pcuwz2SGb3y+nKfXKqGxCWB27RizfKMrpLEsrEzGnJQGmzp2obNcivbWHWant3VZYEmRT3cYya2d1Ge38XHe9tEHuQ9ppQbvbYRunYRwcHqZIg+NKx0jFU2oLHWxD8GIYuIO2F8IZK2g8kOSD3iWtM9jCepZ6k1UYsY+Q5dtvP7Gnb3zvfY0TieSFOC6xmdQt3IivMc\\\\\\\/MU3ls\\\\\\\/UGUPSvDQa+i7oztU1H9J8Ez9\\\\\\\/ZDS71RBWNv\\\\\\\/fWlaTs1Urn5EYTS+5q2k3+13bex3auhULNvJJQZ94PaLKlmVD3J+fUWAlDIeHydcQGPge5ho8rkVpP2hIPDag3hGQNpYi8VH62QgxMWSEIr5ra4BS00MFl0IfW10lCtK1lxjU4Sn7xOZAx1qYmLcsz5eg7VhhdImR3PMVuPhxixGxCbcB4h6CnFjif\\\\\\\/cxZbwcvOkAyCSkBJavnw4\\\\\\\/pl97rITF\\\\\\\/cdKm3iN63kazTopGIqALQHIZGu4qiI3bLgZKiARN+JfyoUaTkk+yq4v8Sn6plsxTN5E2tHsc9wC3StX94M5EhuZks6p68mH8aQilDX0AuG9g\\\\\\\/8m4obgn74bLAMWeWxcQOEF5pjDtLJRaaIG26pzsEuryOx3AgCp0mh27htUzQuC7uHSb+76or6rLqgY\\\\\\\/mOZFPC9Bo3ewGIqmaWcCm\\\\\\\/CRWdc6i6o5NrfROYuG75JYiNp7OuvGvJo1JfOlr27BEW\\\\\\\/NT8DQc4PNyi\\\\\\\/hrCR6Z6jyQUK1+Je9oSHdiI8uPTsaM3rayEcicdt\\\\\\\/7ISMIY93cgXGDQMztHdb6XdDZ1gXGP95bew9kchHduikuM8D60zpvcM\\\\\\\/bBRTZ7EmsAbgzSuHC1jEb+4gfjLw3Uev44KqG0OvXrgwBLQnnHJtMhUKRGq0sq\\\\\\\/Ovl8nlvEmZ3a8aVoViaccnpcMCILBreksHlYNsyQAoNtEN8AFlNZBLo7adJkNZ7EMT255y4PrVYwzu9lIiHbqc2UggAQ7+TXPHZ79exJ3DjpDweJdIP\\\\\\\/RmeFuewDkoXS1ajvBQLULPkJLNEF\\\\\\\/ZJLXs6URjpK0eoUOFQOZemudSaVzJ7jrp55eiMi46OyruSsbvgJCq7wvuq+tlQ9rwfyjEJfB4vUc\\\\\\\/TRVK9wwxGmNqJbsPzfFCKc4kh0D3aWB4SfAZBAFsTXxl7Ev0wYzGcq3WOij1hFncnnszd3J46sT75ogeDFhhQhMtmkHSbIcCqN11WJ3F2AJi02VmBr\\\\\\\/tCH2Vj5W+VZ\\\\\\\/JL7VV3xCwfZRXgAQkNAnxDsAdA9hWNFHl+XDRnDIteZFTuyLOYk\\\\\\\/b+T38N1UvYSIRItLQkKNDSppcAQQYWUsRafRgX5RX3X1dTZGYtkxFMZVdVNCtGyiFzd5bXY0SwpmSm0KQc9avUNxkqgf5XgzjrHJ1JaQ3l3Il0pyyvUM8FKwJwAmMnxPxPJXb6Y4f\\\\\\\/j5DQQdbVkwbNdqyq+GwVN+JDw9DLIEBnT5OSJw428wm\\\\\\\/pkcLtWgHTyQ8PXIyYK7AGjPj23Pk\\\\\\\/qZ6kwdbwJJg\\\\\\\/\\\\\\\/ds9CAq79v53ZqGi8iU3Dn2kf2YNqQMeLCTp8K+bU+jGzqV70Z6jEzjla9bp1FXGQk+IsYad8THCVGdbIRU0IpplXx1tjz7zYMb+Ha6+3hyp6IVCEVaXfu2obsTIk39GIw0OgAn5P0WfFdOdmyVEYeJKio49H6VpBi96kNtqOzQg1LgVjKixkWJ5+FkIHFYAApQZUUoxqySYeVtQYgcQK4exvXtMdHsf34ES4zgGj3Kr5O0sf9w71QYzTQsLcEPyKs6FiH5il0PA\\\\\\\/bb2ggCXU5y6r77wOZaqOS7HlH6mXhKaKvAEyRFofA8vwqYgG3zc2aCbRUg8G6zfrjGUT7CDOX6tYBRkhhZbZoEP6PtaHjelUR3XGgxdJuosJUEUhFQM3iFRr81YlFdFdoDS\\\\\\\/mNRAtkGPb915ffwuauftCf+bXZpwy3CumSi7ard5yoP72EJkx6v0pvX9KLErtwHgk\\\\\\\/XGA9mObc1i2B1ZxkYIZO0hTB8ZpX6DKEn2YIBZ4lg8qmzj+8fEa53XdymWM3kdPpKBodygH2kZ73nrv1RdjdSOteNMniX2\\\\\\\/f\\\\\\\/byupbtZ7rRJ1F9WCP6ZjeY8mt\\\\\\\/LnR7e5n913Rclh6cL4xUSAKTWfqYuPC3ClWbp8noGZl\\\\\\\/+OzK9Wrq+1+lsE3l3Rql7RYkbiQr\\\\\\\/YzKauGBPVVoXrDnFeqU6o6\\\\\\\/XzLDx0BveeitC0hoh5gNunm4ulWiPJkWUfZPlqmAlcivjF\\\\\\\/20oe0ZzwEXyU2LgrxmAllAp6D5t83TA19GjgteLzIRXiyw2T20BWRj3e5eTW72EIWd26EiJrFIU+JZC7csMbE3lBUgLYCmFLaU4sv0nG8UD4jrR3Set9oEZ2SJil\\\\\\\/eUu7StZNPo7\\\\\\\/er0NG0yNqw2E5m7IY1JyQ84CaXgojCYyGsMUCvsIAzeAJhmEHTbNGgHdRcEte4LtZUFKEQ\\\\\\\/qE7CqR46mxClK8fL0t0l5BqvV40ldcK0TMJ7Sad7nejXf+ENTTmfIqi\\\\\\\/gQX\\\\\\\/Dnw6GfLUHwiNJXQa\\\\\\\/NV6bWkbTyI0\\\\\\\/cxOmvgc+dz4fKetBzJt5fRb8im03mkJ7BvodHQLWDDO5O88AcHEr2LiERSKTlcuEp+SJiy3gQHo9\\\\\\\/1qnXTBHRiDKaoZrBbPlLnEKJQ4gW5Rf90\\\\\\\/FiAk3sV4V\\\\\\\/qJGKbBojdbzWEW2nllLXiaiZJxgGaQjBkznZRjxiYgozBSA+wuY8ft1zpxzRRJBtJFTbzOTwdTvQtaV256RKxeKbyzzX1f1QeOsIgVqBbATIf0jEX03BevyS2kDKh2nMnUOPqKNglAvonYQ2zk\\\\\\\/88arojCM8rSDcYN227Q2Kpl22eO4z5V+XyjhFdgB8RDjUVJYFitvnDvcfcESpGLk6IF5E8U3EjPi6KFDceJ\\\\\\\/5LTdfV\\\\\\\/bN\\\\\\\/c64KPV3OyyqiLIpXBmZAJPOt9gq6cBYijleLrOtZyDCZLn8ukS\\\\\\\/5v1OBH8WdfJQLRQV1NMwzMzr0OxPZ9ToZwieTjcM8pAqkERElw0HTDNmESjbKGQPU8MBTHjTOfdCl\\\\\\\/JrsT4SOZROU3LDhu1vmd\\\\\\\/x8t4y7wzcGm20Kp1MqPUXHQ2AjRSulys0ysLp4i1dMNnP2bEjoZVDlS96eftsMxChp1PGmyP4Nb6g4juoc5dQtCgBbWFKokOHXlZgPtLLNATGvwFtq3SHYfmYoxdoJRJiOFlvSjgho3PfG8p+n2pH6uoc+cXzYkMjwOnTKr4C+NBJiSSW685on95ZFbwQXiT0ver8g5na44PoDMz3b8huO2gUXyp4Ya8FPt51sojF1xTxajzS2ankqI7UcYca0Nje575itmS7o1wQPtETd1fjyW9pztjRzgb4QN3iQh\\\\\\\/IS3swdcrYELFUjRs4b30faBheQ55DWoTQnkrFxguiO2RYvkhRHolSZzdTNB8VwIQb78UF03Lsq05VCRTFocS4ZuugArZLMt+wU4hsCExPS11o+caSjiyId8I\\\\\\\/k61ezo\\\\\\\/dYyTdIG911Vz0S5DCw6j2N0YuivRYCQtSEmui6V6NXNjkjb6oy6juW1tahq7Qw9aZF2Z3jzAsA+B1X\\\\\\\/vwZFdV1HVdWFRgSJUOb9NRf7A8GPvrECTpVHCqpdqM9+5ztBl\\\\\\\/apYaFC93nmSmgfj8Kxgq4L6oVmp1amr+8Uwgu5z8qwBuK\\\\\\\/bJYsUpktiVqvXV7ui+u176ooXX7JDMmYgQByh92yF1zfZNXtmQtV89DxZLByRDOiwIw01ZZipWrMw+vae66weXfSDQIBPEUHGLC26C444BJ5IlDAqkfCrXVNKp15kkUgjt1MfYrXOUOPPsIzaswXVL7PgIu5hBeckqW4Kn3R4IqAD3kSdYzmosnUkSgJua0jXV+4e10muw8lFa159sdyWZVxVVhoN+pTxKHpCWjevEQDYDm+xP86ziFwlvTAbF5wwn9pRrrertOOSxs4O8AropdBlhfo4+Pjkdj15F7T97X0GKMieEPaHXBXATDBajilikuPpOx2WPxDROwHisoodAqCpPgl5Fs7oEePSAx6uF+rHoLes5wRcrqcYqZ1QqQ6JocbPk9O71Iog18QxnU\\\\\\\/hwdrQOSgEui7QOrDz5YKrLueGhFmoJFYv52lyQlLdPjrIaicCjGG1DRzXMEQyWVyBpXyITckciAFe1jwS5aog0PKwAJ9W3txjvgDpddJFNohT+FuZ0UoZ0kbfFdr2GwhmOxcIxgWFfZm5AkyaxEZCxBklyM+GPqaxb86VdPTziSCpTLjp7OOfdR2pg\\\\\\\/amsdZNGtgDa55edfb\\\\\\\/ASLsQVoFCFTyhbdlxJxWiKzxEeIF9EBSKuZRXr7Z6Ka94Zk35DTb9oAX47FMCpgSx5aubu66I3OUihbQzkdnl+IxdwuufbsyhwEavKMlpH61LLPjpA8hW\\\\\\\/BLFtYhnxZqk4H39fsda+Gk7zD9LUzQI0bTIS0OBDfGDXzshLdSLLZ8qbmaqfGBXwccL8kHPko\\\\\\\/KSS2UcJX0NjvVuTRytjoLQHc7GOw+3TeZhXD6A4Uia6GR3Wv4jy4oouU3MG\\\\\\\/9vcK8AyhNuh3GdYP4EHDB3YxPkiKT4QDeixqWjFnkB8KL2nO51biFtZHZzd8HRL9kLKlqvJL5Vb8WCvWFcx\\\\\\\/afnKd28UbdyFIfrsQTUrMmr\\\\\\\/gndcN16hkxp+XPt2ftC7mClT8kyGWifzYeX34Svspeoopcs6n8XLAJgcOcHeuQxNzKVb\\\\\\\/9oEBmpw8u4lQgRciRMYfaUPSFH\\\\\\\/3gSItZoDg9J\\\\\\\/NIBclS7ekH50CGqIj0o5trg9nzeF\\\\\\\/fH3TUkviEuKZxBXD\\\\\\\/Q2ztASK3ypyWN8ysqLP3bMYrjOMLeUGN7qxLIZ9ZoqYVN\\\\\\\/PECjClOVWraPok108X8AxqwrI0NyOChTx3gi5gvdR6Rs6F2i2nqjuJIP24cOah\\\\\\\/XdyPHvDEefFIl2\\\\\\\/OAhwRXNc6zOZiNMcXzoGa6w1JfCFyKY0FMnWTDSm5PR10KN7qKQSrHQcRQz+fX0fa5I5ikpvxjDjIdPO+hXneBbey\\\\\\\/gCcvyqikgpmJ8BLMmjjxMWDyjoDu0Q8+yqMwzeANzh6SDxiurzssI+NFd4PXS66xDfxbbA1ZPDCpvAqxtMAm+4GIoDXN6YdFAaa9hb6bi3onh6wRd0PbtSFPdtCIVFnQCB+eahcwMqYSraWTqtxB++66m5mj9\\\\\\\/uSf9Iukz9PwuL8xoPzY7L57i3vtp+\\\\\\\/9bFMdfMuqB2lVYZh+RFyaKBS9a+nzIw5CWMleqBQyFDhei5ZNUNCs+Vez1TG7pGJJ3Zuw3wcXUtY+hBy1YVQfuJ9LaIuh74TK4GasvNQmyEGtjaz7fiZx5KGFXeGsE0FiSJQ470f7JBiV1neOGchHpX0TLOq\\\\\\\/RE6oVh3QrLI8eO3IB0b9OX5yOjE\\\\\\\/+nWlE56alxmb1mB3PjN8nyYY3GL8jlGqZiTPiNnVHs9N+liXfVgQ2ntcRp\\\\\\\/Nsf+daiGj5FLAfAZZ\\\\\\\/EodoXr9KBIj5bmIGrWCZo23Urm3SEvMJpnOczo0nol5UyQcLS5KSaaY5AnbhNfNVvm0Sp1xg1SFDFo5QoQL5SL7jbmmDsFCG7RulmFgdw793GgPuvVJEhXjX\\\\\\\/Rhnq\\\\\\\/3+jJM\\\\\\\/p6if2FZeyKiRxEL\\\\\\\/oByu\\\\\\\/ufr0mkHLDSRamXKDhK792ddLnbJW9ugAfjNrIwPACQ1EHbTL8ctBcIE2JQkzqegbUFTaouxJ+otBDH5YegiHFa2NkDOizjbgHUFP+C5ecaphGPY69vGB9kv7oWaamKCvc4RziOUOHeUfiH3bbLC1t6tfa1U8ViTnmgAx9XlrkGhG0UBOD0Yjsr0no8LXbRZFJl9QSNeP6YNUY24vRUwAKXUw4ndqmCSqVq2IABnQGF0mpmOyuNTT2uEXZkcnuA2XnniGVTJFcDS9gso7nrpe2nnlmXxt++gHUtNQKLe6QpH\\\\\\\/k71xXQS+lwTxd6u4kE0QPRk\\\\\\\/4ySooGob90h0++f4l8csjRWpdWbucmOGBDnKAlOqf75BdXskzO+ayBGnrJRjNrBCuOg+1l02VQhjnWy5iPHgKdFbbch4PrS\\\\\\\/UQn8ZCbyc7oTqPEVmUxmwH0+Q8K42r9s2sQ+43yED5\\\\\\\/Lz9IEqiPKvOt29\\\\\\\/eGMxfa0O7UHqORDWP4cG5GVkdhow\\\\\\\/O356Fd+c8s7lg+v29o\\\\\\\/\\\\\\\/mhBmzZ9raeWwkruXvBRcoDOzhW79sAfEKKaeqRvl1x3WvsVIEpa+CWUe0HivPSSPlglBc9isE4Mt8mRlVziG1h0I1TkkOAGUYUElkYdbF6DpAG80xmZ\\\\\\\/HZ1EmTsb6gcRQfBD3\\\\\\\/FgronKzd3PgISfzv5jzuZBjCQLEd22aBLRfjzeaaB7sY6Am9BmGQBeHgoKomVPD0pQwUThqU16SDt3nc8xU0tHG7FnSZZqfrI1sxCVpGNmUbo939mwI+vizF9kxPUqJxn6zzefciDbh7lj6cHnBMFg2zMMh\\\\\\\/F2mf8FJhUP2aB7GDym+ca1Lzf2g1weHkyrVTtuxH3X93LFmPX6M3w5oZwuiQb4JFaycqT5sDLzc6wKphsvAne5LjlqCVYv7zhsf80tmyfo\\\\\\\/UFAl1rHW6BHNAX\\\\\\\/KyGt2GG8w8Kr\\\\\\\/v1s556ZWskV+0DH04UUg+w7joJ7JbY7cFlQ8+agQO\\\\\\\/3\\\\\\\/z0tQDlMhv6gjwRxC3Ae5a+wHLvUA92izx7hhGwBe1ZbI2Qumm5nNeOokHkxdqzJM5gQY+gMRD\\\\\\\/82RIR1OVVQUBiWR5qst40LtHF3IqEwDmOP5VFYb57WPb1MTLpHIt1iQoJf47DwGHJjaopaCk6NkxD2dEjzW+j6GSl\\\\\\\/2Wcidz8cUTv9Zr4Lcg+6dUCAsCKH3SRoBtVkYlq4CRFMcIh2OtdjjJszPUsYSsNBTZ7x\\\\\\\/O\\\\\\\/yu39ApdpDy8vS3LoaaNt3pkSPO\\\\\\\/IfCt4\\\\\\\/\\\\\\\/LQmifabZuw47SZzp6GHUfZD6jBvTByBsYQ52m8znl4a0LJgW9HZzTo3SB0bmfZ5477eS1iVEReHw\\\\\\\/z2rASDRcsUY4VA1HIuZkv48flMU6qgVzIYTAHIrP6AVlYKcea1V9KCrzZi0s8A3Qvp3Q8H4LIqmmaCbHyqB0MOaeckOkFhI+f4ncZqM0GXhK8u\\\\\\\/TdvHsynU2GpdLjVB5URkNOVwgg2qfGbjdvm5YAZe9gk9r+2WzELyyuJtexp6DJcSkGqAhC3YsQ0nwRJrYhH2pRd7caR0Rf0SoS4hmDM2B9irCbgdDdkQcsslnFSfG7eekKbJLskf8OWDCeU0LoavdTSNirrAhonXVUgz3ESvVdwlew4EYrDoaGwRWoxsdXmfeVOO7yt3PCbW6WgEgWy1rRppEtWB4DzgfSDTulC7rdVIHtJrw7XydNZYGyfCgJmCd9DcHlmhfAMgkU1zIQciSfq6akNUqSfgL8Y4Cd96R5AI822njyekuSjL5fyRqj\\\\\\\/SBImPtWadSO\\\\\\\/jtbnBCOUjmgIHfNTEHfc\\\\\\\/e\\\\\\\/MihTw0e4zu6OgCsMgV7ee4Thz9+c9PlZAnlhwrF7YbDpy0uig+NJIt\\\\\\\/lp5u5cy\\\\\\\/eR+WKqFjqMjaWQTdGq+GOQjwJ2uTK0A9vjp1\\\\\\\/Pr7ql5Sp8Jc8K\\\\\\\/Er1WOBi+rPJU1Ra8PIm+X6b\\\\\\\/pxD3Z3JGov1ptZe04QrCL1Fv+LoX6BWnF3ZlRmq76Rp4s6mO3Gn3Rcpw6M6sdLsYY41U6AD0CBLKmNrmTsUt9YUd11cR3a06DLTBKCan5YkOcTyxXHlJAGsYkqdI01SmoX4ykS4KKNKgQXnusjHL3U6AbUUCOWXT0CIfMKpepDOkQN9Oi7r\\\\\\\/\\\\\\\/\\\\\\\/2CD6yk+Vq9EmUY96d+\\\\\\\/lHdmYF63HA5AdNnc2ZhlfBBM7p7Ma\\\\\\\/NCiw\\\\\\\/u2Z9XFQ2T\\\\\\\/2zSfJ1+PAn2WtuOlmR9xnPIk9dhjY3j7dmfSaLquVLgJ2rZkKJYNdCf242sueP\\\\\\\/QrbIAqKCQWxDOS4AVr+lC1IsPoeKcKiHdlXRBaIvV7Skk03rh2uOwyZpasGtVzNwEF06+KsjSfTV7eNz\\\\\\\/VTB\\\\\\\/WwbYXWjNohp50g6HZ3BobP2PCKFXoeOV9A+fy+8DRQ53+gaYcc3y3fXtsdtvrYv4veoEHMi4HrvvMnBkpxuDYTlbr+VtHO5gpcEgH0otSBCVVG31kN2PzyYXIhgq3\\\\\\\/ekXeu3pLn03+K+e8eZseDac5uqgQmSe146NEj+mNcq0pF\\\\\\\/7f2ursrRJ0iS8yDXvJSB\\\\\\\/aK7lw8eGgvV8TX+mnyCB1XYYFE1s3m4bQV0sMhKOYLAa+aI3bLjzrKPdeQdClBtjbjb4fkjeTh3vBTsAFtG4mcWqJRwdwEUgC+zFw4QImXeWE17Q7GB5UjmiqAmVEfw2+YfGJrN2o2Vk\\\\\\\/FJs9QQDtVhgygjpnd2K8tDfNHFXBF3ivO7e\\\\\\\/76KSdgfAT6qgAz+4UlUYSXZ4tDQckUWco+q5CaO\\\\\\\/kdF1pDVsr9DPkzXJ\\\\\\\/mPF5Lf7PghjyN6WL24PmZduMowXsJ6rb0DIUr4qNITJPkFZECSAm\\\\\\\/pcGuJiPUTsZdU4Lj7tGQXWYt4upPimoFviD8ZafnV1a+GsWC1DsZMTkgyuK3F1kRYfNSxkq5kxbKLYJ1iZcj6+kkONTk1gGaH5fI7Q5vhEvUKs18cToi94jSnUE6BMSLxmn6pb2G0Y\\\\\\\/xqKqYhCj5nwUMj3F1z1oYMr3ToLp737pV5lr0G7Tu5NhDIpI5VeOlfly+34E\\\\\\\/0Jg2ADK2aM7w8d9Id7TNKwvuTu7nrE1ImZs9eDsjUNdnmjLV0fMV0Djjtg+bgWv\\\\\\\/gUx4WIUthwDElHlSSf2kFISnxvAVfVlq2ESK7NEpqRMqWufhaUvvDzXk8ao10RMzMWZDhUbBYsuGio4LuxYqbjBaZJNrkMUNezymKmvXUf0rRWcgfpCg8NmJdQJJwh+LR\\\\\\\/sDHUSsGpZMzlCBt8L4BFNlF0nqTF1emXXIZkcolkU8KVAf1M7zdyzzVLK1eIY+cPXDwpEadbJmaTeGwUvvbEYhYd7XboWyMOSvi0QCKgZSp0UmFeM8MtYO\\\\\\\/yJTrES9qC2sS\\\\\\\/saE3pK0QuHg0NJOSjvRH4rUlNiWkWfCqohZK7AFSI7Zy1\\\\\\\/eXk2\\\\\\\/\\\\\\\/2jjs5x9pNyFIPiewMXBELYBd7+dhX\\\\\\\/2j4V3rDqMRlHFEnxty6VwuGMi6kbnbzJPzXwSLFPkb0c4xic7mKgjSEwcvF6\\\\\\\/b+UbjLkg13NcFmADICWASnzPoUfF+NgMCBJNw8bOWlFKASDgLnVPQUeqjI1WoI8lLRffTRAM05NnXl\\\\\\\/xRYKSz9sNv8xlM9fT314rqLbdEAIZ4oHAh1N5Z5vKiLW8gxVUjHzNu0arvaa42yzxlahBtMJUYXv+03sREROovAK+P6zeu8oj770fXGGKzAM2NK4wH0iO1QttGyB5t8NEbnN6hkzkYUOMSI\\\\\\\/D\\\\\\\/9jH\\\\\\\/I9megEgvMxFD0HIiXI55SN1a3RAP+oMaFnz9RcQ8pIqkCxyV+6rtt+uuFtQegNN8F8Kd+SgAhxOpOcdNBZ+ZGUSTIKANWK7IMS+TbPFSb7KYI5NayOoolJPEFLAc2jZTZqZjq2KjEqQRO42kS3x\\\\\\\/kEA6YOw0ZNBLgRAz9\\\\\\\/XDD64NbLkxWSOpPyiCAWU97jCpunntQvE5idRBPLiWQ5OYPxiGRyq2EwHzv7KZLrkdDP1T8oSD9ID5KMoPLzYbWQO6zparcOjJdnM6cX+HOgTFzPMoZUoRZA098UH4bpLH4MkjWKX48EXoziXWhGjo0E8gMUuXq8Cnzi\\\\\\\/8oLcRXH6SQrYCvkZpD+qXsQfwAOd7drq+NoTWsr0+eA+Vaf7snEnjyV4xLfsCXlbiV7KujjNPb\\\\\\\/b86+2an0OEP968\\\\\\\/0nllaTYGStgY6rAPl3v1T8BL+F+M\\\\\\\/BqYm88XByy0EBePGs6W0ug+8BsDubnUJxcGP5XxvEDqB4Vv0Qobo2+gei2\\\\\\\/3wmU9n685wQc7eo\\\\\\\/IfPLt7qxnveXRp4O3Bp\\\\\\\/7MDkWSph9ZS28DVWRhUcsL\\\\\\\/XV8dNAf9AymGMtO\\\\\\\/vK19m3vaCu053xEtxYxPhqa5FWFCY4us6jP4sOHLZnQycra61AQyFE2DZQ6gvlkIOm1dgU7c33KPzwbE1pHgD+gCoMtb2dgUHjnA+N67xHbLBMcuTwyhFrg\\\\\\\/zfhghBe378izgVcT1W6sQ4m+dDNhpLm1Fu16ak3fI91qPXZnx2R06lrlfKbFj5nQb\\\\\\\/RiWHXnRP9FivBayRjCu8Fj3R4xMyYlnY+0orXTkYBL9E5aK\\\\\\\/Y+SxmjCok2aksdippJSDQGVBOig9OgwNKS6VkI\\\\\\\/+Vf8cjgOc03kzq5VQT2W4GYrkQevH3yCXMOd+152ClkWUbCfxLrng2j+vJwkIokaZsRboTzyHreB5pvqNAUcRiAyNsT0Lcyb6PYo59Ik7ZiZB\\\\\\\/tf8L2jcwVajMZAc6ACqOqBV52jdSuNvqHZIVCGe2DxUxhn1a6pCy38nJyj0MJvKhNdM0Kw6XdSCBkRGnAdmjpRUyQS7YZxO8YM6979H\\\\\\\/zuWvmRgn2N\\\\\\\/c8iSATjhjNytV9pu7m0BpG4dm+RELc70MdyM5x08sZlvMkDiHAD7qylDYoEYnoqE7WfmkiJzidX93oSoI6fz+xcCI3ytVN2Amc4lczYSXS4hc+Ix2vLW5fvNcEMrzcVG20qxwHFz0AfygDaJ5Jj4ickdcRoL18fjW0V688s4yVMYyaVmSxopTQfoB9WqhzadGCvbMnr6UhTEE7bBUDeyU4FxSE8Pqh+8tRNL7arLEvoa+s2wx6BGcO4zq3XOJ\\\\\\\/npT8NIPxoHOJU+irYjRu4Pcq5pdl1bcCX1yt6KViabR4zHsyVAHmIeaspfXgiV3IwcgFNEq5KHUQhfZFrXkFnw4rx3KQnmsIQtfR3BtTeaFUfLenCf1C1+M2xAbMEO9fRxnAHKTqOwzavCdTDg5sDHB\\\\\\\/MEgkuy\\\\\\\/\\\\\\\/ptIyX8gMsjqXUvFpakuhNZD+E4rIwasE4dUrX8s+gBsnK3NaG92SS7OidaS+730QudihmjU26ve9ISoDIU2LBK8xMMR7q0ertk6DzXB6XQb5PyCvY1RG8psqkIiZpKPwANR+tr8XncPVJZAeo1F7FRdET7gle2wbf2iDqXYH7LwThyPa95ltkklj4nvUBFkvUNKzsVfqgVy78YAXXMTJcljkYk5EXFQXO1\\\\\\\/4jrbJERfw43QzMmjJyKG+8aRD\\\\\\\/eh8VrjlnlBGoQJYThs8KH5njH8BlLOEojiVHicJ2\\\\\\\/vGCoOM+QLk6weOoA\\\\\\\/7saXILTXOyo+\\\\\\\/Gr7uGpZJt6KKDeq\\\\\\\/tCsPnRZL2CuD3IVnCqDS3DfRziC\\\\\\\/FgX1PWhpv+8QC0s97C1akZ2DKauTH4EHjLimuhyFeg+gO0SES1XHCTiBzCVr1u7GVajXyc6J+uydACYJ5N84\\\\\\\/w0yDBTKKS7p2B7bsgF+iJyJBQPi7L5jwWbXD0h1PMbeNAA8Jn87dt9Wumf\\\\\\\/tSjsgyX+nyQODRa\\\\\\\/3x3JgeP\\\\\\\/FzSQ7D\\\\\\\/hhfapTHeut3Xzrra5oLR5gVMVQQIz6sjdF1OZ24w4LfALIvHLqD4V778xauAEtc2YNF73g1Q1AaH2CmSbsYRDB7m+qnGnb4EDxcLRfa33ilPw6ug7naygpZGl0ov2MqRFBjt0sRA5yC3kSohpZPjZTmCENQ4mkLW8KKpvrtFYKJVQT2ferFCFglqj0fQAt4G7gMkgkV3N5uiJGrodDMhkLvSGXu9z\\\\\\\/gvQLtxQqUokeh7eCTQ9s7zX7HUy4SP9wOrPccP4xcxWjZ9SYg7fR5xb6TErufOfqp1ZtRkfZlhq8Y68SKKMXZMWTFh8vuB485gncowocTJBaXz1q0AcmELqoFbASgsh+uQfdZmqbF6oD5+tLaLZsnaDz\\\\\\\/MWfdVM\\\\\\\/qg3bgoXWb6Pl0QZ4hMl2vbgydtU5Li8R2TAGEIloGKRVvTlr3PJ5GpSE4oNVyBcSP+HyfrzO0\\\\\\\/9KIV17\\\\\\\/Q6leJETfQmSL\\\\\\\/+aubbs7kneev705TCsuiWO1km1M208jTe4BYCujT58ML8GnYG28IIC072pHMZHzAM1lBpMqk+0j2Z5iuFeudtVNEsEKU4YOdCiZR8tgKTS4SYaZECPIvLRLJJYIWm4hjrjywRIBu0o0aTZFBdBOFxC+VT0laNNnI7zeNfxdaLtIcPxjp+37jbd1M5i5RcTXqZUx3ryyBC550hNfNxSFpakjpA4A2bdCMLHGqxgYJnqN1lPINiUDbNcv90\\\\\\\/ojT6qyJgMkadrzLpSoQR0w3hR+xo8MOryrxKaezWFtEsqB+7kEWKw6VBQ3XbfZAy0If9P0+AssqwVLwBL5v1OvLZzKitwtcqS7fuBIgCAFxdNqF9p+IuRzc+Mq0q+Dc0TEPAQoTVI1gc\\\\\\\/42Disznqcvj70ISmW8od+6exgQLiqUYxzTA+jOcltEcOvfZOzA41lByvLLaZbmWowSIUtFBmvajtgD0hKZCb3IpIHxFm9QWRefljGWmkwshORdYyzir89xEVNB44j\\\\\\\/Ubvty\\\\\\\/GntbWIJmmXIL6Fjd6JexdEJu0E8n6yPdtm6LwQ3Wkq5dfFT\\\\\\\/eG1QPv25MCZBsaFt65ZDeIKanKzC9B0YHn9pRNpK4o08JU\\\\\\\/I7+AijAZnNdsIttM8yfaXPZWQ4wVPtiQImooyt2L2M+SLkPVpA3jbvxQs4lvTFTNvopOdINZML\\\\\\\/brA+ypNl2bJmV9jlD5q5drLyklxLiJmhGzecx5fGEBbqkx9jPjAWhX9e1OksPQV2CzmFNSjBqOl8q0VMtf2U7c8ByVeis1UvuFpxkZh7hwqJc+U5r2fjzR5S26uoz+g7ncJTL\\\\\\\/CQGOGD6slJTcglDZyia9gBOdeEmQVOgoc7M3z9oexkv0joWkkUTCG1wgixJ+McwCuqceCE+sK\\\\\\\/afzbDUxYise4kTLYm3gPnUBhFU0oXcZY1Z9yMg6h8f93\\\\\\\/9HZgTA3CUSHQaNZLfuBX3Ms+xvYabLVkOFO2WtZNAeRonHtzwE54BdkWzulk7TScb8FQtGF5weZ4h2TEyW+4uMX8KSurutVvla8uEhDSejpiwoQ+N7uypDPo5ViyJgHMnjffKpynhNA\\\\\\\/b1WFVg08ZBSCReVmAmrdDfLsyAw0djw096mUsHT8v3QmjMkbQapzzbB9sW6JM6B29L0F8CXy5BT7\\\\\\\/49y1eYKT3Axh0EjzMCnOeh0GrqUI+HAVFahQaJBzykUSEzGcCBKHXPXNDPuOUsfkg0XYLgA6XcYOSs7Jdk9dqD+bTjBMUbvxmRqHIdcXf0a1tf8d00bhMAoHQeurATfbfdQPWvnMyxg5l2hlsP7lJkROscoKHCA6DBEa3MA85lqIHNFRiF9VlPF30KKg+MfS+aqXg9112ManCREkJNG8nZWPTJb5MY\\\\\\\/7jRbRjJmZT+snMHTUClm1pBQx3WCb3d4mq\\\\\\\/MHXCWuK9b+H0nR3gizpOkdIfHP02NXvzqqZ6SeyEljFNsUBg5lNzOHUA7LUUQU9WjkVpOpKin9dUzCDWvxazRQ4L6c2t8K0RqIm9rMXceY0WgIbsILvdo\\\\\\\/l24XFqtElM5bI\\\\\\\/s6yrtb9GJAtjb\\\\\\\/sFJjSYg2wWF8SslRinnAHQyvql\\\\\\\/57vH1lgC99tPdGvp2ttefwq44xAf71rHjiT1YYw8uIxe8rz0\\\\\\\/E9klz+xjdbKs4iGHJO2XphO2FJTEHD68fUEsXucAjVdu+FE6Pndcbcho0EnGuNn5xBq7TQ5ToUJ9EQOs4w\\\\\\\/+oK4C\\\\\\\/APwKTzXR7WQJ\\\\\\\/pprXVYru9tdWWrVwCMAgvB\\\\\\\/1AUWNOchszpmammzhr9ZWZRZwfjvQ8hLqPzDA6uUr0a7mtJekuVtfsX7YdDWbs8MakDxOjJ+RRyl4yB7IRArmIvmm1dzQ0\\\\\\\/bNVPbPma7NO047j06\\\\\\\/s4ipNpFgqRW9iWXR55fjv4UbPvNNffU+EB9LBMh74MF7rkmfrpmcnq8nQZifNpz+7iPQCjFk9Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022O0BRSEi7QKVwDGWDeeWVQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cd0c6468bc5622c4aa6fb73d6d7b83b6b4867abce313a706b3efd7a99d127f95\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pTRdjRNxrps\\\\\\\/t9RZ+6aQ\\\\\\\/Dp\\\\\\\/m1oYiKg5ZoDMG2\\\\\\\/0+epU358SeysTuMONtqZYpkibCaSjngP9HajPhyQV5SMmZoUVj1hbTCXv9QLkY+VSVPIHOZUPhepipXGXO0aF3ym78Lv0RCH4HGlfZb9bD6slwN7t9IcRW11JdxhIOXe3ewXQbtX77RWnanoXhDwquwkDEIGhuS2G3shBCkUd3CgOV+hISCuPyNclzu\\\\\\\/Pq1YVBV+NQcG+hps4FitgDbqjjCoqdvnyZ8PgSo4kRRmnLSFfaR1pb7o2VzUXvs\\\\\\\/XngTzdOqOkBIwy7p8kJ+pTMVet5s+FqbS3D7hYbYi6bZKb3sl\\\\\\\/5RtLWQZu4B8wjYi8FO8w796kHvH5X7Z95h0Xb\\\\\\\/AvXmkSIdvc0WA2VW\\\\\\\/BGHor5yjtqReNKdOfUIbL9dEYqyWAYvTQMqxQ2c1OBYZWXAA\\\\\\\/fA\\\\\\\/hVRgU2KFO+hV7ExbwSEqycRGMoniJO5AwcBBwS462cep\\\\\\\/h+NbVflfzLYvoyFwu\\\\\\\/ZE17YjI7vx\\\\\\\/p8EhiVp9KLfuIZK2iE8pOEk3wFNB+WlxqrQ4841Bxrmqp\\\\\\\/LHkl7T+aLspiGYkt1dgZAudOyLvVjGehDrFhmfiqmXHXMsV88q+cukDRitk46bfp27gZN6psjIx0uEu6\\\\\\\/\\\\\\\/Kzdg3UaniAJdjRU8h+LMcnVH3Wqu7Ak9DvhwLhXuSB\\\\\\\/ICO1szpSs\\\\\\\/NMYGn56vH9QPfnO4i9UY8Qy7uPTIOYzDL9pIWY5tlnQ3c5\\\\\\\/luGIu6w85vcgFYtYYq9wEFZI+qF5KCWcLWXhX8niFpRYV2zbfp2zqfy+aSNK9AgTosyapywv9e8tUr+R0DHhAzsXUT9r5eMvg45JXQXUWVpJ8k6uY7pNFpeCGOUEeyvLbtrd9y5i6ytSJcTp6M2Z6Dtc+Hd8+oJ5bnSNrjKnYOQ+VoOiJXHR3RY72cw0WYLRmaj8E1bRg2cydkrBnDhkqkuF58vXJ9O8JS98pGgSggUnhR+GCTTekFx8IY\\\\\\\/I7MnIeHnACtU9Gu7wBFdgwnyc1M6Z7QsFQo4TjEYD3JCYCUuzHvBuKU3IvA5urcA1s2BjpWozocuRkDFaL1goOaca889kAyvxUo\\\\\\\/gWPybnNi\\\\\\\/\\\\\\\/KLN1FVZOGg4ay1+3paCsid4o+JxRqau8phcXABSlluR7ORJUiLmT\\\\\\\/yDkr+6+H7EPq7elF2WuR+l\\\\\\\/W\\\\\\\/h4Ui6UvBFsHg0pPUyeb1zxjFZ9V0Li9wXYILn6X9Nbx\\\\\\\/rGc0x14himcG6VKO+o2Stw8R6xYPLBJ9Grr+1SyhAG7U\\\\\\\/y9FNuaRBwKCSTBmi71FlMLDSYbH3Q0iBUJymStmkwkU2V9LJExCzGrECsHrPJ3HkzrTvRou3dQAddMBC8Y8HdaUbzCoFzNLF43zA3uUg+zzOX60n+unO8IcjOayX\\\\\\\/rkLiCUFVqm02D5EREXoTdi0nrWOxqPvxTnOsAPQ60q3ofHYxZcCck5iA\\\\\\\/AkaG6vLmBpDoV45Y0BLboRZxsH2OfjxRJoCHDm1DAlXS9chGEtUVpLNRaeIX5ROTjXmrvIfj8ucmwicaaGoyucCv1TL7xDhB9ZJEszlyuXMCW0+JkHroJWtDdbr2omVVH0ax24oW2Wy0xRxI061wGOpS6eFBQqBXxCS+9SWXpPz6D0KY3eai6EFSlgFnXKeTOfJy3TR0pxjTbnEEGNXM2XhrdxTmzAPs9AxubtXfj5BpgGOMfx2JU9e2arV9djeeT6piEMyJGTeCURbFoNYtMP8KR5W5xjEzRDh84LmuxAYUMrbepn7G1SkILdl+53r5Egqyfk8FE7v96HvhhrEV+1tgiOezqbov\\\\\\\/pvE8HLiSA45pBvRyjzF2LizosLYxeDk6yWUrrs3tHo9zuVIHgayVnWOMz4\\\\\\\/K+jFvtNIRByMEhWIL\\\\\\\/G2AvJWd6ngPzTeisJfDN1xnICCzm\\\\\\\/wOadrrZGR6nSp1OWGS4EGt9Cj6dRYwG1bSweAgwL9Xr6iOvwuurK8JHS0v5hFNpYkbajkL5sx4KJQELi+OpX5CA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mlNUq0YJCVW90AIt0lg8Fw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022922bb879d3cb65bea3f87b5d5f40a8a92418df16b1f64b84b3bbd22a04d61d22\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493673 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2071" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8hPJwhs9JkcGNCykVlg0-L-qMk2m70lACGMRLzB4Jk8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2071" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ybKHYM1_tMQ2NIzct2ltclOAOz30ELzo64DI24qlGgs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2061" title="A Genetic Screen Using the Drosophila melanogaster TRiP RNAi Collection To Identify Metabolic Enzymes Required for Eye Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2089" title="ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493673" id="node493673" data-pisa="ggg;9/7/2071" data-pisa-master="ggg;g3.119.400166" data-apath="/ggg/9/7/2071.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nikita A. Sakhanenko</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gareth A. Cromie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span> and <span class="highwire-citation-author" data-delta="3">David J. Galas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2071-2088;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400166 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493673" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nikita A. Sakhanenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikita%2BA.%2BSakhanenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sakhanenko%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANikita%2BA.%2BSakhanenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:nsakhanenko@pnri.org" class="" data-icon-position="" data-hide-link-title="0">nsakhanenko@pnri.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gareth A. Cromie</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:nsakhanenko@pnri.org" class="" data-icon-position="" data-hide-link-title="0">nsakhanenko@pnri.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David J. Galas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BGalas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galas%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BGalas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2071" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493673" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493673/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2071.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493673" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493673/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2071.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493673" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493673/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2071.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493673" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493673/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We describe an information-theory-based method and associated software for computationally identifying sister spores derived from the same meiotic tetrad. The method exploits specific DNA sequence features of tetrads that result from meiotic centromere and allele segregation patterns. Because the method uses only the genomic sequence, it alleviates the need for tetrad-specific barcodes or other genetic modifications to the strains. Using this method, strains derived from randomly arrayed spores can be efficiently grouped back into tetrads.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/yeast-tetrad-analysis" class="hw-term hw-article-keyword hw-article-keyword-yeast-tetrad-analysis" rel="nofollow">yeast tetrad analysis</a></li><li class="kwd"><a href="/keyword/genetic-segregation-patterns" class="hw-term hw-article-keyword hw-article-keyword-genetic-segregation-patterns" rel="nofollow">genetic segregation patterns</a></li><li class="kwd"><a href="/keyword/information-theory" class="hw-term hw-article-keyword hw-article-keyword-information-theory" rel="nofollow">information theory</a></li><li class="kwd"><a href="/keyword/dependency-detection" class="hw-term hw-article-keyword hw-article-keyword-dependency-detection" rel="nofollow">dependency detection</a></li></ul><p id="p-3">In many eukaryotes, including the genetically tractable yeasts <em>Saccharomyces cerevisiae</em> and <em>Schizosaccharomyces pombe</em>, the filamentous fungus <em>Neurospora crassa</em>, and the unicellular green alga <em>Chlamydomonas reinhardtii</em>, it is possible to recover all four of the haploid products of a single meiosis, tetrads. These tetrads can be characterized genetically and phenotypically. Tetrad analysis is a powerful technique that is routinely used to make associations between genetic variation and phenotype, uncover gene-gene interactions, and identify non-reciprocal meiotic recombination events (<em>e.g.</em>, gene conversions).</p><p id="p-4">The manual processes of isolating, disrupting, and arraying spores in conventional tetrad analysis have limited its application to relatively small-scale studies. The conventional method has two steps that are difficult to automate, isolating tetrads away from unsporulated cells in the culture and capturing the sister spore relationships of the resulting progeny strains by arraying the spores in a gridded pattern. We previously described a method, BEST (Barcode Enabled Sorting of Tetrads) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ludlow <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Scott <em>et al.</em> 2014</a>), that uses a meiosis-specific GFP fusion protein to isolate tetrads by fluorescence-activated cell sorting of tetrads and molecular barcodes to identify sister spores of the same tetrad by DNA sequencing.</p><p id="p-5">Although plasmid-borne, tetrad-specific molecular barcodes are well suited for laboratory strains of <em>S. cerevisiae</em>, they may not be as useful for organisms that are less genetically tractable (<em>e.g.</em>, with low transformation efficiency or poor maintenance of extrachromosomal plasmids) or for the construction of non-genetically modified strains. The problem of reconstructing tetrad information from a large set of randomly arrayed spores can be viewed as two sub-problems: (1) finding a reliable measure that identifies groups of four sister spores (a tetrad); and (2) defining an appropriate search strategy to efficiently traverse a very large set of possible spore groupings while applying this measure. Here, we describe an information-theory-based metric that solves the first sub-problem and software implementing a search strategy utilizing this metric that solves the second. Because the method uses only the genomic DNA sequence of the meiotic products, it can be applied to strains or organisms for which genetic manipulation is difficult or undesirable.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>The mechanisms of recombination and chromosome segregation produce unique genotypic signatures for each meiosis</h3><p id="p-6">Meiosis is a process in which a diploid cell undergoes one round of DNA replication followed by two rounds of chromosome segregation and cell division to produce four recombinant haploid progeny. In the first meiotic division, the two homologs of each chromosome recombine and then segregate to the opposite poles of the meiotic spindle. In the second meiotic division, the two chromatids of each recombinant chromosome segregate with no further recombination (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Therefore, in the absence of rare gene-conversion events, at each position that is heterozygous in the original diploid, exactly two spores will inherit allele “A” and exactly two will inherit allele “B” (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Additionally, because sister chromatids only segregate from each other at the second meiotic division, two of the spores will have identical centromeric alleles for every chromosome, while the other two spores will both have the mirror image of this pattern (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Thus, centromere allele segregation patterns, constrained allele frequencies, and patterns of recombination breakpoints can be viewed as genotypic signatures for individual meioses. These signatures could, in principle, be used to reconstruct tetrads computationally from the DNA sequences of randomly arrayed spores.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Behavior of a single chromosome during meiosis. In the initial heterozygous diploid (top) there are two copies of the &#x201C;A&#x201D; haplotype (blue chromatids) and two copies of the &#x201C;B&#x201D; haplotypes (red chromatids). Centromeres are shown as circles. The two &#x201C;A&#x201D; centromeres remain paired until the second meiotic division, as do the two &#x201C;B&#x201D; centromeres. Spores (haploid meiotic products) are shown as dotted ovals. (B) Segregation pattern shown for 3 chromosomes. For each chromosome, segregation of the &#x201C;red&#x201D; or &#x201C;blue&#x201D; homologs to the left or to the right side at the first meiotic division occurs at random, but for each chromosome the two sister &#x201C;red&#x201D; and the two sister &#x201C;blue&#x201D; centromeres always remain paired until the second meiotic division. Therefore, at each centromere the two leftward spores always share the same allele and the two rightward spores share the other allele." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Behavior of a single chromosome during meiosis. In the initial heterozygous diploid (top) there are two copies of the &#x201C;A&#x201D; haplotype (blue chromatids) and two copies of the &#x201C;B&#x201D; haplotypes (red chromatids). Centromeres are shown as circles. The two &#x201C;A&#x201D; centromeres remain paired until the second meiotic division, as do the two &#x201C;B&#x201D; centromeres. Spores (haploid meiotic products) are shown as dotted ovals. (B) Segregation pattern shown for 3 chromosomes. For each chromosome, segregation of the &#x201C;red&#x201D; or &#x201C;blue&#x201D; homologs to the left or to the right side at the first meiotic division occurs at random, but for each chromosome the two sister &#x201C;red&#x201D; and the two sister &#x201C;blue&#x201D; centromeres always remain paired until the second meiotic division. Therefore, at each centromere the two leftward spores always share the same allele and the two rightward spores share the other allele.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F1.medium.gif" width="440" height="384"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2071/F1.medium.gif" width="440" height="384"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493614" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">(A) Behavior of a single chromosome during meiosis. In the initial heterozygous diploid (top) there are two copies of the “A” haplotype (blue chromatids) and two copies of the “B” haplotypes (red chromatids). Centromeres are shown as circles. The two “A” centromeres remain paired until the second meiotic division, as do the two “B” centromeres. Spores (haploid meiotic products) are shown as dotted ovals. (B) Segregation pattern shown for 3 chromosomes. For each chromosome, segregation of the “red” or “blue” homologs to the left or to the right side at the first meiotic division occurs at random, but for each chromosome the two sister “red” and the two sister “blue” centromeres always remain paired until the second meiotic division. Therefore, at each centromere the two leftward spores always share the same allele and the two rightward spores share the other allele.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Here we report a computational method, hereafter called “tetrad reconstruction”, that uses the pattern of centromere segregation and the constrained allele frequencies within a tetrad to infer the original sister spore relationships of recombinant progeny. Our method takes genotype data for all of the progeny strains as input and then proceeds in two steps. First, the centromere segregation pattern is used to reduce the number of potential spore patterns to be searched. Then, the constrained allele segregation patterns are used as the signal to identify members of the same tetrad.</p></div><div id="sec-3" class="subsection"><h3>Using centromere segregation patterns to reduce the search space for spores from the same tetrad</h3><p id="p-9">With an appropriate metric based on allele frequencies or recombination breakpoints, it is possible to distinguish a group of spores from the same tetrad from other groups of spores. However, applying this metric in an exhaustive, brute force search across all possible groups of spores in a large dataset is computationally demanding. To reduce this search space and simplify the computational problem, we implemented an efficient heuristic based on the segregation patterns of centromeres in tetrads, <em>i.e.</em>, the fact that two spores of a tetrad harbor the same alleles at each centromere and the other two spores both share the opposite pattern (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). Our heuristic search leverages this property by first attempting to partition the set of all spores into clusters of spores whose centromere-flanking markers are either a perfect match or a complete mismatch. Unless a polymorphic marker is present at the centromere itself, it is not possible to determine the haplotype origin (“A” or “B”) of centromeres with absolute certainty. Therefore, we compute the probability of each centromere coming from haplotype “A” or haplotype “B” based on the alleles of the flanking markers and the empirically-estimated recombination frequency between them and the centromere (<strong>Appendix G</strong>). Given these probabilities, we derive a similarity coefficient between all spores (<strong>Appendix B</strong>) and use a fast greedy algorithm (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Clauset <em>et al.</em> 2004</a>) to cluster spores based on these similarity coefficients.</p><p id="p-10">There are <em>2<sup>N</sup></em> unique segregation patterns, where <em>N</em> is the number of chromosomes. However, only half of these patterns (<em>2<sup>N-1</sup></em>) can uniquely identify a tetrad, since each tetrad contains two patterns opposite to one another (see <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). Therefore, for organisms with a sufficiently large number of chromosomes, our clustering algorithm based on the centromere segregation heuristic should produce small clusters that contain all members of a given tetrad. For example, in <em>S. cerevisiae</em>, which has sixteen chromosomes, the chance of two tetrads sharing a centromere segregation pattern is 1/2<sup>15</sup>. However, in an organism with fewer chromosomes, such as <em>S. pombe</em>, which has only three, more than one tetrad will often be assigned to a single cluster. In addition, factors such as sequencing errors, or crossovers between a centromere and its flanking markers, can also lead to false positive or false negative tetrad assignments. Therefore, although it reduces the search space for subsequent steps, this heuristic alone is not sufficient to accurately reconstruct tetrads.</p></div><div id="sec-4" class="subsection"><h3>Using information theory to reconstruct tetrad relationships based on 2:2 allele segregation</h3><p id="p-11">The metric that we chose to use for unambiguously identifying members of the same tetrad is based on the fact that at each marker locus in a tetrad, two spores inherit the “A” allele and two spores inherit the “B” allele (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). Thus, the allele patterns within a tetrad are constrained and knowledge of the genotype of one spore changes the allele probabilities for the other three spores. For example, at every position where an “A” allele is observed in one spore, the probability of the “A” allele in any of the remaining three spores of the same tetrad changes from 50 to 33%. In contrast, knowledge of the genotype of a spore from one tetrad does not affect the allele probabilities in spores from different tetrads. As such, tetrad-specific relationships can be viewed as dependencies among the four allele-vectors of a tetrad (one vector for each spore genotype), and such dependencies can be detected using methods from information theory. In contrast to the centromere heuristic, this constrained allele frequency approach uses a much larger number of genotyped markers, making the approach less sensitive to individual genotyping errors and more successful at disambiguating tetrad assignments.</p><p id="p-12">Mutual information is a well-known measure that quantifies the amount of dependency between two categorical variables (<strong>Appendix A</strong>), and interaction information (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">McGill 1954</a>) is a multivariable generalization of this measure (<strong>Appendix A</strong>). While interaction information has a number of drawbacks (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bell 2003</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Jakulin and Bratko 2004</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Sakhanenko and Galas 2011</a>), it can be used in principle to devise measures of dependency among any number of variables. Because the genotypes of a group of spores from the same tetrad are a set of dependent variables, they should produce a strong interaction information signal. In contrast, if the genotype of a spore from a different tetrad (an independent variable) is then added to the group, the interaction information should be close to zero. Thus, an interaction information approach might be able to identify groups of four strains that were sister spores from the same tetrad. Furthermore, in cases where spores are missing or inviable, the allele vectors of the remaining spores will still show dependencies among one another, detectable by mutual information between two spores or interaction information between three spores.</p><p id="p-13">To test the ability of interaction information to identify groups of spores from the same tetrad, we generated a simulated dataset of 100 <em>S. cerevisiae</em> tetrads with 1000 markers, 1% noise (genotyping errors), and 5% missing data (<strong>Appendix F</strong>). We then calculated interaction information for the real tetrads and for groups of 4-spores derived from different tetrads. On average, true tetrads scored highly by this metric, while incorrectly grouped spores had scores centered on zero. However, while most incorrect groups of spores scored poorly, there were a number of false positives, <em>i.e.</em>, incorrect groupings that scored as highly as some of the true tetrad groups (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). A similar result was seen with groups of three spores from the same tetrad <em>vs.</em> incorrect groupings of 3-spores, and the overlap was even stronger in this case (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). These results suggest that noise in the data limits the ability of interaction information to cleanly distinguish correct groups of three or four spores derived from the same tetrad from incorrect groupings.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction Information computed on groups of (A) 4 spores and (B) 3 spores. All measures were computed on the simulated S. cerevisiae data (1000 markers, 100 tetrads, 1% noise, and 5% missing data). Panel (A) distinguishes four possible categories of 4-spore groups based on the number of spores derived from the same tetrad (shown on the x-axis) and shows a letter-value (LV) plot for each category. The purple LV plot shows the Interaction Information scores for all possible real tetrads. Similarly, panel (B) shows the scores for three different categories of 3-spore groups. For the categories where all spores are from different tetrads (the red LV plots in (A) and (B)), only 2 million randomly sampled groups are shown. The Interaction Information signal associated with true sister spores is negative at the 3-spore level and positive at the 4-spore level, while it is clustered around zero in both cases when sets contain non-sister spores." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interaction Information computed on groups of (A) 4 spores and (B) 3 spores. All measures were computed on the simulated &lt;em&gt;S. cerevisiae&lt;/em&gt; data (1000 markers, 100 tetrads, 1% noise, and 5% missing data). Panel (A) distinguishes four possible categories of 4-spore groups based on the number of spores derived from the same tetrad (shown on the x-axis) and shows a letter-value (LV) plot for each category. The purple LV plot shows the Interaction Information scores for all possible real tetrads. Similarly, panel (B) shows the scores for three different categories of 3-spore groups. For the categories where all spores are from different tetrads (the red LV plots in (A) and (B)), only 2 million randomly sampled groups are shown. The Interaction Information signal associated with true sister spores is negative at the 3-spore level and positive at the 4-spore level, while it is clustered around zero in both cases when sets contain non-sister spores.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F2.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2071/F2.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493553" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Interaction Information computed on groups of (A) 4 spores and (B) 3 spores. All measures were computed on the simulated <em>S. cerevisiae</em> data (1000 markers, 100 tetrads, 1% noise, and 5% missing data). Panel (A) distinguishes four possible categories of 4-spore groups based on the number of spores derived from the same tetrad (shown on the x-axis) and shows a letter-value (LV) plot for each category. The purple LV plot shows the Interaction Information scores for all possible real tetrads. Similarly, panel (B) shows the scores for three different categories of 3-spore groups. For the categories where all spores are from different tetrads (the red LV plots in (A) and (B)), only 2 million randomly sampled groups are shown. The Interaction Information signal associated with true sister spores is negative at the 3-spore level and positive at the 4-spore level, while it is clustered around zero in both cases when sets contain non-sister spores.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">To explore this problem further, we considered the behavior of interaction information at multiple levels of complexity. As expected, a real tetrad has strong interaction information signal at the 4-spore level as well as at the 3-spore level for all subgroups of three spores (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-fig-14-1" class="xref-fig" href="#F14">14</a>). In contrast, when an incorrectly assembled tetrad has a relatively high interaction information signal at the 4-spore level due to noise, this signal does not extend to its 3-spore subgroups (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, green oval). We also observed similar, but noisier, patterns between the 2- and 3-spore levels (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>, green oval). Thus, we hypothesized that combining interaction information at the 4-spore and 3-spore levels or from the 3-spore and 2-spore levels should substantially strengthen the signal separating real 4- and 3-spore tetrads from false ones.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the amount of information between (A) 2-spore and 3-spore levels and between (B) 3-spore and 4-spore levels as measured by interaction information. All measures were computed on the same data as in Figure 2. Panel (A) shows the scatter plot of scores computed on various groups of 4 spores and their 3-spore subsets while panel (B) shows the scores for groups of 3 spores and their 2-spore subsets. Each group is colored red if all spores of the group came from the same tetrad and blue if at least two spores came from a different tetrad. The scores of the red sets are plotted in their entirety, whereas for the blue sets we randomly selected 2 million groups. Green ovals indicate the situation when, due to noise, the interaction information shows relatively high signal at one level for a group of spores derived from more than one tetrad. At the level below, the signal from these groups is close to zero." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the amount of information between (A) 2-spore and 3-spore levels and between (B) 3-spore and 4-spore levels as measured by interaction information. All measures were computed on the same data as in Figure 2. Panel (A) shows the scatter plot of scores computed on various groups of 4 spores and their 3-spore subsets while panel (B) shows the scores for groups of 3 spores and their 2-spore subsets. Each group is colored red if all spores of the group came from the same tetrad and blue if at least two spores came from a different tetrad. The scores of the red sets are plotted in their entirety, whereas for the blue sets we randomly selected 2 million groups. Green ovals indicate the situation when, due to noise, the interaction information shows relatively high signal at one level for a group of spores derived from more than one tetrad. At the level below, the signal from these groups is close to zero.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F3.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2071/F3.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493612" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-16" class="first-child">Comparison of the amount of information between (A) 2-spore and 3-spore levels and between (B) 3-spore and 4-spore levels as measured by interaction information. All measures were computed on the same data as in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. Panel (A) shows the scatter plot of scores computed on various groups of 4 spores and their 3-spore subsets while panel (B) shows the scores for groups of 3 spores and their 2-spore subsets. Each group is colored red if all spores of the group came from the same tetrad and blue if at least two spores came from a different tetrad. The scores of the red sets are plotted in their entirety, whereas for the blue sets we randomly selected 2 million groups. Green ovals indicate the situation when, due to noise, the interaction information shows relatively high signal at one level for a group of spores derived from more than one tetrad. At the level below, the signal from these groups is close to zero.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To combine interaction information at different spore-number levels, we used a measure based on differential interaction information, called “delta” (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Galas <em>et al.</em> 2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Sakhanenko and Galas 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Galas and Sakhanenko 2019</a>). Differential interaction information quantifies the change in interaction information that occurs when a new variable is added to a set of existing variables. Unlike the interaction information measure, differential interaction information is not symmetric, but is specific to which variable is added. A symmetric measure results from the product of differential interaction information with all possible choices of the added target variable, and this product is “delta” (<strong>Appendix A</strong>).</p><p id="p-18">Consistent with our hypothesis, delta performed significantly better than interaction information in distinguishing groups of spores from the same tetrad from incorrect groupings (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, also <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figures 13</a> and <a id="xref-fig-14-2" class="xref-fig" href="#F14">14</a>). By combining information at different degree levels, the delta measure allowed us to distinguish the real tetrads from all other 4-spore groups, with a false positive rate of zero using the test dataset. Furthermore, the difference between real tetrads and incorrect 4-spore groupings was orders of magnitude larger when measured with delta as opposed to interaction information. This difference can also be seen at the 3-spore level: groups of 3 sister spores are separated away from other 3-spore groups if we use delta (see <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4D</a>) as opposed to interaction information (see <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>) where these two distributions of 3-spore groups are more ambiguous.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the ability of Interaction Information and the delta measure to distinguish groups of sister spores from all other groups. Panel (A) shows LV plots of Interaction Information scores computed on groups of 4 spores from the same tetrad (in blue) and on all other groups of 4 spores (in orange). Panel (B) shows delta scores computed on the same 4-spore sets. Similarly, panels (C) and (D) show Interaction Information and delta scores computed on triplets of sister spores (in blue) and triplets of spores from different tetrads (in orange). All measures were computed on the same data as in Figure 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the ability of Interaction Information and the delta measure to distinguish groups of sister spores from all other groups. Panel (A) shows LV plots of Interaction Information scores computed on groups of 4 spores from the same tetrad (in blue) and on all other groups of 4 spores (in orange). Panel (B) shows delta scores computed on the same 4-spore sets. Similarly, panels (C) and (D) show Interaction Information and delta scores computed on triplets of sister spores (in blue) and triplets of spores from different tetrads (in orange). All measures were computed on the same data as in Figure 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F4.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2071/F4.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493550" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-19" class="first-child">Comparison of the ability of Interaction Information and the delta measure to distinguish groups of sister spores from all other groups. Panel (A) shows LV plots of Interaction Information scores computed on groups of 4 spores from the same tetrad (in blue) and on all other groups of 4 spores (in orange). Panel (B) shows delta scores computed on the same 4-spore sets. Similarly, panels (C) and (D) show Interaction Information and delta scores computed on triplets of sister spores (in blue) and triplets of spores from different tetrads (in orange). All measures were computed on the same data as in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Thresholds for identifying spores from the same tetrad and validation using 2:2 allele segregation</h3><p id="p-20">To construct a classifier that uses delta for tetrad reconstruction, we need to identify a threshold that distinguishes true-positive tetrads from false-positive groups of spores with high likelihood. The null hypothesis is that there is no difference in delta scores between groups of sister spores and groups of spores that were selected by chance. We therefore create a null distribution whose elements are groups of randomly selected spores, which are mostly non-tetrad groups. Computing delta scores for all the elements of the null distribution, the delta threshold is then defined based on a user-defined <em>p</em>-value. The default for 4-spore tetrads is 0.05, but our software allows the user to adjust this parameter. A group, whose delta score is above the threshold, is then identified as a candidate tetrad for validation. We note that the null distributions obtained by this permutation also include a small proportion of groups of spores from the same tetrad, so the cutoffs are, in practice, slightly conservative.</p><p id="p-21">In a real tetrad with no gene conversions every marker segregates 2:2, <em>i.e.</em>, two spores inherit the “A” allele and two spores inherit the “B” allele. Since this information is not explicitly used in calculating the delta scores (which reflect the dependencies between the spore genotypes, but not the exact form of the dependencies), it can be used as a subsequent validation test. For a candidate tetrad to be validated and labeled as a real tetrad, we require the fraction of markers with 2:2 segregation to be close to 1 (above 0.9 by default). For a partial, 3-spore tetrad the process is similar, but the method uses 2:1 segregation and a cutoff of 0.95. We refer to the process of identifying candidate tetrads, performing validation, labeling real tetrads and removing them from the pool of spores used for further consideration as <em>tetrad verification</em>.</p></div><div id="sec-6" class="subsection"><h3>Software implementation</h3><p id="p-22">Our software implementation combines the previously described methods as follows (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). First, spore genotypes are preprocessed to remove duplicate spores (identified using the edit-distance between allele vectors) and spores with a large number of missing values. Next, the remaining spores are clustered based on their centromere-flanking markers. Then, the software searches for real tetrads within each cluster based on the delta score of each spore group. Afterward, all remaining spores of all clusters that have not been assigned to a tetrad are analyzed together to identify any remaining tetrads and then any <em>partial tetrads</em>, <em>i.e.</em>, groups of three sister spores (with one spore missing from the dataset) and groups of two sister spores (with two spores missing from the dataset). Finally, spores are assigned tetrad labels and output to the user. A detailed flow chart of the search is presented in <strong>Appendix D</strong> (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure 12</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A general overview of the flow of the software for tetrad detection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A general overview of the flow of the software for tetrad detection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F5.medium.gif" width="224" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2071/F5.medium.gif" width="224" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493567" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-23" class="first-child">A general overview of the flow of the software for tetrad detection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The default method for identifying true tetrads within each centromere-cluster is to carry out a <em>direct</em> search of all possible combinations of 4 spores using delta. Any set of 4-spore combinations passing the delta score threshold then undergoes the tetrad verification process and, if it passes, is labeled a tetrad and removed from further analysis. The search then continues among the remaining spores. However, for large clusters, exhaustively combing the 4-spore search space in this way is computationally expensive. Therefore, for clusters containing over five spores, tetrads are instead identified in an <em>indirect</em>, two-step process. First, because the 3-spore search space is much smaller than the equivalent 4-spore space, the software computes the delta measure on all possible 3-spore combinations and detects those that could be part of true tetrads (hereafter <em>triplets</em>). Next, the remaining single spores in the cluster are added to each triplet, and the delta scores are recalculated. Any 4-spore groups that pass the delta score threshold and tetrad verification process are then labeled a full tetrad and removed from further analysis.</p><p id="p-25">The indirect search method described above is an example of a <em>shadow</em> search (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Sakhanenko and Galas 2015</a>). The approach leverages the fact that a functional dependency of <em>N</em>-variables usually has a detectable signal at a lower degree (fewer variables). For tetrad detection with the delta measure, this is in fact the case, since triplets (3-spore groups that are subsets of a real tetrad) have strong delta scores (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a> and <a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure 13A</a>). Therefore, full tetrads can be assembled by identifying high scoring 3-spore subsets within each cluster and then identifying which of the remaining spores belong in a tetrad with each 3-spore group. While this method is more computationally efficient, it relies on the ability of the 3-spore delta measure to distinguish groups of spores from the same tetrad from incorrect groupings, and this discrimination is not as accurate as the one that uses delta calculated on 4-spores (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a> <em>vs.</em> <a id="xref-fig-4-6" class="xref-fig" href="#F4">4D</a>). Thus, when computationally feasible, the exhaustive 4-spore approach should be used.</p><p id="p-26">After the direct or indirect identification of tetrads within the centromere clusters, the software searches for tetrads in which only three of the four spores have been placed in the same initial cluster (likely due to noise or a small number of markers). This is also done using the shadow approach. The software computes the delta measure on all possible combinations of three spores in the centromere cluster. Then, 3-spore groups that pass the significance filter are combined with spores from the set of unclustered spores. The software computes the delta measure on these new sets of four spores and performs tetrad verification.</p><p id="p-27">Once every cluster of spores from the centromere clusters has been analyzed, the software moves on to an exhaustive search for remaining tetrads by collecting all the unlabeled spores into a single cluster and applying a combination of shadow and direct searches. If at any point in the search a candidate tetrad fails the verification step based on 2:2 marker segregation, its spores are put back into the search space. This approach is then repeated for triplets, consisting of three sister spores. Finally, any remaining pairs of two sister spores are identified using mutual information, leaving only the unclassifiable single spores.</p><p id="p-28">The software takes as input a tab delimited text file containing the genotype of each spore. The parameters of the software are defined in a configuration file and preset to the most frequently used values by default. A user can adjust these values in the configuration file to get a better performance for specific situations. Some of the parameters of interest are</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-29">CEN_CALLING controls whether to estimate the recombination frequency from the data or skip it and use the <em>S. cerevisiae</em> default specified by COS_PER_MEGA. Estimating recombination frequency on large datasets could be slow, so one might want to skip it, use a published value, or estimate it once before varying other parameters.</p></li><li id="list-item-2"><p id="p-30">CLUSTERING controls whether we employ the centromere-based clustering first, or go straight to the exhaustive search.</p></li><li id="list-item-3"><p id="p-31">SIMILARITY_COEFFICIENT is a threshold for centromere similarity between two spores, controlling when the software calls two spores similar. Using the similarity coefficient, we are able to considerably reduce the search space speeding up the processing of large data sets. Smaller centromere similarity coefficient cutoffs result in larger numbers of smaller clusters. Increasing this cutoff will merge the clusters together until they become one cluster – the entire data set. The user should set the cutoff such that there are many medium size (under 300 spores) clusters.</p></li><li id="list-item-4"><p id="p-32">D4_PVALUE and D3_PVALUE specify the cutoffs for the p-value of the Delta4 and Delta3 scores. To avoid false positives, the user can increase the stringency of the search by lowering the p-value of the Delta4 and Delta3 scores. The user can also adjust the segregation cutoffs (SEG_CUTOFF_D4 and SEG_CUTOFF_D3) to filter out the false positives.</p></li></ul><p id="p-33">The full list of the parameters with their descriptions is given in the <em>readme</em> file of the software. The software also comes with the set of examples described in this paper and the corresponding configuration files to process these examples. The information-theory-based dependency detection is implemented in <em>C</em>, whereas the rest of the software is implemented in <em>Python</em>. The software is available at FigShare.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-34">FileS1.rar contains the code of the tetrad assembly software, as well as code used to generate the simulated data, the information files and the test data. FileS2.rar contains all the simulated data used in the paper to test our software. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8111405">https://doi.org/10.25387/g3.8111405</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Testing the software on simulated and real datasets</h3><p id="p-35">We first applied the software to simulated datasets. We generated three sets of error-free <em>S. cerevisiae</em> tetrad genotypes (see <strong>Appendix F</strong> for more details), including recombination events and recording spore relationships: <em>Small</em> (100 tetrads and 1,000 markers), <em>Medium</em> (1,000 tetrads and 10,000 markers), and <em>Large</em> (1,000 tetrads and 100,000 markers). For each of these datasets we created multiple test cases by reducing the number of markers and adding various amounts of missing values and levels of noise (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493493/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493493/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The tetrad detection software applied to various simulated test sets. Three simulated datasets are shown (indicated in the column Set): <em>Small</em> (100 tetrads, 1000 markers), <em>Medium</em> (1000 tetrads, 10000 markers), and <em>Large</em> (1000 tetrads, 100000 markers). The first four columns of the table show the various parameters of the simulated data considered: N<sub>t</sub> shows the number of tetrads in the set, N<sub>m</sub> shows the number of randomly selected markers as well as the percentage of the original set of markers, and columns Noise and Missing show the amount of noise and missing values (in percent) added to the data. The software was run on these test sets with different parameters. The table shows only the parameter settings that produced the optimal result. For the results of the software with other settings see Table 4 in Appendix H. Column “Cent calling” shows whether the software estimates the recombination frequency empirically or uses its default value derived from published data (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Mancera <em>et al.</em> 2008</a>). The column “Sim coeff.” shows the similarity threshold being used. The columns 2:2 cutoff and 2:1 cutoff show the cutoff values for 2:2 and 2:1 segregation score for the software to call two spores sisters. The last two columns show the performance of the software, the total runtime (Intel Core i7-7820X CPU @ 3.60GHz and 64 GB RAM) and the percent of total tetrads detected by the software. Note that in all the cases when the software detected 100% of tetrads, there were no false positives. In the case when 99.8% of tetrads were detected, there was 1 false positive tetrad and 1 false positive triplet (with 1 spore unassigned)</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">We used this simulated data to perform a thorough evaluation of the individual components of the software in various situations. Specifically, we analyzed the performance of allele calling at centromeres, spore clustering based on similarity coefficients, and tetrad verification using segregation scores. <strong>Appendix C</strong> shows the details of the evaluation on the simulated dataset <em>Small</em>.</p><p id="p-37">In general, the total number of markers in the data affected the estimation of the allele calls at centromeres and consequently the initial clustering of the spores. Low numbers of markers caused problems in clustering through several mechanisms. These included poor precision in estimating global recombination parameters and difficulties in accurately estimating the allele at the centromere due to a lack of flanking markers or because the flanking markers were too far apart.</p><p id="p-38">The proportion of missing data, on the other hand, did not have a strong effect on centromere allele call predictions or spore clustering, and any effect was fully overcome by using more markers. The noise in the data (genotyping errors), however, had a notable impact on the performance of the software. High levels of noise strongly affected the estimation of the alleles at the centromeres and spore clustering (<strong>Appendix C</strong>). This problem can, however, also be overcome by increasing the number of markers used. Furthermore, the noise affected the strength of the true tetrad signal relative to the background, making it harder to distinguish the true tetrads statistically from the other 4-spore sets. We note that this was the case for both the delta measure as well as the 2:2 segregation measure. The quantitation of this effect is described in <strong>Appendix C</strong>.</p><p id="p-39">After testing the individual components of our software in this way, we then tested our software on a handful of simulated test sets derived from the sets <em>Small</em>, <em>Medium</em>, and <em>Large</em> and compared the final tetrad assignments to ground truth from the simulations. All the tests were performed on a desktop with Intel Core i7-7820X CPU @ 3.60GHz (8 cores, 16 threads) and 64 GB RAM. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> summarizes these tests and shows the corresponding optimal parameter settings of the software and the resulting performance. A comprehensive list of the tests performed is presented in Table 4 of <strong>Appendix H</strong>.</p><p id="p-40"><a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> shows that the software is able to achieve 100% accuracy with no false positives in almost all the tests: only in tests on the <em>Medium</em> set with only 10% of the original marker set and with 10% noise was the accuracy lower, albeit still 99.8%. In these tests some 4-spore groups scored as high as the real tetrads due to noise in the relatively small number of markers, resulting in two missed tetrads, one false positive tetrad assignment, and one false positive triplet assignment.</p><p id="p-41">The runtime of all the tests without noise was very reasonable: <em>Small</em> tests took under 12 sec, <em>Medium</em> tests took under 4 min, and <em>Large</em> tests took under 11 min. Although with the addition of noise, the runtime of <em>Small</em> tests did not change much due to the small size, it changed drastically for <em>Medium</em> and <em>Large</em> tests. At 5% noise, <em>Medium</em> and <em>Large</em> tests took under 28 min and 37 min correspondingly. Increasing noise to 10% made the runtime go up considerably: <em>Medium</em> tests took 14-27 hr (depending on the size) and <em>Large</em> tests took 31-94 hr. In general, including more markers in the test sets increases the time it takes to calculate each delta score. Without noise, the clustering of spores is very effective, allowing for the search space to be divided into multiple small clusters fully containing tetrads, thus keeping the number of delta calculations low and resulting in a short runtime. With noise however, the initial clustering does not work as well, which inevitably increases the total number of delta calculations necessary and thus increases the runtime.</p><p id="p-42">To achieve the best performance, we varied two software parameters, the similarity coefficient threshold and the segregation cutoffs. In test cases where the number of markers was large, the default similarity coefficient threshold resulted in a small number of very large clusters, causing the software to take a considerable amount of time to complete a run (<strong>Appendix C</strong>). Lowering this threshold increased the number of clusters while reducing their size, which divided the search space more efficiently and resulted in faster processing. Users of this software should consider these options, since the level of noise in experimental datasets is usually not known.</p><p id="p-43">When no noise is present, clustering allows the software to subdivide the spores into sets of same-tetrad spores, identifying all the tetrads at the divide-and-conquer phase (see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). In this situation, keeping the clusters small results in smaller processing time. For the test cases with noise however the clustering does not work as well and, as a result, only some tetrads are identified at the divide-and-conquer phase leading to more time spent in the exhaustive search phase (see <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). In this case making the clusters larger allows the software to extract more tetrads at the divide-and-conquer phase, thus reducing the amount of computation at the exhaustive search phase and the overall processing time. Therefore, for the cases with noise (in particular when the noise level is 10%) the similarity threshold had to be increased relative to the same sized dataset without noise resulting in larger clusters. Furthermore, in the presence of noise the default values of the segregation cutoffs are too stringent, resulting in poor accuracy, since many true tetrads were dismissed. See <strong>Appendix C</strong> for more details about the effects of the similarity coefficient and the segregation cutoffs.</p><p id="p-44">We then applied the software to real biological data from two published datasets. Dataset D1 consists of 412 spores from manually dissected tetrads and the tetrad relationship between spores was recorded by the experimenter (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Sirr <em>et al.</em> 2017</a>). Dataset D2 consists of 3,200 spores that were randomly arrayed on an agar plate, but that contained a tetrad-specific plasmid barcode (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Ludlow <em>et al.</em> 2013</a>). In both cases, the genotype markers were generated by RAD-seq as described in the original publications (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Ludlow <em>et al.</em> 2013</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Sirr <em>et al.</em> 2017</a>). Using these datasets allowed us to compare the effectiveness of our method to the experimentally derived tetrad assignments, which we treated as “ground truth” (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493623/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493623/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The tetrad detection software applied to real biological datasets. (a) Evaluation of the software on dataset D1 using the default parameter setting. Since all the spores in D1 are already labeled according to their tetrad assignment (from manual dissection), we were able to count the number of tetrads identified by the software that were either in agreement with the data labels (correct tetrads), or not (false tetrads), as well as the number of tetrads that the software was not able to identify (missed tetrads). The triplets were counted in the same way. (b) Evaluation of the software on dataset D2 using either the default parameter setting or filtering markers with too many missing values (corresponding to the second and third columns). Although there is no known tetrad assignment of the spores in D2, the spores were barcoded, with spores derived from the same tetrad sharing a barcode. As a result we counted the number of tetrads identified by the software comprised of same-barcode spores (confirmed tetrads). We also counted consistent tetrads comprised of either same-barcode spores or spores with no barcode and incorrect tetrads whose spores have different barcodes. The sum of confirmed, consistent and incorrect tetrads is the total number of tetrads identified by the software. The triplets were counted in the same way</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">For dataset D1, using the default parameters the software achieved over 98% full-tetrad accuracy and over 96% accuracy of identifying triplets (tetrads missing 1 spore). For dataset D2, after filtering out the duplicate spores, our software with default parameters showed 100% agreement between identified tetrads and the barcodes (sum of number of confirmed and consistent tetrads from <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). However, almost 60% of triplets identified by the software in default mode were in disagreement with the barcodes. Filtering to remove markers with large numbers of missing values improved the 3-spore performance on dataset D2 by almost 30%. Note also that filtering markers allowed identifying considerably more tetrads and triplets.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-46">We have described a computational method, based on our previously developed information theory dependency analysis that reconstitutes tetrads using only information from genome sequencing of the meiotic progeny. This method avoids the need for tetrad-specific barcoding or genetic modification of any kind. Instead, our method uses information associated with specific features of tetrad genome sequences, features that result from the mechanisms of meiosis. The software reported here is both simple to use and highly effective in reconstituting tetrads. This approach can significantly increase the power of tetrad analysis in several ways, most notably by vastly increasing the numbers of tetrads that can be analyzed with minimal effort.</p><p id="p-47">Our software integrates a heuristic method for clustering, using centromere proximal markers, with an information theory-based method for signal detection. On both simulated and real data, the software achieves a remarkably high success rate, even in the presence of lost spores and sources of experimental noise. The applications for this software include, the analysis of meiotic recombination and the study of gene conversions or other non-reciprocal genetic events in which a subset of markers deviate from the expected 2:2 segregation pattern. Although the software operates under the assumption that there is no gene conversion, it can successfully handle cases where some amount of gene conversion is present. Since gene conversion results in some markers deviating from the 2:2 segregation pattern, the software will perform in the same way it does in the presence of a comparable amount of noise.</p><p id="p-48">The software can also be applied to any organism in which it is possible to isolate the four products of meiosis. Our software is based on an information theory method, which does not make any assumptions about the data and its underlying structure. Although the genome structure of <em>S. pombe</em> is significantly different from that of <em>S. cerevisiae</em>, the analysis method described here will work equally well on this organism. Because the software only uses the centromere allele patterns of the organism to reduce the search space and thus decrease run times, it can be effectively applied to organisms with smaller numbers of chromosomes. For example, in <em>S. pombe</em>, which has only three chromosomes, the method would still produce several, albeit large, clusters of spores and thus productively divide the search space. In such cases, the software would be able to produce the same results, although with longer run times. Another difference is that <em>S. pombe</em> has a lower recombination rate than <em>S. cerevisiae</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Clément-Ziza <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Fowler <em>et al.</em> 2014</a>) and thus a smaller number of 2:2 segregation patterns. In such organisms, the performance of our software will be similar to its performance in <em>S. cerevisiae</em> with fewer markers.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-11"><h3>Information theory based measures of dependence.</h3><p id="p-49">In the analysis of complex biological systems we need measures that can detect synergistic, multiple variable dependencies. Mutual information is a well-known measure that quantifies the amount of dependency between two variables:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-2.gif"/></noscript></span></span> is entropy of variable X defined as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-3.gif"/></noscript></span></span>. Interaction information has been proposed (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">McGill 1954</a>) as a multivariable generalization of mutual information. This measure has a number of advantages and drawbacks (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bell 2003</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Jakulin and Bratko 2004</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Sakhanenko and Galas 2011</a>) but can be used to devise powerful measures of dependency for any number of variables. The interaction information for three variables, for example, quantifies the difference between the two-variable interaction information (mutual information), with and without knowledge of the third variable:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Here <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-5.gif"/></noscript></span></span> is conditional mutual information, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-6.gif"/></noscript></span></span> is entropy of variable X and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-7.gif"/></noscript></span></span> is a joint entropy of the three variables. Note that the conditional mutual information is actually a difference between interaction informations for two and three variables – a <em>differential interaction information</em>. A general form of interaction information for the set of <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-8.gif"/></noscript></span></span> variables, in terms of marginal entropies can be written as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>In this paper we use a symmetric product of <em>differential interaction information</em>, which we call “delta” (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Galas <em>et al.</em> 2014</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Sakhanenko and Galas 2015</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Galas and Sakhanenko 2019</a>). Differential interaction information quantifies the change in interaction information that occurs when we add another variable to a set of variables, so for three variables it is defined as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>If <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-12.gif"/></noscript></span></span>, then the differential interaction information can be defined in general as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Note that, unlike interaction information, differential interaction information is not symmetric, since <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-14.gif"/></noscript></span></span> in <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">equation 5</a> is a special variable. In order to create a symmetric measure, we take the product of differential interaction information with all possible choices of the target variable:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>We refer to <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-16.gif"/></noscript></span></span> as the <em>delta</em> measure, for m variables. Although this is a general, multi-variable measure, in this paper we focus on delta computed only on 3- and 4-variable sets. We use 3 and 4-variable <em>delta</em>, as well as the pair-wise measure, mutual information, to scan the data from large sets of yeast spores and detect and assemble spore tetrads and their components.</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-12"><h3>Spore similarity</h3><p id="p-50">In order to be able to cluster the spores, we need to define a measure of similarity of two spores based on allelic information at the centromeres. We first start with a deterministic case, when the alleles near the centromeres are known.</p><p id="p-51">Consider two vectors X and Y, whose elements are either 1 or 2, representing allele calls at the centromere for the two corresponding spores. We can then define a vector <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-17.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-18.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-19.gif"/></noscript></span></span>, and 0 otherwise, where N is the number of centromeres. Note that if we define a vector <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-20.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-21.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-22.gif"/></noscript></span></span>, and 1 otherwise, then <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-23.gif"/></noscript></span></span></p><p id="p-52">Consequently, we define S, the coefficient of similarity of X and Y, as follows<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-24.gif"/></noscript></span></span>The coefficient of anti-similarity U is defined similarly to S, but using vector b instead. Note that by definition <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-25.gif"/></noscript></span></span>.</p><p id="p-53">Given the allele calls at the centromeres, we can compute S for all possible pairs of spores and then cluster the spores based on their similarity at the centromeres. In most biological examples, however, the allele values at the centromeres are unknown. We can however estimate what the allele is at each centromere based on the recombination frequency in a given data set and the allele calls at the markers flanking the centromeres (see <strong>Appendix G</strong>), and consequently we can estimate the coefficient of similarity between spores.</p><p id="p-54">Given the probabilities <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-27.gif"/></noscript></span></span> that spores X and Y have alleles a and b at the centromere i, we can define the probability that X and Y are identical at ith position as <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-28.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-29.gif"/></noscript></span></span>If <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-31.gif"/></noscript></span></span> are independent, then the definition above can be simplified as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-32.gif"/></noscript></span></span>Since we do not know the values of <em>X</em> and <em>Y</em>, we cannot compute the similarity coefficient directly, thus we need to estimate it. An estimated coefficient of similarity <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-33.gif"/></noscript></span></span> is computed by taking the expectation of <em>S</em>:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-34.gif"/></noscript></span></span>Note that an estimated coefficient of anti-similarity is <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-35.gif"/></noscript></span></span>.</p><p id="p-55">As an example, we apply this similarity coefficient to the simulated data set of 400 spores and 500 markers with no noise or missing values. For each pair of spores we compute the similarity coefficient based on the centromere allele estimates. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the distributions of coefficients for the spores from the same tetrad and for the spores from different tetrads.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of estimated similarity coefficients for spore pairs from the same tetrad (in red) and from different tetrads (in blue) computed on the simulated data set (500 markers, 400 spores, 0% noise, 0% missing)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of estimated similarity coefficients for spore pairs from the same tetrad (in red) and from different tetrads (in blue) computed on the simulated data set (500 markers, 400 spores, 0% noise, 0% missing).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F6.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/7/2071/F6.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493637" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-56" class="first-child">Distribution of estimated similarity coefficients for spore pairs from the same tetrad (in red) and from different tetrads (in blue) computed on the simulated data set (500 markers, 400 spores, 0% noise, 0% missing).</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">This figure shows the expected pattern that, for centromeric alleles, each tetrad consists of two pairs of identical spores that are reflections of one another. The reflected spores in each tetrad form a distribution near 0, whereas the completely identical spores form a distribution near 1. On the other hand, the spores from different tetrads form a normal distribution centered around 0.5.</p><p id="p-58">From <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> it is clear that we can use the similarity coefficient to cluster the spores from the same tetrads. The distributions from <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> can be transformed by “folding” it in on itself as follows, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-36.gif"/></noscript></span></span>, such that <em>S</em>’ is close to 0 when spores are from the same tetrads and around 0.5 when the spores are from different tetrads. The scores in between correspond to situations when the allele call estimates at the centromere are not enough to identify the spores from the same tetrad due to noise and missing data. We need to find a threshold <em>T</em> that simultaneously maximizes the number of same tetrad spores with <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-37.gif"/></noscript></span></span> as well as the number of spores from different tetrads with <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-38.gif"/></noscript></span></span>. We determined that <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-39.gif"/></noscript></span></span> is the optimal threshold to correctly estimate the centromere allele calls when the number of markers is sufficient and the noise and missing data are minimal.</p></div></div><div class="section app" id="app-3"><h2 class="">Appendix C</h2><div class="section" id="sec-13"><h3>Effects of noise and missing values on software components</h3><p id="p-59">We now investigate how the noise and missing values affect the components of the software. We generated a number of simulated data sets with varying number of markers and spores, as indicated in the text. We treated the data set as a long vector and added noise by flipping the values of the vector at randomly selected positions. Similarly, we added missing values by erasing the values of the vector at randomly selected positions. We considered different levels of noise and missing values, up to 25% for both.</p><div id="sec-14" class="subsection"><h4>Allele calling at centromeres:</h4><p id="p-60">Using the simulated data we looked at the effects of noise and missing data on the estimation of alleles at the centromeres. As expected, increasing noise makes the centromere allele estimates worse (see <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). On the other hand, the estimates do not seem to be affected by the amount of missing data.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average (A) and standard deviation (B) of the number of incorrect allele calls at the centromeres computed on the simulated data set (500 markers, 400 spores). The x-axis corresponds to the fraction of the missing values in the data set. Each line (and the color gradient) corresponds to the noise level, ranging from 0 to 25%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average (A) and standard deviation (B) of the number of incorrect allele calls at the centromeres computed on the simulated data set (500 markers, 400 spores). The x-axis corresponds to the fraction of the missing values in the data set. Each line (and the color gradient) corresponds to the noise level, ranging from 0 to 25%.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F7.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/7/2071/F7.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-61" class="first-child">Average (A) and standard deviation (B) of the number of incorrect allele calls at the centromeres computed on the simulated data set (500 markers, 400 spores). The x-axis corresponds to the fraction of the missing values in the data set. Each line (and the color gradient) corresponds to the noise level, ranging from 0 to 25%.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">We also observed that a smaller number of markers makes the estimate worse. This is because with fewer markers, some chromosomes either lack flanking markers or their flanking markers are very far apart, making predicting centromere alleles difficult. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> shows that as the distance between flanking markers increases, the number of incorrect allele calls tends to increase.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distance between the flanking markers vs. the fraction of incorrect allele calls for each centromere computed on the simulated data set (500 markers, 400 spores, no noise, and the fraction of the missing values ranging from 0 to 25%)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distance between the flanking markers &lt;em&gt;vs.&lt;/em&gt; the fraction of incorrect allele calls for each centromere computed on the simulated data set (500 markers, 400 spores, no noise, and the fraction of the missing values ranging from 0 to 25%).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F8.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/7/2071/F8.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-63" class="first-child">The distance between the flanking markers <em>vs.</em> the fraction of incorrect allele calls for each centromere computed on the simulated data set (500 markers, 400 spores, no noise, and the fraction of the missing values ranging from 0 to 25%).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>Similarity coefficients:</h4><p id="p-64">We now look at the similarity coefficients. Since the centromere allele estimates become less precise when too few markers are selected, it is expected that the centromere similarity coefficients will also not perform well. <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> shows that the range of similarity scores narrows when fewer markers are used and the coefficient distribution of spores from the same tetrads starts to overlap that of spores from different tetrads, making it harder to cluster the spores. Note that, on the one hand, a value around 0.2 is optimal in the case of 500 markers, but on the other hand, it is not even nearly optimal in the case of 100 markers.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of centromere similarity coefficients between pairs of spores from the same tetrad (in red) and from different tetrads (in blue). Panel (A) shows the similarity coefficients computed from the simulated data set consisting of 400 spores and 100 randomly selected markers, whereas 500 markers were selected to compute the coefficients in (B). The data set had no noise or missing values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distributions of centromere similarity coefficients between pairs of spores from the same tetrad (in red) and from different tetrads (in blue). Panel (A) shows the similarity coefficients computed from the simulated data set consisting of 400 spores and 100 randomly selected markers, whereas 500 markers were selected to compute the coefficients in (B). The data set had no noise or missing values.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F9.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/9/7/2071/F9.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493522" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-65" class="first-child">Distributions of centromere similarity coefficients between pairs of spores from the same tetrad (in red) and from different tetrads (in blue). Panel (A) shows the similarity coefficients computed from the simulated data set consisting of 400 spores and 100 randomly selected markers, whereas 500 markers were selected to compute the coefficients in (B). The data set had no noise or missing values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">A similar effect is observed when we add noise to the data set. <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a> shows that the distribution of similarity coefficients for spores from the same tetrads gradually moves from the extremes toward the middle and becomes indistinguishable from the distribution of spores from different tetrads. Note that the 0.2 threshold is still usable when the noise level is relatively low, and stops working when the noise is over 5%.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of centromere similarity coefficients between pairs of spores from the same tetrad (in red) and from different tetrads (in blue). The simulated data of 500 markers and 400 spores was generated at 4 different levels of noise, (A) 0%, (B) 5%, (C) 20%, and (D) 25%. The data sets had no missing values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distributions of centromere similarity coefficients between pairs of spores from the same tetrad (in red) and from different tetrads (in blue). The simulated data of 500 markers and 400 spores was generated at 4 different levels of noise, (A) 0%, (B) 5%, (C) 20%, and (D) 25%. The data sets had no missing values.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F10.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/9/7/2071/F10.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493566" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-67" class="first-child">Distributions of centromere similarity coefficients between pairs of spores from the same tetrad (in red) and from different tetrads (in blue). The simulated data of 500 markers and 400 spores was generated at 4 different levels of noise, (A) 0%, (B) 5%, (C) 20%, and (D) 25%. The data sets had no missing values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">On the other hand, as <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11</a> shows, missing data generally does not have a significant effect on the similarity coefficients. Moreover, this is the case even when rather few markers are considered (100 markers).</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Similar to Figure 10, but this time the simulated data were generated using 4 different levels of missing information, (A) 0%, (B) 5%, (C) 20%, and (D) 25%, and no noise was added." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Similar to Figure 10, but this time the simulated data were generated using 4 different levels of missing information, (A) 0%, (B) 5%, (C) 20%, and (D) 25%, and no noise was added.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F11.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/9/7/2071/F11.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493532" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-69" class="first-child">Similar to <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10</a>, but this time the simulated data were generated using 4 different levels of missing information, (A) 0%, (B) 5%, (C) 20%, and (D) 25%, and no noise was added.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Segregation scores:</h4><p id="p-70">For a 4-spore set, the 2:2 segregation score measures the fraction of genetic positions where two spores have allele “A” and the other two spores have allele “B”. Therefore, for a perfect tetrad the 2:2 segregation score is 1. This changes however when we add noise to the data set. If p is the probability of a genetic position having noise (probability of a flipped value), then the 2:2 segregation score of a tetrad is<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-40.gif"/></noscript></span></span>Similarly, for a 3-spore subset of a tetrad, the 2:1 segregation score decreases from 1 to: <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-41.gif"/></noscript></span></span>. Note that both 2:2 and 2:1 segregation scores are minimal when <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-42.gif"/></noscript></span></span>. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows the 2:2 and 2:1 segregation scores for true tetrads under different levels of noise.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493592/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493592/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The change of the 2:2 and 2:1 segregation values depending on the noise level</span><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section app" id="app-4"><h2 class="">Appendix D</h2><div class="section" id="sec-17"><h3>Components of the software</h3><p id="p-71">The tetrad software consists of four main components: the preprocessing steps, the heuristic search, the direct search, and the post-processing steps (see <a id="xref-fig-12-2" class="xref-fig" href="#F12">Figure 12</a>).</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detailed view of the software for tetrad detection. The flow is shown sequentially top to bottom, left to right, with the steps labeled with a letter/number. Four different components of the software are identified with different colors (see the legend on the top of the figure): the preprocessing is in green, the heuristic search is in red, the direct search is in blue, and the post-processing is in yellow. The general functions are shown in the boxes with their resulting sets of tuples shown in ovals. The thin arrows show where the tuple sets are being used as inputs to functions. Most of the general functions shown are self-explanatory. Function Search(N,Set) searches for N-tuples of spores from Set. A thin arrow entering Search(4,Set) function indicates that this is a shadow search that traverses all possible quads of spores consisting of a triplet identified by the arrow and a spore from Set. Function Global Search(N) corresponds to a search for N-tuples among all the remaining spores. Set ClusterN in the argument of a search function traverses all clusters of spores with size N = 3, 4, or 5+." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Detailed view of the software for tetrad detection. The flow is shown sequentially top to bottom, left to right, with the steps labeled with a letter/number. Four different components of the software are identified with different colors (see the legend on the top of the figure): the preprocessing is in green, the heuristic search is in red, the direct search is in blue, and the post-processing is in yellow. The general functions are shown in the boxes with their resulting sets of tuples shown in ovals. The thin arrows show where the tuple sets are being used as inputs to functions. Most of the general functions shown are self-explanatory. Function &lt;em&gt;Search(N,Set)&lt;/em&gt; searches for &lt;em&gt;N&lt;/em&gt;-tuples of spores from &lt;em&gt;Set&lt;/em&gt;. A thin arrow entering &lt;em&gt;Search(4,Set)&lt;/em&gt; function indicates that this is a shadow search that traverses all possible quads of spores consisting of a triplet identified by the arrow and a spore from &lt;em&gt;Set&lt;/em&gt;. Function &lt;em&gt;Global Search(N)&lt;/em&gt; corresponds to a search for &lt;em&gt;N&lt;/em&gt;-tuples among all the remaining spores. Set &lt;em&gt;Cluster&lt;sup&gt;N&lt;/sup&gt;&lt;/em&gt; in the argument of a search function traverses all clusters of spores with size N = 3, 4, or 5+.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F12.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12" src="https://www.g3journal.org/content/ggg/9/7/2071/F12.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493587" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12</span> <p id="p-72" class="first-child">Detailed view of the software for tetrad detection. The flow is shown sequentially top to bottom, left to right, with the steps labeled with a letter/number. Four different components of the software are identified with different colors (see the legend on the top of the figure): the preprocessing is in green, the heuristic search is in red, the direct search is in blue, and the post-processing is in yellow. The general functions are shown in the boxes with their resulting sets of tuples shown in ovals. The thin arrows show where the tuple sets are being used as inputs to functions. Most of the general functions shown are self-explanatory. Function <em>Search(N,Set)</em> searches for <em>N</em>-tuples of spores from <em>Set</em>. A thin arrow entering <em>Search(4,Set)</em> function indicates that this is a shadow search that traverses all possible quads of spores consisting of a triplet identified by the arrow and a spore from <em>Set</em>. Function <em>Global Search(N)</em> corresponds to a search for <em>N</em>-tuples among all the remaining spores. Set <em>Cluster<sup>N</sup></em> in the argument of a search function traverses all clusters of spores with size N = 3, 4, or 5+.</p><div class="sb-div caption-clear"></div></div></div><p id="p-73">The preprocessing component (steps A-C in green in <a id="xref-fig-12-3" class="xref-fig" href="#F12">Figure 12</a>) reads the data in and cleans it by removing spores with too many missing data points, while flagging any duplicate spores. At the next step, the software pre-computes the significance thresholds – the delta scores that correspond to a user-defined level of significance (<em>p-value</em>=0.05 by default set by the software parameters D4_PVALUE, D3_PVALUE, D2_PVALUE) for quads, triplets, and pairs. These thresholds correspond to the minimal delta scores required for a tuple to be considered a real tetrad or part of a real tetrad. To compute the thresholds, the software generates a set of <em>N</em> random tuples set by the parameter RANDOM_SAMPLE_SIZE (equal 10000 by default), calculates delta scores, and finds the score corresponding to the preset p-value. Finally, the preprocessing component attempts to cluster the spores based on their centromere similarity (the user may choose to switch the clustering off by setting the parameter CLUSTERING to 0). To cluster the spores, the software first estimates the recombination frequency as a function of physical distance (see <strong>Appendix G</strong>). On large datasets this step can take too long, therefore the user can skip it (by setting CEN_CALLING to 0) and use the default value of the conversion factor (COS_PER_MEGA=3.7) derived from (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Mancera <em>et al.</em> 2008</a>). Given the recombination frequency, the physical position of centromere-flanking markers and the alleles observed there (see <strong>Appendix G</strong>), the software computes similarity coefficient between all pairs of spores (see <strong>Appendix B</strong>) and constructs a graph of spores such that two spores represented by nodes are connected with an edge if their similarity coefficient is above the threshold SIMILARITY_COEFFICIENT (the default value is 0.2). Note that this threshold can be adjusted by the user for better performance (see <strong>Appendix C</strong>). Given the graph, the software detects clusters of highly connected spores using the fast algorithm for community detection in large graphs (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Clauset <em>et al.</em> 2004</a>). We used the implementation of the algorithm from the python <em>igraph</em> library (function <em>community_fastgreedy</em>).</p><p id="p-74">If the user chooses to use clustering of the spores, then the software continues by executing the heuristic search component (steps 1-13 in red in <a id="xref-fig-12-4" class="xref-fig" href="#F12">Figure 12</a>). During the heuristic search the software attempts to detect all tetrads within each cluster either by searching for the tetrads directly (step 1) or indirectly using a shadow approach by first computing all triplets (step 3) and then looking for tetrads as a combination of a triplet with another spore from the cluster (step 4). After that, the heuristic search attempts to use the remaining triplets detected within each cluster to detect the tetrads outside the clusters by combining each triplet with each spore outside a cluster – a version of a shadow search (steps 6, 9, and 12). The tetrads, detected at each step of the heuristic search, are removed from the downstream analysis, which considerably reduces the size of the search at later steps.</p><p id="p-75">Once the heuristic search is complete, the software proceeds to execute the direct search component (steps 14-22 in blue in <a id="xref-fig-12-5" class="xref-fig" href="#F12">Figure 12</a>). The search takes all the remaining spores and uses the shadow approach to first find candidate triplets (step 14) and then to detect the tetrads that consist of a triplet and another spore. Once the shadow search is complete, the software proceeds to search for tetrads exhaustively (step 17). After that, the software extracts the remaining unused triplets, consisting of three sister spores, and searches for pairs of sister spores among the remaining group.</p><p id="p-76">The last component of the software (step D in yellow in <a id="xref-fig-12-6" class="xref-fig" href="#F12">Figure 12</a>) assembles all the detected tuples, labels them, and outputs in a user-friendly format.</p></div></div><div class="section app" id="app-5"><h2 class="">Appendix E</h2><div class="section" id="sec-18"><h3>Comparison between Interaction Information and Delta at different levels</h3><p id="p-77"><a id="xref-fig-13-3" class="xref-fig" href="#F13">Figure 13</a> shows a general comparison between Interaction Information and Delta at 3-variable level as well as 4-variable level. Conversely, <a id="xref-fig-14-3" class="xref-fig" href="#F14">Figure 14</a> shows the relationship between 3-variable and 4-variable (as well as 2-variable and 3-variable) Interaction Informations and Deltas.</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of delta with interaction information for 3- and 4-spore cases. All measures were computed on the simulated data (1000 markers, 100 tetrads, 1% noise, and 5% missing data). Panel (A) shows the scatter plot of Interaction Information scores vs. delta scores computed on all possible groups of 3 spores. Panel (B) shows the scatter plot of the scores computed on all possible groups of 4 spores. Each group is colored based on the number of sister spores in a tuple. Note that only 1 million randomly sampled tuples are shown for the groups of tuples with no sister spores." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of delta with interaction information for 3- and 4-spore cases. All measures were computed on the simulated data (1000 markers, 100 tetrads, 1% noise, and 5% missing data). Panel (A) shows the scatter plot of Interaction Information scores &lt;em&gt;vs.&lt;/em&gt; delta scores computed on all possible groups of 3 spores. Panel (B) shows the scatter plot of the scores computed on all possible groups of 4 spores. Each group is colored based on the number of sister spores in a tuple. Note that only 1 million randomly sampled tuples are shown for the groups of tuples with no sister spores.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F13.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13" src="https://www.g3journal.org/content/ggg/9/7/2071/F13.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493526" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13</span> <p id="p-78" class="first-child">Comparison of delta with interaction information for 3- and 4-spore cases. All measures were computed on the simulated data (1000 markers, 100 tetrads, 1% noise, and 5% missing data). Panel (A) shows the scatter plot of Interaction Information scores <em>vs.</em> delta scores computed on all possible groups of 3 spores. Panel (B) shows the scatter plot of the scores computed on all possible groups of 4 spores. Each group is colored based on the number of sister spores in a tuple. Note that only 1 million randomly sampled tuples are shown for the groups of tuples with no sister spores.</p><div class="sb-div caption-clear"></div></div></div><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the amount of information between 2-spore and 3-spore levels (A-B) and between 3-spore and 4-spore levels (C-D) as measured by interaction information (A, C) and delta (B, D). All measures were computed on the same data as in Figure 2. Panels (A-B) show the scatter plot of scores computed on all possible groups of 3 spores and their two-spore subsets while panels (C-D) show the scores for groups of 4 spores and their 3-spore subsets. Each group is colored red if all spores of the group came from the same tetrad, blue if all spores except one came from the same tetrad, and green otherwise. The scores of the blue and red sets are plotted in their entirety, whereas for the green sets we randomly selected 1 million groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-10316313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the amount of information between 2-spore and 3-spore levels (A-B) and between 3-spore and 4-spore levels (C-D) as measured by interaction information (A, C) and delta (B, D). All measures were computed on the same data as in Figure 2. Panels (A-B) show the scatter plot of scores computed on all possible groups of 3 spores and their two-spore subsets while panels (C-D) show the scores for groups of 4 spores and their 3-spore subsets. Each group is colored red if all spores of the group came from the same tetrad, blue if all spores except one came from the same tetrad, and green otherwise. The scores of the blue and red sets are plotted in their entirety, whereas for the green sets we randomly selected 1 million groups.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 14" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2071/F14.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 14" src="https://www.g3journal.org/content/ggg/9/7/2071/F14.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2071/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493633" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 14</span> <p id="p-79" class="first-child">Comparison of the amount of information between 2-spore and 3-spore levels (A-B) and between 3-spore and 4-spore levels (C-D) as measured by interaction information (A, C) and delta (B, D). All measures were computed on the same data as in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>. Panels (A-B) show the scatter plot of scores computed on all possible groups of 3 spores and their two-spore subsets while panels (C-D) show the scores for groups of 4 spores and their 3-spore subsets. Each group is colored red if all spores of the group came from the same tetrad, blue if all spores except one came from the same tetrad, and green otherwise. The scores of the blue and red sets are plotted in their entirety, whereas for the green sets we randomly selected 1 million groups.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section app" id="app-6"><h2 class="">Appendix F</h2><div class="section" id="sec-19"><h3>Yeast data simulation</h3><p id="p-80">Error-free budding yeast tetrad genotypes were simulated in the form of a table using a custom R script (see <em>tetrad_sim_1_commented.R</em> file in the software package), with each row representing a single spore and each column a randomly generated marker position. The number of marker positions and tetrads are specified by the user. Tetrads are encoded as consecutive groups of 4 spores with the first 2 spore rows having the same centromere alleles and the second 2 having the mirror pattern. Parental alleles are encoded as “1” and “2”. In every tetrad, each chromosome experienced one randomly placed obligatory crossover, plus an average (Poisson) of 6 further randomly placed crossovers per megabase (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Mancera <em>et al.</em> 2008</a>).</p></div></div><div class="section app" id="app-7"><h2 class="">Appendix G</h2><div class="section" id="sec-20"><h3>Probability of crossovers</h3><p id="p-81">We estimated the probability that each centromere was derived from the “A” or “B” haplotypes based on the alleles observed at the markers flanking each centromere and the probability of crossovers occurring in the centromere-marker intervals. These probabilities were calculated from the physical sizes of the intervals (in basepairs) using a global estimate of the relationship between physical and genetic distance. This estimate (b) was derived by fitting Haldane’s formula with physical distance replacing genetic distance to the observed recombinant fraction between all markers within 250kb of one another. Specifically the formula<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2071/embed/mml-math-43.gif"/></noscript></span></span>where r is a recombinant fraction, p is physical distance in bp, and b is the conversion factor between genetic and physical distance, was fit using the nls (non-linear least squares) command in R.</p><p id="p-82">As an alternative to this approach, the estimate of total crossovers per meiosis from (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Mancera <em>et al.</em> 2008</a>) can be used to provide a coefficient for converting physical to genetic distances in <em>S. cerevisiae</em>.</p></div></div><div class="section app" id="app-8"><h2 class="">Appendix H</h2><div class="section" id="sec-21"><h3>Complete set of simulated tests</h3><p id="p-83">Our software was tested on a simulated test sets derived from the sets <em>Small</em>, <em>Medium</em>, and <em>Large</em> and compared the final tetrad assignments to ground truth from the simulations. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> presents the comprehensive list performed tests.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493563/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493563/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">The tetrad detection software applied to various simulated test sets using multiple different parameter settings</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-84">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8111405">https://doi.org/10.25387/g3.8111405</a>.</p></li><li class="fn" id="fn-2"><p id="p-85"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-08"><span class="received-label">Received </span>March 8, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-12"><span class="accepted-label">Accepted </span>May 12, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Sakhanenko <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.7.2071.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> The co-information lattice. Proceedings, 4th International Symposium on Independent Component Analysis and Blind Source Separation (ICA2003), Nara, Japan: <span class="cit-fpage">921</span>–<span class="cit-lpage">926</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.2" data-doi="10.1103/PhysRevE.70.066111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clauset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">M. E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Finding community structure in very large networks.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</abbr> <span class="cit-vol">70</span>: <span class="cit-elocation-id">066111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1103/PhysRevE.70.066111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2Breview.%2BE%252C%2BStatistical%252C%2Bnonlinear%252C%2Band%2Bsoft%2Bmatter%2Bphysics%26rft.stitle%253DPhys%2BRev%2BE%2BStat%2BNonlin%2BSoft%2BMatter%2BPhys%26rft.aulast%253DClauset%26rft.auinit1%253DA.%26rft.volume%253D70%26rft.issue%253D6%2BPt%2B2%26rft.spage%253D066111%26rft.epage%253D066111%26rft.atitle%253DFinding%2Bcommunity%2Bstructure%2Bin%2Bvery%2Blarge%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.70.066111%26rft_id%253Dinfo%253Apmid%252F15697438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevE.70.066111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15697438&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2071.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.3" data-doi="10.15252/msb.20145123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clément-Ziza</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsellach</span>  <span class="cit-name-given-names">F. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadakis</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhardt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural genetic variation impacts expression levels of coding, non-coding, and antisense transcripts in fission yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15252/msb.20145123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DClement-Ziza%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D764%26rft.epage%253D764%26rft.atitle%253DNatural%2Bgenetic%2Bvariation%2Bimpacts%2Bexpression%2Blevels%2Bof%2Bcoding%252C%2Bnon-coding%252C%2Band%2Bantisense%2Btranscripts%2Bin%2Bfission%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20145123%26rft_id%253Dinfo%253Apmid%252F25432776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzExLzc2NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMDcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.4" data-doi="10.1101/gr.172122.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fowler</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary diverse determinants of meiotic DNA break and recombination landscapes across the genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1650</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.172122.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.172122.114%26rft_id%253Dinfo%253Apmid%252F25024163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMC8xNjUwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIwNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.5" data-doi="10.1089/cmb.2013.0039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galas</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakhanenko</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skupin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ignac</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Describing the Complexity of Systems: Multivariable “Set Complexity” and the Information Basis of Systems Biology.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/cmb.2013.0039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BBiol.%26rft.volume%253D21%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2013.0039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2013.0039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.6" data-doi="10.3390/e21010088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galas</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakhanenko</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Symmetries among Multivariate Information Measures Explored Using Möbius Operators.</span> <abbr class="cit-jnl-abbrev">Entropy (Basel)</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/e21010088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEntropy%2B%2528Basel%2529%26rft.volume%253D21%26rft.spage%253D88%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fe21010088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/e21010088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2071.7"><div class="cit-metadata unstructured">Jakulin, A., and I. Bratko, 2004 Quantifying and visualizing attribute interactions: an approach based on entropy. Computing Research Repository <a href="http://cs.AI/0308002">cs.AI/0308002</a> v3. <a href="http://arxiv.org/abs/cs.AI/0308002">http://arxiv.org/abs/cs.AI/0308002</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.8" data-doi="10.1038/nmeth.2479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-throughput tetrad analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2479</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D671%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2479%26rft_id%253Dinfo%253Apmid%252F23666411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23666411&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2071.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.9" data-doi="10.1038/nature07135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourgon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of meiotic crossovers and non-crossovers in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMancera%26rft.auinit1%253DE.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bnon-crossovers%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07135%26rft_id%253Dinfo%253Apmid%252F18615017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18615017&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2071.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.10" data-doi="10.1007/BF02289159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGill</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">Multivariate information transmission.</span> <abbr class="cit-jnl-abbrev">Psychometrika</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02289159</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D19%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02289159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02289159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954YE14000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.7.2071.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakhanenko</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galas</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Interaction information in the discretization of quantitative phenotype data. Proceedings, 8th International Workshop on Computational Systems Biology, Zurich, Switzerland: <span class="cit-fpage">161</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.12" data-doi="10.1089/cmb.2015.0051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakhanenko</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galas</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Biological data analysis as an information theory problem: Multivariable dependence measures and the Shadows algorithm.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1024</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/cmb.2015.0051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BBiol.%26rft.volume%253D22%26rft.spage%253D1005%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2015.0051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2015.0051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.13" data-doi="10.1534/g3.117.300392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sirr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahyong</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Natural variation in SER1 and ENA6 underlie condition-specific growth defects in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300392%26rft_id%253Dinfo%253Apmid%252F29138237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjgvMS8yMzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjA3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2071.14" data-doi="10.3791/51401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">BEST: Barcode Enabled Sequencing of Tetrads.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">51401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3791/51401</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft.volume%253D87%26rft.spage%253D51401%26rft_id%253Dinfo%253Adoi%252F10.3791%252F51401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3791/51401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2071.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2061" title="A Genetic Screen Using the Drosophila melanogaster TRiP RNAi Collection To Identify Metabolic Enzymes Required for Eye Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2089" title="ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2071" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7BcOZPKrXpTJGtcRUNa2t63PCJTYO7SJzWx4tDZeILk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2071" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2071" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2071" class="active" data-icon-position="" data-hide-link-title="0">Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493673" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SoAiTKl3Wrfwo5eLn0vSLTDRhhF-etmNAvDoYnYXA2c" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2071.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2071" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Vf-h6RURJu7iEQDtPY4DaMy5kxgLwNSE24xvimlhwBY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2071" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2071.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493673" id="node493673--2" data-pisa="ggg;9/7/2071" data-pisa-master="ggg;g3.119.400166" data-apath="/ggg/9/7/2071.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nikita A. Sakhanenko</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gareth A. Cromie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span> and <span class="highwire-citation-author" data-delta="3">David J. Galas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2071-2088;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400166 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493673--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nikita A. Sakhanenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikita%2BA.%2BSakhanenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sakhanenko%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANikita%2BA.%2BSakhanenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:nsakhanenko@pnri.org" class="" data-icon-position="" data-hide-link-title="0">nsakhanenko@pnri.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gareth A. Cromie</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:nsakhanenko@pnri.org" class="" data-icon-position="" data-hide-link-title="0">nsakhanenko@pnri.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David J. Galas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BGalas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galas%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BGalas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2071" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2071&amp;title=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2071&amp;title=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2071&amp;title=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2071&amp;text=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2071&amp;title=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2071&amp;t=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2071&amp;title=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2071&amp;title=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493673/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493673" id="node493673--3" data-pisa="ggg;9/7/2071" data-pisa-master="ggg;g3.119.400166" data-apath="/ggg/9/7/2071.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Computational Inference Software for Tetrad Assembly from Randomly Arrayed Yeast Colonies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nikita A. Sakhanenko</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gareth A. Cromie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span> and <span class="highwire-citation-author" data-delta="3">David J. Galas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2071-2088;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400166 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493673--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nikita A. Sakhanenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikita%2BA.%2BSakhanenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sakhanenko%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANikita%2BA.%2BSakhanenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:nsakhanenko@pnri.org" class="" data-icon-position="" data-hide-link-title="0">nsakhanenko@pnri.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gareth A. Cromie</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:nsakhanenko@pnri.org" class="" data-icon-position="" data-hide-link-title="0">nsakhanenko@pnri.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David J. Galas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Pacific Northwest Research Institute, Seattle, WA 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BGalas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galas%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BGalas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2071&amp;rft.epage=2088&amp;rft.atitle=Computational%20Inference%20Software%20for%20Tetrad%20Assembly%20from%20Randomly%20Arrayed%20Yeast%20Colonies&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sakhanenko%2C+Nikita+A.&amp;rft.au=Cromie%2C+Gareth+A.&amp;rft.au=Dudley%2C+Aim%C3%A9e+M.&amp;rft.au=Galas%2C+David+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493673/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493673/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493673/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493673/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493673/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493673/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493673/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493673/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493673/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493673/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493673/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493673/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/933" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2071" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="odd"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix C</a></li><li class="even"><a href="#app-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix D</a></li><li class="odd"><a href="#app-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix E</a></li><li class="even"><a href="#app-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix F</a></li><li class="odd"><a href="#app-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix G</a></li><li class="even"><a href="#app-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix H</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2071.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2071.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2071.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
