<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2349" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2349.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2349.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2349.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2349.atom" />
<meta name="HW.pisa" content="ggg;9/7/2349" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth Achroia grisella" />
<meta name="DC.Identifier" content="10.1534/g3.119.400090" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Koseva et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Specific characteristics of the male Achroia grisella acoustic mating signal determine a male’s attractiveness toward females. These features are genetically variable in populations, and mapping experiments have been used to identify loci contributing to song variation, and understand the evolutionary forces acting on this important sexual trait. Here we built on this foundation and carried out QTL (Quantitative Trait Locus) mapping using &gt;1,000 recombinant individuals, genotyping this large cohort at thousands of sequence-based markers covering the entire collection of 30 A. grisella chromosomes. This dense marker set, coupled with our development of an annotated, draft genome of A. grisella , allowed us to link &gt;3,000 genome scaffolds, &gt;10,000 predicted genes, and close to 275Mb of genome sequence to chromosomes. Our QTL mapping confirmed a fraction of the QTL identified in a previous study, and additionally revealed novel loci. Collectively, QTL explained only small fractions of the phenotypic variance, suggesting many more causative factors remain below the detection threshold of our study. A surprising, and ultimately challenging feature of our study was the low level of intrachromosomal recombination present in our mapping population. This led to difficulty ordering markers along linkage groups, necessitating a chromosome-by-chromosome mapping approach, rather than true interval mapping, and precluded confident ordering/orienting of scaffolds along each chromosome. Nonetheless, our study increased the genomic resources available for the A. grisella system. Enabled by ever more powerful technologies, future investigators will be able to leverage our data to provide more detailed genetic dissection of male song variation in A. grisella ." />
<meta name="DC.Contributor" content="Boryana S. Koseva" />
<meta name="DC.Contributor" content="Jennifer L. Hackett" />
<meta name="DC.Contributor" content="Yihong Zhou" />
<meta name="DC.Contributor" content="Bethany R. Harris" />
<meta name="DC.Contributor" content="John K. Kelly" />
<meta name="DC.Contributor" content="Michael D. Greenfield" />
<meta name="DC.Contributor" content="Jennifer M. Gleason" />
<meta name="DC.Contributor" content="Stuart J. Macdonald" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth Achroia grisella" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Specific characteristics of the male &lt;i&gt;Achroia grisella&lt;/i&gt; acoustic mating signal determine a male’s attractiveness toward females. These features are genetically variable in populations, and mapping experiments have been used to identify loci contributing to song variation, and understand the evolutionary forces acting on this important sexual trait. Here we built on this foundation and carried out QTL (Quantitative Trait Locus) mapping using &amp;gt;1,000 recombinant individuals, genotyping this large cohort at thousands of sequence-based markers covering the entire collection of 30 &lt;i&gt;A. grisella&lt;/i&gt; chromosomes. This dense marker set, coupled with our development of an annotated, draft genome of &lt;i&gt;A. grisella&lt;/i&gt;, allowed us to link &amp;gt;3,000 genome scaffolds, &amp;gt;10,000 predicted genes, and close to 275Mb of genome sequence to chromosomes. Our QTL mapping confirmed a fraction of the QTL identified in a previous study, and additionally revealed novel loci. Collectively, QTL explained only small fractions of the phenotypic variance, suggesting many more causative factors remain below the detection threshold of our study. A surprising, and ultimately challenging feature of our study was the low level of intrachromosomal recombination present in our mapping population. This led to difficulty ordering markers along linkage groups, necessitating a chromosome-by-chromosome mapping approach, rather than true interval mapping, and precluded confident ordering/orienting of scaffolds along each chromosome. Nonetheless, our study increased the genomic resources available for the &lt;i&gt;A. grisella&lt;/i&gt; system. Enabled by ever more powerful technologies, future investigators will be able to leverage our data to provide more detailed genetic dissection of male song variation in &lt;i&gt;A. grisella&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2349" />
<meta name="citation_id" content="9/7/2349" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2349" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2349.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2349.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2349.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400090" />
<meta name="citation_pmid" content="31101652" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2349" />
<meta name="citation_lastpage" content="2361" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Boryana S. Koseva" />
<meta name="citation_author_institution" content="K-INBRE Bioinformatics Core, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author" content="Jennifer L. Hackett" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author" content="Yihong Zhou" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author" content="Bethany R. Harris" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author" content="John K. Kelly" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author" content="Michael D. Greenfield" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author_institution" content="Equipe Neuro-Ethologie Sensorielle, ENES/Neuro-PSI, CNRS UMR 9197, Université de Lyon/Saint-Etienne, 42023 Saint Etienne, France" />
<meta name="citation_author" content="Jennifer M. Gleason" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0544-3404" />
<meta name="citation_author" content="Stuart J. Macdonald" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045" />
<meta name="citation_author_institution" content="Center for Computational Biology, University of Kansas, Lawrence, Kansas 66047" />
<meta name="citation_author_email" content="sjmac@ku.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9421-002X" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=V. Ahola;citation_author=R. Lehtonen;citation_author=P. Somervuo;citation_author=L. Salmela;citation_author=P. Koskinen;citation_title=The Glanville fritillary genome retains an ancient karyotype and reveals selective chromosomal fusions in Lepidoptera.;citation_pages=4737;citation_volume=5;citation_year=2014;citation_pmid=25189940;citation_doi=10.1038/ncomms5737" />
<meta name="citation_reference" content="citation_journal_title=Behav. Ecol. Sociobiol.;citation_author=S. Alem;citation_author=K. Koselj;citation_author=B. Siemers;citation_author=M. D. Greenfield;citation_title=Bat predation and the evolution of leks in acoustic moths.;citation_pages=2105-2116;citation_volume=65;citation_year=2011;citation_doi=10.1007/s00265-011-1219-x" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=S. Alem;citation_author=R. Streiff;citation_author=B. Courtois;citation_author=S. Zenboudji;citation_author=D. Limousin;citation_title=Genetic architecture of sensory exploitation: QTL mapping of female and male receiver traits in an acoustic moth.;citation_pages=2581-2596;citation_volume=26;citation_year=2013;citation_pmid=24118224;citation_doi=10.1111/jeb.12252" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Andolfatto;citation_author=D. Davison;citation_author=D. Erezyilmaz;citation_author=T. T. Hu;citation_author=J. Mast;citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping.;citation_volume=21;citation_year=2011;citation_pmid=21233398;citation_doi=10.1101/gr.115402.110" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=P. L. Babb;citation_author=N. F. Lahens;citation_author=S. M. Correa-Garhwal;citation_author=D. N. Nicholson;citation_author=E. J. Kim;citation_title=The Nephila clavipes genome highlights the diversity of spider silk genes and their complex expression.;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3852" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. G. Baldwin-Brown;citation_author=S. C. Weeks;citation_author=A. D. Long;citation_title=A new standard for crustacean genomes: the highly contiguous, annotated genome assembly of the clam shrimp Eulimnadia texana reveals HOX gene order and identifies the sex chromosome.;citation_pages=143-156;citation_volume=10;citation_year=2018;citation_doi=10.1093/gbe/evx280" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=J. R. Barber;citation_author=A. Y. Kawahara;citation_title=Hawkmoths produce anti-bat ultrasound.;citation_volume=9;citation_year=2013;citation_pmid=23825084;citation_doi=10.1098/rsbl.2013.0161" />
<meta name="citation_reference" content="citation_author=W. D. Beavis;citation_pages=250-266;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. D. Beavis;citation_author=D. Grant;citation_author=M. Albertsen;citation_author=R. Fincher;citation_title=Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci.;citation_pages=141-145;citation_volume=83;citation_year=1991;citation_pmid=24202349;citation_doi=10.1007/BF00215714" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=L. S. E. Brandt;citation_author=M. D. Greenfield;citation_title=Condition-dependent traits and the capture of genetic variance in male advertisement song.;citation_pages=821-828;citation_volume=17;citation_year=2004;citation_issue=4;citation_pmid=15271081;citation_doi=10.1111/j.1420-9101.2004.00716.x" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. W. Broman;citation_author=Ś. Sen;citation_year=2009;citation_doi=10.1007/978-0-387-92125-9" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=D. C. Card;citation_author=D. R. Schield;citation_author=R. H. Adams;citation_author=A. B. Corbin;citation_author=B. W. Perry;citation_title=Phylogeographic and population genetic analyses reveal multiple species of Boa and independent origins of insular dwarfism.;citation_pages=104-116;citation_volume=102;citation_year=2016;citation_doi=10.1016/j.ympev.2016.05.034" />
<meta name="citation_reference" content="citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_year=2011;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Chakraborty;citation_author=N. W. VanKuren;citation_author=R. Zhao;citation_author=X. Zhang;citation_author=S. Kalsow;citation_title=Hidden genetic variation shapes the structure of functional elements in Drosophila.;citation_pages=20-25;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-017-0010-y" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Chikhi;citation_author=P. Medvedev;citation_title=Informed and automated k-mer size selection for genome assembly.;citation_volume=30;citation_year=2014;citation_pmid=23732276;citation_doi=10.1093/bioinformatics/btt310" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. C. Chiu;citation_author=X. Jiang;citation_author=L. Zhao;citation_author=C. A. Hamm;citation_author=J. M. Cridland;citation_title=Genome of Drosophila suzukii, the spotted wing Drosophila.;citation_volume=3;citation_year=2013;citation_pmid=24142924;citation_doi=10.1534/g3.113.008185" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. D. Collins;citation_author=Y. Jang;citation_author=K. Reinhold;citation_author=M. D. Greenfield;citation_title=Quantitative genetics of ultrasonic advertisement signalling in the Lesser Waxmoth Achroia grisella (Lepidoptera: Pyralidae).;citation_pages=644-651;citation_volume=83;citation_year=1999;citation_pmid=10651908;citation_doi=10.1038/sj.hdy.6885540" />
<meta name="citation_reference" content="citation_journal_title=Annual review of entomology;citation_journal_abbrev=Annu Rev Entomol;citation_author=W. E. Conner;citation_author=A. J. Corcoran;citation_title=Sound strategies: the 65-million-year-old battle between bats and insects.;citation_pages=21-39;citation_volume=57;citation_year=2012;citation_pmid=21888517;citation_doi=10.1146/annurev-ento-121510-133537" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.;citation_pages=1199-1207;citation_volume=141;citation_year=1995;citation_issue=3;citation_pmid=8582624" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. W. Davey;citation_author=M. Chouteau;citation_author=S. L. Barker;citation_author=L. Maroja;citation_author=S. W. Baxter;citation_title=Major improvements to the Heliconius melpomene genome assembly used to confirm 10 chromosome fusion events in 6 million years of butterfly evolution.;citation_volume=6;citation_year=2016;citation_pmid=26772750;citation_doi=10.1534/g3.115.023655" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=K. Eilbeck;citation_author=B. Moore;citation_author=C. Holt;citation_author=M. Yandell;citation_title=Quantitative measures for the management and comparison of annotated genomes.;citation_pages=67-67;citation_volume=10;citation_year=2009;citation_pmid=19236712;citation_doi=10.1186/1471-2105-10-67" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=H. Ellegren;citation_title=GENOME SEQUENCING AND POPULATION GENOMICS IN NON-MODEL ORGANISMS;citation_pages=51-63;citation_volume=29;citation_year=2014;citation_pmid=24139972;citation_doi=10.1016/j.tree.2013.09.008" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_title=Genome 10K: A proposal to obtain whole-genome sequence for 10,000 vertebrate species.;citation_volume=100;citation_year=2009;citation_pmid=19892720;citation_doi=10.1093/jhered/esp086" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. M. Gleason;citation_author=Y. Zhou;citation_author=J. L. Hackett;citation_author=B. R. Harris;citation_author=M. D. Greenfield;citation_title=Development of a genomic resource and quantitative trait loci mapping of male calling traits in the Lesser Wax Moth, Achroia grisella.;citation_pages=e0147014;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0147014" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Fla. Entomol.;citation_author=M. D. Greenfield;citation_title=Moth sex pheromones: An evolutionary perspective.;citation_pages=4-17;citation_volume=64;citation_year=1981;citation_doi=10.2307/3494597" />
<meta name="citation_reference" content="citation_journal_title=Behaviour;citation_author=M. D. Greenfield;citation_author=J. A. Coffelt;citation_title=Reproductive behaviour of the Lesser Waxmoth, Achroia grisella (Pyralidae: Galleriinae): Signalling, pair formation, male interactions, and mate guarding.;citation_pages=287-315;citation_volume=84;citation_year=1983;citation_doi=10.1163/156853983X00534" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Butterfly genome reveals promiscuous exchange of mimicry adaptations among species.;citation_pages=94-98;citation_volume=487;citation_year=2012;citation_issue=7405;citation_pmid=22722851;citation_doi=10.1038/nature11041" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_title=The i5K Initiative: Advancing arthropod genomics for knowledge, human health, agriculture, and the environment.;citation_volume=104;citation_year=2013;citation_pmid=23940263;citation_doi=10.1093/jhered/est050" />
<meta name="citation_reference" content="citation_journal_title=Anim. Behav.;citation_author=Y. Jang;citation_author=M. D. Greenfield;citation_title=Ultrasonic communication and sexual selection in wax moths: female choice based on energy and asynchrony of male signals.;citation_pages=1095-1106;citation_volume=51;citation_year=1996;citation_doi=10.1006/anbe.1996.0111" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=Y. Jang;citation_author=M. D. Greenfield;citation_title=Absolute vs. relative measurements of sexual selection: Assessing the contributions of ultrasonic signal characters to mate attraction in Lesser Wax Moths, Achroia grisella (Lepidoptera: Pyralidae).;citation_pages=1383-1393;citation_volume=52;citation_year=1998;citation_doi=10.2307/2411308" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=E. D. Jarvis;citation_title=Perspectives from the avian phylogenomics project: Questions that can be answered with sequencing all genomes of a vertebrate class.;citation_pages=45-59;citation_volume=4;citation_year=2016;citation_doi=10.1146/annurev-animal-021815-111216" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. R. Kelley;citation_author=M. C. Schatz;citation_author=S. L. Salzberg;citation_title=Quake: quality-aware detection and correction of sequencing errors.;citation_pages=R116-R116;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=21114842;citation_doi=10.1186/gb-2010-11-4-116" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=K. P. Koepfli;citation_author=B. Paten;citation_author=S. J. O’Brien;citation_title=The Genome 10K Project: A way forward.;citation_pages=57-111;citation_volume=3;citation_year=2015;citation_pmid=25689317;citation_doi=10.1146/annurev-animal-090414-014900" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. D. Kosambi;citation_title=The estimation of map distances from recombination values.;citation_pages=172-175;citation_volume=12;citation_year=1943;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=B. Koseva;citation_author=D. J. Crawford;citation_author=K. E. Brown;citation_author=M. E. Mort;citation_author=J. K. Kelly;citation_title=The genetic breakdown of sporophytic self-incompatibility in Tolpis coronopifolia (Asteraceae).;citation_pages=1256-1267;citation_volume=216;citation_year=2017;citation_doi=10.1111/nph.14759" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=G. M. Lathrop;citation_author=J. M. Lalouel;citation_author=C. Julier;citation_author=J. Ott;citation_title=Strategies for multilocus linkage analysis in humans.;citation_pages=3443-3446;citation_volume=81;citation_year=1984;citation_issue=11;citation_pmid=6587361;citation_doi=10.1073/pnas.81.11.3443" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Biology;citation_journal_abbrev=J. Exp. Biol.;citation_author=D. Limousin;citation_author=M. D. Greenfield;citation_title=Evaluation of amplitude in male song: female waxmoths respond to fortissimo notes;citation_pages=4091-4100;citation_volume=212;citation_year=2009;citation_issue=24;citation_pmid=19946089;citation_doi=10.1242/jeb.035345" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Limousin;citation_author=R. Streiff;citation_author=B. Courtois;citation_author=V. Dupuy;citation_author=S. Alem;citation_title=Genetic architecture of sexual selection: QTL mapping of male song and female receiver traits in an acoustic moth.;citation_pages=e44554-e44554;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=22957082;citation_doi=10.1371/journal.pone.0044554" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=T. F. Mackay;citation_title=THE GENETIC ARCHITECTURE OF QUANTITATIVE TRAITS;citation_pages=303-339;citation_volume=35;citation_year=2001;citation_issue=1;citation_pmid=11700286;citation_doi=10.1146/annurev.genet.35.102401.090633" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=P. J. Monnahan;citation_author=J. Colicchio;citation_author=J. K. Kelly;citation_title=A genomic selection component analysis characterizes migration‐selection balance.;citation_pages=1713-1727;citation_volume=69;citation_year=2015;citation_pmid=26082096;citation_doi=10.1111/evo.12698" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=H. Nie;citation_author=X. Yan;citation_author=Z. Huo;citation_author=L. Jiang;citation_author=P. Chen;citation_title=Construction of a high-density genetic map and quantitative trait locus mapping in the manila clam Ruditapes philippinarum.;citation_pages=229;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-00246-0" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Parra;citation_author=K. Bradnam;citation_author=I. Korf;citation_title=CEGMA: A pipeline to accurately annotate core genes in eukaryotic genomes.;citation_volume=23;citation_year=2007;citation_pmid=17332020;citation_doi=10.1093/bioinformatics/btm071" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. Parra;citation_author=K. Bradnam;citation_author=Z. Ning;citation_author=T. Keane;citation_author=I. Korf;citation_title=Assessing the gene space in draft genomes.;citation_volume=37;citation_year=2009;citation_pmid=19042974;citation_doi=10.1093/nar/gkn916" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. A. Pevzner;citation_author=H. Tang;citation_author=M. S. Waterman;citation_title=An Eulerian path approach to DNA fragment assembly;citation_pages=9748-9753;citation_volume=98;citation_year=2001;citation_issue=17;citation_pmid=11504945;citation_doi=10.1073/pnas.171285098" />
<meta name="citation_reference" content="R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Rastas;citation_author=L. Paulin;citation_author=I. Hanski;citation_author=R. Lehtonen;citation_author=P. Auvinen;citation_title=Lep-MAP: Fast and accurate linkage map construction for large SNP datasets.;citation_volume=29;citation_year=2013;citation_pmid=24078685;citation_doi=10.1093/bioinformatics/btt563" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Rastas;citation_author=F. C. F. Calboli;citation_author=B. Guo;citation_author=T. Shikano;citation_author=J. Merilä;citation_title=Construction of ultradense linkage maps with Lep-MAP2: Stickleback F2 recombinant crosses as an example.;citation_volume=8;citation_year=2016;citation_pmid=26668116;citation_doi=10.1093/gbe/evv250" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. E. Robinson;citation_author=K. J. Hackett;citation_author=M. Purcell-Miramontes;citation_author=S. J. Brown;citation_author=J. D. Evans;citation_title=Creating a Buzz About Insect Genomes;citation_pages=1386-1386;citation_volume=331;citation_year=2011;citation_issue=6023;citation_pmid=21415334;citation_doi=10.1126/science.331.6023.1386" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. L. Salzberg;citation_author=A. M. Phillippy;citation_author=A. Zimin;citation_author=D. Puiu;citation_author=T. Magoc;citation_title=GAGE: A critical evaluation of genome assemblies and assembly algorithms.;citation_volume=22;citation_year=2012;citation_pmid=22147368;citation_doi=10.1101/gr.131383.111" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=Ś. Sen;citation_author=J. M. Satagopan;citation_author=K. W. Broman;citation_author=G. A. Churchill;citation_title=R/qtlDesign: inbred line cross experimental design.;citation_pages=87-93;citation_volume=18;citation_year=2007;citation_issue=2;citation_pmid=17347894;citation_doi=10.1007/s00335-006-0090-y" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. T. Simpson;citation_author=K. Wong;citation_author=S. D. Jackman;citation_author=J. E. Schein;citation_author=S. J. Jones;citation_title=ABySS: A parallel assembler for short read sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19251739;citation_doi=10.1101/gr.089532.108" />
<meta name="citation_reference" content="Smit, A. F. A., and R. Hubley, 2008–2015 RepeatModeler Open-1.0. Repeat Masker Website: http://www.repeatmasker.org" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Entomol.;citation_author=H. G. Spangler;citation_author=M. D. Greenfield;citation_author=A. Takessian;citation_title=Ultrasonic mate calling in the Lesser Wax Moth.;citation_pages=87-95;citation_volume=9;citation_year=1984;citation_doi=10.1111/j.1365-3032.1984.tb00684.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=B. Morgenstern;citation_title=AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints;citation_pages=W465-W467;citation_volume=33;citation_year=2005;citation_issue=suppl_2;citation_pmid=15980513;citation_doi=10.1093/nar/gki458" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_author=E. Suomalainen;citation_author=L. M. Cook;citation_author=J. R. G. Turner;citation_title=Achiasmatic oogenesis in the Heliconiine butterflies.;citation_pages=302-304;citation_volume=74;citation_year=1973;citation_doi=10.1111/j.1601-5223.1973.tb01134.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H. Tang;citation_author=X. Zhang;citation_author=C. Miao;citation_author=J. Zhang;citation_author=R. Ming;citation_title=ALLMAPS: Robust scaffold ordering based on multiple maps.;citation_pages=3;citation_volume=16;citation_year=2015;citation_pmid=25583564;citation_doi=10.1186/s13059-014-0573-1" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. M. Waterhouse;citation_author=F. Tegenfeldt;citation_author=J. Li;citation_author=E. M. Zdobnov;citation_author=E. V. Kriventseva;citation_title=OrthoDB: A hierarchical catalog of animal, fungal and bacterial orthologs.;citation_volume=41;citation_year=2013;citation_pmid=23180791;citation_doi=10.1093/nar/gks1116" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Q. Xia;citation_author=Z. Zhou;citation_author=C. Lu;citation_author=D. Cheng;citation_author=F. Dai;citation_title=A Draft Sequence for the Genome of the Domesticated Silkworm (Bombyx mori);citation_pages=1937-1940;citation_volume=306;citation_year=2004;citation_issue=5703;citation_pmid=15591204;citation_doi=10.1126/science.1102210" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=Z. B. Zeng;citation_title=Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.;citation_pages=10972-10976;citation_volume=90;citation_year=1993;citation_issue=23;citation_pmid=8248199;citation_doi=10.1073/pnas.90.23.10972" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. B. Zeng;citation_title=Precision mapping of quantitative trait loci.;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013918" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Zhan;citation_author=C. Merlin;citation_author=J. L. Boore;citation_author=S. M. Reppert;citation_title=The monarch butterfly genome yields insights into long-distance migration.;citation_pages=1171-1185;citation_volume=147;citation_year=2011;citation_issue=5;citation_pmid=22118469;citation_doi=10.1016/j.cell.2011.09.052" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. Zhang;citation_author=C. Rahbek;citation_author=G. R. Graves;citation_author=F. Lei;citation_author=E. D. Jarvis;citation_title=Genomics: Bird sequencing project takes off.;citation_pages=34;citation_volume=522;citation_year=2015;citation_doi=10.1038/522034d" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Zhang;citation_author=A. Martin;citation_author=M. W. Perry;citation_author=K. R. L. van der Burg;citation_author=Y. Matsuoka;citation_title=Genetic basis of melanin pigmentation in butterfly wings.;citation_volume=205;citation_year=2017;citation_pmid=28193726;citation_doi=10.1534/genetics.116.196451" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=Y. Zhou;citation_author=H. K. Kuster;citation_author=J. S. Pettis;citation_author=R. G. Danka;citation_author=J. M. Gleason;citation_title=Reaction norm variants for male calling song in populations of Achroia grisella (Lepidoptera: Pyralidae): toward a resolution of the lek paradox.;citation_pages=1317-1334;citation_volume=62;citation_year=2008;citation_issue=6;citation_pmid=18346222;citation_doi=10.1111/j.1558-5646.2008.00371.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Zollner;citation_author=J. K. Pritchard;citation_title=Overcoming the winner's curse: estimating penetrance parameters from case-control data.;citation_pages=605-615;citation_volume=80;citation_year=2007;citation_issue=4;citation_pmid=17357068;citation_doi=10.1086/512821" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth Achroia grisella" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="Specific characteristics of the male Achroia grisella acoustic mating signal determine a male’s attractiveness toward females. These features are genetically variable in populations, and mapping experiments have been used to identify loci contributing to song variation, and understand the evolutionary forces acting on this important sexual trait. Here we built on this foundation and carried out QTL (Quantitative Trait Locus) mapping using &gt;1,000 recombinant individuals, genotyping this large cohort at thousands of sequence-based markers covering the entire collection of 30 A. grisella chromosomes. This dense marker set, coupled with our development of an annotated, draft genome of A. grisella , allowed us to link &gt;3,000 genome scaffolds, &gt;10,000 predicted genes, and close to 275Mb of genome sequence to chromosomes. Our QTL mapping confirmed a fraction of the QTL identified in a previous study, and additionally revealed novel loci. Collectively, QTL explained only small fractions of the phenotypic variance, suggesting many more causative factors remain below the detection threshold of our study. A surprising, and ultimately challenging feature of our study was the low level of intrachromosomal recombination present in our mapping population. This led to difficulty ordering markers along linkage groups, necessitating a chromosome-by-chromosome mapping approach, rather than true interval mapping, and precluded confident ordering/orienting of scaffolds along each chromosome. Nonetheless, our study increased the genomic resources available for the A. grisella system. Enabled by ever more powerful technologies, future investigators will be able to leverage our data to provide more detailed genetic dissection of male song variation in A. grisella ." />
<meta name="og-title" property="og:title" content="Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth Achroia grisella" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2349" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Specific characteristics of the male Achroia grisella acoustic mating signal determine a male’s attractiveness toward females. These features are genetically variable in populations, and mapping experiments have been used to identify loci contributing to song variation, and understand the evolutionary forces acting on this important sexual trait. Here we built on this foundation and carried out QTL (Quantitative Trait Locus) mapping using &gt;1,000 recombinant individuals, genotyping this large cohort at thousands of sequence-based markers covering the entire collection of 30 A. grisella chromosomes. This dense marker set, coupled with our development of an annotated, draft genome of A. grisella , allowed us to link &gt;3,000 genome scaffolds, &gt;10,000 predicted genes, and close to 275Mb of genome sequence to chromosomes. Our QTL mapping confirmed a fraction of the QTL identified in a previous study, and additionally revealed novel loci. Collectively, QTL explained only small fractions of the phenotypic variance, suggesting many more causative factors remain below the detection threshold of our study. A surprising, and ultimately challenging feature of our study was the low level of intrachromosomal recombination present in our mapping population. This led to difficulty ordering markers along linkage groups, necessitating a chromosome-by-chromosome mapping approach, rather than true interval mapping, and precluded confident ordering/orienting of scaffolds along each chromosome. Nonetheless, our study increased the genomic resources available for the A. grisella system. Enabled by ever more powerful technologies, future investigators will be able to leverage our data to provide more detailed genetic dissection of male song variation in A. grisella ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth Achroia grisella | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EPrw1v4c3n7cMnv3uz0ZHcSv_Ab-qh1yIPMBSsslzj4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493670","apath":"\/ggg\/9\/7\/2349.atom","pisa":"ggg;9\/7\/2349","ac":{"\/ggg\/9\/7\/2349.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2349.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2349"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2349"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2349":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mzMbuamrzDMvJsB\\\\\\\/GOUgP5FeEwtuyHDKBvTgXW1RCor+DmMz2lM\\\\\\\/k+5nSdDj+PsB7QD1P1H27VeYII22K0cYyaep\\\\\\\/8wD0w5xN2Mut3WKQpjky+zzP2OzrNYo48tUvIOnD0pGT4j7i+BAs83rVkIjzKYJPpplHZxHINFAepSc2o9gNTod6OlGdy2VOea9NVCIUDnRmJ\\\\\\\/dEpydVqjxDxBsKmSbBsYmkdPPI7yhRThP1Qw+BEM\\\\\\\/Jsd4pKxayruvZn600bOxrKvebvqgy69bZFOenQbN37faZOQImWYc58aISSmEVyr2IOw0sw7kYHxp8NRBw61E6kwH+3aDNn7tTBs+ViKbkwnT2KrUJcK55hZaL4Z+dROvlplZHTSG+qUHFMGfhrwyj0IiXXUm498VC55FPLVBDGrG9gdielqgLHik\\\\\\\/44FIhfu\\\\\\\/8idYTSjzxV9hq6Ce8Fcvmk0lU45vTza9sKPgSMX1ekrk5po7rZT9LDypcbihBP6TDaCyszpNEu64MLn2XNhDBuLw9AOdSxdjpg1YLDaSY1jqXbA+7MRYq\\\\\\\/AbkErU0jJoPg\\\\\\\/tZ3Hd1ASL9JNqX5ER9+SClxO8SjYhq1JKAzOwzs3L0Ur\\\\\\\/\\\\\\\/X2X93JSLZKgVCzaFFhFVFFQeGlXAUuRUm9dEIyfjin\\\\\\\/Lh8Zs5iLlleQBpluMZpMn2Nx6FjcsQ8MTrhx1UN5pPV0xYg3YH3Vrlhefita3bLAXpjlUAp43gKqlh0ZiqWI9Nvj683NAQxHoMLgw\\\\\\\/DB6KI4I4BGdfvTIN9mK5kh7BOSexEl6dd\\\\\\\/ytKNazwc6LeR1\\\\\\\/BbjBgrflkGyHnEfqb1mqbo\\\\\\\/2OHk0M6qKSJCNtn5jOeoj03v6rHshDrGyoq7Lsu3KIYPqH41\\\\\\\/J1y9Mz6SZL9Orw+DMGOBQUHD\\\\\\\/T\\\\\\\/Ch+U0hRHAF8JAuQr+94Xa+Loq3P6zYiPzTvqdvjVahd5jylCwmoxCaeZ4I8X9IVGHZhAOo+ffdpviHGMjApRutBZRt\\\\\\\/iGsSF+ul2BdynLSdGUyU2toWLljzXclzdfZ38TZNakNOvBWCAcyD\\\\\\\/2uqGcLPEB+BmsaPZId+9p7p4vYQ69vgYQ+UFN\\\\\\\/SxCPPtfqMIpOoaYrt1JsgR4mXh8Q5EODgO8PH0B97Gw90dqgO7BCVRiUx26c144mtSzxAjRM9uyU5o25\\\\\\\/r08hAE3xjOA3ESWv9iFEbIFhNbe6mxxzueB639f9n6i2ONYknkeuw2VlQ8H5oTsTUbSu7CikbGgeeJqrYr4ZcPBlrvFiTsr4F0RBQUoMnL2pJwU7lUs\\\u0022,\\\u0022iv\\\u0022:\\\u0022CwxiCfb+HD8tBUTVDhmEqg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002220f36a8f745fb8f051013d60c71d88aeeb25a6a555f831ab310ce89869dc6081\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jELBkJAoDMuULKQSFrL7MjNJyypnLG\\\\\\\/jH\\\\\\\/jXDgbeQfBfWn\\\\\\\/O8f5huqyqPcontGgcjnmPYiMxSZxCvS4OeF\\\\\\\/Tw+vbqTAlqigwho8X0Eyjg9Te9V9JhcdOklwWawh0dwLa8hEqMyCIcY\\\\\\\/weoK1rhlmj49ZYRaOg0WhEy0h8n1NQxE7C3Jt3\\\\\\\/7HDU5s8Zr008YVLSVuCpCKtUO5oehzJs7uJrVO8cxAvqwUQgM97UyCxIScENjzPY+TwuVwsuqN7bAi5kWfPb6T4X+3snOE+aJZYQMRdvDtrNLtbUrfSVf5wz9EG1wM9a47h\\\\\\\/yJvS\\\\\\\/7HVS7\\\\\\\/\\\\\\\/WEh6Jii4N7qjplJckfqNs63wQQB52plvy8fMRn\\\\\\\/fbz94nTtAFmjfpyQq+p7maIhjfXElOhb7vpqQOy9nanb+aKpS+2MQbo\\\\\\\/L4yLTnkTSa00Yt27OreO0+3xJxPwbmUT1t0xAh5dt\\\\\\\/+E\\\\\\\/6CZsI7zTL3tbIrCiTD5Wp3omsKagURPTyiX5snua69Yt+bLyF\\\\\\\/WXMrelybAEUMreGc7KT7JvIDMkZKI7A7\\\\\\\/rolSNGF96oj\\\\\\\/2Fy6SzyLzY8PoEnS2LspPPsVn\\\\\\\/f5AieeLSkVkvizK7Ju6mhMck6V1P4hRpeBaP9W5\\\\\\\/OKo9z4ziYpFZAH6Qdci48dc6SEiw29\\\\\\\/nN+7\\\\\\\/isQSgLbysKUDFM9ZXI9\\\\\\\/L9NhYEov794nGNsjoTWDphPA+d4VR1\\\\\\\/YcIuLLNVfiKGGlaWyjA9odK2jlmlFJfjxUpV7sgPmJvHOH7F\\\\\\\/2kJDDPZ48DME5z6u22YdsTp2TzSzIK0IxTk3pVtsnuvGYOZi6fDcbF38xeTKFXDHCWcj6Uq1+EVh1r0i681VJzciKOhnAc+ytvpxvp5iy6626oDaj0CHRYjEt+\\\\\\\/4KJjGXNzEu1FKy0FtDwPSWgmjmoUuRcrBVspPQJI+\\\\\\\/4zs55FOZGkkX59flzsaAO59aRkyf4i3DGhMGZQpODgGKTFXE0gWLapfqIN1JhSGdRwbGl4AgvExl3T2wiy9yzTdQ+vJYW9JpcVBfhly4BO7SsQNZkPKr2Mn4LAeRQv8BLn6O\\\\\\\/g\\\\\\\/QTIUX\\\\\\\/CV5R2YWiCeWakJLLzlItnKcX2rUiOg4Qdgpx8UDDdr9rDXMU6H4GHXJ4kZ2HkUOb3t44to7YUb1YEZSDaBecU+oeiQ0DKB3RbzD6Lu2nTTNohmyQFvbcsWd7BVKbIQcuG9IJxrn9+Zye5VX+dmK2vlvxXFat1lVJdJX4fLguGG1LrMXxnDnhp7mojNdiRvVoxUe5YR7DJX0v1CC6zDkLMpeWeZ5ovCio5fzhxzOk0iGQSw\\\\\\\/NmQUDQoh+FiFVVU9JOPiGxJEWmapaz\\\\\\\/rPMWkleAAj2A32UqTIAnQV+2wQseu8EhlY1fnO8B0jtjbZKGHy4qzBhoAq9+g5c\\\\\\\/vAnXFd5OoFb\\\\\\\/2VI0SuTQMRyCtS\\\\\\\/QaTkYFzTZBNaouv+0VWO3w2jVIcPb6HECHcI4X57MpLa5RttwHp9mfb5hKdkwBP+oKHCEvX24K7w24ByixWVih0\\\\\\\/OLpvGJLk7ayWO+67NNFCxFxiEz4gZji\\\\\\\/U3EhdvM3XmfsFrOyM3G3oSebQjKrw2kfqx9VZ6HnNg9BhAsUhmNhGlLdCoD452F3+E317vNZFvBTj70RZzfblFAuKdrYUIUPAqFq6OaYnwEdQfkCHMB+zW+WjyuC1opkH9PiZ64IoHWPitEm1am9CAGe0FlyNouy9Omb84Nj+husWXDoMh3OnTcKkl5jzEmFflOwa0OeZziYHdfvjWoLxAY\\\\\\\/gnJdEPtEetiJBPnomPdBGCIHudVQVxYkX3aCIO8BPjmYJpFT\\\\\\\/cpbyYqaDkAcSxjCvviJgfkkfcXMiizRtJ5YGSe9+oqzFVK5YDXRZx2gedd1QUB3XSQ7V4sOqP\\\\\\\/NDAbc7Dynm7pf4OLCe3lakf7tIOxcYHkY4klR3je6O+4ZJQgzz3kTkou4Mmu1pXqNMCkHpd8XValn+URJGIHyiRWbSh88fEVHYLGPUZ54UYlGJkvFeoZQuxMfAgtolF2qQhWJnLszgznau4CinORpnEl1qcShSACAFvwPXDiWWF50b8RTkapBRFD2xvU47aABjJxdRcS8Dfzb\\\\\\\/Tt6svFj8gM+G7\\\\\\\/X\\\\\\\/ZRq1EhxP0r55gr3PUj3FMDkufGRb4ZGOm0s9s2Zznhx4x2LfRLdXOMPe\\\\\\\/N+R4PLWb0eAa1KUpS\\\\\\\/3nnxBXImTAc62Sy\\\\\\\/iDai8k5bVL\\\\\\\/8bYD2Zd8aO+sHthNa7IZdyPzY+dTo6IYbfWG9nIwphoQImLij\\\\\\\/Fp4g8DQjKSGeRAIwGERllEK+vsuZ9NSfJL0Cy84z4EQ7pKTUTJeZgYydq3t8Jbkswr+LX6VGypCAvW56XJwg2STif5r3gjOEpFRkqy0ZyMIUUVnnUIcpXcl81AI9LL\\\\\\\/K\\\\\\\/juDq2NuoOV36YAyh2SX6a2Nj2f5Q2SZ7joxVX\\\\\\\/qIJ6uOzm29lvwXkUVbQyy0HR3EgG1Jdk+foV0u5FtCKG+Isz2sChK4i7Q8LUZumFLdkZXvlOdpSRUCcPAVITfAvb97MvIiyQrHA9\\\\\\\/Un0zym2Ft\\\\\\\/9xF4fTaj++GotQI43eOrtnAVMKxiVRRQ11WntnZb39\\\\\\\/69NR\\\\\\\/lNw95yqWWBDJyK19CEEgy6MUe4h0\\\\\\\/fONuVUludXdAYs3INCh3L4Od3D8a7S2PZsRuncItorvYoGYHFSOaKZOoEIr27qg6zt8YKQaTBNGyZIgb7dgHckxG9loznBRPefaI2Y5VEUH4xgxmdE25x7IQRTl7MctDe95djq7H9QED3Q8RhP5xJFllcrpi2Ca0J064hihApJq75ptK2ueWnk7m06Bavn6ZmdPecJV7Pmn0FI7eBe9bKUHC68jxbQe6eCCOcaplGmGhuaSVhZUKPcKEa8PedPffsMkeObqzVw1GVkareswfIARyC\\\\\\\/Oj6PzZsd3\\\\\\\/s3F\\\\\\\/KpR3+d3ZndQ0Fc4jC6dg0f6WauZFvp8WG0A3PZqYk4G1LE0Cd\\\\\\\/BmMlTfRdb\\\\\\\/tC+WgrETNAAn3hu21GWpt+c9tHSU72EXeHNFp4AdFYxnhMc\\\\\\\/1X7ywGNZ7uXEx1Hwk99sXzwrD7M3xC+ZmRPsiisi+sGe9YCQIQ6XTDkUmZhI0GoPYbuHBa1finWzMmF5RNZP\\\\\\\/+gSFl7Qczd46uYUKvxC1kYF2L3epEcdNnAIjlIc4apbZ28hlBAO9voOirItLsiAtt4dGRbMp6\\\\\\\/VsvtpmyEL22gxYl1VYs0mYQLU1VtfdRyVHsC8xbhckPFkV2DZsPy0slPZ9RYKg\\\\\\\/MyUD3ww7Fdodl\\\\\\\/hrbdFTAsrEbrouWNlCiO7lebFDUHgHpX8rZABTfI+gXgkrfR3\\\\\\\/0rgySAzdyGeNBj7CUyMGRcFYfulVNeIMBLdP\\\\\\\/ZNz7w1K7FbzzfOmq5ZfKW1jgVmEOGWlPga7I4Q6j+378MW\\\\\\\/b9o2W686JEHod3Zm2u0dWbqfPMoG5vGJCrnNpI\\\\\\\/S1aHy9luqwDZXDzAvCkzUGd8bOSjkc7g8xL485jSPgaCxQTGl1v0ZcVgciAho7gOemdNtH2vkIt50qNSdxM1NvMuFB30L6klE44hcLjvwzwvUyHq8c5ZIEuh6\\\\\\\/DKmGiH78HYyhQaq1GlUYqxLsptpHr+8LttC3HC1x8m\\\\\\\/+SQxzmOSIH3VYCZ5J7rvLLvfdcApSdq6GMAiluWgPwCKAi8QGkdyyto1Hr8PBqA+hQ7R3rUxmw+rsUYwdTEMlYZ57H7wSaUnImO\\\\\\\/oYaxlUs1vf\\\\\\\/C9JwJ0YnrmRK3EgcwsNovHMCIzNDvMQQiXEB\\\\\\\/CxB\\\\\\\/ubL\\\\\\\/ixw5MxkAi3aFXD\\\\\\\/PN4SL4KaeFopKhyyo26aUY5vmmneTmnnDzYhoaF9v\\\\\\\/04EdfNNQDpNpEuC5wV7kqRRBtCD5KSRwALZiF3R9jr+dx1y+ZJiVp52OMWYqmMKLb4LZLrnUvOElTlGHIb0p0VUryy5Jt85wD\\\\\\\/yPCrN3Q1v4BAFYOWIeYw2TBEVqssxb2+wh1CG\\\\\\\/QqQQmYQAZ6M9733vHX+NWU25XuUcQ\\\\\\\/Usc1On1h\\\\\\\/EH\\\\\\\/vqu3zKywqMx2ViNB5fr5eUdTLfwk+gVaepvmjlfL0NQv2R7MsnLwCwfzdAyJHns6snMaPn13G25ggEagKCJNoOruhQXti9BKdrlA07sVQZ5Y8XmbVQXHpVD0yqoAyrzgEJ3IQ6yC2qVvvmgD2vnyEVPUyeir9xqmzayK5lepjSOWNaHMKPE\\\\\\\/hgUK3PZ\\\\\\\/d\\\\\\\/qpfEZgvXoX5hRsL3YrSWz7vmm\\\\\\\/x+hEe1VszoDerlOhcVP3ZI1yW0+jMAuFR+KmCIFzids93x\\\\\\\/+f983PUdYKH8dmkOWLU9voEPHa0PFFNk+1RlCzQtoyJh4jKBaOvaCyamX59MQo6HgeRlYbQq4uCk3gI6hDUOLVpbnFv\\\\\\\/3Agk\\\\\\\/V0f4CJeKK1isX6dYpx64va1CqkMUwZ9g1ZLgjtORdWEEai9NtA0KXJBjdwpxwRZheJQT5iKG0PahIbQSu5I7rYkUkudpL3XxA\\\\\\\/CFmXZUnpp2nRBtHyX37yebEpzmKhwovs9TfUYfbQw1IAQXEKmXNe5hluqxFMtBxkU2F5rqkWVzt5IfxqZq7eeLYviUMGp1VfU7GktjHfw+JD09yzg1bI1X\\\\\\\/OkMGFB7i9355DyrS\\\\\\\/bG9apflyOYt9JiV7rpeclHjYJwjPA1rWNN0ZFvh\\\\\\\/koXhqwWaPCAjfq+\\\\\\\/UwpvFXonqC+fgsgqBPLj4NUNk3WESFPlBT4sxNGSDUpLBpMi2q8w4U2hW2jRx9Pg5jl\\\\\\\/1uCRZhgBKKlsc8EPYOzC0FIkXtEzjaqnYveC09AfJDlHlkLZ\\\\\\\/6paeen8sP6vYn9CDYP\\\\\\\/adixCWrzc5TgZCG+FWyeMI7uV46jALKp7iyd2o+JwKDUChIO50J3BanjL2XdfcPCe9\\\\\\\/sknfhOdQKUrol1syO1OMPIZdmPkDbxBRmnC7OJz5EhDGJ3jfcWdAccvOvHKX0\\\\\\\/zZydaw4\\\\\\\/NbVaH0CxGrR86Wk+8sBSWhNEB7r0NlDEcDqQ+2mlkRwT+Uzz0CEB5LTzgh0QlEGtSQHB7zWGE\\\\\\\/2oAQrXARJSgE6K4i8y\\\\\\\/nbAnVFjfBMTmG\\\\\\\/9UAPe\\\\\\\/1O7E0uCyfNgY\\\\\\\/iqkqtO3aKIqL3r7g3+b2vaWzPRBKVjsjDWN43RwPyWL\\\\\\\/7l2s1daGcYoUzcBaoO6nIFslYNT6EMQvL0XkbfxHYr4ybJvEH3JS4dzY+UFm9uakuyeMDUYb+tKpCFMmz3RM7Pb4A5D\\\\\\\/BA7cdO5uwFwlempS\\\\\\\/2iCVXxObfaRjvW9eHKPEgyTwC68l3xvkFi75AuEcrN5rjPEO0+R8pexqMfBBMbohpDLS8vk55Y4K4Dr7N\\\\\\\/FgEsf14tQW8Oam225RGjgxx1y3x9MMGt3JN1LetiCNLM4LT4JWR2iok65N2vzoe0CajzpzIkXYIONvLmXtnh1puc\\\\\\\/4OswJvHSIgmUox9vIVr14gXayEoQiGz9MVN9cMtcx8RPApKArItQd98yToP\\\\\\\/iAGxGdKvMEf7e0ubaO23yr78pcbLprUtPB8QXiQihKqO7hjAxK6rVR2TZyvJgOli3lJYJq9y\\\\\\\/7HBVLRfObw+6f811qsHWIKCGr7ylp9BkcYMiAkOCRhWT0C\\\\\\\/cVX9vY6cqtZZUl9HUBqfh3cwEkpdFoJwPHQ6Prj0hg3m1RwtwGvcvtosIlyYr9xyv7v75WOs1jwNc4wMp96w96XBo9ZTWxQbPYYkWJ9lbjYMLnX7P5s7UDxEXyErUJ9X5wsBG8yjDf8GQ5lMduQaGvsWCJaNClOLcbcJ+QFZJ8ecTeYQsrY\\\\\\\/uje2oLu30f51FQQxRvW77usp1xDYuaJVCQa1IMeCtLbEf0wJWCs2yfeJil5v21gOcRQ83QgmECg1A9zaR7AuAqz6+8DS1eOlWNvGs9ldci6dDUhPzmRpaxDg2pavPqr6zMV\\\\\\\/tMSMCW\\\\\\\/GXSYCrBTUO2J+EahAutfpaKq06N2MiVJqF2r\\\\\\\/m7M9qimzjy6fslMZi96KfDqFf6xMzve4uFp6sbUobSXxIg537B6PHPVr+V84aSDEHkz5PjHRRjUkH4UKgaq96QMh\\\\\\\/G9oive7m0qMxjvXtO5JmB1tV8UrePG6LszqQYL4TAWDqjVnrL51Kkq+Icc\\\\\\\/isrzEB\\\\\\\/a\\\\\\\/udwB6j334jLJAi1JgB51cpI+DdehW81UdXd02b161cxpGn4CL2+HH6bfisBGPjVWaXOqD01NjHdVvSt1ms9vbRxz6BQlaZqiofnWkxoV4m8sUG7YfW+MgUeCEU7HnTWDIRwc1HPb4AVKsQK0yQjWviyUlpv7Dg31HCnvZh+lRdceJlDNeb24K7YGPSgZ9+jqqxY\\\\\\\/3hEq7bkLoeA1346N35PuyWoIhJwhD3Oo\\\\\\\/UhLxk3oTGFA3Xq0XgKFR6JbBpFU\\\\\\\/8KKW+cp4i+HAfR9XC56Tpqmb+lHdcCrD5SX4yl0Jqd0BwXh7ZufIZjEWvI+gA4fs76KUP0ZpAn2C1cnjdPcvZlWHN4+4N6kx5VOZZPVyeeGlT7++g8lACnR3qPDAKpKZiAgAe6qzlKF6MuvJ4O+R7KEhpQX4pmtfTlIUjHT9bfHsqpMgafiyrqm58auaaUztvhMAlZo\\\\\\\/\\\\\\\/lewEpjCxleUGxkb25SzhN5QL\\\\\\\/SgOZJ5pzUt72U5jU3CvqqVhGTNj5oflv5JdCblCZcCiNBU8eqYNOaSQNiYxkX6k1n+7M9V2Gvcy6KSiK\\\\\\\/3mOLCGn8C921vmK2B8jwzHYPX89bKrMGsamRQaABQUC+TdGORUE6mpvN79hGkP7JQYl1Xjvvjzs+4WDjVmUBNfTpU\\\\\\\/uQkdfVFjso3Ro7Zk6jRcbRSt6GxPUqJ7JgGkWaybXjQYcdVbY3zg\\\\\\\/3WzFymL6SBWPBOPq2kZBMY3RYhbyr0XQqqFT2+pcrVrMZOScPhIGK+AIc5RbX1VtLq9Jd0Y5xdOOO3KP\\\\\\\/+E87XHWyfzkju29NkgNgg2kCpndmmELOxtG23NZykIb10w+b9MY5mPjfmvdQE5bcabeHnbG3OwzvS7V4rmt4kEfOP\\\\\\\/VV12rHWWSixw67fY2I5P8yFAM+Jz9YYoBrVNa5i3s7GjtPvMvGgCHSmLhtjLn+esj8PseAmEWCRd9XKEFOeqXMxneDA8rfFVia7xZLCefF1Xk4sYRwXhsSsIR6cFdq5Z4ozO\\\\\\\/B1dB7ZGSOEubWE7ImobKqwUk+YRHi6ysFHpbjPUf8TiaDSPaTnQNhY5DFm4gnHBh2bUAijMnrv\\\\\\\/9nBixEZnY+iQ+eL3f6NZ8dUH4Mgws1ioi2k71BCGeIWwY0MEXJBV7uL1cCBSjHhBVkXE2h1rzXyrBfEVxIk+32c+XL0HhyUn1NYbdnyofE41XeRCYyH6q2vR4f+LSS9fD99Pfr83r1tC1Kay0iWjoZZyMBOJ+R2bslYtQD\\\\\\\/a26+hJdsPaaOwR0uiqCuY96ke19wou6A0rGMKHW1fK4ReYqQ4HwXbMP4s\\\\\\\/J4DFiECk\\\\\\\/af4mS8wruyjihTKku3QXFFpWCoaqLqwAIm0ESSMuszOX2SR2s1HOTmAHYuf5nViuWWfChhO+nV++kyaMyZD9MqOQTCBKH2IIP9pEGSpvGq+6\\\\\\\/JmUAy6Ac9ZXqyWdqnuYWaAyiZyhHK+7KO01ctSi7LPKU8IMMEmspBJH5fBlQ5bRKicpUjBURiC\\\\\\\/933ahB0mZHyK9t\\\\\\\/jBje7uzfGXL4CaWatBO2FBlLZw4tuZxSpLIvojBQunXPOIQaqX8hgg6R+JG0JMfxc\\\\\\\/KrtCZ7N1JeL+VCZI\\\\\\\/YdSd4dRYPW0nbNokq9\\\\\\\/MHZSK6CZoYhT8MeTBxplMPHGQFPnPbW6sc196GKm67VsuMfS2ExwzVPliLV2U3sfXPp\\\\\\\/uPiHxTtFY2U4ea+zBszWQq0EiwqMfGqglOL8rgPSdeVLenSLdQRN2otBIQezQpwyCIjCY1B+wUmkIKYjgFciEU3JEZzBJ1TOVsXyuBY8bkrbpf1OUxtzVssT8RyNWPRZIKy4JLuZEEa7nSDvb06Vo85R5FQy5Ofa\\\\\\\/17u2\\\\\\\/thXKlzSHxMouWcpT4pNHMeDqLdNRVC00A7HmT8SYEnw+fOncNnyAga0W5D3D6a\\\\\\\/0XQmSojlTDUdfSqQSf8+VNV1IeUUzpYNDb3qZouhbMSkJUYD+Jb6FWzu+D44cS57K2B8Wt\\\\\\\/qCne\\\\\\\/QhWQW5I\\\\\\\/Ad9z1MGzHxYQ2wWWZ6uYRf+1FKgwkswLKrmuO6u7tguhS94heEKxoGeSJTONlvYCWVU76EN7wnNSVBKbJhPkYMi4XKRCT2FoOMsCPXw5LpsUxEqpvv55dslGAFieX6mQnvAAo+cfR+gGokroqFxiTS1fhoDVE4EczlGBsndzY\\\\\\\/9KbMYETso6ONu2jQKEgZvh66BdxzqUPxCTb0\\\\\\\/9Bx59teGLklGN0Mbwm1wTpPWe37LhXI7yvYkA\\\\\\\/Tzdyt\\\\\\\/EVI\\\\\\\/37A6Tlv5Aivs6Jq6rW8O8vjWaIlHukCo9E0RUXOARJUHLGKS574\\\\\\\/Y95hURH2q\\\\\\\/nAARD79QGDJcA90QUAouWNf59JVylTPpZKhISLuZVEav0ZbYO7lDYud91k6YH29r9S1i8xvyfOK0FhdV+KCZniqAQa26y9AOcqOPEyvNdJdbLvY0Dt5DB2uO3bpnUdqdAI0Xk2elbdGBJ+yE78e7sCJ8pu5fdZEBss+nRkRY74j9uBQkoS2r7qPOYdbEPd+jEAabcXa1+6KTTMfj2lEVJXV4nHuLjDGbSsJjvI5BkUIutQXvb7gya96MOi02oylIGFQ6fBMcIyqM8sNT6cKFDFH67lOg2alucbGuRkJhK3FmQqIg6hOB2C22U0xjKgT241GtBf0J30IJEvJttZRx9gtrgYVBvyGsA2mvZdqSJQeLKOtqoXU7XZuxfBTgbUMt76pKxl6HiqCeS8KxGgQASMyr5jxOt52cTMOhWNtLKTJHHHOAAMfUJgzXRIrJDijtTWZimjf26S4jqZB3E82JiM2ClzkUMDrcCK5fqKOvkiJG7I6dpm2wtLR50n5r\\\\\\\/O\\\\\\\/+RLFcdZ4ZNQWxeIsAANYodNbSk3DSklWU0v3uKtkliO7EXwGzPN9BrbUxmjNbTSQUOjAyhHR1pd+oP6bu7SeFExRB66gY2OBPiX7EP\\\\\\\/5czUf3kMZT2yPrigemVSZx4FK4smLGZkvh0Kp7lDFxOOqwXvD5\\\\\\\/OoQGj2VusmbTu0gVA+K55Y1+4Hnf2qS9m58IeWu+goGLpus36yXL9srxNXCFm4qkWwyY+fYksqcNP0WYlb3nzicaOvqM2I5m+KHw7K1hYSV9u7N\\\\\\\/mUl3QYj+6YadwrAdRYXHYssTMi5JP7FCjVE9a6EXf9bCPSRvZPt7nDH9xLjB+L\\\\\\\/XFXryPyRb8ztuyhpmgf1uCEaqt\\\\\\\/8XjMSzG3xvqy7huvlRmtxB30o0ZaJ0wb8C+fA6ivFtkROUH236Zm91Bxk1uwTN+ccz1wVqNzGrZmA+EHYAbxYTPoSY3\\\\\\\/9bj8tqurG1g5vVOLID8L3Gbq7Hje+a\\\\\\\/2CbeeY4k3s7n0zxYThpDf1AtYVlgX3+UWBnDlDnLP3vheULrlTELVV7s5CPEwlH3gqI4o0uwM\\\\\\\/eSmqIA11mxo1PCXZQGI+dYPWxd85woyp6q5LMwV6+iz7OyrJRPkTCPxuRwcOrSZKgZRmSc8K4IxTEXKwI7FJMM1WAro\\\\\\\/Bl4yFXgL3KiP5ChOAuIQgLfySCCCDQVgMqQYtwBo01e7MNMwez28gdRFEtG\\\\\\\/KC3Z3y3+2vtJZ6RpavQVD6h7wIhRsXsnR2iWiv74FlFCGOu3C1FSqu7BQpy6OsZNUQSCMtPJ+Ou\\\\\\\/E+cr1CMHU3I1zKY7UkvQnqVuHVbq5l8Fc3K0gwzdrC2YjUfCyliN7q9kG9eQyILgwEORwoDrut3yM8eibGJEbMqyXX3hRXugJyzG1ltScDuTipEqHa92RfxsqVg3Evzz8b2TZm0pZ2KYFb1A56bD0EMWYYzDmDL5WX4UB9EY0BFMLb2VFP7NR4fQkinOtiafU\\\\\\\/UM8PkRuxb7qcVhdCXRyTI7gs821qoGzNpVEmKDPzjJJ7\\\\\\\/FHhmcMi+i7f\\\\\\\/f3p2HVoQL5iQBOdw8hUvRkyqLsvz+LYXTiGODhue5XegGF6EMWN7RIZf2Jqe1mb\\\\\\\/vRW8KkS5DE0pArjV4i9tdIbojb+vBEI92AOy5CIVK8o0I++zW9QXStCtEIfgAXYZbayxXfQoRxrA+kfcZ8PjXJ1dqO4zUMepc58uxZWm4VL+1O\\\\\\\/P+5o64o5FnwF7LJpAtgQVcFBfwcQLrb3qCbpMHEEVbjwlX\\\\\\\/qUutkbuNad2vZCWS4\\\\\\\/an+tNlA1omYC1HcvetAo6PFOZHoN7bNerstHUZQ7BYc+d6MFL\\\\\\\/TAIyIZ9xo0BfY\\\\\\\/BtFNfTzrAdmq1xZ5yTpvnt0wepioZWWNlOCHxht6Uxm8iJwObEZnCEI5G25J0bxczjMptns6DbkRpgvukD8sBV3qAgKa7zTqeB7T3immaj0ikYFz6vpRiRB8ckL1egeFVSWCIceuKwcbvlUwPsv3XKMtLBkSxNay6a1nuIGq38ZRZoxhwLBUWN\\\\\\\/cogdwOhzgV6JR9jmcY6dMbTJr9ZJkasTIef3BmnP0X518mer30JGgUe3f11pMvCAxMVowgKmxxDiDBcUlnYNpDttSm01RTXYseGhsV\\\\\\\/FS6vSGm8hFwbsqQvmuvampCiNuxE0dnbE1DqtGNUf35fMNdbY\\\\\\\/Q3TvlRtdVxSfZ7FcYxMcxcJBzQ6KgXX95FjZHJPp7tR0U\\\\\\\/b8mKDEvPNEa54Cv9g1o39CipJzWae4VI3yJer2Yx4+VArx+gWmvCF7VugNi+mPCwQP1qfTygHKF65htagS1tSuJL8deWd+wk1e9ZsVMejQ\\\\\\\/6hxRWM9H6vLF1DWjhTYyQt0owsWVuRBmlmZqAXDaiyQvUZ5q3e0Hn3qSHVU+PNo\\\\\\\/04dlCbt3ludIyjX0JqusCdDC7LbOa2HJVCEgi46bQ27KFE2hs5US6i3qdMYJRZA+ukhw8P1Q1TZSm68DsinpzQwUJfvbgl56sHqvzA8hWWF87DNhtxrZud6pfkixjW4pWNCs62fXPZL7jkIcuutz1SFbpGLUa3zAR33umznmPzp54BWjwcfKoQ3cuN+uFr90guxx8\\\\\\\/45a7mDaSkeLpElr1kR\\\\\\\/QKRGw8BS5djN+geWE85XHluApbs8YMcdGqw4MYicO1ei9PKVNrCTcuIlOJTCZbGojnKDyK+fRAbbgvsJh4Ekc\\\\\\\/kvgEAQjB\\\\\\\/hXakU8l7ax8ExHfoW2LNWLRuhsVcUY30tva3T4h7YaQ81FdpX\\\\\\\/DHGYzI1pV4f9TpMwLO9fkeJr01CjFjg\\\\\\\/65BAvQuJMUsSPDkQIY85Ufzi98mkp1rkZqO0YQKMFJoC2caplQeVW9r6M9RZwpghEaPYQLU52nvT\\\\\\\/1rg1lNAObFa\\\\\\\/RVd0JXcI3RuuQ4O\\\\\\\/xcobb+rhx6nQBmZqbspwb4XMlovZt5h\\\\\\\/qtAugXYY\\\\\\\/jbgxlTNviWzKfOq3WMJpa3KEmfGqhLFBLuwTaaoGKtX+uI6iT1BrmQlavS149Rmr5jQXNyi8S32O9tVB\\\\\\\/Y+EXQXZ23Ww1a4vgGdRVOWqa8o0xa2pMDyDk4RlpSLzMV+MXwlP6tIqsdR76So3c1ODclmPBCuvDkFNI\\\\\\\/iFaA0qSuANoFQu8HCJ+fE5nxeiZ41y9Th98nMM0yAZ9x1ioUn2P+wh2p\\\\\\\/DWd2bxVAkIBBD4mIgDemSY+Ep1mloAupNKsBGfaPUl439OC3XiS0S8vL0E7rWnJwWkqjPl9g\\\\\\\/FPDqKeOYmBRzzJAorueirFvVPFu4X4Zs7Atar8VJ+xGiONeTKbCv7q+\\\\\\\/jc5TKUHaOOb+sizrBkKOUmYdzwy2X1t9daMlYprFGsWiePMkcllGzVAMa\\\\\\\/xjAVcnsS081DApsF\\\\\\\/cYjbpslEgHYgvD+DFGBJZBVV3UzprK0s8r4OF4af5oave7mb1BnyQYLn4mZkvicFJyQzHHTOXBGv3+RPhgPnM6b5ggydrpUpPhlvbiCr2LMb5bx7gbsvXI6FV2CKfAOoljsE3rrNvjDX0WoRIz6+tjFPMcLxJ4CxR1elGCznu6flAiZYwTRj6RQaoI5KXvkCfWDY3a9ZKHWHyk02BKpqtdbsG6cEzq+ilWwMZdG8NW\\\\\\\/rU1+vcSvPx5cajZnazfZB17YBf4cJG0UTPS\\\\\\\/ufUA6KxPxQiWI\\\\\\\/JciaS8qGnFjv6PW1i2dvD1VJFNSQen2KSBTNBzOnw9t30nFLqE2beqr9bRX3bx3W0mBGaz\\\\\\\/03klplqdz+AgbXjp+jCTbp0YcyfxQR1LzaDPrlUluC86fPEpJrdyE7X8WTVFFYJ7tzROTEluyg4x4zTSBNNE6MZFjpUPmyx9Q8wPi4UiBn1xAUiIuqPIH18xvGwa26hRg613ClrniACkfk6\\\\\\\/J4Vq\\\\\\\/lhA\\\\\\\/wXpcO7QQlKZsdYMPMZDV9X3lUR4AQFpV88cHiCQIYvZyW9K\\\\\\\/tPjCp3Ts\\\\\\\/B5nqzKkwkON5uqLt7\\\\\\\/PXlsuB62GrAobzdeNE0YSxb4Q8pT0DuQKXvKe7RWh44xUbTGs6KrWr38Q+WxNizswJ4P8eOQNn1+JyQCXdSLPtZHm5W5b22MfjUJKG4SZQvKDXpSDXKXAg1+vV6nWphtW2NabGnbFKJdAapz7YAi5dP7I3RsPKG9JsJZWka+53uw0MTmj0n2g1ciGvd6s9FUVYN1\\\\\\\/lpYQFyq8GA3A6vx3P8e93YONX1TYjfJjpbr2o83g9imOs9Ju8e5xrT2nzIgdh8Pxlc4mesg7OJavI8pL6LhkOVoqQTkdFjQXkJIAs5ZBJ5dYIUqx0V2W\\\\\\\/InPeO+WtwWuf1psmsWkfBvXl+CLyQp4tcMJEMdUtx3ISRGZTSMbd+qHd6R3pnF2mWf6HpjaVYeijVUD1tqdckx7aKPItTdMRfaC+pbo5UL8QVrbOw5N38a8s\\\\\\\/C6llHQ7ST45fJW23VTy+9ik6yQRV5HHVAsnJYIMXVW3KJDQ5tMMOjUDcdb7Cer9QUHkVfs\\\\\\\/TUouv41Wbx81qL9XQ7+iu1iwpsHO0K5aE\\\\\\\/iq16uz1eQ2aVuM9wHJFoNE+gFHup2Udrx9L8QWIvm8BxtDDrC5BXT1SeqQ+hTZYmubtaSnuZfx4Q2u3H50puzWlMN7wx96IroqlRzmMeOUks+1NpyRwBSEAt+rD+vZ69k+6KYldRgy9EI3sGjS562ctABGK2etgKFzplTaL9tOU8lGAngEagATt\\\\\\\/L4O493Tt0dbI3mTVZ9KAZJBAY6IghkYzA2sV0LueA1LkwrsqAY1dK39Sa0SuMSQJzYJHDL1Rnh5uO8PP\\\\\\\/n\\\\\\\/eeI2SY4CVovs\\\\\\\/2fLXuI2AEDegzU35WT966tIlMM4C\\\\\\\/oOvyJXlgXpYCTYUtEMIHU9eGzP4DlbfVFNqxfgUGUIBdwqNDyXU2berRni\\\\\\\/8KBVhZXEsi+b6EJyK38\\\\\\\/EPN1ZNqAGZTmRGmpD752xWZN9jSG4opPh3rs5T7eyGjSbN5YMTIU8BjrPUM0bhsMx0cD26B7eTuKsNq1rJbcVs6obsUkgTM1Eahwvr0g0yHLha64m4gkcyyo6mYcbm\\\\\\\/VoSkQRDQJGuawlfXy798uTaMFhwJIeSXZZIB+2ZdJOH7yv6N9iVjyRXbpjHBtjW\\\\\\\/f3BivofoP5+D8C1\\\\\\\/KnXAUaqlD6R9A7IVjwQF2VWrkrzAlhSwKGLH+6P2HdYTsOWksMgBiHoY3bdahMqKhTx9l69G6EfKm6zIWXECUbu57iec\\\\\\\/zAb03nI6i1G0IUaB3hWkIW71roD2LT77O5to5EaShFFC3LlslVJWhC7HPDXweKmrbSHxhJR5xbGIBeW\\\\\\\/Nr4QIvtIT+c0302FUD6e3w18rVyBvQxnmGnWw5G2kwiiw8fJtYQrzZB4JJvQquzRw0Z2Md7ucExM0ocjOn98VjSLosj4MLU5eLydBbaEEQ4fU0OfBiYiKQKxVFabl5SIkWmrMI+97w\\\\\\\/QFISTMy87RF2MAI4b8CpaUdKSLYpFzdx8O\\\\\\\/kZmGHbLeNoP09Hgt2A8mdKtmqwwl6rjrEqxjXansd5C4sah3DnyjE6UC2LVbtTX4DtVhKFiQ6wyF38nYGWFWHy04Refah9lU58JGfPa3AZECxeEn1RxWWAZehadf2d+QT3EHmqmbxGixxlkoSVW+jcPatDTh+ls52ucQ9kX0FK2ZY6mx7HfhyZgxNjm10OkftTbg6fC8yy5OKVkORPs\\\\\\\/edFjkzU3GFbeCV4vkWaHR6rfFaSBydVnsMlbq2AYqODiOmsAjgA0Qf0nZ4tPTxb32gEEPhUYCsw6Ikz7dlHS2L4dPVlpMw7KmlYprO+vNk5VTuBnMYhSD5UAEel5ieuoe1Z10LwP0XUaLCmiF3RMpKuLP8g4+nOzP5LldesyiqHABnycduQDaeCSU+JluOB\\\\\\\/AgVLh8nhN\\\\\\\/dZ7jhvDEFnQSQaKRUu8Jo0Qp2D6rMlEUQTeKRc2H5gmOTFStENEg9tW3BqewVtbtechd450tfbk7hMRyr9FkAljMXGHOHn50peOCnEidFXmk\\\\\\\/mK+Q+zlhFZpltAykxdqPkswp2BMQfxaIahVRtSQT9b1ojuze+xR0qh\\\\\\\/NGw8brwbiTSshHG5WMN\\\\\\\/vMH6ZXc5DZ7u2uw+j0clLuxUXbOUWQrqtuPvmhAY2KPiLe3Z96hMrJ8xUyzZ8Xg8towE\\\\\\\/1Le1z4H0KsplU5go31T7L5Chk0pSRPgDYbzF1U74EKKJJprQtceeV3b1eyk\\\\\\\/HWqCpO5fERRlmmhFgo7uiKcrw4TqLJUsyfvF2KV6oEjTtF8cRviXR6pky6d6Alz\\\\\\\/7ZvfYBUaisWNBVKk3YXpaGOsW5XHegZLh8YU1mMSlZZBx44LaEQM+vFx0Y4bOj3v620rpeBR9kghn\\\\\\\/itR7eQNF+KYmN+LVwXjQJ0Bb2lIJEv31oaH\\\\\\\/giErONT+buENxAMmvBHIN2KFXtlYElReW\\\\\\\/r4jaCeUpWdSQjDAECMAkQiidbHIbqXsqbPFJizdAFDKh6VwHAv0qFwbT168vZcA5MbHgUHbQr9011i9GlmQ\\\\\\\/XGaoryZslJqLHSFvYjScT3se+I5J6YXsizhGa1n+rqKZ7hAuSSbl0b329krY0V7vCieFMbZtUHLZVF1B1FQWVltTpTV8GJKloAdBBPkrdkzQba7xG89HjtMD3NBTPK2U+zw\\\\\\\/Cya5y4V9aJANo5w\\\\\\\/cLewtk8d1tB6G4TRApWrh\\\\\\\/AXqeC0RtuAuim952qNPV62e9++rz4RkBBh02QHZLLCcKJ+cX8MRBIGn\\\\\\\/pX1qf4MS2GhFHonOfpBZHeIrMJGosr\\\\\\\/Jy6RXTMY\\\\\\\/OPc0L8O11sAGKvjH9KRyB4E9u5NUoDaTlh5s8Rov+2BhTUT+sDPT3mZ7GFirq7U+r3O9PdKZRJEgpXXgdaixMGAhldyi0oi0ee\\\\\\\/Wv1PS4CfVcMef4OZoD2kjYB5+bVE6M\\\\\\\/f+apELuFCMPWNCU68LTcr+hh75OoZRJzXxL3tVAhHKKp8hWWImNR9g+FbzzNa0Q1ZgCV0I11WwKdySFrLDmoizNk5CKRpT1ij+6b4\\\\\\\/jH5Ds87MfqbXQk3xQACXHBPAzGL7JholnqieVSiRL+JSSGGh3NNR1jcb3JjYGoqZsSjvCtHrPZMfk\\\\\\\/mBFVCdiPmJGv0Em6EYHWDJUNNe32w3xfmHOhNaVIsydi2hN1FF5EMKqE++1SQgQnOTgEYAhXJIk9Q\\\\\\\/UKhPwZKQw7JFmd488Luz6Euw3AM4fR67Ov6246oKotieJ\\\\\\\/rwBtM5sfEeY0xk8NiGATqcge39IHs0H60k31QQfltMpDCk5Jc86OfkKTiWsrIxcCUZsymea6P+o716NwWNm2O2eYNmlMbQbr9nhyGif8FWJvI8PQBJIiD8ftzS4KKBzK7JndFEUQzZBVNj6xFrNhXajyHb3GMd8ATzbnrmu6MLt4+TQqnOnomA4QS4QcvTFsEr8LU+6aZNgjvmzSGGv8LxJmtMn4wG0G153NR1rj1A5tEFzVD4rzEdHzD4gCXQ+nRvMEGhT+ZVLhPeWcIf8bNXq0l+HY7Cj117DKkcKv1\\\\\\\/+0egQdjR4YDgt4Gj34P3m1JQ3Ep5LYIXBByS4MMQLEPfkcaXaqJqPHFQUbW03hYmnuMjUe+X3s9pHPyzN2ZmvDyHfiPB8oEWfEt2mvgPyfEuKosFaRv3I+8JqQo5hmSs1ujAs0cUNDrxuxf6NkJeJAENhIpKY5xNHRa7A2UXBiVin9\\\\\\\/HlrwEVXxGG3u3tcYiEmAKbIETE3uGfJG\\\\\\\/BJ0ZmOnmCGA7namWNPHLgHrvfZlEXJCUyZQ62CJdCrbbitBD9dIh0ddvxRuQGaHfQsNH1N+w5JGz7yt0OTyN+KRKwG2yd+F7VboY9qCwsJYL2J0NPPW1A9UHH\\\\\\\/H6WfRsupibSGh1ccPESPIL1L6+X+x4DymNkmnH2qpiRILVs8A\\\\\\\/axNkDpwLTlOHgEcLiTlJTMnSV6NK\\\\\\\/0cqJUoktNDmEQ52TyYWVIEgL2HD2D7eagmA0IdUsfKBQH4GsQU\\\\\\\/tghtL2SxiVBd\\\\\\\/CrSd2YqiWqsqAIMg4uinDz6kgIIUSFaGSGLSbRUDXA4pPRBIlyb0KAD1JMrkxIoXZVxTZqH3gEoiMabZURZgt5xuNT3AxWqidEYHilGNmKTD+fwqC2n2xwmJrAcqdcy7aDndXEyCp20MorshX3tJ\\\\\\\/1dLXQG8lPrI\\\\\\\/+UXe2Kzonw0V3UxCBV8IrreTvddZEBEZ2Or4bpmQLMW+epvbzD0b0akWXHrqB5SLujw8\\\\\\\/NsJV4N0z5xGgf0+m+6CofkI+2ub2QOnT\\\\\\\/TnVixV\\\\\\\/ecyd+Bq2zElmyAVz+484S\\\\\\\/AZF90B1xwuxxnfkPJTlyhh01EpNmwAp2lSVYj1f0tKc9W4ZDGMCZ1JVFJ7lIIY8skrdymH2bUjiDm6JRJj4QvNBGMPUzUkvooEM57ojT788MZ3EkH6++lWewt594JyFjVDVZWx+dP54n\\\\\\\/uO0TGxmc8CqKbgGqK8o61q0pWnqFGoA2Qt97AHmscS3RRtHncZsNvs3+8UKhU2daBklColjgg4J11C4J0xFvFOa5jSGKo005KrhpuDgqyHS7XYSWG\\\\\\\/6XWanfXY0fEw+rvpSiiJ8t6b\\\\\\\/l8WDP4cX4LXPRTyDX+NV1Ht+BLKNAIvoFXcSOPLLo53bm\\\\\\\/qbZFrP8rIuOD+LTCRvtLgeQW1ESEmm1TytApO1m7Qte\\\\\\\/CuL5RODY+2dZn28\\\\\\\/dAJbsK9l+OO4NwRnrz01bqnBUAdU939gd0BXOa2VlY8YEMqL8E12KdNzunZyfVKqzveUNvRXfzi9Q9IgUF0F2PFN7Alld3LTU6nFxm8ra1+Zu9IuPdq\\\\\\\/XXNMqUj2Bbpt1FBwgZpGaSU7hBUFlFWw9ObQoXUSU9UCEDytKtgYx5S+F7ubtkD4dNz2WgXS4KM7HqGIu7lPGe7WtzeLq9xI2Rn1tiQATnFXjdg3SbqivBu7oBGn5ppogKg\\\\\\\/fHlg3rqk86YrtAnuS+cYJ3OF9m2f6AqYpaw0vAZruiUbLPv511n3E3VW6SS+w\\\\\\\/4NNWZyO0f+C\\\\\\\/v7E3zujFg8UzOwpmk8zz89PqxiXcdXVaOCrkbe92INj9YemFPQ3NSeYUleXyyKJZ7\\\\\\\/4cdkkDodhnwvPBg8MK3\\\\\\\/NoZVoBHl6H4L1syZjPm87yk0e5OWumBRFpC22SG5ZNfRkjOdc8CGKR8+PTKuugJijKcfaKe165oszpo+m1HAoy0gf5z6QMQBEao637kLRry3Snzdb8PENojxIDc16xtyn5SN4lMqD7h8\\\\\\\/Dzkp3Fp6M+DSmSTzZ64WIZ64n7+7MtAXTnNzgDhZapdhQhlJR\\\\\\\/ApwS1DTdm70+yFeMJKJ8cNklXOO5CFy7P34+r0w5droJCMEmZZUp1cv+5LmIyQ3ygaX\\\\\\\/F4\\\\\\\/DoaYyfnAosgYGsQ6\\\\\\\/DIGTYQUlEB4FsHFuHqT9vyzgOM35SnBh4bl2iRTit2ZmxR1RnjD9wDndqi7LpoVLb80zXWy9KcBykjpaS+\\\\\\\/milNQcVMHYQeae0lkeHnxFuNTm9JtLcXXZ4nL4QQljx8+OVmsc6wAN+qqYSzYTVqU9kbu+TAXjpji+S4XHk8rnEQaC7hu7HMihwTi0umZztSSh8vQj4X1kHOvNQH\\\\\\\/ovvd0iykUJD93FggMkPWp47LvJ6hnc\\\\\\\/79N9dxNn6GeAGaZYp+sTAiBLgBR\\\\\\\/5upaCkkRM60tyvfoLGcV\\\\\\\/kyOfygVVZVjX1EIZWx\\\\\\\/sGVYM\\\\\\\/QPNqEnNBXNyBWamOLEuuTigAmi7c\\\\\\\/3nUw2\\\\\\\/p82uqd8ePCBzFP\\\\\\\/HLaL\\\\\\\/WwvGdQrmq0JxaA1fcRgI+pmRCdwre4xMNbl6TFonnm97pcMAKAbgT7SgFeAiE95oEpkuXq9kV1ltamh1rlfOMwFi5GrcNZL5wBPlipEk8IBTecUr\\\\\\\/hNzNS6lLiLp\\\\\\\/GZvtZrQHIoB2y6y6TMU2AJjGx+PV6KSz2ZYj4QxUwtPu+Mk7Myz8sd\\\\\\\/m3XI2KX6ut39uxf+gJVsYjT+BCArNgGF0AQIxTM5m2W\\\\\\\/IbhDd12nLBg3Whcz9MPqAGtb++XncoU+RrTkXY3gQ\\\\\\\/gPQ6iKoqV8oLfzIlO4qIpSY8CPVjeVCOq2Lxr3K+BjoWcw52dixGNC8WnDK6U+bxg\\\\\\\/5vzkdce5xw\\\\\\\/xKj8HbRg4kIw7IUZ4v4E\\\\\\\/WP8fKfHQXay5e1zCtAY2vQdcnwT+Ows8\\\\\\\/SbL88A3a1VyC5Ad1gJB8\\\\\\\/af7667QbmObsV4DMArdxvR3bmF4U+wQDE4ix0kySryrzVgwYFcohcpCKGJPUeBElAAauMjSWxuHp8IIZPO1sDjoWYnwwtEbrfcyn4SAMsLEk7AG47rqZ4oJtLT46wKfyKQPa6mZ+Tqa3e\\\\\\\/pg7Oi82zkSwPG2m7E\\\\\\\/\\\\\\\/Uffx64UI1rlnh0qppe8w70SarHpwAU+uiUIH5ffpry45OFKwq7q5Y8CQYG\\\\\\\/ZiP+yDmAnxIQwyJdeJG+9cT5rWPzyJgTEkZ\\\\\\\/acjcUrd\\\\\\\/3yS6y2Q0e9OL+26A6lP1k6vNRbsPQFAmfyk1Y6t+SOrXBmFkUYE\\\\\\\/kuFJRC8U\\\\\\\/NBeUiP1GTJCZS0vej1IMKtjXHN6Hkk+2FxheIiduEz6rSJsOUCChRFe2gnJem4zvAjdFqAL5Czpjhe\\\\\\\/9fHlpudScdECdeKmN45jb5J0GD7v3rFpKQsSW9Me88FrEtki8Kq4Yh3u2SxjmEqE5fWg8lJygogmNqQmN9M1l\\\\\\\/F\\\\\\\/69BNYU0zBhkAm6reaE9RIs6mwJ0vYRnkUY3HJJaA2Wo+mOu0GQolg2bka5fQLP1VUuqL3WOVFfcBvBmmoIaLFuVOscPQgvmmkNnfjYRlBWHXHlqUIY6rgBlzQXTPaiHtD1A7IUDAydVIiJkHwexpV9mKBWC+WkI+U\\\\\\\/YleL190qtu3jOmDVkKGxKk6INZO+nZPXBiTxM1\\\\\\\/xS5cCp26RPGn3MlnbBxlyJeyYpS6t8j6YygK6gWYGF9HTXVnOtC6XWQ9yHtPcZrkUvQfqCEr1pg4SE7RWIhKTwzV6D6yO5PsZVOU\\\\\\\/2JQk6tWFYSduhOQCdCI5uMbyRzWwcK7NeFq46\\\\\\\/GVmAds7zwwK9YItUVbperifS6ytaz2uV0gUo4y0RRYClLgIoVYRTiXU9rF1N8GtSzPGA1RBCAWYPxwpM24ZcGrZDFFDh10e+maIp7oNsJT0wrWW3K2xu39ALmMjl\\\\\\\/GhqhrbQo9A3Fc1K7QAZZgcipZVZZ3Rvrz\\\\\\\/hSml+kbvPO6FyFF8tXZcSbuGul8DfxaqelClMISmxgucCUxggV725LpLglHvcFSxO6EvRvfCssaLDLV8c3IY8nRs5zhwRjnLJMNBt29HgqYFsM19zFt\\\\\\\/XBe0bZokGuucyKKTxvVWCSRQ6QqK3rDQlW\\\\\\\/IhtbT\\\\\\\/PMfKDue5z6xXVlKkZT35zCaNWENN9nQhS9uo\\\\\\\/6JCZhLk+KzLK60yCCd1rDZ\\\\\\\/MYpPkvo3yj+WFORHctMTtz7CcpPBk08KDAtG0zGdUgCAfxRquKnPMU\\\\\\\/8tYSSwuWMi2a3fzNOqbEoTnq+wuM+PqHapsZ98m1KmOEFalnwrM0g\\\\\\\/+7JSjYmyaVALP9vjPddwosLr+29PI8O2Xcip+Gy3zmJJwlL31u1RnBoeO1ETVLTofffbPAUbitCGryqFKtgeXX6VVimMXqdnzIH7bBZiMF3rw6KChsjfjW4\\\\\\\/b4aMMBXogb93yybBi7AmS6m0KCzurCZcDgccZH3Kqv2TUNNagRTdjvJOAfKYuzG1YLctWEoysZH0Iwbo+yrZIdmNVKj6\\\\\\\/Ku2dUM86fsrTaHtil7M\\\\\\\/nYsEyjQmvQ\\\\\\\/lfABSYOfrYxDmYsL9t+6ihYjvmOq5aWXkkE0M+Ag3hUiXwcD0JWOre2r0\\\\\\\/EFzFwVHbEb1\\\\\\\/hsBV6PhEq5hsXfOS\\\\\\\/k9uGG8smfV68JpqK7rc5o0fKqHDGDcHpK4flgdS0lxheVMwbhPkczzqEIO++XNIAdOkMS38jMbPbhNwofnHkPxfVqNE8HUF2RQDUoxejeRzh40wSxQ2NauuEcT6WH4DEnpLZ4Sy4GSKwnd17Xzg+8dbJ60WVkrJe5Tzq\\\\\\\/Uw2D71yXJsZnS6S5\\\\\\\/St+Ul65Sf9tv64BgH2sOtXmfwrzz0Ew2Yh\\\\\\\/U6e0ze+OYpJxLM7feRRdNBYhJxI4Hibg7WD8tQ1SnZ2B\\\\\\\/EoplMrvzsdJWAktGqoy3viNaq1wAoa73UwqN0nVL1WZGV7GlROiC0phgGWHqRXRjwaqkMrSATKF8gwttW+bje0fXQFu4Fan7SnrkP8TR5WqoLdmDSRrePViV9WVH\\\\\\\/dZlWn3mFdBJroj7OVpx\\\\\\\/r\\\\\\\/udhxQxQpf73Tf0jAF8dvEUHeHdqavnB8pnfUc+P7DVXFC3YWsM7u2w6hWIIMnWopHtuzxOM+aotH44Z7hIy7B4rVi\\\\\\\/7P4L\\\\\\\/tG\\\\\\\/mQpXjLLoHtJ0mGGRg\\\\\\\/0+zZzNGkYbvUjXXU2i24fWee0wFo5wzh9VKq3ov6JJ0B8kig1Tim3qo\\\\\\\/J88wyuadKH3A5CyAAtW1lpFvSagsHy1hEY7vDPTbl8EIivWU7A4fabp5rQ8w4gAn4wYtT9gczT5VlXiDLrEAiUJehRqQ5SnCm0mM7Ps5Te1N3WfVELOrUZQb3Ba5fzeI4RZVO1d2LXsOmYTiJOjTlxPPzuNPPRPszvUnHPi6wb2sVf+tJxxKVK2QbL+khfX0QMXYjsyOkILnTVXH7j9ZMIR4pE9H6o4mspRNXWY1GvO0VJW4V4OND1J73L56gyNH9E3ZXLDcJmdkfsTAF4hSAzAGWHjzVXvMrwp8C03PJnWrxtlhqs2g084PhdQPJumI4rf4L2gwx2xcxBkHl4r88DqPv15GfloanknyHClUQNum9umuw6oReSRDLnbZWVkharkfxPaDLBd1LyMtQdbHKwxcWb1C4E2QPItpHe2n1QrkoNX3amy\\\\\\\/xEdnJt6K2nglPySZYC\\\\\\\/VXWGUy44IOHsW3oOysp8HsG3o3SBRVl0cDzHJS9a8mwlHt\\\\\\\/jpsRjzW37+e202hBLO3qKrMp2j1cgXPOMNsBw8c584RGeO9fBPotDTuyPBJiM1rKWIRipDAaIyG1Bsb5UaficB05W7i1WtcMK+6ntWad3FBfbZ7vH8mhN8KYrl9pWh4qH0A9bamvQ4khRe7huQcLhcOA5qJa2gkIK3iC2QVkJfLT2CewlfozfYuCsFs7XRVjVJK+RhSd8I0KF7n2hOH3ZBcQzUDGirdt9shVs1J+G93zmTNJ112Hky0\\\\\\\/sFW\\\\\\\/3xWb\\\\\\\/Lydzf2gd2P2X9RXouU3BQR+aw62jvdfP1WnVwKUHwvcNVu6tivjVUVyEBGMlcCMFvv6EY5z\\\\\\\/9LHOPqdDr8Z3603R0ht9EYjR4JuI6c0GY80uAPQMinEC6QjWxbG4oHSUfriHL6o2yGwx17Zb\\\\\\\/rG9Wt\\\\\\\/fojMaHRCAdeiNaPFGZlHWgMkWvYzbBH6WjtH9\\\\\\\/QSCOXQ8NxXk4snP48SSO4WzTHOZQvVLaNcGRhvKxqjFqE\\\\\\\/f7HJD0Zsw7f+LruL5f1KIumb65XEuwAX9ioEA0FIiqOr2FIWj8IZezFuV3vM\\\\\\\/TeGXjO\\\\\\\/OpqFuyBPJUw3UDIFq1CGmj0BZFHyYw\\\\\\\/WdpRQi+ak0d7G1LtYwVu7ZkxZ8iKpJnY3pXkvmr1j3RhkP6i9sHSnLMPg6Gyte5z8fg6Px\\\\\\\/G29XQwfxWEHjkI0pQ0PNuyUJDmtb8PvXBKDDCFyWGmQKPFbn4m6gtCQmI6VegPXBVVeqrU7KpuTSfT\\\\\\\/L2Xi1RYoppoWABv1gcjKh6YqvQAiy7iQ+diAqhhme5tJEa3ToXjyYavQpk1zhg1WEJ6CmMQZEXfR4LEuAA7m\\\\\\\/PueYYvkpu3XjSWZDopk7LzrsUdY7wNSN7bmiVZlchEimIgGilHGsgHvaXm0FFQZJrhMYG6Et0iO0TZkAUR4EeYrzHTh7hL1trrHP5BdTcZzwk66KMdPkjnxJSaTLf4KYInOM40uJ8Rjxg+BAXCX5khSOaEIJKFjBkcD9mGK8hC1vVe03nYkGMqAblcLPJx5+QljAc8VZa\\\\\\\/fpRzng6rU\\\\\\\/V+h05NntEBy9eZvK6fbXK7\\\\\\\/X0DhJ\\\\\\\/xk7MA99Rtw6dYt5ezhH6Fl2UXW2PXcWBHabWc2HAFN8uylUc6+F1PIynTs1rfhiTubxi7qBTtmAD7y9AIdyBwOITcIjRfmdef5rsMdlqRoLVRWHtDYFXzLh2uKnS4Crx8xalsaRcN0h2IYglbvt3IyZ8+C5eBoqLidOafRbdVbUlZq8bxJ3K6gS6pN6MYYMiGkS7ioRMfiy1rZWTJckoKSzB1XKLygvNSmBexIuSRXnFF3TBLy7zmSNEiE8hamQRaPHcFjU5knD3p\\\\\\\/6hjJPCSqVHSzEE67zW1kCxjgTcRJc+j9ykt6lQ+dF3s+7\\\\\\\/\\\\\\\/pFmFI5R14olr4bgIsqiskQvyFp0GqDjw\\\\\\\/8vmLaXEYRVRwPIxxvt4VqnhWQHq1Hn8K2rqYXgiM+GKbGoMeeq+asUBuP+lWpmaT5KYmLWr8XzhENS0p\\\\\\\/Hw2a5F7PBbVvTl6BtfnEql6ET3QyBGtcHP9De9ZiAkuC8X6vjbFBwCqCXwKQ5fIBrzeieoaqxQcmoiPARXI7\\\\\\\/iYbAqve6lX+mGd0PsbQA7rrhiNm4muvW4TBDpTLQoci1LkLHsrmXBoVh\\\\\\\/TTFfKEQ5S\\\\\\\/Q6Dtb22InMo2C1CPrXaTL4LDp+tbVoJ2N6Y+mldXgB\\\\\\\/32FiXIz2EoLmmu1so9sKmMk\\\\\\\/v2kFpJUeFowPlb6Qo3d2Y1zMV0YO2rwQpBohcJLB91KZtQSB\\\\\\\/UAtqvZvogyn5tpFjJahMm2WO0Uqgv2NQ76uJ2xSKnYfTpPd+WCcYB303OBuTW59Yc2xKMpKI16W6v3vhwapIs+BIIfO9OtJeC5aySByGoDxdVlhQjYV+UrHfV3PWvSB39cERk09RLSCNuKSAHeFA7kNgCUMNIDstr4o4cED6gnSWNl7HCqcX7fx6dWsVKr2dZaCShN8mSJbuxA6no5i16+5ZcNfflXnQeuTqVQZxfWPlmr3ERSFj4ECy0ludGZOZPayRNMCmyHuWPjpmR9vJt1VDJjAjcAbMslREKpGhh3pZsd\\\\\\\/oEElZb8Ta8Y4F\\\\\\\/lDh5LCwWC\\\\\\\/KSP1oXQEObhDEPMltNTHMPTPicUpRSnP+9FDoyoqruCiiWMxcgtA7IiG9q3QwDn9FVbtgsL5Z\\\\\\\/8fIKfxrMHww3lrivcrotXMw4gBMKaL1bqaC5jLQryhuIWHgwHOPuII+uS0hsv0Dl57+tcBq8mXzVj\\\\\\\/jcnihY2Wzjr6FadCmNlDF\\\\\\\/WYu9SjY58v3VB5i5TIr\\\\\\\/m0AdPWeOHvJYhmKzQnBS6ozf1oI7cVwrEU4ZovI43zKbMbjhLqn5AEy1bwP82qX5Gx0OvA0z8ejSTB5tBucrJVMOu\\\\\\\/GWR2DhXSL8ODvidprif4jzpLX3C7\\\\\\\/bt\\\\\\\/V9qcxOdlTMg53k1zzuySyVuDqSlDjvb7brcfKIGwqSRlSazjIf5owi7wlMnQC5xiVB5A5d\\\\\\\/zj0Qhe6qhwJtp40XGxJHg5FEIfroyWDZ\\\\\\\/VuLUSPb9JchTF1gLoEI7Ofn0qvQfGan+FhPqgPooM2KSnOUFZ3hRu+nSyJl55vlAwZx8tPPTtokQd9SAlJ1NsfZXuBzygyYYVRN3+rKyzRsaL\\\\\\\/+x47MgW2LsaHlxvTsmwurHFXbpDSnbgCBPYpBpygolmetqPiq4+8q\\\\\\\/yYoCRJa9pHzHQuMZG2Flz1ZDAuRzBkNaeErElu0oHM0SPzoMqV+q995GCvO2iqYIDzsF2hmWrC22o5d0dvw7bEzYuygBAtA3PRqMzioKerrMyRWUy0w7tH\\\\\\\/Ir+6aU6b3SHH7hISF4UEkQDYnkc8BvYHAf4PeGOI98qv4PESFKdBECJf\\\\\\\/uSiGFjIskxrkDbXCaH4bSTIji1\\\\\\\/WsadPhyYOSgFum43Ko9G9HrXPPN1Hcyleku\\\\\\\/96GxabI5P\\\\\\\/qilr6zOwNutx\\\\\\\/c4lUqKuQOfsJ3e0qMnKmPj0IQ4y3xP\\\\\\\/xZD4ctwYcErYgWrxXaPJZSmx9FvDuaGfsXzDHx3KF9qph1Fggef+Nyn0Yna7NrZGN6XyUs3NpKY\\\\\\\/X57JSM233LVZ195mFNT0LGMwFOAe\\\\\\\/zE+yaFzixOd5VnsAp6K6etYjQ2hGGSXXK2UmD4iH05xSRGBe5T7hUAdaSK4veLF7xnG\\\\\\\/Lkvk5jOtqcrRBDO6oZlr2oBUJLNmf9WFw+ZfQW4G6nCuFPWHcBC8otrjgYqWjViT5pFq\\\\\\\/LNk0McOEIJeDFsXMtK1plDn\\\\\\\/jyEhxVWx8F\\\\\\\/zF9K+HPmcB\\\\\\\/5XpGT4C8wpLwQ\\\\\\\/oDfvdbF1h7o9jX84Ush7i2rholGfa9j7gQOKBxWDau8BHeWXq9ue2mL1r7GqshmIVzbPbcWOrmqjDUARUFkyOkpSdfINpzZnivgqQgwR46BnFqZ8ciudc8lpJ5Q6rvuu\\\\\\\/el5r0zztVq9jY7b1kXWsYa61uXNxB5VdqTu9LvER+hfswz1f36XFywdn5YZKG0YAMhcFDywMFXCYCSRv4hUnbfYz9mPrNf26H77ulR0uDhuIxlXt7TeDcaHwx40RhF66+ouMYfB9kbZ0xKC\\\\\\\/Zo5zgdkjdqo6MNhiUvcBwEU1BV1CkI9X\\\\\\\/AWDMSqrrhZJ2xC2V9J76xncxG8u5UgbCCUcGV8EaCKC\\\\\\\/NJTO8bh\\\\\\\/1ic7ZrsIyK8XMwqoWHqRCIhS9ewfOPsafTJEWLUeUFfSKkHM3Zkc7pEHoAfUapgQWS4s3RboDob\\\\\\\/hGTE32Pjpue9g1DbWk\\\\\\\/H5VejMeQ7pQmNbj0wUY8olQ1YbI8MMOkuAj0tLjxKJh063FRRf+Hopgxb3KdRiTZ5JSLRDEPaH7RQy8F6URJEJNqmlYVYMdYrW9kHQGliLl4Y0q4lV1rMc2bcEPzZbsB7xgyoDhwFeqqXLUli6LVNMDMKfQPkL9MnPRyIvBVOKy9wpF659J0ZHk4jHqkdgx3iZlPnWCPhvlIbsVW34nInrxakFNkhwjaUWoCVsEWVDvImB2Gjgw+SmrgP+m\\\\\\\/D5vqw6LU2+T+7zWbtrdQk84P1maa2bTVUZEpuseGNgZvvPh2zyzCai+kImQepLUQCl86SKmn21ZYXSxlysnjCFHiEnozhNZ4AEhUji5Z8kJZhZToa+kWxhhWgBUG2y2lzK8YS9OqbJhElOzQmuShRXrZo4FqtPHjjO4xBB0AspsrQWdsmiuiYqNT879dHywvcgoWiS45wghSvadq96HZ8NHT5rKWGeiU9wTWkj1+tOumXmgyxmIXXeeVFJWm+s+oANK0WrF8txpaknuUaU9OLnnPpfKA4RSGHIv7UlnmG524tbHzacHi\\\\\\\/829ic4r6qSaNCzA6jYU5fT4j+wgCCCPE69M5xY87i2BreWnQVJZurBqT6yQdXoBNO00\\\\\\\/Nf+TBpJGChTgRaJ0L7FGe0VB\\\\\\\/gJmyU\\\\\\\/hJXOgvxao\\\\\\\/NaylpNZFHzCsJmAiBhTKMK7rsaSzGlSC\\\\\\\/B2umxt\\\\\\\/Tg86TYLSCQplm1hkBGkvTu\\\\\\\/qJQi5AKtyrmI9pOCGcYT9+khv0np3O875CUog5EF2ZW7tZZlqhO\\\\\\\/j3Kwv7RMsWRQ0ILiWLDflzXtiR4SWlwXzpD0qhBoKYLnc29HW5fXoGI48yyJpeb9ThSS9qhb6HWboVQOjnnZz+7fIAh+zwVgfwVafMdV4t7EQ2seoJ0yQoOGGs3a+VXo0WvHOIs5zNTzI\\\\\\\/k1PHaxbFnXun+qarbEvTbWkDBAkpeGsfV\\\\\\\/LKc3EHzYNibpm5GEk5EArzZ6ew2NjJu5XytfOfi8ivsj7Lv\\\\\\\/TwnjHwIKItyNlFIvHPsXHQBtxZJuOc\\\\\\\/KAjAvAajFD8MxNC3DQXvLGfjRQTjEAM9bei\\\\\\\/Xby6mfroBBbq4MK2nmP7N1q2jNOcLwqFrSUnuBIZqwcuWAz+6VbiDbUkWPiEtea7WdPsWH\\\\\\\/h4TMWk3f5La4GSfalkrBtrZu3ijbM1IBtspE+zDYg9P6lHZyuDfx+jFgwR22gFz+eLHHWAtGi1WROtAcrsEuj47ddqdpFObpjl+iKsz\\\\\\\/tbFnbyzCbLRcCQUhbSxCbNegQO52za7t4dVvgF0J7nNlYFaGRfaSkqDlRwTdgdsT8+fxYf4GIG4XP5Mtj\\\\\\\/DkrFn8ZRqecaekjHjnxPKqE9Exi9UvNhW\\\\\\\/1+31M1na9jQLbWfJl0eITxNvMbj6tX8wGtH9GYbNJZPCsQolLTSfPmFXn2emRzaD5V\\\\\\\/Y94aMKAwpt2f5QYthP\\\\\\\/ZIl4+RrYZEfcYW+DxOZuXUv0x2OFA259ZiNg\\\\\\\/nLOXxF3GM5GUlYzXFlQl7IKvB5RpMyvXTYwapmZFFw0Fn6ObaPdOvoAFBB5pYsWROEjp8vJotpc9M9TGlBcQmG0loushwrduzHJ3ti6dOmgihm\\\\\\\/4kr8jRdgFycQDvYEr0rKfEfb6w8YiZ0Gbr1DthTbTJWypuB27mohMqpFHsqxhC9KUBu87qd3igr+oi9oU6eocGBO\\\\\\\/VlPynYIv4Mhj9lhaxynCde5sfMr6iUjvmCRmY2M2PCVzaejB3e1xVeHinFmZZFTbFduQwWQ1\\\\\\\/jU1yAXIUJr2nCwyAnBuOcDM1WAziOMnyGpcIG6itdjbWlkTn9\\\\\\\/zlslQDxDAQYxZoBiNTrnR0QKYMwZTnJPoslbEh\\\\\\\/8C0oGTyXCHI8sTS9gAa3F38Wgkps9RrkIeI3x1g+mXnOIReF7mrjIpWKOY8HpB\\\\\\\/UbNMSKjejpDXS3Q6WAUkCSu4hBu4emJPVvqAq8fhh5sYAdDZPBOno1q8vyb1BsUL8lSiwXmx15DVIZ1lo\\\\\\\/17MLKoYvh3TK5soTv3ONeIFBeI5tbhu38DX7kVWoiW5MOSaAGBWcE9Eyah9xH3aFzQwciz1LC1K9gwx+ZqE3J56xD5QBr\\\\\\\/+aELv02APiHtGtzV9it5TlLDoBatDyLmLCzdWLnbb3vg8uTmJr+s\\\\\\\/YoZz+63lOYBs7kibRC5f4P8CVXJXPQkupU3EXCNfY2tr6Xu+YKo7hilVpbYD\\\\\\\/rWH8YM8CTwoDXtdmIOIASfIahZxb39IjP0IWg+Woe+BkC71Fzw1xe0G3GhINwwO65NIflg5vXi6p3Lh2BJxM52YtwHRxVLtQ3xnpchOvX5cqpWkZ9Ac4Md98JNwoocpPcbb75r4yBh40qzhkGsiLiKUou7hJSY\\\\\\\/KN8oY4q3VCKrDY5BoiIruV20lcgr+a+btO2A2AzXbGcsYL1avfA9ckti\\\\\\\/9lLrKVx8v823afyO6Pa8oTd+51pmwbHvQxMQ+wib8O+6aj6MdiImd8zuUJ0RyDwg6x3sfbTK+OI5DroZHs+wKtwV+v8g+sWU\\\\\\\/j897XoZf22a+m2rBpoggkZ8AOWeSaON2CbuGOQhDH02BRmskC+oeb4XMeQXvG7mFXfciHH5BSJH46oFVReDTkJhhp3T0ZC\\\\\\\/YN1IKRwqTcetA3d4cW2tOSmWnALNHepwiFht6Sxh\\\\\\\/M\\\\\\\/p3VPFYpWAHxuYP5UhBa2jqG4BHL1Nf\\\\\\\/M5u2X4OadA6JUbrQGEdO0TFoqFWkoWPOQZNg+Bc42gGUP1MFKc4rywakPD\\\\\\\/5XZxJmVCrm1gGiDyN0PR0UdfVwRpcaSeVaVU9LMzO54W6+xUPgKh0yo93wPgHxZPm+xX2D4HwFQxBFqMfpT\\\\\\\/51xPH2kbQd5la9sf15Bjjl7FnmBKF+lrrRWGDW1Lrf8O+8f70\\\\\\\/hqZ1Ex0ZeQYPZbjygOGBRfmf4Qc0tpNTLJhrkDPvJtJGuTNI9g4nzg7l\\\\\\\/ZxryxZbXbBNBLfNrtkmJdv8o9tw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XfCTNAsA+3AxTMmz3jVK3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225c4a984cfdb53524439f19c3d9edea4fc8d9bf37bdbc20db2553c8bdca332311\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R0e0O+97rDzA7C4mEe5zC0Fy\\\\\\\/BFnW4+ASrn9QAVaAVhInovwdOh\\\\\\\/+ZvfQVQ4IPPzeZyVtJhY5ZXQrk8aKI8GJio8jGgvxu297sUKWaemrqHXEnuP6NlDwtIFvgXqt8AgY+dhIMbp8z+UGivNW1Zwj76k1L1psCH3EZuj5mrltS+9tXVHAN+DLSWRLpKr+yOSvKqhTPe9GmqoPw3P3PBzu5HkbZx0xyLMtd\\\\\\\/6ulSl2287WVd0e7UYt\\\\\\\/hnH3H8wRSjM6ujej04VIwYclssLXxRMu8R03igMu8F+bdzETIwXD1O41rr5DLba33DHlNLP7dlkI4mK+ojG1ghQn\\\\\\\/LnLOYxv7P0TSWnXWMzLFvbcTIHbVdXKs\\\\\\\/4x+jg\\\\\\\/iKWsVpwlB9GiDD51kCkBHXtDPe\\\\\\\/dqpjOwHtY0pkzuvnTX19++nYZ7sOajVvYR2myFKTWNbtt9qLWiemFuu\\\\\\\/E6j4c4AUxMAtyzm9GHnZv4lEv39mkFf33VGyCYSfIwGW+P5WGYcky0K\\\\\\\/hGFU09KjhDW6u0wPhA3wL17FWvmAWmbX39ldu5wVYfejx79Kjx4G4dzauwfE+FuABH56I8wXxZm0Nm6Hmz+1Inr\\\\\\\/at58O7FCipcDV2ULcu6kZF8SrkLa8XadCK39G\\\\\\\/Eai7g2iCuQ+6F574hnDwqmaVEC05Uh5rInnWnfDgumGMiawKDT5fJ9eQju4sFq4xOLtn9HDFifdCN3mfgnq0uEBHv7+e9ChOgGKjLRTvPWynU5INlQGbBGw4FVpctEHsuxn8bNvqVbwTxYFaflYkzsSoUGgHvMAP92U45gAhBsEr61Z63XF86r+FL2k8B4a9BUoGAX+S\\\\\\\/T7JlgVD4KlrS79Fg\\\\\\\/RCBDJ5nYeWiMY51sCAgxSXtt4jBXiqaZ44SPETq1cENhCpeO8vbBtbH5M+04skAHPqj0DKTLG7EI+yBjvnm6Wp8+L2DaGxWYS22h8wm2moMPMft7vDIbx9+pSKN9+J1s67neHX13faEjdlAKQrR3fdCZ\\\\\\\/d4uXI1FtaRuCF+F7V6XC9THZiqYxNoa63jE8ydyTIc5LsuF77Nj25n73UkPxGuWg3k151MCQLzxMq9a9rPTGAyRlxGvyaOYmY3ZD2Jjrp7dMwbYLE9PPh1ayBytZnxJmB6KeQfbNdwDhJi\\\\\\\/2v1tf9KK+EJQpvzP+mYyU+Ngjyt1tm3mGBiAOnsCGtudopSq2T5XscGVfdQ1jQkZVmsycIh1RNs4TxTyHrTw3DOAufZbVG\\\\\\\/UZShc5WG\\\\\\\/dRBJiDkdUCvshbX+gKk4vEft74DCSHXhy1lpU\\\\\\\/2BFuH9jkKgvCYr+3aaEQnDpcOVhy\\\\\\\/Wt6DQeMDaEGeVe0o5Amj1Lv92Q4U98JMoO14YdYIGYhHeqN2JbphrPHIxswb2urCDY7WdrDwERYlrBbobzgqCEggbCgsomvzw+sQAZuxK+3NRivWXAmuESAzR7EORt+rFO2ulv20OfZ6GUB0FhyftoTzj+Yz8toSqR5\\\\\\\/l8y+SIjCL+VQKngg\\\\\\\/cjYzzTq+LwFBet2WNnvyq57kmNjreukQ\\\\\\\/GcyWciNrWYDdyi\\\\\\\/raO3hdivYHLieHlX3QUYNCg3+eNyn6yCKAVIQfFsiSjviOYOCtIVPVzNx5ZBOKYmq1kb0oKFSHgh4YxoC0flPnU2lVjWBde82Z4rOZ0NQTgyuHlaC1V8UJFWXVfpsi8O5WcXa9HoVcMBJ3eIQYwhUP69CEDJQ6tqEJ\\\\\\\/Xi80YW\\\\\\\/fB2v+zdEE5QDq2C5\\\\\\\/LLFIoLK\\\\\\\/BmNwS6vowxzqQ+miLO85y0vQ19kT1x+ZS3kjzVQfG\\\\\\\/KV9ISYX63mPXzXdpvVjFXC\\\\\\\/vZ3VSRRphPvvrpoX+VyZLkrbV0siafvdYpGSR0GK9XiJA2Xh7tpR5K4sgzY6jMYV\\\\\\\/MrPiSF0umryClCW8Iz5qk6jlcXifUZFbXSRm7P6ugqzolkl49UTSDDw7uyjb\\\\\\\/xiuptFM7vmT0YrqGl+JCJhPFQ5diKgbM9AE7xL2t6yPz2dAsamrxgarXAP7JteGGGBA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vGOFcmlybIFtW8+WylXJpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022faf62c91a69c9831a0b5fe344cb3f7bf3a0ddcaacc252e6b36cb9c20d2f18686\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493670 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2349" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4317neZy7BNIhkCAbSsYLm10QvD4V8FMo5_rYQettWQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2349" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gxFoTrrVwXTh7Qi7-L8MrRNJcwvtXaUfaoXNTSHxmxI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2337" title="Global Shifts in Gene Expression Profiles Accompanied with Environmental Changes in Cnidarian-Dinoflagellate Endosymbiosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2363" title="Differentially and Co-expressed Genes in Embryo, Germ-Line and Somatic Tissues of Tribolium castaneum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493670" id="node493670" data-pisa="ggg;9/7/2349" data-pisa-master="ggg;g3.119.400090" data-apath="/ggg/9/7/2349.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth <em>Achroia grisella</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Boryana S. Koseva</span>, <span class="highwire-citation-author" data-delta="1">Jennifer L. Hackett</span>, <span class="highwire-citation-author" data-delta="2">Yihong Zhou</span>, <span class="highwire-citation-author" data-delta="3">Bethany R. Harris</span>, <span class="highwire-citation-author" data-delta="4">John K. Kelly</span>, <span class="highwire-citation-author" data-delta="5">Michael D. Greenfield</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0544-3404" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jennifer M. Gleason</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9421-002X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stuart J. Macdonald</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2349-2361;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400090 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493670" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Boryana S. Koseva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>K-INBRE Bioinformatics Core, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boryana%2BS.%2BKoseva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koseva%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoryana%2BS.%2BKoseva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jennifer L. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BL.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BL.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yihong Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yihong%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYihong%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bethany R. Harris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bethany%2BR.%2BHarris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harris%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABethany%2BR.%2BHarris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John K. Kelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BK.%2BKelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelly%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BK.%2BKelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael D. Greenfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Equipe Neuro-Ethologie Sensorielle, ENES/Neuro-PSI, CNRS UMR 9197, Université de Lyon/Saint-Etienne, 42023 Saint Etienne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BGreenfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenfield%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BGreenfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jennifer M. Gleason </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BM.%2BGleason%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gleason%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BM.%2BGleason%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0544-3404" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jennifer M. Gleason</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Stuart J. Macdonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computational Biology, University of Kansas, Lawrence, Kansas 66047</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stuart%2BJ.%2BMacdonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macdonald%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStuart%2BJ.%2BMacdonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9421-002X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stuart J. Macdonald</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sjmac@ku.edu" class="" data-icon-position="" data-hide-link-title="0">sjmac@ku.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2349" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493670" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493670/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2349.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493670" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493670/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2349.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493670" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493670/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2349.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493670" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493670/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Specific characteristics of the male <em>Achroia grisella</em> acoustic mating signal determine a male’s attractiveness toward females. These features are genetically variable in populations, and mapping experiments have been used to identify loci contributing to song variation, and understand the evolutionary forces acting on this important sexual trait. Here we built on this foundation and carried out QTL (Quantitative Trait Locus) mapping using &gt;1,000 recombinant individuals, genotyping this large cohort at thousands of sequence-based markers covering the entire collection of 30 <em>A. grisella</em> chromosomes. This dense marker set, coupled with our development of an annotated, draft genome of <em>A. grisella</em>, allowed us to link &gt;3,000 genome scaffolds, &gt;10,000 predicted genes, and close to 275Mb of genome sequence to chromosomes. Our QTL mapping confirmed a fraction of the QTL identified in a previous study, and additionally revealed novel loci. Collectively, QTL explained only small fractions of the phenotypic variance, suggesting many more causative factors remain below the detection threshold of our study. A surprising, and ultimately challenging feature of our study was the low level of intrachromosomal recombination present in our mapping population. This led to difficulty ordering markers along linkage groups, necessitating a chromosome-by-chromosome mapping approach, rather than true interval mapping, and precluded confident ordering/orienting of scaffolds along each chromosome. Nonetheless, our study increased the genomic resources available for the <em>A. grisella</em> system. Enabled by ever more powerful technologies, future investigators will be able to leverage our data to provide more detailed genetic dissection of male song variation in <em>A. grisella</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/male-song" class="hw-term hw-article-keyword hw-article-keyword-male-song" rel="nofollow">male song</a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li></ul><p id="p-3">The advent of relatively inexpensive, high-throughput sequencing technologies has paved the way for sophisticated genetic, genomic, and evolutionary analysis in organisms outside of humans and the traditional model systems (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ellegren 2014</a>). Indeed, such approaches have enabled the assembly and annotation of genomes for an increasingly large number of invertebrates, such as the non-model Drosophilid <em>Drosophila suzukii</em>, an invasive pest species (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Chiu <em>et al.</em> 2013</a>), the clam shrimp <em>Eulimnadia texana</em>, which has a unique sex-determination system (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Baldwin-Brown <em>et al.</em> 2018</a>), and the orb-weaving spider <em>Nephila clavipes</em>, which generates several different silks with diverse properties (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Babb <em>et al.</em> 2017</a>). In concert, several major community efforts are underway to sequence thousands of invertebrate and vertebrate species (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Genome 10K Community of Scientists 2009</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">i5K Consortium 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Koepfli <em>et al.</em> 2015</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zhang <em>et al.</em> 2015</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jarvis 2016</a>). Sequencing technologies are also being used to characterize genomewide variation among individuals/strains via genotyping-by-sequencing (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andolfatto <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Elshire <em>et al.</em> 2011</a>), and such data allow a range of population genomic and phylogenomic questions to be addressed. For instance, these approaches have been used to examine genomic signatures of local adaptation during invasion to new habitats in the monkeyflower <em>Mimulus guttatus</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Monnahan <em>et al.</em> 2015</a>), and to understand the diversification of snake species in the <em>Boa</em> complex (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Card <em>et al.</em> 2016</a>). Genotyping-by-sequencing has also been successful in building genetic linkage maps (<em>e.g.</em>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Koseva <em>et al.</em> 2017</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nie <em>et al.</em> 2017</a>) improving on maps generated using anonymous AFLP markers or small numbers of sequence-based variants (<em>e.g.</em>, microsatellites). Merging genome assemblies with linkage maps constructed using large numbers of sequence-based markers can facilitate the dissection of complex traits in any organism.</p><p id="p-4">We leverage advances in sequencing to explore genetic variation for a complex reproductive behavior exhibited by the lesser wax moth, <em>Achroia grisella</em> (Lepidoptera: Pyralidae), a honeybee symbiont. In most moth species females attract males by emitting long-distance advertisement pheromones (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Greenfield 1981</a>), and in some species the male produces a courtship sound when he arrives in the vicinity of the female. In contrast, <em>A. grisella</em> males remains stationary and attract females over a long distance largely via an intense ultrasonic advertisement song (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Greenfield and Coffelt 1983</a>). While intense sound production is found in various moth species, such sounds are normally produced by both sexes, during flight, and in the context of interacting with, and deterring predatory bats (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Conner and Corcoran 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Barber and Kawahara 2013</a>). Although <em>A. grisella</em> males and females do hear and avoid echolocating bats (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alem <em>et al.</em> 2011</a>), no evidence indicates that the male <em>A. grisella</em> courtship song functions as a bat deterrent.</p><p id="p-5">Male <em>A. grisella</em> generate their sounds using a pair of organs called tymbals located at the base of each forewing (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Spangler <em>et al.</em> 1984</a>). Fanning of the wings causes the tymbals to buckle both during the upward and downward wing strokes, in each case generating a pulse of high-frequency sound (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Spangler <em>et al.</em> 1984</a>). Despite an additional pheromonal release by the male, playback experiments using a loudspeaker have demonstrated that the male call alone is sufficient to attract female <em>A. grisella</em> (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Spangler <em>et al.</em> 1984</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jang and Greenfield 1996</a>). Male songs can vary in the loudness, or amplitude of the acoustic signal, in the rate with which the song is produced - termed the pulse pair rate, and in the time between the paired pulses - termed the asynchrony interval (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jang and Greenfield 1996</a>). A series of playback experiments employing synthetic songs have shown that females prefer signals that are louder (<em>i.e.</em>, have higher amplitude), that are delivered at a faster pulse pair rate, and that include longer intervals between the pairs of pulses (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Jang and Greenfield 1996</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">1998</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Limousin and Greenfield 2009</a>).</p><p id="p-6">Each song trait exhibits significant additive genetic variance (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Collins <em>et al.</em> 1999</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brandt and Greenfield 2004</a>), and three previous studies have genetically dissected this variation, mapped QTL (Quantitative Trait Loci), and gained insight into the selective forces acting on male song (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Limousin <em>et al.</em> 2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alem <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> 2016</a>). However, the limited genetic toolbox of <em>A. grisella</em> hindered these efforts. While all three prior mapping studies generated a genetic linkage map on which to place mapped QTL, in two cases (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Limousin <em>et al.</em> 2012</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Alem <em>et al.</em> 2013</a>) the maps were constructed with a genomewide set of AFLP markers, and in the other case (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> 2016</a>) the map is based on a modest number of EST-based markers. Thus, many QTL for male song traits are mapped to anonymous locations, and cannot be compared across studies, and some true, causative loci may have been missed due to a failure to tag all chromosomes. In addition, without a genome reference underlying the genetic map, one can only speculate about the genes involved.</p><p id="p-7">In this work we expand upon the mapping study reported by <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>, examining a larger pool of phenotyped animals derived from the same mapping cross. We employ a genomewide set of markers derived from genotyping-by-sequencing, and create a sequence-based genetic linkage map. We also build a draft, <em>de novo</em> assembly of the <em>A. grisella</em> genome with a set of robust gene annotations, and tie the annotated genome scaffolds to the linkage map. Finally, we identify additional QTL not identified in the original study by <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>, and provide intriguing evidence supporting a very low crossover rate in the mapping population of <em>A. grisella</em> used in our study.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>QTL mapping populations</h3><p id="p-8">Mapping populations were derived by intercrossing inbred strains derived from individuals collected in Kansas (KS) and in Florida (FL), as described in <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. KS strain females were crossed to FL strain males to generate F<sub>1</sub> progeny, and F<sub>1</sub> animals were backcrossed individually to either the KS or FL strain to generate a series of experimental families. The males of these families were phenotyped and genotyped for QTL mapping. Because <em>A. grisella</em> females show no germline meiotic crossing over (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Suomalainen <em>et al.</em> 1973</a>), to generate recombinant individuals, F<sub>1</sub> males were backcrossed to KS females (hereafter “Kansas Backcross” or “KS-BC”) or to FL females (hereafter “Florida Backcross” or “FL-BC”). In addition, F<sub>1</sub> females were crossed to KS males to produce a population of segregant individuals (hereafter “Kansas Segregants” or “KS-SG”).</p></div><div id="sec-3" class="subsection"><h3>Phenotypes</h3><p id="p-9">All experimental backcross and segregant males were phenotyped for three song traits that influence male attractiveness to females (pulse-pair rate, asynchrony interval, and peak amplitude), along with development time and body weight. Pulse-pair rate is the rate (in msec) at which the male song pulses are emitted. Pulses are normally produced in left-right pairs, and asynchrony interval is the time (in μsec) between these paired pulses. Peak amplitude is the highest sound pressure level recorded during a pulse. Development time is the number of days between oviposition and adult eclosion, and body weight (in mg) is measured immediately following eclosion. Full details of the phenotyping is described by <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. Recombinant males were reared and phenotyped across 2 years, with around half of the experimental families being produced in 2007 and half in 2008. All segregants were phenotyped in a single year.</p></div><div id="sec-4" class="subsection"><h3>Genotyping-by-sequencing library construction</h3><p id="p-10">We isolated DNA from 17 KS individuals, 14 FL individuals, 5 F<sub>1</sub> offspring, 447 KS-BC recombinants, 465 FL-BC recombinants, and 198 KS-SG segregants. The DNA was then used to generate a multiplexed genotyping-by-sequencing library using a modified MSG protocol (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Andolfatto <em>et al.</em> 2011</a>). We made two modifications. First, we employed the restriction enzyme <em>AseI</em>, rather than <em>MseI</em>, as the former was expected to cut less frequently, and lead to higher read counts at marker sites. Second, we used a set of 48, 6-mer in-line barcodes (corresponding to the first 6-bp of the Illumina Read1 sequence) in combination with a set of 24 i7 index sequences added during PCR, to allow all of our test samples to be multiplexed and sequenced together. The multiplexed pool was sequenced over multiple lanes of an Illumina HiSeq2500 instrument (KU Genome Sequencing Core) generating 100-bp single-end reads.</p></div><div id="sec-5" class="subsection"><h3>Marker discovery</h3><p id="p-11">For genotyping-by-sequencing analysis we made use of the Stacks pipeline (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Catchen <em>et al.</em> 2011</a>). We demultiplexed reads with the <em>process_radtags</em> algorithm, and merged all reads associated with the KS and FL parental individuals into two pools (KS-pool = 16.3 million reads, FL-pool = 13.8 million reads). We then used <em>ustacks</em> to <em>de novo</em> assemble loci for each parental pooled sample. We parameterized <em>ustacks</em> to construct stacks with a minimum coverage (<em>m</em>) of 5, a maximum number of differences between reads within a locus (<em>M</em>) of 2, no secondary alignments (<em>N</em> = 0), a maximum number of stacks per <em>de novo</em> locus (<em>max_locus_stacks</em>) equal to 2, and with the “Removal” and “Deleveraging” algorithms enabled (−<em>r</em> and –<em>d</em>, respectively). The Removal algorithm excludes stacks that are highly repetitive, while the deleveraging algorithm attempts to resolve over-merged stacks. In total, 148,448 and 143,201 stacks were assembled for the KS and FL parental strains, respectively. We then used <em>cstacks</em> to merge loci from the two parental pooled samples. We set the maximum distance allowed between loci (<em>n</em>) to 2, allowing for the alleles at any heterozygous loci in the parental strains to be merged. We interrogated this catalog and identified 72,076 entries in which each parent contributed a single monomorphic allele. We excluded all other loci because they could represent heterozygous loci, alleles sampled in only one of the two parental lines, or incorrectly-merged paralogous sequences. Of the 72,076 loci, 26,905 (37.3%) were polymorphic between the FL and KS parental lines, and represent the set of informative markers used to genotype the recombinant and segregant progeny.</p></div><div id="sec-6" class="subsection"><h3>Marker genotyping</h3><p id="p-12">To call genotypes in the recombinant and segregant populations, we first <em>de novo</em> assembled loci for each individual using <em>ustacks</em> with similar parameterization to that used for the KS and FL parental samples described above (<em>m</em> = 2, <em>N</em> = 2, –<em>r</em>, –<em>d</em>, –<em>max_locus_stacks</em> = 2). The loci for each individual were then matched against the catalog of informative loci generated using only the parental lines. Finally, we used the <em>genotypes</em> routine within Stacks to generate genotype calls for all recombinants and segregants. Genotyping rates in the three mapping populations were 49.5% (KS-BC), 42.0% (FL-BC), and 34.2% (KS-SG).</p></div><div id="sec-7" class="subsection"><h3>Assigning markers to linkage groups</h3><p id="p-13">To localize markers we used Lep-MAP2 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rastas <em>et al.</em> 2013</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">2016</a>), first converting the Stacks <em>genotypes</em> output file - containing progeny and parents - to the required LINKAGE pedigree format (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lathrop <em>et al.</em> 1984</a>). Within Lep-MAP2 we first used the <em>Filtering</em> module to remove loci showing segregation distortion (<em>dataTolerance</em> = 0.01) in the two recombinant backcrosses, and to remove loci/individuals with limited genotyping data. Different filtering criteria were employed for each backcross to maintain similar levels of missing data across the three sets of genotypes. For the KS-BC we retained 8,132 markers and 339 individuals, for the FL-BC we retained 5,970 markers and 313 individuals, and for the KS-SG we retained 13,295 markers and 198 individuals.</p><p id="p-14">We assigned markers to linkage groups independently for each of the three mapping populations using the <em>SeparateChromosomes</em> Lep-MAP2 module, requiring a minimum LOD (logarithm of odds) score of 20, and a minimum of 20 markers per linkage group. To establish linkage group homology across populations we compared the assignment of markers to linkage groups in each map. Eight (0.2%) of the 3,204 overlapping markers between KS-BC and FL-BC were removed entirely from the dataset because they were inconsistent in their placement, <em>i.e.</em>, a marker was placed on different linkage groups in the two maps. Based on the remaining overlapping markers, linkage groups in KS-BC and FL-BC were renamed to reflect the respective linkage groups in KS-SG.</p><p id="p-15">For the pair of recombinant backcross populations, markers associated with each linkage group were then separately ordered with <em>OrderMarkers</em> using the Kosambi function (<em>useKosambi</em> = 1; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kosambi 1943</a>), taking into account the achiasmatic meiosis in females (<em>initRecombination</em> = 0.05 0, where the first number is the <em>a priori</em> probability of crossing over in males, and the second is the same probability in females), and removing markers positioned at identical genetic positions within each linkage group (<em>removeDuplicates</em> = 1). Following this procedure, we immediately noticed that marker order within a given linkage group was very different between backcrosses, particularly for markers in the middle of each linkage group. We examined this observation in multiple ways (see “Results and Discussion”), concluding there are remarkably few crossover events evident the data, precluding confident marker ordering within linkage groups.</p></div><div id="sec-8" class="subsection"><h3>QTL mapping</h3><p id="p-16">Due to the limited number of crossovers, and the inability to order markers, we elected not to attempt traditional interval QTL mapping (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lander and Botstein 1989</a>) for the backcross populations. Instead, we ignored all crossing over, and assigned a single consensus genotype to each linkage group in each individual based on three filters: (1) at least 80% of the markers on the linkage group were given a genotype call, (2) the minimum number of called markers for that linkage group was 30, and (3) at least 90% of the markers on that linkage group had the same genotype. We used these “collapsed” genotypes to perform marker regression in R/qtl (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Broman and Sen 2009</a>), effectively mapping QTL to the chromosome level for each phenotype. Given the family structure of the mapping populations, and given this structure is confounded with the year in which the experimental individuals were generated, we included family membership as a covariate in all QTL analyses. We additionally performed QTL mapping for both pulse-pair rate and peak amplitude after correcting for body weight variation (see “Results and Discussion”). Within each mapping population we regressed weight on each of these song traits individually using the <em>glm</em> function in R (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">R Core Team 2018</a>), extracted the residuals, and used these weight-corrected phenotypes for mapping. Significance thresholds for QTL mapping were established by running 1,000 permutations of the data (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Churchill and Doerge 1994</a>).</p><p id="p-17">We used R/qtlDesign (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sen <em>et al.</em> 2007</a>) to calculate the statistical power we had to detect each of the QTL mapped in the study. We used the function <em>powercalc</em> for each QTL we detected with the following parameters: (1) sample size, (2) error variance, as calculated via the R/qtl <em>fitqtl</em> function for each QTL using the Haley-Knott regression method, and (3) the QTL effect, also calculated via <em>fitqtl</em>. The cross type was set to ‘bc’ for all power calculations.</p></div><div id="sec-9" class="subsection"><h3>Genome sequencing data collection</h3><p id="p-18">DNA was isolated from multiple males from the KS <em>A. grisella</em> inbred line (<a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> 2016</a>) and pooled. We employed males, the homogametic (ZZ) sex, to ensure that the Z had similar coverage to each autosome in subsequent sequencing. Two Illumina sequencing libraries were generated (Cofactor Genomics, Inc.); a short insert size paired end (hereafter “PE”) library with 280-504bp inserts, and a long insert size mate pair (hereafter “MP”) library with 3-5kb inserts. Each library was sequenced on three lanes of an Illumina HiSeq2500 instrument, one lane at Cofactor Genomics, Inc. and two lanes at the KU Genome Sequencing Core. We obtained 291 million and 405 million 101-bp read pairs for the PE and MP library, respectively (Table S1).</p></div><div id="sec-10" class="subsection"><h3>Genome sequencing data processing</h3><p id="p-19">Read quality has a major effect on the result of <em>de novo</em> genome assembly (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Salzberg <em>et al.</em> 2012</a>), therefore we first preprocessed the raw FASTQ files. Initially, we removed adaptor sequences using Scythe (<a href="https://github.com/vsbuffalo/scythe">https://github.com/vsbuffalo/scythe</a>) and quality trimmed reads via Sickle (<a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a>), eliminating any reads containing uncalled positions (<em>i.e.</em>, “N” bases), or with a trimmed length below 80-bp. Any reads whose pairs were discarded in the trimming process were saved in a separate FASTQ file which was used alongside the pairs in downstream analyses. Subsequently, we used bowtie2 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Langmead and Salzberg 2012</a>) with default settings to align preprocessed reads to the PhiX (<em>Escherichia</em> virus phiX174) reference genome (NCBI Reference Sequence: NC_001422.1) that was run together with our sample as a control during Illumina sequencing, removing any contaminating reads, and storing the unmapped reads. Finally, we corrected the set of uncontaminated, quality-trimmed reads using Quake, a maximum-likelihood based tool for detecting and correcting sequencing errors (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kelley <em>et al.</em> 2010</a>), setting <em>k</em> to 18. Overall, preprocessing removed 10% of the original reads (Table S1), slightly improving the average base quality (Figure S1).</p></div><div id="sec-11" class="subsection"><h3>Genome assembly and evaluation</h3><p id="p-20">To assemble processed reads into scaffolds we used the de Bruijn assembler ABySS (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Simpson <em>et al.</em> 2009</a>), selecting this software both due to its low error rate when assembling a human chromosome (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Salzberg <em>et al.</em> 2012</a>), and its parallel processing ability and low memory requirements (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Simpson <em>et al.</em> 2009</a>). ABySS uses PE reads to assemble contigs, and then MP reads to construct scaffolds. In common with most de Bruijn assemblers (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pevzner <em>et al.</em> 2001</a>) ABySS requires that the user specify a <em>k</em>-mer size, where the optimal <em>k</em> depends on the repetitiveness of the genome, its heterozygosity, and technology-specific error rates (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Chikhi and Medvedev 2014</a>). We used KmerGenie (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Chikhi and Medvedev 2014</a>) to estimate the appropriate <em>k</em>-mer as 93, then ran ABySS with this value, otherwise employing the default parameters, on a single cluster node with 16 processors and 32GB of RAM. To assess completeness of the final draft assembly we used CEGMA (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Parra <em>et al.</em> 2007</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">2009</a>) with default parameters.</p></div><div id="sec-12" class="subsection"><h3>RNAseq and transcriptome assembly</h3><p id="p-21">To assist with genome annotation, we collected RNAseq data and assembled transcripts from <em>A. grisella</em>. Total RNA was individually isolated from two pupae from strain “Louisiana line 112” (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zhou <em>et al.</em> 2008</a>). A poly-A selected, unstranded TruSeq Illumina RNAseq library was constructed for each individual, and the pair of libraries were sequenced over a single lane of a HiSeq2500 instrument to generate around 175 million paired-end 100-bp reads (KU Genome Sequencing Core). Quality trimming via Sickle (<a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a>, window-wise quality threshold parameter <em>q</em> = 40, minimum read length post-trimming = 50-bp) retained around 96% of the reads. All quality-trimmed reads were merged into a single FASTA file. To assemble transcripts we used Trinity (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Grabherr <em>et al.</em> 2011</a>) with default parameters, except we turned on read normalization, and set the maximum read coverage for normalization to 50. The assembly took &lt;6 hr on a 16-core node with 256 GB of RAM, and generated 96,420 transcripts with an N50 of 2551-bp and a mean scaffold length of 1178.45 bp.</p></div><div id="sec-13" class="subsection"><h3>Genome annotation</h3><p id="p-22">We annotated our <em>de novo</em> genome assembly using MAKER2 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Holt and Yandell 2011</a>). We provided MAKER2 with the Trinity-assembled <em>A. grisella</em> transcripts, protein databases from the lepidopterans <em>Heliconius melpomene</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Davey <em>et al.</em> 2016</a>), <em>Danaus plexippus</em> (OGS2.0; <a href="http://monarchbase.umassmed.edu/">http://monarchbase.umassmed.edu/</a>), and <em>Bombyx mori</em> (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Xia <em>et al.</em> 2004</a>), and a repeat database generated by RepeatModeler (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Smit and Hubley 2008-2015</a>) that contains short and long interspersed nuclear elements, long terminal repeat elements, small RNAs, and other unclassified repeats. Within MAKER2 we used two gene predictors, Augustus (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Stanke and Morgenstern 2005</a>) and SNAP (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Korf 2004</a>). For Augustus, we employed a publicly-available parameter set developed for <em>H. melpomene</em>, which is distributed with the software. For SNAP, we used an HMM file generated by bootstrap training of the gene predictor over three runs of MAKER2. Assessment of the completeness of the genome annotation was accomplished using BUSCO (version 3, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Simão <em>et al.</em> 2015</a>) using single-copy orthologs specific to the phylum Arthropoda from OrthoDB (version 9, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Waterhouse <em>et al.</em> 2013</a>) as our reference gene set.</p></div><div id="sec-14" class="subsection"><h3>Associating genome scaffolds with linkage groups</h3><p id="p-23">We created a nucleotide database from the genome assembly using <em>makeblastdb</em> on a local BLAST (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Altschul <em>et al.</em> 1990</a>) installation. The sequences of markers that had been placed on linkage groups were then extracted from the Stacks <em>cstacks</em> catalog file, and formatted as a FASTA file, adding the linkage group for each marker to the sequence header. Each marker sequence was then mapped to the assembled genome using BLASTN (−<em>evalue</em> 1e-30), and 9,746 markers hit just one scaffold (59 markers with significant hits to more than one scaffold were ignored), enabling us to place scaffolds onto linkage groups. Similarly, we placed the sequences of the 75 EST-derived markers used by <a id="xref-ref-29-8" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> onto scaffolds. This allowed us to translate among the linkage group identifiers assigned in our two studies (Table S2).</p></div><div id="sec-15" class="subsection"><h3>Comparing genotypes across studies</h3><p id="p-24">In their QTL mapping study <a id="xref-ref-29-9" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> employed some of the same recombinant individuals we used here, genotyping a set of markers using Illumina BeadXpress technology. Because we know the scaffolds on which all markers from this and the present study reside (above), we could compare the accuracy of genotype calls at pairs of markers on the same scaffold. We note that the assumption such markers should have the same genotype is dependent on scaffolds rarely being chimeric, and on crossovers being infrequent at the scale of a given scaffold. We examined only those 32 <a id="xref-ref-29-10" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> markers present on scaffolds associated with the 16 linkage groups showing an unambiguous one-to-one relationship among studies (Table S2), minimizing the potential for marker-to-scaffold mismapping to falsely indicate an apparently high rate of genotyping error. When multiple markers from the present study were on one of these scaffolds we assigned a consensus genotype call for each individual, assigning a no-call in the case that marker-specific calls were not all identical.</p></div><div id="sec-16" class="subsection"><h3>Associating A. grisella linkage groups with chromosomes from sequenced lepidopterans</h3><p id="p-25">The genome of <em>B. mori</em> (version Jan. 2017) was downloaded from SilkBase (<a href="http://silkbase.ab.a.u-tokyo.ac.jp/cgi-bin/index.cgi">http://silkbase.ab.a.u-tokyo.ac.jp/cgi-bin/index.cgi</a>), and the genome of <em>H. melpomene</em> (version 2.5), along with the accompanying AGP file (A Golden Path file, which is a description of the assembly), was downloaded from the Butterfly Genome Database (<a href="http://butterflygenome.org/">http://butterflygenome.org/</a>). We generated a BLASTable database for each genome, and subsequently mapped repeat-masked <em>A. grisella</em> scaffolds (with repetitive sequence replaced by runs of “N” bases) against each database using BLAST+ with default parameters (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Camacho <em>et al.</em> 2009</a>). We ignored alignments based on the following criteria: (a) alignments of <em>A. grisella</em> scaffolds not assigned to linkage groups, (b) alignments to <em>B. mori</em> sequences not assigned to chromosomes, (c) alignments with an identity percentage of &lt;90%, and (d) alignments &lt;125-bp long (for <em>B. mori</em>) or &lt;100-bp long (for <em>H. melpomene</em>). The remaining alignments were considered, and an <em>A. grisella</em> linkage group was associated with a chromosome from one of the other species if at least 15 alignments were identified.</p></div><div id="sec-17" class="subsection"><h3>Independent identification of the Z chromosome</h3><p id="p-26">To identify the Z using an orthogonal approach, we extracted protein sequences of Z-linked genes from the two lepidopteran species <em>B. mori</em> (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Xia <em>et al.</em> 2004</a>) and <em>Melitaea cinxia</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> 2014</a>). Each set of protein sequences was aligned to our assembled genome using TBLASTN (−<em>evalue</em> 1e-50). Proteins mapping to a single scaffold were used to identify a set of <em>A. grisella</em> scaffolds containing sequences with strong homology to known Z-linked proteins. Since scaffolds are associated with linkage groups (see above), we confirmed the identity of the linkage group representing the <em>A. grisella</em> Z chromosome.</p></div><div id="sec-18" class="subsection"><h3>Data availability</h3><p id="p-27">We have deposited raw FASTQ files from our genome sequencing, RNAseq, and MSG in the NCBI Sequence Read Archive under accession SRP158931. In addition, we have deposited a data package on FigShare that contains the following information, resources and scripts: (1) all supplementary tables and figures described in the text, (2) phenotypes for all individuals, (3) genotypes for all markers associated with linkage groups, (4) genotypes for the consensus, linkage group-specific markers used for QTL mapping, (5) sequences of all markers, all assembled genome scaffolds, and all assembled transcripts, (6) the genome annotation, and (7) custom Python scripts employed. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8072405">https://doi.org/10.25387/g3.8072405</a>.</p></div></div><div class="section results-discussion" id="sec-19"><h2 class="">Results and Discussion</h2><p id="p-28">This study develops the previous genetic mapping of male song and life history characters in the lesser wax moth <em>Achroia grisella</em>, expanding on the sample of recombinant individuals originally described by <a id="xref-ref-29-11" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. We employed around 1,000 phenotyped males from three mapping populations, each established from the same pair of parental strains, used genotyping-by-sequencing to generate markers and resolve linkage groups for the full complement of chromosomes, and subsequently identified several loci contributing to trait variation. In addition, by generating a draft, annotated <em>de novo</em> genome assembly for <em>A. grisella</em>, and linking large numbers of scaffolds to linkage groups, we facilitate continued genetic analysis of male song in this non-model insect species.</p><div id="sec-20" class="subsection"><h3>Phenotypic variation</h3><p id="p-29">Three male song traits (pulse-pair rate, peak amplitude, asynchrony interval) and two life history traits (development time and body weight) were measured in two recombinant backcross populations (KS-BC and FL-BC) and in one segregant population (KS-SG). All traits exhibited substantial variation within populations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), as was previously observed by <a id="xref-ref-29-12" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. Using simple linear models we found that “family” influenced development time in all three populations, body weight in KS-BC and KS-SG, and peak amplitude in KS-BC (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>; <em>P</em> &lt; 0.001 in each case).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493524/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493524/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Phenotype means for each family and mapping population</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">We examined correlations among all quantitative traits in each mapping population, as this can show how variation of one trait influences that of another, and potentially also indicates genetic correlations among traits. The pattern of correlations among pairs of traits was similar across the three mapping panels (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), suggesting phenotypic associations are broadly maintained. Development time and body weight were significantly negatively correlated in all three populations (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), a correlation previously reported by <a id="xref-ref-29-14" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>, implying longer development time results in lower body weight. Both backcross populations, and to some extent the segregant population, show a positive correlation between peak amplitude and body weight (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). This correlation was already noted in several previous studies in the <em>A. grisella</em> system (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Jang and Greenfield 1998</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Brandt and Greenfield 2004</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Alem <em>et al.</em> 2013</a>; <a id="xref-ref-29-15" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> 2016</a>), and suggests larger males are able to emit songs with a higher peak amplitude. Body weight also shows a weak negative correlation with pulse-pair rate in all three populations (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), as previously shown by <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Brandt and Greenfield (2004)</a>, implying larger males generate songs with lower pulse-pair rates on average. The effects of body weight suggest that at least some of the variation we see in song structure in our mapping panels is due to variation in body size.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493494/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493494/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Correlations among phenotypes within each population</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">We found just two, relatively weak correlations among the three song traits (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>); a negative correlation between pulse-pair rate and peak amplitude in the FL-BC population, and a negative correlation between pulse-pair rate and asynchrony interval in the KS-BC population, the latter previously reported both by <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Collins <em>et al.</em> (1999)</a> and by <a id="xref-ref-29-16" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. The relationships among these traits have the same sign in the other populations, but are not significant. Thus, any genetic association between song traits is at best extremely subtle, and these traits are most likely impacted by independent genetic and environmental factors.</p></div><div id="sec-21" class="subsection"><h3>Placing markers on linkage groups</h3><p id="p-35">Previous attempts to genetically dissect phenotypic variation in the <em>A. grisella</em> system have employed AFLP-based maps (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Limousin <em>et al.</em> 2012</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Alem <em>et al.</em> 2013</a>) or relatively few markers that collectively have not tagged the full complement of chromosomes in the system (<a id="xref-ref-29-17" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> 2016</a>). To mark all linkage groups with sequence-based markers we used a genotyping-by-sequencing approach to generate thousands of markers discriminating the KS and FL parental strains, and then used Lep-MAP2 (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rastas <em>et al.</em> 2013</a>, <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">2016</a>) to place 5,721-12,801 markers on 30 linkage groups in each population (Table S3). Thirty linkage groups is consistent with the haploid number of chromosomes observed through karyotyping by <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Limousin <em>et al.</em> (2012)</a>.</p></div><div id="sec-22" class="subsection"><h3>Associating linkage groups with chromosomes from other sequenced lepidopterans</h3><p id="p-36">To facilitate future exploration of the <em>A. grisella</em> genome in the context of other lepidopterans, we tied our linkage groups to the chromosomes of <em>Heliconius melpomene</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Heliconius Genome Consortium 2012</a>) and <em>Bombyx mori</em> (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Xia <em>et al.</em> 2004</a>) by leveraging our draft genome assembly (for full details of the assembly see below). First, we used BLAST to place marker sequences on genome scaffolds, thereby tying scaffolds to linkage groups. Subsequently, and again using BLAST, we associated those linkage group-associated <em>A. grisella</em> genome scaffolds with <em>H. melpomene</em> and <em>B. mori</em>. This resulted in connecting 24/30 of our linkage groups to <em>H. melpomene</em> chromosomes and 20/30 to <em>B. mori</em> chromosomes (Table S4). Our results are consistent with the previously reported homology between the chromosomes of <em>H. melpomene</em> and <em>B. mori</em> (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Heliconius Genome Consortium 2012</a>).</p><p id="p-37">To confirm that this analysis correctly identified the <em>A. grisella</em> Z chromosome, we extracted protein sequences from known Z-linked genes from <em>B. mori</em> and <em>Melitaea cinxia</em> (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> 2014</a>), and used BLAST to associate them with our draft genome. We found that 206/654 <em>B. mori</em> proteins aligned to 149 <em>A. grisella</em> scaffolds, and 141/572 <em>M. cinxia</em> proteins aligned to 116 <em>A. grisella</em> scaffolds. Seventy-three of these scaffolds are in common, strongly suggesting they reside on the Z in <em>A. grisella</em>. Indeed, we associated 48/73 of these scaffolds to linkage groups (see below), and 43/48 are placed on the Z chromosome.</p><p id="p-38">Our use of a population of segregants allowed us to find markers linked to the Z chromosome, and support these homology-based analyses. All males from the KS-SG population had one, intact Kansas-derived Z chromosome and one, intact Florida-derived Z chromosome. The specific pattern of inheritance of Z-linked markers in this population initially, and incorrectly appeared as severe segregation distortion during the Lep-MAP2 linkage group assignment, but ultimately this property allowed us to confirm that such markers were on the Z.</p></div><div id="sec-23" class="subsection"><h3>Difficulty ordering markers within linkage groups</h3><p id="p-39">In examining marker order between the genetic maps derived from the two backcross populations (FL-BC and KS-BC) we found considerable inconsistency. While markers at the termini of each linkage group were largely consistent in order between the two backcrosses, markers in the middle of each linkage group were scrambled. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> highlights this phenomenon for linkage group 1. If marker order were preserved between backcross populations, markers would fall along a line. We observed the same pattern when using the software ALLMAPS (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Tang <em>et al.</em> 2015</a>), which allowed us to map genetic markers to the set of physical <em>A. grisella</em> scaffolds from our genome assembly. ALLMAPS showed that markers common to both backcross populations always mapped to the same scaffold, but that the marker order defined by each backcross-specific genetic map was distinct (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> demonstrates this pattern for linkage group 18). To explore this pattern further we plotted genotypes for all individuals in a population at all markers, in the order defined by the genetic map. As exemplified by linkage group 1 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), crossovers appear to be relatively rare in the pair of backcross populations, but as expected are absent in the segregant population. Most backcross individuals are <em>either</em> homozygous <em>or</em> heterozygous for an entire linkage group, and when crossovers are evident they are near the ends of the linkage groups. The apparent scarcity of crossover events provided minimal information to Lep-MAP2 to assist with marker ordering, and is likely why markers in the middle of the linkage groups are inconsistently-ordered between backcross populations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker order is not preserved between backcross populations. The genetic positions (in cM) of all markers on linkage group 1 (LG1) that are shared between the FL-BC and KS-BC populations. If relative marker position was consistent between backcrosses, points would appear on a line/curve." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281274679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Marker order is not preserved between backcross populations. The genetic positions (in cM) of all markers on linkage group 1 (LG1) that are shared between the FL-BC and KS-BC populations. If relative marker position was consistent between backcrosses, points would appear on a line/curve.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2349/F1.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2349/F1.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493648" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-40" class="first-child">Marker order is not preserved between backcross populations. The genetic positions (in cM) of all markers on linkage group 1 (LG1) that are shared between the FL-BC and KS-BC populations. If relative marker position was consistent between backcrosses, points would appear on a line/curve.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker order inconsistency between backcross populations. ALLMAPS was used to map genetic markers to our genome scaffolds separately for each backcross. Blocks in the central chromosome represent scaffolds, and lines connect these physical positions to the genetic positions in each map for markers on linkage group 18. The presence of crossed lines indicates map order inconsistency, which is particularly evident in the central portion of the chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281274679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Marker order inconsistency between backcross populations. ALLMAPS was used to map genetic markers to our genome scaffolds separately for each backcross. Blocks in the central chromosome represent scaffolds, and lines connect these physical positions to the genetic positions in each map for markers on linkage group 18. The presence of crossed lines indicates map order inconsistency, which is particularly evident in the central portion of the chromosome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2349/F2.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2349/F2.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493685" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Marker order inconsistency between backcross populations. ALLMAPS was used to map genetic markers to our genome scaffolds separately for each backcross. Blocks in the central chromosome represent scaffolds, and lines connect these physical positions to the genetic positions in each map for markers on linkage group 18. The presence of crossed lines indicates map order inconsistency, which is particularly evident in the central portion of the chromosome.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Limited numbers of crossovers in backcross populations. The panels on the left show genotypes for individuals (rows) and markers (columns) for each mapping population for linkage group 1 (LG1). Heterozygous genotypes (FL/KS) are shown in blue, homozygous genotypes (either FL/FL or FL/KS) are shown in orange, and no-calls are shown in white. Only a subset of individuals and markers are presented in the figure to minimize the number of no-call genotypes shown. In the recombinant, backcross populations, with few exceptions (generally near the ends of the linkage groups), the majority of individuals exhibit the same genotype call for the entire chromosome. This is particular clear in the panels on the right, which show the fraction of called genotypes in each individual that are heterozygous." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281274679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Limited numbers of crossovers in backcross populations. The panels on the left show genotypes for individuals (rows) and markers (columns) for each mapping population for linkage group 1 (LG1). Heterozygous genotypes (FL/KS) are shown in blue, homozygous genotypes (either FL/FL or FL/KS) are shown in orange, and no-calls are shown in white. Only a subset of individuals and markers are presented in the figure to minimize the number of no-call genotypes shown. In the recombinant, backcross populations, with few exceptions (generally near the ends of the linkage groups), the majority of individuals exhibit the same genotype call for the entire chromosome. This is particular clear in the panels on the right, which show the fraction of called genotypes in each individual that are heterozygous.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2349/F3.medium.gif" width="253" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2349/F3.medium.gif" width="253" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2349/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493570" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Limited numbers of crossovers in backcross populations. The panels on the left show genotypes for individuals (rows) and markers (columns) for each mapping population for linkage group 1 (LG1). Heterozygous genotypes (FL/KS) are shown in blue, homozygous genotypes (either FL/FL or FL/KS) are shown in orange, and no-calls are shown in white. Only a subset of individuals and markers are presented in the figure to minimize the number of no-call genotypes shown. In the recombinant, backcross populations, with few exceptions (generally near the ends of the linkage groups), the majority of individuals exhibit the same genotype call for the entire chromosome. This is particular clear in the panels on the right, which show the fraction of called genotypes in each individual that are heterozygous.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Genotyping error could also lead to a spurious reduction in crossover rate and incorrect marker order, and we evaluated the level of genotyping error in our dataset using two strategies. First, we took advantage of the fact that <a id="xref-ref-29-18" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> used a set of KS-BC individuals that overlapped with the set we employed here, but genotyped a different series of markers with an entirely different technology. We compared genotypes between the two studies at markers jointly mapped to the same genome scaffold via BLAST. Across 32 positions, and considering 161-359 individuals per position, the mean percentage of identical genotypes was 96.9%, with just 1/32 having a percent identity below 94% (Table S5). Thus, the marker genotype data we employ in this study appears to be accurate.</p><p id="p-44">Second, we examined the error rate of our genotyping pipeline using the KS-SG segregant population, as here the genotypes of all markers on a given chromosome from a given individual should be identical. Indeed, this is the pattern we observe, with all genotypes on an individual chromosome either being typically homozygous, or typically heterozygous (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>; Figure S2). Nonetheless, genotyping error is not absent from our study. In linkage group 1 of the KS-SG population (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) a greater number of spurious “homozygous” (orange) calls occur in individuals with majority heterozygous (blue) chromosomes than the reverse. Additionally, when we consider only those chromosomes from KS-SG where at least 30% of markers receive a genotype call, the average frequency of heterozygous calls on the Z is 93.8% (given the cross design, all KS-SG Z-linked loci should be heterozygous), on majority heterozygous autosomes is 93.2%, and on majority homozygous autosomes is 0.01%. These observations imply that our genotyping pipeline undercalls heterozygotes. This is a common issue when applying genotyping-by-sequencing technologies to non-inbred diploid organisms such as our backcross individuals, because a true heterozygote may appear as a homozygote if the alternative allele is simply not sampled by a read, whereas for a true homozygote to appear as a heterozygote a specific sequencing error must occur.</p><p id="p-45">Our use of a fairly common-cutting restriction enzyme (<em>Ase</em>I, AT^TAAT) likely contributed to our biased genotyping error, since it resulted in a very large number of markers, each of which we covered only shallowly with sequencing reads. To test whether greater numbers of crossover events, and enhanced consistency in marker order between backcrosses, could be achieved using a dataset with a reduced potential for genotyping error, we repeated the genotype calling step in all three populations, but increased the read depth requirement from 5 to 8 (using <em>genotypes</em> in Stacks, see “Materials and Methods”). This new set of genotype calls yielded a similar pattern of apparently rare crossovers and backcross-to-backcross marker order inconsistency.</p><p id="p-46">Overall, it does not appear that genotyping error has dramatically affected reconstruction of the haplotypes in our experimental individuals. Instead, it appears that our <em>A. grisella</em> backcross mapping populations are subject to relatively low effective levels of intrachromosomal recombination over the bulk of the physical length of each chromosome. This observation could be due to crossing over events occurring normally, but being localized nearly exclusively to the very ends of chromosomes. Alternatively, our population could be subject to a very low rate of recombination initiation, such that very few Holliday junctions are formed, or the events that do occur could be preferentially resolved into non-crossover molecules. Regardless of the mechanism, the relatively short <em>A. grisella</em> genetic map in the present study is supported by the map generated by <a id="xref-ref-29-19" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>, who genotyped 75 SNP markers in a set of individuals that encompasses the KS-BC population we employ here. Their map yielded 20 linkage groups, 17 of which had lengths below 20-cM. The AFLP-based mapping study of <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Alem <em>et al.</em> (2013)</a> also yielded a short map, with linkage groups of 12-66 cM in length. These studies suggest that laboratory intercross populations of <em>A. grisella</em>, or potentially <em>A. grisella</em> as a species, generate relatively few crossovers in each meiosis that provide utility for genetic mapping. However, further study would be required to establish the generality of our observation, and understand the biological basis of the phenomenon.</p></div><div id="sec-24" class="subsection"><h3>Markers for QTL mapping</h3><p id="p-47">The modest number of informative crossover events per linkage group makes it challenging to identify QTL containing limited collections of genes. Furthermore, given we cannot be confident of genetic marker order across the bulk of each chromosome, any sub-chromosomal positions may be incorrect. Thus, we collapsed all genotyping data for each individual for each linkage group to a single, consensus genotype (see “Materials and Methods”). This approach has the advantage that it yields a much smaller number of markers to be tested in QTL mapping (<em>i.e.</em>, 30), reducing the multiple testing burden over a mapping design with hundreds to thousands of markers. Additionally, in the event that all causative loci on a linkage group act in the same direction (<em>e.g.</em>, all KS-derived alleles increase phenotype), testing for associations between phenotypes and entire chromosomes increases power to detect effects, since the effects of multiple, small-effect QTL on a chromosome are aggregated. Of course, since our approach relies on the net effect of a chromosome being different from zero, chromosomes harboring loci of equal and opposite effect on phenotype will not be detected.</p><p id="p-48">A clear drawback of our approach is that we cannot map QTL to sub-chromosomal positions, although we contend that the recombination landscape of our <em>A. grisella</em> populations does not readily allow this. Nonetheless, considering the number of linkage groups (N = 30) and the number of genes in our genome annotation (estimated to be 15,848 - see below), mapping a QTL to a chromosome resolves to a few hundred genes. This is approximately the same resolution achievable in an equivalent backcross or F<sub>2</sub> QTL study in the elite <em>Drosophila melanogaster</em> model system (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mackay 2001</a>).</p></div><div id="sec-25" class="subsection"><h3>QTL mapping results</h3><p id="p-49">Using marker regression, and accounting for variation due to family by including a covariate during analysis, we mapped variation for our five phenotypes to chromosomes in each of the three mapping panels. Additionally, given the significant correlations between body weight and both pulse-pair rate and peak amplitude (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>), we also attempted to map QTL for these song traits after correcting for body weight variation (see “Materials and Methods”). We set genomewide LOD thresholds for significance via permutation testing (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Churchill and Doerge 1994</a>), employing three thresholds; α=0.05, the generally-accepted significance level for detection of QTL, and additionally α=0.1 and α=0.2 that allowed us to explore weaker QTL effects.</p><p id="p-50">We identified a number of strongly-supported and suggestive chromosomal effects for the three male song traits, pulse-pair rate, peak amplitude, and asynchrony interval, and the two life history traits, development time and body weight (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Considering only those effects surviving the most stringent level of statistical significance (α=0.05) we identified two QTL for pulse-pair rate on linkage groups 11 and 13 in KS-BC (both of which are retained following correction for body weight, Table S6), one for asynchrony interval on linkage group 20 in KS-BC, one for development time on linkage group 7 in KS-BC, one for body weight on linkage group 12 in KS-BC, and an additional body weight QTL on linkage group 28 in KS-SG. The effects of these QTL are all fairly modest, each explaining 2.37–8.68% of the phenotypic variance in the mapping population (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), with the largest effect seen for the development time QTL in KS-BC. Similarly low effect sizes were estimated at QTL mapped in <a id="xref-ref-29-20" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. Given the relatively low mapping power we have to identify small-effect loci (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), and since our target traits show significant heritability (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Collins <em>et al.</em> 1999</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Brandt and Greenfield 2004</a>), our results suggest all traits measured are highly polygenic, with genetic contributions to phenotype from an array of variants, many with very small effects on phenotype.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493650/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493650/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of mapped QTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-57">To attempt to uncover other phenotypically-relevant loci, and specifically to identify associations with other traits at those QTL positions that survive a more rigorous genomewide threshold, we employed less stringent levels of statistical significance (α=0.1, α=0.2). In the KS-BC population we identified a suggestive QTL for body weight on linkage group 7 that co-localized with the large development time QTL (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). <a id="xref-ref-29-22" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> also identified this pair of QTL (see Table S7 for a list of the QTL mapped in this prior study, and their relationship to peaks identified in the current work). We also saw a suggestive QTL for development time in KS-SG on the same chromosome (linkage group 12) that harbors a body weight QTL in KS-BC. These pair of overlapping QTL may reflect the strong correlation between these two life history traits (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>), and suggest either the presence of a locus controlling variation in both traits, or closely-linked loci with distinct effects on each trait. The only other evidence for overlapping QTL was the presence of a weak QTL for peak amplitude on linkage group 11 in the KS-SG population (maintained following body weight correction, Table S6), coincident with a QTL for pulse-pair rate in KS-BC (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Given the minimal evidence for phenotypic correlation between these song traits (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>), the positional overlap is most likely due to chance, and not due to any similarity in the genetic control of trait variation.</p><p id="p-58">Aside from these three instances of across-trait or across-mapping population QTL overlap, even when considering suggestive QTL, no other pairs of QTL map to the same chromosome. <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Alem <em>et al.</em> (2013)</a> similarly observed limited QTL overlap at the chromosomal level in a previous study that examined the same male song and life history traits in an independent <em>A. grisella</em> mapping population. There are several explanations for this lack of co-localization. First, some traits may be under independent genetic control, and overlap would not be expected. Indeed, <a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a> shows that variation for many of our traits is not strongly correlated in our mapping panels. Second, even when the phenotypic correlation of two traits is high, the QTL we map have modest effects (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). Our power to detect such QTL was often low (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>), limiting our ability to detect the same effect in more than one population. Third, like any QTL study we are subject to the Beavis effect (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Beavis <em>et al.</em> 1991</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Beavis 1994</a>), the phenomenon that the percentage of phenotypic variance explained by QTL that reach a specified level of statistical significance will be overestimated, particularly if the study has a modest sample size. (Other studies have referred to this property as the “winner’s curse”, <em>e.g.</em>, <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zollner and Pritchard 2007</a>). This suggests that even our small estimated QTL effects are larger than the true effects, compounding the difficulty resolving overlapping pairs of QTL. Fourth, dominance may be a barrier to identifying QTL across reciprocal backcross populations. For instance, even a large effect, but dominantly-acting QTL mapped in the KS-BC population is expected to be invisible in the FL-BC population. Finally, an important technical concern with our chromosome-by-chromosome mapping approach is that if more than one causative locus is present on a chromosome, and the direction of the individual effects sums close to zero, a single consensus chromosome test will reveal no effect. Only much larger mapping panels that incorporate much greater numbers of crossover events, perhaps generated by some kind of advanced intercross design (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Darvasi and Soller 1995</a>), are likely to be successful at identifying small effect loci impacting behavioral and morphological variation in <em>A. grisella</em>.</p></div><div id="sec-26" class="subsection"><h3>Comparison to previous QTL maps</h3><p id="p-59">We compared our QTL results with other studies mapping the same traits in <em>A. grisella</em>. Two were based on genetic maps derived solely from anonymous AFLP markers (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Limousin <em>et al.</em> 2012</a>; <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Alem <em>et al.</em> 2013</a>), so we were unable to resolve chromosome homology among studies, making chromosome-by-chromosome comparison impossible. We were able to directly compare our data to <a id="xref-ref-29-23" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> who used gene-based markers, so we could establish which of their markers map to our <em>de novo</em> assembled scaffolds, and translate among their and our linkage group identifiers (Table S2).</p><p id="p-60"><a id="xref-ref-29-24" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> identified eight QTL in a Kansas backcross mapping population, of which our KS-BC individuals were a subset. Just two QTL were apparently identified in both studies (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>, Table S7); our development time QTL on linkage group 7 (also our largest-effect QTL), and our suggestive QTL for body weight also on linkage group 7 overlapped QTL for the same traits in <a id="xref-ref-29-25" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. The other six QTL identified by <a id="xref-ref-29-26" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> are not recapitulated in the current study. In two cases - QTL for body weight and pulse rate - this may be because the linkage group these QTL are mapped to in <a id="xref-ref-29-27" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> is split into two linkage groups in our genetic map (Table S2, Table S7). A possibility is that these QTL are spurious, and were generated by incorrectly joining linkage groups due to limited marker density in the previous study. The remaining four QTL identified by <a id="xref-ref-29-28" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>, and not identified here, all have small effects (Table S7) and may not have been found simply due to power deficits, particularly because the number of KS-BC individuals we employed was slightly lower than used by <a id="xref-ref-29-29" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. In addition, the two studies used radically different marker sets (75 SNPs <em>vs.</em> thousands of markers), and employed very different analytical methodologies; <a id="xref-ref-29-30" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a> used composite interval mapping (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zeng 1993</a>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">1994</a>), whereas we used chromosome-by-chromosome marker regression. Such technical differences could easily explain the differences in result, particularly if our traits are highly polygenic, because the identification of small-effect functional loci might be particularly sensitive to the precise mapping strategy applied.</p><p id="p-61">Similar methodological and power concerns might explain why we were able to identify novel QTL (<a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>) not identified by <a id="xref-ref-29-31" class="xref-bibr" href="#ref-29">Gleason <em>et al.</em> (2016)</a>. In addition, we also examined the reciprocal backcross (FL-BC), so if any loci we map segregate for dominantly-acting alleles in the cross, the ability to find such variants in reciprocal backcross populations will differ.</p></div><div id="sec-27" class="subsection"><h3>Genome assembly and annotation</h3><p id="p-62">A goal of any mapping project is to enable the identification of genes contributing to trait variation. Hence, we assembled a draft genome of <em>A. grisella</em> (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) using both short- and long-insert sequencing libraries and short-read (100-bp) Illumina sequencing. The total, end-to-end length of the assembled scaffolds is 418-Mb, half of the bases are in scaffolds 87.3-Kb or longer, and the assembly has a GC content of 32.4%, which is on par with other sequenced lepidopteran genomes (<em>e.g.</em>, <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zhan <em>et al.</em> 2011</a>). To assess completeness of the set of scaffolds we used CEGMA (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Parra <em>et al.</em> 2007</a>, <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">2009</a>), and showed that our assembled scaffolds contain 196/248 (79.03%) intact core eukaryotic genes. Investigators could likely obtain a more contiguous, and more complete assembly by adding long-read, single molecule sequencing data to our short-read sequencing dataset in the future (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chakraborty <em>et al.</em> 2018</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Baldwin-Brown <em>et al.</em> 2018</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493683/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493683/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Genome assembly statistics</span><div class="sb-div caption-clear"></div></div></div><p id="p-65">To localize scaffolds to linkage groups we mapped our set of genetic markers to the genome assembly using BLAST, tying 3,099 scaffolds to linkage groups using this approach. While this represents a small minority of the total number of assembled scaffolds (4.2%, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), the scaffolds linked to linkage groups comprise 63.1% of the total length of the assembly. Thus, our genetic markers placed the majority of long scaffolds onto linkage groups. Unfortunately, given our inability to confidently generate an ordered genetic map due to the apparent lack of crossovers in our mapping population, we cannot accurately order or orient scaffolds within linkage groups. If very long scaffolds could be produced in the future via single molecule sequencing, it would be straightforward to physically order markers along linkage groups, better connect the physical and genetic maps, and perhaps increase the resolution of the QTL maps we were able to produce. To facilitate such work we have released all raw and processed genome data associated with this study (see “Data availability”).</p><p id="p-66">To annotate scaffolds we used MAKER2 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Holt and Yandell 2011</a>), identifying 15,848 predicted genes. The number of genes we found is not dissimilar to the 12,669 predicted protein-coding genes identified in <em>H. melpomene</em> (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Heliconius Genome Consortium 2012</a>) or the 12,901 identified in <em>B. mori</em> (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Xia <em>et al.</em> 2004</a>). MAKER2 scores annotated gene models on a 0-1 Annotation Edit Distance (AED) scale (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Eilbeck <em>et al.</em> 2009</a>), where lower values indicate better agreement between the annotation and the supporting evidence. The AED scale provides a useful statistic for understanding the quality of an annotation (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Holt and Yandell 2011</a>). We found that 91.5% of the predicted genes in our <em>A. grisella</em> draft genome have AED values less than or equal to 0.5, suggesting that a large fraction were accurate. Using BUSCO (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Simão <em>et al.</em> 2015</a>) we sought to identify known single-copy, arthropod genes (see <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Waterhouse <em>et al.</em> 2013</a>) among our annotated gene set. BUSCO identified 80.7% (860/1066) of such genes as complete, with all but 18 of the 860 being single copy in our annotation. A further 12.6% (134/1066) of the test genes were present in our annotation, but fragmented, while 6.8% (72/1066) were missing entirely. Thus, our annotation pipeline has likely identified the bulk of the protein-coding genes in <em>A. grisella</em>.</p><p id="p-67">Nearly two thousand of the genes were assigned a predicted function based on sequence similarity with a gene annotated in a related organism, including an array of conserved enzyme genes, genes encoding subunits of the basal transcription machinery, genes for cuticular proteins, detoxification cascade components (<em>e.g.</em>, cytochrome P450s), odor and gustatory receptors, and so on. Yet despite this sophisticated genome annotation, most of the genes in the <em>A. grisella</em> draft genome were not associated with any predicted function. Notably, even in elite model genetic systems such as <em>D. melanogaster</em>, subjected to significant gene-by-gene and genomewide functional exploration, a significant fraction of known genes still have only a basic annotation. The functional annotation of the <em>A. grisella</em> draft genome we have constructed would be enhanced by a more detailed comparison of the predicted gene sequences with those from the array of related, lepidopteran and insect genomes that have now been sequenced, and the data we provide should facilitate such comparisons. However, even then there will be no substitute for detailed, functional gene characterization directly within the <em>A. grisella</em> system, using both genome-scale technologies (<em>e.g.</em>, RNAseq, ATACseq), and gene-specific functional tools (<em>e.g.</em>, CRISPR/Cas9 editing which has been successful in lepidopteran systems, see <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zhang <em>et al.</em> 2017</a>).</p><p id="p-68">By virtue of linking scaffolds to linkage groups we were able to associate between 100 and 576 genes with each linkage group (Table S8), although just over five thousand genes are resident on scaffolds that could not be placed on linkage groups using the present set of markers. While we have only succeeded in elucidating a fraction of the genes on each linkage group, these will ultimately assist with associating genes to the loci mapped for male song and life history traits in this, and in future studies in the <em>A. grisella</em> system.</p></div><div id="sec-28" class="subsection"><h3>Concluding thoughts</h3><p id="p-69">Our study brought the strengths of next-generation sequencing technologies to a non-model insect species, <em>Achroia grisella</em>, to better characterize the genome of the organism, improve the genetic and genomic resources available to the community, and build upon previous work dissecting the genetic basis of a sexually-selected behavioral phenotype exhibited by males of the species. We were able to assemble an accurate, although fragmented, draft genome of <em>A. grisella</em> that has an N50 length of &gt;87Kb. Following annotation of the assembled scaffolds we identified nearly 16,000 genes, which evidence suggests represent the bulk of the protein-coding genes of the organism. By virtue of generating a large number of progeny from a cross between a pair of inbred lines, and genotyping these animals for a genomewide set of sequence-based markers, we were also able to assign hundreds of markers to each of the 30 linkage groups harbored by <em>A. grisella</em>. In turn, the high marker density allowed us to assign &gt;3,000 long scaffolds to the linkage groups, and associate &gt;63% of the total length of the <em>de novo</em> assembly, and &gt;10,000 annotated genes with chromosomes. To facilitate future exploration by investigators we have tied our linkage groups to chromosomes from the sequenced lepidopterans <em>H. melpomene</em> and <em>B. mori</em>, and have made all our data publicly available.</p><p id="p-70">The principal difficulty we faced, which made it challenging to produce a high-resolution genetic linkage map, order and orient scaffolds along the length of chromosomes, and carry out QTL mapping at sub-chromosomal resolution, was the surprisingly low frequency of crossovers we observed in our data. Multiple lines of evidence indicated this observation was due to rare intra-chromosomal recombination in individuals from our mapping population, and was not the result of inaccurate genotype data. While previous genetic mapping studies in <em>A. grisella</em> also indicate a relatively short map length, indicative of low crossover frequency, given the modest sample of genotypes interrogated by laboratory mapping studies, we cannot be confident our results highlight a species-wide phenomenon. Using the resources we outline in the present study future investigations could examine crossover frequency in backcross or F<sub>2</sub> populations derived from an array of <em>A. grisella</em> strains, or perhaps directly examine recombination rate in outbred, wild-caught individuals. If low effective crossover rates are a feature of the species, then whatever the mechanistic basis behind this phenomenon, genetic dissection of male song in <em>A. grisella</em> will continue to be challenging. Advanced generation recombinant mapping populations will need to be established by multiple generations of interbreeding in order to produce a mapping population harboring larger numbers of recombination breakpoints throughout the physical length of the genome. To take advantage of these additional crossovers, long-read, single molecule sequencing would additionally be desirable. Such data would lead to a more contiguous genome, would allow markers to be physically ordered along chromosomes, and would allow true interval QTL mapping techniques to be employed.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-71">Phenotyping and DNA isolation was supported by NSF IOB-0516634 to M.D.G. and J.M.G. Sequencing data for this project was generated by the KU Genome Sequencing Core (supported by NIGMS award P20GM103638), and data analysis was facilitated by the Kansas IDeA Network of Biomedical Research Excellence (supported by NIGMS award P20GM103418). Additional support for the project was provided by NIH R01 GM085260 to S.J.M. and NIH R01 OD010974 to S.J.M. and Anthony D. Long (UC Irvine).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-12"><p id="p-72">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8072405">https://doi.org/10.25387/g3.8072405</a>.</p></li><li class="fn" id="fn-13"><p id="p-73"><em>Communicating editor: P. Morrell</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-12"><span class="received-label">Received </span>February 12, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-15"><span class="accepted-label">Accepted </span>May 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Koseva <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.1" data-doi="10.1038/ncomms5737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahola</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehtonen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somervuo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmela</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koskinen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Glanville fritillary genome retains an ancient karyotype and reveals selective chromosomal fusions in Lepidoptera.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4737</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms5737</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D4737%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5737%26rft_id%253Dinfo%253Apmid%252F25189940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25189940&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.2" data-doi="10.1007/s00265-011-1219-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alem</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koselj</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siemers</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bat predation and the evolution of leks in acoustic moths.</span> <abbr class="cit-jnl-abbrev">Behav. Ecol. Sociobiol.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00265-011-1219-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BEcol.%2BSociobiol.%26rft.volume%253D65%26rft.spage%253D2105%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00265-011-1219-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00265-011-1219-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.3" data-doi="10.1111/jeb.12252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alem</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streiff</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenboudji</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limousin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic architecture of sensory exploitation: QTL mapping of female and male receiver traits in an acoustic moth.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2581</span>–<span class="cit-lpage">2596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12252</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D26%26rft.spage%253D2581%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12252%26rft_id%253Dinfo%253Apmid%252F24118224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24118224&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.4" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.5" data-doi="10.1101/gr.115402.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erezyilmaz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mast</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.115402.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115402.110%26rft_id%253Dinfo%253Apmid%252F21233398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.6" data-doi="10.1038/ng.3852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babb</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahens</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa-Garhwal</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The <em>Nephila clavipes</em> genome highlights the diversity of spider silk genes and their complex expression.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">903</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3852</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3852&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.7" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.8" data-doi="10.1093/gbe/evx280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baldwin-Brown</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A new standard for crustacean genomes: the highly contiguous, annotated genome assembly of the clam shrimp <em>Eulimnadia texana</em> reveals HOX gene order and identifies the sex chromosome.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evx280</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.9" data-doi="10.1098/rsbl.2013.0161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawahara</span>  <span class="cit-name-given-names">A. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hawkmoths produce anti-bat ultrasound.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">20130161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsbl.2013.0161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2013.0161%26rft_id%253Dinfo%253Apmid%252F23825084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2013.0161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23825084&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.7.2349.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> The power and deceit of QTL experiments: Lessons from comparative QTL studies, pp. <span class="cit-fpage">250</span>–<span class="cit-lpage">266</span> in <em>Proceedings of the Forty-Ninth Annual Corn and Sorghum Industry Research Conference</em>, <span class="cit-publ-name">American Seed Trade Association</span>, <span class="cit-publ-loc">Chicago, IL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.11" data-doi="10.1007/BF00215714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albertsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fincher</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00226242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00215714%26rft_id%253Dinfo%253Apmid%252F24202349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00215714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24202349&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.12" data-doi="10.1111/j.1420-9101.2004.00716.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">L. S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Condition-dependent traits and the capture of genetic variance in male advertisement song.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">828</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1420-9101.2004.00716.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DBrandt%26rft.auinit1%253DL.%2BS.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D821%26rft.epage%253D828%26rft.atitle%253DCondition-dependent%2Btraits%2Band%2Bthe%2Bcapture%2Bof%2Bgenetic%2Bvariance%2Bin%2Bmale%2Badvertisement%2Bsong.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2004.00716.x%26rft_id%253Dinfo%253Apmid%252F15271081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2004.00716.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15271081&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222052100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2349.13" data-doi="10.1007/978-0-387-92125-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">A Guide to QTL Mapping with R/qtl</span>, <span class="cit-publ-name">Springer New York</span>, <span class="cit-publ-loc">New York, NY</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-92125-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2BGuide%2Bto%2BQTL%2BMapping%2Bwith%2BR%252Fqtl%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-92125-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-92125-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.14" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: Architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.15" data-doi="10.1016/j.ympev.2016.05.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Card</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schield</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbin</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phylogeographic and population genetic analyses reveal multiple species of Boa and independent origins of insular dwarfism.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2016.05.034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D102%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2016.05.034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2016.05.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.16" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <em>Stacks</em>: Building and genotyping loci <em>de novo</em> from short-read sequences. G3: (Bethesda) 1: 171–182.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.000240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.17" data-doi="10.1038/s41588-017-0010-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanKuren</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalsow</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Hidden genetic variation shapes the structure of functional elements in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-017-0010-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-017-0010-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-017-0010-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.18" data-doi="10.1093/bioinformatics/btt310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medvedev</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Informed and automated <em>k</em>-mer size selection for genome assembly.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt310%26rft_id%253Dinfo%253Apmid%252F23732276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23732276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329059700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.19" data-doi="10.1534/g3.113.008185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamm</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cridland</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome of <em>Drosophila suzukii</em>, the spotted wing <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2257</span>–<span class="cit-lpage">2271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008185%26rft_id%253Dinfo%253Apmid%252F24142924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTIvMjI1NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.20" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.21" data-doi="10.1038/sj.hdy.6885540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhold</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Quantitative genetics of ultrasonic advertisement signalling in the Lesser Waxmoth <em>Achroia grisella</em> (Lepidoptera: Pyralidae).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">651</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2540.1999.00554.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D83%26rft.spage%253D644%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6885540%26rft_id%253Dinfo%253Apmid%252F10651908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6885540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10651908&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085242700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.22" data-doi="10.1146/annurev-ento-121510-133537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conner</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcoran</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sound strategies: The 65-million-year-old battle between bats and insects.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-ento-121510-133537</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.aulast%253DConner%26rft.auinit1%253DW.%2BE.%26rft.volume%253D57%26rft.spage%253D21%26rft.epage%253D39%26rft.atitle%253DSound%2Bstrategies%253A%2Bthe%2B65-million-year-old%2Bbattle%2Bbetween%2Bbats%2Band%2Binsects.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ento-121510-133537%26rft_id%253Dinfo%253Apmid%252F21888517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ento-121510-133537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21888517&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299834000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Advanced intercross lines, an experimental population for fine genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1199%26rft.epage%253D1207%26rft.atitle%253DAdvanced%2Bintercross%2Blines%252C%2Ban%2Bexperimental%2Bpopulation%2Bfor%2Bfine%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F8582624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzExOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.24" data-doi="10.1534/g3.115.023655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chouteau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maroja</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Major improvements to the <em>Heliconius melpomene</em> genome assembly used to confirm 10 chromosome fusion events in 6 million years of butterfly evolution.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">708</span> (erratum: G3 (Bethesda) 6: 1489).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.023655</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.023655%26rft_id%253Dinfo%253Apmid%252F26772750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMy82OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.25" data-doi="10.1186/1471-2105-10-67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eilbeck</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative measures for the management and comparison of annotated genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-67</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEilbeck%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.spage%253D67%26rft.epage%253D67%26rft.atitle%253DQuantitative%2Bmeasures%2Bfor%2Bthe%2Bmanagement%2Band%2Bcomparison%2Bof%2Bannotated%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-67%26rft_id%253Dinfo%253Apmid%252F19236712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-67&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19236712&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.26" data-doi="10.1016/j.tree.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing and population genomics in non-model organisms.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2013.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D29%26rft.spage%253D51%26rft.atitle%253DGENOME%2BSEQUENCING%2BAND%2BPOPULATION%2BGENOMICS%2BIN%2BNON-MODEL%2BORGANISMS%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139972&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330148000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.27" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.28" data-doi="10.1093/jhered/esp086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Genome 10K Community of Scientists</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome 10K: A proposal to obtain whole-genome sequence for 10,000 vertebrate species.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">674</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esp086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesp086%26rft_id%253Dinfo%253Apmid%252F19892720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esp086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19892720&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271817600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.29" data-doi="10.1371/journal.pone.0147014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gleason</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development of a genomic resource and quantitative trait loci mapping of male calling traits in the Lesser Wax Moth, <em>Achroia grisella</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0147014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0147014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0147014%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0147014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0147014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.30" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.31" data-doi="10.2307/3494597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Moth sex pheromones: An evolutionary perspective.</span> <abbr class="cit-jnl-abbrev">Fla. Entomol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/3494597</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFla.%2BEntomol.%26rft.volume%253D64%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3494597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3494597&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LM04100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.32" data-doi="10.1163/156853983X00534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffelt</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Reproductive behaviour of the Lesser Waxmoth, <em>Achroia grisella</em> (Pyralidae: Galleriinae): Signalling, pair formation, male interactions, and mate guarding.</span> <abbr class="cit-jnl-abbrev">Behaviour</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1163/156853983X00534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehaviour%26rft.volume%253D84%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1163%252F156853983X00534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1163/156853983X00534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.33" data-doi="10.1038/nature11041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Heliconius Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Butterfly genome reveals promiscuous exchange of mimicry adaptations among species.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">487</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDasmahapatra%26rft.auinit1%253DK.%2BK.%26rft.volume%253D487%26rft.issue%253D7405%26rft.spage%253D94%26rft.epage%253D98%26rft.atitle%253DButterfly%2Bgenome%2Breveals%2Bpromiscuous%2Bexchange%2Bof%2Bmimicry%2Badaptations%2Bamong%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11041%26rft_id%253Dinfo%253Apmid%252F22722851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22722851&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305982900057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.34" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.35" data-doi="10.1093/jhered/est050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">i5K Consortium</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The i5K Initiative: Advancing arthropod genomics for knowledge, human health, agriculture, and the environment.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">600</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/est050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fest050%26rft_id%253Dinfo%253Apmid%252F23940263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/est050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23940263&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323294400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.36" data-doi="10.1006/anbe.1996.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Ultrasonic communication and sexual selection in wax moths: female choice based on energy and asynchrony of male signals.</span> <abbr class="cit-jnl-abbrev">Anim. Behav.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1095</span>–<span class="cit-lpage">1106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/anbe.1996.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BBehav.%26rft.volume%253D51%26rft.spage%253D1095%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fanbe.1996.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/anbe.1996.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UQ64900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.37" data-doi="10.2307/2411308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Absolute <em>vs.</em> relative measurements of sexual selection: Assessing the contributions of ultrasonic signal characters to mate attraction in Lesser Wax Moths, <em>Achroia grisella</em> (Lepidoptera: Pyralidae).</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1383</span>–<span class="cit-lpage">1393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1998.tb02020.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D52%26rft.spage%253D1383%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076931100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.38" data-doi="10.1146/annurev-animal-021815-111216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Perspectives from the avian phylogenomics project: Questions that can be answered with sequencing all genomes of a vertebrate class.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-animal-021815-111216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D4%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-021815-111216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-021815-111216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.39" data-doi="10.1186/gb-2010-11-4-116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quake: Quality-aware detection and correction of sequencing errors.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2010-11-11-r116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKelley%26rft.auinit1%253DD.%2BR.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253DR116%26rft.epage%253DR116%26rft.atitle%253DQuake%253A%2Bquality-aware%2Bdetection%2Band%2Bcorrection%2Bof%2Bsequencing%2Berrors.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-4-116%26rft_id%253Dinfo%253Apmid%252F21114842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-4-116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21114842&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.40" data-doi="10.1146/annurev-animal-090414-014900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koepfli</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paten</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth cit-collab">Genome 10K Community of Scientists</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Genome 10K Project: A way forward.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-animal-090414-014900</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D3%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-090414-014900%26rft_id%253Dinfo%253Apmid%252F25689317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-090414-014900&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689317&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.41" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.42" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1943.tb02321.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.43" data-doi="10.1111/nph.14759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koseva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mort</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The genetic breakdown of sporophytic self-incompatibility in <em>Tolpis coronopifolia</em> (Asteraceae).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">1256</span>–<span class="cit-lpage">1267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14759</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D216%26rft.spage%253D1256%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.45" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.46" data-doi="10.1073/pnas.81.11.3443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lathrop</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalouel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ott</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Strategies for multilocus linkage analysis in humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">3443</span>–<span class="cit-lpage">3446</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.81.11.3443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLathrop%26rft.auinit1%253DG.%2BM.%26rft.volume%253D81%26rft.issue%253D11%26rft.spage%253D3443%26rft.epage%253D3446%26rft.atitle%253DStrategies%2Bfor%2Bmultilocus%2Blinkage%2Banalysis%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.11.3443%26rft_id%253Dinfo%253Apmid%252F6587361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODEvMTEvMzQ0MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.47" data-doi="10.1242/jeb.035345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limousin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evaluation of amplitude in male song: Female waxmoths respond to fortissimo notes.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">4091</span>–<span class="cit-lpage">4100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.035345</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DLimousin%26rft.auinit1%253DD.%26rft.volume%253D212%26rft.issue%253D24%26rft.spage%253D4091%26rft.epage%253D4100%26rft.atitle%253DEvaluation%2Bof%2Bamplitude%2Bin%2Bmale%2Bsong%253A%2Bfemale%2Bwaxmoths%2Brespond%2Bto%2Bfortissimo%2Bnotes%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.035345%26rft_id%253Dinfo%253Apmid%252F19946089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjExOiIyMTIvMjQvNDA5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.48" data-doi="10.1371/journal.pone.0044554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limousin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streiff</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupuy</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alem</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of sexual selection: QTL mapping of male song and female receiver traits in an acoustic moth.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e44554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0044554</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLimousin%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De44554%26rft.epage%253De44554%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bsexual%2Bselection%253A%2BQTL%2Bmapping%2Bof%2Bmale%2Bsong%2Band%2Bfemale%2Breceiver%2Btraits%2Bin%2Ban%2Bacoustic%2Bmoth.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0044554%26rft_id%253Dinfo%253Apmid%252F22957082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0044554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22957082&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.49" data-doi="10.1146/annurev.genet.35.102401.090633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The genetic architecture of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.35.102401.090633</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%2BC.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D303%26rft.epage%253D339%26rft.atitle%253DTHE%2BGENETIC%2BARCHITECTURE%2BOF%2BQUANTITATIVE%2BTRAITS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.35.102401.090633%26rft_id%253Dinfo%253Apmid%252F11700286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.35.102401.090633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172991500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.50" data-doi="10.1111/evo.12698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monnahan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colicchio</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A genomic selection component analysis characterizes migration‐selection balance.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1713</span>–<span class="cit-lpage">1727</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D1713%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12698%26rft_id%253Dinfo%253Apmid%252F26082096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26082096&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.51" data-doi="10.1038/s41598-017-00246-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Construction of a high-density genetic map and quantitative trait locus mapping in the manila clam <em>Ruditapes philippinarum</em>.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">229</span> (erratum: Sci. Rep. 8: 11953).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-00246-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-00246-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-00246-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.52" data-doi="10.1093/bioinformatics/btm071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CEGMA: A pipeline to accurately annotate core genes in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm071</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm071%26rft_id%253Dinfo%253Apmid%252F17332020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17332020&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246773300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.53" data-doi="10.1093/nar/gkn916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Assessing the gene space in draft genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">297</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkn916</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn916%26rft_id%253Dinfo%253Apmid%252F19042974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19042974&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262335700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.54" data-doi="10.1073/pnas.171285098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pevzner</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">An Eulerian path approach to DNA fragment assembly.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">9748</span>–<span class="cit-lpage">9753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.171285098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPevzner%26rft.auinit1%253DP.%2BA.%26rft.volume%253D98%26rft.issue%253D17%26rft.spage%253D9748%26rft.epage%253D9753%26rft.atitle%253DAn%2BEulerian%2Bpath%2Bapproach%2Bto%2BDNA%2Bfragment%2Bassembly%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.171285098%26rft_id%253Dinfo%253Apmid%252F11504945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTcvOTc0OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2349.55"><div class="cit-metadata unstructured">R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.R-project.org/">https://www.R-project.org/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.56" data-doi="10.1093/bioinformatics/btt563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehtonen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auvinen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lep-MAP: Fast and accurate linkage map construction for large SNP datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3128</span>–<span class="cit-lpage">3134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt563</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt563%26rft_id%253Dinfo%253Apmid%252F24078685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24078685&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327714700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.57" data-doi="10.1093/gbe/evv250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calboli</span>  <span class="cit-name-given-names">F. C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of ultradense linkage maps with Lep-MAP2: Stickleback F2 recombinant crosses as an example.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv250%26rft_id%253Dinfo%253Apmid%252F26668116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26668116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.58" data-doi="10.1126/science.331.6023.1386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell-Miramontes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Creating a buzz about insect genomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">1386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.331.6023.1386</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRobinson%26rft.auinit1%253DG.%2BE.%26rft.volume%253D331%26rft.issue%253D6023%26rft.spage%253D1386%26rft.epage%253D1386%26rft.atitle%253DCreating%2Ba%2BBuzz%2BAbout%2BInsect%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.331.6023.1386%26rft_id%253Dinfo%253Apmid%252F21415334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzEvNjAyMy8xMzg2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.59" data-doi="10.1101/gr.131383.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magoc</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAGE: A critical evaluation of genome assemblies and assembly algorithms.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.131383.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131383.111%26rft_id%253Dinfo%253Apmid%252F22147368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.60" data-doi="10.1007/s00335-006-0090-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satagopan</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">R/qtlDesign: Inbred line cross experimental design.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-006-0090-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DSen%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D87%26rft.epage%253D93%26rft.atitle%253DR%252FqtlDesign%253A%2Binbred%2Bline%2Bcross%2Bexperimental%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-006-0090-y%26rft_id%253Dinfo%253Apmid%252F17347894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-006-0090-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17347894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245627200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.61" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.62" data-doi="10.1101/gr.089532.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">ABySS: A parallel assembler for short read sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.089532.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089532.108%26rft_id%253Dinfo%253Apmid%252F19251739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTExNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNy8yMzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2349.63"><div class="cit-metadata unstructured">Smit, A. F. A., and R. Hubley, 2008–2015 RepeatModeler Open-1.0. Repeat Masker Website: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.64" data-doi="10.1111/j.1365-3032.1984.tb00684.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spangler</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenfield</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takessian</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Ultrasonic mate calling in the Lesser Wax Moth.</span> <abbr class="cit-jnl-abbrev">Physiol. Entomol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-3032.1984.tb00684.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BEntomol.%26rft.volume%253D9%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3032.1984.tb00684.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3032.1984.tb00684.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984SH81300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.65" data-doi="10.1093/nar/gki458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">AUGUSTUS: A web server for gene prediction in eukaryotes that allows user-defined constraints.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">W465</span>–<span class="cit-lpage">W467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gki458</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW465%26rft.epage%253DW467%26rft.atitle%253DAUGUSTUS%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bgene%2Bprediction%2Bin%2Beukaryotes%2Bthat%2Ballows%2Buser-defined%2Bconstraints%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki458%26rft_id%253Dinfo%253Apmid%252F15980513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980513&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400094&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.66" data-doi="10.1111/j.1601-5223.1973.tb01134.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suomalainen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">J. R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Achiasmatic oogenesis in the Heliconiine butterflies.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-5223.1973.tb01134.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.volume%253D74%26rft.spage%253D302%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1973.tb01134.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.1973.tb01134.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1973R187800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.67" data-doi="10.1186/s13059-014-0573-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ALLMAPS: Robust scaffold ordering based on multiple maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0573-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0573-1%26rft_id%253Dinfo%253Apmid%252F25583564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0573-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583564&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.68" data-doi="10.1093/nar/gks1116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tegenfeldt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">OrthoDB: A hierarchical catalog of animal, fungal and bacterial orthologs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D358</span>–<span class="cit-lpage">D365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks1116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1116%26rft_id%253Dinfo%253Apmid%252F23180791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23180791&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.69" data-doi="10.1126/science.1102210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A draft sequence for the genome of the domesticated silkworm (<em>Bombyx mori</em>).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">1937</span>–<span class="cit-lpage">1940</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1102210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBiology%2Banalysis%2Bgroup%26rft.auinit1%253D%2B%26rft.volume%253D306%26rft.issue%253D5703%26rft.spage%253D1937%26rft.epage%253D1940%26rft.atitle%253DA%2BDraft%2BSequence%2Bfor%2Bthe%2BGenome%2Bof%2Bthe%2BDomesticated%2BSilkworm%2B%2528Bombyx%2Bmori%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1102210%26rft_id%253Dinfo%253Apmid%252F15591204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTcwMy8xOTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIzNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.70" data-doi="10.1073/pnas.90.23.10972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10972</span>–<span class="cit-lpage">10976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.90.23.10972</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D90%26rft.issue%253D23%26rft.spage%253D10972%26rft.epage%253D10976%26rft.atitle%253DTheoretical%2Bbasis%2Bfor%2Bseparation%2Bof%2Bmultiple%2Blinked%2Bgene%2Beffects%2Bin%2Bmapping%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.23.10972%26rft_id%253Dinfo%253Apmid%252F8248199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTAvMjMvMTA5NzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1457%26rft.epage%253D1468%26rft.atitle%253DPrecision%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.72" data-doi="10.1016/j.cell.2011.09.052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merlin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boore</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reppert</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Monarch butterfly genome yields insights into long-distance migration.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1171</span>–<span class="cit-lpage">1185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2011.09.052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZhan%26rft.auinit1%253DS.%26rft.volume%253D147%26rft.issue%253D5%26rft.spage%253D1171%26rft.epage%253D1185%26rft.atitle%253DThe%2Bmonarch%2Bbutterfly%2Bgenome%2Byields%2Binsights%2Binto%2Blong-distance%2Bmigration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.09.052%26rft_id%253Dinfo%253Apmid%252F22118469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.09.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22118469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297376600026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.73" data-doi="10.1038/522034d"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahbek</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomics: Bird sequencing project takes off.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">522</span>: <span class="cit-fpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/522034d</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D522%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.1038%252F522034d%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/522034d&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.74" data-doi="10.1534/genetics.116.196451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Burg</span>  <span class="cit-name-given-names">K. R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic basis of melanin pigmentation in butterfly wings.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1537</span>–<span class="cit-lpage">1550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.196451</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.196451%26rft_id%253Dinfo%253Apmid%252F28193726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS80LzE1MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjM0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.75" data-doi="10.1111/j.1558-5646.2008.00371.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuster</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettis</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danka</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleason</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reaction norm variants for male calling song in populations of <em>Achroia grisella</em> (Lepidoptera: Pyralidae): Toward a resolution of the lek paradox.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1317</span>–<span class="cit-lpage">1334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2008.00371.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DZhou%26rft.auinit1%253DY.%26rft.volume%253D62%26rft.issue%253D6%26rft.spage%253D1317%26rft.epage%253D1334%26rft.atitle%253DReaction%2Bnorm%2Bvariants%2Bfor%2Bmale%2Bcalling%2Bsong%2Bin%2Bpopulations%2Bof%2BAchroia%2Bgrisella%2B%2528Lepidoptera%253A%2BPyralidae%2529%253A%2Btoward%2Ba%2Bresolution%2Bof%2Bthe%2Blek%2Bparadox.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00371.x%26rft_id%253Dinfo%253Apmid%252F18346222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00371.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18346222&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256356300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2349.76" data-doi="10.1086/512821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zollner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Overcoming the winner’s curse: estimating penetrance parameters from case-control data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/512821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DZollner%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D605%26rft.epage%253D615%26rft.atitle%253DOvercoming%2Bthe%2Bwinner%2527s%2Bcurse%253A%2Bestimating%2Bpenetrance%2Bparameters%2Bfrom%2Bcase-control%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F512821%26rft_id%253Dinfo%253Apmid%252F17357068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/512821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17357068&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2349.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2349.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2337" title="Global Shifts in Gene Expression Profiles Accompanied with Environmental Changes in Cnidarian-Dinoflagellate Endosymbiosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2363" title="Differentially and Co-expressed Genes in Embryo, Germ-Line and Somatic Tissues of Tribolium castaneum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2349" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QRyniWtlqwoU-bwckh0v4Bj3iJ_el9QaqqWC5XO1kMY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2349" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2349" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2349" class="active" data-icon-position="" data-hide-link-title="0">Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth Achroia grisella</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493670" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-69sWBO0EoVU2F4ALEjSHJZmuxbQw18WfzsX2zFGW6_8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2349.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2349" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-A1yRroOrRPWqsh1mRLitlybqzf_IzBGlDy8KI9vMHYk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2349" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2349.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493670" id="node493670--2" data-pisa="ggg;9/7/2349" data-pisa-master="ggg;g3.119.400090" data-apath="/ggg/9/7/2349.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth <em>Achroia grisella</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Boryana S. Koseva</span>, <span class="highwire-citation-author" data-delta="1">Jennifer L. Hackett</span>, <span class="highwire-citation-author" data-delta="2">Yihong Zhou</span>, <span class="highwire-citation-author" data-delta="3">Bethany R. Harris</span>, <span class="highwire-citation-author" data-delta="4">John K. Kelly</span>, <span class="highwire-citation-author" data-delta="5">Michael D. Greenfield</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0544-3404" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jennifer M. Gleason</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9421-002X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stuart J. Macdonald</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2349-2361;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400090 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493670--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Boryana S. Koseva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>K-INBRE Bioinformatics Core, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boryana%2BS.%2BKoseva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koseva%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoryana%2BS.%2BKoseva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jennifer L. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BL.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BL.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yihong Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yihong%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYihong%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bethany R. Harris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bethany%2BR.%2BHarris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harris%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABethany%2BR.%2BHarris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John K. Kelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BK.%2BKelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelly%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BK.%2BKelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael D. Greenfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Equipe Neuro-Ethologie Sensorielle, ENES/Neuro-PSI, CNRS UMR 9197, Université de Lyon/Saint-Etienne, 42023 Saint Etienne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BGreenfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenfield%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BGreenfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jennifer M. Gleason </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BM.%2BGleason%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gleason%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BM.%2BGleason%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0544-3404" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jennifer M. Gleason</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Stuart J. Macdonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computational Biology, University of Kansas, Lawrence, Kansas 66047</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stuart%2BJ.%2BMacdonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macdonald%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStuart%2BJ.%2BMacdonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9421-002X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stuart J. Macdonald</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sjmac@ku.edu" class="" data-icon-position="" data-hide-link-title="0">sjmac@ku.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2349" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2349&amp;title=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2349&amp;title=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2349&amp;title=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2349&amp;text=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2349&amp;title=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2349&amp;t=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2349&amp;title=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2349&amp;title=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493670/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493670" id="node493670--3" data-pisa="ggg;9/7/2349" data-pisa-master="ggg;g3.119.400090" data-apath="/ggg/9/7/2349.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genetic Mapping and Genome Assembly in the Lesser Wax Moth <em>Achroia grisella</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Boryana S. Koseva</span>, <span class="highwire-citation-author" data-delta="1">Jennifer L. Hackett</span>, <span class="highwire-citation-author" data-delta="2">Yihong Zhou</span>, <span class="highwire-citation-author" data-delta="3">Bethany R. Harris</span>, <span class="highwire-citation-author" data-delta="4">John K. Kelly</span>, <span class="highwire-citation-author" data-delta="5">Michael D. Greenfield</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0544-3404" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jennifer M. Gleason</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9421-002X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stuart J. Macdonald</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2349-2361;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400090 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493670--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Boryana S. Koseva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>K-INBRE Bioinformatics Core, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boryana%2BS.%2BKoseva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koseva%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoryana%2BS.%2BKoseva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jennifer L. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BL.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BL.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yihong Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yihong%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYihong%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bethany R. Harris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bethany%2BR.%2BHarris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harris%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABethany%2BR.%2BHarris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John K. Kelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BK.%2BKelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelly%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BK.%2BKelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael D. Greenfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Equipe Neuro-Ethologie Sensorielle, ENES/Neuro-PSI, CNRS UMR 9197, Université de Lyon/Saint-Etienne, 42023 Saint Etienne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BGreenfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenfield%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BGreenfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jennifer M. Gleason </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, Kansas 66045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BM.%2BGleason%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gleason%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BM.%2BGleason%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0544-3404" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jennifer M. Gleason</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Stuart J. Macdonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, University of Kansas, Lawrence, Kansas 66045</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computational Biology, University of Kansas, Lawrence, Kansas 66047</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stuart%2BJ.%2BMacdonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macdonald%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStuart%2BJ.%2BMacdonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9421-002X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stuart J. Macdonald</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sjmac@ku.edu" class="" data-icon-position="" data-hide-link-title="0">sjmac@ku.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2349&amp;rft.epage=2361&amp;rft.atitle=Quantitative%20Genetic%20Mapping%20and%20Genome%20Assembly%20in%20the%20Lesser%20Wax%20Moth%20Achroia%20grisella&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Koseva%2C+Boryana+S.&amp;rft.au=Hackett%2C+Jennifer+L.&amp;rft.au=Zhou%2C+Yihong&amp;rft.au=Harris%2C+Bethany+R.&amp;rft.au=Kelly%2C+John+K.&amp;rft.au=Greenfield%2C+Michael+D.&amp;rft.au=Gleason%2C+Jennifer+M.&amp;rft.au=Macdonald%2C+Stuart+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493670/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493670/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493670/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493670/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493670/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493670/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493670/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493670/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493670/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493670/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493670/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493670/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2349" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2349.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2349.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2349.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
