<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/7/2253" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/7/2253.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/7/2253.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/7/2253.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/7/2253/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/7/2253.atom" />
<meta name="HW.pisa" content="ggg;9/7/2253" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program" />
<meta name="DC.Identifier" content="10.1534/g3.118.200917" />
<meta name="DC.Date" content="2019-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Stewart-Brown et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) has become viable for selection of quantitative traits for which marker-assisted selection has often proven less effective. The potential of GS for soybean was characterized using 483 elite breeding lines, genotyped with BARCSoySNP6K iSelect BeadChips. Cross validation was performed using RR-BLUP and predictive abilities ( r MP) of 0.81, 0.71, and 0.26 for protein, oil, and yield, were achieved at the largest tested training set size. Minimal differences were observed when comparing different marker densities and there appeared to be inflation in r MP due to population structure. For comparison purposes, two additional methods to predict breeding values for lines of four bi-parental populations within the GS dataset were tested. The first method predicted within each bi-parental population (WP method) and utilized a training set of full-sibs of the validation set. The second method utilized a training set of all remaining breeding lines except for full-sibs of the validation set to predict across populations (AP method). The AP method is more practical as the WP method would likely delay the breeding cycle and leverage smaller training sets. Averaging across populations for protein and oil content, r MP for the AP method (0.55, 0.30) approached r MP for the WP method (0.60, 0.52). Though comparable, r MP for yield was low for both AP and WP methods (0.12, 0.13). Based on increases in r MP as training sets increased and the effectiveness of WP vs. AP method, the AP method could potentially improve with larger training sets and increased relatedness between training and validation sets." />
<meta name="DC.Contributor" content="Benjamin B. Stewart-Brown" />
<meta name="DC.Contributor" content="Qijian Song" />
<meta name="DC.Contributor" content="Justin N. Vaughn" />
<meta name="DC.Contributor" content="Zenglu Li" />
<meta name="article:published_time" content="2019-07-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) has become viable for selection of quantitative traits for which marker-assisted selection has often proven less effective. The potential of GS for soybean was characterized using 483 elite breeding lines, genotyped with BARCSoySNP6K iSelect BeadChips. Cross validation was performed using RR-BLUP and predictive abilities (&lt;i&gt;r&lt;/i&gt;&lt;sub&gt;MP&lt;/sub&gt;) of 0.81, 0.71, and 0.26 for protein, oil, and yield, were achieved at the largest tested training set size. Minimal differences were observed when comparing different marker densities and there appeared to be inflation in &lt;i&gt;r&lt;/i&gt;&lt;sub&gt;MP&lt;/sub&gt; due to population structure. For comparison purposes, two additional methods to predict breeding values for lines of four bi-parental populations within the GS dataset were tested. The first method predicted within each bi-parental population (WP method) and utilized a training set of full-sibs of the validation set. The second method utilized a training set of all remaining breeding lines except for full-sibs of the validation set to predict across populations (AP method). The AP method is more practical as the WP method would likely delay the breeding cycle and leverage smaller training sets. Averaging across populations for protein and oil content, &lt;i&gt;r&lt;/i&gt;&lt;sub&gt;MP&lt;/sub&gt; for the AP method (0.55, 0.30) approached &lt;i&gt;r&lt;/i&gt;&lt;sub&gt;MP&lt;/sub&gt; for the WP method (0.60, 0.52). Though comparable, &lt;i&gt;r&lt;/i&gt;&lt;sub&gt;MP&lt;/sub&gt; for yield was low for both AP and WP methods (0.12, 0.13). Based on increases in &lt;i&gt;r&lt;/i&gt;&lt;sub&gt;MP&lt;/sub&gt; as training sets increased and the effectiveness of WP &lt;i&gt;vs.&lt;/i&gt; AP method, the AP method could potentially improve with larger training sets and increased relatedness between training and validation sets.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/07/01" />
<meta name="citation_mjid" content="ggg;9/7/2253" />
<meta name="citation_id" content="9/7/2253" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/7/2253" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/7/2253.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/7/2253.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/7/2253.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200917" />
<meta name="citation_pmid" content="31088906" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="2253" />
<meta name="citation_lastpage" content="2265" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Benjamin B. Stewart-Brown" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7551-9833" />
<meta name="citation_author" content="Qijian Song" />
<meta name="citation_author_institution" content="Soybean Genomics and Improvement Lab, USDA-ARS, Beltsville, MD 20705" />
<meta name="citation_author" content="Justin N. Vaughn" />
<meta name="citation_author_institution" content="Genomics and Bioinformatics Research Unit, USDA-ARS, Center for Applied Genetic Technologies, Athens, GA 30602" />
<meta name="citation_author" content="Zenglu Li" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602" />
<meta name="citation_author_email" content="zli@uga.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4114-9509" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H. J. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="American Soybean Association, 2018 2016 Soy Highlights. The American Soybean Association. http://soystats.com/ 2013-highlights/ (accessed 20 Jan. 2018)." />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. M. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. C. Barrett;citation_author=B. Fry;citation_author=J. Maller;citation_author=M. J. Daly;citation_title=Haploview: analysis and visualization of LD and haplotype maps.;citation_volume=21;citation_year=2005;citation_pmid=15297300;citation_doi=10.1093/bioinformatics/bth457" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Maechler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting linear mixed-effects models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015;citation_doi=10.18637/jss.v067.i01" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=H. R. Boerma;citation_author=R. S. Hussey;citation_author=D. V. Phillips;citation_author=E. D. Wood;citation_author=G. B. Rowan;citation_title=Registration of ‘Boggs’ soybean.;citation_pages=294;citation_volume=40;citation_year=2000;citation_doi=10.2135/cropsci2000.0008rcv" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. A. Brim;citation_title=A Modified Pedigree Method of Selection in Soybeans;citation_pages=220-220;citation_volume=6;citation_year=1966;citation_issue=2;citation_doi=10.2135/cropsci1966.0011183X000600020041x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. A. Brim;citation_author=J. W. Burton;citation_title=Recurrent Selection in Soybeans. II. Selection for Increased Percent Protein in Seeds;citation_pages=494-498;citation_volume=19;citation_year=1979;citation_issue=4;citation_doi=10.2135/cropsci1979.0011183X001900040016x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. C. Brummer;citation_author=G. L. Graef;citation_author=J. Orf;citation_author=J. R. Wilcox;citation_author=R. C. Shoemaker;citation_title=Mapping QTL for Seed Protein and Oil Content in Eight Soybean Populations;citation_pages=370-378;citation_volume=37;citation_year=1997;citation_issue=2;citation_doi=10.2135/cropsci1997.0011183X003700020011x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. W. Burton;citation_author=T. E. Carter;citation_author=M. O. Fountain;citation_author=D. T. Bowman;citation_title=Registration of ‘NC-Raleigh’ soybean.;citation_pages=130;citation_volume=13;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=T. E. Carter;citation_author=J. W. Burton;citation_author=D. T. Bowman;citation_author=Z. Cui;citation_author=X. Zhou;citation_title=Registration of 'N7001' Soybean;citation_pages=1126-1127;citation_volume=43;citation_year=2003;citation_issue=3;citation_doi=10.2135/cropsci2003.1126" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. H. Chung;citation_author=L. Babka;citation_author=G. L. Graef;citation_author=P. E. Staswick;citation_author=D. J. Lee;citation_title=The Seed Protein, Oil, and Yield QTL on Soybean Linkage Group I;citation_pages=1053-1067;citation_volume=43;citation_year=2003;citation_issue=3;citation_doi=10.2135/cropsci2003.1053" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. A. Clark;citation_author=J. M. Hickey;citation_author=H. D. Daetwyler;citation_author=J. van der Werf;citation_title=The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes.;citation_pages=4-4;citation_volume=44;citation_year=2012;citation_pmid=22321529;citation_doi=10.1186/1297-9686-44-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Gene;citation_author=E. Combs;citation_author=R. Bernardo;citation_title=Accuracy of genome-wide selection for different traits with constant population size, heritability, and numbers of markers.;citation_pages=1-7;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Perez;citation_author=J. Hickey;citation_author=J. Burgueno;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. I. de Bakker;citation_author=R. Yelensky;citation_author=I. Pe’er;citation_author=S. B. Gabriel;citation_author=M. J. Daly;citation_title=Efficiency and power in genetic association studies.;citation_pages=1217-1223;citation_volume=37;citation_year=2005;citation_issue=11;citation_pmid=16244653;citation_doi=10.1038/ng1669" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de Los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic heritability: what is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de Los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="de Mendiburu, F., 2017 Agricolae: statistical procedures for agricultural research. R package version 1.2–8. https://CRAN.R-project.org/package=agricolae." />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=J. Dekkers;citation_title=Prediction of response to marker-assisted and genomic selection using selection index theory.;citation_pages=331-341;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076470;citation_doi=10.1111/j.1439-0388.2007.00701.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge Regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genes Genom. Genet.;citation_author=J. B. Endelman;citation_author=J. L. Jannink;citation_title=Shrinkage estimation of the realized relationship matrix. G3;citation_pages=1405-1413;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. G. Guo;citation_author=D. M. Tucker;citation_author=J. W. Lu;citation_author=V. Kishore;citation_author=G. Gray;citation_title=Evaluation of genome-wide selection efficiency in maize nested association mapping populations.;citation_pages=261-275;citation_volume=124;citation_year=2012;citation_issue=2;citation_pmid=21938474;citation_doi=10.1007/s00122-011-1702-9" />
<meta name="citation_reference" content="citation_journal_title=Genes Genom. Genet.;citation_author=Z. Guo;citation_author=D. Wang;citation_author=C. Basten;citation_author=E. Ersoz;citation_author=W. Briggs;citation_title=Accuracy of across-environment genome-wide prediction in maize nested association mapping populations. G3;citation_pages=263-272;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=J. L. Jannink;citation_author=H. Iwata;citation_author=E. Souza;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy for grain quality traits in biparental wheat populations.;citation_pages=2597-2606;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.05.0253" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost.;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_title=Practical Handbook of Soybean Processing and Utilization;citation_author=P. Huth;citation_title=Nutritional aspects of soybean oil and soy protein;citation_year=1995;citation_doi=10.1016/B978-0-935315-63-9.50027-9" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=E. Y. Hwang;citation_author=Q. Song;citation_author=G. Jia;citation_author=J. E. Specht;citation_author=D. L. Hyten;citation_title=A genome-wide association study of seed protein and oil content in soybean.;citation_pages=1;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J. L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Jacobson;citation_author=L. Lian;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=General combining ability model for genomewide selection in a biparental cross.;citation_pages=895-905;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.11.0774" />
<meta name="citation_reference" content="Jain, A., M. Roorkiwal, M. K. Pandey, and R. K. Varshney, 2017 Current status and prospects of genomic selection in legumes, pp. 131–148 in Genomic Selection for Crop Improvement, edited by R. K. Varshney, M. Roorkiwal, and M. E. Sorrells. Springer International Publishing, Cham, Switzerland. 10.1007/978-3-319-63170-7_6" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J. L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. Jarquín;citation_author=K. Kocak;citation_author=L. Posadas;citation_author=K. Hyma;citation_author=J. Jedlicka;citation_title=Genotyping by sequencing for genomic prediction in a soybean breeding population.;citation_pages=740;citation_volume=15;citation_year=2014;citation_pmid=25174348;citation_doi=10.1186/1471-2164-15-740" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="Kassambara, A., 2017 ggpubr: ‘ggplot2’ based publication ready plots. R package version 0.1.6.99. http://www.sthda.com/english/rpkgs/ggpubr." />
<meta name="citation_reference" content="citation_journal_title=Soybean Genet. Newsl.;citation_author=P. Keim;citation_author=R. C. Shoemaker;citation_author=T. C. Olson;citation_title=A rapid protocol for isolating soybean DNA.;citation_pages=150-152;citation_volume=15;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=L. Lian;citation_author=A. Jacobson;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=Genomewide prediction accuracy within 969 maize biparental populations.;citation_pages=1514-1522;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.12.0856" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_title=Advances in Agronomy;citation_title=Genomic selection in plant breeding: knowledge and prospects;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_pages=77-123;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J. L. Jannink;citation_title=Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=Y. Ma;citation_author=J. C. Reif;citation_author=Y. Jiang;citation_author=Z. Wen;citation_author=D. Wang;citation_title=Potential of marker selection to increase prediction accuracy of genomic selection in soybean (Glycine max L.).;citation_pages=113;citation_volume=36;citation_year=2016;citation_pmid=WOS:00038214;citation_doi=10.1007/s11032-016-0504-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. M. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Y. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Riedelsheimer;citation_author=J. B. Endelman;citation_author=M. Stange;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic predictability of interconnected biparental maize populations.;citation_volume=194;citation_year=2013;citation_pmid=23535384;citation_doi=10.1534/genetics.113.150227" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_author=J. Poland;citation_title=Efficient use of historical data for genomic selection: a case study of stem rust resistance in wheat.;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.09.0046" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schmutz;citation_author=S. B. Cannon;citation_author=J. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean.;citation_pages=178-183;citation_volume=463;citation_year=2010;citation_issue=7278;citation_pmid=20075913;citation_doi=10.1038/nature08670" />
<meta name="citation_reference" content="citation_title=Designing soybeans for 21st century markets;citation_author=S. A. Sebastian;citation_author=L. Feng;citation_author=L. C. Kuhlman;citation_title=Accelerated Yield Technology: a platform for marker assisted selection of simple and complex traits;citation_year=2012;citation_doi=10.1016/B978-0-9830791-0-1.50020-0" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.;citation_pages=e54985;citation_volume=8;citation_year=2013;citation_pmid=23372807;citation_doi=10.1371/journal.pone.0054985" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Q. Song;citation_author=J. Jenkins;citation_author=G. Jia;citation_author=D. L. Hyten;citation_author=V. Pantalone;citation_title=Construction of high resolution genetic linkage maps to improve the soybean genome sequence assembly Glyma1.01.;citation_pages=33;citation_volume=17;citation_year=2016;citation_pmid=26739042;citation_doi=10.1186/s12864-015-2344-0" />
<meta name="citation_reference" content="USDA National Agricultural Statistics Service (USDA-NASS), 2017 Quick stats. USDA-NASS, Washington, DC. https://usda.mannlib.cornell.edu/usda/current/Acre/Acre-06–30–2017.pdf (accessed 20 Jan. 2018)." />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. C. Whittaker;citation_author=R. Thompson;citation_author=M. C. Denham;citation_title=Marker-assisted selection using ridge regression.;citation_pages=249-252;citation_volume=75;citation_year=2000;citation_issue=2;citation_pmid=10816982;citation_doi=10.1017/S0016672399004462" />
<meta name="citation_reference" content="citation_journal_title=Genes Genom. Genet.;citation_author=A. Xavier;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=Assessing predictive properties of genome-wide selection in soybeans. G3;citation_pages=2611-2616;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. Zhang;citation_author=H. Wang;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=Y. Liu;citation_title=Effect of trait heritability, training population size and marker density on genomic prediction accuracy estimation in 22 bi-parental tropical maize populations.;citation_pages=1916;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01916" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) has become viable for selection of quantitative traits for which marker-assisted selection has often proven less effective. The potential of GS for soybean was characterized using 483 elite breeding lines, genotyped with BARCSoySNP6K iSelect BeadChips. Cross validation was performed using RR-BLUP and predictive abilities ( r MP) of 0.81, 0.71, and 0.26 for protein, oil, and yield, were achieved at the largest tested training set size. Minimal differences were observed when comparing different marker densities and there appeared to be inflation in r MP due to population structure. For comparison purposes, two additional methods to predict breeding values for lines of four bi-parental populations within the GS dataset were tested. The first method predicted within each bi-parental population (WP method) and utilized a training set of full-sibs of the validation set. The second method utilized a training set of all remaining breeding lines except for full-sibs of the validation set to predict across populations (AP method). The AP method is more practical as the WP method would likely delay the breeding cycle and leverage smaller training sets. Averaging across populations for protein and oil content, r MP for the AP method (0.55, 0.30) approached r MP for the WP method (0.60, 0.52). Though comparable, r MP for yield was low for both AP and WP methods (0.12, 0.13). Based on increases in r MP as training sets increased and the effectiveness of WP vs. AP method, the AP method could potentially improve with larger training sets and increased relatedness between training and validation sets." />
<meta name="og-title" property="og:title" content="Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/7/2253" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) has become viable for selection of quantitative traits for which marker-assisted selection has often proven less effective. The potential of GS for soybean was characterized using 483 elite breeding lines, genotyped with BARCSoySNP6K iSelect BeadChips. Cross validation was performed using RR-BLUP and predictive abilities ( r MP) of 0.81, 0.71, and 0.26 for protein, oil, and yield, were achieved at the largest tested training set size. Minimal differences were observed when comparing different marker densities and there appeared to be inflation in r MP due to population structure. For comparison purposes, two additional methods to predict breeding values for lines of four bi-parental populations within the GS dataset were tested. The first method predicted within each bi-parental population (WP method) and utilized a training set of full-sibs of the validation set. The second method utilized a training set of all remaining breeding lines except for full-sibs of the validation set to predict across populations (AP method). The AP method is more practical as the WP method would likely delay the breeding cycle and leverage smaller training sets. Averaging across populations for protein and oil content, r MP for the AP method (0.55, 0.30) approached r MP for the WP method (0.60, 0.52). Though comparable, r MP for yield was low for both AP and WP methods (0.12, 0.13). Based on increases in r MP as training sets increased and the effectiveness of WP vs. AP method, the AP method could potentially improve with larger training sets and increased relatedness between training and validation sets." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7.cover-source.jpg" />
    <title>Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zOK29458e21O6SCLtm0ETdkLyQAyzveO-reUJS12t3A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493678","apath":"\/ggg\/9\/7\/2253.atom","pisa":"ggg;9\/7\/2253","ac":{"\/ggg\/9\/7\/2253.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/7\/2253.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2253"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/7\/2253"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/7\/2253":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+hn0t5WlUvGT54T05aD49sZmTJcev8iIuAZWYrPLHCIa1qYlGG7cPnhf7kWFI8VvUz+h9CSboSr9sYkAksu+4isI2Oo5kvmpKpGnpqohm6Bq3yRXpM\\\\\\\/TpS6QFnW3DCg8oQWCvvU+XjqsDs\\\\\\\/aNh8YjiqGB5QHFZ0NtORPwCVxYC7lrNbpciIm\\\\\\\/oeKeumb6bSx9KTenjvG6CsBEA2wYv6EoGeIavqwVxiUrpD79ehLldSC\\\\\\\/zFwC30i\\\\\\\/SajOhcAyxorLEyGst7Diw67KKqkwOUqq5pslmMGhA\\\\\\\/T9gaJbhp0B+HNCwoFLOlX5y0fvyguHPer6hZvCkXttZbdMWsNOv2VKQt7c+7eocOQbxPEyrou8bk8LnXkRCcABoXJOwRlbg2izJzarIIbNjzIGiqkCOw8hRpHdTC\\\\\\\/9Jxl3dSDdSQzDKf0uqvmOiRd8PfXOjXB2asHUhr4e2kzFzthjppCE392gAd0X8rLu7PiBV0RxesLrQU2tTqql4KVcog2kL\\\\\\\/7PnwZBOf7sOm2gUhsSOACzRYuRjf9glv7aU3LJb8Ih0QbONyhbkzNn4LDlNzch\\\\\\\/ROKCgZTgS6jahoR5nqlJcY80mEjGq\\\\\\\/xm9FTaDzXBiz3LT+boEvkBqGDK955SAV9AvVSCK5bG2MU0rQSQGqz6Wp7ucmoBP\\\\\\\/ln2w4jjGe9Oun0BdGizstNM5vggiqKrT3Pf8Bx1M5D8yLqujE2ejl81KVMyQa3gRi82rJcXCwgm+JAEPq09fGjrLNUeYv4kdweYndWzbhavvf0vhHov58JWAmfDcgSsAxO6BqRyTHI55GUO4mYIfAY2dyugrz6oKG0H6Vq5Wd52UJ73L9YTKG80d3luK2RaGt46RJcGzaZtMs6ms7LxWQULmcqT+NBl5yjRkVxKxRD4B7jD3WqELCg\\\\\\\/FtZEaddwzlL2RMpavocDliICarhkkrpL7aYTlPOBcgAPMSl7IszaNmlCLaAfRJo30r+jWzYx3Vtl86d7lE5yKoj2artEMHTbLR3Oz3+\\\\\\\/6YTMCHRor6nIVJ59VAQCLcpAOQEbUV1XcC0B8HhJkLKzugSob\\\\\\\/Q+BsOSVNkkjYL2DcQw1JVfuD7lc8PRRzgnbsor2apLhHO8+LcD8wPWjTeQkOQMl+llGSLtZVN3D4K8oQRYseSqUXeZOM4i0C0Gen65jq2D1tuIrOKD2W0i\\\\\\\/VdB+ScrvTcal4XDGI+uccdoIlPMXIU3Oo64DWisWD8L7O8JG9gp7xSevtcBKX2IgKuloyjXKkMh+RYavVKU6s\\\\\\\/2XKi6TLoJb\\\u0022,\\\u0022iv\\\u0022:\\\u0022XI70GW1ZNYuxf4JUlGDALg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d45227a7171815689705e7550774a06e809e7be795037b655f0f31b30748adfb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229GjSFH\\\\\\\/6eb3gQe84sbniZTUUXawuLzUqc8GsSxkQj3fWbV+8gL+QQVFFuTlygn8EvaLQ7Tiyu6Dj0yBNObJNE37wl\\\\\\\/pDZejz1vzOlLW1CXuuuj7\\\\\\\/lVWeEONr1yzH2CXy2s9LbI+H7\\\\\\\/3mhEnqQJB\\\\\\\/cZhU6Fo1SLzTtJEjyYji\\\\\\\/iaCCf8ZS687Iz14L7zLsadoxlI+JiejWmuWzBRwNWyz79H95a7okwG52whq+FjUR7cTqIjTlAkDZgtgs8u4CJuaGKcQiBd+eXoyBhD59EDl0AdJu\\\\\\\/p7PsZz601dB\\\\\\\/qoWXetHLVyR9dx\\\\\\\/hPQU3lTgP3AuECDwFlv6UDeH\\\\\\\/Q3WkZ9HGthd\\\\\\\/\\\\\\\/QLNROcsVzPhkB8uquzuIgJPfKwi6x00NVsPRUvmGZIP8oq1H7pSt09ow9D2ORTzTOsOfeTfPI1DGtkdRolz8ymFVvUuoqd\\\\\\\/c+XpgizpXsRwgaBxHvEp86SJ5LcEGDvNWzoF0Cr\\\\\\\/JMOKSwncv\\\\\\\/zdKJgl05q\\\\\\\/ph1suFBMKPsjihATg6TxEaeB8aNRvbrMdOP3bHK9wzeVmssg\\\\\\\/Ias5Wv+LLL+7UoegQ6ovXl1kLXuE9YGdIXqffAcSnQnN+GMsoydtfl664EZuLuz73LdpuzR93tztmDehkXpUDXNS1ShycBPTylAxGNhaXD3jEWAyh88oOhSYOLEZqbVKm4F0tL5ZGb7OkmB0rG+7f7j2Zhes\\\\\\\/QIWcoNNoqVkZeBdgiOifC+teVTwuduh\\\\\\\/p+RVaqeeA7uVmaiT16a5ehYP4LiJ+dPm8pP412vSrsmxWrs125yclih49lf7rv6rc0vsuows8CTVmn17OEk8c8sAy6EZUxGsaE2FutsSnlqbMEK8rVuBJegToUaAD1HK2C61mgm2RBfeXaMcdiRgOczMmTq5JKCpGOJBX7ZCR5YolE2eyoHP68nqQProUzmqKUY9GAHoU7J0\\\\\\\/Ab8UkJIyQ26Q1E\\\\\\\/rSdL1TQErNvgviFiR7b9Xx\\\\\\\/7tnrCRj+J5WM71JKkPRmnwCk3NT0YjjdC5BatYWuiiOby94Roe3r\\\\\\\/zAep3jEIMd+ipT9oE11JFGp83rymresRaEBmxRUqS093T8KgmXKG+DGtrydeIQw\\\\\\\/rdzfy2q2vLtokgmnnKbkao7RvspUtnuKCCHDFGjWj62CplduwUmfknl5PRXIqo6+pjeKJZJSvlSwPZRHpzk4c1GlAlvDS8lS8gPaBLR22K46F9aL1wU9EB3PrjhZlw6EBSdlSI+XJTzuV1CRX7DvLruqd\\\\\\\/4k5GkmjQWnvpL0puDLtOF9u9kK4SusEk4NC+ub\\\\\\\/oX15VUdYbXJa+INHqyicmOQqPWpAT\\\\\\\/BK9+CQByUp8SgO4wKuBrhDpX3fCnUZ7DQ1WQecNGI\\\\\\\/PIfhPd5JH7VROX6J\\\\\\\/L+VYhkAUk8f\\\\\\\/fNt66LAvnBItpJSaPFTzrZ1UG10MZwmIXnR3Wrh89lzTgYcG64rvmgvedLhiaGGq8+OKdDabNX\\\\\\\/H\\\\\\\/hTgJTg2jYKXXWHAeei+sNB4rXCDDt6f97WTEgPgsMxxnkPjwPMtobTP5fck7P6xf7GzGfXmekXC6LfvJJMtnJvym684V4Xnm1svu1LTErqiqk2wYzgT0N3ZIG4GxR51BMa0DDBlnR93EFqrOhwfBUJ4R3jPk9CTHClel31xxhKSPbliagH\\\\\\\/HlaCvmf8WWe8fC1pBdj6Tpv1ZqmiKTGH+xUT+8v30Oh0uKH84KRl6rb3\\\\\\\/mHiISvUiKFPghxZhCGDOZiD6HBw3GwysvAaoEeCE\\\\\\\/a2xSz2Jjnnxf4pJFZnws8k\\\\\\\/Xcd6IXRAFqcd5Xa8QUCRF1FFfNWKQFmF9356\\\\\\\/6MjXSzJWlhlVSMSdNbwr71tg0vKa2XVfPuLwY+uzOTPdnLNdtqwE2upmIEK5M6hfj1r5xZv3GKtlHSXQaE3Sc671rk5acPzytFOnSL+T4l8iCDtLEbeObkzw2GXjjqBm67lbck4hE9nyjRQZ6M+2unmGVXHujz1sI4naq6G0vEsp90HhJ\\\\\\\/Mg\\\\\\\/LGpjK36AQk+jqcIOYyZJbRTPH23vK2Uisc7v\\\\\\\/iKr5zgP8Yxc7ld0WJFgEhiMZumoQaBwKYWlcItEQfge12bQbc\\\\\\\/Epft2r1hQGScyi5pi53Y2YA7NmGhWLRYj8FrV3OmYpjfrPLkj\\\\\\\/jYXGS2aLvvIXCHSYNtfZFCgtGljJZe3xfnx0ZQ55dRskWJCXmKbGQL0siL5CYRLQ0c3ou3Tesz8o8+EoWwq3ji56FVl\\\\\\\/1xN1+\\\\\\\/RzjXWftXnbxVEwg8ocTsC9B1HiAiB1xDzaoSvz67DFQA\\\\\\\/zDMAGgCmXC5SpIiczQbQPJOSeH8pvzEWNuJjEIdjbcX3IdU52vVXM0l+USWInHRjDKL87yrKjhq0xfbeR5p6Gu7Cld8pEAWxvY5arbMsmG767WOsKsSyNO2LEk+RFaW7J\\\\\\\/ZS5IEA8q+rZ462QfTQ0FagMPoVkO87oP7fDvKvX\\\\\\\/DtNO2I1Chb+pkmHjfdOYnaiKBmoCGLtKl7yHQNt8YMT+45LCTB7\\\\\\\/ImMAj5GeV5J0eOwiPOJQ8rxMgHwJJHymd9b5A0nsOADYFeJWTwymgNaGwV9zHe4ZBFKfq50c5eiapjDDdRMYnBN5nAJyJ+LNIaj62vEsr0VAl8alaab8u4nn+dwnTmFb7Qv93SK54drYGPtrOzQ1YNBQ+4LuFDy5lkQRnv77xBy5LjvmcNeVUnJ\\\\\\\/6rtVoommH9NqUY2lXOMRTASxoZkGDaK3Z4Fq7aZHpz3Ah2MzxgXw3dg1juGC7t5F+UlgqHmLMhZ3IP9AValqcPMAIyiB+tF2BSTamsWNyyfQs2U\\\\\\\/L88QJLzpeWMOVtukeHN+gmJriQg3EJg1Y1SiPyiLWu1ph41lYSTehrwGBchwX30jRsnM7sjuAOnwna30VdWAQ\\\\\\\/bhUjMZc7LdwuyydBQYhSICr7r8HBRyM5ZAWVHECTmYoE3llxXUYWaOKZX5ntQ24I9vbtKb00SAsD+yDbKB\\\\\\\/2D2YoOiGPPXf3m40RBlPC1XcbyzJTi21Jc0TAhCevDe0Va3o0Yad9RTXcSb1AheWjIPUNhNVjpL+5zGkgyCcdHHdFi7w6UuLBkDxjDWFkyQXQe5PfMZXZjlqRBO13o0rnsWnilfMZ2sCWL9CzwuMHegIZeczfo5o+o+SMPD+N+vwb6zLvXoCijxSC104lM18AkapxRVVG8rZCYsKsXjM6YZXK3ToIiWFfnQ5OWnToOMII8iMyrgTMlhhu5+1camdKX08YhBy\\\\\\\/HmT100B8qEDyd7MFNHiQbq8HB2FRg8vPw\\\\\\\/ehL7XuuEMFY++iMqNoLk8mgRC5JRXGhagJ+WKnjJVnwLTw7aEWuO73TCftW3IoOtpaYvHT4CFDaUiuQLkWxG+2FGiRY0V0eoY2iQQmLAQzI6ZkK9K+ZjKI9dEmLG\\\\\\\/zW1FXlvkf1APZcQitSZnoPntRHJt4pp+sIeBCTX8wV1BQTDYUAeBiGnT5DVKK92oMGZblMG1mxX5TPmlzjtHQLLiofUklvRBurjJ\\\\\\\/XoQ3bIO0uZ2aU2rwgehOLGJ8kzkqlwb3n8eYpJabU0633WNbh4uMa62VMglkS8ozGsDeYSIIu9Xxs+UsyxL\\\\\\\/6fQz23C+qQ2QwbADuahZz00l3YXoXpdFXYAK5VkGUd1jyDyfIqZ7GhMPlNU9CSNHZlDOjE5pVSzMVjylxgHdUqhUivQiJLTAtmfv4HA4NScPTGu1p43ip0AhHcHYQrk5iLAZY8FavFj7Yqge7WcgVNb4OXUnkKnH+zFu8NDiEpkhVoULulq5UPuDjLQOHlMCKwjCscJa7jwXn4oPrQxWNKi\\\\\\\/v31nGnDl2JvZPDahLrdhndhQ9JkxHSJXYTxbNOXzZ07TW9omxBDOQrJAlsr\\\\\\\/mPqFgwgPggJ5b\\\\\\\/Fn3Y\\\\\\\/WQm6zqItifGuw3eK1LhBYaagvxBRG6aopzNEOL5VSNbPoQSKx\\\\\\\/j+R7jeH1TMZG1p3\\\\\\\/3tFtaOYqILTeP75dKHWDD7v0Uh7MNAEeEiAwXMmqD2Meo6T7EQ1o4PI6YTmfa20gfyglbDv9B91WUFNguDFSD80KLDM7CrHqHYu2U1Au0W844QHDRF01j8tZnOcACCrWuxaI7kBCJ15RjVXL9pgKfmF1NN7miC4PCtRyP5pfRCR8VqvGBnRRVLYh8EGh1Ly5VgcwvhKKGD2VF\\\\\\\/+yZ2\\\\\\\/tcv4dd1C6MWTU1bKnpPDZXAOQ9rFoODuegqcLYYmmdJwlIc6FBbDaXvmpeUjHyB43ITVip+Zk7CfDPAUL8zTbRBV0v10srQ5+whLVgciuqqUf2X6BQ12MrXJ5NyvWFLeUvp1Q\\\\\\\/\\\\\\\/FqXU9dACr1oJE2Iq16Ug0JODiERk8owKBmVhNBgb\\\\\\\/8T\\\\\\\/1bEP\\\\\\\/o9qnjw9tAneNGNBmd2EYCQiAW8+yJf6fdqfNq93EjD9u08x5cq6o4woCHkRTMk+ivuvasAhO0QTzzvRqXF1b\\\\\\\/BMrzbOsFH6vzfgRrlmPHbtIHdRW4+ip15q\\\\\\\/85fR0AVCvBmS6LK15faQnZMTuyt8Ary4mi4KLnqErO7DPs0tAPvexVcmVJhIvDuQq\\\\\\\/7+hS2yA1NVcFfn2+uTOFtV5jzSxYHyg2D6Ljj\\\\\\\/5JybcTrNRRlUiuTaopFWtBZ3wtyFlRsyeSCShbQtWTsReI5AzvFO+Ui74IdO3M4fZA+7by8zYDJfT+1kUBbnwJwh3IKvDNZBS+QqVuXTDXIEVFktqMDCcJ6WDBzzD2yQt6l1fPOVDtjoUq4eUWbEX\\\\\\\/Nf97RYXTdA7h2R+o\\\\\\\/Yo75f6ABB6F271WchWH1mAEZJ\\\\\\\/VbzKYjH4nSI2jg6DWIiEWbVsrYjtuEeLMMU+DrTavS7D+uhLzie35QyH39ZTEkjmcUhSgoVILzt3Mxez6XsGYmqK3DLj83d02Xj\\\\\\\/WCcATwEkrfIgZBrf6i6TpZCd625l0jbPZCUzZN4RvMpiIpNhlPiCI+MNeQrusdGe6uW\\\\\\\/DfaDTF5jOsuBJdvDE0cccol3J\\\\\\\/2HTgUJEwrxRUqERUlbifwbLrUxGdqFEydeDtDimAbuImeibg4eIuzn6rD7+mBSirdMzxVzRb4P5XtnzQG3MP\\\\\\\/GtMOfIVC9VRj+NXNkxzSVNxzj9IoOAXsMN93\\\\\\\/JtFj+vZUqKJ\\\\\\\/sKfeMZh58zbjChz2B3LKUnR0GKkZ7nUJCDPsMxXc5Hr\\\\\\\/P9kyBNV2NaQUx8NlqTKuEft1lqzU+vbPCq1mRuF6WX90LofL6lXV0oT4PTzc1SKIbUPiJp62G+QAplvtZWgx800JamPayBINRUi\\\\\\\/9Ea+ynYhjlyiOuz+TxGj9u1U5Q6PpRv2QTDXdu9+tRJlR+32eFR\\\\\\\/Wz5Gw2ztP6HOGWqdHVNH99cmb395r6ju02dreLhw+i21Zl7Bga+3MxQ8liRUi6favTAO\\\\\\\/WXYpPrWkF6f149+HU7+Uzn5ITcRUIgE5gztEYKuZlwgpqbn3Kpsj6sFDjbsdmaCNEj8D4V5AxYb5b5fzi9XYBbHy+FerI1drSb5f7rsBz4EU20jBIRqSaYNPz9MdvzUh5rpQbLD\\\\\\\/UdklHm4HrlpUjuDdTRd6nHo9hcY6iPvtX1oQjyHx8iwH+6ZJNIYd+1zZ63O4LAB+NYzK4RcHr\\\\\\\/o\\\\\\\/kW75qengarfAhyOA45hKuY9n8dtt6yHnYu5SNORyYgk6INWCg8Igo1mBiIt1Kgnynxz+maETSB4rsKkSEyi+Zvq6Ef3sgkWLE4B0jDrsjIAa\\\\\\\/jGo\\\\\\\/G3NUyPUKCcyeXrs9\\\\\\\/dW+cked2LGEln1q3+p0TsN32ZTxErtz7VVRG8gILoaCqrIm6WSfv2QhwAjTdWBwir2GKlw5VMKQifeBKHEaoq1b83AHz0cQsTDrkqcRztaIJWLT5Y+H10fIytxiIlw1XsLMh0v8OTNX5CYxdIyhxjwGt6UaU6bGk4IZsGrg8CglWZg0ZjOcqhx1lRxEelAs5GwNy2nUOsCssOy48Wz+rag6icRjSwOIMp7dP0HqJWGgr5rqbiOAz7B4QRlcpMNbLc\\\\\\\/7yjio9opVeQ\\\\\\\/V38BWX2RQUew0cKmklHQaW7JDjLGhUlzoD+M8uaJzSjNNY16dyBd6vO\\\\\\\/xs\\\\\\\/HlgCjh8OKZN0WZFlxu5So0DrIfW8uxyb7JRKZnTPsrINasAwkhxNP8ohQWxnbZAGrqGop7+pBzp1QoEYCsIk4LxHYy3SVMkvImPbGTr4QJ6rtdjtgtnn3jmqXALx9NzkMvMVQ8fV8JT50s4up6OSPUdLW3+icUx+2bvuJet8gwIXvcay\\\\\\\/WM3FC94GRLLMju+CNhQIdKp6TJoILh8+WaJZbT8R7\\\\\\\/xpyWL5JAh6CKlbNomPaCg1emkNB4xPLshaShb5vAkKWnpxkXq4ylSWkqm48Khubmf6gbo6b8Qx3NppzvfdMdj4C0l\\\\\\\/ni0ueSountWzLgv15lWBj7xcu5vOjWiDepCcAjBZlAzIgtW3Oj1duJVi0JzLxcmr+NZK1KjC\\\\\\\/rVQYvGMMI+GepAwRNZc4auggM9rbj8af1nTbRHyEasoXAGYri+9RKhddlIVpVC2CuDxs2sly1L+XsKN10UQHGrmt7wHE17YXV1qFJ8pyS6MAgjoGLDFAk\\\\\\\/FpWhKT\\\\\\\/IsBxlKm3io2AHO2+R29ip65QjyKa2pLV7kgOsz2LcNM+sA42ZA9KhuNFBZeNkjSLw1sWSyhO9TzgkfJ3UdvrRgzxprMjOl6X7GiDZ+3rA6AHAtsKLxmYDSo+ZTd1\\\\\\\/vofzON7\\\\\\\/sevpGuM\\\\\\\/QGxDXyjEAdhzTYzXylVHo\\\\\\\/hf6HisoS4Ym44nPGMFO6g05s9SrxM6G5SEfPnqgDPVtbKskzh3zhkbbTqpsJ8ha5rJE1Em2GbWXqZERe221ZGS1\\\\\\\/wnkYR9eaD\\\\\\\/ZoWxbuBfqEE5ryF1SUAjQhxp8SDi6rhnDIgOBTRS1F4q1PXHFJyU+9CHQ4WyhDQdCCkzign\\\\\\\/GGspT3xzs1ZST5wO\\\\\\\/pxEU7tcmTW+f4kkB6GI\\\\\\\/zWgYIdm0viG29f2\\\\\\\/Tzksph3DUkGRTxRJEUUj1BE6f+vQzVqlb5Syf4\\\\\\\/0awhgwTxzObA05KFTRBpnmkantb1qo+Go5uDSegmCMh58pt54TWJCs\\\\\\\/BkRdGI7l\\\\\\\/Ac3pIMARlBbtcV6vATWDXhAQiurPUJrytUpLr38UW1pHujJf4kSzTgho9Dqj7ymyPSSVAV7io2IV3kG3gQk2jxgfwG+eDu1i5WATelnjB1BDuZ0mFVYnvxA3+kkJuS1t+wN86PTbH5X8rqRQOojNbVR4v\\\\\\\/wA0jl2SKKaa+CihuV+TLufn0swLwjVW7DY68k6ix1r4Wo+t9\\\\\\\/sqZ1zz8ge5FPC54wfgRffuVfpWlUw6EPnQCJHTNKK1v\\\\\\\/WXt4oxC+VTRwIQVyZvnqni34TXMGT7w7oiLD5dtV9WBAfobTIcaU1s+Cx\\\\\\\/REMyBUGcLqgcl5QH30Uo835j600Uqd1RTZckJRHdqe2u194Togrf3JPVywdO8aWFTeCR0c7BA4H+vHxZKsyNeXUbAX\\\\\\\/jAPqNPhuLZzWSFi7WgWC1D7Cpnik3ALwib0Hps1SobChXnw62C+IdJFYcH0ntFwtYZfPsw7AbGF9BMsB6W1UXewjfpAE5\\\\\\\/\\\\\\\/WKG0kWlqWIdtXilyKgwqBFdSfW6Llxz9Dirccl+KgLpG58GY2GlK0gaRRfT4zVZ+klSselocOchKzMrUEyMJakohZi20\\\\\\\/\\\\\\\/Pii01HdF8VDbSPgHYir+ToADIxLn3\\\\\\\/y51CWolXB\\\\\\\/zgGT+LeaM5pDZLQdjwGCvC7HOOHL2fqYPAIBnJw1FuQYocKA3tnLpDi02zZK1LgPcaNkwnOWq5R+HWbt+HyxY\\\\\\\/e9I6pZd\\\\\\\/mS3Sx8EwFwYj9CBvl63bt77mFP\\\\\\\/0BMgo2jaDS1XunOFHlEUq0JZcLnfgOjl1a+3ZrNuNu79cKujJcv9lKD7AaW8+fZaeR\\\\\\\/91JTsKjBS4\\\\\\\/d59Lu0aRk6Y0\\\\\\\/9saaqVntT\\\\\\\/W23ycVv7NcD85qIqR7i6g\\\\\\\/UcKHBn0qdDZ71WBs\\\\\\\/wAqH0nXF4bmviP\\\\\\\/g0ZELGZFlHVwYeYIwcbL6O61CaZEL1\\\\\\\/6h1ce53hK9mGIrzZb3Fv2AboddOEZGMQHXlEAZSD5wxE9krLUVsAaM1Tm31W3zGsnx3rJpRMaP08b\\\\\\\/OX2licbtPS6g9MhYz3oAoTVepECUG9DOj0JTsNaQ05aRzF4vC7zxsLMCpCB1Lt+aOydpsnXUf+tXNsq8Q2y3MNFzGJUKCR54Mc0tVjW6DdwiUZFmFcHGQfOG7bNRr\\\\\\\/bQAzp9xsqMcu7ePo\\\\\\\/b3jWLWc+Q1cxoLQIc7sJQe7D5gjvHSbcODWga6ix0i\\\\\\\/mru7dkOZvb762ibNOTqYGO3IL7fzcM8hJtEU4YeiLxmo507l9bQUrBA7FEQomlE3Pu4qxk6221h6\\\\\\\/SG6o3GCl4RmAts0XAmlZzPsGf5IB8wVj1dUdlR9delvf658bvkWy505mvTOwYb9Rer2Q\\\\\\\/mFS7a8\\\\\\\/IpzVByt6vMDWx2szYW8v00loWN0qMPSKa18Gr2zfvv2qBfMctbPBKOn0DLn7NRsggk4hqrqAi9BOoKJY2oYa6zXeniot6ARPOCai43V7h04saHKWi1Xnpgl3IqMsu4A1LWAO3g1NU1W\\\\\\\/Duka86xFNjLqp9t3WIIPYvS0KKMNhcDjY1IqB5M6T4096k3WbOjkpB3hStFhKTBVAeGniDDjWf6QtZkT8jqiTPQhyfXZAP334lpwgKRuOPdgnWL7cj5KhH7cJq+4Nm86g\\\\\\\/1MJN7q6tvPpzeqmUyifT\\\\\\\/mUEcoWBxwtkmeiECJCwtBS\\\\\\\/etPBIuKCprVL4Mgcb41M41cwxSfa1E\\\\\\\/fB+oWPYYmxE1vs+ZJC6etbZfIspVcG02RYAuTCkDYT+QwA4y54Jo+j\\\\\\\/BiMWLihfKPJq8XNTu75rdKKZyPqp\\\\\\\/4vNPwUXwdToKYHxNaQzP5dLn5N3fsLSu8gs3Wsl1wfliqSoRcV5erA2V2fKykMA\\\\\\\/7H9k8\\\\\\\/2\\\\\\\/NY0MVkps3PJpfK8UrT+ekOwakbFLsdOLueLTSk5Kxd\\\\\\\/22e3UBCnKmkpktum4eRUe2xt3imhIcPNZHpu7MwqZFv62eSMvkq3ckLzjK7JCetmlQfrbMoJrXwSrogt9umYLkZBGmmlSbck++qUEE5UwPfmPZhMoGr5esf5mrT5eeg0a66X+XGp4XHfVcKNe23TFSE3pb2rE4g602MTw7ktiCBm10\\\\\\\/djAXK1sOXR+mz11SiNMJaJSC5xv8bCFrGfDV7dIq7AlFJOmShRk2ICBE4fP4ZLyUUDgyf1w4Ud31IRlsAGHCDejfhiojPwLkDSBEytdvYaoccdb5toP2kOHg0WU8S2LWDfaCLmmQgs2Yi8YJDuQkfD+xWan4uAyZCK0H8Q87mCNgHlXrjjjlv+UBjAu6ufxx92EltY2M8dwZ4Qu4LWvFx0gQHWLbT72wBvCght4f8HGOsY8GVXuWGiLZHlG3hzF3XRKFIAI9ZSIlIbM0JiW5uPoMMKQQhA2OcBCEjFTBs8VtpiPpbbkCLVCoZ54ICAKYfUOR27\\\\\\\/LPavSkUcXvbUNSs73jikwdscUpG2m8PjBohSj9dsVMqd1xCNoGPGYcx4A079aZ9CyTHhqHwI1UEbnxqN6L5mWWtmIY9AVTJOdTINzujPGrnCfmieAAEUXzsVyxBzzuj4LiK9C0smPeJthF9nvhLWSCIwdev0tneB0wgz7CPBHsnz5QVLau6iQ\\\\\\\/Hedy5Aq2uhqivv9p+BWrUygWQNubLPZhS0dDa6HjmTOG7k4WGqsk0i54AFNegRo+AFbSqDtN2owH5uUJ0RGT+bP7Lu2nEK2fIyNoj0TzwI00kBMFBM+vp7l\\\\\\\/iRiFGkXMQU6GMoilhMToqt8vsRRvsFQwVqzOkZqPU2EF7Nx1t4Xm4ds6ayY6fW2HawmhpZVKktr6ex7xyBJgcKgIgNJLV1OHIp1a9202ysQS4vdyi2bmObSG6U2XMnrvvFEao+W0\\\\\\\/fbcMWo\\\\\\\/xYj4GnW6NEVeAXeDxtX+3u4DCq\\\\\\\/NyQk4x7zO++RC3k8wakF8+LWcYpf1phf7jBW1nH175k+UEDnSSz6MQntPZE+T7ZeoJqK0di6jRZr0eu0iJqo3v6+2x5B4+mxRNmmTN2w2GwpaDFkbHj8Ee+qgTjDQirzcja1agqaK2SVmmM8FoULw1eTjznEYPZ+QO78BhuUcUj+Ot1oOUu8yGcZiI3FSFOmYilekr32\\\\\\\/pH663Fb0SFod3z5FKhBFJTNr4X5LmRxaiCO9D3reSaRRdFW9CI0vvnSdB\\\\\\\/ch7by1ToW2YOppZYvwoG4Z33owElaCN8GY8YtJvEBwqJFvmFnjwXzCdtidVa1AJww5mVaIu\\\\\\\/Bjwi+cZw3HKX8IGrwGiJ8fhYtGqQ2oWY33vlEYxjN3pr79T8I8mWVyb4nVhaCkUrezN+e1i\\\\\\\/ZHB1EpokpYJLpy8vFM2TMCVGerAo5SHLyaHT1avc+r0qxFLa4tjwWlACy9+9RTxFy\\\\\\\/qY5NFQ0jcPZDOCtc7gNenlcwF+zWpUCbEIuoPNnwa68un\\\\\\\/VR4e0HpBYs7ArI1O\\\\\\\/8Uwi08Lp2dDd5Y4+4J7OJWXVDehKbINsdqJTwFw7c2\\\\\\\/+f0YsglvZ4NQumLZesztMvZaCrNd9x4mjsJ\\\\\\\/u3SPNcSnP0o5+B3yZZJCoffitx19J9njsBubxRk3iGsJLNsndv2MBssjXVVlo4MAdkloJ6oCqwQKAFufQfADdCY9fgr+zYboS7I2plcz3z3XvDk4h3sBFMSW3MH8nWhs+KN5ik52J2dFEgPD+PPgYIOicr3e9mV3CB9kdsWWE9pUkgj8Z8dYFNAm4JCS\\\\\\\/wvWLxGGzacygJP2aNm4qdOjlPPCuGG8nzzNtxxT5RxvzaYe+AdxAOo7DdFgg1QT7Cje0O+hIn4JNnfAN0r1oP+GHfBWu0YfDNfDhWw1+6YItlZsiS+7ffH6wmk68bUpucQ6EYzx0MIQQMuHc\\\\\\\/51z+in4WHHDvJZGYhJQEpF9Qfa1WR9imnEX3zL\\\\\\\/5FmOYDmpuyKAOAWKIDuv3\\\\\\\/fuDNofH9qmfGol7xRdRcXZ\\\\\\\/weV6udMFwhMbr8bYMK49XAwDtdb2ikuXDKFeAtXOY\\\\\\\/XRfuuqdBZ+SjQD1Kld1VkPOjdI8SWxBXjPu38xKo+M+irYz6NKZap4YiXTyKlQ+V+6oarcU7he7IGKRCPRTvZKbRAn04pv8n2GZXwpWxgQTIF2pRv3PEjgIFQNus3\\\\\\\/yPyrzYvpX2ZaDuAQZLu9xmcKcvXi6plk1dHAAEMW6l3ynq95jdyKjrwA6DzFI2609rSwSy0vUsfURjpG3Cdr53WelMLhIuAbLkH1\\\\\\\/K19ej59GzcAqFuZHkxK8XjcrJWaLkZmDsx\\\\\\\/1OFJJtvvp1mrSHR63cOI2tiJV2\\\\\\\/7oYXcYbhLZV5JrOmvDRgSbFwV3iqeFtIrN6oMn2bZyL7j56YrPZHkNQlCDS2NzH7coVq67x5KEvlZNz+9+88DkZJlyfbgK2+nJuAf7QBEIf344j+ota1RdONMKCITo5buDBWn0+D06nancXzH+3vchN3C7kDS7zzz0CO1LkmhYcYYOxuTn9z4M0F171r6miN0ylep5Db+NufJSHJenaeRNxCHYS92xynZEpg\\\\\\\/gx1AGE3hcTd8pzmPM1ErqqAN98hXO6DU6DPudqd6ZWPNPbrC7xWg\\\\\\\/G9vbceJmP7cdhimMD3enXt7UsJygplemPfFwAhhP1GxwpjBz5+EhUeajdE2diYGcB+02kVGJPgXXNOj+i2Hupjk2CpXV5Z79uNebtIvTRE+W0z7sfnQE\\\\\\\/EuE\\\\\\\/ifRu7i8XXTsymyD4zyZ4G7N+8AHie0LWWQjB10RN4ec75Ye7R8uew4HhdqgyaVYQbY2qBnZsXv8Vue6\\\\\\\/q93Y47ZKW1wceAZrA0pcGAGqFfgfyXyf2StgX22nJo6wdF031OSBRhtPGmnQwRQTIVveNhd3jBfirsb+1pCMB89RQ6pTqV3a608L7D8KvKGIjIHovK\\\\\\\/ExAtuf9mOgkI6os291PxEWwIDhTFZ2\\\\\\\/KzY3swtze9caUOzceIc0nmV5PmBRficQuFX8dHxdoG9JBP7M74HJkuQ1gf0uNnEi0XQNFsRGci+\\\\\\\/4nXKBLfmxEV\\\\\\\/0AyyV5evgZvsbbSYO1q7U2jom5Km+5iI9nUkLM2JEOw9csTxRApkfoQmg34qlIoMyegKrjo\\\\\\\/Q3pPbIZgELwd3K+Lx2mEYG+UHmOAi4bVPCcDOPLbRMQdBrpSm+MIMtWP7AXxkvGcYQ9EOWnCIS4FgwDotrdGyMPv9EZ+vhL9WhEpM9N4qunzOBSwzZNKpI5JBxo6UlRAmSTpQtbDqW5\\\\\\\/xxPw0ZDzYT2KMMYl1WzqJYpUCXUIgKKP+MyN4ho19NlSYCG2sK0J1KBaz402miCcnA9dO994zVDo\\\\\\\/uVjYvKM2Xhn6efyPR+7bAJLlKTZ94yWXv4WjW2OeRh7+rEALL1kyh+P9HXK3ao17WBW\\\\\\\/vU685b1WuIp6AYc90uS4PI\\\\\\\/YNRHOOYkFSOj\\\\\\\/9i5ikuWprJNeS\\\\\\\/zeTejlsERXIJUpmmH9UbyTxpvDKgtYrhi9ldTtHezfPvP5F\\\\\\\/thd7M9vpXC\\\\\\\/QoT4SnDowLOJ63UIZfSrODckUZqBT2hZDHNrHY1NTBwkRUSo1EGJ4ggHJwri\\\\\\\/8jwVHwZB7E5V5XYA7d2nAfJd1+4WhPJXveWwspOW20kM\\\\\\\/2u0\\\\\\\/WMtmLqzk224tjuxlXW5lkUYzT6hS4D0xOzCfNMLC3c3iq0cXMaOPSYaNJRlRcPcKh5VclmIsj2CgmGZgdoTMHNSJ50nVR9GSKV02i\\\\\\\/BgnxcM1VnTfcY32oK\\\\\\\/JOzJcVglg+KNemvYLuv3xyzzjrdCPjJTx56dYKwZov5FQABZdbGw96l5tKfeNVVyPQgUoH5vdNweerHY+if3F\\\\\\\/qIz93554zHTtbtVMWb9tww+EeOt1JpiUoTownJ4QBtXFJ0HekyXlKqvYQhDAaUpDNh\\\\\\\/jphfvBzCCrEwhxS1UjNpeCFgB15dNisvLluCi\\\\\\\/bAc6BE4GwS4+nd52WL61LNQ+o2CWFY6ub18qJP\\\\\\\/WbilfFJnqZdQXy6F8MyqyMiDfYSkvvbMe5VeOE0UQe8VQquQfBhfkp\\\\\\\/FO\\\\\\\/O10jGrfs0gU\\\\\\\/zMTIbV9I9edPSxbTKbd+HkKO3Zw8ZFL6jmC24VUqzLi6kg4Luv3FAFKTRwvEXUeaFVLeIrlAdQAn7ZxRYXWtxnWWnXL0FZY\\\\\\\/mdbi+\\\\\\\/UPu5PmAec9DH189+6jzH3BNS9mo3iSu8s\\\\\\\/A5d9oXRlh\\\\\\\/xaJVJAJD5kFDZTpNw4XUdMT8AWJEnNpj6uHhHXN5QdjEw\\\\\\\/RUpRxMhliijkGIbx3qJWqFLps7Y9XReEnux0gR0QOICj0rCne19Ofy25dgZy6\\\\\\\/kYIZRe3hQhiJGqDuclbCpv9qbDoMRUPMYJTRLL\\\\\\\/TDncTdJ5AXT2zozCcx48DoTMJoFvU5Uli5HPCTCLdzwZkYbqVqF0ErP6dVSrhRXOeg6VOqq8CRF+3OK48UVoDNMTvo+I9Rf0iaXWB3gTfYFSF2Pbgs+qez0JUkDkokM2HUah31GGGP\\\\\\\/\\\\\\\/kAVM4EdP0Lro0hBopQazdSPi0zMqFzrnkD6JnLP07GONYY4cTXKhJfa4DuiYrZxQJyWkem9jnvz8Q52G4DviRQyiEuB7eiG3SGDZorm4BrnkuYnRW3wvjxbKyAosdiAVWqR\\\\\\\/\\\\\\\/t0nFmBjENb3vkOibHLj5XC6psXomxd375HEG+uc2nuPcejrRm6K62yS8gX2RQ0diJ\\\\\\\/dcHwDudsywV31hX9wzWt7Q1qm3Lw8tBdKwuEo6Sl\\\\\\\/Soei3sSj68u8dQ4xbabkeM3Oz6GgbHu3kEmO6r+Aef5dM9y4Vgwz\\\\\\\/q4rdFpqdmQIqGvllsW6KkO+8eeAhOrDy0A1ONtb+yFMYLR\\\\\\\/91wMHdzEqhBIPg54orMoXwhXQfZ4TevM0535G65mZ9Q9Yhrvlc9Eo6oBLBog9JIpB8eQBu8FNgfQ+n2kZX55lKugAldV7DAs+iJ\\\\\\\/Y3Bn434JvQ5vwE7N\\\\\\\/ASY6ljWJuABrz\\\\\\\/INuQ4XTWq+3mszEX+zdkC8zt7ti9KE\\\\\\\/2wSHvbWDez6q\\\\\\\/GCW0anwMNoZ4ez9EoGym2lEg4gbObFSnu\\\\\\\/4Itmv3\\\\\\\/EZ+Is8SkvN9T5v\\\\\\\/XDcNIpG+S9JEHQhPgSMzKNHpvnleq8lslTzPPrr05VBRV342mz3G0TYTKMncCSf1yUk9OVfH0Mr\\\\\\\/aCXkwQ0FJ2kTdo0zJKHB9bpw4n611YG7Tb0WHDfiec9OF1u\\\\\\\/9kjRd0hkl8jPXDJp6NyCPvyJRXmvj8ZJev0ofsWDfPPCKGgCrTbAFgP27qVqxqTkaALQpoOFW6mTWnsNYkPuSE+trVMmICG10gvrtwwNyl+Fy\\\\\\\/3YYY1er8bnMqj4fVYaX5zO2gBaVhohKne9848gFbrlxT3DRJq\\\\\\\/KUeG9b4QEJBzuAYxNkwPHru4ZHm3jWAlvvdZMJfDvWUkNLYWiOlbeY6sv6+lBQwCNpJPv0TsQDSev+F0PnlRxlIm5VuEmiytFyjoauBpICWVyzDb88W8f2N4l4L6cXsNkrCAwqLkAfnUOh1jc2L4cwqlE1LqzVcTzHOpm\\\\\\\/Y57Q0QFRdeMJmBJHizo7Opmg+yFXydGZwJelsUyY4OChT0Tn\\\\\\\/XfX\\\\\\\/K4SUWhB6CkeDogIdkW8qG3ml7EAkbYifoWDwuz+3C0otxIoIC9VbPa9BINX3XAjFoE+cprhvfvElx1aw2XXxphfK\\\\\\\/cUlCBEJseV6Rf84\\\\\\\/AqBICO86UncgkdBo+hwHUo2Bcs2Moc6TDV5be\\\\\\\/zft8vqlAjAaWTC6EYe3+yruaIgZB2\\\\\\\/i1\\\\\\\/ELWAJGOcSgkbotPOj1HJ4yddKjBXVXR6Z1iMM7lbtfbqcwbCXcM2zFsGScawQHwS5prbroQ0HYRqpZl9Mx28vFVbVVIW3PTU3b2q2ZFkycOKEHoxtRapa0NwBaJueAlgrTMeXtH0MGeUiNt2P3PF\\\\\\\/QG78yu0fmOGwOxmHoeuwT1F5NdkvdDpjNrCecw5YDC4Ap0T+PUgohSkEWkjNTvXxEz4yVEySi1sFgOe\\\\\\\/Xb\\\\\\\/NrwkZOuAAodHMaTo+NA57HkDwjE9gOIs+5SVbsSuY7GJQUZ3989NyfcCi+mc9GKegcLOZxJ\\\\\\\/Rkj0AntFIb9wpCakgRKyC74lksYgPOHZ1xK5ELzn6+3SltNYAflo\\\\\\\/02IaDWknAu8JtcZZmSo6RGL\\\\\\\/Q8TY4JuGvOm9+2nrGU5Ei1Mqzublnt1rwLQdl\\\\\\\/g55WX\\\\\\\/73bd1WLgha7OmYPkVNNaResCJ8rFjAz7hCbF0uug6b33q3PwaPLMoH5PfCSBZ33J7ni4VSbALO6otd6G6SKaEdEjcKJD8rDRJmQwh7RWHSoqRDyHKQoPjsM0RsJvu84fNBgfKOdnwbFB2pVUnbEldGd00dM0K2stqDvgD87uPER8hmCyvh2uZdLoppQZiuazT+gSfl4++OeN9lgzBYlezuZr4QEw2xL6SYXL\\\\\\\/dYAxThyLVpziBNBrK+lpKNaqLfB5EqGjRSMbBdRaFN0g5D9EWXYT2pRnPy\\\\\\\/G5Jb6ReTWfT02yZBOxVzZey2a6qGLZFeMgeCtjmt5iBvK0F\\\\\\\/bfGtkzLuDbOseDUgDgEBKCAHhP5uheIasI3jVUUyq0tLntpwNOxRbVvns\\\\\\\/MHGszXzO\\\\\\\/PHqsTMqAV2DyMj0nov7MlAsuVgvwcfqyWXd\\\\\\\/t+\\\\\\\/1CC9ZCykHdQOJHqkdFSrrRRwrugQWxBT0D5aLYf2TRnXXccxPGk1fQv\\\\\\\/+hHghn2MIIfNwvytgVAnApV\\\\\\\/c8\\\\\\\/2XvLMbqVd7FK+n1mVMRf98jBLmgEr0fuAABMV+Xl8iW4anwMBTWm25JWnqxyr8lQR2VCSs9edxkTik9ZLLpQAwWlOdZQLtD7iS7lO23PJTo6BX2jMvXWu9f3DVqgglmltFZ19YwbIR9gl1epNHNp5cXeCxBe7eS+I40X\\\\\\\/CqfKvZc1tLEFfRvUFFI9S8lYUULRD7D+n+o2B\\\\\\\/NqFhbvvuaxJ8ncMqxGs3+bD1AfPGa0Q121caZd5GdZ9K10OSx4DYvmd2hsahGh7QVveFMV2iei9jg6G9v7FP\\\\\\\/yT1gYLUyQ8iNfOLfzMdDm5wGgYh8VPfHqz7Tm0TDvKKYdu6YOuWgzN+aKQiO4W1debhIddZtuR6KP85hNf2njFGX6xLP0y8nL4rxdSUHBe9\\\\\\\/IrNaUzNLAS6j83kJC\\\\\\\/KfeaBUlqzCc\\\\\\\/suTuLLMb3MK0Hii6rPGiB6bY5t4XelAWhyljit3a5fGLXVvd\\\\\\\/qxgpDkdIOh\\\\\\\/dYJh4q8WMhK+1PoehfxupTspx4ycCQ3qGVc9rcuaa680g9DdrMlRorNw5L9amgZIflpr6DVL6iJFVjj1fqVMLhStrC5ggqdvbElT2sTAI\\\\\\\/09jgabls3UnVSsoCKP9Zz3Qyil86TeSTwYKs\\\\\\\/jseMnlX9EmT8BayUCqgGqO9odkmgBxQsxOyD5ivqLsp7tHRV8qhB+9siLdWWHFBB4qUkycsHR46xLkKqkhixMaOJiI6rMwl6MCDKf83w7EaiL83KcF2Lu\\\\\\\/662ViDo8UtrJw1Zo3P3Xo8uUq0pmpSPgmBWk9KfQ0s4lTGWap4gfVFdKK5zu9MqwXBuYgwIh2LJr69IyV+Ptq01H2LLNsXpoKQ2Pl8O+w+8rjRIRHb2rfmPTrjLlF3b8StxMTbSmMQ\\\\\\\/1+ur3TlpgQupak1VfAqT1jEicjK151VP1tcDHqAkKQAGlhHNg8BcRgfWkSw37Qnitr8gGxCKrNhSfiZNE5KfQe33pnFvyDkG+i2HAi6RMLAE8ljPTxbGT1aIPBr+HEu5z\\\\\\\/6o\\\\\\\/3EVx8OB0MvPIf\\\\\\\/CrcRhuQOcY6MdldxSg0l+xKv6\\\\\\\/gAJTsprOhcGQ4ZmH6bsfGnIgq+PCUfcsLEmWCM6bGXog19bYYTHN8Jf5TEAV\\\\\\\/KwFa6DI58HCkE1JF\\\\\\\/Me4QEZcNE5rfKGY6sgf5pBrAcbqqMxmwm5A6To669OF9LYvkdR4xNrE8KNB7KYkawIiQ3bU4PsHkKdmOB8Fs47QsnaR7oEE6YTqLCtdHzHBNxu4Rm9UrTRlK46e1tzWGyx\\\\\\\/MxV4oKnegngihdy8wIeJ\\\\\\\/9+8Z0qs2OrjkwpuhG4eo0hxBaEfFASNJVl3mdV2vJabgA9hlrdvUnxojL2mtjja98C4ifRDELtmPH+ky+PU6TcaNa1+lYJj+hgg3d1mt6L9y0T1V9mu2T9mh1wpTOm3SeiRdPlUfme9Z\\\\\\\/XYt8lGHKe7JK0mRZdiqJ2QkmgnkNrjC172OgWcpAQznN\\\\\\\/lG1YvXFaWQrzPqiIlmSLE0pVROgj9NlOsPGxroU5uK+fSQiVE\\\\\\\/K9gCm2Pk7ysBIy0YndDkBSHXfPjzX4DvfzvJqaB0VEza1F7NZ203SZm3K177Kfh7xGFOHB9r78GsIgyvSn6xd6MtyX99OHbfGji\\\\\\\/+uAnk5ayZ6wUOZShCgZMz1LIT6HS+avIRMnZB2WUfRxpneNzAsLv+SzCf9G8yFpZJLQiuZDHr40A\\\\\\\/zla33DqInBbPiOGtDtuVrA9dqaa8mHU7lgvBaLwLkWQYlNli1GwCIMr\\\\\\\/KzTYRHunNXtaVm3VVLXEkodDAID9TQUbdRL54doVvPAbzOQ7N7hwhlGod4917IJ8agq\\\\\\\/atGfy3HELTpWyQHUfqfD0PthGomJk04KcSdepCWu+fE09ACFKZRmEVJVwIHedv\\\\\\\/tlH\\\\\\\/w\\\\\\\/Sj\\\\\\\/rqYDxND5qDElKqD+bCKegLHpQpy7jd\\\\\\\/1jN\\\\\\\/2pUb4ZDUmjCOSrHu0zheRM9KOozcA3m0fcmxcgNL1W+y7Mfag0PSRYkl6aFy7jfe5ghLHK1LUiBWZCwl7ncCvYKZ7LGGhrTFLbQvdZdjTG7k2bCYBGnynZ\\\\\\\/7iV3ksJJJAEYspCWl0r0Z7EKu+qtijaVVPNY61GC7tXNiZ6stplCKkHXea4rHeQKu1lo95UDIdrR4mZbnzokFfEUD20Tzi0m7TxQwO4IUhHVojU3gbzH\\\\\\\/ba+JQXH+2AamJGK5jpUiiONApmTg+4UvCxei\\\\\\\/cEMiGOBpUmeMPneOXwD\\\\\\\/M7Les6wLN8awEydsbuIAUMl9CoPEl2GFiVaYsJz2hWurM9ubC0\\\\\\\/5F73UJhYXtauNpLnHVKg\\\\\\\/Vp\\\\\\\/i7QvnPwb4cH7AYR5zRNjtqIj\\\\\\\/OcLFnzxsZx3kD6vCFF9yKquzY3ILdIn8lmCQ2rNRBEmBKVQc9f4HldCpQO8NICAWIk2jCKePKwhlASqp2o125uHz\\\\\\\/fJ5qJi\\\\\\\/T1SeGjMsbfC4dM7CFqYNqS17EIWEeAFhNk\\\\\\\/n8443oV+yyYVgwZMJ1DWXYQLAmqok3f2MpXEHn+umxk5MRFcLcH74ZeGOn1plhAxkHL1gBdlZ\\\\\\\/HQsP09rx9W2yDkxNk98g4mBOBsvDTfekBWyooBD18GcsVD5BeAP1jwDoAsJ7vsO010X6HnG8W8pKGpJgXECVbu41+Jb4W+LcKhXIezitE1pf7qxBRe2lJFrUeBF2dduc8sitoBuyCYhAVit6xY3YOX\\\\\\\/Gt7EbC3QUjbly3CTCtsKUkl2RtPlvLEAtRosjAN8AZvEpeKLd5fW58wgREQbwe0jIccZexeP8r3ITARWNjAi8H7pJ\\\\\\\/8OEYzuOMJeuvAHAb3G6RD6C9vLbZ0d36xa9TC4Q3N3FmMhkHmPlYYSBf2Be3iMMjcf3IhkaAV21V9xItEhpN29nzvv1qDJEh3m2eIdTLhL5QZhjJH\\\\\\\/2rOPEcR7jkDw5h2UGK3cjncCF\\\\\\\/k9VapH8KjrHV\\\\\\\/tkfA916XWX2iHfdDb6Imn14+57igcX3mGpPgtT+o1WruXL7k9E9pLpk\\\\\\\/PDM5LCYc5Y8EaZBAwnyKUAyDX5JoKMk3349mKmpNgTBk4wO3X1Zxyql6vS1o1HVY3pya2p7cZw0xlDE\\\\\\\/hR1qlZILQfL8rpIzJ+QHNFd92ThkVud6YsM\\\\\\\/vEAtWe1TJYP+zhdAxreYGnYoQPTMX6HrhY9rq9jyPZ0s+xi7jvgnBceBymTlyvpJvJFdYXE95CI+4RDQ7hS8z1ByiGzz\\\\\\\/\\\\\\\/yCQ0nB98RwlFqN\\\\\\\/jS782UmLZQXhzmYwmBygtZfL8dzYoSpn5AeeXnC5jQDwRZ71X4DHKakrZ6GKCbaVF\\\\\\\/33wiVZ4ugyoIRA\\\\\\\/bvKs7CwmvwclDCR\\\\\\\/fKWreWthUcP3Zt7DEDdtAKJDGFovZRldungt032yXUZ5umc0n5BqLSCO5xe0ADIF4FQlKLso9ZK1TCI\\\\\\\/TjWLMNB+a3KN6fW6mvl3bnY+9fnMsJRTMDZwizTxdIREb0ir\\\\\\\/Hf+FApazqcfQBtnqV7SF\\\\\\\/W2aKkWGLnxg0FcRx8IIclf\\\\\\\/mArp8e13Q0owEXI3q2lrCJCKOZ75336sZUjEAkHjJr2PlkGUY67F6+IoidaI\\\\\\\/IYRdcDYq59ppKCR8hs\\\\\\\/FGjrWj6fauPk7170phmeqT4a2rC59PLJGejmZ7iUTEbI4Z+GahxHHcDfwzlcmtsKk2dNdiJHHZp\\\\\\\/mCn58GSq2Mt7aKSdq0hQwSbQUeg7ZEVzB+k0ooNqQgD+0836AD4m10yMkL3ivfnrpdnnygExZHSEwrcsvQfRk02wwUieEM2HIGtkZrg7TK7UNWOqmsJMEqVDyDAnhJIt6KAVauDm5e\\\\\\\/+UtO+Cwkw76ETRcoELAG7B7bu\\\\\\\/zccOVy59o5N\\\\\\\/9\\\\\\\/nGwtqDlfJHq7Kg+mgtmBpW+s20k5KoBpWp\\\\\\\/KZZ9ySdhDIgqGTFBSaMgi7NVuZ36yymLdD4XzjCQIIKpf4Ha9KBOoovt2mZ8\\\\\\\/8UqJ7QIYgWi6+t3ic9A7NUoaMax6dcmf8QiE7WHFPMb6bbjhyvuWtYvhQ0MRryNCIFcdiQoHOsRG9M9Z2K0FqpOX6FyPrIImTlOQPc3zg\\\\\\\/wXw9sIWuVXDf0sqZ9DJqJFZOSKbP+svn1cLSDv7FYHgV5pGAXDLCA5bbbRwnEM\\\\\\\/TXBg+CbKCLAA6iJRvbIJgHdlPjRnHcMTL6FUn3EQuoom34cR2q36QWyM+Tuz3I+\\\\\\\/mGuq29bRVcnj6fwwCjU7fUBl788M7hIggTCwBfuKJeuRddceNZewLbBqE5OiPMcQuH1prTXpuC9j\\\\\\\/8Yn15DLbKceO+zXRKDxU5Plwk0mn+Zk+4hZ3wFKdb7oidOc\\\\\\\/PHwf4EeuxhkKfZKP5TpvPpCXZoKmeWEgW0AbTBw4tPejlkMgtyl4x5XCyHxb67uk12ZDNuqynjOANYyzHabAbnbjkBdULWU648FQS\\\\\\\/GPgn60t3goxvrcXymPGQ81lsuvhiZPM7IfW3774yub6wkkHg2ChCCC7JOKLmL0Kx0K93+v2AWwyPIhlHlMfmAlhqOWVekDekfw\\\\\\\/2Kch240HzP6Ikbqe4r23+APWWveZDnLZTb8Fwwq6fpt\\\\\\\/qNmI\\\\\\\/XuW14XO83IG9P5t\\\\\\\/pDfpfOViJ97RoXApaGKypIf9OLuB5yqwY4YekNxfIK7QuxmSc5kRWd5U6KaE8NxOLgDceLurxNLDiPqRO521mxbBoa4DNC1+6BzzS8sv++Z3NqAEj1Fw3pTkgFAtjgWM6b6dQRdXlYb9Kr5D3iNGypDddZR3YQ76vFNusMUuIxEWw0JqFhYOxSuE7KYOAPFd7ODa1BV+t1KVPTBzBqAt1EOD+XmInes3tS0lGkLKO421yiN9NxIrfswHLwpP95aVLGN09Pmtk8cMV6229lU9t5VhX3SimS8QteQaOmtHnN1t0+lBWv7nLPCIsXlWC62voHD6ShLM31rFLtPTqsiVreZI7Y1U50wRKhrXxOxMEKqOLAUpfSCjDsLwEvKLh8yMlFRNawEAsZ9qyCNGNOVSPiEzx4QAU0IONEZ9k+NnqgQqZAj2J7\\\\\\\/Nu0\\\\\\\/heH1yP8LGicXRklR+MgkFU8lJp+TFDFKW60kdtdIveSSYqX4AuGA3duTJntlSuZ\\\\\\\/YOhI+hQdYIQIDBQ\\\\\\\/axTfGilIU1OUUgI0prWvXzDe\\\\\\\/qVOCUpXSGifzADTDEHael8IhRIJePh71NElGNqbQ0h0BtER03k8pklijlOmkaxDWNVuQJm0soETj8CPWk569Ibw62Gn91KRSzpIeT6sUJ8CbUD0IOJShkgs9C085JD1NnMu4mafEFSN391yfEreQ\\\\\\\/CgyVrgPGGoduHVHE4WxSVrrtq3vhSH\\\\\\\/jm0UoDuDyMt2fMfvqQ4bt5USFrfFABXIzI169FEQINCA2JblfZ15qHefCJLdtmAQAAR3bPZXsy5+o\\\\\\\/5\\\\\\\/KyC5Z7f1oMdd1DgD9vIs+5CMGknhyLRrz2MdGuKfew19NdivYyKG7o43aSWXKQXIEcAMW4wPYKa\\\\\\\/MgUGwDL9vBFDhQTY34KSDSnQAZgTquMcdGxw9liBACWuN15k\\\\\\\/XQEGF5YUdHeYr\\\\\\\/iqX+RT8urVllQCzoY6hTwr01IwgJCZkIsluDv2LsjpGcSgV4B7qeDfgas13KKHkZEG3Wvcu1rpLiCnsWLS6UjtOyHzvJVjgtvxAMLldAtHn0DR8Jzg8pNk82X8ifzJKg2xBUjf4GqjduYqdbL1tjTkq\\\\\\\/m0zfZ7tNKLcFqnnYoEpsoAkFpX9uY9HyfStcYlB3BGg+AxnurEVLf+75aucDqBekygFAGZPbLxYeDXUPn4MasUP\\\\\\\/zPWpIY8L1YZ7Rxn\\\\\\\/rOa47pMXvWdZrosmg5v9bphFwhK6b9vxgZkCDWG7z0i25Fh\\\\\\\/gsCqWq6OZ6oeC6f7DLioLKMMUW80qhgrsX\\\\\\\/r8xbafxFA3fvKgzV20SaC8Aj8gJkZ\\\\\\\/N\\\\\\\/VOim1xuhQpk3BkvCVEFwOn7h7F19Dpvz+s1CbD9zHT5fmnjcgOLC1y5zG97A5Xgd2+SvoFGDYe+v1PWDV8oQ1pbUtcXdSQ+eu2l2EllIF3ogHR7fl0xUKHAL\\\\\\\/Yy7uTaRidHNdAZfLcMMpE1Xq7M\\\\\\\/REfHEzZASUadZgKLQYUXsEIpJdKXixwWFicjDArWOat7ubT6lT9JQOXUIEZFF5jtXT93cmy0QAUcnHZAPVGsVP3096Rfkaei0QjHdpDooHJBRKPfwNAha38bLO\\\\\\\/6VwsMrdEdp+sBivZhD4HctKMCasHE3hjX0YtxqFKXbOMIgFYypROSkJ+S0xR69a66s6M+C4gpGGFg1MlWtuLBjG9vv7iEIhzXof1h7eyx20rgh1hZQUZEgqpA4AoPI1Dagt3hmOKxZQCiLmg7Gax0xdmFZir9tOr2OiauB7U45jqG7uKRJuRirRn3VMbPOkszrM7vm\\\\\\\/HxFVEpoUjuakwU+mNWN+rIn\\\\\\\/hcVTFZzZRRErPotCCngU5SouxSyvZPEBXzWMLIOxAh3JBU5Y68Nlj1rPLtQYu2kw3e\\\\\\\/59LhXgwmQyavX7iDOTEj22eYYzk6J3YMm4YBYJKTQ\\\\\\\/rGYseQQA1oOk1aNglf2rgd0BwhpLkRltLt7kJGu1Q3h8JcOb6Pj4LINRaWTo9ooVaWFtP7bRjx+AtLz41vn69m8OsT23l+\\\\\\\/eTOVgy+GjUkmPulIMQl3rqa4SAVwFS4lVReqZHC55Sc08lWLtI7uWnkk3Ry4Iqp5blUzMhuugphwA6tsii70gawUxKz\\\\\\\/Wk9RbAoeYyhqE6+dt3Uv684+ZYAolq5DTfO0aL75HbW2ydjeeMwL3o6A\\\\\\\/swGN+dkXR3iSo+Xfq4WH\\\\\\\/iBIvPoAEF5M8fnAzT8U+ZyTpyKFTGqOhrgcUQUkxG6q2AX3N3Nbo3kYLhs1101+Lk4AhBkkDTRNEN1m2zYGg\\\\\\\/+37NnncLzZOd25d4a6bNIqvCySKooRGnJAQ\\\\\\\/ejXPQNfT7QC2MNWbTnmL6H2AmQy2BDSCy05wX8y71y6jvEcfGCz6hyxwm6j4kYSiodBCT7uRsU+6da++cV+K43F8JxkqzW1mBI0BuvNWqK2iAlSzk4xME0G8ArPInqeQqMtb6ka2PzbY2B+Xky06yQY6VVBUUTIouNMRvR9h1b44FSymAV5NpVxoNWTqlUD6kMz8AtaRPwQO2xO7ZUweP0fqKgJXsNb9IxS1qINlmWVolzQA8B9Jyl1yVDeKBlvJJOSuOWW+u66MDi1QyncByDSQOW9TecHLqm2TlyHWRqMziIkRXTsQ5XpWflIcOlFTy96GrIMFIc7QP+gUh0udbyvEq3jMc+MMaBb8fMRt9E+0HMMOGqgcItI+9q+K8AUYmeBsIDG\\\\\\\/B1BM4DSGqsbOjH3h0dsUkR9SR0bb8AxO7dmGFtVg8Pqf4wKLHjhdrLDW2b+gwYgSIQla5nFandXu8WeKOoU11N5OnPI\\\\\\\/F2z8jURMmcNz0jUkbbG+uqXHskx3Tpp7HwNM9PQOG9njvJaqOQfLpaONOlfA5VVaU8Kpni9Sg2ymamvFUTvYxScmWIDxV8OLHTXRmdK1rQmbYC+8ewOb90QyZAa3hWl3wzjsiiuwlyn4C2mK5k+q+8o\\\\\\\/Q5XBfZJBjsyzKBeAoZS8MqI6Qdqy9QTZr7nCzmX3XDv8+OAJq6E3ckZ11+OFYkBm9SD\\\\\\\/Um+cFIAlPX\\\\\\\/F6430qlhkEmugQTjExhjJVH6NLrDWip9aX86Yc9gDzSz3lyDvp4+dvQ9XueeZ7IZQ2i0\\\\\\\/GlBuUvCk88ELT8TTJRXUTT\\\\\\\/gAKgOqVhrQe3CR3ISVpr38V\\\\\\\/f5rDkHHvKOxxiCjXkZeL0J5dFcBKJIbna6NNeU0wKb27niVMHH4jat+SR5N32F8N1xVVNt8qzd+Z+8B5YbWh7bQAcyuPNefjsDfBqjltEZQJ0n5s5k4SVpSD1KbkEgKSyhJ4IVmvRAuT6KVz0tt\\\\\\\/fPeaLXVYj0KzwebwqMZXPGQGumimBxky5\\\\\\\/\\\\\\\/nVuejQNi+ZW8pIAabMSAAQQoLYlWtNWMOY07INpgPywwDEIMzfpQ\\\\\\\/9deewDYULqvySZHjK2AjQRn2OA8\\\\\\\/mkVdwptQ00NBxKcQrX25YpDX2u2GWR3K4sKseP38dVE3Pp5emSeWdxbl\\\\\\\/m+KierRaBzb\\\\\\\/J\\\\\\\/Rqdul\\\\\\\/VPs81PPCO41k1YfvFXuCtb7NhanPy1T7ZD5hGqFPmWTKwB9BmUfU5+1tswHc4vwtkhLgR3VMCeKqkUO0kr+4O894SHXO5px22ns0uLAuQRjxyP8rNG9VdCPnbQrqvadalNfMR6vepOI6tKJL1SkIgGegxOG\\\\\\\/N\\\\\\\/eaRdTymt9HFc6qeovsLrePZPw\\\\\\\/GGdigLV6K27LXG5fwHVn\\\\\\\/Huewn6e19mMk9k+ixPuKE5Rs7O02RzWAFCgATlRBHdGangf+iEiuT5rAXe4M8LEkXEQevCpBdeOun5kqWI5oQNkbUpw1HkidxLbIfsFX35\\\\\\\/u0KElRHhj0v99+eyw+XJK4w+8w1y\\\\\\\/9tTOhQinMulHwl5DWnN8YZTjntz9ToK7mEOGIyO6wSL3mrxHjkWoZg3G4nhGjl0pljhBbt1Bxmc9RlIbDZsJDZoLLmb\\\\\\\/tpGFJ3oyEXJkfFMNuZwGddDzMG4YYKAlJASXG2Mw+76\\\\\\\/cSvWQJoRQ5eoCa05jqCKVXGRKVKagL3ywcsrXKbZ4sedP5KzEu4p0DHqF4j+QGZeAFcdxne9\\\\\\\/ZIpFRs1YFQFl4Eo1DMbeYNV6ojiOnDOz91djVXa4tjK9NUGjsNY\\\\\\\/LTVUJSzJRdSH\\\\\\\/c4fsGyq4qcBs7OtjLYQ+7ZQW1fwmvU8nMicDCp1fUAYM1PQpg5o6VFmB+IPu9Vckw\\\\\\\/bW5RRIn0A\\\\\\\/5ZUlyZYBePWWc1YIfnaviwwSXWq49aZg671qLJZ8EtDU+zIn9U+H3qrEa9u7t19TKl3WjK6lA1dqGRUJLoXWzsAFz9wo07KLwrpAkSwDZCXsBgQ4ArWjqMJw1NF+hQnJimhCTlJZ4UHjfrCpbL4gfPRBh8QxKON48S5x778\\\\\\\/JF2mEz+gbQyIpo1QCFid85AYYnCtuqD7r93HllB\\\\\\\/DwTnkwsQgcXnK0+1rF7K0kIP7zGxbyjeiPYCzwcNOVBcWstSyqBuVhdnyjfTg\\\\\\\/F1\\\\\\\/ZSjqmhBBHoyn\\\\\\\/rxGVfYQk8Nb2Nx\\\\\\\/sYqRvfffFxakAJAdzu0OstgQb7V85vy6l9ML4t3eF0hXoyiZNnT\\\\\\\/QYbzPzJZeR2QDA6uf46NCZBtTO2y+hkdhPJF6iF82qCCRRk+bCb2gJ\\\\\\\/UNMaDkRCzBrtF40b9V\\\\\\\/CGe\\\\\\\/H+I3XI1OLM8e+zcat8A\\\\\\\/XjGNxCJenyY9s8YK8Fy5lswRfuJjrtxBO+y\\\\\\\/p6tVv+fyWPzmoLrRRDcDMyOIpMyHkyKp6DtHfj+iFGiErUKrSIKrv7sjTStazu4T26cDx8dxRS9i\\\\\\\/S2mhpxqPlHIGczXxww5gs2yPZ9rRe+5CFB14OMIp6jByCOPWp84ViLc3FTy\\\\\\\/8nzAnMO6bZNRblXGR1GHtwQqRaob5l0eTgmaO6MVSCa+hs1fnFeOPR4xP6zfnM2HmjaXKzUMHoNPiNeSXaElPyuHm3FZlPH1fSh0qOpEVvScqiq5vmJOoTsQMVP4DLa6ciyfVoBRGESfDzzBAh4ppT1O\\\\\\\/Dl+sUFSR4vE3kpXGtF6YRB257hVTM2ygN90hZlBYf66138rQbhvwMrQ83+KVDrE+CDWGRZu05uy7m8+Gw0mlTWQTlHl3dG5UIW++tu09d4RPTgPprrOPYuiwZVJk5UeGuuQS30Vi7nYI0IJG33wN58f+0Ma\\\\\\\/ZGpqtSfv0VolGSG2zHNcBNJXu\\\\\\\/FEOwcK\\\\\\\/Y3IORitRDlels7hC1tIvhgdT8fxAIWB22FNaspE8xnnZDpZwM5rcxhVl15PKxubwNBoS4h5GaAMyQ9Ibv7JEINED9HxCr\\\\\\\/hrf73bboNoX4mekc8P+3zeGMPMtVQ9Pv9kGyhmv\\\\\\\/gXw46xk9gdI7DSeN\\\\\\\/Rw0vEbbFQj8MAMAI6zSKdP97G3H4chr+K5LtcdWgkkFB9lm9kWD8w2viD7\\\\\\\/gbkq9+msMYYaf8fRR+OViZnWItbysN9calOhOMYX4opVaQnillPXRKutDJZ9ZeXc8mxTAJl2KhIs6znBvdZPe8c8RPAB30ycIzKcjnTJVGR5GTYLqABK30jiv68p\\\\\\\/atVEOYAIZLAcnG98NyJ7skIhVYg3R7ezBjtHR7orQbmLbnkbmgrgYzIlJrHAcDc2dngdlwWc2vV+BWPKlrbQYQk\\\\\\\/uoPhRJSO2IsjTiP8XHWFMvN2WqcPeXJKGajkr1\\\\\\\/f4II7txdKncEH\\\\\\\/Zj9SqAaB7ipV+qBr35nx0qKg1Wr5NKpAlp4nHLQbVT13bzLzYU1FG6\\\\\\\/I1J7g\\\\\\\/KAQsJcIdej9mdTtK+WSTW9fy7A4MFnmZQSxb\\\\\\\/6MCs5op8856K+1gxemv5L+uqZi2MG\\\\\\\/gdsGZjzlxDrFYpwISzNDOcTj5NKX5RuI\\\\\\\/vglV8\\\\\\\/M29i6AspPJz95Hep1ArBkTkCAz4DKAY\\\\\\\/E+Z1albxJULgytu8mPO2JtO6i+fZlSXqcB88m7KEXLIQw+pqWF+waSu2sHcDjdxXnBHGt1ECVcqOw6+7cvkn0gvWXAbpZ1xl+NNf9+C4Ns8GYURy9qnmQcxJ0C5pu2lt8G8KE5kR8pBBcDxRtgTDeSTT5HKQx0MczS82cit\\\\\\\/l+yi1WgLK0zGJmiATjN+QOc3W92Tq8+rlYTWy\\\\\\\/zuOaHIn1Zu\\\\\\\/JowiX1uUGW1SrGdRrM\\\\\\\/FCNBo0zXPYppcXfhgU5P+gCpNRTnLRUi7YhkwoMS4WA1AdOE+degHO86yaxheRNhC9ugN5bo3ZvfkJxur2PvuRC9zxt8t6PiswHgFbzABYhUqQCeLtksm67yS0t0cn\\\\\\\/QY9l4S+NJTGInvv0w+KnqfjXt+V4CIffwNPutOvFWMVClk0nQ1ACJ52QSrEZTgyyTs6VmFMpZbHIDhUjhFqJNTdM3wS79ruOsrSlk8R6U\\\\\\\/j1mPTLgJ2CfhaKroXfeKnvlz1x1kFSJ11LRfzQHj1wluXW3YsPufhK\\\\\\\/lLIWqvUXZc3MDcrt71+zd7UtAEtpAmCrs9IRNymuTuT9ipP1dr5ErRIqYce2mqMJkzuSUZL8tL8yGGb3IMspYM4ZwkvAjPUcifNAWFzRHL\\\\\\\/2ldKqGF8RYxzgP3XgByb9IUCUMSErwaYFJ0K3y4tucb+Qw7HRd9XCrGiNSVv6V9hcHDTTg3Y6HpEWDRIEOGCTjZaoEDiyVw2\\\\\\\/69mgbKMEVpGWaiYjZ5sJ\\\\\\\/leLnVywst20MmtrUinvpjQHgQadZUmYmPuDHw+Ep8vDDT5HSypc\\\\\\\/WHguhaHwE6t9AocKI0lsusvqB+oGPAhROjWeNVrsJlp1uX1DFP8jPXxcHlhD5\\\\\\\/BcAQ+5TOyUxoyRfVsU+lJf+5kPhbVB817EoYpvgq2r1hGxwOcCIKM8zfJt2SYf8OhU4tdtM2cMVvR7HawK2J1UpzGhHe\\\\\\\/qMscJOx3LMrUdG3FdTdP2XF32rqZA6g3B9lproxgc6Jpn4DUQOfubfOG\\\\\\\/21XpPUa9Y1I0ynAS\\\\\\\/YcWlJUUxQaORcwMjTn2Yta5Y0kCne2vv6avFvotHGS6Y\\\\\\\/6U8w9dfA0vAUjXlTTZVCQvjaGdatNWQScHA7OfC14fzbhdGAT5bZOmhv7Y7eMEH7FVos43Z\\\\\\\/xdKraegz+fsddng+kUaGwbaq9m7PuVmdjOymx2weP1wDFoW73ZMTTJGkaf7HwmpAGecyoFQfNHZGHPSBylkp1xy21MTMRpm2dESLLuYziY5ACa+orX+cCn4S2hpxv8G7byccnhy5ZPV2Q8sZh0mhXxyE5F5oJYfyOTG0KOGGytZ6enWYyT4A5dnSh1v6exMbNmNoVVqcRR5OdRGQB1ghR0Nn8jN42tIEF\\\\\\\/E6AqgY9AdCYh7tcfVGkbK4jE6o41iFaQoXsmqWVYxvIGrnNuhJKEm1yBBNLnZSkCBjP36w6F\\\\\\\/tUdqDTcaAV+XMITNU3sEgAjRIVWyVqk0XZer3uN7Ak1WmNmXAAc1Ld3ac7lAyUrcpQtP3yuJNI6UvY5uDGUTPb\\\\\\\/oVVQokIor5fhdypVynuPabG9Lp1sXXnthzGq8Q5SIy8yTYeEk2D8Nt3T22JqvKEVDjPnjtzDv4K16FI05UPL6PIIUzWrp84f0X7wLvgZ37d4UqgSH72sKs1SFXDvqtrmlEFO3UYBim8fKz9qgZhi+GEZ0k1zfVAlR7\\\\\\\/BPQS+NDhgLWFSoErvLxZsqc8CVrB7av+wmyrcGPzHIw9wXV4FFPl6zt3Dslu\\\\\\\/LGKSwiZ7YMNqvH28e1\\\\\\\/2W40wkQW8HFM\\\\\\\/K3qlU7htFh0nYjHBLMNsANSZ1kEY9TQpCm9kKDU79l44WT7MeQ427F0bv6VI9mJIQ2JjiPeNNiAkmmKtwaNu+9PJQorBxnq7xWW1khb9DfrLpwR9JadlmhNr1m1zo38W9n6TwGbWoMZpkTmzz5y2\\\\\\\/SeQugWmWHjy00g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ETvQF3slVTujJdtWlBqRMA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224fc44e2ecd7bc04ad0cd1015ddadc266cd40e51228c8c10ef2050da857761545\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uqm7xOSm5tmMTwbZA7k8s3zMGTAaQ2f6Oxvd5YvS1\\\\\\\/Yyzp++O7fsfuqXxct0fmAzf\\\\\\\/JoZK6X6J62hI7Ca1JVX8+Q\\\\\\\/oOWZipjNfBs10kCodThz8fthaNIfIyh8qKReAmXGws5nSD4fFZvVUWcajCpzIZAXH9DXsSPCJHjwZaqQ51tDV2GQLd486VF96hv5WIGbzXvEPt0UJiBGacX47+RQiSf46+g11FBb1t+sQeLm7V89ENAWMWnSdjj7TvX8LHEK0sviRU997pfm8\\\\\\\/aI6AzOWp1thNyqZb6nbUc0y9q\\\\\\\/ebtawsQ\\\\\\\/feXIqI0NTq6nHrAS+AFhjke7YGrekF7LLk7vE3jdYh8LfSFDqCIlGCCZD51PX7rAu1L2YPLvuPTe\\\\\\\/vpsEp+lrH\\\\\\\/nPHFgT4Ly+HuzCKLymzGPSy55KyTuAEfOQuyfOMvwLJi+b8+dhZNJfJNzgoQGWXunF8SCiis1+IhffGKDy3vj8PcxDXqmukQ9zpGEfDOocA1mx5NEZibcN9O62mCI2GCATWU3nV\\\\\\\/\\\\\\\/KnlpkjX2f5L7jmbtfnriKiTan8t4PmSOGe4Y8uA8nTmBlv720TI51L\\\\\\\/reYf\\\\\\\/lf3egA6O0NgfljKGCIQLaj4rsuBSmd0NgdBDMCXt0Ej1hYbaiDRk+UTLoa6IyBf9JqKAafZ525e4X6sqZZ68sDf4W6KA8HB\\\\\\\/VyajNyUTidmfIyJZKmWdOFUFNgwq7Sc3rqscagw\\\\\\\/j4cxxfijHXPtFnGdEqTlzSLFbOXz7iyjlXbcGp4eHiB\\\\\\\/nM3yGEtU5bn274Xr2hv7BPY59\\\\\\\/YpCN23NcUixlsvwn1yD7SZTW48EZmR9ITj9iMjJtd7MeQrDqAaSLXM9CyUOZiBr\\\\\\\/vwWhclJygboAFFBs3sCvnzfM9IC31PsnmRLd\\\\\\\/HODh5T2Zj1rgEhu7afyX+YvH7SrH7Q8K3kByfiOrLe2AvaVKp1EY45pRy8oMjc8wJkNPReek6GDKHEBuXoVRV5KOJ6BY8XzQ2DE48bf0mfY6U3jLrP5ZzgybXPfUN2O24Se47wVIUu1lwkVCQj9sMgYapWRxHGzHWY0GE7bSvOYoFPaPRxd8RgNLrs4Ug21dlYHoXTIOD10tOX1DOyc0THvT3Uqu2XFZbCkQKQE42crrmZzfWExSVJ6VMijKlP54QBBXNK5MTVJxd9avX808qLpiDLAQQxgY+bAp\\\\\\\/E9wGnG506ShZ+MsWnBZvVt2zo6ECC1mPByuGmuZTVVNxD+LUN6VTSZy+38hb8wl+P\\\\\\\/lotPchucVNmsEvWTdbqjDzplOWPAB+3ZzscyiJuvEoScWPPdY5G8HEw72QbWROX+6ld\\\\\\\/NIbGww9qM4h6I3g\\\\\\\/gSXZNbNfcCKudPPNv6iAk1cnmbOoOqnj56P2tZIFfUJntK3GM9dAmBhrADNhH82XCBPjCqEUjF5Ns0DEL47+DyBg+Yz9aJ5Jjlo8DeOJH9Ix4O2pLkZEfgZX74Jb0ghemHmzIdvQb\\\\\\\/mHpIuSrrAxnICW9ZuUl1oFVXm5Qrdj8Q+PxDiu6EZnvYiV2UBRtMYwYIgZwR5J0wmEn4eValkbI3qideefs14uFy\\\\\\\/ekI1DyyO+GSgWmBblKDnHGy9cnUwK+7CvLcl5AWkW6b4\\\\\\\/+TfDfuo1TLQ5KxbgNLrAfgv8d+QxKsgISdVBxfH9LqgtWUoqxSjdkj5oDeGdsCG9aNwSyJiwVohjOsZ2Fpg9\\\\\\\/G91kWnH74S04O1CgHr2WE1u1IkundHMFeRD92OlohsvqAlUidtoieiFRySOw4H0w0CaVRzZHpahB8pkZCk5NcqSKzIHuai0VPgQSVyPkpNWJzesH1jfhx4aI\\\\\\\/pNPhrF1M83tiowlPoucP0iY3tMHQX8s7picJdJGpWh2EqWhqdkACMwucCkNJJ266+gwq8TTd7jWjhoDrCKbv5611ZTHcwDdv7841JlDjIBDBfYW+H2ikXG7\\\\\\\/uWIcRnVtQ5QlUKkZHzvR7KANEUT2HvryXVrxMa4GYa7W+ZJGlhQCufJiYzwkG+xwRxsSg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FBe3YbRM+58pAQeE7fnqYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e789fc5216b72fe53a5ce9756d6bcdea1c18eb12c4e49da37875f859457384aa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493678 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2253" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vgou2k89e-LUX-vCNR6Z0sBubrTuSWzknty_xdb1DGk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2253" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bTPeHJIvSsw_5eyn7UUeFv2lpxxliriCPRMA4KrJmks" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2245" title="Asymmetric Redundancy of ZERZAUST and ZERZAUST HOMOLOG in Different Accessions of Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2267" title="Expression Signatures of Cisplatin- and Trametinib-Treated Early-Stage Medaka Melanomas" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493678" id="node493678" data-pisa="ggg;9/7/2253" data-pisa-master="ggg;g3.118.200917" data-apath="/ggg/9/7/2253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7551-9833" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin B. Stewart-Brown</span>, <span class="highwire-citation-author" data-delta="1">Qijian Song</span>, <span class="highwire-citation-author" data-delta="2">Justin N. Vaughn</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4114-9509" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zenglu Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2253-2265;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200917 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493678" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin B. Stewart-Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BB.%2BStewart-Brown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stewart-Brown%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BB.%2BStewart-Brown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7551-9833" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin B. Stewart-Brown</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Qijian Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Lab, USDA-ARS, Beltsville, MD 20705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qijian%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQijian%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin N. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Genomics and Bioinformatics Research Unit, USDA-ARS, Center for Applied Genetic Technologies, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BN.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BN.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zenglu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zenglu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZenglu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4114-9509" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zenglu Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zli@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zli@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/7/2253" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493678" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493678/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2253.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493678" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493678/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/7/2253.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493678" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493678/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/7/2253.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493678" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493678/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection (GS) has become viable for selection of quantitative traits for which marker-assisted selection has often proven less effective. The potential of GS for soybean was characterized using 483 elite breeding lines, genotyped with BARCSoySNP6K iSelect BeadChips. Cross validation was performed using RR-BLUP and predictive abilities (<em>r</em><sub>MP</sub>) of 0.81, 0.71, and 0.26 for protein, oil, and yield, were achieved at the largest tested training set size. Minimal differences were observed when comparing different marker densities and there appeared to be inflation in <em>r</em><sub>MP</sub> due to population structure. For comparison purposes, two additional methods to predict breeding values for lines of four bi-parental populations within the GS dataset were tested. The first method predicted within each bi-parental population (WP method) and utilized a training set of full-sibs of the validation set. The second method utilized a training set of all remaining breeding lines except for full-sibs of the validation set to predict across populations (AP method). The AP method is more practical as the WP method would likely delay the breeding cycle and leverage smaller training sets. Averaging across populations for protein and oil content, <em>r</em><sub>MP</sub> for the AP method (0.55, 0.30) approached <em>r</em><sub>MP</sub> for the WP method (0.60, 0.52). Though comparable, <em>r</em><sub>MP</sub> for yield was low for both AP and WP methods (0.12, 0.13). Based on increases in <em>r</em><sub>MP</sub> as training sets increased and the effectiveness of WP <em>vs.</em> AP method, the AP method could potentially improve with larger training sets and increased relatedness between training and validation sets.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li><li class="kwd"><a href="/keyword/rr-blup" class="hw-term hw-article-keyword hw-article-keyword-rr-blup" rel="nofollow">RR-BLUP</a></li><li class="kwd"><a href="/keyword/seed-composition" class="hw-term hw-article-keyword hw-article-keyword-seed-composition" rel="nofollow">Seed composition</a></li><li class="kwd"><a href="/keyword/seed-yield" class="hw-term hw-article-keyword hw-article-keyword-seed-yield" rel="nofollow">Seed yield</a></li><li class="kwd"><a href="/keyword/soybean" class="hw-term hw-article-keyword hw-article-keyword-soybean" rel="nofollow">Soybean</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Quantitative traits have proven difficult to select for using marker-assisted selection (MAS) based on the fact that they are polygenic and loci responsible for variation in these traits often have small effects. <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Meuwissen <em>et al.</em> (2001)</a> introduced the concept of genomic selection (GS) to take advantage of genotypic data to predict the performance of genotypes for complex traits. The main difference between MAS and GS, is that GS utilizes all markers across the genome to predict the performance of traits of interest, while MAS relies on a few markers to select specific QTL often associated with qualitative traits. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Heffner <em>et al.</em> (2010)</a> reported that GS provided threefold and twofold genetic gain per year compared to MAS for maize and winter wheat when costs were equivalent. With the advent of new genotyping platforms, such as single nucleotide polymorphism (SNP) beadchip arrays, Diversity array Technology (DArT), and genotyping-by-sequencing (GBS), high-throughput genotyping has made GS more affordable and efficient (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jain <em>et al.</em> 2017</a>). The basic concept behind GS is that a set of breeding materials is used as a training set (TS). The TS is both genotyped and phenotyped for traits of interest in order to calculate marker effects which then predict performance of a test set that has been genotyped but not phenotyped. These phenotypic predictions are often referred to as genomic estimated breeding values (GEBVs). To evaluate the effectiveness of GS, a process referred to as cross-validation is often implemented. Cross-validation involves bisecting a set of lines which has been both genotyped and phenotyped into a TS and a validation set (VS). The TS is used to estimate marker effects to calculate GEBVs for the VS. The GEBVs are correlated with the observed phenotypic values of the VS and this determines predictive ability (<em>r</em><sub>MP</sub>) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Combs and Bernardo 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jacobson <em>et al.</em> 2014</a>). The higher the correlation coefficient, the higher the predictive ability, and the more successful prediction is deemed to be. Prediction accuracy (<em>r</em><sub>MG</sub>) is sometimes estimated as <em>r</em><sub>MP</sub> divided by the square root of heritability (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-1.gif"/></noscript></span></span>) as a way to estimate success relative to phenotypic selection (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dekkers 2007</a>). Studies have extensively explored GS across many crops but most extensively in maize (<em>Zea mays</em> L.) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo and Yu 2007</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Crossa <em>et al.</em> 2014</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Jacobson <em>et al.</em> 2014</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lian <em>et al.</em> 2014</a>) and wheat (<em>Triticum aestivum</em> L.) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de Los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Heffner <em>et al.</em> 2011a</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Heffner <em>et al.</em> 2011b</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Crossa <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Heslot <em>et al.</em> 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rutkoski <em>et al.</em> 2015</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Isidro <em>et al.</em> 2015</a>). There are several factors that often influence the accuracy of GS. These factors include but are not limited to trait architecture and heritability, training set size and composition, marker density, and statistical model for estimation of marker effects (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jannink <em>et al.</em> 2010</a>).</p><p id="p-4">Soybean (<em>Glycine max</em> L. merr) accounted for 61% of the world’s oilseed production in 2016 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">American Soybean Association 2018</a>) and is a vital source of both protein meal for animal feed and vegetable oil for human consumption (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Huth 1995</a>). There have been several studies examining the potential for GS in soybean but relatively few compared to maize and wheat. <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jarquín <em>et al.</em> (2014)</a> was one of the first studies examining the potential for GS in soybean for seed yield prediction. They reported a prediction accuracy of 0.64 for seed yield across 301 experimental lines from the University of Nebraska-Lincoln soybean breeding program and found little improvement in accuracy when training set size (<em>N</em><sub>P</sub>) exceeded 100 breeding lines. Predicted success when performing GS tends to be higher in studies reporting results with prediction accuracy <em>vs.</em> predictive ability, especially for lower heritability traits. The BARCSoySNP6K iSelect BeadChip was used to genotype a mixed population of 235 soybean cultivars by <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ma <em>et al.</em> (2016)</a> and potential for GS was examined for plant height and seed yield. They reported an increase in prediction accuracy of 4% for yield when using haplotype block-based markers (<em>r</em><sub>MG</sub> = 0.49) <em>vs.</em> random (<em>r</em><sub>MG</sub> = 0.48) or equidistant marker sampling (<em>r</em><sub>MG</sub> = 0.47). The potential to utilize GS has also been investigated within larger populations such as the SoyNAM population which is composed of over 5500 lines across 40 bi-parental populations (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> 2016</a>). Traits investigated were seed yield, days to maturity, plant height, pod number, node number, and pods per node. They detected minimal difference in accuracy across 14 statistical models (<em>r</em><sub>MG</sub> = 0.60 - 0.61) as well as minimal difference between genotyping densities of 4077 (<em>r</em><sub>MG</sub> = 0.60) <em>vs.</em> 1020 SNPs (<em>r</em><sub>MG</sub> = 0.61). They determined the most important factor for improving accuracy was to increase training set size as they examined <em>N</em><sub>P</sub>’s from 250 (<em>r</em><sub>MG</sub> = 0.38) up to 4000 lines (<em>r</em><sub>MG</sub> = 0.75). They reported significant improvements in accuracy up to 2000 individuals. Thus far, there is no GS study in soybean with materials from later maturity groups. Soybean was second behind corn in terms of total acreage planted in 2017 (USDA-NASS 2017) and considering the importance of soybean on a national and global level, more studies are needed to characterize the potential for GS in soybean for complex traits.</p><p id="p-5">The objective of this study was to characterize the ability to perform GS in later maturity groups within an applied soybean breeding program at the University of Georgia (UGA) and explore the effects of trait architecture and heritability, training set size and composition, and genotyping marker density on prediction of seed yield (yield). Protein and oil content (protein and oil) were also predicted as these traits are important from a breeder’s perspective because of the dependence on soybean as a protein source in animal feed and as a source of vegetable oil. Predictive ability of these two traits has yet to be investigated in a soybean GS study.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant Materials</h3><p id="p-6">The original GS dataset consisted of 14 distinct experimental sets which included 540 RILs from 26 pedigrees (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Set1-8 formed four bi-parental populations (Pop1-4) composed of 84 F<sub>5:7</sub> RILs each. Two sets were stratified based on the maturity within each population. Pop1 was developed from a cross of AU02-3104 × G00-3213. AU02-3104 is a MG VII soybean line developed at Auburn University which was derived from a cross of ‘NC-Raleigh’ (PI 641156) × G92-1110 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Burton <em>et al.</em> 2006</a>). G00-3213 is also a MG VII soybean line but was developed at UGA and was derived from a cross of ‘Boggs’ × ‘N7001’ (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boerma <em>et al.</em> 2000</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Carter <em>et al.</em> 2003</a>). Pop2, 3, and 4 were developed from crosses of G10PR-10 × G10PR-56389R2, G10PR-56248R2 × G10PR-56389R2, and G93-2225 × G09PR-54329R2, respectively. These parents are MG VII soybean lines developed at UGA. These four populations were advanced using a modified single-seed descent method (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brim 1966</a>) and were within their initial year of replicated yield testing. Set9-14 consisted of 34 advanced F<sub>5:8</sub> RILs each, from multiple pedigrees, which had undergone an additional round of breeding selection based on the first year of replicated yield testing. Set9-11 consisted of RILs from 12 separate pedigrees (Ped 1-12) as well as breeding selections from Pop1 and 4 and these RILs were stratified into equal sets based on early, middle, and late maturity. Set12-14 was divided similarly but consisted of RILs from 10 separate pedigrees and breeding selections from Pop2 and 3. These 540 breeding lines represented a large portion of the diversity in the UGA Soybean Breeding pipeline. Fifty-five breeding lines were present in two separate sets (Set1-8 and Set9-14) and phenotypic data for these lines remained for best linear unbiased predictor (BLUP) calculation, but these breeding lines were removed from Set9-14 in the GS dataset to avoid biasing results by cause of having the same genotypes in both the TS and VS during prediction. Two lines from Pop2 were removed from the dataset based on improper clustering according to a principle component analysis (PCA) using genotypic data and a total of 483 lines remained within the GS dataset for analysis (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493686/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493686/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of genomic selection (GS) dataset</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Genotyping and Population Structure Analysis</h3><p id="p-7">Four hundred and eighty-five RILs were genotyped for the original prediction dataset. For each RIL, 20 seeds were planted in Styrofoam cups in a University of Georgia greenhouse facility. Once seedling were 3 weeks old, leaf tissue was harvested in 50-ml Falcon tubes, lyophilized, and ground into fine powder for DNA extraction. DNA was extracted using a modified CTAB (cetyl trim ethyl ammonium bromide) method (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Keim <em>et al.</em> 1988</a>). Genotyping was performed at Soybean Genomics and Improvement Lab at USDA-ARS, Beltsville, MD using BARCSoySNP6K iSelect BeadChips, returning 5403 SNPs (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Song <em>et al.</em> 2013</a>). Physical distances of SNPs were initially from the genome assembly version Glyma.Wm82.a1 (Gmax1.01) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schmutz <em>et al.</em> 2010</a>) and were then converted to version Glyma.Wm82.a2 (Gmax2.0). SNPs mapped in Gmax1.01 but not Gmax2.0, were excluded from the analysis (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Song <em>et al.</em> 2016</a>).</p><p id="p-8">In addition to monomorphic SNPs, SNPs with &gt; 10% heterozygous genotypes, &gt; 80% missing data, or minor allele frequency (MAF) &lt; 0.05 were removed. There were 2647 polymorphic SNPs remaining for GS. Various marker densities (<em>N</em><sub>M</sub>) were investigated for their effect on <em>r</em><sub>MP</sub>. The <em>N</em><sub>M</sub> categories tested were 1) all SNPs (2647 SNPs); 2) tag SNPs (yield: 1459 SNPs, protein and oil: 1435 SNPs); 3) half tag (yield: 748 SNPs, protein and oil content: 718 SNPs); 4) 4<sup>th</sup> tag (yield: 374 SNPs, protein and oil: 359 SNPs); 5) 8<sup>th</sup> tag (yield: 187 SNPs, protein and oil: 180 SNPs). Each tag SNP represents a genomic region with high linkage disequilibrium (LD). Depending on the trait, the number of tag SNPs varied as a result of variation in the composition of the GS dataset. Tag SNPs were determined using tagger in Haploview using pairwise tagging only and an r<sup>2</sup> threshold set at 0.8 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">de Bakker <em>et al.</em> 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrett <em>et al.</em> 2005</a>). From the set of tag SNPs, every other marker was selected as half tag, every fourth marker as 4<sup>th</sup> tag, and every eighth as 8<sup>th</sup> tag. Population structure was examined using BARCSoySNP6K iSelect BeadChip data and the GAPIT R package (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lipka <em>et al.</em> 2012</a>). PCA’s were plotted for visualization using TIBCO Spotfire 6.5.1 (2014).</p></div><div id="sec-4" class="subsection"><h3>Phenotyping</h3><p id="p-9">For Pop1-4, 84 RILs from each population were divided into two equal sets of 42 based on maturity for yield trials, and two elite checks were included in each set. Yield trials were conducted in two locations in Georgia (Athens and Plains) for Set1-8 over 2 years. For Set1-2, yield evaluations were conducted in 2014 and 2016, while Set3-6 were evaluated in 2015 and 2016. Set7-8 were evaluated in 2014 but only Athens in 2016 due to lack of seed. Sets evaluated in 2014 or 2015 were replicated in two blocks per environment in a randomized complete block design (RCBD) and sets evaluated in 2016 were replicated three blocks per environment. In addition to select lines from the Pop1-4, breeding lines from Ped1-12 and Ped13-22 were allocated to Set9-11 and Set12-14 by maturity. Yield evaluations were performed in 2015 for Set9-11 and in 2016 for Set12-14. Each set consisted of 34 RILs and two elite checks which were replicated in three blocks per environment in an RCBD and were evaluated at three of four locations (Athens, Plains, and Tifton, GA or Florence, SC).</p><p id="p-10">Set1-8 were planted in two-row plots, 4.9 m long and 76 cm apart. Plots were end trimmed to 3.7 m at R5 or R6 stage. Both rows were harvested for yield determination and adjusted to 13% moisture. For Georgia locations, Set9-14 were planted in the same way except in four-row plots and the plots at the Tifton location were not end-trimmed. For the Florence location, RILs were planted in four-row plots which were 6.1 m long and 76 cm apart. Plots were end-trimmed to 5.5 m at R5 or R6 stage. The middle two rows were harvested for yield determination and adjusted to 13% moisture.</p><p id="p-11">Days to maturity was defined as the number of days from 1 September to maturity and was recorded on all blocks at the Athens location. Seed composition (protein and oil content) were measured from the same seed sources harvested for yield evaluation. Seed composition was not measured for Set 9-11 because of seed quality issues in 2015, resulting in no seed composition measurements obtained for any genotypes that year. For Set1-6 and 12-14, seed composition was measured from both Athens and Plains in 2016. Set1-2 also had seed composition measured from both locations in 2014. Seed composition was measured for Set 7-8 in 2014 and 2016 but only from Athens. Crude protein and oil were analyzed on a sample of ∼250 seeds from each plot using a DA 7250 NIR analyzer (Perten, Springfield, IL).</p></div><div id="sec-5" class="subsection"><h3>BLUP and Heritability</h3><p id="p-12">BLUP values were calculated using the lme4 package (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bates <em>et al.</em> 2015</a>) in R for each genotype and trait to account for variation resulting from environmental factors and maturity. Factors in the random model for yield included genotype, environment (a combination of year and location), genotype x environment interaction, set within environment, and days to maturity. Factors in the random models for protein and oil content included genotype, environment, genotype × environment interaction, and set within environment. To investigate the normality of BLUP values for each phenotypic trait, kernel density plots were created in R.</p><p id="p-13">Heritability was estimated for each trait utilizing the rrBLUP package (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Endelman 2011</a>) implemented in R. An additive relationship matrix was created using the A.mat function. Utilizing the additive relationship matrix and phenotypic BLUP values for each genotype, genetic and error variances were estimated using the kin.blup function. Narrow-sense heritability was then estimated using the additive genetic (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-2.gif"/></noscript></span></span> and error variance (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-3.gif"/></noscript></span></span> outputs from kin.blup using the following equation: <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-4.gif"/></noscript></span></span> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Endelman and Jannink 2012</a>). Utilizing the efficient mixed model association algorithm (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kang <em>et al.</em> 2008</a>), restricted maximum log-likelihood was used to calculate variance components from a mixed model. The mixed model implemented is defined here as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-5.gif"/></noscript></span></span>), where y is a vector of phenotypic values, 1 is a column vector of ones, μ is a fixed effect for the overall mean, Z is the genotypic incidence matrix, and G is the genomic relationship matrix. This notation for calculation of genomic heritability (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Los Campos <em>et al.</em> 2015</a>) is also represented in <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> (2016)</a>.</p></div><div id="sec-6" class="subsection"><h3>Genomic Prediction</h3><p id="p-14">The ridge regression best linear unbiased prediction (RR-BLUP) modeling methodology was utilized for GS using the rrBLUP package (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Endelman 2011</a>) implemented in R. Three different genomic prediction methods were investigated for three traits: yield, protein, and oil. In the first two methods, both TS and VS were established, and cross-validation was conducted by taking a random sample at various TS sizes and predicting GEBVs of genotypes in the VS. The third method was performed in the same manner, but with the TS and VS from separate pools. The correlation between GEBVs and the observed BLUP values was recorded. The procedure was replicated 100 times and <em>r</em><sub>MP</sub> was the average of these 100 replications.</p><div id="sec-7" class="subsection"><h4>Genomic prediction across entire genomic selection dataset (EGSD method):</h4><p id="p-15">The first prediction method examined the ability to predict when the TS and VS were pulled from the entire dataset at random. Predicting across mixed populations is a common approach for evaluating <em>r</em><sub>MP</sub> and provides a general idea of how well GS can function across all breeding materials (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> 2016</a>). For the EGSD method, <em>r</em><sub>MP</sub> for each trait was calculated across the entire dataset of 483 RILs for yield and 401 RILs for protein and oil (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The VS was composed of 50 randomly selected breeding lines from the entire dataset. The TS was composed of randomly selected breeding lines from the remaining genotypes at various TS sizes. Predictive ability was measured with marker density fixed at all SNPs at the following TS sizes: 50, 100, 150, 200, 250, 300, 350, and 400. For protein and oil, TS size was maximum at 350 as a result of a smaller subset of genotypes. Utilizing the maximum TS size for each trait, the effect of marker density on r<sub>MP</sub> was investigated for various numbers of SNPs including all, tag, half tag, 4<sup>th</sup> tag, and 8<sup>th</sup> tag SNPs.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram displaying the three methods performed for estimating predictive ability within the genomic selection dataset. (A) Perform cross-validation using the entire mixed population as both the validation set and training set (EGSD method), (B) Perform cross-validation within bi-parental populations using Pop1-4 individually as the validation set and training set (WP method); and (C) Predict across populations using one of Pop1-4 as the validation set and the remaining breeding lines as the training set (AP method)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1191541004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diagram displaying the three methods performed for estimating predictive ability within the genomic selection dataset. (A) Perform cross-validation using the entire mixed population as both the validation set and training set (EGSD method), (B) Perform cross-validation within bi-parental populations using Pop1-4 individually as the validation set and training set (WP method); and (C) Predict across populations using one of Pop1-4 as the validation set and the remaining breeding lines as the training set (AP method).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2253/F1.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/7/2253/F1.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493516" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Diagram displaying the three methods performed for estimating predictive ability within the genomic selection dataset. (A) Perform cross-validation using the entire mixed population as both the validation set and training set (EGSD method), (B) Perform cross-validation within bi-parental populations using Pop1-4 individually as the validation set and training set (WP method); and (C) Predict across populations using one of Pop1-4 as the validation set and the remaining breeding lines as the training set (AP method).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h4>Genomic prediction within bi-parental RIL population (WP method):</h4><p id="p-17">Another goal of this study was to examine how well GS would function for predicting GEBVs within specifically each of the four bi-parental populations (Pop1-4), named as WP method. Both the TS and VS were the same bi-parental population and thus, full-sibs were used to predict full-sibs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The GS dataset contained four bi-parental populations with 84 RILs each (Pop1-4). Cross-validation was performed similar to the EGSD method, except within Pop1-4 and a VS size of 20 RILs was used. Predictive ability was measured with marker density fixed at all SNPs at a TS size of 50 RILs. A limiting factor for the WP method is that TS size becomes restricted by the size of the bi-parental population. Another reason that this method may not be ideal is that the breeder would need replicated yield trials on a subset of a bi-parental population to generate phenotypic data and to train a model with which they can return to remnant seed of additional RILs to decide which plant rows to select for advancement (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Jannink <em>et al.</em> 2010</a>). For the purpose of this study, this strategy served mainly as a contrast to the third and most ideal GS method in which one of the four populations (Pop1-4) was the VS and all remaining breeding lines were compiled as the TS (AP method).</p></div><div id="sec-9" class="subsection"><h4>Genomic prediction across bi-parental RIL populations (AP method):</h4><p id="p-18">The AP method examined predictive ability when the VS was created from one of Pop1-4, but the TS was developed from the remaining breeding lines (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). This method simulated a situation similar to how GS would actually be implemented in a breeding program in order to select better breeding lines within a newly developed population when no phenotypic data are available. Predictive ability was measured with marker density fixed at all SNPs and examined the following TS sizes: 50, 100, 150, 200, 250, 300, 350. For protein and oil, the largest TS size tested was 300, due to a smaller subset of genotypes having been phenotyped for these traits. Comparing WP and AP methods allows for an investigation of the ability to compensate for a decrease in genetic relatedness with an increase in TS size that can be achieved when using the AP method.</p></div></div><div id="sec-10" class="subsection"><h3>Statistical Analysis</h3><p id="p-19">All significance tests of correlation were calculated using the Pearson’s product-moment correlation method via the ggpubr package (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kassambara 2017</a>) in R. ANOVA was performed for each trait using the agricolae package (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">de Mendiburu 2017</a>) to examine if there were statistical differences in <em>r</em><sub>MP</sub> resulting from changes in TS sizes and marker sets. For the ANOVA model, the dependent variable was predictive ability from each replication cycle and the independent variable was the factor of interest. A Fisher’s LSD multiple comparison test was performed to test differences of the means between different levels of each factor (α = 0.05).</p></div><div id="sec-11" class="subsection"><h3>Data Availability</h3><p id="p-20">All data and code required to replicate the analyses are available in Files S1-5. File S1 contains the raw phenotypic data for calculation of BLUP values. File S2 contains the phenotypic BLUP values used for each method of GS. Worksheet 1 provides additional information for each genotype while Worksheets 2 and 3 contain information used for GS. Supplemental Data File S3 and S4 contains the genotypic data files used for prediction of seed yield and protein/oil content for each method of GS. Worksheet 1 of each file provides additional information for each genotype while Worksheets 1-10 contain genotypic data for the following marker densities: all SNPs, tag SNPs, half tag SNPs, 4<sup>th</sup> tag SNPs, and 8<sup>th</sup> tag SNPs. Odd sheets contain extra information while even sheets contain data used for GS. File S5 provides the r code used for calculation of predictive abilities which can be adapted to test all methods. Supplemental data files along with supplemental figures and tables are available at FigShare. R code used for calculation of predictive abilities is also available at FigShare. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8121251">https://doi.org/10.25387/g3.8121251</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Population structure and genomic heritability</h3><p id="p-21">The GS dataset showed significant population structure due to the presence of four bi-parental RIL populations (Pop1-4) composing more than half of the entire dataset. The first, second, and third principal components explained 12.9, 9.5, and 7.0% of variation within the dataset, respectively (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). There was clear clustering within each of the four bi-parental populations and population structure among some of the advanced breeding lines from Ped5-22 was observed as several lines shared the same parentage. High genetic relatedness among many breeding lines was observed that led to clustering among several advanced breeding lines with Pop1-4.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principle component analysis of genomic selection dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1191541004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principle component analysis of genomic selection dataset.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2253/F2.medium.gif" width="440" height="373"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/7/2253/F2.medium.gif" width="440" height="373"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493565" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Principle component analysis of genomic selection dataset.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">BLUP values for each trait followed a normal distribution (Fig. S1). When RILs were separated by pedigree, it became evident that pedigrees varied in terms of their mean BLUP values for each trait (Fig. S2). When focusing on Pop1-4 which contributed to a majority of the GS dataset, there was evidence of populations which were numerically different in terms of protein and oil. Pop1 had higher oil and lower protein content compared to Pop2-4 in terms of mean BLUP values (Fig. S2B; Fig. S2C). Overall, protein and oil were significantly negatively correlated (r = −0.62; <em>P</em> &lt; 1× 10<sup>−15</sup>). This supports prior reports of the inverse relationship between protein and oil content in soybean (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brummer <em>et al.</em> 1997</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brim and Burton 1979</a>). Yield varied across pedigrees but the relative differences in yield between the four bi-parental populations compared to across all pedigrees was minimal (Fig. S2A). Though the correlations were not as strong, yield had a significant negative correlation with protein content (r = −0.10; <em>P</em> = 4.6 × 10<sup>−2</sup>) and had a significant positive correlation with oil content (r = 0.11; <em>P</em> = 3.4 × 10<sup>−2</sup>), which is consistent with previous reports (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chung <em>et al.</em> 2003</a>).</p><p id="p-24">BLUP values and an additive matrix of breeding material were used to compute genomic heritability for each trait via the kin.blup function in rrBLUP (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Endelman 2011</a>). Protein had the highest heritability with a genomic heritability of 0.82. Oil had a genomic heritability of 0.78 and yield had the lowest heritability trait at 0.17. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hwang <em>et al.</em> (2014)</a> reported similarly high heritability estimates for protein and oil content and it is widely reported that yield is a low heritability trait for many crops, including soybean.</p></div><div id="sec-14" class="subsection"><h3>Predictive ability across entire GS dataset (EGSD)</h3><p id="p-25">Predictive ability for yield increased by 364% from 0.06 (<em>N</em><sub>P</sub> = 50) to 0.26 (<em>N</em><sub>P</sub> = 400) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, percentages/significance tests were based on <em>r</em><sub>mp</sub> values in Table S1). As TS size increased by 50, predictive ability increased on average by 0.03. There were no significant differences in <em>r</em><sub>MP</sub> from a TS size of 300 (<em>r</em><sub>MP</sub> = 0.24) to 400 (<em>r</em><sub>MP</sub> = 0.26). Marker density appeared to have less impact on predictive ability compared to TS size. When comparing different marker densities, <em>r</em><sub>MP</sub> ranged from 0.30 (<em>N</em><sub>M</sub> = 8<sup>th</sup> tag SNPs) to 0.24 (<em>N</em><sub>M</sub> = half tag SNPs) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, percentages/significance tests were based on <em>r</em><sub>mp</sub> values in Table S2). Utilizing 8<sup>th</sup> tag SNPs was only 0.04 greater in terms of <em>r</em><sub>MP</sub> compared to utilizing all SNPs so minor differences were present among marker densities.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots of the effect of training set size (NP) on predictive ability (rMP) for each trait when utilizing the entire genomic selection dataset (EGSD) method. Solid line represents median and dotted line represents mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1191541004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Boxplots of the effect of training set size (&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;P&lt;/sub&gt;) on predictive ability (&lt;em&gt;r&lt;/em&gt;&lt;sub&gt;MP&lt;/sub&gt;) for each trait when utilizing the entire genomic selection dataset (EGSD) method. Solid line represents median and dotted line represents mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2253/F3.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/7/2253/F3.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493483" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Boxplots of the effect of training set size (<em>N</em><sub>P</sub>) on predictive ability (<em>r</em><sub>MP</sub>) for each trait when utilizing the entire genomic selection dataset (EGSD) method. Solid line represents median and dotted line represents mean.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots of the effect of marker density (NM) on predictive ability (rMP) for each trait when utilizing the entire genomic selection dataset (EGSD) method. Number of markers indicated in parentheses. Solid line represents median and dotted line represents average." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1191541004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Boxplots of the effect of marker density (&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;M&lt;/sub&gt;) on predictive ability (&lt;em&gt;r&lt;/em&gt;&lt;sub&gt;MP&lt;/sub&gt;) for each trait when utilizing the entire genomic selection dataset (EGSD) method. Number of markers indicated in parentheses. Solid line represents median and dotted line represents average.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2253/F4.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/7/2253/F4.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493511" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Boxplots of the effect of marker density (<em>N</em><sub>M</sub>) on predictive ability (<em>r</em><sub>MP</sub>) for each trait when utilizing the entire genomic selection dataset (EGSD) method. Number of markers indicated in parentheses. Solid line represents median and dotted line represents average.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">For protein, TS size had a significant impact as well, evidenced by an increase in <em>r</em><sub>MP</sub> of 29% from 0.63 (<em>N</em><sub>P</sub> = 50) to 0.81 (<em>N</em><sub>P</sub> = 350) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and Table S1). The average increase in <em>r</em><sub>MP</sub> for each increase in TS size of 50 was 0.03, but gains were higher during the initial increase from 50 (<em>r</em><sub>MP</sub> = 0.63) to 100 (<em>r</em><sub>MP</sub> = 0.70). Predictive ability for protein began to diminish at larger TS sizes as <em>r</em><sub>MP</sub> only increased from 0.80 to 0.81 when TS size increased from 250 to 350. Marker density also had less impact on <em>r</em><sub>MP</sub> compared to TS size as predictive ability for protein decreased by only 8% from 0.81 (<em>N</em><sub>M</sub> = all SNPs) to 0.74 (<em>N</em><sub>M</sub> = 8<sup>th</sup> tag SNPs) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and Table S2).</p><p id="p-29">Oil was no exception to the trend of larger TS sizes resulting in higher predictive ability. Predictive ability increased by 31% from 0.54 (<em>N</em><sub>P</sub> = 50) to 0.71 (<em>N</em><sub>P</sub> = 350) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and Table S1). Similar to protein, the average gain in <em>r</em><sub>MP</sub> for each increase in TS size of 50 was 0.03, but the largest increase was observed from 50 (<em>r</em><sub>MP</sub> = 0.54) to 100 (<em>r</em><sub>MP</sub> = 0.61). Increases in <em>r</em><sub>MP</sub> were minimal as TS size increased from 250 (<em>r</em><sub>MP</sub> = 0.68) to 350 (<em>r</em><sub>MP</sub> = 0.71). As marker density decreased so did <em>r</em><sub>MP</sub> but the decrease was only 9% from 0.71 (<em>N</em><sub>M</sub> = all SNPs) to 0.64 (<em>N</em><sub>M</sub> = 8<sup>th</sup> tag SNPs) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and Table S2).</p><p id="p-30">There appeared to be a direct relationship between heritability and <em>r</em><sub>MP</sub> as the highest heritability traits (oil and protein) were more predictive than yield which had a lower heritability. By cause of the larger number of breeding lines which had been phenotyped for yield, a slightly larger TS size was tested compared to the other traits but when comparing traits at equal TS sizes, protein and oil were consistently higher than yield. For protein and oil, the highest predictive ability was achieved with all SNPs, while the highest predictive ability for yield was achieved with the lowest marker density of 8<sup>th</sup> tag SNPs (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> and Table S2). Considering the SNP distribution decreased from ∼130 SNPs per chromosome (all SNPs) to ∼10 SNPs per chromosome (8<sup>th</sup> tag SNPs), a more dramatic decrease in <em>r</em><sub>MP</sub> across all traits may have been anticipated. Overall, though statistical differences were present, it did not appear that decreasing marker density had a drastic effect on <em>r</em><sub>MP</sub> for any trait. On average, the difference in <em>r</em><sub>MP</sub> between the highest and lowest marker density across all traits was only 0.03.</p></div><div id="sec-15" class="subsection"><h3>Predictive ability of individual bi-parental populations (WP <em>vs.</em> AP method)</h3><div id="sec-16" class="subsection"><h4>Predictive ability averaged across populations (Pop1-4):</h4><p id="p-31">The ability to predict lines within a bi-parental population using full-sib members of that population (WP method) <em>vs.</em> using the remaining breeding lines (AP method) was examined. For the initial analysis, r<sub>MP</sub> was averaged across Pop1-4 for each trait at each TS size. Yield was the lowest heritability trait and achieved the lowest r<sub>MP</sub> for WP (r<sub>MP</sub> = 0.13) and AP (r<sub>MP</sub> = 0.12) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, percentages/significance tests were based on r<sub>mp</sub> values in Table S3). Predictive ability for the AP method ranged from 0.04 (N<sub>P</sub> = 50) to 0.12 (N<sub>P</sub> =350). There were no statistical differences in predictive ability between a TS size of 300 or 350 for the AP method and a TS size of 50 for the WP method, as each achieved an r<sub>mp</sub> of 0.13. When comparing both methods at an equal TS size (N<sub>P</sub> = 50), predictive ability for the WP method was 205% higher than the AP method (0.13 <em>vs.</em> 0.04). Though this difference was significant, the WP method was still quite low in terms of predictive ability.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graph displaying the effect of training set size (NP) on predictive ability (rMP) for each trait when contrasting the within population (WP) method vs. the across population (AP) method. rMP was averaged across the four validation sets (Pop1-4). The WP method was indicated with a horizontal dashed line while the AP method was indicated with a solid trend line across TS sizes. For the WP method, a single training set size of 50 breeding lines was used." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1191541004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graph displaying the effect of training set size (&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;P&lt;/sub&gt;) on predictive ability (&lt;em&gt;r&lt;/em&gt;&lt;sub&gt;MP&lt;/sub&gt;) for each trait when contrasting the within population (WP) method &lt;em&gt;vs.&lt;/em&gt; the across population (AP) method. &lt;em&gt;r&lt;/em&gt;&lt;sub&gt;MP&lt;/sub&gt; was averaged across the four validation sets (Pop1-4). The WP method was indicated with a horizontal dashed line while the AP method was indicated with a solid trend line across TS sizes. For the WP method, a single training set size of 50 breeding lines was used.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2253/F5.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/7/2253/F5.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493680" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Graph displaying the effect of training set size (<em>N</em><sub>P</sub>) on predictive ability (<em>r</em><sub>MP</sub>) for each trait when contrasting the within population (WP) method <em>vs.</em> the across population (AP) method. <em>r</em><sub>MP</sub> was averaged across the four validation sets (Pop1-4). The WP method was indicated with a horizontal dashed line while the AP method was indicated with a solid trend line across TS sizes. For the WP method, a single training set size of 50 breeding lines was used.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Protein was the highest heritability trait and achieved the highest <em>r</em><sub>MP</sub> for both WP (<em>r</em><sub>MP</sub> = 0.60) and AP methods (<em>r</em><sub>MP</sub> = 0.55). Predictive ability for the AP method ranged from 0.34 (<em>N</em><sub>P</sub> = 50) to 0.55 (<em>N</em><sub>P</sub> = 300) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and Table S3). For the AP method, the largest TS sizes of 250 and 300 were statistically equivalent in terms of <em>r</em><sub>MP</sub> (0.53 and 0.55). There was a 9% increase in <em>r</em><sub>MP</sub> when implementing the WP (<em>r</em><sub>MP</sub> = 0.60) <em>vs.</em> the AP method (<em>r</em><sub>MP</sub> = 0.55) at the maximum TS size (<em>N</em><sub>P</sub> = 300). When comparing both methods at an equal TS size (<em>N</em><sub>P</sub> = 50), predictive ability for the WP method was 80% higher than the AP method (0.60 <em>vs.</em> 0.34).</p><p id="p-34">Oil was the second highest heritability trait and achieved the second highest <em>r</em><sub>MP</sub> for WP (<em>r</em><sub>MP</sub> = 0.52) and AP (<em>r</em><sub>MP</sub> = 0.30) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> and Table S3). Predictive ability for the WP method was comparable to protein but when comparing values for the AP method was almost half. Predictive ability for the AP method ranged from 0.21 (<em>N</em><sub>P</sub> = 50) to 0.30 (<em>N</em><sub>P</sub> = 300) and TS sizes from 200 to 300 were statistically equivalent in terms of <em>r</em><sub>MP</sub> (0.27 to 0.30). There was an increase in <em>r</em><sub>MP</sub> of 76% when implementing the WP (<em>r</em><sub>MP</sub> = 0.52) <em>vs.</em> the AP method (<em>r</em><sub>MP</sub> = 0.30) at the maximum TS size (<em>N</em><sub>P</sub> = 300). When comparing both methods at an equal TS size (<em>N</em><sub>P</sub> = 50), predictive ability for the WP method was 149% higher than the AP method (0.52 <em>vs.</em> 0.21), comparable to protein. Both protein and oil had smaller increases in percentage compared to yield, but this was largely influenced by how low the predictive ability was for yield when utilizing the AP method.</p><p id="p-35">For each trait, a higher or at least equivalent predictive ability was achievable when implementing WP <em>vs.</em> AP even though the maximum TS size achievable for AP was significantly larger (oil and protein: 50 <em>vs.</em> 300, yield: 50 <em>vs.</em> 350). When comparing both methods at an equal TS size of 50 (max <em>N</em><sub>P</sub> for WP), predictive ability was higher when implementing WP <em>vs.</em> AP for all traits, further highlighting the advantage of the WP <em>vs.</em> AP method.</p></div><div id="sec-17" class="subsection"><h4>Predictive ability of each individual bi-parental population (Pop1-4):</h4><p id="p-36">After investigating how the WP and AP methods compared on average across Pop1-4, individual populations were investigating to see if there were trends unique to any individual population. For yield, Pop1-4 achieved an <em>r</em><sub>MP</sub> of 0.04, 0.21, 0.25, and 0.01, respectively, when utilizing the WP method. (Fig. S3A, percentages/significance tests were based on <em>r</em><sub>mp</sub> values in Table S4). For the AP method, Pop1-4 achieved a maximum <em>r</em><sub>MP</sub> of 0.12 (<em>N</em><sub>P</sub> = 250 or 350), 0.10 (<em>N</em><sub>P</sub> = 350), 0.11 (<em>N</em><sub>P</sub> = 300), and 0.18 (NP = 350), respectively (Fig. S3A and Table S4). Predictive ability for yield was overall significantly lower than those for protein and oil in each population (Fig. S3A). As TS size increased for AP, <em>r</em><sub>MP</sub> tended to increase but fluctuated drastically throughout this trend. The WP method was significantly more effective in Pop2 and Pop3 compared to the AP method. For Pop1 and Pop4, the WP method performed poorly, and the highest prediction was achieved when implementing the AP method. Yield was far more population dependent compared to protein and oil in terms of prediction.</p><p id="p-37">For protein, Pop1-4 achieved an <em>r</em><sub>MP</sub> of 0.64, 0.73, 0.61, and 0.43, respectively, when utilizing the WP method (Fig. S3B and Table S4). For the AP method, Pop1-4 achieved a maximum <em>r</em><sub>MP</sub> of 0.57 (<em>N</em><sub>P</sub> = 300), 0.55 (<em>N</em><sub>P</sub> = 300), 0.64 (<em>N</em><sub>P</sub> = 300), and 0.45 (<em>N</em><sub>P</sub> = 300), respectively (Fig. S3B and Table S4). As TS size increased, <em>r</em><sub>MP</sub> tended to increase when utilizing the AP method (Fig. S3B). For Pop1 and Pop2, <em>r</em><sub>MP</sub> for WP was significantly higher compared to AP when comparing the highest measured <em>r</em><sub>MP</sub> for each population. When comparing <em>r</em><sub>MP</sub> for Pop4, there was no significant difference between WP and AP at the largest tested TS size. Utilizing AP for Pop3, predictive ability of WP was surpassed starting at a TS size of 200. Though predictive ability for AP was higher, there were no significant differences when compared to WP at a TS size of 50.</p><p id="p-38">For oil, Pop1-4 achieved an <em>r</em><sub>MP</sub> of 0.64, 0.36, 0.63, and 0.46, respectively, when utilizing the WP method (Fig. S3C and Table S4). When utilizing the AP method, Pop1-4 achieved a maximum <em>r</em><sub>MP</sub> of 0.12 (<em>N</em><sub>P</sub> = 50 or 200), 0.25 (<em>N</em><sub>P</sub> = 250 or 300), 0.48 (<em>N</em><sub>P</sub> = 250) and 0.36 (<em>N</em><sub>P</sub> = 300) (Fig. S3C and Table S4). Similar to protein, as TS size increased, <em>r</em><sub>MP</sub> tended to increase for AP (Fig. S3C). Even though the highest TS size did not always possess the highest <em>r</em><sub>MP</sub> in each individual population, it was statistically equivalent for each. WP was significantly more effective for prediction of Pop1-4 compared to AP. When comparing the highest <em>r</em><sub>MP</sub> for each population, Pop1 showed the largest discrepancy in ability to predict as AP was 18% of WP in terms of <em>r</em><sub>MP</sub>. Predictive ability utilizing AP for Pop2-4 was on average 76% of <em>r</em><sub>MP</sub> utilizing WP. Also, Pop1 seemed to be the only population where predictive ability stagnated completely as there were no significant differences from 50 to 300 lines.</p></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-39">In previous literature, GS has shown potential to improve the rate of genetic gain over MAS for quantitative traits. Studies have been performed extensively in crops such as maize and wheat, but soybean has had comparably few studies investigating the potential for GS. Predictive ability for yield was targeted in this study as increasing yield is a primary focus of soybean breeders. The potential to perform GS for protein and oil was also investigated as it is important to increase protein and oil considering soybean is the main source of protein for animal feed and a major source of vegetable oil.</p><p id="p-40">Three distinct methods of evaluating potential for GS were tested. The EGSD method was the most traditional approach in which the entire dataset was sampled for both the TS and VS. Two additional methods were then compared to examine how GS performed within bi-parental populations when genetic relationships were strongest, compared to a realistic scenario in which GEBVs were predicted for RILs within each bi-parental population using all other breeding lines as a training population. This last method demonstrates the most efficient way that GS could be implemented within a breeding program for plant row selection in order to make more informative decisions on which genotypes should be placed into advanced yield trials in cooperation with breeder notes.</p><div id="sec-19" class="subsection"><h3>Predictive ability across entire GS dataset (EGSD method)</h3><p id="p-41">When performing cross-validation across the entire dataset, increasing TS size showed continuous increases in <em>r</em><sub>MP</sub> for all three traits of interest. Predictive abilities of 0.81, 0.71, and 0.26 for protein, oil, and yield, respectively, were achieved at the largest tested training set size. <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> (2016)</a> reported prediction accuracies of 0.64 and 0.75 for yield, which were calculated using <em>r</em><sub>MP</sub> divided by <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-6.gif"/></noscript></span></span>. Prediction accuracy, especially for lower heritability traits such as yield, is often much higher than predictive ability as a result of dividing by <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/7/2253/embed/mml-math-7.gif"/></noscript></span></span>. This was observed in this study as prediction accuracies of 0.89, 0.80, and 0.63 were calculated for protein, oil, and yield, respectively, which is comparable with previous reports (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> 2016</a>). Though increases in <em>r</em><sub>MP</sub> continued as TS size increased, it appeared that gains for each trait diminished around 250 to 300 RILs. <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Jarquín <em>et al.</em> (2014)</a> performed a cross-validation analysis in a mixed soybean population for yield and reported a similar result in that prediction accuracy increased as TS size increased, yet they witnessed a plateau in yield prediction around 100 breeding lines. <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> (2016)</a> performed cross-validation across the entire SoyNAM population for yield and reported significant increases in prediction accuracy up to 2000 RILs. Different populations contain different levels of LD and substructure, so the ideal TS size for GS may be population dependent. Many studies have corroborated though that an increase in TS size will often result in an increase of <em>r</em><sub>MP</sub> with eventual diminishing returns (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Heffner <em>et al.</em> 2011a</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Heffner <em>et al.</em> 2011b</a>; <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> 2016</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zhang <em>et al.</em> 2017</a>). The improved ability to predict GEBVs as TS size increased is a reflection of the fact that there is an increased replication of alleles within a TS, allowing for a well-trained GS model. At smaller sizes, breeding lines with poor phenotypic data can negatively influence accurate estimations of allele effects. These outlier breeding lines are offset by increased replication as the TS size increases (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Muir 2007</a>). Also, as TS size increases, rare allele frequencies increase, which will help improve estimations of these marker effects (<a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Jarquín <em>et al.</em> 2014</a>).</p><p id="p-42">Genomic heritability was calculated for each trait utilizing BLUP phenotypic values for each genotype and an additive kinship matrix via the kin.blup function (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Endelman 2011</a>). The narrow sense heritability estimates for protein, oil, and yield were 0.82, 0.78, and 0.17. It is not surprising that protein and oil content have higher heritability relative to yield based on the complex trait architecture and interactions both epistatically and environmentally that are often associated with yield. Heritability estimates for yield were low compared to previous GS studies investigating prediction potential for yield in soybean. <a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> (2016)</a> calculated heritability in a similar manner, estimating a heritability of 0.49 in 2013 and 0.41 in 2014 for yield. Since their heritability estimates were broken up by year, this eliminated the variance associated with genotype × year interactions.</p><p id="p-43">Traits with higher heritability having higher <em>r</em><sub>MP</sub> values is a common occurrence in GS studies. <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Combs and Bernardo (2013)</a> observed this trend with few exceptions when analyzing <em>r</em><sub>MP</sub> in maize, barley (<em>Hordeum vulgare</em> L.) and wheat populations. <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Heffner <em>et al.</em> (2011b)</a> and <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> (2011)</a> also reported similar results. As there is often a strong relationship reported between heritability and <em>r</em><sub>MP</sub>, increasing the heritability of the trait by improving phenotyping accuracy utilized in a GS model can be useful for better prediction. As a breeding program increases the size of the TS used for GS, one should investigate environments (location × year combinations) in which phenotypic traits have shown to have unusually low heritability as this may have been caused by odd environmental factors specific to that location within that year.</p><p id="p-44">When holding TS size constant at the highest tested size for each trait, protein and oil content showed a decrease in <em>r</em><sub>MP</sub> as marker density decreased from all SNPs (2647 SNPs) to 8<sup>th</sup> tag SNPs (180 SNPs), but the decrease in <em>r</em><sub>MP</sub> was only 0.07 for oil and 0.06 for protein. For yield, there were slight fluctuations in this trend and the highest <em>r</em><sub>MP</sub> was achieved with the lowest <em>N</em><sub>M</sub>. <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Muir (2007)</a> reported that increasing marker density can actually lead to a decrease in prediction accuracy in some situations and this is related to the increase in collinearity between markers (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Whittaker <em>et al.</em> 2000</a>). If TS sizes are not large enough, it is also possible that marker effects can be overestimated and this problem is confounded by the increased number of markers used for genotyping. <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Lorenzana and Bernardo (2009)</a> demonstrated fluctuations in prediction accuracy related to marker density as they evaluated prediction of several agronomic traits within maize and barley populations. Within the maize population BM-TC1, they reported a higher accuracy at a marker density of 256 SNPs (<em>r</em><sub>MG</sub> = 0.56) compared to marker densities of 512 (<em>r</em><sub>MG</sub> = 0.55) and 768 SNPs (<em>r</em><sub>MG</sub> = 0.54). When assessing prediction of glucose concentration within the same population, they reported the highest accuracy achieved at a marker density of 512 SNPs (<em>r</em><sub>MG</sub> = 0.69), which was higher than the accuracy reported at the highest marker density of 1024 SNPs (<em>r</em><sub>MG</sub> = 0.67). Within a barley population derived from ‘Steptoe’ × ‘Morex’, a higher or equivalent accuracy was reported for grain yield and grain protein at 128 SNPs (<em>r</em><sub>MG</sub> = 0.62, 0.82) <em>vs.</em> the highest density at 223 SNPs (<em>r</em><sub>MG</sub> = 0.62). In this study, the difference between using all SNPs (2647) and 8<sup>th</sup> tag SNPs (187 SNPs) was only an increase in <em>r</em><sub>MP</sub> of 0.04, even less than the difference for protein and oil. Several studies have reported that decreasing marker density can have minimal impacts on prediction (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lorenz <em>et al.</em> 2011</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Heffner <em>et al.</em> 2011b</a>). It is important that marker density is high enough to have linkage with QTL which may be responsible for variance in the quantitative trait of interest. Considering soybean has considerably high LD relative to other crops, it is not surprising that marker density seemed to have little effect on improving <em>r</em><sub>MP</sub>.</p><p id="p-45">When examining the effects of different marker densities, there was a minimal change in <em>r</em><sub>MP</sub> even at the lowest marker density. It was hypothesized this may have partially been related to the strong population structure present within the GS dataset. The ability to differentiate bi-parental populations from each other <em>vs.</em> prediction within populations may be affecting predictive ability. Oil content was investigated to illustrate this concern. Population structure was first visualized via PCA at all SNPs compared to the lowest marker density, 8<sup>th</sup> tag SNPs (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A and B</a>). There was still identifiable population structure at the lowest marker density, indicating that an ability to differentiate each of the four bi-parental populations (Pop1-4) from each other at the lowest marker density remained. When examining the original oil BLUP values for each genotype, it was evident that Pop1 was higher in oil content compared to Pop2-4 (Fig. S2C). Thus, if there was an ability to genetically differentiate breeding lines from Pop1 compared to Pop2-4, these lines would be predicted to be higher in oil content compared to the other three populations. Pop1-4 influenced a large portion of <em>r</em><sub>MP</sub> because they composed ∼83% lines of the entire GS dataset for oil.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of population structure on prediction of oil content when utilizing the entire genomic selection dataset (EGSD) method. (A) PCA of genomic prediction population using all SNPs. (B) PCA of genomic prediction population using 8th tag SNPs. (C) Average predicted GEBV vs. observed BLUP values when using all SNPs. (D) Average predicted GEBV vs. observed BLUP values when using 8th tag SNPs. (E) Average predicted GEBV vs. observed BLUP within Pop1-4 when using all SNPs. (F) Average predicted GEBV vs. observed BLUP within Pop1-4 when using 8th tag SNPs. Correlation coefficients presented within scatterplots (C-F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1191541004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of population structure on prediction of oil content when utilizing the entire genomic selection dataset (EGSD) method. (A) PCA of genomic prediction population using all SNPs. (B) PCA of genomic prediction population using 8&lt;sup&gt;th&lt;/sup&gt; tag SNPs. (C) Average predicted GEBV &lt;em&gt;vs.&lt;/em&gt; observed BLUP values when using all SNPs. (D) Average predicted GEBV &lt;em&gt;vs.&lt;/em&gt; observed BLUP values when using 8&lt;sup&gt;th&lt;/sup&gt; tag SNPs. (E) Average predicted GEBV &lt;em&gt;vs.&lt;/em&gt; observed BLUP within Pop1-4 when using all SNPs. (F) Average predicted GEBV &lt;em&gt;vs.&lt;/em&gt; observed BLUP within Pop1-4 when using 8&lt;sup&gt;th&lt;/sup&gt; tag SNPs. Correlation coefficients presented within scatterplots (C-F).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/7/2253/F6.medium.gif" width="422" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/7/2253/F6.medium.gif" width="422" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/7/2253/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-46" class="first-child">Effects of population structure on prediction of oil content when utilizing the entire genomic selection dataset (EGSD) method. (A) PCA of genomic prediction population using all SNPs. (B) PCA of genomic prediction population using 8<sup>th</sup> tag SNPs. (C) Average predicted GEBV <em>vs.</em> observed BLUP values when using all SNPs. (D) Average predicted GEBV <em>vs.</em> observed BLUP values when using 8<sup>th</sup> tag SNPs. (E) Average predicted GEBV <em>vs.</em> observed BLUP within Pop1-4 when using all SNPs. (F) Average predicted GEBV <em>vs.</em> observed BLUP within Pop1-4 when using 8<sup>th</sup> tag SNPs. Correlation coefficients presented within scatterplots (C-F).</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">The average predicted oil GEBVs for each RIL were plotted against the observed oil BLUP values (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6C and D</a>). For all SNPs, the correlation coefficient between the average predicted oil GEBVs and observed oil BLUP values of the entire GS dataset was 0.71. For 8<sup>th</sup> tag SNPs, the correlation between the observed and predicted values was 0.63 which was a 11% decrease. Within Pop1-4, decreases in correlation were 41% (0.69 to 0.41), 27% (0.56 to 0.41), 38% (0.64 to 0.40), and 24% (0.54 to 0.41), respectively (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6E and F</a>). Correlation coefficients decreased more within each individual population compared to across all populations, indicating that <em>r</em><sub>MP</sub> may be affected by differences between high and low oil populations as well as high and low oil breeding lines within these populations.</p><p id="p-48">The EGSD approach is a common method to examine the potential for GS within a mixed population of breeding materials. It was observed that population structure can have a strong influence on <em>r</em><sub>MP</sub> and inflate confidence in prediction. Predicting across different populations also has the possibility of inflating <em>r</em><sub>MP</sub> due to the TS possibly containing full-sibs to genotypes placed in the VS unless precautions are taken to avoid this. Since breeders are often applying GS to make predictions in new unique parental combinations, having full-sibs in both the TS and VS is rare. Population structure can be accounted for by including population as an effect in a BLUP model. This would possibly mitigate an ability to identify that the worst line in one population may be better than the best line in another population. Caution should be used when assessing <em>r</em><sub>MP</sub> across different populations as one may be detecting more population differences than differences among the best and worst breeding lines within each population. Not only may this phenomenon be accounting for a lack of significant decreases in <em>r</em><sub>MP</sub> at extremely low marker densities, but it is most likely inflating <em>r</em><sub>MP</sub> at each level of marker density for the same reasons.</p><p id="p-49">Predictive ability for yield was lower compared to protein and oil content by cause of the complexity and low heritability of yield but also partially because Pop1-4, which dominated the GS dataset, had similar mean yield BLUP values, so mean yield differences among these populations was not driving prediction as much as it appeared to be for protein and oil. Predictive ability is most likely inflated for many studies which combine multiple bi-parental populations in GS datasets, specifically when phenotypic means vary across populations. Previous literature has discussed the issues of population structure within GS but this usually refers to substructure within the VS, which is not properly represented within the TS and thus marker effects are not properly estimated (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Guo <em>et al.</em> 2013</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Crossa <em>et al.</em> 2014</a>). Though this is an issue in GS, it is not the population structure related issue referred to here.</p></div><div id="sec-20" class="subsection"><h3>Predictive ability of individual bi-parental populations (WP method <em>vs.</em> AP method)</h3><p id="p-50">The most common breeding pipeline for soybean begins with developing F<sub>1</sub>’s from unique parental combinations. The single seed descent method (SSD) advances lines until the F<sub>4</sub> or F<sub>5</sub> generation (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brim 1966</a>). At this stage, hundreds of single plants are selected based on visual assessment of plants in the field. Selected single plants become plant rows which undergo another round of visual selection for key agronomic traits (<em>i.e.</em>, plant height, lodging, maturity) or plant row yield tests. Many plant rows across populations are often discarded based on breeder notes that can be heavily influenced by environmental factors including but not limited to soil conditions, field slope, mechanical damage, or disease/insect pressure. Single row measurements for traits such as yield, are time-consuming, labor-intensive, and often not reliable estimates (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sebastian <em>et al.</em> 2012</a>). GS has the advantage of leveraging years and locations of replicated field trials to estimate marker effects in order to predict GEBVs for these plant rows that are ideally more reliable than simple visual assessments or single plot phenotyping. The advantage in utilizing GS at this stage <em>vs.</em> a visual assessment should warrant the cost, labor, and time associated with genotyping these plant rows if one is to effectively implement GS at this stage of their breeding program.</p><p id="p-51">Two methods were compared for prediction of each individual bi-parental population (Pop1-4). For comparison purposes, maker density was fixed at all SNPs. Predictive ability was higher for higher heritability traits for both the WP and AP approaches. Utilizing the WP approach was often higher than prediction utilizing the AP approach. For WP, TS size was 50 and when averaging across Pop1-4, this was superior or at least statistically equivalent to a max TS size of 300 for protein and oil and 350 for yield. When comparing the two methodologies at the same TS size, the advantage of WP over AP was even more drastic for all traits. This was most likely resulting from the genetic relatedness between the TS and VS when using full-sibs via the WP approach. For WP, markers were in LD with QTL controlling variation for the traits of interest. Once unrelated materials were brought into the TS in the AP method, the loss in genetic relatedness between TS and VS resulted in a decrease in <em>r</em><sub>MP</sub> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clark <em>et al.</em> 2012</a>). This decrease in relatedness was most likely harming prediction as markers were in LD with QTL specific to populations in the TS and these QTL might not be represented in the VS (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lorenz <em>et al.</em> 2012</a>). The strong subpopulation structure due to having large bi-parental populations in the TS exacerbated the issue as allele effects became increasingly biased toward the allele effects within these larger populations which were not represented in the VS (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Guo <em>et al.</em> 2013</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Crossa <em>et al.</em> 2014</a>).</p><p id="p-52">For higher heritability traits (<em>i.e.</em>, protein and oil), the AP method approached the WP method by taking advantage of larger TS sizes. This was likely due to the added replication of alleles allowing for more accurate estimates of allele effects. The high heritability of these traits implied that a large amount of variation controlled by genetics made marker effect estimates more accurate for prediction. Though this appeared to be the trend on average across populations, certain specific bi-parental populations could not approach predictive ability for the WP method, even at large TS sizes for high heritability traits. The AP approach for predicting oil content for Pop1 showed little success. This may have been a result of unique alleles specific to oil content being present in Pop1 yet largely absent from other breeding lines which composed the TS. This same trend did not occur for the other high heritability trait, protein, so it appeared to be specific to oil and not related to overall genetic relatedness between Pop1 and the other breeding lines.</p><p id="p-53">Yield proved especially difficult to predict as prediction for both methods for each population was comparatively low. As TS size increased during implementation of the AP method, trends in <em>r</em><sub>MP</sub> varied far more for individual bi-parental populations for yield compared to protein and oil. There was some success using the WP approach for Pop2 and 3, but Pop1 and 4 did not predict well. <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lian <em>et al.</em> (2014)</a> predicted within 969 maize bi-parental populations and reported <em>r</em><sub>MP</sub> ranging from -0.34 to 0.89, providing evidence of the variability in predictive ability that can occur for yield even when predicting within populations. The AP methodology was largely unsuccessful for yield and seemed to only surpass WP in situations where <em>r</em><sub>MP</sub> was extremely low such as Pop1 and Pop4. It is possible the high level of structure within the training set may have attributed to this overall lack of success as allele effects were biased toward the bi-parental populations present within the TS. The complexity and low heritability of yield made variation in <em>r</em><sub>MP</sub> of different populations far greater compared to higher heritability traits such as protein and oil. Also, genotype × environment interactions were most likely harming prediction as alleles in one environment may have had opposing effects on yield in another for certain breeding lines.</p><p id="p-54">There has been success predicting for various traits using approaches similar to the AP method but they have had the advantage of leveraging larger numbers of more related plant materials. <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Riedelsheimer <em>et al.</em> (2013)</a> performed cross-validation within a mixed population of 635 maize doubled-haploid (DH) lines for several yield component traits. They reported that across all traits, a TS composed of DH lines which were full-sibs predicted significantly better than a TS composed of DH lines which were half-sibs and prediction was even worse if unrelated breeding lines were placed into the TS. They also reported having half-sibs present for both VS parents was significantly better than having half-sibs for one of the VS parents. <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Jacobson <em>et al.</em> (2014)</a> developed a general combing ability (GCA) model in which maize inbreds were placed into a TS which were half-sibs with the VS and compared to pooling random inbreds in the TS. The GCA model significantly outperformed the random inbred model across 30 test populations for yield, moisture, and test weight. <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Jacobson <em>et al.</em> (2014)</a> was able to leverage 970 testcross populations made available by Monsanto and although the approach was promising, this current study would have needed more extensive genotyping and phenotyping of material to have implemented a similar study in soybean. For the AP method, there were half-sibs present within the TS which may have led to some of the success in prediction, but not close to the numbers observed in these aforementioned studies.</p><p id="p-55">Constructing GS models using full-sibs (WP method) appears to be effective for GS. This most likely delays the breeding cycle compared to leveraging previous phenotypes and genotypes to make predictions (AP method). Prediction of protein content showed the most promise for GS via the AP method as on average, the AP method predicted comparably to the WP method. For oil content, the same could largely be said, but one population decreased average AP predictive ability significantly, indicating that variability in prediction can occur depending on the population being predicted even for high heritably traits. Though successful prediction was achieved for protein and oil, the primary objective for soybean breeders is to make selections based on yield. It is assumed that the level of success achieved for yield within this study may not justify the cost and time needed to impose GS for yield. Though <em>r</em><sub>MP</sub> for yield was low compared to protein and oil, populations on average showed an upward trend and still made gains in r<sub>MP</sub> at the highest TS size. Simply increasing TS size may not be the best solution as the literature has shown the benefits of increasing genetic relatedness between TS and VS. Studies in maize have reported success predicting across populations when leveraging half-sibs that represent both parents in the cross in addition to increasing TS size. Targeting this approach may be the best strategy for improving GS for yield in soybean in the future. A study investigating this has not yet been shown in soybean as many previous studies have evaluated prediction across mixed populations.</p><p id="p-56">The success in prediction of protein and oil content alone may not warrant the application of GS as NIR spectrometry provides good estimates of these phenotypes with minimal time, labor, and cost. If predictive ability for yield could be increased enough to warrant genotyping of single-plant rows, acquiring predictions of protein and oil would be a logical additional step with minimal additional efforts.</p></div><div id="sec-21" class="subsection"><h3>CONCLUSION</h3><p id="p-57">This study illustrated use of genomic selection for prediction of yield within a soybean breeding program. This was the first report indicating the success that can be achieved for higher heritability traits such as protein and oil content. Predictive ability can be inflated when there is population structure present in combination with differences in trait means across populations. Increased success across all traits can be attributed to increasing training set size more so than increased marker density, though benefits associated with training set size had eventual diminishing returns. Predictive ability can also be increased by building training sets with increased relatedness to validation sets. Yield was difficult to predict and this is most likely related to complex genotype × environment interactions, its highly quantitative nature, and a biasing of allele effects toward populations which dominated the training set. For future success, a larger training set size in combination with increased genetic relatedness between training and validation set could improve predictive ability in soybean as it has in maize.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-58">Technical support was provided by Dale Wood, Earl Baxter, Brice Wilson, and Gina Bishop of the University of Georgia. Genotyping was performed at the USDA-ARS Soybean Genomics and Improvement Lab by Chuck Quigley. Research funding was provided by the United Soybean Board and Georgia Agricultural Community Commission for Soybeans. Special thanks to the Glenn and Helen Burton Scholarship Fund provided by the College of Agricultural and Environmental Sciences at the University of Georgia.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8121251">https://doi.org/10.25387/g3.8121251</a>.</p></li><li class="fn" id="fn-2"><p id="p-60"><em>Communicating editor: J. Birchler</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-22"><span class="received-label">Received </span>December 22, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-10"><span class="accepted-label">Accepted </span>May 10, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Stewart-Brown <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.1" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-based prediction of testcross values in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1587-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2253.2"><div class="cit-metadata unstructured">American Soybean Association, 2018 2016 Soy Highlights. The American Soybean Association. <a href="http://soystats.com/2013-highlights/">http://soystats.com/ 2013-highlights/</a> (accessed 20 Jan. 2018).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.3" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2006.11.0690</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.4" data-doi="10.1093/bioinformatics/bth457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Haploview: analysis and visualization of LD and haplotype maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">265</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bth457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth457%26rft_id%253Dinfo%253Apmid%252F15297300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15297300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226308500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.5" data-doi="10.18637/jss.v067.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maechler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting linear mixed-effects models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v067.i01</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v067.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v067.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.6" data-doi="10.2135/cropsci2000.0008rcv"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boerma</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussey</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowan</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Registration of ‘Boggs’ soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2000.0008rcv</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D40%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2000.0008rcv%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2000.0008rcv&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.7" data-doi="10.2135/cropsci1966.0011183X000600020041x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brim</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">A modified pedigree method of selection in soybeans.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1966.0011183X000600020041x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBrim%26rft.auinit1%253DC.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D220%26rft.epage%253D220%26rft.atitle%253DA%2BModified%2BPedigree%2BMethod%2Bof%2BSelection%2Bin%2BSoybeans%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1966.0011183X000600020041x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1966.0011183X000600020041x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19667816000041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.8" data-doi="10.2135/cropsci1979.0011183X001900040016x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brim</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Recurrent selection in soybeans. II. Selection for increased percent protein in seeds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1979.0011183X001900040016x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBrim%26rft.auinit1%253DC.%2BA.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D494%26rft.epage%253D498%26rft.atitle%253DRecurrent%2BSelection%2Bin%2BSoybeans.%2BII.%2BSelection%2Bfor%2BIncreased%2BPercent%2BProtein%2Bin%2BSeeds%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1979.0011183X001900040016x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1979.0011183X001900040016x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979HJ51900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.9" data-doi="10.2135/cropsci1997.0011183X003700020011x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graef</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mapping QTL for seed protein and oil content in eight soybean populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">370</span>–<span class="cit-lpage">378</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1997.0011183X003700020011x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBrummer%26rft.auinit1%253DE.%2BC.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D370%26rft.epage%253D378%26rft.atitle%253DMapping%2BQTL%2Bfor%2BSeed%2BProtein%2Band%2BOil%2BContent%2Bin%2BEight%2BSoybean%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1997.0011183X003700020011x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1997.0011183X003700020011x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WU90500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fountain</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Registration of ‘NC-Raleigh’ soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D13%26rft.spage%253D130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.11" data-doi="10.2135/cropsci2003.1126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Registration of ‘N7001’ soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1126</span>–<span class="cit-lpage">1127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2003.1126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCarter%26rft.auinit1%253DT.E.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D1126%26rft.epage%253D1127%26rft.atitle%253DRegistration%2Bof%2B%2527N7001%2527%2BSoybean%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000182540300055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.12" data-doi="10.2135/cropsci2003.1053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babka</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graef</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staswick</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The seed protein, oil, and yield QTL on soybean linkage group I.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1067</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2003.1053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DChung%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D1053%26rft.epage%253D1067%26rft.atitle%253DThe%2BSeed%2BProtein%252C%2BOil%252C%2Band%2BYield%2BQTL%2Bon%2BSoybean%2BLinkage%2BGroup%2BI%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000182540300040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.13" data-doi="10.1186/1297-9686-44-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The importance of information on relatives for the prediction of genomic breeding values and the implication for the makeup of reference data sets in livestock breeding schemes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DClark%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DThe%2Bimportance%2Bof%2Binformation%2Bon%2Brelatives%2Bfor%2Bthe%2Bprediction%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Band%2Bthe%2Bimplications%2Bfor%2Bthe%2Bmakeup%2Bof%2Breference%2Bdata%2Bsets%2Bin%2Blivestock%2Bbreeding%2Bschemes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-4%26rft_id%253Dinfo%253Apmid%252F22321529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22321529&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of genome-wide selection for different traits with constant population size, heritability, and numbers of markers.</span> <abbr class="cit-jnl-abbrev">Plant Gene</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGene%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.15" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.16" data-doi="10.1038/ng1669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Bakker</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yelensky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Efficiency and power in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1669</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253Dde%2BBakker%26rft.auinit1%253DP.%2BI.%26rft.volume%253D37%26rft.issue%253D11%26rft.spage%253D1217%26rft.epage%253D1223%26rft.atitle%253DEfficiency%2Band%2Bpower%2Bin%2Bgenetic%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1669%26rft_id%253Dinfo%253Apmid%252F16244653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16244653&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233045200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.17" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic heritability: what is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.18" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2253.19"><div class="cit-metadata unstructured">de Mendiburu, F., 2017 Agricolae: statistical procedures for agricultural research. R package version 1.2–8. <a href="https://CRAN.R-project.org/package=agricolae">https://CRAN.R-project.org/package=agricolae</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.20" data-doi="10.1111/j.1439-0388.2007.00701.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prediction of response to marker-assisted and genomic selection using selection index theory.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00701.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DDekkers%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D331%26rft.epage%253D341%26rft.atitle%253DPrediction%2Bof%2Bresponse%2Bto%2Bmarker-assisted%2Band%2Bgenomic%2Bselection%2Busing%2Bselection%2Bindex%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00701.x%26rft_id%253Dinfo%253Apmid%252F18076470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00701.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.21" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shrinkage estimation of the realized relationship matrix. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genom. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenom.%2BGenet.%26rft.volume%253D2%26rft.spage%253D1405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.23" data-doi="10.1007/s00122-011-1702-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishore</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genome-wide selection efficiency in maize nested association mapping populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1702-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D261%26rft.epage%253D275%26rft.atitle%253DEvaluation%2Bof%2Bgenome-wide%2Bselection%2Befficiency%2Bin%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1702-9%26rft_id%253Dinfo%253Apmid%252F21938474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1702-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21938474&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of across-environment genome-wide prediction in maize nested association mapping populations. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genom. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenom.%2BGenet.%26rft.volume%253D3%26rft.spage%253D263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.25" data-doi="10.2135/cropsci2011.05.0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection accuracy for grain quality traits in biparental wheat populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.05.0253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.05.0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.05.0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000295839200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.26" data-doi="10.3835/plantgenome2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2010.12.0029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.27" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.11.0662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.28" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2231-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2253.29" data-doi="10.1016/B978-0-935315-63-9.50027-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huth</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Nutritional aspects of soybean oil and soy protein</span>, pp. <span class="cit-fpage">460</span>–<span class="cit-lpage">478</span> in <span class="cit-source">Practical Handbook of Soybean Processing and Utilization</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">D. R.</span></span> <span class="cit-publ-name">AOCS Press</span>. <span class="cit-publ-loc">Champaign, IL</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-935315-63-9.50027-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPractical%2BHandbook%2Bof%2BSoybean%2BProcessing%2Band%2BUtilization%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-935315-63-9.50027-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-935315-63-9.50027-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.30" data-doi="10.1186/1471-2164-15-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide association study of seed protein and oil content in soybean.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.31" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2418-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.32" data-doi="10.2135/cropsci2013.11.0774"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">General combining ability model for genomewide selection in a biparental cross.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">905</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.11.0774</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.11.0774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.11.0774&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2253.33"><div class="cit-metadata"><cite>Jain, A., M. Roorkiwal, M. K. Pandey, and R. K. Varshney, 2017 Current status and prospects of genomic selection in legumes, pp. 131–148 in <em>Genomic Selection for Crop Improvement</em>, edited by R. K. Varshney, M. Roorkiwal, and M. E. Sorrells. Springer International Publishing, Cham, Switzerland.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-63170-7_6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.34" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.35" data-doi="10.1186/1471-2164-15-740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posadas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jedlicka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping by sequencing for genomic prediction in a soybean breeding population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D740%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-740%26rft_id%253Dinfo%253Apmid%252F25174348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-740&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25174348&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.36" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.080101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjI1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-web" id="cit-9.7.2253.37"><div class="cit-metadata unstructured">Kassambara, A., 2017 ggpubr: ‘ggplot2’ based publication ready plots. R package version 0.1.6.99. <a href="http://www.sthda.com/english/rpkgs/ggpubr">http://www.sthda.com/english/rpkgs/ggpubr</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keim</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">A rapid protocol for isolating soybean DNA.</span> <abbr class="cit-jnl-abbrev">Soybean Genet. Newsl.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoybean%2BGenet.%2BNewsl.%26rft.volume%253D15%26rft.spage%253D150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.39" data-doi="10.2135/cropsci2013.12.0856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide prediction accuracy within 969 maize biparental populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1514</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.12.0856</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1514%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.12.0856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.12.0856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.40" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22796960&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2253.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects</span>, pp. <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span> in <span class="cit-source">Advances in Agronomy</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">D. L.</span></span> <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">Waltham, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.42" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.09.0503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.43" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1166-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.44" data-doi="10.1007/s11032-016-0504-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Potential of marker selection to increase prediction accuracy of genomic selection in soybean (<em>Glycine max</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-016-0504-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-016-0504-9%26rft_id%253Dinfo%253Apmid%252FWOS%253A00038214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-016-0504-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=WOS:00038214&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.45" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzcvMjI1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.46" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00700.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.47" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.06.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.48" data-doi="10.1534/genetics.113.150227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic predictability of interconnected biparental maize populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.150227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150227%26rft_id%253Dinfo%253Apmid%252F23535384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS83LzIyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.49" data-doi="10.3835/plantgenome2014.09.0046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient use of historical data for genomic selection: a case study of stem rust resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.09.0046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.09.0046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.09.0046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.50" data-doi="10.1038/nature08670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the palaeopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>. Erratum: 120.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-9.7.2253.51" data-doi="10.1016/B978-0-9830791-0-1.50020-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sebastian</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhlman</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accelerated Yield Technology: a platform for marker assisted selection of simple and complex traits</span>, pp. <span class="cit-fpage">297</span>–<span class="cit-lpage">305</span> in <span class="cit-source">Designing soybeans for 21<sup>st</sup> century markets</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. F.</span></span> <span class="cit-publ-name">Elsevier Inc.</span>, <span class="cit-publ-loc">Urbana, IL</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-9830791-0-1.50020-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDesigning%2Bsoybeans%2Bfor%2B21st%2Bcentury%2Bmarkets%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-9830791-0-1.50020-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-9830791-0-1.50020-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.52" data-doi="10.1371/journal.pone.0054985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e54985</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0054985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54985%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054985%26rft_id%253Dinfo%253Apmid%252F23372807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372807&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.53" data-doi="10.1186/s12864-015-2344-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pantalone</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of high resolution genetic linkage maps to improve the soybean genome sequence assembly Glyma1.01.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2344-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2344-0%26rft_id%253Dinfo%253Apmid%252F26739042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2344-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26739042&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-9.7.2253.54"><div class="cit-metadata unstructured">USDA National Agricultural Statistics Service (USDA-NASS), 2017 Quick stats. USDA-NASS, Washington, DC. <a href="https://usda.mannlib.cornell.edu/usda/current/Acre/Acre-06%E2%80%9330%E2%80%932017.pdf">https://usda.mannlib.cornell.edu/usda/current/Acre/Acre-06–30–2017.pdf</a> (accessed 20 Jan. 2018).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.55" data-doi="10.1017/S0016672399004462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denham</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection using ridge-regression.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672399004462</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWhittaker%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D75%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D252%26rft.atitle%253DMarker-assisted%2Bselection%2Busing%2Bridge%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004462%26rft_id%253Dinfo%253Apmid%252F10816982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10816982&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F7%2F2253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086569100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rainey</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessing predictive properties of genome-wide selection in soybeans. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genom. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenom.%2BGenet.%26rft.volume%253D6%26rft.spage%253D2611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.7.2253.57" data-doi="10.3389/fpls.2017.01916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Effect of trait heritability, training population size and marker density on genomic prediction accuracy estimation in 22 bi-parental tropical maize populations.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1916</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01916</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1916%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/7/2253.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/7/2245" title="Asymmetric Redundancy of ZERZAUST and ZERZAUST HOMOLOG in Different Accessions of Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/7/2267" title="Expression Signatures of Cisplatin- and Trametinib-Treated Early-Stage Medaka Melanomas" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/7"><span>Volume 9 Issue 7, July 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/7.cover-source.jpg?itok=ecxKsDdx" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/7/2253" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BQE1b_2iSVHTGnQ-20naxfWDmX2zsedX6UWgj97bQqw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/7/2253" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/7/2253" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/7/2253" class="active" data-icon-position="" data-hide-link-title="0">Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493678" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-73xDZRQQBD1Vn_uW7pg-UbY-s-NHUVeDu0F-8XtxXKU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2253.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/7/2253" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-CevJJfORoNwwJ6qbdHu_fo9TmPe7GWUu6SPnj83Jfbs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/7/2253" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/7/2253.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493678" id="node493678--2" data-pisa="ggg;9/7/2253" data-pisa-master="ggg;g3.118.200917" data-apath="/ggg/9/7/2253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7551-9833" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin B. Stewart-Brown</span>, <span class="highwire-citation-author" data-delta="1">Qijian Song</span>, <span class="highwire-citation-author" data-delta="2">Justin N. Vaughn</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4114-9509" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zenglu Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2253-2265;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200917 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493678--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin B. Stewart-Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BB.%2BStewart-Brown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stewart-Brown%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BB.%2BStewart-Brown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7551-9833" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin B. Stewart-Brown</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Qijian Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Lab, USDA-ARS, Beltsville, MD 20705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qijian%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQijian%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin N. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Genomics and Bioinformatics Research Unit, USDA-ARS, Center for Applied Genetic Technologies, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BN.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BN.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zenglu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zenglu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZenglu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4114-9509" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zenglu Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zli@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zli@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/7/2253" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/7/2253&amp;title=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/7/2253&amp;title=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/7/2253&amp;title=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/7/2253&amp;text=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/7/2253&amp;title=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/7/2253&amp;t=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/7/2253&amp;title=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/7/2253&amp;title=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493678/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493678" id="node493678--3" data-pisa="ggg;9/7/2253" data-pisa-master="ggg;g3.118.200917" data-apath="/ggg/9/7/2253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection for Yield and Seed Composition Traits Within an Applied Soybean Breeding Program</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7551-9833" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin B. Stewart-Brown</span>, <span class="highwire-citation-author" data-delta="1">Qijian Song</span>, <span class="highwire-citation-author" data-delta="2">Justin N. Vaughn</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4114-9509" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zenglu Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2253-2265;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200917 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493678--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin B. Stewart-Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BB.%2BStewart-Brown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stewart-Brown%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BB.%2BStewart-Brown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7551-9833" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin B. Stewart-Brown</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Qijian Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Soybean Genomics and Improvement Lab, USDA-ARS, Beltsville, MD 20705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qijian%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQijian%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin N. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Genomics and Bioinformatics Research Unit, USDA-ARS, Center for Applied Genetic Technologies, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BN.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BN.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zenglu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Genetics and Genomics and Dep. of Crop and Soil Sci., University of Georgia, Athens, GA 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zenglu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZenglu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4114-9509" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zenglu Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zli@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zli@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2253&amp;rft.epage=2265&amp;rft.atitle=Genomic%20Selection%20for%20Yield%20and%20Seed%20Composition%20Traits%20Within%20an%20Applied%20Soybean%20Breeding%20Program&amp;rft.volume=9&amp;rft.issue=7&amp;rft.date=2019-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Stewart-Brown%2C+Benjamin+B.&amp;rft.au=Song%2C+Qijian&amp;rft.au=Vaughn%2C+Justin+N.&amp;rft.au=Li%2C+Zenglu"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493678/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493678/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493678/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493678/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493678/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493678/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493678/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493678/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493678/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493678/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493678/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493678/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/7/2253" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/7/2253.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/7/2253.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/7/2253.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
