<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/4/487" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/4/487.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/4/487.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/4/487.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/4/487.atom" />
<meta name="HW.pisa" content="ggg;2/4/487" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in Arabidopsis" />
<meta name="DC.Identifier" content="10.1534/g3.111.001362" />
<meta name="DC.Date" content="2012-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Kurihara et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Eukaryotes possess several RNA surveillance mechanisms that prevent undesirable aberrant RNAs from accumulating. Arabidopsis XRN2, XRN3, and XRN4 are three orthologs of the yeast 5′-to-3′ exoribonuclease, Rat1/Xrn2, that function in multiple RNA decay pathways. XRN activity is maintained by FIERY1 (FRY1), which converts the XRN inhibitor, adenosine 3′, 5′-bisphosphate (PAP), into 5′AMP. To identify the roles of XRNs and FRY1 in suppression of non-coding RNAs, strand-specific genome-wide tiling arrays and deep strand-specific RNA-Seq analyses were carried out in fry1 and xrn single and double mutants. In fry1-6 , about 2000 new transcripts were identified that extended the 3′ end of specific mRNAs; many of these were also observed in genotypes that possess the xrn3-3 mutation, a partial loss-of-function allele. Mutations in XRN2 and XRN4 in combination with xrn3-3 revealed only a minor effect on 3′ extensions, indicating that these genes may be partially redundant with XRN3 . We also observed the accumulation of 3′ remnants of many DCL1-processed microRNA (miRNA) precursors in fry1-6 and xrn3-3 . These findings suggest that XRN3, in combination with FRY1, is required to prevent the accumulation of 3′ extensions that arise from thousands of mRNA and miRNA precursor transcripts." />
<meta name="DC.Contributor" content="Yukio Kurihara" />
<meta name="DC.Contributor" content="Robert J. Schmitz" />
<meta name="DC.Contributor" content="Joseph R. Nery" />
<meta name="DC.Contributor" content="Matthew D. Schultz" />
<meta name="DC.Contributor" content="Emiko Okubo-Kurihara" />
<meta name="DC.Contributor" content="Taeko Morosawa" />
<meta name="DC.Contributor" content="Maho Tanaka" />
<meta name="DC.Contributor" content="Tetsuro Toyoda" />
<meta name="DC.Contributor" content="Motoaki Seki" />
<meta name="DC.Contributor" content="Joseph R. Ecker" />
<meta name="article:published_time" content="2012-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in Arabidopsis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Eukaryotes possess several RNA surveillance mechanisms that prevent undesirable aberrant RNAs from accumulating. &lt;i&gt;Arabidopsis&lt;/i&gt; XRN2, XRN3, and XRN4 are three orthologs of the yeast 5′-to-3′ exoribonuclease, Rat1/Xrn2, that function in multiple RNA decay pathways. XRN activity is maintained by FIERY1 (FRY1), which converts the XRN inhibitor, adenosine 3′, 5′-bisphosphate (PAP), into 5′AMP. To identify the roles of XRNs and FRY1 in suppression of non-coding RNAs, strand-specific genome-wide tiling arrays and deep strand-specific RNA-Seq analyses were carried out in &lt;i&gt;fry1&lt;/i&gt; and &lt;i&gt;xrn&lt;/i&gt; single and double mutants. In &lt;i&gt;fry1-6&lt;/i&gt;, about 2000 new transcripts were identified that extended the 3′ end of specific mRNAs; many of these were also observed in genotypes that possess the &lt;i&gt;xrn3-3&lt;/i&gt; mutation, a partial loss-of-function allele. Mutations in &lt;i&gt;XRN2&lt;/i&gt; and &lt;i&gt;XRN4&lt;/i&gt; in combination with &lt;i&gt;xrn3-3&lt;/i&gt; revealed only a minor effect on 3′ extensions, indicating that these genes may be partially redundant with &lt;i&gt;XRN3&lt;/i&gt;. We also observed the accumulation of 3′ remnants of many DCL1-processed microRNA (miRNA) precursors in &lt;i&gt;fry1-6&lt;/i&gt; and &lt;i&gt;xrn3-3&lt;/i&gt;. These findings suggest that XRN3, in combination with FRY1, is required to prevent the accumulation of 3′ extensions that arise from thousands of mRNA and miRNA precursor transcripts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/04/01" />
<meta name="citation_mjid" content="ggg;2/4/487" />
<meta name="citation_id" content="2/4/487" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/4/487" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/4/487.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/4/487.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/4/487.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001362" />
<meta name="citation_pmid" content="22540040" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="4" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="487" />
<meta name="citation_lastpage" content="498" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yukio Kurihara" />
<meta name="citation_author_institution" content="Plant Biology Laboratory" />
<meta name="citation_author" content="Robert J. Schmitz" />
<meta name="citation_author_institution" content="Plant Biology Laboratory" />
<meta name="citation_author_institution" content="Genomic Analysis Laboratory, and" />
<meta name="citation_author" content="Joseph R. Nery" />
<meta name="citation_author_institution" content="Genomic Analysis Laboratory, and" />
<meta name="citation_author" content="Matthew D. Schultz" />
<meta name="citation_author_institution" content="Genomic Analysis Laboratory, and" />
<meta name="citation_author" content="Emiko Okubo-Kurihara" />
<meta name="citation_author_institution" content="Plant Biology Laboratory" />
<meta name="citation_author" content="Taeko Morosawa" />
<meta name="citation_author_institution" content="Genomic Analysis Laboratory, and" />
<meta name="citation_author" content="Maho Tanaka" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and" />
<meta name="citation_author" content="Tetsuro Toyoda" />
<meta name="citation_author_institution" content="RIKEN Plant Science Center" />
<meta name="citation_author" content="Motoaki Seki" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and" />
<meta name="citation_author" content="Joseph R. Ecker" />
<meta name="citation_author_institution" content="Plant Biology Laboratory" />
<meta name="citation_author_institution" content="Genomic Analysis Laboratory, and" />
<meta name="citation_author_institution" content="Bioinformatics and Systems Engineering Division, RIKEN Yokohama Institute, Yokohama, Kanagawa 230-0045, Japan" />
<meta name="citation_author_email" content="ecker@salk.edu" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Alonso;citation_author=A. N. Stepanova;citation_author=T. J. Leisse;citation_author=C. J. Kim;citation_author=H. Chen;citation_title=Genome-Wide Insertional Mutagenesis of Arabidopsis thaliana;citation_pages=653-657;citation_volume=301;citation_year=2003;citation_issue=5633;citation_pmid=12893945;citation_doi=10.1126/science.1086391" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=I. Behm-Ansmant;citation_author=I. Kashima;citation_author=J. Rehwinkel;citation_author=J. Sauliere;citation_author=N. Wittkopp;citation_title=mRNA quality control: an ancient machinery recognizes and degrades mRNAs with nonsense codons.;citation_pages=2845-2853;citation_volume=581;citation_year=2007;citation_issue=15;citation_pmid=17531985;citation_doi=10.1016/j.febslet.2007.05.027" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=E. J. Chapman;citation_author=J. C. Carrington;citation_title=Specialization and evolution of endogenous small RNA pathways.;citation_pages=884-896;citation_volume=8;citation_year=2007;citation_issue=11;citation_pmid=17943195;citation_doi=10.1038/nrg2179" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Chekanova;citation_author=B. D. Gregory;citation_author=S. V. Reverdatto;citation_author=H. Chen;citation_author=R. Kumar;citation_title=Genome-wide high-resolution mapping of exosome substrates reveals hidden features in the Arabidopsis transcriptome.;citation_pages=1340-1353;citation_volume=131;citation_year=2007;citation_issue=7;citation_pmid=18160042;citation_doi=10.1016/j.cell.2007.10.056" />
<meta name="citation_reference" content="citation_journal_title=Plant, Cell, and Environment (Print);citation_journal_abbrev=Plant, Cell, and Environment (Print);citation_author=H. Chen;citation_author=L. Xiong;citation_title=The bifunctional abiotic stress signalling regulator and endogenous RNA silencing suppressor FIERY1 is required for lateral root formation.;citation_pages=2180-2190;citation_volume=33;citation_year=2010;citation_issue=12;citation_pmid=20807376;citation_doi=10.1111/j.1365-3040.2010.02218.x" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Biol.;citation_author=Y. Chiba;citation_author=P. J. Green;citation_title=mRNA degradation machinery in plants;citation_pages=114-124;citation_volume=52;citation_year=2009;citation_doi=10.1007/s12374-009-9021-2" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=B. Dichtl;citation_author=A. Stevens;citation_author=D. Tollervey;citation_title=Lithium toxicity in yeast is due to the inhibition of RNA processing enzymes.;citation_pages=7184-7195;citation_volume=16;citation_year=1997;citation_issue=23;citation_pmid=9384595;citation_doi=10.1093/emboj/16.23.7184" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=N. L. Garneau;citation_author=J. Wilusz;citation_author=C. J. Wilusz;citation_title=The highways and byways of mRNA decay.;citation_pages=113-126;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17245413;citation_doi=10.1038/nrm2104" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Gazzani;citation_author=T. Lawrenson;citation_author=D. Woodward;citation_author=R. Sablowski;citation_title=A Link Between mRNA Turnover and RNA Interference in Arabidopsis;citation_pages=1046-1048;citation_volume=306;citation_year=2004;citation_issue=5698;citation_pmid=15528448;citation_doi=10.1126/science.1101092" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=R. Gil-Mascarell;citation_author=J. M. Lopez-Coronado;citation_author=J. M. Belles;citation_author=R. Serrano;citation_author=P. L. Rodriguez;citation_title=The Arabidopsis HAL2-like gene family includes a novel sodium-sensitive phosphatase.;citation_pages=373-383;citation_volume=17;citation_year=1999;citation_issue=4;citation_pmid=10205895;citation_doi=10.1046/j.1365-313X.1999.00385.x" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=B. D. Gregory;citation_author=R. C. O’Malley;citation_author=R. Lister;citation_author=M. A. Urich;citation_author=J. Tonti-Filippini;citation_title=A link between RNA metabolism and silencing affecting Arabidopsis development.;citation_pages=854-866;citation_volume=14;citation_year=2008;citation_issue=6;citation_pmid=18486559;citation_doi=10.1016/j.devcel.2008.04.005" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=I. Gy;citation_author=V. Gasciolli;citation_author=D. Lauressergues;citation_author=J.-B. Morel;citation_author=J. Gombert;citation_title=Arabidopsis FIERY1, XRN2, and XRN3 are endogenous RNA silencing suppressors;citation_volume=19;citation_year=2007;citation_pmid=17993620;citation_doi=10.1105/tpc.107.055319" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Hirsch;citation_author=J. Misson;citation_author=P. A. Crisp;citation_author=P. David;citation_author=V. Bayle;citation_title=A novel fry1 allele reveals the existence of a mutant phenotype unrelated to 5'-&amp;gt;3' exoribonuclease (XRN) activities in Arabidopsis thaliana roots.;citation_pages=e16724-e16724;citation_volume=6;citation_year=2011;citation_issue=2;citation_pmid=21304819;citation_doi=10.1371/journal.pone.0016724" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=C. Hornyik;citation_author=L. C. Terzi;citation_author=G. G. Simpson;citation_title=The spen family protein FPA controls alternative cleavage and polyadenylation of RNA.;citation_pages=203-213;citation_volume=18;citation_year=2010;citation_issue=2;citation_pmid=20079695;citation_doi=10.1016/j.devcel.2009.12.009" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Houseley;citation_author=D. Tollervey;citation_title=The many pathways of RNA degradation.;citation_pages=763-776;citation_volume=136;citation_year=2009;citation_issue=4;citation_pmid=19239894;citation_doi=10.1016/j.cell.2009.01.019" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. P. Kastenmayer;citation_author=P. J. Green;citation_title=Novel features of the XRN-family in Arabidopsis: Evidence that AtXRN4, one of several orthologs of nuclear Xrn2p/Rat1p, functions in the cytoplasm;citation_pages=13985-13990;citation_volume=97;citation_year=2000;citation_issue=25;citation_pmid=11106401;citation_doi=10.1073/pnas.97.25.13985" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=B.-H. Kim;citation_author=A. G. von Arnim;citation_title=FIERY1 regulates light-mediated repression of cell elongation and flowering time via its 3'(2'),5'-bisphosphate nucleotidase activity.;citation_pages=208-219;citation_volume=58;citation_year=2009;citation_issue=2;citation_pmid=19077168;citation_doi=10.1111/j.1365-313X.2008.03770.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kim;citation_author=N. J. Krogan;citation_author=L. Vasiljeva;citation_author=O. J. Rando;citation_author=E. Nedea;citation_title=The yeast Rat1 exonuclease promotes transcription termination by RNA polymerase II.;citation_pages=517-522;citation_volume=432;citation_year=2004;citation_issue=7016;citation_pmid=15565157;citation_doi=10.1038/nature03041" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=M. Koornneef;citation_author=C. J. Hanhart;citation_author=J. H. van der Veen;citation_title=A genetic and physiological analysis of late flowering mutants in Arabidopsis thaliana.;citation_pages=57-66;citation_volume=229;citation_year=1991;citation_issue=1;citation_pmid=1896021;citation_doi=10.1007/BF00264213" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Kurihara;citation_author=Y. Watanabe;citation_title=Arabidopsis micro-RNA biogenesis through Dicer-like1 protein functions;citation_volume=101;citation_year=2004;citation_pmid=15314213;citation_doi=10.1073/pnas.0403115101" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Kurihara;citation_author=A. Matsui;citation_author=K. Hanada;citation_author=M. Kawashima;citation_author=J. Ishida;citation_title=Genome-wide suppression of aberrant mRNA-like noncoding RNAs by NMD in Arabidopsis;citation_volume=106;citation_year=2009;citation_pmid=19181858;citation_doi=10.1073/pnas.0808902106" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Lister;citation_author=R. C. O’Malley;citation_author=J. Tonti-Filippini;citation_author=B. D. Gregory;citation_author=C. C. Berry;citation_title=Highly integrated single-base resolution maps of the epigenome in Arabidopsis.;citation_pages=523-536;citation_volume=133;citation_year=2008;citation_issue=3;citation_pmid=18423832;citation_doi=10.1016/j.cell.2008.03.029" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Lister;citation_author=M. Pelizzola;citation_author=Y. S. Kida;citation_author=R. D. Hawkins;citation_author=J. R. Nery;citation_title=Hotspots of aberrant epigenomic reprogramming in human induced pluripotent stem cells.;citation_pages=68-73;citation_volume=471;citation_year=2011;citation_issue=7336;citation_pmid=21289626;citation_doi=10.1038/nature09798" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=F. Liu;citation_author=V. Quesada;citation_author=P. Crevillen;citation_author=I. Baurle;citation_author=S. Swiezawski;citation_title=The Arabidopsis RNA-binding protein FCA requires a lysine-specific demethylase 1 homolog to downregulate FLC.;citation_pages=398-407;citation_volume=28;citation_year=2007;citation_issue=3;citation_pmid=17996704;citation_doi=10.1016/j.molcel.2007.10.018" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=F. Liu;citation_author=S. Marquardt;citation_author=C. Lister;citation_author=S. Swiezewski;citation_author=C. Dean;citation_title=Targeted 3′ processing of antisense transcripts triggers Arabidopsis FLC chromatin silencing;citation_volume=327;citation_year=2010;citation_pmid=19965720;citation_doi=10.1126/science.1180278" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. C. Loke;citation_author=E. A. Stahlberg;citation_author=D. G. Strenski;citation_author=B. J. Haas;citation_author=P. C. Wood;citation_title=Compilation of mRNA polyadenylation signals in Arabidopsis revealed a new signal element and potential secondary structures;citation_volume=138;citation_year=2005;citation_pmid=15965016;citation_doi=10.1104/pp.105.060541" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=F. D. Lucia;citation_author=C. Dean;citation_title=Long non-coding RNAs and chromatin regulation.;citation_pages=168-173;citation_volume=14;citation_year=2011;citation_issue=2;citation_pmid=21168359;citation_doi=10.1016/j.pbi.2010.11.006" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=A. Matsui;citation_author=J. Ishida;citation_author=T. Morosawa;citation_author=Y. Mochizuki;citation_author=E. Kaminuma;citation_title=Arabidopsis transcriptome analysis under drought, cold, high-salinity and ABA treatment conditions using a tiling array;citation_volume=49;citation_year=2008;citation_pmid=18625610;citation_doi=10.1093/pcp/pcn101" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=L. E. Maquat;citation_title=Nonsense-mediated mRNA decay: splicing, translation and mRNP dynamics.;citation_pages=89-99;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=15040442;citation_doi=10.1038/nrm1310" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Olmedo;citation_author=H. Guo;citation_author=B. D. Gregory;citation_author=S. D. Nourizadeh;citation_author=L. Aguilar-Henonin;citation_title=ETHYLENE-INSENSITIVE5 encodes a 5′→3′ exoribonuclease required for regulation of the EIN3-targeting F-box proteins EBF1/2;citation_volume=103;citation_year=2006;citation_pmid=16920797;citation_doi=10.1073/pnas.0605528103" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. Roman;citation_author=B. Lubarsky;citation_author=J. J. Kieber;citation_author=M. Rothenberg;citation_author=J. R. Ecker;citation_title=Genetic analysis of ethylene signal transduction in Arabidopsis thaliana: five novel mutant loci integrated into a stress response pathway.;citation_pages=1393-1409;citation_volume=139;citation_year=1995;citation_issue=3;citation_pmid=7768447" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=L. A. Rymarquis;citation_author=F. F. Souret;citation_author=P. J. Green;citation_title=Evidence that XRN4, an Arabidopsis homolog of exoribonuclease XRN1, preferentially impacts transcripts with certain sequences or in particular functional categories;citation_volume=17;citation_year=2011;citation_pmid=21224377;citation_doi=10.1261/rna.2467911" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=R. J. Schmitz;citation_author=X. Zhang;citation_title=High-throughput approaches for plant epigenomic studies.;citation_pages=130-136;citation_volume=14;citation_year=2011;citation_issue=2;citation_pmid=21470901;citation_doi=10.1016/j.pbi.2011.03.010" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=L. Song;citation_author=M. J. Axtell;citation_author=N. V. Fedoroff;citation_title=RNA secondary structural determinants of miRNA precursor processing in Arabidopsis.;citation_pages=37-41;citation_volume=20;citation_year=2010;citation_issue=1;citation_pmid=20015653;citation_doi=10.1016/j.cub.2009.10.076" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Sonmez;citation_author=I. Baurle;citation_author=A. Magusin;citation_author=R. Dreos;citation_author=S. Laubinger;citation_title=RNA 3′ processing functions of Arabidopsis FCA and FPA limit intergenic transcription;citation_volume=108;citation_year=2011;citation_pmid=21536901;citation_doi=10.1073/pnas.1105334108" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=F. Souret;citation_author=J. P. Kastenmayer;citation_author=P. J. Green;citation_title=AtXRN4 degrades mRNA in Arabidopsis and its substrates include selected miRNA targets.;citation_pages=173-183;citation_volume=15;citation_year=2004;citation_issue=2;citation_pmid=15260969;citation_doi=10.1016/j.molcel.2004.06.006" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=T. Toyoda;citation_author=K. Shinozaki;citation_title=Tiling array-driven elucidation of transcriptional structures based on maximum-likelihood and Markov models.;citation_pages=611-621;citation_volume=43;citation_year=2005;citation_issue=4;citation_pmid=16098113;citation_doi=10.1111/j.1365-313X.2005.02470.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=Tophat: discovering splice junctions with RNA-Seq;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. L. van Dijk;citation_author=C. L. Chen;citation_author=Y. d’Aubenton-Carafa;citation_author=S. Gourvennec;citation_author=M. Kwapisz;citation_title=XUTs are a class of Xrn1-sensitive antisense regulatory non-coding RNA in yeast.;citation_pages=114-117;citation_volume=475;citation_year=2011;citation_issue=7354;citation_pmid=21697827;citation_doi=10.1038/nature10118" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=O. Voinnet;citation_title=Origin, biogenesis, and activity of plant microRNAs.;citation_pages=669-687;citation_volume=136;citation_year=2009;citation_issue=4;citation_pmid=19239888;citation_doi=10.1016/j.cell.2009.01.046" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. West;citation_author=N. Gromak;citation_author=N. J. Proudfoot;citation_title=Human 5' --&gt; 3' exonuclease Xrn2 promotes transcription termination at co-transcriptional cleavage sites.;citation_pages=522-525;citation_volume=432;citation_year=2004;citation_issue=7016;citation_pmid=15565158;citation_doi=10.1038/nature03035" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=L. Xiong;citation_author=B. Lee;citation_author=M. Ishitani;citation_author=H. Lee;citation_author=C. Zhang;citation_title=FIERY1 encoding an inositol polyphosphate 1-phosphatase is a negative regulator of abscisic acid and stress signaling in Arabidopsis;citation_pages=1971-1984;citation_volume=15;citation_year=2001;citation_issue=15;citation_pmid=11485991;citation_doi=10.1101/gad.891901" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=L. Xiong;citation_author=B. Lee;citation_author=R. Huang;citation_author=J. K. Zhu;citation_title=A single amino acid substitution in the Arabidopsis FIERY1/HOS2 protein confers cold signaling specificity and lithium tolerance.;citation_pages=536-545;citation_volume=40;citation_year=2004;citation_issue=4;citation_pmid=15500469;citation_doi=10.1111/j.1365-313X.2004.02225.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. Yamada;citation_author=J. Lim;citation_author=J. M. Dale;citation_author=H. Chen;citation_author=P. Shinn;citation_title=Empirical Analysis of Transcriptional Activity in the Arabidopsis Genome;citation_pages=842-846;citation_volume=302;citation_year=2003;citation_issue=5646;citation_pmid=14593172;citation_doi=10.1126/science.1088305" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Zakrzewska-Placzek;citation_author=F. F. Souret;citation_author=G. J. Sobczyk;citation_author=P. J. Green;citation_author=J. Kufel;citation_title=Arabidopsis thaliana XRN2 is required for primary cleavage in the pre-ribosomal RNA;citation_volume=38;citation_year=2010;citation_pmid=20338880;citation_doi=10.1093/nar/gkq172" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=H. Zhang;citation_author=J. K. Zhu;citation_title=RNA-directed DNA methylation.;citation_pages=142-147;citation_volume=14;citation_year=2011;citation_issue=2;citation_pmid=21420348;citation_doi=10.1016/j.pbi.2011.02.003" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Zhang;citation_author=J. Yazaki;citation_author=A. Sundaresan;citation_author=S. Cokus;citation_author=S. W. Chan;citation_title=Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis.;citation_pages=1189-1201;citation_volume=126;citation_year=2006;citation_issue=6;citation_pmid=16949657;citation_doi=10.1016/j.cell.2006.08.003" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in Arabidopsis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
<meta name="twitter:description" content="Eukaryotes possess several RNA surveillance mechanisms that prevent undesirable aberrant RNAs from accumulating. Arabidopsis XRN2, XRN3, and XRN4 are three orthologs of the yeast 5′-to-3′ exoribonuclease, Rat1/Xrn2, that function in multiple RNA decay pathways. XRN activity is maintained by FIERY1 (FRY1), which converts the XRN inhibitor, adenosine 3′, 5′-bisphosphate (PAP), into 5′AMP. To identify the roles of XRNs and FRY1 in suppression of non-coding RNAs, strand-specific genome-wide tiling arrays and deep strand-specific RNA-Seq analyses were carried out in fry1 and xrn single and double mutants. In fry1-6 , about 2000 new transcripts were identified that extended the 3′ end of specific mRNAs; many of these were also observed in genotypes that possess the xrn3-3 mutation, a partial loss-of-function allele. Mutations in XRN2 and XRN4 in combination with xrn3-3 revealed only a minor effect on 3′ extensions, indicating that these genes may be partially redundant with XRN3 . We also observed the accumulation of 3′ remnants of many DCL1-processed microRNA (miRNA) precursors in fry1-6 and xrn3-3 . These findings suggest that XRN3, in combination with FRY1, is required to prevent the accumulation of 3′ extensions that arise from thousands of mRNA and miRNA precursor transcripts." />
<meta name="og-title" property="og:title" content="Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in Arabidopsis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/4/487" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Eukaryotes possess several RNA surveillance mechanisms that prevent undesirable aberrant RNAs from accumulating. Arabidopsis XRN2, XRN3, and XRN4 are three orthologs of the yeast 5′-to-3′ exoribonuclease, Rat1/Xrn2, that function in multiple RNA decay pathways. XRN activity is maintained by FIERY1 (FRY1), which converts the XRN inhibitor, adenosine 3′, 5′-bisphosphate (PAP), into 5′AMP. To identify the roles of XRNs and FRY1 in suppression of non-coding RNAs, strand-specific genome-wide tiling arrays and deep strand-specific RNA-Seq analyses were carried out in fry1 and xrn single and double mutants. In fry1-6 , about 2000 new transcripts were identified that extended the 3′ end of specific mRNAs; many of these were also observed in genotypes that possess the xrn3-3 mutation, a partial loss-of-function allele. Mutations in XRN2 and XRN4 in combination with xrn3-3 revealed only a minor effect on 3′ extensions, indicating that these genes may be partially redundant with XRN3 . We also observed the accumulation of 3′ remnants of many DCL1-processed microRNA (miRNA) precursors in fry1-6 and xrn3-3 . These findings suggest that XRN3, in combination with FRY1, is required to prevent the accumulation of 3′ extensions that arise from thousands of mRNA and miRNA precursor transcripts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
    <title>Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in Arabidopsis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VIUasOW7mywbmMXHnthqM20mYFoV2udXFpDhW7vzr-k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459266","apath":"\/ggg\/2\/4\/487.atom","pisa":"ggg;2\/4\/487","ac":{"\/ggg\/2\/4\/487.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/4\/487.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/4\/487"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/4\/487"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/4\/487":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229rBxmpkEpqamctTvsgzwDfLdJU+tKZvIWxNME62EowFxEK6F9nm4in0R+JhY3L8kEoXnT2UENI+yJ83aWUINUheXQMdIGUppXOAhPyQibwK8LH2OLszZu5OfzyPTg4XNUNtB10skaP1uPD6CTxT8z44AX\\\\\\\/v5Zp0WGQ6jSN4TYMIOZ5f3MOm6MoKk4rCGzp9fLrTU\\\\\\\/2f5v6FxRoM3Hkur+Jk+wmA28GqzjvGYFICAYN1oaH7gKipiUS3mAyJGznDGITFuyWF3PiCYGg8vGI0VV9pTlZR8oSC189FY9bSXt9Jxj7l8xFL7krYQ74A8oQNQVr4AINLFNv0MVTevTPy6B91NGbKyJ3MTrrYe2\\\\\\\/tf3YJ+X2vqlQF4fUeStcPKzm37lQB9Y8ItnnkrUjGJ9Dx+fHnCk4ioKIUD4LxeRoi3vPE4Ivmqy46FIQmZT8Zj4BCR03lMRhnrMx\\\\\\\/S\\\\\\\/pw4uhVh2p5zWuKaxTjUbxYyen0jqrqv9lZEOfUQE2Q3dwH5kY9HbNKGS3OjX5bK0zwMQlKGPvR1oSXx0EQQmRYTyPUrDr0TgfkkFPJXQddQJlIBH6ogXKg4GgSuU3KRx\\\\\\\/3R93l81mk9vJOkgwp6qjGo2BgZO002emdbIgddzrp88CY6gbpqLvJcjwuvTbeZof5x41t3MSXfsXMASupM+eRS5eBGVkyNPygAYeSw6mTPzHbvhoZ\\\\\\\/kQbF2d7JTyPUc336dWlEMYOTsYGQ\\\\\\\/M+r3eBAESyrUI+qDw8qXofY1kv0oqRNK6KbwpKtI5Dm2qcmfbSrt2MwuJSHgBrOUNI9AkHCQVLEudMQl6c8x43A249W5NvxG0sO36SNinyY7JQ7n2ucOzIfa1\\\\\\\/hqb6\\\\\\\/0P5cn7MI0D5KdRaAdOZXbvN4Bl0QwmX\\\\\\\/Te6kDBBWmG0Nl6WuJJKpbTv2F7BIHBDwDIwt07+xVG2o8wv7vib+RKP46wJ\\\\\\\/klbQE1SM0QHF0oFI37xxoNlewoLGVwvkn4d\\\\\\\/ZEnbPYORUmOAM9HrYzqhJWKPxBEb26GGGnbt\\\\\\\/dZ0y70PRfvHa5dm8LWrDuYFt1ZTHalnxtQ8IHO6S3PUymRAwwxlrs6PVs6mQolVEvBCLI9i5rSv1b3F8JtyMbQgjRHjSPJ6spU6uaLDx5uxdc82jEUJ4NjNQol1jUCi6AEh2P9eq1KJx8a28j+g2+HiTqHPa8haMgFsOCQv7W0Ed6uLoj1lIdS3ldOpyBHkiyJ3JlaXq+uSQmPIpeK96tGItPTA06zt+wnlh9mylpBuimUfGTCnuorFKlZSn2h\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022KjKpMSIyX+M\\\\\\\/sgD0QsDgrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022701c89c1c8ae0c0e31def9db8769cd62f68611838318a24cbf704a99eb777bfc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228A29xWqeWS3+roiOzAf2OjGFyAH9vIhHzVTKHJM+yty+imHlU5QmmFupDFbvdCTnuaq9y3RlSogwJReNu4\\\\\\\/aWG9gVzw4PLgxGIszBEQ1r52+vYzscQTVxlWlwnF9WT+9tolrDirncig3qynzdeEVkCtVbLTGhXg7+YIaJSlwEjH2vGo1ES1aszKG3BicpKt9o2nq8oC3S6BH9tC\\\\\\\/ne8K4UAOLygQ\\\\\\\/y84NR6czyZnRWTx5MqDtFmCuWtqxOX0McNyLGJaewJEacX1ayfrUZoJ8tcwEYATGDe\\\\\\\/Xn9M7fqZlA8U9o9Aw9RgdZOtz233t1ZyENeDQwdiFBWup6C0GC7GjRIi3KbPlF5owitd8c0S5tP19UBaF+5AuP4Q7QVnU4KgrAS2QDSQmD0Qufs4NM3Hk0Y99R7vk5opyVXBIuD1wNLbJ2QYIrH7WD1tQlxbSaXsKL\\\\\\\/N6L1xbUFPp9d6iPB30pMbLXObA8kCyO0ioHlJumgl+U7vI4uJy0jalsGr16EAmeWzwdvoxPSo3oIWiR9lhyHyUsSydcq7BkuGqbbHUzZjzoYgj9HPV+g+Tr0Un1MCL95QxWfUGAqe3gAguRJtjL3zcaklkJ+nlqZYYN\\\\\\\/YC6VLhG7awX8rGx1X\\\\\\\/Pb1WVabijWyEDcyd79bBTBO4ifQSwpWWQT6WLxUthKMEbVkLqaxrvGE0C2fomXbAw5aE9HVkGi3NbiEuATMLQ6I99r9uFRU6aRkZLduan0Rd2bwVaMqHxGLBAai6O6Ny8ilU7F6KzCchBABgMnW0yVk0KttRk1JJ48o\\\\\\\/eoQS+CTG2x8hEd9SV0mwvnBdBrtVTP83hYa6jU1Bx6VNV6in6AIZSvkHz513rWlaHDw7OVJDXXMmzUD4DagMPi1iexWnogsn4S1Ep9rhaMIA5FcpJzTtGYkifla+ewuvofbJlkmXdHUzbVQXQyyr796qbuJlSiCfWELUthcTVHRwITV0z0jKM5KanyNL1wiqf1\\\\\\\/HA71bIx9YyVyWj\\\\\\\/\\\\\\\/p\\\\\\\/6SoHNFrCDf0ZwOlzl9R\\\\\\\/xJKmHzgfNbo+KnZUs+9aPHkCe5M7jZlAVmJR1YOmPZV+oSiTz7zdmol+NUdGlgsOgImgJg5p0XqsNGHMYAc0E2subWFhCLNkzCfDyjBC5A+G8a0WhsBk+3pNMcXR+KdiA7BvWuqTyj3KRreO2gXMB2CJ0+K5JF0oJ+zzvqA0bHSeCtuo4IUtszhaqxC9o++99Q\\\\\\\/2Gq5m8P8+IJqA8RqYGFpMHtVvUABmwPlt9RoobZsmMOEuLgunFAGNQpVXK7Jhnby\\\\\\\/4t8v\\\\\\\/V5fPnbaZGygFM7vD7kM1DhULS0U4MOSynsvT17\\\\\\\/HAjjSBq2A1ZcS\\\\\\\/3eWbw+eCmrcNfUX5d\\\\\\\/0\\\\\\\/vQshg7vlF4S3ote0aCvh4zyDD3L\\\\\\\/BXXNdEaic4H6BaEUviTvgwo1BjWd1WgOWnwXNdQw5OAl6RNv\\\\\\\/nEHNf4pkVxiGwpP3hI6zS8dsROF1a3fAoPjaPHWOn9o41Zde\\\\\\\/oV0DuqMJk7xJgeD3diV9y8aI2txP0VjD\\\\\\\/xBP1cn1fOJM0mPGZhIZcRa3t8TERegkVbWCcldYf7QDHQhSVGKVjQeQsrkgFsA0zNLI0fbL9oNf3pU2zrW5oU4t59qNA45wtGOzC1Ry1Yrjkyx2j8aj3i7etjJyQ2m2pvhYaI\\\\\\\/pJoXWz+zKcAZ1h14F67nblayHdIxgzp+\\\\\\\/5c7BlS16hYmmn5jeOV5HR0tkFl1FqNVWU+CFMpZDIEKzBBnUnMm9JAVbBq3SkagALbYliFVznWGKzmSb5\\\\\\\/rhKy1sDDzemfkf\\\\\\\/XmT3l9Sj8WXTDWXS83Fm5TtR1x72OE4KjFN55Y1rj76C8uUyOmT5ZYS7U0yvGxOOXowyCixoJHNLCPv4xldFaCDCKii7PnJc8o2jmc9060j\\\\\\\/+WVjITfXtXmMe6o+ivQuEW+4pKZQygSa92x0s273rWKRc6VUqXo43uZFrxl9CS5f7A+TcJRsU8DYXumphMwYqBjUsDpcMxq+Dp\\\\\\\/aId5L9Dw5VU+IC6pOmO9wmiV\\\\\\\/CkcmtBkMZVN1NOO9MferMPEgjrAwWdSmW8\\\\\\\/k7bgmc1NqT9X2KSkIkY6uctUpefZYlUlOwNpqhr3j3wt0pk+pyo7189rsGSp1KzQ2zzDsjPqeGVwDJ6qWDw5i7B\\\\\\\/4ZBJRgQ30ShsCz5ExdimGlgbUQdMHloGDjG98RAe3\\\\\\\/7rTiwo3vQhHpP4\\\\\\\/UJdtU0B2jlXNjYftKwP9PSgKS2MNSkxMcxFEhxVXdKGrgJSS6oINU0V7hWe5+Qm4ZjBIW2Hjb4O91pfoHM8Zth9K271Q6i0DopB8i5qQ8as1WNQoRugAAEyjCAArRyU1Q54yWaE5sGf6XkRLA2Y1\\\\\\\/hSb6yOq4ZSfLg7pScU7IbqSPxvO+qlAiuCeYebWEVYgPUTF\\\\\\\/V4ZdWQbNZcLOAvYtr7WpipyUnywzBuBn9TqcEBWA5wIjG3KYHKwsseFrB8nDBBbyHKXSkpUw46RTsXvAzWDM4wKopCqElVOG7aXKFAGio12Po0\\\\\\\/NIZOgVKAL1T4fpBiPfPpMNLtNkIax30a1\\\\\\\/sjN6Qi3S5xVO0MeUxpEeRo4PEJO\\\\\\\/CIymVDMKiV4OHydl96P4NbOaOGTTqQpK1H++QzVSUjrjRpCMbnML1NDLxR8u7zGIIn7sZr0wECKwZH8fkqPk4tD29qxYsU4kfAB\\\\\\\/4DTnaaZK168sx4lz1RgRGdQp38EMUrDHgLlLsFdx53S5Xf7u1xrUFGdrieluhiizjWBDAnrnrgBDTzFzasFAXsZqFQpwMfwEiJMU57OBLu1G05anVrq74tqypNvv1JbDynmdGCkJ1kr2myXIOIaDiM7kXWAc42ABc+PIt2tZZ5Ly9Z5MGjBLlqfwBTqveqP8Gr8FA9oT4nxpxBOnXKDcy0k5tGPExxHyI4DLmhJeTZBQrbZgB0ETjyltgnZXRMF83cKhfNVuyX0oQV+cMwqblT+NViRNIQTjR2uK\\\\\\\/SdON2QPDF7HScxqtISMEDRBsfAX0jK4Ek7kIF1nJfT8pw+NRh1Z853SbaKWMDNIbTOX+Vc72e8T7+MiP4btcSVdn2ObVeSvqTCym+r\\\\\\\/kuUi+iJa2AtjMCmXF6zCgHvWcvu3nlq+H6n+VhZvLUlb5LCQt4b5BQA4hq1YXKPLcsEfVC\\\\\\\/rEie\\\\\\\/u9\\\\\\\/wIJnvhtodkSkGAOpQbr9v2Z46NxofhhgCJnYQJkz\\\\\\\/ovIdjaAPqK4NbYskr+X2zjcJn0s7ICibg1fN5o50VDpTJVhnfmmTi56xG2rBDJwL4D6qaN4l05HInoSCwebPV20RlsLS6b1WHqAftNDM084dPJ7fUz2OWCFjkDZn6qbbjJ8KB8apCA0K10qODCsuw5ykEocT7mn0ngyDuIWZzX\\\\\\\/Rz8VeykO+5sCLVrFOJHA42XPdAZc7uRMishMHA+EUttRrB\\\\\\\/fIvpZSKjR69lwljtKUDfJlTlfMdW0HmbeL8umxaEvF0JiooNdtyqRzFg9RYhene6\\\\\\\/MVN99mtBdBgTu\\\\\\\/LXu51zmDhdWuFyVUGBpvy6bMnHweU8ucUJDXCezjXxMz7wYALC3G9+WSup\\\\\\\/Ye9YmLUOkPLANWB5ZbavmcWTGK1QtDNJtlwciFCBus8NV2gYKfD10dqQXvyKGhK2vKBWVEoMPuKkqXBIz9eUl0srw3ieFE0K08bao2CldxVm1yxCLQ+mlKH1vyfEVc56ZsyV0sQfjayot1shnvt\\\\\\\/hFMWBdZEWB\\\\\\\/ZABQjLbad7kzyxOdmmQrDH3mDT3O8YtzJf2cDmT6QkfrrLbXavUljy\\\\\\\/3ZpRIGVSm4H5OtLGylkDjzIaawcAtnrT6JoXhOnurylg54DXKTF\\\\\\\/gJAZp8sNHzG81j9sDYLdQGQcXQdvOVMTwX3\\\\\\\/eu2NHTDa3GONyLHrQTns5kHm\\\\\\\/E5sh+E4f0CF3+QlJvBB6JJY+tt\\\\\\\/z+PvJFSEoanyEn4bzFAMySvH+QSqZEWhD5a55XmSl6uurfVOT8af+w1Fw4LOBfUdzIefnDiFK0SHRykY8OsuKWBge7UEQJVf3i1bVJB2mKWqU0MsO+CkoHk58P7i0CUrbeiGZedpxN5sBDBOrAKwX1WSYF2yHNb4m5BQTOma3W3eiFqWr6CgklvYTgjGBGczUULFVXmfunreGuqfl+YkT5ZT60RQyxBtQVwKKKcwGrS0O5yeLfOohY661bXUdEXMCXBk8O9QW59YGjEZDFi0dd2bxiiz3B4SztLUTUuSiw320P6FNleWdN+fEWhUN8iPuOM7+t9RdYJewTPvnwN0y105gQEDDn+Ve7ymt0zSnxfuzw92U5x2ZE\\\\\\\/1+JT8ZzbnChIPz+IH\\\\\\\/S+74sieSRZfDTGh6fxGPtH836XziEN\\\\\\\/ST+\\\\\\\/Wips519O8wYsHEC4qhqSXPBaenWgKxalketGD8ijX3BGJFynR7dwnq\\\\\\\/MSodzkzy9zx99f+3VKAslXvIBBDtwPxfhlcKbs3E4OuiP7wLEIqcE95OIZ4Z2v0\\\\\\\/2z3TN2olDVz7drHxzXcURJS8+YbCZvKzmZ8\\\\\\\/wxy1U+Emk1ckROeooy0GK7MiK2gJthC74SyOHCHp+RJOsgyuus6bsI3HmDCPKRwSplF5DCLWC4wzqRE+2EOQmo228BAZREk66fOuvUs9+Xf10IeXy5nGdMDfZ4igs10eGmhjIVerdoT9X\\\\\\\/EZoOdy+9xwRs+Kt3+3weQyNKyqubaPL0D6HoR2gnvBWqBkNdawcSR3BSoVmXhguoh67ahnpWpguK+DPQgoImbCWtglnvygiND0+8I+O9bcWaPbcxIAnQySIhHIi0JATAnppM+sF1JZ7\\\\\\\/NZYzqbk+juI6mxQlzXjpgm1rsxL0e+MFqApR0UTlunYkufumXOROTkwTSItgsQe4obrOfWcSPf05H29IhnCOU3auYgtgHApPDgb2KIznpHK5IZsH2WcoPnxRPE\\\\\\\/HvIDDtDF\\\\\\\/yveeFe909xdwGhjsswhdaTuG0CNYdIbn13plGg3JpjCKRrBhqrsXvxZeNjWWhyqsOcGJIUVECkpZR217tK0ikBS\\\\\\\/21PgkWg6KF5viUvpqFITpexMXmmrCFbjPdU4WV+toPmor5RuOYEvuHbw0K8BCg5R\\\\\\\/aGVYqJ4rP8aqYZp4I468xnkRkMv58wb+FtxnUffenqVxZiSYbJpGnRfgUd3CMZgktvaKq0ZzEH0AQIT0eam9rIulsCvS8ih584A7gto9RrBQJBwPJvlRF4JY58FOeokvFROlaSaB4sgXTxt1ql1A39z8RgTi2OE1S7TKpdeWvCOih1TfJk+HzLy0h0LT8uqDehjIUa4UN0XhSYLVotCRT0L9Dq1+WtSy8\\\\\\\/1Zn1oXNOHtFDcY9IdB6q2Lwsuv0t4QLsBBetefXc2sxxgeNTGWYTQHs+w9bQpdbVAwteS7n1E7ye2RuhlaI+wEtdSAfcirHho8gBJcY0ur+Kp4MOtH+VKM8XFI8LZv+7raO3zy7jCnDPdm6oTv4ZxwPFB0BzsSzS4nNlRxctVuS0Nlkpw1TkpVOO75vkHaq+kPOLWiBxYp1OVJ6oH7Yh7y139M9mkbDE+6r6\\\\\\\/CTHIgwkynHIxb4vPyg95PO2nLLvXH+vrUenW2sHsPzP42NmHCUk+HOpRxjpw3cBVWz7f8Vwfgb3le8slQf8kM\\\\\\\/4bK59EBs9QmQggWSoIf0Y3Z+HZ2NxIXKajHeDv5lbUJs1IkjnknWOq4icYc4qzLsa8S4mSkAFe0zPfYL2Z+rTkjWtRZySVK8WmVOOOGbdtJVVUYzlr8vZjIIbO7CRWt1sy+ZDQSp269vTJYRc2FyqFFK82p2z08wp0B0uquXlfFg9VcP02TTLsbF0BWal8kEaGM6dsTr4jg8Rvylub1vfbog0Y4TjG+kVijLdjCkR2vLlMiGFW0IwwpVS8\\\\\\\/NwmKZxOeeZsUs1K8nlOcFVDy8tamTBjYpYDwHj9AzGexwHscUSoVrTJVA2lmaTplPruaB+osKgV93Ii9I3FcQnVQwhtuahXy9nlsOsr8fUObCE7Yl95MajnQzNDKsV1GobuPRfcK5Cv4sRQQS1GpZHassKhsPL1wEgvDLQb5YAWYD7mBC8QP4x89NQC\\\\\\\/l8QFQtJ4Oz0DjTJeioM38VFV+RJ47V0\\\\\\\/bNznKSmtsC6IK7v4BELe5ZleFgFqej8DXl\\\\\\\/QHamXIyMIW5Ywn7mM5Ibsgl9NkzHbEQna\\\\\\\/OImWJ7FJSD0Dw66xyYCVoJsC4\\\\\\\/NE3gCJaUYUZf57M1OhdbWR8lY692sK24\\\\\\\/CAUdUPsNCVork6yXlAmTNoaVUh\\\\\\\/QgW5R8V1jZRw3Xg+kmhuupVoBKOcDG57q73if3WKpM4VZW4NFlqvb2fXbiW\\\\\\\/nT3ppCGi2Ai0hvBI0q2D9XlQfAz2pnSjCz1L7sq1GIRSLpQ10PiI65KMW90sw4tF8j\\\\\\\/clPacdiiRwv6zTQ7JJNB3pvfIT6rDQs04HKLvWwG8iBeHpo5y36HkYxtic1Njr8yFQoW0EZ8JPuI50CI5hAXO69CGEZtxFEUnmYFEaVFh+92b1Q7u9SZXQNF0Ex6fg2Sxx0RHmV\\\\\\\/aB\\\\\\\/yZDYfbxzf8VEazyFCLUPZrqLpUqd53vFvI0cC3BywfxG42z2d3cLIoM6kyoJ5HEDqSqCPbYgPsyY3l6NiuEi9KfVyQAYrdWVgBLO6BwXwFbZWNvcCUrWF4DTIy6NINvdnKz1MjPqGG6lpMl\\\\\\\/oy2WYVOr8f7tz1Xe0M07ApuluWBWNa5wYYB1JFIBnLmykfBXtxWp05Pw8GGP8UGyURpcMYnLHnRfhvwsEq5ej8xy1PaM1kr+0l90xbjxUG0y0ZsPcJtOde4S8TyNFsDk0R2w\\\\\\\/yAhX6d1HjODua+X9abXJrMlRI0dMb71tlM4\\\\\\\/tH73vtQap4rh8CVWi275wsPHrLvqih+v7g9pSCMyvC2yT4i\\\\\\\/pd903myqTWdoh3A\\\\\\\/v2UkmR1okm9vr+pZxW9knc2UcTy8dW3KC7hOdbfI\\\\\\\/bRwtJ29toM2vzQN5x2AMHl9fYNsyyirSJoGrg3abCRShcw7+nj1SE14\\\\\\\/0jbv0mDtIhbxyXYvMbnEXPSRC7yW+9W8NyDKLeP3vI\\\\\\\/6z7gD5K5lAYnxiHF3\\\\\\\/4oSnEY1+C9YpgRHg+\\\\\\\/Q8WH40AGmlGSCQys01hIiGa\\\\\\\/8lBX\\\\\\\/BA5piR+o5BiAEODyaHE4Aoj8p3Nt\\\\\\\/UvAnSqV7GObF5sRawvUuhnVJnBtIHtjUH1YUFU6WJlzxcC28lQkJFsYZA8SmsKh9R\\\\\\\/2nPJd6D8pgpyAcfk9oKWGnl9X5wSMuGXFtfVVorhifwBryZ8QECaIPFIl5cW+fELuq9E4rs1TY1W1i6lxzOywQp6PMXDtL5xZOCpzPfozmQTojlSwn2GAKDiEMZbFPljhgmuxRgDUhZyE2BkCbMVSFGTcquPC\\\\\\\/uhgTJ8OUc0rJVch5y6RWdAYm6hKds\\\\\\\/Lr9HGsfDDlzhFo5zE0X\\\\\\\/qt\\\\\\\/PTZtK2PykS7W7cj7aeptXPQ7cqiNFmGgRPIEJ5w8bFB7n7+wBWb82eI\\\\\\\/M0i70+h7WgqOcWyC5vIk0qw+0eUnktmICkPZYLj1SGssYRmDxtfh5fJxeFUhy9thzXmsahbht2lx3rRoKiIA1Tbbw+3\\\\\\\/7hVNn\\\\\\\/UzyNLU7JXrQJhCHua94O4wIljy1EYBn2SobzJdZrxViQRO213ExdlVjFF2nCtAxz6dZdSu5SDKj9r8QcF1\\\\\\\/CjkRRExYUd8nBWfBEYqtkT1HiPN3NPnmI39BC0t4xh65VlhfdWCQQpum6TraK7bKMUIvJFLFDO2J2320Hz\\\\\\\/E2fKEOIARyjHapap\\\\\\\/0CeXao9Wm3cLlg75ZVxH80+Akdu5YzgOzUP5vTL5cmSddWLUe1fjiKKatV+dosVy4QV6usscgqi78EYihnDYX1eHUok3s3bMUwecz3iefrakrT5sUK0aCN6eeYP3+3kRue8BjYrBCnFaIS5Rk0tn6\\\\\\\/Gasr0m6vGZezRfFrGcrhO\\\\\\\/oe31IdidH9qZN9gpC\\\\\\\/TlIf5wOQCDaweQvl1qy0I0\\\\\\\/gqhxef2B14B54oUknJAQrKE6QpvWow\\\\\\\/UVasB6MdGiHMSM7D2x0RRsbDCgaWBjrxs4qLo\\\\\\\/\\\\\\\/IxoarUVA4S18HMQGQo17Yvs9IphIe3Ur4eiETHopScgpSV\\\\\\\/oc4Njdvcum\\\\\\\/uuQ\\\\\\\/DDkc7tzS0IxLQPPfNkrtNzY5BT1LrL0dj2qdPpcyjiChvKkWHNrckwgET7PD2GqffMQ5kiwAIh0hxmLrN5625N4bL9BK\\\\\\\/C9NtLKAlpBBF7hZhaGxnsfjV5JLwNqxJMG8Mu3ZvV+P+EdfWLCJJVhyRwBPvjKJDSHTLSa4KQMMWjLCJQMwv+aMHW8GBhOpQ9xS47UgORF8X0gl5NjxieM+J3SrsM7VJVqFqv27wpz6g9FxS4YdDZjHmumpG3BZtcTIRqva6+JxwJdV+QPepFjanXOgEWfa\\\\\\\/Te7lhmDdVAABWmS\\\\\\\/65Wq9pxiiOUkECbuMFSREcYlvBBQ\\\\\\\/HmSnYgo10ePkqxs7wLsk5REmO+Eq7e2kaOR23Gexq6r8a2Mpe1UsnZA\\\\\\\/4JVnI33tLc97xIvgkVHLtcXLoU5yPo+MHruWbV8ezWMH6D2QOH+iVB9MKcot7Hz0ukV\\\\\\\/P\\\\\\\/1v7viCRPgFl70TKwkcWH4Mvb5vuMmrYCIZMraQch0FHPNHrVaKp5e1eCog3AhUwwZRx\\\\\\\/OS3ZVy6IO4Hh2tXi\\\\\\\/qxAUvcuKLXV\\\\\\\/35o96hLXOIxaUhxVvozH3C0uwTKn1wNkdiMqPg0cjj8QjMB7SSjcyw244DhokUhDC40t04CazRuMGaEJ86O7Y42+HEaKPMaTMC9\\\\\\\/\\\\\\\/JJMENUov\\\\\\\/v0CWrtR5tTKTtzARu0gGcF5IAesR6U2lp11f1U26svM\\\\\\\/jvIVRspMy4Opg5uAcutAAwqmLZFpvX1JaWIN+K+DTZ4oHtk9HOO5eIoQ+aEgBwqwRIWkUwDJy2Qi5AXDGMOYQ9fXdxTi\\\\\\\/y+lfxFWn+yCGOS+Lj2D9GGmA9o8w+2ePBt8TZr\\\\\\\/\\\\\\\/tSKIHZPlfpdDuSSAMnut95ekmCOy5u2IULzYHjWF8UMaq9Yy5hh87FbjTzM0r\\\\\\\/hPKNi\\\\\\\/XNez1C3mNtg3kxRBPuZPLSMDslR3nQQxGHhw8+49xWg0oJpOfU33n0sIOVKTAu+p6BbR7DOs+EmBeOzFe9xYRlec7AFA+WHi1z92oU9H4xkQYLB2LWu2J6c+fpFJFIyYK58veJD164+5mNi4MWUrgGd9D4ZV3K5hsPe3BUwGiTC5ssEy8UwYbjhfc0y5JDHdBxAR2+C1EzuT3w7z30mqbjSWanC+MBsNXIMPUA2X6jG2dKpaiSp08f2dKHEUhFuI\\\\\\\/x8eoh9EtquGJ0dFU0X4k50KdTzuPFNwpVm+ul8O3xODVHcuRhVdoweUrB9ReVhbrQbTPBeNvl\\\\\\\/VB3\\\\\\\/G\\\\\\\/nFdEfKH\\\\\\\/IP5qrc0y+AwmkoQZ1smen9qeh1QutuFbLoDJeLwOiu+4G3DFZCEfsTcvtQNyiwrzl61r+G9xBVpjmjMGyz+QOhucDfWtwlu\\\\\\\/P1npE8FOssCvn4DAKuUKxD3D0DvTZJ+CS+By9Gnpo8BHBvgfTZ6Dj4zVkv\\\\\\\/z2lYSTLTqx4Deq49j87aqYjneWz\\\\\\\/Y3J3ez9JUi9BUiAoDezsLZmjvblvqCVFm1kyf2UhRxXAIvFmOyUHiv6EbeEum5CPfg3sHrrcdejFhR\\\\\\\/Oe+aJ53rSV5XXTr9XS9QO4u21QjJsGJcYiJpJHDjNBOigi+ZeqCunKGIm7mithA39dwt\\\\\\\/8IYZemQJUtVhPO+FuqnU\\\\\\\/s1voZR6gEXHS3awSy5FVlYIkwGhV79C\\\\\\\/jdDcPIptazD+Z2MS\\\\\\\/4uo5aX\\\\\\\/31Ryo3g45TIIfKWxh8vNzHzU2uSXY9YHKC90VRMu5z+pHftVSBypw3wwxZ9VIZeXh4DeVhD0GBqzxq4lm75iWCaBmbVs8MFN8lPKXZZTiUQJLUnVajaDaMPgHQ1Ezm6cTodzuNbvVk3fdF4G8pCA8SUk0+x\\\\\\\/K9+xTb2rq3jBQBXXoM5QhaMS67pecOMoiCJ\\\\\\\/syS8u9YmovJnBqnG5ZbJmjg62AHJl9vYWjAiwf8l+YBzAB\\\\\\\/htvxxsekbY98t+hoGBIJSmUhk1ZrgbAY\\\\\\\/gboLw1cAoqu5DHnbQ2JoIXOQPRGhkHxaULK+sNCuih5zmRkl2q4jn5pXtnIFWLG8EVfd4YV\\\\\\\/Qm8kIqC7shLlFaKtSo2eZf51rGOL6OhB\\\\\\\/M8HFy6gc+xlSRyzAjPqWRumwpjEsO4fZi\\\\\\\/4k0cvO89YyqU4cfH4SUHxfA6Oz\\\\\\\/oX5p71fv1M20XgH2oQ6pn16K3sPkAhAapvI3x3P6e0lGW+AD0mahAvHWVDsPtETuQOTazgyvEaOh6g0m3lZ1aeadxKSR9033WbdKwYYLYLIA8PKbIfesJJlBig2QRXd5l+tnta52YfC0Iivcd8H6YzgY0\\\\\\\/MnmKbDSr8qzAm5pubDa51MZ7T9wnjKAAvTqphymrLrXEpMbE+E5wJ16o0dnenqw3RFAofv8aWMrzHsSLdibYoaDEJeWwiwpTGR\\\\\\\/aA0Zq0buxz5wufb8EabW0Ny34QMjbCMyrMr+TVGgg0NY1w+tCx1AcmijPebqlTyEg3eBG\\\\\\\/dtdfWvSXTqjKkEhTdVRSSOY8aX7BzQPhMOUpXYLzyaCwWko+zS+mBzQhzXQ1tDKpXd1D4g2zLo22MkY7qImlAaYLV507u3CD\\\\\\\/DPMcpEDjRo8V03gOALEGWX\\\\\\\/R0lDoDC9\\\\\\\/np6UdQG+bHojEcF\\\\\\\/AR\\\\\\\/JlZVzbCZzNvt\\\\\\\/CxWnaSMKmVDzmuQ2EimYZzLi1wNbObbxbdpJgatTFpDmp2B4+1TThR9piXlftPCpK4i9qdwWsn6XfsytIXjhTFA5fiSRBOcWXW8YGV9WyIxvZ\\\\\\\/BD6bAp3Nq61O00HgOHVN5na++AKN8nt8zMlmqfUI51D73Q8dRLiiC3FfrUHzSr09di4LTcTulea+6Q7Prjg+zb3vAL5MkL8Nib1xzkzKxKqwlb2SdjDBVppBpA3zecLSPYmmWFuRHAL12CJf3nZquyiNXBatQkFcrphKMfYc5SzKR\\\\\\\/X0Bx9K2b9ZY9n28IiyZQoZKwGZo3Y2ULHMzRtKG6RvKXoksjZY09w76A\\\\\\\/6W1CtftiPutCYaN571TAhqHfBqOCs2BPBZyMEJTz1viTzSCR9gzE6VqZnC\\\\\\\/zxGZMkceQMgNS\\\\\\\/yfmcLvt0Ahbw7TCg+s45yMAzni+w34mDnaf51ZEsAuWOQS6gCVJCNUFeHtAAQ4WRMVVkydsnk1m5znXPU\\\\\\\/NpXVW4mD5q7IbrsDpj7Ou2wer3ZDJgP+2J6hlmhcz\\\\\\\/lv5ddck0NcqqzOMiBjT\\\\\\\/kFjGtJ70lt5hzTKP4hCOUBeea24PB8CbMFiOIpN6bNL4H1R7ot1K0x\\\\\\\/A20ooWye5cjRCIzXrWuB\\\\\\\/Exy9E26rX8SL1eOcL1voBZMT4sKKqz7omdLHzp9f37VNgGU+ibR4eFWwUim+x58TNlKoP6+zKgpEMLfnN+MRjPeNvVQqup9z9G8KL8QrjBLPm9WrWUvAqv3gw\\\\\\\/Jk8ejP37r4a\\\\\\\/ZyBsMMsaQQ7N5lID4CiOmLb+GVg33eInXRUEUu3UnzoPKMB3rM2zTBGnrgojRi6yTQsn3cOooZpEvl2LeTBIyebEPr3\\\\\\\/rEMCnapYDNY3RcxkQ0FrB3ubzxoE09\\\\\\\/3fe8MWJqWFMhTHebjaaSbHHB+ixKxNkL8tkwvzXYbT0t0xmZOMHv1bvUvcbg6B64dueaRl1w7E5buX+9PyJavB\\\\\\\/Rb4KeWyUBFBbfzMbO1iuKqbM9YuvXnlO\\\\\\\/uaTZheOPxtc708V9wSyo6Whgnxa4lMqzgZvoMn9cMfP3Qma5JprnAmAjL+c5t9aZM3uwMb7Aadgxguk2SwaoJ1GI4Z2HHN33\\\\\\\/801qcZK0Mna6ifooe\\\\\\\/Pi51T8LtlKtnd5wRyD2B06kQZM\\\\\\\/G7qYMi4zLLWqaW8qC1N5hgvE53XpCJvw4c8D\\\\\\\/VD228IJQNIrjt64pub9lVAe9aGsId9AEeHjfdoo3zVGtZ11VxsDsCwOOzO729\\\\\\\/+TkzgcZGpSJocQ1ZJbsoprmVcNt\\\\\\\/L36kqQ1vl8M7UyLID519WFiVHyJEPnYjJ0VXbvDhbey+AyMO\\\\\\\/n8AtlacpbIf9BYL0ByBzsZH+ujzQzsOJ9Oda8+4LGhNbf+c+6sSMBfKr4HBqbDenM74FNiri6ZEuA94yblq+h0gWuNUIU9qyqB\\\\\\\/bhC+0\\\\\\\/nFaj5YqWxnqQtwy4uZfL8yYUU225cKaqHFzyc2QCEG2fc\\\\\\\/A4uBNEpwgTbhJivPr37KGxBhURnAAFbzvFvpULIFwm46tEgNUpKj9MHPVrq+0so97sSg8I2qBB0LnIAjISD+suSIuWJ6xnUKtHoYsqhTwhu\\\\\\\/ktfA9aHVLquF6x6EnQlK9gvPaVeJ2XAjQSLY84zdx9GGkP+WLgD542prGQacMh8ADzMFetgqatRO1hBUlwbABfnbz72YmaePc63d3Y\\\\\\\/mI5o22Hx7MxdSWB1T+i6Tz5mudmoiCT3XOIG0+VawDk+zRQFv+g2nbLJgcir9YRyBh9seGj7g34kxMPVob1KunfHM5euSZnws9xddBG0HTGIODuuiGJXomoVW8nXl8PjBiKnPySp8QqdymD6AdWcJljcraJjQL3IQQAmTFQx49pEhCYkAkJLF6z+xyjTnp979Te9AyFtKtwN9JV0z5DWA9Kr\\\\\\\/1Jbf\\\\\\\/HjJ7Y6+q6qVbyMf835uGJkbi6OwusPKBlpZtRHN76KxYywyu+ftYGmVU1sHmquQYlqjseLhZsL\\\\\\\/5nxM7DfUthlDzMgqLKnnwqyEs5BOxgpp\\\\\\\/Zsp90OCZ85ay11ZmTGtEjMKI7fj5vjcbrs1XrOyQsRXtJpKmgUY5FNis1oJVrnBCAOvJ52a4SgpYgv0QpnCyzcVq69mpVtqoMAhbMKTNfH7nSPQA01jRP6M2vrqnWGZk1d0TjfjcscXTJEOh0v0JuLZ4azl+x1caar4DGPng5hb\\\\\\\/D80qzqE7dPTPIhTTx5AMycqdrK4D8YbP\\\\\\\/TPK3b\\\\\\\/Y5fEPK4mHqyI00pK\\\\\\\/YSKu0B4U+xv4RqdrQSXIVlyOIuFuytXQLHpJJWYKOF5\\\\\\\/k1VsvnjXAWzGhR7fNt9dKhZfMNtVrnNlIzrYGPZUKXpUzcO0ji3qqHohjoT8cOFx38tqX1amO\\\\\\\/WxxKjqqu9ENEIptUcjhna97THgvTXN80mI4f5YTpmpzq64XzmDJ6zeFeVMpV\\\\\\\/XZrGE5jwO0N+sdUpAkz+29Kh1V10Gd9mInVmk\\\\\\\/IjZaoo2bS7BFZHYACwu1ASTJiSeOC+NJxUrG3okivGxOWI6upnNXmJjG+WnQ9NZfkfsIroeRJVmAkQhNnmruAIrq7q51HIInFN1FX\\\\\\\/RP8H7+p\\\\\\\/kD47oM2P8FK3ypfiVrCwNadkEBC9F0HkZpjQVc0+BjZg707ohu\\\\\\\/dzxDouxS1vU1pwf5BSTWBD6TtW1RmYRkBgcW1i7Mx6CDQNFlET5cJNNTCQw\\\\\\\/1gRzBESJrLPgd\\\\\\\/BEH3sBgGcIRnfEcwjQBainRoLJ\\\\\\\/tYg5jiwHkf\\\\\\\/tdKXtfnENY7GTlx051wrS7kfxHWxKMUBchCAqjqb3KGLNlCIR+FXw3eaM6+0336BbmGzmSxENJV8uPmSbJPcsHdi8vJazAlVFcN\\\\\\\/oIn9Ew+z4G8H2uU4DdjJN20KXMUb1tSGrPGX7HukNMAYwTFKE5v+fUbFlOUk2sEF820a7GDlrOVu8WwTvDmfRly0ifgP0gvGBK5QTxxEdJvXHKeHFBrfzaVdxD0s5uGkZ0N9JFTAWndcBrjCJvWLZJmNk5HxV833SlbDIMfh8DhQSFhxyjYTV88RTOuezfhUcrocauKHjAk3SPM8G5iY9nuxCD6iSxvCKS6Vdymc\\\\\\\/3u74pioNa3L8823hN0SiPMBZekcCfv\\\\\\\/QCELZknYPLVIS9RJ77pzjDsuMEbEZiTW6aOkdU0VbYgOD+AVHh1AIGAe46QiQDyw9XVQaIXzekoGFE3GCA8BpuHBk+98R4u7wZjLZOVpD0\\\\\\\/aOBpgf1+eVDQoUOZScGMc0JnIjRcf8dVPrNGiqkgeU3mOnBOCF62Fq50YHoNV\\\\\\\/OY2QUFs+TglOf+RikuJue9cPPq8T08Hwk9BC4AigNrkzhLN8+VABCsEyCxRIwfRUHGGA5\\\\\\\/YyFBbPJG79bCjdAZXw1cJpxjsythj0i7VKJRRHVmXxx63XUzCdzjoDU9dFKpbpFeYdGhH3MjLsCy1Bp0p+lTblV+vU2P1Av+JI2jxaCKDrAvOQ1BC3Lc\\\\\\\/Ke1emeAQ+frTobbyh\\\\\\\/Bg4\\\\\\\/afasZlcBNy1T7tvYMipPZABzb5QAd4ZdJKdWUPT92bPHhN7VDW\\\\\\\/LDzatSmSHHCTREnXTt8wMhGHoKbPL+vVFISxhLCC6PHubQGLq9Rv7ijsInb3S11+RAOTWD9ehFXzm4emJN340lS+C0\\\\\\\/ZwaDwqg3Cu357fcIWDNV\\\\\\\/YQTE2vFkSLvr9HxFwJP38FVhSbcvOWufiN+m5tAQrDmJmm+j8df3EIgyjZatgef1wqMFL4Fff\\\\\\\/JIEv\\\\\\\/jLLZ0AsAU9frQRHIm+qoorA8jqYe3q2IbfkYkseUOgmCPuJoOZkaomu50CLttxuNeK+Nn5eNd4ZupbPQqTP\\\\\\\/RDq\\\\\\\/IO8v8P4EO3H5TZlSNmIKJuDU85GWUmoW33jkHZG4w+OScp3hSoviokXS5rn45WAGLbDXDl0O7b7XIXEjQBaux7iMTWrpAIkWMMeueyDfKjyPUARHrXL8ZJdRvEwVZNNTuW0hg4hDovp\\\\\\\/s4y5MkAp8yWJChsL+P9vlM2eeEqoymY3kc7mLHJUcdWaGeiMTdEqsBrZy5RswQl8OlHCdgZR\\\\\\\/D\\\\\\\/ik8fFz2JJf2+hGRzzVA+jPyi3tQxanFdPocowUWI1aP9K5zIR4XqbOuJCXU+1tUX5NJycpC5elaVVCPLCuWpld7\\\\\\\/eb5ngZ4eHAKoqHK8WrbmYMXXE4+ZhtUiXQnE\\\\\\\/R8+pCJHXFYGHOSAImfR2c89\\\\\\\/TTaMSSVUDB17tWoFSw61J3oF9G0GyIgvRoIUKf0Drnd0sYZ65dBA7vIUuyqOLtKnfPivx\\\\\\\/W66OnJX9T1ELknqusgQILEJBsHKBroGV+SBGRPTNU6\\\\\\\/992kkIJUe4c3Mh9IeDbbO98ghqHpFRAe7zLHL953yvc5\\\\\\\/WC+mnVXVylsst35jv0xD\\\\\\\/WdZRCYXqd3d4oAClYR9de5tJzE8HLw\\\\\\\/g36AFsAQrSaEo52hKTwAIaYzSsOT\\\\\\\/Js2evVDq6qAaohVPml0YU7xnrnSz0hWTrOp5HKyQM6CjiBW2iZhvgF\\\\\\\/L6jsZrFV1QClFxZa2RrwaokKzvwM3ivbv03dW6HnjjqukTKfjqcCmI5rpHLS\\\\\\\/lc+M2MIVRpvVtNk2gx6iluiHFHc\\\\\\\/NZhtPPKUSQ7Zu7aat8eoNyB3CnGuo9AhqG6y90jtgCPjy81zVbJO5ccXItYly\\\\\\\/2F6V2g6n94crwFLGuuMUjzLEBQQgZm4btWr\\\\\\\/g6sBZzb7OmcLcDiVKATa4EOfcSaOtUylqXpyopc6PcxteyDoHq\\\\\\\/ZbVfzAYI3lvAChdkCX14ZdFWiduO1XXj8Mvs73zscgHTMaKO79P\\\\\\\/sMUeWgK+RyghciWJjbBhj1FJmp+zJ7P6V++yQGK\\\\\\\/+U4RICSD8EFHyagjomoH0CuL9ZCNqM8WGGtp\\\\\\\/3z2jo6wifPMoF4tciNQfJtoip5weMAbXR50BXJg+XXuiEKv2j2cojsNFLg8QmzSdvSDJiwi+TVCKWj+igbJQPw4DCehFgruhsG1CU9o10Q8IvYNew1T62a0rjDY6ryERoTupTNSCd\\\\\\\/+SMIkKfsbN4N0j2cx9CyaRN60oal6\\\\\\\/srWaAuklyw5rPCd5lZ7+mJqPgJWN17bq+H9cKvxIsSd93v+LgPZS31OuDjkiOQ4OzVqChukEMgVAcLSfNEk+oWtFKRuhdFjHYTKdzuq8l1SoUtgMUkSsvzI+HECishr1q4xsm0v3NLasZWh+XPnzFHrr+k3QF1nfmJAHzYqJgyBkkycTQy55cDuRduJ7LrqJXe3nXVl4y3fbukSK82qVVBqo7jNBp3MFjE3i4NUzu3h7WEtF32JJPdqtNEgCmAkpDVIl9LHzkXgz8c6XaonzvnzU9YRZojYjFkGB8VaxlaP7MEuLEq3CPjsWZ+1Z+C3rRN0YoaB5I\\\\\\\/LQoMfsCfpE+6I9FHreM6MQEA3efFnboZ56YFXsNL3oHg6lByNoihFO+pq1QdXZ6osrUGaj15TPS1gYdMECmTcoRnj20t16j3q\\\\\\\/nYlquyQ3lOC6vYayoHl7GE6NI3cVaHOSSVquOj68wn7p+KbKfm7w+xPj1TtTvBzW9CFdmlXw1nIO8yyu0GqEDTOAe+8ZrK\\\\\\\/cSWsgbnf4Qeu\\\\\\\/Og5oGGkHD2y9a9ap5cxnK1bLb+SZr0MWq6zMxcQQ5mXeizXRtqrnbcRtWNA4IlDS1tse7DW+AAyntxDppJZhZIqv9jafszOP5gSn04JeuLYzwgaukJF4bGFbxD6Hh63VDldf2hPU3Z8xPvMvhQ0b34EdcPAjiP1oqpvXRc8nVv4a3jouOh0z1ei0pIX6sIeervVBF2MGjUFbuBMHd3y+fb7lW1cCbRDMffXFzQAvxkzBzgkf\\\\\\\/MbboI9jRbgsNYkY63dap\\\\\\\/a92YC2YhC0redBLwYvmGX+vSu292XHKkBwl75GdEb7tBhywlUdV3Q+lHhLGhURMTmY1NAnQXaij2LNBVOto2opKOVO\\\\\\\/v0r6ec08w7Q0d2FSus7wiHCQJn+U\\\\\\\/SXkU0YjvSTWLolUVUEeauL1FlKA11tJtMdNcaA1Hj9qKtB6JlamYxKBdoa\\\\\\\/6MAeXSe2gp\\\\\\\/ccN0wBYhvldW8afTZYpRLc2Jed6PCxh+z4\\\\\\\/Rw8K8u2+NotFIJDo1A5IC4\\\\\\\/Q7HOt\\\\\\\/NIgN5l2dVXBT6j6lrls+HINxSJIyhJc0hT4KZVEqJtStmpjQd0i1q+nhir3X1vh6DY9O9Hqda2D7mX6rbEiEPP35B0ILwzx7JX6lc7xn+q+2Lvv+JrYm\\\\\\\/rwvHgP1AtYLv7D1Fzd8ARYtHOC12i2s2WO9TUcL4\\\\\\\/dH1K5lFW++7xDfegiCCjbSxcCdpUQ642RJIZJbCLNPcOhIa50WI3CzQ9YDCrbn6YM+qx+9iYuQk4yGLiLKjhEIekYQuhFj++SHybMcHC7s5tU4tjYDpxWufOhHfHQu4CK8N6YcnfOlmqu2b7o68hTldeQ+xv5Hma9cMlvAGqlc6I1zX5jF41oEdvPG9KvFf+JvMUzprRt\\\\\\\/DBLM5egM3Rqw45CZplIfPh2vHtFX\\\\\\\/tD58t03UnA34z1V5ynyUw2sPmHbfGPyUuHolOisPJWP8GFm38uwI3uLPV0CmogAfxqnVrAUtVcxdUg72xUGUY165dVypmfULlPc7RrkBhHBdrZ6CHX1G\\\\\\\/V6DcCxlpwRjBTGYKrWfjJ8Vjaar8uikC0D6M9BqY038qywSQ0JcVbOI+TcfXeWlUqp\\\\\\\/Jm6BsTUf0PRx6RaUr880Nh7m1TYuDnLF3GO\\\\\\\/61sh\\\\\\\/twrgTozHqqM7v4EtNfiD1W50fttW\\\\\\\/zCgUNlAFjqabRvIwGcX9tLDqDZcCUjN1bfjWwwvzXIekVu3UetzJK0DLb7Fsx34iq\\\\\\\/ZCz9tyoVR1zzhEsGMGM6xYGJWJfRy3GO9ZzYesuoomEHjwpNJNMTyJcctrx5KDs+tFn25rHbwISwc6PqLWtxTavKP1H1G+rGMjU21QSxIBqMLGP\\\\\\\/E8THxH5E+Ydh3BAE6HDn1PHkKLpkqnjgHtHyr9mxXDHZWdHYuitQIjIZmwFTmS6UOboHpoY+aUeepK4434RkeAvvZ2+TgZNXXrE5f6EuG8BN0EmJK54RDQiZFl8OOMKHFNaZvEYDa76Z4EhggxEhnwUVtWby9F2YQOgJ5r+kMPJ3xyh2reoqH7jHYqc+4x73PRDiGp0\\\\\\\/HW8WyZxuHX0HenGRItGAD38\\\\\\\/UrNTZ47d2XcVgLgDDZslSGOaC9UuCHpIGShNKAwwlQc7N6aCN6mbSWTggPEVylP4TM7tOsnH+xkd+a+io2jcUgZgd1ALRYaDHeZvrQJhbECYQtETqAmCYAoLEyjcdkrrTihrEKhjciDLfeNaGXsO2HdStrGvcg1zQgm1\\\\\\\/1kgg+XiX4eNJU8nCaDyJv5wPKqujn59OtLhM2d9Z\\\\\\\/qZ8vGfTnfNiR\\\\\\\/1KL6SMh2AipCOAP1DlO9inmnynz6U0IzNon2mGgq24DVRd5bEG\\\\\\\/vGFo5fRnf1N9gpnJQ2Ia8PMbbGmsgqczc9BVR1tGDDb42n83uT1CZg+TFCJy0vR7+nL+lAKz\\\\\\\/AFb6UiOw5DI86sjK+QJu52oeeAQaPSWDThzm4bFNr70Icil3OuQVJQ6TH6LBDSNGdzm3NJUWoASP9G2aCKpKdnLfuS3Yp1NDY9Jerilvx5AbmbF1ibodwDdAl2ViDwu76K+hpcZkD9N7+\\\\\\\/DSu05vFvdDTXWTj28L1z3OZG3s5fIBODx6smFgIl7gJmQP1MB9bDGAG6nv+yOtS8+4cj5eKnN2GZObgxEOBVnIhZ8Pv\\\\\\\/MzxLA\\\\\\\/OK4CiVt5oXiDS1k5W6zT+1Bf549i0JUnx8ByaFCEQqJkk+GcKZhM7C7rownsk1OibddYNrsZVbd1\\\\\\\/etL+G\\\\\\\/Gv2uqj3qNu8vMEs3OAEYE6tjCQ6UQZWZuWiNqHYCOkGo\\\\\\\/jqOiMCGUcY10\\\\\\\/IMdYtzWBgFckkG4vJQJm5quFkOFL2oFd7HPOWRFq6xnes4HzBWaOcATMrwqWtqc+ynwlXlBgcWwyQI6Ha12Ar2szVTjgs8nzoj8bFmXoTpDT1YUW7p9Pts5q7CszzINWr8I19v\\\\\\\/mv7GRgApqd9ljMGlMKhKlhTn9ARZSsMalRrsDolY5mCx6H4Z56ql7n6t2NYzKmySsZdBYoTmVBNHf5wgKCgK6kLmktLrkU0O1e4a+1JPXVVfnNsCs6qWJQbmuWdF9RYIAW4XGYOHmVpdSeTE01ratM8M7TC4vFbEHorXGEhEcMovXtBoeKvUL2pHxawtbO9rjB0uUUSsjxVQfJj1ecslUgQK+ALOg9RSs5t+edWHM97oFbP2GfIwgXdLhrCCq9YPoMZ3Q2CdjedRDnG7DGa+hoHYc\\\\\\\/IKGVloY6AIS0h42WvRrWc33HTFmhbWCsYm19FWYTPNcX5Hbh+nMS9pfCGw5XJOFCfRlouUeotFj2CNBR4VrfKZHeFcpbWyuEFZxQExX8LQKhMSWj1PHuwguHGDW7nv0QL4\\\\\\\/94prULoJ3MlJQg3FpPr8aFUwbtJtbnKaA0JlSKt0TQttbsUrijdWFlRDnSsIQUHzqq0Ad5X2ZEx0WyY2z5BqKBVxWUkrQlQSupbJmUe65tQ5yqW0t3vvprrb3GtmDw\\\\\\\/vLvSYVbCbAMGp5LDYyd2Z4xhhXixZPA9AuBJS\\\\\\\/E56n2Y\\\\\\\/VkqDgYhq9CmINo2aLIq3dg6dqDJztChiuO3a42nKqQWVZKYm0zJdwWEwjS5h+VqbKUKo8nEG\\\\\\\/tWvkX2tNhET9RzQkCV\\\\\\\/aznw7bnWfBdCqcD9lgGyYxZRegpwBdJWnBIlDbA8+dxoT568APj4q7X2kPVSICptRY966xn9WcwH+Ypqzm1lea7cI4rulB0J7y\\\\\\\/FewBXtaoqOdCzFcs5m68xHAptw4AGmummiK9WiAcCMytvIl8USIuN9tZ+tAWxn0JNQeZPxFqTWaMKHFmOv\\\\\\\/uOIUv+58hwunS2yHoR563UEye3WPQYxhfWCDgl48Jk7jV8LbrfbviDNPIVYBhWZ0+lliEH\\\\\\\/LEQVLGGZ3s3z9Q19twzhfAEGO7tGx8X7FdzM89ma2gSTXgXcPnLPLhxxD\\\\\\\/oepEd4vmJn8fvE3+jVNolIWG0VLhVcQTBItiQRT7edkvXR2UNGyJ3go2hUy291Z3QmolQC4er6qYPIuzDVEC6FZ7l2KU5XISTuz1UaV1mB4HXz3cLUq1cWFLrxpxMZRHJznIpRpyi6J6FjPnsry6tjwzswl3pB7pRd33gP3xFZmV7OIyxacCDZMF9RabMa78WmTuPm\\\\\\\/2XUZjv7cDSyrXgbWlK5tYvzkJqV3UJ7CbW\\\\\\\/PaIwVD0eNQbcKdq\\\\\\\/poiERRK2FioWCX3YPFJSGB1K97CJWzuAFG5YqH4R2DgjOOB9Bwua6dhLdmd8oEO7hQ8ujXcFaMiYtjsRRGl47zXeubGQMedMNFFpCsHAyH+j4O0LAUfqNhoZ3v3\\\\\\\/z5r4ORIta7v6MN+yTq1r0GzLca\\\\\\\/iEF5mzdYHK6aNKD36mPqRhZWp3DJhSMGZA027FKBxh2uy0i51rtXv7R\\\\\\\/XdZS3d\\\\\\\/HssdmtiNugqw7cDFCoMPh2rLGxNWBVSYemZVMOJtbb5w421jPR+uGjsVJwWTVlt+Ca3XyuhROoL73Mq+LKl9OKKuIehp75B9hfx875nxkXYwQBhqZk6x7w7jXaO90t+8OxqVHSNFjjJT4yHl4qR2ZJ0tyBRFEkKKKLX8IAVzDKzFwBQSmKVS6vpDLTyQIhA5Qg3UOCuPKy1CCaK4gI4EH4JBA0hp7a+TU89Ym0Zeu3jjU2CQkjqULWDjV3OTMIqMY\\\\\\\/p1HWN3iZm1SCeqO3IjAH4\\\\\\\/xvPAWEGr1FnfUBUCXdIXEvJIRhZ8Ep\\\\\\\/7rqqTSYNc\\\\\\\/WghxVXxIAlsqL0YIbfyMmlpMHmIeKgkDwXAAleqWtAh\\\\\\\/tyZOPpThyZ29JanrGpE2rqIglgjhydQqecy+w2ZrmfjWmnzPRnrwmMpVQ3I+FTggKJWpAUZgBnvnpwwkb1o6kXAQJTIenKxlo3LNwDMrzfwu\\\\\\\/VgJInI0evi0QhF48tEqUTOK6diRk\\\\\\\/OZsvxjOur+MYgS0daGvESJusP2Rx+G3O07YC5tY2z66bXUV9Qov7lVaMyMer5\\\\\\\/gvlsJ80R+sB\\\\\\\/N2OAH4rwU9IXAgDDOAgR16DZcpqVS7tbxpHy3vmf8FLs3KMB1I+ZLj6akfTpFAMd\\\\\\\/K4h7gLAmfoZnQWfALFZNUPZOzJYe1s1hPLrMAVe6ePgS53YJJfSR7Xe0fOnISx0UWGtAJUPv8pVJwP+DaUWjO6j80d+GPbHc+TVspJm5Dvy3vU8pKzLHGsGgF9ia3wjG0se0UAdTHgzwxEHDuukmoTXwCbFwU5smzOFFwQSRJKxefvUiW2TdCeS6G8OBMF8BRhTRhzxcJT8JVoeLL0TwvmtUEwz1fNiTGd9oUs66l4\\\\\\\/h7hg9NbY7fP\\\\\\\/PNcDhpWK3q3IuVh4YgiU2ghHAkJUYFOf1eoiALZx5cJrDpHT++L3kuHpcQMxEKBWDZQD3PHxY3Hm+zW99rWozgBVSsBZsMT2r2CozYrdX37hQVxy6SMzg155gWYHuSW+FnBvG\\\\\\\/CyIJmHjL7SFyXtcjpwRQXFHG4VVF4wKYL7fs19NOjQYBrb1dGAYEnkkWyb0sNxBPJ57YZFR+y526FAu2K\\\\\\\/KY4CptkDKTxwX0DNnfKFLZSS7poT1IcSpVkfxEJ\\\\\\\/r4K1V2wta060EJEVxKkkM9uHX8fLGcWS5OOTyalP2Cnyyb0+F21S2kCw3W8kJ2Z8lHkXl9kn6OcoZr+APJDuO\\\\\\\/KNSX9nf3n\\\\\\\/vRzfHIOsI7vbK4soWmjwTdsvMPIZZtBYBsIM0rGdDGmaiO9bP0Mp7yuLH9\\\\\\\/Vfyaz5gUpoIeeSeYa5ocpBDziKGcrMfsylVD8bQi6WAIp871PZphoYzXReKopkFhe9q6ixOGOoAX1rvjW7gpzd9qNNU5eLonLo873j5HBqWDjzuuGHmvG2Dcr\\\\\\\/DTN6RpeoUS1Z+ZbbEiGp\\\\\\\/2fF42wgur0ygSAR4M2EFr6FulKt6HG3pDGgkrE\\\\\\\/lObelKRkl3AKoOGLQUsX\\\\\\\/uuvV7ly\\\\\\\/6IeF5Ts3zw3XrG4MfpXSYh2FQ\\\\\\\/xkMf2+z6g3XBAZX8HTyps\\\\\\\/65OAiqNev\\\\\\\/Je4G9KFgbvJvkkYzuwIzsZRZUrHsxWyd52wD1ssfV0eGfESvqS6MweV90CZ3iCCY+YwmYGrivzgVkQ0eHMPa3YczZwKNedkivgXVooY0JEue4UVfu2TAG8hurSEJvMCFMnIndYga7ZG3ArF49+oyrRFxjq5Zc+tywVEH5YlYS7U2N+5LTuY7Yv7tRO0R4frukzO8F4uk7oCrqM4jvndy3uwmuJX7Rgasz8hz+dMWhP\\\\\\\/7ez3fsZ1YqOT\\\\\\\/D8VS\\\\\\\/OPNbff6QKhjWIQTd6FI3zaMbbPqeFdE9iXO7V0l60WJFzrlEibuPMz8hxdGKFQJvxI+04srZI\\\\\\\/Qnyk5RJxALO+BSN0g\\\\\\\/AsC7THUtV\\\\\\\/nTvRGdK6XmYMG9aTYepfJ4mvfVpsJlhzSAiKbxQu2itfBRNAC2t6yLOSO+qQdEZRaQrKWdpXT\\\\\\\/ZPiWqQAbqXvygbh1aPnk8pJ\\\\\\\/dQIlNjelAPhJEeyPyHixzhoSiONmtmh\\\\\\\/NQGIa9DmZknulgHiVdOIhzOC3VhLL3sGPSzmRjLItZuXRrjLD4b5ILnud653\\\\\\\/YV9i86\\\\\\\/crz4fdG\\\\\\\/BUdutRjasXJdl9hTKm0eTLwoLkFiQmIjarkp9fEXXuh4IDl9diGbPQ79ilxILwWFaKleXQRIuFxd6QlO1\\\\\\\/DpnclU1fGhZzVO\\\\\\\/vfdFbI75lqIo\\\\\\\/1dCSVktl16M4zyQCN2lANxvi9SPaayKBmQuoz6gNVeR5AfmDs\\\\\\\/2xxUFae4U4\\\\\\\/dvbupUFpANV1ZyC+3uZRCnHNqNrV2aSRO13rZ561ESFFOUF2E0ppnFwWlRqQYJbh8qPguVR6ND2npN0pyPGRVDT6Si5B+WF8Ttci7LcFsnMfTRMUziMrGB32hX5bccK36ldS4uzE0BQvKyDwns\\\\\\\/+atdBG36LpvPluCownpudETML7AqxdJW3QOYZSPvFGpj6Aq0AedtrAtKhbffBIrEXI6Jr3IH4r2D827wNPOWdDOibAYJevy9Z7OTJM3r4UkbgF7UGO\\\\\\\/jDe3r1lr4UK3X11LD6\\\\\\\/H1w8+8PLb9O5b4ZDEMHrEM\\\\\\\/S7bvcEPLPPiW+UHGNF0BmIwo+k+5pveLCkdAFFMzscv7cBlDxJDzMC\\\\\\\/d7jZNRIl4dBT5ZjktVHOzWhbCU\\\\\\\/5x+1tVbZSfzxNRHNVSKJzJhxcbEAUayMi9LK3ThcjeXoIlDixLD2GF7FGTl2M3cR8KqtmV7AplxXRyctgDPxTOIXcZfeqeT5s8flIhoVFK62UVEt+JEMRGq+XAzs\\\\\\\/03DCpPdoBn467Kc1O+gtR5jlLm4pLRH\\\\\\\/HTrTazvgLt1VUwgknnHDMdGZebzplu5Hicpi2mZ4WHdbrPO7GGznPtqtY5uOkCZ3cyrxx6t3h\\\\\\\/4PAcIztbaYFVsDWMsj1DXCUhe6A\\\\\\\/nkKXuVV1xiMo5GBt8URf+lnvjmYrVCX9kjvxjhnWazRRTcIRZUgLMjZm1f3qCRupqYt4dTVG+6ONyKl26JfB3tDPmzgtCy+obrbO8WR0L+8DNWzj3pgdJlQ923E04PXgo5A6AwQZnOZV+c2QjMPBlvfj7JnvTBaRsk52Fzo1O3HqcYa56UJVCN99+6Z32dCGDj+DPu0SRRIa2xrkh2un0ku0IXOPDGa5yDYaoW3WD0Jn0chIfvYO3FpEUMo6+lBw3B8Tsf8YHyU3qalrgrkbG7GBkaiQ0CxutXj+Uk0QaUQLYxeDnvjnaIt4ANlU8CQu5upt8EvvALl4SI+4cI23d6GGz3oPe2amKFZjoPDBKVQhlTtYk87mvMuDC8uoOCI3y\\\\\\\/fK7D6H77iCgRN20QzGG\\\\\\\/8ZGlkbbSDR\\\\\\\/FtXDUQLejFI5Vi1x8jbUHA4h2t2d3zz0rAf+javaLyCKl5I3Gi5rfkwsHB540DzeTeSwZ4Dhk+1TRsU896Prxl9cvn7j7c+mHqgQ+P+Qv\\\\\\\/TVGoFeQiSMa1eKvFJl\\\\\\\/i+BKB5ThR\\\\\\\/u7kvjVbMzt\\\\\\\/X2z4arK+BR9YwrTC6Fer0rHRADb8aw+rWzPXXXKbc23MhcN06vdWq5sHTjzs2g5HrCgJDyywQJu\\\\\\\/2OJXpEygBFhBBy7BYTbveGLftImRLQ3nlT6a7fkea1PrCCLl9mjIUCGLdC7POChMV0xaR7rL3Fj+64Ru\\\\\\\/mxIh7KMQtguNzJczYyUkOqFjUu5dhfYmdO16NP\\\\\\\/0eGl9+0ClqkXbu3aEiHi4zrGpXEKaOlTyHRWAXJwWFdmAviyc172VKZBHtXm0sONAoAh\\\\\\\/\\\\\\\/sEXCGDA0Td\\\\\\\/i9si1MuCNmKNH6hQt4pH4JwsIHLILkkC5IiCQLv0lNGRxxnSzsTmeSgwDC2Dk0yQb3DPjnUjU+TGiuJh2N6723RrSYSQCVOi0gnBfZdC+eY9EaG\\\\\\\/vTwRwpBLBUlrISwf3cqcZCZJxMBnTdtR2YPr5QerizBOHOkwexn2NSKohdV\\\\\\\/PwQD7aX5l8LGR0tj3FAhtX1MFGGFTwz92jQq9XPlbZc\\\\\\\/zd4Ca3bFJE3MkHOX+vFIRYQVEedwZzrNa6yK4miIMf1MDofybNauxDqhjpYjuaAsLrCS9zfEMV8zF+tOAeUDGSA3YiQbgvmzcwYEiOwKHMRfQVjpwVnoQBIFKAm8FWNtiLxJNRPdhg9M9lMVJ7tV0rQ7sj+Nkr7a6VqLx3VGAYwz7xf9qrpbda9SdroYSSSZ4EbEzE9roPg3uYikhajWRhduXvUXz325rGn2vxFzsf0ZKuSg1T0GlJATGJoQj13XyOKclOTI3kv4fJDJazEApSHMH+dswNsWnt\\\\\\\/CrQyY7Th2WlvPC96Vbs0IVqiH4xXkvnvKR8NeQ7Xy3ejG55gNBQEAI7E802vlIb6eIZwNZnyLCXg\\\\\\\/iZsgFQZODRVs5rUiYmy63Yx6UelqZfJeC6rgIkm9QyBTGQCvHOlHq4B+pBtuxTcekJVTzKnnLjByBlZ7czGWSnpdguCgWSkPfSvQTD9BgNB1ScpV6bO4X15NiCZg2zm9BBWmWC1FBDiq4Rg0kmyjTjkYRkPjmKZYrGix3cUvvwobdn5AonsBrXK5RrKXl4mn8o\\\\\\\/aZILa0DysbIENdKGXJA+nWSKAD8aFQ\\\\\\\/mBu+swxzgXVfxfQYCnODuvu7FQpFdZso1OAtUyIqt7ymm1UGluZ3VSvJidE6FU3hAhhWQyv8eTacBoaGh819L4S9Hnx5QpN7Fi6+iuQ2UT+df8+8\\\\\\\/kKRCicyFb9SnmGNn6SpjhotqAocvLtebkQhldrYseULy1GKJVenj+wwddSI72TsMdVS6sPnTLDilkUxkI7DLMKBCUwtYirFgq2ktkOUj+2K7uu\\\\\\\/OvTVVC5CUmwY9DnYv0innMlwqkdmvslx5nUhDZtghcM6PGQQsLnk7B9sbnPPXmoVapvb+aaPSu9WCAVaGYyac2B+3EFRHJiRiGEZ7fzoRRZXePIQ82tNB10wyljibfBNxHTvNfF\\\\\\\/R7vHgzPHCSZ8HaXBB5lsKFrYlQX0s1bbxS7SyLoFwkhk7LpBTH2bvtXzxRCCU21Ai372rl5\\\\\\\/9RbZnelAnUHperibMR73RTAquGRxVAlEe2aaRyXVbZxy1mumWjel6p4HU9ohc69xGN6NFDNODAjHWzV3PTeWy24sIDh\\\\\\\/zoiiYXgU2RtEAJOqjzl9YFwgaLbPedi3OoOR0En7V4WlMjK6nkwN9X8Wcz9MTI1HaUF\\\\\\\/jApiQ+6UYGDM\\\\\\\/7gbgwZSetAk+hJRna6cdDHShfU5qvlXixuE41bytjv9n+gpcFG\\\\\\\/4iijViExXjSbVarrioQpE6B1DAJfq37MtKetIwobTmG9wc+OFQTQnyr\\\\\\\/+d7gDLvR1n5gElSBSBpQ9J2RnOf6WrdxwQAwtvNdF83o4EdUh7f5HOjRPheke9043Or1H6S43UkiBg7n7gSNAtcAzAlMQY9Ray\\\\\\\/usjnFwnN3lKQo50PuCG0KTPrkbRIfGgdIUJ3nefdnNosxB2PD7y7oks1C2qmIXfiHrL4sunfG+PCsd9mkCu\\\\\\\/VAkJkrT5d5PX4etQ8Rvc5BM8HZA++vX5vLNmKyC7jZpImpn8WzEa\\\\\\\/gsBoYHm9ZwheCG5+slkysmluIvgpWriGl2BBSoWibM93kvGG5qUFvLiwyyr6\\\\\\\/qbTkCYayWX2shczXpnQw+q4hlXeB8oqNLQAVoooI1JaMccBsHB4X8kFaTFqpV++hduFP0OJz2N4fpRdc8zgNhIG2\\\\\\\/FYjwA22EmKvQsfYeUI7UZVql1H57olI45E6L+0FRRLHtld4MZY4nMPqDQmFV8\\\\\\\/jUCnvgw7NTMTT5bI1+so5TBnozwImJpXoanG1oFIKzabWXjEvYeRvsd0\\\\\\\/cHxDH34o2HmTRC6w25O7tGUN4Dr4qU7PMHaD+0w4\\\\\\\/ctaGZZ7aPbgcNzN6rUDpnleC4\\\\\\\/eKJyr5nqEtqmjiiL7Fos81tubTv2huNrQ1axzG\\\\\\\/s\\\\\\\/GkegkHXdbyL1LxFmFLdUOA0Cr7vNtvg9XSC+2r4Gx0yzzOJbdrt\\\\\\\/JZBp9Fufrlrx0sR6us\\\\\\\/mVM9jXLxOU+dvXiTm\\\\\\\/sq3SdVnqz2PIqilIHWEDwnqjjELPsTaPjCqV6CY1fM4TmjXE02lyKNVPUTXp10wVmaBhzZhnqRnasx7Jym9X8P88fPVdCxudDjJvUID0JPYfrP6pxcZVg\\\\\\\/Ky6eduAivR9DA3pIbiPX7cisQNAvN5oQ6q1WyOmOy12v6AZjCnvvh4MdiF95vZS0eGiBcg7vSwPtmphYaif7ZKnt77XnUHALCJGpW0aRYQEsqYPzK8KQtOjCJq2T63TJLeIdOVxPjglNNcXxAQ7fzyA8ZEMdjLoHjb2KqYIaljLl9xq3osOExOjipcu0gwhQoY3Z7gw1fWhbcmKwRhHjAbYqaYCV5zTCAfRFJzxo2UP0kXBaUvz\\\\\\\/o6bzxoMGpzITryNTw85EyJAwnaEuhAVWueD\\\\\\\/Xbu7vOayBSjefaEPpuQ4LVhopC5XVMcEnawrWjv19wXvxL4ha6J+x3DvgPysvzVmfoT1mzCeKKUzDA1pld1886IkZfuTLJpdHa12Qz3PCcz5xlBocafoCfVcnflFYPzJJyxvhwhlXhV22YhLN9cWhukgFMjOb8\\\\\\\/mdotBlgL\\\\\\\/ftwyOJnGAew+6GMvGQUPj1AYUgznx6QWWl3\\\\\\\/cCO4loJjaYEl5f\\\\\\\/8YKlqYN8aeZqFHVG4ZO51CbeLxG3TAQHmA\\\\\\\/\\\\\\\/Y0gimfopj\\\\\\\/ezf\\\\\\\/Q1+3aUHlQqsww+Pv2wwvIbhILeglElY1q+DyOlmXXOHqlYnVG1LRl\\\\\\\/\\\\\\\/8KrxdAb92\\\\\\\/biFrIjDY1fBxBp83yM6xtt8UefEa6qhE4x52cJlcNWwhrAGdjjbcP0rlFU9pYsz2rJolYd8ovPV5Yc9zzdM0IBoL0FjZgQq645LCvw2PkBtNhkk5Mul13jS0kw\\\\\\\/RmRD5kAtRS9ej1cuQERN4kn2hIwMUpzVUmNSMVCnwt4rBkWja+\\\\\\\/MjPAGE+UfcaV7QraSMJHE5nQsxf5xj8qUl\\\\\\\/APianXUUMtDcotsnoQLjD+MjvNdHG5DpC1DD06+jiH7CKwuETbWXwjxrgMN\\\\\\\/6nQqJJO6CzMZf1mYW1hF7V+EnDEA2G3yAD0UVb6ta1GV8PU3M6bfeCUO\\\\\\\/3uX8Rshk0YcplFsq8yWui6ZLnhTtLWnJnRbNYqeOHlzwUQlh99VO34UaTve7lidffR5ez7pjlpuDQJgIpSTVLvRKP+ZNxmLy2g+zWtkJ6TvLK5p3AVcDYBVr3UsZVBCRjTXe3JFNcoh8EbJRsECjSaoh7HS0O9tGhxqvoRgsaXLaQolrUbBXtPZhbtw3teVfeJklrn7EUPh1OizJayuhZ1js\\\\\\\/ZCMG7Efl4JsKX9SEgEHot0JRiDg996+UyG5RH6upgxhxS3NbwjW0JOocBDrf3QbKpQCWqQVnoW0FCXdiS7SNfoN7mt0GlTc31vlyiCBeSDT2zoyQIKo8zx5nZ56+dNZe82b+QMuR4Z+R94qNOw+12cQGRjHpTrWlCG7mNQmIPRHTTubRanZvlEdWNH5jXqKK1c7NB00GpYD0ooD38LEW\\\\\\\/442eja5DZGnDsjDdJztvQ7bP8aDGQ6cc9FyGmhj6kROsC+VKvMkV\\\\\\\/LDyL4hv7Dl\\\\\\\/OT1ycAjXQDcGVLh2sRiXIn+CHnY3h0SvMe3kjIeUfAqy6jSqw9RvfIDEPdKdhejya2olX5P8GuMJuTegnOmiWEss9XUtbN6gnsHNPWY3Q4aEp7hhh5pasT7JgCfzhSirczT3u2ysjCg5bbjcW8S8V0p1NvyLvn2NFMOPoHEpeiLpMAeOtjIqy91eVf8m\\\\\\\/4LxwAVJ7OQm0+qk\\\\\\\/0NHZkb+vDHTs2E3w5HQy3h7OY0V9zRgyyiq0HY8TVtgr\\\\\\\/9IQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022fMRfPpynBdJsOZO0LAmm9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224b75a0192f48b68dcd64f8c777743cf52697a8188521a27ca58aae00102898e6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r7Wg03uWNJLEFioxyAGtdqhKdgITrq+38BzUtyy4Jg767qB1b+ojt28Ni7eC+yWrqquMIsdSP9xcfpqWy5zsks38Wuu9Cv6RomWxAseFi\\\\\\\/YNEYXWhQydRz+P8y6cT\\\\\\\/TxCIZ83logBnD5CkFm4Y+oEY\\\\\\\/f6jPsxcnRsQS4yFlwTgLX+NTjgNbErR6r7z93cONzhPW3AlSh42h2jQUbBzYoxhJlWB8nkFHNBh1L9w8m13pm33Mq74un2OMaaQGkdhSY53YGtkoJV\\\\\\\/Lwm8+VLrX0DNvsyn6XCPw8scmIOjkAU6VCRUa3BqFmX3ZdaWBGfnZtjmD7+5Y2pB4Wy2TRtdZkiU2Zk4IfXT29AGBKO8532cudOKPlVEOvGQqy9LmVgNVjQlAArVZPmjxt77PubxK8ixQ53NsvK0wfQDBoLJ6LH2zW5Uk8ARUNRD2NanhhniTw5qk\\\\\\\/6q+UTBazW7WtQwL02UyhbhFQnW+4F+6jaaYw9mDxQKhMjvlm+8SiCMAeRmTGHASCOoxTPJDl5rSICJgQi0J6Uk0H2Qq7QJiRwrR8XqWKhRIzGmrnpmY+kg2WtT7AyRvCPdxwt7raxXUQ3Mmg1uipko\\\\\\\/BzYy7KLleiO0L7ifrUZvK2K2UHcgN\\\\\\\/xELZZbdsvD7VzIL+TXwnA7FOPi7JA87y+9WSJxZUooPTW9HOOPnrHVl0oGdqXkUlKTGFh9mnICsiSUMPfsk0rqxd\\\\\\\/lVNPKw8VLRLVUoAZ4JQft+FZgY25Ux1eGlCH8oRdhJFmg2+596eNpfHTz\\\\\\\/0YGTLm84knupVy9CbuIO9dLpwkjkeYvW5B9kQXOjwq1\\\\\\\/ywLQZnZ4Fras9OXunRE94F4ha1Cz0r5a+MueYqq4eNhMzFS1iwjNsQKrDtlYU4OqxkMhhSHQwNS\\\\\\\/sP4iSnr6LHECldjVf6Td6q3L\\\\\\\/64iMv1cLLbWha28gk2bAAeCk7cIHMyzzVFp0XxbSyp47UWfu7Wvpdt+\\\\\\\/4w6kkJb8pLJbkcqpM+moqF2NMDyvwxaauhshqRK90pN30Kb2UN9lrXG3azlpCWtcu9L0RLcEXz5pI+aNurtJa8qtXUwELOqPuEi\\\\\\\/Zr0g2DYBxE7\\\\\\\/yLLRPQ9PeCrkrSHiqgqPy9XgSIDvOwZS+DuCAEGGhBiePv0KKMdLEmuYHXGwAtnSLMjHV+sWM96nyQvzvwdpa5r6O\\\\\\\/idlfzXSaTJFzw5eLYHyeemW0M6sGtz4jKZIwydB+XMLTQL+hbu4CRckiFuWeXIc\\\\\\\/FlAog8irM9rhWVZAqm\\\\\\\/IznVlShDkpfIVZAEqiT20KS\\\\\\\/YhTJWvY8FZVoc9Hnd+XhzSFsO1ch6lXjfJ0QIqMjR5hRBHTe5tExgFaumW0SY2yk9bJXDVOE6CMyZ6GKKsy5FxUJNCpnkcQR+EKWVfbxZu354yffRx8RfggU4JLm79aX0BZHnsHdQ8HTQrb51\\\\\\\/3tT3xyrCrmIwl87shqU+XnNvwe9QvVzBABSaKDEreh5\\\\\\\/aNSg5HzLrgcNzlz7LkJLoe1z2t8A5yim+7IO16zzkOGyh4yizH\\\\\\\/1\\\\\\\/MRgWz7GFwx2RcfGZVyz35qLsKfTYYmyGZPyOdJtlGGplz\\\\\\\/tMLopMDfcO5waCHAvIZzMm3hMV8lYWN0E8RZcNvYWa75RcYR7bebUuNLPyq8ZvbWX5RfIKTl7IiG5CrvGafvaCfBxRTR576HApQYwOq9by4YrYC5yfJqr0XiL7oEWhO9eSdcm7XkPFewgcU2mcs8bEsDeT\\\\\\\/B4uelW8Lc4JPV\\\\\\\/Y3riOxd575MtKO1EDQztwJJuqSaLopMuCxS78NNZNqX1RXCMRUDZh7Q6ukTO2v\\\\\\\/hT\\\\\\\/u4RzqlQuyX7BPzjb9781iVktjkB8w+ALBBcibMffi35Xrbzhi8eYLiNda5X53lDjyK4z4u9tB2xg0unsPVv84L7Ky4cymiVpXKUzcZfLfLqfaIex\\\\\\\/H4LQ45swiqo0ToOBdMs6yoppyu3k3jO9CWZuMwwUqYlkkTBL+XVD+qsymh1v0nQN0fTtqsAzTtPgTXg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022L1WOT6fJncUNG31EwOqG\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022126f9bea1f969d355b695286c15bf2eb196f1b684a5e80cf3126f655b703823e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459266 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/487" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--7qNGjOOazIbLoDskKcE18DmM2EsF5mY9p_7kaMq3-I" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/487" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RU6pIZYMLeG8Wo_lJgMMl9YX8PtfgtOBAtrdzsIdZh8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/4/483" title="Contrasting Mutation Rates from Specific-Locus and Long-Term Mutation-Accumulation Procedures" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/4/499" title="Hypopigmentation and Maternal-Zygotic Embryonic Lethality Caused by a Hypomorphic Mbtps1 Mutation in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459266" id="node459266" data-pisa="ggg;2/4/487" data-pisa-master="ggg;2/4/487" data-apath="/ggg/2/4/487.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in <em>Arabidopsis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yukio Kurihara</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Schmitz</span>, <span class="highwire-citation-author" data-delta="2">Joseph R. Nery</span>, <span class="highwire-citation-author" data-delta="3">Matthew D. Schultz</span>, <span class="highwire-citation-author" data-delta="4">Emiko Okubo-Kurihara</span>, <span class="highwire-citation-author" data-delta="5">Taeko Morosawa</span>, <span class="highwire-citation-author" data-delta="6">Maho Tanaka</span>, <span class="highwire-citation-author" data-delta="7">Tetsuro Toyoda</span>, <span class="highwire-citation-author" data-delta="8">Motoaki Seki</span> and <span class="highwire-citation-author" data-delta="9">Joseph R. Ecker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">487-498;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001362 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459266" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yukio Kurihara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yukio%2BKurihara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurihara%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYukio%2BKurihara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Schmitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BSchmitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmitz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BSchmitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joseph R. Nery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BNery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nery%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BR.%2BNery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew D. Schultz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BD.%2BSchultz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schultz%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BD.%2BSchultz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Emiko Okubo-Kurihara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emiko%2BOkubo-Kurihara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Okubo-Kurihara%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmiko%2BOkubo-Kurihara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Taeko Morosawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taeko%2BMorosawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morosawa%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaeko%2BMorosawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Maho Tanaka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maho%2BTanaka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanaka%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaho%2BTanaka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Tetsuro Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>RIKEN Plant Science Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tetsuro%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATetsuro%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Motoaki Seki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Motoaki%2BSeki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seki%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMotoaki%2BSeki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Joseph R. Ecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Bioinformatics and Systems Engineering Division, RIKEN Yokohama Institute, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BEcker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ecker%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BR.%2BEcker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecker@salk.edu" class="" data-icon-position="" data-hide-link-title="0">ecker@salk.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/4/487" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459266" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459266/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/487.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459266" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459266/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/487.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459266" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459266/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/487.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459266" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459266/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/4/487.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459266" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459266/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Eukaryotes possess several RNA surveillance mechanisms that prevent undesirable aberrant RNAs from accumulating. <em>Arabidopsis</em> XRN2, XRN3, and XRN4 are three orthologs of the yeast 5′-to-3′ exoribonuclease, Rat1/Xrn2, that function in multiple RNA decay pathways. XRN activity is maintained by FIERY1 (FRY1), which converts the XRN inhibitor, adenosine 3′, 5′-bisphosphate (PAP), into 5′AMP. To identify the roles of XRNs and FRY1 in suppression of non-coding RNAs, strand-specific genome-wide tiling arrays and deep strand-specific RNA-Seq analyses were carried out in <em>fry1</em> and <em>xrn</em> single and double mutants. In <em>fry1-6</em>, about 2000 new transcripts were identified that extended the 3′ end of specific mRNAs; many of these were also observed in genotypes that possess the <em>xrn3-3</em> mutation, a partial loss-of-function allele. Mutations in <em>XRN2</em> and <em>XRN4</em> in combination with <em>xrn3-3</em> revealed only a minor effect on 3′ extensions, indicating that these genes may be partially redundant with <em>XRN3</em>. We also observed the accumulation of 3′ remnants of many DCL1-processed microRNA (miRNA) precursors in <em>fry1-6</em> and <em>xrn3-3</em>. These findings suggest that XRN3, in combination with FRY1, is required to prevent the accumulation of 3′ extensions that arise from thousands of mRNA and miRNA precursor transcripts.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fry1" class="hw-term hw-article-keyword hw-article-keyword-fry1" rel="nofollow">FRY1</a></li><li class="kwd"><a href="/keyword/xrn3" class="hw-term hw-article-keyword hw-article-keyword-xrn3" rel="nofollow">XRN3</a></li><li class="kwd"><a href="/keyword/genome-wide" class="hw-term hw-article-keyword hw-article-keyword-genome-wide" rel="nofollow">genome-wide</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/arabidopsis" class="hw-term hw-article-keyword hw-article-keyword-arabidopsis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Arabidopsis</em></a></li><li class="kwd"><a href="/keyword/methylation" class="hw-term hw-article-keyword hw-article-keyword-methylation" rel="nofollow">methylation</a></li></ul><p id="p-5">During developmental transitions and environmental fluctuations, unnecessary RNAs are destined for degradation, and new functional RNAs are transcribed. Several RNA turnover pathways are involved in controlling gene expression profiles, which is important to maintain normal cells and tissues (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Houseley and Tollervey 2009</a>). Generally, mRNA decay starts by deadenylation of the 3′ poly(A) tail and then by decapping of the 5′ cap structure, followed by degradation in the 5′-to-3′ and/or 3′-to-5′ directions by exoribonucleases (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chiba and Green 2009</a>).</p><p id="p-6">In yeast, cytoplasmic Xrn1 and nuclear Rat1 are particularly prominent 5′-to-3′ exoribonucleases. Xrn1 catalyzes 5′-to-3′ mRNA degradation in multiple decay pathways, whereas Rat1 functions in the processing of ribosomal RNAs (rRNA) and small nucleolar RNAs (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Garneau <em>et al.</em> 2007</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Houseley and Tollervey 2009</a>). The <em>Arabidopsis thaliana</em> genome contains three <em>XRN</em> genes named XRN2, XRN3, and XRN4, which are structurally similar to Rat1 in yeast (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kastenmayer and Green 2000</a>). XRN2 and XRN3 are localized in the nucleus, whereas XRN4 is localized in the cytoplasm. XRN4 not only acts as an mRNA-degrading enzyme similar to the yeast Xrn1 enzyme but also acts to degrade the 3′ products that result from microRNA (miRNA)-mediated cleavage of target mRNAs (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Souret <em>et al.</em> 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gregory <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rymarquis <em>et al.</em> 2011</a>). XRN4, also referred to as ETHYLENE INSENSITIVE 5 (EIN5), is required for proper ethylene signaling. It functions by directly or indirectly promoting the degradation of mRNAs of two F-box proteins that mediate protein degradation of ETHYLENE INSENSITIVE3 (EIN3), a transcription factor that elicits the ethylene response (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Roman <em>et al.</em> 1995</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Olmedo <em>et al.</em> 2006</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Gregory <em>et al.</em> 2008</a>). XRN2 is required for primary cleavage of pre-ribosomal RNAs and redundantly acts with XRN3 in pre-ribosomal RNA processing (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zakrzewska-Placzek <em>et al.</em> 2010</a>). In addition to the respective functions of each family member, all XRN proteins redundantly act as endogenous RNA silencing suppressors, probably through eliminating the free 5′ ends of single-stranded RNA templates that can be recognized by RNA-dependent RNA polymerases (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gazzani <em>et al.</em> 2004</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>). Although XRN3 has limited roles in cleavage of pre-ribosomal RNAs, its primary role in RNA processing has yet to be determined.</p><p id="p-7"><em>FIERY1</em> (<em>FRY1</em>), which is an ortholog of <em>HAL2</em> from yeast, was first identified as a negative regulator of gene expression during stress responses (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Xiong <em>et al.</em> 2001</a>). This gene family encodes a 3′(2′),5′-bisphosphate nucleotidase that catalyzes 3′-phosphoadenosine 5′-phosphate (PAP), a product of sulfur assimilation, into 5′AMP and Pi (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dichtl <em>et al.</em> 1997</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gil-Mascarell <em>et al.</em> 1999</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>). FRY1 was identified as an endogenous RNA silencing suppressor similar to the XRN gene family, because PAP is a strong inhibitor of XRN enzymatic activity (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>). Therefore, repression of FRY1 activity leads to dysfunction of all XRN proteins through PAP overaccumulation. This effect also causes accumulation of looped RNA molecules derived from miR164b and miR168a precursors in <em>fry1-6</em> as well as slight accumulation in <em>xrn2xrn3</em> double mutants (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>). Moreover, <em>fry1</em> mutants show severe developmental defects, such as altered root architecture, reduced growth, late flowering, and an ethylene-insensitive phenotype likely due to inhibition of XRN4/EIN5 activity (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kim <em>et al.</em> 2009</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Olmedo <em>et al.</em> 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen and Xiong 2010</a>). <em>fry1</em> mutants also exhibit drought resistance, which can be mimicked by the <em>xrn2xrn3xrn4</em> triple mutant (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hirsch <em>et al.</em> 2011</a>).</p><p id="p-8">Several recent reports revealed that numerous long non-coding RNAs, including intergenic and antisense transcripts, are abundant in the transcriptomes of many organisms, including <em>Arabidopsis thaliana</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yamada <em>et al.</em> 2003</a>; Luica and Dean 2011). Some of these transcripts possess important developmental functions through gene regulation by way of chromatin modifications. For example, a non-coding RNA arises from the antisense strand of <em>FLOWERING LOCUS C</em> (<em>FLC</em>), a major repressor of the floral transition (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2010</a>). This antisense transcript uses two proximal and distal polyadenylation sites that are controlled by two RNA binding proteins, FCA and FPA, which in turn promote polyadenylation specifically at the proximal site (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liu <em>et al.</em> 2007</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hornyik <em>et al.</em> 2010</a>). The antisense transcript that is adenylated at the proximal site triggers histone 3 lysine 4 demethylation and transcriptional deactivation of <em>FLC</em>, thereby inducing the transition to floral development.</p><p id="p-9">However, many non-coding RNAs can be recognized by RNA surveillance mechanisms and eliminated from cells (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chekanova <em>et al.</em> 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kurihara <em>et al.</em> 2009</a>). One such RNA surveillance mechanism is nonsense-mediated decay (NMD), which fundamentally eliminates aberrant mRNAs with premature termination codons or relatively long 3′ UTRs (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Maquat 2004</a>). <em>UP-FRAMESHIFT</em> (<em>UPF</em>) genes (<em>UPF1</em>, <em>UPF2</em>, and <em>UPF3</em>) are essential for normal NMD and are evolutionarily conserved in eukaryotic organisms (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Behm-Ansmant <em>et al.</em> 2007</a>). Previous reports using genome-wide tiling arrays showed that many of the mRNA-like non-coding RNAs, including antisense transcripts, overaccumulate in <em>Arabidopsis upf1</em> and <em>upf3</em> knockdown mutants. This is likely due to the long 3′ UTRs that many of these mRNA-like non-coding RNAs possess downstream of short ORFs, which do not encode proteins and can act as a trigger for NMD (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kurihara <em>et al.</em> 2009</a>). These results also reveal that NMD eliminates non-coding RNAs as well as aberrant mRNAs.</p><p id="p-10">The exosome, a 3′-to-5′ exribonuclease complex, also plays a principal role in eliminating non-coding RNAs. Previous genome-wide tiling array analysis using inducible RNAi mutants of <em>RRP4</em> and <em>RRP41</em>, genes that encode for components of the exosome core, detected not only accumulation of hundreds of mRNAs, 5′ remnants of miRNA precursors, and several classes of structural RNAs (such as small nuclear RNAs, small nucleolar RNAs, and transfer RNAs) but also accumulation of large classes of uncharacterized non-coding RNAs (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chekanova <em>et al.</em> 2007</a>). Many of these RNAs are transcribed from repetitive elements and siRNA-generating loci of which genomic DNA is often highly methylated, indicating a close relationship between exosome-mediated RNA decay and DNA methylation via siRNAs. The other non-coding RNAs that accumulate in <em>rrp4</em> and <em>rrp41</em> RNAi mutants include aberrant transcripts that originate from the 5′ ends (first exon) of protein-coding genes. Thus, the exosome acts as a quality control of several kinds of non-coding RNAs. Studies of the exosome and NMD have uncovered hidden layers of eukaryotic transcriptomes that comprise many classes of non-coding RNAs as well as canonical transcripts.</p><p id="p-11">As described above, elimination of non-coding RNAs is often an essential mechanism for maintenance of gene expression. Here, we investigated the roles of the FRY1- and XRN-mediated non-coding RNA regulation using both whole-genome tiling arrays and next-generation RNA-sequencing (RNA-Seq) methods. In <em>fry1</em>, we detected the accumulation of several thousand non-coding transcripts that mapped to the 3′ ends of genes. We also identified accumulation of many of these same transcripts in <em>xrn3</em>, but we did not detect significantly more accumulation in <em>xrn2xrn3</em> and <em>xrn3xrn4</em> double mutants than in <em>xrn3</em>, indicating that the main activity of XRN3 is to eliminate these non-coding transcripts. Additionally, we detected accumulation of the 3′ remnants of miRNA precursors in <em>fry1</em> and <em>xrn3</em>. Therefore, we suggest that FRY1 and XRN3 are required to prevent spurious accumulation of non-coding transcripts from the 3′ end of genes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-12">All the wild-type (WT) and mutant <em>Arabidopsis</em> plants used in this study are from the Columbia (Col-0) background. <em>fry1-6</em> (Salk_020882), <em>xrn2-1</em> (Salk_041148), <em>xrn3-3</em> (Sail_1172C07), <em>xrn4-6</em> (Salk_014209), <em>xrn2-1xrn3-3</em>, <em>xrn2-1xrn4-6</em>, and <em>xrn3-3xrn4-6</em> were described previously (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>). <em>xrn2-4</em> (Salk_073255) was recovered from the Salk T-DNA insertion collection [<a href="http://signal.salk.edu/cgi-bin/tdnaexpress">http://signal.salk.edu/cgi-bin/tdnaexpress</a> and <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alonso <em>et al.</em> (2003)</a>].</p></div><div id="sec-3" class="subsection"><h3>Tiling array analysis</h3><p id="p-13">The GeneChip <em>Arabidopsis</em> tiling array set (1.0F Array and 1.0R Array; Affymetrix, Santa Clara, CA) was used (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhang <em>et al.</em> 2006</a>). Plant growth conditions, RNA extraction, probe synthesis, and array hybridization were carried out as described previously (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Matsui <em>et al.</em> 2008</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kurihara <em>et al.</em> 2009</a>). The ARTADE-based method (P-initial &lt; 10<sup>−8</sup>) was used to predict novel (unannotated) transcripts from the expression data (ARTADE v1.2.1.1) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Toyoda and Shinozaki 2005</a>). The <em>Arabidopsis</em> genome annotation used in tiling array analysis was based on the TAIR8 (<a href="ftp://ftp.arabidopsis.org/home/tair/Genes/TAIR8_genome_release/">ftp://ftp.arabidopsis.org/home/tair/Genes/TAIR8_genome_release/</a>). The transcripts predominantly upregulated in the mutants were identified by the Mann-Whitney <em>U</em>-test (FDR α = 0.05).</p></div><div id="sec-4" class="subsection"><h3>RNA-Seq analysis</h3><p id="p-14">Wild-type, <em>fry1-6</em>, <em>xrn3-3</em>, <em>xrn2-1xrn4-6</em>, and <em>xrn3-3xrn4-6</em> were grown in Linsmaier and Skoog (LS; Caisson Laboratories, North Logan, UT) media containing 1% sucrose and 0.85% agar at 23° under conditions of 16 hr of light and 8 hr of darkness for 2 weeks. Wild-type (WT3w) and <em>xrn2-1xrn3-3</em> were grown in soil (Metro Mix 250; Grace-Sierra, Boca Raton, FL) at 23° under conditions of 16 hr of light and 8 hr of darkness for 3 weeks. Total RNA was extracted from the plants using Trizol reagent (Invitrogen, Carlsbad, CA). Poly(A)<sup>+</sup> fraction was separated from 80 μg of total RNA using Oligotex mRNA Mini Kit (Qiagen, Valencia, CA) and fragmented using Fragmentation Reagents (Applied Biosystems/Ambion, Austin, TX) at 70° for 15 min. Fifty nanograms of the fragmented RNA was used to generate strand-specific RNA-Seq library according to the Directional mRNA-Seq Library Preparation Protocol (Illumina). RNA-Seq libraries were sequenced for 42 cycles using the Illumina Genome Analyzer IIx (WT and <em>fry1-6</em>) or for 50 cycles using the HiSeq 2000 (<em>xrn3-3</em>, <em>xrn3-3xrn4-6</em>, 3-week-old WT and <em>xrn2-1xrn3-3</em>) according to manufacturer’s instruction. Image analysis and base calling were performed with the standard Illumina pipeline. Read sequences were aligned with the Tophat software to the TAIR9 reference genome (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Trapnell <em>et al.</em> 2009</a>). Reads that aligned to multiple positions were discarded. <span class="underline">R</span>eads <span class="underline">p</span>er <span class="underline">k</span>ilobase of transcript per <span class="underline">m</span>illion (RPKM) values were calculated using the Refiner Genome module developed by Genedata Expressionist (Genedata Inc., Lexington, MA). The <em>Arabidopsis</em> genome annotation used was based on the TAIR9 (<a href="ftp://ftp.arabidopsis.org/home/tair/Genes/TAIR9_genome_release/">ftp://ftp.arabidopsis.org/home/tair/Genes/TAIR9_genome_release/</a>). The transcripts predominantly upregulated in the mutants were identified by using a Student <em>t</em>-test (P-value &lt; 0.1).</p></div><div id="sec-5" class="subsection"><h3>MethylC-Seq analysis</h3><p id="p-15">Wild-type and <em>fry1-6</em> were grown in LS (Caisson Laboratories) media containing 1% sucrose and 0.85% agar at 23° under conditions of 16 hr of light and 8 hr of darkness for 2 weeks. One microgram of gDNA was isolated from this tissue using the Qiagen Plant DNeasy Kit (Qiagen). DNA was sheared to ∼100 bp using the Covaris S2 System using the following parameters: cycle number = 6, duty cycle = 20%, intensity = 5, cycles/burst = 200, and time = 60 sec. Sonicated DNA was purified with a mini-elute column (Qiagen). Sequencing libraries were constructed using the NEBNext DNA Sample Prep Reagent Set 1 (New England Biolabs, Ipswich, MA) following the manufacturer’s instructions, except for the following modifications. Methylated adapters were used instead of the standard genomic DNA adapters from Illumina. Ligations were purified with AMPure XP beads (Beckman, Brea, CA). A total of 450 ng of DNA was bisulfite treated using the MethylCode Kit (Invitrogen) according to the manufacturer’s instructions and then PCR amplified using Pfu Cx Turbo (Agilent, Santa Clara, CA) instead of using the Phusion Taq included in the NEBNext kit using the following PCR conditions (2 min at 95°, 4 cycles of 15 sec at 98°, 30 sec at 60°, 4 min at 72°, and 10 min at 72°).</p></div><div id="sec-6" class="subsection"><h3>Analysis of differentially methylated regions</h3><p id="p-16">DMRs were identified using the methylPipe package in R (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lister <em>et al.</em> 2011</a>; Pelizzola, unpublished results). Each specific methylation context (CG, CHG, and CHH) was scanned genome-wide requiring at least 10 mC differences within a 100 bp window. The methylation level of the sites within a window was then compared across all samples using a using a Kruskal-Wallis test. Next, these potential DMRs were consolidated by joining neighboring DMRs that occur within 100 bp of each other. The P-values of joined DMRs were combined using Fisher’s Method. The P-values of these joined DMRs were then adjusted for multiple hypotheses testing with the Benjamini-Hochburg method as implemented in R, and any DMR with an adjusted P-value below 0.01 was retained. Furthermore, a stringent requirement of an 8-fold difference in methylation density between the least methylated and most methylated sample was required.</p></div><div id="sec-7" class="subsection"><h3>Northern blot analysis</h3><p id="p-17">Fifteen micrograms of total RNA were loaded into 1% denaturing agarose gel containing 18% formaldehyde and MOPS buffer (0.2 M MOPS, 80 mM Na-acetate, and 10 mM EDTA, pH 7.0), subjected to electrophoresis at 100 V in MOPS buffer, and transferred onto Hybond N+ membrane (GE Healthcare, Piscataway, NJ) by capillary blotting method in 20 × SSC buffer (3 M sodium chloride, 300 mM trisodium citrate, pH 7.0), followed by UV-crosslinking with 120,000 μJ/cm<sup>2</sup>. Prehybridization and hybridization with each specific probe were performed with the PerfectHyb hybridization buffer (Sigma-Aldrich, St. Louis, MO) at 65° for 1 hr and overnight, respectively. The membrane was washed in the first wash buffer (2 × SSC, 0.1% SDS) for 10 min twice and then in the second wash buffer (0.5 × SSC, 0.1% SDS) for 20 min three times. The membrane was exposed to Imaging Plate BAS-MS2040 for 2 hr and the signals were detected using Bio-Image analyzer BAS-2500 (Fujifilm, Tokyo, Japan).</p><p id="p-18">DNA probes were constructed by random priming reactions with a <sup>32</sup>P-dCTP using the Megaprime DNA-labeling system (GE Healthcare) according to manufacturer’s instruction. DNA fragments used for these reactions were PCR amplified from genomic DNA using the following primer sets: mRNA_F and mRNA_R for mRNA detection, 3′ext_F and 3′ext_R for 3′ extension detection, and TUB8_F and TUB8_R for <em>TUB8</em> mRNA detection (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/0013622SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS1.xls">Table S1</a>).</p></div><div id="sec-8" class="subsection"><h3>Self-ligation–mediated RT-PCR</h3><p id="p-19">Total RNA was extracted using Trizol reagent (Invitrogen) from 2-week-old seedlings grown in LS media as described above. Poly(A)<sup>+</sup> RNA was separated from total RNA using Poly(A) Purist kit (Applied Biosystems/Ambion) according to the manufacturer’s instructions. The RNA (500 ng) was treated with or without calf intestinal phosphatase (CIP, 10 U; New England Biolabs, Cambridge, MA) in a 20 μL volume and then tobacco acid pyrophosphatase (TAP, 2 U; Epicentre Biotechnologies, Madison, WI) in a 30 μL volume. The modified or unmodified RNA molecules were self-ligated using T4 RNA ligase (20 U; Promega) in a 50 μL volume, followed by reverse-transcription using SuperScript II Reverse Transcriptase and Oligo dT primer in a 20 μL volume. As a PCR template, 0.25 μL of RT product was used. The PCR was carried out in a 20 μL volume using Phire Hot Start DNA polymerase (New England Biolabs) and the following primer set (circle_F and circle_R; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS1.xls">Table S1</a>). The amplification program was 35 cycles of 5 sec at 98°, 15 sec at 55°, and 20 sec at 72°. The resulting PCR product was loaded into a 1.5% agarose gel, subjected to electrophoresis in TBE (50 mM Tris-HCl, 48.5 mM boric acid, 2 mM EDTA, pH 8.0) buffer at 100 V and visualized under UV light.</p></div><div id="sec-9" class="subsection"><h3>Quantitative RT-PCR analysis</h3><p id="p-20">Total RNA was extracted from 20-day-old seedlings grown in LS media containing 1% sucrose and 0.85% agar at 23° under conditions of 16 hr of light and 8 hr of darkness. Ten micrograms of total RNA were digested in 50 μL of TURBO DNase (Applied Biosystems/Ambion) for 30 min and recovered by acidic phenol:chloroform extractions and ethanol precipitation. Two micrograms of the DNase-treated RNA was used for the reverse-transcription reaction in 20 μL using the SuperScript II reverse transcriptase (Invitrogen) and Oligo dT primer according to the manufacturer’s instructions. As a template for PCR amplification, 0.25 μL of the RT product was used. The PCR was carried out in 20 μL using Taq DNA polymerase for amplification of mRNAs and 3′ extension of At1g16410, or using Phire Hot Start DNA polymerase (New England Biolabs) for amplification of 3′ extensions (with the exception of At1g16410), pri-miRNA_5′s, and pri-miRNA_3′s. Primer sets used here and cycle numbers of PCR are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS1.xls">Table S1</a>.</p></div><div id="sec-10" class="subsection"><h3>Accession numbers</h3><p id="p-21">Raw data from tiling arrays and RNA-Seq were deposited in Gene Expression Omnibus under the accession numbers GSE32977 and GSE34654, respectively. Raw data from MethylC-Seq were deposited in the Sequence Read Archive (SRA) of NCBI under accession number SRA049100.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Identification of novel transcripts extending from the 3′ ends of mRNAs</h3><p id="p-22">Strand-specific whole-genome tiling array analysis of <em>fry1-6</em> and <em>xrn</em> single mutants (<em>xrn2-4</em>, <em>xrn3-3</em>, <em>xrn4-6</em>) was performed to identify the roles of XRNs and FRY1 in transcript accumulation. The <em>xrn2-4</em> allele (Salk_073255) has a T-DNA insertion in the 12<sup>th</sup> intron, resulting in extremely reduced mRNA expression (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS1.pdf">Figure S1</a>A). These data were normalized together with previous tiling array data generated for <em>upf</em> mutants (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Kurihara <em>et al.</em> 2009</a>) to reduce misidentification of unannotated transcripts, such as non-coding RNAs. The ARTADE program, which predicts novel transcripts <em>de novo</em> (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Toyoda and Shinozaki 2005</a>) uncovered 171 novel transcripts located less than 300 nucleotides (nt) downstream of the 3′ ends of upstream mRNAs in <em>fry1-6</em> [Fold (<em>fry1-6</em>/WT) &gt; 2, P-initial &lt; 10<sup>−8</sup>, FDR α = 0.05] (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS2.xls">Table S2</a>). Henceforth, we will refer to these novel transcripts as 3′ extensions. Interestingly, 61% (104) of the estimated lengths of the 3′ extensions were less than 800 nt (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The results from the tiling array analyses can be observed on OmicBrowse at <a href="http://omicspace.riken.jp/gps/group/psca7">http://omicspace.riken.jp/gps/group/psca7</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/487/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of novel transcripts extending the 3&#x2032; ends of mRNAs. (A) Example (At1g16410 locus) of novel 3&#x2032; extended signals (3&#x2032; extensions) identified by the ARTADE program (Toyoda and Shinozaki 2005) in tiling array data from fry1-6. Vertical red bars indicate signal intensities from each probe. Horizontal blue and light blue regions are exons and introns, respectively, in the gene model of At1g16410. The horizontal orange region indicated by the arrow is a predicted 3&#x2032; extension. The length of black scale bar is 500 nt. (B) Distribution of estimated lengths of 171 3&#x2032; extensions identified in the tiling array analysis of fry1-6. (C) A schematic of the 500 nt downstream region used for calculation of RPKM values. (D) Comparison of accumulation profiles in the range from the 3&#x2032; ends of genes to 500 nt downstream between wild-type and fry1-6 (RNA-Seq). RPKM values of all expressed transcripts (total 19,104) in wild-type or fry1-6 were classified into four categories. (E) A scatter plot of average RPKM values between wild-type and fry1-6. The 3&#x2032; regions with RPKM values less than 500 were plotted. (F) Examples of 3&#x2032; extensions identified in fry1-6 by filtering through four parameters [Fold (fry1-6/WT) &gt; 5, RPKM average (fry1) &gt; 1, P-value &lt; 0.1, non-pri-miRNA]. Pink regions are 3&#x2032; extensions. The length of the black scale bar is 500 nt." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30288879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of novel transcripts extending the 3&#x2032; ends of mRNAs. (A) Example (At1g16410 locus) of novel 3&#x2032; extended signals (3&#x2032; extensions) identified by the ARTADE program (Toyoda and Shinozaki 2005) in tiling array data from &lt;em&gt;fry1-6&lt;/em&gt;. Vertical red bars indicate signal intensities from each probe. Horizontal blue and light blue regions are exons and introns, respectively, in the gene model of At1g16410. The horizontal orange region indicated by the arrow is a predicted 3&#x2032; extension. The length of black scale bar is 500 nt. (B) Distribution of estimated lengths of 171 3&#x2032; extensions identified in the tiling array analysis of &lt;em&gt;fry1-6&lt;/em&gt;. (C) A schematic of the 500 nt downstream region used for calculation of RPKM values. (D) Comparison of accumulation profiles in the range from the 3&#x2032; ends of genes to 500 nt downstream between wild-type and &lt;em&gt;fry1-6&lt;/em&gt; (RNA-Seq). RPKM values of all expressed transcripts (total 19,104) in wild-type or &lt;em&gt;fry1-6&lt;/em&gt; were classified into four categories. (E) A scatter plot of average RPKM values between wild-type and &lt;em&gt;fry1-6&lt;/em&gt;. The 3&#x2032; regions with RPKM values less than 500 were plotted. (F) Examples of 3&#x2032; extensions identified in &lt;em&gt;fry1-6&lt;/em&gt; by filtering through four parameters [Fold (&lt;em&gt;fry1-6&lt;/em&gt;/WT) &gt; 5, RPKM average (&lt;em&gt;fry1&lt;/em&gt;) &gt; 1, P-value &lt; 0.1, non-pri-miRNA]. Pink regions are 3&#x2032; extensions. The length of the black scale bar is 500 nt.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/487/F1.medium.gif" width="355" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/4/487/F1.medium.gif" width="355" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/487/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/487/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461338" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Identification of novel transcripts extending the 3′ ends of mRNAs. (A) Example (At1g16410 locus) of novel 3′ extended signals (3′ extensions) identified by the ARTADE program (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Toyoda and Shinozaki 2005</a>) in tiling array data from <em>fry1-6</em>. Vertical red bars indicate signal intensities from each probe. Horizontal blue and light blue regions are exons and introns, respectively, in the gene model of At1g16410. The horizontal orange region indicated by the arrow is a predicted 3′ extension. The length of black scale bar is 500 nt. (B) Distribution of estimated lengths of 171 3′ extensions identified in the tiling array analysis of <em>fry1-6</em>. (C) A schematic of the 500 nt downstream region used for calculation of RPKM values. (D) Comparison of accumulation profiles in the range from the 3′ ends of genes to 500 nt downstream between wild-type and <em>fry1-6</em> (RNA-Seq). RPKM values of all expressed transcripts (total 19,104) in wild-type or <em>fry1-6</em> were classified into four categories. (E) A scatter plot of average RPKM values between wild-type and <em>fry1-6</em>. The 3′ regions with RPKM values less than 500 were plotted. (F) Examples of 3′ extensions identified in <em>fry1-6</em> by filtering through four parameters [Fold (<em>fry1-6</em>/WT) &gt; 5, RPKM average (<em>fry1</em>) &gt; 1, P-value &lt; 0.1, non-pri-miRNA]. Pink regions are 3′ extensions. The length of the black scale bar is 500 nt.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">We suspected that many low abundance 3′ transcript extensions may not be detected using tiling array analysis. To overcome this limitation, strand-specific RNA-Seq was carried out using an Illumina Genome Analyzer IIx to investigate these RNA populations at single base resolution. Unlike the tiling array technique, the RNA-Seq method does not suffer from background noise (cross-hybridization of RNAs with multiple probes) because the vast majority of reads can be uniquely mapped in the <em>Arabidopsis</em> reference genome (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lister <em>et al.</em> 2008</a>). First, three biological replicates of RNA-Seq were carried out using mRNA prepared using wild-type and <em>fry1-6</em> seedlings. We identified 33.2 and 35.0 million sequenced reads that uniquely aligned to the TAIR9 reference genome sequence in wild-type and <em>fry1-6</em>, respectively. For quantification of transcript levels, RPKM values were calculated for the regions extending 500 nt downstream from the 3′ ends of all annotated genes. (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). This range was selected because the lengths of most of 3′ extensions observed from previous tiling array analysis were less than 800 nt in length (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). RPKM values of these 3′ regions tended to be higher in <em>fry1-6</em> than in wild-type (data are displayed by classifying them into four categories: RPKM ≦ 0.2. 0.2 &lt; RPKM ≤ 1, 1 &lt; RPKM ≤ 10, and RPKM &gt; 10; <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). In <em>fry1-6</em>, novel transcripts were detected at the 3′ ends of 6901 genes (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1E</a>, see vertical 0 line of WT, indicated by an arrow). RNA-Seq data were then further filtered by four parameters [Fold change (<em>fry1-6</em>/WT) &gt; 5, RPKM average (<em>fry1</em>) &gt; 1, P-value &lt; 0.1, non-pri-miRNA], which resulted in the identification of 2230 high-quality candidate 3′ transcript extensions (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1F</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS2.xls">Table S2</a>). The RNA-Seq data can be visualized using the AnnoJ genome browser located at <a href="http://neomorph.salk.edu/dev/pages/XRN.html">http://neomorph.salk.edu/dev/pages/XRN.html</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461333/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461333/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of 3′ extensions in RNA-Seq</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Because of the possibility that these 5′ mRNAs, which are accompanied with 3′ extensions, and the candidate 3′ extensions may, in fact, be two separate transcripts rather than one extended transcript, northern blot analysis was carried out on two genes (At1g16410 and At3g45160) using two probes covering the mRNA or its 3′ extension. We did not uncover evidence for single transcripts from either of the genes tested (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Furthermore, RT-PCR across the junction between each mRNA and its 3′ extension using an additional six transcript pairs did not provide evidence for gene transcript extension (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS1.pdf">Figure S1</a>B). Therefore, we conclude that the majority of 5′ mRNAs and 3′ “extensions” are generally two separate transcripts, although we cannot rule out the possibility that low levels of single transcripts exist or that they exist for a limited number of untested loci.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/487/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of 3&#x2032; extensions. (A) Northern blot of mRNAs and 3&#x2032; extensions of At1g16410 and At3g45160. Positions of probes used are shown in the illustration of the mRNA and the 3&#x2032; extension. Samples from two biological replicates are loaded on each blot. TUB8 mRNA and ribosomal RNA were used as internal controls. (B) Self-ligation&#x2013;mediated RT-PCR analysis. The experimental procedure is shown in Figure S1C. ACT2 mRNA was used as an internal control. CIP, calf intestinal phosphatase; TAP, tobacco acid pyrophosphatase." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30288879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Characterization of 3&#x2032; extensions. (A) Northern blot of mRNAs and 3&#x2032; extensions of At1g16410 and At3g45160. Positions of probes used are shown in the illustration of the mRNA and the 3&#x2032; extension. Samples from two biological replicates are loaded on each blot. &lt;em&gt;TUB8&lt;/em&gt; mRNA and ribosomal RNA were used as internal controls. (B) Self-ligation&#x2013;mediated RT-PCR analysis. The experimental procedure is shown in Figure S1C. &lt;em&gt;ACT2&lt;/em&gt; mRNA was used as an internal control. CIP, calf intestinal phosphatase; TAP, tobacco acid pyrophosphatase.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/487/F2.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/4/487/F2.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/487/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/487/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461421" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Characterization of 3′ extensions. (A) Northern blot of mRNAs and 3′ extensions of At1g16410 and At3g45160. Positions of probes used are shown in the illustration of the mRNA and the 3′ extension. Samples from two biological replicates are loaded on each blot. <em>TUB8</em> mRNA and ribosomal RNA were used as internal controls. (B) Self-ligation–mediated RT-PCR analysis. The experimental procedure is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS1.pdf">Figure S1</a>C. <em>ACT2</em> mRNA was used as an internal control. CIP, calf intestinal phosphatase; TAP, tobacco acid pyrophosphatase.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Previous reports suggest that the exoribonuclease Rat1/Xrn2 degrades 3′ read-through products after endonucleolytic cleavage at the poly(A) site of these RNA polymerase II (Pol. II)–transcribed mRNAs (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2004</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">West <em>et al.</em> 2004</a>). Such 3′ extensions are uncapped, which is an important signature of exoribonuclease substrates (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Gregory <em>et al.</em> 2008</a>). To examine whether the 5′ end of the 3′ extension transcripts contains a canonical 5′ cap, we performed self-ligation–mediated RT-PCR analysis for the 3′ extensions of At1g16410, At3g45160, and At3g56360 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS1.pdf">Figure S1</a>C). In this assay, the poly(A)-enriched RNA fraction is treated with or without CIP and TAP enzymes and then self-ligated to make a circle, followed by RT-PCR using a primer set that amplifies fragments across the gap between 5′ and 3′ ends of the RNA. Only capped RNA species can be amplified after treatment with CIP and TAP enzymes, whereas only uncapped RNA species can be amplified without CIP and TAP treatment. Specific amplification products from 3′ extensions were detected only in <em>fry1-6</em>, and not in wild-type, when RNAs were not pretreated with these enzymes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The smeary bands observed in lane 4 most likely indicate that the position of 5′ and/or 3′ ends of 3′ extensions is not uniform, which is probably due to degradation and/or variable poly(A) tail and nonstop PCR amplification. On the other hand, when RNAs were pretreated with TAP and CIP, specific amplification products from <em>ACT2</em> mRNA were detected in both WT and <em>fry1-6</em>, demonstrating that the enzyme treatments were effective. These results indicate that the 3′ extensions detected in <em>fry1-6</em> mutants are very likely equivalent to Pol II 3′ read-through products.</p></div><div id="sec-13" class="subsection"><h3>mRNA 3′extensions often occur in abundantly expressed transcripts</h3><p id="p-30">When the set of 2230 mRNAs with 3′ extensions were analyzed by gene ontology (GO) annotations (categories cellular component, molecular component, and biological component), there were few significant differences in represented GO terms between all <em>Arabidopsis</em> genes expressed in <em>fry1-6</em> and the 2230 mRNAs. The only exception was that in all three categories, the percentages of genes of unknown function were significantly lower for the 2230 mRNAs with 3′ extensions than for all genes expressed in <em>fry1-6</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS2.pdf">Figure S2</a>A). Therefore, these data suggest that FRY1 does not target specific classes of genes but, instead, is required for general surveillance that prevents these 3′ extensions.</p><p id="p-31">Next, we compared the accumulation of these 2230 mRNAs and their 3′ extensions between wild-type and <em>fry1-6</em>. Although accumulation of 3′ extensions was apparently higher in <em>fry1-6</em> than in wild-type, accumulation of the 5′ mRNAs was comparable between both genotypes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). However, the majority of the mRNAs with 3′ extensions have relatively higher RPKM values compared with all expressed genes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS2.pdf">Figure S2</a>B), indicating that actively transcribed genes tend to possess 3′ extensions.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/487/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Many actively transcribed genes possess 3&#x2032; RNA extensions. Comparison of accumulation profiles of 2230 3&#x2032; extensions and the 5&#x2032; mRNAs between wild-type and fry1-6. RPKM values of 3&#x2032; extensions and the mRNAs were classified into four and six categories, respectively. (B) Distribution of RPKM values of mRNAs tailed with 2230 3&#x2032; extensions. The population &#x201C;all_gene&#x201D; comprising mRNAs with RPKM values in fry1-6 was used for control of comparison. ext, extension." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30288879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Many actively transcribed genes possess 3&#x2032; RNA extensions. Comparison of accumulation profiles of 2230 3&#x2032; extensions and the 5&#x2032; mRNAs between wild-type and &lt;em&gt;fry1-6&lt;/em&gt;. RPKM values of 3&#x2032; extensions and the mRNAs were classified into four and six categories, respectively. (B) Distribution of RPKM values of mRNAs tailed with 2230 3&#x2032; extensions. The population &#x201C;all_gene&#x201D; comprising mRNAs with RPKM values in &lt;em&gt;fry1-6&lt;/em&gt; was used for control of comparison. ext, extension.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/487/F3.medium.gif" width="440" height="151"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/4/487/F3.medium.gif" width="440" height="151"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/487/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/487/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461415" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Many actively transcribed genes possess 3′ RNA extensions. Comparison of accumulation profiles of 2230 3′ extensions and the 5′ mRNAs between wild-type and <em>fry1-6</em>. RPKM values of 3′ extensions and the mRNAs were classified into four and six categories, respectively. (B) Distribution of RPKM values of mRNAs tailed with 2230 3′ extensions. The population “all_gene” comprising mRNAs with RPKM values in <em>fry1-6</em> was used for control of comparison. ext, extension.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>XRN3 represses 3′ extension transcripts</h3><p id="p-33">FRY1 is required for catalyzing conversion of the XRN inhibitor PAP into 5′AMP, thereby promoting XRN activities (<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Kim <em>et al.</em> 2009</a>). In addition to the <em>fry1-6</em> mutant, slightly increased accumulation of some 3′ extensions in <em>xrn3-3</em> were observed from the tiling array analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS3.pdf">Figure S3</a>A). However, it was difficult to detect 3′ extensions in <em>xrn3-3</em>, possibly because <em>xrn3-3</em> is a knockdown allele with a T-DNA insertion in the promoter region that results in reduced expression of <em>XRN3</em> mRNA rather than in a complete knockout (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS1.pdf">Figure S1</a>A; <a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>). Therefore, the effect on accumulation of 3′ extensions might be limited due to the allele used and low detection sensitivity of the tiling array in this experiment. As knockout mutant of XRN3 gene is lethal, we could not use it here (<a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>).</p><p id="p-34">To increase detection sensitivity, two biological replicates of directional RNA-Seq (on the Illumina HiSeq 2000 platform) were performed using <em>xrn3-3</em> to examine whether XRN3 activity degrades the 3′ read-through products described above. Additionally, two biological replicates of RNA-Seq using <em>xrn2-1xrn3-3</em>, <em>xrn2-1xrn4-6</em>, and <em>xrn3-3xrn4-6</em> double mutants were carried out to investigate possible redundancies among the three XRN enzymes. The total numbers of sequenced reads mapped to the TAIR9 reference genome sequence for each genotype were 65.7, 58.7, 60.8, 35.7, and 58.5 million in <em>xrn3-3</em>, <em>xrn3xrn4</em>, <em>xrn2xrn4</em>, 3-week-old wild-type, and <em>xrn2xrn3</em>, respectively. Analyses of the results revealed that RPKM values of the 3′ regions composed of the 500 nt regions downstream of genes were typically greater in <em>xrn3-3</em>, <em>xrn2xrn3</em>, and <em>xrn3xrn4</em> (but not <em>xrn2xrn4</em>) when compared with wild-type (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). When compared with <em>fry1-6</em> (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a>), RPKM values were lower, which is consistent with a previous report that <em>xrn3-3</em> is a weak allele that results in fewer morphological defects than <em>fry1-6</em> (<a id="xref-ref-12-11" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/487/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The major activity of XRN3 is to eliminate 3&#x2032; extensions. (A) Comparison of accumulation profiles in wild-type and each mutant in the range from the 3&#x2032; end of genes to 500 nt downstream. RPKM values of all expressed transcripts in wild-type or each mutant were classified into four categories. Total numbers are 18,431, 17,247, 18,480, and 15,323 in wild-type or xrn3-3, xrn2xrn3, xrn3xrn4, and xrn2xrn4, respectively. Averages RPKM values of two RNA-Seq replicates were used. Tissues use for wild-type (WT), xrn3-3, xrn3xrn4, and xrn2xrn4 were 2-week-old plants grown on nutrient medium, whereas xrn2xrn3 and the control wild-type (WT3w) were 3-week-old plants grown in soil. (B) Venn diagrams of the overlap between 3&#x2032; extensions identified in fry1-6 and those identified in xrn3-3, xrn2xrn3, and xrn3xrn4, and the overlap between in xrn3-3 and in xrn3 double mutants. (C) Quantitative RT-PCR analysis of four 3&#x2032; extensions and the corresponding 5&#x2032; adjacent mRNAs. Positions of primers used are shown in the illustration of the mRNA and the 3&#x2032; extension. Vertical axes show relative accumulation normalized against ACT2 expression. x2x3 = xrn2xrn3; x2x4 = xrn2xrn4; x3x4 = xrn3xrn4. Additional qRT-PCR data are presented in Figure S3C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30288879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The major activity of XRN3 is to eliminate 3&#x2032; extensions. (A) Comparison of accumulation profiles in wild-type and each mutant in the range from the 3&#x2032; end of genes to 500 nt downstream. RPKM values of all expressed transcripts in wild-type or each mutant were classified into four categories. Total numbers are 18,431, 17,247, 18,480, and 15,323 in wild-type or &lt;em&gt;xrn3-3&lt;/em&gt;, &lt;em&gt;xrn2xrn3&lt;/em&gt;, &lt;em&gt;xrn3xrn4&lt;/em&gt;, and &lt;em&gt;xrn2xrn4&lt;/em&gt;, respectively. Averages RPKM values of two RNA-Seq replicates were used. Tissues use for wild-type (WT), &lt;em&gt;xrn3-3&lt;/em&gt;, &lt;em&gt;xrn3xrn4&lt;/em&gt;, and &lt;em&gt;xrn2xrn4&lt;/em&gt; were 2-week-old plants grown on nutrient medium, whereas &lt;em&gt;xrn2xrn3&lt;/em&gt; and the control wild-type (WT3w) were 3-week-old plants grown in soil. (B) Venn diagrams of the overlap between 3&#x2032; extensions identified in &lt;em&gt;fry1-6&lt;/em&gt; and those identified in &lt;em&gt;xrn3-3&lt;/em&gt;, &lt;em&gt;xrn2xrn3&lt;/em&gt;, and &lt;em&gt;xrn3xrn4&lt;/em&gt;, and the overlap between in &lt;em&gt;xrn3-3&lt;/em&gt; and in &lt;em&gt;xrn3&lt;/em&gt; double mutants. (C) Quantitative RT-PCR analysis of four 3&#x2032; extensions and the corresponding 5&#x2032; adjacent mRNAs. Positions of primers used are shown in the illustration of the mRNA and the 3&#x2032; extension. Vertical axes show relative accumulation normalized against &lt;em&gt;ACT2&lt;/em&gt; expression. x2x3 = &lt;em&gt;xrn2xrn3&lt;/em&gt;; x2x4 = &lt;em&gt;xrn2xrn4&lt;/em&gt;; x3x4 = &lt;em&gt;xrn3xrn4&lt;/em&gt;. Additional qRT-PCR data are presented in Figure S3C.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/487/F4.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/4/487/F4.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/487/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/487/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461328" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">The major activity of XRN3 is to eliminate 3′ extensions. (A) Comparison of accumulation profiles in wild-type and each mutant in the range from the 3′ end of genes to 500 nt downstream. RPKM values of all expressed transcripts in wild-type or each mutant were classified into four categories. Total numbers are 18,431, 17,247, 18,480, and 15,323 in wild-type or <em>xrn3-3</em>, <em>xrn2xrn3</em>, <em>xrn3xrn4</em>, and <em>xrn2xrn4</em>, respectively. Averages RPKM values of two RNA-Seq replicates were used. Tissues use for wild-type (WT), <em>xrn3-3</em>, <em>xrn3xrn4</em>, and <em>xrn2xrn4</em> were 2-week-old plants grown on nutrient medium, whereas <em>xrn2xrn3</em> and the control wild-type (WT3w) were 3-week-old plants grown in soil. (B) Venn diagrams of the overlap between 3′ extensions identified in <em>fry1-6</em> and those identified in <em>xrn3-3</em>, <em>xrn2xrn3</em>, and <em>xrn3xrn4</em>, and the overlap between in <em>xrn3-3</em> and in <em>xrn3</em> double mutants. (C) Quantitative RT-PCR analysis of four 3′ extensions and the corresponding 5′ adjacent mRNAs. Positions of primers used are shown in the illustration of the mRNA and the 3′ extension. Vertical axes show relative accumulation normalized against <em>ACT2</em> expression. x2x3 = <em>xrn2xrn3</em>; x2x4 = <em>xrn2xrn4</em>; x3x4 = <em>xrn3xrn4</em>. Additional qRT-PCR data are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS3.pdf">Figure S3</a>C.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">When these RNA-Seq data were stringently filtered using four parameters (Fold change &gt; 5, RPKM average &gt; 1, P-value &lt; 0.1, non-pri-miRNA), we identified 528, 473, and 392 3′ extension candidates in <em>xrn3</em>, <em>xrn2xrn3</em>, and <em>xrn3xrn4</em>, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS3.xls">Table S3</a>). These candidates significantly overlapped with the 2230 3′ extensions observed in <em>fry1-6</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). The abundance of the 2230 mRNAs with 3′ extensions identified in <em>fry1-6</em> showed little change in <em>xrn3</em>, <em>xrn2xrn3</em>, <em>xrn2xrn4</em>, and <em>xrn3xrn4</em> compared with wild-type. However, the abundance of the concomitant 2230 3′ extensions was notably increased in <em>xrn3</em>, <em>xrn2xrn3</em>, and <em>xrn3xrn4</em>, but not in <em>xrn2xrn4</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS3.pdf">Figure S3</a>B). The candidates in <em>xrn3-3</em> do not seem to widely overlap with those in <em>xrn3</em> double mutants (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). Of the 528 candidates in <em>xrn3-3</em>, 20.3% and 19.1% overlap with those in <em>xrn2xrn3</em> and <em>xrn3xrn4</em>, respectively. However, this result is due, in part, to the limitation of threshold values used for candidate identification. For example, when alternative (less stringent) threshold values are used to identify 3′ extension candidates (Fold change &gt; 2, RPKM average &gt; 1, non-pri-miRNA), 44.8% and 51.5% of newly calculated candidates in <em>xrn3-3</em> overlap with those in <em>xrn2xrn3</em> and <em>xrn3xrn4</em>, respectively. These results reveal that significant redundancy in suppressing 3′ extensions was not detected between XRN3 and XRN2/XRN4, indicating the XRN3 enzyme possesses unique specificity in 3′ end processing/turnover.</p><p id="p-37">Quantitative RT-PCR (qRT-PCR) analysis was used to examine the abundance of selected 5′ mRNAs and 3′ extensions in all genotypes. Two primer sets were used to amplify the 5′ mRNAs (mRNA_F and mRNA_R) and 3′ extensions (3′ext_F and 3′ext_R), respectively (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS3.xls">Table S3</a>). These experiments revealed the accumulation of 3′ extensions was not correlated with accumulation of the 5′ mRNAs, consistent with the results of the RNA-Seq analyses (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS3.pdf">Figure S3</a>C). In this assay, we observed either compensation for reduced XRN3 activity in <em>xrn3-3</em> by XRN2/XRN4 or a dependency on the combination of mutations in the double mutants. For example, in the case of At3g45160, accumulation of the 3′ extension in <em>xrn3xrn4</em> was higher than in <em>xrn3-3</em>. By contrast, in the case of At1g19670, accumulation of the 3′ extension observed in <em>xrn3xrn4</em> was lower than in <em>xrn3-3</em>. These results suggest that XRN3 is the main component eliminating 3′ extensions and that other XRNs redundantly act to compensate, in some cases, for reduced XRN3 activity in the <em>xrn3-3</em> mutant. In this context, it is apparent that XRN3 possesses at least of one of the features that define yeast Rat1-type enzymes: involvement in termination of transcription. However, the other <em>Arabidopsis</em> Rat1 orthologs, XRN2 and XRN4, exhibit little or none of this function. Surprisingly, XRN2 does not seem to act redundantly with XRN3 in targeting of 3′ extensions for degradation, even though both XRN2 and XRN3 localize to the nucleus (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Kastenmayer and Green 2000</a>).</p></div><div id="sec-15" class="subsection"><h3>Transcript 3′ extensions do not lead to DNA methylation of the genomic region of origin</h3><p id="p-38">A total of 953 3′ extension candidates potentially overlapped with flanking opposite-strand transcripts from neighboring genes while a total of 24 pairs of 3′ extension candidates were confirmed to overlap with each other (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). We defined these 3′ extensions by postulating that their lengths are ∼500 nt. This range was used because the predicted lengths of most of 3′ extensions observed in tiling array analysis were less than 800 nt (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1B</a>). It is hypothesized that overlap between neighboring Xrn-sensitive antisense transcripts provides a source of double-stranded RNAs which may lead to pools of siRNA species that in turn could mediate <em>de novo</em> DNA methylation of these 3′ extension-generating loci (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhang and Zhu 2011</a>). To investigate whether these 3′ extensions correlated with <em>de novo</em> DNA methylation, we performed genome-wide MethylC-Seq analysis in <em>fry1-6</em> and wild-type (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lister <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schmitz and Zhang 2011</a>). A total of 52.8 and 53.0 million reads were aligned to the reference genome, resulting in ∼42× coverage (∼21× per strand coverage) of both wild-type and <em>fry1-6</em>, respectively. In total, we identified 35 differentially methylated regions (DMR) in <em>fry1-6</em> compared with wild-type (P-value &gt; 0.01, 8-fold difference in methylation levels as determined by combining levels from all three methylation contexts, CG, CHG, and CHH). However, none of the DMRs correlated with the identified 3′ extensions from our previous analysis (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS4.xls">Table S4</a>), indicating that 3′ extensions do not lead to <em>de novo</em> DNA methylation of the 3′ extension originating loci.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/487/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="With few exceptions, 3&#x2032; extensions do not lead to de novo DNA methylation of the 3&#x2032; extension originating loci. (A) Examples of RNA-Seq and MethylC-Seq data from loci that exhibit overlapping transcripts. Pink regions are 3&#x2032; extensions. (B) At1TE93275 locus. The length of black scale bar is 500 nt. (C) Quantitative RT-PCR analysis of the antisense transcript of At1TE93275. Vertical axes show relative accumulation normalized against ACT2 expression. x2x3 = xrn2xrn3; x2x4 = xrn2xrn4; x3x4 = xrn3xrn4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30288879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;With few exceptions, 3&#x2032; extensions do not lead to &lt;em&gt;de novo&lt;/em&gt; DNA methylation of the 3&#x2032; extension originating loci. (A) Examples of RNA-Seq and MethylC-Seq data from loci that exhibit overlapping transcripts. Pink regions are 3&#x2032; extensions. (B) At1TE93275 locus. The length of black scale bar is 500 nt. (C) Quantitative RT-PCR analysis of the antisense transcript of At1TE93275. Vertical axes show relative accumulation normalized against &lt;em&gt;ACT2&lt;/em&gt; expression. x2x3 = &lt;em&gt;xrn2xrn3&lt;/em&gt;; x2x4 = &lt;em&gt;xrn2xrn4&lt;/em&gt;; x3x4 = &lt;em&gt;xrn3xrn4&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/487/F5.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/4/487/F5.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/487/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/487/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461345" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">With few exceptions, 3′ extensions do not lead to <em>de novo</em> DNA methylation of the 3′ extension originating loci. (A) Examples of RNA-Seq and MethylC-Seq data from loci that exhibit overlapping transcripts. Pink regions are 3′ extensions. (B) At1TE93275 locus. The length of black scale bar is 500 nt. (C) Quantitative RT-PCR analysis of the antisense transcript of At1TE93275. Vertical axes show relative accumulation normalized against <em>ACT2</em> expression. x2x3 = <em>xrn2xrn3</em>; x2x4 = <em>xrn2xrn4</em>; x3x4 = <em>xrn3xrn4</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">A recent report using transcriptome tiling array analysis (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sonmez <em>et al.</em> 2011</a>) revealed that expression of some 3′ downstream regions were increased in an <em>fcafpa</em> double mutant, resulting in 3′ transcript extensions similar to those reported here. Moreover, the expression of one transposon (At1TE93275) was reported to be increased, and DNA methylation at this locus was reduced in <em>fcafpa</em>. Similarly, we found that expression of antisense transcripts originating from this transposon was increased, and DNA methylation was drastically reduced in <em>fry1-6</em> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). However, <em>xrn2</em> and <em>xrn4</em> mutations more strongly affected the expression of antisense transcripts originating from this locus compared with <em>xrn3</em> mutations, and the combination of the <em>xrn3</em> mutation with <em>xrn2</em> or <em>xrn4</em> mutations suppressed the expression of this transcript by way of an unknown mechanism (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS4.pdf">Figure S4</a>). Therefore, the mechanism regulating expression of this transposon seems to be independent of the XRN3-controlled 3′ extension.</p></div><div id="sec-16" class="subsection"><h3>FRY1 and XRN3 repress 3′ remnants of miRNA precursors</h3><p id="p-41">MicroRNAs (miRNAs) are regulatory small RNAs produced through DICER-LIKE1 (DCL1)–mediated cleavage of the stem-loop structure of primary miRNA precursors (pri-miRNA) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kurihara and Watanabe 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Song <em>et al.</em> 2010</a>). They execute endonucleolytic cleavage of target mRNAs with target sites complementary to the miRNAs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chapman and Carrington 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Voinnet 2009</a>). Previous reports have demonstrated that XRN4 and FRY1 cooperatively repress the accumulation of some 3′ products of miRNA-mediated cleavage of target loci (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Souret <em>et al.</em> 2004</a>; <a id="xref-ref-12-12" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Gregory <em>et al.</em> 2008</a>). In our tiling array and RNA-Seq analyses, we also detected overaccumulation of such 3′ products of miRNA targets in <em>xrn4-6</em> and <em>fry1-6</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS5.pdf">Figure S5</a>A). We calculated RPKM values of the previously predicted miRNA targets in two ranges: 200 nt downstream of start sites of genes, and 200 nt upstream of 3′ ends of genes in wild-type and <em>fry1-6</em>. The results showed that many of RPKM values of start sites of 165 miRNA targets were comparable, but RPKM values of their 3′ ends were higher in <em>fry1-6</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS5.pdf">Figure S5</a>B). These results indicate and support a model in which the FRY1- and XRN4-mediated decay pathway plays a major role in degrading 3′ products of miRNA-mediated cleavage of targets.</p><p id="p-42">FRY1 and three XRN exoribonucleases cooperatively repress some loop structures released after DCL1-mediated cleavage of pri-miRNAs in the miRNA processing step (<a id="xref-ref-12-13" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>). Our previous report also showed that 5′ cleavage remnants of pri-miRNAs are degraded by the exosome complex (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Chekanova <em>et al.</em> 2007</a>). Our current analysis has newly revealed overaccumulation of sequenced reads that map to pri-miRNA loci in the <em>fry1-6</em> mutant compared with wild-type (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). The enriched transcript regions are located downstream of the stem-loop structures of pri-miRNAs. As most of the pri-miRNA gene models in TAIR9 include stem-loop regions but not 3′ downstream regions adjacent to the stem-loop regions, we calculated RPKM values of pri-miRNAs for the region from the start sites of the gene models to 500 nt downstream of the end of the gene models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS6.pdf">Figure S6</a>A and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS4.xls">Table S4</a>). These data showed that the tendency toward increased accumulation of the 3′ region of pri-miRNAs observed in <em>fry1-6</em> was also detected in <em>xrn3-3</em>, <em>xrn2xrn3</em>, and <em>xrn3xrn4</em> mutants (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>), suggesting that XRN3 also provides a critical activity for degrading 3′ RNA remnants of pri-miRNAs.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/487/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FRY1 and XRN3 repress 3&#x2032; remnants of miRNA precursors. (A) Examples of pri-miRNAs (pri-miR156c and pri-miR172a) showing increased accumulation. (B) Comparison of accumulation profiles of 151 expressed pri-miRNAs. Averages of RPKM values of three replicates of RNA-Seq in wild-type and fry1-6 and two replicates of RNA-Seq in xrn3-3, xrn3xrn4, 3-week-old wild-type, and xrn2xrn3 were used. (C) Quantitative RT-PCR analysis of six pri-miRNA_3&#x2032;s and the pri-miRNA_5&#x2032;s. Positions of primers used are shown in the illustration of pri-miRNA. Vertical axes show relative accumulation normalized against ACT2 expression. x2x3 = xrn2xrn3; x2x4 = xrn2xrn4; x3x4 = xrn3xrn4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30288879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;FRY1 and XRN3 repress 3&#x2032; remnants of miRNA precursors. (A) Examples of pri-miRNAs (pri-miR156c and pri-miR172a) showing increased accumulation. (B) Comparison of accumulation profiles of 151 expressed pri-miRNAs. Averages of RPKM values of three replicates of RNA-Seq in wild-type and &lt;em&gt;fry1-6&lt;/em&gt; and two replicates of RNA-Seq in &lt;em&gt;xrn3-3&lt;/em&gt;, &lt;em&gt;xrn3xrn4&lt;/em&gt;, 3-week-old wild-type, and &lt;em&gt;xrn2xrn3&lt;/em&gt; were used. (C) Quantitative RT-PCR analysis of six pri-miRNA_3&#x2032;s and the pri-miRNA_5&#x2032;s. Positions of primers used are shown in the illustration of pri-miRNA. Vertical axes show relative accumulation normalized against &lt;em&gt;ACT2&lt;/em&gt; expression. x2x3 = &lt;em&gt;xrn2xrn3&lt;/em&gt;; x2x4 = &lt;em&gt;xrn2xrn4&lt;/em&gt;; x3x4 = &lt;em&gt;xrn3xrn4&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/487/F6.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/4/487/F6.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/487/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/487/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461346" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">FRY1 and XRN3 repress 3′ remnants of miRNA precursors. (A) Examples of pri-miRNAs (pri-miR156c and pri-miR172a) showing increased accumulation. (B) Comparison of accumulation profiles of 151 expressed pri-miRNAs. Averages of RPKM values of three replicates of RNA-Seq in wild-type and <em>fry1-6</em> and two replicates of RNA-Seq in <em>xrn3-3</em>, <em>xrn3xrn4</em>, 3-week-old wild-type, and <em>xrn2xrn3</em> were used. (C) Quantitative RT-PCR analysis of six pri-miRNA_3′s and the pri-miRNA_5′s. Positions of primers used are shown in the illustration of pri-miRNA. Vertical axes show relative accumulation normalized against <em>ACT2</em> expression. x2x3 = <em>xrn2xrn3</em>; x2x4 = <em>xrn2xrn4</em>; x3x4 = <em>xrn3xrn4</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">We performed qRT-PCR analysis of pri-miRNAs in all genotypes to further verify our hypothesis that XRN3 degrades 3′ RNA remnants of pri-miRNAs (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). Two primer sets were used: a first primer set detected accumulation of the 5′ pri-miRNA region (pri-miRNA_5′), including the full stem loop; a second primer set detected accumulation of the 3′ region (pri-miRNA_3′), including the entire pri-miRNA and the 3′ remnants of DCL1-mediated cleavage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS1.xls">Table S1</a>). The results revealed that, irrespective of accumulation of the pri-miRNA_5′ regions, accumulation of pri-miR160a_3′, pri-miR168a_3′, and pri-miR172a_3′ were clearly increased in <em>xrn3-3</em>, <em>xrn2xrn3</em>, and <em>fry1-6</em> and slightly increased in <em>xrn3xrn4</em> compared with those in wild-type. By contrast, it was difficult to judge whether accumulation of pri-miR156c_3′, pri-miR158a_3′, and pri-miR164b_3′ were dependent on XRN3 activity in the RT-PCR data, although increased accumulation of these transcripts was faintly detected in a few <em>xrn3-3</em> allele-containing plants in the RNA-Seq data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS6.pdf">Figure S6</a>B and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/TableS5.xls">Table S5</a>). This result is likely due to the remaining XRN3 activity in <em>xrn3-3</em>, which is enough to repress the accumulation of the 3′ remnants of pri-miRNAs, preventing examination of whether other exoribonucleases act redundantly to repress the 3′ remnants. Taken together, the results indicate that XRN3 degrades unnecessary 3′ remnants of DCL1-mediated cleavage of pri-miRNAs.</p></div><div id="sec-17" class="subsection"><h3>Lithium induces accumulation of 3′ extension transcripts</h3><p id="p-45">A previous report showed that lithium ion could inhibit activity of HAL2, a yeast FRY1 ortholog, and then deplete activity of exoribonucleases, such as Rat1 and Xrn1, which induced accumulation of many of non-coding RNAs in yeast (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Dichtl <em>et al.</em> 1997</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Van Dijk <em>et al.</em> 2011</a>). It was also reported that lithium and sodium ions could inhibit the recombinant FRY1 protein from catalyzing PAP, although the effect of sodium was weaker than that of lithium (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Xiong <em>et al.</em> 2004</a>). The depletion of FRY1 activity by lithium in wild-type plants may induce accumulation of 3′ extension transcripts through suppression of XRN exoribonuclease activities including XRN3 (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). To test this hypothesis, accumulation of some 3′ extensions and pri-miRNA_3′ under some abiotic stresses in wild-type seedlings was examined using quantitative RT-PCR. Accumulation of 3′ transcripts was strongly induced by LiCl stress in all cases and by NaCl stress in some cases (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>), regardless of accumulation of 5′ transcripts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS7.pdf">Figure S7</a>). This result supports the idea that depletion of FRY1 activity leads to accumulation of the 3′ transcripts by following suppression of XRN activities.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/487/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lithium induces accumulation of 3&#x2032; extension transcripts. (A) Model for inhibition of FIERY1 activity by lithium. (B) Quantitative RT-PCR analysis of 3&#x2032; extensions and pri-miRNA_3&#x2032;s under some stressful conditions. Wild-type seedlings were soaked in water, 200 mM LiCl, 200 mM NaCl, 100 &#x3BC;M abscisic acid (ABA), or 300 mM mannitol solutions, and incubated for 5 hr." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30288879" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Lithium induces accumulation of 3&#x2032; extension transcripts. (A) Model for inhibition of FIERY1 activity by lithium. (B) Quantitative RT-PCR analysis of 3&#x2032; extensions and pri-miRNA_3&#x2032;s under some stressful conditions. Wild-type seedlings were soaked in water, 200 mM LiCl, 200 mM NaCl, 100 &#x3BC;M abscisic acid (ABA), or 300 mM mannitol solutions, and incubated for 5 hr.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/487/F7.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/2/4/487/F7.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/487/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/487/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461379" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-46" class="first-child">Lithium induces accumulation of 3′ extension transcripts. (A) Model for inhibition of FIERY1 activity by lithium. (B) Quantitative RT-PCR analysis of 3′ extensions and pri-miRNA_3′s under some stressful conditions. Wild-type seedlings were soaked in water, 200 mM LiCl, 200 mM NaCl, 100 μM abscisic acid (ABA), or 300 mM mannitol solutions, and incubated for 5 hr.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-47">Using two genome-wide technologies, tiling arrays and RNA-Seq, we report the identification of transcript extensions from the 3′ ends of thousands of genes as well as 3′ remnants of pri-miRNAs that accumulate in <em>fry1-6</em> mutants. We also observed accumulation of these non-coding RNAs in <em>xrn3-3</em> mutants. These studies may suggest that XRN3 and FRY1 act cooperatively to eliminate two kinds of non-coding RNAs in plants: 3′ extensions from transcribed mRNAs and 3′ remnants of pri-miRNAs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1/FigureS8.pdf">Figure S8</a>). This finding is supported by the fact that accumulation of the 3′ transcripts was induced in wild-type plant under high-salinity stresses that are inhibitors of FRY1 activity (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Most notably, this is the first report of endogenous XRN3-specific activity in plants and genome-wide base resolution identification of thousands of 3′ gene transcript extensions.</p><div id="sec-19" class="subsection"><h3>Gene transcript extensions are equivalent to 3′ read-through products</h3><p id="p-48">In yeast, the exoribonuclease Rat1 degrades 3′ read-through products that are transcribed by RNA polymerase II after endonucleolytic cleavage of the mRNA at the poly(A) site, and transcription terminates upon complete degradation by Rat1 (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2004</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">West <em>et al.</em> 2004</a>). This Rat1-mediated reaction is closely coupled with transcription. It is very likely that the 3′ extensions detected in <em>fry1-6</em> mutants are equivalent to such 3′ read-through products and that only XRN3 (not XRN2 or XRN4) exclusively possesses this Rat1 function which can terminate the transcription reaction in <em>Arabidopsis</em>, regardless of the fact that XRN2 and XRN4 are also Rat1 orthologs (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Kastenmayer and Green 2000</a>). This indicates that some yeast Rat1 functions are divided into two proteins, XRN2 and XRN3, in <em>Arabidopsis</em>.</p></div><div id="sec-20" class="subsection"><h3>XRN2 and XRN4 are not fully redundant with XRN3 for all functions</h3><p id="p-49">It was previously reported that XRN3 acts redundantly with XRN2 and XRN4 to suppress RNA silencing, to eliminate some loop structures of pri-miRNAs, and to process pre-ribosomal RNAs (<a id="xref-ref-12-14" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Zakrzewska-Placzek <em>et al.</em> 2010</a>). Interestingly, XRN2, another Rat1 ortholog that localizes to the nucleus together with XRN3, as well as cytoplasmic XRN4 compensate poorly for reduced XRN3 activity in the turnover of 3′ extensions, although it is likely that they redundantly act with XRN3 in a limited number of cases (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C</a>, <em>eg</em> At3g45160). This observation supports the idea that elimination of unnecessary 3′ transcripts is the primary role of XRN3.</p><p id="p-50">RNA-Seq data showed that the numbers of 3′ extension transcripts deregulated in <em>xrn3xrn4</em> (392) and <em>xrn2xrn3</em> (473) are lower than those deregulated in the <em>xrn3-3</em> single mutant (528). This indicates that the three <em>xrn</em> mutations genetically interfere with each other’s function(s) in an unknown manner. Interestingly, previous studies have shown that whereas the <em>xrn2xrn3</em> double mutant is infertile (<a id="xref-ref-12-15" class="xref-bibr" href="#ref-12">Gy <em>et al.</em> 2007</a>), the <em>xrn2xrn3xrn4</em> triple mutant is, in fact, fertile (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hirsch <em>et al.</em> 2011</a>). These findings may be partly consistent with our results from qRT-PCR experiments, which reveal that <em>xrn2</em> and <em>xrn4</em> mutations can, in some cases, enhance or suppress the effect of the <em>xrn3-3</em> mutation on the accumulation of 3′ extension transcripts.</p><p id="p-51">On the other hand, it is likely that XRN2, XRN4, or other exoribonucleases redundantly act with XRN3 to eliminate 3′ remnants of pri-miRNAs, because we could not detect significantly increased accumulation of pri-miR156c_3′, pri-miR158a, or pri-miR164b in qRT-PCR analysis. Furthermore, RNA-Seq data showed that some pri-miRNAs accumulated in the <em>xrn2xrn4</em> double mutant compared with wild-type. This observation provides support for the idea that, like <em>S. cerevisiae</em> Rat1, XRN3-mediated degradation of 3′ extensions is probably coupled with transcription and that degradation of 3′ remnants of pri-miRNAs is not necessarily coupled with transcription. If DCL1-mediated cleavage of pri-miRNA occurs before transcription terminates, XRN3 may capture the 5′ end of the 3′ remnant, whereas if DCL1-mediated cleavage occurs after transcription terminates, other exoribonucleases may participate in this degradation.</p></div><div id="sec-21" class="subsection"><h3>Characteristics of the mRNAs with 3′extensions</h3><p id="p-52">Many, but not all, mRNAs showed 3′ extensions in the <em>xrn</em> or <em>fry1</em> mutants. When we examined whether mRNAs tailed with 3′ extensions possess specific characteristics, we noticed a significant number of the mRNAs with 3′ extensions had relatively higher RPKM values compared with all expressed genes. These data revealed that actively transcribed genes tend to possess 3′ extensions.</p><p id="p-53">Importantly, mRNAs with 3′ extensions showed comparable expression in wild-type and <em>fry1-6</em>, regardless of increased accumulation of 3′ extensions. This result indicates that 3′ extensions themselves do not affect the expression levels of the 5′ mRNAs.</p><p id="p-54">It has been proposed that elevated accumulation of PAP inhibits XRN3 activity, as well as XRN2 and XRN4 activities, in <em>fry1</em> and that 3′ extensions being transcribed could partially escape from degradation by reduced XRN3 activity in both <em>fry1</em> and <em>xrn3</em>. However, transcription might terminate at unknown downstream positions, such as poly(A) addition sites. <em>Arabidopsis</em> uses not only the AAUAAA consensus sequence conserved in all eukaryotes but also several other AU-rich consensus sequences as poly(A) signals that often appear in intergenic regions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Loke <em>et al.</em> 2005</a>). This mechanism of alternative transcription termination could define the length of 3′ extensions in <em>fry1</em> and <em>xrn3</em> mutant backgrounds. Further studies are needed to identify why some mRNAs may possess 3′ extensions whereas others do not in these mutants.</p></div><div id="sec-22" class="subsection"><h3>The role of XRN3- and FRY1-mediated non-coding RNA suppression</h3><p id="p-55">The RNA-binding proteins FCA and FPA are known as positive regulators of the floral transition by downregulating expression of the MADS box floral repressor FLC (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Koornneef <em>et al.</em> 1991</a>). A recent study using tiling analysis detected 3′ read-through signals at the 3′ ends of several genes in an <em>fcafpa</em> double mutant (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Sonmez <em>et al.</em> 2011</a>). However, these findings differ from our study in that the 3′ extensions we identified are generally independent molecules from 5′ mRNAs, whereas 3′ read-through products in <em>fcafpa</em> mutants are likely connected directly with mRNAs.</p><p id="p-56"><a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Sonmez <em>et al.</em> (2011)</a> also found accumulation of transcripts of one transposon (At1TE93275) that was associated with reduced DNA methylation in <em>fcafpa</em>. Our study also revealed much greater accumulation of this antisense transcript originating from this transposon in <em>fry1-6</em>, <em>xrn2-4</em>, and <em>xrn4-6</em> compared with those in other backgrounds, as well as strongly reduced DNA methylation in <em>fry1-6</em>, indicating that regulation of this transposon is quite different from that of 3′ extensions, which is primarily mediated by XRN3 activity. In future studies, it will be interesting to reveal the relationship between FRY1- and XRN3-mediated transcription termination and the RNA-binding proteins FCA and FPA.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We are grateful for the gift of the <em>xrn2-1xrn3-3</em>, <em>xrn2-1xrn4-6</em>, and <em>xrn3-3xrn4-6</em> double mutants from Herve Vaucheret and Allison Mallory (French National Institute for Agricultural Research/INRA). We also thank Mathew G. Lewsey and Jose M. Alvarez for their comments on this manuscript. This work was supported by grants from the Division of Chemical Sciences, Geosciences, and Biosciences, Office of Basic Energy Sciences of the U.S. Department of Energy through Grant DE-FG02-04ER15517 for funding the molecular genetic and RNA-Seq analyses, and the National Science Foundation (NSF MCB-1122246) for supporting the MethylC-Seq analysis, the Gordon and Betty Moore Foundation (GBMF), and the Howard Hughes Medical Institute (HHMI) to J.R.E. and by a grant from RIKEN Plant Science Center to M.S. Y.K. is supported by Japanese Society for the Promotion of Science (JSPS) Postdoctoral Fellowship for Research Abroad. R.J.S. was supported by an National Institutes of Health (NIH) National Research Service Award (NRSA) postdoctoral fellowship (F32-HG004830). M.D.S. was supported by a National Science Foundation IGERT training grant (DGE-0504645). J.R.E. is an HHMI-GBMF investigator.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001362/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Arrays and sequences have been submitted to the GEO and SRA databases at NCBI as GSE32977, GSE34654, and SRA049100.</p></li><li class="fn" id="fn-5"><p id="p-58"><em>Communicating editor: C. S. Pikaard</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-10-06"><span class="received-label">Received </span>October 6, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-02-11"><span class="accepted-label">Accepted </span>February 11, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Kurihara <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.1" data-doi="10.1126/science.1086391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stepanova</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leisse</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide insertional mutagenesis of <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAlonso%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D301%26rft.issue%253D5633%26rft.spage%253D653%26rft.epage%253D657%26rft.atitle%253DGenome-Wide%2BInsertional%2BMutagenesis%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1086391%26rft_id%253Dinfo%253Apmid%252F12893945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDEvNTYzMy82NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.2" data-doi="10.1016/j.febslet.2007.05.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behm-Ansmant</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashima</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rehwinkel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauliere</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">mRNA quality control: an ancient machinery recognizes and degrades mRNAs with nonsense codons</span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">581</span>: <span class="cit-fpage">2845</span>–<span class="cit-lpage">2853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DBehm-Ansmant%26rft.auinit1%253DI.%26rft.volume%253D581%26rft.issue%253D15%26rft.spage%253D2845%26rft.epage%253D2853%26rft.atitle%253DmRNA%2Bquality%2Bcontrol%253A%2Ban%2Bancient%2Bmachinery%2Brecognizes%2Band%2Bdegrades%2BmRNAs%2Bwith%2Bnonsense%2Bcodons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2007.05.027%26rft_id%253Dinfo%253Apmid%252F17531985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2007.05.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17531985&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247479900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.3" data-doi="10.1038/nrg2179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrington</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Specialization and evolution of endogenous small RNA pathways</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">884</span>–<span class="cit-lpage">896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChapman%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D884%26rft.epage%253D896%26rft.atitle%253DSpecialization%2Band%2Bevolution%2Bof%2Bendogenous%2Bsmall%2BRNA%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2179%26rft_id%253Dinfo%253Apmid%252F17943195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943195&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250281000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.4" data-doi="10.1016/j.cell.2007.10.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chekanova</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reverdatto</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide high-resolution mapping of exosome substrates reveals hidden features in <em>Arabidopsis</em> transcriptome</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1340</span>–<span class="cit-lpage">1353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChekanova%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D131%26rft.issue%253D7%26rft.spage%253D1340%26rft.epage%253D1353%26rft.atitle%253DGenome-wide%2Bhigh-resolution%2Bmapping%2Bof%2Bexosome%2Bsubstrates%2Breveals%2Bhidden%2Bfeatures%2Bin%2Bthe%2BArabidopsis%2Btranscriptome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.056%26rft_id%253Dinfo%253Apmid%252F18160042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18160042&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252217200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.5" data-doi="10.1111/j.1365-3040.2010.02218.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The bifunctional abiotic stress signaling regulator and endogenous RNA silencing suppressor FIERY1 is required for laternal root formation</span>. <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2180</span>–<span class="cit-lpage">2190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DChen%26rft.auinit1%253DH.%26rft.volume%253D33%26rft.issue%253D12%26rft.spage%253D2180%26rft.epage%253D2190%26rft.atitle%253DThe%2Bbifunctional%2Babiotic%2Bstress%2Bsignalling%2Bregulator%2Band%2Bendogenous%2BRNA%2Bsilencing%2Bsuppressor%2BFIERY1%2Bis%2Brequired%2Bfor%2Blateral%2Broot%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2010.02218.x%26rft_id%253Dinfo%253Apmid%252F20807376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2010.02218.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20807376&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284166500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.6" data-doi="10.1007/s12374-009-9021-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiba</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">mRNA degradation machinery in plants</span>. <abbr class="cit-jnl-abbrev">J. Plant Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BBiol.%26rft.volume%253D52%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12374-009-9021-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12374-009-9021-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264838600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.7" data-doi="10.1093/emboj/16.23.7184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dichtl</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Lithium toxicity in yeast is due to the inhibition of RNA processing enzymes</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">7184</span>–<span class="cit-lpage">7195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DDichtl%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D23%26rft.spage%253D7184%26rft.epage%253D7195%26rft.atitle%253DLithium%2Btoxicity%2Bin%2Byeast%2Bis%2Bdue%2Bto%2Bthe%2Binhibition%2Bof%2BRNA%2Bprocessing%2Benzymes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.23.7184%26rft_id%253Dinfo%253Apmid%252F9384595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTYvMjMvNzE4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.8" data-doi="10.1038/nrm2104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garneau</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilusz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilusz</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The highways and byways of mRNA decay</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DGarneau%26rft.auinit1%253DN.%2BL.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D113%26rft.epage%253D126%26rft.atitle%253DThe%2Bhighways%2Band%2Bbyways%2Bof%2BmRNA%2Bdecay.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2104%26rft_id%253Dinfo%253Apmid%252F17245413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17245413&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.9" data-doi="10.1126/science.1101092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gazzani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrenson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodward</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sablowski</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A link between mRNA turnover and RNA interference in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">1046</span>–<span class="cit-lpage">1048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGazzani%26rft.auinit1%253DS.%26rft.volume%253D306%26rft.issue%253D5698%26rft.spage%253D1046%26rft.epage%253D1048%26rft.atitle%253DA%2BLink%2BBetween%2BmRNA%2BTurnover%2Band%2BRNA%2BInterference%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101092%26rft_id%253Dinfo%253Apmid%252F15528448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTY5OC8xMDQ2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.10" data-doi="10.1046/j.1365-313X.1999.00385.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gil-Mascarell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Coronado</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belles</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The <em>Arabidopsis HAL2</em>-like gene family includes a novel sodium-sensitive phosphatase</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DGil-Mascarell%26rft.auinit1%253DR.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D373%26rft.epage%253D383%26rft.atitle%253DThe%2BArabidopsis%2BHAL2-like%2Bgene%2Bfamily%2Bincludes%2Ba%2Bnovel%2Bsodium-sensitive%2Bphosphatase.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1999.00385.x%26rft_id%253Dinfo%253Apmid%252F10205895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1999.00385.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10205895&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079406200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.11" data-doi="10.1016/j.devcel.2008.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urich</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonti-Filippini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A link between RNA metabolism and silencing affecting <em>Arabidopsis</em> development</span>. <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">854</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DGregory%26rft.auinit1%253DB.%2BD.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D854%26rft.epage%253D866%26rft.atitle%253DA%2Blink%2Bbetween%2BRNA%2Bmetabolism%2Band%2Bsilencing%2Baffecting%2BArabidopsis%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2008.04.005%26rft_id%253Dinfo%253Apmid%252F18486559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2008.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18486559&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256652200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.12" data-doi="10.1105/tpc.107.055319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gy</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasciolli</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauressergues</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morel</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gombert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>Arabidopsis</em> FIERY1, XRN2, and XRN3 are endogenous RNA silencing suppressors</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3451</span>–<span class="cit-lpage">3461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.055319%26rft_id%253Dinfo%253Apmid%252F17993620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8zNDUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.13" data-doi="10.1371/journal.pone.0016724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisp</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayle</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel fry1 allele reveals the existence of a mutant phenotype unrelated to 5′-&gt;3′ exoribonuclease (XRN) activities in Arabidopsis thaliana roots</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e16724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHirsch%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De16724%26rft.epage%253De16724%26rft.atitle%253DA%2Bnovel%2Bfry1%2Ballele%2Breveals%2Bthe%2Bexistence%2Bof%2Ba%2Bmutant%2Bphenotype%2Bunrelated%2Bto%2B5%2527-%2526gt%253B3%2527%2Bexoribonuclease%2B%2528XRN%2529%2Bactivities%2Bin%2BArabidopsis%2Bthaliana%2Broots.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016724%26rft_id%253Dinfo%253Apmid%252F21304819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304819&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.14" data-doi="10.1016/j.devcel.2009.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hornyik</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terzi</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">G. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Spen family protein FPA controls alternative cleavage and polyadenylation of RNA</span>. <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DHornyik%26rft.auinit1%253DC.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D213%26rft.atitle%253DThe%2Bspen%2Bfamily%2Bprotein%2BFPA%2Bcontrols%2Balternative%2Bcleavage%2Band%2Bpolyadenylation%2Bof%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2009.12.009%26rft_id%253Dinfo%253Apmid%252F20079695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2009.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20079695&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275100400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.15" data-doi="10.1016/j.cell.2009.01.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houseley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The many pathways of RNA degradation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHouseley%26rft.auinit1%253DJ.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D763%26rft.epage%253D776%26rft.atitle%253DThe%2Bmany%2Bpathways%2Bof%2BRNA%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.01.019%26rft_id%253Dinfo%253Apmid%252F19239894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.01.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19239894&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263688200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.16" data-doi="10.1073/pnas.97.25.13985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kastenmayer</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Novel features of the XRN-family in Arabidopsis: evidence that AtXRN4, one of several orthologs of nuclear Xrn2p/Rat1p, functions in the cytoplasm</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">13985</span>–<span class="cit-lpage">13990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKastenmayer%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D97%26rft.issue%253D25%26rft.spage%253D13985%26rft.epage%253D13990%26rft.atitle%253DNovel%2Bfeatures%2Bof%2Bthe%2BXRN-family%2Bin%2BArabidopsis%253A%2BEvidence%2Bthat%2BAtXRN4%252C%2Bone%2Bof%2Bseveral%2Borthologs%2Bof%2Bnuclear%2BXrn2p%252FRat1p%252C%2Bfunctions%2Bin%2Bthe%2Bcytoplasm%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.25.13985%26rft_id%253Dinfo%253Apmid%252F11106401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjUvMTM5ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.17" data-doi="10.1111/j.1365-313X.2008.03770.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Arnim</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>FIERY1</em> regulates light-mediated repression of cell elongation and flowering time via 3′(2’),5′-bisphosphate nucleotidase activity</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKim%26rft.auinit1%253DB.%2BH.%26rft.volume%253D58%26rft.issue%253D2%26rft.spage%253D208%26rft.epage%253D219%26rft.atitle%253DFIERY1%2Bregulates%2Blight-mediated%2Brepression%2Bof%2Bcell%2Belongation%2Band%2Bflowering%2Btime%2Bvia%2Bits%2B3%2527%25282%2527%2529%252C5%2527-bisphosphate%2Bnucleotidase%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2008.03770.x%26rft_id%253Dinfo%253Apmid%252F19077168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2008.03770.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19077168&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265040700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.18" data-doi="10.1038/nature03041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasiljeva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rando</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nedea</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The yeast Rat1 exonuclease promotes transcription termination by RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKim%26rft.auinit1%253DM.%26rft.volume%253D432%26rft.issue%253D7016%26rft.spage%253D517%26rft.epage%253D522%26rft.atitle%253DThe%2Byeast%2BRat1%2Bexonuclease%2Bpromotes%2Btranscription%2Btermination%2Bby%2BRNA%2Bpolymerase%2BII.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03041%26rft_id%253Dinfo%253Apmid%252F15565157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15565157&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225322100050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.19" data-doi="10.1007/BF00264213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koornneef</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanhart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Veen</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Agenetic and physiological analysis of late flowering mutants in Arabiodopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">229</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DKoornneef%26rft.auinit1%253DM.%26rft.volume%253D229%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D66%26rft.atitle%253DA%2Bgenetic%2Band%2Bphysiological%2Banalysis%2Bof%2Blate%2Bflowering%2Bmutants%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00264213%26rft_id%253Dinfo%253Apmid%252F1896021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00264213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1896021&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GF17600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.20" data-doi="10.1073/pnas.0403115101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurihara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Arabidopsis</em> micro-RNA biogenesis through Dicer-like1 protein functions</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">12753</span>–<span class="cit-lpage">12758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403115101%26rft_id%253Dinfo%253Apmid%252F15314213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM0LzEyNzUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.21" data-doi="10.1073/pnas.0808902106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurihara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsui</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishida</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide suppression of aberrant mRNA-like noncoding RNAs by NMD in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">2453</span>–<span class="cit-lpage">2458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808902106%26rft_id%253Dinfo%253Apmid%252F19181858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzcvMjQ1MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.22" data-doi="10.1016/j.cell.2008.03.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonti-Filippini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Highly integrated single-base resolution maps of the epigenome in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLister%26rft.auinit1%253DR.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D523%26rft.epage%253D536%26rft.atitle%253DHighly%2Bintegrated%2Bsingle-base%2Bresolution%2Bmaps%2Bof%2Bthe%2Bepigenome%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.03.029%26rft_id%253Dinfo%253Apmid%252F18423832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.03.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18423832&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255503900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.23" data-doi="10.1038/nature09798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelizzola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kida</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nery</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hotspots of aberrant epigenomic reprogramming in human induced pluripotent stem cells</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLister%26rft.auinit1%253DR.%26rft.volume%253D471%26rft.issue%253D7336%26rft.spage%253D68%26rft.epage%253D73%26rft.atitle%253DHotspots%2Bof%2Baberrant%2Bepigenomic%2Breprogramming%2Bin%2Bhuman%2Binduced%2Bpluripotent%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09798%26rft_id%253Dinfo%253Apmid%252F21289626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21289626&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287924100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.24" data-doi="10.1016/j.molcel.2007.10.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crevillen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baurle</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swiezawski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The <em>Arabidopsis</em> RNA-binding protein FCA requires a lysine-specific demethylase 1 homolog to downregulate <em>FLC</em></span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLiu%26rft.auinit1%253DF.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D398%26rft.epage%253D407%26rft.atitle%253DThe%2BArabidopsis%2BRNA-binding%2Bprotein%2BFCA%2Brequires%2Ba%2Blysine-specific%2Bdemethylase%2B1%2Bhomolog%2Bto%2Bdownregulate%2BFLC.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.10.018%26rft_id%253Dinfo%253Apmid%252F17996704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.10.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17996704&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250776800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.25" data-doi="10.1126/science.1180278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marquardt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swiezewski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Targeted 3′ processing of antisense transcripts triggers <em>Arabidopsis FLC</em> chromatin silencing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180278%26rft_id%253Dinfo%253Apmid%252F19965720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjcvNTk2MS85NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.26" data-doi="10.1104/pp.105.060541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loke</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahlberg</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strenski</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Compilation of mRNA polyadenylation signals in Arabidopsis revealed a new signal element and potential secondary structures</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.060541%26rft_id%253Dinfo%253Apmid%252F15965016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM4LzMvMTQ1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.487.27" data-doi="10.1016/j.pbi.2010.11.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lucia</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Long non-coding RNAs and chromatin regulation</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DDe%2BLucia%26rft.auinit1%253DF.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D168%26rft.epage%253D173%26rft.atitle%253DLong%2Bnon-coding%2BRNAs%2Band%2Bchromatin%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2010.11.006%26rft_id%253Dinfo%253Apmid%252F21168359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2010.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21168359&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.28" data-doi="10.1093/pcp/pcn101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsui</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishida</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morosawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mochizuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaminuma</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Arabidopsis</em> transcriptome analysis under drought, cold, high-salinity and ABA treatment conditions using a tiling array</span>. <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcn101%26rft_id%253Dinfo%253Apmid%252F18625610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGNlbGxwaHlzIjtzOjU6InJlc2lkIjtzOjk6IjQ5LzgvMTEzNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.29" data-doi="10.1038/nrm1310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maquat</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nonsense-mediated mRNA decay: splicing, translation and mRNP dynamics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DMaquat%26rft.auinit1%253DL.%2BE.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D89%26rft.epage%253D99%26rft.atitle%253DNonsense-mediated%2BmRNA%2Bdecay%253A%2Bsplicing%252C%2Btranslation%2Band%2BmRNP%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1310%26rft_id%253Dinfo%253Apmid%252F15040442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15040442&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188854500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.30" data-doi="10.1073/pnas.0605528103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olmedo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nourizadeh</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar-Henonin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>ETHYLENE-INSENSITIVE5</em> encodes a 5′→3′ exoribonuclease required for regulation of the EIN3-targeting F-box proteins EBF1/2</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">13286</span>–<span class="cit-lpage">13293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605528103%26rft_id%253Dinfo%253Apmid%252F16920797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM2LzEzMjg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roman</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubarsky</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kieber</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothenberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic analysis of ethylene transduction in Arabidopsis thaliana: five novel mutant loci integrated into a stress response pathway</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1393</span>–<span class="cit-lpage">1409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRoman%26rft.auinit1%253DG.%26rft.volume%253D139%26rft.issue%253D3%26rft.spage%253D1393%26rft.epage%253D1409%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bethylene%2Bsignal%2Btransduction%2Bin%2BArabidopsis%2Bthaliana%253A%2Bfive%2Bnovel%2Bmutant%2Bloci%2Bintegrated%2Binto%2Ba%2Bstress%2Bresponse%2Bpathway.%26rft_id%253Dinfo%253Apmid%252F7768447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8zLzEzOTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.32" data-doi="10.1261/rna.2467911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rymarquis</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souret</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence that XRN4, an <em>Arabidopsis</em> homolog of exoribonuclease XRN1, preferentially impacts transcripts with certain sequences or in particular functional categories</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2467911%26rft_id%253Dinfo%253Apmid%252F21224377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE3LzMvNTAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.33" data-doi="10.1016/j.pbi.2011.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput approaches for plant epigenetic studies</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DSchmitz%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D130%26rft.epage%253D136%26rft.atitle%253DHigh-throughput%2Bapproaches%2Bfor%2Bplant%2Bepigenomic%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2011.03.010%26rft_id%253Dinfo%253Apmid%252F21470901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2011.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21470901&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.34" data-doi="10.1016/j.cub.2009.10.076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedoroff</span>  <span class="cit-name-given-names">N. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RNA secondary structure determinants of miRNA precursor processing in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSong%26rft.auinit1%253DL.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D41%26rft.atitle%253DRNA%2Bsecondary%2Bstructural%2Bdeterminants%2Bof%2BmiRNA%2Bprecursor%2Bprocessing%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.10.076%26rft_id%253Dinfo%253Apmid%252F20015653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.10.076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20015653&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273697500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.35" data-doi="10.1073/pnas.1105334108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonmez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baurle</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magusin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laubinger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA 3′ processing functions of <em>Arabidopsis</em> FCA and FPA limit intergenic transcription</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">8508</span>–<span class="cit-lpage">8513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1105334108%26rft_id%253Dinfo%253Apmid%252F21536901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzIwLzg1MDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.36" data-doi="10.1016/j.molcel.2004.06.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souret</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kastenmayer</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">AtXRN4 degrades mRNA in <em>Arabidopsis</em> and its substrates include selected miRNA targets</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSouret%26rft.auinit1%253DF.%2BF.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D183%26rft.atitle%253DAtXRN4%2Bdegrades%2BmRNA%2Bin%2BArabidopsis%2Band%2Bits%2Bsubstrates%2Binclude%2Bselected%2BmiRNA%2Btargets.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.06.006%26rft_id%253Dinfo%253Apmid%252F15260969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15260969&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222958300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.37" data-doi="10.1111/j.1365-313X.2005.02470.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinozaki</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Tiling array-driven elucidation of transcriptional structures based on mazimum-likelihood and Markov models</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DToyoda%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.issue%253D4%26rft.spage%253D611%26rft.epage%253D621%26rft.atitle%253DTiling%2Barray-driven%2Belucidation%2Bof%2Btranscriptional%2Bstructures%2Bbased%2Bon%2Bmaximum-likelihood%2Band%2BMarkov%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02470.x%26rft_id%253Dinfo%253Apmid%252F16098113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02470.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16098113&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230889200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.38" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Tophat: discovering splice junctions with RNA-Seq</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.39" data-doi="10.1038/nature10118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Dijk</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d’Aubenton-Carafa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gourvennec</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwapisz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">XUTs are a class of Xrn1-sensitive antisense regulatory non-coding RNA in yeast</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253Dvan%2BDijk%26rft.auinit1%253DE.%2BL.%26rft.volume%253D475%26rft.issue%253D7354%26rft.spage%253D114%26rft.epage%253D117%26rft.atitle%253DXUTs%2Bare%2Ba%2Bclass%2Bof%2BXrn1-sensitive%2Bantisense%2Bregulatory%2Bnon-coding%2BRNA%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10118%26rft_id%253Dinfo%253Apmid%252F21697827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21697827&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292461300056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.40" data-doi="10.1016/j.cell.2009.01.046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voinnet</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Origin, biogenesis, and activity of plant microRNAs</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DVoinnet%26rft.auinit1%253DO.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D669%26rft.epage%253D687%26rft.atitle%253DOrigin%252C%2Bbiogenesis%252C%2Band%2Bactivity%2Bof%2Bplant%2BmicroRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.01.046%26rft_id%253Dinfo%253Apmid%252F19239888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.01.046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19239888&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263688200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.41" data-doi="10.1038/nature03035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gromak</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proudfoot</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Human 5′→3′ exonuclease Xrn2 promotes transcription termination at co-transcriptional cleavage sites</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">522</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWest%26rft.auinit1%253DS.%26rft.volume%253D432%26rft.issue%253D7016%26rft.spage%253D522%26rft.epage%253D525%26rft.atitle%253DHuman%2B5%2527%2B--%253E%2B3%2527%2Bexonuclease%2BXrn2%2Bpromotes%2Btranscription%2Btermination%2Bat%2Bco-transcriptional%2Bcleavage%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03035%26rft_id%253Dinfo%253Apmid%252F15565158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15565158&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225322100051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.42" data-doi="10.1101/gad.891901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishitani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title"><em>FIERY1</em> encoding an inositol polyphosphate 1-phosphatase is a negative regulator of abscisic acid and stress signaling in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1971</span>–<span class="cit-lpage">1984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DXiong%26rft.auinit1%253DL.%26rft.volume%253D15%26rft.issue%253D15%26rft.spage%253D1971%26rft.epage%253D1984%26rft.atitle%253DFIERY1%2Bencoding%2Ban%2Binositol%2Bpolyphosphate%2B1-phosphatase%2Bis%2Ba%2Bnegative%2Bregulator%2Bof%2Babscisic%2Bacid%2Band%2Bstress%2Bsignaling%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.891901%26rft_id%253Dinfo%253Apmid%252F11485991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzE1LzE5NzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.43" data-doi="10.1111/j.1365-313X.2004.02225.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A single amino acid substitution in the Arabidopsis FIERY1/HOS2 protein confers cold signaling specificity and lithium tolerance</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">536</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DXiong%26rft.auinit1%253DL.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D536%26rft.epage%253D545%26rft.atitle%253DA%2Bsingle%2Bamino%2Bacid%2Bsubstitution%2Bin%2Bthe%2BArabidopsis%2BFIERY1%252FHOS2%2Bprotein%2Bconfers%2Bcold%2Bsignaling%2Bspecificity%2Band%2Blithium%2Btolerance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2004.02225.x%26rft_id%253Dinfo%253Apmid%252F15500469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2004.02225.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15500469&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224632700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.44" data-doi="10.1126/science.1088305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dale</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinn</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Empirical analysis of transcriptional activity in the <em>Arabidopsis</em> genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">842</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYamada%26rft.auinit1%253DK.%26rft.volume%253D302%26rft.issue%253D5646%26rft.spage%253D842%26rft.epage%253D846%26rft.atitle%253DEmpirical%2BAnalysis%2Bof%2BTranscriptional%2BActivity%2Bin%2Bthe%2BArabidopsis%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1088305%26rft_id%253Dinfo%253Apmid%252F14593172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDIvNTY0Ni84NDIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.45" data-doi="10.1093/nar/gkq172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakrzewska-Placzek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souret</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobczyk</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kufel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Arabidopsis thaliana</em> XRN2 is required for primary cleavage in the pre-ribosomal RNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">4487</span>–<span class="cit-lpage">4502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq172%26rft_id%253Dinfo%253Apmid%252F20338880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xMy80NDg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.46" data-doi="10.1016/j.pbi.2011.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA-directed DNA methylation</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DH.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D142%26rft.epage%253D147%26rft.atitle%253DRNA-directed%2BDNA%2Bmethylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2011.02.003%26rft_id%253Dinfo%253Apmid%252F21420348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2011.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21420348&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.487.47" data-doi="10.1016/j.cell.2006.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yazaki</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundaresan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cokus</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide high-resolution mapping and functional analysis of DNA methylation in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZhang%26rft.auinit1%253DX.%26rft.volume%253D126%26rft.issue%253D6%26rft.spage%253D1189%26rft.epage%253D1201%26rft.atitle%253DGenome-wide%2Bhigh-resolution%2Bmapping%2Band%2Bfunctional%2Banalysis%2Bof%2BDNA%2Bmethylation%2Bin%2Barabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.08.003%26rft_id%253Dinfo%253Apmid%252F16949657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16949657&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240897600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/4/487.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/4/483" title="Contrasting Mutation Rates from Specific-Locus and Long-Term Mutation-Accumulation Procedures" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/4/499" title="Hypopigmentation and Maternal-Zygotic Embryonic Lethality Caused by a Hypomorphic Mbtps1 Mutation in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/4"><span>Volume 2 Issue 4, April 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/4.cover-source.jpg?itok=0lh-7MFD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/487" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bhXaWEHQzXo-_eN4SPKyXNytdsEEEoqcqhz-M7kIfYk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/4/487" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/4/487" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/4/487" class="active" data-icon-position="" data-hide-link-title="0">Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in Arabidopsis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459266" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-DA4V-TDiZFFxCggLL_24FKKpY7wmFz8B2IVTZkkfxWk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/4/487.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/4/487" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-08D_Su6F8P5w8uMgZT1M1-PiBZ7OxhRhs38Z07p38oM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/4/487" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/4/487.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459266" id="node459266--2" data-pisa="ggg;2/4/487" data-pisa-master="ggg;2/4/487" data-apath="/ggg/2/4/487.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in <em>Arabidopsis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yukio Kurihara</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Schmitz</span>, <span class="highwire-citation-author" data-delta="2">Joseph R. Nery</span>, <span class="highwire-citation-author" data-delta="3">Matthew D. Schultz</span>, <span class="highwire-citation-author" data-delta="4">Emiko Okubo-Kurihara</span>, <span class="highwire-citation-author" data-delta="5">Taeko Morosawa</span>, <span class="highwire-citation-author" data-delta="6">Maho Tanaka</span>, <span class="highwire-citation-author" data-delta="7">Tetsuro Toyoda</span>, <span class="highwire-citation-author" data-delta="8">Motoaki Seki</span> and <span class="highwire-citation-author" data-delta="9">Joseph R. Ecker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">487-498;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001362 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459266--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yukio Kurihara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yukio%2BKurihara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurihara%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYukio%2BKurihara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Schmitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BSchmitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmitz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BSchmitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joseph R. Nery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BNery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nery%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BR.%2BNery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew D. Schultz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BD.%2BSchultz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schultz%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BD.%2BSchultz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Emiko Okubo-Kurihara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emiko%2BOkubo-Kurihara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Okubo-Kurihara%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmiko%2BOkubo-Kurihara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Taeko Morosawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taeko%2BMorosawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morosawa%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaeko%2BMorosawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Maho Tanaka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maho%2BTanaka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanaka%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaho%2BTanaka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Tetsuro Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>RIKEN Plant Science Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tetsuro%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATetsuro%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Motoaki Seki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Motoaki%2BSeki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seki%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMotoaki%2BSeki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Joseph R. Ecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Bioinformatics and Systems Engineering Division, RIKEN Yokohama Institute, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BEcker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ecker%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BR.%2BEcker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecker@salk.edu" class="" data-icon-position="" data-hide-link-title="0">ecker@salk.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/4/487" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/4/487&amp;title=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/4/487&amp;title=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/4/487&amp;title=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/4/487&amp;text=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/4/487&amp;title=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/4/487&amp;t=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/4/487&amp;title=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/4/487&amp;title=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459266/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459266" id="node459266--3" data-pisa="ggg;2/4/487" data-pisa-master="ggg;2/4/487" data-apath="/ggg/2/4/487.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Surveillance of 3′ Noncoding Transcripts Requires FIERY1 and XRN3 in <em>Arabidopsis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yukio Kurihara</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Schmitz</span>, <span class="highwire-citation-author" data-delta="2">Joseph R. Nery</span>, <span class="highwire-citation-author" data-delta="3">Matthew D. Schultz</span>, <span class="highwire-citation-author" data-delta="4">Emiko Okubo-Kurihara</span>, <span class="highwire-citation-author" data-delta="5">Taeko Morosawa</span>, <span class="highwire-citation-author" data-delta="6">Maho Tanaka</span>, <span class="highwire-citation-author" data-delta="7">Tetsuro Toyoda</span>, <span class="highwire-citation-author" data-delta="8">Motoaki Seki</span> and <span class="highwire-citation-author" data-delta="9">Joseph R. Ecker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">487-498;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001362 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459266--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yukio Kurihara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yukio%2BKurihara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurihara%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYukio%2BKurihara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Schmitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BSchmitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmitz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BSchmitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joseph R. Nery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BNery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nery%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BR.%2BNery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew D. Schultz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BD.%2BSchultz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schultz%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BD.%2BSchultz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Emiko Okubo-Kurihara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emiko%2BOkubo-Kurihara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Okubo-Kurihara%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmiko%2BOkubo-Kurihara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Taeko Morosawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taeko%2BMorosawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morosawa%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaeko%2BMorosawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Maho Tanaka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maho%2BTanaka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanaka%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaho%2BTanaka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Tetsuro Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>RIKEN Plant Science Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tetsuro%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATetsuro%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Motoaki Seki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, California 92037, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Motoaki%2BSeki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seki%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMotoaki%2BSeki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Joseph R. Ecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Laboratory</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Genomic Analysis Laboratory, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Bioinformatics and Systems Engineering Division, RIKEN Yokohama Institute, Yokohama, Kanagawa 230-0045, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BEcker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ecker%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BR.%2BEcker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecker@salk.edu" class="" data-icon-position="" data-hide-link-title="0">ecker@salk.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=487&amp;rft.epage=498&amp;rft.atitle=Surveillance%20of%203%E2%80%B2%20Noncoding%20Transcripts%20Requires%20FIERY1%20and%20XRN3%20in%20Arabidopsis&amp;rft.volume=2&amp;rft.issue=4&amp;rft.date=2012-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kurihara%2C+Yukio&amp;rft.au=Schmitz%2C+Robert+J.&amp;rft.au=Nery%2C+Joseph+R.&amp;rft.au=Schultz%2C+Matthew+D.&amp;rft.au=Okubo-Kurihara%2C+Emiko&amp;rft.au=Morosawa%2C+Taeko&amp;rft.au=Tanaka%2C+Maho&amp;rft.au=Toyoda%2C+Tetsuro&amp;rft.au=Seki%2C+Motoaki&amp;rft.au=Ecker%2C+Joseph+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459266/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459266/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459266/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459266/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459266/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459266/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459266/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459266/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459266/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459266/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459266/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459266/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/4/487" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/4/487.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/4/487.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/4/487.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/4/487.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
