<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/4/429" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/4/429.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/4/429.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/4/429.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/4/429/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/4/429.atom" />
<meta name="HW.pisa" content="ggg;2/4/429" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Common Dataset for Genomic Analysis of Livestock Populations" />
<meta name="DC.Identifier" content="10.1534/g3.111.001453" />
<meta name="DC.Date" content="2012-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Cleveland et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Although common datasets are an important resource for the scientific community and can be used to address important questions, genomic datasets of a meaningful size have not generally been available in livestock species. We describe a pig dataset that PIC (a Genus company) has made available for comparing genomic prediction methods. We also describe genomic evaluation of the data using methods that PIC considers best practice for predicting and validating genomic breeding values, and we discuss the impact of data structure on accuracy. The dataset contains 3534 individuals with high-density genotypes, phenotypes, and estimated breeding values for five traits. Genomic breeding values were calculated using BayesB, with phenotypes and de-regressed breeding values, and using a single-step genomic BLUP approach that combines information from genotyped and un-genotyped animals. The genomic breeding value accuracy increased with increased trait heritability and with increased relationship between training and validation. In nearly all cases, BayesB using de-regressed breeding values outperformed the other approaches, but the single-step evaluation performed only slightly worse. This dataset was useful for comparing methods for genomic prediction using real data. Our results indicate that validation approaches accounting for relatedness between populations can correct for potential overestimation of genomic breeding value accuracies, with implications for genotyping strategies to carry out genomic selection programs." />
<meta name="DC.Contributor" content="Matthew A. Cleveland" />
<meta name="DC.Contributor" content="John M. Hickey" />
<meta name="DC.Contributor" content="Selma Forni" />
<meta name="article:published_time" content="2012-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="A Common Dataset for Genomic Analysis of Livestock Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Although common datasets are an important resource for the scientific community and can be used to address important questions, genomic datasets of a meaningful size have not generally been available in livestock species. We describe a pig dataset that PIC (a Genus company) has made available for comparing genomic prediction methods. We also describe genomic evaluation of the data using methods that PIC considers best practice for predicting and validating genomic breeding values, and we discuss the impact of data structure on accuracy. The dataset contains 3534 individuals with high-density genotypes, phenotypes, and estimated breeding values for five traits. Genomic breeding values were calculated using BayesB, with phenotypes and de-regressed breeding values, and using a single-step genomic BLUP approach that combines information from genotyped and un-genotyped animals. The genomic breeding value accuracy increased with increased trait heritability and with increased relationship between training and validation. In nearly all cases, BayesB using de-regressed breeding values outperformed the other approaches, but the single-step evaluation performed only slightly worse. This dataset was useful for comparing methods for genomic prediction using real data. Our results indicate that validation approaches accounting for relatedness between populations can correct for potential overestimation of genomic breeding value accuracies, with implications for genotyping strategies to carry out genomic selection programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/04/01" />
<meta name="citation_mjid" content="ggg;2/4/429" />
<meta name="citation_id" content="2/4/429" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/4/429" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/4/429.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/4/429.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/4/429.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001453" />
<meta name="citation_pmid" content="22540034" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="4" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="429" />
<meta name="citation_lastpage" content="435" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Matthew A. Cleveland" />
<meta name="citation_author_institution" content="Genus plc, Hendersonville, Tennessee 37075" />
<meta name="citation_author_email" content="matthew.cleveland@genusplc.com" />
<meta name="citation_author" content="John M. Hickey" />
<meta name="citation_author_institution" content="School of Environmental and Rural Science, University of New England, Armidale NSW 2351, Australia" />
<meta name="citation_author" content="Selma Forni" />
<meta name="citation_author_institution" content="Genus plc, Hendersonville, Tennessee 37075" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. A. Clark;citation_author=J. M. Hickey;citation_author=J. H. J. van der Werf;citation_title=Different models of genetic variation and their effect on genomic evaluation;citation_pages=18;citation_volume=48;citation_year=2011" />
<meta name="citation_reference" content="citation_author=M. A. Cleveland;citation_author=S. Forni;citation_author=D. J. Garrick;citation_author=N. Deeb;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=P. Bijma;citation_author=J. A. Woolliams;citation_title=Inbreeding in genome-wide selection.;citation_pages=369-376;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076474" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Public Health;citation_journal_abbrev=AJPH;citation_author=T. R. Dawber;citation_author=G. F. Meadors;citation_author=F. E. Moore;citation_title=Epidemiological Approaches to Heart Disease: The Framingham Study;citation_pages=279-286;citation_volume=41;citation_year=1951;citation_issue=3;citation_pmid=14819398;citation_doi=10.2105/AJPH.41.3.279" />
<meta name="citation_reference" content="citation_author=N. Deeb;citation_author=S. Forni;citation_author=M. A. Cleveland;citation_year=2010" />
<meta name="citation_reference" content="citation_author=N. Deeb;citation_author=M. A. Cleveland;citation_author=A. Mileham;citation_author=C. D. Gladney;citation_author=N. Yu;citation_year=2011" />
<meta name="citation_reference" content="citation_author=R. Fernando;citation_author=D. Garrick;citation_year=2009" />
<meta name="citation_reference" content="citation_author=S. Forni;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=N. Deeb;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Forni;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=M. E. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. González-Recio;citation_author=S. Forni;citation_title=Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.;citation_pages=7-7;citation_volume=43;citation_year=2011;citation_pmid=21329522;citation_doi=10.1186/1297-9686-43-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Habier;citation_author=J. Tetens;citation_author=F. Seefried;citation_author=P. Lichtner;citation_author=G. Thaller;citation_title=The impact of genetic relationship information on genomic breeding values in German Holstein cattle.;citation_pages=5-5;citation_volume=42;citation_year=2010;citation_pmid=20170500;citation_doi=10.1186/1297-9686-42-5" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M.E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_title=A phasing and imputation method for pedigreed populations that results in a single stage genomic evaluation method;citation_author=J. M. Hickey;citation_author=B. P. Kinghorn;citation_author=B. Tier;citation_author=J. H. J. van der Werf;citation_author=M. A. Cleveland;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=The International HapMap Consortium;citation_title=The International HapMap Project.;citation_pages=789-796;citation_volume=426;citation_year=2003;citation_issue=6968;citation_pmid=14685227;citation_doi=10.1038/nature02168" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=T. Luan;citation_author=J. A. Woolliams;citation_author=S. Lien;citation_author=M. Kent;citation_author=M. Svendsen;citation_title=The accuracy of genomic selection in Norwegian Red cattle assessed by cross-validation;citation_volume=183;citation_year=2009;citation_pmid=19704013;citation_doi=10.1534/genetics.109.107391" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M.E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Misztal;citation_author=A. Legarra;citation_author=I. Aguilar;citation_title=Computing procedures for genetic evaluation including phenotypic, full pedigree, and genomic information;citation_pages=4648-4655;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700728;citation_doi=10.3168/jds.2009-2064" />
<meta name="citation_reference" content="citation_author=The 1000 Genomes Project Consortium;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. M. Ramos;citation_author=R. P. M. A. Crooijmans;citation_author=N. A. Affara;citation_author=A. J. Amaral;citation_author=A. L. Archibald;citation_title=Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.;citation_pages=e6524-e6524;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19654876;citation_doi=10.1371/journal.pone.0006524" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. Van Tassell;citation_author=G. R. Wiggans;citation_author=T.S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Common Dataset for Genomic Analysis of Livestock Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
<meta name="twitter:description" content="Although common datasets are an important resource for the scientific community and can be used to address important questions, genomic datasets of a meaningful size have not generally been available in livestock species. We describe a pig dataset that PIC (a Genus company) has made available for comparing genomic prediction methods. We also describe genomic evaluation of the data using methods that PIC considers best practice for predicting and validating genomic breeding values, and we discuss the impact of data structure on accuracy. The dataset contains 3534 individuals with high-density genotypes, phenotypes, and estimated breeding values for five traits. Genomic breeding values were calculated using BayesB, with phenotypes and de-regressed breeding values, and using a single-step genomic BLUP approach that combines information from genotyped and un-genotyped animals. The genomic breeding value accuracy increased with increased trait heritability and with increased relationship between training and validation. In nearly all cases, BayesB using de-regressed breeding values outperformed the other approaches, but the single-step evaluation performed only slightly worse. This dataset was useful for comparing methods for genomic prediction using real data. Our results indicate that validation approaches accounting for relatedness between populations can correct for potential overestimation of genomic breeding value accuracies, with implications for genotyping strategies to carry out genomic selection programs." />
<meta name="og-title" property="og:title" content="A Common Dataset for Genomic Analysis of Livestock Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/4/429" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Although common datasets are an important resource for the scientific community and can be used to address important questions, genomic datasets of a meaningful size have not generally been available in livestock species. We describe a pig dataset that PIC (a Genus company) has made available for comparing genomic prediction methods. We also describe genomic evaluation of the data using methods that PIC considers best practice for predicting and validating genomic breeding values, and we discuss the impact of data structure on accuracy. The dataset contains 3534 individuals with high-density genotypes, phenotypes, and estimated breeding values for five traits. Genomic breeding values were calculated using BayesB, with phenotypes and de-regressed breeding values, and using a single-step genomic BLUP approach that combines information from genotyped and un-genotyped animals. The genomic breeding value accuracy increased with increased trait heritability and with increased relationship between training and validation. In nearly all cases, BayesB using de-regressed breeding values outperformed the other approaches, but the single-step evaluation performed only slightly worse. This dataset was useful for comparing methods for genomic prediction using real data. Our results indicate that validation approaches accounting for relatedness between populations can correct for potential overestimation of genomic breeding value accuracies, with implications for genotyping strategies to carry out genomic selection programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
    <title>A Common Dataset for Genomic Analysis of Livestock Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8Ey-kYiTRtKJxbiQFvGkTVG9KJFdXzKY_QsaFWQh4Dw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459263","apath":"\/ggg\/2\/4\/429.atom","pisa":"ggg;2\/4\/429","ac":{"\/ggg\/2\/4\/429.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/4\/429.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/4\/429"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/4\/429"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/4\/429":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w0CprmcC5dBaWiLmaLEZefuTXP+7wV7OElqWKZk4PlMYey83HM3alJcOi2Vr2PCuL\\\\\\\/vCQry6ig4v3Cbdr3Z6iQsDJqg5NZNR+OBgAF91aMScPdjpWC3NssMZTyLLrXOBGU\\\\\\\/08emZuMYwAGMhFyxFe+qvqTjLF0R787yQs1fnIvFCRo4Hh2EQK0p4GJOtDWj32ubhdQdFyikN3AEhQgtHlB9HvSw9jutMU75IT98IO8MVvcP4Og29sqV1bXdln6+rck8DXhC14ctm4eF4IQE3Jusf\\\\\\\/5wzSeeR5V+13qsWsEh0V1gH\\\\\\\/JfEZ6JB8axDOnrLiFXTSbWfmV\\\\\\\/0v1L41XUR4ETcwRBuzwqWRnk\\\\\\\/iKcFxXr+\\\\\\\/d21i9B0r4FyvaHsQjIiMgGJ2JDkbRAKuXh3V7a4wYJRBE8gVrm+FWS7fWkgEPPtp27y3q4bB6HQIhYWK7HjpKjh0tOZfgnWUePEAAOt69BYek8pz4hupK+cHznjgaT3fzyoJlp7synzekIhgmEenZ9q\\\\\\\/nACDzONRFIeEt6BDOlw\\\\\\\/WBlsKsrWsjMbPQrtj+UXD9Km\\\\\\\/43OWQrpC3xvNeRVZ9tk9YQbFTIhWTk6SOvleAz4TnlfHQ16tkFPDaTvXTBVCbux6UfV0OBoWE7Xh6Q0Wphw6IwFXUv0inf3Q4h8rxQCZIbTU5z0N+FGE6bGCg9LrjsWgxpms\\\\\\\/cGfo7UbYxni67BJCC68kc\\\\\\\/tVXWsbBZXhauguei+iqGGPSrY19pRUL\\\\\\\/Zw0N\\\\\\\/XnIju9\\\\\\\/7diWDZjESNA1krGVqbsSj\\\\\\\/MnzUJf5533+wk9G\\\\\\\/KIv1upGwW+TaTs+l2P+VvMZyeH+52KTC5iNOfq4CEW6BIr69dGVOEVeKoGoMrtBMQbyy2SgoCJ3T+avms+Dj\\\\\\\/iJVs+03EjdMCYeLy\\\\\\\/hYnMt5rtAdV18odT87l0hCmc1eS6m4qkdYcq5NUvP6RSd\\\\\\\/CMbnJR2X+PQsCoHlf1YUD25GEcTUqu\\\\\\\/S4KhHtDtdJybhMydz0tlFpMoP6pmTwGjuYrTZisjV7BxuB4EWllTHHnVLQZDcy6hI1+pspQZAJh92IwXZWs7cl1Bg69izHJ4a+\\\\\\\/SIlxw1szHRMmt0A8+tK2r98rDY4D44puNAh+z0oaaSHNCj34AVNIa9SfOxXkBTJc\\\\\\\/88z+meBUb9AvYIVhIx5gsTpJG4RWk+zzf7butcIGKY\\\\\\\/K7VpKSTRVbNClvRk8S+fRz8gVCuaeMW2NEDj5HXLEZiKX\\\\\\\/8m9pxXPncn0+el84KA\\\\\\\/RC39571A0NgxWJJBUPSStW0MQ6mYzA\\\u0022,\\\u0022iv\\\u0022:\\\u0022BemecRZxVOmjNNTqKbhsLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224f41eb89575f94823ccacc09d7f4139d4105bc4bb624ad2cd17144ea7ca9594c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L5lPwKoYWjFmy0Iep3ryHtgDbWcH8yD0yimqU++qt\\\\\\\/G69UipYO1PeCk12z2d3E0i70laoH3IOX9dfTk7N4veR2cm2N2FZTwKRq9X47c5jZ8XdV4MsWek3eQlE6gkzpgNCsYNNUSHXWPrcgRdgvy67yyCqwY8U6GSf80gwnKHmqqATq1xn9+nNzb4TFhO7WofhfVZN4c+z3VO28PYSrcXB0CkVSq9h8rmCR95UIqUnoQO0MalX7CVfya91rV5K\\\\\\\/JjQv+pyTx+jjg3GpDrItFk6uYVE6wctz6BRjwVy8bEbXHpa6\\\\\\\/AWVWiaTHBr7DTL6DLAn92whpyk1rMmz5ZLwZt2tnhGYhuV3F3QkNfMXavsgUZ9D+jg9UOk8KGY+YyU9bHXrC9+JNK9jZJ+Bx4hzDqRVSUlZTEjyXvB6cGKefIVyjbOm4H1PZjwOXYW9EpeYO4A+52e9yx25vNcaH8aYidoGHWY3FolwvyYxsxzdqfVqa4sklAFhKufI4P9IWPTgV94Vspf8r9FgiuyzJhjtDdz0ybA6CUmO1FigxCgmJ7cU78TL3RZv5bWHclB9ygSANh2ttXRPN9UeISmEmrMDQJ2ahwrT8rxSLBnLwV92nkbDhRSCpTovEUf4rcTR8axSM5yExSguyWGXK3WKbRBzUA2x\\\\\\\/NzUONrg74XuyovoqSj2+ue\\\\\\\/RqCqVT3Isld87C+uuqJ19hhq5RY4vuKvDmpcnLDTSOm+vReoix84TVpW06YArvtgjZJ5Hd36O4WHHvGcTMjQATnVpohilRPi50x8lVB364JSB2j1cPFvkEKIP9U6YkzBcAs+kgeL2aYSnfBW7wJ78mC5HM5MbxZo8IxjMR2vvF70gqIK8hTEJeXnC7oGeI5WCP93pJPZkuGAwnq2tT2rvcEKqAS6reSdLItH3ARVLcWu\\\\\\\/KYph4OfZ+nybdlIYwzbTjTaMRjpUcZgubVgktNbBCclCqBNwt+6mMtdrBgTSk9jsVBLbvcoi10WuRZxng6CxRoK9tYnxdRipWCACWI2ZJsyMoJPTLBbwJGBk98y+2j1gK+zo5bSY3nqb8\\\\\\\/00\\\\\\\/5kJovBvLyk1GHdyxhOocuEP8lPzGBjGcgXMnxArmB3e9xfiYF0QzufzSImUJonLuXggAWGnroMUXQN6SWHYX4PqCatDNdZwM7pQnaf5xqXnRXBD9vui8uHhSH4Rqv8qAkRS2vH0AO1fKhYzHWfOaiHAVWr\\\\\\\/tkwtjocsz6H2vhwlRYssE07E776sKPWsb9yFqG87VRFGqotmihurbr8uGynzNwhpV1UXe0VNYy0IZVNKehp5JCTK185MOAW5f3QpLmfap5ZBXKIWo59dbrSEUh0aGiLvY8WhflMRSf7BrggSUwWK3qVuOHOYTbYf4xWAvpN2KM0q+aLrOTqCMR6z1mKOLgxqrApOtacTCI2NEHP2hGg+Q9Nl1FlJiwsrvqo9CQ1VVWUklqBeR+y7nKg9PG41pxcvWiEfvYN\\\\\\\/gAfuJNMiHZU2ncg9bqmI7tziUefoG4Dipf\\\\\\\/2GE3O0E\\\\\\\/IkgLyFTY0jrtRfOaQtxxQ0sFKxT8eK5eLe3zGd71xorDS7AoRSKC1HhCnDHVYHVQm0kkvSm7GOjVn84hhmhZouYHDkyBK+7t2T66D\\\\\\\/7zcdan6D4+4RqnEUIMlFdlzeifhCSt\\\\\\\/eTzjI3YA4TFPgRNzOH2uUQ0pVDh211IQPk\\\\\\\/eGS22UUgEPXZSg9MvpuZAHNRKppZq6SVGMY9xZaU5R9dBwETxD8CIEB3vV3tBifRydNb7XM0JiT4G7qbAXUwMvykfbaz3hLLGyx5hihwAAlP2rguLNjybwqTnNEOP9ZEH+UgOAwm1xlw\\\\\\\/Vqh53R0FGGeE38VXspQ4qemW\\\\\\\/b5Uc04WFIB53hKKTPQikww63d5N+HMt7RdqJYV33OzmrhgR3Dh2zjI2bxQxoRQ4NcjbK5uCGHF5D7nk0u2kBcbJKz7qPQgnOF\\\\\\\/aF4nB\\\\\\\/pj92FrNOctQDAx6ormWnuJr2x11mjyiaGqOJWjtyFWs\\\\\\\/yDPHj4LYviQCvk6XqfZhuoFzbbE05lNbA2d+\\\\\\\/106IbfdgXSSFMbaRaebPcNKlKhkIJZwdznd6t2EGcDQv1rygX\\\\\\\/VUKbDf0\\\\\\\/b\\\\\\\/ObuPFGXaiWmm0umaY1W4C+3aFgOV7Zt6c58E1xwItaxRBV4P3oANuFsJjhJVCVPuLFJXptetjzHdjxlW34oz6WfGlzPmm3h6XFQ4r4GaYV95b5+qu0Rc1AV4hYrEfzBjWCMSZJ5vj9duiPfeXcsjK77RUj3PLpnwa3q\\\\\\\/xmCeumhzTqmTRZ4HOj7J\\\\\\\/BrDJuBd\\\\\\\/TfEPH0hcE3NNa9hf7Y71ZOHiJ7QcD8GQsNVNc8akX5OD0Iut\\\\\\\/lSueQ\\\\\\\/SGeWLpXaKQj67Hpd\\\\\\\/4456gYidP+RBEcfnPO+hbKFQRMkTTl0jvUvq1Hj25hk9g4II6gvBtbJs0pY2WNfTEM3xZ5GwEQR0qohk+G0C2Fb8AMX\\\\\\\/JiYfsbs5eaOpzY0lFtFiakUOOB4FLNL2S0e1+5fB5KFun4qdmZcYE8T4R\\\\\\\/cWYMfD6S5PEJ6R51urlh3NymiLHy4SrinzQ\\\\\\\/0CcupmsurX3RTqCqrJx1LVsCX5b0kmtr\\\\\\\/wAKlmHOtLmysQzjiVABWpU6d8cO5cjvjyrn0BZF+9hyfPitIv9EHzZoV5OU8N71sMesXVIMoOEHQsrHxqRJnDvRXE3RvWQEQGDR5\\\\\\\/cz3g\\\\\\\/X3dcrAFrQWwb\\\\\\\/XMqCXFp058nD8FsqHoxOT+\\\\\\\/FwPH45uK35YUVNEGQnten666udAojb3ban60WPx29FO9PEOrhLBGPKKzu3\\\\\\\/PAVc4QnqIVpuZYgBxd7gGUpm46MreK2TMWB+AMCJRBKV4N0fL2nA5yF+bVvi8IcCzMB+N99Zbe6MSqyA\\\\\\\/jVmd550b06j4wKxHK1GKrelroXmidubihxG5YST5UKd\\\\\\\/pZ7cyNc9+PceI0kv0J8I7blfYMciUlly\\\\\\\/qsBzHRjRWkqCIXncz1tRgm1Blv8eTdgmGGYxk12NxIn3N4p2Xjkfh7Z5kmmICbfvvvITpzBeVppod4E1j2omTVkxtdLuD0U2zvzqOrbldgmZ2uvzJ9aWyg8H0dC1dlp+rORVRFO38NUt0IZ7yAkAXd7pSmDD5hXJoSE+qpM4HXCGkWJ8mKVF1lNDWnuRhJ0uK2n\\\\\\\/bBlrbsOwdYkqCLsGMONIjBIz5dg\\\\\\\/lgiWmx0OxQPXdvi1jCOl+uloYe0l6NbmFTqiWwQBAbeiztIA4B1czj+luriAPReh9G09o3mfJI0McdSKu5Zwccnt+pTtBp\\\\\\\/QbvBQsrQSB52hkuv29rgNV8NpU8VkYfZyxtsypiYCNjVtnqDw52MRjTE36LaeUfF2F\\\\\\\/FBLhmUmBoUxOqRlE4qwRpQrN53+rNAARgVn7IE480747xUcFsR7OlFRr7rRbfpfBJ3eBqWrS\\\\\\\/nz7GgWSDMzqXaKdIAbhN7O7\\\\\\\/xmSKmHUF8cjaUdbx4Xzhk8MhPdI3chIWKSe322TBtDHDgUWecKFkawxRO6WDAEkhMzMMfd7OgRjoziQfJLTOrkIahnva4N6nScXPhymILuzL7e1MilbeQ0JXFocTnqUpgclgtSRgHbah+LV2fEA6BVqitmFOgx0m+scxpJWaJ9C+DsaIcf8KKQ3qI2bS0LMqZ\\\\\\\/JfGAlBDGat2O8LfSiESms920F1sYoIdDvv1lBhduFQtzuIuZpQJfbQwiooYhXNkfbel8Zmt2hqoqaQ3FUx5ouj8asCD+hgH8WlKQ49Kt+6KMUQOx0h2DXq1xO5sy7Rf+xdIJufHThh23UkCYwTk4KX5EOvq+tS35cFOEOZnlEXGptY+9mlJpMUlW6ZZiTJssGZOIZ7kqa1eQB78lVjh002xrQaK4lzs3S3gnYYreTodLF5+d8NNDIRId9jkINFUL4hFTdMsS4OPwNZWh0uO351StAo16SSlHqU6u\\\\\\\/wWQ8jXJoSdaQL7iLFQPMG7YeERp5OQBhAT3MNPcgayLcpTbt7jeFIm5BvDvHfm2ONB3hJqYfJ8ak2V\\\\\\\/Qhtrk3mfNNBp+ydXPaKL0lhy+ehsHK7vGcaXz3j995NwB0cOHa7acC6X06phQx8zHdZ6XMWAk\\\\\\\/7\\\\\\\/ikEx9KRZqLZTvzmkEVIgtNwA9t6IJsuZdrellOLRRhOs5i2dTIMhpSlh7bsE4g8l2B6ZYSGNWJbXcc\\\\\\\/mQQq9besQAETDuwS3AsYKNFC00v8eNhKAvHXOslxljKUazXrQMg0KUv3RcxwCIbFngJzmINR4q+V5XgF4j0+hsC3g22ZYrNPfJ6LIJdLzdZFv2v61ySZsKJLAv\\\\\\\/27rMuH1IoFaCz6jyvNlbiQ4kF87fhdfpErfIhNyaegW9gd0J3jySc2JEPia40x8yZL4eeL1flr1GCZo0zD8SIFQmRC\\\\\\\/+2v8v4OUSJLXUYQ4ONuHnilDyrx4lQ8H7aIKzjUx36yXdVaJ5334aknTVnCLNV6NKhbWhTNIReDEud5vUjg0kcbxtTDPLwl5UvbLUF301D4gJq6bDoyeVbT6Gg27jeUEVQddi\\\\\\\/UdcHGL9q0ODVGsCsNKvBYi5gYSABs9JLSIqp657R\\\\\\\/fOeGszq5rPvezt5ZGpgANi8AhrEZWB2RZHn7O0+X+zPC4cReSikAqIgLJyVcX87dQ09SSUY6dtub4YjtwZiFr3m08DVSCb7yED4aOV+wxIRKxZd79uqEnYkFiUhp2bKspM2rPyXsjCjcGOffZdpAC7\\\\\\\/qd02vfxeyqcOgVXbAYE\\\\\\\/WiEnvh6uX2a81LWNpvC60wrTmL6SnCpGQOanAlXJwLqD7\\\\\\\/uThWf2r2up1nWI2uo1I6PkzlRF9\\\\\\\/vAE3T2oTxtXlgQksjQZPCweb\\\\\\\/bhBiRZrsgAx6yjPXmYZ4c4WB0lCK4ur7ANgL6DEaIpvSNabKzMMEdCvBJQjMA2GAGPJ7N3CdpxW3dAAwPMqoZx9tX7JdMlhZrXz\\\\\\\/pvF6JMdYnqaMlh5+umE3DlDcKLNi9nj8z6GghdwHrUgq5b\\\\\\\/LHaLlJMnQUlEB\\\\\\\/wIv0uasZoqSfWNP6Hlk3axj7JsXxpgBz+ymgz+opDAL\\\\\\\/FSlXwcP04mtYvRiTnOAWtJjrPOpLCnpoLhKX8dlzD4hprJUMTAeKzKkqBXAUHcIW6XOiekNB9d575\\\\\\\/I8XCMFdQ6ZX4rU\\\\\\\/eaIPiwgiRLMZYvUGL3HU3rsoiFdK+IrZNezkMiR03S1rpEll4MP9L+Mu+oLK2HvIU7z3a3q4STM21ZsN0Qo1CCvXdG+846tI0f5YycSkQ813ETFmROTUydOP5wk2BPVx8n8j2GOlKNJdmtaQke\\\\\\\/RwsFB3EvPOrYTws9AFQ0+IGQ\\\\\\\/Q4D64I\\\\\\\/VWzJvz6v+\\\\\\\/huIiTQHVHXDsEPhZSuDb0ZN1rV6A5fvueAWGnr6OygQsr\\\\\\\/ZBQIFg\\\\\\\/v8eLwOEPhm0l27q16dp3Ag2Fs1T56VhqOqZRJaX0Xo3fSTyHCh5lAdKWloM9hNduSJdvOA4DlfYFtT9WJ4mv1PV45NYyKubU6DvQX6LwN5GFtecWnOSn4OS68YaYvXW2RnvfT0LXtOLPlePxmTVhtvwumx+35bXEdUgCbxeGF8S7uKypBhG5GVWavwurAi\\\\\\\/AeAYjAe1nDRqEFpEasWN8Js0ei4hRSbZnC9r+0TLRbJi9hkRnj39crOKzHHLugShLSdhGbLnEiJTGST2xyHBAWgGXOTKXgmKpwPSV7lBQhxmicQLGFUUdXEzDCQgl+a4YyGbzJNdKyAAWz7RxoVYexpujrn0fxH+GeXLdXkwuGXlhtFRWLylj8jW5djt6VaUVt46Fbk0T41vcZU+xpOdOQtRntjJBzMG3RQMITPIiiXkniw9Hsb1GMNEit7nSdluH0cG3m+sLrTHOaIqVH0Cq8ECX7zBrHocWQHjhZAxVwsX\\\\\\\/VraR21aDYEFaNkfR7AoY1AFUYXispozoUdipfz+5ykoY9gt01lDJ8ETcNnDH2oI25W6f3dLNmPvNpHKKzTKPnXyhMQM+mVx6Mm4fl6KXNXPIQ17wHc9PrtbA0F498nyb+YWP0FoL9zFz7w+MXp7UWKZ\\\\\\\/X2S0nNyn28SVZZ3GV28srjQqL6uRj33nWX5S4e4lar\\\\\\\/YTolJLZgxvPhpleg1HMvFuSjRUDKuf4TkVopPmUr\\\\\\\/F3Pbbq21lCGAp4pGm8UJlaMJd4S5n+Ezd04WY3mrEf1x\\\\\\\/ansvFlhVg71Lrf+3tBRnosqsPRvJikuHYtv\\\\\\\/F2CIvuAFXqEBrr5I+iuD6gc4QPIKji9MsbgUtfzmsmCmL7OtWJWtsYp8RzohY0qAD07sKlqxOfV2ss\\\\\\\/HEj4\\\\\\\/PEtSCGzatHrY9EBEVVE396n8NGFQ1SejpPODvEPyfKNWliiEljmnCnu1M1R3yqbo6YFzDqdfHnG+sZYsNP532aHOTcRYX5nzOcviMCvdNq5Lk1N3OlMfDx82DB\\\\\\\/FrV\\\\\\\/8qCGrRnC6DFfJL8TXdXeHKgp1G6foRvLYx5Rfk2\\\\\\\/RVKHUkAuGwMqVN+Shtp+NQlIH45fh8azhim2jWEFsQH24UczhqOxxwEwLv9QMyKSH6uthmtgUZ2YwHPdnsEBTzwvtywkJFFtENLHvMoLxs\\\\\\\/jEAEQIhcEWhaun\\\\\\\/ePyjqsFPRvwM2ObAZrK5ZM0PIbc75JAchI8U0rQV5bdxE22qZHLFtzdzPyttlX16gyFNCgNd\\\\\\\/QrbOwjoIxbtTRjO+Duc\\\\\\\/wvlXV1lZ9reqWQCrPctKO67tV21ymYuJKWG48KRg36pDZgpo3Bq76PndeUdflyd+lx5Bz7JfscT5GYGBvUrFyCG6D1j5kskFq8GCBCxITzdqQIS1+E07JRRF8+oLQehCJDrvoAOQJIxc6SFQHymmijQJtiREJXhHgtzr0iBDvyRZgeWW+3ZJrU3yY\\\\\\\/slk9swwJI9qj3vAIJz+3ZouCK7N6tvvupZTVjUDmX1j8S9rNKdkvyP5J\\\\\\\/sl6kW8wq8IPRahBLJM71TH2eljkzZv1gm1BSmZ1xjwatYGRdOg4OsPY6+cye7uCrvPGL9npRri3QZ8748aOSp9kw0cRJBTSezRIdiIDd38lJWkC8vgRDs2F4Drnd9cMgrUgyPSg4tijWMklO2gb\\\\\\\/IccP6FOfctmP92f1FHowQ8snDB3UJCoIK+\\\\\\\/8Bgi7em+Id0W7POUzK\\\\\\\/87TVfLedZhGvGHTiO9KZ\\\\\\\/n6+1uBtNJ50CgLuTTKRWGqO61KqvJNd4Vw1yQM2rKEhVOgyTgVo6whwSrANF1xgK41mA9WBqcSgavuKWrnTT9mwgIt\\\\\\\/DEA30UsgMdXYCT673Xz6mqnzE0zwDBnZGDfTfEQ142I6\\\\\\\/m\\\\\\\/EltKkTpjEpzJmXkrtmgf6OVhfe38s1PIiygD8tZbCzd2ofBldXOj59xfufaCUUHg31Sm7q3nmQfsRfudRcTqjzUWI8SpHOUMD1CktrXj2sru07uZRBnbodSlSscaTYOUhVqzPQBhOtuR0u2ZIlIbC+Wy1f399xbYDOX2dBqcnOVqcPUqeadsK32KOCUDxrBVgtEhgn+I14ImaygnqqUybQemtiwtDqpCOihS4eSLyB1AW9SBHAES8qw2bQf5M+6aK\\\\\\\/Sk07vlpx60KBjdWqXxw0nYDl5Gy2kA0urDZ1C7BqUfAA34M2giE3H9aTHqlSr8ianhvfJlZrMqQtfl9vm4OewDk\\\\\\\/RCtv\\\\\\\/axEJWF+xSmC\\\\\\\/l9HIiBIYqZkyvwVUpuF8kS\\\\\\\/ue0r0Ca5sXHMtl6Oifl0BNezMChjkE9Or3PHRUbBLU61euKLuSpFIwxXcEF12BXE3GV+ZdHaAB\\\\\\\/\\\\\\\/IeeFCF9WDcW3YgNmSn0UYgnEfI40Ykf1HlFDFBTKUFJXae\\\\\\\/LEE4ExxMKFmEioLNUMJQ4Xq1rouHdM4loWkwF0dMb6yrHmhsTgHJBlEimRRB0BU6rHNVNWUDxZTgrU39DXtN+obb5ugUhUMyvmG\\\\\\\/TCX02aICNbLibXMNM0mN1vXqJeMuhH1I5P0flOGAax81Lg8ipMIg8IcC14nNDgZoY5xeH8VihAiWGficByMRSR7bWyNX1aTLWuiJHGVH\\\\\\\/IPtQoGl98tDlxVGLrohDxqOaB\\\\\\\/372I7Bmrot9UhFWjSYOVAMtAw6bAxjdn8aL6Cr13MLCSXQihIagsjDFec1vmPAQtkbyboOQ9edn9oha384jDqDpaYyh70FV2uRcaRs3\\\\\\\/mbD5+KRYgU2iswASY6zm\\\\\\\/MeunGQ3rc+UILwf33yRg5J6GYf6C9dv\\\\\\\/fFD1zKSwxAFO4yr+GZA\\\\\\\/710VT9\\\\\\\/ejh1vlMB0iQCrwLIg1zJM6f72Q7JW+NGPvZv2UgU4PvDvQnDq5mWEB3Ughs5\\\\\\\/dqIiWZeI3C6q9ocu1H2ajGMUngKi14o0XJOUzRN0cXrHeYk1JBF45tc41uW0UtVeuVM4LMHYMIz73KdB47mo1nBZwubmU\\\\\\\/GoBLjmPgKgqgNO9oL4eBfDcxyBJHXBtnzqnyq55szLGGMDmwyvo005Y79qot1aPQzLmE7BZBZhNXWusj5QGZGOKeZItFEmmRZUJ6lLqXxEgTWbAExKHGUCYIYUv6EdIbDY6I\\\\\\\/yoNOADYgqNISnMEOd3kVe68ZecNS6Se2hwRFpPSVX\\\\\\\/E1PZ0jzW41Xk\\\\\\\/RS7TSaZKn06Sc0THyPPTiDqbzfrvWdawa1gur7EpfHN1y\\\\\\\/juWl7lQnjqBAeWE9uo6T688fbX9c47mJ0r4d0whU69CgCvRx9k+NYKI2QvC+15CVZUNm1dFyvc4ARVUNUxUBl3VbKB2rXhB4hb0JjaorznknPHh1A6D6BE+BmUQi+wtLFpAHY4zK2axisHZS492sZz54saMU8mMLkK\\\\\\\/WIx0oLiH4ISqcpVOy6YaAtSJd27u30eS58f3Uu4qMAPLfv5iRgKb0og6EzmmgDqJUoO2Qc72w80lJmNGraA13+g1a9dJs8CsSO01kj9DHfcCLW2Shxj7wGJsr5aIQaEeirkJy4+cEDXC5TiVcwE+rLzhnGSynhXrAMTaQJv4lpZopapHK+UYZdtYhlAESV7Dkaavcg3T8NMmfADvqKe7\\\\\\\/zAr2W5ZVR9t6l4CsrMR2jPJ3\\\\\\\/Nz6CxJAhL4KkZ9hgweVqLoU\\\\\\\/3E24Ql8qFpLiLWbO+4\\\\\\\/0g3kduZEClhTw76gWYZhwtDjjyg5yf42ayEESnyGg35XdtJQJTiM3rBiRDCGPcKiYw94o81VrG4O6gtNOctemJ+mt5eP0nvMPxNqqT7fkiHlWGCmIIs\\\\\\\/CW4njINv\\\\\\\/ECRBq+MV\\\\\\\/\\\\\\\/SZe9jmx\\\\\\\/l3jvrI7F8VQhAiVSFI6lqb7sVxOpsMwkA1\\\\\\\/AxNvzR6xnm+cMt1CiULfo9SBPQgiPbr\\\\\\\/K6u\\\\\\\/f3cqvoOAVL\\\\\\\/Pm7pLX9IxfWyH+ATv+PX7IzJOzKwcGc8WlMcqtpCUI\\\\\\\/KGsk8XZqCNty1Vm\\\\\\\/VqRxOn14QRKE4sIC\\\\\\\/kXL4GTcDDnAzBtEeS\\\\\\\/DzCuM8ycKhghLIn\\\\\\\/4OaT+8+bHddn1Db1bitMfNffswF9qwJhxiQw49AH0lVQL9rP7SKZH9xyZkkBmNQJ9zUmlBHKRFYOT62IPtJg7ybLo5BSJ2xDrvAXf6eMOMoab0nTkA4W5xD5x+H382KJ4yXFs4NOhM73\\\\\\\/rQsdbW924tssX4F8288NrhdOi03Apoo\\\\\\\/l3SAEj2iC\\\\\\\/aaGlIyMKEFo9Ms8P2R7IJKHX4WRkZVCP\\\\\\\/EAQ+jPdxQsTz30bNrHPFqDeAsGRATyVMf+GbFq+UFqpiua4g+cAZappIwtxcq38Qlb5g9THsN2ImZQSYqVR8wXP3Nlfj1WGdsHaruBoZfafM2Cl84YKEg8nv7Dc9zwWSh7zTERUwuRwymLslstsevcHrhZaPOohBPDTTCdUsyIuCsmreG5JJjdaW1jlNVduGprTSw0VM45dqPSCu9Gpec7oo4\\\\\\\/hY3ETa3gJola\\\\\\\/Q+W31yvIRbV0nV+ltqUWMtfv5b6Ts1r2Of7bw4+tDPbECmtmmCdF7SDja3YHL7zdHsAxJl04DZS0naKOpcSnCFNf5RltLpCkUqQkxnRLFFwIyrxNwkwU9nt7mO1QwBfKZsT8x1ddED7mrm55a3rDcD+DU8uwoqOad71+m5vCc0cKHOIA5wxwzKk\\\\\\\/4luqVjMA2xTaeqJ26nFUCHwEcBPy5gecwJfr\\\\\\\/r4TA1v+uo43Jtg9Bx3fsle8VhsmysY1I69rb\\\\\\\/jNNR4Mt17xTXTyG9d3PWdtQEESqMnLFeX9wFeRYD6nPzCOEds+b5yOlwvebY0AIFOtF24QULry05amDjHQdEtqI\\\\\\\/\\\\\\\/ChckQ\\\\\\\/NtaQ3Fqt\\\\\\\/W+uVcLspfdUb4YeKn1izsSihrPnCNYSk7rWUckY2hwSWrL\\\\\\\/aAAB2u5fJRLqoD\\\\\\\/\\\\\\\/fzOw3LQf6SkhuAiEu7wSxAGnTWCxHJp2qdM16q3pkLo76pG3F8UhsqEhVjeHmndYJnW03zwxZ3N0p6tPIHkPhWpW1UJKddJpZ1QSXlQBe\\\\\\\/Zex0gFdXNalLMJova7XFjnjO8H14NC1wzJyJ\\\\\\\/zAyCJGujZaI4gwvKs31RMn+2JByLeck64mM8g2JFcy+b1ynx34nMtjqSqVxqx5gtCavjKI2dO1UZO+a\\\\\\\/rbaoT7L6y9Qkn\\\\\\\/aTsdj\\\\\\\/HDiuBEcCxnZfKYghkEkkruQeeIsJT4srMCQbkJ+UucEgsHhLYYvdh5uARlRN1MdCBBYAYFK5a2LRXC2KR9Opf43JrKt2QveFyUzTpoXqFU+qt9h7KY5DOgQ6eV1uNpkaoGWLdNeiZcoAtwymroLqsRlrY2fyfE0a0XKVAhI72zuAz3uJK\\\\\\\/s+XP3g\\\\\\\/TDHcBvGbyGD5ruAbN0mJM0rhd4j6v95A0OOVrWW+GS6X8VlnlAVl15b+\\\\\\\/FKtyCTSx1SPorOBqXCKpijSNk41+hSZHeamFiLsydUMXjeF1XBaEWmBuh9UZneFvtIixlLCGwgbX9CyvMxive30athaRsxy2FFcv\\\\\\\/aXn8VjvRG86vs0v\\\\\\\/p0t6dqL35sHMtZ6NyXVKQm4+vL8hDJy+5Mh6COfsdQUxpGvJc9fRP2ETRTgGlkWKk3B7i5+FXFGJE0O2LoDR+fNb0StesIX5fr98pe4sgNAWMLEAFcrXbJi0Yetrb+0ZvOZZQynRgFusoUZurXpHLTvj7WsVS6Z5tXRsPneJS2E\\\\\\\/7WoRHiz2v6jBkBHhZ0R2tpQtxApTHs1cBFgNgK5+QRWd\\\\\\\/mEZ0Pu+B2kWtNooyq0Oj7zLe+H84E6QIzfebxG7mthlSaKd+xJXTq1IzpQKkTQhdHoe+YIww47B1StXVEpEGrm9Cw9SfIunQSj4gPZgh8s1TibXyh8dDj\\\\\\\/oRLLERKRvlCvwJjGsFk6m67SHsEuvcJuX8nEtsZGSs9BFUEzW9H0p2lcxeCZEMP+sSS4fwDwSGfr5\\\\\\\/0tKiYKFzNuuUolyFZmJWGAgB2+BtIWHr0Y1wtCaSE9I43NuJf+MGblckkZG3hHt8lNMVnSqgrUdKzbGhIUBxL8fQ0uG71YCF9Z\\\\\\\/4xBciPs1KAVt9OHHyPGKnWiL95tvdodzXNjfAr8FJU\\\\\\\/5EoywYykzPUOWCCOZMKxWAudvSgi7fUNBC8sYr80xMiSwbuWOR93lAcSMjVxUp92WqrCrpXktZtUguuK63MEbkmnNR5fm0Mb6mPCNu3YlAtrK3tHTvbUkXZ\\\\\\\/KgaCNFRBScZ3B1zi+x98LiLXGKv8LIeUaiurC\\\\\\\/ZThNxOkws3rv0i3zgzatu9rNSGTZGTwhvCKfCPB9jRnk4Ko6NhGg7h06CcpYqeyYm+vLH8WLLoWSbs8hLgk3E7PpVS14bR94bWSBhA1mUVE3iKgCGg4zlv+bfu05P+gZCUBCy+Lq3djL0FckyXuAOebMul6EkqMCflI5bH3K3WH4PJ3hbWQgtuLsLfVDPJwMK6Xu0Lu6YrR6\\\\\\\/R10vVQ8nSrwo8MUFmEPbyaRltC94\\\\\\\/mw7GvAzM7uBFIUFEFSSEiYwPhXJBxHz6w7ISmJOtNCL5B2T6YrBa2gSrlBwVK4Z94lcot3+yDz6I20GGEwgSEJdIoA1K30pDh3Xb\\\\\\\/PpicT412ZQfp7xD\\\\\\\/J1bnif4IgvdK7VVB8UOl+qdSvP3+ofWyoPRoX7lrpFYZDGtXwvO\\\\\\\/nWtyBhI91XvVT8mdxrKzenzTNwDHJIlwSVnu7WzpjkESYovnWvJZ92piXkC0dTOor8XTdiNa42c\\\\\\\/FFgzgM87LxLbGcZVFiXn0FNriFHYP2Ny77dWdNCHYg71x0N3mMVjARuu6mIEMzYt33NYfUlSyq3WdS2K70XGwJHMe\\\\\\\/BLztFYvbnoS5QkVnFB3QrF1N5lNxI7vNeYWzbixRQ3tInDZJb4nGHeoio0ZE2QoTaqKzUStp4opJPVGeaKulN14ZnZ\\\\\\\/qrrd+6su8Ryj7pT9WCUo3QjkkFwNp15xWUTuNmkiG2y92RxyXV\\\\\\\/9sjno7ZhbmfjjDkVobVY3lhoVF82rE8HeJH+bcODWjuV3Vf5FKea81ltcuxFOFRF3WZwRRumo99VDjNCELugj1K8C7YskiPiDfVgB\\\\\\\/QKR67H0Y0KaYvPkzfbiGdhLBnF3eLnyfnVHEaaTk4sifgnpwSUi2ZHDqeJNbP39PrtP3qJNaxdQP3sBtip9BM9BbDqPs07EE0e9Wy5fD\\\\\\\/Xx74nZIqrvf0F2FhzYi\\\\\\\/jRLFzm9CNBanNm+RScBzgMT1ZC5rTxwxaHanc3x9WCr80T+sVhRhup2e5cnResR0URsc+0ds5VvhNM6DCpHLTLplU0zps1YWrFm2BME4jCye84oW9kDYRbLTJDp+nI+lH6Z1rPhDCRtzYH3aL3Jgd0i1+oVik4WK1\\\\\\\/prgCBGpWeBFTlsIoI9IuPXFVzE0K+64PSkZrcC8sP+0SaEC3JbNRNiWRY3RRoZ6wllRUxQlTao5lIi4Xpo133XPj6tNJjN716es4QEs3vDF+KQagTK+1nJzLEgtUQgd\\\\\\\/741iaZoKW5zmb529jZlhKJ5DcBIZZ3q19vs\\\\\\\/gGH4wyheH\\\\\\\/bbNHRtpYhQlDnELNlocPyUNUI7NkRtIg8Q\\\\\\\/RfGOLFFa31oflKwzhiJIExSE0QVFaiyT4ZP91mmT++ovLPBAQ\\\\\\\/DIE9g9PoTPSMY\\\\\\\/qTVuqGPwb6fktClO7vCc+JCkPFXDD+Rtow++MoL0EJrmpC2+uv4Qo4LJ\\\\\\\/uGi5EJhlxGPgJhlCk8tQ5J7yysd4lp+H7WKqCC96cPdm99co6MwHK7Hq2roKH8lPdparyYhmKXTwYZvEVTAbopW8YOAzqDO1EaeiAqJ5+KY8RRb5Qq+SnFmeZtWRGbC\\\\\\\/UyHQXrCmlHsRIeArmtG98i0eXtkg\\\\\\\/5nH5N9Qx3f0knzndFEnioekalS4n3RiiRJK2iWdASLumR5zvvp11tvaI9GMSgPyMntPfXbJ7vxOYfbQfqB513g\\\\\\\/iVdj7+toFZcL2jdDzE1B\\\\\\\/qN5\\\\\\\/jAYNdyVNeJOf5WdX3PUEVv4fAVuO9okACLqYQlN7nXfSV0QN2BNQh+Y\\\\\\\/DnKJ3G6L+o2hR9KBirGDZsvNSYIxNxyUTcv6YNX\\\\\\\/i\\\\\\\/dHnPLGHtwd0VX48wj23B4Mh\\\\\\\/IeQDd4XC+g0aOvLBgKjpt1Mirnu4MYnJ84Ngx\\\\\\\/6Ex7mrAVD\\\\\\\/OD9SXS9jgZjO2zOnB9U+yqve5XZH63lIAiddDTilkxIE7j94EQdsvSvRaVNyPRw31Et+vL504zZg9qmDLloK4JV8nGwdSxzUMrlDuYrhnCcX1JoFzg5OgyJ\\\\\\\/o\\\\\\\/EaMN7jMXUSap6vWqxzKiGsgu1PniGTlsJSbLyVn+MBVbuzgg8hTFxjFLu4GITHkDrsJExGkuvRvfJYMKIeCwfbUIdeaONK6SuuVm8I3xuJ7TlRUAr44KjNM\\\\\\\/DGqKH\\\\\\\/7y7Zm1PbHZnuyVwmTdDrKTneSuUYi0xv1PtRqVAHZZ2hFDpI83xHUOlJtdXsPx\\\\\\\/ptpQFsIav\\\\\\\/ydcJi9Kxv4\\\\\\\/RVBUY\\\\\\\/47p5Zr5ltVqqOPHUIs65SjgjvF3BfbWIQjJUYftLbcPv3MG7DHGyI7O\\\\\\\/shFR+twVIJX6ENz7Xxb4xrD0p1QdJJ9gd9y\\\\\\\/4xP2DaUGGKVYk96Yu3TVRWMsOkxAIknGGufYuTob2o6WotEJRPPO9KR2CRTgl+mwbj6dTGP5DlaTT\\\\\\\/23wP7lKNLeMfEBxouXUGrPrpfWbF7GBYt\\\\\\\/m8nqz7w8+HaO+BbG90np60G8gRxjL\\\\\\\/4XXdNHHj1fYoHfWt4z1EQkacmc\\\\\\\/Hlbhg5ae53WDQWl6V9RU++qRnDnWNy1oCIZKTS8zrxcI6VPSigsazwerxmR9zGZgf5dCVnHs1mvyh+EwTGBYK+WJUvjLx\\\\\\\/+WfN2Gtq23n8nkQRNK9zJBy0RH7tPiun4yM02yMFsJwMpCiMUT6GiJ5VlIHz45cyZf7Dl2cFy3V4Sxsd8dKxzEAcI1CKF1tDMwdo3FyBrXPhI0N7qSPgX4j75b7xo00cr27bfKb\\\\\\\/dvOO4psSkX77z\\\\\\\/zsqrFrvQ8oehNFKFs62bBcDvyAkayzM4VAAEtvc07qhCULrU5YgkWC0BXvNtxcAnIwOetTQSnWN0pH0\\\\\\\/2FpxV54beHuVQ9+0D8v9RCyaKE7GKxSTZeKd8\\\\\\\/YK8IAP+VMXPukDl+by6tczRYGT9YyFHVH4V4Q2EA8YvvMd27WEys4mLUMi20g3lsmGZPfoDX+nc\\\\\\\/GZPE9Z2h8hGRngs11qJgcQmmUGxHnV8fPLgu5JezFSktGDM6Bizv\\\\\\\/hleZpeR8KQGxVu6i203S1jsoS1dYfLODxKKjgunneJT4DZXBC8dqFG93nlSR0M1utHUQOFBLJmB5hfB3tHQ8aBQlOqUSXGkL3LHLTeNgSM4n1mrZDCTCjkYXCseYPx2aC5aEsj7VlF1ONN69WWdfPeW+FzCD6buqX9CWsrrWeH95mKW2DkI\\\\\\\/aWTV0tyEjvLOg5xRtS2OO\\\\\\\/HBHcKI+bM5vL0WrvPtm1XXEWuuij\\\\\\\/sk3EAaJb8LfbOQNmQplmZivictPWlRQl3DxC2DIWu3D1DQt70SHscfNvRujq2Die2GaapjKJU2Pur3ZBw\\\\\\\/xxCGZUcNhu5wU8n6dd8BcwV48mHJSV+s9wT+Xx+UJegIAByPn5P4Dxf+mlkDV4xZBUzoFWyibqu3jRWGxVQ+HIkbs3nbFK+ZvSOlLxPpAyG8EchMVwxTrXsxi1Y04M\\\\\\\/uGVO7eW7oMDinUsXjTDv3QnMP\\\\\\\/yMiW3YTEyBFshYQlx265rZwvTaSI9xQBMcKOqFgvQLxBW6ZBkzpjY8IFCio+cSJ6QBwTSb713sB9gCY3cKrjDFviDN0qVVwnOqvYsxgikmx1i5KRfKge1LTzIzqb1cr7QOWZK1lC+KYu5PMSbY53ztQOIXBTYcc5We3cPwiwMGMosbglLT01+HwPJqQpmtlgLHJXtGujOrXaY4NQ49gXq66RZ2PcKu1G1xUM1TaZ5Eqx4NjVtIT1Xwss2wc6cEle3a24UygkvTIs6uz5MYxj9skfTilsfFxNxW8XKZ6UvMvPnNt6avhF\\\\\\\/03KWZoFByrFUP8uzoUaaNWXab1GDuNBupvylc9u0Yw+BRiK+gf3cYJZQDLX4CwBDIu0XZY0d+iicoe5iNM70xBmyEhNHRy1WkRLOUtdJN9xVlgsmPO8qjqhxkuZRjYmDCOwUhd+hAkhBEsoxUS92QUnGFrrJO657O92\\\\\\\/B4Vp+AJw7g1ogoj\\\\\\\/ZkB\\\\\\\/obbg\\\\\\\/H5wFD5WWnlGGLG72CcaETao6TFqoLYW\\\\\\\/uv8WhmIbBHBJ4ywmKJQ9Zj4C3HjU6BjiKbTBhzpz\\\\\\\/qr9FvSkEguRAaxhgajxwOcufINeHwX6mJ5gTvyGDDEEFByrn8dJITnMkEZbARJl554Vy3gkLKGN\\\\\\\/P09R0\\\\\\\/4gE3QVEZb54Rlhq\\\\\\\/SRx0SFExJneh0VKkBUeV1QWWtQgYqeMX4S\\\\\\\/vCH140HbK8s\\\\\\\/Q57tQ6lsW7QwlRzz37khd1n03hFxltd0ncq9lCfmi3QRWFvcMjGH3nfXULFpk8DtqGsyml2kBia1ODqIKwPePVzYe+Fi5hddzA0ppQL8lMPHsxl2DFWaU9QuYbbUM9vfwlPgAK+bw2ai87vxoThs1VASYhM5oMxds+0Mfn4cpTPR\\\\\\\/hYI38nxF6Zeyf\\\\\\\/uXre0Df9927vf+NgvLPhYWYNNXwPycE2zbyobwH8+L6FvBl4XdJ7wgx0oII6wzWsGn7XfkGBECZU+ILK8Cwpl\\\\\\\/xlQrdrtEI056MSIUmHnrKVSIJ9gquhkMmaF0kgvlmtYF4S1AZqRhVNm5y3OFENU1gDuaw9C70AIoKez1Y8DchliDrNVYNAMjiop3Q9ojJtw9+KfIED2loj68ECVJBFzYHWVCLcYOiQcT3owxucjyr+ciVPpWV+O4AnwXQouvV3FLrlCWqnSV4l+IlQzfO1uUNF9529KrYOxTWu14\\\\\\\/dcZHhjxCUT4mPZLOC7ys3t3kU2C8ihJbZ8oAPzqJP+Cc+JujXl2nhjaRk76P4GVMg9ZlrF8Opy0HLF03\\\\\\\/+bJ868RXREztLsP7RRnssq10\\\\\\\/EVCIlCpkhgAgtW97pOworsnL9P2ihiytJX38Uu+9HS0FqHpA+0KC8rtJirqHB5laWR5\\\\\\\/6H8XxZDdo4giNulaakjiGCbtfRPychSaPcGqdhafruTYyA4psrh2qLUAni+KEqXTKcTrbJLt6XXlPhRUdfRYIEEd3YizCI1Hl8n2uZpCcLBLZbOHRxgFNiq5nJ2wgj2R1P7gE2gL9zuVVFvUs1YX11+49Q9oc9PDLzsAzdlo+ThS6VYPoE5hlzQNFlZNy9Z3yzMYBVL+Bj+h4YoB2QSMHIo\\\\\\\/\\\\\\\/FHjC\\\\\\\/iKaXzYOMeBykFWDR2CiogsmgIhMNez\\\\\\\/wEUrWNAGSHrkNOhuNHInNaxGosrR1cFBfRxv0hFkUFldFC\\\\\\\/YpcYaMesbi1QQ19+smyPu7I2BfcSwrFPPK0kwnywm+DTn23Mx23JLei8C5W47pu2uxV+coxZxMPr8Zt+pOXnJLQ0bj69bTqf+Gzz2rG9jQ0+4j6QC6jdjX7YtOagUJg9x+O3l4yLev94ug0JRVDz+HW8Ga73a3CdZhTeYJxBT1+yh0XTSYvlV3zQhj+4RkvnQqZ6pzbfEIoNY8LPuMVNTwfJ4L+ojpSKvqPX8Sjar9ZsQmTxuiqd6KRk+eChjL6BoFyri6vY4rg5mVxS3Q6z+S7mCEiblnMHGB5zuSTHlRQCn48NEQCpi7gwLiXdZdZdpjpHbw0xIzXr5Z63WNjpGhKLLD\\\\\\\/kYlQd6kKijpJRN4mv\\\\\\\/inU1dYWpzJmQYVbdeRHNUkBNE40t23hlIyKT53kDiyoWIErd4yvPEvdLIggSFpR9ya7LG++CUwGj+rZoYhgAJXeWzPvWEd6I2naV31Uu\\\\\\\/8+OxFpUdGTegltZI8oJqGMaAIUQM+XtDv4Uwz2ieJNXluUPLLjgTlCGqU4Vjdlq803N18RNsWyuo8M3nCrMAs55DXeYnByqXwMSlmQolHPcEY4UEV81POk4ZPwARdNWyHu12Dje2QbLjUeViXhaq1Xncq2stPPtH5eqLDcgwb7BJSl6zAjKp6gq\\\\\\\/yBKIlPlvozgWpLwSICnK9e5MjpZ5SZWEH+lh\\\\\\\/mLolW8W1gIvMHZuyzUbbR60+E4qhhkk0u1X43jlhFmdHRC\\\\\\\/n0GcN1nDKjfjQxi7ib6+bjWr9uUAC8bdN69C8GGtIyMta346fJd\\\\\\\/mKy1yaPASwEyoE9UdvTmSqRC+gygp4S+G8xizpMsl72ulTd1oI5j9gUioyhNt+ABpAkNj1+lgr0Un1xDc+I1wEg9oTzh7q6bJ3tNHcPnKPzwUrt1nDhYFSRXHcWs9Nh4CymG2X7YNeWgyW5b09y8EBKw4HpgpwgqFu4QON6jnhismnHY\\\\\\\/ENe0FR7A5nXpyHebXNti0wep4j0EFxt+zEt\\\\\\\/7KheM\\\\\\\/1VMZss+xlGZ4bcKC1Z+6dXPn9hKDWAe4Rl1O1e3BrkPEl95E+fzZXTI8RPPXZTkVvAKSmhLTvd+7jkfQarLDv8S28EPWSkaaaaiKnB68CplGv3lZOFybWlpHU68S857J1PduEKm4rGH5kvV2uft3nVA9TSgC4XZodDkjKw63Fd+uGX3bKxyQ35Ap8eDhw2173cJr05M8d5GGmsfPpJTQnpmy5cigD+tC9vmpimrn4XsVI7hlSTYkMey14SMx9ZsTukAqKcpVhXDIOnOVlTFEXdM7rgDQ7ApqcVCOLqHiCJgv5dQVVvdv6mbQg8plZUaAeO7pUmSUyN0hF7ed3yoqUEWLOr29nz41BVM5\\\\\\\/pY4mcMZ4ZQR\\\\\\\/3lhvoLAXpcEG682v8d6dSurUfZGTyMeGAi5xtdlcw4tE6iVDGOR63JkbcgPXaOk7szcVNexiuWJ30qsPfV9CWpZzdfzTV\\\\\\\/tnrCsXpFw0oP7lxf4XwSiBTux4nT6HyKvn5tPS+m3YRfws9vbH5zXB3KYvA94ltYBJvZfUjuBo24vrdXn1V8xy9lOInU774fOrQNnANd16BwPgKd2jAliWf0NYYL\\\\\\\/CcCKWCPRmqmW5ZpIMHfnSnWVf0F6st9qCzgsDV4q0LQ7ibNR7U0g79Z7EhslUnw7RErRRfNdusqC0XgQ6Vb21RH2Hu3IBccxLCpf8JTRMTMpGhG+pQMlSPDf1c7rxFiRQBeXaKzZeX\\\\\\\/Oeb5fZgxKpsI8AA\\\\\\\/fPqbxUpLjU2Lr5zsU01P0OJVoXbKrrxbpDRQFcDkXXPN8hNtBbIrZEEPpdiP0F6D6yvJ2g7gPGc59EaxmbOUQgbrPc60o\\\\\\\/knQYIYwZ8baPgbWB68VDPMEDRfUG9mtFAa9WuVT7232pLr02V+glNrxRtaDBkaUlyoEZWti3HP0DGLmpa5OyMz6o8BK3X7hW1T5lnS5PI\\\\\\\/xfGy9WTrJwOVCfn1vi9tZEKP8QSYTzIT2+ARiSIfDdLAtgV2RBV5\\\\\\\/iQfruByYr1JfdBxOZiiZdZT6mKy\\\\\\\/9z1xv\\\\\\\/30TusSvc5YEgCTEEXkcuS0aJ1Ak8pqOhnRC0weupV441M9pnoE6ZfbF8C304NbZgtWxE5SFnNKet2349gwXGYoiFmN14nW5KVU34TIWUS+bUbFgd8bxCqM\\\\\\\/6x2BgR5M7DR8dpnjy2dGvnpSqWMaka0rz3O7eCL6QwkKj\\\\\\\/BftsLVTxw\\\\\\\/U5noWoWdotnYTxXBvtpFrXiIH\\\\\\\/n+X4GwqbAGBt\\\\\\\/JokceFRhVRoJonFIBOLwK7+ooTHHOUbVRqm\\\\\\\/8By2M694Y2yGzXHx877XQqTMTIdGCk3yLBmNO9hU5haRX606vx8bBX2DxKY6XavseW+llARX3Dbap3dS5X8RCTQGy9Mqlhq6ShZvWyl\\\\\\\/HfAzWQOtJkcWFPTWIQYjT5S8vpcIOSR3ll0HJWyQ6krQxXum5Xm4pNSwvXBTctfMyABjT3YHXDb8CqZiCuSmFsobcNJb3oEv6og3TXbXIcIeympbHgknXehc0+XsfdlBLRlljs4hEIc\\\\\\\/LE7cg7PMcCooBxIXDnvZ0JmRn92zyz\\\\\\\/U8oC7eKB1iKavOzjvRnm1jnpqws1GV5QH4f9HyytQJNWSMJY3idrAaSf5no28gDoPggqK5Kqc7xTgippkyEmTohUbEZS1B1NcCQ\\\\\\\/i\\\\\\\/\\\\\\\/ArqO9lk0tEu6YhER2nFDUNjYvrpG9xCJyHbij7ev65GCwFn69l+lEPhDPDHOHwp94LLqzo0Od3QdYylQGs7jeB+fBs\\\\\\\/dgQfYvxrCELtBesEpAs0laJGq6ac\\\\\\\/Tcqvf56Cp+S2Q81XEVJqRQot3UPHE\\\\\\\/69aJRPG8wc3P1qFDOYM2kYV5xlZZuM0zgbrtWV6oKXf6jDJTy8IshJAEO0yjT1iikykFKYf0bbGa7bemV5fHSUXkSbjcYvofmEAZYNWs2Y\\\\\\\/HvygkZ\\\\\\\/CyEcx3LfK6wsYLpD5YTrI1Fmh2idaGHjnXQefOvwqf90pGpktJyJqQst3prMGJQDuw\\\\\\\/sPmBT7gYNexYDBeCoh3rrB7s9NzpyrjaOlLJYGj264woFMiY\\\\\\\/+winRCUD81uf96eRVxyi9urq7+gF6mVJkCJRg68ylJ8kMk7Nr5oNtxTgGJaJFxH7QqNqwyDR9FrbxOVhUijspWreSC+STPKyAHdDsjMI8QhCKu6NAxIHTc0N4VDQT1RlOjyml7ZIeBC8LNNn4UsWkAJCWQsNHcbOMkG\\\\\\\/gafjz60P9JkB7JnKqrLDxxW1BanyFLcfGcH2NkE6AE1K2WogJ9jWfoZ5fUEunrub\\\\\\\/XK3aA4mCk2\\\\\\\/50nrbcP3ZrkoioD8AZ73aYGXIAh7m7tP9TiQ6vF7xkQmCWYBnUZSbq2sWa5Xrqcx8rF+RBvmao6kI240R9D3J1AmMrzys0qiHp0CXDvK4ZpZGano1uTaogQb3MURSUPIU7R0O9loUXd\\\\\\\/FzEmzLXUCPw\\\\\\\/tz1pk7vR87s0k8pEcV7jn8TfiyKd040Qr3r4Jn1zaJtpBEXQqbIw1b5uc12XPOe2nPx1lLLzZ5qyA0TdwArnuPfQNU0E09pcyjpBqEuhXZy9hOBtIoPONLXvMQs6s9dpONfbYosz4TnJBL7yBUsaAgn2Vc1FjRDFPGnFGA1N9J8F4Wyx90PaYuTL3ms98qqDmbSLFCfbKRKNrs\\\\\\\/G4Fh5\\\\\\\/8zMp3ily8lwo0whor43cp0QR39MD7OJ5l3BL+pli+QCrFyXoQnpkKQZ1K4pckKnPEqFokVbaTzSI8xd3G9wCPbJX3Vnu71F7FGTTQTL4uaelfpMSE4uAGFTh2cNiqDUlLZFrQcZnQS+\\\\\\\/3\\\\\\\/AhszHJth025dUUdS3AVMwq\\\\\\\/kdQeMixac+mCuvwc45+DCXLAcUZ7UI5k0+gEj1poMXPov7FEI84nTMbed\\\\\\\/bRI2SCOL4MSOJHzipq+\\\\\\\/ysu6PZezxBCDEs3VFaV4XGayb6fDGJIBRoPY3JotX2nNs3VU+IKxg+rh1Z+8IWpjQpu9B7a8MLMcQ7W9dkvD7\\\\\\\/wUN3UpY3bPFxIP\\\\\\\/i8M8arp82D\\\\\\\/FCn1Zi9JFOz2QMXe3JaLmakUW\\\\\\\/4lNkkkIQqiCSlQgtO0GbaEJRmtuuiZwRRA0YLGUZYtZUn7hxxR93ROfr0zHqhPT1JJF6XotmFE7U3Fzz\\\\\\\/33pQN7bmcfaDIt+UoQepEFu5ZVazyf+tQykgIL6okOPmPY0hHWpYhUJd7hpqnSCUNAqs8DVzEVKSMfMBUooZxgsMJQ1eSWQrMZdefUD3bmsz34Bo7+Kjc3Khu\\\\\\\/HkUPKsXU50K7GUhLliseFrKRssLbjnwWmYeHPRswC0phBC0rR9Uf+wSWS1xEQxnNoY6AS\\\\\\\/FbwLjxuNDF6BVcydO5k2h28PhLCcOFvsrIx7UVMbv3RnAjHU5x\\\\\\\/inyNIssu7iAaDyh\\\\\\\/KlOtPYBBGmqbRxvK\\\\\\\/3pk1vfUtyK2lxdSai31XkECUC6CnhgE9DIv1ScIcmzYl5YF\\\\\\\/PDb9Bcm9csJqRvl06GclwIwQvnWm4wiqa7\\\\\\\/8zc8EQCBGFIR7Z0VLA1Uc\\\\\\\/lUjyD42hTl2QxaAdtnXBOzKE3V+Beirwb5z8EtQRRxawa4kiuAUSb1W8ozzsYTM\\\\\\\/qLiLgsdrdt98lCSGVd7wf1d+Ohpe54x0fS3YcSh3FzHTda1tnqmDHlb4OEXQqRkMn9lR2Laa6GneIqCmuZC+\\\\\\\/KXpg3dKRNZq3i+8U+1pj8QD5\\\\\\\/cNzPzxRjOcZUkAZMKGFVTRBeN1Lw9UOeN3xMj6tqK3D7yNEH\\\\\\\/+6Rme\\\\\\\/Olkk7iBGmTjaVUGQT0Y5UqW4FryP5DPhJ4oLkHdCHRcDI1etKzNwfl8n53Iqt+CLRGBDwSc6inFKruxX9aA4LKqWGoliWnn8\\\\\\\/fgQ8rOJl+ZLZkxmS8qZRM83AsgVxs17MpIU3Zhv0udse+XmdCCQZNAZ12CrrWqvfwZSfSFu1z2NqzLJr88nAeI5LZinZmsu2prq2a5Xo2MVq9a3ZI0e8tj\\\\\\\/1NP0fMuKKWE4d+os1EsXpNqVlAbN6OUK4fmSrxvY3nNRv3KNqq7VKC\\\\\\\/2LjQ5jl9QzE9oDlaVfE0EDHV1v+muO2fnfneLOOnhfcH10ChMhL9rMAMpdi5RgFcnZF5nzJrSK26dHEFYQJ\\\\\\\/w12kwC0cAcuPh\\\\\\\/Z2Jsr7IyOxt6SFou7DfVUHnDsxjw9+cw6aDpsrL7kec7fr1zXmsG4R33PzSTNpRFZehKyHQoa67oJxP9ID9lT3gGbHkR4csDAvGckM8BKd6uA6kiC0sbRNfeSbHmFMMbX2rclQxUTnYtgzguCoHnJ16JtryYhiiy9sBpSa2hJi246vGSiOBXt54tGd\\\\\\\/d\\\\\\\/yak7sMDLvgIBoYmIrl9GmhyxpLzSnlJaz0nybF8I5MiY+zWHzLilxScuSU8yG\\\\\\\/8CYsZ69bXO6IdHXYW+xJ1HHA+31cS32kZbr0wbOY4kjFeSjpq23NUy+s9WGtJQ1U4ruUeiLv2DIz7L14RnYT0Wa+dN+XIejZ+yQDWR+5MB9TXdegNnbmGta0lAZU5EhSh3ibVfo9LiF7qvzoOTZWEbnQNBEsn5RkMjDBRwwstMsPbj9ZSa1cNiF0VlVvyUa3TaR1mxR5u+y+OYsb9y+2zkQh9U2rJ1m7wNmlBn3Mrx1Z9DDui\\\\\\\/EtogAP\\\\\\\/GN0wo4iJ5Jz4FW33dY\\\\\\\/QaHZ5qp0ctZkczS6Th\\\\\\\/hjaPzvix4hRmpcFTk14TqMm\\\\\\\/uvd1HJ6IYFDU5FH+hO4zFsGsi0J+rFAGDbDoLPSKgEEEjKSkp\\\\\\\/pzYeP9foccTEdxl3x4aU5wEgW5yhjU\\\\\\\/IxsiqrfwvLsZh0Tl1esDjIwiWM4smxsCPtXo+XPob19ipv38B6rr21HZ2KAyp6tkCICLPparEgbhUaSJ019IGoCE4JmH60yvJw7Toj4OGlUaNnhIXiE+4cqlaXIlVFYKp2ow4A2Jjaf95bww9oR8kUEmvw+G4zMEJ4tOcnZM6AVPBqwO4xXWINotiCPR1vuUYf6EwKHUUK3NgK+J5MBXu+Gbax5v1ThME1ByWkJkyWeRDyUPTK7Mr8d3476OJtBXOeLVQLNFQ4\\\\\\\/5tiakVlri9\\\\\\\/EYzWH\\\\\\\/3vYwR29GW2QCuCgK7qYn9QxrRaIwMh\\\\\\\/cgboHJl4JDzwAIYPXfbaz7gCAm25XA3un90IoH9n\\\\\\\/mmDGhQukC\\\\\\\/eFzBj8giQab2nb+uu9K13KPExeA1FQjEPcjSsy11CqrNEfvYS66ySuppiNOp3T5v0vHIryeoU6b+hDlxJkGfiSQIYKpSYFab9md5hgulx69OdwVHKsTyKeo+y4mj1YzrJF96yV1MUCOLXpG2KXKDp+uZsnO9FTx0ggkyPtdmvsKKquxx4ty1GsFAEMM67MjYBhoKF5bauZ6GxAKJa8PotKuEciCEEoqPzDbOQw9hbIGRwmaYrAwFlDxCyxlsGLnAiI+9zozstL9BY0Ck\\\\\\\/MXYReCJwOMYrpJVpZc\\\\\\\/B4CgMmziPpcIbeLbGJ3Zf0WcBcEA\\\\\\\/B5of9cSSV8xfmhFsRFggMljLwn8g8v\\\\\\\/K7+x\\\\\\\/irrzjzt74MDeF8Ki2hXQMIXIma1Jrajkg\\\\\\\/xh4xYwXGoKEDB3iVyBU757EDnEW5nKPPWAkL\\\\\\\/NIs\\\\\\\/d2sMSRIZ8XUqR9RFR+T199KnHxkIYdBMZRLodgXS3NeX5tcl4e1yuPM3QUYvgZvatHJJ748IHgWPpajAANWgR\\\\\\\/S8bl8D9Eq4AIKAwa0k3ak7IOm6wjc+p56LBBevMQO4knhxF9t9SSSESVGhdtogunXw0g8dwdMiD+Rte3HZod7iYJINPeR2\\\\\\\/mLtOAa1hbmIY3jqa3CpKau0o441Q6bYU9bWcgm61QQhdomYfKg0GXNBYRzKL\\\\\\\/uPloN8EWMTWGNNEZwVWDGgjW8Vz2h2IsIYPTordBTxJH6S7CEF6i2ZgZ0yb0SVmhVZnsTKRj7a+RvF5EyrfnjJ43JqC+DITQ1b8uthW6W+Mpldog+oHH\\\\\\\/BCp9SqKEfHsV1A3SVJhvxThgAC66bpuRgKZkspWWKe35LgKrUSsAs7xUr6VI+P1afSxMdFEkLFSZbqMJvBjLIoV8c8pcadGKzs9JrF3vAHLBwHa\\\\\\\/5I3Hwib00jDF5Qv+uTSqDzOzOpA0fBaM3YiFWqdjA05AQT6uma680vtxK92ZBNEWNYmTcHOiIKCM2NqEGiMmm2ay2HdJWWRSvjDZrJ6I+dX6Kdr9KcjhYQlpvgT5MJZWyhk9o7IbGu6gVKGpLhu+FVt6a1Tnzik364helgbw2emFB9eItIRfAoCK4Dnzu\\\\\\\/crVaRBq93ilhwgrWiRTnAkJzNetPMthsqzIcflV9BTHC1PeEYFKgNnFY2pbiV14R+ubCy169ntIXXLDX+uxxV4s\\\\\\\/XFMavfz3Ci\\\\\\\/3gC5KcrS96AEIsDG5J5A4f1ycPXpBWpVy72yn4WbhLJHN3COn4LCAIKN0XeOVCOaOWKvWBQKRvVoyuDmVT+nilng0ucUFfQO0vEOnkVJzFhUkYohax+ipCjkARagnG9dtSiLyJQdcw82K4x0he\\\\\\\/mjTxnM+FJNpNvIabh1lVIy8othbbn7DcSo8SFtNQUjMjStklSlQiDE8jEdVVIz0xM20h7DIomlBidzuBS9+OgGw4QgkvNxAQIJ6yv8ybHqDHy7vw1G4fcor3iC7+zM53Let477r31yJJ1J+nqWgEk12tR8HTpyxO5QxHXcsgEGLlDyvXeqOdOai3jHXNTLSO1cQHb9xeLW97P5qxsfRLVGZfbXdqsG1zYcXEpbGtBDtD7rJL5fWlYx5HDNlZqyRf5kOgdlwy4UcImR22HLICK9JkeCVc6fSZwCbUKW9DYCy9CPmy1F6GkjDixRxUEHfb7NodbPthQHI9fhlmoVfogsFSLnBmjgtzlf6HU6Ef4PjL9Xm2yKGxWc3PTv42Dae6tPudfOyWdedZo04wUQpYOufLerUoD81IRlWqjvhAktFT1Gysl7YWvxHlnEc\\\\\\\/6r3YAiR+lPRskCU6j9+2uRm0z+avpdIJTLhjndbTrM9+hsHTHEJVGHHderERQ+SEnf\\\\\\\/Rk0TZYhDEpSKqxTjrsp71qQ6i9Pf8cpkFQxYCEtu+7kPTurR4GJbrYYaBhYXo257rPfvOMCls67+5vTR91nOpoOui\\\\\\\/zpXezprnzVO2XDdl8oL4mICaLdf9UXW\\\\\\\/0s6C4YfvK5mu6bafGFOV0X1qwrb4H6IiiqlpHnVjvyxMdDI\\\\\\\/3D1oI838tmRC3K4g6DfYlh6f+Rl0vMsgxDZxD5ISkqEJPI4Ed6\\\\\\\/ry2loJfMp\\\\\\\/TbF3MpnQNCqu4fVhMDNJ3y1Kyk1PC+Etjt87gbwIWW424T0KOM0twql0bKpg1yHZcL6uEK2HiuCKUdEt0R5T0Ud+KZRnVFVoKCRg2LzykPwS5ywxRLTGe+0ZUTXTL2dZEXM32VFIuEQMnt4W5Bs\\\\\\\/uUf6Kms0Fk\\\\\\\/8cEXirHxCUYw6R8RHX3\\\\\\\/t1cAJmwM7Jh6mGbk0iE86E5GiP6hbZXe5D+OVrBm+kNpnXvUVTR6wHUDIIC3A45FOWEycAJ2zUkQNPm\\\\\\\/3r9BTT38IifLZQtaCBNSDjutHWHrd62b7P7ZSidtxf5CT8wCA9lSWqD\\\\\\\/Gs9\\\\\\\/uSjXKg9DPNHSbU8V52iOvAfJepvbQxOhYD18Vex\\\\\\\/IzzLpMT5EZg8MVxGaHq6LuSq+yVgztr4k14N3xb9G7M4SkdY3NecWqzJ4KBuQk3phx3wf27\\\\\\\/pud810qL8NzzRZel1yohO5CIZOOloi5fOy+nih5Mg1HRhkQ1s744WZE63YY0CPYd2bQLaqZOCl3bNsAy5eb+JLTVjsCMvmCR1wJgNPTT9aZ9BNi\\\\\\\/5DuBTHsQjIysiG+VpAL7tIYx\\\\\\\/Y\\\\\\\/QoHZkyuyPrtxgpaJWp6pB1L+Gtu407Qcx4U\\\\\\\/ogv7TfKgmiGNm19KsxEwTBJWImvnr56kksArJx2u9DxSxv2IuI+M0Ev9yfwh8Y+z9hE\\\\\\\/qlD\\\\\\\/Qxp2uIHqBA\\\\\\\/4E+LMT\\\\\\\/KtBASi\\\\\\\/Fjrtt\\\\\\\/rDlJlupJEQrS3HwxONlQB7tZmYrbsfR0uTZPyDjdc+UnikSSBM4AF+BwxqTOV0fQ8ikZNAY7NwEmVe85KSBHOZILEN\\\\\\\/jxxm0OnY3w1GM3UAwVjALjXD168mv65x4E0\\\\\\\/yvo\\\\\\\/IaeEB4kamT+dC0MjNoLhEPPwA3MEBwOKyEOOcXOSYjCD80muc4CqvvoNPeBZuJwK0pQrGj5EISLZLqnevwF17Aw2aqyk7TDvNShI\\\\\\\/v7YbIuCNcRQ3aCVAOK2K3xZnrYW3s0ZGPVdxb4Gv6ij1tW65SeHH5JGka7Dd9h4rp2F8zxkOuFZIHr1o9jPx6uO4B6e97qRSnmMLtJF9\\\\\\\/XnvynJP6Zq3IoOpHDNk2KG6265uZ7Dext8\\\\\\\/7bd5Vkra5q1LZBXTlvPmp6mkJn0SiidVPZvnRokJMTdOK0fDlX6sMVlPo4\\\\\\\/nqLikls2yMYKGQBSIRB4bOjSXH0sZhG\\\\\\\/RiNQCTG0NV++cUTuGH5zGy62xfXrSb+ThZkFxyLv7eP6Psk3ncxFiJPe9LsIfpAuE\\\\\\\/K7\\\\\\\/OOoo1d4YYAZMtPd0dLVh8xdqWDzy79ICwf0koWa6tygdjkHNudsD3Hfs1PefSxKWjU8fDTMIg2WMkpq1SKDiq6ib9CPGtJMTCEesCOUtiuBPPrb6X6VZ+F1uc7L5utC+8vxoz9uGHnvfFJ42NLthRyJf\\\\\\\/inuk09hpXbhlIZ22OnKYqrCLj4MrdBsGOKz3e3Xw70dqzIYMUwiosov3moLrL2Bf8++jxiW4JATYSEQ+8j+LReWcuoZrigrwWjODr+f5d82ULWDeBeFIM5iX4Gm+bPBMzZQ6gZXD9hzuWqFbl8FqSaE0FPGAkBELlAPAlfDs4Q1o73SFisPw+SftZIRQTQ4FZQoyzhZvhIaKy7W7lQo\\\\\\\/YNPba8LZMWcN1IPUgm3ksx30bn9usAsBG3RBynmEXolm80l\\\\\\\/CGNpb276PM\\\\\\\/3FiB69YhzWkKE9qmmmBh0CDP\\\\\\\/FlfXCH8WYn2MzEceG4dboyMud4eGAMxJE2oH1Y5WuWKq+ntfQ8O0qPqfOtAWLS9p4Pl20bXLpv0KNZmioMfGqBgIy3lY5jW9uhWW6QXxB4SHHcnjr\\\\\\\/h250BfuBYk9K6P+O1aqB6gwPjdWYPPSNoc6pUDp6i1R0Td6iM9V63AO\\\\\\\/BsGYpRX6fzOtR4Vmd3+jvLKh+DZ689zvl1hJA\\\\\\\/LajpnDIub3ZOB7UD6pb5rwQ3HYbg5kiDfJs+bTxeWeBpNGkD2iucRfnw9VoYEMS7lj27pP79rDNthZ3pXPa2wQ3obd4EBAB8MkvrY68eB1oUISFxFyUuKxM5Mk3T\\\\\\\/NWj\\\\\\\/k9OiHR\\\\\\\/Lw6R7+FEDNYhze85HBt+ZJuZYWLkU8kgLVP1Ss0qy7IPShnz1rRAza7YqW2tzRxCjCUwkRqZ5rxanmZkaZKaNcdgkI1FihWrLMokggONsCQ\\\\\\\/i8Mk7uGPoO19k3qE3aTx4hI4sHmcY+vKob90+nahqXsZdnItfltoArYAikpwTNDeavVnY9YV2qtu+e2UTNpDeywytBJ817iO2iC3CYBJ6VA2VMrk0D34mOWqKxq9wyQj3Z4ZtpVte7V2632JK+kCkY0Zvo24Ps2U\\\\\\\/Pt\\\\\\\/3nHOyh3N\\\\\\\/FkvRpw53xgD\\\\\\\/Lng5cciA4JLGl1z0E2r1Gb\\\\\\\/PbiiqF1D7b\\\\\\\/E419ViexE7hw1oChheO71mauMxsqK02KVTa3otUmWnSu0wmX368AKnngHEKdzitalhKKsWHC\\\\\\\/8vFNTfciBHP5C+ppy20wx2z1hrp92WgfQfEmm6Ct0Zm\\\\\\\/ioCgsFc7CPukcwNUJwLVXZevNDUhxScb34GK\\\\\\\/c3OJ3qKT5MmSSqm5Le7vZEMsEThcbxNNecU3mpD3\\\\\\\/H9aZQ2uYNuCsNQyeTaF+JqpYOR+7R0lZYM56DOvfhEzo4kr7uiQknrNN7tC\\\\\\\/5bc39pLtUXYPfTMFQt9uzoy6EurIw9cWGkIVUF9bDQy7iNse79HrS7Jmn0pcXbT8V2PBqgh9ChC5QPlX4hYtAD3zK0kG58iAQ3YychkPYaEO4lV4HZwB1vi62+og7d+xmHkmz2ob97CtmoLqYPM44g1OQDzz5bAztmXNSKXbnU9ARQy597H6D+iAZeHgU\\\\\\\/MzLQhdRqOF8sn9QZB4NCdouFuACidp2311EveyJKahbGPcFiN75Vuf+pRro+iJP1p2b1\\\\\\\/z9DGhpNQLFGr3yzTB6XhdhhyYHthowxmnoMrHeJw63THFpuILikhc2CfXNikLdMmV\\\\\\\/Klu018Iz7ed\\\\\\\/hIOCLSEJfG29YfGvlfAZ6mhUqginWKd2Hiw1LAGUXbBNizUbj01cpVACJyNbjiPwSL7bbO\\\\\\\/Wxy8VdNetwgfnpyjGRXqKWnqJ6MpPWtbQtgKnFl4JVDCA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022JjIFQxcUA8m8+5Tp6zPnNw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220ba89294f06690fc0657e8292b209e62dc9fecc9b3f6885fc0950ebf964b8b1b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227d55jPxnvcLmunQYM7g6uBB05Tp6t6Dzhivv+AK7tTASlhBh9qsXFtQuT2h5918CxRv0jpBXwcd10XH7A01B2yDVZXyZQ2JYEEQQ5KNvC+gwaCP4oTPw30SLe5m8lsu7NF5HQxQSVsmvGdWK5YjtijiGa9oashI2djp0heP1k551Vl7mJKkQJEHBPgkvOv5jfDadPKc90jbs\\\\\\\/ecrbl7FtkGN8p6JfV+7kQ3uqSTXDyx8qaXFD2QaZ7GcKL3lpc3fW84TLeRxGNZ4iLjxo2HUe5sC2VVvqJBTRRc1Oj3K\\\\\\\/LhWWYNgkUC+eA3UUgOhKrMEQ+nWFGfRki5UljaJ\\\\\\\/THQbsYQahE+3pqTFoTS4w\\\\\\\/ErmPdY5gSWDii5kZMfNJ+QNivRusj2hxDlmqGQFc9+G8FL5yWnG6stejz5gYfqX5FjU55DdNZQV1thyiAivNA\\\\\\\/j\\\\\\\/252NjlSkGUiONesBjK5Sh7UzSKoD26wrYgKav1MLRGK7D9Medhqor5zv\\\\\\\/AP6ywVOCfT9A7e08ZMlH2feqOiNGSYLlPDqu5He+8BFqZU9npRtBnahtj4Q+RlDQZkofxf39we5m0u+dIXrnTwDP+4VJdboge\\\\\\\/wY5sgOgcwJjvrMVktuOZ3+8fI\\\\\\\/r7+n0xB\\\\\\\/h8EVPSs2fpHyE\\\\\\\/cmHaqsc\\\\\\\/weWhBlKXb50n5\\\\\\\/snqkspyQXjmI8TXdNGYz7u9VXKpoYJVWTaayeU3\\\\\\\/kuSxH8VQZyPxOIsw5yYFtbeUEJ\\\\\\\/\\\\\\\/6Dt4qW+\\\\\\\/9upUxXK6W0SbboXOqq2G7o+Yhe6npysPlGTfhymHfjm\\\\\\\/7qvpekJOyeacyeL52ethXhRGnGtKE4NmaFhRuQXw1xMEkzbqWQK80I4rBhL6NNov+la1WcLQW1xbbsBaNehJ4sTL2jyb+0xaKVWpPxlGKUZcqk8\\\\\\\/o\\\\\\\/Mi8VxUG5BDV89RroEcA1v7aZB06p6m1OmCDwSjq0qDap1AEoHn11Yd7z6U6uFAhteaNO4kdY3sWDKOgk8gFr1V8n1+kD0e2WYtzYkxPDOXZ6Z3JvWwWNnYN0VR46gTgAcNPEv6i8NOF5yLh9pCsdtOGTftHERZL9e60XCD1FOky14kpYeY8Kq42kY1hmPPAzOCbejerPkO1uUwDIDuocRRdQCO9s8PPaWv3A\\\\\\\/qFdoxOGjIwzp9bmgqtmE75ln0UHTMGeRwBPVWPmMTFu5qpObfKQFZoETvIjMD4vAnaTA59vx6SbD\\\\\\\/PZlvKyhZ2RgHTmG6jbgB6\\\\\\\/7U0dy4GIusuH7jKIUCIQjBGY8+5n4RoHJRp\\\\\\\/ow4+VRCXzHdU9b1pa3gDP15rYRnsDOBamjXz2LHOzADYQTICDFzYSj0iUPkxMdMWT35IuydDBzQqNRI9aujmH7nk4MpxSHHwlAk30yj7EeRKL1EM21h4PUlNQHBSgOV0H40Kf0kWr1qTe+tywxfz1c7HcJGUXFLbcPCOtcEhLuKD2nDk33YRoioTzKv6kezR7YesvDojbkyQKJedXAD\\\\\\\/m32yUXfarG21pTceJsniCJ8PIiD3jx6LQyUIxJSD+Wb\\\\\\\/XABA8oP4xzdS\\\\\\\/Nuk\\\\\\\/RiBXTIvdNydsc0mcyq\\\\\\\/+c28H83OCdGiEOnHHAqoIua\\\\\\\/sp0TsEjjjHNtXnjDxucPLGPr0dw18WYi7wqFzV4NdE2VhaRxtYZhpToaFCC8dyDyX1peTIZDfPdismNMpER4yf+MwdyTLR66LvWDArd5c1mKJIv0S1\\\\\\\/1MOP8+HdQwKyWf5WLUQvjRN17YO2YSHTS\\\\\\\/FHt6kwr3LyHeaDSLggx5s\\\\\\\/L4PKAjbVVSopMWHkg0ZdiH2sd\\\\\\\/BryqwAOToQU0jlvLTi51PJA5OptjhUoEA+5lypfQBkf46jY1EfjKYZgf2niPxoNGPF5kGqqPl8Dn3LDNrZ2cm1OUDbQIrvk\\\\\\\/ZX70rOAKava9k7fdz8Yg3k29f8AEwvmKOQeAVloAuPaXmmLocC9k3xqB0dce+e4H7GF\\\\\\\/UfFc3y\\\\\\\/\\\\\\\/UQrCGv\\\\\\\/Ll1krKpoKAsXDSIlRSusSu4zewaGyOs37P65\\\\\\\/3xE9DHg==\\\u0022,\\\u0022iv\\\u0022:\\\u00220OONWtOlT+Cs92b8p5v5ZQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a663b7d846c5b67a94bc5e0768a10c9fe2aca20b61aef024c79e90713f2adc8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459263 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/429" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rm1ehs_q4BXR82kQTNLqS7WgHvVdrb6PupeBg8tL0e0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0BiB1D8vPI5RJgK_sATrwP3F3gTYKxf-OoKcUEfSlwo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/4/425" title="Simulated Data for Genomic Selection and Genome-Wide Association Studies Using a Combination of Coalescent and Gene Drop Methods" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/4/437" title="A Systematic Genetic Screen to Dissect the MicroRNA Pathway in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459263" id="node459263" data-pisa="ggg;2/4/429" data-pisa-master="ggg;2/4/429" data-apath="/ggg/2/4/429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Common Dataset for Genomic Analysis of Livestock Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew A. Cleveland</span>, <span class="highwire-citation-author" data-delta="1">John M. Hickey</span> and <span class="highwire-citation-author" data-delta="2">Selma Forni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">429-435;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459263" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew A. Cleveland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genus plc, Hendersonville, Tennessee 37075</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BA.%2BCleveland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cleveland%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BA.%2BCleveland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:matthew.cleveland@genusplc.com" class="" data-icon-position="" data-hide-link-title="0">matthew.cleveland@genusplc.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environmental and Rural Science, University of New England, Armidale NSW 2351, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Selma Forni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genus plc, Hendersonville, Tennessee 37075</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Selma%2BForni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forni%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASelma%2BForni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/4/429" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459263" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/429.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459263" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/429.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459263" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/429.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459263" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/4/429.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459263" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459263/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Although common datasets are an important resource for the scientific community and can be used to address important questions, genomic datasets of a meaningful size have not generally been available in livestock species. We describe a pig dataset that PIC (a Genus company) has made available for comparing genomic prediction methods. We also describe genomic evaluation of the data using methods that PIC considers best practice for predicting and validating genomic breeding values, and we discuss the impact of data structure on accuracy. The dataset contains 3534 individuals with high-density genotypes, phenotypes, and estimated breeding values for five traits. Genomic breeding values were calculated using BayesB, with phenotypes and de-regressed breeding values, and using a single-step genomic BLUP approach that combines information from genotyped and un-genotyped animals. The genomic breeding value accuracy increased with increased trait heritability and with increased relationship between training and validation. In nearly all cases, BayesB using de-regressed breeding values outperformed the other approaches, but the single-step evaluation performed only slightly worse. This dataset was useful for comparing methods for genomic prediction using real data. Our results indicate that validation approaches accounting for relatedness between populations can correct for potential overestimation of genomic breeding value accuracies, with implications for genotyping strategies to carry out genomic selection programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pig" class="hw-term hw-article-keyword hw-article-keyword-pig" rel="nofollow">pig</a></li><li class="kwd"><a href="/keyword/genomic-relationships" class="hw-term hw-article-keyword hw-article-keyword-genomic-relationships" rel="nofollow">genomic relationships</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/cross-validation" class="hw-term hw-article-keyword hw-article-keyword-cross-validation" rel="nofollow">cross-validation</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li></ul><p id="p-4">The availability of common datasets is important to the scientific community. These datasets can be used to compare and benchmark methods, to provide links between different datasets facilitating meta-analysis, and to enable researchers who lack such resources to test hypotheses that address important questions. In human genetics, many common datasets are available, <em>e.g.</em> International HapMap Project (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">The International HapMap Consortium 2003</a>), 1000 Genomes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">The 1000 Genomes Project Consortium 2010</a>), Framingham Heart Study (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dawber <em>et al.</em> 1951</a>), Mouse Genomes Project (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Keane <em>et al.</em> 2011</a>), and have led to numerous discoveries.</p><p id="p-5">Because commercial data may hold economic value, common datasets of meaningful size have in general been unavailable in livestock, particularly during the genomics era in which organizations have made large investments in genotypes and phenotypes. Efforts to implement genomic selection in a number of livestock species have generated datasets that contain multigenerational phenotypes, genotypes, pedigree, and progeny test-derived estimated breeding values (EBV). Such commonly available datasets could be used in numerous ways, in particular to evaluate new methods to estimate genomic breeding values (gEBV). Additionally, important questions relating to the nature and properties of genomic selection (GS) could be assessed with such a dataset, including the effect on inbreeding (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Daetwyler <em>et al.</em> 2007</a>), the importance of close relatives when training prediction equations (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Clark <em>et al.</em> 2011</a>), and the dynamics of long-term response to selection (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jannink 2010</a>).</p><p id="p-6">As part of the effort to implement GS in pigs (<em>Sus scrofa</em>), PIC (a Genus company) created several datasets containing individuals with phenotypes for a number of traits, high-density genotypes (60k), polygenic estimated breeding values (pEBV; no genomic information) for pure and crossbred traits, and complete pedigree. PIC has implemented several methods for routine genomic evaluation, including a two-step approach using BayesA/BayesB (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meuwissen <em>et al.</em> 2001</a>) and the single-step evaluation described by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Misztal <em>et al.</em> (2009)</a>, which, based on extensive internal testing, PIC considers current best practice for predicting gEBV in a practical setting. Genomic breeding values from these methods can be routinely predicted through a process that includes quality checks and data editing, imputation of missing genotypes, and validation in “selection candidates” when pEBV accuracies approach progeny test proofs. These validations can be problematic in pig populations in which the structure of the genetic improvement program yields few individuals with high accuracy pEBV across traits. Thus, we require optimal validation procedures that avoid overestimating the expected accuracy of the gEBV.</p><p id="p-7">This article describes a pig dataset that PIC has made available to the scientific community for testing and validating alternative methods for genomic prediction (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001453/-/DC1/FileS1.zip">File S1</a>). In addition, current best-practice methods were applied to the data to predict and validate gEBV. We discuss peculiarities particular to the data structure, with an emphasis on their impact on gEBV accuracy from validation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><p id="p-8">The dataset consisted of 3534 animals from a single PIC nucleus pig line with genotypes from the Illumina PorcineSNP60 chip (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ramos <em>et al.</em> 2009</a>) and a pedigree including parents and grandparents of the genotyped animals (N = 6473). The majority of genotyped animals were selected for the genomic evaluation of a specific trait, and the remaining were added as part of a strategy to “fill-in” missing herd sires and sows to calculate genomic breeding values for selection candidates. The sample consisted of male and female pigs born since 2000, with varying pedigree relationships among animals, although the original selection avoided sampling multiple members of full-sib families.</p><div id="sec-3" class="subsection"><h4>Phenotypes:</h4><p id="p-9">Genotyped animals had phenotypes for five purebred traits (phenotypes in a single nucleus line), with heritability ranging from 0.07 to 0.62 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), which represent a small number of phenotypes that are routinely collected from birth in the genetic nucleus. Each phenotype was either corrected for environmental factors (<em>e.g.</em> year of birth or farm) and rescaled by correcting for the overall mean (traits 3, 4, and 5) or was a rescaled, weighted mean of corrected progeny phenotypes (traits 1 and 2), for which many animals have no individual performance data. Each genotyped animal also had pEBV and accuracies from single-trait pedigree-based BLUP evaluations. The models to calculate corrected phenotypes and the models to predict pEBV included the full PIC pedigree and all data used in a typical production run for each trait, excluding any genomic information. Accuracy was of the form <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4/429/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4/429/embed/mml-math-1.gif"/></noscript></span></span>, where PEV is the prediction error variance and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4/429/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4/429/embed/mml-math-2.gif"/></noscript></span></span> is the additive genetic variance. The distribution of accuracy for all traits is depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Animals were originally selected to maximize the pEBV accuracy of trait 2.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461428/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461428/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-10" class="first-child">Summary of phenotype data</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/429/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy (r2) of estimated breeding values for genotyped animals (N = 3534)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2079668376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accuracy (r&lt;sup&gt;2&lt;/sup&gt;) of estimated breeding values for genotyped animals (N = 3534).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/429/F1.medium.gif" width="189" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/4/429/F1.medium.gif" width="189" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/429/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/429/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461351" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Accuracy (r<sup>2</sup>) of estimated breeding values for genotyped animals (N = 3534).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h4>Genotypes:</h4><p id="p-14">Genotypes available from the PorcineSNP60 chip (N = 64,233) were filtered for extreme minor allele frequency (&lt;0.001) and proportion missing genotypes by SNP (&gt;10%). Additionally, markers on the X or Y chromosome were excluded, yielding 52,842 total SNP. The overall missing genotype rate was less than 1%, but many GS methods require complete nonmissing genotypes. AlphaImpute (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hickey <em>et al.</em> 2011</a>) was used to calculate probabilities of each parental allele, which are combined to fill in any missing genotypes. An imputed genotype based on a probability score [the sum of the allele probabilities (SAP) yields a non-integer genotype ranging from 0 to 2] was used to replace missing information. SNP with both known and unknown position were included and imputed, but the map order was randomized and SNP identity was recoded.</p></div><div id="sec-5" class="subsection"><h4>Genotyping strategy:</h4><p id="p-15">The original selection of animals for genotyping aimed to minimize pedigree connections between selected individuals, but a high level of relatedness remained due to the breeding schemes used in genetic nucleus pig lines (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The populations in the nucleus are relatively small by line, and the turnover in breeding animals can be rapid, which makes it difficult to identify animals with high-accuracy breeding values to construct a training set. The animals selected for genotyping tend to be used widely and have more close relatives than others in the population. One advantage of applying GS in pig populations, though, is the high levels of relatedness between genotyped and phenotyped training animals and genotyped selection candidates. The gEBV accuracy in selection candidates can therefore be maintained through incremental increases to the existing training dataset by adding small numbers of herd boars (and potentially sows) to sustain connections across generations. This structure, however, makes validation of SNP effects in a less-related population problematic, and thus inferences about the usefulness of alternative methods in other livestock species may be difficult.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461365/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461365/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <p id="p-16" class="first-child">Percentage of genotyped individuals with relatives genotyped in dataset, by category of relative</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-6" class="subsection"><h3>Genomic breeding value prediction</h3><p id="p-19">Prediction of gEBV in PIC has taken two forms since the development of and subsequent large-scale genotyping on the PorcineSNP60 chip. Breeding animals in major lines have been routinely genotyped for 60k SNP and used as training populations to estimate SNP effects and to identify important markers for traits of interest using de-regressed breeding values and Bayesian analysis approaches [<em>e.g.</em> GenSel; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fernando and Garrick (2009)</a>]. Based on results from training populations, a large number of selection candidates have been genotyped for smaller trait-specific panels. The resulting gEBV have then been blended with the polygenic breeding value, using an approach similar to <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">VanRaden <em>et al.</em> (2009)</a>, and incorporated into the overall index for selection. Alternatively, the single-step approach of <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Misztal <em>et al.</em> (2009)</a> has been implemented to reduce the computational and logistical requirements of the multistage Bayesian approaches. This method uses all available SNP to construct a genomic relationship matrix among genotyped individuals, which is then combined with the standard numerator relationship matrix that includes all un-genotyped individuals. Genomic breeding values are then predicted for all animals, regardless of genotyping status. In this situation, selection candidates can be genotyped for a small panel (<em>e.g.</em> &lt;1,000 SNP) and 60k genotypes imputed to improve the predictive power.</p><p id="p-20">To predict gEBV using these general approaches, the data were further filtered to exclude genotypes with extreme minor allele frequency (&lt;0.02) and large values for the Pearson chi-squared test statistic (&gt;300), indicating extreme deviation from the expected genotype proportions. A total of 48,866 SNP remained for analysis. To evaluate the accuracy of methods to predict gEBV, a 6-fold cross-validation procedure was used (XVal), in which all animals appear in the training set and in the validation set. Animals were randomly assigned an integer from the set (1,6) for assignment to a fold. Each analysis was performed to predict gEBV for the validation set using all remaining individuals as the training set. This was repeated so that each animal was in the validation set one time and in the training set five times. The training set can be defined as animals with phenotypes and high-density genotypes, whereas animals in the validation set have only high-density genotypes. Each training set, therefore, consisted of 2945 genotyped animals (reduced to the number of animals with phenotypes in that set), with the remaining in validation where phenotypes were removed. Genomic breeding values were estimated using BayesB (as implemented in GenSel) on both phenotype (BayesB_ph) and de-regressed pEBV (BayesB_ebv) of genotyped animals and using phenotypes in the single-step approach (SStep). Additionally, a standard BLUP using phenotypes but no genomic information was performed (BLUP). EBV and gEBV were calculated for all genotyped animals.</p></div><div id="sec-7" class="subsection"><h3>Genomic breeding value validation</h3><p id="p-21">Accuracy was estimated as the correlation between gEBV and high accuracy pEBV, which is a substitute for the true breeding value. In this case, many animals did not have accuracies that could be considered high (<em>e.g.</em> &gt;0.90); therefore, a subset of each validation set was selected based on pEBV accuracy. The top 75 animals by pEBV accuracy for each trait and cross-validation were selected (N = 450) to correlate with the pEBV. The mean pEBV accuracy (r<sup>2</sup>) for animals used to determine the gEBV accuracy is in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The validation subset was then further divided into categories based on genotyping of relatives in the training set. Validation animals were identified as having at least one parent (P), a least one parent and at least one offspring (PO), at least one offspring (O), or no parent and no offspring (N) genotyped in the training set. Accuracies of gEBV were then evaluated based on these categories, which is similar in principle to <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> (2010)</a>. Additionally, genomic relationship coefficients were calculated between training and validation animals [using <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">VanRaden (2008)</a>], and the number of coefficients exceeding a threshold of 0.45 in each validation animal was determined, indicating the number of animals with which an individual was considered highly related. This value was then used as an additional categorization of the gEBV accuracy to determine the impact of knowing the actual relationship between the validation and training sets <em>vs.</em> knowing only the average relationship from the pedigree. The results of the 6-fold cross-validation were then compared with an approach where young animals with no progeny in the data compose the validation set (N = 509) and their parents (and other older animals) are in the training set (YoungVal). This approach is commonly used to validate gEBV in other livestock species in which progeny-tested pEBV are routinely available (<em>e.g.</em> <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">VanRaden <em>et al.</em> 2009</a>). Accuracies of gEBV were estimated using the animals with the highest pEBV accuracies for each trait (N = 30). Because of very low pEBV accuracies (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), Traits 1 and 2 were not included.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461431/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461431/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <p id="p-22" class="first-child">Mean accuracy (r<sup>2</sup>) of estimated breeding values for individuals in the validation sets of the 6-fold cross-validation (XVal) and the validation using young animals (YoungVal), for all traits</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Comparison of methods</h3><p id="p-23">The gEBV accuracy generally increased with increasing heritability of the trait (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), corresponding to an increase in the mean pEBV accuracy for the validation samples (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). The maximum accuracy was observed for trait (T)4 and the minimum for T1, across all methods (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). All correlations (between gEBV and pEBV) were different than zero (results not shown), but the low mean pEBV accuracy, especially for T1, suggests that the gEBV accuracy for this trait may be somewhat underestimated. The gEBV accuracy for T2 was nearly as high as the accuracy for T4, which was unexpected due to the lower mean EBV accuracy and the lower heritability of T2.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/429/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic breeding value accuracy of the 6-fold cross-validation (XVal), for all traits, using a standard polygenic BLUP (BLUP), BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2079668376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic breeding value accuracy of the 6-fold cross-validation (XVal), for all traits, using a standard polygenic BLUP (BLUP), BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/429/F2.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/4/429/F2.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/429/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/429/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461383" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Genomic breeding value accuracy of the 6-fold cross-validation (XVal), for all traits, using a standard polygenic BLUP (BLUP), BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Across methods, the gEBV accuracy generally increased as additional information was implicitly utilized by the different approaches. Accuracies for all traits improved using SStep, compared with BayesB_ph, and improved again using BayesB_ebv, except for T5, in which there was a small decrease. The BayesB_ebv approach used information from progeny and a multigenerational pedigree to generate a higher accuracy “phenotype” (in the form of a de-regressed EBV), whereas the SStep approach used phenotypes much less representative of the additive value but included genomic relationships. The difference in accuracy between the two methods was generally small, with the exception of T1. The difference between methods differed across traits, with only small changes when the accuracy was higher but much larger changes with lower starting accuracies (and trait heritability). The addition of genomic data was expected to increase accuracy in all cases, compared with BLUP, but the phenotype-based BayesB_ph analysis resulted in accuracies that were actually lower (or nearly the same) than the standard analysis using pedigree alone.</p></div><div id="sec-10" class="subsection"><h3>Relatedness between training and validation</h3><p id="p-26">The gEBV accuracies increased with an increase in the relatedness between the two datasets (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The change in accuracy with higher relatedness tended to be smaller as the trait heritability increased, as did the differences between methods. Very high gEBV accuracies were evident for some traits (<em>e.g.</em> T1) when including offspring in the training set, where accuracies approached 0.90. A gEBV accuracy of this magnitude is substantially larger than what would generally be expected, especially for a lowly heritable trait. An additional separation of accuracy by number of parents genotyped in training for T4 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) shows that much of the accuracy increase is being driven by having both parents genotyped in the training population, which can be an advantage for populations that routinely genotype both sires and dams.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/429/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for individuals with at least one parent (P), at least one offspring (O), at least one parent and one offspring (PO), or no parents or offspring (N) genotyped in the training set. All traits were analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2079668376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for individuals with at least one parent (P), at least one offspring (O), at least one parent and one offspring (PO), or no parents or offspring (N) genotyped in the training set. All traits were analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/429/F3.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/4/429/F3.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/429/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/429/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461375" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for individuals with at least one parent (P), at least one offspring (O), at least one parent and one offspring (PO), or no parents or offspring (N) genotyped in the training set. All traits were analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/429/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for individuals with one parent (1P), both parents (2P), at least one offspring (O), at least one parent and one offspring (PO), or no parents or offspring (N) genotyped in the training set, for T4. The trait was analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2079668376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for individuals with one parent (1P), both parents (2P), at least one offspring (O), at least one parent and one offspring (PO), or no parents or offspring (N) genotyped in the training set, for T4. The trait was analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/429/F4.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/4/429/F4.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/429/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/429/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461371" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for individuals with one parent (1P), both parents (2P), at least one offspring (O), at least one parent and one offspring (PO), or no parents or offspring (N) genotyped in the training set, for T4. The trait was analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">The evaluation of relatedness using an index of the genomic relationship coefficients showed an increase in the gEBV accuracy for T4 for all methods as the magnitude of the genomic relationship increased (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The other traits showed the same general trend but were slightly less consistent (results not shown). The minimum accuracies were lower and maximums higher when evaluating actual genomic relationships compared with the same approach using pedigree relationships (<em>e.g.</em> <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), indicating that a more precise definition of the relationship between training and validation can impact the estimation of expected gEBV accuracy.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/429/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for bins representing the number of validation animals having genomic relationship coefficients with animals in training that exceed 0.45, for T4. The trait was analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2079668376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for bins representing the number of validation animals having genomic relationship coefficients with animals in training that exceed 0.45, for T4. The trait was analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/429/F5.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/4/429/F5.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/429/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/429/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461326" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">Genomic breeding value accuracy of the 6-fold cross-validation (XVal) for bins representing the number of validation animals having genomic relationship coefficients with animals in training that exceed 0.45, for T4. The trait was analyzed with BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">In the validation using young animals (YoungVal), there was an increase in gEBV accuracy moving from BLUP to BayesB_ebv (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), similar to the trend observed in XVal (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), although the correlations (accuracies) for BLUP were not different than zero (<em>P</em> &gt; 0.05) in this case. The ranking of traits for gEBV accuracy was equivalent to the XVal for BLUP and BayesB_ebv, but reversed for BayesB_ph and SStep. Overall, the gEBV accuracies were shifted downward compared with XVal, likely due to a combination of lower accuracy pEBV and a reduction in the level of relatedness between training and validation sets. The gEBV accuracy for BayesB_ebv on T4 was similar to the accuracy obtained from the (N) and (P) categories in XVal (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), which was expected based on the relationships between the datasets in this validation, but the gEBV accuracies using BayesB_ph and SStep were much lower (∼0.20 compared with ∼0.70). The accuracies for T3 and T5 were more similar, but smaller than the cross-validation (XVal).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/429/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic breeding value accuracy of the validation set containing young animals (YoungVal), for all traits, using a standard polygenic BLUP (BLUP), BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2079668376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic breeding value accuracy of the validation set containing young animals (YoungVal), for all traits, using a standard polygenic BLUP (BLUP), BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/429/F6.medium.gif" width="440" height="423"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/4/429/F6.medium.gif" width="440" height="423"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/429/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/429/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461393" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-32" class="first-child">Genomic breeding value accuracy of the validation set containing young animals (YoungVal), for all traits, using a standard polygenic BLUP (BLUP), BayesB with phenotypes (BayesB_ph), the single-step approach (SStep), and BayesB with estimated breeding values (BayesB_ebv).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-33">A dataset was created and a strategy developed to evaluate the accuracy of genomic selection approaches in livestock populations, especially when targeting populations where progeny-tested breeding values are not available on large numbers of individuals. The dataset was established for the purpose of implementing genomic selection in a production environment and so research utilizing the data can have direct implications for the application of genomic tools in breeding programs. These, and similar, data have been extensively analyzed within PIC and have been used to show the potential value of genomics in pig breeding, in terms of increased EBV accuracy at an early age (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Cleveland <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Forni <em>et al.</em> 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Deeb <em>et al.</em> 2011</a>), to describe the genetic architecture of a commercial pig population (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Deeb <em>et al.</em> 2010</a>), and to develop alternative methodologies for genomic prediction (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Forni <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">González-Recio and Forni 2011</a>). The structure of this dataset allows for testing approaches considering varying levels of pedigree and/or genomic relationships because animals from multiple generations and both sexes have high-density genotypes. The structure also highlights the need to develop methods to account for a distribution of EBV accuracies when validating the accuracy of gEBV. Genus plc encourages researchers who wish to explore this (and other) datasets further to discuss potential collaborations.</p><p id="p-34">The dataset enabled the testing of popular genomic prediction methods using real data, while the availability of both phenotypes and progeny-tested pEBV allowed for inferences about the value of sources of phenotypic information to different methods. The BayesB approach using de-regressed pEBV (BayesB_ebv) outperformed the other methods across almost all traits, which was not unexpected due to the nature of the data. Because they result from a production genetic evaluation using all available data, the pEBV contain information on the individual and its relatives in the full PIC pedigree. The SStep approach yielded accuracies similar to BayesB_ebv, but smaller in most cases. No strong evidence exists to support the existence of major QTL for these traits, and therefore, BayesB would not be expected to have a large advantage (results not shown). For traits affected by large numbers of loci (genes), the expectation is that a genomic BLUP approach should perform similarly to BayesB (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Luan <em>et al.</em> 2009</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2010</a>). In SStep, however, genomic relationships are augmented by pedigree relationships between genotyped and un-genotyped animals for a more accurate modeling of individual relationships, especially when only a subset of the population is genotyped (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Legarra <em>et al.</em> 2009</a>). Assuming no large QTL exist, any large differences between BayesB_ebv and SStep may be due to differences in the information content of the phenotypes used by each method and to differences in the training set size relative to differences in effective heritability. SStep outperformed BLUP, indicating that genomic information was useful, but only when the heritability was high. These findings, taken together, highlight the need for additional genotyping to improve lowly heritable traits using genomics, particularly when only phenotypes are available.</p><p id="p-35">Including genomic information in a genetic evaluation is expected to increase the accuracy of the EBV, but the results presented here (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) illustrate the potential to actually decrease the accuracy when analyzing phenotypes, especially when the heritability is low. When heritability is not high, large numbers of genotyped and phenotyped animals are needed to achieve even moderate gEBV accuracy (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Goddard 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Goddard and Hayes 2009</a>). Such numbers were not available in this dataset. Alternatively, the heritability of the trait can be increased by using de-regressed progeny test EBV as phenotypes, in which the effective heritability is proportional to the pEBV accuracy (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Garrick <em>et al.</em> 2009</a>). The expected accuracy of the gEBV will then be based on this higher heritability (<em>e.g.</em> BayesB_ph <em>vs.</em> BayesB_ebv). Often, the number of available genotyped samples is based on practical or budgetary considerations, and the decision to incorporate genomic information for a particular trait and the method that will be used depends on the heritability and the availability of progeny and pedigree information.</p><p id="p-36">The gEBV accuracy based on relatedness between training and validation sets increased with an increase in pedigree relationship (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">4</a>). Using a count of genomic coefficients exceeding a threshold showed this relationship even more clearly (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). These results indicate that expected gEBV accuracies from validation could be biased when highly related individuals are in both training and validation. This relatedness may cause an overestimation of expected accuracies when the target application consists of less-related samples (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2010</a>). In practice, the population used for training is often composed of the parents and ancestors of the individuals for which gEBV are required. A certain amount of relatedness between populations is appropriate, but the validation should aim to simulate the application as closely as possible. Interpretation of the accuracy of gEBV without knowledge of the pedigree relationships within the population is not optimal.</p><p id="p-37">For pig populations, where genomic predictions of young selection candidates within a single line are often desired, creating a validation dataset using young animals with high accuracy pEBV is difficult due to the structure of the breeding program. Because relatively few individuals have large amounts of progeny information, it is generally impossible to high-density genotype large numbers of individuals with high-accuracy pEBV. The younger animals in the dataset that may be used in validation are typically not suitable because of low-accuracy pEBV for many traits, which would provide a poor estimate of gEBV accuracy. Using older animals in this role would require removing their progeny from the training set, reducing the training set size to unacceptable levels. The small test performed here resulted in gEBV accuracies from YoungVal (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>) that were smaller than those in XVal (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), but when accuracies were evaluated in XVal considering pedigree and genomic relationships, they were similar to those from YoungVal. These results suggest that when it is not possible to simulate the application of a genomic evaluation for validation purposes, such as for T1 and T2 in this study, a k-fold cross-validation can provide appropriate estimates of gEBV accuracy when specifying the relatedness between training and validation. Additionally, genomic relationships may help determine the bounds of potential gEBV accuracy given the projected relatedness between datasets.</p><div id="sec-12" class="subsection"><h3>Implications</h3><p id="p-38">The results of this study indicate that the differences between methods to predict gEBV depend primarily on the information content of the phenotype, rather than on the relationship between training and validation, as expressed by the minimal level of reranking between methods. The conclusions drawn concerning the impact of relatedness on expected accuracies from validation can also be applied to genotyping strategies for implementing genomic selection programs. The relatedness between the training and prediction populations will influence the gEBV accuracy, and therefore, high levels of relatedness are desirable for breeding program design. Such high levels are most easily accomplished by genotyping both parents of prediction animals, but more selective genotyping may be possible by maximizing genomic relationships between the populations in which more distant pedigree relationships may be useful.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">Nader Deeb, Olwen Southwood, and Lizhen Wang are acknowledged for their assistance in creating the dataset. J.M.H. was funded by Australian Research Council project LP100100880, of which Genus plc is a co-funder.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001453/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001453/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-40"><em>Edited by Dirk-Jan de Koning and Lauren M. McIntyre</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-10-20"><span class="received-label">Received </span>October 20, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-12-19"><span class="accepted-label">Accepted </span>December 19, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Cleveland <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different models of genetic variation and their effect on genomic evaluation</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deeb</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-comment">Prediction of genomic breeding values in a commercial pig population. Proceedings of the 9<sup>th</sup> World Congress on Genetics Applied to Livestock Production. Leipzig, Germany, August 1–6, 2010, Paper 0266</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inbreeding in genome-wide selection</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D369%26rft.epage%253D376%26rft.atitle%253DInbreeding%2Bin%2Bgenome-wide%2Bselection.%26rft_id%253Dinfo%253Apmid%252F18076474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18076474&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.4" data-doi="10.2105/AJPH.41.3.279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dawber</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadors</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">F. E.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">Epidemiological approaches to heart disease: the Framingham Study</span>. <abbr class="cit-jnl-abbrev">Am. J. Public Health</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPublic%2BHealth%26rft.stitle%253DAJPH%26rft.aulast%253DDawber%26rft.auinit1%253DT.%2BR.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D279%26rft.epage%253D286%26rft.atitle%253DEpidemiological%2BApproaches%2Bto%2BHeart%2BDisease%253A%2BThe%2BFramingham%2BStudy%26rft_id%253Dinfo%253Adoi%252F10.2105%252FAJPH.41.3.279%26rft_id%253Dinfo%253Apmid%252F14819398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2105/AJPH.41.3.279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14819398&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1951XR29800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deeb</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-comment">Linkage disequilibrium decay in commercial pigs. Proceedings of the International Plant and Animal Genome XVIII Conference. San Diego, CA, January 9–13, 2010, P602</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deeb</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mileham</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gladney</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-comment">Genomic selection using small panels. Proceedings of the International Plant and Animal Genome XIX Conference. San Diego, CA, January 15–19, 2011, P606</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-comment"><em>GenSel: User Manual for a Portfolio of Genomic Selection-related Analyses</em>. Ed. 2 for v. 2.12. Animal Breeding and Genetics, Iowa State University, Ames, IA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deeb</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-comment">Scaling the genomic relationship matrix for single-step evaluation using phenotypic, pedigree and genomic information. Proceedings of the 9<sup>th</sup> World Congress on Genetics Applied to Livestock Production. Leipzig, Germany, August 1–6, 2010, Paper PP2–152</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.9" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.10" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.11" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.12" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.13" data-doi="10.1186/1297-9686-43-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide prediction of discrete traits using Bayesian regressions and machine learning</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGonzalez-Recio%26rft.auinit1%253DO.%26rft.volume%253D43%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DGenome-wide%2Bprediction%2Bof%2Bdiscrete%2Btraits%2Busing%2BBayesian%2Bregressions%2Band%2Bmachine%2Blearning.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-7%26rft_id%253Dinfo%253Apmid%252F21329522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329522&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.14" data-doi="10.1186/1297-9686-42-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichtner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic relationship information on genomic breeding values in German Holstein cattle</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D42%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenetic%2Brelationship%2Binformation%2Bon%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BGerman%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-5%26rft_id%253Dinfo%253Apmid%252F20170500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20170500&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.15" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection in dairy cattle: progress and challenges</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A phasing and imputation method for pedigreed populations that results in a single stage genomic evaluation method</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> (<span class="cit-comment">in review</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.17" data-doi="10.1038/nature02168"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The International HapMap Consortium</span>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The International HapMap Project</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D426%26rft.issue%253D6968%26rft.spage%253D789%26rft.epage%253D796%26rft.atitle%253DThe%2BInternational%2BHapMap%2BProject.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02168%26rft_id%253Dinfo%253Apmid%252F14685227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14685227&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187342000046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.18" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.19" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.20" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.21" data-doi="10.1534/genetics.109.107391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svendsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The accuracy of genomic selection in Norwegian Red cattle assessed by cross-validation</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.107391%26rft_id%253Dinfo%253Apmid%252F19704013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My8zLzExMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.23" data-doi="10.3168/jds.2009-2064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Computing procedures for genetic evaluation including phenotypic, full pedigree, and genomic information</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4648</span>–<span class="cit-lpage">4655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMisztal%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4648%26rft.epage%253D4655%26rft.atitle%253DComputing%2Bprocedures%2Bfor%2Bgenetic%2Bevaluation%2Bincluding%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2064%26rft_id%253Dinfo%253Apmid%252F19700728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700728&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.24"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The 1000 Genomes Project Consortium</span>, <span class="cit-pub-date">2010</span> <span class="cit-comment">A map of human genome variation from population-scale sequencing. Nature 467: 1061–1073</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.25" data-doi="10.1371/journal.pone.0006524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affara</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amaral</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRamos%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6524%26rft.epage%253De6524%26rft.atitle%253DDesign%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bin%2Bthe%2Bpig%2Busing%2BSNPs%2Bidentified%2Band%2Bcharacterized%2Bby%2Bnext%2Bgeneration%2Bsequencing%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006524%26rft_id%253Dinfo%253Apmid%252F19654876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19654876&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.26" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.429.27" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T.S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions for North American Holstein bulls</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/4/429.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/4/425" title="Simulated Data for Genomic Selection and Genome-Wide Association Studies Using a Combination of Coalescent and Gene Drop Methods" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/4/437" title="A Systematic Genetic Screen to Dissect the MicroRNA Pathway in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/4"><span>Volume 2 Issue 4, April 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/4.cover-source.jpg?itok=0lh-7MFD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pyVFp2sY5l90Kt-1BxWdmtAeLNhE3Tf8FOqG37MbzBo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/4/429" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/4/429" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/4/429" class="active" data-icon-position="" data-hide-link-title="0">A Common Dataset for Genomic Analysis of Livestock Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459263" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-E5P7IazTYE5v23r3IJtVIUCtIh5-qclDoqCFugErqjE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/4/429.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/4/429" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-BjWM5pSfDPvWClURVMePXTv7125fQ7674aSFvWEHwGk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/4/429" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/4/429.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459263" id="node459263--2" data-pisa="ggg;2/4/429" data-pisa-master="ggg;2/4/429" data-apath="/ggg/2/4/429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Common Dataset for Genomic Analysis of Livestock Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew A. Cleveland</span>, <span class="highwire-citation-author" data-delta="1">John M. Hickey</span> and <span class="highwire-citation-author" data-delta="2">Selma Forni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">429-435;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459263--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew A. Cleveland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genus plc, Hendersonville, Tennessee 37075</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BA.%2BCleveland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cleveland%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BA.%2BCleveland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:matthew.cleveland@genusplc.com" class="" data-icon-position="" data-hide-link-title="0">matthew.cleveland@genusplc.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environmental and Rural Science, University of New England, Armidale NSW 2351, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Selma Forni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genus plc, Hendersonville, Tennessee 37075</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Selma%2BForni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forni%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASelma%2BForni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/4/429" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/4/429&amp;title=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/4/429&amp;title=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/4/429&amp;title=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/4/429&amp;text=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/4/429&amp;title=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/4/429&amp;t=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/4/429&amp;title=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/4/429&amp;title=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459263/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459263" id="node459263--3" data-pisa="ggg;2/4/429" data-pisa-master="ggg;2/4/429" data-apath="/ggg/2/4/429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Common Dataset for Genomic Analysis of Livestock Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew A. Cleveland</span>, <span class="highwire-citation-author" data-delta="1">John M. Hickey</span> and <span class="highwire-citation-author" data-delta="2">Selma Forni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">429-435;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459263--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew A. Cleveland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genus plc, Hendersonville, Tennessee 37075</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BA.%2BCleveland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cleveland%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BA.%2BCleveland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:matthew.cleveland@genusplc.com" class="" data-icon-position="" data-hide-link-title="0">matthew.cleveland@genusplc.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environmental and Rural Science, University of New England, Armidale NSW 2351, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Selma Forni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genus plc, Hendersonville, Tennessee 37075</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Selma%2BForni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forni%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASelma%2BForni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=429&amp;rft.epage=435&amp;rft.atitle=A%20Common%20Dataset%20for%20Genomic%20Analysis%20of%20Livestock%20Populations&amp;rft.volume=2&amp;rft.issue=4&amp;rft.date=2012-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cleveland%2C+Matthew+A.&amp;rft.au=Hickey%2C+John+M.&amp;rft.au=Forni%2C+Selma"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459263/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459263/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459263/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459263/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459263/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459263/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459263/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459263/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459263/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459263/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459263/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459263/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/4/429" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/4/429.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/4/429.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/4/429.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/4/429.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
