<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/4/453" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/4/453.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/4/453.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/4/453.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/4/453.atom" />
<meta name="HW.pisa" content="ggg;2/4/453" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Predicting the Fission Yeast Protein Interaction Network" />
<meta name="DC.Identifier" content="10.1534/g3.111.001560" />
<meta name="DC.Date" content="2012-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Pancaldi et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A systems-level understanding of biological processes and information flow requires the mapping of cellular component interactions, among which protein–protein interactions are particularly important. Fission yeast ( Schizosaccharomyces pombe ) is a valuable model organism for which no systematic protein-interaction data are available. We exploited gene and protein properties, global genome regulation datasets, and conservation of interactions between budding and fission yeast to predict fission yeast protein interactions in silico . We have extensively tested our method in three ways: first, by predicting with 70–80% accuracy a selected high-confidence test set; second, by recapitulating interactions between members of the well-characterized SAGA co-activator complex; and third, by verifying predicted interactions of the Cbf11 transcription factor using mass spectrometry of TAP-purified protein complexes. Given the importance of the pathway in cell physiology and human disease, we explore the predicted sub-networks centered on the Tor1/2 kinases. Moreover, we predict the histidine kinases Mak1/2/3 to be vital hubs in the fission yeast stress response network, and we suggest interactors of argonaute 1, the principal component of the siRNA-mediated gene silencing pathway, lost in budding yeast but preserved in S. pombe . Of the new high-quality interactions that were discovered after we started this work, 73% were found in our predictions. Even though any predicted interactome is imperfect, the protein network presented here can provide a valuable basis to explore biological processes and to guide wet-lab experiments in fission yeast and beyond. Our predicted protein interactions are freely available through PInt, an online resource on our website ([www.bahlerlab.info/PInt][1]).

 [1]: http://www.bahlerlab.info/PInt" />
<meta name="DC.Contributor" content="Vera Pancaldi" />
<meta name="DC.Contributor" content="Ömer S. Saraç" />
<meta name="DC.Contributor" content="Charalampos Rallis" />
<meta name="DC.Contributor" content="Janel R. McLean" />
<meta name="DC.Contributor" content="Martin Převorovský" />
<meta name="DC.Contributor" content="Kathleen Gould" />
<meta name="DC.Contributor" content="Andreas Beyer" />
<meta name="DC.Contributor" content="Jürg Bähler" />
<meta name="article:published_time" content="2012-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Predicting the Fission Yeast Protein Interaction Network" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A systems-level understanding of biological processes and information flow requires the mapping of cellular component interactions, among which protein–protein interactions are particularly important. Fission yeast (&lt;i&gt;Schizosaccharomyces pombe&lt;/i&gt;) is a valuable model organism for which no systematic protein-interaction data are available. We exploited gene and protein properties, global genome regulation datasets, and conservation of interactions between budding and fission yeast to predict fission yeast protein interactions &lt;i&gt;in silico&lt;/i&gt;. We have extensively tested our method in three ways: first, by predicting with 70–80% accuracy a selected high-confidence test set; second, by recapitulating interactions between members of the well-characterized SAGA co-activator complex; and third, by verifying predicted interactions of the Cbf11 transcription factor using mass spectrometry of TAP-purified protein complexes. Given the importance of the pathway in cell physiology and human disease, we explore the predicted sub-networks centered on the Tor1/2 kinases. Moreover, we predict the histidine kinases Mak1/2/3 to be vital hubs in the fission yeast stress response network, and we suggest interactors of argonaute 1, the principal component of the siRNA-mediated gene silencing pathway, lost in budding yeast but preserved in &lt;i&gt;S. pombe&lt;/i&gt;. Of the new high-quality interactions that were discovered after we started this work, 73% were found in our predictions. Even though any predicted interactome is imperfect, the protein network presented here can provide a valuable basis to explore biological processes and to guide wet-lab experiments in fission yeast and beyond. Our predicted protein interactions are freely available through PInt, an online resource on our website (www.bahlerlab.info/PInt).&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/04/01" />
<meta name="citation_mjid" content="ggg;2/4/453" />
<meta name="citation_id" content="2/4/453" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/4/453" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/4/453.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/4/453.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/4/453.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001560" />
<meta name="citation_pmid" content="22540037" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="4" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="453" />
<meta name="citation_lastpage" content="467" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vera Pancaldi" />
<meta name="citation_author_institution" content="Department of Genetics, Evolution, and Environment and" />
<meta name="citation_author_institution" content="UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom" />
<meta name="citation_author_email" content="v.pancaldi@ucl.ac.uk" />
<meta name="citation_author" content="Ömer S. Saraç" />
<meta name="citation_author_institution" content="Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and" />
<meta name="citation_author" content="Charalampos Rallis" />
<meta name="citation_author_institution" content="Department of Genetics, Evolution, and Environment and" />
<meta name="citation_author_institution" content="UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom" />
<meta name="citation_author" content="Janel R. McLean" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute" />
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232" />
<meta name="citation_author" content="Martin Převorovský" />
<meta name="citation_author_institution" content="Department of Genetics, Evolution, and Environment and" />
<meta name="citation_author_institution" content="UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom" />
<meta name="citation_author" content="Kathleen Gould" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute" />
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232" />
<meta name="citation_author" content="Andreas Beyer" />
<meta name="citation_author_institution" content="Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and" />
<meta name="citation_author" content="Jürg Bähler" />
<meta name="citation_author_institution" content="Department of Genetics, Evolution, and Environment and" />
<meta name="citation_author_institution" content="UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=P. Aloy;citation_author=R. B. Russell;citation_title=The third dimension for protein interactions and complexes.;citation_pages=633-638;citation_volume=27;citation_year=2002;citation_issue=12;citation_pmid=12468233;citation_doi=10.1016/S0968-0004(02)02204-1" />
<meta name="citation_reference" content="citation_journal_title=Bioscience, Biotechnology, and Biochemistry;citation_journal_abbrev=Bioscience, Biotechnology, and Biochemistry;citation_author=K. Aoyama;citation_author=H. Aiba;citation_author=T. Mizuno;citation_title=Genetic analysis of the His-to-Asp phosphorelay implicated in mitotic cell cycle control: involvement of histidine-kinase genes of Schizosaccharomyces pombe.;citation_pages=2347-2352;citation_volume=65;citation_year=2001;citation_issue=10;citation_pmid=11758939;citation_doi=10.1271/bbb.65.2347" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. Ball;citation_author=J. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_abbrev=Future Oncol;citation_author=I. A. Babar;citation_author=F. J. Slack;citation_author=J. B. Weidhaas;citation_title=miRNA modulation of the cellular stress response.;citation_pages=289-298;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18407740;citation_doi=10.2217/14796694.4.2.289" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. H. Bayne;citation_author=S. A. White;citation_author=A. Kagansky;citation_author=D. A. Bijos;citation_author=L. Sanchez-Pulido;citation_title=Stc1: a critical link between RNAi and chromatin modification required for heterochromatin integrity.;citation_pages=666-677;citation_volume=140;citation_year=2010;citation_issue=5;citation_pmid=20211136;citation_doi=10.1016/j.cell.2010.01.038" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=A. Ben-Hur;citation_author=W. Noble;citation_title=Kernel methods for predicting protein-protein interactions;citation_pages=i38-i46;citation_volume=21;citation_year=2005;citation_issue=suppl_1;citation_pmid=15961482;citation_doi=10.1093/bioinformatics/bti1016" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=A. Ben-Hur;citation_author=W. Noble;citation_title=Choosing negative examples for the prediction of protein-protein interactions;citation_pages=S2;citation_volume=7;citation_year=2006;citation_pmid=16723005" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Cancer;citation_journal_abbrev=Nat Rev Cancer;citation_author=M.-A. Bjornsti;citation_author=P. J. Houghton;citation_title=The TOR pathway: a target for cancer therapy.;citation_pages=335-348;citation_volume=4;citation_year=2004;citation_issue=5;citation_pmid=15122205;citation_doi=10.1038/nrc1362" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Metab;citation_author=N. D. Bonawitz;citation_author=M. Chatenay-Lapointe;citation_author=Y. Pan;citation_author=G. S. Shadel;citation_title=Reduced TOR signaling extends chronological life span via increased respiration and upregulation of mitochondrial gene expression.;citation_pages=265-277;citation_volume=5;citation_year=2007;citation_issue=4;citation_pmid=17403371;citation_doi=10.1016/j.cmet.2007.02.009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. I. Boyle;citation_author=S. Weng;citation_author=J. Gollub;citation_author=H. Jin;citation_author=D. Botstein;citation_title=GO::TermFinder—open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes;citation_volume=20;citation_year=2004;citation_pmid=15297299;citation_doi=10.1093/bioinformatics/bth456" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=P. Braun;citation_author=M. Tasan;citation_author=M. Dreze;citation_author=M. Barrios-Rodiles;citation_author=I. Lemmens;citation_title=An experimentally derived confidence score for binary protein-protein interactions.;citation_pages=91-97;citation_volume=6;citation_year=2009;citation_issue=1;citation_pmid=19060903;citation_doi=10.1038/nmeth.1281" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random Forests;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Breitkreutz;citation_author=H. Choi;citation_author=J. R. Sharom;citation_author=L. Boucher;citation_author=V. Neduva;citation_title=A Global Protein Kinase and Phosphatase Interaction Network in Yeast;citation_pages=1043-1046;citation_volume=328;citation_year=2010;citation_issue=5981;citation_pmid=20489023;citation_doi=10.1126/science.1176495" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=V. Buck;citation_author=J. Quinn;citation_author=T. Soto Pino;citation_author=H. Martin;citation_author=J. Saldanha;citation_title=Peroxide Sensors for the Fission Yeast Stress-activated Mitogen-activated Protein Kinase Pathway;citation_pages=407-419;citation_volume=12;citation_year=2001;citation_issue=2;citation_pmid=11179424;citation_doi=10.1091/mbc.12.2.407" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=M. Bühler;citation_author=N. Spies;citation_author=D. P. Bartel;citation_author=D. Moazed;citation_title=TRAMP-mediated RNA surveillance prevents spurious entry of RNAs into the Schizosaccharomyces pombe siRNA pathway.;citation_pages=1015-1023;citation_volume=15;citation_year=2008;citation_issue=10;citation_pmid=18776903;citation_doi=10.1038/nsmb.1481" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=L. Burger;citation_author=E. van Nimwegen;citation_title=Accurate prediction of protein-protein interactions from sequence alignments using a Bayesian method;citation_pages=165-165;citation_volume=4;citation_year=2008;citation_issue=1;citation_pmid=18277381;citation_doi=10.1038/msb4100203" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. S. Burguete;citation_author=T. D. Fenn;citation_author=A. T. Brunger;citation_author=S. R. Pfeffer;citation_title=Rab and Arl GTPase family members cooperate in the localization of the golgin GCC185.;citation_pages=286-298;citation_volume=132;citation_year=2008;citation_issue=2;citation_pmid=18243103;citation_doi=10.1016/j.cell.2007.11.048" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=Y. Cao;citation_author=J. A. Espinola;citation_author=E. Fossale;citation_author=A. C. Massey;citation_author=A. M. Cuervo;citation_title=Autophagy is disrupted in a knock-in mouse model of juvenile neuronal ceroid lipofuscinosis;citation_volume=281;citation_year=2006;citation_pmid=16714284;citation_doi=10.1074/jbc.M602180200" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=D. Chen;citation_author=W. M. Toone;citation_author=J. Mata;citation_author=R. Lyne;citation_author=G. Burns;citation_title=Global transcriptional responses of fission yeast to environmental stress;citation_volume=14;citation_year=2003;citation_pmid=12529438;citation_doi=10.1091/mbc.E02-08-0499" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=D. Chen;citation_author=C. R. Wilkinson;citation_author=S. Watt;citation_author=C. J. Penkett;citation_author=W. M. Toone;citation_title=Multiple pathways differentially regulate global oxidative stress responses in fission yeast;citation_volume=19;citation_year=2008;citation_pmid=18003976;citation_doi=10.1091/mbc.E07-08-0735" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=C. Cortes;citation_author=V. Vapnik;citation_title=Support-vector networks;citation_pages=273-297;citation_volume=20;citation_year=1995;citation_doi=10.1007/BF00994018" />
<meta name="citation_reference" content="citation_journal_title=BMC cell biology [electronic resource];citation_journal_abbrev=BMC Cell Biol;citation_author=B. Cuenca-Bono;citation_author=V. Garcia-Molinero;citation_author=P. Pascual-Garcia;citation_author=E. Garcia-Oliver;citation_author=A. Llopis;citation_title=A novel link between Sus1 and the cytoplasmic mRNA decay machinery suggests a broad role in mRNA metabolism.;citation_pages=19-19;citation_volume=11;citation_year=2010;citation_pmid=20230609;citation_doi=10.1186/1471-2121-11-19" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. De Bodt;citation_author=S. Proost;citation_author=K. Vandepoele;citation_author=P. Rouze;citation_author=Y. Van de Peer;citation_title=Predicting protein-protein interactions in Arabidopsis thaliana through integration of orthology, gene ontology and co-expression.;citation_pages=288-288;citation_volume=10;citation_year=2009;citation_pmid=19563678;citation_doi=10.1186/1471-2164-10-288" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Desikan;citation_author=J. Horak;citation_author=C. Chaban;citation_author=V. Mira-Rodado;citation_author=J. Witthoft;citation_title=The histidine kinase AHK5 integrates endogenous and environmental signals in Arabidopsis guard cells.;citation_pages=e2491-e2491;citation_volume=3;citation_year=2008;citation_issue=6;citation_pmid=18560512;citation_doi=10.1371/journal.pone.0002491" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. J. Dixon;citation_author=Y. Fedyshyn;citation_author=J. L. Koh;citation_author=T. S. Prasad;citation_author=C. Chahwan;citation_title=Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes;citation_volume=105;citation_year=2008;citation_pmid=18931302;citation_doi=10.1073/pnas.0806261105" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. P. Fernandes;citation_author=A. Annibale;citation_author=J. Kleinjung;citation_author=A. C. C. Coolen;citation_author=F. Fraternali;citation_title=Protein networks reveal detection bias and species consistency when analysed by information-theoretic methods.;citation_pages=e12083-e12083;citation_volume=5;citation_year=2010;citation_issue=8;citation_pmid=20805870;citation_doi=10.1371/journal.pone.0012083" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Formstecher;citation_author=S. Aresta;citation_author=V. Collura;citation_author=A. Hamburger;citation_author=A. Meil;citation_title=Protein interaction mapping: a Drosophila case study;citation_volume=15;citation_year=2005;citation_pmid=15710747;citation_doi=10.1101/gr.2659105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. L. Freeman-Cook;citation_author=E. B. Gomez;citation_author=E. J. Spedale;citation_author=J. Marlett;citation_author=S. L. Forsburg;citation_title=Conserved locus-specific silencing functions of Schizosaccharomyces pombe sir2+;citation_volume=169;citation_year=2005;citation_pmid=15545655;citation_doi=10.1534/genetics.104.032714" />
<meta name="citation_reference" content="citation_journal_title=Aging cell;citation_journal_abbrev=Aging Cell;citation_author=B. Q. Gan;citation_author=B. L. Tang;citation_title=Sirt1âs beneficial roles in neurodegenerative diseases - a chaperonin containing TCP-1 (CCT) connection?;citation_pages=924-929;citation_volume=9;citation_year=2010;citation_issue=5;citation_pmid=20569238;citation_doi=10.1111/j.1474-9726.2010.00597.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A.-C. Gavin;citation_author=M. Bosche;citation_author=R. Krause;citation_author=P. Grandi;citation_author=M. Marzioch;citation_title=Functional organization of the yeast proteome by systematic analysis of protein complexes.;citation_pages=141-147;citation_volume=415;citation_year=2002;citation_issue=6868;citation_pmid=11805826;citation_doi=10.1038/415141a" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=A.-C. Gavin;citation_author=K. Maeda;citation_author=S. Kühner;citation_title=Recent advances in charting protein-protein interaction: mass spectrometry-based approaches.;citation_pages=42-49;citation_volume=22;citation_year=2011;citation_issue=1;citation_pmid=20934865;citation_doi=10.1016/j.copbio.2010.09.007" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=G. Hart;citation_author=A. Ramani;citation_author=E. Marcotte;citation_title=How complete are current yeast and human protein-interaction networks?;citation_pages=120-120;citation_volume=7;citation_year=2006;citation_issue=11;citation_pmid=17147767;citation_doi=10.1186/gb-2006-7-11-120" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=A. Hayashi;citation_author=D. Da-Qiao;citation_author=C. Tsutsumi;citation_author=Y. Chikashige;citation_author=H. Masuda;citation_title=Localization of gene products using a chromosomally tagged GFP-fusion library in the fission yeast Schizosaccharomyces pombe.;citation_pages=217-225;citation_volume=14;citation_year=2009;citation_issue=2;citation_pmid=19170768;citation_doi=10.1111/j.1365-2443.2008.01264.x" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. Helmlinger;citation_author=S. Marguerat;citation_author=J. Villén;citation_author=S. P. Gygi;citation_author=J. Bähler;citation_title=The S. pombe SAGA complex controls the switch from proliferation to sexual differentiation through the opposing roles of its subunits Gcn5 and Spt8;citation_pages=3184-3195;citation_volume=22;citation_year=2008;citation_issue=22;citation_pmid=19056896;citation_doi=10.1101/gad.1719908" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=D. Helmlinger;citation_author=S. Marguerat;citation_author=J. Villen;citation_author=D. L. Swaney;citation_author=S. P. Gygi;citation_title=Tra1 has specific regulatory roles, rather than global functions, within the SAGA co-activator complex.;citation_pages=2843-2852;citation_volume=30;citation_year=2011;citation_issue=14;citation_pmid=21642955;citation_doi=10.1038/emboj.2011.181" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=K. Ingvarsdottir;citation_author=N. J. Krogan;citation_author=N. C. T. Emre;citation_author=A. Wyce;citation_author=N. J. Thompson;citation_title=H2B Ubiquitin Protease Ubp8 and Sgf11 Constitute a Discrete Functional Module within the Saccharomyces cerevisiae SAGA Complex;citation_pages=1162-1172;citation_volume=25;citation_year=2005;citation_issue=3;citation_pmid=15657441;citation_doi=10.1128/MCB.25.3.1162-1172.2005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Ito;citation_author=T. Chiba;citation_author=R. Ozawa;citation_author=M. Yoshida;citation_author=M. Hattori;citation_title=A comprehensive two-hybrid analysis to explore the yeast protein interactome;citation_volume=98;citation_year=2001;citation_pmid=11283351;citation_doi=10.1073/pnas.061034498" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Jansen;citation_author=H. Yu;citation_author=D. Greenbaum;citation_author=Y. Kluger;citation_author=N. J. Krogan;citation_title=A Bayesian Networks Approach for Predicting Protein-Protein Interactions from Genomic Data;citation_pages=449-453;citation_volume=302;citation_year=2003;citation_issue=5644;citation_pmid=14564010;citation_doi=10.1126/science.1087361" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. J. Jensen;citation_author=M. Kuhn;citation_author=M. Stark;citation_author=S. Chaffron;citation_author=C. Creevey;citation_title=STRING 8—a global view on proteins and their functional interactions in 630 organisms;citation_volume=37;citation_year=2009;citation_pmid=18940858;citation_doi=10.1093/nar/gkn760" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=E. Koutelou;citation_author=C. L. Hirsch;citation_author=S. Y. R. Dent;citation_title=Multiple faces of the SAGA complex.;citation_pages=374-382;citation_volume=22;citation_year=2010;citation_issue=3;citation_pmid=20363118;citation_doi=10.1016/j.ceb.2010.03.005" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=T. Krell;citation_author=J. Lacal;citation_author=A. Busch;citation_author=H. Silva-Jimenez;citation_author=M. E. Guazzaroni;citation_title=Bacterial sensor kinases: diversity in the recognition of environmental signals.;citation_pages=539-559;citation_volume=64;citation_year=2010;citation_pmid=20825354;citation_doi=10.1146/annurev.micro.112408.134054" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. J. Krogan;citation_author=G. Cagney;citation_author=H. Yu;citation_author=G. Zhong;citation_author=X. Guo;citation_title=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae.;citation_pages=637-643;citation_volume=440;citation_year=2006;citation_issue=7084;citation_pmid=16554755;citation_doi=10.1038/nature04670" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=D. H. Lackner;citation_author=T. H. Beilharz;citation_author=S. Marguerat;citation_author=J. Mata;citation_author=S. Watt;citation_title=A network of multiple regulatory layers shapes gene expression in fission yeast.;citation_pages=145-155;citation_volume=26;citation_year=2007;citation_issue=1;citation_pmid=17434133;citation_doi=10.1016/j.molcel.2007.03.002" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=C. H. Lee;citation_author=K. Inoki;citation_author=M. Karbowniczek;citation_author=E. Petroulakis;citation_author=N. Sonenberg;citation_title=Constitutive mTOR activation in TSC mutants sensitizes cells to energy starvation and genomic damage via p53.;citation_pages=4812-4823;citation_volume=26;citation_year=2007;citation_issue=23;citation_pmid=17962806;citation_doi=10.1038/sj.emboj.7601900" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Li;citation_author=C. M. Armstrong;citation_author=N. Bertin;citation_author=H. Ge;citation_author=S. Milstein;citation_title=A map of the interactome network of the metazoan C. elegans;citation_volume=303;citation_year=2004;citation_pmid=14704431;citation_doi=10.1126/science.1091403" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. Liu;citation_author=L. Larsson;citation_author=A. Caballero;citation_author=X. Hao;citation_author=D. Öling;citation_title=The polarisome is required for segregation and retrograde transport of protein aggregates.;citation_pages=257-267;citation_volume=140;citation_year=2010;citation_issue=2;citation_pmid=20141839;citation_doi=10.1016/j.cell.2009.12.031" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Lopez-Maury;citation_author=S. Marguerat;citation_author=J. Bahler;citation_title=Tuning gene expression to changing environments: from rapid responses to evolutionary adaptation.;citation_pages=583-593;citation_volume=9;citation_year=2008;citation_issue=8;citation_pmid=18591982;citation_doi=10.1038/nrg2398" />
<meta name="citation_reference" content="citation_journal_title=Journal of proteome research;citation_journal_abbrev=J Proteome Res;citation_author=Z.-Q. Ma;citation_author=S. Dasari;citation_author=M. C. Chambers;citation_author=M. D. Litton;citation_author=S. M. Sobecki;citation_title=IDPicker 2.0: Improved protein assembly with high discrimination peptide identification filtering.;citation_pages=3872-3881;citation_volume=8;citation_year=2009;citation_issue=8;citation_pmid=19522537;citation_doi=10.1021/pr900360j" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. A. Martienssen;citation_author=M. Zaratiegui;citation_author=D. B. Goto;citation_title=RNA interference and heterochromatin in the fission yeast Schizosaccharomyces pombe.;citation_pages=450-456;citation_volume=21;citation_year=2005;citation_issue=8;citation_pmid=15979194;citation_doi=10.1016/j.tig.2005.06.005" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Matsuyama;citation_author=R. Arai;citation_author=Y. Yashiroda;citation_author=A. Shirai;citation_author=A. Kamata;citation_title=ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe.;citation_pages=841-847;citation_volume=24;citation_year=2006;citation_issue=7;citation_pmid=16823372;citation_doi=10.1038/nbt1222" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mass Spectrom.;citation_author=W. H. McDonald;citation_author=R. Ohi;citation_author=D. T. Miyamoto;citation_author=T. J. Mitchison;citation_author=J. R. Yates;citation_title=Comparison of three directly coupled HPLC MS/MS strategies for identification of proteins from complex mixtures: single-dimension LC-MS/MS, 2-phase MudPIT, and 3-phase MudPIT;citation_pages=245-251;citation_volume=219;citation_year=2002;citation_doi=10.1016/S1387-3806(02)00563-8" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=T. Motoyama;citation_author=K. Kadokura;citation_author=T. Ohira;citation_author=A. Ichiishi;citation_author=M. Fujimura;citation_title=A two-component histidine kinase of the rice blast fungus is involved in osmotic stress response and fungicide action.;citation_pages=200-212;citation_volume=42;citation_year=2005;citation_issue=3;citation_pmid=15707841;citation_doi=10.1016/j.fgb.2004.11.002" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=H. Mukaiyama;citation_author=M. Nakase;citation_author=T. Nakamura;citation_author=Y. Kakinuma;citation_author=K. Takegawa;citation_title=Autophagy in the fission yeast Schizosaccharomyces pombe.;citation_pages=1327-1334;citation_volume=584;citation_year=2010;citation_issue=7;citation_pmid=20036658;citation_doi=10.1016/j.febslet.2009.12.037" />
<meta name="citation_reference" content="citation_journal_title=Bioscience, Biotechnology, and Biochemistry;citation_journal_abbrev=Bioscience, Biotechnology, and Biochemistry;citation_author=N. Nakamichi;citation_author=H. Yanada;citation_author=H. Aiba;citation_author=K. Aoyama;citation_author=R. Ohmiya;citation_title=Characterization of the Prr1 response regulator with special reference to sexual development in Schizosaccharomyces pombe.;citation_pages=547-555;citation_volume=67;citation_year=2003;citation_issue=3;citation_pmid=12723602;citation_doi=10.1271/bbb.67.547" />
<meta name="citation_reference" content="citation_journal_title=Aging (Albany NY);citation_author=Y. Pan;citation_author=G. S. Shadel;citation_title=Extension of chronological life span by reduced TOR signaling requires down-regulation of Sch9p and involves increased mitochondrial OXPHOS complex density;citation_pages=131-145;citation_volume=1;citation_year=2009;citation_pmid=20157595" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Biosyst;citation_author=V. Pancaldi;citation_author=F. Schubert;citation_author=J. Bahler;citation_title=Meta-analysis of genome regulation and expression variability across hundreds of environmental and genetic perturbations in fission yeast.;citation_pages=543-552;citation_volume=6;citation_year=2010;citation_issue=3;citation_pmid=20174682;citation_doi=10.1039/b913876p" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=S. Patrick Walton;citation_author=M. Wu;citation_author=J. A. Gredell;citation_author=C. Chan;citation_title=Designing highly active siRNAs for therapeutic applications.;citation_pages=4806-4813;citation_volume=277;citation_year=2010;citation_issue=23;citation_pmid=21078115;citation_doi=10.1111/j.1742-4658.2010.07903.x" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=P. Perez;citation_author=S. A. Rincón;citation_title=Rho GTPases: regulation of cell polarity and growth in yeasts;citation_volume=426;citation_year=2010;citation_pmid=20175747;citation_doi=10.1042/BJ20091823" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Society Transactions;citation_journal_abbrev=Biochm. Soc. Trans.;citation_author=J. Petersen;citation_title=TOR signalling regulates mitotic commitment through stress-activated MAPK and Polo kinase in response to nutrient stress.;citation_pages=273-277;citation_volume=37;citation_year=2009;citation_issue=Pt 1;citation_pmid=19143645;citation_doi=10.1042/BST0370273" />
<meta name="citation_reference" content="citation_author=S. Pitre;citation_author=M. Alamgir;citation_author=J. R. Green;citation_author=M. Dumontier;citation_author=F. Dehne;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. Posas;citation_author=S. M. Wurgler-Murphy;citation_author=T. Maeda;citation_author=E. A. Witten;citation_author=T. C. Thai;citation_title=Yeast HOG1 MAP kinase cascade is regulated by a multistep phosphorelay mechanism in the SLN1-YPD1-SSK1 &quot;two-component&quot; osmosensor.;citation_pages=865-875;citation_volume=86;citation_year=1996;citation_issue=6;citation_pmid=8808622;citation_doi=10.1016/S0092-8674(00)80162-2" />
<meta name="citation_reference" content="citation_journal_title=Experimental cell research;citation_journal_abbrev=Exp Cell Res;citation_author=M. Převorovský;citation_author=T. Grousl;citation_author=J. Stanurová;citation_author=J. Rynes;citation_author=W. Nellen;citation_title=Cbf11 and Cbf12, the fission yeast CSL proteins, play opposing roles in cell adhesion and coordination of cell and nuclear division.;citation_pages=1533-1547;citation_volume=315;citation_year=2009;citation_issue=8;citation_pmid=19101542;citation_doi=10.1016/j.yexcr.2008.12.001" />
<meta name="citation_reference" content="citation_title=Evaluation of different biological data and computational classification methods for use in protein interaction prediction.;citation_author=Y. Qi;citation_author=Z. Bar-Joseph;citation_author=J. Klein-Seetharaman;citation_pages=490-500;citation_volume=63;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and metabolism;citation_journal_abbrev=Mol Genet Metab;citation_author=D. Rakheja;citation_author=S. B. Narayan;citation_author=M. J. Bennett;citation_title=The function of CLN3P, the Batten disease protein.;citation_pages=269-274;citation_volume=93;citation_year=2008;citation_issue=3;citation_pmid=18688960;citation_doi=10.1016/j.ymgme.2008.01.001" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=G. Rigaut;citation_author=A. Schevchenko;citation_author=B. Rutz;citation_author=M. Wilm;citation_author=M. Mann;citation_title=A generic protein purification method for protein complex characterization and proteome exploration.;citation_pages=1030-1032;citation_volume=17;citation_year=1999;citation_issue=10;citation_pmid=10504710;citation_doi=10.1038/13732" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=V. Risson;citation_author=L. Mazelin;citation_author=M. Roceri;citation_author=H. Sanchez;citation_author=V. Moncollin;citation_title=Muscle inactivation of mTOR causes metabolic and dystrophin defects leading to severe myopathy;citation_volume=187;citation_year=2009;citation_pmid=20008564;citation_doi=10.1083/jcb.200903131" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=R. H. Roberts-Galbraith;citation_author=J.-S. Chen;citation_author=J. Wang;citation_author=K. L. Gould;citation_title=The SH3 domains of two PCH family members cooperate in assembly of the Schizosaccharomyces pombe contractile ring;citation_volume=184;citation_year=2009;citation_pmid=19139265;citation_doi=10.1083/jcb.200806044" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=A. Roguev;citation_author=A. Shevchenko;citation_author=D. Schaft;citation_author=H. Thomas;citation_author=A. F. Stewart;citation_title=A comparative analysis of an orthologous proteomic environment in the yeasts Saccharomyces cerevisiae and Schizosaccharomyces pombe;citation_volume=3;citation_year=2004;citation_pmid=14617822" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Roguev;citation_author=S. Bandyopadhyay;citation_author=M. Zofall;citation_author=K. Zhang;citation_author=T. Fischer;citation_title=Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast;citation_volume=322;citation_year=2008;citation_pmid=18818364;citation_doi=10.1126/science.1162609" />
<meta name="citation_reference" content="citation_journal_title=Mini reviews in medicinal chemistry;citation_journal_abbrev=Mini Rev Med Chem;citation_author=S. L. Rowland;citation_author=G. F. King;citation_title=Histidine kinases as antimicrobial targets: prospects and pitfalls.;citation_pages=1144-1154;citation_volume=7;citation_year=2007;citation_issue=11;citation_pmid=18045218;citation_doi=10.2174/138955707782331759" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J.-F. Rual;citation_author=K. Venkatesan;citation_author=T. Hao;citation_author=T. Hirozane-Kishikawa;citation_author=A. Dricot;citation_title=Towards a proteome-scale map of the human protein-protein interaction network.;citation_pages=1173-1178;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16189514;citation_doi=10.1038/nature04209" />
<meta name="citation_reference" content="citation_journal_title=Science Signaling;citation_journal_abbrev=Sci Signal;citation_author=J. L. Santos;citation_author=K. Shiozaki;citation_title=Fungal Histidine Kinases;citation_pages=re1-re1;citation_volume=2001;citation_year=2001;citation_issue=98;citation_pmid=11752677;citation_doi=10.1126/stke.2001.98.re1" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=M. Schreiber;citation_author=I. Res;citation_author=A. Matter;citation_title=Protein kinases as antibacterial targets.;citation_pages=325-330;citation_volume=21;citation_year=2009;citation_issue=2;citation_pmid=19246185;citation_doi=10.1016/j.ceb.2009.01.026" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Shen;citation_author=J. Zhang;citation_author=X. Luo;citation_author=W. Zhu;citation_author=K. Yu;citation_title=Predicting protein-protein interactions based only on sequences information;citation_volume=104;citation_year=2007;citation_pmid=17360525;citation_doi=10.1073/pnas.0607879104" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Shevchenko;citation_author=A. Roguev;citation_author=D. Schaft;citation_author=L. Buchanan;citation_author=B. Habermann;citation_title=CHROMATIN CENTRAL: TOWARDS THE COMPARATIVE PROTEOME BY ACCURATE MAPPING OF THE YEAST PROTEOMIC ENVIRONMENT;citation_pages=R167;citation_volume=9(11);citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Singh;citation_author=D. Park;citation_author=J. Xu;citation_author=R. Hosur;citation_author=B. Berger;citation_title=Struct2Net: a web service to predict protein–protein interactions using a structure-based approach;citation_volume=38;citation_year=2010;citation_pmid=20513650;citation_doi=10.1093/nar/gkq481" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. E. Smoot;citation_author=K. Ono;citation_author=J. Ruscheinski;citation_author=P.-L. Wang;citation_author=T. Ideker;citation_title=Cytoscape 2.8: new features for data integration and network visualization;citation_volume=27;citation_year=2011;citation_pmid=21149340;citation_doi=10.1093/bioinformatics/btq675" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=K. Song;citation_author=K. E. Mach;citation_author=C.-Y. Chen;citation_author=T. Reynolds;citation_author=C. F. Albright;citation_title=A novel suppressor of ras1 in fission yeast, byr4, is a dosage- dependent inhibitor of cytokinesis;citation_pages=1307-1319;citation_volume=133;citation_year=1996;citation_issue=6;citation_pmid=8682866;citation_doi=10.1083/jcb.133.6.1307" />
<meta name="citation_reference" content="citation_author=C. Stark;citation_author=B.-J. Breitkreutz;citation_author=A. Chatr-aryamontri;citation_author=L. Boucher;citation_author=R. Oughtred;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=U. Stelzl;citation_author=U. Worm;citation_author=M. Lalowski;citation_author=C. Haenig;citation_author=F. Brembeck;citation_title=A human protein-protein interaction network: a resource for annotating the proteome.;citation_pages=957-968;citation_volume=122;citation_year=2005;citation_issue=6;citation_pmid=16169070;citation_doi=10.1016/j.cell.2005.08.029" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in pharmacology;citation_journal_abbrev=Curr Opin Pharmacol;citation_author=K. Stephenson;citation_author=J. A. Hoch;citation_title=Two-component and phosphorelay signal-transduction systems as therapeutic targets.;citation_pages=507-512;citation_volume=2;citation_year=2002;citation_issue=5;citation_pmid=12324251;citation_doi=10.1016/S1471-4892(02)00194-7" />
<meta name="citation_reference" content="citation_journal_title=Current medicinal chemistry;citation_journal_abbrev=Curr Med Chem;citation_author=K. Stephenson;citation_author=J. A. Hoch;citation_title=Developing inhibitors to selectively target two-component and phosphorelay signal transduction systems of pathogenic microorganisms.;citation_pages=765-773;citation_volume=11;citation_year=2004;citation_issue=6;citation_pmid=15032730;citation_doi=10.2174/0929867043455765" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Strobl;citation_author=A.-L. Boulesteix;citation_author=T. Kneib;citation_author=T. Augustin;citation_author=A. Zeileis;citation_title=Conditional variable importance for random forests.;citation_pages=307-307;citation_volume=9;citation_year=2008;citation_pmid=18620558;citation_doi=10.1186/1471-2105-9-307" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Szklarczyk;citation_author=A. Franceschini;citation_author=M. Kuhn;citation_author=M. Simonovic;citation_author=A. Roth;citation_title=The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored;citation_volume=39;citation_year=2011;citation_pmid=21045058;citation_doi=10.1093/nar/gkq973" />
<meta name="citation_reference" content="citation_journal_title=Journal of proteome research;citation_journal_abbrev=J Proteome Res;citation_author=D. L. Tabb;citation_author=C. G. Fernando;citation_author=M. C. Chambers;citation_title=MyriMatch: highly accurate tandem mass spectral peptide identification by multivariate hypergeometric analysis.;citation_pages=654-661;citation_volume=6;citation_year=2007;citation_issue=2;citation_pmid=17269722;citation_doi=10.1021/pr0604054" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=H. Tatebe;citation_author=S. Morigasaki;citation_author=S. Murayama;citation_author=C. T. Zeng;citation_author=K. Shiozaki;citation_title=Rab-family GTPase regulates TOR complex 2 signaling in fission yeast.;citation_pages=1975-1982;citation_volume=20;citation_year=2010;citation_issue=22;citation_pmid=21035342;citation_doi=10.1016/j.cub.2010.10.026" />
<meta name="citation_reference" content="citation_author=R. I. Tuxworth;citation_author=H. Chen;citation_author=V. Vivancos;citation_author=N. Carvajal;citation_author=X. Huang;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Uetz;citation_author=L. Giot;citation_author=G. Cagney;citation_author=T. Mansfield;citation_author=R. Judson;citation_title=A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae.;citation_pages=623-627;citation_volume=403;citation_year=2000;citation_issue=6770;citation_pmid=10688190;citation_doi=10.1038/35001009" />
<meta name="citation_reference" content="citation_journal_title=Science Signaling;citation_journal_abbrev=Sci Signal;citation_author=T. Urao;citation_author=K. Yamaguchi-Shinozaki;citation_author=K. Shinozaki;citation_title=Plant Histidine Kinases: An Emerging Picture of Two-Component Signal Transduction in Hormone and Environmental Responses;citation_pages=re18-re18;citation_volume=2001;citation_year=2001;citation_issue=109;citation_pmid=11717470;citation_doi=10.1126/stke.2001.109.re18" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Wang;citation_author=C. Li;citation_author=E. Wang;citation_author=X. Wang;citation_title=Uncovering the rules for protein-protein interactions from yeast genomic data;citation_volume=106;citation_year=2009;citation_pmid=19237561;citation_doi=10.1073/pnas.0806427106" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=T. Wang;citation_author=U. Lao;citation_author=B. A. Edgar;citation_title=TOR-mediated autophagy regulates cell death in Drosophila neurodegenerative disease;citation_volume=186;citation_year=2009;citation_pmid=19720874;citation_doi=10.1083/jcb.200904090" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=K. P. Wedaman;citation_author=A. Reinke;citation_author=S. Anderson;citation_author=J. Yates;citation_author=J. M. McCaffery;citation_title=Tor kinases are in distinct membrane-associated protein complexes in Saccharomyces cerevisiae;citation_volume=14;citation_year=2003;citation_pmid=12631735;citation_doi=10.1091/mbc.E02-09-0609" />
<meta name="citation_reference" content="citation_author=V. Wood;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. Yu;citation_author=P. Braun;citation_author=M. A. Yıldırım;citation_author=I. Lemmens;citation_author=K. Venkatesan;citation_title=High-quality binary protein interaction map of the yeast interactome network;citation_volume=322;citation_year=2008;citation_pmid=18719252;citation_doi=10.1126/science.1158684" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Yu;citation_author=M. Guo;citation_author=C. J. Needham;citation_author=Y. Huang;citation_author=L. Cai;citation_title=Simple sequence-based kernels do not predict protein-protein interactions;citation_volume=26;citation_year=2010;citation_pmid=20801913;citation_doi=10.1093/bioinformatics/btq483" />
<meta name="citation_reference" content="citation_journal_title=Journal of proteome research;citation_journal_abbrev=J Proteome Res;citation_author=B. Zhang;citation_author=M. C. Chambers;citation_author=D. L. Tabb;citation_title=Proteomic parsimony through bipartite graph analysis improves accuracy and transparency.;citation_pages=3549-3557;citation_volume=6;citation_year=2007;citation_issue=9;citation_pmid=17676885;citation_doi=10.1021/pr070230d" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Predicting the Fission Yeast Protein Interaction Network" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
<meta name="twitter:description" content="A systems-level understanding of biological processes and information flow requires the mapping of cellular component interactions, among which protein–protein interactions are particularly important. Fission yeast ( Schizosaccharomyces pombe ) is a valuable model organism for which no systematic protein-interaction data are available. We exploited gene and protein properties, global genome regulation datasets, and conservation of interactions between budding and fission yeast to predict fission yeast protein interactions in silico . We have extensively tested our method in three ways: first, by predicting with 70–80% accuracy a selected high-confidence test set; second, by recapitulating interactions between members of the well-characterized SAGA co-activator complex; and third, by verifying predicted interactions of the Cbf11 transcription factor using mass spectrometry of TAP-purified protein complexes. Given the importance of the pathway in cell physiology and human disease, we explore the predicted sub-networks centered on the Tor1/2 kinases. Moreover, we predict the histidine kinases Mak1/2/3 to be vital hubs in the fission yeast stress response network, and we suggest interactors of argonaute 1, the principal component of the siRNA-mediated gene silencing pathway, lost in budding yeast but preserved in S. pombe . Of the new high-quality interactions that were discovered after we started this work, 73% were found in our predictions. Even though any predicted interactome is imperfect, the protein network presented here can provide a valuable basis to explore biological processes and to guide wet-lab experiments in fission yeast and beyond. Our predicted protein interactions are freely available through PInt, an online resource on our website ([www.bahlerlab.info/PInt][1]).

 [1]: http://www.bahlerlab.info/PInt" />
<meta name="og-title" property="og:title" content="Predicting the Fission Yeast Protein Interaction Network" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/4/453" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A systems-level understanding of biological processes and information flow requires the mapping of cellular component interactions, among which protein–protein interactions are particularly important. Fission yeast ( Schizosaccharomyces pombe ) is a valuable model organism for which no systematic protein-interaction data are available. We exploited gene and protein properties, global genome regulation datasets, and conservation of interactions between budding and fission yeast to predict fission yeast protein interactions in silico . We have extensively tested our method in three ways: first, by predicting with 70–80% accuracy a selected high-confidence test set; second, by recapitulating interactions between members of the well-characterized SAGA co-activator complex; and third, by verifying predicted interactions of the Cbf11 transcription factor using mass spectrometry of TAP-purified protein complexes. Given the importance of the pathway in cell physiology and human disease, we explore the predicted sub-networks centered on the Tor1/2 kinases. Moreover, we predict the histidine kinases Mak1/2/3 to be vital hubs in the fission yeast stress response network, and we suggest interactors of argonaute 1, the principal component of the siRNA-mediated gene silencing pathway, lost in budding yeast but preserved in S. pombe . Of the new high-quality interactions that were discovered after we started this work, 73% were found in our predictions. Even though any predicted interactome is imperfect, the protein network presented here can provide a valuable basis to explore biological processes and to guide wet-lab experiments in fission yeast and beyond. Our predicted protein interactions are freely available through PInt, an online resource on our website ([www.bahlerlab.info/PInt][1]).

 [1]: http://www.bahlerlab.info/PInt" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/4.cover-source.jpg" />
    <title>Predicting the Fission Yeast Protein Interaction Network | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_lyA9yD2yTpZVRkt6WYxO3BnxkY3dVrfdzc5O4fv5eY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459267","apath":"\/ggg\/2\/4\/453.atom","pisa":"ggg;2\/4\/453","ac":{"\/ggg\/2\/4\/453.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/4\/453.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/4\/453"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/4\/453"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/4\/453.full":true,"\/content\/2\/4\/453":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZOz9xA0DGVzppuU+B9mb\\\\\\\/z1vROOmyPy7G1We+3csvTI2DHG2QU65kudfOIR9qSnuk1Sa5XS2iDwcaI42NxrY5W6RSrxjMHt9C93nFuHjMEwIL7xk\\\\\\\/DkriB21j9BO3+IJgMHjTAr7thICluYYCHrkELYRpdLBmPzG1VqDhRJsKfym8lN9TxA0QrBX5ni9KgBXNyHJ7oBpkoEsfLMrmP1sqb2zRowCZfbiicvvStFoLx1qMjvQgRXNeiTvYS3dCEHdlN95JBoqIct+vzqT61a+uArzEXwlmrHFyhnauE3blTCSXSMd5xmR30fYhfGINk8O9PLUH9iN6\\\\\\\/FZ3HDNXo6Fb0q7t2gry69I3IkpNJJqjrOUwokvshx+vqrKeGrpllKV\\\\\\\/jB4kUHVnrRkMiHAd3QqlIrk87Wdvgfj2RvBQDTJDDE6cXwRe5IHJQlukSuhLlfehZ7sJ\\\\\\\/M3m1GhSQzLgMBwjNdAWM+vIvvNyRxhRVUJUz0hDJG+X23tW0oPv7Lg+kmdu\\\\\\\/I\\\\\\\/UebJAE00VvZiFQbrPfFv42Z6w3MdBXzDGZZ2IIQGB+JY6\\\\\\\/ufeScrz5ZMX0TxzJDFbSKYdRG1BuHE\\\\\\\/TV4yA7rMk\\\\\\\/swBi++MdW0TfP89dhEHY+GahCyAtHLOj1fRpETPmyCwJvUv1JWMV\\\\\\\/l\\\\\\\/afe0n6JjAF57mRFiQDF3isY28MjN7LNcb+nm12vn9+5vkkp5AEoOZjfVlJx\\\\\\\/bemsLzYU5v0\\\\\\\/kTAj9WHvIdvram4QQzCtHhUyqUfbIlmS4flG4VQkFpRl\\\\\\\/1hjjdYzCPcuYUktb3MzIvTtpwzgKUQfQtNlFtpSAxm2vt9e3eQwJQ1ChF5d2w6+2V6cz3a\\\\\\\/A4JdfYEBQ3XMNYOt+g0xjsDFswpmumNPNQIxm8CVGMd1jTGCyJAhNBR4LILbEe0rrA0c\\\\\\\/JY2UmeVCvBHuZE6uRSu1rM3\\\\\\\/Il8asWf0ExFxnrMXurKudO1n6cqvtIkmblDdDi2d3EJrHa\\\\\\\/ZeyT93ObHK5ouOqQvdCMRI20vp1AEzEnej1uv\\\\\\\/CLq7nWRhLMicsWAOjUE26ELnzQq9FZSXdRIWV\\\\\\\/\\\\\\\/mit02Bxbe7n\\\\\\\/tbkZnic6RloGf\\\\\\\/4kk0EznxyCwDVJjMYTQN6+KBPo+xpZcM7qmvzqpelXJY+y90j9pCZ+j8\\\\\\\/ZpxBP3bWVwhOlKl4pnRVstnLwmqSmxaYzPM3wPJ9S+njtxOFx12xEVKaT\\\\\\\/ZGCfBqYx\\\\\\\/QzXUdqeWWVbKYIXW7TKaQKphgM9zLd4grI6cP2B0JsV3KbdhYpCdgUcJzIxw\\\\\\\/DsVDT5\\\u0022,\\\u0022iv\\\u0022:\\\u0022AmChKtNv40Fhlg6KMEhsmw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226418c24f750c7c934daa8f51deb12c2975d62cac8781a1986b82760608b19366\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P3LUfYVmUo1ud9VugC2Y29xF3r4ORljrckh00yVO2p6vW6YHBU0cIDvyQ03c+0wknYnA3nW64wlhnardgvQeofuyKcSqHvFk6ncNclr\\\\\\\/zs7TYcC5FWq+pK7WTyGtrBPMfh7u1me1PN\\\\\\\/h\\\\\\\/\\\\\\\/Z2ZHZHdOQa0cUbeV3lS3\\\\\\\/ibVBVsOKuDZJO2kYvG\\\\\\\/HSmWkDrt5yrUOXAPIz269cjcOW0xkx3DQLBZCU9n3O4pDLttmaVkZyDwzKclcP3MsYWdSVZNIub\\\\\\\/zNfqpzqUijMO66kKuDJz2r8bTRb\\\\\\\/SBeq8eDeEF092J9mqwyA2p9f9Oh34Qy2uhOXPGTG8CRL2jKceB2R8nYJAxSTRLIwJ8UrGuFtPuz2wRY8h9mr1IVgaQduL\\\\\\\/aKzYXPBmWsqrmF04ez+HVqX4ERO094dOuwsSxtvQFOkV5Lvq\\\\\\\/Sxm5bycT+ZWPbcsWhD1hdHzA1fAe9ynyKu\\\\\\\/BsfL8+YWf55JYTmacBaVNVtq8XMz5MIoX0ZzPq11Edf2vcVqNvjcMGOxLhzf1zhrWTOXBUzQATpmtfknAfoog\\\\\\\/3rgbRN1LpI+nUbbuF4tqdZuJXmtmT70WP6EkZd2yMf9htOZdvMUEcTw5VnZ31xoRs\\\\\\\/ORghpptPT2M1b3r9fePmCC+VYEJsPOKXSpM9vBez+BNwVUB8+6ScUmEu1rRdkmCc6RM0G\\\\\\\/IpZTY3QVmCCTHxMhRjssD2LHti4MwQ+Hac58TNZJZBVshiQLfbWz6TsEMhFizYdEt\\\\\\\/S5hQGDYQ777twUR612JtuBvj1ENW7xJGQoLG4qiLPZk5nib1JqVkzWhAicYDsoBIHU30bvnlOI7mbawhkK3wV62yhgBI0Wr9MZBWZk75Nt284ycNSSLB0GeKqzTJ6kYFzmywEPDSoFwmvAkRIoktXWTeWYxS2xBrrfWmUOTDW7FxFFhVlnLZV8TADEywgUyHrEpGot+2Ef54mWaTxK7pDnOogAIoDYgSYxjyvNh2KXfR+OxXaL\\\\\\\/2TN7NG1p01TAzu\\\\\\\/Vw00s3fIzQLfTHODUKiUGaM9SfUdgnVFDFbX84HbQY6YXCTuNPK56A0zRpPy2dj+Uvf715GFeC7GxC\\\\\\\/AwEkjDqpxzSSaLJytVbZoicvP\\\\\\\/2yqtsgEyRFha2\\\\\\\/H6TwAN\\\\\\\/m0\\\\\\\/epSKsWFc\\\\\\\/Pis7paO3ZRagtVvan\\\\\\\/trJnrWlG1V9H21un8YMI1t1zNbuoXmQwPubcgMZdWUJjsi6o6D79NBXezZfZu924LlmP2FiG91RJC24YR5q09eMPgTfdnviQ4LAraFFBoCYSxFZOGkYieiuZ29BJaw8LxEVvDd+cVQfV5Ul+v0Dv\\\\\\\/iZcUlrsOfOgxJhRynaKLppLUp3+6\\\\\\\/UlOwlcYFzWb6tWPaYtcv3nBna007ht6\\\\\\\/81gRabt0n\\\\\\\/Boz8KOgTVNoJLIzfRTdewsExVjwLS8tZLZmeePLWKm91F1LMgg1Tv9gz8bHTlAYfCcRT8k9OGpoUNK5E\\\\\\\/hlXvuTx7TC5\\\\\\\/tjz6Uequfc\\\\\\\/JRwr0xDpxN6BQvgmAJ4Wh6\\\\\\\/e4tmwkHCeZ\\\\\\\/MbjKeMKwLK1fGvlWpYuYzYfbDZ0rA6TFtcWT0ohacYh\\\\\\\/HKMODkvGdqu1uLhtypnQfsWuQZdjmxZfybT0IjhLHZzb3TXTnrj5QIsQocloBwCgQveN1W4wq8NczhR1iiAlnuo03N6FX5\\\\\\\/6suRXewlWV7ZlnXnwm6kb1H4Q3jdKa+MvsS+bI8wy9s\\\\\\\/f6MdkbdI76vBS6BkeSQ5gWW13Fk+FAXo3584Su2NTwCX8HglWJdJZf\\\\\\\/ZNjmMAfMn2k0R4PDxuvXc75qwnQiveqBVZBBA9ChdVw\\\\\\\/DuFlb1gJI9nKTdymtCN6JolEhHrc2IoNV8VjlA6A+DOpQu+UsI\\\\\\\/NCSKgBUfGZO7lhYBKvL8AFRwKbhfCGn32AjQ3wY2MM3a3KvF1JHj3Nl9ms8iwmdERU3lmWmHxk04RTz8OIuiHrT7\\\\\\\/Qtjfd6VPTqpZ9I5WkHFHIMtb64cOYfJEZEjkzbqF5BKT6m8RtZSp2qdoYJXY6PywB9dmmdemCnPoEy25yQ1Q\\\\\\\/6KvgGmRDAQvb1vYDKI9WRyL7fq8mFLZP18rzWITzwJV2T1z6jipjT6Fl4cIwQkwGbpzcNrRfNDKJMS+3pNQBHxHtaPXg9eeSKANDVYInIty+KP\\\\\\\/2bMQqrM4O0rOTtbjwdm+LrxAyN82o5Kqa1oR2\\\\\\\/mvhF5SgwyMBmr+fQYjimqJdZe3o+BEBhXAmnmS9p5WfvLiZ+hxFf7qzBTGOBSDaTeulv6hPCMUUCd9GNbWjVyLu9kzsVM2inxeRRFNw7wW\\\\\\\/7yqXLSDaUqvLD8SBiUVoO9NoMm6WwIP0GdArnkq2g9A3hlOaxKhPHQVM5ZG\\\\\\\/yS\\\\\\\/\\\\\\\/ZDPJ+yF7s7jmGX5N4SpJCFUVWep1jT3PcqbyIXS0mwBKBuLQJwO3PLihK+skCYyE03EBcORDxtvO9OnoDZcS5ZPnvTbbcfT0AcqATFK9VuW175UZON5q33E9QM+AJgyokU0oxZEn41Odkum6LBjUMhA9WLFQxHXkb+OEjm4RPU74axaKDhETncswfyysvUVg\\\\\\\/QPaneI3K0rt072mS5OT0o0sUVkftlMQ6593EMIh4mJo7vMvgdZTrE7cxNyAmWTFrcAcy7\\\\\\\/t\\\\\\\/g1vMJGPat7kPuQq0x9Hh0tGuoJqH1X4yU1ORZJgl88xshHvAqTZNg0dzdElxtWdFYnL9ymexVQoFKfUvEnYkBPP40PqPdFb93+serDiyj8HroWzCLMZW25tx5IjadC8tAXKeILO2VrdZPLI4YcSIYD4\\\\\\\/gy2eL7gZtKq5NzFttVvt5rxyaacEwTnPj\\\\\\\/SncGdC8AsS9k\\\\\\\/jFqB+jG4KkUX1bu3k7e7SWj9EukSVKZCfnAdDMh9j8hihCbcbjkay\\\\\\\/BuE6RcCMiw\\\\\\\/vyxRB6tZiLHIAYAIqLPXBcPQkojEWNIV5hUV6BpCXutdpzOFLjtWc+WKo+tpx06oXAvhJmBjOrP49BvoHz5Do02XX3OS8GDRa1qIXqywsVGdU0ndXlQPoiClzxnU+rcVPFyLPvnlZieSi6VF3DLWY1P91FrRv1eppWEnlV1JzU1duHkcD5\\\\\\\/glUy4Sg8u9vmUWoGSmj\\\\\\\/uhrdWxM2hPw9Fq\\\\\\\/7OytwPKPxObMaZiDILbA1skngsU86asWTFZ\\\\\\\/8oC8iTzHE0rTRFOfnHHVVpXQd9YTtKvsGULBByG5vd30VlAN1fONYM67tmgHTvL2fFqmMZKcFI+2S5VC6i5oJKnpg22MGEHEkB30IbMtbAccZrSmXQI3ue\\\\\\\/U5pQX+RhzZgfmcn\\\\\\\/1plZmLbpcWsIVzev2YQdI1miN0uFlPSfDGUGcmA2vCTJ4RLOjj4Fo6QD0lz5sUNjKG2raWA2xb\\\\\\\/uhMXuihOzgSs7D9KmrIckMMbQCP98150xVALvM8nn76\\\\\\\/p14N1yFJIDLwJ6my+PQCygCicAQshb8XaBknjYanG8\\\\\\\/live6xI6xU6\\\\\\\/dOb\\\\\\\/8jYindjRK4hxe8WZBe9IWinQgq7B3hJq1nhFSP\\\\\\\/4WB\\\\\\\/uv74nyLGQjPNmelPCN8G\\\\\\\/Ph\\\\\\\/B04h7bxyni8SGN8Rl+3IUypVtZ\\\\\\\/bddr1kvdx7INLYw2G36VQgJWgfCbfzq\\\\\\\/GiVlHXwKoE4CuJpsxLIBRCZq0ZVbMyVLzwhoKPL82eCbLYh8982p8dDaV9gxHNi+gKEcaKyGIUqHL6C5\\\\\\\/Ys44pqQooc0HI5TTiKGRs6gb3Bj8NmQQoc+SBzoJ8d+kdxbkWH\\\\\\\/bzyJYKiZ6KuwLTspCYHCxNgCVKXDXgx\\\\\\\/bU0JavZ3rHd0U3ymiyaIoDgZNFZ9Z+WLscKofE7LtYYY5rP7aRTXNjZJNBrmHSQwKax\\\\\\\/SngwLQu9TaiIw2eifwE8UuWE77TwsRQU4MwmfReA+LLXwA6+MfFdaMDbFxC77+JedIZ31vIf6dE8gPAktCxvqoTEeqiW7aTKXsPRxSi6bMcsbbkZpOQYAWusjYUOmurb+DgUSF2nQJv\\\\\\\/RqQ1lOIYxHy9pbtRo5NpqNWFniwy548ylJi6RVXSXvm5rU2kA8Ne6GMU2N3iWmtFmsySQsMwTvRJ1rL7jrR5kEyRQFHuVdqfsozQGk9dRAmGopAF5yAI9OxuSsHPWAcFs\\\\\\\/HbL3HdlJ0vWQGiI9nmTje4vs4eQ9MrvZHy8lSIIMiEZA5O672xUaFu9szGXM1fVElzwmRjSJtG0SS8ji+CDHEs7zhm\\\\\\\/7+A5mkp2NqTKwgHFDGIwIMNBt3sRRRn0S39Cu87g8319zefe0kwUAwsNskV7i4jicd0MtTuarOPRqdGyUEPgMP5KyqH8bTrvl8eftT1wRXCu2VqLyEX1TnbVhvTRg1IltVahllF7ae\\\\\\\/wIlTUsp1q98NZyJvmHkUpDHLNOE58q\\\\\\\/BOXu5THAFN0oucxKnNOdlPsws0ziCy\\\\\\\/Jm18FKuuBagdqt075xHWZt4GCunDWgsYuto5956v1uc4y2EdmEl0l5Sv+0Pn5SBpD0B3sWZPstRQi92e59w4jN90pqS5G5kVB2HMT5Zf8HCj7yhFaFVff\\\\\\\/oX0t8aP7UeefQnlP5emp5n32lGx1CmJ5nbHd9xsMW\\\\\\\/4Zgc1o8z9uKS7lZpS02KNSRjJZ0JVFMl2vkLZaKsib+ZrHZ1EkGJPiKWZHfLJSzBn1HFGMHcGbUUFV6OmKG5tzIWU3ebyT6Ub2HP1xlHv7IbHRa2oVTbMJe+l8wM2e4tM4ZKuF\\\\\\\/PXD1zGXXzS2KqmS\\\\\\\/aFsWJlpKYYyX\\\\\\\/bzq2G8Kh0UGN6zvJ+iTLSqctPZYLButPAklHCG7i2iibt0yGojmjxc7e061Uuv0Z+bURSgc1ZmXRf9nydeZOd4AKjZLACaFyrjMzGasJacoNE7+u8OZy0+xoW\\\\\\\/RiMuOGc+o7AGp5cUsEDrXCmypG6OjwCPHWIPf35+o1TMfUEmVO1\\\\\\\/YPaRNTgaJdFQsgvsm02HpkddybxpS\\\\\\\/IN+RA6\\\\\\\/e1b3FTht3+ZE0dGaYZj8tp46dTMKL6+iCt2UWLj21x\\\\\\\/Oc8gWJ22gtngACOf71kFW1LcpFvMdtx4p7yP2j+V35gZlRy4+N6kUKcxZ\\\\\\\/sDvSLLZIcb2+v89J5x9M5E6CF9iE8v37NU8YHZE5QqqZcFJ8IbvmB6uqVNWEi3Lfs\\\\\\\/IAw1DuFMspxFRdbyW9i5eWVpcdlJan5kxl9VOjhmyd5tBpOWD7ONHBDdwsiVSJeP62UuZOtNC78IHx3ofwbxep\\\\\\\/dmcPh8bvSHBQaSWpkAN+Xon80WuzPEk1By+2xTQbM8iR0v71hqHOrQC1YCRZhS4YlwgvkEXOiAe9iZY89VzSXGkAQPSmDKPs61KWJYk4E45WjH44Qp99L7JkkD3zKbHtldKZwL8frVqfuBlIqz2oViSTip83yd+9NmcwkxQ4b5bQD4EzmNNsIigd5j1mMVlQVTdcR0MOjNOLGaTnq3CokEyhUKLjJdqlf8JRdjcEkLa7UsleVTFuSxFjzFg+jp2rRXUex+Q162QzSsmhlRIP3UL2I8fk+OJHEBLNVIO1Tleba2Z\\\\\\\/c0\\\\\\\/59RSaS6K0AVMs8GbEIIMEXauDgrpNOmcMJVygbb2+5AeW7dfoV81An4\\\\\\\/4id8w1VXffNZT\\\\\\\/ZVnDVOnHQ\\\\\\\/18TUdKfhVpBER1BrlGhbY6nppjrkMJ6AwnXfMnUIOaeQ8ZllZV72BUich+auAqTThihfjxo48CVgFQ+4OYSA+MMxE4wWstJEefyto\\\\\\\/OjSL4cbPA5GogKpItiHwGW2xpvHJzE7AHjkeS+PtAi1wG5KMtK0dIeTseTvID+Qkz+upjzUJWiuoaX\\\\\\\/8L\\\\\\\/smSvYzqqJ3H4kff97NVtUmflMvP\\\\\\\/kNlr8mviX4u7YuKdeQiy2o9Ct9Cv06idLJ3CdV+jVcqq\\\\\\\/LVVUH3lN75dOMMhZ7E8RhywdF+K6ievoti8pXAsd+N3KMMD9QrPhzwuBvuG+gnhklgdlCqu\\\\\\\/sji+zFQjtMQYzcuidxdbAxrg4Q\\\\\\\/bH9SjsWvv\\\\\\\/+qxr4NO\\\\\\\/58JMiZqwHjb798fR3X8vEaXQDcU3gz2ZfUFgCsOQ2iy9hfrf0GrlND1JI0v4xPRMbTFQSUmDX+WzeIuj96cXh8tIvxwx0gk1mQFfXUsORz3Fr7MEsRy1kybsphyI+BKjJKuGADcYTZpSyvI4Q+g2VkrWmhddtZBhaXNNE69D+uskiA69Mh7i4H+GqYu8eaNt5BQHJBfZZRZH8Ou\\\\\\\/olPWOzitBMHfAaKvrPVCQBkZo3OH5gTsh7+dQx3iC3w5nU+SRzAMp8K8HXdB2TlBq0HZq7F38RXe7V6wymasQSs3pqLgQj2Q2rAU3urYFzXqr7abuEfh+A53esmfQeeKnC9CzOQq+2ZOQJkpVHRuNI43jiUxxpvJrhWxEORA40u\\\\\\\/sk8KNefqqvAR+w7plhVXb1Ae88p8wb8o7dq3uUnfrBG6VAjtWMLpg2XyFPqldtiPIq+5kLtr1d+4H5\\\\\\\/3oeuAqhP9lGkYs6o75jjerg38PxTumOgjbakeLmWFja7j3Y48VJEAyYtTtMymQnKDLjPvNw02lnaGjtkrPu44C6\\\\\\\/iBWcvZ7qU4n3U\\\\\\\/QFH\\\\\\\/WFzCgn5BEYhCyYd2DwEdo4SJsXK6dqzevQZJ29WBC6dWBXhB244vhhEN7UKvaSkRCDUkJJLaIEq9AmbWjdar7OKIy62iavPaB+HrLCoCQARSwQUZuDu8rZCP8VLFhwcqq70ic+Bxus4c89VkPoW3IfS4bvWEN1MoCuVPwjNe8HnFesXRl22Yrzx7USlAnh2hSSDL\\\\\\\/poZrINrcBZKeGRBopktG\\\\\\\/OrWJOKrWC+ad+SSWsLUq4OJ79RBTfOnhu8iuMqWbwDPl8txftbYdMGnNdzu3rufc9BJEfP7CkT3G\\\\\\\/H4vgQ28fcQdaLmTd4hH\\\\\\\/Gtw+bDaCNwoNtGCUdv\\\\\\\/E\\\\\\\/E+d98JUHfUdePN6tOVInKxYA3X7\\\\\\\/nM5r8oEhq2s0KoiyPRXpegSOepL5mxctPhAVwBF+WxXFNCJkyI3\\\\\\\/e30sUafJVYziiPrAbFcY2TVzyFOq4HXI2H+KlxkM1CTzaA99bB4eiOQt9Y4iC8pSZF0JiDENQMNQnqeKdoDMeH\\\\\\\/Emb4dcps\\\\\\\/qmkxjJ1KODMVOS5\\\\\\\/4Q4BXSYYlm+qoktXYEO3uoIf5+NFm6kPscPeiQ8vtu47OEMJFcpEwAEqUFr9x6JOHV0GM7aMJe7B\\\\\\\/Mi9xD136rvdpPXwunBkF0kk8dDpHT9BRclpaQal3M\\\\\\\/7QJNu+Fn0LgDISSSYEV2zYmivnUFgwTulMwonu73mJtrudtEDkPhhtknfRuy3KsYx8ssbqr9VevzYBoiq2DDOh0K78+D6AKSXBOMO6jXhPTFo0UsbLhPJAGp2uDLh\\\\\\\/MqEJN5qcd+HAj1CCAOn1e1JO68XT1PEk97N2NM4inX3Mf4sXIYzeYewA00ph+xqeyVjDFNpCcO0dD39kY2vCMVAXzORBCLz3XehixR4soj62Apbls1uIOBxhxqCpWFGB4kuWfcnl\\\\\\\/7LsRlR+uYDw4AUJCj7YR\\\\\\\/C88unWP95HTjrMLe5IaQnWeZ8NzNrplYXR0o3X5+B\\\\\\\/qyvOyTqto8nrz5No\\\\\\\/3K4zugJ7kSj+c4H88RybcwwfFlN3PxA5OVigdZVs2y70uou\\\\\\\/dzO5krl16wRAinur+6deEpUjXB25Zy\\\\\\\/6xPR4CH4AI0+Bc6\\\\\\\/R4bnU4s\\\\\\\/0RWlbveNneKt9j2W4vf018t1QoB+j6AxHT85sCEdnbje0LJIq\\\\\\\/X2Yym+mNuEjJtd+zsU0tbjmb9caaOVOQyxszs\\\\\\\/nQSRR+EXkXWihswkU07FX3cXH3004Vt4mgmy0qpejqAD64\\\\\\\/5bcYzUu1ib9rG1l1GK+7fc7OL5tUtuCTRXX0JjABOCVpwkPPRu2NnvQPYCfzo3bpBsNi3t0hgPwCOHA0r2xsl+SsS6nOC+RCnUWTDZ9KfZTVifAwxqO9PTQP1lkfBBZwgYXKJLwOJnM70OCfuHaNEUfnipyCo3EzFHdo1fXZBUpsL\\\\\\\/\\\\\\\/G9VU8spkRbOoSSOHsGJZj0gVMBactmQ3eVQiwPfkyJL8IkO\\\\\\\/WhpdUtQW3didxg\\\\\\\/Q\\\\\\\/ThwNfBA7CVmsU8qOcgETAdTWqywzNpeeazo8CI8TrdeA7JK+W2iLfKdYuDq5kj0DpIP1ddacq6GmT6gmYeV7lwELenax0okxA8JOi\\\\\\\/Qbqie0+OwpnyRZ2Pk2hOLhBfCxR+pgaahlYg+7hHlMGOWqmYpgUMQLn0aG4+FflU8z+dE2THb80QkNu69BKX5BWsY9E+b\\\\\\\/iJXza8FECLPYMDq4RvXQ7Zgo9vzP9goy8oGq3\\\\\\\/FstXy\\\\\\\/rUaIb8yytLqeNXQdrLzbnxBk++EJgNrLHcHjCfd9EbkujldpFK+CAGb1uLpzC6xZfy4fNpv1PMeGF9txIzxhURO4kEFd3PCLwnW5xcGMJo3W5Tz2TAUNZKRdSgB0blJGxkfUmd9BYk6gEncM5nSIGZXMywI2zuEqVvINMZ1EHeT2+WF3gh72sL979BeArhecqMr5zh2HB7p\\\\\\\/ZL1bhWcRAYlGLx6LbLxIKTLRY2jBzNmEhjMXHcwL5phVSZDI9+XrN7N9zD9WmTVVnZ2n4vjnelkLlKNVXiQxtxN0\\\\\\\/B6PoZZNXXPkoH9ncO7f+phJjTPrcZJFNjH+3RxSkm3eQSwFnjQsLYjMNWoiDbEw6HhtR9I5yQPuZF12VSl72IYuFsWkTuEjKolwQcC1LiOa9FFCB98P29+FPOdGc0CKMQPxvDyb0COU5xdyScfG6f4szY7NUTRp4bY83BeryfG6f23QR7DlTB7xZ7ePCEb+ehqCJmio9SUK4m3ETK6CLwd+Ne8EFYggJXGRkfOzlM70ryU67TmfU7m0mOh8dKE7KsylQwa3h0X5j15f8OfHUdN8qq6ASyq0KPQX29KmZ6gIdXINgLaFjWDE6oJY+KgWhDOayI3wMQyESvNujbD7ByOlRli5zUQejgiUTLaSyvBozHcgpHI7cgIZIkhg+7\\\\\\\/i1Fhw8cj3BPWI6LfYseC\\\\\\\/htD0DhbfnOwlkbgqNUN330q+VjVE9az5Z1\\\\\\\/ohRTE+8zGZmVclXEauUDgmSM8K\\\\\\\/nMmrKixKN8ZUO8eMOeFFbWj93FNKM4gy+yUih\\\\\\\/nuplN4QErUZExl7UQX8q37an3ziw4wsTg\\\\\\\/PKcd8+Jr5uH6QTEacrTrVM8KRn31tjl7txIaVkLSZU5DyV8lyIl459Zsw750Y0USaKsqJOq4CoEGAGEhEDYR0QiJaU2W9j6kbCO7g7SFU13OzRUv1+Dxlr0zCaYfZVpD1wVFME6MSbGfBbkkyil6HJhxbi7Amu3GEQ41HZY7dhidKvJ5IykhOUcD7WsD\\\\\\\/i5OfKJrhCCgG7VvWxMxkAtESaEg2nPfLlIPMBWSHjje6oGzylb2n4cnSGOHBffqJV\\\\\\\/rPQ+SJeUqMyyzDPdTRXws3FCWFcAfftBw0+2m394pSeVkRKucslQTleR4LwG1GTx1Uzj9\\\\\\\/38\\\\\\\/j56f4nOKRMo4rp2R4Jklm7KEU76j7kRbQ42x65hHEklZeMaa0qMxGuyIuQ+z4Z853IFPjKjzelDogucTVZmeCYWLAZo1UZedR3iLBHiQJmRoyve5uyGxApL3aNsnt471Kqpm06eRGnjzmFtfzzn8Nn5JE4oieCbv+1kxVbRkckL+kZ6BBg3Io2EDuatrDLXIw3WfbDy0uKzHky5m9VufqXS4HOr++CjDkOa4G5ZNXFq2ybLkq\\\\\\\/CyO9\\\\\\\/i15OBdtu6y3e7x9KbWpZGxYTb7pxNfcauNJYr7\\\\\\\/tNjMW93qkXzw76TYerGIIu2sNZ4+RFD7\\\\\\\/BJen\\\\\\\/pxzcq\\\\\\\/y774anXlPDCuFyngnXnIp\\\\\\\/xDhAQvdasFEDTwbpAOziS6wmVtjYhYYQCqLLvmXp1zzyDW\\\\\\\/t3ciTXV6FEhuqpeLGxG2ksAX9smOhf7gXUpomABnucXPyXJIoKaRFqvecF73K3CoNl4wn60i8D3GAOFDlXFX7aRs8zWkL2CqMacU9Yyar6aiNNo49Ps80ZAf\\\\\\\/KlhMlqFcsYN5nu4G5AFKEdncydEidpyGFDRysWTJjixt3IiqQ5y8+7WFIl8iNDeDjLfZsLghsi3EUbDO\\\\\\\/7NuEDBqy9Tn5UCEZGVxDtOD\\\\\\\/zGAdoGFdlRBY1Asjzrxnx9VZwvBaxgsXFzmH9BveGCBFhSMbhHNsggEbMf7oY7iDiN72EOtieXY6qbsVO4owY\\\\\\\/VDIQvolYPXe97XhmBCWhOyfmJVoMcJ1lNXiuJy7vm11z9fXm0B6s1d+SfcY9RiN7aDsOqWvWiJJMG9YCpt1se3EwgG7GLsMjfd2rdEU3BGynXrcGzNNWGRsaY\\\\\\\/GlHVRbzV3v90K0G74k4MqIL23PC991I6uLVFu\\\\\\\/bQpFzAABSFHsfndOKrtGJRnFZZbVEEqEvpQfErCeMcOgEDlEx8Cy712jjpbai5zOTDmK0r4kB2ZQ8Z4IC4EHL7TTKRNkb1mPG4CqW4f33Kym7V+RtLZkDAdo4Tl6nt0iN8DfPZrXK55fQOu1oPRiCpgRDZISOamNQ8GXPWkpywEV\\\\\\\/FEft59\\\\\\\/LlUCkvTVnwLnfS83XgM+adHvBQa9vMP8i0e1VzyUJOWjUA1frS+7wAOTFLyNqHPJlkujcFRVnl\\\\\\\/pPyf10IByKqOsdPXObMLsBT+JtnZWy2\\\\\\\/4RMkOrRUVc76dYmXruhzvPpv+M7ovUmDkJc3daf9rx6\\\\\\\/PN2txtkj4xXasGTd79uibt2vzNwg569Hktkurc\\\\\\\/3glPwK9Ac3\\\\\\\/rZLz37hrGGalL\\\\\\\/WBL8WRsxw\\\\\\\/20TGQqjaztCJ2OiC8dsPioILbf5lN3DREeqZgh89dtmTnSFfZvL35nZHrOPZvGlgdw+8ccM6mOJ78nFMXqdKoQLosHahYSw\\\\\\\/WVEDOyWxDLzpkFrrLNKXh3i1bvLO5bY7finQ4zaWD9Za35VSwF55cg2L7b0HfCnMiySZMVfcoU8Su4rJurC9rq7wlo1vX9IBO2eRzz16DcwGRYw\\\\\\\/RjAuKJXj82igczm9SuNRfCtAsBtNkTER0eCESjhEVVQA5IixWs4SSNTMVBjHnesRjY97JhCm5J19D+0torGjwcmWM8CKkrShzcAnJQz\\\\\\\/FA8sF0N0tir8\\\\\\\/j6Xy9mDkRIBRSwwid5ud2SK9XNWoe9ehm8KzXy\\\\\\\/Gefvjfdf517GnGpzkJoBEGv129MEoTy2wpDQjwSrL3u3+k+B2\\\\\\\/eqkVVdHQLH\\\\\\\/5Wcfj1022Tex8BwP\\\\\\\/jIk98AYvMXxxvJNNYu5h1aJUWS8vL+kB9tDCmRGsOvKIbqCQ1MHuHKnxHMkAYEaXbVaenWvYrfclf3BRQDQ86f825CHi9x\\\\\\\/G1J0tM7xPuyAH12D4xVqADkXyzOWlelEM9Ue475JF86ysU8o1ATnDUvm63Kvl\\\\\\\/OjPpkWcCL4bRxA1WF\\\\\\\/VIT9C3ap3a+BA\\\\\\\/7pIjOOWvNq3lECx4sEpYbHTNrUJD5wkqzWAuXsuH8+swQOMpPh6RNgb2+chrDrcc2uYfdICKlGTCJEoOJ0wHhCwlD9U5rqBRAQ26JUQbTU\\\\\\\/qtX31fbh+8YYFBNUfLmv1AwY7o3ogzgTTBOWMyCF7aLfXYeWW5SRb9t5sBNMBbDANKylJ+XK1ssLX5Aq52QEd6EVUmMRKujSwL2PsrDWCCkZCZ8iADEzkdVG0dDW+ZSLtNhcmavFSbM7COrHK9ZsfPdBed2eMHkQagcIUYw38SAIOY2cYWF4Bx4xn33iWAL1A4XdksYrEE\\\\\\\/U11eJAwU4FC6ZJeOOLAfKD2ojsoR0nsDFipVgMxsigLckYXfb2WBrbqKeqYdq3k+0N8\\\\\\\/eIpn+W3IXgdxg\\\\\\\/9qSf1ftJ918nyhgh6vyt9DeskILNay8YHRMRiwRkJrS84YJD+1hqL0gFTf00nXf9traeJEGa0giixLyyt\\\\\\\/+lKo5Sli4ufu10g3EXTVpBiE\\\\\\\/wpLeKqep1dc4sM8AkRyiFwhlJV5x5e1QbiQMB\\\\\\\/zmwwhulnETRnUPzCYtN8lSYCQx2xj2kDHgvPB5z7\\\\\\\/Taj32GqpZvvrCNB9WVSCNA+0NekXPc1eJQDLR2FIYe\\\\\\\/gTMB8Y\\\\\\\/Hie+ETNLxcJDib3CHPuY5CMizglgUKXwTvMtGhWX1sG5p4982EsVgtDWhSN3EMOpdjqwNSFe9igEse5cIpI\\\\\\\/Z5bD7zaz+U6w9S9Myu\\\\\\\/+1fgeOLUWYn49s\\\\\\\/XCjW26GHU4yenoAznNDYPT+Z9rd2M+DLr9iMnxgXBtzcB6voxrmKGPWyKSP5GSCh5ZoTBOW3fPk6rzbaqz4PuL\\\\\\\/7CvLxPo1vAlpxFpDf5bkNbBN3QfMGee2RtTDrWVcsd6H7Apmqt\\\\\\\/TXYxmvKTHAdnxCtxbuMInipIgsuqzMvKdnR37lebPpu\\\\\\\/Qrbh3JKJsCv2kpq5t6G4ZHbEzP6OcAy8hPyW96chdK4OotyFmeMYLHDkAcLEd+I2VlA+JGxiw5fMz7xD7OofRUwC9O6kbgbLQp8m3hfGr4qeSwRfjqy7TL\\\\\\\/9faqhh9GWoTBqFbJKj0kxioXnP6qqJGAST\\\\\\\/2nb6iWk6hqkYG5OOjN3J4bAvnjOpM1uqeS0Ph9S++kU6FTVrofo+P+6qJYBnVz2VLf39QYJ25RGRdtvzV\\\\\\\/b3Cei3pX7aDks3h0tp3vW0gUa8wDGTtqZ603WbqjrYfEXlPthuxrvApldh2KSDZfbH61p6iew1h69D+a3f26uDfGmjQ61YQELcjhZCcG\\\\\\\/4HksdmwzZTE5aDtDat4LN1InViG0otTPKTxfm8XTumC9JZsySUvED3brjdcdtMwYrtiFcHet3+eLdSf7Lu0VR\\\\\\\/cNrthysZ0ms6stWQ4Ne2ixuuMFVQsrmSsmJV1co3xSd\\\\\\\/3OJgo\\\\\\\/hEz5UXkD31D4EJf9aOIHmxEX9CmzvFRM6lptA9UomRt9uYQuNRYOq3AXpgMv8d9qV7JL6E1nTRc2H6DL736oof05UhNUM10wZ4qh6Qrgj6ed3L0WX59llPyIORsA\\\\\\\/+AWK\\\\\\\/upkDBDbSCag83vHdAX42bjy7bJUPG1yOpvqro2GwgGxdzRzG3i\\\\\\\/XW9vRbEkWCyimrUwRn435HwXylekW4Yln6erD\\\\\\\/N\\\\\\\/ez6On6M+vsg3YhBxw\\\\\\\/u+K8jPYNDqBFIdOssacAxuOMvoEvlwiWfIm8\\\\\\\/E5rQaKxV54X6IIr8MjLeBzZp8Dp1fLGjsbYToeJrwB4DT9Es1Gcvp3dD+Ckd15gPp9wBx9beWxpB90UqL5066pABboUDqLAWyQmuXD6WoWh52dF3pZDQBBCC\\\\\\\/Lx77s2w5LS4xfnv\\\\\\\/lh2djcdUrwq62fTeljvMAf5+hJAG8TQhbpTEvGSQKdTjE5RtqdSz4m0ztfk\\\\\\\/Qim49No1agk\\\\\\\/\\\\\\\/bFCnOh\\\\\\\/yS20BfQOouBPnQGPdHN+vERp9beHD8G1aYtV0\\\\\\\/rrGj+\\\\\\\/ORGAO4q0pGkpf+14CVWOjebIZNpH0L1rkoDq7JJ6Li6+H\\\\\\\/jcHjrqaJeZUWXx+oOzGKikuy0b\\\\\\\/ZxbXLjSq0UtaeZP98zvz9dcnyPwhLjzVZspU7bN2UiwxfIkl64ipOu82DTICvIQxiBM6tgSnJxgLlJdmIczScwKsTlMoG5Yp+cSsN21\\\\\\\/uXQ3bmuWC9Hxh1uvfF0O2AmmIwaR6jtWXCO5eplJs2bhdBpiS4\\\\\\\/WNroOyummoqptZ7x3wIDgF8oyqxC83RVlHVFFHKOBU\\\\\\\/AE9WAwDBlKeHaqtl7vcCqYBY2igHs+4kRGT4JEyYtbVjqGaYu6+DpCZsWm9V3moH\\\\\\\/yHqkRoH5cfK5JuRy0N3+FVl\\\\\\\/dduq\\\\\\\/a+3imaG84zFo3GbOntfyns4DiOlDs6p5rRa08xELDvEpkxwdpybfpFq6kmkMZmOPI4mUORf57DobpLkIrD3hwVIR9KfQb+vGo71uvFkrVFzQFwJnQhH911fgX0jLvXJm4wfAVK+o2pxtcBg5ajiQDqucM4q+fIe20mcyLW6aUGFSi91qaCul0oeMF8l+lCvtiMlitC1EqEqCL3cxCfSS1++cc\\\\\\\/OaxIjz1hbVa3GJLjyhJu1rNbtCRbouMql5YgngCN6omLBYKVDegl2N70ovV8lLc7KWJ4JmkcNnGqrX2rRU\\\\\\\/guasoZVqUuq6ePYzLq4ScbI+vYrGBqX2uF1rUAoKVwAsWIIYjVAF0oWoKaSid544erDqo3n8TTwsTL1f7VjnxMTUPssvQSZzWPbgGquu5TJNh+3FgmaVTkOQ3vSANtO7W4x+d6rlIRLj4g7To0ea4f+3Us0EKTxdq0TOOPfJuHvmdU7Mtce3ae6pJ4l6CfwnrnMO2hcIY1\\\\\\\/W54Q2PDCtwrUKCebW84FXhh0pdOn9AWSrx9ftILLO2uH2Oi8SmlvXK\\\\\\\/P3eOcveVczvtU2B8xMeI5HcmbNy3SGrSuTKypO9Ssk6AjCkZ9frC1BqEmNrrXtoA88LMl3G\\\\\\\/aVTVeG0PZelafZenujpFPOHP1zQHi1MHIVqG2WPrJUKs3pCDXVUAT92YdKNyIU5IL1mHgTm4S9D8NQQqd1W8DRSweZM218JzYRSU5tLNV51AcHoRmkImVzAszsM1DaKPrgQqiaS7imGtTTF9yY2xrjIbCBUzN3lZ+olrRpxFIY8ltvYLQW85HtFDmB2e+7ENn4m0UWBXPIovHVe44gNPgzmN6nmUhWys7m05K0NBun0eqA79WojH2e37roU1HYm7XD0GWxYOpUDHM1nsfgMpu52ARoj\\\\\\\/YIUvHaxd5CobbLZu3xhQi\\\\\\\/\\\\\\\/u6UlNtEK8zKDmBkT3pjZU8byXjSddCrTEAJQT2vxCfBkRieQ6SaLwB1pepjn1lsrP5zKMshQqmVh7VqcJL\\\\\\\/WzXLY40Pj1U4sdCC19YmraU9oTmyz7afM8L5s3iJVeXBntEwNpXVWklqzAPpbRwJzu8OCs45\\\\\\\/qQlPwMEqZVczUvv2wj5W2YhUbIWlQMQNW13Xycawqqf4qOckpRgePf+mqplfEy6Q4h5OqoL07n9XMu6DnnEwtyxqcS9xnLAbTn7k85Ca3d56IH3flAWcS4I+ug+NawrezsUfAEYvIaUFWbcoXDnzzSiTvGb8b8B7CApZV6eOq\\\\\\\/s5iuJEK8BSjrBZZSL9beECtyd8qaPK7DRqVcJMNsQscIMPaBiRqiss4gSUqehutij6xM+VNitGnTyEeW0XLAmWeRbNuPuRoMVR0TFXX5IPrwQVLz93Oyz243RsmSPj77NQdhxHJAT2CpPDpb8qu2ukb56jdv8V48U+crYZgfJGblT2M67ltziBT\\\\\\\/qYl8d6odHv4HcRJFAk+\\\\\\\/GHDyVuBtc4zV+gPiPxSqUdWr33NtabbXlm7Gt6WgFOgYY7ZGZMCZUg6ImGXnUVTxHmos+tYfEbRMwJM1Udf7AYJFMC1McdI2S6O1lNjq5muMpuKn0oIVfWVsKobPkBVbObRE3NdPcO8LvkTkSAsvg0aRymvktKzIAGXA404+5l8y0B4yPAVko6oOLbLVrxhvJVv2Jy0XxVo5exzhExyv7+tVZX7K80q9epNaXZp+zRDosAGPJmTRp4Vtvz5y4QsMEqiu980xUFj87ZAYbx5JZun3LnU4k\\\\\\\/nUvDqPlgT12Ref00bS8QC4cfAWRV1MScL3ciAJvp+RRNhaCES\\\\\\\/cCu5Xp6Ja1Xf4KK31DhFA2KWAKgKwFGribTxb2ClzbrrDV6cYzUTukUVao3DybbFbTR4efkFLQwMF4NclJPgSoHLtIn3qNEmap4\\\\\\\/Fin4+woxEvaM30qDni6ZoFB\\\\\\\/6nNLJItLjCVHlqozSqkPIvjxWC5wczMqpz6XOSvtwbIIsNqq9x5JT9UiWTlpqn+8ao+UClcdffpsN67EtcP9WHVNSHXB89YRXMc2gKclxex2DewHp1kHT7g+p441R8IuUwwgYfHba8v7lqGSJu+Gpiec+fq787UikkCTkfbEZBY1AsH8FXjkXYrwAJAQBuH4ahWnCzhb2bhZH4mwgc+zyUz8bk3pHDxa+CsQDHDFPVjx\\\\\\\/+kRXh1YWTfgmf54gO8PtVIB2JzXIwHWpon1xuVhUPLtTkUvCPFZJL52ablIY5IlTWbXM6uC4fNzy1zii+8T86WdH4hOlxQ4HcgSSjeLJfCGItzOL66G21mgRBOrGCYJQILwMO6xdHuFT1klzTjxDkNwFugUtpCDDP5WPPe803xthdj+BH+xd6O0LO5fJj3SdnRBhXT9oTKbN7WAtXk\\\\\\\/H3gCGoXMszoiPMEXX6hM0lJcpYHZtci41WUbEgsbytxLR4QKOBuIAaC2mxx35YuOF42WTste4kuczKssGCKlV9H7qVtUcz2G06QD2eeM0jNxVDALcCB5DWxohPU7b2mrY20l3RtU8+\\\\\\\/AyQPqbc3j7ZWrEP5deYmHFp57tg1MVqDrI5xvP9iEru6DTf+LTuOVVyRpqPBTnEZhMK9wOqSLOLchoeXAOKsm0m1A9zwzoYgewa4DFvBmUV9O6tyFepL+MekzS6NJFeYXU6hGtupE1wn9iKSKC4fRg5ePrzJiLRgbDmUlaWy8rbmCfOv3LwgTycyWXekAYo2P323vem3YLeGGWh0sip\\\\\\\/wafCZkuaOD1isH88vjQkj1r863myPBX84T8J9Qryuq95t5NmT7Mu92DCY1fo5\\\\\\\/NvxQJWFG8xwPKWHRy04yyeTutsZnpU6hii0CyaRPpvMz7rSmcFIEJbrF7En+8o+JdtP1KoRbLwneh01TEZTIZibF5Jmodv1SgZUuKWCQT8ziFcLHmsd2wdaMMhiVAoxmEikx7HHN1ZH7V2Mwnpcuk8Y\\\\\\\/AVrD4M9w4Snk0tRk7gHcFREHTt3TBWir7\\\\\\\/2novwypjzE6T\\\\\\\/uowNhCtWUzzsHaxNYTNzeqpMw4FpJpmhIlIWX3XO2d4jNRCKHuBnApVTQHbQo5\\\\\\\/kx+lWBKQiaKOSSkeXJmcF\\\\\\\/XFPnKldRuBmtkqm1rHEetClyEC0ZYg9WBsV7gGbU3ogJSfsIr0u5RZqxKkYFruNnkDtKmjq7\\\\\\\/lXf0LS91W3aYBjVBvNTOa+7EsCYqAACGRIfwHPTuWwR8t+\\\\\\\/+i2Wbu2vifwvZ\\\\\\\/MFkRzlqq1YTXr4TvX0g6FwKAqjgVW2tq7CGKpKouP1Jq2J7oVYRWZ7lGw5126vgjoXM2qWPOkcABVpmt4dbChxdo6H2dcFVMmT3lHtKI6FnsMDmk2\\\\\\\/yJVAZVWPrlc+EQx\\\\\\\/QFSouLwiRZ9rqDeZCR35yUa7OiNlNrnYILDQcKK8iH9RW7kAfabQCucBJFmY8OcdKz5hz72y6w++D\\\\\\\/7wNDkRA+yhHit0tmM\\\\\\\/OcZ1rs9ZLj3uo83Jzs\\\\\\\/uucmd7Atk42\\\\\\\/r47g4+zgNHLcefYNE2\\\\\\\/66ENvcszFM\\\\\\\/4aHoRddIteO7CmWH\\\\\\\/v9\\\\\\\/p+Wq9Gh9Qb9tL1h8vOFlYqdbml0QTcWrMPc6tkvBTrwj8L\\\\\\\/xqHjoJh4TI98cYHXRJyfWuhk9VkQ2RaBP2JrF+uWmDUhQNZD6yFXrZmTNNeX\\\\\\\/8aYIvzHfUHN38LJvvbL9NS2yv9xtyqzKwOZDc3s2wnBhCmppwUzj3YyEaCAOBplPGyupEvJeZxCg\\\\\\\/SlliHuJ6S2TI0Kt0FeyO8MB5mXqEnFp1Asx1tBrrmrr6MbNFX6CmE+YJaj2ntShiMo6NkGt72LYKppPLWT1S11NoqSZgFaPGPUv3412\\\\\\\/d1UHdO3xjtC+93W9SGmTtDEZJazq4GjIAjziarC38Zkd8cYdr09GIXNW5POI3MZy3IQ+YbgWKJsiIPTSc9PFFqOYmEZXTsxvJrKalepBMH4lvR2cp+\\\\\\\/NlXdYW3m3\\\\\\\/J7RO9mz2CuzVuEX8dklqQrrJngoXm8Ue4bKWUbbXdWMABD1B9kzvko+iVSHkeU2eRcBRVlvqFAApEEFeGFr+YHpTsx\\\\\\\/txUy1Pgnqb2z7+8MnevmYYh+qlDfxMYZ6XxOzMyEIdu\\\\\\\/pnh4NcPHvYp5A+l+KgBIK79iZBhryp4g5IDLXpAzaCBTLARzygcakTaFtcLIQdv0hlnHDN6hQyopYrgHeMR3YhmZuCWW1L8RtkCXUR+KW\\\\\\\/u4Ahs7aNhluR5MH0qwDRFV0p21jCvospNwNTHLQEcpwCujudG52gh5tcEpCAMjz6xhCCddiXRW56EeE1RwRZ1Zgkfv8rrol+bsrdwzhJ\\\\\\\/OswcSRy+TKdjJ\\\\\\\/AqXghpXxRmcG8KakNPh3935KNuHQ0+QyYw0X\\\\\\\/29eJTFFmr7tIXC8acaFTaQ\\\\\\\/UAR6t8DW527\\\\\\\/O\\\\\\\/DEpSlPdw\\\\\\\/OtdImOcY3lgO5CHqgFm0s+QZtaztfZpEkktpkAkM5efKKXceZUfgvrG0RuAe3oyL9uafoyNXisHtxtFk6ciDUmtTS+DcNFyKW8tKtIYSvXAkFlPKfZ1aZHbTqAwZqLrQKYFN+9Wvr3377bKKHUwLcDka6ZmMxz26806nr+tQgzAF7lYd1acUPDgxqLfo\\\\\\\/xzeH7SiVL+WcY5ng+X9WqsrZfYkg2bexgVmkY0Mc5lYBGy1ymmnx\\\\\\\/qsg5\\\\\\\/zvhMS3qOJXNcnnnLTEpQlECDuZX7G4EGGoDG\\\\\\\/B4hstyxD+Q72tBpxyuot1SQTGWn03lQX2Rykp92xULhXvdHQNDnmwgeTKX3Su9zfKrz0dtl0DXi4c4yvxJhPiEZNIa5AZp\\\\\\\/pQgouW50zfk4wGGy7u1MqQ1Tyj6AcdE7wEUedK0m\\\\\\\/IqG0nk44f57RV3eVfKAy3SzMIMj5aoXbW8JVfjxllSaEe8mWdq8ndZd\\\\\\\/uFY25MkVX8S79kn24GuRYSZdLM\\\\\\\/tNdn0RcHcS0I8mLNHsRRSwHA\\\\\\\/DZIOBQGLvqw21j\\\\\\\/Y9BujKcF5+ubdNa33IEXSrxUgtoaDqrpV4NLyXmHHYgcRKlux3VIPTjlwdkzqRAX+LLxf0g+y5GZkA3Uhw6qz1rzvVVX0mNNiEJPRDA8e+StA6KINO5lM\\\\\\\/OoIRi7dUepACTj6AlF51sw05PHkXkOVca5x9TnD+fJOHnuod4+c2gxAuSTAd7jRF6L0q0BzpZv4+AjJrPYFrsaH5BOAm5woKHq+S+gqzuuOK2JFik4tS1pIoLZOmjJAqUL0saHyZWTTh9Uz2ohPRfmxDNENPETShNvIc9uG95dQTwbQX7BbHAZ44GBsNd0baZdcshSeCY\\\\\\\/lJAkl+yAyZSQfjk8GG9EzzWV1iZ2BWcUKWN8OWoDYY8mIE9EDOHqSP5fP6Em0mt5VUBNm39p9aXXgAYSwqGAyDCmjV5oCbT+lqfJKsxtKuNgvaz\\\\\\\/+awW0Oek60xyBXI3zKLpWH4IL2MyGyMJLjSXjpsMc\\\\\\\/fVRn8WcedtHzmL1eKohKQT1vLuuJ9v2HcVL+Fp3ZnDDLNpxwo6yqmDttukFD\\\\\\\/oAL59az5UfTUi8J\\\\\\\/OQ8u3rccql\\\\\\\/R1B8xMOL9FO8KTftTF75WwpwAmt\\\\\\\/\\\\\\\/HsTUFdWzn7b3J+BNotmjUFTAdGYSqkSqTcvclu6esY39Xe96IrPI4pDXFPiU2RcGDenTxYdC9o6dTdObiLabRlGOy40IWn1vUudhDyjmklPE05o8yzL4iqkDoPt6p6w9MrthPDPDzCPZu9RWmqqootQUx1f+0\\\\\\\/rlNNcdjr2GoQjT+oyXdFFdr7HXC9UAQIfUKW9J19XT\\\\\\\/WCPLUcasBXRjH22k+aYnkD2n+OJPIgAGU4KIo90cm8a4nihmXDV2C6ZrM9W7ARAfiVBAZBCW0zq9hWkscb174KznN0VuOY35BcqrJA0MojNWs4f0OXIDqKIQK4BdujoJrjDMtvVPlMTz0Ie3FadgcdESphlyHlLxeBrVz15zn5z06DihrShBUh93Zu9OeDVgTCto248cyTFrbilqPL8zSj4kj59ycpl95CzWi\\\\\\\/YlQL\\\\\\\/E73JXuPwsUBOOM\\\\\\\/z26S5gyE5qmI4Xyz+RVNH9VUVGpTTVzbNt42VLdu50OHr+QHsvcjtuMZokJ5WxjrkOdpqIQRXZH4G5Oo0+ax9sh5vqFKXlbsTJInA7uszwI7U2\\\\\\\/j7PVHdIyBVLz9Mj2lNkEOTGHtc4n6VQr\\\\\\\/bLFCpYelDYgrAslwYqrgoR0gXqjSVcOt4Zrs95h2bTXQbnx9rfi26ry06DgEzSzYBdrtwK+FIC4AMgWvKPq5RbZHIOMx26RIEm4n17bZBDNrJvuNbLo\\\\\\\/9ryL8kOyLLUGTnLg0iy5YfU4VBbdnHHzvjyYHeJoF68Gai3oOSK+1OH2qZIe7bA2FaBdaDTN73DMRN6dTvdYMd2wp7GzjhrwodK3XyX8tunrhu5Q94Km29zbfZVjv9j+CvY0wR4PxYhBKFsSI4yHvRZuD3f0EP5nn3OWcUrVVrb9ylJa1FP2x+kQ9w9VBThwaY3gTr7IuwibL71ndXnsCkUvNlob6aIIcH3nk7+zTEaUewAEl1T4O8k74XjUPHViKWhFI2gcLWFzBjkdnwQ+EKpNqsE7FFYT8h6oo6dYnJ4kSr9gzY21Sr8vYcjg0CcUlIuTFJI33JCuQbiotJWYGnH5HNs6HMrJ6kO0lLEHlc\\\\\\\/OdFhOrBU\\\\\\\/RedqigOp\\\\\\\/6oHKsYBhYBhEHQPmt2k6qiQblYDIcipWvlcV1\\\\\\\/ysvFtEpAU189JS3X1idTte\\\\\\\/Yryc7ozQ1ouIjJNZgK4dm7TowLaWrXOrHDj7DNKAroL4ED+INOitIX+qB7avoVoWT4oUPHjp37dlXwz6\\\\\\\/txeC2FpFN7B5HHLhJkGT6PTZELJsyPntS8TKp8v5oTx3\\\\\\\/B7VtkzVPkEWL\\\\\\\/w6g2HeKzy6DaTTkIBjTnKOfdGMLopgA9+k86zOYbRT76cU5vjlwX5ThkY91j+RU7c6ZSu5peWGpFiYJZnu3huTksqz5EROGMYoXBeDFbF2fZ3eX5ex9cX9AAA4Mep70srX\\\\\\\/fyD31b+Uj\\\\\\\/q8LVPsc8DRvREmE2koKsYH0aIOKmhlNmnOYNPML35Fh6\\\\\\\/Iqa4Cny6WXJX+pjHpoVeZlfE0rHGRn2DfNrA\\\\\\\/ytSlXNr3oCVAesfC5RBwZvFBdQh6epJG6BHze8ntSr6S7j\\\\\\\/zjr5IjlkAESeOzxUl5ViNMeRotqbUC3AImtXRqgGy6ybc6uoePItznZSDn\\\\\\\/Xop9163cjEz3JhO6sSckbAYUXBOl9uwS6smPGfsT7vU69\\\\\\\/MRUkvPLc3k7a7PWieH1NOc0F0E3tNYyyWtqQfcE7K3kOFIdjWOzzCVERVulcZM67ZMuWkV8+V6t9delfUir39coOBDxIYf7aFtQY5HPTlAFQ3EzZJhizl\\\\\\\/5i1TtAXuay8H4ecMQk8Me8EM+KeIx93GEEJmBblwm0G27xZ6bKK+RwjvC6T7RJ4QDoDKaCbRNfMCqcQc4\\\\\\\/ZkHxP0INKK5oP5oscbSTwU\\\\\\\/98JQ+WNVaDNrmDY7ukt46X6LxksYP749R7qizElpbS7GZ+to6G1gnwIOGsfSQkR3LJmfaWE0g2bnRMZ9q5ADKkSkNHzl21\\\\\\\/SmiYZHiIAvA5n1PQSG+gRtvwV2pHb8FU1PJnkwtzwedPaRptZfzrs92RfACJ+\\\\\\\/h9bbOo7W9JTfXeRHY8DcuLBTK9NN4dNwQ2fbwpBuVHW3OpefBrpPAv7X3oDzz4NaPQw5eeMGxKbOLnVfagU+M7DYiYukc3\\\\\\\/VJUwIqD6d\\\\\\\/c0Gyv2Yhm5ErUMdtn7w4ASW4aFCj5wv9od6a00iykOwpchlzM8x04ebM0RkY9l3FJJ8TfHMhTTocMa8+erdQ8cUeDui7NcGyjCY38jEKYdPK8yR92r10xmbR+IQZd349hWiXHoyuAdKSrRYXI8j\\\\\\\/ra+9lIwOU9WENBRthClbMPFjlRoCxwuraA1CtzX1RirTzU+4bTxyBj64m8QhTMB+bdAaN1aCK7g1hnrEt45TmYXbTs\\\\\\\/nTcjDCOcWC8xlkWzBH6jRyaUtFoApx5nOWZHMW+ZM7GK\\\\\\\/Jdi3Ok2L8RHWwv1g9KcawA9xnBX0snaZIuAnP9O6mP09fUPA6jP5LhZ6UvReEJ\\\\\\\/4K7\\\\\\\/cSQGhwGCs1hpGoszNcV1sbgZpz4Ff1CvNg4yGXBT1PwdXw7fAjSR79XB5msU5kJVCEcLZqzPvbTbY4CXOh6YVfArBrHCzLJizz5fqE3DH0DUEqcWg6NHf5PmIUbph+M4j\\\\\\\/MC4lK6MmiseXb8LQKkBRHYewB4aU0WV4xON7wL\\\\\\\/WYWYWXAstrhh5W+oGFxRLdKhK\\\\\\\/uP8P0\\\\\\\/URDzGTVIWL\\\\\\\/ql8fmJ2pitV7OZbi8WzqhgJupZ7v7IlyTNP6BxjpDx+6F6YLN8uf\\\\\\\/ECJE2JXH4dtPM8UKxVm5c3aFfC0siZtjamYwe3g8nOTu1iVcdMCFHsrGXCquzegEuqT0HiymI+f\\\\\\\/UxF8rv8hU\\\\\\\/PFVKt0\\\\\\\/lYD2Z3SiAPwxuErkrF5708slhMkEvw3RLbhc6YoyANEi+lqWg9VBGcdpfCNqprHpmvdDszfHuqXAgQLRh1mTw9QJxjU2DvOgDRlaJ2+D2Os8rcQPvyeSzgcHhY+6D55FQpx4jujC9rQYKAbdlH1WrdxGPqm9xqIDgnReb1GIaM9mry16254Bwrh8zcVU5spYMo9sLiabiHfHWLVRy7GVy1kdZyfu0t7HAHOZYdnxY9T3ggdOd2+W8asJq6T9Ut8tSa5nLR13otx\\\\\\\/\\\\\\\/+nSWKlZPjj9q8D1RnFhlUEob4b\\\\\\\/8eiz8skNPuWhJAMBDFHpbGRFXKRfswbjuaKhtgHQ9pxclDzMjSUFeR7HhckT4GLYoK5MFuulq8Sd2nwqpG4DqxPUVrHQekwXQu9Ll70b3JipmSBwWjrOH\\\\\\\/1uouhttMM5nVQpl+NOrIpZAi17QHjN70uGhFa7Xnck7v4xWWULSq9R3k6knkpqZywatUi0T0mwIKmrVXvSu3d1M08OHw3AZe3B4vWP83MY\\\\\\\/0cnWxFX2GLNEpx46m5xVMRn0jIP6iTHBzwizAXpdvKomVGC3mHAe+js33hYdc1pzElWvkGzwZeCsn2PiNmTDgi45OQZ1G+GDvss9RL7zdQSSXnwxJFnVBlBZciD+3CUGbW1ANQjX0O6RQqEeyZNzZemmnfRUr3Weyz\\\\\\\/SLVBl9Am64oTQHiBrI7PpVrJH\\\\\\\/C\\\\\\\/WFpULFySZsGHn+jl+BFm+Dz\\\\\\\/BzwCclZHVu1ibA2xWquN8\\\\\\\/4Gjbd9jO10cQyuXKPhKuga0G6z1FHcYIul6O2VZXt7lmjURMK0v3jJF63zgjJTQJs6ye+uOfJ\\\\\\\/sjE1eAqHpLDiCMcmQB31O9QXI95RiVAHMRrjkMQznu1delFRBP5Vw8y6fS77xqfM94qdJ5j2bAKoBmgnQvrpmORk53sJE3foHsw0kHQkWsoO7rRJHl88qYr6GDLTXqczspoHaFyR231Dlyi08Ka\\\\\\\/eixMulUO0RZMWPg9yy9U8ry6KF6dxFcDnfkPYC6\\\\\\\/S2hkYufXu1vEyM+lOwAl5iK5YXr54jrZbfdP5Mrd33W52paZq\\\\\\\/xT3hbNJFs1kJc\\\\\\\/xKFy0mQMYh8GHBhH9V2RzToEF2ZVK6x8WpEhldq4QwMTpJODAWDCMvtpnNczyTH1EoGcaDTqMENXA2qbphpAxJZ9lsXColQK1w\\\\\\\/rCBTn\\\\\\\/8E2vya1T7aCuM9s4+SFK09hvimgEahPsjrfswQCfiqe877nVN20JpEdex37H0nNcPa+LhXRGWh0etOcfmI4RR3PkITKPHJ6qDN9mqwIgRGpsuIYLYGF1UAZIXuh3u65OmLPM4Dk4kNOLgmklakXSvGXcm5LNRZ7jvx41jnpL8H3VCuRkY9tacidDg4oF5mhSaV8iRg\\\\\\\/YF1p9AiyWFzMzbVsiKV68zDLBqpnA41AOVzlYlyCjtvpI15Nw597BVpWw8jcANl\\\\\\\/J2a4z1PULDyHZmhr9TIZEbYJHy4oNq9bBj75JFgWUVyoJ6EcvHIIBzVaxl4HQv8PJy9rnoYQK4lG6LaINoWsNo3Sk3Y0ha2IEmsyRgy0l4R7l+kAESW39IYClz8ZxgUhjQNpo5pG+0xZCKD\\\\\\\/h7fYsZCfACF7wYKmysYKkL9Nmu+8iVtFi+wA0b4sngKDS9HvEDWa5Z5Cly373rwANm+C5Pixj6bsxqtW\\\\\\\/zCV1uX8GVRHuhZdTZ7cPVOvEnOx\\\\\\\/25aI5Ga\\\\\\\/hBGKbet1BEx2QI1Jecv9JksvpV2wlmQW4act\\\\\\\/s9hWRAkCbmJJ3+vq6qaNDl1\\\\\\\/IH+H0ScZrPN+fTgrwli4058oS7JiDt2aLySChqwdCPe21m4UhmK0t6yhtdU\\\\\\\/VL8tDUAYOLn3ieOC4hueQfoCm1dN\\\\\\\/7yX5LxisXAguzVYUDtanx4eV0BRUlHF3oZMQyv7miOXAZg2dHJvkdavtscO\\\\\\\/jECNYeLoN1PoanCUd6pZH6dGtN1PcgMNlFczssEwczWAJ+zBrvqJ7wFzFrGjak\\\\\\\/agLIx5l4zuOfXhV4Kl+vSVe8EKEQ8kqyzKp4KudrguRK\\\\\\\/5JgZchdDyDJo+gNjbbFyCYWaqugtvgUPcrwjgqgpYbOg5oznX8XBZz26EvqPaTuHfxi6czf8TVNW5Vro\\\\\\\/VVmU8EjHt9R96Usd4kFeumu+a9YAab\\\\\\\/98ojSeyPL2DufNPjLI4wq0zW+0pUmx8G2aA2YLUi9h+B\\\\\\\/Vh5zPQWpJ7i6H6AA\\\\\\\/oqCzuRjJtsCBuKkFbfuiZle3\\\\\\\/I4r7Nd1LNUx9hK9DGh5pW2ombmq+6yC4yint5r8MxegZBnk7cQ6yDwMSqeVPnpGQY1fMQl3cGr+oJHt+WmdRwHfEo1JWBbSQMFVaGyt0fLH7d16oNQWN7Lth0IRSBDxQlP5TIHC3RoU3omCMs6AQfb5SoVYWeP\\\\\\\/rT2ijMcc2dA8yICAo+GokQ2VSZ\\\\\\\/JAlPOqxEAEIJoTTg+1oVmCcBYOUL4dw2XoZOpDugzpsl4D3higIm0sW48N3uZz4ak4MzkAqkBwdIrdc7uFu1YA0kLplk6yi7cC7hZ2OVWGtF+0bQF1PzQtm8reDevIoPA6wx7LWf4vQRVRzlwEqozdXL11tCpkiFM3jpmAKkJgRiB8Jb6Z08CpVHtOCctpZqomqyE+KZcRS60hLcann0GDi6ZbgwteXHdW6gloaUAX8KFFKLTk1vYfWpSQERWz6azZ1\\\\\\\/uk8Bbkeic84auUIoKeYj1CxWFYwuc+DPsKht3TDzl37WZWhh\\\\\\\/3MyZot012fpZpKVU95T0HORpb07LwufItKowdr2o5zjZ475OzXs\\\\\\\/hN+sMxxriW\\\\\\\/uVWb8S9OkM+qgS96KRTICypAvPA5DdAgy7uPd6gQmBZvAkeFf34MqJZkePixzjUsCDrZQ1xDVtWUtRhTXMfNtIJvWj9rgqHzELeBYUJzq4N9oVsC37Z0inVMrGFLE\\\\\\\/Ww8RqgIL9vU+MyR+7GHQOyW6h8iRagP\\\\\\\/a01m6MyEqaFRkNh8dNVLMlspZQjKb2Wjjyj3+qSuwpHOqM5z3zS+x+xzyB3NgV728gAnEAvD\\\\\\\/2YqoOl2MCCtn5oegEceENsy\\\\\\\/2Ev3ZloU60wkBz7LP5YEETwTDuCtrtgursXrmPuTG5EB4AgwHyot8Qvat\\\\\\\/YKpk4DwtM7CuG0rpAZPBQaVPiBK2HBB3LDmCBRcHGaKeaCDcaDzd1MTBbBVMPXfzUK5MosE50MFHH79cDKskJw2rqoT6EzrVHuTRUN6hctgW2lHmHFz6L4pFnDSulTNrR+i7wFxiW63ezVldzZaVh8oIp9BRkUC6Uo2FRnil3FIoH9Pwvdshb\\\\\\\/zRPaU5SCWAlrr\\\\\\\/HuQZwnoMz0bf2WUUs0d0vb\\\\\\\/cM8ENw9U7RzTCShhAh37C1jWj1LhCgm2EmwGXfdt4gpT11BRKu9jm+DojNrLShkYrlRmGp8PRxqc06KxgOLCzTpCmbsLGUriNPd92OU2iUPOv0fLYGV\\\\\\\/lCiuehBH55b8qaq2DbaqGRQkq+2uC0IEi1QOAeGgwzNHEuTAhmvpNVliFlEnO4mFmcv24FFjVyN59l3yuQSBzHbj6sNGTcIpG31xYLI51cKhkR22MPFTX1eLhsrL5D1oJykJkQ3gXQ8hMzJWZXbW6tnZLtZ1SfXjKCuaFJeICnqa44yQ+9meH8TQOq9m44TbZTbhYcAkMyDiy\\\\\\\/jHUO75kaXVM8zfAIxYFXWFdDNp91swTzjV1XCCyaCsP5lleTKbLWAYUU3bMHEiz3hBbrZ09CW6RVYnErGWgAXZh4mHdjqtTK03oCgwh7YbQlICIewENu5foDk899yn6aDBx7dYT3xct5jQImJL0\\\\\\\/AIIuJvyihI9TMpLGi5ApnRHGNNsS0bkgtBik\\\\\\\/7n2R2W4Lv4H2l9VUDaOja3zYMzHwS1I3Gnpdg8l2iEHrxp9CV7+vL3OwrLWQ3zXcbM4arlH5Yw8TkAz8yjrHC5458tOPl6fHQkbgKGl86S5FCwRg7S6zr6jrtE9qnlZJN6UGwRGIr9tzfcA4Eq2Hsi843iACy2vQsX4OisC96DCMkSPUM42AP+avwkT\\\\\\\/al9qcFS2Qnj82xABnlKl2Kcy65W98KszeLGQmSH6oFOVpTd5va\\\\\\\/RoyLfCymyuzCtlHpJbwESyyXI8KsJFBkk7zf18WvwNKtET9lNWdyFZYhPLqVGb05j3H+AnWUDojAM3wvj9IjsHOaBRSc0X0+kiqBqRiSOE25Xdi6aTNKEjIGUWrkzUw+QgM4L4\\\\\\\/yleHwSw4WXuNvARMhKzBNWTvXiN\\\\\\\/EgUgeId+wgEu7HpJQXOB8xXgw+K3VBGzvnvpfM8cZuxnfArn6MuAE8rKVkW6wGRCDemPe+bvE6b0MWChQV9C+o7\\\\\\\/l2VHhsuCF\\\\\\\/0A+Bj3XwVQJVKTcx\\\\\\\/y9Rqh9eBTPPXB7T1uRkReBUB\\\\\\\/W65j1+XX6H++reyUj+5E1H36JkHfkOuZGawxADuOrWexMyXFSWz32j\\\\\\\/VZUxFmaiM5JuGGTzW+26Yfxn6lA25MrYp0PAf4Tx1zXpbAA8G8snMsC6RP623pQPkEVaj9Uz0nz8rASpRvM3Bls1S3q4Mt5MxKPFXcpimjRsKFisexdqPF0\\\\\\\/81pIN1Fn8OqgNTV2MpFeEjH+HmieZiUKTIM07AB5WInx+tgi5+s95JC\\\\\\\/F31LIRTIprXrheyQXRFuhTYq5LDsfy\\\\\\\/8z0nM+Mrny0OFfeo08s1uI1qxuIThDI\\\\\\\/fCdXqVPiaFjqVwGx3HRt1qfuQdAJmsOCpoI95YvzRhgDKYrvtNMs\\\\\\\/zTnX6yfrdmea7udA\\\\\\\/+yRJ8b3OVg7QdpIkkPczvy08DzsSXzBGkJXZYezOc9gx+a06IDYtcEqf+p5HI3FgeusfGtrnfAsv4LgELxbQ5fO6EROikLBXeiX+qtBe1pKNT4NJmk2yr+cPc2PY\\\\\\\/PRlgcKy5CLSYmkqM5N1xgFXChGaREaSNhIC28u6Y\\\\\\\/+OFBofnKGxIlwsEByABF9ud8e73IdTDywuSKq7lvlEpSJKUaS8YJJsLR+DfJ1U5n2aMVnmxOvIm2wZaRCXdivpjHZOIFpOFXpihVjOaHz9nd6+pdIEK2kPS4U8gQdcNoyFf4zlXEyA3iKLC\\\\\\\/7JI+xaPdnMEZYRADw6RtKhwliZt+3u3tjac7DMGZdkUwPnuoI1tl37gqlNl9Ex7SqP+W0Tv\\\\\\\/oShIMr+VtzWM4GeEyTxfamPy8eQiLV7ScJxmPT91eEuUHcA53Gw4rxmgULQ0LsF0qBuC+y3dPK0+oDledGlgW14ofSf6MWiUwNIJ9Se4B+c4yLs0xaQsfw\\\\\\\/y0fI\\\\\\\/LTvazBiHboPEhPKW2gj875c84yMnDf3xHAjqVIifJUdJeMTn8gfjCCrueJw\\\\\\\/jipu7f47xHdt4rnKdOavRy8cNQt4vl3Eool09gQ7RHnoyUpANWmODdF06kBxAHp58Is0v8xhWZ0AX8PqazAV2elaWQfde1XrnAUZVbYfFOet9GjRdjmdErlS0g1E505U8ADgWR2WaCFNjLPZDmCfDCVFZHJbaaUkN\\\\\\\/pFI0T17XlYTNavey7PfdlYMke2+wluValPwQN8CEYZ\\\\\\\/bnSQw11am69EFkq4\\\\\\\/UbYCCgAT3mpCGlzTN56JobQnnhOi+XCOJFPhcGTpXPW7+G6HmQXJo3Tq02uJzs7MRQZlE9UKDaz\\\\\\\/AGs3R\\\\\\\/BUt089NWeFdHBIke9cRN1wx67\\\\\\\/\\\\\\\/8EoPDMnK2EUXpd7o9JB5oUA2PNF5tOlMhhC3XNp2L9jNEJjsykUiSNdWGg+UZEsAZiRPbl\\\\\\\/+Ba8dBRa2p7iKpifYYvqhQhFwBcLflUlAJFyFIkoinvWiS1VE+zKkEIiGpoMyvz3DSCKx0L5dlgsb4+5dbCkiAUm0xd9u4cVWF3GKrqeAYcCO5LwI639nyC2TnNBl6UCmpURw\\\\\\\/WXgXHUW0X0Jk65ik1vXdlAeeqTvZVqWM9xbnsZboUTVcj99bEf0iNAxWdozsG519L5dcRUvnTDLYLN1BsL3Xt+jVGY+ONPk3luSXNUgVfgVGS+5IYSm8OGAn36s2STqdCoXCYshL290OPlw9M\\\u0022,\\\u0022iv\\\u0022:\\\u0022mEhZG3Dw9XbEIC3\\\\\\\/hD7Ong==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bcc77376a4c13a36cb96b2b9e48f6097aa219d0c6faf8b6741db8c7b93c6e2e7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QJ\\\\\\\/0AEl02ND7oT2O+4B8fjLNWT3fQAW7\\\\\\\/hww3JrOVpKaIFEKmFtIaCY++1B07xTNk91Ij6\\\\\\\/+5xgYGkjPUMKpGuUARLvZnWc4k3VszVcplknAjbs7qOB8XwAGYUG91Fy6ywwxNCtAGHBmkssjTCPHDfNvVfEZk6ouDKY2xOHZs2h7Xwtu0xpHjzphE\\\\\\\/xo4j79bTaxvaRC6fTY9oi7+sZFqbNz7S\\\\\\\/M4HwQI50oEl6bWex0KISX+uOhq3yHDDmyjItaj5Y7\\\\\\\/18+k1WgYRyRHo33RwZFptxj+nQFxYJQAnUOr0H43xEoZNWMGBzQ6Ml+UweRooIyCvy5mFpbBSCGh8vMMlWKwZrlzM1pSf1YoDDjB4EbBmv4x9qpHTGoMu74uF0UqgwO+Yg+mBPV0lLrEkrIoyklXlrFjqtBXgF3EtD4Pgg7r3iGz149h9JDr9\\\\\\\/Q+9+TD6JwKw76ALCnWSnxj9vcfsjPTmXI5wbYqhFo\\\\\\\/yh3NKWPfUDZnpTUBOnYzWX5cKZ1NCZ5gIZM0ZfxpYVoLy\\\\\\\/F67YyEK4zmqjYv9x9HWHRxIuaBGGFZ5xV8MZKUiLYM+i\\\\\\\/NVUXo6trLRiDiKasKz7dbLIuRmqfFBFB5ixFo8wbH6Wx003pBSGP4ITv9V+m+9784pLnzS0YVLy8TpfYJZS6r2Qlme+IHInvVn0trhtoG1juNRYOEy8U2suOTYFyg1J+q0Y5NQJiRjs1WP6O3xkmueRLbxXHFAPi3ogELLFx8K3ADY5X7IEI40fcNoKp3YAz+p0lUrFhnqjJ9ScVqSxkJKOf0yq6vXEsZZL17GmqNhZRYue9m+f88CDbgh80RE5R57RFhcY8EIHnjn6wqhflh9m+JKaa6P8CM1B+ovEJeqhb4LX2uZiFHJRXGMwbAvkLJ+y0gVdUIcw30L5nK30iTIUCIZD4y9fnsCtYqdEMRGyPrPmzbUHcaPd0gqEzMtVIPahKa5pZKMFG1cvot1JdEY7\\\\\\\/Tckk2hXHmQyJ9jFB1U03n9QGZq37\\\\\\\/7mYUyTotegT5F4bveWHKqvDZCLP+b+rsOEFlIFDKqGGPClmg\\\\\\\/ZG89ilKgB4pB32ouPCn7LPLWQHXGdZLkZPiVdpiWmhZSNLB4qL9GqAkvo855r1wxZF73h9mYdTsIb54q7AvBJr3VbP56mz0cVhvaSiLVg7nhkIdxoDhcbahndzoeYNXnhr1YbHtDMhrgGr\\\\\\\/RnWWH+0Ll\\\\\\\/eyG5tUthkoD+c0hUd4i4xZ7wOdw0Vx6gIfzjggNkCeMMsyKutyB72bUwXQcZE+22wI9YHrPZWtlAH1xzeeupIOweVoqWpBcK1l9mClW6CIs6f\\\\\\\/13zOvrAVcCCqYRCug7rNcdC\\\\\\\/8YUm\\\\\\\/5ZWWc4GtgmH\\\\\\\/FEUtSX9BD+ChDLlXgVRQVMuX8FjbBE5vRkd2xBGRwK3VcH2HaEy7tPyiTxyPlq886fpV8Tl6juE\\\\\\\/f0otY8Pz9XvbfjYI0o7A64otKrlkJfyzK479AgkUe2hPoqJ\\\\\\\/qnAOtC5lOTNSm2Uqu1ACV1G8t7gieCsqwnJ03PVZPXHWmfHPJD4UHavpQYef9r9weIqDf9RkTxRJkABQ3ndZqUmXzh\\\\\\\/NmVC79eNzL+KEIucDNnYXwvIL5zChbRhPf9Vz8A3WXMnpvVOP9OctJl4fR6LgXvo\\\\\\\/EPe2t0lED3hv1hQd86qnG2SiSrs5JKoPgu\\\\\\\/eySSLYu3ta\\\\\\\/rwFKtcIkPb7D3gAiOL1uQSORhVIDl2c\\\\\\\/bdqZHPEL8z6MW9kwnTde2nrVLcDTv6nmox3JwBB4jBAsI1mXhN8Xn+nuvHb+6SiNKJ1L5+kY\\\\\\\/CYXi8qwMwHzy4USXhTK7lrSCMswZbL7V9nFFTlAbrDAoE94GJ7qgCffksRBuS\\\\\\\/z4T9bgyL41nM9TDt3KqPWv1VRUILZvCdLhC3IZSxAkCoIoKfiro9ET1LPORkP3evwRrQ3xEhnLOovVh5KHilASkFVd+a3mFNjgbIsYbVDsa2mGc9G1NTtJ3jFAdRY2yaFFhtwvK+jTbR0ZVhDGzGU\\\\\\\/lC7LS19p8BD0xMo0zM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022eBfbpb94z924\\\\\\\/r1EbuFuUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225ec87a79985e2db5301a6af85df11308da63342a253286c5ce340c9dd2eb3c1d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459267 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/453" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bZlxrtdYSdyzolqgUIV-ZDYm_CoXfEmT2Fnx9xoAnRk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/453" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jk53K925JsdEQsZYefPjFfrHJbnIVtNBMR9BcW1LDPg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/4/449" title="FASTER MT: Isolation of Pure Populations of a and α Ascospores from Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/4/469" title="Conserved Motifs and Prediction of Regulatory Modules in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459267" id="node459267" data-pisa="ggg;2/4/453" data-pisa-master="ggg;2/4/453" data-apath="/ggg/2/4/453.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting the Fission Yeast Protein Interaction Network</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vera Pancaldi</span>, <span class="highwire-citation-author" data-delta="1">Ömer S. Saraç</span>, <span class="highwire-citation-author" data-delta="2">Charalampos Rallis</span>, <span class="highwire-citation-author" data-delta="3">Janel R. McLean</span>, <span class="highwire-citation-author" data-delta="4">Martin Převorovský</span>, <span class="highwire-citation-author" data-delta="5">Kathleen Gould</span>, <span class="highwire-citation-author" data-delta="6">Andreas Beyer</span> and <span class="highwire-citation-author" data-delta="7">Jürg Bähler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">453-467;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001560 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459267" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vera Pancaldi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vera%2BPancaldi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pancaldi%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVera%2BPancaldi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:v.pancaldi@ucl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">v.pancaldi@ucl.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ömer S. Saraç </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96mer%2BS.%2BSara%C3%A7%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sara%C3%A7%20%C3%96S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596mer%2BS.%2BSara%25C3%25A7%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charalampos Rallis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charalampos%2BRallis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rallis%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharalampos%2BRallis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Janel R. McLean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Janel%2BR.%2BMcLean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLean%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJanel%2BR.%2BMcLean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin Převorovský </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BP%C5%99evorovsk%C3%BD%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C5%99evorovsk%C3%BD%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BP%25C5%2599evorovsk%25C3%25BD%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kathleen Gould </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BGould%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gould%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BGould%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Andreas Beyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BBeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BBeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jürg Bähler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%BCrg%2BB%C3%A4hler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%A4hler%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25BCrg%2BB%25C3%25A4hler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/4/453" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459267" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459267/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/453.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459267" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459267/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/453.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459267" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459267/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/4/453.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459267" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459267/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/4/453.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459267" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459267/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">A systems-level understanding of biological processes and information flow requires the mapping of cellular component interactions, among which protein–protein interactions are particularly important. Fission yeast (<em>Schizosaccharomyces pombe</em>) is a valuable model organism for which no systematic protein-interaction data are available. We exploited gene and protein properties, global genome regulation datasets, and conservation of interactions between budding and fission yeast to predict fission yeast protein interactions <em>in silico</em>. We have extensively tested our method in three ways: first, by predicting with 70–80% accuracy a selected high-confidence test set; second, by recapitulating interactions between members of the well-characterized SAGA co-activator complex; and third, by verifying predicted interactions of the Cbf11 transcription factor using mass spectrometry of TAP-purified protein complexes. Given the importance of the pathway in cell physiology and human disease, we explore the predicted sub-networks centered on the Tor1/2 kinases. Moreover, we predict the histidine kinases Mak1/2/3 to be vital hubs in the fission yeast stress response network, and we suggest interactors of argonaute 1, the principal component of the siRNA-mediated gene silencing pathway, lost in budding yeast but preserved in <em>S. pombe</em>. Of the new high-quality interactions that were discovered after we started this work, 73% were found in our predictions. Even though any predicted interactome is imperfect, the protein network presented here can provide a valuable basis to explore biological processes and to guide wet-lab experiments in fission yeast and beyond. Our predicted protein interactions are freely available through PInt, an online resource on our website (<a href="http://www.bahlerlab.info/PInt">www.bahlerlab.info/PInt</a>).</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cbf11" class="hw-term hw-article-keyword hw-article-keyword-cbf11" rel="nofollow">Cbf11</a></li><li class="kwd"><a href="/keyword/tor" class="hw-term hw-article-keyword hw-article-keyword-tor" rel="nofollow">TOR</a></li><li class="kwd"><a href="/keyword/mak123" class="hw-term hw-article-keyword hw-article-keyword-mak1-2-3" rel="nofollow">Mak1/2/3</a></li><li class="kwd"><a href="/keyword/support-vector-machine" class="hw-term hw-article-keyword hw-article-keyword-support-vector-machine" rel="nofollow">support vector machine</a></li><li class="kwd"><a href="/keyword/random-forest" class="hw-term hw-article-keyword hw-article-keyword-random-forest" rel="nofollow">random forest</a></li></ul><p id="p-4">Progress in experimental techniques has uncovered highly interconnected protein networks. Protein interactions not only hold together protein complexes, the operational units within cells, but also mediate cellular information flow along signaling pathways and networks. Acquiring a global knowledge of protein interconnections is important for understanding biological processes and their disruption in disease.</p><p id="p-5">Several high-throughput approaches for experimental detection of physical protein interactions have been applied, such as affinity capture followed by mass spectrometry (MS) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gavin <em>et al.</em> 2002</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Krogan <em>et al.</em> 2006</a>), yeast two-hybrid (Y2H) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ito <em>et al.</em> 2001</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Uetz <em>et al.</em> 2000</a>), and kinase interaction assays (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Breitkreutz <em>et al.</em> 2010</a>). These techniques have now provided protein interactomes for several organisms (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Formstecher <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Li <em>et al.</em> 2004</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Rual <em>et al.</em> 2005</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Stelzl <em>et al.</em> 2005</a>). Many additional protein interactions are known from various small-scale experiments, such as co-immunoprecipitations or affinity capture, which provide us with data for species in which large-scale studies are not yet available. The budding yeast <em>Saccharomyces cerevisiae</em> is the model organism in which the largest portion of protein interactions has been studied experimentally, covering ∼56,363 unique interactions among 5786 proteins (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Stark <em>et al.</em> 2011</a>). Typical overlaps between different Y2H screens are around 20% (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Yu <em>et al.</em> 2008</a>), and similar for TAP-MS (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gavin <em>et al.</em> 2011</a>). In addition, the experimental exploration of the interactome is prone to some bias, because better-studied proteins are more likely to be used as baits in experiments (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Yu <em>et al.</em> 2008</a>) as well as because of technical biases. For example, a network distance estimation method applied to interaction datasets obtained with different techniques on different species produced clusters based on the interaction detection method rather than on the species (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fernandes <em>et al.</em> 2010</a>). A recent report has developed a confidence score for each experimentally verified interaction (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Braun <em>et al.</em> 2009</a>), the use of which could potentially mitigate some of these biases. However, paying attention to the quality of the data is not common practice. We are still far from a complete and reliable picture of the protein interactome, even for the intensely studied budding yeast.</p><p id="p-6">The fission yeast <em>Schizosaccharomyces pombe</em> is only remotely related to budding yeast and provides a powerful complementary model system. However, no high-throughput protein interaction screens have been performed in fission yeast, and relatively few small-scale experimental data on protein interactions are available: a total of 2719 unique interactions involving 1468 proteins have been reported so far (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Stark <em>et al.</em> 2011</a>). Orthology relationships have been used for mapping interactions measured in one species onto other species (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Jensen <em>et al.</em> 2009</a>). The considerable level of conservation at the protein complex level suggests that such inter-species mapping could successfully update our knowledge about protein binding in fission yeast based on data in budding yeast. However, such mapping will be restricted to network modules existing in both species. Hence, it is unavoidable that modules within this network that are different between the two organisms will remain incorrectly or incompletely characterized (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Roguev <em>et al.</em> 2004</a>).</p><p id="p-7">Assaying all possible protein interactions experimentally in several species, in multiple repeats and under different conditions remains an enormously expensive and time-consuming challenge. Hence, there is a need for developing approaches that can predict physical protein interactions based on some pre-existing experimental data. <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pitre <em>et al.</em> (2008)</a> have reviewed the main types of prediction approaches that show promising results, and <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Qi <em>et al.</em> (2006)</a> conducted a comparison of the available methods. Different approaches have used protein sequences (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Shen <em>et al.</em> 2007</a>), functions (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ben-Hur and Noble 2005</a>) or three-dimensional structures (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aloy and Russell 2002</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Burger and Van Nimwegen 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">De Bodt <em>et al.</em> 2009</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jansen <em>et al.</em> 2003</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Singh <em>et al.</em> 2010</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Wang <em>et al.</em> 2009a</a>). In this article, we suggest a supervised machine learning approach for the prediction of physical protein-protein interactions based on a collection of more than 100 different protein and gene features, using negative sets that preserve the degree of each protein as in the positive set. We present the first large-scale protein interaction network for fission yeast. Contrary to the existing fission yeast interactome available through STRING (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Szklarczyk <em>et al.</em> 2011</a>), we only consider physical protein interactions, and we do not assume all interactions in budding yeast to be conserved. We present experimental verification of some of our predictions, and we make the whole dataset available for further exploration on our website (<a href="http://www.bahlerlab.info/PInt">www.bahlerlab.info/PInt</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-2" class="subsection"><h3>Assembling the features</h3><p id="p-8">The machine learning approach suggested requires positive and negative training sets of interactions as well as a vector of features for each pair of proteins. The feature vector was assembled by concatenating the features of the first and second protein as well as the corresponding pair features. However, to ensure consistency in defining symmetric interactions (if A interacts with B, it follows that B interacts with A), we ordered the features such that the smaller value always came first. In a previous version, the two symmetric interactions were both included, but this slowed down the training procedure and did not lead to better results.</p></div><div id="sec-3" class="subsection"><h3>Assembling the training sets</h3><p id="p-9">We then used these features to classify any two proteins into either interacting or not interacting pairs. The starting point was a positive training set (list of known interacting pairs) and a negative training set (list of pairs believed not to interact).</p></div><div id="sec-4" class="subsection"><h3>Positive training set</h3><p id="p-10">In recent years, numerous interaction databases have attempted to summarize the knowledge on protein interactions in different species. We used data from BioGRID (<a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Stark <em>et al.</em> 2011</a>), which aims to integrate the information coming from different sources with a systematic evidence code. The main types of documented physical interactions are subdivided into two classes: large-scale experiments, usually less accurate but unbiased, and small-scale experiments, probably of higher quality but with much less coverage and obvious biases dictated by the higher biological interest of certain proteins. Of all the types of small-scale experiments included in BioGRID, we only considered the ones that aim to prove a physical interaction, thus including co-immunoprecipitation, capture experiments of different types, and complex reconstitution, but excluding co-localization. As far as the large-scale experiments are concerned, the most common experiments are affinity capture-MS and Y2H. Some of the interactions in databases are of dubious reproducibility, and especially, Y2H and capture methods are known to have different technical biases. We thus took a strict approach, including these interactions in our training sets only if they were verified by both Y2H and a capture technique, either large-scale, like affinity capture-MS, or small-scale, like affinity capture. This filtering reduced our training sets considerably, but it increased the quality of the training data. We chose to consider only orthologs from budding yeast as we assumed that complexes in the two species are highly conserved, which was shown to be particularly true between the two yeasts (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Dixon <em>et al.</em> 2008</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Roguev <em>et al.</em> 2008</a>). Only unique orthologs were considered, and orthology mapping was performed with a list of one-to-one orthologs curated by an expert (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Wood 2006</a>). A previous approach mapping interactions from more species led to worse results. The resulting high-quality positive training set consisted of 1097 interactions among 660 proteins.</p></div><div id="sec-5" class="subsection"><h3>Negative training set</h3><p id="p-11">A common strategy to generate pairs of non-interacting proteins is to choose proteins with different GO annotation or different localization. However, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ben-Hur and Noble (2006)</a> have shown that this introduces biases in the sets that can confound the algorithm, as there could be differences in proteins that are found in different cell compartments. Moreover, in species with few known interactions, we do not want to introduce any bias in the type of proteins for which we predict interactions. The commonly used alternative is to create random protein pairs, on the assumption that it is highly unlikely that randomly chosen pairs physically interact, given the space of possible interactions is of the order of the square of the number of proteins. However, a recent report has shown that the resulting randomized negative sets are not comparable to the positive sets, unless the randomization preserves the number of connections for each protein (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Yu <em>et al.</em> 2010</a>). For this reason, the construction of a degree-balanced negative set, which does not give more or less importance to the interactions involving different proteins, has been advocated (<a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Yu <em>et al.</em> 2010</a>). Hence, our negative set was constructed by shuffling the links between the proteins in the positive set while keeping the degree fixed, <em>i.e.</em> the number of connections for each protein. We also removed from the pool of possible negative pairs any interactions that have been experimentally shown at least once with any method in either of the two yeasts. The negative sets were generated using BRS-nonint (<a id="xref-ref-95-3" class="xref-bibr" href="#ref-95">Yu <em>et al.</em> 2010</a>).</p></div><div id="sec-6" class="subsection"><h3>Support Vector Machines and Random Forest</h3><p id="p-12">Using the features and training sets described above, we trained a support vector machine (SVM) and a random forest (RF). The features for all protein pairs were concatenated in feature vectors for the positive and negative training sets, and only pairs for which none of the features were missing were used for the classification. Training and testing of the SVM was performed using a radial basis function kernel with the e1071 package for R. Optimal parameters were chosen using the tune function in a cross-validation framework. The random forests were built using the package randomForest, and the cforest function from the party package was used for estimating the feature importance (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Strobl <em>et al.</em> 2008</a>).</p></div><div id="sec-7" class="subsection"><h3>Classification performance measures and ROC curves</h3><p id="p-13">The performance of the classification was estimated using standard measures. Once a specific threshold is chosen for the interaction score, we can define the following: true positives (TP) as the number of predicted interactions which are true interactions; true negatives (TN) as the number of false interactions correctly not predicted; false positives (FP) as the number of false interactions incorrectly predicted; and false negatives (FN) as the number of true interactions not predicted. Using these measures, we can define the following: specificity = TN/(TN+FP); sensitivity = TP/(TP+FN); accuracy = (TP+TN)/(TP+TN+FP+FN); and false-positive rate (FPR) = FP/N = FP/(FP+TN). Moreover, we used Receiver Operator Characteristic curves (ROC), Areas Under the Curve (AUC), areas under the partial curve (AUC50), and precision-recall curves as a threshold-independent measure of the performance, using the ROCR package for R. The main purpose of providing these measures for our classification was to be able to compare the RF and SVM methods.</p></div><div id="sec-8" class="subsection"><h3>Test sets</h3><p id="p-14">We constructed a positive test set by collecting high-confidence protein interactions in fission yeast. We found 204 pairs of proteins that were shown to interact both by Y2H and a different capture experiment in fission yeast (BioGRID). Any interactions involving proteins that appear in this high-confidence fission yeast interactome were eliminated from our training sets. We then proceeded to build negative test sets.</p><p id="p-15">The first negative test set was constructed making random pairs of the proteins from the positive set, eliminating all interactions documented at least once in budding or fission yeast and preserving the degree of each protein. This ensures that the positive and negative test sets contain the same proteins (none of them present in the training set) and have the same topological features.</p><p id="p-16">In a more realistic scenario, there should be a strong imbalance between the number of positive and negative interactions, so we proceeded to also generate a negative set to reproduce this bias. To construct this negative test set, we made random pairs of the proteins known to interact in budding or fission yeast paired at random, maintaining the same degree for each protein. This degree-balanced negative set comprises 32,482 pairs among ∼4500 genes, which, combined with the 204 positives, leads to a ratio between positives and total pairs of ∼0.006, the same as what we expect for the real interactome [12.5 million possible pairs; an upper estimate of between 60,000 and 80,000 true interactions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hart <em>et al.</em> 2006</a>)].</p></div><div id="sec-9" class="subsection"><h3>Prediction of SAGA, a large co-activator complex</h3><p id="p-17">The Spt-Ada-Gcn5 acetyltransferase (SAGA) complex is involved in regulating RNA polymerase II transcription through chromatin remodeling and histone modification. It is one of the largest and best-annotated protein complexes in budding and fission yeast (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Helmlinger <em>et al.</em> 2008</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Koutelou <em>et al.</em> 2010</a>) and, as such, suitable for testing our method. To test our method using the SAGA complex, we constructed a test set of all possible pairs between the fission yeast SAGA components and an equal number of control proteins picked at random. The positive and degree-balanced negative training sets were constructed so that they would not include any of the test proteins. We then predicted all possible interactions among the SAGA proteins and the control proteins. We also predicted interactions for the SAGA proteins with all fission yeast proteins and compared the overlap between the predicted interactions and the real list of SAGA interactions.</p></div><div id="sec-10" class="subsection"><h3>Cbf11 predictions</h3><p id="p-18">As there were no interactions for Cbf11 in the training set, the original training pairs were used to make predictions about Cbf11 interactions with all the 4920 fission yeast proteins for which data were available. We predicted interactions using both SVM and RF, applying a score threshold of 0.5 to separate positives from negatives. We then tested these predictions against the results of an affinity capture-MS experiment that used TAP-tagged Cbf11 as bait for the affinity purification. We also tested the enrichment of our predicted interaction partners for experimentally verified interactors and compared this overlap with expected overlaps of randomized lists of interactions.</p><p id="p-19">The predicted and experimentally verified interactors were checked against a list of known complexes in fission yeast to examine the coverage of the different complexes within the experimentally verified associated proteins. Gene Ontology (GO) enrichment analysis was performed using GO TermFinder (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boyle <em>et al.</em> 2004</a>).</p></div><div id="sec-11" class="subsection"><h3>Protein purification and mass spectrometry</h3><p id="p-20">Fission yeast strain (JB795) expressing a C-terminally TAP-tagged version of Cbf11 from its endogenous chromosomal locus was grown to mid-exponential phase in 4× concentrated YES medium (Formedium), washed twice with water, and snap-frozen in liquid nitrogen. Cells were broken in the RM 200 mortar (Retch), and lysates were clarified by ultracentrifugation in the Sorvall Discovery 90SE ultracentrifuge (Sorvall T-647.5 rotor) at 35,000 rpm for 90 min. Due to low abundance of the bait protein, only the first purification step of the standard TAP protocol (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Rigaut <em>et al.</em> 1999</a>) was employed. Briefly, the tagged protein with its interactors were captured on IgG-coated magnetic beads (1:1 pan mouse IgG and M-280 sheep anti-rabbit IgG dynalbeads, Invitrogen), washed three times with NP-40 lysis buffer, three times with TEV cleavage buffer, and eluted in 2× 500 µl elution buffer (0.5 M NH<sub>4</sub>OH, 0.5 mM EDTA). The purified proteins were then TCA-precipitated and digested with trypsin (Promega). The resultant peptides were subjected to two-dimensional LC-MS/MS analysis on a Thermo LTQ as previously detailed (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mcdonald <em>et al.</em> 2002</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Roberts-Galbraith <em>et al.</em> 2009</a>). Thermo RAW files were converted to MZML files using Scansifter (software developed at the Vanderbilt University Medical Center). Spectra with fewer than 20 peaks were excluded from our analysis. The <em>S. pombe</em> database (<a href="http://www.sanger.ac.uk">http://www.sanger.ac.uk</a>, October 2009) was searched with the MyriMatch algorithm (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Tabb <em>et al.</em> 2007</a>) v1.6.33 on a high-performance computing cluster (Advanced Computing Center for Research and Education at Vanderbilt University). We added contaminant proteins (<em>e.g.</em>, keratin, IgG) to the complete <em>S. pombe</em> database and reversed and concatenated all sequences to allow estimation of FDRs (10,186 entries). MyriMatch parameters were as follows: strict tryptic cleavage; modification of methionine (oxidation, dynamic modification, +16 Da), S/T/Y (phosphorylation, dynamic modification, +80 Da), and cysteine (carboxamidomethylation, static modification, +57 Da) was allowed; precursor ions were required to be within <em>m/z</em> 0.6 of the peptide monoisotopic mass; fragment ions were required to fall within <em>m/z</em> 0.5 of the expected monoisotopic mass. IDPicker (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ma <em>et al.</em> 2009</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Zhang <em>et al.</em> 2007</a>) v2.5.180 was used to filter peptide matches with the following parameters: maximum FDR per result, 0.05; maximum ambiguous IDs per result, 2; minimum peptide length per result, 5; minimum distinct peptides per protein, 2; minimum additional peptides per protein group, 1; minimum number of spectra per protein, 2; indistinct modifications M 15.994 Da, C 57.05 Da; and distinct modifications S/T/Y 80 Da. IDPicker results were processed in Excel (Microsoft) to generate protein identification lists. Proteins identified as contaminants (<em>e.g.</em> keratin) were removed from the final protein list.</p></div><div id="sec-12" class="subsection"><h3>Statistics of degree correlation</h3><p id="p-21">Wilcoxon rank-sum tests and Spearman correlation tests were performed using R to assess the correlation of the degree of a node in a specific network with any other feature used in this work.</p></div><div id="sec-13" class="subsection"><h3>Genome-wide interaction predictions</h3><p id="p-22">Protein interactions among all of the fission yeast proteins were predicted. Each protein was assigned a GO superslim term. We used these terms in the following order to ensure that the most specific term was used: GO:0007005, mitochondrion organization and metabolism; GO:0007165, signal transduction; GO:0006350, transcription; GO:0016070, RNA metabolism; GO:000412, protein biosynthesis; GO:0006259, DNA metabolism; GO:0007049, cell cycle; GO:0006810, transport; GO:0044238, primary metabolism; and GO: 0008150, biological process.</p></div></div><div class="section results" id="sec-14"><h2 class="">RESULTS</h2><div id="sec-15" class="subsection"><h3>Classification approach to predict protein interactions</h3><p id="p-23">We interpreted the protein prediction problem as a classification task on protein pairs that can be interacting or not. We required a training set to perform the classification that was composed of known interactions (positive training set) and a corresponding set of supposedly non-interacting pairs (negative training set), as well as a list of features. We constructed both a support vector machine (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cortes and Vapnik 1995</a>) and a random forest (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Breiman 2001</a>) with the same input data and compared the results. These methods use an assembly of features from protein pairs to learn the difference between interacting and non-interacting pairs, and then score the new interaction predictions.</p></div><div id="sec-16" class="subsection"><h3>Features used</h3><p id="p-24">As a basis for the machine learning approaches, we compiled two types of features, describing either the fission yeast proteins themselves (<em>e.g.</em> sequence-based features) or their interactions (<em>e.g.</em> co-expression). An overview of all features is presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS1.xls">File S1</a>. These features include physical position on the chromosome, physical properties of the protein (such as length, mass, and chemical properties), functional category information (through GO Superslim classification), as well as protein pair information, such as expression correlation over hundreds of different conditions, co-localization, distance along the chromosome, and the presence of genetic interactions (see <em>Materials and Methods</em> for more details).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461356/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461356/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of features used in the prediction of the interactions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Model cross-validation</h3><p id="p-27">Both the SVM and RF classification methods output a probability for the interaction of any tested pair, which we interpreted as a score. Cross-validation is useful to assess the performance of the predictor on a subset of the data while the rest of it is used for training. Two-fold cross-validation, which uses half of the data for training and half for testing, was performed 10 times for both the SVM and the RF models. The average accuracy of the SVM was 0.76, with an AUC of 0.81 (compared with 0.5 expected by chance and 1 for perfect classification). The results for RF in this cross-validation test were marginally worse (accuracy, 0.73; AUC, 0.79; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS2.xls">File S2</a>). Considering predictions with a score above 0.5, the specificity (0.80 for SVM and 0.74 for RF) was better than the sensitivity (0.71 for both SVM and RF), suggesting that the false negatives were higher than the false positives at this threshold. We can see from the ROC curves (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>) and precision-recall curves (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, C and D</a>) that the results were robust with respect to different splitting of the data into training and test sets. We conclude that our machine learning approach manages to successfully predict many protein-protein interactions based on the features used.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/453/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model cross-validation and model testing on an independent test set. (A, B) ROC curves of 10 repeats with 2-fold cross-validation tests performed with SVM and RF, respectively. ROC curves show how the relationship between the true-and false-positive rate changes as a function of the probability threshold. If we are interested in making a few predictions with the smallest possible chance for errors, we should consider a high threshold. (C, D) Precision-recall curves corresponding to A and B. Precision-recall curves show the rate of correctly predicted interactions vs. the rate of predicted interactions. (E, F) Model testing using an independent fission yeast data set and an equally large degree balanced negative test set with the same proteins. (G, H) ROC and precision-recall curves for a second test set composed of the 204 high-confidence interactions and &#x223C;32,000 random negative pairs, which are assumed not to be interacting. In this test, the ratio of positives to total pairs is similar to what we expect in predicting the whole interactome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670881907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Model cross-validation and model testing on an independent test set. (A, B) ROC curves of 10 repeats with 2-fold cross-validation tests performed with SVM and RF, respectively. ROC curves show how the relationship between the true-and false-positive rate changes as a function of the probability threshold. If we are interested in making a few predictions with the smallest possible chance for errors, we should consider a high threshold. (C, D) Precision-recall curves corresponding to A and B. Precision-recall curves show the rate of correctly predicted interactions &lt;em&gt;vs.&lt;/em&gt; the rate of predicted interactions. (E, F) Model testing using an independent fission yeast data set and an equally large degree balanced negative test set with the same proteins. (G, H) ROC and precision-recall curves for a second test set composed of the 204 high-confidence interactions and &#x223C;32,000 random negative pairs, which are assumed not to be interacting. In this test, the ratio of positives to total pairs is similar to what we expect in predicting the whole interactome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/453/F1.medium.gif" width="237" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/4/453/F1.medium.gif" width="237" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/453/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/453/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461342" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Model cross-validation and model testing on an independent test set. (A, B) ROC curves of 10 repeats with 2-fold cross-validation tests performed with SVM and RF, respectively. ROC curves show how the relationship between the true-and false-positive rate changes as a function of the probability threshold. If we are interested in making a few predictions with the smallest possible chance for errors, we should consider a high threshold. (C, D) Precision-recall curves corresponding to A and B. Precision-recall curves show the rate of correctly predicted interactions <em>vs.</em> the rate of predicted interactions. (E, F) Model testing using an independent fission yeast data set and an equally large degree balanced negative test set with the same proteins. (G, H) ROC and precision-recall curves for a second test set composed of the 204 high-confidence interactions and ∼32,000 random negative pairs, which are assumed not to be interacting. In this test, the ratio of positives to total pairs is similar to what we expect in predicting the whole interactome.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Model testing on proteins not present in training sets</h3><p id="p-29">Cross-validation results cannot estimate the power of the method to generalize to unseen data; hence, it is necessary to test it on a completely independent set of interactions. We report the classification performance on the test set with a one-to-one ratio between positive and negative pairs.</p><p id="p-30"><a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> shows the ROC curve for SVM compared with RF (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1E</a>) with the corresponding precision-recall curve (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1F</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS2.xls">File S2</a> summarizes the statistics of the tests performed. RF and SVM achieved comparable results. In these tests, SVM showed higher accuracy (0.71) and higher AUC50 (0.291) compared with RF (0.68 and 0.286, respectively). The specificities ranged from 0.66 to 0.81, and the sensitivities ranged from 0.63 to 0.73. Generally, RF tended to produce fewer false positives, with the consequence of missing more of the true interactions compared with SVM.</p></div><div id="sec-19" class="subsection"><h3>Test with imbalance between positive and negative pairs</h3><p id="p-31">Protein interaction networks are generally sparse; therefore, we expect a small fraction of real interactions among all possible protein pairs in fission yeast. To account for the excess of negative interactions, we also tested the performance of our method using a negative test set much larger than the positive test set. The results showed that, even with a large imbalance between the number of true and false interactions in the test sets, our methods can successfully limit the false-positive rate while correctly identifying the true positives. Although both the RF and SVM methods showed comparable performance (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, G and H</a>), RF achieved a better accuracy (0.81) compared with SVM (0.75) and slightly better AUC50 (0.327 for RF and 0.32 for SVM).</p></div><div id="sec-20" class="subsection"><h3>Feature importance</h3><p id="p-32">We estimated the importance of each feature using a method that was proven to correctly take into account redundancy between the features and a mixture of feature types (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Strobl <em>et al.</em> 2008</a>). The features that are most influential in making the predictions are expression correlation; various GO terms, mostly related to metabolism and transport; and the length, abundance, and localization of proteins (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Similar results were obtained by measuring the mean decrease in accuracy as the trees were being built (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS3.xls">File S3</a>). To quantify the predictive power of each feature alone, we conducted <em>t</em>-tests and Fisher’s tests estimating the significance of the differences between feature values in the positive and negative training sets. This analysis generally confirms our initial results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS3.xls">File S3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/453/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Importance of features in the RF classifier. Only the most important features are shown. Expression correlation and GO functional categories are the most important features, followed by protein length, mRNA levels, and protein co-localization (File S3; see File S1 for explanation of feature names). These values are the average of the importance of each feature in 10 realizations of the 2-fold cross-validation test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670881907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Importance of features in the RF classifier. Only the most important features are shown. Expression correlation and GO functional categories are the most important features, followed by protein length, mRNA levels, and protein co-localization (File S3; see File S1 for explanation of feature names). These values are the average of the importance of each feature in 10 realizations of the 2-fold cross-validation test.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/453/F2.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/4/453/F2.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/453/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/453/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461360" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Importance of features in the RF classifier. Only the most important features are shown. Expression correlation and GO functional categories are the most important features, followed by protein length, mRNA levels, and protein co-localization (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS3.xls">File S3</a>; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS1.xls">File S1</a> for explanation of feature names). These values are the average of the importance of each feature in 10 realizations of the 2-fold cross-validation test.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Predictions of SAGA complex interactions</h3><p id="p-34">Next we focused on the prediction of interactors of the SAGA complex. At a score threshold of 0.5, we only failed to predict two interactions with either SVM or RF: one between Ubp8 and Sus1 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cuenca-Bono <em>et al.</em> 2010</a>), and one between Ubp8 and Tra1. The consistency of these negative results between SVM and RF raises the possibility that these SAGA subunits are part of two different sub-complexes or modules with specialized functions. This possibility is in line with findings in budding yeast, which place <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004836">Ubp8</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005968">Sgf11</a> in a discrete functional module within SAGA (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ingvarsdottir <em>et al.</em> 2005</a>). More recently, an analysis of deletion mutants for a few components of the fission yeast SAGA complex has also highlighted the presence of different modules within the complex. Intriguingly, we recover these modules to some extent when looking at the similarity of interaction profiles of the subunits. For example, we see the separate role of Ubp8 and Tra1, and the close link of Spt7 and Hfi1 and Sgf73 and Sgf29 (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Helmlinger <em>et al.</em> 2011</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FigureS1.pdf">Figure S1</a>). At a score threshold of 0.9, SVM predicted 92 interactions, 86 of which were part of the 171 total known SAGA interactions, whereas no interactions were predicted with RF.</p><p id="p-35">RF scores generally tended to be lower than SVM scores so that choosing a specific score threshold led to a different number of predictions in the two methods. However, a correlation between the ranking of interactions according to SVM and RF was evident (Spearman r = 0.88, <em>P</em> &lt; 10<sup>−16</sup>). To further check the specificity of our predictions, we calculated precision and recall curves for identifying the SAGA interactions from the total of the possible interactions of SAGA units with other proteins (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-36">We predicted 132 and 126 additional interactions with SVM and RF, respectively, for which at least one of the proteins was in the control set. Notably, 105 of these interactions involving non-SAGA proteins were predicted both by SVM and RF, 28 of them with SVM scores &gt; 0.8. The control proteins predicted to interact with SAGA components were characterized by involvement in chromatin and DNA damage repair (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS4.xls">File S4</a>). For example, many of the highest scoring interactions were between known SAGA proteins and Phf1, a histone H3-K9 demethylase. We therefore hypothesize that some of these predicted interactions could in fact be true and thus point to potential new SAGA components or to proteins transiently interacting with the SAGA complex. For example, we have found some evidence for Phf1, Cdc17, and Pku80 to be interacting with SAGA in other organisms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS9.pdf">File S9</a>). In conclusion, our classification approaches succeeded in recapitulating most of the SAGA complex, and they also suggested potential additional proteins that plausibly could associate with SAGA.</p></div><div id="sec-22" class="subsection"><h3>Prediction of Cbf11 interactors</h3><p id="p-37">Cbf11 is a fission yeast member of the <span class="underline">C</span>BF1/RBP-Jκ/<span class="underline">S</span>uppressor of Hairless/<span class="underline">L</span>ag-1 (CSL) family of transcription factors. In metazoans, the CSL proteins serve as effectors of the Notch signaling pathway, which is critical for development. This protein family has been lost in budding yeast, and it seems to exert pleiotropic functions in fission yeast (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Převorovský <em>et al.</em> 2009</a>).</p><p id="p-38">A purification experiment identified ∼300 potential components of the Cbf11 complex. The performance of the prediction compared with experimental results is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> with ROC curves, precision-recall curves, and Venn diagrams for SVM and RF. It is possible that the experiment would itself have false negatives, as the experiments carried out in a specific experimental condition do not necessarily identify all the partners of a protein, especially the less stable ones. Notably, a significant correlation was found between the SVM prediction score and the number of peptides identified in the affinity capture-MS experiment (Spearman r = 0.25, <em>P</em> = 0.007), suggesting that our confidence scores could also reflect the strength of the experimental evidence. The scores obtained with RF, however, did not correlate with the peptide count (<em>P</em> &gt; 0.5).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/453/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimation of performance of the predictions for the interaction partners of Cbf11. (A) ROC curves for RF (upper curve) and SVM (lower curve) obtained by comparing the predicted Cbf11 interactions with the experimentally verified targets. (B) Corresponding precision recall curves. (C) Distributions of the interactors predicted by RF (red line) and SVM (blue line) and of the corresponding 100 sets of genes picked at random from the fission yeast proteome (RF, pink line; SVM, light-blue line). (D) Overlap between the predicted interactors and the &#x223C;300 experimentally verified targets (solid lines, RF, red; SVM, blue) and overlaps for each of the 100 random sets (RF, pink squares; SVM light-blue triangles). Lower panel: Venn diagrams showing overlaps between experimentally identified targets and the predictions, SVM (E), RF (F), and overlap of RF and SVM (G)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670881907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimation of performance of the predictions for the interaction partners of Cbf11. (A) ROC curves for RF (upper curve) and SVM (lower curve) obtained by comparing the predicted Cbf11 interactions with the experimentally verified targets. (B) Corresponding precision recall curves. (C) Distributions of the interactors predicted by RF (red line) and SVM (blue line) and of the corresponding 100 sets of genes picked at random from the fission yeast proteome (RF, pink line; SVM, light-blue line). (D) Overlap between the predicted interactors and the &#x223C;300 experimentally verified targets (solid lines, RF, red; SVM, blue) and overlaps for each of the 100 random sets (RF, pink squares; SVM light-blue triangles). Lower panel: Venn diagrams showing overlaps between experimentally identified targets and the predictions, SVM (E), RF (F), and overlap of RF and SVM (G).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/453/F3.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/4/453/F3.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/453/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/453/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461417" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Estimation of performance of the predictions for the interaction partners of Cbf11. (A) ROC curves for RF (upper curve) and SVM (lower curve) obtained by comparing the predicted Cbf11 interactions with the experimentally verified targets. (B) Corresponding precision recall curves. (C) Distributions of the interactors predicted by RF (red line) and SVM (blue line) and of the corresponding 100 sets of genes picked at random from the fission yeast proteome (RF, pink line; SVM, light-blue line). (D) Overlap between the predicted interactors and the ∼300 experimentally verified targets (solid lines, RF, red; SVM, blue) and overlaps for each of the 100 random sets (RF, pink squares; SVM light-blue triangles). Lower panel: Venn diagrams showing overlaps between experimentally identified targets and the predictions, SVM (E), RF (F), and overlap of RF and SVM (G).</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">SVM and RF contributed 15 and 10 unique predictions, respectively, that were confirmed experimentally. Thus, the two methods were somewhat complementary and, if used together, may provide better coverage of true predictions. On the other hand, using the overlap of predictions from both SVM and RF provides a more conservative and, hence, more reliable list of protein interactions that could be used as a starting point for further investigations.</p><p id="p-41">The Cbf11 interactors predicted were significantly enriched for the experimentally determined targets, both in the case of SVM (Fisher’s exact test, <em>P</em> = 10<sup>−14</sup>), RF (Fisher’s <em>P</em> = 10<sup>−28</sup>), and in the overlap of the two methods (Fisher’s <em>P</em> = 10<sup>−28</sup>). So far, there are no known genetic interactions for Cbf11 and no functional interactions due to the protein not being conserved in budding yeast. For this reason, our method for predicting its physical association partners can only be compared with selecting proteins at random from the whole genome. The results of this test are shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, C and D</a>, revealing that the scores of the predicted targets are much higher than scores for other equally large sets of genes picked at random (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). On closer analysis, two peaks can be identified in the score distribution of the predicted set, suggesting that by considering only scores above 0.7, we are likely to eliminate many of the false positives. Our predictions offer almost three times as many verified targets as random lists of genes. The presence of a peak of high scores in the random sets’ distribution (continuous curves in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>) and the overlaps between random sets and the verified targets (circles in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>) are consistent evidence for the likely presence of a small number of real interactors or false positives in the random sets, which can be expected.</p><p id="p-42">We performed a GO term enrichment analysis on the predicted Cbf11 interactors as well as on the experimentally identified interactors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS4.xls">File S4</a>). Both these lists were significantly enriched for transcriptional control, RNA processing, and chromatin remodeling; these enrichments were particularly significant among the 97 proteins that were both predicted by the two techniques and experimentally confirmed as Cbf11 interactors. The terms that we found in the enrichment are of finer level compared with the ones used as features in the training, so this enrichment is not trivial.</p><p id="p-43">The proteins that are not predicted to interact but are experimentally found to co-purify with Cbf11 were only weakly enriched for organelle and chromosome organization and for metabolic proteins, suggesting that some of these interactions could be false positives of the affinity capture-MS method or background interactions. Consistent with the latter hypothesis, more than 20% of these interactions were with proteins that could be considered non-specific background (<em>i.e.</em> identified in no TAP tag negative controls or in most other unrelated TAP/LC-MS/MS analyses performed in our laboratory; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS4.xls">File S4</a>). Notably, there were also 398 interactions predicted with both SVM and RF, but not confirmed experimentally, which showed strong enrichment for the same GO terms that characterize the experimentally verified interactors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS4.xls">File S4</a>). These genes showed significantly lower mRNA levels (Wilcoxon rank-sum test, <em>P</em> = 10<sup>−5</sup>) and shorter mRNA half-lives (<em>P</em> = 10<sup>−5</sup>), raising the possibility that they escaped the purification steps or were below the detection sensitivity of the affinity capture-MS approach. We conclude that our list of predicted targets contains approximately one third of the experimentally verified Cbf11 interactors and is not biased by protein levels that could affect the affinity capture-MS data.</p><p id="p-44">Moreover, our list of predictions contains whole complexes that could interact through some of their units with Cbf11. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the predicted Cbf11 interactors that are annotated to be part of a complex in fission yeast. In some cases, like the SAGA complex, the vast majority of the subunits of the complex are predicted to interact. In others, like the pre-replication complex, only a subset is predicted. Looking at these complexes shows that, within the set of predicted interactions, clear biological modules can be identified. Using a relatively low cut-off of 0.5 (applied to SVM and RF) leads to as many as 951 predicted interactors, raising the possibility that these complexes are just predicted by chance. However, even raising the SVM threshold to 0.7 (as suggested by the score distributions of RF and SVM shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>), we observe many components of the same complexes among the predicted interactors. Moreover, these complexes were also present in the list of experimentally verified Cbf11 interactors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS4.xls">File S4</a>). The general theme of chromatin remodeling among Cbf11 interactors, with an emphasis on histone modification and silencing, suggests a role for this transcription factor in epigenetic processes, which could explain the wide range of phenotypes that have been associated with it.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/453/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Known interactions among predicted Cbf11 interactors. The proteins shown are the subset of the predicted Cbf11 interactors that are annotated to be part of a complex in fission yeast. Many complexes are predicted to interact with Cbf11. In some cases, almost all the subunits are predicted interactors (File S4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670881907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Known interactions among predicted Cbf11 interactors. The proteins shown are the subset of the predicted Cbf11 interactors that are annotated to be part of a complex in fission yeast. Many complexes are predicted to interact with Cbf11. In some cases, almost all the subunits are predicted interactors (File S4).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/453/F4.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/4/453/F4.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/453/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/453/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461423" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">Known interactions among predicted Cbf11 interactors. The proteins shown are the subset of the predicted Cbf11 interactors that are annotated to be part of a complex in fission yeast. Many complexes are predicted to interact with Cbf11. In some cases, almost all the subunits are predicted interactors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS4.xls">File S4</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Validation of other lower-confidence complex interactions from the literature</h3><p id="p-46">Using GO annotations, a curated list of fission yeast complexes can be compiled (GO complex interactions). Assuming that all members of a complex interact with each other and not with members of other complexes, we constructed a set of 28,771 interactions and 1,374,879 non-interactions. Using a threshold of 0.5 for both SVM and RF, we confirmed 14,720 of the interactions and 1,191,506 of the non-interactions, with a precision of 0.07 and an accuracy of ∼0.86 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FigureS2.pdf">Figure S2</a>). Importantly, the assumption that there are no interactions between units of these different complexes and that all complex units interact directly is very likely wrong, showing that these estimates are only very rough approximations of the real performance.</p></div><div id="sec-24" class="subsection"><h3>Validation of new high-confidence interactions</h3><p id="p-47">Looking at interactions from the GO complex list described above, which are also verified in a study for interaction of chromatin-associated proteins (<a id="xref-ref-83-4" class="xref-bibr" href="#ref-83">Shevchenko <em>et al.</em> 2008</a>), we predicted 39 out of 58. Additionally, as of April 2011, 15 more pairs can be added to the list of 204 fission yeast interactions, with two different lines of evidence (capture and Y2H), which was used for our testing in 2010. Of these, 11 are predicted by both RF and SVM, and 5 have an SVM score higher than 0.8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS5.xls">File S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-25" class="subsection"><h3>Node degree is correlated with protein length and not abundance</h3><p id="p-48">We then checked whether particular types of proteins were predicted to have many interactors. To answer this question, we examined the tendency of proteins with specific feature values to have many interacting partners in three different networks: (1) a network of protein interactions in fission yeast from databases (including the ones mapped from budding yeast); (2) the filtered subset of this known network used as our positive training set (two independent lines of evidence); and (3) the network of predicted interactions with scores higher than 0.9. In the protein interaction network built from databases, we find that proteins with more connections have a higher relative polymerase II occupancy (Spearman r = 0.075, <em>P</em> = 10<sup>−5</sup>), suggesting that they are relatively more abundant. However, this is not the case in either of the other networks. It is hard to determine whether this trend is biologically relevant. The correlation of the degree with the protein level is to be expected in experiments, where capture techniques are more sensitive to abundant proteins (especially with high-throughput methods such as mass spectrometry), but it was not found in our predictions or in a high-quality dataset. This finding is, therefore, consistent with a bias in the experimental data. Interestingly, highly connected proteins tended to be longer, both in the known network (Spearman r = 0.09, <em>P</em> = 10<sup>−7</sup>) and in the predicted network (Spearman r = 0.16, <em>P</em> = 10<sup>−20</sup>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS6.xls">File S6</a>), which could reflect the presence of more interaction domains on longer proteins.</p></div><div id="sec-26" class="subsection"><h3>Global prediction of the protein interaction network in fission yeast</h3><p id="p-49">Having assessed the accuracy of our prediction method, we calculated probability scores for all possible fission yeast protein interactions. The almost 5000 proteins can potentially interact in over 12.46 million possible pairs. We performed the predictions with both SVM and RF and used the overlap as a conservative estimate of the fission yeast protein interactome. We chose a score threshold (0.87) that resulted in a number of interactions comparable to the ∼60,000 interactions known for budding yeast (<a id="xref-ref-79-5" class="xref-bibr" href="#ref-79">Stark <em>et al.</em> 2011</a>) and within other estimates (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hart <em>et al.</em> 2006</a>). As we have seen while examining the predictions for Cbf11 interactions against experimental data, the score assigned by the SVM to each pair appears to be a reliable measure of the strength of the interaction and an indication of whether it is direct. The network of all interactions passing our filtering criteria showed coherent modules characterized by GO terms (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This finding may reflect that GO terms are features in the classification, but we still observed these modules to some extent when the GO terms were omitted from the training (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FigureS4.pdf">Figure S4</a>). This result indicated the presence of redundant information in the different features used.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/453/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted network with interactions confirmed by RF and SVM. The network includes 3438 proteins and 37,325 interactions for clarity. The Cytoscape organic layout was used for visualization (Smoot et al. 2011). The emerging clusters reflect highly connected portions of the network. These clusters coincide with the GO superslim functional categories of the proteins and are color-coded as indicated (Ashburner et al. 2000)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670881907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predicted network with interactions confirmed by RF and SVM. The network includes 3438 proteins and 37,325 interactions for clarity. The Cytoscape organic layout was used for visualization (Smoot &lt;em&gt;et al.&lt;/em&gt; 2011). The emerging clusters reflect highly connected portions of the network. These clusters coincide with the GO superslim functional categories of the proteins and are color-coded as indicated (Ashburner &lt;em&gt;et al.&lt;/em&gt; 2000).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/453/F5.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/4/453/F5.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/453/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/453/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461425" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">Predicted network with interactions confirmed by RF and SVM. The network includes 3438 proteins and 37,325 interactions for clarity. The Cytoscape organic layout was used for visualization (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Smoot <em>et al.</em> 2011</a>). The emerging clusters reflect highly connected portions of the network. These clusters coincide with the GO superslim functional categories of the proteins and are color-coded as indicated (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ashburner <em>et al.</em> 2000</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">Although budding yeast interactions were used in the training sets, the predictions were based entirely on fission yeast features. The resulting predicted interactome, therefore, offers a species-specific picture. The light green GO:transcription cluster appears to be split into two sub-clusters (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). A GO term enrichment analysis confirms the difference between the two sets of proteins: the smaller one enriched for chromatin organization and modification, and the larger one enriched for transcription and RNA biosynthesis (data not shown). This example shows how the clustering of the predicted interactome is not uniquely determined by the GO features used in the training, confirming the importance of the other features. It also indicates that our approach provides insight into the higher-order organization of biological processes within the cell that might be incompletely described by GO.</p><p id="p-52">The interaction score provided by the SVM can be exploited using two complementary approaches. First, we can use a large number of predicted interactions to explore the context of a biological pathway, aware of the presence of false positives. Alternatively, we can select a few high-confidence predictions and verify them experimentally. It should be mentioned that most of our false positives are likely to be functionally related proteins. Filtering the positive predictions for a protein of interest based on the presence of compatible interaction domains on the proteins would help to distinguish more confidently the true direct physical interactions. In the following, we used the predicted interactome to investigate the stress response sub-network.</p></div><div id="sec-27" class="subsection"><h3>Stress response sub-network</h3><p id="p-53">The cellular stress response is a universal mechanism of pivotal significance for both cellular physiology and pathology. It represents a defense reaction of cells to maintain homeostasis and survive environmental challenges. The response of fission yeast to environmental stress has been the subject of multiple studies that provide a framework to explore the predicted protein interaction network (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). We therefore focused on the neighborhood of proteins related to stress response. Any change of the environment activates a core environmental stress response (CESR) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Chen <em>et al.</em> 2003</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2008</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lopez-Maury <em>et al.</em> 2008</a>). The large gene expression response is bidirectional, inducing cellular stress defense mechanisms as well as repressing growth-related processes.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/4/453/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The stress response sub-network. (A) Known interactions of stress-related genes (from BioGRID). (B) Network in A expanded using our predictions. (C) Detail of the network for Tor1 and Tor2 kinases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670881907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The stress response sub-network. (A) Known interactions of stress-related genes (from BioGRID). (B) Network in A expanded using our predictions. (C) Detail of the network for Tor1 and Tor2 kinases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/4/453/F6.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/4/453/F6.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/4/453/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/4/453/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461400" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-54" class="first-child">The stress response sub-network. (A) Known interactions of stress-related genes (from BioGRID). (B) Network in A expanded using our predictions. (C) Detail of the network for Tor1 and Tor2 kinases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">We compiled a list of 32 proteins known or suspected to be involved in stress response from the literature. Most of these proteins are highly conserved, with some of them having multiple paralogs in budding and fission yeast. <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> shows the sub-network of fission yeast proteins that are recorded in databases (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a>) or were predicted here (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>) to interact with any of the selected 32 stress-response proteins, defining a stress response network. Besides several previously reported interactions [<em>e.g.</em> the one between Prr1 and Mpr1 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aoyama <em>et al.</em> 2001</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Nakamichi <em>et al.</em> 2003</a>)], our network contains numerous potentially new interactions (<em>e.g.</em> between Prr1 and Pka1, Cyr1 and Ssu72). As examples for how our predictions can be used for the generation of new biological hypotheses, we highlight below a few of the predicted interactions and discuss them within their biological context.</p></div><div id="sec-28" class="subsection"><h3>Predicted TOR interactions</h3><p id="p-56">The evolutionary conserved target of rapamycin (Tor) pathway mediates fundamental cellular activities, such as translation, spatial and temporal growth, and cell-cycle progression in response to nutrient signals. Fission yeast Tor1 and Tor2 proteins are serine/threonine kinases of the phosphatidylinositol kinase–related family (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Lopez-Maury <em>et al.</em> 2008</a>) that function in the multiprotein complexes TORC1 and TORC2 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bjornsti and Houghton 2004</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Petersen 2009</a>). Although the Tor signaling pathway is of great interest and subject to intense research in several model organisms, the list of known molecular players is far from complete. Proteins that were predicted to interact with both <em>S. pombe</em> Tor kinases are involved in cell shape determination; cortical patch localization and cell polarity; septum formation; vacuole targeting and sorting; endocytosis; and autophagy (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). A large number of predictions included GTPases and GTPase activators or inhibitors, such as the ras activator GAP protein Byr4 (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Song <em>et al.</em> 1996</a>). GTPases (<em>e.g.</em> of the Rho family, present in our list) are central for the regulation of spatial cytoarchitecture (polarity and growth) via regulation of cytoskeletal elements (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Perez and Rincón 2010</a>), autophagy (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mukaiyama <em>et al.</em> 2010</a>), and trafficking (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Burguete <em>et al.</em> 2008</a>). Our data raise the possibility that these GTPases may be regulated by Tor. Interestingly, the predictions also included proteins, such as Rab GTPases, that may themselves regulate Tor signaling (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Tatebe <em>et al.</em> 2010</a>). Although autophagy mechanisms have been described for <em>S. pombe</em>, our understanding of this process and its regulation has remained elusive. For example, Tor kinases are localized in distinct membrane-associated protein complexes (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Wedaman <em>et al.</em> 2003</a>), but the nature of these complexes is unknown. Using our method, Tor1 kinase was predicted to interact with Atg20 and Atg24 (sorting nexins), providing clues for the composition of such complexes. In addition, our list includes one of the two pH sensor bro1 domain proteins involved in ascospore formation and protein processing (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS7.xls">File S7</a>). These predictions thus provide an intriguing list of candidates to follow up with targeted wet-lab experiments. Given the strong conservation of the Tor pathway, our predicted interactions should also be fruitful to pursue in other organisms.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461402/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Predicted interactions for both Tor1 and Tor2 obtained with both RF and SVM (score &gt; 0.87)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h3>Connections with disease</h3><p id="p-57">The mTOR pathway in humans is an important contributor to disease in various cancers, including hematologic malignancies and neurodegenerative diseases, such as Huntington disease (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Wang <em>et al.</em> 2009b</a>), via still unclear mechanisms of suppression of autophagy and myopathies (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Risson <em>et al.</em> 2009</a>). Tor signaling is shown to be directly implicated in neurodegenerative disorders (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Wang <em>et al.</em> 2009b</a>) and may include additional interactions and pathways. Constitutive activation of mTOR is shown to sensitize cells to genomic damage (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lee <em>et al.</em> 2007</a>). A protein-protein interaction map for Tor is therefore of significance for understanding the action mechanisms of this kinase in pathological conditions. Human homologs of proteins predicted with our tool to physically interact with <em>S. pombe</em> Tor1 kinase are implicated in disease and have Online Mendelian Inheritance in Man (OMIM) entries related to various cancers, neuronal diseases or syndromes, myopathies, and porphyria (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461404/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461404/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Predicted interaction partners of the Tor kinases whose orthologs are associated with human disease</span><div class="sb-div caption-clear"></div></div></div><p id="p-58">The predicted Tor interactions with proteins that function in mitochondria might be unexpected. Although a possible relation seems unlikely, it is already reported that Tor signaling is related to mitochondrial function, and mutations that cripple Tor signaling have effects on mitochondrial respiration. Indeed, increased oxygen consumption is a signature of Tor mutants (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bonawitz <em>et al.</em> 2007</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pan and Shadel 2009</a>). Our protein-protein interaction predictions suggest relations and mechanisms that could be used for further disease studies and potential targeted drug design.</p></div><div id="sec-30" class="subsection"><h3>Predicted Mak1/2/3 interactions</h3><p id="p-59">Histidine kinases mediate a histidyl-aspartyl phosphorelay, also known as a two-component system (TCS). We predicted the <em>S. pombe</em> histidine kinases Mak1, Mak2, and Mak3 to be central in the stress response sub-network (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS7.xls">File S7</a>). Histidine kinases are reported to have important roles in bacteria, other fungal species, and plants in regulation of osmotic and oxidative stress response (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Desikan <em>et al.</em> 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Krell <em>et al.</em> 2010</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Motoyama <em>et al.</em> 2005</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Santos and Shiozaki 2001</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Urao <em>et al.</em> 2001</a>), functioning as parts of TCSs. While mammals developed complex signaling systems involving serine/threonine/tyrosine kinases for stress response, bacterial and fungal TCSs represent a simple but elegant prototype of signal transduction machineries. Due to their significance in bacterial and fungal virulence and the development of antibiotic resistance, they are of great interest in the development of antimicrobial and antifungal agents (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Rowland and King 2007</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Schreiber <em>et al.</em> 2009</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Stephenson and Hoch 2002</a>, <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">2004</a>). Fission yeast Mak2/3 have been reported to play important roles as oxidative stress sensors by mediating phosphorylation of the Atf1 transcription factor via the Sty1 kinase (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Buck <em>et al.</em> 2001</a>). However, the role of Mak1 is different and required for Pap1- and Prr1-dependent transcription (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Buck <em>et al.</em> 2001</a>). The only histidine kinase in budding yeast, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001409">Sln1</a>, plays a role as an osmosensor in response to osmotic stress (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Posas <em>et al.</em> 1996</a>). We predicted the three <em>S. pombe</em> histidine kinases to interact with 127 proteins that were enriched for signal transduction, cytokinesis, septation, RNA metabolism, and response to chemical stimuli (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS7.xls">File S7</a>). Within these 127 proteins, we found 41 genetic interactions and 37 physical interactions inferred from budding yeast (as documented in BioGRID). As many as 18 transcription factors were predicted to be interacting with the Mak proteins, 6 of them specifically with Mak2 (Atf31, Hsr1, Rsv1, Res1, Moc3, and Prr1). We conclude that the predicted stress response sub-network is a useful framework to generate specific hypotheses for this important biological process.</p></div><div id="sec-31" class="subsection"><h3>Predicted argonaute interactions</h3><p id="p-60">Cellular response to stress is highly dynamic, allowing the cells to effectively counteract diverse stresses. It is evident from mammalian studies that siRNAs are a new class of genes involved in stress response, apart from regulation of gene expression and anti-virulence action (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Babar <em>et al.</em> 2008</a>). siRNAs are of immense interest as a gene function research tool but also as active agents in a new variety of therapeutics (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Walton <em>et al.</em> 2010</a>). Interestingly, <em>S. pombe</em> has the machinery for siRNA production and processing that <em>S. cerevisiae</em> lacks and has, thus, emerged as a useful model for RNA interference: it has single-copy genes for components of the RNAi pathway, such as argonaute, dicer, and RNA-dependent RNA polymerase (RdRP) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Bühler <em>et al.</em> 2008</a>). Functions for RNAi revealed in <em>S. pombe</em>, such as heterochromatic silencing and chromosome segregation, are likely to be ancient because they are shared with other eukaryotes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Martienssen <em>et al.</em> 2005</a>). Importantly, a link between RNAi and heterochromatin integrity is reported (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bayne <em>et al.</em> 2010</a>). It is therefore interesting to examine physical interactors of proteins actively involved in this process, to gain insight on different mechanisms of gene regulation and stress response. argonaute 1 (Ago1 or eIF2c) is required for siRNA-mediated gene silencing in a range of biological models. However, only a few interactors of Ago1 are known. Using our tool, we identify a series of proteins involved in chromatin regulation as Ago1 interactors, notably numerous SAGA complex components, the histone deacetylases Hst2 and Sir2, and the chromatin silencing protein Clr2. Sir2 is involved in silencing at centromeres and telomeres (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Freeman-Cook <em>et al.</em> 2005</a>) and in processes that affect replicative lifespan (<a id="xref-ref-91-4" class="xref-bibr" href="#ref-91">Wang <em>et al.</em> 2009b</a>). Interestingly, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002200">Sir2</a> in <em>S. cerevisiae</em> is associated with the polarisome and plays an important role in segregation of damaged proteins between mother and daughter cells and the modulation of replicative lifespan (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Liu <em>et al.</em> 2010</a>), and it may have beneficial roles against neurodegeneration (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gan and Tang 2010</a>). It would be interesting to elucidate the potential link between Sir2 actions and the siRNA machinery.</p></div></div><div class="section discussion" id="sec-32"><h2 class="">DISCUSSION</h2><p id="p-61">We present the first approach to predict physical protein interactions in fission yeast based on a large number of experimental and sequence features. The expression correlation, functional characterization, and length of the interacting proteins emerge as the major factors for the predictions. The method can produce predictions even for proteins that lack functional annotation, as was demonstrated for Cbf11. Although we trained our method to identify physical interactions, the lowest-scoring predicted partners are likely to be proteins that are at least functionally interacting with the protein of interest. We used a training set that included the overlap between two very different types of evidence of interaction to reduce the number of false positives. In theory, proteins that are shown to interact both by Y2H and other capture methods should be stable and direct interactors, but we expect to find positive predictions even for unstable and co-complex interactions. Including a very coarse version the Gene Ontology in our features suggests that some of our false positives correspond to functional interactions. In fission yeast, even functional interactions are poorly characterized, especially the ones involving non-conserved proteins, such as Cbf11. Our method can hence also be used, to some extent, to explore the functional network neighborhood of uncharacterized proteins. Different training sets would need to be employed to improve the performance of this task, which is beyond the purpose of this work. A flexible use of the prediction scores should help in changing the focus from an exploratory list of possible interactions to a more conservative list. For an even stricter filter on proteins that interact physically and directly with the protein of interest, an assessment of the presence of compatible domains could be carried out on the predicted list.</p><div id="sec-33" class="subsection"><h3>The usefulness of RF and SVM predictions</h3><p id="p-62">Having used SVM and RF in all analyses allows us to compare the two techniques. There is an advantage in using both techniques as each produced unique predictions that were verified experimentally. The overlap of predictions made by SVM and RF can be used as a high-confidence set. Generally, RF produces fewer predictions than SVM, eliminating both false positives and true positives and leading to better ROC and precision-recall curves. However, according to our validation by affinity capture-MS experiments, the probabilities returned by the SVM algorithm seem to correlate weakly but significantly with the peptide counts, a useful proxy for the strength of the experimental interaction. In the case of RF, this correlation was not significant.</p></div><div id="sec-34" class="subsection"><h3>Choice of threshold for interaction score</h3><p id="p-63">An important issue in establishing a correct use of these predictions is to understand how changing the threshold affects their accuracy. To this end, we calculated our performance measures on the imbalanced sets for two additional thresholds: 0.7 and 0.9 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS2.xls">File S2</a>). For example, the accuracy for the RF method is 0.81, 0.71, and 0.75 for thresholds of 0.5, 0.7, and 0.9, respectively, whereas the SVM method records accuracies of 0.77, 0.92, and 0.97 respectively. The large difference between SVM accuracies at 0.5 and 0.7 thresholds suggests that 0.5 is probably too low a cut-off. For the RF case, on the other hand, a threshold of 0.5 effectively limits the number of false positives, as shown by a high accuracy. Notably, the score assigned by SVM to Cbf11 interactors correlates significantly with the strength of the experimental evidence for these interactions. The classification underlying the SVM provides a score which should reflect the confidence of the predictions, whereas RF results do not guarantee this. We suggest that users of our tool consider the ranking of the predictions and determine an appropriate threshold based on the purpose of the prediction. Using the data provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS2.xls">File S2</a> should further guide the interpretation of the predictions.</p></div><div id="sec-35" class="subsection"><h3>Performance in recapitulating the SAGA complex and characterizing Cbf11</h3><p id="p-64">Almost all interactions within the SAGA complex are recapitulated, and additional plausible connections with other proteins are predicted. Of course, in the case of this well-annotated complex, most of these results could be obtained by mapping the budding yeast SAGA complex members into fission yeast orthologs. However, having eliminated all SAGA proteins from the training, we demonstrated how the method can be used to reconstruct uncharacterized complexes.</p><p id="p-65">Moreover, we predicted one out of three interactions for Cbf11 that were produced as hits from an experimental approach involving affinity (TAP) purification followed by mass spectroscopy. In this way, a role in chromatin-related processes is suggested for this uncharacterized transcription factor. Many of our predicted targets that are not found in the experimental hit lists were enriched for similar specific GO processes and are likely to be at least functionally related to Cbf11. Moreover, we showed how some low-abundance proteins could be false negatives of the experiment and stressed the fact that high-abundance proteins can bias specific experimental approaches.</p></div><div id="sec-36" class="subsection"><h3>General considerations on method performance</h3><p id="p-66">There are no other predicted physical interaction networks for fission yeast, with the only available functional interactions available from STRING (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Szklarczyk <em>et al.</em> 2011</a>) likely to miss important species-specific subsets of the network. A comparison of our predictions with the current version of STRING for fission yeast shows a reassuring correlation of the STRING combined scores with our predicted scores for SVM (Spearman r &gt; 0.25) and for RF (r &gt; 0.2). We predict 67% of the pairs with a STRING score &gt; 800/1000 but only 46% of the pairs with a STRING score &gt; 200/1000, showing that our method is mostly capturing physical interactions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS8.xls">File S8</a>).</p><p id="p-67">Using interaction domain information as well as sequence features, <a id="xref-ref-95-4" class="xref-bibr" href="#ref-95">Yu <em>et al.</em> (2010)</a> predicted budding yeast interactions reporting AUCs of up to 0.77, whereas we obtained AUCs of around 0.8 without including any domain information. This could be due to our careful construction of the training sets, which included only interactions with at least two kinds of experimental evidence for physical interaction.</p><p id="p-68">Of the 15 new interactions that were added with two lines of evidence to BioGRID since the start of this project, 11 had been predicted by both our methods.</p><p id="p-69">Estimating the real size of the interactome is not trivial and by necessity renders the accurate estimation of our method’s performance subject to large error margins (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1/FileS10.pdf">File S10</a>). The low precision recorded (2%, reaching 7% when taking only the top 10% of predictions) can be attributed to this lack of knowledge and is characteristic of predicted networks, which can still be of use to guide hypothesis generation and experiments. The experimental validations presented in this article offer a practical example of the validity of the method and show possible uses of our predictions.</p></div><div id="sec-37" class="subsection"><h3>New interactions suggested by the predicted fission yeast interactome</h3><p id="p-70">To demonstrate the biological use of the obtained interactome, we discussed predicted partners of the Tor proteins, which are involved in the regulation of growth and stress response, of the histidine kinases Mak1/2/3, which are involved in sensing of environmental perturbations, and of the only argonaute protein present in fission yeast. Although such predictions for physical interactions contain inevitable false positives and false negatives, we conclude that the predicted protein network can provide a useful framework to explore biological processes, expand the functional network around a protein of interest, and develop hypotheses to guide wet-lab experiments.</p></div><div id="sec-38" class="subsection"><h3>A new resource for the fission yeast community and beyond</h3><p id="p-71">We believe that this tool will be of particular use to the yeast community to generate hypotheses and to investigate the role of yet uncharacterized proteins in fission yeast, like Cbf11, for which no physical interaction information is available and for which even functional interactions are unknown (<a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">Szklarczyk <em>et al.</em> 2011</a>). We make our predictions available online for perusal by colleagues through a user friendly network visualization tool (<a href="http://www.bahlerlab.info/PInt/">http://www.bahlerlab.info/PInt/</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-72">The authors gratefully acknowledge the contribution of Falk Schubert in the initial planning of the study, Luis López-Maury for helping with the interpretation of the interaction predictions, and Sanjay Khadayate and Fengyuan Hu for help with setting up the web interface. Moreover, the authors acknowledge the useful comments from anonymous referees. Research in the Bähler Lab was funded by Cancer Research UK and by PhenOxiGEn, an EU FP7 research project. JRM was supported by National Institutes of Health (NIH) training grant T32CA009385-25. Funding for work in the Gould Lab was provided by the Howard Hughes Medical Institute of which KLG is an investigator. A.B. and O.S.S. acknowledge funding from the Klaus Tschira Foundation and the EU FP7 (PhenOxiGEn).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001560/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-73"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-11-08"><span class="received-label">Received </span>November 8, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-01-31"><span class="accepted-label">Accepted </span>January 31, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Pancaldi <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.1" data-doi="10.1016/S0968-0004(02)02204-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aloy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The third dimension for protein interactions and complexes</span>. <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DAloy%26rft.auinit1%253DP.%26rft.volume%253D27%26rft.issue%253D12%26rft.spage%253D633%26rft.epage%253D638%26rft.atitle%253DThe%2Bthird%2Bdimension%2Bfor%2Bprotein%2Binteractions%2Band%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0968-0004%252802%252902204-1%26rft_id%253Dinfo%253Apmid%252F12468233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0968-0004(02)02204-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12468233&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180216200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.2" data-doi="10.1271/bbb.65.2347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aoyama</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aiba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizuno</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic analysis of the His-to-Asp phosphorelay implicated in mitotic cell cycle control: involvement of histidine-kinase genes of Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Biosci. Biotechnol. Biochem.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">2347</span>–<span class="cit-lpage">2352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.stitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.aulast%253DAoyama%26rft.auinit1%253DK.%26rft.volume%253D65%26rft.issue%253D10%26rft.spage%253D2347%26rft.epage%253D2352%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2BHis-to-Asp%2Bphosphorelay%2Bimplicated%2Bin%2Bmitotic%2Bcell%2Bcycle%2Bcontrol%253A%2Binvolvement%2Bof%2Bhistidine-kinase%2Bgenes%2Bof%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1271%252Fbbb.65.2347%26rft_id%253Dinfo%253Apmid%252F11758939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1271/bbb.65.2347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11758939&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.3" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.4" data-doi="10.2217/14796694.4.2.289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babar</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slack</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weidhaas</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">miRNA modulation of the cellular stress response</span>. <abbr class="cit-jnl-abbrev">Future Oncol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFuture%2BOncol%26rft.aulast%253DBabar%26rft.auinit1%253DI.%2BA.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D298%26rft.atitle%253DmiRNA%2Bmodulation%2Bof%2Bthe%2Bcellular%2Bstress%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.2217%252F14796694.4.2.289%26rft_id%253Dinfo%253Apmid%252F18407740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/14796694.4.2.289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18407740&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261698600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.5" data-doi="10.1016/j.cell.2010.01.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bayne</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kagansky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijos</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Pulido</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Stc1: a critical link between RNAi and chromatin modification required for heterochromatin integrity</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">666</span>–<span class="cit-lpage">677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBayne%26rft.auinit1%253DE.%2BH.%26rft.volume%253D140%26rft.issue%253D5%26rft.spage%253D666%26rft.epage%253D677%26rft.atitle%253DStc1%253A%2Ba%2Bcritical%2Blink%2Bbetween%2BRNAi%2Band%2Bchromatin%2Bmodification%2Brequired%2Bfor%2Bheterochromatin%2Bintegrity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.01.038%26rft_id%253Dinfo%253Apmid%252F20211136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.01.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20211136&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275197400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.6" data-doi="10.1093/bioinformatics/bti1016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Hur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Kernel methods for predicting protein-protein interactions</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">i38</span>–<span class="cit-lpage">i46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBen-Hur%26rft.auinit1%253DA.%26rft.volume%253D21%26rft.issue%253Dsuppl_1%26rft.spage%253Di38%26rft.epage%253Di46%26rft.atitle%253DKernel%2Bmethods%2Bfor%2Bpredicting%2Bprotein-protein%2Binteractions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti1016%26rft_id%253Dinfo%253Apmid%252F15961482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxNDoiMjEvc3VwcGxfMS9pMzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Hur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Choosing negative examples for the prediction of protein-protein interactions</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">S2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D7%26rft.spage%253DS2%26rft_id%253Dinfo%253Apmid%252F16723005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16723005&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.8" data-doi="10.1038/nrc1362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bjornsti</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houghton</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The TOR pathway: a target for cancer therapy</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DBjornsti%26rft.auinit1%253DM.%2BA.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D335%26rft.epage%253D348%26rft.atitle%253DThe%2BTOR%2Bpathway%253A%2Ba%2Btarget%2Bfor%2Bcancer%2Btherapy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc1362%26rft_id%253Dinfo%253Apmid%252F15122205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc1362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15122205&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221158400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.9" data-doi="10.1016/j.cmet.2007.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonawitz</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatenay-Lapointe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadel</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reduced TOR signaling extends chronological life span via increased respiration and upregulation of mitochondrial gene expression</span>. <abbr class="cit-jnl-abbrev">Cell Metab.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BMetab%26rft.aulast%253DBonawitz%26rft.auinit1%253DN.%2BD.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D265%26rft.epage%253D277%26rft.atitle%253DReduced%2BTOR%2Bsignaling%2Bextends%2Bchronological%2Blife%2Bspan%2Bvia%2Bincreased%2Brespiration%2Band%2Bupregulation%2Bof%2Bmitochondrial%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmet.2007.02.009%26rft_id%253Dinfo%253Apmid%252F17403371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmet.2007.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17403371&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245484100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.10" data-doi="10.1093/bioinformatics/bth456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gollub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">GO::TermFinder—open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3710</span>–<span class="cit-lpage">3715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth456%26rft_id%253Dinfo%253Apmid%252F15297299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTgvMzcxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.11" data-doi="10.1038/nmeth.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tasan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreze</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrios-Rodiles</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemmens</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An experimentally derived confidence score for binary protein-protein interactions</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBraun%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D91%26rft.epage%253D97%26rft.atitle%253DAn%2Bexperimentally%2Bderived%2Bconfidence%2Bscore%2Bfor%2Bbinary%2Bprotein-protein%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1281%26rft_id%253Dinfo%253Apmid%252F19060903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19060903&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262370200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.12" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random Forests</span>. <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.13" data-doi="10.1126/science.1176495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharom</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neduva</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A global protein kinase and phosphatase interaction network in yeast</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">1043</span>–<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBreitkreutz%26rft.auinit1%253DA.%26rft.volume%253D328%26rft.issue%253D5981%26rft.spage%253D1043%26rft.epage%253D1046%26rft.atitle%253DA%2BGlobal%2BProtein%2BKinase%2Band%2BPhosphatase%2BInteraction%2BNetwork%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1176495%26rft_id%253Dinfo%253Apmid%252F20489023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjgvNTk4MS8xMDQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.14" data-doi="10.1091/mbc.12.2.407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soto Pino</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saldanha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Peroxide sensors for the fission yeast stress-activated mitogen-activated protein kinase pathway</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DBuck%26rft.auinit1%253DV.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D407%26rft.epage%253D419%26rft.atitle%253DPeroxide%2BSensors%2Bfor%2Bthe%2BFission%2BYeast%2BStress-activated%2BMitogen-activated%2BProtein%2BKinase%2BPathway%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.12.2.407%26rft_id%253Dinfo%253Apmid%252F11179424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjEyLzIvNDA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.15" data-doi="10.1038/nsmb.1481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bühler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spies</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moazed</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">TRAMP-mediated RNA surveillance prevents spurious entry of RNAs into the Schizosaccharomyces pombe siRNA pathway</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DBuhler%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D10%26rft.spage%253D1015%26rft.epage%253D1023%26rft.atitle%253DTRAMP-mediated%2BRNA%2Bsurveillance%2Bprevents%2Bspurious%2Bentry%2Bof%2BRNAs%2Binto%2Bthe%2BSchizosaccharomyces%2Bpombe%2BsiRNA%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1481%26rft_id%253Dinfo%253Apmid%252F18776903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18776903&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259777000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.16" data-doi="10.1038/msb4100203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Nimwegen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accurate prediction of protein-protein interactions from sequence alignments using a Bayesian method</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DBurger%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D165%26rft.epage%253D165%26rft.atitle%253DAccurate%2Bprediction%2Bof%2Bprotein-protein%2Binteractions%2Bfrom%2Bsequence%2Balignments%2Busing%2Ba%2BBayesian%2Bmethod%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100203%26rft_id%253Dinfo%253Apmid%252F18277381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjQvMS8xNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.17" data-doi="10.1016/j.cell.2007.11.048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burguete</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fenn</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunger</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeffer</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rab and Arl GTPase family members cooperate in the localization of the golgin GCC185</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBurguete%26rft.auinit1%253DA.%2BS.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D286%26rft.epage%253D298%26rft.atitle%253DRab%2Band%2BArl%2BGTPase%2Bfamily%2Bmembers%2Bcooperate%2Bin%2Bthe%2Blocalization%2Bof%2Bthe%2Bgolgin%2BGCC185.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.11.048%26rft_id%253Dinfo%253Apmid%252F18243103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.11.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18243103&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253427700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.18" data-doi="10.1074/jbc.M602180200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espinola</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fossale</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massey</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuervo</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Autophagy is disrupted in a knock-in mouse model of juvenile neuronal ceroid lipofuscinosis</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20483</span>–<span class="cit-lpage">20493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M602180200%26rft_id%253Dinfo%253Apmid%252F16714284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjkvMjA0ODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.19" data-doi="10.1091/mbc.E02-08-0499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toone</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mata</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global transcriptional responses of fission yeast to environmental stress</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">214</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E02-08-0499%26rft_id%253Dinfo%253Apmid%252F12529438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzEvMjE0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.20" data-doi="10.1091/mbc.E07-08-0735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penkett</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toone</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple pathways differentially regulate global oxidative stress responses in fission yeast</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">308</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-08-0735%26rft_id%253Dinfo%253Apmid%252F18003976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMzA4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.21" data-doi="10.1007/BF00994018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Support-vector networks</span>. <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D20%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00994018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00994018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RX35400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.22" data-doi="10.1186/1471-2121-11-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuenca-Bono</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Molinero</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascual-Garcia</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Oliver</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llopis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A novel link between Sus1 and the cytoplasmic mRNA decay machinery suggests a broad role in mRNA metabolism</span>. <abbr class="cit-jnl-abbrev">BMC Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bcell%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BCell%2BBiol%26rft.aulast%253DCuenca-Bono%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DA%2Bnovel%2Blink%2Bbetween%2BSus1%2Band%2Bthe%2Bcytoplasmic%2BmRNA%2Bdecay%2Bmachinery%2Bsuggests%2Ba%2Bbroad%2Brole%2Bin%2BmRNA%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2121-11-19%26rft_id%253Dinfo%253Apmid%252F20230609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2121-11-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20230609&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.23" data-doi="10.1186/1471-2164-10-288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Bodt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proost</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandepoele</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouze</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting protein-protein interactions in Arabidopsis thaliana through integration of orthology, gene ontology and co-expression</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDe%2BBodt%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D288%26rft.epage%253D288%26rft.atitle%253DPredicting%2Bprotein-protein%2Binteractions%2Bin%2BArabidopsis%2Bthaliana%2Bthrough%2Bintegration%2Bof%2Borthology%252C%2Bgene%2Bontology%2Band%2Bco-expression.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-288%26rft_id%253Dinfo%253Apmid%252F19563678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19563678&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.24" data-doi="10.1371/journal.pone.0002491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desikan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaban</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mira-Rodado</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witthoft</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The histidine kinase AHK5 integrates endogenous and environmental signals in Arabidopsis guard cells</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e2491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDesikan%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De2491%26rft.epage%253De2491%26rft.atitle%253DThe%2Bhistidine%2Bkinase%2BAHK5%2Bintegrates%2Bendogenous%2Band%2Benvironmental%2Bsignals%2Bin%2BArabidopsis%2Bguard%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0002491%26rft_id%253Dinfo%253Apmid%252F18560512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0002491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18560512&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.25" data-doi="10.1073/pnas.0806261105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedyshyn</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chahwan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">16653</span>–<span class="cit-lpage">16658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806261105%26rft_id%253Dinfo%253Apmid%252F18931302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQzLzE2NjUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.26" data-doi="10.1371/journal.pone.0012083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Annibale</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinjung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coolen</span>  <span class="cit-name-given-names">A. C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraternali</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Protein networks reveal detection bias and species consistency when analysed by information-theoretic methods</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFernandes%26rft.auinit1%253DL.%2BP.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De12083%26rft.epage%253De12083%26rft.atitle%253DProtein%2Bnetworks%2Breveal%2Bdetection%2Bbias%2Band%2Bspecies%2Bconsistency%2Bwhen%2Banalysed%2Bby%2Binformation-theoretic%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012083%26rft_id%253Dinfo%253Apmid%252F20805870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20805870&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.27" data-doi="10.1101/gr.2659105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formstecher</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aresta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamburger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Protein interaction mapping: a Drosophila case study</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2659105%26rft_id%253Dinfo%253Apmid%252F15710747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzMvMzc2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.28" data-doi="10.1534/genetics.104.032714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman-Cook</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spedale</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marlett</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forsburg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Conserved locus-specific silencing functions of Schizosaccharomyces pombe sir2+</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032714%26rft_id%253Dinfo%253Apmid%252F15545655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzEyNDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.29" data-doi="10.1111/j.1474-9726.2010.00597.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">B. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sirt1’s beneficial roles in neurodegenerative diseases – a chaperonin containing TCP-1 (CCT) connection?</span> <abbr class="cit-jnl-abbrev">Aging Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">924</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAging%2Bcell%26rft.stitle%253DAging%2BCell%26rft.aulast%253DGan%26rft.auinit1%253DB.%2BQ.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D924%26rft.epage%253D929%26rft.atitle%253DSirt1%25E2%2580%2599s%2Bbeneficial%2Broles%2Bin%2Bneurodegenerative%2Bdiseases%2B-%2Ba%2Bchaperonin%2Bcontaining%2BTCP-1%2B%2528CCT%2529%2Bconnection%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1474-9726.2010.00597.x%26rft_id%253Dinfo%253Apmid%252F20569238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1474-9726.2010.00597.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20569238&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.30" data-doi="10.1038/415141a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gavin</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosche</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marzioch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional organization of the yeast proteome by systematic analysis of protein complexes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGavin%26rft.auinit1%253DA.%2BC.%26rft.volume%253D415%26rft.issue%253D6868%26rft.spage%253D141%26rft.epage%253D147%26rft.atitle%253DFunctional%2Borganization%2Bof%2Bthe%2Byeast%2Bproteome%2Bby%2Bsystematic%2Banalysis%2Bof%2Bprotein%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F415141a%26rft_id%253Dinfo%253Apmid%252F11805826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/415141a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11805826&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173159300033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.31" data-doi="10.1016/j.copbio.2010.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gavin</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kühner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recent advances in charting protein-protein interaction: mass spectrometry-based approaches</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DGavin%26rft.auinit1%253DA.%2BC.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D42%26rft.epage%253D49%26rft.atitle%253DRecent%2Badvances%2Bin%2Bcharting%2Bprotein-protein%2Binteraction%253A%2Bmass%2Bspectrometry-based%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2010.09.007%26rft_id%253Dinfo%253Apmid%252F20934865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2010.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20934865&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287840700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.32" data-doi="10.1186/gb-2006-7-11-120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">How complete are current yeast and human protein-interaction networks?</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHart%26rft.auinit1%253DG.%2BT.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253D120%26rft.epage%253D120%26rft.atitle%253DHow%2Bcomplete%2Bare%2Bcurrent%2Byeast%2Band%2Bhuman%2Bprotein-interaction%2Bnetworks%253F%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-11-120%26rft_id%253Dinfo%253Apmid%252F17147767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-11-120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17147767&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.33" data-doi="10.1111/j.1365-2443.2008.01264.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Da-Qiao</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsutsumi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikashige</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Localization of gene products using a chromosomally tagged GFP-fusion library in the fission yeast Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DHayashi%26rft.auinit1%253DA.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D225%26rft.atitle%253DLocalization%2Bof%2Bgene%2Bproducts%2Busing%2Ba%2Bchromosomally%2Btagged%2BGFP-fusion%2Blibrary%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2443.2008.01264.x%26rft_id%253Dinfo%253Apmid%252F19170768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2443.2008.01264.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19170768&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.34" data-doi="10.1101/gad.1719908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Helmlinger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marguerat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villén</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bähler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The S. pombe SAGA complex controls the switch from proliferation to sexual differentiation through the opposing roles of its subunits Gcn5 and Spt8</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3184</span>–<span class="cit-lpage">3195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHelmlinger%26rft.auinit1%253DD.%26rft.volume%253D22%26rft.issue%253D22%26rft.spage%253D3184%26rft.epage%253D3195%26rft.atitle%253DThe%2BS.%2Bpombe%2BSAGA%2Bcomplex%2Bcontrols%2Bthe%2Bswitch%2Bfrom%2Bproliferation%2Bto%2Bsexual%2Bdifferentiation%2Bthrough%2Bthe%2Bopposing%2Broles%2Bof%2Bits%2Bsubunits%2BGcn5%2Band%2BSpt8%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1719908%26rft_id%253Dinfo%253Apmid%252F19056896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIyLzIyLzMxODQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.35" data-doi="10.1038/emboj.2011.181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Helmlinger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marguerat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swaney</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Tra1 has specific regulatory roles, rather than global functions, within the SAGA co-activator complex</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2843</span>–<span class="cit-lpage">2852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DHelmlinger%26rft.auinit1%253DD.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D2843%26rft.epage%253D2852%26rft.atitle%253DTra1%2Bhas%2Bspecific%2Bregulatory%2Broles%252C%2Brather%2Bthan%2Bglobal%2Bfunctions%252C%2Bwithin%2Bthe%2BSAGA%2Bco-activator%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2011.181%26rft_id%253Dinfo%253Apmid%252F21642955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2011.181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21642955&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.36" data-doi="10.1128/MCB.25.3.1162-1172.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsdottir</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emre</span>  <span class="cit-name-given-names">N. C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyce</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">H2B ubiquitin protease Ubp8 and Sgf11 constitute a discrete functional module within the Saccharomyces cerevisiae SAGA complex</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DIngvarsdottir%26rft.auinit1%253DK.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D1162%26rft.epage%253D1172%26rft.atitle%253DH2B%2BUbiquitin%2BProtease%2BUbp8%2Band%2BSgf11%2BConstitute%2Ba%2BDiscrete%2BFunctional%2BModule%2Bwithin%2Bthe%2BSaccharomyces%2Bcerevisiae%2BSAGA%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.3.1162-1172.2005%26rft_id%253Dinfo%253Apmid%252F15657441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI1LzMvMTE2MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.37" data-doi="10.1073/pnas.061034498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozawa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A comprehensive two-hybrid analysis to explore the yeast protein interactome</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">4569</span>–<span class="cit-lpage">4574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.061034498%26rft_id%253Dinfo%253Apmid%252F11283351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC84LzQ1NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.38" data-doi="10.1126/science.1087361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kluger</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A Bayesian networks approach for predicting protein-protein interactions from genomic data</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJansen%26rft.auinit1%253DR.%26rft.volume%253D302%26rft.issue%253D5644%26rft.spage%253D449%26rft.epage%253D453%26rft.atitle%253DA%2BBayesian%2BNetworks%2BApproach%2Bfor%2BPredicting%2BProtein-Protein%2BInteractions%2Bfrom%2BGenomic%2BData%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1087361%26rft_id%253Dinfo%253Apmid%252F14564010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDIvNTY0NC80NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.39" data-doi="10.1093/nar/gkn760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaffron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Creevey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">STRING 8—a global view on proteins and their functional interactions in 630 organisms</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">D412</span>–<span class="cit-lpage">D416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn760%26rft_id%253Dinfo%253Apmid%252F18940858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNy9zdXBwbF8xL0Q0MTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.40" data-doi="10.1016/j.ceb.2010.03.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koutelou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dent</span>  <span class="cit-name-given-names">S. Y. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multiple faces of the SAGA complex</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DKoutelou%26rft.auinit1%253DE.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D374%26rft.epage%253D382%26rft.atitle%253DMultiple%2Bfaces%2Bof%2Bthe%2BSAGA%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2010.03.005%26rft_id%253Dinfo%253Apmid%252F20363118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2010.03.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20363118&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.41" data-doi="10.1146/annurev.micro.112408.134054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva-Jimenez</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guazzaroni</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bacterial sensor kinases: diversity in the recognition of environmental signals</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DKrell%26rft.auinit1%253DT.%26rft.volume%253D64%26rft.spage%253D539%26rft.epage%253D559%26rft.atitle%253DBacterial%2Bsensor%2Bkinases%253A%2Bdiversity%2Bin%2Bthe%2Brecognition%2Bof%2Benvironmental%2Bsignals.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.112408.134054%26rft_id%253Dinfo%253Apmid%252F20825354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.112408.134054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20825354&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284030600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.42" data-doi="10.1038/nature04670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cagney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Global landscape of protein complexes in the yeast Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKrogan%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D440%26rft.issue%253D7084%26rft.spage%253D637%26rft.epage%253D643%26rft.atitle%253DGlobal%2Blandscape%2Bof%2Bprotein%2Bcomplexes%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04670%26rft_id%253Dinfo%253Apmid%252F16554755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16554755&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236350400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.43" data-doi="10.1016/j.molcel.2007.03.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lackner</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beilharz</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marguerat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mata</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A network of multiple regulatory layers shapes gene expression in fission yeast</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLackner%26rft.auinit1%253DD.%2BH.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D155%26rft.atitle%253DA%2Bnetwork%2Bof%2Bmultiple%2Bregulatory%2Blayers%2Bshapes%2Bgene%2Bexpression%2Bin%2Bfission%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.03.002%26rft_id%253Dinfo%253Apmid%252F17434133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17434133&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246075400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.44" data-doi="10.1038/sj.emboj.7601900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karbowniczek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petroulakis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonenberg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Constitutive mTOR activation in TSC mutants sensitizes cells to energy starvation and genomic damage via p53</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4812</span>–<span class="cit-lpage">4823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DLee%26rft.auinit1%253DC.%2BH.%26rft.volume%253D26%26rft.issue%253D23%26rft.spage%253D4812%26rft.epage%253D4823%26rft.atitle%253DConstitutive%2BmTOR%2Bactivation%2Bin%2BTSC%2Bmutants%2Bsensitizes%2Bcells%2Bto%2Benergy%2Bstarvation%2Band%2Bgenomic%2Bdamage%2Bvia%2Bp53.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601900%26rft_id%253Dinfo%253Apmid%252F17962806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7601900&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17962806&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251282400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.45" data-doi="10.1126/science.1091403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A map of the interactome network of the metazoan C. elegans</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1091403%26rft_id%253Dinfo%253Apmid%252F14704431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1Ny81NDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.46" data-doi="10.1016/j.cell.2009.12.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Öling</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The polarisome is required for segregation and retrograde transport of protein aggregates</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLiu%26rft.auinit1%253DB.%26rft.volume%253D140%26rft.issue%253D2%26rft.spage%253D257%26rft.epage%253D267%26rft.atitle%253DThe%2Bpolarisome%2Bis%2Brequired%2Bfor%2Bsegregation%2Band%2Bretrograde%2Btransport%2Bof%2Bprotein%2Baggregates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.12.031%26rft_id%253Dinfo%253Apmid%252F20141839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.12.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20141839&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273826500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.47" data-doi="10.1038/nrg2398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Maury</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marguerat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahler</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Tuning gene expression to changing environments: from rapid responses to evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLopez-Maury%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D583%26rft.epage%253D593%26rft.atitle%253DTuning%2Bgene%2Bexpression%2Bto%2Bchanging%2Benvironments%253A%2Bfrom%2Brapid%2Bresponses%2Bto%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2398%26rft_id%253Dinfo%253Apmid%252F18591982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18591982&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257758400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.48" data-doi="10.1021/pr900360j"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Z.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dasari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Litton</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobecki</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">IDPicker 2.0: improved protein assembly with high discrimination peptide identification filtering</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3872</span>–<span class="cit-lpage">3881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DMa%26rft.auinit1%253DZ.%2BQ.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D3872%26rft.epage%253D3881%26rft.atitle%253DIDPicker%2B2.0%253A%2BImproved%2Bprotein%2Bassembly%2Bwith%2Bhigh%2Bdiscrimination%2Bpeptide%2Bidentification%2Bfiltering.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr900360j%26rft_id%253Dinfo%253Apmid%252F19522537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr900360j&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19522537&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268661800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.49" data-doi="10.1016/j.tig.2005.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martienssen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaratiegui</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">RNA interference and heterochromatin in the fission yeast Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMartienssen%26rft.auinit1%253DR.%2BA.%26rft.volume%253D21%26rft.issue%253D8%26rft.spage%253D450%26rft.epage%253D456%26rft.atitle%253DRNA%2Binterference%2Band%2Bheterochromatin%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2005.06.005%26rft_id%253Dinfo%253Apmid%252F15979194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2005.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15979194&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231209200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.50" data-doi="10.1038/nbt1222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yashiroda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamata</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMatsuyama%26rft.auinit1%253DA.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D841%26rft.epage%253D847%26rft.atitle%253DORFeome%2Bcloning%2Band%2Bglobal%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1222%26rft_id%253Dinfo%253Apmid%252F16823372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16823372&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239025100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.51" data-doi="10.1016/S1387-3806(02)00563-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyamoto</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchison</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Comparison of three directly coupled HPLC MS/MS strategies for identification of proteins from complex mixtures: single-dimension LC-MS/MS, 2-phase MudPIT, and 3-phase MudPIT</span>. <abbr class="cit-jnl-abbrev">Int. J. Mass Spectrom.</abbr> <span class="cit-vol">219</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMass%2BSpectrom.%26rft.volume%253D219%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1387-3806%252802%252900563-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1387-3806(02)00563-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000177860600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.52" data-doi="10.1016/j.fgb.2004.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Motoyama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadokura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohira</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichiishi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A two-component histidine kinase of the rice blast fungus is involved in osmotic stress response and fungicide action</span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DMotoyama%26rft.auinit1%253DT.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D200%26rft.epage%253D212%26rft.atitle%253DA%2Btwo-component%2Bhistidine%2Bkinase%2Bof%2Bthe%2Brice%2Bblast%2Bfungus%2Bis%2Binvolved%2Bin%2Bosmotic%2Bstress%2Bresponse%2Band%2Bfungicide%2Baction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2004.11.002%26rft_id%253Dinfo%253Apmid%252F15707841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2004.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15707841&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.53" data-doi="10.1016/j.febslet.2009.12.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukaiyama</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakase</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakinuma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takegawa</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Autophagy in the fission yeast Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">584</span>: <span class="cit-fpage">1327</span>–<span class="cit-lpage">1334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DMukaiyama%26rft.auinit1%253DH.%26rft.volume%253D584%26rft.issue%253D7%26rft.spage%253D1327%26rft.epage%253D1334%26rft.atitle%253DAutophagy%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2009.12.037%26rft_id%253Dinfo%253Apmid%252F20036658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2009.12.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20036658&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275738900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.54" data-doi="10.1271/bbb.67.547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamichi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanada</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aiba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aoyama</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohmiya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Characterization of the Prr1 response regulator with special reference to sexual development in Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Biosci. Biotechnol. Biochem.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.stitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.aulast%253DNakamichi%26rft.auinit1%253DN.%26rft.volume%253D67%26rft.issue%253D3%26rft.spage%253D547%26rft.epage%253D555%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BPrr1%2Bresponse%2Bregulator%2Bwith%2Bspecial%2Breference%2Bto%2Bsexual%2Bdevelopment%2Bin%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1271%252Fbbb.67.547%26rft_id%253Dinfo%253Apmid%252F12723602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1271/bbb.67.547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12723602&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadel</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Extension of chronological life span by reduced TOR signaling requires down-regulation of Sch9p and involves increased mitochondrial OXPHOS complex density</span>. <abbr class="cit-jnl-abbrev">Aging (Albany NY)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAging%2B%2528Albany%2BNY%2529%26rft.volume%253D1%26rft.spage%253D131%26rft_id%253Dinfo%253Apmid%252F20157595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20157595&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.56" data-doi="10.1039/b913876p"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pancaldi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahler</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Meta-analysis of genome regulation and expression variability across hundreds of environmental and genetic perturbations in fission yeast</span>. <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DPancaldi%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D543%26rft.epage%253D552%26rft.atitle%253DMeta-analysis%2Bof%2Bgenome%2Bregulation%2Band%2Bexpression%2Bvariability%2Bacross%2Bhundreds%2Bof%2Benvironmental%2Band%2Bgenetic%2Bperturbations%2Bin%2Bfission%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fb913876p%26rft_id%253Dinfo%253Apmid%252F20174682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/b913876p&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20174682&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274654400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.57" data-doi="10.1111/j.1742-4658.2010.07903.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patrick Walton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gredell</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Designing highly active siRNAs for therapeutic applications</span>. <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">4806</span>–<span class="cit-lpage">4813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DPatrick%2BWalton%26rft.auinit1%253DS.%26rft.volume%253D277%26rft.issue%253D23%26rft.spage%253D4806%26rft.epage%253D4813%26rft.atitle%253DDesigning%2Bhighly%2Bactive%2BsiRNAs%2Bfor%2Btherapeutic%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2010.07903.x%26rft_id%253Dinfo%253Apmid%252F21078115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2010.07903.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21078115&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.58" data-doi="10.1042/BJ20091823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rincón</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rho GTPases: regulation of cell polarity and growth in yeasts</span>. <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">426</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20091823%26rft_id%253Dinfo%253Apmid%252F20175747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDI2LzMvMjQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.59" data-doi="10.1042/BST0370273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TOR signalling regulates mitotic commitment through stress-activated MAPK and Polo kinase in response to nutrient stress</span>. <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BSociety%2BTransactions%26rft.stitle%253DBiochm.%2BSoc.%2BTrans.%26rft.aulast%253DPetersen%26rft.auinit1%253DJ.%26rft.volume%253D37%26rft.issue%253DPt%2B1%26rft.spage%253D273%26rft.epage%253D277%26rft.atitle%253DTOR%2Bsignalling%2Bregulates%2Bmitotic%2Bcommitment%2Bthrough%2Bstress-activated%2BMAPK%2Band%2BPolo%2Bkinase%2Bin%2Bresponse%2Bto%2Bnutrient%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0370273%26rft_id%253Dinfo%253Apmid%252F19143645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1042/BST0370273&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19143645&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-2.4.453.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pitre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alamgir</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumontier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehne</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-comment">Computational methods for predicting protein–protein interactions, pp. 247–267 in <em>Protein–Protein Interaction</em>, edited by M. Werther and H. Seitz. Springer, Berlin/Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.61" data-doi="10.1016/S0092-8674(00)80162-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Posas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wurgler-Murphy</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thai</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Yeast HOG1 MAP kinase cascade is regulated by a multistep phosphorelay mechanism in the SLN1-YPD1-SSK1 “two-component” osmosensor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPosas%26rft.auinit1%253DF.%26rft.volume%253D86%26rft.issue%253D6%26rft.spage%253D865%26rft.epage%253D875%26rft.atitle%253DYeast%2BHOG1%2BMAP%2Bkinase%2Bcascade%2Bis%2Bregulated%2Bby%2Ba%2Bmultistep%2Bphosphorelay%2Bmechanism%2Bin%2Bthe%2BSLN1-YPD1-SSK1%2B%2522two-component%2522%2Bosmosensor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980162-2%26rft_id%253Dinfo%253Apmid%252F8808622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80162-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8808622&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VJ44300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.62" data-doi="10.1016/j.yexcr.2008.12.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Převorovský</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grousl</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanurová</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rynes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nellen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cbf11 and Cbf12, the fission yeast CSL proteins, play opposing roles in cell adhesion and coordination of cell and nuclear division</span>. <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">315</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bcell%2Bresearch%26rft.stitle%253DExp%2BCell%2BRes%26rft.aulast%253DPrevorovsky%26rft.auinit1%253DM.%26rft.volume%253D315%26rft.issue%253D8%26rft.spage%253D1533%26rft.epage%253D1547%26rft.atitle%253DCbf11%2Band%2BCbf12%252C%2Bthe%2Bfission%2Byeast%2BCSL%2Bproteins%252C%2Bplay%2Bopposing%2Broles%2Bin%2Bcell%2Badhesion%2Band%2Bcoordination%2Bof%2Bcell%2Band%2Bnuclear%2Bdivision.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yexcr.2008.12.001%26rft_id%253Dinfo%253Apmid%252F19101542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.yexcr.2008.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19101542&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bar-Joseph</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein-Seetharaman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evaluation of different biological data and computational classification methods for use in protein interaction prediction.</span> Proteins <span class="cit-vol">63</span>: <span class="cit-fpage">490</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.64" data-doi="10.1016/j.ymgme.2008.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rakheja</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narayan</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The function of CLN3P, the Batten disease protein</span>. <abbr class="cit-jnl-abbrev">Mol. Genet. Metab.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bmetabolism%26rft.stitle%253DMol%2BGenet%2BMetab%26rft.aulast%253DRakheja%26rft.auinit1%253DD.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D274%26rft.atitle%253DThe%2Bfunction%2Bof%2BCLN3P%252C%2Bthe%2BBatten%2Bdisease%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymgme.2008.01.001%26rft_id%253Dinfo%253Apmid%252F18688960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymgme.2008.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18688960&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253603300117&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.65" data-doi="10.1038/13732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rigaut</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schevchenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A generic purification method for protein complex characterization and proteome exploration</span>. <abbr class="cit-jnl-abbrev">Nat. Biotech.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1030</span>–<span class="cit-lpage">1032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRigaut%26rft.auinit1%253DG.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D1030%26rft.epage%253D1032%26rft.atitle%253DA%2Bgeneric%2Bprotein%2Bpurification%2Bmethod%2Bfor%2Bprotein%2Bcomplex%2Bcharacterization%2Band%2Bproteome%2Bexploration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F13732%26rft_id%253Dinfo%253Apmid%252F10504710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/13732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10504710&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083015700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.66" data-doi="10.1083/jcb.200903131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Risson</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazelin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roceri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moncollin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Muscle inactivation of mTOR causes metabolic and dystrophin defects leading to severe myopathy</span>. <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200903131%26rft_id%253Dinfo%253Apmid%252F20008564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE4Ny82Lzg1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.67" data-doi="10.1083/jcb.200806044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts-Galbraith</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The SH3 domains of two PCH family members cooperate in assembly of the Schizosaccharomyces pombe contractile ring</span>. <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200806044%26rft_id%253Dinfo%253Apmid%252F19139265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE4NC8xLzExMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaft</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A comparative analysis of an orthologous proteomic environment in the yeasts Saccharomyces cerevisiae and Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Apmid%252F14617822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjc6IjMvMi8xMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.69" data-doi="10.1126/science.1162609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zofall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162609%26rft_id%253Dinfo%253Apmid%252F18818364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTkwMC80MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.70" data-doi="10.2174/138955707782331759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowland</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">G. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Histidine kinases as antimicrobial targets: prospects and pitfalls</span>. <abbr class="cit-jnl-abbrev">Mini Rev. Med. Chem.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1144</span>–<span class="cit-lpage">1154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMini%2Breviews%2Bin%2Bmedicinal%2Bchemistry%26rft.stitle%253DMini%2BRev%2BMed%2BChem%26rft.aulast%253DRowland%26rft.auinit1%253DS.%2BL.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253D1144%26rft.epage%253D1154%26rft.atitle%253DHistidine%2Bkinases%2Bas%2Bantimicrobial%2Btargets%253A%2Bprospects%2Band%2Bpitfalls.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138955707782331759%26rft_id%253Dinfo%253Apmid%252F18045218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138955707782331759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18045218&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.71" data-doi="10.1038/nature04209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rual</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatesan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirozane-Kishikawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dricot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Towards a proteome-scale map of the human protein-protein interaction network</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRual%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1173%26rft.epage%253D1178%26rft.atitle%253DTowards%2Ba%2Bproteome-scale%2Bmap%2Bof%2Bthe%2Bhuman%2Bprotein-protein%2Binteraction%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04209%26rft_id%253Dinfo%253Apmid%252F16189514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16189514&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.72" data-doi="10.1126/stke.2001.98.re1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiozaki</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Fungal histidine kinases</span>. <abbr class="cit-jnl-abbrev">Sci. STKE</abbr> <span class="cit-vol">2001</span>: <span class="cit-fpage">re1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DSantos%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D2001%26rft.issue%253D98%26rft.spage%253Dre1%26rft.epage%253Dre1%26rft.atitle%253DFungal%2BHistidine%2BKinases%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fstke.2001.98.re1%26rft_id%253Dinfo%253Apmid%252F11752677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6MTE6IjIwMDEvOTgvcmUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.73" data-doi="10.1016/j.ceb.2009.01.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Res</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matter</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Protein kinases as antibacterial targets</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DSchreiber%26rft.auinit1%253DM.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D325%26rft.epage%253D330%26rft.atitle%253DProtein%2Bkinases%2Bas%2Bantibacterial%2Btargets.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2009.01.026%26rft_id%253Dinfo%253Apmid%252F19246185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2009.01.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19246185&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.74" data-doi="10.1073/pnas.0607879104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Predicting protein-protein interactions based only on sequences information</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">4337</span>–<span class="cit-lpage">4341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0607879104%26rft_id%253Dinfo%253Apmid%252F17360525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzExLzQzMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaft</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Habermann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chromatin Central: towards the comparative proteome by accurate mapping of the yeast proteomic environment</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9(11)</span>: <span class="cit-fpage">R167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D9%252811%2529%26rft.spage%253DR167%26rft.atitle%253DCHROMATIN%2BCENTRAL%253A%2BTOWARDS%2BTHE%2BCOMPARATIVE%2BPROTEOME%2BBY%2BACCURATE%2BMAPPING%2BOF%2BTHE%2BYEAST%2BPROTEOMIC%2BENVIRONMENT%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.76" data-doi="10.1093/nar/gkq481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosur</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Struct2Net: a web service to predict protein–protein interactions using a structure-based approach</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W508</span>–<span class="cit-lpage">W515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq481%26rft_id%253Dinfo%253Apmid%252F20513650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8yL1c1MDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.77" data-doi="10.1093/bioinformatics/btq675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruscheinski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ideker</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cytoscape 2.8: new features for data integration and network visualization</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq675%26rft_id%253Dinfo%253Apmid%252F21149340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy8zLzQzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.78" data-doi="10.1083/jcb.133.6.1307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mach</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albright</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A novel suppressor of <em>ras1</em> in fission yeast, <em>byr4</em>, is a dosage-dependent inhibitor of cytokinesis</span>. <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DSong%26rft.auinit1%253DK.%26rft.volume%253D133%26rft.issue%253D6%26rft.spage%253D1307%26rft.epage%253D1319%26rft.atitle%253DA%2Bnovel%2Bsuppressor%2Bof%2Bras1%2Bin%2Bfission%2Byeast%252C%2Bbyr4%252C%2Bis%2Ba%2Bdosage-%2Bdependent%2Binhibitor%2Bof%2Bcytokinesis%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.133.6.1307%26rft_id%253Dinfo%253Apmid%252F8682866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMzMvNi8xMzA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi80LzQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">B.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatr-aryamontri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oughtred</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-comment">The BioGRID Interaction Database: 2011 update. Nucleic Acids Res. 39: D698–D704</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.80" data-doi="10.1016/j.cell.2005.08.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stelzl</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worm</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalowski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haenig</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brembeck</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A human protein-protein interaction network: a resource for annotating the proteome</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">957</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DStelzl%26rft.auinit1%253DU.%26rft.volume%253D122%26rft.issue%253D6%26rft.spage%253D957%26rft.epage%253D968%26rft.atitle%253DA%2Bhuman%2Bprotein-protein%2Binteraction%2Bnetwork%253A%2Ba%2Bresource%2Bfor%2Bannotating%2Bthe%2Bproteome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.08.029%26rft_id%253Dinfo%253Apmid%252F16169070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.08.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16169070&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232130200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.81" data-doi="10.1016/S1471-4892(02)00194-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephenson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoch</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Two-component and phosphorelay signal-transduction systems as therapeutic targets</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Pharmacol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bpharmacology%26rft.stitle%253DCurr%2BOpin%2BPharmacol%26rft.aulast%253DStephenson%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D507%26rft.epage%253D512%26rft.atitle%253DTwo-component%2Band%2Bphosphorelay%2Bsignal-transduction%2Bsystems%2Bas%2Btherapeutic%2Btargets.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1471-4892%252802%252900194-7%26rft_id%253Dinfo%253Apmid%252F12324251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1471-4892(02)00194-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12324251&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177897400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.82" data-doi="10.2174/0929867043455765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephenson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoch</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Developing inhibitors to selectively target two-component and phosphorelay signal transduction systems of pathogenic microorganisms</span>. <abbr class="cit-jnl-abbrev">Curr. Med. Chem.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bmedicinal%2Bchemistry%26rft.stitle%253DCurr%2BMed%2BChem%26rft.aulast%253DStephenson%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D765%26rft.epage%253D773%26rft.atitle%253DDeveloping%2Binhibitors%2Bto%2Bselectively%2Btarget%2Btwo-component%2Band%2Bphosphorelay%2Bsignal%2Btransduction%2Bsystems%2Bof%2Bpathogenic%2Bmicroorganisms.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F0929867043455765%26rft_id%253Dinfo%253Apmid%252F15032730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/0929867043455765&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15032730&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220240400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.83" data-doi="10.1186/1471-2105-9-307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strobl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulesteix</span>  <span class="cit-name-given-names">A.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kneib</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Augustin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeileis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Conditional variable importance for random forests</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStrobl%26rft.auinit1%253DC.%26rft.volume%253D9%26rft.spage%253D307%26rft.epage%253D307%26rft.atitle%253DConditional%2Bvariable%2Bimportance%2Bfor%2Brandom%2Bforests.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-307%26rft_id%253Dinfo%253Apmid%252F18620558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18620558&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.84" data-doi="10.1093/nar/gkq973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franceschini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D561</span>–<span class="cit-lpage">D568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq973%26rft_id%253Dinfo%253Apmid%252F21045058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8xL0Q1NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.85" data-doi="10.1021/pr0604054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabb</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MyriMatch: highly accurate tandem mass spectral peptide identification by multivariate hypergeometric analysis</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">654</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DTabb%26rft.auinit1%253DD.%2BL.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D654%26rft.epage%253D661%26rft.atitle%253DMyriMatch%253A%2Bhighly%2Baccurate%2Btandem%2Bmass%2Bspectral%2Bpeptide%2Bidentification%2Bby%2Bmultivariate%2Bhypergeometric%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr0604054%26rft_id%253Dinfo%253Apmid%252F17269722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr0604054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17269722&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243927900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.86" data-doi="10.1016/j.cub.2010.10.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatebe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morigasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murayama</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiozaki</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rab-Family GTPase regulates TOR complex 2 signaling in fission yeast</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1975</span>–<span class="cit-lpage">1982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DTatebe%26rft.auinit1%253DH.%26rft.volume%253D20%26rft.issue%253D22%26rft.spage%253D1975%26rft.epage%253D1982%26rft.atitle%253DRab-family%2BGTPase%2Bregulates%2BTOR%2Bcomplex%2B2%2Bsignaling%2Bin%2Bfission%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.10.026%26rft_id%253Dinfo%253Apmid%252F21035342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.10.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21035342&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.4.453.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuxworth</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vivancos</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvajal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-comment">The Batten disease gene CLN3 is required for the response to oxidative stress. Hum. Mol. Genet. 20: 2037–2047</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.88" data-doi="10.1038/35001009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uetz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giot</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cagney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansfield</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Judson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">403</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DUetz%26rft.auinit1%253DP.%26rft.volume%253D403%26rft.issue%253D6770%26rft.spage%253D623%26rft.epage%253D627%26rft.atitle%253DA%2Bcomprehensive%2Banalysis%2Bof%2Bprotein-protein%2Binteractions%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35001009%26rft_id%253Dinfo%253Apmid%252F10688190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35001009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10688190&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085288200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.89" data-doi="10.1126/stke.2001.109.re18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi-Shinozaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinozaki</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Plant histidine kinases: an emerging picture of two-component signal transduction in hormone and environmental responses</span>. <abbr class="cit-jnl-abbrev">Sci. STKE</abbr> <span class="cit-vol">2001</span>: <span class="cit-fpage">re18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DUrao%26rft.auinit1%253DT.%26rft.volume%253D2001%26rft.issue%253D109%26rft.spage%253Dre18%26rft.epage%253Dre18%26rft.atitle%253DPlant%2BHistidine%2BKinases%253A%2BAn%2BEmerging%2BPicture%2Bof%2BTwo-Component%2BSignal%2BTransduction%2Bin%2BHormone%2Band%2BEnvironmental%2BResponses%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fstke.2001.109.re18%26rft_id%253Dinfo%253Apmid%252F11717470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6MTM6IjIwMDEvMTA5L3JlMTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.90" data-doi="10.1073/pnas.0806427106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Uncovering the rules for protein-protein interactions from yeast genomic data</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">3752</span>–<span class="cit-lpage">3757</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806427106%26rft_id%253Dinfo%253Apmid%252F19237561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzEwLzM3NTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.91" data-doi="10.1083/jcb.200904090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lao</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">TOR-mediated autophagy regulates cell death in Drosophila neurodegenerative disease</span>. <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200904090%26rft_id%253Dinfo%253Apmid%252F19720874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE4Ni81LzcwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.92" data-doi="10.1091/mbc.E02-09-0609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wedaman</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">J.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCaffery</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Tor kinases are in distinct membrane-associated protein complexes in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1204</span>–<span class="cit-lpage">1220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E02-09-0609%26rft_id%253Dinfo%253Apmid%252F12631735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE0LzMvMTIwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-book" id="cit-2.4.453.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-comment">Schizosaccharomyces pombe comparative genomics; from sequence to systems, pp. 233–285 in <em>Comparative Genomics: Using Fungi as Models</em>, edited by P. Sunnerhagen and J. Piskur. Springer, Berlin/Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.94" data-doi="10.1126/science.1158684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yıldırım</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemmens</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatesan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-quality binary protein interaction map of the yeast interactome network</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158684%26rft_id%253Dinfo%253Apmid%252F18719252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTg5OC8xMDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzQvNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.95" data-doi="10.1093/bioinformatics/btq483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Needham</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Simple sequence-based kernels do not predict protein-protein interactions</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2610</span>–<span class="cit-lpage">2614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq483%26rft_id%253Dinfo%253Apmid%252F20801913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjAvMjYxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNC80NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-2.4.453.96" data-doi="10.1021/pr070230d"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabb</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Proteomic parsimony through bipartite graph analysis improves accuracy and transparency</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3549</span>–<span class="cit-lpage">3557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D3549%26rft.epage%253D3557%26rft.atitle%253DProteomic%2Bparsimony%2Bthrough%2Bbipartite%2Bgraph%2Banalysis%2Bimproves%2Baccuracy%2Band%2Btransparency.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr070230d%26rft_id%253Dinfo%253Apmid%252F17676885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr070230d&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676885&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F4%2F453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249371000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/4/453.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/4/449" title="FASTER MT: Isolation of Pure Populations of a and α Ascospores from Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/4/469" title="Conserved Motifs and Prediction of Regulatory Modules in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/4"><span>Volume 2 Issue 4, April 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/4.cover-source.jpg?itok=0lh-7MFD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/4/453.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o-uyRt9e7QjodPYeK_knmaWnOUGhR5QW4JEa3sJ4mak" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/4/453" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/4/453" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/4/453" class="active" data-icon-position="" data-hide-link-title="0">Predicting the Fission Yeast Protein Interaction Network</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459267" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zeFJNK71X-ZYpuV0aLnEpm6p7FSHBvasQaWvgtchq_g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/4/453.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/4/453" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-blyvx9V2qS3emCaMvXh3qBbP1ccymk0x-A-iuTGGbtQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/4/453" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/4/453.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459267" id="node459267--2" data-pisa="ggg;2/4/453" data-pisa-master="ggg;2/4/453" data-apath="/ggg/2/4/453.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting the Fission Yeast Protein Interaction Network</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vera Pancaldi</span>, <span class="highwire-citation-author" data-delta="1">Ömer S. Saraç</span>, <span class="highwire-citation-author" data-delta="2">Charalampos Rallis</span>, <span class="highwire-citation-author" data-delta="3">Janel R. McLean</span>, <span class="highwire-citation-author" data-delta="4">Martin Převorovský</span>, <span class="highwire-citation-author" data-delta="5">Kathleen Gould</span>, <span class="highwire-citation-author" data-delta="6">Andreas Beyer</span> and <span class="highwire-citation-author" data-delta="7">Jürg Bähler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">453-467;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001560 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459267--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vera Pancaldi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vera%2BPancaldi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pancaldi%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVera%2BPancaldi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:v.pancaldi@ucl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">v.pancaldi@ucl.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ömer S. Saraç </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96mer%2BS.%2BSara%C3%A7%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sara%C3%A7%20%C3%96S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596mer%2BS.%2BSara%25C3%25A7%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charalampos Rallis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charalampos%2BRallis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rallis%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharalampos%2BRallis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Janel R. McLean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Janel%2BR.%2BMcLean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLean%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJanel%2BR.%2BMcLean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin Převorovský </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BP%C5%99evorovsk%C3%BD%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C5%99evorovsk%C3%BD%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BP%25C5%2599evorovsk%25C3%25BD%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kathleen Gould </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BGould%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gould%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BGould%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Andreas Beyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BBeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BBeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jürg Bähler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%BCrg%2BB%C3%A4hler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%A4hler%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25BCrg%2BB%25C3%25A4hler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/4/453" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/4/453&amp;title=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/4/453&amp;title=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/4/453&amp;title=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/4/453&amp;text=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/4/453&amp;title=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/4/453&amp;t=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/4/453&amp;title=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/4/453&amp;title=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459267/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459267" id="node459267--3" data-pisa="ggg;2/4/453" data-pisa-master="ggg;2/4/453" data-apath="/ggg/2/4/453.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting the Fission Yeast Protein Interaction Network</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vera Pancaldi</span>, <span class="highwire-citation-author" data-delta="1">Ömer S. Saraç</span>, <span class="highwire-citation-author" data-delta="2">Charalampos Rallis</span>, <span class="highwire-citation-author" data-delta="3">Janel R. McLean</span>, <span class="highwire-citation-author" data-delta="4">Martin Převorovský</span>, <span class="highwire-citation-author" data-delta="5">Kathleen Gould</span>, <span class="highwire-citation-author" data-delta="6">Andreas Beyer</span> and <span class="highwire-citation-author" data-delta="7">Jürg Bähler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">453-467;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001560 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459267--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vera Pancaldi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vera%2BPancaldi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pancaldi%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVera%2BPancaldi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:v.pancaldi@ucl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">v.pancaldi@ucl.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ömer S. Saraç </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96mer%2BS.%2BSara%C3%A7%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sara%C3%A7%20%C3%96S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596mer%2BS.%2BSara%25C3%25A7%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charalampos Rallis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charalampos%2BRallis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rallis%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharalampos%2BRallis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Janel R. McLean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Janel%2BR.%2BMcLean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLean%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJanel%2BR.%2BMcLean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin Převorovský </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BP%C5%99evorovsk%C3%BD%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C5%99evorovsk%C3%BD%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BP%25C5%2599evorovsk%25C3%25BD%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kathleen Gould </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, Tennessee 37232</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BGould%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gould%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BGould%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Andreas Beyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular Networks and Systems Biology, Biotechnology Center, Dresden University of Technology (TU Dresden), Dresden 01307, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BBeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BBeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jürg Bähler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Evolution, and Environment and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>UCL Cancer Institute, University College London, London WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%BCrg%2BB%C3%A4hler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%A4hler%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25BCrg%2BB%25C3%25A4hler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=453&amp;rft.epage=467&amp;rft.atitle=Predicting%20the%20Fission%20Yeast%20Protein%20Interaction%20Network&amp;rft.volume=2&amp;rft.issue=4&amp;rft.date=2012-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Pancaldi%2C+Vera&amp;rft.au=Sara%C3%A7%2C+%C3%96mer+S.&amp;rft.au=Rallis%2C+Charalampos&amp;rft.au=McLean%2C+Janel+R.&amp;rft.au=P%C5%99evorovsk%C3%BD%2C+Martin&amp;rft.au=Gould%2C+Kathleen&amp;rft.au=Beyer%2C+Andreas&amp;rft.au=B%C3%A4hler%2C+J%C3%BCrg"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459267/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459267/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459267/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459267/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459267/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459267/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459267/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459267/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459267/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459267/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459267/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459267/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/4/453" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MATERIALS AND METHODS</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-32" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/4/453.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/4/453.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/4/453.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/4/453.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
